<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3155193/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>GWAS Findings for Human Iris Patterns: Associations with Variants in Genes that Influence Normal Neuronal Pattern Development</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3155193/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="GWAS Findings for Human Iris Patterns: Associations with Variants in Genes that Influence Normal Neuronal Pattern Development"><meta name="citation_authors" content="Mats Larsson, David&nbsp;L. Duffy, Gu Zhu, Jimmy&nbsp;Z. Liu, Stuart Macgregor, Allan&nbsp;F. McRae, Margaret&nbsp;J. Wright, Richard&nbsp;A. Sturm, David&nbsp;A. Mackey, Grant&nbsp;W. Montgomery, Nicholas&nbsp;G. Martin, Sarah&nbsp;E. Medland"><meta name="citation_date" content="12 August 2011"><meta name="citation_issue" content="2"><meta name="citation_volume" content="89"><meta name="citation_firstpage" content="334"><meta name="citation_doi" content="10.1016/j.ajhg.2011.07.011"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3155193/?report=abstract"><meta name="citation_pmid" content="21835309"><meta name="DC.Title" content="GWAS Findings for Human Iris Patterns: Associations with Variants in Genes that Influence Normal Neuronal Pattern Development"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Mats Larsson"><meta name="DC.Contributor" content="David&nbsp;L. Duffy"><meta name="DC.Contributor" content="Gu Zhu"><meta name="DC.Contributor" content="Jimmy&nbsp;Z. Liu"><meta name="DC.Contributor" content="Stuart Macgregor"><meta name="DC.Contributor" content="Allan&nbsp;F. McRae"><meta name="DC.Contributor" content="Margaret&nbsp;J. Wright"><meta name="DC.Contributor" content="Richard&nbsp;A. Sturm"><meta name="DC.Contributor" content="David&nbsp;A. Mackey"><meta name="DC.Contributor" content="Grant&nbsp;W. Montgomery"><meta name="DC.Contributor" content="Nicholas&nbsp;G. Martin"><meta name="DC.Contributor" content="Sarah&nbsp;E. Medland"><meta name="DC.Date" content="2011 Aug 12"><meta name="DC.Identifier" content="10.1016/j.ajhg.2011.07.011"><meta name="DC.Language" content="en"><meta property="og:title" content="GWAS Findings for Human Iris Patterns: Associations with Variants in Genes that Influence Normal Neuronal Pattern Development"><meta property="og:type" content="article"><meta property="og:description" content="Human iris patterns are highly variable. The origins of this variation are of interest in the study of iris-related eye diseases and forensics, as well as from an embryological developmental perspective, with regard to their possible relationship to fundamental ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3155193/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3155193/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3155193/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A8EB8DD52720100000000074204C9.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3155193%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/199276/">v.89(2); 2011 Aug 12</a></li><li class="accid">PMC3155193</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2011 Aug 12; 89(2): 334–343. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2011.07.011" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2011.07.011</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3155193</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21835309">21835309</a></div></div></div></div><h1 class="content-title">GWAS Findings for Human Iris Patterns: Associations with Variants in Genes that Influence Normal Neuronal Pattern Development</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Larsson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960089577888" co-class="co-affbox">Mats Larsson</a>,<sup>1,</sup><sup>2,</sup><sup>∗</sup> <a href="/pubmed/?term=Duffy%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960091229568" co-class="co-affbox">David&nbsp;L. Duffy</a>,<sup>1</sup> <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960091227584" co-class="co-affbox">Gu Zhu</a>,<sup>1</sup> <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960165972624" co-class="co-affbox">Jimmy&nbsp;Z. Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Macgregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960092331328" co-class="co-affbox">Stuart Macgregor</a>,<sup>1</sup> <a href="/pubmed/?term=McRae%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960090329568" co-class="co-affbox">Allan&nbsp;F. McRae</a>,<sup>1</sup> <a href="/pubmed/?term=Wright%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960090327696" co-class="co-affbox">Margaret&nbsp;J. Wright</a>,<sup>1</sup> <a href="/pubmed/?term=Sturm%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960095092800" co-class="co-affbox">Richard&nbsp;A. Sturm</a>,<sup>3</sup> <a href="/pubmed/?term=Mackey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960095583568" co-class="co-affbox">David&nbsp;A. Mackey</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Montgomery%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960083529456" co-class="co-affbox">Grant&nbsp;W. Montgomery</a>,<sup>1</sup> <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960083527584" co-class="co-affbox">Nicholas&nbsp;G. Martin</a>,<sup>1</sup> and  <a href="/pubmed/?term=Medland%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309" class="affpopup" co-rid="_co_idm139960087778512" co-class="co-affbox">Sarah&nbsp;E. Medland</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139960089577888"><h3 class="no_margin">Mats Larsson</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><p><sup>2</sup>Center for Developmental Research, School of Law, Psychology and Social Work, Örebro University, 701 82 Örebro, Sweden</p><div>Find articles by <a href="/pubmed/?term=Larsson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Mats Larsson</a></div></div><div id="_co_idm139960091229568"><h3 class="no_margin">David&nbsp;L. Duffy</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Duffy%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">David&nbsp;L. Duffy</a></div></div><div id="_co_idm139960091227584"><h3 class="no_margin">Gu Zhu</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Gu Zhu</a></div></div><div id="_co_idm139960165972624"><h3 class="no_margin">Jimmy&nbsp;Z. Liu</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Jimmy&nbsp;Z. Liu</a></div></div><div id="_co_idm139960092331328"><h3 class="no_margin">Stuart Macgregor</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Macgregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Stuart Macgregor</a></div></div><div id="_co_idm139960090329568"><h3 class="no_margin">Allan&nbsp;F. McRae</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=McRae%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Allan&nbsp;F. McRae</a></div></div><div id="_co_idm139960090327696"><h3 class="no_margin">Margaret&nbsp;J. Wright</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Wright%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Margaret&nbsp;J. Wright</a></div></div><div id="_co_idm139960095092800"><h3 class="no_margin">Richard&nbsp;A. Sturm</h3><p><sup>3</sup>Institute for Molecular Bioscience, The University of Queensland, Brisbane 4072, Australia</p><div>Find articles by <a href="/pubmed/?term=Sturm%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Richard&nbsp;A. Sturm</a></div></div><div id="_co_idm139960095583568"><h3 class="no_margin">David&nbsp;A. Mackey</h3><p><sup>4</sup>Centre for Eye Research Australia, University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne 3002, Australia</p><p><sup>5</sup>Lions Eye Institute, University of Western Australia, Centre for Ophthalmology and Visual Science, Perth 6009, Australia</p><div>Find articles by <a href="/pubmed/?term=Mackey%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">David&nbsp;A. Mackey</a></div></div><div id="_co_idm139960083529456"><h3 class="no_margin">Grant&nbsp;W. Montgomery</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Grant&nbsp;W. Montgomery</a></div></div><div id="_co_idm139960083527584"><h3 class="no_margin">Nicholas&nbsp;G. Martin</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Martin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Nicholas&nbsp;G. Martin</a></div></div><div id="_co_idm139960087778512"><h3 class="no_margin">Sarah&nbsp;E. Medland</h3><p><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Medland%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21835309">Sarah&nbsp;E. Medland</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139960091859312_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139960091859312_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139960091859312_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139960091859312_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Queensland Institute of Medical Research, Brisbane 4029, Australia</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Center for Developmental Research, School of Law, Psychology and Social Work, Örebro University, 701 82 Örebro, Sweden</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Institute for Molecular Bioscience, The University of Queensland, Brisbane 4072, Australia</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Centre for Eye Research Australia, University of Melbourne, Royal Victorian Eye and Ear Hospital, Melbourne 3002, Australia</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Lions Eye Institute, University of Western Australia, Centre for Ophthalmology and Visual Science, Perth 6009, Australia</div><div><span class="contrib-email" id="idm139960089577888">Mats Larsson: <a href="mailto:dev@null" data-email="es.uro@nossral.stam" class="oemail">es.uro@nossral.stam</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="es.uro@nossral.stam" class="oemail">es.uro@nossral.stam</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139960091859312_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Feb 15; Revised 2011 Jul 15; Accepted 2011 Jul 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139960091859312_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2011 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3155193/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Five Figures and 15 Tables</div><div><a href="/pmc/articles/PMC3155193/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (439K)</span></div><div class="small guid">GUID:&nbsp;7E623EBF-2498-4D1B-8B50-250444AD07F4</div></div></dd></dl></div><div id="idm139960091357392" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960091357392title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Human iris patterns are highly variable. The origins of this variation are of interest in the study of iris-related eye diseases and forensics, as well as from an embryological developmental perspective, with regard to their possible relationship to fundamental processes of neurodevelopment. We have performed genome-wide association scans on four iris characteristics (crypt frequency, furrow contractions, presence of peripupillary pigmented ring, and number of nevi) in three Australian samples of European descent. Both the discovery (n = 2121) and replication (n = 499 and 73) samples showed evidence for association between (1) crypt frequency and variants in the axonal guidance gene SEM<em>A3A</em> (p = 6.6&nbsp;× 10<sup>−11</sup>), (2) furrow contractions and variants within the cytoskeleton gene <em>TRAF3IP1</em> (p&nbsp;= 2.3&nbsp;× 10<sup>−12</sup>), and (3) the pigmented ring and variants in the well-known pigmentation gene <em>SLC24A4</em> (p = 7.6&nbsp;× 10<sup>−21</sup>). These replicated findings individually accounted for around 1.5%–3% of the variance for these iris characteristics. Because both SEMA3A and TRAFIP1 are implicated in pathways that control neurogenesis, neural migration, and synaptogenesis, we also examined the evidence of enhancement among such genes, finding enrichment for crypts and furrows. These findings suggest that genes involved in normal neuronal pattern development may also influence tissue structures in the human iris.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Main Text</h2><p id="__p2" class="p p-first">Using genotyped SNPs to predict complex iris phenotypes<sup><a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353344">1</a></sup> has proved to be very successful in studies with European samples. So far, 48 SNPs in 11 different regions have been significantly associated with eye (iris) color, and a combination of six of these is able to predict blue-, green/hazel-, and brown-colored eyes with an overall accuracy of 91%, 72%, and 93%, respectively,<sup><a href="#bib2" rid="bib2 bib3 bib4" class=" bibr popnode">2–4</a></sup> demonstrating feasibility for forensic applications.<sup><a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353342">5</a></sup> Furthermore, pigmentation SNPs in <em>ASIP</em> (MIM <a href="/omim/600201" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600201</a>), <em>TYR</em> (MIM <a href="/omim/606933" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606933</a>), <em>TYRP1</em> (MIM <a href="/omim/115501" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">115501</a>), <em>MC1R</em> (MIM <a href="/omim/155555" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">155555</a>), <em>OCA2</em> (MIM <a href="/omim/611409" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">611409</a>), and <em>SLC45A2</em> (MIM <a href="/omim/606202" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606202</a>) account for 12% of the familial risk of malignant melanoma (MIM <a href="/omim/155600" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">155600</a>) in sun-exposed populations of European decent.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337046157">6</a></sup> However,&nbsp;very little is known about genes that influence iris characteristics other than eye color. These additional phenotypes are worthy of attention, because the embryological origin of the iris makes it ideal for studying genes that are expressed early in development of the neural crest and neural ectoderm,<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353356">7</a></sup> believed to be of importance for both ophthalmological and neurological diseases. For example, mutations in <em>PAX6</em> (MIM <a href="/omim/607108" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607108</a>) have been found to cause aniridia (partial or complete loss of the iris)<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353353">8</a></sup> (MIM <a href="/omim/106210" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">106210</a>) as well as neural loss in the frontal lobe,<sup><a href="#bib9" rid="bib9 bib10" class=" bibr popnode">9,10</a></sup> and tissue markers in the iris have been associated with neurological diseases, including Down syndrome<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> (MIM <a href="/omim/190685" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">190685</a>), neurofibromatosis type 1<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353358">12</a></sup> (MIM <a href="/omim/162200" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">162200</a>), and Gillespie syndrome<sup><a href="#bib13" rid="bib13 bib14" class=" bibr popnode">13,14</a></sup> (MIM <a href="/omim/206700" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">206700</a>), suggesting that development of the iris and the brain may be linked. Dysfunctional development of the cell layers in the iris also contributes to ocular melanoma<sup><a href="#bib15" rid="bib15 bib16" class=" bibr popnode">15,16</a></sup> (MIM <a href="/omim/155720" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">155720</a>), glaucoma<sup><a href="#bib17" rid="bib17 bib18" class=" bibr popnode">17,18</a></sup> (MIM <a href="/omim/137750" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">137750</a>, <a href="/omim/231300" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">231300</a>, <a href="/omim/137760" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">137760</a>], pigment dispersion (MIM <a href="/omim/600510" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600510</a>), pseudoexfoliation syndrome<sup><a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353348">19</a></sup> (MIM <a href="/omim/177650" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">177650</a>) and Axenfeld-Rieger syndrome<sup><a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353336">18</a></sup> (MIM <a href="/omim/62482" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">62482</a>, <a href="/omim/80500" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">80500</a>) (other iris-related eye diseases are listed in NIEBank). We used a GWAS approach to identify genes associated with texture and pigment characteristics of the human iris and to explore the involvement of these genes in pathways for normal neurodevelopment.</p><p id="__p3">We conducted a multistage study of 2693 individuals to detect genetic variation in four general iris characteristics. The discovery sample, consisting of 1652 individuals (age range 9–23 years, mean 13.23, SD 1.83) from the Brisbane Adolescent Twin Study (BATS) and 469 individuals (age range 5–50 years, mean 17.28, SD 10.49) from the Twin Eye Study in Tasmania (TEST). We validated the results from the discovery sample, using two independent replication samples. We directly genotyped the nine most promising SNPs in an independent replication sample of 499 Australian adolescent twins and siblings (age range 9–20 years, mean 13.50, SD 1.96), and conducted in&nbsp;silico replication in a second independent GWAS sample in which phenotypes were available for 73 individuals (age range 5–19 years, mean 10.76, SD 4.14). A full description of the Australian twin cohorts is given elsewhere.<sup><a href="#bib20" rid="bib20 bib21" class=" bibr popnode">20,21</a></sup> All three samples were drawn from the general population. All participants provided written informed consent, and the studies were approved by the respective local research ethics committees. An overview of the samples is presented in <a href="/pmc/articles/PMC3155193/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p45">Overview of Discovery and Replication Twin-Family Samples Used for GWAS of Iris Pattern Phenotypes and of the Reduced Samples of Independent Subjects used for Polygenic Prediction Analyses</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960094854672" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Sample</strong></th><th rowspan="1" colspan="1"><strong>MZ Twins</strong></th><th rowspan="1" colspan="1"><strong>DZ Twins</strong></th><th rowspan="1" colspan="1"><strong>Other Sibling</strong></th><th rowspan="1" colspan="1"><strong>N of Families</strong></th><th rowspan="1" colspan="1"><strong>Mean Family Size</strong></th><th rowspan="1" colspan="1"><strong>Effective Sample Size</strong></th><th rowspan="1" colspan="1"><strong>No. of SNPs Tested</strong></th></tr></thead><tbody><tr><td colspan="8" rowspan="1"><strong>GWAS and VEGAS Analyses</strong></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Discovery Sample: BATS</td><td rowspan="1" colspan="1">500</td><td rowspan="1" colspan="1">907</td><td rowspan="1" colspan="1">245</td><td rowspan="1" colspan="1">745</td><td rowspan="1" colspan="1">2.21</td><td rowspan="1" colspan="1">1652</td><td rowspan="1" colspan="1">2,383,268<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">Discovery Sample: TEST</td><td rowspan="1" colspan="1">190</td><td rowspan="1" colspan="1">255</td><td rowspan="1" colspan="1">24</td><td rowspan="1" colspan="1">244</td><td rowspan="1" colspan="1">1.92</td><td rowspan="1" colspan="1">469</td><td rowspan="1" colspan="1">2,383,268<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">Replication Sample 1</td><td rowspan="1" colspan="1">127</td><td rowspan="1" colspan="1">242</td><td rowspan="1" colspan="1">130</td><td rowspan="1" colspan="1">222</td><td rowspan="1" colspan="1">2.24</td><td rowspan="1" colspan="1">499</td><td rowspan="1" colspan="1">11<sup>c</sup></td></tr><tr><td rowspan="1" colspan="1">Replication Sample 2<sup>d</sup></td><td rowspan="1" colspan="1">29</td><td rowspan="1" colspan="1">27</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">41</td><td rowspan="1" colspan="1">1.78</td><td rowspan="1" colspan="1">73</td><td rowspan="1" colspan="1">9<sup>b</sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td colspan="8" rowspan="1"><strong>Polygenic Prediction Analyses</strong><sup>a</sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Discovery Sample: BATS</td><td rowspan="1" colspan="1">251</td><td rowspan="1" colspan="1">440</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">691</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">691</td><td rowspan="1" colspan="1">383<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">Discovery Sample: TEST</td><td rowspan="1" colspan="1">96</td><td rowspan="1" colspan="1">133</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">229</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">229</td><td rowspan="1" colspan="1">383<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1">Replication Sample 2<sup>d</sup></td><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">-</td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">36</td><td rowspan="1" colspan="1">383<sup>b</sup></td></tr></tbody></table></div><div id="largeobj_idm139960094854672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3155193/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139960089540096"><p id="__p46" class="p p-first-last">MZ, monozygotic; DZ, dizygotic.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>Conducted with the use of only one person per family.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Both genotyped and imputed SNPs, present in HapMap I+II (r22b36).</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>c</sup>Genotyped SNPs only.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>d</sup>These individuals entered the study after the genotyping of BATS and TEST had finished.</div></div></div><p id="__p4">Genotyping was performed with Illumina HumanHap 610W Quad arrays. Genotypic data were screened through a series of quality control criteria, including Mendelian errors, minor allele frequency (MAF) ≥ 1%, p-value of a Hardy-Weinberg equilibrium (HWE) test ≥ 10<sup>−6</sup>, SNP call rate &gt; 95%, and Illumina Beadstudio GenCall score ≥ 0.7. We screened the genotypic data for ancestral outliers by using principal-component analysis,<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353339">22</a></sup> comparing the genotyped data in the discovery sample with 16 global populations sourced from HapMap Phase 3 and Northern European subpopulations from a previous study.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353357">23</a></sup> Approximately 2% of the samples were excluded as ancestral outliers. As described elsewhere,<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353359">24</a></sup> the genotypic data were imputed to HapMap CEU I+II (release 22, build 36) with the use of MACH with a set of 274,604 SNPs that were common to the seven waves of genotyping available when this project began at QIMR.<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353349">24</a></sup> The same quality control pipeline was used to clean the genotypic data of the second replication sample, resulting in a set of 469,117 SNPs that were used for imputation via the same procedures that were used in the discovery sample. After quality control (r<sup>2</sup> ≥ 0.3; MAF ≥ 0.01; HWE p &gt; 10<sup>−6</sup>), the imputed data in the discovery sample and in replication sample 2 contained 2,383,268 and 2,379,931 SNPs, respectively.</p><p id="__p5">The phenotypes used within this study were derived from close-up high-resolution digital color photos of the iris. Four ordinal scales, depicted in <a href="/pmc/articles/PMC3155193/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figures 1–4</span></a>, were used by M.L. to rate the iris characteristics of interest: crypt frequency (<a href="/pmc/articles/PMC3155193/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>) measures different degrees of hypoplasia in the two top cell layers in the iris (i.e., anterior border layer and underlying stroma), the extension and evenness of the peripupillary pigmented ring (<a href="/pmc/articles/PMC3155193/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>) indexes the amount of melanin that circumscribes the pupil, the distinction and extension of furrow contractions (<a href="/pmc/articles/PMC3155193/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>) is a measure of the overall thickness of the iris, and frequency of iris nevi (<a href="/pmc/articles/PMC3155193/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>) is a measure of the number of melanin accumulations on the anterior border layer. The scale construction, the rating procedure, and the interrater reliability of the rating procedure have been reported in detail elsewhere.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353360">25</a></sup> The photos of each participant's right and left iris were viewed simultaneously by the rater during the rating procedure. However because the polychoric correlations for the iris characteristics from the right or left iris ranged from 0.91 to 0.95 (<a href="#app2" rid="app2" class=" sec">Table S1</a> available online), only ratings for the right iris were used in the genetic analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3155193/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960088482448" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC3155193/bin/gr1.jpg"></div></a><div id="largeobj_idm139960088482448" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3155193/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC3155193/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3155193/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p35">Crypt Frequency</p><p id="__p36">Crypt frequency measures the degree of hypoplasia in the two top cell layers in the iris; i.e., the anterior border layer and the underlying stroma. The arrows indicate examples of crypt markings of interest in the iris tissue. The categories above depict (A) only parallel and/or densely packed curly fibers, (B) spots of wavy fibers and 1–3 shallow crypts, (C) at least four shallow crypts, (D) at least five marked crypts and/or a tissue texture that is more permeable than in category 3, (E) at least four large crypts that extend at least 2/3 of the distance between the collarette and the periphery of the iris.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3155193/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960092439024" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC3155193/bin/gr2.jpg"></div></a><div id="largeobj_idm139960092439024" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3155193/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC3155193/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3155193/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p37">Extension and Evenness of the Pigmented Ring</p><p id="__p38">The pigmented ring measures the amount of melanin that surrounds the pupil and distinguishes different shades of green and hazel eye color (91% of people judged to have a green or hazel eye color by a distant observer in this sample had a pigmented ring). The categories above depict (A) the absence of a pigmented ring; (B) an uneven pigmented ring, covering 18–348 degrees; (C) an even pigmented ring, covering &gt; 348 degrees.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3155193/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960092891152" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC3155193/bin/gr3.jpg"></div></a><div id="largeobj_idm139960092891152" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3155193/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC3155193/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3155193/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p39">Distinction and Extension of Furrow Contractions</p><p id="__p40">Furrows manifest as the result of a tendency of the iris to fold in exactly the same location each time the iris adapts to different light conditions. They relate to the overall thickness and density of the iris (thicker irises have more extended furrows). The arrows indicate example of furrows on the iris. The categories above depict (A) the absence of furrows, or short furrows extending &lt; 90 degrees; (B) furrows extending between 90 and 288 degrees; (C) distinct furrows, extending at least 288 degrees; (D) more marked furrows than in category 3, extending at least 324 degrees; (E) at least two concentric furrows, with a primary furrow of &gt; 324 degrees and a second furrow of&nbsp;&gt; 288 degrees.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC3155193/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960088275568" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" src="/pmc/articles/PMC3155193/bin/gr4.jpg"></div></a><div id="largeobj_idm139960088275568" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3155193/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a target="object" href="/pmc/articles/PMC3155193/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC3155193/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p41">Frequency of Iris Nevi</p><p id="__p42">Iris nevi are melanin accumulations on the anterior border layer. The arrows point to examples of iris nevi. The categories above depict (A) the absence of iris nevi, (B) at least one iris nevus, (C) at least three iris nevi, (D) at least five iris nevi.</p></div></div></div><p id="__p6">To establish the test-retest reliabilities of the ratings in this sample, a representative sample of 400 iris photos of crypts, furrows, and nevi were rated twice by the first author, five weeks apart, the second rating blind to the first. For the pigment ring, a randomized subsample of 982 iris photos was rated independently by M.L. and R.A.S. The polychoric test-retest and interobserver reliabilities ranged from 0.91 to 0.96 (<a href="#app2" rid="app2" class=" sec">Table S1</a>). The distributions of the iris characteristics in the discovery and replication samples are presented in <a href="#app2" rid="app2" class=" sec">Table S2</a>. Because relatively few participants had iris characteristics that corresponded with the highest categories, the data were recoded to produce three categories for crypts and furrows and two for iris nevi.</p><p id="__p7">The phenotypic correlations between the iris characteristics for males and females are shown in <a href="#app2" rid="app2" class=" sec">Table S3</a>. For estimation of the heritability of the iris characteristics in our sample, heritability analyses were undertaken in the context of a classical twin design with the use of standard structural equation modeling techniques for analysis of the data from monozygotic (MZ) and dizygotic (DZ) twin pairs in the discovery sample.<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> To enable the analysis of the ordinal data, we used a multifactorial threshold model, which assumes that the observed data reflect an underlying continuous distribution that cannot be quantified but arises from the sum of numerous factors of small effect. Under this model, the ordered classes are parameterized as thresholds required to divide a standard normal distribution into categories with cumulative probability equal to the frequency of the categories in the observed data. The best-fitting model incorporating additive (A) and nonadditive (D) genetic and individual environment (E) variance components are presented in <a href="#app2" rid="app2" class=" sec">Table S4</a>. The broad-sense heritabilities for crypt, furrows, pigmented ring, and iris nevi were 84%, 87%, 93%, and 71%, respectively, somewhat higher than previously reported for these scales in a German twin sample.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353350">25</a></sup> Furthermore, age and sex differences were observed for the threshold estimates: males scored higher than females on crypts, but they received somewhat lower scores on nevi, furrows, and the pigmented ring than females. Despite the fairly narrow age range of our sample (87.6% of the sample were between 10 and 16 years), older participants also tend to receive higher scores on nevi and furrows than younger participants, but no age differences were found for the other iris characteristics.</p><p id="__p8">GWAS analyses in the discovery stage were conducted with the use of quality-controlled hard-call genotypes in MERLIN (fastassoc),<sup><a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353347">27</a></sup> which controls for family structure and zygosity by modeling kinship within each family.<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353340">28</a></sup> We included sex, age, eye color, and the six SNPs in <em>HERC2</em> (MIM <a href="/omim/605837" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605837</a>) and <em>OCA2</em> that have been shown to be most strongly associated with eye color (rs12913832, rs7183532, rs7495174, rs11852452, rs7179994, rs7183877) as covariates in the GWAS analysis.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337046158">6</a></sup></p><p id="__p9">In the second stage of the study, we examined evidence for replication by genotyping the nine most promising SNPs in an independent replication sample (replication1) and through in&nbsp;silico replication of a small independent GWAS sample. For both replication samples, the analyses were conducted with MERLIN. The results from the three samples were combined using an inverse standard-error-weighted meta-analysis within METAL.<sup><a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353355">29</a></sup></p><p id="__p10">The results for the strongest association signals in the discovery sample as well as the result in the replication samples and the meta-analysis are presented in <a href="/pmc/articles/PMC3155193/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>. The third strongest association signal for crypts in the discovery sample, rs10235789 (7q21.11; p = 2.7&nbsp;× 10<sup>−6</sup>), was supported in both replication samples (p = 2.0&nbsp;× 10<sup>−4</sup> and p = 2.5&nbsp;× 10<sup>−3</sup>), with meta-analysis of the three samples yielding a p value of 6.6&nbsp;× 10<sup>−11</sup> (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>). rs10235789 is located in intron 6 of SEM<em>A3A</em>, which has previously been associated with schizophrenia<sup><a href="#bib30" rid="bib30 bib31" class=" bibr popnode">30,31</a></sup> (MIM <a href="/omim/181500" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">181500</a>) and the progression of Alzheimer disease<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353341">32</a></sup> (MIM <a href="/omim/104300" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">104300</a>). SEM<em>A3A</em> inhibits the migration of the neural crest cells that grow into the periocular region to form the pupillary membrane, which later develops into the anterior border layer and stroma in the iris, these being the cell layers where crypts occur.<sup><a href="#bib7" rid="bib7 bib33" class=" bibr popnode">7,33</a></sup> Thus, it is likely that SEM<em>A3A</em>, which in the brain influences apical dendritic growth and can function as either a chemorepulsive or chemoattractive agent for axonal outgrowth,<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353352">34</a></sup> also influences processes that contribute to the formation of crypts in the human iris. About 1.5% of the variance for crypts could be explained by the replicated SNP findings on 7q21.11 (rs10235789). A number of other promising association signals were observed in the discovery sample (rs13140875, rs205611, rs10874518). However, these results were not replicated in the other two samples.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p51">Results for the Strongest Association Signals in the Discovery Sample, Association in Replication Sample 1 and Replication Sample 2, and the Meta-Analyses of These Samples</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960095398336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Sample</strong></th><th colspan="3" rowspan="2"><strong>MAF Alleles (Effect Null)</strong></th><th colspan="4" rowspan="1"><strong>Crypts</strong><hr></th><th colspan="4" rowspan="1"><strong>Furrows</strong><hr></th><th colspan="4" rowspan="1"><strong>Pigmented Ring</strong><hr></th><th colspan="4" rowspan="1"><strong>Iris Nevi</strong><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>P</strong></th><th rowspan="1" colspan="1"><strong>β</strong></th><th rowspan="1" colspan="1"><strong>SE</strong></th><th rowspan="1" colspan="1"><strong>r<sup>2</sup> (%)</strong></th><th rowspan="1" colspan="1"><strong>P</strong></th><th rowspan="1" colspan="1"><strong>β</strong></th><th rowspan="1" colspan="1"><strong>SE</strong></th><th rowspan="1" colspan="1"><strong>r<sup>2</sup> (%)</strong></th><th rowspan="1" colspan="1"><strong>P</strong></th><th rowspan="1" colspan="1"><strong>β</strong></th><th rowspan="1" colspan="1"><strong>SE</strong></th><th rowspan="1" colspan="1"><strong>r<sup>2</sup> (%)</strong></th><th rowspan="1" colspan="1"><strong>P</strong></th><th rowspan="1" colspan="1"><strong>β</strong></th><th rowspan="1" colspan="1"><strong>SE</strong></th><th rowspan="1" colspan="1"><strong>r<sup>2</sup> (%)</strong></th></tr></thead><tbody><tr><td colspan="20" rowspan="1"><strong>rs13140875 (Closest Gene: ANXA10)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">9.7&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.55</td><td rowspan="1" colspan="1">0.100</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.270</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.310</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.06</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.85</td><td rowspan="1" colspan="1">0.700</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.870</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.190</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.36</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.292</td><td rowspan="1" colspan="1">−0.18</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">2.51</td><td rowspan="1" colspan="1">0.759</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.349</td><td rowspan="1" colspan="1">−0.13</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.99</td><td rowspan="1" colspan="1">0.606</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.51</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">2.5&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.091</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.436</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.100</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs205611 (Closest Gene: <a id="__tag_380052447" class="tag_hotlink" href="/nuccore/AC007387.1" ref="/nuccore/AC007387.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC007387.1","term_id":"4680449","term_text":"AC007387.1"}}</span>AC007387.1</a>)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">2.1&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.13</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.43</td><td rowspan="1" colspan="1">1.3&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.330</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.500</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.190</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.480</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.250</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.290</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.23</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.1&nbsp;× 10<sup>−2</sup></td><td rowspan="1" colspan="1">−0.30</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">9.36</td><td rowspan="1" colspan="1">0.915</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.495</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.68</td><td rowspan="1" colspan="1">0.922</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.2&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">6.4&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.729</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.418</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs10235789 (Closest Gene: SEMA3A)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2.7&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.27</td><td rowspan="1" colspan="1">9.3&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">0.620</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.540</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2.0&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">2.92</td><td rowspan="1" colspan="1">0.750</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.200</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">0.096</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.59</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2.5&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">13.9</td><td rowspan="1" colspan="1">1.3&nbsp;× 10<sup>−2</sup></td><td rowspan="1" colspan="1">−0.30</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">9.54</td><td rowspan="1" colspan="1">0.384</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">1.17</td><td rowspan="1" colspan="1">0.349</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">1.02</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">6.6&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">3.2&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.886</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.548</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs2189849, rs1357813</strong><sup>a</sup><strong>, rs2189849 (Closest Gene: SEMA3A)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">3.1 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">2.1&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">−0.08</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.53</td><td rowspan="1" colspan="1">0.680</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.170</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.10</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.680</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.620</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.920</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.530</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.09</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.128</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">4.33</td><td rowspan="1" colspan="1">0.403</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.32</td><td rowspan="1" colspan="1">0.555</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.927</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">3.2&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">6.5&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.685</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.306</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs4732540 (Closest Gene: SEMA3A)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">8.5&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.86</td><td rowspan="1" colspan="1">6.1&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">0.990</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.260</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.07</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.140</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.090</td><td rowspan="1" colspan="1">−0.09</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">0.160</td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">0.210</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.33</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.368</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">0.593</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">0.422</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.280</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">1.31</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.088</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">4.7&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.470</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">−0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs10874518 (Closest Gene: OLFM3)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">3.5&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.10</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.26</td><td rowspan="1" colspan="1">0.840</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.970</td><td rowspan="1" colspan="1">−0.00</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.460</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.330</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">0.320</td><td rowspan="1" colspan="1">−0.05</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.160</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">0.340</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.17</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.401</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">1.23</td><td rowspan="1" colspan="1">0.358</td><td rowspan="1" colspan="1">−0.12</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">1.48</td><td rowspan="1" colspan="1">0.105</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">4.61</td><td rowspan="1" colspan="1">0.552</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.52</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.2&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">−0.07</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.686</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.410</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.184</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs3739070, rs11900963</strong><sup>b</sup><strong>, rs3739070 (Closest Gene: TRAF3IP1)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">4.1&nbsp;× 10<sup>−3</sup></td><td rowspan="1" colspan="1">−0.11</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">6.2&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">1.70</td><td rowspan="1" colspan="1">0.460</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.170</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.11</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">5.8&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">−0.28</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">3.39</td><td rowspan="1" colspan="1">1.3&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">3.07</td><td rowspan="1" colspan="1">0.370</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.540</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.10</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.189</td><td rowspan="1" colspan="1">−0.27</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">3.86</td><td rowspan="1" colspan="1">0.654</td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.086</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">6.67</td><td rowspan="1" colspan="1">0.911</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">6.1&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">−0.16</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">2.3&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.967</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.167</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs4900109 (Closest Gene: SLC324A4)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.060</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.880</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">2.9&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">3.23</td><td rowspan="1" colspan="1">0.450</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.42</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.190</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">0.990</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">5.7&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">3.45</td><td rowspan="1" colspan="1">0.150</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.44</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.742</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.096</td><td rowspan="1" colspan="1">−0.22</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">4.92</td><td rowspan="1" colspan="1">0.737</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.962</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.826</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">7.7&nbsp;× 10<sup>−21</sup></td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.879</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−</td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td colspan="20" rowspan="1"><strong>rs11630290 (Closest Gene: HERC1)</strong></td></tr><tr><td colspan="20" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">D</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.060</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.630</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.076</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">6.3×10<sup>−6</sup></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.13</td></tr><tr><td rowspan="1" colspan="1">R1</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.210</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">0.290</td><td rowspan="1" colspan="1">−0.06</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">0.980</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.00</td><td rowspan="1" colspan="1">0.430</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">0.12</td></tr><tr><td rowspan="1" colspan="1">R2</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.336</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.61</td><td rowspan="1" colspan="1">0.339</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">1.60</td><td rowspan="1" colspan="1">0.696</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">0.921</td><td rowspan="1" colspan="1">−0.01</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">MA</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.031</td><td rowspan="1" colspan="1">−0.04</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">−0.00</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">0.125</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−</td><td rowspan="1" colspan="1">1.3&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">−</td></tr></tbody></table></div><div id="largeobj_idm139960095398336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3155193/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139960091068896"><p id="__p52" class="p p-first-last">D, discovery sample; R1, replication sample 1; R2, and replication sample 2; MA, meta-analysis; MAF, minor allele frequency.</p></div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>a</sup>Proxy for rs2189849, r2&nbsp;= 0.66.</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>b</sup>Proxy for rs3739070, r2&nbsp;= 1.00.</div></div></div><p id="__p11">For furrows, the most strongly associated SNP was rs3739070 (p = 6.2&nbsp;× 10<sup>−8</sup>), a nonsynonymous coding SNP in <em>TRAF3IP1</em> (MIM <a href="/omim/607380" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607380</a>) on chromosome 2q37.3 (<a href="/pmc/articles/PMC3155193/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>). This finding was replicated with the use of a proxy SNP in replication sample 1 (rs11900963, p = 1.3&nbsp;× 10<sup>−4</sup>, r<sup>2</sup> = 1.00, D′ = 1.00, HapMap II r22b36 CEU). Meta-analysis across the three samples yielded a p value of 2.3&nbsp;× 10<sup>−12</sup>, suggesting that <em>TRAF3IP1</em> may contribute to the overall thickness and density of the iris. <em>TRAF3IP1</em> is also implicated in neurogenesis, neural migration, and synaptogenesis.<sup><a href="#bib35" rid="bib35 bib36" class=" bibr popnode">35,36</a></sup> About 1.7% of the variance in the discovery sample and 3% in the first replication sample were explained by rs3739070.</p><p id="__p12">The SNP most strongly associated with the pigmented ring, rs4900109 (p = 2.90&nbsp;× 10<sup>−13</sup>), exceeded our threshold for genome-wide significance in the discovery sample and was also found to be strongly associated with the pigmented ring in the replication sample 1 (p = 5.70&nbsp;× 10<sup>−5</sup>; <a href="/pmc/articles/PMC3155193/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). Not surprisingly, meta-analysis of the three samples was highly significant (7.6&nbsp;× 10<sup>−21</sup>). This SNP is situated in an intergenic region on chromosome 14q32.12 upstream of the well-known pigmentation gene <em>SLC24A4</em> (MIM <a href="/omim/609840" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609840</a>) and is in high LD with rs12896399 in the same region (rs4900109, rs12896399, r<sup>2</sup> = 1.00, D′ = 1.00 within the HapMap CEU sample), which has been previously associated with blue versus brown eye color, blue versus green eye color, hair color, skin sensitivity to sun, and increased risk for cutaneous malignant melanoma.<sup><a href="#bib4" rid="bib4 bib6 bib37 bib38 bib39" class=" bibr popnode">4,6,37–39</a></sup> Thus, rs4900109 may influence <em>where</em> in the iris the pigment is placed, rather than overall eye color per se. The finding that 91% of people judged to have a green or hazel eye color by a distant observer in this sample have a pigmented ring supports this notion. Furthermore, 12 SNPs in the GWAS (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>)&nbsp;had a p value less than 1.0&nbsp;× 10<sup>−5</sup>, and all except&nbsp;rs4900109 were situated in regions that have not previously been associated with iris pigmentation. These SNPs may influence where melanin is deposited in the iris and may also be modifying risk alleles for cutaneous melanoma, because three of the top findings are in well-known regions associated with cutaneous melanoma: 6q16.3 (rs9322729, p = 8.2&nbsp;× 10<sup>−6</sup>),<sup><a href="#bib40" rid="bib40 bib41" class=" bibr popnode">40,41</a></sup> 7q21.3 (rs10808093, p = 5.6&nbsp;× 10<sup>−6</sup>),<sup><a href="#bib42" rid="bib42 bib43" class=" bibr popnode">42,43</a></sup> and 7q31.1 (rs6957003, p = 6.4&nbsp;× 10<sup>−6</sup>).<sup><a href="#bib44" rid="bib44 bib45" class=" bibr popnode">44,45</a></sup> In both the discovery sample and the larger replication sample, rs4900109 accounted for around 3.2% of the variance in the pigmentation ring.</p><p id="__p13">For iris nevi, rs11630290, located in <em>HERC1</em> (MIM <a href="/omim/605109" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605109</a>; 15q22.31), showed the strongest association signal (p = 6.3&nbsp;× 10<sup>−6</sup>; <a href="/pmc/articles/PMC3155193/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>). However, this association did not replicate, although the direction of effect was consistent in the larger replication sample and the meta-analysis yielded a p value of 1.28&nbsp;× 10<sup>−6</sup>. <em>HERC1</em> is distinct from the well-known pigmentation gene <em>HERC2</em>, which is situated 35 Mbp downstream of <em>HERC1</em> on chromosome 15. Little is known about <em>HERC1</em>, but it is believed to be involved in different membrane transport processes.<sup><a href="#bib46" rid="bib46 bib47" class=" bibr popnode">46,47</a></sup> Other suggestive SNP findings and the quantile-quantile (Q-Q) and Manhattan plots for each iris characteristic are presented in <a href="#app2" rid="app2" class=" sec">Figures S1–S4</a>.</p><p id="__p14">To complement our genome-wide association analysis, we performed genome-wide gene-based tests by using VEGAS,<sup><a href="#bib48" rid="bib48 bib49" class=" bibr popnode">48,49</a></sup> which accounts for the size of genes, SNP density, and the LD between SNPs. Depending on the underlying genetic architecture, this approach can be more powerful than traditional SNP-level analyses, such as when, for example, a gene contains two or more semi-independent causal variants of moderate significance. For each trait, the p values for the genic SNPs from the GWAS analysis of the discovery sample were assigned to&nbsp;genes according to positions on the UCSC Genome Browser hg18 assembly. In order to capture regulatory regions and SNPs in LD, we used the default setting in VEGAS, which defines gene boundaries as ± 50 kb of 5′ and 3′ UTRs. <a href="#app2" rid="app2" class=" sec">Tables S5–S8</a> present the results from VEGAS,<sup><a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353345">48</a></sup> summarizing the&nbsp;50 most associated genes for each iris characteristic.</p><p id="__p15">The only gene with an empirical p value ≤ 3.0&nbsp;× 10<sup>−6</sup> after 10<sup>6</sup> simulations (our Bonferroni-corrected threshold that corrects for the 17,787 genes tested) was SEM<em>A3A</em>, and its association with crypts and this may suggest that there are other SNPs in SEM<em>A3A</em> with small effects that influence crypts and are in low LD with rs10235789 (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>). SEM<em>A3A</em> influences the two top cell layers in the human iris where crypts occur<sup><a href="#bib7" rid="bib7 bib33" class=" bibr popnode">7,33</a></sup> but also the cell layers in cerebral cortex,<sup><a href="#bib34" rid="bib34 bib50" class=" bibr popnode">34,50</a></sup> hippocampus,<sup><a href="#bib51" rid="bib51 bib52" class=" bibr popnode">51,52</a></sup> cerebellum,<sup><a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353343">30</a></sup> and amygdala.<sup><a href="#bib52" rid="bib52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353337">52</a></sup> Therefore, we decided to investigate whether crypts also are associated with other genes that, like SEM<em>A3A</em>, have been associated with decreased cortical width and depleted dendritic branching in the medial forebrain.<sup><a href="#bib53" rid="bib53 bib54 bib55" class=" bibr popnode">53–55</a></sup> This was accomplished by testing of a “gene set” derived from an experiment conducted by Krasnova et&nbsp;al.<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353338">55</a></sup></p><p id="__p16">Krasnova and colleagues<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267353354">55</a></sup> injected 6-hydroxydopamine into the medial forebrain of mice on postnatal day 1, which resulted in a decrease in the dopamine levels in the cortex and striatum of 78% and 38%, respectively (on postnatal day 7), as well as changes in the frontocortical width and somatosensory parietal cortex. Expression microarrays were then used to identify genes that were up- or downregulated in response to this treatment in the medial forebrain. They found expression differences in SEM<em>A3A</em> and 228 other genes. SNP data for 219 of the orthologous human genes were available in our discovery sample (<a href="#app2" rid="app2" class=" sec">Table S9</a>). Consequently, in order to test whether the 24,812 SNPs derived from this gene set were enriched for association with crypts, and potentially also with furrows, we examined the distribution of test statistics for these SNPs as compared to (1) the expectation under the null hypothesis and (2) the distribution of p values for the SNPs from 1,000 randomly selected sets of 219 genes of similar size and with similar numbers of SNPs.</p><p id="__p17">As shown in the Q-Q plots in <a href="/pmc/articles/PMC3155193/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure&nbsp;5</span></a>, the distributions of p values for this candidate SNP set was enriched for crypts. This enrichment is reflected in the high λ values of these SNP sets, indicating that this enrichment is not due to a small number of large effects, but rather to a systematically higher degree of association throughout the candidate SNP set. Importantly, this enhanced association is absent when considering the λ from the full GWAS analysis (i.e., for crypts λ<sub>GWAS</sub> = 1.0051 [<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>A], whereas λ<sub>Krasnova Set</sub> = 1.1362), suggesting that this is unlikely to be due to a methodological artifact. We also used bootstrapping (sampling with replacement) to examine evidence for enhancement, comparing the number of SNPs reaching a nominal p value of .01 observed in genes within the Krasnova set to that observed in 1,000 randomly selected sets of 219 genes of similar size. As shown in <a href="#app2" rid="app2" class=" sec">Table S14</a>, the bootstrapping analyses suggested that a distribution this enriched for crypt and furrow was unlikely to arise by chance alone (empirical p value = 0.019 and 0.034, respectively).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC3155193/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139960087049824" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" src="/pmc/articles/PMC3155193/bin/gr5.jpg"></div></a><div id="largeobj_idm139960087049824" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3155193/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a target="object" href="/pmc/articles/PMC3155193/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC3155193/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure&nbsp;5</a></div><!--caption a7--><div class="caption"><p id="__p43">Q-Q Plots for the 24,812 SNPs Located within the 219 Genes in the Krasnova et&nbsp;al. Set<sup>55</sup></p><p id="__p44">See <a href="#app2" rid="app2" class=" sec">Table S9</a> as well.</p></div></div></div><p id="__p18">To further investigate the hypothesis that these neuronal pathway genes are associated with iris characteristics, we used PLINK to calculate polygenic profile scores by using the maximally associated SNP from the 50 most significant genes derived from the VEGAS analyses of the discovery sample (<a href="#app2" rid="app2" class=" sec">Tables S5–S8</a>) and the 50 most significantly associated genes from Krasnova's gene set (<a href="#app2" rid="app2" class=" sec">Tables S10–13</a>). The profile score from the Krasnova set accounted for 4% of the variation in the crypts in the replication sample (r = 0.2), as compared to the 1.7% accounted for by the empirically derived gene set (<a href="#app2" rid="app2" class=" sec">Table S15</a>). These results are somewhat preliminary, but our results suggest that genes influencing morphogenesis in the developing cortex also influence human iris patterns. Although the current findings suggest that human orthologs of genes responsible for neuronal morphogenesis in the mouse cortex are associated with crypt frequency and furrows, it is possible that the current findings may reflect the effect of a variant in long-range LD or trans effects. Thus, these findings warrant replication using both genetic and brain imaging techniques, which can evaluate whether SNPs associated with iris characteristics are also associated with brain morphology and function.</p><p id="__p19" class="p p-last">In conclusion, we report two quantitative trait loci that affect visible tissue characteristics in the human iris. These loci account for about 1.5% of the variance for crypt and furrows. The variants associated with crypts fall within SEM<em>A3A</em>, which is expressed in the cell layers where crypts&nbsp;occur. Variants associated with furrows fall within <em>TRAF3IP1</em>, which is essential for cytoskeleton function. Both SEM<em>A3A</em> and <em>TRAF3IP1</em> are implicated in pathways that control neurogenesis, neural migration, and synaptogenesis. In addition, we found that variants associated with the pigmented ring, which lie upstream of <em>SLC24A4</em>, are in strong LD with variants that have been associated with eye color, suggesting that these variants may influence where in the iris the pigment is placed, rather than overall eye color per se.</p></div><div id="idm139960092429520" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960092429520title">Acknowledgments</h2><div class="sec"><p id="__p33">The authors thank Manuel Ferreira for helpful discussions and the twins and their families for their participation. We also thank Anjali Henders, Ann Eldridge, Marlene Grace, Kerrie McAloney (sample collection); Lisa Bowdler, Steven Crooks (DNA processing); David Smyth, Harry Beeby, and Daniel Park (IT support).</p><p id="__p34">Funding was provided by the Australian National Health and Medical Research Council (241944, 339462, 389927,389875, 389891, 389892, 389938, 442915, 442981, 496739,552485, 552498), the Australian Research Council (A7960034, A79906588, A79801419, DP0770096, DP0212016, DP0343921),the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254), and the U.S. National Institutes of Health (NIH grants AA07535, AA10248, AA13320, AA13321, AA13326, AA14041, MH66206). S.E.M., D.R.N., A.F.M., M.A.R.F., S.M., D.L.D., and G.W.M. are supported by the National Health and Medical Research Council (NHMRC) Fellowship Scheme. M.L. is a Postdoctoral Fellow supported by the Swedish Research Council.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Five Figures and 15 Tables:</strong><div class="sup-box half_rhythm" id="idm139960084679280"><a href="/pmc/articles/PMC3155193/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(439K, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p21" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p22">Entrez Gene, <a href="http://www.ncbi.nlm.nih.gov/gene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene</a></p></li><li id="u0015"><p id="__p23">GeneCards, <a href="http://www.genecards.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genecards.org/</a></p></li><li id="u0020"><p id="__p24">Haploview, <a href="http://www.broad.mit.edu/mpg/haploview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.broad.mit.edu/mpg/haploview/</a></p></li><li id="u0025"><p id="__p25">Merlin, <a href="http://www.sph.umich.edu/csg/abecasis/Merlin/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Merlin/</a></p></li><li id="u0030"><p id="__p26">Metal, <a href="http://www.sph.umich.edu/csg/abecasis/Metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/</a></p></li><li id="u0035"><p id="__p27">Mx, <a href="http://www.vcu.edu/mx/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.vcu.edu/mx/</a></p></li><li id="u0040"><p id="__p28">NCIB Unigene, <a href="http://www.ncbi.nlm.nih.gov/unigene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/unigene</a></p></li><li id="u0045"><p id="__p29">NIEBank, <a href="http://neibank.nei.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://neibank.nei.nih.gov</a></p></li><li id="u0050"><p id="__p30">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org</a></p></li><li id="u0055"><p id="__p31">Plink, <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a></p></li><li id="u0060"><p id="__p32">SNAP plots, <a href="http://www.broadinstitute.org/mpg/snap/ldplot.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/snap/ldplot.php</a></p></li></ul><p></p></div><div id="idm139960088368816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139960088368816title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Sturm R.A., Larsson M. Genetics of human iris colour and patterns. <span><span class="ref-journal">Pigment cell &amp; melanoma research. </span>2009;<span class="ref-vol">22</span>:544–562.</span> [<a href="/pubmed/19619260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pigment+cell+&amp;+melanoma+research&amp;title=Genetics+of+human+iris+colour+and+patterns&amp;author=R.A.+Sturm&amp;author=M.+Larsson&amp;volume=22&amp;publication_year=2009&amp;pages=544-562&amp;pmid=19619260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Liu F., van Duijn K., Vingerling J.R., Hofman A., Uitterlinden A.G., Janssens A.C., Kayser M. Eye color and the prediction of complex phenotypes from genotypes. <span><span class="ref-journal">Curr. Biol. </span>2009;<span class="ref-vol">19</span>:R192–R193.</span> [<a href="/pubmed/19278628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol.&amp;title=Eye+color+and+the+prediction+of+complex+phenotypes+from+genotypes&amp;author=F.+Liu&amp;author=K.+van+Duijn&amp;author=J.R.+Vingerling&amp;author=A.+Hofman&amp;author=A.G.+Uitterlinden&amp;volume=19&amp;publication_year=2009&amp;pages=R192-R193&amp;pmid=19278628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Liu F., Wollstein A., Hysi P.G., Ankra-Badu G.A., Spector T.D., Park D., Zhu G., Larsson M., Duffy D.L., Montgomery G.W. Digital quantification of human eye color highlights genetic association of three new loci. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865509/">PMC free article</a>]</span> [<a href="/pubmed/20463881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Digital+quantification+of+human+eye+color+highlights+genetic+association+of+three+new+loci&amp;author=F.+Liu&amp;author=A.+Wollstein&amp;author=P.G.+Hysi&amp;author=G.A.+Ankra-Badu&amp;author=T.D.+Spector&amp;volume=6&amp;publication_year=2010&amp;pages=e1000934&amp;pmid=20463881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Sulem P., Gudbjartsson D.F., Stacey S.N., Helgason A., Rafnar T., Magnusson K.P., Manolescu A., Karason A., Palsson A., Thorleifsson G. Genetic determinants of hair, eye and skin pigmentation in Europeans. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1443–1452.</span> [<a href="/pubmed/17952075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+determinants+of+hair,+eye+and+skin+pigmentation+in+Europeans&amp;author=P.+Sulem&amp;author=D.F.+Gudbjartsson&amp;author=S.N.+Stacey&amp;author=A.+Helgason&amp;author=T.+Rafnar&amp;volume=39&amp;publication_year=2007&amp;pages=1443-1452&amp;pmid=17952075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Walsh S., Liu F., Ballantyne K.N., van Oven M., Lao O., Kayser M. IrisPlex: A sensitive DNA tool for accurate prediction of blue and brown eye colour in the absence of ancestry information. <span><span class="ref-journal">Forensic Sci Int Genet. </span>2010;<span class="ref-vol">5</span>:170–180.</span> [<a href="/pubmed/20457092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Forensic+Sci+Int+Genet&amp;title=IrisPlex:+A+sensitive+DNA+tool+for+accurate+prediction+of+blue+and+brown+eye+colour+in+the+absence+of+ancestry+information&amp;author=S.+Walsh&amp;author=F.+Liu&amp;author=K.N.+Ballantyne&amp;author=M.+van+Oven&amp;author=O.+Lao&amp;volume=5&amp;publication_year=2010&amp;pages=170-180&amp;pmid=20457092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Duffy D.L., Zhao Z.Z., Sturm R.A., Hayward N.K., Martin N.G., Montgomery G.W. Multiple pigmentation gene polymorphisms account for a substantial proportion of risk of cutaneous malignant melanoma. <span><span class="ref-journal">J. Invest. Dermatol. </span>2010;<span class="ref-vol">130</span>:520–528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3672059/">PMC free article</a>]</span> [<a href="/pubmed/19710684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=Multiple+pigmentation+gene+polymorphisms+account+for+a+substantial+proportion+of+risk+of+cutaneous+malignant+melanoma&amp;author=D.L.+Duffy&amp;author=Z.Z.+Zhao&amp;author=R.A.+Sturm&amp;author=N.K.+Hayward&amp;author=N.G.+Martin&amp;volume=130&amp;publication_year=2010&amp;pages=520-528&amp;pmid=19710684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Cvekl A., Tamm E.R. Anterior eye development and ocular mesenchyme: new insights from mouse models and human diseases. <span><span class="ref-journal">Bioessays. </span>2004;<span class="ref-vol">26</span>:374–386.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2094210/">PMC free article</a>]</span> [<a href="/pubmed/15057935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Anterior+eye+development+and+ocular+mesenchyme:+new+insights+from+mouse+models+and+human+diseases&amp;author=A.+Cvekl&amp;author=E.R.+Tamm&amp;volume=26&amp;publication_year=2004&amp;pages=374-386&amp;pmid=15057935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Simpson T.I., Price D.J. Pax6; a pleiotropic player in development. <span><span class="ref-journal">Bioessays. </span>2002;<span class="ref-vol">24</span>:1041–1051.</span> [<a href="/pubmed/12386935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Pax6;+a+pleiotropic+player+in+development&amp;author=T.I.+Simpson&amp;author=D.J.+Price&amp;volume=24&amp;publication_year=2002&amp;pages=1041-1051&amp;pmid=12386935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Ellison-Wright Z., Heyman I., Frampton I., Rubia K., Chitnis X., Ellison-Wright I., Williams S.C., Suckling J., Simmons A., Bullmore E. Heterozygous PAX6 mutation, adult brain structure and fronto-striato-thalamic function in a human family. <span><span class="ref-journal">Eur. J. Neurosci. </span>2004;<span class="ref-vol">19</span>:1505–1512.</span> [<a href="/pubmed/15066147" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Neurosci.&amp;title=Heterozygous+PAX6+mutation,+adult+brain+structure+and+fronto-striato-thalamic+function+in+a+human+family&amp;author=Z.+Ellison-Wright&amp;author=I.+Heyman&amp;author=I.+Frampton&amp;author=K.+Rubia&amp;author=X.+Chitnis&amp;volume=19&amp;publication_year=2004&amp;pages=1505-1512&amp;pmid=15066147&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Osumi N., Shinohara H., Numayama-Tsuruta K., Maekawa M. Concise review: Pax6 transcription factor contributes to both embryonic and adult neurogenesis as a multifunctional regulator. <span><span class="ref-journal">Stem Cells. </span>2008;<span class="ref-vol">26</span>:1663–1672.</span> [<a href="/pubmed/18467663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stem+Cells&amp;title=Concise+review:+Pax6+transcription+factor+contributes+to+both+embryonic+and+adult+neurogenesis+as+a+multifunctional+regulator&amp;author=N.+Osumi&amp;author=H.+Shinohara&amp;author=K.+Numayama-Tsuruta&amp;author=M.+Maekawa&amp;volume=26&amp;publication_year=2008&amp;pages=1663-1672&amp;pmid=18467663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Donaldson D.D. The significance of spotting of the iris in mongolids: Brushfield spots. <span><span class="ref-journal">Arch. Ophthalmol. </span>1961;<span class="ref-vol">65</span>:26–31.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Ophthalmol.&amp;title=The+significance+of+spotting+of+the+iris+in+mongolids:+Brushfield+spots&amp;author=D.D.+Donaldson&amp;volume=65&amp;publication_year=1961&amp;pages=26-31&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Lee M.J., Stephenson D.A. Recent developments in neurofibromatosis type 1. <span><span class="ref-journal">Curr. Opin. Neurol. </span>2007;<span class="ref-vol">20</span>:135–141.</span> [<a href="/pubmed/17351482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Neurol.&amp;title=Recent+developments+in+neurofibromatosis+type+1&amp;author=M.J.+Lee&amp;author=D.A.+Stephenson&amp;volume=20&amp;publication_year=2007&amp;pages=135-141&amp;pmid=17351482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Mariën P., Brouns R., Engelborghs S., Wackenier P., Verhoeven J., Ceulemans B., De Deyn P.P. Cerebellar cognitive affective syndrome without global mental retardation in two relatives with Gillespie syndrome. <span><span class="ref-journal">Cortex. </span>2008;<span class="ref-vol">44</span>:54–67.</span> [<a href="/pubmed/18387531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cortex&amp;title=Cerebellar+cognitive+affective+syndrome+without+global+mental+retardation+in+two+relatives+with+Gillespie+syndrome&amp;author=P.+Mariën&amp;author=R.+Brouns&amp;author=S.+Engelborghs&amp;author=P.+Wackenier&amp;author=J.+Verhoeven&amp;volume=44&amp;publication_year=2008&amp;pages=54-67&amp;pmid=18387531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Ticho B.H., Hilchie-Schmidt C., Egel R.T., Traboulsi E.I., Howarth R.J., Robinson D. Ocular findings in Gillespie-like syndrome: association with a new PAX6 mutation. <span><span class="ref-journal">Ophthalmic Genet. </span>2006;<span class="ref-vol">27</span>:145–149.</span> [<a href="/pubmed/17148041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmic+Genet.&amp;title=Ocular+findings+in+Gillespie-like+syndrome:+association+with+a+new+PAX6+mutation&amp;author=B.H.+Ticho&amp;author=C.+Hilchie-Schmidt&amp;author=R.T.+Egel&amp;author=E.I.+Traboulsi&amp;author=R.J.+Howarth&amp;volume=27&amp;publication_year=2006&amp;pages=145-149&amp;pmid=17148041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Houlston R.S., Damato B.E. Genetic predisposition to ocular melanoma. <span><span class="ref-journal">Eye (Lond.) </span>1999;<span class="ref-vol">13</span>:43–46.</span> [<a href="/pubmed/10396383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eye+(Lond.)&amp;title=Genetic+predisposition+to+ocular+melanoma&amp;author=R.S.+Houlston&amp;author=B.E.+Damato&amp;volume=13&amp;publication_year=1999&amp;pages=43-46&amp;pmid=10396383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Richtig E., Langmann G., Müllner K., Smolle J. Ocular melanoma: epidemiology, clinical presentation and relationship with dysplastic nevi. <span><span class="ref-journal">Ophthalmologica. </span>2004;<span class="ref-vol">218</span>:111–114.</span> [<a href="/pubmed/15004500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmologica&amp;title=Ocular+melanoma:+epidemiology,+clinical+presentation+and+relationship+with+dysplastic+nevi&amp;author=E.+Richtig&amp;author=G.+Langmann&amp;author=K.+Müllner&amp;author=J.+Smolle&amp;volume=218&amp;publication_year=2004&amp;pages=111-114&amp;pmid=15004500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Chang B., Smith R.S., Hawes N.L., Anderson M.G., Zabaleta A., Savinova O., Roderick T.H., Heckenlively J.R., Davisson M.T., John S.W. Interacting loci cause severe iris atrophy and glaucoma in DBA/2J mice. <span><span class="ref-journal">Nat. Genet. </span>1999;<span class="ref-vol">21</span>:405–409.</span> [<a href="/pubmed/10192392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Interacting+loci+cause+severe+iris+atrophy+and+glaucoma+in+DBA/2J+mice&amp;author=B.+Chang&amp;author=R.S.+Smith&amp;author=N.L.+Hawes&amp;author=M.G.+Anderson&amp;author=A.+Zabaleta&amp;volume=21&amp;publication_year=1999&amp;pages=405-409&amp;pmid=10192392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Davis-Silberman N., Ashery-Padan R. Iris development in vertebrates; genetic and molecular considerations. <span><span class="ref-journal">Brain Res. </span>2008;<span class="ref-vol">1192</span>:17–28.</span> [<a href="/pubmed/17466284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res.&amp;title=Iris+development+in+vertebrates;+genetic+and+molecular+considerations&amp;author=N.+Davis-Silberman&amp;author=R.+Ashery-Padan&amp;volume=1192&amp;publication_year=2008&amp;pages=17-28&amp;pmid=17466284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Asano N., Schlötzer-Schrehardt U., Naumann G.O. A histopathologic study of iris changes in pseudoexfoliation syndrome. <span><span class="ref-journal">Ophthalmology. </span>1995;<span class="ref-vol">102</span>:1279–1290.</span> [<a href="/pubmed/9097764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ophthalmology&amp;title=A+histopathologic+study+of+iris+changes+in+pseudoexfoliation+syndrome&amp;author=N.+Asano&amp;author=U.+Schlötzer-Schrehardt&amp;author=G.O.+Naumann&amp;volume=102&amp;publication_year=1995&amp;pages=1279-1290&amp;pmid=9097764&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Mackey D.A., Mackinnon J.R., Brown S.A., Kearns L.S., Ruddle J.B., Sanfilippo P.G., Sun C., Hammond C.J., Young T.L., Martin N.G., Hewitt A.W. Twins eye study in Tasmania (TEST): rationale and methodology to recruit and examine twins. <span><span class="ref-journal">Twin Res. Hum. Genet. </span>2009;<span class="ref-vol">12</span>:441–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3855231/">PMC free article</a>]</span> [<a href="/pubmed/19803772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.+Hum.+Genet.&amp;title=Twins+eye+study+in+Tasmania+(TEST):+rationale+and+methodology+to+recruit+and+examine+twins&amp;author=D.A.+Mackey&amp;author=J.R.+Mackinnon&amp;author=S.A.+Brown&amp;author=L.S.+Kearns&amp;author=J.B.+Ruddle&amp;volume=12&amp;publication_year=2009&amp;pages=441-454&amp;pmid=19803772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Wright M.J., Martin N.G. Brisbane Adolescent Twin Study: Outline of study methods and research projects. <span><span class="ref-journal">Aust. J. Psychol. </span>2004;<span class="ref-vol">56</span>:65–78.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aust.+J.+Psychol.&amp;title=Brisbane+Adolescent+Twin+Study:+Outline+of+study+methods+and+research+projects&amp;author=M.J.+Wright&amp;author=N.G.+Martin&amp;volume=56&amp;publication_year=2004&amp;pages=65-78&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">McEvoy B.P., Montgomery G.W., McRae A.F., Ripatti S., Perola M., Spector T.D., Cherkas L., Ahmadi K.R., Boomsma D., Willemsen G. Geographical structure and differential natural selection among North European populations. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:804–814.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2675969/">PMC free article</a>]</span> [<a href="/pubmed/19265028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Geographical+structure+and+differential+natural+selection+among+North+European+populations&amp;author=B.P.+McEvoy&amp;author=G.W.+Montgomery&amp;author=A.F.+McRae&amp;author=S.+Ripatti&amp;author=M.+Perola&amp;volume=19&amp;publication_year=2009&amp;pages=804-814&amp;pmid=19265028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Medland S.E., Nyholt D.R., Painter J.N., McEvoy B.P., McRae A.F., Zhu G., Gordon S.D., Ferreira M.A., Wright M.J., Henders A.K. Common variants in the trichohyalin gene are associated with straight hair in Europeans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">85</span>:750–755.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775823/">PMC free article</a>]</span> [<a href="/pubmed/19896111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Common+variants+in+the+trichohyalin+gene+are+associated+with+straight+hair+in+Europeans&amp;author=S.E.+Medland&amp;author=D.R.+Nyholt&amp;author=J.N.+Painter&amp;author=B.P.+McEvoy&amp;author=A.F.+McRae&amp;volume=85&amp;publication_year=2009&amp;pages=750-755&amp;pmid=19896111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Larsson M., Pedersen N.L., Stattin H. Importance of genetic effects for characteristics of the human iris. <span><span class="ref-journal">Twin Res. </span>2003;<span class="ref-vol">6</span>:192–200.</span> [<a href="/pubmed/12855068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.&amp;title=Importance+of+genetic+effects+for+characteristics+of+the+human+iris&amp;author=M.+Larsson&amp;author=N.L.+Pedersen&amp;author=H.+Stattin&amp;volume=6&amp;publication_year=2003&amp;pages=192-200&amp;pmid=12855068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="mixed-citation">Neale, M., Boker, S., Xie, G., and Maes, H. (1999). Mx: Statistical Modeling. 5th Edition (Box 126 MCV, Richmond, VA 23298, Department of Psychiatry: Medical College of Virginia).</span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Chen W.M., Abecasis G.R. Family-based association tests for genomewide association scans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:913–926.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Family-based+association+tests+for+genomewide+association+scans&amp;author=W.M.+Chen&amp;author=G.R.+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-926&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Pe'er I., Yelensky R., Altshuler D., Daly M.J. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I.+Pe'er&amp;author=R.+Yelensky&amp;author=D.+Altshuler&amp;author=M.J.+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Eastwood S.L., Law A.J., Everall I.P., Harrison P.J. The axonal chemorepellant semaphorin 3A is increased in the cerebellum in schizophrenia and may contribute to its synaptic pathology. <span><span class="ref-journal">Mol. Psychiatry. </span>2003;<span class="ref-vol">8</span>:148–155.</span> [<a href="/pubmed/12610647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=The+axonal+chemorepellant+semaphorin+3A+is+increased+in+the+cerebellum+in+schizophrenia+and+may+contribute+to+its+synaptic+pathology&amp;author=S.L.+Eastwood&amp;author=A.J.+Law&amp;author=I.P.+Everall&amp;author=P.J.+Harrison&amp;volume=8&amp;publication_year=2003&amp;pages=148-155&amp;pmid=12610647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Wedenoja J., Loukola A., Tuulio-Henriksson A., Paunio T., Ekelund J., Silander K., Varilo T., Heikkilä K., Suvisaari J., Partonen T. Replication of linkage on chromosome 7q22 and association of the regional Reelin gene with working memory in schizophrenia families. <span><span class="ref-journal">Mol. Psychiatry. </span>2008;<span class="ref-vol">13</span>:673–684.</span> [<a href="/pubmed/17684500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Replication+of+linkage+on+chromosome+7q22+and+association+of+the+regional+Reelin+gene+with+working+memory+in+schizophrenia+families&amp;author=J.+Wedenoja&amp;author=A.+Loukola&amp;author=A.+Tuulio-Henriksson&amp;author=T.+Paunio&amp;author=J.+Ekelund&amp;volume=13&amp;publication_year=2008&amp;pages=673-684&amp;pmid=17684500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Good P.F., Alapat D., Hsu A., Chu C., Perl D., Wen X., Burstein D.E., Kohtz D.S. A role for semaphorin 3A signaling in the degeneration of hippocampal neurons during Alzheimer's disease. <span><span class="ref-journal">J. Neurochem. </span>2004;<span class="ref-vol">91</span>:716–736.</span> [<a href="/pubmed/15485501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurochem.&amp;title=A+role+for+semaphorin+3A+signaling+in+the+degeneration+of+hippocampal+neurons+during+Alzheimer's+disease&amp;author=P.F.+Good&amp;author=D.+Alapat&amp;author=A.+Hsu&amp;author=C.+Chu&amp;author=D.+Perl&amp;volume=91&amp;publication_year=2004&amp;pages=716-736&amp;pmid=15485501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">Lwigale P.Y., Bronner-Fraser M. Semaphorin3A/neuropilin-1 signaling acts as a molecular switch regulating neural crest migration during cornea development. <span><span class="ref-journal">Dev. Biol. </span>2009;<span class="ref-vol">336</span>:257–265.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2800376/">PMC free article</a>]</span> [<a href="/pubmed/19833121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=Semaphorin3A/neuropilin-1+signaling+acts+as+a+molecular+switch+regulating+neural+crest+migration+during+cornea+development&amp;author=P.Y.+Lwigale&amp;author=M.+Bronner-Fraser&amp;volume=336&amp;publication_year=2009&amp;pages=257-265&amp;pmid=19833121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Polleux F., Morrow T., Ghosh A. Semaphorin 3A is a chemoattractant for cortical apical dendrites. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">404</span>:567–573.</span> [<a href="/pubmed/10766232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Semaphorin+3A+is+a+chemoattractant+for+cortical+apical+dendrites&amp;author=F.+Polleux&amp;author=T.+Morrow&amp;author=A.+Ghosh&amp;volume=404&amp;publication_year=2000&amp;pages=567-573&amp;pmid=10766232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation">Callicott J.H., Straub R.E., Pezawas L., Egan M.F., Mattay V.S., Hariri A.R., Verchinski B.A., Meyer-Lindenberg A., Balkissoon R., Kolachana B. Variation in DISC1 affects hippocampal structure and function and increases risk for schizophrenia. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2005;<span class="ref-vol">102</span>:8627–8632.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1143583/">PMC free article</a>]</span> [<a href="/pubmed/15939883" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Variation+in+DISC1+affects+hippocampal+structure+and+function+and+increases+risk+for+schizophrenia&amp;author=J.H.+Callicott&amp;author=R.E.+Straub&amp;author=L.+Pezawas&amp;author=M.F.+Egan&amp;author=V.S.+Mattay&amp;volume=102&amp;publication_year=2005&amp;pages=8627-8632&amp;pmid=15939883&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation">Deutsch S.I., Burket J.A., Katz E. Does subtle disturbance of neuronal migration contribute to schizophrenia and other neurodevelopmental disorders? Potential genetic mechanisms with possible treatment implications. <span><span class="ref-journal">Eur. Neuropsychopharmacol. </span>2010;<span class="ref-vol">20</span>:281–287.</span> [<a href="/pubmed/20207112" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Neuropsychopharmacol.&amp;title=Does+subtle+disturbance+of+neuronal+migration+contribute+to+schizophrenia+and+other+neurodevelopmental+disorders?+Potential+genetic+mechanisms+with+possible+treatment+implications&amp;author=S.I.+Deutsch&amp;author=J.A.+Burket&amp;author=E.+Katz&amp;volume=20&amp;publication_year=2010&amp;pages=281-287&amp;pmid=20207112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation">Eriksson N., Macpherson J.M., Tung J.Y., Hon L.S., Naughton B., Saxonov S., Avey L., Wojcicki A., Pe'er I., Mountain J. Web-based, participant-driven studies yield novel genetic associations for common traits. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2891811/">PMC free article</a>]</span> [<a href="/pubmed/20585627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Web-based,+participant-driven+studies+yield+novel+genetic+associations+for+common+traits&amp;author=N.+Eriksson&amp;author=J.M.+Macpherson&amp;author=J.Y.+Tung&amp;author=L.S.+Hon&amp;author=B.+Naughton&amp;volume=6&amp;publication_year=2010&amp;pages=e1000993&amp;pmid=20585627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation">Han J., Kraft P., Nan H., Guo Q., Chen C., Qureshi A., Hankinson S.E., Hu F.B., Duffy D.L., Zhao Z.Z. A genome-wide association study identifies novel alleles associated with hair color and skin pigmentation. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2367449/">PMC free article</a>]</span> [<a href="/pubmed/18483556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+identifies+novel+alleles+associated+with+hair+color+and+skin+pigmentation&amp;author=J.+Han&amp;author=P.+Kraft&amp;author=H.+Nan&amp;author=Q.+Guo&amp;author=C.+Chen&amp;volume=4&amp;publication_year=2008&amp;pages=e1000074&amp;pmid=18483556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation">Sturm R.A. Human ‘coat colour’ genetics. <span><span class="ref-journal">Pigment cell &amp; melanoma research. </span>2008;<span class="ref-vol">21</span>:115–116.</span> [<a href="/pubmed/18426404" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pigment+cell+&amp;+melanoma+research&amp;title=Human+‘coat+colour’+genetics&amp;author=R.A.+Sturm&amp;volume=21&amp;publication_year=2008&amp;pages=115-116&amp;pmid=18426404&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation">Miele M.E., Jewett M.D., Goldberg S.F., Hyatt D.L., Morelli C., Gualandi F., Rimessi P., Hicks D.J., Weissman B.E., Barbanti-Brodano G., Welch D.R. A human melanoma metastasis-suppressor locus maps to 6q16.3-q23. <span><span class="ref-journal">Int. J. Cancer. </span>2000;<span class="ref-vol">86</span>:524–528.</span> [<a href="/pubmed/10797266" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=A+human+melanoma+metastasis-suppressor+locus+maps+to+6q16.3-q23&amp;author=M.E.+Miele&amp;author=M.D.+Jewett&amp;author=S.F.+Goldberg&amp;author=D.L.+Hyatt&amp;author=C.+Morelli&amp;volume=86&amp;publication_year=2000&amp;pages=524-528&amp;pmid=10797266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation">Shirasaki F., Takata M., Hatta N., Takehara K. Loss of expression of the metastasis suppressor gene KiSS1 during melanoma progression and its association with LOH of chromosome 6q16.3-q23. <span><span class="ref-journal">Cancer Res. </span>2001;<span class="ref-vol">61</span>:7422–7425.</span> [<a href="/pubmed/11606374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Loss+of+expression+of+the+metastasis+suppressor+gene+KiSS1+during+melanoma+progression+and+its+association+with+LOH+of+chromosome+6q16.3-q23&amp;author=F.+Shirasaki&amp;author=M.+Takata&amp;author=N.+Hatta&amp;author=K.+Takehara&amp;volume=61&amp;publication_year=2001&amp;pages=7422-7425&amp;pmid=11606374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation">de Snoo F.A., Hottenga J.J., Gillanders E.M., Sandkuijl L.A., Jones M.P., Bergman W., van der Drift C., van Leeuwen I., van Mourik L., Huurne J.A. Genome-wide linkage scan for atypical nevi in p16-Leiden melanoma families. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2008;<span class="ref-vol">16</span>:1135–1141.</span> [<a href="/pubmed/18398432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Genome-wide+linkage+scan+for+atypical+nevi+in+p16-Leiden+melanoma+families&amp;author=F.A.+de+Snoo&amp;author=J.J.+Hottenga&amp;author=E.M.+Gillanders&amp;author=L.A.+Sandkuijl&amp;author=M.P.+Jones&amp;volume=16&amp;publication_year=2008&amp;pages=1135-1141&amp;pmid=18398432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation">McCannel T.A., Burgess B.L., Nelson S.F., Eskin A.F., Straatsma B.R. Genomic Identification of Significant Targets in Ciliochoroidal Melanoma. <span><span class="ref-journal">Invest Ophthalmol Vis Sci. </span>2010;<span class="ref-vol">52</span>:3018–3022.</span> [<a href="/pubmed/20688739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Invest+Ophthalmol+Vis+Sci&amp;title=Genomic+Identification+of+Significant+Targets+in+Ciliochoroidal+Melanoma&amp;author=T.A.+McCannel&amp;author=B.L.+Burgess&amp;author=S.F.+Nelson&amp;author=A.F.+Eskin&amp;author=B.R.+Straatsma&amp;volume=52&amp;publication_year=2010&amp;pages=3018-3022&amp;pmid=20688739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation">Segura M.F., Hanniford D., Menendez S., Reavie L., Zou X., Alvarez-Diaz S., Zakrzewski J., Blochin E., Rose A., Bogunovic D. Aberrant miR-182 expression promotes melanoma metastasis by repressing FOXO3 and microphthalmia-associated transcription factor. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:1814–1819.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2634798/">PMC free article</a>]</span> [<a href="/pubmed/19188590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Aberrant+miR-182+expression+promotes+melanoma+metastasis+by+repressing+FOXO3+and+microphthalmia-associated+transcription+factor&amp;author=M.F.+Segura&amp;author=D.+Hanniford&amp;author=S.+Menendez&amp;author=L.+Reavie&amp;author=X.+Zou&amp;volume=106&amp;publication_year=2009&amp;pages=1814-1819&amp;pmid=19188590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation">Treszl A., Ladanyi A., Rakosy Z., Buczko Z., Adany R., Balazs M. Molecular cytogenetic characterization of a novel cell line established from a superficial spreading melanoma. <span><span class="ref-journal">Front. Biosci. </span>2006;<span class="ref-vol">11</span>:1844–1853.</span> [<a href="/pubmed/16368560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Biosci.&amp;title=Molecular+cytogenetic+characterization+of+a+novel+cell+line+established+from+a+superficial+spreading+melanoma&amp;author=A.+Treszl&amp;author=A.+Ladanyi&amp;author=Z.+Rakosy&amp;author=Z.+Buczko&amp;author=R.+Adany&amp;volume=11&amp;publication_year=2006&amp;pages=1844-1853&amp;pmid=16368560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation">Rosa J.L., Barbacid M. A giant protein that stimulates guanine nucleotide exchange on ARF1 and Rab proteins forms a cytosolic ternary complex with clathrin and Hsp70. <span><span class="ref-journal">Oncogene. </span>1997;<span class="ref-vol">15</span>:1–6.</span> [<a href="/pubmed/9233772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=A+giant+protein+that+stimulates+guanine+nucleotide+exchange+on+ARF1+and+Rab+proteins+forms+a+cytosolic+ternary+complex+with+clathrin+and+Hsp70&amp;author=J.L.+Rosa&amp;author=M.+Barbacid&amp;volume=15&amp;publication_year=1997&amp;pages=1-6&amp;pmid=9233772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation">Mashimo T., Hadjebi O., Amair-Pinedo F., Tsurumi T., Langa F., Serikawa T., Sotelo C., Guénet J.L., Rosa J.L. Progressive Purkinje cell degeneration in tambaleante mutant mice is a consequence of a missense mutation in HERC1 E3 ubiquitin ligase. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000784.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2791161/">PMC free article</a>]</span> [<a href="/pubmed/20041218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Progressive+Purkinje+cell+degeneration+in+tambaleante+mutant+mice+is+a+consequence+of+a+missense+mutation+in+HERC1+E3+ubiquitin+ligase&amp;author=T.+Mashimo&amp;author=O.+Hadjebi&amp;author=F.+Amair-Pinedo&amp;author=T.+Tsurumi&amp;author=F.+Langa&amp;volume=5&amp;publication_year=2009&amp;pages=e1000784&amp;pmid=20041218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation">Liu J.Z., McRae A.F., Nyholt D.R., Medland S.E., Wray N.R., Brown K.M., Hayward N.K., Montgomery G.W., Visscher P.M., Martin N.G., Macgregor S., AMFS Investigators  A versatile gene-based test for genome-wide association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">87</span>:139–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896770/">PMC free article</a>]</span> [<a href="/pubmed/20598278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+versatile+gene-based+test+for+genome-wide+association+studies&amp;author=J.Z.+Liu&amp;author=A.F.+McRae&amp;author=D.R.+Nyholt&amp;author=S.E.+Medland&amp;author=N.R.+Wray&amp;volume=87&amp;publication_year=2010&amp;pages=139-145&amp;pmid=20598278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation">Neale B.M., Sham P.C. The future of association studies: gene-based analysis and replication. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">75</span>:353–362.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182015/">PMC free article</a>]</span> [<a href="/pubmed/15272419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=The+future+of+association+studies:+gene-based+analysis+and+replication&amp;author=B.M.+Neale&amp;author=P.C.+Sham&amp;volume=75&amp;publication_year=2004&amp;pages=353-362&amp;pmid=15272419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation">Whitford K.L., Dijkhuizen P., Polleux F., Ghosh A. Molecular control of cortical dendrite development. <span><span class="ref-journal">Annu. Rev. Neurosci. </span>2002;<span class="ref-vol">25</span>:127–149.</span> [<a href="/pubmed/12052906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Neurosci.&amp;title=Molecular+control+of+cortical+dendrite+development&amp;author=K.L.+Whitford&amp;author=P.+Dijkhuizen&amp;author=F.+Polleux&amp;author=A.+Ghosh&amp;volume=25&amp;publication_year=2002&amp;pages=127-149&amp;pmid=12052906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation">Chédotal A., Del Rio J.A., Ruiz M., He Z., Borrell V., de Castro F., Ezan F., Goodman C.S., Tessier-Lavigne M., Sotelo C., Soriano E. Semaphorins III and IV repel hippocampal axons via two distinct receptors. <span><span class="ref-journal">Development. </span>1998;<span class="ref-vol">125</span>:4313–4323.</span> [<a href="/pubmed/9753685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Semaphorins+III+and+IV+repel+hippocampal+axons+via+two+distinct+receptors&amp;author=A.+Chédotal&amp;author=J.A.+Del+Rio&amp;author=M.+Ruiz&amp;author=Z.+He&amp;author=V.+Borrell&amp;volume=125&amp;publication_year=1998&amp;pages=4313-4323&amp;pmid=9753685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation">Giger R.J., Pasterkamp R.J., Heijnen S., Holtmaat A.J., Verhaagen J. Anatomical distribution of the chemorepellent semaphorin III/collapsin-1 in the adult rat and human brain: predominant expression in structures of the olfactory-hippocampal pathway and the motor system. <span><span class="ref-journal">J. Neurosci. Res. </span>1998;<span class="ref-vol">52</span>:27–42.</span> [<a href="/pubmed/9556027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.+Res.&amp;title=Anatomical+distribution+of+the+chemorepellent+semaphorin+III/collapsin-1+in+the+adult+rat+and+human+brain:+predominant+expression+in+structures+of+the+olfactory-hippocampal+pathway+and+the+motor+system&amp;author=R.J.+Giger&amp;author=R.J.+Pasterkamp&amp;author=S.+Heijnen&amp;author=A.J.+Holtmaat&amp;author=J.+Verhaagen&amp;volume=52&amp;publication_year=1998&amp;pages=27-42&amp;pmid=9556027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation">Tamamaki N., Fujimori K., Nojyo Y., Kaneko T., Takauji R. Evidence that Sema3A and Sema3F regulate the migration of GABAergic neurons in the developing neocortex. <span><span class="ref-journal">J. Comp. Neurol. </span>2003;<span class="ref-vol">455</span>:238–248.</span> [<a href="/pubmed/12454988" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comp.+Neurol.&amp;title=Evidence+that+Sema3A+and+Sema3F+regulate+the+migration+of+GABAergic+neurons+in+the+developing+neocortex&amp;author=N.+Tamamaki&amp;author=K.+Fujimori&amp;author=Y.+Nojyo&amp;author=T.+Kaneko&amp;author=R.+Takauji&amp;volume=455&amp;publication_year=2003&amp;pages=238-248&amp;pmid=12454988&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation">Pascual M., Pozas E., Soriano E. Role of class 3 semaphorins in the development and maturation of the septohippocampal pathway. <span><span class="ref-journal">Hippocampus. </span>2005;<span class="ref-vol">15</span>:184–202.</span> [<a href="/pubmed/15386596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hippocampus&amp;title=Role+of+class+3+semaphorins+in+the+development+and+maturation+of+the+septohippocampal+pathway&amp;author=M.+Pascual&amp;author=E.+Pozas&amp;author=E.+Soriano&amp;volume=15&amp;publication_year=2005&amp;pages=184-202&amp;pmid=15386596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation">Krasnova I.N., Betts E.S., Dada A., Jefferson A., Ladenheim B., Becker K.G., Cadet J.L., Hohmann C.F. Neonatal dopamine depletion induces changes in morphogenesis and gene expression in the developing cortex. <span><span class="ref-journal">Neurotox. Res. </span>2007;<span class="ref-vol">11</span>:107–130.</span> [<a href="/pubmed/17449454" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurotox.+Res.&amp;title=Neonatal+dopamine+depletion+induces+changes+in+morphogenesis+and+gene+expression+in+the+developing+cortex&amp;author=I.N.+Krasnova&amp;author=E.S.+Betts&amp;author=A.+Dada&amp;author=A.+Jefferson&amp;author=B.+Ladenheim&amp;volume=11&amp;publication_year=2007&amp;pages=107-130&amp;pmid=17449454&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3155193&amp;issue-id=199276&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139960091357392title">Abstract</a></li><li><a href="#sec1title">Main Text</a></li><li><a href="#idm139960092429520title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm139960088368816title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3155193/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3155193/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3155193/pdf/main.pdf">PDF (687K)</a></li> | <li><a href="#" data-citationid="PMC3155193" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3155193%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3155193%2F&amp;text=GWAS%20Findings%20for%20Human%20Iris%20Patterns%3A%20Associations%20with%20Variants%20in%20Genes%20that%20Influence%20Normal%20Neuronal%20Pattern%20Development" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3155193%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21835309" data-aiid="3155193" data-aid="3155193" data-iid="199276" data-domainid="203" data-domain="ajhg" data-accid="PMC3155193" data-md5="e55d2db8d25b2fe2c5c4342fedba4c28"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:32:06-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A8EB8DD52720100000000074204C9&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3155193%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3155193/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>