<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jnephrol"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3013538/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>HLA Has Strongest Association with IgA Nephropathy in Genome-Wide Analysis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013538/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of the American Society of Nephrology : JASN"><meta name="citation_title" content="HLA Has Strongest Association with IgA Nephropathy in Genome-Wide Analysis"><meta name="citation_authors" content="John Feehally, Martin Farrall, Anne Boland, Daniel P. Gale, Ivo Gut, Simon Heath, Ashish Kumar, John F. Peden, Patrick H. Maxwell, David L. Morris, Sandosh Padmanabhan, Timothy J. Vyse, Anna Zawadzka, Andrew J. Rees, Mark Lathrop, Peter J. Ratcliffe"><meta name="citation_date" content="October 2010"><meta name="citation_issue" content="10"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="1791"><meta name="citation_doi" content="10.1681/ASN.2010010076"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3013538/?report=abstract"><meta name="citation_pmid" content="20595679"><meta name="DC.Title" content="HLA Has Strongest Association with IgA Nephropathy in Genome-Wide Analysis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society of Nephrology"><meta name="DC.Contributor" content="John Feehally"><meta name="DC.Contributor" content="Martin Farrall"><meta name="DC.Contributor" content="Anne Boland"><meta name="DC.Contributor" content="Daniel P. Gale"><meta name="DC.Contributor" content="Ivo Gut"><meta name="DC.Contributor" content="Simon Heath"><meta name="DC.Contributor" content="Ashish Kumar"><meta name="DC.Contributor" content="John F. Peden"><meta name="DC.Contributor" content="Patrick H. Maxwell"><meta name="DC.Contributor" content="David L. Morris"><meta name="DC.Contributor" content="Sandosh Padmanabhan"><meta name="DC.Contributor" content="Timothy J. Vyse"><meta name="DC.Contributor" content="Anna Zawadzka"><meta name="DC.Contributor" content="Andrew J. Rees"><meta name="DC.Contributor" content="Mark Lathrop"><meta name="DC.Contributor" content="Peter J. Ratcliffe"><meta name="DC.Date" content="2010 Oct"><meta name="DC.Identifier" content="10.1681/ASN.2010010076"><meta name="DC.Language" content="en"><meta property="og:title" content="HLA Has Strongest Association with IgA Nephropathy in Genome-Wide Analysis"><meta property="og:type" content="article"><meta property="og:description" content="Demographic and family studies support the existence of a genetic contribution to the pathogenesis of IgA nephropathy, but results from genetic association studies of candidate genes are inconsistent. To systematically survey common genetic variation ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013538/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jnephrol.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3013538/epub/">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D55F4DD50E5510000000004F70360.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013538%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/612/">J Am Soc Nephrol</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/192995/">v.21(10); 2010 Oct</a></li><li class="accid">PMC3013538</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jnephrol.gif" alt="Logo of jnephrol" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://jasn.asnjournals.org/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Am Soc Nephrol</a></span>. 2010 Oct; 21(10): 1791–1797. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1681%2FASN.2010010076" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1681/ASN.2010010076</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3013538</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20595679">20595679</a></div></div></div></div><h1 class="content-title">HLA Has Strongest Association with IgA Nephropathy in Genome-Wide Analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Feehally%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333959798240" co-class="co-affbox">John Feehally</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>*</sup> <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333918723632" co-class="co-affbox">Martin Farrall</a>,<sup>†</sup> <a href="/pubmed/?term=Boland%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333919041552" co-class="co-affbox">Anne Boland</a>,<sup>‡</sup> <a href="/pubmed/?term=Gale%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333915999744" co-class="co-affbox">Daniel P. Gale</a>,<sup>§</sup> <a href="/pubmed/?term=Gut%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333915997392" co-class="co-affbox">Ivo Gut</a>,<sup>‡</sup> <a href="/pubmed/?term=Heath%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333919134048" co-class="co-affbox">Simon Heath</a>,<sup>‡</sup> <a href="/pubmed/?term=Kumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333919528288" co-class="co-affbox">Ashish Kumar</a>,<sup>†</sup> <a href="/pubmed/?term=Peden%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333957388528" co-class="co-affbox">John F. Peden</a>,<sup>†</sup> <a href="/pubmed/?term=Maxwell%20PH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333911436832" co-class="co-affbox">Patrick H. Maxwell</a>,<sup>§</sup> <a href="/pubmed/?term=Morris%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333959174320" co-class="co-affbox">David L. Morris</a>,<sup>‖</sup> <a href="/pubmed/?term=Padmanabhan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333915793008" co-class="co-affbox">Sandosh Padmanabhan</a>,<sup>¶</sup> <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333913125280" co-class="co-affbox">Timothy J. Vyse</a>,<sup>‖</sup> <a href="/pubmed/?term=Zawadzka%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333918932416" co-class="co-affbox">Anna Zawadzka</a>,<sup>†</sup> <a href="/pubmed/?term=Rees%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333915728560" co-class="co-affbox">Andrew J. Rees</a>,<sup>**</sup> <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333954161696" co-class="co-affbox">Mark Lathrop</a>,<sup>‡</sup> and  <a href="/pubmed/?term=Ratcliffe%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679" class="affpopup" co-rid="_co_idm140333910902800" co-class="co-affbox">Peter J. Ratcliffe</a><sup>††</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140333959798240"><h3 class="no_margin">John Feehally</h3><p><sup>*</sup>The John Walls Renal Unit, Leicester General Hospital and Department of Infection, Immunity, and Inflammation, University of Leicester, Leicester, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Feehally%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">John Feehally</a></div></div><div id="_co_idm140333918723632"><h3 class="no_margin">Martin Farrall</h3><p><sup>†</sup>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Farrall%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Martin Farrall</a></div></div><div id="_co_idm140333919041552"><h3 class="no_margin">Anne Boland</h3><p><sup>‡</sup>Centre National de Génotypage, Evry Cedex, France; </p><div>Find articles by <a href="/pubmed/?term=Boland%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Anne Boland</a></div></div><div id="_co_idm140333915999744"><h3 class="no_margin">Daniel P. Gale</h3><p><sup>§</sup>Centre for Cell Signalling and Molecular Genetics, Rayne Institute, University College London, London, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Gale%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Daniel P. Gale</a></div></div><div id="_co_idm140333915997392"><h3 class="no_margin">Ivo Gut</h3><p><sup>‡</sup>Centre National de Génotypage, Evry Cedex, France; </p><div>Find articles by <a href="/pubmed/?term=Gut%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Ivo Gut</a></div></div><div id="_co_idm140333919134048"><h3 class="no_margin">Simon Heath</h3><p><sup>‡</sup>Centre National de Génotypage, Evry Cedex, France; </p><div>Find articles by <a href="/pubmed/?term=Heath%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Simon Heath</a></div></div><div id="_co_idm140333919528288"><h3 class="no_margin">Ashish Kumar</h3><p><sup>†</sup>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Kumar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Ashish Kumar</a></div></div><div id="_co_idm140333957388528"><h3 class="no_margin">John F. Peden</h3><p><sup>†</sup>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Peden%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">John F. Peden</a></div></div><div id="_co_idm140333911436832"><h3 class="no_margin">Patrick H. Maxwell</h3><p><sup>§</sup>Centre for Cell Signalling and Molecular Genetics, Rayne Institute, University College London, London, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Maxwell%20PH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Patrick H. Maxwell</a></div></div><div id="_co_idm140333959174320"><h3 class="no_margin">David L. Morris</h3><p><sup>‖</sup>Rheumatology Section, Hammersmith Hospital, Faculty of Medicine, Imperial College London, London, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Morris%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">David L. Morris</a></div></div><div id="_co_idm140333915793008"><h3 class="no_margin">Sandosh Padmanabhan</h3><p><sup>¶</sup>BHF Glasgow Cardiovascular Research Centre, University of Glasgow, Glasgow, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Padmanabhan%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Sandosh Padmanabhan</a></div></div><div id="_co_idm140333913125280"><h3 class="no_margin">Timothy J. Vyse</h3><p><sup>‖</sup>Rheumatology Section, Hammersmith Hospital, Faculty of Medicine, Imperial College London, London, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Timothy J. Vyse</a></div></div><div id="_co_idm140333918932416"><h3 class="no_margin">Anna Zawadzka</h3><p><sup>†</sup>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom; </p><div>Find articles by <a href="/pubmed/?term=Zawadzka%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Anna Zawadzka</a></div></div><div id="_co_idm140333915728560"><h3 class="no_margin">Andrew J. Rees</h3><p><sup>**</sup>Institute of Clinical Pathology, Medical University of Vienna, Vienna, Austria; and </p><div>Find articles by <a href="/pubmed/?term=Rees%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Andrew J. Rees</a></div></div><div id="_co_idm140333954161696"><h3 class="no_margin">Mark Lathrop</h3><p><sup>‡</sup>Centre National de Génotypage, Evry Cedex, France; </p><div>Find articles by <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Mark Lathrop</a></div></div><div id="_co_idm140333910902800"><h3 class="no_margin">Peter J. Ratcliffe</h3><p><sup>††</sup>Nuffield Department of Medicine, University of Oxford, Oxford, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Ratcliffe%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20595679">Peter J. Ratcliffe</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140333959722848_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140333959722848_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140333959722848_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140333959722848_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>*</sup>The John Walls Renal Unit, Leicester General Hospital and Department of Infection, Immunity, and Inflammation, University of Leicester, Leicester, United Kingdom; </div><div class="fm-affl" id="aff2" lang="en"><sup>†</sup>Department of Cardiovascular Medicine, The Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, United Kingdom; </div><div class="fm-affl" id="aff3" lang="en"><sup>‡</sup>Centre National de Génotypage, Evry Cedex, France; </div><div class="fm-affl" id="aff4" lang="en"><sup>§</sup>Centre for Cell Signalling and Molecular Genetics, Rayne Institute, University College London, London, United Kingdom; </div><div class="fm-affl" id="aff5" lang="en"><sup>‖</sup>Rheumatology Section, Hammersmith Hospital, Faculty of Medicine, Imperial College London, London, United Kingdom; </div><div class="fm-affl" id="aff6" lang="en"><sup>¶</sup>BHF Glasgow Cardiovascular Research Centre, University of Glasgow, Glasgow, United Kingdom; </div><div class="fm-affl" id="aff7" lang="en"><sup>**</sup>Institute of Clinical Pathology, Medical University of Vienna, Vienna, Austria; and </div><div class="fm-affl" id="aff8" lang="en"><sup>††</sup>Nuffield Department of Medicine, University of Oxford, Oxford, United Kingdom</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm140333909506848"><strong>Correspondence:</strong> Dr. John Feehally, The John Walls Renal Unit, Leicester General Hospital, Gwendolen Road, Leicester LE5 4PW, UK.,  Phone: 44-116-258-4132; Fax: 44-116-258-4764; E-mail: <a href="mailto:dev@null" data-email="ku.ca.el@72fj" class="oemail">ku.ca.el@72fj</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140333959722848_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Jan 18; Accepted 2010 May 11.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140333959722848_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2010 by the American Society of Nephrology</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3013538/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               [Supplemental Data]
            </div>
            
            <div><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_index.html" data-ga-action="click_feat_suppl">ASN.2010010076_index.html</a><span style="color:gray"> (1.1K)</span></div><div class="small guid">GUID:&nbsp;BA70C1A7-9C4E-4AE9-AD6A-1A7942A601E5</div>
            <div><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_FigureS1.tif" data-ga-action="click_feat_suppl">ASN.2010010076_FigureS1.tif</a><span style="color:gray"> (362K)</span></div><div class="small guid">GUID:&nbsp;C10504FF-24CE-40DF-BEBD-3F5ADAD57BB9</div>
            <div><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_FigureS2.tif" data-ga-action="click_feat_suppl">ASN.2010010076_FigureS2.tif</a><span style="color:gray"> (425K)</span></div><div class="small guid">GUID:&nbsp;EE15AB32-D6BC-4992-9AEC-AFA25536D29C</div>
            <div><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_FigureS3.tif" data-ga-action="click_feat_suppl">ASN.2010010076_FigureS3.tif</a><span style="color:gray"> (481K)</span></div><div class="small guid">GUID:&nbsp;861AEC21-C380-4E10-B9BB-C2BED23213F7</div>
            <div><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_FigureS4.tif" data-ga-action="click_feat_suppl">ASN.2010010076_FigureS4.tif</a><span style="color:gray"> (481K)</span></div><div class="small guid">GUID:&nbsp;29CBC366-C68E-4FBB-8C4C-3376C4B65DEC</div>
         </div></dd></dl></div><div id="idm140333912825936" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333912825936title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Demographic and family studies support the existence of a genetic contribution to the pathogenesis of IgA nephropathy, but results from genetic association studies of candidate genes are inconsistent. To systematically survey common genetic variation in this disease, we performed a genome-wide analysis in a cohort of patients with IgA nephropathy selected from the UK Glomerulonephritis DNA Bank. We used two groups of controls: parents of affected individuals and previously genotyped, unaffected, ancestry-matched individuals from the 1958 British Birth Cohort and the UK Blood Service. We genotyped 914 affected or family controls for 318,127 single nucleotide polymorphisms (SNPs). Filtering for low genotype call rates and inferred non-European ancestry left 533 genotyped individuals (187 affected children) for the family-based association analysis and 244 cases and 4980 controls for the case-control analysis. A total of 286,200 SNPs with call rates &gt;95% were available for analysis. Genome-wide analysis showed a strong signal of association on chromosome 6p in the region of the MHC (<em>P</em> = 1 × 10<sup>−9</sup>). The two most strongly associated SNPs showed consistent association in both family-based and case-control analyses. HLA imputation analysis showed that the strongest association signal arose from a combination of DQ loci with some support for an independent HLA-B signal. These results suggest that the HLA region contains the strongest common susceptibility alleles that predispose to IgA nephropathy in the European population.</p></div></div><div id="idm140333914033584" class="tsec sec headless whole_rhythm"><p id="__p2" class="p p-first">IgA nephropathy (IgAN) is the most common form of glomerulonephritis and is an important cause of ESRD. Clinical presentation is usually with hematuria with a variable degree of proteinuria; progression to ESRD occurs in ∼25% of those affected in the 20 years after diagnosis.<sup><sup><a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234213">1</a></sup></sup> Pathologically, IgAN is characterized by deposition of polymeric IgA1 in the renal mesangium, accompanied by mesangial hypercellularity, mesangial matrix expansion, and varying degrees of glomerulosclerosis and interstitial fibrosis. The pathogenesis is unclear, and diversity in the clinico-pathologic disease spectrum has suggested etiologic heterogeneity.</p><p id="__p3">A number of observations suggest that there is a significant genetic contribution to the pathogenesis of IgAN. These include differences in the prevalence of IgAN across ancestry groups, increased disease prevalence in relatives of affected individuals, and reports of large pedigrees containing multiple affected individuals.<sup><sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234196">2</a></sup></sup> To date, the strongest direct evidence for the existence of genetic factors in the development and/or progression of IgAN has been provided by linkage analyses in such families.<sup><sup><a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_703145421">3</a></sup><sup><a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234208"></a></sup>–<sup><a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234183">5</a></sup></sup> A major disease locus designated <em>IGAN1</em> on chromosome 6q22–23 was defined in a study of white families.<sup><sup><a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234199">6</a></sup></sup> However, only 60% of the families in the study were linked to this locus, and, to date, no disease-specific genes associated with disease/susceptibility have been identified within the linkage interval. This locus was excluded by linkage analysis in a Japanese family,<sup><sup><a href="#B7" rid="B7" class=" bibr popnode">7</a></sup></sup> whereas in Italian families, linkage has been reported at two different IgAN loci on 4q26–31 and 17q12–22.<sup><sup><a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234203">8</a></sup></sup> A Canadian family study has localized another IgAN susceptibility locus to chromosome 2q36,<sup><sup><a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234204">9</a></sup></sup> whereas analysis of a large Lebanese kindred did not provide evidence for linkage at 6q22–23, 2q36, or 4q22–31.<sup><sup><a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234212">10</a></sup></sup></p><p id="__p4">These studies therefore suggested that at least part of the genetic basis of IgAN is specified by variants that are confined to specific families or discrete populations (and therefore might not be fundamental to the disease process as a whole), and it is unclear whether there are common IgAN susceptibility haplotypes that operate across large populations. This led to increasing interest in defining the existence and extent of IgAN susceptibility loci in large populations using genetic association. Most studies conducted to date examined small numbers of candidate genes, although some surveyed candidate genes more extensively<sup><sup><a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234193">11</a></sup></sup> or conducted limited genome-wide analyses.<sup><sup><a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234211">12</a></sup></sup> Although a number of potential associations with polymorphisms at candidate gene loci were reported (reviewed in references <sup><a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234179">2</a></sup><sup>,</sup><sup><a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234178">13</a></sup><sup>,</sup><sup><a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234181">14</a></sup>), these associations have not all been reproduced, and there is no systematic genome-wide analysis of susceptibility to IgAN.</p><p id="__p5">To evaluate evidence for genetic association with IgAN, we undertook a genome-wide analysis in patients with European ancestry using a panel of ∼300,000 tagging single nucleotide polymorphisms (SNPs), designed to maximize coverage of common human haplotypes based on linkage disequilibrium intervals.<sup><sup><a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234190">15</a></sup></sup></p><p id="__p6" class="p">We performed both family-based and case-control association studies, using cases and nuclear families ascertained from the UK MRC/Kidney Research UK National DNA Bank for Glomerulonephritis and controls from the 1958 British Birth Cohort and the UK National Blood Service. Strong signals of association were observed in the MHC on chromosome 6p but not other loci, indicating that, in the UK population, the MHC contains the strongest common susceptibility alleles for IgAN.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Results</h2><div id="__sec2" class="sec sec-first"><h3 id="__sec2title">Genome-Wide Association Analysis</h3><p id="__p7" class="p p-first">In total, 318,127 autosomal and X-chromosomal SNPs were typed in 914 samples that self-reportedly comprised 181 complete parent-child trios, 24 parent-child pairs, and 323 unrelated samples. Control genotypes for 5069 individuals from the 1958 British Birth Cohort and UK Blood Service that have been genotyped using the same platform were obtained for the 314,656 SNPs that were typed in both studies.</p><p id="__p8">We next applied a series of quality controls to the data to ensure reliable population structure and genotyping. Principal components analysis identified 37 putative non-Europeans in the IgA nephropathy samples compared with the North West European, East Asian, South Asian, and West African reference populations<sup><sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234177">16</a></sup></sup> (see Supplementary Figure S1). After quality control filtering of individuals with low genotype call-rates and non-European ancestry, and revision of family structures based on identity-by-state information, there were 187 affected children available for family-based association analysis and 244 cases with 4980 controls for case-control analysis. After removal of SNPs with &lt;95% call rates, a minimum allele frequency (MAF) &lt;5%, or major deviation from Hardy-Weinberg equilibrium (<em>P</em> value &lt;1 × 10<sup>−6</sup> in controls), a total of 286,200 SNPs were available for analysis in both the family-based and case-control data; 3 SNPs that showed extreme heterogeneity of association (<em>P</em> &lt; 1 × 10<sup>−6</sup>) in the two studies were eliminated from further analysis.</p><p id="__p9">In the family-based study, testing for linkage and association was performed using the program TRANSMIT; in the case-control study, logistic regression analysis was performed using Stata<sup>TM</sup> (version 9.2). To maximize the power to detect association signals, we also combined the results from the family-based and case-control samples using meta-analysis techniques. The distribution of the association statistics (asymptotically a one degree of freedom, χ<sup>2</sup> distribution) showed little evidence of overdispersion (Supplementary Figure S2, left panel) with a genomic control parameter (λ) equal to 1.02; this indicates that there was no evidence of population stratification (admixture) or cryptic relatedness or other sources of systematic bias that might inflate the type 1 (false positive) error. The λ value for the family-based statistics alone was 1.02 and for the case-control statistics was 1.02, which confirms that the type 1 error was tightly controlled in both arms of the study.</p><p id="__p10">The genome-wide association plot of the association statistics for the combined data are shown in <a href="/pmc/articles/PMC3013538/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. There is a strong signal of association on chromosome 6p defined by a group of strongly associated SNPs (minimum <em>P</em> value = 1 × 10<sup>−9</sup>). The most strongly associated SNPs in this group, rs3115573 and rs3130315, showed clear and consistent evidence for association in both the family-based and case-control data (<a href="/pmc/articles/PMC3013538/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These two SNPs were in absolute linkage disequilibrium, and the small differences in association detected by each SNP were explained by minor differences in genotype call-rates (Supplementary Table S1).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3013538/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333910553424" class="figure"><a href="/pmc/articles/PMC3013538/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3013538_asn0101044910001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is asn0101044910001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3013538/bin/asn0101044910001.jpg"></a></div><div id="largeobj_idm140333910553424" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3013538/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3013538/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p11">Strong signal of association on chromosome 6p with IgA nephropathy. “Manhattan” plot summarizing the association of IgA nephropathy with 286,197 SNPs distributed across the genome. The support for genetic association is expressed on the <em>y</em>-axis as (−log10 <em>P</em> value); the <em>x</em>-axis shows the relative physical position of each SNP within each chromosome.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p12">Genotypic and statistical data for SNPs on chromosome 6p that were most strongly associated with IgAN</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333954738752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" valign="middle" align="center">SNP</th><th rowspan="2" colspan="1" valign="middle" align="center">Location (bp)</th><th rowspan="2" colspan="1" valign="middle" align="center">Alleles Major/Minor</th><th rowspan="2" colspan="1" valign="middle" align="center">MAF</th><th colspan="2" rowspan="1" align="center">Family Data (186 Nuclear Families)<hr></th><th colspan="2" rowspan="1" align="center">Case-Control Data (244 Cases; 4980 Controls)<hr></th><th colspan="2" rowspan="1" align="center">Family + Case-Control Meta-Analysis<hr></th></tr><tr><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th><th rowspan="1" colspan="1" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">rs3115573</td><td rowspan="1" colspan="1" align="center">32,326,821</td><td rowspan="1" colspan="1" align="center">A/G</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">1.88 (1.37 to 2.57)</td><td rowspan="1" colspan="1" align="center">6 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.55 (1.29 to 1.85)</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.62 (1.39 to 1.90)</td><td rowspan="1" colspan="1" align="center">1 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">rs3130315</td><td rowspan="1" colspan="1" align="center">32,328,663</td><td rowspan="1" colspan="1" align="center">G/A</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">1.87 (1.37 to 2.56)</td><td rowspan="1" colspan="1" align="center">7 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.55 (1.29 to 1.86)</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.62 (1.39 to 1.90)</td><td rowspan="1" colspan="1" align="center">1 × 10<sup>−9</sup></td></tr></tbody></table></div><div id="largeobj_idm140333954738752" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3013538/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="__p13">A conditional association analysis of the combined family-based and case-control data was performed to search for signals secondary to that detected by the lead SNP (rs3115573) across the 997 SNPs that mapped to the greater MHC region on chromosome 6p (29.0 to 33.4 million bp). Three SNPs (rs13209234, rs11244, and rs2857106) showed significant conditional associations (<em>P</em> &lt; 0.05 after correction for multiple testing); these SNPs showed very little linkage disequilibrium with the lead SNP (<em>r</em><sup>2</sup> ≤ 0.02) or with each other (0.03 ≤ <em>r</em><sup>2</sup> ≤ 0.32). These results suggest that multiple discrete loci conferring susceptibility to IgAN map to the greater MHC region.</p><p id="__p14" class="p p-last">Further review of the genome-wide analysis showed no evidence of statistically significant association outside the MHC. Supplementary Table S2 lists the association statistics for the combined datasets at 72 loci, which were selected because of previous reported associations in case control studies or because of plausible candidacy based on current understanding of the pathogenesis of IgAN. None of these statistics was close to the significance of the signal at HLA. <em>Post hoc</em> power calculations suggested that the study was powered (≥80%) to detect common susceptibility alleles (MAF ≥ 0.2) with genome-wide levels of statistical significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) that increase risk by at least 70%. (<em>i.e.</em>, odds ratio ≥ 1.7) and low-frequency alleles (MAF ≥ 0.05) that increase risk by at least 120% (<em>i.e.</em>, odds ratio ≥ 2.2).</p></div><div id="__sec3" class="sec sec-last"><h3 id="__sec3title">HLA Imputation Analysis</h3><p id="__p15" class="p p-first">Multiple SNPs were associated with IgAN across the MHC region with the strongest support (<em>P</em> &lt; 10<sup>−6</sup>) in the interval 31.7 to 32.5 million bp (<a href="/pmc/articles/PMC3013538/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). To explore the chromosome 6p association signal further, we applied HLA imputation techniques to infer SNP association results in terms of classic HLA alleles,<sup><sup><a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234215">17</a></sup></sup> pooling independent (<em>i.e.</em>, founder) haplotypes inferred in families with those inferred in cases and controls. Imputed counts of HLA alleles for HLA-A, HLA-B, HLA-C, DQ-A, DQ-B, and DR-B in cases and controls are shown in Supplementary Table S3. Analysis of the individual HLA loci showed strong association at HLA-B (<em>P</em> &lt; 10<sup>−8</sup>), HLA-DRB (<em>P</em> &lt; 10<sup>−9</sup>), HLA-DQA (<em>P</em> &lt; 10<sup>−8</sup>), and HLA-DQB (<em>P</em> &lt; 10<sup>−9</sup>) (<a href="/pmc/articles/PMC3013538/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>; Supplementary Table S3). There is strong linkage disequilibrium between these loci, so a series of simultaneous analyses of the six HLA loci was performed (<a href="/pmc/articles/PMC3013538/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Models 1 to 6 show that the effects of dropping a single HLA locus are similar, consistent with individual effects being masked by strong linkage disequilibrium between the loci. Analysis of nested models suggested that the strongest association was detected by a combination of DRB, DQ, and HLA-B loci (models 7 to 9; <a href="/pmc/articles/PMC3013538/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) However, the strongest association was with HLA-DQ, and imputation of alleles at this locus showed significant positive association with DQB1*0501 (odds ratio [OR] = 1.51; 95% confidence interval [CI], 1.14 to 2.01) and negative association with DQB1*0201 (OR = 0.66; 95% CI, 0.5 to 0.87; <a href="/pmc/articles/PMC3013538/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3013538/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333917632480" class="figure"><a href="/pmc/articles/PMC3013538/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3013538_asn0101044910002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is asn0101044910002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3013538/bin/asn0101044910002.jpg"></a></div><div id="largeobj_idm140333917632480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3013538/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3013538/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><p id="__p16">Multiple SNPs are associated with IgA nephropathy across the MHC region on chromosome 6p. Support for genetic association (−log10 <em>P</em> value) is plotted against physical position (million bp; NCBI build 36). Results for individual SNPs are shown in blue; results for imputed classic HLA loci are shown by red lines. The density of SNPs in the vicinity of HLA-DRB is relatively low in the HumanHap300 BeadChip.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p17">Association analysis of IgAN with classic HLA loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333950863360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" align="left">Model</th><th rowspan="1" colspan="1" align="center">Tested Loci</th><th rowspan="1" colspan="1" align="center"><em>F</em>-Statistic</th><th rowspan="1" colspan="1" align="center">df1</th><th rowspan="1" colspan="1" align="center">df2</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">HLAA, HLAB, HLAC, DQA, DQB, DRB</td><td rowspan="1" colspan="1" align="center">3.05</td><td rowspan="1" colspan="1" align="center">82</td><td rowspan="1" colspan="1" align="center">5428</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−18</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="left">HLAA</td><td rowspan="1" colspan="1" align="center">1.82</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">5496</td><td rowspan="1" colspan="1" align="center">3 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">3</td><td rowspan="1" colspan="1" align="left">HLAB</td><td rowspan="1" colspan="1" align="center">2.76</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">5489</td><td rowspan="1" colspan="1" align="center">3 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">4</td><td rowspan="1" colspan="1" align="left">HLAC</td><td rowspan="1" colspan="1" align="center">1.85</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">5495</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">5</td><td rowspan="1" colspan="1" align="left">DQA</td><td rowspan="1" colspan="1" align="center">4.41</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">5504</td><td rowspan="1" colspan="1" align="center">2 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="left">DQB</td><td rowspan="1" colspan="1" align="center">2.46</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">5500</td><td rowspan="1" colspan="1" align="center">6 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">7</td><td rowspan="1" colspan="1" align="left">DRB</td><td rowspan="1" colspan="1" align="center">2.53</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">5494</td><td rowspan="1" colspan="1" align="center">7 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">8</td><td rowspan="1" colspan="1" align="left">DQA, DQB</td><td rowspan="1" colspan="1" align="center">3.6</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">5494</td><td rowspan="1" colspan="1" align="center">1 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">9</td><td rowspan="1" colspan="1" align="left">DQA, DQB, DRB</td><td rowspan="1" colspan="1" align="center">3.22</td><td rowspan="1" colspan="1" align="center">32</td><td rowspan="1" colspan="1" align="center">5478</td><td rowspan="1" colspan="1" align="center">3 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" align="left">10</td><td rowspan="1" colspan="1" align="left">HLAB, DQA, DQB, DRB</td><td rowspan="1" colspan="1" align="center">3.21</td><td rowspan="1" colspan="1" align="center">53</td><td rowspan="1" colspan="1" align="center">5457</td><td rowspan="1" colspan="1" align="center">6 × 10<sup>−14</sup></td></tr></tbody></table></div><div id="largeobj_idm140333950863360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3013538/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140333911069248"><p id="__p18" class="p p-first-last">The results are derived from a series of logistic regression analyses comparing a general model that includes all six classic HLA loci with nested models to show the effects of dropping one or more loci. Haplotypes for classic HLA loci were imputed (predicted) using chromosome 6p SNP genotype data. df1 and df2 denote the numerator and denominator degrees of freedom associated with the <em>F</em>-statistic comparing general and nested model.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p19">Association analysis of HLA-DQB imputed alleles</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140333917951856" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="2" colspan="1" valign="middle" align="center">HLA-DQB</th><th rowspan="2" colspan="1" valign="middle" align="center">OR</th><th colspan="3" rowspan="1" align="center">Linearized<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">95% CI</th></tr><tr><th rowspan="1" colspan="1" align="center">SE</th><th rowspan="1" colspan="1" align="center"><em>t</em></th><th rowspan="1" colspan="1" align="center"><em>P</em>&gt; <em>t</em></th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" align="left">*0201</td><td rowspan="1" colspan="1" align="center">0.66</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">−2.89</td><td rowspan="1" colspan="1" align="center">0.004</td><td rowspan="1" colspan="1" align="center">0.50 to 0.87</td></tr><tr><td rowspan="1" colspan="1" align="left">*0301</td><td rowspan="1" colspan="1" align="center">1.26</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">1.69</td><td rowspan="1" colspan="1" align="center">0.091</td><td rowspan="1" colspan="1" align="center">0.96 to 1.65</td></tr><tr><td rowspan="1" colspan="1" align="left">*0303</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">−0.76</td><td rowspan="1" colspan="1" align="center">0.448</td><td rowspan="1" colspan="1" align="center">0.55 to 1.30</td></tr><tr><td rowspan="1" colspan="1" align="left">*0402</td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="center">0.20</td><td rowspan="1" colspan="1" align="center">−1.56</td><td rowspan="1" colspan="1" align="center">0.118</td><td rowspan="1" colspan="1" align="center">0.30 to 1.15</td></tr><tr><td rowspan="1" colspan="1" align="left">*0501</td><td rowspan="1" colspan="1" align="center">1.51</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">2.86</td><td rowspan="1" colspan="1" align="center">0.004</td><td rowspan="1" colspan="1" align="center">1.14 to 2.01</td></tr><tr><td rowspan="1" colspan="1" align="left">*0502</td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="center">0.17</td><td rowspan="1" colspan="1" align="center">−1.86</td><td rowspan="1" colspan="1" align="center">0.062</td><td rowspan="1" colspan="1" align="center">0.33 to 1.03</td></tr><tr><td rowspan="1" colspan="1" align="left">*0503</td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="center">0.30</td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">0.257</td><td rowspan="1" colspan="1" align="center">0.83 to 2.02</td></tr><tr><td rowspan="1" colspan="1" align="left">*0602</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">0.13</td><td rowspan="1" colspan="1" align="center">−1.41</td><td rowspan="1" colspan="1" align="center">0.157</td><td rowspan="1" colspan="1" align="center">0.58 to 1.09</td></tr><tr><td rowspan="1" colspan="1" align="left">*0603</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">−0.5</td><td rowspan="1" colspan="1" align="center">0.614</td><td rowspan="1" colspan="1" align="center">0.62 to 1.33</td></tr><tr><td rowspan="1" colspan="1" align="left">*0604</td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">0.16</td><td rowspan="1" colspan="1" align="center">−0.73</td><td rowspan="1" colspan="1" align="center">0.468</td><td rowspan="1" colspan="1" align="center">0.62 to 1.25</td></tr></tbody></table></div><div id="largeobj_idm140333917951856" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3013538/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140333918030224"><p id="__p20" class="p p-first-last">Reference allele = *0302.</p></div></div></div></div></div><div id="__sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Discussion</h2><p id="__p21" class="p p-first">Here we report the results of the first genome-wide association study of IgAN. The study encompassed both family-based and case-control methodology, using affected individuals and families ascertained from the UK Glomerulonephritis DNA Bank. The diagnosis of IgAN was supported in every case by renal biopsy. Rigorous genetic criteria were used to remove a small number of individuals from the analysis who, despite ascertainment as white Europeans, most likely contained non-European genetic admixture; analysis of the association statistics following these exclusions showed no evidence of population stratification.</p><p id="__p22">A strong SNP association signal was observed in both the family-based and case-control studies in the MHC, demonstrating the existence of a reproducible, statistically robust association of IgAN with HLA loci in the UK white population. Imputation techniques were used to infer classic HLA alleles, which showed that association signals were observed across the HLA-B, DRB1, DQA, and DQB loci.</p><p id="__p23">Previous studies of HLA association with IgAN in Europeans have sometimes, but not always, identified associations with the HLA class II genes<sup><sup><a href="#B18" rid="B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234198">18</a></sup><sup><a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234206"></a></sup><sup><a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234195"></a></sup><sup><a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234187"></a></sup><sup><a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304863353"></a></sup><sup><a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234205"></a></sup>–<sup><a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234194">24</a></sup></sup> but not consistently with any allele. Both the family-based and the case-control components of this analysis were larger than any previous study in Europeans, and both components showed very similar and highly significant associations with HLA genes; the strongest signals were observed at the HLA-DQ locus. An analysis of imputed HLA-DQB alleles in our data showed significant positive and negative associations with HLA-DQB*0501 and HLA-DQB*0201, respectively (<em>P</em> &lt; 0.005 for both). Interestingly, in Europeans, HLA-DQB*0201 lies in linkage disequilibrium with the HLA-B8-C4A*Q0-HLA-DRB1*0301 haplotype that is a risk allele for several autoimmune diseases.<sup><sup><a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234209">25</a></sup></sup> In keeping with this, our HLA imputation showed reduced frequencies of HLA-B*0801 and HLA-DRB1*0301 in IgAN cases, suggesting that individuals bearing this haplotype are less susceptible to IgAN.</p><p id="__p24">An association of IgAN with HLA-DR4 and HLA-DQ4, which are in strong linkage equilibrium, has been consistently identified in Japanese cohorts using both serologic and molecular typing strategies<sup><sup><a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234207">26</a></sup>,<sup><a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234182">27</a></sup></sup> (for review see reference<sup><a href="#B28" rid="B28" class=" bibr popnode">28</a></sup>). Furthermore, a more recent SNP-based study of IgAN in the Japanese population identified an association with the DRA locus that is in strong linkage disequilibrium with DQ, but DQ loci were not directly probed by any of the SNPs used.<sup><sup><a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234180">29</a></sup></sup> There are differences in the prevalence and gender distribution of IgAN in East Asians and Europeans; IgAN is more common in East Asians in whom it occurs equally in males and females, whereas it occurs predominantly in males in Europeans. Despite these differences, our study suggests that similar loci within the HLA region may predispose to IgAN in both populations. Further studies will be required to test this directly and to determine the molecular basis of predisposition.</p><p id="__p25">Conditional analysis of the SNP associations across the greater MHC in our study suggested the presence of more than one susceptibility locus within this region. The two most strongly associated SNPs, rs3115573 and rs3130315, map within 1.6 kb of each other, ∼27 kb upstream of the <em>neurogenic locus notch homolog protein 4</em> gene. Of the three secondary signals, rs13209234 maps to an intergenic interval 3.1 kb downstream of HLA-DRA and 69 kb downstream of HLA-DRB, rs11244 maps to the 3′ UTR of HLA-DQB, and rs2857106 maps to an intergenic interval 2.7 kb upstream of HLA-DQB and 2 kb downstream of the TAP2 gene encoding antigen peptide transporter 2. Further studies will be required to determine the mechanisms underlying these associations.</p><p id="__p26">Survey of the association statistics across the genome (<a href="/pmc/articles/PMC3013538/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) did not reveal any other association of comparable strength to that in the MHC. In keeping with this, the excess of χ<sup>2</sup> association statistics &gt;10.8 (<em>P</em> &lt; 0.001) in the χ<sup>2</sup> distribution plot can be seen to be almost entirely caused by the chromosome 6p association signal (Supplementary Figure S2, right panel). This indicates that, despite reported evidence of larger gene-specific risks at a number of loci in smaller candidate gene studies, at least in the UK population, any common susceptibility haplotypes at extra-HLA loci are likely to have substantially smaller effect sizes. We did not observe significant associations across any of the IgAN loci identified in linkage analyses of multiplex pedigrees, suggesting that, consistent with evidence of heterogeneity in these studies, linkage does not arise from co-inheritance of a common susceptibility haplotype but more likely discrete predisposing genotypes that are specific to certain of these pedigrees. None of the loci selected on the basis of putative mechanistic involvement in the pathogenesis of IgAN or previously reported genetic association (Supplementary Table S2) manifested an association statistic that was significant, the smallest values being of the order of <em>P</em> = 10<sup>−3</sup>, well below that at the HLA locus (<em>P</em> = 10<sup>−9</sup>). Although the size of the study places limits on the ability to detect or exclude small associations, we found little support for association at loci that have been reported in previous studies of IgAN. Although the Hap300 BeadChip provides good coverage of common genetic variation (75% coverage at <em>r</em><sup>2</sup> ≥ 0.8 evaluated in HapMap2),<sup><sup><a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234192">30</a></sup></sup> it is inevitable that some disease-associated variants were neither genotyped nor reliably tagged in this study. In the future, meta-analyses of combined datasets including ours, and further genotyping of this and other large collections will aid to confirm our conclusions of lack of strong association with IgAN at these loci.</p><p id="__p27" class="p p-last">Overall, our data provided clear evidence for the association of IgAN with genes at the HLA locus but not other regions of the genome. The full set of association results, and the genotype dataset, available at the European Genome-phenome Archive <a href="http://www.ebi.ac.uk/ega" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ebi.ac.uk/ega</a>, should be of use in combination with future studies to define the existence or otherwise of extra-HLA common susceptibility loci and to analyze the HLA association in greater detail.</p></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Concise Methods</h2><p id="__p28" class="p p-first">This study was conducted in accordance with the Declaration of Helsinki and approved by a UK multicenter research ethics committee (MREC) and local research ethics committees. Informed consent was obtained from all participants.</p><div id="__sec6" class="sec"><h3 id="__sec6title">Study Design, Patients, and Controls</h3><p id="__p29" class="p p-first">The MRC//Kidney Research UK National DNA Bank for Glomerulonephritis established collections in five common glomerular diseases, of which one is IgAN. Participants of presumed European ancestry were recruited through four UK centers (Glasgow, Leicester, London, and Oxford) through probands with renal biopsy-proven IgAN &lt;50 years of age at the time of diagnosis and &gt;18 years of age at the time of recruitment; gender distribution and recruitment by center are shown in Supplementary Table S4. Individuals with evidence of liver disease or Henoch-Schönlein purpura were excluded. Diagnosis was confirmed in all cases by direct review of renal biopsy histopathology reports and clinical case records. DNA was extracted from venous blood samples by standard methods.</p><p id="__p30" class="p p-last">Where available, samples were also collected from the parents of affected individuals and, in some cases, other nuclear family members, to enable a study of family-based association. Individuals for whom suitable family DNA samples could not be obtained were entered into a separate case-control analysis, using control genotypes for individuals from the 1958 British Birth Cohort and UK Blood Service that have been genotyped using the same platform. The cases used in the family-based and case-control studies did not overlap; the two studies were analyzed separately, after which the data were combined by meta-analysis.</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Genotyping</h3><p id="__p31" class="p p-first-last">Nine hundred and fourteen individuals were genotyped for 318,127 autosomal and X-chromosomal SNPs with the Illumina Sentrix HumanHap300 BeadChip<sup><sup><a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234191">31</a></sup></sup>; genotypes were automatically called using the proprietary BeadStudio software. This chip is composed of a panel of tagging SNPs based on the phase I HapMap analysis (<a href="http://www.HapMap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.HapMap.org</a>). Control genotypes for 5069 individuals (from the 1958 British Birth Cohort and UK Blood Service) were determined using a custom Illumina 1.2M Beadchip (<a href="http://www.wtccc.org.uk/ccc2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk/ccc2/</a>); genotypes were automatically called using the Illuminus algorithm.<sup><sup><a href="#B32" rid="B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234201">32</a></sup></sup> This chip includes 314,656 SNPs that are available on the HumanHap300 BeadChip.</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Statistical Analysis</h3><p id="__p32" class="p p-first-last">SNPs with genotype call-rates &lt;95% and samples with call-rates &lt;95% were eliminated from the analysis. Incompatibilities between recorded gender and that inferred from X-chromosome genotype data were resolved in favor of the genotype data. Family relationships were confirmed or revised based on the results of an identity-by-state (IBS) analysis. An ancestry analysis was carried out using the EIGENSTRAT2.0 software<sup><sup><a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234189">16</a></sup></sup> using HapMap data (CEU, YRI, GIH, JPT, and CHB) as representatives of West European, West African, South Asian, and East Asian populations to infer ancestry informative principal components, which were projected onto the case and control samples. Putative non-European samples were flagged as “outliers” and eliminated from subsequent analyses.</p></div><div id="__sec9" class="sec"><h3 id="__sec9title">Autosomal SNP Association Analysis</h3><p id="__p33" class="p p-first-last">A family-based test of linkage and association was performed using the computer program TRANSMIT.<sup><sup><a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234216">33</a></sup></sup> TRANSMIT calculates a score-test statistic that omits terms that are sensitive to population stratification (admixture) and incorporates nuclear families with incomplete genotype data and multiple affected offspring. Estimates of the genetic effect size (per-allele odds ratio and the SE of this estimate) were calculated from the estimated transmission proportion of alleles to affected offspring and the equivalent number of informative transmissions.<sup><sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234186">34</a></sup></sup> Case-control data were analyzed using the logistic regression function in Stata™ (version 9.2), with indicator variables defined to model a multiplicative genetic risk model (which models additive genetic effects on the log risk scale). Finally, the estimates of the per-allele odds ratio from the family-based (TRANSMIT) and case-control (logistic regression) analyses were combined in a fixed-effects meta-analysis using inverse-variance weighting with standard formulae.<sup><sup><a href="#B35" rid="B35" class=" bibr popnode">35</a></sup></sup></p></div><div id="__sec10" class="sec"><h3 id="__sec10title">X-Linked SNP Association Analysis</h3><p id="__p34" class="p p-first">Because TRANSMIT does not compute appropriate statistics for X-linked SNP data, the family-based and case-control X-linked SNP data were pooled for logistic regression analysis with indicator variables defined to model a multiplicative genetic risk model, with an assumption of common male-hemizygote and female-homozygote risks. Robust sandwich estimation of the variance<sup><sup><a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234185">36</a></sup></sup> was used to model nuclear family level clustering of genotypes; this approach has previously been shown to efficiently absorb variance inflation (overdispersion) caused by relatedness.<sup><sup><a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234202">37</a></sup></sup></p><p id="__p35" class="p p-last">Hardy-Weinberg equilibrium was tested in unrelated controls using an exact algorithm implemented in the Stata procedure <em>genhwi</em>.<sup><sup><a href="#B38" rid="B38" class=" bibr popnode">38</a></sup></sup> X-linked SNP data in trios and cases and controls were pooled and analyzed with logistic regression models with indicator variables defined to model a multiplicative genetic risk model and assuming common male-hemizygote and female-homozygote risks. Estimates of the per-allele odds ratio from family-based and case-control analyses were combined in a fixed-effects meta-analysis using inverse-variance weighting with standard formulae.<sup><sup><a href="#B38" rid="B38" class=" bibr popnode">38</a></sup></sup> Variance inflation (overdispersion) of the test statistics, which under the null hypothesis are asymptotically distributed as χ<sup>2</sup> statistics with 1 degree of freedom, was assessed by inspection of quantile-quantile plots (Supplementary Figure S2) and by calculation of a genomic control parameter λ = minimum [1,median(χ<sup>2</sup>)/0.456].<sup><sup><a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234188">39</a></sup></sup></p></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">HLA Imputation</h3><p id="__p36" class="p p-first-last">Genotypes for a set of 999 SNPs spanning the greater MHC region on chromosome 6p (29.0 to 33.4 million bp) were extracted for HLA imputation. Initially, missing SNP genotypes were imputed and SNP haplotypes were phased using the Beagle program<sup><sup><a href="#B40" rid="B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234197">40</a></sup>,<sup><a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234200">41</a></sup></sup>; this method can simultaneously accommodate both family-based and unrelated data. HLA prediction<sup><sup><a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234184">34</a></sup></sup> was performed on each haplotype output from Beagle, with 30 CEPH North West European (CEU) families from the HapMap (<a href="http://www.HapMap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.HapMap.org</a>) used as training data. Subsets of SNPs were chosen for prediction based on cross-validation performance within the training data. Prediction accuracy within the training data were 94.86, 91.62, 96.63, 92.49, 95.56, and 98.87%, using 20, 21, 13, 51, 15, and 40 SNPs for HLA-A, HLA-B, HLA-C, DRB, DQA, and DQB, respectively. We used these same SNPs to predict HLA types in our data, with the predicted alleles being the ones with maximum posterior probability. For families, four founder haplotypes, two corresponding to the transmitted (to the affected child) and two corresponding to the untransmitted haplotypes, were recorded.<sup><sup><a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_206234210">42</a></sup></sup> For unrelated cases and controls, two independent haplotypes were recorded for each individual.</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Disclosures</h2><p id="__p37" class="p p-first-last">None.</p></div><div id="__sec13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>[Supplemental Data]</strong>
            
            <div class="sup-box half_rhythm" id="idm140333913011792"><a href="/pmc/articles/PMC3013538/bin/ASN.2010010076_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
            
            
            
         </div></div><div id="idm140333917685344" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333917685344title">Acknowledgments</h2><div class="sec"><p id="__p40">The development of the MRC/Kidney Research UK National DNA Bank for Glomerulonephritis was supported by the UK Medical Research Council and Kidney Research UK. The authors thank the patients, their relatives, and their physicians who enabled the collection of the UK National DNA Bank for Glomerulonephritis. This work was supported by Kidney Research UK (SP/DNAmapping/2006), the Wellcome Trust (075491/Z/04), and European Union Marie Curie Excellence Chair to A.J.R. (MEXC-CT-2006-042742). This study makes use of data generated by the Wellcome Trust Case Control Consortium (WTCCC). A full list of the investigators who contributed to the generation of the WTCCC data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk</a>.</p></div></div><div id="idm140333913312160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333913312160title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140333913312032"></p><p id="__p38" class="p p-first-last">Published online ahead of print. Publication date available at <a href="http://www.jasn.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.jasn.org</a>.</p><p></p><p class="fn sec" id="idm140333914758688"></p><p id="__p39" class="p p-first-last">Supplemental information for this article is available online at <a href="http://www.jasn.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.jasn.org/</a>.</p><p></p></div></div><div id="idm140333917682912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140333917682912title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">
Barratt J, Feehally J: <span class="ref-title">IgA nephropathy</span>. <span class="ref-journal">J Am Soc Nephrol</span> <span class="ref-vol">16</span>: 2088–2097, 2005 [<a href="/pubmed/15930092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=IgA+nephropathy&amp;author=J+Barratt&amp;author=J+Feehally&amp;volume=16&amp;publication_year=2005&amp;pages=2088-2097&amp;pmid=15930092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="mixed-citation">
Hsu SI: <span class="ref-title">Racial and genetic factors in IgA nephropathy</span>. <span class="ref-journal">Semin Nephrol</span> <span class="ref-vol">28</span>: 48–57, 2008 [<a href="/pubmed/18222346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Nephrol&amp;title=Racial+and+genetic+factors+in+IgA+nephropathy&amp;author=SI+Hsu&amp;volume=28&amp;publication_year=2008&amp;pages=48-57&amp;pmid=18222346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="mixed-citation">
Izzi C, Sanna-Cherchi S, Prati E, Belleri R, Remedio A, Tardanico R, Foramitti M, Guerini S, Viola BF, Movilli E, Beerman I, Lifton R, Leone L, Gharavi A, Scolari F: <span class="ref-title">Familial aggregation of primary glomerulonephritis in an Italian population isolate: Valtrompia study</span>. <span class="ref-journal">Kidney Int</span> <span class="ref-vol">69</span>: 1033–1040, 2006 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5849459/">PMC free article</a>]</span> [<a href="/pubmed/16528253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Familial+aggregation+of+primary+glomerulonephritis+in+an+Italian+population+isolate:+Valtrompia+study&amp;author=C+Izzi&amp;author=S+Sanna-Cherchi&amp;author=E+Prati&amp;author=R+Belleri&amp;author=A+Remedio&amp;volume=69&amp;publication_year=2006&amp;pages=1033-1040&amp;pmid=16528253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="mixed-citation">
Julian BA, Quiggins PA, Thompson JS, Woodford SY, Gleason K, Wyatt RJ: <span class="ref-title">Familial IgA nephropathy. Evidence of an inherited mechanism of disease</span>. <span class="ref-journal">N Engl J Med</span> <span class="ref-vol">312</span>: 202–208, 1985 [<a href="/pubmed/3855328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Familial+IgA+nephropathy.+Evidence+of+an+inherited+mechanism+of+disease&amp;author=BA+Julian&amp;author=PA+Quiggins&amp;author=JS+Thompson&amp;author=SY+Woodford&amp;author=K+Gleason&amp;volume=312&amp;publication_year=1985&amp;pages=202-208&amp;pmid=3855328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="mixed-citation">
Scolari F: <span class="ref-title">Inherited forms of IgA nephropathy</span>. <span class="ref-journal">J Nephrol</span> <span class="ref-vol">16</span>: 317–320, 2003 [<a href="/pubmed/12768083" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Nephrol&amp;title=Inherited+forms+of+IgA+nephropathy&amp;author=F+Scolari&amp;volume=16&amp;publication_year=2003&amp;pages=317-320&amp;pmid=12768083&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="mixed-citation">
Gharavi AG, Yan Y, Scolari F, Schena FP, Frasca GM, Ghiggeri GM, Cooper K, Amoroso A, Viola BF, Battini G, Caridi G, Canova C, Farhi A, Subramanian V, Nelson-Williams C, Woodford S, Julian BA, Wyatt RJ, Lifton RP: <span class="ref-title">IgA nephropathy, the most common cause of glomerulonephritis, is linked to 6q22–23</span>. <span class="ref-journal">Nat Genet</span> <span class="ref-vol">26</span>: 354–357, 2000 [<a href="/pubmed/11062479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=IgA+nephropathy,+the+most+common+cause+of+glomerulonephritis,+is+linked+to+6q22–23&amp;author=AG+Gharavi&amp;author=Y+Yan&amp;author=F+Scolari&amp;author=FP+Schena&amp;author=GM+Frasca&amp;volume=26&amp;publication_year=2000&amp;pages=354-357&amp;pmid=11062479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="mixed-citation">
Tsukaguchi H: <span class="ref-title">A genetic mapping for a familial IgA nephropathy</span>. <span class="ref-journal">Nephrology (Carlton)</span> <span class="ref-vol">9</span>[<span class="ref-iss">Suppl 2</span>]: A65, 2004 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrology+(Carlton)&amp;title=A+genetic+mapping+for+a+familial+IgA+nephropathy&amp;author=H+Tsukaguchi&amp;volume=9&amp;issue=Suppl+2&amp;publication_year=2004&amp;pages=A65&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="mixed-citation">
Bisceglia L, Cerullo G, Forabosco P, Torres DD, Scolari F, Di Perna M, Foramitti M, Amoroso A, Bertok S, Floege J, Mertens PR, Zerres K, Alexopoulos E, Kirmizis D, Ermelinda M, Zelante L, Schena FP; European IgAN Consortium: <span class="ref-title">Genetic heterogeneity in Italian families with IgA nephropathy: Suggestive linkage for two novel IgA nephropathy loci</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">79</span>: 1130–1134, 2006 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1698717/">PMC free article</a>]</span> [<a href="/pubmed/17186473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+heterogeneity+in+Italian+families+with+IgA+nephropathy:+Suggestive+linkage+for+two+novel+IgA+nephropathy+loci&amp;author=L+Bisceglia&amp;author=G+Cerullo&amp;author=P+Forabosco&amp;author=DD+Torres&amp;author=F+Scolari&amp;volume=79&amp;publication_year=2006&amp;pages=1130-1134&amp;pmid=17186473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="mixed-citation">
Paterson AD, Liu XQ, Wang K, Magistroni R, Song X, Kappel J, Klassen J, Cattran D, St George-Hyslop P, Pei Y: <span class="ref-title">Genome-wide linkage scan of a large family with IgA nephropathy localizes a novel susceptibility locus to chromosome 2q36</span>. <span class="ref-journal">J Am Soc Nephrol</span> <span class="ref-vol">18</span>: 2408–2415, 2007 [<a href="/pubmed/17634434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Genome-wide+linkage+scan+of+a+large+family+with+IgA+nephropathy+localizes+a+novel+susceptibility+locus+to+chromosome+2q36&amp;author=AD+Paterson&amp;author=XQ+Liu&amp;author=K+Wang&amp;author=R+Magistroni&amp;author=X+Song&amp;volume=18&amp;publication_year=2007&amp;pages=2408-2415&amp;pmid=17634434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="mixed-citation">
Karnib HH, Sanna-Cherchi S, Zalloua PA, Medawar W, D'Agati VD, Lifton RP, Badr K, Gharavi AG: <span class="ref-title">Characterization of a large Lebanese family segregating IgA nephropathy</span>. <span class="ref-journal">Nephrol Dial Transplant</span> <span class="ref-vol">22</span>: 772–777, 2007 [<a href="/pubmed/17172253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=Characterization+of+a+large+Lebanese+family+segregating+IgA+nephropathy&amp;author=HH+Karnib&amp;author=S+Sanna-Cherchi&amp;author=PA+Zalloua&amp;author=W+Medawar&amp;author=VD+D'Agati&amp;volume=22&amp;publication_year=2007&amp;pages=772-777&amp;pmid=17172253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="mixed-citation">
Liu XQ, Paterson AD, He N, St George-Hyslop P, Rauta V, Gronhagen-Riska C, Laakso M, Thibaudin L, Berthoux F, Cattran D, Pei Y: <span class="ref-title">IL5RA and TNFRSF6B gene variants are associated with sporadic IgA nephropathy</span>. <span class="ref-journal">J Am Soc Nephrol</span> <span class="ref-vol">19</span>: 1025–1033, 2008 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2386732/">PMC free article</a>]</span> [<a href="/pubmed/18256354" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=IL5RA+and+TNFRSF6B+gene+variants+are+associated+with+sporadic+IgA+nephropathy&amp;author=XQ+Liu&amp;author=AD+Paterson&amp;author=N+He&amp;author=P+St+George-Hyslop&amp;author=V+Rauta&amp;volume=19&amp;publication_year=2008&amp;pages=1025-1033&amp;pmid=18256354&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="mixed-citation">
Obara W, Iida A, Suzuki Y, Tanaka T, Akiyama F, Maeda S, Ohnishi Y, Yamada R, Tsunoda T, Takei T, Ito K, Honda K, Uchida K, Tsuchiya K, Yumura W, Ujiie T, Nagane Y, Nitta K, Miyano S, Narita I, Gejyo F, Nihei H, Fujioka T, Nakamura Y: <span class="ref-title">Association of single-nucleotide polymorphisms in the polymeric immunoglobulin receptor gene with immunoglobulin A nephropathy (IgAN) in Japanese patients</span>. <span class="ref-journal">J Hum Genet</span> <span class="ref-vol">48</span>: 293–299, 2003 [<a href="/pubmed/12740691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=Association+of+single-nucleotide+polymorphisms+in+the+polymeric+immunoglobulin+receptor+gene+with+immunoglobulin+A+nephropathy+(IgAN)+in+Japanese+patients&amp;author=W+Obara&amp;author=A+Iida&amp;author=Y+Suzuki&amp;author=T+Tanaka&amp;author=F+Akiyama&amp;volume=48&amp;publication_year=2003&amp;pages=293-299&amp;pmid=12740691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="mixed-citation">
Maxwell PH, Wang Y: <span class="ref-title">Genetic studies of IgA nephropathy</span>. <span class="ref-journal">Nephron Exp Nephrol</span> <span class="ref-vol">102</span>: e76–e80, 2006 [<a href="/pubmed/16282702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephron+Exp+Nephrol&amp;title=Genetic+studies+of+IgA+nephropathy&amp;author=PH+Maxwell&amp;author=Y+Wang&amp;volume=102&amp;publication_year=2006&amp;pages=e76-e80&amp;pmid=16282702&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="mixed-citation">
Beerman I, Novak J, Wyatt RJ, Julian BA, Gharavi AG: <span class="ref-title">The genetics of IgA nephropathy</span>. <span class="ref-journal">Nat Clin Pract Nephrol</span> <span class="ref-vol">3</span>: 325–338, 2007 [<a href="/pubmed/17525715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Clin+Pract+Nephrol&amp;title=The+genetics+of+IgA+nephropathy&amp;author=I+Beerman&amp;author=J+Novak&amp;author=RJ+Wyatt&amp;author=BA+Julian&amp;author=AG+Gharavi&amp;volume=3&amp;publication_year=2007&amp;pages=325-338&amp;pmid=17525715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="mixed-citation">
International HapMap Consortium: <span class="ref-title">A haplotype map of the human genome</span>. <span class="ref-journal">Nature</span> <span class="ref-vol">437</span>: 1299–1320, 2005 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="mixed-citation">
Patterson N, Price AL, Reich D: <span class="ref-title">Population structure and eigenanalysis</span>. <span class="ref-journal">PLoS Genet</span> <span class="ref-vol">2</span>: e190, 2006 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="mixed-citation">
Leslie S, Donnelly P, McVean G: <span class="ref-title">A statistical method for predicting classical HLA alleles from SNP data</span>. <span class="ref-journal">Am J Human Genet</span> <span class="ref-vol">82</span>: 48–56, 2008 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Human+Genet&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S+Leslie&amp;author=P+Donnelly&amp;author=G+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="mixed-citation">
Li PK-T, Bunrs AP, So AKL, Pusey CD, Feehally J, Rees AJ: <span class="ref-title">The DQw7 allele at the HLA-DQB locus is associated with susceptibility to IgA nephropathy in Caucasians</span>. <span class="ref-journal">Kidney Int</span> <span class="ref-vol">39</span>: 961–965, 1991 [<a href="/pubmed/1676769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=The+DQw7+allele+at+the+HLA-DQB+locus+is+associated+with+susceptibility+to+IgA+nephropathy+in+Caucasians&amp;author=PK-T+Li&amp;author=AP+Bunrs&amp;author=AKL+So&amp;author=CD+Pusey&amp;author=J+Feehally&amp;volume=39&amp;publication_year=1991&amp;pages=961-965&amp;pmid=1676769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="mixed-citation">
Raguénès O, Mercier B, Clèdes J, Whebe B, Férec C: <span class="ref-title">HLA class II typing and idiopathic IgA nephropathy (IgAN): DQB1*0301, a possible marker of unfavorable outcome</span>. <span class="ref-journal">Tissue Antigens</span> <span class="ref-vol">45</span>: 246–249, 1995 [<a href="/pubmed/7638860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tissue+Antigens&amp;title=HLA+class+II+typing+and+idiopathic+IgA+nephropathy+(IgAN):+DQB1*0301,+a+possible+marker+of+unfavorable+outcome&amp;author=O+Raguénès&amp;author=B+Mercier&amp;author=J+Clèdes&amp;author=B+Whebe&amp;author=C+Férec&amp;volume=45&amp;publication_year=1995&amp;pages=246-249&amp;pmid=7638860&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="mixed-citation">
Moore RH, Hitman GA, Lucas EY, Richards NT, Venning MC, Papiha S, Goodship TH, Fidler A, Awad J, Festenstein H, Cunningham J, Marsh FP: <span class="ref-title">HLA DQ region polymorphisms associated with primary IgA nephropathy</span>. <span class="ref-journal">Kidney Int</span> <span class="ref-vol">37</span>: 991–995, 1990 [<a href="/pubmed/1968992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=HLA+DQ+region+polymorphisms+associated+with+primary+IgA+nephropathy&amp;author=RH+Moore&amp;author=GA+Hitman&amp;author=EY+Lucas&amp;author=NT+Richards&amp;author=MC+Venning&amp;volume=37&amp;publication_year=1990&amp;pages=991-995&amp;pmid=1968992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="mixed-citation">
Moore RH, Hitman GA, Medcraft J, Sinico RA, Mustonen J, Lucas EY, D'Amico G: <span class="ref-title">HLA-DP region gene polymorphism in primary IgA nephropathy: No association</span>. <span class="ref-journal">Nephrol Dial Transplant</span> <span class="ref-vol">7</span>: 200–204, 1992 [<a href="/pubmed/1314989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephrol+Dial+Transplant&amp;title=HLA-DP+region+gene+polymorphism+in+primary+IgA+nephropathy:+No+association&amp;author=RH+Moore&amp;author=GA+Hitman&amp;author=J+Medcraft&amp;author=RA+Sinico&amp;author=J+Mustonen&amp;volume=7&amp;publication_year=1992&amp;pages=200-204&amp;pmid=1314989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="mixed-citation">
Rambausek MH, Waldherr R, Ritz E: <span class="ref-title">Immunogenetic findings in glomerulonephritis</span>. <span class="ref-journal">Kidney Int</span> <span class="ref-vol">39</span>: S3–S8, 1993 [<a href="/pubmed/8468922" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=Immunogenetic+findings+in+glomerulonephritis&amp;author=MH+Rambausek&amp;author=R+Waldherr&amp;author=E+Ritz&amp;volume=39&amp;publication_year=1993&amp;pages=S3-S8&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="mixed-citation">
Luger AM, Komathireddy G, Walker RE, Pandey JP, Hoffman RW: <span class="ref-title">Molecular and serological analysis of HLA genes and immunoglobulin allotypes in IgA nephropathy</span>. <span class="ref-journal">Autoimmunity</span> <span class="ref-vol">19</span>: 1–5, 1994 [<a href="/pubmed/7749037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmunity&amp;title=Molecular+and+serological+analysis+of+HLA+genes+and+immunoglobulin+allotypes+in+IgA+nephropathy&amp;author=AM+Luger&amp;author=G+Komathireddy&amp;author=RE+Walker&amp;author=JP+Pandey&amp;author=RW+Hoffman&amp;volume=19&amp;publication_year=1994&amp;pages=1-5&amp;pmid=7749037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="mixed-citation">
Fennessy M, Hitman GA, Moore RH, Metcalfe K, Medcraft J, Sinico RA, Mustonen JT, D'Amico G: <span class="ref-title">HLA-DQ polymorphism in primary IgA disease in three European populations</span>. <span class="ref-journal">Kidney Int</span> <span class="ref-vol">49</span>: 477–480, 1996 [<a href="/pubmed/8821832" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Kidney+Int&amp;title=HLA-DQ+polymorphism+in+primary+IgA+disease+in+three+European+populations&amp;author=M+Fennessy&amp;author=GA+Hitman&amp;author=RH+Moore&amp;author=K+Metcalfe&amp;author=J+Medcraft&amp;volume=49&amp;publication_year=1996&amp;pages=477-480&amp;pmid=8821832&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="mixed-citation">
Candore G, Lio D, Romano GC, Caruso C: <span class="ref-title">Pathogenesis of autoimmune diseases associated with 8.1 ancestral haplotype: effect of multiple gene interactions</span>. <span class="ref-journal">Autoimmun Rev</span> <span class="ref-vol">1</span>: 29–35, 2002 [<a href="/pubmed/12849055" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmun+Rev&amp;title=Pathogenesis+of+autoimmune+diseases+associated+with+8.1+ancestral+haplotype:+effect+of+multiple+gene+interactions&amp;author=G+Candore&amp;author=D+Lio&amp;author=GC+Romano&amp;author=C+Caruso&amp;volume=1&amp;publication_year=2002&amp;pages=29-35&amp;pmid=12849055&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="mixed-citation">
Hiki Y, Kobayashi Y, Ookubo M, Obata F, Kashiwagi N: <span class="ref-title">Association of HLA-DQw4 with IgA nephropathy in the Japanese population</span>. <span class="ref-journal">Nephron</span> <span class="ref-vol">58</span>: 109–111, 1991 [<a href="/pubmed/1713306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephron&amp;title=Association+of+HLA-DQw4+with+IgA+nephropathy+in+the+Japanese+population&amp;author=Y+Hiki&amp;author=Y+Kobayashi&amp;author=M+Ookubo&amp;author=F+Obata&amp;author=N+Kashiwagi&amp;volume=58&amp;publication_year=1991&amp;pages=109-111&amp;pmid=1713306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="mixed-citation">
Abe J, Kohsaka T, Tanaka M, Kobayashi N: <span class="ref-title">Genetic study on HLA class II and class III region in the disease associated with IgA nephropathy</span>. <span class="ref-journal">Nephron</span> <span class="ref-vol">65</span>: 17–22, 1993 [<a href="/pubmed/8105396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nephron&amp;title=Genetic+study+on+HLA+class+II+and+class+III+region+in+the+disease+associated+with+IgA+nephropathy&amp;author=J+Abe&amp;author=T+Kohsaka&amp;author=M+Tanaka&amp;author=N+Kobayashi&amp;volume=65&amp;publication_year=1993&amp;pages=17-22&amp;pmid=8105396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="mixed-citation">
Phelps R, Rees A: <span class="ref-title">The HLA complex in renal disease</span>. In: <span class="ref-journal">HLA in Health and Disease</span>, 2nd Ed, edited by Lechler R, Warren A, editors.  Oxford, Academic Press, 2000 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=HLA+in+Health+and+Disease&amp;author=R+Phelps&amp;author=A+Rees&amp;publication_year=2000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="mixed-citation">
Akiyama F, Tanaka T, Yamada R, Ohnishi Y, Tsunoda T, Maeda S, Takei T, Obara W, Ito K, Honda K, Uchida K, Tsuchiya K, Nitta K, Yumura W, Nihei H, Ujiie T, Nagane Y, Miyano S, Suzuki Y, Fujioka T, Narita I, Gejyo F, Nakamura Y: <span class="ref-title">Single-nucleotide polymorphisms in the class II region of the major histocompatibility complex in Japanese patients with immunoglobulin A nephropathy</span>. <span class="ref-journal">J Hum Genet</span> <span class="ref-vol">47</span>: 532–538, 2002 [<a href="/pubmed/12376743" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=Single-nucleotide+polymorphisms+in+the+class+II+region+of+the+major+histocompatibility+complex+in+Japanese+patients+with+immunoglobulin+A+nephropathy&amp;author=F+Akiyama&amp;author=T+Tanaka&amp;author=R+Yamada&amp;author=Y+Ohnishi&amp;author=T+Tsunoda&amp;volume=47&amp;publication_year=2002&amp;pages=532-538&amp;pmid=12376743&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="mixed-citation">
Barrett JC, Cardon LR: <span class="ref-title">Evaluating coverage of genome-wide association studies</span>. <span class="ref-journal">Nat Genet</span> <span class="ref-vol">38</span>: 659–662, 2006 [<a href="/pubmed/16715099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Evaluating+coverage+of+genome-wide+association+studies&amp;author=JC+Barrett&amp;author=LR+Cardon&amp;volume=38&amp;publication_year=2006&amp;pages=659-662&amp;pmid=16715099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="mixed-citation">
Steemers FJ, Chang W, Lee G, Barker DL, Shen R, Gunderson KL: <span class="ref-title">Whole-genome genotyping with the single-base extension assay</span>. <span class="ref-journal">Nat Methods</span> <span class="ref-vol">3</span>: 31–33, 2006 [<a href="/pubmed/16369550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Whole-genome+genotyping+with+the+single-base+extension+assay&amp;author=FJ+Steemers&amp;author=W+Chang&amp;author=G+Lee&amp;author=DL+Barker&amp;author=R+Shen&amp;volume=3&amp;publication_year=2006&amp;pages=31-33&amp;pmid=16369550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="mixed-citation">
Teo YY, Inouye M, Small KS, Gwilliam R, Deloukas P, Kwiatkowski DP, Clark TG: <span class="ref-title">A genotype calling algorithm for the Illumina BeadArray platform</span>. <span class="ref-journal">Bioinformatics</span> <span class="ref-vol">23</span>: 2741–2746, 2007 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;author=M+Inouye&amp;author=KS+Small&amp;author=R+Gwilliam&amp;author=P+Deloukas&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="mixed-citation">
Clayton DA: <span class="ref-title">Generalization of the transmission/disequilibrium test for uncertain-haplotype transmission</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">65</span>: 1170–1177, 1999 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1288250/">PMC free article</a>]</span> [<a href="/pubmed/10486336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Generalization+of+the+transmission/disequilibrium+test+for+uncertain-haplotype+transmission&amp;author=DA+Clayton&amp;volume=65&amp;publication_year=1999&amp;pages=1170-1177&amp;pmid=10486336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="mixed-citation">
Kazeem GR, Farrall M: <span class="ref-title">Integrating case-control and TDT studies</span>. <span class="ref-journal">Ann Hum Genet</span> <span class="ref-vol">69</span>: 329–335, 2005 [<a href="/pubmed/15845037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Integrating+case-control+and+TDT+studies&amp;author=GR+Kazeem&amp;author=M+Farrall&amp;volume=69&amp;publication_year=2005&amp;pages=329-335&amp;pmid=15845037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="mixed-citation">
Cochrane WG: <span class="ref-title">Some methods for strengthening the common chi-squared tests</span>, <span class="ref-journal">Biometrics</span> <span class="ref-vol">10</span>: 417–451, 1954 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Some+methods+for+strengthening+the+common+chi-squared+tests&amp;author=WG+Cochrane&amp;volume=10&amp;publication_year=1954&amp;pages=417-451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="mixed-citation">
Williams RL: <span class="ref-title">A note on robust variance estimation for cluster-correlated data</span>. <span class="ref-journal">Biometrics</span> <span class="ref-vol">56</span>: 645–646, 2000 [<a href="/pubmed/10877330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=A+note+on+robust+variance+estimation+for+cluster-correlated+data&amp;author=RL+Williams&amp;volume=56&amp;publication_year=2000&amp;pages=645-646&amp;pmid=10877330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="mixed-citation">
Moffatt MF, Kabesch M, Liang L, Dixon AL, Strachan D, Heath S, Depner M, von Berg A, Bufe A, Rietschel E, Heinzmann A, Simma B, Frischer T, Willis-Owen SA, Wong KC, Illig T, Vogelberg C, Weiland SK, von Mutius E, Abecasis GR, Farrall M, Gut IG, Lathrop GM, Cookson WO: <span class="ref-title">Genetic variants regulating ORMDL3 expression contribute to the risk of childhood asthma</span>. <span class="ref-journal">Nature</span> <span class="ref-vol">448</span>: 470–473, 2007 [<a href="/pubmed/17611496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variants+regulating+ORMDL3+expression+contribute+to+the+risk+of+childhood+asthma&amp;author=MF+Moffatt&amp;author=M+Kabesch&amp;author=L+Liang&amp;author=AL+Dixon&amp;author=D+Strachan&amp;volume=448&amp;publication_year=2007&amp;pages=470-473&amp;pmid=17611496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="mixed-citation">
Cleves MA: <span class="ref-title">Hardy-Weinberg equilibrium tests and allele frequency estimation</span>. <span class="ref-journal">STATA Tech Bull</span> <span class="ref-vol">48</span>: 34–37, 1999 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=STATA+Tech+Bull&amp;title=Hardy-Weinberg+equilibrium+tests+and+allele+frequency+estimation&amp;author=MA+Cleves&amp;volume=48&amp;publication_year=1999&amp;pages=34-37&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="mixed-citation">
Bacanu SA, Devlin B, Roeder K: <span class="ref-title">The power of genomic control</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">66</span>: 1933–1944, 2000 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378064/">PMC free article</a>]</span> [<a href="/pubmed/10801388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=The+power+of+genomic+control&amp;author=SA+Bacanu&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=66&amp;publication_year=2000&amp;pages=1933-1944&amp;pmid=10801388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="mixed-citation">
Browning SR, Browning BL: <span class="ref-title">Rapid and accurate haplotype phasing and missing data inference for whole genome association studies using localized haplotype clustering</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">81</span>: 1084–1097, 2007 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265661/">PMC free article</a>]</span> [<a href="/pubmed/17924348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rapid+and+accurate+haplotype+phasing+and+missing+data+inference+for+whole+genome+association+studies+using+localized+haplotype+clustering&amp;author=SR+Browning&amp;author=BL+Browning&amp;volume=81&amp;publication_year=2007&amp;pages=1084-1097&amp;pmid=17924348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="mixed-citation">
Browning BL, Browning SR: <span class="ref-title">A unified approach to genotype imputation and haplotype phase inference for large data sets of trios and unrelated individuals</span>. <span class="ref-journal">Am J Hum Genet</span> <span class="ref-vol">84</span>: 210–223, 2009 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype+phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-223&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="mixed-citation">
Falk CT, Rubinstein P: <span class="ref-title">Haplotype relative risks: an easy reliable way to construct a proper control sample for risk calculations</span>. <span class="ref-journal">Ann Hum Genet</span> <span class="ref-vol">51</span>: 227–233, 1987 [<a href="/pubmed/3500674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Haplotype+relative+risks:+an+easy+reliable+way+to+construct+a+proper+control+sample+for+risk+calculations&amp;author=CT+Falk&amp;author=P+Rubinstein&amp;volume=51&amp;publication_year=1987&amp;pages=227-233&amp;pmid=3500674&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3013538&amp;issue-id=192995&amp;journal-id=612&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140333912825936title">Abstract</a></li><li><a href="#__sec1title">Results</a></li><li><a href="#__sec4title">Discussion</a></li><li><a href="#__sec5title">Concise Methods</a></li><li><a href="#__sec12title">Disclosures</a></li><li><a href="#__sec13title">Supplementary Material</a></li><li><a href="#idm140333917685344title">Acknowledgments</a></li><li><a href="#idm140333913312160title">Footnotes</a></li><li><a href="#idm140333917682912title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of the American Society of Nephrology : JASN</span> are provided here courtesy of <strong>American Society of Nephrology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3013538/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3013538/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3013538/?report=printable">Printer Friendly</a></li> | <li><a href="#" data-citationid="PMC3013538" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013538%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013538%2F&amp;text=HLA%20Has%20Strongest%20Association%20with%20IgA%20Nephropathy%20in%20Genome-Wide%20Analysis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013538%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20595679" data-aiid="3013538" data-aid="3013538" data-iid="192995" data-domainid="612" data-domain="jnephrol" data-accid="PMC3013538" data-md5="67e0dd50c78593f1a54f90054d01082d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:31:54-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D55F4DD50E5510000000004F70360&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3013538%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jnephrol&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3013538/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>