<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="cbt"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3280918/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>New variants at 10q26 and 15q21 are associated with aggressive prostate cancer in a genome-wide association study from a prostate biopsy screening cohort</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280918/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Cancer Biology &amp; Therapy"><meta name="citation_title" content="New variants at 10q26 and 15q21 are associated with aggressive prostate cancer in a genome-wide association study from a prostate biopsy screening cohort"><meta name="citation_authors" content="Robert K. Nam, William Zhang, Katherine Siminovitch, Adam Shlien, Michael W. Kattan, Laurence H. Klotz, John Trachtenberg, Yan Lu, Jinyi Zhang, Changhong Yu, Ants Toi, D. Andrew Loblaw, Vasundara Venkateswaran, Aleksandra Stanimirovic, Linda Sugar, David Malkin, Arun Seth, Steven A. Narod"><meta name="citation_date" content="1 December 2011"><meta name="citation_issue" content="11"><meta name="citation_volume" content="12"><meta name="citation_firstpage" content="997"><meta name="citation_doi" content="10.4161/cbt.12.11.18366"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3280918/?report=abstract"><meta name="citation_pmid" content="22130093"><meta name="DC.Title" content="New variants at 10q26 and 15q21 are associated with aggressive prostate cancer in a genome-wide association study from a prostate biopsy screening cohort"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Taylor &amp; Francis"><meta name="DC.Contributor" content="Robert K. Nam"><meta name="DC.Contributor" content="William Zhang"><meta name="DC.Contributor" content="Katherine Siminovitch"><meta name="DC.Contributor" content="Adam Shlien"><meta name="DC.Contributor" content="Michael W. Kattan"><meta name="DC.Contributor" content="Laurence H. Klotz"><meta name="DC.Contributor" content="John Trachtenberg"><meta name="DC.Contributor" content="Yan Lu"><meta name="DC.Contributor" content="Jinyi Zhang"><meta name="DC.Contributor" content="Changhong Yu"><meta name="DC.Contributor" content="Ants Toi"><meta name="DC.Contributor" content="D. Andrew Loblaw"><meta name="DC.Contributor" content="Vasundara Venkateswaran"><meta name="DC.Contributor" content="Aleksandra Stanimirovic"><meta name="DC.Contributor" content="Linda Sugar"><meta name="DC.Contributor" content="David Malkin"><meta name="DC.Contributor" content="Arun Seth"><meta name="DC.Contributor" content="Steven A. Narod"><meta name="DC.Date" content="2011 Dec 1"><meta name="DC.Identifier" content="10.4161/cbt.12.11.18366"><meta name="DC.Language" content="en"><meta property="og:title" content="New variants at 10q26 and 15q21 are associated with aggressive prostate cancer in a genome-wide association study from a prostate biopsy screening cohort"><meta property="og:type" content="article"><meta property="og:description" content="Purpose: To identify and examine polymorphisms of genes associated with aggressive and clinical significant forms of prostate cancer among a screening cohort.

&nbsp;
&nbsp;
Experimental Design: We conducted a genome-wide association study among ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280918/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-cbt.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3280918/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3280918/pdf/cbt-12-997.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8824D9DD53862100000000025801A9.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3280918%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1361/">Cancer Biol Ther</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/205681/">v.12(11); 2011 Dec 1</a></li><li class="accid">PMC3280918</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-cbt.gif" alt="Logo of cbt" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.tandfonline.com/loi/kcbt20" target="pmc_ext" ref="reftype=publisher&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Cancer Biol Ther</a></span>. 2011 Dec 1; 12(11): 997–1004. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.4161%2Fcbt.12.11.18366" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.4161/cbt.12.11.18366</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3280918</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22130093">22130093</a></div></div></div></div><h1 class="content-title">New variants at 10q26 and 15q21 are associated with aggressive prostate cancer in a genome-wide association study from a prostate biopsy screening cohort</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Nam%20RK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884675548160" co-class="co-affbox">Robert K. Nam</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>
1
,</sup><sup>
2
</sup> <a href="/pubmed/?term=Zhang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884675544448" co-class="co-affbox">William Zhang</a>,<sup>
1
</sup> <a href="/pubmed/?term=Siminovitch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884745784816" co-class="co-affbox">Katherine Siminovitch</a>,<sup>
3
</sup> <a href="/pubmed/?term=Shlien%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673898976" co-class="co-affbox">Adam Shlien</a>,<sup>
4
</sup> <a href="/pubmed/?term=Kattan%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673896368" co-class="co-affbox">Michael W. Kattan</a>,<sup>
5
</sup> <a href="/pubmed/?term=Klotz%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673893760" co-class="co-affbox">Laurence H. Klotz</a>,<sup>
1
</sup> <a href="/pubmed/?term=Trachtenberg%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673891152" co-class="co-affbox">John Trachtenberg</a>,<sup>
6
</sup> <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673888544" co-class="co-affbox">Yan Lu</a>,<sup>
3
</sup> <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884673885936" co-class="co-affbox">Jinyi Zhang</a>,<sup>
3
</sup> <a href="/pubmed/?term=Yu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677980320" co-class="co-affbox">Changhong Yu</a>,<sup>
5
</sup> <a href="/pubmed/?term=Toi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677977712" co-class="co-affbox">Ants Toi</a>,<sup>
7
</sup> <a href="/pubmed/?term=Loblaw%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677975104" co-class="co-affbox">D. Andrew Loblaw</a>,<sup>
8
</sup> <a href="/pubmed/?term=Venkateswaran%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677972496" co-class="co-affbox">Vasundara Venkateswaran</a>,<sup>
1
</sup> <a href="/pubmed/?term=Stanimirovic%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677969888" co-class="co-affbox">Aleksandra Stanimirovic</a>,<sup>
1
</sup> <a href="/pubmed/?term=Sugar%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884677967280" co-class="co-affbox">Linda Sugar</a>,<sup>
9
</sup> <a href="/pubmed/?term=Malkin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884670552240" co-class="co-affbox">David Malkin</a>,<sup>
4
</sup> <a href="/pubmed/?term=Seth%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884670549632" co-class="co-affbox">Arun Seth</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>
10
</sup> and  <a href="/pubmed/?term=Narod%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093" class="affpopup" co-rid="_co_idm139884670546784" co-class="co-affbox">Steven A. Narod</a><sup>
11
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139884675548160"><h3 class="no_margin">Robert K. Nam</h3><p><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </p><p><sup>(2) </sup>Department of Health Policy Management and Evaluation; University of Toronto; Toronto ON Canada</p><div>Find articles by <a href="/pubmed/?term=Nam%20RK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Robert K. Nam</a></div></div><div id="_co_idm139884675544448"><h3 class="no_margin">William Zhang</h3><p><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </p><div>Find articles by <a href="/pubmed/?term=Zhang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">William Zhang</a></div></div><div id="_co_idm139884745784816"><h3 class="no_margin">Katherine Siminovitch</h3><p><sup>(3) </sup>Department of Medicine, Immunology and Molecular Genetics; University Health Network; The Cleveland Clinic; Cleveland, OH USA</p><div>Find articles by <a href="/pubmed/?term=Siminovitch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Katherine Siminovitch</a></div></div><div id="_co_idm139884673898976"><h3 class="no_margin">Adam Shlien</h3><p><sup>(4) </sup>Division of Hematology/Oncology; Genetics and Genome Biology Program; The Hospital for Sick Children; Department of Pediatrics and Medical Biophysics; University of Toronto; Toronto ON Canada</p><div>Find articles by <a href="/pubmed/?term=Shlien%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Adam Shlien</a></div></div><div id="_co_idm139884673896368"><h3 class="no_margin">Michael W. Kattan</h3><p><sup>(5) </sup>Quantitative Health Sciences; The Cleveland Clinic; Cleveland, OH USA; 5 Division of Molecular and Cellular Biology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada</p><div>Find articles by <a href="/pubmed/?term=Kattan%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Michael W. Kattan</a></div></div><div id="_co_idm139884673893760"><h3 class="no_margin">Laurence H. Klotz</h3><p><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </p><div>Find articles by <a href="/pubmed/?term=Klotz%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Laurence H. Klotz</a></div></div><div id="_co_idm139884673891152"><h3 class="no_margin">John Trachtenberg</h3><p><sup>(6) </sup>Division of Urology; University Health Network; The Cleveland Clinic; Cleveland, OH USA</p><div>Find articles by <a href="/pubmed/?term=Trachtenberg%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">John Trachtenberg</a></div></div><div id="_co_idm139884673888544"><h3 class="no_margin">Yan Lu</h3><p><sup>(3) </sup>Department of Medicine, Immunology and Molecular Genetics; University Health Network; The Cleveland Clinic; Cleveland, OH USA</p><div>Find articles by <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Yan Lu</a></div></div><div id="_co_idm139884673885936"><h3 class="no_margin">Jinyi Zhang</h3><p><sup>(3) </sup>Department of Medicine, Immunology and Molecular Genetics; University Health Network; The Cleveland Clinic; Cleveland, OH USA</p><div>Find articles by <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Jinyi Zhang</a></div></div><div id="_co_idm139884677980320"><h3 class="no_margin">Changhong Yu</h3><p><sup>(5) </sup>Quantitative Health Sciences; The Cleveland Clinic; Cleveland, OH USA; 5 Division of Molecular and Cellular Biology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada</p><div>Find articles by <a href="/pubmed/?term=Yu%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Changhong Yu</a></div></div><div id="_co_idm139884677977712"><h3 class="no_margin">Ants Toi</h3><p><sup>(7) </sup>Department of Medical Imaging; University Health Network; The Cleveland Clinic; Cleveland, OH USA</p><div>Find articles by <a href="/pubmed/?term=Toi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Ants Toi</a></div></div><div id="_co_idm139884677975104"><h3 class="no_margin">D. Andrew Loblaw</h3><p><sup>(8) </sup>Department of Radiation Oncology; Sunnybrook Health Sciences Center; University of Toronto; Toronto, ON Canada</p><div>Find articles by <a href="/pubmed/?term=Loblaw%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">D. Andrew Loblaw</a></div></div><div id="_co_idm139884677972496"><h3 class="no_margin">Vasundara Venkateswaran</h3><p><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </p><div>Find articles by <a href="/pubmed/?term=Venkateswaran%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Vasundara Venkateswaran</a></div></div><div id="_co_idm139884677969888"><h3 class="no_margin">Aleksandra Stanimirovic</h3><p><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </p><div>Find articles by <a href="/pubmed/?term=Stanimirovic%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Aleksandra Stanimirovic</a></div></div><div id="_co_idm139884677967280"><h3 class="no_margin">Linda Sugar</h3><p><sup>(9) </sup>Department of Pathology; Sunnybrook Health Sciences Center; University of Toronto; Toronto, ON Canada</p><div>Find articles by <a href="/pubmed/?term=Sugar%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Linda Sugar</a></div></div><div id="_co_idm139884670552240"><h3 class="no_margin">David Malkin</h3><p><sup>(4) </sup>Division of Hematology/Oncology; Genetics and Genome Biology Program; The Hospital for Sick Children; Department of Pediatrics and Medical Biophysics; University of Toronto; Toronto ON Canada</p><div>Find articles by <a href="/pubmed/?term=Malkin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">David Malkin</a></div></div><div id="_co_idm139884670549632"><h3 class="no_margin">Arun Seth</h3><p><sup>(10) </sup>Division of Molecular and Cellular Biology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada</p><div>Find articles by <a href="/pubmed/?term=Seth%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Arun Seth</a></div></div><div id="_co_idm139884670546784"><h3 class="no_margin">Steven A. Narod</h3><p><sup>(11) </sup>Department of Public Health Sciences; University of Toronto; Toronto ON Canada</p><div>Find articles by <a href="/pubmed/?term=Narod%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22130093">Steven A. Narod</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139884670514624_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139884670514624_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139884670514624_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>(1) </sup>Division of Urology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada </div><div class="fm-affl" id="A2" lang="en"><sup>(2) </sup>Department of Health Policy Management and Evaluation; University of Toronto; Toronto ON Canada</div><div class="fm-affl" id="A3" lang="en"><sup>(3) </sup>Department of Medicine, Immunology and Molecular Genetics; University Health Network; The Cleveland Clinic; Cleveland, OH USA</div><div class="fm-affl" id="A4" lang="en"><sup>(4) </sup>Division of Hematology/Oncology; Genetics and Genome Biology Program; The Hospital for Sick Children; Department of Pediatrics and Medical Biophysics; University of Toronto; Toronto ON Canada</div><div class="fm-affl" id="A5" lang="en"><sup>(5) </sup>Quantitative Health Sciences; The Cleveland Clinic; Cleveland, OH USA; 5 Division of Molecular and Cellular Biology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada</div><div class="fm-affl" id="A6" lang="en"><sup>(6) </sup>Division of Urology; University Health Network; The Cleveland Clinic; Cleveland, OH USA</div><div class="fm-affl" id="A7" lang="en"><sup>(7) </sup>Department of Medical Imaging; University Health Network; The Cleveland Clinic; Cleveland, OH USA</div><div class="fm-affl" id="A8" lang="en"><sup>(8) </sup>Department of Radiation Oncology; Sunnybrook Health Sciences Center; University of Toronto; Toronto, ON Canada</div><div class="fm-affl" id="A9" lang="en"><sup>(9) </sup>Department of Pathology; Sunnybrook Health Sciences Center; University of Toronto; Toronto, ON Canada</div><div class="fm-affl" id="A10" lang="en"><sup>(10) </sup>Division of Molecular and Cellular Biology; Sunnybrook Research Institute; University of Toronto; Toronto, ON Canada</div><div class="fm-affl" id="A11" lang="en"><sup>(11) </sup>Department of Public Health Sciences; University of Toronto; Toronto ON Canada</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm139884670538096">Correspondence to: Arun Seth; Email: <a href="mailto:dev@null" data-email="ac.otnorotu@htes.nura" class="oemail">ac.otnorotu@htes.nura</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139884670514624_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2011 Landes Bioscience</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3280918/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Note</div><p>Supplementary materials can be found at: <a href="http://www.landesbioscience.com/journals/cbt/article/18366/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.landesbioscience.com/journals/cbt/article/18366/</a>.</p></div></dd></dl></div><div id="idm139884712439584" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712439584title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first">Purpose: To identify and examine polymorphisms of genes associated with aggressive and clinical significant forms of prostate cancer among a screening cohort.
</p><p id="__p2">
&nbsp;</p><p id="__p3">
&nbsp;</p><p id="__p4">
Experimental Design: We conducted a genome-wide association study among patients with aggressive forms of prostate cancer and biopsy-proven normal controls ascertained from a prostate cancer screening program. We then examined significant associations of specific polymorphisms among a prostate cancer screened cohort to examine their predictive ability in detecting prostate cancer.</p><p id="__p5">
Results: We found significant associations between aggressive prostate cancer and five single nucleotide polymorphisms (SNPs) in the 10q26 (rs10788165, rs10749408, and rs10788165, p value for association 1.3 × 10<sup>−10 </sup> to 3.2 × 10<sup>−11</sup>) and 15q21 (rs4775302 and rs1994198, p values for association 3.1 × 10<sup>−8 </sup> to 8.2 × 10<sup>−9</sup>) regions. Results of a replication study done in 3439 patients undergoing a prostate biopsy, revealed certain combinations of these SNPs to be significantly associated not only with prostate cancer but with aggressive forms of prostate cancer using an established classification criterion for prostate cancer progression (odds ratios for intermediate to high-risk disease 1.8–3.0, p value 0.003–0.001). These SNP combinations were also important clinical predictors for prostate cancer detection based on nomogram analysis that assesses prostate cancer risk.</p><p id="__p6" class="p p-last">
Conclusions: Five SNPs were found to be associated with aggressive forms of prostate cancer. We demonstrated potential clinical applications of these associations.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p7" class="p p-first">Several genome-wide association studies (GWAS) have identified a number of genomic variants which are associated with an increased risk of prostate cancer, particularly from the 8q24 region.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198906">1</a></sup><sup>-</sup><sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198890">3</a></sup> Although these associations are statistically significant, it remains unclear to what extent these high-risk genotypes are associated with aggressive forms of prostate cancer.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198905">4</a></sup><sup>,</sup><sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267819053">5</a></sup></p><p id="__p8">A primary goal of evaluating biomarkers for the early detection of prostate cancer is to distinguish patients who will eventually develop metastases from those with more indolent forms of cancer. Recently, a large, multi-centered study, including subjects from the Physicians’ Health Study, failed to show any associations with aggressive and lethal forms of prostate cancer with these and other SNPs found from past GWAS.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198888">4</a></sup> Also, a recent study examining the most significant SNPs found by past GWAS study found no associations to prostate cancer outcomes including measures of aggressiveness and cancer-specific mortality.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267819052">5</a></sup> Thus, although new SNP associations have been numerous, the clinical benefit has been limited since in part that these SNPs cannot identify patients with aggressive forms of prostate cancer.</p><p id="__p9">In a typical GWAS, the cases and controls are not derived from the same patient sample; however, this is not the case when screened subjects are studied. In a screening study, controls may be selected from men who screen negative for the cancer. Recent data from the Prostate Cancer Prevention Trial<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198895">6</a></sup> has established that men who are judged to be at low risk for prostate cancer (i.e., normal patients), but who undergo a prostate biopsy, have a prevalence rate of 25% for prostate cancer. One-quarter of these are high-grade, aggressive cancers.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198887">7</a></sup> Thus, the misclassification of cases as controls may diminish the potential of discovering SNPs that help to identify men with aggressive, high grade prostate cancer. By using men with negative biopsies as controls, the potential for misclassification is minimized.</p><p id="__p10" class="p p-last">To identify new SNP variants for aggressive prostate cancer, we conducted a GWAS among men who had a prostate biopsy, using a two-stage approach. In the first stage, 316 cases and 229 controls were genotyped using the Affymetrix 500K SNP array (443,816 SNPs). Cases were patients with aggressive forms of prostate cancer using the established D’Amico classification criteria,<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198907">8</a></sup> and controls were biopsy proven normal patients. In the second stage, we genotyped positive SNPs found from stage 1 among 3439 patients who underwent prostate biopsy for prostate cancer screening. We investigated their clinical significance by examining their association with D’Amico criteria outcomes and by nomogram analysis in predicting prostate risk.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Results</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Stage 1: identification of novel SNPs from GWAS</h3><p id="__p11" class="p p-first">We chose three criteria to select positive SNPs from the 316 cases and 229 controls that underwent GWAS analysis and chose 20 SNPs from them (<a href="/pmc/articles/PMC3280918/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>): (1) a Bonferroni corrected p value of less than 0.01, and the SNP was from a region which has previously been shown to harbor a locus associated with prostate cancer (n = 5); (2) two or more SNPs were in linkage disequilibrium and the Bonferroni corrected p value for each was less than 0.01 (n = 9); or (3) the Bonferroni corrected p value was 10<sup>−5</sup> or less (n = 6).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>List of 20 SNPs found by genome wide association study (GWAS) using the 500K Affymetrix single nucleotide polymorphism (SNP) chip to be associated with aggressive prostate cancer.</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884712274896" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">SNP<hr></th><th rowspan="1" colspan="1" valign="top" align="center">p value/Bonferroni-corrected p value<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Chromosomal region<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Minor allele frequency<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Odds ratio (95% C.I.)<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Associated gene (if applicable)<hr></th></tr><tr><th colspan="6" rowspan="1" valign="top" align="center">1st Criteria: Bonf. p value &lt; 0.01 with region close to known prostate cancer gene (n = 5)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs12699509<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.4 × 10<sup>−8</sup>/0.006<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7p21.2<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.29<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.7 (1.9.–3.8)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">ETV1<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3114316<hr></td><td rowspan="1" colspan="1" valign="top" align="center">9.2 × 10<sup>−9</sup>/0.004<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7q11.23<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.12<hr></td><td rowspan="1" colspan="1" valign="top" align="center">3.4 (2.2–5.2)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">KIAA1505<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs3852402<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 × 10<sup>−8</sup>/0.01<hr></td><td rowspan="1" colspan="1" valign="top" align="center">9q23.32<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.38<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.2 (1.7–2.9)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">FANCC<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2226016<hr></td><td rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−8</sup>/0.01<hr></td><td rowspan="1" colspan="1" valign="top" align="center">11p14.1<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.34<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.3 (1.7–3.2)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">FSHB<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4281668</td><td rowspan="1" colspan="1" valign="top" align="center">9.9 × 10<sup>−9</sup>/0.004</td><td rowspan="1" colspan="1" valign="top" align="center">15q25.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">2.9 (2.0–4.1)</td><td rowspan="1" colspan="1" valign="top" align="center">AKAP13</td></tr></tbody></table></div><div id="largeobj_idm139884712274896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/table/T1/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884677997760" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="top" align="center">2nd Criteria: Two or more SNPs in LD and Bonf. p &lt; 0.01 (n = 9)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2053140<hr></td><td rowspan="1" colspan="1" valign="top" align="center">5.7 × 10<sup>−9</sup>/0.002<hr></td><td rowspan="1" colspan="1" valign="top" align="center">8q13.2<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.47<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 (1.8–3.7)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">DEPDC2<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4131931<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.8 × 10<sup>−8</sup>/0.01<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">(8 kb apart)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs7089868<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.8 × 10<sup>−8</sup>/0.007<hr></td><td rowspan="1" colspan="1" valign="top" align="center">10q23.1<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.47<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.8 (2.0–4.1)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">KIAA1128<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11596082<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.7 × 10<sup>−8</sup>/0.007<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">(7 kb apart)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2351337<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.8 × 10<sup>−8</sup>/0.007<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">(40 kb apart)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11199874<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7.2 × 10<sup>−11</sup>/3.0x10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">10q26.12<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.29<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 (2.1–4.1)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">No description <hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10788165<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−10</sup>/5.5x10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;<hr></td><td rowspan="1" colspan="1" valign="top" align="center">(12 kb apart)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1994198<hr></td><td rowspan="1" colspan="1" valign="top" align="center">8.2 × 10<sup>−9</sup>/0.003<hr></td><td rowspan="1" colspan="1" valign="top" align="center">15q21.1<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.43<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 (1.8–3.7)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">No description<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4775302</td><td rowspan="1" colspan="1" valign="top" align="center">3.1 × 10<sup>−8</sup>/0.01</td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td rowspan="1" colspan="1" valign="top" align="center">&nbsp;</td><td rowspan="1" colspan="1" valign="top" align="center">(13 kb apart)</td></tr></tbody></table></div><div id="largeobj_idm139884677997760" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/table/T1/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884675321888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="6" rowspan="1" valign="top" align="center">3rd Criteria: Bonf. corrected p value &lt; 10<sup>−5</sup> (n = 6)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs17018760<hr></td><td rowspan="1" colspan="1" valign="top" align="center">4.0 × 10<sup>−12</sup>/1.7 × 10<sup>−6</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">2p12<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.11<hr></td><td rowspan="1" colspan="1" valign="top" align="center">4.4 (2.8–6.9)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">CTNNA2<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs410259<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.2 × 10<sup>−11</sup>/1.0 × 10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">2p22.1<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.12<hr></td><td rowspan="1" colspan="1" valign="top" align="center">4.0 (2.6–6.2)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">SLC8A<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10749408<hr></td><td rowspan="1" colspan="1" valign="top" align="center">3.2 × 10<sup>−11</sup>/1.0 × 10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">10q26.12<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.34<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.5 (1.9–3.3)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">No description<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2429763<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>−11</sup>/1.0 × 10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">11p14.3<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.18<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 (2.1–4.0)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">No description<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs595018<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.0 × 10<sup>−11</sup>/1.0 × 10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">11q12.2<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.19<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.9 (2.1–4.0)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">CCDC86<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs2347306</td><td rowspan="1" colspan="1" valign="top" align="center">3.0 × 10<sup>−11</sup>/1.0 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">12q24.32</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">5.5 (3.2–9.7)</td><td rowspan="1" colspan="1" valign="top" align="center">No description</td></tr></tbody></table></div><div id="largeobj_idm139884675321888" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1n1"><p id="__p12" class="p p-first-last">C.I., confidence interval.</p></div></div></div></div><div id="__sec4" class="sec sec-last"><h3 id="__sec4title">Stage 2: replication of 20 SNPs from prostate biopsy cohort</h3><p id="__p13" class="p p-first">The 20 SNPs were tested in an independent sample of 3439 men of various ethnicities. All 3439 men underwent a prostate biopsy; 1663 (48.4%) were diagnosed with prostate cancer, and 1776 (51.6%) did not have any evidence of cancer from biopsy (<a href="/pmc/articles/PMC3280918/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Among the 1663 men with cancer, 413 (24.8%) had a low risk cancer (based on the D’Amico classification of prostate cancer aggressiveness), 944 (56.8%) had an intermediate risk cancer, and 306 (18.4%) had a high risk prostate cancer.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparison between cases and controls of risk factors, tumor markers for prostate cancer and the genotype frequencies of selected SNPs</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884745873536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Factor<br>total n = 3439</th><th rowspan="1" colspan="1" valign="top" align="center">Cancer<br>n = 1663 (48.4%)</th><th rowspan="1" colspan="1" valign="top" align="center">No cancer<br>n = 1776 (51.6%)</th><th rowspan="1" colspan="1" valign="top" align="center">p value*</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">Age Group (years)<br>≤ 50<br>51–60<br>61–70<br>&gt; 70<hr></td><td rowspan="1" colspan="1" valign="top" align="center">38 (2.3%)<br>397 (23.9%)<br>718 (43.2%)<br>510 (30.7%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">96 (5.4%)<br>554 (31.2%)<br>776 (43.7%)<br>350 (19.7%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.3 × 10<sup>−21</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Family History of PC<br>Absent<br>Present<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1376 (82.7%)<br>287 (17.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1545 (87.0%)<br>231 (13.0%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.0005<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">Ethnicity<br>Asian<br>Caucasian<br>Black<br>Other<hr></td><td rowspan="1" colspan="1" valign="top" align="center">55 (3.3%)<br>1382 (83.1%)<br>184 (11.1%)<br>42 (2.5%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">134 (7.6%)<br>1428 (80.4%)<br>127 (7.1%)<br>87 (4.9%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7.4 × 10<sup>−17</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">LUTS<br>≤ 7<br>&gt; 7<hr></td><td rowspan="1" colspan="1" valign="top" align="center">926 (55.7%)<br>737 (44.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">864 (48.7%)<br>912 (51.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">5.5 × 10<sup>−6</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">DRE<br>No Nodule<br>Nodule<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1206 (72.5%)<br>457 (27.5%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1500 (84.5%)<br>276 (15.5%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7.3 × 10<sup>−18</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">PSA (ng/mL)<br>≤ 4.0<br>4.1–10.0<br>10.1–20.0<br>&gt; 20.0</td><td rowspan="1" colspan="1" valign="top" align="center">135 (8.1%)<br>980 (58.9%)<br>387 (23.3%)<br>161 (9.7%)</td><td rowspan="1" colspan="1" valign="top" align="center">353 (19.9%)<br>1011 (56.9%)<br>345 (19.4%)<br>67 (3.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">2.1 × 10<sup>−34</sup></td></tr></tbody></table></div><div id="largeobj_idm139884745873536" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3280918/table/T2/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884711215136" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="4" rowspan="1" valign="top" align="center">SNP genotyping frequency</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11199874, 10q26<br>GG<br>AG<br>AA<hr></td><td rowspan="1" colspan="1" valign="top" align="center">962 (57.9%)<br>567 (35.9%)<br>104 (6.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">869 (48.9%)<br>740 (41.7%)<br>167 (9.4%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 × 10<sup>−10</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10749408, 10q26<br>TT<br>CT<br>CC<hr></td><td rowspan="1" colspan="1" valign="top" align="center">813 (48.9%)<br>697 (41.9%)<br>153 (9.2%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">775 (43.6%)<br>803 (45.2%)<br>198 (11.2%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">6.9 × 10<sup>−6</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10788165, 10q26<br>TT<br>GT<br>GG<hr></td><td rowspan="1" colspan="1" valign="top" align="center">740 (44.5%)<br>734 (44.1%)<br>189 (11.4%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">673 (47.6%)<br>842 (47.4%)<br>261 (14.7%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.2 × 10<sup>−7</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4775302, 15q21<br>AA<br>AG<br>GG<hr></td><td rowspan="1" colspan="1" valign="top" align="center">496 (29.8%)<br>828 (49.8%)<br>339 (20.4%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">467 (26.3%)<br>833 (46.9%)<br>476 (26.8%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">4.1 × 10<sup>−8</sup><hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1994198, 15q21<br>TT<br>CT<br>CC</td><td rowspan="1" colspan="1" valign="top" align="center">492 (29.6%)<br>802 (48.2%)<br>369 (22.2%)</td><td rowspan="1" colspan="1" valign="top" align="center">498 (28.0%)<br>783 (44.1%)<br>495 (27.9%)</td><td rowspan="1" colspan="1" valign="top" align="center">5.8 × 10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm139884711215136" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3280918/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t2n1"><sup>*</sup>p value calculation based on Fisher’s exact test.</div><div id="t2n2"><sup>**</sup>LUTS, lower urinary tract symptoms.</div></div></div><p id="__p16">In the replication set, nine of the 20 SNPs were significantly associated with prostate cancer. The strongest associations were found from five SNPs: three SNPs at region 10q26 (p value 6 × 10<sup>−7</sup> to 3 × 10<sup>−10</sup>) and two SNPs at 15q21 (p value 7 × 10<sup>−6</sup> to 1 × 10<sup>−7</sup>) (<a href="/pmc/articles/PMC3280918/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The remaining four SNPs were from other regions, at 10q23 (rs7089868, rs11596082 and rs2351337) and at 8q13 (rs2053140), but the p values of association ranged from 0.05–0.02. Because of the large difference in the level of significance from the first five SNPs, the remaining four SNPs were excluded from further study.</p><p id="__p17">The associations between the five SNPs and prostate cancer were only present among white subjects. No associations were found for men of Asian or African ancestries, although the sample sizes for these ethnic groups were smaller (Table S1). No other significant association was found with the other 15 SNPs. Of the three SNPs from region 10q26, two SNPs (rs11199874 and rs10788165) were in strong linkage disequilibrium (r<sup>2</sup> = 0.97 in controls). The third SNP, rs10749408, was not in linkage disequilibrium (LD) with either of the other two. The two SNPs from region 15q21, rs4775302 and rs1994198, were in strong linkage disequilibrim (r<sup>2</sup> = 0.99 in controls).</p><p id="__p18">We asked whether these SNPs were associated with prostate cancer, after adjusting for established risk factors and other variables [age, family history of prostate cancer, ethnicity, urinary symptoms, prostate specific antigen (PSA) level and digital rectal examination]. The adjusted odds ratios, based on the risk genotype classes for the five SNPs, ranged from 1.26–1.42 (<a href="/pmc/articles/PMC3280918/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). To determine whether these variants were also associated with aggressive prostate cancer, we used the D’Amico criteria to divide the cases into low, intermediate and high-risk categories for prostate cancer progression. This criteria combines Gleason Score grade, clinical stage and PSA level at diagnosis into risk group categories and is a well-established method of predicting prostate cancer mortality.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198904">8</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Crude and adjusted odds ratio based on variant genotypes. Analysis for 10q26 was based on a recessive model and for 15q21 on a dominant model. Multivariate model variables included age at biopsy, ethnicity, family history of prostate cancer, the presence of urinary voiding symptoms, prostate specific antigen (PSA) level and digital rectal exam (DRE) status.</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884672701184" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">SNP Information<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Alternative Alleles<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Assoc. Allele<hr></th><th colspan="2" rowspan="1" valign="top" align="center">Frequency<hr></th><th colspan="2" rowspan="1" valign="top" align="center">Genotype<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Odds Ratio (95% CI)<hr></th><th rowspan="1" colspan="1" valign="top" align="center">p value<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Adjusted Odds Ratio (95% C.I.)<hr></th><th rowspan="1" colspan="1" valign="top" align="center">p value<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">Cases</th><th rowspan="1" colspan="1" valign="top" align="center">Controls</th><th rowspan="1" colspan="1" valign="top" align="center">Reference</th><th rowspan="1" colspan="1" valign="top" align="center">Associated</th><th rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">rs11199874 10q26<hr></td><td rowspan="1" colspan="1" valign="top" align="center">A/G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.75<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.70<hr></td><td rowspan="1" colspan="1" valign="top" align="center">A/A, A/G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">G/G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.42 (1.2−1.6)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">3.1 × 10<sup>−8</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.27 (1.1−1.5)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.001<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10749408 10q26<hr></td><td rowspan="1" colspan="1" valign="top" align="center">C/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.70<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.66<hr></td><td rowspan="1" colspan="1" valign="top" align="center">C/C, C/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">T/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.26 (1.1−1.4)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">6.0 × 10<sup>−5</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.0−1.4)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.02<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs10788165 10q26<hr></td><td rowspan="1" colspan="1" valign="top" align="center">G/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.67<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.62<hr></td><td rowspan="1" colspan="1" valign="top" align="center">G/G, G/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">T/T<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.34 (1.2−1.5)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.6 × 10<sup>−6</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (1.1−1.5)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.0005<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs4775302 15q21<hr></td><td rowspan="1" colspan="1" valign="top" align="center">A/G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">A<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.59<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.50<hr></td><td rowspan="1" colspan="1" valign="top" align="center">G/G<hr></td><td rowspan="1" colspan="1" valign="top" align="center">A/G, A/A<hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.2−1.6)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">2.4 × 10<sup>−6</sup><hr></td><td rowspan="1" colspan="1" valign="top" align="center">1.31 (1.1−1.6)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">0.001<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">rs1994198 15q21</td><td rowspan="1" colspan="1" valign="top" align="center">C/T</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">C/C</td><td rowspan="1" colspan="1" valign="top" align="center">C/T, T/T</td><td rowspan="1" colspan="1" valign="top" align="center">1.34 (1.1 – 1.6)</td><td rowspan="1" colspan="1" valign="top" align="center">3.4 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.33 (1.1 – 1.6)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0009</td></tr></tbody></table></div><div id="largeobj_idm139884672701184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3280918/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t3n1"><p id="__p19" class="p p-first-last">C.I., confidence interval.</p></div></div></div><p id="__p20">We chose one SNP to represent each of the two haplotype blocks (the one with the highest odds ratio for prostate cancer) resulting in three SNPs (rs11199874, rs10749408, and rs4775302). We combined the risk alleles from the three SNPs and compared the frequencies of patients with low, intermediate, high risk prostate cancer and with no cancer, by the number of risk alleles (<a href="/pmc/articles/PMC3280918/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). Within each risk category, the odds ratios for cancer increased by the number of variant alleles (Table S2). In particular, for patients who were diagnosed with high risk prostate cancer, those with three variant alleles had a 3-fold increase in risk (odds ratio being 3.0, 95% CI: 1.5–5.8) for having high risk prostate cancer, compared with patients with no variant allele. The odds ratios for high risk cancer for patients with two variant alleles was 1.8 (95% CI: 0.9–3.4) and for one variant allele was 1.2 (95% CI: 0.6–2.4) with a significant increase in trend by the number of variant alleles (p &lt; 0.0001) (Table S2). There was a significant positive trend in the odds ratios for having low, intermediate and high risk prostate cancer for patients with the three variant risk alleles (p = 0.0002). No significant trends were observed for patients with one or two variant alleles across disease risk categories. Overall, there was a global positive trend association by risk group and number of variant risk alleles (p = 0.048).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Distribution of patients by aggressiveness of prostate cancer using low, intermediate and high risk categories for prostate cancer (D’Amico Classification). Variant alleles based on a combination of a 3 SNP model (rs11199874, rs10749408 and rs4775302)</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884676523232" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Number of Variant Alleles (Frequency Distribution)<hr></th><th rowspan="1" colspan="1" valign="top" align="center">Number of patients with NO CANCER (n = 1776)<hr></th><th colspan="3" rowspan="1" valign="top" align="center">Number of Patients with CANCER* (n = 1663)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">LOW RISK** (n = 413) Gleason Score 6, PSA &lt; 10, AND Stage T1c</th><th rowspan="1" colspan="1" valign="top" align="center">INTERMEDIATE RISK** (n = 944) Gleason Score 7, PSA 10–20, OR Stage T2a/b</th><th rowspan="1" colspan="1" valign="top" align="center">HIGH RISK** (n = 306) Gleason Score 8–10, PSA &gt; 20, OR Stage T2c</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">0 (7.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">154 (8.7%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">25 (6.1%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">55 (5.8%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">18 (5.9%)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1 (34.3%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">669 (37.7%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">138 (33.4%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">293 (31.0%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">78 (25.5%)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2 (33.8%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">584 (32.9%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">148 (36.1%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">330 (34.9%)<hr></td><td rowspan="1" colspan="1" valign="top" align="center">100 (32.7%)<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3 (24.6%)</td><td rowspan="1" colspan="1" valign="top" align="center">369 (20.8%)</td><td rowspan="1" colspan="1" valign="top" align="center">101 (24.5%)</td><td rowspan="1" colspan="1" valign="top" align="center">267 (28.3%)</td><td rowspan="1" colspan="1" valign="top" align="center">110 (36.0%)</td></tr></tbody></table></div><div id="largeobj_idm139884676523232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t4n1"><sup>*</sup>Comparison between patients with and without cancer by the number of variant alleles (2 × 4 table): chi-square = 44.6, p &lt; 0.0001.</div><div id="t4n2"><sup>**</sup>Comparison between patients without cancer and cancer risk groups by the number of variant alleles (4 × 4 table): chi-square = 58.1, p &lt; 0.0001. Comparison between risk groups (cases only, 3 × 4 table): chi-square = 12.6, p = 0.05.</div></div></div><p id="__p23" class="p">To determine whether or not these SNPs could be used clinically in diagnosing prostate cancer, we examined the effect of SNPs within a multivariate logistic regression model in predicting prostate cancer using nomograms, and also assessed the clinical validity of these SNPs using sensitivity, specificity, positive predictive value (NPV) and negative predictive (NPV) analysis, as proposed by Kraft et al.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267819054">14</a></sup> To construct the nomogram, all 20 SNPs were tested within the multivariate model. Based on the concordance indices of bias-corrected probabilities for any and aggressive prostate cancer, for each of these SNPs, the performance of each of the 20 SNPs and the three variant risk alleles were similar. Among white subjects, the nomogram with the three risk alleles was a significant predictor (p = 0.0002) of any cancer and high grade (Gleason Score 7 or more) cancer (<a href="/pmc/articles/PMC3280918/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig.&nbsp;1</span></a>). When examining the importance of each predictor from area under the curve (AUC) analysis, the three SNP model was the fourth (out of six) most important predictor for cancer (incremental drop of AUC being 0.004, <a href="/pmc/articles/PMC3280918/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig.&nbsp;1</span></a>). It was a more important predictor than family history of prostate cancer and urinary symptoms. When considering the clinical performance of the three SNP model alone, the sensitivity was 94.4% for any cancer for patients with one or more variant alleles and the specificity was 78.2% for any cancer for patients with three variant alleles (<a href="/pmc/articles/PMC3280918/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). The positive and negative predictive values improved with the number of variant alleles, but the absolute levels were low, compared with the baseline prevalence. Also, the ROC curve for the 3 SNP model was better at diagnosing aggressive cancer than for any cancer (<a href="/pmc/articles/PMC3280918/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig.&nbsp;2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3280918/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884673806704" class="figure"><a href="/pmc/articles/PMC3280918/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3280918_cbt-12-997-g1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is cbt-12-997-g1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3280918/bin/cbt-12-997-g1.jpg"></a></div><div id="largeobj_idm139884673806704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3280918/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p24">Nomogram analysis based on 2,810 white subjects: 1,428 with no evidence of cancer, 637 with low grade PCa (GS &lt; = 6) and 745 with high grade PCa (GS 7–10). The nomogram is used by first locating a patient’s position for each variable on its horizontal scale and then a point value is assigned according to the points scale (top axis) and summed for all variables. Total points correspond to a probability value for having prostate cancer or aggressive prostate cancer. PSA, prostate specific antigen; FH, family history of prostate cancer; DRE, digital rectal examination; AUA Score, American Urological Prostate Urinary Symptom Score; and MOD3, 3 SNP risk allele model. For calibration plots, see Figure&nbsp;S2.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table&nbsp;5.</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Comparisons of sensitivity, specificity, positive predictive value, and negative predictive value based on the combination of the risk alleles from the three SNP model among white subjects only</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884673803792" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Predictor variable<hr></th><th colspan="3" rowspan="1" valign="top" align="center">Sensitivity (%)<hr></th><th colspan="3" rowspan="1" valign="top" align="center">Specificity (%)<hr></th><th colspan="3" rowspan="1" valign="top" align="center">Positive predictive value (%)<hr></th><th colspan="3" rowspan="1" valign="top" align="center">Negative predictive value (%)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">&nbsp;</th><th rowspan="1" colspan="1" valign="top" align="center">Any cancer</th><th rowspan="1" colspan="1" valign="top" align="center">GS 7</th><th rowspan="1" colspan="1" valign="top" align="center">GS 8–10</th><th rowspan="1" colspan="1" valign="top" align="center">Any cancer</th><th rowspan="1" colspan="1" valign="top" align="center">GS 7</th><th rowspan="1" colspan="1" valign="top" align="center">GS 8–10</th><th rowspan="1" colspan="1" valign="top" align="center">Any cancer</th><th rowspan="1" colspan="1" valign="top" align="center">GS 7<br>(26.5%)</th><th rowspan="1" colspan="1" valign="top" align="center">GS 8–10<br>(5.6%)</th><th rowspan="1" colspan="1" valign="top" align="center">Any cancer</th><th rowspan="1" colspan="1" valign="top" align="center">GS 7<br>(73.5%)</th><th rowspan="1" colspan="1" valign="top" align="center">GS 8–10<br>(94.4%)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">1 Variant allele cut-off<hr></td><td rowspan="1" colspan="1" valign="top" align="center">94.4<hr></td><td rowspan="1" colspan="1" valign="top" align="center">94.5<hr></td><td rowspan="1" colspan="1" valign="top" align="center">95.6<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">7.2<hr></td><td rowspan="1" colspan="1" valign="top" align="center">6.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">49.8<hr></td><td rowspan="1" colspan="1" valign="top" align="center">26.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">5.8<hr></td><td rowspan="1" colspan="1" valign="top" align="center">59.5<hr></td><td rowspan="1" colspan="1" valign="top" align="center">78.4<hr></td><td rowspan="1" colspan="1" valign="top" align="center">96.3<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">2 Variant alleles cut-off<hr></td><td rowspan="1" colspan="1" valign="top" align="center">62.0<hr></td><td rowspan="1" colspan="1" valign="top" align="center">62.0<hr></td><td rowspan="1" colspan="1" valign="top" align="center">70.4<hr></td><td rowspan="1" colspan="1" valign="top" align="center">46.6<hr></td><td rowspan="1" colspan="1" valign="top" align="center">43.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">43.1<hr></td><td rowspan="1" colspan="1" valign="top" align="center">52.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">28.5<hr></td><td rowspan="1" colspan="1" valign="top" align="center">6.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">55.9<hr></td><td rowspan="1" colspan="1" valign="top" align="center">76.2<hr></td><td rowspan="1" colspan="1" valign="top" align="center">96.1<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3 Variant alleles cut-off</td><td rowspan="1" colspan="1" valign="top" align="center">29.5</td><td rowspan="1" colspan="1" valign="top" align="center">30.7</td><td rowspan="1" colspan="1" valign="top" align="center">37.1</td><td rowspan="1" colspan="1" valign="top" align="center">78.2</td><td rowspan="1" colspan="1" valign="top" align="center">76.3</td><td rowspan="1" colspan="1" valign="top" align="center">75.1</td><td rowspan="1" colspan="1" valign="top" align="center">56.8</td><td rowspan="1" colspan="1" valign="top" align="center">31.9</td><td rowspan="1" colspan="1" valign="top" align="center">8.2</td><td rowspan="1" colspan="1" valign="top" align="center">53.4</td><td rowspan="1" colspan="1" valign="top" align="center">75.3</td><td rowspan="1" colspan="1" valign="top" align="center">95.2</td></tr></tbody></table></div><div id="largeobj_idm139884673803792" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3280918/table/T5/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3280918/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139884712592416" class="figure"><a href="/pmc/articles/PMC3280918/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3280918_cbt-12-997-g2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is cbt-12-997-g2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3280918/bin/cbt-12-997-g2.jpg"></a></div><div id="largeobj_idm139884712592416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3280918/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3280918/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p25">ROC plot of 3 SNP model based on method by Kraft et al. among white subjects only. For ROC plots of nomogram, see Figure&nbsp;S3.</p></div></div></div></div></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Discussion</h2><p id="__p26" class="p p-first">It is well established that men diagnosed with intermediate or high-risk prostate cancer based on the D’Amico criteria have a high chance for progression to metastasis. It is of primary importance to identify these men in a screening program. Our study is the first to demonstrate association of 10q26 and 15q21 region SNPs with prostate cancer, and to show that a combination of these risk alleles (three SNP model) is associated with aggressive prostate cancer. From nomogram analysis, the clinical performance of our SNP variants was not superior to PSA or age, but was important as other established risk factors, such as family history of prostate cancer.</p><p id="__p27">The stage 1 analysis of this study did not reveal any associations between prostate cancer and previously-identified risk loci on 8q24 and 17q. This discrepancy likely reflects our specific analysis of patients with aggressive prostate cancer, as our prior analysis of the patient cohort included in stage 2, did reveal associations between 8q24 and 17q SNPs and prostate cancer, but not aggressive cancer.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198892">15</a></sup> Similarly, results of two recent large, multi-centered studies revealed no association of prostate cancer aggressiveness and mortality with these or other putative prostate cancer SNPs found by past large GWAS.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198903">4</a></sup><sup>,</sup><sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198886">16</a></sup></p><p id="__p28">The odds ratios for the five SNPs identified by our Stage 1 analysis were larger than those identified in Stage 2. This result could reflect a “Winner’s Curse” phenomenon,<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_382986229">17</a></sup><sup>,</sup><sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198902">18</a></sup> wherein the estimated effect of a marker allele from the initial GWAS may be exaggerated related to the estimated effect in a confirmatory study. Such a possibility could relate to our selection of “hypernormal” controls in Stage 1—men who had normal PSA levels or had multiple biopsies with no evidence of cancer.</p><p id="__p29">No previous studies have reported prostate cancer association with the chromosomal 10q26 and 15q21 regions or with genes in these regions. The three disease-associated SNPs on chromosome 10q26 map within 70 kb from one another and the two SNPs in LD (rs11199874 and rs10788165) are separated by 12 kb. These three SNPs span a 590 kb region encompassing two genes, WDR11 and FGFR2, that have been linked to glioblastomas<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198894">19</a></sup> and breast cancer.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198889">20</a></sup> The two SNPs on 15q21 are 13 kb apart and map 880 kb from the closest gene, GATM, which encodes a mitochondrial enzyme.</p><p id="__p30">A limitation of our study is that a proportion of our controls may be misclassified due to false-negative results of the prostate biopsy. Approximately 10–20% of patients have been found to have cancer from repeat prostate biopsy after an initial negative biopsy.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198891">21</a></sup><sup>,</sup><sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198901">22</a></sup> In Stage 1, this bias was minimized by selecting control patients who had a low risk for having prostate cancer with low PSA levels (&lt; 10 ng/mL, with half having a normal PSA level of &lt; 4.0 ng/mL).</p><p id="__p31" class="p p-last">Thus, from a GWAS based on cases with aggressive prostate cancer and controls with no evidence of cancer from biopsy, we have found new associations of SNPs at 10q26 and 15q21. Certain combinations of these SNPs are associated with aggressive forms of prostate cancer and have similar clinical importance compared with other risk factors for prostate cancer. Further elucidation of additional SNPs or genes in these new regions would be important to examine before these and potentially other SNPs from these regions can be used clinically to predict prostate cancer.</p></div><div id="__sec6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec6title">Materials and Methods</h2><div id="__sec7" class="sec sec-first"><h3 id="__sec7title">Study subjects</h3><p id="__p32" class="p p-first-last">Patients were drawn from a sample of 4573 men who underwent a prostate biopsy from a prostate cancer screening program between 1999 and 2008 within an urban, North American-based population. Patients of any age were included in the study if they had an abnormal prostate specific antigen (PSA) value (&gt; 4.0 ng/mL) or digital rectal exam (DRE). Patients were also included if they had a normal PSA level (&lt; 4.0 ng/mL) or DRE, but were willing to undergo a prostate biopsy for the purpose of prostate cancer screening (n = 408). Patients eligible for this study were unselected and were accrued consecutively. No patient had a past history of prostate cancer. All patients underwent one or more transrectal ultrasonography (TRUS)-guided needle core biopsies. Patients were excluded if they were not capable of giving consent to participate in a research study (n = 46); or if they could not provide sufficient baseline information (n = 53). Blood samples were obtained prior to prostate biopsy. Six to fifteen ultrasound-guided needle core biopsies were performed (median = 8), using an 18-gauge spring loaded biopsy device. Samples were obtained using a systematic pattern and additional targeted samples were obtained from suspicious areas. The primary endpoint was the histologic presence of adenocarcinoma of the prostate in the biopsy specimen. All grading was based on the Gleason scoring system.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198897">9</a></sup> A urological voiding history (AUA Symptom Score<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198885">10</a></sup>), DRE results, serum PSA level, family history of prostate cancer information, and ethnic background were obtained. This study was approved by the research ethics board (Sunnybrook Health Sciences Center).</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Stage analysis</h3><p id="__p33" class="p p-first">To maximize the probability of identifying SNPs that can predict patients with aggressive forms of prostate cancer we conducted our study in two stages. Stage 1 consisted of a GWAS to identify novel SNPs. We restricted our GWAS analysis to patients diagnosed with intermediate to high risk prostate cancer for progression to metastasis defined by the D`Amico criteria<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198896">8</a></sup> of prostate cancer risk in order to maximize the probability of finding clinically significant SNPs (rather than those associated with indolent forms of prostate cancer). Stage 2 consisted of a replication study among a separate cohort to determine whether the SNPs from Stage 1 could predict aggressive forms of prostate cancer among a prostate biopsy screening cohort who underwent a prostate biopsy. To fully evaluate whether SNPs can be used in a clinical setting within the context of prostate cancer detection and screening, we evaluated whether these SNPs could predict any prostate cancer and its aggressive forms.</p><p id="__p34">For Stage 1, subjects were derived from the first 1,000 subjects who underwent a prostate biopsy. We identified 545 cases and controls from the 1,000 eligible patients for SNP array analysis. Of the 545 patients, 316 cases and 229 controls were genotyped using the Affymetrix 500K SNP array (443,816 SNPs). Cases were white patients with screen-detected prostate cancer and who had a high risk of progression based on the D’Amico criteria which is and established method of risk stratification for prostate cancer progression; i.e., either a Gleason Score of 7 and a PSA level of &gt; 10 ng/mL or a Gleason Score of 8–10. Controls were white patients who had no evidence of cancer, based on one or more systematic multi-core (median number = 8 cores per biopsy session) prostate biopsies; 105 controls had a normal PSA level and 124 controls had a PSA level between 4.0 and 10.0 ng/mL.</p><p id="__p35" class="p p-last">For Stage 2, the remaining 3573 patients were included in the replication study. The primary endpoint was the presence of adenocarcinoma of the prostate. We assessed whether the SNPs from Stage 1 was associated with any prostate cancer and aggressive forms defined by the D’Amico criteria. This criteria categorizes patients with prostate cancer into three groups: low, intermediate and high risk based Gleason score, clinical stage and PSA level. Because of the three categories of aggressiveness and the potential multiple number of SNP genotypes found, to assess the association with aggressive forms of prostate cancer, we grouped the SNPs in combination and compared them to the D’Amico risk groups. We also examined their ability to predict prostate and aggressive forms of prostate within a nomogram model to assess its clinical significance.</p></div><div id="__sec9" class="sec"><h3 id="__sec9title">SNP array analysis and genotyping</h3><p id="__p36" class="p p-first">In Stage 1, using the 500K Affymetrix SNP array, 443,816 SNPs were analyzed. Of those SNPs, 401,593 were retained after being excluded for not being in Hardy-Weinberg equilibrium (HWE)(p &gt; 0.001), had a minor allele frequency of &lt; 0.05, or a call rate of &lt; 95%. To deal with population stratification, we used principal component analysis to adjust for variances within ethnic subgroups with EIGENSTRAT (Golden HelixTree, V6.4.1).<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198899">11</a></sup> Five eigenvectors were used and three clusters were observed. A Q-Q plot is provided in Figure&nbsp;S1.</p><p id="__p37" class="p p-last">Genotyping was conducted using mass-spectrometry-based genotyping analysis and matrix-assisted laser desorption ionization—time of flight (MassArray System, Sequenom Inc.) following the manufacturer’s instructions. A standard protocol for multiplex homogeneous mass extend assay developed by Sequenom Inc. was utilized and modified according to designed primers. For quality control, we assigned negative controls for each test plate (Microseal TM 384 V2.0).</p></div><div id="__sec10" class="sec"><h3 id="__sec10title">Data analysis for replication study</h3><p id="__p38" class="p p-first-last">For Stage 2, we excluded 40 samples that failed on two or more of the assays used. The call rates were at least 95% for each SNP. Genotype distributions for all tested SNPs were in Hardy-Weinberg Equilibrium (HWE). Cases were defined as patients with prostate cancer and controls were men with no evidence of cancer. Allele frequencies for each SNP were calculated for cases and controls and the distributions were compared. Allelic odds ratios were calculated based on a multiplicative model. For genotypes, frequency comparisons were compared for an additive, dominant or recessive genetic model for each of the SNPs with unconditional logistic regression. The model with the highest likelihood was considered to be the best fitting model for each SNP.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198900">12</a></sup> We tested the cumulative effects of selected SNPs for each model by counting the number of genotypes associated with prostate cancer on the basis of the best-fitting genetic model from single-SNP analysis; the odds ratios for prostate cancer for patients with one or more variant genotypes were estimated. In the multivariate analysis, we adjusted for age, ethnic group, family history of prostate cancer, the presence of lower urinary tract voiding symptoms, the total PSA level, and the digital rectal examination. Unconditional logistic regression analysis was used to examine how each of these factors, alone and in combination, would predict the presence of prostate cancer and aggressive forms of prostate cancer, defined by the D’Amico classification.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256198893">8</a></sup></p></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Nomogram construction</h3><p id="__p39" class="p p-first-last">To develop a clinical instrument that incorporates the SNP findings, we considered all SNPs in a nomogram multivariate model, restricted to white subjects only. This nomogram was designed to predict both prostate cancer and aggressive cancer defined as having intermediate to high-grade prostate cancer (Gleason Score 7 or more). Ordinal logistic regression was used to model the probability of having low or high grade cancer. Three outcome levels were defined: (1) no cancer; (2) low-grade cancer (Gleason Score 6 or less; and (3) intermediate to high-grade cancer (Gleason Score 7 or more). Continuous variables were modeled with restricted cubic splines to avoid linearity assumptions. The logistic regression model was the basis for constructing a nomogram. All patients were used to develop and validate the nomogram. Bootstrapping was used to correct for optimism in the evaluation of discrimination and calibration. All analyses were performed using S-Plus 2000 Professional software (Statistical Sciences) with the Design and Hmisc libraries added.<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a></sup></p></div></div><div id="__sec12" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="idm139884712091968"><!--caption a9--><strong>Note:</strong><p id="__p40">Supplementary materials can be found at: <a href="http://www.landesbioscience.com/journals/cbt/article/18366/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.landesbioscience.com/journals/cbt/article/18366/</a>.</p></div></div><div id="idm139884712224816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712224816title">Acknowledgments</h2><div class="sec"><p id="__p41">This study was funded in part by the Canadian Institute of Health Research, Operating Grant MOP-79530 and the National Cancer Institute Grants 010294 and 017092 and CCSRI 019038</p></div></div><div id="idm139884712221696" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712221696title">Disclosure of Potential Conflicts of Interest</h2><p id="__p43">No potential conflicts of interest were disclosed.</p></div><div id="idm139884712220816" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712220816title">Note</h2><p id="__p44">Supplemental material can be found at: <a href="http://www.landesbioscience.com/journals/cbt/article/18366/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.landesbioscience.com/journals/cbt/article/18366/</a>.</p></div><div id="idm139884712223744" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712223744title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm139884712223616"></p><p id="__p42" class="p p-first-last">Previously published online: <a href="http://www.landesbioscience.com/journals/cbt/article/18366/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.landesbioscience.com/journals/cbt/article/18366</a></p><p></p></div></div><div id="idm139884712218672" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139884712218672title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Eeles RA, Durocher F, Edwards S, Teare D, Badzioch M, Hamoudi R, et al.  Linkage analysis of chromosome 1q markers in 136 prostate cancer families. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">62</span>:653–8. doi: 10.1086/301745.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1376940/">PMC free article</a>]</span> [<a href="/pubmed/9497242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1086%2F301745" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Linkage+analysis+of+chromosome+1q+markers+in+136+prostate+cancer+families&amp;author=RA+Eeles&amp;author=F+Durocher&amp;author=S+Edwards&amp;author=D+Teare&amp;author=M+Badzioch&amp;volume=62&amp;publication_year=1998&amp;pages=653-8&amp;pmid=9497242&amp;doi=10.1086/301745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Gudmundsson J, Sulem P, Manolescu A, Amundadottir LT, Gudbjartsson D, Helgason A, et al.  Genome-wide association study identifies a second prostate cancer susceptibility variant at 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:631–7. doi: 10.1038/ng1999.</span> [<a href="/pubmed/17401366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng1999" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+second+prostate+cancer+susceptibility+variant+at+8q24&amp;author=J+Gudmundsson&amp;author=P+Sulem&amp;author=A+Manolescu&amp;author=LT+Amundadottir&amp;author=D+Gudbjartsson&amp;volume=39&amp;publication_year=2007&amp;pages=631-7&amp;pmid=17401366&amp;doi=10.1038/ng1999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Yeager M, Orr N, Hayes RB, Jacobs KB, Kraft P, Wacholder S, et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:645–9. doi: 10.1038/ng2022.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng2022" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M+Yeager&amp;author=N+Orr&amp;author=RB+Hayes&amp;author=KB+Jacobs&amp;author=P+Kraft&amp;volume=39&amp;publication_year=2007&amp;pages=645-9&amp;pmid=17401363&amp;doi=10.1038/ng2022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Penney KL, Salinas CA, Pomerantz M, Schumacher FR, Beckwith CA, Lee GS, et al.  Evaluation of 8q24 and 17q risk loci and prostate cancer mortality. <span><span class="ref-journal">Clin Cancer Res. </span>2009;<span class="ref-vol">15</span>:3223–30. doi: 10.1158/1078-0432.CCR-08-2733.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2878092/">PMC free article</a>]</span> [<a href="/pubmed/19366828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1158%2F1078-0432.CCR-08-2733" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Evaluation+of+8q24+and+17q+risk+loci+and+prostate+cancer+mortality&amp;author=KL+Penney&amp;author=CA+Salinas&amp;author=M+Pomerantz&amp;author=FR+Schumacher&amp;author=CA+Beckwith&amp;volume=15&amp;publication_year=2009&amp;pages=3223-30&amp;pmid=19366828&amp;doi=10.1158/1078-0432.CCR-08-2733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Wiklund FE, Adami HO, Zheng SL, Stattin P, Isaacs WB, Gronberg H, et al.  Established prostate cancer susceptibility variants are not associated with disease outcome. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2009;<span class="ref-vol">18</span>:1659–62. doi: 10.1158/1055-9965.EPI-08-1148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3353269/">PMC free article</a>]</span> [<a href="/pubmed/19423541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1158%2F1055-9965.EPI-08-1148" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Established+prostate+cancer+susceptibility+variants+are+not+associated+with+disease+outcome&amp;author=FE+Wiklund&amp;author=HO+Adami&amp;author=SL+Zheng&amp;author=P+Stattin&amp;author=WB+Isaacs&amp;volume=18&amp;publication_year=2009&amp;pages=1659-62&amp;pmid=19423541&amp;doi=10.1158/1055-9965.EPI-08-1148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Thompson IM, Goodman PJ, Tangen CM, Lucia MS, Miller GJ, Ford LG, et al.  The influence of finasteride on the development of prostate cancer. <span><span class="ref-journal">N Engl J Med. </span>2003;<span class="ref-vol">349</span>:215–24. doi: 10.1056/NEJMoa030660.</span> [<a href="/pubmed/12824459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1056%2FNEJMoa030660" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=The+influence+of+finasteride+on+the+development+of+prostate+cancer&amp;author=IM+Thompson&amp;author=PJ+Goodman&amp;author=CM+Tangen&amp;author=MS+Lucia&amp;author=GJ+Miller&amp;volume=349&amp;publication_year=2003&amp;pages=215-24&amp;pmid=12824459&amp;doi=10.1056/NEJMoa030660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Thompson IM, Pauler DK, Goodman PJ, Tangen CM, Lucia MS, Parnes HL, et al.  Prevalence of prostate cancer among men with a prostate-specific antigen level &lt; or =4.0 ng per milliliter. <span><span class="ref-journal">N Engl J Med. </span>2004;<span class="ref-vol">350</span>:2239–46. doi: 10.1056/NEJMoa031918.</span> [<a href="/pubmed/15163773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1056%2FNEJMoa031918" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Prevalence+of+prostate+cancer+among+men+with+a+prostate-specific+antigen+level+<+or+=4.0+ng+per+milliliter&amp;author=IM+Thompson&amp;author=DK+Pauler&amp;author=PJ+Goodman&amp;author=CM+Tangen&amp;author=MS+Lucia&amp;volume=350&amp;publication_year=2004&amp;pages=2239-46&amp;pmid=15163773&amp;doi=10.1056/NEJMoa031918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">D'Amico AV, Whittington R, Malkowicz SB, Schultz D, Blank K, Broderick GA, et al.  Biochemical outcome after radical prostatectomy, external beam radiation therapy, or interstitial radiation therapy for clinically localized prostate cancer. <span><span class="ref-journal">JAMA. </span>1998;<span class="ref-vol">280</span>:969–74. doi: 10.1001/jama.280.11.969.</span> [<a href="/pubmed/9749478" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1001%2Fjama.280.11.969" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Biochemical+outcome+after+radical+prostatectomy,+external+beam+radiation+therapy,+or+interstitial+radiation+therapy+for+clinically+localized+prostate+cancer&amp;author=AV+D'Amico&amp;author=R+Whittington&amp;author=SB+Malkowicz&amp;author=D+Schultz&amp;author=K+Blank&amp;volume=280&amp;publication_year=1998&amp;pages=969-74&amp;pmid=9749478&amp;doi=10.1001/jama.280.11.969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Gleason DF, Mellinger GT. Prediction of prognosis for prostatic adenocarcinoma by combined histological grading and clinical staging. <span><span class="ref-journal">J Urol. </span>1974;<span class="ref-vol">111</span>:58–64.</span> [<a href="/pubmed/4813554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Urol&amp;title=Prediction+of+prognosis+for+prostatic+adenocarcinoma+by+combined+histological+grading+and+clinical+staging&amp;author=DF+Gleason&amp;author=GT+Mellinger&amp;volume=111&amp;publication_year=1974&amp;pages=58-64&amp;pmid=4813554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Barry MJ, Fowler FJJ, O'Leary MP, Bruskewitz RC, Holtgrewe HL, Mebust WK, et al.  The American Urological Association symptom index for benign prostatic hyperplasia. The Measurement Committee of the American Urological Association. <span><span class="ref-journal">J Urol. </span>1992;<span class="ref-vol">148</span>:1549–57.</span> [<a href="/pubmed/1279218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Urol&amp;title=The+American+Urological+Association+symptom+index+for+benign+prostatic+hyperplasia.+The+Measurement+Committee+of+the+American+Urological+Association&amp;author=MJ+Barry&amp;author=FJJ+Fowler&amp;author=MP+O'Leary&amp;author=RC+Bruskewitz&amp;author=HL+Holtgrewe&amp;volume=148&amp;publication_year=1992&amp;pages=1549-57&amp;pmid=1279218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9. doi: 10.1038/ng1847.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng1847" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;doi=10.1038/ng1847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Zheng SL, Sun J, Wiklund F, Smith S, Stattin P, Li G, et al.  Cumulative association of five genetic variants with prostate cancer. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:910–9. doi: 10.1056/NEJMoa075819.</span> [<a href="/pubmed/18199855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1056%2FNEJMoa075819" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Cumulative+association+of+five+genetic+variants+with+prostate+cancer&amp;author=SL+Zheng&amp;author=J+Sun&amp;author=F+Wiklund&amp;author=S+Smith&amp;author=P+Stattin&amp;volume=358&amp;publication_year=2008&amp;pages=910-9&amp;pmid=18199855&amp;doi=10.1056/NEJMoa075819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Harrell FE. Regression modeling strategies with applications to linear models. In: Logistic Regression, and Survival Analysis. New York: Springer-Verlag 2001. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Harrell+FE.+Regression+modeling+strategies+with+applications+to+linear+models.+In:+Logistic+Regression,+and+Survival+Analysis.+New+York:+Springer-Verlag+2001.+" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kraft P, Wacholder S, Cornelis MC, Hu FB, Hayes RB, Thomas G, et al.  Beyond odds ratios–communicating disease risk based on genetic profiles. <span><span class="ref-journal">Nat Rev. </span>2009;<span class="ref-vol">10</span>:264–9.</span> [<a href="/pubmed/19238176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev&amp;title=Beyond+odds+ratios–communicating+disease+risk+based+on+genetic+profiles&amp;author=P+Kraft&amp;author=S+Wacholder&amp;author=MC+Cornelis&amp;author=FB+Hu&amp;author=RB+Hayes&amp;volume=10&amp;publication_year=2009&amp;pages=264-9&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Nam RK, Zhang WW, Trachtenberg J, Seth A, Klotz LH, Stanimirovic A, et al.  Utility of Incorporating Genetic Variants for the Early Detection of Prostate Cancer. <span><span class="ref-journal">Clin Cancer Res. </span>2009;<span class="ref-vol">15</span>:1787–93. doi: 10.1158/1078-0432.CCR-08-1593.</span> [<a href="/pubmed/19223501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1158%2F1078-0432.CCR-08-1593" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Utility+of+Incorporating+Genetic+Variants+for+the+Early+Detection+of+Prostate+Cancer&amp;author=RK+Nam&amp;author=WW+Zhang&amp;author=J+Trachtenberg&amp;author=A+Seth&amp;author=LH+Klotz&amp;volume=15&amp;publication_year=2009&amp;pages=1787-93&amp;pmid=19223501&amp;doi=10.1158/1078-0432.CCR-08-1593&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Kader AK, Sun J, Isaacs SD, Wiley KE, Yan G, Kim ST, et al.  Individual and cumulative effect of prostate cancer risk-associated variants on clinicopathologic variables in 5,895 prostate cancer patients. <span><span class="ref-journal">Prostate. </span>2009 doi: 10.1002/pros.20970.</span> In press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852875/">PMC free article</a>]</span> [<a href="/pubmed/19434657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1002%2Fpros.20970" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prostate&amp;title=Individual+and+cumulative+effect+of+prostate+cancer+risk-associated+variants+on+clinicopathologic+variables+in+5,895+prostate+cancer+patients&amp;author=AK+Kader&amp;author=J+Sun&amp;author=SD+Isaacs&amp;author=KE+Wiley&amp;author=G+Yan&amp;publication_year=2009&amp;pmid=19434657&amp;doi=10.1002/pros.20970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Kraft P. Curses—winner's and otherwise—in genetic epidemiology. In: Epidemiology. Cambridge, Mass 2008:649-51; discussion 57-8. [<a href="/pubmed/18703928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Kraft+P.+Curses—winner's+and+otherwise—in+genetic+epidemiology.+In:+Epidemiology.+Cambridge,+Mass+2008:649-51;+discussion+57-8.+" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Lohmueller KE, Pearce CL, Pike M, Lander ES, Hirschhorn JN. Meta-analysis of genetic association studies supports a contribution of common variants to susceptibility to common disease. <span><span class="ref-journal">Nat Genet. </span>2003;<span class="ref-vol">33</span>:177–82. doi: 10.1038/ng1071.</span> [<a href="/pubmed/12524541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng1071" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genetic+association+studies+supports+a+contribution+of+common+variants+to+susceptibility+to+common+disease&amp;author=KE+Lohmueller&amp;author=CL+Pearce&amp;author=M+Pike&amp;author=ES+Lander&amp;author=JN+Hirschhorn&amp;volume=33&amp;publication_year=2003&amp;pages=177-82&amp;pmid=12524541&amp;doi=10.1038/ng1071&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Chernova OB, Hunyadi A, Malaj E, Pan H, Crooks C, Roe B, et al.  A novel member of the WD-repeat gene family, WDR11, maps to the 10q26 region and is disrupted by a chromosome translocation in human glioblastoma cells. <span><span class="ref-journal">Oncogene. </span>2001;<span class="ref-vol">20</span>:5378–92. doi: 10.1038/sj.onc.1204694.</span> [<a href="/pubmed/11536051" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fsj.onc.1204694" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=A+novel+member+of+the+WD-repeat+gene+family,+WDR11,+maps+to+the+10q26+region+and+is+disrupted+by+a+chromosome+translocation+in+human+glioblastoma+cells&amp;author=OB+Chernova&amp;author=A+Hunyadi&amp;author=E+Malaj&amp;author=H+Pan&amp;author=C+Crooks&amp;volume=20&amp;publication_year=2001&amp;pages=5378-92&amp;pmid=11536051&amp;doi=10.1038/sj.onc.1204694&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Liang J, Chen P, Hu Z, Zhou X, Chen L, Li M, et al.  Genetic variants in fibroblast growth factor receptor 2 (FGFR2) contribute to susceptibility of breast cancer in Chinese women. <span><span class="ref-journal">Carcinogenesis. </span>2008;<span class="ref-vol">29</span>:2341–6. doi: 10.1093/carcin/bgn235.</span> [<a href="/pubmed/18845558" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1093%2Fcarcin%2Fbgn235" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Genetic+variants+in+fibroblast+growth+factor+receptor+2+(FGFR2)+contribute+to+susceptibility+of+breast+cancer+in+Chinese+women&amp;author=J+Liang&amp;author=P+Chen&amp;author=Z+Hu&amp;author=X+Zhou&amp;author=L+Chen&amp;volume=29&amp;publication_year=2008&amp;pages=2341-6&amp;pmid=18845558&amp;doi=10.1093/carcin/bgn235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Nam RK, Toi A, Trachtenberg J, Jewett MA, Klotz L, Fleshner N, et al.  Variation in patterns of practice in diagnosing screen-detected prostate cancer. <span><span class="ref-journal">BJU Int. </span>2004;<span class="ref-vol">94</span>:1239–44. doi: 10.1111/j.1464-410X.2004.05150.x.</span> [<a href="/pubmed/15610097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fj.1464-410X.2004.05150.x" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BJU+Int&amp;title=Variation+in+patterns+of+practice+in+diagnosing+screen-detected+prostate+cancer&amp;author=RK+Nam&amp;author=A+Toi&amp;author=J+Trachtenberg&amp;author=MA+Jewett&amp;author=L+Klotz&amp;volume=94&amp;publication_year=2004&amp;pages=1239-44&amp;pmid=15610097&amp;doi=10.1111/j.1464-410X.2004.05150.x&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Presti JC., Jr Prostate biopsy strategies. <span><span class="ref-journal">Nat Clin Pract Urol. </span>2007;<span class="ref-vol">4</span>:505–11. doi: 10.1038/ncpuro0887.</span> [<a href="/pubmed/17823604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fncpuro0887" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Clin+Pract+Urol&amp;title=Prostate+biopsy+strategies&amp;author=JC+Presti&amp;volume=4&amp;publication_year=2007&amp;pages=505-11&amp;pmid=17823604&amp;doi=10.1038/ncpuro0887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3280918&amp;issue-id=205681&amp;journal-id=1361&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139884712439584title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Results</a></li><li><a href="#__sec5title">Discussion</a></li><li><a href="#__sec6title">Materials and Methods</a></li><li><a href="#__sec12title">Supplementary Material</a></li><li><a href="#idm139884712224816title">Acknowledgments</a></li><li><a href="#idm139884712221696title">Disclosure of Potential Conflicts of Interest</a></li><li><a href="#idm139884712220816title">Note</a></li><li><a href="#idm139884712223744title">Footnotes</a></li><li><a href="#idm139884712218672title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Cancer Biology &amp; Therapy</span> are provided here courtesy of <strong>Taylor &amp; Francis</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3280918/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3280918/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3280918/pdf/cbt-12-997.pdf">PDF (299K)</a></li> | <li><a href="#" data-citationid="PMC3280918" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3280918%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3280918%2F&amp;text=New%20variants%20at%2010q26%20and%2015q21%20are%20associated%20with%20aggressive%20prostate%20cancer%20in%20a%20genome-wide%20association%20study%20from%20a%20prostate%20biopsy%20screening%20cohort" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3280918%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22130093" data-aiid="3280918" data-aid="3280918" data-iid="205681" data-domainid="1361" data-domain="cbt" data-accid="PMC3280918" data-md5="a4e75c5d0fa88f1ae90ea56c86d1bac6"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:12:55-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8824D9DD53862100000000025801A9&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3280918%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=cbt&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3280918/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>