<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3713791/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome- and Phenome-Wide Analysis of Cardiac Conduction Identifies Markers of Arrhythmia Risk</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3713791/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Circulation"><meta name="citation_title" content="Genome- and Phenome-Wide Analysis of Cardiac Conduction Identifies Markers of Arrhythmia Risk"><meta name="citation_authors" content="Marylyn D. Ritchie, Joshua C. Denny, Rebecca L. Zuvich, Dana C. Crawford, Jonathan S. Schildcrout, Lisa Bastarache, Andrea H. Ramirez, Jonathan D. Mosley, Jill M. Pulley, Melissa A. Basford, Yuki Bradford, Luke V. Rasmussen, Jyotishman Pathak, Christopher G. Chute, Iftikhar J. Kullo, Catherine A. McCarty, Rex L. Chisholm, Abel N. Kho, Christopher S. Carlson, Eric B. Larson, Gail P. Jarvik, Nona Sotoodehnia, Teri A. Manolio, Rongling Li, Daniel R. Masys, Jonathan L. Haines, Dan M. Roden"><meta name="citation_date" content="2 April 2013"><meta name="citation_issue" content="13"><meta name="citation_volume" content="127"><meta name="citation_firstpage" content="1377"><meta name="citation_doi" content="10.1161/CIRCULATIONAHA.112.000604"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3713791/?report=abstract"><meta name="citation_pmid" content="23463857"><meta name="DC.Title" content="Genome- and Phenome-Wide Analysis of Cardiac Conduction Identifies Markers of Arrhythmia Risk"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Marylyn D. Ritchie"><meta name="DC.Contributor" content="Joshua C. Denny"><meta name="DC.Contributor" content="Rebecca L. Zuvich"><meta name="DC.Contributor" content="Dana C. Crawford"><meta name="DC.Contributor" content="Jonathan S. Schildcrout"><meta name="DC.Contributor" content="Lisa Bastarache"><meta name="DC.Contributor" content="Andrea H. Ramirez"><meta name="DC.Contributor" content="Jonathan D. Mosley"><meta name="DC.Contributor" content="Jill M. Pulley"><meta name="DC.Contributor" content="Melissa A. Basford"><meta name="DC.Contributor" content="Yuki Bradford"><meta name="DC.Contributor" content="Luke V. Rasmussen"><meta name="DC.Contributor" content="Jyotishman Pathak"><meta name="DC.Contributor" content="Christopher G. Chute"><meta name="DC.Contributor" content="Iftikhar J. Kullo"><meta name="DC.Contributor" content="Catherine A. McCarty"><meta name="DC.Contributor" content="Rex L. Chisholm"><meta name="DC.Contributor" content="Abel N. Kho"><meta name="DC.Contributor" content="Christopher S. Carlson"><meta name="DC.Contributor" content="Eric B. Larson"><meta name="DC.Contributor" content="Gail P. Jarvik"><meta name="DC.Contributor" content="Nona Sotoodehnia"><meta name="DC.Contributor" content="Teri A. Manolio"><meta name="DC.Contributor" content="Rongling Li"><meta name="DC.Contributor" content="Daniel R. Masys"><meta name="DC.Contributor" content="Jonathan L. Haines"><meta name="DC.Contributor" content="Dan M. Roden"><meta name="DC.Date" content="2013 Apr 2"><meta name="DC.Identifier" content="10.1161/CIRCULATIONAHA.112.000604"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome- and Phenome-Wide Analysis of Cardiac Conduction Identifies Markers of Arrhythmia Risk"><meta property="og:type" content="article"><meta property="og:description" content="Electrocardiographic QRS duration, a measure of cardiac intraventricular conduction, varies ~2-fold in individuals without cardiac disease. Slow conduction may promote reentrant arrhythmias.We performed a genome-wide association study (GWAS) to identify ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3713791/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3713791/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3713791/pdf/nihms-484529.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A65B6DD55795100000000028E01A2.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3713791%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3713791</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Circulation</a></span>. Author manuscript; available in PMC 2014 Apr 2.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23463857" target="pmc_ext" ref="reftype=publisher&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Circulation. 2013 Apr 2; 127(13): 1377–1385. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Mar 5. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1161%2FCIRCULATIONAHA.112.000604" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1161/CIRCULATIONAHA.112.000604</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3713791</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS484529</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23463857">23463857</a></div></div></div></div><h1 class="content-title">Genome- and Phenome-Wide Analysis of Cardiac Conduction Identifies Markers of Arrhythmia Risk</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680573709792" co-class="co-affbox">Marylyn D. Ritchie</a>, PhD,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680577572832" co-class="co-affbox">Joshua C. Denny</a>, MD MS,<sup>2,</sup><sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Zuvich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680577569344" co-class="co-affbox">Rebecca L. Zuvich</a>, PhD,<sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680578350000" co-class="co-affbox">Dana C. Crawford</a>, PhD,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Schildcrout%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680568735280" co-class="co-affbox">Jonathan S. Schildcrout</a>, PhD,<sup>6</sup> <a href="/pubmed/?term=Bastarache%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680581684720" co-class="co-affbox">Lisa Bastarache</a>, MS,<sup>2</sup> <a href="/pubmed/?term=Ramirez%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680581682480" co-class="co-affbox">Andrea H. Ramirez</a>, MD,<sup>3</sup> <a href="/pubmed/?term=Mosley%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680582251824" co-class="co-affbox">Jonathan D. Mosley</a>, MD PhD,<sup>3</sup> <a href="/pubmed/?term=Pulley%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680582249584" co-class="co-affbox">Jill M. Pulley</a>, MBA,<sup>7</sup> <a href="/pubmed/?term=Basford%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680579157616" co-class="co-affbox">Melissa A. Basford</a>, MBA,<sup>7</sup> <a href="/pubmed/?term=Bradford%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680579155376" co-class="co-affbox">Yuki Bradford</a>, MS,<sup>5</sup> <a href="/pubmed/?term=Rasmussen%20LV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680576445168" co-class="co-affbox">Luke V. Rasmussen</a>,<sup>8</sup> <a href="/pubmed/?term=Pathak%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680576443184" co-class="co-affbox">Jyotishman Pathak</a>, PhD,<sup>9</sup> <a href="/pubmed/?term=Chute%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680562922848" co-class="co-affbox">Christopher G. Chute</a>, MD DrPH,<sup>9</sup> <a href="/pubmed/?term=Kullo%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680562920608" co-class="co-affbox">Iftikhar J. Kullo</a>, MD,<sup>10</sup> <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680570543488" co-class="co-affbox">Catherine A. McCarty</a>, PhD,<sup>11</sup> <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680579650048" co-class="co-affbox">Rex L. Chisholm</a>, PhD,<sup>12</sup> <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680579647808" co-class="co-affbox">Abel N. Kho</a>, MD MS,<sup>13</sup> <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680575743936" co-class="co-affbox">Christopher S. Carlson</a>, PhD,<sup>14</sup> <a href="/pubmed/?term=Larson%20EB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680575741696" co-class="co-affbox">Eric B. Larson</a>, MD MPH,<sup>15</sup> <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680572983248" co-class="co-affbox">Gail P. Jarvik</a>, MD PhD,<sup>16,</sup><sup>19</sup> <a href="/pubmed/?term=Sotoodehnia%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680572980272" co-class="co-affbox">Nona Sotoodehnia</a>, MD MPH,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Manolio%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680648448528" co-class="co-affbox">Teri A. Manolio</a>, MD PhD,<sup>21</sup> <a href="/pubmed/?term=Li%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680648446288" co-class="co-affbox">Rongling Li</a>, PhD,<sup>21</sup> <a href="/pubmed/?term=Masys%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680576502896" co-class="co-affbox">Daniel R. Masys</a>, MD,<sup>20</sup> <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680576500656" co-class="co-affbox">Jonathan L. Haines</a>, PhD,<sup>4,</sup><sup>5</sup> and  <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857" class="affpopup" co-rid="_co_idm139680568224864" co-class="co-affbox">Dan M. Roden</a>, MD<sup>3,</sup><sup>22</sup>, on behalf of the CHARGE QRS Group</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139680573709792"><h3 class="no_margin">Marylyn D. Ritchie</h3><p><sup>1</sup>Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA </p><div>Find articles by <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Marylyn D. Ritchie</a></div></div><div id="_co_idm139680577572832"><h3 class="no_margin">Joshua C. Denny</h3><p><sup>2</sup>Departments of Biomedical Informatics, Vanderbilt University School of Medicine, Nashville, TN </p><p><sup>3</sup>Departments of Medicine, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Joshua C. Denny</a></div></div><div id="_co_idm139680577569344"><h3 class="no_margin">Rebecca L. Zuvich</h3><p><sup>4</sup>Departments of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Zuvich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Rebecca L. Zuvich</a></div></div><div id="_co_idm139680578350000"><h3 class="no_margin">Dana C. Crawford</h3><p><sup>4</sup>Departments of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Nashville, TN </p><p><sup>5</sup>Departments of Center for Human Genetics Research, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Dana C. Crawford</a></div></div><div id="_co_idm139680568735280"><h3 class="no_margin">Jonathan S. Schildcrout</h3><p><sup>6</sup>Departments of Biostatistics, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Schildcrout%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Jonathan S. Schildcrout</a></div></div><div id="_co_idm139680581684720"><h3 class="no_margin">Lisa Bastarache</h3><p><sup>2</sup>Departments of Biomedical Informatics, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Bastarache%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Lisa Bastarache</a></div></div><div id="_co_idm139680581682480"><h3 class="no_margin">Andrea H. Ramirez</h3><p><sup>3</sup>Departments of Medicine, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Ramirez%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Andrea H. Ramirez</a></div></div><div id="_co_idm139680582251824"><h3 class="no_margin">Jonathan D. Mosley</h3><p><sup>3</sup>Departments of Medicine, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Mosley%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Jonathan D. Mosley</a></div></div><div id="_co_idm139680582249584"><h3 class="no_margin">Jill M. Pulley</h3><p><sup>7</sup>Departments of Office of Research, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Pulley%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Jill M. Pulley</a></div></div><div id="_co_idm139680579157616"><h3 class="no_margin">Melissa A. Basford</h3><p><sup>7</sup>Departments of Office of Research, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Basford%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Melissa A. Basford</a></div></div><div id="_co_idm139680579155376"><h3 class="no_margin">Yuki Bradford</h3><p><sup>5</sup>Departments of Center for Human Genetics Research, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Bradford%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Yuki Bradford</a></div></div><div id="_co_idm139680576445168"><h3 class="no_margin">Luke V. Rasmussen</h3><p><sup>8</sup>Essentia Institute of Rural Health, Department of Preventive Medicine, Northwestern University School of Medicine, Chicago IL </p><div>Find articles by <a href="/pubmed/?term=Rasmussen%20LV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Luke V. Rasmussen</a></div></div><div id="_co_idm139680576443184"><h3 class="no_margin">Jyotishman Pathak</h3><p><sup>9</sup>Divisions of Biomedical Informatics and Statistics, Rochester MN </p><div>Find articles by <a href="/pubmed/?term=Pathak%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Jyotishman Pathak</a></div></div><div id="_co_idm139680562922848"><h3 class="no_margin">Christopher G. Chute</h3><p><sup>9</sup>Divisions of Biomedical Informatics and Statistics, Rochester MN </p><div>Find articles by <a href="/pubmed/?term=Chute%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Christopher G. Chute</a></div></div><div id="_co_idm139680562920608"><h3 class="no_margin">Iftikhar J. Kullo</h3><p><sup>10</sup>Divisions of Cardiology, Mayo Clinic, Rochester MN </p><div>Find articles by <a href="/pubmed/?term=Kullo%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Iftikhar J. Kullo</a></div></div><div id="_co_idm139680570543488"><h3 class="no_margin">Catherine A. McCarty</h3><p><sup>11</sup>Essentia Institute of Rural Health </p><div>Find articles by <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Catherine A. McCarty</a></div></div><div id="_co_idm139680579650048"><h3 class="no_margin">Rex L. Chisholm</h3><p><sup>12</sup>Departments of Cell and Molecular Biology, Northwestern University School of Medicine, Chicago IL </p><div>Find articles by <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Rex L. Chisholm</a></div></div><div id="_co_idm139680579647808"><h3 class="no_margin">Abel N. Kho</h3><p><sup>13</sup>Departments of Medicine, Northwestern University School of Medicine, Chicago IL </p><div>Find articles by <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Abel N. Kho</a></div></div><div id="_co_idm139680575743936"><h3 class="no_margin">Christopher S. Carlson</h3><p><sup>14</sup>Departments of Fred Hutchinson Cancer Research Center, Seattle, WA </p><div>Find articles by <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Christopher S. Carlson</a></div></div><div id="_co_idm139680575741696"><h3 class="no_margin">Eric B. Larson</h3><p><sup>15</sup>Group Health Research Institute, Seattle, WA University of Washington </p><div>Find articles by <a href="/pubmed/?term=Larson%20EB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Eric B. Larson</a></div></div><div id="_co_idm139680572983248"><h3 class="no_margin">Gail P. Jarvik</h3><p><sup>16</sup>Department of Medicine (Division of Medical Genetics Health Informatics, Seattle, WA </p><p><sup>19</sup>Department of Genome Sciences, and Health Informatics, Seattle, WA </p><div>Find articles by <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Gail P. Jarvik</a></div></div><div id="_co_idm139680572980272"><h3 class="no_margin">Nona Sotoodehnia</h3><p><sup>17</sup>Department of Cardiology, and Health Informatics, Seattle, WA </p><p><sup>18</sup>Department of Cardiovascular Health Research Unit) Health Informatics, Seattle, WA </p><div>Find articles by <a href="/pubmed/?term=Sotoodehnia%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Nona Sotoodehnia</a></div></div><div id="_co_idm139680648448528"><h3 class="no_margin">Teri A. Manolio</h3><p><sup>21</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, MD </p><div>Find articles by <a href="/pubmed/?term=Manolio%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Teri A. Manolio</a></div></div><div id="_co_idm139680648446288"><h3 class="no_margin">Rongling Li</h3><p><sup>21</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, MD </p><div>Find articles by <a href="/pubmed/?term=Li%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Rongling Li</a></div></div><div id="_co_idm139680576502896"><h3 class="no_margin">Daniel R. Masys</h3><p><sup>20</sup>Department of Biomedical and Health Informatics, Seattle, WA </p><div>Find articles by <a href="/pubmed/?term=Masys%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Daniel R. Masys</a></div></div><div id="_co_idm139680576500656"><h3 class="no_margin">Jonathan L. Haines</h3><p><sup>4</sup>Departments of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Nashville, TN </p><p><sup>5</sup>Departments of Center for Human Genetics Research, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Haines%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Jonathan L. Haines</a></div></div><div id="_co_idm139680568224864"><h3 class="no_margin">Dan M. Roden</h3><p><sup>3</sup>Departments of Medicine, Vanderbilt University School of Medicine, Nashville, TN </p><p><sup>22</sup>Department of Pharmacology, Vanderbilt University School of Medicine, Nashville, TN </p><div>Find articles by <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23463857">Dan M. Roden</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139680577971824_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139680577971824_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139680577971824_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Departments of Biomedical Informatics, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Departments of Medicine, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Departments of Molecular Physiology and Biophysics, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Departments of Center for Human Genetics Research, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Departments of Biostatistics, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Departments of Office of Research, Vanderbilt University School of Medicine, Nashville, TN </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Essentia Institute of Rural Health, Department of Preventive Medicine, Northwestern University School of Medicine, Chicago IL </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Divisions of Biomedical Informatics and Statistics, Rochester MN </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Divisions of Cardiology, Mayo Clinic, Rochester MN </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Essentia Institute of Rural Health </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Departments of Cell and Molecular Biology, Northwestern University School of Medicine, Chicago IL </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Departments of Medicine, Northwestern University School of Medicine, Chicago IL </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Departments of Fred Hutchinson Cancer Research Center, Seattle, WA </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Group Health Research Institute, Seattle, WA University of Washington </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Medicine (Division of Medical Genetics Health Informatics, Seattle, WA </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Cardiology, and Health Informatics, Seattle, WA </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Cardiovascular Health Research Unit) Health Informatics, Seattle, WA </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Genome Sciences, and Health Informatics, Seattle, WA </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Department of Biomedical and Health Informatics, Seattle, WA </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>National Human Genome Research Institute, National Institutes of Health, Bethesda, MD </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Pharmacology, Vanderbilt University School of Medicine, Nashville, TN </div><div id="CR1"><strong>Correspondence</strong>: Dan M. Roden, MD Professor of Medicine and Pharmacology Director, Oates Institute for Experimental Therapeutics Assistant Vice-Chancellor for Personalized Medicine Vanderbilt University School of Medicine 1285 Medical Research Building IV Nashville, TN 37232-0575 Phone: 615-322-0067 Fax: 615-343-4522 <a href="mailto:dev@null" data-email="ude.tlibrednav@nedor.nad" class="oemail">ude.tlibrednav@nedor.nad</a></div><div id="FN1"><sup>*</sup>Contributed equally</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139680577971824_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23463857" target="pmc_ext" ref="reftype=publisher&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Circulation</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3713791/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01: </strong><strong>Supplemental Figure 1:</strong> LocusZoom plot of the region in chromosome 3 in which most of the replicated associations (18/23) were identified. All variants with p&lt;10<sup>−4</sup> in the eMERGE data set were replicated at p&lt;5×10<sup>−8</sup> in the CHARGE data set.<p>Supplemental Figure 2: Phenome-wide association study plot for <em>CDKN1A</em> (rs1321313).</p><p>Supplemental Figure 3: Phenome-wide association study plot for <em>C6orf204</em> (rs6906287).</p><p>Supplemental Figure 4: Phenome-wide association study plot for <em>NFIA</em> (rs2207790).</p><p>Supplemental Table 1: Replicated SNPs in the QRS GWAS analysis. Reported betas and pvalues for eMERGE analysis are adjusted for sex, and all betas for CHARGE and eMERGE analyses are for the coded allele.</p><p>Supplemental Table 2: PheWAS associations for replicated SNPs (N=23). Data for all associations with P&lt;0.01 are shown. ICD9 codes for given associations can be downloaded from <a href="http://knowledgemap.mc.vanderbilt.edu/research/content/phewas" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://knowledgemap.mc.vanderbilt.edu/research/content/phewas</a>.</p></div><div><a href="/pmc/articles/PMC3713791/bin/NIHMS484529-supplement-01.pdf" data-ga-action="click_feat_suppl">NIHMS484529-supplement-01.pdf</a><span style="color:gray"> (1010K)</span></div><div class="small guid">GUID:&nbsp;7CC2D49E-D5BB-42BE-B5E0-FA53927CDBDB</div></div></dd></dl></div><div id="idm139680567731296" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139680567731296title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">Electrocardiographic QRS duration, a measure of cardiac intraventricular conduction, varies ~2-fold in individuals without cardiac disease. Slow conduction may promote reentrant arrhythmias.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods and Results</h3><p id="P3" class="p p-first-last">We performed a genome-wide association study (GWAS) to identify genomic markers of QRS duration in 5,272 individuals without cardiac disease selected from electronic medical record (EMR) algorithms at five sites in the Electronic Medical Records and Genomics (eMERGE) network. The most significant loci were evaluated within the CHARGE consortium QRS GWAS meta-analysis. Twenty-three single nucleotide polymorphisms in 5 loci, previously described by CHARGE, were replicated in the eMERGE samples; 18 SNPs were in the chromosome 3 <em>SCN5A</em> and <em>SCN10A</em> loci, where the most significant SNPs were rs1805126 in <em>SCN5A</em> with p=1.2×10<sup>−8</sup> (eMERGE) and p=2.5×10<sup>−20</sup> (CHARGE) and rs6795970 in <em>SCN10A</em> with p=6×10<sup>−6</sup> (eMERGE) and p=5×10<sup>−27</sup> (CHARGE). The other loci were in <em>NFIA</em>, near <em>CDKN1A</em>, and near <em>C6orf204.</em> We then performed phenome-wide association studies (PheWAS) on variants in these five loci in 13,859 European Americans to search for diagnoses associated with these markers. PheWAS identified atrial fibrillation and cardiac arrhythmias as the most common associated diagnoses with <em>SCN10A</em> and <em>SCN5A</em> variants. <em>SCN10A</em> variants were also associated with subsequent development of atrial fibrillation and arrhythmia in the original 5,272 “heart-healthy” study population.</p></div><div id="S3" class="sec sec-last"><h3 id="S3title">Conclusions</h3><p id="P4" class="p p-first-last">We conclude that DNA biobanks coupled to EMRs provide a platform not only for GWAS but may also allow broad interrogation of the longitudinal incidence of disease associated with genetic variants. The PheWAS approach implicated sodium channel variants modulating QRS duration in subjects without cardiac disease as predictors of subsequent arrhythmias.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">cardiac conduction, QRS duration, atrial fibrillation, genome-wide association study, phenome-wide association study, electronic medical records</span></div></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Introduction</h2><p id="P5" class="p p-first">Electrocardiographic (ECG) parameters of cardiac conduction and repolarization, PR, QRS, and QT intervals, are widely used in clinical medicine and display substantial variability when measured across large populations. QRS duration represents activation time in the cardiac ventricle, and prolongation of this interval – representing global or regional slow conduction – has been associated with adverse outcomes such as sudden cardiac death.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373221">1</a></sup> This variability reflects modulators such as abnormal electrolytes, underlying heart disease, or concomitant drug therapy, as well as heritable components; published estimates suggest that up to 40% of variability in the QRS interval is heritable.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373261">2</a>–<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373274">4</a></sup> Using automated methods described further below, we have shown that 99% of QRS durations in over 30,000 normal subjects not receiving confounding medications fall between 65 to 108 msec.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373225">5</a></sup></p><p id="P6">DNA repositories linked to Electronic Medical Record (EMR) systems have been proposed as one potential source of subjects for analyzing the relationship between genetic variation and a range of human traits.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_761438697">6</a>–<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373265">10</a></sup> The advantages of this approach may include rapid generation of patient sets for study (since electronic data are already in place), and the ability to study large numbers of subjects accrued without bias with respect to factors such as disease or age. The National Human Genome Research Institute’s electronic MEdical Records and GEnomics (eMERGE) Network<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373233">11</a></sup> has, as one of its primary goals, the evaluation of the utility of EMR systems coupled to DNA repositories as a tool for genome science. Initial studies from eMERGE sites support the potential utility of EMR systems for discovery and validation of genotype-phenotype associations.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373248">12</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373259">16</a></sup></p><p id="P7" class="p p-last">We report here a GWAS of QRS duration in European-descent subjects whose first ECG in an EMR system was normal, and who at the time of the ECG lacked evidence of heart disease, potentially confounding medications, and electrolyte abnormalities. We then used a phenome-wide association study (PheWAS)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373263">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373214">17</a></sup> to demonstrate that polymorphisms associated with QRS variability in subjects without cardiac disease were also associated with subsequent diagnoses of cardiac arrhythmias. This coupling of GWAS and PheWAS further validates the relationship between abnormal conduction and arrhythmias, and points to the development of genome-based predictors of arrhythmia susceptibility.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Normal QRS algorithm</h3><p id="P8" class="p p-first">We developed and deployed an algorithm to identify individuals with normal ECGs and without any cardiac disease, abnormal electrolyte values, or QRS-active medications – across the five eMERGE-1 sites identified 5,272 Caucasian patients (2,488 males and 2,784 females; <a href="/pmc/articles/PMC3713791/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The algorithm was developed and validated in the Synthetic Derivative (SD), a de-identified image of the Vanderbilt EMR that currently contains over 120 million documents on about 2 million patients.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358524285">7</a></sup> The SD is refreshed regularly to add new clinical information from the EMR as it is accrued.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">eMERGE sites contributing samples. All patients were of European ancestry.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680568240816" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Site</th><th rowspan="1" colspan="1" valign="top" align="center">Primary site<br>phenotype</th><th rowspan="1" colspan="1" valign="top" align="center">Mean Age<br>(SD)</th><th colspan="3" rowspan="1" valign="top" align="center">Genotyped samples meeting normal QRS<br>algorithm definition</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center">Males</th><th rowspan="1" colspan="1" valign="top" align="center">Females</th><th rowspan="1" colspan="1" valign="top" align="center">Total</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Group Health</td><td rowspan="1" colspan="1" valign="top" align="center">Dementia</td><td rowspan="1" colspan="1" valign="top" align="center">72.58 (7.23)</td><td rowspan="1" colspan="1" valign="top" align="center">187</td><td rowspan="1" colspan="1" valign="top" align="center">351</td><td rowspan="1" colspan="1" valign="top" align="center">538</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Marshfield</td><td rowspan="1" colspan="1" valign="top" align="center">Cataract</td><td rowspan="1" colspan="1" valign="top" align="center">56.73 (9.73)</td><td rowspan="1" colspan="1" valign="top" align="center">69</td><td rowspan="1" colspan="1" valign="top" align="center">149</td><td rowspan="1" colspan="1" valign="top" align="center">218</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Mayo Clinic</td><td rowspan="1" colspan="1" valign="top" align="center">Peripheral<br>artery disease</td><td rowspan="1" colspan="1" valign="top" align="center">58.81 (8.52)</td><td rowspan="1" colspan="1" valign="top" align="center">1056</td><td rowspan="1" colspan="1" valign="top" align="center">730</td><td rowspan="1" colspan="1" valign="top" align="center">1786</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Northwestern</td><td rowspan="1" colspan="1" valign="top" align="center">Type 2<br>diabetes</td><td rowspan="1" colspan="1" valign="top" align="center">52.49 (10.54)</td><td rowspan="1" colspan="1" valign="top" align="center">99</td><td rowspan="1" colspan="1" valign="top" align="center">118</td><td rowspan="1" colspan="1" valign="top" align="center">217</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Vanderbilt</td><td rowspan="1" colspan="1" valign="top" align="center">Normal QRS</td><td rowspan="1" colspan="1" valign="top" align="center">49.95 (14.6)</td><td rowspan="1" colspan="1" valign="top" align="center">1077</td><td rowspan="1" colspan="1" valign="top" align="center">1436</td><td rowspan="1" colspan="1" valign="top" align="center">2513</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Total</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>2488</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>2784</strong>
</td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>5272</strong>
</td></tr></tbody></table></div><div id="largeobj_idm139680568240816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3713791/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P9">The study population consisted of subjects with a normal ECG without evidence of cardiac disease any time before or within one month following the ECG, concurrent use of medications that interfere with ventricular conduction, and who did not have abnormal potassium, calcium, or magnesium lab values at the time of the ECG. The algorithm has been described in detail previously.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373262">13</a></sup> The algorithm used natural language processing (NLP)<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373226">18</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373238">19</a></sup> to analyze narrative text, billing code queries, and lab queries to exclude any subjects with evidence of arrhythmia, heart failure, cardiomyopathy, myocardial ischemia/infarct, or cardiac conduction defect. The algorithm considered all physician-generated clinical documentation, including clinical notes and cardiologist-generated ECG impressions. Patients with family histories of cardiac disease were allowed by the NLP algorithm. In addition, ECGs had normal Bazett’s corrected QT intervals (&lt;450ms), heart rates (between 50-100 bpm), and QRS (60-120 ms). The algorithm was reviewed by two physicians not involved in algorithm development, and achieved a PPV of 97% to identify patients with normal ECGs who did not have known exclusions on a random selection of 100 subjects.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373247">13</a></sup> Analysis of clinical covariates in ~30,000 records with algorithm-defined normal ECGs identified gender and ancestry as modulators of QRS duration.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373228">5</a></sup> Complete details of the algorithm are available from PheKB (<a href="http://phekb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://phekb.org/</a>).</p><p id="P10">The final algorithm was applied in BioVU, the Vanderbilt DNA databank that links DNA extracted from discarded blood samples to the SD.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358524286">7</a></sup> Patients at Vanderbilt were genotyped specifically for the purpose of studying normal QRS duration. The algorithm was then deployed across the DNA repositories at the other four eMERGE-I sites (Marshfield Clinic, Northwestern University, Mayo Clinic, Group Health Research Institute) to identify subjects with extant eMERGE-based genotyping data (based on other phenotypes; as shown in <a href="/pmc/articles/PMC3713791/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) who met algorithm-defined criteria for normal QRS. The eMERGE cohorts are described in more detail in McCarty et al. 2011<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373212">11</a></sup> and at the Phenotype Knowledge Base (PheKB.org). Thus, all eMERGE individuals used in the analysis underwent the same algorithm to select those with normal ECGs and without prior heart disease, interfering medications, and abnormal electrolytes. To assess the performance of the algorithm when applied within external EMR systems, trained chart abstracters at Northwestern and Marshfield reviewed randomly-selected subsets of 100 subjects at Marshfield and 45 subjects at Northwestern to determine the algorithm’s accuracy at external sites. Northwestern’s evaluation also included an independent review by a board-certified internal medicine physician, with discrepancies resolved by consensus. This study included only subjects designated as “non-Hispanic white” European American in the EMR from each site. We have previously shown the EMR ancestry performs similar to self-report.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373241">20</a></sup></p><p id="P11" class="p p-last">This study was approved by each site’s Institutional Review Board. Because BioVU is de-identified and accrues individuals through left-over blood remaining after routine clinical testing, it operates as non-human subjects research according to the provisions of 45 CFR 46, as described previously.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358524287">7</a></sup> Individuals at other eMERGE sites were consented as part of each site’s DNA biobank.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373216">11</a></sup></p></div><div id="S7" class="sec"><h3 id="S7title">Genotyping and data analysis</h3><p id="P12" class="p p-first">Genotyping was performed at the Center for Genotyping and Analysis at the Broad Institute and the Center for Inherited Disease Research (CIDR) at Johns Hopkins University. Samples of European ancestry or unknown ancestry were analyzed using the Illumina Human660W-Quadv1_A genotyping platform, consisting of 561,490 SNPs and 95,876 intensity-only probes. Data were cleaned using the quality control (QC) pipeline developed by the eMERGE Genomics Working Group.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373243">21</a></sup> This process includes evaluation of sample and marker call rate, gender mismatch and anomalies, duplicate and HapMap concordance, batch effects, Hardy-Weinberg equilibrium (HWE), sample relatedness, and population stratification. After QC, 528,508 SNPs were used for analysis based on the following QC criteria: SNP call rate &gt;99%, sample call rate &gt;99%, minor allele frequency &gt; 0.0001, unrelated samples only (removing all parent-offspring, full and half siblings), and individuals of European-descent only (based on STRUCTURE<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373258">22</a></sup> analysis of &gt;90% probability of being in the CEU cluster).</p><p id="P13">Each eMERGE site used the QC pipeline to clean their initial datasets prior to merging all the samples. QC procedures were then performed on the merged eMERGE dataset in which data from all five sites were combined, and no significant differences across sites or genotyping center were identified. As well, all sites had comparable QC results including similar SNP and sample call rates, HWE p-values overall, and minor allele frequencies. The detailed QC report on the merged dataset will be deposited in dbGaP along with the merged dataset.</p><p id="P14">Single-locus tests of association were performed using linear regression assuming an additive genetic model for all 528,508 SNPs in a total of 5,272 individuals with a normal QRS duration. Our studies of ECG intervals in 32,949 normal individuals identified sex as a major modulator of normal QRS duration, with minor effects of age and ancestry.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373237">5</a></sup> All analyses were performed unadjusted and then adjusted for age, sex, BMI, and the first principal component from Eigenstrat<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373251">23</a></sup> to adjust for potential population stratification, without significantly changing the key results. Since only sex is significantly associated with QRS duration via the literature, we report that here. Analyses were also performed adjusting for height and/or BMI, but these did not change the results.</p><p id="P15">Associations with the lowest P values (p&lt;10<sup>−4</sup>) were then submitted to the recent QRS Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) meta-analysis group<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373232">24</a></sup> and a table of P-values from that analysis was generated. The CHARGE meta-analysis of QRS duration has been described in detail previously<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373246">24</a></sup>; briefly, it involved 40,407 individuals selected from 15 sites restricted to those of European ancestry. Individuals with prior myocardial infarction, heart failure, arrhythmias, pacemakers, antiarrhythmic medication use, or whose QRS durations &gt;120 ms were excluded.</p><p id="P16" class="p p-last">To calculate the variance explained by all SNPs in the dataset, we analyzed the data using GCTA.<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373277">25</a></sup> Only those SNPs with minor allele frequency &gt; 0.01, genotyping efficiency &gt; 99.9 and HWE &gt; 0.001 were included in the analysis (n=505,502 SNPs). The genetic relationship matrix (GRM) was computed for all 5272 subjects and all SNPs using GCTA. In order to eliminate possible cryptic relationships, subjects with GRM&gt;0.25 were pruned from the analysis, which removed 310 subjects. The proportion of variance explained by either all SNPs or all SNPs excluding the subset of 23 SNPs significant in the CHARGE GWAS was computed on the remaining subjects for QRS duration. We compared this to a linear regression analysis using the five SNPs in <a href="/pmc/articles/PMC3713791/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> to estimate the proportion of variance explained by these loci. All analyses were adjusted for age, gender and the first principal component (previously computed).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p7">Replicated SNPs in the QRS GWAS analysis. Reported betas and p-values for eMERGE analysis are adjusted for sex, and all betas for CHARGE and eMERGE analyses are for the coded allele. Each SNP represents the strongest association in the region, and was the target of a PheWAS (as shown in <a href="/pmc/articles/PMC3713791/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680580935152" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="4" rowspan="1" valign="top" align="center">eMERGE QRS</th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th rowspan="1" colspan="1" valign="top" align="center"></th><th colspan="2" rowspan="1" valign="top" align="center">CHARGE QRS</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="center">CHR</th><th rowspan="1" colspan="1" valign="top" align="center">SNP</th><th rowspan="1" colspan="1" valign="top" align="center">Location</th><th rowspan="1" colspan="1" valign="top" align="center">Coded<br>Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Coded<br>Allele<br>Frequency</th><th rowspan="1" colspan="1" valign="top" align="center">BETA<br>(msec)</th><th rowspan="1" colspan="1" valign="top" align="center">p-value</th><th rowspan="1" colspan="1" valign="top" align="center">Coded<br>Allele</th><th rowspan="1" colspan="1" valign="top" align="center">Coded<br>Allele<br>Frequency</th><th rowspan="1" colspan="1" valign="top" align="center">BETA<br>(msec)</th><th rowspan="1" colspan="1" valign="top" align="center">p-value</th><th rowspan="1" colspan="1" valign="top" align="center">Nearest<br>Gene</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs1805126</td><td rowspan="1" colspan="1" valign="top" align="center">38567410</td><td rowspan="1" colspan="1" valign="top" align="center">T<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.665</td><td rowspan="1" colspan="1" valign="top" align="center">−1.002</td><td rowspan="1" colspan="1" valign="top" align="center">1.45E-08</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.655</td><td rowspan="1" colspan="1" valign="top" align="center">−0.6568</td><td rowspan="1" colspan="1" valign="top" align="center">2.52E-20</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SCN5A</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">rs6795970</td><td rowspan="1" colspan="1" valign="top" align="center">38741679</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.401</td><td rowspan="1" colspan="1" valign="top" align="center">0.765</td><td rowspan="1" colspan="1" valign="top" align="center">6.60E-06</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.396</td><td rowspan="1" colspan="1" valign="top" align="center">0.7476</td><td rowspan="1" colspan="1" valign="top" align="center">5.08E-27</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>SCN10A</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs6906287</td><td rowspan="1" colspan="1" valign="top" align="center">119069433</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.454</td><td rowspan="1" colspan="1" valign="top" align="center">0.717</td><td rowspan="1" colspan="1" valign="top" align="center">2.26E-05</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.451</td><td rowspan="1" colspan="1" valign="top" align="center">0.5383</td><td rowspan="1" colspan="1" valign="top" align="center">5.56E-16</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>C6orf204</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">rs1321313</td><td rowspan="1" colspan="1" valign="top" align="center">36726799</td><td rowspan="1" colspan="1" valign="top" align="center">G<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.760</td><td rowspan="1" colspan="1" valign="top" align="center">−0.793</td><td rowspan="1" colspan="1" valign="top" align="center">6.13E-05</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.742</td><td rowspan="1" colspan="1" valign="top" align="center">−0.8129</td><td rowspan="1" colspan="1" valign="top" align="center">4.60E-25</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>CDKN1A</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="center">rs2207790</td><td rowspan="1" colspan="1" valign="top" align="center">61670555</td><td rowspan="1" colspan="1" valign="top" align="center">T<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.482</td><td rowspan="1" colspan="1" valign="top" align="center">−0.622</td><td rowspan="1" colspan="1" valign="top" align="center">2.07E-04</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.461</td><td rowspan="1" colspan="1" valign="top" align="center">−0.5956</td><td rowspan="1" colspan="1" valign="top" align="center">6.31E-18</td><td rowspan="1" colspan="1" valign="top" align="center">
<em>NFIA</em>
</td></tr></tbody></table></div><div id="largeobj_idm139680580935152" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3713791/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>This SNP was coded on the opposite strand from that in the CHARGE study. Therefore, while the allele is not identical, the direction of effect is the same</div></div></div></div><div id="S8" class="sec"><h3 id="S8title">Phenome-wide association study of QRS-associated SNPs</h3><p id="P17" class="p p-first">We selected the most significant SNP associations for analysis by PheWAS.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373256">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373253">17</a></sup> For this analysis, we combined the entire eMERGE cohort of European American individuals (n=13,859) identified across the five eMERGE sites. These individuals represent a superset of the 5,272 individuals with normal ECGs and without heart disease used for the GWAS. To define diseases, we queried all International Classification of Disease (ICD), 9<sup>th</sup> edition, codes from the respective EMRs of the five eMERGE sites.</p><p id="P18" class="p p-last">The PheWAS software uses occurrences of ICD codes to classify each person as having one or more of 778 possible clinical phenotypes (typically diseases). For each disease, the PheWAS algorithm constructs a control population by selecting all patients that do not have the case disease or closely related diseases (e.g., a patient with a bundle branch block cannot serve as a control for complete heart block). The PheWAS methodology has previously been validated through rediscovery of known associations.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373242">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373276">17</a></sup> Analysis of each phenotype then proceeds using a pairwise analysis of all case and control groups for each tested SNP (n=23). We have observed that positive predictive values increase when individual codes are present more than once in the EMR, and here we required each case to have at least four instances of the same ICD code in a PheWAS case group. In addition, we did not analyze phenotypes occurring in less than 50 patients (a prevalence of 0.36% in the dataset). Association analyses were performed with PLINK using logistic regression adjusted for age, gender, and the first three principal component analyses as calculated by Eigenstrat, since on this larger population, the third principal component was statistically significant.<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373230">23</a></sup> Analysis adjusted with and without principal components did not substantively change the results. After identification of PheWAS case and control groups using the PheWAS software, the association analyses were performed using PLINK.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373267">26</a></sup></p></div><div id="S9" class="sec sec-last"><h3 id="S9title">Survival analysis of QRS population</h3><p id="P19" class="p p-first-last">Following PheWAS analysis, we analyzed the original set of 5272 patients that met our algorithm definition for normal cardiac conduction/normal heart for subsequent development of atrial fibrillation and cardiac arrhythmias with the <em>SCN5A</em> rs1805126 and <em>SCN10A</em> rs6795970 SNPs. Phenotype definitions were drawn from the PheWAS analysis using billing codes. Kaplan-Meier analysis and Cox proportional hazard models were calculated, using the starting time as the initial normal ECG with a time-to-event analysis. Cox proportional hazard models were adjusted for age, sex, principal components as calculated above, and QRS duration.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Results</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Population identification</h3><p id="P20" class="p p-first-last">We identified 5,272 Caucasian patients (2,488 males and 2,784 females; <a href="/pmc/articles/PMC3713791/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) across the five eMERGE-I sites. The positive predictive value (PPV) of the automated phenotype algorithm to find cases with normal ECGs and without exclusions at the development site, Vanderbilt, to identify study subjects was 97% (95% confidence interval [CI] 91-99%).<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373250">13</a></sup> The PPV at Northwestern University and Marshfield Clinic were 97% (95% CI 83%-100%) and 100% (95% CI 96%-100%), respectively. Combining all reviewed samples across the three sites, the PPV would be 98% (95% CI 96%-100%). The mean QRS duration was 87.9 msec (standard deviation 9.5 msec; median 88.0 msec; <a href="/pmc/articles/PMC3713791/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3713791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680611943792" class="figure"><a href="/pmc/articles/PMC3713791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3713791_nihms-484529-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-484529-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3713791/bin/nihms-484529-f0001.jpg"></a></div><div id="largeobj_idm139680611943792" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3713791/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3713791/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p3">Panel A. Distribution of QRS durations in 5,272 normal ECGs. Panel B. Genome-wide association analysis of QRS duration, using sex-adjusted linear regression. The red line indicates genome-wide significance (p=5×10<sup>−8</sup>). The points in green are those SNPs that were also identified at genome-wide significance in the CHARGE consortium QRS meta-analysis as described in the text.</p></div></div></div></div><div id="S12" class="sec"><h3 id="S12title">GWAS results</h3><p id="P21" class="p p-first">A total of 528,508 SNPs passed quality control of eMERGE-supported Illumina 660Quad genotyping data in these subjects. <a href="/pmc/articles/PMC3713791/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a> shows the genome-wide association analysis for QRS duration adjusted for sex; the findings were near-identical for the unadjusted analysis. There was a single association between QRS duration and a SNP (rs1805126) in <em>SCN5A</em>, encoding the cardiac sodium channel gene, that survived Bonferroni correction (beta=1.002 msec per copy of the T allele, p=1.45 × 10<sup>−8</sup>).</p><p id="P22">The set taken forward to the CHARGE QRS meta-analysis consortium included 108 SNPs with P-values &lt;10<sup>−4</sup>. The retrieved P-values for this set divided into two distinct groups: 23 SNPs with P-values in the CHARGE set from 10<sup>−8</sup> to 10<sup>−27</sup>, and 85 with P-values &gt; 0.003. These 23 associations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) are located in the five loci with the lowest P values reported by the CHARGE consortium: 18/23 are in the chromosome 3 locus that includes <em>SCN5A</em> and <em>SCN10A</em>, as well as other genes (e.g. <em>EXOG</em> and <em>XYLB1</em>). The other three loci are near <em>SLC35F1</em> and <em>C6orf204</em> (chromosome 6), near <em>CDKN1A</em> (chromosome 6) and in <em>NFIA</em> (chromosome 1). The most significant SNP for each locus is presented in <a href="/pmc/articles/PMC3713791/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The locus zoom plot (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>) shows little linkage disequilibrium (LD) in the chromosome 3 region in HapMap Phase III (CEU), consistent with the suggestion that the <em>SCN5A-10A</em> finding may actually indicate multiple independent associations.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373224">24</a></sup> Specifically, the most significant variants in <em>SCN5A</em> (rs1805126) and <em>SCN10A</em> (rs6795970) are not in LD (r<sup>2</sup>&lt;0.20).</p><p id="P23" class="p p-last">Using the GTCA<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373269">25</a></sup> approach, we estimated heritability for QRS at 31.1% (standard error [SE] 6.9%, p=5.7 × 10<sup>−7</sup>) using all SNPs in the dataset. Conducting the analysis without the 23 SNPs significant in CHARGE decreased the estimated heritability to 30.3%, a decrease of 0.8%. This was somewhat conservative compared to a linear regression model, which estimated an adjusted r-square value of 1.6% for the five loci in <a href="/pmc/articles/PMC3713791/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p></div><div id="S13" class="sec"><h3 id="S13title">PheWAS analysis</h3><p id="P24" class="p p-first-last">The PheWAS dataset consisted of 13,859 European-American subjects in the entire genotyped eMERGE cohort. The analysis focused on the most significant SNPs in each of the five loci associated with QRS (<a href="/pmc/articles/PMC3713791/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). While no associations survived a strict Bonferroni correction for significance (p=0.05/778/5=1.3×10<sup>−5</sup>), the most significant associations were particularly relevant to cardiac disease and demonstrated significantly different patterns of associations for the five QRS-associated loci. The strongest associations for the SNPs in both <em>SCN5A</em> and <em>SCN10A</em> were with the diagnoses of cardiac arrhythmias (p=7.21×10<sup>−4</sup> for <em>SCN10A</em> and p=1.1×10<sup>−3</sup> for <em>SCN5A</em>) and, for <em>SCN10A</em>, atrial fibrillation (p=8.5×10<sup>−4</sup>, <a href="/pmc/articles/PMC3713791/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). <a href="/pmc/articles/PMC3713791/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> lists associations for the most significant <em>SCN5A</em> (rs1805126) and <em>SCN10A</em> (rs6795970) SNPs, as well as those at the other QRS-associated loci, chromosome 1 (rs2207790) and the two chromosome 6 loci (rs6906287 and rs1321313), also graphed in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 2</a>-<a href="#SD1" rid="SD1" class=" supplementary-material">4</a>. The <em>CDKN1</em> and <em>C6orf204</em> loci were not associated with cardiac arrhythmias (p&gt;0.3, with 80% power to detect and OR&gt;1.12 at p=0.05), and <em>NFIA</em> was weakly associated with cardiac arrhythmias (OR=0.91, p=0.02). <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> presents PheWAS association data for all 23 SNPs significant in CHARGE. While most SNPs in a given gene displayed similar patterns of PheWAS associations, rs11129801, the strongest <em>SCN10A</em> SNP in our adjusted analysis but a lesser association in CHARGE, had a very different PheWAS pattern, with the strongest associations being epilepsy, uterine cancer, and migraines; atrial fibrillation was not associated (OR=1.07, p=0.18). In agreement with these data, rs11129801 was only in weak linkage disequilibrium to the other <em>SCN10A</em> SNPs, such as rs6800541 (r<sup>2</sup>=0.20). Likewise, the <em>EXOG</em> locus had a very different PheWAS pattern of associations (prostatic hyperplasia, sexual and gender identity disorders, liver disease, kidney disease, cerebral degenerations, diarrhea) despite being nearby the <em>SCN5A-10A</em> region.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3713791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680570193872" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-484529-f0002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-484529-f0002.jpg" src="/pmc/articles/PMC3713791/bin/nihms-484529-f0002.jpg"></div></a><div id="largeobj_idm139680570193872" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3713791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3713791/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3713791/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p4">Phenome-wide association study plots of most significant SNPs associated with QRS duration. Panel A. <em>SCN5A</em> (rs1805126). Panel B. <em>SCN10A</em> (rs6795970). The blue lines indicate p&lt;0.05.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p8">Most significant associations between SNPs predicting normal QRS duration and diagnostic codes. For each SNP displayed, all PheWAS associations p&lt;0.01 are displayed.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680570732496" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Associated Phenotype</th><th rowspan="1" colspan="1" valign="middle" align="left">Case count</th><th rowspan="1" colspan="1" valign="middle" align="left">Odds ratio</th><th rowspan="1" colspan="1" valign="middle" align="left">P</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"><em>SCN5A</em> (Chr 3, rs1805126)</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cardiac arrhythmias</td><td rowspan="1" colspan="1" valign="middle" align="left">3075</td><td rowspan="1" colspan="1" valign="middle" align="left">0.877</td><td rowspan="1" colspan="1" valign="middle" align="left">1.10E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Other diseases of upper respiratory tract</td><td rowspan="1" colspan="1" valign="middle" align="left">509</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8131</td><td rowspan="1" colspan="1" valign="middle" align="left">3.83E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Benign prostatic hypertrophy</td><td rowspan="1" colspan="1" valign="middle" align="left">1615</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8527</td><td rowspan="1" colspan="1" valign="middle" align="left">5.54E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Chronic kidney disease</td><td rowspan="1" colspan="1" valign="middle" align="left">1212</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8772</td><td rowspan="1" colspan="1" valign="middle" align="left">6.02E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Melanoma</td><td rowspan="1" colspan="1" valign="middle" align="left">151</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7033</td><td rowspan="1" colspan="1" valign="middle" align="left">8.18E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Dermatophytosis</td><td rowspan="1" colspan="1" valign="middle" align="left">1229</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8791</td><td rowspan="1" colspan="1" valign="middle" align="left">8.77E-03</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>SCN10A</em> (Chr 3, rs6795970)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cardiac arrhythmias</td><td rowspan="1" colspan="1" valign="middle" align="left">3075</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8781</td><td rowspan="1" colspan="1" valign="middle" align="left">7.21E-04</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Atrial fibrillation and flutter</td><td rowspan="1" colspan="1" valign="middle" align="left">1758</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8519</td><td rowspan="1" colspan="1" valign="middle" align="left">8.45E-04</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Arterial embolism and thrombosis</td><td rowspan="1" colspan="1" valign="middle" align="left">150</td><td rowspan="1" colspan="1" valign="middle" align="left">0.6928</td><td rowspan="1" colspan="1" valign="middle" align="left">3.20E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cholecystitis</td><td rowspan="1" colspan="1" valign="middle" align="left">121</td><td rowspan="1" colspan="1" valign="middle" align="left">0.6627</td><td rowspan="1" colspan="1" valign="middle" align="left">3.44E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Convulsions</td><td rowspan="1" colspan="1" valign="middle" align="left">311</td><td rowspan="1" colspan="1" valign="middle" align="left">1.249</td><td rowspan="1" colspan="1" valign="middle" align="left">7.01E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Aphasia</td><td rowspan="1" colspan="1" valign="middle" align="left">69</td><td rowspan="1" colspan="1" valign="middle" align="left">0.5999</td><td rowspan="1" colspan="1" valign="middle" align="left">7.25E-03</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>C6orf204</em> (Chr 6, rs6906287)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Colorectal cancer</td><td rowspan="1" colspan="1" valign="middle" align="left">316</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7592</td><td rowspan="1" colspan="1" valign="middle" align="left">9.90E-04</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Benign prostatic hypertrophy</td><td rowspan="1" colspan="1" valign="middle" align="left">1615</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8576</td><td rowspan="1" colspan="1" valign="middle" align="left">4.19E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Penicillin allergy</td><td rowspan="1" colspan="1" valign="middle" align="left">65</td><td rowspan="1" colspan="1" valign="middle" align="left">1.68</td><td rowspan="1" colspan="1" valign="middle" align="left">4.01E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Melanoma</td><td rowspan="1" colspan="1" valign="middle" align="left">151</td><td rowspan="1" colspan="1" valign="middle" align="left">1.383</td><td rowspan="1" colspan="1" valign="middle" align="left">5.68E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Disorders of pancreatic secretion</td><td rowspan="1" colspan="1" valign="middle" align="left">84</td><td rowspan="1" colspan="1" valign="middle" align="left">0.6421</td><td rowspan="1" colspan="1" valign="middle" align="left">6.22E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Nasal polyps</td><td rowspan="1" colspan="1" valign="middle" align="left">127</td><td rowspan="1" colspan="1" valign="middle" align="left">1.413</td><td rowspan="1" colspan="1" valign="middle" align="left">7.15E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Prostatitis</td><td rowspan="1" colspan="1" valign="middle" align="left">150</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7166</td><td rowspan="1" colspan="1" valign="middle" align="left">8.07E-03</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>NFIA</em> (Chr 1, rs2207790)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Postinflammatory pulmonary fibrosis</td><td rowspan="1" colspan="1" valign="middle" align="left">173</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7048</td><td rowspan="1" colspan="1" valign="middle" align="left">1.76E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Multiple myeloma</td><td rowspan="1" colspan="1" valign="middle" align="left">54</td><td rowspan="1" colspan="1" valign="middle" align="left">0.5425</td><td rowspan="1" colspan="1" valign="middle" align="left">2.80E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Neutropenia and leukopenia</td><td rowspan="1" colspan="1" valign="middle" align="left">238</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7545</td><td rowspan="1" colspan="1" valign="middle" align="left">2.81E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Facial nerve disorders</td><td rowspan="1" colspan="1" valign="middle" align="left">67</td><td rowspan="1" colspan="1" valign="middle" align="left">1.677</td><td rowspan="1" colspan="1" valign="middle" align="left">3.70E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cardiomyopathy</td><td rowspan="1" colspan="1" valign="middle" align="left">420</td><td rowspan="1" colspan="1" valign="middle" align="left">0.8033</td><td rowspan="1" colspan="1" valign="middle" align="left">4.40E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Pneumothorax</td><td rowspan="1" colspan="1" valign="middle" align="left">80</td><td rowspan="1" colspan="1" valign="middle" align="left">0.6391</td><td rowspan="1" colspan="1" valign="middle" align="left">6.59E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Disorders of function of stomach</td><td rowspan="1" colspan="1" valign="middle" align="left">355</td><td rowspan="1" colspan="1" valign="middle" align="left">1.23</td><td rowspan="1" colspan="1" valign="middle" align="left">7.73E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Prostatitis</td><td rowspan="1" colspan="1" valign="middle" align="left">150</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7239</td><td rowspan="1" colspan="1" valign="middle" align="left">8.85E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Toxic diffuse goiter</td><td rowspan="1" colspan="1" valign="middle" align="left">76</td><td rowspan="1" colspan="1" valign="middle" align="left">1.541</td><td rowspan="1" colspan="1" valign="middle" align="left">9.47E-03</td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"><em>CDKN1A</em> (Chr 6, rs1321313)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td colspan="4" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Anorexia</td><td rowspan="1" colspan="1" valign="middle" align="left">53</td><td rowspan="1" colspan="1" valign="middle" align="left">0.3649</td><td rowspan="1" colspan="1" valign="middle" align="left">1.57E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Bacteremia</td><td rowspan="1" colspan="1" valign="middle" align="left">118</td><td rowspan="1" colspan="1" valign="middle" align="left">0.5684</td><td rowspan="1" colspan="1" valign="middle" align="left">2.01E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Anal fissure and fistula</td><td rowspan="1" colspan="1" valign="middle" align="left">108</td><td rowspan="1" colspan="1" valign="middle" align="left">1.552</td><td rowspan="1" colspan="1" valign="middle" align="left">3.04E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Abnormal loss of weight and underweight</td><td rowspan="1" colspan="1" valign="middle" align="left">398</td><td rowspan="1" colspan="1" valign="middle" align="left">0.7642</td><td rowspan="1" colspan="1" valign="middle" align="left">3.39E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Dementias</td><td rowspan="1" colspan="1" valign="middle" align="left">841</td><td rowspan="1" colspan="1" valign="middle" align="left">0.833</td><td rowspan="1" colspan="1" valign="middle" align="left">5.69E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Overweight, obesity and other</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">hyperalimentation</td><td rowspan="1" colspan="1" valign="middle" align="left">2501</td><td rowspan="1" colspan="1" valign="middle" align="left">1.105</td><td rowspan="1" colspan="1" valign="middle" align="left">7.39E-03</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Spondylosis and allied disorders</td><td rowspan="1" colspan="1" valign="middle" align="left">1244</td><td rowspan="1" colspan="1" valign="middle" align="left">1.156</td><td rowspan="1" colspan="1" valign="middle" align="left">9.39E-03</td></tr></tbody></table></div><div id="largeobj_idm139680570732496" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3713791/table/T3/?report=objectonly">Open in a separate window</a></div></div></div><div id="S14" class="sec sec-last"><h3 id="S14title">Analysis of QRS population for arrhythmias</h3><p id="P25" class="p p-first-last">After PheWAS analysis, we analyzed the original set of 5,272 patients that met our algorithm definition for normal cardiac conduction/normal heart for subsequent development of atrial fibrillation and cardiac arrhythmias with the <em>SCN5A</em> rs1805126 and <em>SCN10A</em> rs6795970. In this population, 173 (3%) developed atrial fibrillation or atrial flutter at some point at least one month following the normal ECG, and 605 (11%) were coded as having any arrhythmia. QRS duration itself was associated with future development of atrial fibrillation (p=0.015) by logistic regression. As with the eMERGE PheWAS, <em>SCN10A</em> rs6795970 was associated with both arrhythmias (hazard ratio [HR]=0.81 per copy of the A allele, p=0.002) and atrial fibrillation/flutter (HR=0.67 per copy of the A allele, p=0.001). Moreover, this association was essentially unchanged when QRS was also included in the model (HR=0.68), indicating that the association between rs6795970 and atrial fibrillation is independent of the association between QRS and rs6795970. Similarly, the association between rs6795970 and cardiac arrhythmias was independent of QRS (HR=0.80 without QRS). Our analysis did not demonstrate an association between <em>SCN5A</em> rs1805126 and either atrial fibrillation (HR=1.2, p=0.14) or cardiac arrhythmias (1.03, p=0.66) in the normal QRS population. <a href="/pmc/articles/PMC3713791/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> presents a Kaplan-Meier plot for the relationship between rs6795970 and development of atrial fibrillation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3713791/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139680570190528" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms-484529-f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms-484529-f0003.jpg" src="/pmc/articles/PMC3713791/bin/nihms-484529-f0003.jpg"></div></a><div id="largeobj_idm139680570190528" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3713791/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC3713791/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3713791/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p5">Kaplan-Meier estimate of development Atrial Fibrillation per <em>SCN10A</em> rs6795970 genotype. All patients were judged free of cardiac disease (including arrhythmias) at the time of the initial normal ECG.</p></div></div></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Discussion</h2><p id="P26" class="p p-first">The current study demonstrates common variants in the <em>SCN5A-SCN10A</em> locus are associated with QRS duration in subjects without clinical evidence of prior heart disease. These patients were derived from clinical practice settings, adding to the growing body of evidence of supporting the utility of EMR-based genomic analysis.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373245">12</a>–<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373213">16</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373264">27</a></sup> The data replicate findings from a large meta-analysis<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373219">24</a></sup> drawn from multiple community populations, where information on potential QRS modulators such as heart disease status and medications was not as precisely controlled or excluded from all included studies. The major new finding here is that using the PheWAS study paradigm, we were able to examine the longitudinal associations of these genomic variants on disease in a hypothesis-free manner. This analysis revealed that SNPs in <em>SCN10A</em> (rs6795970) and <em>SCN5A</em> (rs1805126) strongly associated with QRS duration, are also associated with subsequent cardiac arrhythmias. <em>SCN10A</em> specifically is associated with atrial fibrillation. These associations between rs6795970 and atrial fibrillation and cardiac arrhythmias were also seen specifically in the original “heart healthy” study population, and were independent of the SNP’s association with QRS duration. The latter finding suggests that while variants at the <em>SCN5A-SCN10A</em> locus determine QRS and subsequent arrhythmia susceptibility, they may do so by divergent (pleiotropic) pathways, or that conduction slowing occurs not only in the ventricle but also in the atrium where it contributes to susceptibility to atrial fibrillation. Importantly, the selection logic for the case selection algorithm in our GWAS required the absence of cardiovascular disease at the time of the ECG. Therefore, these associations represent subsequent development of cardiac arrhythmias in subjects with these variants. This result highlights the potential of the EMR, with multiple diagnoses and longitudinal follow-up, to identify not only variants associated with disease susceptibility or trait variability, but also subsequent outcomes associated with these variants.</p><p id="P27">Drugs that block <em>SCN5A</em>-encoded sodium channels slow ventricular conduction, prolong QRS duration,<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373227">28</a></sup> and increased mortality following myocardial infarction in the Cardiac Arrhythmia Suppression Trial (CAST).<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373260">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373275">30</a></sup> Available evidence supports the view that slow conduction, particularly in the setting of scarred or ischemic myocardium, promotes reentrant excitation that leads to fatal arrhythmias,<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373223">31</a>–<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373231">33</a></sup> and in CAST, longer QRS durations also predicted increased mortality among patients treated with placebo.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373271">1</a></sup> In addition, a genetic disease caused by <em>SCN5A</em> loss-of-function mutations (Brugada Syndrome) is characterized by slowed ventricular conduction and an increased risk for fatal arrhythmias.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373255">34</a></sup> Interestingly, previous analyses of variable PR duration have also identified strong associations with variants in <em>SCN10A</em>,<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373244">13</a>,<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373268">35</a>–<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373220">37</a></sup> and multiple mechanisms are currently being examined to explain this effect: expression of SCN10A-encoded channels in cardiomyocytes and/or cardiac neurons, or regulation of SCN5A-10A expression.<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373266">38</a>–<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373234">40</a></sup></p><p id="P28">Our PheWAS analysis suggests that <em>SCN10A, SCN5A</em>, and <em>EXOG</em> variants are associated with a cardiac arrhythmia billing codes (entered either by physicians or professional coders); this includes atrial fibrillation and flutter, supraventricular and ventricular tachycardia, cardiac arrest, and other unspecified arrhythmias. <em>SCN10A</em> rs6795970 was specifically associated with atrial fibrillation and flutter, which was noted by Pfeufer et al.<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373239">36</a></sup> but not Holm et al.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373249">35</a></sup> Chambers et al.<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373240">37</a></sup> previously demonstrated associations between <em>SCN10A</em> rs6795970 and heart block and ventricular fibrillation; we also noted an association with first-degree atrioventricular block (p=0.009, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Mouse studies have demonstrated expression of Nav1.8 in vagal and spinal afferents in gastrointestinal mucosa and myenteric plexes,<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373272">41</a></sup> and interestingly <em>SCN10A</em> was also associated with cholecystitis in the PheWAS. Variants in <em>CDKN1A</em> and <em>C6orf204</em>, however, were not associated with cardiac arrhythmias, although we cannot exclude that weak associations may exist. In contrast, the <em>C6orf204</em> locus seems most associated with neoplastic disorders (colorectal cancer, prostatic hypertrophy, and melanoma); its strongest cardiovascular disease association was atherosclerosis (odds ratio 1.094, p=0.03). Thus, PheWAS suggests that although all these regions may be associated with QRS interval, only those SNPs in the <em>SCN5A-10A</em> region seem significantly associated with subsequent development of arrhythmias.</p><p id="P29">Recent growth in large GWAS meta-analysis has shown the power of large numbers to find genomic influences of given traits and disease. In this study, the development of the phenotype algorithm at one site was followed by its use at four other sites with different EMR systems. Algorithm performance was similar to find patients meeting inclusion and exclusion criteria at the three sites who evaluated the algorithm, providing further validation of the transportability of EMR phenotype algorithms.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373217">16</a>,<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373254">42</a></sup> Estimates of heritability of QRS using all SNPs are consistent with other reports; the heritability associated with the very limited subset of 23 SNPs implicated here appears to explain a surprisingly large proportion of this heritability estimate. This analysis also indicates that, as with other traits, there is extensive “missing heritability” when QRS duration is analyzed as a function of common genomic variation.</p><p id="P30">This report highlights both limitations and real and potential advantages of EMR-based genomic research. The present study involved analysis of subjects accrued in the initial stages of the eMERGE network, and as such a major limitation was the relatively small size of the study set. Large consortia such as CHARGE have used meta-analysis to aggregate individual datasets across many sites and have demonstrated the power of the large numbers to generate highly significant results by this approach. One of the key lessons in the eMERGE experience to date has been that algorithms to identify cases and controls for genomic or other study can be successfully deployed across multiple EMR systems.<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373257">16</a>,<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373215">43</a>,<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373236">44</a></sup> Thus, as the number of subjects with dense genomic information across multiple EMR systems grows, this and other EMR-based studies highlight the potential for accrual of increasingly large sample sets to identify genomic predictors of variability in phenotypes such as physiologic traits or disease susceptibility.</p><p id="P31">Further, EMRs hold the promise, as suggested here, of examining longitudinal healthcare outcomes, such as disease complications or response to drug therapies. Identifying cases for such studies requires especially large resources, since subsets of subsets (e.g. drug response <em>X</em> in disease <em>Y</em>) are required. Current efforts demonstrate the feasibility of accrual of DNA collections coupled to EMRs large enough to support statistically valid analyses of rare variants and/or rare clinical events. In the current eMERGE network, there are 9 sites with EMR-based biobanks that include &gt;250,000 subjects, and &gt;50,000 have been genotyped on a genome-wide platform. Other resources that should expand the reach of EMR-based genomic research include the Kaiser Northern California biobank (&gt;100,000 individuals),<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup> the Million Veteran’s Project (currently &gt;100,000 individuals),<sup><a href="#R46" rid="R46" class=" bibr popnode">46</a></sup> and biobanks that will be coupled to national healthcare systems (e.g. the UK Biobank<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373273">47</a></sup>).</p><p id="P32">Studies in the EMR environment enable PheWAS analyses: a PheWAS experiment cannot be executed in the absence of diverse diagnoses across many diagnostic classes in study subjects. The PheWAS-based associations reported here did not achieve significance using a strict Bonferroni correction; however, <em>SCN10A</em> rs6795970 was strongly associated with both atrial fibrillation and cardiac arrhythmias in the normal QRS population as well. The PheWAS approach is still in development and it is likely that the strategy of using only standard diagnostic codes is a limitation. The refinement of disease classifications and abstraction methods will enable more granular phenotypic subsetting, particularly when applied to increasingly large datasets described above.</p><p id="P33">A fundamental limitation of the PheWAS technique is that every diagnosis is not explicitly included or excluded in each record. Another potential limitation of EMR-based phenotyping is the accuracy of the information contained in the record. Extracting phenotypes from the EMR can result in errors if the data in the source EMR are incorrect (e.g., the EMR specifies a disease the patient does not have). Gender and ancestry testing suggest these demographic features are rarely incorrect in an EMR.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_358524288">7</a>,<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373218">20</a></sup> Similarly, the electronic phenotyping experience in eMERGE indicates that recurrent mentions of specific diagnoses or combinations of diverse data types (such as medications plus free text plus diagnostic codes) greatly improve diagnostic accuracy of electronic phenotyping algorithms when assessed as positive predictive value using hand curation as the gold standard.<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_324373270">48</a></sup> EMRs contain data on subjects exposed to a healthcare system and as such EMR-based studies may not be generalizable to a broad population. The conduct of EMR-based studies across a variety of geographical locations and practice settings (as in eMERGE) potentially mitigates this issue.</p><p id="P34" class="p p-last">In summary, a genome-wide association study conducted across multiple EMR systems replicated known associations for a readily available index of cardiac conduction, the QRS duration. The algorithm deployed allowed us to analyze subjects with normal electrocardiograms, and no evidence of heart disease or confounding drugs or electrolyte abnormalities, and the phenome-wide association established genomic variants predicting slower conduction in this population also associated with subsequent development of arrhythmias. Thus, the present findings are consonant with a view that individual susceptibility to serious arrhythmias is determined in part by genetically-determined variability in cardiac electrophysiologic behaviors. Furthermore, this study highlights the advantages of a genotyped EMR population to explore the subsequent emergence of clinically-important phenotypes not ascertained in the original study design.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm139680580353520title">Clinical Perspective</h3></div><p id="__p2">QRS duration, a measure of cardiac intraventricular conduction, varies ~2-fold in individuals without cardiac disease. Slow conduction may promote reentrant arrhythmias. We identified 5,272 individuals with normal ECGs and no evidence of cardiac disease from electronic medical records at five institutions and performed genome-wide association analysis. We found variants in 5 loci associated with QRS duration in normal individuals, including <em>SCN5A, SCN10A, NFIA</em>, near <em>CDKN1A</em>, and near <em>SLC35F1</em> that were replicated in the CHARGE consortium QRS meta-analysis. Subsequently, we performed phenome-wide association studies on associated SNPs. These analyses demonstrated with <em>SCN5A-10A</em> variants were associated with future development of atrial fibrillation and cardiac arrhythmias. We conclude that DNA biobanks coupled to EMRs provide a platform not only for GWAS but also for broadly interrogating the longitudinal incidence of disease associated with genetic variants.</p></div></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><p id="P35"><strong>Supplemental Figure 1:</strong> LocusZoom plot of the region in chromosome 3 in which most of the replicated associations (18/23) were identified. All variants with p&lt;10<sup>−4</sup> in the eMERGE data set were replicated at p&lt;5×10<sup>−8</sup> in the CHARGE data set.</p><p id="P36">Supplemental Figure 2: Phenome-wide association study plot for <em>CDKN1A</em> (rs1321313).</p><p id="P37">Supplemental Figure 3: Phenome-wide association study plot for <em>C6orf204</em> (rs6906287).</p><p id="P38">Supplemental Figure 4: Phenome-wide association study plot for <em>NFIA</em> (rs2207790).</p><p id="P39">Supplemental Table 1: Replicated SNPs in the QRS GWAS analysis. Reported betas and pvalues for eMERGE analysis are adjusted for sex, and all betas for CHARGE and eMERGE analyses are for the coded allele.</p><p id="P40">Supplemental Table 2: PheWAS associations for replicated SNPs (N=23). Data for all associations with P&lt;0.01 are shown. ICD9 codes for given associations can be downloaded from <a href="http://knowledgemap.mc.vanderbilt.edu/research/content/phewas" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://knowledgemap.mc.vanderbilt.edu/research/content/phewas</a>.</p><div class="sup-box half_rhythm" id="d36e1185"><a href="/pmc/articles/PMC3713791/bin/NIHMS484529-supplement-01.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1010K, pdf)</sup></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Acknowledgments</h2><div class="sec"><p id="__p1"><strong>Funding Sources:</strong> This work was supported by the electronic MEdical Records and GEnomics (eMERGE) Network, initiated and funded by the National Human Genome Research Institute, with additional funding from the National Institute of General Medical Sciences, through the following grants: U01-HG-004610 (Group Health Cooperative); U01-HG-004608 (Marshfield Clinic); U01-HG-04599 (Mayo Clinic); U01-HG-004609 (Northwestern University); U01-HG-04603 (Vanderbilt University, also serving as the Administrative Coordinating Center). BioVU also receives support through the National Center for Research Resources UL1 RR024975, which is now at the National Center for Advancing Translational Sciences, 2 UL1 TR000445. Dr. Sotoodehnia was supported by R01-HL088456. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div></div><div id="idm139680573221552" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139680573221552title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P41" class="p p-first-last"><strong>Conflict of Interest Disclosures:</strong> None.</p><p></p><p class="fn sec" id="FN3"></p><p id="P42" class="p p-first-last">This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm139680573218704" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139680573218704title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Capone RJ, Pawitan Y, el-Sherif N, Geraci TS, Handshaw K, Morganroth J, Schlant RC, Waldo AL. Events in the cardiac arrhythmia suppression trial: baseline predictors of mortality in placebo-treated patients. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>1991;<span class="ref-vol">18</span>:1434–1438.</span> [<a href="/pubmed/1939943" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol&amp;title=Events+in+the+cardiac+arrhythmia+suppression+trial:+baseline+predictors+of+mortality+in+placebo-treated+patients&amp;author=RJ+Capone&amp;author=Y+Pawitan&amp;author=N+el-Sherif&amp;author=TS+Geraci&amp;author=K+Handshaw&amp;volume=18&amp;publication_year=1991&amp;pages=1434-1438&amp;pmid=1939943&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Hanson B, Tuna N, Bouchard T, Heston L, Eckert E, Lykken D, Segal N, Rich S. Genetic factors in the electrocardiogram and heart rate of twins reared apart and together. <span><span class="ref-journal">Am. J. Cardiol. </span>1989;<span class="ref-vol">63</span>:606–609.</span> [<a href="/pubmed/2919564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol&amp;title=Genetic+factors+in+the+electrocardiogram+and+heart+rate+of+twins+reared+apart+and+together&amp;author=B+Hanson&amp;author=N+Tuna&amp;author=T+Bouchard&amp;author=L+Heston&amp;author=E+Eckert&amp;volume=63&amp;publication_year=1989&amp;pages=606-609&amp;pmid=2919564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Busjahn A, Knoblauch H, Faulhaber HD, Boeckel T, Rosenthal M, Uhlmann R, Hoehe M, Schuster H, Luft FC. QT interval is linked to 2 long-QT syndrome loci in normal subjects. <span><span class="ref-journal">Circulation. </span>1999;<span class="ref-vol">99</span>:3161–3164.</span> [<a href="/pubmed/10377080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=QT+interval+is+linked+to+2+long-QT+syndrome+loci+in+normal+subjects&amp;author=A+Busjahn&amp;author=H+Knoblauch&amp;author=HD+Faulhaber&amp;author=T+Boeckel&amp;author=M+Rosenthal&amp;volume=99&amp;publication_year=1999&amp;pages=3161-3164&amp;pmid=10377080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Russell MW, Law I, Sholinsky P, Fabsitz RR. Heritability of ECG measurements in adult male twins. <span><span class="ref-journal">J Electrocardiol. </span>1998;<span class="ref-vol">30</span>(Suppl):64–68.</span> [<a href="/pubmed/9535482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Electrocardiol&amp;title=Heritability+of+ECG+measurements+in+adult+male+twins&amp;author=MW+Russell&amp;author=I+Law&amp;author=P+Sholinsky&amp;author=RR+Fabsitz&amp;volume=30&amp;issue=Suppl&amp;publication_year=1998&amp;pages=64-68&amp;pmid=9535482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Ramirez AH, Schildcrout JS, Blakemore DL, Masys DR, Pulley JM, Basford MA, Roden DM, Denny JC. Modulators of normal electrocardiographic intervals identified in a large electronic medical record. <span><span class="ref-journal">Heart Rhythm. </span>2011;<span class="ref-vol">8</span>:271–277.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3057498/">PMC free article</a>]</span> [<a href="/pubmed/21044898" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heart+Rhythm&amp;title=Modulators+of+normal+electrocardiographic+intervals+identified+in+a+large+electronic+medical+record&amp;author=AH+Ramirez&amp;author=JS+Schildcrout&amp;author=DL+Blakemore&amp;author=DR+Masys&amp;author=JM+Pulley&amp;volume=8&amp;publication_year=2011&amp;pages=271-277&amp;pmid=21044898&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">McCarty CA, Wilke RA, Giampietro PF, Wesbrook SD, Caldwell MD. Marshfield Clinic Personalized Medicine Research Project (PMRP): design, methods and recruitment for a large population-based biobank. <span><span class="ref-journal">Personalized Medicine. </span>2005;<span class="ref-vol">2</span>:49–79.</span> [<a href="/pubmed/29793241" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Personalized+Medicine&amp;title=Marshfield+Clinic+Personalized+Medicine+Research+Project+(PMRP):+design,+methods+and+recruitment+for+a+large+population-based+biobank&amp;author=CA+McCarty&amp;author=RA+Wilke&amp;author=PF+Giampietro&amp;author=SD+Wesbrook&amp;author=MD+Caldwell&amp;volume=2&amp;publication_year=2005&amp;pages=49-79&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, Masys DR. Development of a large-scale de-identified DNA biobank to enable personalized medicine. <span><span class="ref-journal">Clin. Pharmacol. Ther. </span>2008;<span class="ref-vol">84</span>:362–369.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3763939/">PMC free article</a>]</span> [<a href="/pubmed/18500243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Pharmacol.+Ther&amp;title=Development+of+a+large-scale+de-identified+DNA+biobank+to+enable+personalized+medicine&amp;author=DM+Roden&amp;author=JM+Pulley&amp;author=MA+Basford&amp;author=GR+Bernard&amp;author=EW+Clayton&amp;volume=84&amp;publication_year=2008&amp;pages=362-369&amp;pmid=18500243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">McCarty CA, Chapman-Stone D, Derfus T, Giampietro PF, Fost N. Community consultation and communication for a population-based DNA biobank: the Marshfield clinic personalized medicine research project. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2008;<span class="ref-vol">146A</span>:3026–3033.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2588646/">PMC free article</a>]</span> [<a href="/pubmed/19006210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=Community+consultation+and+communication+for+a+population-based+DNA+biobank:+the+Marshfield+clinic+personalized+medicine+research+project&amp;author=CA+McCarty&amp;author=D+Chapman-Stone&amp;author=T+Derfus&amp;author=PF+Giampietro&amp;author=N+Fost&amp;volume=146A&amp;publication_year=2008&amp;pages=3026-3033&amp;pmid=19006210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Ginsburg GS, Burke TW, Febbo P. Centralized biorepositories for genetic and genomic research. <span><span class="ref-journal">JAMA. </span>2008;<span class="ref-vol">299</span>:1359–1361.</span> [<a href="/pubmed/18349099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Centralized+biorepositories+for+genetic+and+genomic+research&amp;author=GS+Ginsburg&amp;author=TW+Burke&amp;author=P+Febbo&amp;volume=299&amp;publication_year=2008&amp;pages=1359-1361&amp;pmid=18349099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Lemke AA, Wolf WA, Hebert-Beirne J, Smith ME. Public and biobank participant attitudes toward genetic research participation and data sharing. <span><span class="ref-journal">Public Health Genomics. </span>2010;<span class="ref-vol">13</span>:368–377.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2951726/">PMC free article</a>]</span> [<a href="/pubmed/20805700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Public+Health+Genomics&amp;title=Public+and+biobank+participant+attitudes+toward+genetic+research+participation+and+data+sharing&amp;author=AA+Lemke&amp;author=WA+Wolf&amp;author=J+Hebert-Beirne&amp;author=ME+Smith&amp;volume=13&amp;publication_year=2010&amp;pages=368-377&amp;pmid=20805700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">McCarty CA, Chisholm RL, Chute CG, Kullo IJ, Jarvik GP, Larson EB, Li R, Masys DR, Ritchie MD, Roden DM, Struewing JP, Wolf WA. The eMERGE Network: A consortium of biorepositories linked to electronic medical records data for conducting genomic studies. <span><span class="ref-journal">BMC Med Genomics. </span>2011;<span class="ref-vol">4</span>:13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038887/">PMC free article</a>]</span> [<a href="/pubmed/21269473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genomics&amp;title=The+eMERGE+Network:+A+consortium+of+biorepositories+linked+to+electronic+medical+records+data+for+conducting+genomic+studies&amp;author=CA+McCarty&amp;author=RL+Chisholm&amp;author=CG+Chute&amp;author=IJ+Kullo&amp;author=GP+Jarvik&amp;volume=4&amp;publication_year=2011&amp;pages=13&amp;pmid=21269473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kullo IJ, Ding K, Shameer K, McCarty CA, Jarvik GP, Denny JC, Ritchie MD, Ye Z, Crosslin DR, Chisholm RL, Manolio TA, Chute CG. Complement receptor 1 gene variants are associated with erythrocyte sedimentation rate. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">89</span>:131–138.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135803/">PMC free article</a>]</span> [<a href="/pubmed/21700265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Complement+receptor+1+gene+variants+are+associated+with+erythrocyte+sedimentation+rate&amp;author=IJ+Kullo&amp;author=K+Ding&amp;author=K+Shameer&amp;author=CA+McCarty&amp;author=GP+Jarvik&amp;volume=89&amp;publication_year=2011&amp;pages=131-138&amp;pmid=21700265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Denny JC, Ritchie MD, Crawford DC, Schildcrout JS, Ramirez AH, Pulley JM, Basford MA, Masys DR, Haines JL, Roden DM. Identification of genomic predictors of atrioventricular conduction: using electronic medical records as a tool for genome science. <span><span class="ref-journal">Circulation. </span>2010;<span class="ref-vol">122</span>:2016–2021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991609/">PMC free article</a>]</span> [<a href="/pubmed/21041692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Identification+of+genomic+predictors+of+atrioventricular+conduction:+using+electronic+medical+records+as+a+tool+for+genome+science&amp;author=JC+Denny&amp;author=MD+Ritchie&amp;author=DC+Crawford&amp;author=JS+Schildcrout&amp;author=AH+Ramirez&amp;volume=122&amp;publication_year=2010&amp;pages=2016-2021&amp;pmid=21041692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kullo IJ, Ding K, Jouni H, Smith CY, Chute CG.  [cited 2010 Oct 29];A genome-wide association study of red blood cell traits using the electronic medical record. <span><span class="ref-journal">PLoS ONE. </span>2010 :5.</span> [Internet] Available from:  <a href="http://www.ncbi.nlm.nih.gov/pubmed/20927387" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/20927387</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2946914/">PMC free article</a>]</span> [<a href="/pubmed/20927387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=A+genome-wide+association+study+of+red+blood+cell+traits+using+the+electronic+medical+record&amp;author=IJ+Kullo&amp;author=K+Ding&amp;author=H+Jouni&amp;author=CY+Smith&amp;author=CG+Chute&amp;publication_year=2010&amp;pages=5&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Ritchie MD, Denny JC, Crawford DC, Ramirez AH, Weiner JB, Pulley JM, Basford MA, Brown-Gentry K, Balser JR, Masys DR, Haines JL, Roden DM. Robust replication of genotype-phenotype associations across multiple diseases in an electronic medical record. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:560–572.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850440/">PMC free article</a>]</span> [<a href="/pubmed/20362271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Robust+replication+of+genotype-phenotype+associations+across+multiple+diseases+in+an+electronic+medical+record&amp;author=MD+Ritchie&amp;author=JC+Denny&amp;author=DC+Crawford&amp;author=AH+Ramirez&amp;author=JB+Weiner&amp;volume=86&amp;publication_year=2010&amp;pages=560-572&amp;pmid=20362271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Denny JC, Crawford DC, Ritchie MD, Bielinski SJ, Basford MA, Bradford Y, Chai HS, Bastarache L, Zuvich R, Peissig P, Carrell D, Ramirez AH, Pathak J, Wilke RA, Rasmussen L, Wang X, Pacheco JA, Kho AN, Hayes MG, Weston N, Matsumoto M, Kopp PA, Newton KM, Jarvik GP, Li R, Manolio TA, Kullo IJ, Chute CG, Chisholm RL, Larson EB, McCarty CA, Masys DR, Roden DM, De Andrade M. Variants Near FOXE1 Are Associated with Hypothyroidism and Other Thyroid Conditions: Using Electronic Medical Records for Genome- and Phenome-wide Studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">89</span>:529–542.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3188836/">PMC free article</a>]</span> [<a href="/pubmed/21981779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Variants+Near+FOXE1+Are+Associated+with+Hypothyroidism+and+Other+Thyroid+Conditions:+Using+Electronic+Medical+Records+for+Genome-+and+Phenome-wide+Studies&amp;author=JC+Denny&amp;author=DC+Crawford&amp;author=MD+Ritchie&amp;author=SJ+Bielinski&amp;author=MA+Basford&amp;volume=89&amp;publication_year=2011&amp;pages=529-542&amp;pmid=21981779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Denny JC, Ritchie MD, Basford MA, Pulley JM, Bastarache L, Brown-Gentry K, Wang D, Masys DR, Roden DM, Crawford DC. PheWAS: demonstrating the feasibility of a phenome-wide scan to discover gene-disease associations. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:1205–1210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2859132/">PMC free article</a>]</span> [<a href="/pubmed/20335276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PheWAS:+demonstrating+the+feasibility+of+a+phenome-wide+scan+to+discover+gene-disease+associations&amp;author=JC+Denny&amp;author=MD+Ritchie&amp;author=MA+Basford&amp;author=JM+Pulley&amp;author=L+Bastarache&amp;volume=26&amp;publication_year=2010&amp;pages=1205-1210&amp;pmid=20335276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Denny JC, Spickard A, Johnson KB, Peterson NB, Peterson JF, Miller RA. Evaluation of a method to identify and categorize section headers in clinical documents. <span><span class="ref-journal">J Am Med Inform Assoc. </span>2009;<span class="ref-vol">16</span>:806–815.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3002123/">PMC free article</a>]</span> [<a href="/pubmed/19717800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Inform+Assoc&amp;title=Evaluation+of+a+method+to+identify+and+categorize+section+headers+in+clinical+documents&amp;author=JC+Denny&amp;author=A+Spickard&amp;author=KB+Johnson&amp;author=NB+Peterson&amp;author=JF+Peterson&amp;volume=16&amp;publication_year=2009&amp;pages=806-815&amp;pmid=19717800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Denny JC, Miller RA, Waitman LR, Arrieta MA, Peterson JF. Identifying QT prolongation from ECG impressions using a general-purpose Natural Language Processor. <span><span class="ref-journal">Int J Med Inform. </span>2009;<span class="ref-vol">78 Suppl 1</span>:S34–42.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2728459/">PMC free article</a>]</span> [<a href="/pubmed/18938105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Med+Inform&amp;title=Identifying+QT+prolongation+from+ECG+impressions+using+a+general-purpose+Natural+Language+Processor&amp;author=JC+Denny&amp;author=RA+Miller&amp;author=LR+Waitman&amp;author=MA+Arrieta&amp;author=JF+Peterson&amp;volume=78+Suppl+1&amp;publication_year=2009&amp;pages=S34-42&amp;pmid=18938105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Dumitrescu L, Ritchie MD, Brown-Gentry K, Pulley JM, Basford M, Denny JC, Oksenberg JR, Roden DM, Haines JL, Crawford DC. Assessing the accuracy of observer-reported ancestry in a biorepository linked to electronic medical records. <span><span class="ref-journal">Genet Med. </span>2010;<span class="ref-vol">12</span>:648–650.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2952033/">PMC free article</a>]</span> [<a href="/pubmed/20733501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med&amp;title=Assessing+the+accuracy+of+observer-reported+ancestry+in+a+biorepository+linked+to+electronic+medical+records&amp;author=L+Dumitrescu&amp;author=MD+Ritchie&amp;author=K+Brown-Gentry&amp;author=JM+Pulley&amp;author=M+Basford&amp;volume=12&amp;publication_year=2010&amp;pages=648-650&amp;pmid=20733501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Turner S, Armstrong LL, Bradford Y, Carlson CS, Crawford DC, Crenshaw AT, De Andrade M, Doheny KF, Haines JL, Hayes G, Jarvik G, Jiang L, Kullo IJ, Li R, Ling H, Manolio TA, Matsumoto M, McCarty CA, McDavid AN, Mirel DB, Paschall JE, Pugh EW, Rasmussen LV, Wilke RA, Zuvich RL, Ritchie MD. Quality control procedures for genome-wide association studies. <span><span class="ref-journal">Curr Protoc Hum Genet. </span>2011</span> Chapter 1:Unit1.19. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3066182/">PMC free article</a>]</span> [<a href="/pubmed/21234875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Hum+Genet&amp;title=Quality+control+procedures+for+genome-wide+association+studies&amp;author=S+Turner&amp;author=LL+Armstrong&amp;author=Y+Bradford&amp;author=CS+Carlson&amp;author=DC+Crawford&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Sotoodehnia N, Isaacs A, De Bakker PIW, Dörr M, Newton-Cheh C, Nolte IM, Van der Harst P, Müller M, Eijgelsheim M, Alonso A, Hicks AA, Padmanabhan S, Hayward C, Smith AV, Polasek O, Giovannone S, Fu J, Magnani JW, Marciante KD, Pfeufer A, Gharib SA, Teumer A, Li M, Bis JC, Rivadeneira F, Aspelund T, Köttgen A, Johnson T, Rice K, Sie MPS, Wang YA, Klopp N, Fuchsberger C, Wild SH, Mateo Leach I, Estrada K, Völker U, Wright AF, Asselbergs FW, Qu J, Chakravarti A, Sinner MF, Kors JA, Petersmann A, Harris TB, Soliman EZ, Munroe PB, Psaty BM, Oostra BA, Cupples LA, Perz S, De Boer RA, Uitterlinden AG, Völzke H, Spector TD, Liu F-Y, Boerwinkle E, Dominiczak AF, Rotter JI, Van Herpen G, Levy D, Wichmann H-E, Van Gilst WH, Witteman JCM, Kroemer HK, Kao WHL, Heckbert SR, Meitinger T, Hofman A, Campbell H, Folsom AR, Van Veldhuisen DJ, Schwienbacher C, O’Donnell CJ, Volpato CB, Caulfield MJ, Connell JM, Launer L, Lu X, Franke L, Fehrmann RSN, Te Meerman G, Groen HJM, Weersma RK, Van den Berg LH, Wijmenga C, Ophoff RA, Navis G, Rudan I, Snieder H, Wilson JF, Pramstaller PP, Siscovick DS, Wang TJ, Gudnason V, Van Duijn CM, Felix SB, Fishman GI, et al.  Common variants in 22 loci are associated with QRS duration and cardiac ventricular conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:1068–1076.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3338195/">PMC free article</a>]</span> [<a href="/pubmed/21076409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+22+loci+are+associated+with+QRS+duration+and+cardiac+ventricular+conduction&amp;author=N+Sotoodehnia&amp;author=A+Isaacs&amp;author=PIW+De+Bakker&amp;author=M+Dörr&amp;author=C+Newton-Cheh&amp;volume=42&amp;publication_year=2010&amp;pages=1068-1076&amp;pmid=21076409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A Tool for Genome-wide Complex Trait Analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+A+Tool+for+Genome-wide+Complex+Trait+Analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MAR, Bender D, Maller J, Sklar P, De Bakker PIW, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MAR+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Kurreeman F, Liao K, Chibnik L, Hickey B, Stahl E, Gainer V, Li G, Bry L, Mahan S, Ardlie K, Thomson B, Szolovits P, Churchill S, Murphy SN, Cai T, Raychaudhuri S, Kohane I, Karlson E, Plenge RM. Genetic basis of autoantibody positive and negative rheumatoid arthritis risk in a multi-ethnic cohort derived from electronic health records. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:57–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014362/">PMC free article</a>]</span> [<a href="/pubmed/21211616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+basis+of+autoantibody+positive+and+negative+rheumatoid+arthritis+risk+in+a+multi-ethnic+cohort+derived+from+electronic+health+records&amp;author=F+Kurreeman&amp;author=K+Liao&amp;author=L+Chibnik&amp;author=B+Hickey&amp;author=E+Stahl&amp;volume=88&amp;publication_year=2011&amp;pages=57-69&amp;pmid=21211616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Johnson EA, McKinnon MG. The differential effect of quinidine and pyrilamine on the myocardial action potential at various rates of stimulation. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>1957;<span class="ref-vol">120</span>:460–468.</span> [<a href="/pubmed/13476371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=The+differential+effect+of+quinidine+and+pyrilamine+on+the+myocardial+action+potential+at+various+rates+of+stimulation&amp;author=EA+Johnson&amp;author=MG+McKinnon&amp;volume=120&amp;publication_year=1957&amp;pages=460-468&amp;pmid=13476371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Preliminary report: effect of encainide and flecainide on mortality in a randomized trial of arrhythmia suppression after myocardial infarction. The Cardiac Arrhythmia Suppression Trial (CAST) Investigators. <span><span class="ref-journal">N Engl J Med. </span>1989;<span class="ref-vol">321</span>:406–412.</span> [<a href="/pubmed/2473403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Preliminary+report:+effect+of+encainide+and+flecainide+on+mortality+in+a+randomized+trial+of+arrhythmia+suppression+after+myocardial+infarction.+The+Cardiac+Arrhythmia+Suppression+Trial+(CAST)+Investigators&amp;volume=321&amp;publication_year=1989&amp;pages=406-412&amp;pmid=2473403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Epstein AE, Hallstrom AP, Rogers WJ, Liebson PR, Seals AA, Anderson JL, Cohen JD, Capone RJ, Wyse DG. Mortality following ventricular arrhythmia suppression by encainide, flecainide, and moricizine after myocardial infarction. The original design concept of the Cardiac Arrhythmia Suppression Trial (CAST) <span><span class="ref-journal">JAMA. </span>1993;<span class="ref-vol">270</span>:2451–2455.</span> [<a href="/pubmed/8230622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Mortality+following+ventricular+arrhythmia+suppression+by+encainide,+flecainide,+and+moricizine+after+myocardial+infarction.+The+original+design+concept+of+the+Cardiac+Arrhythmia+Suppression+Trial+(CAST)&amp;author=AE+Epstein&amp;author=AP+Hallstrom&amp;author=WJ+Rogers&amp;author=PR+Liebson&amp;author=AA+Seals&amp;volume=270&amp;publication_year=1993&amp;pages=2451-2455&amp;pmid=8230622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Akiyama T, Pawitan Y, Greenberg H, Kuo CS, Reynolds-Haertle RA. Increased risk of death and cardiac arrest from encainide and flecainide in patients after non-Q-wave acute myocardial infarction in the Cardiac Arrhythmia Suppression Trial. CAST Investigators. <span><span class="ref-journal">Am J Cardiol. </span>1991;<span class="ref-vol">68</span>:1551–1555.</span> [<a href="/pubmed/1720917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Cardiol&amp;title=Increased+risk+of+death+and+cardiac+arrest+from+encainide+and+flecainide+in+patients+after+non-Q-wave+acute+myocardial+infarction+in+the+Cardiac+Arrhythmia+Suppression+Trial.+CAST+Investigators&amp;author=T+Akiyama&amp;author=Y+Pawitan&amp;author=H+Greenberg&amp;author=CS+Kuo&amp;author=RA+Reynolds-Haertle&amp;volume=68&amp;publication_year=1991&amp;pages=1551-1555&amp;pmid=1720917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Coromilas J, Saltman AE, Waldecker B, Dillon SM, Wit AL. Electrophysiological effects of flecainide on anisotropic conduction and reentry in infarcted canine hearts. <span><span class="ref-journal">Circulation. </span>1995;<span class="ref-vol">91</span>:2245–2263.</span> [<a href="/pubmed/7697855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Electrophysiological+effects+of+flecainide+on+anisotropic+conduction+and+reentry+in+infarcted+canine+hearts&amp;author=J+Coromilas&amp;author=AE+Saltman&amp;author=B+Waldecker&amp;author=SM+Dillon&amp;author=AL+Wit&amp;volume=91&amp;publication_year=1995&amp;pages=2245-2263&amp;pmid=7697855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Greenberg HM, Dwyer EM, Jr, Hochman JS, Steinberg JS, Echt DS, Peters RW. Interaction of ischaemia and encainide/flecainide treatment: a proposed mechanism for the increased mortality in CAST I. <span><span class="ref-journal">Br Heart J. </span>1995;<span class="ref-vol">74</span>:631–635.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC484119/">PMC free article</a>]</span> [<a href="/pubmed/8541168" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+Heart+J&amp;title=Interaction+of+ischaemia+and+encainide/flecainide+treatment:+a+proposed+mechanism+for+the+increased+mortality+in+CAST+I&amp;author=HM+Greenberg&amp;author=EM+Dwyer&amp;author=JS+Hochman&amp;author=JS+Steinberg&amp;author=DS+Echt&amp;volume=74&amp;publication_year=1995&amp;pages=631-635&amp;pmid=8541168&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Antzelevitch C, Brugada P, Borggrefe M, Brugada J, Brugada R, Corrado D, Gussak I, LeMarec H, Nademanee K, Perez Riera AR, Shimizu W, Schulze-Bahr E, Tan H, Wilde A. Brugada syndrome: report of the second consensus conference: endorsed by the Heart Rhythm Society and the European Heart Rhythm Association. <span><span class="ref-journal">Circulation. </span>2005;<span class="ref-vol">111</span>:659–670.</span> [<a href="/pubmed/15655131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Brugada+syndrome:+report+of+the+second+consensus+conference:+endorsed+by+the+Heart+Rhythm+Society+and+the+European+Heart+Rhythm+Association&amp;author=C+Antzelevitch&amp;author=P+Brugada&amp;author=M+Borggrefe&amp;author=J+Brugada&amp;author=R+Brugada&amp;volume=111&amp;publication_year=2005&amp;pages=659-670&amp;pmid=15655131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Holm H, Gudbjartsson DF, Arnar DO, Thorleifsson G, Thorgeirsson G, Stefansdottir H, Gudjonsson SA, Jonasdottir A, Mathiesen EB, Njølstad I, Nyrnes A, Wilsgaard T, Hald EM, Hveem K, Stoltenberg C, Løchen M-L, Kong A, Thorsteinsdottir U, Stefansson K. Several common variants modulate heart rate, PR interval and QRS duration. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:117–122.</span> [<a href="/pubmed/20062063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Several+common+variants+modulate+heart+rate,+PR+interval+and+QRS+duration&amp;author=H+Holm&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=G+Thorleifsson&amp;author=G+Thorgeirsson&amp;volume=42&amp;publication_year=2010&amp;pages=117-122&amp;pmid=20062063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Pfeufer A, Van Noord C, Marciante KD, Arking DE, Larson MG, Smith AV, Tarasov KV, Müller M, Sotoodehnia N, Sinner MF, Verwoert GC, Li M, Kao WHL, Köttgen A, Coresh J, Bis JC, Psaty BM, Rice K, Rotter JI, Rivadeneira F, Hofman A, Kors JA, Stricker BHC, Uitterlinden AG, Van Duijn CM, Beckmann BM, Sauter W, Gieger C, Lubitz SA, Newton-Cheh C, Wang TJ, Magnani JW, Schnabel RB, Chung MK, Barnard J, Smith JD, Van Wagoner DR, Vasan RS, Aspelund T, Eiriksdottir G, Harris TB, Launer LJ, Najjar SS, Lakatta E, Schlessinger D, Uda M, Abecasis GR, Müller-Myhsok B, Ehret GB, Boerwinkle E, Chakravarti A, Soliman EZ, Lunetta KL, Perz S, Wichmann H-E, Meitinger T, Levy D, Gudnason V, Ellinor PT, Sanna S, Kääb S, Witteman JCM, Alonso A, Benjamin EJ, Heckbert SR. Genome-wide association study of PR interval. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:153–159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850197/">PMC free article</a>]</span> [<a href="/pubmed/20062060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+PR+interval&amp;author=A+Pfeufer&amp;author=C+Van+Noord&amp;author=KD+Marciante&amp;author=DE+Arking&amp;author=MG+Larson&amp;volume=42&amp;publication_year=2010&amp;pages=153-159&amp;pmid=20062060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Chambers JC, Zhao J, Terracciano CMN, Bezzina CR, Zhang W, Kaba R, Navaratnarajah M, Lotlikar A, Sehmi JS, Kooner MK, Deng G, Siedlecka U, Parasramka S, El-Hamamsy I, Wass MN, Dekker LRC, De Jong JSSG, Sternberg MJE, McKenna W, Severs NJ, De Silva R, Wilde AAM, Anand P, Yacoub M, Scott J, Elliott P, Wood JN, Kooner JS. Genetic variation in SCN10A influences cardiac conduction. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:149–152.</span> [<a href="/pubmed/20062061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+SCN10A+influences+cardiac+conduction&amp;author=JC+Chambers&amp;author=J+Zhao&amp;author=CMN+Terracciano&amp;author=CR+Bezzina&amp;author=W+Zhang&amp;volume=42&amp;publication_year=2010&amp;pages=149-152&amp;pmid=20062061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">London B. Whither Art Thou, SCN10A, and What Art Thou Doing? <span><span class="ref-journal">Circ Res. </span>2012;<span class="ref-vol">111</span>:268–270.</span> [<a href="/pubmed/22821905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Whither+Art+Thou,+SCN10A,+and+What+Art+Thou+Doing?&amp;author=B+London&amp;volume=111&amp;publication_year=2012&amp;pages=268-270&amp;pmid=22821905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Yang T, Atack TC, Stroud DM, Zhang W, Hall L, Roden DM. Blocking Scn10a channels in heart reduces late sodium current and is antiarrhythmic. <span><span class="ref-journal">Circ Res. </span>2012;<span class="ref-vol">111</span>:322–332.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3412150/">PMC free article</a>]</span> [<a href="/pubmed/22723299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Blocking+Scn10a+channels+in+heart+reduces+late+sodium+current+and+is+antiarrhythmic&amp;author=T+Yang&amp;author=TC+Atack&amp;author=DM+Stroud&amp;author=W+Zhang&amp;author=L+Hall&amp;volume=111&amp;publication_year=2012&amp;pages=322-332&amp;pmid=22723299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Verkerk AO, Remme CA, Schumacher CA, Scicluna BP, Wolswinkel R, De Jonge B, Bezzina CR, Veldkamp MW. Functional Nav1.8 channels in intracardiac neurons: the link between SCN10A and cardiac electrophysiology. <span><span class="ref-journal">Circ Res. </span>2012;<span class="ref-vol">111</span>:333–343.</span> [<a href="/pubmed/22723301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Functional+Nav1.8+channels+in+intracardiac+neurons:+the+link+between+SCN10A+and+cardiac+electrophysiology&amp;author=AO+Verkerk&amp;author=CA+Remme&amp;author=CA+Schumacher&amp;author=BP+Scicluna&amp;author=R+Wolswinkel&amp;volume=111&amp;publication_year=2012&amp;pages=333-343&amp;pmid=22723301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Gautron L, Sakata I, Udit S, Zigman JM, Wood JN, Elmquist JK. Genetic tracing of Nav1.8-expressing vagal afferents in the mouse. <span><span class="ref-journal">J Comp Neurol. </span>2011;<span class="ref-vol">519</span>:3085–3101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3306808/">PMC free article</a>]</span> [<a href="/pubmed/21618224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Comp+Neurol&amp;title=Genetic+tracing+of+Nav1.8-expressing+vagal+afferents+in+the+mouse&amp;author=L+Gautron&amp;author=I+Sakata&amp;author=S+Udit&amp;author=JM+Zigman&amp;author=JN+Wood&amp;volume=519&amp;publication_year=2011&amp;pages=3085-3101&amp;pmid=21618224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Carroll RJ, Thompson WK, Eyler AE, Mandelin AM, Cai T, Zink RM, Pacheco JA, Boomershine CS, Lasko TA, Xu H, Karlson EW, Perez RG, Gainer VS, Murphy SN, Ruderman EM, Pope RM, Plenge RM, Kho AN, Liao KP, Denny JC. Portability of an algorithm to identify rheumatoid arthritis in electronic health records. <span><span class="ref-journal">JAMIA. </span>2012;<span class="ref-vol">19</span>:e162–e169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3392871/">PMC free article</a>]</span> [<a href="/pubmed/22374935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMIA&amp;title=Portability+of+an+algorithm+to+identify+rheumatoid+arthritis+in+electronic+health+records&amp;author=RJ+Carroll&amp;author=WK+Thompson&amp;author=AE+Eyler&amp;author=AM+Mandelin&amp;author=T+Cai&amp;volume=19&amp;publication_year=2012&amp;pages=e162-e169&amp;pmid=22374935&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kho AN, Hayes MG, Rasmussen-Torvik L, Pacheco JA, Thompson WK, Armstrong LL, Denny JC, Peissig PL, Miller AW, Wei W-Q, Bielinski SJ, Chute CG, Leibson CL, Jarvik GP, Crosslin DR, Carlson CS, Newton KM, Wolf WA, Chisholm RL, Lowe WL. Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study. <span><span class="ref-journal">J Am Med Inform Assoc. </span>2012;<span class="ref-vol">19</span>:212–218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3277617/">PMC free article</a>]</span> [<a href="/pubmed/22101970" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Inform+Assoc&amp;title=Use+of+diverse+electronic+medical+record+systems+to+identify+genetic+risk+for+type+2+diabetes+within+a+genome-wide+association+study&amp;author=AN+Kho&amp;author=MG+Hayes&amp;author=L+Rasmussen-Torvik&amp;author=JA+Pacheco&amp;author=WK+Thompson&amp;volume=19&amp;publication_year=2012&amp;pages=212-218&amp;pmid=22101970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Carroll RJ, Thompson WK, Eyler AE, Mandelin AM, Cai T, Zink RM, Pacheco JA, Boomershine CS, Lasko TA, Xu H, Karlson EW, Perez RG, Gainer VS, Murphy SN, Ruderman EM, Pope RM, Plenge RM, Kho AN, Liao KP, Denny JC.  [cited 2012 Mar 13];Portability of an algorithm to identify rheumatoid arthritis in electronic health records. <span><span class="ref-journal">JAMIA. </span>2012 </span> [Internet] Available from:  <a href="http://www.ncbi.nlm.nih.gov/pubmed/22374935" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/pubmed/22374935</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3392871/">PMC free article</a>]</span> [<a href="/pubmed/22374935" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMIA&amp;title=Portability+of+an+algorithm+to+identify+rheumatoid+arthritis+in+electronic+health+records&amp;author=RJ+Carroll&amp;author=WK+Thompson&amp;author=AE+Eyler&amp;author=AM+Mandelin&amp;author=T+Cai&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation"> [cited 2011 Sep 13];Kaiser Permanente, UCSF Scientists Complete NIH-Funded Genomics Project Involving 100,000 People [Internet] <span></span> Available from:  <a href="http://www.dor.kaiser.org/external/news/press_releases/Kaiser_Permanente,_UCSF_Scientists_Complete_NIH-Funded_Genomics_Project_Involving_100,000_People/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.dor.kaiser.org/external/news/press_releases/Kaiser_Permanente,_UCSF_Scientists_ Complete_NIH-Funded_Genomics_Project_Involving_100,000_People/</a></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation"> [cited 2012 Jun 20];Million Veteran Program (MVP) [Internet] <span></span> Available from:  <a href="http://www.research.va.gov/mvp/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.research.va.gov/mvp/</a></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Collins R. What makes UK Biobank special? <span><span class="ref-journal">Lancet. </span>2012;<span class="ref-vol">379</span>:1173–1174.</span> [<a href="/pubmed/22463865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=What+makes+UK+Biobank+special?&amp;author=R+Collins&amp;volume=379&amp;publication_year=2012&amp;pages=1173-1174&amp;pmid=22463865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Kho AN, Pacheco JA, Peissig PL, Rasmussen L, Newton KM, Weston N, Crane PK, Pathak J, Chute CG, Bielinski SJ, Kullo IJ, Li R, Manolio TA, Chisholm RL, Denny JC. Electronic Medical Records for Genetic Research: Results of the eMERGE Consortium. <span><span class="ref-journal">Sci Transl Med. </span>2011;<span class="ref-vol">3</span>:79re1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3690272/">PMC free article</a>]</span> [<a href="/pubmed/21508311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Electronic+Medical+Records+for+Genetic+Research:+Results+of+the+eMERGE+Consortium&amp;author=AN+Kho&amp;author=JA+Pacheco&amp;author=PL+Peissig&amp;author=L+Rasmussen&amp;author=KM+Newton&amp;volume=3&amp;publication_year=2011&amp;pages=79re1&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3713791&amp;issue-id=225173&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139680567731296title">Abstract</a></li><li><a href="#S4title">Introduction</a></li><li><a href="#S5title">Methods</a></li><li><a href="#S10title">Results</a></li><li><a href="#S15title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S16title">Acknowledgments</a></li><li><a href="#idm139680573221552title">Footnotes</a></li><li><a href="#idm139680573218704title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3713791/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3713791/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3713791/pdf/nihms-484529.pdf">PDF (541K)</a></li> | <li><a href="#" data-citationid="PMC3713791" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3713791%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3713791%2F&amp;text=Genome-%20and%20Phenome-Wide%20Analysis%20of%20Cardiac%20Conduction%20Identifies%20Markers%20of%20Arrhythmia%20Risk" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3713791%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23463857" data-aiid="3713791" data-aid="3713791" data-iid="225173" data-domainid="319" data-domain="nihpa" data-accid="PMC3713791" data-md5="15ab7ed5985a017012a18dd877c70c31"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:24:49-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A65B6DD55795100000000028E01A2&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3713791%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3713791/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>