<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3518930/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Interleukin 28B polymorphisms are the only common genetic variants associated with low-density lipoprotein cholesterol (LDL-C) in genotype-1 chronic hepatitis C and determine the association between LDL-C and treatment response</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518930/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of viral hepatitis"><meta name="citation_title" content="Interleukin 28B polymorphisms are the only common genetic variants associated with low-density lipoprotein cholesterol (LDL-C) in genotype-1 chronic hepatitis C and determine the association between LDL-C and treatment response"><meta name="citation_authors" content="P. J. Clark, A. J. Thompson, M. Zhu, D. M. Vock, Q. Zhu, D. Ge, K. Patel, S. A. Harrison, T. J. Urban, S. Naggie, J. Fellay, H. L. Tillmann, K. Shianna, S. Noviello, L. D. Pedicone, R. Esteban, P. Kwo, M. S. Sulkowski, N. Afdhal, J. K. Albrecht, D. B. Goldstein, J. G. McHutchison, A. J. Muir"><meta name="citation_date" content="May 2012"><meta name="citation_issue" content="5"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="332"><meta name="citation_doi" content="10.1111/j.1365-2893.2011.01553.x"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3518930/?report=abstract"><meta name="citation_pmid" content="22497812"><meta name="DC.Title" content="Interleukin 28B polymorphisms are the only common genetic variants associated with low-density lipoprotein cholesterol (LDL-C) in genotype-1 chronic hepatitis C and determine the association between LDL-C and treatment response"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="P. J. Clark"><meta name="DC.Contributor" content="A. J. Thompson"><meta name="DC.Contributor" content="M. Zhu"><meta name="DC.Contributor" content="D. M. Vock"><meta name="DC.Contributor" content="Q. Zhu"><meta name="DC.Contributor" content="D. Ge"><meta name="DC.Contributor" content="K. Patel"><meta name="DC.Contributor" content="S. A. Harrison"><meta name="DC.Contributor" content="T. J. Urban"><meta name="DC.Contributor" content="S. Naggie"><meta name="DC.Contributor" content="J. Fellay"><meta name="DC.Contributor" content="H. L. Tillmann"><meta name="DC.Contributor" content="K. Shianna"><meta name="DC.Contributor" content="S. Noviello"><meta name="DC.Contributor" content="L. D. Pedicone"><meta name="DC.Contributor" content="R. Esteban"><meta name="DC.Contributor" content="P. Kwo"><meta name="DC.Contributor" content="M. S. Sulkowski"><meta name="DC.Contributor" content="N. Afdhal"><meta name="DC.Contributor" content="J. K. Albrecht"><meta name="DC.Contributor" content="D. B. Goldstein"><meta name="DC.Contributor" content="J. G. McHutchison"><meta name="DC.Contributor" content="A. J. Muir"><meta name="DC.Date" content="2012 May"><meta name="DC.Identifier" content="10.1111/j.1365-2893.2011.01553.x"><meta name="DC.Language" content="en"><meta property="og:title" content="Interleukin 28B polymorphisms are the only common genetic variants associated with low-density lipoprotein cholesterol (LDL-C) in genotype-1 chronic hepatitis C and determine the association between LDL-C and treatment response"><meta property="og:type" content="article"><meta property="og:description" content="Low-density lipoprotein cholesterol (LDL-C) levels and interleukin 28B (IL28B) polymorphism are associated with sustained viral response (SVR) to peginterferon/ribavirin (pegIFN/RBV) for chronic hepatitis C (CHC) infection. IL28B has been linked with ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518930/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3518930/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3518930/pdf/nihms380108.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A3241DD53F1710000000002F10204.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3518930%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3518930</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Viral Hepat</a></span>. Author manuscript; available in PMC 2012 Dec 11.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22497812" target="pmc_ext" ref="reftype=publisher&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">J Viral Hepat. 2012 May; 19(5): 332–340. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Feb 22. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fj.1365-2893.2011.01553.x" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/j.1365-2893.2011.01553.x</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3518930</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS380108</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22497812">22497812</a></div></div></div></div><h1 class="content-title">Interleukin 28B polymorphisms are the only common genetic variants associated with low-density lipoprotein cholesterol (LDL-C) in genotype-1 chronic hepatitis C and determine the association between LDL-C and treatment response</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Clark%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613039072" co-class="co-affbox">P. J. Clark</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Thompson%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067611234384" co-class="co-affbox">A. J. Thompson</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Zhu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067604226592" co-class="co-affbox">M. Zhu</a>,<sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Vock%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067604223872" co-class="co-affbox">D. M. Vock</a>,<sup>1</sup> <a href="/pubmed/?term=Zhu%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067616564448" co-class="co-affbox">Q. Zhu</a>,<sup>3</sup> <a href="/pubmed/?term=Ge%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067616562464" co-class="co-affbox">D. Ge</a>,<sup>3</sup> <a href="/pubmed/?term=Patel%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615181680" co-class="co-affbox">K. Patel</a>,<sup>1</sup> <a href="/pubmed/?term=Harrison%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615179696" co-class="co-affbox">S. A. Harrison</a>,<sup>4</sup> <a href="/pubmed/?term=Urban%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613792080" co-class="co-affbox">T. J. Urban</a>,<sup>3</sup> <a href="/pubmed/?term=Naggie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613790096" co-class="co-affbox">S. Naggie</a>,<sup>1</sup> <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613788112" co-class="co-affbox">J. Fellay</a>,<sup>3</sup> <a href="/pubmed/?term=Tillmann%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615643456" co-class="co-affbox">H. L. Tillmann</a>,<sup>1</sup> <a href="/pubmed/?term=Shianna%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615641472" co-class="co-affbox">K. Shianna</a>,<sup>3</sup> <a href="/pubmed/?term=Noviello%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067610815600" co-class="co-affbox">S. Noviello</a>,<sup>5</sup> <a href="/pubmed/?term=Pedicone%20LD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067610813616" co-class="co-affbox">L. D. Pedicone</a>,<sup>5</sup> <a href="/pubmed/?term=Esteban%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615360752" co-class="co-affbox">R. Esteban</a>,<sup>6</sup> <a href="/pubmed/?term=Kwo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067615358768" co-class="co-affbox">P. Kwo</a>,<sup>7</sup> <a href="/pubmed/?term=Sulkowski%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613270352" co-class="co-affbox">M. S. Sulkowski</a>,<sup>8</sup> <a href="/pubmed/?term=Afdhal%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613268368" co-class="co-affbox">N. Afdhal</a>,<sup>9</sup> <a href="/pubmed/?term=Albrecht%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067617240496" co-class="co-affbox">J. K. Albrecht</a>,<sup>5</sup> <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067617238512" co-class="co-affbox">D. B. Goldstein</a>,<sup>3</sup> <a href="/pubmed/?term=McHutchison%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613394224" co-class="co-affbox">J. G. McHutchison</a>,<sup>1</sup> and  <a href="/pubmed/?term=Muir%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812" class="affpopup" co-rid="_co_idm140067613392240" co-class="co-affbox">A. J. Muir</a><sup>1</sup>, for the IDEAL investigators<sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140067613039072"><h3 class="no_margin">P. J. Clark</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><p><sup>2</sup>Kirby Institute for Infection and Immunity in Society, University of New South Wales, Sydney, NSW, Australia</p><div>Find articles by <a href="/pubmed/?term=Clark%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">P. J. Clark</a></div></div><div id="_co_idm140067611234384"><h3 class="no_margin">A. J. Thompson</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Thompson%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">A. J. Thompson</a></div></div><div id="_co_idm140067604226592"><h3 class="no_margin">M. Zhu</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Zhu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">M. Zhu</a></div></div><div id="_co_idm140067604223872"><h3 class="no_margin">D. M. Vock</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Vock%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">D. M. Vock</a></div></div><div id="_co_idm140067616564448"><h3 class="no_margin">Q. Zhu</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Zhu%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">Q. Zhu</a></div></div><div id="_co_idm140067616562464"><h3 class="no_margin">D. Ge</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Ge%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">D. Ge</a></div></div><div id="_co_idm140067615181680"><h3 class="no_margin">K. Patel</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Patel%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">K. Patel</a></div></div><div id="_co_idm140067615179696"><h3 class="no_margin">S. A. Harrison</h3><p><sup>4</sup>Brooke Army Medical Center, Fort Sam Houston, TX</p><div>Find articles by <a href="/pubmed/?term=Harrison%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">S. A. Harrison</a></div></div><div id="_co_idm140067613792080"><h3 class="no_margin">T. J. Urban</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Urban%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">T. J. Urban</a></div></div><div id="_co_idm140067613790096"><h3 class="no_margin">S. Naggie</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Naggie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">S. Naggie</a></div></div><div id="_co_idm140067613788112"><h3 class="no_margin">J. Fellay</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Fellay%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">J. Fellay</a></div></div><div id="_co_idm140067615643456"><h3 class="no_margin">H. L. Tillmann</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Tillmann%20HL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">H. L. Tillmann</a></div></div><div id="_co_idm140067615641472"><h3 class="no_margin">K. Shianna</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Shianna%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">K. Shianna</a></div></div><div id="_co_idm140067610815600"><h3 class="no_margin">S. Noviello</h3><p><sup>5</sup>Schering-Plough Corporation, now Merck &amp; Co., Inc., Whitehouse Station, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Noviello%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">S. Noviello</a></div></div><div id="_co_idm140067610813616"><h3 class="no_margin">L. D. Pedicone</h3><p><sup>5</sup>Schering-Plough Corporation, now Merck &amp; Co., Inc., Whitehouse Station, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Pedicone%20LD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">L. D. Pedicone</a></div></div><div id="_co_idm140067615360752"><h3 class="no_margin">R. Esteban</h3><p><sup>6</sup>Hospital General Universitario Valle de Hebron, Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Esteban%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">R. Esteban</a></div></div><div id="_co_idm140067615358768"><h3 class="no_margin">P. Kwo</h3><p><sup>7</sup>Indiana University School of Medicine, Indianapolis, IN</p><div>Find articles by <a href="/pubmed/?term=Kwo%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">P. Kwo</a></div></div><div id="_co_idm140067613270352"><h3 class="no_margin">M. S. Sulkowski</h3><p><sup>8</sup>Johns Hopkins University School of Medicine, Baltimore, MD</p><div>Find articles by <a href="/pubmed/?term=Sulkowski%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">M. S. Sulkowski</a></div></div><div id="_co_idm140067613268368"><h3 class="no_margin">N. Afdhal</h3><p><sup>9</sup>Beth Israel Deaconess Medical Center, Harvard University Boston, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Afdhal%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">N. Afdhal</a></div></div><div id="_co_idm140067617240496"><h3 class="no_margin">J. K. Albrecht</h3><p><sup>5</sup>Schering-Plough Corporation, now Merck &amp; Co., Inc., Whitehouse Station, NJ, USA</p><div>Find articles by <a href="/pubmed/?term=Albrecht%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">J. K. Albrecht</a></div></div><div id="_co_idm140067617238512"><h3 class="no_margin">D. B. Goldstein</h3><p><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</p><div>Find articles by <a href="/pubmed/?term=Goldstein%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">D. B. Goldstein</a></div></div><div id="_co_idm140067613394224"><h3 class="no_margin">J. G. McHutchison</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=McHutchison%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">J. G. McHutchison</a></div></div><div id="_co_idm140067613392240"><h3 class="no_margin">A. J. Muir</h3><p><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</p><div>Find articles by <a href="/pubmed/?term=Muir%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22497812">A. J. Muir</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140067610787856_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140067610787856_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140067610787856_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Duke Clinical Research Institute, Duke University, Durham, NC, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Kirby Institute for Infection and Immunity in Society, University of New South Wales, Sydney, NSW, Australia</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Center for Human Genome Variation, Duke University, Durham, NC</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Brooke Army Medical Center, Fort Sam Houston, TX</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Schering-Plough Corporation, now Merck &amp; Co., Inc., Whitehouse Station, NJ, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Hospital General Universitario Valle de Hebron, Barcelona, Spain</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Indiana University School of Medicine, Indianapolis, IN</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Johns Hopkins University School of Medicine, Baltimore, MD</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Beth Israel Deaconess Medical Center, Harvard University Boston, Boston, MA, USA</div><div id="FN1">Correspondence: Paul Clark, Duke Clinical Research Institute, Duke University Medical Center, PO Box 17969, Durham, NC 27715, USA. <a href="mailto:dev@null" data-email="ude.ekud.md@kralc.luap" class="oemail">ude.ekud.md@kralc.luap</a></div><div id="FN2"><sup>*</sup>These authors contributed equally to the work.</div><div id="FN3"><sup>†</sup>Members of the IDEAL investigators are listed in <a href="#APP1" rid="APP1" class=" app">Appendix 1</a>.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140067610787856_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22497812" target="pmc_ext" ref="reftype=publisher&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">J Viral Hepat</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3518930/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supporting Information: Additional Supporting Information may be found in the online version of this article:<p><strong>Fig. S1</strong> Quartiles of LDL_C by HCV RNA level during treatment at week 2, week 4 and week 12 timepoints.</p><p><strong>Fig. S2</strong> LDL_C (&lt;/&gt;130 mg/dL) by HCV RNA level during treatment at week 2, week 4 and week 12 time-points.</p><p><strong>Table S1:</strong> Measures of linkage disequilibrium between genome wide significant SNPs and rs12980275 by race.</p><p><strong>Table S2:</strong> Multiple regression model for baseline LDL in Caucasians.</p><p><strong>Table S3:</strong> Top GWAS association polymorphisms in association with total cholesterol in Caucasians.</p><p><strong>Table S4:</strong> Top GWAS association polymorphisms in association with serum HDL-cholesterol in Caucasians.</p><p><strong>Table S5:</strong> Top GWAS association polymorphisms in association with serum triglycerides in Caucasisans.</p><p><strong>Table S6:</strong> Multiple regression model for LDL at 24 weeks post treatment for non-SVR patients.</p><p><strong>Table S7:</strong> Multiple regression model for LDL at 24 weeks post treatment for patients who attained SVR.</p><p><strong>Table S8:</strong> Comparison between treatment response at the timepoint LDL was measured.</p><p><strong>Table S9:</strong> Comparison between genotypes within treatment response levels at the timepoint LDL was measured.</p><p><strong>Table S10:</strong> Multiple logistic regression model for SVR.</p><p><strong>Table S11:</strong> Multiple logistic regression model results testing interaction between LDL and IL28B genotype (rs12980275) for SVR.</p><p><strong>Table S12:</strong> Univariate comparison of SVR rates by each <em>IL28B</em> genotype (rs12980275).</p></div><div><a href="/pmc/articles/PMC3518930/bin/NIHMS380108-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS380108-supplement-supplement_1.pdf</a><span style="color:gray"> (31K)</span></div><div class="small guid">GUID:&nbsp;F6313A50-FAC5-47B0-B779-18D6E410DF5D</div></div></dd></dl></div><div id="idm140067615278640" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140067615278640title">Summary</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Low-density lipoprotein cholesterol (LDL-C) levels and interleukin 28B (<em>IL28B</em>) polymorphism are associated with sustained viral response (SVR) to peginterferon/ribavirin (pegIFN/RBV) for chronic hepatitis C (CHC) infection. <em>IL28B</em> has been linked with LDL-C levels using a candidate gene approach, but it is not known whether other genetic variants are associated with LDL-C, nor how these factors definitively affect SVR. We assessed genetic predictors of serum lipid and triglyceride levels in 1604 patients with genotype 1 (G1) chronic hepatitis C virus (HCV) infection by genome-wide association study and developed multivariable predictive models of SVR. <em>IL28B</em> polymorphisms were the only common genetic variants associated with pretreatment LDL-C level in Caucasians (rs12980275, P = 4.7 × 10<sup>−17</sup>, poor response <em>IL28B</em> variants associated with lower LDL-C). The association was dependent on HCV infection, <em>IL28B</em> genotype was no longer associated with LDL-C in SVR patients after treatment, while the association remained significant in non-SVR patients (P &lt; 0.001). LDL-C was significantly associated with SVR for heterozygous <em>IL28B</em> genotype patients (P &lt; 0.001) but not for homozygous genotypes. SVR modelling suggested that <em>IL28B</em> heterozygotes with LDL-C &gt; 130 mg/dL and HCV RNA ≤600 000 IU/mL may anticipate cure rates &gt;80%, while the absence of these two criteria was associated with an SVR rate of &lt;35%. <em>IL28B</em> polymorphisms are the only common genetic variants associated with pretreatment LDL-C in G1-HCV. LDL-C remains significantly associated with SVR for heterozygous <em>IL28B</em> genotype patients, where LDL-C and HCV RNA burden may identify those patients with high or low likelihood of cure with pegIFN/RBV therapy.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">cholesterol, hepatitis C virus, interleukin 28B, lipids, sustained viral response</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">The hepatitis C virus (HCV) is able to co-opt and disrupt host lipid metabolism to facilitate cell entry [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017215">1</a>], assembly [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017243">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017235">3</a>], replication and secretion [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017239">4</a>–<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017225">6</a>]. Lower pretreatment serum low-density lipoprotein cholesterol (LDL-C) levels have been associated with poor response to peginterferon/ribavirin (pegIFN/RBV) therapy for patients with chronic hepatitis C (CHC) [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017248">7</a>–<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017245">9</a>]. This lipid-lowering effect of HCV typically corrects after eradication of the virus, supporting the assertion that these clinical effects are virally mediated [<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017228">7</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017254">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481744507">11</a>].</p><p id="P3">Genome-wide association studies (GWAS) independently identified single nucleotide polymorphisms (SNPs) near the interleukin 28B (<em>IL28B</em>) gene (chromosome 19) that are strongly associated with response to pegIFN/RBV treatment for patients infected with genotype-1 HCV [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017253">12</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017218">14</a>]. The polymorphisms share a haplotype block around the <em>IL28B</em> gene with the causal variant(s) yet to be identified. In a candidate gene approach, <em>IL28B</em> polymorphism rs12979860 was found to be associated with LDL-C levels in genotype 1 (G1) CHC [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017227">15</a>] and has been associated with hepatic steatosis [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017224">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343810190">17</a>]. However, the association of other common genetic variants with lipid levels in HCV has not been tested. Furthermore, the relationship between genetic polymorphism and lipid levels during and after treatment and the interactions with sustained viral response (SVR) prediction have not been explored.</p><p id="P4" class="p p-last">We therefore sought to identify whether any other common genetic variants may contribute to serum lipid and triglyceride levels by assessing whole-genome variation by GWAS in the IDEAL pharmacogenomics cohort [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017241">18</a>]. The large size and well-characterized nature of the cohort enabled us to analyse LDL-C levels during and after treatment to characterize the host–virus interdependence of this association, which has not previously been studied. Finally, we assessed the clinical utility of LDL-C in the prediction of SVR in the light of <em>IL28B</em> genetic associations, modelling specific clinical parameters to help define how LDL-C may be associated with SVR.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Study cohort</h3><p id="P5" class="p p-first">The IDEAL trial was a multi-centre, randomized control trial comparing efficacy and adverse events in 3070 treatment naïve patients with CHC (ClinicalTrials.gov number, <a id="__tag_538503877" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT00081770" ref="reftype=extlink-clinical-trial&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT00081770","term_id":"NCT00081770"}}</span>NCT00081770</a>) and has previously been described [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017252">18</a>]. Patients chronically infected with genotype-1 HCV were randomized to one of three treatment arms: peginterferon alfa-2b at standard dose (1.5 <em>μ</em>g per kilogram body weight per week); or peginterferon alfa-2b at a lower dose (1.0 <em>μ</em>g per kilogram per week), both with ribavirin doses between 800–1400 mg per day; or peginterferon alfa-2a at standard dose (180 <em>μ</em>g/week) and ribavirin dose of 1000–1200 mg/day. Study patients with evaluable GWAS samples were predominantly Caucasian males in their forties with mean BMI 28.2 kg/m<sup>2</sup> [standard deviation (SD) 4.8] mean HCV RNA of 6.36 log<sub>10</sub> IU/mL (SD 0.62) and 11% had advanced fibrosis (METAVIR fibrosis stages 3 or 4) (<a href="/pmc/articles/PMC3518930/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Clinical characteristics of the study population for genome-wide association studies related to serum lipids</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067612732816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left">All</th><th rowspan="1" colspan="1" valign="middle" align="left">Caucasians</th><th rowspan="1" colspan="1" valign="middle" align="left">African Americans</th><th rowspan="1" colspan="1" valign="middle" align="left">Hispanics</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">N</td><td rowspan="1" colspan="1" valign="top" align="left">1319</td><td rowspan="1" colspan="1" valign="top" align="left">1017</td><td rowspan="1" colspan="1" valign="top" align="left">207</td><td rowspan="1" colspan="1" valign="top" align="left">95</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Gender (M/F)</td><td rowspan="1" colspan="1" valign="top" align="left">803/516</td><td rowspan="1" colspan="1" valign="top" align="left">622/395</td><td rowspan="1" colspan="1" valign="top" align="left">121/86</td><td rowspan="1" colspan="1" valign="top" align="left">60/35</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age, years (mean)</td><td rowspan="1" colspan="1" valign="top" align="left">47.4</td><td rowspan="1" colspan="1" valign="top" align="left">47.2</td><td rowspan="1" colspan="1" valign="top" align="left">49.7</td><td rowspan="1" colspan="1" valign="top" align="left">44.3</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI, kg/m<sup>2</sup> (mean)</td><td rowspan="1" colspan="1" valign="top" align="left">28.2</td><td rowspan="1" colspan="1" valign="top" align="left">27.8</td><td rowspan="1" colspan="1" valign="top" align="left">29.7</td><td rowspan="1" colspan="1" valign="top" align="left">29.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Advanced fibrosis (&gt;F2), <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="left">148 (11)</td><td rowspan="1" colspan="1" valign="top" align="left">115 (11)</td><td rowspan="1" colspan="1" valign="top" align="left">20 (10)</td><td rowspan="1" colspan="1" valign="top" align="left">13 (14)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Steatosis (&gt;0%), <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="left">810 (61)</td><td rowspan="1" colspan="1" valign="top" align="left">607 (60)</td><td rowspan="1" colspan="1" valign="top" align="left">134 (65)</td><td rowspan="1" colspan="1" valign="top" align="left">69 (73)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Activity grade (% with grade 2–3)</td><td rowspan="1" colspan="1" valign="top" align="left">82.6</td><td rowspan="1" colspan="1" valign="top" align="left">82.0</td><td rowspan="1" colspan="1" valign="top" align="left">85.5</td><td rowspan="1" colspan="1" valign="top" align="left">83.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Baseline viral load (log<sub>10</sub>) (mean)</td><td rowspan="1" colspan="1" valign="top" align="left">6.36</td><td rowspan="1" colspan="1" valign="top" align="left">6.36</td><td rowspan="1" colspan="1" valign="top" align="left">6.36</td><td rowspan="1" colspan="1" valign="top" align="left">6.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Baseline fasting blood glucose, mm (mean)</td><td rowspan="1" colspan="1" valign="top" align="left">5.3</td><td rowspan="1" colspan="1" valign="top" align="left">5.2</td><td rowspan="1" colspan="1" valign="top" align="left">5.4</td><td rowspan="1" colspan="1" valign="top" align="left">5.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Baseline low-density lipoprotein cholesterol, mg/dL (mean)</td><td rowspan="1" colspan="1" valign="top" align="left">103.2</td><td rowspan="1" colspan="1" valign="top" align="left">104.0</td><td rowspan="1" colspan="1" valign="top" align="left">100.5</td><td rowspan="1" colspan="1" valign="top" align="left">98.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Sustained viral response (%)</td><td rowspan="1" colspan="1" valign="top" align="left">51</td><td rowspan="1" colspan="1" valign="top" align="left">55</td><td rowspan="1" colspan="1" valign="top" align="left">26</td><td rowspan="1" colspan="1" valign="top" align="left">51</td></tr></tbody></table></div><div id="largeobj_idm140067612732816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P6">The IDEAL cohort has previously been analysed for non-genomic, clinical relationships between serum cholesterol, statin use and SVR [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017230">9</a>]. A pharmacogenomics cohort from IDEAL consented to DNA testing (<em>n</em> = 1604) and has been analysed for genetic associations with treatment response [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017216">12</a>] and ribavirin-induced haemolytic anaemia [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017242">19</a>].</p><p id="P7" class="p p-last">From the pharmacogenomics cohort, patients on statin therapy at any time during the study period (<em>n</em> = 46) were excluded from the GWAS analysis to avoid potential confounding. Patients were included if all covariate data were available for the relevant models and GWAS genotyping quality control protocols were satisfied (<em>n</em> = 1319) [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017249">12</a>]. For the SVR analysis in addition to further adjusting for self-declared race rather than genetically inferred ancestry and pegIFN dose received, we considered all patients with covariate data on an intention-to-treat basis, irrespective of treatment compliance, ethnicity or statin therapy (<em>n</em> = 1473).</p></div><div id="S4" class="sec"><h3 id="S4title">Genetic analysis</h3><p id="P8" class="p p-first">Patients were genotyped using the Illumina Human610-quad BeadChip (Illumina, San Diego, CA, USA). After quality control, 97.5% or 565 759 SNPs were analysed with multivariable linear and logistic regression models. The primary association model used single-marker genotype trend tests in three independent ethnic groups (Caucasians, African Americans and Hispanics). Ethnicity was genetically inferred and a modified Eigenstrat method corrected for population substratification [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017223">20</a>].</p><p id="P9" class="p p-last">Clinically important covariates adjusted for in the models included age; gender; body mass index (kg/m<sup>2</sup>); baseline HCV viral load (log<sub>10</sub> IU/mL); fibrosis (binary variable as METAVIR stage F0–2 <em>vs</em> F3–4); inflammation (METAVIR grade 0–1 <em>vs</em> 2–3), alanine transaminase (ALT) values, baseline fasting blood glucose levels and respective genetic ancestry subpopulation (or Eigen) sets. Plasma HCV RNA concentrations were measured using the COBAS Taqman assay (lower limit of quantitation of 27 IU/mL; Roche Diagnostics, Indianapolis, IN, USA).</p></div><div id="S5" class="sec"><h3 id="S5title">Phenotypes</h3><p id="P10" class="p p-first">Fasting levels of total cholesterol (TC), high-density lipoprotein cholesterol (HDL-C), LDL-C and triglycerides (TG) were analysed at baseline through GWAS using a linear regression model. Serum lipid measurements were considered as continuous variables, and LDL-C was log transformed to fit a normal distribution.</p><p id="P11">Serum LDL-C was further analysed by <em>IL28B</em> genotype during treatment at 12 weeks (<em>n</em> = 1385), 24 weeks (<em>n</em> = 1045) and 48 weeks (<em>n</em> = 797), then at 24 weeks follow-up post-treatment (<em>n</em> = 1079). Follow-up LDL-C was analysed based on SVR status, defined as undetectable HCV RNA 24 weeks after treatment completion (or 12 weeks after completion if post-treatment 24 week HCV RNA data were missing).</p><p id="P12" class="p p-last">Multivariable logistic regression models of SVR adjusted for pegIFN alfa-2b dose (low <em>vs</em> standard dose) in addition to the covariates in the GWAS. Patients were considered on an intention to treat basis irrespective of treatment adherence, with ethnicity defined by patient declaration rather than inferred from genotype data as in the GWAS. Patients on statins were reincluded and statin use was considered as a covariate.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Statistical analyses</h3><p id="P13" class="p p-first-last">The Bonferroni method corrected for multiple testing with genome-wide significance predefined at <em>P</em> &lt; 4.4 × 10<sup>−8</sup> (<em>α</em> = 0.05). Student's <em>T</em>-test was used to compare LDL levels between <em>IL28B</em> genotype (good <em>vs</em> poor response genotypes) during treatment and when comparing between SVR and non-SVR patients at follow-up. Linear and logistic regression modelling was conducted with PLINK [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017229">21</a>], STATA (Stata-Corp, College Station, TX, USA) and SAS (SAS Institute, Cary, NC, USA) software. The study met local Institutional Review Board requirements and the 1975 Declaration of Helsinki guidelines.</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Results</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Serum lipid and triglyceride genome-wide association studies</h3><div id="S9" class="sec sec-first"><p></p><h4 id="S9title" class="inline">IL28B polymorphisms are the only common variants with genome-wide significant association with baseline serum low-density lipoprotein cholesterol levels </h4><p id="P14" class="p p-first">Four SNPs demonstrated genome-wide significant association with pretreatment LDL-C (<a href="/pmc/articles/PMC3518930/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). These SNPs are in the region of the <em>IL28B</em> gene on chromosome 19 and are all in linkage disequilibrium with the top SNP (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>), and all have previously been associated with pegIFN/RBV treatment outcome [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017250">12</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017233">14</a>].</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Genome-wide significant associations with pretreatment low-density lipoprotein cholesterol levels</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067611084480" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left">Single nucleotide polymorphism</th><th colspan="4" rowspan="1" valign="bottom" align="left"><em>P</em>-value</th></tr><tr><th colspan="4" rowspan="1" valign="bottom">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Combined (<em>n</em> = 1319)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Caucasians (<em>n</em> = 1017)</th><th rowspan="1" colspan="1" valign="bottom" align="left">African Americans (<em>n</em> = 207)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Hispanics (<em>n</em> = 95)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12980275</td><td rowspan="1" colspan="1" valign="bottom" align="left">4.7 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">6.9 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">0.017</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.63</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12979860</td><td rowspan="1" colspan="1" valign="bottom" align="left">4.9 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">1.3 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">0.014</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.26</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs8099917</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.2 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">1.2 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">0.33</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.086</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">rs12972991</td><td rowspan="1" colspan="1" valign="bottom" align="left">7.0 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">2.0 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">0.21</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.48</td></tr></tbody></table></div><div id="largeobj_idm140067611084480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P15">For the most significant SNP rs12980275, located 2.4 kb downstream from the <em>IL28B</em> gene, the G ‘poor response’ allele was associated with lower serum LDL-C levels across combined ethnic groups (G allele, all ethnicities <em>P</em> = 4.7 × 10<sup>−17</sup>). In subanalysis by ethnicity, the association was significant in Caucasians (<em>n =</em> 1017, P = 6.9 × 10<sup>−16</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>), but not genome-wide significant in the smaller cohorts of African American (<em>n</em> = 207, <em>P</em> = 0.017) and Hispanic patients (<em>n</em> = 95, <em>P</em> = 0.63), although there was a consistent direction of effect. Importantly, no other common genetic variants on genome-wide testing were associated with LDL-C on genome-wide testing.</p><p id="P16">These results likely reflect an <em>IL28B</em> haplotype effect rather than being SNP-specific, given the degree of linkage disequilibrium demonstrated. As the most significant genetic association for LDL-C in the GWAS was from rs12980275, further analysis will focus on this SNP. To minimize confusion in allele description of ‘introducing’ a new <em>IL28B</em> allele, the variant will be discussed in terms of its association with treatment response to pegIFN/RBV as good or poor response. Results will thus be described as ‘good response’ variants corresponding to patients homozygous for the ‘A’ allele at rs12980275 (with equivalent direction of effect for the better known ‘C’ allele at rs12979860; or the ‘T’ allele at rs8099917) and the presence of the G allele (G/G or A/G) corresponding to ‘poor response’.</p><p id="P17" class="p p-last">Low-density lipoprotein cholesterol levels above 130 mg/dL have previously been associated with SVR [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017231">9</a>]. For Caucasians, twice as many patients with the good response <em>IL28B</em> genotype had LDL-C above this clinical threshold than poor response genotypes (A/A <em>vs</em> A/G and G/G, 33.4% <em>vs</em> 14.3%, respectively, <em>P</em> = 9.92 × 10<sup>−6</sup>).</p></div><div id="S10" class="sec"><p></p><h4 id="S10title" class="inline">Baseline serum total cholesterol, high-density lipoprotein cholesterol and triglycerides analyses do not demonstrate genome-wide criteria but identify significant candidate genetic associations </h4><p id="P18" class="p p-first-last">No genome-wide significant associations for other lipids or triglyceride levels were found on GWAS. Several previously identified associations satisfy less stringent candidate gene criteria (results for Caucasians presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Tables S3–S5</a>). For the HDL-C GWAS, the top SNP (rs3764261, P = 5.80 × 10<sup>−7</sup>) was located upstream from the cholesterylester transfer protein (<em>CETP</em>) gene which has previously been linked to HDL-C levels [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017237">22</a>]. For TG, the second ranked SNP was close to the <em>APOA5</em> gene, a gene previously associated with hypertriglyceridemia (rs6589567, P = 4.00 × 10<sup>−6</sup>) [<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017255">23</a>].</p></div><div id="S11" class="sec sec-last"><p></p><h4 id="S11title" class="inline">Genetic association between IL28B genotype and low-density lipoprotein cholesterol is hepatitis C virus-dependent </h4><p id="P19" class="p p-first">Although there were <em>IL28B</em> genotype-based differences in mean LDL-C levels at baseline (P = 6.9 × 10<sup>−16</sup>), during treatment with exogenous peginterferon there were no significant differences in mean serum LDL-C by <em>IL28B</em> genotype (rs12980275; good response genotype <em>vs</em> poor response genotype, <a href="/pmc/articles/PMC3518930/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). We found no differences based on either viral response or <em>IL28B</em> genotype when comparing LDL-C levels by on-treatment milestones [early viral responses (EVRs) <em>vs</em> LDL-C at 12 weeks and end of treatment responses <em>vs</em> week 48 LDL-C level, <a href="#SD1" rid="SD1" class=" supplementary-material">Tables S8 &amp; S9</a>]. However the potentially confounding effect of exogenous interferon [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017246">24</a>] was a limitation for this on-treatment analysis relative to the pretreatment and SVR/non-SVR LDL-C analyses.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3518930/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067616678016" class="figure"><a href="/pmc/articles/PMC3518930/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3518930_nihms380108f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms380108f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3518930/bin/nihms380108f1.jpg"></a></div><div id="largeobj_idm140067616678016" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3518930/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p29">Mean serum low-density lipoprotein (LDL) (mM) at baseline, during treatment and at follow-up by host <em>IL28B</em> genotype [rs12980275 A/A (white) <em>vs</em> non-A/A (grey)] in Caucasians within the IDEAL pharmacogenomics cohort.</p></div></div></div><p id="P20" class="p p-last">For Caucasians, there was a significant <em>IL28B</em> genotype-based difference in mean serum LDL-C at 24 weeks follow-up after treatment in non-SVR patients (<em>P</em> = 0.0002), but in patients who achieved SVR, <em>IL28B</em> was not associated with LDL-C (<em>P</em> = 0.287). This was further tested in a multivariable model of follow-up LDL-C, where formal interaction testing confirmed a significant interaction between treatment outcome and <em>IL28B</em> (P = 0.02 for interaction term) with no significant relationship in SVR patients (<a href="#SD1" rid="SD1" class=" supplementary-material">Tables S6 and S7</a>).</p></div></div><div id="S12" class="sec sec-last"><h3 id="S12title">Defining the role of low-density lipoprotein cholesterol and IL28B in predicting sustained viral response</h3><p id="P21" class="p p-first">Given the recognized genetic association between <em>IL28B</em> polymorphisms and treatment response [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017240">12</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017214">14</a>] and also LDL-C levels and treatment response [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017247">8</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017211">9</a>], we undertook multivariable modelling to more definitively assess the association of pretreatment LDL-C on SVR in this large genotype-informed cohort. We assessed models before and after the inclusion of host <em>IL28B</em> genotype as a covariate and compared each model's performance [<em>r</em><sup>2</sup> and area under the receiver operating curve (ROC)].</p><p id="P22" class="p">A baseline logistic regression model of SVR adjusted for the covariates of LDL-C, HCV RNA (log10 IU/mL), race, age, gender, significant fibrosis (METAVIR &gt;F2), presence of any steatosis (&gt;0%), fasting blood glucose level, alanine aminotransferase (ALT), low dose pegIFN and statin therapy but did <em>not</em> consider <em>IL28B</em> genotype. Higher pretreatment LDL-C was significantly associated with better treatment response [OR 1.16 per 10 mg/dL LDL (95%CI 1.11–1.20); <em>P</em> &lt; 0.0001] with reasonable model performance (<em>r</em><sup>2</sup> 19.2% ROC 0.721), confirming the previous association with SVR [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017236">9</a>]. Including <em>IL28B</em> into the model improved the model's predictive performance (<em>r</em><sup>2</sup> 30.6% ROC 78.3%), with significant interaction between LDL-C and <em>IL28B</em> (<em>P</em>-value for interaction = 0.029), reducing the effect and significance of LDL-C in SVR prediction, which was subsequently explored further.</p><div id="S13" class="sec sec-last"><p></p><h4 id="S13title" class="inline">Low-density lipoprotein cholesterol helps identify heterozygous IL28B genotype patients with high or low chance of sustained viral response </h4><p id="P23" class="p p-first">The significance of the association of LDL-C with SVR is dependent upon <em>IL28B</em> genotype. For heterozygous patients, LDL-C level was a significant predictor of SVR (per 10 mg/dL increase in LDL-C, OR 1.18 (95% CI 1.11–1.25), <em>P</em> &lt; 0.001). In contrast, for patients with homozygous <em>IL28B</em> genotypes LDL-C did not aid in predicting response (<em>IL28B</em> good/poor response genotypes <em>P</em> = 0.097 and <em>P</em> = 0.308, respectively; <a href="#SD1" rid="SD1" class=" supplementary-material">Tables S10–S12</a>).</p><p id="P24">Other recognized clinical and demographic factors continued to be significant in the prediction of SVR after inclusion of <em>IL28B</em> and LDL-C. Significant covariates associated with a better chance of SVR included Caucasian and Hispanic race relative to African American race (OR 2.11, <em>P</em> &lt; 0.001 and OR 2.61, <em>P</em> = 0.008, respectively), fibrosis stage of ≤F2 (OR 2.48, <em>P</em> &lt; 0.001); normal fasting plasma glucose (OR 1.33, <em>P</em> = 0.001) and lower HCV RNA (per 1 log<sub>10</sub> IU/mL decrease, OR 2.15, <em>P</em> &lt; 0001).</p><p id="P25">Because LDL-C remained a significant predictor of SVR for heterozygous <em>IL28B</em> genotype patients, we sought to better define the clinical utility of LDL-C in predicting SVR for this specific patient group. Dichotomizing the cohort into high LDL-C (&gt;130 mg/dL) and low LDL-C (≤130 mg/dL), we considered a number of treatment endpoints. While there was no difference in rapid viral response, significantly higher rates of EVR, end of treatment responses (EOTR), and as expected SVR were seen in patients with LDL &gt;130 mg/dL, while relapse rates were significantly lower (<a href="/pmc/articles/PMC3518930/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Baseline LDL-C either as quartiles or at the 130 mg/dL cut-off point differentiated HCV viraemia across HCV RNA measurement timepoints (treatment weeks 2, 4, 12, <a href="#SD1" rid="SD1" class=" supplementary-material">Figures S1 &amp; S2</a>). Importantly, however, the LDL-C cut-off did not help to discriminate eventual SVR responses in slow responder patients with partial EVR (&gt;2 log IU/mL reduction in HCV RNA but remaining HCV RNA positive after 12 weeks of therapy) and thus is unlikely to be useful to identify a treatment futility rule.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Univariate analysis of on-treatment, end of treatment and sustained viral responses and relapse rates compared by baseline low-density lipoprotein cholesterol (LDL-C) thresholds in <em>IL28B</em> heterozygous patients</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067616327104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Week 4 response (RVR)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Week 12 response (cEVR)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Week 48 response (EOTR)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Week 24 follow-up response (SVR)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Relapse</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL-C &gt; 130 mg/dL</td><td rowspan="1" colspan="1" valign="middle" align="left">7.14% (7/98)</td><td rowspan="1" colspan="1" valign="middle" align="left">55.4% (51/92)</td><td rowspan="1" colspan="1" valign="middle" align="left">74.1% (60/81)</td><td rowspan="1" colspan="1" valign="middle" align="left">47.1% (49/104)</td><td rowspan="1" colspan="1" valign="middle" align="left">18.3% (11/49)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">LDL-C ≤ 130 mg/dL</td><td rowspan="1" colspan="1" valign="middle" align="left">5.12% (32/625)</td><td rowspan="1" colspan="1" valign="middle" align="left">33.4% (204/611)</td><td rowspan="1" colspan="1" valign="middle" align="left">47.6% (267/581)</td><td rowspan="1" colspan="1" valign="middle" align="left">28.7% (185/644)</td><td rowspan="1" colspan="1" valign="middle" align="left">32.6% (87/267)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Chi-square test <em>P</em>-value</td><td rowspan="1" colspan="1" valign="middle" align="left">0.5594</td><td rowspan="1" colspan="1" valign="middle" align="left">&lt;0.0001<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="left">&lt;0.0001</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0002</td><td rowspan="1" colspan="1" valign="middle" align="left">0.0432</td></tr></tbody></table></div><div id="largeobj_idm140067616327104" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p31" class="p p-first-last">RVR, rapid viral response; EOTR, end of treatment responses; SVR, sustained viral response; EVR, early viral response; pEVR, partial EVR; cEVR, HCV RNA negative after 12 weeks of therapy.</p></div><div id="TFN2"><sup>*</sup>The test considers all three levels of EVR (cEVR, pEVR, and no EVR).</div></div></div><p id="P26">We further assessed SVR rates observed in the trial compared across a number of important clinical predictors of SVR (<a href="/pmc/articles/PMC3518930/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). LDL-C and high/low HCV RNA burden (≤600 000 IU/mL) were key factors. SVR decreased from 71% when comparing patients with high LDL-C and low HCV RNA, to 24% in patients with neither of these factors. In univariate analysis of the trial data, the high/low LDL-C threshold helped to differentiate SVR rates when considered across a number of other clinical factors such as significant fibrosis (&gt;F2 METAVIR stage), African American ethnicity and abnormal fasting blood glucose.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Univariate analysis of sustained viral response (SVR) rates observed from the IDEAL pharmacogenomics cohort for <em>IL28B</em> heterozygous patients with high or low low-density lipoprotein cholesterol (LDL-C) (≤/&gt;130 mg/dL) compared with other prognostic factors associated with SVR</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067613918896" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Clinical factor</th><th rowspan="1" colspan="1" valign="bottom" align="left">LDL-C &gt; 130 mg/dL SVR % (<em>n</em>/N)</th><th rowspan="1" colspan="1" valign="bottom" align="left">LDL-C ≤ 130 mg/dL SVR % (<em>n</em>/N)</th><th rowspan="1" colspan="1" valign="bottom" align="left"><em>P</em>-value<sup>*</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">HCV RNA ≤600 000 IU/mL</td><td rowspan="1" colspan="1" valign="middle" align="left">71% (12/17)</td><td rowspan="1" colspan="1" valign="middle" align="left">53% (59/112)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00014</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">HCV RNA &gt;600 000 IU/mL</td><td rowspan="1" colspan="1" valign="middle" align="left">43% 37/87</td><td rowspan="1" colspan="1" valign="middle" align="left">24% (126/532)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">F0–F2 METAVIR fibrosis</td><td rowspan="1" colspan="1" valign="middle" align="left">47.9% (45/94)</td><td rowspan="1" colspan="1" valign="middle" align="left">30.6% (175/571)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00033</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">F3–F4 METAVIR fibrosis</td><td rowspan="1" colspan="1" valign="middle" align="left">40.0% (4/10)</td><td rowspan="1" colspan="1" valign="middle" align="left">13.7% (10/73)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Non-African American</td><td rowspan="1" colspan="1" valign="middle" align="left">53.8% (42/78)</td><td rowspan="1" colspan="1" valign="middle" align="left">30.6% (165/533)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00008</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">African American</td><td rowspan="1" colspan="1" valign="middle" align="left">26.9% (7/26)</td><td rowspan="1" colspan="1" valign="middle" align="left">18.0% (10/73)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Normal fasting blood glucose</td><td rowspan="1" colspan="1" valign="middle" align="left">50.0% (43/86)</td><td rowspan="1" colspan="1" valign="middle" align="left">31.5% (165/524)</td><td rowspan="1" colspan="1" valign="middle" align="left">0.00030</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Abnormal fasting blood glucose</td><td rowspan="1" colspan="1" valign="middle" align="left">33.3% (6/18)</td><td rowspan="1" colspan="1" valign="middle" align="left">16.7% (20/120)</td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140067613918896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>Cochran–Mantel–Haenszel test compares SVR rates for high <em>vs</em> low LDL-C after adjusting for hepatitis C virus (HCV) RNA level, METAVIR fibrosis, race, abnormal fasting glucose.</div></div></div><p id="P27" class="p p-last">Finally, we sought to consider these variables together in a multivariable model using binary clinical thresholds as might be carried out in clinic. For model parsimony, we assumed no hepatic steatosis and normal glucose. This model confirmed that for <em>IL28B</em> heterozygous patients, the LDL-C threshold helped to differentiate the predicted treatment responses across the continuum of clinical and racial factors associated with poor response (<em>R</em><sup>2</sup> = 0.261, ROC = 76.0%) (<a href="/pmc/articles/PMC3518930/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Patients with high LDL-C, low viral load, without advanced fibrosis (≤F2) and non-African American ethnicity had predicted SVR rates of 81%, while for patients without any of these positive treatment prognostic factors anticipated SVR is below 10%.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3518930/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140067616674352" class="figure"><a href="/pmc/articles/PMC3518930/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3518930_nihms380108f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms380108f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3518930/bin/nihms380108f2.jpg"></a></div><div id="largeobj_idm140067616674352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3518930/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3518930/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p30">Low-density lipoprotein cholesterol (LDL-C) level above 130 mg/dL (light grey line) or below 130 mg/dL (dark grey line) differentiates sustained viral response (SVR) predication for <em>IL28B</em> heterozygous patients across the spectrum of adverse clinical predicators including high viral load [hepatitis C virus (HCV) RNA &gt;600 000 IU/mL], advanced fibrosis (&gt;METAVIR F2) and African American race in a logistic regression model of SVR.</p></div></div></div></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Discussion</h2><p id="P28" class="p p-first">Altered lipid levels in patients infected with HCV have been thought primarily due to disruption of host lipid synthetic pathways mediated by viral factors such as the HCV core [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017213">25</a>] and nonstructural (e.g. NS5A) [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017244">26</a>] proteins. Through a genome-wide association approach, we have demonstrated host genetics are also critical. <em>IL28B</em> polymorphisms are the only common genetic variants significantly associated with host serum LDL-C levels in genotype-1 HCV infection. We thus validate the previously reported candidate gene association of <em>IL28B</em> polymorphism with LDL-C levels [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017232">15</a>] and are able to extend this, demonstrating for the first time that there are no other common variants associated with LDL-C in genotype-1 HCV infection.</p><p id="P29">In this retrospective analysis, we have demonstrated that LDL-C's association with SVR is conditional upon <em>IL28B</em> genotype. LDL-C has predictive utility for patients with heterozygous <em>IL28B</em> genotype only and is not associated with SVR for patients homozygous for either the good or poor response <em>IL28B</em> alleles. In a multivariable model of SVR in <em>IL28B</em> heterozygous patients, established cut-offs for LDL (&gt;130 mg/dL) and HCV RNA (≤600 000 IU/mL) provided a clear distinction of patients with high or low predicted SVR rate, independently of fibrosis and race.</p><p id="P30">Heterozygous <em>IL28B</em> genotype is the most common <em>IL28B</em> genotype in Caucasians. The heterozygous <em>IL28B</em> genotype is less clear in its predictive discrimination of SVR compared with the homozygous genotypes. Our data suggest patients with high LDL-C, low viral load and heterozygous <em>IL28B</em> genotype may anticipate much higher rates of SVR with existing pegIFN/RBV therapy. This distinction may be important for planning regimens for patients at more risk of toxicity or in treatment settings where direct acting antivirals (DAAs) may not be available. For the majority of heterozygous patients with low LDL-C and/or high HCV RNA, expected treatment responses to pegIFN/RBV are below the anticipated SVR rates seen with regimens including DAAs. Further prospective testing is necessary to validate the association of SVR with LDL-C and HCV RNA level for pegIFN/RBV regimens and to explore its use in DAA-inclusive regimens. <em>IL28B</em> may continue to play a significant, though attenuated role in DAA-inclusive peg-IFN/RBV regimens [<a href="#R27" rid="R27" class=" bibr popnode">27</a>,<a href="#R28" rid="R28" class=" bibr popnode">28</a>]. To define the clinical utility of LDL-C, our SVR modelling concentrated on <em>IL28B</em> heterozygous genotype patients based on data from a treatment naive cohort; however, more extensive modelling of SVR in larger cohorts including patients with more advanced fibrosis or prior nonresponse is needed, ideally prospectively.</p><p id="P31">In the GWAS analysis of other lipid measurements, no novel variants met genome-wide significance criteria; however, two polymorphisms implicated in previous genetic studies satisfied candidate gene criteria for significance. HDL-C was significantly associated with <em>CETP</em> variants and genetic variation near the <em>APOA5</em> gene with hypertriglyceridaemia [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017234">22</a>,<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017222">23</a>]. This demonstrates that common genetic variants important in non-HCV-infected populations may also have important effects on HDL and TG for patients with HCV.</p><p id="P32">Our data suggest that <em>IL28B</em> polymorphisms influence the biological associations between HCV infection and serum LDL-C, and the clinical utility of LDL-C for predicting treatment response to pegIFN/RBV therapy. Exogenous IFN therapy has been previously observed to decrease serum lipid levels, which may occur via reduced activities of lipoprotein lipase (LPL) and hepatic triglyceride lipase (HTGL) [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017212">24</a>]. This may account for the lower LDL-C levels seen during therapy irrespective of host genotype or on-treatment responses. In contrast, LDL-C levels differ in HCV-infected <em>vs</em> cured patients, suggesting a direct role for the virus. Furthermore, in HCV-infected patients (whether pretreatment or non-SVR post-treatment) the effect of the virus on LDL-C level differs significantly by host IL28B genotype. The underlying mechanism of how <em>IL28B</em> genetic variation is associated with treatment-induced and spontaneous HCV clearance remains unclear, although it is presumed to be immune related. The viral dependency of the association between host genetics and LDL-C and the inverse correlation between HCV RNA levels and LDL-C (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S1 &amp; S2</a>), suggests LDL-C is a marker of HCV interference in lipid metabolism, mediated by host genetics.</p><p id="P33" class="p p-last">Increased expression of interferon-stimulated genes (ISGs) provides a possible functional explanation and weak correlation has previously been found between ISG expression and LDL-C levels. Higher levels of ISG expression pretreatment have been associated with nonresponse to treatment [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017220">29</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017219">30</a>] and have also been correlated with poor response <em>IL28B</em> genotype [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017238">31</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343810191">32</a>], although a recent study has suggested these factors may work independently [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289017226">33</a>]. Unfortunately, ISG expression data were not available in this clinical trial cohort. The functional mechanisms underlying these genetic associations remain undetermined. Further molecular and clinical research is required to understand how host genotype underlies the complex interaction between the HCV, the host immune system and lipid metabolism.</p></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Conclusions</h2><p id="P34" class="p p-first-last"><em>IL28B</em> polymorphisms are the only common variants associated with LDL-C in G1-HCV. This association is HCV-dependent and was no longer significant after SVR. The clinical utility of serum LDL-C for predicting SVR is apparent only for patients with heterozygous <em>IL28B</em> genotype. In multivariable modelling of SVR, LDL-C level was significant for heterozygous <em>IL28B</em> patients and may potentially allow better prognostication of treatment response. Established thresholds for LDL-C and HCV RNA levels help to better identify heterozygous <em>IL28B</em> genotype patients with high or low anticipated SVR rates to pegIFN/RBV therapy and may help to better individualize care.</p></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><p id="P35">Supporting Information: Additional Supporting Information may be found in the online version of this article:</p><p id="P36"><strong>Fig. S1</strong> Quartiles of LDL_C by HCV RNA level during treatment at week 2, week 4 and week 12 timepoints.</p><p id="P37"><strong>Fig. S2</strong> LDL_C (&lt;/&gt;130 mg/dL) by HCV RNA level during treatment at week 2, week 4 and week 12 time-points.</p><p id="P38"><strong>Table S1:</strong> Measures of linkage disequilibrium between genome wide significant SNPs and rs12980275 by race.</p><p id="P39"><strong>Table S2:</strong> Multiple regression model for baseline LDL in Caucasians.</p><p id="P40"><strong>Table S3:</strong> Top GWAS association polymorphisms in association with total cholesterol in Caucasians.</p><p id="P41"><strong>Table S4:</strong> Top GWAS association polymorphisms in association with serum HDL-cholesterol in Caucasians.</p><p id="P42"><strong>Table S5:</strong> Top GWAS association polymorphisms in association with serum triglycerides in Caucasisans.</p><p id="P43"><strong>Table S6:</strong> Multiple regression model for LDL at 24 weeks post treatment for non-SVR patients.</p><p id="P44"><strong>Table S7:</strong> Multiple regression model for LDL at 24 weeks post treatment for patients who attained SVR.</p><p id="P45"><strong>Table S8:</strong> Comparison between treatment response at the timepoint LDL was measured.</p><p id="P46"><strong>Table S9:</strong> Comparison between genotypes within treatment response levels at the timepoint LDL was measured.</p><p id="P47"><strong>Table S10:</strong> Multiple logistic regression model for SVR.</p><p id="P48"><strong>Table S11:</strong> Multiple logistic regression model results testing interaction between LDL and IL28B genotype (rs12980275) for SVR.</p><p id="P49"><strong>Table S12:</strong> Univariate comparison of SVR rates by each <em>IL28B</em> genotype (rs12980275).</p><div class="sup-box half_rhythm" id="d35e1080"><a href="/pmc/articles/PMC3518930/bin/NIHMS380108-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(31K, pdf)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p3">The authors would like to thank the patients in the IDEAL trial as well as the principal investigators, study coordinators and nurses involved. The authors thank Megan Koehler, DCRI for her statistical review.</p><p id="__p4">Funding: The study was funded by Schering-Plough Research Institute, Kenilworth, NJ (now Merck and Co.). Drs Clark and Thompson received funding support from the Duke Clinical Research Institute, the Richard Boebel Family Fund, the National Health and Medical Research Council of Australia (PJC: APP1017139) and the Gastroenterological Society of Australia. Dr Clark received funding from the National Centre in HIV Epidemiology and Clinical Research (now The Kirby Institute for Infection and Immunity in Society), University of New South Wales, Australia, and the AASLD/LIFER Clinical and Translational Research Fellowship in Liver Diseases Award. Dr Thompson received funding from the Royal Australian College of Physicians.</p></div></div><div id="GL" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="GLtitle">Abbreviations</h2><div class="bk-sec"><div><table class="default-table glossary"><tbody><tr><td id="G1">ALT</td><td>alanine aminotransferase</td></tr><tr><td id="G2"><em>CETP</em></td><td>cholesterylester transfer protein</td></tr><tr><td id="G3">CHC</td><td>chronic hepatitis C</td></tr><tr><td id="G4">DAA</td><td>direct acting antiviral</td></tr><tr><td id="G5">EOTR</td><td>end of treatment responses</td></tr><tr><td id="G6">EVR</td><td>early viral response</td></tr><tr><td id="G7">G1</td><td>genotype 1</td></tr><tr><td id="G8">GWAS</td><td>genome-wide association studies</td></tr><tr><td id="G9">HCV</td><td>hepatitis C virus</td></tr><tr><td id="G10">HDL-C</td><td>high-density lipoprotein cholesterol</td></tr><tr><td id="G11">HTGL</td><td>hepatic triglyceride lipase</td></tr><tr><td id="G12"><em>IL28B</em></td><td>interleukin 28B</td></tr><tr><td id="G13">ISG</td><td>interferon-stimulated gene</td></tr><tr><td id="G14">LDL-C</td><td>low-density lipoprotein cholesterol</td></tr><tr><td id="G15">LPL</td><td>lipoprotein lipase</td></tr><tr><td id="G16">pegIFN/RBV</td><td>peginterferon/ribavirin</td></tr><tr><td id="G17">ROC</td><td>receiver operating curve</td></tr><tr><td id="G18">SD</td><td>standard deviation</td></tr><tr><td id="G19">SNP</td><td>single nucleotide polymorphisms</td></tr><tr><td id="G20">SVR</td><td>sustained viral response</td></tr><tr><td id="G21">TC</td><td>total cholesterol</td></tr><tr><td id="G22">TG</td><td>triglycerides</td></tr></tbody></table></div></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">Appendix 1</h2><!--/article/back/app-group/app/--><p id="P50">Members of the IDEAL investigators: Ira Jacobson, Weill Cornell Medical College, New York, NY, USA; Fred Poor-dad, Cedars-Sinai Medical Center, Los Angeles, CA, USA; Eric Lawitz, Alamo Medical Research, San Antonio, TX, USA; Jonathan McCone, Mt. Vernon Endoscopy Center, Alexandria, VA, USA; Mitchell L. Shiffman, Virginia Commonwealth University, Richmond, VA, USA; Greg W. Galler, Kelsey Research Foundation, Houston, TX, USA; William M. Lee, University of Texas Southwestern Medical Center, Dallas, TX, USA, Robert Reindollar, Piedmont Healthcare, Statesville, NC, USA; John King, Louisiana State University, Shreveport, LA, USA; Reem Ghalib, The Liver Institute at Methodist Dallas Medical Center, Dallas, TX, USA; Bradley Freilich, Kansas City Gastroenterology and Hepatology, Kansas City, MO, USA; Lisa M. Nyberg, Kaiser Permanente, San Diego, CA, USA; Zachary Goodman, Armed Forces Institute of Pathology, Washington, DC, USA; Navdeep Boparai, Kenneth Koury, Clifford A. Brass, Schering-Plough Corporation, now Merck &amp; Co., Inc., Whitehouse Station, NJ, USA.</p></div><div id="idm140067614439024" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140067614439024title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"> </p><p id="__p5" class="p p-first-last">Conflict of Interest: The authors disclose the following, Drs Thompson, Sulkowski, Afdhal, McHutchison and Muir report having received research and grant support from Schering-Plough (now Merck); Drs Afdhal, Harrison, McHutchison, Goldstein and Muir have received consulting fees or acted in an advisory capacity for Schering-Plough or Merck. Drs Albrecht and Pedicone are employees of Schering-Plough (now Merck &amp; Co.) and are stockholders in this entity. Dr McHutchison is now employed by Gilead Sciences. Dr Noviello is a former employee of Schering-Plough and is now a consultant to Merck &amp; Co. Drs Thompson, Ge, Urban, McHutchison and Goldstein are co-inventors of a patent application based on the <em>IL28B</em> finding.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p6" class="p p-first-last">Please note: Wiley-Blackwell are not responsible for the content or functionality of any supporting materials supplied by the authors. Any queries (other than missing material) should bedirected to the corresponding author for the article.</p><p></p></div></div><div id="idm140067604080272" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140067604080272title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Agnello V, Ábel G, Elfahal M, Knight GB, Zhang QX. Hepatitis C virus and other Flaviviridae viruses enter cells via low density lipoprotein receptor. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1999;<span class="ref-vol">96</span>(22):12766–12771.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC23090/">PMC free article</a>]</span> [<a href="/pubmed/10535997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Hepatitis+C+virus+and+other+Flaviviridae+viruses+enter+cells+via+low+density+lipoprotein+receptor&amp;author=V+Agnello&amp;author=G+Ábel&amp;author=M+Elfahal&amp;author=GB+Knight&amp;author=QX+Zhang&amp;volume=96&amp;issue=22&amp;publication_year=1999&amp;pages=12766-12771&amp;pmid=10535997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Boulant S, Montserret R, Hope RG, et al.  Structural determinants that target the hepatitis c virus core protein to lipid droplets. <span><span class="ref-journal">J Biol Chem. </span>2006;<span class="ref-vol">281</span>(31):22236–22247.</span> [<a href="/pubmed/16704979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Structural+determinants+that+target+the+hepatitis+c+virus+core+protein+to+lipid+droplets&amp;author=S+Boulant&amp;author=R+Montserret&amp;author=RG+Hope&amp;volume=281&amp;issue=31&amp;publication_year=2006&amp;pages=22236-22247&amp;pmid=16704979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Miyanari Y, Atsuzawa K, Usuda N, et al.  The lipid droplet is an important organelle for hepatitis C virus production. <span><span class="ref-journal">Nat Cell Biol. </span>2007;<span class="ref-vol">9</span>(9):1089–1097.</span> [<a href="/pubmed/17721513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=The+lipid+droplet+is+an+important+organelle+for+hepatitis+C+virus+production&amp;author=Y+Miyanari&amp;author=K+Atsuzawa&amp;author=N+Usuda&amp;volume=9&amp;issue=9&amp;publication_year=2007&amp;pages=1089-1097&amp;pmid=17721513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gastaminza P, Cheng G, Wieland S, Zhong J, Liao W, Chisari FV. Cellular determinants of hepatitis C virus assembly, maturation, degradation, and secretion. <span><span class="ref-journal">J Virol. </span>2008;<span class="ref-vol">82</span>(5):2120–2129.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2258938/">PMC free article</a>]</span> [<a href="/pubmed/18077707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Virol&amp;title=Cellular+determinants+of+hepatitis+C+virus+assembly,+maturation,+degradation,+and+secretion&amp;author=P+Gastaminza&amp;author=G+Cheng&amp;author=S+Wieland&amp;author=J+Zhong&amp;author=W+Liao&amp;volume=82&amp;issue=5&amp;publication_year=2008&amp;pages=2120-2129&amp;pmid=18077707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Barth H, Liang TJ, Baumert TF. Hepatitis C virus entry: molecular biology and clinical implications. <span><span class="ref-journal">Hepatology. </span>2006;<span class="ref-vol">44</span>(3):527–535.</span> [<a href="/pubmed/16941688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Hepatitis+C+virus+entry:+molecular+biology+and+clinical+implications&amp;author=H+Barth&amp;author=TJ+Liang&amp;author=TF+Baumert&amp;volume=44&amp;issue=3&amp;publication_year=2006&amp;pages=527-535&amp;pmid=16941688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Syed GH, Amako Y, Siddiqui A. Hepatitis C virus hijacks host lipid metabolism. <span><span class="ref-journal">Trends Endocrinol Metab. </span>2010;<span class="ref-vol">21</span>(1):33–40.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2818172/">PMC free article</a>]</span> [<a href="/pubmed/19854061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Endocrinol+Metab&amp;title=Hepatitis+C+virus+hijacks+host+lipid+metabolism&amp;author=GH+Syed&amp;author=Y+Amako&amp;author=A+Siddiqui&amp;volume=21&amp;issue=1&amp;publication_year=2010&amp;pages=33-40&amp;pmid=19854061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Poynard T, Ratziu V, McHutchison J, et al.  Effect of treatment with peginterferon or interferon alfa-2b and ribavirin on steatosis in patients infected with hepatitis C. <span><span class="ref-journal">Hepatology. </span>2003;<span class="ref-vol">38</span>(1):75–85.</span> [<a href="/pubmed/12829989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Effect+of+treatment+with+peginterferon+or+interferon+alfa-2b+and+ribavirin+on+steatosis+in+patients+infected+with+hepatitis+C&amp;author=T+Poynard&amp;author=V+Ratziu&amp;author=J+McHutchison&amp;volume=38&amp;issue=1&amp;publication_year=2003&amp;pages=75-85&amp;pmid=12829989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Ramcharran D, Wahed AS, Conjeevaram HS, et al.  Associations between serum lipids and hepatitis C antiviral treatment efficacy. <span><span class="ref-journal">Hepatology. </span>2010;<span class="ref-vol">52</span>(3):854–863.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2938827/">PMC free article</a>]</span> [<a href="/pubmed/20690192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Associations+between+serum+lipids+and+hepatitis+C+antiviral+treatment+efficacy&amp;author=D+Ramcharran&amp;author=AS+Wahed&amp;author=HS+Conjeevaram&amp;volume=52&amp;issue=3&amp;publication_year=2010&amp;pages=854-863&amp;pmid=20690192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Harrison SA, Rossaro L, Hu KQ, et al.  Serum cholesterol and statin use predict virological response to peginterferon and ribavirin therapy. <span><span class="ref-journal">Hepatology. </span>2010;<span class="ref-vol">52</span>(3):864–874.</span> [<a href="/pubmed/20568303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Serum+cholesterol+and+statin+use+predict+virological+response+to+peginterferon+and+ribavirin+therapy&amp;author=SA+Harrison&amp;author=L+Rossaro&amp;author=KQ+Hu&amp;volume=52&amp;issue=3&amp;publication_year=2010&amp;pages=864-874&amp;pmid=20568303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Serfaty L, Andreani T, Giral P, Carbonell N, Chazouilleres O, Poupon R. Hepatitis C virus induced hypobeta-lipoproteinemia: a possible mechanism for steatosis in chronic hepatitis C. <span><span class="ref-journal">J Hepatol. </span>2001;<span class="ref-vol">34</span>(3):428–434.</span> [<a href="/pubmed/11322205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Hepatitis+C+virus+induced+hypobeta-lipoproteinemia:+a+possible+mechanism+for+steatosis+in+chronic+hepatitis+C&amp;author=L+Serfaty&amp;author=T+Andreani&amp;author=P+Giral&amp;author=N+Carbonell&amp;author=O+Chazouilleres&amp;volume=34&amp;issue=3&amp;publication_year=2001&amp;pages=428-434&amp;pmid=11322205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Corey KE, Kane E, Munroe C, Barlow LL, Zheng H, Chung RT. Hepatitis C virus infection and its clearance alter circulating lipids: implications for long-term follow-up. <span><span class="ref-journal">Hepatology. </span>2009;<span class="ref-vol">50</span>(4):1030–1037.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4330996/">PMC free article</a>]</span> [<a href="/pubmed/19787818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Hepatitis+C+virus+infection+and+its+clearance+alter+circulating+lipids:+implications+for+long-term+follow-up&amp;author=KE+Corey&amp;author=E+Kane&amp;author=C+Munroe&amp;author=LL+Barlow&amp;author=H+Zheng&amp;volume=50&amp;issue=4&amp;publication_year=2009&amp;pages=1030-1037&amp;pmid=19787818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Ge D, Fellay J, Thompson AJ, et al.  Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>(7262):399–401.</span> [<a href="/pubmed/19684573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+variation+in+IL28B+predicts+hepatitis+C+treatment-induced+viral+clearance&amp;author=D+Ge&amp;author=J+Fellay&amp;author=AJ+Thompson&amp;volume=461&amp;issue=7262&amp;publication_year=2009&amp;pages=399-401&amp;pmid=19684573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Tanaka Y, Nishida N, Sugiyama M, et al.  Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>(10):1105–1109.</span> [<a href="/pubmed/19749757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+of+IL28B+with+response+to+pegylated+interferon-alpha+and+ribavirin+therapy+for+chronic+hepatitis+C&amp;author=Y+Tanaka&amp;author=N+Nishida&amp;author=M+Sugiyama&amp;volume=41&amp;issue=10&amp;publication_year=2009&amp;pages=1105-1109&amp;pmid=19749757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Suppiah V, Moldovan M, Ahlenstiel G, et al.  IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>(10):1100–1104.</span> [<a href="/pubmed/19749758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=IL28B+is+associated+with+response+to+chronic+hepatitis+C+interferon-alpha+and+ribavirin+therapy&amp;author=V+Suppiah&amp;author=M+Moldovan&amp;author=G+Ahlenstiel&amp;volume=41&amp;issue=10&amp;publication_year=2009&amp;pages=1100-1104&amp;pmid=19749758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Li JH, Lao XQ, Tillmann HL, et al.  Interferon-lambda genotype and low serum low-density lipoprotein cholesterol levels in patients with chronic hepatitis C infection. <span><span class="ref-journal">Hepatology. </span>2010;<span class="ref-vol">51</span>(6):1904–1911.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2921623/">PMC free article</a>]</span> [<a href="/pubmed/20235331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Interferon-lambda+genotype+and+low+serum+low-density+lipoprotein+cholesterol+levels+in+patients+with+chronic+hepatitis+C+infection&amp;author=JH+Li&amp;author=XQ+Lao&amp;author=HL+Tillmann&amp;volume=51&amp;issue=6&amp;publication_year=2010&amp;pages=1904-1911&amp;pmid=20235331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Cai T, Dufour JF, Muellhaupt B, et al.  Viral genotype-specific role of PNPLA3, PPARG, MTTP and IL28B in hepatitis C virus-associated steatosis. <span><span class="ref-journal">J Hepatol. </span>2011;<span class="ref-vol">55</span>(3):529–535.</span> [<a href="/pubmed/21236304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Viral+genotype-specific+role+of+PNPLA3,+PPARG,+MTTP+and+IL28B+in+hepatitis+C+virus-associated+steatosis&amp;author=T+Cai&amp;author=JF+Dufour&amp;author=B+Muellhaupt&amp;volume=55&amp;issue=3&amp;publication_year=2011&amp;pages=529-535&amp;pmid=21236304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Tillmann HL, Patel K, Muir AJ, et al.  Beneficial IL28B genotype associated with lower frequency of hepatic steatosis in patients with chronic hepatitis C. <span><span class="ref-journal">J Hepatol. </span>2011;<span class="ref-vol">55</span>(6):1195–1200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3763913/">PMC free article</a>]</span> [<a href="/pubmed/21703198" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Beneficial+IL28B+genotype+associated+with+lower+frequency+of+hepatic+steatosis+in+patients+with+chronic+hepatitis+C&amp;author=HL+Tillmann&amp;author=K+Patel&amp;author=AJ+Muir&amp;volume=55&amp;issue=6&amp;publication_year=2011&amp;pages=1195-1200&amp;pmid=21703198&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">McHutchison JG, Lawitz EJ, Shiffman ML, et al.  Peginterferon alfa-2b or alfa-2a with ribavirin for treatment of hepatitis C infection. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">361</span>(6):580–593.</span> [<a href="/pubmed/19625712" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Peginterferon+alfa-2b+or+alfa-2a+with+ribavirin+for+treatment+of+hepatitis+C+infection&amp;author=JG+McHutchison&amp;author=EJ+Lawitz&amp;author=ML+Shiffman&amp;volume=361&amp;issue=6&amp;publication_year=2009&amp;pages=580-593&amp;pmid=19625712&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Fellay J, Thompson AJ, Ge D, et al.  ITPA gene variants protect against anaemia in patients treated for chronic hepatitis C. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>(7287):405–408.</span> [<a href="/pubmed/20173735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=ITPA+gene+variants+protect+against+anaemia+in+patients+treated+for+chronic+hepatitis+C&amp;author=J+Fellay&amp;author=AJ+Thompson&amp;author=D+Ge&amp;volume=464&amp;issue=7287&amp;publication_year=2010&amp;pages=405-408&amp;pmid=20173735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(8):904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Brown ML, Inazu A, Hesler CB, et al.  Molecular basis of lipid transfer protein deficiency in a family with increased high-density lipoproteins. <span><span class="ref-journal">Nature. </span>1989;<span class="ref-vol">342</span>(6248):448–451.</span> [<a href="/pubmed/2586614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Molecular+basis+of+lipid+transfer+protein+deficiency+in+a+family+with+increased+high-density+lipoproteins&amp;author=ML+Brown&amp;author=A+Inazu&amp;author=CB+Hesler&amp;volume=342&amp;issue=6248&amp;publication_year=1989&amp;pages=448-451&amp;pmid=2586614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Pennacchio LA, Olivier M, Hubacek JA, et al.  An apolipoprotein influencing triglycerides in humans and mice revealed by comparative sequencing. <span><span class="ref-journal">Science. </span>2001;<span class="ref-vol">294</span>(5540):169–173.</span> [<a href="/pubmed/11588264" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An+apolipoprotein+influencing+triglycerides+in+humans+and+mice+revealed+by+comparative+sequencing&amp;author=LA+Pennacchio&amp;author=M+Olivier&amp;author=JA+Hubacek&amp;volume=294&amp;issue=5540&amp;publication_year=2001&amp;pages=169-173&amp;pmid=11588264&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Shinohara E, Yamashita S, Kihara S, et al.  Interferon alpha induces disorder of lipid metabolism by lowering postheparin lipases and cholesteryl ester transfer protein activities in patients with chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>1997;<span class="ref-vol">25</span>(6):1502–1506.</span> [<a href="/pubmed/9185775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=Interferon+alpha+induces+disorder+of+lipid+metabolism+by+lowering+postheparin+lipases+and+cholesteryl+ester+transfer+protein+activities+in+patients+with+chronic+hepatitis+C&amp;author=E+Shinohara&amp;author=S+Yamashita&amp;author=S+Kihara&amp;volume=25&amp;issue=6&amp;publication_year=1997&amp;pages=1502-1506&amp;pmid=9185775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Moriishi K, Mochizuki R, Moriya K, et al.  Critical role of PA28<em>γ</em> in hepatitis C virus-associated steatogenesis and hepatocarcinogenesis. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2007;<span class="ref-vol">104</span>(5):1661–1666.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1776165/">PMC free article</a>]</span> [<a href="/pubmed/17234812" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Critical+role+of+PA28γ+in+hepatitis+C+virus-associated+steatogenesis+and+hepatocarcinogenesis&amp;author=K+Moriishi&amp;author=R+Mochizuki&amp;author=K+Moriya&amp;volume=104&amp;issue=5&amp;publication_year=2007&amp;pages=1661-1666&amp;pmid=17234812&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Wang AG, Lee DS, Moon HB, et al.  Non-structural 5A protein of hepatitis C virus induces a range of liver pathology in transgenic mice. <span><span class="ref-journal">J Pathol. </span>2009;<span class="ref-vol">219</span>(2):253–262.</span> [<a href="/pubmed/19621337" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pathol&amp;title=Non-structural+5A+protein+of+hepatitis+C+virus+induces+a+range+of+liver+pathology+in+transgenic+mice&amp;author=AG+Wang&amp;author=DS+Lee&amp;author=HB+Moon&amp;volume=219&amp;issue=2&amp;publication_year=2009&amp;pages=253-262&amp;pmid=19621337&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Poordad F, Bronowicki JP, Gordon SC, et al.  IL28B polymorphism predicts virologic response in patients with hepatitis C genoytpe 1 treated with Boceprevir (BOC) combination therapy. <span><span class="ref-journal">J Hepatol. </span>2011;<span class="ref-vol">54</span>(Suppl. 1):S6.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=IL28B+polymorphism+predicts+virologic+response+in+patients+with+hepatitis+C+genoytpe+1+treated+with+Boceprevir+(BOC)+combination+therapy&amp;author=F+Poordad&amp;author=JP+Bronowicki&amp;author=SC+Gordon&amp;volume=54&amp;issue=Suppl.+1&amp;publication_year=2011&amp;pages=S6&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Pol S, Aerssens J, Zeuzem S, et al.  Similar SVR rates in IL28B CC, CT or TT prior relapser, partial- or null responder patients treated with telaprevir/peginterferon/ribavirin: retrospective analysis of the REALIZE study. <span><span class="ref-journal">J Hepatol. </span>2011;<span class="ref-vol">54</span>(Suppl. 1):S6.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hepatol&amp;title=Similar+SVR+rates+in+IL28B+CC,+CT+or+TT+prior+relapser,+partial-+or+null+responder+patients+treated+with+telaprevir/peginterferon/ribavirin:+retrospective+analysis+of+the+REALIZE+study&amp;author=S+Pol&amp;author=J+Aerssens&amp;author=S+Zeuzem&amp;volume=54&amp;issue=Suppl.+1&amp;publication_year=2011&amp;pages=S6&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Sarasin-Filipowicz M, Oakeley EJ, Duong FHT, et al.  Interferon signaling and treatment outcome in chronic hepatitis C. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">105</span>(19):7034–7039.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2383932/">PMC free article</a>]</span> [<a href="/pubmed/18467494" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Interferon+signaling+and+treatment+outcome+in+chronic+hepatitis+C&amp;author=M+Sarasin-Filipowicz&amp;author=EJ+Oakeley&amp;author=FHT+Duong&amp;volume=105&amp;issue=19&amp;publication_year=2008&amp;pages=7034-7039&amp;pmid=18467494&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Chen L, Borozan I, Sun J, et al.  Cell-type specific gene expression signature in liver underlies response to interferon therapy in chronic hepatitis C infection. <span><span class="ref-journal">Gastroenterology. </span>2010;<span class="ref-vol">138</span>(3):1123–1133.</span> [<a href="/pubmed/19900446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Cell-type+specific+gene+expression+signature+in+liver+underlies+response+to+interferon+therapy+in+chronic+hepatitis+C+infection&amp;author=L+Chen&amp;author=I+Borozan&amp;author=J+Sun&amp;volume=138&amp;issue=3&amp;publication_year=2010&amp;pages=1123-1133&amp;pmid=19900446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Honda M, Sakai A, Yamashita T, et al.  Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. <span><span class="ref-journal">Gastroenterology. </span>2010;<span class="ref-vol">139</span>(2):499–509.</span> [<a href="/pubmed/20434452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Hepatic+ISG+expression+is+associated+with+genetic+variation+in+interleukin+28B+and+the+outcome+of+IFN+therapy+for+chronic+hepatitis+C&amp;author=M+Honda&amp;author=A+Sakai&amp;author=T+Yamashita&amp;volume=139&amp;issue=2&amp;publication_year=2010&amp;pages=499-509&amp;pmid=20434452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Urban TJ, Thompson AJ, Bradrick SS, et al.  IL28B genotype is associated with differential expression of intrahepatic interferon-stimulated genes in patients with chronic hepatitis C. <span><span class="ref-journal">Hepatology. </span>2010;<span class="ref-vol">52</span>(6):1888–1896.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3653303/">PMC free article</a>]</span> [<a href="/pubmed/20931559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatology&amp;title=IL28B+genotype+is+associated+with+differential+expression+of+intrahepatic+interferon-stimulated+genes+in+patients+with+chronic+hepatitis+C&amp;author=TJ+Urban&amp;author=AJ+Thompson&amp;author=SS+Bradrick&amp;volume=52&amp;issue=6&amp;publication_year=2010&amp;pages=1888-1896&amp;pmid=20931559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Dill MT, Duong FHT, Vogt JE, et al.  Interferon-induced gene expression is a stronger predictor of treatment response than IL28B genotype in patients with hepatitis C. <span><span class="ref-journal">Gastroenterology. </span>2011;<span class="ref-vol">140</span>(3):1021–1031.</span> [<a href="/pubmed/21111740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Interferon-induced+gene+expression+is+a+stronger+predictor+of+treatment+response+than+IL28B+genotype+in+patients+with+hepatitis+C&amp;author=MT+Dill&amp;author=FHT+Duong&amp;author=JE+Vogt&amp;volume=140&amp;issue=3&amp;publication_year=2011&amp;pages=1021-1031&amp;pmid=21111740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3518930&amp;issue-id=217129&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140067615278640title">Summary</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and Methods</a></li><li><a href="#S7title">Results</a></li><li><a href="#S14title">Discussion</a></li><li><a href="#S15title">Conclusions</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#GLtitle">Abbreviations</a></li><li><a href="#APP1title">Appendix 1</a></li><li><a href="#idm140067614439024title">Footnotes</a></li><li><a href="#idm140067604080272title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3518930/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3518930/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3518930/pdf/nihms380108.pdf">PDF (571K)</a></li> | <li><a href="#" data-citationid="PMC3518930" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3518930%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3518930%2F&amp;text=Interleukin%2028B%20polymorphisms%20are%20the%20only%20common%20genetic%20variants%20associated%20with%20low-density%20lipoprotein%20cholesterol%20(LDL-C)%20in%20genotype-1%20chronic%20hepatitis%20C%20and%20determine%20the%20association%20between%20LDL-C%20and%20treatment%20response" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3518930%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22497812" data-aiid="3518930" data-aid="3518930" data-iid="217129" data-domainid="319" data-domain="nihpa" data-accid="PMC3518930" data-md5="f76dee0447dae4745f0e972d27062ce8"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:44:32-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A3241DD53F1710000000002F10204&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3518930%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3518930/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>