<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3605033/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies multiple loci influencing human serum metabolite levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3605033/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies multiple loci influencing human serum metabolite levels"><meta name="citation_authors" content="Johannes Kettunen, Taru Tukiainen, Antti-Pekka Sarin, Alfredo Ortega-Alonso, Emmi Tikkanen, Leo-Pekka Lyytikäinen, Antti J Kangas, Pasi Soininen, Peter Würtz, Kaisa Silander, Danielle M Dick, Richard J Rose, Markku J Savolainen, Jorma Viikari, Mika Kähönen, Terho Lehtimäki, Kirsi H Pietiläinen, Michael Inouye, Mark I McCarthy, Antti Jula, Johan Eriksson, Olli T Raitakari, Veikko Salomaa, Jaakko Kaprio, Marjo-Riitta Järvelin, Leena Peltonen, Markus Perola, Nelson B Freimer, Mika Ala-Korpela, Aarno Palotie, Samuli Ripatti"><meta name="citation_issue" content="3"><meta name="citation_volume" content="44"><meta name="citation_firstpage" content="269"><meta name="citation_doi" content="10.1038/ng.1073"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3605033/?report=abstract"><meta name="citation_pmid" content="22286219"><meta name="DC.Title" content="Genome-wide association study identifies multiple loci influencing human serum metabolite levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Johannes Kettunen"><meta name="DC.Contributor" content="Taru Tukiainen"><meta name="DC.Contributor" content="Antti-Pekka Sarin"><meta name="DC.Contributor" content="Alfredo Ortega-Alonso"><meta name="DC.Contributor" content="Emmi Tikkanen"><meta name="DC.Contributor" content="Leo-Pekka Lyytikäinen"><meta name="DC.Contributor" content="Antti J Kangas"><meta name="DC.Contributor" content="Pasi Soininen"><meta name="DC.Contributor" content="Peter Würtz"><meta name="DC.Contributor" content="Kaisa Silander"><meta name="DC.Contributor" content="Danielle M Dick"><meta name="DC.Contributor" content="Richard J Rose"><meta name="DC.Contributor" content="Markku J Savolainen"><meta name="DC.Contributor" content="Jorma Viikari"><meta name="DC.Contributor" content="Mika Kähönen"><meta name="DC.Contributor" content="Terho Lehtimäki"><meta name="DC.Contributor" content="Kirsi H Pietiläinen"><meta name="DC.Contributor" content="Michael Inouye"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Antti Jula"><meta name="DC.Contributor" content="Johan Eriksson"><meta name="DC.Contributor" content="Olli T Raitakari"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Jaakko Kaprio"><meta name="DC.Contributor" content="Marjo-Riitta Järvelin"><meta name="DC.Contributor" content="Leena Peltonen"><meta name="DC.Contributor" content="Markus Perola"><meta name="DC.Contributor" content="Nelson B Freimer"><meta name="DC.Contributor" content="Mika Ala-Korpela"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="Samuli Ripatti"><meta name="DC.Identifier" content="10.1038/ng.1073"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies multiple loci influencing human serum metabolite levels"><meta property="og:type" content="article"><meta property="og:description" content="Nuclear magnetic resonance assays allow for measurement of a wide range of metabolic phenotypes. We report here the results of a GWAS on 8,330 Finnish individuals genotyped and imputed at 7.7 million SNPs for a range of 216 serum metabolic phenotypes ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3605033/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3605033/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3605033/pdf/nihms444574.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B2C09DD5384810000000004C1031F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3605033%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3605033</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Mar 21.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22286219" target="pmc_ext" ref="reftype=publisher&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2012 Jan 29; 44(3): 269–276. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Jan 29. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.1073" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.1073</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3605033</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS444574</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22286219">22286219</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies multiple loci influencing human serum metabolite levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kettunen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378902512384" co-class="co-affbox">Johannes Kettunen</a>,<sup>1,</sup><sup>2,</sup><sup>37</sup> <a href="/pubmed/?term=Tukiainen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378902510144" co-class="co-affbox">Taru Tukiainen</a>,<sup>1,</sup><sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>37</sup> <a href="/pubmed/?term=Sarin%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378867444816" co-class="co-affbox">Antti-Pekka Sarin</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Ortega-Alonso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378867442096" co-class="co-affbox">Alfredo Ortega-Alonso</a>,<sup>1,</sup><sup>6</sup> <a href="/pubmed/?term=Tikkanen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378867439376" co-class="co-affbox">Emmi Tikkanen</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Lyytik%26%23x000e4%3Binen%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378860942016" co-class="co-affbox">Leo-Pekka Lyytikäinen</a>,<sup>7</sup> <a href="/pubmed/?term=Kangas%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378860940032" co-class="co-affbox">Antti J Kangas</a>,<sup>5</sup> <a href="/pubmed/?term=Soininen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378860938048" co-class="co-affbox">Pasi Soininen</a>,<sup>5,</sup><sup>8</sup> <a href="/pubmed/?term=W%26%23x000fc%3Brtz%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378864597888" co-class="co-affbox">Peter Würtz</a>,<sup>1,</sup><sup>3,</sup><sup>5</sup> <a href="/pubmed/?term=Silander%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378864594432" co-class="co-affbox">Kaisa Silander</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Dick%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378868641440" co-class="co-affbox">Danielle M Dick</a>,<sup>9</sup> <a href="/pubmed/?term=Rose%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378868639456" co-class="co-affbox">Richard J Rose</a>,<sup>6,</sup><sup>10</sup> <a href="/pubmed/?term=Savolainen%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378868636736" co-class="co-affbox">Markku J Savolainen</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Viikari%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378901865120" co-class="co-affbox">Jorma Viikari</a>,<sup>13</sup> <a href="/pubmed/?term=K%26%23x000e4%3Bh%26%23x000f6%3Bnen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378901863136" co-class="co-affbox">Mika Kähönen</a>,<sup>14</sup> <a href="/pubmed/?term=Lehtim%26%23x000e4%3Bki%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378901861152" co-class="co-affbox">Terho Lehtimäki</a>,<sup>7</sup> <a href="/pubmed/?term=Pietil%26%23x000e4%3Binen%20KH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378901859168" co-class="co-affbox">Kirsi H Pietiläinen</a>,<sup>1,</sup><sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859232048" co-class="co-affbox">Michael Inouye</a>,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859229328" co-class="co-affbox">Mark I McCarthy</a>,<sup>19,</sup><sup>20</sup> <a href="/pubmed/?term=Jula%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378863595792" co-class="co-affbox">Antti Jula</a>,<sup>2</sup> <a href="/pubmed/?term=Eriksson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378863593808" co-class="co-affbox">Johan Eriksson</a>,<sup>21,</sup><sup>22,</sup><sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Raitakari%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859253456" co-class="co-affbox">Olli T Raitakari</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859250736" co-class="co-affbox">Veikko Salomaa</a>,<sup>2</sup> <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859248752" co-class="co-affbox">Jaakko Kaprio</a>,<sup>1,</sup><sup>6,</sup><sup>27</sup> <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378865927648" co-class="co-affbox">Marjo-Riitta Järvelin</a>,<sup>3,</sup><sup>12,</sup><sup>28,</sup><sup>30</sup> <a href="/pubmed/?term=Peltonen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Leena Peltonen</a>,<sup>36</sup> <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378865921472" co-class="co-affbox">Markus Perola</a>,<sup>1,</sup><sup>2,</sup><sup>31</sup> <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378857704032" co-class="co-affbox">Nelson B Freimer</a>,<sup>32</sup> <a href="/pubmed/?term=Ala-Korpela%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378857702048" co-class="co-affbox">Mika Ala-Korpela</a>,<sup>5,</sup><sup>8,</sup><sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859035600" co-class="co-affbox">Aarno Palotie</a>,<sup>1,</sup><sup>33,</sup><sup>34,</sup><sup>35</sup> and  <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219" class="affpopup" co-rid="_co_idm140378859031408" co-class="co-affbox">Samuli Ripatti</a><sup>1,</sup><sup>2,</sup><sup>33</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140378902512384"><h3 class="no_margin">Johannes Kettunen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Kettunen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Johannes Kettunen</a></div></div><div id="_co_idm140378902510144"><h3 class="no_margin">Taru Tukiainen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>3</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</p><p><sup>4</sup>Department of Biomedical Engineering and Computational Science, School of Science, Aalto University, Espoo, Finland</p><p><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Tukiainen%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Taru Tukiainen</a></div></div><div id="_co_idm140378867444816"><h3 class="no_margin">Antti-Pekka Sarin</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Sarin%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Antti-Pekka Sarin</a></div></div><div id="_co_idm140378867442096"><h3 class="no_margin">Alfredo Ortega-Alonso</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>6</sup>Department of Public Health, University of Helsinki, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Ortega-Alonso%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Alfredo Ortega-Alonso</a></div></div><div id="_co_idm140378867439376"><h3 class="no_margin">Emmi Tikkanen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Tikkanen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Emmi Tikkanen</a></div></div><div id="_co_idm140378860942016"><h3 class="no_margin">Leo-Pekka Lyytikäinen</h3><p><sup>7</sup>Department of Clinical Chemistry, University of Tampere and Tampere University Hospital, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Lyytik%26%23x000e4%3Binen%20LP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Leo-Pekka Lyytikäinen</a></div></div><div id="_co_idm140378860940032"><h3 class="no_margin">Antti J Kangas</h3><p><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Kangas%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Antti J Kangas</a></div></div><div id="_co_idm140378860938048"><h3 class="no_margin">Pasi Soininen</h3><p><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</p><p><sup>8</sup>Nuclear Magnetic Resonance (NMR) Metabonomics Laboratory, Department of Biosciences, University of Eastern Finland, Kuopio, Finland</p><div>Find articles by <a href="/pubmed/?term=Soininen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Pasi Soininen</a></div></div><div id="_co_idm140378864597888"><h3 class="no_margin">Peter Würtz</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>3</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</p><p><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=W%26%23x000fc%3Brtz%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Peter Würtz</a></div></div><div id="_co_idm140378864594432"><h3 class="no_margin">Kaisa Silander</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Silander%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Kaisa Silander</a></div></div><div id="_co_idm140378868641440"><h3 class="no_margin">Danielle M Dick</h3><p><sup>9</sup>Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, Virginia, USA</p><div>Find articles by <a href="/pubmed/?term=Dick%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Danielle M Dick</a></div></div><div id="_co_idm140378868639456"><h3 class="no_margin">Richard J Rose</h3><p><sup>6</sup>Department of Public Health, University of Helsinki, Helsinki, Finland</p><p><sup>10</sup>Department of Psychological and Brain Sciences, Indiana University, Bloomington, Indiana, USA</p><div>Find articles by <a href="/pubmed/?term=Rose%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Richard J Rose</a></div></div><div id="_co_idm140378868636736"><h3 class="no_margin">Markku J Savolainen</h3><p><sup>11</sup>Department of Internal Medicine, Clinical Research Center, University of Oulu, Oulu, Finland</p><p><sup>12</sup>Biocenter Oulu, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Savolainen%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Markku J Savolainen</a></div></div><div id="_co_idm140378901865120"><h3 class="no_margin">Jorma Viikari</h3><p><sup>13</sup>Department of Medicine, University of Turku and Turku University Hospital, Turku, Finland</p><div>Find articles by <a href="/pubmed/?term=Viikari%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Jorma Viikari</a></div></div><div id="_co_idm140378901863136"><h3 class="no_margin">Mika Kähönen</h3><p><sup>14</sup>Department of Clinical Physiology, University of Tampere and Tampere University Hospital, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000e4%3Bh%26%23x000f6%3Bnen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Mika Kähönen</a></div></div><div id="_co_idm140378901861152"><h3 class="no_margin">Terho Lehtimäki</h3><p><sup>7</sup>Department of Clinical Chemistry, University of Tampere and Tampere University Hospital, Tampere, Finland</p><div>Find articles by <a href="/pubmed/?term=Lehtim%26%23x000e4%3Bki%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Terho Lehtimäki</a></div></div><div id="_co_idm140378901859168"><h3 class="no_margin">Kirsi H Pietiläinen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>15</sup>Department of Medicine, Division of Internal Medicine, Obesity Research Unit, Helsinki University Central Hospital, Helsinki, Finland</p><p><sup>16</sup>Department of Public Health, Hjelt Institute, University of Helsinki, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Pietil%26%23x000e4%3Binen%20KH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Kirsi H Pietiläinen</a></div></div><div id="_co_idm140378859232048"><h3 class="no_margin">Michael Inouye</h3><p><sup>17</sup>Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia</p><p><sup>18</sup>Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Inouye%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Michael Inouye</a></div></div><div id="_co_idm140378859229328"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>19</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, Churchill Hospital, Oxford, UK</p><p><sup>20</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, UK</p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Mark I McCarthy</a></div></div><div id="_co_idm140378863595792"><h3 class="no_margin">Antti Jula</h3><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Jula%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Antti Jula</a></div></div><div id="_co_idm140378863593808"><h3 class="no_margin">Johan Eriksson</h3><p><sup>21</sup>Unit of Chronic Disease Epidemiology and Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><p><sup>22</sup>Department of General Practice and Primary Health Care, University of Helsinki, Helsinki, Finland</p><p><sup>23</sup>Unit of General Practice, Helsinki University Central Hospital, Helsinki, Finland</p><p><sup>24</sup>Folkhälsan Research Centre, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Eriksson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Johan Eriksson</a></div></div><div id="_co_idm140378859253456"><h3 class="no_margin">Olli T Raitakari</h3><p><sup>25</sup>Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Turku, Finland</p><p><sup>26</sup>Department of Clinical Physiology, Turku University Hospital, Turku, Finland</p><div>Find articles by <a href="/pubmed/?term=Raitakari%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Olli T Raitakari</a></div></div><div id="_co_idm140378859250736"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Veikko Salomaa</a></div></div><div id="_co_idm140378859248752"><h3 class="no_margin">Jaakko Kaprio</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>6</sup>Department of Public Health, University of Helsinki, Helsinki, Finland</p><p><sup>27</sup>Department of Mental Health and Alcohol Abuse Services, National Institute for Health and Welfare, Helsinki, Finland</p><div>Find articles by <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Jaakko Kaprio</a></div></div><div id="_co_idm140378865927648"><h3 class="no_margin">Marjo-Riitta Järvelin</h3><p><sup>3</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</p><p><sup>12</sup>Biocenter Oulu, University of Oulu, Oulu, Finland</p><p><sup>28</sup>Medical Research Council (MRC) Health Protection Agency (HPA) Centre for Environment and Health, Imperial College London, London, UK</p><p><sup>30</sup>Department of Lifecourse and Services, National Institute of Health and Welfare, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Marjo-Riitta Järvelin</a></div></div><div id="_co_idm140378865921472"><h3 class="no_margin">Markus Perola</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><p><sup>31</sup>Estonian Genome Centre, University of Tartu, Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Markus Perola</a></div></div><div id="_co_idm140378857704032"><h3 class="no_margin">Nelson B Freimer</h3><p><sup>32</sup>Center for Neurobehavioral Genetics, Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Freimer%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Nelson B Freimer</a></div></div><div id="_co_idm140378857702048"><h3 class="no_margin">Mika Ala-Korpela</h3><p><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</p><p><sup>8</sup>Nuclear Magnetic Resonance (NMR) Metabonomics Laboratory, Department of Biosciences, University of Eastern Finland, Kuopio, Finland</p><p><sup>11</sup>Department of Internal Medicine, Clinical Research Center, University of Oulu, Oulu, Finland</p><p><sup>12</sup>Biocenter Oulu, University of Oulu, Oulu, Finland</p><div>Find articles by <a href="/pubmed/?term=Ala-Korpela%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Mika Ala-Korpela</a></div></div><div id="_co_idm140378859035600"><h3 class="no_margin">Aarno Palotie</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>33</sup>Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK</p><p><sup>34</sup>Department of Medical Genetics, University of Helsinki and the Helsinki University Hospital, Helsinki, Finland</p><p><sup>35</sup>The Broad Institute of MIT and Harvard University, Cambridge, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Aarno Palotie</a></div></div><div id="_co_idm140378859031408"><h3 class="no_margin">Samuli Ripatti</h3><p><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</p><p><sup>33</sup>Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK</p><div>Find articles by <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22286219">Samuli Ripatti</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140378864731520_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140378864731520_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140378864731520_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Epidemiology and Biostatistics, Imperial College London, London, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Biomedical Engineering and Computational Science, School of Science, Aalto University, Espoo, Finland</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Computational Medicine Research Group, Institute of Clinical Medicine, University of Oulu and Biocenter Oulu, Oulu, Finland</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Public Health, University of Helsinki, Helsinki, Finland</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Clinical Chemistry, University of Tampere and Tampere University Hospital, Tampere, Finland</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Nuclear Magnetic Resonance (NMR) Metabonomics Laboratory, Department of Biosciences, University of Eastern Finland, Kuopio, Finland</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Virginia Institute for Psychiatric and Behavioral Genetics, Virginia Commonwealth University, Richmond, Virginia, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Psychological and Brain Sciences, Indiana University, Bloomington, Indiana, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Internal Medicine, Clinical Research Center, University of Oulu, Oulu, Finland</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Biocenter Oulu, University of Oulu, Oulu, Finland</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Medicine, University of Turku and Turku University Hospital, Turku, Finland</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Clinical Physiology, University of Tampere and Tampere University Hospital, Tampere, Finland</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Medicine, Division of Internal Medicine, Obesity Research Unit, Helsinki University Central Hospital, Helsinki, Finland</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Public Health, Hjelt Institute, University of Helsinki, Helsinki, Finland</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, Churchill Hospital, Oxford, UK</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Headington, UK</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Unit of Chronic Disease Epidemiology and Prevention, National Institute for Health and Welfare, Helsinki, Finland</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of General Practice and Primary Health Care, University of Helsinki, Helsinki, Finland</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Unit of General Practice, Helsinki University Central Hospital, Helsinki, Finland</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Folkhälsan Research Centre, Helsinki, Finland</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Turku, Finland</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Clinical Physiology, Turku University Hospital, Turku, Finland</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Department of Mental Health and Alcohol Abuse Services, National Institute for Health and Welfare, Helsinki, Finland</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Medical Research Council (MRC) Health Protection Agency (HPA) Centre for Environment and Health, Imperial College London, London, UK</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Institute of Health Sciences, University of Oulu, Oulu, Finland</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Department of Lifecourse and Services, National Institute of Health and Welfare, Oulu, Finland</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Estonian Genome Centre, University of Tartu, Tartu, Estonia</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Center for Neurobehavioral Genetics, Semel Institute for Neuroscience and Human Behavior, University of California, Los Angeles, Los Angeles, California, USA</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Human Genetics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, UK</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Department of Medical Genetics, University of Helsinki and the Helsinki University Hospital, Helsinki, Finland</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>The Broad Institute of MIT and Harvard University, Cambridge, Massachusetts, USA</div><div id="FN1">Correspondence should be addressed to: S.R. (<a href="mailto:dev@null" data-email="if.mmif@ittapir.ilumas" class="oemail">if.mmif@ittapir.ilumas</a>)</div><div id="FN2"><sup>36</sup>Deceased.</div><div id="FN3"><sup>37</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140378864731520_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22286219" target="pmc_ext" ref="reftype=publisher&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3605033/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplementary table 1.</strong></div><div><a href="/pmc/articles/PMC3605033/bin/NIHMS444574-supplement-supplementary_table_1.pdf" data-ga-action="click_feat_suppl">NIHMS444574-supplement-supplementary_table_1.pdf</a><span style="color:gray"> (3.3M)</span></div><div class="small guid">GUID:&nbsp;78D1FF59-D006-46A3-9ACA-FA7600DBC6A1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplementary table 2.</strong></div><div><a href="/pmc/articles/PMC3605033/bin/NIHMS444574-supplement-supplementary_table_2.xlsx" data-ga-action="click_feat_suppl">NIHMS444574-supplement-supplementary_table_2.xlsx</a><span style="color:gray"> (2.4M)</span></div><div class="small guid">GUID:&nbsp;E62A7DCC-C79E-4BDF-8CA6-9D4588671CFC</div></div></dd></dl></div><div id="idm140378869193584" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140378869193584title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Nuclear magnetic resonance assays allow for measurement of a wide range of metabolic phenotypes. We report here the results of a GWAS on 8,330 Finnish individuals genotyped and imputed at 7.7 million SNPs for a range of 216 serum metabolic phenotypes assessed by NMR of serum samples. We identified significant associations (<em>P</em> &lt; 2.31 × 10<sup>−10</sup>) at 31 loci, including 11 for which there have not been previous reports of associations to a metabolic trait or disorder. Analyses of Finnish twin pairs suggested that the metabolic measures reported here show higher heritability than comparable conventional metabolic phenotypes. In accordance with our expectations, SNPs at the 31 loci associated with individual metabolites account for a greater proportion of the genetic component of trait variance (up to 40%) than is typically observed for conventional serum metabolic phenotypes. The identification of such associations may provide substantial insight into cardiometabolic disorders.</p></div></div><div id="idm140378869189072" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Circulating metabolites have key roles in numerous biological pathways and consequently contribute to risk for many diseases, particularly disorders of the metabolic and cardiovascular systems<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301360">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301378">2</a></sup>. Such metabolites have long been used for clinical risk assessment, diagnosis, prognosis and evaluation of treatment efficacy. Genome-wide association studies (GWAS) have discovered numerous genomic regions associated with clinically relevant metabolites, with recent large-scale meta-analyses having identified, in total, over 100 loci associated with serum concentrations of individual metabolites, such as glucose, insulin, lipids and uric acid<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301348">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301328">5</a></sup>. Nevertheless, our understanding of the genetic basis and pathophysiological impact of variations in metabolite levels remains far from complete, and recent studies suggest the importance of investigating metabolite phenotypes beyond those used in traditional genetic studies. For example, in a recent longitudinal study, amino acids identified by metabolite profiling were shown to be associated with the risk for developing type 2 diabetes (T2D) in cohorts of apparently healthy individuals<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301332">6</a></sup>.</p><p id="P3" class="p">Until recently, the search for metabolic risk variants had focused on only a few metabolites at a time, but recent technological developments in NMR and mass spectrometry have made possible the quantification of over 100 metabolites in a single analytical procedure, allowing both broader and deeper molecular profiling of large cohorts<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301375">7</a></sup>. Previous genome-wide studies of high-throughput blood metabolites have identified over 40 loci associated with such measures<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703834">8</a>–<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301325">10</a></sup>. We present here the results of the largest genetic investigation of metabolomic phenotypes reported to date, including 8,330 unrelated individuals and 561 twin pairs sampled from the Finnish population who were analyzed for both genome-wide SNP genotypes and for 216 phenotypic measures obtained from an NMR-based metabolomic screen of fasting serum samples. We identify 31 loci associated with one or more phenotypes at a genome-wide significance level, including 6 newly identified loci for serum amino acids, 1 for citrate and 4 for serum lipoprotein and lipid metabolites. Using the twin samples, we estimate the heritability of each of the 216 metabolic measures and determine the proportions of overall and genetic variance that can be explained by the significantly associated loci.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><p id="P4" class="p p-first">We present genome-wide association results for 117 directly measured metabolites and 99 variables derived from these measures (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>). The 117 metabolites consist of 80 lipoproteins, 15 lipids and 22 low-molecular-weight metabolites (with the latter including pyruvic acid, amino acids and other small molecules participating in glycolysis, the citric acid cycle or the urea cycle). The derived variables (mainly ratios of directly measured metabolites) were selected on the basis of their prior utility in characterizing metabolic function in either normal or disease states. For example, an increase in the ratio of branched-chain amino acids (valine, leucine and isoleucine) to aromatic amino acids (phenylalanine and tyrosine) in the serum, termed Fischer’s ratio, is characteristic of liver fibrosis and is hypothesized to contribute to hepatic encephalopathy<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301380">11</a></sup>. The rationale for the specific ratios that we analyzed is presented (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>). The ratios selected in this study involve metabolites implicated in lipolysis, proteolysis, ketogenesis and glycolysis, as well as reagents and products of enzymatic reactions.</p><div id="S2" class="sec"><h3 id="S2title">Heritability of metabolic measures</h3><p id="P5" class="p p-first">To assess the heritability of each measure, we estimated intrapair metabolite correlations for 221 monozygotic and 340 dizygotic twin sets, aged 22–25, from the Finnish Twin Cohort. For amino acids and other small-molecule metabolites, the heritability estimates ranged between 0.23–0.55. Heritability estimates were higher for both lipids (range of 0.48–0.62) and lipoproteins (range of 0.50–0.76) (<a href="/pmc/articles/PMC3605033/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3605033/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140378863183344" class="figure"><a href="/pmc/articles/PMC3605033/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3605033_nihms444574f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms444574f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3605033/bin/nihms444574f1.jpg"></a></div><div id="largeobj_idm140378863183344" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3605033/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3605033/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p10">The heritability estimates and proportion of variance explained for all traits. Heritability estimates are presented for lipids, small molecules, ratios used in this study and lipoproteins. Labels in the lipoprotein subclasses describe the properties measured in each subfraction (<em>P</em>, concentration of particles; L, total lipids; PL, phospholipids; C, total cholesterol; CE, cholesterol esters; FC, free cholesterol; TG, triglycerides). Abbreviations are explained in detail in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 1</a>.</p></div></div></div><p id="P6" class="p p-last">Because of the high heritability estimated for many of the lipoprotein subclasses, we also compared the heritability of composite lipid phenotypes derived from NMR measures with that determined using conventional lipid measures (enzymatically measured lipid levels), using data from a subset of the twins (<em>n</em> = 256) for whom data from both types of assays were available. The NMR-based and enzymatic measures gave similar heritability estimates, except for triglycerides, for which estimated heritability was 0.68 for the NMR-based measure and 0.55 for the enzymatic measure (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Tables 4 and 5</a>).</p></div><div id="S3" class="sec"><h3 id="S3title">Genome-wide association analysis identifies 31 metabolite loci</h3><p id="P7" class="p p-first">We used stochastic imputation methods to augment the directly genotyped SNPs in the various cohorts (see <a href="/pmc/articles/PMC3605033/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> for a complete list of the cohorts) to generate a marker set for association analyses consisting of 7.7 million SNPs. The genotype imputation panel that we employed incorporated phased haplotypes from the 1000 Genomes Project<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301374">12</a></sup>, HapMap 3 (ref. <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301338">13</a>) and the Finnish extension to HapMap 3 (ref. <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301329">14</a>). Using an additive genetic model, we tested for univariate associations between these 7.7 million genetic markers and 216 metabolic measures. To correct for multiple testing, genome-wide statistical significance was set to <em>P</em> &lt; 2.31 × 10<sup>−10</sup> (standard univariate genome-wide significance threshold of 5 × 10<sup>−8</sup> / 216 phenotypes tested). Genome-wide inflation factors ranged from 0.99 to 1.06 (all are listed in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 6</a>). Thirty-one loci were significantly associated with at least one metabolic measure (<a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3);</span></span><span>3</span></a>); all SNPs that reached the genome-wide level of significance are presented in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>). The associations for these SNPs are shown in 2-Mb windows surrounding the SNP with the lowest <em>P</em> value to provide a graphical view of the associated regions (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>), and quantile-quantile plots were created for the lead traits in <a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>) to graphically present the overall inflation of the test statistics. For each cohort, box plots of raw phenotype values for each genotype class of the SNP with the lowest <em>P</em> value are provided (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>) to show the effects of the SNPs per genotype class per cohort. Four of the 11 newly associated loci presented here (1-159807481, rs17610395, 17-7083575 and rs6917603; indicated in <a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3)</span></span><span>3</span></a>) would not have been identified using only the HapMap 2 imputation panel instead of the expanded panel that we employed for our association testing (see Online Methods).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p12">Sample demographics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140378867566640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Study</th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>n</em></th><th rowspan="1" colspan="1" valign="bottom" align="left">Mean age (years ± s.d.)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Female (%)</th><th rowspan="1" colspan="1" valign="bottom" align="center">BMI ± s.d.</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">NFBC1966</td><td rowspan="1" colspan="1" valign="top" align="left">Northern Finland Birth Cohort 1966</td><td rowspan="1" colspan="1" valign="top" align="right">4,703</td><td rowspan="1" colspan="1" valign="top" align="left">31.0 ± 0.0</td><td rowspan="1" colspan="1" valign="top" align="center">51</td><td rowspan="1" colspan="1" valign="top" align="center">24.6 ± 4.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">YF</td><td rowspan="1" colspan="1" valign="top" align="left">The Cardiovascular Risk in Young Finns Study</td><td rowspan="1" colspan="1" valign="top" align="right">1,904</td><td rowspan="1" colspan="1" valign="top" align="left">37.7 ± 5.0</td><td rowspan="1" colspan="1" valign="top" align="center">54</td><td rowspan="1" colspan="1" valign="top" align="center">26.0 ± 4.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HBCS</td><td rowspan="1" colspan="1" valign="top" align="left">Helsinki Birth Cohort Study</td><td rowspan="1" colspan="1" valign="top" align="right">708</td><td rowspan="1" colspan="1" valign="top" align="left">61.3 ± 2.9</td><td rowspan="1" colspan="1" valign="top" align="center">60</td><td rowspan="1" colspan="1" valign="top" align="center">27.1 ± 4.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">GenMets</td><td rowspan="1" colspan="1" valign="top" align="left">Health2000 GenMets Study</td><td rowspan="1" colspan="1" valign="top" align="right">572</td><td rowspan="1" colspan="1" valign="top" align="left">55.8 ± 7.3</td><td rowspan="1" colspan="1" valign="top" align="center">57</td><td rowspan="1" colspan="1" valign="top" align="center">27.2 ± 4.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DILGOM</td><td rowspan="1" colspan="1" valign="top" align="left">The Dietary, Lifestyle, and Genetic determinants of Obesity and Metabolic Syndrome</td><td rowspan="1" colspan="1" valign="top" align="right">443</td><td rowspan="1" colspan="1" valign="top" align="left">50.1 ± 13.5</td><td rowspan="1" colspan="1" valign="top" align="center">56</td><td rowspan="1" colspan="1" valign="top" align="center">26.5 ± 4.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Twins</td><td rowspan="1" colspan="1" valign="top" align="left">FinnTwin12 (FT12) and FinnTwin16 (FT16) cohort studies</td><td rowspan="1" colspan="1" valign="top" align="right">1,269</td><td rowspan="1" colspan="1" valign="top" align="left">23.9 ± 2.1</td><td rowspan="1" colspan="1" valign="top" align="center">55</td><td rowspan="1" colspan="1" valign="top" align="center">23.1 ± 3.7</td></tr></tbody></table></div><div id="largeobj_idm140378867566640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3605033/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p13" class="p p-first-last">BMI, body mass index; <em>n</em>, number of individuals with both genotype and metabolic phenotypes included in this study.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p14">Fourteen loci associated with small-molecule phenotypes with genome-wide significance in meta-analysis of five Finnish cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140378857125344" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="13" rowspan="1" valign="bottom" align="center">Newly identified loci</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="left">Lead trait</th><th rowspan="1" colspan="1" valign="bottom" align="right">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Other allele</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">CAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect size</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.m.</th><th rowspan="1" colspan="1" valign="bottom" align="center">eQTL</th><th rowspan="1" colspan="1" valign="bottom" align="center">Candidate gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genotyped</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2160387</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">Ala/Val</td><td rowspan="1" colspan="1" valign="top" align="right">65074414</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">2.62 × 10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">−0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC1A4</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs10211524</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1440581</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">Fischer’s ratio</td><td rowspan="1" colspan="1" valign="top" align="right">89445446</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.96 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PPM1K</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs6841731</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2545801</td><td rowspan="1" colspan="1" valign="top" align="right">5</td><td rowspan="1" colspan="1" valign="top" align="left">Phe</td><td rowspan="1" colspan="1" valign="top" align="right">176773945</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">8.70 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.73</td><td rowspan="1" colspan="1" valign="top" align="center">−0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>F12</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs2731672</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2297644</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="left">Gln/His</td><td rowspan="1" colspan="1" valign="top" align="right">99349729</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DHDPSL</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs11817730</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4788815</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="left">Phe/Tyr</td><td rowspan="1" colspan="1" valign="top" align="right">70192312</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">1.54 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TAT</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs4788547</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">17-7083575 (rs117616209)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="left">Fischer’s ratio</td><td rowspan="1" colspan="1" valign="top" align="right">7083575</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">2.64 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">−0.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC2A4</em></td><td rowspan="1" colspan="1" valign="top" align="center">–<sup>b</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs807669</td><td rowspan="1" colspan="1" valign="top" align="right">22</td><td rowspan="1" colspan="1" valign="top" align="left">Citrate</td><td rowspan="1" colspan="1" valign="top" align="right">17534522</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3.30 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.62</td><td rowspan="1" colspan="1" valign="top" align="center">−0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center"><em>CLTCL1</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC25A1</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs712964</td></tr><tr><td colspan="13" rowspan="1" valign="top" align="center">Known loci</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP</td><td rowspan="1" colspan="1" valign="top" align="right">Chr.</td><td rowspan="1" colspan="1" valign="top" align="left">Lead trait</td><td rowspan="1" colspan="1" valign="top" align="right">Position</td><td rowspan="1" colspan="1" valign="top" align="center">Coded allele</td><td rowspan="1" colspan="1" valign="top" align="center">Other allele</td><td rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" valign="top" align="center">CAF</td><td rowspan="1" colspan="1" valign="top" align="center">Effect size</td><td rowspan="1" colspan="1" valign="top" align="center">s.e.m.</td><td rowspan="1" colspan="1" valign="top" align="center">eQTL</td><td rowspan="1" colspan="1" valign="top" align="center">Candidate gene, trait</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1260326</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">Ala/Gln</td><td rowspan="1" colspan="1" valign="top" align="right">27584444</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">2.59 × 10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65</td><td rowspan="1" colspan="1" valign="top" align="center">−0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>GCKR</em>, TG<sup>31</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs560887</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="left">Glc</td><td rowspan="1" colspan="1" valign="top" align="right">169471394</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">2.19 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>G6PC2</em>, Glc<sup>20</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4241816</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">His/Val</td><td rowspan="1" colspan="1" valign="top" align="right">187389321</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">5.58 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.57</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KLKB1</em>, bradykinin<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6900341</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">Ala/Tyr</td><td rowspan="1" colspan="1" valign="top" align="right">111901102</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">3.68 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC16A10</em>, Ile, Tyr<sup>8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10830963</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">Glc</td><td rowspan="1" colspan="1" valign="top" align="right">92348358</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">3.19 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MTNR1B</em>, Glc<sup>21</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2638315</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="left">Gln/Glc</td><td rowspan="1" colspan="1" valign="top" align="right">55151323</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2.43 × 10<sup>−35</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">−0.29</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SPRYD4</em></td><td rowspan="1" colspan="1" valign="top" align="center"><em>GLS2</em>, Gln<sup>8</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140378857125344" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3605033/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p15" class="p p-first-last">Chr., chromosome; position, SNP position in NCBI human genome build 36; CAF, coded allele frequency; candidate gene, potential candidate gene in the region; candidate gene, trait, gene reported for the known locus and lead trait with which the gene was associated; genotyped, genome-wide significant SNP genotyped in at least three study cohorts in the region of association; Ala, alanine; Glc, glucose; Gln, glutamine; His, histidine; Ile, isoleucine; Phe, phenylalanine; TG, total triglyceride content; Tyr, tyrosine; Val, valine.</p></div><div id="TFN3"><sup>a</sup>This locus would not have been identified using the HapMap 2 imputation reference panel.</div><div id="TFN4"><sup>b</sup>Imputed SNPs that were validated by sequencing.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p18">Genome-wide significant loci identified for the lipid metabolism phenotypes in the meta-analysis of five Finnish cohorts</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140378864875776" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="13" rowspan="1" valign="bottom" align="center">New loci
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="right">Chr.</th><th rowspan="1" colspan="1" valign="bottom" align="left">Lead trait</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Coded allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Other allele</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">CAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">Effect size</th><th rowspan="1" colspan="1" valign="bottom" align="center">s.e.m.</th><th rowspan="1" colspan="1" valign="bottom" align="center">eQTL</th><th rowspan="1" colspan="1" valign="bottom" align="left">Candidate gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">Genotyped</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1-159807481 (rs67418890)<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">XL-HDL-CE</td><td rowspan="1" colspan="1" valign="top" align="center">159807481</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">1.21 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center"><em>FCGR2B</em>, <em>FCGR2A</em></td><td rowspan="1" colspan="1" valign="top" align="left"><em>FCGR2B</em>, <em>FCGR2A</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs6427609</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4-73541429 (rs115136538)</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="left">Albumin</td><td rowspan="1" colspan="1" valign="top" align="center">73541429</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">4.84 × 10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">−0.51</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ALB</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs16850360</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6917603<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="left">XXL-VLDL-<em>P</em></td><td rowspan="1" colspan="1" valign="top" align="center">30125050</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">2.81 × 10<sup>−29</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">−0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PPP1R11</em></td><td rowspan="1" colspan="1" valign="top" align="center">rs6917603</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17610395<sup>a</sup></td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">LA/PUFA</td><td rowspan="1" colspan="1" valign="top" align="center">68318904</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">7.57 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CPT1A</em></td><td rowspan="1" colspan="1" valign="top" align="center">–<sup>b</sup></td></tr><tr><td colspan="13" rowspan="1" valign="top" align="center">Known loci</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP</td><td rowspan="1" colspan="1" valign="top" align="right">Chr.</td><td rowspan="1" colspan="1" valign="top" align="left">Lead trait</td><td rowspan="1" colspan="1" valign="top" align="center">Position</td><td rowspan="1" colspan="1" valign="top" align="center">Coded allele</td><td rowspan="1" colspan="1" valign="top" align="center">Other allele</td><td rowspan="1" colspan="1" valign="top" align="center"><em>P</em> value</td><td rowspan="1" colspan="1" valign="top" align="center">CAF</td><td rowspan="1" colspan="1" valign="top" align="center">Effect size</td><td rowspan="1" colspan="1" valign="top" align="center">s.e.m.</td><td rowspan="1" colspan="1" valign="top" align="center">eQTL</td><td rowspan="1" colspan="1" valign="top" align="left">Candidate gene, trait</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">1-55889093 (rs72669744)</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">L-LDL-FC</td><td rowspan="1" colspan="1" valign="top" align="center">55889093</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.10 × 10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">−0.59</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PCSK9</em>, LDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1168029</td><td rowspan="1" colspan="1" valign="top" align="right">1</td><td rowspan="1" colspan="1" valign="top" align="left">MobCH</td><td rowspan="1" colspan="1" valign="top" align="center">62741990</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2.66 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.70</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ANGPTL3</em>, TG<sup>31</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs13247874</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="left">VLDL-D</td><td rowspan="1" colspan="1" valign="top" align="center">72648378</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">8.43 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">−0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MLXIPL</em>, TG<sup>31</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-19956650 (rs115849089)</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="left">M-VLDL-PL</td><td rowspan="1" colspan="1" valign="top" align="center">19956650</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">4.26 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">−0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LPL</em>, TG<sup>31</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2575876</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="left">Tot-C/Est-C</td><td rowspan="1" colspan="1" valign="top" align="center">106705560</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">1.63 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">−0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ABCA1</em>, HDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs174547</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">LA/PUFA</td><td rowspan="1" colspan="1" valign="top" align="center">61327359</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">8.02 × 10<sup>−262</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.57</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FADS1</em>-<em>FADS2</em>-<em>FADS3</em>, <em>PC</em><sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs651821</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="left">Val/serum TG</td><td rowspan="1" colspan="1" valign="top" align="center">116167789</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">7.98 × 10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.91</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOA1</em>-<em>APOC3</em>-<em>APOA4</em>- <em>APOA5</em>, LDL<sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs35853021</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="left">XL-HDL-TG</td><td rowspan="1" colspan="1" valign="top" align="center">56467935</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">7.11 × 10<sup>−76</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.39</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LIPC</em>, HDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11075253</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="left">LA/PUFA</td><td rowspan="1" colspan="1" valign="top" align="center">15056147</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">4.98 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">−0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PDXC1</em>, C20:2 and C20:3 fatty acid metabolism<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3764261</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="left">HDL-C</td><td rowspan="1" colspan="1" valign="top" align="center">55550825</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>−36</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CETP</em>, HDL<sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7228085*</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="left">XL-HDL-TG</td><td rowspan="1" colspan="1" valign="top" align="center">45414812</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">6.70 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LIPG</em>, HDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs55791371</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">M-LDL-C/M-LDL-PL</td><td rowspan="1" colspan="1" valign="top" align="center">11049153</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">8.21 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">−0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LDLR</em>, LDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7412</td><td rowspan="1" colspan="1" valign="top" align="right">19</td><td rowspan="1" colspan="1" valign="top" align="left">L-LDL-FC</td><td rowspan="1" colspan="1" valign="top" align="center">50103919</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">2.52 × 10<sup>−58</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">−0.75</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">–</td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOE</em>-<em>APOC1</em>-<em>APOC2</em>, LDL<sup><a href="#R31" rid="R31" class=" bibr popnode">31</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6065904</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="left">L-HDL-L/M-HDL-L</td><td rowspan="1" colspan="1" valign="top" align="center">43968058</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">2.29 × 10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">−0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PLTP</em></td><td rowspan="1" colspan="1" valign="top" align="left"><em>PLTP</em>, TG<sup>32</sup></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140378864875776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3605033/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN5"><p id="__p19" class="p p-first-last">Chr., chromosome; position, SNP position in NCBI human genome build 36; CAF, coded allele frequency; candidate gene, potential candidate gene in the region; candidate gene, trait: gene reported for the known locus and lead trait the gene was associated with; genotyped, genome-wide significant SNP genotyped in at least three study cohorts in the region of association; L-HDL-L, lipid content in L-HDL; M-HDL-L, lipid content in medium HDL; FC, free cholesterol; LA, linoleic acid; PUFA, polyunsaturated fatty acids; C, cholesterol; PL, phospholipid; MobCH, double-bond protons of mobile lipids; Tot-C, total cholesterol; Est-C, esterified cholesterol; Val, valine; serum TG, serum total triglyceride content; D, particle diameter; CE, cholesterol ester; <em>P</em>, particle concentration.</p></div><div id="TFN6"><sup>a</sup>These loci would not have been identified using the HapMap 2 imputation reference panel.</div><div id="TFN7"><sup>b</sup>Imputed SNPs that were validated by sequencing.</div></div></div><p id="P8" class="p p-last">The 31 loci for which we detected association to NMR-based measures are shown in relation to the relevant metabolic pathways (<a href="/pmc/articles/PMC3605033/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a>). Multiple loci had not previously been observed to be associated with metabolic measures, either in the National Human Genome Research Institute (NHGRI) catalog of published GWAS<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301333">15</a></sup> or in previous blood metabolomics genome-wide screens<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703833">8</a>–<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301351">10</a></sup>. Thirteen of these loci, seven of which were newly identified here, were associated with amino acids and other small molecules (<a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The remaining 18 loci were associated with NMR-based measures of lipid metabolites (<a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Fourteen of these loci, three of which were newly identified here, were most strongly associated with lipoprotein measures, and four loci, including one newly identified locus, showed the strongest associations to other lipid-related measures. Below, we categorize the loci reaching genome-wide significance by phenotype and present potential candidate genes from the associated regions.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3605033/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140378858235408" class="figure"><a href="/pmc/articles/PMC3605033/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3605033_nihms444574f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms444574f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3605033/bin/nihms444574f2.jpg"></a></div><div id="largeobj_idm140378858235408" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3605033/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3605033/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p11">Overall summary of basic metabolism, key constituents of the NMR-measurable serum metabolome and associated genetic loci. The green shapes represent various dietary ingredients and systemic metabolic measures reflecting human body functions. A key selection of metabolites quantified in each metabolic category is given. Red rectangles represent the newly identified genetic loci found in this work, and blue rectangles indicate loci that were previously identified. The genes are categorized by the lead trait associations given in <a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. In particular, <em>ALB</em> and <em>FCGR2B</em> were associated with serum cholesterol, <em>PPP1R11</em> with very high VLDL measures, <em>CPT1A</em> with the LA/PUFA ratio, <em>SLC25A1</em> with citrate, <em>F12</em> with phenylalanine, <em>TAT</em> with tyrosine, <em>DHDPSL</em> with glutamine, <em>SLC1A4</em> with valine and <em>PPM1K</em> and <em>SLC2A4</em> with Fischer’s ratio.</p></div></div></div></div><div id="S4" class="sec"><h3 id="S4title">Associations with amino acids and other small molecules</h3><p id="P9" class="p p-first">For the metabolites involved in processes such as glycolysis, the citric acid cycle and amino acid metabolism, we identified, in total, seven new and six previously described loci (<a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Five of these associations (to valine, phenylalanine, tyrosine, isoleucine and leucine or their composite measure, Fischer’s ratio) involve amino acids previously shown to be associated with the risk for T2D<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301324">6</a></sup>. In addition, we identified one new locus associated with serum citrate levels and one new locus associated with serum glutamine levels.</p><p id="P10">The strongest associations at chromosome 2p14 (rs2160387) were with the ratio of alanine to valine (<em>P</em> = 2.6 × 10<sup>−22</sup>) and with circulating valine levels (<em>P</em> = 8.4 × 10<sup>−11</sup>). The associated SNP is located in the first intron of <em>SLC1A4</em> (encoding solute carrier family 1 member 4), a neutral amino acid transporter. A marker (rs1440581) at 4q22 was associated with both Fischer’s ratio (<em>P</em> = 2.0 × 10<sup>−16</sup>) and valine (<em>P</em> = 6.4 × 10<sup>−14</sup>), and a SNP (rs2545801) at 5q35 was associated with phenylalanine (<em>P</em> = 8.7 × 10<sup>−11</sup>). The rs4788815 SNP at 16q22 was associated with both tyrosine levels (<em>P</em> = 1.2 × 10<sup>−10</sup>) and the ratio of phenylalanine to tyrosine (<em>P</em> = 1.5 × 10<sup>−17</sup>). This marker is located 25 kb upstream of <em>TAT</em> (encoding tyrosine aminotransferase), an enzyme that catalyzes the conversion of tyrosine to hydroxyphenylpyruvate. Mutations in <em>TAT</em> have been shown to cause type 2 tyrosinemia (MIM 276600), whose symptoms include keratitis, painful palmoplantar hyperkeratosis, intellectual disability and elevated serum tyrosine levels<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301337">16</a></sup>. Two additional markers at this locus (rs34042070 and rs3213423) were independently associated with the ratio of glycoproteins and total cholesterol and remained significantly associated (<em>P</em> = 2.4 × 10<sup>−14</sup> and 3.2 × 10<sup>−12</sup>, respectively) after conditioning on rs4788815. These associations with the ratio of glycoproteins to total cholesterol may reflect a known association with total cholesterol at this locus<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301352">5</a></sup>. At 17p13, a SNP located 45 kb from <em>SLC2A4</em>, encoding a facilitated glucose transporter, was associated with Fischer’s ratio (17-7083575; rs117616209; <em>P</em> = 2.6 × 10<sup>−14</sup>).</p><p id="P11">Two additional loci were associated with amino acid levels, small molecules or measures derived from these metabolite levels. The rs807669 SNP near <em>SLC25A1</em> was associated with serum citrate concentration (<em>P</em> = 3.3 × 10<sup>−16</sup>), and rs2297644 in the first intron of <em>DHDPSL</em> (encoding DHDPS-like protein isoform 1) was associated with the ratio of glutamine to histidine (<em>P</em> = 1.2 × 10<sup>−12</sup>), as well as with serum glutamine levels (<em>P</em> = 4.7 × 10<sup>−11</sup>).</p><p id="P12">Six of the 13 loci determined by this study to be associated with small molecules have previously been reported. At chromosome 6q21, rs6900341 showed association with the ratio of alanine to tyrosine (<em>P</em> = 3.7 × 10<sup>−15</sup>), and, in a previous study, the rs7760535 SNP in the same locus associated with the ratio of isoleucine to tyrosine<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703837">8</a></sup>. rs6900341 is located within an intron of <em>REV3L</em> (encoding the REV3-like catalytic subunit of DNA polymerase ζ) and is 250 kb from a possible functional candidate gene, <em>SLC16A10</em> (encoding solute carrier family 16 member 10), which transports aromatic amino acids across the plasma membrane. In addition, in line with previous findings<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703836">8</a></sup>, we found that rs2638315 in the 3′ UTR of <em>GLS2</em> (encoding mitochondrial phosphate-activated glutaminase) was associated with serum glutamine levels (<em>P</em> = 8.6 × 10<sup>−28</sup>), as well as with the ratio of glutamine to glucose (<em>P</em> = 2.4 × 10<sup>−35</sup>). Moreover, we extend a previous finding of an association on chromosome 4 between a SNP in <em>KLKB1</em> and bradykinin. In our study, the rs4241816 SNP in the same locus was associated with both serum histidine levels (<em>P</em> = 2.2 × 10<sup>−11</sup>) and the ratio of histidine to valine (<em>P</em> = 5.6 × 10<sup>−13</sup>). The decarboxylation of histidine by the HDC enzyme leads to the formation of histamine, a central proinflammatory mediator. A possible functional candidate gene, <em>TLR3</em> (encoding Toll-like receptor 3), lies 145 kb from the associated SNP. <em>TLR3</em> encodes a receptor that regulates the release of histamine-rich granules from mast cells<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301385">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301355">18</a></sup>. A previous study provided further support for the potential involvement of <em>TLR3</em>, having shown that bradykinin regulates histamine release from mast cells through a mechanism that remains unclear<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301381">19</a></sup>.</p><p id="P13" class="p p-last">The SNPs in <em>G6PC2</em> and <em>MTNR1B</em> that showed associations with serum glucose in previous GWAS<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301371">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301372">21</a></sup> were also associated with NMR-measured glucose levels in this study. Previous studies have shown association between SNPs in <em>GCKR</em> (encoding glucokinase regulator) and a variety of metabolic traits, including triglycerides<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301357">5</a></sup>, glucose<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301386">3</a></sup> and the glucose to mannose ratio<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703835">8</a></sup>. In our study, a SNP in <em>GCKR</em> (rs1260326) was associated with the ratio of alanine to glutamine and other amino acids measures, in addition to measures of very-low-density lipoprotein (VLDL) and total triglycerides.</p></div><div id="S5" class="sec"><h3 id="S5title">Associations with NMR-based measures of lipid metabolism</h3><p id="P14" class="p p-first">Previous investigations have identified nearly 100 genetic loci associated with measures of serum lipid concentrations that are typically used in clinical practice<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301335">5</a></sup>. These measures (total cholesterol, high-density lipoprotein (HDL), low-density lipoprotein (LDL) and triglycerides), however, are heterogeneous aggregates that reflect multiple biological processes. We assessed here association to a much wider range of lipoprotein and lipid measures (<em>n</em> = 95) hypothesized to quantify more homogenous phenotypes. We identified a total of 18 such associations, 4 of which have not previously been reported (<a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). These four loci include one associated with a cholesterol measure, one associated with a VLDL measure, one associated with the ratio of linoleic acid to other polyunsaturated fatty acids (LA/PUFA) and one associated with serum albumin levels as well as lipoprotein measures.</p><p id="P15">The dosage of the C allele at the 4-73541429 SNP (rs115136538) was associated with decreased concentration of serum albumin (<em>P</em> = 4.8 × 10<sup>−18</sup>) and with elevated concentration of 37 other metabolites, including apoB-containing lipoprotein particles, several cholesterol- related measures and sphingomyelin (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>). This SNP was also positively associated with the levels of enzymatically measured total cholesterol (<em>P</em> = 1.5 × 10<sup>−19</sup>) and LDL cholesterol (<em>P</em> = 2.64 × 10<sup>−13</sup>). These observations are in contrast to the mostly positive correlations observed in the metabolic phenotype data between albumin and apoB-containing lipoprotein measures (range for correlations of −0.03 to 0.5; mean = 0.31; median = 0.37). Rare variants in the albumin gene have previously been associated with analbuminemia, hypercholesterolemia and hyperlipidemia<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301370">22</a>–<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301365">24</a></sup>; however, to our knowledge, this is the first report of a common genetic variant at <em>ALB</em> being associated with any lipoprotein or lipid measure or albumin levels.</p><p id="P16">The cholesterol ester content of extra large HDL (XL-HDL-CE) particles was associated (<em>P</em> = 1.2 × 10<sup>−10</sup>) with a single SNP (1-159807481; rs67418890). In a subset of the samples (<em>n</em> = 585) with leukocyte expression data and genome-wide genotype data available, we identified a <em>cis</em> expression quantitative trait locus (eQTL) for this SNP in the <em>FCGR2A</em> and <em>FCGR2B</em> genes that encode the α and β subunits of the Fc fragment of the IgG low-affinity II receptor, respectively (linear regression <em>P</em> = 3.9 × 10<sup>−10</sup> and 1.3 × 10<sup>−20</sup>, respectively; Online Methods and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>). There were several SNPs that acted as eQTLs in the <em>FCGR2B</em> locus, including our lead SNP (rs67418890). All of the best associated SNPs with eQTL activity are in tight correlation with each other (<em>r</em> &gt; 0.87), suggesting that they represent the same possibly functional variant in this shared haplotype. <em>FCGR2A</em> and <em>FCGR2B</em> encode the components of the CD32 cell surface receptor, an IgG-mediated B-cell coreceptor that represses antibody production in the presence of IgG; however, CD32 also has a role in cardiovascular disease, with <em>FCGR2B</em> having been shown to modify the size of atherosclerotic plaques in mice<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301369">25</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301387">26</a></sup>. Our findings suggest a role for CD32 in the cholesterol balance in peripheral tissue and potentially in reverse cholesterol transport.</p><p id="P17">A class I major histocompatibility (MHC) locus <em>HLA-A</em> also harbored metabolite signals, with the top SNP, rs6917603, being strongly associated with the concentration of chylomicrons and extremely large VLDL particles (XXL-VLDL-<em>P</em>) (<em>P</em> = 2.8 × 10<sup>−29</sup>). The associated variant is upstream of <em>PPP1R11</em>, an inhibitor of PP1, a highly conserved serine/threonine phosphatase with a central role in glycogen metabolism and blood glucose levels<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301367">27</a></sup>. The association of rs6917603 with VLDL was also independent of the two other SNPs in the human leukocyte antigen (HLA) locus that have been shown to be associated with triglycerides, total cholesterol and LDL cholesterol<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301339">5</a></sup>.</p><p id="P18">A SNP at 11q13.2 was found to be associated with the LA/PUFA ratio (rs17610395; <em>P</em> = 7.6 × 10<sup>−12</sup>). The associated SNP is within 6 Mb of the <em>FADS</em> gene cluster (encoding fatty acid desaturase), a known lipoprotein locus, and it is likely to be independent of the previously identified SNP in <em>FADS</em> (rs174547). In our study, the association remained significant after conditioning on the earlier marker (<em>P</em> = 7.6 × 10<sup>−12</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333834152">9</a>,<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301377">28</a>,<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301336">29</a></sup>. rs17610395 is a nonsynonymous SNP (encoding a p.Ala275Thr amino acid change) located in <em>CPT1A</em>, a gene encoding carnitine palmitoyltransferase IA, a liver-expressed enzyme involved in long-chain fatty acid oxidation<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301373">30</a></sup>. Rare mutations in <em>CPT1A</em> cause CPT IA deficiency in an autosomal recessive metabolic disorder of long-chain fatty acid oxidation (MIM 255120).</p><p id="P19" class="p p-last">Fourteen loci that were reported in previous GWAS to be associated with lipid measures were also associated with NMR-based lipid measures in this study. <em>FADS1</em> was originally associated with a range of glycero-phosphatidylcholines<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301326">10</a></sup>. The authors in the previous study also found that a SNP in <em>FADS1</em> was associated with several lipid and phosphatidylcholine measures, whereas we showed association with measures of fatty acid saturation and with omega-3 fatty acids and several omega-3 fatty acid ratios. <em>PDXDC1</em> (encoding pyridoxal-dependent decarboxylase domain–containing protein 1) was recently shown to associate with the ratio of eicosatrienoylglycerophospholipids<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_372703832">8</a></sup>, indicating a role for it in the metabolism of 20:2 and 20:3 fatty acids. In our study, the same locus was associated with the LA/PUFA ratio, with linoleic acid being one of the metabolites involved in eicosanoid synthesis. Four loci previously reported to be associated with serum triglycerides (<em>ANGPTL3</em>, <em>MLXIPL</em>, <em>LPL</em> and <em>PLTP</em>)<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_631571987">31</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301341">32</a></sup> were associated with VLDL measures in our study. The <em>PLTP</em> locus was also associated with a large number of HDL measures. Four loci have previously been associated with total LDL cholesterol (LDL-C) (<em>PCSK9</em>, the <em>APOA1</em> region, <em>LDLR</em> and the <em>APOE</em> region)<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_631571989">31</a>,<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301384">33</a></sup>. In our study, the <em>APOA1</em> region showed association with a large number of VLDL and triglyceride measures, whereas the other three loci showed association with LDL and cholesterol measures. Finally, associations to four loci previously determined to be involved in total HDL cholesterol (HDL-C) levels (<em>ABCA1</em>, <em>LIPC</em>, <em>CETP</em> and <em>LIPG</em>) are replicated in our data<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_631571988">31</a>,<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301322">32</a></sup>.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Proportion of variance explained</h3><p id="P20" class="p p-first">For each phenotype, we estimated the proportion of variance explained by the 33 significantly associated SNPs in an independent sample of 436 individuals from the Finnish twin cohort (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>). For the direct metabolite measures, the proportion of variance explained by the SNPs ranged between 0.2–9.1% for HDL subclasses, 5.0–8.0% for LDL subclasses, 0.5–8.2% for VLDL subclasses, 4.8–9.5% for intermediate-density lipoprotein (IDL) subclasses and up to 7.6% for the other lipids and molecules. When comparing the proportion of variance explained in the lipoprotein subclasses and the composite measures for the corresponding lipids, associated SNPs explained much more of the variance for the larger size lipoprotein subclasses than for the corresponding composite measures (<a href="/pmc/articles/PMC3605033/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P21" class="p p-last">For derived measures, the SNPs explained up to 25% (18–32%, 95% confidence interval (CI)) of the total variance in the LA/ PUFA ratio, corresponding to 40% of the heritability. This particularly high proportion of explained variance is driven by the association in the <em>FADS1</em>-<em>FADS2</em>-<em>FADS3</em> locus, where each risk allele was accompanied by an increase of 0.57 s.d. in the fatty acid ratio in a common variant (coded allele frequency (CAF) = 42%). This strong association shows that a few variants in specific metabolite measures can explain a high proportion of variation, as has been shown previously for the <em>FADS</em> gene cluster<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333834149">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301350">10</a></sup>.</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">DISCUSSION</h2><p id="P22" class="p p-first">In this study, we report the heritability of metabolite measures assayed by NMR and identify 11 new genetic associations for these measures through genome-wide analysis of five population-based cohorts comprising &gt;8,000 individuals. We further show that, compared to the associations typically observed for complex traits in comparably sized samples, the associations identified in this study explain a greater proportion of trait variance.</p><p id="P23">The results support several conclusions. First, almost 40% of the metabolites assayed in this study showed an estimated heritability of &gt;0.6, a figure higher on average than that typically reported for clinically used measures (heritability estimates of plasma lipids have been reported to range from 0.39 to 0.62 for total cholesterol (TC), 0.39 to 0.83 for HDL-C, 0.24 to 0.50 for LDL-C, 0.20 to 0.55 for triglycerides and 0.07 to 0.28 for fasting plasma glucose)<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333834150">34</a>,<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301340">35</a></sup>. This observation supports the hypothesis that the more detailed metabolite measures obtained by NMR are more reflective of underlying biology than are the composite measures used in clinical practice. This idea is in line with the heritability estimates of composite measures being similar between enzymatic- and NMR-based measures of HDL-C and LDL-C. For triglycerides, the higher heritability estimate for NMR-based measures may reflect its smaller variance compared to enzymatic triglyceride measures. Smaller variance for the NMR-based measure has also been shown previously<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301368">28</a></sup>.</p><p id="P24">Second, we have identified newly associated loci that may help to characterize the biochemical pathways by which serum amino acids influence risk for T2D. Five newly identified loci were associated with metabolic measures of tyrosine, phenylalanine and valine, three of the five amino acids recently shown to be associated with the risk for developing T2D<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301342">6</a></sup>.</p><p id="P25">Third, by employing NMR-based metabolite measures for GWAS, we were able to identify new loci associated with lipid metabolism using a much smaller sample size than has been required to identify most such loci for composite lipid traits. There are several potential explanations for why these signals have not previously been detected by large GWAS consortia. For example, the loci containing <em>FCGR2B</em> and <em>PPP1R11</em> showed association patterns in the lipoprotein spectrum that likely confounded composite measures, and the <em>CPT1A</em> locus was associated with a specific metabolic measure (the LA/PUFA ratio) that is not usually considered in the analyses of lipid phenotypes. These observations suggest that further enhancing the range of measures of lipid metabolism used in genetic association analyses will likely provide a more complete picture of the loci modifying lipids and lipoproteins.</p><p id="P26">The results reported here represent only the first step in the genetic dissection of high-resolution metabolic phenotypes. The metabolome contains many more types of molecules and particles than those measured in the 216 phenotypes assessed here. The phenotypes assessed here were assayed in a high-throughput and cost-effective manner in order to attain the large sample numbers required for genome-wide association analyses. Measurements using mass spectrometry methods broaden the range of metabolites beyond those identifiable using NMR methods, and both approaches are required to obtain a more comprehensive view of the metabolome<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_333834151">9</a></sup>. NMR, which is less expensive and more automatable than mass spectrometry, is therefore more suitable for large-scale genetic investigations. It can also extract information from a range of lipoprotein particles not assayable using mass spectrometry. On the other hand, mass spectrometry provides analytical opportunities that are outside the reach of NMR spectroscopy. Mass spectrometry is several fold more sensitive and can be used as a discovery tool to identify new compounds as well as assay a wider spectrum of known molecules.</p><p id="P27">The use in this study of the most comprehensive available imputation panel increased the opportunities for identifying new loci, as evidenced by the fact that 4 of the 11 newly associated loci reported here would not have been detected with the commonly used HapMap 2 panel. On the other hand, imputation on this scale can also enhance the chances of identifying false positive loci. For this reason, we applied particularly stringent quality control filters to prevent false positives caused by imputation errors. For example, we restricted our search to only variants that had a minor allele frequency (MAF) of &gt;0.01, good imputation quality and similar effect sizes in all cohorts. Although 9 of the 11 newly identified loci showed the strongest association with an imputed marker, all but two loci also showed association to directly genotyped SNPs in three or more cohorts. The two remaining loci showed high accuracy of imputation in sequencing undertaken to validate the imputed genotypes. Thus, we used highly conservative procedures to diminish the risk of false positive associations, and these strict filters may have abolished some true positive signals.</p><p id="P28" class="p">In conclusion, the study highlights the value of enhancing the specificity of metabolic phenotyping for genetic association analyses. The availability of 216 metabolite measures by NMR provides a substantial enhancement over the measures obtained by classical clinical chemistry methods that are typically available in large population cohorts. Using these phenotypes with the higher-resolution genetic analysis now possible using the 1000 Genomes Project variant catalog, we identified several new loci associated with serum metabolites, some of which may be important in risk for T2D.</p><div id="S8" class="sec sec-last"><h3 id="S8title">URLs</h3><p id="P29" class="p p-first-last">1000 Genomes Project imputation reference panel, <a href="http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_pilot_plus_hapmap3.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/data_download_1000G_pilot_plus_hapmap3.html</a>.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">ONLINE METHODS</h2><p id="P31" class="p p-first">We performed a GWAS for metabolites and metabolite ratios in five cohorts from Finland totaling 8,330 individuals (cohorts are described in <a href="/pmc/articles/PMC3605033/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>). Written informed consent was obtained from all participants. Studies were approved by the following ethical committees: Ethical Committee of Oulu University Faculty of Medicine for NFBC 1966, Ethics Committee of the National Public Health Institute for Health2000 and HBCS, Helsinki University Hospital Coordinating Ethical Committee for DILGOM and Twins and Ethics Committee of the Hospital District of Southwest Finland for YF. The genomic positions indicated throughout this study are based on NCBI human genome build 36. Some of the markers from the 1000 Genomes imputation reference set highlighted in this manuscript have since been assigned rs numbers (1-55889093, rs72669744; 1-159807481, rs67418890; 4-73541429, rs115136538; 8-19956650, rs115849089; and 17-7083575, rs117616209).</p><div id="S11" class="sec"><h3 id="S11title">Genotyping and imputation</h3><p id="P32" class="p p-first">All cohorts were genotyped using commercially available Illumina genotyping arrays. NFBC1966 was genotyped using the HumanHap CNV 370k array. DILGOM and GenMets were genotyped with the Illumina HumanHap 610k array. YF and HBCS were genotyped using a custom-generated HumanHap 670k array that largely overlapped with the HumanHap 610k array but had additional copy-number probes. Quality control analysis was performed independently for each study before imputation. Poor quality markers (for which genotyping failed in &gt;5% of samples) and poor quality DNA samples (for which genotyping failed at &gt;5% of markers) were removed from further analysis. In addition, individuals with excessive genome-wide heterozygosity (indicating sample contamination) or gender discrepancies as well, as closely related individuals, were removed from the data. Imputation was performed on the cleaned data using IMPUTE<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301359">36</a></sup>. Imputation included a 1000 Genomes imputation reference and a HapMap 3 imputation reference, which included an additional Finnish imputation reference in HapMap 3 depth. The reference used included HapMap3 and 1000 Genomes in NCBI build 36, where the HapMap 3 files were from release 2 (February 2009) and the 1000 Genomes files were from the low-coverage pilot genotypes released in March 2010 (see URLs). The benefit of the additional Finnish reference set has previously been discussed in detail<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301366">14</a></sup>. The regional plots of the associated loci and genotyping status or imputation reference are presented (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). The SNPs reported in <a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3,</span></span><span>3</span></a>, other than those directly genotyped, have imputation quality of &gt;0.7. The quantile-quantile plots for phenotypes that show the strongest association with newly identified loci are presented (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>), as are the box plots of the associations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>).</p><p id="P33">To assess the accuracy of imputation, we compared imputed genotypes for 316 markers showing genome-wide significance with directly genotyped Cardiometabochip SNPs available for the DILGOM study sample (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). The concordance between genotyped SNPs on the Cardiometabochip and imputed genotypes in DILGOM was high (94% of the SNPs had <em>r</em><sup>2</sup> &gt; 0.8), indicating imputation was accurate for the reported SNPs. Imputation accuracy was also relatively similar across the whole region of MAFs ranging from 1% to 50% (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>).</p><p id="P34" class="p p-last">We tested the effect of using the comprehensive imputation reference panel described above by analyzing the same 31 loci reported in this study using the HapMap 2 release 22 Utah residents of Northern and Western European ancestry (CEU) imputation reference panel with the lead trait for each locus reported in <a href="/pmc/articles/PMC3605033/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Tables 2</span></a> and <a href="/pmc/articles/PMC3605033/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. Individual cohorts were imputed with the HapMap 2 reference panel using MACH<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301345">37</a></sup> and analyzed using ProbABEL<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301354">38</a></sup>, and the results from individual cohorts were combined with an inverse variance meta-analysis using GWAMA<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301363">39</a></sup>.</p></div><div id="S12" class="sec"><h3 id="S12title">Serum metabolomics</h3><p id="P35" class="p p-first-last">All cohorts were analyzed using the same high-throughput serum NMR metabolomics platform in the same analysis laboratory as described previously<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301358">40</a></sup>. This methodology provided information on 117 serum measures, including lipoprotein subclass distribution and lipoprotein particle concentration, low molecular weight metabolites, such as amino acids, 3-hydroxybutyrate and creatinine, and detailed molecular information on serum lipids, including free and esterified cholesterol, sphingomyelin and fatty acid saturation. Further details for the NMR spectroscopy and data analyses are provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>.</p></div><div id="S13" class="sec"><h3 id="S13title">Genome-wide association analyses and meta-analysis</h3><p id="P36" class="p p-first">Data from individuals using lipid-lowering medication or pregnant individuals were removed before analysis. In all of the studies, metabolomic phenotypes were measured from fasting serum samples. Within each study, residual metabolomic concentrations or residuals of ratios were determined after regression adjustment using R software<sup><a href="#R41" rid="R41" class=" bibr popnode">41</a></sup>. The selected ratios of metabolites were calculated by taking a ratio of particular metabolite measures. Outliers (≥4 s.d. from the mean) were removed from resulting ratio values before covariate adjustment, as the calculation is sensitive to very strong outliers, and strong outliers hamper the covariate correction. To calculate residuals for all metabolites and ratios, each study included as covariates age (except in NFBC66, where all individuals were examined at the age of 31 years), the first ten principal components from genetic data to correct for possible population stratification and sex. Residuals were normalized to have a mean of 0 and s.d. of 1 using inverse normal transformation. The resulting normal distributions of all 216 phenotypes were correlated against genotypes, assuming an additive genetic model using the SNPTEST program<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301382">36</a></sup>. To combine the effect estimates from five distinct studies, we conducted a fixed-effects inverse variance meta-analysis using META<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301379">36</a></sup> for each phenotype. Only good-quality SNPs were included in further evaluations on the basis of the following criteria: imputation informativeness was &gt;0.4, there was no heterogeneity in the effect sizes for the SNP between cohorts (Cochran’s <em>Q</em> statistic <em>P</em> value &lt; 1 × 10<sup>−5</sup>), and the SNP had to have a result in all five cohorts. For SNPs successfully analyzed in each cohort, the number excluded as a result of heterogeneity in effect size ranged between 28 and 720 for different traits. The genome-wide inflation factors were calculated from each meta-analysis (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 4</a>). All meta-analysis test statistics were corrected by using a genomic inflation factor for each trait. A stringent genome-wide significance level of 2.31 × 10<sup>−10</sup> was set to correct for multiple testing of the 216 phenotypes.</p><p id="P37" class="p p-last"><em>P</em>-gain values were calculated for the ratios by taking the minimum of the two <em>P</em> values for the individual metabolite associations divided by the <em>P</em> value of the ratio between both. If the <em>P</em>-gain value was &gt;1, then the ratio had better power for the association than did either of the direct metabolite measures. For all ratios for which an association was presented or discussed in the manuscript, the <em>P</em>-gain value was &gt;1. <em>P</em>-gain values are presented in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 7</a>.</p></div><div id="S14" class="sec"><h3 id="S14title"><em>Cis</em> EQTL analysis</h3><p id="P38" class="p p-first-last">We had whole-blood expression data available in the DILGOM cohort for 585 individuals. The expression data have been described previously<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301376">42</a></sup>. In brief, to obtain stabilized total RNA in study III, the PAXgene Blood RNA System (PreAnalytiX) was used. Biotinylated cDNA (750 ng) was hybridized to Illumina HumanHT-12 Expression BeadChips according to the manufacturer’s protocol. The correlation of SNPs, which had significant association with metabolites, was tested for correlation with expression. All expression probes within 1 Mb of the SNP were tested for correlation with the SNP allele dosage using Spearman rank correlation in R. All eQTL associations reaching a genome-wide level of significance (<em>P</em> &lt; 9 × 10<sup>−7</sup>) with a 1% genome-wide false discovery rate<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301349">43</a></sup> are presented (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 8</a>).</p></div><div id="S15" class="sec"><h3 id="S15title">Heritability estimates</h3><p id="P39" class="p p-first-last">Briefly, we used 561 twin pairs (221 monozygotic pairs and 340 dizygotic pairs, aged 22–25) from the Finnish population to estimate intrapair metabolite correlation for monozygotic and dizygotic pairs. For each phenotype, models estimating the hypothetical combinations of the different genetic and environmental sources of influence (including, additive genetic influences (A), shared environmental influences (C), dominance genetic influences (D) and unique environmental influences (E)) were built and tested against a saturated model, where no inference on the underlying architecture of the phenotype was assumed. The estimation of heritability is presented in detail in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Note</a>, as is the comparison between clinical and NMR-based lipid measures.</p></div><div id="S16" class="sec"><h3 id="S16title">Proportion of variance explained</h3><p id="P40" class="p p-first-last">A subset of the twin cohort used for the heritability estimates also had genome-wide SNP data available. The genotyping was performed using Illumina HumanHap 670k custom arrays, and genotypes were clustered using the Illuminus algorithm<sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301361">44</a></sup>. The part of the twin cohort that was used for heritability estimation was stratified for alcohol drinking. To match the population-level alcohol consumption distribution, we randomly chose one member of the twin pair from this subset of twins. This resulted in a random population sample of 436 individuals with both metabolite data and genotypes. We assessed the proportion of variance explained in this independent population sample of 436 individuals by the 33 SNPs (31 from the GWAS and 2 from fine mapping) by creating a genetic risk score for each trait for which the significant SNP associated with a nominal genome-wide level of significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>).</p></div><div id="S17" class="sec sec-last"><h3 id="S17title">Clinical lipid measures in population samples</h3><p id="P41" class="p p-first-last">The enzymatic lipoprotein measures were analyzed in the same study sample as the metabolomic phenotypes, but additional individuals (<em>n</em> = 2,247) were also available, who were not part of the metabolomic studies. The results from all these individuals were used to test for the association of the <em>ALB</em> locus with enzymatic lipoprotein measures. The sample cohorts were the same as those used in this study. Data from individuals who were pregnant or using lipid-lowering medication were removed before analysis, and lipid measures were adjusted for age, sex and the ten first principal components and transformed using inverse normal transformation. Enzymatic lipoprotein measures have been described in an earlier study<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303301353">5</a></sup>.</p></div></div><div id="S18" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplementary table 1</h4><div class="sup-box half_rhythm" id="d35e1509"><a href="/pmc/articles/PMC3605033/bin/NIHMS444574-supplement-supplementary_table_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.3M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>supplementary table 2</h4><div class="sup-box half_rhythm" id="d35e1513"><a href="/pmc/articles/PMC3605033/bin/NIHMS444574-supplement-supplementary_table_2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.4M, xlsx)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p3">We thank all the Finnish volunteers who participated in the studies. We thank the IT Center for Science and the technology center of the Institute for Molecular Medicine Finland for providing the computational facilities required in this study. The expert technical assistance for statistical analyses provided by A. Vikman, I. Lisinen, V. Aalto and the Genotyping Facilities at the Wellcome Trust Sanger Institute are gratefully acknowledged. The study was supported through funds from The European Community’s Seventh Framework Programme (FP7/2007-2013), the BioSHaRE Consortium (261433), the Sigrid Juselius Foundation (251217 to S.R.), the Academy of Finland (137870 to P.S. and 135973 to P.W.), the Responding to Public Health Challenges Research Programme of the Academy of Finland (129269 to M.J.S., 129429 to M.A.-K., 129322 to M.P. and 139635 to V.S.), the Academy of Finland Center of Excellence in Complex Disease Genetics (213506 and 129680 to A.P., J. Kaprio, L.P., K.S. and S.R.), the Finnish Foundation for Cardiovascular Research (to M.J.S., M.A.-K., M.P., S.R. and K.H.P), the Jenny and Antti Wihuri Foundation (to A.J.K.), the Instrumentarium Science Foundation (to T.T. and P.W.), the Finnish Cultural Foundation (to T.T. and T.L.), an Aalto University School of Science and Technology researcher training scholarship (to T.T.) and the Wellcome Trust (098051 to A.P.). The Young Finns Study has been financially supported by the Academy of Finland (126925, 121584, 124282, 129378 (Salve), 117787 (Gendi) and 41071 (Skidi)), the Social Insurance Institution of Finland, the Turku University Foundation, the Yrjö Jahnsson Foundation, the Emil Aaltonen Foundation (to T.L.), the Medical Research Fund of Tampere University Hospital, the Turku University Hospital Medical Fund, the Juho Vainio Foundation, the Finnish Foundation for Cardiovascular Research (to T.L.) and the Tampere Tuberculosis Foundation (to T.L. and M.K.). The Helsinki Birth Cohort Study has been supported by grants from the Academy of Finland (120386, 125876 and 126775 to J.E.), the Finnish Diabetes Research Society, the Novo Nordisk Foundation, the European Science Foundation (EuroSTRESS), the Wellcome Trust (89061/Z/09/Z and 089062/Z/09/Z), the Samfundet Folkhälsan and the Finska Läkaresällskapet. The FINRISK/DILGOM study was supported by the Academy of Finland (118081). Data collection for FinnTwin12 and FinnTwin16 were supported by the National Institute on Alcohol Abuse and Alcoholism (NIAAA) (AA-12502, AA-09203 and AA-08315 to R.J.R. and AA-15416 to D.M.D.) and the Academy of Finland (100499, 205585, 118555 and 141054 (Skidi-Kids) to J. Kaprio). The Finnish Twin cohorts are also supported by the Novo Nordisk Foundation, the Diabetes Research Foundation, Biomedicum Helsinki and Helsinki University Central Hospital grants (all to K.H.P.). NFBC1966 received financial support from the Academy of Finland (104781, 120315, 129269, 1114194, 139900 and SALVE to M.-R.J. and Center of Excellence in Complex Disease Genetics to L.P.), University Hospital Oulu, Biocenter, University of Oulu (75617 to M.-R.J. and M.J.S.), the European Commission EURO-BLCS Framework 5 award (QLG1-CT-2000-01643 to M.-R.J.), the US National Heart, Lung, and Blood Institute (NHLBI) (5R01HL087679), the US National Institute of Mental Health (NIMH) (1RL1MH083268), European Network for Genetic and Genomic Epidemiology (ENGAGE) (HEALTH-F4-2007-201413 to L.P. and M.-R.J.), the MRC UK (G0500539, G0600705 and PrevMetSyn/Salve to M.-R.J.) and the Wellcome Trust (GR069224).</p></div></div><div id="idm140378858864032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140378858864032title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN4"></p><p id="__p4" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p><div><p id="__p5" class="p p-first"><strong>AUTHOR CONTRIBUTIONS</strong></p>Experiments were designed by L.P., M.P., M.A.-K., A.P. and S.R. Statistical analyses were performed by J. Kettunen, T.T., A.O.-A., E.T. and L.-P.L. Materials and/or analysis tools were contributed by J. Kettunen, T.T., A.-P.S., P.S., A.J.K., P.W., K.S., D.M.D., R.J.R., M.J.S., J.V., M.K., T.L., K.H.P., M.I.M., A.J., J.E., O.T.R., V.S., J. Kaprio, M.-R.J., N.B.F., M.A.-K., A.P. and S.R. The manuscript was written by J. Kettunen, T.T., A.J.K., M.I., N.B.F., M.A.-K., A.P. and S.R. All authors reviewed the manuscript.</div><p class="fn sec" id="FN6"> </p><p id="__p7" class="p p-first"><strong>COMPETING FINANCIAL INTERESTS</strong></p><p id="__p8" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN7"></p><p id="__p9" class="p p-first-last">Reprints and permissions information is available online at <a href="http://www.nature.com/reprints/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nature.com/reprints/index.html</a>.</p><p></p></div></div><div id="idm140378868513136" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140378868513136title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Schlessinger BS, Wilson FH, Jr, Milch LJ. Serum parameters as discriminators between normal and coronary groups. <span><span class="ref-journal">Circulation. </span>1959;<span class="ref-vol">19</span>:265–268.</span> [<a href="/pubmed/13629788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Serum+parameters+as+discriminators+between+normal+and+coronary+groups&amp;author=BS+Schlessinger&amp;author=FH+Wilson&amp;author=LJ+Milch&amp;volume=19&amp;publication_year=1959&amp;pages=265-268&amp;pmid=13629788&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Stumvoll M, Goldstein BJ, van Haeften TW. Type 2 diabetes: principles of pathogenesis and therapy. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:1333–1346.</span> [<a href="/pubmed/15823385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Type+2+diabetes:+principles+of+pathogenesis+and+therapy&amp;author=M+Stumvoll&amp;author=BJ+Goldstein&amp;author=TW+van+Haeften&amp;volume=365&amp;publication_year=2005&amp;pages=1333-1346&amp;pmid=15823385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Dupuis J, et al.  New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:105–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3018764/">PMC free article</a>]</span> [<a href="/pubmed/20081858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+genetic+loci+implicated+in+fasting+glucose+homeostasis+and+their+impact+on+type+2+diabetes+risk&amp;author=J+Dupuis&amp;volume=42&amp;publication_year=2010&amp;pages=105-116&amp;pmid=20081858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Kolz M, et al.  Meta-analysis of 28,141 individuals identifies common variants within five new loci that influence uric acid concentrations. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683940/">PMC free article</a>]</span> [<a href="/pubmed/19503597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Meta-analysis+of+28,141+individuals+identifies+common+variants+within+five+new+loci+that+influence+uric+acid+concentrations&amp;author=M+Kolz&amp;volume=5&amp;publication_year=2009&amp;pages=e1000504&amp;pmid=19503597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Teslovich TM, et al.  Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=TM+Teslovich&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Wang TJ, et al.  Metabolite profiles and the risk of developing diabetes. <span><span class="ref-journal">Nat Med. </span>2011;<span class="ref-vol">17</span>:448–453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3126616/">PMC free article</a>]</span> [<a href="/pubmed/21423183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=Metabolite+profiles+and+the+risk+of+developing+diabetes&amp;author=TJ+Wang&amp;volume=17&amp;publication_year=2011&amp;pages=448-453&amp;pmid=21423183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Dunn WB, Broadhurst DI, Atherton HJ, Goodacre R, Griffin JL. Systems level studies of mammalian metabolomes: the roles of mass spectrometry and nuclear magnetic resonance spectroscopy. <span><span class="ref-journal">Chem Soc Rev. </span>2011;<span class="ref-vol">40</span>:387–426.</span> [<a href="/pubmed/20717559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem+Soc+Rev&amp;title=Systems+level+studies+of+mammalian+metabolomes:+the+roles+of+mass+spectrometry+and+nuclear+magnetic+resonance+spectroscopy&amp;author=WB+Dunn&amp;author=DI+Broadhurst&amp;author=HJ+Atherton&amp;author=R+Goodacre&amp;author=JL+Griffin&amp;volume=40&amp;publication_year=2011&amp;pages=387-426&amp;pmid=20717559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Suhre K, et al.  Human metabolic individuality in biomedical and pharmaceutical research. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">477</span>:54–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832838/">PMC free article</a>]</span> [<a href="/pubmed/21886157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+metabolic+individuality+in+biomedical+and+pharmaceutical+research&amp;author=K+Suhre&amp;volume=477&amp;publication_year=2011&amp;pages=54-60&amp;pmid=21886157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Illig T, et al.  A genome-wide perspective of genetic variation in human metabolism. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:137–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773904/">PMC free article</a>]</span> [<a href="/pubmed/20037589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+perspective+of+genetic+variation+in+human+metabolism&amp;author=T+Illig&amp;volume=42&amp;publication_year=2010&amp;pages=137-141&amp;pmid=20037589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Gieger C, et al.  Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+a+genome-wide+association+study+of+metabolite+profiles+in+human+serum&amp;author=C+Gieger&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Soeters PB, Fischer JE. Insulin, glucagon, aminoacid imbalance, and hepatic encephalopathy. <span><span class="ref-journal">Lancet. </span>1976;<span class="ref-vol">2</span>:880–882.</span> [<a href="/pubmed/62115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Insulin,+glucagon,+aminoacid+imbalance,+and+hepatic+encephalopathy&amp;author=PB+Soeters&amp;author=JE+Fischer&amp;volume=2&amp;publication_year=1976&amp;pages=880-882&amp;pmid=62115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">International HapMap 3 Consortium. Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Surakka I, et al.  Founder population–specific HapMap panel increases power in GWA studies through improved imputation accuracy and CNV tagging. <span><span class="ref-journal">Genome Res. </span>2010;<span class="ref-vol">20</span>:1344–1351.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2945183/">PMC free article</a>]</span> [<a href="/pubmed/20810666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Founder+population–specific+HapMap+panel+increases+power+in+GWA+studies+through+improved+imputation+accuracy+and+CNV+tagging&amp;author=I+Surakka&amp;volume=20&amp;publication_year=2010&amp;pages=1344-1351&amp;pmid=20810666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Hindorff LA, et al.  Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;pmid=19474294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Natt E, Kida K, Odievre M, Di Rocco M, Scherer G. Point mutations in the tyrosine aminotransferase gene in tyrosinemia type II. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1992;<span class="ref-vol">89</span>:9297–9301.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC50113/">PMC free article</a>]</span> [<a href="/pubmed/1357662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Point+mutations+in+the+tyrosine+aminotransferase+gene+in+tyrosinemia+type+II&amp;author=E+Natt&amp;author=K+Kida&amp;author=M+Odievre&amp;author=M+Di+Rocco&amp;author=G+Scherer&amp;volume=89&amp;publication_year=1992&amp;pages=9297-9301&amp;pmid=1357662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Orinska Z, et al.  TLR3-induced activation of mast cells modulates CD8<sup>+</sup> T-cell recruitment. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:978–987.</span> [<a href="/pubmed/15840693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=TLR3-induced+activation+of+mast+cells+modulates+CD8++T-cell+recruitment&amp;author=Z+Orinska&amp;volume=106&amp;publication_year=2005&amp;pages=978-987&amp;pmid=15840693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Marone G, Galli SJ, Kitamura Y. Probing the roles of mast cells and basophils in natural and acquired immunity, physiology and disease. <span><span class="ref-journal">Trends Immunol. </span>2002;<span class="ref-vol">23</span>:425–427.</span> [<a href="/pubmed/12200056" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Immunol&amp;title=Probing+the+roles+of+mast+cells+and+basophils+in+natural+and+acquired+immunity,+physiology+and+disease&amp;author=G+Marone&amp;author=SJ+Galli&amp;author=Y+Kitamura&amp;volume=23&amp;publication_year=2002&amp;pages=425-427&amp;pmid=12200056&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Cross LJ, Heaney LG, Ennis M. Histamine release from human bronchoalveolar lavage mast cells by neurokinin A and bradykinin. <span><span class="ref-journal">Inflamm Res. </span>1997;<span class="ref-vol">46</span>:306–309.</span> [<a href="/pubmed/9297575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Inflamm+Res&amp;title=Histamine+release+from+human+bronchoalveolar+lavage+mast+cells+by+neurokinin+A+and+bradykinin&amp;author=LJ+Cross&amp;author=LG+Heaney&amp;author=M+Ennis&amp;volume=46&amp;publication_year=1997&amp;pages=306-309&amp;pmid=9297575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Bouatia-Naji N, et al.  A polymorphism within the <em>G6PC2</em> gene is associated with fasting plasma glucose levels. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">320</span>:1085–1088.</span> [<a href="/pubmed/18451265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+polymorphism+within+the+G6PC2+gene+is+associated+with+fasting+plasma+glucose+levels&amp;author=N+Bouatia-Naji&amp;volume=320&amp;publication_year=2008&amp;pages=1085-1088&amp;pmid=18451265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Prokopenko I, et al.  Variants in <em>MTNR1B</em> influence fasting glucose levels. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:77–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682768/">PMC free article</a>]</span> [<a href="/pubmed/19060907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+in+MTNR1B+influence+fasting+glucose+levels&amp;author=I+Prokopenko&amp;volume=41&amp;publication_year=2009&amp;pages=77-81&amp;pmid=19060907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Shin Y, Vaziri ND, Willekes N, Kim CH, Joles JA. Effects of gender on hepatic HMG-CoA reductase, cholesterol 7α-hydroxylase, and LDL receptor in hereditary analbuminemia. <span><span class="ref-journal">Am J Physiol Endocrinol Metab. </span>2005;<span class="ref-vol">289</span>:E993–E998.</span> [<a href="/pubmed/16030068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Endocrinol+Metab&amp;title=Effects+of+gender+on+hepatic+HMG-CoA+reductase,+cholesterol+7α-hydroxylase,+and+LDL+receptor+in+hereditary+analbuminemia&amp;author=Y+Shin&amp;author=ND+Vaziri&amp;author=N+Willekes&amp;author=CH+Kim&amp;author=JA+Joles&amp;volume=289&amp;publication_year=2005&amp;pages=E993-E998&amp;pmid=16030068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Rosipal S, Debreova M, Rosipal R. A speculation about hypercholesterolemia in congenital analbuminemia. <span><span class="ref-journal">Am J Med. </span>2006;<span class="ref-vol">119</span>:181–182.</span> [<a href="/pubmed/16443434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med&amp;title=A+speculation+about+hypercholesterolemia+in+congenital+analbuminemia&amp;author=S+Rosipal&amp;author=M+Debreova&amp;author=R+Rosipal&amp;volume=119&amp;publication_year=2006&amp;pages=181-182&amp;pmid=16443434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Koot BG, Houwen R, Pot DJ, Nauta J. Congenital analbuminaemia: biochemical and clinical implications. A case report and literature review. <span><span class="ref-journal">Eur J Pediatr. </span>2004;<span class="ref-vol">163</span>:664–670.</span> [<a href="/pubmed/15300429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Pediatr&amp;title=Congenital+analbuminaemia:+biochemical+and+clinical+implications.+A+case+report+and+literature+review&amp;author=BG+Koot&amp;author=R+Houwen&amp;author=DJ+Pot&amp;author=J+Nauta&amp;volume=163&amp;publication_year=2004&amp;pages=664-670&amp;pmid=15300429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Zhao M, et al.  FcγRIIB inhibits the development of atherosclerosis in low- density lipoprotein receptor–deficient mice. <span><span class="ref-journal">J Immunol. </span>2010;<span class="ref-vol">184</span>:2253–2260.</span> [<a href="/pubmed/20097865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=FcγRIIB+inhibits+the+development+of+atherosclerosis+in+low-+density+lipoprotein+receptor–deficient+mice&amp;author=M+Zhao&amp;volume=184&amp;publication_year=2010&amp;pages=2253-2260&amp;pmid=20097865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Hernández-Vargas P, et al.  Fcγ receptor deficiency confers protection against atherosclerosis in apolipoprotein E knockout mice. <span><span class="ref-journal">Circ Res. </span>2006;<span class="ref-vol">99</span>:1188–1196.</span> [<a href="/pubmed/17053192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Res&amp;title=Fcγ+receptor+deficiency+confers+protection+against+atherosclerosis+in+apolipoprotein+E+knockout+mice&amp;author=P+Hernández-Vargas&amp;volume=99&amp;publication_year=2006&amp;pages=1188-1196&amp;pmid=17053192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Cohen PT. Protein phosphatase 1—targeted in many directions. <span><span class="ref-journal">J Cell Sci. </span>2002;<span class="ref-vol">115</span>:241–256.</span> [<a href="/pubmed/11839776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Sci&amp;title=Protein+phosphatase+1—targeted+in+many+directions&amp;author=PT+Cohen&amp;volume=115&amp;publication_year=2002&amp;pages=241-256&amp;pmid=11839776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Chasman DI, et al.  Forty-three loci associated with plasma lipoprotein size, concentration, and cholesterol content in genome-wide analysis. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000730.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2777390/">PMC free article</a>]</span> [<a href="/pubmed/19936222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Forty-three+loci+associated+with+plasma+lipoprotein+size,+concentration,+and+cholesterol+content+in+genome-wide+analysis&amp;author=DI+Chasman&amp;volume=5&amp;publication_year=2009&amp;pages=e1000730&amp;pmid=19936222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Aulchenko YS, et al.  Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:47–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;author=YS+Aulchenko&amp;volume=41&amp;publication_year=2009&amp;pages=47-55&amp;pmid=19060911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Bergstrom JD, Reitz RC. Studies on carnitine palmitoyl transferase: the similar nature of CPTi (inner form) and CPTo (outer form) <span><span class="ref-journal">Arch Biochem Biophys. </span>1980;<span class="ref-vol">204</span>:71–79.</span> [<a href="/pubmed/7425647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Biochem+Biophys&amp;title=Studies+on+carnitine+palmitoyl+transferase:+the+similar+nature+of+CPTi+(inner+form)+and+CPTo+(outer+form)&amp;author=JD+Bergstrom&amp;author=RC+Reitz&amp;volume=204&amp;publication_year=1980&amp;pages=71-79&amp;pmid=7425647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Willer CJ, et al.  Newly identified loci that influence lipid concentrations and risk of coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:161–169.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;author=CJ+Willer&amp;volume=40&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Kathiresan S, et al.  Six new loci associated with blood low-density lipoprotein cholesterol, high-density lipoprotein cholesterol or triglycerides in humans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:189–197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Six+new+loci+associated+with+blood+low-density+lipoprotein+cholesterol,+high-density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;author=S+Kathiresan&amp;volume=40&amp;publication_year=2008&amp;pages=189-197&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Wallace C, et al.  Genome-wide association study identifies genes for biomarkers of cardiovascular disease: serum urate and dyslipidemia. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:139–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253977/">PMC free article</a>]</span> [<a href="/pubmed/18179892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+study+identifies+genes+for+biomarkers+of+cardiovascular+disease:+serum+urate+and+dyslipidemia&amp;author=C+Wallace&amp;volume=82&amp;publication_year=2008&amp;pages=139-149&amp;pmid=18179892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Zhang S, et al.  Genetic and environmental contributions to phenotypic components of metabolic syndrome: a population-based twin study. <span><span class="ref-journal">Obesity (Silver Spring) </span>2009;<span class="ref-vol">17</span>:1581–1587.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3766632/">PMC free article</a>]</span> [<a href="/pubmed/19407809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity+(Silver+Spring)&amp;title=Genetic+and+environmental+contributions+to+phenotypic+components+of+metabolic+syndrome:+a+population-based+twin+study&amp;author=S+Zhang&amp;volume=17&amp;publication_year=2009&amp;pages=1581-1587&amp;pmid=19407809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Isaacs A, et al.  Heritabilities, apolipoprotein E, and effects of inbreeding on plasma lipids in a genetically isolated population: the Erasmus Rucphen Family Study. <span><span class="ref-journal">Eur J Epidemiol. </span>2007;<span class="ref-vol">22</span>:99–105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2797851/">PMC free article</a>]</span> [<a href="/pubmed/17318336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Epidemiol&amp;title=Heritabilities,+apolipoprotein+E,+and+effects+of+inbreeding+on+plasma+lipids+in+a+genetically+isolated+population:+the+Erasmus+Rucphen+Family+Study&amp;author=A+Isaacs&amp;volume=22&amp;publication_year=2007&amp;pages=99-105&amp;pmid=17318336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Li Y, Willer CJ, Ding J, Scheet P, Abecasis GR. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y+Li&amp;author=CJ+Willer&amp;author=J+Ding&amp;author=P+Scheet&amp;author=GR+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Mägi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R+Mägi&amp;author=AP+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Soininen P, et al.  High-throughput serum NMR metabonomics for cost-effective holistic studies on systemic metabolism. <span><span class="ref-journal">Analyst. </span>2009;<span class="ref-vol">134</span>:1781–1785.</span> [<a href="/pubmed/19684899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=High-throughput+serum+NMR+metabonomics+for+cost-effective+holistic+studies+on+systemic+metabolism&amp;author=P+Soininen&amp;volume=134&amp;publication_year=2009&amp;pages=1781-1785&amp;pmid=19684899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">R Development Core Team. <span class="ref-journal">R: A Language and Environment for Statistical Computing.</span> R Foundation for Statistical Computing; Vienna, Austria: 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+Language+and+Environment+for+Statistical+Computing&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Inouye M, et al.  An immune response network associated with blood lipid levels. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2936545/">PMC free article</a>]</span> [<a href="/pubmed/20844574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=An+immune+response+network+associated+with+blood+lipid+levels&amp;author=M+Inouye&amp;volume=6&amp;publication_year=2010&amp;pages=e1001113&amp;pmid=20844574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Ding J, et al.  Gene expression in skin and lymphoblastoid cells: refined statistical method reveals extensive overlap in <em>cis</em>-eQTL signals. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:779–789.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997368/">PMC free article</a>]</span> [<a href="/pubmed/21129726" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Gene+expression+in+skin+and+lymphoblastoid+cells:+refined+statistical+method+reveals+extensive+overlap+in+cis-eQTL+signals&amp;author=J+Ding&amp;volume=87&amp;publication_year=2010&amp;pages=779-789&amp;pmid=21129726&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Teo YY, et al.  A genotype calling algorithm for the Illumina BeadArray platform. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:2741–2746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3605033&amp;issue-id=220951&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140378869193584title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S7title">DISCUSSION</a></li><li><a href="#S10title">ONLINE METHODS</a></li><li><a href="#S18title">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm140378858864032title">Footnotes</a></li><li><a href="#idm140378868513136title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3605033/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3605033/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3605033/pdf/nihms444574.pdf">PDF (1.8M)</a></li> | <li><a href="#" data-citationid="PMC3605033" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3605033%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3605033%2F&amp;text=Genome-wide%20association%20study%20identifies%20multiple%20loci%20influencing%20human%20serum%20metabolite%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3605033%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22286219" data-aiid="3605033" data-aid="3605033" data-iid="220951" data-domainid="319" data-domain="nihpa" data-accid="PMC3605033" data-md5="3111e733da070855f9c32be662a8f22a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:27:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B2C09DD5384810000000004C1031F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3605033%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3605033/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>