<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3772310/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Multiple type 2 diabetes susceptibility genes following genome-wide association scan in UK samples</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772310/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Science (New York, N.Y.)"><meta name="citation_title" content="Multiple type 2 diabetes susceptibility genes following genome-wide association scan in UK samples"><meta name="citation_authors" content="Eleftheria Zeggini, Michael N. Weedon, Cecilia M. Lindgren, Timothy M. Frayling, Katherine S. Elliott, Hana Lango, Nicholas J. Timpson, John R.B. Perry, Nigel W. Rayner, Rachel M. Freathy, Jeffrey C. Barrett, Beverley Shields, Andrew P. Morris, Sian Ellard, Christopher J. Groves, Lorna W. Harries, Jonathan L. Marchini, Katharine R. Owen, Beatrice Knight, Lon R. Cardon, Mark Walker, Graham A. Hitman, Andrew D. Morris, Alex S.F. Doney, The Wellcome Trust Case Control Consortium, Mark I. McCarthy, Andrew T. Hattersley"><meta name="citation_date" content="1 June 2007"><meta name="citation_issue" content="5829"><meta name="citation_volume" content="316"><meta name="citation_firstpage" content="1336"><meta name="citation_doi" content="10.1126/science.1142364"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3772310/?report=abstract"><meta name="citation_pmid" content="17463249"><meta name="DC.Title" content="Multiple type 2 diabetes susceptibility genes following genome-wide association scan in UK samples"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Eleftheria Zeggini"><meta name="DC.Contributor" content="Michael N. Weedon"><meta name="DC.Contributor" content="Cecilia M. Lindgren"><meta name="DC.Contributor" content="Timothy M. Frayling"><meta name="DC.Contributor" content="Katherine S. Elliott"><meta name="DC.Contributor" content="Hana Lango"><meta name="DC.Contributor" content="Nicholas J. Timpson"><meta name="DC.Contributor" content="John R.B. Perry"><meta name="DC.Contributor" content="Nigel W. Rayner"><meta name="DC.Contributor" content="Rachel M. Freathy"><meta name="DC.Contributor" content="Jeffrey C. Barrett"><meta name="DC.Contributor" content="Beverley Shields"><meta name="DC.Contributor" content="Andrew P. Morris"><meta name="DC.Contributor" content="Sian Ellard"><meta name="DC.Contributor" content="Christopher J. Groves"><meta name="DC.Contributor" content="Lorna W. Harries"><meta name="DC.Contributor" content="Jonathan L. Marchini"><meta name="DC.Contributor" content="Katharine R. Owen"><meta name="DC.Contributor" content="Beatrice Knight"><meta name="DC.Contributor" content="Lon R. Cardon"><meta name="DC.Contributor" content="Mark Walker"><meta name="DC.Contributor" content="Graham A. Hitman"><meta name="DC.Contributor" content="Andrew D. Morris"><meta name="DC.Contributor" content="Alex S.F. Doney"><meta name="DC.Contributor" content="The Wellcome Trust Case Control Consortium"><meta name="DC.Contributor" content="Mark I. McCarthy"><meta name="DC.Contributor" content="Andrew T. Hattersley"><meta name="DC.Date" content="2007 Jun 1"><meta name="DC.Identifier" content="10.1126/science.1142364"><meta name="DC.Language" content="en"><meta property="og:title" content="Multiple type 2 diabetes susceptibility genes following genome-wide association scan in UK samples"><meta property="og:type" content="article"><meta property="og:description" content="The molecular mechanisms involved in the development of type 2 diabetes are poorly understood. Starting from genome-wide genotype data for 1,924 diabetic cases and 2,938 population controls generated by the Wellcome Trust Case Control Consortium, we set ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3772310/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3772310/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3772310/pdf/emss-54288.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A771EDD3E93610000000003E20254.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3772310%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene">Gene</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3772310</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Science</a></span>. Author manuscript; available in PMC 2013 Sep 13.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17463249" target="pmc_ext" ref="reftype=publisher&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Science. 2007 Jun 1; 316(5829): 1336–1341. </span></a></div><span class="fm-vol-iss-date">Published online 2007 Apr 26. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1126%2Fscience.1142364" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1126/science.1142364</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3772310</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS54288</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/17463249">17463249</a></div></div></div></div><h1 class="content-title">Multiple type 2 diabetes susceptibility genes following genome-wide association scan in UK samples</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zeggini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001531940816" co-class="co-affbox">Eleftheria Zeggini</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001498919120" co-class="co-affbox">Michael N. Weedon</a>,<sup>3,</sup><sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001497388544" co-class="co-affbox">Cecilia M. Lindgren</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001531943872" co-class="co-affbox">Timothy M. Frayling</a>,<sup>3,</sup><sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=Elliott%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001566033488" co-class="co-affbox">Katherine S. Elliott</a>,<sup>2</sup> <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001532780048" co-class="co-affbox">Hana Lango</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001532710736" co-class="co-affbox">Nicholas J. Timpson</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001492121664" co-class="co-affbox">John R.B. Perry</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Rayner%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001491153024" co-class="co-affbox">Nigel W. Rayner</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001566083376" co-class="co-affbox">Rachel M. Freathy</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Barrett%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001485521056" co-class="co-affbox">Jeffrey C. Barrett</a>,<sup>2</sup> <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001498947424" co-class="co-affbox">Beverley Shields</a>,<sup>4</sup> <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001500487872" co-class="co-affbox">Andrew P. Morris</a>,<sup>2</sup> <a href="/pubmed/?term=Ellard%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001493320752" co-class="co-affbox">Sian Ellard</a>,<sup>4,</sup><sup>6</sup> <a href="/pubmed/?term=Groves%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001499251136" co-class="co-affbox">Christopher J. Groves</a>,<sup>1</sup> <a href="/pubmed/?term=Harries%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001494883024" co-class="co-affbox">Lorna W. Harries</a>,<sup>4</sup> <a href="/pubmed/?term=Marchini%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001495279184" co-class="co-affbox">Jonathan L. Marchini</a>,<sup>7</sup> <a href="/pubmed/?term=Owen%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001500002544" co-class="co-affbox">Katharine R. Owen</a>,<sup>1</sup> <a href="/pubmed/?term=Knight%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001492957904" co-class="co-affbox">Beatrice Knight</a>,<sup>4</sup> <a href="/pubmed/?term=Cardon%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001492507952" co-class="co-affbox">Lon R. Cardon</a>,<sup>2</sup> <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001496001696" co-class="co-affbox">Mark Walker</a>,<sup>8</sup> <a href="/pubmed/?term=Hitman%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001500098544" co-class="co-affbox">Graham A. Hitman</a>,<sup>9</sup> <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001500096560" co-class="co-affbox">Andrew D. Morris</a>,<sup>10</sup> <a href="/pubmed/?term=Doney%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001496011488" co-class="co-affbox">Alex S.F. Doney</a>,<sup>10</sup> The Wellcome Trust Case Control Consortium,<sup>11</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001493599440" co-class="co-affbox">Mark I. McCarthy</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> and  <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249" class="affpopup" co-rid="_co_idm140001492983872" co-class="co-affbox">Andrew T. Hattersley</a><sup>3,</sup><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140001531940816"><h3 class="no_margin">Eleftheria Zeggini</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Zeggini%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Eleftheria Zeggini</a></div></div><div id="_co_idm140001498919120"><h3 class="no_margin">Michael N. Weedon</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Weedon%20MN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Michael N. Weedon</a></div></div><div id="_co_idm140001497388544"><h3 class="no_margin">Cecilia M. Lindgren</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Cecilia M. Lindgren</a></div></div><div id="_co_idm140001531943872"><h3 class="no_margin">Timothy M. Frayling</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Frayling%20TM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Timothy M. Frayling</a></div></div><div id="_co_idm140001566033488"><h3 class="no_margin">Katherine S. Elliott</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Elliott%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Katherine S. Elliott</a></div></div><div id="_co_idm140001532780048"><h3 class="no_margin">Hana Lango</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Lango%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Hana Lango</a></div></div><div id="_co_idm140001532710736"><h3 class="no_margin">Nicholas J. Timpson</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><p><sup>5</sup>The MRC Centre for Causal Analyses in Translational Epidemiology, Bristol University, Canynge Hall, Whiteladies Rd, Bristol, BS2 8PR, UK </p><div>Find articles by <a href="/pubmed/?term=Timpson%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Nicholas J. Timpson</a></div></div><div id="_co_idm140001492121664"><h3 class="no_margin">John R.B. Perry</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Perry%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">John R.B. Perry</a></div></div><div id="_co_idm140001491153024"><h3 class="no_margin">Nigel W. Rayner</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Rayner%20NW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Nigel W. Rayner</a></div></div><div id="_co_idm140001566083376"><h3 class="no_margin">Rachel M. Freathy</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Freathy%20RM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Rachel M. Freathy</a></div></div><div id="_co_idm140001485521056"><h3 class="no_margin">Jeffrey C. Barrett</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Barrett%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Jeffrey C. Barrett</a></div></div><div id="_co_idm140001498947424"><h3 class="no_margin">Beverley Shields</h3><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Shields%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Beverley Shields</a></div></div><div id="_co_idm140001500487872"><h3 class="no_margin">Andrew P. Morris</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Andrew P. Morris</a></div></div><div id="_co_idm140001493320752"><h3 class="no_margin">Sian Ellard</h3><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><p><sup>6</sup>The Molecular Genetics Laboratory, Royal Devon and Exeter NHS Foundation Trust, Old Pathology Building, Barrack Road, Exeter, EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Ellard%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Sian Ellard</a></div></div><div id="_co_idm140001499251136"><h3 class="no_margin">Christopher J. Groves</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><div>Find articles by <a href="/pubmed/?term=Groves%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Christopher J. Groves</a></div></div><div id="_co_idm140001494883024"><h3 class="no_margin">Lorna W. Harries</h3><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Harries%20LW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Lorna W. Harries</a></div></div><div id="_co_idm140001495279184"><h3 class="no_margin">Jonathan L. Marchini</h3><p><sup>7</sup>Department of Statistics, University of Oxford, 1 South Parks Road, Oxford, OX1 3TG, UK </p><div>Find articles by <a href="/pubmed/?term=Marchini%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Jonathan L. Marchini</a></div></div><div id="_co_idm140001500002544"><h3 class="no_margin">Katharine R. Owen</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><div>Find articles by <a href="/pubmed/?term=Owen%20KR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Katharine R. Owen</a></div></div><div id="_co_idm140001492957904"><h3 class="no_margin">Beatrice Knight</h3><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Knight%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Beatrice Knight</a></div></div><div id="_co_idm140001492507952"><h3 class="no_margin">Lon R. Cardon</h3><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=Cardon%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Lon R. Cardon</a></div></div><div id="_co_idm140001496001696"><h3 class="no_margin">Mark Walker</h3><p><sup>8</sup>Diabetes Research Group, School of Clinical Medical Sciences, Newcastle University, Framlington Place, Newcastle upon Tyne NE2 4HH, UK </p><div>Find articles by <a href="/pubmed/?term=Walker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Mark Walker</a></div></div><div id="_co_idm140001500098544"><h3 class="no_margin">Graham A. Hitman</h3><p><sup>9</sup>Centre for Diabetes and Metabolic Medicine, Barts and The London, Royal London Hospital, Whitechapel, London, E1 1BB, UK </p><div>Find articles by <a href="/pubmed/?term=Hitman%20GA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Graham A. Hitman</a></div></div><div id="_co_idm140001500096560"><h3 class="no_margin">Andrew D. Morris</h3><p><sup>10</sup>Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, DD1 9SY, UK </p><div>Find articles by <a href="/pubmed/?term=Morris%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Andrew D. Morris</a></div></div><div id="_co_idm140001496011488"><h3 class="no_margin">Alex S.F. Doney</h3><p><sup>10</sup>Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, DD1 9SY, UK </p><div>Find articles by <a href="/pubmed/?term=Doney%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Alex S.F. Doney</a></div></div><div id="_co_idm140001497251520"><h3 class="no_margin">The Wellcome Trust Case Control Consortium</h3><p><sup>11</sup>Membership of the WTCCC is listed in <a href="#SD1" rid="SD1" class=" supplementary-material">supporting online text</a></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">The Wellcome Trust Case Control Consortium</a></div></div><div id="_co_idm140001493599440"><h3 class="no_margin">Mark I. McCarthy</h3><p><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </p><p><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Mark I. McCarthy</a></div></div><div id="_co_idm140001492983872"><h3 class="no_margin">Andrew T. Hattersley</h3><p><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </p><p><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </p><div>Find articles by <a href="/pubmed/?term=Hattersley%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=17463249">Andrew T. Hattersley</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140001490360000_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140001490360000_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140001490360000_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Oxford Centre for Diabetes, Endocrinology and Medicine, University of Oxford, Churchill Hospital, Oxford, OX3 7LJ, UK </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford, OX3 7BN, UK </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Genetics of Complex Traits, Institute of Biomedical and Clinical Science, Peninsula Medical School, Magdalen Road, Exeter EX1 2LU, UK </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Diabetes Genetics group, Institute of Biomedical and Clinical Science, Peninsula Medical School, Barrack Road, Exeter EX2 5DW, UK </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>The MRC Centre for Causal Analyses in Translational Epidemiology, Bristol University, Canynge Hall, Whiteladies Rd, Bristol, BS2 8PR, UK </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>The Molecular Genetics Laboratory, Royal Devon and Exeter NHS Foundation Trust, Old Pathology Building, Barrack Road, Exeter, EX2 5DW, UK </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Statistics, University of Oxford, 1 South Parks Road, Oxford, OX1 3TG, UK </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Diabetes Research Group, School of Clinical Medical Sciences, Newcastle University, Framlington Place, Newcastle upon Tyne NE2 4HH, UK </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Centre for Diabetes and Metabolic Medicine, Barts and The London, Royal London Hospital, Whitechapel, London, E1 1BB, UK </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Diabetes Research Group, Division of Medicine and Therapeutics, Ninewells Hospital and Medical School, Dundee, DD1 9SY, UK </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Membership of the WTCCC is listed in <a href="#SD1" rid="SD1" class=" supplementary-material">supporting online text</a></div><div id="CR1"> Corresponding author: Prof Mark I McCarthy Oxford Centre for Diabetes, Endocrinology and Metabolism University of Oxford Old Road, Headington, Oxford OX3 7LJ UK Tel: (44) 1865 857298 Fax: (44) 1865 857299 <a href="mailto:dev@null" data-email="ku.ca.xo.lrd@yhtraccm.kram" class="oemail">ku.ca.xo.lrd@yhtraccm.kram</a></div><div id="FN2"><sup>*</sup>EZ, MNW, CML, TMF contributed equally to the work described. MIMcC and ATH contributed equally.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140001490360000_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=17463249" target="pmc_ext" ref="reftype=publisher&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Science</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 317 on&nbsp;page&nbsp;1035.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3772310/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01: </strong>Figure S1: Overview of <em>FTO</em> signal region<p>A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p>B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p>C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p>D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p>E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p>F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p>Figure S2: Expression patterns of <em>CDKAL1, CDKN2A, CDKN2B, HHEX, IGF2BP2.</em> Messenger RNA expression profiles are shown for the genes listed for a range of human tissues, as determined by RT-PCR. Figures on the y axes refer to the test transcript levels relative to two separate endogenous control genes (beta glucuronidase [<em>BGUS</em>] and beta 2 microglobulin [<em>B2M</em>]). Each test:control ratio was then normalized to that of adult human pancreas for tissue to tissue comparison.</p><p>Figure S3: Overview of <em>HHEX</em> signal region</p><p>A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples. Note that rs5015480 was typed in the WTCCC scan and rs1111875 in the replication set, so the meta-analysis result is based on a combined analysis of the two (r<sup>2</sup>=1 in HapMap CEU) and the position of this signal denoted at both locations.</p><p>B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p>C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p>D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p>E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p>F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p>Figure S4: Overview of <em>IGF2BP2</em> signal region</p><p>A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p>B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p>C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p>D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p>E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p>F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p>Figure S5: Overview of <em>VEGFA</em> signal region</p><p>A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p>B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p>C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p>D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p>E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p>F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p>Figure S6: Quantile-Quantile plots for the 393,453 SNPs passing T2D-specific quality control. The plot compares the observed and expected distributions for the 1df chi-square statistics generated from single-point Cochran-Armitage tests.</p><p>Figure S7: Correlation plot of association statistics for the WTCCC scan genotypes before and after adjusting for population structure. The plot compares the 1df χ<sup>2</sup> (single-point Cochran-Armitage) values obtained from a naïve trend test for the 393.453 SNPs passing T2D-specific quality control, with the equivalent statistics generated after correcting for population substructure using Eigenstrat (<em>S8</em>). The high correlation overall (r<sup>2</sup>&gt;0.99), and in particular the strong linearity for high χ<sup>2</sup> values (top right) indicates minimal confounding from population substructure after implementation of the various QC measures described.</p><p>Figure S8: Single-point and haplotype-based analysis results for the chr9 signal region using GENEBPM. Circles denote single-point analysis results and the continuous line represents 5 SNP sliding window haplotype-based analyses using GENEBPM (<em>S15</em>). The two peaks of association are separated by a recombination hotspot (<a href="/pmc/articles/PMC3772310/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Multipoint analyses reveal much stronger evidence for association at the 3′ peak in this region. Common (&gt;0.01) high risk haplotypes in the 3′ signal share alleles T and T at SNPs rs10811661 and rs10757283 respectively. In the WTCCC scan (<em>S1</em>), Cochran-Armitage p values for rs2891169 were similar to those of SNPs rs10811661 and rs10757283, which were selected for replication. In contrast, GENEBPM analysis indicated stronger evidence for single-point association for rs2891169.</p><p><strong>Table S1: Clinical characteristics of UK samples.</strong> WTCCC controls came from two sources. No data on age, waist circumference or BMI are available for the UK Blood Service controls. Control individuals from the 1958 Birth Cohort were last reviewed at age 41. Under the terms of access, waist circumference and BMI values from these controls are not available to WTCCC researchers. Only 46% (all male) of the RS2 control individuals had available waist circumference measures.</p><p>Table S2: Replication results for SNPs selected for Cochran-Armitage p&lt;10<sup>−5</sup> on WTCCC scan. Alleles in this table are named alphabetically (as per the forward strand) with the ancestral allele underlined (where known). A2F denotes allele 2 frequency. For consistency, all ORs in this table are reported for allele 2 (and may therefore be the reciprocal of the ORs reported in the text and <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). It was not possible to design a working replication assay for rs5015480, and the UK meta-analysis for this signal combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU).</p><p>Table S3: Confirmed T2D susceptibility signals: SNPs reported for the DGI and FUSION studies. As DGI and FUSION did not always type the same SNPs as the UK study in all their samples, results in <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> include data from the SNPs generating the strongest association in their respective studies. Table S3 gives details of the SNPs reported for DGI and FUSION, and their LD relationships (based on HapMap CEU and/or genome-wide or imputation data as available) with the UK index SNP. In all cases these proxies were SNPs in strong LD (r<sup>2</sup>&gt;0.95, except <em>TCF7L2</em>) and showed consistent direction of effect with the SNP reported in the UK data.</p><p>Table S4: Replication results for SNPs selected for the “second wave” of replication. Alleles in this table are named alphabetically (as per the forward strand) with the ancestral allele underlined (where known). A2F denotes allele 2 frequency. For consistency, all ORs in this table are reported for allele 2 (and may therefore be the reciprocal of the ORs reported in the text). These signals were (with the exception of rs13266634 in <em>SLC30A8</em>) selected on the basis of Cochran-Armitage test p values between 10<sup>−2</sup> and 10<sup>−5</sup> on the WTCCC scan, prioritized on the basis of biological candidacy, multiple independent associations and/or support from DGI and/or FUSION scans. It was not possible to design a working replication assay for rs11140802, and the UK meta-analysis for this signal combines data from rs11140802 and rs12346884 (r<sup>2</sup>=1 in HapMap CEU). Rs13266634 in <em>SLC30A8</em> was not captured on the Affymetrix chip but was selected for replication on the basis of the associations in French (<em>S18</em>) and FUSION (<em>S17</em>) subjects.</p><p>Table S5: Associations between T2D susceptibility variants and (a) BMI, (b) waist circumference in cases and controls. Analyses were performed using linear regression on (a) log<sub>10</sub>-transformed BMI values and (b) on waist circumference values (in cm) using gender as a covariate. Beta values, 95% CIs and asymptotic p values (t statistic) are reported. Fixed-effects meta-analyses are shown. BMI and waist circumference information was not available for WTCCC controls. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU). Rs8050136 was not typed in RS3. At rs13266634, in the WTCCC cases, the common allele (C; T2D risk allele) homozygotes have a waist circumference that is on average 3.2 cm and a BMI that is 2.0 kg/m<sup>2</sup> less than the rare allele (TT) homozygotes.</p><p>Table S6: Effects of adjusting T2D associations for BMI and waist circumference. In this table, ORs and 95% CIs are reported with respect to the risk allele (denoted in bold, with the ancestral allele underlined where known). Analyses report ORs and CIs before and after adjustment for log<sub>10</sub>BMI or waist circumference and gender, by logistic regression, and fixed-effects meta-analysis. These analyses are only possible for the replication sets, since BMI and waist circumference values were not available in the WTCCC controls. Only the T2D associations at <em>FTO</em> are attenuated by adjustment for BMI and waist circumference.</p><p>Table S7: Associations between T2D susceptibility variants and age of diagnosis in cases. Analyses were performed using linear regression on square root-transformed age of diagnosis values using gender as a covariate. Beta values, 95% CIs and asymptotic p values (t statistic) are reported. Fixed-effects meta-analyses are shown. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU). Rs8050136 were not typed in RS3. T2D- and adiposity-predisposing variants at <em>FTO</em> are associated with earlier age of diagnosis. At rs8050136, rare allele (A; T2D risk allele) homozygotes have an age of diagnosis that is on average 1.7 years earlier than the common allele (CC) homozygotes.</p><p>Table S8: Genotype counts, association p values under different genetic models and test of departure from additivity for robustly replicating signals. The most significant model for each SNP is shown in bold. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU).</p><p>Table S9: Selection criteria used for “second wave” SNPs.</p><p>Table S10: Pairwise interaction analyses of replicating SNPs, and known T2D susceptibility variants in <em>TCF7L2</em>, <em>KCNJ11</em> and <em>PPARG</em>. Odds ratios (for interactions under a log additive model) are reported to the risk allele at each SNP, as defined in <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. In the replication sets, the analyses were adjusted for the three strata. The results are ordered by interaction p value. Rs1801282, rs5215 and rs7901695 were not typed in the replication sets. Rs1111875 was typed as an r<sup>2</sup>=1 proxy (HapMap CEU) for rs5015480 in the replication sets.</p></div><div><a href="/pmc/articles/PMC3772310/bin/NIHMS54288-supplement-01.pdf" data-ga-action="click_feat_suppl">NIHMS54288-supplement-01.pdf</a><span style="color:gray"> (541K)</span></div><div class="small guid">GUID:&nbsp;675DA591-214D-4549-BC98-6944F85896E8</div></div></dd></dl></div><div id="idm140001497137360" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140001497137360title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">The molecular mechanisms involved in the development of type 2 diabetes are poorly understood. Starting from genome-wide genotype data for 1,924 diabetic cases and 2,938 population controls generated by the Wellcome Trust Case Control Consortium, we set out to detect replicated diabetes association signals through analysis of 3,757 additional cases and 5,346 controls, and by integration of our findings with equivalent data from other international consortia. We detected diabetes susceptibility loci in and around the genes <em>CDKAL1</em>, <em>CDKN2A/CDKN2B</em> and <em>IGF2BP2</em> and confirmed the recently described associations at <em>HHEX/IDE</em> and <em>SLC30A8</em>. Our findings provide insights into the genetic architecture of type 2 diabetes, emphasizing the contribution of multiple variants of modest effect. The regions identified underscore the importance of pathways influencing pancreatic beta cell development and function in the etiology of type 2 diabetes.</p></div></div><div id="idm140001494005760" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">The pathophysiological basis of type 2 diabetes (T2D) remains unclear despite its growing global importance (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644018">1</a>). Candidate gene and positional cloning efforts have suggested many putative susceptibility variants, but unequivocal replications are so far limited to variants in just three genes: <em>PPARG</em>, <em>KCNJ11</em> and <em>TCF7L2</em> (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644022">2</a>-<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644012">4</a>).</p><p id="P3">Improved understanding of the correlation between genetic variants (linkage disequilibrium [LD]), allied to advances in genotyping technology, have enabled systematic searches for disease-associated common variants on a genome-wide scale. The Wellcome Trust Case Control Consortium (WTCCC) recently completed such a genome-wide association (GWA) scan in 1,924 T2D cases and 2,938 population controls from the UK, using the Affymetrix GeneChip Human Mapping 500k Array Set (<a href="#R5" rid="R5" class=" bibr popnode">5</a>). The strongest association signals genome-wide were observed for single nucleotide polymorphisms (SNPs) in <em>TCF7L2</em> (e.g. rs7901695 OR 1.37 [95% CI 1.25-1.49], p=6.7×10<sup>−13</sup>). The other known T2D susceptibility variants were detected with effect sizes consistent with previous reports (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644025">2</a>, <a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644016">3</a>).</p><p id="P4">Here, we describe how integration of data from the WTCCC scan and our own replication studies with similar information generated by the Diabetes Genetics Initiative [DGI] (<a href="#R6" rid="R6" class=" bibr popnode">6</a>) and the Finland-United States Investigation of NIDDM Genetics [FUSION] (<a href="#R7" rid="R7" class=" bibr popnode">7</a>) has identified several additional susceptibility variants for T2D.</p><p id="P5">In the WTCCC study, analysis of 490,032 autosomal SNPs in 16,179 samples yielded 459,448 SNPs that passed initial quality control (<a href="#R5" rid="R5" class=" bibr popnode">5</a>). We considered only the 393,453 autosomal SNPs with minor allele frequency (MAF) exceeding 1% in both cases and controls and no extreme departure from Hardy Weinberg equilibrium (HWE) (p&lt;10<sup>−4</sup> in cases or controls) (<a href="#R8" rid="R8" class=" bibr popnode">8</a>). This T2D-specific dataset shows no evidence of substantial confounding from population substructure and genotyping biases (<a href="#R8" rid="R8" class=" bibr popnode">8</a>).</p><p id="P6">To distinguish true associations from those reflecting fluctuations under the null or residual errors arising from aberrant allele calling, we first submitted putative signals from the WTCCC study to additional quality control including cluster plot visualization and validation genotyping on a second platform (<a href="#R8" rid="R8" class=" bibr popnode">8</a>). Next, we attempted replication of selected signals in up to 3,757 additional cases and 5,346 controls (replication sets RS1-RS3). RS1 comprised 2,022 cases and 2,037 controls from the UK Type 2 Diabetes Genetics Consortium collection (UKT2DGC: all from Tayside, Scotland). RS2 included 632 additional T2D cases and 1,750 population controls from the Exeter Family Study of Child Health (EFSOCH). A subset of SNPs were typed in RS3, comprising a further 1,103 cases and 1,559 controls from the UKT2DGC (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>).</p><p id="P7">The first wave of validated SNPs sent for replication was selected from the 30 SNPs, in 9 distinct chromosomal regions (excluding <em>TCF7L2</em>), which had, in the WTCCC scan alone, attained the most extreme (p&lt;10<sup>−5</sup>) significance values on Cochran-Armitage tests of association. Genotyping of 21 representative SNPs generated evidence of replication (p&lt;0.05) for three of these 9 regions (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S2</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Confirmed T2D susceptibility variants</strong><p id="__p11">Representative SNPs are shown for each signal with ORs and 95% CIs reported (for the Cochran-Armitage 1df test) with respect to the risk allele (denoted in bold, with the ancestral allele underlined where known). SNPs selected for inclusion are those with the strongest evidence for association in the UK datasets (except in the case of <em>TCF7L2</em>, where, to maximize consistency across the datasets rs7901695 is presented). In the case of <em>HHEX,</em> the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU). Since DGI and FUSION had not typed the identical SNPs in all samples, results shown for those studies feature the SNP generating the strongest association: in all cases these were SNPs in strong LD (minimum r<sup>2</sup> 0.95, except <em>TCF7L2</em>) and with consistent direction of effect with the SNP reported in the UK data (see <strong>Table S3</strong> for details). The use of different SNPs may result in slightly different estimates of p values and OR between the three studies. Combined estimates of the ORs were calculated by weighting the logORs of each study by the inverse of their variance.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140001486475824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">WTCCC</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">Replication<br>meta-analysis</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">All UK sample<br>meta-analysis</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">DGI</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">FUSION</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">All Combined</th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">1924 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">3757 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">5681 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">6529 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">2376 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">14586 cases</th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">2938 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">5346 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">8284 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">7252 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">2432 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">17968 controls</th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">rs</th><th rowspan="1" colspan="1" valign="middle" align="center">chr</th><th rowspan="1" colspan="1" valign="middle" align="center">position</th><th rowspan="1" colspan="1" valign="middle" align="center">A1</th><th rowspan="1" colspan="1" valign="middle" align="center">A2</th><th rowspan="1" colspan="1" valign="middle" align="center">Region</th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95%Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95%Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95%Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95%Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95%Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th><th rowspan="1" colspan="1" valign="middle" align="center">OR<br>(95% Cis)</th><th rowspan="1" colspan="1" valign="middle" align="center">P<sub>add</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs8050136</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center">52373776</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">A</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>FTO</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.27<br>(1.16-1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.0×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22<br>(1.12-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23<br>(1.18-1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.3×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.03<br>(0.91-1.17)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11<br>(1.02-1.20)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.017</td><td rowspan="1" colspan="1" valign="middle" align="center">1.17<br>(1.12-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10946398</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">20769013</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">C</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CDKAL1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20<br>(1.10-1.31)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.5×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14<br>(1.07-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.4×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.16<br>(1.10-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.08<br>(1.03-1.14)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.4×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.03-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">9.5×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.08,1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.1×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs5015480</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">94455539</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">C</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>HHEX</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22<br>(1.12-1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1111875</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">94452862</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>C</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<span class="underline">T</span>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>HHEX</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.08<br>(1.01-1.15)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.020</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13<br>(1.07-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.6×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14<br>(1.06-1.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.10<br>(1.01-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.025</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13<br>(1.08-1.17)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.7×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs10811661</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center">22124094</td><td rowspan="1" colspan="1" valign="middle" align="center">
<span class="underline">C</span>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CDKN2B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22<br>(1.09-1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18<br>(1.08-1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.19<br>(1.11-1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.9×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20<br>(1.12-1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20<br>(1.07-1.36)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.20<br>(1.14-1.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.8−10<sup>−15</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs564398</td><td rowspan="1" colspan="1" valign="middle" align="center">9</td><td rowspan="1" colspan="1" valign="middle" align="center">22019547</td><td rowspan="1" colspan="1" valign="middle" align="center">
<span class="underline">C</span>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CDKN2B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.16<br>(1.07-1.27)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.05-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.13<br>(1.08-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.05<br>(0.94-1.17)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.5</td><td rowspan="1" colspan="1" valign="middle" align="center">1.13<br>(1.01-1.27)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.039</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.07-1.17)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs4402960</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">186994389</td><td rowspan="1" colspan="1" valign="middle" align="center">
<span class="underline">G</span>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>T</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>IGF2BP2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15<br>(1.05-1.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.09<br>(1.01-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.018</td><td rowspan="1" colspan="1" valign="middle" align="center">1.11<br>(1.05-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.17<br>(1.11-1.23)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18<br>(1.08-1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.4×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14<br>(1.11-1.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.6×10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs13266634</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">118253964</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">C</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SLC30A8</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.02-1.23)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.020</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.04-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.05-1.18)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.0×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.07<br>(1.00-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.047</td><td rowspan="1" colspan="1" valign="middle" align="center">1.18<br>(1.09-1.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.0×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12<br>(1.07-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.3×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs7901695</td><td rowspan="1" colspan="1" valign="middle" align="center">10</td><td rowspan="1" colspan="1" valign="middle" align="center">114744078</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">C</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TCF7L2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.37<br>(1.25-1.49)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.7×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.38<br>(1.31-1.46)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.34<br>(1.21-1.49)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.37<br>(1.31-1.43)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0×10<sup>−48</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs5215</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center">17365206</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>C</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>KCNJ11</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15<br>(1.05-1.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15<br>(1.09-1.21)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0−10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.11<br>(1.02-1.20)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.014</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14<br>(1.10-1.19)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.0×10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1801282</td><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">12368125</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong><span class="underline">C</span></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PPARG</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1.23<br>(1.09-1.41)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">-</td><td rowspan="1" colspan="1" valign="middle" align="center">1.09<br>(1.01-1.16)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.019</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20<br>(1.07-1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.14<br>(1.08-1.20)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140001486475824" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3772310/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P8">Rs8050136 (mapping to the <em>FTO</em> [fat mass associated with obesity] gene region on chr16) was among a cluster of SNPs generating the strongest evidence for association outside <em>TCF7L2</em> in the original scan (risk allele OR 1.27 [1.16-1.37] p=2.0×10<sup>−8</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a>). This SNP showed strong replication (OR 1.22 [1.12-1.32], p=5.4×10<sup>−7</sup>). As we recently reported (<a href="#R9" rid="R9" class=" bibr popnode">9</a>), this effect on T2D risk is mediated through a primary effect on adiposity, and adjustment for BMI abolishes the T2D association. Replication was also obtained for SNPs within the <em>CDKAL1</em> locus on chromosome 6, including rs9465871 and rs10946398. Although rs9465871 generated the stronger signal in the WTCCC scan, replication at this SNP was modest (p=0.023). The replication signal at rs10946398 was more striking (OR 1.14 [1.07-1.22], p=8.4×10<sup>−5</sup>) (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S2</a>). Consistent evidence of association is provided by the DGI (p=4.1×10<sup>−4</sup> at rs7754840) and FUSION groups (p=9.5×10<sup>−3</sup> at rs471253) (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>) (<a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>), both SNPs being strong (r<sup>2</sup>&gt;0.99) proxies for rs10946398. Across all studies, combined evidence for association at <em>CDKAL1</em> is compelling (p~4.1×10<sup>−11</sup>). All associated SNPs map to a large (90kb) intron within <em>CDKAL1</em> (<a href="/pmc/articles/PMC3772310/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Flanking recombination hotspots define a 200kb interval likely to contain the etiological variant(s). <em>CDKAL1</em> (cyclin-dependent kinase 5 [CDK5] regulatory subunit associated protein 1-like 1) encodes a 579-residue, 65kD protein of unknown function. We have detected expression of <em>CDKAL1</em> mRNA in human pancreatic islet and skeletal muscle (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>). CDKAL1 shares considerable protein domain and amino acid homology with CDK5 regulatory subunit associated protein 1 (CDK5RAP1), a known inhibitor of CDK5 activation. CDK5 has been implicated in the regulation of pancreatic beta cell function, through formation of p35/CDK5 complexes that downregulate insulin expression (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644029">11</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644013">12</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3772310/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140001491093504" class="figure"><a href="/pmc/articles/PMC3772310/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3772310_emss-54288-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54288-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3772310/bin/emss-54288-f0001.jpg"></a></div><div id="largeobj_idm140001491093504" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3772310/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3772310/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overview of <em>CDKAL1</em> signal region</strong><p id="__p4">A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p id="__p5">B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p id="__p6">C MULTIZ (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644027">24</a>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p id="__p7">D GoldSurfer2 (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644033">25</a>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p id="__p8">E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644021">26</a>).</p><p id="__p9">F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p></div></div></div><p id="P9">The third replicated association maps to the <em>HHEX</em> (homeobox, hematopoietically expressed) gene region on chromosome 10. This gene showed both strong association in the WTCCC scan (rs5015480: risk allele OR 1.22 [1.12-1.33], p=5.4×10<sup>−6</sup>) and is a powerful biological candidate (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644023">13</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644026">14</a>). We could not optimize a replication assay for rs5015480, but observed evidence for replication at a perfect proxy, rs1111875 (risk allele OR 1.08 [1.01-1.15], p=0.02) (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S2</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>). Both DGI and FUSION studies showed modest, but consistent association signals generating strong combined evidence (p~5.7×10<sup>−10</sup>) for a role in T2D susceptibility (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>). A fourth genome-wide association scan, in French subjects, recently reported independent evidence for a T2D signal in this region (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644028">10</a>). The signal resides within an extended (295kb) region of LD containing not only <em>HHEX</em> (highly expressed in fetal and adult pancreas [<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>]) but also the genes encoding kinesin-interacting factor (<em>KIF11</em>) and insulin degrading enzyme <em>(IDE</em>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>). <em>IDE</em> represents a second strong biological candidate given postulated effects on both insulin signalling and islet function, and data from rodent models (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644034">15</a>-<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644031">17</a>).</p><p id="P10">Of the remaining regions selected in the first wave, none showed any evidence of replication in UK samples (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>), and for none was there strong support from the DGI and FUSION scans.</p><p id="P11">The relatively strict thresholds imposed for SNP selection in the first wave (i.e. point-wise p&lt;10<sup>−5</sup>) help to limit false discovery, but many genuine susceptibility variants will fail to reach them. We initiated a second wave of replication based around SNPs for which the WTCCC scan generated more modest evidence for association (Cochran-Armitage p ~10<sup>−2</sup> to 10<sup>−5</sup>). We prioritized the 5367 SNPs in this range, using additional criteria: (a) evidence of association in DGI and FUSION (<a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>); (b) presence of multiple, independent (r<sup>2</sup>&lt;0.4) associations within the same locus; and (c) biological candidacy (<a href="#R8" rid="R8" class=" bibr popnode">8</a>, <a href="#R18" rid="R18" class=" bibr popnode">18</a>).</p><p id="P12">Analysis of the 56 SNPs, representing 49 putative signals, selected for this “second wave” of replication (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>) yielded two further regions implicated in T2D-susceptibility. A cluster of SNPs on chromosome 9 (represented by rs10811661) generated a promising signal in all three scans. Replication was observed in UK samples (rs10811661:OR 1.18 [1.08-1.28], p=1.7×10<sup>−4</sup>), as well as DGI (p=2.2×10<sup>−5</sup>) and FUSION follow-up studies (rs2383208, p=9.7×10<sup>−3</sup>). A second signal from the WTCCC scan located ~100kb 5′ (rs564398, OR 1.16 [1.07-1.27], p=3.2×10<sup>−4</sup>) was weakly supported in the FUSION, but not the DGI scan (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>) and replicated in the UK RS samples (OR 1.12 [1.05-1.19], p=8.6×10<sup>−4</sup>) (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>).</p><p id="P13">These two association signals are separated by a recombination hotspot (D′ between rs10811661 and rs564398 is 0.057, r<sup>2</sup>&lt;0.001) (<a href="/pmc/articles/PMC3772310/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Across all studies, the combined evidence for association is stronger for the 3′ (p~7.8×10<sup>−15</sup>) than the 5′ (p~1.2×10<sup>−7</sup>) peak (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The 3′ signal maps to sequence with no characterized genes, while the recombination interval enclosing the 5′ signal includes the full coding sequences of <em>CDKN2B</em> and <em>CDKN2A</em> (encoding p15<sup>INK4b</sup> and p16<sup>INK4a</sup> respectively). <em>CDKN2A</em> is a known tumour suppressor and its product, p16<sup>INK4a</sup>, inhibits CDK4 (cyclin-dependent kinase 4), a powerful regulator of pancreatic beta cell replication (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644019">19</a>-<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644014">21</a>). Overexpression of <em>Cdkn2a</em> leads to decreased islet proliferation in ageing mice (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644015">22</a>). <em>Cdkn2b</em> overexpression is also causally related to islet hypoplasia and diabetes in murine models (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644024">23</a>). Both <em>CDKN2B</em> and <em>CDKN2A</em> display high levels of expression in pancreatic islets and pituitary (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3772310/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140001485528160" class="figure"><a href="/pmc/articles/PMC3772310/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3772310_emss-54288-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54288-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3772310/bin/emss-54288-f0002.jpg"></a></div><div id="largeobj_idm140001485528160" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3772310/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3772310/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Overview of chr9 signal region</strong><p id="__p10">Panel layout as per <a href="/pmc/articles/PMC3772310/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a></p></div></div></div><p id="P14">A fifth replicated association lies within the <em>IGF2BP2</em> gene on chromosome 3. We observed some evidence of association for SNPs in this region in the WTCCC scan (<a href="#R5" rid="R5" class=" bibr popnode">5</a>) (e.g. rs4402960: OR 1.15 [1.05-1.25], p=1.7×10<sup>−3</sup>). Consistent associations in the DGI and FUSION scans (<a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>) and the biological candidacy of the gene (a known regulator of insulin-like growth factor 2 [IGF2] translation), prompted replication. We obtained only modest evidence for replication at rs4402960 (OR 1.09 [1.01-1.16], p=0.018) (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S4</a>), but combined evidence across all studies (p~8.6×10<sup>−16</sup>) establishes this as a genuine T2D signal (<a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S3</a>). The associated SNPs map to a 57kb region spanning the promoter and first 2 exons of <em>IGF2BP2</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4</a>).</p><p id="P15">Most of the remaining 50 “second wave” SNPs can be discounted as susceptibility variants based on their failure to replicate (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S4</a>), though some merit further consideration. One such example is rs9369425, located 57kb downstream of the <em>VEGFA</em> (vascular endothelial growth factor A) gene on chromosome 6 (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5</a>). Evidence for association in the WTCCC scan (OR 1.16 [1.06-1.27], p=8.6×10<sup>−4</sup>) is supported by nominal replication in UK samples (1.08 [1.01-1.15], p=0.03) and by DGI scan results (1.17 [1.04-1.32], p=4.4×10<sup>−3</sup>). While no signal is apparent in the FUSION study, this does not allow us to reject this association. For 80% power to detect an OR of 1.11 (α=0.05), over 3,000 case-control pairs are needed.</p><p id="P16">In the French genome-wide scan (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644020">10</a>), variants in both the <em>HHEX</em> and <em>SLC30A8</em> genes were implicated in T2D susceptibility. As the associated SNPs in <em>SLC30A8</em> are poorly captured on the Affymetrix chip (r<sup>2</sup>&lt;0.01), the WTCCC scan was not informative for this locus. However, we genotyped rs13266634 independently and obtained replication of the finding (risk allele OR 1.12 [1.05-1.18], p=7.0×10<sup>−5</sup> in all UK data) and across all three studies (p~5.3×10<sup>−8</sup>, <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S4</a>).</p><p id="P17">The present analysis has contributed to identification of several confirmed T2D susceptibility loci. One of these (<em>FTO</em>) exerts its primary effect on T2D risk through an impact on adiposity (<a href="#R9" rid="R9" class=" bibr popnode">9</a>): none of the other signals was attenuated by adjustment for BMI or waist circumference (<a href="#SD1" rid="SD1" class=" supplementary-material">Tables S5</a>-<a href="#SD1" rid="SD1" class=" supplementary-material">S7</a>). One of the remaining four loci (<em>HHEX/IDE</em>) represents a strong replication of findings recently reported (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644032">10</a>). The other three loci (near <em>CDKAL1, IGF2BP2</em> and <em>CDKN2A</em>), all showing extensive replication across the three studies represent novel T2D susceptibility loci.</p><p id="P18">Across the four T2D scans completed (<a href="#R5" rid="R5" class=" bibr popnode">5</a>, <a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644030">10</a>), <em>TCF7L2</em> clearly emerges as the largest association signal. On current evidence, all other confirmed loci display more modest effect sizes (between 1.10 and 1.25 per allele). Extensive resequencing and fine-mapping will be required to define the full spectrum of etiological variation at each locus and these may yet identify variants with greater impact. Our findings offer clear lessons for the design of future studies. Robust identification of variants with such effect sizes is only feasible with large-scale sample sets (13,965 individuals were typed in the present study). Further, the exchange of data between groups (providing data on up to 32,554 samples) was key to the rapid and unequivocal identification of the signals we report.</p><p id="P19" class="p">As a result of the four GWA studies reported to date (<a href="#R5" rid="R5" class=" bibr popnode">5</a>, <a href="#R6" rid="R6" class=" bibr popnode">6</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_332644017">10</a>), the number of genuine, replicated T2D susceptibility signals has climbed from 3 to 9 (adding <em>HHEX/IDE, SLC30A8, CDKAL1, CDKN2A, IGF2BP2</em> and <em>FTO</em>). However, these loci explain only a small proportion of the observed familiality (the sibling relative risk, λ<sub>s</sub>, attributable to all loci in the UK samples is only ~1.07). We expect additional loci to be revealed by further rounds of replication initiated by more systematic meta-analysis of these and other scans. Our study provides an important validation of the genome-wide indirect association mapping approach and a demonstration of the value of aggressive data sharing efforts. It also generates insights into T2D pathogenesis emphasizing the likely importance of pathways involved in pancreatic beta cell development, regeneration and function. In-depth physiological and functional studies are now needed to establish the precise mechanisms involved.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><p id="P20">Figure S1: Overview of <em>FTO</em> signal region</p><p id="P21">A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p id="P22">B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p id="P23">C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p id="P24">D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p id="P25">E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p id="P26">F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p id="P27">Figure S2: Expression patterns of <em>CDKAL1, CDKN2A, CDKN2B, HHEX, IGF2BP2.</em> Messenger RNA expression profiles are shown for the genes listed for a range of human tissues, as determined by RT-PCR. Figures on the y axes refer to the test transcript levels relative to two separate endogenous control genes (beta glucuronidase [<em>BGUS</em>] and beta 2 microglobulin [<em>B2M</em>]). Each test:control ratio was then normalized to that of adult human pancreas for tissue to tissue comparison.</p><p id="P28">Figure S3: Overview of <em>HHEX</em> signal region</p><p id="P29">A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples. Note that rs5015480 was typed in the WTCCC scan and rs1111875 in the replication set, so the meta-analysis result is based on a combined analysis of the two (r<sup>2</sup>=1 in HapMap CEU) and the position of this signal denoted at both locations.</p><p id="P30">B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p id="P31">C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p id="P32">D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p id="P33">E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p id="P34">F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p id="P35">Figure S4: Overview of <em>IGF2BP2</em> signal region</p><p id="P36">A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p id="P37">B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p id="P38">C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p id="P39">D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p id="P40">E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p id="P41">F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p id="P42">Figure S5: Overview of <em>VEGFA</em> signal region</p><p id="P43">A Plot of −log(p values) for T2D (Cochran-Armitage test for trend) against chromosome position in Mb. Blue diamonds represent primary scan results and pink triangles denote meta-analysis results across all UK samples.</p><p id="P44">B Genomic location of genes showing intron and exon structure (NCBI Build 35). Pink triangles show position of replication SNPs relative to gene structure.</p><p id="P45">C MULTIZ (<em>S24</em>) vertebrate alignment of 17 species showing evolutionary conservation.</p><p id="P46">D GoldSurfer2 (<em>S19</em>) plot of linkage disequilibrium (r<sup>2</sup>) for SNPs genotyped in WTCCC scan (passing T2D-specific quality control) in WTCCC T2D cases.</p><p id="P47">E Recombination rate given as cM/MB. Red boxes represent recombination hotspots (<em>S25</em>).</p><p id="P48">F GoldSurfer2 plot of linkage disequilibrium (r<sup>2</sup>) for all HapMap SNPs across the region (HapMap CEU data).</p><p id="P49">Figure S6: Quantile-Quantile plots for the 393,453 SNPs passing T2D-specific quality control. The plot compares the observed and expected distributions for the 1df chi-square statistics generated from single-point Cochran-Armitage tests.</p><p id="P50">Figure S7: Correlation plot of association statistics for the WTCCC scan genotypes before and after adjusting for population structure. The plot compares the 1df χ<sup>2</sup> (single-point Cochran-Armitage) values obtained from a naïve trend test for the 393.453 SNPs passing T2D-specific quality control, with the equivalent statistics generated after correcting for population substructure using Eigenstrat (<em>S8</em>). The high correlation overall (r<sup>2</sup>&gt;0.99), and in particular the strong linearity for high χ<sup>2</sup> values (top right) indicates minimal confounding from population substructure after implementation of the various QC measures described.</p><p id="P51">Figure S8: Single-point and haplotype-based analysis results for the chr9 signal region using GENEBPM. Circles denote single-point analysis results and the continuous line represents 5 SNP sliding window haplotype-based analyses using GENEBPM (<em>S15</em>). The two peaks of association are separated by a recombination hotspot (<a href="/pmc/articles/PMC3772310/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Multipoint analyses reveal much stronger evidence for association at the 3′ peak in this region. Common (&gt;0.01) high risk haplotypes in the 3′ signal share alleles T and T at SNPs rs10811661 and rs10757283 respectively. In the WTCCC scan (<em>S1</em>), Cochran-Armitage p values for rs2891169 were similar to those of SNPs rs10811661 and rs10757283, which were selected for replication. In contrast, GENEBPM analysis indicated stronger evidence for single-point association for rs2891169.</p><p id="P52"><strong>Table S1: Clinical characteristics of UK samples.</strong> WTCCC controls came from two sources. No data on age, waist circumference or BMI are available for the UK Blood Service controls. Control individuals from the 1958 Birth Cohort were last reviewed at age 41. Under the terms of access, waist circumference and BMI values from these controls are not available to WTCCC researchers. Only 46% (all male) of the RS2 control individuals had available waist circumference measures.</p><p id="P53">Table S2: Replication results for SNPs selected for Cochran-Armitage p&lt;10<sup>−5</sup> on WTCCC scan. Alleles in this table are named alphabetically (as per the forward strand) with the ancestral allele underlined (where known). A2F denotes allele 2 frequency. For consistency, all ORs in this table are reported for allele 2 (and may therefore be the reciprocal of the ORs reported in the text and <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). It was not possible to design a working replication assay for rs5015480, and the UK meta-analysis for this signal combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU).</p><p id="P54">Table S3: Confirmed T2D susceptibility signals: SNPs reported for the DGI and FUSION studies. As DGI and FUSION did not always type the same SNPs as the UK study in all their samples, results in <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> include data from the SNPs generating the strongest association in their respective studies. Table S3 gives details of the SNPs reported for DGI and FUSION, and their LD relationships (based on HapMap CEU and/or genome-wide or imputation data as available) with the UK index SNP. In all cases these proxies were SNPs in strong LD (r<sup>2</sup>&gt;0.95, except <em>TCF7L2</em>) and showed consistent direction of effect with the SNP reported in the UK data.</p><p id="P55">Table S4: Replication results for SNPs selected for the “second wave” of replication. Alleles in this table are named alphabetically (as per the forward strand) with the ancestral allele underlined (where known). A2F denotes allele 2 frequency. For consistency, all ORs in this table are reported for allele 2 (and may therefore be the reciprocal of the ORs reported in the text). These signals were (with the exception of rs13266634 in <em>SLC30A8</em>) selected on the basis of Cochran-Armitage test p values between 10<sup>−2</sup> and 10<sup>−5</sup> on the WTCCC scan, prioritized on the basis of biological candidacy, multiple independent associations and/or support from DGI and/or FUSION scans. It was not possible to design a working replication assay for rs11140802, and the UK meta-analysis for this signal combines data from rs11140802 and rs12346884 (r<sup>2</sup>=1 in HapMap CEU). Rs13266634 in <em>SLC30A8</em> was not captured on the Affymetrix chip but was selected for replication on the basis of the associations in French (<em>S18</em>) and FUSION (<em>S17</em>) subjects.</p><p id="P56">Table S5: Associations between T2D susceptibility variants and (a) BMI, (b) waist circumference in cases and controls. Analyses were performed using linear regression on (a) log<sub>10</sub>-transformed BMI values and (b) on waist circumference values (in cm) using gender as a covariate. Beta values, 95% CIs and asymptotic p values (t statistic) are reported. Fixed-effects meta-analyses are shown. BMI and waist circumference information was not available for WTCCC controls. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU). Rs8050136 was not typed in RS3. At rs13266634, in the WTCCC cases, the common allele (C; T2D risk allele) homozygotes have a waist circumference that is on average 3.2 cm and a BMI that is 2.0 kg/m<sup>2</sup> less than the rare allele (TT) homozygotes.</p><p id="P57">Table S6: Effects of adjusting T2D associations for BMI and waist circumference. In this table, ORs and 95% CIs are reported with respect to the risk allele (denoted in bold, with the ancestral allele underlined where known). Analyses report ORs and CIs before and after adjustment for log<sub>10</sub>BMI or waist circumference and gender, by logistic regression, and fixed-effects meta-analysis. These analyses are only possible for the replication sets, since BMI and waist circumference values were not available in the WTCCC controls. Only the T2D associations at <em>FTO</em> are attenuated by adjustment for BMI and waist circumference.</p><p id="P58">Table S7: Associations between T2D susceptibility variants and age of diagnosis in cases. Analyses were performed using linear regression on square root-transformed age of diagnosis values using gender as a covariate. Beta values, 95% CIs and asymptotic p values (t statistic) are reported. Fixed-effects meta-analyses are shown. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU). Rs8050136 were not typed in RS3. T2D- and adiposity-predisposing variants at <em>FTO</em> are associated with earlier age of diagnosis. At rs8050136, rare allele (A; T2D risk allele) homozygotes have an age of diagnosis that is on average 1.7 years earlier than the common allele (CC) homozygotes.</p><p id="P59">Table S8: Genotype counts, association p values under different genetic models and test of departure from additivity for robustly replicating signals. The most significant model for each SNP is shown in bold. In the case of <em>HHEX</em>, the UK meta-analysis combines data from rs5015480 and rs1111875 (r<sup>2</sup>=1 in HapMap CEU).</p><p id="P60">Table S9: Selection criteria used for “second wave” SNPs.</p><p id="P61">Table S10: Pairwise interaction analyses of replicating SNPs, and known T2D susceptibility variants in <em>TCF7L2</em>, <em>KCNJ11</em> and <em>PPARG</em>. Odds ratios (for interactions under a log additive model) are reported to the risk allele at each SNP, as defined in <a href="/pmc/articles/PMC3772310/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. In the replication sets, the analyses were adjusted for the three strata. The results are ordered by interaction p value. Rs1801282, rs5215 and rs7901695 were not typed in the replication sets. Rs1111875 was typed as an r<sup>2</sup>=1 proxy (HapMap CEU) for rs5015480 in the replication sets.</p><div class="sup-box half_rhythm" id="d36e1192"><a href="/pmc/articles/PMC3772310/bin/NIHMS54288-supplement-01.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(541K, pdf)</sup></div></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Acknowledgments</h2><div class="sec"><p id="__p1">We are grateful to all the study participants. We acknowledge the support of Diabetes UK, BDA Research, the UK Medical Research Council, the Wellcome Trust, European Commission (EURODIA LSHG-CT-2004-518153) and Peninsula Medical School. Personal funding comes from the Wellcome Trust (EZ, ATH, LRC); UK Medical Research Council (JRBP); Diabetes UK (RMF); Throne-Holst Foundation (CML); Leverhulme Trust (APM); Research Councils UK (LWH); and the Scottish Executive Generations Scotland Initiative (CNAP, ADM). MNW is Vandervell Foundation Research Fellow at the Peninsula Medical School. We are grateful to members of the DGI and FUSION teams for sharing data.</p></div></div><div id="idm140001492779904" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140001492779904title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P62" class="p p-first">Supporting Online Material</p><p id="P63" class="p"><a href="http://www.sciencemag.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.sciencemag.org</a></p><p id="P64" class="p">Materials and Methods</p><p id="P65" class="p">Replication set-only combined effect size estimation</p><p id="P66" class="p">Haplotype-based analysis results</p><p id="P67" class="p">Overlap between association and linkage signals</p><p id="P68" class="p">Quantitative trait analysis results</p><p id="P69" class="p">Departures from additivity</p><p id="P70" class="p">Data access details</p><p id="P71" class="p">Membership of WTCCC</p><p id="P72" class="p">Detailed acknowledgements</p><p id="P73" class="p">References</p><p id="P74" class="p">Figs. S1, S2, S3, S4, S5, S6, S7, S8</p><p id="P75" class="p p-last">Tables S1, S2, S3, S4, S5, S6, S7, S8, S9, S10</p><p></p></div></div><div id="idm140001485870320" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140001485870320title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Stumvoll M, Goldstein BJ, van Haeften TW. <span><span class="ref-journal">Lancet. </span>2005;<span class="ref-vol">365</span>:1333–1346.</span> [<a href="/pubmed/15823385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;author=M+Stumvoll&amp;author=BJ+Goldstein&amp;author=TW+van+Haeften&amp;volume=365&amp;publication_year=2005&amp;pages=1333-1346&amp;pmid=15823385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Altshuler D, et al.  <span><span class="ref-journal">Nat. Genet. </span>2000;<span class="ref-vol">26</span>:76–80.</span> [<a href="/pubmed/10973253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;author=D+Altshuler&amp;volume=26&amp;publication_year=2000&amp;pages=76-80&amp;pmid=10973253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Gloyn AL, et al.  <span><span class="ref-journal">Diabetes. </span>2003;<span class="ref-vol">52</span>:568–572.</span> [<a href="/pubmed/12540637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;author=AL+Gloyn&amp;volume=52&amp;publication_year=2003&amp;pages=568-572&amp;pmid=12540637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Grant SF, et al.  <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:320–323.</span> [<a href="/pubmed/16415884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;author=SF+Grant&amp;volume=38&amp;publication_year=2006&amp;pages=320-323&amp;pmid=16415884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Donnelly P, the WTCCC, personal communication  <span class="ref-journal">Data from the Wellcome Trust Case Control Consortium scan.</span> </span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Diabetes Genetics Initiative  <span><span class="ref-journal">Science. </span>(this issue)</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;issue=this+issue&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Scott LJ, et al.  <span><span class="ref-journal">Science. </span>(this issue)</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=LJ+Scott&amp;issue=this+issue&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span>Materials and methods are available as <a href="#SD1" rid="SD1" class=" supplementary-material">supporting material</a> on Science Online</span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Frayling TM, et al.  <span class="ref-journal">Science.</span>  Published online April 12 2007.</span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Sladek R, et al.  <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">445</span>:881–885.</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;author=R+Sladek&amp;volume=445&amp;publication_year=2007&amp;pages=881-885&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Ubeda M, Rukstalis JM, Habener JF. <span><span class="ref-journal">J. Biol. Chem. </span>2006;<span class="ref-vol">281</span>:28858–28864.</span> [<a href="/pubmed/16887799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;author=M+Ubeda&amp;author=JM+Rukstalis&amp;author=JF+Habener&amp;volume=281&amp;publication_year=2006&amp;pages=28858-28864&amp;pmid=16887799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Wei FY, et al.  <span><span class="ref-journal">Nat. Med. </span>2005;<span class="ref-vol">11</span>:1104–1108.</span> [<a href="/pubmed/16155576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Med&amp;author=FY+Wei&amp;volume=11&amp;publication_year=2005&amp;pages=1104-1108&amp;pmid=16155576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Foley AC, Mercola M. <span><span class="ref-journal">Genes Dev. </span>2005;<span class="ref-vol">19</span>:387–396.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546516/">PMC free article</a>]</span> [<a href="/pubmed/15687261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;author=AC+Foley&amp;author=M+Mercola&amp;volume=19&amp;publication_year=2005&amp;pages=387-396&amp;pmid=15687261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Bort R, Martinez-Barbera JP, Beddington RS, Zaret KS. <span><span class="ref-journal">Development. </span>2004;<span class="ref-vol">131</span>:797–806.</span> [<a href="/pubmed/14736744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;author=R+Bort&amp;author=JP+Martinez-Barbera&amp;author=RS+Beddington&amp;author=KS+Zaret&amp;volume=131&amp;publication_year=2004&amp;pages=797-806&amp;pmid=14736744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Fakhrai-Rad H, et al.  <span><span class="ref-journal">Hum. Mol. Genet. </span>2000;<span class="ref-vol">9</span>:2149–2158.</span> [<a href="/pubmed/10958757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;author=H+Fakhrai-Rad&amp;volume=9&amp;publication_year=2000&amp;pages=2149-2158&amp;pmid=10958757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Seta KA, Roth RA. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>1997;<span class="ref-vol">231</span>:167–71.</span> [<a href="/pubmed/9070242" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun&amp;author=KA+Seta&amp;author=RA+Roth&amp;volume=231&amp;publication_year=1997&amp;pages=167-71&amp;pmid=9070242&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Farris W, et al.  <span><span class="ref-journal">Am. J. Pathol. </span>2004;<span class="ref-vol">164</span>:1425–1434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1615329/">PMC free article</a>]</span> [<a href="/pubmed/15039230" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Pathol&amp;author=W+Farris&amp;volume=164&amp;publication_year=2004&amp;pages=1425-1434&amp;pmid=15039230&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span>GeneSniffer ( <a href="http://www.genesniffer.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.genesniffer.org</a>) was used to prioritize genes for biological candidacy by integrating information from diverse online databases (including Entrez Gene, OMIM, PubMed and MGI).</span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Rane SG, et al.  <span><span class="ref-journal">Nat. Genet. </span>1999;<span class="ref-vol">22</span>:44–52.</span> [<a href="/pubmed/10319860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;author=SG+Rane&amp;volume=22&amp;publication_year=1999&amp;pages=44-52&amp;pmid=10319860&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Mettus RV, Rane SG. <span><span class="ref-journal">Oncogene. </span>2003;<span class="ref-vol">22</span>:8413–8421.</span> [<a href="/pubmed/14627982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;author=RV+Mettus&amp;author=SG+Rane&amp;volume=22&amp;publication_year=2003&amp;pages=8413-8421&amp;pmid=14627982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Marzo N, et al.  <span><span class="ref-journal">Diabetologia. </span>2004;<span class="ref-vol">47</span>:686–694.</span> [<a href="/pubmed/15298346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;author=N+Marzo&amp;volume=47&amp;publication_year=2004&amp;pages=686-694&amp;pmid=15298346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Krishnamurthy J, et al.  <span><span class="ref-journal">Nature. </span>2006;<span class="ref-vol">443</span>:453–457.</span> [<a href="/pubmed/16957737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;author=J+Krishnamurthy&amp;volume=443&amp;publication_year=2006&amp;pages=453-457&amp;pmid=16957737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Moritani M, et al.  <span><span class="ref-journal">Mol. Cell Endocrinol. </span>2005;<span class="ref-vol">229</span>:175–184.</span> [<a href="/pubmed/15607541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell+Endocrinol&amp;author=M+Moritani&amp;volume=229&amp;publication_year=2005&amp;pages=175-184&amp;pmid=15607541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Blanchette M, et al.  <span><span class="ref-journal">Genome Res. </span>2004;<span class="ref-vol">14</span>:708–715.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC383317/">PMC free article</a>]</span> [<a href="/pubmed/15060014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;author=M+Blanchette&amp;volume=14&amp;publication_year=2004&amp;pages=708-715&amp;pmid=15060014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Pettersson F, Jonsson O, Cardon LR. <span><span class="ref-journal">Bioinformatics. </span>2004;<span class="ref-vol">20</span>:3241–3243.</span> [<a href="/pubmed/15201180" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;author=F+Pettersson&amp;author=O+Jonsson&amp;author=LR+Cardon&amp;volume=20&amp;publication_year=2004&amp;pages=3241-3243&amp;pmid=15201180&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Myers S, Bottolo L, Freeman C, McVean G, Donnelly P. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">310</span>:321.</span> [<a href="/pubmed/16224025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;author=S+Myers&amp;author=L+Bottolo&amp;author=C+Freeman&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=310&amp;publication_year=2005&amp;pages=321&amp;pmid=16224025&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3772310&amp;issue-id=227449&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140001497137360title">Abstract</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S1title">Acknowledgments</a></li><li><a href="#idm140001492779904title">Footnotes</a></li><li><a href="#idm140001485870320title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3772310/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3772310/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3772310/pdf/emss-54288.pdf">PDF (962K)</a></li> | <li><a href="#" data-citationid="PMC3772310" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3772310%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3772310%2F&amp;text=Multiple%20type%202%20diabetes%20susceptibility%20genes%20following%20genome-wide%20association%20scan%20in%20UK%20samples" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3772310%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="17463249" data-aiid="3772310" data-aid="3772310" data-iid="227449" data-domainid="358" data-domain="wtpa" data-accid="PMC3772310" data-md5="469f09c183955e64a418bcee56aea832"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T08:30:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A771EDD3E93610000000003E20254&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3772310%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3772310/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>