<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="pnas"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3545809/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3545809/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association"><meta name="citation_authors" content="Karl C. Desch, Ayse B. Ozel, David Siemieniak, Yossi Kalish, Jordan A. Shavit, Courtney D. Thornburg, Anjali A. Sharathkumar, Caitlin P. McHugh, Cathy C. Laurie, Andrew Crenshaw, Daniel B. Mirel, Yoonhee Kim, Cheryl D. Cropp, Anne M. Molloy, Peadar N. Kirke, Joan E. Bailey-Wilson, Alexander F. Wilson, James L. Mills, John M. Scott, Lawrence C. Brody, Jun Z. Li, David Ginsburg"><meta name="citation_date" content="8 January 2013"><meta name="citation_issue" content="2"><meta name="citation_volume" content="110"><meta name="citation_firstpage" content="588"><meta name="citation_doi" content="10.1073/pnas.1219885110"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3545809/?report=abstract"><meta name="citation_pmid" content="23267103"><meta name="DC.Title" content="Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Karl C. Desch"><meta name="DC.Contributor" content="Ayse B. Ozel"><meta name="DC.Contributor" content="David Siemieniak"><meta name="DC.Contributor" content="Yossi Kalish"><meta name="DC.Contributor" content="Jordan A. Shavit"><meta name="DC.Contributor" content="Courtney D. Thornburg"><meta name="DC.Contributor" content="Anjali A. Sharathkumar"><meta name="DC.Contributor" content="Caitlin P. McHugh"><meta name="DC.Contributor" content="Cathy C. Laurie"><meta name="DC.Contributor" content="Andrew Crenshaw"><meta name="DC.Contributor" content="Daniel B. Mirel"><meta name="DC.Contributor" content="Yoonhee Kim"><meta name="DC.Contributor" content="Cheryl D. Cropp"><meta name="DC.Contributor" content="Anne M. Molloy"><meta name="DC.Contributor" content="Peadar N. Kirke"><meta name="DC.Contributor" content="Joan E. Bailey-Wilson"><meta name="DC.Contributor" content="Alexander F. Wilson"><meta name="DC.Contributor" content="James L. Mills"><meta name="DC.Contributor" content="John M. Scott"><meta name="DC.Contributor" content="Lawrence C. Brody"><meta name="DC.Contributor" content="Jun Z. Li"><meta name="DC.Contributor" content="David Ginsburg"><meta name="DC.Date" content="2013 Jan 8"><meta name="DC.Identifier" content="10.1073/pnas.1219885110"><meta name="DC.Language" content="en"><meta property="og:title" content="Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association"><meta property="og:type" content="article"><meta property="og:description" content="The plasma glycoprotein von Willebrand factor (VWF) exhibits fivefold antigen level variation across the normal human population determined by both genetic and environmental factors. Low levels of VWF are associated with bleeding and elevated levels with ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3545809/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3545809/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3545809/pdf/pnas.201219885.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5B57DD5513310000000003B6027B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3545809%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="gene" class="last">Gene</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/218256/">v.110(2); 2013 Jan 8</a></li><li class="accid">PMC3545809</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="306,2,479,64" alt="PNAS" title="PNAS" href="http://www.pnas.org" ref="reftype=publisher&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="59,34,142,50" alt="Info for Authors" title="Info for Authors" href="http://www.pnas.org/site/misc/iforc.shtml" ref="reftype=publisher&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="143,34,200,50" alt="Subscriptions" title="Subscriptions" href="http://www.pnas.org/site/subscriptions/" ref="reftype=publisher&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="201,34,239,50" alt="About" title="About" href="http://www.pnas.org/site/misc/about.shtml" ref="reftype=publisher&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,34,57,50" alt="This Article" title="This Article" href="//dx.doi.org/10.1073/pnas.1219885110" ref="reftype=publisher&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Content%20Provider%7CArticle%7CFullText" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Proc Natl Acad Sci U S A</a></span>. 2013 Jan 8; 110(2): 588–593. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Dec 24. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1073%2Fpnas.1219885110" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.1219885110</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3545809</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23267103">23267103</a></div></div></div><div class="fm-doctopic no_margin">Genetics</div></div><h1 class="content-title">Linkage analysis identifies a locus for plasma von Willebrand factor undetected by genome-wide association</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Desch%20KC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703054054016" co-class="co-affbox">Karl C. Desch</a>,<sup>a</sup> <a href="/pubmed/?term=Ozel%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703036854240" co-class="co-affbox">Ayse B. Ozel</a>,<sup>b</sup> <a href="/pubmed/?term=Siemieniak%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703052312832" co-class="co-affbox">David Siemieniak</a>,<sup>c</sup> <a href="/pubmed/?term=Kalish%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703045352400" co-class="co-affbox">Yossi Kalish</a>,<sup>d</sup> <a href="/pubmed/?term=Shavit%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703045350288" co-class="co-affbox">Jordan A. Shavit</a>,<sup>a</sup> <a href="/pubmed/?term=Thornburg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703038546048" co-class="co-affbox">Courtney D. Thornburg</a>,<sup>e</sup> <a href="/pubmed/?term=Sharathkumar%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703087489936" co-class="co-affbox">Anjali A. Sharathkumar</a>,<sup>f</sup> <a href="/pubmed/?term=McHugh%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703087487824" co-class="co-affbox">Caitlin P. McHugh</a>,<sup>g</sup> <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703052418448" co-class="co-affbox">Cathy C. Laurie</a>,<sup>g</sup> <a href="/pubmed/?term=Crenshaw%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703038611632" co-class="co-affbox">Andrew Crenshaw</a>,<sup>h</sup> <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703038609520" co-class="co-affbox">Daniel B. Mirel</a>,<sup>h</sup> <a href="/pubmed/?term=Kim%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703045333360" co-class="co-affbox">Yoonhee Kim</a>,<sup>i</sup> <a href="/pubmed/?term=Cropp%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703052978304" co-class="co-affbox">Cheryl D. Cropp</a>,<sup>i</sup> <a href="/pubmed/?term=Molloy%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703052976192" co-class="co-affbox">Anne M. Molloy</a>,<sup>j</sup> <a href="/pubmed/?term=Kirke%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703052321360" co-class="co-affbox">Peadar N. Kirke</a>,<sup>k</sup> <a href="/pubmed/?term=Bailey-Wilson%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703051117728" co-class="co-affbox">Joan E. Bailey-Wilson</a>,<sup>i</sup> <a href="/pubmed/?term=Wilson%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703051115728" co-class="co-affbox">Alexander F. Wilson</a>,<sup>i</sup> <a href="/pubmed/?term=Mills%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703053156768" co-class="co-affbox">James L. Mills</a>,<sup>l</sup> <a href="/pubmed/?term=Scott%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703053154656" co-class="co-affbox">John M. Scott</a>,<sup>m</sup> <a href="/pubmed/?term=Brody%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703040867616" co-class="co-affbox">Lawrence C. Brody</a>,<sup>n</sup> <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703046621280" co-class="co-affbox">Jun Z. Li</a>,<sup>b,</sup><sup>o,</sup><sup>1</sup> and  <a href="/pubmed/?term=Ginsburg%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103" class="affpopup" co-rid="_co_idm140703053122832" co-class="co-affbox">David Ginsburg</a><sup>a,</sup><sup>b,</sup><sup>c,</sup><sup>p,</sup><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140703054054016"><h3 class="no_margin">Karl C. Desch</h3><p><sup>a</sup>Department of Pediatrics and Communicable Diseases,</p><div>Find articles by <a href="/pubmed/?term=Desch%20KC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Karl C. Desch</a></div></div><div id="_co_idm140703036854240"><h3 class="no_margin">Ayse B. Ozel</h3><p><sup>b</sup>Department of Human Genetics,</p><div>Find articles by <a href="/pubmed/?term=Ozel%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Ayse B. Ozel</a></div></div><div id="_co_idm140703052312832"><h3 class="no_margin">David Siemieniak</h3><p><sup>c</sup>Howard Hughes Medical Institute, Ann Arbor, MI, 48109;</p><div>Find articles by <a href="/pubmed/?term=Siemieniak%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">David Siemieniak</a></div></div><div id="_co_idm140703045352400"><h3 class="no_margin">Yossi Kalish</h3><p><sup>d</sup>Hadassah-Hebrew University Medical Center, Jerusalem 91120, Israel;</p><div>Find articles by <a href="/pubmed/?term=Kalish%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Yossi Kalish</a></div></div><div id="_co_idm140703045350288"><h3 class="no_margin">Jordan A. Shavit</h3><p><sup>a</sup>Department of Pediatrics and Communicable Diseases,</p><div>Find articles by <a href="/pubmed/?term=Shavit%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Jordan A. Shavit</a></div></div><div id="_co_idm140703038546048"><h3 class="no_margin">Courtney D. Thornburg</h3><p><sup>e</sup>Department of Pediatrics, Duke University, Durham, NC,  27710;</p><div>Find articles by <a href="/pubmed/?term=Thornburg%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Courtney D. Thornburg</a></div></div><div id="_co_idm140703087489936"><h3 class="no_margin">Anjali A. Sharathkumar</h3><p><sup>f</sup>Department of Pediatrics, Northwestern University, Chicago, IL,  60611;</p><div>Find articles by <a href="/pubmed/?term=Sharathkumar%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Anjali A. Sharathkumar</a></div></div><div id="_co_idm140703087487824"><h3 class="no_margin">Caitlin P. McHugh</h3><p><sup>g</sup>Department of Biostatistics, University of Washington, Seattle, WA,  98105;</p><div>Find articles by <a href="/pubmed/?term=McHugh%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Caitlin P. McHugh</a></div></div><div id="_co_idm140703052418448"><h3 class="no_margin">Cathy C. Laurie</h3><p><sup>g</sup>Department of Biostatistics, University of Washington, Seattle, WA,  98105;</p><div>Find articles by <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Cathy C. Laurie</a></div></div><div id="_co_idm140703038611632"><h3 class="no_margin">Andrew Crenshaw</h3><p><sup>h</sup>Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</p><div>Find articles by <a href="/pubmed/?term=Crenshaw%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Andrew Crenshaw</a></div></div><div id="_co_idm140703038609520"><h3 class="no_margin">Daniel B. Mirel</h3><p><sup>h</sup>Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</p><div>Find articles by <a href="/pubmed/?term=Mirel%20DB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Daniel B. Mirel</a></div></div><div id="_co_idm140703045333360"><h3 class="no_margin">Yoonhee Kim</h3><p><sup>i</sup>Inherited Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Baltimore, MD,  21224;</p><div>Find articles by <a href="/pubmed/?term=Kim%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Yoonhee Kim</a></div></div><div id="_co_idm140703052978304"><h3 class="no_margin">Cheryl D. Cropp</h3><p><sup>i</sup>Inherited Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Baltimore, MD,  21224;</p><div>Find articles by <a href="/pubmed/?term=Cropp%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Cheryl D. Cropp</a></div></div><div id="_co_idm140703052976192"><h3 class="no_margin">Anne M. Molloy</h3><p><sup>j</sup>School of Medicine and</p><div>Find articles by <a href="/pubmed/?term=Molloy%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Anne M. Molloy</a></div></div><div id="_co_idm140703052321360"><h3 class="no_margin">Peadar N. Kirke</h3><p><sup>k</sup>Child Health Epidemiology Unit, Health Research Board, Dublin, Ireland;</p><div>Find articles by <a href="/pubmed/?term=Kirke%20PN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Peadar N. Kirke</a></div></div><div id="_co_idm140703051117728"><h3 class="no_margin">Joan E. Bailey-Wilson</h3><p><sup>i</sup>Inherited Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Baltimore, MD,  21224;</p><div>Find articles by <a href="/pubmed/?term=Bailey-Wilson%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Joan E. Bailey-Wilson</a></div></div><div id="_co_idm140703051115728"><h3 class="no_margin">Alexander F. Wilson</h3><p><sup>i</sup>Inherited Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Baltimore, MD,  21224;</p><div>Find articles by <a href="/pubmed/?term=Wilson%20AF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Alexander F. Wilson</a></div></div><div id="_co_idm140703053156768"><h3 class="no_margin">James L. Mills</h3><p><sup>l</sup>Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Bethesda, MD,  20892; and</p><div>Find articles by <a href="/pubmed/?term=Mills%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">James L. Mills</a></div></div><div id="_co_idm140703053154656"><h3 class="no_margin">John M. Scott</h3><p><sup>m</sup>School of Biochemistry and Immunology, Trinity College, Dublin 2, Ireland;</p><div>Find articles by <a href="/pubmed/?term=Scott%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">John M. Scott</a></div></div><div id="_co_idm140703040867616"><h3 class="no_margin">Lawrence C. Brody</h3><p><sup>n</sup>Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892</p><div>Find articles by <a href="/pubmed/?term=Brody%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Lawrence C. Brody</a></div></div><div id="_co_idm140703046621280"><h3 class="no_margin">Jun Z. Li</h3><p><sup>b</sup>Department of Human Genetics,</p><p><sup>o</sup>Department of Computational Medicine and Bioinformatics, and</p><div>Find articles by <a href="/pubmed/?term=Li%20JZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">Jun Z. Li</a></div></div><div id="_co_idm140703053122832"><h3 class="no_margin">David Ginsburg</h3><p><sup>a</sup>Department of Pediatrics and Communicable Diseases,</p><p><sup>b</sup>Department of Human Genetics,</p><p><sup>p</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, MI,  48109;</p><p><sup>c</sup>Howard Hughes Medical Institute, Ann Arbor, MI, 48109;</p><div>Find articles by <a href="/pubmed/?term=Ginsburg%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23267103">David Ginsburg</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140703124735760_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140703124735760_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140703124735760_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Department of Pediatrics and Communicable Diseases,</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Department of Human Genetics,</div><div class="fm-affl" id="aff3" lang="en"><sup>o</sup>Department of Computational Medicine and Bioinformatics, and</div><div class="fm-affl" id="aff4" lang="en"><sup>p</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, MI,  48109;</div><div class="fm-affl" id="aff5" lang="en"><sup>c</sup>Howard Hughes Medical Institute, Ann Arbor, MI, 48109;</div><div class="fm-affl" id="aff6" lang="en"><sup>d</sup>Hadassah-Hebrew University Medical Center, Jerusalem 91120, Israel;</div><div class="fm-affl" id="aff7" lang="en"><sup>e</sup>Department of Pediatrics, Duke University, Durham, NC,  27710;</div><div class="fm-affl" id="aff8" lang="en"><sup>f</sup>Department of Pediatrics, Northwestern University, Chicago, IL,  60611;</div><div class="fm-affl" id="aff9" lang="en"><sup>g</sup>Department of Biostatistics, University of Washington, Seattle, WA,  98105;</div><div class="fm-affl" id="aff10" lang="en"><sup>h</sup>Broad Institute of MIT and Harvard, Cambridge, MA,  02142;</div><div class="fm-affl" id="aff11" lang="en"><sup>i</sup>Inherited Disease Research Branch, National Human Genome Research Institute, National Institutes of Health, Baltimore, MD,  21224;</div><div class="fm-affl" id="aff12" lang="en"><sup>j</sup>School of Medicine and</div><div class="fm-affl" id="aff13" lang="en"><sup>m</sup>School of Biochemistry and Immunology, Trinity College, Dublin 2, Ireland;</div><div class="fm-affl" id="aff14" lang="en"><sup>k</sup>Child Health Epidemiology Unit, Health Research Board, Dublin, Ireland;</div><div class="fm-affl" id="aff15" lang="en"><sup>l</sup>Eunice Kennedy Shriver National Institute of Child Health and Human Development, National Institutes of Health, Bethesda, MD,  20892; and</div><div class="fm-affl" id="aff16" lang="en"><sup>n</sup>Genome Technology Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, MD 20892</div><div id="cor1"><sup>1</sup>To whom correspondence may be addressed. E-mail: <a href="mailto:dev@null" data-email="ude.hcimu@ilznuj" class="oemail">ude.hcimu@ilznuj</a> or <a href="mailto:dev@null" data-email="ude.hcimu@grubsnig" class="oemail">ude.hcimu@grubsnig</a>.</div><div id="idm140703043923072">Contributed by David Ginsburg, November 19, 2012 (sent for review September 20, 2012)</div><div><p id="__p2" class="p p-first-last">Author contributions: K.C.D., A.B.O., J.A.S., C.D.T., A.A.S., J.Z.L., and D.G. designed research; K.C.D., A.B.O., D.S., Y. Kalish, A.A.S., A.C., D.B.M., and J.Z.L. performed research; K.C.D., A.B.O., D.S., A.C., D.B.M., Y. Kim, C.D.C., A.M.M., P.N.K., J.E.B.-W., A.F.W., J.L.M., J.M.S., and L.C.B. contributed new reagents/analytic tools; K.C.D., A.B.O., C.P.M., C.C.L., J.Z.L., and D.G. analyzed data; and K.C.D., A.B.O., L.C.B., J.Z.L., and D.G. wrote the paper.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140703124735760_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Freely available online through the PNAS open access option.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3545809/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supporting Information
</div>

<div><a href="/pmc/articles/PMC3545809/bin/supp_110_2_588__index.html" data-ga-action="click_feat_suppl">supp_110_2_588__index.html</a><span style="color:gray"> (6.9K)</span></div><div class="small guid">GUID:&nbsp;114DC7CE-2249-420D-A985-C4909B021AB5</div>
<div><a href="/pmc/articles/PMC3545809/bin/1219885110_sapp.pdf" data-ga-action="click_feat_suppl">1219885110_sapp.pdf</a><span style="color:gray"> (5.6M)</span></div><div class="small guid">GUID:&nbsp;C693980E-FB6E-4A66-B296-69420BB59032</div>
</div></dd></dl></div><div id="idm140703087463280" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140703087463280title" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">The plasma glycoprotein von Willebrand factor (VWF) exhibits fivefold antigen level variation across the normal human population determined by both genetic and environmental factors. Low levels of VWF are associated with bleeding and elevated levels with increased risk for thrombosis, myocardial infarction, and stroke. To identify additional genetic determinants of VWF antigen levels and to minimize the impact of age and illness-related environmental factors, we performed genome-wide association analysis in two young and healthy cohorts (<em>n</em> = 1,152 and <em>n</em> = 2,310) and identified signals at <em>ABO</em> (<em>P</em> &lt; 7.9E-139) and <em>VWF</em> (<em>P</em> &lt; 5.5E-16), consistent with previous reports. Additionally, linkage analysis based on sibling structure within the cohorts, identified significant signals at chromosome 2q12–2p13 (LOD score 5.3) and at the <em>ABO</em> locus on chromosome 9q34 (LOD score 2.9) that explained 19.2% and 24.5% of the variance in VWF levels, respectively. Given its strong effect, the linkage region on chromosome 2 could harbor a potentially important determinant of bleeding and thrombosis risk. The absence of a chromosome 2 association signal in this or previous association studies suggests a causative gene harboring many genetic variants that are individually rare, but in aggregate common. These results raise the possibility that similar loci could explain a significant portion of the “missing heritability” for other complex genetic traits.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genome-wide association study, linkage study, venous thromboembolic disease, von Willebrand disease, quantitative trait loci</span></div></div><div id="idm140703089066784" class="tsec sec headless whole_rhythm"><p id="__p4" class="p p-first">Von Willebrand factor (VWF) is a multimeric plasma glycoprotein that plays a central role in hemostasis by acting as a molecular bridge tethering platelets to injured endothelium and as a carrier molecule for coagulation factor VIII (<a href="#r1" rid="r1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419726">1</a>). Quantitative or qualitative deficiencies in VWF lead to von Willebrand Disease (VWD), the most common inherited bleeding disorder, with an estimated prevalence of 0.002–0.01% worldwide (<a href="#r1" rid="r1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419739">1</a>, <a href="#r2" rid="r2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419743">2</a>). Type I VWD is characterized by mild to moderate bleeding and low circulating VWF levels. This form of VWD is generally associated with haploinsufficiency for <em>VWF</em> and is characterized by incomplete penetrance. In contrast, elevated levels of plasma VWF are an independent risk factor for venous thromboembolic disease (<a href="#r3" rid="r3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419767">3</a>), myocardial infarction (<a href="#r4" rid="r4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419737">4</a>), stroke (<a href="#r5" rid="r5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419756">5</a>, <a href="#r6" rid="r6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419728">6</a>), and also complicate anticoagulant management (<a href="#r7" rid="r7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419730">7</a>).</p><p id="__p5">Plasma VWF levels vary by approximately fivefold in healthy populations and are influenced by both environmental and inherited factors. Increased levels of VWF occur with advancing age (<a href="#r8" rid="r8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419776">8</a>), may rise acutely because of inflammation or infection, and may serve as a surrogate marker for endothelial dysfunction and atherosclerosis (<a href="#r9" rid="r9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419742">9</a><a href="#r10" rid="r10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419724"></a>–<a href="#r11" rid="r11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419740">11</a>). Estimates for the heritability of plasma VWF levels in the general population from previous family-based studies range from 32–75%. A 1985 study in Norwegian twins reported the heritability of VWF at 66%, with 30% of this effect attributable to ABO blood type (<a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419747">12</a>). More recent studies estimated the heritability of VWF levels to be as high as 75% in United Kingdom twins (<a href="#r13" rid="r13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419722">13</a>) and as low as 53% in elderly Danish twins (<a href="#r14" rid="r14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419750">14</a>). In contrast, analysis in 21 Spanish families calculated VWF level heritability at only 32% (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419713">15</a>), identifying with significant linkage only observed at the <em>ABO</em> locus (LOD = 3.46).</p><p id="__p6">The connection between VWF antigen levels and ABO blood group has been well studied, with VWF levels reduced by ∼30% in type O individuals compared with most other individuals with non-O blood types (<a href="#r16" rid="r16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419760">16</a>). Approximately 20% of the mass of circulating VWF is composed of carbohydrate side chains, with <em>N</em>-linked ABO blood-group glycans representing ∼13% of the total glycan structures (<a href="#r17" rid="r17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419712">17</a>). The ABO enzyme is a glycosyltransferase that attaches <em>N</em>-acetyl galactosamine (A allele) or simply galactose (B allele) to the H antigen (oligosaccharide that ends in a fucose linked to galactose) on target proteins. The common O allele results from a single nucleotide deletion (G261del) in exon 6 of <em>ABO</em>, which creates an early stop codon and a nonfunctioning transferase (<a href="#r18" rid="r18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419763">18</a>). Although the precise molecular mechanisms remain unclear, individuals with type O blood group (35–45% of the population) exhibit a shorter VWF half-life, suggesting that the major effect of <em>ABO</em> on VWF level is through alteration of clearance rates (<a href="#r16" rid="r16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419762">16</a>). The A2 allele encodes a hypomorphic version of A1 with 30- to 50-fold less enzyme activity (<a href="#r19" rid="r19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419738">19</a>). The relationship of <em>ABO</em> variants and VWF levels fits an autosomal recessive pattern, with similarly elevated VWF levels observed in A1O, BO, and AB individuals.</p><p id="__p7">A genome-wide association study (GWAS) meta-analysis of 17,596 individual adults (average age 58 y) from several large cardiovascular disease cohorts confirmed the influence of common variants at the <em>ABO</em> locus on plasma VWF level and demonstrated additional smaller association signals at <em>VWF</em> and six other loci (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419774">20</a>). However, the total variance in plasma VWF explained by all of the significant loci in this study was estimated at only 12.8%, notably less than the heritability estimates of 32–75% previously reported in literature.</p><p id="__p8" class="p">To identify additional quantitative trait loci (QTL) for VWF antigen variation and to minimize the impact of age and illness-related environmental factors, we studied two independent healthy young cohorts, the Genes and Blood-Clotting Study (GABC, <em>n</em> = 1,152) and the Trinity Student Study (TSS, <em>n</em> = 2,310) (<a href="#r21" rid="r21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419721">21</a><a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419717"></a>–<a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419748">23</a>). In addition to confirming the association signals at <em>ABO</em> and <em>VWF</em>, linkage analysis identified a previously unidentified QTL on chromosome 2 (Chr2) that was undetected in our GWAS or previous studies. This locus explains 19.2% of additional plasma VWF variance and suggests that the use of cohorts with family structure can identify missing genetic determinants for other complex traits with similar allelic architecture.</p></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title" style="text-transform: uppercase;">Results</h2><div id="__sec1" class="sec sec-first"><p></p><h3 id="__sec1title" class="inline">Study Cohorts. </h3><p id="__p9" class="p p-first"><a href="/pmc/articles/PMC3545809/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a> displays demographic information for the two study cohorts. The median age of participants was 21 y (Q1:19, Q3:23) for the GABC cohort and 22 y (Q1:21, Q3:24) for the TSS. Although the TSS cohort was 100% ethnically Irish, the GABC cohort was of mixed ancestry, consistent with the University of Michigan student population from which participants were recruited (<a href="#r24" rid="r24" class=" bibr popnode">24</a>). Genotyping data identified 81.5% (940 of 1,152) of the GABC cohort as European ancestry, in close agreement with self-report. Of the 502 GABC families (1,152 individuals) who passed genotyping quality control, 13 were singletons, 366 were pairs, and 94, 22, 5, and 2 were comprised of 3, 4, 5, and 6 members, respectively. In the TSS cohort, 2,310 participants were successfully genotyped, including 71 sibling pairs and one sibling trio, for a total of 145 full siblings.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t01"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p10">Characteristics of study participants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140703040702624" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Cohort</td><td rowspan="1" colspan="1" align="center">GABC</td><td rowspan="1" colspan="1" align="center">TSS</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Subject counts, <em>n</em></td><td rowspan="1" colspan="1" align="center">1,152</td><td rowspan="1" colspan="1" align="center">2,310</td></tr><tr><td rowspan="1" colspan="1">Age, y (Q1, Q3)</td><td rowspan="1" colspan="1" align="center">21 (19, 23)</td><td rowspan="1" colspan="1" align="center">22 (21, 24)</td></tr><tr><td rowspan="1" colspan="1">Female (%)</td><td rowspan="1" colspan="1" align="center">721 (63)</td><td rowspan="1" colspan="1" align="center">1,310 (59)</td></tr><tr><td rowspan="1" colspan="1">Weight, lb (Q1, Q3)</td><td rowspan="1" colspan="1" align="center">145 (125, 165)</td><td rowspan="1" colspan="1" align="center">148 (133, 167)</td></tr><tr><td rowspan="1" colspan="1">Body mass index, kg/m<sup>2</sup> (Q1, Q3)</td><td rowspan="1" colspan="1" align="center">22.5 (20.7, 25.0)</td><td rowspan="1" colspan="1" align="center">22.6 (21.0, 24.4)</td></tr><tr><td rowspan="1" colspan="1">Current smoker, <em>n</em>/total cohort (%)</td><td rowspan="1" colspan="1" align="center">55/1,152 (4.8)</td><td rowspan="1" colspan="1" align="center">737/2,299 (32.1)</td></tr><tr><td rowspan="1" colspan="1">Sibship size, (<em>n</em> sibships)</td><td rowspan="1" colspan="1" align="center">2 (366); 3 (94); 4 (22);5 (5); 6 (2)</td><td rowspan="1" colspan="1" align="center">2 (71); 3 (1)</td></tr><tr><td rowspan="1" colspan="1">VWF, IU/dL (Q1, Q3)</td><td rowspan="1" colspan="1" align="center">108.1 (78.3, 129.7)</td><td rowspan="1" colspan="1" align="center">108.4 (79.3, 128.2)</td></tr></tbody></table></div><div id="largeobj_idm140703040702624" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3545809/table/t01/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140703087430288"><p id="__p11" class="p p-first-last">Median values for age, weight, body mass index, and VWF levels are reported with quartiles in parentheses.</p></div></div></div></div><div id="__sec2" class="sec"><p></p><h3 id="__sec2title" class="inline">VWF Antigen Levels. </h3><p id="__p12" class="p p-first-last"><a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S1</a> displays the distribution of untransformed VWF antigen levels. The median VWF levels were 108.1 IU/dL and 108.4 IU/dL for the GABC and TSS cohorts, respectively (<a href="/pmc/articles/PMC3545809/table/t01/" target="table" class="fig-table-link figpopup" rid-figpopup="t01" rid-ob="ob-t01" co-legend-rid=""><span>Table 1</span></a>). The 5th and 95th percentiles of the distribution spanned a 3.4-fold difference, from 54.3 IU/dL to 187.1 IU/dL (GABC), and 3.2-fold difference, from 59.1 IU/dL to 187.2 IU/dL (TSS). There was no significant difference in the distributions of VWF levels between GABC and TSS (Kolmogorov–Smirnov test, <em>P</em> = 0.16). Although 23% were regular tobacco smokers (4.8% GABC, 32% TSS) based on self-report and confirmed with serum cotinine levels (TSS only), there was no significant difference in the distribution of VWF levels between smokers and nonsmokers (Kolmogorov–Smirnov test, <em>P</em> = 0.40). Narrow sense heritability (<em>h</em><sup>2</sup>) of the VWF antigen level in the combined GABC and TSS dataset was first estimated by using the known sibship relationships, yielding an intraclass correlation-based upper-bound estimate of 64.5%. This finding is consistent with the estimates derived from the genome-wide genotyping data for all individuals: 66.3% using MERLIN, and 64.9% using GCTA (<a href="#s3" rid="s3" class=" sec"><em>Materials and Methods</em></a>) (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419718">25</a>, <a href="#r26" rid="r26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419765">26</a>).</p></div><div id="__sec3" class="sec"><p></p><h3 id="__sec3title" class="inline">Association Studies, GABC Cohort. </h3><p id="__p13" class="p p-first-last">In the GABC dataset, 38 SNPs were significantly associated with VWF antigen level (<em>P</em> &lt; 5.0E-8) after adjustment for age, sex, and population structure (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2<em>A</em></a>). All 38 SNPs reside in a 300-kb region on Chr 9q34, with 31 of 38 located within the <em>ABO</em> gene (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2<em>B</em></a>), consistent with the association signal for <em>ABO</em> reported in a previous meta-analysis (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419745">20</a>). The G allele of the top SNP, rs687289, was associated with decreased VWF antigen level and tags the O allele of <em>ABO</em> (β = −0.36 ± 0.022 IU/dL per allele in an additive model, <em>P</em> = 1.7E-52) (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S1</a><strong>)</strong>, consistent with the known association between type O blood group and the shorter half-life of VWF. The lowest <em>P</em> value outside of the 9q34-associated region was 1.5E-7, not reaching genome-wide significance. The Q-Q plot (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2<em>C</em></a>) of observed versus expected –log<sub>10</sub>(<em>P</em> value) demonstrates a large deviation from expectation as a result of the significant signals at the <em>ABO</em> locus (in red) and a slight signal inflation, possibly because of family structure. As the initial analysis treated all samples as unrelated, we applied multiple approaches to examine the impact of sample relatedness. These approaches include GWAF (genome-wide association analyses with family data) and EMMAX (<a href="#s3" rid="s3" class=" sec"><em>Materials and Methods</em></a>), the former using a linear mixed effect model and the latter using a variance components-based model. Log-scatter pair-wise comparisons (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2 <em>D</em> and <em>E</em></a>) and Q-Q plot comparisons (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2 <em>F</em> and <em>G</em></a>) showed that results obtained by considering relatedness were highly similar to those not considering relatedness. We therefore proceeded to use the original results (assuming unrelated samples) in the meta-analysis below.</p></div><div id="__sec4" class="sec"><p></p><h3 id="__sec4title" class="inline">Association Studies, TSS Cohort. </h3><p id="__p14" class="p p-first-last">The TSS cohort revealed two significantly associated regions (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3<em>A</em></a>), with 31 SNPs overlapping the <em>ABO</em> locus (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3<em>B</em></a>) and 10 SNPs overlapping the <em>VWF</em> gene (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3<em>C</em></a>). The Q-Q plot (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3<em>D</em></a>) of observed versus expected –log<sub>10</sub>(<em>P</em> value) exhibits a large deviation from expectation as a result of the significant signals at the <em>ABO</em> and <em>VWF</em> loci. The association statistics for the lead SNPs are shown in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S2</a>. The top <em>ABO</em> SNP in TSS, rs687289, was the same as the top <em>ABO</em> SNP in GABC, and demonstrated a similar effect size for the G allele (β = −0.33 ± 0.016, <em>P</em> = 3.7E-89). In the VWF region, the highest associated SNP was rs1063856 (β = −0.12 ± 0.015, <em>P</em> = 8.5E-14). This SNP did not reach genome-wide significance in GABC (β = 0.095 ± 0.024, <em>P</em> = 1.1E-4) likely because of the latter’s 2.4-fold smaller sample size compared with TSS.</p></div><div id="__sec5" class="sec"><p></p><h3 id="__sec5title" class="inline">Meta-Analysis of GABC and TSS. </h3><p id="__p15" class="p p-first">Seventy-three SNPs were significantly associated (<em>P</em> &lt; 5.0E-8) with VWF level in the meta-analysis of GABC and TSS (<a href="/pmc/articles/PMC3545809/figure/fig01/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig01" rid-ob="ob-fig01" co-legend-rid="lgnd_fig01"><span>Fig. 1</span></a> and <a href="/pmc/articles/PMC3545809/table/t02/" target="table" class="fig-table-link figpopup" rid-figpopup="t02" rid-ob="ob-t02" co-legend-rid=""><span>Table 2</span></a>). These SNPs collectively explained ∼18.7% of the VWF level variation in an analysis using GCTA (a tool for genome-wide complex trait analysis). Of the 73 SNPs, 58 were in the associated region surrounding the <em>ABO</em> gene (including <em>ADAMTS13</em>) (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S4<em>A</em></a>), with the remaining 15 overlapping the <em>VWF</em> locus (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S4<em>B</em></a>). The strongest signal outside of the <em>ABO</em> and <em>VWF</em> loci had a <em>P</em> value of 1.3E-6, not reaching genome-wide significance.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig01" co-legend-rid="lgnd_fig01"><a href="/pmc/articles/PMC3545809/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140703048615616" class="figure"><a href="/pmc/articles/PMC3545809/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3545809_pnas.1219885110fig01.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1219885110fig01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3545809/bin/pnas.1219885110fig01.jpg"></a></div><div id="largeobj_idm140703048615616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3545809/figure/fig01/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig01"><div><a class="figpopup" href="/pmc/articles/PMC3545809/figure/fig01/" target="figure" rid-figpopup="fig01" rid-ob="ob-fig01">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p16">GABC and TSS association meta-analysis (∼724 K SNPs) using age-, sex-, and principal component-adjusted VWF values. Genome-wide −log<sub>10</sub>(<em>P</em> value) plot. The horizontal line marks the 5E-08 threshold of genome-wide significance.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t02"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p17">Top meta-analysis SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140703048612800" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td colspan="6" rowspan="1">Meta-analysis<hr></td><td colspan="3" rowspan="1" align="center">GABC<hr></td><td colspan="3" rowspan="1" align="center">TSS<hr></td></tr><tr><td rowspan="1" colspan="1">Chr</td><td rowspan="1" colspan="1" align="center">SNP</td><td rowspan="1" colspan="1" align="center">Position<sup>*</sup></td><td rowspan="1" colspan="1" align="center">Allele<sup>†</sup></td><td rowspan="1" colspan="1" align="center"><em>P</em> value<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">Closest gene</td><td rowspan="1" colspan="1" align="center">Freq</td><td rowspan="1" colspan="1" align="center">β (SE)</td><td rowspan="1" colspan="1" align="center"><em>P</em> value<sup>§</sup></td><td rowspan="1" colspan="1" align="center">Freq</td><td rowspan="1" colspan="1" align="center">β (SE)</td><td rowspan="1" colspan="1" align="center"><em>P</em> value<sup>§</sup></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs687289</td><td rowspan="1" colspan="1" align="center">135126927</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.3E-128</td><td rowspan="1" colspan="1"><em>ABO</em></td><td rowspan="1" colspan="1" align="center">0.643</td><td rowspan="1" colspan="1" align="center">0.36 (0.022)</td><td rowspan="1" colspan="1" align="center">1.6E-52</td><td rowspan="1" colspan="1" align="center">0.751</td><td rowspan="1" colspan="1" align="center">0.33 (0.016)</td><td rowspan="1" colspan="1" align="center">3.7E-89</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs11244079</td><td rowspan="1" colspan="1" align="center">135174347</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">4.4E-36</td><td rowspan="1" colspan="1"><em>LOC653163</em></td><td rowspan="1" colspan="1" align="center">0.949</td><td rowspan="1" colspan="1" align="center">0.33 (0.055)</td><td rowspan="1" colspan="1" align="center">4.6E-09</td><td rowspan="1" colspan="1" align="center">0.937</td><td rowspan="1" colspan="1" align="center">0.35 (0.030)</td><td rowspan="1" colspan="1" align="center">3.9E-31</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs4962153</td><td rowspan="1" colspan="1" align="center">135313575</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">1.9E-31</td><td rowspan="1" colspan="1"><em>ADAMTS13</em></td><td rowspan="1" colspan="1" align="center">0.839</td><td rowspan="1" colspan="1" align="center">0.208 (0.032)</td><td rowspan="1" colspan="1" align="center">1.6E-10</td><td rowspan="1" colspan="1" align="center">0.894</td><td rowspan="1" colspan="1" align="center">0.25 (0.024)</td><td rowspan="1" colspan="1" align="center">1.2E-24</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs11244035</td><td rowspan="1" colspan="1" align="center">135071140</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">1.9E-25</td><td rowspan="1" colspan="1"><em>OBP2B</em></td><td rowspan="1" colspan="1" align="center">0.885</td><td rowspan="1" colspan="1" align="center">0.27 (0.039)</td><td rowspan="1" colspan="1" align="center">3.2E-12</td><td rowspan="1" colspan="1" align="center">0.915</td><td rowspan="1" colspan="1" align="center">0.23 (0.027)</td><td rowspan="1" colspan="1" align="center">3.4E-17</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs45618736</td><td rowspan="1" colspan="1" align="center">137580983</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">4.6E-24</td><td rowspan="1" colspan="1"><em>OBP2A</em></td><td rowspan="1" colspan="1" align="center">0.890</td><td rowspan="1" colspan="1" align="center">0.26 (0.040)</td><td rowspan="1" colspan="1" align="center">1.3E-10</td><td rowspan="1" colspan="1" align="center">0.918</td><td rowspan="1" colspan="1" align="center">0.24 (0.029)</td><td rowspan="1" colspan="1" align="center">5.4E-17</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs28602591</td><td rowspan="1" colspan="1" align="center">135236487</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">3.3E-20</td><td rowspan="1" colspan="1"><em>C9orf96</em></td><td rowspan="1" colspan="1" align="center">0.886</td><td rowspan="1" colspan="1" align="center">0.19 (0.038)</td><td rowspan="1" colspan="1" align="center">5.7E-07</td><td rowspan="1" colspan="1" align="center">0.928</td><td rowspan="1" colspan="1" align="center">0.24 (0.029)</td><td rowspan="1" colspan="1" align="center">4.8E-16</td></tr><tr><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">rs1063856</td><td rowspan="1" colspan="1" align="center">6023795</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">4.9E-16</td><td rowspan="1" colspan="1"><em>VWF</em></td><td rowspan="1" colspan="1" align="center">0.360</td><td rowspan="1" colspan="1" align="center">−0.095 (0.024)</td><td rowspan="1" colspan="1" align="center">1.1E-04</td><td rowspan="1" colspan="1" align="center">0.395</td><td rowspan="1" colspan="1" align="center">−0.12 (0.015)</td><td rowspan="1" colspan="1" align="center">8.5E-14</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs4507838</td><td rowspan="1" colspan="1" align="center">135092020</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">1.54E-13</td><td rowspan="1" colspan="1"><em>LOC286310</em></td><td rowspan="1" colspan="1" align="center">0.875</td><td rowspan="1" colspan="1" align="center">0.081 (0.036)</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">0.867</td><td rowspan="1" colspan="1" align="center">0.17 (0.022)</td><td rowspan="1" colspan="1" align="center">4.8E-14</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs554710</td><td rowspan="1" colspan="1" align="center">135171669</td><td rowspan="1" colspan="1" align="center">T</td><td rowspan="1" colspan="1" align="center">4.7E-12</td><td rowspan="1" colspan="1"><em>SURF6</em></td><td rowspan="1" colspan="1" align="center">0.281</td><td rowspan="1" colspan="1" align="center">0.14 (0.027)</td><td rowspan="1" colspan="1" align="center">1.5E-07</td><td rowspan="1" colspan="1" align="center">0.286</td><td rowspan="1" colspan="1" align="center">0.088 (0.017)</td><td rowspan="1" colspan="1" align="center">1.8E-07</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs11655</td><td rowspan="1" colspan="1" align="center">135197664</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">1.9E-09</td><td rowspan="1" colspan="1"><em>SURF5</em></td><td rowspan="1" colspan="1" align="center">0.885</td><td rowspan="1" colspan="1" align="center">−0.16 (0.037)</td><td rowspan="1" colspan="1" align="center">3.2E-05</td><td rowspan="1" colspan="1" align="center">0.885</td><td rowspan="1" colspan="1" align="center">−0.11 (0.024)</td><td rowspan="1" colspan="1" align="center">1.8E-06</td></tr></tbody></table></div><div id="largeobj_idm140703048612800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3545809/table/t02/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140703052877504"><p id="__p18" class="p p-first-last">Genome-wide significant (<em>P</em> value &lt; 5.0E-8) SNPs in GABC+TSS meta-analysis and individual cohorts, sorted by <em>P</em> value, with respect to their relationship to the nearest gene.</p></div><div id="tfn1"><p id="__p19" class="p p-first-last">*Positions are in National Center for Biotechnology Information Build 36 coordinates.</p></div><div id="tfn2"><p id="__p20" class="p p-first-last"><sup>†</sup>Tested allele.</p></div><div id="tfn3"><p id="__p21" class="p p-first-last"><sup>‡</sup>Meta-analysis <em>P</em> value.</p></div><div id="tfn4"><p id="__p22" class="p p-first-last"><sup>§</sup>Cohort specific <em>P</em> value.</p></div></div></div><p id="__p23" class="p p-last">Thirty-nine of the 73 significant SNPs in the meta-analysis were not significant in the GABC cohort, whereas 12 of the 73 were not significant in TSS. However, the allelic effect sizes and directions of the top SNPs in each cohort showed excellent agreement (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S5</a>). <em>R</em><sup>2</sup> values were 0.95 and 0.90 for the β-values of the top 38 GABC and top 49 TSS SNPs, respectively, suggesting that the apparent difference in significance levels are mainly due to sample size differences.</p></div><div id="__sec6" class="sec"><p></p><h3 id="__sec6title" class="inline">Conditional Analysis. </h3><p id="__p24" class="p p-first-last">To screen for potential secondary signals masked by the strong primary signal at <em>ABO</em>, the top SNP, rs687289, was introduced as a covariate in a conditional association analysis in the joint set of GABC and TSS cohorts (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S6 <em>A</em> and <em>D</em></a>). A new signal emerged at chromosome 9 in the <em>ABO</em> locus, with rs8176704 as the lead SNP (<em>P</em> = 4.1E-34), which was not significant in the original test (<em>P</em> = 0.18) (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S6 <em>B</em> and <em>C</em></a>). The A allele of rs8176704 tags the A2 allele of <em>ABO</em> and is associated with a decrease in VWF levels (β = −0.34), consistent with the hypomorphic effect of the A2 allele. The significant SNPs in the <em>ADAMTS13</em> locus in the meta-analysis were no longer significant in the conditional analysis, suggesting that the <em>ADAMTS13</em> signal may be because of its linkage disequilibrium with <em>ABO</em>. No other new significant genome-wide signals were detected in the conditional analysis.</p></div><div id="__sec7" class="sec"><p></p><h3 id="__sec7title" class="inline">ABO Haplotype Effects. </h3><p id="__p25" class="p p-first">Four major ABO SNP haplotypes, tagged by three ABO SNPs: rs8176704, rs8176749, and rs687289 (<a href="#r27" rid="r27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419741">27</a>), determine the majority of the ABO blood group serotypes (A1, A2, B, and O). We investigated their association with VWF antigen levels. Using the haplotypes inferred from genotype data we deduced each subject’s most likely ABO serotypes. The overall ABO blood group allele frequencies were similar to those reported in the Atherosclerosis Risk in Communities Study (ARIC) (<a href="#r28" rid="r28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419757">28</a>) and Framingham Heart Study (FHS) (<a href="#r29" rid="r29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419775">29</a>) cohort (<a href="/pmc/articles/PMC3545809/table/t03/" target="table" class="fig-table-link figpopup" rid-figpopup="t03" rid-ob="ob-t03" co-legend-rid=""><span>Table 3</span></a>), and were in concordance with the expected frequencies in populations with European ancestry (<a href="#r30" rid="r30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419778">30</a>). As expected, the A1 and B alleles were positively and significantly associated with the VWF antigen levels (β = 0.36 and <em>P</em> = 3.5E-98; β = 0.36 and <em>P</em> = 7.2E-50, respectively) and O and A2 alleles were negatively associated, but only the O allele was significant (β = −0.34 and <em>P</em> = 2.3E-138; β = −0.038 and <em>P</em> = 0.18, respectively).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t03"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p26"><em>ABO</em> haplotype association</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140703089112352" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1" align="center">Allele/haplotype frequencies<hr></td><td colspan="2" rowspan="1" align="center">GABC+TSS haplotype association<hr></td></tr><tr><td rowspan="1" colspan="1"><em>ABO</em>-allele</td><td rowspan="1" colspan="1" align="center">Haplotype<sup>*</sup></td><td rowspan="1" colspan="1" align="center">ARIC</td><td rowspan="1" colspan="1" align="center">FHS</td><td rowspan="1" colspan="1" align="center">GABC + TSS</td><td rowspan="1" colspan="1" align="center">β</td><td rowspan="1" colspan="1" align="center"><em>P</em> value</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">O</td><td rowspan="1" colspan="1">GGC</td><td rowspan="1" colspan="1" align="center">0.650</td><td rowspan="1" colspan="1" align="center">0.673</td><td rowspan="1" colspan="1" align="center">0.720</td><td rowspan="1" colspan="1" align="center">−0.34</td><td rowspan="1" colspan="1" align="center">2.3E-138</td></tr><tr><td rowspan="1" colspan="1">A1</td><td rowspan="1" colspan="1">AGC</td><td rowspan="1" colspan="1" align="center">0.209</td><td rowspan="1" colspan="1" align="center">0.196</td><td rowspan="1" colspan="1" align="center">0.155</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">3.5E-98</td></tr><tr><td rowspan="1" colspan="1">B</td><td rowspan="1" colspan="1">AGT</td><td rowspan="1" colspan="1" align="center">0.0730</td><td rowspan="1" colspan="1" align="center">0.0770</td><td rowspan="1" colspan="1" align="center">0.0722</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">7.2E-50</td></tr><tr><td rowspan="1" colspan="1">A2</td><td rowspan="1" colspan="1">AAC</td><td rowspan="1" colspan="1" align="center">0.0680</td><td rowspan="1" colspan="1" align="center">0.0540</td><td rowspan="1" colspan="1" align="center">0.0531</td><td rowspan="1" colspan="1" align="center">−0.038</td><td rowspan="1" colspan="1" align="center">1.8E-01</td></tr></tbody></table></div><div id="largeobj_idm140703089112352" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3545809/table/t03/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140703034261312"><p id="__p27" class="p p-first-last">ABO-alleles, tagging haplotypes, and inferred frequencies in the FHS, ARIC, and GABC+TSS cohorts, and association results in GABC+TSS.</p></div><div id="tfn5"><p id="__p28" class="p p-first-last">*rs687289, rs8176704, rs8176749.</p></div></div></div></div><div id="__sec8" class="sec"><p></p><h3 id="__sec8title" class="inline">Focused Analysis at the VWF Locus. </h3><p id="__p29" class="p p-first">The second significant region in the TSS cohort and in the meta-analysis was the <em>VWF</em> locus. The lead SNP in the meta-analysis was rs1063856 (<em>P</em> = 5.5E-16), a coding nonsynonymous SNP at the <em>VWF</em> gene. It is in strong linkage disequilibrium (LD) with the second highly associated SNP, rs1063857 (<em>r</em><sup>2</sup> = 1.00, <em>P</em> = 6.5E-16), a coding synonymous SNP. Although this region was not significant in the GABC cohort, a power analysis using the Genetic Power Calculator (<a href="#r31" rid="r31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419761">31</a>) showed that given the effect size of β = 0.012, as seen in TSS, and given the sample size of GABC, the power to detect this VWF signal was only 38.4% in GABC.</p><p id="__p30" class="p p-last">Similar to the haplotype analysis at <em>ABO</em>, we extended the association analysis at <em>VWF</em> to haplotypes using the combined set of GABC and TSS. From the genotype data for the 71 available <em>VWF</em> SNPs we constructed a haplotype map of the VWF gene using Haploview and identified 12 LD blocks (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S7</a>). Of the 15 most highly associated SNPs in the meta-analysis, 11 reside in block 6, two reside in block 7 (the top two SNPs), and the remaining two reside in block 8. These three blocks span exons 14–22, which encode the D′, D2, and D3 domains of VWF and contain the VWF propeptide and factor VIII binding domains. Haplotype association tests of these blocks yielded <em>P</em> values of 8.8E-3, 6.3E-7, and 7.9E-3 for the most frequent haplotypes in block 6, block 7, and block 8, respectively.</p></div><div id="__sec9" class="sec"><p></p><h3 id="__sec9title" class="inline">Linkage Analysis. </h3><p id="__p31" class="p p-first">To scan for linkage signals for plasma VWF levels we analyzed the genotype data of siblings from the GABC and TSS cohorts, for a total of 561 sibships and 1,284 individuals. Linkage LOD scores from an initial scan using the unpruned dataset of ∼760 K SNP were likely inflated (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S8<em>A</em></a>), because of strong LD between many SNPs and missing parental genotypes (<a href="#r32" rid="r32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419733">32</a>). This explanation was supported by the reduced significance levels observed in a revised scan using an LD pruned dataset (1 SNP/Mb, <em>r</em><sup>2</sup> &lt; 0.001, 2.7 K SNPs) (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S8<em>B</em></a>).</p><p id="__p32">To incorporate data-derived LD patterns, we applied the LD-modeling algorithm in MERLIN (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419725">25</a>) to define independent LD clusters and conducted linkage analysis using inferred haplotype frequencies within clusters (<a href="#s3" rid="s3" class=" sec"><em>Materials and Methods</em></a>). First, markers were organized into clusters using an LD criterion of <em>r</em><sup>2</sup> = 0.001. This process produced ∼37 K nearly independent clusters; and the linkage analysis on each yielded its LOD score and <em>P</em> value. The per-cluster LOD scores (<a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>) revealed a strong linkage signal (peak LOD ∼5.3) crossing the centromere of Chr2 (2q12-2p13), as well as a strong signal on Chr9 (9q34, peak LOD ∼2.9). The latter region contains the <em>ABO</em> gene.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig02" co-legend-rid="lgnd_fig02"><a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140703051038496" class="figure"><a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3545809_pnas.1219885110fig02.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is pnas.1219885110fig02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3545809/bin/pnas.1219885110fig02.jpg"></a></div><div id="largeobj_idm140703051038496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3545809/figure/fig02/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig02"><div><a class="figpopup" href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" rid-figpopup="fig02" rid-ob="ob-fig02">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p33">Linkage analysis results for GABC+TSS using the age-, sex-, and principal component-adjusted VWF values. Genome-wide LOD Scores for 37 K “clusters,” defined in <em>MERLIN</em> to model the LD among SNPs (<a href="#s3" rid="s3" class=" sec"><em>Materials and Methods</em></a>).</p></div></div></div><p id="__p34">We evaluated the genome-wide significance of the linkage results with a locus-counting approach proposed by Wiltshire et al. (<a href="#r33" rid="r33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419771">33</a>) comparing the observed LOD scores of the top independent regions of linkage (IRLs) with the null distributions of LOD scores for their corresponding equal-ranked IRLs over 1,000 simulated datasets with randomized phenotypes. The IRLs were defined as the 40-cM interval around the location of the maximum LOD score. This analysis showed that the three highest IRLs in our study had higher LOD scores than the 95th percentile of their respective equal-ranked LOD score null distributions (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>A</em></a>). The highest signal, at ∼82.4 Mb on Chr2, had a LOD score of 5.27 (original MERLIN <em>P</em> value = 4.2E-7; simulation-based empirical <em>P</em> value = 0.0015) (<a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a>, and <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>A</em></a>). The second strongest signal, at ∼135.3 Mb on chromosome 9, had a LOD score of 2.87 (<em>P</em> = 1.4E-4; empirical <em>P</em> = 0.048) (<a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a> and <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>A</em></a>). The third strongest signal was at ∼104.46 Mb on Chr2, with a LOD score of 2.687 (<em>P</em> = 2.2E-4; empirical <em>P</em> = 0.020) (<a href="/pmc/articles/PMC3545809/figure/fig02/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig02" rid-ob="ob-fig02" co-legend-rid="lgnd_fig02"><span>Fig. 2</span></a> and <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>A</em></a>).</p><p id="__p35">The analysis above used a fixed width (± 20 cM) to define IRL boundaries. An alternative approach is to apply a LOD score cutoff, such as one unit below the maxima. This LOD-based approach yielded an overlapping interval for the first and third significant IRLs, suggesting that the two intervals may be part of the same linkage region. The combined linkage region spans 34 Mb on 2q12-2p13 (positions 74.98 Mb–108.95 Mb), crossing over the centromere (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>B</em></a>). Using the same LOD score cutoff, the second top significant IRL spans 790 Kb on 9q34 (positions 135.03 Mb–135.82 Mb) and includes the <em>ABO</em> locus (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S9<em>C</em></a>).</p><p id="__p36" class="p p-last">By using GCTA we estimated that the LOD score-based linkage intervals on Chr2 and 9 explained 19.2% and 24.5% of the variation in the VWF levels, respectively. Furthermore, by expanding from the core region on 2q12–2p13, we found that the variance explained changed minimally beyond the 34-Mb core interval, increasing from 19.2–19.7% when expanded to 48 cM, suggesting that the 34-Mb interval contains the majority of the linkage signal.</p></div><div id="__sec10" class="sec sec-last"><p></p><h3 id="__sec10title" class="inline">Focused Analysis in the Chr2 Linkage Region. </h3><p id="__p37" class="p p-first">For the Chr2 linkage region (74.98 Mb–108.95 Mb), the phased genotype data defined 886 haplotype blocks, containing 4,074 distinct haplotypes with frequency &gt;1% in our study (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix, SI Methods</em></a>). We performed haplotype-based association tests for VWF levels in each block. None of the tests reached region-wide significance (<em>P</em> &lt; 5.6E-5) (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S10<em>A</em></a>), and few blocks produced substantially smaller haplotype-based <em>P</em> values than single-SNP <em>P</em> values in the same block (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S10<em>B</em></a>). Thus, these results did not provide clear evidence of association to common haplotypes that could explain the linkage signal in Chr2.</p><p id="__p38" class="p p-last">Next, we screened for sets of at least three SNPs with different LD structures between high-VWF and low-VWF individuals in the GABC cohort and identified six SNP sets in the linkage interval (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix, SI Methods</em></a>). We performed association tests using the long-range haplotypes formed by these SNP sets and discovered two sets with <em>P</em> values smaller than 0.001 (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Table S3</a>). Simulation results showed that these two sets reached an empirical significance level of 0.05. The first set (empirical <em>P</em> = 0.006) was anchored by the index SNP rs6547231 (association <em>P</em> = 5.8E-4) and spanned a 987-kb interval from 78.963 Mb to 79.951 Mb. The second set (empirical <em>P</em> = 0.002), anchored by the index SNP rs7566719 (association <em>P</em> = 3.3E-4), covered an ∼973-kb interval from 76.240 Mb to 77.214 Mb.</p></div></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title" style="text-transform: uppercase;">Discussion</h2><p id="__p39" class="p p-first">Our association analysis of VWF variation in healthy young subjects identified two major signals at the <em>ABO</em> and <em>VWF</em> loci, confirming results from a previously published meta-analysis. Taking advantage of the sibling structure of our cohorts, we also identified a unique QTL on Chr2, which was undetected in previous studies (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419746">15</a>, <a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419734">20</a>, <a href="#r34" rid="r34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419768">34</a>, <a href="#r35" rid="r35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419716">35</a>).</p><p id="__p40">With a total sample size of 3,250, our GWAS was underpowered to detect the several smaller-effect loci identified in the larger CHARGE (Cohorts for Heart and Aging Research in Genome Epidemiology) consortium meta-analysis (<em>n</em> = 17,596). However, the total variation explained by the genetic loci identified in our GWAS analysis (18.7%) was notably larger than that explained by the similar analysis in the CHARGE study (12.8%). The young age and healthy status of our subjects may have increased the contribution of the heritable components of VWF levels by limiting the impact of known environmental effects on VWF levels, including age and common illnesses, such as diabetes and heart disease. For example, contrary to studies in older cohorts (<a href="#r36" rid="r36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419773">36</a>, <a href="#r37" rid="r37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419735">37</a>), we found no associations between smoking and VWF levels in this study, suggesting that the previously observed association may be a proxy for vascular disease related to chronic smoking. Consistent with this notion, the estimated heritability of VWF variation of ∼65% from our study is at the upper end of the range previously reported (<a href="#r13" rid="r13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419752">13</a><a href="#r14" rid="r14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419766"></a>–<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419736">15</a>).</p><p id="__p41">The strongest association signals in our analysis were at the <em>ABO</em> locus. <em>ABO</em> haplotype analysis confirmed that SNPs tagging the O and A2 alleles were associated with lower VWF levels, but those tagging the B and A1 alleles were associated with higher VWF levels. Thus, the association signals at <em>ABO</em> are consistent with known alteration of VWF levels through specific ABO glycosylation patterns. Although we were able to detect the association of the A2 allele (minor allele frequency 0.05) after a conditional analysis, we were unable to detect several other less common alleles at <em>ABO</em> (<a href="#r30" rid="r30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419720">30</a>), likely because of their lower allele frequencies. The significantly associated SNPs at the VWF locus tag three of seven haplotype blocks, consistent with a similar analysis of the ARIC cohort (<a href="#r38" rid="r38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419719">38</a>, <a href="#r39" rid="r39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419758">39</a>). Although the causative SNPs are not yet at this point, they likely act via alterations in VWF biosynthesis, secretion, or clearance from plasma (<a href="#r40" rid="r40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419753">40</a>).</p><p id="__p42">Previous studies have reported significant single-gene associations between plasma VWF levels and variants at the <em>FUT2</em> locus (<a href="#r41" rid="r41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419727">41</a>, <a href="#r42" rid="r42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419769">42</a>) lipoprotein receptor-related protein (<em>LRP1</em>) (<a href="#r43" rid="r43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419759">43</a>), angiotensin-converting enzyme (<em>ACE</em>) (<a href="#r44" rid="r44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419714">44</a>), and arginine vasopressin 2 receptor (<em>AVPR2)</em> (<a href="#r45" rid="r45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419754">45</a>). None of these loci were detected in our analysis or in the CHARGE GWAS (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419729">20</a>), although the low minor allele frequency for the <em>LRP1</em> variant (&lt;2%) and X-chromosomal location of A<em>VPR2</em> limited the power for their detection.</p><p id="__p43">In addition to GWAS, the sibling structure of our GABC and TSS cohorts enabled linkage analysis, which identified a highly significant unique QTL on Chr2. The effect size of this locus on VWF variation (19.2% variance explained) was comparable to the effect of the <em>ABO</em> locus (24.5%). Variants in this linkage interval were not detected in our GWAS analysis of the same subjects, nor were they detected in the CHARGE study (<a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419772">20</a>). A previously reported linkage analysis in 21 families (the GAIT study) only identified significant linkage at the <em>ABO</em> locus (LOD = 3.46). Although a peak with LOD = 1.65 was reported on Chr2 (2q33.2), it did not overlap with the Chr2 peak in our study (<a href="#r15" rid="r15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419751">15</a>).</p><p id="__p44">The linkage peak on Chr2 spans ∼34 Mb and contains over 100 annotated genes, with plausible candidates for altering plasma VWF including several glycosyltransferases, sialyltransferases, and SNARE complex proteins potentially involved in protein secretion. Conventional haplotype-based analyses revealed no region-wide significant association in the Chr2 linkage region. However, analysis of differences in LD structure between high- and low-VWF individuals (<a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix, SI Methods</em></a>) identified two ∼1-Mb intervals that might harbor rare causal variants contributing to the linkage signal. The first region contained seven RefSeq genes (<em>REG3G</em>, <em>REG1B</em>, <em>REG1A</em>, <em>REG1P</em>, <em>REG3A</em>, <em>CTNNA2</em>, <em>MIR4264</em>) and the second region one RefSeq gene, <em>LRRTM4.</em> Although none of the genes play an obvious role in VWF biology, these regions represent targets for future study.</p><p id="__p45">We estimate that variants in the Chr2 linkage interval alter steady-state VWF level with a magnitude similar to ABO (20–30%). Therefore, detection as a Mendelian bleeding disorder similar to VWD (VWF levels reduced by &gt;50%) would be unlikely. However, this locus, together with ABO blood group, could represent an important modifier of VWD severity and penetrance, as well as the thrombosis risk associated with elevated VWF. Identification of the underlying gene and its mechanism of action could also lead to improved treatment for VWD and venous thromboembolic disease.</p><p id="__p46" class="p p-last">Mendelian genes for complex traits, such as type 2 diabetes, are often undetected by GWAS (<a href="#r46" rid="r46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419732">46</a>, <a href="#r47" rid="r47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419744">47</a>). However, mutations at these loci are generally very rare and contribute only a small part of the overall population variance for that trait. The large contribution of the Chr2 locus to plasma VWF variation, comparable to the effect of ABO but undetectable even by a well-powered GWAS, is surprising and not readily explained by known mechanisms of VWF homeostasis. We hypothesize that this locus may contain a critical VWF regulatory gene harboring rare causal variants in a situation analogous to allelic heterogeneity, thus having reduced power of detection in association tests. However, these variants might have sufficiently large effect size in each individual family and accrued linkage signals across families, thus contributing to the observed Chr2 linkage results. A similar pattern of a strong linkage signal without corresponding evidence for association was recently reported for a QTL for cystic fibrosis disease severity (<a href="#r48" rid="r48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419715">48</a>). Additionally, in silico experiments predict the presence rare variants detected by linkage that contribute to variation in complex genetic traits (<a href="#r49" rid="r49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_493546155">49</a>). Taken together, these findings suggest that similar loci could explain a significant portion of the “missing heritability” for other complex genetic traits.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title" style="text-transform: uppercase;">Materials and Methods</h2><div id="__sec11" class="sec sec-first"><p></p><h3 id="__sec11title" class="inline">Genes and Blood-Clotting Study. </h3><p id="__p47" class="p p-first-last">A cohort of healthy siblings was recruited from the University of Michigan, Ann Arbor, between June 26, 2006 and January 30, 2009. Participants were between the ages of 14 and 35 y, and had at least one eligible healthy sibling. Subjects who indicated that they were pregnant, had a known bleeding or blood-clotting disorder, or any illness requiring regular medical care were excluded. All participants provided informed consent by a process that was previously described (<a href="#r22" rid="r22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419755">22</a>). Subjects completed an online phenotyping survey and donated a blood sample for DNA extraction and plasma biochemical phenotyping. Details of the sample collection, genotyping, and data-cleaning process for the GABC cohort are described in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="__sec12" class="sec"><p></p><h3 id="__sec12title" class="inline">Trinity Student Study. </h3><p id="__p48" class="p p-first-last">A cohort of 2,524 healthy, ethnically Irish individuals, attending the University of Dublin, Trinity College, with ages between 18 and 28 y, was recruited over one academic year in 2003–2004 (<a href="#r21" rid="r21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419770">21</a>, <a href="#r23" rid="r23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419777">23</a>). Ethical approval was obtained from the Dublin Federated Hospitals Research Ethics Committee, which is affiliated with the Trinity College, and reviewed by the Office of Human Subjects Research at the National Institutes of Health. Written informed consent was obtained from participants before recruitment. Details of the sample collection, genotyping, and data-cleaning process are described in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="__sec13" class="sec"><p></p><h3 id="__sec13title" class="inline">VWF Antigen Level. </h3><p id="__p49" class="p p-first-last">VWF antigen levels were measured in both the GABC and TSS cohorts using a custom AlphaLISA (Perkin-Elmer) assay. For further details on this assay, refer to <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div><div id="__sec14" class="sec"><p></p><h3 id="__sec14title" class="inline">Phenotype Data Processing. </h3><p id="__p50" class="p p-first-last">Consistent with previous studies (<a href="#r12" rid="r12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419731">12</a>, <a href="#r20" rid="r20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419764">20</a>), the raw VWF antigen level distribution in our cohorts was not normal. Therefore, the antigen-level distribution was normalized by log transformation. The associations of smoking with raw VWF level distribution were analyzed with Kolmogorov–Smirnov tests. The effects of age, sex, and population structure on the VWF antigen levels were determined by separate single-factor linear regression analysis. <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a> provides further details of phenotype data processing.</p></div><div id="__sec15" class="sec"><p></p><h3 id="__sec15title" class="inline">Heritability of VWF Antigen Levels. </h3><p id="__p51" class="p p-first-last">For the combined set of GABC and TSS, the sib-pair intraclass correlation coefficient (ICC) was calculated using the log-transformed, age-, sex-, and PC3-adjusted values in GABC, and age- and sex-adjusted values in TSS. As there were sibships with more than two members, we randomly sampled two sibs from each family, calculated ICC, and repeated the process 500 times to obtain the average ICC. The upper-bound of the narrow sense heritability (<em>h</em><sup>2</sup>) was estimated by multiplying the average ICC by 2. In addition to the pedigree-based estimate described above, independent estimates were also calculated using the genotype and phenotype data in MERLIN-REGRESS and GCTA using all individuals (<a href="#r25" rid="r25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419749">25</a>, <a href="#r26" rid="r26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_294419723">26</a>).</p></div><div id="__sec16" class="sec sec-last"><p></p><h3 id="__sec16title" class="inline">Computational Analyses. </h3><p id="__p52" class="p p-first-last">A detailed description of the computational methods used to assess population substructure, association analysis, meta-analysis, linkage analysis, variance-explained calculations, and haplotype-based association analysis are provided in <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental/sapp.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em></a>.</p></div></div><div id="__sec17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec17title" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supporting Information:</strong>

<div class="sup-box half_rhythm" id="idm140703051053200"><a href="/pmc/articles/PMC3545809/bin/supp_110_2_588__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140703045709712" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140703045709712title" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p id="__p53">We thank the participants in the Genes and Blood-Clotting Study and the Trinity Student Study; and the high-throughput screening laboratory in the University of Michigan’s Center for Chemical Genomics for their assistance with the performance of high-throughput von Willebrand factor assays. This study was supported by the Intramural Research Programs of the National Institutes of Health, the Eunice Kennedy Shriver National Institute of Child Health and Human Development, and the National Human Genome Research Institute; and by National Institute of Health Grants 5K12HD028820-15 (to K.C.D.) and 2RO1HL039693 (to D.G.). D.G. is a Howard Hughes Investigator.</p></div></div><div id="idm140703045708304" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140703045708304title" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140703045708176"> </p><p id="__p54" class="p p-first-last">The authors declare no conflict of interest.</p><p></p><p class="fn sec" id="idm140703048114944"></p><p id="__p55" class="p p-first-last">Data deposition: The sequence reported in this paper has been deposited in the database of Genotypes and Phenotypes, <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ncbi.nlm.nih.gov/gap</a> (dbGaP study accession <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000304.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">phs000304.v1.p1</a>).</p><p></p><p class="fn sec" id="idm140703048111728"></p><p id="__p56" class="p p-first-last">This article contains supporting information online at <a href="http://www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.pnas.org/lookup/suppl/doi:10.1073/pnas.1219885110/-/DCSupplemental</a>.</p><p></p></div></div><div id="idm140703042837216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140703042837216title" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="element-citation">Sadler JE. von Willebrand factor: Two sides of a coin. <span><span class="ref-journal">J Thromb Haemost. </span>2005;<span class="ref-vol">3</span>(8):1702–1709.</span> [<a href="/pubmed/16102036" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=von+Willebrand+factor:+Two+sides+of+a+coin&amp;author=JE+Sadler&amp;volume=3&amp;issue=8&amp;publication_year=2005&amp;pages=1702-1709&amp;pmid=16102036&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="element-citation">Nichols WL, et al.  von Willebrand disease (VWD): Evidence-based diagnosis and management guidelines, the National Heart, Lung, and Blood Institute (NHLBI) Expert Panel report (USA) <span><span class="ref-journal">Haemophilia. </span>2008;<span class="ref-vol">14</span>(2):171–232.</span> [<a href="/pubmed/18315614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haemophilia&amp;title=von+Willebrand+disease+(VWD):+Evidence-based+diagnosis+and+management+guidelines,+the+National+Heart,+Lung,+and+Blood+Institute+(NHLBI)+Expert+Panel+report+(USA)&amp;author=WL+Nichols&amp;volume=14&amp;issue=2&amp;publication_year=2008&amp;pages=171-232&amp;pmid=18315614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="element-citation">Koster T, Blann AD, Briët E, Vandenbroucke JP, Rosendaal FR. Role of clotting factor VIII in effect of von Willebrand factor on occurrence of deep-vein thrombosis. <span><span class="ref-journal">Lancet. </span>1995;<span class="ref-vol">345</span>(8943):152–155.</span> [<a href="/pubmed/7823669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Role+of+clotting+factor+VIII+in+effect+of+von+Willebrand+factor+on+occurrence+of+deep-vein+thrombosis&amp;author=T+Koster&amp;author=AD+Blann&amp;author=E+Briët&amp;author=JP+Vandenbroucke&amp;author=FR+Rosendaal&amp;volume=345&amp;issue=8943&amp;publication_year=1995&amp;pages=152-155&amp;pmid=7823669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="element-citation">Morange PE, et al.  PRIME Study Group  Endothelial cell markers and the risk of coronary heart disease: The Prospective Epidemiological Study of Myocardial Infarction (PRIME) study. <span><span class="ref-journal">Circulation. </span>2004;<span class="ref-vol">109</span>(11):1343–1348.</span> [<a href="/pubmed/15023872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Endothelial+cell+markers+and+the+risk+of+coronary+heart+disease:+The+Prospective+Epidemiological+Study+of+Myocardial+Infarction+(PRIME)+study&amp;author=PE+Morange&amp;volume=109&amp;issue=11&amp;publication_year=2004&amp;pages=1343-1348&amp;pmid=15023872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="element-citation">Wieberdink RG, et al.  High von Willebrand factor levels increase the risk of stroke: The Rotterdam study. <span><span class="ref-journal">Stroke. </span>2010;<span class="ref-vol">41</span>(10):2151–2156.</span> [<a href="/pubmed/20798373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=High+von+Willebrand+factor+levels+increase+the+risk+of+stroke:+The+Rotterdam+study&amp;author=RG+Wieberdink&amp;volume=41&amp;issue=10&amp;publication_year=2010&amp;pages=2151-2156&amp;pmid=20798373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="element-citation">van Schie MC, et al.  Active von Willebrand factor and the risk of stroke. <span><span class="ref-journal">Atherosclerosis. </span>2010;<span class="ref-vol">208</span>(2):322–323.</span> [<a href="/pubmed/19732894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Active+von+Willebrand+factor+and+the+risk+of+stroke&amp;author=MC+van+Schie&amp;volume=208&amp;issue=2&amp;publication_year=2010&amp;pages=322-323&amp;pmid=19732894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="element-citation">Roldán V, et al.  Plasma von Willebrand factor levels are an independent risk factor for adverse events including mortality and major bleeding in anticoagulated atrial fibrillation patients. <span><span class="ref-journal">J Am Coll Cardiol. </span>2011;<span class="ref-vol">57</span>(25):2496–2504.</span> [<a href="/pubmed/21497043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Plasma+von+Willebrand+factor+levels+are+an+independent+risk+factor+for+adverse+events+including+mortality+and+major+bleeding+in+anticoagulated+atrial+fibrillation+patients&amp;author=V+Roldán&amp;volume=57&amp;issue=25&amp;publication_year=2011&amp;pages=2496-2504&amp;pmid=21497043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="element-citation">de Lange M, et al.  Genetic influences on fibrinogen, tissue plasminogen activator-antigen and von Willebrand factor in males and females. <span><span class="ref-journal">Thromb Haemost. </span>2006;<span class="ref-vol">95</span>(3):414–419.</span> [<a href="/pubmed/16525567" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=Genetic+influences+on+fibrinogen,+tissue+plasminogen+activator-antigen+and+von+Willebrand+factor+in+males+and+females&amp;author=M+de+Lange&amp;volume=95&amp;issue=3&amp;publication_year=2006&amp;pages=414-419&amp;pmid=16525567&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="element-citation">Jilma B, et al.  High dose dexamethasone increases circulating P-selectin and von Willebrand factor levels in healthy men. <span><span class="ref-journal">Thromb Haemost. </span>2005;<span class="ref-vol">94</span>(4):797–801.</span> [<a href="/pubmed/16270633" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=High+dose+dexamethasone+increases+circulating+P-selectin+and+von+Willebrand+factor+levels+in+healthy+men&amp;author=B+Jilma&amp;volume=94&amp;issue=4&amp;publication_year=2005&amp;pages=797-801&amp;pmid=16270633&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="element-citation">Lip GY, Foster W, Blann AD. Plasma von Willebrand factor levels and surrogates of atherosclerosis. <span><span class="ref-journal">J Thromb Haemost. </span>2005;<span class="ref-vol">3</span>(4):659–661.</span> [<a href="/pubmed/15842350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Plasma+von+Willebrand+factor+levels+and+surrogates+of+atherosclerosis&amp;author=GY+Lip&amp;author=W+Foster&amp;author=AD+Blann&amp;volume=3&amp;issue=4&amp;publication_year=2005&amp;pages=659-661&amp;pmid=15842350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="element-citation">Goligorsky MS, Patschan D, Kuo MC. Weibel-Palade bodies—Sentinels of acute stress. <span><span class="ref-journal">Nat Rev Nephrol. </span>2009;<span class="ref-vol">5</span>(7):423–426.</span> [<a href="/pubmed/19556996" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Nephrol&amp;title=Weibel-Palade+bodies—Sentinels+of+acute+stress&amp;author=MS+Goligorsky&amp;author=D+Patschan&amp;author=MC+Kuo&amp;volume=5&amp;issue=7&amp;publication_year=2009&amp;pages=423-426&amp;pmid=19556996&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="element-citation">Orstavik KH, et al.  Factor VIII and factor IX in a twin population. Evidence for a major effect of ABO locus on factor VIII level. <span><span class="ref-journal">Am J Hum Genet. </span>1985;<span class="ref-vol">37</span>(1):89–101.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684532/">PMC free article</a>]</span> [<a href="/pubmed/3919575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Factor+VIII+and+factor+IX+in+a+twin+population.+Evidence+for+a+major+effect+of+ABO+locus+on+factor+VIII+level&amp;author=KH+Orstavik&amp;volume=37&amp;issue=1&amp;publication_year=1985&amp;pages=89-101&amp;pmid=3919575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="element-citation">de Lange M, Snieder H, Ariëns RA, Spector TD, Grant PJ. The genetics of haemostasis: A twin study. <span><span class="ref-journal">Lancet. </span>2001;<span class="ref-vol">357</span>(9250):101–105.</span> [<a href="/pubmed/11197396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=The+genetics+of+haemostasis:+A+twin+study&amp;author=M+de+Lange&amp;author=H+Snieder&amp;author=RA+Ariëns&amp;author=TD+Spector&amp;author=PJ+Grant&amp;volume=357&amp;issue=9250&amp;publication_year=2001&amp;pages=101-105&amp;pmid=11197396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="element-citation">Bladbjerg EM, et al.  Genetic influence on thrombotic risk markers in the elderly–A Danish twin study. <span><span class="ref-journal">J Thromb Haemost. </span>2006;<span class="ref-vol">4</span>(3):599–607.</span> [<a href="/pubmed/16371117" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Genetic+influence+on+thrombotic+risk+markers+in+the+elderly–A+Danish+twin+study&amp;author=EM+Bladbjerg&amp;volume=4&amp;issue=3&amp;publication_year=2006&amp;pages=599-607&amp;pmid=16371117&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="element-citation">Souto JC, et al.  Genome-wide linkage analysis of von Willebrand factor plasma levels: Results from the GAIT project. <span><span class="ref-journal">Thromb Haemost. </span>2003;<span class="ref-vol">89</span>(3):468–474.</span> [<a href="/pubmed/12624629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Haemost&amp;title=Genome-wide+linkage+analysis+of+von+Willebrand+factor+plasma+levels:+Results+from+the+GAIT+project&amp;author=JC+Souto&amp;volume=89&amp;issue=3&amp;publication_year=2003&amp;pages=468-474&amp;pmid=12624629&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="element-citation">Gallinaro L, et al.  A shorter von Willebrand factor survival in O blood group subjects explains how ABO determinants influence plasma von Willebrand factor. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">111</span>(7):3540–3545.</span> [<a href="/pubmed/18245665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+shorter+von+Willebrand+factor+survival+in+O+blood+group+subjects+explains+how+ABO+determinants+influence+plasma+von+Willebrand+factor&amp;author=L+Gallinaro&amp;volume=111&amp;issue=7&amp;publication_year=2008&amp;pages=3540-3545&amp;pmid=18245665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="element-citation">Canis K, et al.  The plasma von Willebrand factor O-glycome comprises a surprising variety of structures including ABH antigens and disialosyl motifs. <span><span class="ref-journal">J Thromb Haemost. </span>2010;<span class="ref-vol">8</span>(1):137–145.</span> [<a href="/pubmed/19874459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=The+plasma+von+Willebrand+factor+O-glycome+comprises+a+surprising+variety+of+structures+including+ABH+antigens+and+disialosyl+motifs&amp;author=K+Canis&amp;volume=8&amp;issue=1&amp;publication_year=2010&amp;pages=137-145&amp;pmid=19874459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="element-citation">Nossent AY, Vos HL, Rosendaal FR, Bertina RM, Eikenboom JC. Aquaporin 2 gene variations, risk of venous thrombosis and plasma levels of von Willebrand factor and factor VIII. <span><span class="ref-journal">Haematologica. </span>2008;<span class="ref-vol">93</span>(6):959–960.</span> [<a href="/pubmed/18515885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Haematologica&amp;title=Aquaporin+2+gene+variations,+risk+of+venous+thrombosis+and+plasma+levels+of+von+Willebrand+factor+and+factor+VIII&amp;author=AY+Nossent&amp;author=HL+Vos&amp;author=FR+Rosendaal&amp;author=RM+Bertina&amp;author=JC+Eikenboom&amp;volume=93&amp;issue=6&amp;publication_year=2008&amp;pages=959-960&amp;pmid=18515885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="element-citation">Yamamoto F, McNeill PD, Hakomori S. Human histo-blood group A2 transferase coded by A2 allele, one of the A subtypes, is characterized by a single base deletion in the coding sequence, which results in an additional domain at the carboxyl terminal. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>1992;<span class="ref-vol">187</span>(1):366–374.</span> [<a href="/pubmed/1520322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Human+histo-blood+group+A2+transferase+coded+by+A2+allele,+one+of+the+A+subtypes,+is+characterized+by+a+single+base+deletion+in+the+coding+sequence,+which+results+in+an+additional+domain+at+the+carboxyl+terminal&amp;author=F+Yamamoto&amp;author=PD+McNeill&amp;author=S+Hakomori&amp;volume=187&amp;issue=1&amp;publication_year=1992&amp;pages=366-374&amp;pmid=1520322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="element-citation">Smith NL, et al.  Wellcome Trust Case Control Consortium  Novel associations of multiple genetic loci with plasma levels of factor VII, factor VIII, and von Willebrand factor: The CHARGE (Cohorts for Heart and Aging Research in Genome Epidemiology) Consortium. <span><span class="ref-journal">Circulation. </span>2010;<span class="ref-vol">121</span>(12):1382–1392.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2861278/">PMC free article</a>]</span> [<a href="/pubmed/20231535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Novel+associations+of+multiple+genetic+loci+with+plasma+levels+of+factor+VII,+factor+VIII,+and+von+Willebrand+factor:+The+CHARGE+(Cohorts+for+Heart+and+Aging+Research+in+Genome+Epidemiology)+Consortium&amp;author=NL+Smith&amp;volume=121&amp;issue=12&amp;publication_year=2010&amp;pages=1382-1392&amp;pmid=20231535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="element-citation">Stone N, et al.  Bioinformatic and genetic association analysis of microRNA target sites in one-carbon metabolism genes. <span><span class="ref-journal">PLoS ONE. </span>2011;<span class="ref-vol">6</span>(7):e21851.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3134459/">PMC free article</a>]</span> [<a href="/pubmed/21765920" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Bioinformatic+and+genetic+association+analysis+of+microRNA+target+sites+in+one-carbon+metabolism+genes&amp;author=N+Stone&amp;volume=6&amp;issue=7&amp;publication_year=2011&amp;pages=e21851&amp;pmid=21765920&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="element-citation">Desch K, et al.  Analysis of informed consent document utilization in a minimal-risk genetic study. <span><span class="ref-journal">Ann Intern Med. </span>2011;<span class="ref-vol">155</span>(5):316–322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3540806/">PMC free article</a>]</span> [<a href="/pubmed/21893624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Intern+Med&amp;title=Analysis+of+informed+consent+document+utilization+in+a+minimal-risk+genetic+study&amp;author=K+Desch&amp;volume=155&amp;issue=5&amp;publication_year=2011&amp;pages=316-322&amp;pmid=21893624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="element-citation">Mills JL, et al.  Do high blood folate concentrations exacerbate metabolic abnormalities in people with low vitamin B-12 status? <span><span class="ref-journal">Am J Clin Nutr. </span>2011;<span class="ref-vol">94</span>(2):495–500.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142724/">PMC free article</a>]</span> [<a href="/pubmed/21653798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Clin+Nutr&amp;title=Do+high+blood+folate+concentrations+exacerbate+metabolic+abnormalities+in+people+with+low+vitamin+B-12+status?&amp;author=JL+Mills&amp;volume=94&amp;issue=2&amp;publication_year=2011&amp;pages=495-500&amp;pmid=21653798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="element-citation">University of Michigan, Office of the Registrar  Ethnicity of University of Michigan students. 2010.   <a href="http://www.umich.edu/~regoff/enrollment/ethnicity.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.umich.edu/~regoff/enrollment/ethnicity.php</a>. Accessed 11-14-2012.</span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="element-citation">Abecasis GR, Cherny SS, Cookson WO, Cardon LR. Merlin—Rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">30</span>(1):97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Merlin—Rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=GR+Abecasis&amp;author=SS+Cherny&amp;author=WO+Cookson&amp;author=LR+Cardon&amp;volume=30&amp;issue=1&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: A tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>(1):76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+A+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="element-citation">Barbalic M, et al.  Large-scale genomic studies reveal central role of ABO in sP-selectin and sICAM-1 levels. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>(9):1863–1872.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850624/">PMC free article</a>]</span> [<a href="/pubmed/20167578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Large-scale+genomic+studies+reveal+central+role+of+ABO+in+sP-selectin+and+sICAM-1+levels&amp;author=M+Barbalic&amp;volume=19&amp;issue=9&amp;publication_year=2010&amp;pages=1863-1872&amp;pmid=20167578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="element-citation">Folsom AR, Wu KK, Rosamond WD, Sharrett AR, Chambless LE. Prospective study of hemostatic factors and incidence of coronary heart disease: The Atherosclerosis Risk in Communities (ARIC) Study. <span><span class="ref-journal">Circulation. </span>1997;<span class="ref-vol">96</span>(4):1102–1108.</span> [<a href="/pubmed/9286936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Prospective+study+of+hemostatic+factors+and+incidence+of+coronary+heart+disease:+The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study&amp;author=AR+Folsom&amp;author=KK+Wu&amp;author=WD+Rosamond&amp;author=AR+Sharrett&amp;author=LE+Chambless&amp;volume=96&amp;issue=4&amp;publication_year=1997&amp;pages=1102-1108&amp;pmid=9286936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="element-citation">Feng D, et al.  Factor VII gene polymorphism, factor VII levels, and prevalent cardiovascular disease: The Framingham Heart Study. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2000;<span class="ref-vol">20</span>(2):593–600.</span> [<a href="/pubmed/10669660" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Factor+VII+gene+polymorphism,+factor+VII+levels,+and+prevalent+cardiovascular+disease:+The+Framingham+Heart+Study&amp;author=D+Feng&amp;volume=20&amp;issue=2&amp;publication_year=2000&amp;pages=593-600&amp;pmid=10669660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="element-citation">Chester MA, Olsson ML. The ABO blood group gene: A locus of considerable genetic diversity. <span><span class="ref-journal">Transfus Med Rev. </span>2001;<span class="ref-vol">15</span>(3):177–200.</span> [<a href="/pubmed/11471121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfus+Med+Rev&amp;title=The+ABO+blood+group+gene:+A+locus+of+considerable+genetic+diversity&amp;author=MA+Chester&amp;author=ML+Olsson&amp;volume=15&amp;issue=3&amp;publication_year=2001&amp;pages=177-200&amp;pmid=11471121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="element-citation">Ahlen MT, et al.  The development of severe neonatal alloimmune thrombocytopenia due to anti-HPA-1a antibodies is correlated to maternal ABO genotypes. <span><span class="ref-journal">Clin Dev Immunol. </span>2012;<span class="ref-vol">2012</span>:156867.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3216343/">PMC free article</a>]</span> [<a href="/pubmed/22110529" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Dev+Immunol&amp;title=The+development+of+severe+neonatal+alloimmune+thrombocytopenia+due+to+anti-HPA-1a+antibodies+is+correlated+to+maternal+ABO+genotypes&amp;author=MT+Ahlen&amp;volume=2012&amp;publication_year=2012&amp;pages=156867&amp;pmid=22110529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="element-citation">Boyles AL, et al.  Linkage disequilibrium inflates type I error rates in multipoint linkage analysis when parental genotypes are missing. <span><span class="ref-journal">Hum Hered. </span>2005;<span class="ref-vol">59</span>(4):220–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1224705/">PMC free article</a>]</span> [<a href="/pubmed/16093727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Linkage+disequilibrium+inflates+type+I+error+rates+in+multipoint+linkage+analysis+when+parental+genotypes+are+missing&amp;author=AL+Boyles&amp;volume=59&amp;issue=4&amp;publication_year=2005&amp;pages=220-227&amp;pmid=16093727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="element-citation">Wiltshire S, Cardon LR, McCarthy MI. Evaluating the results of genomewide linkage scans of complex traits by locus counting. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>(5):1175–1182.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC385093/">PMC free article</a>]</span> [<a href="/pubmed/12355401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Evaluating+the+results+of+genomewide+linkage+scans+of+complex+traits+by+locus+counting&amp;author=S+Wiltshire&amp;author=LR+Cardon&amp;author=MI+McCarthy&amp;volume=71&amp;issue=5&amp;publication_year=2002&amp;pages=1175-1182&amp;pmid=12355401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="element-citation">Antoni G, et al.  Combined analysis of three genome-wide association studies on vWF and FVIII plasma levels. <span><span class="ref-journal">BMC Med Genet. </span>2011;<span class="ref-vol">12</span>:102.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3163514/">PMC free article</a>]</span> [<a href="/pubmed/21810271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Combined+analysis+of+three+genome-wide+association+studies+on+vWF+and+FVIII+plasma+levels&amp;author=G+Antoni&amp;volume=12&amp;publication_year=2011&amp;pages=102&amp;pmid=21810271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="element-citation">Zabaneh D, et al.  Genetic variants associated with Von Willebrand factor levels in healthy men and women identified using the HumanCVD BeadChip. <span><span class="ref-journal">Ann Hum Genet. </span>2011;<span class="ref-vol">75</span>(4):456–467.</span> [<a href="/pubmed/21534939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Genetic+variants+associated+with+Von+Willebrand+factor+levels+in+healthy+men+and+women+identified+using+the+HumanCVD+BeadChip&amp;author=D+Zabaneh&amp;volume=75&amp;issue=4&amp;publication_year=2011&amp;pages=456-467&amp;pmid=21534939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r36">36. <span class="element-citation">Liu J, et al.  Relationship between biomarkers of cigarette smoke exposure and biomarkers of inflammation, oxidative stress, and platelet activation in adult cigarette smokers. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2011;<span class="ref-vol">20</span>(8):1760–1769.</span> [<a href="/pubmed/21708936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Relationship+between+biomarkers+of+cigarette+smoke+exposure+and+biomarkers+of+inflammation,+oxidative+stress,+and+platelet+activation+in+adult+cigarette+smokers&amp;author=J+Liu&amp;volume=20&amp;issue=8&amp;publication_year=2011&amp;pages=1760-1769&amp;pmid=21708936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r37">37. <span class="element-citation">Jefferis BJ, et al.  Secondhand smoke (SHS) exposure is associated with circulating markers of inflammation and endothelial function in adult men and women. <span><span class="ref-journal">Atherosclerosis. </span>2010;<span class="ref-vol">208</span>(2):550–556.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2822956/">PMC free article</a>]</span> [<a href="/pubmed/19700161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Secondhand+smoke+(SHS)+exposure+is+associated+with+circulating+markers+of+inflammation+and+endothelial+function+in+adult+men+and+women&amp;author=BJ+Jefferis&amp;volume=208&amp;issue=2&amp;publication_year=2010&amp;pages=550-556&amp;pmid=19700161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r38">38. <span class="element-citation">Campos M, et al.  Genetic determinants of plasma von Willebrand factor antigen levels: A target gene SNP and haplotype analysis of ARIC cohort. <span><span class="ref-journal">Blood. </span>2011;<span class="ref-vol">117</span>(19):5224–5230.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3109544/">PMC free article</a>]</span> [<a href="/pubmed/21343614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+determinants+of+plasma+von+Willebrand+factor+antigen+levels:+A+target+gene+SNP+and+haplotype+analysis+of+ARIC+cohort&amp;author=M+Campos&amp;volume=117&amp;issue=19&amp;publication_year=2011&amp;pages=5224-5230&amp;pmid=21343614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r39">39. <span class="element-citation">Campos M, et al.  Influence of single nucleotide polymorphisms in factor VIII and von Willebrand factor genes on plasma factor VIII activity: The ARIC Study. <span><span class="ref-journal">Blood. </span>2012;<span class="ref-vol">119</span>(8):1929–1934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3293647/">PMC free article</a>]</span> [<a href="/pubmed/22219226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Influence+of+single+nucleotide+polymorphisms+in+factor+VIII+and+von+Willebrand+factor+genes+on+plasma+factor+VIII+activity:+The+ARIC+Study&amp;author=M+Campos&amp;volume=119&amp;issue=8&amp;publication_year=2012&amp;pages=1929-1934&amp;pmid=22219226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r40">40. <span class="element-citation">Sadler JE. von Willebrand factor assembly and secretion. <span><span class="ref-journal">J Thromb Haemost. </span>2009;<span class="ref-vol">7</span>(Suppl 1):24–27.</span> [<a href="/pubmed/19630761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=von+Willebrand+factor+assembly+and+secretion&amp;author=JE+Sadler&amp;volume=7&amp;issue=Suppl+1&amp;publication_year=2009&amp;pages=24-27&amp;pmid=19630761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r41">41. <span class="element-citation">Orstavik KH, Kornstad L, Reisner H, Berg K. Possible effect of secretor locus on plasma concentration of factor VIII and von Willebrand factor. <span><span class="ref-journal">Blood. </span>1989;<span class="ref-vol">73</span>(4):990–993.</span> [<a href="/pubmed/2493271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Possible+effect+of+secretor+locus+on+plasma+concentration+of+factor+VIII+and+von+Willebrand+factor&amp;author=KH+Orstavik&amp;author=L+Kornstad&amp;author=H+Reisner&amp;author=K+Berg&amp;volume=73&amp;issue=4&amp;publication_year=1989&amp;pages=990-993&amp;pmid=2493271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r42">42. <span class="element-citation">O’Donnell J, Boulton FE, Manning RA, Laffan MA. Genotype at the secretor blood group locus is a determinant of plasma von Willebrand factor level. <span><span class="ref-journal">Br J Haematol. </span>2002;<span class="ref-vol">116</span>(2):350–356.</span> [<a href="/pubmed/11841438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Genotype+at+the+secretor+blood+group+locus+is+a+determinant+of+plasma+von+Willebrand+factor+level&amp;author=J+O’Donnell&amp;author=FE+Boulton&amp;author=RA+Manning&amp;author=MA+Laffan&amp;volume=116&amp;issue=2&amp;publication_year=2002&amp;pages=350-356&amp;pmid=11841438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r43">43. <span class="element-citation">Morange PE, et al.  Biological and genetic factors influencing plasma factor VIII levels in a healthy family population: Results from the Stanislas cohort. <span><span class="ref-journal">Br J Haematol. </span>2005;<span class="ref-vol">128</span>(1):91–99.</span> [<a href="/pubmed/15606554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Biological+and+genetic+factors+influencing+plasma+factor+VIII+levels+in+a+healthy+family+population:+Results+from+the+Stanislas+cohort&amp;author=PE+Morange&amp;volume=128&amp;issue=1&amp;publication_year=2005&amp;pages=91-99&amp;pmid=15606554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r44">44. <span class="element-citation">Kario K, et al.  Endothelial cell damage and angiotensin-converting enzyme insertion/deletion genotype in elderly hypertensive patients. <span><span class="ref-journal">J Am Coll Cardiol. </span>1998;<span class="ref-vol">32</span>(2):444–450.</span> [<a href="/pubmed/9708474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Endothelial+cell+damage+and+angiotensin-converting+enzyme+insertion/deletion+genotype+in+elderly+hypertensive+patients&amp;author=K+Kario&amp;volume=32&amp;issue=2&amp;publication_year=1998&amp;pages=444-450&amp;pmid=9708474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r45">45. <span class="element-citation">Nossent AY, et al.  Functional variation in the arginine vasopressin 2 receptor as a modifier of human plasma von Willebrand factor levels. <span><span class="ref-journal">J Thromb Haemost. </span>2010;<span class="ref-vol">8</span>(7):1547–1554.</span> [<a href="/pubmed/20403097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Thromb+Haemost&amp;title=Functional+variation+in+the+arginine+vasopressin+2+receptor+as+a+modifier+of+human+plasma+von+Willebrand+factor+levels&amp;author=AY+Nossent&amp;volume=8&amp;issue=7&amp;publication_year=2010&amp;pages=1547-1554&amp;pmid=20403097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r46">46. <span class="element-citation">Frazer KA, Murray SS, Schork NJ, Topol EJ. Human genetic variation and its contribution to complex traits. <span><span class="ref-journal">Nat Rev Genet. </span>2009;<span class="ref-vol">10</span>(4):241–251.</span> [<a href="/pubmed/19293820" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Human+genetic+variation+and+its+contribution+to+complex+traits&amp;author=KA+Frazer&amp;author=SS+Murray&amp;author=NJ+Schork&amp;author=EJ+Topol&amp;volume=10&amp;issue=4&amp;publication_year=2009&amp;pages=241-251&amp;pmid=19293820&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r47">47. <span class="element-citation">Manolio TA, et al.  Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>(7265):747–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;volume=461&amp;issue=7265&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r48">48. <span class="element-citation">Wright FA, et al.  Genome-wide association and linkage identify modifier loci of lung disease severity in cystic fibrosis at 11p13 and 20q13.2. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>(6):539–546.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3296486/">PMC free article</a>]</span> [<a href="/pubmed/21602797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+and+linkage+identify+modifier+loci+of+lung+disease+severity+in+cystic+fibrosis+at+11p13+and+20q13.2&amp;author=FA+Wright&amp;volume=43&amp;issue=6&amp;publication_year=2011&amp;pages=539-546&amp;pmid=21602797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r49">49. <span class="element-citation">Hinrichs AL, Suarez BK. Incorporating linkage information into a common disease/rare variant framework. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>(Suppl 1):S74–S79.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4558895/">PMC free article</a>]</span> [<a href="/pubmed/22128063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Incorporating+linkage+information+into+a+common+disease/rare+variant+framework&amp;author=AL+Hinrichs&amp;author=BK+Suarez&amp;volume=35&amp;issue=Suppl+1&amp;publication_year=2011&amp;pages=S74-S79&amp;pmid=22128063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3545809&amp;issue-id=218256&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140703087463280title">Abstract</a></li><li><a href="#s1title">Results</a></li><li><a href="#s2title">Discussion</a></li><li><a href="#s3title">Materials and Methods</a></li><li><a href="#__sec17title">Supplementary Material</a></li><li><a href="#idm140703045709712title">Acknowledgments</a></li><li><a href="#idm140703045708304title">Footnotes</a></li><li><a href="#idm140703042837216title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3545809/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3545809/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3545809/pdf/pnas.201219885.pdf">PDF (662K)</a></li> | <li><a href="#" data-citationid="PMC3545809" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3545809%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3545809%2F&amp;text=Linkage%20analysis%20identifies%20a%20locus%20for%20plasma%20von%20Willebrand%20factor%20undetected%20by%20genome-wide%20association" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3545809%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23267103" data-aiid="3545809" data-aid="3545809" data-iid="218256" data-domainid="2" data-domain="pnas" data-accid="PMC3545809" data-md5="a3a18c9e75b7553d2756d170f79278e1"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:07:05-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5B57DD5513310000000003B6027B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3545809%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=pnas&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3545809/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>