<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3832838/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Human metabolic individuality in biomedical and pharmaceutical research</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832838/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature"><meta name="citation_title" content="Human metabolic individuality in biomedical and pharmaceutical research"><meta name="citation_authors" content="Karsten Suhre, So-Youn Shin, Ann-Kristin Petersen, Robert P. Mohney, David Meredith, Brigitte Wägele, Elisabeth Altmaier, CARDIoGRAM, Panos Deloukas, Jeanette Erdmann, Elin Grundberg, Christopher J. Hammond, Martin Hrabé de Angelis, Gabi Kastenmüller, Anna Köttgen, Florian Kronenberg, Massimo Mangino, Christa Meisinger, Thomas Meitinger, Hans-Werner Mewes, Michael V. Milburn, Cornelia Prehn, Johannes Raffler, Janina S. Ried, Werner Römisch-Margl, Nilesh J. Samani, Kerrin S. Small, H.-Erich Wichmann, Guangju Zhai, Thomas Illig, Tim D. Spector, Jerzy Adamski, Nicole Soranzo, Christian Gieger"><meta name="citation_issue" content="7362"><meta name="citation_volume" content="477"><meta name="citation_doi" content="10.1038/nature10354"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3832838/?report=abstract"><meta name="citation_pmid" content="21886157"><meta name="DC.Title" content="Human metabolic individuality in biomedical and pharmaceutical research"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Karsten Suhre"><meta name="DC.Contributor" content="So-Youn Shin"><meta name="DC.Contributor" content="Ann-Kristin Petersen"><meta name="DC.Contributor" content="Robert P. Mohney"><meta name="DC.Contributor" content="David Meredith"><meta name="DC.Contributor" content="Brigitte Wägele"><meta name="DC.Contributor" content="Elisabeth Altmaier"><meta name="DC.Contributor" content="CARDIoGRAM"><meta name="DC.Contributor" content="Panos Deloukas"><meta name="DC.Contributor" content="Jeanette Erdmann"><meta name="DC.Contributor" content="Elin Grundberg"><meta name="DC.Contributor" content="Christopher J. Hammond"><meta name="DC.Contributor" content="Martin Hrabé de Angelis"><meta name="DC.Contributor" content="Gabi Kastenmüller"><meta name="DC.Contributor" content="Anna Köttgen"><meta name="DC.Contributor" content="Florian Kronenberg"><meta name="DC.Contributor" content="Massimo Mangino"><meta name="DC.Contributor" content="Christa Meisinger"><meta name="DC.Contributor" content="Thomas Meitinger"><meta name="DC.Contributor" content="Hans-Werner Mewes"><meta name="DC.Contributor" content="Michael V. Milburn"><meta name="DC.Contributor" content="Cornelia Prehn"><meta name="DC.Contributor" content="Johannes Raffler"><meta name="DC.Contributor" content="Janina S. Ried"><meta name="DC.Contributor" content="Werner Römisch-Margl"><meta name="DC.Contributor" content="Nilesh J. Samani"><meta name="DC.Contributor" content="Kerrin S. Small"><meta name="DC.Contributor" content="H.-Erich Wichmann"><meta name="DC.Contributor" content="Guangju Zhai"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Tim D. Spector"><meta name="DC.Contributor" content="Jerzy Adamski"><meta name="DC.Contributor" content="Nicole Soranzo"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Identifier" content="10.1038/nature10354"><meta name="DC.Language" content="en"><meta property="og:title" content="Human metabolic individuality in biomedical and pharmaceutical research"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) have identified many risk loci for complex diseases, but effect sizes are typically small and information on the underlying biological processes is often lacking. Associations with metabolic traits as functional ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3832838/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3832838/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3832838/pdf/emss-54209.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BF518DD5285C100000000078F04D5.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832838%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3832838</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nature</a></span>. Author manuscript; available in PMC 2013 Nov 19.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21886157" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nature. 2011 Aug 31; 477(7362): 10.1038/nature10354. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Aug 31. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fnature10354" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/nature10354</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3832838</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS54209</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21886157">21886157</a></div></div></div></div><h1 class="content-title">Human metabolic individuality in biomedical and pharmaceutical research</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353670799792" co-class="co-affbox">Karsten Suhre</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>+</sup> <a href="/pubmed/?term=Shin%20SY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353719753728" co-class="co-affbox">So-Youn Shin</a>,<sup>#</sup><sup>4</sup> <a href="/pubmed/?term=Petersen%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353686309552" co-class="co-affbox">Ann-Kristin Petersen</a>,<sup>#</sup><sup>5</sup> <a href="/pubmed/?term=Mohney%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353686307328" co-class="co-affbox">Robert P. Mohney</a>,<sup>6</sup> <a href="/pubmed/?term=Meredith%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353689430928" co-class="co-affbox">David Meredith</a>,<sup>7</sup> <a href="/pubmed/?term=W%26%23x000e4%3Bgele%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353757581136" co-class="co-affbox">Brigitte Wägele</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=Altmaier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353759102960" co-class="co-affbox">Elisabeth Altmaier</a>,<sup>1</sup> CARDIoGRAM,<sup>9</sup> <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353759096672" co-class="co-affbox">Panos Deloukas</a>,<sup>4</sup> <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353759094688" co-class="co-affbox">Jeanette Erdmann</a>,<sup>10</sup> <a href="/pubmed/?term=Grundberg%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353673489232" co-class="co-affbox">Elin Grundberg</a>,<sup>4,</sup><sup>11</sup> <a href="/pubmed/?term=Hammond%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353758297456" co-class="co-affbox">Christopher J. Hammond</a>,<sup>11</sup> <a href="/pubmed/?term=de%20Angelis%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353688187808" co-class="co-affbox">Martin Hrabé de Angelis</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Kastenm%26%23x000fc%3Bller%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353689161600" co-class="co-affbox">Gabi Kastenmüller</a>,<sup>1</sup> <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353689159616" co-class="co-affbox">Anna Köttgen</a>,<sup>14</sup> <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353681555872" co-class="co-affbox">Florian Kronenberg</a>,<sup>15</sup> <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353681874352" co-class="co-affbox">Massimo Mangino</a>,<sup>11</sup> <a href="/pubmed/?term=Meisinger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353681872368" co-class="co-affbox">Christa Meisinger</a>,<sup>16</sup> <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353683375536" co-class="co-affbox">Thomas Meitinger</a>,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Mewes%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353719419712" co-class="co-affbox">Hans-Werner Mewes</a>,<sup>1,</sup><sup>8</sup> <a href="/pubmed/?term=Milburn%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353686670192" co-class="co-affbox">Michael V. Milburn</a>,<sup>6</sup> <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353686668208" co-class="co-affbox">Cornelia Prehn</a>,<sup>12</sup> <a href="/pubmed/?term=Raffler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353688766848" co-class="co-affbox">Johannes Raffler</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Ried%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353687342816" co-class="co-affbox">Janina S. Ried</a>,<sup>4</sup> <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353687733872" co-class="co-affbox">Werner Römisch-Margl</a>,<sup>1</sup> <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353687731888" co-class="co-affbox">Nilesh J. Samani</a>,<sup>19</sup> <a href="/pubmed/?term=Small%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353688114928" co-class="co-affbox">Kerrin S. Small</a>,<sup>11</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353682827424" co-class="co-affbox">H.-Erich Wichmann</a>,<sup>20,</sup><sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Zhai%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353682878464" co-class="co-affbox">Guangju Zhai</a>,<sup>11</sup> <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353721320896" co-class="co-affbox">Thomas Illig</a>,<sup>23</sup> <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353721318912" co-class="co-affbox">Tim D. Spector</a>,<sup>11</sup> <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353681340624" co-class="co-affbox">Jerzy Adamski</a>,<sup>12</sup> <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353682762784" co-class="co-affbox">Nicole Soranzo</a>,<sup>#</sup><sup>4,</sup><sup>+</sup> and  <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157" class="affpopup" co-rid="_co_idm140353683718816" co-class="co-affbox">Christian Gieger</a><sup>#</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140353670799792"><h3 class="no_margin">Karsten Suhre</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>2</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </p><p><sup>3</sup>Department of Physiology and Biophysics, Weill Cornell Medical College in Qatar, Education City - Qatar Foundation, Doha, Qatar </p><div>Find articles by <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Karsten Suhre</a></div></div><div id="_co_idm140353719753728"><h3 class="no_margin">So-Youn Shin</h3><p><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><div>Find articles by <a href="/pubmed/?term=Shin%20SY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">So-Youn Shin</a></div></div><div id="_co_idm140353686309552"><h3 class="no_margin">Ann-Kristin Petersen</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Petersen%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Ann-Kristin Petersen</a></div></div><div id="_co_idm140353686307328"><h3 class="no_margin">Robert P. Mohney</h3><p><sup>6</sup>Metabolon Inc., Durham, North Carolina, USA </p><div>Find articles by <a href="/pubmed/?term=Mohney%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Robert P. Mohney</a></div></div><div id="_co_idm140353689430928"><h3 class="no_margin">David Meredith</h3><p><sup>7</sup>School of Life Sciences, Oxford Brookes University, Headington, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Meredith%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">David Meredith</a></div></div><div id="_co_idm140353757581136"><h3 class="no_margin">Brigitte Wägele</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>8</sup>Department of Genome-oriented Bioinformatics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=W%26%23x000e4%3Bgele%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Brigitte Wägele</a></div></div><div id="_co_idm140353759102960"><h3 class="no_margin">Elisabeth Altmaier</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Altmaier%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Elisabeth Altmaier</a></div></div><div id="_co_idm140353759100976"><h3 class="no_margin">CARDIoGRAM</h3><p><sup>9</sup>The member list of the CARDIoGRAM consortium is provided as <strong>Supplemental Information</strong></p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">CARDIoGRAM</a></div></div><div id="_co_idm140353759096672"><h3 class="no_margin">Panos Deloukas</h3><p><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><div>Find articles by <a href="/pubmed/?term=Deloukas%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Panos Deloukas</a></div></div><div id="_co_idm140353759094688"><h3 class="no_margin">Jeanette Erdmann</h3><p><sup>10</sup>Universität zu Lübeck, Medizinische Klinik II, Lübeck, Germany </p><div>Find articles by <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Jeanette Erdmann</a></div></div><div id="_co_idm140353673489232"><h3 class="no_margin">Elin Grundberg</h3><p><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Grundberg%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Elin Grundberg</a></div></div><div id="_co_idm140353758297456"><h3 class="no_margin">Christopher J. Hammond</h3><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Hammond%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Christopher J. Hammond</a></div></div><div id="_co_idm140353688187808"><h3 class="no_margin">Martin Hrabé de Angelis</h3><p><sup>12</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>13</sup>Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=de%20Angelis%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Martin Hrabé de Angelis</a></div></div><div id="_co_idm140353689161600"><h3 class="no_margin">Gabi Kastenmüller</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Kastenm%26%23x000fc%3Bller%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Gabi Kastenmüller</a></div></div><div id="_co_idm140353689159616"><h3 class="no_margin">Anna Köttgen</h3><p><sup>14</sup>Renal Division, University Hospital Freiburg, Germany </p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f6%3Bttgen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Anna Köttgen</a></div></div><div id="_co_idm140353681555872"><h3 class="no_margin">Florian Kronenberg</h3><p><sup>15</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Innsbruck, Austria </p><div>Find articles by <a href="/pubmed/?term=Kronenberg%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Florian Kronenberg</a></div></div><div id="_co_idm140353681874352"><h3 class="no_margin">Massimo Mangino</h3><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Massimo Mangino</a></div></div><div id="_co_idm140353681872368"><h3 class="no_margin">Christa Meisinger</h3><p><sup>16</sup>Institute of Epidemiology II, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Meisinger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Christa Meisinger</a></div></div><div id="_co_idm140353683375536"><h3 class="no_margin">Thomas Meitinger</h3><p><sup>17</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>18</sup>Institute of Human Genetics, Klinikum rechts der Isar, Technische Universität München, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Meitinger%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Thomas Meitinger</a></div></div><div id="_co_idm140353719419712"><h3 class="no_margin">Hans-Werner Mewes</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>8</sup>Department of Genome-oriented Bioinformatics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </p><div>Find articles by <a href="/pubmed/?term=Mewes%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Hans-Werner Mewes</a></div></div><div id="_co_idm140353686670192"><h3 class="no_margin">Michael V. Milburn</h3><p><sup>6</sup>Metabolon Inc., Durham, North Carolina, USA </p><div>Find articles by <a href="/pubmed/?term=Milburn%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Michael V. Milburn</a></div></div><div id="_co_idm140353686668208"><h3 class="no_margin">Cornelia Prehn</h3><p><sup>12</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Cornelia Prehn</a></div></div><div id="_co_idm140353688766848"><h3 class="no_margin">Johannes Raffler</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>2</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </p><div>Find articles by <a href="/pubmed/?term=Raffler%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Johannes Raffler</a></div></div><div id="_co_idm140353687342816"><h3 class="no_margin">Janina S. Ried</h3><p><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><div>Find articles by <a href="/pubmed/?term=Ried%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Janina S. Ried</a></div></div><div id="_co_idm140353687733872"><h3 class="no_margin">Werner Römisch-Margl</h3><p><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Werner Römisch-Margl</a></div></div><div id="_co_idm140353687731888"><h3 class="no_margin">Nilesh J. Samani</h3><p><sup>19</sup>Department of Cardiovascular Sciences, University of Leicester, and Leicester NIHR Biomedical Research Unit in Cardiovascular Disease, Glenfield Hospital, Leicester, UK </p><div>Find articles by <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Nilesh J. Samani</a></div></div><div id="_co_idm140353688114928"><h3 class="no_margin">Kerrin S. Small</h3><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Small%20KS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Kerrin S. Small</a></div></div><div id="_co_idm140353682827424"><h3 class="no_margin">H.-Erich Wichmann</h3><p><sup>20</sup>Institute of Epidemiology I, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>21</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany </p><p><sup>22</sup>Klinikum Grosshadern, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">H.-Erich Wichmann</a></div></div><div id="_co_idm140353682878464"><h3 class="no_margin">Guangju Zhai</h3><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Zhai%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Guangju Zhai</a></div></div><div id="_co_idm140353721320896"><h3 class="no_margin">Thomas Illig</h3><p><sup>23</sup>Unit for Molecular Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Thomas Illig</a></div></div><div id="_co_idm140353721318912"><h3 class="no_margin">Tim D. Spector</h3><p><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </p><div>Find articles by <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Tim D. Spector</a></div></div><div id="_co_idm140353681340624"><h3 class="no_margin">Jerzy Adamski</h3><p><sup>12</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Jerzy Adamski</a></div></div><div id="_co_idm140353682762784"><h3 class="no_margin">Nicole Soranzo</h3><p><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </p><div>Find articles by <a href="/pubmed/?term=Soranzo%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Nicole Soranzo</a></div></div><div id="_co_idm140353683718816"><h3 class="no_margin">Christian Gieger</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21886157">Christian Gieger</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140353681352208_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140353681352208_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140353681352208_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried, Germany </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Physiology and Biophysics, Weill Cornell Medical College in Qatar, Education City - Qatar Foundation, Doha, Qatar </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>The Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton UK </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Metabolon Inc., Durham, North Carolina, USA </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>School of Life Sciences, Oxford Brookes University, Headington, Oxford, UK </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Genome-oriented Bioinformatics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>The member list of the CARDIoGRAM consortium is provided as <strong>Supplemental Information</strong></div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Universität zu Lübeck, Medizinische Klinik II, Lübeck, Germany </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Twin Research &amp; Genetic Epidemiology, King’s College London, UK </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Institute of Experimental Genetics, Life and Food Science Center Weihenstephan, Technische Universität München, Freising-Weihenstephan, Germany </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Renal Division, University Hospital Freiburg, Germany </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Division of Genetic Epidemiology, Department of Medical Genetics, Molecular and Clinical Pharmacology, Innsbruck Medical University, Innsbruck, Austria </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Institute of Epidemiology II, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Institute of Human Genetics, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Institute of Human Genetics, Klinikum rechts der Isar, Technische Universität München, Munich, Germany </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Cardiovascular Sciences, University of Leicester, and Leicester NIHR Biomedical Research Unit in Cardiovascular Disease, Glenfield Hospital, Leicester, UK </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Institute of Epidemiology I, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Klinikum Grosshadern, Munich, Germany </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Unit for Molecular Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg, Germany </div><div><sup>#</sup>Contributed equally.</div><div id="CR1"><sup>+</sup>corresponding authors: <a href="mailto:dev@null" data-email="rf.erhus@netsrak" class="oemail">rf.erhus@netsrak</a> (KS), <a href="mailto:dev@null" data-email="ku.ca.regnas@6sn" class="oemail">ku.ca.regnas@6sn</a> (NS) </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140353681352208_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21886157" target="pmc_ext" ref="reftype=publisher&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nature</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3832838/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm140353689983616" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353689983616title">SUMMARY</h2><!--article-meta--><div><p id="P6" class="p p-first-last">Genome-wide association studies (GWAS) have identified many risk loci for complex diseases, but effect sizes are typically small and information on the underlying biological processes is often lacking. Associations with metabolic traits as functional intermediates can overcome these problems and potentially inform individualized therapy. Here we report a comprehensive analysis of genotype-dependent metabolic phenotypes using a GWAS with non-targeted metabolomics. We identified 37 genetic loci associated with blood metabolite concentrations, of which 25 exhibit effect sizes that are unusually high for GWAS and account for 10-60% of metabolite levels per allele copy. Our associations provide new functional insights for many disease-related associations that have been reported in previous studies, including cardiovascular and kidney disorders, type 2 diabetes, cancer, gout, venous thromboembolism, and Crohn’s disease. Taken together our study advances our knowledge of the genetic basis of metabolic individuality in humans and generates many new hypotheses for biomedical and pharmaceutical research.</p></div></div><div id="idm140353682737888" class="tsec sec headless whole_rhythm"><p id="P7" class="p p-first">Understanding the role of genetic predispositions and their interaction with environmental factors in complex chronic diseases is key in the development of safe and efficient therapies, diagnosis and prevention. Genome-wide association studies (GWAS) have identified hundreds of disease risk loci <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687802">1</a></sup>. However, functional information on the underlying biological processes is often lacking <sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687798">2</a></sup>. Previously, we have shown the promise of using associations with blood metabolites as functional intermediate phenotypes, the so-called genetically determined metabotypes (GDMs), to understand the potential relevance of these genetic variants for biomedical and pharmaceutical research <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687812">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687811">4</a></sup>. Building on this early work, we present here the most comprehensive evaluation of genetic variance in human metabolism to date, combining genetics and metabolomics for hypothesis generation in a GWAS. We used an extensive, non-targeted and metabolome-wide panel of small molecules, analyzing &gt;250 metabolites from 60 biochemical pathways in serum samples of 2,820 individuals from two large population-based European cohorts. We identified 37 genetic loci significant at a stringent genome-wide threshold. In contrast to most GWAS, these loci exhibited exceptionally large effect sizes of 10-60% per allele copy in 25 loci. In the majority of cases a protein biochemically related to the associated metabolic traits is encoded at these loci. As a proof of principle validation of new discoveries, we experimentally validated the predicted function of <em>SLC16A9</em> as a carnitine efflux transporter. We further cross-referenced these loci with databases of disease-related and pharmaceutically-relevant genetic associations, uncovering hitherto unknown links and providing new hypotheses into the function of these loci. Finally, we made publically available a knowledge-base resource via a web-server to aid future functional studies and biological as well as clinical interpretation of GWAS findings. In summary, this study provides compelling evidence for novel associations of metabolic traits at a wide range of loci of biomedical and pharmaceutical interest, and suggests a powerful new paradigm for dissecting human metabolic and disease pathways.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">METHODS</h2><p id="P8" class="p p-first-last">Metabolic profiling was done on fasting serum from participants of the German KORA F4 study (n=1,768) and the British TwinsUK study (n=1,052) using ultrahigh performance liquid-phase chromatography and gas chromatography separation coupled with tandem mass spectrometry <sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687785">5</a>-<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687783">7</a></sup>. We achieved highly efficient profiling (24 minutes/sample) with low median process variability (&lt;12%) of more than 250 metabolites, covering over 60 biochemical pathways of human metabolism (<strong>Supplemental Table 1</strong>). Based on our previous observation that ratios between metabolite concentrations can strengthen the association signal and provide new information about possible metabolic pathways <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687825">4</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687809">8</a></sup>, we included all pairs of ratios between these metabolites in the genome-wide statistical analysis. To reduce the computational and data storage burden associated with meta-analyzing over 37,000 metabolites and ratios, we applied a staged approach for selection of promising association signals (<strong>Supplemental Figure 1</strong>). In the initial screening stage we assessed associations of approximately 600,000 genotyped SNPs with over 37,000 metabolic traits (concentrations and their ratios) by fitting linear models separately in both cohorts to log-transformed metabolic traits, adjusting for age, gender and family structure (<strong>Supplemental Figure 2 &amp; Supplemental Table 2</strong>). Next, we selected all association signals having suggestive evidence for association with a metabolic trait in both cohorts (p&lt;10<sup>−6</sup> in both cohorts or p&lt;10<sup>−3</sup> in one and p&lt;10<sup>−9</sup> in the other). For each of these loci, we then re-assessed the amount of association signals through fixed-effects inverse variance meta-analysis of the two cohorts for all 37,000 available traits using imputed SNPs relative to HapMap2 data (see <strong>Online Methods</strong> for details). The SNP/trait combination yielding the smallest P-value in this meta-analysis was finally selected for each locus. To account for multiple testing we applied conservative Bonferroni correction leading to an adjusted threshold for genome-wide significance of p &lt; 2.0×10<sup>−12</sup>.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">RESULTS</h2><p id="P9" class="p p-first">We identified a total of 37 independent loci that reached genome-wide significance in the meta-analysis (<a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <strong>Supplemental Tables 3&amp;4</strong>). 23 of these loci describe new genetic associations with metabolic traits, and 14 replicate and extend our knowledge of known GDMs, including 10 from our own studies <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687805">3</a>,<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687767">4</a></sup>. We used information on SNP location within genes, known gene function and regional association plots <strong>(Supplemental Figure 2</strong>) to prioritize plausible candidate genes within associated loci. In most cases our annotation was further supported by a statistical analysis of association of gene relationships in published literature <sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687779">9</a></sup> (<strong>Supplemental Table 5</strong>). Associations with additional metabolic traits at the 37 loci presented in <a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> may capture further biochemical information and are provided as <strong>Supplemental Table 6.</strong> At 30 loci the sentinel SNP mapped to a protein that was biochemically linked to the associating metabolites, for instance because responsible for their synthesis, degradation or metabolism. We next extensively searched literature and databases (<strong>see web-links</strong>) to identify which of these 37 loci were previously reported as associated with a clinical endpoint, a medically relevant intermediate phenotype, or a pharmacogenetic effect. Associations of metabolites at disease loci can be used to gain novel information on possible metabolic changes associated with biological processes underlying that association (<a href="/pmc/articles/PMC3832838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, <a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <strong>Supplemental Table 7</strong>). In 15 cases such a relationship could be identified based on the association of the lead SNP or a proxy (r<sup>2</sup>≥0.8) with the disease-associated SNPs, including cardiovascular disease, kidney disease, Crohn’s disease, gout, cancer, pharmacogenomics, and predisposing risk factors for diabetes and cardiovascular disease. Except for three loci, all SNPs are common with minor allele frequencies over 10%. In 25 cases the effect size per allele copy is larger than 10%, and up to 60% in the case of the <em>ACADS</em> locus.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3832838/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353717907472" class="figure"><a href="/pmc/articles/PMC3832838/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832838_emss-54209-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54209-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832838/bin/emss-54209-f0001.jpg"></a></div><div id="largeobj_idm140353717907472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832838/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3832838/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic basis of human metabolic individuality and its overlap with loci of biomedical and pharmaceutical interest</strong><p id="__p3">Over 100 years ago Archibald Garrod realized that inborn errors in human metabolism were <em>‘merely extreme examples of variations of chemical behaviour which are probably everywhere present in minor degrees’</em> and that this <em>‘chemical individuality [confers] predisposition to and immunities from the various mishaps which are spoken of as diseases’</em>
<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687790">36</a></sup>. The 37 genetically determined metabotypes (GDMs) we reported here explain a highly relevant amount of the total variation in the studied population and therefore contribute substantially to the genetic part of human metabolic individuality. GDMs are presented here color-coded <strong>(a)</strong> by general metabolic pathways together with selected associated metabolic traits, highlighting the relationship between gene function and the associated metabolic trait (see column 4 in <a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), and <strong>(b)</strong> by overlap with associations in previous GWAS with disease [red], intermediate disease risk factors [yellow], and other traits [green]. Locus overlap is defined here by the lead SNP reported in the NHGRI GWAS catalogue being highly correlated (R<sup>2</sup>&gt;=0.8) with the most associated SNP in the metabolomics scan (see column 5 in <a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <strong>Suppl. Table 7</strong>). Note that the overlap between the metabolomics loci and the loci reported by the NHGRI GWAS catalogue is highly significant when compared to a draw of 37 randomly selected SNPs with similar properties (p&lt;3×10<sup>−6</sup>, see <strong>Methods</strong>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>37 loci that displayed genome-wide significance in the meta-analysis</strong><p id="__p5">The metabolic trait with the strongest association at the discovery stage in both studies is reported together with the SNP identifier and the p-value of association from the meta-analysis. Full association data are available in <strong>Supplemental Tables 3 &amp; 5</strong> and via a web-server (<a href="http://www.gwas.eu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwas.eu</a>). The loci are labeled by the gene that is considered most likely to carry the causative SNP. Overlap with associations from other GWAS studies are highlighted in bold (R<sup>2</sup>&gt;0.8, details are in <strong>Supplemental Table 6</strong>). Where the metabolic trait is consistent with a nearby gene’s function, details are provided in the column labeled ‘Relationship between gene function and the associated metabolic traits’. Metabolic traits that are associated with the SNP at the corresponding locus are marked with a superscript ‘+’. Further information and full bibliographic references are presented in <strong>Supplemental Table 4</strong>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353754780576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center">Locus &amp;<br>SNP id</th><th rowspan="1" colspan="1" valign="middle" align="center">Metabolic trait</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value</th><th rowspan="1" colspan="1" valign="middle" align="center">Relationship between gene function and the associated<br>metabolic traits</th><th rowspan="1" colspan="1" valign="middle" align="center">Biomedical and pharmaceutical interest</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ACADS</em><br>rs2066938</td><td rowspan="1" colspan="1" valign="middle" align="center">butyrylcarnitine /<br>propionylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">&lt;4.4×10<sup>−305</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Butyrylcarnitine<sup>+</sup> and propionylcarnitine<sup>+</sup> are substrates/products<br>of ACADS</td><td rowspan="1" colspan="1" valign="middle" align="center">ACADS is a key enzyme in the mitochondrial fatty acid<br>beta-oxidation</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>NAT8</em><br>rs13391552</td><td rowspan="1" colspan="1" valign="middle" align="center">N-acetylornithine</td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−252</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">N-acetyltransferase function of NAT8 matches the associating<br>metabolite N-acetylornithine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with glomerular filtration and<br>CKD; association of N-acetylornithine<sup>+</sup> with eGFR in this<br>study</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>FADS1</em><br>rs174547</td><td rowspan="1" colspan="1" valign="middle" align="center">1-arachidonoylglycero-<br>phosphoethanolamine/<br>1-linoleoylglycerophospho-<br>ethanolamine</td><td rowspan="1" colspan="1" valign="middle" align="center">8.5×10<sup>−116</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">FADS1 substrate/product pair ratio arachidonate (20:4n6)<sup>+</sup> /<br>dihomo-linolenate (20:3n3 or n6)<sup>+</sup> is among the top associations</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with LDL cholesterol, HDL cholesterol &amp;<br>triglycerides, fasting glucose &amp; HOMA-B, Crohn’s<br>disease, resting heart rate</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>UGT1A</em><br>rs887829</td><td rowspan="1" colspan="1" valign="middle" align="center">bilirubin (E,E) / oleoylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9×10<sup>−74</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Bilirubin<sup>+</sup> is a substrate of UGT1A1</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with hyperbilirubinemia; low serum<br>concentration of bilirubin associate with increased risk of<br>coronary artery disease; a SNP in <em>UGT1A1</em> is a<br>pharmacogenetic risk factor for irinotecan toxicity</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ACADM</em><br>rs211718</td><td rowspan="1" colspan="1" valign="middle" align="center">hexanoylcarnitine / oleate<br>(18:1n9)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.2×10<sup>−71</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Hexanoylcarnitine<sup>+</sup> is a substrate of ACADM</td><td rowspan="1" colspan="1" valign="middle" align="center">ACADM is a key enzyme in the mitochondrial fatty acid<br>beta-oxidation</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>OPLAH</em><br>rs6558295</td><td rowspan="1" colspan="1" valign="middle" align="center">5-oxoproline</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5×10<sup>−59</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">5-oxoproline<sup>+</sup> is a substrate of 5-oxoprolinase OPLAH</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SCD</em><br>rs603424</td><td rowspan="1" colspan="1" valign="middle" align="center">myristate (14:0) / myristoleate<br>(14:1n5)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9×10<sup>−57</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SCD catalyzes the delta-9-desaturation of fatty acids, such as<br>myristate (14:0)<sup>+</sup> to myristoleate (14:1n5)<sup>+</sup><br>and palmitate (16:0)<sup>+</sup> to palmitoleate (16:1n7)<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Palmitoleate (16:1n7) is a lipokine linking adipose tissue<br>to systemic metabolism</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>GCKR</em><br>rs780094</td><td rowspan="1" colspan="1" valign="middle" align="center">glucose / mannose</td><td rowspan="1" colspan="1" valign="middle" align="center">5.5×10<sup>−53</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">GCKR plays a role in glucose homeostasis, strong association with<br>mannose<sup>+</sup> to glucose<sup>+</sup> ratios matches the gene’s function</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with type 2 diabetes, fasting glucose, fasting<br>insulin; serum uric acid; triglyceride levels; C-reactive<br>protein; eGFRcrea; Crohn’s disease;<br>hypertriglyceridemia</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>NAT2</em><br>rs1495743</td><td rowspan="1" colspan="1" valign="middle" align="center">1-methylxanthine /<br>4-acetamidobutanoate</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−40</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">4-acetamidobutanoate<sup>+</sup>, 1-methylxanthine<sup>+</sup>, and 1-methylurate<sup>+</sup><br>are linked to NAT2 in the xenobiotics pathways</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with triglyceride levels and CAD; bladder<br>cancer; toxicities to docetaxel and thalidomide treatment</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>CYP3A4</em><br>rs17277546</td><td rowspan="1" colspan="1" valign="middle" align="center">androsterone sulfate</td><td rowspan="1" colspan="1" valign="middle" align="center">8.7×10<sup>−40</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">CYP3A cytochrome P450 proteins metabolize androsterone<br>sulfate<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Genetic variance in androsterone metabolism is linked to<br>the incidence of prostate cancer</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ABO</em><br>rs612169</td><td rowspan="1" colspan="1" valign="middle" align="center">ADpSGEGDFXAEGGGVR /<br>ADSGEGDFXAEGGGVR</td><td rowspan="1" colspan="1" valign="middle" align="center">9.1×10<sup>−40</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Polymorphisms in ABO determine the blood group; association to<br>fibrinogen peptide phosphorylation<sup>+</sup>; additive effect on fibrinogen<br>A-alpha phosphorylation together with <em>FUT2</em> and <em>ALPL</em></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with blood ALP level; pancreatic cancer;<br>venous thromboembolism; phytosterol levels</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC2A9</em><br>rs4481233</td><td rowspan="1" colspan="1" valign="middle" align="center">urate</td><td rowspan="1" colspan="1" valign="middle" align="center">5.5×10<sup>−34</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SLC2A9 (GLUT9) transports uric acid<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with gout; several SNPs in <em>SLC2A9</em> associate<br>with etoposide IC<sub>50</sub></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>CYP4A</em><br>rs9332998</td><td rowspan="1" colspan="1" valign="middle" align="center">10-nonadecenoate (19:1n9) /<br>10-undecenoate (11:1n1)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.1×10<sup>−32</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Cytochrome P450, family 4, subfamily A, are fatty acid omega-<br>hydroxylases; 10-undecenoate (11:1n1)<sup>+</sup> is biochemically related<br>to omega-hydroxylated C10 fatty acids</td><td rowspan="1" colspan="1" valign="middle" align="center">Possible role in the etiology of hepatic steatosis in<br>interaction with <em>SCD</em></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>CPS1</em><br>rs2216405</td><td rowspan="1" colspan="1" valign="middle" align="center">glycine</td><td rowspan="1" colspan="1" valign="middle" align="center">1.6×10<sup>−27</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with glycine<sup>+</sup> and creatine<sup>+</sup>; creatine is produced from<br>glycine; glycine is metabolically related to carbamoyl phosphate,<br>which is the product of CPS1 and the entry point of ammonia into<br>the urea cycle</td><td rowspan="1" colspan="1" valign="middle" align="center">Metabolomics data suggests that this association is<br>related to a perturbed ammonia metabolism</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>LACTB</em><br>rs2652822</td><td rowspan="1" colspan="1" valign="middle" align="center">succinylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">7.2×10<sup>−27</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with succinylcarnitine<sup>+</sup>; perturbed hepatic gene<br>expression in transgenic <em>LACTB</em> mice suggests a role of LACTB in<br>butanoate/succinate<sup>+</sup> pathway</td><td rowspan="1" colspan="1" valign="middle" align="center"><em>LACTB</em><sup>tg</sup> mice are obese</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC22A1</em><br>rs662138</td><td rowspan="1" colspan="1" valign="middle" align="center">isobutyrylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">7.3×10<sup>−25</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SLC22A1 (OCT1) translocates a broad array of organic cations,<br>possibly also isobutyrylcarnitine<sup>+</sup> or related metabolites</td><td rowspan="1" colspan="1" valign="middle" align="center">Genetic variation in SLC22A1/SLC22A2 region are<br>determinants of metformin pharmacokinetics</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLCO1B1</em><br>rs4149081</td><td rowspan="1" colspan="1" valign="middle" align="center">eicosenoate (20:1n9 or 11) /<br>tetradecanedioate</td><td rowspan="1" colspan="1" valign="middle" align="center">2.8×10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SLCO1B1 (OATP2, OATP-C) is an organic anion transporter</td><td rowspan="1" colspan="1" valign="middle" align="center">Common variants in <em>SLCO1B1</em> are strongly associated<br>with an increased risk of statin-induced myopathy</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>FUT2</em><br>rs503279</td><td rowspan="1" colspan="1" valign="middle" align="center">ADpSGEGDFXAEGGGVR /<br>ADSGEGDFXAEGGGVR</td><td rowspan="1" colspan="1" valign="middle" align="center">4.3×10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">FUT2 is involved in the creation of a precursor of a H antigen,<br>additive effect on fibrinogen A-alpha phosphorylation together<br>with <em>ABO</em> and <em>ALPL</em>.</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with vitamin B12 levels,<br>total cholesterol, Crohn’s disease; vitamin B12<br>deficiency is associated with cognitive decline, cancer<br>and CAD</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ACE</em><br>rs4329</td><td rowspan="1" colspan="1" valign="middle" align="center">aspartylphenylalanine</td><td rowspan="1" colspan="1" valign="middle" align="center">8.2×10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Angiotensin I converting enzyme (peptidyl-dipeptidase A) 1 is<br>associated with the dipeptide aspartylphenylalanine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with angiotensin-converting enzyme<br>activity, potential genetic interaction with <em>KLKB1</em> locus</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>PHGDH</em><br>rs477992</td><td rowspan="1" colspan="1" valign="middle" align="center">serine</td><td rowspan="1" colspan="1" valign="middle" align="center">2.6×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">PHGDH catalyses the first and rate-limiting step in the<br>phosphorylated pathway of serine<sup>+</sup> biosynthesis</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ENPEP</em><br>rs2087160</td><td rowspan="1" colspan="1" valign="middle" align="center">ADpSGEGDFXAEGGGVR /<br>DSGEGDFXAEGGGVR</td><td rowspan="1" colspan="1" valign="middle" align="center">6.5×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">ENPEP (APA, Aminopeptidase A) is an N-terminal amino peptidase;<br>association with ratios between fibrinogen A-alpha peptide<br>ADSGEGDFXAEGGGVR<sup>+</sup> and its N-terminal cleaved form<br>DSGEGDFXAEGGGVR<sup>+</sup> suggests that fibrinogen is a substrate of<br>ENPEP</td><td rowspan="1" colspan="1" valign="middle" align="center">ENPEP plays a role in the catabolic pathway of the renin-<br>angiotensin system and regulates blood pressure,<br>association with blood pressure in Asian population</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>AKR1C</em><br>rs2518049</td><td rowspan="1" colspan="1" valign="middle" align="center">androsterone sulfate /<br>epiandrosterone sulfate</td><td rowspan="1" colspan="1" valign="middle" align="center">6.7×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">AKR1C isoforms play a role in androgen+ metabolism</td><td rowspan="1" colspan="1" valign="middle" align="center">AKR1C plays a role in the etiology of different cancers,<br>including prostate, brain, breast, bladder and leukemia;<br>potential target of jasmonates in cancer cells</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>NT5E</em><br>rs494562</td><td rowspan="1" colspan="1" valign="middle" align="center">inosine</td><td rowspan="1" colspan="1" valign="middle" align="center">7.4×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Inosine<sup>+</sup> is a substrate of the 5′-nucleotidase NT5E</td><td rowspan="1" colspan="1" valign="middle" align="center">NT5E is involved in purine salvage</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>PRODH</em><br>rs2023634</td><td rowspan="1" colspan="1" valign="middle" align="center">proline</td><td rowspan="1" colspan="1" valign="middle" align="center">2.0×10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">PRODH catalyzes the first step in proline<sup>+</sup> degradation</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>HPS5</em><br>rs2403254</td><td rowspan="1" colspan="1" valign="middle" align="center">alpha-hydroxyisovalerate</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0×10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Alpha-hydroxyisovalerate<sup>+</sup> is found in urine of patients with<br>phenylketonuria, phenylalanine is required for melatonin<br>biosynthesis</td><td rowspan="1" colspan="1" valign="middle" align="center">Melatonin homeostasis is deranged in patients with loss<br>of <em>HPS</em> genes (albinism)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ALPL</em><br>rs10799701</td><td rowspan="1" colspan="1" valign="middle" align="center">ADpSGEGDFXAEGGGVR /<br>DSGEGDFXAEGGGVR</td><td rowspan="1" colspan="1" valign="middle" align="center">2.9×10<sup>−20</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">ALPL is a phosphatase and associates with A-alpha fibrinogen<br>phosphorylation<sup>+</sup>; additive effect on fibrinogen A-alpha<br>phosphorylation together with <em>ABO</em> and of <em>FUT2</em>.</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC7A6</em><br>rs6499165</td><td rowspan="1" colspan="1" valign="middle" align="center">glutaroyl carnitine / lysine</td><td rowspan="1" colspan="1" valign="middle" align="center">9.8×10<sup>−19</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Glutaryl-CoA<sup>+</sup> is an intermediate in the metabolism of lysine<sup>+</sup> and<br>tryptophan;</td><td rowspan="1" colspan="1" valign="middle" align="center">Deficiencies in glutaryl-CoA DH are linked to metabolic<br>disorders</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>KLKB1</em><br>rs4253252</td><td rowspan="1" colspan="1" valign="middle" align="center">bradykinin, des-arg(9)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.6×10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Kallikrein B, plasma (Fletcher factor) 1; kallikrein-kininogen<br>complex binds to cell surface receptors leading to the targeted<br>action of bradykinin<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association of bradykinin<sup>+</sup> with hypertension confirmed<br>in this study; potential genetic interaction with ACE locus</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>GLS2</em><br>rs2657879</td><td rowspan="1" colspan="1" valign="middle" align="center">glutamine</td><td rowspan="1" colspan="1" valign="middle" align="center">3.1×10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">GLS2 catalyzes the hydrolysis of glutamine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>PDXDC1</em><br>rs7200543</td><td rowspan="1" colspan="1" valign="middle" align="center">1-eicosatrienoylglyceropho-<br>sphocholine /<br>1-linoleoylglycerophos-<br>phocholine</td><td rowspan="1" colspan="1" valign="middle" align="center">4.5×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with 1-eicosadienoyl- to 1-<br>eicosatrienoyl-glycerophosphocholine<sup>+</sup> ratio suggests role of<br>PDXDC1 in the metabolism of C20:2 and C20:3 fatty acids</td><td rowspan="1" colspan="1" valign="middle" align="center">Association with body height</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC22A4</em><br>rs272889</td><td rowspan="1" colspan="1" valign="middle" align="center">isovalerylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">7.4×10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SLC22A4 (OCTN1) transports isovalerylcarnitine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Association with body height</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>AHR</em><br>rs12670403</td><td rowspan="1" colspan="1" valign="middle" align="center">caffeine / quinate</td><td rowspan="1" colspan="1" valign="middle" align="center">4.8×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">AHR is a transcription factor for CYP1A1, which metabolizes<br>caffeine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ETFDH</em><br>rs8396</td><td rowspan="1" colspan="1" valign="middle" align="center">decanoylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">5.5×10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Decanoylcarnitine<sup>+</sup> used for energy production via beta oxidation<br>to electron transfer complex</td><td rowspan="1" colspan="1" valign="middle" align="center">ETFDH is a key enzyme in the mitochondrial fatty acid<br>beta-oxidation</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>ELOVL2</em><br>rs9393903</td><td rowspan="1" colspan="1" valign="middle" align="center">docosahexaenoate<br>(DHA; 22:6n3) /<br>eicosapentaenoate<br>(EPA; 20:5n3)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.7×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">EPA (20:5n3)<sup>+</sup> is a substrate of ELOVL2, DHA (22:6n3)<sup>+</sup> is related to<br>its product through by a single desaturation reaction</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC16A9</em><br>rs7094971</td><td rowspan="1" colspan="1" valign="middle" align="center">carnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">3.4×10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">SLC16A9 (MCT9) transports free carnitine<sup>+</sup><br>(shown in this paper)</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>IVD</em><br>rs10518693</td><td rowspan="1" colspan="1" valign="middle" align="center">3-(4-hydroxyphenyl)lactate /<br>isovalerylcarnitine</td><td rowspan="1" colspan="1" valign="middle" align="center">1.1×10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">Isovalerylcarnitine<sup>+</sup> is a transport form of isovalerate, which is the<br>substrate isovaleryl coenzyme A dehydrogenase (IVD)</td><td rowspan="1" colspan="1" valign="middle" align="center">IVD is a key enzyme in the mitochondrial fatty acid beta-<br>oxidation</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC16A10</em><br>rs7760535</td><td rowspan="1" colspan="1" valign="middle" align="center">isoleucine / tyrosine</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"><em>SLC16A10</em> encodes the T-type amino acid transporter-1 (TAT1);<br>this transporter transports tyrosine<sup>+</sup> and phenylalanine<sup>+</sup></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140353754780576" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3832838/table/T1/?report=objectonly">Open in a separate window</a></div></div><div id="S3" class="sec"><h3 id="S3title">Overlap with chronic disease loci</h3><p id="P10" class="p p-first-last">Many genetic risk loci for heart disease, kidney failure, diabetes and other complex disorders have been identified by GWAS. However, the etiology of these common diseases is complex and testable hypotheses are needed in order to develop new avenues for diagnosis and therapy. Associations of known disease risk loci with metabolic traits allow identifying new and potentially relevant biological processes and pathways. Below we report some examples from our study that illustrate this idea, with the full association dataset being freely available for further analysis and reference at <a href="http://www.gwas.eu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.gwas.eu</a>.</p></div><div id="S4" class="sec"><h3 id="S4title">Detoxification and kidney failure</h3><p id="P11" class="p p-first-last">N-acetylation is an important mechanism to detoxify numerous nephrotoxic medications and environmental toxins. A reduced ability to detoxify such substances could lead to impaired kidney function. A key GDM is the N-acetylase <em>NAT8</em> locus, which was reported to associate with kidney function <sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687826">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687769">11</a></sup>. Here we found a highly-significant association of variation at the <em>NAT8</em> locus with N-acetylornithine. Using this information we then asked whether N-acetylornithine concentrations were associated with kidney function. We found a clear association in both our studies with estimated glomerular filtration rate (eGFR), whereby higher levels of N-acetylornithine were correlated with lower eGFR (p<sub>KORA</sub>=7.6×10<sup>−4</sup>, p<sub>TwinsUK</sub>=3.6×10<sup>−8</sup> after adjusting for age and gender). In accord with the genetic effect of the <em>NAT8</em> polymorphism in the chronic kidney disease (CKD) association, the risk allele associated here with higher N-acetylornithine concentrations. Although causality cannot be inferred from this kind of association studies, the role of ornithine acetylation in the etiology of CKD warrants further exploration.</p></div><div id="S5" class="sec"><h3 id="S5title">Diabetes</h3><p id="P12" class="p p-first-last"><em>GCKR</em> is a major pleiotropic risk locus associated with diabetes- and cardiometabolic-related traits, such as fasting glucose and insulin <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687788">12</a></sup>, triglyceride levels <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687791">13</a></sup>, and CKD <sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687827">11</a></sup>. Here we identified a highly significant association of this locus with mannose to glucose ratios. Fasting mannose is lower in carriers of the risk allele, as opposed to glucose. Interestingly, we also observed a 3.3% increase in lactate concentrations per copy of the risk allele at the same locus. Little is known about the physiological role of mannose other than its use in protein glycosylation. Mannose enters the cell via a specific transporter that is insensitive to glucose <sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687819">14</a></sup>, and hepatic glycogen breakdown is implicated in the maintenance of plasma mannose concentrations <sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687799">15</a></sup>. These observations and the association with <em>GCKR</em> observed here, which is even stronger than that of glucose with <em>GCKR</em>, suggest the need for further investigations on the role of mannose as a differential biomarker or even as a point of intervention in diabetes care.</p></div><div id="S6" class="sec"><h3 id="S6title">Venous thromboembolism</h3><p id="P13" class="p p-first-last">With the mass-spectrometry method used here, different forms of the abundant fibrinogen A-alpha peptides can be detected. Fibrinogen plays a role in blood clot formation. Its active form, the fibrinogen A-alpha chain ADSGEGDFXAEGGGVR can be phosphorylated at Serine-3 to ADpSGEGDFXAEGGGVR <sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687830">16</a></sup>. The ratio between the concentrations of these fibrinogen A-alpha peptides provides a measure for fibrinogen A-alpha phosphorylation (FAaP). Increased levels of FAaP have been observed under different physiological and pathophysiological conditions <sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687815">17</a></sup>. Here, three loci (<em>ABO, ALPL, FUT2</em>) associated with FAaP. Intriguingly, these three genes are functionally linked: <em>ABO</em> and <em>FUT2</em> are involved in determining the blood group, and the <em>ABO</em> locus is associated with blood levels of the phosphatase ALPL <sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687781">18</a></sup>. The association of <em>ALPL</em> with FAaP may be explained by either a genotype-dependent dephosphorylation of fibrinogen by ALPL, or a genotype-dependent change in the phosphorus pool available for FAaP. Variants in the <em>ABO</em> gene are associated with many different outcomes, including venous thromboembolism (VTE) <sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687822">19</a></sup>. The association of <em>ABO</em> with FAaP, and thus modified blood coagulation properties, provides a functional explanation for the reported association of <em>ABO</em> with VTE risk. Moreover, if FAaP is at the basis of VTE, then <em>FUT2</em> and <em>ALPL</em> should also be investigated as VTE risk genes, which is a hypothesis that may now be tested in the respective patient groups.</p></div><div id="S7" class="sec"><h3 id="S7title">Coronary artery disease</h3><p id="P14" class="p p-first-last">We have shown previously <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687818">4</a></sup> that strong associations with metabolic traits can point to interesting associations in GWAS with clinical endpoints that otherwise would not be considered as relevant. A recent meta-analysis with lipid traits <sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687801">20</a></sup> identified several genetic loci also affecting risk of CAD in the CARDIoGRAM study <sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687784">21</a></sup> using a similar strategy. Six of such loci are also reported here (<em>ABO, NAT2, CPS1, NAT8, ALPL, KLKB1</em>), albeit some of them showed only weak evidence for association (p &lt; 0.01) with CAD in the CARDIoGRAM study (<strong>Supplemental Table 8</strong>). Although not statistically strong, the biochemical function of the associated metabolic traits identified here may support a possible role in heart disease. For instance, <em>NAT8</em> may be linked to CKD via ornithine acetylation (see above). <em>KLKB1</em> controls blood pressure via the bradykinin pathway. In this study a genetic variant in <em>KLKB1</em> associated with bradykinin concentrations and we also confirmed the expected directional association of bradykinin with hypertension in both our studies (p<sub>KORA</sub>=1.7×10<sup>−9</sup>, p<sub>TwinsUK</sub>=0.0495, with covariates age and gender). <em>ABO</em> and <em>ALPL</em> associated with FAaP, and it may therefore be speculated that genetically determined differences in FAaP and resulting blood coagulation properties may be at the basis of these associations with CAD. Furthermore, our associations suggest that the role of FAaP as a biomarker for acute myocardial infarction, and the combined additive genetic effect of <em>ABO, ALPL,</em> and <em>FUT2</em> loci (<strong>Supplemental Figure 4</strong>) on CAD risk, should be investigated in greater detail.</p></div><div id="S8" class="sec"><h3 id="S8title">New biological and functional insights derived from this study</h3><p id="P15" class="p p-first-last">Genome-wide association studies uncover merely statistically significant associations and thereby are only able to generate biological hypotheses. While it is clear that providing experimental validation of all associations is beyond what can be achieved in a single study, we nevertheless attempted to show that in principle this is possible. The association of SNP rs7094971 in <em>SLC16A9</em> (<em>MCT9</em>) with carnitine suggested that this metabolite is the substrate of this hitherto uncharacterized monocarboxylic acid transporter. We therefore tested [<sup>3</sup>H]-carnitine uptake by <em>SLC16A9</em>-expressing <em>Xenopus</em> oocytes. As shown in <a href="/pmc/articles/PMC3832838/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, our data shows that SLC16A9 is a sodium- and pH-independent carnitine efflux transporter, possibly responsible for carnitine efflux from absorptive epithelia into the blood. Another prominent example is the highly significant association of increased urate levels and their clinical complication of gout with variants in the <em>SLC2A9</em> gene <sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687800">22</a></sup>, the former of which we also observe here. Although previously annotated as a glucose transporter, <em>SLC2A9</em> was later shown <sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687816">23</a></sup> to encode a high-capacity urate transporter. Similar characterization experiments by specialists in the related fields shall be motivated and guided by our association data. Among the 37 GDMs reported here, we suggest that the associations with coarsely-characterized enzyme and transporter genes that are known disease risk loci may warrant further experimental investigation, for instance in experiments using isotope-labeled derivates of the associated metabolites reported here as putative target substrates. For the reasons detailed above we deem <em>NAT8</em> to be a prime candidate for such a study.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3832838/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140353683052512" class="figure"><a href="/pmc/articles/PMC3832838/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3832838_emss-54209-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-54209-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3832838/bin/emss-54209-f0002.jpg"></a></div><div id="largeobj_idm140353683052512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3832838/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3832838/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Experimental evidence for SLC16A9 (MCT9) as a carnitine efflux transporter</strong><p id="__p4">When incubated in uptake medium containing [<sup>3</sup>H]-carnitine (4μCi/ml) there was no significant uptake indicating that MCT9 does not mediate carnitine uptake. As some of the previously characterised MCTs are proton-coupled <sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687786">37</a></sup>, uptake was measured at both pH<sub>out</sub> 7.4 and 5.5, but no significant difference was observed (data not shown). However, when 4.6nl of [<sup>3</sup>H]-carnitine was injected into the oocyte followed by incubation in medium for 90 minutes, efflux was significantly higher in oocytes expressing MCT9 than in the non-injected (NI, <strong>Figure a</strong>) or water-injected (WI, <strong>Figure b</strong>) controls, while again changing the pH<sub>out</sub> had no effect (<strong>Figure a</strong>). In agreement with the lack of uptake of [<sup>3</sup>H]-carnitine, external unlabelled carnitine was unable to <em>trans</em>-stimulate [<sup>3</sup>H]-carnitine efflux with no significant difference in efflux between MCT9-expressing oocytes in the absence or presence of 5mM carnitine (MCT9 vs. MCT9+carn, respectively, <strong>Figure b</strong>). Data are means ± SEM of 6-10 oocytes per data point from 2 oocyte preparations. Y-axis on plots represents remaining [<sup>3</sup>H]-Carnitine (cpm/oocytes). Statistical significance was determined by the Student’s t test. Taken together, these results are consistent with MCT9 acting as a unidirectional carnitine efflux system when expressed in <em>Xenopus</em> oocytes. Note that additional experiments are required to establish the full substrate specificity of MCT9. If future studies show an appropriate cellular distribution, MCT9 could be responsible for carnitine efflux across the basolateral membrane of absorptive epithelial cells following absorption via the well-characterized apical epithelial proton-coupled carnitine transporters OCTNs / SLC22 family <sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687806">38</a></sup>.</p></div></div></div></div><div id="S9" class="sec sec-last"><h3 id="S9title">Pharmacogenomics</h3><p id="P16" class="p p-first-last">Using the <em>Pharmacogenomics Knowledge Base</em>
<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687803">24</a></sup> we identified six GDMs as previously associated with toxicity or adverse reactions to medication. Noteworthy are polymorphisms in the <em>NAT2</em> and in <em>CYP4A</em> loci that associated with toxicities to docetaxel and thalidomide treatment <sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687792">25</a></sup>, the <em>UGT1A</em> locus with irinotecan toxicity <sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687813">26</a></sup>, <em>SLC2A9</em> with etoposide IC<sub>50</sub><sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687777">27</a></sup>, <em>SLC22A1</em> with metformin pharmacokinetics <sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687828">28</a>,<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687821">29</a></sup>, and <em>SLCO1B1</em> with statin-induced myopathy <sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687778">30</a></sup>. In all cases our associations with metabolic traits at these loci provide a possible novel biochemical basis for the genotype-dependant reaction to drug treatment, such as the association of <em>SLCO1B1</em> with a series of fatty acids, including tetradecanedioate and hexadecanedioate. This information can be used to support redesign of the respective drug molecules to avoid adverse reactions. Moreover, systematic inclusion of biochemically relevant GDMs as candidate SNPs during drug trials may permit early identification of potentially adverse pharmacogenetic effects. Concretely this applies to <em>AKR1C</em>, which is a novel target of jasmonates in cancer cells <sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687765">31</a></sup>. We reported a GDM associated with <em>AKR1C</em> with a large effect size on androgen metabolism. Influence of SNP rs2518049 in <em>AKR1C</em> on the drug’s efficiency and potential side effects should therefore be assessed in upcoming clinical trials.</p></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P17" class="p p-first-last">Due to their large effect size and high explained variance, the 37 genetically determined metabotypes (GDMs) reported in this study indicate key genetic loci underpinning differences in human metabolism. Inclusion of these genetic variants in the statistical analysis of pre-clinical and clinical studies may facilitate identification of genotype-dependent outcomes, such as disease complications and adverse drug reactions. In two cases we could establish a direct functional link, supported by both our studies, between a genetic variant, an intermediate metabolic trait, and a disease relevant endpoint: <em>KLKB1</em>-bradykinin-hypertension and <em>NAT8</em>-N-acetylornithine-eGFR. We note that by discussing only associations that are supported by two independent studies at genome-wide significance we have chosen to take a very conservative approach. Based on QQ-plots and coarse assumptions, we estimate that over 500 loci with signals of association below that conservative threshold may be confirmed as GDMs in future, more highly powered studies. On a more technical note it is worthwhile mentioning that by using a single study to metabolically profile 2,820 individuals, based on only 100 micro-liters of blood serum, we replicated in this study a wide series of findings from previous large GWAS with quantitative traits, including serum fasting glucose <sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687766">12</a></sup>, bilirubin <sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687804">32</a>,<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687773">33</a></sup>, urate <sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687808">34</a></sup>, and dehydroisoandrosterone sulfate <sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687793">35</a></sup> levels. Taken together, our study shows how GWAS with intermediate traits that are close to the underlying biological processes provide significant new functional insights into associations from GWAS with complex chronic disease endpoints and drug toxicity. Future GWAS that combine multiple Omics-technologies in a single study, including transcriptomics, proteomics, metabolomics and recent technologies for determining epigenetic modifications on a genome-wide scale are likely the next big step towards a full understanding of the interaction of genetic predispositions with environmental factors in complex chronic diseases and safe and efficient therapies, diagnosis and prevention.</p></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">ONLINE METHODS</h2><div id="S13" class="sec sec-first"><h3 id="S13title">1. STUDY DESIGN</h3><div id="S14" class="sec sec-first"><p></p><h4 id="S14title" class="inline">Study populations </h4><p id="P19" class="p p-first-last">The KORA S4 survey, an independent population-based sample from the general population living in the region of Augsburg, Southern Germany, was conducted in 1999–2001. The study design and standardized examinations of the survey (4,261 participants, response 67%) have been described in detail (ref. <sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687797">39</a></sup> and the references therein). A total of 3,080 subjects participated in a follow-up examination KORA F4 in 2006–2008 comprising individuals who, at that time, were aged 32–81 years. The TwinsUK cohort is an adult twin British registry in the age range 8-102 years and 84% are female. The samples used in this study are aged 23-85 (mean 48 years) and 97% female. These unselected twins were recruited from the general population through national media campaigns in the United Kingdom and were shown to be comparable to age-matched population singletons in terms of disease-related and lifestyle characteristics <sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687807">40</a></sup>. In both studies written informed consent has been given by all participants and the studies have been approved by the local ethics committees (Bayerische Landesärztekammer for KORA and Guy’s and St. Thomas’ Hospital Ethics Committee for TwinsUK).</p></div><div id="S15" class="sec sec-last"><p></p><h4 id="S15title" class="inline">Blood sampling </h4><p id="P20" class="p p-first-last">Blood samples for metabolic analysis and DNA extraction from KORA were collected between 2006 and 2008 as part of the KORA F4 follow-up. To avoid variation due to circadian rhythm, blood was drawn in the morning between 8:00 a.m. and 10:30 a.m. after a period of at least 10 hours overnight fasting. Material was drawn into serum gel tubes, gently inverted two times and then allowed to rest for 30 min at room temperature (18–25 °C) to obtain complete coagulation. The material was then centrifuged for 10 min (2,750<em>g</em> at 15 °C). Serum was divided into aliquots and kept for a maximum of 6 h at 4 °C, after which it was deep frozen to −80 °C until analysis. For the TwinsUK study, blood samples were taken after at least 6 h of fasting. The samples were immediately inverted three times, followed by 40 min resting at 4 °C to obtain complete coagulation. The samples were then centrifuged for 10 min at 2,000<em>g</em>. Serum was removed from the centrifuged brown-topped tubes as the top, yellow, translucent layer of liquid. Four aliquots of 1.5 ml were placed into skirted microcentrifuge tubes and then stored in a −45 °C freezer until sampling.</p></div></div><div id="S16" class="sec"><h3 id="S16title">2. GENETIC AND METABOLOMICS DATA COLLECTION</h3><div id="S17" class="sec sec-first"><p></p><h4 id="S17title" class="inline">Metabolomics measurements </h4><p id="P21" class="p p-first-last">Metabolon, an US- based commercial supplier of metabolic analyses, developed a platform that integrates the chemical analysis, including identification and relative quantification, data reduction, and quality assurance components of the process. The analytical platform incorporates two separate ultrahigh performance liquid chromatography/tandem mass spectrometry (UHPLC/MS/MS2) injections and one GC/MS injection per sample. The UHPLC injections are optimized for basic and acidic species. A total of 295 metabolites were measured, spanning several relevant classes (amino acids, acylcarnitines, sphingomyelins, glycerophospholipids, carbohydrates, vitamins, lipids, nucleotides, peptide, xenobiotics and steroids; a full list of metabolites is given in <strong>Supplemental Table 1</strong>). The detection of the entire panel was carried out with 24 min instrument analysis time (two injections at 12 min each), while maintaining low median process variability (&lt;12% across all compounds). The resulting MS/MS<sup>2</sup> data were searched against a standard library generated by Metabolon that included retention time, molecular weight (m/z), preferred adducts, and in-source fragments as well as their associated MS/MS spectra for all molecules in the library. The library allowed for the identification of the experimentally detected molecules based on a multiparameter match without need for additional analyses. Metabolon has shown in a recent publication that their integrated platform enabled the high-throughput collection and relative quantitative analysis of analytical data and identified a large number and broad spectrum of molecules with a high degree of confidence <sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687810">5</a></sup>. The Metabolon platform has, among other studies, been successfully applied in the analysis of the adult human plasma metabolome <sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687768">41</a></sup> and the identification of sarcosine as a biomarker for prostate cancer <sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687796">42</a></sup>.</p></div><div id="S18" class="sec"><p></p><h4 id="S18title" class="inline">Metabolomics data QC </h4><p id="P22" class="p p-first-last">For this study we measured the Metabolon panel in human blood from 1,768 individuals of the KORA cohort and in 1,052 individuals of the TwinsUK cohort. Quality control data (%RSD, upper and lower 95% confidence interval, minimum and maximum observed values in QC samples) are reported in <strong>Supplemental Table 1</strong>. In order to avoid spurious false positive associations due to small sample sizes, only metabolic traits with at least 300 non-missing values were included and data-points of metabolic traits that lay more than 3 standard deviations off the mean were excluded by setting them to missing in the analysis. 276 of 295 available metabolites and 37,179 metabolite ratios satisfied this criterion in KORA, resulting in a total of 37,455 metabolic traits. For the TwinsUK study, identical selection criteria for metabolic traits were used, resulting in 258 metabolites and 32,499 metabolite ratios, and a total of 32,757 metabolic traits.</p></div><div id="S19" class="sec sec-last"><p></p><h4 id="S19title" class="inline">Genotyping and imputation </h4><div id="S20" class="sec sec-first"><p></p><h5 id="S20title" class="inline">KORA </h5><p id="P23" class="p p-first-last">For all individuals profiled in this study, genome-wide SNP data were already available. GWAS data of KORA and TwinsUK have been used and described extensively in the past in the context of numerous genome-wide association studies and meta-analyses <sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687794">3</a>,<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687782">34</a>,<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687823">43</a></sup>. We therefore only summarize the essential details here. Genotyping of the KORA F4 population was carried out using the Affymetrix GeneChip array 6.0. Genotypes were determined using Birdseed2 clustering algorithm. For quality assurance we applied as filters for SNP quality: call rate &gt; 95% and p(HWE) &gt; 10<sup>−6</sup>. 655,658 autosomal SNPs satisfied these criteria. These genotyped SNPs were used for genome-wide analysis of the metabolic traits. For selection of the best associated SNP in a meta-analysis of KORA and TwinsUK within a region we used genotyped as well dosages of imputed SNPs. In KORA F4 imputation was done using IMPUTE v0.4.2 <sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687829">44</a></sup> based on HapMap2 (see below).</p></div><div id="S21" class="sec sec-last"><p></p><h5 id="S21title" class="inline">TwinsUK </h5><p id="P24" class="p p-first-last">Genotyping of the TwinsUK dataset was done with a combination of Illumina arrays (HumanHap300, HumanHap610Q, 1M-Duo and 1.2MDuo 1M) <sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687772">45</a>,<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687776">46</a></sup>. We pooled the normalised intensity data for each of the three arrays separately (with 1M-Duo and 1.2MDuo 1M pooled together). For each dataset we used the Illluminus calling algorithm <sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687814">47</a></sup> to assign genotypes in the pooled data. No calls were assigned if an individual’s most likely genotyped was called with less than a posterior probability threshold of 0.95. Validation of pooling was achieved via a visual inspection of 100 random, shared SNPs for overt batch effects. Finally, intensity cluster plots of significant SNPs were visually inspected for over-dispersion biased no calling, and/or erroneous genotype assignment. SNPs exhibiting any of these characteristics were discarded. We applied similar exclusion criteria to each of the three dataset separately. <em>Samples</em>: Exclusion criteria were: (i) sample call rate &lt;98%, (ii) heterozygosity across all SNPs ≥2 s.d. from the sample mean; (iii) evidence of non-European ancestry as assessed by PCA comparison with HapMap3 populations; (iv) observed pairwise IBD probabilities suggestive of sample identity errors; (v). We corrected misclassified monozygotic and dizygotic twins based on IBD probabilities. <em>SNPs</em>. Exclusion criteria were (i) Hardy-Weinberg p-value&lt;10<sup>−6</sup>, assessed in a set of unrelated samples; (ii) MAF&lt;1%, assessed in a set of unrelated samples; (iii) SNP call rate &lt;97% (SNPs with MAF≥5%) or &lt; 99% (for 1% ≤MAF &lt; 5%). Alleles of all three datasets were aligned to HapMap2 or HapMap3 fwd strand alleles. Prior to merging, we performed pairwise comparison among the three datasets and further excluded SNPs and samples to avoid spurious genotyping effects, indentified as follows: (i) concordance at duplicate samples &lt;1%; (ii) concordance at duplicate SNPs &lt;1%; (iii) visual inspection of QQ plots for logistic regression applied to all pairwise dataset comparisons; (iv) Hardy-Weinberg p-value&lt;10<sup>−6</sup>, assessed in a set of unrelated samples; (v) observed pairwise IBD probabilities suggestive of sample identity errors. We then merged the three datasets, keeping individuals typed at the largest number of SNPs when an individual was typed at two different arrays. The merged dataset consists of 5,654 individuals (2,040 from the HumanHap300, 3,461 from the HumanHap610Q and 153 from the HumanHap1M and 1.M arrays) and up to 874,733 SNPs depending on the dataset (HumanHap300: 303,940, HumanHap610Q: 553,487, HumanHap1M and 1.M: 874,733). Imputation was performed using the IMPUTE software package (v2) <sup><a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687820">44</a></sup> using two reference panels, P0 (HapMap2, rel 22, combined CEU+YRI+ASN panels) and P1 (610k+, including the combined HumanHap610k and 1M reduced to 610k SNP content). 534,665 autosomal SNPs were used for the analysis of this study (basically 610K SNPs extracted from the final merged data set).</p></div></div></div><div id="S22" class="sec sec-last"><h3 id="S22title">3. DATA ANALYSIS</h3><div id="S23" class="sec sec-first"><p></p><h4 id="S23title" class="inline">Statistical analyses </h4><p id="P25" class="p p-first-last">The primary association testing was carried out using linear regressions on all metabolite concentrations and all possible ratios of metabolite concentrations. This was motivated by our previous observation <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687817">4</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687771">8</a></sup> that the use of ratios may lead to a strong reduction in the overall trait variance. A test of normality showed that in 29,338 cases the log-transformed ratio distribution was significantly better represented by a normal distribution than when untransformed ratios were used. In 5,145 cases untransformed distribution was closer to a normal distribution. For concentrations 149 were closer to a lognormal distribution while 124 were better represented by a normal distribution. Based on this observation, and also for sake of simplicity, we decided to log-transform all metabolites and their ratios. We used the p-gain statistics <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687770">4</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687775">8</a></sup> to quantify the decrease in p-value for the association with the ratio compared to the p-values of the two corresponding concentrations. A high p-gain (above 250) indicates that two metabolites are more likely to be functionally linked in a metabolic pathway that is impacted the associating genotype. KORA and TwinsUK are population-based studies. They comprise only individuals who are not displaying any severe clinical symptoms at the time of sampling. Therefore, disease state has not considered as a confounding factor in the statistical analysis. In KORA, the software PLINK (version 1.06)<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687780">48</a></sup> and SNPTEST was used with age and gender as covariates. In order to account for the family structure in the TwinsUK study, we used variance components applied to a score test implemented in the software Merlin <sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687789">49</a></sup>.</p></div><div id="S24" class="sec"><p></p><h4 id="S24title" class="inline">Correction for multiple testing </h4><p id="P26" class="p p-first-last">We applied a conservative Bonferroni correction to control for false positive error rates deriving from multiple testing. Using the KORA study as reference, we corrected for tests on 655,658 SNPs and 37,455 metabolic traits, thus obtaining a Bonferroni-adjusted p-value of p = 2.04×10<sup>−12</sup>. For ratios we required in addition that the increase in the strength of association, expressed as the change in p-value when using ratios compared to the larger of the two p-values when using two metabolite concentrations individually (p-gain), be larger than the number of tested metabolic traits (p-gain&gt;250) <sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687787">4</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687824">8</a></sup>. This limit is considered as a Bonferroni-type conservative cut-off for identifying those metabolite concentration pairs for which the use of ratios strongly improves the strength of association. Others than the strongest associating metabolic trait often provide additional insight into the underlying biochemical processes. In such cases we consider a p-value of p = 1.33×10<sup>−6</sup> to represent a conservative level of significance (Bonferroni correction for 37,455 tests at a nominal significance level of 5%).</p></div><div id="S25" class="sec"><p></p><h4 id="S25title" class="inline">Inflation </h4><p id="P27" class="p p-first-last">In most cases the assumption of a linear additive model was valid (see box plots in <strong>Supplemental Figure 3</strong>) and there was no inflation of summary statistics which could be indicative of population stratification (see QQ-plots in <strong>Supplemental Figure 3</strong>). Lambda values ranged from 0.965 to 1.024 (median=1.006) in KORA and from 0.940 to 1.013 (median=0.985) in TwinsUK.</p></div><div id="S26" class="sec"><p></p><h4 id="S26title" class="inline">Candidate gene selection and overlap with disease loci </h4><p id="P28" class="p p-first-last">Regional association plots (<strong>Supplemental Figure 3)</strong> were created using imputed and meta-analyzed data. Within this region the SNP with the strongest signal of association in the meta-analysis was retained as the final SNP to be reported. Association data for all metabolic traits at the 37 SNPs reported in <a href="/pmc/articles/PMC3832838/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> (for KORA, TwinsUK and meta-analysis), limited to associations with p&lt;1.33×10<sup>−6</sup> (Bonferroni correction for multiple testing of metabolic traits at a single locus) and p-gain&gt;250 (for ratios) in the meta-analysis are reported in <strong>Supplemental Table 4.</strong> For the strongest associating trait box plots were plotted to visualize the actual quantitative dependence of the trait on genotype (<strong>Supplemental Figure 3</strong>). Based on association data alone, it is in most cases not possible to identify the implicated gene within a locus that causes the association. However, using knowledge on the function of genes within linkage disequilibrium of the reported SNP as well as the biochemical characteristic of the associating metabolite, it is in many cases possible to identify a single most likely candidate gene. These cases are tagged as ‘match between gene function and metabolic trait’ and are supported by arguments provided as supplemental text (e.g. association of a SNP in LD with <em>OPLAH</em> (oxoprolinase) and oxoproline concentrations). At two loci (<em>CYP4A</em> and <em>UGT1A</em>) variants with alternative splice variants exist. We named these loci without attempting to specify the exact variant.</p></div><div id="S27" class="sec"><p></p><h4 id="S27title" class="inline">GWAS catalog </h4><p id="P29" class="p p-first-last">Using the catalogue of published genome-wide association studies (accessed 10 October 2010) <sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687795">1</a></sup> we identified for each entry the SNPs in the KORA and TwinsUK studies that correlate most strongly (r<sup>2</sup> ≥ 0.5) and that was present in our association database (p&lt;10<sup>−3</sup>, p-gain&gt;10). The resulting associations are available online on our GWAS server. New associations shall be included as the database of published GWAS is updated.</p></div><div id="S28" class="sec"><p></p><h4 id="S28title" class="inline">Enrichment analysis </h4><p id="P30" class="p p-first">We downloaded the actual version of the GWAS catalogue from NHGRI and deleted all records that correspond to our previous studies. As a sampling dataset, we chose the 655,658 SNP from the Affymetrix 6.0 array, which have been tested in the KORA part of this study. The 37 SNPs that we report are from this array and can thus be considered as representing one draw out of this set. We then drew 1,000,000 sets of 37 SNPs at random (with replacement) from this sampling dataset. To account for comparable MAF distributions between the reference and the random set we then rejected all draws where the mean or the variance of the MAF distributions were significantly different (p&lt;0.05) between the random and the reference set. 330,775 random sets were hence retained. Using an LD criterion of r<sup>2</sup>&gt;0.8 (based on HapMap2 release #27, NCBI B36, CEU population), we then counted for every random set the overlap with the GWAS catalogue. The reference set was included as a technical positive control in the computations. For the 330,775 tested random sets, at most six overlapping SNPs were found (8 times), and in over half of the cases no overlapping SNPs were present in the sampled dataset (see Table below).
</p><table class="array small"><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">number of SNPs<br>overlapping with<br>the NHGRI GWAS<br>catalogue</td><td rowspan="1" colspan="1" valign="middle" align="center">number of random<br>occurrences</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">182,924</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">109,288</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">31,744</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">3</td><td rowspan="1" colspan="1" valign="middle" align="center">5,931</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">778</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">102</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">total</td><td rowspan="1" colspan="1" valign="middle" align="center">330,775</td></tr></tbody></table><p></p><p id="P31" class="p p-last">For our reported 37 metabolomics SNPs, we identified 14 overlapping SNPs (note that we report 15 overlapping loci <a href="/pmc/articles/PMC3832838/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>; the <em>ENPEP</em> locus was not yet included in the GWAS catalogue and was not used in this analysis). As we never found 14 overlapping loci by chance, the p-value of our observations being due to chance is below p = 1/330,775 = 3×10<sup>−6</sup>.</p></div><div id="S29" class="sec"><p></p><h4 id="S29title" class="inline">Functional characterization of SLC16A9 </h4><p id="P32" class="p p-first-last">The <em>SLC16A9</em> (<em>MCT9</em>) clone (IMAGE ID 40146598) was purchased from Autogen Bioclear (Wiltshire, UK). Plasmid was linearised with SpeI restriction enzyme (New englan Biolabs, UK) and cRNA synthesised in vitro using the T7 mMachine in vitro transcription system (Ambion, Applied Biosystems, Warrington, UK). <em>MCT9</em> was expressed in <em>Xenopus laevis</em> oocytes as described previously (Meredith 2004). Briefly, stage V-VI oocytes were injected with 10ng of <em>MCT9</em> cRNA and incubated in modified Barth’s solution for 3-4 days at 18°C with the medium changed daily. Control oocytes had either no injection (NI) or an injection of an equal volume (50nl) of distilled H<sub>2</sub>O (WI) and were incubated for the same length of time. Uptake and efflux experiments were performed similarly to those described previously <sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_341687774">50</a></sup> except the substrate was [<sup>3</sup>H]-carnitine (specific activity 81Ci/mmol, GE Healthcare, UK).</p></div><div id="S30" class="sec sec-last"><p></p><h4 id="S30title" class="inline">Data access </h4><p id="P33" class="p p-first-last">This study generated millions of individual data points through the profiling of <em>n</em> metabolites and <em>n</em>*(<em>n</em>-1)/2 ratios in ~3,000 individuals, and the subsequent associations with millions of genetic variants from GWAS. We created a web-based interface and visualization tools for the dissemination of results to the scientific community, with the aims of allowing rapid storage and retrieval of data as well as managing the integration of metabolomics summary statistics vis-a-vis published GWAS studies. The association data is freely available at through the server <a href="http://metabolomics.helmholtz-muenchen.de/gwa/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://metabolomics.helmholtz-muenchen.de/gwa/</a> and mirror sites located at the Wellcome Trust Sanger Institute and King’s College London sites.</p></div></div></div><div id="S31" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S31title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p1">We gratefully acknowledge the contributions of P. Lichtner, G. Eckstein, Guido Fischer, T. Strom and all other members of the Helmholtz Zentrum München genotyping staff in generating the SNP dataset, as well as the contribution of all members of field staffs who were involved in the planning and conduct of the MONICA/KORA Augsburg studies. The KORA group consists of H.E. Wichmann (speaker), A. Peters, C. Meisinger, T. Illig, R. Holle, J. John and their co-workers who are responsible for the design and conduct of the KORA studies. For TwinsUK we thank the staff from the Genotyping Facilities at the Wellcome Trust Sanger Institute for sample preparation, quality control and genotyping. Guido Fischer (KORA) and Gabriela Surdulescu (TwinsUK) selected the samples, sample handling and shipment was organized by Humberto Chavez (KORA) and Dylan Hodgkiss (TwinsUK), and Ulrike Goebel (Helmholtz) provided administrative support. Special thanks go to Daniel Garcia-West for his role in facilitating this study. We are grateful to the CARDIoGRAM investigators for access to their dataset. Finally, we wish to express our appreciation to all study participants of the KORA and the TwinsUK studies for donating their blood and time.</p><p id="__p2"><strong>FUNDING</strong> The KORA research platform (KORA: <em>Kooperative Gesundheitsforschung in der Region Augsburg</em>) and the MONICA Augsburg studies (Monitoring trends and determinants on cardiovascular diseases) were initiated and financed by the <em>Helmholtz Zentrum München - National Research Center for Environmental Health</em>, which is funded by the German Federal Ministry of Education, Science, Research and Technology and by the State of Bavaria. This study was supported by a grant from the German Federal Ministry of Education and Research (BMBF) to the German Center for Diabetes Research (DZD e.V.). Part of this work was financed by the German National Genome Research Network (NGFNPlus: 01GS0823). Computing resources have been made available by the Leibniz Supercomputing Centre of the Bavarian Academy of Sciences and Humanities (HLRB project h1231) and the DEISA Extreme Computing Initiative (project PHAGEDA). Part of this research was supported within the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. The TwinsUK study was funded by the Wellcome Trust; European Community’s Seventh Framework Programme (FP7/2007-2013)/grant agreement HEALTH-F2-2008-201865-GEFOS and (FP7/2007-2013) and the FP-5 GenomEUtwin Project (QLG2-CT-2002-01254). The study also receives support from the Dept of Health via the National Institute for Health Research (NIHR) comprehensive Biomedical Research Centre award to Guy’s &amp; St Thomas’ NHS Foundation Trust in partnership with King’s College London. TDS is an NIHR senior Investigator. The project also received support from a <em>Biotechnology and Biological Sciences Research Council</em> (BBSRC) project grant (<a id="__tag_783289445" class="tag_hotlink" href="/nuccore/G20234" ref="/nuccore/G20234"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"G20234","term_id":"1340584","term_text":"G20234"}}</span>G20234</a>). Both studies received support from ENGAGE project grant agreement HEALTH-F4-2007-201413. NJS holds a British Heart Foundation Chair and is an NIHR Senior Investigator and is supported by the Leicester NIHR Biomedical Research Unit in Cardiovascular Disease. The authors acknowledge the funding and support of the National Eye Institute via an NIH/CIDR genotyping project (PI: Terri Young). Genotyping was also performed by CIDR as part of an NEI/NIH project grant. DM received support from the Early Career Researcher Scheme, Oxford Brookes University, UK. JR is supported by DFG Graduiertenkolleg “GRK 1563, Regulation and Evolution of Cellular Systems” (RECESS), EA by BMBF Grant no. 0315494A (project SysMBo), WRM by BMBF grant no. 03IS2061B (project Gani_Med), and BW by Era-Net grant no. 0315442A (project PathoGenoMics). AK is supported by the Emmy Noether Programme of the DFG (KO-3598/2-1) and FK by grants from the “Genomics of Lipid-associated Disorders – GOLD” of the “Austrian Genome Research Programme GEN-AU”. NS is supported by the Wellcome Trust (Core Grant Number 091746/Z/10/Z).</p></div></div><div id="idm140353680345824" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353680345824title"><span class="title" id="idm140353680345824title">AUTHOR CONTRIBUTION (in alphabetic order) </span></h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P1" class="p p-first"><strong>Designed the study:</strong> Jerzy Adamski, Christian Gieger, Thomas Illig, David Meredith, Nicole Soranzo, Karsten Suhre</p><p id="P2" class="p"><strong>Conducted the experiments:</strong> David Meredith, Michael V. Milburn, Robert P. Mohney</p><p id="P3" class="p"><strong>Analyzed the data:</strong> Jerzy Adamski, Elisabeth Altmaier, Christian Gieger, Gabi Kastenmüller, Anna Köttgen, Florian Kronenberg, Christa Meisinger, David Meredith, Ann-Kristin Petersen, Cornelia Prehn, Johannes Raffler, Janina S. Ried, Werner Römisch-Margl, So-Youn Shin, Karsten Suhre, Brigitte Wägele</p><p id="P4" class="p"><strong>Provided material / data / analysis tools:</strong> The CARDIoGRAM consortium, Panos Deloukas, Jeanette Erdmann, Elin Grundberg, Christopher J. Hammond, Martin Hrabé de Angelis, Thomas Illig, Massimo Mangino, Thomas Meitinger, Hans-Werner Mewes, Nilesh Samani, Kerrin S. Small, Tim D. Spector, H.-Erich Wichmann, Guangju Zhai</p><p id="P5" class="p p-last"><strong>Wrote the paper:</strong> Christian Gieger, Nicole Soranzo, Karsten Suhre</p><p></p></div></div><div id="idm140353758767488" class="tsec sec headless whole_rhythm"><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P34" class="p p-first-last"><strong>DISCLOSURES AND COMPETING FINANCIAL INTERESTS</strong> Michael V. Milburn and Robert P. Mohney are employees of Metabolon Inc.</p><p></p><p class="fn sec" id="FN3"></p><p id="P35" class="p p-first-last"><strong>Web links</strong>
</p><ul class="unordered" style="list-style-type:disc" id="L1"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P36">GWAS server: <a href="http://metabolomics.helmholtz-muenchen.de/gwa/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://metabolomics.helmholtz-muenchen.de/gwa/</a></p></li><li><p id="P37">SNAP: <a href="http://www.broadinstitute.org/mpg/snap/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/snap/</a></p></li><li><p id="P38">NHGRI catalog of published GWAS: <a href="http://www.genome.gov/gwastudies/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/gwastudies/</a></p></li><li><p id="P39">eQTL : <a href="http://www.sanger.ac.uk/Software/analysis/genevar/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sanger.ac.uk/Software/analysis/genevar/</a></p></li><li><p id="P40">GRAIL : <a href="http://www.broadinstitute.org/mpg/grail/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/grail/</a></p></li><li><p id="P41">IPA : Ingenuity Pathway Analysis : <a href="http://ingenuity.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ingenuity.com</a></p></li><li><p id="P42">OMIM : <a href="http://www.ncbi.nlm.nih.gov/omim" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/omim</a></p></li><li><p id="P43">yED network editor : <a href="http://www.yworks.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.yworks.com</a></p></li><li><p id="P44">BioGPS : <a href="http://biogps.gnf.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biogps.gnf.org</a></p></li><li><p id="P45">Genecards : <a href="http://www.genecards.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genecards.org</a></p></li><li><p id="P46">WikiGenes : <a href="http://www.wikigenes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wikigenes.org</a></p></li><li><p id="P47">Pharmacogenomics Knowledge Base : <a href="http://www.pharmgkb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.pharmgkb.org</a></p></li><li><p id="P48">R statistical analysis system: <a href="http://www.r-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org</a></p></li><li><p id="P49">KORA study population : <a href="http://www.helmholtz-muenchen.de/kora/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.helmholtz-muenchen.de/kora/</a></p></li><li><p id="P50">TwinsUK study : <a href="http://www.twinsuk.ac.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.twinsuk.ac.uk</a></p></li><li><p id="P51">Metabolon Inc. : <a href="http://www.metabolon.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.metabolon.com</a></p></li><li><p id="P52">MERLIN: <a href="http://www.sph.umich.edu/csg/abecasis/Merlin" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Merlin</a></p></li><li><p id="P53">PLINK: <a href="http://pngu.mgh.harvard.edu/~purcell/plink" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink</a></p></li><li><p id="P54">R: <a href="http://www.r-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org</a></p></li><li><p id="P55">SNPTEST: <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a></p></li></ul><p></p><p></p></div></div><div id="idm140353680330528" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140353680330528title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Hindorff LA, et al.  Potential etiologic and functional implications of genome-wide association loci for human diseases and traits. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Potential+etiologic+and+functional+implications+of+genome-wide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;pmid=19474294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Newgard CB, Attie AD. Getting biological about the genetics of diabetes. <span><span class="ref-journal">Nat Med. </span>2010;<span class="ref-vol">16</span>:388–391.</span> [<a href="/pubmed/20376050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=Getting+biological+about+the+genetics+of+diabetes&amp;author=CB+Newgard&amp;author=AD+Attie&amp;volume=16&amp;publication_year=2010&amp;pages=388-391&amp;pmid=20376050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Illig T, et al.  A genome-wide perspective of genetic variation in human metabolism. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:137–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773904/">PMC free article</a>]</span> [<a href="/pubmed/20037589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+perspective+of+genetic+variation+in+human+metabolism&amp;author=T+Illig&amp;volume=42&amp;publication_year=2010&amp;pages=137-141&amp;pmid=20037589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gieger C, et al.  Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+a+genome-wide+association+study+of+metabolite+profiles+in+human+serum&amp;author=C+Gieger&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Evans AM, DeHaven CD, Barrett T, Mitchell M, Milgram E. Integrated, Nontargeted Ultrahigh Performance Liquid Chromatography/Electrospray Ionization Tandem Mass Spectrometry Platform for the Identification and Relative Quantification of the Small-Molecule Complement of Biological Systems. <span><span class="ref-journal">Analytical Chemistry. </span>2009;<span class="ref-vol">81</span>:6656–6667.</span> [<a href="/pubmed/19624122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Analytical+Chemistry&amp;title=Integrated,+Nontargeted+Ultrahigh+Performance+Liquid+Chromatography/Electrospray+Ionization+Tandem+Mass+Spectrometry+Platform+for+the+Identification+and+Relative+Quantification+of+the+Small-Molecule+Complement+of+Biological+Systems&amp;author=AM+Evans&amp;author=CD+DeHaven&amp;author=T+Barrett&amp;author=M+Mitchell&amp;author=E+Milgram&amp;volume=81&amp;publication_year=2009&amp;pages=6656-6667&amp;pmid=19624122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Ohta T, et al.  Untargeted metabolomic profiling as an evaluative tool of fenofibrate-induced toxicology in Fischer 344 male rats. <span><span class="ref-journal">Toxicol Pathol. </span>2009;<span class="ref-vol">37</span>:521–535.</span> [<a href="/pubmed/19458390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Toxicol+Pathol&amp;title=Untargeted+metabolomic+profiling+as+an+evaluative+tool+of+fenofibrate-induced+toxicology+in+Fischer+344+male+rats&amp;author=T+Ohta&amp;volume=37&amp;publication_year=2009&amp;pages=521-535&amp;pmid=19458390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Suhre K, et al.  Metabolic footprint of diabetes: a multiplatform metabolomics study in an epidemiological setting. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e13953.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978704/">PMC free article</a>]</span> [<a href="/pubmed/21085649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Metabolic+footprint+of+diabetes:+a+multiplatform+metabolomics+study+in+an+epidemiological+setting&amp;author=K+Suhre&amp;volume=5&amp;publication_year=2010&amp;pages=e13953&amp;pmid=21085649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Altmaier E, et al.  Bioinformatics analysis of targeted metabolomics--uncovering old and new tales of diabetic mice under medication. <span><span class="ref-journal">Endocrinology. </span>2008;<span class="ref-vol">149</span>:3478–3489.</span> [<a href="/pubmed/18372322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=Bioinformatics+analysis+of+targeted+metabolomics--uncovering+old+and+new+tales+of+diabetic+mice+under+medication&amp;author=E+Altmaier&amp;volume=149&amp;publication_year=2008&amp;pages=3478-3489&amp;pmid=18372322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Raychaudhuri S, et al.  Identifying relationships among genomic disease regions: predicting genes at pathogenic SNP associations and rare deletions. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000534.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694358/">PMC free article</a>]</span> [<a href="/pubmed/19557189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Identifying+relationships+among+genomic+disease+regions:+predicting+genes+at+pathogenic+SNP+associations+and+rare+deletions&amp;author=S+Raychaudhuri&amp;volume=5&amp;publication_year=2009&amp;pages=e1000534&amp;pmid=19557189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Chambers JC, et al.  Genetic loci influencing kidney function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:373–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3748585/">PMC free article</a>]</span> [<a href="/pubmed/20383145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+loci+influencing+kidney+function+and+chronic+kidney+disease&amp;author=JC+Chambers&amp;volume=42&amp;publication_year=2010&amp;pages=373-375&amp;pmid=20383145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Kottgen A, et al.  New loci associated with kidney function and chronic kidney disease. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:376–384.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2997674/">PMC free article</a>]</span> [<a href="/pubmed/20383146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+loci+associated+with+kidney+function+and+chronic+kidney+disease&amp;author=A+Kottgen&amp;volume=42&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20383146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Dupuis J, et al.  New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:105–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3018764/">PMC free article</a>]</span> [<a href="/pubmed/20081858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+genetic+loci+implicated+in+fasting+glucose+homeostasis+and+their+impact+on+type+2+diabetes+risk&amp;author=J+Dupuis&amp;volume=42&amp;publication_year=2010&amp;pages=105-116&amp;pmid=20081858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Aulchenko YS, et al.  Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:47–55.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;author=YS+Aulchenko&amp;volume=41&amp;publication_year=2009&amp;pages=47-55&amp;pmid=19060911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Panneerselvam K, Freeze HH. Mannose enters mammalian cells using a specific transporter that is insensitive to glucose. <span><span class="ref-journal">J Biol Chem. </span>1996;<span class="ref-vol">271</span>:9417–9421.</span> [<a href="/pubmed/8621609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Mannose+enters+mammalian+cells+using+a+specific+transporter+that+is+insensitive+to+glucose&amp;author=K+Panneerselvam&amp;author=HH+Freeze&amp;volume=271&amp;publication_year=1996&amp;pages=9417-9421&amp;pmid=8621609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Taguchi T, et al.  Hepatic glycogen breakdown is implicated in the maintenance of plasma mannose concentration. <span><span class="ref-journal">Am J Physiol Endocrinol Metab. </span>2005;<span class="ref-vol">288</span>:E534–540.</span> [<a href="/pubmed/15536204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Endocrinol+Metab&amp;title=Hepatic+glycogen+breakdown+is+implicated+in+the+maintenance+of+plasma+mannose+concentration&amp;author=T+Taguchi&amp;volume=288&amp;publication_year=2005&amp;pages=E534-540&amp;pmid=15536204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Blombaeck B, Blombaeck M, Edman P, Hessel B. Amino-acid sequence and the occurrence of phosphorus in human fibrinopeptides. <span><span class="ref-journal">Nature. </span>1962;<span class="ref-vol">193</span>:833–834.</span> [<a href="/pubmed/13870090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Amino-acid+sequence+and+the+occurrence+of+phosphorus+in+human+fibrinopeptides&amp;author=B+Blombaeck&amp;author=M+Blombaeck&amp;author=P+Edman&amp;author=B+Hessel&amp;volume=193&amp;publication_year=1962&amp;pages=833-834&amp;pmid=13870090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Martin SC, Ekman P, Forsberg PO, Ersmark H. Increased phosphate content of fibrinogen in vivo correlates with alteration in fibrinogen behaviour. <span><span class="ref-journal">Thromb Res. </span>1992;<span class="ref-vol">68</span>:467–473.</span> [<a href="/pubmed/1341057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thromb+Res&amp;title=Increased+phosphate+content+of+fibrinogen+in+vivo+correlates+with+alteration+in+fibrinogen+behaviour&amp;author=SC+Martin&amp;author=P+Ekman&amp;author=PO+Forsberg&amp;author=H+Ersmark&amp;volume=68&amp;publication_year=1992&amp;pages=467-473&amp;pmid=1341057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Yuan X, et al.  Population-based genome-wide association studies reveal six loci influencing plasma levels of liver enzymes. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:520–528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561937/">PMC free article</a>]</span> [<a href="/pubmed/18940312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Population-based+genome-wide+association+studies+reveal+six+loci+influencing+plasma+levels+of+liver+enzymes&amp;author=X+Yuan&amp;volume=83&amp;publication_year=2008&amp;pages=520-528&amp;pmid=18940312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Tregouet DA, et al.  Common susceptibility alleles are unlikely to contribute as strongly as the FV and ABO loci to VTE risk: results from a GWAS approach. <span><span class="ref-journal">Blood. </span>2009;<span class="ref-vol">113</span>:5298–5303.</span> [<a href="/pubmed/19278955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Common+susceptibility+alleles+are+unlikely+to+contribute+as+strongly+as+the+FV+and+ABO+loci+to+VTE+risk:+results+from+a+GWAS+approach&amp;author=DA+Tregouet&amp;volume=113&amp;publication_year=2009&amp;pages=5298-5303&amp;pmid=19278955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Teslovich TM, et al.  Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=TM+Teslovich&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Schunkert H, et al.  Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:333–338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3119261/">PMC free article</a>]</span> [<a href="/pubmed/21378990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Large-scale+association+analysis+identifies+13+new+susceptibility+loci+for+coronary+artery+disease&amp;author=H+Schunkert&amp;volume=43&amp;publication_year=2011&amp;pages=333-338&amp;pmid=21378990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Doring A, et al.  SLC2A9 influences uric acid concentrations with pronounced sex-specific effects. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:430–436.</span> [<a href="/pubmed/18327256" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=SLC2A9+influences+uric+acid+concentrations+with+pronounced+sex-specific+effects&amp;author=A+Doring&amp;volume=40&amp;publication_year=2008&amp;pages=430-436&amp;pmid=18327256&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Caulfield MJ, et al.  SLC2A9 is a high-capacity urate transporter in humans. <span><span class="ref-journal">PLoS Med. </span>2008;<span class="ref-vol">5</span>:e197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561076/">PMC free article</a>]</span> [<a href="/pubmed/18842065" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med&amp;title=SLC2A9+is+a+high-capacity+urate+transporter+in+humans&amp;author=MJ+Caulfield&amp;volume=5&amp;publication_year=2008&amp;pages=e197&amp;pmid=18842065&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Klein TE, et al.  Integrating genotype and phenotype information: an overview of the PharmGKB project. Pharmacogenetics Research Network and Knowledge Base. <span><span class="ref-journal">Pharmacogenomics J. </span>2001;<span class="ref-vol">1</span>:167–170.</span> [<a href="/pubmed/11908751" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J&amp;title=Integrating+genotype+and+phenotype+information:+an+overview+of+the+PharmGKB+project.+Pharmacogenetics+Research+Network+and+Knowledge+Base&amp;author=TE+Klein&amp;volume=1&amp;publication_year=2001&amp;pages=167-170&amp;pmid=11908751&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Deeken JF, et al.  A pharmacogenetic study of docetaxel and thalidomide in patients with castration-resistant prostate cancer using the DMET genotyping platform. <span><span class="ref-journal">Pharmacogenomics J. </span>2010;<span class="ref-vol">10</span>:191–199.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6631360/">PMC free article</a>]</span> [<a href="/pubmed/20038957" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J&amp;title=A+pharmacogenetic+study+of+docetaxel+and+thalidomide+in+patients+with+castration-resistant+prostate+cancer+using+the+DMET+genotyping+platform&amp;author=JF+Deeken&amp;volume=10&amp;publication_year=2010&amp;pages=191-199&amp;pmid=20038957&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Lankisch TO, et al.  Gilbert’s Syndrome and irinotecan toxicity: combination with UDP-glucuronosyltransferase 1A7 variants increases risk. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2008;<span class="ref-vol">17</span>:695–701.</span> [<a href="/pubmed/18349289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Gilbert’s+Syndrome+and+irinotecan+toxicity:+combination+with+UDP-glucuronosyltransferase+1A7+variants+increases+risk&amp;author=TO+Lankisch&amp;volume=17&amp;publication_year=2008&amp;pages=695-701&amp;pmid=18349289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Huang RS, et al.  A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2007;<span class="ref-vol">104</span>:9758–9763.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1887589/">PMC free article</a>]</span> [<a href="/pubmed/17537913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+genome-wide+approach+to+identify+genetic+variants+that+contribute+to+etoposide-induced+cytotoxicity&amp;author=RS+Huang&amp;volume=104&amp;publication_year=2007&amp;pages=9758-9763&amp;pmid=17537913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Chen Y, et al.  Effect of genetic variation in the organic cation transporter 2 on the renal elimination of metformin. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2009;<span class="ref-vol">19</span>:497–504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3104496/">PMC free article</a>]</span> [<a href="/pubmed/19483665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Effect+of+genetic+variation+in+the+organic+cation+transporter+2+on+the+renal+elimination+of+metformin&amp;author=Y+Chen&amp;volume=19&amp;publication_year=2009&amp;pages=497-504&amp;pmid=19483665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Shu Y, et al.  Effect of genetic variation in the organic cation transporter 1, OCT1, on metformin pharmacokinetics. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">83</span>:273–280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2976713/">PMC free article</a>]</span> [<a href="/pubmed/17609683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Effect+of+genetic+variation+in+the+organic+cation+transporter+1,+OCT1,+on+metformin+pharmacokinetics&amp;author=Y+Shu&amp;volume=83&amp;publication_year=2008&amp;pages=273-280&amp;pmid=17609683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Link E, et al.  SLCO1B1 variants and statin-induced myopathy--a genomewide study. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">359</span>:789–799.</span> [<a href="/pubmed/18650507" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=SLCO1B1+variants+and+statin-induced+myopathy--a+genomewide+study&amp;author=E+Link&amp;volume=359&amp;publication_year=2008&amp;pages=789-799&amp;pmid=18650507&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Davies NJ, et al.  AKR1C isoforms represent a novel cellular target for jasmonates alongside their mitochondrial-mediated effects. <span><span class="ref-journal">Cancer Res. </span>2009;<span class="ref-vol">69</span>:4769–4775.</span> [<a href="/pubmed/19487289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=AKR1C+isoforms+represent+a+novel+cellular+target+for+jasmonates+alongside+their+mitochondrial-mediated+effects&amp;author=NJ+Davies&amp;volume=69&amp;publication_year=2009&amp;pages=4769-4775&amp;pmid=19487289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Sanna S, et al.  Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:2711–2718.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701337/">PMC free article</a>]</span> [<a href="/pubmed/19419973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+variants+in+the+SLCO1B3+locus+are+associated+with+bilirubin+levels+and+unconjugated+hyperbilirubinemia&amp;author=S+Sanna&amp;volume=18&amp;publication_year=2009&amp;pages=2711-2718&amp;pmid=19419973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Johnson AD, et al.  Genome-wide association meta-analysis for total serum bilirubin levels. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:2700–2710.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701336/">PMC free article</a>]</span> [<a href="/pubmed/19414484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+meta-analysis+for+total+serum+bilirubin+levels&amp;author=AD+Johnson&amp;volume=18&amp;publication_year=2009&amp;pages=2700-2710&amp;pmid=19414484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Kolz M, et al.  Meta-analysis of 28,141 individuals identifies common variants within five new loci that influence uric acid concentrations. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683940/">PMC free article</a>]</span> [<a href="/pubmed/19503597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Meta-analysis+of+28,141+individuals+identifies+common+variants+within+five+new+loci+that+influence+uric+acid+concentrations&amp;author=M+Kolz&amp;volume=5&amp;publication_year=2009&amp;pages=e1000504&amp;pmid=19503597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zhai G, et al.  Eight Common Genetic Variants Associated with Serum DHEAS Levels Suggest a Key Role in Ageing Mechanisms. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002025.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077384/">PMC free article</a>]</span> [<a href="/pubmed/21533175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Eight+Common+Genetic+Variants+Associated+with+Serum+DHEAS+Levels+Suggest+a+Key+Role+in+Ageing+Mechanisms&amp;author=G+Zhai&amp;volume=7&amp;publication_year=2011&amp;pages=e1002025&amp;pmid=21533175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Mootha VK, Hirschhorn JN. Inborn variation in metabolism. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:97–98.</span> [<a href="/pubmed/20104246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Inborn+variation+in+metabolism&amp;author=VK+Mootha&amp;author=JN+Hirschhorn&amp;volume=42&amp;publication_year=2010&amp;pages=97-98&amp;pmid=20104246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Meredith D, Christian HC. The SLC16 monocaboxylate transporter family. <span><span class="ref-journal">Xenobiotica. </span>2008;<span class="ref-vol">38</span>:1072–1106.</span> [<a href="/pubmed/18668440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Xenobiotica&amp;title=The+SLC16+monocaboxylate+transporter+family&amp;author=D+Meredith&amp;author=HC+Christian&amp;volume=38&amp;publication_year=2008&amp;pages=1072-1106&amp;pmid=18668440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Koepsell H, Endou H. The SLC22 drug transporter family. <span><span class="ref-journal">Pflugers Arch. </span>2004;<span class="ref-vol">447</span>:666–676.</span> [<a href="/pubmed/12883891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pflugers+Arch&amp;title=The+SLC22+drug+transporter+family&amp;author=H+Koepsell&amp;author=H+Endou&amp;volume=447&amp;publication_year=2004&amp;pages=666-676&amp;pmid=12883891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Wichmann HE, Gieger C, Illig T. KORA-gen--resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(Suppl 1):S26–30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen--resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Andrew T, et al.  Are twins and singletons comparable? A study of disease-related and lifestyle characteristics in adult women. <span><span class="ref-journal">Twin Res. </span>2001;<span class="ref-vol">4</span>:464–477.</span> [<a href="/pubmed/11780939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res&amp;title=Are+twins+and+singletons+comparable?+A+study+of+disease-related+and+lifestyle+characteristics+in+adult+women&amp;author=T+Andrew&amp;volume=4&amp;publication_year=2001&amp;pages=464-477&amp;pmid=11780939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Lawton KA, et al.  Analysis of the adult human plasma metabolome. <span><span class="ref-journal">Pharmacogenomics. </span>2008;<span class="ref-vol">9</span>:383–397.</span> [<a href="/pubmed/18384253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Analysis+of+the+adult+human+plasma+metabolome&amp;author=KA+Lawton&amp;volume=9&amp;publication_year=2008&amp;pages=383-397&amp;pmid=18384253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Sreekumar A, et al.  Metabolomic profiles delineate potential role for sarcosine in prostate cancer progression. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">457</span>:910–914.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2724746/">PMC free article</a>]</span> [<a href="/pubmed/19212411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Metabolomic+profiles+delineate+potential+role+for+sarcosine+in+prostate+cancer+progression&amp;author=A+Sreekumar&amp;volume=457&amp;publication_year=2009&amp;pages=910-914&amp;pmid=19212411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Soranzo N, et al.  A genome-wide meta-analysis identifies 22 loci associated with eight hematological parameters in the HaemGen consortium. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1182–1190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3108459/">PMC free article</a>]</span> [<a href="/pubmed/19820697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+meta-analysis+identifies+22+loci+associated+with+eight+hematological+parameters+in+the+HaemGen+consortium&amp;author=N+Soranzo&amp;volume=41&amp;publication_year=2009&amp;pages=1182-1190&amp;pmid=19820697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Richards JB, et al.  Bone mineral density, osteoporosis, and osteoporotic fractures: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:1505–1512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2679414/">PMC free article</a>]</span> [<a href="/pubmed/18455228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Bone+mineral+density,+osteoporosis,+and+osteoporotic+fractures:+a+genome-wide+association+study&amp;author=JB+Richards&amp;volume=371&amp;publication_year=2008&amp;pages=1505-1512&amp;pmid=18455228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Soranzo N, et al.  Meta-analysis of genome-wide scans for human adult stature identifies novel Loci and associations with measures of skeletal frame size. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2661236/">PMC free article</a>]</span> [<a href="/pubmed/19343178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Meta-analysis+of+genome-wide+scans+for+human+adult+stature+identifies+novel+Loci+and+associations+with+measures+of+skeletal+frame+size&amp;author=N+Soranzo&amp;volume=5&amp;publication_year=2009&amp;pages=e1000445&amp;pmid=19343178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Teo YY, et al.  A genotype calling algorithm for the Illumina BeadArray platform. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:2741–2746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Abecasis GR, Cherny SS, Cookson WO, Cardon LR. Merlin--rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Merlin--rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=GR+Abecasis&amp;author=SS+Cherny&amp;author=WO+Cookson&amp;author=LR+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Meredith D. Site-directed mutation of arginine 282 to glutamate uncouples the movement of peptides and protons by the rabbit proton-peptide cotransporter PepT1. <span><span class="ref-journal">J Biol Chem. </span>2004;<span class="ref-vol">279</span>:15795–15798.</span> [<a href="/pubmed/14715671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Site-directed+mutation+of+arginine+282+to+glutamate+uncouples+the+movement+of+peptides+and+protons+by+the+rabbit+proton-peptide+cotransporter+PepT1&amp;author=D+Meredith&amp;volume=279&amp;publication_year=2004&amp;pages=15795-15798&amp;pmid=14715671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3832838&amp;issue-id=229910&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140353689983616title">SUMMARY</a></li><li><a href="#S1title">METHODS</a></li><li><a href="#S2title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#S12title">ONLINE METHODS</a></li><li><a href="#S31title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140353680345824title">AUTHOR CONTRIBUTION (in alphabetic order) </a></li><li><a href="#idm140353680330528title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3832838/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3832838/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3832838/pdf/emss-54209.pdf">PDF (839K)</a></li> | <li><a href="#" data-citationid="PMC3832838" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832838%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832838%2F&amp;text=Human%20metabolic%20individuality%20in%20biomedical%20and%20pharmaceutical%20research" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832838%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21886157" data-aiid="3832838" data-aid="3832838" data-iid="229910" data-domainid="358" data-domain="wtpa" data-accid="PMC3832838" data-md5="c9e0235534408dee9151c932a6eae9f1"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:38:32-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BF518DD5285C100000000078F04D5&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3832838%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3832838/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>