<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3632836/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies five new breast cancer susceptibility loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632836/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies five new breast cancer susceptibility loci"><meta name="citation_authors" content="Clare Turnbull, Shahana Ahmed, Jonathan Morrison, David Pernet, Anthony Renwick, Mel Maranian, Sheila Seal, Maya Ghoussaini, Sarah Hines, Catherine S Healey, Deborah Hughes, Margaret Warren-Perry, William Tapper, Diana Eccles, D Gareth Evans, The Breast Cancer Susceptibility Collaboration (UK), Maartje Hooning, Mieke Schutte, Ans van den Ouweland, Richard Houlston, Gillian Ross, Cordelia Langford, Paul D P Pharoah, Michael R Stratton, Alison M Dunning, Nazneen Rahman, Douglas F Easton"><meta name="citation_date" content="June 2010"><meta name="citation_issue" content="6"><meta name="citation_volume" content="42"><meta name="citation_firstpage" content="504"><meta name="citation_doi" content="10.1038/ng.586"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3632836/?report=abstract"><meta name="citation_pmid" content="20453838"><meta name="DC.Title" content="Genome-wide association study identifies five new breast cancer susceptibility loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Clare Turnbull"><meta name="DC.Contributor" content="Shahana Ahmed"><meta name="DC.Contributor" content="Jonathan Morrison"><meta name="DC.Contributor" content="David Pernet"><meta name="DC.Contributor" content="Anthony Renwick"><meta name="DC.Contributor" content="Mel Maranian"><meta name="DC.Contributor" content="Sheila Seal"><meta name="DC.Contributor" content="Maya Ghoussaini"><meta name="DC.Contributor" content="Sarah Hines"><meta name="DC.Contributor" content="Catherine S Healey"><meta name="DC.Contributor" content="Deborah Hughes"><meta name="DC.Contributor" content="Margaret Warren-Perry"><meta name="DC.Contributor" content="William Tapper"><meta name="DC.Contributor" content="Diana Eccles"><meta name="DC.Contributor" content="D Gareth Evans"><meta name="DC.Contributor" content="The Breast Cancer Susceptibility Collaboration (UK)"><meta name="DC.Contributor" content="Maartje Hooning"><meta name="DC.Contributor" content="Mieke Schutte"><meta name="DC.Contributor" content="Ans van den Ouweland"><meta name="DC.Contributor" content="Richard Houlston"><meta name="DC.Contributor" content="Gillian Ross"><meta name="DC.Contributor" content="Cordelia Langford"><meta name="DC.Contributor" content="Paul D P Pharoah"><meta name="DC.Contributor" content="Michael R Stratton"><meta name="DC.Contributor" content="Alison M Dunning"><meta name="DC.Contributor" content="Nazneen Rahman"><meta name="DC.Contributor" content="Douglas F Easton"><meta name="DC.Date" content="2010 Jun"><meta name="DC.Identifier" content="10.1038/ng.586"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies five new breast cancer susceptibility loci"><meta property="og:type" content="article"><meta property="og:description" content="Breast cancer is the most common cancer in women in developed countries. To identify common breast cancer susceptibility alleles, we conducted a genome-wide association study in which 582,886 SNPs were genotyped in 3,659 cases with a family history of ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3632836/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3632836/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3632836/pdf/emss-52673.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88C57BDD50C0210000000004BD0369.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3632836%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="mesh">MeSH</option><option value="pubmed">PubMed</option><option value="snp" class="last">SNP</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3632836</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2013 Apr 23.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20453838" target="pmc_ext" ref="reftype=publisher&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2010 Jun; 42(6): 504–507. </span></a></div><span class="fm-vol-iss-date">Published online 2010 May 9. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.586" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.586</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3632836</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS52673</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20453838">20453838</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies five new breast cancer susceptibility loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Turnbull%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528488368" co-class="co-affbox">Clare Turnbull</a>,<sup>1</sup> <a href="/pubmed/?term=Ahmed%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528486512" co-class="co-affbox">Shahana Ahmed</a>,<sup>2</sup> <a href="/pubmed/?term=Morrison%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528484976" co-class="co-affbox">Jonathan Morrison</a>,<sup>3</sup> <a href="/pubmed/?term=Pernet%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528483440" co-class="co-affbox">David Pernet</a>,<sup>1</sup> <a href="/pubmed/?term=Renwick%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528481904" co-class="co-affbox">Anthony Renwick</a>,<sup>1</sup> <a href="/pubmed/?term=Maranian%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857528480368" co-class="co-affbox">Mel Maranian</a>,<sup>2</sup> <a href="/pubmed/?term=Seal%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857458467136" co-class="co-affbox">Sheila Seal</a>,<sup>1</sup> <a href="/pubmed/?term=Ghoussaini%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857458465264" co-class="co-affbox">Maya Ghoussaini</a>,<sup>2</sup> <a href="/pubmed/?term=Hines%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857458463280" co-class="co-affbox">Sarah Hines</a>,<sup>1</sup> <a href="/pubmed/?term=Healey%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857458461296" co-class="co-affbox">Catherine S Healey</a>,<sup>2</sup> <a href="/pubmed/?term=Hughes%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453425568" co-class="co-affbox">Deborah Hughes</a>,<sup>1</sup> <a href="/pubmed/?term=Warren-Perry%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453423584" co-class="co-affbox">Margaret Warren-Perry</a>,<sup>1</sup> <a href="/pubmed/?term=Tapper%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453421600" co-class="co-affbox">William Tapper</a>,<sup>4</sup> <a href="/pubmed/?term=Eccles%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453419616" co-class="co-affbox">Diana Eccles</a>,<sup>4</sup> <a href="/pubmed/?term=Evans%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453417632" co-class="co-affbox">D Gareth Evans</a>,<sup>5</sup> The Breast Cancer Susceptibility Collaboration (UK),<sup>1,</sup><sup>10</sup> <a href="/pubmed/?term=Hooning%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453413552" co-class="co-affbox">Maartje Hooning</a>,<sup>6</sup> <a href="/pubmed/?term=Schutte%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453411568" co-class="co-affbox">Mieke Schutte</a>,<sup>6</sup> <a href="/pubmed/?term=van%20den%20Ouweland%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453409584" co-class="co-affbox">Ans van den Ouweland</a>,<sup>7</sup> <a href="/pubmed/?term=Houlston%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453407600" co-class="co-affbox">Richard Houlston</a>,<sup>1</sup> <a href="/pubmed/?term=Ross%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857453405616" co-class="co-affbox">Gillian Ross</a>,<sup>8</sup> <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454890736" co-class="co-affbox">Cordelia Langford</a>,<sup>9</sup> <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454888752" co-class="co-affbox">Paul D P Pharoah</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Stratton%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454886032" co-class="co-affbox">Michael R Stratton</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Dunning%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454883312" co-class="co-affbox">Alison M Dunning</a>,<sup>2</sup> <a href="/pubmed/?term=Rahman%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454881328" co-class="co-affbox">Nazneen Rahman</a>,<sup>1</sup> and  <a href="/pubmed/?term=Easton%20DF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838" class="affpopup" co-rid="_co_idm139857454879344" co-class="co-affbox">Douglas F Easton</a><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139857528488368"><h3 class="no_margin">Clare Turnbull</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Turnbull%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Clare Turnbull</a></div></div><div id="_co_idm139857528486512"><h3 class="no_margin">Shahana Ahmed</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Ahmed%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Shahana Ahmed</a></div></div><div id="_co_idm139857528484976"><h3 class="no_margin">Jonathan Morrison</h3><p><sup>3</sup>Department of Public Health and Primary Care, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Morrison%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Jonathan Morrison</a></div></div><div id="_co_idm139857528483440"><h3 class="no_margin">David Pernet</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Pernet%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">David Pernet</a></div></div><div id="_co_idm139857528481904"><h3 class="no_margin">Anthony Renwick</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Renwick%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Anthony Renwick</a></div></div><div id="_co_idm139857528480368"><h3 class="no_margin">Mel Maranian</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Maranian%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Mel Maranian</a></div></div><div id="_co_idm139857458467136"><h3 class="no_margin">Sheila Seal</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Seal%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Sheila Seal</a></div></div><div id="_co_idm139857458465264"><h3 class="no_margin">Maya Ghoussaini</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Ghoussaini%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Maya Ghoussaini</a></div></div><div id="_co_idm139857458463280"><h3 class="no_margin">Sarah Hines</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Hines%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Sarah Hines</a></div></div><div id="_co_idm139857458461296"><h3 class="no_margin">Catherine S Healey</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Healey%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Catherine S Healey</a></div></div><div id="_co_idm139857453425568"><h3 class="no_margin">Deborah Hughes</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Hughes%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Deborah Hughes</a></div></div><div id="_co_idm139857453423584"><h3 class="no_margin">Margaret Warren-Perry</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Warren-Perry%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Margaret Warren-Perry</a></div></div><div id="_co_idm139857453421600"><h3 class="no_margin">William Tapper</h3><p><sup>4</sup>Academic Unit of Genetic Medicine, University of Southampton, Southampton General Hospital, Southampton, UK </p><div>Find articles by <a href="/pubmed/?term=Tapper%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">William Tapper</a></div></div><div id="_co_idm139857453419616"><h3 class="no_margin">Diana Eccles</h3><p><sup>4</sup>Academic Unit of Genetic Medicine, University of Southampton, Southampton General Hospital, Southampton, UK </p><div>Find articles by <a href="/pubmed/?term=Eccles%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Diana Eccles</a></div></div><div id="_co_idm139857453417632"><h3 class="no_margin">D Gareth Evans</h3><p><sup>5</sup>Department of Genetic Medicine, St. Mary’s Hospital, Manchester, UK </p><div>Find articles by <a href="/pubmed/?term=Evans%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">D Gareth Evans</a></div></div><div id="_co_idm139857453415648"><h3 class="no_margin">The Breast Cancer Susceptibility Collaboration (UK)</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">The Breast Cancer Susceptibility Collaboration (UK)</a></div></div><div id="_co_idm139857453413552"><h3 class="no_margin">Maartje Hooning</h3><p><sup>6</sup>Department of Medical Oncology, Erasmus University Medical Center, Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Hooning%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Maartje Hooning</a></div></div><div id="_co_idm139857453411568"><h3 class="no_margin">Mieke Schutte</h3><p><sup>6</sup>Department of Medical Oncology, Erasmus University Medical Center, Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Schutte%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Mieke Schutte</a></div></div><div id="_co_idm139857453409584"><h3 class="no_margin">Ans van den Ouweland</h3><p><sup>7</sup>Department of Clinical Genetics, Erasmus University Medical Center, Rotterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=van%20den%20Ouweland%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Ans van den Ouweland</a></div></div><div id="_co_idm139857453407600"><h3 class="no_margin">Richard Houlston</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Houlston%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Richard Houlston</a></div></div><div id="_co_idm139857453405616"><h3 class="no_margin">Gillian Ross</h3><p><sup>8</sup>Breast Cancer Unit, Royal Marsden National Health Service Foundation Trust, London, UK </p><div>Find articles by <a href="/pubmed/?term=Ross%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Gillian Ross</a></div></div><div id="_co_idm139857454890736"><h3 class="no_margin">Cordelia Langford</h3><p><sup>9</sup>Wellcome Trust Sanger Institute, Hinxton, UK </p><div>Find articles by <a href="/pubmed/?term=Langford%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Cordelia Langford</a></div></div><div id="_co_idm139857454888752"><h3 class="no_margin">Paul D P Pharoah</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><p><sup>3</sup>Department of Public Health and Primary Care, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Pharoah%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Paul D P Pharoah</a></div></div><div id="_co_idm139857454886032"><h3 class="no_margin">Michael R Stratton</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><p><sup>9</sup>Wellcome Trust Sanger Institute, Hinxton, UK </p><div>Find articles by <a href="/pubmed/?term=Stratton%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Michael R Stratton</a></div></div><div id="_co_idm139857454883312"><h3 class="no_margin">Alison M Dunning</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Dunning%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Alison M Dunning</a></div></div><div id="_co_idm139857454881328"><h3 class="no_margin">Nazneen Rahman</h3><p><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </p><div>Find articles by <a href="/pubmed/?term=Rahman%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Nazneen Rahman</a></div></div><div id="_co_idm139857454879344"><h3 class="no_margin">Douglas F Easton</h3><p><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><p><sup>3</sup>Department of Public Health and Primary Care, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Easton%20DF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20453838">Douglas F Easton</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139857525611104_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139857525611104_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139857525611104_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Section of Cancer Genetics, The Institute of Cancer Research, Sutton, Surrey, UK </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Oncology, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Public Health and Primary Care, University of Cambridge, Strangeways Research Laboratory, Cambridge, UK </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Academic Unit of Genetic Medicine, University of Southampton, Southampton General Hospital, Southampton, UK </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Genetic Medicine, St. Mary’s Hospital, Manchester, UK </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Medical Oncology, Erasmus University Medical Center, Rotterdam, The Netherlands </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Clinical Genetics, Erasmus University Medical Center, Rotterdam, The Netherlands </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Breast Cancer Unit, Royal Marsden National Health Service Foundation Trust, London, UK </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Wellcome Trust Sanger Institute, Hinxton, UK </div><div id="FN1"><sup>10</sup>A full list of members is provided in the <strong>supplementary Note</strong>.</div><div id="CR1">Correspondence should be addressed to D.F.E. (<a href="mailto:dev@null" data-email="ku.ca.mac.lrs@salguod" class="oemail">ku.ca.mac.lrs@salguod</a>).</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139857525611104_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20453838" target="pmc_ext" ref="reftype=publisher&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3632836/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm139857456168848" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139857456168848title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Breast cancer is the most common cancer in women in developed countries. To identify common breast cancer susceptibility alleles, we conducted a genome-wide association study in which 582,886 SNPs were genotyped in 3,659 cases with a family history of the disease and 4,897 controls. Promising associations were evaluated in a second stage, comprising 12,576 cases and 12,223 controls. We identified five new susceptibility loci, on chromosomes 9, 10 and 11 (<em>P</em> = 4.6 × 10<sup>−7</sup> to <em>P</em> = 3.2 × 10<sup>−15</sup>). We also identified SNPs in the 6q25.1 (rs3757318, <em>P</em> = 2.9 × 10<sup>−6</sup>), 8q24 (rs1562430, <em>P</em> = 5.8 × 10<sup>−7</sup>) and <em>LSP1</em> (rs909116, <em>P</em> = 7.3 × 10<sup>−7</sup>) regions that showed more significant association with risk than those reported previously. Previously identified breast cancer susceptibility loci were also found to show larger effect sizes in this study of familial breast cancer cases than in previous population-based studies, consistent with polygenic susceptibility to the disease.</p></div></div><div id="idm139857457368672" class="tsec sec headless whole_rhythm"><p id="P3" class="p p-first">Genome-wide association studies (GWAS) provide a powerful approach to identify common disease alleles. Recent GWAS have identified common variants at 12 loci that are associated with an increased risk of breast cancer, and an additional locus, <em>CASP8</em> (specifically, a polymorphism resulting in a D302H substitution), has been identified through a candidate-gene association study<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534744">1</a>-<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534723">8</a></sup>. However, because the risks associated with these variants are modest (per-allele odds ratios (OR) &lt;1.3), they explain only a small fraction of the estimated twofold familial relative risk of breast cancer in first-degree relatives of affected women. Moreover, the GWAS conducted to date have been relatively small, and it is likely that many susceptibility variants have been missed due to lack of power in these studies. In an attempt to identify additional breast cancer loci, we conducted a GWAS that was substantially larger than those conducted to date.</p><p id="P4">We studied 3,960 cases of breast cancer from the UK, selected for a positive family history of breast cancer. We selected cases with a positive family history because, under a polygenic model of susceptibility, this is expected to increase the effect size and hence improve study power<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534753">9</a></sup>. DNA samples from these women were genotyped using an Illumina Infinium 660k array. Case genotypes were compared with those from 5,069 controls, drawn from two UK population-based studies. After quality control exclusions, we utilized data on 582,886 SNPs in 3,659 cases and 4,897 controls (Online Methods).</p><p id="P5">Genotype frequencies in cases and controls were compared using a 1-degree-of-freedom (d.f.) Cochran-Armitage trend test (<a href="/pmc/articles/PMC3632836/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>; for the quantile-quantile plot see <strong>Supplementary Fig. 1</strong>). There was modest evidence for inflation in the test statistic (<em>λ</em> = 1.12, which is equivalent to <em>λ</em><sub>1,000</sub> = 1.03 for a study of 1,000 cases and 1,000 controls). Adjustment for differential population structure using the first two components based on a principal-components analysis of uncorrelated SNPs reduced the inflation to <em>λ</em> = 1.06 (Online Methods).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3632836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139857453384208" class="figure"><a href="/pmc/articles/PMC3632836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3632836_emss-52673-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-52673-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3632836/bin/emss-52673-f0001.jpg"></a></div><div id="largeobj_idm139857453384208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3632836/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3632836/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan plot of 1-d.f. Cochran-Armitage <em>P</em> values for association by genomic position.</p></div></div></div><p id="P6">We observed evidence of association for all 12 of the susceptibility loci identified through previous GWAS, using the same SNP as that previously identified or a strongly correlated SNP (<em>P</em> = 0.02 to <em>P</em> = 3.6 × 10<sup>−31</sup>; <a href="/pmc/articles/PMC3632836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Seven of these loci reached <em>P</em> &lt; 10<sup>−4</sup>, among which five have previously been evaluated in large collaborative analyses of case-control studies by the Breast Cancer Association Consortium (BCAC). The BCAC analyses involved more than 20,000 cases and 20,000 controls, providing a reliable estimate of the per-allele OR<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534741">1</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534735">5</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534737">10</a></sup>. For each of these five SNPs, the per-allele OR in the current study was higher than that estimated from the population-based studies by BCAC by a factor of 1.46-fold to 1.75-fold (<em>P</em> &lt; 0.05 for difference in OR for all SNPs except rs13281615; <strong>Supplementary Table 1</strong>). This enrichment is broadly consistent with the selection of cases with a family history, assuming a multiplicative polygenic model (which predicts a 1.5-fold higher excess relative risk for the associated SNP for women with one affected first-degree relative and a twofold higher excess relative risk for women with two affected first-degree relatives)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534725">9</a></sup>. The loci on 5p12 (rs7716600, a surrogate for rs10941679) and 1p11.2 do not conform to this pattern, having smaller ORs than those published previously (a 1.5-fold higher excess OR can be excluded here in each case, <em>P</em> = 0.018 and <em>P</em> = 0.015, respectively). These results suggest either that the initial effect sizes were overestimated (perhaps due to ‘winner’s curse’) or that these loci have weaker than expected effects in women with a family history due to a different model of susceptibility than is applicable for the other loci. We also found limited evidence in support of the association with the <em>CASP8</em> D302H polymorphism (<em>P</em> = 0.14; <a href="/pmc/articles/PMC3632836/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>)<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534754">8</a></sup>. Consistent with previous results, the two loci showing the largest effect sizes and most significant associations in this GWAS were on chromosome 10, in intron 2 of <em>FGFR2</em> (rs2981579, <em>P</em> = 3.6 × 10<sup>−31</sup>) and at the <em>TOX3</em> locus on 16q (rs3803662, <em>P</em> = 3.2 × 10<sup>−15</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p3">Associations in the current study at previously known breast cancer loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139857453381632" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="left">Strongest association in current study<hr></th><th colspan="4" rowspan="1" valign="bottom" align="left">Published association<hr></th><th colspan="4" rowspan="1" valign="bottom" align="left">Association for published SNP in current study<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Locus</th><th rowspan="1" colspan="1" valign="bottom" align="left">Most signifcant <br>SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Alleles<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Per-allele OR <br>(95% CI)<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="left">Published<br>SNP</th><th rowspan="1" colspan="1" valign="bottom" align="left">Alleles<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">(<em>r</em><sup>2</sup>)<sup>c</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Published OR</th><th rowspan="1" colspan="1" valign="bottom" align="left">Best tag in GWAS (r <sup>2</sup>)<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Alleles<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Per-allele OR<br>(95% CI)<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">
<em>P</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>FGFR2</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs2981579</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.42)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.43<br>(1.35–1.53)</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.6 × 10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs2981582<sup>e</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.38)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.26<br>(1.22–1.29)<sup>1</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs2981579<br><em>(r<sup>2</sup></em> = 1.0)</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.42)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.43<br>(1.35–1.53)</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.6 × 10<sup>−31</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>TOX3</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3803662</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A (0.26)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.30<br>(1.22–1.39)</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.2 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3803662</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.25)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.19<br>(1.15–1.23)<sup>1</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3803662</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.26)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.30<br>(1.22–1.39)</td><td rowspan="1" colspan="1" valign="bottom" align="left">3.2 × 10<sup>−15</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>MAP3K1</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs889312</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/C<br>(0.28)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.22<br>(1.14–1.30)</td><td rowspan="1" colspan="1" valign="bottom" align="left">4.6 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs889312</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/C<br>(0.38)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.12<br>(1.08–1.16)<sup>1</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs889312</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/C<br>(0.28)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.22<br>(1.14–1.30)</td><td rowspan="1" colspan="1" valign="bottom" align="left">4.6 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">8q24</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs1562430</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.58)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.17<br>(1.10–1.25)</td><td rowspan="1" colspan="1" valign="bottom" align="left">5.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs13281615</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.40)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.42</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.08<br>(1.05–1.12)<sup>1</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs13281615</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.41)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.14<br>(1.07–1.21)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.2 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">2q35</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs13387042</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.49)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.21<br>(1.14–1.29)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.0 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs13387042</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.49)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.12<br>(1.09–1.15)<sup>10</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs13387042</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.49)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.21<br>(1.14–1.29)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.0 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>LSP1</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs909116</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.53)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.17<br>(1.10–1.24)</td><td rowspan="1" colspan="1" valign="bottom" align="left">7.3 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3817198</td><td rowspan="1" colspan="1" valign="bottom" align="left">T/C<br>(0.30)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.23</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.07<br>(1.04–1.11)<sup>1</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3817198</td><td rowspan="1" colspan="1" valign="bottom" align="left">T/C<br>(0.33)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.12<br>(1.05–1.19)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0006</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">5pl2</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs9790879</td><td rowspan="1" colspan="1" valign="bottom" align="left">T/C<br>(0.40)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.10<br>(1.03–1.17)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0032</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs10941679</td><td rowspan="1" colspan="1" valign="bottom" align="left">(A/G)<br>(0.25)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.48</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.19<br>(1.11–1.28)<sup>4</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs7716600<br><em>(r<sup>2</sup></em> = 0.75)</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/A<br>(0.22)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.11<br>(1.04–1.19)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0034</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">6q25.1</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs3757318</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.07)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.30<br>(1.17–1.46)</td><td rowspan="1" colspan="1" valign="bottom" align="left">2.9 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs2046210</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.34)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.088</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.15<sup>f</sup><br>(1.03–1.28)<sup>7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs6900157<br><em>(r<sup>2</sup></em> = 0.96)</td><td rowspan="1" colspan="1" valign="bottom" align="left">T/C<br>(0.35)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.15<br>(1.08–1.22)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.8 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>SLC4A7</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs4973768</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T (0.47)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.16<br>(1.10–1.24)</td><td rowspan="1" colspan="1" valign="bottom" align="left">5.8 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs4973768</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.46)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.11<br>(1.08–1.13)<sup>5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">Rs4973768</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.47)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.16<br>(1.10–1.24)</td><td rowspan="1" colspan="1" valign="bottom" align="left">5.8 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>COX11</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs1156287</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.29)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.91<br>(0.85–0.97)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0058</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs6504950</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.27)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.91</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.95<br>(0.92–0.97)<sup>5</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs7222197<br><em>(r<sup>2</sup></em> = 1.0)</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/A<br>(0.28)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.92<br>(0.86–0.99)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.021</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>RAD51L1</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs8009944</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/A<br>(0.75)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.88<br>(0.82–0.95)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0004</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs999737</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.24)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.13</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.94<br>(0.88–0.99)<sup>6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs999737</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.25)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.89<br>(0.83–0.95)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.0009</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">lpll.2</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs11249433</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.42)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.08<br>(1.02–1.15)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.010</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs11249433</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.39)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.0</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.16<br>(1.09–1.24)<sup>6</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs11249433</td><td rowspan="1" colspan="1" valign="bottom" align="left">A/G<br>(0.42)</td><td rowspan="1" colspan="1" valign="bottom" align="left">1.08<br>(1.02–1.15)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.010</td></tr><tr><td rowspan="1" colspan="1" valign="bottom" align="left">
<em>CASP8</em>
</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs10931936</td><td rowspan="1" colspan="1" valign="bottom" align="left">T/C<br>(0.74)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.88<br>(0.82–0.94)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.00015</td><td rowspan="1" colspan="1" valign="bottom" align="left">rs1045485</td><td rowspan="1" colspan="1" valign="bottom" align="left">G/C<br>(0.13)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.083</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.88<br>(0.84–0.92)<sup>8</sup></td><td rowspan="1" colspan="1" valign="bottom" align="left">rs17468277<br><em>(r<sup>2</sup></em> = 1.0)</td><td rowspan="1" colspan="1" valign="bottom" align="left">C/T<br>(0.13)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.93<br>(0.85–1.02)</td><td rowspan="1" colspan="1" valign="bottom" align="left">0.14</td></tr></tbody></table></div><div id="largeobj_idm139857453381632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3632836/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Allele (frequency of the second listed allele).</div><div id="TFN2"><sup>b</sup>Per-allele OR for the second listed allele, relative to the first. In each case the second listed allele was that which correlated with the second-listed published allele.</div><div id="TFN3"><sup>c</sup><em>r</em><sup>2</sup> between the published SNP and most significant SNP in this study based on HapMap CEU.</div><div id="TFN4"><sup>d</sup><em>r</em><sup>2</sup> between the published SNP and the best tagSNP in this study based on HapMap CEU.</div><div id="TFN5"><sup>e</sup>Note that fine-mapping and functional analyses suggest that the strongest association for breast cancer is with rs2981578<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534724">25</a></sup>. It is correlated with rs2981579 and rs2981582 at <em>r</em><sup>2</sup> = 0.85. No more strongly correlated tag for rs2981578 was typed in the GWAS.</div><div id="TFN6"><sup>f</sup>Estimated OR in Europeans. Estimated OR in Chinese was 1.36.</div></div></div><p id="P7">For three loci (6q25.1, <em>LSP1</em> and 8q24) we identified a SNP that showed a more significant association than the SNP originally reported associated to breast cancer susceptibility. The SNP with the lowest <em>P</em> value at 6q25.1 (rs3757318, <em>P</em> = 2.9 × 10<sup>−6</sup>) lies ~200 kb upstream of <em>ESR1</em> in an intron of <em>C6orf97</em>. In Europeans, rs3757318 is only weakly correlated with rs2046210, which has previously been identified as a susceptibility SNP<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534746">7</a></sup> in a study from Shanghai (<em>r</em><sup>2</sup> = 0.088), though these two SNPs are more strongly correlated in an East Asian population (<em>r</em><sup>2</sup> = 0.48 in HapMap CHB). Both rs3757318 and rs6900157 (a surrogate for rs2046210 with <em>r</em><sup>2</sup> = 0.96) remained significantly associated with breast cancer after multiple logistic regression analysis (<em>P</em> = 0.0003 and <em>P</em> = 0.002, respectively). These results suggest either the presence of a single causal variant that is more strongly correlated with rs3757318 than rs2046210 in Europeans or the presence of two causal variants. The more strongly associated SNPs that we identified in the 8q24 and <em>LSP1</em> regions lie within the same linkage disequilibrium (LD) blocks as the originally identified SNP, and in each case, the original SNP was not significantly associated with risk after adjusting for the new SNP. Thus, these results may reflect the same underlying association and should assist in narrowing the search for the true causal variants. A more strongly associated variant, rs10931936, was also identified at the <em>CASP8</em> locus (<em>P</em> = 0.0014, <em>r</em><sup>2</sup> = 0.13).</p><p id="P8">After eliminating SNPs in previously identified susceptibility regions, we identified 28 SNPs in 13 regions of LD that were significant at <em>P</em> &lt; 0.00001. After eliminating SNPs that were strongly correlated, we attempted to replicate these associations by genotyping 15 SNPs in a second stage involving 11,431 cases and 11,081 controls from four studies in the UK and The Netherlands (Online Methods). We also incorporated available data from 1,145 cases and 1,142 controls from the Cancer Genetic Markers of Susceptibility (CGEMS) study. Six SNPs from five regions on chromosomes 9,10 and 11 showed clear evidence of replication in stage 2 (<em>P</em> = 0.0017 or better and in the same direction as stage 1) and reached significance levels over both stages combined of <em>P</em> = 4.6 × 10<sup>−7</sup> to <em>P</em> = 3.2 × 10<sup>−15</sup> (<a href="/pmc/articles/PMC3632836/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <strong>Supplementary Tables 2</strong> and <strong>3</strong>). rs614367 and rs624797, which both showed strong evidence of association, were correlated, and rs624797 showed no independent association after adjustment for rs614367. The per-allele OR was higher in stage 1 than stage 2 for each SNP (<em>P</em> &lt; 0.05 in each case; <strong>Supplementary Table 2</strong>). This may reflect either winner’s curse or the enrichment of stage 1 for cases with a positive family history. There was no evidence for heterogeneity in the per-allele ORs among the stage 2 samples, with the exception of the weak evidence shown for rs10995190 (<em>P</em> = 0.08; <strong>Supplementary Table 2</strong>). There was no evidence for departure from a log-additive model for any SNP (that is, the OR for rare homozygotes did not differ significantly from the square of the OR for heterozygotes). There was weak evidence of a decrease in the per-allele OR with age for rs1011970 and of an increase in the per-allele OR with age for rs614367 (<em>P</em> = 0.071 and <em>P</em> = 0.068; <strong>Supplementary Table 4</strong>). rs614367 and rs624797 (but no other SNPs) showed a consistently stronger association with a positive family history in both stages (for rs614367, <em>P</em> = 0.006 and <em>P</em> = 0.00016, respectively; for rs624797, <em>P</em> = 0.012 and <em>P</em> = 0.001, respectively; <strong>Supplementary Table 4</strong>). For four of the SNPs (rs10995190, rs1011970, rs614367 and rs624797), the estimated per-allele ORs were higher for estrogen receptor–positive disease and showed little association in estrogen receptor–negative breast cancer, consistent with the pattern seen for the majority of breast cancer loci identified to date. For rs2380205 and rs704010, the per-allele ORs for estrogen receptor–positive and estrogen receptor–negative disease were similar, but the number of estrogen receptor–negative cases used was too small to draw firm conclusions on the effect sizes for this subset (<strong>Supplementary Table 4</strong>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p4">Associations between genotype and breast cancer risk for six SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139857456377360" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">Marker</th><th rowspan="2" colspan="1" valign="bottom" align="left">Chromosome position</th><th rowspan="2" colspan="1" valign="bottom" align="left">Stage<sup>a</sup></th><th rowspan="2" colspan="1" valign="bottom" align="right">Cases/controls</th><th rowspan="2" colspan="1" valign="bottom" align="left">MAF<sup>b</sup></th><th rowspan="2" colspan="1" valign="bottom" align="right">Per-allele OR (95% CI)</th><th rowspan="2" colspan="1" valign="bottom" align="right">Heterozygous OR<br>(95% CI)</th><th rowspan="2" colspan="1" valign="bottom" align="right">Homozygous OR<br>(95%CI)</th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>P</em>value<sup>c</sup><hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Stage</th><th rowspan="1" colspan="1" valign="bottom" align="right">Combined</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1011970 G/T</td><td rowspan="1" colspan="1" valign="top" align="left">9<br>22,052,134</td><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="left">3,730/4,894</td><td rowspan="1" colspan="1" valign="top" align="left">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">1.20<br>(1.11–1.30)</td><td rowspan="1" colspan="1" valign="top" align="left">1.19<br>(1.08–1.31)</td><td rowspan="1" colspan="1" valign="top" align="left">1.45<br>(1.13–1.86)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6&nbsp;×&nbsp;10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="left">12,253/12,000</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td><td rowspan="1" colspan="1" valign="top" align="left">1.09<br>(1.04–1.14)</td><td rowspan="1" colspan="1" valign="top" align="left">1.07<br>(1.01–1.13)</td><td rowspan="1" colspan="1" valign="top" align="left">1.29<br>(1.12–1.50)</td><td rowspan="1" colspan="1" valign="top" align="left">0.00026</td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2380205 C/T</td><td rowspan="1" colspan="1" valign="top" align="left">10<br>5,926,740</td><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="left">3,730/4,895</td><td rowspan="1" colspan="1" valign="top" align="left">0.44</td><td rowspan="1" colspan="1" valign="top" align="left">0.86<br>(0.81–0.92)</td><td rowspan="1" colspan="1" valign="top" align="left">0.86<br>(0.78–0.95)</td><td rowspan="1" colspan="1" valign="top" align="left">0.75<br>(0.66–0.85)</td><td rowspan="1" colspan="1" valign="top" align="left">7.9 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="left">12,235/11,961</td><td rowspan="1" colspan="1" valign="top" align="left">0.43</td><td rowspan="1" colspan="1" valign="top" align="left">0.94<br>(0.91–0.98)</td><td rowspan="1" colspan="1" valign="top" align="left">0.95<br>(0.90–1.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.89<br>(0.82–0.95)</td><td rowspan="1" colspan="1" valign="top" align="left">0.0017</td><td rowspan="1" colspan="1" valign="top" align="left">4.6 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10995190 G/A</td><td rowspan="1" colspan="1" valign="top" align="left">10<br>63,948,688</td><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="left">3,731/4,891</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.76<br>(0.70–0.84)</td><td rowspan="1" colspan="1" valign="top" align="left">0.77<br>(0.69–0.86)</td><td rowspan="1" colspan="1" valign="top" align="left">0.55<br>(0.40–0.77)</td><td rowspan="1" colspan="1" valign="top" align="left">6.1 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="left">12,261/12,000</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">0.86<br>(0.82–0.91)</td><td rowspan="1" colspan="1" valign="top" align="left">0.84<br>(0.79–0.89)</td><td rowspan="1" colspan="1" valign="top" align="left">0.83<br>(0.69–1.00)</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.1 × 10<sup>−15</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs704010 G/A</td><td rowspan="1" colspan="1" valign="top" align="left">10<br>80,511,154</td><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="left">3,726/4,893</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">1.15<br>(1.09–1.23)</td><td rowspan="1" colspan="1" valign="top" align="left">1.05<br>(0.95–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">1.38<br>(1.22–1.57)</td><td rowspan="1" colspan="1" valign="top" align="left">3.5 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="left">12,222/11,992</td><td rowspan="1" colspan="1" valign="top" align="left">0.39</td><td rowspan="1" colspan="1" valign="top" align="left">1.07<br>(1.03–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">1.11<br>(1.05–1.17)</td><td rowspan="1" colspan="1" valign="top" align="left">1.13<br>(1.04–1.21)</td><td rowspan="1" colspan="1" valign="top" align="left">0.00026</td><td rowspan="1" colspan="1" valign="top" align="left">3.7 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs614367<br>C/T</td><td rowspan="1" colspan="1" valign="top" align="left">11<br>69,037,945</td><td rowspan="1" colspan="1" valign="top" align="left">Stage 1</td><td rowspan="1" colspan="1" valign="top" align="left">3,723/4,882</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">1.30<br>(1.20–1.41)</td><td rowspan="1" colspan="1" valign="top" align="left">1.24<br>(1.13–1.37)</td><td rowspan="1" colspan="1" valign="top" align="left">2.02<br>(1.56–2.64)</td><td rowspan="1" colspan="1" valign="top" align="left">3.9 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Stage 2</td><td rowspan="1" colspan="1" valign="top" align="left">12,114/11,967</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">1.15<br>(1.10–1.20)</td><td rowspan="1" colspan="1" valign="top" align="left">1.16<br>(1.10–1.23)</td><td rowspan="1" colspan="1" valign="top" align="left">1.27<br>(1.10–1.47)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.2 × 10<sup>−15</sup></td></tr></tbody></table></div><div id="largeobj_idm139857456377360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3632836/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><sup>a</sup>Stage 2 includes genotype data in SEARCH, RBCS and FBCS together with publicly available data from CGEMS.</div><div id="TFN8"><sup>b</sup>MAF, frequency of the minor (second listed) allele.</div><div id="TFN9"><sup>c</sup>Adjusted 1-d.f. <em>P</em> trend (see Online Methods).</div></div></div><p id="P9">To examine whether there was evidence for a more strongly associated variant in any of the above regions, we used imputation to estimate the genotype probabilities in the stage 1 data at known SNPs in region using the HapMap CEU data as a framework. On chromosome 11, we identified four SNPs that showed a more significant association than rs614367 (most significantly associated SNP rs6610204; <em>P</em> = 4.6 × 10<sup>−14</sup>; <strong>Supplementary Table 5</strong>). In the other regions, no SNPs showed associations that were more significant than the original SNP. We also estimated the ORs associated with haplotypes of SNPs in each of the five regions (<strong>Supplementary Table 6</strong>). In each case, the association was present on more than one haplotype carrying the risk allele for the initially associated SNP, suggesting that the associations are unlikely to be driven by a single rare, high penetrance variant. For the chromosome 11 region, there was evidence of association with risk for two related haplotypes carrying the T allele of rs614367 with a combined frequency of 4%, suggesting that the causal variant may be somewhat rarer than the 15% minor allele frequency of rs614367.</p><p id="P10">SNP rs1011970 lies in a 180-kb block on 9p21 that includes <em>CDKN2A</em> and <em>CDKN2B</em>. These two genes encode cyclin-dependent kinase inhibitors and are frequently mutated or deleted in a wide variety of human tumors<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534738">11</a></sup>. Germline mutations in <em>CDKN2A</em> predispose to malignant melanoma and pancreatic cancer<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534734">12</a></sup>, and recent GWAS also identified rs1011970 to be associated with melanoma risk<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534750">13</a></sup>; SNPs within this same region are associated with nevus density and melanoma<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534732">14</a></sup>, basal cell carcinoma<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534727">15</a></sup>, glioma<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481089907">16</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534742">17</a></sup>, diabetes<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342832327">18</a></sup> and coronary heart disease<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534752">19</a></sup>. This is the first example of the same common variant predisposing to breast cancer and another cancer type rs10757278, which is correlated with rs1011970 (<em>r</em><sup>2</sup> = 0.7), is associated with levels of expression in lymphocytes of <em>CDKN2A, CDKN2B</em> and a noncoding RNA in the same block, <em>CDKN2BAS</em> (also known as <em>ANR1L</em>)<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534730">20</a></sup>.</p><p id="P11">rs614367 on 11q13 lies in an LD block of ~166 kb that contains no annotated genes. This region is frequently amplified in human tumors, including breast cancers<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534731">21</a></sup>. Plausible genes flanking this block include: proximally, <em>MYEOV</em>, a gene overexpressed in myeloma; distally, <em>CCND1</em>, encoding cyclin D1, a protein critical for cell-cycle control that is somatically altered in many tumor types; <em>ORAOV1</em>, a gene overexpressed in oral cancer; and three genes encoding fibroblast growth factors, <em>FGF19, FGF4</em> and <em>FGF3</em>. FGF3 and FGF4 are oncogenic growth factors that bind distinct FGFR2 isoforms, providing a possible link with the <em>FGFR2</em> susceptibility locus<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534743">22</a></sup>.</p><p id="P12">rs10995190 on chromosome 10 lies within intron 4 of <em>ZNF365</em>, which encodes zinc finger protein 365. An amino acid substitution in this gene has been associated with uric acid nephrolithiasis<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534729">23</a></sup>. Recent GWAS have identified another variant within this gene, rs10995271, located 159 kb downstream of rs10995190, to be associated with Crohn’s disease<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534749">24</a></sup>. rs2380205 lies in a 105-kb block on chromosome 10 containing the genes <em>ANKRD16</em> (encoding ankyrin repeat domain 16) and <em>FBXO18</em> (encoding the F-box protein, helicase 18). rs704010 on chromosome 10 lies in a 20-kb block 90 kb upstream of <em>ZMIZ1</em> (encoding zinc finger MIZ-type containing 1).</p><p id="P13">Based on the estimated per-allele ORs from stage 2 of our study, the newly identified loci explain approximately 1.2% of the familial risk of breast cancer, though the overall contribution may be larger because the true causal variants may be more strongly associated with disease than the SNPs tagging them in this study. Taken together with estimates from previous studies, the 18 confirmed breast cancer susceptibility loci explain approximately 8% of the familial risk of breast cancer, whereas rarer mutations in the known high risk loci (principally <em>BRCA1</em> and <em>BRCA2</em>) and moderate risk loci explain a further ~20%. This is by far the largest breast cancer GWAS to date and confirms that the <em>FGFR2</em> and <em>TOX3</em> loci (conferring per-allele ORs between 1.2 and 1.3) have the largest effect sizes from among the common susceptibility loci that are detectable with the current high-coverage genome-wide SNP sets. The residual familial risk is therefore likely to be due to a combination of a large number of common variants with smaller effects together with rarer variants not testable with current arrays. It is likely that many additional loci will be identifiable through more extensive follow-up of data from this and other GWAS.</p><p id="P14" class="p"><strong>URLs.</strong> CGEMS, <a href="http://cgems.cancer.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov/</a>; Welcome Trust Case Control Consortium (WTCCC), <a href="http://www.wtccc.org.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk/</a>; Nurses Health Study, <a href="http://www.channing.harvard.edu/nhs/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.channing.harvard.edu/nhs/</a>; Mach, <a href="http://www.sph.umich.edu/csg/abecasis/MaCH/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MaCH/index.html</a>; data access from this GWAS, <a href="http://www.srl.cam.ac.uk/genepi/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.srl.cam.ac.uk/genepi/</a>.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">ONLINE METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Samples</h3><p id="P16" class="p p-first">Three thousand nine hundred and sixty breast cancer cases were used in stage 1, of which 3,652 were from cancer genetics clinics in the UK recruited through the Familial Breast Cancer Study (FBCS) and 308 were from oncology clinics in the UK recruited through the Prospective study of Outcomes in Sporadic versus Hereditary breast cancer (POSH) study. Cases were preferentially selected to have at least two affected first- or second-degree relatives. The majority of cases were screened and found to be negative for germline mutations, including large rearrangements, in <em>BRCA1</em> and <em>BRCA2</em>. A minority of samples were not tested for <em>BRCA1</em> or <em>BRCA2</em> mutations. All carriers of disease-associated mutations in <em>BRCA1</em> and <em>BRCA2</em> were excluded. We also excluded all cases with self-reported non-European ancestry.</p><p id="P17">Controls for stage 1 were drawn from two sources: 2,930 controls were drawn from the 1958 Birth Cohort (1958BC), a population-based study in the United Kingdom of individuals born in 1 week in 1958 (ref. <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534748">26</a>). The remaining 2,737 controls were identified through the UK National Blood Service (NBS)<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534739">19</a></sup>. These samples were genotyped as part of the Wellcome Trust Case Control Consortium (WTCCC2; see URLs)<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534751">27</a></sup>. The analyses presented here are based on 2,482 1958BC and 2,587 NBS controls for which genotype data were available at the time of analysis.</p><p id="P18">Samples for stage 2 were drawn from six sources: (i) the SEARCH study, a population-based study of cases from East Anglia (<em>n</em> = 6,640); controls (<em>n</em> = 6,832) were drawn from the European Prospective Investigation into Cancer and Nutrition (EPIC) study, a population-based cohort study of diet and cancer from general practices contributing to SEARCH; (ii) the Rotterdam breast cancer study (RBCS) (799 cases, 800 controls); (iii) the Familial Breast Cancer Study (FBCS), consisting of additional cases ascertained through UK cancer genetics clinics (<em>n</em> = 2,009); (iv) the RMH breast cancer series (<em>n</em> = 1,732); and (v) the Prospective study of Outcomes in Sporadic vs. Hereditary breast cancer (POSH) study (<em>n</em> = 251). The combined samples from these latter three series (<em>n</em> = 3,992) were analyzed in a single replication experiment together with additional controls selected through the 1958BC (<em>n</em> = 3,450), none of which were included in stage 1. For stage 2, we also incorporated data on the relevant SNPs from the CGEMS study (see URLs). CGEMS is based on 1,145 cases and 1,142 controls drawn from the Nurses Health Study (see URLs) which were genotyped using the Illumina 550k array.</p><p id="P19" class="p p-last">All studies were approved by the appropriate ethics committees.</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P20" class="p p-first">Genotypes for stage 1 cases were generated using a custom Illumina Infinium 670k array and controls were genotyped using an Illumina Infinium 1.2M array at the Wellcome Trust Sanger Institute. For this analysis, we analyzed data on 594,375 SNPs that were successfully genotyped on both arrays. Genotypes for both arrays were called using the Illuminus algorithm<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534733">28</a></sup>. We used genotypes for which Illuminus generated a posterior probability of &gt;0.95. Cluster plots were inspected manually for all SNPs considered for inclusion in stage 2.</p><p id="P21" class="p p-last">Genotyping for stage 2 was performed by 5′ exonuclease assay (Taqman) using the ABI Prism 7900HT Sequence Detection System according to the manufacturer’s instructions. Primers and probes were supplied directly by Applied Biosystems as Assays-By-Design. Assays included at least two negative controls and 2% to 5% duplicates per plate. Genotyping for one marker, rs1866823, failed for the SEARCH and RBCS studies, and the marker was replaced by rs2246873 (<em>r</em><sup>2</sup> = 0.94 in HapMap CEU).</p></div><div id="S5" class="sec"><h3 id="S5title">Analyses</h3><p id="P22" class="p p-first">We restricted analyses to individuals who were called on &gt;97% of successfully genotyped SNPs. To identify close relatives, we computed identity-by-state (IBS) probabilities for all pairs. We confirmed 2 case monozygotic twin (MZ) pairs, 22 duplicate case pairs and 24 first-degree relative pairs (IBS &gt; 0.86). We also identified 4 probable case-control and 44 probable control-control sibling pairs. We excluded the control from the case-control pairs and the sample with the lower call rate from the remaining pairs. By computing IBS scores between participants and individuals in HapMap and by using multidimensional scaling, we identified 89 individuals who appeared to have substantial Asian or African ancestry (defined as approximately &gt;15% non-European ancestry, comprising 69 cases, 4 individuals from 1958BC and 16 NBS). After these exclusions, 3,659 cases and 4,897 controls were used in the final analysis.</p><p id="P23" class="p p-last">We filtered out all SNPs with, in either cases or controls, a MAF &lt; 1%, a call rate of &lt; 99% and a MAF &lt; 5%, or a call rate &lt; 95% and MAF ≥ 5%. We also excluded SNPs whose frequencies departed from HWE at <em>P</em> &lt; 0.00001 in controls or <em>P</em> &lt; 10<sup>−12</sup> in cases. After these exclusions, we used data on 582,886 SNPs. Duplicate concordance was 99.99%.</p></div><div id="S6" class="sec sec-last"><h3 id="S6title">Statistical methods</h3><p id="P24" class="p p-first">We first assessed associations between each SNP and breast cancer at stage 1 using a 1-d.f. Cochran-Armitage trend test and a general 2-d.f. <em>χ</em><sup>2</sup> test. Inflation in the <em>χ</em><sup>2</sup> statistic was assessed using the genomic control approach; we derived an inflation factor <em>λ</em> by dividing the median of the lowest 90% of the 1-d.f. statistics by the 45% percentile of a 1-d.f. <em>χ</em><sup>2</sup> distribution (0.357). We have also presented the equivalent inflation factor for a study of 1,000 cases and 1,000 controls (<em>λ</em><sub>1,000</sub>) calculated by <em>λ</em><sub>1,000</sub> = 1 + 500 (1 / <em>N</em><sub>cases</sub>+ 1 / <em>N</em><sub>controls</sub>) / (<em>λ</em> − 1), where <em>N</em><sub>cases</sub> and <em>N</em><sub>controls</sub> are the number of cases and controls, respectively.</p><p id="P25">To correct for potential inflation due to population structure, we performed a principal-components analysis based on the genotypes of a subset of 35,797 uncorrelated SNPs (<em>r</em><sup>2</sup> &lt; 0.1)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534745">29</a></sup>. We then computed 1-d.f. score tests for each SNP, adjusting for progressively larger numbers of principal components as covariates. Adjustment for the first two components reduced the inflation slightly (to 1.06); however, adjustment for further components did not reduce the inflation further. Adjusted significance tests were therefore calculated from the score tests adjusted for two principal components. To allow for the residual inflation, we adjusted the resulting test statistics using the genomic control approach by dividing the test statistic by the inflation factor.</p><p id="P26">SNPs were selected for evaluation in stage 2 on the basis of a significance level of <em>P</em> &lt; 10<sup>−5</sup> based on the unadjusted 1-d.f. trend test. Where two or more SNPs were selected from the same region, we used multiple logistic regression to determine a minimal set of SNPs that showed evidence of association after adjustment for other SNPs. In practice, one SNP was selected in each region except in the case of one region, in which two SNPs were genotyped.</p><p id="P27" class="p p-last">After stage 2, overall 1-d.f. and 2-d.f. tests of association were derived, stratified by stage and study. Adjusted tests of association were derived by adjusting in stage 1 for principal components and genomic control as described above. In the combined analysis, the effect size in stage 1 was weighted by a factor of 2 relative to that in stage 2, consistent with the effect size expected under a polygenic model. A criterion of <em>P</em> &lt; 5 × 10<sup>−7</sup> was used for genome-wide significance<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534736">19</a></sup>, and ORs and 95% confidence limits were estimated using unconditional logistic regression, stratified by study. In the text, we have reported the combined tests of association over both stages, but we have emphasized the OR estimates from stage 2 to minimize the effect of winner’s curse. Tests of homogeneity of the ORs across strata were assessed using likelihood ratio tests. The associations between genotype and family history in stage 2, and between genotype and estrogen receptor status, were assessed using a case-only analysis (that is, treating family history or estrogen receptor status as the outcome variable and estimating a per-allele OR for each SNP using logistic regression). For stage 1, the effect of family history was analyzed using a family history score, derived as the total number of affected relatives weighted by their degree of relationship to the case. The effect of family history score on per-allele OR was assessed using constrained polytomous regression. Modification of the ORs by age at diagnosis was assessed using a case-only analysis, assessing the association between age and SNP genotype in the cases using polytomous regression. The contribution of the loci to the familial risk of breast cancer was estimated by first computing the familial risk to a daughter of an affected individual that was attributable to each locus (<em>λ</em><sub>1</sub>) from the allele frequency and the estimated per-allele OR in the SEARCH study, which was largest study contributing to stage 2 and which is population based. The proportion of the familial risk due to each locus was then calculated as ln(<em>λ</em><sub>1</sub>) / ln(2), assuming an overall familial relative risk of 2. The combined effect of all loci was then derived by summing the locus-specific contributions (that is, assuming a log-additive model). Imputed genotypes for non-typed SNPs were estimated using Mach (see URLs), using the HapMap CEU data as a framework. Haplotype analyses were conducted in haplo.stats<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534728">30</a></sup>. Haplotypes were based on SNPs in each region that were significantly associated with breast cancer at <em>P</em> &lt; 0.001, after eliminating perfectly correlated SNPs. Per-haplotype ORs were estimated using the haplo.cc routine. Other analyses were performed in R, principally using GenABEL<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308534747">31</a></sup>, and Stata (R, <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a>; Stata, <a href="http://www.stata.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stata.com/</a>).</p></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p1">This work was supported by the Wellcome Trust and by Cancer Research UK. D.F.E. is a Principal Research Fellow of Cancer Research UK. C.T. is funded by a Medical Research Council Clinical Research Fellowship. The samples were collected and screened for <em>BRCA</em> mutations through funding from Cancer Research UK; US Military Acquisition (ACQ) Activity, Era of Hope Award (W81XWH-05-1-0204) and the Institute of Cancer Research (UK). This study makes use of data generated by the Wellcome Trust Case Control Consortium (WTCCC) 2. A full list of the investigators who contributed to the generation of the data is available from the WTCCC website. We acknowledge use of DNA from the British 1958 Birth Cohort collection, funded by the Medical Research Council grant G0000934 and the Wellcome Trust grant 068545/Z/02. Funding for this project was provided by the Wellcome Trust under award 085475. We thank the SEARCH team and Eastern Cancer Registry and Information Centre (ECRIC) for recruitment of the SEARCH cases. We acknowledge the clinicians from the Rotterdam Family Cancer Clinic who were involved in collecting the RBCS samples: C. Seynaeve, J. Klijn, J. Collee and R. Oldenburg.</p></div></div><div id="idm139857457883184" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139857457883184title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P28" class="p p-first-last">Note: Supplementary information is available on the Nature Genetics website.</p><p></p><p class="fn sec" id="FN3"></p><p id="P29" class="p p-first-last">AUTHOR CONTRIBUTIONS: D.F.E., N.R., M.R.S., P.D.P.P. and A.M.D. obtained funding for the study. D.F.E. designed the study and drafted the manuscript. D.F.E. and C.T. conducted the statistical analyses. J.M. provided data management and bioinformatics support. C.T. and N.R. coordinated the Familial Breast Cancer Study (FBCS). D.P., A.R., S.S., S.H., D.H., M.W.-P., C.T. and N.R. coordinated the FBCS genotyping. P.D.P.P. and D.F.E. coordinated Studies of Epidemiology and Risk Factors in Cancer Heredity (SEARCH). S.A., M.M., M.G., C.S.H. and A.M.D. coordinated the stage 2 genotyping of the SEARCH and RBCS samples. M.H., M.S. and A.v.d.O. coordinated and provided samples and data from RBCS. C.L. coordinated the stage 1 genotyping. G.R. and R.H. provided data and samples from the Royal Marsden Hospital (RMH) study. W.T. and D.E. provided data and samples from the Prospective study of Outcomes in Sporadic vs. Hereditary breast cancer (POSH) study. D.E. and D.G.E. provided data and samples for FBCS. All authors provided critical review of the manuscript.</p><p></p><p class="fn sec" id="FN4"></p><p id="P31" class="p p-first-last">COMPETING FINANCIAL INTERESTS: The authors declare no competing financial interests.</p><p></p></div></div><div id="idm139857453513440" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139857453513440title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Easton DF, et al.  Genome-wide association study identifies novel breast cancer susceptibility loci. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:1087–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+identifies+novel+breast+cancer+susceptibility+loci&amp;author=DF+Easton&amp;volume=447&amp;publication_year=2007&amp;pages=1087-1093&amp;pmid=17529967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Hunter DJ, et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=DJ+Hunter&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Stacey SN, et al.  Common variants on chromosomes 2q35 and 16q12 confer susceptibility to estrogen receptor-positive breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:865–869.</span> [<a href="/pubmed/17529974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+on+chromosomes+2q35+and+16q12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=SN+Stacey&amp;volume=39&amp;publication_year=2007&amp;pages=865-869&amp;pmid=17529974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Stacey SN, et al.  Common variants on chromosome 5p12 confer susceptibility to estrogen receptor-positive breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:703–706.</span> [<a href="/pubmed/18438407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Common+variants+on+chromosome+5p12+confer+susceptibility+to+estrogen+receptor-positive+breast+cancer&amp;author=SN+Stacey&amp;volume=40&amp;publication_year=2008&amp;pages=703-706&amp;pmid=18438407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Ahmed S, et al.  Newly discovered breast cancer susceptibility loci on 3p24 and 17q23.2. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:585–590.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748125/">PMC free article</a>]</span> [<a href="/pubmed/19330027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Newly+discovered+breast+cancer+susceptibility+loci+on+3p24+and+17q23.2&amp;author=S+Ahmed&amp;volume=41&amp;publication_year=2009&amp;pages=585-590&amp;pmid=19330027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Thomas G, et al.  A multistage genome-wide association study in breast cancer identifies two new risk alleles at 1p11.2 and 14q24.1 (RAD51L1) <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:579–584.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928646/">PMC free article</a>]</span> [<a href="/pubmed/19330030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+multistage+genome-wide+association+study+in+breast+cancer+identifies+two+new+risk+alleles+at+1p11.2+and+14q24.1+(RAD51L1)&amp;author=G+Thomas&amp;volume=41&amp;publication_year=2009&amp;pages=579-584&amp;pmid=19330030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Zheng W, et al.  Genome-wide association study identifies a new breast cancer susceptibility locus at 6q25.1. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:324–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754845/">PMC free article</a>]</span> [<a href="/pubmed/19219042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+a+new+breast+cancer+susceptibility+locus+at+6q25.1&amp;author=W+Zheng&amp;volume=41&amp;publication_year=2009&amp;pages=324-328&amp;pmid=19219042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Cox A, et al.  A common coding variant in CASP8 is associated with breast cancer risk. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:352–358.</span> [<a href="/pubmed/17293864" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+common+coding+variant+in+CASP8+is+associated+with+breast+cancer+risk&amp;author=A+Cox&amp;volume=39&amp;publication_year=2007&amp;pages=352-358&amp;pmid=17293864&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Antoniou AC, Easton DF. Polygenic inheritance of breast cancer: implications for design of association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2003;<span class="ref-vol">25</span>:190–202.</span> [<a href="/pubmed/14557987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Polygenic+inheritance+of+breast+cancer:+implications+for+design+of+association+studies&amp;author=AC+Antoniou&amp;author=DF+Easton&amp;volume=25&amp;publication_year=2003&amp;pages=190-202&amp;pmid=14557987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Milne RL, et al.  Risk of estrogen receptor-positive and -negative breast cancer and single-nucleotide polymorphism 2q35-rs13387042. <span><span class="ref-journal">J. Natl. Cancer Inst. </span>2009;<span class="ref-vol">101</span>:1012–1018.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2724850/">PMC free article</a>]</span> [<a href="/pubmed/19567422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl.+Cancer+Inst&amp;title=Risk+of+estrogen+receptor-positive+and+-negative+breast+cancer+and+single-nucleotide+polymorphism+2q35-rs13387042&amp;author=RL+Milne&amp;volume=101&amp;publication_year=2009&amp;pages=1012-1018&amp;pmid=19567422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Kamb A, et al.  A cell cycle regulator potentially involved in genesis of many tumor types. <span><span class="ref-journal">Science. </span>1994;<span class="ref-vol">264</span>:436–440.</span> [<a href="/pubmed/8153634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+cell+cycle+regulator+potentially+involved+in+genesis+of+many+tumor+types&amp;author=A+Kamb&amp;volume=264&amp;publication_year=1994&amp;pages=436-440&amp;pmid=8153634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kamb A, et al.  Analysis of the p16 gene (CDKN2) as a candidate for the chromosome 9p melanoma susceptibility locus. <span><span class="ref-journal">Nat. Genet. </span>1994;<span class="ref-vol">8</span>:23–26.</span> [<a href="/pubmed/7987388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Analysis+of+the+p16+gene+(CDKN2)+as+a+candidate+for+the+chromosome+9p+melanoma+susceptibility+locus&amp;author=A+Kamb&amp;volume=8&amp;publication_year=1994&amp;pages=23-26&amp;pmid=7987388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Bishop DT, et al.  Genome-wide association study identifies three loci associated with melanoma risk. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:920–925.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2741419/">PMC free article</a>]</span> [<a href="/pubmed/19578364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+three+loci+associated+with+melanoma+risk&amp;author=DT+Bishop&amp;volume=41&amp;publication_year=2009&amp;pages=920-925&amp;pmid=19578364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Falchi M, et al.  Genome-wide association study identifies variants at 9p21 and 22q13 associated with development of cutaneous nevi. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:915–919.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080738/">PMC free article</a>]</span> [<a href="/pubmed/19578365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+9p21+and+22q13+associated+with+development+of+cutaneous+nevi&amp;author=M+Falchi&amp;volume=41&amp;publication_year=2009&amp;pages=915-919&amp;pmid=19578365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Stacey SN, et al.  New common variants affecting susceptibility to basal cell carcinoma. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:909–914.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2973331/">PMC free article</a>]</span> [<a href="/pubmed/19578363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=New+common+variants+affecting+susceptibility+to+basal+cell+carcinoma&amp;author=SN+Stacey&amp;volume=41&amp;publication_year=2009&amp;pages=909-914&amp;pmid=19578363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Shete S, et al.  Genome-wide association study identifies five susceptibility loci for glioma. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:899–904.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4501476/">PMC free article</a>]</span> [<a href="/pubmed/19578367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+five+susceptibility+loci+for+glioma&amp;author=S+Shete&amp;volume=41&amp;publication_year=2009&amp;pages=899-904&amp;pmid=19578367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Wrensch M, et al.  Variants in the <em>CDKN2B</em> and <em>RTEL1</em> regions are associated with high-grade glioma susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:905–908.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2923561/">PMC free article</a>]</span> [<a href="/pubmed/19578366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Variants+in+the+CDKN2B+and+RTEL1+regions+are+associated+with+high-grade+glioma+susceptibility&amp;author=M+Wrensch&amp;volume=41&amp;publication_year=2009&amp;pages=905-908&amp;pmid=19578366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Zeggini E, et al.  Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">The Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Liu Y, et al.  INK4/ARF transcript expression is associated with chromosome 9p21 variants linked to atherosclerosis. <span><span class="ref-journal">PLoS One. </span>2009;<span class="ref-vol">4</span>:e5027.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2660422/">PMC free article</a>]</span> [<a href="/pubmed/19343170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=INK4/ARF+transcript+expression+is+associated+with+chromosome+9p21+variants+linked+to+atherosclerosis&amp;author=Y+Liu&amp;volume=4&amp;publication_year=2009&amp;pages=e5027&amp;pmid=19343170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Karlseder J, et al.  Patterns of DNA amplification at band q13 of chromosome 11 in human breast cancer. <span><span class="ref-journal">Genes Chromosom. Cancer. </span>1994;<span class="ref-vol">9</span>:42–48.</span> [<a href="/pubmed/7507699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosom.+Cancer&amp;title=Patterns+of+DNA+amplification+at+band+q13+of+chromosome+11+in+human+breast+cancer&amp;author=J+Karlseder&amp;volume=9&amp;publication_year=1994&amp;pages=42-48&amp;pmid=7507699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Ornitz DM, et al.  Receptor specificity of the fibroblast growth factor family. <span><span class="ref-journal">J. Biol. Chem. </span>1996;<span class="ref-vol">271</span>:15292–15297.</span> [<a href="/pubmed/8663044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Receptor+specificity+of+the+fibroblast+growth+factor+family&amp;author=DM+Ornitz&amp;volume=271&amp;publication_year=1996&amp;pages=15292-15297&amp;pmid=8663044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Gianfrancesco F, et al.  Identification of a novel gene and a common variant associated with uric acid nephrolithiasis in a Sardinian genetic isolate. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2003;<span class="ref-vol">72</span>:1479–1491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180308/">PMC free article</a>]</span> [<a href="/pubmed/12740763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Identification+of+a+novel+gene+and+a+common+variant+associated+with+uric+acid+nephrolithiasis+in+a+Sardinian+genetic+isolate&amp;author=F+Gianfrancesco&amp;volume=72&amp;publication_year=2003&amp;pages=1479-1491&amp;pmid=12740763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Barrett JC, et al.  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn’s+disease&amp;author=JC+Barrett&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Udler MS, et al.  FGFR2 variants and breast cancer risk: fine-scale mapping using African American studies and analysis of chromatin conformation. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:1692–1703.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2733817/">PMC free article</a>]</span> [<a href="/pubmed/19223389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=FGFR2+variants+and+breast+cancer+risk:+fine-scale+mapping+using+African+American+studies+and+analysis+of+chromatin+conformation&amp;author=MS+Udler&amp;volume=18&amp;publication_year=2009&amp;pages=1692-1703&amp;pmid=19223389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Power C, Elliott J. Cohort profile: 1958 British birth cohort (National Child Development Study) <span><span class="ref-journal">Int. J. Epidemiol. </span>2006;<span class="ref-vol">35</span>:34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol&amp;title=Cohort+profile:+1958+British+birth+cohort+(National+Child+Development+Study)&amp;author=C+Power&amp;author=J+Elliott&amp;volume=35&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Teo YY, et al.  A genotype calling algorithm for the Illumina BeadArray platform. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:2741–2746.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666488/">PMC free article</a>]</span> [<a href="/pubmed/17846035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+the+Illumina+BeadArray+platform&amp;author=YY+Teo&amp;volume=23&amp;publication_year=2007&amp;pages=2741-2746&amp;pmid=17846035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome- wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-+wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when linkage phase is ambiguous. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">70</span>:425–434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384917/">PMC free article</a>]</span> [<a href="/pubmed/11791212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Score+tests+for+association+between+traits+and+haplotypes+when+linkage+phase+is+ambiguous&amp;author=DJ+Schaid&amp;author=CM+Rowland&amp;author=DE+Tines&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=70&amp;publication_year=2002&amp;pages=425-434&amp;pmid=11791212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. GenABEL: an R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenABEL:+an+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3632836&amp;issue-id=222125&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139857456168848title">Abstract</a></li><li><a href="#S2title">ONLINE METHODS</a></li><li><a href="#S7title">ACKNOWLEDGMENTS</a></li><li><a href="#idm139857457883184title">Footnotes</a></li><li><a href="#idm139857453513440title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3632836/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3632836/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3632836/pdf/emss-52673.pdf">PDF (454K)</a></li> | <li><a href="#" data-citationid="PMC3632836" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3632836%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3632836%2F&amp;text=Genome-wide%20association%20study%20identifies%20five%20new%20breast%20cancer%20susceptibility%20loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3632836%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20453838" data-aiid="3632836" data-aid="3632836" data-iid="222125" data-domainid="358" data-domain="wtpa" data-accid="PMC3632836" data-md5="2a416a758b3374758813ffd27f5704d5"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T05:19:19-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88C57BDD50C0210000000004BD0369&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3632836%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3632836/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>