<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3298855/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies novel loci predisposing to cutaneous melanoma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3298855/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study identifies novel loci predisposing to cutaneous melanoma"><meta name="citation_authors" content="Christopher I. Amos, Li-E Wang, Jeffrey E. Lee, Jeffrey E. Gershenwald, Wei V. Chen, Shenying Fang, Roman Kosoy, Mingfeng Zhang, Abrar A. Qureshi, Selina Vattathil, Christopher W. Schacherer, Julie M. Gardner, Yuling Wang, D. Tim Bishop, Jennifer H. Barrett, Stuart MacGregor, Nicholas K. Hayward, Nicholas G. Martin, David L. Duffy, Graham J. Mann, Anne Cust, John Hopper, Kevin M. Brown, Elizabeth A. Grimm, Yaji Xu, Younghun Han, Kaiyan Jing, Caitlin McHugh, Cathy C. Laurie, Kim F. Doheny, Elizabeth W. Pugh, Michael F. Seldin, Jiali Han, Qingyi Wei, GenoMEL Investigators, Q-Mega Investigators, AMFS Investigators"><meta name="citation_date" content="15 December 2011"><meta name="citation_issue" content="24"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="5012"><meta name="citation_doi" content="10.1093/hmg/ddr415"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3298855/?report=abstract"><meta name="citation_pmid" content="21926416"><meta name="DC.Title" content="Genome-wide association study identifies novel loci predisposing to cutaneous melanoma"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Li-E Wang"><meta name="DC.Contributor" content="Jeffrey E. Lee"><meta name="DC.Contributor" content="Jeffrey E. Gershenwald"><meta name="DC.Contributor" content="Wei V. Chen"><meta name="DC.Contributor" content="Shenying Fang"><meta name="DC.Contributor" content="Roman Kosoy"><meta name="DC.Contributor" content="Mingfeng Zhang"><meta name="DC.Contributor" content="Abrar A. Qureshi"><meta name="DC.Contributor" content="Selina Vattathil"><meta name="DC.Contributor" content="Christopher W. Schacherer"><meta name="DC.Contributor" content="Julie M. Gardner"><meta name="DC.Contributor" content="Yuling Wang"><meta name="DC.Contributor" content="D. Tim Bishop"><meta name="DC.Contributor" content="Jennifer H. Barrett"><meta name="DC.Contributor" content="Stuart MacGregor"><meta name="DC.Contributor" content="Nicholas K. Hayward"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="David L. Duffy"><meta name="DC.Contributor" content="Graham J. Mann"><meta name="DC.Contributor" content="Anne Cust"><meta name="DC.Contributor" content="John Hopper"><meta name="DC.Contributor" content="Kevin M. Brown"><meta name="DC.Contributor" content="Elizabeth A. Grimm"><meta name="DC.Contributor" content="Yaji Xu"><meta name="DC.Contributor" content="Younghun Han"><meta name="DC.Contributor" content="Kaiyan Jing"><meta name="DC.Contributor" content="Caitlin McHugh"><meta name="DC.Contributor" content="Cathy C. Laurie"><meta name="DC.Contributor" content="Kim F. Doheny"><meta name="DC.Contributor" content="Elizabeth W. Pugh"><meta name="DC.Contributor" content="Michael F. Seldin"><meta name="DC.Contributor" content="Jiali Han"><meta name="DC.Contributor" content="Qingyi Wei"><meta name="DC.Contributor" content="GenoMEL Investigators"><meta name="DC.Contributor" content="Q-Mega Investigators"><meta name="DC.Contributor" content="AMFS Investigators"><meta name="DC.Date" content="2011 Dec 15"><meta name="DC.Identifier" content="10.1093/hmg/ddr415"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies novel loci predisposing to cutaneous melanoma"><meta property="og:type" content="article"><meta property="og:description" content="We performed a multistage genome-wide association study of melanoma. In a discovery cohort of 1804 melanoma cases and 1026 controls, we identified loci at chromosomes 15q13.1 (HERC2/OCA2 region) and 16q24.3 (MC1R) regions that reached genome-wide significance ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3298855/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3298855/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3298855/pdf/ddr415.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89FE04DD5319C100000000022F0160.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3298855%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3298855</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2011 Dec 15; 20(24): 5012–5023. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Sep 17. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddr415" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddr415</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3298855</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21926416">21926416</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies novel loci predisposing to cutaneous melanoma<sup><sup>†</sup></sup></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352880471888" co-class="co-affbox">Christopher I. Amos</a>,<sup>1,</sup><sup>*</sup><sup>‡</sup> <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352865641264" co-class="co-affbox">Li-E Wang</a>,<sup>1,</sup><sup>‡</sup> <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352950279312" co-class="co-affbox">Jeffrey E. Lee</a>,<sup>2,</sup><sup>‡</sup> <a href="/pubmed/?term=Gershenwald%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352873636624" co-class="co-affbox">Jeffrey E. Gershenwald</a>,<sup>2</sup> <a href="/pubmed/?term=Chen%20WV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352949819536" co-class="co-affbox">Wei V. Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Fang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352949817312" co-class="co-affbox">Shenying Fang</a>,<sup>1</sup> <a href="/pubmed/?term=Kosoy%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352875099872" co-class="co-affbox">Roman Kosoy</a>,<sup>4</sup> <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352871050560" co-class="co-affbox">Mingfeng Zhang</a>,<sup>5</sup> <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352878473120" co-class="co-affbox">Abrar A. Qureshi</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Vattathil%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352869843456" co-class="co-affbox">Selina Vattathil</a>,<sup>1</sup> <a href="/pubmed/?term=Schacherer%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352877674928" co-class="co-affbox">Christopher W. Schacherer</a>,<sup>2</sup> <a href="/pubmed/?term=Gardner%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352879471072" co-class="co-affbox">Julie M. Gardner</a>,<sup>2</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352879468848" co-class="co-affbox">Yuling Wang</a>,<sup>2</sup> <a href="/pubmed/?term=Tim%20Bishop%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352874468096" co-class="co-affbox">D. Tim Bishop</a>,<sup>7</sup> <a href="/pubmed/?term=Barrett%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352877747008" co-class="co-affbox">Jennifer H. Barrett</a>,<sup>7</sup> <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352872815920" co-class="co-affbox">Stuart MacGregor</a>,<sup>8</sup> <a href="/pubmed/?term=Hayward%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352875844240" co-class="co-affbox">Nicholas K. Hayward</a>,<sup>8</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352875842016" co-class="co-affbox">Nicholas G. Martin</a>,<sup>8</sup> <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352865539120" co-class="co-affbox">David L. Duffy</a>,<sup>8</sup> <a href="/pubmed/?term=Mann%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352874167824" co-class="co-affbox">Graham J. Mann</a>,<sup>9</sup> <a href="/pubmed/?term=Cust%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352915498432" co-class="co-affbox">Anne Cust</a>,<sup>10</sup> <a href="/pubmed/?term=Hopper%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352916261456" co-class="co-affbox">John Hopper</a>,<sup>11</sup> <a href="/pubmed/?term=Brown%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352881994464" co-class="co-affbox">Kevin M. Brown</a>,<sup>12</sup> <a href="/pubmed/?term=Grimm%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352881992240" co-class="co-affbox">Elizabeth A. Grimm</a>,<sup>3</sup> <a href="/pubmed/?term=Xu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352872892480" co-class="co-affbox">Yaji Xu</a>,<sup>1</sup> <a href="/pubmed/?term=Han%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352876060096" co-class="co-affbox">Younghun Han</a>,<sup>1</sup> <a href="/pubmed/?term=Jing%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352916883952" co-class="co-affbox">Kaiyan Jing</a>,<sup>1</sup> <a href="/pubmed/?term=McHugh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352951326080" co-class="co-affbox">Caitlin McHugh</a>,<sup>13</sup> <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352951323856" co-class="co-affbox">Cathy C. Laurie</a>,<sup>13</sup> <a href="/pubmed/?term=Doheny%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352876383488" co-class="co-affbox">Kim F. Doheny</a>,<sup>14</sup> <a href="/pubmed/?term=Pugh%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352876328960" co-class="co-affbox">Elizabeth W. Pugh</a>,<sup>14</sup> <a href="/pubmed/?term=Seldin%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352874964800" co-class="co-affbox">Michael F. Seldin</a>,<sup>4,</sup><sup>‡</sup> <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352873591328" co-class="co-affbox">Jiali Han</a>,<sup>5,</sup><sup>6,</sup><sup>15,</sup><sup>‡</sup> and  <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416" class="affpopup" co-rid="_co_idm140352875159056" co-class="co-affbox">Qingyi Wei</a><sup>1,</sup><sup>*</sup><sup>‡</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140352880471888"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Christopher I. Amos</a></div></div><div id="_co_idm140352865641264"><h3 class="no_margin">Li-E Wang</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Li-E Wang</a></div></div><div id="_co_idm140352950279312"><h3 class="no_margin">Jeffrey E. Lee</h3><p><sup>2</sup>Department of Surgical Oncology,  and</p><div>Find articles by <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Jeffrey E. Lee</a></div></div><div id="_co_idm140352873636624"><h3 class="no_margin">Jeffrey E. Gershenwald</h3><p><sup>2</sup>Department of Surgical Oncology,  and</p><div>Find articles by <a href="/pubmed/?term=Gershenwald%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Jeffrey E. Gershenwald</a></div></div><div id="_co_idm140352949819536"><h3 class="no_margin">Wei V. Chen</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Chen%20WV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Wei V. Chen</a></div></div><div id="_co_idm140352949817312"><h3 class="no_margin">Shenying Fang</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Fang%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Shenying Fang</a></div></div><div id="_co_idm140352875099872"><h3 class="no_margin">Roman Kosoy</h3><p><sup>4</sup>Rowe Program in Genetics, University of California, UC Davis, Davis, CA, USA,</p><div>Find articles by <a href="/pubmed/?term=Kosoy%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Roman Kosoy</a></div></div><div id="_co_idm140352871050560"><h3 class="no_margin">Mingfeng Zhang</h3><p><sup>5</sup>Department of Dermatology,  and</p><div>Find articles by <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Mingfeng Zhang</a></div></div><div id="_co_idm140352878473120"><h3 class="no_margin">Abrar A. Qureshi</h3><p><sup>5</sup>Department of Dermatology,  and</p><p><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA 02115, USA,</p><div>Find articles by <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Abrar A. Qureshi</a></div></div><div id="_co_idm140352869843456"><h3 class="no_margin">Selina Vattathil</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Vattathil%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Selina Vattathil</a></div></div><div id="_co_idm140352877674928"><h3 class="no_margin">Christopher W. Schacherer</h3><p><sup>2</sup>Department of Surgical Oncology,  and</p><div>Find articles by <a href="/pubmed/?term=Schacherer%20CW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Christopher W. Schacherer</a></div></div><div id="_co_idm140352879471072"><h3 class="no_margin">Julie M. Gardner</h3><p><sup>2</sup>Department of Surgical Oncology,  and</p><div>Find articles by <a href="/pubmed/?term=Gardner%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Julie M. Gardner</a></div></div><div id="_co_idm140352879468848"><h3 class="no_margin">Yuling Wang</h3><p><sup>2</sup>Department of Surgical Oncology,  and</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Yuling Wang</a></div></div><div id="_co_idm140352874468096"><h3 class="no_margin">D. Tim Bishop</h3><p><sup>7</sup>Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, UK,</p><div>Find articles by <a href="/pubmed/?term=Tim%20Bishop%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">D. Tim Bishop</a></div></div><div id="_co_idm140352877747008"><h3 class="no_margin">Jennifer H. Barrett</h3><p><sup>7</sup>Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, UK,</p><div>Find articles by <a href="/pubmed/?term=Barrett%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Jennifer H. Barrett</a></div></div><div id="_co_idm140352872815920"><h3 class="no_margin">Stuart MacGregor</h3><p><sup>8</sup>Queensland Institute of Medical Research, Brisbane QLD 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=MacGregor%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Stuart MacGregor</a></div></div><div id="_co_idm140352875844240"><h3 class="no_margin">Nicholas K. Hayward</h3><p><sup>8</sup>Queensland Institute of Medical Research, Brisbane QLD 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Hayward%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Nicholas K. Hayward</a></div></div><div id="_co_idm140352875842016"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>8</sup>Queensland Institute of Medical Research, Brisbane QLD 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Nicholas G. Martin</a></div></div><div id="_co_idm140352865539120"><h3 class="no_margin">David L. Duffy</h3><p><sup>8</sup>Queensland Institute of Medical Research, Brisbane QLD 4029, Australia,</p><div>Find articles by <a href="/pubmed/?term=Duffy%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">David L. Duffy</a></div></div><div id="_co_idm140352874167824"><h3 class="no_margin">Graham J. Mann</h3><p><sup>9</sup>Westmead Institute of Cancer Research, University of Sydney at Westmead Millennium Institute and Melanoma Institute Australia, Westmead, NSW 2145, Australia,</p><div>Find articles by <a href="/pubmed/?term=Mann%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Graham J. Mann</a></div></div><div id="_co_idm140352915498432"><h3 class="no_margin">Anne Cust</h3><p><sup>10</sup>Cancer Epidemiology and Services Research (CESR), Sydney School of Public Health, The University of Sydney, Sydney, Australia,</p><div>Find articles by <a href="/pubmed/?term=Cust%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Anne Cust</a></div></div><div id="_co_idm140352916261456"><h3 class="no_margin">John Hopper</h3><p><sup>11</sup>Centre for Molecular, Environmental, Genetic and Analytic (MEGA) Epidemiology, Melbourne School of Population Health, University of Melbourne, Melbourne, Australia,</p><div>Find articles by <a href="/pubmed/?term=Hopper%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">John Hopper</a></div></div><div id="_co_idm140352881994464"><h3 class="no_margin">Kevin M. Brown</h3><p><sup>12</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MD, USA,</p><div>Find articles by <a href="/pubmed/?term=Brown%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Kevin M. Brown</a></div></div><div id="_co_idm140352881992240"><h3 class="no_margin">Elizabeth A. Grimm</h3><p><sup>3</sup>Department of Melanoma Medical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA,</p><div>Find articles by <a href="/pubmed/?term=Grimm%20EA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Elizabeth A. Grimm</a></div></div><div id="_co_idm140352872892480"><h3 class="no_margin">Yaji Xu</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Xu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Yaji Xu</a></div></div><div id="_co_idm140352876060096"><h3 class="no_margin">Younghun Han</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Han%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Younghun Han</a></div></div><div id="_co_idm140352916883952"><h3 class="no_margin">Kaiyan Jing</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Jing%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Kaiyan Jing</a></div></div><div id="_co_idm140352951326080"><h3 class="no_margin">Caitlin McHugh</h3><p><sup>13</sup>Department of Biostatistics, University of Washington, Seattle, WA, USA,</p><div>Find articles by <a href="/pubmed/?term=McHugh%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Caitlin McHugh</a></div></div><div id="_co_idm140352951323856"><h3 class="no_margin">Cathy C. Laurie</h3><p><sup>13</sup>Department of Biostatistics, University of Washington, Seattle, WA, USA,</p><div>Find articles by <a href="/pubmed/?term=Laurie%20CC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Cathy C. Laurie</a></div></div><div id="_co_idm140352876383488"><h3 class="no_margin">Kim F. Doheny</h3><p><sup>14</sup>Center for Inherited Disease Research, School of Medicine, Johns Hopkins University, Baltimore, MD, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Doheny%20KF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Kim F. Doheny</a></div></div><div id="_co_idm140352876328960"><h3 class="no_margin">Elizabeth W. Pugh</h3><p><sup>14</sup>Center for Inherited Disease Research, School of Medicine, Johns Hopkins University, Baltimore, MD, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Pugh%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Elizabeth W. Pugh</a></div></div><div id="_co_idm140352874964800"><h3 class="no_margin">Michael F. Seldin</h3><p><sup>4</sup>Rowe Program in Genetics, University of California, UC Davis, Davis, CA, USA,</p><div>Find articles by <a href="/pubmed/?term=Seldin%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Michael F. Seldin</a></div></div><div id="_co_idm140352873591328"><h3 class="no_margin">Jiali Han</h3><p><sup>5</sup>Department of Dermatology,  and</p><p><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA 02115, USA,</p><p><sup>15</sup>Department of Epidemiology, Channing Laboratory, Harvard University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Jiali Han</a></div></div><div id="_co_idm140352875159056"><h3 class="no_margin">Qingyi Wei</h3><p><sup>1</sup>Department of Epidemiology,</p><div>Find articles by <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21926416">Qingyi Wei</a></div></div></div></div><div class="half_rhythm"><div class="contrib-group fm-author">GenoMEL Investigators, Q-Mega Investigators, and  AMFS Investigators<div class="fm-panel half_rhythm"><div class="fm-authors-info fm-panel half_rhythm"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Epidemiology,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Surgical Oncology,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Melanoma Medical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Rowe Program in Genetics, University of California, UC Davis, Davis, CA, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Dermatology,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Channing Laboratory, Department of Medicine, Brigham and Women's Hospital, Harvard Medical School, Boston, MA 02115, USA,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, UK,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Queensland Institute of Medical Research, Brisbane QLD 4029, Australia,</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Westmead Institute of Cancer Research, University of Sydney at Westmead Millennium Institute and Melanoma Institute Australia, Westmead, NSW 2145, Australia,</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Cancer Epidemiology and Services Research (CESR), Sydney School of Public Health, The University of Sydney, Sydney, Australia,</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Centre for Molecular, Environmental, Genetic and Analytic (MEGA) Epidemiology, Melbourne School of Population Health, University of Melbourne, Melbourne, Australia,</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Laboratory of Translational Genomics, Division of Cancer Epidemiology and Genetics, National Cancer Institute, Bethesda, MD, USA,</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Department of Biostatistics, University of Washington, Seattle, WA, USA,</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Center for Inherited Disease Research, School of Medicine, Johns Hopkins University, Baltimore, MD, USA,  and</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Department of Epidemiology, Channing Laboratory, Harvard University, Boston, MA, USA</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm"></div></div></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140352879268928_ai idm140352876254384_ai idm140352869855392_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140352879268928_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140352879268928_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140352879268928_ai" style="display:none"><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Epidemiology, Unit 1340, The University of Texas MD Anderson Cancer Center, 1155 Pressler Boulevard, Houston, TX 77030, USA. Tel: +1 7137452480; Fax: +1 7137928261; Email: <a href="mailto:dev@null" data-email="gro.nosrednadm@somac" class="oemail">gro.nosrednadm@somac</a> (C.I.A.); <a href="mailto:dev@null" data-email="gro.nosrednadm@iewq" class="oemail">gro.nosrednadm@iewq</a> (Q.W.)</div><div id="AN1"><sup>†</sup>See Appendix for the list of investigators involved in the study.</div><div id="AN2"><sup>‡</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140352879268928_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Jun 27; Accepted 2011 Sep 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140352879268928_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3298855/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3298855/bin/supp_20_24_5012__index.html" data-ga-action="click_feat_suppl">supp_20_24_5012__index.html</a><span style="color:gray"> (792 bytes)</span></div><div class="small guid">GUID:&nbsp;F7805DF9-2E54-45F9-92E6-646644C43102</div>
<div><a href="/pmc/articles/PMC3298855/bin/supp_ddr415_ddr415supp.doc" data-ga-action="click_feat_suppl">supp_ddr415_ddr415supp.doc</a><span style="color:gray"> (941K)</span></div><div class="small guid">GUID:&nbsp;62FE25B0-5A10-49BC-9673-FC2266CD20A8</div>
</div></dd></dl></div><div id="idm140352867994704" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140352867994704title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">We performed a multistage genome-wide association study of melanoma. In a discovery cohort of 1804 melanoma cases and 1026 controls, we identified loci at chromosomes 15q13.1 (<em>HERC2/OCA2</em> region) and 16q24.3 (<em>MC1R</em>) regions that reached genome-wide significance within this study and also found strong evidence for genetic effects on susceptibility to melanoma from markers on chromosome 9p21.3 in the <em>p16/ARF</em> region and on chromosome 1q21.3 (<em>ARNT/LASS2/ANXA9</em> region). The most significant single-nucleotide polymorphisms (SNPs) in the 15q13.1 locus (rs1129038 and rs12913832) lie within a genomic region that has profound effects on eye and skin color; notably, 50% of variability in eye color is associated with variation in the SNP rs12913832. Because eye and skin colors vary across European populations, we further evaluated the associations of the significant SNPs after carefully adjusting for European substructure. We also evaluated the top 10 most significant SNPs by using data from three other genome-wide scans. Additional <em>in silico</em> data provided replication of the findings from the most significant region on chromosome 1q21.3 rs7412746 (<em>P</em> = 6 × 10<sup>−10</sup>). Together, these data identified several candidate genes for additional studies to identify causal variants predisposing to increased risk for developing melanoma.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p4" class="p p-first">Cutaneous melanoma (CM) is among the more common cancers, with more than 70 000 estimated new cases in the USA (excluding melanoma <em>in situ</em>) in 2011 (<a href="#DDR415C1" rid="DDR415C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538866">1</a>). The risk for siblings of index cases who have CM is ∼2-fold higher than that for the general population (<a href="#DDR415C2" rid="DDR415C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538852">2</a>). Linkage studies and genome-wide association studies (GWASs) have identified several loci influencing CM risk. Cyclin-dependent kinase inhibitor 2A (<em>CDKN2A</em> or <em>p16</em>), an alternate reading frame of <em>CDKN2A</em> (<em>ARF</em> or <em>p14</em>), and cyclin-dependent kinase 4 (<em>CDK4</em>) in region 9p21.3 have been identified as high-penetrance CM susceptibility genes (<a href="#DDR415C3" rid="DDR415C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538864">3</a>), but mutations in these genes are rare and explain only a small fraction of familial CM risk. However, GWASs found evidence that common variation in the <em>CDKN2A</em> region also commonly influence risk for melanoma (<a href="#DDR415C4" rid="DDR415C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538867">4</a>,<a href="#DDR415C5" rid="DDR415C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538878">5</a>). Mutations in the melanocortin 1 receptor (<em>MC1R</em>) gene in the chromosomal region 16q24.3 influence skin pigmentation, and a candidate gene analysis followed by several confirmatory studies documented an ∼40% increased risk of CM in those who carried the variants conferring lighter skin pigmentation according to specific mutations (<a href="#DDR415C6" rid="DDR415C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538851">6</a>–<a href="#DDR415C8" rid="DDR415C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538854">8</a>).</p><p id="__p5" class="p p-last">In addition, GWASs have identified several other loci that may contribute to CM risk. GWAS analyses have revealed several additional genes associated with skin pigmentation, including <em>TYR</em> (tyrosinase, chromosome 11q21) (<a href="#DDR415C9" rid="DDR415C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304778333">9</a>), <em>TYRP1</em> (tyrosinase-related protein 1, chromosome 9p23) (<a href="#DDR415C10" rid="DDR415C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538874">10</a>), <em>SLC45A2</em> (solute carrier family 45, member 2, chromosome 5p13.3), <em>SLC24A4</em> (solute carrier family 24, member 4, chromosome 14q32.12) and <em>IRF4</em> (interferon regulatory factor 4, chromosome 6p25.3). A case–control study (<a href="#DDR415C11" rid="DDR415C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538862">11</a>) that used a pooling approach to identify common variants influencing CM risk found several genetic factors influencing risk across a broad region of chromosome 20q11.2 that encompasses the genetic loci, <em>PIGU, EIF2S2, NCOA6</em> and <em>MYH17B</em>. Finally, the locus <em>PLA2G6</em> (<a href="#DDR415C4" rid="DDR415C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538868">4</a>) on chromosome 22q13 was also previously associated with CM risk in a GWAS. The goal of the current GWAS was to identify novel variants associated with CM risk by using dense single-nucleotide polymorphism (SNP) arrays on a large population of CM cases that were collected uniformly from a single cancer center in the USA and to then validate these findings with use of additional data from US, Australian and European populations.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p6" class="p p-first">To identify risk variants for CM, we used an Illumina Omni1-Quad_v1-0_B array to genotype 1 016 423 SNPs in 3115 participants. After the application of quality control (QC) criteria (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a> and Fig.&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F1" rid-ob="ob-DDR415F1" co-legend-rid="lgnd_DDR415F1"><span>1</span></a>), genotypes were available for 1804 Caucasian CM patients presenting to clinics (other than dermatology) at The University of Texas MD Anderson Cancer Center and 1026 Caucasian cancer-free controls who were friends or acquaintances accompanying patients at their clinical visits. We analyzed 1 012 904 probes with a mean sample call rate of 99.66%. After applying QC filters, we retained data for analysis from 818 977 SNPs (818 237 autosomal or X chromosome SNPs and 740 psuedoautosomal SNPs) that had a minor allele frequency (MAF) of &gt;1%, were genotyped for 95% or more of retained participants and did not deviate from Hardy–Weinberg equilibrium (<em>P</em>&gt; 1 × 10<sup>−5</sup>). Individual genotypes that had Illumina GenTrain version 1.0 quality scores of ≤0.15 were set to missing. Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F1" rid-ob="ob-DDR415F1" co-legend-rid="lgnd_DDR415F1"><span>1</span></a> and Supplementary Tables <a href="/pmc/articles/PMC3298855/table/DDR415TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB1" rid-ob="ob-DDR415TB1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">Tables1</span></span><span>1</span></a> and <a href="/pmc/articles/PMC3298855/table/DDR415TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB2" rid-ob="ob-DDR415TB2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and2</span></span><span>2</span></a> present an overall description of the sample processing and <em>in silico</em> replication phases of this study.


</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR415TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p7">Associations of SNPs previously found in GWASs to be associated with CM risk, A1 is the minor allele in forward strand</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352873458176" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Chr.</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Source</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left"><em>r</em><sup>2</sup></th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Reference</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Ref. All.</th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">95% CI</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs16891982</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>SLC45A2</em></td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">33 987 450</td><td rowspan="1" colspan="1">0.0026</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.54</td><td rowspan="1" colspan="1">0.37–0.81</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs35391</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>SLC45A2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">33 991 430</td><td rowspan="1" colspan="1">0.020</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.37–0.91</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">rs28777</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>SLC45A2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">33 994 716</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.56</td><td rowspan="1" colspan="1">0.37–0.84</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs1408799</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>TYRP1</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">9,10, 13</td><td rowspan="1" colspan="1">12 662 097</td><td rowspan="1" colspan="1">0.70</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.87–1.10</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs2218220</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1">near <em>MTAP</em></td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">21 746 089</td><td rowspan="1" colspan="1">0.00014</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">0.73–0.90</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs1335510</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1">near <em>MTAP</em></td><td rowspan="1" colspan="1">1.00</td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">21 747 803</td><td rowspan="1" colspan="1">5.01 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.71–0.89</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs935053</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1">near <em>MTAP</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">21 773 922</td><td rowspan="1" colspan="1">4.49 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.71–0.89</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs10757257</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>MTAP</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.38</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">21 796 564</td><td rowspan="1" colspan="1">1.96 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.78</td><td rowspan="1" colspan="1">0.70–0.88</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs7023329</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>MTAP</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">21 806 528</td><td rowspan="1" colspan="1">1.86 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.79</td><td rowspan="1" colspan="1">0.71–0.88</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1042602</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.36</td><td rowspan="1" colspan="1">4, 10</td><td rowspan="1" colspan="1">88 551 344</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.89</td><td rowspan="1" colspan="1">0.79–1.00</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1393350</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">4, 10</td><td rowspan="1" colspan="1">88 650 694</td><td rowspan="1" colspan="1">0.0085</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">1.04–1.33</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1847142</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">88 661 222</td><td rowspan="1" colspan="1">0.0021</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.20</td><td rowspan="1" colspan="1">1.07–1.35</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs10830253</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">88 667 691</td><td rowspan="1" colspan="1">0.0022</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.20</td><td rowspan="1" colspan="1">1.07–1.34</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs1806319</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>TYR</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.39</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">88 677 584</td><td rowspan="1" colspan="1">0.0050</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.17</td><td rowspan="1" colspan="1">1.05–1.31</td></tr><tr><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">rs12896399</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>SLC24A4</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.46</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">91 843 416</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.06</td><td rowspan="1" colspan="1">0.95–1.18</td></tr><tr><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">rs1800407</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>OCA2</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">25 903 913</td><td rowspan="1" colspan="1">0.67</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.95</td><td rowspan="1" colspan="1">0.78–1.17</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs1805006</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>MC1R</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">88 513 419</td><td rowspan="1" colspan="1">0.053</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.67</td><td rowspan="1" colspan="1">0.97–2.86</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs1805007</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>MC1R</em></td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">10, 13</td><td rowspan="1" colspan="1">88 513 618</td><td rowspan="1" colspan="1">4.1 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.74</td><td rowspan="1" colspan="1">1.44–2.11</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs1805008</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>MC1R</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">10,13</td><td rowspan="1" colspan="1">88 513 645</td><td rowspan="1" colspan="1">0.001</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.37</td><td rowspan="1" colspan="1">1.13–1.67</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs4911414</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>LOC729547</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32 193 105</td><td rowspan="1" colspan="1">0.090</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.11</td><td rowspan="1" colspan="1">0.98–1.24</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs1015362</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>EIF2S2</em></td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32 202 273</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.94</td><td rowspan="1" colspan="1">0.83–1.06</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs910873</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>PIGU</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32 635 433</td><td rowspan="1" colspan="1">0.00017</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.42</td><td rowspan="1" colspan="1">1.18–1.71</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs17305573</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>PIGU</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32 643 813</td><td rowspan="1" colspan="1">0.00017</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.42</td><td rowspan="1" colspan="1">1.18–1.71</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs4911442</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>NCOA6</em></td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32 818 707</td><td rowspan="1" colspan="1">0.0029</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.27</td><td rowspan="1" colspan="1">1.08–1.49</td></tr><tr><td rowspan="1" colspan="1">20</td><td rowspan="1" colspan="1">rs1885120</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>MYH7B</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">33 040 650</td><td rowspan="1" colspan="1">0.00016</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.43</td><td rowspan="1" colspan="1">1.18–1.72</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs2284063</td><td rowspan="1" colspan="1">Imputed</td><td rowspan="1" colspan="1"><em>PLA2G6</em></td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">36 874 244</td><td rowspan="1" colspan="1">0.0045</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.75–0.95</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs6001027</td><td rowspan="1" colspan="1">Typed</td><td rowspan="1" colspan="1"><em>PLA2G6</em></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.34</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">36 875 565</td><td rowspan="1" colspan="1">0.0039</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.75–0.95</td></tr></tbody></table></div><div id="largeobj_idm140352873458176" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3298855/table/DDR415TB1/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR415TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p8">Association of <em>HERC2</em> rs12913832 with pigmentary phenotypes in the MD Anderson study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352871998800" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">GWAS phenotype</th><th rowspan="1" colspan="1" align="left">WT/VT</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Estimate</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Hair color (blonde to black)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2625</td><td rowspan="1" colspan="1">0.3924</td><td rowspan="1" colspan="1">0.0407</td><td rowspan="1" colspan="1">5.35 × 10<sup>−22</sup></td></tr><tr><td rowspan="1" colspan="1">Eye color (blue to brown)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2620</td><td rowspan="1" colspan="1">0.9193</td><td rowspan="1" colspan="1">0.0283</td><td rowspan="1" colspan="1">1.79 × 10<sup>−231</sup></td></tr><tr><td rowspan="1" colspan="1">Tanning ability (none to deep)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2589</td><td rowspan="1" colspan="1">0.2008</td><td rowspan="1" colspan="1">0.0489</td><td rowspan="1" colspan="1">4.07 × 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140352871998800" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/table/DDR415TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140352879649008"><p id="__p9" class="p p-first-last">WT, wild-type; VT, variant type; SE, standard error.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F1" co-legend-rid="lgnd_DDR415F1"><a href="/pmc/articles/PMC3298855/figure/DDR415F1/" target="figure" rid-figpopup="DDR415F1" rid-ob="ob-DDR415F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352872736528" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F1/" target="figure" rid-figpopup="DDR415F1" rid-ob="ob-DDR415F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41501.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41501.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41501.jpg"></a></div><div id="largeobj_idm140352872736528" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F1"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F1/" target="figure" rid-figpopup="DDR415F1" rid-ob="ob-DDR415F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p10">Description of the flow of samples and markers through the genotyping study reported for CM cases and controls from MD Anderson Cancer Center.</p></div></div></div><p id="__p11">Initial analyses were performed using the additive logistic regression model implemented in PLINK (<a href="#DDR415C12" rid="DDR415C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538860">12</a>). The Q–Q plots for the discovery sample are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figures S1 and S2</a>. The genomic inflation factor (<em>λ</em>) for the discovery sample was 1.020, and this was reduced when the first two principal components (PCs) were included as covariates (<em>λ</em> = 1.011). Association results across the genome after correcting for the two PCs are shown in Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F2" rid-ob="ob-DDR415F2" co-legend-rid="lgnd_DDR415F2"><span>2</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a> (association tests with <em>P</em> &lt; 1 × 10<sup>−4</sup>). The two regions reaching genome-wide significance (5 × 10<sup>−8</sup>) are located on 15q13.1 (<em>HERC2/OCA2</em> region) and 16q24.3 (<em>MC1R</em> region) centered at or near the <em>MC1R</em> and <em>HERC2/OCA2</em> genes, with a third highly significant region at chromosome 9p21.3 (near <em>CDKN2A/ARF</em> genes). In Table&nbsp;<a href="/pmc/articles/PMC3298855/table/DDR415TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB1" rid-ob="ob-DDR415TB1" co-legend-rid=""><span>1</span></a>, we present the results for previously reported associations of CM risk with SNPs. Of the previously reported 27 risk-associated SNPs, 21 were significant in our study at <em>P</em> &lt; 0.05. Individual SNPs that were previously associated with CM risk but did not reach significance in our study included rs1408799 in <em>TYRP1</em>, rs12896399 in <em>SLC24A4</em>, rs1800407 in <em>OCA2</em>, rs1805006 in <em>MC1R</em>, rs4911414 in <em>LOC729547</em> and rs1015362 in <em>EIF2S2</em>, but associations for most previously reported SNPs were supported in our study, and except for the region around <em>SLC24A4</em> on chromosome 14q32.12 there was at least one SNP in a region showing <em>P</em> &lt; 1 × 10<sup>−4</sup>.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F2" co-legend-rid="lgnd_DDR415F2"><a href="/pmc/articles/PMC3298855/figure/DDR415F2/" target="figure" rid-figpopup="DDR415F2" rid-ob="ob-DDR415F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352874552880" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F2/" target="figure" rid-figpopup="DDR415F2" rid-ob="ob-DDR415F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41502.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41502.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41502.jpg"></a></div><div id="largeobj_idm140352874552880" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F2"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F2/" target="figure" rid-figpopup="DDR415F2" rid-ob="ob-DDR415F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p12">Results from genome-wide association analysis of 818 977 (818 237 + 740 pseudo-autochromosomal) genotypes from 1804 CM cases and 1026 controls, after adjusting for two PCs.</p></div></div></div><p id="__p13">To understand whether associations in regions that were previously reported to be associated with CM risk were likely to be due to effects from a single or multiple variants, we performed analyses in which the association of SNPs in a region was conditioned on the most significant SNP in that region. Conditioning on the most significant SNP in <em>MC1R</em> reduced the level of significance for all other SNPs, but some SNPs retained significance levels of 10<sup>−3</sup> or higher (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). Similarly, conditioning on the most significant SNP in the <em>p16</em> region removed some, but not all, of the evidence for association in that region. These results suggest that multiple variants influence CM risk in these regions, or the SNP on which we conditioned may not be the causal variant, but was only in strong linkage disequilibrium (LD) with the causal variant, so that there remained some residual association. For 9p21.3 (<em>CDKN2A</em> region), associations were detected across a broad region that includes recombination hotspots even after conditioning on the most significant SNP, suggesting that multiple variants are involved.</p><p id="__p14">Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F3" rid-ob="ob-DDR415F3" co-legend-rid="lgnd_DDR415F3"><span>3</span></a> shows results from an association analysis of CM with SNPs in the <em>HERC2/OCA2</em> region in samples from MD Anderson Cancer Center. The <em>HERC2</em> SNP rs1129038 was the most significantly associated variant in the region [<em>P</em> = 2.58 × 10<sup>−8</sup>; odds ratio (OR) = 0.69; 95% confidence interval (CI) = 0.61–0.79]. The next most significant genotyped SNP in this region was rs12913832 (<em>P</em> = 4.31 × 10<sup>−8</sup>; OR = 0.69; 95% CI = 0.61–0.79). The variants rs1129038 and rs12913832 are in very high LD (<em>r</em><sup>2</sup> = 98.5%), but rs1129038 was not genotyped on other platforms used in subsequent studies; hence, further analyses were restricted to rs12913832.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F3" co-legend-rid="lgnd_DDR415F3"><a href="/pmc/articles/PMC3298855/figure/DDR415F3/" target="figure" rid-figpopup="DDR415F3" rid-ob="ob-DDR415F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352951750288" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F3/" target="figure" rid-figpopup="DDR415F3" rid-ob="ob-DDR415F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41503.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41503.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41503.jpg"></a></div><div id="largeobj_idm140352951750288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F3"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F3/" target="figure" rid-figpopup="DDR415F3" rid-ob="ob-DDR415F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p15">Association tests for the <em>HERC2/OCA2</em> region on chromosome 15q13.1. Genotyped SNPs are indicated as diamonds and imputed SNPs as circles. The most significant SNP in the region is <em>HERC2</em> rs1129038. The strength of the pairwise correlation between the surrounding markers and the most significant SNP is depicted by the size of the symbols: the larger the size, the stronger the LD. Genes in the region are annotated with location, range and orientation.</p></div></div></div><p id="__p16">The SNP rs1800407 in the flanking region of 15q13.1 (<em>HERC2</em>/<em>OCA2</em> region) showed moderate evidence (<em>P</em> = 0.004) for association with CM risk in a candidate gene study but was not associated with CM risk in this study (<em>P</em> = 0.62, OR = 0.95; 95% CI = 0.78–1.16). The most significant SNPs near the <em>OCA2</em> gene yielded <em>P</em>-values (rs73377792, <em>P</em> = 6.3 × 10<sup>−4</sup>; rs4778138, <em>P</em> = 6.52 × 10<sup>−4</sup>) that were less extreme than those observed in the 15q13.1 (<em>HERC2/OCA2</em>) region in our study population. Further analyses conditioning on rs1129038 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>) removed all remaining significant SNPs in the region.</p><p id="__p17">Because the <em>HERC2</em> gene was previously shown to be associated with pigmentary phenotypes (<a href="#DDR415C13" rid="DDR415C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337277946">13</a>), we further evaluated the association of the rs12913832 SNP with pigmentary phenotypes by using samples from MD Anderson (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). Table&nbsp;<a href="/pmc/articles/PMC3298855/table/DDR415TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB2" rid-ob="ob-DDR415TB2" co-legend-rid=""><span>2</span></a> describes additional findings from the analysis of SNP rs12913832 in the MD Anderson participants. Results showed overwhelming evidence associating <em>HERC2</em> rs12913832 with skin, eye and hair color phenotypes. Further analysis to evaluate the effect that this SNP had on these phenotypes showed that rs12913832 alone explained 50% of eye color (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>). In addition, we evaluated the association of <em>HERC2</em> rs12913832 with pigmentary phenotypes in 10 183 participants in the Nurses' Health Study (NHS) and Health Professionals Follow-Up Study (HPFS). In these analyses, 567 cases and 7329 controls were genotyped using Illumina Bead arrays for <em>HERC2</em> SNP rs12913832. We found extremely significant associations (Table&nbsp;<a href="/pmc/articles/PMC3298855/table/DDR415TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB3" rid-ob="ob-DDR415TB3" co-legend-rid=""><span>3</span></a>). Stratifying by skin color, we observed strongest effects of the <em>HERC2</em> SNP in lighter-skinned individuals (<em>n</em> = 1009, <em>P</em> = 0.0003, OR = 0.66, 95% CI = 0.53–0.83), a weaker effect in medium-skinned individuals (<em>n</em> = 828, <em>P</em> = 0.02, OR = 0.77, 95% CI = 0.61–0.96) and no effect in dark-skinned individuals (<em>n</em> = 117, <em>P</em> = 0.75, OR = 1.10, 95% CI = 0.61–2.00). In the Harvard cohort studies, the strongest effect was observed in light-skinned individuals (<em>n</em> = 2173, <em>P</em> = 0.08 OR = 0.80, 95% CI = 0.63–1.03), and no effect was observed in medium-skinned (<em>n</em> = 1916, <em>P</em>= 0.64, OR = 1.05, 95% CI = 0.85–1.32) or dark-skinned individuals (<em>n</em> = 1916, <em>P</em>= 0.72, OR = 0.94, 95% CI = 0.67–1.32).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR415TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p18">Association of <em>HERC2</em> rs12913832 with pigmentary phenotypes in the NHS and the HPFS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352873008080" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">GWAS phenotype</th><th rowspan="1" colspan="1" align="left">WT/VT</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Hair color (blonde to black)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2497</td><td rowspan="1" colspan="1">0.3872</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">1.39 × 10<sup>−167</sup></td></tr><tr><td rowspan="1" colspan="1">Eye color (blue to brown)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2647</td><td rowspan="1" colspan="1">0.8571</td><td rowspan="1" colspan="1">0.0175</td><td rowspan="1" colspan="1">7.68 × 10<sup>−523</sup></td></tr><tr><td rowspan="1" colspan="1">Tanning ability (none to deep)</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.2534</td><td rowspan="1" colspan="1">0.1359</td><td rowspan="1" colspan="1">0.0139</td><td rowspan="1" colspan="1">1.44 × 10<sup>−22</sup></td></tr></tbody></table></div><div id="largeobj_idm140352873008080" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/table/DDR415TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140352881191232"><p id="__p19" class="p p-first-last">WT, wild-type; VT, variant type; SE, standard error.</p></div></div></div><p id="__p20">Because skin pigmentation and the <em>HERC2</em> SNP frequency (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>) showed highly significant variability across European populations (<a href="#DDR415C14" rid="DDR415C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538873">14</a>) and the CM risk is higher for northern European than for southern European populations, a potential concern can be raised that northern/southern European ancestry may be a confounder, causing a spurious association to be observed. We therefore derived the most likely European ancestry of the study participants (Fig.&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F4" rid-ob="ob-DDR415F4" co-legend-rid="lgnd_DDR415F4"><span>4</span></a>) by principal component analysis (PCA) with additional control samples from European groups. Results of conditioning on the most likely European ancestry showed no evidence for heterogeneity in the association of rs12913832 with CM risk by derived European populations, and stratified results were virtually identical to those obtained with allowing for European stratification (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). In contrast, significant variation in the association between CM risk and SNPs in the <em>p16</em> region did vary significantly among derived European populations, suggesting potential allelic heterogeneity in European populations for this SNP in <em>CDKN2A</em> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F4" co-legend-rid="lgnd_DDR415F4"><a href="/pmc/articles/PMC3298855/figure/DDR415F4/" target="figure" rid-figpopup="DDR415F4" rid-ob="ob-DDR415F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352879407456" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F4/" target="figure" rid-figpopup="DDR415F4" rid-ob="ob-DDR415F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41504.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41504.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41504.jpg"></a></div><div id="largeobj_idm140352879407456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F4"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F4/" target="figure" rid-figpopup="DDR415F4" rid-ob="ob-DDR415F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p21">PCA of CM cases and controls showing the relationship among populations of European and Middle-Eastern origin. (<strong>A</strong>) Analyses of 1804 CM cases and 1026 controls, with an additional 5001 Caucasians of known origin, are shown for PC1 and PC2 (A1, A2, A3), and for PC1 and PC3 (A4, A5, A6). The results are shown with either the known population groups color-coded (A1 and A4) or with controls color-coded (A2 and A5) or cases color-coded (A3 and A6). The population groups are indicated in the legend key. AJA 4GP are Ashkenazi Jewish Americans with four grandparents of Ashkenazi heritage. The group denoted ‘ALL’ are Americans of European ancestry. Results show similarity of PCA loadings for cases and controls in A2–A6. (<strong>B</strong>) The color code indicates the correspondence of controls or cases to seven different population groups with graphs for PC1 and PC2 (B1 and B3), and PC1 and PC3 (B2 and B4). The population groups were based on correspondence to PCA. The population groups corresponded to the range of the self-identified population group member defined by the first three PCs (see A). Where overlaps were present, the midpoints in the overlapped region were used to define the boundary between the population group assignment. The legend key shows the corresponding population groups with abbreviations for the following groups: Ashkenazi Jewish (ASHKEN), eastern European (EEUR), German (GER), Scandinavian (SCAN) and United Kingdom (UK). Analysis of CM case–control data alone yielded two PCs, whereas analysis including additional European controls yielded the additional third PC, shown in this figure.</p></div></div></div><p id="__p22">We also sought <em>in silico</em> replication of the 10 most significant SNPs from the MD Anderson GWAS, excluding those loci that have been previously identified (in particular, we excluded the <em>MC1R</em> and <em>CDKN2A</em> regions, since these have been extensively resequenced in other studies), provided there was a second SNP with an <em>r</em><sup>2</sup> value of &gt;0.8 and a <em>P</em>-value of &lt;10<sup>−4</sup>. Results were provided to us by the GenoMEL consortium (based on European samples), an Australian collaboration, and from the Harvard cohorts (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). To further evaluate effects for two most significant regions on chromosomes 1 and 15, we obtained data from the three collaborating groups for ∼1 Mb regions flanking the most significant SNP in our study. As shown in Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F5" rid-ob="ob-DDR415F5" co-legend-rid="lgnd_DDR415F5"><span>5</span></a>, the most significant SNPs in the chromosome 15q region were rs1129038 in an intron of the <em>HERC2</em> gene (<em>P</em> = 2.00 × 10<sup>−6,</sup> OR = 0.77, 95% CI = 0.69–0.85), but this SNP was genotyped only in the US and a subset of Australian samples and showed heterogeneity, with the Australian samples giving an OR of 0.97 (95% CI = 0.80–1.17). The next most significant SNP is rs4778138 in <em>OCA2</em> (OR = 0.86, 95% CI = 0.80–0.92, <em>P</em> = 2.2 × 10<sup>−5</sup>). This SNP showed strong associations in the MD Anderson sample (<em>P</em> = 6.5 × 10<sup>−4</sup>, OR = 0.75, 95% CI = 0.64–0.89) and UK sample (<em>P</em> = 0.0064, OR = 0.88, 95% CI = 0.80–0.97) but weak association in the Australian sample (<em>P</em> = 0.14; OR = 0.89; 95% CI = 0.78–1.04) and was not genotyped in the UK sample. The next most significant SNP in the US study, rs12913832, was not associated with CM risk in the Australian samples (<em>P</em> = 0.63; OR = 0.98, 95% CI = 0.89–1.07) or UK samples (<em>P</em> = 0.44; OR = 0.96; 95% CI = 0.86–1.07), but showed a trend towards association in the Harvard cohort studies (<em>P</em> = 0.12; OR = 0.88, 95% CI = 0.76–1.03).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F5" co-legend-rid="lgnd_DDR415F5"><a href="/pmc/articles/PMC3298855/figure/DDR415F5/" target="figure" rid-figpopup="DDR415F5" rid-ob="ob-DDR415F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352879962912" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F5/" target="figure" rid-figpopup="DDR415F5" rid-ob="ob-DDR415F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41505.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41505.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41505.jpg"></a></div><div id="largeobj_idm140352879962912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F5"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F5/" target="figure" rid-figpopup="DDR415F5" rid-ob="ob-DDR415F5">Figure&nbsp;5.</a></div><!--caption a7--><div class="caption"><p id="__p23">Association tests from four studies and from fixed-effects meta-analysis for a region of chromosome 15 surrounding the <em>HERC2/OCA2</em> region. The region telomeric to the <em>HERC2</em> gene contains a region of many complex rearrangements including CHRFAM7A—a rearrangement of nicotinic acetylcholine receptor 7 and FAM7A. The gap in genotyping in the region between 25 168 374 and 25 271 522&nbsp;base pairs overlays a region with some reported copy number variants in the <em>GABRG3</em> gene.</p></div></div></div><p id="__p24" class="p p-last">As shown in Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F6" rid-ob="ob-DDR415F6" co-legend-rid="lgnd_DDR415F6"><span>6</span></a>, a region of chromosome 1q21.3 near the <em>ARNT</em> and <em>LASS2</em> genes around 149.1 Mb was highly associated with CM risk in the samples from MD Anderson Cancer Center. The most significant SNP in the 1 Mb region including data from the other studies (Fig.&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F7" rid-ob="ob-DDR415F7" co-legend-rid="lgnd_DDR415F7"><span>7</span></a>) was rs7412746 (<em>P</em> = 6.17 × 10<sup>−10</sup>, OR = 0.88; 95% CI = 0.84–0.91), which significantly replicated in all of the studies but Harvard: MD Anderson Cancer Center (<em>P</em> = 0.003; OR = 0.85; 95% CI = 0.76–0.94), Australia (<em>P</em> = 2.52 × 10<sup>−7</sup>; OR = 0.82; 95% CI = 0.76–0.89), GenoMEL (<em>P</em> = 0.014; OR = 0.92; 95% CI = 0.86–0.98), but not in the smaller Harvard study (<em>P</em> = 0.12, OR = 1.11; 95% CI = 0.97–1.27). The most significant SNP in the MD Anderson Cancer Center study, rs11204756 (<em>P</em> = 3.61 × 10<sup>−5</sup>, OR = 1.26; 95% CI = 1.13–1.40) in the <em>ANXA9</em> gene, had partially imputed data (<em>r</em><sup>2</sup> = 0.99) by the Australian study (<em>P</em> = 8 × 10<sup>−4</sup>; OR = 1.13; 95% CI = 1.05–1.22), yielding an overall association (<em>P</em> = 3.59 × 10<sup>−7</sup>, OR = 1.17, 95% CI = 1.10–1.24) and is, therefore, worthy of future investigations.

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F6" co-legend-rid="lgnd_DDR415F6"><a href="/pmc/articles/PMC3298855/figure/DDR415F6/" target="figure" rid-figpopup="DDR415F6" rid-ob="ob-DDR415F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352879947024" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F6/" target="figure" rid-figpopup="DDR415F6" rid-ob="ob-DDR415F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41506.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41506.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41506.jpg"></a></div><div id="largeobj_idm140352879947024" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F6"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F6/" target="figure" rid-figpopup="DDR415F6" rid-ob="ob-DDR415F6">Figure&nbsp;6.</a></div><!--caption a7--><div class="caption"><p id="__p25">Association tests for a region of chromosome 1 surrounding the <em>LASS2/ANXA9</em> region. Genotyped SNPs are indicated as diamonds and imputed SNPs as circles. The most significant SNP in the region is rs1722784. The overall structure of the LD with SNPs in this region is reflected by estimated recombination rates. The strength of the pairwise correlation between the surrounding markers and the most significant SNP is depicted by the size of the symbols: the larger the size, the stronger the LD. Genes in the region are annotated with location, range and orientation.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR415F7" co-legend-rid="lgnd_DDR415F7"><a href="/pmc/articles/PMC3298855/figure/DDR415F7/" target="figure" rid-figpopup="DDR415F7" rid-ob="ob-DDR415F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140352877338800" class="figure"><a href="/pmc/articles/PMC3298855/figure/DDR415F7/" target="figure" rid-figpopup="DDR415F7" rid-ob="ob-DDR415F7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3298855_ddr41507.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr41507.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3298855/bin/ddr41507.jpg"></a></div><div id="largeobj_idm140352877338800" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3298855/figure/DDR415F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR415F7"><div><a class="figpopup" href="/pmc/articles/PMC3298855/figure/DDR415F7/" target="figure" rid-figpopup="DDR415F7" rid-ob="ob-DDR415F7">Figure&nbsp;7.</a></div><!--caption a7--><div class="caption"><p id="__p26">Association tests from four studies using fixed-effects meta-analysis for a region of chromosome 1 spanning the <em>ARNT/LASS2/ANXA9</em> region.</p></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p27" class="p p-first">This study provided confirmatory evidence of associations between genetic loci in several previously identified chromosome regions and CM risk, including the <em>SLC45A2</em> region on chromosome 5p13.3, the 9p21.3 region encompassing <em>CDKN2A</em>, the region of <em>TYR</em> on chromosome 11q21, the region on chromosome 16q24.3 encompassing <em>MC1R</em>, a broad region on chromosome 20q11.2 and a region of 22q13 encompassing <em>PLA2G6.</em> Our study focused primarily on the discovery of novel variants influencing CM risk and we identified strong associations of SNPs in the <em>HERC2/OCA2</em> region on chromosome 15q13.1 as well as a region of chromosome 1q21.3 that had not previously been well characterized. The <em>HERC2</em> SNPs had previously been noted to associate strongly with pigmentation (<a href="#DDR415C5" rid="DDR415C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538850">5</a>), but our finding is the first report that these SNPs influence CM risk in US populations. In the previous analysis of pigmentary phenotypes and SNPs in the <em>HERC2/OCA2</em> region, the strongest association was found with SNPs rs12913832 and rs1129038 (<a href="#DDR415C13" rid="DDR415C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337277945">13</a>). These SNPs lie in a region that is in LD with both exons of <em>HERC2</em> as well as the regulatory region of <em>OCA2</em>, so that further studies are needed to identify whether variation in <em>HERC2</em> or <em>OCA2</em> accounts for CM risk and pigmentary changes in this region. We also identified the SNP rs4778138 in <em>OCA2</em>, which showed a suggestive association with CM risk among the studies in which it was genotyped. These results implicate <em>HERC2/OCA2</em> variation as a pigment-related gene that affects CM risk. Risk of CM could also be influenced by the effects of <em>HERC2</em>, which has recently been shown to play a role in ubiquitination after DNA damage (<a href="#DDR415C15" rid="DDR415C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538861">15</a>) and to post-translationally regulate levels of XPA (<a href="#DDR415C16" rid="DDR415C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538872">16</a>), a core protein involved in nucleotide-excision repair. <em>HERC2</em> variants could reduce CM risk by increasing levels of XPA, a hypothesis that needs further mechanistic studies. Variation in the association of the <em>HERC2</em> region SNPs among US, Australian and European regions may reflect varying patterns of sun exposure and thus further gene–environment studies are needed.</p><p id="__p28">Pigmentation varies among European populations, as does the frequency of variants in rs12913832, but cases and controls from the two US studies were collected from the same institution and cohorts, respectively, limiting possible confounding effects. The GenoMEL study encompasses several countries within Europe. Differences in allelic background as well as variation in sun exposure may underlie the differences between US and Australian or GenoMEL study findings for the effect of <em>HERC2/OCA2</em>, chromosome 15q13.1 region SNPs on the risk of CM.</p><p id="__p29" class="p p-last">For the region on chromosome 1q21.3 that was highly associated with CM risk, the evidence of an association was the strongest for a region that includes <em>ANXA9</em>, <em>LASS2</em>, <em>SETDB1</em> and <em>ARNT</em>, which are closely located. <em>ANXA9</em> is an annexin that has previously been associated with TH2-related responses that are induced during pemphigus (<a href="#DDR415C17" rid="DDR415C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538858">17</a>). <em>LASS2</em> is a ceramide synthase that has previously been shown to influence apoptosis after ionizing irradiation (<a href="#DDR415C18" rid="DDR415C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_449569363">18</a>) and to play a role in tumor suppression in hepatocellular carcinoma cell lines (<a href="#DDR415C19" rid="DDR415C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538870">19</a>). <em>SETDB1</em> plays a role in trimethylation required for proviral silencing, which would seem a less likely candidate for influencing CM risk. <em>ARNT</em> is the aryl hydrocarbon nuclear translocator and plays a key role in regulating response to exogenous compounds that are catabolized by P450 enzymes. In summary, this study identified signals in several chromosomal regions and suggests that candidate genes including <em>OCA2</em>, <em>HERC2</em>, <em>ARNT</em>, <em>ANXA9</em> and <em>LASS2</em> warrant additional investigations.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">MD Anderson Cancer Center Study</h3><p id="__p30" class="p p-first-last">The study participants for the discovery analysis were from a hospital-based case–control study of CM, for which cases were recruited from non-Hispanic white patients and controls at MD Anderson between March 1998 and August 2008. Samples and data were available from 931 CM patients and 1026 cancer-free controls (friends or acquaintances of patients reporting to other clinics), who were frequency-matched on age and sex, completed a comprehensive skin lifestyle questionnaire and passed QC filters for genotyping. This questionnaire was administered by an interviewer to 70% of patients and controls and was self-administered for the remaining 30%. An additional case series comprising 873 individuals presenting for the treatment for CM at MD Anderson was also included, bringing the total number of CM patients to 1804. The study protocols were approved by the Institutional Review Board at MD Anderson, and informed consent was obtained from all participants.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping and data QC</h3><p id="__p31" class="p p-first">Tissue samples were collected as whole blood, using various DNA extraction methods (including Gentra, Qiagen and phenol/chloroform). DNA samples for the first-stage GWAS were genotyped with use of the Illumina HumanOmni1-Quad_v1-0_B array and were called using the BeadStudio algorithm, at the John Hopkins University Center for Inherited Disease Research (CIDR). We were able to satisfactorily genotype 1 012 904 of the 1 016 423 SNPs attempted (99.6%) with a mean sample call rate of 99.86% (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a> summarizes a series of SNP filters applied to the original 1 140 419 SNPs and CNV probes. Before data release, 3519 SNPs failed the CIDR QC process with a missing call rate of &gt;5%. SNPs with an MAF of ≤0.01, call rate of &lt;95% or Hardy–Weinberg equilibrium in controls with a <em>P-</em>value of ≤10<sup>−5</sup> were excluded.</p><p id="__p32">After the above criteria were applied, 818 237 genotyped autosomal or X chromosome SNPs and 740 pseudo-autosomal SNPs were available for the final association analysis. Imputation of ungenotyped SNPs in candidate regions (at least two genotyped SNPs in one region having <em>P</em> &lt; 10<sup>−4</sup>) was performed for case and control samples with the use of the MACH program (<a href="#DDR415C20" rid="DDR415C20" class=" bibr popnode">20</a>) using 1000 Genomes CEU (March 2010 release) as a reference panel. An additional 12 009 SNPs were imputed from those regions. In addition, genome-wide imputation using MACH and Hapmap2 Release 22 using the CEU population has been completed and data are available upon request for 2 373 692 SNPs with <em>r</em><sup>2</sup> &gt; 0.8, with an average quality score of 99.35%.For the most significant SNP in the <em>HERC2</em> region, rs12913832, we performed confirmatory genotyping of 528 cases and controls using TaqMan. One subject was misclassified between the two platforms, one subject was genotyped as G/G by TaqMan and A/G by Illumina and one subject could not be genotyped by TaqMan but was found to have the G/G genotype by Illumina.</p><p id="__p33" class="p p-last">Of the samples that were genotyped, 41 failed genotyping with a &gt;10% missing rate across all SNPs; 11 samples had identity problems that could not be resolved. For this study, the identity-by-descent (IBD) coefficients were estimated using 116 002 autosomal SNPs in PLINK (<a href="#DDR415C12" rid="DDR415C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538857">12</a>) and 5 unexpected duplicates and 15 related samples were removed. In total, 126 duplicated (67 expected duplicates), related (IBD) or outliers identified by PCA were excluded from the study. After these exclusions, 1952 cases and 1026 controls remained. <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a> summarizes the whole-sample filters. From the total 2978 case and control subjects with data after QC, 138 <em>in situ</em> cases were removed from the study because they had indeterminate phenotypes; in addition, 10 patients with atypical melanocytic proliferation were excluded because they did not have invasive cancers. Data from 1804 cases and 1026 controls were analyzed for the association study of CM susceptibility.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">PCA and population structure of the MD Anderson sample</h3><p id="__p34" class="p p-first-last">We used an LD-pruned SNP set provided by the GENEVA (<a href="#DDR415C21" rid="DDR415C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538875">21</a>) coordinating center for PCA adjustment to evaluate population structure. To select a set of SNPs for identifying population stratification, GENEVA thinned the initial SNPs to reduce LD to a set of 75 210 SNPs. SNPs were pruned using pairwise genotypic correlation in PLINK. The pruning procedure consisted of two stages. In the first stage, short-range LD was removed. We used a window size of 50 SNPs with a 5 SNP offset and <em>r</em><sup>2</sup> cut-off point of 0.2. For the second stage, we aimed to remove long-range LD. We used a window size of 150 SNPs with an offset of 5 Mb and <em>r</em><sup>2</sup> cut-off point of 0.2. We adjusted for the first two components from PCA in our GWAS. No other PCs varied significantly between cases and controls. Those two eigenvectors were treated as covariables to adjust for population structure among study subjects. The GENEVA coordinating center provided an initial quality assessment of data and helped to organize the data for submission to dbGAP.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Association analysis of the MD Anderson sample</h3><p id="__p35" class="p p-first">Association analysis with CM of genotyped SNPs or most likely genotypes from the imputation study was performed using the PLINK –logistic and –covar options (<a href="#DDR415C12" rid="DDR415C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538856">12</a>). A logistic regression model was built to measure the additive effect of each SNP on susceptibility to CM. A likelihood ratio test was performed under the null hypothesis of <em>χ</em><sup>2</sup> distribution with one degree of freedom. The first two PCs were included to adjust for population structure. Q–Q plots portraying the associations of markers with CM risk are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S1</a>, including all of the directly genotyped markers, and in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S2</a> after excluding previously identified genes strongly associated with CM risk (<em>MC1R</em> and <em>p16/arf</em>). Significant regions with multiple SNPs that each had a <em>P</em>-value of &lt;10<sup>−4</sup> were selected for further validation in other independent studies.</p><p id="__p36" class="p p-last">To evaluate associations with pigmentary phenotypes, we regressed ordinal coding for hair color (1 = blonde, 2 = red, 3 = brown and 4 = black), eye color (1 = blue/gray, 2 = hazel/green and 3 = brown/black) and frequency of burns on a 1–4 scale (1 = always or usually burn, 2 = moderately burn, 3 = minimally burn and 4 = rarely or never burn). We used linear regression to test the association between minor allele counts and pigmentary phenotypes, and the model was adjusted for age and sex.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Genomic imputation</h3><p id="__p37" class="p p-first-last">MACH was used to impute ungenotyped SNPs in the candidate regions according to 1000 Genomes Project data. Based on the reference data with a denser set of markers, those untyped markers can be filled in by means of maximum-likelihood estimation. Imputation was run in one step since it was for small candidate regions. A total of 1682 SNPs with <em>P</em>-values of &lt;10<sup>−4</sup> were found among 16 regions on 12 chromosomes, excluding chromosome 9 near the centromere where <em>p16/ARF/CDKN2A</em> is located and chromosome 16 near <em>MC1R</em>, each of which contained at least two significant SNPs. Within a region of 50 kb from each side of these top SNPs, untyped genotypes were inferred with the use of MACH. Finally 12 009 SNPs were obtained with average <em>r</em><sup>2</sup>&gt; 0.875 for those candidate regions. The average posterior probability for the most likely genotype was 97.7%. As newer versions of the 1000 Genomes Project become available, better inference from it should be possible.</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Further adjustment for population structure in the MD Anderson sample</h3><p id="__p38" class="p p-first-last">Because skin pigmentation and CM risk show a North–South gradient across European populations, and allele frequencies in genes associated with skin pigmentation also show variation in allele frequencies, confounding between allele frequency and northern/southern European ancestry is a concern. We therefore performed additional studies to characterize the European ethnic background of study participants from MD Anderson and to allow for this inferred ancestry. Results, shown in Figure&nbsp;<a href="/pmc/articles/PMC3298855/figure/DDR415F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR415F4" rid-ob="ob-DDR415F4" co-legend-rid="lgnd_DDR415F4"><span>4</span></a>, demonstrate the inferred origins of study participants from MD Anderson according to the first two PCs. <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a> displays results from analyses that were conditioned on the inferred most likely ancestral origin of each participant. The approach to selecting populations for characterizing European ancestry has been previously presented (<a href="#DDR415C22" rid="DDR415C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538855">22</a>). Cases and controls that did not cluster within European ancestry groups were removed from analysis. In this analysis, 1795 cases and 931 controls were retained, whereas an additional 1587 controls genotyped on 610K platforms that had genotypes close to the existing cases according to the first six PCs were used as additional controls in the analysis.</p></div><div id="s4g" class="sec"><h3 id="s4gtitle">Materials and methods from Harvard cohort studies</h3><p id="__p39" class="p p-first">We have conducted several GWASs on different disease outcomes [NHS: breast cancer by Illumina 550, and pancreatic cancer by Illumina 550; HPFS: advanced prostate cancer by Illumina 610; both NHS (part of the CGEMS) (<a href="#DDR415C23" rid="DDR415C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538865">23</a>–<a href="#DDR415C25" rid="DDR415C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303002302">25</a>) and HPFS: coronary heart disease by Affymetrix 6.0, type 2 diabetes by Affymetrix 6.0, kidney stone by Illumina 610 and glaucoma by Illumina 660]. We included only controls in each study, except for the kidney stone study, in which we used both cases and controls. We excluded those with personal history of squamous cell carcinoma and basal cell carcinoma. Participants with CM diagnosis were the CM cases in this study, and those without CM diagnosis were the controls. In addition, we genotyped the rest of the CM cases identified in both cohorts who were not in these previous GWASs by Illumina 610. Finally, we included 494 CM cases and 5628 controls.</p><p id="__p40">Based on the genotyped SNPs and haplotype information in the NCBI Build 35 of phase II Hapmap CEU data, we imputed genotypes for &gt;2.5 million SNPs using the program MACH (<a href="#DDR415C20" rid="DDR415C20" class=" bibr popnode">20</a>). Only SNPs with imputation quality <em>r</em><sup>2</sup>&gt; 0.95 and with MAF &gt; 2.5% in each study were included in the final analysis. A total of 1 579 307 (NHS: 1 518 067; HPFS: 1 533 499) SNPs were included in the final meta-analysis of the NHS set and HPFS set.</p><p id="__p41">Information on natural hair color at age 20 years and on childhood and adolescent tanning tendency was collected in both the NHS and HPFS prospective questionnaires, and information on natural eye color was collected in the HPFS only. The summary of the basic information of the five-component cohort studies used in the GWAS of pigmentation is shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>. The imputed dosage data on chromosome 15 were used for the association study. We regressed ordinal coding for hair color (1 = blonde, 2 = light brown, 3 = dark brown and 4 = black; the participants with red hair color were excluded from the hair color GWAS), eye color (1 = blue/light, 2 = hazel/green/medium, 3 = brown/dark) and tanning ability (1 = practically none, 2 = light tan, 3 = average tan, 4 = deep tan in NHS; and 1 = pain: burn/peel, 2 = burn then tan, 3 = tan without burn in HPFS). We used linear regression to test the association between minor allele counts and pigmentary phenotypes, and the model was adjusted for the first four PCs separately for each SNP and trait; within-cohort association results were combined in an inverse variance weighted meta-analysis, and the software METAL (<a href="http://www.sph.umich.edu/csg/abecasis/Metal/index.html">http://www.sph.umich.edu/csg/abecasis/Metal/index.html</a>) was used for the meta-analysis across the different cohorts that were analyzed. Controlling for case–control status within each collection (e.g. type 2 diabetes in the HPFS) made no material difference to results (hence no adjustment was made). A summary of information from the five component studies used for SNP studies of pigmentation is shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>.</p><p id="__p42">For the study on CM risk, the eligible cases consisted of participants in the NHS and the HPFS with pathologically confirmed CM, diagnosed any time after the baseline up to the 2006 follow-up cycle (for both cohorts), who had no previously diagnosed cancer. The controls were randomly selected from participants in the same cohorts who were free of cancer up to their participation and including the questionnaire cycle in which the case was diagnosed. In addition, after excluding the breast cancer cases in the NHS-BC set and the individuals diagnosed with skin cancer (CM, basal cell carcinoma, or squamous cell carcinoma), the participants in the five GWAS sets mentioned above were also included as controls. All subjects were US non-Hispanic Caucasians. We included 585 CM cases (mean age at diagnosis, 61.5 years) and 7363 controls. Laboratory personnel were blinded to case–control status, and blinded QC samples were inserted to validate genotyping procedures; concordance for the blinded samples was 100%. Primers, probes and conditions for genotyping assays are available upon request.</p><p id="__p43" class="p p-last">We regressed a binary coding for CM case and control (0 or 1) on each SNP (dosage file used) that passed QC filters. Among the 10 largest PCs, the first 5 were significantly associated with nevus count at a two-sided alpha of 0.05. Therefore, we adjusted for top five PCs of genetic variation in the regression model along with age. These PCs were calculated for all individuals on the basis of approximately 10 000 unlinked markers using the EIGENSTRAT software.</p></div><div id="s4h" class="sec"><h3 id="s4htitle"><em>In silico</em> replication studies</h3><p id="__p44" class="p p-first-last">Confirmatory <em>in silico</em> replication was sought from GenoMEL, Harvard cohorts and the Australian consortium for the top 10 SNPs of significance for which another nearby SNP with LD <em>r</em><sup>2</sup>&gt; 0.8 also reached a significance level of &lt;10<sup>−4</sup>.</p></div><div id="s4i" class="sec"><h3 id="s4ititle">Australian sample</h3><p id="__p45" class="p p-first">DNA was extracted from peripheral blood or saliva samples. Australian twin and endometriosis sample controls were genotyped at deCODE Genetics (Reykjavik, Iceland) on the Illumina HumanHap610W Quad and Illumina HumanHap660 Quad Beadarrays, respectively. AMFS controls were genotyped by Illumina (San Diego) on Illumina Omni1-Quad arrays. Cases were genotyped by Illumina (San Diego) on Illumina Omni1-Quad (568 AMFS cases, 699 Q-MEGA cases) and HumanHap610W Quad arrays (998 Q-MEGA cases). All genotypes were called with the Illumina BeadStudio software. SNPs with a mean BeadStudio GenCall score &lt;0.7 were excluded from the control data sets. All samples had successful genotypes for &gt;95% of SNPs. SNPs with call rates either &lt;0.95 (MAF &gt; 0.05) or &lt;0.99 (MAF &gt; 0.01), Hardy–Weinberg equilibrium in controls <em>P</em> &lt; 10<sup>−6</sup> and/or MAF &lt; 0.01 were excluded. Cryptic relatedness between individuals was assessed through the production of a full identity-by-state matrix. Ancestry outliers were identified by PC analysis, using data from 11 populations of the HapMap 3 Project and 5 northern European populations genotyped by the GenomeEUtwin consortium, using the EIGENSOFT package. Individuals lying ≥2 standard deviations from the mean PC1 and PC2 scores were excluded from subsequent analyses. Following these exclusions, there were 2168 case samples (1242 typed on the Omni1-Quad and 926 typed on the Hap610 arrays, respectively) and 4387 controls (431 typed on the Omni1-Quad and 3956 typed on the Hap610 arrays) retained for subsequent analyses (Table&nbsp;<a href="/pmc/articles/PMC3298855/table/DDR415TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR415TB1" rid-ob="ob-DDR415TB1" co-legend-rid=""><span>1</span></a>). Individuals typed on the Omni1-Quad array had genotypes for up to 816 169 SNPs, whereas individuals typed on the Hap610/670 arrays had genotypes for up to 544 483 SNPs. There were 299 394 SNPs passing QC and overlapping between these arrays (and hence directly genotyped on all Australian samples).</p><p id="__p46">Imputation for the Australian samples was performed using MACH (<a href="#DDR415C20" rid="DDR415C20" class=" bibr popnode">20</a>) with 1000 Genomes Project [June 2010 release] data obtained from people of northern and western European ancestry collected by the Centre d'Etude du Polymorphisme Humain. Imputation was based on a set of autosomal SNPs common to all CM case–control samples (<em>n</em> = 292 043). Imputation was run in two stages. First, data from a set of representative Australian sample individuals were compared with the phased haplotype data from the 1000 Genomes data to generate recombination and error maps. For the second stage, data were imputed for all individuals using the phased 1000 Genomes data as the reference panel and the recombination as well as error files generated in the first stage. In total, 5 480 804 1000 Genomes SNPs could be imputed with imputation <em>r</em><sup>2</sup> &gt; 0.5.</p><p id="__p47" class="p p-last">Association analysis of genotyped SNPs was performed using the PLINK –assoc option (<a href="#DDR415C12" rid="DDR415C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538876">12</a>). Analysis of dosage scores from the imputation analysis was done using mach2dat.</p></div><div id="s4j" class="sec sec-last"><h3 id="s4jtitle">GenoMEL sample</h3><p id="__p48" class="p p-first-last">The GenoMEL data incorporated in the <em>in silico</em> replication came from a GWAS of 2804 cases and 1835 controls for CM collected at 11 different centers across Europe and Israel. These were genotyped in two phases, phase 1 on the Illumina HumanHap300 BeadChip version 2 duo array (317K SNPs) and phase 2 on the Illumina Human610 quad array (610K SNPs). These are supplemented by 3878 controls from the WTCCC study [WTCCC (<a href="#DDR415C26" rid="DDR415C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538863">26</a>)] genotyped on the Illumina HumanHap1.2 million array and 1905 French controls genotyped by Centre National de Genotypage on the Illumina Humancnv370k array. Individuals were excluded for low call rate (&lt;97% on the array on which the sample was genotyped), non-European ethnicity (determined by PCA), sex discrepancy with recorded phenotype information, first-degree or closer relatedness with another sample or recommendation for exclusion by WTCCC (<a href="#DDR415C26" rid="DDR415C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_253538849">26</a>). QC was applied to SNPs separately for each genotyping platform. SNPs were excluded for a low call rate (&lt;97%), Hardy–Weinberg equilibrium <em>P</em>-value of &lt;10<sup>−20</sup> or recommendation for exclusion by WTCCC (usually on the basis of poor clustering). A trend test was applied to each SNP in turn stratified by broad geographic region (eight regions pre-specified).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">WEB RESOURCES</h2><p id="__p49" class="p p-first">EIGENSOFT: <a href="http://genepath.med.harvard.edu/~reich/Software.htm">http://genepath.med.harvard.edu/~reich/Software.htm</a>.</p><p id="__p50">MACH: <a href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</a>, release MACH1.016.</p><p id="__p51">1000 Genomes: <a href="http://www.1000genomes.org">http://www.1000genomes.org</a>.</p><p id="__p52">The genotyping data and phenotypes that have been presented in this analysis are available online through dbGaP (study accession: phs000187.v1.p1).</p><p id="__p53" class="p p-last">Chromosomal region graphs were developed using the software developed by the Diabetes Genome Initiative (<a href="http://www.broadinstitute.org/science/projects/diabetes-genetics-initiative/plotting-genome-wide-association-results">http://www.broadinstitute.org/science/projects/diabetes-genetics-initiative/plotting-genome-wide-association-results</a>).</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">SUPPLEMENTARY MATERIAL</h2><p id="__p54" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddr415/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title">NOTE ADDED IN PROOF</h2><p id="__p55" class="p p-first">Dr Houvras brought to our attention an elegant work that was performed showing that the <em>SETDB1</em> gene (which lies within our region of association on chromosome 1q21.3) accelerates melanoma in cooperation with BRAF, increases invasiveness of the lesions and allows cells to bypass a senescence arrest. Therefore, this locus is also an excellent candidate for future studies to identify its role in melanoma susceptibility (<a href="#DDR415C27" rid="DDR415C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_265770964">27</a>).</p><p id="__p56" class="p p-last"><em>Conflict of Interest statement</em>. None declared.</p></div><div id="s8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s8title">FUNDING</h2><p id="__p57" class="p p-first">Research at MD Anderson Cancer Center was partially supported by NIH grants <span>R01CA100264</span>;, <span>2P50CA093459</span>;, <span>P30CA016672</span>;, <span>R01CA133996</span> and by the Marit Peterson Fund for Melanoma Research.</p><p id="__p58">The Center for Inherited Disease Research is supported by contract <span>HHSN268200782096C</span>. Geneva is supported by NIH grant <span>HG004446</span>.</p><p id="__p59">The Australian Twin Registry is supported by an Australian National Health and Medical Research Council (NHMRC) Enabling Grant (2004–2009).</p><p id="__p60">The QIMR Study was supported by grants from the Melanoma Research Alliance, the National Health and Medical Research Council (NHMRC) of Australia (<span>241944, 339462, 389927,389875, 389891, 389892, 389938, 443036, 442915, 442981, 496610, 496739, 552485, 552498</span>), the Cooperative Research Centre for Discovery of Genes for Common Human Diseases (CRC), Cerylid Biosciences (Melbourne), and donations from Neville and Shirley Hawkins.</p><p id="__p61">S.M. is supported by NHMRC Career Development Awards (<span>496674</span>;, <span>613705</span>). G.W.M. is supported by the NHMRC Fellowships Scheme.</p><p id="__p62">The AMFS study is funded by the National Health and Medical Research Council of Australia (NHMRC) (project grants <span>566946, 107359, 211172</span>; and program grant number <span>402761</span> to G.J.M. and R.F.K.); the Cancer Council New South Wales (<span>project grant 77/00, 06/10</span>), the Cancer Council Victoria and the Cancer Council Queensland (<span>project grant 371</span>); and the US National Institutes of Health (via RO1 grant <span>CA-83115-01A2</span> to the international Melanoma Genetics Consortium - GenoMEL). A.E.C. is the recipient of an NHMRC public health postdoctoral fellowship (<span>520018</span>) and a National Cancer Institute NSW Early Career Development Fellowship (<span>10/ECF/2-06</span>). B.K.A.'s research is supported by a University of Sydney Medical Foundation Program Grant, and J.L.H. is an Australia Fellow of the NHMRC.</p><p id="__p63">The GenoMEL study was funded by the European Commission under the 6th Framework Programme, contract no: <span>LSHC-CT-2006-018702</span>; by Cancer Research UK Programme Awards, <span>C588/A4994</span>;, and <span>C588/A10589</span>, and Cancer Research UK Project Grant <span>C8216/A6129</span>; and by US National Institutes of Health
<span>R01 CA83115</span>. This research was also supported by the Intramural Research Program of the NIH, NCI and DCEG.</p><p id="__p64">Emilia-Romagna: National Cancer Institute grants
<span>RO1 CA5558</span> to Maria Teresa Landi.</p><p id="__p65">Genoa: IRCSS 2007 Italian Ministry of Health
<span>DGRST.4/4235-P1.9.A.B</span>, Fondazione CARIGE, <span>PRIN 2008</span> to G.B.-S.</p><p id="__p66">Leeds: Cancer Research UK Programme grants Genetic Epidemiology of Cancer
<span>C588/A10589, C588/A4994</span> and Cancer Research UK Project grant <span>C8216/A6129</span>.</p><p id="__p67">Lund: Swedish Cancer Society, Swedish Research Council, Region Skåne funds, Kamprad Foundation.</p><p id="__p68">Norway: Grants from the Comprehensive Cancer Center, Oslo University Hospital and the University of Bergen.</p><p id="__p69" class="p p-last">Paris: Grants from Institut National du Cancer (<span>INCa-PL016</span>) and Ligue Nationale Contre Le Cancer (<span>PRE05/FD</span>; and <span>PRE 09/FD</span>) to Florence Demenais, Programme Hospitalier de Recherche Clinique (<span>PHRC 2007/AOM-07-195</span>) to Marie-Françoise Avril and Florence Demenais, Institut National du Cancer (Melanoma Network RS Number 13), Association pour la Recherche sur le Cancer (<span>ARC No. A09/5/5003</span>), and Société Française de Dermatologie (<span>SFD2009</span>) to Brigitte Bressac-de Paillerets. Brigitte Bressac-de Paillerets has been awarded an INSERM Research Fellowship for hospital-based scientists.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140352876172288"><a href="/pmc/articles/PMC3298855/bin/supp_20_24_5012__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140352874293568" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140352874293568title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p70">We acknowledge with appreciation all of the women who participated in the QIMR, OXEGENE and NHS studies. We thank Endometriosis Associations for supporting study recruitment as well as Sullivan Nicolaides and Queensland Medical Laboratory for <em>pro bono</em> collection and delivery of blood samples and other pathology services associated with blood collection. In addition, we thank Margaret J. Wright, Megan J. Campbell, Anthony Caracella, Margaret Lung, Zhensheng Liu, Yawei Qiao, Min Zhao, Dakai Zhu, B. Haddon, D. Smyth, H. Beeby, O. Zheng and B. Chapman for their input into project management, databases, sample processing and genotyping. We are grateful to the many research assistants and interviewers for assistance with the studies contributing to the QIMR collection.</p></div></div><div id="idm140352951741120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140352951741120title">APPENDIX</h2><!--/article/back/app-group/app/--><div id="s9" class="sec sec-first"><h3 id="s9title">MEMBERS OF THE GENOMEL COLLABORATION</h3><p id="__p71" class="p p-first">Australian Melanoma Family Study (AMFS): Graham J. Mann, John L. Hopper, Joanne F. Aitken, Bruce K. Armstrong, Graham G. Giles, Richard F. Kefford, Anne E. Cust, Mark A. Jenkins, Helen Schmid.</p><p id="__p72">Barcelona: The participants of GenoMEL in Barcelona: Paula Aguilera, Celia Badenas, Cristina Carrera, Francisco Cuellar, Daniel Gabriel, Estefania Martinez, Melinda Gonzalez, Pablo Iglesias, Josep Malvehy, Rosa Marti-Laborda, Montse Mila, Zighe Ogbah, Joan-Anton Puig Butille, Susana Puig, and other members of the Melanoma Unit: Llúcia Alós, Ana Arance, Pedro Arguís, Antonio Campo, Teresa Castel, Carlos Conill, Jose Palou, Ramon Rull, Marcelo Sánchez, Sergi Vidal-Sicart, Antonio Vilalta and Ramon Vilella.</p><p id="__p73">Brisbane: The Queensland study of Melanoma: Environmental and Genetic Associations (Q-MEGA) Principal Investigators: Nicholas G. Martin, Grant W. Montgomery, David L. Duffy, David C. Whiteman, Stuart MacGregor and Nicholas K. Hayward.</p><p id="__p74">The Australian Cancer Study (ACS) Principal Investigators: David Whiteman, Penny Webb, Adele Green, Peter Parsons, David Purdie and Nicholas Hayward.</p><p id="__p75">Emilia-Romagna: Maria Teresa Landi, Donato Calista, Giorgio Landi, Paola Minghetti, Fabio Arcangeli and Pier Alberto Bertazzi.</p><p id="__p76">Genoa: Department of Internal Medicine (DIMI), University of Genoa: Giovanna Bianchi-Scarra, Paola Ghiorzo, Lorenza Pastorino, William Bruno, Linda Battistuzzi, Sara Gargiulo, Sabina Nasti, Sara Gliori, Paola Origone, Virginia Andreotti. Medical Oncology Unit, National Institute for Cancer Research: Paola Queirolo.</p><p id="__p77">Glasgow: Rona Mackie, Julie Lang.</p><p id="__p78">Leeds: Julia A. Newton Bishop, Paul Affleck, Jennifer H. Barrett, D. Timothy Bishop, Jane Harrison, Mark M. Iles, Juliette Randerson-Moor, Mark Harland, John C. Taylor, Linda Whittaker, Kairen Kukalizch, Susan Leake, Birute Karpavicius, Sue Haynes, Tricia Mack, May Chan, Yvonne Taylor, John Davies and Paul King.</p><p id="__p79">Leiden: Department of Dermatology, Leiden University Medical Centre: Nelleke A. Gruis, Frans A. van Nieuwpoort, Coby Out, Clasine van der Drift, Wilma Bergman, Nicole Kukutsch and Jan Nico Bouwes Bavinck. Department of Clinical Genetics, Centre of Human and Clinical Genetics, Leiden University Medical Centre: Bert Bakker, Nienke van der Stoep, Jeanet ter Huurne. Department of Dermatology, HAGA Hospital, The Hague: Han van der Rhee. Department of Dermatology, Reinier de Graaf Groep, Delft: Marcel Bekkenk. Department of Dermatology, Sint Franciscus Gasthuis, Rotterdam: Dyon Snels, Marinus van Praag. Department of Dermatology, Ghent University Hospital, Ghent, Belgium: Lieve Brochez and colleagues. Department of Dermatology, St Radboud University Medical Centre, Nijmegen: Rianne Gerritsen and colleagues. Department of Dermatology, Rijnland Hospital, Leiderdorp: Marianne Crijns and colleagues. Dutch Patient Organization, Stichting Melanoom, Purmerend. The Netherlands Foundation for the Detection of Hereditary Tumors, Leiden: Hans Vasen.</p><p id="__p80">Lund: Lund Melanoma Study Group: Håkan Olsson, Christian Ingvar, Göran Jönsson, Åke Borg, Anna Måsbäck, Lotta Lundgren, Katja Baeckenhorn, Kari Nielsen, Anita Schmidt Casslén.</p><p id="__p81">Norway: Oslo University Hospital: Per Helsing, Per Arne Andresen, Helge Rootwelt.</p><p id="__p82">University of Bergen: Lars A. Akslen, Anders Molven.</p><p id="__p83">Paris: Marie-Françoise Avril, Brigitte Bressac-de Paillerets, Valérie Chaudru, Nicolas Chateigner, Eve Corda, Patricia Jeannin, Fabienne Lesueur, Mahaut de Lichy, Eve Maubec, Hamida Mohamdi, Florence Demenais, and the French Family Study Group including the following oncogeneticists and dermatologists: Pascale Andry-Benzaquen, Bertrand Bachollet, Frédéric Bérard, Pascaline Berthet, Françoise Boitier, Valérie Bonadona, Jean-Louis Bonafé, Jean-Marie Bonnetblanc, Frédéric Cambazard, Olivier Caron, Frédéric Caux, Jacqueline Chevrant-Breton, Agnès Chompret (deceased), Stéphane Dalle, Liliane Demange, Olivier Dereure, Martin-Xavier Doré, Marie-Sylvie Doutre, Catherine Dugast, Laurence Faivre, Florent Grange, Philippe Humbert, Pascal Joly, Delphine Kerob, Christine Lasset, Marie Thérèse Leccia, Gilbert Lenoir, Dominique Leroux, Julien Levang, Dan Lipsker, Sandrine Mansard, Ludovic Martin, Tanguy Martin-Denavit, Christine Mateus, Jean-Loïc Michel, Patrice Morel, Laurence Olivier-Faivre, Jean-Luc Perrot, Caroline Robert, Sandra Ronger-Savle, Bruno Sassolas, Pierre Souteyrand, Dominique Stoppa-Lyonnet, Luc Thomas, Pierre Vabres and Eva Wierzbicka.</p><p id="__p84">Philadelphia: David Elder, Peter Kanetsky, Jillian Knorr, Michael Ming, Nandita Mitra, Althea Ruffin and Patricia Van Belle.</p><p id="__p85">Poland: Tadeusz Dębniak, Jan Lubiński, Aneta Mirecka and Sławomir Ertmański.</p><p id="__p86">Slovenia: Srdjan Novakovic, Marko Hocevar, Barbara Peric and Petra Cerkovnik.</p><p id="__p87">Stockholm: Veronica Höiom and Johan Hansson.</p><p id="__p88">Sydney: Graham J. Mann, Richard F. Kefford, Helen Schmid and Elizabeth A. Holland.</p><p id="__p89">Tel Aviv: Esther Azizi, Gilli Galore-Haskel, Eitan Friedman, Orna Baron-Epel, Alon Scope, Felix Pavlotsky, Emanuel Yakobson, Irit Cohen-Manheim, Yael Laitman, Roni Milgrom, Iris Shimoni and Evgeniya Kozlovaa.</p><p id="__p90" class="p p-last">See also: <a href="http://www.GenoMEL.org">www.GenoMEL.org</a>.</p></div></div><div id="idm140352876655488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140352876655488title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDR415C1">1. <span class="element-citation">Siegel R., Ward E., Brawley O., Jemal A. Cancer statistics, 2011: the impact of eliminating socioeconomic and racial disparities on premature cancer deaths. <span><span class="ref-journal">CA Cancer J. Clin. </span>2011;<span class="ref-vol">61</span>:212–236.</span> [<a href="/pubmed/21685461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J.+Clin.&amp;title=Cancer+statistics,+2011:+the+impact+of+eliminating+socioeconomic+and+racial+disparities+on+premature+cancer+deaths&amp;author=R.+Siegel&amp;author=E.+Ward&amp;author=O.+Brawley&amp;author=A.+Jemal&amp;volume=61&amp;publication_year=2011&amp;pages=212-236&amp;pmid=21685461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C2">2. <span class="element-citation">Cust A.E., Schmid H., Maskiell J.A., Jetann J., Ferguson M., Holland E.A., Agha-Hamilton C., Jenkins M.A., Kelly J., Kefford R.F., et al.  Population-based, case-control-family design to investigate genetic and environmental influences on melanoma risk: Australian Melanoma Family Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>2009;<span class="ref-vol">170</span>:1541–1554.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2800270/">PMC free article</a>]</span> [<a href="/pubmed/19887461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Population-based,+case-control-family+design+to+investigate+genetic+and+environmental+influences+on+melanoma+risk:+Australian+Melanoma+Family+Study&amp;author=A.E.+Cust&amp;author=H.+Schmid&amp;author=J.A.+Maskiell&amp;author=J.+Jetann&amp;author=M.+Ferguson&amp;volume=170&amp;publication_year=2009&amp;pages=1541-1554&amp;pmid=19887461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C3">3. <span class="element-citation">Meyle K.D., Guldberg P. Genetic risk factors for melanoma. <span><span class="ref-journal">Hum. Genet. </span>2009;<span class="ref-vol">126</span>:499–510.</span> [<a href="/pubmed/19585149" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genetic+risk+factors+for+melanoma&amp;author=K.D.+Meyle&amp;author=P.+Guldberg&amp;volume=126&amp;publication_year=2009&amp;pages=499-510&amp;pmid=19585149&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C4">4. <span class="element-citation">Bishop D.T., Demenais F., Iles M.M., Harland M., Taylor J.C., Corda E., Randerson-Moor J., Aitken J.F., Avril M.F., Azizi E., et al.  Genome-wide association study identifies three loci associated with melanoma risk. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:920–925.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2741419/">PMC free article</a>]</span> [<a href="/pubmed/19578364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+three+loci+associated+with+melanoma+risk&amp;author=D.T.+Bishop&amp;author=F.+Demenais&amp;author=M.M.+Iles&amp;author=M.+Harland&amp;author=J.C.+Taylor&amp;volume=41&amp;publication_year=2009&amp;pages=920-925&amp;pmid=19578364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C5">5. <span class="element-citation">Sturm R.A., Duffy D.L., Zhao Z.Z., Leite F.P., Stark M.S., Hayward N.K., Martin N.G., Montgomery G.W. A single SNP in an evolutionary conserved region within intron 86 of the HERC2 gene determines human blue-brown eye color. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:424–431.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427173/">PMC free article</a>]</span> [<a href="/pubmed/18252222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+single+SNP+in+an+evolutionary+conserved+region+within+intron+86+of+the+HERC2+gene+determines+human+blue-brown+eye+color&amp;author=R.A.+Sturm&amp;author=D.L.+Duffy&amp;author=Z.Z.+Zhao&amp;author=F.P.+Leite&amp;author=M.S.+Stark&amp;volume=82&amp;publication_year=2008&amp;pages=424-431&amp;pmid=18252222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C6">6. <span class="element-citation">Fargnoli M.C., Argenziano G., Zalaudek I., Peris K. High- and low-penetrance cutaneous melanoma susceptibility genes. <span><span class="ref-journal">Expert Rev. Anticancer Ther. </span>2006;<span class="ref-vol">6</span>:657–670.</span> [<a href="/pubmed/16759158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Rev.+Anticancer+Ther.&amp;title=High-+and+low-penetrance+cutaneous+melanoma+susceptibility+genes&amp;author=M.C.+Fargnoli&amp;author=G.+Argenziano&amp;author=I.+Zalaudek&amp;author=K.+Peris&amp;volume=6&amp;publication_year=2006&amp;pages=657-670&amp;pmid=16759158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C7">7. <span class="element-citation">Kanetsky P.A., Rebbeck T.R., Hummer A.J., Panossian S., Armstrong B.K., Kricker A., Marrett L.D., Millikan R.C., Gruber S.B., Culver H.A., et al.  Population-based study of natural variation in the melanocortin-1 receptor gene and melanoma. <span><span class="ref-journal">Cancer Res. </span>2006;<span class="ref-vol">66</span>:9330–9337.</span> [<a href="/pubmed/16982779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Population-based+study+of+natural+variation+in+the+melanocortin-1+receptor+gene+and+melanoma&amp;author=P.A.+Kanetsky&amp;author=T.R.+Rebbeck&amp;author=A.J.+Hummer&amp;author=S.+Panossian&amp;author=B.K.+Armstrong&amp;volume=66&amp;publication_year=2006&amp;pages=9330-9337&amp;pmid=16982779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C8">8. <span class="element-citation">Pho L., Grossman D., Leachman S.A. Melanoma genetics: a review of genetic factors and clinical phenotypes in familial melanoma. <span><span class="ref-journal">Curr. Opin. Oncol. </span>2006;<span class="ref-vol">18</span>:173–179.</span> [<a href="/pubmed/16462187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Oncol.&amp;title=Melanoma+genetics:+a+review+of+genetic+factors+and+clinical+phenotypes+in+familial+melanoma&amp;author=L.+Pho&amp;author=D.+Grossman&amp;author=S.A.+Leachman&amp;volume=18&amp;publication_year=2006&amp;pages=173-179&amp;pmid=16462187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C9">9. <span class="element-citation">Gudbjartsson D.F., Sulem P., Stacey S.N., Goldstein A.M., Rafnar T., Sigurgeirsson B., Benediktsdottir K.R., Thorisdottir K., Ragnarsson R., Sveinsdottir S.G., et al.  ASIP and TYR pigmentation variants associate with cutaneous melanoma and basal cell carcinoma (vol 40, pg 886, 2008) <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1029.</span> [<a href="/pubmed/18488027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=ASIP+and+TYR+pigmentation+variants+associate+with+cutaneous+melanoma+and+basal+cell+carcinoma+(vol+40,+pg+886,+2008)&amp;author=D.F.+Gudbjartsson&amp;author=P.+Sulem&amp;author=S.N.+Stacey&amp;author=A.M.+Goldstein&amp;author=T.+Rafnar&amp;volume=40&amp;publication_year=2008&amp;pages=1029&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C10">10. <span class="element-citation">Sulem P., Gudbjartsson D.F., Stacey S.N., Helgason A., Rafnar T., Magnusson K.P., Manolescu A., Karason A., Palsson A., Thorleifsson G., et al.  Genetic determinants of hair, eye and skin pigmentation in Europeans. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1443–1452.</span> [<a href="/pubmed/17952075" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+determinants+of+hair,+eye+and+skin+pigmentation+in+Europeans&amp;author=P.+Sulem&amp;author=D.F.+Gudbjartsson&amp;author=S.N.+Stacey&amp;author=A.+Helgason&amp;author=T.+Rafnar&amp;volume=39&amp;publication_year=2007&amp;pages=1443-1452&amp;pmid=17952075&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C11">11. <span class="element-citation">Brown K.M., Macgregor S., Montgomery G.W., Craig D.W., Zhao Z.Z., Iyadurai K., Henders A.K., Homer N., Campbell M.J., Stark M., et al.  Common sequence variants on 20q11.22 confer melanoma susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:838–840.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755512/">PMC free article</a>]</span> [<a href="/pubmed/18488026" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+sequence+variants+on+20q11.22+confer+melanoma+susceptibility&amp;author=K.M.+Brown&amp;author=S.+Macgregor&amp;author=G.W.+Montgomery&amp;author=D.W.+Craig&amp;author=Z.Z.+Zhao&amp;volume=40&amp;publication_year=2008&amp;pages=838-840&amp;pmid=18488026&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C12">12. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A.R., Bender D., Maller J., Sklar P., de Bakker P.I.W., Daly M.J., Sham P.C. PLINK: A tool set for whole-genome association and population-based linkage analyses 1. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses+1&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.R.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C13">13. <span class="element-citation">Duffy D.L., Zhao Z.Z., Sturm R.A., Hayward N.K., Martin N.G., Montgomery G.W. Multiple pigmentation gene polymorphisms account for a substantial proportion of risk of cutaneous malignant melanoma. <span><span class="ref-journal">J. Invest. Dermatol. </span>2010;<span class="ref-vol">130</span>:520–528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3672059/">PMC free article</a>]</span> [<a href="/pubmed/19710684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=Multiple+pigmentation+gene+polymorphisms+account+for+a+substantial+proportion+of+risk+of+cutaneous+malignant+melanoma&amp;author=D.L.+Duffy&amp;author=Z.Z.+Zhao&amp;author=R.A.+Sturm&amp;author=N.K.+Hayward&amp;author=N.G.+Martin&amp;volume=130&amp;publication_year=2010&amp;pages=520-528&amp;pmid=19710684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C14">14. <span class="element-citation">Cann H.M., de Toma C., Cazes L., Legrand M.F., Morel V., Piouffre L., Bodmer J., Bodmer W.F., Bonne-Tamir B., Cambon-Thomsen A., et al.  A human genome diversity cell line panel. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:261–262.</span> [<a href="/pubmed/11954565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+human+genome+diversity+cell+line+panel&amp;author=H.M.+Cann&amp;author=C.+de+Toma&amp;author=L.+Cazes&amp;author=M.F.+Legrand&amp;author=V.+Morel&amp;volume=296&amp;publication_year=2002&amp;pages=261-262&amp;pmid=11954565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C15">15. <span class="element-citation">Bekker-Jensen S., Rendtlew D.J., Fugger K., Gromova I., Nerstedt A., Lukas C., Bartek J., Lukas J., Mailand N. HERC2 coordinates ubiquitin-dependent assembly of DNA repair factors on damaged chromosomes. <span><span class="ref-journal">Nat. Cell Biol. </span>2010;<span class="ref-vol">12</span>:80–86.</span> [<a href="/pubmed/20023648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Cell+Biol.&amp;title=HERC2+coordinates+ubiquitin-dependent+assembly+of+DNA+repair+factors+on+damaged+chromosomes&amp;author=S.+Bekker-Jensen&amp;author=D.J.+Rendtlew&amp;author=K.+Fugger&amp;author=I.+Gromova&amp;author=A.+Nerstedt&amp;volume=12&amp;publication_year=2010&amp;pages=80-86&amp;pmid=20023648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C16">16. <span class="element-citation">Kang T.H., Lindsey-Boltz L.A., Reardon J.T., Sancar A. Circadian control of XPA and excision repair of cisplatin-DNA damage by cryptochrome and HERC2 ubiquitin ligase. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:4890–4895.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2841896/">PMC free article</a>]</span> [<a href="/pubmed/20304803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Circadian+control+of+XPA+and+excision+repair+of+cisplatin-DNA+damage+by+cryptochrome+and+HERC2+ubiquitin+ligase&amp;author=T.H.+Kang&amp;author=L.A.+Lindsey-Boltz&amp;author=J.T.+Reardon&amp;author=A.+Sancar&amp;volume=107&amp;publication_year=2010&amp;pages=4890-4895&amp;pmid=20304803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C17">17. <span class="element-citation">Caproni M., Giomi B., Cardinali C., Salvatore E., Pestelli E., D'Agata A., Bianchi B., Toto P., Feliciani C., Fabbri P. Further support for a role for Th2-like cytokines in blister formation of pemphigus. <span><span class="ref-journal">Clin. Immunol. </span>2001;<span class="ref-vol">98</span>:264–271.</span> [<a href="/pubmed/11161984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Immunol.&amp;title=Further+support+for+a+role+for+Th2-like+cytokines+in+blister+formation+of+pemphigus&amp;author=M.+Caproni&amp;author=B.+Giomi&amp;author=C.+Cardinali&amp;author=E.+Salvatore&amp;author=E.+Pestelli&amp;volume=98&amp;publication_year=2001&amp;pages=264-271&amp;pmid=11161984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C18">18. <span class="element-citation">Mesicek J., Lee H., Feldman T., Jiang X., Skobeleva A., Berdyshev E.V., Haimovitz-Friedman A., Fuks Z., Kolesnick R. Ceramide synthases 2, 5, and 6 confer distinct roles in radiation-induced apoptosis in HeLa cells. <span><span class="ref-journal">Cell. Signal. </span>2010;<span class="ref-vol">22</span>:1300–1307.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4348005/">PMC free article</a>]</span> [<a href="/pubmed/20406683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Signal.&amp;title=Ceramide+synthases+2,+5,+and+6+confer+distinct+roles+in+radiation-induced+apoptosis+in+HeLa+cells&amp;author=J.+Mesicek&amp;author=H.+Lee&amp;author=T.+Feldman&amp;author=X.+Jiang&amp;author=A.+Skobeleva&amp;volume=22&amp;publication_year=2010&amp;pages=1300-1307&amp;pmid=20406683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C19">19. <span class="element-citation">Tang N., Jin J., Deng Y., Ke R.H., Shen Q.J., Fan S.H., Qin W.X. LASS2 interacts with V-ATPase and inhibits cell growth of hepatocellular carcinoma. <span><span class="ref-journal">Sheng Li Xue Bao. </span>2010;<span class="ref-vol">62</span>:196–202.</span> [<a href="/pubmed/20571735" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sheng+Li+Xue+Bao&amp;title=LASS2+interacts+with+V-ATPase+and+inhibits+cell+growth+of+hepatocellular+carcinoma&amp;author=N.+Tang&amp;author=J.+Jin&amp;author=Y.+Deng&amp;author=R.H.+Ke&amp;author=Q.J.+Shen&amp;volume=62&amp;publication_year=2010&amp;pages=196-202&amp;pmid=20571735&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C20">20. <span class="element-citation">Li Y., Abecasis G. Mach 1.0: rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mach+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C21">21. <span class="element-citation">Cornelis M.C., Agrawal A., Cole J.W., Hansel N.N., Barnes K.C., Beaty T.H., Bennett S.N., Bierut L.J., Boerwinkle E., Doheny K.F., et al.  The Gene, environment association studies consortium (GENEVA): maximizing the knowledge obtained from GWAS by collaboration across studies of multiple conditions. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:364–372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860056/">PMC free article</a>]</span> [<a href="/pubmed/20091798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=The+Gene,+environment+association+studies+consortium+(GENEVA):+maximizing+the+knowledge+obtained+from+GWAS+by+collaboration+across+studies+of+multiple+conditions&amp;author=M.C.+Cornelis&amp;author=A.+Agrawal&amp;author=J.W.+Cole&amp;author=N.N.+Hansel&amp;author=K.C.+Barnes&amp;volume=34&amp;publication_year=2010&amp;pages=364-372&amp;pmid=20091798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C22">22. <span class="element-citation">Tian C., Kosoy R., Nassir R., Lee A., Villoslada P., Klareskog L., Hammarstrom L., Garchon H.J., Pulver A.E., Ransom M., et al.  European population genetic substructure: further definition of ancestry informative markers for distinguishing among diverse European ethnic groups. <span><span class="ref-journal">Mol. Med. </span>2009;<span class="ref-vol">15</span>:371–383.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2730349/">PMC free article</a>]</span> [<a href="/pubmed/19707526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Med.&amp;title=European+population+genetic+substructure:+further+definition+of+ancestry+informative+markers+for+distinguishing+among+diverse+European+ethnic+groups&amp;author=C.+Tian&amp;author=R.+Kosoy&amp;author=R.+Nassir&amp;author=A.+Lee&amp;author=P.+Villoslada&amp;volume=15&amp;publication_year=2009&amp;pages=371-383&amp;pmid=19707526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C23">23. <span class="element-citation">Han J., Kraft P., Nan H., Guo Q., Chen C., Qureshi A., Hankinson S.E., Hu F.B., Duffy D.L., Zhao Z.Z., et al.  A genome-wide association study identifies novel alleles associated with hair color and skin pigmentation. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2367449/">PMC free article</a>]</span> [<a href="/pubmed/18483556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+identifies+novel+alleles+associated+with+hair+color+and+skin+pigmentation&amp;author=J.+Han&amp;author=P.+Kraft&amp;author=H.+Nan&amp;author=Q.+Guo&amp;author=C.+Chen&amp;volume=4&amp;publication_year=2008&amp;pages=e1000074&amp;pmid=18483556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C24">24. <span class="element-citation">Nan H., Kraft P., Qureshi A.A., Guo Q., Chen C., Hankinson S.E., Hu F.B., Thomas G., Hoover R.N., Chanock S., et al.  Genome-wide association study of tanning phenotype in a population of European ancestry. <span><span class="ref-journal">J. Invest. Dermatol. </span>2009;<span class="ref-vol">129</span>:2250–2257.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758927/">PMC free article</a>]</span> [<a href="/pubmed/19340012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=Genome-wide+association+study+of+tanning+phenotype+in+a+population+of+European+ancestry&amp;author=H.+Nan&amp;author=P.+Kraft&amp;author=A.A.+Qureshi&amp;author=Q.+Guo&amp;author=C.+Chen&amp;volume=129&amp;publication_year=2009&amp;pages=2250-2257&amp;pmid=19340012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C25">25. <span class="element-citation">Hunter D.J., Kraft P., Jacobs K.B., Cox D.G., Yeager M., Hankinson S.E., Wacholder S., Wang Z., Welch R., Hutchinson A., et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=D.J.+Hunter&amp;author=P.+Kraft&amp;author=K.B.+Jacobs&amp;author=D.G.+Cox&amp;author=M.+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C26">26. <span class="element-citation">WTCCC. Genome-wide association study of 14 000 cases of seven common diseases and 3000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14+000+cases+of+seven+common+diseases+and+3000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR415C27">27. <span class="element-citation">Ceol C.J., Houvras Y., Jane-Valbuena J., Bilodeau S., Orlando D.A., Battisti V., Fritsch L., Lin W.M., Hollmann T.J., Ferré F., et al.  The histone methyltransferase SETDB1 is recurrently amplified in melanoma and accelerates its onset. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">471</span>:513–517.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3348545/">PMC free article</a>]</span> [<a href="/pubmed/21430779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+histone+methyltransferase+SETDB1+is+recurrently+amplified+in+melanoma+and+accelerates+its+onset&amp;author=C.J.+Ceol&amp;author=Y.+Houvras&amp;author=J.+Jane-Valbuena&amp;author=S.+Bilodeau&amp;author=D.A.+Orlando&amp;volume=471&amp;publication_year=2011&amp;pages=513-517&amp;pmid=21430779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140352867994704title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">WEB RESOURCES</a></li><li><a href="#s6title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s7title">NOTE ADDED IN PROOF</a></li><li><a href="#s8title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140352874293568title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140352951741120title">APPENDIX</a></li><li><a href="#idm140352876655488title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3298855/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3298855/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3298855/pdf/ddr415.pdf">PDF (523K)</a></li> | <li><a href="#" data-citationid="PMC3298855" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3298855%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3298855%2F&amp;text=Genome-wide%20association%20study%20identifies%20novel%20loci%20predisposing%20to%20cutaneous%20melanoma" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3298855%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21926416" data-aiid="3298855" data-aid="3298855" data-iid="202806" data-domainid="740" data-domain="hmg" data-accid="PMC3298855" data-md5="ef10d7422cf6d100b681bab8f7cf67bb">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-5" aria-hidden="true"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="3298855" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/20629734" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genetic analysis of three important genes in pigmentation and melanoma susceptibility: CDKN2A, MC1R and HERC2/OCA2.</a><span class="source">[Exp Dermatol. 2010]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Ibarrola-Villava M, Fernandez LP, Pita G, Bravo J, Floristan U, Sendagorta E, Feito M, Avilés JA, Martin-Gonzalez M, Lázaro P, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Exp Dermatol. 2010 Sep; 19(9):836-44. Epub 2010 Jul 14.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21983785" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association study identifies a new melanoma susceptibility locus at 1q21.3.</a><span class="source">[Nat Genet. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Macgregor S, Montgomery GW, Liu JZ, Zhao ZZ, Henders AK, Stark M, Schmid H, Holland EA, Duffy DL, Zhang M, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nat Genet. 2011 Oct 9; 43(11):1114-8. Epub 2011 Oct 9.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/23548203" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association studies identify several new loci associated with pigmentation traits and skin cancer risk in European Americans.</a><span class="source">[Hum Mol Genet. 2013]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Zhang M, Song F, Liang L, Nan H, Zhang J, Liu H, Wang LE, Wei Q, Lee JE, Amos CI, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Mol Genet. 2013 Jul 15; 22(14):2948-59. Epub 2013 Apr 1.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21693730" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Comprehensive field synopsis and systematic meta-analyses of genetic association studies in cutaneous melanoma.</a><span class="source">[J Natl Cancer Inst. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Chatzinasiou F, Lill CM, Kypreou K, Stefanaki I, Nicolaou V, Spyrou G, Evangelou E, Roehr JT, Kodela E, Katsambas A, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">J Natl Cancer Inst. 2011 Aug 17; 103(16):1227-35. Epub 2011 Jun 21.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/20546537" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association studies of pigmentation and skin cancer: a review and meta-analysis.</a><span class="source">[Pigment Cell Melanoma Res. 2010]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Gerstenblith MR, Shi J, Landi MT. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Pigment Cell Melanoma Res. 2010 Oct; 23(5):587-606. Epub 2010 Jul 16.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=21926416&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=21926416&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6824721/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genetic variants in ELOVL2 and HSD17B12 predict melanoma-specific survival</a><span class="source">[International journal of cance...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Dai W, Liu H, Xu X, Jie G, Luo S, Zhu D, Amos CI, Fang S, Lee JE, Li X, Nan H, Li C, Wei Q. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">International journal of cancer. 2019 Feb 20; 145(10): 2619-2628</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6814532/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Role of immune response, inflammation and tumor immune response–related cytokines/chemokines in melanoma progression</a><span class="source">[The Journal of investigative d...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Fang S, Xu T, Xiong M, Zhou X, Wang Y, Haydu LE, Ross MI, Gershenwald JE, Prieto VG, Cormier JN, Wargo J, Sui D, Wei Q, Amos CI, Lee JE. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">The Journal of investigative dermatology. 2019 Jun 7; 139(11): 2352-2358.e3</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6761051/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association studies and polygenic risk scores for skin cancer: clinically useful yet?</a><span class="source">[The British journal of dermato...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Roberts MR, Asgari MM, Toland AE. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">The British journal of dermatology. 2019 Mar 25;10.1111/bjd.17917</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6735918/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">An accurate and powerful method for copy number variation detection</a><span class="source">[Bioinformatics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Xiao F, Luo X, Hao N, Niu YS, Xiao X, Cai G, Amos CI, Zhang H. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Bioinformatics. 2019 Jan 14; 35(17): 2891-2898</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6692442/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">HERCing: Structural and Functional Relevance of the Large HERC Ubiquitin Ligases</a><span class="source">[Frontiers in Physiology. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">García-Cano J, Martinez-Martinez A, Sala-Gaston J, Pedrazza L, Rosa JL. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Frontiers in Physiology. 2019 Aug 7; 10: 1014</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=gene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_gene&amp;IdsFromResult=3298855" ref="reftype=Gene&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Gene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Gene<div class="brieflinkpopdesc">Gene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=geoprofiles&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_geoprofiles&amp;IdsFromResult=3298855" ref="reftype=GEO%20Profiles&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">GEO Profiles</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">GEO Profiles<div class="brieflinkpopdesc">Related GEO records</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=homologene&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_homologene&amp;IdsFromResult=3298855" ref="reftype=HomoloGene&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|HomoloGene&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">HomoloGene</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">HomoloGene<div class="brieflinkpopdesc">HomoloGene links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=3298855" ref="reftype=MedGen&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-22" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/21926416/" ref="reftype=PubMed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-23" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=snp&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_snp&amp;IdsFromResult=3298855" ref="reftype=SNP&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|SNP&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SNP</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-24" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">SNP<div class="brieflinkpopdesc">PMC to SNP links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=pcsubstance&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_pcsubstance&amp;IdsFromResult=3298855" ref="reftype=Substance&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Substance</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-25" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Substance<div class="brieflinkpopdesc">PubChem Substance links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=3298855" ref="reftype=Taxonomy&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-26" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-27" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-27" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd534f7dde0893a8b4c0080">Genome-wide association study identifies novel loci predisposing to cutaneous me...</a><div class="ralinkpop offscreen_noflow">Genome-wide association study identifies novel loci predisposing to cutaneous melanoma<div class="brieflinkpopdesc">Human Molecular Genetics. 2011 Dec 15; 20(24)5012</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd534de39a66eb03d288240">Convergent Genomic Studies Identify Association of GRIK2 and NPAS2 with Chronic ...</a><div class="ralinkpop offscreen_noflow">Convergent Genomic Studies Identify Association of GRIK2 and NPAS2 with Chronic Fatigue Syndrome<div class="brieflinkpopdesc">Neuropsychobiology. 2011 Sep; 64(4)183</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd534a3dd2142621d09aac1">Genome-wide association study identifies a susceptibility locus for thoracic aor...</a><div class="ralinkpop offscreen_noflow">Genome-wide association study identifies a susceptibility locus for thoracic aortic aneurysms and aortic dissections spanning FBN1 at 15q21.1<div class="brieflinkpopdesc">NIHPA Author Manuscripts. ; 43(10)996</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd5347cdde0893a8b381554">Genome-wide association study identifies breast cancer risk variant at 10q21.2: ...</a><div class="ralinkpop offscreen_noflow">Genome-wide association study identifies breast cancer risk variant at 10q21.2: results from the Asia Breast Cancer Consortium<div class="brieflinkpopdesc">Human Molecular Genetics. 2011 Dec 15; 20(24)4991</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd5345575f66d018f21f14e">Identification of IL6R and chromosome 11q13.5 as risk loci for asthma</a><div class="ralinkpop offscreen_noflow">Identification of IL6R and chromosome 11q13.5 as risk loci for asthma<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2011 Sep 10; 378(9795)1006</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p4" name="crb--__p4" rid="__p4" style="position: absolute; visibility: visible; width: 370px; top: 1612px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="DDR415C1">
<a href="/pubmed/21685461/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Cancer statistics, 2011: the impact of eliminating socioeconomic and racial disparities on premature cancer deaths.</a><span class="one_line_source">[CA Cancer J Clin.  2011]</span><div class="alt-note">
<div class="authors">Siegel R, Ward E, Brawley O, Jemal A</div>
<div class="citation">CA Cancer J Clin. 2011 Jul-Aug; 61(4):212-36.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C2">
<a href="/pubmed/19887461/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Population-based, case-control-family design to investigate genetic and environmental influences on melanoma risk: Australian Melanoma Family Study.</a><span class="one_line_source">[Am J Epidemiol.  2009]</span><div class="alt-note">
<div class="authors">Cust AE, Schmid H, Maskiell JA, Jetann J, Ferguson M, Holland EA, Agha-Hamilton C, Jenkins MA, Kelly J, Kefford RF, Giles GG, Armstrong BK, Aitken JF, Hopper JL, Mann GJ</div>
<div class="citation">Am J Epidemiol. 2009 Dec 15; 170(12):1541-54.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C3">
<a href="/pubmed/19585149/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Genetic risk factors for melanoma.</a><span class="one_line_source">[Hum Genet.  2009]</span><div class="alt-note">
<div class="authors">Meyle KD, Guldberg P</div>
<div class="citation">Hum Genet. 2009 Oct; 126(4):499-510.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C4">
<a href="/pubmed/19578364/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study identifies three loci associated with melanoma risk.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Bishop DT, Demenais F, Iles MM, Harland M, Taylor JC, Corda E, Randerson-Moor J, Aitken JF, Avril MF, Azizi E, Bakker B, Bianchi-Scarrà G, Bressac-de Paillerets B, Calista D, Cannon-Albright LA, Chin-A-Woeng T, Debniak T, Galore-Haskel G, Ghiorzo P, Gut I, Hansson J, Hocevar M, Höiom V, Hopper JL, Ingvar C, Kanetsky PA, Kefford RF, Landi MT, Lang J, Lubiński J, Mackie R, Malvehy J, Mann GJ, Martin NG, Montgomery GW, van Nieuwpoort FA, Novakovic S, Olsson H, Puig S, Weiss M, van Workum W, Zelenika D, Brown KM, Goldstein AM, Gillanders EM, Boland A, Galan P, Elder DE, Gruis NA, Hayward NK, Lathrop GM, Barrett JH, Bishop JA</div>
<div class="citation">Nat Genet. 2009 Aug; 41(8):920-5.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C5">
<a href="/pubmed/18252222/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A single SNP in an evolutionary conserved region within intron 86 of the HERC2 gene determines human blue-brown eye color.</a><span class="one_line_source">[Am J Hum Genet.  2008]</span><div class="alt-note">
<div class="authors">Sturm RA, Duffy DL, Zhao ZZ, Leite FP, Stark MS, Hayward NK, Martin NG, Montgomery GW</div>
<div class="citation">Am J Hum Genet. 2008 Feb; 82(2):424-31.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C6" style="display: none;">
<a href="/pubmed/16759158/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> High- and low-penetrance cutaneous melanoma susceptibility genes.</a><span class="one_line_source">[Expert Rev Anticancer Ther.  2006]</span><div class="alt-note">
<div class="authors">Fargnoli MC, Argenziano G, Zalaudek I, Peris K</div>
<div class="citation">Expert Rev Anticancer Ther. 2006 May; 6(5):657-70.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C8" style="display: none;">
<a href="/pubmed/16462187/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal"><span class="invert">Review</span> Melanoma genetics: a review of genetic factors and clinical phenotypes in familial melanoma.</a><span class="one_line_source">[Curr Opin Oncol.  2006]</span><div class="alt-note">
<div class="authors">Pho L, Grossman D, Leachman SA</div>
<div class="citation">Curr Opin Oncol. 2006 Mar; 18(2):173-9.</div>
</div>
</li>
</ul><a href="" class="seemore">See more ...</a></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p5" name="crb--__p5" rid="__p5" style="position: absolute; visibility: visible; width: 370px; top: 1865px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul>
<li class="two_line" reference_id="DDR415C9">
<a href="/pubmed/18488027/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">ASIP and TYR pigmentation variants associate with cutaneous melanoma and basal cell carcinoma.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Gudbjartsson DF, Sulem P, Stacey SN, Goldstein AM, Rafnar T, Sigurgeirsson B, Benediktsdottir KR, Thorisdottir K, Ragnarsson R, Sveinsdottir SG, Magnusson V, Lindblom A, Kostulas K, Botella-Estrada R, Soriano V, Juberías P, Grasa M, Saez B, Andres R, Scherer D, Rudnai P, Gurzau E, Koppova K, Kiemeney LA, Jakobsdottir M, Steinberg S, Helgason A, Gretarsdottir S, Tucker MA, Mayordomo JI, Nagore E, Kumar R, Hansson J, Olafsson JH, Gulcher J, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2008 Jul; 40(7):886-91.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C10">
<a href="/pubmed/17952075/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genetic determinants of hair, eye and skin pigmentation in Europeans.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Sulem P, Gudbjartsson DF, Stacey SN, Helgason A, Rafnar T, Magnusson KP, Manolescu A, Karason A, Palsson A, Thorleifsson G, Jakobsdottir M, Steinberg S, Pálsson S, Jonasson F, Sigurgeirsson B, Thorisdottir K, Ragnarsson R, Benediktsdottir KR, Aben KK, Kiemeney LA, Olafsson JH, Gulcher J, Kong A, Thorsteinsdottir U, Stefansson K</div>
<div class="citation">Nat Genet. 2007 Dec; 39(12):1443-52.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C11">
<a href="/pubmed/18488026/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Common sequence variants on 20q11.22 confer melanoma susceptibility.</a><span class="one_line_source">[Nat Genet.  2008]</span><div class="alt-note">
<div class="authors">Brown KM, Macgregor S, Montgomery GW, Craig DW, Zhao ZZ, Iyadurai K, Henders AK, Homer N, Campbell MJ, Stark M, Thomas S, Schmid H, Holland EA, Gillanders EM, Duffy DL, Maskiell JA, Jetann J, Ferguson M, Stephan DA, Cust AE, Whiteman D, Green A, Olsson H, Puig S, Ghiorzo P, Hansson J, Demenais F, Goldstein AM, Gruis NA, Elder DE, Bishop JN, Kefford RF, Giles GG, Armstrong BK, Aitken JF, Hopper JL, Martin NG, Trent JM, Mann GJ, Hayward NK</div>
<div class="citation">Nat Genet. 2008 Jul; 40(7):838-40.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C4">
<a href="/pubmed/19578364/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study identifies three loci associated with melanoma risk.</a><span class="one_line_source">[Nat Genet.  2009]</span><div class="alt-note">
<div class="authors">Bishop DT, Demenais F, Iles MM, Harland M, Taylor JC, Corda E, Randerson-Moor J, Aitken JF, Avril MF, Azizi E, Bakker B, Bianchi-Scarrà G, Bressac-de Paillerets B, Calista D, Cannon-Albright LA, Chin-A-Woeng T, Debniak T, Galore-Haskel G, Ghiorzo P, Gut I, Hansson J, Hocevar M, Höiom V, Hopper JL, Ingvar C, Kanetsky PA, Kefford RF, Landi MT, Lang J, Lubiński J, Mackie R, Malvehy J, Mann GJ, Martin NG, Montgomery GW, van Nieuwpoort FA, Novakovic S, Olsson H, Puig S, Weiss M, van Workum W, Zelenika D, Brown KM, Goldstein AM, Gillanders EM, Boland A, Galan P, Elder DE, Gruis NA, Hayward NK, Lathrop GM, Barrett JH, Bishop JA</div>
<div class="citation">Nat Genet. 2009 Aug; 41(8):920-5.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p11" name="crb--__p11" rid="__p11" style="position: absolute; visibility: visible; width: 370px; top: 4055px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul><li class="two_line" reference_id="DDR415C12">
<a href="/pubmed/17701901/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PLINK: a tool set for whole-genome association and population-based linkage analyses.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC</div>
<div class="citation">Am J Hum Genet. 2007 Sep; 81(3):559-75.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p17" name="crb--__p17" rid="__p17" style="position: absolute; visibility: visible; width: 370px; top: 6115px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="DDR415C13">
<a href="/pubmed/19710684/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple pigmentation gene polymorphisms account for a substantial proportion of risk of cutaneous malignant melanoma.</a><span class="one_line_source">[J Invest Dermatol.  2010]</span><div class="alt-note">
<div class="authors">Duffy DL, Zhao ZZ, Sturm RA, Hayward NK, Martin NG, Montgomery GW</div>
<div class="citation">J Invest Dermatol. 2010 Feb; 130(2):520-8.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p20" name="crb--__p20" rid="__p20" style="position: absolute; visibility: visible; width: 370px; top: 6871px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-32"><ul><li class="two_line" reference_id="DDR415C14">
<a href="/pubmed/11954565/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A human genome diversity cell line panel.</a><span class="one_line_source">[Science.  2002]</span><div class="alt-note">
<div class="authors">Cann HM, de Toma C, Cazes L, Legrand MF, Morel V, Piouffre L, Bodmer J, Bodmer WF, Bonne-Tamir B, Cambon-Thomsen A, Chen Z, Chu J, Carcassi C, Contu L, Du R, Excoffier L, Ferrara GB, Friedlaender JS, Groot H, Gurwitz D, Jenkins T, Herrera RJ, Huang X, Kidd J, Kidd KK, Langaney A, Lin AA, Mehdi SQ, Parham P, Piazza A, Pistillo MP, Qian Y, Shu Q, Xu J, Zhu S, Weber JL, Greely HT, Feldman MW, Thomas G, Dausset J, Cavalli-Sforza LL</div>
<div class="citation">Science. 2002 Apr 12; 296(5566):261-2.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p27" name="crb--__p27" rid="__p27" style="position: absolute; visibility: visible; width: 370px; top: 10542px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-33"><ul>
<li class="two_line" reference_id="DDR415C5">
<a href="/pubmed/18252222/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A single SNP in an evolutionary conserved region within intron 86 of the HERC2 gene determines human blue-brown eye color.</a><span class="one_line_source">[Am J Hum Genet.  2008]</span><div class="alt-note">
<div class="authors">Sturm RA, Duffy DL, Zhao ZZ, Leite FP, Stark MS, Hayward NK, Martin NG, Montgomery GW</div>
<div class="citation">Am J Hum Genet. 2008 Feb; 82(2):424-31.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C13">
<a href="/pubmed/19710684/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Multiple pigmentation gene polymorphisms account for a substantial proportion of risk of cutaneous malignant melanoma.</a><span class="one_line_source">[J Invest Dermatol.  2010]</span><div class="alt-note">
<div class="authors">Duffy DL, Zhao ZZ, Sturm RA, Hayward NK, Martin NG, Montgomery GW</div>
<div class="citation">J Invest Dermatol. 2010 Feb; 130(2):520-8.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C15">
<a href="/pubmed/20023648/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">HERC2 coordinates ubiquitin-dependent assembly of DNA repair factors on damaged chromosomes.</a><span class="one_line_source">[Nat Cell Biol.  2010]</span><div class="alt-note">
<div class="authors">Bekker-Jensen S, Rendtlew Danielsen J, Fugger K, Gromova I, Nerstedt A, Lukas C, Bartek J, Lukas J, Mailand N</div>
<div class="citation">Nat Cell Biol. 2010 Jan; 12(1):80-6; sup pp 1-12.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C16">
<a href="/pubmed/20304803/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Circadian control of XPA and excision repair of cisplatin-DNA damage by cryptochrome and HERC2 ubiquitin ligase.</a><span class="one_line_source">[Proc Natl Acad Sci U S A.  2010]</span><div class="alt-note">
<div class="authors">Kang TH, Lindsey-Boltz LA, Reardon JT, Sancar A</div>
<div class="citation">Proc Natl Acad Sci U S A. 2010 Mar 16; 107(11):4890-5.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p29" name="crb--__p29" rid="__p29" style="position: absolute; visibility: visible; width: 370px; top: 11158px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-34"><ul>
<li class="two_line" reference_id="DDR415C17">
<a href="/pubmed/11161984/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Further support for a role for Th2-like cytokines in blister formation of pemphigus.</a><span class="one_line_source">[Clin Immunol.  2001]</span><div class="alt-note">
<div class="authors">Caproni M, Giomi B, Cardinali C, Salvatore E, Pestelli E, D'Agata A, Bianchi B, Toto P, Feliciani C, Fabbri P</div>
<div class="citation">Clin Immunol. 2001 Feb; 98(2):264-71.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C18">
<a href="/pubmed/20406683/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Ceramide synthases 2, 5, and 6 confer distinct roles in radiation-induced apoptosis in HeLa cells.</a><span class="one_line_source">[Cell Signal.  2010]</span><div class="alt-note">
<div class="authors">Mesicek J, Lee H, Feldman T, Jiang X, Skobeleva A, Berdyshev EV, Haimovitz-Friedman A, Fuks Z, Kolesnick R</div>
<div class="citation">Cell Signal. 2010 Sep; 22(9):1300-7.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C19">
<a href="/pubmed/20571735/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">[LASS2 interacts with V-ATPase and inhibits cell growth of hepatocellular carcinoma].</a><span class="one_line_source">[Sheng Li Xue Bao.  2010]</span><div class="alt-note">
<div class="authors">Tang N, Jin J, Deng Y, Ke RH, Shen QJ, Fan SH, Qin WX</div>
<div class="citation">Sheng Li Xue Bao. 2010 Jun 25; 62(3):196-202.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p33" name="crb--__p33" rid="__p33" style="position: absolute; visibility: visible; width: 370px; top: 12207px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-35"><ul><li class="two_line" reference_id="DDR415C12">
<a href="/pubmed/17701901/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PLINK: a tool set for whole-genome association and population-based linkage analyses.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC</div>
<div class="citation">Am J Hum Genet. 2007 Sep; 81(3):559-75.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p34" name="crb--__p34" rid="__p34" style="position: absolute; visibility: visible; width: 370px; top: 12478px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-36"><ul><li class="two_line" reference_id="DDR415C21">
<a href="/pubmed/20091798/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The Gene, Environment Association Studies consortium (GENEVA): maximizing the knowledge obtained from GWAS by collaboration across studies of multiple conditions.</a><span class="one_line_source">[Genet Epidemiol.  2010]</span><div class="alt-note">
<div class="authors">Cornelis MC, Agrawal A, Cole JW, Hansel NN, Barnes KC, Beaty TH, Bennett SN, Bierut LJ, Boerwinkle E, Doheny KF, Feenstra B, Feingold E, Fornage M, Haiman CA, Harris EL, Hayes MG, Heit JA, Hu FB, Kang JH, Laurie CC, Ling H, Manolio TA, Marazita ML, Mathias RA, Mirel DB, Paschall J, Pasquale LR, Pugh EW, Rice JP, Udren J, van Dam RM, Wang X, Wiggs JL, Williams K, Yu K, GENEVA Consortium.</div>
<div class="citation">Genet Epidemiol. 2010 May; 34(4):364-72.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p35" name="crb--__p35" rid="__p35" style="position: absolute; visibility: visible; width: 370px; top: 12806px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-37"><ul><li class="two_line" reference_id="DDR415C12">
<a href="/pubmed/17701901/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PLINK: a tool set for whole-genome association and population-based linkage analyses.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC</div>
<div class="citation">Am J Hum Genet. 2007 Sep; 81(3):559-75.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p38" name="crb--__p38" rid="__p38" style="position: absolute; visibility: visible; width: 370px; top: 13451px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-38"><ul><li class="two_line" reference_id="DDR415C22">
<a href="/pubmed/19707526/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">European population genetic substructure: further definition of ancestry informative markers for distinguishing among diverse European ethnic groups.</a><span class="one_line_source">[Mol Med.  2009]</span><div class="alt-note">
<div class="authors">Tian C, Kosoy R, Nassir R, Lee A, Villoslada P, Klareskog L, Hammarström L, Garchon HJ, Pulver AE, Ransom M, Gregersen PK, Seldin MF</div>
<div class="citation">Mol Med. 2009 Nov-Dec; 15(11-12):371-83.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p39" name="crb--__p39" rid="__p39" style="position: absolute; visibility: visible; width: 370px; top: 13767px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-39"><ul>
<li class="two_line" reference_id="DDR415C23">
<a href="/pubmed/18483556/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A genome-wide association study identifies novel alleles associated with hair color and skin pigmentation.</a><span class="one_line_source">[PLoS Genet.  2008]</span><div class="alt-note">
<div class="authors">Han J, Kraft P, Nan H, Guo Q, Chen C, Qureshi A, Hankinson SE, Hu FB, Duffy DL, Zhao ZZ, Martin NG, Montgomery GW, Hayward NK, Thomas G, Hoover RN, Chanock S, Hunter DJ</div>
<div class="citation">PLoS Genet. 2008 May 16; 4(5):e1000074.</div>
</div>
</li>
<li class="two_line" reference_id="DDR415C25">
<a href="/pubmed/17529973/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer.</a><span class="one_line_source">[Nat Genet.  2007]</span><div class="alt-note">
<div class="authors">Hunter DJ, Kraft P, Jacobs KB, Cox DG, Yeager M, Hankinson SE, Wacholder S, Wang Z, Welch R, Hutchinson A, Wang J, Yu K, Chatterjee N, Orr N, Willett WC, Colditz GA, Ziegler RG, Berg CD, Buys SS, McCarty CA, Feigelson HS, Calle EE, Thun MJ, Hayes RB, Tucker M, Gerhard DS, Fraumeni JF Jr, Hoover RN, Thomas G, Chanock SJ</div>
<div class="citation">Nat Genet. 2007 Jul; 39(7):870-4.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p47" name="crb--__p47" rid="__p47" style="position: absolute; visibility: visible; width: 370px; top: 15611px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-40"><ul><li class="two_line" reference_id="DDR415C12">
<a href="/pubmed/17701901/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">PLINK: a tool set for whole-genome association and population-based linkage analyses.</a><span class="one_line_source">[Am J Hum Genet.  2007]</span><div class="alt-note">
<div class="authors">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC</div>
<div class="citation">Am J Hum Genet. 2007 Sep; 81(3):559-75.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p48" name="crb--__p48" rid="__p48" style="position: absolute; visibility: visible; width: 370px; top: 15707px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-41"><ul><li class="two_line" reference_id="DDR415C26">
<a href="/pubmed/17554300/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls.</a><span class="one_line_source">[Nature.  2007]</span><div class="alt-note">
<div class="authors">Wellcome Trust Case Control Consortium.</div>
<div class="citation">Nature. 2007 Jun 7; 447(7145):661-78.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p55" name="crb--__p55" rid="__p55" style="position: absolute; visibility: visible; width: 370px; top: 16393px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-42"><ul><li class="two_line" reference_id="DDR415C27">
<a href="/pubmed/21430779/" ref="reftype=pubmed&amp;article-id=3298855&amp;issue-id=202806&amp;journal-id=740&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The histone methyltransferase SETDB1 is recurrently amplified in melanoma and accelerates its onset.</a><span class="one_line_source">[Nature.  2011]</span><div class="alt-note">
<div class="authors">Ceol CJ, Houvras Y, Jane-Valbuena J, Bilodeau S, Orlando DA, Battisti V, Fritsch L, Lin WM, Hollmann TJ, Ferré F, Bourque C, Burke CJ, Turner L, Uong A, Johnson LA, Beroukhim R, Mermel CH, Loda M, Ait-Si-Ali S, Garraway LA, Young RA, Zon LI</div>
<div class="citation">Nature. 2011 Mar 24; 471(7339):513-7.</div>
</div>
</li></ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:43:35-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89FE04DD5319C100000000022F0160&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3298855%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3298855/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>