<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3322238/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Combined Analysis of Genome-wide Association Studies for Crohn Disease and Psoriasis Identifies Seven Shared Susceptibility Loci</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322238/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Combined Analysis of Genome-wide Association Studies for Crohn Disease and Psoriasis Identifies Seven Shared Susceptibility Loci"><meta name="citation_authors" content="David Ellinghaus, Eva Ellinghaus, Rajan&nbsp;P. Nair, Philip&nbsp;E. Stuart, Tõnu Esko, Andres Metspalu, Sophie Debrus, John&nbsp;V. Raelson, Trilokraj Tejasvi, Majid Belouchi, Sarah&nbsp;L. West, Jonathan&nbsp;N. Barker, Sulev Kõks, Külli Kingo, Tobias Balschun, Orazio Palmieri, Vito Annese, Christian Gieger, H.&nbsp;Erich Wichmann, Michael Kabesch, Richard&nbsp;C. Trembath, Christopher&nbsp;G. Mathew, Gonçalo&nbsp;R. Abecasis, Stephan Weidinger, Susanna Nikolaus, Stefan Schreiber, James&nbsp;T. Elder, Michael Weichenthal, Michael Nothnagel, Andre Franke"><meta name="citation_date" content="6 April 2012"><meta name="citation_issue" content="4"><meta name="citation_volume" content="90"><meta name="citation_firstpage" content="636"><meta name="citation_doi" content="10.1016/j.ajhg.2012.02.020"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3322238/?report=abstract"><meta name="citation_pmid" content="22482804"><meta name="DC.Title" content="Combined Analysis of Genome-wide Association Studies for Crohn Disease and Psoriasis Identifies Seven Shared Susceptibility Loci"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="David Ellinghaus"><meta name="DC.Contributor" content="Eva Ellinghaus"><meta name="DC.Contributor" content="Rajan&nbsp;P. Nair"><meta name="DC.Contributor" content="Philip&nbsp;E. Stuart"><meta name="DC.Contributor" content="Tõnu Esko"><meta name="DC.Contributor" content="Andres Metspalu"><meta name="DC.Contributor" content="Sophie Debrus"><meta name="DC.Contributor" content="John&nbsp;V. Raelson"><meta name="DC.Contributor" content="Trilokraj Tejasvi"><meta name="DC.Contributor" content="Majid Belouchi"><meta name="DC.Contributor" content="Sarah&nbsp;L. West"><meta name="DC.Contributor" content="Jonathan&nbsp;N. Barker"><meta name="DC.Contributor" content="Sulev Kõks"><meta name="DC.Contributor" content="Külli Kingo"><meta name="DC.Contributor" content="Tobias Balschun"><meta name="DC.Contributor" content="Orazio Palmieri"><meta name="DC.Contributor" content="Vito Annese"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="H.&nbsp;Erich Wichmann"><meta name="DC.Contributor" content="Michael Kabesch"><meta name="DC.Contributor" content="Richard&nbsp;C. Trembath"><meta name="DC.Contributor" content="Christopher&nbsp;G. Mathew"><meta name="DC.Contributor" content="Gonçalo&nbsp;R. Abecasis"><meta name="DC.Contributor" content="Stephan Weidinger"><meta name="DC.Contributor" content="Susanna Nikolaus"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="James&nbsp;T. Elder"><meta name="DC.Contributor" content="Michael Weichenthal"><meta name="DC.Contributor" content="Michael Nothnagel"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Date" content="2012 Apr 6"><meta name="DC.Identifier" content="10.1016/j.ajhg.2012.02.020"><meta name="DC.Language" content="en"><meta property="og:title" content="Combined Analysis of Genome-wide Association Studies for Crohn Disease and Psoriasis Identifies Seven Shared Susceptibility Loci"><meta property="og:type" content="article"><meta property="og:description" content="Psoriasis (PS) and Crohn disease (CD) have been shown to be epidemiologically, pathologically, and therapeutically connected, but little is known about their shared genetic causes. We performed meta-analyses of five published genome-wide association studies ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3322238/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3322238/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3322238/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE88A2F0DD540FA100000000015C00E1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3322238%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books">Books</option><option value="taxonomy">Taxonomy</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/208649/">v.90(4); 2012 Apr 6</a></li><li class="accid">PMC3322238</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2012 Apr 6; 90(4): 636–647. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2012.02.020" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2012.02.020</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3322238</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22482804">22482804</a></div></div></div></div><h1 class="content-title">Combined Analysis of Genome-wide Association Studies for Crohn Disease and Psoriasis Identifies Seven Shared Susceptibility Loci</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408032671008" co-class="co-affbox">David Ellinghaus</a>,<sup>1</sup> <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408032669136" co-class="co-affbox">Eva Ellinghaus</a>,<sup>1</sup> <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408032667600" co-class="co-affbox">Rajan&nbsp;P. Nair</a>,<sup>2</sup> <a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408032665840" co-class="co-affbox">Philip&nbsp;E. Stuart</a>,<sup>2</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408041216096" co-class="co-affbox">Tõnu Esko</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408041213376" co-class="co-affbox">Andres Metspalu</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408041210656" co-class="co-affbox">Sophie Debrus</a>,<sup>5</sup> <a href="/pubmed/?term=Raelson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408041208672" co-class="co-affbox">John&nbsp;V. Raelson</a>,<sup>6</sup> <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408041206688" co-class="co-affbox">Trilokraj Tejasvi</a>,<sup>2</sup> <a href="/pubmed/?term=Belouchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408113545360" co-class="co-affbox">Majid Belouchi</a>,<sup>7</sup> <a href="/pubmed/?term=West%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408113543376" co-class="co-affbox">Sarah&nbsp;L. West</a>,<sup>8</sup> <a href="/pubmed/?term=Barker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408113541392" co-class="co-affbox">Jonathan&nbsp;N. Barker</a>,<sup>8</sup> <a href="/pubmed/?term=K%26%23x000f5%3Bks%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408113539408" co-class="co-affbox">Sulev Kõks</a>,<sup>9</sup> <a href="/pubmed/?term=Kingo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408113537424" co-class="co-affbox">Külli Kingo</a>,<sup>10</sup> <a href="/pubmed/?term=Balschun%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408110626096" co-class="co-affbox">Tobias Balschun</a>,<sup>1</sup> <a href="/pubmed/?term=Palmieri%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408110624112" co-class="co-affbox">Orazio Palmieri</a>,<sup>11</sup> <a href="/pubmed/?term=Annese%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408110622128" co-class="co-affbox">Vito Annese</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408110619408" co-class="co-affbox">Christian Gieger</a>,<sup>13</sup> <a href="/pubmed/?term=Wichmann%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408110617424" co-class="co-affbox">H.&nbsp;Erich Wichmann</a>,<sup>14,</sup><sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Kabesch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408042938880" co-class="co-affbox">Michael Kabesch</a>,<sup>17</sup> <a href="/pubmed/?term=Trembath%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408042936896" co-class="co-affbox">Richard&nbsp;C. Trembath</a>,<sup>8</sup> <a href="/pubmed/?term=Mathew%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408042934912" co-class="co-affbox">Christopher&nbsp;G. Mathew</a>,<sup>8</sup> <a href="/pubmed/?term=Abecasis%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408042932928" co-class="co-affbox">Gonçalo&nbsp;R. Abecasis</a>,<sup>18</sup> <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408042930944" co-class="co-affbox">Stephan Weidinger</a>,<sup>19</sup> <a href="/pubmed/?term=Nikolaus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408115093856" co-class="co-affbox">Susanna Nikolaus</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408115091136" co-class="co-affbox">Stefan Schreiber</a>,<sup>1,</sup><sup>21</sup> <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408115088416" co-class="co-affbox">James&nbsp;T. Elder</a>,<sup>2,</sup><sup>22</sup> <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408115085696" co-class="co-affbox">Michael Weichenthal</a>,<sup>19</sup> <a href="/pubmed/?term=Nothnagel%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408114978192" co-class="co-affbox">Michael Nothnagel</a>,<sup>23,</sup><sup>24</sup> and  <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804" class="affpopup" co-rid="_co_idm140408114975472" co-class="co-affbox">Andre Franke</a><sup>1,</sup><sup>24,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140408032671008"><h3 class="no_margin">David Ellinghaus</h3><p><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">David Ellinghaus</a></div></div><div id="_co_idm140408032669136"><h3 class="no_margin">Eva Ellinghaus</h3><p><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Eva Ellinghaus</a></div></div><div id="_co_idm140408032667600"><h3 class="no_margin">Rajan&nbsp;P. Nair</h3><p><sup>2</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Rajan&nbsp;P. Nair</a></div></div><div id="_co_idm140408032665840"><h3 class="no_margin">Philip&nbsp;E. Stuart</h3><p><sup>2</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Philip&nbsp;E. Stuart</a></div></div><div id="_co_idm140408041216096"><h3 class="no_margin">Tõnu Esko</h3><p><sup>3</sup>Estonian Genome Center, University of Tartu, 50409 Tartu, Estonia</p><p><sup>4</sup>Institute of Molecular and Cell Biology, University of Tartu, 50409 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Tõnu Esko</a></div></div><div id="_co_idm140408041213376"><h3 class="no_margin">Andres Metspalu</h3><p><sup>3</sup>Estonian Genome Center, University of Tartu, 50409 Tartu, Estonia</p><p><sup>4</sup>Institute of Molecular and Cell Biology, University of Tartu, 50409 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Andres Metspalu</a></div></div><div id="_co_idm140408041210656"><h3 class="no_margin">Sophie Debrus</h3><p><sup>5</sup>Gatineau, QC J9J 2X6, Canada</p><div>Find articles by <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Sophie Debrus</a></div></div><div id="_co_idm140408041208672"><h3 class="no_margin">John&nbsp;V. Raelson</h3><p><sup>6</sup>PGX-Services, Montreal, QC H2T 1S1, Canada</p><div>Find articles by <a href="/pubmed/?term=Raelson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">John&nbsp;V. Raelson</a></div></div><div id="_co_idm140408041206688"><h3 class="no_margin">Trilokraj Tejasvi</h3><p><sup>2</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Trilokraj Tejasvi</a></div></div><div id="_co_idm140408113545360"><h3 class="no_margin">Majid Belouchi</h3><p><sup>7</sup>Genizon BioSciences, Inc., St. Laurent, QC H4T 2C7, Canada</p><div>Find articles by <a href="/pubmed/?term=Belouchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Majid Belouchi</a></div></div><div id="_co_idm140408113543376"><h3 class="no_margin">Sarah&nbsp;L. West</h3><p><sup>8</sup>Division of Genetics and Molecular Medicine, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=West%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Sarah&nbsp;L. West</a></div></div><div id="_co_idm140408113541392"><h3 class="no_margin">Jonathan&nbsp;N. Barker</h3><p><sup>8</sup>Division of Genetics and Molecular Medicine, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Barker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Jonathan&nbsp;N. Barker</a></div></div><div id="_co_idm140408113539408"><h3 class="no_margin">Sulev Kõks</h3><p><sup>9</sup>Department of Physiology, Centre of Translational Medicine and Centre of Translational Genomics, University of Tartu, 50409 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f5%3Bks%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Sulev Kõks</a></div></div><div id="_co_idm140408113537424"><h3 class="no_margin">Külli Kingo</h3><p><sup>10</sup>Department of Dermatology and Venerology, University of Tartu, 50409 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Kingo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Külli Kingo</a></div></div><div id="_co_idm140408110626096"><h3 class="no_margin">Tobias Balschun</h3><p><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Balschun%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Tobias Balschun</a></div></div><div id="_co_idm140408110624112"><h3 class="no_margin">Orazio Palmieri</h3><p><sup>11</sup>Division of Gastroenterology, Istituto di Ricovero e Cura a Carattere Scientifico-Casa Sollievo della Sofferenza Hospital, San Giovanni Rotondo 71013, Italy</p><div>Find articles by <a href="/pubmed/?term=Palmieri%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Orazio Palmieri</a></div></div><div id="_co_idm140408110622128"><h3 class="no_margin">Vito Annese</h3><p><sup>11</sup>Division of Gastroenterology, Istituto di Ricovero e Cura a Carattere Scientifico-Casa Sollievo della Sofferenza Hospital, San Giovanni Rotondo 71013, Italy</p><p><sup>12</sup>Unit of Gastroenterology SOD2, Azienda Ospedaliero Universitaria Careggi, Florence 50134, Italy</p><div>Find articles by <a href="/pubmed/?term=Annese%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Vito Annese</a></div></div><div id="_co_idm140408110619408"><h3 class="no_margin">Christian Gieger</h3><p><sup>13</sup>Institute of Genetic Epidemiology, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Christian Gieger</a></div></div><div id="_co_idm140408110617424"><h3 class="no_margin">H.&nbsp;Erich Wichmann</h3><p><sup>14</sup>Institute of Epidemiology I, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>15</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</p><p><sup>16</sup>Klinikum Grosshadern, 81377 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">H.&nbsp;Erich Wichmann</a></div></div><div id="_co_idm140408042938880"><h3 class="no_margin">Michael Kabesch</h3><p><sup>17</sup>Department of Paediatric Pneumology, Allergy and Neonatology, Hannover Medical School, 30625 Hannover, Germany</p><div>Find articles by <a href="/pubmed/?term=Kabesch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Michael Kabesch</a></div></div><div id="_co_idm140408042936896"><h3 class="no_margin">Richard&nbsp;C. Trembath</h3><p><sup>8</sup>Division of Genetics and Molecular Medicine, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Trembath%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Richard&nbsp;C. Trembath</a></div></div><div id="_co_idm140408042934912"><h3 class="no_margin">Christopher&nbsp;G. Mathew</h3><p><sup>8</sup>Division of Genetics and Molecular Medicine, King's College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Mathew%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Christopher&nbsp;G. Mathew</a></div></div><div id="_co_idm140408042932928"><h3 class="no_margin">Gonçalo&nbsp;R. Abecasis</h3><p><sup>18</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Gonçalo&nbsp;R. Abecasis</a></div></div><div id="_co_idm140408042930944"><h3 class="no_margin">Stephan Weidinger</h3><p><sup>19</sup>Department of Dermatology, Allergology, and Venerology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Stephan Weidinger</a></div></div><div id="_co_idm140408115093856"><h3 class="no_margin">Susanna Nikolaus</h3><p><sup>20</sup>PopGen Biobank, Christian-Albrechts-University Kiel, 24105 Kiel, Germany</p><p><sup>21</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Nikolaus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Susanna Nikolaus</a></div></div><div id="_co_idm140408115091136"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><p><sup>21</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Stefan Schreiber</a></div></div><div id="_co_idm140408115088416"><h3 class="no_margin">James&nbsp;T. Elder</h3><p><sup>2</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><p><sup>22</sup>Ann Arbor Veterans Affairs Hopital, Ann Arbor, MI 48105, USA</p><div>Find articles by <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">James&nbsp;T. Elder</a></div></div><div id="_co_idm140408115085696"><h3 class="no_margin">Michael Weichenthal</h3><p><sup>19</sup>Department of Dermatology, Allergology, and Venerology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Michael Weichenthal</a></div></div><div id="_co_idm140408114978192"><h3 class="no_margin">Michael Nothnagel</h3><p><sup>23</sup>Institute of Medical Informatics and Statistics, Christian-Albrechts University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Nothnagel%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Michael Nothnagel</a></div></div><div id="_co_idm140408114975472"><h3 class="no_margin">Andre Franke</h3><p><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22482804">Andre Franke</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140408032427120_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140408032427120_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140408032427120_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140408032427120_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Estonian Genome Center, University of Tartu, 50409 Tartu, Estonia</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Institute of Molecular and Cell Biology, University of Tartu, 50409 Tartu, Estonia</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Gatineau, QC J9J 2X6, Canada</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>PGX-Services, Montreal, QC H2T 1S1, Canada</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Genizon BioSciences, Inc., St. Laurent, QC H4T 2C7, Canada</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Division of Genetics and Molecular Medicine, King's College London, London SE1 9RT, UK</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Physiology, Centre of Translational Medicine and Centre of Translational Genomics, University of Tartu, 50409 Tartu, Estonia</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Dermatology and Venerology, University of Tartu, 50409 Tartu, Estonia</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Division of Gastroenterology, Istituto di Ricovero e Cura a Carattere Scientifico-Casa Sollievo della Sofferenza Hospital, San Giovanni Rotondo 71013, Italy</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Unit of Gastroenterology SOD2, Azienda Ospedaliero Universitaria Careggi, Florence 50134, Italy</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Institute of Genetic Epidemiology, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Institute of Epidemiology I, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Klinikum Grosshadern, 81377 Munich, Germany</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Department of Paediatric Pneumology, Allergy and Neonatology, Hannover Medical School, 30625 Hannover, Germany</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Department of Dermatology, Allergology, and Venerology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>PopGen Biobank, Christian-Albrechts-University Kiel, 24105 Kiel, Germany</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Department of General Internal Medicine, University Hospital Schleswig-Holstein, 24105 Kiel, Germany</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Ann Arbor Veterans Affairs Hopital, Ann Arbor, MI 48105, USA</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Institute of Medical Informatics and Statistics, Christian-Albrechts University, 24105 Kiel, Germany</div><div><span class="contrib-email" id="idm140408114975472">Andre Franke: <a href="mailto:dev@null" data-email="ed.asocum@eknarf.a" class="oemail">ed.asocum@eknarf.a</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ed.asocum@eknarf.a" class="oemail">ed.asocum@eknarf.a</a></div><div id="fn1"><sup>24</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140408032427120_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Oct 28; Revised 2012 Jan 30; Accepted 2012 Feb 16.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140408032427120_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3322238/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1–S6 and Table S1–S5</div><div><a href="/pmc/articles/PMC3322238/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (1.9M)</span></div><div class="small guid">GUID:&nbsp;43E74981-1C56-4B51-8A3F-D80295F06F83</div></div></dd></dl></div><div id="idm140408040171520" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140408040171520title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Psoriasis (PS) and Crohn disease (CD) have been shown to be epidemiologically, pathologically, and therapeutically connected, but little is known about their shared genetic causes. We performed meta-analyses of five published genome-wide association studies on PS (2,529 cases and 4,955 controls) and CD (2,142 cases and 5,505 controls), followed up 20 loci that showed strongest evidence for shared disease association and, furthermore, tested cross-disease associations for previously reported PS and CD risk alleles in additional&nbsp;6,115 PS cases, 4,073 CD cases, and 10,100 controls. We identified seven susceptibility loci outside the human leukocyte antigen region (9p24 near <em>JAK2,</em> 10q22 at <em>ZMIZ1</em>, 11q13 near <em>PRDX5</em>, 16p13 near <em>SOCS1</em>, 17q21 at <em>STAT3</em>, 19p13 near <em>FUT2</em>, and 22q11 at <em>YDJC</em>) shared between PS and CD with genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>) and confirmed four already established PS and CD risk loci (<em>IL23R</em>, <em>IL12B</em>, <em>REL</em>, and <em>TYK2</em>). Three of the shared loci are also genome-wide significantly associated with PS alone (10q22 at <em>ZMIZ1</em>, p<sub>rs1250544</sub> = 3.53&nbsp;×&nbsp;10<sup>−8</sup>, 11q13 near <em>PRDX5</em>, p<sub>rs694739</sub> = 3.71&nbsp;× 10<sup>−09</sup>, 22q11 at <em>YDJC</em>, p<sub>rs181359</sub> = 8.02&nbsp;× 10<sup>−10</sup>). In addition, we identified one susceptibility locus for CD (16p13 near <em>SOCS1</em>, p<sub>rs4780355</sub> = 4.99&nbsp;× 10<sup>−8</sup>). Refinement of association signals identified shared genome-wide significant associations for exonic SNPs at 10q22 (<em>ZMIZ1</em>) and in&nbsp;silico expression quantitative trait locus analyses revealed that the associations at <em>ZMIZ1</em> and near <em>SOCS1</em> have a potential functional effect on gene expression. Our results show the usefulness of joint analyses of clinically distinct immune-mediated diseases and enlarge the map of shared genetic risk loci.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p3" class="p p-first">Psoriasis (PS [MIM <a href="/omim/177900" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">177900</a>]) and Crohn disease (CD [MIM <a href="/omim/266600" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">266600</a>]) are both chronic inflammatory epithelial disorders that are triggered by an activated cellular immune system and have an estimated sibling relative risk (λ<sub>s</sub>) of&nbsp;4–11<sup><a href="#bib1" rid="bib1 bib2" class=" bibr popnode">1,2</a></sup> and 25–42,<sup><a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988784">3</a></sup> respectively, and a prevalence of 2%–3% and about 0.1%, respectively, in populations of European ancestry.<sup><a href="#bib4" rid="bib4 bib5" class=" bibr popnode">4,5</a></sup> PS is a common hyperproliferative disorder of the skin, characterized by red scaly plaques, typically occurring on the elbows, knees, scalp, and lower back.<sup><a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988808">6</a></sup> In contrast, CD is primarily a gut disorder affecting any aspect of the gastrointestinal tract but with extraintestinal manifestations which might also affect the skin (e.g., erythema nodosum and pyoderma gangraenosum).<sup><a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988807">7</a></sup> It results from the interaction of environmental factors, including the commensal microflora, with host immune mechanisms in a genetically susceptible host.<sup><a href="#bib8" rid="bib8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988816">8</a></sup> Although PS and CD are clinically distinct&nbsp;diseases, they are observed together more frequently than expected by chance, which could indicate shared genetic factors acting in the etiology of both diseases.<sup><a href="#bib9" rid="bib9 bib10 bib11" class=" bibr popnode">9–11</a></sup> Recently, several genome-wide association studies (GWASs) have successfully been carried out separately for CD and&nbsp;PS<sup><a href="#bib12" rid="bib12 bib13 bib14 bib15 bib16 bib17 bib18 bib19 bib20" class=" bibr popnode">12–20</a></sup> and identified shared susceptibility genes, such&nbsp;as <em>IL23R</em> (MIM <a href="/omim/607562" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607562</a>), <em>IL12B</em> (MIM <a href="/omim/161561" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">161561</a>), <em>REL</em>&nbsp;(MIM <a href="/omim/164910" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">164910</a>), and <em>TYK2</em> (MIM <a href="/omim/176941" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">176941</a>), thereby providing further evidence for a genetic overlap of both&nbsp;diseases.<sup><a href="#bib13" rid="bib13 bib16 bib21 bib22 bib23" class=" bibr popnode">13,16,21–23</a></sup> One of the best characterized risk&nbsp;loci for both CD and PS is&nbsp;<em>IL23R</em>, located in&nbsp;a drug-targetable pathway.<sup><a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988793">24</a></sup> IL-23,&nbsp;a pro-inflammatory cytokine, is thought to be a key player driving autoimmunity in human&nbsp;disease.<sup><a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988805">25</a></sup> A recent functional characterization of the amino acid substitution R381Q in <em>IL23R</em> suggests that IL-23-induced Th17 cell effector function is reduced in protective allele carriers and leads to protection against several autoimmune diseases, including PS, CD, and ankylosing spondylitis.<sup><a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988772">26</a></sup></p><p id="__p4">So far, shared susceptibility loci for CD and PS have been identified by single-disease GWAS for CD or PS separately, and established risk SNPs for one disease are usually tested for association in another disease,<sup><a href="#bib27" rid="bib27 bib28 bib29 bib30 bib31" class=" bibr popnode">27–31</a></sup> rather than in&nbsp;a combined systematic approach. Combined GWASs were only conducted across clinically related phenotypes, such as CD and ulcerative colitis (UC),<sup><a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988788">32</a></sup> CD and sarcoidosis (SA)<sup><a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988819">27</a></sup> or CD and celiac disease (CelD).<sup><a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988790">33</a></sup> Recently, Zhernakova et&nbsp;al.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988797">34</a></sup> performed a meta-analysis with a similar systematic approach that combined genome-wide genotype data from two autoimmune diseases affecting different organs, namely CelD and rheumatoid arthritis (RA). They identified eight shared risk loci outside the human leukocyte antigen (HLA) region for CelD and RA, four of them previously not known to be associated with either CelD or RA. Based on genome-wide SNP data for CelD and RA, the authors identified an increased probability for CelD risk SNPs to confer also an increase in risk for RA and vice versa. Zhernakova et&nbsp;al.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988801">34</a></sup> postulated criteria for declaring a SNP as being a shared risk factor for two clinically distinct diseases, namely that shared SNPs (1) have to reach genome-wide significance in the combined analysis of the initial GWAS screening stage and the replication stage of the two distinct diseases (p<sub>GWAS+Repl</sub> &lt; 5&nbsp;× 10<sup>−8</sup>) and (2) have to achieve, for each disease separately, nominal significance in the replication stage (p<sub>Repl</sub> &lt; 0.05) as well as p<sub>GWAS+Repl</sub> &lt; 10<sup>−3</sup> in the combined analysis of screening and replication stage.</p><p id="__p5" class="p p-last">We used the criteria proposed by Zhernakova et&nbsp;al.<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988800">34</a></sup> in a genome-wide association analysis combining CD and PS to systematically identify shared risk loci associated with both diseases. A two-fold strategy was employed: in a first approach (<em>OVERLAP</em>), we tested established non-HLA CD risk SNPs for association with PS and vice versa, thereby seeking confirmation of whether known risk loci for one disease also play a role in the etiology of the other disease, disregarding the direction of effect. In a second approach (<em>COMBINED</em>), we performed a meta-analysis for the combined phenotype based on genome-wide data sets of both CD and PS in order to increase power for the detection of new shared risk alleles because of an increased sample size. The latter approach allows consideration of same-direction as well as opposing-direction allelic effects of putative shared markers between CD and PS through the use of suitable allele coding. We followed up 20 loci that showed the strongest association in the <em>COMBINED</em> approach and that had not been previously reported as&nbsp;being risk factors for either CD or PS. Follow-up was&nbsp;performed in independent replication panels from Germany, Estonia, Italy, United Kingdom, and the United States (see <a href="#app2" rid="app2" class=" sec">Table S1</a>, available online).</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Subjects and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Study Subjects</h3><p id="__p6" class="p p-first">We analyzed a collection of different data sets. <a href="/pmc/articles/PMC3322238/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a> details the different panels and their use in this study. For discovery, we combined genome-wide case-control data of single-nucleotide polymorphisms (SNP) for psoriasis (PS; panel A) and Crohn disease (CD; panel B) (see <a href="#app2" rid="app2" class=" sec">Table S1</a>) respectively, conducted genome-wide meta-analyses on PS and CD, respectively, and employed two strategies (<em>COMBINED</em> and <em>OVERLAP</em>, see below) to systematically identify risk loci associated with both diseases. Replication was performed in independent replication panels from Germany (see panels C–E), Estonia (panels C and D), Italy (panel E), United Kingdom (panels C and E), and the United States (panel C) (see <a href="#app2" rid="app2" class=" sec">Table S1</a>). Written, informed consent was obtained from all study participants and all protocols were approved by the institutional ethical review committees of the participating centers.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3322238/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408044246320" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC3322238/bin/gr1.jpg"></div></a><div id="largeobj_idm140408044246320" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3322238/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC3322238/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3322238/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p40">Study Design for the Combined Analysis of CD and PS</p><p id="__p41">For discovery, we conducted a PS and a CD GWAS meta-analysis (panel A and B in <a href="#app2" rid="app2" class=" sec">Table S1</a>), respectively, and employed two strategies (<em>OVERLAP</em> and <em>COMBINED</em>) to systematically search for shared risk loci. In a first approach (<em>OVERLAP</em>), we tested established non-HLA PS risk SNPs for potential association (p &lt; 0.01) with CD and vice versa. In a second approach (<em>COMBINED</em>), we selected SNPs from 20 loci for being nominally associated in each of the single-disease meta-analyses (p<sub>panel A</sub> &lt; 0.05, p<sub>panel B</sub> &lt; 0.05) and for being significantly associated in the combined-phenotype association analysis at the 10<sup>−4</sup> level (p<sub>panel A&amp;B</sub> &lt; 10<sup>−4</sup>). For replication, follow-up SNP genotyping was performed for CD and PS in independent replication panels (panels C–E in <a href="#app2" rid="app2" class=" sec">Table S1</a>). The following abbreviations are&nbsp;used: PS-GER, German PS GWAS; PS-US, United States PS GWAS; PS-Canada, Canadian PS&nbsp;GWAS; CD-GER, German CD GWAS; and CD-UK, United Kingdom CD GWAS. For each panel, numbers of cases/controls are displayed in parentheses.</p></div></div></div><div id="sec2.1.1" class="sec"><p></p><h4 id="sec2.1.1title" class="inline">Initial GWAS and German Replication Data </h4><p id="__p7" class="p p-first-last">All German CD patients in discovery and replication panels (B&nbsp;and C–E, respectively) were recruited either at the Department of General Internal Medicine, Christian-Albrechts-University, Kiel, and the Charité University Hospital, Berlin; through local outpatient services; or nationwide with the support of the German Crohn and Colitis Foundation. German PS cases in discovery and replication panels (A and C and D, respectively) were recruited either at the Department of Dermatology, Christian-Albrechts-University, Kiel, or the Department of Dermatology and Allergy, Technical University, Munich, or through local outpatient services. Individuals were considered to be affected by PS if chronic plaque or guttate psoriasis lesions covered more than 1% of the total body surface area or if at least two skin, scalp, nail, or joint lesions were clinically diagnosed as psoriasis. The 4,680 German healthy control individuals in discovery and replication panels (A–E) were obtained from the Popgen biobank.<sup><a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988799">35</a></sup> The additional 3,391 German healthy controls (after quality control measures) in the discovery panels (A&nbsp;and B) were selected from the KORA S3+S4 survey, an independent population-based sample from the general population living in the region of Augsburg, southern Germany.<sup><a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988829">36</a></sup> Another 674 German healthy controls in the discovery panels (A and B) were selected from ISAAC Phase II study.<sup><a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988824">37</a></sup> German GWAS controls of the discovery phase were randomly assigned to panels A and B at equal proportions, while ensuring that controls in panel A did not overlap with German GWAS controls used in the independent genome-wide meta-analysis on CD.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988828">13</a></sup> The Collaborative Association Study of Psoriasis (CASP) samples<sup><a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988809">14</a></sup> consisted of 1,303 PS cases and 1,322 controls after quality control measures and are part of panel A in our study. The data sets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype (dbGaP). The genotyping of samples was provided through the Genetic Association Information Network (GAIN).<sup><a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988833">38</a></sup> The Canadian samples (from Genizon BioSciences and M. Belouchi, unpublished data) consisted of 757 PS cases and 987 controls sampled from the Québec founder population (QFP) after quality control measures. They are part of panel A. Membership in the QFP was defined as having four grandparents with French-Canadian family names who were born in the Province of Québec, Canada, or in adjacent areas in the provinces of New Brunswick and Ontario or in New England or New York state. This criterion assured that all subjects were descendants of French-Canadians living before the 1960s, after which time admixture with non-French-Canadians became more common. CD cases and controls from the United Kingdom were recruited from the 1958 birth cohort&nbsp;and UK National Blood Service for the Welcome Trust Case Controls Consortium (WTCCC) (described in details in WTCCC<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988822">28</a></sup>). The WTCCC1 CD samples consisted of 1,662 CD cases and 2,860 healthy controls after quality control procedures and entered the analysis as part of panel B.</p></div><div id="sec2.1.2" class="sec sec-last"><p></p><h4 id="sec2.1.2title" class="inline">Additional Replication Data </h4><p id="__p8" class="p p-first">A number of collaborative data sets were used as replication&nbsp;panels. The Estonian samples used in the <em>OVERLAP</em> and <em>COMBINED</em> approaches (part of panels C and D) were collected at the Department of Dermatology and Venerology and at the Department of Physiology and Centre of Translational Medicine at the University of Tartu.</p><p id="__p9" class="p p-last">Additional Estonian samples (part of panel C) used for replication in the <em>OVERLAP</em> approach consisted of samples provided by the population-based biobank of the Estonian Genome Center, University of Tartu. Subjects were recruited by general practitioners (GP) and physicians in the hospitals. Participants in the hospitals were randomly selected from individuals visiting GP offices or hospitals. Diagnosis of PS on the basis of clinical symptoms was posed by a general practitioner and confirmed by a dermatologist. At the moment of recruitment, the controls did not report diagnosis of osteoarthritis, psoriasis, or autoimmune diseases. The United States samples (part of panel C) used for replication in the <em>OVERLAP</em> approach consisted of 2,137 PS cases and 1,903 controls of white European ancestry from the United States. The Italian samples (panel E) used in the <em>COMBINED</em> approach consisted of 688 CD cases and 879 healthy controls that were used in the independent genome-wide meta-analysis on CD.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988802">13</a></sup> The psoriasis data set from the United Kingdom consisted of 2,178 PS cases collected through the Genetic Analysis of Psoriasis Consortium (GAPC) and 2,657 controls from the WTCCC2 common control set, used as the GWAS discovery set described in Strange et&nbsp;al. in 2010.<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343190026">16</a></sup> Only controls that did not overlap with WTCCC1 controls were used. UK cases and controls entered the analysis as part of panel C and E.</p></div></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Quality Control and Genome-wide Genotype Imputation</h3><p id="__p10" class="p p-first-last">Quality control (QC) was performed for each sample set separately. In each sample set samples with more than 5% missing&nbsp;data were&nbsp;excluded before genotype imputation. We also excluded individuals from each pair of unexpected duplicates or relatives, as well as outlier individuals with average marker heterozygosities of ±5 standard deviation away from the sample mean. The remaining samples were tested for population stratification with the principal components stratification method as implemented in EIGENSTRAT,<sup><a href="#bib39" rid="bib39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988832">39</a></sup> and population outliers were subsequently excluded. SNPs that had more than 5% missing data, a minor allele frequency less than 1% or deviated from Hardy-Weinberg equilibrium (exact p &lt; 10<sup>−4</sup> in controls) per sample set were excluded with the PLINK software version 1.07.<sup><a href="#bib40" rid="bib40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988782">40</a></sup> SNP imputation was carried out with the BEAGLE v.3.1.1<sup><a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988831">41</a></sup> software package and 690 HapMap3 reference haplotypes from the CEU, TSI, MEX, and GIH cohorts<sup><a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988818">42</a></sup> to predict missing autosomal genotypes in&nbsp;silico. We subsequently analyzed only those SNPs that could be imputed with moderate confidence (INFO score r<sup>2</sup>&nbsp;&gt; 0.3) and had a minor allele frequency more than 1% in cases or in controls. To take imputation uncertainty into account, phenotypic association was tested for allele dosage data separately for each of the five GWAS data sets in panels A and B through the use of PLINK's logistic regression framework for dosage data. To control potentially confounding effects due to population stratification, we adjusted for the top ten eigenvectors from EIGENSTRAT in the regression analysis. The genomic inflation factor λ is defined as the ratio of the medians of the&nbsp;sample χ<sup>2</sup> test statistics and the 1 degree of freedom χ<sup>2</sup> distribution (0.455).<sup><a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988780">43</a></sup> Because the estimated genomic inflation factor λ scales with sample size, it is informative to report the inflation factor for an equivalent study of 1,000 cases and 1,000 controls (λ<sub>1000</sub>) by rescaling λ.<sup><a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988774">44</a></sup></p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Meta-Analyses</h3><p id="__p11" class="p p-first-last">Meta-analyses were performed with PLINK's meta-analysis function and with its standard error of odds ratio weighting option (inverse variance weighting), which implicitly deals with imputation uncertainty. For the combined-phenotype analysis, we performed two sorts of meta-analysis in order to detect associations of SNPs with either the same or opposite allelic effects in the two diseases. For the same-effect analysis, the meta-analysis was carried out as usual. For the opposite-effect analysis, first we flipped minor and major alleles of each biallelic SNP in the CD data sets to mimic an opposite-direction effect of the allele in CD and performed a meta-analysis afterward. For both effects models, we considered only those SNPs whose genotypes were available from at least four out of the five GWAS data sets in panels A and B.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Follow-Up Genotyping</h3><p id="__p12" class="p p-first-last">Genotyping was carried out with our Sequenom iPlex platform&nbsp;from Sequenom and TaqMan technology from Applied Biosystems. Individuals with more than 3% missing data were removed. SNPs that had more than 3% missing data, a minor allele frequency less than 1% or deviated from Hardy-Weinberg equilibrium (exact p &lt; 10<sup>−4</sup> in controls) per sample set were excluded. p values for allele-based tests of phenotypic association for each single-replication sample sets (panels C–E) were calculated with PLINK. PLINK's meta-analysis function was used to obtain p&nbsp;values for the replication data set (p<sub>Repl</sub>) and for the combined discovery-replication data set (p<sub>GWAS+Repl</sub>).</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Regional Imputation Based on the 1000 Genomes Project Reference</h3><p id="__p13" class="p p-first-last">To enable imputation based on the 1000 Genomes project data,&nbsp;SNP positions referring to NCBI build 36 were mapped to build 37. SNP imputation was carried out with the BEAGLE software package v.3.1.1<sup><a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988796">41</a></sup> and 566 EUR (European) haplotypes generated by the 1000 Genomes Project.<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988830">45</a></sup> We analyzed only imputed SNPs with moderate imputation confidence (INFO score r<sup>2</sup> &gt; 0.3) and a minor allele frequency more than1% in cases or in controls.</p></div><div id="sec2.6" class="sec sec-last"><h3 id="sec2.6title">Gene Relationships across Implicated Loci Pathway Analysis</h3><p id="__p14" class="p p-first-last">The Gene Relationships Across Implicated Loci (GRAIL) software<sup><a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988821">46</a></sup> quantifies functional similarity between genes by applying established statistical text mining methods to the PubMed database of published scientific abstracts. As input we used the&nbsp;following list of SNPs: rs2201841, rs2082412, rs702873, rs12720356, rs10758669, rs694739, rs281379, rs181359, rs4780355, rs744166, and rs1250544. GRAIL was run with the following settings: HapMap release = HapMap release 22/hg18; HapMap population = CEU (Utah residents with ancestry from northern and western Europe from the Centre d′Étude du Polymorphisme Humain collection); functional data source = PubMed Text (April 2011); and gene size correction = on. GRAIL output results were visualized with VIZ-GRAIL.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Preparation of Single-Disease Meta-Analyses for Discovery Phase by Means of HapMap3 Imputation</h3><p id="__p15" class="p p-first">The overall study workflow for the combined analysis of CD and PS is displayed in <a href="/pmc/articles/PMC3322238/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>. For discovery, we conducted a meta-analysis on PS comprising 2,529 PS cases and 4,955 controls from three previously published GWASs,<sup><a href="#bib14" rid="bib14 bib15" class=" bibr popnode">14,15</a></sup> all of European descent (panel A in <a href="#app2" rid="app2" class=" sec">Table S1</a>). SNP data were combined with genotype imputation based&nbsp;on the HapMap3 reference. We subsequently used standard meta-analysis methodology (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). In total, 1,121,166 quality-controlled autosomal-imputed SNP markers were available for the analysis on PS. To control for potential population stratification, we adjusted association test statistics by means of principal component analysis (PCA) (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). A quantile-quantile (Q-Q) plot of the meta-analysis revealed a marked excess of significant associations in the tail of the&nbsp;distribution (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>A), which is primarily due to thousands of highly significant association signals from the HLA region. Genetic heterogeneity was low; there was an estimated genomic inflation factor of λ<sub>1000</sub> = 1.027<sup><a href="#bib43" rid="bib43 bib44" class=" bibr popnode">43,44</a></sup> (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). Results of the meta-analysis on PS are summarized in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>A.</p><p id="__p16" class="p p-last">In the same way, we performed a meta-analysis on CD by using 1,034,639 quality-controlled autosomal-imputed markers from a German GWAS<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988777">13</a></sup> and a previously published UK GWAS,<sup><a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988798">28</a></sup> consisting of 2,142 CD cases and 5,505 controls in total (panel B in <a href="#app2" rid="app2" class=" sec">Table S1</a>). Again, we observed low genomic inflation (λ<sub>1000</sub> = 1.032, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>B). Results of the meta-analysis on CD are summarized in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>B.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title"><em>OVERLAP</em> Approach: Cross-Disease Analysis of Established Risk SNPs</h3><p id="__p17" class="p p-first-last">So far, four established GWAS risk loci that are located outside the HLA region and shared between CD and PS have been reported in the literature, namely <em>IL23R</em>, <em>IL12B</em>, <em>REL</em>, and <em>TYK2</em> (<a href="/pmc/articles/PMC3322238/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). Although the markers that showed the strongest association differed between the two diseases at each of the first three loci, we found the same SNP rs12720356 at <em>TYK2</em> to be associated with both CD and PS. To seek confirmation of whether known risk loci for CD also play a role in the etiology of PS and vice versa, we checked whether markers that were significant (p &lt; 0.01) in our PS meta-analysis were among the 71 established risk SNPs previously implicated in CD<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988789">13</a></sup> and whether significant markers (p &lt; 0.01) from our CD meta-analysis were among the 25 established PS risk SNPs.<sup><a href="#bib14" rid="bib14 bib15 bib16 bib17 bib18 bib19 bib20" class=" bibr popnode">14–20</a></sup> The four already established shared risk SNPs (<a href="/pmc/articles/PMC3322238/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) were excluded. Given the well-established and heterogeneous allelic associations of the HLA region on chromosome 6 with both CD and PS, we also excluded all markers from the extended HLA region (chr6:25-34 Mb). Although none of the known PS SNPs were significantly associated with CD in our analysis, five out of the 71 known CD risk SNPs met our criterion of&nbsp;significance, namely rs10758669 (<em>JAK2</em> [MIM <a href="/omim/147796" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147796</a>]), rs694739 (<em>PRDX5</em> [MIM <a href="/omim/606583" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606583</a>]), rs281379 (<em>FUT2</em> [MIM <a href="/omim/182100" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">182100</a>]), rs744166 (<em>STAT3</em> [MIM <a href="/omim/102582" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">102582</a>]), and rs181359 (<em>YDJC;</em>HGNC 27158). We genotyped these five SNPs by using TaqMan technology in a large independent replication panel comprising 3,937 PS cases and 4,847 controls but also used summary statistics data of the five SNPs from an independent GWAS on PS<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343190025">16</a></sup> comprising 2,178 PS cases and 2,657 controls. The overall replication panel consisted of 6,115 PS cases and 7,504 controls (panel C, <a href="#app2" rid="app2" class=" sec">Table S1</a>). We performed single-marker association tests for panel C (p<sub>PS-Repl</sub>) and conducted a meta-analysis (p<sub>PS-GWAS+Repl</sub>) by combining association results from the&nbsp;GWAS (p<sub>PS-GWAS</sub>) and the replication (p<sub>PS-Repl</sub>) stages (<a href="/pmc/articles/PMC3322238/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table&nbsp;2</span></a>). All of the CD risk SNPs were also significantly associated with PS at the previously proposed level<sup><a href="#bib34" rid="bib34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988792">34</a></sup> of p<sub>PS-Repl</sub> &lt; 0.05 and p<sub>PS-GWAS+Repl</sub> &lt; 10<sup>−3</sup> (rs10758669 near <em>JAK2</em>, rs694739 near <em>PRDX5</em>, rs281379 near <em>FUT2</em>, and rs181359 at <em>YDJC</em>, rs744166 at&nbsp;<em>STAT3</em>). These five SNPs have already been reported to&nbsp;show significant association at the genome-wide level (p<sub>CD-GWAS+Repl</sub> &lt; 5&nbsp;× 10<sup>−8</sup>, p<sub>CD-Repl</sub> &lt; 0.05, and p<sub>CD-GWAS+Repl</sub> &lt; 10<sup>−3</sup>) in a very large, independent genome-wide meta-analysis on CD roughly three times the size of this one, that is comprising 6,333 CD cases and 15,056 controls.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988783">13</a></sup> All five SNPs achieved genome-wide significance in the combined analysis of PS discovery panel A, PS replication panel C, and CD discovery data from Franke et&nbsp;al.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988776">13</a></sup> (p<sub>CDPS-GWAS+Repl</sub> &lt; 5&nbsp;× 10<sup>−8</sup>). Furthermore, SNP rs694739, 7.9 kb downstream of <em>PRDX5</em>, as well as SNP rs181359, 53.7 kb downstream of&nbsp;<em>YDJC</em>, reached genome-wide significance for PS only (p<sub>rs694739</sub> = 3.71&nbsp;× 10<sup>−09</sup> and p<sub>rs181359</sub> = 8.02&nbsp;× 10<sup>−10</sup>). We also observed a highly significant association at the <em>FUT2</em> locus (p<sub>rs281379</sub> = 7.86&nbsp;× 10<sup>−08</sup>) for PS&nbsp;only.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p46">Established Risk Loci Shared between CD and PS from Published Studies on CD and PS, Respectively</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408031691152" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Locus</strong></th><th rowspan="1" colspan="1"><strong>Genes of Interest</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Top CD dbSNP ID</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>Risk Allele</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>Top PS dbSNP ID</strong><sup>b</sup></th><th rowspan="1" colspan="1"><strong>Risk Allele</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>LD between CD and PS SNPs (D′/r<sup>2</sup>)</strong></th><th rowspan="1" colspan="1"><strong>Comment</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1p31</td><td rowspan="1" colspan="1"><em>IL23R</em></td><td rowspan="1" colspan="1">rs11209026<sup>c</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">2.66</td><td rowspan="1" colspan="1">rs2201841<sup>f</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1">1.0/0.018</td><td rowspan="1" colspan="1">different markers</td></tr><tr><td rowspan="1" colspan="1">5q33</td><td rowspan="1" colspan="1"><em>IL12B</em></td><td rowspan="1" colspan="1">rs6556412<sup>d</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1">rs2082412<sup>f</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">1.44</td><td rowspan="1" colspan="1">0.715/0.269</td><td rowspan="1" colspan="1">different markers</td></tr><tr><td rowspan="1" colspan="1">2p16</td><td rowspan="1" colspan="1"><em>REL</em></td><td rowspan="1" colspan="1">rs10181042<sup>e</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1">rs702873<sup>g</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1">0.036/0.001</td><td rowspan="1" colspan="1">different markers</td></tr><tr><td rowspan="1" colspan="1">19p13</td><td rowspan="1" colspan="1"><em>TYK2</em></td><td rowspan="1" colspan="1">rs12720356<sup>e</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1">rs12720356<sup>g</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1" align="center">1.40</td><td rowspan="1" colspan="1">same marker</td><td rowspan="1" colspan="1">same marker, opposite direction of effect</td></tr></tbody></table></div><div id="largeobj_idm140408031691152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3322238/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>Candidate genes of interest are listed for the locus.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Lead SNP with most significant association within a locus, as stated in the reference publication.</div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>c</sup>See Franke et&nbsp;al.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988825">13</a></sup> and Duerr et&nbsp;al.<sup><a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481391834">21</a></sup></div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>d</sup>See Barrett et&nbsp;al.<sup><a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988823">12</a></sup> and Franke et&nbsp;al.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988814">13</a></sup></div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>e</sup>See Franke et&nbsp;al.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988827">13</a></sup></div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>f</sup>See Cargill et&nbsp;al.<sup><a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988787">22</a></sup> and Nair et&nbsp;al.<sup><a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988820">23</a></sup></div></div><div class="tblwrap-foot"><div id="tblfn7"><sup>g</sup>See Strange et&nbsp;al.<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343190023">16</a></sup></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p54">Association Results of <em>OVERLAP</em> Approach from Cross-Disease Comparison of Established Risk Markers</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408042161696" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Chr</strong></th><th rowspan="2" colspan="1"><strong>SNP</strong></th><th rowspan="2" colspan="1"><strong>A1</strong></th><th rowspan="2" colspan="1"><strong>Locus</strong></th><th colspan="2" rowspan="1"><strong>CD GWAS Meta-Analysis</strong><sup>a</sup><strong>(6,333/15,056)</strong><hr></th><th colspan="2" rowspan="1"><strong>PS GWAS (2,529/4,955)</strong><hr></th><th colspan="2" rowspan="1"><strong>PS Replication (6,115/7,504)</strong><hr></th><th colspan="2" rowspan="1"><strong>PS GWAS and Repl (8,644/12,459)</strong><hr></th><th colspan="2" rowspan="1"><strong>CD GWAS Meta-Analysis</strong><sup>a</sup><strong>+ PS GWAS and Repl (14,977/27,515)</strong><hr></th><th rowspan="2" colspan="1"><strong>Status Now</strong><sup>b</sup></th></tr><tr><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs10758669</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>JAK2</em></td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">2.43&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">2.47&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">2.69&nbsp;× 10<sup>−05</sup></td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">1.30&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs694739</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>PRDX5</em></td><td rowspan="1" colspan="1" align="center">3.4&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">1.13&nbsp;× 10<sup>−04</sup></td><td rowspan="1" colspan="1" align="center">0.86</td><td rowspan="1" colspan="1" align="center">6.12&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1" align="center">3.71&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">2.41&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">0.89</td><td rowspan="1" colspan="1">PS, CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs281379</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>FUT2</em></td><td rowspan="1" colspan="1" align="center">8.6&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">3.22&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">7.12&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">7.86&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">1.32&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs181359</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>YDJC</em></td><td rowspan="1" colspan="1" align="center">6.3&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.83</td><td rowspan="1" colspan="1" align="center">4.83&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">3.54&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">8.02&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">1.33&nbsp;× 10<sup>−21</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1">PS, CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs744166<sup>c</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>STAT3</em></td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">2.44&nbsp;× 10<sup>−04</sup></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">1.49&nbsp;× 10<sup>−02</sup></td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">5.30&nbsp;× 10<sup>−05</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">5.48&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.90</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr></tbody></table></div><div id="largeobj_idm140408042161696" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3322238/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140408032965600"><p id="__p55" class="p p-first-last">The following abbreviations are used: Chr, chromosome of marker; SNP, rs ID; A1, minor allele; Locus, one candidate gene in the region; p/OR, p value and corresponding odds ratio with respect to minor allele for the large GWAS meta-analysis of CD,<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988771">13</a></sup> GWAS meta-analysis of PS (panel A), PS replication analysis (panel C), combined analysis of PS GWAS meta-analysis (panel A) and PS replication (panel C), and combined analysis of CD GWAS meta-analysis<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988815">13</a></sup> and panels A and C. For each panel, numbers of cases/controls are displayed in parentheses.</p></div></div><div class="tblwrap-foot"><div id="tblfn8"><sup>a</sup>See Franke et&nbsp;al.<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988812">13</a></sup></div></div><div class="tblwrap-foot"><div id="tblfn9"><sup>b</sup>Status now: new status of association with CD and/or PS. All SNPs are established CD risk SNPs with p &lt; 5&nbsp;× 10<sup>−8 13</sup> that were significant (p &lt; 0.01) in our PS meta-analysis. All SNPs, except for rs744166, showed the same direction of effect for CD and PS. None of the SNPs showed an exact Hardy-Weinberg p value &lt; 0.01 in the PS replication (panel C).</div></div><div class="tblwrap-foot"><div id="tblfn10"><sup>c</sup>Minor and major alleles of rs744166 were flipped in the CD GWAS meta-analysis in order to calculate the combined-phenotype p value and odds ratio.</div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title"><em>COMBINED</em> Approach, Part 1: Meta-Analysis Considering Same-Direction Effects</h3><p id="__p18" class="p p-first-last">In order to identify additional shared genetic susceptibility loci in CD and PS, we performed a meta-analysis of the combined phenotype where CD and PS were considered as a single phenotype. The disease-specific meta-analyses (panels A and B) were merged to form a combined-phenotype meta-analysis discovery panel comprising 2,142 CD cases, 2,529 PS cases, and 10,460 healthy controls. In total, 1,123,777 quality-controlled autosomal markers were available for the analysis in at least four out of the five GWAS data sets. As with the <em>OVERLAP</em> approach, we excluded all markers from the extended HLA region (chr6:25-34Mb), leaving 1,116,213 autosomal SNPs for screening of shared risk loci. We observed only low genomic inflation for the same-direction meta-analysis (λ<sub>1000</sub> = 1.023; <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>A). After exclusion of established loci for PS and CD, the inflation factors further decreased (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>C). To provide proof of principle for our approach, we first examined association signals at the three established shared risk loci with same-direction effects of alleles for CD and PS (see <a href="/pmc/articles/PMC3322238/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). We observed highly significant association signals for all three loci (p<em><sub>IL23R</sub></em> = 1.82&nbsp;× 10<sup>−22</sup>, p<em><sub>IL12B</sub></em> = 3.32&nbsp;× 10<sup>−7</sup>, and p<em><sub>REL</sub></em> = 1.53&nbsp;× 10<sup>−7</sup>; <a href="#app2" rid="app2" class=" sec">Figures S4</a>A–S4C). Subsequently, we selected SNPs for being nominally associated in each of the single-disease meta-analyses (p<sub>CD-GWAS</sub> &lt; 0.05, p<sub>PS-GWAS</sub> &lt; 0.05) and for being significantly associated in the combined-phenotype association analysis at the 10<sup>−4</sup> level (p<sub>CDPS-GWAS</sub> &lt; 1&nbsp;× 10<sup>−4</sup>). This resulted in 17 SNPs located at 17 distinct loci. Except for the four known shared loci (see&nbsp;<a href="/pmc/articles/PMC3322238/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>), we did not exclude established risk loci from either CD or PS to maintain the chance of detecting shared risk alleles at these loci. Because one of the 17 SNPs is located at 10q22 (<em>ZMIZ1</em> [MIM <a href="/omim/607159" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607159</a>]), which is an established CD risk locus, we added the established CD-associated SNP rs1250550 from this region to the list of follow-up SNPs. We then genotyped these 18 SNPs in an independent panel of 1,713 CD cases, 1,009 PS cases and 3,565 controls (panel D, <a href="#app2" rid="app2" class=" sec">Table S1</a>D) by using the Sequenom iPlex platform. Association results for all 18 SNPs are shown in <a href="#app2" rid="app2" class=" sec">Table S2</a>. The strongest association was observed at <em>ZMIZ1</em> for SNP rs1250544 (p<sub>CDPS-GWAS</sub> = 1.12&nbsp;× 10<sup>−5</sup> and p<sub>CDPS-GWAS+Repl</sub> = 2.66&nbsp;× 10<sup>−10</sup>; see <a href="#app2" rid="app2" class=" sec">Table&nbsp;S2</a> and <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>A) and yielded genome-wide significance&nbsp;in the same-effect combined-phenotype analysis of discovery panels A and B and replication panel D. SNP rs1250544 reached also genome-wide significance for PS alone (p<sub>PS-GWAS+Repl</sub> = 3.90&nbsp;× 10<sup>−8</sup>). A robust association with CD, but not with PS, was observed at <em>SOCS1</em> (MIM <a href="/omim/603597" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603597</a>) on chromosomal region 16p13 (p<sub>CDPS-GWAS</sub> = 9.36&nbsp;× 10<sup>−7</sup>, p<sub>CD-GWAS</sub> = 1.47&nbsp;× 10<sup>−3</sup>, and p<sub>CD-GWAS+Repl</sub> = 1.01&nbsp;× 10<sup>−7</sup> for rs4780355; see&nbsp;<a href="#app2" rid="app2" class=" sec">Table S2</a> and <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S5</a>B). We further corroborated both association signals by genotyping SNPs rs1250544 and rs4780355 in additional sample sets from Germany and Italy, comprising 2,360 CD cases and 1,015 healthy controls, but we also used&nbsp;summary statistics data of the two SNPs from the independent GWAS on PS<sup><a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_343190024">16</a></sup> comprising 2,178 PS cases and 2,657 controls (panel E in <a href="#app2" rid="app2" class=" sec">Table S1</a>, the same cases and controls from the United Kingdom, as described in panel C). In the combined analysis of discovery panels A and&nbsp;B&nbsp;and replication panels D and E (<a href="/pmc/articles/PMC3322238/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Tables 3</span></a> and <a href="/pmc/articles/PMC3322238/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4),</span></span><span>4</span></a>), SNP rs4780355 achieved genome-wide significance (p<sub>CDPS-GWAS+Repl</sub> = 1.37&nbsp;× 10<sup>−13</sup>) but also attained genome-wide significance for CD alone (p<sub>CD-GWAS+Repl</sub> = 4.99&nbsp;× 10<sup>−8</sup>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p59">Association Results of Combined-Phenotype Meta-Analysis Considering Same-Direction Effects of Alleles from <em>COMBINED</em> Approach</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408076407280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Chr</strong></th><th rowspan="2" colspan="1"><strong>SNP</strong></th><th rowspan="2" colspan="1"><strong>A1</strong></th><th rowspan="2" colspan="1"><strong>Locus</strong></th><th colspan="2" rowspan="1"><strong>CD+PS Discovery GWAS (4,671/10,460)</strong><hr></th><th colspan="2" rowspan="1"><strong>PS GWAS (2,529/4,955)</strong><hr></th><th colspan="2" rowspan="1"><strong>CD GWAS (2,142/5,505)</strong><hr></th><th colspan="2" rowspan="1"><strong>PS Replication (3,187/4,759)</strong><hr></th><th colspan="2" rowspan="1"><strong>CD Replication (4,073/2,478)</strong><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250544<sup>a</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">1.12&nbsp;× 10<sup>−05</sup></td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">3.85&nbsp;× 10<sup>−05</sup></td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">3.31&nbsp;× 10<sup>−02</sup></td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">1.94&nbsp;× 10<sup>−04</sup></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">5.28&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">1.22</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250560<sup>b</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">3.13&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">4.31&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">4.87&nbsp;× 10<sup>−02</sup></td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">1.06&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">8.06&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250559<sup>b</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">1.53&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">4.58&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">0.82</td><td rowspan="1" colspan="1" align="center">3.46&nbsp;× 10<sup>−02</sup></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">1.16&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">4.10&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.79</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs4780355<sup>a</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>SOCS1</em></td><td rowspan="1" colspan="1" align="center">9.36&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">1.72&nbsp;× 10<sup>−04</sup></td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">1.47&nbsp;× 10<sup>−03</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">7.53&nbsp;× 10<sup>−04</sup></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">7.40&nbsp;× 10<sup>−06</sup></td><td rowspan="1" colspan="1" align="center">1.19</td></tr></tbody></table></div><div id="largeobj_idm140408076407280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3322238/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140408033621136"><p id="__p60" class="p p-first-last">The following abbreviations are used: Chr: chromosome of marker; SNP: rs ID; A1: minor allele; Locus: one candidate gene in the region; p/OR: p value and corresponding odds ratio with respect to minor allele for the combined-phenotype GWAS meta-analysis of CD and PS (panels A and B), GWAS meta-analysis of PS (panel A), GWAS meta-analysis of CD (panel B), PS replication analysis (as part of panel D), CD replication analysis (part of panel D, panel E). For each panel, numbers of cases/controls are displayed in parentheses. None of the SNPs showed an exact Hardy-Weinberg p&nbsp;value &lt; 0.01 in the PS and CD replication panels (panels D and E).</p></div></div><div class="tblwrap-foot"><div id="tblfn11"><sup>a</sup>SNPs were identified via genotype imputation based on the HapMap3 reference and P/OR are given according to that analysis.</div></div><div class="tblwrap-foot"><div id="tblfn12"><sup>b</sup>SNPs were identified via genotype imputation based on the 1000 Genomes reference and P/OR are given according to that analysis.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p63">Association Results of Combined-Phenotype Meta-Analysis Considering Same-Direction Effects of Alleles from <em>COMBINED</em> Approach</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408033613936" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Chr</strong></th><th rowspan="2" colspan="1"><strong>SNP</strong></th><th rowspan="2" colspan="1"><strong>A1</strong></th><th rowspan="2" colspan="1"><strong>Locus</strong></th><th colspan="2" rowspan="1"><strong>PS GWAS and Repl (5,716/9,714)</strong><hr></th><th colspan="2" rowspan="1"><strong>CD GWAS and Repl (6,215/7,983)</strong><hr></th><th colspan="2" rowspan="1"><strong>CD+PS GWAS and Repl (11,931/17,697)</strong><hr></th><th rowspan="2" colspan="1"><strong>Status Now</strong></th></tr><tr><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250544<sup>a</sup></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">3.53&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">2.56&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">7.32&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1">PS, CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250560<sup>b</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">3.03&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">4.10&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">7.34&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs1250559<sup>b</sup></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>ZMIZ1</em></td><td rowspan="1" colspan="1" align="center">3.63&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">1.24&nbsp;× 10<sup>−09</sup></td><td rowspan="1" colspan="1" align="center">0.84</td><td rowspan="1" colspan="1" align="center">2.78&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr><tr><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">rs4780355<sup>a</sup></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>SOCS1</em></td><td rowspan="1" colspan="1" align="center">5.30&nbsp;× 10<sup>−07</sup></td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">4.99&nbsp;× 10<sup>−08</sup></td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1" align="center">1.37&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1">CD-PS, CD</td></tr></tbody></table></div><div id="largeobj_idm140408033613936" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3322238/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140408042024384"><p id="__p64" class="p p-first-last">For abbreviations used, see <a href="/pmc/articles/PMC3322238/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>. Combined analysis of PS GWAS meta-analysis (panel A) and PS replication (part of panel D), combined analysis of CD GWAS meta-analysis (panel B) and CD replication (part of panel D, panel E), combined analysis of CD+PS GWAS meta-analysis (panels A and B) and CD+PS replication (panels D and E).</p></div></div><div class="tblwrap-foot"><div id="tblfn13"><sup>a</sup>SNPs were identified via genotype imputation based on the HapMap3 reference and p/OR are given according to that analysis.</div></div><div class="tblwrap-foot"><div id="tblfn14"><sup>b</sup>SNPs were identified via genotype imputation based on the 1000 Genomes reference and p/OR are given according to that analysis.</div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title"><em>COMBINED</em> Approach, Part 2: Meta-Analysis Assuming Opposite-Direction Effects</h3><p id="__p19" class="p p-first-last">An allele might confer a risk for CD while protecting against PS and vice versa, as is the case for <em>TYK2</em>. Therefore, we also screened our combined-phenotype meta-analysis data (panels A and B) while coding alleles in such a way as to consider the opposite effects of them in the two diseases (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). We observed low genomic inflation for the opposite-direction meta-analysis (λ<sub>1000</sub> = 1.009, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>B). After excluding established shared loci for PS and CD, the inflation factors further decreased (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>D). In a first step, we checked the known risk SNP rs12720356 (<em>TYK2</em>; see <a href="/pmc/articles/PMC3322238/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>) for opposite direction of effects. SNP rs12720356 had a p value of 4.09&nbsp;× 10<sup>−5</sup> in the combined analysis of panels A and B (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>D); there was an odds ratio (OR) of 1.29 (95% confidence interval [CI] [1.10,1.51]) for allele A in panel A (p<sub>PS-GWAS</sub>&nbsp;= 1.39&nbsp;× 10<sup>−3</sup>) and of 0.78 (95% CI [0.65,0.94]) in panel B (p<sub>CD-GWAS</sub> = 1.01&nbsp;× 10<sup>−2</sup>). We then selected three SNPs for subsequent genotyping (with Sequenom) and testing in replication panel D (see&nbsp;<a href="#app2" rid="app2" class=" sec">Table S1</a>D). The selection criteria were the same as for the&nbsp;same-direction effect meta-analysis. However, none&nbsp;of the three SNPs replicated in both diseases at p&nbsp;value &lt;&nbsp;0.05 (see <a href="#app2" rid="app2" class=" sec">Table S3</a>).</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">In Silico Fine-Mapping: Refinement of Association Signals of <em>COMBINED</em> Approach</h3><p id="__p20" class="p p-first">For refinement of the association signals at <em>ZMIZ1</em> and <em>SOCS1</em>, we imputed a region of about ±1 Mb around the strongest signals from the discovery panels A and B (see <a href="#app2" rid="app2" class=" sec">Table S1</a>) by using the EUR reference from the 1000 Genomes Project<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988775">45</a></sup> (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). In silico fine-mapping of the region around <em>SOCS1</em> via standard meta-analysis methodology (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>) confirmed rs4780355 to be highly significant in this region (p<sub>GWAS</sub> = 4.04&nbsp;× 10<sup>−7</sup>). Additionally, another SNP, rs2021511, which is located in the same intron as rs4780355 (2.9 kb downstream of rs4780355) showed the&nbsp;same magnitude of association (p<sub>GWAS</sub> = 1.58&nbsp;× 10<sup>−7</sup>; <a href="/pmc/articles/PMC3322238/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>) but was not selected for further replication because of the high linkage disequilibrium (LD) between SNPs rs4780355 and rs2021511 (r<sup>2</sup> = 0.934) according to the 1000 Genomes Project EUR reference. Screening of the imputed region of <em>SOCS1</em> for coding SNPs revealed one missense SNP with p &lt; 10<sup>−4</sup> within the <em>TNP2</em> gene, namely rs11640138. We genotyped this SNP in replication panel D, but it did not replicate in either disease at p&nbsp;value&nbsp;&lt; 0.05.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3322238/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408031701552" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC3322238/bin/gr2.jpg"></div></a><div id="largeobj_idm140408031701552" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3322238/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC3322238/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3322238/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p42">Regional Association Plots of In Silico Fine-Mapping for Newly Detected Shared Risk Loci from <em>COMBINED</em> Approach</p><p id="__p43">Shared risk loci for CD and PS at (A) 10q22 (<em>ZMIZ1</em>) and (B) 16p13 (near <em>SOCS1</em>). eQTL analyses revealed a potential effect of the associations at <em>ZMIZ1</em> and near <em>SOCS1</em> on gene expression. p values (−log<sub>10</sub><em>p</em>) are depicted with regard to the physical location of markers and are based on imputed genotypes. SNP genotypes were imputed with the EUR reference from 1000 Genomes Project<sup><a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988778">45</a></sup> (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>). The following abbreviations are used: blue-filled circle, lead SNP of the combined-phenotype data (panels A and B); other filled circles, analyzed SNPs of the combined-phenotype data (panels A and B) where the fill color corresponds to the strength of linkage disequilibrium (r<sup>2</sup>) with the lead SNP (for color coding see legend in the upper right corner of each plot); green triangles, analyzed SNPs of the meta-analysis on PS (panel A); gray squares, analyzed SNPs of the meta-analysis on CD (panel&nbsp;B); and blue line, recombination intensity (cM/Mb). Positions and gene annotations are according to NCBI's build 37 (hg19).</p></div></div></div><p id="__p21" class="p p-last">In silico fine-mapping of <em>ZMIZ1</em> with the same methodology narrowed down the association signal to two coding SNPs, namely rs1250559 (p<sub>CDPS-GWAS</sub> = 1.53&nbsp;× 10<sup>−7</sup>, <a href="/pmc/articles/PMC3322238/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>) and rs1250560 (p<sub>CDPS-GWAS</sub> = 3.13&nbsp;× 10<sup>−6</sup>). According to the 1000 Genomes Project EUR reference, both SNPs are in near perfect LD (r<sup>2</sup> = 0.948). Depending on different splice variants of <em>ZMIZ1</em>, rs1250559 is either intronic or located in the 3-untranslated region (3-UTR), whereas rs1250560 is either an intronic SNP or a missense SNP located in exon 5. The intronic <em>ZMIZ1</em> SNP rs1250544, which yielded the strongest signal from the initial same-effect combined-phenotype meta-analysis, and the missense SNP rs1250560 are 20.6 kb apart and in moderate LD (r<sup>2</sup> = 0.682). In order to substantiate our findings from the in&nbsp;silico analyses, we genotyped both <em>ZMIZ1</em> SNPs in replication panels D and E (see <a href="#app2" rid="app2" class=" sec">Table S1</a>). As shown in <a href="/pmc/articles/PMC3322238/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Tables 3</span></a> and <a href="/pmc/articles/PMC3322238/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4,</span></span><span>4</span></a>, both SNPs were associated with PS at the 0.05 level and even showed genome-wide significance&nbsp;with CD (p<sub>CD-Repl</sub> = 8.06&nbsp;× 10<sup>−10</sup> at rs1250560, p<sub>CD-Repl</sub> = 4.10&nbsp;× 10<sup>−10</sup> at rs1250559). Interestingly, the association signals of these two SNPs were much stronger in the initial analysis of PS panel A than of CD panel B. The combined analysis of discovery panels A and B and replication panels D and E yielded genome-wide significance for rs1250560 (p<sub>CDPS-GWAS+Repl</sub> = 7.34&nbsp;× 10<sup>−16</sup>) and rs1250559 (p<sub>CDPS-GWAS+Repl</sub> = 2.78&nbsp;× 10<sup>−16</sup>), both of which are of higher significance than was observed for rs1250544 (p<sub>CDPS-GWAS+Repl</sub> = 7.32&nbsp;× 10<sup>−14</sup>) (<a href="/pmc/articles/PMC3322238/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Tables 3</span></a> and <a href="/pmc/articles/PMC3322238/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4</span></span><span>4</span></a>).</p></div><div id="sec3.6" class="sec sec-last"><h3 id="sec3.6title">Effect on Gene Expression</h3><p id="__p22" class="p p-first-last">We subsequently assessed a potential functional effect of the four SNPs showing association for both CD and PS with the same direction of effects, namely rs1250544, rs1250559, rs1250560 (<em>ZMIZ1</em>), and rs4780355 (near <em>SOCS1</em>). To this end, we investigated the correlation of SNP genotypes with gene expression levels by means of in&nbsp;silico expression quantitative trait locus (eQTL) analysis by using the mRNAbySNPBrowser software.<sup><a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988813">47</a></sup> This program utilizes genotype data from 408,273 SNPs and gene&nbsp;expression data from Epstein-Barr-virus-transformed lymphoblastoid cell lines that were collected from 400&nbsp;children and measured with the Affymetrix HG-U133 Plus 2.0 chip. Significant evidence (uncorrected p<sub>Expression</sub>&nbsp;&lt; 10<sup>−4</sup>) for causing differential expression of <em>ZMIZ1</em> was observed for SNP rs1250546 (p<sub>rs1250546</sub> = 8.10&nbsp;× 10<sup>−5</sup>), which is in high LD with our lead SNP rs1250544 (r<sup>2</sup>&nbsp;= 0.829). Also, we found an even stronger evidence for association between expression of <em>C16ORF75</em> (MIM <a href="/omim/612426" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612426</a>), which is located 90 kb upstream of <em>SOCS1</em>, and SNP rs243323 (p<sub>rs243323</sub> = 1.10&nbsp;× 10<sup>−8</sup>). This SNP is also in high LD with our lead SNP rs4780355 (r<sup>2</sup> = 0.931). Both proxy SNPs rs1250546 and rs243323 were also significantly associated in our same-effect combined-phenotype analysis of discovery panels A and B (p<sub>GWAS</sub> = 7.15&nbsp;× 10<sup>−5</sup> for rs1250546, p<sub>GWAS</sub> = 1.80&nbsp;× 10<sup>−6</sup> for rs243323). This in&nbsp;silico eQTL analysis supports the notion that our four reported SNPs might affect the expression of <em>ZMIZ1</em> and <em>C16ORF75</em>. The full list of significant associations between SNP genotypes and gene expression levels is shown in <a href="#app2" rid="app2" class=" sec">Table S4</a>.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p23" class="p p-first">In a large combined sample set of 6,215 CD cases, 8,644 PS cases and 20,560 healthy controls, we have identified seven non-HLA susceptibility loci shared between CD and PS (9p24 near <em>JAK2,</em> 10q22 at <em>ZMIZ1,</em> 11q13 near <em>PRDX5,</em> 16p13 near <em>SOCS1</em>, 19p13 near <em>FUT2,</em> 17q21 at <em>STAT3</em>, 22q11 at <em>YDJC</em>). These loci, except for <em>SOCS1</em>, were already known to play&nbsp;a role in CD etiology, but were of unknown significance for PS<sup><a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988803">13</a></sup> (see also <a href="#app2" rid="app2" class=" sec">Table S5</a> for associations with other diseases). Notably, three of these loci showed genome-wide significance when tested for association with PS alone (10q22 at <em>ZMIZ1</em>, 11q13 near <em>PRDX5</em>, and 22q11 at <em>YDJC</em>). Furthermore, we revealed a risk locus for CD (16p13 near <em>SOCS1</em>). The identified shared risk loci point to functionally very interesting genes that might play a role in the pathogenesis of both CD and PS. The gene <em>ZMIZ1</em> (also known as <em>hZIMP10</em> or <em>TRAFIP10</em>) encodes for the protein zinc finger MIZ type 1, which is a member of the protein inhibitor of activated STAT (PIAS) family. The protein regulates the activity of several transcription factors such as the androgen receptor, Smad3/4, and p53; regulates TGF-β/SMAD signaling; and is induced by retinoic acid.<sup><a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988770">48</a></sup>
<em>FUT2</em> encodes α-(1,2)fucosyltransferase (FUT2), a physiological trait that regulates expression of the Lewis human blood group of antigens on the surface of epithelial cells and in body fluids. Genetic variants in <em>FUT2</em> have been implicated in susceptibility to infections with Norovirus<sup><a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988826">49</a></sup> and <em>Helicobacter pylori</em>.<sup><a href="#bib50" rid="bib50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988806">50</a></sup>
<em>PRDX5</em> encodes Peroxiredoxin-5, which belongs to the peroxiredoxin family of antioxidant enzymes that reduce hydrogen peroxide and alkyl hydroperoxides and might play&nbsp;a protective role during inflammatory processes. <em>SOCS1</em> encodes the suppressor of cytokine signaling 1 (SOCS1), a protein that is member of the STAT-induced STAT inhibitor (SSI), also known as suppressor of cytokine signaling (SOCS) family. SOCS1 is&nbsp;a cytokine-inducible negative regulator of cytokine signaling.<sup><a href="#bib51" rid="bib51 bib52" class=" bibr popnode">51,52</a></sup> Cytokines such as IL2, IL3, erythropoietin, and interferon-gamma can induce expression of <em>SOCS1</em>.<sup><a href="#bib53" rid="bib53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988785">53</a></sup> Moreover, a potential functional effect of the associations at <em>ZMIZ1</em> and near <em>SOCS1</em> on gene expression was found by an in&nbsp;silico eQTL analysis.</p><p id="__p24">The present study has increased the number of known shared CD and PS susceptibility loci to eleven (<em>IL12B</em>, <em>IL23R</em>, <em>REL</em>, <em>TYK2</em>, <em>JAK2</em>, <em>ZMIZ1</em>, <em>PRDX5</em>, <em>SOCS1</em>, <em>STAT3</em>, <em>FUT2</em>, and <em>YDJC</em>). To quantify the degree of relatedness between genes within the eleven loci, we used a published statistical genomics method, namely GRAIL (gene relationships across implicated loci),<sup><a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988791">46</a></sup> that applies statistical text mining to PubMed abstracts (see <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a> and <a href="/pmc/articles/PMC3322238/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>). GRAIL highlights a number of nonrandom and evidence-based connections between the genes within the nine loci that might indicate overlap in the pathways acting in the etiology of CD and PS. Multiple genes (<em>IL12B</em>, <em>IL23R</em>, <em>TYK2</em>, <em>JAK2</em>, <em>SOCS1</em>, and <em>STAT3</em>) are involved in IL23/Th17 signaling and play a critical role in the principal signaling mechanism for a wide array of cytokines and growth factors. It is noteworthy that genes <em>CDC37</em> ([MIM <a href="/omim/605065" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605065</a>] 21.7 kb downstream of the established shared risk locus at <em>TYK2</em>) and <em>STIP1</em> ([MIM <a href="/omim/605063" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605063</a>] 113.5 kb upstream of the identified shared risk at <em>PRDX5</em>) were found by GRAIL to be significantly connected. <em>CDC37</em> and <em>STIP1</em> encode CDC37 and STI1, respectively, two of several auxiliary proteins that associate with the heat-shock protein 90 (HSP90) molecular chaperone and thus are collectively referred to as HSP90 cochaperones.<sup><a href="#bib54" rid="bib54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988781">54</a></sup> HSP90 itself is an abundant, evolutionarily conserved molecular chaperone that acts mainly as a cofactor for the folding of polyproteins into functional, stable, mature proteins, and it physically associates with JAK1 and probably JAK2,<sup><a href="#bib55" rid="bib55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988810">55</a></sup> demonstrating that JAK1/2 are client&nbsp;proteins of HSP90. A study in mice and in patient samples suggested that HSP90 inhibitors might help treat JAK2-dependent myeloproliferative neoplasms (MPNs).<sup><a href="#bib56" rid="bib56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988768">56</a></sup> Moreover, inhibition of HSP90 was found to block Nod2-mediated activation of the transcription factor NF-κB and reduce NALP3-mediated gout-like inflammation in mice,<sup><a href="#bib57" rid="bib57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988786">57</a></sup> and mutations in the gene encoding NALP3,&nbsp;a member of the Nod-like receptor (NLR) protein family, are associated with several autoinflammatory disorders.<sup><a href="#bib58" rid="bib58 bib59" class=" bibr popnode">58,59</a></sup> Our hypothesis that <em>CDC37</em> and <em>STIP1</em> are potential joint risk factors for CD and PS is substantiated by an association peak within <em>CDC37</em> in our same-effect combined-phenotype analysis of discovery panels A and B (p<sub>GWAS</sub> = 1.61&nbsp;× 10<sup>−3</sup> for rs11879191, <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S6</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC3322238/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408031694160" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC3322238/bin/gr3.jpg"></div></a><div id="largeobj_idm140408031694160" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3322238/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC3322238/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC3322238/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p44">Gene Relationships across the 11 Shared Risk Loci of CD and PS Identified by GRAIL Analysis</p><p id="__p45">GRAIL<sup><a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988817">46</a></sup> is a statistical text-mining approach to quantify the degree of relatedness among genes in genomic disease regions. It estimates the statistical significance of the number of observed relationships with a null model in which relationships between the genes occur by random chance. A significance score p<sub>text</sub>, which is adjusted for multiple hypothesis testing, represents the output GRAIL score. p<sub>text</sub> values approximately estimate type-I error rates. Outer circle: lead SNPs from shared risk loci of both diseases; each box represents a SNP. Inner circle: genes of the genomic regions around lead SNPs that were identified based on LD properties; each box represents a gene; genes that were scored at p<sub>text</sub> &lt; 0.05 are significantly linked to genes in the other disease regions and are indicated in bold type. Lines: the lines between genes represent significant connections, with the thickness and redness of the lines being inversely proportional to the probability that a text-based connection would be seen by chance.</p></div></div></div><p id="__p25">It is worth noting that we used a two tier strategy to identify shared disease risk loci: Both approaches turned out to be effective and complementary tools for gaining insights into the postulated shared pathogenesis of CD and PS. Application of only a single strategy would have decreased the number of identified loci. Although the <em>OVERLAP</em> approach represents a simple and cost-effective strategy (cross-disease comparison of known risk SNPs), the <em>COMBINED</em> approach provides the power to identify shared susceptibility loci even if association signals are heterogeneous between diseases, that is the particular SNP showing the smallest p value at the considered locus, as was the case, for example, for the identified risk locus at <em>ZMIZ1</em>. This heterogeneity of most strongly associated SNPs could be due to interactions with other genetic variants or environmental factors, to differences in the distribution or effect size of causal alleles, or to the fact the identified SNPs show an association signal only because they are in LD with the actual causal variant. In particular, the increase of power due to increased sample sizes makes the <em>COMBINED</em> approach a potentially powerful tool to detect shared risk loci that might be missed in disease-specific GWASs that are often underpowered because of their comparatively smaller sample sizes.</p><p id="__p26">Because we did not search for loci harboring association signals with different and independent SNPs in terms of LD associated with CD and PS, there is room for improvement. For instance, in a simple rank approach with regard to single-marker association p values, different disease-associated markers for the same locus could be determined when they rank high with regard to their p value in association scans of CD and PS, respectively. This would allow detecting shared susceptibility loci even if association signals are heterogeneous between diseases. An approach to meet the challenge of the heterogeneity of genetic effects of the same markers between different diseases was proposed by Morris et&nbsp;al.<sup><a href="#bib60" rid="bib60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988804">60</a></sup> The authors developed a test of association within a multinomial regression framework and demonstrated the improved power of their multinomial regression-based analysis over existing methods.</p><p id="__p27" class="p p-last">It is likely that future studies will identify additional shared disease loci for CD and PS by further increasing the sample size of analyzed case-control panels or, for example, by applying the suggested rank approach. Evidence for a shared etiological basis among several autoimmune and inflammatory diseases is growing. For Crohn disease, for example, Lees and colleagues currently reported that 51 of the known 71 loci overlap with more than 23 distinct diseases, comprising also several nonautoimmune conditions.<sup><a href="#bib61" rid="bib61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_256988773">61</a></sup> Given the success of this study, we expect the same for the investigation of further combinations of such diseases for shared risk factors.</p></div><div id="idm140408044674256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140408044674256title">Acknowledgments</h2><div class="sec"><p id="__p39">We thank all individuals with psoriasis or CD, their families, control individuals and clinicians for their participation in this project. We thank the WTCCC consortium for the access to the CD case/control data. We acknowledge the cooperation of Genizon Biosciences. We wish to thank Tanja Wesse, Tanja Henke and Susan Ehlers for expert technical help. We acknowledge EGCUT and Estonian Biocentre personnel, especially Ms. M. Hass and Mr. V. Soo. A list of funding sources is included in the Supplemental Data.</p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1–S6 and Table S1–S5:</strong><div class="sup-box half_rhythm" id="idm140408040242832"><a href="/pmc/articles/PMC3322238/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.9M, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p29" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p30">1000 Genomes Project, <a href="http://www.1000genomes.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org/</a></p></li><li id="u0015"><p id="__p31">BEAGLE, <a href="http://faculty.washington.edu/browning/beagle/beagle.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://faculty.washington.edu/browning/beagle/beagle.html</a></p></li><li id="u0020"><p id="__p32">dbGaP, <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a></p></li><li id="u0025"><p id="__p33">EIGENSTRAT, <a href="http://genepath.med.harvard.edu/~reich/Software.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/∼reich/Software.htm</a></p></li><li id="u0030"><p id="__p34">GRAIL, <a href="http://www.broadinstitute.org/mpg/grail/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/grail/</a></p></li><li id="u0035"><p id="__p35">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org</a></p></li><li id="u0040"><p id="__p36">PLINK, <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a></p></li><li id="u0045"><p id="__p37">PopGen Biobank, <a href="http://www.popgen.de" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.popgen.de</a></p></li><li id="u0050"><p id="__p38">VIZ-GRAIL, <a href="http://www.broadinstitute.org/mpg/grail/vizgrail.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/grail/vizgrail.html</a></p></li></ul><p></p></div><div id="idm140408044296624" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140408044296624title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Bhalerao J., Bowcock A.M. The genetics of psoriasis: A complex disorder of the skin and immune system. <span><span class="ref-journal">Hum. Mol. Genet. </span>1998;<span class="ref-vol">7</span>:1537–1545.</span> [<a href="/pubmed/9735374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=The+genetics+of+psoriasis:+A+complex+disorder+of+the+skin+and+immune+system&amp;author=J.+Bhalerao&amp;author=A.M.+Bowcock&amp;volume=7&amp;publication_year=1998&amp;pages=1537-1545&amp;pmid=9735374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Elder J.T., Nair R.P., Guo S.W., Henseler T., Christophers E., Voorhees J.J. The genetics of psoriasis. <span><span class="ref-journal">Arch. Dermatol. </span>1994;<span class="ref-vol">130</span>:216–224.</span> [<a href="/pubmed/8304761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.&amp;title=The+genetics+of+psoriasis&amp;author=J.T.+Elder&amp;author=R.P.+Nair&amp;author=S.W.+Guo&amp;author=T.+Henseler&amp;author=E.+Christophers&amp;volume=130&amp;publication_year=1994&amp;pages=216-224&amp;pmid=8304761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Russell R.K., Satsangi J. IBD: A family affair. <span><span class="ref-journal">Best Pract. Res. Clin. Gastroenterol. </span>2004;<span class="ref-vol">18</span>:525–539.</span> [<a href="/pubmed/15157825" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Best+Pract.+Res.+Clin.+Gastroenterol.&amp;title=IBD:+A+family+affair&amp;author=R.K.+Russell&amp;author=J.+Satsangi&amp;volume=18&amp;publication_year=2004&amp;pages=525-539&amp;pmid=15157825&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Griffiths C.E., Barker J.N. Pathogenesis and clinical features of psoriasis. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">370</span>:263–271.</span> [<a href="/pubmed/17658397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pathogenesis+and+clinical+features+of+psoriasis&amp;author=C.E.+Griffiths&amp;author=J.N.+Barker&amp;volume=370&amp;publication_year=2007&amp;pages=263-271&amp;pmid=17658397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Logan I., Bowlus C.L. The geoepidemiology of autoimmune intestinal diseases. <span><span class="ref-journal">Autoimmun. Rev. </span>2010;<span class="ref-vol">9</span>:A372–A378.</span> [<a href="/pubmed/19903540" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmun.+Rev.&amp;title=The+geoepidemiology+of+autoimmune+intestinal+diseases&amp;author=I.+Logan&amp;author=C.L.+Bowlus&amp;volume=9&amp;publication_year=2010&amp;pages=A372-A378&amp;pmid=19903540&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Sagoo G.S., Cork M.J., Patel R., Tazi-Ahnini R. Genome-wide studies of psoriasis susceptibility loci: A review. <span><span class="ref-journal">J. Dermatol. Sci. </span>2004;<span class="ref-vol">35</span>:171–179.</span> [<a href="/pubmed/15381238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Dermatol.+Sci.&amp;title=Genome-wide+studies+of+psoriasis+susceptibility+loci:+A+review&amp;author=G.S.+Sagoo&amp;author=M.J.+Cork&amp;author=R.+Patel&amp;author=R.+Tazi-Ahnini&amp;volume=35&amp;publication_year=2004&amp;pages=171-179&amp;pmid=15381238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Najarian D.J., Gottlieb A.B. Connections between psoriasis and Crohn's disease. <span><span class="ref-journal">J. Am. Acad. Dermatol. </span>2003;<span class="ref-vol">48</span>:805–821.</span> quiz 822–804. [<a href="/pubmed/12789169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Acad.+Dermatol.&amp;title=Connections+between+psoriasis+and+Crohn's+disease&amp;author=D.J.+Najarian&amp;author=A.B.+Gottlieb&amp;volume=48&amp;publication_year=2003&amp;pages=805-821&amp;pmid=12789169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Khor B., Gardet A., Xavier R.J. Genetics and pathogenesis of inflammatory bowel disease. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">474</span>:307–317.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3204665/">PMC free article</a>]</span> [<a href="/pubmed/21677747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics+and+pathogenesis+of+inflammatory+bowel+disease&amp;author=B.+Khor&amp;author=A.+Gardet&amp;author=R.J.+Xavier&amp;volume=474&amp;publication_year=2011&amp;pages=307-317&amp;pmid=21677747&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Yates V.M., Watkinson G., Kelman A. Further evidence for an association between psoriasis, Crohn's disease and ulcerative colitis. <span><span class="ref-journal">Br. J. Dermatol. </span>1982;<span class="ref-vol">106</span>:323–330.</span> [<a href="/pubmed/7066192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=Further+evidence+for+an+association+between+psoriasis,+Crohn's+disease+and+ulcerative+colitis&amp;author=V.M.+Yates&amp;author=G.+Watkinson&amp;author=A.+Kelman&amp;volume=106&amp;publication_year=1982&amp;pages=323-330&amp;pmid=7066192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Bernstein C.N., Wajda A., Blanchard J.F. The clustering of other chronic inflammatory diseases in inflammatory bowel disease: A population-based study. <span><span class="ref-journal">Gastroenterology. </span>2005;<span class="ref-vol">129</span>:827–836.</span> [<a href="/pubmed/16143122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=The+clustering+of+other+chronic+inflammatory+diseases+in+inflammatory+bowel+disease:+A+population-based+study&amp;author=C.N.+Bernstein&amp;author=A.+Wajda&amp;author=J.F.+Blanchard&amp;volume=129&amp;publication_year=2005&amp;pages=827-836&amp;pmid=16143122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Weng X., Liu L., Barcellos L.F., Allison J.E., Herrinton L.J. Clustering of inflammatory bowel disease with immune mediated diseases among members of a northern california-managed care organization. <span><span class="ref-journal">Am. J. Gastroenterol. </span>2007;<span class="ref-vol">102</span>:1429–1435.</span> [<a href="/pubmed/17437504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Gastroenterol.&amp;title=Clustering+of+inflammatory+bowel+disease+with+immune+mediated+diseases+among+members+of+a+northern+california-managed+care+organization&amp;author=X.+Weng&amp;author=L.+Liu&amp;author=L.F.+Barcellos&amp;author=J.E.+Allison&amp;author=L.J.+Herrinton&amp;volume=102&amp;publication_year=2007&amp;pages=1429-1435&amp;pmid=17437504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Barrett J.C., Hansoul S., Nicolae D.L., Cho J.H., Duerr R.H., Rioux J.D., Brant S.R., Silverberg M.S., Taylor K.D., Barmada M.M., NIDDK IBD Genetics Consortium. Belgian-French IBD Consortium. Wellcome Trust Case Control Consortium  Genome-wide association defines more than 30 distinct susceptibility loci for Crohn's disease. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:955–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574810/">PMC free article</a>]</span> [<a href="/pubmed/18587394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+defines+more+than+30+distinct+susceptibility+loci+for+Crohn's+disease&amp;author=J.C.+Barrett&amp;author=S.+Hansoul&amp;author=D.L.+Nicolae&amp;author=J.H.+Cho&amp;author=R.H.+Duerr&amp;volume=40&amp;publication_year=2008&amp;pages=955-962&amp;pmid=18587394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Franke A., McGovern D.P., Barrett J.C., Wang K., Radford-Smith G.L., Ahmad T., Lees C.W., Balschun T., Lee J., Roberts R. Genome-wide meta-analysis increases to 71 the number of confirmed Crohn's disease susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1118–1125.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3299551/">PMC free article</a>]</span> [<a href="/pubmed/21102463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+meta-analysis+increases+to+71+the+number+of+confirmed+Crohn's+disease+susceptibility+loci&amp;author=A.+Franke&amp;author=D.P.+McGovern&amp;author=J.C.+Barrett&amp;author=K.+Wang&amp;author=G.L.+Radford-Smith&amp;volume=42&amp;publication_year=2010&amp;pages=1118-1125&amp;pmid=21102463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Nair R.P., Duffin K.C., Helms C., Ding J., Stuart P.E., Goldgar D., Gudjonsson J.E., Li Y., Tejasvi T., Feng B.J., Collaborative Association Study of Psoriasis  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=R.P.+Nair&amp;author=K.C.+Duffin&amp;author=C.+Helms&amp;author=J.+Ding&amp;author=P.E.+Stuart&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Ellinghaus E., Ellinghaus D., Stuart P.E., Nair R.P., Debrus S., Raelson J.V., Belouchi M., Fournier H., Reinhard C., Ding J. Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:991–995.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3136364/">PMC free article</a>]</span> [<a href="/pubmed/20953188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+a+psoriasis+susceptibility+locus+at+TRAF3IP2&amp;author=E.+Ellinghaus&amp;author=D.+Ellinghaus&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=S.+Debrus&amp;volume=42&amp;publication_year=2010&amp;pages=991-995&amp;pmid=20953188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Strange A., Capon F., Spencer C.C., Knight J., Weale M.E., Allen M.H., Barton A., Band G., Bellenguez C., Bergboer J.G., Genetic Analysis of Psoriasis Consortium &amp; the Wellcome Trust Case Control Consortium 2  A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:985–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749730/">PMC free article</a>]</span> [<a href="/pubmed/20953190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+new+psoriasis+susceptibility+loci+and+an+interaction+between+HLA-C+and+ERAP1&amp;author=A.+Strange&amp;author=F.+Capon&amp;author=C.C.+Spencer&amp;author=J.+Knight&amp;author=M.E.+Weale&amp;volume=42&amp;publication_year=2010&amp;pages=985-990&amp;pmid=20953190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Sun L.D., Cheng H., Wang Z.X., Zhang A.P., Wang P.G., Xu J.H., Zhu Q.X., Zhou H.S., Ellinghaus E., Zhang F.R. Association analyses identify six new psoriasis susceptibility loci in the Chinese population. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1005–1009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140436/">PMC free article</a>]</span> [<a href="/pubmed/20953187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Association+analyses+identify+six+new+psoriasis+susceptibility+loci+in+the+Chinese+population&amp;author=L.D.+Sun&amp;author=H.+Cheng&amp;author=Z.X.+Wang&amp;author=A.P.+Zhang&amp;author=P.G.+Wang&amp;volume=42&amp;publication_year=2010&amp;pages=1005-1009&amp;pmid=20953187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Stuart P.E., Nair R.P., Ellinghaus E., Ding J., Tejasvi T., Gudjonsson J.E., Li Y., Weidinger S., Eberlein B., Gieger C. Genome-wide association analysis identifies three psoriasis susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1000–1004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965799/">PMC free article</a>]</span> [<a href="/pubmed/20953189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+three+psoriasis+susceptibility+loci&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=E.+Ellinghaus&amp;author=J.+Ding&amp;author=T.+Tejasvi&amp;volume=42&amp;publication_year=2010&amp;pages=1000-1004&amp;pmid=20953189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Hüffmeier U., Uebe S., Ekici A.B., Bowes J., Giardina E., Korendowych E., Juneblad K., Apel M., McManus R., Ho P. Common variants at TRAF3IP2 are associated with susceptibility to psoriatic arthritis and psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:996–999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2981079/">PMC free article</a>]</span> [<a href="/pubmed/20953186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+at+TRAF3IP2+are+associated+with+susceptibility+to+psoriatic+arthritis+and+psoriasis&amp;author=U.+Hüffmeier&amp;author=S.+Uebe&amp;author=A.B.+Ekici&amp;author=J.+Bowes&amp;author=E.+Giardina&amp;volume=42&amp;publication_year=2010&amp;pages=996-999&amp;pmid=20953186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Zhang X.J., Huang W., Yang S., Sun L.D., Zhang F.Y., Zhu Q.X., Zhang F.R., Zhang C., Du W.H., Pu X.M. Psoriasis genome-wide association study identifies susceptibility variants within LCE gene cluster at 1q21. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:205–210.</span> [<a href="/pubmed/19169255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Psoriasis+genome-wide+association+study+identifies+susceptibility+variants+within+LCE+gene+cluster+at+1q21&amp;author=X.J.+Zhang&amp;author=W.+Huang&amp;author=S.+Yang&amp;author=L.D.+Sun&amp;author=F.Y.+Zhang&amp;volume=41&amp;publication_year=2009&amp;pages=205-210&amp;pmid=19169255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Duerr R.H., Taylor K.D., Brant S.R., Rioux J.D., Silverberg M.S., Daly M.J., Steinhart A.H., Abraham C., Regueiro M., Griffiths A. A genome-wide association study identifies IL23R as an inflammatory bowel disease gene. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>:1461–1463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+identifies+IL23R+as+an+inflammatory+bowel+disease+gene&amp;author=R.H.+Duerr&amp;author=K.D.+Taylor&amp;author=S.R.+Brant&amp;author=J.D.+Rioux&amp;author=M.S.+Silverberg&amp;volume=314&amp;publication_year=2006&amp;pages=1461-1463&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Cargill M., Schrodi S.J., Chang M., Garcia V.E., Brandon R., Callis K.P., Matsunami N., Ardlie K.G., Civello D., Catanese J.J. A large-scale genetic association study confirms IL12B and leads to the identification of IL23R as psoriasis-risk genes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:273–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M.+Cargill&amp;author=S.J.+Schrodi&amp;author=M.+Chang&amp;author=V.E.+Garcia&amp;author=R.+Brandon&amp;volume=80&amp;publication_year=2007&amp;pages=273-290&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Nair R.P., Ruether A., Stuart P.E., Jenisch S., Tejasvi T., Hiremagalore R., Schreiber S., Kabelitz D., Lim H.W., Voorhees J.J. Polymorphisms of the IL12B and IL23R genes are associated with psoriasis. <span><span class="ref-journal">J. Invest. Dermatol. </span>2008;<span class="ref-vol">128</span>:1653–1661.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2739284/">PMC free article</a>]</span> [<a href="/pubmed/18219280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Invest.+Dermatol.&amp;title=Polymorphisms+of+the+IL12B+and+IL23R+genes+are+associated+with+psoriasis&amp;author=R.P.+Nair&amp;author=A.+Ruether&amp;author=P.E.+Stuart&amp;author=S.+Jenisch&amp;author=T.+Tejasvi&amp;volume=128&amp;publication_year=2008&amp;pages=1653-1661&amp;pmid=18219280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Mannon P.J., Fuss I.J., Mayer L., Elson C.O., Sandborn W.J., Present D., Dolin B., Goodman N., Groden C., Hornung R.L., Anti-IL-12 Crohn's Disease Study Group  Anti-interleukin-12 antibody for active Crohn's disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2004;<span class="ref-vol">351</span>:2069–2079.</span> [<a href="/pubmed/15537905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Anti-interleukin-12+antibody+for+active+Crohn's+disease&amp;author=P.J.+Mannon&amp;author=I.J.+Fuss&amp;author=L.+Mayer&amp;author=C.O.+Elson&amp;author=W.J.+Sandborn&amp;volume=351&amp;publication_year=2004&amp;pages=2069-2079&amp;pmid=15537905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Abraham C., Cho J.H. IL-23 and autoimmunity: New insights into the pathogenesis of inflammatory bowel disease. <span><span class="ref-journal">Annu. Rev. Med. </span>2009;<span class="ref-vol">60</span>:97–110.</span> [<a href="/pubmed/18976050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Med.&amp;title=IL-23+and+autoimmunity:+New+insights+into+the+pathogenesis+of+inflammatory+bowel+disease&amp;author=C.+Abraham&amp;author=J.H.+Cho&amp;volume=60&amp;publication_year=2009&amp;pages=97-110&amp;pmid=18976050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Di Meglio P., Di Cesare A., Laggner U., Chu C.C., Napolitano L., Villanova F., Tosi I., Capon F., Trembath R.C., Peris K., Nestle F.O. The IL23R R381Q gene variant protects against immune-mediated diseases by impairing IL-23-induced Th17 effector response in humans. <span><span class="ref-journal">PLoS ONE. </span>2011;<span class="ref-vol">6</span>:e17160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3043090/">PMC free article</a>]</span> [<a href="/pubmed/21364948" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=The+IL23R+R381Q+gene+variant+protects+against+immune-mediated+diseases+by+impairing+IL-23-induced+Th17+effector+response+in+humans&amp;author=P.+Di+Meglio&amp;author=A.+Di+Cesare&amp;author=U.+Laggner&amp;author=C.C.+Chu&amp;author=L.+Napolitano&amp;volume=6&amp;publication_year=2011&amp;pages=e17160&amp;pmid=21364948&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Franke A., Fischer A., Nothnagel M., Becker C., Grabe N., Till A., Lu T., Müller-Quernheim J., Wittig M., Hermann A. Genome-wide association analysis in sarcoidosis and Crohn's disease unravels a common susceptibility locus on 10p12.2. <span><span class="ref-journal">Gastroenterology. </span>2008;<span class="ref-vol">135</span>:1207–1215.</span> [<a href="/pubmed/18723019" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Genome-wide+association+analysis+in+sarcoidosis+and+Crohn's+disease+unravels+a+common+susceptibility+locus+on+10p12.2&amp;author=A.+Franke&amp;author=A.+Fischer&amp;author=M.+Nothnagel&amp;author=C.+Becker&amp;author=N.+Grabe&amp;volume=135&amp;publication_year=2008&amp;pages=1207-1215&amp;pmid=18723019&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Wellcome Trust Case Control Consortium  Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Franke A., Balschun T., Karlsen T.H., Hedderich J., May S., Lu T., Schuldt D., Nikolaus S., Rosenstiel P., Krawczak M., Schreiber S. Replication of signals from recent studies of Crohn's disease identifies previously unknown disease loci for ulcerative colitis. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:713–715.</span> [<a href="/pubmed/18438405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Replication+of+signals+from+recent+studies+of+Crohn's+disease+identifies+previously+unknown+disease+loci+for+ulcerative+colitis&amp;author=A.+Franke&amp;author=T.+Balschun&amp;author=T.H.+Karlsen&amp;author=J.+Hedderich&amp;author=S.+May&amp;volume=40&amp;publication_year=2008&amp;pages=713-715&amp;pmid=18438405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Wang K., Baldassano R., Zhang H., Qu H.Q., Imielinski M., Kugathasan S., Annese V., Dubinsky M., Rotter J.I., Russell R.K. Comparative genetic analysis of inflammatory bowel disease and type 1 diabetes implicates multiple loci with opposite effects. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2059–2067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860894/">PMC free article</a>]</span> [<a href="/pubmed/20176734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Comparative+genetic+analysis+of+inflammatory+bowel+disease+and+type+1+diabetes+implicates+multiple+loci+with+opposite+effects&amp;author=K.+Wang&amp;author=R.+Baldassano&amp;author=H.+Zhang&amp;author=H.Q.+Qu&amp;author=M.+Imielinski&amp;volume=19&amp;publication_year=2010&amp;pages=2059-2067&amp;pmid=20176734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Cotsapas C., Voight B.F., Rossin E., Lage K., Neale B.M., Wallace C., Abecasis G.R., Barrett J.C., Behrens T., Cho J., FOCiS Network of Consortia  Pervasive sharing of genetic effects in autoimmune disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002254.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3154137/">PMC free article</a>]</span> [<a href="/pubmed/21852963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Pervasive+sharing+of+genetic+effects+in+autoimmune+disease&amp;author=C.+Cotsapas&amp;author=B.F.+Voight&amp;author=E.+Rossin&amp;author=K.+Lage&amp;author=B.M.+Neale&amp;volume=7&amp;publication_year=2011&amp;pages=e1002254&amp;pmid=21852963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Imielinski M., Baldassano R.N., Griffiths A., Russell R.K., Annese V., Dubinsky M., Kugathasan S., Bradfield J.P., Walters T.D., Sleiman P., Western Regional Alliance for Pediatric IBD. International IBD Genetics Consortium. NIDDK IBD Genetics Consortium. Belgian-French IBD Consortium. Wellcome Trust Case Control Consortium  Common variants at five new loci associated with early-onset inflammatory bowel disease. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1335–1340.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3267927/">PMC free article</a>]</span> [<a href="/pubmed/19915574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+at+five+new+loci+associated+with+early-onset+inflammatory+bowel+disease&amp;author=M.+Imielinski&amp;author=R.N.+Baldassano&amp;author=A.+Griffiths&amp;author=R.K.+Russell&amp;author=V.+Annese&amp;volume=41&amp;publication_year=2009&amp;pages=1335-1340&amp;pmid=19915574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">Festen E.A., Goyette P., Green T., Boucher G., Beauchamp C., Trynka G., Dubois P.C., Lagacé C., Stokkers P.C., Hommes D.W. A meta-analysis of genome-wide association scans identifies IL18RAP, PTPN2, TAGAP, and PUS10 as shared risk loci for Crohn's disease and celiac disease. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1001283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3029251/">PMC free article</a>]</span> [<a href="/pubmed/21298027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+meta-analysis+of+genome-wide+association+scans+identifies+IL18RAP,+PTPN2,+TAGAP,+and+PUS10+as+shared+risk+loci+for+Crohn's+disease+and+celiac+disease&amp;author=E.A.+Festen&amp;author=P.+Goyette&amp;author=T.+Green&amp;author=G.+Boucher&amp;author=C.+Beauchamp&amp;volume=7&amp;publication_year=2011&amp;pages=e1001283&amp;pmid=21298027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Zhernakova A., Stahl E.A., Trynka G., Raychaudhuri S., Festen E.A., Franke L., Westra H.J., Fehrmann R.S., Kurreeman F.A., Thomson B. Meta-analysis of genome-wide association studies in celiac disease and rheumatoid arthritis identifies fourteen non-HLA shared loci. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3044685/">PMC free article</a>]</span> [<a href="/pubmed/21383967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Meta-analysis+of+genome-wide+association+studies+in+celiac+disease+and+rheumatoid+arthritis+identifies+fourteen+non-HLA+shared+loci&amp;author=A.+Zhernakova&amp;author=E.A.+Stahl&amp;author=G.+Trynka&amp;author=S.+Raychaudhuri&amp;author=E.A.+Festen&amp;volume=7&amp;publication_year=2011&amp;pages=e1002004&amp;pmid=21383967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation">Krawczak M., Nikolaus S., von Eberstein H., Croucher P.J., El Mokhtari N.E., Schreiber S. PopGen: Population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. <span><span class="ref-journal">Community Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> [<a href="/pubmed/16490960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Community+Genet.&amp;title=PopGen:+Population-based+recruitment+of+patients+and+controls+for+the+analysis+of+complex+genotype-phenotype+relationships&amp;author=M.+Krawczak&amp;author=S.+Nikolaus&amp;author=H.+von+Eberstein&amp;author=P.J.+Croucher&amp;author=N.E.+El+Mokhtari&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;pmid=16490960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation">Wichmann H.E., Gieger C., Illig T., MONICA/KORA Study Group  KORA-gen—resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(Suppl 1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen—resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=H.E.+Wichmann&amp;author=C.+Gieger&amp;author=T.+Illig&amp;volume=67&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation">Weiland S.K., Björkstén B., Brunekreef B., Cookson W.O., von Mutius E., Strachan D.P., International Study of Asthma and Allergies in Childhood Phase II Study Group  Phase II of the International Study of Asthma and Allergies in Childhood (ISAAC II): Rationale and methods. <span><span class="ref-journal">Eur. Respir. J. </span>2004;<span class="ref-vol">24</span>:406–412.</span> [<a href="/pubmed/15358699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Respir.+J.&amp;title=Phase+II+of+the+International+Study+of+Asthma+and+Allergies+in+Childhood+(ISAAC+II):+Rationale+and+methods&amp;author=S.K.+Weiland&amp;author=B.+Björkstén&amp;author=B.+Brunekreef&amp;author=W.O.+Cookson&amp;author=E.+von+Mutius&amp;volume=24&amp;publication_year=2004&amp;pages=406-412&amp;pmid=15358699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation">Manolio T.A., Rodriguez L.L., Brooks L., Abecasis G., Ballinger D., Daly M., Donnelly P., Faraone S.V., Frazer K., Gabriel S., GAIN Collaborative Research Group. Collaborative Association Study of Psoriasis. International Multi-Center ADHD Genetics Project. Molecular Genetics of Schizophrenia Collaboration. Bipolar Genome Study. Major Depression Stage 1 Genomewide Association in Population-Based Samples Study. Genetics of Kidneys in Diabetes (GoKinD) Study  New models of collaboration in genome-wide association studies: The Genetic Association Information Network. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1045–1051.</span> [<a href="/pubmed/17728769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=New+models+of+collaboration+in+genome-wide+association+studies:+The+Genetic+Association+Information+Network&amp;author=T.A.+Manolio&amp;author=L.L.+Rodriguez&amp;author=L.+Brooks&amp;author=G.+Abecasis&amp;author=D.+Ballinger&amp;volume=39&amp;publication_year=2007&amp;pages=1045-1051&amp;pmid=17728769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: A tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+A+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation">Browning B.L., Browning S.R. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">84</span>:210–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=B.L.+Browning&amp;author=S.R.+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-223&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation">Altshuler D.M., Gibbs R.A., Peltonen L., Altshuler D.M., Gibbs R.A., Peltonen L., Dermitzakis E., Schaffner S.F., Yu F., Peltonen L., International HapMap 3 Consortium  Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;author=D.M.+Altshuler&amp;author=R.A.+Gibbs&amp;author=L.+Peltonen&amp;author=D.M.+Altshuler&amp;author=R.A.+Gibbs&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation">Devlin B., Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation">de Bakker P.I., Ferreira M.A., Jia X., Neale B.M., Raychaudhuri S., Voight B.F. Practical aspects of imputation-driven meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>(R2):R122–R128.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782358/">PMC free article</a>]</span> [<a href="/pubmed/18852200" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Practical+aspects+of+imputation-driven+meta-analysis+of+genome-wide+association+studies&amp;author=P.I.+de+Bakker&amp;author=M.A.+Ferreira&amp;author=X.+Jia&amp;author=B.M.+Neale&amp;author=S.+Raychaudhuri&amp;volume=17&amp;issue=R2&amp;publication_year=2008&amp;pages=R122-R128&amp;pmid=18852200&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation">1000 Genomes Project Consortium  A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation">Raychaudhuri S., Plenge R.M., Rossin E.J., Ng A.C., Purcell S.M., Sklar P., Scolnick E.M., Xavier R.J., Altshuler D., Daly M.J., International Schizophrenia Consortium  Identifying relationships among genomic disease regions: Predicting genes at pathogenic SNP associations and rare deletions. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000534.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694358/">PMC free article</a>]</span> [<a href="/pubmed/19557189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Identifying+relationships+among+genomic+disease+regions:+Predicting+genes+at+pathogenic+SNP+associations+and+rare+deletions&amp;author=S.+Raychaudhuri&amp;author=R.M.+Plenge&amp;author=E.J.+Rossin&amp;author=A.C.+Ng&amp;author=S.M.+Purcell&amp;volume=5&amp;publication_year=2009&amp;pages=e1000534&amp;pmid=19557189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation">Dixon A.L., Liang L., Moffatt M.F., Chen W., Heath S., Wong K.C., Taylor J., Burnett E., Gut I., Farrall M. A genome-wide association study of global gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1202–1207.</span> [<a href="/pubmed/17873877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+of+global+gene+expression&amp;author=A.L.+Dixon&amp;author=L.+Liang&amp;author=M.F.+Moffatt&amp;author=W.+Chen&amp;author=S.+Heath&amp;volume=39&amp;publication_year=2007&amp;pages=1202-1207&amp;pmid=17873877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation">Li X., Thyssen G., Beliakoff J., Sun Z. The novel PIAS-like protein hZimp10 enhances Smad transcriptional activity. <span><span class="ref-journal">J. Biol. Chem. </span>2006;<span class="ref-vol">281</span>:23748–23756.</span> [<a href="/pubmed/16777850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+novel+PIAS-like+protein+hZimp10+enhances+Smad+transcriptional+activity&amp;author=X.+Li&amp;author=G.+Thyssen&amp;author=J.+Beliakoff&amp;author=Z.+Sun&amp;volume=281&amp;publication_year=2006&amp;pages=23748-23756&amp;pmid=16777850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation">Carlsson B., Kindberg E., Buesa J., Rydell G.E., Lidón M.F., Montava R., Abu Mallouh R., Grahn A., Rodríguez-Díaz J., Bellido J. The G428A nonsense mutation in&nbsp;FUT2 provides strong but not absolute protection against symptomatic GII.4 Norovirus infection. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e5593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680586/">PMC free article</a>]</span> [<a href="/pubmed/19440360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=The+G428A+nonsense+mutation+in&nbsp;FUT2+provides+strong+but+not+absolute+protection+against+symptomatic+GII.4+Norovirus+infection&amp;author=B.+Carlsson&amp;author=E.+Kindberg&amp;author=J.+Buesa&amp;author=G.E.+Rydell&amp;author=M.F.+Lidón&amp;volume=4&amp;publication_year=2009&amp;pages=e5593&amp;pmid=19440360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation">Ikehara Y., Nishihara S., Yasutomi H., Kitamura T., Matsuo K., Shimizu N., Inada K., Kodera Y., Yamamura Y., Narimatsu H. Polymorphisms of two fucosyltransferase genes (Lewis and Secretor genes) involving type I Lewis antigens are associated with the presence of anti-Helicobacter pylori IgG antibody. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2001;<span class="ref-vol">10</span>:971–977.</span> [<a href="/pubmed/11535550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Polymorphisms+of+two+fucosyltransferase+genes+(Lewis+and+Secretor+genes)+involving+type+I+Lewis+antigens+are+associated+with+the+presence+of+anti-Helicobacter+pylori+IgG+antibody&amp;author=Y.+Ikehara&amp;author=S.+Nishihara&amp;author=H.+Yasutomi&amp;author=T.+Kitamura&amp;author=K.+Matsuo&amp;volume=10&amp;publication_year=2001&amp;pages=971-977&amp;pmid=11535550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation">Starr R., Willson T.A., Viney E.M., Murray L.J., Rayner J.R., Jenkins B.J., Gonda T.J., Alexander W.S., Metcalf D., Nicola N.A., Hilton D.J. A family of cytokine-inducible inhibitors of signalling. <span><span class="ref-journal">Nature. </span>1997;<span class="ref-vol">387</span>:917–921.</span> [<a href="/pubmed/9202125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+family+of+cytokine-inducible+inhibitors+of+signalling&amp;author=R.+Starr&amp;author=T.A.+Willson&amp;author=E.M.+Viney&amp;author=L.J.+Murray&amp;author=J.R.+Rayner&amp;volume=387&amp;publication_year=1997&amp;pages=917-921&amp;pmid=9202125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation">Yasukawa H., Sasaki A., Yoshimura A. Negative regulation of cytokine signaling pathways. <span><span class="ref-journal">Annu. Rev. Immunol. </span>2000;<span class="ref-vol">18</span>:143–164.</span> [<a href="/pubmed/10837055" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Immunol.&amp;title=Negative+regulation+of+cytokine+signaling+pathways&amp;author=H.+Yasukawa&amp;author=A.+Sasaki&amp;author=A.+Yoshimura&amp;volume=18&amp;publication_year=2000&amp;pages=143-164&amp;pmid=10837055&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation">Krebs D.L., Hilton D.J. SOCS: Physiological suppressors of cytokine signaling. <span><span class="ref-journal">J. Cell Sci. </span>2000;<span class="ref-vol">113</span>:2813–2819.</span> [<a href="/pubmed/10910765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Sci.&amp;title=SOCS:+Physiological+suppressors+of+cytokine+signaling&amp;author=D.L.+Krebs&amp;author=D.J.+Hilton&amp;volume=113&amp;publication_year=2000&amp;pages=2813-2819&amp;pmid=10910765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation">Abbas-Terki T., Briand P.A., Donzé O., Picard D. The Hsp90 co-chaperones Cdc37 and Sti1 interact physically and genetically. <span><span class="ref-journal">Biol. Chem. </span>2002;<span class="ref-vol">383</span>:1335–1342.</span> [<a href="/pubmed/12437126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Chem.&amp;title=The+Hsp90+co-chaperones+Cdc37+and+Sti1+interact+physically+and+genetically&amp;author=T.+Abbas-Terki&amp;author=P.A.+Briand&amp;author=O.+Donzé&amp;author=D.+Picard&amp;volume=383&amp;publication_year=2002&amp;pages=1335-1342&amp;pmid=12437126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation">Shang L., Tomasi T.B. The heat shock protein 90-CDC37 chaperone complex is required for signaling by types I and II interferons. <span><span class="ref-journal">J. Biol. Chem. </span>2006;<span class="ref-vol">281</span>:1876–1884.</span> [<a href="/pubmed/16280321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+heat+shock+protein+90-CDC37+chaperone+complex+is+required+for+signaling+by+types+I+and+II+interferons&amp;author=L.+Shang&amp;author=T.B.+Tomasi&amp;volume=281&amp;publication_year=2006&amp;pages=1876-1884&amp;pmid=16280321&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation">Marubayashi S., Koppikar P., Taldone T., Abdel-Wahab O., West N., Bhagwat N., Caldas-Lopes E., Ross K.N., Gönen M., Gozman A. HSP90 is a therapeutic target in JAK2-dependent myeloproliferative neoplasms in mice and humans. <span><span class="ref-journal">J. Clin. Invest. </span>2010;<span class="ref-vol">120</span>:3578–3593.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947224/">PMC free article</a>]</span> [<a href="/pubmed/20852385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest.&amp;title=HSP90+is+a+therapeutic+target+in+JAK2-dependent+myeloproliferative+neoplasms+in+mice+and+humans&amp;author=S.+Marubayashi&amp;author=P.+Koppikar&amp;author=T.+Taldone&amp;author=O.+Abdel-Wahab&amp;author=N.+West&amp;volume=120&amp;publication_year=2010&amp;pages=3578-3593&amp;pmid=20852385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation">Mayor A., Martinon F., De Smedt T., Pétrilli V., Tschopp J. A crucial function of SGT1 and HSP90 in inflammasome activity links mammalian and plant innate immune responses. <span><span class="ref-journal">Nat. Immunol. </span>2007;<span class="ref-vol">8</span>:497–503.</span> [<a href="/pubmed/17435760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol.&amp;title=A+crucial+function+of+SGT1+and+HSP90+in+inflammasome+activity+links+mammalian+and+plant+innate+immune+responses&amp;author=A.+Mayor&amp;author=F.+Martinon&amp;author=T.+De+Smedt&amp;author=V.+Pétrilli&amp;author=J.+Tschopp&amp;volume=8&amp;publication_year=2007&amp;pages=497-503&amp;pmid=17435760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation">Hoffman H.M., Mueller J.L., Broide D.H., Wanderer A.A., Kolodner R.D. Mutation of a new gene encoding a putative pyrin-like protein causes familial cold autoinflammatory syndrome and Muckle-Wells syndrome. <span><span class="ref-journal">Nat. Genet. </span>2001;<span class="ref-vol">29</span>:301–305.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4322000/">PMC free article</a>]</span> [<a href="/pubmed/11687797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Mutation+of+a+new+gene+encoding+a+putative+pyrin-like+protein+causes+familial+cold+autoinflammatory+syndrome+and+Muckle-Wells+syndrome&amp;author=H.M.+Hoffman&amp;author=J.L.+Mueller&amp;author=D.H.+Broide&amp;author=A.A.+Wanderer&amp;author=R.D.+Kolodner&amp;volume=29&amp;publication_year=2001&amp;pages=301-305&amp;pmid=11687797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">59. <span class="element-citation">Hawkins P.N., Lachmann H.J., Aganna E., McDermott M.F. Spectrum of clinical features in Muckle-Wells syndrome and response to anakinra. <span><span class="ref-journal">Arthritis Rheum. </span>2004;<span class="ref-vol">50</span>:607–612.</span> [<a href="/pubmed/14872505" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Spectrum+of+clinical+features+in+Muckle-Wells+syndrome+and+response+to+anakinra&amp;author=P.N.+Hawkins&amp;author=H.J.+Lachmann&amp;author=E.+Aganna&amp;author=M.F.+McDermott&amp;volume=50&amp;publication_year=2004&amp;pages=607-612&amp;pmid=14872505&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib60">60. <span class="element-citation">Morris A.P., Lindgren C.M., Zeggini E., Timpson N.J., Frayling T.M., Hattersley A.T., McCarthy M.I. A powerful approach to sub-phenotype analysis in population-based genetic association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:335–343.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2964510/">PMC free article</a>]</span> [<a href="/pubmed/20039379" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=A+powerful+approach+to+sub-phenotype+analysis+in+population-based+genetic+association+studies&amp;author=A.P.+Morris&amp;author=C.M.+Lindgren&amp;author=E.+Zeggini&amp;author=N.J.+Timpson&amp;author=T.M.+Frayling&amp;volume=34&amp;publication_year=2010&amp;pages=335-343&amp;pmid=20039379&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib61">61. <span class="element-citation">Lees C.W., Barrett J.C., Parkes M., Satsangi J. New IBD genetics: Common pathways with other diseases. <span><span class="ref-journal">Gut. </span>2011;<span class="ref-vol">60</span>:1739–1753.</span> [<a href="/pubmed/21300624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=New+IBD+genetics:+Common+pathways+with+other+diseases&amp;author=C.W.+Lees&amp;author=J.C.+Barrett&amp;author=M.+Parkes&amp;author=J.+Satsangi&amp;volume=60&amp;publication_year=2011&amp;pages=1739-1753&amp;pmid=21300624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3322238&amp;issue-id=208649&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140408040171520title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Subjects and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#idm140408044674256title">Acknowledgments</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm140408044296624title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3322238/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3322238/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3322238/pdf/main.pdf">PDF (822K)</a></li> | <li><a href="#" data-citationid="PMC3322238" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3322238%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3322238%2F&amp;text=Combined%20Analysis%20of%20Genome-wide%20Association%20Studies%20for%20Crohn%20Disease%20and%20Psoriasis%20Identifies%20Seven%20Shared%20Susceptibility%20Loci" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3322238%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22482804" data-aiid="3322238" data-aid="3322238" data-iid="208649" data-domainid="203" data-domain="ajhg" data-accid="PMC3322238" data-md5="980610b472953283e2b557e075e39b44"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:43:02-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE88A2F0DD540FA100000000015C00E1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3322238%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3322238/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>