<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3910086/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genomewide Association for Schizophrenia in the CATIE Study: Results
of Stage 1</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910086/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular psychiatry"><meta name="citation_title" content="Genomewide Association for Schizophrenia in the CATIE Study: Results
of Stage 1"><meta name="citation_authors" content="Patrick F. Sullivan, Danyu Lin, Jung-Ying Tzeng, Edwin van den Oord, Diana Perkins, T. Scott Stroup, Michael Wagner, Seunggeun Lee, Fred A. Wright, Fei Zou, Wenlei Liu, AnnCatherine M. Downing, Jeffrey Lieberman, Sandra L. Close"><meta name="citation_date" content="June 2008"><meta name="citation_issue" content="6"><meta name="citation_volume" content="13"><meta name="citation_firstpage" content="570"><meta name="citation_doi" content="10.1038/mp.2008.25"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3910086/?report=abstract"><meta name="citation_pmid" content="18347602"><meta name="DC.Title" content="Genomewide Association for Schizophrenia in the CATIE Study: Results
of Stage 1"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Patrick F. Sullivan"><meta name="DC.Contributor" content="Danyu Lin"><meta name="DC.Contributor" content="Jung-Ying Tzeng"><meta name="DC.Contributor" content="Edwin van den Oord"><meta name="DC.Contributor" content="Diana Perkins"><meta name="DC.Contributor" content="T. Scott Stroup"><meta name="DC.Contributor" content="Michael Wagner"><meta name="DC.Contributor" content="Seunggeun Lee"><meta name="DC.Contributor" content="Fred A. Wright"><meta name="DC.Contributor" content="Fei Zou"><meta name="DC.Contributor" content="Wenlei Liu"><meta name="DC.Contributor" content="AnnCatherine M. Downing"><meta name="DC.Contributor" content="Jeffrey Lieberman"><meta name="DC.Contributor" content="Sandra L. Close"><meta name="DC.Date" content="2008 Jun"><meta name="DC.Identifier" content="10.1038/mp.2008.25"><meta name="DC.Language" content="en"><meta property="og:title" content="Genomewide Association for Schizophrenia in the CATIE Study: Results
of Stage 1"><meta property="og:type" content="article"><meta property="og:description" content="Little is known for certain about the genetics of schizophrenia. The
advent of genomewide association has been widely anticipated as holding
promise as a means to identify reproducible DNA sequence variation
associated with this important and debilitating ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910086/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3910086/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3910086/pdf/nihms319115.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE890EA4DD3FACF100000000003D0025.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3910086%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3910086</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Psychiatry</a></span>. Author manuscript; available in PMC 2014 Feb 3.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18347602" target="pmc_ext" ref="reftype=publisher&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Mol Psychiatry. 2008 Jun; 13(6): 570–584. </span></a></div><span class="fm-vol-iss-date">Published online 2008 Mar 18. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fmp.2008.25" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/mp.2008.25</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3910086</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS319115</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/18347602">18347602</a></div></div></div></div><h1 class="content-title">Genomewide Association for Schizophrenia in the CATIE Study: Results
of Stage 1</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sullivan%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876098363456" co-class="co-affbox">Patrick F. Sullivan</a>, MD, FRANZCP,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Lin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876056750208" co-class="co-affbox">Danyu Lin</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Tzeng%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876056747968" co-class="co-affbox">Jung-Ying Tzeng</a>, PhD,<sup>4</sup> <a href="/pubmed/?term=van%20den%20Oord%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876064094432" co-class="co-affbox">Edwin van den Oord</a>, PhD,<sup>5</sup> <a href="/pubmed/?term=Perkins%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876060907888" co-class="co-affbox">Diana Perkins</a>, MD,<sup>6</sup> <a href="/pubmed/?term=Stroup%20TS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876097723136" co-class="co-affbox">T. Scott Stroup</a>, MD,<sup>6</sup> <a href="/pubmed/?term=Wagner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876097720896" co-class="co-affbox">Michael Wagner</a>, PhD,<sup>7</sup> <a href="/pubmed/?term=Lee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876054990304" co-class="co-affbox">Seunggeun Lee</a>,<sup>3</sup> <a href="/pubmed/?term=Wright%20FA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876065178368" co-class="co-affbox">Fred A. Wright</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Zou%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876063649936" co-class="co-affbox">Fei Zou</a>, PhD,<sup>3</sup> <a href="/pubmed/?term=Liu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876048686592" co-class="co-affbox">Wenlei Liu</a>, PhD,<sup>8</sup> <a href="/pubmed/?term=Downing%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876048684352" co-class="co-affbox">AnnCatherine M. Downing</a>, PharmD,<sup>9</sup> <a href="/pubmed/?term=Lieberman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876058751264" co-class="co-affbox">Jeffrey Lieberman</a>, MD,<sup>10</sup> and  <a href="/pubmed/?term=Close%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602" class="affpopup" co-rid="_co_idm139876064043664" co-class="co-affbox">Sandra L. Close</a>, PhD<sup>9</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139876098363456"><h3 class="no_margin">Patrick F. Sullivan</h3><p><sup>1</sup>Departments of Genetics, Psychiatry, &amp; Epidemiology, University of North Carolina at Chapel Hill</p><p><sup>2</sup>Department of Medical Epidemiology &amp; Biostatistics, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Sullivan%20PF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Patrick F. Sullivan</a></div></div><div id="_co_idm139876056750208"><h3 class="no_margin">Danyu Lin</h3><p><sup>3</sup>Department of Biostatistics, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Lin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Danyu Lin</a></div></div><div id="_co_idm139876056747968"><h3 class="no_margin">Jung-Ying Tzeng</h3><p><sup>4</sup>Department of Statistics, North Carolina State University</p><div>Find articles by <a href="/pubmed/?term=Tzeng%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Jung-Ying Tzeng</a></div></div><div id="_co_idm139876064094432"><h3 class="no_margin">Edwin van den Oord</h3><p><sup>5</sup>Department of Pharmacy, Virginia Commonwealth University</p><div>Find articles by <a href="/pubmed/?term=van%20den%20Oord%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Edwin van den Oord</a></div></div><div id="_co_idm139876060907888"><h3 class="no_margin">Diana Perkins</h3><p><sup>6</sup>Department of Psychiatry, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Perkins%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Diana Perkins</a></div></div><div id="_co_idm139876097723136"><h3 class="no_margin">T. Scott Stroup</h3><p><sup>6</sup>Department of Psychiatry, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Stroup%20TS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">T. Scott Stroup</a></div></div><div id="_co_idm139876097720896"><h3 class="no_margin">Michael Wagner</h3><p><sup>7</sup>School of Pharmacy, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Wagner%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Michael Wagner</a></div></div><div id="_co_idm139876054990304"><h3 class="no_margin">Seunggeun Lee</h3><p><sup>3</sup>Department of Biostatistics, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Lee%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Seunggeun Lee</a></div></div><div id="_co_idm139876065178368"><h3 class="no_margin">Fred A. Wright</h3><p><sup>3</sup>Department of Biostatistics, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Wright%20FA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Fred A. Wright</a></div></div><div id="_co_idm139876063649936"><h3 class="no_margin">Fei Zou</h3><p><sup>3</sup>Department of Biostatistics, University of North Carolina at Chapel Hill</p><div>Find articles by <a href="/pubmed/?term=Zou%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Fei Zou</a></div></div><div id="_co_idm139876048686592"><h3 class="no_margin">Wenlei Liu</h3><p><sup>8</sup>Statistics, Eli Lilly and Company, Indianapolis, IN</p><div>Find articles by <a href="/pubmed/?term=Liu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Wenlei Liu</a></div></div><div id="_co_idm139876048684352"><h3 class="no_margin">AnnCatherine M. Downing</h3><p><sup>9</sup>Department of Experimental Medicine, Eli Lilly and Company, Indianapolis, IN</p><div>Find articles by <a href="/pubmed/?term=Downing%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">AnnCatherine M. Downing</a></div></div><div id="_co_idm139876058751264"><h3 class="no_margin">Jeffrey Lieberman</h3><p><sup>10</sup>Department of Psychiatry, Columbia University</p><div>Find articles by <a href="/pubmed/?term=Lieberman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Jeffrey Lieberman</a></div></div><div id="_co_idm139876064043664"><h3 class="no_margin">Sandra L. Close</h3><p><sup>9</sup>Department of Experimental Medicine, Eli Lilly and Company, Indianapolis, IN</p><div>Find articles by <a href="/pubmed/?term=Close%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=18347602">Sandra L. Close</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139876059955152_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139876059955152_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139876059955152_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Departments of Genetics, Psychiatry, &amp; Epidemiology, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Medical Epidemiology &amp; Biostatistics, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Biostatistics, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Statistics, North Carolina State University</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Pharmacy, Virginia Commonwealth University</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Psychiatry, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>School of Pharmacy, University of North Carolina at Chapel Hill</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Statistics, Eli Lilly and Company, Indianapolis, IN</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Experimental Medicine, Eli Lilly and Company, Indianapolis, IN</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Psychiatry, Columbia University</div><div id="cor1">Correspond with Dr. Sullivan and Dr. Close. Dr. Sullivan is in the Department of Genetics, CB#7264, 4109D Neurosciences Research Building, University of North Carolina, Chapel Hill, NC, 27599-7264, USA. Voice: +919-966-3358, FAX: +919-966-3630. <a href="mailto:dev@null" data-email="ude.cnu.dem@villusfp" class="oemail">ude.cnu.dem@villusfp</a>. Dr. Close is in the Department of Experimental Medicine, Eli Lilly and Company, Indianapolis, IN, 46285, USA. Voice: +317-651-3050, FAX: +317-651-3051. <a href="mailto:dev@null" data-email="moc.yllil@ardnas_doowkrik" class="oemail">moc.yllil@ardnas_doowkrik</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139876059955152_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=18347602" target="pmc_ext" ref="reftype=publisher&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Mol Psychiatry</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 14 on&nbsp;page&nbsp;1144.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3910086/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>31 pages including 9 tables, methods and references.</strong></div><div><a href="/pmc/articles/PMC3910086/bin/NIHMS319115-supplement-31_pages_including_9_tables__methods_and_references.doc" data-ga-action="click_feat_suppl">NIHMS319115-supplement-31_pages_including_9_tables__methods_and_references.doc</a><span style="color:gray"> (491K)</span></div><div class="small guid">GUID:&nbsp;A652B24B-3FE7-4274-9BA2-ADE6BEF7E826</div></div></dd></dl></div><div id="idm139876053304256" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139876053304256title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Little is known for certain about the genetics of schizophrenia. The
advent of genomewide association has been widely anticipated as holding
promise as a means to identify reproducible DNA sequence variation
associated with this important and debilitating disorder.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">738 cases with DSM-IV schizophrenia (all participants in the CATIE
study) and 733 group-matched controls were genotyped for 492,900 single
nucleotide polymorphisms (SNPs) using the Affymetrix 500K two chip
genotyping platform plus a custom 164K fill-in chip. Following multiple
quality control steps for both subjects and SNPs, logistic regression
analyses were used to assess the evidence for association of all SNPs with
schizophrenia.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">We identified a number of promising SNPs for follow-up studies,
although no SNP or multi-marker combination of SNPs achieved genomewide
statistical significance. Although a few signals coincided with genomic
regions previously implicated in schizophrenia, chance could not be
excluded.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">These data do not provide evidence for the involvement of any genomic
region with schizophrenia detectable with moderate sample size. However,
planned GWAS for response phenotypes and inclusion of individual phenotype
and genotype data from this study in meta-analyses holds promise for the
eventual identification of susceptibility and protective variants.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">schizophrenia, genome-wide association, CATIE</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">In the past several decades, it has become generally accepted that
schizophrenia (SCZ) is a complex trait with a substantial genetic component. The
accumulated evidence suggests that SCZ is a relatively common disorder with lifetime
morbid risk of 0.72% (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313928">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313847">2</a>). Genetic factors have been strongly and
consistently implicated via family, adoption, and twin studies (<a href="#R3" rid="R3" class=" bibr popnode">3</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313937">4</a>). A
meta-analytic estimate of the broad sense heritability (<a href="#R5" rid="R5" class=" bibr popnode">5</a>) of SCZ was 81% (95% CI,
73%–90%) (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313932">4</a>) and the
recurrence risk to siblings (λ<sub>sibs</sub>) (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313917">6</a>) was 8.55 (95% CI 7.86–9.57) in a
population-based Swedish national sample of over 7.7 million individuals (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313858">2</a>), consistent with prior estimates (<a href="#R3" rid="R3" class=" bibr popnode">3</a>).</p><p id="P6">These results have been used in support of the application of an array of
linkage and association methods (<a href="#R7" rid="R7" class=" bibr popnode">7</a>) attempting
to identify genomic regions that confer risk for or protection against the
development of SCZ. These efforts have been considerable: we have identified 31
independent samples in which genomewide linkage has been applied (3,108 multiplex
SCZ pedigrees and over 8.3 million genotypes) <sup><sup><a href="#FN1" rid="FN1" class=" fn">†</a></sup></sup> and &gt;1,100 association studies of 525
candidate genes for SCZ have been published <sup><sup><a href="#FN2" rid="FN2" class=" fn">‡</a></sup></sup>.</p><p id="P7">Despite these efforts, little is known for certain about the genetics of SCZ.
The genomewide linkage studies have not converged to identify a set of high priority
genomic regions (<a href="#R8" rid="R8" class=" bibr popnode">8</a>) ; indeed, no genomic
region has been implicated in more than four of 31 genomewide linkage studies and a
perhaps implausibly large 58% of the genome has been implicated at least
once. The accumulated evidence for specific candidate genes such as
<em>NRG1</em> (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313845">9</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313839">10</a>), <em>DTNBP1</em> (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313919">11</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313935">12</a>),
or <em>DISC1</em> (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313944">13</a>) does not
constitute unequivocal and consistent replication (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313862">14</a>). These findings stand in contrast to other results in human complex
trait genetics – for example, genetic variation in intron 1
<em>FTO</em> was associated with body mass index in 13 cohorts and
38,759 individuals with remarkable consistency (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313897">15</a>). The possibility that all members of a consensus set of the best
candidate genes for SCZ are false positives has not been convincingly excluded.</p><p id="P8">In the past two years, the widely anticipated method of genomewide
association study (GWAS) has become technically and economically feasible. These
studies entail individual genotyping of considerable numbers of cases and controls
for 100,000 or more genetic markers (single nucleotide polymorphisms, SNPs). Evident
successes in identifying highly compelling candidate genes for age-related macular
degeneration (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313949">16</a>), body mass index (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313838">15</a>), inflammatory bowel disease (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_500239030">17</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313851">18</a>),
type 1 diabetes mellitus (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313846">18</a>), and type 2
diabetes mellitus (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313909">19</a>–<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313916">22</a>) support the utility of GWAS. To date, two
GWAS that employed individual genotyping have been published for psychiatric
disorders – a small study of SCZ (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313832">23</a>)
and the Wellcome Trust Case-Control Consortium study of bipolar disorder (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313920">18</a>). Multiple GWAS are known to be in progress
for attention-deficit hyperactivity disorder, autism, bipolar disorder, and major
depressive disorder as well as for SCZ.</p><p id="P9">In 2006, academic investigators from the NIMH-funded Clinical Antipsychotic
Trials of Intervention Effectiveness project (CATIE) (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313866">24</a>, <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313887">25</a>) entered into a
scientific collaboration with Eli Lilly and Company to conduct individual GWAS
genotyping and joint analyses on the CATIE samples. The collaboration contract
required that genotype and phenotype data be made available to the scientific
community with intellectual property rights consistent with NIH policies intended to
maximize the public benefit resulting from the research. The genotype and phenotype
data reported here were deposited with the controlled-access repository of the NIMH
in 6/2007 <sup><sup><a href="#FN3" rid="FN3" class=" fn">†</a></sup></sup>.</p><p id="P10" class="p p-last">We report here the primary analyses aimed at identifying SNPs associated
with susceptibility to SCZ using 492,900 SNPs that were genotyped in 738
participants with SCZ and 733 group-matched controls from a United States population
based sample.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><p id="P11" class="p p-first">We have attempted to follow published guidelines for GWAS (26, Box 1).</p><div id="S7" class="sec"><h3 id="S7title">Subjects</h3><p id="P12" class="p p-first">All cases were participants in the CATIE project (NIMH contract NO1
MH90001) which was conducted between 1/2001-12/2004. CATIE was a multi-phase
randomized controlled trial of antipsychotic medications involving 1,460 persons
with SCZ followed for up to 18 months (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313943">24</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313855">27</a>). The philosophy of the
trial was to assess controlled treatment with antipsychotic drugs in a broad
range of patients with SCZ under “real world” conditions. To
maximize representativeness, subjects were ascertained from an array of clinical
settings across the US. 1,894 subjects were evaluated and 1,460 (77.0%)
entered into CATIE. No subject was known to be related to any other subject. The
optional genetic sub-study began about a year after the trial began and
51% of CATIE participants donated a DNA sample.</p><p id="P13">Preliminary diagnoses were established by referring psychiatrists and
final study diagnoses of DSM-IV SCZ (<a href="#R28" rid="R28" class=" bibr popnode">28</a>)
were independently established by CATIE personnel using the Structured Clinical
Interview for DSM-IV (SCID) (<a href="#R29" rid="R29" class=" bibr popnode">29</a>)
including review of all available information (including psychiatric and general
medical records) along with one or more subject interviews. Interviewers were
experienced Master’s-level clinicians who were trained to criterion via
a standard protocol (<a href="#R29" rid="R29" class=" bibr popnode">29</a>). Any diagnostic
uncertainties were resolved via discussion with one of the CATIE senior
clinicians. Study inclusion criteria were: definite diagnosis of SCZ (<a href="#R28" rid="R28" class=" bibr popnode">28</a>, <a href="#R29" rid="R29" class=" bibr popnode">29</a>), age 18–67 years, clinical decision that oral medication
was appropriate, adequate decisional capacity, and provision of written informed
consent. Exclusion criteria are detailed elsewhere (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313908">24</a>). Briefly, patients were excluded if they had: received
a diagnosis of schizoaffective disorder, mental retardation, or another
cognitive disorder; a history of serious adverse reactions to the proposed
treatments; only one psychotic episode; a history of treatment resistance
(defined by the persistence of severe symptoms despite adequate trials of one of
the proposed treatments or prior treatment with clozapine); pregnant or
breastfeeding; or a serious and unstable medical condition. Individuals with
psychoactive drug use disorders were included but only when there was positive
evidence that SCZ was an independent diagnosis.</p><p id="P14" class="p p-last">Controls were ascertained from a US national sampling frame as part of
the NIMH Genetics repository (<a id="__tag_801498395" class="tag_hotlink" href="/nuccore/MH059571" ref="/nuccore/MH059571"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH059571","term_id":"1554614518","term_text":"MH059571"}}</span>MH059571</a>, PI Dr. Pablo Gejman, release v4.0,
6/2006). Controls were collected by Knowledge Networks (KN), a survey and market
research company whose panel contains approximately 60,000 households
(&gt;120,000 unrelated adults). Households were selected via random digit
dialing and proportionally from 25 major US population areas and financial
incentives were provided for participation. The KN panel is generally
representative of the US population but with a slight bias toward higher income
and education. To be eligible for matching to cases, we required that potential
controls deny any history of SCZ, schizoaffective disorder, bipolar disorder,
auditory hallucinations, and delusional beliefs. These controls are being used
for multiple additional GWAS and meta-analysts need to use caution when
combining results or data across studies that use these common controls.</p></div><div id="S8" class="sec"><h3 id="S8title">Case-control matching</h3><p id="P15" class="p p-first-last">There were 3,487 individuals in v4.0 of the control dataset and 2,645
controls were eligible for matching to CATIE cases – 842 individuals
were removed from consideration due to at least one of the following: a
self-reported possible history of a psychotic disorder (SCZ, schizoaffective
disorder, bipolar disorder, delusional disorder, or auditory hallucinations),
age &lt;18 or age &gt;67, lack of documentation of informed consent,
or missing data for a matching variable (age, sex, or self-reported race). Cases
were group-matched to controls by five-year age band, sex, and self-reported
race. Most cases (91.4%) were successfully matched to controls. The
exception was for 66 cases (8.6%) that could not be matched due to a
deficiency of African-American males aged 18–38 years in v4.0 of the
control cohort. Rather than eliminate cases, African-American females in this
age band were selected as controls.</p></div><div id="S9" class="sec"><h3 id="S9title">DNA Sampling &amp; Cell Line Establishment</h3><p id="P16" class="p p-first-last">Peripheral venous blood samples were sent to the Rutgers University Cell
and DNA Repository (RUCDR) where cell lines were established via EBV
transformation. RUCDR employs stringent quality control procedures and the
success rate for immortalization exceeds 99%. Sample DNA concentrations
were quantified and normalized via the use of Picogreen dsDNA Quantitation Kits
(Molecular Probes, Eugene, OR).</p></div><div id="S10" class="sec"><h3 id="S10title">Ethical Issues</h3><p id="P17" class="p p-first-last">The CATIE study was approved by the institutional review board at each
site, and written informed consent was obtained from the subjects or their legal
guardians (including an additional consent for genetic studies). Written
informed consent was obtained from all controls. All biological samples,
phenotypes, and genotypes were de-identified before deposit into the NIMH/RUDCR
repositories. Although the analyses described here are generally considered not
to be “human subjects research” as defined by the relevant US
statutes (45 CFR part 46), this research was reviewed and approved by the IRB at
UNC-Chapel Hill.</p></div><div id="S11" class="sec"><h3 id="S11title">GWAS Genotyping</h3><p id="P18" class="p p-first-last">Individual genotyping was conducted by Perlegen Sciences (Mountain View,
CA, USA) using three genotyping chips – Affymetrix 500K
“A” chipset (<em>Nsp I</em> and <em>Sty I</em>
chips), Santa Clara, CA, USA) (<a href="#R30" rid="R30" class=" bibr popnode">30</a>) and a
custom 164K chip created by Perlegen (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313836">31</a>)
to provide additional genome coverage. Genotype calls were generated with a
proprietary Perlegen algorithm (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313898">32</a>)
applied to the .cel files from cases and controls together. The Perlegen calling
algorithm shows excellent agreement with genotype calls from BRLMM (<a href="#R33" rid="R33" class=" bibr popnode">33</a>) in a subset of the controls used in
this study (see below) as well as in genotyping of 270 HapMap samples as part of
GAIN (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313899">34</a>) <sup><sup><a href="#FN4" rid="FN4" class=" fn">†</a></sup></sup>. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods Part
1</a> provides additional details.</p></div><div id="S12" class="sec"><h3 id="S12title">Genotyping Quality Control</h3><p id="P19" class="p p-first">There were 500,568 SNPs on the Affymetrix 500K chips and 164,871 on the
Perlegen custom chip (665,439 SNPs in total). Of these, 157,048 SNPs failed
Perlegen’s quality control (20.2% of the Affymetrix 500K and
33.9% of the Perlegen 164K SNPs). The QC process is detailed below
(<a href="#R35" rid="R35" class=" bibr popnode">35</a>).</p><p id="P20">First, the Perlegen genotyping algorithm yielded a quality score for
each individual genotype; after inspection of the quality score distributions, a
more stringent quality score cutoff (≥7) than that originally applied by
Perlegen was used.</p><p id="P21">Second, duplicate agreement was investigated. DNA samples from cases and
controls were on 18 × 96 well plates. Two samples from each plate were
chosen at random and included a second time on that plate (from the same DNA
aliquot) for a total of 36 pairs of samples that were genotyped twice, 18 pairs
for cases and 18 pairs for controls. The proportion of SNPs with non-missing
genotype calls that disagreed in these duplicated samples was 0.00291 for cases
and 0.00339 for controls.</p><p id="P22">Third, in order to investigate the impact of the genotype calling
algorithm and genotyping facility, Perlegen genotype calls were directly
compared to BRLMM (<a href="#R33" rid="R33" class=" bibr popnode">33</a>) genotype calls. Of
the controls used here, 277 individuals were independently genotyped using the
Affymetrix 500K “A” chips at the MIT/Harvard Broad Institute by
Dr. Pamela Sklar <sup><sup><a href="#FN5" rid="FN5" class=" fn">‡</a></sup></sup>.
There were 370,704 pairs of SNPs (168,299,616 genotypes) that passed quality
control at both genotyping centers. The proportion of genotypes that were called
at both centers was 0.98407 and the proportion of disagreements among called
genotypes was 0.00731. These results suggest that the impact of calling
algorithm and genotyping facility was not substantial.</p><p id="P23">Fourth, the genotyping data were used to select the final set of
subjects for analysis. The identity-by-state matrix for all autosomal genotypes
for all pairwise combinations of subjects was generated using PLINK (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313872">36</a>). Four pairs of cases were cryptic
duplicates and one member of each pair was removed. The numbers of heterozygous
and homozygous genotypes on chrX (excluding the pseudo-autosomal regions) and
chrY for each subject were counted and compared to phenotypic sex – 21
subjects (3 cases and 18 controls) with sex discrepancies were removed. Twelve
subjects (9 cases and 3 controls) were removed for excessive missing genotypes
(≥0.05). After application of these filters, there were 1,471 subjects
(738 cases and 733 controls).</p><p id="P24" class="p p-last">Fifth, the final set of SNPs was selected for analysis. Beginning with
the 508,286 SNPs delivered by Perlegen, 76 duplicated SNPs, 3,803 SNPs with
minor allele frequencies &lt;0.01, 9,279 SNPs with missingness
&gt;0.05362 (the 99.5<sup>th</sup> percentile in this dataset), and 2,272
SNPs that led to ≥2 disagreements in 36 pairs of duplicated samples were
removed. SNPs were not excluded based solely on deviations from Hardy-Weinberg
Equilibrium (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313905">37</a>) given the ancestries of
the subjects and as there are informative reasons for departures from HWE (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313925">38</a>). After application of these filters,
there were 492,900 SNPs for analysis (44 SNPs were excluded for multiple
reasons). We investigated batch effects and found none.</p></div><div id="S13" class="sec"><h3 id="S13title">Control of Population Stratification</h3><p id="P25" class="p p-first">Cases and controls are of mixed ancestry. Using the US census
“race” categories, 56.3% of subjects described
themselves as only White, 29.6% as only Black/African-American, and
14.1% selected some other or more than one racial category (i.e.,
American Indian/Alaska Native, Asian, Black/African American, Native
Hawaiian/Other Pacific Islander; White, and Hispanic/Latino). GWAS in a sample
of mixed ancestry can yield false positive findings due to population
stratification if there are sufficiently large differences in the phenotype and
genotype distributions within the strata defined by ancestry (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313892">39</a>, <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313889">40</a>).</p><p id="P26">There were two critical decisions in regard to the analysis of these
GWAS data. First, rather than stratifying the analysis by self-reported
"race" (an imperfect proxy for ancestry) (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313891">41</a>), the entire cohort was analyzed together in order to
maximize statistical power. Second, the available statistical methods for
population stratification control were evaluated. At least five major methods
for population stratification control have been described – genomic
control (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313933">42</a>), structured association
(<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313864">43</a>), principal components (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313873">44</a>), multi-dimensional scaling (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313931">36</a>), and partial least squares relating
phenotype to ancestry-informative markers (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313934">45</a>, <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313859">46</a>). These methods were
evaluated extensively using these GWAS data – see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods Part
2</a> and Lee et al. (in preparation) for details. Briefly, the central
criteria in evaluating these methods were control of Type 1 error while
preserving statistical power. To avoid bias, these comparisons were based on
lists of p-values alone and were blinded to all SNP rs numbers and annotation
information.</p><p id="P27" class="p p-last">The consensus among the statistical geneticists involved in these
evaluations was to use principal components (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313833">44</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313893">47</a>) in order robustly to
control for stratification effects while preserving statistical power. The
consensus was to use seven principal components as covariates in all logistic
regression models (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods Part 2</a>). Moreover, as has been reported
elsewhere (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313828">18</a>, <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313831">44</a>), the principal components method can capture both
subtle and extensive variation due to both genomic and experimental features
and, with the availability of &gt;10<sup>5</sup> genetic markers,
self-reported “race” is no longer required as a proxy for
ancestry (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313875">41</a>). The principal components
method is computationally efficient and uses the genotyping matrix to infer
continuous axes of genetic variation (eigenvectors) which then serve as
covariates. The potential weakness of this method is lowered statistical power
in some genomic regions – if a SNP is highly stratified, then it could
contribute strongly to a principal component and induce collinearity. This
concern is mitigated by the fact that large numbers of SNPs contribute to each
principal component. All autosomal GWAS SNPs were used as input to EigenSoft
<sup><sup><a href="#FN6" rid="FN6" class=" fn">†</a></sup></sup> (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313863">44</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313894">47</a>) and default parameters were used except that the outlier removal
option was turned off in order to generate estimates for all subjects.</p></div><div id="S14" class="sec"><h3 id="S14title">Single Marker Analyses</h3><p id="P28" class="p p-first-last">All autosomal and pseudo-autosomal SNPs that passed quality control
checks were tested for association with SCZ using logistic regression under a
log-additive mode of inheritance (1 df, each SNP coded as the number of minor
alleles). All regression models included the first seven principal components as
covariates in order to adjust for stratification and any artifacts detectible in
the GWAS data. Of the different inheritance models that could be considered,
log-additive model appear to minimize the number of comparisons while detecting
effects under a range of unknown inheritance models (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313924">48</a>). For chrX SNPs, a sex term was added as a covariate and
the number of copies of the minor allele were coded as 0 or 2 for males and 0,
1, or 2 for females. Analyses of chrY SNPs were necessarily limited to
males.</p></div><div id="S15" class="sec"><h3 id="S15title">Control of False Discoveries</h3><p id="P29" class="p p-first-last">Given the ~500,000 statistical comparisons in a GWAS, highly
significant findings (p&lt;10<sup>−6</sup>) are expected by chance.
To control the risk of false discoveries, q-values (<a href="#R49" rid="R49" class=" bibr popnode">49</a>, <a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313853">50</a>) were
computed for all p-values for single-marker tests of association. A q-value is
an estimate of the proportion of false discoveries among all significant
markers. In other words, a q-value is the false discovery rate (FDR) for the
corresponding p-value. As argued elsewhere (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313844">51</a>, <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313921">52</a>), the use of q-values
is preferable to more traditional multiple testing controls because q-values: a)
provide a better balance between the competing goals of finding true positives
versus controlling false discoveries; b) allow more similar comparisons across
studies because proportions of false discoveries are much less dependent on the
number of tests conducted; c) are relatively robust against the effects of
correlated tests (<a href="#R49" rid="R49" class=" bibr popnode">49</a>, <a href="#R53" rid="R53" class=" bibr popnode">53</a>–<a href="#R60" rid="R60" class=" bibr popnode">60</a>); and d) provide a more nuanced picture regarding the possible
relevance of the tested markers rather than an all-or-nothing conclusion about
whether a study produces significant results. The q-value threshold for
declaring significance was 0.10 – i.e., the top 10% of the
significant findings are, on average, allowed to be false discoveries (see
references (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313910">51</a>, <a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313861">56</a>) for a thorough consideration of these issues). FDR
thresholds &lt;0.10 result in a disproportional drop in power to detect
true effects.</p></div><div id="S16" class="sec"><h3 id="S16title">Multi-Marker Analyses</h3><p id="P30" class="p p-first-last">The score test of Schaid et al. (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313824">61</a>) implemented in the R (<a href="#R62" rid="R62" class=" bibr popnode">62</a>)
(v2.5.1) function "haplo.score" (v1.3.1) <sup><sup><a href="#FN7" rid="FN7" class=" fn">†</a></sup></sup> was used for multi-marker
association analyses. This test uses unphased genotypes and probabilistic
haplotype assignments in order to estimate haplotype-phenotype association
within a given genomic region. Overlapping sliding windows over the genome (500
kb segments plus 200 kb segments centered on boundaries between the 500 kb
blocks) were used to estimate linkage disequilibrium (LD) and to form haplotypes
via the Gabriel method (<a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313896">63</a>) implemented
in HaploView (<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313835">64</a>). LD blocks were
estimated in European subjects, the predominant ancestry group in this sample.
The resulting 74,411 haplotypes have a median length of 8.2 kb and
inter-quartile range of 2.7–20.0 kb. These haplotypes contained 335,539
SNPs (69.23% of all autosomal SNPs) and covered 1.368 mb (47.78%
of the autosomal genome). As haplotype frequencies differ by ancestry (<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313871">65</a>), an assumption of haplo.score –
that all subgroups share the same haplotype frequencies – was violated.
Therefore, GWAS multi-marker analyses were run in separate subgroups, and a
combined test statistic is reported.</p></div><div id="S17" class="sec"><h3 id="S17title">Statistical Power</h3><p id="P31" class="p p-first-last">Quanto (<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313940">66</a>, <a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313895">67</a>) was used to generate an illustrative approximation of
statistical power. An FDR framework was used to determine the α level
– assuming that ~25 of 492,900 GWAS SNPs have true effects
(i.e., the proportion of SNPs without true effects p<sub>0</sub>=0.99995),
two-tailed α=4.4 × 10<sup>−6</sup>. Additional
assumptions were: 738 cases and 733 controls, lifetime morbid risk of SCZ of
0.0072 (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313825">1</a>, <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313902">2</a>), and a log additive genetic model. For statistical power of 0.80
(β=0.20), the minimum detectible genotypic relative risks are 1.83,
1.56, and 1.50 for minor allele frequencies of 0.10, 0.25, and 0.40. While the
logistic model employed in our study includes seven EigenSoft principal
components as covariates, the power analyses remain approximately correct as the
additional covariates subtract only a very modest number of degrees of freedom
from the test statistic (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>). Moreover, the use of principal components
could have reduced power if they accounted for variance due to case-control
status.</p></div><div id="S18" class="sec"><h3 id="S18title">Software</h3><p id="P32" class="p p-first-last">PLINK (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313874">36</a>), SAS (v9.1.3) (<a href="#R68" rid="R68" class=" bibr popnode">68</a>), JMP (<a href="#R69" rid="R69" class=" bibr popnode">69</a>), and UCSC Genome Graphs (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313936">70</a>) were used for data management, quality control, statistical
analyses, and graphics. Statistical analyses conducted using PLINK (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313884">36</a>) and the SAS LOGISTIC procedure (<a href="#R68" rid="R68" class=" bibr popnode">68</a>) produced identical results.</p></div><div id="S19" class="sec"><h3 id="S19title">Bioinformatics</h3><p id="P33" class="p p-first">All genomic locations are based on NCBI Build 35 (<a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313869">71</a>) (UCSC hg17) (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313837">70</a>). Pseudo-autosomal region 1 (PAR1) is assumed to be located on
chrX:1–2,692,881 and chrY:1–2,692,881 and PAR2 on
chrX:154,494,747-154,824,264 and chrY:57,372,174-57,701,691 (<a href="#R72" rid="R72" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313843">72</a>). SNP annotations were created using the
TAMAL database (<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313829">73</a>) based chiefly on UCSC
genome browser files (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313915">70</a>), HapMap (<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313941">65</a>), and dbSNP (<a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313865">71</a>) using a set of SAS programs. These annotation files are
available for download and there is a searchable repository for genomewide
studies in neuropsychiatry at the same site <sup><sup><a href="#FN8" rid="FN8" class=" fn">†</a></sup></sup>. All gene names referenced are the standard
names set by the HUGO Gene Nomenclature Committee.</p><p id="P34" class="p p-last">Fisher’s method (<a href="#R74" rid="R74" class=" bibr popnode">74</a>) was
used to combine lists of p-values for the same SNPs (e.g., from different GWAS)
or for the same haplotypes (i.e., from AFR and EUR subjects in this GWAS). The
test statistic for <em>K</em> independent p-values is. <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" overflow="scroll"><msubsup><mi>χ</mi><mrow><mn>2</mn><mi>K</mi></mrow><mn>2</mn></msubsup><mo>=</mo><mo>−</mo><mn>2</mn><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></munderover><msub><mtext>log</mtext><mi>e</mi></msub><mo stretchy="false">(</mo><msub><mi>p</mi><mi>i</mi></msub><mo stretchy="false">)</mo></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" overflow="scroll"><msubsup><mi>χ</mi><mrow><mn>2</mn><mi>K</mi></mrow><mn>2</mn></msubsup><mo>=</mo><mo>−</mo><mn>2</mn><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></munderover><msub><mtext>log</mtext><mi>e</mi></msub><mo stretchy="false">(</mo><msub><mi>p</mi><mi>i</mi></msub><mo stretchy="false">)</mo></math></script>.</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Project Context</h3><p id="P35" class="p p-first-last">A list of completed and proposed genotyping projects using the CATIE
samples is available on-line <sup><sup><a href="#FN9" rid="FN9" class=" fn">‡</a></sup></sup>. Future manuscripts using these GWAS data will
investigate copy number variation and genetic influence on treatment response,
tardive dyskinesia, and neurocognition. The NIMH controls are being used in
multiple other projects.</p></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Results</h2><div id="S22" class="sec sec-first"><h3 id="S22title">Sample Description</h3><p id="P36" class="p p-first-last"><a href="/pmc/articles/PMC3910086/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> presents descriptive data
for cases and controls. Cases and controls were well-matched for age and
ancestry but not for sex (due to insufficient numbers of African-American males
in the control pool as described in the Methods). There were large differences
between cases and controls in education, marital status, and employment
consistent with the adverse effects of a chronic mental illness with onset in
early adulthood. Cases had been ill for a mean of 14 years and the mean PANSS
scores (<a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313904">75</a>) are consistent with a
moderately ill sample. As described previously (<a href="#R76" rid="R76" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313841">76</a>), CATIE subjects who provided DNA samples had lower symptom
severity (PANSS total 74 vs. 77), lesser current drug/alcohol abuse/dependence
(29% vs. 36%) and less likely to describe themselves as
African-American (29% vs. 40%) in comparison to CATIE subjects
who did not provide a DNA sample.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P79">Descriptive data for cases with schizophrenia and controls included in
GWAS.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876098371904" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Subject Descriptor</th><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th><th rowspan="1" colspan="1" align="left">Test</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">Number of subjects genotyped</td><td rowspan="1" colspan="1" align="left">738</td><td rowspan="1" colspan="1" align="left">733</td><td rowspan="1" colspan="1" align="left">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Mean age in years (SD) <sup>†</sup></td><td rowspan="1" colspan="1" align="left">40.9 (11.1)</td><td rowspan="1" colspan="1" align="left">41.0 (11.6)</td><td rowspan="1" colspan="1" align="left"><em>F</em><sub>1,1469</sub> =0.09 p=0.77</td></tr><tr><td rowspan="1" colspan="1" align="left">Proportion male (N) <sup>†</sup></td><td rowspan="1" colspan="1" align="left">0.74 (544)</td><td rowspan="1" colspan="1" align="left">0.67 (493)</td><td rowspan="1" colspan="1" align="left"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>7.37</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>=</mo><mn>0.007</mn></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>7.37</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>=</mo><mn>0.007</mn></math></script></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Ancestry proportions (inferred from
self-report) <sup>†</sup><br>&nbsp;&nbsp;African only
(N)<br>&nbsp;&nbsp;European only
(N)<br>&nbsp;&nbsp;Other (N)</td><td rowspan="1" colspan="1" valign="middle" align="left"><br>0.29 (217)<br>0.57
(417)<br>0.14 (104)</td><td rowspan="1" colspan="1" valign="middle" align="left"><br>0.30 (219)<br>0.56
(411)<br>0.14 (103)</td><td rowspan="1" colspan="1" valign="middle" align="left"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" overflow="scroll"><msubsup><mi>χ</mi><mn>2</mn><mn>2</mn></msubsup><mo>=</mo><mn>0.04</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>=</mo><mn>0.98</mn></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M4" overflow="scroll"><msubsup><mi>χ</mi><mn>2</mn><mn>2</mn></msubsup><mo>=</mo><mn>0.04</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>=</mo><mn>0.98</mn></math></script>
</td></tr><tr><td rowspan="1" colspan="1" align="left">Proportion with high school degree or more (N)</td><td rowspan="1" colspan="1" align="left">0.74 (543)</td><td rowspan="1" colspan="1" align="left">0.93 (684)</td><td rowspan="1" colspan="1" align="left"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>100</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>100</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></script></td></tr><tr><td rowspan="1" colspan="1" align="left">Proportion married (N)</td><td rowspan="1" colspan="1" align="left">0.11 (78)</td><td rowspan="1" colspan="1" align="left">0.57 (415)</td><td rowspan="1" colspan="1" align="left"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>350</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>350</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></script></td></tr><tr><td rowspan="1" colspan="1" align="left">Proportion employed (N)</td><td rowspan="1" colspan="1" align="left">0.06 (43)</td><td rowspan="1" colspan="1" align="left">0.75 (548)</td><td rowspan="1" colspan="1" align="left"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>727</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M7" overflow="scroll"><msubsup><mi>χ</mi><mn>1</mn><mn>2</mn></msubsup><mo>=</mo><mn>727</mn><mo>,</mo><mi mathvariant="normal">p</mi><mo>&lt;</mo><mn>0.0001</mn></math></script></td></tr><tr><td rowspan="1" colspan="1" align="left">Mean years since first antipsychotic prescribed</td><td rowspan="1" colspan="1" align="left">14.2 (10.8)</td><td rowspan="1" colspan="1" align="left">-</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Mean PANSS total score</td><td rowspan="1" colspan="1" align="left">74.0 (17.2)</td><td rowspan="1" colspan="1" align="left">-</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Mean PANSS positive symptom score</td><td rowspan="1" colspan="1" align="left">17.8 (5.5)</td><td rowspan="1" colspan="1" align="left">-</td><td rowspan="1" colspan="1" align="left"></td></tr><tr><td rowspan="1" colspan="1" align="left">Mean PANSS negative symptom score</td><td rowspan="1" colspan="1" align="left">19.9 (6.4)</td><td rowspan="1" colspan="1" align="left">-</td><td rowspan="1" colspan="1" align="left"></td></tr></tbody></table></div><div id="largeobj_idm139876098371904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3910086/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>†</sup>Matching variable.</div></div></div></div><div id="S23" class="sec"><h3 id="S23title">SNP Description</h3><p id="P37" class="p p-first-last">The analysis SNP set had 492,900 SNPs including 484,664 autosomal SNPs,
8,084 SNPs on chrX, 143 SNPs on chrXY/PAR1, 9 SNPs on chrY, and 0 SNPs on PAR2
and chrM (mitochondrial DNA). <a href="/pmc/articles/PMC3910086/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>
depicts summary missingness and MAF estimates from the GWAS analysis set in the
entire sample and separately for cases and controls; on average across the
genome, SNP missingness appears to be acceptably low. The average marker density
over the genome was 1 SNP every 6.2 kb (=3,077,088,087 bases / 492,900 SNPs).
The median inter-marker distance was 2.8 kb with an interquartile range
0.8–7.1 kb and the 99<sup>th</sup> percentile was 39.7 kb.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="P81">Distribution descriptors for missingness and MAF in GWAS SNP set <sup>†</sup>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876054806288" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="center">Property</th><th rowspan="1" colspan="1" align="center">SNP Location</th><th rowspan="1" colspan="1" align="center">All subjects</th><th rowspan="1" colspan="1" align="center">Cases</th><th rowspan="1" colspan="1" align="center">Controls</th></tr></thead><tbody><tr><td rowspan="4" colspan="1" valign="top" align="center">Missingness</td><td rowspan="1" colspan="1" valign="top" align="center">chr1-22</td><td rowspan="1" colspan="1" valign="top" align="center">0.0048 (0.0014–0.0116)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0041 (0.0014–0.0108)</td><td rowspan="1" colspan="1" valign="top" align="center">0.0041 (0.0014–0.0123)</td></tr><tr><td rowspan="1" colspan="1" align="center">chrX</td><td rowspan="1" colspan="1" align="center">0.0082 (0.0027–0.0177)</td><td rowspan="1" colspan="1" align="center">0.0081 (0.0027–0.0163)</td><td rowspan="1" colspan="1" align="center">0.0082 (0.0027–0.0177)</td></tr><tr><td rowspan="1" colspan="1" align="center">PAR1</td><td rowspan="1" colspan="1" align="center">0.0082 (0.0048–0.0180)</td><td rowspan="1" colspan="1" align="center">0.0081 (0.0041–0.0149)</td><td rowspan="1" colspan="1" align="center">0.0082 (0.0041–0.0177)</td></tr><tr><td rowspan="1" colspan="1" align="center">chrY <sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.0010 (0.0005–0.0039)</td><td rowspan="1" colspan="1" align="center">0.0000 (0.0000–0.0018)</td><td rowspan="1" colspan="1" align="center">0.0020 (0.0000–0.0071)</td></tr><tr><td rowspan="4" colspan="1" valign="top" align="center">MAF</td><td rowspan="1" colspan="1" valign="top" align="center">chr1-22</td><td rowspan="1" colspan="1" valign="top" align="center">0.212 (0.102–0.347)</td><td rowspan="1" colspan="1" valign="top" align="center">0.213 (0.103–0.347)</td><td rowspan="1" colspan="1" valign="top" align="center">0.212 (0.100–0.346)</td></tr><tr><td rowspan="1" colspan="1" align="center">chrX</td><td rowspan="1" colspan="1" align="center">0.241 (0.121–0.371)</td><td rowspan="1" colspan="1" align="center">0.241 (0.122–0.369)</td><td rowspan="1" colspan="1" align="center">0.240 (0.119–0.370)</td></tr><tr><td rowspan="1" colspan="1" align="center">PAR1</td><td rowspan="1" colspan="1" align="center">0.240 (0.073–0.372)</td><td rowspan="1" colspan="1" align="center">0.241 (0.071–0.375)</td><td rowspan="1" colspan="1" align="center">0.244 (0.073–0.374)</td></tr><tr><td rowspan="1" colspan="1" align="center">chrY <sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.218 (0.055–0.462)</td><td rowspan="1" colspan="1" align="center">0.252 (0.052–0.465)</td><td rowspan="1" colspan="1" align="center">0.181 (0.057–0.456)</td></tr></tbody></table></div><div id="largeobj_idm139876054806288" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3910086/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>†</sup>Data are medians (25<sup>th</sup>-75th percentiles). MAF=minor
allele frequency.</div><div id="TFN3"><sup>‡</sup>Males only.</div></div></div></div><div id="S24" class="sec"><h3 id="S24title">Single Marker Association Tests</h3><p id="P38" class="p p-first">We used logistic regression to test for association of the 492,900 SNPs
in the GWAS dataset with case/control status (with seven principal components
(<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313854">44</a>) included as covariates to
account for population stratification). The minimum p-value obtained was 1.71
× 10<sup>−6</sup>. There were 26,738 p-values &lt;0.05
– six p-values in the 10<sup>−6</sup> bin, 42 in the
10<sup>−5</sup> bin, 486 in the 10<sup>−4</sup> bin, 4,845
in the 10<sup>−3</sup> bin, and 21,359 in the interval
[0.01–0.05). The GWAS results are depicted in <a href="/pmc/articles/PMC3910086/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>. Panel A shows the QQ plot (<a href="#R77" rid="R77" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313870">77</a>) for GWAS for case-control status. The QQ plot suggests
that the observed p-values do not strongly depart from the p-value distribution
expected by chance. Panel B shows the
−log<sub>10</sub>(<em>p</em>) for the 26,738 p-values
&lt; 0.05 (5.425%) in the context of the human genome in order to
make spatial clustering evident. To facilitate comparisons with other datasets,
inclusion in meta-analyses, and further investigation of these findings, all
p-values are available in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a> and the data used to create <a href="/pmc/articles/PMC3910086/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1b</span></a> are contained in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>.
SNPs that failed quality control are contained in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3910086/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876098374768" class="figure"><a href="/pmc/articles/PMC3910086/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3910086_nihms319115f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms319115f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3910086/bin/nihms319115f1.jpg"></a></div><div id="largeobj_idm139876098374768" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3910086/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3910086/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="P78">Results of single-marker association tests of case-control status for
492,900 SNPs. Panel A. shows the QQ plot (77) of the observed p-values,
−log<sub>10</sub> (<em>p</em>), versus those expected by
chance, <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" overflow="scroll"><mo>−</mo><msub><mtext>log</mtext><mn>10</mn></msub><mo>(</mo><mfrac><mi>i</mi><mrow><mi>L</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>)</mo></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" overflow="scroll"><mo>−</mo><msub><mtext>log</mtext><mn>10</mn></msub><mo>(</mo><mfrac><mi>i</mi><mrow><mi>L</mi><mo>+</mo><mn>1</mn></mrow></mfrac><mo>)</mo></math></script>, where p is the asymptotic p-value from the additive test that
the SNP coefficient is zero, i is the rank for each SNP p-value (1=smallest,
L=largest), and L is the number of SNPs. The dashed lines show the expected 95%
probability interval for ordered p-values. Panel B depicts
−log<sub>10</sub> (<em>p</em>) for the 26,738 SNPs with
p-values &lt; 0.05 in the context of the human genome in order to make
spatial clustering more evident.</p></div></div></div><p id="P39">For an FDR threshold of 0.10, the proportion of all SNPs without true
effects (p<sub>0</sub>) can be estimated from the GWAS results (<a href="#R60" rid="R60" class=" bibr popnode">60</a>) and was found to be p<sub>0</sub>=
0.9999904. This result is consistent with the presence of ~5 SNPs with
true effects in these GWAS data for SCZ. Q-values were calculated for all
p-values under the conservative assumption that p<sub>0</sub>=1, and no SNPs
reached genomewide significance as the minimum q-values of 0.45 (rs10911902 and
rs16977195) did not exceed the pre-specified FDR threshold of 0.10 (<a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Nearly identical results were
obtained using p<sub>0</sub>=0.9999904 (data not shown).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="P84">Top 25 results from GWAS for SCZ in the CATIE study <sup>†</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876054408512" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP data</th><th rowspan="1" colspan="1" align="center"></th><th rowspan="1" colspan="1" align="right"></th><th rowspan="1" colspan="1" align="left">Gene data</th><th colspan="7" rowspan="1" align="center">Logistic regression tests for
association</th><th colspan="4" rowspan="1" align="center">Allele frequency data</th><th colspan="3" rowspan="1" align="center">HapMap Allele FreqsComments <sup>‡</sup></th><th rowspan="1" colspan="1" align="center">Context</th><th rowspan="1" colspan="1" align="center"></th><th colspan="3" rowspan="1" align="center">SNP annotations</th><th rowspan="1" colspan="1" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">SNP ID</td><td rowspan="1" colspan="1" align="center">Chromosome</td><td rowspan="1" colspan="1" align="right">Position</td><td rowspan="1" colspan="1" align="left">Transcripts in which SNP locate</td><td rowspan="1" colspan="1" align="center">Odds Ratio</td><td rowspan="1" colspan="1" align="center">Wald test</td><td rowspan="1" colspan="1" align="center">p-value</td><td rowspan="1" colspan="1" align="center">q-value</td><td rowspan="1" colspan="1" align="center">Rank</td><td rowspan="1" colspan="1" align="center">AFR odds ratio</td><td rowspan="1" colspan="1" align="center">EUR odds ratio</td><td rowspan="1" colspan="1" align="center">Cases</td><td rowspan="1" colspan="1" align="center">Controls</td><td rowspan="1" colspan="1" align="center">CATIE AFR</td><td rowspan="1" colspan="1" align="center">CATIE EUR</td><td rowspan="1" colspan="1" align="center">AFR/YRI</td><td rowspan="1" colspan="1" align="center">EUR/CEU</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">SNPs with p&lt;0.0</td><td rowspan="1" colspan="1" align="center">Proxy SNP ≤0.00</td><td rowspan="1" colspan="1" align="center">CNV</td><td rowspan="1" colspan="1" align="center">Conserved p95</td><td rowspan="1" colspan="1" align="center">Regulatory potential</td><td rowspan="1" colspan="1" align="center">Transfactor binding site</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4846033</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">11,722,830</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.348</td><td rowspan="1" colspan="1" align="center">−4.594</td><td rowspan="1" colspan="1" align="center">4.36E-06</td><td rowspan="1" colspan="1" align="center">0.56</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">0.401</td><td rowspan="1" colspan="1" align="center">0.146</td><td rowspan="1" colspan="1" align="center">0.025</td><td rowspan="1" colspan="1" align="center">0.053</td><td rowspan="1" colspan="1" align="center">0.103</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.009</td><td rowspan="1" colspan="1" align="center">0.179</td><td rowspan="1" colspan="1" align="center">0.009</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">Isolated finding</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10911902</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">183,363,974</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.560</td><td rowspan="1" colspan="1" align="center">−4.770</td><td rowspan="1" colspan="1" align="center">1.85E-06</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">0.627</td><td rowspan="1" colspan="1" align="center">0.523</td><td rowspan="1" colspan="1" align="center">0.099</td><td rowspan="1" colspan="1" align="center">0.171</td><td rowspan="1" colspan="1" align="center">0.046</td><td rowspan="1" colspan="1" align="center">0.173</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.200</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2 SNPs, 0.4 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">Yes</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9309325</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">60,492,482</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.727</td><td rowspan="1" colspan="1" align="center">−4.076</td><td rowspan="1" colspan="1" align="center">4.58E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">0.704</td><td rowspan="1" colspan="1" align="center">0.770</td><td rowspan="1" colspan="1" align="center">0.382</td><td rowspan="1" colspan="1" align="center">0.455</td><td rowspan="1" colspan="1" align="center">0.414</td><td rowspan="1" colspan="1" align="center">0.414</td><td rowspan="1" colspan="1" align="center">0.433</td><td rowspan="1" colspan="1" align="center">0.458</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">6 SNPs, 7 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1569351</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">60,287,457</td><td rowspan="1" colspan="1" align="left">FHIT</td><td rowspan="1" colspan="1" align="center">0.697</td><td rowspan="1" colspan="1" align="center">−4.225</td><td rowspan="1" colspan="1" align="center">2.39E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">0.609</td><td rowspan="1" colspan="1" align="center">0.729</td><td rowspan="1" colspan="1" align="center">0.337</td><td rowspan="1" colspan="1" align="center">0.415</td><td rowspan="1" colspan="1" align="center">0.173</td><td rowspan="1" colspan="1" align="center">0.470</td><td rowspan="1" colspan="1" align="center">0.192</td><td rowspan="1" colspan="1" align="center">0.433</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">12 SNPs, 31 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs4568102</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">72,070,679</td><td rowspan="1" colspan="1" align="left"></td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">4.118</td><td rowspan="1" colspan="1" align="center">4.103</td><td rowspan="1" colspan="1" align="center">4.08E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">19</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">4.947</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.663</td><td rowspan="1" colspan="1" align="center">0.035</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.008</td><td rowspan="1" colspan="1" align="center">0.062</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.003</td><td rowspan="1" colspan="1" align="center">0.117</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">1, 2</td><td rowspan="1" colspan="1" align="center">2 SNPs, 5 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1380272</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">21,431,975</td><td rowspan="1" colspan="1" align="left">KCNIP4</td><td rowspan="1" colspan="1" align="center">0.522</td><td rowspan="1" colspan="1" align="center">−4.397</td><td rowspan="1" colspan="1" align="center">1.10E-05</td><td rowspan="1" colspan="1" align="center">0.74</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">0.425</td><td rowspan="1" colspan="1" align="center">0.610</td><td rowspan="1" colspan="1" align="center">0.057</td><td rowspan="1" colspan="1" align="center">0.107</td><td rowspan="1" colspan="1" align="center">0.074</td><td rowspan="1" colspan="1" align="center">0.088</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.102</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">4 SNPs, 18 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs1495716</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="right">177,103,725</td><td rowspan="1" colspan="1" align="left">GPM6A</td><td rowspan="1" colspan="1" align="center">1.423</td><td rowspan="1" colspan="1" align="center">4.223</td><td rowspan="1" colspan="1" align="center">2.41E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="center">1.915</td><td rowspan="1" colspan="1" align="center">1.277</td><td rowspan="1" colspan="1" align="center">0.365</td><td rowspan="1" colspan="1" align="center">0.296</td><td rowspan="1" colspan="1" align="center">0.208</td><td rowspan="1" colspan="1" align="center">0.391</td><td rowspan="1" colspan="1" align="center">0.142</td><td rowspan="1" colspan="1" align="center">0.425</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2 SNPs, 8 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9295938</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">31,061,084</td><td rowspan="1" colspan="1" align="left">C6orf205</td><td rowspan="1" colspan="1" align="center">1.642</td><td rowspan="1" colspan="1" align="center">4.138</td><td rowspan="1" colspan="1" align="center">3.51E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">1.288</td><td rowspan="1" colspan="1" align="center">1.769</td><td rowspan="1" colspan="1" align="center">0.143</td><td rowspan="1" colspan="1" align="center">0.097</td><td rowspan="1" colspan="1" align="center">0.072</td><td rowspan="1" colspan="1" align="center">0.143</td><td rowspan="1" colspan="1" align="center">0.033</td><td rowspan="1" colspan="1" align="center">0.192</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">7 SNPs, 17 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs9400690</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">114,453,545</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">1.534</td><td rowspan="1" colspan="1" align="center">4.052</td><td rowspan="1" colspan="1" align="center">5.08E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">24</td><td rowspan="1" colspan="1" align="center">1.588</td><td rowspan="1" colspan="1" align="center">1.635</td><td rowspan="1" colspan="1" align="center">0.193</td><td rowspan="1" colspan="1" align="center">0.151</td><td rowspan="1" colspan="1" align="center">0.057</td><td rowspan="1" colspan="1" align="center">0.230</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.267</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3 SNPs, 6 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs16917897</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">52,856,709</td><td rowspan="1" colspan="1" align="left"><a id="__tag_352314001" class="tag_hotlink" href="/nuccore/D45864" ref="/nuccore/D45864"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"D45864","term_id":"1255601","term_text":"D45864"}}</span>D45864</a> PRKG1 <a id="__tag_352314002" class="tag_hotlink" href="/nuccore/Y07512" ref="/nuccore/Y07512"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"Y07512","term_id":"31708","term_text":"Y07512"}}</span>Y07512</a> BC062</td><td rowspan="1" colspan="1" align="center">2.750</td><td rowspan="1" colspan="1" align="center">4.078</td><td rowspan="1" colspan="1" align="center">4.55E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="center">2.864</td><td rowspan="1" colspan="1" align="center">1.650</td><td rowspan="1" colspan="1" align="center">0.049</td><td rowspan="1" colspan="1" align="center">0.019</td><td rowspan="1" colspan="1" align="center">0.099</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.005</td><td rowspan="1" colspan="1" align="center">0.119</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">1, 6</td><td rowspan="1" colspan="1" align="center">Isolated finding</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs297257</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="right">127,298,704</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.689</td><td rowspan="1" colspan="1" align="center">−4.129</td><td rowspan="1" colspan="1" align="center">3.64E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">0.727</td><td rowspan="1" colspan="1" align="center">0.705</td><td rowspan="1" colspan="1" align="center">0.452</td><td rowspan="1" colspan="1" align="center">0.498</td><td rowspan="1" colspan="1" align="center">0.805</td><td rowspan="1" colspan="1" align="center">0.318</td><td rowspan="1" colspan="1" align="center">0.883</td><td rowspan="1" colspan="1" align="center">0.367</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3 SNPs, 12 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9512730</td><td rowspan="1" colspan="1" valign="middle" align="center">13</td><td rowspan="1" colspan="1" valign="middle" align="right">26,975,144</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">1.515</td><td rowspan="1" colspan="1" valign="middle" align="center">4.586</td><td rowspan="1" colspan="1" valign="middle" align="center">4.52E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">0.56</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">1.724</td><td rowspan="1" colspan="1" valign="middle" align="center">1.385</td><td rowspan="1" colspan="1" valign="middle" align="center">0.274</td><td rowspan="1" colspan="1" valign="middle" align="center">0.211</td><td rowspan="1" colspan="1" valign="middle" align="center">0.212</td><td rowspan="1" colspan="1" valign="middle" align="center">0.262</td><td rowspan="1" colspan="1" valign="middle" align="center">0.229</td><td rowspan="1" colspan="1" valign="middle" align="center">0.263</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="2" colspan="1" valign="middle" align="center">6 SNPs, 38 kb</td><td rowspan="2" colspan="1" valign="middle" align="center">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs942348</td><td rowspan="1" colspan="1" valign="middle" align="center">13</td><td rowspan="1" colspan="1" valign="middle" align="right">26,976,147</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">0.718</td><td rowspan="1" colspan="1" valign="middle" align="center">−4.194</td><td rowspan="1" colspan="1" valign="middle" align="center">2.74E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">1.641</td><td rowspan="1" colspan="1" valign="middle" align="center">1.399</td><td rowspan="1" colspan="1" valign="middle" align="center">0.451</td><td rowspan="1" colspan="1" valign="middle" align="center">0.521</td><td rowspan="1" colspan="1" valign="middle" align="center">0.311</td><td rowspan="1" colspan="1" valign="middle" align="center">0.547</td><td rowspan="1" colspan="1" valign="middle" align="center">0.220</td><td rowspan="1" colspan="1" valign="middle" align="center">0.500</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" valign="middle" align="center">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17070578</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="right">78,431,983</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">1.903</td><td rowspan="1" colspan="1" align="center">4.069</td><td rowspan="1" colspan="1" align="center">4.72E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">23</td><td rowspan="1" colspan="1" align="center">1.560</td><td rowspan="1" colspan="1" align="center">1.589</td><td rowspan="1" colspan="1" align="center">0.080</td><td rowspan="1" colspan="1" align="center">0.052</td><td rowspan="1" colspan="1" align="center">0.029</td><td rowspan="1" colspan="1" align="center">0.092</td><td rowspan="1" colspan="1" align="center">0.008</td><td rowspan="1" colspan="1" align="center">0.117</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">3 SNPs, 9 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17095545</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">58,555,364</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">1.648</td><td rowspan="1" colspan="1" align="center">4.204</td><td rowspan="1" colspan="1" align="center">2.62E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">1.501</td><td rowspan="1" colspan="1" align="center">1.746</td><td rowspan="1" colspan="1" align="center">0.153</td><td rowspan="1" colspan="1" align="center">0.099</td><td rowspan="1" colspan="1" align="center">0.205</td><td rowspan="1" colspan="1" align="center">0.094</td><td rowspan="1" colspan="1" align="center">0.217</td><td rowspan="1" colspan="1" align="center">0.100</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Isolated finding</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs7144633</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="right">58,582,146</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">1.784</td><td rowspan="1" colspan="1" align="center">4.309</td><td rowspan="1" colspan="1" align="center">1.64E-05</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">1.246</td><td rowspan="1" colspan="1" align="center">1.987</td><td rowspan="1" colspan="1" align="center">0.117</td><td rowspan="1" colspan="1" align="center">0.076</td><td rowspan="1" colspan="1" align="center">0.095</td><td rowspan="1" colspan="1" align="center">0.102</td><td rowspan="1" colspan="1" align="center">0.100</td><td rowspan="1" colspan="1" align="center">0.100</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Isolated finding</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs16977195</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="right">84,785,244</td><td rowspan="1" colspan="1" align="left">AGBL1</td><td rowspan="1" colspan="1" align="center">6.005</td><td rowspan="1" colspan="1" align="center">4.785</td><td rowspan="1" colspan="1" align="center">1.71E-06</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2.028</td><td rowspan="1" colspan="1" align="center">6.801</td><td rowspan="1" colspan="1" align="center">0.035</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.006</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.003</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.017</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">2 SNPs, 0.6 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs234993</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">20,555,145</td><td rowspan="1" colspan="1" align="left">ACSM1 BUCS1</td><td rowspan="1" colspan="1" align="center">0.524</td><td rowspan="1" colspan="1" align="center">−4.036</td><td rowspan="1" colspan="1" align="center">5.44E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">25</td><td rowspan="1" colspan="1" align="center">0.263</td><td rowspan="1" colspan="1" align="center">0.596</td><td rowspan="1" colspan="1" align="center">0.049</td><td rowspan="1" colspan="1" align="center">0.091</td><td rowspan="1" colspan="1" align="center">0.034</td><td rowspan="1" colspan="1" align="center">0.083</td><td rowspan="1" colspan="1" align="center">0.017</td><td rowspan="1" colspan="1" align="center">0.042</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">2 SNPs, 0.4 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs151222</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="right">20,581,993</td><td rowspan="1" colspan="1" align="left">ACSM1 BUCS1</td><td rowspan="1" colspan="1" align="center">0.477</td><td rowspan="1" colspan="1" align="center">−4.524</td><td rowspan="1" colspan="1" align="center">6.08E-06</td><td rowspan="1" colspan="1" align="center">0.57</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">0.310</td><td rowspan="1" colspan="1" align="center">0.549</td><td rowspan="1" colspan="1" align="center">0.045</td><td rowspan="1" colspan="1" align="center">0.093</td><td rowspan="1" colspan="1" align="center">0.038</td><td rowspan="1" colspan="1" align="center">0.080</td><td rowspan="1" colspan="1" align="center">0.017</td><td rowspan="1" colspan="1" align="center">0.042</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">3 SNPs, 2 kb</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs17455133</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">49,719,513</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.643</td><td rowspan="1" colspan="1" align="center">−4.142</td><td rowspan="1" colspan="1" align="center">3.44E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">0.357</td><td rowspan="1" colspan="1" align="center">0.713</td><td rowspan="1" colspan="1" align="center">0.138</td><td rowspan="1" colspan="1" align="center">0.211</td><td rowspan="1" colspan="1" align="center">0.090</td><td rowspan="1" colspan="1" align="center">0.216</td><td rowspan="1" colspan="1" align="center">0.075</td><td rowspan="1" colspan="1" align="center">0.233</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">19 SNPs, 60 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs2824301</td><td rowspan="1" colspan="1" align="center">21</td><td rowspan="1" colspan="1" align="right">17,738,145</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.720</td><td rowspan="1" colspan="1" align="center">−4.103</td><td rowspan="1" colspan="1" align="center">4.09E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">1.917</td><td rowspan="1" colspan="1" align="center">1.386</td><td rowspan="1" colspan="1" align="center">0.421</td><td rowspan="1" colspan="1" align="center">0.484</td><td rowspan="1" colspan="1" align="center">0.267</td><td rowspan="1" colspan="1" align="center">0.525</td><td rowspan="1" colspan="1" align="center">0.192</td><td rowspan="1" colspan="1" align="center">0.552</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">6 SNPs, 47 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs10521865</td><td rowspan="1" colspan="1" align="center">X</td><td rowspan="1" colspan="1" align="right">146,501,918</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.550</td><td rowspan="1" colspan="1" align="center">−4.125</td><td rowspan="1" colspan="1" align="center">3.70E-05</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">0.809</td><td rowspan="1" colspan="1" align="center">0.304</td><td rowspan="1" colspan="1" align="center">0.028</td><td rowspan="1" colspan="1" align="center">0.078</td><td rowspan="1" colspan="1" align="center">0.013</td><td rowspan="1" colspan="1" align="center">0.071</td><td rowspan="1" colspan="1" align="center">0.000</td><td rowspan="1" colspan="1" align="center">0.067</td><td rowspan="1" colspan="1" align="center">1, 3, 4, 6</td><td rowspan="1" colspan="1" align="center">Isolated finding</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">No</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2159767</td><td rowspan="1" colspan="1" valign="middle" align="center">X</td><td rowspan="1" colspan="1" valign="middle" align="right">147,086,567</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">0.752</td><td rowspan="1" colspan="1" valign="middle" align="center">−4.496</td><td rowspan="1" colspan="1" valign="middle" align="center">6.94E-06</td><td rowspan="1" colspan="1" valign="middle" align="center">0.57</td><td rowspan="1" colspan="1" valign="middle" align="center">6</td><td rowspan="1" colspan="1" valign="middle" align="center">0.288</td><td rowspan="1" colspan="1" valign="middle" align="center">0.753</td><td rowspan="1" colspan="1" valign="middle" align="center">0.430</td><td rowspan="1" colspan="1" valign="middle" align="center">0.549</td><td rowspan="1" colspan="1" valign="middle" align="center">0.274</td><td rowspan="1" colspan="1" valign="middle" align="center">0.618</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" valign="middle" align="center">0.144</td><td rowspan="1" colspan="1" valign="middle" align="center">0.678</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="2" colspan="1" valign="middle" align="center">2 SNPs 13 kb</td><td rowspan="2" colspan="1" valign="middle" align="center">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" valign="middle" align="center">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2536589</td><td rowspan="1" colspan="1" valign="middle" align="center">X</td><td rowspan="1" colspan="1" valign="middle" align="right">147,099,127</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">0.757</td><td rowspan="1" colspan="1" valign="middle" align="center">−4.377</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21E-05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.74</td><td rowspan="1" colspan="1" valign="middle" align="center">8</td><td rowspan="1" colspan="1" valign="middle" align="center">0.285</td><td rowspan="1" colspan="1" valign="middle" align="center">0.762</td><td rowspan="1" colspan="1" valign="middle" align="center">0.412</td><td rowspan="1" colspan="1" valign="middle" align="center">0.529</td><td rowspan="1" colspan="1" valign="middle" align="center">0.291</td><td rowspan="1" colspan="1" valign="middle" align="center">0.639</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" valign="middle" align="center">0.156</td><td rowspan="1" colspan="1" valign="middle" align="center">0.678</td><td rowspan="1" colspan="1" valign="middle" align="center">5</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" valign="middle" align="center">Yes</td><td rowspan="1" colspan="1" valign="middle" align="center">no</td></tr><tr><td rowspan="1" colspan="1" align="left">rs952515</td><td rowspan="1" colspan="1" align="center">X</td><td rowspan="1" colspan="1" align="right">147,142,998</td><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="center">0.759</td><td rowspan="1" colspan="1" align="center">−4.321</td><td rowspan="1" colspan="1" align="center">1.56E-05</td><td rowspan="1" colspan="1" align="center">0.81</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">0.337</td><td rowspan="1" colspan="1" align="center">0.711</td><td rowspan="1" colspan="1" align="center">0.424</td><td rowspan="1" colspan="1" align="center">0.546</td><td rowspan="1" colspan="1" align="center">0.273</td><td rowspan="1" colspan="1" align="center">0.625</td><td style="background-color:#FFFF99" rowspan="1" colspan="1" align="center">0.144</td><td rowspan="1" colspan="1" align="center">0.678</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">7 SNPs, 24 kb</td><td rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td><td rowspan="1" colspan="1" align="center">no</td><td style="background-color:#99FFCC" rowspan="1" colspan="1" align="center">Yes</td><td rowspan="1" colspan="1" align="center">no</td></tr></tbody></table></div><div id="largeobj_idm139876054408512" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3910086/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><sup>†</sup>All allele frequency data were standardized to the same reference
alleles. Yellow boxes highlight issues of concern. Although SNPs with
overall MAF&lt;0.01 were dropped In the QC process, subgroups may have
MAF&lt;0.01. Some MAF&gt;0.5 as the reference allele may be
different in subgroups. EUR=European ancestry, AFR=African ancestry.</div><div id="TFN5"><sup>‡</sup>1=rare allele, 2=direction of effects different in AFR and EUR
subgroups, 3=low mean SNP quality scores, 4=greater SNP missingness in cases
than controls, 5=frequency difference of &gt;10% versus HapMap
6=suboptimal clustering of genotype groups on manual inspection of
scatterplots</div></div></div><p id="P40">Additional data about the SNPs with the 25 smallest p-values are shown
in <a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>. First, the allele frequency
differences between CATIE AFR and EUR subjects were generally similar (median
difference 0.09, inter-quartile range 0.05–0.25) suggesting that one
precondition for population stratification artifacts was not universally met.
Moreover, the SNP with the greatest MAF difference (rs297257,
|MAF<sub>AFR</sub>-MAF<sub>EUR</sub>|=0.49) yielded similar
odds ratio estimates in AFR and EUR subjects (0.73 and 0.71) despite the allele
frequency difference. Second, critically, the odds ratios in <a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> were generally of similar direction
and magnitude for the overall sample (including seven principal components) as
well as in logistic regressions without covariates for AFR and EUR subjects
separately (note that significance tests were not conducted for the stratified
logistic regressions). There was one exception as noted in <a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> (rs4568102). These findings support
the approach to population stratification control used here, and suggest that
the overall odds ratios were generally unlikely to be caused by stratification
artifacts despite different allele frequencies in subjects of EUR and AFR
ancestries.</p><p id="P41">Third, SNP allele frequencies in CATIE were reassuringly similar to
those in the HapMap panels with the possible exception of three chrX SNPs in
CATIE AFR subjects. Fourth, some results in <a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> may be problematical due to lack of robustness of odds ratio
estimates for uncommon alleles (rs4846033, rs4568102, rs16917897, rs16977195,
rs10521865), sub-optimal performance of the calling algorithm upon manual review
(rs1380272, rs16917897, and rs17070578), and differential missingness in cases
and controls (rs10521865). The existence of a high LD “proxy”
SNP that also shows association reduces the chance of a false positive due to
genotyping artifact.</p><p id="P42">Fifth, most of the top 25 findings were not located within the
transcript of a known gene and, for the 11 genes listed in <a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, searches of PubMed and SZGene
identified no published studies of schizophrenia although prior linkage studies
implicated these genomic regions. Sixth, two pairs of SNPs in the top 25 were
located near one another and two additional SNPs clustered with over 10 SNPs
that were nominally significant and in relatively close proximity. Finally,
<a href="/pmc/articles/PMC3910086/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> also presents SNP
annotations; two SNPs were in copy number variant regions, one SNP was modestly
conserved, one SNP was in a predicted transfactor binding site, and 17 of the
top 25 SNPs were predicted to be in regions with regulatory potential.</p><p id="P43" class="p p-last">Spatial clustering of SNPs with significant associations may be an
indication that a genomic region does not represent a spurious finding,
particularly if the degree of linkage disequilibrium is not very high. As a
descriptive approach prior to multi-marker analyses, SNP clusters with p-values
&lt;0.05 and with distances to the next significant marker ≤15 kb
were identified. For the 26,738 SNPs with p-values &lt;0.05, 11,638
occurred in isolation, and there were 4,427 clusters containing 2–4
SNPs, 550 clusters containing 5–9 SNPs, 60 clusters containing
10–14 SNPs, and 12 clusters containing 15–22 SNPs. Additional
data about the latter set of clusters (15–22 SNPs, p&lt;0.05, and
inter-marker distances of ≤15 kb) are shown in <a href="/pmc/articles/PMC3910086/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>. The overlap of the genes in these regions with
genes previously studied for SCZ is limited to a single study of
<em>FMO3</em> (<a href="#R78" rid="R78" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313903">78</a>). Several
regions would appear intriguing, particularly chr7:31,322,276-31,375,664 and
chr18:49,714,826-49,774,618.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="P87">Clusters containing ≥15 SNPs (all p&lt;0.05 and
inter-marker distances of &lt;15 kb).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876048788320" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="center">Start</th><th rowspan="1" colspan="1" valign="middle" align="center">End</th><th rowspan="1" colspan="1" valign="middle" align="center">Distance</th><th rowspan="1" colspan="1" valign="middle" align="center">SNPs in<br>cluster (all
p&lt;0.05)</th><th rowspan="1" colspan="1" valign="middle" align="center">SNPs<br>with<br>p&lt;0.001</th><th rowspan="1" colspan="1" valign="middle" align="center">Minimum p</th><th rowspan="1" colspan="1" valign="middle" align="center">RefSeq genes</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="right">167,784,811</td><td rowspan="1" colspan="1" align="right">167,850,858</td><td rowspan="1" colspan="1" align="right">66,047</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">1.9E–03</td><td rowspan="1" colspan="1" align="left"><em>FMO3 FMO6</em></td></tr><tr><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="right">10,878,985</td><td rowspan="1" colspan="1" align="right">10,902,509</td><td rowspan="1" colspan="1" align="right">23,524</td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">2.4E-02</td><td rowspan="1" colspan="1" align="left"><em>PDIA6</em></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="right">4,012,045</td><td rowspan="1" colspan="1" align="right">4,070,654</td><td rowspan="1" colspan="1" align="right">58,609</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">5.1E-04</td><td rowspan="1" colspan="1" align="left"><em>PECI C6orf146 C6orf201</em></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6</td><td rowspan="1" colspan="1" valign="middle" align="right">30,052,958</td><td rowspan="1" colspan="1" valign="middle" align="right">30,154,225</td><td rowspan="1" colspan="1" valign="middle" align="right">101,267</td><td rowspan="1" colspan="1" valign="middle" align="center">22</td><td rowspan="1" colspan="1" valign="middle" align="center">0</td><td rowspan="1" colspan="1" valign="middle" align="center">8.4E-03</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>HLA-A29.1 HCG9 ZNRD1 PPP1R11
RNF39</em></td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="right">30,429,340</td><td rowspan="1" colspan="1" align="right">30,500,227</td><td rowspan="1" colspan="1" align="right">70,887</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">3.3E-03</td><td rowspan="1" colspan="1" align="left">-</td></tr><tr><td rowspan="1" colspan="1" align="left">6</td><td rowspan="1" colspan="1" align="right">32,447,818</td><td rowspan="1" colspan="1" align="right">32,497,626</td><td rowspan="1" colspan="1" align="right">49,808</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">4.2E-03</td><td rowspan="1" colspan="1" align="left"><em>BTNL2</em></td></tr><tr><td rowspan="1" colspan="1" align="left">7</td><td rowspan="1" colspan="1" align="right">31,322,276</td><td rowspan="1" colspan="1" align="right">31,375,664</td><td rowspan="1" colspan="1" align="right">53,388</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">1.4E-04</td><td rowspan="1" colspan="1" align="left"><em>LOC223075 CCDC129</em></td></tr><tr><td rowspan="1" colspan="1" align="left">11</td><td rowspan="1" colspan="1" align="right">126,178,796</td><td rowspan="1" colspan="1" align="right">126,246,694</td><td rowspan="1" colspan="1" align="right">67,898</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">5.9E-03</td><td rowspan="1" colspan="1" align="left"><em>KIRREL3</em></td></tr><tr><td rowspan="1" colspan="1" align="left">16</td><td rowspan="1" colspan="1" align="right">80,722,907</td><td rowspan="1" colspan="1" align="right">80,789,170</td><td rowspan="1" colspan="1" align="right">66,263</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">6.6E-04</td><td rowspan="1" colspan="1" align="left"><em>MPHOSPH6</em></td></tr><tr><td rowspan="1" colspan="1" align="left">18</td><td rowspan="1" colspan="1" align="right">49,714,826</td><td rowspan="1" colspan="1" align="right">49,774,618</td><td rowspan="1" colspan="1" align="right">59,792</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="center">3.4E-05</td><td rowspan="1" colspan="1" align="left">-</td></tr><tr><td rowspan="1" colspan="1" align="left">21</td><td rowspan="1" colspan="1" align="right">24,343,408</td><td rowspan="1" colspan="1" align="right">24,426,048</td><td rowspan="1" colspan="1" align="right">82,640</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">1.2E-03</td><td rowspan="1" colspan="1" align="left">-</td></tr><tr><td rowspan="1" colspan="1" align="left">21</td><td rowspan="1" colspan="1" align="right">43,307,905</td><td rowspan="1" colspan="1" align="right">43,360,473</td><td rowspan="1" colspan="1" align="right">52,568</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">5.8E-03</td><td rowspan="1" colspan="1" align="left"><em>PKNOX1 CBS</em></td></tr></tbody></table></div><div id="largeobj_idm139876048788320" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3910086/table/T4/?report=objectonly">Open in a separate window</a></div></div></div><div id="S25" class="sec"><h3 id="S25title">Positive Controls &amp; Bioinformatic Comparisons</h3><p id="P44" class="p p-first">It is highly desirable to compare GWAS findings to “positive
controls” (i.e., genomic regions with very strong prior evidence for
association like <em>APOE*ε4</em> and Alzheimer’s
disease (<a href="#R79" rid="R79" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313857">79</a>) or
<em>FTO</em>/intron 1 and body mass index (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313876">15</a>)). At the time of this writing in 10/2007, there are no
such regions for SCZ although there are a few genomic regions with multiple
positive but inconsistent findings.</p><p id="P45">First, the GWAS platform used here had at least one SNP in 25,287
“known genes” (<a href="#R80" rid="R80" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313883">80</a>). The
median number of SNPs per gene was 4, inter-quartile range 2–11, and
range 1–1,088. The number of SNPs per gene is strongly related to gene
size (Spearman ρ=0.79) meaning that larger genes have more SNPs on
average.</p><p id="P46">Second, the GWAS findings were compared to a list of 525 candidate genes
for SCZ that had been investigated at least once in a published report
<sup><sup><a href="#FN10" rid="FN10" class=" fn">†</a></sup></sup>. This list
includes studies with positive or negative findings and thus represents
candidate genes that at least one set of investigators believed to be relevant
to the etiology of SCZ. The number of studies per gene varied widely –
from 302 genes studied in single reports to genes that had been studied
&gt;50 times (52 reports for <em>HTR2A</em>, 57 reports for
<em>DRD2</em>, 65 reports for <em>DRD3</em>, and 68 reports
for <em>COMT</em>). In the GWAS results, there were 534 SNPs with
p-values &lt;0.001 (an arbitrary choice) that implicated 249 genes (one SNP
could be located in multiple genes) – of these, only 6 (2.41%)
had been the subject of a prior study (<em>ARMC3, CACNA1A, FEZ1, NRG1,
PIWIL2</em>, and <em>VDR</em>). Although a GWAS platform can
provide very good or excellent coverage over the genome on average, there may be
important candidate genes with sub-optimal SNP coverage. Of the 525 candidate
genes previously studied in the literature, 84 genes had no SNPs that passed
quality control, including eight genes that had been studied in ≥10
reports (<em>APOE, CNTF, CYP2D6, DRD1, DRD4, GRIN1, TH</em>, and
<em>TNF</em>).</p><p id="P47">Third, we focused more closely on a consensus set of 15 candidate genes
for SCZ with the best evidence of association – 12 candidate genes were
selected from a review (<a href="#R8" rid="R8" class=" bibr popnode">8</a>),
<em>CSF2RA</em> and <em>IL3RA</em> were from a published SCZ
GWAS (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313856">23</a>), and <em>PLXNA2</em>
was from a large-scale candidate gene study for SCZ (<a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313942">81</a>) (<a href="/pmc/articles/PMC3910086/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). Of
the 249 genes with one or more p-values &lt;0.001 in this GWAS, only 1 was
on this list of 15 candidate genes (<em>NRG1</em>). Although the GWAS
platform we used had generally good coverage across the genome (1 SNP/6.2 kb on
average), six of these 15 genes had inadequate coverage and nine genes had SNP
densities better than the GWAS average (2.2–5.0 SNPs/kb). For this
subset of the GWAS data, the proportion of SNPs without true effects
(p<sub>0</sub>) (<a href="#R60" rid="R60" class=" bibr popnode">60</a>) was estimated at
0.997 for an FDR threshold of 0.10 and the minimum q-value was 0.30. In
comparison to all GWAS SNPs, these results suggest that these candidate regions
may be enriched for genetic variants influencing susceptibility to SCZ.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="P88">GWAS results for a selected set of candidate genes for SCZ.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139876053652304" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Adequate<br>coverage</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene<br>name</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene<br>product</th><th rowspan="1" colspan="1" valign="middle" align="center">Gene<br>size</th><th rowspan="1" colspan="1" valign="middle" align="center">Size<br>percentile</th><th rowspan="1" colspan="1" valign="middle" align="center">Number<br>of SNPs</th><th rowspan="1" colspan="1" valign="middle" align="center">SNP<br>Density<br>(kb/SNP)</th><th rowspan="1" colspan="1" valign="middle" align="center">P-values<br>&lt;0.05</th><th rowspan="1" colspan="1" valign="middle" align="center">P-values<br>&lt;0.001</th><th rowspan="1" colspan="1" valign="middle" align="center">Minimum<br>p-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>AKT1</em></td><td rowspan="1" colspan="1" align="left">v-akt murine thymoma oncogene homolog 1</td><td rowspan="1" colspan="1" align="center">24,250</td><td rowspan="1" colspan="1" align="center">53</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">12.1</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.0316</td></tr><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>CSF2RA</em></td><td rowspan="1" colspan="1" align="left">colony stimulating factor 2 receptor, alpha</td><td rowspan="1" colspan="1" align="center">41,089</td><td rowspan="1" colspan="1" align="center">68</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>IL3RA</em></td><td rowspan="1" colspan="1" align="left">interleukin 3 receptor, alpha</td><td rowspan="1" colspan="1" align="center">46,220</td><td rowspan="1" colspan="1" align="center">71</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">46.2</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.41</td></tr><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>PRODH</em></td><td rowspan="1" colspan="1" align="left">proline dehydrogenase 1</td><td rowspan="1" colspan="1" align="center">23,772</td><td rowspan="1" colspan="1" align="center">53</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">7.9</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.023</td></tr><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>RGS4</em></td><td rowspan="1" colspan="1" align="left">regulator of G-protein signaling 4</td><td rowspan="1" colspan="1" align="center">7,228</td><td rowspan="1" colspan="1" align="center">26</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">7.2</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1" align="left">No</td><td rowspan="1" colspan="1" align="left"><em>ZDHHC8</em></td><td rowspan="1" colspan="1" align="left">zinc finger, DHHC-type containing 8</td><td rowspan="1" colspan="1" align="center">16,165</td><td rowspan="1" colspan="1" align="center">43</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td><td rowspan="1" colspan="1" align="center">-</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>COMT</em></td><td rowspan="1" colspan="1" align="left">catechol-O-methyltransferase</td><td rowspan="1" colspan="1" align="center">27,222</td><td rowspan="1" colspan="1" align="center">56</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">3.0</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.016</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>DAOA</em></td><td rowspan="1" colspan="1" align="left">D-amino acid oxidase activator</td><td rowspan="1" colspan="1" align="center">25,168</td><td rowspan="1" colspan="1" align="center">54</td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1" align="center">2.8</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.12</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>DISC1</em></td><td rowspan="1" colspan="1" align="left">disrupted in schizophrenia 1</td><td rowspan="1" colspan="1" align="center">414,456</td><td rowspan="1" colspan="1" align="center">98</td><td rowspan="1" colspan="1" align="center">99</td><td rowspan="1" colspan="1" align="center">4.2</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.0011</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>DRD3</em></td><td rowspan="1" colspan="1" align="left">dopamine receptor D3</td><td rowspan="1" colspan="1" align="center">50,200</td><td rowspan="1" colspan="1" align="center">72</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">3.9</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>DTNBP1</em></td><td rowspan="1" colspan="1" align="left">dystrobrevin binding protein 1</td><td rowspan="1" colspan="1" align="center">140,231</td><td rowspan="1" colspan="1" align="center">91</td><td rowspan="1" colspan="1" align="center">28</td><td rowspan="1" colspan="1" align="center">5.0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.064</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>HTR2A</em></td><td rowspan="1" colspan="1" align="left">serotonin receptor 2A</td><td rowspan="1" colspan="1" align="center">62,663</td><td rowspan="1" colspan="1" align="center">77</td><td rowspan="1" colspan="1" align="center">29</td><td rowspan="1" colspan="1" align="center">2.2</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.15</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>NRG1</em></td><td rowspan="1" colspan="1" align="left">neuregulin 1</td><td rowspan="1" colspan="1" align="center">1,124,806</td><td rowspan="1" colspan="1" align="center">99</td><td rowspan="1" colspan="1" align="center">293</td><td rowspan="1" colspan="1" align="center">3.8</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">0.00091</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>PLXNA2</em></td><td rowspan="1" colspan="1" align="left">plexin A2</td><td rowspan="1" colspan="1" align="center">216,898</td><td rowspan="1" colspan="1" align="center">95</td><td rowspan="1" colspan="1" align="center">70</td><td rowspan="1" colspan="1" align="center">3.1</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.013</td></tr><tr><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="left"><em>SLC6A4</em></td><td rowspan="1" colspan="1" align="left">serotonin transporter</td><td rowspan="1" colspan="1" align="center">37,800</td><td rowspan="1" colspan="1" align="center">65</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">4.7</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">0.52</td></tr></tbody></table></div><div id="largeobj_idm139876053652304" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3910086/table/T5/?report=objectonly">Open in a separate window</a></div></div><p id="P48">None of the results in <a href="/pmc/articles/PMC3910086/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>
survive FDR multiple comparison correction. Nonetheless, we investigated a few
of these results further (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 7</a>). Of the 146 SNPs in the vicinity of
<em>DISC1</em> (minimum p-value 0.001), the significant findings
clustered around the chr1 (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313930">1</a>;<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313918">11</a>) (q42;q14.3) translocation break point
(<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313927">13</a>) as follows: rs2738875 (p=0.001)
– 5.0 kb – translocation break point – 3.0 kb –
rs11122342 (p=0.62) – 0.6 kb – rs6672782 (p=0.016) – 0.1
kb – start of haplotype HEP1 – 0.8 kb – rs11588937
(p=0.83) – 8.6 kb – end of haplotype HEP1 – 8.0 kb
– rs12744978 (p=0.048). These data are not conclusive, but the locations
of the significant <em>DISC1</em> SNPs coincide very closely with the
<em>DISC1</em> breakpoint (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313868">13</a>) and the HEP1 haplotype implicated in the etiology of SCZ (<a href="#R82" rid="R82" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313922">82</a>) and reduced prefrontal cortex gray
matter density (<a href="#R83" rid="R83" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313840">83</a>). Of the 401 SNPs in
the vicinity of <em>NRG1</em> (minimum p-value 0.0009), there were 15
SNPs (6 with p&lt;0.05 including the most significant <em>NRG1</em>
SNP, rs16879809) in a cluster at the 3’ end of <em>NRG1</em>.
This cluster was 875 kb from the 5’ portion of NRG1 that been of
particular interest (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313888">9</a>, <a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313929">84</a>, <a href="#R85" rid="R85" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313948">85</a>). Three of 53 SNPs in the “HapIce” haplotype
(<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313830">9</a>) had p-values between
0.01–0.05. The SNP rs6994992 (<a href="#R84" rid="R84" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313923">84</a>,
<a href="#R85" rid="R85" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313852">85</a>) was not genotyped in this GWAS
but had been done in these samples previously: rs6994992 was weakly associated
with SCZ but in the opposite direction than has been reported (<a href="#R86" rid="R86" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313906">86</a>). Of the 13 SNPs in the vicinity of
<em>COMT</em>, two consecutive exonic SNPs 1 kb apart both had
p=0.02. These were rs4633 (synonymous) and rs4680 (val158met) which has been
widely studied as a genetic risk factor for SCZ and other disorders (<a href="#R87" rid="R87" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313882">87</a>). Finally, of the 84 SNPs in the
vicinity of <em>PLXNA2</em>, the significant SNPs were 80 kb away from
those highlighted in the initial report (<a href="#R81" rid="R81" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313867">81</a>) and whose most notable SNP (rs752016) was not significant in
this GWAS (p=0.90). It is important to stress that none of these findings met
genome-wide significance and the overlap with prior studies could have been
merely due to chance.</p><p id="P49" class="p p-last">Fourth, these findings were compared with the findings of two published
GWAS. The Affymetrix 500K “A” chips were also used in these
studies facilitating direct comparisons. For a SCZ GWAS (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313842">23</a>), the list of p-values was not available from the
authors but the most significant association was for the PAR1 SNP rs4129148 (P=
3.7×10<sup>−7</sup>) (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313879">23</a>). This finding was not replicated in the CATIE GWAS (p=0.41).
Indeed, only 5 of 143 SNPs in the pseudo-autosomal region had association
p-values &lt;0.05 (none within 600 kb of rs4129148) and there were no
pseudo-autosomal region p-values &lt;0.001. In comparing the CATIE GWAS
with the Wellcome Trust bipolar disorder GWAS (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313834">18</a>), the QQ plot for the 370,216 common to both studies is shown in
<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure
8</a>. This comparison was motivated by suspicions that SCZ and bipolar
disorder might share etiological genetic risk factors; however, there was no
substantial divergence of the observed from the expected p-value
distributions.</p></div><div id="S26" class="sec sec-last"><h3 id="S26title">Multi-Marker Analyses</h3><p id="P50" class="p p-first-last">The QQ plot for combined p-values from genomewide multi-marker analyses
conducted on AFR and EUR subjects is shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 9</a>;
there was no marked deviation of the observed from the expected p-value
distributions. Moreover, there was no overlap of the 12 genes listed in <a href="/pmc/articles/PMC3910086/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a> or the 525 genes that have been
investigated in studies of SCZ with the 100 haplotypes with p&lt;0.001.</p></div></div><div id="S27" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S27title">Discussion</h2><p id="P51" class="p p-first">The present genomewide association study (GWAS) for schizophrenia (SCZ) had
multiple notable features. It is the second published GWAS for SCZ, and the first
for which all individual genotype and phenotype data were made available to the
research community under an “open source” philosophy.</p><p id="P52">Moreover, this project was conceptualized from the beginning as a two-stage
study (<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313901">56</a>, <a href="#R88" rid="R88" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313848">88</a>). The present report constitutes the first stage and, as anticipated,
there are no “slam-dunk” findings that meet genomewide significance;
however, it is very possible that there exist true findings in these results that
may not be impressive in any single study but that only emerge by comparing multiple
large GWAS.</p><p id="P53">It is hoped that the process of gene-finding for SCZ will mirror that of
type 2 diabetes mellitus (T2DM). In 2006, three genes had accumulated strong
evidence in support of association – after the publication of six T2DM GWAS
in 2007 (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313890">18</a>–<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313938">22</a>, <a href="#R89" rid="R89" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313850">89</a>), there are now
eight (and perhaps as many as 11) genes with highly compelling support (<a href="#R90" rid="R90" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313945">90</a>). Notably, several of the initial T2DM GWAS
had QQ plots very similar to that in <a href="/pmc/articles/PMC3910086/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1a</span></a>
(<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313914">19</a>, <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313860">20</a>) and only after “aggressive data sharing” (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313907">22</a>) across studies and additional genotyping in
thousands of additional samples did multiple high confidence findings emerge (<a href="#R90" rid="R90" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313912">90</a>). Indeed, some of the findings that
eventually proved to be highly significant had initial p-value ranks in the hundreds
or even thousands.</p><p id="P54">Therefore, these data from the CATIE GWAS will be part of an inclusive
meta-analysis of individual phenotype and genotype data from all available SCZ GWAS
that will be done in Q1 2008 under the auspices of the Psychiatric GWAS Consortium.
It is anticipated that there will be well over 10,000 cases plus controls available
for meta-analysis – particularly large-scale projects include the GAIN SCZ
samples (<a href="#R91" rid="R91" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313886">91</a>) and a consortium lead by Dr.
Pamela Sklar at the Broad Institute). Drs. van den Oord and Sullivan hope to conduct
large-scale Stage 2 genotyping to confirm and refine results from the SCZ
meta-analysis. The CATIE samples are being used for deep re-sequencing of candidate
genes under an award from the Medical Re-sequencing/Allelic Spectrum project
supported by the NHGRI and can be used to discover variants not previously
identified.</p><p id="P55">Additional notable features of this project include that, at the time
genotyping was conducted, the GWAS platform used here had the best genomic coverage
for common genomic variation with r<sup>2</sup>≥0.8 for 86% of
genome in subjects with European ancestry, 79% for East Asian ancestry, and
49% for African ancestry (<a href="#R92" rid="R92" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313911">92</a>). The
CATIE sample was ascertained from diverse sites across the continental United States
in order to accrue a “real-world” sample of patients in treatment
for chronic SCZ (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313900">27</a>) and a rich set of
phenotypes are available for all CATIE subjects – i.e., treatment response
in a randomized, double-blind clinical trial (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313926">24</a>), multiple assessments of tardive dyskinesia (<a href="#R93" rid="R93" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313827">93</a>), and repeated assessments of neurocognition (<a href="#R94" rid="R94" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313880">94</a>).</p><p id="P56">The principal finding of the present study was that no SNP or multi-marker
combination of SNPs achieved genomewide significance. Moreover, there was no
important overlap of our findings with those from published GWAS for SCZ (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313885">23</a>) or bipolar disorder (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313947">18</a>). However, some findings with uncorrected p-values
&lt;0.05 overlapped with regions of <em>DISC1</em> and
<em>COMT</em> that have been highlighted in prior studies. Of the 146
SNPs in the vicinity of <em>DISC1</em> (minimum genewise p-value 0.001), the
significant findings clustered around the chr1 translocation break point (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313878">13</a>) and the HEP1haplotype implicated in the
etiology of SCZ (<a href="#R82" rid="R82" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313877">82</a>) and associated with
reduced prefrontal cortex gray matter density (<a href="#R83" rid="R83" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313849">83</a>). This observation is bedeviling – these statistical signals
could easily be due to chance (even the best finding would not survive multiple
comparison correction for SNPs in <em>DISC1</em> alone much less for the
492,900 SNPs in this GWAS) and yet the location of the signals is intriguing.
Whether the observed overlap is merely due to chance (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313881">14</a>) or reflects genetic influences on liability to SCZ will
require the meta-analyses described above.</p><p id="P57">There are three broad explanations for the pattern of findings observed in
this SCZ GWAS. The first and most optimistic possibility is that there are true
findings relevant to the etiology of SCZ imbedded in these results but that it will
require careful and rigorous comparisons with other GWAS datasets along with
additional genotyping in new samples to delineate true from false positive
findings.</p><p id="P58">Statistical power is a particular concern. The sample size of 738 SCZ cases
and 733 group-matched controls and a genomewide set of 492,900 SNPs provided the
capacity to detect genetic effects of moderate size for reasonably common
polymorphisms (i.e., minor allele frequencies exceeding 10%). True genetic
effects influencing case-control status might not have been detected in this study
for reasons predictable from the design of this study. Non-detection could have
occurred if the genotypic effect size was below the detection threshold, if the
effect was located in a genomic region for which there was poor SNP coverage, if the
effect was a genetic variant other than a SNP (e.g., a copy number variant) and if
there was low LD with genotyped SNPs, or in the presence of excessive phenotypic or
locus heterogeneity. Additionally, it is possible that true positive findings could
have been obscured by the use of principal components analysis in this heterogeneous
sample particularly for SNPs in strongly stratified genomic regions.</p><p id="P59">The second possibility is that there are true genetic effects for SCZ but
that assumptions fundamental to GWAS are incorrect. It is possible that current
definitions of SCZ (<a href="#R28" rid="R28" class=" bibr popnode">28</a>, <a href="#R95" rid="R95" class=" bibr popnode">95</a>) lack validity. If true, attempting to identify genetic
variants associated with “caseness” in a GWAS may prove fruitless as
case classifications based on signs and symptoms are poorly correlated with genetic
etiological factors. It is possible that the fundamental model is incorrect
– GWAS are predicated under the common disease/common variant model whereby
prevalent human diseases are caused by polymorphisms of relatively modest effects.
If SCZ is caused by multiple rare variants (causation due to multiple quite uncommon
genetic variants of very strong effect) (<a href="#R96" rid="R96" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313913">96</a>)
then the GWAS design is inappropriate to the fundamental genetic phenomenon. It is
possible that liability to SCZ is mostly or entirely due to interactions between
genomic regions or between genomic regions and environmental factors and that these
must be explicitly modeled in order to be detected. Each of these instances provides
an explanation why true effects were not detected.</p><p id="P60">Finally, the least optimistic possibility is that, despite the indirect
evidence from family, adoption, and twin studies (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313946">4</a>), there exist no true genetic effects causal to SCZ. This option does
not appear consistent with a sizeable body of genetic epidemiological data; however,
data from quasi-experimental studies do not constitute proof of the involvement of
specific genetic variants in the etiopathology of SCZ.</p><p id="P61" class="p p-last">It could prove possible to determine of which of these three possibilities
are operative for SCZ on near horizon. In the meantime, SCZ genetics needs no more
false claims (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352313826">14</a>) – there is an
urgent need to identify replicable findings associated with SCZ. Consistent with
this belief, these analyses have been interpreted with caution and the individual
phenotype and genotype data made available to the scientific community <sup><sup><a href="#FN11" rid="FN11" class=" fn">†</a></sup></sup>.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>31 pages including 9 tables, methods and references</h4><div class="sup-box half_rhythm" id="d36e1295"><a href="/pmc/articles/PMC3910086/bin/NIHMS319115-supplement-31_pages_including_9_tables__methods_and_references.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(491K, doc)</sup></div></div></div><div id="S28" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S28title">Acknowledgments</h2><div class="sec"><p id="P73"><strong>Acknowledgements</strong></p><p id="P74">Dr. Sullivan was supported by R01s <a id="__tag_801498394" class="tag_hotlink" href="/nuccore/MH074027" ref="/nuccore/MH074027"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH074027","term_id":"1389435019","term_text":"MH074027"}}</span>MH074027</a> and <a id="__tag_801498391" class="tag_hotlink" href="/nuccore/MH077139" ref="/nuccore/MH077139"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH077139","term_id":"1364874077","term_text":"MH077139"}}</span>MH077139</a>, Dr. Zou by
<a id="__tag_801498397" class="tag_hotlink" href="/nuccore/GM074175" ref="/nuccore/GM074175"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"GM074175","term_id":"222006391","term_text":"GM074175"}}</span>GM074175</a>, and Dr. Wright by P30 HD003110. The CATIE project was funded by NIMH
contract N01 MH90001. We thank Dr. Nick Patterson for assistance with the EigenSoft
program.</p><p id="P75">We are indebted to the "Molecular Genetics of Schizophrenia
II" (MGS-2) collaboration for their exceptional collegiality in making the
control samples available to the research community. Control subjects were from the
National Institute of Mental Health Schizophrenia Genetics Initiative, and
phenotypes and DNA samples were collected by the MGS-2 collaboration whose
investigators and co-investigators are: ENH/Northwestern University, Evanston, IL,
<a id="__tag_801498388" class="tag_hotlink" href="/nuccore/MH059571" ref="/nuccore/MH059571"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH059571","term_id":"1554614518","term_text":"MH059571"}}</span>MH059571</a>, Pablo V. Gejman, M.D. (Collaboration Coordinator; PI), Alan R. Sanders,
M.D.; Emory University School of Medicine, Atlanta, GA, MH59587, Farooq Amin, M.D.
(PI); Louisiana State University Health Sciences Center; New Orleans, Louisiana,
<a id="__tag_801498393" class="tag_hotlink" href="/nuccore/MH067257" ref="/nuccore/MH067257"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH067257","term_id":"1428143801","term_text":"MH067257"}}</span>MH067257</a>, Nancy Buccola APRN, BC, MSN (PI); University of California-Irvine, Irvine,
CA, MH60870, William Byerley, M.D. (PI); Washington University, St. Louis, MO, U01,
<a id="__tag_801498390" class="tag_hotlink" href="/nuccore/MH060879" ref="/nuccore/MH060879"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH060879","term_id":"1384494008","term_text":"MH060879"}}</span>MH060879</a>, C. Robert Cloninger, M.D. (PI); University of Iowa, Iowa, IA, MH59566,
Raymond Crowe, M.D. (PI), Donald Black, M.D.; University of Colorado, Denver, CO,
<a id="__tag_801498392" class="tag_hotlink" href="/nuccore/MH059565" ref="/nuccore/MH059565"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH059565","term_id":"1489134353","term_text":"MH059565"}}</span>MH059565</a>, Robert Freedman, M.D. (PI); University of Pennsylvania, Philadelphia, PA,
<a id="__tag_801498396" class="tag_hotlink" href="/nuccore/MH061675" ref="/nuccore/MH061675"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH061675","term_id":"1360457698","term_text":"MH061675"}}</span>MH061675</a>, Douglas Levinson M.D. (PI); University of Queensland, Queensland,
Australia, <a id="__tag_801498389" class="tag_hotlink" href="/nuccore/MH059588" ref="/nuccore/MH059588"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"MH059588","term_id":"1534500118","term_text":"MH059588"}}</span>MH059588</a>, Bryan Mowry, M.D. (PI); Mt. Sinai School of Medicine, New York,
NY, MH59586, Jeremy Silverman, Ph.D. (PI).</p><p id="P76"><strong>Financial Disclosures</strong></p><p id="P77">Eli Lilly funded the GWAS genotyping done at Perlegen Sciences. Dr. Sullivan
reports receiving research funding from Eli Lilly in connection with this project.
Dr. Stroup reports having received research funding from Eli Lilly and consulting
fees from Janssen Pharmaceutica, GlaxoSmithKline, and Bristol-Myers Squibb. Dr.
Lieberman reports having received research funding from AstraZeneca Pharmaceuticals,
Bristol-Myers Squibb, GlaxoSmithKline, Janssen Pharmaceutica, and Pfizer and
consulting and educational fees from AstraZeneca Pharmaceuticals, Bristol-Myers
Squibb, Eli Lilly, Forest Pharmaceuticals, GlaxoSmithKline, Janssen Pharmaceutica,
Novartis, Pfizer, and Solvay.</p></div></div><div id="idm139876057432288" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139876057432288title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"><sup>†</sup><a href="http://slep.unc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://slep.unc.edu</a> (accessed 28JUN2007)</p><p class="fn sec" id="FN2"><sup>‡</sup>Â‡ <a href="http://www.schizophreniaforum.org/res/sczgene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.schizophreniaforum.org/res/sczgene</a> (accessed
28JUN2007)</p><p class="fn sec" id="FN3"><sup>†</sup><a href="http://www.nimhgenetics.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nimhgenetics.org</a> (accessed 28JUN2007)</p><p class="fn sec" id="FN4"><sup>†</sup>Available at dbGaP, <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/sites/entrez?db=gap</a> (accessed
28JUN2007)</p><p class="fn sec" id="FN5"><sup>‡</sup><a href="http://www.nimhgenetics.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nimhgenetics.org</a> (accessed 28JUN2007)</p><p class="fn sec" id="FN6"><sup>†</sup><a href="http://genepath.med.harvard.edu/~reich/Software.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genepath.med.harvard.edu/~reich/Software.htm</a>, v1.0
(accessed 28JUN2007)</p><p class="fn sec" id="FN7"><sup>†</sup><a href="http://cancercenter.mayo.edu/mayo/research/schaid_lab/software.cfm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cancercenter.mayo.edu/mayo/research/schaid_lab/software.cfm</a>,
v1.3.1 (accessed 28JUN2007)</p><p class="fn sec" id="FN8"><sup>†</sup><a href="https://slep.unc.edu/evidence" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://slep.unc.edu/evidence</a> (downloads tab, accessed
28JUN2007)</p><p class="fn sec" id="FN9"><sup>‡</sup><a href="https://slep.unc.edu/evidence" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://slep.unc.edu/evidence</a> (downloads tab, accessed
02OCT2007)</p><p class="fn sec" id="FN10"><sup>†</sup><a href="http://www.schizophreniaforum.org/res/sczgene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.schizophreniaforum.org/res/sczgene</a> (accessed
28JUN2007)</p><p class="fn sec" id="FN11"><sup>†</sup><a href="http://www.nimhgenetics.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nimhgenetics.org</a> (accessed 28JUN2007)</p></div></div><div id="idm139876055345264" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139876055345264title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Saha S, Chant D, Welham J, McGrath J. A systematic review of the prevalence of
schizophrenia. <span><span class="ref-journal">PLoS Medicine. </span>2005;<span class="ref-vol">2</span>(5):e141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1140952/">PMC free article</a>]</span> [<a href="/pubmed/15916472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Medicine&amp;title=A+systematic+review+of+the+prevalence+of
schizophrenia&amp;author=S+Saha&amp;author=D+Chant&amp;author=J+Welham&amp;author=J+McGrath&amp;volume=2&amp;issue=5&amp;publication_year=2005&amp;pages=e141&amp;pmid=15916472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Lichtenstein P, Bjork C, Hultman CM, Scolnick EM, Sklar P, Sullivan PF. Recurrence risks for schizophrenia in a Swedish national
cohort. <span><span class="ref-journal">Psychological Medicine. </span>2006;<span class="ref-vol">36</span>:1417–1426.</span> [<a href="/pubmed/16863597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychological+Medicine&amp;title=Recurrence+risks+for+schizophrenia+in+a+Swedish+national
cohort&amp;author=P+Lichtenstein&amp;author=C+Bjork&amp;author=CM+Hultman&amp;author=EM+Scolnick&amp;author=P+Sklar&amp;volume=36&amp;publication_year=2006&amp;pages=1417-1426&amp;pmid=16863597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Gottesman II, Shields J.  <span class="ref-journal">Schizophrenia: The Epigenetic Puzzle.</span> Cambridge, UK: Cambridge University Press; 1982.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Schizophrenia:+The+Epigenetic+Puzzle&amp;author=II+Gottesman&amp;author=J+Shields&amp;publication_year=1982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Sullivan PF, Kendler KS, Neale MC. Schizophrenia as a complex trait: evidence from a meta-analysis
of twin studies. <span><span class="ref-journal">Archives of General Psychiatry. </span>2003;<span class="ref-vol">60</span>:1187–1192.</span> [<a href="/pubmed/14662550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Archives+of+General+Psychiatry&amp;title=Schizophrenia+as+a+complex+trait:+evidence+from+a+meta-analysis
of+twin+studies&amp;author=PF+Sullivan&amp;author=KS+Kendler&amp;author=MC+Neale&amp;volume=60&amp;publication_year=2003&amp;pages=1187-1192&amp;pmid=14662550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Falconer DS, Mackay TFC.  <span class="ref-journal">Introduction to Quantitative Genetics.</span> 4th ed. London: Longman Group Ltd.; 1996.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Introduction+to+Quantitative+Genetics&amp;author=DS+Falconer&amp;author=TFC+Mackay&amp;publication_year=1996&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Risch N. Linkage strategies for genetically complex traits: I. Multilocus
models. <span><span class="ref-journal">American Journal of Human Genetics. </span>1990;<span class="ref-vol">46</span>:222–228.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1684987/">PMC free article</a>]</span> [<a href="/pubmed/2301392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Linkage+strategies+for+genetically+complex+traits:+I.+Multilocus
models&amp;author=N+Risch&amp;volume=46&amp;publication_year=1990&amp;pages=222-228&amp;pmid=2301392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Neale B, Ferreira M, Medland S, Posthuma D.  <span class="ref-journal">Statistical Genetics: Gene Mapping through Linkage and
Association.</span> London: Taylor and Francis; 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+Genetics:+Gene+Mapping+through+Linkage+and
Association&amp;author=B+Neale&amp;author=M+Ferreira&amp;author=S+Medland&amp;author=D+Posthuma&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Sullivan PF. The genetics of schizophrenia. <span><span class="ref-journal">PLoS Medicine. </span>2005;<span class="ref-vol">2</span>:614–618.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Medicine&amp;title=The+genetics+of+schizophrenia&amp;author=PF+Sullivan&amp;volume=2&amp;publication_year=2005&amp;pages=614-618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Stefansson H, Sigurdsson E, Steinthorsdottir V, Bjornsdottir S, Sigmundsson T, Ghosh S, et al.  Neuregulin 1 and susceptibility to schizophrenia. <span><span class="ref-journal">American Journal of Human Genetics. </span>2002;<span class="ref-vol">71</span>(4):877–892.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378543/">PMC free article</a>]</span> [<a href="/pubmed/12145742" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Neuregulin+1+and+susceptibility+to+schizophrenia&amp;author=H+Stefansson&amp;author=E+Sigurdsson&amp;author=V+Steinthorsdottir&amp;author=S+Bjornsdottir&amp;author=T+Sigmundsson&amp;volume=71&amp;issue=4&amp;publication_year=2002&amp;pages=877-892&amp;pmid=12145742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Li D, Collier DA, He L. Meta-analysis shows strong positive association of the neuregulin
1 (NRG1) gene with schizophrenia. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>(12):1995–2002.</span> [<a href="/pubmed/16687441" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Meta-analysis+shows+strong+positive+association+of+the+neuregulin
1+(NRG1)+gene+with+schizophrenia&amp;author=D+Li&amp;author=DA+Collier&amp;author=L+He&amp;volume=15&amp;issue=12&amp;publication_year=2006&amp;pages=1995-2002&amp;pmid=16687441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Straub RE, Jiang Y, MacLean CJ, Ma Y, Webb BT, Myakishev MV, et al.  Genetic variation in the 6p22.3 gene <em>DTNBP1</em>, the
human ortholog of the mouse Dysbindin gene, is associated with
schizophrenia. <span><span class="ref-journal">American Journal of Human Genetics. </span>2002;<span class="ref-vol">71</span>(2):337–348.</span> Erratum in American Journal of Human Genetics 2002
Oct;72(4):1007. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379166/">PMC free article</a>]</span> [<a href="/pubmed/12098102" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Genetic+variation+in+the+6p22.3+gene+DTNBP1,+the
human+ortholog+of+the+mouse+Dysbindin+gene,+is+associated+with
schizophrenia&amp;author=RE+Straub&amp;author=Y+Jiang&amp;author=CJ+MacLean&amp;author=Y+Ma&amp;author=BT+Webb&amp;volume=71&amp;issue=2&amp;publication_year=2002&amp;pages=337-348&amp;pmid=12098102&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Mutsuddi M, Morris DW, Waggoner SG, Daly MJ, Scolnick EM, Sklar P. Analysis of high-resolution HapMap of DTNBP1 (Dysbindin) suggests
no consistency between reported common variant associations and
schizophrenia. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">79</span>(5):903–909.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1698571/">PMC free article</a>]</span> [<a href="/pubmed/17033966" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Analysis+of+high-resolution+HapMap+of+DTNBP1+(Dysbindin)+suggests
no+consistency+between+reported+common+variant+associations+and
schizophrenia&amp;author=M+Mutsuddi&amp;author=DW+Morris&amp;author=SG+Waggoner&amp;author=MJ+Daly&amp;author=EM+Scolnick&amp;volume=79&amp;issue=5&amp;publication_year=2006&amp;pages=903-909&amp;pmid=17033966&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Millar JK, Wilson-Annan JC, Anderson S, Christie S, Taylor MS, Semple CA, et al.  Disruption of two novel genes by a translocation co-segregating
with schizophrenia. <span><span class="ref-journal">Human Molecular Genetics. </span>2000;<span class="ref-vol">9</span>(9):1415–1423.</span> [<a href="/pubmed/10814723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Molecular+Genetics&amp;title=Disruption+of+two+novel+genes+by+a+translocation+co-segregating
with+schizophrenia&amp;author=JK+Millar&amp;author=JC+Wilson-Annan&amp;author=S+Anderson&amp;author=S+Christie&amp;author=MS+Taylor&amp;volume=9&amp;issue=9&amp;publication_year=2000&amp;pages=1415-1423&amp;pmid=10814723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Sullivan PF. Spurious genetic associations. <span><span class="ref-journal">Biological Psychiatry. </span>2007;<span class="ref-vol">61</span>:1121–1126.</span> [<a href="/pubmed/17346679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biological+Psychiatry&amp;title=Spurious+genetic+associations&amp;author=PF+Sullivan&amp;volume=61&amp;publication_year=2007&amp;pages=1121-1126&amp;pmid=17346679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Frayling TM, Timpson NJ, Weedon MN, Zeggini E, Freathy RM, Lindgren CM, et al.  A common variant in the <em>FTO</em> gene is associated
with body mass index and predisposes to childhood and adult
obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>(5826):889–894.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated
with+body+mass+index+and+predisposes+to+childhood+and+adult
obesity&amp;author=TM+Frayling&amp;author=NJ+Timpson&amp;author=MN+Weedon&amp;author=E+Zeggini&amp;author=RM+Freathy&amp;volume=316&amp;issue=5826&amp;publication_year=2007&amp;pages=889-894&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Klein RJ, Zeiss C, Chew EY, Tsai JY, Sackler RS, Haynes C, et al.  Complement factor H polymorphism in age-related macular
degeneration. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">308</span>(5720):385–389.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1512523/">PMC free article</a>]</span> [<a href="/pubmed/15761122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Complement+factor+H+polymorphism+in+age-related+macular
degeneration&amp;author=RJ+Klein&amp;author=C+Zeiss&amp;author=EY+Chew&amp;author=JY+Tsai&amp;author=RS+Sackler&amp;volume=308&amp;issue=5720&amp;publication_year=2005&amp;pages=385-389&amp;pmid=15761122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Duerr RH, Taylor KD, Brant SR, Rioux JD, Silverberg MS, Daly MJ, et al.  A genome-wide association study identifies IL23R as an
inflammatory bowel disease gene. <span><span class="ref-journal">Science. </span>2006;<span class="ref-vol">314</span>(5804):1461–1463.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4410764/">PMC free article</a>]</span> [<a href="/pubmed/17068223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+identifies+IL23R+as+an
inflammatory+bowel+disease+gene&amp;author=RH+Duerr&amp;author=KD+Taylor&amp;author=SR+Brant&amp;author=JD+Rioux&amp;author=MS+Silverberg&amp;volume=314&amp;issue=5804&amp;publication_year=2006&amp;pages=1461-1463&amp;pmid=17068223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">WTCCC. Genome-wide association study of 14,000 cases of seven common
diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>(7145):661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common
diseases+and+3,000+shared+controls&amp;volume=447&amp;issue=7145&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Saxena R, Voight BF, Lyssenko V, Burtt NP, de Bakker PI, Chen H, et al.  Genome-Wide Association Analysis Identifies Loci for Type 2
Diabetes and Triglyceride Levels. <span><span class="ref-journal">Science. </span>2007</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-Wide+Association+Analysis+Identifies+Loci+for+Type+2
Diabetes+and+Triglyceride+Levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;author=NP+Burtt&amp;author=PI+de+Bakker&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Scott LJ, Mohlke KL, Bonnycastle LL, Willer CJ, Li Y, Duren WL, et al.  A Genome-Wide Association Study of Type 2 Diabetes in Finns
Detects Multiple Susceptibility Variants. <span><span class="ref-journal">Science. </span>2007</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+Genome-Wide+Association+Study+of+Type+2+Diabetes+in+Finns
Detects+Multiple+Susceptibility+Variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;author=CJ+Willer&amp;author=Y+Li&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Steinthorsdottir V, Thorleifsson G, Reynisdottir I, Benediktsson R, Jonsdottir T, Walters GB, et al.  A variant in CDKAL1 influences insulin response and risk of type
2 diabetes. <span><span class="ref-journal">Nat Genet. </span>2007</span> [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type
2+diabetes&amp;author=V+Steinthorsdottir&amp;author=G+Thorleifsson&amp;author=I+Reynisdottir&amp;author=R+Benediktsson&amp;author=T+Jonsdottir&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Zeggini E, Weedon MN, Lindgren CM, Frayling TM, Elliott KS, Lango H, et al.  Replication of Genome-Wide Association Signals in U.K. Samples
Reveals Risk Loci for Type 2 Diabetes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1336–1341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+Genome-Wide+Association+Signals+in+U.K.+Samples
Reveals+Risk+Loci+for+Type+2+Diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;author=TM+Frayling&amp;author=KS+Elliott&amp;volume=316&amp;publication_year=2007&amp;pages=1336-1341&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Lencz T, Morgan TV, Athanasiou M, Dain B, Reed CR, Kane JM, et al.  Converging evidence for a pseudoautosomal cytokine receptor gene
locus in schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2007;<span class="ref-vol">12</span>(6):572–580.</span> [<a href="/pubmed/17522711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Converging+evidence+for+a+pseudoautosomal+cytokine+receptor+gene
locus+in+schizophrenia&amp;author=T+Lencz&amp;author=TV+Morgan&amp;author=M+Athanasiou&amp;author=B+Dain&amp;author=CR+Reed&amp;volume=12&amp;issue=6&amp;publication_year=2007&amp;pages=572-580&amp;pmid=17522711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Stroup TS, McEvoy JP, Swartz MS, Byerly MJ, Glick ID, Canive JM, et al.  The National Institute of Mental Health Clinical Antipsychotic
Trials of Intervention Effectiveness (CATIE) project: schizophrenia trial
design and protocol development. <span><span class="ref-journal">Schizophr Bull. </span>2003;<span class="ref-vol">29</span>(1):15–31.</span> [<a href="/pubmed/12908658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr+Bull&amp;title=The+National+Institute+of+Mental+Health+Clinical+Antipsychotic
Trials+of+Intervention+Effectiveness+(CATIE)+project:+schizophrenia+trial
design+and+protocol+development&amp;author=TS+Stroup&amp;author=JP+McEvoy&amp;author=MS+Swartz&amp;author=MJ+Byerly&amp;author=ID+Glick&amp;volume=29&amp;issue=1&amp;publication_year=2003&amp;pages=15-31&amp;pmid=12908658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Lieberman JA. Comparative effectiveness of antipsychotic drugs. A commentary
on: Cost Utility Of The Latest Antipsychotic Drugs In Schizophrenia Study
(CUtLASS 1) and Clinical Antipsychotic Trials Of Intervention Effectiveness
(CATIE) <span><span class="ref-journal">Arch Gen Psychiatry. </span>2006;<span class="ref-vol">63</span>(10):1069–1072.</span> [<a href="/pubmed/17015808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Comparative+effectiveness+of+antipsychotic+drugs.+A+commentary
on:+Cost+Utility+Of+The+Latest+Antipsychotic+Drugs+In+Schizophrenia+Study
(CUtLASS+1)+and+Clinical+Antipsychotic+Trials+Of+Intervention+Effectiveness
(CATIE)&amp;author=JA+Lieberman&amp;volume=63&amp;issue=10&amp;publication_year=2006&amp;pages=1069-1072&amp;pmid=17015808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Chanock SJ, Manolio T, Boehnke M, Boerwinkle E, Hunter DJ, Thomas G, et al.  Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>(7145):655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;author=SJ+Chanock&amp;author=T+Manolio&amp;author=M+Boehnke&amp;author=E+Boerwinkle&amp;author=DJ+Hunter&amp;volume=447&amp;issue=7145&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Lieberman JA, Stroup TS, McEvoy JP, Swartz MS, Rosenheck RA, Perkins DO, et al.  Effectiveness of antipsychotic drugs in patients with chronic
schizophrenia. <span><span class="ref-journal">N Engl J Med. </span>2005;<span class="ref-vol">353</span>:1209–1223.</span> [<a href="/pubmed/16172203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Effectiveness+of+antipsychotic+drugs+in+patients+with+chronic
schizophrenia&amp;author=JA+Lieberman&amp;author=TS+Stroup&amp;author=JP+McEvoy&amp;author=MS+Swartz&amp;author=RA+Rosenheck&amp;volume=353&amp;publication_year=2005&amp;pages=1209-1223&amp;pmid=16172203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">American Psychiatric Association. <span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders.</span> Fourth Edition ed. Washington, DC: American Psychiatric Association; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">First M, Spitzer R, Gibbon M, Williams J.  <span class="ref-journal">Structured Clinical Interview for DSM-IV Axis I
Disorders--Administration Booklet.</span> Washington, DC: American Psychiatric Press, Inc.; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Structured+Clinical+Interview+for+DSM-IV+Axis+I
Disorders--Administration+Booklet&amp;author=M+First&amp;author=R+Spitzer&amp;author=M+Gibbon&amp;author=J+Williams&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Affymetrix. <span class="ref-journal">GeneChip<sup>®</sup> Mapping 500K Assay Manual.</span> Santa Clara, CA: 2006.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=GeneChip®+Mapping+500K+Assay+Manual&amp;publication_year=2006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Bierut LJ, Madden PA, Breslau N, Johnson EO, Hatsukami D, Pomerleau OF, et al.  Novel genes identified in a high-density genome wide association
study for nicotine dependence. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>(1):24–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2278047/">PMC free article</a>]</span> [<a href="/pubmed/17158188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Novel+genes+identified+in+a+high-density+genome+wide+association
study+for+nicotine+dependence&amp;author=LJ+Bierut&amp;author=PA+Madden&amp;author=N+Breslau&amp;author=EO+Johnson&amp;author=D+Hatsukami&amp;volume=16&amp;issue=1&amp;publication_year=2007&amp;pages=24-35&amp;pmid=17158188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Hinds DA, Stuve LL, Nilsen GB, Halperin E, Eskin E, Ballinger DG, et al.  Whole-genome patterns of common DNA variation in three human
populations. <span><span class="ref-journal">Science. </span>2005;<span class="ref-vol">307</span>(5712):1072–1079.</span> [<a href="/pubmed/15718463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Whole-genome+patterns+of+common+DNA+variation+in+three+human
populations&amp;author=DA+Hinds&amp;author=LL+Stuve&amp;author=GB+Nilsen&amp;author=E+Halperin&amp;author=E+Eskin&amp;volume=307&amp;issue=5712&amp;publication_year=2005&amp;pages=1072-1079&amp;pmid=15718463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Affymetrix. <span class="ref-journal">BRLMM: an Improved Genotype Calling Method for the
GeneChip<sup>®</sup> Human Mapping 500K Array Set.</span> Santa Clara, CA: Apr 14, 2006.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=BRLMM:+an+Improved+Genotype+Calling+Method+for+the
GeneChip®+Human+Mapping+500K+Array+Set&amp;publication_year=2006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">GAIN Collaborative Research Group. <span class="ref-journal">New models of collaboration in genome-wide association studies: the
Genetic Association Information Network (GAIN)</span>  Submitted. [<a href="/pubmed/17728769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=New+models+of+collaboration+in+genome-wide+association+studies:+the
Genetic+Association+Information+Network+(GAIN)&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Sullivan PF, Purcell S.  In: <span class="ref-journal">Statistical Genetics: Gene Mapping through Linkage and
Association.</span> Neale B, Ferreira M, Medland S, Posthuma D, editors. London: Taylor and Francis; 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+Genetics:+Gene+Mapping+through+Linkage+and
Association&amp;author=PF+Sullivan&amp;author=S+Purcell&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira M, Bender D, et al.  PLINK: a toolset for whole-genome association and
population-based linkage analysis. <span><span class="ref-journal">American Journal of Human Genetics. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=PLINK:+a+toolset+for+whole-genome+association+and
population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=M+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Wigginton JE, Cutler DJ, Abecasis GR. A note on exact tests of Hardy-Weinberg
equilibrium. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">76</span>(5):887–883.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1199378/">PMC free article</a>]</span> [<a href="/pubmed/15789306" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+note+on+exact+tests+of+Hardy-Weinberg
equilibrium&amp;author=JE+Wigginton&amp;author=DJ+Cutler&amp;author=GR+Abecasis&amp;volume=76&amp;issue=5&amp;publication_year=2005&amp;pages=887-883&amp;pmid=15789306&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Wittke-Thompson JK, Pluzhnikov A, Cox NJ. Rational inferences about departures from Hardy-Weinberg
equilibrium. <span><span class="ref-journal">Am J Hum Genet. </span>2005;<span class="ref-vol">76</span>(6):967–986.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1196455/">PMC free article</a>]</span> [<a href="/pubmed/15834813" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Rational+inferences+about+departures+from+Hardy-Weinberg
equilibrium&amp;author=JK+Wittke-Thompson&amp;author=A+Pluzhnikov&amp;author=NJ+Cox&amp;volume=76&amp;issue=6&amp;publication_year=2005&amp;pages=967-986&amp;pmid=15834813&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Marchini J, Cardon LR, Phillips MS, Donnelly P. The effects of human population structure on large genetic
association studies. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>(5):512–517.</span> [<a href="/pubmed/15052271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+effects+of+human+population+structure+on+large+genetic
association+studies&amp;author=J+Marchini&amp;author=LR+Cardon&amp;author=MS+Phillips&amp;author=P+Donnelly&amp;volume=36&amp;issue=5&amp;publication_year=2004&amp;pages=512-517&amp;pmid=15052271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Heiman GA, Hodge SE, Gorroochurn P, Zhang J, Greenberg DA. Effect of population stratification on case-control association
studies. I. Elevation in false positive rates and comparison to confounding
risk ratios (a simulation study) <span><span class="ref-journal">Hum Hered. </span>2004;<span class="ref-vol">58</span>(1):30–39.</span> [<a href="/pubmed/15604562" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Effect+of+population+stratification+on+case-control+association
studies.+I.+Elevation+in+false+positive+rates+and+comparison+to+confounding
risk+ratios+(a+simulation+study)&amp;author=GA+Heiman&amp;author=SE+Hodge&amp;author=P+Gorroochurn&amp;author=J+Zhang&amp;author=DA+Greenberg&amp;volume=58&amp;issue=1&amp;publication_year=2004&amp;pages=30-39&amp;pmid=15604562&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Shields AE, Fortun M, Hammonds EM, King PA, Lerman C, Rapp R, et al.  The use of race variables in genetic studies of complex traits
and the goal of reducing health disparities. <span><span class="ref-journal">American Psychologist. </span>2005;<span class="ref-vol">60</span>:77–103.</span> [<a href="/pubmed/15641924" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Psychologist&amp;title=The+use+of+race+variables+in+genetic+studies+of+complex+traits
and+the+goal+of+reducing+health+disparities&amp;author=AE+Shields&amp;author=M+Fortun&amp;author=EM+Hammonds&amp;author=PA+King&amp;author=C+Lerman&amp;volume=60&amp;publication_year=2005&amp;pages=77-103&amp;pmid=15641924&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Devlin B, Roeder K. Genomic control for association studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+control+for+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Pritchard JK, Stephens M, Rosenberg NA, Donnelley P. Association mapping in structured populations. <span><span class="ref-journal">American Journal of Human Genetics. </span>2000;<span class="ref-vol">67</span>:170–181.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1287075/">PMC free article</a>]</span> [<a href="/pubmed/10827107" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Association+mapping+in+structured+populations&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=NA+Rosenberg&amp;author=P+Donnelley&amp;volume=67&amp;publication_year=2000&amp;pages=170-181&amp;pmid=10827107&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in
genome-wide association studies. <span><span class="ref-journal">Nature Genetics. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Principal+components+analysis+corrects+for+stratification+in
genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Epstein MP, Allen AS, Satten GA. A simple and improved correction for population stratification in
case-control studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>(5):921–930.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852732/">PMC free article</a>]</span> [<a href="/pubmed/17436246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+simple+and+improved+correction+for+population+stratification+in
case-control+studies&amp;author=MP+Epstein&amp;author=AS+Allen&amp;author=GA+Satten&amp;volume=80&amp;issue=5&amp;publication_year=2007&amp;pages=921-930&amp;pmid=17436246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Lee S, Sullivan PF, Zou F, Wright FA. Comment on "A Simple and Improved Correction of
Population Stratification". <span><span class="ref-journal">American Journal of Human Genetics. </span></span> In press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427240/">PMC free article</a>]</span> [<a href="/pubmed/18252233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Comment+on+&quot;A+Simple+and+Improved+Correction+of
Population+Stratification&quot;&amp;author=S+Lee&amp;author=PF+Sullivan&amp;author=F+Zou&amp;author=FA+Wright&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">22</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=22&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Lettre G, Lange C, Hirschhorn JN. Genetic model testing and statistical power in population-based
association studies of quantitative traits. <span><span class="ref-journal">Genet Epidemiol. </span>2007;<span class="ref-vol">31</span>(4):358–362.</span> [<a href="/pubmed/17352422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Genetic+model+testing+and+statistical+power+in+population-based
association+studies+of+quantitative+traits&amp;author=G+Lettre&amp;author=C+Lange&amp;author=JN+Hirschhorn&amp;volume=31&amp;issue=4&amp;publication_year=2007&amp;pages=358-362&amp;pmid=17352422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Storey JD. The positive false discovery rate: a Bayesian interpretation and
the q-value. <span><span class="ref-journal">Annals of Statistics. </span>2003;<span class="ref-vol">31</span>:2013–2035.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Statistics&amp;title=The+positive+false+discovery+rate:+a+Bayesian+interpretation+and
the+q-value&amp;author=JD+Storey&amp;volume=31&amp;publication_year=2003&amp;pages=2013-2035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Storey JD, Tibshirani R. Statistical significance for genomewide studies. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2003;<span class="ref-vol">100</span>(16):9440–9445.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC170937/">PMC free article</a>]</span> [<a href="/pubmed/12883005" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Statistical+significance+for+genomewide+studies&amp;author=JD+Storey&amp;author=R+Tibshirani&amp;volume=100&amp;issue=16&amp;publication_year=2003&amp;pages=9440-9445&amp;pmid=12883005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">van den Oord EJ, Sullivan PF. False discoveries and models for gene discovery. <span><span class="ref-journal">Trends Genet. </span>2003;<span class="ref-vol">19</span>(10):537–542.</span> [<a href="/pubmed/14550627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=False+discoveries+and+models+for+gene+discovery&amp;author=EJ+van+den+Oord&amp;author=PF+Sullivan&amp;volume=19&amp;issue=10&amp;publication_year=2003&amp;pages=537-542&amp;pmid=14550627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">van den Oord EJCG. Controlling false discoveries in genetic studies. <span><span class="ref-journal">American Journal of Medical Genetics (Neuropsychiatric
Genetics) </span></span> In press. [<a href="/pubmed/18092307" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Medical+Genetics+(Neuropsychiatric
Genetics)&amp;title=Controlling+false+discoveries+in+genetic+studies&amp;author=EJCG+van+den+Oord&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful
approach to multiple testing. <span><span class="ref-journal">Journal of the Royal Statistical Society (Series B) </span>1995;<span class="ref-vol">57</span>:289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+the+Royal+Statistical+Society+(Series+B)&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful
approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Brown BW, Russell K. Methods of correcting for multiple testing: operating
characteristics. <span><span class="ref-journal">Stat Med. </span>1997;<span class="ref-vol">16</span>(22):2511–2528.</span> [<a href="/pubmed/9403953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Methods+of+correcting+for+multiple+testing:+operating
characteristics&amp;author=BW+Brown&amp;author=K+Russell&amp;volume=16&amp;issue=22&amp;publication_year=1997&amp;pages=2511-2528&amp;pmid=9403953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Fernando RL, Nettleton D, Southey BR, Dekkers JC, Rothschild MF, Soller M. Controlling the proportion of false positives in multiple
dependent tests. <span><span class="ref-journal">Genetics. </span>2004;<span class="ref-vol">166</span>(1):611–619.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1470704/">PMC free article</a>]</span> [<a href="/pubmed/15020448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Controlling+the+proportion+of+false+positives+in+multiple
dependent+tests&amp;author=RL+Fernando&amp;author=D+Nettleton&amp;author=BR+Southey&amp;author=JC+Dekkers&amp;author=MF+Rothschild&amp;volume=166&amp;issue=1&amp;publication_year=2004&amp;pages=611-619&amp;pmid=15020448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">van den Oord EJ, Sullivan PF. A framework for controlling false discovery rates and minimizing
the amount of genotyping in the search for disease mutations. <span><span class="ref-journal">Hum Hered. </span>2003;<span class="ref-vol">56</span>(4):188–199.</span> [<a href="/pubmed/15031620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=A+framework+for+controlling+false+discovery+rates+and+minimizing
the+amount+of+genotyping+in+the+search+for+disease+mutations&amp;author=EJ+van+den+Oord&amp;author=PF+Sullivan&amp;volume=56&amp;issue=4&amp;publication_year=2003&amp;pages=188-199&amp;pmid=15031620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Tsai CA, Hsueh HM, Chen JJ. Estimation of false discovery rates in multiple testing:
application to gene microarray data. <span><span class="ref-journal">Biometrics. </span>2003;<span class="ref-vol">59</span>(4):1071–1081.</span> [<a href="/pubmed/14969487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Estimation+of+false+discovery+rates+in+multiple+testing:
application+to+gene+microarray+data&amp;author=CA+Tsai&amp;author=HM+Hsueh&amp;author=JJ+Chen&amp;volume=59&amp;issue=4&amp;publication_year=2003&amp;pages=1071-1081&amp;pmid=14969487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">van den Oord EJ. Controlling false discoveries in candidate gene
studies. <span><span class="ref-journal">Mol Psychiatry. </span>2005;<span class="ref-vol">10</span>(3):230–231.</span> [<a href="/pubmed/15738930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Controlling+false+discoveries+in+candidate+gene
studies&amp;author=EJ+van+den+Oord&amp;volume=10&amp;issue=3&amp;publication_year=2005&amp;pages=230-231&amp;pmid=15738930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Sabatti C, Service S, Freimer N. False discovery rate in linkage and association genome screens
for complex disorders. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>(2):829–833.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462572/">PMC free article</a>]</span> [<a href="/pubmed/12807801" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=False+discovery+rate+in+linkage+and+association+genome+screens
for+complex+disorders&amp;author=C+Sabatti&amp;author=S+Service&amp;author=N+Freimer&amp;volume=164&amp;issue=2&amp;publication_year=2003&amp;pages=829-833&amp;pmid=12807801&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Meinhausen N, Rice J. Estimating the proportion of false null hypotheses among a large
number of independently tested hypotheses. <span><span class="ref-journal">Annals of Statistics. </span>2006;<span class="ref-vol">34</span>:373–393.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Statistics&amp;title=Estimating+the+proportion+of+false+null+hypotheses+among+a+large
number+of+independently+tested+hypotheses&amp;author=N+Meinhausen&amp;author=J+Rice&amp;volume=34&amp;publication_year=2006&amp;pages=373-393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Schaid DJ, Rowland CM, Tines DE, Jacobson RM, Poland GA. Score tests for association between traits and haplotypes when
linkage phase is ambiguous. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">70</span>(2):425–434.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC384917/">PMC free article</a>]</span> [<a href="/pubmed/11791212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Score+tests+for+association+between+traits+and+haplotypes+when
linkage+phase+is+ambiguous&amp;author=DJ+Schaid&amp;author=CM+Rowland&amp;author=DE+Tines&amp;author=RM+Jacobson&amp;author=GA+Poland&amp;volume=70&amp;issue=2&amp;publication_year=2002&amp;pages=425-434&amp;pmid=11791212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">R Development Core Team. <span class="ref-journal">R: A language and environment for statistical computing.</span> Vienna, Austria: R Foundation for Statistical Computing; 2005.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+language+and+environment+for+statistical+computing&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Gabriel SB, Schaffner SF, Nguyen H, Moore JM, Roy J, Blumenstiel B, et al.  The structure of haplotype blocks in the human
genome. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>(5576):2225–2229.</span> [<a href="/pubmed/12029063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+structure+of+haplotype+blocks+in+the+human
genome&amp;author=SB+Gabriel&amp;author=SF+Schaffner&amp;author=H+Nguyen&amp;author=JM+Moore&amp;author=J+Roy&amp;volume=296&amp;issue=5576&amp;publication_year=2002&amp;pages=2225-2229&amp;pmid=12029063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype
maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype
maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Altshuler D, Brooks LD, Chakravarti A, Collins FS, Daly MJ, Donnelly P. A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>(7063):1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;author=D+Altshuler&amp;author=LD+Brooks&amp;author=A+Chakravarti&amp;author=FS+Collins&amp;author=MJ+Daly&amp;volume=437&amp;issue=7063&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Gauderman WJ. Sample size requirements for association studies of gene-gene
interaction. <span><span class="ref-journal">Am J Epidemiol. </span>2002;<span class="ref-vol">155</span>(5):478–484.</span> [<a href="/pubmed/11867360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Sample+size+requirements+for+association+studies+of+gene-gene
interaction&amp;author=WJ+Gauderman&amp;volume=155&amp;issue=5&amp;publication_year=2002&amp;pages=478-484&amp;pmid=11867360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="element-citation">Gauderman WJ. Sample size requirements for matched case-control studies of
gene-environment interaction. <span><span class="ref-journal">Stat Med. </span>2002;<span class="ref-vol">21</span>(1):35–50.</span> [<a href="/pubmed/11782049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Sample+size+requirements+for+matched+case-control+studies+of
gene-environment+interaction&amp;author=WJ+Gauderman&amp;volume=21&amp;issue=1&amp;publication_year=2002&amp;pages=35-50&amp;pmid=11782049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="element-citation">SAS Institute Inc. <span class="ref-journal">SAS/STATÂ® Software: Version 9.</span> Cary, NC: SAS Institute, Inc.; 2004.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SAS/STATÂ®+Software:+Version+9&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">69. <span class="element-citation">SAS Institute Inc. <span class="ref-journal">JMP User's Guide (Version 6)</span> Cary, NC: SAS Institute, Inc.; 2005.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=JMP+User's+Guide+(Version+6)&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">70. <span class="element-citation">Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, et al.  The UCSC Genome Browser Database: update 2006. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>(Database issue):D590–D598.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1347506/">PMC free article</a>]</span> [<a href="/pubmed/16381938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+UCSC+Genome+Browser+Database:+update+2006&amp;author=AS+Hinrichs&amp;author=D+Karolchik&amp;author=R+Baertsch&amp;author=GP+Barber&amp;author=G+Bejerano&amp;volume=34&amp;issue=Database+issue&amp;publication_year=2006&amp;pages=D590-D598&amp;pmid=16381938&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">71. <span class="element-citation">Wheeler DL, Barrett T, Benson DA, Bryant SH, Canese K, Chetvernin V, et al.  Database resources of the National Center for Biotechnology
Information. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>(Database issue):D173–D180.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1347520/">PMC free article</a>]</span> [<a href="/pubmed/16381840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Database+resources+of+the+National+Center+for+Biotechnology
Information&amp;author=DL+Wheeler&amp;author=T+Barrett&amp;author=DA+Benson&amp;author=SH+Bryant&amp;author=K+Canese&amp;volume=34&amp;issue=Database+issue&amp;publication_year=2006&amp;pages=D173-D180&amp;pmid=16381840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">72. <span class="element-citation">Blaschke RJ, Rappold G. The pseudoautosomal regions, SHOX and disease. <span><span class="ref-journal">Curr Opin Genet Dev. </span>2006;<span class="ref-vol">16</span>(3):233–239.</span> [<a href="/pubmed/16650979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Genet+Dev&amp;title=The+pseudoautosomal+regions,+SHOX+and+disease&amp;author=RJ+Blaschke&amp;author=G+Rappold&amp;volume=16&amp;issue=3&amp;publication_year=2006&amp;pages=233-239&amp;pmid=16650979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">73. <span class="element-citation">Hemminger BM, Saelim B, Sullivan PF. TAMAL: An integrated approach to choosing SNPs for genetic
studies of human complex traits. <span><span class="ref-journal">Bioinformatics. </span>2006;<span class="ref-vol">22</span>:626–627.</span> [<a href="/pubmed/16418238" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TAMAL:+An+integrated+approach+to+choosing+SNPs+for+genetic
studies+of+human+complex+traits&amp;author=BM+Hemminger&amp;author=B+Saelim&amp;author=PF+Sullivan&amp;volume=22&amp;publication_year=2006&amp;pages=626-627&amp;pmid=16418238&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">74. <span class="element-citation">Fisher RA.  <span class="ref-journal">Statistical Methods for Reesearch Workers.</span> 11th edition. London: Oliver and Boyd; 1950.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Statistical+Methods+for+Reesearch+Workers&amp;author=RA+Fisher&amp;publication_year=1950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">75. <span class="element-citation">Kay SR, Fiszbein A, Opler LA. The positive and negative syndrome scale (PANSS) for
schizophrenia. <span><span class="ref-journal">Schizophr Bull. </span>1987;<span class="ref-vol">13</span>(2):261–276.</span> [<a href="/pubmed/3616518" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr+Bull&amp;title=The+positive+and+negative+syndrome+scale+(PANSS)+for
schizophrenia&amp;author=SR+Kay&amp;author=A+Fiszbein&amp;author=LA+Opler&amp;volume=13&amp;issue=2&amp;publication_year=1987&amp;pages=261-276&amp;pmid=3616518&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R76">76. <span class="element-citation">Sullivan PF, Keefe RSE, Lange LA, Lange EM, Stroup TS, Lieberman JA, et al.  
<em>NCAM1</em> and neurocognition in schizophrenia. <span><span class="ref-journal">Biological Psychiatry. </span>2007;<span class="ref-vol">Volume 61</span>(7):902–910.</span> [<a href="/pubmed/17161382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biological+Psychiatry&amp;title=
NCAM1+and+neurocognition+in+schizophrenia&amp;author=PF+Sullivan&amp;author=RSE+Keefe&amp;author=LA+Lange&amp;author=EM+Lange&amp;author=TS+Stroup&amp;volume=Volume+61&amp;issue=7&amp;publication_year=2007&amp;pages=902-910&amp;pmid=17161382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R77">77. <span class="element-citation">Balding DJ. A tutorial on statistical methods for population association
studies. <span><span class="ref-journal">Nat Rev Genet. </span>2006;<span class="ref-vol">7</span>(10):781–791.</span> [<a href="/pubmed/16983374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=A+tutorial+on+statistical+methods+for+population+association
studies&amp;author=DJ+Balding&amp;volume=7&amp;issue=10&amp;publication_year=2006&amp;pages=781-791&amp;pmid=16983374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R78">78. <span class="element-citation">Sachse C, Ruschen S, Dettling M, Schley J, Bauer S, Muller-Oerlinghausen B, et al.  Flavin monooxygenase 3 (FMO3) polymorphism in a white population:
allele frequencies, mutation linkage, and functional effects on clozapine
and caffeine metabolism. <span><span class="ref-journal">Clin Pharmacol Ther. </span>1999;<span class="ref-vol">66</span>(4):431–438.</span> [<a href="/pubmed/10546928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Flavin+monooxygenase+3+(FMO3)+polymorphism+in+a+white+population:
allele+frequencies,+mutation+linkage,+and+functional+effects+on+clozapine
and+caffeine+metabolism&amp;author=C+Sachse&amp;author=S+Ruschen&amp;author=M+Dettling&amp;author=J+Schley&amp;author=S+Bauer&amp;volume=66&amp;issue=4&amp;publication_year=1999&amp;pages=431-438&amp;pmid=10546928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R79">79. <span class="element-citation">Farrer LA, Cupples LA, Haines JL, Hyman B, Kukull WA, Mayeux R, et al.  Effects of age, sex, and ethnicity on the association between
apolipoprotein E genotype and Alzheimer disease. A meta-analysis. APOE and
Alzheimer Disease Meta Analysis Consortium. <span><span class="ref-journal">Jama. </span>1997;<span class="ref-vol">278</span>:1349–1356.</span> [<a href="/pubmed/9343467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Jama&amp;title=Effects+of+age,+sex,+and+ethnicity+on+the+association+between
apolipoprotein+E+genotype+and+Alzheimer+disease.+A+meta-analysis.+APOE+and
Alzheimer+Disease+Meta+Analysis+Consortium&amp;author=LA+Farrer&amp;author=LA+Cupples&amp;author=JL+Haines&amp;author=B+Hyman&amp;author=WA+Kukull&amp;volume=278&amp;publication_year=1997&amp;pages=1349-1356&amp;pmid=9343467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R80">80. <span class="element-citation">Hsu F, Kent WJ, Clawson H, Kuhn RM, Diekhans M, Haussler D. The UCSC Known Genes. <span><span class="ref-journal">Bioinformatics. </span>2006;<span class="ref-vol">22</span>(9):1036–1046.</span> [<a href="/pubmed/16500937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The+UCSC+Known+Genes&amp;author=F+Hsu&amp;author=WJ+Kent&amp;author=H+Clawson&amp;author=RM+Kuhn&amp;author=M+Diekhans&amp;volume=22&amp;issue=9&amp;publication_year=2006&amp;pages=1036-1046&amp;pmid=16500937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R81">81. <span class="element-citation">Mah S, Nelson MR, Delisi LE, Reneland RH, Markward N, James MR, et al.  Identification of the semaphorin receptor PLXNA2 as a candidate
for susceptibility to schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2006</span> [<a href="/pubmed/16402134" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Identification+of+the+semaphorin+receptor+PLXNA2+as+a+candidate
for+susceptibility+to+schizophrenia&amp;author=S+Mah&amp;author=MR+Nelson&amp;author=LE+Delisi&amp;author=RH+Reneland&amp;author=N+Markward&amp;publication_year=2006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R82">82. <span class="element-citation">Hennah W, Varilo T, Kestila M, Paunio T, Arajarvi R, Haukka J, et al.  Haplotype transmission analysis provides evidence of association
for DISC1 to schizophrenia and suggests sex-dependent
effects. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>(23):3151–3159.</span> [<a href="/pubmed/14532331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Haplotype+transmission+analysis+provides+evidence+of+association
for+DISC1+to+schizophrenia+and+suggests+sex-dependent
effects&amp;author=W+Hennah&amp;author=T+Varilo&amp;author=M+Kestila&amp;author=T+Paunio&amp;author=R+Arajarvi&amp;volume=12&amp;issue=23&amp;publication_year=2003&amp;pages=3151-3159&amp;pmid=14532331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R83">83. <span class="element-citation">Cannon TD, Hennah W, van Erp TG, Thompson PM, Lonnqvist J, Huttunen M, et al.  Association of DISC1/TRAX haplotypes with schizophrenia, reduced
prefrontal gray matter, and impaired short- and long-term
memory. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2005;<span class="ref-vol">62</span>(11):1205–1213.</span> [<a href="/pubmed/16275808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Association+of+DISC1/TRAX+haplotypes+with+schizophrenia,+reduced
prefrontal+gray+matter,+and+impaired+short-+and+long-term
memory&amp;author=TD+Cannon&amp;author=W+Hennah&amp;author=TG+van+Erp&amp;author=PM+Thompson&amp;author=J+Lonnqvist&amp;volume=62&amp;issue=11&amp;publication_year=2005&amp;pages=1205-1213&amp;pmid=16275808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R84">84. <span class="element-citation">Law AJ, Lipska BK, Weickert CS, Hyde TM, Straub RE, Hashimoto R, et al.  Neuregulin 1 transcripts are differentially expressed in
schizophrenia and regulated by 5’ SNPs associated with the
disease. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2006;<span class="ref-vol">103</span>(17):6747–6752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1458952/">PMC free article</a>]</span> [<a href="/pubmed/16618933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Neuregulin+1+transcripts+are+differentially+expressed+in
schizophrenia+and+regulated+by+5’+SNPs+associated+with+the
disease&amp;author=AJ+Law&amp;author=BK+Lipska&amp;author=CS+Weickert&amp;author=TM+Hyde&amp;author=RE+Straub&amp;volume=103&amp;issue=17&amp;publication_year=2006&amp;pages=6747-6752&amp;pmid=16618933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R85">85. <span class="element-citation">Hall J, Whalley HC, Job DE, Baig BJ, McIntosh AM, Evans KL, et al.  A neuregulin 1 variant associated with abnormal cortical function
and psychotic symptoms. <span><span class="ref-journal">Nat Neurosci. </span>2006;<span class="ref-vol">9</span>(12):1477–1478.</span> [<a href="/pubmed/17072305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=A+neuregulin+1+variant+associated+with+abnormal+cortical+function
and+psychotic+symptoms&amp;author=J+Hall&amp;author=HC+Whalley&amp;author=DE+Job&amp;author=BJ+Baig&amp;author=AM+McIntosh&amp;volume=9&amp;issue=12&amp;publication_year=2006&amp;pages=1477-1478&amp;pmid=17072305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R86">86. <span class="element-citation">Crowley JJ, Keefe RS, Perkins DO, Stroup TS, Lieberman JA, Sullivan PF. <span><span class="ref-journal">The neuregulin 1 promoter polymorphism rs6994992 is not associated with
chronic schizophrenia or neurocognition. </span></span> Submitted. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3875465/">PMC free article</a>]</span> [<a href="/pubmed/18286587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+neuregulin+1+promoter+polymorphism+rs6994992+is+not+associated+with
chronic+schizophrenia+or+neurocognition&amp;author=JJ+Crowley&amp;author=RS+Keefe&amp;author=DO+Perkins&amp;author=TS+Stroup&amp;author=JA+Lieberman&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R87">87. <span class="element-citation">Shifman S, Bronstein M, Sternfeld M, Pisante-Shalom A, Lev-Lehman E, Weizman A, et al.  A highly significant association between a COMT haplotype and
schizophrenia. <span><span class="ref-journal">American Journal of Human Genetics. </span>2002;<span class="ref-vol">71</span>:1296–1302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC378567/">PMC free article</a>]</span> [<a href="/pubmed/12402217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=A+highly+significant+association+between+a+COMT+haplotype+and
schizophrenia&amp;author=S+Shifman&amp;author=M+Bronstein&amp;author=M+Sternfeld&amp;author=A+Pisante-Shalom&amp;author=E+Lev-Lehman&amp;volume=71&amp;publication_year=2002&amp;pages=1296-1302&amp;pmid=12402217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R88">88. <span class="element-citation">Skol AD, Scott LJ, Abecasis GR, Boehnke M. Joint analysis is more efficient than replication-based analysis
for two-stage genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span> [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis
for+two-stage+genome-wide+association+studies&amp;author=AD+Skol&amp;author=LJ+Scott&amp;author=GR+Abecasis&amp;author=M+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R89">89. <span class="element-citation">Sladek R, Rocheleau G, Rung J, Dina C, Shen L, Serre D, et al.  A genome-wide association study identifies novel risk loci for
type 2 diabetes. <span><span class="ref-journal">Nature. </span>2007</span> [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for
type+2+diabetes&amp;author=R+Sladek&amp;author=G+Rocheleau&amp;author=J+Rung&amp;author=C+Dina&amp;author=L+Shen&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R90">90. <span class="element-citation">Frayling TM. Genome-wide association studies provide new insights into type 2
diabetes aetiology. <span><span class="ref-journal">Nat Rev Genet. </span>2007;<span class="ref-vol">8</span>(9):657–662.</span> [<a href="/pubmed/17703236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+provide+new+insights+into+type+2
diabetes+aetiology&amp;author=TM+Frayling&amp;volume=8&amp;issue=9&amp;publication_year=2007&amp;pages=657-662&amp;pmid=17703236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R91">91. <span class="element-citation">Manolio TA, Rodriguez LL, Brooks L, Abecasis G, Ballinger D, Daly M, et al.  New models of collaboration in genome-wide association studies:
the Genetic Association Information Network. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>(9):1045–1051.</span> [<a href="/pubmed/17728769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+models+of+collaboration+in+genome-wide+association+studies:
the+Genetic+Association+Information+Network&amp;author=TA+Manolio&amp;author=LL+Rodriguez&amp;author=L+Brooks&amp;author=G+Abecasis&amp;author=D+Ballinger&amp;volume=39&amp;issue=9&amp;publication_year=2007&amp;pages=1045-1051&amp;pmid=17728769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R92">92. <span class="element-citation">Barrett JC, Cardon LR. Evaluating coverage of genome-wide association
studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>(6):659–662.</span> [<a href="/pubmed/16715099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Evaluating+coverage+of+genome-wide+association
studies&amp;author=JC+Barrett&amp;author=LR+Cardon&amp;volume=38&amp;issue=6&amp;publication_year=2006&amp;pages=659-662&amp;pmid=16715099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R93">93. <span class="element-citation">Miller D, McEvoy JP, Davis SM, Caroff SN, Saltz BL, Chakos MH, et al.  Clinical correlates of tardive dyskinesia in schizophrenia:
baseline data from the CATIE schizophrenia trial. <span><span class="ref-journal">Schizophr Res. </span>2005;<span class="ref-vol">80</span>(1):33–43.</span> [<a href="/pubmed/16171976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr+Res&amp;title=Clinical+correlates+of+tardive+dyskinesia+in+schizophrenia:
baseline+data+from+the+CATIE+schizophrenia+trial&amp;author=D+Miller&amp;author=JP+McEvoy&amp;author=SM+Davis&amp;author=SN+Caroff&amp;author=BL+Saltz&amp;volume=80&amp;issue=1&amp;publication_year=2005&amp;pages=33-43&amp;pmid=16171976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R94">94. <span class="element-citation">Keefe RS, Bilder RM, Harvey PD, Davis SM, Palmer BW, Gold JM, et al.  Baseline neurocognitive deficits in the CATIE schizophrenia
trial. <span><span class="ref-journal">Neuropsychopharmacology. </span>2006;<span class="ref-vol">31</span>:2033–2046.</span> [<a href="/pubmed/16641947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&amp;title=Baseline+neurocognitive+deficits+in+the+CATIE+schizophrenia
trial&amp;author=RS+Keefe&amp;author=RM+Bilder&amp;author=PD+Harvey&amp;author=SM+Davis&amp;author=BW+Palmer&amp;volume=31&amp;publication_year=2006&amp;pages=2033-2046&amp;pmid=16641947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R95">95. <span class="element-citation">World Health Organization. <span class="ref-journal">The ICD-10 Classification of Mental and Behavioural Disorders:
Diagnostic Criteria for Research.</span> Geneva: World Health Organization; 1993.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+ICD-10+Classification+of+Mental+and+Behavioural+Disorders:
Diagnostic+Criteria+for+Research&amp;publication_year=1993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R96">96. <span class="element-citation">McClellan JM, Susser E, King MC. Schizophrenia: a common disease caused by multiple rare
alleles. <span><span class="ref-journal">Br J Psychiatry. </span>2007;<span class="ref-vol">190</span>:194–199.</span> [<a href="/pubmed/17329737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Psychiatry&amp;title=Schizophrenia:+a+common+disease+caused+by+multiple+rare
alleles&amp;author=JM+McClellan&amp;author=E+Susser&amp;author=MC+King&amp;volume=190&amp;publication_year=2007&amp;pages=194-199&amp;pmid=17329737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3910086&amp;issue-id=233185&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139876053304256title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S21title">Results</a></li><li><a href="#S27title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S28title">Acknowledgments</a></li><li><a href="#idm139876057432288title">Footnotes</a></li><li><a href="#idm139876055345264title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3910086/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3910086/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3910086/pdf/nihms319115.pdf">PDF (562K)</a></li> | <li><a href="#" data-citationid="PMC3910086" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3910086%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3910086%2F&amp;text=Genomewide%20Association%20for%20Schizophrenia%20in%20the%20CATIE%20Study%3A%20Results%0Aof%20Stage%201" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3910086%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="18347602" data-aiid="3910086" data-aid="3910086" data-iid="233185" data-domainid="319" data-domain="nihpa" data-accid="PMC3910086" data-md5="94f9666d8994ff6e927f5c4b221cd0e4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-19T09:24:33-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE890EA4DD3FACF100000000003D0025&amp;ncbi_session=CE8B7BC8DD3EA361_1439SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3910086%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3910086/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8B7BC8DD3EA361_1439SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>