<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="mayoclinproc"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3127556/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Mayo Genome Consortia: A Genotype-Phenotype Resource for Genome-Wide Association Studies With an Application to the Analysis of Circulating Bilirubin Levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3127556/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Mayo Clinic Proceedings"><meta name="citation_title" content="Mayo Genome Consortia: A Genotype-Phenotype Resource for Genome-Wide Association Studies With an Application to the Analysis of Circulating Bilirubin Levels"><meta name="citation_authors" content="Suzette J. Bielinski, High Seng Chai, Jyotishman Pathak, Jayant A. Talwalkar, Paul J. Limburg, Rachel E. Gullerud, Hugues Sicotte, Eric W. Klee, Jason L. Ross, Jean-Pierre A. Kocher, Iftikhar J. Kullo, John A. Heit, Gloria M. Petersen, Mariza de Andrade, Christopher G. Chute"><meta name="citation_date" content="July 2011"><meta name="citation_issue" content="7"><meta name="citation_volume" content="86"><meta name="citation_firstpage" content="606"><meta name="citation_doi" content="10.4065/mcp.2011.0178"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3127556/?report=abstract"><meta name="citation_pmid" content="21646302"><meta name="DC.Title" content="Mayo Genome Consortia: A Genotype-Phenotype Resource for Genome-Wide Association Studies With an Application to the Analysis of Circulating Bilirubin Levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Mayo Foundation"><meta name="DC.Contributor" content="Suzette J. Bielinski"><meta name="DC.Contributor" content="High Seng Chai"><meta name="DC.Contributor" content="Jyotishman Pathak"><meta name="DC.Contributor" content="Jayant A. Talwalkar"><meta name="DC.Contributor" content="Paul J. Limburg"><meta name="DC.Contributor" content="Rachel E. Gullerud"><meta name="DC.Contributor" content="Hugues Sicotte"><meta name="DC.Contributor" content="Eric W. Klee"><meta name="DC.Contributor" content="Jason L. Ross"><meta name="DC.Contributor" content="Jean-Pierre A. Kocher"><meta name="DC.Contributor" content="Iftikhar J. Kullo"><meta name="DC.Contributor" content="John A. Heit"><meta name="DC.Contributor" content="Gloria M. Petersen"><meta name="DC.Contributor" content="Mariza de Andrade"><meta name="DC.Contributor" content="Christopher G. Chute"><meta name="DC.Date" content="2011 Jul"><meta name="DC.Identifier" content="10.4065/mcp.2011.0178"><meta name="DC.Language" content="en"><meta property="og:title" content="Mayo Genome Consortia: A Genotype-Phenotype Resource for Genome-Wide Association Studies With an Application to the Analysis of Circulating Bilirubin Levels"><meta property="og:type" content="article"><meta property="og:description" content="OBJECTIVE: To create a cohort for cost-effective genetic research, the Mayo Genome Consortia (MayoGC) has been assembled with participants from research studies across Mayo Clinic with high-throughput genetic data and electronic medical record (EMR) data ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3127556/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-mayoclinproc.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3127556/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3127556/pdf/mayoclinproc_86_7_004.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A0998DD5201A100000000086C0539.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127556%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/849/">Mayo Clin Proc</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/198113/">v.86(7); 2011 Jul</a></li><li class="accid">PMC3127556</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-mayoclinproc.gif" alt="Logo of mayoclinproc" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.mayoclinicproceedings.com/" target="pmc_ext" ref="reftype=publisher&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mayo Clin Proc</a></span>. 2011 Jul; 86(7): 606–614. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.4065%2Fmcp.2011.0178" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.4065/mcp.2011.0178</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3127556</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21646302">21646302</a></div></div></div></div><h1 class="content-title">Mayo Genome Consortia: A Genotype-Phenotype Resource for Genome-Wide Association Studies With an Application to the Analysis of Circulating Bilirubin Levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Bielinski%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Suzette J. Bielinski</a>, PhD, <a href="/pubmed/?term=Chai%20HS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">High Seng Chai</a>, PhD, <a href="/pubmed/?term=Pathak%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Jyotishman Pathak</a>, PhD, <a href="/pubmed/?term=Talwalkar%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Jayant A. Talwalkar</a>, MD, <a href="/pubmed/?term=Limburg%20PJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Paul J. Limburg</a>, MD, <a href="/pubmed/?term=Gullerud%20RE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Rachel E. Gullerud</a>, BA, <a href="/pubmed/?term=Sicotte%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Hugues Sicotte</a>, PhD, <a href="/pubmed/?term=Klee%20EW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Eric W. Klee</a>, PhD, <a href="/pubmed/?term=Ross%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Jason L. Ross</a>, MBA, <a href="/pubmed/?term=Kocher%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Jean-Pierre A. Kocher</a>, PhD, <a href="/pubmed/?term=Kullo%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Iftikhar J. Kullo</a>, MD, <a href="/pubmed/?term=Heit%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">John A. Heit</a>, MD, <a href="/pubmed/?term=Petersen%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Gloria M. Petersen</a>, PhD, <a href="/pubmed/?term=de%20Andrade%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Mariza de Andrade</a>, PhD, and  <a href="/pubmed/?term=Chute%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21646302">Christopher G. Chute</a>, MD, DrPH</div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140697031443248_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140697031443248_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140697031443248_ai" style="display:none"><div class="fm-affl" id="idm140697022880384" lang="en">From the Division of Epidemiology (S.J.B., G.M.P.), Division of Biomedical Statistics and Informatics (H.S.C., J.P., R.E.G., H.S., E.W.K., J.L.R., J.-P.A.K., M.d.A., C.G.C.), Division of Gastroenterology and Hepatology (J.A.T., P.J.L.), and Division of Cardiovascular Diseases and Gonda Vascular Center (I.J.K., J.A.H.), Mayo Clinic, Rochester, MN.</div><div id="idm140697022879632">Individual reprints of this article are not available. Address correspondence to Suzette J. Bielinski, PhD, Division of Epidemiology, Mayo Clinic, 200 First Street SW, Rochester, MN 55905 (<a href="mailto:dev@null" data-email="ude.oyam@ettezus.iksnileib" class="oemail">ude.oyam@ettezus.iksnileib</a>).</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140697031443248_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2011 Mayo Foundation for Medical Education and Research</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See "<a href="/pmc/articles/PMC3127553/">Genome-Wide Association Studies Go Green: Novel and Cost-Effective Opportunities for Identifying Genetic Associations</a>"  on&nbsp;page&nbsp;597.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC3127556/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supporting Online Material
            </div>
            
            <div><a href="/pmc/articles/PMC3127556/bin/supp_86_7_606__index.html" data-ga-action="click_feat_suppl">supp_86_7_606__index.html</a><span style="color:gray"> (696 bytes)</span></div><div class="small guid">GUID:&nbsp;1F899540-8E65-40B2-BCE7-6233BF777F3D</div>
            <div><a href="/pmc/articles/PMC3127556/bin/supp_mcp.2011.0178_BielinskiOnlineSupportingMaterial.pdf" data-ga-action="click_feat_suppl">supp_mcp.2011.0178_BielinskiOnlineSupportingMaterial.pdf</a><span style="color:gray"> (583K)</span></div><div class="small guid">GUID:&nbsp;3D2ECE05-7F8A-446B-9BC0-4CD0C96C947F</div>
         </div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Author Interview
            </div>
            
            <div><a href="/pmc/articles/PMC3127556/bin/supp_86_7_606_v2_index.html" data-ga-action="click_feat_suppl">supp_86_7_606_v2_index.html</a><span style="color:gray"> (863 bytes)</span></div><div class="small guid">GUID:&nbsp;6F0B2675-83C8-49E3-8B34-692113D302BE</div>
         </div></dd></dl></div><div id="idm140697028822736" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697028822736title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first">OBJECTIVE: <strong>To create a cohort for cost-effective genetic research, the Mayo Genome Consortia (MayoGC) has been assembled with participants from research studies across Mayo Clinic with high-throughput genetic data and electronic medical record (EMR) data for phenotype extraction.</strong></p><p id="P2">PARTICIPANTS AND METHODS: <strong>Eligible participants include those who gave general research consent in the contributing studies to share high-throughput genotyping data with other investigators. Herein, we describe the design of the MayoGC, including the current participating cohorts, expansion efforts, data processing, and study management and organization. A genome-wide association study to identify genetic variants associated with total bilirubin levels was conducted to test the genetic research capability of the MayoGC.</strong></p><p id="P3">RESULTS: <strong>Genome-wide significant results were observed on 2q37 (top single nucleotide polymorphism, rs4148325; <em>P</em>=5.0 × 10<sup>–62</sup>) and 12p12 (top single nucleotide polymorphism, rs4363657; <em>P</em>=5.1 × 10<sup>–8</sup>) corresponding to a gene cluster of uridine 5′-diphospho-glucuronosyltransferases (the <em>UGT1A cluster</em>) and solute carrier organic anion transporter family, member 1B1 (<em>SLCO1B1</em>), respectively.</strong></p><p id="P4" class="p p-last">CONCLUSION: <strong>Genome-wide association studies have identified genetic variants associated with numerous phenotypes but have been historically limited by inadequate sample size due to costly genotyping and phenotyping. Large consortia with harmonized genotype data have been assembled to attain sufficient statistical power, but phenotyping remains a rate-limiting factor in gene discovery research efforts. The EMR consists of an abundance of phenotype data that can be extracted in a relatively quick and systematic manner. The MayoGC provides a model of a unique collaborative effort in the environment of a common EMR for the investigation of genetic determinants of diseases.</strong></p></div></div><div id="idm140697032924512" class="tsec sec headless whole_rhythm"><div class="boxed-text-box whole_rhythm hide-overflow"><p id="__p1">ABI = ankle-brachial index; BORA = Biologically Oriented Repository Architectures; DUA = data use agreement; eMERGE = Electronic Medical Records and Genomics; EMR = electronic medical record; GENEVA = Gene Environment Association Studies; GWA = genome-wide association; MayoGC = Mayo Genome Consortia; PAD = peripheral arterial disease; SNP = single nucleotide polymorphism</p></div><p id="P5" class="p p-first"><strong>G</strong>enome-wide association (GWA) studies have identified genetic variants associated with numerous diseases and phenotypes. This agnostic approach to discovering genetic variants important to diseases has been historically limited by inadequate sample size due to costly genotyping and phenotyping. Traditionally, GWA studies have been conducted in research cohorts designed to capture and measure outcomes and phenotypes specific to one disease domain. This has prompted the formation of large consortia comprising several research cohorts with harmonized genotype data and similar phenotype measurements to attain the statistical power required for genetic variant discovery.</p><p id="P6">Despite the high initial cost of genotyping, the static nature of germline genotypes allows such data to be mined and fitted to pursue various hypotheses, including those unrelated to the original study. As such, a collection of genotype data from existing GWA studies would be a valuable cost-saving resource for genetic research. In contrast, phenotype acquisition and standardization across different studies, which has been time-consuming, laborious, and prone to error, is a rate-limiting factor in gene discovery research efforts. The electronic medical record (EMR) consists of an abundance of phenotype data that can be extracted in a relatively quick and systematic manner. The Electronic Medical Records and Genomics (eMERGE) Network of the National Human Genome Research Institute (for more information, see <a href="http://www.gwas.org">http://www.gwas.org</a>) is a national consortium formed to develop, disseminate, and apply approaches for genetic research within EMR systems.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239917">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239912">2</a></sup> Successful use of this approach in the eMERGE Network has inspired the creation of the intramural Mayo Genome Consortia (MayoGC). The goal of the MayoGC is to assemble a large cohort of participants from research studies across Mayo Clinic with high-throughput genetic data and to use EMR for phenotype extraction for cost-effective genetic research.</p><p id="P7" class="p">Herein, we describe the design of the MayoGC, including the current participating cohorts, expansion efforts, data processing, and study management and organization. As a test of the genetic research capability of the MayoGC, we conducted a GWA study to identify genetic variants associated with total bilirubin levels. Bilirubin levels have a large variability in the population, with heritability of roughly 0.50.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239921">3</a></sup> Two previous GWA studies identified variants from similar genomic locations with strong and moderate effects on bilirubin levels,<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239913">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239910">5</a></sup> making this phenotype an ideal candidate for testing. The MayoGC provides a model of a unique collaborative effort in the environment of a common EMR for the investigation of genetic determinants of diseases.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">PARTICIPANTS AND METHODS</h2><p id="P8" class="p p-first">MayoGC is a large cohort of Mayo Clinic patients with EMR and genotype data. Eligible participants include those who gave general research (ie, not disease-specific) consent in the contributing studies to share high-throughput genotyping data with other investigators. This cohort is being built in 2 phases. Phase 1, which has been completed, includes participants from 3 studies funded by the National Institutes of Health, which sought to identify genetic determinants of peripheral arterial disease (PAD), venous thromboembolism, and pancreatic cancer, respectively, with a combined total sample size of 6307 unique participants (<a href="/pmc/articles/PMC3127556/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The eMERGE study contributed genotype data for 3336 participants with PAD and control participants recruited from Mayo Clinic's noninvasive vascular and exercise stress testing laboratories, respectively.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239926">2</a></sup> Peripheral arterial disease was defined by documentation of at least 1 of the following: (1) an ankle-brachial index (ABI) of 0.9 or less at rest or 1 minute after exercise, (2) the presence of poorly compressible arteries, or (3) a normal ABI but history of revascularization for PAD. Control participants had a normal ABI and no history of PAD.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239924">2</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>TABLE 1.</h3><!--caption a7--><div class="caption"><p id="__p2"><strong>MayoGC Phase 1 Studies<sup>a,b</sup></strong></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697010566304" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.tbl1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.tbl1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.tbl1.jpg"></a></div><div id="largeobj_idm140697010566304" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3127556/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P9">The GENEVA (Gene Environment Association Studies) Study of Venous Thromboembolism of the National Human Genome Research Institute enrolled consecutive Mayo Clinic outpatients with objectively diagnosed deep venous thrombosis and/or pulmonary embolism who resided in the upper Midwest and had been referred by a Mayo Clinic physician to the Mayo Clinic Special Coagulation Laboratory or to the Mayo Clinic Thrombophilia Center.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239919">6</a></sup> A deep venous thrombosis or pulmonary embolism was categorized as objectively diagnosed (1) when it was confirmed by venography or pulmonary angiography or via a pathology examination of a thrombus removed at surgery or (2) if findings on at least 1 noninvasive test (compression duplex ultrasonography, lung scan, computed tomography, magnetic resonance imaging) were positive. Persons with venous thromboembolism related to active cancer were excluded. A control group was prospectively recruited for this study. Control participants were frequency-matched to the age group (18-29 years, 30-39 years, 40-49 years, 50-59 years, 60-69 years, 70-79 years, and ≥80 years), sex, myocardial infarction or stroke status, and state of residence distribution of the cases. The study selected clinic-based controls using a database of persons undergoing general medical examinations in the Mayo Clinic Divisions of General Internal Medicine and Primary Care Internal Medicine. Additionally, persons undergoing evaluation at the Mayo Clinic Sports Medicine Center and the Department of Family Medicine were screened for inclusion as control participants. Genotype data for 2497 participants were contributed by the GENEVA study.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239928">6</a></sup></p><p id="P10">The Mayo Clinic Molecular Epidemiology of Pancreatic Cancer Study contributed genotype data for 613 control participants to the MayoGC. Details of this study have been described previously.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239909">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239923">8</a></sup> In brief, patients scheduled for a general medical examination between May 2004 and February 2007 were recruited from the Divisions of General Internal Medicine and Primary Care Internal Medicine. Control participants were frequency-matched to pancreatic cancer cases on the basis of sex, state of residence, age at recruitment, and race and ethnicity. At the time of recruitment, control participants had no personal history of cancer except nonmelanoma skin cancer.</p><p id="P11" class="p">Phase 2 is under way with the goal of expanding the MayoGC by recruiting eligible patients from other studies funded by the National Institutes of Health (<a href="/pmc/articles/PMC3127556/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). All of the MayoGC studies were approved by the Mayo Clinic Institutional Review Board, and the participants from each involved study provided written and informed consent for general research.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>TABLE 2.</h3><!--caption a7--><div class="caption"><p id="__p3"><strong>Studies Participating in the MayoGC</strong></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697011617472" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.tbl2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.tbl2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.tbl2.jpg"></a></div><div id="largeobj_idm140697011617472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3127556/table/T2/?report=objectonly">Open in a separate window</a></div></div><div id="S2" class="sec"><h3 id="S2title">M<span class="small-caps">ayo</span>GC O<span class="small-caps">rganization</span></h3><p id="P12" class="p p-first-last">The MayoGC represents a voluntary collaboration of investigators across disciplines at Mayo Clinic. The organizational structure comprises a MayoGC Research Group and a Steering Committee. The Research Group is a collaboration of investigators with expertise in epidemiology, statistical genetics, and bioinformatics who are responsible for the creation, implementation, and maintenance of the consortia. Specifically, members of this group are responsible for study recruitment, development of policies and procedures for widespread use of MayoGC data, harmonization of genetic and phenotype data, and maintenance and storage of study data. The Steering Committee consists of a member from each of the contributing studies and members of the Research Group. This committee is responsible for approving all study policies and procedures and recommending modifications to operational policy as needed. Furthermore, the committee reviews all research proposals that use MayoGC data. Via its representative on the Steering Committee, each contributing study may formally opt out of participation in research proposals that are in conflict with its goals.</p></div><div id="S3" class="sec"><h3 id="S3title">M<span class="small-caps">ayo</span> C<span class="small-caps">linic</span> B<span class="small-caps">iobank</span></h3><p id="P13" class="p p-first-last">The MayoGC is a passive collection of existing data and, as such, does not have stored biological samples on participants in the consortia. However, 6% of the participants in phase 1 were also enrolled in the Mayo Clinic Biobank. The Mayo Clinic Biobank (for more information, see <a href="http://mayoresearch.mayo.edu/biobank/">http://mayoresearch.mayo.edu/biobank/</a>) has enrolled more than 18,000 participants and has the goal of reaching 20,000 participants by the end of 2011 in an effort to support a wide array of health-related research studies throughout the institution. Study participants provide a blood sample for DNA and serum/plasma research, complete a health risk questionnaire, allow access to medical records, and consent to prospective follow-up for health outcomes. Thus, for the subset of the MayoGC participants in the Biobank, biological specimens are available. Likewise, genotype data is available through the MayoGC for users of the Biobank.</p></div><div id="S4" class="sec"><h3 id="S4title">D<span class="small-caps">ata</span> U<span class="small-caps">se</span> A<span class="small-caps">greement</span></h3><p id="P14" class="p p-first-last">To protect the confidentiality and privacy of the MayoGC, investigators who are granted access to MayoGC data will operate under a data use agreement (DUA). The DUA describes the terms and conditions for the following: data use and transferability, publication, termination/expiration of the data agreement, obtaining of informed consent from study participants, and compliance by data recipients with the requirements of the institutional review board of their home institution. Further, the DUA outlines the procedures for amending a current agreement and specifies how the failure to comply with the terms of the DUA will be addressed.</p></div><div id="S5" class="sec"><h3 id="S5title">B<span class="small-caps">ilirubin</span> P<span class="small-caps">henotype</span>—A GWA A<span class="small-caps">pplication</span></h3><p id="P15" class="p p-first-last">For phase 1 of the MayoGC, all bilirubin levels clinically ordered from January 1, 1994, through August 31, 2010, were retrieved from a structured laboratory database. Extracted data included the test code and description, the date and time of the sample, the units of results, the associated reference range and indicators for low and high results, the laboratory accession number, and the results of the test in both character and numeric format. Bilirubin measurements were available for 4195 participants. Because our hypothesis focused on identifying genetic variants that affect bilirubin levels within the normal range, we excluded 726 participants who had at least 1 abnormal bilirubin level from the primary analysis. Of the 3469 participants with normal bilirubin levels, 58% had serial measurements of bilirubin. This analysis includes only the first-ever bilirubin level measured after age 18 years. To further explore the genetic effects on bilirubin levels, secondary analyses included a GWA study of all 4195 participants without exclusions and 2 subsets in which we excluded participants with any abnormal test results for alanine aminotransferase, alkaline phosphatase, aspirate aminotransferase, or γ-glutamyltransferase on the same day as the bilirubin measurement (n=2427) or those without any abnormal liver test results within 1 year before or after the bilirubin measurement (n=2191).</p></div><div id="S6" class="sec"><h3 id="S6title">G<span class="small-caps">enotype</span> H<span class="small-caps">armonization</span></h3><p id="P16" class="p p-first">Participants in the eMERGE Network and GENEVA were genotyped using Illumina HumanHap660-Quad chips (Illumina, San Diego, CA) (<a href="/pmc/articles/PMC3127556/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Participants from the Mayo Clinic Molecular Epidemiology of Pancreatic Cancer Study were typed on the HumanHap550 and the Human 610-Quad chips.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239916">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239920">8</a></sup> PLINK files, in which genotypes were coded as the number of minor alleles, were provided by the individual studies. Of the participants in phase 1 of the MayoGC, 60 also participated in 2 of the contributing studies, and 1 was involved in all 3 studies. These duplicated samples were used to check genotype concordance across studies as well as to inform flipping of minor allele/strand as necessary. Single nucleotide polymorphisms (SNPs) that had more than 2 discordant genotypes among the duplicates or that were monomorphic in all phase 1 samples were excluded from the dataset. Other commonly used SNP-filtering criteria were not imposed because of their dependency on the samples used for a specific hypothesis (ie, the availability of EMR data).</p><p id="P17" class="p">We retained 1 duplicated sample with more nonmissing genotypes from each of the 61 study participants. PLINK was then used for sample-wise quality control on the remaining data. Relatedness was determined on the basis of identity-by-descent estimates generated from the “--Zgenome” option in PLINK. Genotype data of trios from the Centre d'Etude du Polymorphisme Humain collected in Utah, USA, with ancestry from northern and western Europe were used to define an identity-by-descent threshold for relatedness. No cryptic relatedness was detected. Because the patterns of missingness differed somewhat across studies, we excluded samples on the basis of the notion of within-study outlier. This approach eliminated 12 samples with a comparably high missing genotype rate. On the basis of the genotype data, a large majority of the participants in phase 1 of MayoGC were of European ancestry. The 48 samples in which population admixture was evident were filtered out. Cross-checking of the sex of study participants identified and removed 5 with a mismatch between the reported and deduced sex. These quality controls left 6307 study participants in whom 583,129 SNPs were available for analysis. No SNP-wise quality control filtering based on Hardy-Weinberg equilibrium or minor allele frequency was completed at this stage because these measures may change depending on the sample with the phenotype of interest.</p></div><div id="S7" class="sec"><h3 id="S7title">D<span class="small-caps">ata</span> M<span class="small-caps">anagement and</span> D<span class="small-caps">ata</span> S<span class="small-caps">ecurity</span></h3><p id="P18" class="p p-first-last">The Bioinformatics Core at Mayo Clinic has developed a system for the storage of processed genomics data. The Biologically Oriented Repository Architectures (BORA) system includes components to facilitate the processing and analysis of data produced by high-throughput genomics platforms. BORA features fully deployed methods for data security and access control, including user authentication and user authorization. Authentication involves validation of the institution's user ID and password and verification that the user ID is included in the BORA list of authorized users. Authorization of data use is defined on a user and user group level. A user or user group is provided access to explicitly defined samples or sample sets. Designation of specific user or user group authorization to access the MayoGC data is approved by the MayoGC Steering Committee. The BORA system stores and provides access to MayoGC genomics data that have been quality controlled and imputed following the described procedures.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">S<span class="small-caps">tatistical</span> A<span class="small-caps">nalyses</span></h3><p id="P19" class="p p-first-last">Genotype association with bilirubin levels was performed using a linear regression model implemented in PLINK. Age at the time of first bilirubin measurement in adulthood and sex were accommodated in the model as covariates. Latent variables indicative of study of origin were also included to avoid study-induced bias. Bilirubin values were log transformed to accommodate skewness. <em>P</em>&lt;10<sup>–6</sup> is deemed to have genome-wide significance.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">RESULTS</h2><p id="P20" class="p p-first">The MayoGC phase 1 population is 44% female, has a mean age of 60.3 years, and is predominately white (<a href="/pmc/articles/PMC3127556/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). As a whole, the cohort is a rich source of phenotypes, with a mean medical record length of 22.4 years.</p><p id="P21" class="p">Bilirubin levels (0.1-1.1 mg/dL [to convert to μmol/L, multiply by 17.104]) were available for 3469 participants: 1726 in the eMERGE Network, 1316 in GENEVA, and 427 with pancreatic cancer. Of participants in this group, 47% were women, the mean ± SD age at the time of first bilirubin measurement in adulthood was 57±12.3 years, and the mean ± SD bilirubin level was 0.56±0.20 mg/dL. All studies showed significantly higher bilirubin levels in men (<em>P</em>&lt;.001). The top GWA results were similar with and without adjustment for study; thus, only age- and sex-adjusted results are reported (<a href="/pmc/articles/PMC3127556/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Genome-wide significant results were observed on 2q37 (top SNP, rs4148325; <em>P</em>=5.0 × 10<sup>–62</sup>) and 12p12 (top SNP, rs4363657; <em>P</em>=5.1 × 10<sup>–8</sup>), corresponding to a gene cluster of uridine 5′-diphospho-glucuronosyltransferases (the <em>UGT1A</em> cluster) and solute carrier organic anion transporter family, member 1B1 (<em>SLCO1B1</em>), respectively. <a href="/pmc/articles/PMC3127556/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> contains a complete list of significant results, and Figures <a href="/pmc/articles/PMC3127556/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC3127556/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> show the regional association plots for chromosomes 2 and 12. Figures S1 through S3 (see Supporting Online Material, a link to which is provided at the end of this article) illustrate that the signal on chromosome 2 was robust to different exclusion criteria because it was highly significant both for all study participants with bilirubin measurements (N=4195) and for the 2 subsets restricted to participants with normal liver enzymes either measured on the same day as the bilirubin (n=2427) or within 1 year before or after bilirubin measurement (n=2191).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3127556/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697018143456" class="figure"><a href="/pmc/articles/PMC3127556/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.fig1.jpg"></a></div><div id="largeobj_idm140697018143456" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3127556/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3127556/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">FIGURE 1.</a></div><!--caption a7--><div class="caption"><p id="__p4">Genome-wide –log<sub>10</sub> <em>P</em>-value plot for total serum bilirubin levels. Solid horizontal line represents significant level of <em>P</em>=10<sup>–6</sup>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>TABLE 3.</h3><!--caption a7--><div class="caption"><p id="__p5"><strong>Associations of the Top SNPs (<em>P</em>&lt;1.0 x 10<sup>–6</sup>) With Bilirubin Levels<sup>a</sup></strong></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697028849680" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.tbl3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.tbl3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.tbl3.jpg"></a></div><div id="largeobj_idm140697028849680" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3127556/table/T3/?report=objectonly">Open in a separate window</a></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3127556/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697033569552" class="figure"><a href="/pmc/articles/PMC3127556/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.fig2.jpg"></a></div><div id="largeobj_idm140697033569552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3127556/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3127556/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">FIGURE 2.</a></div><!--caption a7--><div class="caption"><p id="__p6">Regional view of <em>UGT1A</em> cluster for associations with total serum bilirubin levels. The gene symbols used here follow the recommendations of the HUGO Gene Nomenclature Committee (HGNC); expansions of the symbols can be searched on the HGNC Web site (<a href="http://www.genenames.org">http://www.genenames.org</a>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3127556/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140697022912464" class="figure"><a href="/pmc/articles/PMC3127556/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3127556_606.fig3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 606.fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3127556/bin/606.fig3.jpg"></a></div><div id="largeobj_idm140697022912464" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3127556/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3127556/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">FIGURE 3.</a></div><!--caption a7--><div class="caption"><p id="__p7">Regional view of <em>SLCO1B1</em> region for associations with total serum bilirubin levels. The gene symbols used here follow the recommendations of the HUGO Gene Nomenclature Committee (HGNC); expansions of the symbols can be searched on the HGNC Web site (<a href="http://www.genenames.org">http://www.genenames.org</a>).</p></div></div></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P22" class="p p-first">Inspired by the National Center for Biotechnology Information's Database of Genotypes and Phenotypes, the MayoGC represents a model for genetic research that capitalizes on existing genetic data in the environment of a common EMR. Furthermore, this cost-effective approach expands the use of genetic data, derived from available data within the medical record, to phenotypes unrelated to the original study. The utility of the MayoGC is demonstrated by genome-wide significant results for clinically ordered serum bilirubin levels that replicate findings from several cohort studies. In addition to classic gene discovery, other applications of the MayoGC resource include replication of results from other studies, association of SNPs with multiple phenotypes, and pilot studies for rare diseases.</p><p id="P23">Recruitment is under way for phase 2 of MayoGC, which is intended to expand the cohort and to develop the infrastructure and processes needed for wider and long-term use of the resource. <a href="/pmc/articles/PMC3127556/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> lists the cohorts that have agreed to participate. Phase 2 is expected to be completed by the end of 2011 and will include more than 10,000 participants with harmonized and imputed high-throughput genetic data. This effort requires processing of a greater variety of platforms. Thus, a fully automated workflow for forward strand mapping and imputation of genotyping data will be implemented. The workflow will standardize the genotyping data stored in BORA to enable integrated analysis of genotypes from different platforms and from multiple studies collected at Mayo Clinic. In addition, current genotyping-focused quality control procedures will be extended with generalized procedures for population structure validation as well as patient relationship cross-checking to monitor for duplicates and relatedness. BORA is also being interfaced with the Enterprise Data Trust, a central data warehouse in which minable clinical information on Mayo Clinic's patients will be stored and can be accessed. The interface between the 2 systems will greatly facilitate phenotype-genotype association and replication studies.</p><p id="P24">Significant associations were observed on chromosome 2 corresponding to the <em>UGT1A</em> cluster (<em>P</em>=5.0 × 10<sup>–62</sup>) and chromosome 12 in the <em>SLCO1B1</em> gene (<em>P</em>=5.1 × 10<sup>–8</sup>), confirming previous results in GWA and linkage studies.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239927">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239922">5</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239918">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239908">10</a></sup> The <em>UGT1A</em> cluster encodes 9 transferase genes (<em>UGT1A<sup>*</sup> 1, 3, 4, 5, 6, 7, 8, 9, 10</em>), of which 8 are known active gene transcripts. All of the UGT1A proteins are identical in the amino acid sequence encoded by exons 2 through 5 with unique alternate first exons.<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239929">11</a></sup> The glucuronidation of bilirubin by <em>UGT1A</em> and specifically by <em>UGT1A1</em> is essential for the elimination of bilirubin from the body.<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239914">12</a></sup> Three heritable forms of hyperbilirubinemia (ie, Crigler-Najjar syndrome types 1 and 2 and Gilbert syndrome) all result from mutations in <em>UGT1A1</em>, and candidate gene studies in multiple ethnic populations have shown that <em>UGT1A1</em> is the major gene influencing total bilirubin levels.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239911">13</a>-<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239925">17</a></sup> The <em>SLCO1B1</em> locus encodes a protein that mediates sodium-dependent uptake of bilirubin. Although not as influential as <em>UGT1A1, SLCO1B1</em> is a major contributor to total bilirubin levels and variations in it have been associated with hyperbilirubinemia.<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239915">18</a>-<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_226239907">20</a></sup></p><p id="P25" class="p p-last">The MayoGC is a collaborative effort for data sharing and, as such, has no stored DNA for the study participants for de novo genotyping. Likewise, phenotyping is restricted to data available within the medical record. For a subset of MayoGC participants, biological specimens are available through the Mayo Clinic Biobank. Biological specimens may also be available in the contributing studies; however, large-scale de novo measurements are likely not practical. The ability to contact and consent participants for additional measurements is possible for MayoGC participants whose contributing study consent did not prohibit recontact or for those who are enrolled in the Mayo Clinic Biobank. Despite these limitations, which are common to genetic consortia, the MayoGC leverages the success of EMR-derived phenotyping with high-throughput genetic data obtained at considerable research costs to enhance genetic research. Furthermore, the relatively stable population in Olmsted County, Minnesota, the county in which Mayo Clinic in Rochester is located, and surrounding counties and the ability to search EMR data spanning decades are unique features of Mayo Clinic that enhance the utility of the MayoGC. However, the MayoGC can serve as a template for other institutions with EMR-phenotyping capability.</p></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">CONCLUSION</h2><p id="P26" class="p p-first-last">Genome-wide association studies have identified genetic variants associated with numerous phenotypes but have been limited historically by inadequate sample size due to costly genotyping and phenotyping. Large consortia with harmonized genotype data have been assembled to attain sufficient statistical power, but phenotyping remains a rate-limiting factor in gene discovery research efforts. The EMR offers an abundance of phenotype data that can be extracted in a relatively quick and systematic manner. The utility of the MayoGC was successfully demonstrated by the replication of GWA results for bilirubin levels obtained from the EMR. Thus, MayoGC provides a model of a unique collaborative effort in the environment of a common EMR for the investigation of genetic determinants of diseases.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supporting Online Material:</strong>
            
            <div class="sup-box half_rhythm" id="idm140697101834304"><a href="/pmc/articles/PMC3127556/bin/supp_86_7_606__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div><div class="sec suppmat" id="PMC_2">
            
               <!--caption a9--><strong>Author Interview:</strong>
            
            <div class="sup-box half_rhythm" id="idm140697027366016"><a href="/pmc/articles/PMC3127556/bin/supp_86_7_606_v2_index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
         </div></div><div id="idm140697024914208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697024914208title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140697024914080"></p><p id="__p8" class="p p-first-last">Funding for this research was provided by the Electronic Medical Records and Genomics (eMERGE) Network of the National Human Genome Research Institute (HG05499), Mayo Clinic Genome-wide Association Study of Venous Thromboembolism (HG04735) from the National Human Genome Research Institute (NHGRI; Gene Environment Association Studies [GENEVA] consortium), Mayo Clinic Specialized Programs of Research Excellence (SPORE) in Pancreatic Cancer (P50CA102701) from the National Cancer Institute, and Mayo Clinic Cancer Center (Genetic Epidemiology and Risk Assessment [GERA] Program). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. Dr Limburg served as a consultant for Genomic Health, Inc from August 12, 2008, through April 19, 2010. Mayo Clinic has licensed Dr Limburg's intellectual property to Exact Sciences, and he and Mayo Clinic have contractual rights to receive royalties through this agreement.</p><p></p><p class="fn sec" id="idm140697027410560"></p><p id="__p9" class="p p-first-last">An earlier version of this article appeared Online First.</p><p></p><p class="fn sec" id="idm140697027409936"></p><p id="__p10" class="p p-first-last"><strong>For editorial comment, see page <a href="/pmc/articles/PMC3127553/">597</a></strong></p><p></p><p class="fn sec" id="idm140697024679552"></p><p id="__p11" class="p p-first"><strong>Supporting Online Material</strong></p><p id="__p12" class="p p-last"><a href="http://www.mayoclinicproceedings.com/content/86/7/606/suppl/DC1">www.mayoclinicproceedings.com/content/86/7/606/suppl/DC1</a> Figures S1-S3</p><p></p></div></div><div id="idm140697065174080" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140697065174080title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">
McCarty CA, Chisholm RL, Chute CG, et al. ; eMerge Team <span class="ref-title">The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies</span>. <span class="ref-journal"><em>BMC Med Genomics</em></span>. 2011;<span class="ref-vol">4</span>(<span class="ref-iss">1</span>):13
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038887/">PMC free article</a>]</span> [<a href="/pubmed/21269473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genomics&amp;title=The+eMERGE+Network:+a+consortium+of+biorepositories+linked+to+electronic+medical+records+data+for+conducting+genomic+studies&amp;author=CA+McCarty&amp;author=RL+Chisholm&amp;author=CG+Chute&amp;volume=4&amp;issue=1&amp;publication_year=2011&amp;pages=13&amp;pmid=21269473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">
Kullo IJ, Fan J, Pathak J, Savova GK, Ali Z, Chute CG. <span class="ref-title">Leveraging informatics for genetic studies: use of the electronic medical record to enable a genome-wide association study of peripheral arterial disease</span>. <span class="ref-journal"><em>J Am Med Inform Assoc</em></span>. 2010;<span class="ref-vol">17</span>(<span class="ref-iss">5</span>):568-574
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2995686/">PMC free article</a>]</span> [<a href="/pubmed/20819866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Inform+Assoc&amp;title=Leveraging+informatics+for+genetic+studies:+use+of+the+electronic+medical+record+to+enable+a+genome-wide+association+study+of+peripheral+arterial+disease&amp;author=IJ+Kullo&amp;author=J+Fan&amp;author=J+Pathak&amp;author=GK+Savova&amp;author=Z+Ali&amp;volume=17&amp;issue=5&amp;publication_year=2010&amp;pages=568-574&amp;pmid=20819866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">
Bathum L, Petersen HC, Rosholm JU, Hyltoft Petersen P, Vaupel J, Christensen K. <span class="ref-title">Evidence for a substantial genetic influence on biochemical liver function tests: results from a population-based Danish twin study</span>. <span class="ref-journal"><em>Clin Chem</em></span>. 2001;<span class="ref-vol">47</span>(<span class="ref-iss">1</span>):81-87
 [<a href="/pubmed/11148181" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Evidence+for+a+substantial+genetic+influence+on+biochemical+liver+function+tests:+results+from+a+population-based+Danish+twin+study&amp;author=L+Bathum&amp;author=HC+Petersen&amp;author=JU+Rosholm&amp;author=P+Hyltoft+Petersen&amp;author=J+Vaupel&amp;volume=47&amp;issue=1&amp;publication_year=2001&amp;pages=81-87&amp;pmid=11148181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">
Johnson AD, Kavousi M, Smith AV, et al.  <span class="ref-title">Genome-wide association meta-analysis for total serum bilirubin levels</span>. <span class="ref-journal"><em>Hum Mol Genet</em></span>. 2009;<span class="ref-vol">18</span>:2700-2710
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701336/">PMC free article</a>]</span> [<a href="/pubmed/19414484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+meta-analysis+for+total+serum+bilirubin+levels&amp;author=AD+Johnson&amp;author=M+Kavousi&amp;author=AV+Smith&amp;volume=18&amp;publication_year=2009&amp;pages=2700-2710&amp;pmid=19414484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">
Sanna S, Busonero F, Maschio A, et al.  <span class="ref-title">Common variants in the SLCO1B3 locus are associated with bilirubin levels and unconjugated hyperbilirubinemia</span>. <span class="ref-journal"><em>Hum Mol Genet</em></span>. 2009;<span class="ref-vol">18</span>:2711-2718
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701337/">PMC free article</a>]</span> [<a href="/pubmed/19419973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+variants+in+the+SLCO1B3+locus+are+associated+with+bilirubin+levels+and+unconjugated+hyperbilirubinemia&amp;author=S+Sanna&amp;author=F+Busonero&amp;author=A+Maschio&amp;volume=18&amp;publication_year=2009&amp;pages=2711-2718&amp;pmid=19419973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">
Cornelis MC, Agrawal A, Cole JW, et al.  <span class="ref-title">The Gene, Environment Association Studies consortium (GENEVA): maximizing the knowledge obtained from GWAS by collaboration across studies of multiple conditions</span>. <span class="ref-journal"><em>Genet Epidemiol</em></span>. 2010;<span class="ref-vol">34</span>(<span class="ref-iss">4</span>):364-372
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860056/">PMC free article</a>]</span> [<a href="/pubmed/20091798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=The+Gene,+Environment+Association+Studies+consortium+(GENEVA):+maximizing+the+knowledge+obtained+from+GWAS+by+collaboration+across+studies+of+multiple+conditions&amp;author=MC+Cornelis&amp;author=A+Agrawal&amp;author=JW+Cole&amp;volume=34&amp;issue=4&amp;publication_year=2010&amp;pages=364-372&amp;pmid=20091798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">
Amundadottir L, Kraft P, Stolzenberg-Solomon RZ, et al.  <span class="ref-title">Genome-wide association study identifies variants in the ABO locus associated with susceptibility to pancreatic cancer</span>. <span class="ref-journal"><em>Nat Genet</em></span>. 2009;<span class="ref-vol">41</span>(<span class="ref-iss">9</span>):986-990
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2839871/">PMC free article</a>]</span> [<a href="/pubmed/19648918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+in+the+ABO+locus+associated+with+susceptibility+to+pancreatic+cancer&amp;author=L+Amundadottir&amp;author=P+Kraft&amp;author=RZ+Stolzenberg-Solomon&amp;volume=41&amp;issue=9&amp;publication_year=2009&amp;pages=986-990&amp;pmid=19648918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">
Petersen GM, Amundadottir L, Fuchs CS, et al.  <span class="ref-title">A genome-wide association study identifies pancreatic cancer susceptibility loci on chromosomes 13q22.1, 1q32.1 and 5p15.33</span>. <span class="ref-journal"><em>Nat Genet</em></span>. 2010;<span class="ref-vol">42</span>(<span class="ref-iss">3</span>):224-228
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2853179/">PMC free article</a>]</span> [<a href="/pubmed/20101243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+pancreatic+cancer+susceptibility+loci+on+chromosomes+13q22.1,+1q32.1+and+5p15.33&amp;author=GM+Petersen&amp;author=L+Amundadottir&amp;author=CS+Fuchs&amp;volume=42&amp;issue=3&amp;publication_year=2010&amp;pages=224-228&amp;pmid=20101243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">
Kang TW, Kim HJ, Ju H, et al.  <span class="ref-title">Genome-wide association of serum bilirubin levels in Korean population</span>. <span class="ref-journal"><em>Hum Mol Genet</em></span>. 2010;<span class="ref-vol">19</span>(<span class="ref-iss">18</span>):3672-3678
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928134/">PMC free article</a>]</span> [<a href="/pubmed/20639394" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+of+serum+bilirubin+levels+in+Korean+population&amp;author=TW+Kang&amp;author=HJ+Kim&amp;author=H+Ju&amp;volume=19&amp;issue=18&amp;publication_year=2010&amp;pages=3672-3678&amp;pmid=20639394&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">
Melton PE, Haack K, Göring HH, et al.  <span class="ref-title">Genetic influences on serum bilirubin in American Indians: the Strong Heart Family Study</span>. <span class="ref-journal"><em>Am J Hum Biol</em></span>. 2011;<span class="ref-vol">23</span>(<span class="ref-iss">1</span>):118-125
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3046552/">PMC free article</a>]</span> [<a href="/pubmed/21080475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Biol&amp;title=Genetic+influences+on+serum+bilirubin+in+American+Indians:+the+Strong+Heart+Family+Study&amp;author=PE+Melton&amp;author=K+Haack&amp;author=HH+Göring&amp;volume=23&amp;issue=1&amp;publication_year=2011&amp;pages=118-125&amp;pmid=21080475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">
Tukey RH, Strassburg CP. <span class="ref-title">Human UDP-glucuronosyltransferases: metabolism, expression, and disease</span>. <span class="ref-journal"><em>Annu Rev Pharmacol Toxicol</em></span>. 2000;<span class="ref-vol">40</span>:581-616
 [<a href="/pubmed/10836148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Pharmacol+Toxicol&amp;title=Human+UDP-glucuronosyltransferases:+metabolism,+expression,+and+disease&amp;author=RH+Tukey&amp;author=CP+Strassburg&amp;volume=40&amp;publication_year=2000&amp;pages=581-616&amp;pmid=10836148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">
Ghosh SS, Lu Y, Lee SW, et al.  <span class="ref-title">Role of cysteine residues in the function of human UDP glucuronosyltransferase isoform 1A1 (UGT1A1)</span>. <span class="ref-journal"><em>Biochem J</em></span>. 2005;<span class="ref-vol">392</span>(<span class="ref-iss">3</span>):685-692
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1316310/">PMC free article</a>]</span> [<a href="/pubmed/16171463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Role+of+cysteine+residues+in+the+function+of+human+UDP+glucuronosyltransferase+isoform+1A1+(UGT1A1)&amp;author=SS+Ghosh&amp;author=Y+Lu&amp;author=SW+Lee&amp;volume=392&amp;issue=3&amp;publication_year=2005&amp;pages=685-692&amp;pmid=16171463&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">
Hong AL, Huo D, Kim HJ, et al.  <span class="ref-title">UDP-glucuronosyltransferase 1A1 gene polymorphisms and total bilirubin levels in an ethnically diverse cohort of women</span>. <span class="ref-journal"><em>Drug Metab Dispos</em></span>. 2007;<span class="ref-vol">35</span>(<span class="ref-iss">8</span>):1254-1261
 [<a href="/pubmed/17478602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Metab+Dispos&amp;title=UDP-glucuronosyltransferase+1A1+gene+polymorphisms+and+total+bilirubin+levels+in+an+ethnically+diverse+cohort+of+women&amp;author=AL+Hong&amp;author=D+Huo&amp;author=HJ+Kim&amp;volume=35&amp;issue=8&amp;publication_year=2007&amp;pages=1254-1261&amp;pmid=17478602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">
Lin JP, O'Donnell CJ, Schwaiger JP, et al.  <span class="ref-title">Association between the UG-T1A1<sup>*</sup>28 allele, bilirubin levels, and coronary heart disease in the Framingham Heart Study</span>. <span class="ref-journal"><em>Circulation</em></span>. 2006;<span class="ref-vol">114</span>(<span class="ref-iss">14</span>):1476-1481
 [<a href="/pubmed/17000907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Association+between+the+UG-T1A1*28+allele,+bilirubin+levels,+and+coronary+heart+disease+in+the+Framingham+Heart+Study&amp;author=JP+Lin&amp;author=CJ+O'Donnell&amp;author=JP+Schwaiger&amp;volume=114&amp;issue=14&amp;publication_year=2006&amp;pages=1476-1481&amp;pmid=17000907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">
Lin R, Wang Y, Wang Y, et al.  <span class="ref-title">Common variants of four bilirubin metabolism genes and their association with serum bilirubin and coronary artery disease in Chinese Han population</span>. <span class="ref-journal"><em>Pharmacogenet Genomics</em></span>. 2009;<span class="ref-vol">19</span>(<span class="ref-iss">4</span>):310-318
 [<a href="/pubmed/19238116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Common+variants+of+four+bilirubin+metabolism+genes+and+their+association+with+serum+bilirubin+and+coronary+artery+disease+in+Chinese+Han+population&amp;author=R+Lin&amp;author=Y+Wang&amp;author=Y+Wang&amp;volume=19&amp;issue=4&amp;publication_year=2009&amp;pages=310-318&amp;pmid=19238116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">
Lingenhel A, Kollerits B, Schwaiger JP, et al.  <span class="ref-title">Serum bilirubin levels, UGT1A1 polymorphisms and risk for coronary artery disease</span>. <span class="ref-journal"><em>Exp Gerontol</em></span>. 2008;<span class="ref-vol">43</span>(<span class="ref-iss">12</span>):1102-1107
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2648823/">PMC free article</a>]</span> [<a href="/pubmed/18790042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp+Gerontol&amp;title=Serum+bilirubin+levels,+UGT1A1+polymorphisms+and+risk+for+coronary+artery+disease&amp;author=A+Lingenhel&amp;author=B+Kollerits&amp;author=JP+Schwaiger&amp;volume=43&amp;issue=12&amp;publication_year=2008&amp;pages=1102-1107&amp;pmid=18790042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">
Borucki K, Weikert C, Fisher E, et al.  <span class="ref-title">Haplotypes in the UGT1A1 gene and their role as genetic determinants of bilirubin concentration in healthy German volunteers</span>. <span class="ref-journal"><em>Clin Biochem</em></span>. 2009;<span class="ref-vol">42</span>(<span class="ref-iss">16-17</span>):1635-1641
 [<a href="/pubmed/19732760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Biochem&amp;title=Haplotypes+in+the+UGT1A1+gene+and+their+role+as+genetic+determinants+of+bilirubin+concentration+in+healthy+German+volunteers&amp;author=K+Borucki&amp;author=C+Weikert&amp;author=E+Fisher&amp;volume=42&amp;issue=16-17&amp;publication_year=2009&amp;pages=1635-1641&amp;pmid=19732760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">
Huang CS, Huang MJ, Lin MS, Yang SS, Teng HC, Tang KS. <span class="ref-title">Genetic factors related to unconjugated hyperbilirubinemia amongst adults</span>. <span class="ref-journal"><em>Pharmacogenet Genomics</em></span>. 2005;<span class="ref-vol">15</span>(<span class="ref-iss">1</span>):43-50
 [<a href="/pubmed/15864125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Genetic+factors+related+to+unconjugated+hyperbilirubinemia+amongst+adults&amp;author=CS+Huang&amp;author=MJ+Huang&amp;author=MS+Lin&amp;author=SS+Yang&amp;author=HC+Teng&amp;volume=15&amp;issue=1&amp;publication_year=2005&amp;pages=43-50&amp;pmid=15864125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">
Ieiri I, Suzuki H, Kimura M, et al.  <span class="ref-title">Influence of common variants in the pharmacokinetic genes (OATP-C, UGT1A1, and MRP2) on serum bilirubin levels in healthy subjects</span>. <span class="ref-journal"><em>Hepatol Res</em></span>. 2004;<span class="ref-vol">30</span>(<span class="ref-iss">2</span>):91-95
 [<a href="/pubmed/15519273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hepatol+Res&amp;title=Influence+of+common+variants+in+the+pharmacokinetic+genes+(OATP-C,+UGT1A1,+and+MRP2)+on+serum+bilirubin+levels+in+healthy+subjects&amp;author=I+Ieiri&amp;author=H+Suzuki&amp;author=M+Kimura&amp;volume=30&amp;issue=2&amp;publication_year=2004&amp;pages=91-95&amp;pmid=15519273&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">
Zhang W, He YJ, Gan Z, et al.  <span class="ref-title">OATP1B1 polymorphism is a major determinant of serum bilirubin level but not associated with rifampicin-mediated bilirubin elevation</span>. <span class="ref-journal"><em>Clin Exp Pharmacol Physiol</em></span>. 2007;<span class="ref-vol">34</span>(<span class="ref-iss">12</span>):1240-1244
 [<a href="/pubmed/17973861" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Exp+Pharmacol+Physiol&amp;title=OATP1B1+polymorphism+is+a+major+determinant+of+serum+bilirubin+level+but+not+associated+with+rifampicin-mediated+bilirubin+elevation&amp;author=W+Zhang&amp;author=YJ+He&amp;author=Z+Gan&amp;volume=34&amp;issue=12&amp;publication_year=2007&amp;pages=1240-1244&amp;pmid=17973861&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3127556&amp;issue-id=198113&amp;journal-id=849&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140697028822736title">Abstract</a></li><li><a href="#S1title">PARTICIPANTS AND METHODS</a></li><li><a href="#S9title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#S11title">CONCLUSION</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140697024914208title">Footnotes</a></li><li><a href="#idm140697065174080title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Mayo Clinic Proceedings</span> are provided here courtesy of <strong>The Mayo Foundation for Medical Education and Research</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3127556/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3127556/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3127556/pdf/mayoclinproc_86_7_004.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC3127556" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127556%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127556%2F&amp;text=Mayo%20Genome%20Consortia%3A%20A%20Genotype-Phenotype%20Resource%20for%20Genome-Wide%20Association%20Studies%20With%20an%20Application%20to%20the%20Analysis%20of%20Circulating%20Bilirubin%20Levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127556%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21646302" data-aiid="3127556" data-aid="3127556" data-iid="198113" data-domainid="849" data-domain="mayoclinproc" data-accid="PMC3127556" data-md5="b9c78646902a380203ee007acc1dc780"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:08:48-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A0998DD5201A100000000086C0539&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3127556%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=mayoclinproc&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3127556/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>