<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hhe"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3725885/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3725885/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Heredity"><meta name="citation_title" content="Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data"><meta name="citation_authors" content="Kiranmoy Das, Jiahan Li, Guifang Fu, Zhong Wang, Rongling Wu"><meta name="citation_date" content="October 2011"><meta name="citation_issue" content="2"><meta name="citation_volume" content="72"><meta name="citation_firstpage" content="110"><meta name="citation_doi" content="10.1159/000330781"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3725885/?report=abstract"><meta name="citation_pmid" content="21996601"><meta name="DC.Title" content="Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Karger Publishers"><meta name="DC.Contributor" content="Kiranmoy Das"><meta name="DC.Contributor" content="Jiahan Li"><meta name="DC.Contributor" content="Guifang Fu"><meta name="DC.Contributor" content="Zhong Wang"><meta name="DC.Contributor" content="Rongling Wu"><meta name="DC.Date" content="2011 Oct"><meta name="DC.Identifier" content="10.1159/000330781"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data"><meta property="og:type" content="article"><meta property="og:description" content="Longitudinal measurements with bivariate response have been analyzed by several authors using two separate models for each response. However, for most of the biological or medical experiments, the two responses are highly correlated and hence a separate ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3725885/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhe.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3725885/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3725885/pdf/hhe-0072-0110.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C2854DD534CF100000000015300E4.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3725885%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Hered%22[journal]" class="navlink">Hum Hered</a></li><li class="accid">PMC3725885</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhe.gif" alt="Logo of hhe" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,100,75" alt="Karger" title="Karger" href="http://www.karger.com/" ref="reftype=publisher&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="poly" coords="210,0,500,0,500,59,295,59,295,75,59,75" alt="Home" title="Home" href="http://www.karger.com/HHE" ref="reftype=publisher&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="306,60,380,75" alt="Alerts" title="Alerts" href="http://content.karger.com/services/alert.asp" ref="reftype=publisher&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="395,60,458,75" alt="Resources" title="Resources" href="http://content.karger.com/resources/index.asp" ref="reftype=publisher&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Hered</a></span>. 2011 Oct; 72(2): 110–120. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Oct 11. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1159%2F000330781" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1159/000330781</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3725885</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21996601">21996601</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Das%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601" class="affpopup" co-rid="_co_idm140119716659504" co-class="co-affbox">Kiranmoy Das</a>,<sup>a,</sup><sup>b,</sup><sup>c,</sup><sup>*</sup> <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601" class="affpopup" co-rid="_co_idm140119756674912" co-class="co-affbox">Jiahan Li</a>,<sup>b,</sup><sup>c</sup> <a href="/pubmed/?term=Fu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601" class="affpopup" co-rid="_co_idm140119714537232" co-class="co-affbox">Guifang Fu</a>,<sup>b,</sup><sup>c</sup> <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601" class="affpopup" co-rid="_co_idm140119714566496" co-class="co-affbox">Zhong Wang</a>,<sup>c,</sup><sup>d</sup> and  <a href="/pubmed/?term=Wu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601" class="affpopup" co-rid="_co_idm140119714563776" co-class="co-affbox">Rongling Wu</a><sup>b,</sup><sup>c,</sup><sup>d</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140119716659504"><h3 class="no_margin">Kiranmoy Das</h3><p><sup>a</sup>Department of Statistics, Temple University, Philadelphia, Pa., USA</p><p><sup>b</sup>Department of Statistics, The Pennsylvania State University, University Park, Pa., USA</p><p><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</p><div>Find articles by <a href="/pubmed/?term=Das%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601">Kiranmoy Das</a></div></div><div id="_co_idm140119756674912"><h3 class="no_margin">Jiahan Li</h3><p><sup>b</sup>Department of Statistics, The Pennsylvania State University, University Park, Pa., USA</p><p><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601">Jiahan Li</a></div></div><div id="_co_idm140119714537232"><h3 class="no_margin">Guifang Fu</h3><p><sup>b</sup>Department of Statistics, The Pennsylvania State University, University Park, Pa., USA</p><p><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</p><div>Find articles by <a href="/pubmed/?term=Fu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601">Guifang Fu</a></div></div><div id="_co_idm140119714566496"><h3 class="no_margin">Zhong Wang</h3><p><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</p><p><sup>d</sup>Department of Public Health Sciences, The Pennsylvania State College of Medicine, Hershey, Pa., USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601">Zhong Wang</a></div></div><div id="_co_idm140119714563776"><h3 class="no_margin">Rongling Wu</h3><p><sup>b</sup>Department of Statistics, The Pennsylvania State University, University Park, Pa., USA</p><p><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</p><p><sup>d</sup>Department of Public Health Sciences, The Pennsylvania State College of Medicine, Hershey, Pa., USA</p><div>Find articles by <a href="/pubmed/?term=Wu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21996601">Rongling Wu</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140119713681280_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140119713681280_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140119713681280_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140119713681280_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Department of Statistics, Temple University, Philadelphia, Pa., USA</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>Department of Statistics, The Pennsylvania State University, University Park, Pa., USA</div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Center for Statistical Genetics, The Pennsylvania State University, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>d</sup>Department of Public Health Sciences, The Pennsylvania State College of Medicine, Hershey, Pa., USA</div><div id="cor1">*Kiranmoy Das, Department of Statistics, Temple University, Philadelphia, PA 19122 (USA), E-Mail <a href="mailto:dev@null" data-email="moc.liamg@sad.yomnarik" class="oemail">moc.liamg@sad.yomnarik</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140119713681280_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Feb 23; Revised 2011 Jul 13</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140119713681280_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2011 by S. Karger AG, Basel</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div><strong>This article has been corrected. </strong>See the correction in volume 73 on&nbsp;page&nbsp;34.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC3725885/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Erratum</div><div><a href="/pmc/articles/PMC3725885/bin/hhe-0072-0110-s01.pdf" data-ga-action="click_feat_suppl">hhe-0072-0110-s01.pdf</a><span style="color:gray"> (24K)</span></div><div class="small guid">GUID:&nbsp;DA6714CC-A18B-411F-813F-C3B31B94028A</div></div></dd></dl></div><div id="idm140119712299776" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140119712299776title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Objective</h3><p id="__p1" class="p p-first-last">Longitudinal measurements with bivariate response have been analyzed by several authors using two separate models for each response. However, for most of the biological or medical experiments, the two responses are highly correlated and hence a separate model for each response might not be a desirable way to analyze such data. A single model considering a bivariate response provides a more powerful inference as the correlation between the responses is modeled appropriately. In this article, we propose a dynamic statistical model to detect the genes controlling human blood pressure (systolic and diastolic).</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Methods</h3><p id="__p2" class="p p-first-last">By modeling the mean function with orthogonal Legendre polynomials and the covariance matrix with a stationary parametric structure, we incorporate the statistical ideas in <em>functional genome-wide association studies</em> to detect SNPs which have significant control on human blood pressure. The traditional false discovery rate is used for multiple comparisons.</p></div><div id="__sec3" class="sec sec-last"><h3 id="__sec3title">Results</h3><p id="__p3" class="p p-first-last">We analyze the data from the Framingham Heart Study to detect such SNPs by appropriately considering gender-gene interaction. We detect 8 SNPs for males and 7 for females which are most significant in controlling blood pressure. The genotype-specific mean curves and additive and dominant effects over time are shown for each significant SNP for both genders. Simulation studies are performed to examine the statistical properties of our model. The current model will be extremely useful in detecting genes controlling different traits and diseases for humans or non-human subjects.</p></div></div><div class="sec"><strong class="kwd-title">Key Words: </strong><span class="kwd-text">Bivariate response, Functional GWAS, Multiple testing, SNP</span></div></div><div id="sec1_1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_1title">Introduction</h2><p id="__p4" class="p p-first">The successful completion of the Human Genome Project (in 2005) made a revolution in detecting genes controlling various traits and diseases by genome-wide association studies (GWAS). In the past few years, GWAS have successfully identified a number of loci responsible for complex traits and diseases and by extensive genome-wide scans of genes, the genetic interactions and anomalous nature of complex genes have been better studied by geneticists and biomedical experts. This revolution is of paramount importance in developing advanced treatment methods and powerful drugs for diseases with high complexity and enticed the researchers to focus on such studies.</p><p id="__p5">Despite their potential effect on biomedical sciences, traditional GWAS suffer from several severe limitations. While in clinical trials each subject is measured at several time points not necessarily evenly spaced, GWAS consider a single time point measurement per subject and hence cannot reflect the underlying subject-specific trajectory for the longitudinal phenotype. Secondly, most GWAS have found only a small proportion of genetic variation and hence cannot explain the considerable genetic variances observed quite often in many traits. Also when correlated phenotypic traits are measured longitudinally for many subjects, traditional GWAS focus on single phenotype-genotype analyses without taking into account the possible dependence among the biomarkers. By integrating the basic idea for GWAS into powerful statistical approaches, Das et al. [<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191233">1</a>] proposed a new so-called <em>functional GWAS</em> (or <em>f</em> GWAS) approach which can well address the fundamental limitations of traditional GWAS.</p><p id="__p6">For the analysis of univariate repeated measures, the linear mixed model-based approach [<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191230">2</a>] is quite popular and readily available in standard statistical packages. When more than one biomarker is measured repeatedly, longitudinal multivariate models should be used. A bivariate linear mixed model approach using the standard statistical package (SAS) was proposed by Thiebaut et al. [<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191235">3</a>]; Sithole and Jones [<a href="#B4" rid="B4" class=" bibr popnode">4</a>] used such a model for detecting prescribing change in two drugs simultaneously with correlated errors. Since then it has been well recognized that for multiple correlated biomarkers, a multivariate linear model is more powerful than using a separate model for each biomarker.</p><p id="__p7" class="p p-last">In this article, we have used a similar bivariate longitudinal model to analyze data from the Framingham Heart Study (FHS) with the aim to unlock the door of the complex genetic architecture of human blood pressure which may lead to serious cardiovascular diseases. Two responses, systolic and diastolic blood pressure, are measured for 977 healthy subjects at different ages. Different subjects are measured at different time points and the numbers of measurements are quite different from subject to subject. Since the body mass index (BMI) is known to have a significant effect on blood pressure, we consider it as a covariate in our model and in order to incorporate the gender-gene interaction, we perform separate analyses for the male and female population. We detect single nucleotide polymorphisms (SNPs) which are most significant for controlling human blood pressure for the male and female population. Simulation studies and power analyses have been performed to verify the effectiveness and importance of our approach.</p></div><div id="sec1_2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_2title">Subjects and Methods</h2><p id="__p8" class="p p-first">We use our model and method to analyze the bivariate longitudinal trait collected in the FHS. The objective of this study was to unlock the door of genetic architecture of cardiovascular diseases. The study began in the year 1948 with 5,209 healthy men and women aged 30–60 from Framingham, Mass., USA. Currently, the FHS project is in its third generation of participants. The study recorded the medical history of all its participants and laboratory tests were conducted to have actual data from the subjects. Different subjects were measured at different time points, but collectively the data provide records of phenotypes at many time points. Our data contain 977 subjects (all Caucasians), 500 males and 477 females, who had their systolic and diastolic blood pressure, BMI and many other variables measured at multiple time points. The number of serial measurements for a subject can be as low as 3, but the times and intervals of measurements are highly variable among the different subjects. Thus, despite high sparsity, our data set contains a large range of time points when all subjects are projected on a single time scale.</p><p id="__p9" class="p">We have approximately 550,000 SNP data from the entire human genome. Following the general tradition of GWAS, we excluded SNPs with minor allele frequencies (MAF) of less than 0.10. The numbers and percentages of non-rare allele SNPs vary among different chromosomes and range from 4,417 to 28,771 and from 0.64 to 0.72, respectively. Our model, based on a direct relationship between phenotype and genotype, is used at the SNP level to detect significant SNPs associated with blood pressure by considering the gender-gene interaction.</p><div id="sec2_1" class="sec"><h3 id="sec2_1title">Statistical Model</h3><p id="__p10" class="p p-first">In the FHS, longitudinal traits are measured at irregular and possibly subject-specific time points. Let <strong>y</strong><sub><em>ik</em></sub> = [<em>y<sub>ik</sub></em>(<em>t<sub>i</sub></em><sub>1</sub>)<em>, y<sub>ik</sub></em> (<em>t<sub>i</sub></em><sub>2</sub>), <em>…, yik</em> (<em>t<sub>i</sub>T</em><sub>i</sub>)] denote the vector of trait of type <em>k</em> (for example, <em>k</em> = 1 for systolic blood pressure and <em>k</em> = 2 for diastolic blood pressure) for the <em>i</em>-th subject measured at time points <em>t<sub>i</sub></em> = [<em>t<sub>i</sub></em><sub>1</sub>, …, <em>t<sub>i</sub>T</em><sub>i</sub>]. Using this denotation, we can capture subject-specific differences in the number and interval of time points. In what follows, we consider <em>n</em> subjects measured at different time points. Consider a SNP with two alleles, A and a, generating three genotypes: AA (coded as 1) with <em>n</em><sub>1</sub> observations, Aa (coded as 2) with <em>n</em><sub>2</sub> observations and aa (coded as 3) with <em>n</em><sub>3</sub> observations. The <em>k</em>-th type phenotypic value for subject <em>i</em> at time <em>t<sub>iτ</sub></em> (<em>τ</em> = 1, …, <em>T<sub>i</sub></em>) is expressed as</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq1" overflow="scroll"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></munderover><mrow><msub><mi>ξ</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mi>μ</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>+</mo><mi>β</mi><msub><mi>x</mi><mi>i</mi></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></mrow></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="uneq1" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="msub" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 1.177em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.5em, 4.337em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.006em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.49em;"><span class="mrow" id="MathJax-Span-6"><span class="mi" id="MathJax-Span-7" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-8" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-9" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-10" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-11"><span class="msub" id="MathJax-Span-12"><span style="display: inline-block; position: relative; width: 1.046em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.33em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-13" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.361em;"><span class="mrow" id="MathJax-Span-14"><span class="mi" id="MathJax-Span-15" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-16" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">τ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-17" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-18" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="munderover" id="MathJax-Span-19" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.444em; height: 0px;"><span style="position: absolute; clip: rect(2.872em, 1001.39em, 4.582em, -1000em); top: -3.977em; left: 0em;"><span class="mo" id="MathJax-Span-20" style="font-family: MathJax_Size2; vertical-align: 0em;">∑</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.351em, 1001.14em, 4.376em, -1000em); top: -2.89em; left: 0.124em;"><span class="mrow" id="MathJax-Span-21"><span class="mi" id="MathJax-Span-22" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span class="mo" id="MathJax-Span-23" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-24" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.252em, 1000.32em, 4.148em, -1000em); top: -5.127em; left: 0.545em;"><span class="mn" id="MathJax-Span-25" style="font-size: 70.7%; font-family: MathJax_Main;">3</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-26" style="padding-left: 0.167em;"><span class="msub" id="MathJax-Span-27"><span style="display: inline-block; position: relative; width: 1.048em; height: 0px;"><span style="position: absolute; clip: rect(3.118em, 1000.44em, 4.337em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-28" style="font-family: MathJax_Math; font-style: italic;">ξ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.005em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.438em;"><span class="mrow" id="MathJax-Span-29"><span class="mi" id="MathJax-Span-30" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-31" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="msub" id="MathJax-Span-32"><span style="display: inline-block; position: relative; width: 1.338em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.58em, 4.348em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-33" style="font-family: MathJax_Math; font-style: italic;">μ</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.603em;"><span class="mrow" id="MathJax-Span-34"><span class="mi" id="MathJax-Span-35" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span class="mi" id="MathJax-Span-36" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-37" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-38" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-39"><span class="msub" id="MathJax-Span-40"><span style="display: inline-block; position: relative; width: 1.046em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.33em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-41" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.361em;"><span class="mrow" id="MathJax-Span-42"><span class="mi" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-44" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">τ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-46" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-47" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span class="msub" id="MathJax-Span-48"><span style="display: inline-block; position: relative; width: 0.891em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.52em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-49" style="font-family: MathJax_Math; font-style: italic;">x</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.572em;"><span class="mi" id="MathJax-Span-50" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-51" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-52" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-53"><span class="msub" id="MathJax-Span-54"><span style="display: inline-block; position: relative; width: 1.046em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.33em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-55" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.361em;"><span class="mrow" id="MathJax-Span-56"><span class="mi" id="MathJax-Span-57" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-58" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">τ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-59" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-60" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-61" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.153em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.43em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-62" style="font-family: MathJax_Math; font-style: italic;">e</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.466em;"><span class="mrow" id="MathJax-Span-63"><span class="mi" id="MathJax-Span-64" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-65" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mrow" id="MathJax-Span-66" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-67" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-68"><span class="msub" id="MathJax-Span-69"><span style="display: inline-block; position: relative; width: 1.046em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.33em, 4.143em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-70" style="font-family: MathJax_Math; font-style: italic;">t</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.361em;"><span class="mrow" id="MathJax-Span-71"><span class="mi" id="MathJax-Span-72" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-73" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">τ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-74" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-75" style="font-family: MathJax_Main; padding-left: 0.167em;">,</span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq1" overflow="scroll"><mrow><msub><mi>y</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></munderover><mrow><msub><mi>ξ</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mi>μ</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>+</mo><mi>β</mi><msub><mi>x</mi><mi>i</mi></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>+</mo><msub><mi>e</mi><mrow><mi>i</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mrow><msub><mi>t</mi><mrow><mi>i</mi><mi>τ</mi></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo></mrow></mrow></math></script></div><div class="l">(1)</div></div><p id="__p11">where </p><div class="disp-formula" id="uneq2"><div class="f">ξ<sub><em>i</em><em>j</em></sub></div></div><p> is an indicator variable taking value 1 if the <em>i</em>-th subject is of genotype <em>j</em> and 0 if otherwise, <em>µ<sub>jk</sub></em>(<em>t<sub>iτ</sub></em>) is the mean value for genotype <em>j</em> of the <em>k</em>-th response at time <em>t<sub>iτ</sub>, x<sub>i</sub></em> is the time-variant covariate, and <em>β</em> is the regression coefficient of the covariate. Since in the present situation the covariate is the BMI which is assumed to have the same effect on both systolic and diastolic blood pressure, we do not use subscript <em>k</em> for <em>β.</em> However, this might not be the case in general and depending on the context, one has to decide whether to include the subscript or not. The residual error for subject <em>i</em>, <em>e<sub>ik</sub></em> (<em>t<sub>iτ</sub></em>), is assumed to be distributed as multivariate normal with mean = 0 and covariance matrix = σ<sub><em>i</em></sub>.</p><p id="__p12">The first important task in <em>f</em> GWAS is to model the genotype-specific mean curves and within-subject covariance structure. For modeling the mean curves, we used a nonparametric approach based on orthogonal Legendre polynomials (LP). These polynomials have already been proven a powerful tool to model longitudinal or functional data by several authors for nonparametric regression [<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191232">5</a>, <a href="#B6" rid="B6" class=" bibr popnode">6</a>, <a href="#B7" rid="B7" class=" bibr popnode">7</a>] and the robustness of nonparametric regression using LP has been investigated effectively. The LP are solutions to the Legendre differential equation</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq3" overflow="scroll"><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>-</mo><msup><mi>x</mi><mn>2</mn></msup></mrow><mo>)</mo></mrow><mfrac><mrow><msup><mi>d</mi><mn>2</mn></msup><mi>z</mi></mrow><mrow><mi>d</mi><msup><mi>x</mi><mn>2</mn></msup></mrow></mfrac><mo>-</mo><mn>2</mn><mi>x</mi><mfrac><mrow><mi>d</mi><mi>z</mi></mrow><mrow><mi>d</mi><mi>x</mi></mrow></mfrac><mo>+</mo><mi>r</mi><mrow><mo>(</mo><mrow><mi>r</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mi>z</mi><mo>=</mo><mn>0.</mn></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq3" overflow="scroll"><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>-</mo><msup><mi>x</mi><mn>2</mn></msup></mrow><mo>)</mo></mrow><mfrac><mrow><msup><mi>d</mi><mn>2</mn></msup><mi>z</mi></mrow><mrow><mi>d</mi><msup><mi>x</mi><mn>2</mn></msup></mrow></mfrac><mo>-</mo><mn>2</mn><mi>x</mi><mfrac><mrow><mi>d</mi><mi>z</mi></mrow><mrow><mi>d</mi><mi>x</mi></mrow></mfrac><mo>+</mo><mi>r</mi><mrow><mo>(</mo><mrow><mi>r</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mi>z</mi><mo>=</mo><mn>0.</mn></mrow></math></script></div></div><p id="__p13">The general form of a LP of order r is given by the following sum</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq4" overflow="scroll"><mrow><msub><mi>P</mi><mi>r</mi></msub><mrow><mo>(</mo><mi>x</mi><mo>)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>0</mn></mrow><mi>L</mi></munderover><mrow><msup><mrow><mrow><mo>(</mo><mrow><mo>-</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow><mn>1</mn></msup><mfrac><mrow><mrow><mo>(</mo><mrow><mn>2</mn><mi>r</mi><mo>-</mo><mn>2</mn><mi>l</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow><mrow><msup><mn>2</mn><mi>r</mi></msup><mi>l</mi><mo>!</mo><mrow><mo>(</mo><mrow><mi>r</mi><mo>-</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>!</mo><mrow><mo>(</mo><mrow><mi>r</mi><mo>-</mo><mn>2</mn><mi>l</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow></mfrac><msup><mi>x</mi><mrow><mi>r</mi><mo>-</mo><mn>21</mn></mrow></msup><mo>,</mo></mrow></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq4" overflow="scroll"><mrow><msub><mi>P</mi><mi>r</mi></msub><mrow><mo>(</mo><mi>x</mi><mo>)</mo></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>l</mi><mo>=</mo><mn>0</mn></mrow><mi>L</mi></munderover><mrow><msup><mrow><mrow><mo>(</mo><mrow><mo>-</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow><mn>1</mn></msup><mfrac><mrow><mrow><mo>(</mo><mrow><mn>2</mn><mi>r</mi><mo>-</mo><mn>2</mn><mi>l</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow><mrow><msup><mn>2</mn><mi>r</mi></msup><mi>l</mi><mo>!</mo><mrow><mo>(</mo><mrow><mi>r</mi><mo>-</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>!</mo><mrow><mo>(</mo><mrow><mi>r</mi><mo>-</mo><mn>2</mn><mi>l</mi></mrow><mo>)</mo></mrow><mo>!</mo></mrow></mfrac><msup><mi>x</mi><mrow><mi>r</mi><mo>-</mo><mn>21</mn></mrow></msup><mo>,</mo></mrow></mrow></math></script></div></div><p id="__p14">where</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq5" overflow="scroll"><mrow><mi>L</mi><mo>=</mo><mfrac><mi>r</mi><mn>2</mn></mfrac><mi>o</mi><mi>r</mi><mfrac><mrow><mi>r</mi><mo>-</mo><mn>1</mn></mrow><mn>2</mn></mfrac></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq5" overflow="scroll"><mrow><mi>L</mi><mo>=</mo><mfrac><mi>r</mi><mn>2</mn></mfrac><mi>o</mi><mi>r</mi><mfrac><mrow><mi>r</mi><mo>-</mo><mn>1</mn></mrow><mn>2</mn></mfrac></mrow></math></script></div></div><p id="__p15">whichever is an integer. These polynomials are defined over [−1, <a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191229">1</a>] and are orthogonal to each other in this interval in the sense that </p><div class="disp-formula" id="uneq6"><div class="f">∫ -&nbsp;1<em> </em><em>P</em><sub><em>r</em></sub>(<em>x</em>)<em>P</em><sub><em>s</em></sub>(<em>d</em><em>x</em>) =&nbsp;0<em> </em><em>w</em><em>h</em><em>e</em><em>n</em><em> </em><em>r</em><em> </em>&nbsp;≠&nbsp;<em>s</em>.</div></div><p> Let <em>P<sub>r</sub></em>(<em>t</em>) denote the <em>r</em>-th order LP at time <em>t.</em> By appropriate transformation of the measurement times, we make proper adjustment to fit the orthogonal function range [−1, <a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191241">1</a>]. This can be achieved very easily by making the following transformation of the original time points <em>t</em><sub>1</sub>, <em>t</em><sub>2</sub>, <em>t</em><sub>3</sub>, to <em>t</em>‘<sub>1</sub>, <em>t</em>‘<sub>2</sub>, <em>t</em>‘<sub>3</sub>, …, where</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq7" overflow="scroll"><mrow><msup><mi>t</mi><mo>'</mo></msup><mo>=</mo><mo>-</mo><mn>1</mn><mo>+</mo><mn>2</mn><mrow><mo>(</mo><mrow><mfrac><mrow><mi>t</mi><mo>-</mo><msub><mi>t</mi><mrow><mo>min</mo></mrow></msub></mrow><mrow><msub><mi>t</mi><mrow><mo>max</mo></mrow></msub><mo>-</mo><msub><mi>t</mi><mrow><mo>min</mo></mrow></msub></mrow></mfrac></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq7" overflow="scroll"><mrow><msup><mi>t</mi><mo>'</mo></msup><mo>=</mo><mo>-</mo><mn>1</mn><mo>+</mo><mn>2</mn><mrow><mo>(</mo><mrow><mfrac><mrow><mi>t</mi><mo>-</mo><msub><mi>t</mi><mrow><mo>min</mo></mrow></msub></mrow><mrow><msub><mi>t</mi><mrow><mo>max</mo></mrow></msub><mo>-</mo><msub><mi>t</mi><mrow><mo>min</mo></mrow></msub></mrow></mfrac></mrow><mo>)</mo></mrow><mo>,</mo></mrow></math></script></div></div><p id="__p16">where <em>t<sub>min</sub></em> and <em>t<sub>max</sub></em> are the shortest and the longest time points, respectively. A family of orthogonal LP is denoted by <em>P</em>(<em>t</em>) = [<em>P</em><sub>0</sub>(<em>t</em>) <em>P</em><sub>1</sub>(<em>t</em>), …, <em>P<sub>r</sub></em>(<em>t</em>)]<sup><em>τ</em></sup> and the genotype-specific mean value is expressed as a linear combination of the polynomials, such as</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq8" overflow="scroll"><mrow><msub><mi>μ</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msubsup><mi>u</mi><mrow><mi>j</mi><mi>k</mi></mrow><mi>T</mi></msubsup><mi>P</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>,</mo></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq8" overflow="scroll"><mrow><msub><mi>μ</mi><mrow><mi>j</mi><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msubsup><mi>u</mi><mrow><mi>j</mi><mi>k</mi></mrow><mi>T</mi></msubsup><mi>P</mi><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>,</mo></mrow></math></script></div><div class="l">(2)</div></div><p id="__p17">where <em>u<sub>jk</sub></em> = (<em>u<sub>jk</sub></em><sub>0</sub>, <em>u<sub>jk</sub></em><sub>1</sub>, …, <em>u<sub>jkr</sub></em>)<sup><em>τ</em></sup> is called base vector. The order of the LP is chosen by information criterion like AIC/BIC, etc.</p><p id="__p18">In the Statistics literature, there are a number of ways to model the time-dependent mean curves for longitudinal data. This includes Fourier series, B-spline, Wavelets, etc. If the trait under consideration is well studied and well understood, then a known parametric structure might be more effective and biologically meaningful to model the mean function. For example, in modeling the growth for plants and animals, a logistic equation might be used [<a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191234">8</a>]. Triple logistic equations might be effective for human body growth [<a href="#B9" rid="B9" class=" bibr popnode">9</a>]. Since these mathematical equations are derived from fundamental biological principles, one can incorporate these models in GWAS or <em>f</em> GWAS to capitalize their power in modeling the trait under consideration in an accurate and precise way. Semiparametric models [<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191231">5</a>] can also be implemented in some situations for a better understanding of the process under consideration.</p><p id="__p19">In longitudinal data analysis, it is also fundamentally important to model the within-subject covariance structure in a robust and powerful way. If the trait variation within subjects is constant, one might use a stationary autoregressive model for modeling the covariance. The advantage of this approach is that one needs to estimate only two parameters, the variance and the correlation for this structure. While the stationary assumption is not worth, nonstationary approaches like the structured antedependence model can be used. In order to handle more complex covariance structures, the autoregressive moving average model (ARMA(<em>p</em>, <em>q</em>)) might be implemented too. More recently, Fan et al. [<a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191236">10</a>] and Fan and Wu [<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191240">11</a>] have developed a semiparametric kernel-based covariance function which is robust and effective for modeling subject-specific covariance structure for longitudinal data analysis.</p><p id="__p20">Here, we model the covariance matrix for the bivariate response longitudinal data as a Kronecker product of two covariance structures following Sithole and Jones [<a href="#B4" rid="B4" class=" bibr popnode">4</a>]. For a fixed <em>k</em> (= 1 and 2), we assume that the longitudinal measurements from the same subject have an AR(1) structure, while the correlation between the systolic and diastolic blood pressure remains the same over time for a fixed subject. This kind of covariance structure can be expressed as <em>UN</em> ⊗ <em>AR</em>(1), where UN is a 2 × 2 symmetric matrix with σ<sup>2</sup><sub><em>s</em></sub> and σ<sup>2</sup><sub><em>d</em></sub> as the diagonal elements and <em>Σ<sub>sd</sub></em> as the off-diagonal element. For instance, if we have three repeated measures from a subject, then we assume a covariance matrix with the structure <em>UN</em> ⊗ <em>AR</em>(1), where UN and AR(1) are</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq9" overflow="scroll"><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><msubsup><mi>σ</mi><mi>s</mi><mn>2</mn></msubsup></mrow></mtd></mtr><mtr><mtd><mrow><msubsup><mi>σ</mi><mrow><mi>s</mi><mi>d</mi></mrow><mn>2</mn></msubsup></mrow></mtd></mtr></mtable><mtable><mtr><mtd><mrow><msub><mi>σ</mi><mrow><mi>s</mi><mi>d</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msubsup><mi>σ</mi><mi>d</mi><mn>2</mn></msubsup></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq9" overflow="scroll"><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><msubsup><mi>σ</mi><mi>s</mi><mn>2</mn></msubsup></mrow></mtd></mtr><mtr><mtd><mrow><msubsup><mi>σ</mi><mrow><mi>s</mi><mi>d</mi></mrow><mn>2</mn></msubsup></mrow></mtd></mtr></mtable><mtable><mtr><mtd><mrow><msub><mi>σ</mi><mrow><mi>s</mi><mi>d</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msubsup><mi>σ</mi><mi>d</mi><mn>2</mn></msubsup></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow></mrow></math></script></div></div><p id="__p21">and</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq10" overflow="scroll"><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><mn>1</mn><mi> </mi><mi>ρ</mi><mi> </mi><msup><mi>ρ</mi><mn>2</mn></msup></mrow></mtd></mtr><mtr><mtd><mrow><mi>ρ</mi><mi> </mi><mn>1</mn><mi> </mi><mi>ρ</mi></mrow></mtd></mtr><mtr><mtd><mrow><msup><mi>ρ</mi><mn>2</mn></msup><mi> </mi><mi>ρ</mi><mi> </mi><mn>1</mn></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow><mo>,</mo></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq10" overflow="scroll"><mrow><mrow><mo>[</mo><mrow><mtable><mtr><mtd><mrow><mn>1</mn><mi> </mi><mi>ρ</mi><mi> </mi><msup><mi>ρ</mi><mn>2</mn></msup></mrow></mtd></mtr><mtr><mtd><mrow><mi>ρ</mi><mi> </mi><mn>1</mn><mi> </mi><mi>ρ</mi></mrow></mtd></mtr><mtr><mtd><mrow><msup><mi>ρ</mi><mn>2</mn></msup><mi> </mi><mi>ρ</mi><mi> </mi><mn>1</mn></mrow></mtd></mtr></mtable></mrow><mo>]</mo></mrow><mo>,</mo></mrow></math></script></div></div><p id="__p22">respectively. This specification of the covariance structure assumes that an intra-blood pressure correlation is the same for systolic and diastolic. Moreover, it assumes that an inter-blood pressure correlation is proportional to an intra-blood pressure correlation. However, this assumption is not too restrictive and such a covariance function can be modeled easily using the standard statistical packages. Such a covariance structure has been proposed by Thiebaut et al. [<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191242">3</a>] and Sithole and Jones [<a href="#B4" rid="B4" class=" bibr popnode">4</a>] for regular longitudinal data when all subjects are measured at the same time points. We performed an exploratory data analysis to investigate the validity of such a structure for irregular longitudinal data using SAS, and it turned out that SAS is equally powerful in this case as well. Hence, when different subjects are measured at different time points (which is true for our FHS), one can model subject-specific covariance matrices in the way mentioned above, assuming the underlying covariance process is the same for all subjects.</p><p id="__p23">The additive (<em>a</em>) and dominant (<em>d</em>) effects over time of the SNP under consideration can be expressed as follows</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq11" overflow="scroll"><mrow><mtable><mtr><mtd><mrow><msub><mi>a</mi><mi>k</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo>[</mo><mrow><msub><mi>μ</mi><mrow><mn>1</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>-</mo><msub><mi>μ</mi><mrow><mn>3</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mo>]</mo></mrow><mo>,</mo></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>d</mi><mi>k</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msub><mi>μ</mi><mrow><mn>2</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>-</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo>[</mo><mrow><msub><mi>μ</mi><mrow><mn>1</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>+</mo><msub><mi>ν</mi><mrow><mn>3</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mo>]</mo></mrow><mo>.</mo></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq11" overflow="scroll"><mrow><mtable><mtr><mtd><mrow><msub><mi>a</mi><mi>k</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo>[</mo><mrow><msub><mi>μ</mi><mrow><mn>1</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>-</mo><msub><mi>μ</mi><mrow><mn>3</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mo>]</mo></mrow><mo>,</mo></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>d</mi><mi>k</mi></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>=</mo><msub><mi>μ</mi><mrow><mn>2</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>-</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><mrow><mo>[</mo><mrow><msub><mi>μ</mi><mrow><mn>1</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow><mo>+</mo><msub><mi>ν</mi><mrow><mn>3</mn><mi>k</mi></mrow></msub><mrow><mo>(</mo><mi>t</mi><mo>)</mo></mrow></mrow><mo>]</mo></mrow><mo>.</mo></mrow></mtd></mtr></mtable></mrow></math></script></div></div><p id="__p24">Assuming the subjects under study are independent from each other, the joint likelihood function can be written as</p><div class="disp-formula"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq12" overflow="scroll"><mrow><mi>L</mi><mrow><mo>(</mo><mi>y</mi><mo>)</mo></mrow><mo>=</mo><munderover><mo>∐</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>1</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>1</mn></msub></mrow><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>2</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>2</mn></msub><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><munderover><mo>∐</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>3</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>3</mn></msub></mrow><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><mo>,</mo></mrow></mrow></math></span><div class="MathJax_Display MathJax_Processing"><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0"></span></div><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="uneq12" overflow="scroll"><mrow><mi>L</mi><mrow><mo>(</mo><mi>y</mi><mo>)</mo></mrow><mo>=</mo><munderover><mo>∐</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>1</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>1</mn></msub></mrow><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>2</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>2</mn></msub><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><munderover><mo>∐</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>n</mi><mn>3</mn></msub></mrow></munderover><mrow><msub><mi>f</mi><mn>3</mn></msub></mrow><mrow><mo>(</mo><mrow><msub><mi>y</mi><mi>i</mi></msub></mrow><mo>)</mo></mrow><mo>,</mo></mrow></mrow></math></script></div><div class="l">(3)</div></div><p id="__p25">where <strong>y</strong><sub><em>i</em></sub> = [<strong>y</strong><sub><em>i</em>1</sub>, <strong>y</strong><sub><em>i</em>2</sub>]<sup><em>τ</em></sup> is the response vector for the <em>i</em>-th subject, <em>f<sub>j</sub></em>(<strong>y</strong><sub><em>i</em></sub>) is the multivariate normal density for the <em>i</em>-th subject carrying SNP genotype <em>j</em> (<em>j</em> = 1, 2, 3), with the following mean vectors</p><p id="__p26">[<em>μ</em><sub>11</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>), <em>µ</em><sub>12</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>)]<sup><em>τ</em></sup>, for genotype AA,</p><p id="__p27">[<em>μ</em><sub>11</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>), <em>µ</em><sub>22</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>)]<sup><em>τ</em></sup>, for genotype Aa,</p><p id="__p28">[<em>μ</em><sub>31</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>), <em>µ</em><sub>32</sub>(<em>t<sub>iτ</sub></em>) + <em>ßx<sub>i</sub></em>(<em>t<sub>iτ</sub></em>)]<sup><em>τ</em></sup>, for genotype aa,</p><p id="__p29" class="p p-last">and subject-specific covariance matrix σ<sub><em>i</em></sub>, modeled as described earlier. In the traditional likelihood-based approach, we maximize the log likelihood to get the maximum likelihood estimate for the model parameters.</p></div><div id="sec2_2" class="sec sec-last"><h3 id="sec2_2title">Hypothesis Tests</h3><p id="__p30" class="p p-first">Once the model parameters are estimated, it is important to perform hypothesis tests in order to detect the significant genetic effect of a particular SNP. The hypotheses can be formulated in the following way: <em>H</em><sub>0</sub>: <em>µ</em><sub>1<em>k</em></sub> = <em>µ</em>2<em>k</em> = <em>µ</em><sub>3<em>k</em></sub>, for <em>k</em> = 1 and 2, versus <em>H</em><sub>1</sub>: at least one equality in <em>H</em><sub>0</sub> does not hold.</p><p id="__p31" class="p p-last">Note that under <em>H</em><sub>0</sub>, the mean functions for the three genotypes are the same, indicating no significant genetic effect for the SNP under consideration. By computing the likelihood function under <em>H</em><sub>0</sub> and <em>H</em><sub>1</sub>, one can compute the likelihood ratio (LR). The LR test statistic −2log(<em>LR</em>) asymptotically follows a <em>χ</em><sup>2</sup> distribution with the degree of freedom equal to the difference in the numbers of unknown parameters under <em>H</em><sub>0</sub> and <em>H</em><sub>1</sub>. Because of the large number of SNPs under consideration, we need to adjust for multiple comparison and here we use a standard false discovery rate (FDR) approach as proposed by Benjamini and Hochberg [<a href="#B12" rid="B12" class=" bibr popnode">12</a>]. However, one can also apply more advanced FDR approaches but we avoid those for computational simplicity.</p></div></div><div id="sec1_3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_3title">Results</h2><div id="sec2_3" class="sec sec-first"><h3 id="sec2_3title">Gene Detection</h3><p id="__p32" class="p p-first">The raw data for systolic and diastolic blood pressure for males and females are shown in figure <a href="/pmc/articles/PMC3725885/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">figure1.</span></span><span>1</span></a>. In the exploratory data analysis, we first look for the optimal order (<em>r</em>) of the LP that fits our data best. To get the optimum order, we compute BIC values for <em>r</em> = 1, 2, 3 and 4 using the standard approach. Table <a href="/pmc/articles/PMC3725885/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1</span></span><span>1</span></a> provides the corresponding BIC values for different orders and we see that a second order polynomial gives the best fit among the four different values of <em>r</em> which is therefore then taken as the optimum order.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3725885/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119722608208" class="figure"><a href="/pmc/articles/PMC3725885/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g01.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g01.jpg"></a></div><div id="largeobj_idm140119722608208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p45">Raw data for systolic and diastolic blood pressure for males (<strong>a</strong>) and females (<strong>b</strong>), respectively.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p54">BIC values for selecting the optimum order (<em>r</em>) of the LP</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119719472640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr valign="top"><th rowspan="1" colspan="1" align="left">Order</th><th rowspan="1" colspan="1" align="left">BIC (male)</th><th rowspan="1" colspan="1" align="left">BIC (female)</th></tr></thead><tbody><tr valign="top"><td rowspan="1" colspan="1" align="left">1</td><td rowspan="1" colspan="1" align="left">4.62</td><td rowspan="1" colspan="1" align="left">3.85</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left">2</td><td rowspan="1" colspan="1" align="left"><strong>1.62</strong></td><td rowspan="1" colspan="1" align="left"><strong>1.95</strong></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left">3</td><td rowspan="1" colspan="1" align="left">3.84</td><td rowspan="1" colspan="1" align="left">2.77</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left">4</td><td rowspan="1" colspan="1" align="left">1.99</td><td rowspan="1" colspan="1" align="left">2.01</td></tr></tbody></table></div><div id="largeobj_idm140119719472640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="__p33">Next, we consider one SNP at a time and use the model as calculated in equation <a href="#uneq1" rid="uneq1" class=" other">1</a> to test for a significant effect of the marker on the blood pressure trajectories. We have different numbers of SNPs for different chromosomes and we test for each SNP in each chromosome. We have the fixed effect of the genotype and also the effect of the BMI as the covariate. After estimating the model parameters, we perform the hypothesis testing as described in the previous section. The multiple testing problem is handled by controlling the FDR using Benjamini and Hochberg's algorithm [<a href="#B12" rid="B12" class=" bibr popnode">12</a>].</p><p id="__p34">As mentioned earlier, we have nearly 550,000 SNPs, we need to adopt a multiple hypothesis-testing approach to have control over the overall type 1 error probability. Although Bonferroni's method is a widely used approach, we did not apply this approach here simply because it is too conservative in nature. We applied a FDR control for the multiple comparisons. With a significance level <em>α</em> = 10<sup>−6</sup> (with an estimated FDR = 0.002), we select 8 significant SNPs for the male and 7 significant SNPs for the female population. The Manhattan plots for males and females are displayed in figures <a href="/pmc/articles/PMC3725885/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">figures2</span></span><span>2</span></a> and <a href="/pmc/articles/PMC3725885/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3,</span></span><span>3</span></a>, respectively. For males, the detected SNPs are rs66475406 on chromosome 2, rs66149495 on chromosome 3, rs66299297 and rs66501706 on chromosome 5, rs66484226 on chromosome 8, rs66379521 on chromosome 12, rs66154967 on chromosome 17 and rs66092412 on chromosome 19. For females, we got rs66076226 on chromosome 1, rs66053327, rs66123984, rs66114880 and rs66347789 on chromosome 2, rs66447584 on chromosome 6 and rs66225752 on chromosome 18. Table <a href="/pmc/articles/PMC3725885/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a> provides the names, positions, MAF and p values for all these sex-specific significant SNPs. Figures <a href="/pmc/articles/PMC3725885/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures4</span></span><span>4</span></a> and <a href="/pmc/articles/PMC3725885/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and5</span></span><span>5</span></a> show the sex-specific and genotype-specific mean trajectories for systolic and diastolic blood pressure. Figures <a href="/pmc/articles/PMC3725885/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3.5em;">Figures6</span></span><span>6</span></a> and <a href="/pmc/articles/PMC3725885/figure/F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="F7" rid-ob="ob-F7" co-legend-rid="lgnd_F7"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and7</span></span><span>7</span></a> show the observed (black) versus expected (grey) p values in –log scale with base 10 for the male and female population, respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3725885/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119755219600" class="figure"><a href="/pmc/articles/PMC3725885/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g02.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g02.jpg"></a></div><div id="largeobj_idm140119755219600" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p46">Manhattan plot for males: selection of the most significant SNPs controlling blood pressure.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3725885/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119717007472" class="figure"><a href="/pmc/articles/PMC3725885/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g03.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g03.jpg"></a></div><div id="largeobj_idm140119717007472" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p47">Manhattan plot for females: selection of the most significant SNPs controlling blood pressure.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3725885/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119717004896" class="figure"><a href="/pmc/articles/PMC3725885/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g04.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g04.jpg"></a></div><div id="largeobj_idm140119717004896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p48">Genotype-specific mean curves for the most significant SNPs for males.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC3725885/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119720613280" class="figure"><a href="/pmc/articles/PMC3725885/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g05.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g05.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g05.jpg"></a></div><div id="largeobj_idm140119720613280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><p id="__p49">Genotype-specific mean curves for the most significant SNPs for females.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC3725885/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119712734160" class="figure"><a href="/pmc/articles/PMC3725885/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g06.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g06.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g06.jpg"></a></div><div id="largeobj_idm140119712734160" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Fig. 6</a></div><!--caption a7--><div class="caption"><p id="__p50">Observed versus expected P-plot for males.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F7" co-legend-rid="lgnd_F7"><a href="/pmc/articles/PMC3725885/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119712731584" class="figure"><a href="/pmc/articles/PMC3725885/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g07.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g07.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g07.jpg"></a></div><div id="largeobj_idm140119712731584" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F7"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F7/" target="figure" rid-figpopup="F7" rid-ob="ob-F7">Fig. 7</a></div><!--caption a7--><div class="caption"><p id="__p51">Observed versus expected P-plot for females.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p55">Associations between SNPs and age-specific changes of blood pressure for males and females</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119723201888" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr valign="top"><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Alleles</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">p value</th></tr></thead><tbody><tr valign="top"><td rowspan="1" colspan="1" align="center">Male-specific</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">rs66475406</td><td rowspan="1" colspan="1" align="right">81,518,943</td><td rowspan="1" colspan="1" align="right">C/T</td><td rowspan="1" colspan="1" align="right">0.1652(C)</td><td rowspan="1" colspan="1" align="right">0.5353 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="right">rs66149495</td><td rowspan="1" colspan="1" align="right">16,140,422</td><td rowspan="1" colspan="1" align="right">G/T</td><td rowspan="1" colspan="1" align="right">0.4038(G)</td><td rowspan="1" colspan="1" align="right">0.1511 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">rs66299297</td><td rowspan="1" colspan="1" align="right">104,206,688</td><td rowspan="1" colspan="1" align="right">A/G</td><td rowspan="1" colspan="1" align="right">0.1137(A)</td><td rowspan="1" colspan="1" align="right">0.3244 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1" align="right">rs66501706</td><td rowspan="1" colspan="1" align="right">147,356,971</td><td rowspan="1" colspan="1" align="right">A/G</td><td rowspan="1" colspan="1" align="right">0.3481(A)</td><td rowspan="1" colspan="1" align="right">0.7316 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="right">rs66484226</td><td rowspan="1" colspan="1" align="right">91,327,474</td><td rowspan="1" colspan="1" align="right">C/A</td><td rowspan="1" colspan="1" align="right">0.1125(C)</td><td rowspan="1" colspan="1" align="right">0.2436 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1" align="right">rs66379521</td><td rowspan="1" colspan="1" align="right">130,758,789</td><td rowspan="1" colspan="1" align="right">T/G</td><td rowspan="1" colspan="1" align="right">0.4158(T)</td><td rowspan="1" colspan="1" align="right">0.5921 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="right">rs66154967</td><td rowspan="1" colspan="1" align="right">29,846,491</td><td rowspan="1" colspan="1" align="right">A/G</td><td rowspan="1" colspan="1" align="right">0.1505(A)</td><td rowspan="1" colspan="1" align="right">0.7514 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="right">rs66092412</td><td rowspan="1" colspan="1" align="right">56,060,316</td><td rowspan="1" colspan="1" align="right">C/T</td><td rowspan="1" colspan="1" align="right">0.3490(C)</td><td rowspan="1" colspan="1" align="right">0.5090 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">Female-specific</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="right">rs66076226</td><td rowspan="1" colspan="1" align="right">180,385,227</td><td rowspan="1" colspan="1" align="right">A/G</td><td rowspan="1" colspan="1" align="right">0.4884(A)</td><td rowspan="1" colspan="1" align="right">0.5618 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">rs66053327</td><td rowspan="1" colspan="1" align="right">10,213,937</td><td rowspan="1" colspan="1" align="right">C/T</td><td rowspan="1" colspan="1" align="right">0.1762(C)</td><td rowspan="1" colspan="1" align="right">0.3833 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">rs66123984</td><td rowspan="1" colspan="1" align="right">10,211,140</td><td rowspan="1" colspan="1" align="right">C/T</td><td rowspan="1" colspan="1" align="right">0.3388(C)</td><td rowspan="1" colspan="1" align="right">0.3266 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">rs66114880</td><td rowspan="1" colspan="1" align="right">10,208,865</td><td rowspan="1" colspan="1" align="right">G/C</td><td rowspan="1" colspan="1" align="right">0.3307(G)</td><td rowspan="1" colspan="1" align="right">0.031 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="right">rs66347789</td><td rowspan="1" colspan="1" align="right">10,193,197</td><td rowspan="1" colspan="1" align="right">A/C</td><td rowspan="1" colspan="1" align="right">0.3702(A)</td><td rowspan="1" colspan="1" align="right">0.00014 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="right">rs66447584</td><td rowspan="1" colspan="1" align="right">95,313,484</td><td rowspan="1" colspan="1" align="right">T/C</td><td rowspan="1" colspan="1" align="right">0.1846(T)</td><td rowspan="1" colspan="1" align="right">0.1409 × 10<sup>−6</sup></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="right">rs66225752</td><td rowspan="1" colspan="1" align="right">58,498,179</td><td rowspan="1" colspan="1" align="right">A/T</td><td rowspan="1" colspan="1" align="right">0.1226(A)</td><td rowspan="1" colspan="1" align="right">0.0566 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140119723201888" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="__p35" class="p p-last">Based on the fitted mean curves for different genotypes, we computed the additive and dominant effects over time. Figure <a href="/pmc/articles/PMC3725885/figure/F8/" target="figure" class="fig-table-link figpopup" rid-figpopup="F8" rid-ob="ob-F8" co-legend-rid="lgnd_F8"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">Figure8</span></span><span>8</span></a> displays the age-specific changes of additive and dominant effects on blood pressure in the different sexes. Those age-specific changes show a complicated pattern and in order to explain the molecular basis of these phenomena, one might want to refer to Kacser and Burns [<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191238">13</a>] and Keightley and Kacser [<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191239">14</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F8" co-legend-rid="lgnd_F8"><a href="/pmc/articles/PMC3725885/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119716589904" class="figure"><a href="/pmc/articles/PMC3725885/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g08.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g08.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g08.jpg"></a></div><div id="largeobj_idm140119716589904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F8/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F8"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F8/" target="figure" rid-figpopup="F8" rid-ob="ob-F8">Fig. 8</a></div><!--caption a7--><div class="caption"><p id="__p52">Additive and dominant effects for the most significant SNPs for males (<strong>a</strong>) and females (<strong>b</strong>), respectively.</p></div></div></div></div><div id="sec2_4" class="sec sec-last"><h3 id="sec2_4title">Simulation, Power and False Positive Rate Analysis</h3><p id="__p36" class="p p-first">Like for most GWAS results in the literature, we also performed a cross-validation analysis to investigate the reproducibility of bivariate <em>f</em> GWAS as proposed in this article. We split the data into the two sex groups which were of nearly equal size. We applied bivariate <em>f</em> GWAS to each group as well as on the complete sample and found that the same SNPs are detected as significant for males and females. To examine the power and the statistical properties of the newly proposed methodology, we performed a simulation study. Sparse trait values were simulated from the model in equation <a href="#uneq1" rid="uneq1" class=" other">1</a> and the covariance structure we used for original data analysis. We assumed a segregating SNP with different MAF associated with the trait under consideration, i.e. blood pressure. Table <a href="/pmc/articles/PMC3725885/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table3</span></span><span>3</span></a> shows the true model parameters, estimated model parameters and their standard errors. We note that the estimated values are very close to the true parameter values with small standard errors, which indicates that the method we propose is accurate and precise enough and hence applicable for practical purposes. In figure <a href="/pmc/articles/PMC3725885/figure/F9/" target="figure" class="fig-table-link figpopup" rid-figpopup="F9" rid-ob="ob-F9" co-legend-rid="lgnd_F9"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">figure9,</span></span><span>9</span></a>, we show the simulation result by plotting actual (solid) versus fitted (dashed) curves for different genotypes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F9" co-legend-rid="lgnd_F9"><a href="/pmc/articles/PMC3725885/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119716131424" class="figure"><a href="/pmc/articles/PMC3725885/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3725885_hhe-0072-0110-g09.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is hhe-0072-0110-g09.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3725885/bin/hhe-0072-0110-g09.jpg"></a></div><div id="largeobj_idm140119716131424" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/figure/F9/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F9"><div><a class="figpopup" href="/pmc/articles/PMC3725885/figure/F9/" target="figure" rid-figpopup="F9" rid-ob="ob-F9">Fig. 9</a></div><!--caption a7--><div class="caption"><p id="__p53">Simulation results: actual (solid) versus fitted (dashed) curves for different genotypes.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p56">Simulation results mimicking the original data</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119719301728" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr valign="top"><th rowspan="1" colspan="1" align="left">Parameter</th><th rowspan="1" colspan="1" align="left">True value</th><th rowspan="1" colspan="1" align="left">Estimate</th><th rowspan="1" colspan="1" align="left">Standard error</th></tr></thead><tbody><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>110</sub></td><td rowspan="1" colspan="1" align="left">27.09</td><td rowspan="1" colspan="1" align="left">26.62</td><td rowspan="1" colspan="1" align="left">0.0990</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>111</sub></td><td rowspan="1" colspan="1" align="left">2.84</td><td rowspan="1" colspan="1" align="left">2.72</td><td rowspan="1" colspan="1" align="left">0.0899</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>112</sub></td><td rowspan="1" colspan="1" align="left">1.78</td><td rowspan="1" colspan="1" align="left">1.64</td><td rowspan="1" colspan="1" align="left">0.0183</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>120</sub></td><td rowspan="1" colspan="1" align="left">21.71</td><td rowspan="1" colspan="1" align="left">20.66</td><td rowspan="1" colspan="1" align="left">0.0617</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>121</sub></td><td rowspan="1" colspan="1" align="left">1.67</td><td rowspan="1" colspan="1" align="left">1.65</td><td rowspan="1" colspan="1" align="left">0.0716</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>122</sub></td><td rowspan="1" colspan="1" align="left">0.97</td><td rowspan="1" colspan="1" align="left">0.94</td><td rowspan="1" colspan="1" align="left">0.0091</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>210</sub></td><td rowspan="1" colspan="1" align="left">29.19</td><td rowspan="1" colspan="1" align="left">28.33</td><td rowspan="1" colspan="1" align="left">0.0697</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>211</sub></td><td rowspan="1" colspan="1" align="left">2.17</td><td rowspan="1" colspan="1" align="left">2.39</td><td rowspan="1" colspan="1" align="left">0.0978</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>212</sub></td><td rowspan="1" colspan="1" align="left">−1.56</td><td rowspan="1" colspan="1" align="left">−1.58</td><td rowspan="1" colspan="1" align="left">0.0361</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>220</sub></td><td rowspan="1" colspan="1" align="left">27.91</td><td rowspan="1" colspan="1" align="left">27.75</td><td rowspan="1" colspan="1" align="left">0.0721</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>221</sub></td><td rowspan="1" colspan="1" align="left">1.77</td><td rowspan="1" colspan="1" align="left">1.81</td><td rowspan="1" colspan="1" align="left">0.0851</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>222</sub></td><td rowspan="1" colspan="1" align="left">−1.41</td><td rowspan="1" colspan="1" align="left">−1.38</td><td rowspan="1" colspan="1" align="left">0.0098</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>310</sub></td><td rowspan="1" colspan="1" align="left">34.19</td><td rowspan="1" colspan="1" align="left">33.17</td><td rowspan="1" colspan="1" align="left">0.0305</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>311</sub></td><td rowspan="1" colspan="1" align="left">1.17</td><td rowspan="1" colspan="1" align="left">1.153</td><td rowspan="1" colspan="1" align="left">0.0503</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>312</sub></td><td rowspan="1" colspan="1" align="left">1.79</td><td rowspan="1" colspan="1" align="left">1.73</td><td rowspan="1" colspan="1" align="left">0.105</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>320</sub></td><td rowspan="1" colspan="1" align="left">30.15</td><td rowspan="1" colspan="1" align="left">31.01</td><td rowspan="1" colspan="1" align="left">0.0104</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>321</sub></td><td rowspan="1" colspan="1" align="left">1.14</td><td rowspan="1" colspan="1" align="left">1.13</td><td rowspan="1" colspan="1" align="left">0.0310</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>u</em><sub>322</sub></td><td rowspan="1" colspan="1" align="left">1.84</td><td rowspan="1" colspan="1" align="left">1.81</td><td rowspan="1" colspan="1" align="left">0.1153</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>Σ<sub>s</sub></em></td><td rowspan="1" colspan="1" align="left">1.34</td><td rowspan="1" colspan="1" align="left">1.38</td><td rowspan="1" colspan="1" align="left">0.0598</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>Σ<sub>d</sub></em></td><td rowspan="1" colspan="1" align="left">1.86</td><td rowspan="1" colspan="1" align="left">1.88</td><td rowspan="1" colspan="1" align="left">0.0255</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>Σ<sub>sd</sub></em></td><td rowspan="1" colspan="1" align="left">1.17</td><td rowspan="1" colspan="1" align="left">1.13</td><td rowspan="1" colspan="1" align="left">0.0658</td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left"><em>ρ</em></td><td rowspan="1" colspan="1" align="left">0.65</td><td rowspan="1" colspan="1" align="left">0.62</td><td rowspan="1" colspan="1" align="left">0.0260</td></tr></tbody></table></div><div id="largeobj_idm140119719301728" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3725885/table/T3/?report=objectonly">Open in a separate window</a></div></div><p id="__p37" class="p p-last">The practical applicability and usefulness of the proposed joint analysis was assessed by further simulation studies. Under two different setups, when the bivariate data are affected by pleiotropic genes and when they are not affected by such genes, power and false positive rate (FPR) analyses are performed with different values for MAF. We considered two different sample sizes (n = 1,000 and 2,000) for each situation. Table <a href="/pmc/articles/PMC3725885/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table4</span></span><span>4</span></a> shows the results when pleiotropic genes are there and it is clear that in such a situation joint analysis performs much better in terms of power and FPR, i.e. the analysis results in higher power and lower FPR. However, table <a href="/pmc/articles/PMC3725885/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">table5</span></span><span>5</span></a> shows the results when there is no pleiotropy: separate analysis should be recommended. In such a situation, joint analysis will have lower power and higher FPR than a separate analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p57">Theoretical power and FPR of bivariate data affected by pleiotropic genes for separate and joint analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119715963040" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr valign="top"><th rowspan="2" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">MAF</th><th colspan="2" rowspan="1" align="left">Separate analysis<hr></th><th colspan="2" rowspan="1" align="left">Joint analysis<hr></th></tr><tr valign="top"><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">power</th><th rowspan="1" colspan="1" align="left">FPR</th><th rowspan="1" colspan="1" align="left">power</th><th rowspan="1" colspan="1" align="left">FPR</th></tr></thead><tbody><tr valign="top"><td rowspan="1" colspan="1" align="left">1,000</td><td rowspan="1" colspan="1" align="left">0.1</td><td rowspan="1" colspan="1" align="left">0.67</td><td rowspan="1" colspan="1" align="left">0.115</td><td rowspan="1" colspan="1" align="left">0.72</td><td rowspan="1" colspan="1" align="left">0.103</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.2</td><td rowspan="1" colspan="1" align="left">0.65</td><td rowspan="1" colspan="1" align="left">0.114</td><td rowspan="1" colspan="1" align="left">0.71</td><td rowspan="1" colspan="1" align="left">0.0914</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.3</td><td rowspan="1" colspan="1" align="left">0.70</td><td rowspan="1" colspan="1" align="left">0.102</td><td rowspan="1" colspan="1" align="left">0.77</td><td rowspan="1" colspan="1" align="left">0.0681</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.4</td><td rowspan="1" colspan="1" align="left">0.73</td><td rowspan="1" colspan="1" align="left">0.085</td><td rowspan="1" colspan="1" align="left">0.82</td><td rowspan="1" colspan="1" align="left">0.0502</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.5</td><td rowspan="1" colspan="1" align="left">0.72</td><td rowspan="1" colspan="1" align="left">0.082</td><td rowspan="1" colspan="1" align="left">0.83</td><td rowspan="1" colspan="1" align="left">0.0601</td></tr><tr valign="top"><td colspan="6" rowspan="1" align="left"><hr></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left">2, 000</td><td rowspan="1" colspan="1" align="left">0.1</td><td rowspan="1" colspan="1" align="left">0.71</td><td rowspan="1" colspan="1" align="left">0.110</td><td rowspan="1" colspan="1" align="left">0.81</td><td rowspan="1" colspan="1" align="left">0.0931</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.2</td><td rowspan="1" colspan="1" align="left">0.71</td><td rowspan="1" colspan="1" align="left">0.098</td><td rowspan="1" colspan="1" align="left">0.80</td><td rowspan="1" colspan="1" align="left">0.0773</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.3</td><td rowspan="1" colspan="1" align="left">0.74</td><td rowspan="1" colspan="1" align="left">0.092</td><td rowspan="1" colspan="1" align="left">0.82</td><td rowspan="1" colspan="1" align="left">0.0504</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.4</td><td rowspan="1" colspan="1" align="left">0.73</td><td rowspan="1" colspan="1" align="left">0.088</td><td rowspan="1" colspan="1" align="left">0.83</td><td rowspan="1" colspan="1" align="left">0.0501</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.5</td><td rowspan="1" colspan="1" align="left">0.72</td><td rowspan="1" colspan="1" align="left">0.081</td><td rowspan="1" colspan="1" align="left">0.83</td><td rowspan="1" colspan="1" align="left">0.0519</td></tr></tbody></table></div><div id="largeobj_idm140119715963040" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/table/T4/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p58">Theoretical power and FPR of bivariate data not affected by pleiotropic genes for separate and joint analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140119715871536" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr valign="top"><th rowspan="2" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">MAF</th><th colspan="2" rowspan="1" align="left">Separate analysis<hr></th><th colspan="2" rowspan="1" align="left">Joint analysis<hr></th></tr><tr valign="top"><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">power</th><th rowspan="1" colspan="1" align="left">FPR</th><th rowspan="1" colspan="1" align="left">power</th><th rowspan="1" colspan="1" align="left">FPR</th></tr></thead><tbody><tr valign="top"><td rowspan="1" colspan="1" align="left">1,000</td><td rowspan="1" colspan="1" align="left">0.1</td><td rowspan="1" colspan="1" align="left">0.68</td><td rowspan="1" colspan="1" align="left">0.112</td><td rowspan="1" colspan="1" align="left">0.61</td><td rowspan="1" colspan="1" align="left">0.118</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.2</td><td rowspan="1" colspan="1" align="left">0.67</td><td rowspan="1" colspan="1" align="left">0.110</td><td rowspan="1" colspan="1" align="left">0.60</td><td rowspan="1" colspan="1" align="left">0.117</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.3</td><td rowspan="1" colspan="1" align="left">0.73</td><td rowspan="1" colspan="1" align="left">0.093</td><td rowspan="1" colspan="1" align="left">0.62</td><td rowspan="1" colspan="1" align="left">0.114</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.4</td><td rowspan="1" colspan="1" align="left">0.74</td><td rowspan="1" colspan="1" align="left">0.094</td><td rowspan="1" colspan="1" align="left">0.63</td><td rowspan="1" colspan="1" align="left">0.112</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.5</td><td rowspan="1" colspan="1" align="left">0.74</td><td rowspan="1" colspan="1" align="left">0.088</td><td rowspan="1" colspan="1" align="left">0.63</td><td rowspan="1" colspan="1" align="left">0.099</td></tr><tr valign="top"><td colspan="6" rowspan="1" align="left"><hr></td></tr><tr valign="top"><td rowspan="1" colspan="1" align="left">2,000</td><td rowspan="1" colspan="1" align="left">0.1</td><td rowspan="1" colspan="1" align="left">0.70</td><td rowspan="1" colspan="1" align="left">0.097</td><td rowspan="1" colspan="1" align="left">0.63</td><td rowspan="1" colspan="1" align="left">0.109</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.2</td><td rowspan="1" colspan="1" align="left">0.69</td><td rowspan="1" colspan="1" align="left">0.089</td><td rowspan="1" colspan="1" align="left">0.62</td><td rowspan="1" colspan="1" align="left">0.098</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.3</td><td rowspan="1" colspan="1" align="left">0.75</td><td rowspan="1" colspan="1" align="left">0.085</td><td rowspan="1" colspan="1" align="left">0.65</td><td rowspan="1" colspan="1" align="left">0.096</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.4</td><td rowspan="1" colspan="1" align="left">0.75</td><td rowspan="1" colspan="1" align="left">0.085</td><td rowspan="1" colspan="1" align="left">0.66</td><td rowspan="1" colspan="1" align="left">0.096</td></tr><tr valign="top"><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="left">0.5</td><td rowspan="1" colspan="1" align="left">0.74</td><td rowspan="1" colspan="1" align="left">0.082</td><td rowspan="1" colspan="1" align="left">0.65</td><td rowspan="1" colspan="1" align="left">0.092</td></tr></tbody></table></div><div id="largeobj_idm140119715871536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3725885/table/T5/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="sec1_4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_4title">Discussion</h2><p id="__p38" class="p p-first">In last three years, there have been a number of successful applications of GWAS which is indeed a consequence of the successful completion of the Human Genome Project and its derivative hapmap project. By analyzing thousands of SNPs, this approach can effectively identify the function of multiple genetic changes that are responsible for polygenic traits and diseases. Despite of several limitations, GWAS play a major role in biomedical researches and in medical genetics too.</p><p id="__p39">Most of the traditional GWAS consider one time point measurement per subject for the trait under consideration. In all senses, a curve is more informative than a single point. This motivated us to develop GWAS with longitudinal traits. In most practical studies, the subjects are measured at irregular time points which give rise to irregular sparse longitudinal data. Advanced statistical approaches can effectively model the mean and covariance structure for these data, and hence they can produce more meaningful inferences. By incorporating the functional aspects of the observed trait in the traditional GWAS, a new approach (<em>f</em> GWAS) has been developed.</p><p id="__p40">When we have a bivariate response variable and we collect data longitudinally per subject, a simple extension of the <em>f</em> GWAS can work better than the traditional approaches. The only challenge here is to model the covariance structure appropriately. The present article deals with nonparametric modeling of the mean function and a parametric covariance structure assuming that the intra-response correlation is the same for each response and the inter-response correlation is directly proportional to the intra-response correlation. However, authors are currently developing a new approach where this assumption will be relaxed and a more powerful <em>f</em> GWAS can be performed in the near future. Once genes having significant effects on blood pressure are identified, heart diseases which are the consequences of abnormal blood pressures can be treated and controlled more effectively. Also the same procedure could be applied to detect genes controlling other traits and diseases too.</p><p id="__p41">In this article, we mainly focused on bivariate traits like systolic and diastolic blood pressure. The underlying assumption for this work is that when we observe one trait for a specific subject at a particular time point, we must also have measurement for the second trait at that point. For traits like blood pressure, this is true in almost all cases, but it might not be the case in a general setting. A more sophisticated method needs to be developed to handle those situations. For our method, both traits need to be measured concurrently. Even a simple extension of this method works for multivariate traits under the same underlying assumption.</p><p id="__p42" class="p p-last">One limitation of the approach we propose in this article is that we treat all subjects under study independent of each other. However, in reality, the subjects might be genetically related or even come from the same family. Once we have the information for genetic dependence of the subjects, we can incorporate that in our method and this will reflect biostatistically more informative results. We will also need to consider the impact of genetic imprinting [<a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_326191237">15</a>]. Once we do that, we can actually address many meaningful and challenging issues in genetics and biomedical sciences.</p></div><div id="__sec4" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec4title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="S1"><p id="__p43">Erratum</p><div class="sup-box half_rhythm" id="idm140119714965024"><a href="/pmc/articles/PMC3725885/bin/hhe-0072-0110-s01.pdf" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(24K, pdf)</sup></div></div></div><div id="idm140119714962944" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140119714962944title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="mixed-citation">Das K, Li J, Wang Z, Tong C, Fu G, Li Y, Xu M, Ahn K, Mauger D, Li R, Wu RL. A dynamic model for genome-wide association studies. <span class="ref-journal">Human Genetics</span> 2011; DOI 10.1007/s00439-011-0960-6. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3103104/">PMC free article</a>]</span> [<a href="/pubmed/21293879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>]</span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Laird NM, Ware JH. Random effects model for longitudinal data. <span><span class="ref-journal">Biometrics. </span>1982;<span class="ref-vol">38</span>:963–974.</span> [<a href="/pubmed/7168798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Random+effects+model+for+longitudinal+data&amp;author=NM+Laird&amp;author=JH+Ware&amp;volume=38&amp;publication_year=1982&amp;pages=963-974&amp;pmid=7168798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Thiebaut R, Jacqmin-Gadda H, Chene G, Leport C, Commenges D. Bivariate linear mixed models using SAS proc MIXED. <span><span class="ref-journal">Comput Methods Programs Biomed. </span>2002;<span class="ref-vol">69</span>:249–256.</span> [<a href="/pubmed/12204452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Comput+Methods+Programs+Biomed&amp;title=Bivariate+linear+mixed+models+using+SAS+proc+MIXED&amp;author=R+Thiebaut&amp;author=H+Jacqmin-Gadda&amp;author=G+Chene&amp;author=C+Leport&amp;author=D+Commenges&amp;volume=69&amp;publication_year=2002&amp;pages=249-256&amp;pmid=12204452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Sithole JS, Jones PW. Bivariate longitudinal model for detecting prescribing change in two drugs simultaneously with correlated errors. <span><span class="ref-journal">J Appl Stat. </span>2007;<span class="ref-vol">34</span>:339–352.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Appl+Stat&amp;title=Bivariate+longitudinal+model+for+detecting+prescribing+change+in+two+drugs+simultaneously+with+correlated+errors&amp;author=JS+Sithole&amp;author=PW+Jones&amp;volume=34&amp;publication_year=2007&amp;pages=339-352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Cui Y, Zhu J, Wu RL. Functional mapping for genetic control of programmed cell death. <span><span class="ref-journal">Physiol Genomics. </span>2006;<span class="ref-vol">25</span>:458–469.</span> [<a href="/pubmed/16464975" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Functional+mapping+for+genetic+control+of+programmed+cell+death&amp;author=Y+Cui&amp;author=J+Zhu&amp;author=RL+Wu&amp;volume=25&amp;publication_year=2006&amp;pages=458-469&amp;pmid=16464975&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Meyer K. Random regressions to model phenotypic variation in monthly weights of Australian beef cows. <span><span class="ref-journal">Livest Prod Science. </span>2000;<span class="ref-vol">65</span>:19–38.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Livest+Prod+Science&amp;title=Random+regressions+to+model+phenotypic+variation+in+monthly+weights+of+Australian+beef+cows&amp;author=K+Meyer&amp;volume=65&amp;publication_year=2000&amp;pages=19-38&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Huskova M, Sen PK. On sequentially adaptive asymptotically efficient rank statistics. <span><span class="ref-journal">Seq Anal. </span>1985;<span class="ref-vol">4</span>:125–151.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Seq+Anal&amp;title=On+sequentially+adaptive+asymptotically+efficient+rank+statistics&amp;author=M+Huskova&amp;author=PK+Sen&amp;volume=4&amp;publication_year=1985&amp;pages=125-151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Guiot C, Degiorgis PG, Delsanto PP, Gabriele P, Deisboeck TS. Does tumor growth follow a ‘universal law’? <span><span class="ref-journal">J Theor Biol. </span>2003;<span class="ref-vol">225</span>:147–151.</span> [<a href="/pubmed/14575649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Theor+Biol&amp;title=Does+tumor+growth+follow+a+‘universal+law’?&amp;author=C+Guiot&amp;author=PG+Degiorgis&amp;author=PP+Delsanto&amp;author=P+Gabriele&amp;author=TS+Deisboeck&amp;volume=225&amp;publication_year=2003&amp;pages=147-151&amp;pmid=14575649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Bock RD, Thissen D. Fitting multi-component models for growth in stature. <span><span class="ref-journal">Proceedings of the Ninth International Biometrics Conference. </span>1976;<span class="ref-vol">1</span>:431–442.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+Ninth+International+Biometrics+Conference&amp;title=Fitting+multi-component+models+for+growth+in+stature&amp;author=RD+Bock&amp;author=D+Thissen&amp;volume=1&amp;publication_year=1976&amp;pages=431-442&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Fan J, Huang T, Li R. Analysis of longitudinal data with semi-parametric estimation of covariance function. <span><span class="ref-journal">J Am Statist Assoc. </span>2007;<span class="ref-vol">102</span>:632–641.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2730591/">PMC free article</a>]</span> [<a href="/pubmed/19707537" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Statist+Assoc&amp;title=Analysis+of+longitudinal+data+with+semi-parametric+estimation+of+covariance+function&amp;author=J+Fan&amp;author=T+Huang&amp;author=R+Li&amp;volume=102&amp;publication_year=2007&amp;pages=632-641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Fan J, Wu Y. Semiparametric estimation of covariance matrixes for longitudinal data. <span><span class="ref-journal">J Am Statist Assoc. </span>2008;<span class="ref-vol">103</span>:1520–1533.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2631936/">PMC free article</a>]</span> [<a href="/pubmed/19180247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Statist+Assoc&amp;title=Semiparametric+estimation+of+covariance+matrixes+for+longitudinal+data&amp;author=J+Fan&amp;author=Y+Wu&amp;volume=103&amp;publication_year=2008&amp;pages=1520-1533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <span><span class="ref-journal">J Roy Statist Soc Ser B. </span>1995;<span class="ref-vol">57</span>:289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Roy+Statist+Soc+Ser+B&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Kacser H, Bums JA. The molecular basis of dominance. <span><span class="ref-journal">Genetics. </span>1981;<span class="ref-vol">97</span>:639–666.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1214416/">PMC free article</a>]</span> [<a href="/pubmed/7297851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The+molecular+basis+of+dominance&amp;author=H+Kacser&amp;author=JA+Bums&amp;volume=97&amp;publication_year=1981&amp;pages=639-666&amp;pmid=7297851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Keightley PD, Kacser H. Dominance, pleiotropy and metabolic structure. <span><span class="ref-journal">Genetics. </span>1987;<span class="ref-vol">117</span>:319–329.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1203207/">PMC free article</a>]</span> [<a href="/pubmed/3666444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Dominance,+pleiotropy+and+metabolic+structure&amp;author=PD+Keightley&amp;author=H+Kacser&amp;volume=117&amp;publication_year=1987&amp;pages=319-329&amp;pmid=3666444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Luedi PP, Hartemink AJ, Jirtle RL. Genome-wide prediction of imprinted murine genes. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:875–884.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1142478/">PMC free article</a>]</span> [<a href="/pubmed/15930497" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Genome-wide+prediction+of+imprinted+murine+genes&amp;author=PP+Luedi&amp;author=AJ+Hartemink&amp;author=RL+Jirtle&amp;volume=15&amp;publication_year=2005&amp;pages=875-884&amp;pmid=15930497&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140119712299776title">Abstract</a></li><li><a href="#sec1_1title">Introduction</a></li><li><a href="#sec1_2title">Subjects and Methods</a></li><li><a href="#sec1_3title">Results</a></li><li><a href="#sec1_4title">Discussion</a></li><li><a href="#__sec4title">Supplementary Material</a></li><li><a href="#idm140119714962944title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Heredity</span> are provided here courtesy of <strong>Karger Publishers</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3725885/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3725885/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3725885/pdf/hhe-0072-0110.pdf">PDF (747K)</a></li> | <li><a href="#" data-citationid="PMC3725885" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3725885%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3725885%2F&amp;text=Genome-Wide%20Association%20Studies%20for%20Bivariate%20Sparse%20Longitudinal%20Data" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3725885%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21996601" data-aiid="3725885" data-aid="3725885" data-iid="206895" data-domainid="1008" data-domain="hhe" data-accid="PMC3725885" data-md5="bacc37b52e22fe49bc7cb6cd8eb6d339">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-5" aria-hidden="true"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="3725885" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/17903302" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Framingham Heart Study 100K Project: genome-wide associations for blood pressure and arterial stiffness.</a><span class="source">[BMC Med Genet. 2007]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Levy D, Larson MG, Benjamin EJ, Newton-Cheh C, Wang TJ, Hwang SJ, Vasan RS, Mitchell GF. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Med Genet. 2007 Sep 19; 8 Suppl 1:S3. Epub 2007 Sep 19.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/24400941" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A semiparametric approach to simultaneous covariance estimation for bivariate sparse longitudinal data.</a><span class="source">[Biometrics. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Das K, Daniels MJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Biometrics. 2014 Mar; 70(1):33-43. Epub 2014 Jan 8.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/27859628" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Longitudinal SNP-set association analysis of quantitative phenotypes.</a><span class="source">[Genet Epidemiol. 2017]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Wang Z, Xu K, Zhang X, Wu X, Wang Z. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genet Epidemiol. 2017 Jan; 41(1):81-93. Epub 2016 Nov 9.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/17848626" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A 100K genome-wide association scan for diabetes and related traits in the Framingham Heart Study: replication and integration with other genome-wide datasets.</a><span class="source">[Diabetes. 2007]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Florez JC, Manning AK, Dupuis J, McAteer J, Irenze K, Gianniny L, Mirel DB, Fox CS, Cupples LA, Meigs JB. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Diabetes. 2007 Dec; 56(12):3063-74. Epub 2007 Sep 11.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/25112192" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Longitudinal data analysis in genome-wide association studies.</a><span class="source">[Genet Epidemiol. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Beyene J, Hamid JS. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Genet Epidemiol. 2014 Sep; 38 Suppl 1:S68-73. </em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=21996601&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=21996601&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-12" aria-hidden="true">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC5133524/" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association of trajectories of systolic blood pressure change</a><span class="source">[BMC Proceedings. 2016]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Justice AE, Howard AG, Chittoor G, Fernandez-Rhodes L, Graff M, Voruganti VS, Diao G, Love SA, Franceschini N, O’Connell JR, Avery CL, Young KL, North KE. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">BMC Proceedings. 2016 Oct 18; 10(Suppl 7): 321-327</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC4622042/" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Multi-Trait GWAS and New Candidate Genes Annotation for Growth Curve Parameters in Brahman Cattle</a><span class="source">[PLoS ONE. 2015]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Crispim AC, Kelly MJ, Guimarães SE, e Silva FF, Fortes MR, Wenceslau RR, Moore S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">PLoS ONE. 2015 Oct 7; 10(10): e0139906</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC3954444/" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A Semiparametric Approach to Simultaneous Covariance Estimation for Bivariate Sparse Longitudinal Data</a><span class="source">[Biometrics. 2014]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Das K, Daniels MJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Biometrics. 2014 Jan 8; 70(1): 33-43</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC3740051/" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A Bayesian Semiparametric Model for Bivariate Sparse Longitudinal Data</a><span class="source">[Statistics in medicine. 2013]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Das K, Li R, Sengupta S, Wu R. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Statistics in medicine. 2013 Apr 4; 32(22): 3899-3910</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-17" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-17" aria-hidden="true"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=3725885" ref="reftype=MedGen&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-18" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/21996601/" ref="reftype=PubMed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=snp&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_snp&amp;IdsFromResult=3725885" ref="reftype=SNP&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|SNP&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">SNP</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">SNP<div class="brieflinkpopdesc">PMC to SNP links</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=3725885" ref="reftype=Taxonomy&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-22" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-22" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd536fedde0893a8ba18b1c">Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data</a><div class="ralinkpop offscreen_noflow">Genome-Wide Association Studies for Bivariate Sparse Longitudinal Data<div class="brieflinkpopdesc">Human Heredity. 2011 Oct; 72(2)110</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd536e6dd2142621d66c685">Genomewide Association between GLCCI1 and Response to Glucocorticoid Therapy in ...</a><div class="ralinkpop offscreen_noflow">Genomewide Association between GLCCI1 and Response to Glucocorticoid Therapy in Asthma<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2011 Sep 29; 365(13)1173</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd536c675f66d018f85252f">Variants Near FOXE1 Are Associated with Hypothyroidism and Other Thyroid Conditi...</a><div class="ralinkpop offscreen_noflow">Variants Near FOXE1 Are Associated with Hypothyroidism and Other Thyroid Conditions: Using Electronic Medical Records for Genome- and Phenome-wide Studies<div class="brieflinkpopdesc">American Journal of Human Genetics. 2011 Oct 7; 89(4)529</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd536ac39a66eb03d74fd1a">A genome-wide association study identifies a potential novel gene locus for kera...</a><div class="ralinkpop offscreen_noflow">A genome-wide association study identifies a potential novel gene locus for keratoconus, one of the commonest causes for corneal transplantation in developed countries<div class="brieflinkpopdesc">Human Molecular Genetics. 2012 Jan 15; 21(2)421</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd53676dde0893a8b8a9526">Genome-wide association study of coronary artery disease in the Japanese</a><div class="ralinkpop offscreen_noflow">Genome-wide association study of coronary artery disease in the Japanese<div class="brieflinkpopdesc">European Journal of Human Genetics. 2012 Mar; 20(3)333</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
<div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p5" name="crb--__p5" rid="__p5" style="position: absolute; visibility: visible; width: 370px; top: 1436px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-23"><ul><li class="two_line" reference_id="B1">
<a href="/pubmed/21293879/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A dynamic model for genome-wide association studies.</a><span class="one_line_source">[Hum Genet.  2011]</span><div class="alt-note">
<div class="authors">Das K, Li J, Wang Z, Tong C, Fu G, Li Y, Xu M, Ahn K, Mauger D, Li R, Wu R</div>
<div class="citation">Hum Genet. 2011 Jun; 129(6):629-39.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p6" name="crb--__p6" rid="__p6" style="position: absolute; visibility: visible; width: 370px; top: 1667px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-24"><ul>
<li class="two_line" reference_id="B2">
<a href="/pubmed/7168798/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Random-effects models for longitudinal data.</a><span class="one_line_source">[Biometrics.  1982]</span><div class="alt-note">
<div class="authors">Laird NM, Ware JH</div>
<div class="citation">Biometrics. 1982 Dec; 38(4):963-74.</div>
</div>
</li>
<li class="two_line" reference_id="B3">
<a href="/pubmed/12204452/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Bivariate linear mixed models using SAS proc MIXED.</a><span class="one_line_source">[Comput Methods Programs Biomed.  2002]</span><div class="alt-note">
<div class="authors">Thiébaut R, Jacqmin-Gadda H, Chêne G, Leport C, Commenges D</div>
<div class="citation">Comput Methods Programs Biomed. 2002 Nov; 69(3):249-56.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p12" name="crb--__p12" rid="__p12" style="position: absolute; visibility: visible; width: 370px; top: 3076px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-25"><ul><li class="two_line" reference_id="B5">
<a href="/pubmed/16464975/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Functional mapping for genetic control of programmed cell death.</a><span class="one_line_source">[Physiol Genomics.  2006]</span><div class="alt-note">
<div class="authors">Cui Y, Zhu J, Wu R</div>
<div class="citation">Physiol Genomics. 2006 May 16; 25(3):458-69.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p15" name="crb--__p15" rid="__p15" style="position: absolute; visibility: visible; width: 370px; top: 3535px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-26"><ul><li class="two_line" reference_id="B1">
<a href="/pubmed/21293879/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">A dynamic model for genome-wide association studies.</a><span class="one_line_source">[Hum Genet.  2011]</span><div class="alt-note">
<div class="authors">Das K, Li J, Wang Z, Tong C, Fu G, Li Y, Xu M, Ahn K, Mauger D, Li R, Wu R</div>
<div class="citation">Hum Genet. 2011 Jun; 129(6):629-39.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p18" name="crb--__p18" rid="__p18" style="position: absolute; visibility: visible; width: 370px; top: 4005px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-27"><ul>
<li class="two_line" reference_id="B8">
<a href="/pubmed/14575649/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Does tumor growth follow a "universal law"?</a><span class="one_line_source">[J Theor Biol.  2003]</span><div class="alt-note">
<div class="authors">Guiot C, Degiorgis PG, Delsanto PP, Gabriele P, Deisboeck TS</div>
<div class="citation">J Theor Biol. 2003 Nov 21; 225(2):147-51.</div>
</div>
</li>
<li class="two_line" reference_id="B5">
<a href="/pubmed/16464975/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Functional mapping for genetic control of programmed cell death.</a><span class="one_line_source">[Physiol Genomics.  2006]</span><div class="alt-note">
<div class="authors">Cui Y, Zhu J, Wu R</div>
<div class="citation">Physiol Genomics. 2006 May 16; 25(3):458-69.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p19" name="crb--__p19" rid="__p19" style="position: absolute; visibility: visible; width: 370px; top: 4214px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-28"><ul>
<li class="two_line" reference_id="B10">
<a href="/pubmed/19707537/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Analysis of Longitudinal Data with Semiparametric Estimation of Covariance Function.</a><span class="one_line_source">[J Am Stat Assoc.  2007]</span><div class="alt-note">
<div class="authors">Fan J, Huang T, Li R</div>
<div class="citation">J Am Stat Assoc. 2007 Jun 1; 102(478):632-641.</div>
</div>
</li>
<li class="two_line" reference_id="B11">
<a href="/pubmed/19180247/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Semiparametric estimation of covariance matrices for longitudinal data.</a><span class="one_line_source">[J Am Stat Assoc.  2008]</span><div class="alt-note">
<div class="authors">Fan J, Wu Y</div>
<div class="citation">J Am Stat Assoc. 2008 Dec 1; 103(484):1520-1533.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p22" name="crb--__p22" rid="__p22" style="position: absolute; visibility: visible; width: 370px; top: 4846px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-29"><ul><li class="two_line" reference_id="B3">
<a href="/pubmed/12204452/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Bivariate linear mixed models using SAS proc MIXED.</a><span class="one_line_source">[Comput Methods Programs Biomed.  2002]</span><div class="alt-note">
<div class="authors">Thiébaut R, Jacqmin-Gadda H, Chêne G, Leport C, Commenges D</div>
<div class="citation">Comput Methods Programs Biomed. 2002 Nov; 69(3):249-56.</div>
</div>
</li></ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p35" name="crb--__p35" rid="__p35" style="position: absolute; visibility: visible; width: 370px; top: 11940px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-30"><ul>
<li class="two_line" reference_id="B13">
<a href="/pubmed/7297851/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">The molecular basis of dominance.</a><span class="one_line_source">[Genetics.  1981]</span><div class="alt-note">
<div class="authors">Kacser H, Burns JA</div>
<div class="citation">Genetics. 1981 Mar-Apr; 97(3-4):639-66.</div>
</div>
</li>
<li class="two_line" reference_id="B14">
<a href="/pubmed/3666444/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Dominance, pleiotropy and metabolic structure.</a><span class="one_line_source">[Genetics.  1987]</span><div class="alt-note">
<div class="authors">Keightley PD, Kacser H</div>
<div class="citation">Genetics. 1987 Oct; 117(2):319-29.</div>
</div>
</li>
</ul></div>
</div>
<div class="portlet brieflink pmc_para_cit" id="crb--__p42" name="crb--__p42" rid="__p42" style="position: absolute; visibility: visible; width: 370px; top: 16036px;">
<div class="portlet_head"></div>
<div class="portlet_content" id="ui-portlet_content-31"><ul><li class="two_line" reference_id="B15">
<a href="/pubmed/15930497/" ref="reftype=pubmed&amp;article-id=3725885&amp;issue-id=206895&amp;journal-id=1008&amp;FROM=Article|CitedRefBlock&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal">Genome-wide prediction of imprinted murine genes.</a><span class="one_line_source">[Genome Res.  2005]</span><div class="alt-note">
<div class="authors">Luedi PP, Hartemink AJ, Jirtle RL</div>
<div class="citation">Genome Res. 2005 Jun; 15(6):875-84.</div>
</div>
</li></ul></div>
</div>
</div>   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:52:14-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C2854DD534CF100000000015300E4&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3725885%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hhe&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3725885/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size2, sans-serif;"></div></div></body></html>