<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="brain"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3613709/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide association study of brain lesion distribution in multiple sclerosis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3613709/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Brain"><meta name="citation_title" content="A genome-wide association study of brain lesion distribution in multiple sclerosis"><meta name="citation_authors" content="Pierre-Antoine Gourraud, Michael Sdika, Pouya Khankhanian, Roland G. Henry, Azadeh Beheshtian, Paul M. Matthews, Stephen L. Hauser, Jorge R. Oksenberg, Daniel Pelletier, Sergio E. Baranzini"><meta name="citation_date" content="April 2013"><meta name="citation_issue" content="4"><meta name="citation_volume" content="136"><meta name="citation_firstpage" content="1012"><meta name="citation_doi" content="10.1093/brain/aws363"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3613709/?report=abstract"><meta name="citation_pmid" content="23412934"><meta name="DC.Title" content="A genome-wide association study of brain lesion distribution in multiple sclerosis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Pierre-Antoine Gourraud"><meta name="DC.Contributor" content="Michael Sdika"><meta name="DC.Contributor" content="Pouya Khankhanian"><meta name="DC.Contributor" content="Roland G. Henry"><meta name="DC.Contributor" content="Azadeh Beheshtian"><meta name="DC.Contributor" content="Paul M. Matthews"><meta name="DC.Contributor" content="Stephen L. Hauser"><meta name="DC.Contributor" content="Jorge R. Oksenberg"><meta name="DC.Contributor" content="Daniel Pelletier"><meta name="DC.Contributor" content="Sergio E. Baranzini"><meta name="DC.Date" content="2013 Apr"><meta name="DC.Identifier" content="10.1093/brain/aws363"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide association study of brain lesion distribution in multiple sclerosis"><meta property="og:type" content="article"><meta property="og:description" content="Brain magnetic resonance imaging is widely used as a diagnostic and monitoring tool in multiple sclerosis and provides a non-invasive, sensitive and reproducible way to track the disease. Topological characteristics relating to the distribution and shape ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3613709/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-brain.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3613709/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3613709/pdf/aws363.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8888B4DD5560010000000002D301DC.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3613709%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Brain%22[journal]" class="navlink">Brain</a></li><li class="accid">PMC3613709</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-brain.gif" alt="Logo of brain" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="10,40,143,75" alt="Instructions to Authors" title="Instructions to Authors" href="http://www.oxfordjournals.org/page/6235/7" ref="reftype=publisher&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="144,40,197,75" alt="Subcribe" title="Subcribe" href="http://www.oxfordjournals.org/page/6235/6" ref="reftype=publisher&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="198,40,239,75" alt="About" title="About" href="http://www.oxfordjournals.org/page/6235/8" ref="reftype=publisher&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="240,40,321,75" alt="Free Editorials" title="Free Editorials" href="http://www.oxfordjournals.org/page/6235/9" ref="reftype=publisher&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,500,75" alt="Brain" title="Brain" href="http://brain.oxfordjournals.org/" ref="reftype=publisher&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Brain</a></span>. 2013 Apr; 136(4): 1012–1024. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Feb 13. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fbrain%2Faws363" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/brain/aws363</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3613709</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23412934">23412934</a></div></div></div></div><h1 class="content-title">A genome-wide association study of brain lesion distribution in multiple sclerosis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gourraud%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909714514928" co-class="co-affbox">Pierre-Antoine Gourraud</a>,<sup>1</sup> <a href="/pubmed/?term=Sdika%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909722078240" co-class="co-affbox">Michael Sdika</a>,<sup>1</sup> <a href="/pubmed/?term=Khankhanian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909718803392" co-class="co-affbox">Pouya Khankhanian</a>,<sup>1</sup> <a href="/pubmed/?term=Henry%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909758014144" co-class="co-affbox">Roland G. Henry</a>,<sup>1</sup> <a href="/pubmed/?term=Beheshtian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909758012480" co-class="co-affbox">Azadeh Beheshtian</a>,<sup>1</sup> <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909716168784" co-class="co-affbox">Paul M. Matthews</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909716165808" co-class="co-affbox">Stephen L. Hauser</a>,<sup>1</sup> <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909756722288" co-class="co-affbox">Jorge R. Oksenberg</a>,<sup>1</sup> <a href="/pubmed/?term=Pelletier%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909795336784" co-class="co-affbox">Daniel Pelletier</a>,<sup>1,</sup><sup>4</sup> and  <a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934" class="affpopup" co-rid="_co_idm139909795333808" co-class="co-affbox">Sergio E. Baranzini</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139909714514928"><h3 class="no_margin">Pierre-Antoine Gourraud</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Gourraud%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Pierre-Antoine Gourraud</a></div></div><div id="_co_idm139909722078240"><h3 class="no_margin">Michael Sdika</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Sdika%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Michael Sdika</a></div></div><div id="_co_idm139909718803392"><h3 class="no_margin">Pouya Khankhanian</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Khankhanian%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Pouya Khankhanian</a></div></div><div id="_co_idm139909758014144"><h3 class="no_margin">Roland G. Henry</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Henry%20RG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Roland G. Henry</a></div></div><div id="_co_idm139909758012480"><h3 class="no_margin">Azadeh Beheshtian</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Beheshtian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Azadeh Beheshtian</a></div></div><div id="_co_idm139909716168784"><h3 class="no_margin">Paul M. Matthews</h3><p>2 GlaxoSmithKline Clinical Imaging Centre, Hammersmith Hospital, London W12 0NN, UK</p><p>3 Department of Medicine, Centre for Neurosciences, Imperial College, London WC12 0NN, UK</p><div>Find articles by <a href="/pubmed/?term=Matthews%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Paul M. Matthews</a></div></div><div id="_co_idm139909716165808"><h3 class="no_margin">Stephen L. Hauser</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Hauser%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Stephen L. Hauser</a></div></div><div id="_co_idm139909756722288"><h3 class="no_margin">Jorge R. Oksenberg</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Oksenberg%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Jorge R. Oksenberg</a></div></div><div id="_co_idm139909795336784"><h3 class="no_margin">Daniel Pelletier</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><p>4 Division of Neuro-immunology, Department of Neurology, Yale School of Medicine, New Haven, CT, USA</p><div>Find articles by <a href="/pubmed/?term=Pelletier%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Daniel Pelletier</a></div></div><div id="_co_idm139909795333808"><h3 class="no_margin">Sergio E. Baranzini</h3><p>1 Department of Neurology, University of California, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Baranzini%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23412934">Sergio E. Baranzini</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139909796136112_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139909796136112_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139909796136112_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139909796136112_ai" style="display:none"><div class="fm-affl" id="aws363-AFF1" lang="en">1 Department of Neurology, University of California, San Francisco, CA, USA</div><div class="fm-affl" id="aws363-AFF2" lang="en">2 GlaxoSmithKline Clinical Imaging Centre, Hammersmith Hospital, London W12 0NN, UK</div><div class="fm-affl" id="aws363-AFF3" lang="en">3 Department of Medicine, Centre for Neurosciences, Imperial College, London WC12 0NN, UK</div><div class="fm-affl" id="aws363-AFF4" lang="en">4 Division of Neuro-immunology, Department of Neurology, Yale School of Medicine, New Haven, CT, USA</div><div id="aws363-COR1">Correspondence to: Sergio E. Baranzini, PhD, Department of Neurology, School of Medicine, University of California, San Francisco, 675 Nelson Rising Lane, Suite 215, San Francisco, CA 94158, USA E-mail: <a href="mailto:dev@null" data-email="ude.fscu.lgc@narabes" class="oemail">ude.fscu.lgc@narabes</a></div><div id="aws363-COR2">Correspondence may also be addressed to: Daniel Pelletier, MD, Division of Neuro-immunology, Department of Neurology, Yale School of Medicine, PO Box 208018, New Haven, CT 06520-8018, USA. E-mail: <a href="mailto:dev@null" data-email="ude.elay@reitellep.leinad" class="oemail">ude.elay@reitellep.leinad</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139909796136112_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Jan 4; Revised 2012 Nov 13; Accepted 2012 Nov 15.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139909796136112_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author (2013). Published by Oxford University Press on behalf of the Guarantors of Brain. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3613709/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3613709/bin/supp_136_4_1012__index.html" data-ga-action="click_feat_suppl">supp_136_4_1012__index.html</a><span style="color:gray"> (2.4K)</span></div><div class="small guid">GUID:&nbsp;62CC629A-9C07-49FD-AD92-CEA43D988A56</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File004.docx" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File004.docx</a><span style="color:gray"> (19K)</span></div><div class="small guid">GUID:&nbsp;666CAEC1-88A1-406C-A866-1AD1ED219518</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File011.jpg" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File011.jpg</a><span style="color:gray"> (3.8M)</span></div><div class="small guid">GUID:&nbsp;33883CAD-D40C-4795-98AB-CFF4CFFC93B5</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File012.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File012.tif</a><span style="color:gray"> (460K)</span></div><div class="small guid">GUID:&nbsp;5E1CC435-B827-4CDF-8DF2-F15E6DCAE1C5</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File013.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File013.tif</a><span style="color:gray"> (2.2M)</span></div><div class="small guid">GUID:&nbsp;992F21C3-4BEE-48D1-BE92-EDF74C1B0619</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File014.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File014.tif</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;1A0A9DB4-B7B6-40A5-A428-C74AF4860306</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File015.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File015.tif</a><span style="color:gray"> (457K)</span></div><div class="small guid">GUID:&nbsp;5E4285E6-647A-4848-9083-6DB76D2100D1</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File016.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File016.tif</a><span style="color:gray"> (1.6M)</span></div><div class="small guid">GUID:&nbsp;6F24865B-E04B-4474-B550-F9FFB52EABBB</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File017.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File017.tif</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;D8E492F3-EE1A-42B5-B7C0-643EE0083221</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File018.tif" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File018.tif</a><span style="color:gray"> (600K)</span></div><div class="small guid">GUID:&nbsp;2C96D470-7937-4FD5-8058-3DBB08E1BD3E</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File019.doc" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File019.doc</a><span style="color:gray"> (49K)</span></div><div class="small guid">GUID:&nbsp;A449A2D1-4290-4832-92A7-0321BD8A5678</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File020.doc" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File020.doc</a><span style="color:gray"> (44K)</span></div><div class="small guid">GUID:&nbsp;13945865-F8DB-40B9-9729-13A8C657599A</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File021.doc" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File021.doc</a><span style="color:gray"> (137K)</span></div><div class="small guid">GUID:&nbsp;6F435791-50B9-44E9-8C65-55834EE5D4A8</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File022.docx" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File022.docx</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;DA691E81-5D99-41BF-8165-1EDCABB72733</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File023.docx" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File023.docx</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;5F5979C6-AFD4-484D-863C-5F27C9281402</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File024.docx" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File024.docx</a><span style="color:gray"> (48K)</span></div><div class="small guid">GUID:&nbsp;8E23CC56-2D49-47D0-8A81-C97163D52964</div>
<div><a href="/pmc/articles/PMC3613709/bin/supp_aws363_brain-2012-00024-File025.docx" data-ga-action="click_feat_suppl">supp_aws363_brain-2012-00024-File025.docx</a><span style="color:gray"> (17K)</span></div><div class="small guid">GUID:&nbsp;5D5E7D67-628E-4CD0-96E5-108C03023821</div>
</div></dd></dl></div><div id="idm139909796192752" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139909796192752title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Brain magnetic resonance imaging is widely used as a diagnostic and monitoring tool in multiple sclerosis and provides a non-invasive, sensitive and reproducible way to track the disease. Topological characteristics relating to the distribution and shape of lesions are recognized as important neuroradiological markers in the diagnosis of multiple sclerosis, although these have been much less well characterized quantitatively than have traditional measures such as T<sub>2</sub> hyperintense or T<sub>1</sub> hypointense lesion volumes. Here, we used voxel-level 3 T magnetic resonance imaging T<sub>1</sub>-weighted scans to reconstruct the 3D topology of lesions in 284 subjects with multiple sclerosis and tested whether this is a heritable phenotype. To this end, we extracted the genotypes from a published genome-wide association study on these same individuals and searched for genetic associations with lesion load, shape and topological distribution. Lesion probability maps were created to identify frequently affected areas and to assess the overall distribution of T<sub>1</sub> lesions in the subject population as a whole. We then developed an original algorithm to cluster adjacent lesional voxels (cluxels) in each subject and tested whether cluxel topology was significantly associated with any single-nucleotide polymorphism in our data set. To focus on patterns of lesion distribution, we computed the first 10 principal components. Although principal component 1 correlated with lesion load, none of the remaining orthogonal components correlated with any other known variable. We then conducted genome-wide association studies on each of these and found 31 significant associations (false discovery rate &lt;0.01) with principal component 8, which represents a mode of variation of lesion topology in the population. The majority of the loci can be linked to genes related to immune cell function and to myelin and neural growth; some (<em>SYK, MYT1L, TRAPPC9, SLITKR6</em> and <em>RIC3</em>) have been previously associated with the distribution of white matter lesions in multiple sclerosis. Finally, we used a bioinformatics approach to identify a network of 48 interacting proteins showing genetic associations (<em>P</em> &lt; 0.01) with cluxel topology in multiple sclerosis. This network also contains proteins expressed in immune cells and is enriched in molecules expressed in the central nervous system that contribute to neural development and regeneration. Our results show how quantitative traits derived from brain magnetic resonance images of patients with multiple sclerosis can be used as dependent variables in a genome-wide association study. With the widespread availability of powerful computing and the availability of genotyped populations, integration of imaging and genetic data sets is likely to become a mainstream tool for understanding the complex biological processes of multiple sclerosis and other brain disorders.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">voxel-wise, GWAS, multiple sclerosis</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">The success of genome-wide association studies (GWAS) in identifying common variants associated with susceptibility to complex genetic disorders has fuelled their application to assess the heritability of a variety of quantitative traits. Genetic associations with secondary or intermediate phenotypes have been reported, and GWAS of human height (<a href="#aws363-B27" rid="aws363-B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735718">Lango Allen <em>et al.</em>, 2010</a>), eye colour (<a href="#aws363-B16" rid="aws363-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735735">Eriksson <em>et al.</em>, 2010</a>), hair colour, freckling (<a href="#aws363-B56" rid="aws363-B56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735752">Sulem <em>et al.</em>, 2008</a>; <a href="#aws363-B16" rid="aws363-B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735773">Eriksson <em>et al.</em>, 2010</a>), digit length ratio (<a href="#aws363-B32" rid="aws363-B32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735765">Medland <em>et al.</em>, 2010</a>), leisure time (<a href="#aws363-B14" rid="aws363-B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735731">De Moor <em>et al.</em>, 2009</a>) and tanning (<a href="#aws363-B36" rid="aws363-B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735744">Nan <em>et al.</em>, 2009</a>) are examples of this approach. The development of novel methods and technologies to quantify RNA, metabolite and protein concentrations in biological fluids, and electrical properties or morphological changes in specific tissues (e.g. heart, brain) have also facilitated integration of these physiological traits with genetic variation (<a href="#aws363-B13" rid="aws363-B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735723">Cheung and Spielman, 2002</a>; <a href="#aws363-B22" rid="aws363-B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735722">Hicks <em>et al.</em>, 2009</a>; <a href="#aws363-B39" rid="aws363-B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735753">Newton-Cheh <em>et al.</em>, 2009</a>; <a href="#aws363-B57" rid="aws363-B57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735732">Teslovich <em>et al.</em>, 2010</a>).</p><p id="__p3">Widespread availability of brain and spinal cord MRI has revolutionized the understanding and, equally important, the diagnosis and management of multiple sclerosis, the most common cause of acquired neurological dysfunction arising during early and mid-adulthood. MRI is highly sensitive in detecting white matter hyperintense/hypointense lesions (plaques) associated with multiple sclerosis neuropathology. Furthermore, anatomical location of an injury is likely to explain, at least in part, the extent and type of neurological dysfunction experienced by a patient. In a large cross-sectional study, <a href="#aws363-B12" rid="aws363-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735785">Charil <em>et al.</em> (2003)</a> elegantly showed an example of this paradigm by demonstrating the close relationship between site of lesions and the type of impairment in subjects with relapsing-remitting multiple sclerosis.</p><p id="__p4">In an early successful integration of MRI-derived phenotypes and genetic information, <a href="#aws363-B40" rid="aws363-B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735746">Okuda <em>et al.</em> (2009)</a> showed that patients with multiple sclerosis carrying the susceptibility allele HLA-DRB1*15:01 display a higher volume of brain lesions than non-carriers. This observation corroborated an earlier study in optic neuritis (<a href="#aws363-B19" rid="aws363-B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735751">Hauser <em>et al.</em>, 2000</a>) and was later confirmed in an independent population (<a href="#aws363-B23" rid="aws363-B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735764">Horakova <em>et al.</em>, 2011</a>). These findings provided evidence that visible lesions in multiple sclerosis may be, at least in part, genetically determined. In a more recent study, we correlated genome-wide genetic variation with brain glutamate assessed <em>in vivo</em> using <sup>1</sup>H magnetic resonance spectroscopy imaging (<a href="#aws363-B4" rid="aws363-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735767">Baranzini <em>et al.</em>, 2010</a>). Using a protein interaction network-based pathway approach, we were able to identify associations in several genes potentially affecting the function of receptors, accessory molecules, transporters and transduction signalling of glutamate.</p><p id="__p5">In a recent report, voxel-level volume differences were used as the phenotype in a GWAS of 731 elderly subjects (<a href="#aws363-B21" rid="aws363-B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735747">Hibar <em>et al.</em>, 2011</a>). Another study from the same group successfully mapped the 3D profile of temporal lobe volume differences from 742 brain MRI scans of patients with Alzheimer’s disease and healthy subjects to a single-nucleotide polymorphism (SNP) in <em>GRIN2B</em>, an ionotropic glutamate receptor. More recently, an exploratory study looking at the correlation of brain lesion distribution in 208 patients with multiple sclerosis with 69 candidate SNPs suggested genotypic influences on spatial lesion distribution (<a href="#aws363-B53" rid="aws363-B53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735741">Sombekke <em>et al.</em>, 2011</a>).</p><p id="__p6" class="p p-last">Here, we tested whether genetic variation is associated with multiple sclerosis lesion topology by a GWAS. To test this hypothesis, we analysed the distribution of multiple sclerosis lesions (captured at voxel resolution of 1 mm<sup>3</sup>) and used that measure as a trait in a GWAS. This kind of approach represents a new trend in genomics, in which more accurate and quantitative tissue phenotypes are used to maximize the power of finding biologically meaningful genetic associations.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Subjects and samples</h3><p id="__p7" class="p p-first-last">The study included 484 subjects of northern European ancestry as described previously (<a href="#aws363-B5" rid="aws363-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_449564476">Baranzini <em>et al.</em>, 2009<em>b</em></a>). Basic demographic and clinical characteristics of the subjects are shown in <a href="/pmc/articles/PMC3613709/table/aws363-T1/" target="table" class="fig-table-link figpopup" rid-figpopup="aws363-T1" rid-ob="ob-aws363-T1" co-legend-rid=""><span>Table 1</span></a>. This information was obtained by means of a longitudinal, prospective, observational on-going multiple sclerosis study at the UCSF Multiple Sclerosis Center where data were acquired according to well-established uniform protocols, stored and quality checked in an integrated computerized database. Although at the time of the study most patients had a relapsing remitting course (<em>n</em> = 343, 71%), our cohort also included individuals with clinically isolated syndrome (<em>n</em> = 76, 16%), secondary progressive (<em>n</em> = 45, 9%) and primary progressive disease (<em>n</em> = 20, 4%) (<a href="#aws363-B31" rid="aws363-B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735761">McDonald <em>et al.</em>, 2001</a>). Clinically isolated syndrome was defined as the first well-established neurological event lasting &gt;48 h, involving optic nerve, spinal cord, brainstem or cerebellum. In patients with clinically isolated syndrome, the presence of two or more hyperintense lesions on a T<sub>2</sub>-weighted MRI sequence was also required for enrolment into the study. Secondary progressive multiple sclerosis was defined by 6 months of worsening neurological disability not explained by clinical relapse. Primary progressive multiple sclerosis was defined both by progressive clinical worsening for &gt;12 months from symptom onset without any relapses, and abnormal CSF as defined by the presence of two or more oligoclonal bands or an elevated immunoglobulin G index. Individuals were excluded if they had experienced a clinical relapse or received treatment with glucocorticosteroids within the previous month of enrolment, as these conditions would have posed significant confounders to our study. The concomitant use of disease-modifying therapies for multiple sclerosis was permitted for inclusion in this study. For all subjects, the expanded disability status scale and multiple sclerosis severity score scores were assessed, and baseline brain MRI scans were performed within 2 weeks of entry into the study. Age of onset was defined as the first episode of focal neurological dysfunction suggestive of CNS demyelinating disease. To reduce the heterogeneity of our cohort, only patients with clinically isolated syndrome and relapsing-remitting multiple sclerosis who had a disease duration of &lt;10 years and age of onset &gt;20 years (<em>n</em> = 284) were studied further. The University of California, San Francisco institutional review board approved the study, and informed consent was obtained from all subjects before participation.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="aws363-T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Cohort characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909716790592" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead align="left"><tr><th rowspan="1" colspan="1">Variable</th><th rowspan="1" colspan="1" align="center">Value</th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">Cohort size (<em>n</em>)</td><td rowspan="1" colspan="1">484</td></tr><tr><td rowspan="1" colspan="1">Age<sup>a</sup> (years), p50 (p25–p75)</td><td rowspan="1" colspan="1">42 (35–50)</td></tr><tr><td rowspan="4" colspan="1">Disease course, <em>n</em> (%)</td><td rowspan="1" colspan="1">CIS: 76 (15.7)</td></tr><tr><td rowspan="1" colspan="1">RRMS: 343 (70.9)</td></tr><tr><td rowspan="1" colspan="1">SPMS: 45 (9.3)</td></tr><tr><td rowspan="1" colspan="1">PPMS: 20 (4.1)</td></tr><tr><td rowspan="2" colspan="1">Gender, <em>n</em> (%)</td><td rowspan="1" colspan="1">Female: 332 (68.6)</td></tr><tr><td rowspan="1" colspan="1">Male: 152 (31.4)</td></tr><tr><td rowspan="3" colspan="1">HLA-DRB1*15:01 dose, <em>n</em> (%)</td><td rowspan="1" colspan="1">0:261 (53.93)</td></tr><tr><td rowspan="1" colspan="1">1:188 (38.84)</td></tr><tr><td rowspan="1" colspan="1">2:35 (7.23)</td></tr><tr><td rowspan="1" colspan="1">Age of onset<sup>a</sup> (years), p50 (p25–p75)</td><td rowspan="1" colspan="1">33 (27.0–39.5)</td></tr><tr><td rowspan="1" colspan="1">MSSS<sup>a</sup>, p50 (p25–p75)</td><td rowspan="1" colspan="1">2.44 (0.91–4.33)</td></tr><tr><td rowspan="1" colspan="1">EDSS<sup>a</sup>, p50 (p25–p75)</td><td rowspan="1" colspan="1">1.5 (1.0–3.0)</td></tr><tr><td rowspan="1" colspan="1">Disease duration<sup>a</sup> (years), p50 (p25–p75)</td><td rowspan="1" colspan="1">5.85 (1.68–12.9)</td></tr><tr><td rowspan="1" colspan="1">Lesion volume<sup>a</sup> (mm<sup>3</sup>), p50 (p25–p75)</td><td rowspan="1" colspan="1">2013 (711–4340)</td></tr><tr><td rowspan="1" colspan="1">Patients on disease-modifying therapy, <em>n</em> (%)</td><td rowspan="1" colspan="1">292 (60)</td></tr></tbody></table></div><div id="largeobj_idm139909716790592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3613709/table/aws363-T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="aws363-TF1"><p id="__p9" class="p p-first-last">a p50 = median; p25 = 1st quartile; p75 = 3rd quartile.</p></div><div id="aws363-TF2"><p id="__p10" class="p p-first-last">CIS = clinically isolated syndrome; EDSS = expanded disability status scale; MSSS = multiple sclerosis severity score; PPMS = primary progressive multiple sclerosis; RRMS = relapsing-remitting multiple sclerosis; SPMS = secondary progressive multiple sclerosis.</p></div></div></div></div><div id="__sec4" class="sec"><h3 id="__sec4title">Imaging data</h3><p id="__p11" class="p p-first">All brain MRI data were derived from high-resolution images acquired on a single 3 T GE Excite scanner (GE Healthcare Technologies) equipped with a phase-array eight-channel coil using a 3D T<sub>1</sub>-weighted inversion recovery spoiled gradient-recalled echo (IRSPGR) sequence yielding a 1-mm<sup>3</sup> isometric voxel size (echo time/repetition time/inversion time = 2/7/400 ms, flip angle = 15°, number of excitations = 1, 180 slices).</p><p id="__p12" class="p p-last">Hemispheric, brainstem and cerebellar white matter lesions were segmented out directly from the high-resolution T<sub>1</sub>-weighted 3D-IRSPGR images based on a semi-automated pixel intensity threshold with manual editing, using in-house software, and T<sub>1</sub> lesion masks were created as reported previously (<a href="#aws363-B10" rid="aws363-B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735720">Blum <em>et al.</em>, 2002</a>; <a href="#aws363-B35" rid="aws363-B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735772">Mowry <em>et al.</em>, 2009</a>).</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">Lesion probability maps</h3><p id="__p13" class="p p-first-last">To create lesion probability maps, lesions were first segmented on each multiple sclerosis subject 3D-IRSPGR scan, inpainted (<a href="#aws363-B48" rid="aws363-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735757">Sdikaand Pelletier, 2009</a>) and non-linearly registered (<a href="#aws363-B47" rid="aws363-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735782">Sdika, 2008</a>) using a healthy subject (female, 42 years old) scan as reference. The lesional map of each subject was then mapped onto the reference scan using the transformation found by the previous step. The registered lesion maps were then analysed on a voxel-by-voxel basis and voxels with <em>P</em>-lesion &gt;0 were considered lesional.</p></div><div id="__sec6" class="sec"><h3 id="__sec6title">Identification of clusters of lesional voxels</h3><p id="__p14" class="p p-first-last">With the goal of accurately representing the 3D shapes of lesions in each subject, lesional voxels that shared a common face (6), edge (12) or vertex (8) in 3D space (26 in total) were grouped into clusters. To accomplish this, we developed an algorithm that starts by considering any lesional voxel and evaluates whether each of its 26 neighbours in 3D is also a lesional voxel, and if so, it merges them. The algorithm continues to aggregate adjacent lesional voxels until the resulting cluster is only surrounded by voxels devoid of lesions. We use the term ‘cluxel’ to describe each such cluster of lesional voxels. For each subject, cluxel topology was defined as the 3D shape of a lesion resulting from the aggregation of neighbouring lesional voxels into cluxels. Analysis of neighbouring voxels was performed in the R statistical package using a custom algorithm (code available on request).</p></div><div id="__sec7" class="sec"><h3 id="__sec7title">Genetic data</h3><p id="__p15" class="p p-first-last">All 484 subjects with multiple sclerosis were typed for 553 139 SNPs using the Illumina 550k platform with &lt; 2% of genotyping failure rate per sample (<a href="#aws363-B5" rid="aws363-B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_449564477">Baranzini <em>et al.</em>, 2009<em>b</em></a>). SNPs (7043) were removed from the data set for missing genotypes in &gt; 2.5% of individuals and 50 340 SNPs were removed for having a minor allele frequency of &lt; 5%. Finally, 443 SNPs that departed from Hardy–Weinberg equilibrium (<em>P</em> &lt; 0.001) were removed. After these quality control steps, 495 313 (99.9%) markers remained. However, to limit the number of association tests, we also eliminated SNPs that were in moderate linkage disequilibrium (<em>r</em><sup>2</sup> &gt; 0.5), and only 208 975 were considered for further analysis. To control for multiple hypothesis testing, the false discovery rate (FDR) correction (<a href="#aws363-B8" rid="aws363-B8" class=" bibr popnode">Benjamini and Hochberg, 1995</a>) was applied to the results of each GWAS. An analysis with all SNPs that passed quality control was also performed, and results are available in <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 3</a>. Clusterplots of significant SNPs are provided in <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 3</a> and regional association plots in <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 4</a>. Distributions of the phenotype with each significant genotype are provided in <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 5</a>.</p></div><div id="__sec8" class="sec"><h3 id="__sec8title">Statistical analysis</h3><p id="__p16" class="p p-first">Of the 484 subjects with available genotypes, 284 fitted our stringent criteria for analysis. GWAS analyses were completed in this subset using PLINK (version 1.06) (<a href="#aws363-B44" rid="aws363-B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735784">Purcell <em>et al.</em>, 2007</a>), and all other tests were performed in the R statistical package (version 2.9). Binary traits were analysed using logistic regression. Quantitative variables (T1LL and T2LL) that did not distribute normally were log transformed before analysis using a linear model. To reduce the heterogeneity of our cohort, we used a combination of subject stratification and covariates in the linear model. We first stratified subjects by disease type (we included only clinically isolated syndrome and relapsing-remitting multiple sclerosis) and then adjusted by disease duration. We tested other variables (gender, disease duration, age, age of onset, expanded disability status scale, immunotherapy, etc.) and found no significant contribution in the measured phenotype (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 2</a>). Given the negligible influence of these variables on the measured outcome, we decided not to adjust for those, to preserve power. Reported <em>P</em>-values were adjusted by the FDR method (<a href="#aws363-B8" rid="aws363-B8" class=" bibr popnode">Benjamini and Hochberg, 1995</a>).</p><p id="__p17" class="p p-last">We performed a simulation to assess the probability of observing a <em>P</em>-value &lt; 1 observed in the data, given an effect size equal to our estimated effect and variance equal to the sample variance of principal component 8 (PC8). In other words, this test shows our ability to find SNPs with such effect sizes given the number of samples and SNP allele frequencies found in our study. Specifically, we simulated the genotypes of 284 individuals to match the maximum autocorrelation factor of each of our hit SNPs. We then simulated a continuous phenotype based on the hit SNP’s effect size and added noise from a normal distribution with mean 0 and variance equal to that of PC8. Finally, we tested for associations between the simulated genotypes and phenotypes. This process was repeated 100 000 times and the distribution of obtained <em>P</em>-values is shown in <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 6</a>.</p></div><div id="__sec9" class="sec"><h3 id="__sec9title">Principal component analysis</h3><p id="__p18" class="p p-first-last">To reduce the dimensionality of the voxel-based analysis, we used principal component analysis. This method relies on orthogonal transformation to convert a set of possibly correlated voxel-based measures into a set of values of linearly uncorrelated variables, the principal components. The first principal component explains the largest amount of variance (i.e. accounts for as much of the variability in the data as possible), and each subsequent component explains the highest remaining variance under the constraint that it is orthogonal (i.e. uncorrelated with) to the preceding components.</p></div><div id="__sec10" class="sec sec-last"><h3 id="__sec10title">Module (sub-network) analysis</h3><p id="__p19" class="p p-first-last">Systematic module searches on a highly curated protein interaction network were conducted as described (<a href="#aws363-B4" rid="aws363-B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735783">Baranzini <em>et al.</em>, 2010</a>). Briefly, each gene product in the network was assigned a number corresponding to the <em>P</em>-value of the most strongly associated SNP for that gene with the trait (only <em>P</em>-values &lt; 0.05 were considered). Next, the Cytoscape (<a href="http://www.cytoscape.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.cytoscape.org</a>) plugin jActive modules (<a href="#aws363-B24" rid="aws363-B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735776">Ideker <em>et al.</em>, 2002</a>) were used to identify groups of interacting gene products that were also associated with PC8 by GWAS. jActive modules convert <em>P</em>-values into <em>z</em>-scores and use a greedy algorithm to grow a sub-network (or module) from a random seed node by sequentially incorporating its neighbours in the protein interaction network. The algorithm then returns the smallest possible module that includes gene products with the most significant associations. A significance (<em>z</em>) score is assigned to each reported sub-network after evaluation of 10 000 random networks of similar size. Only modules with a score &gt; 3 and of size &gt; 5 were considered.</p></div></div><div id="__sec11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Results</h2><div id="__sec12" class="sec sec-first"><h3 id="__sec12title">Lesion probability maps</h3><p id="__p20" class="p p-first-last">We analysed the presence and distribution of white matter lesions in 484 patients with multiple sclerosis at 1 mm<sup>3</sup> voxel resolution. First, the probability of a lesion to be present in each of the 67 000 voxels was calculated across all patients to build a composite lesion probability map (<a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1" co-legend-rid="lgnd_aws363-F1"><span>Fig. 1</span></a>A). As expected, the lesion probability map clearly shows that the probability of a lesion is maximal near the periventricular areas and decays markedly in other brain areas. Following our earlier observation that subjects carrying the HLA-DRB1*15:01 allele consistently show higher lesion volume than those carrying other alleles (<a href="#aws363-B40" rid="aws363-B40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735754">Okuda <em>et al.</em>, 2009</a>), we next evaluated the lesion probability map stratified by the presence of this major susceptibility allele. We observed a moderate but significantly different distribution of lesional voxels in the 15:01 negative (<a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1" co-legend-rid="lgnd_aws363-F1"><span>Fig. 1</span></a>B) versus the 15:01 positive group (<a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1" co-legend-rid="lgnd_aws363-F1"><span>Fig. 1</span></a>C) (<em>P</em> = 2.3 × 10<sup>−</sup><sup>4</sup>, matched Wilcoxon test). Interestingly, we also observed that more voxels with high lesion probability were found in males (<a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1" co-legend-rid="lgnd_aws363-F1"><span>Fig. 1</span></a>E) compared with females (<a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1" co-legend-rid="lgnd_aws363-F1"><span>Fig. 1</span></a>F) (<em>P</em> = 2.2 × 10<sup>−</sup><sup>16</sup>, matched Wilcoxon test). Although this finding could suggest a gender-dependent effect on the total number of lesional voxels and/or their spatial distribution, it is also possible that the observed differences are caused by a larger white matter volume typical in males.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F1" co-legend-rid="lgnd_aws363-F1"><a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909716746144" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f1p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f1p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f1p.jpg"></a></div><div id="largeobj_idm139909716746144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F1"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F1/" target="figure" rid-figpopup="aws363-F1" rid-ob="ob-aws363-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p21">Lesion probability maps. (<strong>A</strong>) Lesion probability map over all individuals shows the highest probability of a lesion to be around the periventricular areas. Individuals carrying at least one 15:01 allele (<strong>B</strong>) display a significantly different lesion distribution than 15:01 negative (<strong>C</strong>) individuals. The colour of each voxel is proportional to the probability of there being a lesion. (red = high, yellow = medium, green = low). (<strong>D</strong>) The difference between panels <strong>A</strong> and <strong>B</strong>. In this lesion probability map, red voxels indicate a higher probability of a lesion for 15:01 positive subjects, whereas blue voxels indicate lower probability of a lesion for these subjects. Males (<strong>E</strong>) have different distribution of probabilities than females (<strong>F</strong>). (<strong>G</strong>) The difference between panels <strong>E</strong> and <strong>F</strong>. In this lesion probability map, red voxels indicate higher probability of a lesion for males, whereas blue voxels indicate lower probability of a lesion for males.</p></div></div></div></div><div id="__sec13" class="sec"><h3 id="__sec13title">Distribution of lesional voxels</h3><p id="__p22" class="p p-first-last">As shown in the lesion probability map, the probability for any given voxel to contain a lesion across all patients is exceedingly small (<a href="/pmc/articles/PMC3613709/figure/aws363-F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F2" rid-ob="ob-aws363-F2" co-legend-rid="lgnd_aws363-F2"><span>Fig. 2</span></a>). Similarly, in any given subject, only a small fraction of voxels carry a lesion (lesional voxel). This limited variance in the distribution of the binary trait (lesion versus no lesion) prevented us from conducting a voxel-based genetic association because most voxels contained no lesions (uninformative). We thus considered an alternative strategy in which a combination of a threshold (T) number of lesional voxels (depicted by coloured lines in <a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3" co-legend-rid="lgnd_aws363-F3"><span>Fig. 3</span></a>) and the <em>n</em>-most lesional voxels across the population (<em>x</em> axis, <a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3" co-legend-rid="lgnd_aws363-F3"><span>Fig. 3</span></a>) defines two subgroups of individuals for analysis. For example, the proportion of patients displaying lesions in at least 13 of the 100 most lesional voxels (T) is ∼50%, thus maximizing the power of finding a genetic association to this trait. Because multiple combinations can result in balanced proportion of observations, we tested this stratification approach over three combinations of T and Z parameters (indicated by the arrows in <a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3" co-legend-rid="lgnd_aws363-F3"><span>Fig. 3</span></a>).

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F2" co-legend-rid="lgnd_aws363-F2"><a href="/pmc/articles/PMC3613709/figure/aws363-F2/" target="figure" rid-figpopup="aws363-F2" rid-ob="ob-aws363-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909722862832" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F2/" target="figure" rid-figpopup="aws363-F2" rid-ob="ob-aws363-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f2p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f2p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f2p.jpg"></a></div><div id="largeobj_idm139909722862832" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F2"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F2/" target="figure" rid-figpopup="aws363-F2" rid-ob="ob-aws363-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p23">Lesional voxels are rare. (<strong>A</strong>) The proportion of voxels that contain a lesion in the given number of patients is exponentially decaying. The inset shows the distribution of voxels that contain lesions in 0–25 patients. (<strong>B</strong>) The proportion of patients with the given number of lesional voxels is also exponentially decaying. The inset shows the number of subjects with lesions in the top 2000 lesional voxels.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F3" co-legend-rid="lgnd_aws363-F3"><a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909720308560" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f3p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f3p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f3p.jpg"></a></div><div id="largeobj_idm139909720308560" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F3"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p24">Proportion of subjects with lesions in top lesional voxels. Each curve shows the proportion of subjects with ≥T lesional voxels versus the number of voxels considered (where the voxels considered are the 200 voxels that contain the most lesions across patients). The colour of each curve represents ‘T’, the threshold minimum number of lesions among the top lesional voxels. The dotted horizontal line indicates combinations of T lesions in top × lesional voxels resulting in a more or less balanced ‘cases’–‘controls’ design for a GWAS. The arrows depict the three balanced combinations chosen to run GWAS. For example, the first arrow indicates that 50% of cases have at least one lesion in any of the top 11 lesional voxels. The second arrow indicates that 50% of the cases have at least 13 lesions in the top 100 lesional voxels. Similarly, the third arrow indicates that 50% of the patients have at least 20 lesions in the top 160 lesional voxels. It can also be seen from this chart that ∼80% of cases have at least one lesion (T = 1, red line) in the top 100 lesional voxels and ∼90% in the top 200 lesional voxels.</p></div></div></div></div><div id="__sec14" class="sec sec-last"><h3 id="__sec14title">Genome-wide association study on lesion distribution</h3><p id="__p25" class="p p-first">We performed a GWAS for each of three combinations of parameters resulting in a balanced proportion of patients (low, mid and high number of lesional voxels, denoted by arrows in <a href="/pmc/articles/PMC3613709/figure/aws363-F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F3" rid-ob="ob-aws363-F3" co-legend-rid="lgnd_aws363-F3"><span>Fig. 3</span></a>). Although several genetic markers were nominally significant in these analyses, none surpassed the FDR correction for multiple comparisons.</p><p id="__p26">As lesional voxels can adopt a variety of topological arrangements and these may be specific to each subject, we developed an algorithm that allowed merging of neighbouring lesional voxels into clusters (cluxels) to identify and characterize these spatial patterns. Using this strategy, the total number of quantitative variables for GWAS (number of cluxels per patient) is reduced by several orders of magnitude relative to a direct, voxel-based analysis and can also be used as a dichotomous trait (e.g. dense versus sparse organization) (<a href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4" co-legend-rid="lgnd_aws363-F4"><span>Fig. 4</span></a>). In addition, this new trait is independent of total lesion volume, as two patients with similar number of lesional voxels can have a markedly different cluxel topology, and the correlation between the number of cluxels and lesion volume is low (<a href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4" co-legend-rid="lgnd_aws363-F4"><span>Fig. 4</span></a>C). We computed the distribution of the number of cluxels over all subjects and chose an arbitrary cut-off (<em>n</em> = 35) to define a dichotomous phenotype (high/low number of cluxels) for a GWAS. (<a href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4" co-legend-rid="lgnd_aws363-F4"><span>Fig. 4</span></a>D). The pairwise minimum distance between cluxel edges and the average size of cluxels were also found to be highly variable across individuals and independent of gender and lesion volume. However, a GWAS on the number of cluxels and on the average minimal distance between cluxels did not identify markers with genome-wide (FDR-corrected) significance, and a GWAS on the average size of cluxels resulted in borderline significant results (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Tables 1</a> and <a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">2</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F4" co-legend-rid="lgnd_aws363-F4"><a href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909722191296" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f4p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f4p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f4p.jpg"></a></div><div id="largeobj_idm139909722191296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F4"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F4/" target="figure" rid-figpopup="aws363-F4" rid-ob="ob-aws363-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p27">Cluxel topology as a trait. 3D representation of neighbouring lesional voxels (cluxels) in two patients with similar lesion volume. (<strong>A</strong>) The patient depicted on the left has 4737 lesional voxels arranged in 13 cluxels. (<strong>B</strong>) Despite having a similar lesion volume (4984 mm<sup>3</sup>), the lesional voxels in the patient depicted on the right are arranged in 79 cluxels. These patient-specific differences form the basis for a newly defined trait in multiple sclerosis. (<strong>C</strong>) The black dotted line shows that the correlation between lesion volume and number of cluxels across all multiple sclerosis subjects is low (R = 0.344). (<strong>D</strong>) The frequency distribution of the number of cluxels per subject.</p></div></div></div><p id="__p28">Although the arrangement of cluxels may be markedly different across individuals, just measuring their number, size or distance from each other may not accurately capture actual cluxel topology. We then hypothesized that if the variance related to lesion volume is accounted for, the remaining variance would be enriched in cluxel topology. We thus used principal component analysis to decompose the total variance in lesional voxels and computed the correlation of each component with other relevant imaging and clinical variables (e.g. total lesion volume, expanded disability status scale, age of onset, T<sub>1</sub> lesion load, T<sub>2</sub> lesion load, total number of lesional voxels, age, disease duration, gender, HLA-DRB1*15:01, treatments and multiple sclerosis type). Almost 10% of the total variance was explained by the first component (PC1), which in turn was highly correlated with lesion volume (<em>R</em><sup>2 </sup>= 88%) (<a href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5" co-legend-rid="lgnd_aws363-F5"><span>Fig. 5</span></a>A and B). Given the wide distribution of lesions observed in patients with multiple sclerosis, this finding is not surprising. However, it allowed us to identify the proportion of variance explained by lesion volume itself and then focus on the remaining (orthogonal) variance, in which lesion topology (and not load) is now more represented.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F5" co-legend-rid="lgnd_aws363-F5"><a href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909791813664" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f5p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f5p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f5p.jpg"></a></div><div id="largeobj_idm139909791813664" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F5"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p29">Principal component analysis on the distribution of lesions across all subjects. (<strong>A</strong>) Amount of variance explained by each of the top 10 principal components. PC1 (PC1, blue bar) explains almost 10% of the variance and is highly correlated with lesion volume (<strong>B</strong>). (<strong>C</strong>) Spatial distribution of voxels with the highest weight in PC1. Each displayed voxel is coloured according to its weight. A ‘hot’ colouring scale (red-orange-yellow) is used for the 2% highest-weighted voxels, and a ‘cold’ colouring scale (purple-blue-cyan) was used for the 2% lowest-weighted voxels. (<strong>D</strong>) Spatial distribution of voxels with the highest weight in PC8, which showed genome-wide (FDR-corrected) association with 31 markers. (<strong>E</strong>) Manhattan plot showing associations with PC8.</p></div></div></div><p id="__p30">We then performed GWAS with each of the remaining components (PC2–PC10) to evaluate genetic associations with orthogonal sources of variance in lesion topology within the multiple sclerosis brain. To control the rate of false discoveries, the location of lesional voxels was permuted across all voxels 100 times, thus keeping the lesional load of each individual intact while removing any spatial structure of lesions across the brain. An enrichment of significant <em>P</em>-values was observed in the original data when compared with the permuted sets (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 8</a>). Specifically, 31 significant associations at FDR-corrected <em>P</em>-value of 0.01 were identified with PC8 (<a href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5" co-legend-rid="lgnd_aws363-F5"><span>Fig. 5</span></a> and <a href="/pmc/articles/PMC3613709/table/aws363-T2/" target="table" class="fig-table-link figpopup" rid-figpopup="aws363-T2" rid-ob="ob-aws363-T2" co-legend-rid=""><span>Table 2</span></a>), suggesting a genetic effect in the variance explained by this component. Although 13 of these associations reach <em>P</em> &lt; 10<sup>−</sup><sup>7</sup>, an average of only 0.64 associations exceeded this threshold in the permuted data set tested with on PC8, and an average of 2.32 associations exceeded this threshold in the permuted data sets with all 10 principal components (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 4</a>). Although the voxels with the highest weight in PC8 are not arranged in a pattern that matches any particular brain structure, the associations showed to be robust to rigorous quality controls (<a href="/pmc/articles/PMC3613709/figure/aws363-F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F5" rid-ob="ob-aws363-F5" co-legend-rid="lgnd_aws363-F5"><span>Fig. 5</span></a>D). None of the remaining principal components yielded any significant associations.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="aws363-T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p31">GWAS with PC8</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909757017088" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead align="left"><tr><th rowspan="1" colspan="1"><strong>Chromosome</strong></th><th rowspan="1" colspan="1"><strong>Position</strong></th><th rowspan="1" colspan="1"><strong>SNP</strong></th><th rowspan="1" colspan="1"><strong>Mapped gene (distance)</strong></th><th rowspan="1" colspan="1"><strong>Beta</strong></th><th rowspan="1" colspan="1"><strong>MAF (%)</strong></th><th rowspan="1" colspan="1"><strong>Raw <em>P</em>-value</strong></th><th rowspan="1" colspan="1"><strong>FDR_BH</strong></th></tr></thead><tbody align="left"><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">174014782</td><td rowspan="1" colspan="1">rs16861690</td><td rowspan="1" colspan="1">CDCA7 (+100 Kb)</td><td rowspan="1" colspan="1">2.203</td><td rowspan="1" colspan="1">9.54</td><td rowspan="1" colspan="1">1.80E−11</td><td rowspan="1" colspan="1">3.77E-06</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">103124572</td><td rowspan="1" colspan="1">rs13410351</td><td rowspan="1" colspan="1">TMEM182 (+400 Kb)</td><td rowspan="1" colspan="1">2.008</td><td rowspan="1" colspan="1">11.3</td><td rowspan="1" colspan="1">6.12E−11</td><td rowspan="1" colspan="1">6.39E-06</td></tr><tr><td rowspan="1" colspan="1"><strong>9</strong></td><td rowspan="1" colspan="1"><strong>92672620</strong></td><td rowspan="1" colspan="1"><strong>rs10119179</strong></td><td rowspan="1" colspan="1"><strong>SYK (0 Kb)</strong></td><td rowspan="1" colspan="1">2.183</td><td rowspan="1" colspan="1">7.12</td><td rowspan="1" colspan="1"><strong>3.65E−09</strong></td><td rowspan="1" colspan="1"><strong>0.000254</strong></td></tr><tr><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">85343994</td><td rowspan="1" colspan="1">rs9602859</td><td rowspan="1" colspan="1">SLITRK6 (−80 Kb)</td><td rowspan="1" colspan="1">1.755</td><td rowspan="1" colspan="1">11.7</td><td rowspan="1" colspan="1">1.04E−08</td><td rowspan="1" colspan="1">0.000341</td></tr><tr><td rowspan="1" colspan="1"><strong>8</strong></td><td rowspan="1" colspan="1"><strong>141380846</strong></td><td rowspan="1" colspan="1"><strong>rs6983731</strong></td><td rowspan="1" colspan="1"><strong>TRAPPC9 (0 Kb)</strong></td><td rowspan="1" colspan="1">1.705</td><td rowspan="1" colspan="1">9.54</td><td rowspan="1" colspan="1"><strong>1.09E−08</strong></td><td rowspan="1" colspan="1"><strong>0.000341</strong></td></tr><tr><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">12046782</td><td rowspan="1" colspan="1">rs169715</td><td rowspan="1" colspan="1">HIVEP1 (−70 Kb)</td><td rowspan="1" colspan="1">1.891</td><td rowspan="1" colspan="1">9.54</td><td rowspan="1" colspan="1">1.14E−08</td><td rowspan="1" colspan="1">0.000341</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">2728526</td><td rowspan="1" colspan="1">rs2053906</td><td rowspan="1" colspan="1">MYT1L (−400 Kb)</td><td rowspan="1" colspan="1">1.778</td><td rowspan="1" colspan="1">9.9</td><td rowspan="1" colspan="1">1.14E−08</td><td rowspan="1" colspan="1">0.000341</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">114836373</td><td rowspan="1" colspan="1">rs10981613</td><td rowspan="1" colspan="1">ZFP37 (10 Kb)</td><td rowspan="1" colspan="1">1.712</td><td rowspan="1" colspan="1">12.01</td><td rowspan="1" colspan="1">1.56E−08</td><td rowspan="1" colspan="1">0.00039</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">8177388</td><td rowspan="1" colspan="1">rs12776126</td><td rowspan="1" colspan="1">GATA3 (20 Kb)</td><td rowspan="1" colspan="1">1.728</td><td rowspan="1" colspan="1">11.66</td><td rowspan="1" colspan="1">1.68E−08</td><td rowspan="1" colspan="1">0.00039</td></tr><tr><td rowspan="1" colspan="1"><strong>1</strong></td><td rowspan="1" colspan="1"><strong>36700319</strong></td><td rowspan="1" colspan="1"><strong>rs7065</strong></td><td rowspan="1" colspan="1"><strong>MRPS15</strong><sup><strong><sup>a</sup></strong></sup><strong> (0 Kb)</strong></td><td rowspan="1" colspan="1">1.574</td><td rowspan="1" colspan="1">10.24</td><td rowspan="1" colspan="1"><strong>3.06E−08</strong></td><td rowspan="1" colspan="1"><strong>0.00064</strong></td></tr><tr><td rowspan="1" colspan="1"><strong>14</strong></td><td rowspan="1" colspan="1"><strong>68189895</strong></td><td rowspan="1" colspan="1"><strong>rs10483818</strong></td><td rowspan="1" colspan="1"><strong>RAD51L1 (0 Kb)</strong></td><td rowspan="1" colspan="1">1.698</td><td rowspan="1" colspan="1">9.9</td><td rowspan="1" colspan="1"><strong>3.62E−08</strong></td><td rowspan="1" colspan="1"><strong>0.000689</strong></td></tr><tr><td rowspan="1" colspan="1"><strong>19</strong></td><td rowspan="1" colspan="1"><strong>61012011</strong></td><td rowspan="1" colspan="1"><strong>rs16986626</strong></td><td rowspan="1" colspan="1"><strong>NLRP11 (0 Kb)</strong></td><td rowspan="1" colspan="1">1.601</td><td rowspan="1" colspan="1">11.7</td><td rowspan="1" colspan="1"><strong>5.72E−08</strong></td><td rowspan="1" colspan="1"><strong>0.000996</strong></td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">81207967</td><td rowspan="1" colspan="1">rs6452434</td><td rowspan="1" colspan="1">ATG10 (100 Kb)</td><td rowspan="1" colspan="1">1.62</td><td rowspan="1" colspan="1">12.01</td><td rowspan="1" colspan="1">9.41E−08</td><td rowspan="1" colspan="1">0.001512</td></tr><tr><td rowspan="1" colspan="1"><strong>12</strong></td><td rowspan="1" colspan="1"><strong>70075768</strong></td><td rowspan="1" colspan="1"><strong>rs10506628</strong></td><td rowspan="1" colspan="1"><strong>TSPAN8 (0 Kb)</strong></td><td rowspan="1" colspan="1"><strong>1.431</strong></td><td rowspan="1" colspan="1">12.37</td><td rowspan="1" colspan="1"><strong>2.82E−07</strong></td><td rowspan="1" colspan="1"><strong>0.004213</strong></td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">13581905</td><td rowspan="1" colspan="1">rs13299116</td><td rowspan="1" colspan="1">FLJ41200 (−150 Kb)</td><td rowspan="1" colspan="1">1.343</td><td rowspan="1" colspan="1">14.89</td><td rowspan="1" colspan="1">3.34E−07</td><td rowspan="1" colspan="1">0.004509</td></tr><tr><td rowspan="1" colspan="1">15</td><td rowspan="1" colspan="1">95430437</td><td rowspan="1" colspan="1">rs11852342</td><td rowspan="1" colspan="1">NR2F2 (900 Kb)</td><td rowspan="1" colspan="1">1.843</td><td rowspan="1" colspan="1">7.77</td><td rowspan="1" colspan="1">3.45E−07</td><td rowspan="1" colspan="1">0.004509</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">43944077</td><td rowspan="1" colspan="1">rs2368524</td><td rowspan="1" colspan="1">LGALS7 (10 Kb)</td><td rowspan="1" colspan="1">1.323</td><td rowspan="1" colspan="1">16.26</td><td rowspan="1" colspan="1">4.77E−07</td><td rowspan="1" colspan="1">0.005863</td></tr><tr><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">91806412</td><td rowspan="1" colspan="1">rs10075974</td><td rowspan="1" colspan="1"><a id="__tag_380286228" class="tag_hotlink" href="/nuccore/AK056485" ref="/nuccore/AK056485"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK056485","term_id":"16551900","term_text":"AK056485"}}</span>AK056485</a> (30 Kb)</td><td rowspan="1" colspan="1">1.486</td><td rowspan="1" colspan="1">11.31</td><td rowspan="1" colspan="1">6.67E−07</td><td rowspan="1" colspan="1">0.007428</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">80588305</td><td rowspan="1" colspan="1">rs13299822</td><td rowspan="1" colspan="1">PSAT1 (500 Kb)</td><td rowspan="1" colspan="1">1.317</td><td rowspan="1" colspan="1">14.49</td><td rowspan="1" colspan="1">6.75E−07</td><td rowspan="1" colspan="1">0.007428</td></tr><tr><td rowspan="1" colspan="1"><strong>11</strong></td><td rowspan="1" colspan="1"><strong>120288578</strong></td><td rowspan="1" colspan="1"><strong>rs17124538</strong></td><td rowspan="1" colspan="1"><strong>GRIK4 (0 Kb)</strong></td><td rowspan="1" colspan="1"><strong>1.439</strong></td><td rowspan="1" colspan="1">10.95</td><td rowspan="1" colspan="1"><strong>7.30E−07</strong></td><td rowspan="1" colspan="1"><strong>0.007628</strong></td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">36028008</td><td rowspan="1" colspan="1">rs10249851</td><td rowspan="1" colspan="1">SEPT7 (−120 Kb)</td><td rowspan="1" colspan="1">1.484</td><td rowspan="1" colspan="1">9.89</td><td rowspan="1" colspan="1">7.89E−07</td><td rowspan="1" colspan="1">0.007807</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">64565272</td><td rowspan="1" colspan="1">rs12132851</td><td rowspan="1" colspan="1">UBE2U (80 Kb)</td><td rowspan="1" colspan="1">1.558</td><td rowspan="1" colspan="1">10.95</td><td rowspan="1" colspan="1">8.23E−07</td><td rowspan="1" colspan="1">0.007807</td></tr><tr><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">246068188</td><td rowspan="1" colspan="1">rs7512555</td><td rowspan="1" colspan="1">OR11L1 (2 Kb)</td><td rowspan="1" colspan="1">1.531</td><td rowspan="1" colspan="1">10.95</td><td rowspan="1" colspan="1">9.31E−07</td><td rowspan="1" colspan="1">0.007807</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">29616136</td><td rowspan="1" colspan="1">rs1930395</td><td rowspan="1" colspan="1">LINGO2 (−800 Kb)</td><td rowspan="1" colspan="1">1.42</td><td rowspan="1" colspan="1">12.02</td><td rowspan="1" colspan="1">9.49E−07</td><td rowspan="1" colspan="1">0.007807</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">151317233</td><td rowspan="1" colspan="1">rs10513360</td><td rowspan="1" colspan="1">PFN2 (−150 Kb)</td><td rowspan="1" colspan="1">1.461</td><td rowspan="1" colspan="1">12.36</td><td rowspan="1" colspan="1">9.50E−07</td><td rowspan="1" colspan="1">0.007807</td></tr><tr><td rowspan="1" colspan="1"><strong>11</strong></td><td rowspan="1" colspan="1"><strong>8140723</strong></td><td rowspan="1" colspan="1"><strong>rs16936464</strong></td><td rowspan="1" colspan="1"><strong>RIC3 (0 Kb)</strong></td><td rowspan="1" colspan="1"><strong>1.548</strong></td><td rowspan="1" colspan="1">10.95</td><td rowspan="1" colspan="1"><strong>9.71E−07</strong></td><td rowspan="1" colspan="1"><strong>0.007807</strong></td></tr><tr><td rowspan="1" colspan="1"><strong>14</strong></td><td rowspan="1" colspan="1"><strong>36397262</strong></td><td rowspan="1" colspan="1"><strong>rs7144374</strong></td><td rowspan="1" colspan="1"><strong>SLC25A21 (0 Kb)</strong></td><td rowspan="1" colspan="1"><strong>1.434</strong></td><td rowspan="1" colspan="1"><strong>12.72</strong></td><td rowspan="1" colspan="1"><strong>1.17E−06</strong></td><td rowspan="1" colspan="1"><strong>0.009081</strong></td></tr><tr><td rowspan="1" colspan="1"><strong>12</strong></td><td rowspan="1" colspan="1"><strong>100625756</strong></td><td rowspan="1" colspan="1"><strong>rs7980436</strong></td><td rowspan="1" colspan="1"><strong>CHPT1 (0 Kb)</strong></td><td rowspan="1" colspan="1"><strong>1.21</strong></td><td rowspan="1" colspan="1">15.9</td><td rowspan="1" colspan="1"><strong>1.32E−06</strong></td><td rowspan="1" colspan="1"><strong>0.009784</strong></td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">126063061</td><td rowspan="1" colspan="1">rs2807064</td><td rowspan="1" colspan="1">OAT (12 Kb)</td><td rowspan="1" colspan="1">1.303</td><td rowspan="1" colspan="1">12.36</td><td rowspan="1" colspan="1">1.42E−06</td><td rowspan="1" colspan="1">0.009784</td></tr><tr><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">75765667</td><td rowspan="1" colspan="1">rs12585734</td><td rowspan="1" colspan="1"><a id="__tag_380286229" class="tag_hotlink" href="/nuccore/AX747676" ref="/nuccore/AX747676"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AX747676","term_id":"32132064","term_text":"AX747676"}}</span>AX747676</a> (300 Kb)</td><td rowspan="1" colspan="1">1.514</td><td rowspan="1" colspan="1">9.9</td><td rowspan="1" colspan="1">1.42E−06</td><td rowspan="1" colspan="1">0.009784</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">83673368</td><td rowspan="1" colspan="1">rs11976275</td><td rowspan="1" colspan="1">SEMA3A (−13 Kb)</td><td rowspan="1" colspan="1">1.453</td><td rowspan="1" colspan="1">12.36</td><td rowspan="1" colspan="1">1.45E−06</td><td rowspan="1" colspan="1">0.009784</td></tr></tbody></table></div><div id="largeobj_idm139909757017088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/table/aws363-T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="aws363-TF3"><p id="__p32" class="p p-first-last">a Denotes a non-synonymous SNP.</p></div><div id="aws363-TF4"><p id="__p33" class="p p-first-last">MAF = maximum autocorrelation factor; bold lettering = SNPs that map within a gene.</p></div></div></div><p id="__p34" class="p p-last">To evaluate the functional relationship among statistically significant associations, we performed a protein interaction network-based pathway analysis. This approach (as described by <a href="#aws363-B3" rid="aws363-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735770">Baranzini <em>et al.</em>, 2009<em>a</em></a>) integrates information of genetic associations with protein interactions to identify groups of physically interacting proteins (sub-networks or modules) that likely work together in a biological pathway. The top-ranked network identified using this approach contained 48 genes (<a href="/pmc/articles/PMC3613709/figure/aws363-F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F6" rid-ob="ob-aws363-F6" co-legend-rid="lgnd_aws363-F6"><span>Fig. 6</span></a>A), most of them expressed in inflammatory cells or in the CNS (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 1</a>), which is encouraging given the suspected involvement of inflammatory and neural pathways in multiple sclerosis pathogenesis. A gene ontology analysis of the elements of this network revealed a significant enrichment in genes associated with immune response and with CNS development and function (<a href="/pmc/articles/PMC3613709/figure/aws363-F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="aws363-F6" rid-ob="ob-aws363-F6" co-legend-rid="lgnd_aws363-F6"><span>Fig. 6</span></a>B). In contrast, the top-scoring network obtained after permutation of <em>P</em>-values among all elements in the protein interaction network did not yield any significant enrichments.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="aws363-F6" co-legend-rid="lgnd_aws363-F6"><a href="/pmc/articles/PMC3613709/figure/aws363-F6/" target="figure" rid-figpopup="aws363-F6" rid-ob="ob-aws363-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139909755215632" class="figure"><a href="/pmc/articles/PMC3613709/figure/aws363-F6/" target="figure" rid-figpopup="aws363-F6" rid-ob="ob-aws363-F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3613709_aws363f6p.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is aws363f6p.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3613709/bin/aws363f6p.jpg"></a></div><div id="largeobj_idm139909755215632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3613709/figure/aws363-F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_aws363-F6"><div><a class="figpopup" href="/pmc/articles/PMC3613709/figure/aws363-F6/" target="figure" rid-figpopup="aws363-F6" rid-ob="ob-aws363-F6">Figure 6</a></div><!--caption a7--><div class="caption"><p id="__p35">Top-scoring sub-network (module). (<strong>A</strong>) Circles and diamond represent proteins, and lines represent physical interactions (green: protein–protein, black: protein–DNA). Gene products with a significant <em>P</em>-value in the GWAS with PC8 are depicted as diamonds. Diamond size is proportional to the –log(10) of the <em>P</em>-value. Genes known to be expressed in the CNS are shown in red. (<strong>B</strong>) Gene ontology analysis. Bars represent gene ontology categories between GO levels 7 and 15. The length of each bar is proportional to the number of genes associated with that term. The percentage of associated genes for each term is indicated inside each bar. Asterisks denote Bonferroni-corrected (Fisher’s exact test) <em>P</em>-values of enrichment for that term (*<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.001). Colours denote gene ontology groups with &gt;50% similarity in their component genes.</p></div></div></div></div></div><div id="__sec15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec15title">Discussion</h2><p id="__p36" class="p p-first">Although genetic association studies have scored notable achievements in the discovery of susceptibility genes in complex diseases, this search has proven more challenging than initially anticipated. The clinical and genetic heterogeneity embedded in these diseases hampers appropriate phenotyping and explains, at least in part, this incomplete success (<a href="#aws363-B58" rid="aws363-B58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735733">van der Sluis <em>et al.</em>, 2010</a>; <a href="#aws363-B26" rid="aws363-B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735724">Kutalik <em>et al.</em>, 2011</a>; <a href="#aws363-B64" rid="aws363-B64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735742">Wood <em>et al.</em>, 2011</a>). In particular, the difficulty in defining a clear phenotype conspires against the basic assumptions of GWAS, which are aimed at identifying excess of sharing in common genetic variants among individuals with the same high-level phenotype (e.g. health or disease). In this regard, the identification of intermediate, more precise and quantitative phenotypes that are more proximal (than a physiological state of health or disease) to the genotype in the chain of events determining causality, is potentially a more informative strategy. Examples of this approach include studies that integrate data on volumetric differences from patients with Alzheimer’s disease at the voxel level with genotypes from an existing GWAS to identify genetic associations involved in brain structure (<a href="#aws363-B9" rid="aws363-B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735760">Biffi <em>et al.</em>, 2010</a>; <a href="#aws363-B50" rid="aws363-B50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735789">Shen <em>et al.</em>, 2010</a>; <a href="#aws363-B55" rid="aws363-B55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735769">Stein <em>et al.</em>, 2010</a>). Similar approaches have been used to find genetic associations with cognition and susceptibility to schizophrenia (<a href="#aws363-B42" rid="aws363-B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735771">Potkin <em>et al.</em>, 2009<em>a</em></a>, <a href="#aws363-B43" rid="aws363-B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735734"><em>b</em></a>).</p><p id="__p37">In multiple sclerosis, the relationship between physical or cognitive impairment and whole-brain white matter lesion volume is generally weak. However, despite the fact that anatomical location of an injury is likely to explain (at least in part) the extent and type of neurological dysfunction, lesion volume rather than location is used as the prevalent metric. In a large cross-sectional study using lesion probability maps, <a href="#aws363-B12" rid="aws363-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735745">Charil <em>et al.</em> (2003)</a> showed a relationship between sites of lesions and major impairments in a group of subjects with relapsing-remitting multiple sclerosis. In this study, we have extended the concept of using lesion distribution as a phenotype to investigate the contribution of genetic variation to the topological distribution of white matter lesions in multiple sclerosis at voxel-level resolution.</p><p id="__p38">To be able to perform genetic associations with white matter lesion distribution across a population, it is critical to place all subject lesion masks in a common reference space (<a href="#aws363-B48" rid="aws363-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735787">Sdika and Pelletier, 2009</a>). In this study, we used image registration, an automated method to perform this task. Given two brain images, registration finds the geometric transformation that maps one brain to the other (<a href="#aws363-B33" rid="aws363-B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735738">Miller <em>et al.</em>, 1993</a>; <a href="#aws363-B45" rid="aws363-B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735740">Rueckert <em>et al.</em>, 2006</a>; <a href="#aws363-B47" rid="aws363-B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735780">Sdika, 2008</a>). Although non-rigid registration algorithms are able to cope with the high inter-subject variability of human brain, this method is sensitive to the presence of lesions, thus most investigators prefer to use affine registration when building multiple sclerosis lesion probability maps (<a href="#aws363-B37" rid="aws363-B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735786">Narayanan <em>et al.</em>, 1997</a>; <a href="#aws363-B12" rid="aws363-B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735756">Charil <em>et al.</em>, 2003</a>; <a href="#aws363-B15" rid="aws363-B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735766">Enzinger <em>et al.</em>, 2006</a>). Indeed, affine registration is robust to the presence of lesions in the brain, but only the global shape of the brain will fit the reference and the inter-subject variability is not taken into account. To further reduce the influence of the lesion in this process, lesional voxels can be masked out of the cost function of the registration. This method improves the registration of focal lesions with respect to affine or non-rigid registration with no cost function masking (<a href="#aws363-B11" rid="aws363-B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735758">Brett <em>et al.</em>, 2001</a>). We recently proposed a method dedicated to the problem of lesion mapping and registration of images of brains of subjects with multiple sclerosis (<a href="#aws363-B48" rid="aws363-B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735774">Sdika and Pelletier, 2009</a>). In this method, all white matter lesions are inpainted using an original algorithm before performing the registration. The intensity of the voxels in the lesions is replaced by the intensity of surrounding white matter to visually remove them from the original image. Any registration method can then be used to register the inpainted image on another image. We have shown that the registration is improved with respect to cost function masking, especially in lesional areas.</p><p id="__p39">We decided to use white matter hypointensity lesions detected on high-resolution T<sub>1</sub>-weighted 3D-IRSPGR images as the concordance between lesions visible on T<sub>2</sub> spin-echo weighted and T<sub>1</sub>-weighted 3D-IRSPGR images is ∼100% (<a href="#aws363-B20" rid="aws363-B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735728">Henry <em>et al.</em>, 2009</a>). These images allowed us to delineate sharp lesion edges, making determination of lesion topology more precise and enabling integration with readily available GWAS information from the same individuals. Results from this approach show significant associations with lesion distribution that are independent of whole-brain lesion volume. A clear limitation of our approach is the lack of an independent replication, which typically constitutes the most reliable evidence of reported genetic associations. However, to replicate this study, a similarly sized (or larger) cohort of subjects with both genome-wide genotypes and high-resolution structural imaging are needed, a data set that currently very few groups in the world are ready to obtain. One possible source of genotyped subjects would be those participating in large GWAS. Unfortunately, these are typically multicentre efforts, and although this is acceptable for DNA collection and analysis, comparing images obtained at different sites is technically challenging and to date there is no consensus as to how to address this problem effectively. Also, although these results are suggestive of a genotypic influence on spatial distribution of lesions in multiple sclerosis, this study may be underpowered to detect unequivocal signals. With a widespread interest in quantitative imaging and broader availability of genomic data, we believe this type of study will become more common and powerful in the near future.</p><p id="__p40">In our genetic study of lesion distribution (i.e. PC8), 31 associations fell below a FDR threshold of 1%. A caveat worth noticing is that most of the associated SNPs were relatively rare in our population (∼5%). Although the clusterplots appeared robust (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 3</a>), the regional association plots (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Fig. 4</a>) uncovered several SNPs in relatively high linkage disequilibrium with the index marker that did not show evidence of association. It is possible that this observation is related to these low-frequency alleles, but further studies will be needed to unequivocally establish these associations. Intriguingly, when analysed altogether in a biological context, these associations are consistent with biological processes involved in brain development and lesion formation or repair (i.e. myelination), thus supporting our original hypothesis of genetic associations with lesion topology. For example, one of the most relevant associations (rs10119179, <em>P</em><em> = </em>3.65 × 10<sup>−</sup><sup>9</sup>) maps within <em>SYK</em> (spleen tyrosine kinase) a kinase involved in tau phosphorylation (<a href="#aws363-B29" rid="aws363-B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735748">Lebouvier <em>et al.</em>, 2009</a>) and amyloid-β oligomer-mediated microglial activation (<a href="#aws363-B54" rid="aws363-B54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735725">Sondag <em>et al.</em>, 2009</a>). The product of <em>SYK</em> has been shown to phosphorylate myelin basic protein (<a href="#aws363-B51" rid="aws363-B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735759">Shimomura <em>et al.</em>, 1993</a>) and α-synuclein thereby preventing its aggregation, thus potentially playing an anti-neurodegenerative role (<a href="#aws363-B38" rid="aws363-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735763">Negro <em>et al.</em>, 2002</a>). Interestingly, DNA methylation in SYK is dynamically regulated in the human cerebral cortex throughout the lifespan, involves differentiated neurons and affects a substantial portion of genes predominantly by an age-related increase (<a href="#aws363-B52" rid="aws363-B52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735779">Siegmund <em>et al.</em>, 2007</a>). T<sub>1</sub> lesions non-specifically define inflammation/oedema, myelin loss and matrix destruction (a consequence both of severity of inflammation and lack of axonal regrowth). Although traditional approaches consider inflammation and tissue response as independent processes, we and others hypothesize that—in fact—they are interactive (<a href="#aws363-B63" rid="aws363-B63" class=" bibr popnode">Waxman, 2005</a>; <a href="#aws363-B18" rid="aws363-B18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735775">Hauser and Oksenberg, 2006</a>). Our findings support this notion by showing joint associations with immune and neural tissue response factors contributing to this complex phenotype. Most of the genes found to be associated with lesion topology (captured by PC8) can be traced to specific neural (e.g. axonogenesis, transmission of nerve impulse) or immune (e.g. NFkB signalling, T-cell regulation) functions.</p><p id="__p41">Convincing evidence for the involvement of <em>SYK</em>, <em>MYT1L</em>, <em>TRAPPC9</em>, <em>SLITKR6</em> and <em>RIC3</em> in the development and distribution of white matter lesions in multiple sclerosis can be gathered from the literature (<a href="#aws363-B51" rid="aws363-B51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735737">Shimomura <em>et al.</em>, 1993</a>; <a href="#aws363-B25" rid="aws363-B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735743">Kim <em>et al.</em>, 1997</a>; <a href="#aws363-B65" rid="aws363-B65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735716">Wrathall <em>et al.</em>, 1998</a>; <a href="#aws363-B38" rid="aws363-B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735729">Negro <em>et al.</em>, 2002</a>; <a href="#aws363-B1" rid="aws363-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735781">Aruga, 2003</a>; <a href="#aws363-B1" rid="aws363-B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735717">Aruga and Mikoshiba, 2003</a>; <a href="#aws363-B49" rid="aws363-B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735726">Severance and Yolken, 2007</a>; <a href="#aws363-B59" rid="aws363-B59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735749">Vana <em>et al.</em>, 2007</a>; <a href="#aws363-B30" rid="aws363-B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735777">Llorens <em>et al.</em>, 2008</a>; <a href="#aws363-B62" rid="aws363-B62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735730">Vrijenhoek <em>et al.</em>, 2008</a>; <a href="#aws363-B3" rid="aws363-B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735778">Baranzini <em>et al.</em>, 2009<em>a</em></a>; <a href="#aws363-B34" rid="aws363-B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735762">Mochida <em>et al.</em>, 2009</a>; <a href="#aws363-B41" rid="aws363-B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735750">Philippe <em>et al.</em>, 2009</a>; <a href="#aws363-B60" rid="aws363-B60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735739">Vierbuchen <em>et al.</em>, 2010</a>; <a href="#aws363-B61" rid="aws363-B61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_380286227">Vilarino-Guell <em>et al.</em>, 2010</a>) (<a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material</a>). The network-based strategy followed here uses a more liberal threshold of associations to include genes that are nominally significant (<em>P</em> &lt; 0.001) so that, by virtue of interacting among each other, define a biologically plausible module. Thus, it is possible that genes that are not significantly associated with the trait are part of any given module if they serve to functionally link two or more associated genes. However, the proportion of these non-associated genes is generally low (15/48 in the top network). Interestingly, 19 of the 30 (63%) genes with modest associations (10<sup>−</sup><sup>7</sup> &gt; <em>P</em> &gt; 10<sup>−</sup><sup>3</sup>) that interact physically with the top associated genes are known to be expressed in the brain. Among these, SEMA3A, RTN4R, GRM7, LRRC4C and FYN are of particular importance in the CNS owing to their role in axon guidance during development.</p><p id="__p42">New multiple sclerosis lesions begin as lymphocytic infiltration around a central vein, with subsequent centrifugal spread into surrounding tissue (<a href="#aws363-B6" rid="aws363-B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735727">Barnett and Prineas, 2004</a>). In more established lesions, and perhaps also in reactivated ones, inflammation is most prominent at the outer margins of the lesion (<a href="#aws363-B17" rid="aws363-B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735755">Gaitan <em>et al.</em>, 2011</a>; <a href="#aws363-B28" rid="aws363-B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735768">Lassmann, 2011</a>). The clinical significance of the heterogeneity described here is uncertain but is likely to reflect the influence of a limited number of gene variants on the propagation of lesions, influencing their initiation, evolution, termination and/or repair.</p><p id="__p43" class="p p-last">A large GWAS on multiple sclerosis susceptibility was recently reported and notably, most of the associated genes have an immunological function (<a href="#aws363-B46" rid="aws363-B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304735736">Sawcer <em>et al.</em>, 2011</a>). Although to date genetic variation has been primarily associated with risk rather than expression of multiple sclerosis, earlier studies of gene-outcome associations have generally relied on bedside measures of clinical disability, which are insensitive to the number, volume or topology of brain lesions. In this study, we have identified an MRI-based, quantitative trait that is associated with common variants in patients with multiple sclerosis. The apparent strength of the associations and a compelling hypothesis for their interaction make these important candidate modifier genes for the multiple sclerosis phenotype, but independent replication of these findings is needed. The data reduction methods proposed substantially improve the potential for prospective studies to test these and related hypotheses. Joint analysis of quantitative imaging traits and genomic data should shed new light to the heterogeneity of multiple sclerosis and its responses to treatment.</p></div><div id="__sec16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec16title">Funding</h2><p id="__p44" class="p p-first-last">National Institutes of Health (<span>RO1NS26799</span>; to <span>S.L.H., R01NS062885</span>; to <span>D.P</span>;<span>.</span>) and National Multiple Sclerosis Society (<span>RG2901</span>; to <span>J.R.O.</span>). S.E.B. is a Harry Weaver Neuroscience Scholar of the US National MS Society.</p></div><div id="__sec17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec17title">Supplementary material</h2><!--/article/body/sec/--><p id="__p45" class="p p-first"><a href="http://brain.oxfordjournals.org/lookup/suppl/doi:10.1093/brain/aws363/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material</a> is available at <em>Brain</em> online.</p><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139909722467008"><a href="/pmc/articles/PMC3613709/bin/supp_136_4_1012__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
















</div></div><div id="idm139909716115600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139909716115600title">Acknowledgements</h2><div class="sec"><p id="__p46">The authors thank R. Lincoln, R. Guerrero, H. Mousavi, R. Gomez and A. Santaniello for sample and database management. The authors thank the patients with multiple sclerosis that participated in the study.</p></div></div><div id="idm139909716114400" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139909716114400title">Glossary</h2><div class="bk-sec"><div><h4>Abbreviations</h4><table class="default-table glossary"><tbody><tr><td id="G2">GWAS</td><td>genome-wide association study</td></tr><tr><td id="G3">IRSPGR</td><td>inversion recovery spoiled gradient-recalled echo</td></tr><tr><td id="G4">SNP</td><td>single-nucleotide polymorphism</td></tr></tbody></table></div></div></div><div id="idm139909715586048" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139909715586048title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="aws363-B1"><span class="element-citation">Aruga J. Slitrk6 expression profile in the mouse embryo and its relationship to that of Nlrr3. <span><span class="ref-journal">Gene Expr Patterns. </span>2003;<span class="ref-vol">3</span>:727–33.</span> [<a href="/pubmed/14643680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene+Expr+Patterns&amp;title=Slitrk6+expression+profile+in+the+mouse+embryo+and+its+relationship+to+that+of+Nlrr3&amp;author=J+Aruga&amp;volume=3&amp;publication_year=2003&amp;pages=727-33&amp;pmid=14643680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B2"><span class="element-citation">Aruga J, Mikoshiba K. Identification and characterization of Slitrk, a novel neuronal transmembrane protein family controlling neurite outgrowth. <span><span class="ref-journal">Mol Cell Neurosci. </span>2003;<span class="ref-vol">24</span>:117–29.</span> [<a href="/pubmed/14550773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Neurosci&amp;title=Identification+and+characterization+of+Slitrk,+a+novel+neuronal+transmembrane+protein+family+controlling+neurite+outgrowth&amp;author=J+Aruga&amp;author=K+Mikoshiba&amp;volume=24&amp;publication_year=2003&amp;pages=117-29&amp;pmid=14550773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B3"><span class="element-citation">Baranzini SE, Galwey NW, Wang J, Khankhanian P, Lindberg R, Pelletier D, et al.  Pathway and network-based analysis of genome-wide association studies in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009a;<span class="ref-vol">18</span>:2078–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2678928/">PMC free article</a>]</span> [<a href="/pubmed/19286671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Pathway+and+network-based+analysis+of+genome-wide+association+studies+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=NW+Galwey&amp;author=J+Wang&amp;author=P+Khankhanian&amp;author=R+Lindberg&amp;volume=18&amp;publication_year=2009a&amp;pages=2078-90&amp;pmid=19286671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B4"><span class="element-citation">Baranzini SE, Srinivasan R, Khankhanian P, Okuda DT, Nelson SJ, Matthews PM, et al.  Genetic variation influences glutamate concentrations in brains of patients with multiple sclerosis. <span><span class="ref-journal">Brain. </span>2010;<span class="ref-vol">133</span>:2603–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2929334/">PMC free article</a>]</span> [<a href="/pubmed/20802204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Genetic+variation+influences+glutamate+concentrations+in+brains+of+patients+with+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=R+Srinivasan&amp;author=P+Khankhanian&amp;author=DT+Okuda&amp;author=SJ+Nelson&amp;volume=133&amp;publication_year=2010&amp;pages=2603-11&amp;pmid=20802204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B5"><span class="element-citation">Baranzini SE, Wang J, Gibson RA, Galwey N, Naegelin Y, Barkhof F, et al.  Genome-wide association analysis of susceptibility and clinical phenotype in multiple sclerosis. <span><span class="ref-journal">Hum Mol Genet. </span>2009b;<span class="ref-vol">18</span>:767–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4334814/">PMC free article</a>]</span> [<a href="/pubmed/19010793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+analysis+of+susceptibility+and+clinical+phenotype+in+multiple+sclerosis&amp;author=SE+Baranzini&amp;author=J+Wang&amp;author=RA+Gibson&amp;author=N+Galwey&amp;author=Y+Naegelin&amp;volume=18&amp;publication_year=2009b&amp;pages=767-78&amp;pmid=19010793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B6"><span class="element-citation">Barnett MH, Prineas JW. Relapsing and remitting multiple sclerosis: pathology of the newly forming lesion. <span><span class="ref-journal">Ann Neurol. </span>2004;<span class="ref-vol">55</span>:458–68.</span> [<a href="/pubmed/15048884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Relapsing+and+remitting+multiple+sclerosis:+pathology+of+the+newly+forming+lesion&amp;author=MH+Barnett&amp;author=JW+Prineas&amp;volume=55&amp;publication_year=2004&amp;pages=458-68&amp;pmid=15048884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B8"><span class="element-citation">Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <span><span class="ref-journal">J R Stat Soc. </span>1995;<span class="ref-vol">57</span>:289–300.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+R+Stat+Soc&amp;title=Controlling+the+false+discovery+rate:+a+practical+and+powerful+approach+to+multiple+testing&amp;author=Y+Benjamini&amp;author=Y+Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B9"><span class="element-citation">Biffi A, Anderson CD, Desikan RS, Sabuncu M, Cortellini L, Schmansky N, et al.  Genetic variation and neuroimaging measures in Alzheimer disease. <span><span class="ref-journal">Arch Neurol. </span>2010;<span class="ref-vol">67</span>:677–85.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2956757/">PMC free article</a>]</span> [<a href="/pubmed/20558387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Genetic+variation+and+neuroimaging+measures+in+Alzheimer+disease&amp;author=A+Biffi&amp;author=CD+Anderson&amp;author=RS+Desikan&amp;author=M+Sabuncu&amp;author=L+Cortellini&amp;volume=67&amp;publication_year=2010&amp;pages=677-85&amp;pmid=20558387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B10"><span class="element-citation">Blum D, Yonelinas AP, Luks T, Newitt D, Oh J, Lu Y, et al.  Dissociating perceptual and conceptual implicit memory in multiple sclerosis patients. <span><span class="ref-journal">Brain Cogn. </span>2002;<span class="ref-vol">50</span>:51–61.</span> [<a href="/pubmed/12372351" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Cogn&amp;title=Dissociating+perceptual+and+conceptual+implicit+memory+in+multiple+sclerosis+patients&amp;author=D+Blum&amp;author=AP+Yonelinas&amp;author=T+Luks&amp;author=D+Newitt&amp;author=J+Oh&amp;volume=50&amp;publication_year=2002&amp;pages=51-61&amp;pmid=12372351&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B11"><span class="element-citation">Brett M, Leff AP, Rorden C, Ashburner J. Spatial normalization of brain images with focal lesions using cost function masking. <span><span class="ref-journal">Neuroimage. </span>2001;<span class="ref-vol">14</span>:486–500.</span> [<a href="/pubmed/11467921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Spatial+normalization+of+brain+images+with+focal+lesions+using+cost+function+masking&amp;author=M+Brett&amp;author=AP+Leff&amp;author=C+Rorden&amp;author=J+Ashburner&amp;volume=14&amp;publication_year=2001&amp;pages=486-500&amp;pmid=11467921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B12"><span class="element-citation">Charil A, Zijdenbos AP, Taylor J, Boelman C, Worsley KJ, Evans AC, et al.  Statistical mapping analysis of lesion location and neurological disability in multiple sclerosis: application to 452 patient data sets. <span><span class="ref-journal">Neuroimage. </span>2003;<span class="ref-vol">19</span>:532–44.</span> [<a href="/pubmed/12880785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Statistical+mapping+analysis+of+lesion+location+and+neurological+disability+in+multiple+sclerosis:+application+to+452+patient+data+sets&amp;author=A+Charil&amp;author=AP+Zijdenbos&amp;author=J+Taylor&amp;author=C+Boelman&amp;author=KJ+Worsley&amp;volume=19&amp;publication_year=2003&amp;pages=532-44&amp;pmid=12880785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B13"><span class="element-citation">Cheung VG, Spielman RS. The genetics of variation in gene expression. <span><span class="ref-journal">Nat Genet. </span>2002;<span class="ref-vol">32</span>(Suppl):522–5.</span> [<a href="/pubmed/12454648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+genetics+of+variation+in+gene+expression&amp;author=VG+Cheung&amp;author=RS+Spielman&amp;volume=32&amp;issue=Suppl&amp;publication_year=2002&amp;pages=522-5&amp;pmid=12454648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B14"><span class="element-citation">De Moor MH, Liu YJ, Boomsma DI, Li J, Hamilton JJ, Hottenga JJ, et al.  Genome-wide association study of exercise behavior in Dutch and American adults. <span><span class="ref-journal">Med Sci Sports Exerc. </span>2009;<span class="ref-vol">41</span>:1887–95.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2895958/">PMC free article</a>]</span> [<a href="/pubmed/19727025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Med+Sci+Sports+Exerc&amp;title=Genome-wide+association+study+of+exercise+behavior+in+Dutch+and+American+adults&amp;author=MH+De+Moor&amp;author=YJ+Liu&amp;author=DI+Boomsma&amp;author=J+Li&amp;author=JJ+Hamilton&amp;volume=41&amp;publication_year=2009&amp;pages=1887-95&amp;pmid=19727025&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B15"><span class="element-citation">Enzinger C, Smith S, Fazekas F, Drevin G, Ropele S, Nichols T, et al.  Lesion probability maps of white matter hyperintensities in elderly individuals: results of the Austrian stroke prevention study. <span><span class="ref-journal">J Neurol. </span>2006;<span class="ref-vol">253</span>:1064–70.</span> [<a href="/pubmed/16607471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurol&amp;title=Lesion+probability+maps+of+white+matter+hyperintensities+in+elderly+individuals:+results+of+the+Austrian+stroke+prevention+study&amp;author=C+Enzinger&amp;author=S+Smith&amp;author=F+Fazekas&amp;author=G+Drevin&amp;author=S+Ropele&amp;volume=253&amp;publication_year=2006&amp;pages=1064-70&amp;pmid=16607471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B16"><span class="element-citation">Eriksson N, Macpherson JM, Tung JY, Hon LS, Naughton B, Saxonov S, et al.  Web-based, participant-driven studies yield novel genetic associations for common traits. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000993.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2891811/">PMC free article</a>]</span> [<a href="/pubmed/20585627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Web-based,+participant-driven+studies+yield+novel+genetic+associations+for+common+traits&amp;author=N+Eriksson&amp;author=JM+Macpherson&amp;author=JY+Tung&amp;author=LS+Hon&amp;author=B+Naughton&amp;volume=6&amp;publication_year=2010&amp;pages=e1000993&amp;pmid=20585627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B17"><span class="element-citation">Gaitan MI, Shea CD, Evangelou IE, Stone RD, Fenton KM, Bielekova B, et al.  Evolution of the blood-brain barrier in newly forming multiple sclerosis lesions. <span><span class="ref-journal">Ann Neurol. </span>2011;<span class="ref-vol">70</span>:22–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3143223/">PMC free article</a>]</span> [<a href="/pubmed/21710622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Evolution+of+the+blood-brain+barrier+in+newly+forming+multiple+sclerosis+lesions&amp;author=MI+Gaitan&amp;author=CD+Shea&amp;author=IE+Evangelou&amp;author=RD+Stone&amp;author=KM+Fenton&amp;volume=70&amp;publication_year=2011&amp;pages=22-9&amp;pmid=21710622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B18"><span class="element-citation">Hauser SL, Oksenberg JR. The neurobiology of multiple sclerosis: genes, inflammation, and neurodegeneration. <span><span class="ref-journal">Neuron. </span>2006;<span class="ref-vol">52</span>:61–76.</span> [<a href="/pubmed/17015227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+neurobiology+of+multiple+sclerosis:+genes,+inflammation,+and+neurodegeneration&amp;author=SL+Hauser&amp;author=JR+Oksenberg&amp;volume=52&amp;publication_year=2006&amp;pages=61-76&amp;pmid=17015227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B19"><span class="element-citation">Hauser SL, Oksenberg JR, Lincoln R, Garovoy J, Beck RW, Cole SR, et al.  Optic Neuritis Study Group. Interaction between HLA-DR2 and abnormal brain MRI in optic neuritis and early MS. <span><span class="ref-journal">Neurology. </span>2000;<span class="ref-vol">54</span>:1859–61.</span> [<a href="/pubmed/10802800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Optic+Neuritis+Study+Group.+Interaction+between+HLA-DR2+and+abnormal+brain+MRI+in+optic+neuritis+and+early+MS&amp;author=SL+Hauser&amp;author=JR+Oksenberg&amp;author=R+Lincoln&amp;author=J+Garovoy&amp;author=RW+Beck&amp;volume=54&amp;publication_year=2000&amp;pages=1859-61&amp;pmid=10802800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B20"><span class="element-citation">Henry RG, Shieh M, Amirbekian B, Chung S, Okuda DT, Pelletier D. Connecting white matter injury and thalamic atrophy in clinically isolated syndromes. <span><span class="ref-journal">J. Neurol Sci. </span>2009;<span class="ref-vol">282</span>:61–6.</span> [<a href="/pubmed/19394969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol+Sci&amp;title=Connecting+white+matter+injury+and+thalamic+atrophy+in+clinically+isolated+syndromes&amp;author=RG+Henry&amp;author=M+Shieh&amp;author=B+Amirbekian&amp;author=S+Chung&amp;author=DT+Okuda&amp;volume=282&amp;publication_year=2009&amp;pages=61-6&amp;pmid=19394969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B21"><span class="element-citation">Hibar DP, Stein JL, Kohannim O, Jahanshad N, Saykin AJ, Shen L, et al.  Voxelwise gene-wide association study (vGeneWAS): multivariate gene-based association testing in 731 elderly subjects. <span><span class="ref-journal">Neuroimage. </span>2011;<span class="ref-vol">56</span>:1875–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3366726/">PMC free article</a>]</span> [<a href="/pubmed/21497199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Voxelwise+gene-wide+association+study+(vGeneWAS):+multivariate+gene-based+association+testing+in+731+elderly+subjects&amp;author=DP+Hibar&amp;author=JL+Stein&amp;author=O+Kohannim&amp;author=N+Jahanshad&amp;author=AJ+Saykin&amp;volume=56&amp;publication_year=2011&amp;pages=1875-91&amp;pmid=21497199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B22"><span class="element-citation">Hicks AA, Pramstaller PP, Johansson A, Vitart V, Rudan I, Ugocsai P, et al.  Genetic determinants of circulating sphingolipid concentrations in European populations. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000672.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745562/">PMC free article</a>]</span> [<a href="/pubmed/19798445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+determinants+of+circulating+sphingolipid+concentrations+in+European+populations&amp;author=AA+Hicks&amp;author=PP+Pramstaller&amp;author=A+Johansson&amp;author=V+Vitart&amp;author=I+Rudan&amp;volume=5&amp;publication_year=2009&amp;pages=e1000672&amp;pmid=19798445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B23"><span class="element-citation">Horakova D, Zivadinov R, Weinstock-Guttman B, Havrdova E, Tamano-Blanco M, Tyblova M, et al.  HLA DRB1*1501 is only modestly associated with lesion burden at the first demyelinating event. <span><span class="ref-journal">J. Neuroimmunol. </span>2011;<span class="ref-vol">236</span>:76–80.</span> [<a href="/pubmed/21621859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neuroimmunol&amp;title=HLA+DRB1*1501+is+only+modestly+associated+with+lesion+burden+at+the+first+demyelinating+event&amp;author=D+Horakova&amp;author=R+Zivadinov&amp;author=B+Weinstock-Guttman&amp;author=E+Havrdova&amp;author=M+Tamano-Blanco&amp;volume=236&amp;publication_year=2011&amp;pages=76-80&amp;pmid=21621859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B24"><span class="element-citation">Ideker T, Ozier O, Schwikowski B, Siegel AF. Discovering regulatory and signalling circuits in molecular interaction networks. <span><span class="ref-journal">Bioinformatics. </span>2002;<span class="ref-vol">18</span>(Suppl 1):S233–40.</span> [<a href="/pubmed/12169552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Discovering+regulatory+and+signalling+circuits+in+molecular+interaction+networks&amp;author=T+Ideker&amp;author=O+Ozier&amp;author=B+Schwikowski&amp;author=AF+Siegel&amp;volume=18&amp;issue=Suppl+1&amp;publication_year=2002&amp;pages=S233-40&amp;pmid=12169552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B25"><span class="element-citation">Kim JG, Armstrong RC, v Agoston D, Robinsky A, Wiese C, Nagle J, et al.  Myelin transcription factor 1 (Myt1) of the oligodendrocyte lineage, along with a closely related CCHC zinc finger, is expressed in developing neurons in the mammalian central nervous system. <span><span class="ref-journal">J Neurosci Res. </span>1997;<span class="ref-vol">50</span>:272–90.</span> [<a href="/pubmed/9373037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Res&amp;title=Myelin+transcription+factor+1+(Myt1)+of+the+oligodendrocyte+lineage,+along+with+a+closely+related+CCHC+zinc+finger,+is+expressed+in+developing+neurons+in+the+mammalian+central+nervous+system&amp;author=JG+Kim&amp;author=RC+Armstrong&amp;author=D+v+Agoston&amp;author=A+Robinsky&amp;author=C+Wiese&amp;volume=50&amp;publication_year=1997&amp;pages=272-90&amp;pmid=9373037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B26"><span class="element-citation">Kutalik Z, Whittaker J, Waterworth D, Beckmann JS, Bergmann S. Novel method to estimate the phenotypic variation explained by genome-wide association studies reveals large fraction of the missing heritability. <span><span class="ref-journal">Genet Epidemiol. </span>2011;<span class="ref-vol">35</span>:341–9.</span> [<a href="/pubmed/21465548" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Novel+method+to+estimate+the+phenotypic+variation+explained+by+genome-wide+association+studies+reveals+large+fraction+of+the+missing+heritability&amp;author=Z+Kutalik&amp;author=J+Whittaker&amp;author=D+Waterworth&amp;author=JS+Beckmann&amp;author=S+Bergmann&amp;volume=35&amp;publication_year=2011&amp;pages=341-9&amp;pmid=21465548&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B27"><span class="element-citation">Lango Allen H, Estrada K, Lettre G, Berndt SI, Weedon MN, Rivadeneira F, et al.  Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H+Lango+Allen&amp;author=K+Estrada&amp;author=G+Lettre&amp;author=SI+Berndt&amp;author=MN+Weedon&amp;volume=467&amp;publication_year=2010&amp;pages=832-8&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B28"><span class="element-citation">Lassmann H. A dynamic view of the blood-brain barrier in active multiple sclerosis lesions. <span><span class="ref-journal">Ann Neurol. </span>2011;<span class="ref-vol">70</span>:1–2.</span> [<a href="/pubmed/21710628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=A+dynamic+view+of+the+blood-brain+barrier+in+active+multiple+sclerosis+lesions&amp;author=H+Lassmann&amp;volume=70&amp;publication_year=2011&amp;pages=1-2&amp;pmid=21710628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B29"><span class="element-citation">Lebouvier T, Scales TM, Williamson R, Noble W, Duyckaerts C, Hanger DP, et al.  The microtubule-associated protein tau is also phosphorylated on tyrosine. <span><span class="ref-journal">J Alzheimers Dis. </span>2009;<span class="ref-vol">18</span>:1–9.</span> [<a href="/pubmed/19542604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Alzheimers+Dis&amp;title=The+microtubule-associated+protein+tau+is+also+phosphorylated+on+tyrosine&amp;author=T+Lebouvier&amp;author=TM+Scales&amp;author=R+Williamson&amp;author=W+Noble&amp;author=C+Duyckaerts&amp;volume=18&amp;publication_year=2009&amp;pages=1-9&amp;pmid=19542604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B30"><span class="element-citation">Llorens F, Gil V, Iraola S, Carim-Todd L, Marti E, Estivill X, et al.  Developmental analysis of Lingo-1/Lern1 protein expression in the mouse brain: interaction of its intracellular domain with Myt1l. <span><span class="ref-journal">Dev Neurobiol. </span>2008;<span class="ref-vol">68</span>:521–41.</span> [<a href="/pubmed/18186492" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Neurobiol&amp;title=Developmental+analysis+of+Lingo-1/Lern1+protein+expression+in+the+mouse+brain:+interaction+of+its+intracellular+domain+with+Myt1l&amp;author=F+Llorens&amp;author=V+Gil&amp;author=S+Iraola&amp;author=L+Carim-Todd&amp;author=E+Marti&amp;volume=68&amp;publication_year=2008&amp;pages=521-41&amp;pmid=18186492&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B31"><span class="element-citation">McDonald WI, Compston A, Edan G, Goodkin D, Hartung HP, Lublin FD, et al.  Recommended diagnostic criteria for multiple sclerosis: guidelines from the International Panel on the diagnosis of multiple sclerosis. <span><span class="ref-journal">Ann Neurol. </span>2001;<span class="ref-vol">50</span>:121–7.</span> [<a href="/pubmed/11456302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Recommended+diagnostic+criteria+for+multiple+sclerosis:+guidelines+from+the+International+Panel+on+the+diagnosis+of+multiple+sclerosis&amp;author=WI+McDonald&amp;author=A+Compston&amp;author=G+Edan&amp;author=D+Goodkin&amp;author=HP+Hartung&amp;volume=50&amp;publication_year=2001&amp;pages=121-7&amp;pmid=11456302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B32"><span class="element-citation">Medland SE, Zayats T, Glaser B, Nyholt DR, Gordon SD, Wright MJ, et al.  A variant in LIN28B is associated with 2D:4D finger-length ratio, a putative retrospective biomarker of prenatal testosterone exposure. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:519–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850436/">PMC free article</a>]</span> [<a href="/pubmed/20303062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+variant+in+LIN28B+is+associated+with+2D:4D+finger-length+ratio,+a+putative+retrospective+biomarker+of+prenatal+testosterone+exposure&amp;author=SE+Medland&amp;author=T+Zayats&amp;author=B+Glaser&amp;author=DR+Nyholt&amp;author=SD+Gordon&amp;volume=86&amp;publication_year=2010&amp;pages=519-25&amp;pmid=20303062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B33"><span class="element-citation">Miller MI, Christensen GE, Amit Y, Grenander U. Mathematical textbook of deformable neuroanatomies. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1993;<span class="ref-vol">90</span>:11944–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC48101/">PMC free article</a>]</span> [<a href="/pubmed/8265653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Mathematical+textbook+of+deformable+neuroanatomies&amp;author=MI+Miller&amp;author=GE+Christensen&amp;author=Y+Amit&amp;author=U+Grenander&amp;volume=90&amp;publication_year=1993&amp;pages=11944-8&amp;pmid=8265653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B34"><span class="element-citation">Mochida GH, Mahajnah M, Hill AD, Basel-Vanagaite L, Gleason D, Hill RS, et al.  A truncating mutation of TRAPPC9 is associated with autosomal-recessive intellectual disability and postnatal microcephaly. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:897–902.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2790576/">PMC free article</a>]</span> [<a href="/pubmed/20004763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+truncating+mutation+of+TRAPPC9+is+associated+with+autosomal-recessive+intellectual+disability+and+postnatal+microcephaly&amp;author=GH+Mochida&amp;author=M+Mahajnah&amp;author=AD+Hill&amp;author=L+Basel-Vanagaite&amp;author=D+Gleason&amp;volume=85&amp;publication_year=2009&amp;pages=897-902&amp;pmid=20004763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B35"><span class="element-citation">Mowry EM, Beheshtian A, Waubant E, Goodin DS, Cree BA, Qualley P, et al.  Quality of life in multiple sclerosis is associated with lesion burden and brain volume measures. <span><span class="ref-journal">Neurology. </span>2009;<span class="ref-vol">72</span>:1760–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683738/">PMC free article</a>]</span> [<a href="/pubmed/19451531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Quality+of+life+in+multiple+sclerosis+is+associated+with+lesion+burden+and+brain+volume+measures&amp;author=EM+Mowry&amp;author=A+Beheshtian&amp;author=E+Waubant&amp;author=DS+Goodin&amp;author=BA+Cree&amp;volume=72&amp;publication_year=2009&amp;pages=1760-5&amp;pmid=19451531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B36"><span class="element-citation">Nan H, Kraft P, Qureshi AA, Guo Q, Chen C, Hankinson SE, et al.  Genome-wide association study of tanning phenotype in a population of European ancestry. <span><span class="ref-journal">J Invest Dermatol. </span>2009;<span class="ref-vol">129</span>:2250–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758927/">PMC free article</a>]</span> [<a href="/pubmed/19340012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=Genome-wide+association+study+of+tanning+phenotype+in+a+population+of+European+ancestry&amp;author=H+Nan&amp;author=P+Kraft&amp;author=AA+Qureshi&amp;author=Q+Guo&amp;author=C+Chen&amp;volume=129&amp;publication_year=2009&amp;pages=2250-7&amp;pmid=19340012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B37"><span class="element-citation">Narayanan S, Fu L, Pioro E, De Stefano N, Collins DL, Francis GS, et al.  Imaging of axonal damage in multiple sclerosis: spatial distribution of magnetic resonance imaging lesions. <span><span class="ref-journal">Ann Neurol. </span>1997;<span class="ref-vol">41</span>:385–91.</span> [<a href="/pubmed/9066360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Imaging+of+axonal+damage+in+multiple+sclerosis:+spatial+distribution+of+magnetic+resonance+imaging+lesions&amp;author=S+Narayanan&amp;author=L+Fu&amp;author=E+Pioro&amp;author=N+De+Stefano&amp;author=DL+Collins&amp;volume=41&amp;publication_year=1997&amp;pages=385-91&amp;pmid=9066360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B38"><span class="element-citation">Negro A, Brunati AM, Donella-Deana A, Massimino ML, Pinna LA. Multiple phosphorylation of alpha-synuclein by protein tyrosine kinase Syk prevents eosin-induced aggregation. <span><span class="ref-journal">FASEB J. </span>2002;<span class="ref-vol">16</span>:210–12.</span> [<a href="/pubmed/11744621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J&amp;title=Multiple+phosphorylation+of+alpha-synuclein+by+protein+tyrosine+kinase+Syk+prevents+eosin-induced+aggregation&amp;author=A+Negro&amp;author=AM+Brunati&amp;author=A+Donella-Deana&amp;author=ML+Massimino&amp;author=LA+Pinna&amp;volume=16&amp;publication_year=2002&amp;pages=210-12&amp;pmid=11744621&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B39"><span class="element-citation">Newton-Cheh C, Eijgelsheim M, Rice KM, de Bakker PI, Yin X, Estrada K, et al.  Common variants at ten loci influence QT interval duration in the QTGEN Study. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:399–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701449/">PMC free article</a>]</span> [<a href="/pubmed/19305408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+ten+loci+influence+QT+interval+duration+in+the+QTGEN+Study&amp;author=C+Newton-Cheh&amp;author=M+Eijgelsheim&amp;author=KM+Rice&amp;author=PI+de+Bakker&amp;author=X+Yin&amp;volume=41&amp;publication_year=2009&amp;pages=399-406&amp;pmid=19305408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B40"><span class="element-citation">Okuda DT, Srinivasan R, Oksenberg JR, Goodin DS, Baranzini SE, Beheshtian A, et al.  Genotype-Phenotype correlations in multiple sclerosis: HLA genes influence disease severity inferred by 1HMR spectroscopy and MRI measures. <span><span class="ref-journal">Brain. </span>2009;<span class="ref-vol">132</span>(Pt 1):250–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2638695/">PMC free article</a>]</span> [<a href="/pubmed/19022862" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Genotype-Phenotype+correlations+in+multiple+sclerosis:+HLA+genes+influence+disease+severity+inferred+by+1HMR+spectroscopy+and+MRI+measures&amp;author=DT+Okuda&amp;author=R+Srinivasan&amp;author=JR+Oksenberg&amp;author=DS+Goodin&amp;author=SE+Baranzini&amp;volume=132&amp;issue=Pt+1&amp;publication_year=2009&amp;pages=250-9&amp;pmid=19022862&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B41"><span class="element-citation">Philippe O, Rio M, Carioux A, Plaza JM, Guigue P, Molinari F, et al.  Combination of linkage mapping and microarray-expression analysis identifies NF-kappaB signaling defect as a cause of autosomal-recessive mental retardation. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:903–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795800/">PMC free article</a>]</span> [<a href="/pubmed/20004764" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Combination+of+linkage+mapping+and+microarray-expression+analysis+identifies+NF-kappaB+signaling+defect+as+a+cause+of+autosomal-recessive+mental+retardation&amp;author=O+Philippe&amp;author=M+Rio&amp;author=A+Carioux&amp;author=JM+Plaza&amp;author=P+Guigue&amp;volume=85&amp;publication_year=2009&amp;pages=903-8&amp;pmid=20004764&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B42"><span class="element-citation">Potkin SG, Turner JA, Fallon JA, Lakatos A, Keator DB, Guffanti G, et al.  Gene discovery through imaging genetics: identification of two novel genes associated with schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2009a;<span class="ref-vol">14</span>:416–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3254586/">PMC free article</a>]</span> [<a href="/pubmed/19065146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Gene+discovery+through+imaging+genetics:+identification+of+two+novel+genes+associated+with+schizophrenia&amp;author=SG+Potkin&amp;author=JA+Turner&amp;author=JA+Fallon&amp;author=A+Lakatos&amp;author=DB+Keator&amp;volume=14&amp;publication_year=2009a&amp;pages=416-28&amp;pmid=19065146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B43"><span class="element-citation">Potkin SG, Turner JA, Guffanti G, Lakatos A, Torri F, Keator DB, et al.  Genome-wide strategies for discovering genetic influences on cognition and cognitive disorders: methodological considerations. <span><span class="ref-journal">Cogn Neuropsychiatry. </span>2009b;<span class="ref-vol">14</span>:391–418.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037334/">PMC free article</a>]</span> [<a href="/pubmed/19634037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cogn+Neuropsychiatry&amp;title=Genome-wide+strategies+for+discovering+genetic+influences+on+cognition+and+cognitive+disorders:+methodological+considerations&amp;author=SG+Potkin&amp;author=JA+Turner&amp;author=G+Guffanti&amp;author=A+Lakatos&amp;author=F+Torri&amp;volume=14&amp;publication_year=2009b&amp;pages=391-418&amp;pmid=19634037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B44"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B45"><span class="element-citation">Rueckert D, Aljabar P, Heckemann RA, Hajnal JV, Hammers A. Diffeomorphic registration using B-splines. <span><span class="ref-journal">Med Image Comput Comput Assist Interv. </span>2006;<span class="ref-vol">9</span>(Pt 2):702–9.</span> [<a href="/pubmed/17354834" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Med+Image+Comput+Comput+Assist+Interv&amp;title=Diffeomorphic+registration+using+B-splines&amp;author=D+Rueckert&amp;author=P+Aljabar&amp;author=RA+Heckemann&amp;author=JV+Hajnal&amp;author=A+Hammers&amp;volume=9&amp;issue=Pt+2&amp;publication_year=2006&amp;pages=702-9&amp;pmid=17354834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B46"><span class="element-citation">Sawcer S, Hellenthal G, Pirinen M, Spencer CC, Patsopoulos NA, Moutsianas L, et al.  Genetic risk and a primary role for cell-mediated immune mechanisms in multiple sclerosis. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">476</span>:214–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3182531/">PMC free article</a>]</span> [<a href="/pubmed/21833088" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+risk+and+a+primary+role+for+cell-mediated+immune+mechanisms+in+multiple+sclerosis&amp;author=S+Sawcer&amp;author=G+Hellenthal&amp;author=M+Pirinen&amp;author=CC+Spencer&amp;author=NA+Patsopoulos&amp;volume=476&amp;publication_year=2011&amp;pages=214-9&amp;pmid=21833088&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B47"><span class="element-citation">Sdika M. A fast nonrigid image registration with constraints on the Jacobian using large scale constrained optimization. <span><span class="ref-journal">IEEE Trans Med Imaging. </span>2008;<span class="ref-vol">27</span>:271–81.</span> [<a href="/pubmed/18334448" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Trans+Med+Imaging&amp;title=A+fast+nonrigid+image+registration+with+constraints+on+the+Jacobian+using+large+scale+constrained+optimization&amp;author=M+Sdika&amp;volume=27&amp;publication_year=2008&amp;pages=271-81&amp;pmid=18334448&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B48"><span class="element-citation">Sdika M, Pelletier D. Nonrigid registration of multiple sclerosis brain images using lesion inpainting for morphometry or lesion mapping. <span><span class="ref-journal">Hum Brain Mapp. </span>2009;<span class="ref-vol">30</span>:1060–7.</span> [<a href="/pubmed/18412131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Brain+Mapp&amp;title=Nonrigid+registration+of+multiple+sclerosis+brain+images+using+lesion+inpainting+for+morphometry+or+lesion+mapping&amp;author=M+Sdika&amp;author=D+Pelletier&amp;volume=30&amp;publication_year=2009&amp;pages=1060-7&amp;pmid=18412131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B49"><span class="element-citation">Severance EG, Yolken RH. Lack of RIC-3 congruence with beta2 subunit-containing nicotinic acetylcholine receptors in bipolar disorder. <span><span class="ref-journal">Neuroscience. </span>2007;<span class="ref-vol">148</span>:454–60.</span> [<a href="/pubmed/17640815" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Lack+of+RIC-3+congruence+with+beta2+subunit-containing+nicotinic+acetylcholine+receptors+in+bipolar+disorder&amp;author=EG+Severance&amp;author=RH+Yolken&amp;volume=148&amp;publication_year=2007&amp;pages=454-60&amp;pmid=17640815&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B50"><span class="element-citation">Shen L, Kim S, Risacher SL, Nho K, Swaminathan S, West JD, et al.  Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: a study of the ADNI cohort. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:1051–63.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2892122/">PMC free article</a>]</span> [<a href="/pubmed/20100581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Whole+genome+association+study+of+brain-wide+imaging+phenotypes+for+identifying+quantitative+trait+loci+in+MCI+and+AD:+a+study+of+the+ADNI+cohort&amp;author=L+Shen&amp;author=S+Kim&amp;author=SL+Risacher&amp;author=K+Nho&amp;author=S+Swaminathan&amp;volume=53&amp;publication_year=2010&amp;pages=1051-63&amp;pmid=20100581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B51"><span class="element-citation">Shimomura R, Sakai K, Tanaka Y, Yonezawa K, Hashimoto E, Kasuga M, et al.  Phosphorylation sites of myelin basic protein by a catalytic fragment of non-receptor type protein-tyrosine kinase p72syk and comparison with those by insulin receptor kinase. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>1993;<span class="ref-vol">192</span>:252–60.</span> [<a href="/pubmed/7682809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=Phosphorylation+sites+of+myelin+basic+protein+by+a+catalytic+fragment+of+non-receptor+type+protein-tyrosine+kinase+p72syk+and+comparison+with+those+by+insulin+receptor+kinase&amp;author=R+Shimomura&amp;author=K+Sakai&amp;author=Y+Tanaka&amp;author=K+Yonezawa&amp;author=E+Hashimoto&amp;volume=192&amp;publication_year=1993&amp;pages=252-60&amp;pmid=7682809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B52"><span class="element-citation">Siegmund KD, Connor CM, Campan M, Long TI, Weisenberger DJ, Biniszkiewicz D, et al.  DNA methylation in the human cerebral cortex is dynamically regulated throughout the life span and involves differentiated neurons. <span><span class="ref-journal">PLoS ONE. </span>2007;<span class="ref-vol">2</span>:e895.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1964879/">PMC free article</a>]</span> [<a href="/pubmed/17878930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=DNA+methylation+in+the+human+cerebral+cortex+is+dynamically+regulated+throughout+the+life+span+and+involves+differentiated+neurons&amp;author=KD+Siegmund&amp;author=CM+Connor&amp;author=M+Campan&amp;author=TI+Long&amp;author=DJ+Weisenberger&amp;volume=2&amp;publication_year=2007&amp;pages=e895&amp;pmid=17878930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B53"><span class="element-citation">Sombekke MH, Vellinga MM, Uitdehaag BM, Barkhof F, Polman CH, Arteta D, et al.  Genetic correlations of brain lesion distribution in multiple sclerosis: an exploratory study. <span><span class="ref-journal">AJNR Am J Neuroradiol. </span>2011;<span class="ref-vol">32</span>:695–703.</span> [<a href="/pubmed/21436341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=AJNR+Am+J+Neuroradiol&amp;title=Genetic+correlations+of+brain+lesion+distribution+in+multiple+sclerosis:+an+exploratory+study&amp;author=MH+Sombekke&amp;author=MM+Vellinga&amp;author=BM+Uitdehaag&amp;author=F+Barkhof&amp;author=CH+Polman&amp;volume=32&amp;publication_year=2011&amp;pages=695-703&amp;pmid=21436341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B54"><span class="element-citation">Sondag CM, Dhawan G, Combs CK. Beta amyloid oligomers and fibrils stimulate differential activation of primary microglia. <span><span class="ref-journal">J Neuroinflammation. </span>2009;<span class="ref-vol">6</span>:1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2632990/">PMC free article</a>]</span> [<a href="/pubmed/19123954" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neuroinflammation&amp;title=Beta+amyloid+oligomers+and+fibrils+stimulate+differential+activation+of+primary+microglia&amp;author=CM+Sondag&amp;author=G+Dhawan&amp;author=CK+Combs&amp;volume=6&amp;publication_year=2009&amp;pages=1&amp;pmid=19123954&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B55"><span class="element-citation">Stein JL, Hua X, Lee S, Ho AJ, Leow AD, Toga AW, et al.  Voxelwise genome-wide association study (vGWAS) <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:1160–74.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900429/">PMC free article</a>]</span> [<a href="/pubmed/20171287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Voxelwise+genome-wide+association+study+(vGWAS)&amp;author=JL+Stein&amp;author=X+Hua&amp;author=S+Lee&amp;author=AJ+Ho&amp;author=AD+Leow&amp;volume=53&amp;publication_year=2010&amp;pages=1160-74&amp;pmid=20171287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B56"><span class="element-citation">Sulem P, Gudbjartsson DF, Stacey SN, Helgason A, Rafnar T, Jakobsdottir M, et al.  Two newly identified genetic determinants of pigmentation in Europeans. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:835–7.</span> [<a href="/pubmed/18488028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Two+newly+identified+genetic+determinants+of+pigmentation+in+Europeans&amp;author=P+Sulem&amp;author=DF+Gudbjartsson&amp;author=SN+Stacey&amp;author=A+Helgason&amp;author=T+Rafnar&amp;volume=40&amp;publication_year=2008&amp;pages=835-7&amp;pmid=18488028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B57"><span class="element-citation">Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, et al.  Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=TM+Teslovich&amp;author=K+Musunuru&amp;author=AV+Smith&amp;author=AC+Edmondson&amp;author=IM+Stylianou&amp;volume=466&amp;publication_year=2010&amp;pages=707-13&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B58"><span class="element-citation">van der Sluis S, Verhage M, Posthuma D, Dolan CV. Phenotypic complexity, measurement bias, and poor phenotypic resolution contribute to the missing heritability problem in genetic association studies. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e13929.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978099/">PMC free article</a>]</span> [<a href="/pubmed/21085666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Phenotypic+complexity,+measurement+bias,+and+poor+phenotypic+resolution+contribute+to+the+missing+heritability+problem+in+genetic+association+studies&amp;author=S+van+der+Sluis&amp;author=M+Verhage&amp;author=D+Posthuma&amp;author=CV+Dolan&amp;volume=5&amp;publication_year=2010&amp;pages=e13929&amp;pmid=21085666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B59"><span class="element-citation">Vana AC, Lucchinetti CF, Le TQ, Armstrong RC. Myelin transcription factor 1 (Myt1) expression in demyelinated lesions of rodent and human CNS. <span><span class="ref-journal">Glia. </span>2007;<span class="ref-vol">55</span>:687–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2789289/">PMC free article</a>]</span> [<a href="/pubmed/17330875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Glia&amp;title=Myelin+transcription+factor+1+(Myt1)+expression+in+demyelinated+lesions+of+rodent+and+human+CNS&amp;author=AC+Vana&amp;author=CF+Lucchinetti&amp;author=TQ+Le&amp;author=RC+Armstrong&amp;volume=55&amp;publication_year=2007&amp;pages=687-97&amp;pmid=17330875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B60"><span class="element-citation">Vierbuchen T, Ostermeier A, Pang ZP, Kokubu Y, Sudhof TC, Wernig M. Direct conversion of fibroblasts to functional neurons by defined factors. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">463</span>:1035–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2829121/">PMC free article</a>]</span> [<a href="/pubmed/20107439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Direct+conversion+of+fibroblasts+to+functional+neurons+by+defined+factors&amp;author=T+Vierbuchen&amp;author=A+Ostermeier&amp;author=ZP+Pang&amp;author=Y+Kokubu&amp;author=TC+Sudhof&amp;volume=463&amp;publication_year=2010&amp;pages=1035-41&amp;pmid=20107439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B61"><span class="element-citation">Vilarino-Guell C, Wider C, Ross OA, Jasinska-Myga B, Kachergus J, Cobb SA, et al.  LINGO1 and LINGO2 variants are associated with essential tremor and Parkinson disease. <span><span class="ref-journal">Neurogenetics. </span>2010;<span class="ref-vol">11</span>:401–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3930084/">PMC free article</a>]</span> [<a href="/pubmed/20369371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurogenetics&amp;title=LINGO1+and+LINGO2+variants+are+associated+with+essential+tremor+and+Parkinson+disease&amp;author=C+Vilarino-Guell&amp;author=C+Wider&amp;author=OA+Ross&amp;author=B+Jasinska-Myga&amp;author=J+Kachergus&amp;volume=11&amp;publication_year=2010&amp;pages=401-8&amp;pmid=20369371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B62"><span class="element-citation">Vrijenhoek T, Buizer-Voskamp JE, van der Stelt I, Strengman E, Sabatti C, Geurts van Kessel A, et al.  Recurrent CNVs disrupt three candidate genes in schizophrenia patients. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:504–10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561936/">PMC free article</a>]</span> [<a href="/pubmed/18940311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Recurrent+CNVs+disrupt+three+candidate+genes+in+schizophrenia+patients&amp;author=T+Vrijenhoek&amp;author=JE+Buizer-Voskamp&amp;author=I+van+der+Stelt&amp;author=E+Strengman&amp;author=C+Sabatti&amp;volume=83&amp;publication_year=2008&amp;pages=504-10&amp;pmid=18940311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B63"><span class="element-citation">Waxman SG.  <span class="ref-journal">From neuroscience to neurology: neuroscience, molecular medicine, and the therapeutic transformation of neurology.</span> Amsterdam; Boston: Elsevier Academic Press; 2005.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=From+neuroscience+to+neurology:+neuroscience,+molecular+medicine,+and+the+therapeutic+transformation+of+neurology&amp;author=SG+Waxman&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B64"><span class="element-citation">Wood AR, Hernandez DG, Nalls MA, Yaghootkar H, Gibbs JR, Harries LW, et al.  Allelic heterogeneity and more detailed analyses of known loci explain additional phenotypic variation and reveal complex patterns of association. <span><span class="ref-journal">Hum Mol Genet. </span>2011;<span class="ref-vol">20</span>:4082–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3177649/">PMC free article</a>]</span> [<a href="/pubmed/21798870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Allelic+heterogeneity+and+more+detailed+analyses+of+known+loci+explain+additional+phenotypic+variation+and+reveal+complex+patterns+of+association&amp;author=AR+Wood&amp;author=DG+Hernandez&amp;author=MA+Nalls&amp;author=H+Yaghootkar&amp;author=JR+Gibbs&amp;volume=20&amp;publication_year=2011&amp;pages=4082-92&amp;pmid=21798870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="aws363-B65"><span class="element-citation">Wrathall JR, Li W, Hudson LD. Myelin gene expression after experimental contusive spinal cord injury. <span><span class="ref-journal">J Neurosci. </span>1998;<span class="ref-vol">18</span>:8780–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6793561/">PMC free article</a>]</span> [<a href="/pubmed/9786985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Myelin+gene+expression+after+experimental+contusive+spinal+cord+injury&amp;author=JR+Wrathall&amp;author=W+Li&amp;author=LD+Hudson&amp;volume=18&amp;publication_year=1998&amp;pages=8780-93&amp;pmid=9786985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3613709&amp;issue-id=221322&amp;journal-id=719&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139909796192752title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec11title">Results</a></li><li><a href="#__sec15title">Discussion</a></li><li><a href="#__sec16title">Funding</a></li><li><a href="#__sec17title">Supplementary material</a></li><li><a href="#idm139909716115600title">Acknowledgements</a></li><li><a href="#idm139909716114400title">Glossary</a></li><li><a href="#idm139909715586048title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Brain</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3613709/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3613709/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3613709/pdf/aws363.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC3613709" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3613709%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3613709%2F&amp;text=A%20genome-wide%20association%20study%20of%20brain%20lesion%20distribution%20in%20multiple%20sclerosis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3613709%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23412934" data-aiid="3613709" data-aid="3613709" data-iid="221322" data-domainid="719" data-domain="brain" data-accid="PMC3613709" data-md5="32c4f6188c1c3bee425badc67cb3d368"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:19:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8888B4DD5560010000000002D301DC&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3613709%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=brain&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3613709/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>