<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3080738/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Loci at 9p21 and 22q13 harbour alleles for development of cutaneous nevi and melanoma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080738/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Loci at 9p21 and 22q13 harbour alleles for development of cutaneous nevi and melanoma"><meta name="citation_authors" content="Falchi Mario, Bataille Veronique, Hayward K. Nicholas, Duffy L. David, Newton Bishop A. Julia, Pastinen Tomi, Cervino Alessandra, Zhao Z. Zhen, Deloukas Panos, Soranzo Nicole, Elder E. David, Barrett H. Jennifer, Martin G. Nicholas, Bishop D. Timothy, Montgomery W. Grant, Spector D. Timothy"><meta name="citation_date" content="August 2009"><meta name="citation_issue" content="8"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="915"><meta name="citation_doi" content="10.1038/ng.410"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3080738/?report=abstract"><meta name="citation_pmid" content="19578365"><meta name="DC.Title" content="Loci at 9p21 and 22q13 harbour alleles for development of cutaneous nevi and melanoma"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Falchi Mario"><meta name="DC.Contributor" content="Bataille Veronique"><meta name="DC.Contributor" content="Hayward K. Nicholas"><meta name="DC.Contributor" content="Duffy L. David"><meta name="DC.Contributor" content="Newton Bishop A. Julia"><meta name="DC.Contributor" content="Pastinen Tomi"><meta name="DC.Contributor" content="Cervino Alessandra"><meta name="DC.Contributor" content="Zhao Z. Zhen"><meta name="DC.Contributor" content="Deloukas Panos"><meta name="DC.Contributor" content="Soranzo Nicole"><meta name="DC.Contributor" content="Elder E. David"><meta name="DC.Contributor" content="Barrett H. Jennifer"><meta name="DC.Contributor" content="Martin G. Nicholas"><meta name="DC.Contributor" content="Bishop D. Timothy"><meta name="DC.Contributor" content="Montgomery W. Grant"><meta name="DC.Contributor" content="Spector D. Timothy"><meta name="DC.Date" content="2009 Aug"><meta name="DC.Identifier" content="10.1038/ng.410"><meta name="DC.Language" content="en"><meta property="og:title" content="Loci at 9p21 and 22q13 harbour alleles for development of cutaneous nevi and melanoma"><meta property="og:type" content="article"><meta property="og:description" content="High number of melanocytic nevi is the most important known risk factor for cutaneous melanoma. We conducted a genome-wide association study for nevus count using 297,108 tag-SNPs in 1,524 twins and validated our results in an independent cohort of 4,107 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3080738/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3080738/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3080738/pdf/nihms131115.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D4E61DD4FEE3100000000005A003A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080738%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3080738</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2011 Apr 21.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19578365" target="pmc_ext" ref="reftype=publisher&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Aug; 41(8): 915–919. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Jul 5. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.410" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.410</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3080738</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS131115</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19578365">19578365</a></div></div></div></div><h1 class="content-title">Loci at 9p21 and 22q13 harbour alleles for development of cutaneous nevi and melanoma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Mario%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878966924400" co-class="co-affbox">Falchi Mario</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Veronique%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878972322512" co-class="co-affbox">Bataille Veronique</a>,<sup>1,</sup><sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Nicholas%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878972758192" co-class="co-affbox">Hayward K. Nicholas</a>,<sup>4,</sup><sup>*</sup> <a href="/pubmed/?term=David%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878975360992" co-class="co-affbox">Duffy L. David</a>,<sup>5</sup> <a href="/pubmed/?term=Julia%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878967425968" co-class="co-affbox">Newton Bishop A. Julia</a>,<sup>6</sup> <a href="/pubmed/?term=Tomi%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878976415712" co-class="co-affbox">Pastinen Tomi</a>,<sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Alessandra%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878976053488" co-class="co-affbox">Cervino Alessandra</a>,<sup>1</sup> <a href="/pubmed/?term=Zhen%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878969571280" co-class="co-affbox">Zhao Z. Zhen</a>,<sup>9</sup> <a href="/pubmed/?term=Panos%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878971067712" co-class="co-affbox">Deloukas Panos</a>,<sup>10</sup> <a href="/pubmed/?term=Nicole%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878970949424" co-class="co-affbox">Soranzo Nicole</a>,<sup>1,</sup><sup>10</sup> <a href="/pubmed/?term=David%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878975809968" co-class="co-affbox">Elder E. David</a>,<sup>11</sup> <a href="/pubmed/?term=Jennifer%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878972536368" co-class="co-affbox">Barrett H. Jennifer</a>,<sup>6</sup> <a href="/pubmed/?term=Nicholas%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878974280480" co-class="co-affbox">Martin G. Nicholas</a>,<sup>5</sup> <a href="/pubmed/?term=Timothy%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878957389664" co-class="co-affbox">Bishop D. Timothy</a>,<sup>6,</sup><sup>*</sup> <a href="/pubmed/?term=Grant%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878965559088" co-class="co-affbox">Montgomery W. Grant</a>,<sup>9,</sup><sup>*</sup> and  <a href="/pubmed/?term=Timothy%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365" class="affpopup" co-rid="_co_idm139878970727664" co-class="co-affbox">Spector D. Timothy</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139878966924400"><h3 class="no_margin">Falchi Mario</h3><p><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</p><p><sup>2</sup> Genomic Medicine, Hammersmith Hospital, Imperial College London, W12 0NN, London, UK</p><div>Find articles by <a href="/pubmed/?term=Mario%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Falchi Mario</a></div></div><div id="_co_idm139878972322512"><h3 class="no_margin">Bataille Veronique</h3><p><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</p><p><sup>3</sup> Dermatology Department, West Hertfordshire NHS Trust, Hemel Hempstead General Hospital, Herts, UK</p><div>Find articles by <a href="/pubmed/?term=Veronique%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Bataille Veronique</a></div></div><div id="_co_idm139878972758192"><h3 class="no_margin">Hayward K. Nicholas</h3><p><sup>4</sup> Oncogenomics, Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Nicholas%20HK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Hayward K. Nicholas</a></div></div><div id="_co_idm139878975360992"><h3 class="no_margin">Duffy L. David</h3><p><sup>5</sup> Genetic Epidemiology, Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=David%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Duffy L. David</a></div></div><div id="_co_idm139878967425968"><h3 class="no_margin">Newton Bishop A. Julia</h3><p><sup>6</sup> Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, U.K.</p><div>Find articles by <a href="/pubmed/?term=Julia%20NB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Newton Bishop A. Julia</a></div></div><div id="_co_idm139878976415712"><h3 class="no_margin">Pastinen Tomi</h3><p><sup>7</sup> Department of Human and Medical Genetics, McGill University, Montréal H3A 1B1, Canada</p><p><sup>8</sup> McGill University and Genome Québec Innovation Centre, Montréal H3A 1A4, Canada</p><div>Find articles by <a href="/pubmed/?term=Tomi%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Pastinen Tomi</a></div></div><div id="_co_idm139878976053488"><h3 class="no_margin">Cervino Alessandra</h3><p><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Alessandra%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Cervino Alessandra</a></div></div><div id="_co_idm139878969571280"><h3 class="no_margin">Zhao Z. Zhen</h3><p><sup>9</sup> Molecular Epidemiology, Queensland Institute of Medical Research, Royal Brisbane Hospital, 300 Herston Rd, Brisbane, QLD 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Zhen%20ZZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Zhao Z. Zhen</a></div></div><div id="_co_idm139878971067712"><h3 class="no_margin">Deloukas Panos</h3><p><sup>10</sup> Human Genetics Department, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</p><div>Find articles by <a href="/pubmed/?term=Panos%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Deloukas Panos</a></div></div><div id="_co_idm139878970949424"><h3 class="no_margin">Soranzo Nicole</h3><p><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</p><p><sup>10</sup> Human Genetics Department, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</p><div>Find articles by <a href="/pubmed/?term=Nicole%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Soranzo Nicole</a></div></div><div id="_co_idm139878975809968"><h3 class="no_margin">Elder E. David</h3><p><sup>11</sup> Dept of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, USA</p><div>Find articles by <a href="/pubmed/?term=David%20EE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Elder E. David</a></div></div><div id="_co_idm139878972536368"><h3 class="no_margin">Barrett H. Jennifer</h3><p><sup>6</sup> Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, U.K.</p><div>Find articles by <a href="/pubmed/?term=Jennifer%20BH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Barrett H. Jennifer</a></div></div><div id="_co_idm139878974280480"><h3 class="no_margin">Martin G. Nicholas</h3><p><sup>5</sup> Genetic Epidemiology, Queensland Institute of Medical Research, Brisbane, Australia</p><div>Find articles by <a href="/pubmed/?term=Nicholas%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Martin G. Nicholas</a></div></div><div id="_co_idm139878957389664"><h3 class="no_margin">Bishop D. Timothy</h3><p><sup>6</sup> Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, U.K.</p><div>Find articles by <a href="/pubmed/?term=Timothy%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Bishop D. Timothy</a></div></div><div id="_co_idm139878965559088"><h3 class="no_margin">Montgomery W. Grant</h3><p><sup>9</sup> Molecular Epidemiology, Queensland Institute of Medical Research, Royal Brisbane Hospital, 300 Herston Rd, Brisbane, QLD 4029, Australia</p><div>Find articles by <a href="/pubmed/?term=Grant%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Montgomery W. Grant</a></div></div><div id="_co_idm139878970727664"><h3 class="no_margin">Spector D. Timothy</h3><p><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</p><div>Find articles by <a href="/pubmed/?term=Timothy%20SD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578365">Spector D. Timothy</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139878966163072_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139878966163072_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139878966163072_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Dept of Twin Research &amp; Genetic Epidemiology, Kings College London, London, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Genomic Medicine, Hammersmith Hospital, Imperial College London, W12 0NN, London, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Dermatology Department, West Hertfordshire NHS Trust, Hemel Hempstead General Hospital, Herts, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Oncogenomics, Queensland Institute of Medical Research, Brisbane, Australia</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Genetic Epidemiology, Queensland Institute of Medical Research, Brisbane, Australia</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Section of Epidemiology and Biostatistics, Leeds Institute of Molecular Medicine, University of Leeds, Leeds, U.K.</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Department of Human and Medical Genetics, McGill University, Montréal H3A 1B1, Canada</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> McGill University and Genome Québec Innovation Centre, Montréal H3A 1A4, Canada</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Molecular Epidemiology, Queensland Institute of Medical Research, Royal Brisbane Hospital, 300 Herston Rd, Brisbane, QLD 4029, Australia</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Human Genetics Department, Wellcome Trust Sanger Institute, Hinxton, Cambridge CB10 1SA, UK</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Dept of Pathology and Laboratory Medicine, University of Pennsylvania, Philadelphia, USA</div><div id="FN1">Corresponding authors: Timothy D Spector, Dept of Twin Research &amp; Genetic Epidemiology, St Thomas' Hospital, KCL, London SE1 7EH, <a href="mailto:dev@null" data-email="ku.ca.lck@rotceps.miT" class="oemail">ku.ca.lck@rotceps.miT</a>, 0044-207-188-6765, Mario Falchi, Genomic Medicine, Hammersmith Hospital, Imperial College London, W12 0NN, London, UK, <a href="mailto:dev@null" data-email="ku.ca.lairepmi@ihclaf.m" class="oemail">ku.ca.lairepmi@ihclaf.m</a>, 0044-207-594-6519</div><div id="FN2"><sup>*</sup>Equal authorship contributions</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139878966163072_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19578365" target="pmc_ext" ref="reftype=publisher&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3080738/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary.</strong></div><div><a href="/pmc/articles/PMC3080738/bin/NIHMS131115-supplement-Supplementary.pdf" data-ga-action="click_feat_suppl">NIHMS131115-supplement-Supplementary.pdf</a><span style="color:gray"> (140K)</span></div><div class="small guid">GUID:&nbsp;C3BD0FC1-244A-4A9D-9B63-F5EE15382FB7</div></div></dd></dl></div><div id="idm139878969279744" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139878969279744title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">High number of melanocytic nevi is the most important known risk factor for cutaneous melanoma. We conducted a genome-wide association study for nevus count using 297,108 tag-SNPs in 1,524 twins and validated our results in an independent cohort of 4,107 subjects. We identified strongly associated variants in <em>MTAP</em>, a gene adjacent to the familial melanoma susceptibility locus <em>CDKN2A</em> on 9p21 (rs4636294, P = 3.4 × 10<sup>-15</sup>). We further identified <em>PLA2G6</em> on 22q13.1 (rs2284063, P = 3.4 × 10<sup>-8</sup>). Both loci were also associated with melanoma risk in 3,131 melanoma cases from two independent studies (odds-ratios 1.23 at rs10757257 and rs132985). About one subject in 11 is homozygous for the variant at both loci with twice the number of nevi compared to those homozygous for the protective alleles, and double the risk for melanoma. These data provide the first evidence for common melanoma alleles whose effects are mediated through nevus number.</p></div></div><div id="idm139878957356416" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">The worldwide incidence of cutaneous melanoma in European descent populations has risen rapidly over the past 30 years, more than for any other malignancy [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644425">1</a>]. The World Health Organization estimates 132,000 new cases of melanoma per year globally. Approximately 10% of malignant melanoma cases occur in multiplex families, with the <em>CDKN2A</em> locus in 9p21 accounting for susceptibility in 20-57% of all melanoma families [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644430">2</a>]. However, remaining cases of familial melanoma do not carry mutations in <em>CDKN2A</em> and additional tumor suppressor genes at 9p21 and elsewhere have been proposed [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644422">3</a>]. Large numbers of nevi are the strongest known risk factor for melanoma [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644435">4</a>] and in many familial cases susceptibility to melanoma is associated with an excess of nevi. Linkage to 9p21 has been found in two genome-wide analyses looking at nevus count [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644420">6</a>,<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644423">7</a>] suggesting that shared genetic factors might be involved in melanoma susceptibility and nevogenesis. Understanding the genetic bases of nevus formation is therefore an important step in understanding melanoma etiology.</p><p id="P3">To search for loci involved in nevus count, we carried out a genome-wide association study (GWAS) in 1,524 healthy adult female twins from the TwinsUK registry using 297,108 SNPs with minor allele frequency (MAF) &gt; 1% on Illumina HumanHap 300k duo chips (See Methods). Total body nevus count was evaluated by trained research nurses and defined as the sum of all nevi at least 2 mm in diameter. Comparison of the observed and expected distributions (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>) showed no evidence for inflation of the test statistics (inflation factor = 1.006) and highlighted 46 SNPs having P values &lt; 10<sup>-4</sup> against the expected 29. We selected twelve SNPs for follow up that clustered in either the 9p21 or 22q13 chromosomal regions (<a href="/pmc/articles/PMC3080738/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). We replicated these SNP associations in an independent sample of 4,107 adolescent subjects of European ancestry from the Brisbane Twin Nevus Study (BTNS) for whom nevus count had been evaluated using a similar protocol (See Methods).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3080738/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878970480832" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131115f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131115f1.jpg" src="/pmc/articles/PMC3080738/bin/nihms131115f1.jpg"></div></a><div id="largeobj_idm139878970480832" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3080738/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3080738/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3080738/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Genome-wide plot for total nevus count in the TwinsUK sample. The SNPs typed and replicated in the Brisbane Adolescent Nevus Study sample are highlighted.</p></div></div></div><p id="P4">At the replication stage (<a href="/pmc/articles/PMC3080738/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) the strongest association signal was observed for rs4636294 (combined P = 3.4 × 10<sup>-15</sup>). This marker is located in the 5′ UTR of the methylthioadenosine phosphorylase (<em>MTAP</em>) gene located at 9p21, next to <em>CDKN2A</em> (<a href="/pmc/articles/PMC3080738/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). The other replicated region on 22q13 (<a href="/pmc/articles/PMC3080738/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2b</span></a>) showed the highest combined association signal with rs2284063 (combined P = 3.4 × 10<sup>-8</sup>). This SNP lies in the intron between the second and third exon of <em>PLA2G6</em>, belonging to the phospholipase A2 (PLA2) superfamily of genes [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355525257">8</a>]. The 9p21 and 22q13 loci accounted for 1.5% (UK) and 3.0% (Australia), and for 1.9% (UK) and 0.7% (Australia) of nevus count variance, respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3080738/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878975257264" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131115f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131115f2.jpg" src="/pmc/articles/PMC3080738/bin/nihms131115f2.jpg"></div></a><div id="largeobj_idm139878975257264" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3080738/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC3080738/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3080738/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Regional plot for the chromosomes 9p21 (a) and 22q13 (b). Meta analysis log<sub>10</sub> P values are plotted as a function of genomic position (Build 36). The P values for the lead SNPs are denoted by large blue (combined discovery and replication) diamonds. Proxies are indicated with diamonds of smaller size, with colours assigned based on the pairwise r<sup>2</sup> values with the lead SNP in the HapMap CEU sample: red (r<sup>2</sup>&gt;0.8), orange (0.5&lt;r<sup>2</sup>&lt;0.8) or yellow (0.2&lt;r<sup>2</sup>&lt;0.5). White indicates either no LD with the lead SNP (r<sup>2</sup>&lt;0.2), or loci where such information was not available. Recombination rate estimates (HapMap Phase II) are given in light blue, Refseq genes (NCBI) in green.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Total nevus count: summary association results and follow-up for the 9p21 and 22q13 regions. Effect size (beta) and standard error (SE) are expressed in log (nevi). The meta-analytic P values are calculated using a weighted Z-score method.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878967743984" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Marker</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">MAF Allele TwinsUK</th><th rowspan="1" colspan="1" valign="top" align="left">Beta (SE) TwinsUK</th><th rowspan="1" colspan="1" valign="top" align="left">P TwinsUK</th><th rowspan="1" colspan="1" valign="top" align="left">Beta (SE) BTNS <sup>*</sup></th><th rowspan="1" colspan="1" valign="top" align="left">P BTNS</th><th rowspan="1" colspan="1" valign="top" align="left">Combined P</th><th rowspan="1" colspan="1" valign="top" align="left">Heterogeneity P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4636294</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.48 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">0.188 (0.042)</td><td rowspan="1" colspan="1" valign="top" align="left">9.8 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.197 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">6.6 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2218220</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.48 (C)</td><td rowspan="1" colspan="1" valign="top" align="left">0.188 (0.043)</td><td rowspan="1" colspan="1" valign="top" align="left">1.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.194 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.7 × 10<sup>-15</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.57</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs751173</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.46 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">0.166 (0.042)</td><td rowspan="1" colspan="1" valign="top" align="left">8.5 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.168 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">9.9 × 10<sup>-14</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.95</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7023329</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.50 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">0.184 (0.042)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.164 (0.036)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10757257</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.40 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.205 (0.042)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.147 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.9 × 10<sup>-11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1335510</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.41 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.198 (0.043)</td><td rowspan="1" colspan="1" valign="top" align="left">3.9 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.145 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">5.3 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.2 × 10<sup>-10 10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1341866</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.41 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.189 (0.043)</td><td rowspan="1" colspan="1" valign="top" align="left">1.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.143 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">5.2 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6.2 × 10<sup>-10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10811629</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">0.41 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.175 (0.042)</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.089 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">4.7 × 10<sup>-3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.3 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2284063</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">0.35 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.209 (0.045)</td><td rowspan="1" colspan="1" valign="top" align="left">2.3 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.078 (0.039)</td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6001027</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">0.35 (G)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.209 (0.045)</td><td rowspan="1" colspan="1" valign="top" align="left">3.8 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.078 (0.039)</td><td rowspan="1" colspan="1" valign="top" align="left">3.6 × 10<sup>-4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.0 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.04</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs132985</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">0.46 (T)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.209 (0.043)</td><td rowspan="1" colspan="1" valign="top" align="left">1.0 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.055 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">1.5 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.0 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">4.0 × 10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs738322</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">0.47 (A)</td><td rowspan="1" colspan="1" valign="top" align="left">-0.209 (0.043)</td><td rowspan="1" colspan="1" valign="top" align="left">1.4 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">-0.056 (0.037)</td><td rowspan="1" colspan="1" valign="top" align="left">4.6 × 10<sup>-2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.5 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.0 × 10<sup>-3</sup></td></tr></tbody></table></div><div id="largeobj_idm139878967743984" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080738/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>BTNS Beta and SE are reported on the log scale to make them comparable to the TwinsUK results. P values have been evaluated using cube-root transformation which better fits the BTNS data (See Methods).</div></div></div><p id="P5">The combined additive effect of rs4636294 and rs2284063 markedly increases the observed number of nevi. Indeed, subjects in the TwinsUK discovery sample homozygous for both variants (7%) had more than double the number of nevi (median number of nevi = 30) compared to subjects homozygous for the protective alleles (median number of nevi = 12.5). Subjects in the Australian BTNS sample are characterized by an overall larger number of nevi (median number of nevi = 93 compared to 21 in the TwinsUK sample). Homozygous subjects for both variants in the Australian population (11%) also showed greater median number of nevi (116) than homozygous individuals for the protective alleles (79) (<a href="/pmc/articles/PMC3080738/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Sun exposure varies greatly between the UK and Australian samples. The number of sunburns over a lifetime and the skin type (measured according to the Fitzpatrick classification) were not related to the number of nevi in either sample (see also <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> for the distributions in the two countries). Therefore, adjusting for these two measures did not affect our results.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3080738/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878974946752" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms131115f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms131115f3.jpg" src="/pmc/articles/PMC3080738/bin/nihms131115f3.jpg"></div></a><div id="largeobj_idm139878974946752" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3080738/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC3080738/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3080738/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Box-plot of number of nevi for subjects in the TwinsUK (left) and BNTS (right) samples homozygous for both <em>MTAP</em> SNP rs4636294 and <em>PLA2G6</em> SNP rs2284063 variants against the subjects homozygous for the protective alleles.</p></div></div></div><p id="P6">We explored whether these two genetic regions were also involved directly or indirectly in melanoma susceptibility by analysing two melanoma case-control samples: the Queensland study of Melanoma, Environment and Genetic Association (Q-MEGA) which included 1,734 cases and 1,811 controls of northern European origin, and the Leeds melanoma case-control study of North UK including 1,274 incident melanoma cases, 123 cases with a family history of melanoma, and 496 controls plus 1,395 controls from the Wellcome Trust Case-Control Consortium (WTCCC) 1958 birth cohort samples. (Details are provided in Methods).</p><p id="P7">We derived joint odds ratios (OR) and confidence intervals under a fixed-effects model for each SNP, and associated P values from the standard normal distribution. As expected, we observed a small degree of heterogeneity for some of the SNPs shown by the lack of complete agreement for the highest association signals in the two samples. The highest combined ORs were 1.23 (95% CI = 1.15 - 1.30) for rs10757257 at 9p21 and 1.23 (95% CI = 1.15 - 1.30) for rs132985 at 22q13 (<a href="/pmc/articles/PMC3080738/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). ORs at the SNPs giving the most significant results in the nevus count analyses were 1.21 (95% CI = 1.14 – 1.28) for rs4636294, and 1.19 (95% CI = 1.12 – 1.27) for rs2284063. Population attributable risk of the two loci reflected their different effect in the Australian and UK populations, ranging from 20.0% (Australia) to 9.0% (UK) for 9p21 and from 14.8 (Australia) to 29.3 (UK) for 22q13.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Melanoma: summary results for the lead SNPs from the 9p21 and 22q13 loci for cutaneous melanoma. Joint odds ratios and confidence intervals were evaluated under a fixed-effects model for each SNP. Associated P values were estimated from the standard normal distribution.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878973324800" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th colspan="3" rowspan="1" valign="top" align="left"></th><th colspan="2" rowspan="1" valign="top" align="left">Allele reference</th><th colspan="2" rowspan="1" valign="top" align="left">Risk allele frequency</th><th colspan="4" rowspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="right"></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Marker</th><th rowspan="1" colspan="1" valign="top" align="left">Chromosome</th><th rowspan="1" colspan="1" valign="top" align="left">Sample</th><th rowspan="1" colspan="1" valign="top" align="left">Base</th><th rowspan="1" colspan="1" valign="top" align="left">Risk</th><th rowspan="1" colspan="1" valign="top" align="left">Controls</th><th rowspan="1" colspan="1" valign="top" align="left">Cases</th><th rowspan="1" colspan="1" valign="top" align="left">OR per risk allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">P</th><th rowspan="1" colspan="1" valign="top" align="left">Combined OR per risk allele (95% CI)</th><th rowspan="1" colspan="1" valign="top" align="left">Combined P</th><th rowspan="1" colspan="1" valign="top" align="right">Heterogeneity P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4636294</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS2</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.48<br>0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.55<br>0.51</td><td rowspan="1" colspan="1" valign="top" align="left">1.31 (1.19, 1.46)<br>1.11 (1.01, 1.22)</td><td rowspan="1" colspan="1" valign="top" align="left">1.7 × 10<sup>-7</sup><br>0.029</td><td rowspan="1" colspan="1" valign="top" align="left">1.21 (1.14, 1.28)</td><td rowspan="1" colspan="1" valign="top" align="left">3.7 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">4.0 × 10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2218220</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS2</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.49<br>0.48</td><td rowspan="1" colspan="1" valign="top" align="left">0.54<br>0.51</td><td rowspan="1" colspan="1" valign="top" align="left">1.29 (1.17, 1.44)<br>1.13 (1.03, 1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">5.7 × 10<sup>-7</sup><br>0.011</td><td rowspan="1" colspan="1" valign="top" align="left">1.21 (1.14, 1.27)</td><td rowspan="1" colspan="1" valign="top" align="left">4.8 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7023329</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS2</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.50<br>0.49</td><td rowspan="1" colspan="1" valign="top" align="left">0.54<br>0.53</td><td rowspan="1" colspan="1" valign="top" align="left">1.22 (1.11, 1.37)<br>1.17 (1.06, 1.29)</td><td rowspan="1" colspan="1" valign="top" align="left">1.2 × 10<sup>-4</sup><br>0.001</td><td rowspan="1" colspan="1" valign="top" align="left">1.20 (1.13, 1.26)</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.48</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10757257</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.58<br>0.58</td><td rowspan="1" colspan="1" valign="top" align="left">0.64<br>0.62</td><td rowspan="1" colspan="1" valign="top" align="left">1.30 (1.17, 1.44)<br>1.16 (1.05, 1.28)</td><td rowspan="1" colspan="1" valign="top" align="left">7.7 × 10<sup>-7</sup><br>0.003</td><td rowspan="1" colspan="1" valign="top" align="left">1.23 (1.15, 1.30)</td><td rowspan="1" colspan="1" valign="top" align="left">3.4 × 10<sup>-8</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs751173</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.46<br>0.46</td><td rowspan="1" colspan="1" valign="top" align="left">0.51<br>0.48</td><td rowspan="1" colspan="1" valign="top" align="left">1.28 (1.16, 1.42)<br>1.07 (0.97, 1.18)</td><td rowspan="1" colspan="1" valign="top" align="left">1.9 × 10<sup>-6</sup><br>0.152</td><td rowspan="1" colspan="1" valign="top" align="left">1.17 (1.11, 1.24)</td><td rowspan="1" colspan="1" valign="top" align="left">5.6 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="right">3.0 × 10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1335510</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0.58<br>0.57</td><td rowspan="1" colspan="1" valign="top" align="left">0.63<br>0.60</td><td rowspan="1" colspan="1" valign="top" align="left">1.26 (1.14, 1.40)<br>1.12 (1.02, 1.24)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>-5</sup><br>0.020</td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.12, 1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1341866</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.58<br>0.58</td><td rowspan="1" colspan="1" valign="top" align="left">0.63<br>0.60</td><td rowspan="1" colspan="1" valign="top" align="left">1.24 (1.13, 1.39)<br>1.13 (1.02, 1.25)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-5</sup><br>0.015</td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.12, 1.26)</td><td rowspan="1" colspan="1" valign="top" align="left">6.9 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10811629</td><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.57<br>0.56</td><td rowspan="1" colspan="1" valign="top" align="left">0.61<br>0.62</td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.08, 1.32)<br>1.28 (1.06, 1.55)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3 × 10<sup>-4</sup><br>0.010</td><td rowspan="1" colspan="1" valign="top" align="left">1.21 (1.12, 1.30)</td><td rowspan="1" colspan="1" valign="top" align="left">1.6 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.41</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2284063</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS2</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.65<br>0.63</td><td rowspan="1" colspan="1" valign="top" align="left">0.66<br>0.69</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 (0.95, 1.19)<br>1.30 (1.18, 1.44)</td><td rowspan="1" colspan="1" valign="top" align="left">0.275<br>2.3 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.12, 1.27)</td><td rowspan="1" colspan="1" valign="top" align="left">1.5 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="right">1.0 × 10<sup>-3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6001027</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS2</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.65<br>0.64</td><td rowspan="1" colspan="1" valign="top" align="left">0.66<br>0.69</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 (0.95, 1.19)<br>1.25 (1.13, 1.38)</td><td rowspan="1" colspan="1" valign="top" align="left">0.275<br>2.0 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.16 (1.09, 1.24)</td><td rowspan="1" colspan="1" valign="top" align="left">6.8 × 10<sup>-5</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs132985</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.53<br>0.52</td><td rowspan="1" colspan="1" valign="top" align="left">0.56<br>0.58</td><td rowspan="1" colspan="1" valign="top" align="left">1.18 (1.05, 1.30)<br>1.26 (1.14, 1.38)</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 × 10<sup>-3</sup><br>2.8 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.23 (1.15, 1.30)</td><td rowspan="1" colspan="1" valign="top" align="left">2.6 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs738322</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Q-MEGA<br>LEEDS1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.53<br>0.52</td><td rowspan="1" colspan="1" valign="top" align="left">0.55<br>0.57</td><td rowspan="1" colspan="1" valign="top" align="left">1.18 (1.05, 1.30)<br>1.23 (1.12, 1.36)</td><td rowspan="1" colspan="1" valign="top" align="left">2.8 × 10<sup>-3</sup><br>1.4 × 10<sup>-6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.21 (1.14, 1.28)</td><td rowspan="1" colspan="1" valign="top" align="left">3.0 × 10<sup>-7</sup></td><td rowspan="1" colspan="1" valign="top" align="right">0.51</td></tr></tbody></table></div><div id="largeobj_idm139878973324800" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3080738/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P8">We used the Leeds sample to evaluate the combined risk for melanoma at both loci (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). Subjects homozygous at the two loci have double the risk (OR = 1.94; 95% CI = 1.19 – 3.16).</p><p id="P9">From the Leeds case-control study 903 cases and 485 controls were assessed for their nevus phenotype through a standardised nevus count protocol, similar to that used in the TwinsUK sample. We used this sub-sample to explore whether the association between the identified loci and melanoma was direct or mediated by nevus count. Using these 485 controls we verified that the effect of the nevus count loci was consistent with what was observed with the TwinsUK and BTNS samples. The results using this sub-sample were still consistent with those reported in <a href="/pmc/articles/PMC3080738/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. Odd ratios for melanoma status were then compared with those obtained when the number of nevi was included in the logistic regression (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). When the association between melanoma and SNP was adjusted for nevus count, the OR for melanoma risk was reduced toward 1.0 and the P value became less significant, consistent with melanoma risk being mediated via nevus count. It is notable that the OR was midway between the unadjusted OR and 1.0 which might be expected as nevus count would not be a perfect measure of risk (including error in its measurement). Indeed, nevus count is likely modified by other genetic influences as well as lifestyle factors. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a> online shows the distribution of nevus count in the Leeds case-control sample stratified by genotype for the highest associations on 9p21 and 22q13.</p><p id="P10">The location of the key SNPs in the 9p21 region is intriguing. They lie in the proximal promoter, 5′UTR and upstream from the coding region of the house keeping gene <em>MTAP</em>, which encodes a key enzyme in the adenine and methionine salvage pathways. <em>MTAP</em> is often co-deleted [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644428">9</a>,<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644413">10</a>] with the adjacent melanoma susceptibility locus <em>CDKN2A</em>, which encodes p16INK4A and p14ARF (<a href="/pmc/articles/PMC3080738/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). The loss of <em>MTAP</em> in some tumors has generally been attributed to its proximity to <em>CDKN2A</em>, but some studies suggest <em>MTAP</em> might function as an independent tumor suppressor [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644439">11</a>-<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644424">12</a>]. A novel fusion transcript has been reported which comprises the first four exons of <em>MTAP</em> plus exons 2 and 3 of the non-coding RNA <em>ANRIL</em>, which has its first exon in the <em>CDKN2A</em> promoter [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644416">13</a>]. This points to complex regulation of several genes within this region (<a href="/pmc/articles/PMC3080738/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). A possible hypothesis parallels common cancer susceptibility variants found earlier where long range regulatory effects on established oncogenes (<em>MYC</em> and <em>FAM84B</em>) have been postulated as a possible mechanism underlying multiple cancer associations mapping to a 1.2 Mb gene desert on 8q24 [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644436">14</a>]</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3080738/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139878973520048" class="figure"><a href="/pmc/articles/PMC3080738/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3080738_nihms131115f4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms131115f4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3080738/bin/nihms131115f4.jpg"></a></div><div id="largeobj_idm139878973520048" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3080738/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3080738/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">Relative positions of genes in the vicinity of the 9p21 association signal (not drawn to scale). Shaded boxes represent exons (ex), which are numbered, and solid lines connecting exons show the normal splicing patterns. Dotted lines between exons (above the figure) denote alternative splicing that generates an MTAP-ANRIL fusion transcript. Arrows indicate the direction of transcription. Gene product names are indicated. Tel (telomere); cen (centromere).</p></div></div></div><p id="P11">The <em>PLA2G6</em> gene on 22p13 belongs to the PLA2 superfamily of genes which encode esterases that cleave glycerophospholipids [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355525258">8</a>]. These enzymes are normally involved in the maintenance of membrane phospholipids, but have recently been shown to regulate cell growth [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_412417693">15</a>] and apoptosis [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644417">16</a>], cell proliferation in human tumors such as insulinoma, colon cancer, ovarian cancer, and prostate cancer [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644432">17</a>-<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644418">19</a>] and to be associated with lung cancer susceptibility [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644415">20</a>].</p><p id="P12">We attempted to refine the location of chromosomes 9 and 22 association signals by imputing genotypes from the HapMap in the TwinsUK sample, and identified ten and four imputed SNPs on 9p21 and 22q13 respectively, which showed P values smaller than the strongest peak observed with the typed genotypes and might be helpful to identify some causal variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>).</p><p id="P13" class="p">In the same issue of this journal Bishop et al [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_355525256">21</a>] report a genome-wide association study for melanoma conducted by the GenoMEL consortium. Melanoma cases from Europe and Australia were carefully selected in order to increase the probability that they were enriched for genetic susceptibility. Three associated regions were replicated in two independent samples, one including cases selected for being genetically enriched while the other one was an unselected population-based case-control study. Among the replicated regions they identified the same locus on chromosome 9p21 described in this study, with top hit at rs7023329 (P = 4.03×10<sup>-7</sup>). The authors also successfully attempted replication of our association on 22q13 at rs2284063 (P = 2.40×10<sup>-9</sup>). The GenoMEL study involves melanoma cases and controls of diverse European ancestry, so the results presented in this paper generalise beyond English and Anglo-Celtic (Australian) ancestry. Understanding how these loci influence the development of nevi could provide important therapeutic clues to treat melanoma but may also be relevant for other cancers.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Subjects and phenotypes</h3><div id="S3" class="sec sec-first-last"><p></p><h4 id="S3title" class="inline">TwinsUK </h4><p id="P14" class="p p-first">The St. Thomas' U.K. adult twin registry (TwinsUK) cohort is unselected for any disease and is representative of the general UK population. Nevus counts were collected at St Thomas Hospital in London and the study was approved by the St Thomas' Hospital Research Ethics Committee. Written informed consent was obtained from every participant to the study. Examination was performed by trained research nurses following a standardized and reproducible nevus count protocol [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644431">22</a>]. The total body nevus count (excluding the genital area, breasts, and posterior scalp) was defined as the sum of all nevi &gt;2 mm in diameter.</p><p id="P15" class="p p-last">Genotypic data were available for 1,524 subjects: 206 singletons, 832 dizygous (DZ) and 486 monozygous (MZ) twins. The median (1st - 3rd) age was 47 (37 – 54). The median (1st - 3rd) number of nevi was 21 (8 – 47).</p></div></div><div id="S4" class="sec"><h3 id="S4title">BTNS and Q-MEGA</h3><p id="P16" class="p p-first">The Brisbane Twin Nevus Study (BTNS) is described in detail elsewhere [<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644411">23</a>] and comprised 4,107 adolescent twins, their siblings and parents (1,148 nuclear families) which have been recruited over sixteen years into an ongoing study of genetic and environmental factors contributing to the development of nevi and other risk factors for skin cancer. The twins are recruited at age twelve years via schools around Brisbane, Australia, and followed up at age fourteen. All controls are screened to be unaffected by cutaneous malignant melanoma (CMM). The sample is overwhelmingly (&gt;95%) of northern European origin (mainly Anglo-Celtic).</p><p id="P17">Total body nevus counting has been performed on the adolescent twins and singleton siblings on two occasions by a trained nurse (at ages 12 and 14). There were 1,979 twins and singleton siblings where nevus counts and genotypes were available. Of the parents of the twins, 1,470 had self-reported nevus score and genotypes available.</p><p id="P18">The Australian melanoma cases (Q-MEGA) were a stratified sample of 1,734 subjects including all cases of melanoma diagnosed in the state of Queensland in the period 1982-1990, as described in detail elsewhere [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_322960508">24</a>]. These individuals were originally studied between 1991-1994, but were recontacted and interviewed in 2002-2004.</p><p id="P19" class="p p-last">Controls for the present analysis come from the parents of the BTNS families. An additional 658 individuals were genotyped and contributed as controls for the melanoma case-control analysis. All cases and controls gave informed consent to participation in this study, and the study protocol was approved by appropriate institutional review boards.</p></div><div id="S5" class="sec"><h3 id="S5title">Leeds Case-Control Study</h3><p id="P20" class="p p-first">1,274 population based incident melanoma cases were recruited in a geographically defined area of Yorkshire and the Northern region of the UK. Overall, the study had a 63% response rate. Patients were diagnosed between September 2000 and December 2006. Cases were identified by clinicians, pathology registers and via the Northern and Yorkshire Cancer Registry and Information Service cancer registry to ensure overall ascertainment. For the first three years of the study all patients with invasive melanoma were invited to participate while subsequently only patients with Breslow thickness of at least 0.75 mm were invited as cases. The case series was supplemented by 123 melanoma cases with a family history of melanoma (at most two affected relatives) who were known to be negative for a <em>CDKN2A</em> mutation. <em>CDKN2A</em> is the most common gene mutated in the germline of multicase melanoma families; such mutations are infrequent in 3 case families in the UK being found in an estimated 21% of families (data not shown).</p><p id="P21">Controls were ascertained by contacting general practitioners to identify eligible individuals. These controls were frequency matched with cases for age and sex from general practitioners who had also had cases as part of their patient register. Overall there was a 55% response rate among controls (496 subjects). Additional controls were supplemented by a population-based group of 574 women who following informed consent agreed to participate in a study involving recording their history of sun exposure and sun bathing including sun bed usage and for whom various measures of skin aging were recorded. DNA samples were also provided.</p><p id="P22">All participating individuals gave written informed consent and regional ethical committee approval was obtained.</p><p id="P23" class="p p-last">The first 903 cases and all the controls were asked about UV exposure and other aspects of lifestyle and provided detailed information on the history of residency. A trained nurse interviewed all cases and controls and formed a standardised examination for skin nevi. Nevi were examined by body sector; nevi greater than 2 mm in diameter were counted.</p></div><div id="S6" class="sec"><h3 id="S6title">Genotyping</h3><p id="P24" class="p p-first">The TwinsUk sample has been genotyped using the Illumina HumanHap 300k duo chip, and population substructure has been detected and removed as described previously [<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644414">25</a>]. The SNPs rs751173, rs4636294, rs2218220, rs1335510, rs1341866, rs10757257, rs7023329, and rs10811629 for <em>MTAP</em> and rs2284063, rs6001027, rs132985, and rs738322 for <em>PLA2G6</em> were genotyped both in the Australian BTNS and Q-MEGA samples in multiplex assays designed using the Sequenom MassARRAY Assay Design software (version 3.0). SNPs were typed using Sequenom iPLEX™ chemistry on a MALDI-TOF Compact Mass Spectrometer (Sequenom Inc, San Diego, CA, USA). The 2.5μL PCR reactions were performed in standard 384-well plates using 12.5 ng genomic DNA, 0.8 unit of Taq polymerase (HotStarTaq, Qiagen, Valencia, CA), 500 μmol of each dNTP, 1.625mM of MgCl2, and 100 nmol of each PCRprimers (Bioneer, Korea). Standard PCR thermal cycling conditions and post-PCR extension reactions were carried out as described previously [<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644429">26</a>]. The iPLEX reaction products were desalted by diluting samples with 15μl of water and adding 3μl of resin. The products were spotted on a SpectroChip (Sequenom Inc, San Diego, CA, USA), and data were processed and analysed by MassARRAY TYPER 3.4 software (Sequenom Inc, San Diego, CA, USA). None of SNPs exhibited Hardy-Weinberg disequilibrium, and there were no effects of ethnicity in both the control and case groups.</p><p id="P25" class="p p-last">For the Leeds sample, genotyping data from the Illumina HumanHap300 v2 array were available for the 123 familial cases and for 148 cases and 167 controls (Leeds1 sample). The SNPs rs4636294, rs2218220, and rs7023329 for <em>MTAP</em> and rs2284063 and rs6001027 for <em>PLA2G6</em> were genotyped in further 1,126 cases and 574 controls from the Leeds case-control study by Taqman with overall call rate greater then 97%. The Leeds1 sample plus the additional genotypes were part of the Leeds2 sample. None of SNPs exhibited deviation from Hardy-Weinberg equilibrium. Participants in the study were all of UK ancestry and principal component analysis (PCA) involving a subset of the samples included in this analysis found no evidence of population stratification. Both Leeds1 and Leeds2 case-control sample were supplemented with genotyping information from the Wellcome Trust Case-Control Consortium (WTCCC) 1958 birth cohort control samples analysed on the Illumina 550k array. While data on 1,447 controls are provided by the WTCCC, these controls were reduced to 1,395 because of low call rate (call rate &lt; 97%, n = 51) and for apparent non-European ancestry on the basis of PCA (n=1).</p></div><div id="S7" class="sec sec-last"><h3 id="S7title">Statistical analyses</h3><p id="P26" class="p p-first">Power analysis for the TwinsUK discovery sample was performed using the Genetic Power Calculator. Assuming that the tested SNP is a good proxy for an untyped functional variant we have 70% to 90% power of identifying a QTN responsible for 1% to 2% of the total genetic variance showing a P value required for a significant result after Bonferroni correction (P = 0.05/297,108 = 1.68 × 10<sup>-7</sup>). The GWA was performed in the TwinsUK sample using a rapid variance component association test implemented in MERLIN software version 1.1.2 [<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644438">27</a>] on the natural log transformed nevus counts and including age in the model. The method estimates the additive effect of each SNP by fitting a simple regression model to the trait while using a variance component approach to model the correlation between different observed phenotypes within pairs. A quantile-quantile (Q-Q) plot was generated using R and visually inspected to detect strong inflation of the test statistics. The genomic inflation factor was calculated as the ratio of the median of the empirically observed distribution of the test statistic to the expected median to quantify the excess false positive rate [<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644412">28</a>] and was found to be 1.006, close to its expected value of 1.0. Simulation analysis by gene dropping was carried out with MERLIN in the TwinsUK sample to assess the empirical P values for the most significant SNPs on chromosome 9 (rs10757257) and 22 (rs132985). By using 1 × 10<sup>-6</sup> simulations we observed a single P value of the same magnitude as what observed using the real data for rs10757257 (empirical P value 2 × 10<sup>-6</sup>), and none for rs132985 (empirical P value 1 × 10<sup>-6</sup>). Replication in the BTNS has been carried out using MENDEL [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644433">29</a>], following cube-root transformation of the counts, and including age, age2, year studied, body surface area, estimated cumulative UV exposure, ancestry, hair and skin colour as covariates.</p><p id="P27">For the Australian data cube-root transformation was significantly better than log transformation based on the Box-Cox analysis, and it closely matched the results of a negative binomial generalized linear mixed model analyses of the BTNS data, which also improved significantly in terms of model fit diagnostics.</p><p id="P28">Empirical P values were evaluated through straight regression of log(nevus count) versus allele count using MENDEL for gene dropping of the most significant SNPs on chromosomes 9 and 22. By using 1 × 10<sup>-6</sup> iterations we obtained empirical P values of 1 × 10<sup>-6</sup> for rs4636294 and 3.6 × 10<sup>-3</sup> for rs2284063.</p><p id="P29">Combined P values from the BTNS and the TwinsUK studies were obtained using METAL. A z statistic is calculated for each marker using the P value and direction of the effect observed in each study. The combined P value is evaluated from the overall z statistic weighting each study proportionally to the square root of the number of subjects. The graphs on <a href="/pmc/articles/PMC3080738/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> were generated by adapting a freely available R script [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644419">30</a>].</p><p id="P30">Given that some cases and all the controls have some relatives in the analysis, the case-control analysis in the Q-MEGA sample was carried out using MENDEL penetrance analysis option, and also the Sib-pair program. The latter was used to calculate simulation-based P values allowing for residual familial resemblance. We selected cases and controls where all four grandparents were reported to be of Northern European ancestry and we also performed quantitative trait TDT analysis of nevus count, confirming that the observed associations were not due to population stratification. For the Leeds analysis, case-control comparisons by SNP for melanoma were made by unconditional logistic regression. Analyses including nevus count involved age and sex adjusted nevus count. Tests of nevus count by SNP genotype were based on a test of linear trend of age and sex adjusted nevus count.</p><p id="P31" class="p p-last">Meta-analysis for the melanoma case control association results was performed using the rmeta package of R. To refine the location of chromosomes 9 and 22 association signals and help the identification of some causal variants we imputed genotypes from the HapMap in the TwinsUK sample, as described previously [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_217644434">31</a>]. An additional 665 and 309 SNPs in the 9p21 and 22q13 regions respectively were tested for association with nevus count.</p></div></div><div id="S8" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary</h4><div class="sup-box half_rhythm" id="N0x142b0990N0x15246b00"><a href="/pmc/articles/PMC3080738/bin/NIHMS131115-supplement-Supplementary.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(140K, pdf)</sup></div></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">Acknowledgments</h2><div class="sec"><p id="__p9">The KCL authors acknowledge financial support from the Wellcome Trust, the Department of Health via the National Institute for Health Research (NIHR) comprehensive Biomedical Research Centre award to Guy's &amp; St Thomas' NHS Foundation Trust in partnership with King's College London, EC Framework 7 Health-2007-A ENGAGE project and the Chronic Disease Research Foundation (CDRF). TDS is an NIHR senior investigator. We would also like to thank all the nurses and Research assistants who collected the skin data, including Ursula Perks, Gail Clement, Dan Glass, Emad Qweitin as well as the volunteer twins who gave their time.</p><p id="__p10">The Leeds case-control study and genotyping was supported by Cancer Research UK Programme Award (C588/A4994), NIH (R01 CA83115) and EU FP6 to GenoMEL (LSHC-CT-2006-018702). This study makes use of data generated by the Wellcome Trust Case Control Consortium. A full list of the investigators who contributed to the generation of the data is available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.wtccc.org.uk</a>. Funding for the project was provided by the Wellcome Trust under award 076113. The Leeds second control series were recruited through an award from the Department of Health in conjunction with Dr Isabel dos Santos Silva and Prof Anthony Swerdlow. We would like to thank John Taylor for statistical analyses relating to the Leeds dataset.</p><p id="__p11">TP holds a Canada Research Chair and is supported by Genome Canada/Quebec and CIHR.</p><p id="__p12">The Australian Studies were supported by the National Institutes of Health/National Cancer Institute (CA88363), the National Health and Medical Research Council of Australia (NHMRC) and the Cancer Council Queensland. DLD, GWM and NKH are supported by the NHMRC Fellowships scheme. We thank David Whiteman, Adele Green, Joanne Aitken, Anjali Henders, Megan Campbell, Mitchell Stark, Amanda Baxter, Monica de Nooyer, Isabel Gardner, Dixie Statham, Barbara Haddon, Jane Palmer, Lisa Bardsley, David Smyth, and Harry Beeby for their input into project management, sample processing, database and questionnaire development. We are grateful to all the participants of the BTNS and Q-MEGA as well as the research interviewers and examiners for these studies.</p></div></div><div id="idm139878972570272" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139878972570272title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div><p id="__p13" class="p p-first"><strong>Author Contributions:</strong> M.F., T.D.S., N.G.M., D.T.B., N.K.H. designed the study.</p>M.F., D.D.L., J.H.B., A.C. analysed the data.<p id="__p15" class="p">B.V, N.G.M., J.A.N.B. contributed to data collection and phenotype definitions.</p><p id="__p16" class="p">T.D.S., E.D., P.D., D.E., N.G.M., N.K.H., J.A.N.B. obtained funding.</p><p id="__p17" class="p">Z.Z., P.D., N.S., G.W.M. contributed to genotyping.</p><p id="__p18" class="p p-last">M.F., V.B., T.D.S. wrote the first draft of the paper and all authors made important contributions to the final version of the paper.</p></div><p class="fn sec" id="FN4"></p><p id="__p19" class="p p-first-last">COMPETING INTERESTS STATEMENT: The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p20" class="p p-first"><strong>URL Genetic Power Calculator:</strong> <a href="http://pngu.mgh.harvard.edu/∼purcell/gpc/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/gpc/</a></p><p id="__p21" class="p"><strong>R:</strong> <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a></p><p id="__p22" class="p"><strong>METAL:</strong> <a href="http://www.sph.umich.edu/csg/abecasis/metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/metal/</a></p><p id="__p23" class="p p-last"><strong>GenoMEL:</strong> <a href="http://www.genomel.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genomel.org/</a></p><p></p></div></div><div id="idm139878974601040" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139878974601040title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="citation">Karim-Kos HE, et al.  Recent trends of cancer in Europe: a combined approach of incidence, survival and mortality for 17 cancer sites since the 1990s. <span><span class="ref-journal">Eur J Cancer. </span>2008;<span class="ref-vol">44</span>:1345–89.</span> [<a href="/pubmed/18280139" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Cancer&amp;title=Recent+trends+of+cancer+in+Europe:+a+combined+approach+of+incidence,+survival+and+mortality+for+17+cancer+sites+since+the+1990s&amp;author=HE+Karim-Kos&amp;volume=44&amp;publication_year=2008&amp;pages=1345-89&amp;pmid=18280139&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="citation">Goldstein AM, et al.  Features associated with germline CDKN2A mutations: a GenoMEL study of melanoma-prone families from three continents. <span><span class="ref-journal">J Med Genet. </span>2007;<span class="ref-vol">44</span>:99–106.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2598064/">PMC free article</a>]</span> [<a href="/pubmed/16905682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Features+associated+with+germline+CDKN2A+mutations:+a+GenoMEL+study+of+melanoma-prone+families+from+three+continents&amp;author=AM+Goldstein&amp;volume=44&amp;publication_year=2007&amp;pages=99-106&amp;pmid=16905682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="citation">Palmieri G, et al.  Definition of the role of chromosome 9p21 in sporadic melanoma through genetic analysis of primary tumours and their metastases. The Melanoma Cooperative Group. <span><span class="ref-journal">Br J Cancer. </span>2000;<span class="ref-vol">83</span>:1707–14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2363459/">PMC free article</a>]</span> [<a href="/pubmed/11104570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Definition+of+the+role+of+chromosome+9p21+in+sporadic+melanoma+through+genetic+analysis+of+primary+tumours+and+their+metastases.+The+Melanoma+Cooperative+Group&amp;author=G+Palmieri&amp;volume=83&amp;publication_year=2000&amp;pages=1707-14&amp;pmid=11104570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="citation">Gandini S, et al.  Meta-analysis of risk factors for cutaneous melanoma: I. Common and atypical naevi. <span><span class="ref-journal">Eur J Cancer. </span>2005;<span class="ref-vol">41</span>:28–44.</span> [<a href="/pubmed/15617989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Cancer&amp;title=Meta-analysis+of+risk+factors+for+cutaneous+melanoma:+I.+Common+and+atypical+naevi&amp;author=S+Gandini&amp;volume=41&amp;publication_year=2005&amp;pages=28-44&amp;pmid=15617989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="citation">Fountain JW, Bale SJ, Housman DE, Dracopoli NC. Genetics of melanoma. <span><span class="ref-journal">Cancer Surv. </span>1990;<span class="ref-vol">9</span>:645–71.</span> [<a href="/pubmed/2101727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Surv&amp;title=Genetics+of+melanoma&amp;author=JW+Fountain&amp;author=SJ+Bale&amp;author=DE+Housman&amp;author=NC+Dracopoli&amp;volume=9&amp;publication_year=1990&amp;pages=645-71&amp;pmid=2101727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="citation">Falchi M, Spector TD, Perks U, Kato BS, Bataille V. Genome-wide search for nevus density shows linkage to two melanoma loci on chromosome 9 and identifies a new QTL on 5q31 in an adult twin cohort. <span><span class="ref-journal">Hum Mol Genet. </span>2006;<span class="ref-vol">15</span>:2975–9.</span> [<a href="/pubmed/16928783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+search+for+nevus+density+shows+linkage+to+two+melanoma+loci+on+chromosome+9+and+identifies+a+new+QTL+on+5q31+in+an+adult+twin+cohort&amp;author=M+Falchi&amp;author=TD+Spector&amp;author=U+Perks&amp;author=BS+Kato&amp;author=V+Bataille&amp;volume=15&amp;publication_year=2006&amp;pages=2975-9&amp;pmid=16928783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="citation">Zhu G, et al.  A genome-wide scan for naevus count: linkage to CDKN2A and to other chromosome regions. <span><span class="ref-journal">Eur J Hum Genet. </span>2007;<span class="ref-vol">15</span>:94–102.</span> [<a href="/pubmed/17063143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=A+genome-wide+scan+for+naevus+count:+linkage+to+CDKN2A+and+to+other+chromosome+regions&amp;author=G+Zhu&amp;volume=15&amp;publication_year=2007&amp;pages=94-102&amp;pmid=17063143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="citation">Ma Z, Wang X, Nowatzke W, Ramanadham S, Turk J. Human pancreatic islets express mRNA species encoding two distinct catalytically active isoforms of group VI phospholipase A2 (iPLA2) that arise from an exon-skipping mechanism of alternative splicing of the transcript from the iPLA2 gene on chromosome 22q13.1. <span><span class="ref-journal">J Biol Chem. </span>1999;<span class="ref-vol">274</span>:9607–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3715997/">PMC free article</a>]</span> [<a href="/pubmed/10092647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Human+pancreatic+islets+express+mRNA+species+encoding+two+distinct+catalytically+active+isoforms+of+group+VI+phospholipase+A2+(iPLA2)+that+arise+from+an+exon-skipping+mechanism+of+alternative+splicing+of+the+transcript+from+the+iPLA2+gene+on+chromosome+22q13.1&amp;author=Z+Ma&amp;author=X+Wang&amp;author=W+Nowatzke&amp;author=S+Ramanadham&amp;author=J+Turk&amp;volume=274&amp;publication_year=1999&amp;pages=9607-16&amp;pmid=10092647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="citation">Garcia-Castellano JM, et al.  Methylthioadenosine phosphorylase gene deletions are common in osteosarcoma. <span><span class="ref-journal">Clin Cancer Res. </span>2002;<span class="ref-vol">8</span>:782–7.</span> [<a href="/pubmed/11895909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Cancer+Res&amp;title=Methylthioadenosine+phosphorylase+gene+deletions+are+common+in+osteosarcoma&amp;author=JM+Garcia-Castellano&amp;volume=8&amp;publication_year=2002&amp;pages=782-7&amp;pmid=11895909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="citation">Hori Y, et al.  The methylthioadenosine phosphorylase gene is frequently co-deleted with the p16INK4a gene in acute type adult T-cell leukemia. <span><span class="ref-journal">Int J Cancer. </span>1998;<span class="ref-vol">75</span>:51–6.</span> [<a href="/pubmed/9426690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Cancer&amp;title=The+methylthioadenosine+phosphorylase+gene+is+frequently+co-deleted+with+the+p16INK4a+gene+in+acute+type+adult+T-cell+leukemia&amp;author=Y+Hori&amp;volume=75&amp;publication_year=1998&amp;pages=51-6&amp;pmid=9426690&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="citation">Christopher SA, Diegelman P, Porter CW, Kruger WD. Methylthioadenosine phosphorylase, a gene frequently codeleted with p16(cdkN2a/ARF), acts as a tumor suppressor in a breast cancer cell line. <span><span class="ref-journal">Cancer Res. </span>2002;<span class="ref-vol">62</span>:6639–44.</span> [<a href="/pubmed/12438261" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Methylthioadenosine+phosphorylase,+a+gene+frequently+codeleted+with+p16(cdkN2a/ARF),+acts+as+a+tumor+suppressor+in+a+breast+cancer+cell+line&amp;author=SA+Christopher&amp;author=P+Diegelman&amp;author=CW+Porter&amp;author=WD+Kruger&amp;volume=62&amp;publication_year=2002&amp;pages=6639-44&amp;pmid=12438261&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="citation">Behrmann I, et al.  Characterization of methylthioadenosin phosphorylase (MTAP) expression in malignant melanoma. <span><span class="ref-journal">Am J Pathol. </span>2003;<span class="ref-vol">163</span>:683–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1868213/">PMC free article</a>]</span> [<a href="/pubmed/12875987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=Characterization+of+methylthioadenosin+phosphorylase+(MTAP)+expression+in+malignant+melanoma&amp;author=I+Behrmann&amp;volume=163&amp;publication_year=2003&amp;pages=683-90&amp;pmid=12875987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="citation">Pasmant E, et al.  Characterization of a germ-line deletion, including the entire INK4/ARF locus, in a melanoma-neural system tumor family: identification of ANRIL, an antisense noncoding RNA whose expression coclusters with ARF. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:3963–9.</span> [<a href="/pubmed/17440112" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Characterization+of+a+germ-line+deletion,+including+the+entire+INK4/ARF+locus,+in+a+melanoma-neural+system+tumor+family:+identification+of+ANRIL,+an+antisense+noncoding+RNA+whose+expression+coclusters+with+ARF&amp;author=E+Pasmant&amp;volume=67&amp;publication_year=2007&amp;pages=3963-9&amp;pmid=17440112&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="citation">Easton DF, Eeles RA. Genome-wide association studies in cancer. <span><span class="ref-journal">Hum Mol Genet. </span>2008;<span class="ref-vol">17</span>:R109–15.</span> [<a href="/pubmed/18852198" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+studies+in+cancer&amp;author=DF+Easton&amp;author=RA+Eeles&amp;volume=17&amp;publication_year=2008&amp;pages=R109-15&amp;pmid=18852198&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="citation">Hooks SB, Cummings BS. Role of Ca2+-independent phospholipase A2 in cell growth and signaling. <span><span class="ref-journal">Biochem Pharmacol. </span>2008;<span class="ref-vol">76</span>:1059–67.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4088984/">PMC free article</a>]</span> [<a href="/pubmed/18775417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Pharmacol&amp;title=Role+of+Ca2+-independent+phospholipase+A2+in+cell+growth+and+signaling&amp;author=SB+Hooks&amp;author=BS+Cummings&amp;volume=76&amp;publication_year=2008&amp;pages=1059-67&amp;pmid=18775417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="citation">Atsumi G, et al.  Fas-induced arachidonic acid release is mediated by Ca2+-independent phospholipase A2 but not cytosolic phospholipase A2, which undergoes proteolytic inactivation. <span><span class="ref-journal">J Biol Chem. </span>1998;<span class="ref-vol">273</span>:13870–7.</span> [<a href="/pubmed/9593733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Fas-induced+arachidonic+acid+release+is+mediated+by+Ca2+-independent+phospholipase+A2+but+not+cytosolic+phospholipase+A2,+which+undergoes+proteolytic+inactivation&amp;author=G+Atsumi&amp;volume=273&amp;publication_year=1998&amp;pages=13870-7&amp;pmid=9593733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="citation">Bao S, et al.  Effects of stable suppression of Group VIA phospholipase A2 expression on phospholipid content and composition, insulin secretion, and proliferation of INS-1 insulinoma cells. <span><span class="ref-journal">J Biol Chem. </span>2006;<span class="ref-vol">281</span>:187–98.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2156593/">PMC free article</a>]</span> [<a href="/pubmed/16286468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Effects+of+stable+suppression+of+Group+VIA+phospholipase+A2+expression+on+phospholipid+content+and+composition,+insulin+secretion,+and+proliferation+of+INS-1+insulinoma+cells&amp;author=S+Bao&amp;volume=281&amp;publication_year=2006&amp;pages=187-98&amp;pmid=16286468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="citation">Song Y, et al.  Inhibition of calcium-independent phospholipase A2 suppresses proliferation and tumorigenicity of ovarian carcinoma cells. <span><span class="ref-journal">Biochem J. </span>2007;<span class="ref-vol">406</span>:427–36.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2049037/">PMC free article</a>]</span> [<a href="/pubmed/17555408" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Inhibition+of+calcium-independent+phospholipase+A2+suppresses+proliferation+and+tumorigenicity+of+ovarian+carcinoma+cells&amp;author=Y+Song&amp;volume=406&amp;publication_year=2007&amp;pages=427-36&amp;pmid=17555408&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="citation">Sun B, Zhang X, Talathi S, Cummings BS. Inhibition of Ca2+-independent phospholipase A2 decreases prostate cancer cell growth by p53-dependent and independent mechanisms. <span><span class="ref-journal">J Pharmacol Exp Ther. </span>2008;<span class="ref-vol">326</span>:59–68.</span> [<a href="/pubmed/18441250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Pharmacol+Exp+Ther&amp;title=Inhibition+of+Ca2+-independent+phospholipase+A2+decreases+prostate+cancer+cell+growth+by+p53-dependent+and+independent+mechanisms&amp;author=B+Sun&amp;author=X+Zhang&amp;author=S+Talathi&amp;author=BS+Cummings&amp;volume=326&amp;publication_year=2008&amp;pages=59-68&amp;pmid=18441250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="citation">Hosgood HD, 3rd, et al.  Pathway-based evaluation of 380 candidate genes and lung cancer susceptibility suggests the importance of the cell cycle pathway. <span><span class="ref-journal">Carcinogenesis. </span>2008;<span class="ref-vol">29</span>:1938–43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722857/">PMC free article</a>]</span> [<a href="/pubmed/18676680" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Pathway-based+evaluation+of+380+candidate+genes+and+lung+cancer+susceptibility+suggests+the+importance+of+the+cell+cycle+pathway&amp;author=HD+Hosgood&amp;volume=29&amp;publication_year=2008&amp;pages=1938-43&amp;pmid=18676680&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="citation">Bishop TD, et al.  <span><span class="ref-journal">Genome-wide association study identifies three loci associated with melanoma risk. </span></span> (Submitteds) <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2741419/">PMC free article</a>]</span> [<a href="/pubmed/19578364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome-wide+association+study+identifies+three+loci+associated+with+melanoma+risk&amp;author=TD+Bishop&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="citation">Bataille V, Snieder H, MacGregor AJ, Sasieni P, Spector TD. Genetics of risk factors for melanoma: an adult twin study of nevi and freckles. <span><span class="ref-journal">J Natl Cancer Inst. </span>2000;<span class="ref-vol">92</span>:457–63.</span> [<a href="/pubmed/10716963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Genetics+of+risk+factors+for+melanoma:+an+adult+twin+study+of+nevi+and+freckles&amp;author=V+Bataille&amp;author=H+Snieder&amp;author=AJ+MacGregor&amp;author=P+Sasieni&amp;author=TD+Spector&amp;volume=92&amp;publication_year=2000&amp;pages=457-63&amp;pmid=10716963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="citation">Sturm RA, et al.  A single SNP in an evolutionary conserved region within intron 86 of the HERC2 gene determines human blue-brown eye color. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">82</span>:424–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427173/">PMC free article</a>]</span> [<a href="/pubmed/18252222" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+single+SNP+in+an+evolutionary+conserved+region+within+intron+86+of+the+HERC2+gene+determines+human+blue-brown+eye+color&amp;author=RA+Sturm&amp;volume=82&amp;publication_year=2008&amp;pages=424-31&amp;pmid=18252222&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="citation">Baxter AJ, et al.  The Queensland Study of Melanoma: environmental and genetic associations (Q-MEGA); study design, baseline characteristics, and repeatability of phenotype and sun exposure measures. <span><span class="ref-journal">Twin Res Hum Genet. </span>2008;<span class="ref-vol">11</span>:183–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3677021/">PMC free article</a>]</span> [<a href="/pubmed/18361720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res+Hum+Genet&amp;title=The+Queensland+Study+of+Melanoma:+environmental+and+genetic+associations+(Q-MEGA);+study+design,+baseline+characteristics,+and+repeatability+of+phenotype+and+sun+exposure+measures&amp;author=AJ+Baxter&amp;volume=11&amp;publication_year=2008&amp;pages=183-96&amp;pmid=18361720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="citation">Richards JB, et al.  Bone mineral density, osteoporosis, and osteoporotic fractures: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">371</span>:1505–12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2679414/">PMC free article</a>]</span> [<a href="/pubmed/18455228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Bone+mineral+density,+osteoporosis,+and+osteoporotic+fractures:+a+genome-wide+association+study&amp;author=JB+Richards&amp;volume=371&amp;publication_year=2008&amp;pages=1505-12&amp;pmid=18455228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="citation">Zhao ZZ, et al.  Genetic variation in tumour necrosis factor and lymphotoxin is not associated with endometriosis in an Australian sample. <span><span class="ref-journal">Hum Reprod. </span>2007;<span class="ref-vol">22</span>:2389–97.</span> [<a href="/pubmed/17595314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Reprod&amp;title=Genetic+variation+in+tumour+necrosis+factor+and+lymphotoxin+is+not+associated+with+endometriosis+in+an+Australian+sample&amp;author=ZZ+Zhao&amp;volume=22&amp;publication_year=2007&amp;pages=2389-97&amp;pmid=17595314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="citation">Chen WM, Abecasis GR. Family-based association tests for genomewide association scans. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:913–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2265659/">PMC free article</a>]</span> [<a href="/pubmed/17924335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Family-based+association+tests+for+genomewide+association+scans&amp;author=WM+Chen&amp;author=GR+Abecasis&amp;volume=81&amp;publication_year=2007&amp;pages=913-26&amp;pmid=17924335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="citation">Devlin B, Roeder K, Wasserman L. Genomic control, a new approach to genetic-based association studies. <span><span class="ref-journal">Theor Popul Biol. </span>2001;<span class="ref-vol">60</span>:155–66.</span> [<a href="/pubmed/11855950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Theor+Popul+Biol&amp;title=Genomic+control,+a+new+approach+to+genetic-based+association+studies&amp;author=B+Devlin&amp;author=K+Roeder&amp;author=L+Wasserman&amp;volume=60&amp;publication_year=2001&amp;pages=155-66&amp;pmid=11855950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="citation">Lange K, Weeks D, Boehnke M. Programs for Pedigree Analysis: MENDEL, FISHER, and dGENE. <span><span class="ref-journal">Genet Epidemiol. </span>1988;<span class="ref-vol">5</span>:471–2.</span> [<a href="/pubmed/3061869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Programs+for+Pedigree+Analysis:+MENDEL,+FISHER,+and+dGENE&amp;author=K+Lange&amp;author=D+Weeks&amp;author=M+Boehnke&amp;volume=5&amp;publication_year=1988&amp;pages=471-2&amp;pmid=3061869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="citation">Saxena R, et al.  Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1331–1336.</span> [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;volume=316&amp;publication_year=2007&amp;pages=1331-1336&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="citation">Prokopenko I, et al.  Variants in MTNR1B influence fasting glucose levels. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:77–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682768/">PMC free article</a>]</span> [<a href="/pubmed/19060907" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+in+MTNR1B+influence+fasting+glucose+levels&amp;author=I+Prokopenko&amp;volume=41&amp;publication_year=2009&amp;pages=77-81&amp;pmid=19060907&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3080738&amp;issue-id=195988&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139878969279744title">Abstract</a></li><li><a href="#S1title">Methods</a></li><li><a href="#S8title">Supplementary Material</a></li><li><a href="#S10title">Acknowledgments</a></li><li><a href="#idm139878972570272title">Footnotes</a></li><li><a href="#idm139878974601040title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3080738/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3080738/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3080738/pdf/nihms131115.pdf">PDF (475K)</a></li> | <li><a href="#" data-citationid="PMC3080738" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080738%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080738%2F&amp;text=Loci%20at%209p21%20and%2022q13%20harbour%20alleles%20for%20development%20of%20cutaneous%20nevi%20and%20melanoma" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080738%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19578365" data-aiid="3080738" data-aid="3080738" data-iid="195988" data-domainid="319" data-domain="nihpa" data-accid="PMC3080738" data-md5="c47edcb005d23df7ea16695dc83d9f31"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:55:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D4E61DD4FEE3100000000005A003A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3080738%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3080738/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>