<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3992201/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genomewide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3992201/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biological psychiatry"><meta name="citation_title" content="Genomewide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways"><meta name="citation_authors" content="Joel Gelernter, Henry R. Kranzler, Richard Sherva, Ryan Koesterer, Laura Almasy, Hongyu Zhao, Lindsay A. Farrer"><meta name="citation_date" content="1 July 2014"><meta name="citation_issue" content="1"><meta name="citation_volume" content="76"><meta name="citation_firstpage" content="66"><meta name="citation_doi" content="10.1016/j.biopsych.2013.08.034"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3992201/?report=abstract"><meta name="citation_pmid" content="24143882"><meta name="DC.Title" content="Genomewide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Joel Gelernter"><meta name="DC.Contributor" content="Henry R. Kranzler"><meta name="DC.Contributor" content="Richard Sherva"><meta name="DC.Contributor" content="Ryan Koesterer"><meta name="DC.Contributor" content="Laura Almasy"><meta name="DC.Contributor" content="Hongyu Zhao"><meta name="DC.Contributor" content="Lindsay A. Farrer"><meta name="DC.Date" content="2014 Jul 1"><meta name="DC.Identifier" content="10.1016/j.biopsych.2013.08.034"><meta name="DC.Language" content="en"><meta property="og:title" content="Genomewide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways"><meta property="og:type" content="article"><meta property="og:description" content="We report a GWAS of two populations, African- and European-American (AA, EA) for opioid dependence (OD) in three sets of subjects, to identify pathways, genes, and alleles important in OD risk.Design employed three phases (based on separate sample collections). ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3992201/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3992201/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3992201/pdf/nihms525621.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DD43FDD5603A100000000071C045A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992201%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="nlmcatalog" class="last">NLM Catalog</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3992201</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biol Psychiatry</a></span>. Author manuscript; available in PMC 2015 Jul 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24143882" target="pmc_ext" ref="reftype=publisher&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Biol Psychiatry. 2014 Jul 1; 76(1): 66–74. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Oct 19. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.biopsych.2013.08.034" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.biopsych.2013.08.034</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3992201</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS525621</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24143882">24143882</a></div></div></div></div><h1 class="content-title">Genomewide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gelernter%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680135548544" co-class="co-affbox">Joel Gelernter</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Kranzler%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680104362400" co-class="co-affbox">Henry R. Kranzler</a>,<sup>2</sup> <a href="/pubmed/?term=Sherva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680104360416" co-class="co-affbox">Richard Sherva</a>,<sup>3</sup> <a href="/pubmed/?term=Koesterer%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680104358432" co-class="co-affbox">Ryan Koesterer</a>,<sup>3</sup> <a href="/pubmed/?term=Almasy%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680104356448" co-class="co-affbox">Laura Almasy</a>,<sup>5</sup> <a href="/pubmed/?term=Zhao%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680103372528" co-class="co-affbox">Hongyu Zhao</a>,<sup>6</sup> and  <a href="/pubmed/?term=Farrer%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882" class="affpopup" co-rid="_co_idm140680103370544" co-class="co-affbox">Lindsay A. Farrer</a><sup>7</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140680135548544"><h3 class="no_margin">Joel Gelernter</h3><p><sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics; and VA CT Healthcare Center; and Departments of Genetics and Neurobiology</p><div>Find articles by <a href="/pubmed/?term=Gelernter%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Joel Gelernter</a></div></div><div id="_co_idm140680104362400"><h3 class="no_margin">Henry R. Kranzler</h3><p><sup>2</sup>University of Pennsylvania Perelman School of Medicine, Department of Psychiatry and Philadelphia VAMC</p><div>Find articles by <a href="/pubmed/?term=Kranzler%20HR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Henry R. Kranzler</a></div></div><div id="_co_idm140680104360416"><h3 class="no_margin">Richard Sherva</h3><p><sup>3</sup>Boston University School of Medicine, Department of Medicine (Biomedical Genetics)</p><div>Find articles by <a href="/pubmed/?term=Sherva%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Richard Sherva</a></div></div><div id="_co_idm140680104358432"><h3 class="no_margin">Ryan Koesterer</h3><p><sup>3</sup>Boston University School of Medicine, Department of Medicine (Biomedical Genetics)</p><div>Find articles by <a href="/pubmed/?term=Koesterer%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Ryan Koesterer</a></div></div><div id="_co_idm140680104356448"><h3 class="no_margin">Laura Almasy</h3><p><sup>5</sup>Texas Biomedical Research Institute, Department of Genetics</p><div>Find articles by <a href="/pubmed/?term=Almasy%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Laura Almasy</a></div></div><div id="_co_idm140680103372528"><h3 class="no_margin">Hongyu Zhao</h3><p><sup>6</sup>Yale School of Public Health, Departments Biostatistics and Genetics</p><div>Find articles by <a href="/pubmed/?term=Zhao%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Hongyu Zhao</a></div></div><div id="_co_idm140680103370544"><h3 class="no_margin">Lindsay A. Farrer</h3><p><sup>7</sup>Boston University Schools of Medicine and Public Health, Departments of Medicine (Biomedical Genetics), Neurology, Ophthalmology, Genetics &amp; Genomics, Epidemiology and Biostatistics</p><div>Find articles by <a href="/pubmed/?term=Farrer%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24143882">Lindsay A. Farrer</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140680175545376_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140680175545376_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140680175545376_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics; and VA CT Healthcare Center; and Departments of Genetics and Neurobiology</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>University of Pennsylvania Perelman School of Medicine, Department of Psychiatry and Philadelphia VAMC</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Boston University School of Medicine, Department of Medicine (Biomedical Genetics)</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>University of Connecticut, School of Engineering, Department of Computer Science and Engineering</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Texas Biomedical Research Institute, Department of Genetics</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Yale School of Public Health, Departments Biostatistics and Genetics</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Boston University Schools of Medicine and Public Health, Departments of Medicine (Biomedical Genetics), Neurology, Ophthalmology, Genetics &amp; Genomics, Epidemiology and Biostatistics</div><div id="FN1">Corresponding Author: Joel Gelernter, Yale University School of Medicine, Department of Psychiatry, Division of Human Genetics in Psychiatry; VA CT 116A2; 950 Campbell Avenue; West Haven, CT 06516; <a href="mailto:dev@null" data-email="ude.elay@retnreleg.leoj" class="oemail">ude.elay@retnreleg.leoj</a>; fax, 203-937-3897</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140680175545376_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN3">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=24143882" target="pmc_ext" ref="reftype=publisher&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3992201/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01.</strong></div><div><a href="/pmc/articles/PMC3992201/bin/NIHMS525621-supplement-01.pdf" data-ga-action="click_feat_suppl">NIHMS525621-supplement-01.pdf</a><span style="color:gray"> (1.4M)</span></div><div class="small guid">GUID:&nbsp;ED8287FE-ECAD-4DF5-8053-15CC206D4641</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>02.</strong></div><div><a href="/pmc/articles/PMC3992201/bin/NIHMS525621-supplement-02.xls" data-ga-action="click_feat_suppl">NIHMS525621-supplement-02.xls</a><span style="color:gray"> (154K)</span></div><div class="small guid">GUID:&nbsp;D3356890-3385-4595-9E6F-212C43CB9D61</div></div></dd></dl></div><div id="idm140680135962832" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140680135962832title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">We report a GWAS of two populations, African- and European-American (AA, EA) for opioid dependence (OD) in three sets of subjects, to identify pathways, genes, and alleles important in OD risk.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">Design employed three phases (based on separate sample collections). <em>Phase 1</em> included our discovery GWAS dataset consisting of 5,697 subjects (58% AA) diagnosed with opioid and/or other substance dependence (SD), and controls. Subjects were genotyped using the Illumina OmniQuad microarray, yielding 890,000 SNPs suitable for analysis. Additional genotypes were imputed using the 1000 Genomes reference panel. Top-ranked findings were further evaluated in <em>Phase 2</em> by incorporating information from the publicly available SAGE dataset, with GWAS data from 4,063 subjects (32% AA). In <em>Phase 3</em>, the most significant SNPs from <em>Phase</em> 2 were genotyped in 2,549 independent subjects (32% AA). Analyses were performed using case-control and ordinal trait designs.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P4" class="p p-first-last">Most significant results emerged from the AA subgroup. Genomewide-significant associations (p&lt;5.0×10<sup>−8</sup>) were observed with SNPs from multiple loci - <em>KCNC1</em>*rs60349741 most significant after combining results from datasets in every phase of the study. The most compelling results were obtained with genes involved in potassium signaling pathways (e.g., <em>KCNC1, KCNG2, and KCNA4</em>). Pathway analysis also implicated genes involved in calcium signaling and long-term potentiation.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P5" class="p p-first-last">This is the first study to identify risk variants for OD using GWAS. Our results strongly implicate risk pathways, provide insights into novel therapeutic and prevention strategies, and may provide biologically bridge OD and other non-SD psychiatric traits where similar pathways have been implicated.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">opioid dependence, genomewide association, complex traits, calcium signaling, potassium, convergence</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">INTRODUCTION</h2><p id="P6" class="p p-first">Opioid dependence (OD) is associated with serious medical, legal, and social problems, and co-occurring psychiatric disorders. The cost of OD to society in 2002 was ~$181 billion (<a href="#R1" rid="R1" class=" bibr popnode">1</a>). Although genomewide association study (GWAS) is the method of choice to identify risk genes for complex traits, none has been published for OD, despite an estimated heritability of &gt;0.60 (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341956">2</a>). The strongest GWAS-derived noteworthy and replicable genomewide significant (GWS) results so far for drug dependence (DD) traits identified a set of loci mapping to a chromosome 15 nicotinic receptor gene cluster (e.g., ref. (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527145446">3</a>)) for nicotine dependence (ND) and related traits; we also reported a GWS association of the <em>FAM53B</em> locus to cocaine dependence (CD) (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341955">4</a>).</p><p id="P7">Few DD GWAS studies have been attempted, and those that have been published are underpowered by modern standards, partly because they used dichotomous traits (i.e., DD diagnoses). Here, we used a relatively large sample and augmented power with an ordinal trait analytic design that allowed us to take into account both the presence or absence of OD and the severity of affection (including the ability to distinguish between subjects with zero and those with one or two symptoms). This increased power by enabling us to use more of the available phenotypic information than standard diagnosis-based analyses. Some of these strategies have been used previously in successful efforts to map ND risk alleles, most notably the use of large clinical samples (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_527145445">3</a>). We further increased our analytic power by including, for some analyses, data from the Study of Addiction: Genetics and Environment (SAGE) sample (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048651">5</a>, <a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048665">6</a>), which includes SD trait information. This dataset is available to the scientific community through an application process and will henceforth be referred to as “public domain.”</p><p id="P8">Our GWAS discovery sample consisted of 2,379 European Americans (EA), including 1,383 subjects with OD; and 3,318 African Americans (AA), including 683 subjects with OD. A second phase sample of 4,603 EAs and AAs from the SAGE study, and a third phase sample including 2,549 EAs and AAs ascertained in a manner identical to that of the discovery sample, were used to replicate and extend our findings.</p><p id="P9" class="p p-last">Thus, our study took place in three “phases” that differed with respect to samples and genotyping. “Phase 1” designates our own GWAS sample. “Phase 2” designates the addition of SNP data from SAGE (which used a very different recruitment strategy, but similarly ascertained subjects, and was genotyped on a different microarray) combined with our sample by meta-analysis; this is the core of the GWAS discovery and replication strategy. “Phase 3” designates our own smaller replication sample, where individual SNPs rather than GWAS arrays were genotyped. With these strategies, we identified genetic variants that increase risk for OD and related heritable traits.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">MATERIALS AND METHODS</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Subjects and Diagnostic Procedures</h3><p id="P10" class="p p-first">The (Phase 1) GWAS discovery sample included 5,697 subjects. A second identically ascertained sample comprising 2,549 subjects was used for replication (Phase 3). All of these subjects were recruited for studies of the genetics of drug (opioid or cocaine) or alcohol dependence (AD). The sample consisted of small nuclear families (SNFs) originally collected for linkage studies, and unrelated individuals. Subjects were recruited at five eastern US sites (<a href="/pmc/articles/PMC3992201/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Our previous OD linkage study (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048657">7</a>) included a subset of the SNFs included in this study. Subjects gave written informed consent as approved by the institutional review board at each site, and certificates of confidentiality were obtained from NIDA and NIAAA.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p11">(a) Demographic and diagnostic information for the entire sample. Yale University School of Medicine (APT Foundation), New Haven, CT; University of Connecticut Health Center, Farmington, CT; the University of Pennsylvania School of Medicine, Philadelphia, PA; the Medical University of South Carolina, Charleston, SC; and McLean Hospital (Harvard Medical School; Belmont, MA.; (b) Subject characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680103533312" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">A. Recruiting sites</th><th colspan="4" rowspan="1" valign="bottom" align="center">Phase 1 (SNFs)</th><th colspan="4" rowspan="1" valign="bottom" align="center">Phase 1 (Unrelateds)</th><th colspan="4" rowspan="1" valign="bottom" align="center">Phase 3</th><th colspan="3" rowspan="1" valign="bottom" align="center">Site Total</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>AA</em></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>EA</em></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>AA</em></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>EA</em></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>AA</em></th><th colspan="2" rowspan="1" valign="bottom" align="center"><em>EA</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>AA</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>EA</em></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>Total</em></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Recruiting site</th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Male</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"><em>Female</em></th><th rowspan="1" colspan="1" valign="bottom" align="right"></th><th rowspan="1" colspan="1" valign="bottom" align="right"></th><th rowspan="1" colspan="1" valign="bottom" align="right"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Yale (APT Foundation)</td><td rowspan="1" colspan="1" valign="top" align="right">199</td><td rowspan="1" colspan="1" valign="top" align="right">257</td><td rowspan="1" colspan="1" valign="top" align="right">141</td><td rowspan="1" colspan="1" valign="top" align="right">108</td><td rowspan="1" colspan="1" valign="top" align="right">453</td><td rowspan="1" colspan="1" valign="top" align="right">370</td><td rowspan="1" colspan="1" valign="top" align="right">485</td><td rowspan="1" colspan="1" valign="top" align="right">290</td><td rowspan="1" colspan="1" valign="top" align="right">223</td><td rowspan="1" colspan="1" valign="top" align="right">198</td><td rowspan="1" colspan="1" valign="top" align="right">474</td><td rowspan="1" colspan="1" valign="top" align="right">477</td><td rowspan="1" colspan="1" valign="top" align="right">1,700</td><td rowspan="1" colspan="1" valign="top" align="right">1,975</td><td rowspan="1" colspan="1" valign="top" align="right">3,675</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">University of CT</td><td rowspan="1" colspan="1" valign="top" align="right">174</td><td rowspan="1" colspan="1" valign="top" align="right">227</td><td rowspan="1" colspan="1" valign="top" align="right">155</td><td rowspan="1" colspan="1" valign="top" align="right">161</td><td rowspan="1" colspan="1" valign="top" align="right">455</td><td rowspan="1" colspan="1" valign="top" align="right">355</td><td rowspan="1" colspan="1" valign="top" align="right">451</td><td rowspan="1" colspan="1" valign="top" align="right">296</td><td rowspan="1" colspan="1" valign="top" align="right">127</td><td rowspan="1" colspan="1" valign="top" align="right">93</td><td rowspan="1" colspan="1" valign="top" align="right">315</td><td rowspan="1" colspan="1" valign="top" align="right">299</td><td rowspan="1" colspan="1" valign="top" align="right">1,431</td><td rowspan="1" colspan="1" valign="top" align="right">1,677</td><td rowspan="1" colspan="1" valign="top" align="right">3,108</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MUSC</td><td rowspan="1" colspan="1" valign="top" align="right">42</td><td rowspan="1" colspan="1" valign="top" align="right">84</td><td rowspan="1" colspan="1" valign="top" align="right">52</td><td rowspan="1" colspan="1" valign="top" align="right">47</td><td rowspan="1" colspan="1" valign="top" align="right">53</td><td rowspan="1" colspan="1" valign="top" align="right">109</td><td rowspan="1" colspan="1" valign="top" align="right">33</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="right">21</td><td rowspan="1" colspan="1" valign="top" align="right">24</td><td rowspan="1" colspan="1" valign="top" align="right">47</td><td rowspan="1" colspan="1" valign="top" align="right">47</td><td rowspan="1" colspan="1" valign="top" align="right">333</td><td rowspan="1" colspan="1" valign="top" align="right">255</td><td rowspan="1" colspan="1" valign="top" align="right">588</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">McLean Hospital</td><td rowspan="1" colspan="1" valign="top" align="right">44</td><td rowspan="1" colspan="1" valign="top" align="right">36</td><td rowspan="1" colspan="1" valign="top" align="right">42</td><td rowspan="1" colspan="1" valign="top" align="right">30</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="right">6</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">98</td><td rowspan="1" colspan="1" valign="top" align="right">106</td><td rowspan="1" colspan="1" valign="top" align="right">204</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Univ. Pennsylvania</td><td rowspan="1" colspan="1" valign="top" align="right">9</td><td rowspan="1" colspan="1" valign="top" align="right">11</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">0</td><td rowspan="1" colspan="1" valign="top" align="right">288</td><td rowspan="1" colspan="1" valign="top" align="right">136</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="right">10</td><td rowspan="1" colspan="1" valign="top" align="right">51</td><td rowspan="1" colspan="1" valign="top" align="right">64</td><td rowspan="1" colspan="1" valign="top" align="right">43</td><td rowspan="1" colspan="1" valign="top" align="right">39</td><td rowspan="1" colspan="1" valign="top" align="right">559</td><td rowspan="1" colspan="1" valign="top" align="right">112</td><td rowspan="1" colspan="1" valign="top" align="right">671</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td colspan="2" rowspan="1" valign="top" align="center"><strong>Phase 2</strong></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SAGE (Phase 2)</td><td colspan="2" rowspan="1" valign="top" align="right">1083</td><td colspan="2" rowspan="1" valign="top" align="right">736</td><td colspan="2" rowspan="1" valign="top" align="right">2235</td><td colspan="2" rowspan="1" valign="top" align="right">1643</td><td rowspan="1" colspan="1" valign="top" align="right">643</td><td rowspan="1" colspan="1" valign="top" align="right">668</td><td rowspan="1" colspan="1" valign="top" align="right">1222</td><td rowspan="1" colspan="1" valign="top" align="right">1530</td><td rowspan="1" colspan="1" valign="top" align="right">1,311</td><td rowspan="1" colspan="1" valign="top" align="right">2,752</td><td rowspan="1" colspan="1" valign="top" align="right">4,063</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Total Phases 1+2+3</strong></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right">5,432</td><td rowspan="1" colspan="1" valign="top" align="right">6,877</td><td rowspan="1" colspan="1" valign="top" align="right">12,309</td></tr></tbody></table></div><div id="largeobj_idm140680103533312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3992201/table/T2/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680175988512" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th colspan="9" rowspan="1" valign="bottom" align="left">B. Subject Characteristics</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center"><em>African Americans</em></th><th colspan="4" rowspan="1" valign="bottom" align="center"><em>European Americans</em></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">Phase/OD Status</th><th rowspan="1" colspan="1" valign="middle" align="center">#</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Male (%)</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Mean age</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Mean symp. count</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>#</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Male (%)</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Mean age</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>Mean symp. count</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 1/OD cases</td><td rowspan="1" colspan="1" valign="top" align="center">683</td><td rowspan="1" colspan="1" valign="top" align="center">62</td><td rowspan="1" colspan="1" valign="top" align="center">44.2</td><td rowspan="1" colspan="1" valign="top" align="center">6.1</td><td rowspan="1" colspan="1" valign="top" align="center">1383</td><td rowspan="1" colspan="1" valign="top" align="center">60</td><td rowspan="1" colspan="1" valign="top" align="center">37.2</td><td rowspan="1" colspan="1" valign="top" align="center">6.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 1/OD Abusers</td><td rowspan="1" colspan="1" valign="top" align="center">49</td><td rowspan="1" colspan="1" valign="top" align="center">76</td><td rowspan="1" colspan="1" valign="top" align="center">44.8</td><td rowspan="1" colspan="1" valign="top" align="center">1.5</td><td rowspan="1" colspan="1" valign="top" align="center">57</td><td rowspan="1" colspan="1" valign="top" align="center">72</td><td rowspan="1" colspan="1" valign="top" align="center">36.9</td><td rowspan="1" colspan="1" valign="top" align="center">1.6</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 1/Controls</td><td rowspan="1" colspan="1" valign="top" align="center">2550</td><td rowspan="1" colspan="1" valign="top" align="center">49</td><td rowspan="1" colspan="1" valign="top" align="center">40.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">854</td><td rowspan="1" colspan="1" valign="top" align="center">55</td><td rowspan="1" colspan="1" valign="top" align="center">39.6</td><td rowspan="1" colspan="1" valign="top" align="center">0.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 1/Exposed Controls</td><td rowspan="1" colspan="1" valign="top" align="center">639</td><td rowspan="1" colspan="1" valign="top" align="center">61</td><td rowspan="1" colspan="1" valign="top" align="center">42.2</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">366</td><td rowspan="1" colspan="1" valign="top" align="center">64</td><td rowspan="1" colspan="1" valign="top" align="center">37.1</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 2/OD cases</td><td rowspan="1" colspan="1" valign="top" align="center">105</td><td rowspan="1" colspan="1" valign="top" align="center">61</td><td rowspan="1" colspan="1" valign="top" align="center">39.9</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">193</td><td rowspan="1" colspan="1" valign="top" align="center">60</td><td rowspan="1" colspan="1" valign="top" align="center">35.6</td><td rowspan="1" colspan="1" valign="top" align="center">5.9</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 2/OD Abusers</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">100</td><td rowspan="1" colspan="1" valign="top" align="center">40.5</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">100</td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">0</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 2/Controls</td><td rowspan="1" colspan="1" valign="top" align="center">1204</td><td rowspan="1" colspan="1" valign="top" align="center">48</td><td rowspan="1" colspan="1" valign="top" align="center">40</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">2554</td><td rowspan="1" colspan="1" valign="top" align="center">43</td><td rowspan="1" colspan="1" valign="top" align="center">38.8</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 2/Exposed Controls</td><td rowspan="1" colspan="1" valign="top" align="center">158</td><td rowspan="1" colspan="1" valign="top" align="center">67</td><td rowspan="1" colspan="1" valign="top" align="center">40.8</td><td rowspan="1" colspan="1" valign="top" align="center">0.2</td><td rowspan="1" colspan="1" valign="top" align="center">383</td><td rowspan="1" colspan="1" valign="top" align="center">69</td><td rowspan="1" colspan="1" valign="top" align="center">35.9</td><td rowspan="1" colspan="1" valign="top" align="center">0.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 3/OD cases</td><td rowspan="1" colspan="1" valign="top" align="center">142</td><td rowspan="1" colspan="1" valign="top" align="center">67</td><td rowspan="1" colspan="1" valign="top" align="center">45.2</td><td rowspan="1" colspan="1" valign="top" align="center">5.9</td><td rowspan="1" colspan="1" valign="top" align="center">407</td><td rowspan="1" colspan="1" valign="top" align="center">60</td><td rowspan="1" colspan="1" valign="top" align="center">35.6</td><td rowspan="1" colspan="1" valign="top" align="center">6.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 3/OD Abusers</td><td rowspan="1" colspan="1" valign="top" align="center">10</td><td rowspan="1" colspan="1" valign="top" align="center">90</td><td rowspan="1" colspan="1" valign="top" align="center">44.8</td><td rowspan="1" colspan="1" valign="top" align="center">2.2</td><td rowspan="1" colspan="1" valign="top" align="center">22</td><td rowspan="1" colspan="1" valign="top" align="center">55</td><td rowspan="1" colspan="1" valign="top" align="center">36.9</td><td rowspan="1" colspan="1" valign="top" align="center">1.1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 3/Controls</td><td rowspan="1" colspan="1" valign="top" align="center">601</td><td rowspan="1" colspan="1" valign="top" align="center">47</td><td rowspan="1" colspan="1" valign="top" align="center">39.9</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td><td rowspan="1" colspan="1" valign="top" align="center">1255</td><td rowspan="1" colspan="1" valign="top" align="center">46</td><td rowspan="1" colspan="1" valign="top" align="center">42.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.02</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Phase 3/Exposed Controls</td><td rowspan="1" colspan="1" valign="top" align="center">85</td><td rowspan="1" colspan="1" valign="top" align="center">78</td><td rowspan="1" colspan="1" valign="top" align="center">42.3</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">137</td><td rowspan="1" colspan="1" valign="top" align="center">61</td><td rowspan="1" colspan="1" valign="top" align="center">36.7</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td></tr></tbody></table></div><div id="largeobj_idm140680175988512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3992201/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P11" class="p p-last">All subjects were interviewed using an electronic version of the Semi-Structured Assessment for Drug Dependence and Alcoholism (SSADDA) (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341953">8</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048663">9</a>) to derive DSM-IV diagnoses (<a href="#R10" rid="R10" class=" bibr popnode">10</a>) of lifetime OD and other major psychiatric traits. The test-retest (κ=0.94) and interrater (κ=0.91) reliability of the OD diagnosis was excellent (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048646">9</a>).</p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping and Quality Control</h3><p id="P12" class="p p-first">Samples for Phase 1 were genotyped on the Illumina HumanOmni1-Quad v1.0 microarray containing 988,306 autosomal SNPs, at the Center for Inherited Disease Research (CIDR) and the Yale Center for Genome Analysis. Genotypes were called using GenomeStudio software V2011.1 and genotyping module V1.8.4 (Illumina, San Diego, CA, USA).</p><p id="P13">Follow-up genotyping (Phase 3 sample) was performed using a custom Illumina GoldenGate Genotyping Universal-32, 1536-plex microarray. Most SNPs included in the custom array were selected for studies of other phenotypes. Additional SNPs were genotyped individually using the TaqMan method (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048656">11</a>).</p><p id="P14">44,644 SNPs on the microarray and 135 individuals with call rates &lt; 98% were excluded, and 62,076 additional SNPs were removed due to minor allele frequencies (MAF) &lt;1%. After data cleaning and quality control, 5,697 individuals and 889,659 SNPs remained for imputation. We identified several instances where identical DNA marker profiles were linked to two different interview forms. When demographic information (sex, date of birth, number of reported children) was consistent across interviews, one sample was randomly removed from analysis; when demographic information was inconsistent, both were removed. Genetic relationships were examined in the family-based sample by calculating pairwise identity by descent (IBD) proportion estimates using PLINK (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048653">12</a>). Pairs of individuals whose IBD proportions did not match their reported genetic relationship were assigned to two different families and pairs of individuals who shared more than 25% of their alleles IBD were assigned to the same family. Self-reported males with X chromosome heterozygosity &gt; 20% and self-reported females with heterozygosity &lt; 20% were excluded unless their true identity could be determined.</p><p id="P15" class="p p-last">To verify and correct the misclassification of self-reported race, we compared GWAS data from all subjects with genotypes from the HapMap 3 reference CEU, YRI, and CHB populations. Principal components (PC) analysis was conducted in the entire GWAS sample using Eigensoft (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048649">13</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048637">14</a>) and 145,472 SNPs that were common to the GWAS dataset and HapMap panel (after pruning the GWAS SNPs for linkage disequilibrium (LD) (r<sup>2</sup>) &gt;80%) to characterize the underlying genetic architecture of the samples. The first PC score distinguished AAs and EAs; these groups were subsequently analyzed separately. We then conducted PC analyses within the two groups and the first three PCs were used in all subsequent analyses to correct for residual population stratification.</p></div><div id="S9" class="sec"><h3 id="S9title">Public Domain GWAS Sample: SAGE</h3><p id="P16" class="p p-first">In the <em>Phase 2</em> analyses described below, we included publically available GWAS data from SAGE (<a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1</a>) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048642">15</a>). The SAGE dataset contained 1,311 AA and 2,750 EA unrelated individuals (<a href="/pmc/articles/PMC3992201/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). SAGE includes individuals from the Collaborative Study on the Genetics of Alcoholism (COGA) (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048666">5</a>), the Family Study of Cocaine Dependence (FSCD) (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048644">6</a>), and the Collaborative Genetic Study of Nicotine Dependence (COGEND) (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048638">15</a>). The COGA sample is a set of unrelated individuals recruited in Indiana, New York, St. Louis, Connecticut, Iowa, and San Diego selected for genotyping from a larger set of 8,000 subjects. Cases met criteria for DSM-IV AD. FSCD contained cases and controls from the greater St. Louis metropolitan area. All cases met criteria for DSM-IV AD and most also met criteria for DSM-IV CD. Controls were from the same communities and had consumed alcohol, but had no lifetime history of substance dependence. A subgroup of FSCD subjects was not alcohol dependent, but had a lifetime diagnosis of DSM-IV cannabis dependence or dependence on another illicit drug. COGEND subjects were recruited in MO and MI. Cases met criteria for DSM-IV alcohol and/or ND. SAGE samples were genotyped on the Illumina Human 1M array containing 1,069,796 total SNPs. Control subjects were selected from the non-dependent population and did not meet criteria for AD, ND, or illicit DD.</p><p id="P17" class="p p-last"><strong>Genotypes were imputed</strong> with IMPUTE2 (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048641">16</a>) using the genotyped SNPs and the 1000 Genomes reference panel released in June 2011 (<a href="http://www.1000genomes.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.1000genomes.org/</a>), which contains phased haplotypes for 1,094 individuals of various ancestries, including 379 samples of European descent (CEU, FIN, GBR, IBS, and TCI), 286 of Asian descent (CHB, JPT, and CHS), 181 admixed American samples (PUR, CLM, and MXL), and 246 samples of African descent (ASW, LWK, YR) (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048645">17</a>). EA and AA samples were imputed separately. Genotyped SNPs with Hardy-Weinberg equilibrium P-values &lt;10<sup>−5</sup> were set to missing and imputed genotypes were used instead. We retained 18,564,419 SNPs with derived information content &gt; 0.8 in at least one of the population groups.</p></div><div id="S10" class="sec"><h3 id="S10title">Analytical Models</h3><ol class="enumerated" style="list-style-type:decimal" id="L1"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><p id="P18">We employed a model with the imputed minor allele dosage as the dependent variable used DSM-IV symptom count (Sympcount<sub>adj</sub>) for OD and each of three other major SD diagnoses (CD, AD, and ND) as ordinal predictors (with adjustment for sex, age, and the first three PCs of ancestry [<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1 in Supplement 1</a>]). The mutual adjustment of OD for measures of dependence on other substances <em>facilitated the identification of genetic risk factors unique to each substance</em> and limited confounding due to comorbid dependencies. All individuals contributed to this analysis, including those meeting DSM-IV criteria for OD (who met 3–7 criteria) and those having no symptoms of OD. The ordinal trait model has greater power to detect genetic associations than a univariate model based on disease status because of greater information content and improved specificity of the dependence measure. The β coefficient and p-value for the OD symptom count (adjusted for the symptom counts for CD, AD, and ND) were used to assess the magnitude and significance of the association, respectively. Ordinal trait data were derived for all samples included in the study.</p></li><li><p id="P19">Case-control status was the outcome in models that included as controls only individuals who had used opioids at least once without becoming dependent. This accounts for the fact that subjects who were never exposed might be better classified as phenotype “unknown” than as “unaffected.”</p></li></ol><p id="P20" class="p p-last">Association tests in the Phase 1 and Phase 2 GWAS datasets were performed using linear or logistic association models embedded in generalized estimating equations (GEE) to correct for correlations among related individuals (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048650">18</a>). The Phase 3 sample containing unrelated individuals only was evaluated using linear and logistic models. All models were adjusted for age, sex, and the first three PCs of ancestry. The data were analyzed separately within population groups and the results were combined by meta-analysis using the inverse variance method implemented in the computer program METAL (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048639">19</a>). In <em>Phase 2</em>, <em>Phase 1</em> SNPs with p-values&lt;10<sup>−4</sup> in either population were tested for association in the SAGE dataset in the specific population group(s) and statistical model(s) yielding the specified result in <em>Phase 1</em>. Tests were restricted in this manner to minimize their number and because it is unlikely that a SNP not meeting the cut-off for follow-up would attain a noteworthy significance level when combining results from the discovery and replication samples. <em>Phase 1</em> and <em>2</em> results were combined within population groups by meta-analysis. In <em>Phase 3</em>, we selected for replication <sup>76</sup> SNPs (Sympcount<sub>adj</sub>, N=44 and case-control, N=28, including 58 unique SNPs) with p&lt;10<sup>−4</sup> in either population in the <em>Phase 2</em> analyses.</p></div><div id="S11" class="sec"><h3 id="S11title">Multiple Testing Considerations</h3><p id="P21" class="p p-first-last">A p-value of 5.0×10<sup>−8</sup> was used as a threshold for GWS. Results were not adjusted for testing in two populations because we tested three distinct <em>a priori</em> hypotheses: (1) SNPs are associated with OD and related traits in AAs, (2) SNPs are associated with OD and related traits in EAs, and (3) associations are evident with the same SNPs in AAs and EAs (in meta-analysis). Results were not adjusted for testing two phenotypes, as the rank order correlation between the symptom count and case-control variables is 1.0 (that is, no affected subject can have a lower symptom count than a control), and the point-biserial correlation is 0.95.</p></div><div id="S12" class="sec sec-last"><h3 id="S12title">Pathway Analysis</h3><p id="P22" class="p p-first-last">Meta-analyzed GWAS results (separately by population) from the discovery and SAGE datasets were used to identify biological pathways related to OD. First, the number of independent SNP association tests for each gene in the genome was computed according to the method of Li and Ji (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048662">20</a>). Next, the smallest p-value for an individual SNP within each gene was multiplied by the number of independent tests in that gene to create a list of genes significantly associated with OD after correcting for the number of tests within that gene (P<sub>adj</sub>&lt;0.05). The significant genes were evaluated by pathway analysis, performed using the Ingenuity Pathway Analysis (IPA) software suite (<a href="http://www.ingenuity.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com</a>) to identify an overrepresentation of selected genes within canonical pathways that were defined using information culled from multiple sources (Kyoto Encyclopedia of Genes and Genomes, interactome studies, manual curation, etc). A Fisher’s exact p-value for each pathway indicated whether the pathway contained more significantly associated genes than expected by chance. To validate the findings, we uploaded the same gene list into the Database for Annotation, Visualization and Integrated Discovery (DAVID; <a href="http://www.david.niaid.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.david.niaid.nih.gov</a>) and performed a gene enrichment analysis using its functional annotation tools.</p></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">RESULTS</h2><p id="P23" class="p p-first">As noted, three independent datasets were employed in these OD-phenotype analyses, in three phases. EAs and AAs were analyzed separately in each phase. In <em>Phase 1</em>, GWAS was conducted in a discovery dataset containing 5,697 subjects. In <em>Phase 2</em>, the first replication phase, SNPs with p-values &lt;10<sup>−4</sup> in either or both populations combined were tested for association in the SAGE dataset including 4,063 subjects. In <em>Phase 3</em>, the second replication phase, SNPs with p-values &lt;10<sup>−4</sup> in either population in the <em>Phase 2</em> analyses were evaluated in a set of 2,549 unrelated subjects ascertained in the same manner as the <em>Phase 1</em> sample. Meta-analyses were conducted in a combined sample of 12,309 subjects. Analyses of symptom count accounted for comorbidity with other SD symptoms (sympcount<sub>adj</sub>).</p><p id="P24">In both <em>Phase 1</em> analyses, we identified population-specific variants and variants associated with risk in <em>both</em> EAs and AAs and (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S2 and S3 in Supplement 1</a>]), including GWS associations in AAs with <em>PITPNM3</em> (rs9913974, p=8.4×10<sup>−9</sup>) for the case-control model and <em>KCNC1</em> (rs60349741, p=1.3×10<sup>−8</sup>) and <em>HHLA2</em> (rs73204138, p=3.7×10<sup>−8</sup>) for the Sympcount<sub>adj</sub> model. The p-values for these findings are negligibly inflated (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S4 and S5 in Supplement 1</a>]) and the SNPs were well imputed (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1 in Supplement 1</a>]). We also identified a possible association with a <em>DISC1</em> polymorphism (rs2738888, AA p=8.47×10<sup>−4</sup>; EA p=1.74×10<sup>−3</sup>; meta p=5.22×10<sup>−6</sup> (discussed below)).</p><p id="P25">In <em>Phase 2</em>, we repeated the primary OD analyses for SNPs with p&lt;10<sup>−4</sup> (403 SNPs for the case-control model, 1,522 SNPs for the Sympcount<sub>adj</sub> model in AAs) in the independent SAGE sample. Four distinct regions contained GWS SNP sympcount<sub>adj</sub> p-values in AAs after meta-analysis of the results from <em>Phases 1</em> and <em>2</em> (<a href="/pmc/articles/PMC3992201/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). These included the regions containing the potassium voltage-gated channel genes <em>KCNG2</em> (rs62103177, p=6.7×10<sup>−10</sup>; <a href="/pmc/articles/PMC3992201/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) and <em>KCNC1</em> (rs60349741, p=7.5×10<sup>−9</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6 in Supplement 1</a>), and <em>APBB2</em> (rs115368721, p=1.8×10<sup>−9</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S7 in Supplement 1</a>), and <em>PARVA</em> (rs73411566, p=1.8×10<sup>−8</sup>; <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S8 in Supplement 1</a>). Several GWS results from <em>Phase 1</em> did not remain so in <em>Phase 2</em>, including SNPs mapped to <em>HHLA2</em> and <em>PITPNM3</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3992201/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680101191344" class="figure"><a href="/pmc/articles/PMC3992201/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3992201_nihms525621f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms525621f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3992201/bin/nihms525621f1.jpg"></a></div><div id="largeobj_idm140680101191344" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3992201/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3992201/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p6">Regional association (Manhattan) plot showing Phase 1+2 meta-analyzed results for association of Sympcount<sub>adj</sub> with SNPs mapped to the <em>KCNG2</em> region on chromosome 18 in the AA population. The SNPs are color coded according to r<sup>2</sup> with the most significant SNP shown in purple. The SNP with the strongest evidence for association is shown twice: once with the Phase 1+2 meta-analysis p-value (purple with “crosshairs”) and once after the inclusion of the Phase 3 samples (purple without “crosshairs”). (They are nearly superimposed in this figure.) The light blue line and right Y-axis show the observed recombination rate in the HapMap YRI samples.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p9">Genomewide significant association results for measures of opioid dependence – adjusted symptom count model. Imputation qualities ranged from 0.91–0.99 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680172197088" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="2" colspan="1" valign="bottom" align="center">BP</th><th rowspan="2" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="2" colspan="1" valign="bottom" align="center">SNP Type</th><th rowspan="2" colspan="1" valign="bottom" align="center">Gene</th><th colspan="2" rowspan="1" valign="bottom" align="center">Phase 1</th><th colspan="2" rowspan="1" valign="bottom" align="center">Phase 2</th><th rowspan="1" colspan="1" valign="bottom" align="center">Phase 1+2</th><th colspan="2" rowspan="1" valign="bottom" align="center">Phase 3</th><th rowspan="1" colspan="1" valign="bottom" align="center">Phase 1+2+3</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th><th rowspan="1" colspan="1" valign="bottom" align="center">P-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">41056716</td><td rowspan="1" colspan="1" valign="top" align="center">rs114070671<sup>*</sup></td><td rowspan="1" colspan="1" valign="top" align="center">intron</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APBB2</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">4.58E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">8.27E-05</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>2.26E-09</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">1.04E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">41056839</td><td rowspan="1" colspan="1" valign="top" align="center">rs115368721</td><td rowspan="1" colspan="1" valign="top" align="center">intron</td><td rowspan="1" colspan="1" valign="top" align="center"><em>APBB2</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">3.74E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">1.16E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.92E-08</td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">--not tested --</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">12454744</td><td rowspan="1" colspan="1" valign="top" align="center">rs73411566</td><td rowspan="1" colspan="1" valign="top" align="center">intron</td><td rowspan="1" colspan="1" valign="top" align="center"><em>PARVA</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">5.48E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">3.06E-05</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.75E-08</strong></td><td rowspan="1" colspan="1" valign="top" align="center"></td><td rowspan="1" colspan="1" valign="top" align="center">--not tested --</td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">17796111</td><td rowspan="1" colspan="1" valign="top" align="center">rs60349741</td><td rowspan="1" colspan="1" valign="top" align="center">UTR_3</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNC1</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.33E-08</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">7.79E-02</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>9.26E-09</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">4.81E-07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">77624479</td><td rowspan="1" colspan="1" valign="top" align="center">rs62103177</td><td rowspan="1" colspan="1" valign="top" align="center">intron</td><td rowspan="1" colspan="1" valign="top" align="center"><em>KCNG2</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">8.00E-05</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-07</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6.68E-10</strong></td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">3.60E-10</td></tr></tbody></table></div><div id="largeobj_idm140680172197088" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3992201/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>LD proxy for rs115368721</div></div></div><p id="P26" class="p">Although no GWS association signals identified in the meta-analysis of the results from <em>Phases1</em>+<em>2</em> were significantly associated with the corresponding trait in the much smaller <em>Phase 3</em> dataset at p&lt;0.05, three of the results improved or remained significant after meta-analysis that included the <em>Phase 3</em> data (<a href="/pmc/articles/PMC3992201/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The associations of rs62103177 in <em>KCNG2</em> with Sympcount<sub>adj</sub> improved to p=3.6×10<sup>−10</sup> (<a href="/pmc/articles/PMC3992201/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Two of the GWS SNPs (rs115368721 in <em>APBB2</em>, rs73411566 in <em>PARVA</em>) in <em>Phase 2</em> were imputed and assays could not be designed to genotype them in <em>Phase 3</em>. LD-based proxies were used for <em>APBB2</em> SNPs, but no suitable proxies (i.e., r<sup>2</sup>&gt;0.4) were available for the <em>PARVA</em> SNP.</p><div id="S14" class="sec sec-last"><h3 id="S14title">Pathway Analysis</h3><p id="P27" class="p p-first-last">Pathway analysis was performed including genes with at least one SNP with a gene-based corrected p-value &lt;0.05. Separate analyses were performed with gene sets obtained from the case-control (62 genes) and Sympcount<sub>adj</sub> (198 genes) models. The most remarkable pathways were identified from the case-control model. The first and third most significant pathways resulting from this analysis were calcium signaling (p=0.002, false discovery rate (FDR)=0.15, <a href="/pmc/articles/PMC3992201/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) and synaptic long-term potentiation (LTP) (p=0.004, FDR=0.17, <a href="/pmc/articles/PMC3992201/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>), respectively. (The second most-significant canonical pathway (p=0.0024) captured genes that influence cardiac hypertrophy.) The Ca<sup>2+</sup> signaling pathway was identified by significant associations in genes encoding calcineurin A (<em>PPP3CA</em>), calcium/calmodulin-dependent protein kinase II beta (<em>CAMK2B</em>), histone deacetyltransferase 9 (<em>HDAC9</em>), and cAMP responsive element binding protein 5 (<em>CREB5</em>). Ca<sup>2+</sup> signaling is also critical to LTP; thus, this pathway emerged as significant in part due to associations with <em>PPP3CA</em>, <em>CAMK2B</em>, and <em>CREB5</em> (<a href="/pmc/articles/PMC3992201/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Reanalysis using the DAVID software revealed that the top gene ontology functional category was “calcium ion binding” (p=0.0089) and the top KEGG pathway was “LTP” (p=0.034). These pathways (complete list, <a href="#SD2" rid="SD2" class=" supplementary-material">Table S2 in Supplement 2</a>) are identical to the pathways identified by IPA.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3992201/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680101184640" class="figure"><a href="/pmc/articles/PMC3992201/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3992201_nihms525621f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms525621f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3992201/bin/nihms525621f2.jpg"></a></div><div id="largeobj_idm140680101184640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3992201/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3992201/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p7">The role of genes identified in GWAS of Phase 1+2 AA subjects using a case-control model in the canonical pathway “calcium signaling.” The gene or gene families with a significant association result are shown in orange. The calcineurin family was identified by an association in <em>PPP3CA</em>, the CREB family by <em>CREB5</em>, the HDAC family by <em>HDAC9</em>, and the CAMK2 family by <em>CAMK2B</em> (see table “pathway SNPs”). For a full description of the functional relationships represented by the various lines and arrows, see:<a href="http://www.biolreprod.org/content/suppl/2010/09/29/biolreprod.110.085910.DC1/biolreprod.110.085910-3.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biolreprod.org/content/suppl/2010/09/29/biolreprod.110.085910.DC1/biolreprod.110.085910-3.pdf</a>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3992201/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680172200768" class="figure"><a href="/pmc/articles/PMC3992201/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3992201_nihms525621f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms525621f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3992201/bin/nihms525621f3.jpg"></a></div><div id="largeobj_idm140680172200768" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3992201/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3992201/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p8">The role of genes identified in GWAS of Phase 1+2 AA subjects using a case-control model in the canonical pathway “synaptic long-term potentiation.” The gene or gene families with a significant association result are shown in orange. The calcineurin family was identified by an association in PPP3CA, the CREB family by <em>CREB5</em>, and the CAMK2 family by <em>CAMK2B</em> (see table “pathway SNPs”). For a full description of the functional relationships represented by the various lines and arrows, see:<a href="http://www.biolreprod.org/content/suppl/2010/09/29/biolreprod.110.085910.DC1/biolreprod.110.085910-3.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.biolreprod.org/content/suppl/2010/09/29/biolreprod.110.085910.DC1/biolreprod.110.085910-3.pdf</a>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p12">Pathway SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140680102791376" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center">chr</th><th rowspan="1" colspan="1" valign="bottom" align="center">SNP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="center">P</th><th rowspan="1" colspan="1" valign="bottom" align="center">BP</th><th rowspan="1" colspan="1" valign="bottom" align="center">Total SNPs</th><th rowspan="1" colspan="1" valign="bottom" align="center">Independent SNPs</th><th rowspan="1" colspan="1" valign="bottom" align="center">Adjusted P</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs2075069</td><td rowspan="1" colspan="1" valign="top" align="center">CAMK2B</td><td rowspan="1" colspan="1" valign="top" align="center">1.27E-06</td><td rowspan="1" colspan="1" valign="top" align="center">44284564</td><td rowspan="1" colspan="1" valign="top" align="center">61</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">2.55E-05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">4-102128801</td><td rowspan="1" colspan="1" valign="top" align="center">PPP3CA</td><td rowspan="1" colspan="1" valign="top" align="center">3.15E-04</td><td rowspan="1" colspan="1" valign="top" align="center">1.02E+08</td><td rowspan="1" colspan="1" valign="top" align="center">67</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="center">5.66E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">rs9639575</td><td rowspan="1" colspan="1" valign="top" align="center">CREB5</td><td rowspan="1" colspan="1" valign="top" align="center">1.19E-04</td><td rowspan="1" colspan="1" valign="top" align="center">28417214</td><td rowspan="1" colspan="1" valign="top" align="center">135</td><td rowspan="1" colspan="1" valign="top" align="center">52</td><td rowspan="1" colspan="1" valign="top" align="center">6.16E-03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">7-18189274</td><td rowspan="1" colspan="1" valign="top" align="center">HDAC9</td><td rowspan="1" colspan="1" valign="top" align="center">2.40E-05</td><td rowspan="1" colspan="1" valign="top" align="center">18189274</td><td rowspan="1" colspan="1" valign="top" align="center">219</td><td rowspan="1" colspan="1" valign="top" align="center">73</td><td rowspan="1" colspan="1" valign="top" align="center">1.75E-03</td></tr></tbody></table></div><div id="largeobj_idm140680102791376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3992201/table/T3/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">DISCUSSION</h2><p id="P28" class="p p-first">We report here results for a GWAS for OD in two different population groups in the United States. We made use of GWAS and replication samples collected by us and available data from the SAGE project, which were collected using a similar psychiatric interview, were used for replication. Several of the top-ranked genes encode proteins that participate in potassium and calcium signaling pathways. Although calcium signaling genes have been studied in addiction biology, they were not previously considered key genetic candidates. These findings therefore add substantially to our knowledge of the biology of OD. Our most compelling results were obtained in the AA population.</p><p id="P29"><em>KCNC1</em> and <em>KCNG2,</em> loci containing some of the most significantly associated SNPs, encode potassium voltage-gated channel subunits. Another locus, <em>PITPNM3,</em> encodes a protein that is involved in phosphatidylinositol transport, but that also binds calcium. Variants located ~196 kb upstream from <em>PPP3CA</em> were associated with OD in the case-control model (although not GWS) in both populations (rs6419156, P<sub>AA</sub>=1.4×10<sup>−6</sup>, P<sub>EA</sub>=5.6×<sup>10−7</sup>); the effect directions were opposite, suggesting that population-specific causal variants or a single causal variant occurring on distinct haplotypes may be responsible. <em>PPP3CA</em> encodes a calcium-dependent, calmodulin-stimulated protein phosphatase involved in calcium signaling. (A trinucleotide repeat at this locus was reported to be nominally associated to SD in AAs but not EAs (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048652">21</a>)).</p><p id="P30">Calcium signaling and LTP (which relies on calcium signaling for neurotransmitter release) were two of the best-supported pathways identified in pathway analysis (although these results did not meet FDR-defined significance after correction for the number of pathways evaluated). Thus, potassium and calcium transport and signaling mechanisms appear to play essential roles in OD risk.</p><p id="P31">Possibly relevant to these findings, <em>CACNA1C</em> (which encodes a voltage-dependent calcium channel) is one of the best-supported GWAS-identified risk genes for bipolar affective disorder (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048640">22</a>) and schizophrenia (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048659">23</a>). Further, a large mega-analysis of psychiatric traits also landed on several calcium-system genes as contributing to risk for a set of psychiatric illnesses: two of the four GWS SNPs mapped to voltage-gated calcium channel subunit genes, and calcium signaling pathways were identified as important for the five disorders studied (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341951">24</a>). Potassium-calcium signaling may couple neuronal signaling to vasodilation in the brain (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341954">25</a>) and opioids can regulate calcium conductance via increasing potassium conductance in μ-opioid receptors (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048661">26</a>). One of the genes in the calcium signaling pathway that is strongly associated with OD risk, <em>CAMK2B</em>, modulates activation of ionotropic (including AMPA) glutamate receptors (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048654">27</a>). AMPA-mediated glutamatergic signaling in the hippocampus plays an important role in context-dependent sensitization to morphine. Behavioral sensitization in animal models is of interest as a neural basis of addiction (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341960">28</a>, <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048636">29</a>). LTP (which underlies learning and memory) was also implicated directly in pathway analysis. These pathways may interact to produce biologically important effects both on behavior following an initial exposure to opioids and on impaired control over use following chronic exposure (i.e., relapse risk, a key issue in the natural history and treatment of OD).</p><p id="P32">There were additional GWS associations (albeit not-yet-replicated) of biological interest. <em>PARVA</em>, encoding alpha parvin, which is reported to play roles in cell adhesion and cytoskeletal organization, was associated in the GWAS for OD (<em>Phase 2</em> meta p=1.75×10<sup>−8</sup>). <em>APBB2</em> (<em>Phase 2</em> meta p=2.26×10<sup>−9</sup>) encodes a member of a family of proteins that bind the amyloid precursor protein that may be important for signal transduction and is associated with Alzheimer disease risk (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048664">30</a>). A closely related gene, <em>APBB1</em>, was shown previously to be associated to ND (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_635893605">31</a>).</p><p id="P33">An interesting possible OD risk gene identified in Phase 1 was <em>DISC1</em>, which (among other functions) regulates development of the cerebral cortex. <em>DISC1</em> was originally identified as a schizophrenia risk gene in a cytogenetic study (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048660">32</a>) and considerable evidence since has tied it to cognitive phenotypes in human (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341957">33</a>) and animal (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341961">34</a>) models. We observed nominally significant associations with <em>DISC1</em> in both EAs and AAs (meta p=5.2×10<sup>−6</sup>). Subsequent to this common-variant association finding, we used deep sequencing at this locus to identify a set of rare variants in <em>DISC1</em> that contribute to risk for OD (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341958">35</a>). Thus, this study ties OD to other cognitive phenotypes via a common gene, and, as schizophrenia was one of the five disorders identified as associated to calcium pathways (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341952">24</a>), also through gene pathways. Previously, we reported that another schizophrenia-associated locus, <em>NRG1</em>, is associated with cannabis dependence in AAs (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_463341959">36</a>).</p><p id="P34">Strengths of this study are a relatively large sample size, including SAGE data to increase power. All subjects were assessed using comprehensive, standardized instruments. Our primary sample has unusually good representation of AA subjects. Most of the strongest findings emerged from this understudied population; some of these were supported by findings from the EAs. We employed several novel analytic strategies, including models defining OD symptoms as a quantitative trait adjusted for addiction to other substances. Such approaches minimize concern about the misclassification of controls because SD requires exposure to the drug. An overall picture emerges of genes and pathways involved in regulating calcium and potassium signaling as major contributors to OD risk.</p><p id="P35">Our study has several limitations. The GWS and other top-ranked findings were identified with imputed SNPs; however, empirically, our results show that the top-ranked SNPs in our study were imputed with excellent quality. Although our strongest results were with imputed SNPs, they are not rare. (Two had MAF=0.08; two, 0.06; and one, 0.03.) The primary concern about rare (MAF&lt;1%) SNPs is the greater rate of false positive results possible due to large effect sizes (i.e., greatly increased risk) derived from small allele frequency differences between cases and controls or genotyping errors. However, studies examining this issue have concluded that nominally significant results occur less frequently than expected, even for low MAF SNPs, provided there is no genotyping or imputation quality bias between cases and controls (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048647">37</a>, <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_383048643">38</a>). Moreover, the concern about false positives is substantially less when the results are replicated or supported by evidence from multiple samples, which is the case for the associations that we reported. Also, although our sample size is reasonable, experience has shown that many true associations are detected only in samples or meta-analyses that are much larger. The relatively small size of the sample used in Phase 3 for the primary trait analyses and as replication for the OD subtype findings is in our view the most likely cause of few replication observations in that sample. Specifically, there were proportionally fewer AAs available for Phase 3 (n= 805, ~24% as large as the discovery sample), which is the part of the sample in which most positive results were initially observed. Ideally, the replication sample would be larger than the discovery sample. It is also noteworthy that there was little evidence of association in EAs for any of our top findings in AAs, however, all but one of the SNPs reported in <a href="/pmc/articles/PMC3992201/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> were monomorphic in EAs. Finally, our findings are not adjusted for testing association in two populations and with two trait models (case-control and ordinal). Bonferroni correction is too conservative given the high correlation between the traits and distinct hypotheses for EAs and AAs, populations that often have distinctive risk loci and/or distinctive risk alleles at the same locus. Future studies in large independent populations are necessary to address some of these concerns.</p><p id="P36" class="p p-last">In conclusion, we identified numerous novel GWS associations with OD. We found compelling evidence for association in several brain systems, most consistently for calcium and potassium signaling pathways, which should provide the impetus for further research on these brain systems in OD. Further, these findings are largely consonant with a set of SNP and pathway-based findings that are emerging for a broader range of neuropsychiatric traits, and as such, may bridge OD and other neuropsychiatric phenotypes.</p></div><div id="S16" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><div class="sup-box half_rhythm" id="d36e861"><a href="/pmc/articles/PMC3992201/bin/NIHMS525621-supplement-01.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.4M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>02</h4><div class="sup-box half_rhythm" id="d36e865"><a href="/pmc/articles/PMC3992201/bin/NIHMS525621-supplement-02.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(154K, xls)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgments</h2><div class="sec"><p id="__p1">This study was supported by National Institutes of Health grants RC2 DA028909, R01 DA12690, R01 DA12849, R01 DA18432, R01 AA11330, R01 AA017535, and the VA Connecticut and Philadelphia VA MIRECCs. We appreciate the work in recruitment and assessment provided at Yale University School of Medicine and the APT Foundation by James Poling, Ph.D.; at McLean Hospital by Roger Weiss, M.D., at the Medical University of South Carolina by Kathleen Brady, M.D., Ph.D. and Raymond Anton, M.D., and at the University of Pennsylvania by David Oslin, M.D. Genotyping services for a part of our GWAS study were provided by the Center for Inherited Disease Research (CIDR) and Yale University (Center for Genome Analysis). We are grateful to Ann Marie Lacobelle, Michelle Cucinelli, Christa Robinson, and Greg Dalton-Kay for their excellent technical assistance, to the SSADDA interviewers, led by Yari Nuñez and Michelle Slivinsky, who devoted substantial time and effort to phenotype the study sample and to John Farrell and Alexan Mardigan for database management assistance.</p><p id="__p2">The publicly available datasets used for the analyses described in this manuscript were obtained from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000092.v1.p1</a> through dbGaP accession number phs000092.v1.p.</p></div></div><div id="idm140680102183728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140680102183728title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p3" class="p p-first"><strong>Financial disclosures</strong></p><p id="__p4" class="p p-last">Dr. Kranzler has served as a consultant or advisory board member for Alkermes, Lilly, Lundbeck, Pfizer and Roche. He is also a member of the American Society of Clinical Psychopharmacology’s Alcohol Clinical Trials Initiative, which is supported by Lilly, Lundbeck, Abbott, and Pfizer. The other authors declare no biomedical financial interests or potential conflicts of interest.</p><p></p><p class="fn sec" id="FN3"></p><p id="__p5" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140680102179872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140680102179872title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Office of National Drug Control Policy. <span class="ref-journal">The Economic Costs of Drug Abuse in the United States, 1992–2002.</span> Washington, DC: Executive Office of the President; 2004.  (Publication No. 207303) <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Economic+Costs+of+Drug+Abuse+in+the+United+States,+1992–2002&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Goldman D, Oroszi G, Ducci F. The genetics of addictions: uncovering the genes. <span><span class="ref-journal">Nature Reviews Genetics. </span>2005;<span class="ref-vol">6</span>:521–532.</span> [<a href="/pubmed/15995696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Reviews+Genetics&amp;title=The+genetics+of+addictions:+uncovering+the+genes&amp;author=D+Goldman&amp;author=G+Oroszi&amp;author=F+Ducci&amp;volume=6&amp;publication_year=2005&amp;pages=521-532&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Thorgeirsson TE, Geller F, Sulem P, Rafnar T, Wiste A, Magnusson KP, et al.  A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:638–642.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539558/">PMC free article</a>]</span> [<a href="/pubmed/18385739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+variant+associated+with+nicotine+dependence,+lung+cancer+and+peripheral+arterial+disease&amp;author=TE+Thorgeirsson&amp;author=F+Geller&amp;author=P+Sulem&amp;author=T+Rafnar&amp;author=A+Wiste&amp;volume=452&amp;publication_year=2008&amp;pages=638-642&amp;pmid=18385739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gelernter J, Sherva R, Koesterer R, Almasy L, Zhao H, Kranzler HR, Farrer LA. Genomewide association study of cocaine dependence and related traits: <em>FAM53B</em> identified as a risk gene. <span><span class="ref-journal">Molecular Psychiatry </span>2013</span> In press. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3865158/">PMC free article</a>]</span> [<a href="/pubmed/23958962" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+Psychiatry&amp;title=Genomewide+association+study+of+cocaine+dependence+and+related+traits:+FAM53B+identified+as+a+risk+gene&amp;author=J+Gelernter&amp;author=R+Sherva&amp;author=R+Koesterer&amp;author=L+Almasy&amp;author=H+Zhao&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Edenberg HJ. The collaborative study on the genetics of alcoholism: an update. <span><span class="ref-journal">Alcohol Res Health. </span>2002;<span class="ref-vol">26</span>:214–218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6683843/">PMC free article</a>]</span> [<a href="/pubmed/12875050" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Res+Health&amp;title=The+collaborative+study+on+the+genetics+of+alcoholism:+an+update&amp;author=HJ+Edenberg&amp;volume=26&amp;publication_year=2002&amp;pages=214-218&amp;pmid=12875050&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Bierut LJ, Strickland JR, Thompson JR, Afful SE, Cottler LB. Drug use and dependence in cocaine dependent subjects, community-based individuals, and their siblings. <span><span class="ref-journal">Drug Alcohol Depend. </span>2008;<span class="ref-vol">95</span>:14–22.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2384165/">PMC free article</a>]</span> [<a href="/pubmed/18243582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Alcohol+Depend&amp;title=Drug+use+and+dependence+in+cocaine+dependent+subjects,+community-based+individuals,+and+their+siblings&amp;author=LJ+Bierut&amp;author=JR+Strickland&amp;author=JR+Thompson&amp;author=SE+Afful&amp;author=LB+Cottler&amp;volume=95&amp;publication_year=2008&amp;pages=14-22&amp;pmid=18243582&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Gelernter J, Panhuysen C, Wilcox M, Hesselbrock V, Rounsaville B, Poling J, et al.  Genomewide linkage scan for opioid dependence and related traits. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>:759–769.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1474044/">PMC free article</a>]</span> [<a href="/pubmed/16642432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genomewide+linkage+scan+for+opioid+dependence+and+related+traits&amp;author=J+Gelernter&amp;author=C+Panhuysen&amp;author=M+Wilcox&amp;author=V+Hesselbrock&amp;author=B+Rounsaville&amp;volume=78&amp;publication_year=2006&amp;pages=759-769&amp;pmid=16642432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Gelernter J, Panhuysen C, Weiss R, Brady K, Hesselbrock V, Rounsaville B, et al.  Genomewide linkage scan for cocaine dependence and related traits: Linkages for a cocaine-related trait and cocaine-induced paranoia. <span><span class="ref-journal">Am J Med Genet Neuropsych Genet. </span>2005;<span class="ref-vol">136</span>:45–52.</span> [<a href="/pubmed/15909294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+Neuropsych+Genet&amp;title=Genomewide+linkage+scan+for+cocaine+dependence+and+related+traits:+Linkages+for+a+cocaine-related+trait+and+cocaine-induced+paranoia&amp;author=J+Gelernter&amp;author=C+Panhuysen&amp;author=R+Weiss&amp;author=K+Brady&amp;author=V+Hesselbrock&amp;volume=136&amp;publication_year=2005&amp;pages=45-52&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Pierucci-Lagha A, Gelernter J, Feinn R, Cubells JF, Pearson D, Pollastri A, et al.  Diagnostic Reliability of the Semi-Structured Assessment for Drug Dependence and Alcoholism (SSADDA) <span><span class="ref-journal">Drug Alcohol Depend. </span>2005;<span class="ref-vol">80</span>:303–12.</span> [<a href="/pubmed/15896927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Alcohol+Depend&amp;title=Diagnostic+Reliability+of+the+Semi-Structured+Assessment+for+Drug+Dependence+and+Alcoholism+(SSADDA)&amp;author=A+Pierucci-Lagha&amp;author=J+Gelernter&amp;author=R+Feinn&amp;author=JF+Cubells&amp;author=D+Pearson&amp;volume=80&amp;publication_year=2005&amp;pages=303-12&amp;pmid=15896927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">American Psychiatric Association. <span class="ref-journal">Diagnostic and Statistical Manual of Mental Disorders.</span> 4. Washington DC: American Psychiatric Press; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Diagnostic+and+Statistical+Manual+of+Mental+Disorders&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Holland PM, Abramson RD, Watson R, Gelfand DH. Detection of specific polymerase chain reaction product by utilizing the 5′ →3′ exonuclease activity of Thermus aquaticus DNA polymerase. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1991;<span class="ref-vol">88</span>:7276–7280.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC52277/">PMC free article</a>]</span> [<a href="/pubmed/1871133" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+USA&amp;title=Detection+of+specific+polymerase+chain+reaction+product+by+utilizing+the+5′+→3′+exonuclease+activity+of+Thermus+aquaticus+DNA+polymerase&amp;author=PM+Holland&amp;author=RD+Abramson&amp;author=R+Watson&amp;author=DH+Gelfand&amp;volume=88&amp;publication_year=1991&amp;pages=7276-7280&amp;pmid=1871133&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al.  PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Bierut LJ, Madden PA, Breslau N, Johnson EO, Hatsukami D, Pomerleau OF, et al.  Novel genes identified in a high-density genome wide association study for nicotine dependence. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>(1):24–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2278047/">PMC free article</a>]</span> [<a href="/pubmed/17158188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Novel+genes+identified+in+a+high-density+genome+wide+association+study+for+nicotine+dependence&amp;author=LJ+Bierut&amp;author=PA+Madden&amp;author=N+Breslau&amp;author=EO+Johnson&amp;author=D+Hatsukami&amp;volume=16&amp;issue=1&amp;publication_year=2007&amp;pages=24-35&amp;pmid=17158188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A Flexible and Accurate Genotype Imputation Method for the Next Generation of Genome-Wide Association Studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529. doi: 10.1371/journal.pgen.1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1371%2Fjournal.pgen.1000529" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+Flexible+and+Accurate+Genotype+Imputation+Method+for+the+Next+Generation+of+Genome-Wide+Association+Studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;doi=10.1371/journal.pgen.1000529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Zeger SL, Liang KY. Longitudinal data analysis for discrete and continuous outcomes. <span><span class="ref-journal">Biometrics. </span>1986;<span class="ref-vol">42</span>(1):121–130.</span> [<a href="/pubmed/3719049" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Longitudinal+data+analysis+for+discrete+and+continuous+outcomes&amp;author=SL+Zeger&amp;author=KY+Liang&amp;volume=42&amp;issue=1&amp;publication_year=1986&amp;pages=121-130&amp;pmid=3719049&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Li J, Ji L. Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix. <span><span class="ref-journal">Heredity (Edinb) </span>2005;<span class="ref-vol">95</span>:221–7.</span> [<a href="/pubmed/16077740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heredity+(Edinb)&amp;title=Adjusting+multiple+testing+in+multilocus+analyses+using+the+eigenvalues+of+a+correlation+matrix&amp;author=J+Li&amp;author=L+Ji&amp;volume=95&amp;publication_year=2005&amp;pages=221-7&amp;pmid=16077740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Chiocco MJ, Zhu X, Walther D, Pletnikova O, Troncoso JC, Uhl GR, et al.  Fine mapping of calcineurin (PPP3CA) gene reveals novel alternative splicing patterns, association of 5′UTR trinucleotide repeat with addiction vulnerability, and differential isoform expression in Alzheimer’s disease. <span><span class="ref-journal">Subst Use Misuse. </span>2010;<span class="ref-vol">45</span>:1809–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3031160/">PMC free article</a>]</span> [<a href="/pubmed/20590401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Subst+Use+Misuse&amp;title=Fine+mapping+of+calcineurin+(PPP3CA)+gene+reveals+novel+alternative+splicing+patterns,+association+of+5′UTR+trinucleotide+repeat+with+addiction+vulnerability,+and+differential+isoform+expression+in+Alzheimer’s+disease&amp;author=MJ+Chiocco&amp;author=X+Zhu&amp;author=D+Walther&amp;author=O+Pletnikova&amp;author=JC+Troncoso&amp;volume=45&amp;publication_year=2010&amp;pages=1809-26&amp;pmid=20590401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Sklar P, Smoller JW, Fan J, Ferreira MA, Perlis RH, Chambert K, et al.  Whole-genome association study of bipolar disorder. <span><span class="ref-journal">Mol Psychiatry. </span>2008;<span class="ref-vol">13</span>:558–69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3777816/">PMC free article</a>]</span> [<a href="/pubmed/18317468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Whole-genome+association+study+of+bipolar+disorder&amp;author=P+Sklar&amp;author=JW+Smoller&amp;author=J+Fan&amp;author=MA+Ferreira&amp;author=RH+Perlis&amp;volume=13&amp;publication_year=2008&amp;pages=558-69&amp;pmid=18317468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Nyegaard M, Demontis D, Foldager L, Hedemand A, Flint TJ, Sørensen KM, et al.  CACNA1C (rs1006737) is associated with schizophrenia. <span><span class="ref-journal">Mol Psychiatry. </span>2010;<span class="ref-vol">15</span>:119–21.</span> [<a href="/pubmed/20098439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=CACNA1C+(rs1006737)+is+associated+with+schizophrenia&amp;author=M+Nyegaard&amp;author=D+Demontis&amp;author=L+Foldager&amp;author=A+Hedemand&amp;author=TJ+Flint&amp;volume=15&amp;publication_year=2010&amp;pages=119-21&amp;pmid=20098439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Cross-Disorder Group of the Psychiatric Genomics Consortium. Identification of risk loci with shared effects on five major psychiatric disorders: a genome-wide analysis. <span><span class="ref-journal">Lancet. </span>2013 Feb 27; doi: 10.1016/S0140-6736(12)62129-1.</span> S0140–6736(12)62129–1. [Epub ahead of print] <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3714010/">PMC free article</a>]</span> [<a href="/pubmed/23453885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1016%2FS0140-6736(12)62129-1" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Identification+of+risk+loci+with+shared+effects+on+five+major+psychiatric+disorders:+a+genome-wide+analysis&amp;publication_year=2013&amp;doi=10.1016/S0140-6736(12)62129-1&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Filosa JA, Bonev AD, Straub SV, Meredith AL, Wilkerson MK, Aldrich RW, et al.  Local potassium signaling couples neuronal activity to vasodilation in the brain. <span><span class="ref-journal">Nat Neurosci. </span>2006;<span class="ref-vol">9</span>:13971403.</span> [<a href="/pubmed/17013381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Local+potassium+signaling+couples+neuronal+activity+to+vasodilation+in+the+brain&amp;author=JA+Filosa&amp;author=AD+Bonev&amp;author=SV+Straub&amp;author=AL+Meredith&amp;author=MK+Wilkerson&amp;volume=9&amp;publication_year=2006&amp;pages=13971403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">North RA, Williams JT. Opiate activation of potassium conductance inhibits calcium action potentials in rat locus coeruleusneurones. <span><span class="ref-journal">Br J Pharmacol. </span>1983;<span class="ref-vol">80</span>:225–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2045013/">PMC free article</a>]</span> [<a href="/pubmed/6652378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Pharmacol&amp;title=Opiate+activation+of+potassium+conductance+inhibits+calcium+action+potentials+in+rat+locus+coeruleusneurones&amp;author=RA+North&amp;author=JT+Williams&amp;volume=80&amp;publication_year=1983&amp;pages=225-8&amp;pmid=6652378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Jiao Y, Jalan-Sakrikar N, Robison AJ, Baucum AJ, 2nd, Bass MA, Colbran RJ. Characterization of a central Ca2+/calmodulin-dependent protein kinase IIalpha/beta binding domain in densin that selectively modulates glutamate receptor subunit phosphorylation. <span><span class="ref-journal">J Biol Chem. </span>2010;<span class="ref-vol">286</span>:24806–18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3137056/">PMC free article</a>]</span> [<a href="/pubmed/21610080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Characterization+of+a+central+Ca2+/calmodulin-dependent+protein+kinase+IIalpha/beta+binding+domain+in+densin+that+selectively+modulates+glutamate+receptor+subunit+phosphorylation&amp;author=Y+Jiao&amp;author=N+Jalan-Sakrikar&amp;author=AJ+Robison&amp;author=AJ+Baucum&amp;author=MA+Bass&amp;volume=286&amp;publication_year=2010&amp;pages=24806-18&amp;pmid=21610080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Steketee JD, Kalivas PW. Drug wanting: Behavioral sensitization and relapse to drug-seeking behavior. <span><span class="ref-journal">Pharmacol Rev. </span>2011;<span class="ref-vol">63</span></span> A–R. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3082449/">PMC free article</a>]</span> [<a href="/pubmed/21490129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Rev&amp;title=Drug+wanting:+Behavioral+sensitization+and+relapse+to+drug-seeking+behavior&amp;author=JD+Steketee&amp;author=PW+Kalivas&amp;volume=63&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Xia Y, Portugal GS, Fakira AK, Melyan Z, Neve R, Lee HT, et al.  Hippocampal GluA1-containing AMPA receptors mediate context-dependent sensitization to morphine. <span><span class="ref-journal">J Neurosci. </span>2011;<span class="ref-vol">31</span>:16279–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3235051/">PMC free article</a>]</span> [<a href="/pubmed/22072679" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Hippocampal+GluA1-containing+AMPA+receptors+mediate+context-dependent+sensitization+to+morphine&amp;author=Y+Xia&amp;author=GS+Portugal&amp;author=AK+Fakira&amp;author=Z+Melyan&amp;author=R+Neve&amp;volume=31&amp;publication_year=2011&amp;pages=16279-91&amp;pmid=22072679&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Li Y, Hollingworth P, Moore P, Foy C, Archer N, Powell J, et al.  Genetic association of the APP binding protein 2 gene (APBB2) with late onset Alzheimer disease. <span><span class="ref-journal">Hum Mutat. </span>2005;<span class="ref-vol">25</span>:270–7.</span> [<a href="/pubmed/15714520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Genetic+association+of+the+APP+binding+protein+2+gene+(APBB2)+with+late+onset+Alzheimer+disease&amp;author=Y+Li&amp;author=P+Hollingworth&amp;author=P+Moore&amp;author=C+Foy&amp;author=N+Archer&amp;volume=25&amp;publication_year=2005&amp;pages=270-7&amp;pmid=15714520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Chen GB, Payne TJ, Lou XY, Ma JZ, Zhu J, Li MD. Association of amyloid precursor protein-binding protein, family B, member 1 with nicotine dependence in African and European American smokers. <span><span class="ref-journal">Hum Genet. </span>2008;<span class="ref-vol">124</span>(4):393–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5576171/">PMC free article</a>]</span> [<a href="/pubmed/18777128" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Association+of+amyloid+precursor+protein-binding+protein,+family+B,+member+1+with+nicotine+dependence+in+African+and+European+American+smokers&amp;author=GB+Chen&amp;author=TJ+Payne&amp;author=XY+Lou&amp;author=JZ+Ma&amp;author=J+Zhu&amp;volume=124&amp;issue=4&amp;publication_year=2008&amp;pages=393-8&amp;pmid=18777128&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">St Clair D, Blackwood D, Muir W, Carothers A, Walker M, Spowart G, et al.  Association within a family of a balanced autosomal translocation with major mental illness. <span><span class="ref-journal">Lancet. </span>1990;<span class="ref-vol">336</span>:13–16.</span> [<a href="/pubmed/1973210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Association+within+a+family+of+a+balanced+autosomal+translocation+with+major+mental+illness&amp;author=D+St+Clair&amp;author=D+Blackwood&amp;author=W+Muir&amp;author=A+Carothers&amp;author=M+Walker&amp;volume=336&amp;publication_year=1990&amp;pages=13-16&amp;pmid=1973210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Thomson PA, Parla JS, McRae AF, Kramer M, Ramakrishnan K, Yao J, et al.  708 Common and 2010 rare DISC1 locus variants identified in 1542 subjects: analysis for association with psychiatric disorder and cognitive traits. <span><span class="ref-journal">Mol Psychiatry. </span>2013 Jun 4; doi: 10.1038/mp.2013.68.</span> Epub ahead of print. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4031635/">PMC free article</a>]</span> [<a href="/pubmed/23732877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fmp.2013.68" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=708+Common+and+2010+rare+DISC1+locus+variants+identified+in+1542+subjects:+analysis+for+association+with+psychiatric+disorder+and+cognitive+traits&amp;author=PA+Thomson&amp;author=JS+Parla&amp;author=AF+McRae&amp;author=M+Kramer&amp;author=K+Ramakrishnan&amp;publication_year=2013&amp;doi=10.1038/mp.2013.68&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Kvajo M, McKellar H, Arguello PA, Drew LJ, Moore H, MacDermott AB, et al.  A mutation in mouse Disc1 that models a schizophrenia risk allele leads to specific alterations in neuronal architecture and cognition. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">13;105</span>:7076–7081.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2383956/">PMC free article</a>]</span> [<a href="/pubmed/18458327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+mutation+in+mouse+Disc1+that+models+a+schizophrenia+risk+allele+leads+to+specific+alterations+in+neuronal+architecture+and+cognition&amp;author=M+Kvajo&amp;author=H+McKellar&amp;author=PA+Arguello&amp;author=LJ+Drew&amp;author=H+Moore&amp;volume=13;105&amp;publication_year=2008&amp;pages=7076-7081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Xie P, Kranzler HR, Krystal JH, Farrer LA, Zhao H, Gelernter J. Deep resequencing of 17 glutamate system genes identifies rare variants in <em>DISC1</em> and <em>GRIN2B</em> affecting risk of opioid dependence. <span><span class="ref-journal">Addict Biol. </span>2013 Jul 16; doi: 10.1111/adb.12072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3815683/">PMC free article</a>]</span> [<a href="/pubmed/23855403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1111%2Fadb.12072" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addict+Biol&amp;title=Deep+resequencing+of+17+glutamate+system+genes+identifies+rare+variants+in+DISC1+and+GRIN2B+affecting+risk+of+opioid+dependence&amp;author=P+Xie&amp;author=HR+Kranzler&amp;author=JH+Krystal&amp;author=LA+Farrer&amp;author=H+Zhao&amp;publication_year=2013&amp;doi=10.1111/adb.12072&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Han S, Yang B-Z, Kranzler HR, Oslin D, Anton R, Farrer LA, Gelernter J. Linkage analysis followed by association show NRG1 associated with cannabis dependence in African-Americans. <span><span class="ref-journal">Biol Psychiatr. </span>2012;<span class="ref-vol">72</span>(8):637–44.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3699339/">PMC free article</a>]</span> [<a href="/pubmed/22520967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatr&amp;title=Linkage+analysis+followed+by+association+show+NRG1+associated+with+cannabis+dependence+in+African-Americans&amp;author=S+Han&amp;author=B-Z+Yang&amp;author=HR+Kranzler&amp;author=D+Oslin&amp;author=R+Anton&amp;volume=72&amp;issue=8&amp;publication_year=2012&amp;pages=637-44&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Moskvina V, Craddock N, Holmans P, Owen MJ, O’Donovan MC. Effects of differential genotyping error rate on the type I error probability of case-control studies. <span><span class="ref-journal">Hum Hered. </span>2006;<span class="ref-vol">61</span>:55–64.</span> [<a href="/pubmed/16612103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Effects+of+differential+genotyping+error+rate+on+the+type+I+error+probability+of+case-control+studies&amp;author=V+Moskvina&amp;author=N+Craddock&amp;author=P+Holmans&amp;author=MJ+Owen&amp;author=MC+O’Donovan&amp;volume=61&amp;publication_year=2006&amp;pages=55-64&amp;pmid=16612103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Tabangin ME, Woo JG, Martin LJ. The effect of minor allele frequency on the likelihood of obtaining false positives. <span><span class="ref-journal">BMC Proc. </span>2009;<span class="ref-vol">15</span>(Suppl 7):S41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795940/">PMC free article</a>]</span> [<a href="/pubmed/20018033" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Proc&amp;title=The+effect+of+minor+allele+frequency+on+the+likelihood+of+obtaining+false+positives&amp;author=ME+Tabangin&amp;author=JG+Woo&amp;author=LJ+Martin&amp;volume=15&amp;issue=Suppl+7&amp;publication_year=2009&amp;pages=S41&amp;pmid=20018033&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3992201&amp;issue-id=236544&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140680135962832title">Abstract</a></li><li><a href="#S5title">INTRODUCTION</a></li><li><a href="#S6title">MATERIALS AND METHODS</a></li><li><a href="#S13title">RESULTS</a></li><li><a href="#S15title">DISCUSSION</a></li><li><a href="#S16title">Supplementary Material</a></li><li><a href="#S17title">Acknowledgments</a></li><li><a href="#idm140680102183728title">Footnotes</a></li><li><a href="#idm140680102179872title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3992201/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3992201/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3992201/pdf/nihms525621.pdf">PDF (548K)</a></li> | <li><a href="#" data-citationid="PMC3992201" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992201%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992201%2F&amp;text=Genomewide%20association%20study%20of%20opioid%20dependence%3A%20multiple%20associations%20mapped%20to%20calcium%20and%20potassium%20pathways" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992201%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24143882" data-aiid="3992201" data-aid="3992201" data-iid="236544" data-domainid="319" data-domain="nihpa" data-accid="PMC3992201" data-md5="8b0d46ec223179c28b7ae9d874d73b30"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:27:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DD43FDD5603A100000000071C045A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3992201%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3992201/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>