<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3319015/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Whole genome association scan for genetic polymorphisms influencing information processing speed</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3319015/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biological psychology"><meta name="citation_title" content="Whole genome association scan for genetic polymorphisms influencing information processing speed"><meta name="citation_authors" content="Michelle Luciano, Narelle K. Hansell, Jari Lahti, Gail Davies, Sarah E. Medland, Katri Räikkönen, Albert Tenesa, Elisabeth Widen, Kevin A. McGhee, Aarno Palotie, David Liewald, David J. Porteous, John M. Starr, Grant W. Montgomery, Nicholas G. Martin, Johan G. Eriksson, Margaret J. Wright, Ian J. Deary"><meta name="citation_date" content="March 2011"><meta name="citation_issue" content="3"><meta name="citation_volume" content="86"><meta name="citation_firstpage" content="193"><meta name="citation_doi" content="10.1016/j.biopsycho.2010.11.008"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3319015/?report=abstract"><meta name="citation_pmid" content="21130836"><meta name="DC.Title" content="Whole genome association scan for genetic polymorphisms influencing information processing speed"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Michelle Luciano"><meta name="DC.Contributor" content="Narelle K. Hansell"><meta name="DC.Contributor" content="Jari Lahti"><meta name="DC.Contributor" content="Gail Davies"><meta name="DC.Contributor" content="Sarah E. Medland"><meta name="DC.Contributor" content="Katri Räikkönen"><meta name="DC.Contributor" content="Albert Tenesa"><meta name="DC.Contributor" content="Elisabeth Widen"><meta name="DC.Contributor" content="Kevin A. McGhee"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="David Liewald"><meta name="DC.Contributor" content="David J. Porteous"><meta name="DC.Contributor" content="John M. Starr"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="Johan G. Eriksson"><meta name="DC.Contributor" content="Margaret J. Wright"><meta name="DC.Contributor" content="Ian J. Deary"><meta name="DC.Date" content="2011 Mar"><meta name="DC.Identifier" content="10.1016/j.biopsycho.2010.11.008"><meta name="DC.Language" content="en"><meta property="og:title" content="Whole genome association scan for genetic polymorphisms influencing information processing speed"><meta property="og:type" content="article"><meta property="og:description" content="Processing speed is an important cognitive function that is compromised in psychiatric illness (e.g., schizophrenia, depression) and old age; it shares genetic background with complex cognition (e.g., working memory, reasoning). To find genes influencing ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3319015/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3319015/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3319015/pdf/ukmss-47308.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8865B3DD52186100000000001D0012.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3319015%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC3319015</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biol Psychol</a></span>. Author manuscript; available in PMC 2012 Apr 4.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21130836" target="pmc_ext" ref="reftype=publisher&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Biol Psychol. 2011 Mar; 86(3): 193–202. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Dec 3. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.biopsycho.2010.11.008" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.biopsycho.2010.11.008</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3319015</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>UKMS47308</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21130836">21130836</a></div></div></div></div><h1 class="content-title">Whole genome association scan for genetic polymorphisms influencing information processing speed</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Luciano%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564727664" co-class="co-affbox">Michelle Luciano</a>,<sup>a,</sup><sup>*,</sup><sup>1</sup> <a href="/pubmed/?term=Hansell%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131562384128" co-class="co-affbox">Narelle K. Hansell</a>,<sup>b,</sup><sup>1</sup> <a href="/pubmed/?term=Lahti%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564592784" co-class="co-affbox">Jari Lahti</a>,<sup>c</sup> <a href="/pubmed/?term=Davies%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564590912" co-class="co-affbox">Gail Davies</a>,<sup>a</sup> <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131557759936" co-class="co-affbox">Sarah E. Medland</a>,<sup>b</sup> <a href="/pubmed/?term=R%26%23x000e4%3Bikk%26%23x000f6%3Bnen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131557758064" co-class="co-affbox">Katri Räikkönen</a>,<sup>c</sup> <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564872832" co-class="co-affbox">Albert Tenesa</a>,<sup>d</sup> <a href="/pubmed/?term=Widen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564870848" co-class="co-affbox">Elisabeth Widen</a>,<sup>e</sup> <a href="/pubmed/?term=McGhee%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564015504" co-class="co-affbox">Kevin A. McGhee</a>,<sup>a</sup> <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564013520" co-class="co-affbox">Aarno Palotie</a>,<sup>e,</sup><sup>f,</sup><sup>g</sup> <a href="/pubmed/?term=Liewald%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131555810576" co-class="co-affbox">David Liewald</a>,<sup>a</sup> <a href="/pubmed/?term=Porteous%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131563959856" co-class="co-affbox">David J. Porteous</a>,<sup>h</sup> <a href="/pubmed/?term=Starr%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131563957872" co-class="co-affbox">John M. Starr</a>,<sup>i</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131550940784" co-class="co-affbox">Grant W. Montgomery</a>,<sup>b</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131550938800" co-class="co-affbox">Nicholas G. Martin</a>,<sup>b</sup> <a href="/pubmed/?term=Eriksson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131564908800" co-class="co-affbox">Johan G. Eriksson</a>,<sup>j,</sup><sup>k,</sup><sup>l,</sup><sup>m,</sup><sup>n</sup> <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131555219712" co-class="co-affbox">Margaret J. Wright</a>,<sup>b</sup> and  <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836" class="affpopup" co-rid="_co_idm140131559073424" co-class="co-affbox">Ian J. Deary</a><sup>a</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140131564727664"><h3 class="no_margin">Michelle Luciano</h3><p><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </p><div>Find articles by <a href="/pubmed/?term=Luciano%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Michelle Luciano</a></div></div><div id="_co_idm140131562384128"><h3 class="no_margin">Narelle K. Hansell</h3><p><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </p><div>Find articles by <a href="/pubmed/?term=Hansell%20NK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Narelle K. Hansell</a></div></div><div id="_co_idm140131564592784"><h3 class="no_margin">Jari Lahti</h3><p><sup>c</sup>Institute of Behavioural Sciences, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Lahti%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Jari Lahti</a></div></div><div id="_co_idm140131564590912"><h3 class="no_margin">Gail Davies</h3><p><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </p><div>Find articles by <a href="/pubmed/?term=Davies%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Gail Davies</a></div></div><div id="_co_idm140131557759936"><h3 class="no_margin">Sarah E. Medland</h3><p><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </p><div>Find articles by <a href="/pubmed/?term=Medland%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Sarah E. Medland</a></div></div><div id="_co_idm140131557758064"><h3 class="no_margin">Katri Räikkönen</h3><p><sup>c</sup>Institute of Behavioural Sciences, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=R%26%23x000e4%3Bikk%26%23x000f6%3Bnen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Katri Räikkönen</a></div></div><div id="_co_idm140131564872832"><h3 class="no_margin">Albert Tenesa</h3><p><sup>d</sup>MRC Human Genetics Unit, The Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh, UK </p><div>Find articles by <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Albert Tenesa</a></div></div><div id="_co_idm140131564870848"><h3 class="no_margin">Elisabeth Widen</h3><p><sup>e</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Widen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Elisabeth Widen</a></div></div><div id="_co_idm140131564015504"><h3 class="no_margin">Kevin A. McGhee</h3><p><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </p><div>Find articles by <a href="/pubmed/?term=McGhee%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Kevin A. McGhee</a></div></div><div id="_co_idm140131564013520"><h3 class="no_margin">Aarno Palotie</h3><p><sup>e</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Finland </p><p><sup>f</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK </p><p><sup>g</sup>Department of Medical Genetics, University of Helsinki and University Central Hospital, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Aarno Palotie</a></div></div><div id="_co_idm140131555810576"><h3 class="no_margin">David Liewald</h3><p><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </p><div>Find articles by <a href="/pubmed/?term=Liewald%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">David Liewald</a></div></div><div id="_co_idm140131563959856"><h3 class="no_margin">David J. Porteous</h3><p><sup>h</sup>University of Edinburgh Molecular Medicine Centre, Western General Hospital, Edinburgh, UK </p><div>Find articles by <a href="/pubmed/?term=Porteous%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">David J. Porteous</a></div></div><div id="_co_idm140131563957872"><h3 class="no_margin">John M. Starr</h3><p><sup>i</sup>Department of Geriatric Medicine, University of Edinburgh, Royal Victoria Hospital, UK </p><div>Find articles by <a href="/pubmed/?term=Starr%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">John M. Starr</a></div></div><div id="_co_idm140131550940784"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Grant W. Montgomery</a></div></div><div id="_co_idm140131550938800"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Nicholas G. Martin</a></div></div><div id="_co_idm140131564908800"><h3 class="no_margin">Johan G. Eriksson</h3><p><sup>j</sup>National Institute for Health and Welfare, Finland </p><p><sup>k</sup>Department of General Practice and Primary Health Care, University of Helsinki, Finland </p><p><sup>l</sup>Helsinki University Central Hospital, Unit of General Practice, Helsinki, Finland </p><p><sup>m</sup>Folkhalsan Research Centre, Helsinki, Finland </p><p><sup>n</sup>Vasa Central Hospital, Vasa, Finland </p><div>Find articles by <a href="/pubmed/?term=Eriksson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Johan G. Eriksson</a></div></div><div id="_co_idm140131555219712"><h3 class="no_margin">Margaret J. Wright</h3><p><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </p><div>Find articles by <a href="/pubmed/?term=Wright%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Margaret J. Wright</a></div></div><div id="_co_idm140131559073424"><h3 class="no_margin">Ian J. Deary</h3><p><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </p><div>Find articles by <a href="/pubmed/?term=Deary%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21130836">Ian J. Deary</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140131560572784_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140131560572784_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140131560572784_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>a</sup>Centre for Cognitive Aging and Cognitive Epidemiology, Department of Psychology, University of Edinburgh, Scotland, UK </div><div class="fm-affl" id="A2" lang="en"><sup>b</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane, Australia </div><div class="fm-affl" id="A3" lang="en"><sup>c</sup>Institute of Behavioural Sciences, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A4" lang="en"><sup>d</sup>MRC Human Genetics Unit, The Institute of Genetics and Molecular Medicine, Western General Hospital, Edinburgh, UK </div><div class="fm-affl" id="A5" lang="en"><sup>e</sup>Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Finland </div><div class="fm-affl" id="A6" lang="en"><sup>f</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge, UK </div><div class="fm-affl" id="A7" lang="en"><sup>g</sup>Department of Medical Genetics, University of Helsinki and University Central Hospital, Helsinki, Finland </div><div class="fm-affl" id="A8" lang="en"><sup>h</sup>University of Edinburgh Molecular Medicine Centre, Western General Hospital, Edinburgh, UK </div><div class="fm-affl" id="A9" lang="en"><sup>i</sup>Department of Geriatric Medicine, University of Edinburgh, Royal Victoria Hospital, UK </div><div class="fm-affl" id="A10" lang="en"><sup>j</sup>National Institute for Health and Welfare, Finland </div><div class="fm-affl" id="A11" lang="en"><sup>k</sup>Department of General Practice and Primary Health Care, University of Helsinki, Finland </div><div class="fm-affl" id="A12" lang="en"><sup>l</sup>Helsinki University Central Hospital, Unit of General Practice, Helsinki, Finland </div><div class="fm-affl" id="A13" lang="en"><sup>m</sup>Folkhalsan Research Centre, Helsinki, Finland </div><div class="fm-affl" id="A14" lang="en"><sup>n</sup>Vasa Central Hospital, Vasa, Finland </div><div id="FN1"><sup>1</sup>These authors contributed equally to this work.</div><div id="CR1"><sup>*</sup>Corresponding author at: Psychology, University of Edinburgh, 7 George Square, Edinburgh EH8 9JZ, UK. Tel.: +44 131 6 51 5040; fax: +44 131 650 8405. <a href="mailto:dev@null" data-email="ku.ca.de@onaicul.ellehcim" class="oemail">ku.ca.de@onaicul.ellehcim</a>. </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140131560572784_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21130836" target="pmc_ext" ref="reftype=publisher&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biol Psychol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3319015/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary data.</strong></div><div><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_data.doc" data-ga-action="click_feat_suppl">NIHMS47308-supplement-Supplementary_data.doc</a><span style="color:gray"> (173K)</span></div><div class="small guid">GUID:&nbsp;AF8D773A-629E-4731-B110-3C957D18B9F2</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary table 1.</strong></div><div><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_table_1.xls" data-ga-action="click_feat_suppl">NIHMS47308-supplement-Supplementary_table_1.xls</a><span style="color:gray"> (154K)</span></div><div class="small guid">GUID:&nbsp;DDB94CED-559B-4D93-925E-B094B5524134</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary table 2.</strong></div><div><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_table_2.xls" data-ga-action="click_feat_suppl">NIHMS47308-supplement-Supplementary_table_2.xls</a><span style="color:gray"> (71K)</span></div><div class="small guid">GUID:&nbsp;68761E9C-CAF5-4BD7-997D-B2EDBDFB7B16</div></div></dd></dl></div><div id="idm140131561838384" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140131561838384title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">Processing speed is an important cognitive function that is compromised in psychiatric illness (e.g., schizophrenia, depression) and old age; it shares genetic background with complex cognition (e.g., working memory, reasoning). To find genes influencing speed we performed a genome-wide association scan in up to three cohorts: Brisbane (mean age 16 years; <em>N</em> = 1659); LBC1936 (mean age 70 years, <em>N</em> = 992); LBC1921 (mean age 82 years, <em>N</em> = 307), and; HBCS (mean age 64 years, <em>N</em> = 1080). Meta-analysis of the common measures highlighted various suggestively significant (<em>p</em> &lt; 1.21 × 10<sup>−5</sup>) SNPs and plausible candidate genes (e.g., <em>TRIB3</em>). A biological pathways analysis of the speed factor identified two common pathways from the KEGG database (cell junction, focal adhesion) in two cohorts, while a pathway analysis linked to the GO database revealed common pathways across pairs of speed measures (e.g., receptor binding, cellular metabolic process). These highlighted genes and pathways will be able to inform future research, including results for psychiatric disease.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Information processing speed, Cognitive ability, Genes, Biological pathways</span></div></div><div id="idm140131596052352" class="tsec sec headless whole_rhythm"><p id="P3" class="p p-first">Information processing speed is considered a lower level cognitive process, although it shares a large portion of its genetic variance with higher order abilities (e.g., reasoning, working memory) (<a href="#R45" rid="R45" class=" bibr popnode">Luciano et al., 2003</a>; <a href="#R53" rid="R53" class=" bibr popnode">Neubauer, 1997</a>). It is a cognitive domain that is particularly prone to deterioration with ageing (<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874827">Salthouse, 1996</a>); for example, simple reaction time (RT) slows at around 50 years of age and choice RT slows throughout the adult range (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874795">Der and Deary, 2006</a>). However, similar to complex cognition with which it is correlated, the genes influencing speed are expected to be stable across the lifespan (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874830">Lyons et al., 2009</a>). Speed is affected (slowed) in psychiatric conditions such as schizophrenia, depression and substance use disorder (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874820">Andersson et al., 2010</a>; <a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874828">Jahshan et al., 2009</a>) and RT has even been shown to account for the relationship between IQ and mortality in a cohort followed up until age 70 (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874816">Deary and Der, 2005</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874806">Latvala et al., 2009</a>). It is therefore an important trait to understand, and in this study, we aim to locate genes influencing chronometric and psychometric processing speed measures via genome wide association. These genes may underlie complex cognition or reflect genetic influences specific to speed because a large portion of genetic variance in processing speed is independent of higher order ability.</p><p id="P4">Processing speed as measured by elementary cognitive tasks has been used to understand the genetic structure of human mental ability (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874808">Baker et al., 1991</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874784">Ho et al., 1988</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874818">Rijsdijk et al., 1998</a>; <a href="#R72" rid="R72" class=" bibr popnode">Wright et al., 2000</a>). Reaction time measures and measures without a RT component (such as inspection time) demonstrate phenotypic correlations ranging between −.30 to −.50 with IQ (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874819">Deary, 2001</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874826">Deary et al., 2001</a>; <a href="#R18" rid="R18" class=" bibr popnode">Deary and Stough, 1996</a>). More complex elementary cognitive tasks correlate more strongly with IQ, for example, choice RT confers a greater correlation with IQ than does simple RT (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874807">Deary et al., 2001</a>; <a href="#R36" rid="R36" class=" bibr popnode">Larson et al., 1988</a>). Psychometric speed measures, such as digit symbol, contribute to the estimation of a person’s full scale IQ, typically forming part of a first-order perceptual speed factor. Multivariate genetic analyses have found evidence for genetic pleiotropy among varied measures of processing speed (chronometric and psychometric) and, importantly, among speed and more complex cognitive abilities, notably IQ (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874790">Luciano et al., 2004</a>; <a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874789">Posthuma et al., 2001</a>; <a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874802">Rijsdijk et al., 1998</a>). For chronometric speed measures, common genes have been shown to influence the variation in processing speed and IQ rather than there being a causal relationship between the two (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874822">Luciano et al., 2005</a>), although in an older sample (50–89 years) and using psychometric speed measures a directional relationship from speed to spatial and memory abilities explained the genetic covariance (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342525213">Finkel et al., 2009</a>). Thus, the possibility exists that some information processing components serve as the bases for complex cognition, with the slowing of processing speed accounting for the ageing of other cognitive abilities.</p><p id="P5">The search for genetic polymorphisms influencing information processing speed variables has been unimpressive: the first whole genome linkage scan (in 378 families) of a collection of speed measures reported no linked chromosomal regions at a genome-wide significance criterion (<a href="#R71" rid="R71" class=" bibr popnode">Wright et al., 2008</a>). However, there were regions of suggestive significance reported for 8-choice RT mean (on chromosomes 1 and 11), 4-choice RT mean (chromosome 8), and a delayed response RT measure (chromosome 14). It was noted that some of these linkage peaks overlapped with peaks observed for IQ, although most of the peaks were specific to individual speed measures. Specific candidate genes influencing processing speed measures have been largely unsubstantiated, but there have been findings of association with the TNF-alpha-308G→A polymorphism and a speed factor in the elderly (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874805">Baune et al., 2008</a>), <em>BDNF</em> and a psychometric speed measure (alphabet-coding task) (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308224689">Miyajima et al., 2008</a>), and between <em>APOE</em> e4 allele variation and both chronometric and psychometric speed measures in a sample of 70 year olds (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874831">Luciano et al., 2009</a>).</p><p id="P6" class="p">The present study uses the technique of genome wide association (GWA) to identify genetic variants influencing processing speed measures. Four cohorts from Australia, Scotland, and Finland were genotyped on the Illumina 610k Quad array and had been measured on information processing speed measures, some of which overlapped in, at most, three cohorts (e.g., choice RT, inspection time, and digit symbol), enabling meta-analysis of these measures. While the cohorts varied widely in age, processing speed has been shown to be fairly stable from childhood to age 70 (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874813">Deary et al., 2010</a>), therefore, our study will be able to detect genetic variants influencing stable variation in processing speed.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">1. Methods</h2><div id="S2" class="sec sec-first"><h3 id="S2title">1.1. GWA study subjects</h3><div id="S3" class="sec sec-first"><p></p><h4 id="S3title" class="inline">1.1.1. Brisbane adolescent twin sample – Australia </h4><p id="P7" class="p p-first-last">Twins and their non-twin siblings were initially recruited as part of ongoing studies of melanoma risk factors and cognition (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874815">McGregor et al., 1999</a>; <a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874793">Wright et al., 2001</a>). For the current study, processing speed and genotypic data, following quality control described below, were available for a maximum of 1659 individuals (from 730 families), of whom 266 were monozygotic twin pairs and for whom phenotypic data were averaged. When data were collected, participants ranged in age from 15.4 to 28.7 (mean = 16.4 years, SD = .8) with the sample being Caucasian, predominantly Anglo-celtic (i.e., ethnic outliers were excluded (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874792">Medland et al., 2009</a>)). Fifty-three percent of the sample was female. Written informed consent was obtained from each participant and their parent/guardian (if younger than 18 years) prior to testing.</p></div><div id="S4" class="sec"><p></p><h4 id="S4title" class="inline">1.1.2. Lothian Birth Cohort 1936 (LBC1936) – Scotland </h4><p id="P8" class="p p-first-last">This cohort consists of 1091 relatively healthy individuals assessed on cognitive and medical traits at about 70 years of age. They were all born in 1936 and most took part at age 11 in the Scottish Mental Survey of 1947. They were Caucasian and almost all lived independently in the Lothian region (Edinburgh city and surrounding area) of Scotland. A full description of participant recruitment and testing can be found elsewhere (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874825">Deary et al., 2007</a>). Genomic DNA was isolated from 1071 participants by standard procedure at the Wellcome Trust Clinical Research Facility (WTCRF) Genetics Core, Western General Hospital, Edinburgh. Twenty-nine samples failed quality control preceding the genotyping procedure. A reduced sample of 992 individuals had both genotype and phenotype data; their mean age was 69.55 years (SD = .84, range = 67.6–71.3 years) at data collection.</p></div><div id="S5" class="sec"><p></p><h4 id="S5title" class="inline">1.1.3. Lothian Birth Cohort 1921 (LBC1921) – Scotland </h4><p id="P9" class="p p-first-last">This cohort comprises 550 individuals, who were tested on cognitive and medical indices on multiple occasions; at first test wave participants were aged ~79 years (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874788">Deary et al., 2004b</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874782">Houlihan et al., 2010</a>). They were all born in 1921 and most took part at age 11 in the Scottish Mental Survey of 1932. They were Caucasian and almost all lived independently in the Lothian region (Edinburgh city and surrounding area) of Scotland. The data for this analysis was based on the second test occasion, when they were measured on all three relevant speed traits. Genomic DNA was isolated from whole blood from 542 participants by standard procedure at Medical Research Council Technology, Western General Hospital, Edinburgh. Sixteen samples failed quality preceding the genotyping procedure. A reduced sample of 302 individuals had both genotype and phenotype data; their mean age was 83.35 years (SD = .54, range = 82.0–84.6 years) at data collection.</p></div><div id="S6" class="sec sec-last"><p></p><h4 id="S6title" class="inline">1.1.4. The Helsinki Birth Cohort Study (HBCS) – Finland </h4><p id="P10" class="p p-first-last">The HBCS cohort is composed of 8760 individuals born between the years 1934 and 1944 in Helsinki University Central Hospital. Between 2001 and 2003, a randomly selected sample of 928 males and 1075 females participated in a clinical study with a focus on cardiovascular, metabolic and reproductive health, cognitive function and depressive symptoms. For the sub-study on cognitive performance we invited those subjects from the original sample, who were still living in the greater Helsinki area (<em>n</em> = 1586). The 1279 subjects who attended were administered a test of cognitive performance in the years 2004–2006. After exclusion of subjects with a history of stroke (<em>n</em> = 29), type 1 diabetes (<em>n</em> = 1) and invalid test results (<em>n</em> = 3), the sample with valid phenotype and genotype data available for analyses consisted of 1080 men (<em>n</em> = 472) and women (<em>n</em> = 608). The mean age of the subjects at time of assessment was 63.9 (SD = 2.8) years. The mean age of the men was 63.6 (SD = 2.6) and of the women was 64.1 (SD = 3.0) years. The subjects who attended the cognitive test were similar in age, sex, educational attainment and prevalence and duration of diabetes to those who did not attend. Detailed information on the selection of the HBCS participants and on the study design can be found elsewhere (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874801">Barker et al., 2005</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874785">Eriksson et al., 2006</a>; <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874810">Paile-Hyvarinen et al., 2009</a>; <a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874800">Raikkonen et al., 2008</a>).</p></div></div><div id="S7" class="sec"><h3 id="S7title">1.2. Measures</h3><div id="S8" class="sec sec-first"><p></p><h4 id="S8title" class="inline">1.2.1. Brisbane adolescent twin sample </h4><p id="P11" class="p p-first-last">Psychometric speed was measured by the digit symbol substitution test, a performance subtest of the Wechsler Adult Intelligence Scale – Revised (<a href="#R68" rid="R68" class=" bibr popnode">Wechsler, 1981</a>) which requires the participant to pair random digits with their matching symbols. Inspection time and choice RT were the elementary cognitive tasks administered. Inspection time was measured using a backward masked, line discrimination task. A parameter estimation by sequential testing procedure was employed (<a href="#R24" rid="R24" class=" bibr popnode">Findlay, 1978</a>; <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874803">Pentland, 1980</a>) to vary stimulus duration. To minimise bias from random responses and lapses in attention/interest factors, inspection time was estimated by fitting post hoc a cumulative normal curve (mean = 0) to accuracy as a function of stimulus onset asynchrony. See <a href="#R44" rid="R44" class=" bibr popnode">Luciano et al. (2001b)</a> for more details. Mean RT of correct responses on a visual reaction time task with three choice conditions (two, four, and eight) was performed on a computer using a keyboard response pad. The number of trials presented in each of the two, four, and eight choice conditions was 96, 48, and 96, respectively. RT trials less than 150 ms or greater than 2000 ms were excluded from the calculation of the mean. See <a href="#R43" rid="R43" class=" bibr popnode">Luciano et al. (2001a)</a> for a fuller description of this task. A general factor, explaining 51% of variance, was calculated from a principal components analysis of digit symbol, 4-choice RT and inspection time.</p></div><div id="S9" class="sec"><p></p><h4 id="S9title" class="inline">1.2.2. LBC1936 </h4><p id="P12" class="p p-first-last">The information processing speed battery comprised two psychometric tests from the WAIS-III<sup>UK</sup> (digit symbol coding and symbol search) and two elementary cognitive tasks, RT and inspection time, which will be described briefly here; a full account of each test is given in <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874787">Deary et al. (2007)</a>. Mean of correct trials on a simple RT (20 trials) and 4-choice RT (40 trials) task were used to assess speed of information processing. For simple RT, the participant is required to press a response key as fast as possible following the occurrence of a zero on the LCD screen. Four-choice RT requires the participant to press the corresponding response key when a target of 1, 2, 3 or 4 appears on the screen. The inspection time task is a two-alternative, forced choice, backward masking, visual discrimination task, requiring participants to make a discrimination, with no pressure on response time, regarding which of two parallel, vertical lines of markedly different lengths was longer. The stimuli and psychophysical procedure were the same as those used in <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874823">Deary et al. (2004a)</a>, with a prescribed number of trials given at different stimulus durations. The correctness of each response across the 150 trials was recorded as the outcome variable. A general factor, explaining 59% of variance, was calculated from a principal components analysis of digit symbol, 4-choice RT mean and inspection time.</p></div><div id="S10" class="sec"><p></p><h4 id="S10title" class="inline">1.2.3. LBC1921 </h4><p id="P13" class="p p-first-last">Digit symbol, simple RT, 4-choice RT and inspection time were measured as per the protocol described for the LBC1936. A general factor, explaining 63% of variance, was calculated from a principal components analysis of digit symbol, 4-choice RT mean and inspection time.</p></div><div id="S11" class="sec sec-last"><p></p><h4 id="S11title" class="inline">1.2.4. HBCS </h4><p id="P14" class="p p-first-last">Psychomotor speed was measured with two tests being part of a standardised language independent computerized battery of cognitive tests (CogState1, version 3.0.5). This battery has been validated and shown to be a sensitive indicator of mild impairments in the following cognitive domains: psychomotor speed, attention, working memory and episodic learning and memory (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874812">Collie et al., 2001</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874797">2003</a>; <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874796">Darby et al., 2002</a>). In the tasks participants were asked to pay attention to playing cards on a computer screen. The test battery takes approximately 15 min to complete and is preceded by a practice round. In the simple RT task, which assesses psychomotor function and speed of processing, a single card is presented face-down on the computer screen. The subject is asked to press the spacebar as soon as the card turns face-up. This is repeated 35 times with random time intervals. This task is administered twice – first and last in the test battery – and the mean RT is calculated. In the 2-choice RT task, subjects are asked to indicate whether the turning playing card on the screen is red by pressing either K (“yes”) or D (“no”). The stimulus is repeated 30 times and the mean RT is calculated.</p></div></div><div id="S12" class="sec"><h3 id="S12title">1.3. Genotyping, quality control and imputation</h3><p id="P15" class="p p-first">DNA was extracted from blood samples and SNP genotyping was performed with the Illumina 610k Quad Bead chip either by deCODE Genetics (Brisbane cohort) or WTCRF (LBC1921 and LBC1921 cohorts) or with a modified Illumina 610k chip by the Wellcome Trust Sanger Institute, Cambridge, UK (HBCS). In the Brisbane adolescent twin cohort, genotype data (1817 samples) were checked for pedigree, sex, and Mendelian errors, and for ancestry (using HapMap3 and GenomeEUTwin individuals as a reference panel). Five individuals were removed because of gender inconsistencies, and 28 individuals (14 twin pairs) because of non-European ancestry. Quality control filters, as previously described (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874798">Medland et al., 2009</a>), ensured no samples had a call rate ≤.95, and that all SNPs included in analyses had the following characteristics: call rate ≥.95, minor allele frequency ≥.01, and HWE test with <em>P</em> ≥ 1 × 10<sup>−6</sup>. In the LBC cohorts (LBC1936 = 1042 and LBC1921 = 526 samples), individuals were checked for disagreement between genetic and reported gender (<em>n</em> = 12 in LBC1936 and <em>n</em> = 1 in LBC1921). Relatedness between subjects was investigated and, for any related pair of individuals, one was removed (<em>n</em> = 8, LBC1936; <em>n</em> = 1, LBC1921). Samples with a call rate ≤.95 (<em>n</em> = 16, LBC1936; <em>n</em> = 6, LBC1921), and those showing evidence of non-caucasian ascent, were also removed (<em>n</em> = 1, LBC1936; <em>n</em> = 2, LBC1921). SNPs were included in the analyses if they met the following conditions: call rate ≥.98, minor allele frequency ≥.01, and HWE test with <em>P</em> ≥.001. Thus after quality control procedures 1005 and 517 samples remained for the LBC1936 and LBC1921 cohorts respectively. In the HBCS, none of the participants showed non-European ancestry. Relatedness of the participants was examined with the pair-wise IBD estimates and closely related individuals were excluded from the analyses. Moreover, participants with X-chromosomal genotypes discrepant with the reported sex were removed (<em>N</em> = 8). After quality control procedures 1728 samples remained for the analyses. SNPs were included in the analyses if they met the following conditions: call rate ≥.95, minor allele frequency ≥.01, and HWE test with <em>p</em> ≥ 1 × 10<sup>−5</sup>.</p><p id="P16" class="p p-last">Because all cohorts used the same genotyping chip, the genotyped marker panel was analysed, but in the Australian and Scottish cohorts imputed genotypes were used where cases and SNPs were missing within samples. This ensured that the full panel of markers was available in each cohort. The HapMap phase II CEU data (NCBI build 36 (UCSC hg18)) was used as the reference sample for imputation using MACH software. In the Australian and Scottish cohorts, SNPs with low imputation (<em>r</em><sup>2</sup> &lt; .30) were excluded, and in all cohorts, SNPs with low minor allele frequency (MAF &lt; .01) were excluded.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">1.4. Statistical analyses</h3><p id="P17" class="p p-first">Genome-wide association analyses were conducted in each study independently using linear regression (under an additive model) of standardised measures and including sex and age as covariates. We used the family based score test implemented in Merlin for the Brisbane cohort, and PLINK software for the Scottish and Finnish cohorts.</p><p id="P18">METAL (<a href="#R1" rid="R1" class=" bibr popnode">Abecasis, 2010</a>) was used to perform meta-analysis of the results; it uses the weighted inverse variance method, computing a pooled effect estimate (ln(beta)), its standard error and its <em>p</em>-value by weighting the effect estimates of each cohort by the inverse of its variance and by incorporating the direction of effect. Automatic correction for any population stratification effects, if necessary, was done by applying genomic control, although individual cohort lambda values for each variable ranged between .97 and 1 (see <a href="#SD1" rid="SD1" class=" supplementary-material">Appendix CSupplementary Fig. 1</a> for genomic control inflation values for reported results). A chi-square test for heterogeneity tested whether the regression coefficients differed significantly between the cohorts contributing to the meta-analysis.</p><p id="P19">We used a genome-wide significance level of <em>p</em> &lt; 1.21 × 10<sup>−8</sup> and a suggestive level of significance of <em>p</em> &lt; 1.21 × 10<sup>−5</sup> as proposed for the Illumina 317K panel of markers (adjusted for SNP non-independence) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874804">Duggal et al., 2008</a>). While this is a liberal criterion because we used a larger chip, no SNPs reached genome-wide significance. Using the genome-wide significance alpha level, power calculations for our maximum sample were 36% to detect an effect size of 1% which increased to 80% for an effect size of 1.5%; the more liberal suggestive significance level gave a power of 83% to detect an effect size of 1%. We also present supplementary results of our top 100 SNPs because evidence is accumulating that for complex traits (e.g., height) effect sizes are too small to pass the stringent significance tests of genome-wide association (<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874817">Yang et al., 2010</a>).</p><p id="P20" class="p p-last">Biological pathways analysis was undertaken for all variables using ALIGATOR (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874791">Holmans et al., 2009</a>) and for the speed factor only using the SNPratio test (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308224690">O’Dushlaine et al., 2009</a>). Because ALIGATOR can use meta-analysis <em>p</em>-values to generate results, all variables were run through this program. SNPratio, on the other hand, is computer intensive requiring raw genotype data, therefore, only one variable was analysed (the composite speed factor) and this was done separately by cohort. In brief, ALIGATOR counts the number of significant genes (based on a nominal SNP association) in Gene Ontology (GO) categories that arise more often than would be expected by chance (empirical <em>p</em>-values of GO category membership are established via permutation of randomly drawn genes). We used a <em>p</em>-value of .01 for inclusion of nominally significant SNPs, 5000 replicate gene lists and 1000 replicate studies (used to judge study-wide significance levels for individual categories). The SNPratio test compares the proportion of nominally significant (<em>p</em> &lt; .05) SNPs within genes to all SNPs within genes that relate to a specific biological pathway (from 220 experimentally validated pathways described in the Kyoto Encyclopaedia of Genes and Genomes (KEGG)). The NCBI definition of gene boundaries (including the 5 kb upstream and 2 kb downstream flanking region), was used to define a gene. An empirical <em>p</em>-value is estimated from the comparisons to ratios in datasets where the trait score has been randomised (1000 permutations). Because PLINK output was used for this analysis, the Brisbane cohort was re-run using PLINK on a randomly selected single member from each family. It should be noted that pathway categories differ between GO and KEGG databases. The GO categories are descriptors of gene products that fall into three broadly defined groups: biological process, cellular component and molecular function, whereas the KEGG database contain pathways that have been constructed through knowledge of molecular interaction and reaction networks.</p></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">2. Results</h2><p id="P21" class="p p-first">Descriptive statistics of the measures, including their correlation with age and with other variables, are shown in <a href="/pmc/articles/PMC3319015/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. No SNPs exceeded the genome-wide significance level for any of the traits; q-q plots with their 95% confidence intervals for all measures can be found in the <a href="#SD1" rid="SD1" class=" supplementary-material">Appendix CSupplementary Fig. 1</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p4">Mean scores of the processing speed measures in the Australian, Scottish and Finnish studies, stratified across sex, and their correlations with age and with other variables.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140131564805744" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>N</em></th><th colspan="2" rowspan="1" valign="middle" align="right">Total sample<hr></th><th colspan="2" rowspan="1" valign="middle" align="right">Men<hr></th><th colspan="2" rowspan="1" valign="middle" align="right">Women<hr></th><th rowspan="1" colspan="1" valign="middle" align="center">Correlation with age</th><th colspan="5" rowspan="1" valign="middle" align="center">Inter-trait correlations (all significant <em>p</em> &lt; .001)<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">SD</th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">SD</th><th rowspan="1" colspan="1" valign="middle" align="right">Mean</th><th rowspan="1" colspan="1" valign="middle" align="right">SD</th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center">1. DS</th><th rowspan="1" colspan="1" valign="middle" align="center">2. IT</th><th rowspan="1" colspan="1" valign="middle" align="center">3. 2CRT</th><th rowspan="1" colspan="1" valign="middle" align="center">4. 4CRT</th><th rowspan="1" colspan="1" valign="middle" align="center">5. 8CRT</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Brisbane</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1.&nbsp;Digit&nbsp;Symbol (no. correct in 90 s)</td><td rowspan="1" colspan="1" valign="middle" align="center">1659</td><td rowspan="1" colspan="1" valign="middle" align="right">59.25</td><td rowspan="1" colspan="1" valign="middle" align="right">11.05</td><td rowspan="1" colspan="1" valign="middle" align="right">54.66</td><td rowspan="1" colspan="1" valign="middle" align="right">10.22</td><td rowspan="1" colspan="1" valign="middle" align="right">63.39</td><td rowspan="1" colspan="1" valign="middle" align="right">10.10</td><td rowspan="1" colspan="1" valign="middle" align="center">.05<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2. Inspection Time (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1388</td><td rowspan="1" colspan="1" valign="middle" align="right">92.40</td><td rowspan="1" colspan="1" valign="middle" align="right">65.79</td><td rowspan="1" colspan="1" valign="middle" align="right">92.41</td><td rowspan="1" colspan="1" valign="middle" align="right">75.26</td><td rowspan="1" colspan="1" valign="middle" align="right">92.39</td><td rowspan="1" colspan="1" valign="middle" align="right">56.19</td><td rowspan="1" colspan="1" valign="middle" align="center">−.05<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">−.21</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3. 2-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1531</td><td rowspan="1" colspan="1" valign="middle" align="right">308.61</td><td rowspan="1" colspan="1" valign="middle" align="right">39.97</td><td rowspan="1" colspan="1" valign="middle" align="right">311.20</td><td rowspan="1" colspan="1" valign="middle" align="right">41.93</td><td rowspan="1" colspan="1" valign="middle" align="right">306.32</td><td rowspan="1" colspan="1" valign="middle" align="right">38.04</td><td rowspan="1" colspan="1" valign="middle" align="center">−.02</td><td rowspan="1" colspan="1" valign="middle" align="center">−.33</td><td rowspan="1" colspan="1" valign="middle" align="center">.15</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4. 4-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1532</td><td rowspan="1" colspan="1" valign="middle" align="right">487.69</td><td rowspan="1" colspan="1" valign="middle" align="right">107.47</td><td rowspan="1" colspan="1" valign="middle" align="right">490.56</td><td rowspan="1" colspan="1" valign="middle" align="right">108.35</td><td rowspan="1" colspan="1" valign="middle" align="right">485.16</td><td rowspan="1" colspan="1" valign="middle" align="right">106.68</td><td rowspan="1" colspan="1" valign="middle" align="center">−.07<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">−.34</td><td rowspan="1" colspan="1" valign="middle" align="center">.21</td><td rowspan="1" colspan="1" valign="middle" align="center">.46</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5. 8-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1382</td><td rowspan="1" colspan="1" valign="middle" align="right">602.66</td><td rowspan="1" colspan="1" valign="middle" align="right">100.47</td><td rowspan="1" colspan="1" valign="middle" align="right">602.85</td><td rowspan="1" colspan="1" valign="middle" align="right">105.39</td><td rowspan="1" colspan="1" valign="middle" align="right">602.49</td><td rowspan="1" colspan="1" valign="middle" align="right">95.93</td><td rowspan="1" colspan="1" valign="middle" align="center">−.06<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">−.35</td><td rowspan="1" colspan="1" valign="middle" align="center">.20</td><td rowspan="1" colspan="1" valign="middle" align="center">.62</td><td rowspan="1" colspan="1" valign="middle" align="center">.64</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6. G speed factor</td><td rowspan="1" colspan="1" valign="middle" align="center">1172</td><td rowspan="1" colspan="1" valign="middle" align="right">.00</td><td rowspan="1" colspan="1" valign="middle" align="right">1.00</td><td rowspan="1" colspan="1" valign="middle" align="right">.20</td><td rowspan="1" colspan="1" valign="middle" align="right">.99</td><td rowspan="1" colspan="1" valign="middle" align="right">−.18</td><td rowspan="1" colspan="1" valign="middle" align="right">.94</td><td rowspan="1" colspan="1" valign="middle" align="center">−.08<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>N</em>
</td><td colspan="2" rowspan="1" valign="middle" align="right">Total sample<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Men<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Women<hr></td><td rowspan="1" colspan="1" valign="middle" align="center">Correlation with age</td><td colspan="5" rowspan="1" valign="middle" align="center">Inter-trait correlations (all significant <em>p</em> &lt; .001)<hr></td></tr><tr><td colspan="14" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">1. DS</td><td rowspan="1" colspan="1" valign="middle" align="center">2. SS</td><td rowspan="1" colspan="1" valign="middle" align="center">3. IT</td><td rowspan="1" colspan="1" valign="middle" align="center">4. SRT</td><td rowspan="1" colspan="1" valign="middle" align="center">5. 4CRT</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>LBC1936</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1. Digit Symbol (no. correct in 120 s)</td><td rowspan="1" colspan="1" valign="middle" align="center">990</td><td rowspan="1" colspan="1" valign="middle" align="right">56.82</td><td rowspan="1" colspan="1" valign="middle" align="right">12.81</td><td rowspan="1" colspan="1" valign="middle" align="right">54.97</td><td rowspan="1" colspan="1" valign="middle" align="right">13.05</td><td rowspan="1" colspan="1" valign="middle" align="right">58.65</td><td rowspan="1" colspan="1" valign="middle" align="right">12.30</td><td rowspan="1" colspan="1" valign="middle" align="center">−.17<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2. Symbol Search</td><td rowspan="1" colspan="1" valign="middle" align="center">991</td><td rowspan="1" colspan="1" valign="middle" align="right">24.82</td><td rowspan="1" colspan="1" valign="middle" align="right">6.28</td><td rowspan="1" colspan="1" valign="middle" align="right">24.85</td><td rowspan="1" colspan="1" valign="middle" align="right">6.48</td><td rowspan="1" colspan="1" valign="middle" align="right">24.79</td><td rowspan="1" colspan="1" valign="middle" align="right">6.09</td><td rowspan="1" colspan="1" valign="middle" align="center">−.23<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">.61</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3. Inspection Time</td><td rowspan="1" colspan="1" valign="middle" align="center">952</td><td rowspan="1" colspan="1" valign="middle" align="right">112.19</td><td rowspan="1" colspan="1" valign="middle" align="right">11.00</td><td rowspan="1" colspan="1" valign="middle" align="right">113.99</td><td rowspan="1" colspan="1" valign="middle" align="right">10.62</td><td rowspan="1" colspan="1" valign="middle" align="right">110.37</td><td rowspan="1" colspan="1" valign="middle" align="right">11.10</td><td rowspan="1" colspan="1" valign="middle" align="center">−.03</td><td rowspan="1" colspan="1" valign="middle" align="center">.30</td><td rowspan="1" colspan="1" valign="middle" align="center">.32</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4. Simple RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">991</td><td rowspan="1" colspan="1" valign="middle" align="right">275.33</td><td rowspan="1" colspan="1" valign="middle" align="right">53.44</td><td rowspan="1" colspan="1" valign="middle" align="right">275.85</td><td rowspan="1" colspan="1" valign="middle" align="right">57.29</td><td rowspan="1" colspan="1" valign="middle" align="right">274.81</td><td rowspan="1" colspan="1" valign="middle" align="right">49.36</td><td rowspan="1" colspan="1" valign="middle" align="center">.06<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">−.27</td><td rowspan="1" colspan="1" valign="middle" align="center">−.24</td><td rowspan="1" colspan="1" valign="middle" align="center">−.19</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5. 4-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">990</td><td rowspan="1" colspan="1" valign="middle" align="right">640.51</td><td rowspan="1" colspan="1" valign="middle" align="right">83.93</td><td rowspan="1" colspan="1" valign="middle" align="right">639.39</td><td rowspan="1" colspan="1" valign="middle" align="right">91.46</td><td rowspan="1" colspan="1" valign="middle" align="right">641.63</td><td rowspan="1" colspan="1" valign="middle" align="right">75.78</td><td rowspan="1" colspan="1" valign="middle" align="center">.12<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">−.50</td><td rowspan="1" colspan="1" valign="middle" align="center">−.46</td><td rowspan="1" colspan="1" valign="middle" align="center">−.36</td><td rowspan="1" colspan="1" valign="middle" align="center">.47</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">6. G speed factor</td><td rowspan="1" colspan="1" valign="middle" align="center">949</td><td rowspan="1" colspan="1" valign="middle" align="right">.00</td><td rowspan="1" colspan="1" valign="middle" align="right">.98</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="right">1.01</td><td rowspan="1" colspan="1" valign="middle" align="right">−.03</td><td rowspan="1" colspan="1" valign="middle" align="right">.95</td><td rowspan="1" colspan="1" valign="middle" align="center">−.17<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>N</em>
</td><td colspan="2" rowspan="1" valign="middle" align="right">Total sample<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Men<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Women<hr></td><td rowspan="1" colspan="1" valign="middle" align="center">Correlation with age</td><td colspan="4" rowspan="1" valign="middle" align="center">Inter-trait Correlations (all significant <em>p</em> &lt; .001)<hr></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">1. DS</td><td rowspan="1" colspan="1" valign="middle" align="center">2. IT</td><td rowspan="1" colspan="1" valign="middle" align="center">3. SRT</td><td rowspan="1" colspan="1" valign="middle" align="center">4. 4CRT</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td colspan="14" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>LBC1921</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1. Digit Symbol (no. correct in 120 s)</td><td rowspan="1" colspan="1" valign="middle" align="center">307</td><td rowspan="1" colspan="1" valign="middle" align="right">41.83</td><td rowspan="1" colspan="1" valign="middle" align="right">12.84</td><td rowspan="1" colspan="1" valign="middle" align="right">42.54</td><td rowspan="1" colspan="1" valign="middle" align="right">12.44</td><td rowspan="1" colspan="1" valign="middle" align="right">41.27</td><td rowspan="1" colspan="1" valign="middle" align="right">13.15</td><td rowspan="1" colspan="1" valign="middle" align="center">.03</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2. Inspection Time</td><td rowspan="1" colspan="1" valign="middle" align="center">305</td><td rowspan="1" colspan="1" valign="middle" align="right">101.48</td><td rowspan="1" colspan="1" valign="middle" align="right">13.59</td><td rowspan="1" colspan="1" valign="middle" align="right">102.05</td><td rowspan="1" colspan="1" valign="middle" align="right">14.18</td><td rowspan="1" colspan="1" valign="middle" align="right">101.01</td><td rowspan="1" colspan="1" valign="middle" align="right">13.12</td><td rowspan="1" colspan="1" valign="middle" align="center">−.02</td><td rowspan="1" colspan="1" valign="middle" align="center">.36</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">3. Simple RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">307</td><td rowspan="1" colspan="1" valign="middle" align="right">312.69</td><td rowspan="1" colspan="1" valign="middle" align="right">84.34</td><td rowspan="1" colspan="1" valign="middle" align="right">307.57</td><td rowspan="1" colspan="1" valign="middle" align="right">63.76</td><td rowspan="1" colspan="1" valign="middle" align="right">316.77</td><td rowspan="1" colspan="1" valign="middle" align="right">97.66</td><td rowspan="1" colspan="1" valign="middle" align="center">−.02</td><td rowspan="1" colspan="1" valign="middle" align="center">−.43</td><td rowspan="1" colspan="1" valign="middle" align="center">−.27</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">4. 4-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">307</td><td rowspan="1" colspan="1" valign="middle" align="right">792.36</td><td rowspan="1" colspan="1" valign="middle" align="right">146.29</td><td rowspan="1" colspan="1" valign="middle" align="right">782.59</td><td rowspan="1" colspan="1" valign="middle" align="right">150.14</td><td rowspan="1" colspan="1" valign="middle" align="right">800.13</td><td rowspan="1" colspan="1" valign="middle" align="right">143.13</td><td rowspan="1" colspan="1" valign="middle" align="center">.04</td><td rowspan="1" colspan="1" valign="middle" align="center">−.61</td><td rowspan="1" colspan="1" valign="middle" align="center">−.34</td><td rowspan="1" colspan="1" valign="middle" align="center">.53</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">5. G speed factor</td><td rowspan="1" colspan="1" valign="middle" align="center">309</td><td rowspan="1" colspan="1" valign="middle" align="right">.01</td><td rowspan="1" colspan="1" valign="middle" align="right">1.01</td><td rowspan="1" colspan="1" valign="middle" align="right">.08</td><td rowspan="1" colspan="1" valign="middle" align="right">1.00</td><td rowspan="1" colspan="1" valign="middle" align="right">−.05</td><td rowspan="1" colspan="1" valign="middle" align="right">1.02</td><td rowspan="1" colspan="1" valign="middle" align="center">−.01</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center">
–
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center">
<em>N</em>
</td><td colspan="2" rowspan="1" valign="middle" align="right">Total sample<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Men<hr></td><td colspan="2" rowspan="1" valign="middle" align="right">Women<hr></td><td rowspan="1" colspan="1" valign="middle" align="center">Correlation with age</td><td colspan="4" rowspan="1" valign="middle" align="center">Inter-trait correlations (all significant <em>p</em> &lt; .001)<hr></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="right">Mean</td><td rowspan="1" colspan="1" valign="middle" align="right">SD</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">1. SRT</td><td rowspan="1" colspan="1" valign="middle" align="center">2. 2CRT</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td colspan="14" rowspan="1" valign="middle" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>HBCS</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">1. Simple RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1080</td><td rowspan="1" colspan="1" valign="middle" align="right">349.42</td><td rowspan="1" colspan="1" valign="middle" align="right">88.81</td><td rowspan="1" colspan="1" valign="middle" align="right">332.84</td><td rowspan="1" colspan="1" valign="middle" align="right">74.22</td><td rowspan="1" colspan="1" valign="middle" align="right">362.25</td><td rowspan="1" colspan="1" valign="middle" align="right">96.76</td><td rowspan="1" colspan="1" valign="middle" align="center">.12<sup>***</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">2. 2-Choice RT mean (ms)</td><td rowspan="1" colspan="1" valign="middle" align="center">1071</td><td rowspan="1" colspan="1" valign="middle" align="right">580.58</td><td rowspan="1" colspan="1" valign="middle" align="right">117.62</td><td rowspan="1" colspan="1" valign="middle" align="right">564.79</td><td rowspan="1" colspan="1" valign="middle" align="right">93.77</td><td rowspan="1" colspan="1" valign="middle" align="right">592.66</td><td rowspan="1" colspan="1" valign="middle" align="right">131.80</td><td rowspan="1" colspan="1" valign="middle" align="center">.09<sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">.63</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr></tbody></table></div><div id="largeobj_idm140131564805744" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3319015/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup><em>p</em> &lt; .05.</div><div id="TFN2"><sup>**</sup><em>p</em> &lt; .001.</div><div id="TFN3"><sup>***</sup><em>p</em> &lt; .0001.</div></div></div><p id="P22" class="p">The results for SNPs exceeding suggestive significance level are shown in <a href="/pmc/articles/PMC3319015/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; see <a href="#SD2" rid="SD2" class=" supplementary-material">Appendix CSupplementary Table 1</a> for top 100 SNPs. For digit symbol, six SNPs showed suggestive significance with three of these SNPs in strong linkage disequilibrium (LD) with each other; they were located in or nearby two known genes, <em>ATRNL1</em> and <em>KRTAP7-1</em>, although none were exonic. Eight SNPs in three genes were shown to be associated (at suggestive levels) with symbol search in the LBC1936 cohort. The effect sizes of these SNPs were relatively large, ranging between .21 and .43, but larger standard errors accompany these estimates demonstrating the reduced power of the analysis for individual cohorts. There were five SNPs that passed the suggestive significance criterion for inspection time, and two of these were located in a single known gene, <em>CRTC3</em>. For simple RT, 11 SNPs were identified at the suggestive level of significance, with a SNP on chromosome 3 located in <em>MYRIP</em>. For 2-choice RT mean, 10 SNPs (on 6 chromosomes and two in <em>DCDC2</em>) showed suggestive association. The beta weights for all top SNPs for each trait were homogenous between cohorts (<em>p</em> &gt; .05). For 4-choice RT, five SNPs showed suggestive significance, they were located in <em>FAM110C</em>, <em>DIP2C</em> and <em>KCTD2</em>. In the Australian cohort, the majority of SNPs showing suggestive association with 8-choice RT mean were in almost perfect LD with each other. SNPs on five chromosomes were identified and two known genes were implicated (<em>RAB11F1P2</em>, <em>LAPTM4A</em>). Minor allele frequencies were above .07 and effect sizes ranged between .18 and .41. Six SNPs were suggestively significant for the processing speed factor; some were located in <em>PDE1C</em> and <em>PKNOX1</em> genes. Only one SNP (rs1375785) showed suggestive association with multiple measures (2- and 8-choice RT).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p5">Description and results (beta, SE, <em>p</em>) of the top SNPs associated with digit symbol, symbol search, inspection time, simple RT, choice RT measures and the speed factor (<em>p</em> &lt; 1.21×10<sup>−5</sup>); bolded gene name indicates that the SNP lies in the gene/locus.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140131555120288" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="right">Ch</th><th rowspan="1" colspan="1" valign="middle" align="right">Position</th><th colspan="2" rowspan="1" valign="middle" align="center">Allele<hr></th><th rowspan="1" colspan="1" valign="middle" align="left">MAF Range</th><th colspan="4" rowspan="1" valign="middle" align="right">Results<hr></th><th colspan="4" rowspan="1" valign="middle" align="left">Individual-analysis <em>p</em><hr></th><th rowspan="1" colspan="1" valign="middle" align="left">Nominally significant p of other measures</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="center">1</th><th rowspan="1" colspan="1" valign="middle" align="center">2</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right">Beta</th><th rowspan="1" colspan="1" valign="middle" align="right">SE</th><th rowspan="1" colspan="1" valign="middle" align="left">Meta-<em>p</em></th><th rowspan="1" colspan="1" valign="middle" align="left">Gene/Locus</th><th rowspan="1" colspan="1" valign="middle" align="left">Brisbane</th><th rowspan="1" colspan="1" valign="middle" align="left">LBC1936</th><th rowspan="1" colspan="1" valign="middle" align="left">LBC1921</th><th rowspan="1" colspan="1" valign="middle" align="left">HBCS</th><th rowspan="1" colspan="1" valign="middle" align="left"></th></tr></thead><tbody><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<em>Psychometric</em>
</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>Digit symbol (DS)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rs10490919</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="right">117512588</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.46–.49</td><td rowspan="1" colspan="1" valign="middle" align="right">−.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.88E-06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>ATRNL1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.20E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">6.98E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">4.59E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SS&nbsp;(.007);&nbsp;CRT4&nbsp;(.048)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rs4247092</td><td rowspan="1" colspan="1" valign="middle" align="right">15</td><td rowspan="1" colspan="1" valign="middle" align="right">96225221</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.46–.48</td><td rowspan="1" colspan="1" valign="middle" align="right">−.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">5.59E–06</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335010" class="tag_hotlink" href="/nuccore/AC015574.1" ref="/nuccore/AC015574.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC015574.1","term_id":"6447113","term_text":"AC015574.1"}}</span>AC015574.1</a></td><td rowspan="1" colspan="1" valign="middle" align="left">4.50E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">8.38E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">2.65E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SS (.003); Factor (.048)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rs2567426</td><td rowspan="1" colspan="1" valign="middle" align="right">15</td><td rowspan="1" colspan="1" valign="middle" align="right">96332861</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.33–.35</td><td rowspan="1" colspan="1" valign="middle" align="right">.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">3.23E–06</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335011" class="tag_hotlink" href="/nuccore/AC015574.2" ref="/nuccore/AC015574.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC015574.2","term_id":"6524246","term_text":"AC015574.2"}}</span>AC015574.2</a></td><td rowspan="1" colspan="1" valign="middle" align="left">3.90E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">5.69E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">2.45E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SS (.002); Factor (.031)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rs7283316</td><td rowspan="1" colspan="1" valign="middle" align="right">21</td><td rowspan="1" colspan="1" valign="middle" align="right">32216253</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.38–.39</td><td rowspan="1" colspan="1" valign="middle" align="right">−.37</td><td rowspan="1" colspan="1" valign="middle" align="right">.08</td><td rowspan="1" colspan="1" valign="middle" align="left">8.71E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">KRTAP7-1</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">8.53E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">1.23E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">IT (.007)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>Symbol search (SS)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Rs6707697</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">109940324</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="left">.07</td><td rowspan="1" colspan="1" valign="middle" align="right">−.43</td><td rowspan="1" colspan="1" valign="middle" align="right">.08</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>SH3RF3</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.29E–07</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT8 (.008)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2255798</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">31629281</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.11</td><td rowspan="1" colspan="1" valign="middle" align="right">.31</td><td rowspan="1" colspan="1" valign="middle" align="right">.07</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>NFKBIL1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.66E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">DS (.027); Factor (.005)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2230365</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">31633427</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.28</td><td rowspan="1" colspan="1" valign="middle" align="right">.06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>NFKBIL1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">6.57E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">Factor (8.38E–05)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7087965</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="right">16689683</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.32</td><td rowspan="1" colspan="1" valign="middle" align="right">−.21</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>RSU1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.93E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.008); DS (.014)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<em>Chronometric</em>
</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>Inspection time (IT)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2392362</td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="right">35401447</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.27–.31</td><td rowspan="1" colspan="1" valign="middle" align="right">−.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.40E–06</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335014" class="tag_hotlink" href="/nuccore/AC007652.1" ref="/nuccore/AC007652.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC007652.1","term_id":"4887764","term_text":"AC007652.1"}}</span>AC007652.1</a></td><td rowspan="1" colspan="1" valign="middle" align="left">3.90E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.46E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.97E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs12915189</td><td rowspan="1" colspan="1" valign="middle" align="right">15</td><td rowspan="1" colspan="1" valign="middle" align="right">91114376</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.29–.33</td><td rowspan="1" colspan="1" valign="middle" align="right">.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.97E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>CRTC3</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.30E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">8.85E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.56E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs3743401</td><td rowspan="1" colspan="1" valign="middle" align="right">15</td><td rowspan="1" colspan="1" valign="middle" align="right">91185994</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.21–.24</td><td rowspan="1" colspan="1" valign="middle" align="right">.15</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">7.10E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>CRTC3</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">8.00E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">1.22E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">7.64E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7219585</td><td rowspan="1" colspan="1" valign="middle" align="right">17</td><td rowspan="1" colspan="1" valign="middle" align="right">72268208</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.13–.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.20</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">1.03E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>DNAI2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.10E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">2.70E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">7.12E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SS (.007); CRT4 (.039); DS (.004); Factor (.003)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1884136</td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="right">10850464</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.09–.11</td><td rowspan="1" colspan="1" valign="middle" align="right">.20</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.65E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">RP4-697P8.2</td><td rowspan="1" colspan="1" valign="middle" align="left">9.60E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.11E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">6.08E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SRT (.03)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>Simple RT (SRT)</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs7566934</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">176654738</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.29–.32</td><td rowspan="1" colspan="1" valign="middle" align="right">−.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">7.03E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>EXTL2P1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">5.23E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">7.92E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">2.40E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">Factor (.038)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1446829</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">176677084</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.31–.32</td><td rowspan="1" colspan="1" valign="middle" align="right">−.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">7.09E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>EXTL2P1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.14E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">6.38E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">6.34E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">Factor (.015)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9985399</td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="right">40008634</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.05–.07</td><td rowspan="1" colspan="1" valign="middle" align="right">−.26</td><td rowspan="1" colspan="1" valign="middle" align="right">.06</td><td rowspan="1" colspan="1" valign="middle" align="left">8.79E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>MYRIP</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.87E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">3.20E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">6.66E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.005); CRT4 (.012); IT (.007); SS (.024)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs523340</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="right">111854246</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.10–.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.21</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">2.43E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>C9orf5</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">5.83E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">8.61E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.52E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">DS (.019); IT (.026)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2908835</td><td rowspan="1" colspan="1" valign="middle" align="right">12</td><td rowspan="1" colspan="1" valign="middle" align="right">11648418</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.23–.24</td><td rowspan="1" colspan="1" valign="middle" align="right">.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">6.51E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">AC007450.3</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.21E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">4.85E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">5.54E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10148969</td><td rowspan="1" colspan="1" valign="middle" align="right">14</td><td rowspan="1" colspan="1" valign="middle" align="right">90968374</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.38–.42</td><td rowspan="1" colspan="1" valign="middle" align="right">−.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.08E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TTC7B</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.03E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">7.46E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">7.20E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT4 (.025); Factor (.012)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs17291845</td><td rowspan="1" colspan="1" valign="middle" align="right">16</td><td rowspan="1" colspan="1" valign="middle" align="right">55245037</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.13–.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">4.70E–06</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335015" class="tag_hotlink" href="/nuccore/AC109462.1" ref="/nuccore/AC109462.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC109462.1","term_id":"18483458","term_text":"AC109462.1"}}</span>AC109462.1</a></td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.48E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.05E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">1.43E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">IT (.022)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>2-choice RT</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6801136</td><td rowspan="1" colspan="1" valign="middle" align="right">3</td><td rowspan="1" colspan="1" valign="middle" align="right">55265321</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.33–.34</td><td rowspan="1" colspan="1" valign="middle" align="right">−.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">9.97E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">RP11-889D3.262</td><td rowspan="1" colspan="1" valign="middle" align="left">2.10E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.17E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">Factor (.045)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6922632</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">24107093</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.10–.16</td><td rowspan="1" colspan="1" valign="middle" align="right">.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">4.18E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>NRSN1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">7.50E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">5.79E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">SRT (.002)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs9460980</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">24233288</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.35–.39</td><td rowspan="1" colspan="1" valign="middle" align="right">−.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">4.34E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>DCDC2</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.60E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.22E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs793834</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">24234933</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.18–.25</td><td rowspan="1" colspan="1" valign="middle" align="right">.15</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.85E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>DCDC2</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">8.70E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">8.97E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">SRT (.041)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1375785<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">83991231</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.08–.12</td><td rowspan="1" colspan="1" valign="middle" align="right">−.20</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.41E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.60E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">6.78E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT8 (1.00E–06); CRT4 (.016)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11212364</td><td rowspan="1" colspan="1" valign="middle" align="right">11</td><td rowspan="1" colspan="1" valign="middle" align="right">97574984</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.03–.11</td><td rowspan="1" colspan="1" valign="middle" align="right">−.26</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">2.96E–06</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335016" class="tag_hotlink" href="/nuccore/AP005434.1" ref="/nuccore/AP005434.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AP005434.1","term_id":"21623541","term_text":"AP005434.1"}}</span>AP005434.1</a></td><td rowspan="1" colspan="1" valign="middle" align="left">2.10E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.82E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">SRT (.008); SS (.036)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs17124581</td><td rowspan="1" colspan="1" valign="middle" align="right">14</td><td rowspan="1" colspan="1" valign="middle" align="right">88826682</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.06</td><td rowspan="1" colspan="1" valign="middle" align="right">−.27</td><td rowspan="1" colspan="1" valign="middle" align="right">.06</td><td rowspan="1" colspan="1" valign="middle" align="left">2.71E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>SPATA7</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.90E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.77E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">Factor (.034)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4815868</td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="right">5840539</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.16</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">8.21E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>C20orf196</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.90E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">9.17E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT4 (.004); CRT8 (.027); DS (.008); SRT (.006)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>4-Choice RT</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11542478</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">38938</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.10–.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">8.89E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>FAM110C</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">8.70E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">6.57E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">1.98E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SRT (.018); IT (.04)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6961611</td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="right">67426323</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.02</td><td rowspan="1" colspan="1" valign="middle" align="right">−.63</td><td rowspan="1" colspan="1" valign="middle" align="right">.14</td><td rowspan="1" colspan="1" valign="middle" align="left">7.33E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">RP11-358M3.1</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">5.83E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">5.04E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">DS (.038); Factor (.0005)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs16939046</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">76147954</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.09–.10</td><td rowspan="1" colspan="1" valign="middle" align="right">.22</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">3.71E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><a id="__tag_387335013" class="tag_hotlink" href="/nuccore/AC022274.1" ref="/nuccore/AC022274.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC022274.1","term_id":"6778595","term_text":"AC022274.1"}}</span>AC022274.1</a></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">7.70E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.83E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">9.54E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.042); CRT8 (.048); IT (.017); SRT (.04)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs17293454</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="right">633495</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.08–.13</td><td rowspan="1" colspan="1" valign="middle" align="right">−.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">1.20E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>DIP2C</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">4.20E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">8.46E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">3.98E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">SS (.018)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11077773</td><td rowspan="1" colspan="1" valign="middle" align="right">17</td><td rowspan="1" colspan="1" valign="middle" align="right">73060073</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.06–.08</td><td rowspan="1" colspan="1" valign="middle" align="right">.24</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.33E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>KCTD2</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">3.00E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">1.21E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">2.94E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">IT (.011); SRT (.043)</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">8-Choice RT</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1191812</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">20224999</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">.25</td><td rowspan="1" colspan="1" valign="middle" align="right">.22</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>LAPTM4A</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.00E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.006); DS (.0008); SS (.014)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11694170</td><td rowspan="1" colspan="1" valign="middle" align="right">2</td><td rowspan="1" colspan="1" valign="middle" align="right">20260512</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.21</td><td rowspan="1" colspan="1" valign="middle" align="right">.23</td><td rowspan="1" colspan="1" valign="middle" align="right">.05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left"><a id="__tag_387335012" class="tag_hotlink" href="/nuccore/AC098828.2" ref="/nuccore/AC098828.2"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC098828.2","term_id":"17998648","term_text":"AC098828.2"}}</span>AC098828.2</a></td><td rowspan="1" colspan="1" valign="middle" align="left">7.00E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.006); DS (.029); IT (.0322)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs13170576</td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="right">8467174</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="left">.07</td><td rowspan="1" colspan="1" valign="middle" align="right">.41</td><td rowspan="1" colspan="1" valign="middle" align="right">.09</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">RP11-480D4.3</td><td rowspan="1" colspan="1" valign="middle" align="left">1.20E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs11156429</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">105364421</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="left">.46</td><td rowspan="1" colspan="1" valign="middle" align="right">−.18</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">C6orf220</td><td rowspan="1" colspan="1" valign="middle" align="left">1.10E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (.003); CRT4 (.00002); DS (.0001); SRT (.031)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1375785<sup>a</sup></td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="right">83991231</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="left">.12</td><td rowspan="1" colspan="1" valign="middle" align="right">.30</td><td rowspan="1" colspan="1" valign="middle" align="right">.06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">1.00E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT2 (8.41E–06); CRT4 (.016)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1419141</td><td rowspan="1" colspan="1" valign="middle" align="right">10</td><td rowspan="1" colspan="1" valign="middle" align="right">119694504</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.34</td><td rowspan="1" colspan="1" valign="middle" align="right">−.19</td><td rowspan="1" colspan="1" valign="middle" align="right">.04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>RAB11FIP2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">5.80E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td colspan="15" rowspan="1" valign="middle" align="left">
<strong>Speed Factor</strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10475598</td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="right">173648438</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.32–.34</td><td rowspan="1" colspan="1" valign="middle" align="right">.15</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">8.10E–07</td><td rowspan="1" colspan="1" valign="middle" align="left">CTC-430J12.2</td><td rowspan="1" colspan="1" valign="middle" align="left">2.30E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">9.55E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">5.41E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">DS (.02)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4710758</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="right">170455060</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.32–.34</td><td rowspan="1" colspan="1" valign="middle" align="right">.14</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">9.77E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">RP11-302L19.1</td><td rowspan="1" colspan="1" valign="middle" align="left">5.30E–04</td><td rowspan="1" colspan="1" valign="middle" align="left">8.52E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">2.35E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT8 (.031)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4723127</td><td rowspan="1" colspan="1" valign="middle" align="right">7</td><td rowspan="1" colspan="1" valign="middle" align="right">32225283</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.43–.44</td><td rowspan="1" colspan="1" valign="middle" align="right">−.13</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">9.54E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>PDE1C</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.30E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">3.92E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.58E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6051520</td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="right">351944</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="left">.21–.24</td><td rowspan="1" colspan="1" valign="middle" align="right">.18</td><td rowspan="1" colspan="1" valign="middle" align="right">.03</td><td rowspan="1" colspan="1" valign="middle" align="left">1.68E–07</td><td rowspan="1" colspan="1" valign="middle" align="left">
<em>TRIB3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="left">3.40E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">6.61E–03</td><td rowspan="1" colspan="1" valign="middle" align="left">6.63E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">CRT8 (.002)</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2839627</td><td rowspan="1" colspan="1" valign="middle" align="right">21</td><td rowspan="1" colspan="1" valign="middle" align="right">44448718</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="left">.11–.12</td><td rowspan="1" colspan="1" valign="middle" align="right">−.27</td><td rowspan="1" colspan="1" valign="middle" align="right">.06</td><td rowspan="1" colspan="1" valign="middle" align="left">6.17E–06</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong><em>PKNOX1</em></strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">2.00E–05</td><td rowspan="1" colspan="1" valign="middle" align="left">8.05E–02</td><td rowspan="1" colspan="1" valign="middle" align="left">1.18E–01</td><td rowspan="1" colspan="1" valign="middle" align="left">
–
</td><td rowspan="1" colspan="1" valign="middle" align="left">IT (.0002)</td></tr></tbody></table></div><div id="largeobj_idm140131555120288" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3319015/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="P35" class="p p-first-last">SNP = single nucleotide polymorphism. Ch = chromosome. A1 = allele 1 (effect allele); A2 = allele 2. MAF=minor allele frequency.</p></div><div id="TFN5"><p id="P36" class="p p-first-last"><em>Note</em>: The following SNPs were in LD (<em>r</em><sup>2</sup> &gt; .80) with significant SNPs reported in the table: Digit Symbol: rs10490919 with rs2804161, rs1157117; Symbol Search: rs2255798 with rs2857708, rs2736189, rs2009658, rs2844482; Inspection Time: rs3743401 with rs8028854, rs13379889; rs12915189 with rs6496696; Simple RT: rs523340 with rs1519466; rs17291845 with rs12926302, rs12918335; rs9985399 with rs6791790; 2-choice RT: rs9460980 with rs9295619, rs793842; 8-choice RT: rs1375785 with rs1449811; rs1375785 with rs4739919, rs896010; rs11694170 with rs1513831; rs11156429 with rs4946651, rs314262, rs31428; Speed Factor: rs10475598 with rs4295404.</p></div><div id="TFN6"><sup>a</sup>Overlapping SNP.</div></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">2.1. Biological pathways analysis</h3><p id="P23" class="p p-first">The results of the biological pathways analysis of all the speed measures – using ALIGATOR software – are shown in <a href="#SD3" rid="SD3" class=" supplementary-material">Appendix CSupplementary Table 2</a>. Pathways which overlapped across measures are shown in <a href="/pmc/articles/PMC3319015/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>. Most of these were part of the biological process category: the three terms associated with inspection time and the speed factor related to vascular process; the two terms associated with simple RT and 4-choice RT were related to biosynthetic process; while the term associated with inspection time and 8-choice RT was a renal system process involved in regulation of systemic arterial blood pressure. The cellular component associated with the choice RT measures comprised terms that were all related to organelle and cell components. The three terms represented in the molecular function category associated with simple RT, choice RT, inspection time and digit symbol were in different pathways.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p6">Enriched GO categories (<em>p</em> &lt; .01) for multiple speed measure.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140131549484848" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">GO number</th><th rowspan="1" colspan="1" valign="top" align="left">Type</th><th rowspan="1" colspan="1" valign="top" align="right">Genes in category</th><th rowspan="1" colspan="1" valign="top" align="right"># genes on list</th><th rowspan="1" colspan="1" valign="top" align="right">Expected # genes on list</th><th rowspan="1" colspan="1" valign="top" align="left"><em>p</em></th><th rowspan="1" colspan="1" valign="top" align="center">Expected hits/study</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0008373</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Sialyltransferase activity</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">2.72</td><td rowspan="1" colspan="1" valign="top" align="left">.0066</td><td rowspan="1" colspan="1" valign="top" align="center">19.59</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">20</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">2.54</td><td rowspan="1" colspan="1" valign="top" align="left">.0000</td><td rowspan="1" colspan="1" valign="top" align="center">.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0030173</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Integral to Golgi membrane</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">41</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">5.48</td><td rowspan="1" colspan="1" valign="top" align="left">.0014</td><td rowspan="1" colspan="1" valign="top" align="center">4.43</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">41</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">5.17</td><td rowspan="1" colspan="1" valign="top" align="left">.0008</td><td rowspan="1" colspan="1" valign="top" align="center">2.73</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0031228</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Intrinsic to Golgi membrane</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">44</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">5.51</td><td rowspan="1" colspan="1" valign="top" align="left">.0018</td><td rowspan="1" colspan="1" valign="top" align="center">5.57</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">44</td><td rowspan="1" colspan="1" valign="top" align="right">12</td><td rowspan="1" colspan="1" valign="top" align="right">5.19</td><td rowspan="1" colspan="1" valign="top" align="left">.0008</td><td rowspan="1" colspan="1" valign="top" align="center">2.73</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0031301</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Integral to organelle membrane</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">100</td><td rowspan="1" colspan="1" valign="top" align="right">16</td><td rowspan="1" colspan="1" valign="top" align="right">8.69</td><td rowspan="1" colspan="1" valign="top" align="left">.007</td><td rowspan="1" colspan="1" valign="top" align="center">20.81</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">CELLULAR</td><td rowspan="1" colspan="1" valign="top" align="right">100</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="right">8.28</td><td rowspan="1" colspan="1" valign="top" align="left">.0068</td><td rowspan="1" colspan="1" valign="top" align="center">19.86</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0044237</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Cellular metabolic process</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">4200</td><td rowspan="1" colspan="1" valign="top" align="right">297</td><td rowspan="1" colspan="1" valign="top" align="right">259.16</td><td rowspan="1" colspan="1" valign="top" align="left">.001</td><td rowspan="1" colspan="1" valign="top" align="center">3.37</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">4179</td><td rowspan="1" colspan="1" valign="top" align="right">278</td><td rowspan="1" colspan="1" valign="top" align="right">244.93</td><td rowspan="1" colspan="1" valign="top" align="left">.0036</td><td rowspan="1" colspan="1" valign="top" align="center">10.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0016226</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Iron-sulfur cluster assembly</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Simple RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">.16</td><td rowspan="1" colspan="1" valign="top" align="left">.008</td><td rowspan="1" colspan="1" valign="top" align="center">23.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">.15</td><td rowspan="1" colspan="1" valign="top" align="left">.008</td><td rowspan="1" colspan="1" valign="top" align="center">24.07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0031163</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Metallo-sulfur cluster assembly</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Simple RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">.16</td><td rowspan="1" colspan="1" valign="top" align="left">.008</td><td rowspan="1" colspan="1" valign="top" align="center">23.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">8</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">.15</td><td rowspan="1" colspan="1" valign="top" align="left">.008</td><td rowspan="1" colspan="1" valign="top" align="center">24.07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0003071</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Renal system process involved</strong>
<br>
<strong>in regulation of systemic</strong>
<br>
<strong>arterial blood pressure</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="right">2</td><td rowspan="1" colspan="1" valign="top" align="right">.19</td><td rowspan="1" colspan="1" valign="top" align="left">.0096</td><td rowspan="1" colspan="1" valign="top" align="center">28.09</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8-choice RT</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">4</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">.18</td><td rowspan="1" colspan="1" valign="top" align="left">.0000</td><td rowspan="1" colspan="1" valign="top" align="center">.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0008889</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Glycerophosphodiester</strong>
<br>
<strong>phosphodiesterase activity</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">.51</td><td rowspan="1" colspan="1" valign="top" align="left">.0076</td><td rowspan="1" colspan="1" valign="top" align="center">22.12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Simple RT</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">7</td><td rowspan="1" colspan="1" valign="top" align="right">3</td><td rowspan="1" colspan="1" valign="top" align="right">.52</td><td rowspan="1" colspan="1" valign="top" align="left">.0066</td><td rowspan="1" colspan="1" valign="top" align="center">19.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0005102</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Receptor binding</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Digit Symbol</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">655</td><td rowspan="1" colspan="1" valign="top" align="right">68</td><td rowspan="1" colspan="1" valign="top" align="right">51.68</td><td rowspan="1" colspan="1" valign="top" align="left">.0054</td><td rowspan="1" colspan="1" valign="top" align="center">15.69</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">FUNCTION</td><td rowspan="1" colspan="1" valign="top" align="right">655</td><td rowspan="1" colspan="1" valign="top" align="right">68</td><td rowspan="1" colspan="1" valign="top" align="right">51.83</td><td rowspan="1" colspan="1" valign="top" align="left">.0054</td><td rowspan="1" colspan="1" valign="top" align="center">15.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0001568</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Blood vessel development</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">191</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="right">18</td><td rowspan="1" colspan="1" valign="top" align="left">.0022</td><td rowspan="1" colspan="1" valign="top" align="center">6.63</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Speed Factor</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">191</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="right">18.37</td><td rowspan="1" colspan="1" valign="top" align="left">.0044</td><td rowspan="1" colspan="1" valign="top" align="center">12.8</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0001944</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Vasculature development</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">195</td><td rowspan="1" colspan="1" valign="top" align="right">29</td><td rowspan="1" colspan="1" valign="top" align="right">18.18</td><td rowspan="1" colspan="1" valign="top" align="left">.0026</td><td rowspan="1" colspan="1" valign="top" align="center">7.74</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Speed Factor</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">195</td><td rowspan="1" colspan="1" valign="top" align="right">30</td><td rowspan="1" colspan="1" valign="top" align="right">18.55</td><td rowspan="1" colspan="1" valign="top" align="left">.0032</td><td rowspan="1" colspan="1" valign="top" align="center">9.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>GO:0048514</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">
<strong>Blood vessel morphogenesis</strong>
</td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="right"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Inspection Time</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">162</td><td rowspan="1" colspan="1" valign="top" align="right">28</td><td rowspan="1" colspan="1" valign="top" align="right">15.73</td><td rowspan="1" colspan="1" valign="top" align="left">.0000</td><td rowspan="1" colspan="1" valign="top" align="center">.77</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Speed Factor</td><td rowspan="1" colspan="1" valign="top" align="left">PROCESS</td><td rowspan="1" colspan="1" valign="top" align="right">162</td><td rowspan="1" colspan="1" valign="top" align="right">27</td><td rowspan="1" colspan="1" valign="top" align="right">16.03</td><td rowspan="1" colspan="1" valign="top" align="left">.0022</td><td rowspan="1" colspan="1" valign="top" align="center">6.58</td></tr></tbody></table></div><div id="largeobj_idm140131549484848" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3319015/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN7"><p id="P38" class="p p-first-last"><em>Note</em>: Results from ALIGATOR using association <em>p</em> values.</p></div></div></div><p id="P24" class="p p-last"><a href="/pmc/articles/PMC3319015/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a> shows the results of the SNPratio pathways analysis undertaken for the speed factor in the Brisbane and LBC1936 cohorts. Two pathways, focal adhesion and cell junction, were identified in both cohorts. Interestingly, the ALIGATOR results for the speed factor also showed an association with the regulation of focal adhesion formation category (<em>p</em> = .004). We searched these two pathways for SNPs that were nominally significant in both cohorts. No SNPs were identified that were exclusively tied to the cell junction pathway, that is, they were also linked with the focal adhesion pathway. There were twenty-seven of these (in five known genes: <em>LAMB4</em>, <em>COL5A1</em>, <em>COL4A1</em>, <em>VTN</em>, and <em>LAMA1</em>). Seventeen SNPs in six genes (<em>MAPK10</em>, <em>ITGA2</em>, <em>PIK3R1</em>, <em>TLN2</em>, <em>PRKCA</em>, and <em>PARVB</em>) were identified only in the focal adhesion pathway. This included one non-synonymous coding SNP (rs704) on chromosome 17 in the <em>VTN</em> gene. The two significant biological pathways identified in the Brisbane and LBC1936 cohorts were not significant in the smaller LBC1921: focal adhesion pathway, <em>p</em> = .85; cell junction pathway, <em>p</em> = .99. Nor were any of the nominally significant SNPs identified within these pathways replicated (<em>p</em> &gt; .05).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p7">Enriched KEGG Pathways (<em>p</em> &lt; .05) for the Speed Factor. Common pathways between cohorts have been bolded.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140131549608608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Brisbane</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>p</em></th><th rowspan="1" colspan="1" valign="middle" align="left">LBC1936</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>p</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">Human diseases, immune disorders</td><td rowspan="1" colspan="1" valign="middle" align="left">.003</td><td rowspan="1" colspan="1" valign="middle" align="left">Primary immunodeficiency</td><td rowspan="1" colspan="1" valign="middle" align="left">.006</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cellular processes, immune system</td><td rowspan="1" colspan="1" valign="middle" align="left">.003</td><td rowspan="1" colspan="1" valign="middle" align="left">Tight junction</td><td rowspan="1" colspan="1" valign="middle" align="left">.009</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Environmental information processing, signal transduction</td><td rowspan="1" colspan="1" valign="middle" align="left">.007</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Focal adhesion</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">.011</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Metabolism, metabolism of other amino acids</td><td rowspan="1" colspan="1" valign="middle" align="left">.014</td><td rowspan="1" colspan="1" valign="middle" align="left">Valine, leucine and isoleucine biosynthesis</td><td rowspan="1" colspan="1" valign="middle" align="left">.014</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Metabolism, glycan biosynthesis and metabolism</td><td rowspan="1" colspan="1" valign="middle" align="left">.019</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Cell junction</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">.02</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Cellular processes, endocrine system</td><td rowspan="1" colspan="1" valign="middle" align="left">.019</td><td rowspan="1" colspan="1" valign="middle" align="left">Extracellular matrix-receptor interaction</td><td rowspan="1" colspan="1" valign="middle" align="left">.035</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Focal adhesion</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>.022</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">Peroxisome proliferator-activated receptors signalling pathway</td><td rowspan="1" colspan="1" valign="middle" align="left">.039</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>Cell junction</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left">
<strong>.03</strong>
</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Genetic information processing, replication and repair</td><td rowspan="1" colspan="1" valign="middle" align="left">.046</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Complement and coagulation cascades</td><td rowspan="1" colspan="1" valign="middle" align="left">.046</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">Glycan structures – biosynthesis 1</td><td rowspan="1" colspan="1" valign="middle" align="left">.05</td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140131549608608" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3319015/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN8"><p id="P39" class="p p-first-last"><em>Note</em>: Results from SNPratio using observed genotypes.</p></div></div></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">3. Discussion</h2><p id="P25" class="p p-first">This was the first genome-wide association scan of chronometric information processing speed measures with a number of SNPs showing suggestive association with various traits in both the adolescent and elderly cohorts. Some of these SNPs were in genes that appeared to be theoretically relevant to cognitive processing. Furthermore, a biological pathways analysis revealed SNPs in the cell junction and focal adhesion pathways to be overrepresented at a nominal significance level for the speed factor. Pathways analysis of the other traits showed that biological process pathways (sets of molecular events with a defined beginning and end) were overrepresented among the associations of pathways with multiple variables.</p><p id="P26">Up to 11 (for 2-choice RT) mostly independent SNPs were identified for the measures at a suggestive significance level. For every SNP located in or nearby a gene a NCBI database search was performed to check the gene function and any previously reported associations with cognition-related traits. A number of candidate genes were suggested by this bioinformatics search (see <a href="/pmc/articles/PMC3319015/table/T5/" target="table" class="fig-table-link figpopup" rid-figpopup="T5" rid-ob="ob-T5" co-legend-rid=""><span>Table 5</span></a>). Some of these have been associated with psychiatric diseases characterised by cognitive impairment of some sort (e.g., dyslexia, Alzheimer’s disease (AD)), while others are involved in metabolic syndromes which are known to have negative effects on cognition. The association with immune system related disorders is also interesting because a polymorphism in <em>TNF-alpha</em> (which encodes a proinflammatory cytokine) has been associated with processing speed in the elderly (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874783">Baune et al., 2008</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T5"><h3>Table 5</h3><!--caption a7--><div class="caption"><p id="__p8">Genes with Known Function from the List of Best Hits in <a href="/pmc/articles/PMC3319015/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140131556142480" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Gene function</th><th rowspan="1" colspan="1" valign="top" align="left">Gene, Chr, Chr-Band<br> ● <em>Meta-analysis association</em><br> ○ <em>Single study association</em><sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Literature supporting gene function</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Neurological</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DCDC2</em>, 6, 6p22.2<br> ● <em>2-Choice RT, p = 2.9E−06</em><br>● <em>Simple RT, p = .041</em></td><td rowspan="1" colspan="1" valign="top" align="left"><em>DCDC2</em> is thought to play a role in neuronal migration, and has been associated with dyslexia in multiple independent studies (e.g., <a href="#R51" rid="R51" class=" bibr popnode">Meng et al., 2005</a>; <a href="#R67" rid="R67" class=" bibr popnode">Schumacher et al., 2006</a>; <a href="#R69" rid="R69" class=" bibr popnode">Wilcke et al., 2009</a>) (<em>p</em> values = .0003, .004, and &lt;.05 respectively). It has been associated with variation in reading and spelling ability in a general population (<em>p</em><br>values &lt;.001) (<a href="#R39" rid="R39" class=" bibr popnode">Lind et al., 2010</a>), with inattention and hyperactivity/impulsivity symptoms of ADHD (<a href="#R10" rid="R10" class=" bibr popnode">Couto et al., 2009</a>), and with the distribution of grey matter in<br>language-related brain regions in healthy individuals (<em>p</em> &lt; .01 for volumetric differences between genotype groups) (<a href="#R49" rid="R49" class=" bibr popnode">Meda et al., 2008</a>).</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Metabolic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIB3</em>, 20, 20p13-p12.2<br> ● <em>Speed factor, p = 1.7E–07</em><br>○ <em>8-Choice RT, p = .002</em></td><td rowspan="1" colspan="1" valign="top" align="left"><em>TRIB3</em> is a member of the Tribbles family of pseudokinases. Numerous studies point to its involvement in metabolic processes, e.g., in a US Caucasian case-control<br>sample, the functional TRIB3 Q84R polymorphism has been nominally associated with: type 2 diabetes (T2D) (OR = 1.17, <em>p</em> = .04), early-onset of T2D (OR = 1.32, <em>p</em> = .002),<br>and among a non-diabetic subset, R84 carriers had higher glucose levels (<em>p</em> = .005) and lower insulinogenic (<em>p</em> = .03) and disposition index (<em>p</em> = .02) during an oral glucose<br>tolerance test (<a href="#R62" rid="R62" class=" bibr popnode">Prudente et al., 2009</a>). Further, in two independent Italian samples, R84 carriers were found to be at higher risk of impaired glucose regulation (OR = 1.54,<br><em>p</em> = .004 and OR= 1.63, <em>p</em> = .027) (<a href="#R61" rid="R61" class=" bibr popnode">Prudente et al., 2010</a>). In a Chinese cohort, individuals with the same variant were found to be at risk for metabolic syndrome<br>(OR = 2.349, <em>p</em> = .018), with a particular predisposition to carotid atherosclerosis, in part due to the effects of abdominal obesity (OR = 2.351, <em>p</em> = .012),<br>hypertriglyceridemia (OR = 2.314, <em>p</em> = .00003), and insulin resistance (OR-1.697, <em>p</em> = .023) (<a href="#R27" rid="R27" class=" bibr popnode">Gong et al., 2009</a>). The same gene has shown nominal association with AD in a<br>cohort of Swedish men (<em>p</em> = .044) – a finding replicated in a Canadian cohort of males and females (<em>p</em> = .001) (<a href="#R26" rid="R26" class=" bibr popnode">Giedraitis et al., 2009</a>). <em>TRIB3</em> has also been implicated in the<br>control of stress response (<a href="#R56" rid="R56" class=" bibr popnode">Ord et al., 2009</a>).</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Immune</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NFKBIL1</em>, 6, 6p21.3<br> ○ <em>Symbol&nbsp;search,&nbsp;p&nbsp;=&nbsp;6.6E–06,&nbsp;p&nbsp;=&nbsp;1.66E–6</em><br> ● <em>Digit symbol, p = .027</em><br> ● <em>Speed factor, p = .005, P = 8.38E–05</em></td><td rowspan="1" colspan="1" valign="top" align="left"><em>NFKBIL1</em> is located on the human leukocyte antigen locus, which is the major histocompatibility complex in humans and contains a large number of genes involved in<br>immune system function. It has been associated with rheumatoid arthritis in case-control Japanese (<em>p</em> = .006) (<a href="#R55" rid="R55" class=" bibr popnode">Okamoto et al., 2003</a>) and Taiwanese cohorts (<em>p</em> = .004)<br>(<a href="#R38" rid="R38" class=" bibr popnode">Lin et al., 2006</a>). Functional characterisation of the gene supports a role in the pathogenesis of rheumatoid arthritis, and further, indicates that NFKBIL1 proteins may be<br>involved in mRNA processing or the regulation of translation (<a href="#R28" rid="R28" class=" bibr popnode">Greetham et al., 2007</a>). <em>NFKBIL1</em> has also been associated with other inflammation-related disorders:<br>chronic thromboembolic pulmonary hypertension in a Japanese cohort (<em>p</em> = .0002) (<a href="#R35" rid="R35" class=" bibr popnode">Kominami et al., 2009</a>), and the development of chronic Chagas’ cardiomyopathy<br>among Trypanosoma cruzi-infected individuals in a Brazilian cohort (<em>p</em> = .009) (<a href="#R64" rid="R64" class=" bibr popnode">Ramasawmy et al., 2008</a>).</td></tr></tbody></table></div><div id="largeobj_idm140131556142480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3319015/table/T5/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">Note: Gene selection was based on searches conducted using the Genetic Association Database (geneticassociationdb.nih.gov). Only genes with multiple, independent indicators of function were included.</p><p id="__p10">a Phenotype available for one cohort only.</p></div></div><p id="P27">The speed measures were inter-correlated, and in the Brisbane sample have been shown to be genetically related (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874821">Luciano et al., 2004</a>), we therefore expected some overlap between these measures in the GWA results. Only one of the top hits overlapped across measures (2-choice and 8-choice RT) and this SNP was not located near any known gene. Importantly, most of the top hits for each measure showed nominally significant associations with other speed measures. The top hits for digit symbol, symbol search and 4-choice RT were all nominally significant with other speed measures; and one of the top hits for inspection time, simple RT, and all choice RT conditions each showed nominal association with four other speed measures. It is interesting to note that for symbol search, which was only measured in the LBC1936, there were associations with variables that were unmeasured in this cohort (i.e., 2-choice and 8-choice RT) suggesting that not all the common associations can be attributed to correlated type 1 error. The speed factor was most frequently nominally associated with the top hits from other variables and these were not restricted to the measures of which it was composed, suggesting that it was tapping reliable variance related to general speed processes.</p><p id="P28">Like our study, a previous genome-wide association study of digit symbol and symbol search did not uncover any genome-wide significant associations (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874814">Cirulli et al., 2010</a>). This study was less powered than ours with a sample size of 1086 and 414 for the respective measures; none of their 100 top SNPs for each analysis overlapped with our top 100. However, for digit symbol eight of their top 100 SNPs (rs9302365, rs2280645, rs10500956, rs1874989, rs17572757, rs4880535, rs9310772, rs9883639) were significant at a nominal level from 89 that we had results for; two of these SNPs were in the <em>ADARB2</em> gene. For symbol search, three of their top 100 SNPs from 92 that we had results for were nominally significant, these included: rs11812109, rs16896091, and rs4723454. Our study agrees with theirs in that common genes do not influence psychometric speed measures with large effect size.</p><p id="P29">Biological pathways analysis linked to the GO database uncovered a number of pathways common to multiple measures, in particular, the choice RT measures. Across the measures, there was a recurrence of associations with gene products related to vascular and biosynthetic processes and to organelle and cell components. Using the manually defined pathways from KEGG to investigate the speed factor, two pathways – cell junction and focal adhesion – were identified in the Australian and Scottish LBC1936 cohorts, and when we traced the significant SNPs in these pathways, 44 SNPs were identified in 11 genes. These genes were all in the focal adhesion pathway, although five of these were also involved in cell junction. Focal adhesions are structures consisting of proteins at cell-extracellular matrix (ECM) adhesions, and they are involved in controlling cell behaviour. This pathway was also implicated for the speed factor in the ALIGATOR analysis using GO terms. Of the significant genes we identified in this pathway, several are particularly interesting due to their link with AD. The alpha 1 laminin isoform is over-expressed in AD frontal cortex and presents as punctate deposits in the senile plaques, and in the astrocytes of grey and white matter (<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874824">Palu and Liesi, 2002</a>), providing a basis for linking <em>LAMA1</em> to the aetiology of AD. Type 4 collagen (<em>COL4A1</em> is part of the gene family coding for this protein) has also been localised within the senile plaques as punctuate deposits (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874829">Kiuchi et al., 2002</a>); and integrins (e.g., <em>ITGA2</em> gene product) have been identified immunohistochemically in cerebal amyloid plaques (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874799">Eikelenboom et al., 1994</a>), again suggesting that variation in these relevant genes might be important. By the same logic, reduced levels of protein kinase C alpha (e.g., <em>PRKCA</em> gene product) have been linked to an altered amyloid precursor protein secretion in fibroblasts from AD patients (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874794">Benussi et al., 1998</a>). The only gene (of those we identified in the focal adhesion pathway) to be directly tested for association with AD is <em>PIK3R1</em> (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_308224688">Liolitsa et al., 2002</a>). Here, the Met326Tyr polymorphism showed association with risk of late-onset AD in women and in patients where the <em>APOE</em> e4 allele was not present; this was argued to relate to impaired insulin mediated signal transduction.</p><p id="P30" class="p p-last">In summary, our study used two approaches to help localise genetic variants influencing information processing speed: association with single SNPs and association within significant biological pathways. Our speed measures shared common variance, and this overlap was demonstrated for most of the top association signals in the SNP analysis, where at least one, and up to four, other variables were nominally associated. Only one SNP reached suggestive association levels for multiple traits (i.e., 2- and 8-choice RT), suggesting that pleiotropic genes have different strengths of association with correlated traits. Further, it might be that much smaller gene effects (and not detectable in our study at genome-wide level) influence pleiotropy among speed traits and, conversely, genes with larger effect influence the unique genetic variance also known to influence speed measures. Five of the candidate genes identified from the biological pathways analysis of the speed factor were implicated in the pathophysiology of AD. Several genes (e.g., <em>APOE</em>, <em>APP</em>) predisposing to AD have been associated with variation in normal cognition, and there is some indication that the effect of these genes may be stronger for processing speed than for general intelligence or even memory (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874786">Harris et al., 2007</a>; <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_287874811">Luciano et al., 2009</a>). Thus, the pursuit of AD genes in normal cognition, and especially processing speed traits, might prove worthwhile.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary data</h4><div class="sup-box half_rhythm" id="d35e897"><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_data.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(173K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary table 1</h4><div class="sup-box half_rhythm" id="d35e901"><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_table_1.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(154K, xls)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supplementary table 2</h4><div class="sup-box half_rhythm" id="d35e905"><a href="/pmc/articles/PMC3319015/bin/NIHMS47308-supplement-Supplementary_table_2.xls" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(71K, xls)</sup></div></div></div><div id="S17" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Acknowledgements</h2><div class="sec"><p id="__p1"><em>QIMR</em>: We especially thank the twins and their families for participation; Marlene Grace, Ann Eldridge, and Kerrie McAloney for phenotype collection; Anjali Henders, Megan Campbell, Lisa Bowdler, Steven Crooks and staff of the Molecular Epidemiology Laboratory for DNA sample processing and preparation; Harry Beeby, David Smyth and Daniel Park for IT support; and Drs Dale R Nyholt and Scott Gordon for their substantial efforts involving the QC and preparation of the GWA datasets. The research was supported by the Australian Research Council (A7960034, A79906588, A79801419, DP0212016, DP0343921), with genotyping funded by the National Health and Medical Research Council (Medical Bioinformatics Genomics Proteomics Program, 389891). The Genetic Cluster Computer (<a href="http://www.geneticcluster.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.geneticcluster.org</a>) which is financially supported by the Netherlands Organization for Scientific Research (NWO 480-05-003), was used for statistical analyses.</p><p id="__p2"><em>LBC</em>: We thank the LBC1921 and LBC1936 participants. We thank the LBC1936 study Secretary Paula Davies. We thank Michelle Taylor, Janie Corley, Caroline Brett, Alison Pattie, Martha Whiteman and Caroline Cameron for data collection and data entry. We thank the staff at Lothian Health Board, and the staff at the SCRE Centre, University of Glasgow; and Jenny Huffman for performing some of the genotyping. The LBC1921 data described here were collected in waves funded by the BBSRC and the Chief Scientist Office of the Scottish Executive. The LBC1936 was supported by a programme grant (to IJD and JMS) from Research Into Ageing, which continues with funding from Help the Aged as The Disconnected Mind. The work was undertaken by The University of Edinburgh Centre for Cognitive Aging and Cognitive Epidemiology, part of the cross council Lifelong Health and Wellbeing Initiative. Funding (to IJD and JMS) from the BBSRC, EPSRC, ESRC and MRC is gratefully acknowledged. ML is a Royal Society of Edinburgh/Lloyds TSB Foundation for Scotland Personal Research Fellow.</p><p id="__p3"><em>HBCS</em>: We thank all study participants as well as everybody involved in the Helsinki Birth Cohort Study. Helsinki Birth Cohort Study has been supported by grants from the Academy of Finland, the Finnish Diabetes Research Society, Folkhälsan Research Foundation, Novo Nordisk Foundation, Finska Läkaresällskapet, Signe and Ane Gyllenberg Foundation, University of Helsinki, European Science Foundation (EUROSTRESS), Ministry of Education, Ahokas Foundation, Emil Aaltonen Foundation, Juho Vainio Foundation, and Wellcome Trust (grant number WT089062).</p></div></div><div id="idm140131562366240" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140131562366240title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P31" class="p p-first-last"><strong>Appendix A. Supplementary data</strong> Supplementary data associated with this article can be found, in the online version, at doi:10.1016/j.biopsycho.2010.11.008.</p><p></p></div></div><div id="idm140131563542352" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140131563542352title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:disc;"><li id="R1"><span class="element-citation">Abecasis G. <span>2010</span>  <a href="http://www.sph.umich.edu/csg/abecasis/Metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/</a></span></li><li id="R2"><span class="element-citation">Andersson S, Lovdahl H, Malt UF. Neuropsychological function in unmedicated recurrent brief depression. <span><span class="ref-journal">Journal of Affective Disorders. </span>2010;<span class="ref-vol">125</span>(1–3):155–164.</span> [<a href="/pubmed/20085849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Affective+Disorders&amp;title=Neuropsychological+function+in+unmedicated+recurrent+brief+depression&amp;author=S+Andersson&amp;author=H+Lovdahl&amp;author=UF+Malt&amp;volume=125&amp;issue=1–3&amp;publication_year=2010&amp;pages=155-164&amp;pmid=20085849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Baker LA, Vernon PA, Ho H. The genetic correlation between intelligence and speed of information processing. <span><span class="ref-journal">Behavior Genetics. </span>1991;<span class="ref-vol">21</span>(4):351–367.</span> [<a href="/pubmed/1953598" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behavior+Genetics&amp;title=The+genetic+correlation+between+intelligence+and+speed+of+information+processing&amp;author=LA+Baker&amp;author=PA+Vernon&amp;author=H+Ho&amp;volume=21&amp;issue=4&amp;publication_year=1991&amp;pages=351-367&amp;pmid=1953598&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Barker DJ, Osmond C, Forsen TJ, Kajantie E, Eriksson JG. Trajectories of growth among children who have coronary events as adults. <span><span class="ref-journal">The New England Journal of Medicine. </span>2005;<span class="ref-vol">353</span>(17):1802–1809.</span> [<a href="/pubmed/16251536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+New+England+Journal+of+Medicine&amp;title=Trajectories+of+growth+among+children+who+have+coronary+events+as+adults&amp;author=DJ+Barker&amp;author=C+Osmond&amp;author=TJ+Forsen&amp;author=E+Kajantie&amp;author=JG+Eriksson&amp;volume=353&amp;issue=17&amp;publication_year=2005&amp;pages=1802-1809&amp;pmid=16251536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Baune BT, Ponath G, Rothermundt M, Riess O, Funke H, Berger K. Association between genetic variants of IL-1beta, IL-6 and TNF-alpha cytokines and cognitive performance in the elderly general population of the MEMO-study. <span><span class="ref-journal">Psychoneuroendocrinology. </span>2008;<span class="ref-vol">33</span>(1):68–76.</span> [<a href="/pubmed/17988804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychoneuroendocrinology&amp;title=Association+between+genetic+variants+of+IL-1beta,+IL-6+and+TNF-alpha+cytokines+and+cognitive+performance+in+the+elderly+general+population+of+the+MEMO-study&amp;author=BT+Baune&amp;author=G+Ponath&amp;author=M+Rothermundt&amp;author=O+Riess&amp;author=H+Funke&amp;volume=33&amp;issue=1&amp;publication_year=2008&amp;pages=68-76&amp;pmid=17988804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Benussi L, Govoni S, Gasparini L, Binetti G, Trabucchi M, Bianchetti A, et al.  Specific role for protein kinase C alpha in the constitutive and regulated secretion of amyloid precursor protein in human skin fibroblasts. <span><span class="ref-journal">Neuroscience Letters. </span>1998;<span class="ref-vol">240</span>(2):97–101.</span> [<a href="/pubmed/9486481" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience+Letters&amp;title=Specific+role+for+protein+kinase+C+alpha+in+the+constitutive+and+regulated+secretion+of+amyloid+precursor+protein+in+human+skin+fibroblasts&amp;author=L+Benussi&amp;author=S+Govoni&amp;author=L+Gasparini&amp;author=G+Binetti&amp;author=M+Trabucchi&amp;volume=240&amp;issue=2&amp;publication_year=1998&amp;pages=97-101&amp;pmid=9486481&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Cirulli ET, Kasperaviciute D, Attix DK, Need AC, Ge D, Gibson G, et al.  Common genetic variation and performance on standardized cognitive tests. <span><span class="ref-journal">European Journal of Human Genetics. </span>2010;<span class="ref-vol">18</span>(7):815–820.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987367/">PMC free article</a>]</span> [<a href="/pubmed/20125193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=European+Journal+of+Human+Genetics&amp;title=Common+genetic+variation+and+performance+on+standardized+cognitive+tests&amp;author=ET+Cirulli&amp;author=D+Kasperaviciute&amp;author=DK+Attix&amp;author=AC+Need&amp;author=D+Ge&amp;volume=18&amp;issue=7&amp;publication_year=2010&amp;pages=815-820&amp;pmid=20125193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Collie A, Darby D, Maruff P. Computerised cognitive assessment of athletes with sports related head injury. <span><span class="ref-journal">British Journal of Sports Medicine. </span>2001;<span class="ref-vol">35</span>(5):297–302.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1724391/">PMC free article</a>]</span> [<a href="/pubmed/11579059" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=British+Journal+of+Sports+Medicine&amp;title=Computerised+cognitive+assessment+of+athletes+with+sports+related+head+injury&amp;author=A+Collie&amp;author=D+Darby&amp;author=P+Maruff&amp;volume=35&amp;issue=5&amp;publication_year=2001&amp;pages=297-302&amp;pmid=11579059&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Collie A, Maruff P, Makdissi M, McCrory P, McStephen M, Darby D. CogSport: reliability and correlation with conventional cognitive tests used in postconcussion medical evaluations. <span><span class="ref-journal">Clinical Journal of Sport Medicine. </span>2003;<span class="ref-vol">13</span>(1):28–32.</span> [<a href="/pubmed/12544161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clinical+Journal+of+Sport+Medicine&amp;title=CogSport:+reliability+and+correlation+with+conventional+cognitive+tests+used+in+postconcussion+medical+evaluations&amp;author=A+Collie&amp;author=P+Maruff&amp;author=M+Makdissi&amp;author=P+McCrory&amp;author=M+McStephen&amp;volume=13&amp;issue=1&amp;publication_year=2003&amp;pages=28-32&amp;pmid=12544161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Couto JM, Gomez L, Wigg K, Ickowicz A, Pathare T, Malone M, et al.  Association of attention-deficit/hyperactivity disorder with a candidate region for reading disabilities on chromosome 6p. <span><span class="ref-journal">Biological Psychiatry. </span>2009;<span class="ref-vol">66</span>(4):368–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5750043/">PMC free article</a>]</span> [<a href="/pubmed/19362708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biological+Psychiatry&amp;title=Association+of+attention-deficit/hyperactivity+disorder+with+a+candidate+region+for+reading+disabilities+on+chromosome+6p&amp;author=JM+Couto&amp;author=L+Gomez&amp;author=K+Wigg&amp;author=A+Ickowicz&amp;author=T+Pathare&amp;volume=66&amp;issue=4&amp;publication_year=2009&amp;pages=368-375&amp;pmid=19362708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Darby D, Maruff P, Collie A, McStephen M. Mild cognitive impairment can be detected by multiple assessments in a single day. <span><span class="ref-journal">Neurology. </span>2002;<span class="ref-vol">59</span>(7):1042–1046.</span> [<a href="/pubmed/12370459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Mild+cognitive+impairment+can+be+detected+by+multiple+assessments+in+a+single+day&amp;author=D+Darby&amp;author=P+Maruff&amp;author=A+Collie&amp;author=M+McStephen&amp;volume=59&amp;issue=7&amp;publication_year=2002&amp;pages=1042-1046&amp;pmid=12370459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Deary IJ. Human intelligence differences: towards a combined experimental-differential approach. <span><span class="ref-journal">Trends in Cognitive Sciences. </span>2001;<span class="ref-vol">5</span>(4):164–170.</span> [<a href="/pubmed/11287270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+in+Cognitive+Sciences&amp;title=Human+intelligence+differences:+towards+a+combined+experimental-differential+approach&amp;author=IJ+Deary&amp;volume=5&amp;issue=4&amp;publication_year=2001&amp;pages=164-170&amp;pmid=11287270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Deary IJ, Der G. Reaction time explains IQ’s association with death. <span><span class="ref-journal">Psychological Science. </span>2005;<span class="ref-vol">16</span>(1):64–69.</span> [<a href="/pubmed/15660853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychological+Science&amp;title=Reaction+time+explains+IQ’s+association+with+death&amp;author=IJ+Deary&amp;author=G+Der&amp;volume=16&amp;issue=1&amp;publication_year=2005&amp;pages=64-69&amp;pmid=15660853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Deary IJ, Der G, Ford G. Reaction times and intelligence differences: a population-based cohort study. <span><span class="ref-journal">Intelligence. </span>2001;<span class="ref-vol">29</span>:389–399.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Intelligence&amp;title=Reaction+times+and+intelligence+differences:+a+population-based+cohort+study&amp;author=IJ+Deary&amp;author=G+Der&amp;author=G+Ford&amp;volume=29&amp;publication_year=2001&amp;pages=389-399&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Deary IJ, Gow AJ, Taylor MD, Corley J, Brett C, Wilson V, et al.  The Lothian Birth Cohort 1936: a study to examine influences on cognitive ageing from age 11 to age 70 and beyond. <span><span class="ref-journal">BMC Geriatrics. </span>2007;<span class="ref-vol">7</span>:28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2222601/">PMC free article</a>]</span> [<a href="/pubmed/18053258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Geriatrics&amp;title=The+Lothian+Birth+Cohort+1936:+a+study+to+examine+influences+on+cognitive+ageing+from+age+11+to+age+70+and+beyond&amp;author=IJ+Deary&amp;author=AJ+Gow&amp;author=MD+Taylor&amp;author=J+Corley&amp;author=C+Brett&amp;volume=7&amp;publication_year=2007&amp;pages=28&amp;pmid=18053258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Deary IJ, Johnson W, Starr JM. Are processing speed tasks biomarkers of cognitive aging? <span><span class="ref-journal">Psychology and Aging. </span>2010;<span class="ref-vol">25</span>(1):219–228.</span> [<a href="/pubmed/20230141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychology+and+Aging&amp;title=Are+processing+speed+tasks+biomarkers+of+cognitive+aging?&amp;author=IJ+Deary&amp;author=W+Johnson&amp;author=JM+Starr&amp;volume=25&amp;issue=1&amp;publication_year=2010&amp;pages=219-228&amp;pmid=20230141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Deary IJ, Simonotto E, Meyer M, Marshall A, Marshall I, Goddard N, et al.  The functional anatomy of inspection time: an event-related fMRI study. <span><span class="ref-journal">Neuroimage. </span>2004a;<span class="ref-vol">22</span>(4):1466–1479.</span> [<a href="/pubmed/15275904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=The+functional+anatomy+of+inspection+time:+an+event-related+fMRI+study&amp;author=IJ+Deary&amp;author=E+Simonotto&amp;author=M+Meyer&amp;author=A+Marshall&amp;author=I+Marshall&amp;volume=22&amp;issue=4&amp;publication_year=2004a&amp;pages=1466-1479&amp;pmid=15275904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Deary IJ, Stough C. Intelligence and inspection time. <span><span class="ref-journal">American Psychologist. </span>1996;<span class="ref-vol">51</span>(6):599–608.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Psychologist&amp;title=Intelligence+and+inspection+time&amp;author=IJ+Deary&amp;author=C+Stough&amp;volume=51&amp;issue=6&amp;publication_year=1996&amp;pages=599-608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Deary IJ, Whiteman MC, Starr JM, Whalley LJ, Fox HC. The impact of childhood intelligence on later life: following up the Scottish mental surveys of 1932 and 1947. <span><span class="ref-journal">Journal of Personality and Social Psychology. </span>2004b;<span class="ref-vol">86</span>(1):130–147.</span> [<a href="/pubmed/14717632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Personality+and+Social+Psychology&amp;title=The+impact+of+childhood+intelligence+on+later+life:+following+up+the+Scottish+mental+surveys+of+1932+and+1947&amp;author=IJ+Deary&amp;author=MC+Whiteman&amp;author=JM+Starr&amp;author=LJ+Whalley&amp;author=HC+Fox&amp;volume=86&amp;issue=1&amp;publication_year=2004b&amp;pages=130-147&amp;pmid=14717632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Der G, Deary IJ. Age and sex differences in reaction time in adulthood: results from the United Kingdom Health and Lifestyle Survey. <span><span class="ref-journal">Psychology and Aging. </span>2006;<span class="ref-vol">21</span>(1):62–73.</span> [<a href="/pubmed/16594792" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychology+and+Aging&amp;title=Age+and+sex+differences+in+reaction+time+in+adulthood:+results+from+the+United+Kingdom+Health+and+Lifestyle+Survey&amp;author=G+Der&amp;author=IJ+Deary&amp;volume=21&amp;issue=1&amp;publication_year=2006&amp;pages=62-73&amp;pmid=16594792&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Duggal P, Gillanders EM, Holmes TN, Bailey-Wilson JE. Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies. <span><span class="ref-journal">BMC Genomics. </span>2008;<span class="ref-vol">9</span>:516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2621212/">PMC free article</a>]</span> [<a href="/pubmed/18976480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Establishing+an+adjusted+p-value+threshold+to+control+the+family-wide+type+1+error+in+genome+wide+association+studies&amp;author=P+Duggal&amp;author=EM+Gillanders&amp;author=TN+Holmes&amp;author=JE+Bailey-Wilson&amp;volume=9&amp;publication_year=2008&amp;pages=516&amp;pmid=18976480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Eikelenboom P, Zhan SS, Kamphorst W, van der Valk P, Rozemuller JM. Cellular and substrate adhesion molecules (integrins) and their ligands in cerebral amyloid plaques in Alzheimer’s disease. <span><span class="ref-journal">Virchows Archiv: An International Journal of Pathology. </span>1994;<span class="ref-vol">424</span>(4):421–427.</span> [<a href="/pubmed/7515758" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Virchows+Archiv:+An+International+Journal+of+Pathology&amp;title=Cellular+and+substrate+adhesion+molecules+(integrins)+and+their+ligands+in+cerebral+amyloid+plaques+in+Alzheimer’s+disease&amp;author=P+Eikelenboom&amp;author=SS+Zhan&amp;author=W+Kamphorst&amp;author=P+van+der+Valk&amp;author=JM+Rozemuller&amp;volume=424&amp;issue=4&amp;publication_year=1994&amp;pages=421-427&amp;pmid=7515758&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Eriksson JG, Osmond C, Kajantie E, Forsen TJ, Barker DJ. Patterns of growth among children who later develop type 2 diabetes or its risk factors. <span><span class="ref-journal">Diabetologia. </span>2006;<span class="ref-vol">49</span>(12):2853–2858.</span> [<a href="/pubmed/17096117" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Patterns+of+growth+among+children+who+later+develop+type+2+diabetes+or+its+risk+factors&amp;author=JG+Eriksson&amp;author=C+Osmond&amp;author=E+Kajantie&amp;author=TJ+Forsen&amp;author=DJ+Barker&amp;volume=49&amp;issue=12&amp;publication_year=2006&amp;pages=2853-2858&amp;pmid=17096117&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Findlay JM. Estimates on probability functions: a more virulent PEST. <span><span class="ref-journal">Perception and Psychophysics. </span>1978;<span class="ref-vol">23</span>(2):181–185.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Perception+and+Psychophysics&amp;title=Estimates+on+probability+functions:+a+more+virulent+PEST&amp;author=JM+Findlay&amp;volume=23&amp;issue=2&amp;publication_year=1978&amp;pages=181-185&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Finkel D, Reynolds CA, McArdle JJ, Hamagami F, Pedersen NL. Genetic variance in processing speed drives variation in aging of spatial and memory abilities. <span><span class="ref-journal">Developmental Psychology. </span>2009;<span class="ref-vol">45</span>(3):820–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3742113/">PMC free article</a>]</span> [<a href="/pubmed/19413434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Developmental+Psychology&amp;title=Genetic+variance+in+processing+speed+drives+variation+in+aging+of+spatial+and+memory+abilities&amp;author=D+Finkel&amp;author=CA+Reynolds&amp;author=JJ+McArdle&amp;author=F+Hamagami&amp;author=NL+Pedersen&amp;volume=45&amp;issue=3&amp;publication_year=2009&amp;pages=820-834&amp;pmid=19413434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Giedraitis V, Kilander L, Degerman-Gunnarsson M, Sundelof J, Axelsson T, Syvanen AC, et al.  Genetic analysis of Alzheimer’s disease in the Uppsala Longitudinal Study of Adult Men. <span><span class="ref-journal">Dementia and Geriatric Cognitive Disorders. </span>2009;<span class="ref-vol">27</span>(1):59–68.</span> [<a href="/pubmed/19141999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dementia+and+Geriatric+Cognitive+Disorders&amp;title=Genetic+analysis+of+Alzheimer’s+disease+in+the+Uppsala+Longitudinal+Study+of+Adult+Men&amp;author=V+Giedraitis&amp;author=L+Kilander&amp;author=M+Degerman-Gunnarsson&amp;author=J+Sundelof&amp;author=T+Axelsson&amp;volume=27&amp;issue=1&amp;publication_year=2009&amp;pages=59-68&amp;pmid=19141999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Gong HP, Wang ZH, Jiang H, Fang NN, Li JS, Shang YY, et al.  TRIB3 functional Q84R polymorphism is a risk factor for metabolic syndrome and carotid atherosclerosis. <span><span class="ref-journal">Diabetes Care. </span>2009;<span class="ref-vol">32</span>(7):1311–1313.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2699701/">PMC free article</a>]</span> [<a href="/pubmed/19389818" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=TRIB3+functional+Q84R+polymorphism+is+a+risk+factor+for+metabolic+syndrome+and+carotid+atherosclerosis&amp;author=HP+Gong&amp;author=ZH+Wang&amp;author=H+Jiang&amp;author=NN+Fang&amp;author=JS+Li&amp;volume=32&amp;issue=7&amp;publication_year=2009&amp;pages=1311-1313&amp;pmid=19389818&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Greetham D, Ellis CD, Mewar D, Fearon U, an Ultaigh SN, Veale DJ, et al.  Functional characterization of NF-kappaB inhibitor-like protein 1 (NFkappa-BIL1), a candidate susceptibility gene for rheumatoid arthritis. <span><span class="ref-journal">Human Molecular Genetics. </span>2007;<span class="ref-vol">16</span>(24):3027–3036.</span> [<a href="/pubmed/17855452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Molecular+Genetics&amp;title=Functional+characterization+of+NF-kappaB+inhibitor-like+protein+1+(NFkappa-BIL1),+a+candidate+susceptibility+gene+for+rheumatoid+arthritis&amp;author=D+Greetham&amp;author=CD+Ellis&amp;author=D+Mewar&amp;author=U+Fearon&amp;author=SN+an+Ultaigh&amp;volume=16&amp;issue=24&amp;publication_year=2007&amp;pages=3027-3036&amp;pmid=17855452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Harris SE, Fox H, Wright AF, Hayward C, Starr JM, Whalley LJ, et al.  A genetic association analysis of cognitive ability and cognitive ageing using 325 markers for 109 genes associated with oxidative stress or cognition. <span><span class="ref-journal">BMC Genetics. </span>2007;<span class="ref-vol">8</span>:43.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1933580/">PMC free article</a>]</span> [<a href="/pubmed/17601350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genetics&amp;title=A+genetic+association+analysis+of+cognitive+ability+and+cognitive+ageing+using+325+markers+for+109+genes+associated+with+oxidative+stress+or+cognition&amp;author=SE+Harris&amp;author=H+Fox&amp;author=AF+Wright&amp;author=C+Hayward&amp;author=JM+Starr&amp;volume=8&amp;publication_year=2007&amp;pages=43&amp;pmid=17601350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Ho H, Baker LA, Decker SN. Covariation between intelligence and speed of cognitive processing: genetic and environmental influences. <span><span class="ref-journal">Behavior Genetics. </span>1988;<span class="ref-vol">18</span>(2):247–261.</span> [<a href="/pubmed/3377737" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behavior+Genetics&amp;title=Covariation+between+intelligence+and+speed+of+cognitive+processing:+genetic+and+environmental+influences&amp;author=H+Ho&amp;author=LA+Baker&amp;author=SN+Decker&amp;volume=18&amp;issue=2&amp;publication_year=1988&amp;pages=247-261&amp;pmid=3377737&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Holmans P, Green EK, Pahwa JS, Ferreira MA, Purcell SM, Sklar P, et al.  Gene ontology analysis of GWA study data sets provides insights into the biology of bipolar disorder. <span><span class="ref-journal">American Journal of Human Genetics. </span>2009;<span class="ref-vol">85</span>(1):13–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2706963/">PMC free article</a>]</span> [<a href="/pubmed/19539887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Gene+ontology+analysis+of+GWA+study+data+sets+provides+insights+into+the+biology+of+bipolar+disorder&amp;author=P+Holmans&amp;author=EK+Green&amp;author=JS+Pahwa&amp;author=MA+Ferreira&amp;author=SM+Purcell&amp;volume=85&amp;issue=1&amp;publication_year=2009&amp;pages=13-24&amp;pmid=19539887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Houlihan LM, Wyatt ND, Harris SE, Hayward C, Gow AJ, Marioni RE, et al.  Variation in the uric acid transporter gene (SLC2A9) is associated with memory performance. <span><span class="ref-journal">Human Molecular Genetics. </span>2010;<span class="ref-vol">19</span>(11):2321–2330.</span> [<a href="/pubmed/20197412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+Molecular+Genetics&amp;title=Variation+in+the+uric+acid+transporter+gene+(SLC2A9)+is+associated+with+memory+performance&amp;author=LM+Houlihan&amp;author=ND+Wyatt&amp;author=SE+Harris&amp;author=C+Hayward&amp;author=AJ+Gow&amp;volume=19&amp;issue=11&amp;publication_year=2010&amp;pages=2321-2330&amp;pmid=20197412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Jahshan C, Heaton RK, Golshan S, Cadenhead KS. Course of neurocognitive deficits in the prodrome and first episode of schizophrenia. <span><span class="ref-journal">Neuropsychology. </span>2009;<span class="ref-vol">24</span>(1):109–120.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2808194/">PMC free article</a>]</span> [<a href="/pubmed/20063952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuropsychology&amp;title=Course+of+neurocognitive+deficits+in+the+prodrome+and+first+episode+of+schizophrenia&amp;author=C+Jahshan&amp;author=RK+Heaton&amp;author=S+Golshan&amp;author=KS+Cadenhead&amp;volume=24&amp;issue=1&amp;publication_year=2009&amp;pages=109-120&amp;pmid=20063952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Kiuchi Y, Isobe Y, Fukushima K. Type IV collagen prevents amyloid beta-protein fibril formation. <span><span class="ref-journal">Life Sciences. </span>2002;<span class="ref-vol">70</span>(13):1555–1564.</span> [<a href="/pubmed/11895106" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Life+Sciences&amp;title=Type+IV+collagen+prevents+amyloid+beta-protein+fibril+formation&amp;author=Y+Kiuchi&amp;author=Y+Isobe&amp;author=K+Fukushima&amp;volume=70&amp;issue=13&amp;publication_year=2002&amp;pages=1555-1564&amp;pmid=11895106&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Kominami S, Tanabe N, Ota M, Naruse TK, Katsuyama Y, Nakanishi N, et al.  HLA-DPB1 and NFKBIL1 may confer the susceptibility to chronic thromboembolic pulmonary hypertension in the absence of deep vein thrombosis. <span><span class="ref-journal">Journal of Human Genetics. </span>2009;<span class="ref-vol">54</span>(2):108–114.</span> [<a href="/pubmed/19165231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Human+Genetics&amp;title=HLA-DPB1+and+NFKBIL1+may+confer+the+susceptibility+to+chronic+thromboembolic+pulmonary+hypertension+in+the+absence+of+deep+vein+thrombosis&amp;author=S+Kominami&amp;author=N+Tanabe&amp;author=M+Ota&amp;author=TK+Naruse&amp;author=Y+Katsuyama&amp;volume=54&amp;issue=2&amp;publication_year=2009&amp;pages=108-114&amp;pmid=19165231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Larson GE, Merritt CR, Williams SE. Information processing and intelligence: some implications of task complexity. <span><span class="ref-journal">Intelligence. </span>1988;<span class="ref-vol">12</span>:133–147.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Intelligence&amp;title=Information+processing+and+intelligence:+some+implications+of+task+complexity&amp;author=GE+Larson&amp;author=CR+Merritt&amp;author=SE+Williams&amp;volume=12&amp;publication_year=1988&amp;pages=133-147&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Latvala A, Castaneda AE, Perala J, Saarni SI, Aalto-Setala T, Lonnqvist J, et al.  Cognitive functioning in substance abuse and dependence: a population-based study of young adults. <span><span class="ref-journal">Addiction. </span>2009;<span class="ref-vol">104</span>(9):1558–1568.</span> [<a href="/pubmed/19686526" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Addiction&amp;title=Cognitive+functioning+in+substance+abuse+and+dependence:+a+population-based+study+of+young+adults&amp;author=A+Latvala&amp;author=AE+Castaneda&amp;author=J+Perala&amp;author=SI+Saarni&amp;author=T+Aalto-Setala&amp;volume=104&amp;issue=9&amp;publication_year=2009&amp;pages=1558-1568&amp;pmid=19686526&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Lin CH, Cho CL, Tsai WC, Ou TT, Wu CC, Yen JH, et al.  Inhibitors of kB-like gene polymorphisms in rheumatoid arthritis. <span><span class="ref-journal">Immunology Letters. </span>2006;<span class="ref-vol">105</span>(2):193–197.</span> [<a href="/pubmed/16644022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunology+Letters&amp;title=Inhibitors+of+kB-like+gene+polymorphisms+in+rheumatoid+arthritis&amp;author=CH+Lin&amp;author=CL+Cho&amp;author=WC+Tsai&amp;author=TT+Ou&amp;author=CC+Wu&amp;volume=105&amp;issue=2&amp;publication_year=2006&amp;pages=193-197&amp;pmid=16644022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Lind PA, Luciano M, Wright MJ, Montgomery GW, Martin NG, Bates TC. Dyslexia and DCDC2: normal variation in reading and spelling is associated with DCDC2 polymorphisms in an Australian population sample. <span><span class="ref-journal">European Journal of Human Genetics. </span>2010;<span class="ref-vol">18</span>(6):668–673.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987340/">PMC free article</a>]</span> [<a href="/pubmed/20068590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=European+Journal+of+Human+Genetics&amp;title=Dyslexia+and+DCDC2:+normal+variation+in+reading+and+spelling+is+associated+with+DCDC2+polymorphisms+in+an+Australian+population+sample&amp;author=PA+Lind&amp;author=M+Luciano&amp;author=MJ+Wright&amp;author=GW+Montgomery&amp;author=NG+Martin&amp;volume=18&amp;issue=6&amp;publication_year=2010&amp;pages=668-673&amp;pmid=20068590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Liolitsa D, Powell J, Lovestone S. Genetic variability in the insulin signalling pathway may contribute to the risk of late onset Alzheimer’s disease. <span><span class="ref-journal">Journal of Neurology, Neurosurgery, and Psychiatry. </span>2002;<span class="ref-vol">73</span>(3):261–266.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1738023/">PMC free article</a>]</span> [<a href="/pubmed/12185156" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Neurology,+Neurosurgery,+and+Psychiatry&amp;title=Genetic+variability+in+the+insulin+signalling+pathway+may+contribute+to+the+risk+of+late+onset+Alzheimer’s+disease&amp;author=D+Liolitsa&amp;author=J+Powell&amp;author=S+Lovestone&amp;volume=73&amp;issue=3&amp;publication_year=2002&amp;pages=261-266&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Luciano M, Gow AJ, Harris SE, Hayward C, Allerhand M, Starr JM, et al.  Cognitive ability at age 11 and 70 years, information processing speed, and APOE variation: the Lothian Birth Cohort 1936 study. <span><span class="ref-journal">Psychology and Aging. </span>2009;<span class="ref-vol">24</span>(1):129–138.</span> [<a href="/pubmed/19290744" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychology+and+Aging&amp;title=Cognitive+ability+at+age+11+and+70+years,+information+processing+speed,+and+APOE+variation:+the+Lothian+Birth+Cohort+1936+study&amp;author=M+Luciano&amp;author=AJ+Gow&amp;author=SE+Harris&amp;author=C+Hayward&amp;author=M+Allerhand&amp;volume=24&amp;issue=1&amp;publication_year=2009&amp;pages=129-138&amp;pmid=19290744&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Luciano M, Posthuma D, Wright MJ, de Geus EJ, Smith GA, Geffen GM, et al.  Perceptual speed does not cause intelligence, and intelligence does not cause perceptual speed. <span><span class="ref-journal">Biological Psychology. </span>2005;<span class="ref-vol">70</span>(1):1–8.</span> [<a href="/pubmed/16038769" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biological+Psychology&amp;title=Perceptual+speed+does+not+cause+intelligence,+and+intelligence+does+not+cause+perceptual+speed&amp;author=M+Luciano&amp;author=D+Posthuma&amp;author=MJ+Wright&amp;author=EJ+de+Geus&amp;author=GA+Smith&amp;volume=70&amp;issue=1&amp;publication_year=2005&amp;pages=1-8&amp;pmid=16038769&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Luciano M, Smith GA, Wright MJ, Geffen GM, Geffen LB, Martin NG. Genetic influence on the relationship between choice reaction time and IQ: a twin study. <span><span class="ref-journal">Foreign Psychology. </span>2001a;<span class="ref-vol">14</span>:49–59.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Foreign+Psychology&amp;title=Genetic+influence+on+the+relationship+between+choice+reaction+time+and+IQ:+a+twin+study&amp;author=M+Luciano&amp;author=GA+Smith&amp;author=MJ+Wright&amp;author=GM+Geffen&amp;author=LB+Geffen&amp;volume=14&amp;publication_year=2001a&amp;pages=49-59&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Luciano M, Smith GA, Wright MJ, Geffen GM, Geffen LB, Martin NG. On the heritability of inspection time and its covariance with IQ: a twin study. <span><span class="ref-journal">Intelligence. </span>2001b;<span class="ref-vol">29</span>(6):443–457.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Intelligence&amp;title=On+the+heritability+of+inspection+time+and+its+covariance+with+IQ:+a+twin+study&amp;author=M+Luciano&amp;author=GA+Smith&amp;author=MJ+Wright&amp;author=GM+Geffen&amp;author=LB+Geffen&amp;volume=29&amp;issue=6&amp;publication_year=2001b&amp;pages=443-457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Luciano M, Smith GA, Wright MJ, Geffen GM, Geffen LB, Martin NG.  Genetic covariance between processing speed and IQ. In: Plomin R, DeFries J, McGuffin P, Craig I, editors. <span class="ref-journal">Behavior Genetics in the Postgenomic Era.</span> APA Books; Washington, DC: 2003. pp. 163–182. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Behavior+Genetics+in+the+Postgenomic+Era&amp;author=M+Luciano&amp;author=GA+Smith&amp;author=MJ+Wright&amp;author=GM+Geffen&amp;author=LB+Geffen&amp;publication_year=2003&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Luciano M, Wright MJ, Geffen GM, Geffen LB, Smith GA, Martin NG. A genetic investigation of the covariation among inspection time, choice reaction time, and IQ subtest scores. <span><span class="ref-journal">Behavior Genetics. </span>2004;<span class="ref-vol">34</span>(1):41–50.</span> [<a href="/pubmed/14739695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behavior+Genetics&amp;title=A+genetic+investigation+of+the+covariation+among+inspection+time,+choice+reaction+time,+and+IQ+subtest+scores&amp;author=M+Luciano&amp;author=MJ+Wright&amp;author=GM+Geffen&amp;author=LB+Geffen&amp;author=GA+Smith&amp;volume=34&amp;issue=1&amp;publication_year=2004&amp;pages=41-50&amp;pmid=14739695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Lyons MJ, York TP, Franz CE, Grant MD, Eaves LJ, Jacobson KC, et al.  Genes determine stability and the environment determines change in cognitive ability during 35 years of adulthood. <span><span class="ref-journal">Psychological Science. </span>2009;<span class="ref-vol">20</span>(9):1146–1152.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2753423/">PMC free article</a>]</span> [<a href="/pubmed/19686293" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychological+Science&amp;title=Genes+determine+stability+and+the+environment+determines+change+in+cognitive+ability+during+35+years+of+adulthood&amp;author=MJ+Lyons&amp;author=TP+York&amp;author=CE+Franz&amp;author=MD+Grant&amp;author=LJ+Eaves&amp;volume=20&amp;issue=9&amp;publication_year=2009&amp;pages=1146-1152&amp;pmid=19686293&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">McGregor B, Pfitzner J, Zhu G, Grace M, Eldridge A, Pearson J, et al.  Genetic and environmental contributions to size, color, shape, and other characteristics of melanocytic naevi in a sample of adolescent twins. <span><span class="ref-journal">Genetic Epidemiology. </span>1999;<span class="ref-vol">16</span>:40–53.</span> [<a href="/pubmed/9915566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiology&amp;title=Genetic+and+environmental+contributions+to+size,+color,+shape,+and+other+characteristics+of+melanocytic+naevi+in+a+sample+of+adolescent+twins&amp;author=B+McGregor&amp;author=J+Pfitzner&amp;author=G+Zhu&amp;author=M+Grace&amp;author=A+Eldridge&amp;volume=16&amp;publication_year=1999&amp;pages=40-53&amp;pmid=9915566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">Meda SA, Gelernter J, Gruen JR, Calhoun VD, Meng H, Cope NA, et al.  Polymorphism of DCDC2 reveals differences in cortical morphology of healthy individuals – a preliminary voxel based morphometry study. <span><span class="ref-journal">Brain Imaging and Behavior. </span>2008;<span class="ref-vol">2</span>(1):21–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2605089/">PMC free article</a>]</span> [<a href="/pubmed/19096528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Imaging+and+Behavior&amp;title=Polymorphism+of+DCDC2+reveals+differences+in+cortical+morphology+of+healthy+individuals+–+a+preliminary+voxel+based+morphometry+study&amp;author=SA+Meda&amp;author=J+Gelernter&amp;author=JR+Gruen&amp;author=VD+Calhoun&amp;author=H+Meng&amp;volume=2&amp;issue=1&amp;publication_year=2008&amp;pages=21-26&amp;pmid=19096528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Medland SE, Nyholt DR, Painter JN, McEvoy BP, McRae AF, Zhu G, et al.  Common variants in the trichohyalin gene are associated with straight hair in Europeans. <span><span class="ref-journal">American Journal of Human Genetics. </span>2009;<span class="ref-vol">85</span>(5):750–755.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775823/">PMC free article</a>]</span> [<a href="/pubmed/19896111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Common+variants+in+the+trichohyalin+gene+are+associated+with+straight+hair+in+Europeans&amp;author=SE+Medland&amp;author=DR+Nyholt&amp;author=JN+Painter&amp;author=BP+McEvoy&amp;author=AF+McRae&amp;volume=85&amp;issue=5&amp;publication_year=2009&amp;pages=750-755&amp;pmid=19896111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="element-citation">Meng H, Smith SD, Hager K, Held M, Liu J, Olson RK, et al.  DCDC2 is associated with reading disability and modulates neuronal development in the brain. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2005;<span class="ref-vol">102</span>(47):17053–17058.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1278934/">PMC free article</a>]</span> [<a href="/pubmed/16278297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=DCDC2+is+associated+with+reading+disability+and+modulates+neuronal+development+in+the+brain&amp;author=H+Meng&amp;author=SD+Smith&amp;author=K+Hager&amp;author=M+Held&amp;author=J+Liu&amp;volume=102&amp;issue=47&amp;publication_year=2005&amp;pages=17053-17058&amp;pmid=16278297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="element-citation">Miyajima F, Ollier W, Mayes A, Jackson A, Thacker N, Rabbitt P, et al.  Brain-derived neurotrophic factor polymorphism Val66Met influences cognitive abilities in the elderly. <span><span class="ref-journal">Genes Brain and Behavior. </span>2008;<span class="ref-vol">7</span>(4):411–417.</span> [<a href="/pubmed/17973920" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+and+Behavior&amp;title=Brain-derived+neurotrophic+factor+polymorphism+Val66Met+influences+cognitive+abilities+in+the+elderly&amp;author=F+Miyajima&amp;author=W+Ollier&amp;author=A+Mayes&amp;author=A+Jackson&amp;author=N+Thacker&amp;volume=7&amp;issue=4&amp;publication_year=2008&amp;pages=411-417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R53"><span class="element-citation">Neubauer AC.  The mental speed approach to the assessment of intelligence. In: Kingma J, Tomic W, editors. <span class="ref-journal">Advances in Cognition and Educational Practice: Reflections on the Concept of Intelligence.</span> vol. 4. JAI Press, Inc.; Greenwhich, CT: 1997. pp. 149–173. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Advances+in+Cognition+and+Educational+Practice:+Reflections+on+the+Concept+of+Intelligence&amp;author=AC+Neubauer&amp;publication_year=1997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R54"><span class="element-citation">O’Dushlaine C, Kenny E, Heron EA, Segurado R, Gill M, Morris DW, et al.  The SNP ratio test: pathway analysis of genome-wide association datasets. <span><span class="ref-journal">Bioinformatics (Oxford, England) </span>2009;<span class="ref-vol">25</span>(20):2762–2763.</span> [<a href="/pubmed/19620097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+(Oxford,+England)&amp;title=The+SNP+ratio+test:+pathway+analysis+of+genome-wide+association+datasets&amp;author=C+O’Dushlaine&amp;author=E+Kenny&amp;author=EA+Heron&amp;author=R+Segurado&amp;author=M+Gill&amp;volume=25&amp;issue=20&amp;publication_year=2009&amp;pages=2762-2763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R55"><span class="element-citation">Okamoto K, Makino S, Yoshikawa Y, Takaki A, Nagatsuka Y, Ota M, et al.  Identification of I kappa BL as the second major histocompatibility complex-linked susceptibility locus for rheumatoid arthritis. <span><span class="ref-journal">American Journal of Human Genetics. </span>2003;<span class="ref-vol">72</span>(2):303–312.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC379224/">PMC free article</a>]</span> [<a href="/pubmed/12509789" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Identification+of+I+kappa+BL+as+the+second+major+histocompatibility+complex-linked+susceptibility+locus+for+rheumatoid+arthritis&amp;author=K+Okamoto&amp;author=S+Makino&amp;author=Y+Yoshikawa&amp;author=A+Takaki&amp;author=Y+Nagatsuka&amp;volume=72&amp;issue=2&amp;publication_year=2003&amp;pages=303-312&amp;pmid=12509789&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R56"><span class="element-citation">Ord T, Ord D, Koivomagi M, Juhkam K. Human TRB3 is upregulated in stressed cells by the induction of translationally efficient mRNA containing a truncated 5 -UTR. <span><span class="ref-journal">Gene. </span>2009;<span class="ref-vol">444</span>(1-2):24–32.</span> [<a href="/pubmed/19505541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Human+TRB3+is+upregulated+in+stressed+cells+by+the+induction+of+translationally+efficient+mRNA+containing+a+truncated+5+-UTR&amp;author=T+Ord&amp;author=D+Ord&amp;author=M+Koivomagi&amp;author=K+Juhkam&amp;volume=444&amp;issue=1-2&amp;publication_year=2009&amp;pages=24-32&amp;pmid=19505541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R57"><span class="element-citation">Paile-Hyvarinen M, Raikkonen K, Kajantie E, Darby D, Yliharsila H, Salonen MK, et al.  Impact of glucose metabolism and birth size on cognitive performance in elderly subjects. <span><span class="ref-journal">Diabetes Research and Clinical Practice. </span>2009;<span class="ref-vol">83</span>(3):379–386.</span> [<a href="/pubmed/19167774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Research+and+Clinical+Practice&amp;title=Impact+of+glucose+metabolism+and+birth+size+on+cognitive+performance+in+elderly+subjects&amp;author=M+Paile-Hyvarinen&amp;author=K+Raikkonen&amp;author=E+Kajantie&amp;author=D+Darby&amp;author=H+Yliharsila&amp;volume=83&amp;issue=3&amp;publication_year=2009&amp;pages=379-386&amp;pmid=19167774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R58"><span class="element-citation">Palu E, Liesi P. Differential distribution of laminins in Alzheimer disease and normal human brain tissue. <span><span class="ref-journal">Journal of Neuroscience Research. </span>2002;<span class="ref-vol">69</span>(2):243–256.</span> [<a href="/pubmed/12111806" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+Neuroscience+Research&amp;title=Differential+distribution+of+laminins+in+Alzheimer+disease+and+normal+human+brain+tissue&amp;author=E+Palu&amp;author=P+Liesi&amp;volume=69&amp;issue=2&amp;publication_year=2002&amp;pages=243-256&amp;pmid=12111806&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R59"><span class="element-citation">Pentland A. Maximum likelihood estimation: the best PEST. <span><span class="ref-journal">Perception and Psychophysics. </span>1980;<span class="ref-vol">28</span>(4):377–379.</span> [<a href="/pubmed/7465322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Perception+and+Psychophysics&amp;title=Maximum+likelihood+estimation:+the+best+PEST&amp;author=A+Pentland&amp;volume=28&amp;issue=4&amp;publication_year=1980&amp;pages=377-379&amp;pmid=7465322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R60"><span class="element-citation">Posthuma D, de Geus EJC, Boomsma DI. Perceptual speed and IQ are associated through common genetic factors. <span><span class="ref-journal">Behavior Genetics. </span>2001;<span class="ref-vol">31</span>(6):593–602.</span> [<a href="/pubmed/11838536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behavior+Genetics&amp;title=Perceptual+speed+and+IQ+are+associated+through+common+genetic+factors&amp;author=D+Posthuma&amp;author=EJC+de+Geus&amp;author=DI+Boomsma&amp;volume=31&amp;issue=6&amp;publication_year=2001&amp;pages=593-602&amp;pmid=11838536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R61"><span class="element-citation">Prudente S, Baratta R, Andreozzi F, Morini E, Farina MG, Nigro A, et al.  TRIB3 R84 variant affects glucose homeostasis by altering the interplay between insulin sensitivity and secretion. <span><span class="ref-journal">Diabetologia. </span>2010;<span class="ref-vol">53</span>(7):1354–1361.</span> [<a href="/pubmed/20393693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=TRIB3+R84+variant+affects+glucose+homeostasis+by+altering+the+interplay+between+insulin+sensitivity+and+secretion&amp;author=S+Prudente&amp;author=R+Baratta&amp;author=F+Andreozzi&amp;author=E+Morini&amp;author=MG+Farina&amp;volume=53&amp;issue=7&amp;publication_year=2010&amp;pages=1354-1361&amp;pmid=20393693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R62"><span class="element-citation">Prudente S, Scarpelli D, Chandalia M, Zhang YY, Morini E, Del Guerra S, et al.  The TRIB3 Q84R polymorphism and risk of early-onset type 2 diabetes. <span><span class="ref-journal">The Journal of Clinical Endocrinology and Metabolism. </span>2009;<span class="ref-vol">94</span>(1):190–196.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2630876/">PMC free article</a>]</span> [<a href="/pubmed/18984671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+Clinical+Endocrinology+and+Metabolism&amp;title=The+TRIB3+Q84R+polymorphism+and+risk+of+early-onset+type+2+diabetes&amp;author=S+Prudente&amp;author=D+Scarpelli&amp;author=M+Chandalia&amp;author=YY+Zhang&amp;author=E+Morini&amp;volume=94&amp;issue=1&amp;publication_year=2009&amp;pages=190-196&amp;pmid=18984671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R63"><span class="element-citation">Raikkonen K, Pesonen AK, Heinonen K, Lahti J, Kajantie E, Forsen T, et al.  Infant growth and hostility in adult life. <span><span class="ref-journal">Psychosomatic Medicine. </span>2008;<span class="ref-vol">70</span>(3):306–313.</span> [<a href="/pubmed/18378874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychosomatic+Medicine&amp;title=Infant+growth+and+hostility+in+adult+life&amp;author=K+Raikkonen&amp;author=AK+Pesonen&amp;author=K+Heinonen&amp;author=J+Lahti&amp;author=E+Kajantie&amp;volume=70&amp;issue=3&amp;publication_year=2008&amp;pages=306-313&amp;pmid=18378874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R64"><span class="element-citation">Ramasawmy R, Fae KC, Cunha-Neto E, Borba SC, Ianni B, Mady C, et al.  Variants in the promoter region of IKBL/NFKBIL1 gene may mark susceptibility to the development of chronic Chagas’ cardiomyopathy among Trypanosoma cruzi-infected individuals. <span><span class="ref-journal">Molecular Immunology. </span>2008;<span class="ref-vol">45</span>(1):283–288.</span> [<a href="/pubmed/17544510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+Immunology&amp;title=Variants+in+the+promoter+region+of+IKBL/NFKBIL1+gene+may+mark+susceptibility+to+the+development+of+chronic+Chagas’+cardiomyopathy+among+Trypanosoma+cruzi-infected+individuals&amp;author=R+Ramasawmy&amp;author=KC+Fae&amp;author=E+Cunha-Neto&amp;author=SC+Borba&amp;author=B+Ianni&amp;volume=45&amp;issue=1&amp;publication_year=2008&amp;pages=283-288&amp;pmid=17544510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R65"><span class="element-citation">Rijsdijk FV, Vernon PA, Boomsma DI. The genetic basis of the relation between speed-of-information-processing and IQ. <span><span class="ref-journal">Behavioural Brain Research. </span>1998;<span class="ref-vol">95</span>:77–84.</span> [<a href="/pubmed/9754879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Behavioural+Brain+Research&amp;title=The+genetic+basis+of+the+relation+between+speed-of-information-processing+and+IQ&amp;author=FV+Rijsdijk&amp;author=PA+Vernon&amp;author=DI+Boomsma&amp;volume=95&amp;publication_year=1998&amp;pages=77-84&amp;pmid=9754879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R66"><span class="element-citation">Salthouse TA. The processing-speed theory of adult age differences in cognition. <span><span class="ref-journal">Psychological Review. </span>1996;<span class="ref-vol">103</span>(3):403–428.</span> [<a href="/pubmed/8759042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychological+Review&amp;title=The+processing-speed+theory+of+adult+age+differences+in+cognition&amp;author=TA+Salthouse&amp;volume=103&amp;issue=3&amp;publication_year=1996&amp;pages=403-428&amp;pmid=8759042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R67"><span class="element-citation">Schumacher J, Anthoni H, Dahdouh F, Konig IR, Hillmer AM, Kluck N, et al.  Strong genetic evidence of DCDC2 as a susceptibility gene for dyslexia. <span><span class="ref-journal">American Journal of Human Genetics. </span>2006;<span class="ref-vol">78</span>(1):52–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380223/">PMC free article</a>]</span> [<a href="/pubmed/16385449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+Journal+of+Human+Genetics&amp;title=Strong+genetic+evidence+of+DCDC2+as+a+susceptibility+gene+for+dyslexia&amp;author=J+Schumacher&amp;author=H+Anthoni&amp;author=F+Dahdouh&amp;author=IR+Konig&amp;author=AM+Hillmer&amp;volume=78&amp;issue=1&amp;publication_year=2006&amp;pages=52-62&amp;pmid=16385449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R68"><span class="element-citation">Wechsler D.  <span class="ref-journal">Wechsler Adult Intelligence Scale – Revised Manual.</span> The Psychological Corporation; New York: 1981.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Wechsler+Adult+Intelligence+Scale+–+Revised+Manual&amp;author=D+Wechsler&amp;publication_year=1981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R69"><span class="element-citation">Wilcke A, Weissfuss J, Kirsten H, Wolfram G, Boltze J, Ahnert P. The role of gene DCDC2 in German dyslexics. <span><span class="ref-journal">Annals of Dyslexia. </span>2009;<span class="ref-vol">59</span>(1):1–11.</span> [<a href="/pubmed/19238550" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annals+of+Dyslexia&amp;title=The+role+of+gene+DCDC2+in+German+dyslexics&amp;author=A+Wilcke&amp;author=J+Weissfuss&amp;author=H+Kirsten&amp;author=G+Wolfram&amp;author=J+Boltze&amp;volume=59&amp;issue=1&amp;publication_year=2009&amp;pages=1-11&amp;pmid=19238550&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R70"><span class="element-citation">Wright MJ, De Geus E, Ando J, Luciano M, Posthuma D, Ono Y, et al.  Genetics of cognition: outline of a collaborative twin study. <span><span class="ref-journal">Twin Research. </span>2001;<span class="ref-vol">4</span>(1):48–56.</span> [<a href="/pubmed/11665325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Research&amp;title=Genetics+of+cognition:+outline+of+a+collaborative+twin+study&amp;author=MJ+Wright&amp;author=E+De+Geus&amp;author=J+Ando&amp;author=M+Luciano&amp;author=D+Posthuma&amp;volume=4&amp;issue=1&amp;publication_year=2001&amp;pages=48-56&amp;pmid=11665325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R71"><span class="element-citation">Wright MJ, Gillespie NA, Luciano M, Zhu G, Martin NG.  Genetics of personality and cognition in adolescents. In: Hudziak JJ, editor. <span class="ref-journal">Developmental Psychopathology and Wellness.</span> American Psychiatric Publishing, Inc.; Arlington, VA: 2008. pp. 85–108. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Developmental+Psychopathology+and+Wellness&amp;author=MJ+Wright&amp;author=NA+Gillespie&amp;author=M+Luciano&amp;author=G+Zhu&amp;author=NG+Martin&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R72"><span class="element-citation">Wright MJ, Smith GA, Geffen GM, Geffen LB, Martin NG. Genetic influence on the variance in coincidence timing and its covariance with IQ: a twin study. <span><span class="ref-journal">Intelligence. </span>2000;<span class="ref-vol">28</span>(4):239–250.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Intelligence&amp;title=Genetic+influence+on+the+variance+in+coincidence+timing+and+its+covariance+with+IQ:+a+twin+study&amp;author=MJ+Wright&amp;author=GA+Smith&amp;author=GM+Geffen&amp;author=LB+Geffen&amp;author=NG+Martin&amp;volume=28&amp;issue=4&amp;publication_year=2000&amp;pages=239-250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R73"><span class="element-citation">Yang J, Benyamin B, McEvoy BP, Gordon S, Henders AK, Nyholt DR, et al.  Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nature Genetics. </span>2010;<span class="ref-vol">42</span>(7):565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J+Yang&amp;author=B+Benyamin&amp;author=BP+McEvoy&amp;author=S+Gordon&amp;author=AK+Henders&amp;volume=42&amp;issue=7&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3319015&amp;issue-id=208455&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140131561838384title">Abstract</a></li><li><a href="#S1title">1. Methods</a></li><li><a href="#S14title">2. Results</a></li><li><a href="#S16title">3. Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S17title">Acknowledgements</a></li><li><a href="#idm140131562366240title">Footnotes</a></li><li><a href="#idm140131563542352title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3319015/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3319015/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3319015/pdf/ukmss-47308.pdf">PDF (322K)</a></li> | <li><a href="#" data-citationid="PMC3319015" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3319015%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3319015%2F&amp;text=Whole%20genome%20association%20scan%20for%20genetic%20polymorphisms%20influencing%20information%20processing%20speed" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3319015%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21130836" data-aiid="3319015" data-aid="3319015" data-iid="208455" data-domainid="358" data-domain="wtpa" data-accid="PMC3319015" data-md5="7ad64bed56cfb19533c2e79ef2fe0ff7"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:21:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8865B3DD52186100000000001D0012&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3319015%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3319015/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>