<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3049353/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association studies of adolescent idiopathic scoliosis suggest candidate susceptibility genes</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049353/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association studies of adolescent idiopathic scoliosis suggest candidate susceptibility genes"><meta name="citation_authors" content="Swarkar Sharma, Xiaochong Gao, Douglas Londono, Shonn E. Devroy, Kristen N. Mauldin, Jessica T. Frankel, January M. Brandon, Dongping Zhang, Quan-Zhen Li, Matthew B. Dobbs, Christina A. Gurnett, Struan F.A. Grant, Hakon Hakonarson, John P. Dormans, John A. Herring, Derek Gordon, Carol A. Wise"><meta name="citation_date" content="1 April 2011"><meta name="citation_issue" content="7"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="1456"><meta name="citation_doi" content="10.1093/hmg/ddq571"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3049353/?report=abstract"><meta name="citation_pmid" content="21216876"><meta name="DC.Title" content="Genome-wide association studies of adolescent idiopathic scoliosis suggest candidate susceptibility genes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Swarkar Sharma"><meta name="DC.Contributor" content="Xiaochong Gao"><meta name="DC.Contributor" content="Douglas Londono"><meta name="DC.Contributor" content="Shonn E. Devroy"><meta name="DC.Contributor" content="Kristen N. Mauldin"><meta name="DC.Contributor" content="Jessica T. Frankel"><meta name="DC.Contributor" content="January M. Brandon"><meta name="DC.Contributor" content="Dongping Zhang"><meta name="DC.Contributor" content="Quan-Zhen Li"><meta name="DC.Contributor" content="Matthew B. Dobbs"><meta name="DC.Contributor" content="Christina A. Gurnett"><meta name="DC.Contributor" content="Struan F.A. Grant"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Contributor" content="John P. Dormans"><meta name="DC.Contributor" content="John A. Herring"><meta name="DC.Contributor" content="Derek Gordon"><meta name="DC.Contributor" content="Carol A. Wise"><meta name="DC.Date" content="2011 Apr 1"><meta name="DC.Identifier" content="10.1093/hmg/ddq571"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association studies of adolescent idiopathic scoliosis suggest candidate susceptibility genes"><meta property="og:type" content="article"><meta property="og:description" content="Adolescent idiopathic scoliosis (AIS) is an unexplained and common spinal deformity seen in otherwise healthy children. Its pathophysiology is poorly understood despite intensive investigation. Although genetic underpinnings are clear, replicated susceptibility ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3049353/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3049353/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3049353/pdf/ddq571.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE887915DD5153910000000009A40643.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3049353%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3049353</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2011 Apr 1; 20(7): 1456–1466. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Jan 7. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddq571" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddq571</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3049353</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21216876">21216876</a></div></div></div></div><h1 class="content-title">Genome-wide association studies of adolescent idiopathic scoliosis suggest candidate susceptibility genes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Sharma%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128392080" co-class="co-affbox">Swarkar Sharma</a>,<sup>1</sup> <a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469126650000" co-class="co-affbox">Xiaochong Gao</a>,<sup>1</sup> <a href="/pubmed/?term=Londono%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128377168" co-class="co-affbox">Douglas Londono</a>,<sup>3</sup> <a href="/pubmed/?term=Devroy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128374848" co-class="co-affbox">Shonn E. Devroy</a>,<sup>1</sup> <a href="/pubmed/?term=Mauldin%20KN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128372576" co-class="co-affbox">Kristen N. Mauldin</a>,<sup>1</sup> <a href="/pubmed/?term=Frankel%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128370304" co-class="co-affbox">Jessica T. Frankel</a>,<sup>1</sup> <a href="/pubmed/?term=Brandon%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128368032" co-class="co-affbox">January M. Brandon</a>,<sup>1</sup> <a href="/pubmed/?term=Zhang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128365760" co-class="co-affbox">Dongping Zhang</a>,<sup>1</sup> <a href="/pubmed/?term=Li%20QZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128363488" co-class="co-affbox">Quan-Zhen Li</a>,<sup>4</sup> <a href="/pubmed/?term=Dobbs%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128361120" co-class="co-affbox">Matthew B. Dobbs</a>,<sup>7,</sup><sup>10</sup> <a href="/pubmed/?term=Gurnett%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128358016" co-class="co-affbox">Christina A. Gurnett</a>,<sup>7,</sup><sup>8,</sup><sup>9</sup> <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469128354032" co-class="co-affbox">Struan F.A. Grant</a>,<sup>11,</sup><sup>13</sup> <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469162879520" co-class="co-affbox">Hakon Hakonarson</a>,<sup>11,</sup><sup>13</sup> <a href="/pubmed/?term=Dormans%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469162876464" co-class="co-affbox">John P. Dormans</a>,<sup>12</sup> <a href="/pubmed/?term=Herring%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469162874096" co-class="co-affbox">John A. Herring</a>,<sup>2,</sup><sup>5</sup> <a href="/pubmed/?term=Gordon%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469162870992" co-class="co-affbox">Derek Gordon</a>,<sup>3</sup> and  <a href="/pubmed/?term=Wise%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876" class="affpopup" co-rid="_co_idm140469162868720" co-class="co-affbox">Carol A. Wise</a><sup>1,</sup><sup>5,</sup><sup>6,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140469128392080"><h3 class="no_margin">Swarkar Sharma</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Sharma%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Swarkar Sharma</a></div></div><div id="_co_idm140469126650000"><h3 class="no_margin">Xiaochong Gao</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Gao%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Xiaochong Gao</a></div></div><div id="_co_idm140469128377168"><h3 class="no_margin">Douglas Londono</h3><p><sup>3</sup>Department of Genetics, Rutgers, The State University of New Jersey, Piscataway, NJ, USA,</p><div>Find articles by <a href="/pubmed/?term=Londono%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Douglas Londono</a></div></div><div id="_co_idm140469128374848"><h3 class="no_margin">Shonn E. Devroy</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Devroy%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Shonn E. Devroy</a></div></div><div id="_co_idm140469128372576"><h3 class="no_margin">Kristen N. Mauldin</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Mauldin%20KN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Kristen N. Mauldin</a></div></div><div id="_co_idm140469128370304"><h3 class="no_margin">Jessica T. Frankel</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Frankel%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Jessica T. Frankel</a></div></div><div id="_co_idm140469128368032"><h3 class="no_margin">January M. Brandon</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Brandon%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">January M. Brandon</a></div></div><div id="_co_idm140469128365760"><h3 class="no_margin">Dongping Zhang</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><div>Find articles by <a href="/pubmed/?term=Zhang%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Dongping Zhang</a></div></div><div id="_co_idm140469128363488"><h3 class="no_margin">Quan-Zhen Li</h3><p><sup>4</sup>UTSW Microarray Core,</p><div>Find articles by <a href="/pubmed/?term=Li%20QZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Quan-Zhen Li</a></div></div><div id="_co_idm140469128361120"><h3 class="no_margin">Matthew B. Dobbs</h3><p><sup>7</sup>Department of Orthopaedics,</p><p><sup>10</sup>St Louis Shriners Hospital for Children, St Louis, MO, USA,</p><div>Find articles by <a href="/pubmed/?term=Dobbs%20MB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Matthew B. Dobbs</a></div></div><div id="_co_idm140469128358016"><h3 class="no_margin">Christina A. Gurnett</h3><p><sup>7</sup>Department of Orthopaedics,</p><p><sup>8</sup>Department of Pediatrics,  and</p><p><sup>9</sup>Department of Neurology, Washington University School of Medicine, St Louis, MO, USA,</p><div>Find articles by <a href="/pubmed/?term=Gurnett%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Christina A. Gurnett</a></div></div><div id="_co_idm140469128354032"><h3 class="no_margin">Struan F.A. Grant</h3><p><sup>11</sup>Center for Applied Genomics and Division of Human Genetics,  and</p><p><sup>13</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Grant%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Struan F.A. Grant</a></div></div><div id="_co_idm140469162879520"><h3 class="no_margin">Hakon Hakonarson</h3><p><sup>11</sup>Center for Applied Genomics and Division of Human Genetics,  and</p><p><sup>13</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Hakon Hakonarson</a></div></div><div id="_co_idm140469162876464"><h3 class="no_margin">John P. Dormans</h3><p><sup>12</sup>Department of Orthopaedic Surgery, The Children's Hospital of Philadelphia, PA, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Dormans%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">John P. Dormans</a></div></div><div id="_co_idm140469162874096"><h3 class="no_margin">John A. Herring</h3><p><sup>2</sup>Department of Orthopaedics, Texas Scottish Rite Hospital for Children, Dallas, TX, USA,</p><p><sup>5</sup>Department of Orthopaedics,  and</p><div>Find articles by <a href="/pubmed/?term=Herring%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">John A. Herring</a></div></div><div id="_co_idm140469162870992"><h3 class="no_margin">Derek Gordon</h3><p><sup>3</sup>Department of Genetics, Rutgers, The State University of New Jersey, Piscataway, NJ, USA,</p><div>Find articles by <a href="/pubmed/?term=Gordon%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Derek Gordon</a></div></div><div id="_co_idm140469162868720"><h3 class="no_margin">Carol A. Wise</h3><p><sup>1</sup>Seay Center for Musculoskeletal Research,  and</p><p><sup>5</sup>Department of Orthopaedics,  and</p><p><sup>6</sup>McDermott Center for Human Growth and Development, University of Texas Southwestern Medical Center at Dallas, Dallas, TX, USA,</p><div>Find articles by <a href="/pubmed/?term=Wise%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21216876">Carol A. Wise</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140469198695392_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140469198695392_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140469198695392_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140469198695392_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Seay Center for Musculoskeletal Research,  and</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Orthopaedics, Texas Scottish Rite Hospital for Children, Dallas, TX, USA,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Genetics, Rutgers, The State University of New Jersey, Piscataway, NJ, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>UTSW Microarray Core,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Orthopaedics,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>McDermott Center for Human Growth and Development, University of Texas Southwestern Medical Center at Dallas, Dallas, TX, USA,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Orthopaedics,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Pediatrics,  and</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Department of Neurology, Washington University School of Medicine, St Louis, MO, USA,</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>St Louis Shriners Hospital for Children, St Louis, MO, USA,</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Center for Applied Genomics and Division of Human Genetics,  and</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Department of Orthopaedic Surgery, The Children's Hospital of Philadelphia, PA, USA,  and</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Department of Pediatrics, University of Pennsylvania School of Medicine, Philadelphia, PA, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Email: <a href="mailto:dev@null" data-email="gro.hrst@esiw.lorac" class="oemail">gro.hrst@esiw.lorac</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140469198695392_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Sep 14; Revised 2010 Nov 9; Accepted 2010 Dec 31.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140469198695392_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3049353/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supplementary Data
            </div>
            
            <div><a href="/pmc/articles/PMC3049353/bin/supp_20_7_1456__index.html" data-ga-action="click_feat_suppl">supp_20_7_1456__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;58B84378-597E-498B-8E2E-D8942A84CF06</div>
            <div><a href="/pmc/articles/PMC3049353/bin/supp_ddq571_ddq571supp.doc" data-ga-action="click_feat_suppl">supp_ddq571_ddq571supp.doc</a><span style="color:gray"> (5.0M)</span></div><div class="small guid">GUID:&nbsp;BFEADAAB-CDBC-4B86-BC1D-CE398A76F469</div>
            <div><a href="/pmc/articles/PMC3049353/bin/supp_ddq571_ddq571supp_table9.xls" data-ga-action="click_feat_suppl">supp_ddq571_ddq571supp_table9.xls</a><span style="color:gray"> (76K)</span></div><div class="small guid">GUID:&nbsp;87211479-5E94-44B1-A228-D9F993E65F15</div>
         </div></dd></dl></div><div id="idm140469162817120" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140469162817120title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Adolescent idiopathic scoliosis (AIS) is an unexplained and common spinal deformity seen in otherwise healthy children. Its pathophysiology is poorly understood despite intensive investigation. Although genetic underpinnings are clear, replicated susceptibility loci that could provide insight into etiology have not been forthcoming. To address these issues, we performed genome-wide association studies (GWAS) of ∼327 000 single nucleotide polymorphisms (SNPs) in 419 AIS families. We found strongest evidence of association with chromosome 3p26.3 SNPs in the proximity of the <em>CHL1</em> gene (<em>P</em> &lt; 8 × 10<sup>−8</sup> for rs1400180). We genotyped additional chromosome 3p26.3 SNPs and tested replication in two follow-up case–control cohorts, obtaining strongest results when all three cohorts were combined (rs10510181 odds ratio = 1.49, 95% confidence interval = 1.29–1.73, <em>P</em> = 2.58 × 10<sup>−8</sup>), but these were not confirmed in a separate GWAS. <em>CHL1</em> is of interest, as it encodes an axon guidance protein related to Robo3. Mutations in the Robo3 protein cause <span class="underline">h</span>orizontal <span class="underline">g</span>aze <span class="underline">p</span>alsy with <span class="underline">p</span>rogressive <span class="underline">s</span>coliosis (HGPPS), a rare disease marked by severe scoliosis. Other top associations in our GWAS were with SNPs in the <em>DSCAM</em> gene encoding an axon guidance protein in the same structural class with Chl1 and Robo3. We additionally found AIS associations with loci in <em>CNTNAP2</em>, supporting a previous study linking this gene with AIS. Cntnap2 is also of functional interest, as it interacts directly with L1 and Robo class proteins and participates in axon pathfinding. Our results suggest the relevance of axon guidance pathways in AIS susceptibility, although these findings require further study, particularly given the apparent genetic heterogeneity in this disease.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Adolescent idiopathic scoliosis (AIS) is the most common pediatric spinal deformity, affecting ∼3% of school age children worldwide. Through unknown mechanisms, the vertebrae become misaligned such that the spine or a segment of the spine deviates from the midline laterally in the coronal plane, is rotated axially toward the side of deviation, and in the sagittal plane may deviate anteriorly (a lordotic deviation) or posteriorly (kyphotic deviation). AIS occurs in otherwise healthy children who bear no obvious deficiencies in the components of the spinal column itself, and the onset is highly correlated with the adolescent growth spurt (<a href="#DDQ571C1" rid="DDQ571C1" class=" bibr popnode">1</a>). School screenings are recommended due to the morbidity and rapid onset of disease (<a href="#DDQ571C2" rid="DDQ571C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275820">2</a>). Intervention is required in cases where the spinal deformity continues to worsen, so-called progressive disease. A first approach is bracing with an orthotic device that is designed to limit progression by holding the torso erect (<a href="#DDQ571C3" rid="DDQ571C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275821">3</a>). However, in extreme cases (∼1% of patients), bracing will not halt the progression of the deformity, and patients risk significant deformity with possible cardiopulmonary compromise. Treatment in these cases typically requires surgical intervention involving spinal instrumentation with fusion of selected vertebrae.</p><p id="__p3">The etiology of AIS is poorly understood as is implied by the name (<a href="#DDQ571C4" rid="DDQ571C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275785">4</a>). Indeed, although at least superficially it is a disease of the spinal column, the underlying pathophysiology is not clear. Lesions of the bony composition of the vertebrae, the vertebral endplates, the paraspinous muscles or the neurologic system each have been proposed at various times to explain disease pathogenesis. However, observational studies have had limited power in discerning whether these features are secondary to AIS or causative (<a href="#DDQ571C5" rid="DDQ571C5" class=" bibr popnode">5</a>). Progress in this regard has been hampered by the lack of an obvious AIS animal model. Many vertebrate strains (i.e. mouse and zebrafish) have been described with deformities of the spine or tail (<a href="#DDQ571C6" rid="DDQ571C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275822">6</a>–<a href="#DDQ571C9" rid="DDQ571C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275823">9</a>). Although it is anticipated that an appropriate vertebrate model will be invaluable for elucidating disease pathogenesis, at present it is not clear which if any of these animals recapitulates the patho-etiology of human AIS.</p><p id="__p4">Genetic contributions to AIS have been described for decades (<a href="#DDQ571C10" rid="DDQ571C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275791">10</a>). About one quarter of AIS patients report a positive family history of disease, and twin studies have consistently supported shared genetic factors in disease (reviewed in <a href="#DDQ571C4" rid="DDQ571C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275814">4</a>,<a href="#DDQ571C5" rid="DDQ571C5" class=" bibr popnode">5</a>). Although extended families are described, the genetic architecture of AIS appears to be generally complex. Accordingly, genome-wide linkage studies of multiplex families have identified many chromosomal regions of interest, but replication is reported for only one linkage peak, on chromosome 9q31.2–34.2 (<a href="#DDQ571C11" rid="DDQ571C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275798">11</a>,<a href="#DDQ571C12" rid="DDQ571C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275819">12</a>). Likewise, replicated causal genes or variants are also lacking. We previously performed targeted linkage studies in a cohort of 53 AIS multiplex families, and in follow-up studies we found association with haplotypes in the <em>CHD7</em> gene encoded on chromosome 8q. Rare mutations in <em>CHD7</em> are responsible for the CHARGE (<span class="underline">c</span>oloboma of the eye, <span class="underline">h</span>eart defects, <span class="underline">a</span>tresia of the choanae, <span class="underline">r</span>etardation of growth and/or development, genital and/or urinary abnormalities, and <span class="underline">e</span>ar abnormalities and deafness) syndrome of multiple anomalies that can include scoliosis (<a href="#DDQ571C13" rid="DDQ571C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275804">13</a>). From this study, we concluded that other genes responsible for rare Mendelian syndromes involving scoliosis might contribute to AIS, and that genome-wide studies of common variation in AIS cohorts would be fruitful in identifying susceptibility loci (<a href="#DDQ571C14" rid="DDQ571C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275837">14</a>).</p><p id="__p5" class="p p-last">In the present study, we searched more comprehensively for common AIS risk loci by performing genome-wide association studies (GWAS) in 419 Texas families. We identified top loci of interest using family-based tests of association that are robust to population stratification. We then tested replication of our most significant findings in additional independent cohorts of cases and controls ascertained in Texas and elsewhere in the USA. To our knowledge, this is the first reported GWAS for AIS. We provide genome-wide data and our top 100 single nucleotide polymorphism (SNP) associations so that others may replicate our findings. Our results provide candidate loci and genes worthy of further study, and particularly underscore genes involved in axon guidance pathways in AIS susceptibility.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p6" class="p p-first">Probands, other affected family members and parents were ascertained primarily in orthopedic clinics in Dallas, Texas and St Louis, Missouri (see Materials and Methods&nbsp;and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table&nbsp;S1</a> for a description of study populations). Samples from Texas AIS family trios (probands and parental controls) were genotyped using the Illumina HumanCNV370-quad platform that interrogates over 370 000 human polymorphisms. After applying stringent quality control, we performed tests of transmission disequilibrium (TDT) for 326 498 SNPs using the PLINK analysis program (<a href="#DDQ571C15" rid="DDQ571C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275832">15</a>) in two ways (see Materials and Methods for description of statistical methods). In the first analysis, genotypes for all 419 families (<em>n</em> = 1122) of all self-reported ethnicities were used, as the TDT statistic is robust to population stratification (<a href="#DDQ571C16" rid="DDQ571C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275816">16</a>). The second analysis was restricted to the subset of self-reported non-Hispanic white families, our largest ethnic group. To define the latter group, we corrected possible stratification by performing identity-by-state (IBS) analysis of unrelated probands using PLINK. Plotting the first three dimensions of a multidimensional scaling analysis of pairwise IBS distances identified one outlier family that was removed from further analyses (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). A plot of resulting −2log(e) <em>P</em>-values against expected results under the null hypothesis (quantile–quantile plot, Fig.&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F1" rid-ob="ob-DDQ571F1" co-legend-rid="lgnd_DDQ571F1"><span>1</span></a>) suggested a modest excess of associations without evidence of stratification (<em>λ</em><sub>GC</sub> = 1.0025) within the non-Hispanic white cohort. TDT results for the two data sets are shown in the form of Manhattan plots in Figures&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2" co-legend-rid="lgnd_DDQ571F2"><span>2</span></a> and <a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3" co-legend-rid="lgnd_DDQ571F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3.</span></span><span>3</span></a>. We also examined cryptic relatedness that could produce overly inflated results. Using pairwise inheritance-by-descent (IBD) estimation, we did not detect closely related pairs, with <em>π</em><sub>hat</sub> values &lt;0.15 for all samples (see Materials and Methods).


</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ571F1" co-legend-rid="lgnd_DDQ571F1"><a href="/pmc/articles/PMC3049353/figure/DDQ571F1/" target="figure" rid-figpopup="DDQ571F1" rid-ob="ob-DDQ571F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140469199329824" class="figure"><a href="/pmc/articles/PMC3049353/figure/DDQ571F1/" target="figure" rid-figpopup="DDQ571F1" rid-ob="ob-DDQ571F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3049353_ddq57101.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq57101.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3049353/bin/ddq57101.jpg"></a></div><div id="largeobj_idm140469199329824" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3049353/figure/DDQ571F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ571F1"><div><a class="figpopup" href="/pmc/articles/PMC3049353/figure/DDQ571F1/" target="figure" rid-figpopup="DDQ571F1" rid-ob="ob-DDQ571F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p7">Quantile–quantile (Q–Q) plot of actual versus expected −2log(e)<em>P</em> under the null hypothesis for a genome-wide data in non-Hispanic white probands. This plot was constructed using WGAViewer software (<a href="#DDQ571C49" rid="DDQ571C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275805">49</a>). The plot suggests departure from the null (<em>λ</em><sub>GC</sub> = 1.0025), with three SNPs (labeled in black) exceeding lift-off.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ571F2" co-legend-rid="lgnd_DDQ571F2"><a href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140469199325680" class="figure"><a href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3049353_ddq57102.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq57102.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3049353/bin/ddq57102.jpg"></a></div><div id="largeobj_idm140469199325680" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3049353/figure/DDQ571F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ571F2"><div><a class="figpopup" href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p8">Scatterplot (plotted using Haploview) of SNP −log10 <em>P</em>-values (Y-axis) obtained by TDT analyses in 419 Texas AIS family trios versus autosomal position (X-axis). <em>P</em>-values for SNPs on same chromosomes are represented in same color.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ571F3" co-legend-rid="lgnd_DDQ571F3"><a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140469199321552" class="figure"><a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3049353_ddq57103.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq57103.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3049353/bin/ddq57103.jpg"></a></div><div id="largeobj_idm140469199321552" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3049353/figure/DDQ571F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ571F3"><div><a class="figpopup" href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p9">Scatterplot (plotted using Haploview) of SNP −log10 <em>P</em>-values (Y-axis) obtained by TDT analyses in 339 (340 families—one outlier) non-Hispanic white AIS trios versus autosomal position (X-axis). <em>P</em>-values for SNPs on same chromosomes are represented in same color.</p></div></div></div><p id="__p10">We estimated that our discovery cohort would provide (dependent on allele frequencies) ∼90% power to detect disease associations with effect sizes [odds ratios (ORs)] ≥ 2.0 at a significance level <em>P</em> = 5.0 × 10<sup>−8</sup> (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2a</a>), and 70% power to detect loci with an effect size of 1.8, but only 10% power to detect weaker effect sizes of 1.5 or less at <em>P</em> = 5.0 × 10<sup>−8</sup> (<a href="#DDQ571C17" rid="DDQ571C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_334130905">17</a>). However, this cohort was potentially enriched for genetic risk factors, as 21% of the cases were familial. Only three SNPs met or exceeded a significance threshold <em>P</em> ≤ 1 × 10<sup>−5</sup>; however, genomic clustering suggested non-random association. Specifically, we noted that in the total data set, our most significant result was obtained for the SNP rs1400180 [OR = 1.92, 95% confidence interval (CI) = 1.48–2.49; <em>P</em> = 6.2 × 10<sup>−7</sup>], with nearby SNP rs10510181 among the top-ranked SNPs (OR = 1.88, 95% CI = 1.42–2.49; <em>P</em> = 7.1 × 10<sup>−6</sup>). These two SNPs are within 21 kb of each other at distal chromosome 3p and were ranked highest in the non-Hispanic white data set (Fig.&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3" co-legend-rid="lgnd_DDQ571F3"><span>3</span></a> and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). The evidence for association for the chromosome 3 SNPs rs1400180 (OR = 2.13; <em>P</em> = 7.9 × 10<sup>−8</sup>) and rs10510181 (OR = 2.03; <em>P</em> = 2.6 × 10<sup>−6</sup>) increased in the non-Hispanic white cohort despite the fact that this subset contained 80 fewer families than the total (results for non-white families are given in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>).</p><p id="__p11">We also imputed genotypes at untyped loci to potentially increase genome-wide coverage, given the relatively low density of the CNV370-quad platform. We imputed 2 271 581 genotypes in the non-Hispanic white families and tested these SNPs for association using the TDT statistic in PLINK as before. This analysis produced additional signals of interest (in terms of clustering and <em>P</em>-values), in particular for loci on chromosomes 1, 6 and 21 (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). However, SNPs clustering in the region of rs1400180 and rs10510181 remained the most significant by imputation (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S3 and S4</a>&nbsp;and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Fig. S4</a>), and we prioritized this region for further study (Fig.&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F4" rid-ob="ob-DDQ571F4" co-legend-rid="lgnd_DDQ571F4"><span>4</span></a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ571F4" co-legend-rid="lgnd_DDQ571F4"><a href="/pmc/articles/PMC3049353/figure/DDQ571F4/" target="figure" rid-figpopup="DDQ571F4" rid-ob="ob-DDQ571F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140469199295904" class="figure"><a href="/pmc/articles/PMC3049353/figure/DDQ571F4/" target="figure" rid-figpopup="DDQ571F4" rid-ob="ob-DDQ571F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3049353_ddq57104.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq57104.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3049353/bin/ddq57104.jpg"></a></div><div id="largeobj_idm140469199295904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3049353/figure/DDQ571F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ571F4"><div><a class="figpopup" href="/pmc/articles/PMC3049353/figure/DDQ571F4/" target="figure" rid-figpopup="DDQ571F4" rid-ob="ob-DDQ571F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p12">TDT results and LD structure for SNPs in the chromosome 3p26.3 (50 000–500 000 bp) region. Results are shown relative to physical locations as denoted in assembly GRCh37, build 37.1. Shown are TDT <em>P</em>-values plotted versus chromosomal location for SNPs present on the Illumina HumanCNV370-quad chip from the region in 419 discovery set families. The respective locations of the two genes <em>CHL1</em> (long arrow) and hypothetical gene <em>LOC642891</em> (short arrowhead) are also shown. Below are the pairwise <em>r</em><sup>2</sup> values for SNPs in the region represented graphically, where darker squares indicate greater correlation between SNPs. Only SNPs with MAF &gt; 0.01 are shown. The inverted triangle highlights the region including the top two SNPs rs1400180 and rs10510181. The call out box is the same region magnified to better show the respective <em>r<sup>2</sup></em> values.</p></div></div></div><p id="__p13">We subsequently tested SNPs rs1400180 and rs10510181 for evidence of allelic association with AIS in additional independent cohorts. In the first replication study, we genotyped samples from 375 Texas cases of non-Hispanic white ethnicity as defined by self-report as well as by IBS analyses using genotypes from 384 ancestry-informative markers (as described above and Materials and Methods, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>). Close relationship within or between this cohort and the discovery cohort was unlikely per extensive review of pedigree and demographic information. For both SNPs, we observed that the same allele that was overtransmitted in families was over-represented in cases when compared with controls (i.e. the same direction of effect). Strongest results were obtained for rs10510181, where the frequency in cases was 0.37 when compared with 0.32 in controls. In addition, a logistic regression analysis incorporating gender and age at onset as covariates yielded essentially the same results (Table&nbsp;<a href="/pmc/articles/PMC3049353/table/DDQ571TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ571TB1" rid-ob="ob-DDQ571TB1" co-legend-rid=""><span>1</span></a>). In the second replication study, we genotyped samples from 187 cases ascertained in US clinics outside of Texas and 222 controls, and again observed the same direction of effect for both SNPs and strongest results for SNP rs10510181 (Table&nbsp;<a href="/pmc/articles/PMC3049353/table/DDQ571TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ571TB1" rid-ob="ob-DDQ571TB1" co-legend-rid=""><span>1</span></a>). Combining the results of the two replication studies (562 cases, 666 controls) yielded OR = 1.36, 95% CI = 1.14–1.61, <em>P</em> = 0.0005 for rs10510181.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ571TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p14">AIS risk association results for chromosome 3p26.3 loci</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140469199287552" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="6" rowspan="1" align="left">Discovery<hr></th><th colspan="5" rowspan="1" align="left">Rep 1<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1" align="left">All families (419)</th><th colspan="3" rowspan="1" align="left">White, non-Hispanic families (339)</th><th colspan="5" rowspan="1" align="left">Dallas (375:444)</th></tr><tr><th rowspan="1" colspan="1" align="left">Marker</th><th rowspan="1" colspan="1" align="left">Location</th><th rowspan="1" colspan="1" align="left">Risk allele</th><th rowspan="1" colspan="1" align="left">T:U</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR<sub>allelic</sub> (95% CI)</th><th rowspan="1" colspan="1" align="left">T:U</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR<sub>allelic</sub> (95% CI)</th><th rowspan="1" colspan="1" align="left">Risk allele Ca:Co</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">OR<sub>allelic</sub> (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value*</th><th rowspan="1" colspan="1" align="left">OR* (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs1400180</td><td rowspan="1" colspan="1">170,968</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">165:86</td><td rowspan="1" colspan="1">6.15 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.92 (1.48–2.49)</td><td rowspan="1" colspan="1">151:71</td><td rowspan="1" colspan="1">7.91 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">2.13 (1.60–2.82)</td><td rowspan="1" colspan="1">0.46:0.41</td><td rowspan="1" colspan="1">0.067</td><td rowspan="1" colspan="1">1.21 (0.98–1.47)</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">1.22 (1.0–1.49)</td></tr><tr><td rowspan="1" colspan="1">rs9754850</td><td rowspan="1" colspan="1">190,290</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">120:80</td><td rowspan="1" colspan="1">0.0047</td><td rowspan="1" colspan="1">1.5 (1.13–1.99)</td><td rowspan="1" colspan="1">0.49:0.43</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">1.26 (1.03–1.54)</td><td rowspan="1" colspan="1">0.013</td><td rowspan="1" colspan="1">1.3 (1.06–1.6)</td></tr><tr><td rowspan="1" colspan="1">rs9754552</td><td rowspan="1" colspan="1">190,411</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">120:80</td><td rowspan="1" colspan="1">0.0047</td><td rowspan="1" colspan="1">1.5 (1.13–1.99)</td><td rowspan="1" colspan="1">0.49:0.44</td><td rowspan="1" colspan="1">0.031</td><td rowspan="1" colspan="1">1.25 (1.02–1.52)</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">1.29 (1.04–1.59)</td></tr><tr><td rowspan="1" colspan="1">rs10510181</td><td rowspan="1" colspan="1">191,047</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">141:75</td><td rowspan="1" colspan="1">7.10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">1.88 (1.42–2.49)</td><td rowspan="1" colspan="1">128:63</td><td rowspan="1" colspan="1">2.56 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">2.03 (1.50–2.75)</td><td rowspan="1" colspan="1">0.37:0.32</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">1.29 (1.04–1.59)</td><td rowspan="1" colspan="1">0.013</td><td rowspan="1" colspan="1">1.3 (1.06–1.62)</td></tr><tr><td colspan="14" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1" align="left">Rep 2</td><td colspan="2" rowspan="1" align="left">Rep 3</td><td colspan="4" rowspan="1" align="left">Combined</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1" align="left">St Louis/USA (187:222)</td><td colspan="2" rowspan="1" align="left">CHOP (137:2126)</td><td colspan="2" rowspan="1" align="left">Discovery white non-Hispanic + Rep1 + Rep2</td><td colspan="3" rowspan="1" align="left">Discovery white non-Hispanic + Rep1 + Rep2 + Rep3</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">Marker<hr></td><td rowspan="1" colspan="1">Location<hr></td><td rowspan="1" colspan="1">Risk allele<hr></td><td rowspan="1" colspan="1">Risk allele Ca:Co<hr></td><td rowspan="1" colspan="1"><em>P</em>-value<hr></td><td rowspan="1" colspan="1">OR<sub>allelic</sub> (95% CI)<hr></td><td rowspan="1" colspan="1">Risk allele Ca:Co<hr></td><td rowspan="1" colspan="1"><em>P</em>-value<hr></td><td rowspan="1" colspan="1"><em>P</em><sub>Fisher</sub><hr></td><td rowspan="1" colspan="1">OR<sub>allelic</sub> (95% CI)<hr></td><td rowspan="1" colspan="1"><em>P</em><sub>Fisher</sub><hr></td><td rowspan="1" colspan="1">OR<sub>Meta</sub> (95% CI)<hr></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs1400180</td><td rowspan="1" colspan="1">170,968</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.43:0.41</td><td rowspan="1" colspan="1">0.56</td><td rowspan="1" colspan="1">1.09 (0.82–1.45)</td><td rowspan="1" colspan="1">0.40:0.44</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">6.35 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">1.37 (1.19–1.58)</td><td rowspan="1" colspan="1">1.33 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">1.30 (1.14–1.48)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9754850</td><td rowspan="1" colspan="1">190,290</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.51:0.44</td><td rowspan="1" colspan="1">0.044</td><td rowspan="1" colspan="1">1.35 (1.01–1.80)</td><td rowspan="1" colspan="1">0.45:0.44</td><td rowspan="1" colspan="1">0.82</td><td rowspan="1" colspan="1">1.50 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">1.34 (1.16–1.54)</td><td rowspan="1" colspan="1">1.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1">1.29 (1.13–1.47)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs9754552</td><td rowspan="1" colspan="1">190,411</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.51:0.44</td><td rowspan="1" colspan="1">0.049</td><td rowspan="1" colspan="1">1.34 (1.00–1.79)</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">2.28 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">1.33 (1.15–1.53)</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1">ND</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs10510181</td><td rowspan="1" colspan="1">191,047</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.38:0.30</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">1.42 (1.05–1.92)</td><td rowspan="1" colspan="1">0.31:0.33</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">2.58 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">1.49 (1.29–1.73)</td><td rowspan="1" colspan="1">8.22 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.37 (1.20–1.58)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140469199287552" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3049353/table/DDQ571TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140469163800656"><p id="__p15" class="p p-first-last">Location, base pair position of SNP (Build GRCh37); T:U = transmitted:untransmitted count; Rep, replication cohort; Ca:Co, case:control risk allele frequencies; <em>P</em>-value, <em>P</em>-value assuming a central chi-square distribution with one degree of freedom and a statistic value equal to TDT-CHISQ; <em>P</em>*-value, <em>P-</em>value adjusted for gender, age and four dimension values c1, c2, c3 and c4 from MDS by logistic regression; OR, odds ratio; OR*, odds ratio adjusted for gender, age and four dimension values c1, c2, c3 and c4 from MDS by logistic regression; 95% CI, 95% confidence interval for ORX; <em>P</em><sub>Fisher</sub>, Fisher-combined <em>P-</em>values.</p></div></div></div><p id="__p16">Two overlapping genes, <em>CHL1</em> and <em>LOC642891</em>, are nearest the region of association that we observed and are predicted to be transcribed in opposite orientation. <em>CHL1</em> encodes <span class="underline">C</span>lose <span class="underline">H</span>omologue of <span class="underline">L1</span>, a member of the family of immunoglobulin-class L1 neural cell adhesion molecules. Chl1 functions in axonal guidance and neuronal migration (<a href="#DDQ571C18" rid="DDQ571C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275831">18</a>,<a href="#DDQ571C19" rid="DDQ571C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275787">19</a>); however, whether <em>LOC642891</em> encodes a functional protein is unknown. We confirmed transcription of both genes in fetal and adult brain (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6 and S7</a>). The <em>CHL1</em> gene spans &gt;212 kb and its putative promoter sequences are ∼45 kb distal to the associated haplotype. The predicted <em>LOC642891</em> gene spans ∼1.5 kb, where its 5′-end lies within <em>CHL1</em> intron 1 and its 3′-end lies within the putative <em>CHL1</em> promoter (Fig.&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F4" rid-ob="ob-DDQ571F4" co-legend-rid="lgnd_DDQ571F4"><span>4</span></a>). Forty-nine imputed SNPs from this region (50–500 kb at chromosome 3p26.3) including the <em>CHL1</em> and <em>LOC642891</em> genes produced <em>P</em>-values &lt; 9 × 10<sup>−4</sup> by TDT analysis (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>) of discovery data. To validate these findings, we selected four imputed SNPs (rs965084, rs1400182, rs9754850 and rs9754552) near rs1400180 and rs10510181 for genotyping in the discovery and Rep1 cohorts (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S4 and S5</a>). SNPs rs9754850 and rs9754552 yielded evidence for association (<em>P</em> &lt; 0.05) with the same direction of effect as in the discovery phase, but SNPs rs965084 and rs1400182 did not. SNPs rs9754850 and rs9754552 are within 800 bp of rs10510181 and are moderately correlated with this SNP: <em>r</em><sup>2</sup> (rs9754850: rs10510181) = 0.56, 0.57, and <em>r</em><sup>2</sup> (rs9754552: rs10510181) = 0.56, 0.58, respectively, in case–control and discovery cohorts. We obtained similar results for rs9754850 and rs9754552 in the Rep2 cohort (Table&nbsp;<a href="/pmc/articles/PMC3049353/table/DDQ571TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ571TB1" rid-ob="ob-DDQ571TB1" co-legend-rid=""><span>1</span></a>).</p><p id="__p17">In a third replication study (Rep 3), we compared our data to a separate GWAS of 137 AIS cases and 2126 controls ascertained at Children's Hospital of Philadelphia (CHOP). Allele frequencies for SNPs rs1400180, rs10510181 and rs9754850 did not differ in cases compared with controls in that study (Table&nbsp;<a href="/pmc/articles/PMC3049353/table/DDQ571TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ571TB1" rid-ob="ob-DDQ571TB1" co-legend-rid=""><span>1</span></a>). To assess potential bias in our locally ascertained controls, we examined SNP risk allele frequencies in available data sets. We found that risk allele frequencies for the four SNPs rs1400180, rs9754850, rs9754552 and rs10510181 were similar between our controls and four control data sets (total <em>n</em> = 3917 individuals), but not HapMap CEU, a difference we attribute to the relative few chromosomes represented in the HapMap data set (<em>n</em> = 60) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>).</p><p id="__p18">Taken together, these results suggest that a genomic region correlated with SNP rs10510181, proximal to the <em>CHL1</em> and <em>LOC642891</em> genes, is associated with increased AIS risk. The lack of replication in the CHOP GWAS may reflect issues of heterogeneity and power to detect modest effect sizes.</p><p id="__p19">Many prior observations have indirectly linked scoliosis and neuropathology (<a href="#DDQ571C4" rid="DDQ571C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275796">4</a>). Clear evidence that improper axonal targeting specifically can evoke scoliosis is evident in the rare autosomal recessive disease <span class="underline">h</span>orizontal <span class="underline">g</span>aze <span class="underline">p</span>alsy with <span class="underline">p</span>rogressive <span class="underline">s</span>coliosis (HGPPS, MIM #607313) that is remarkable for absent horizontal eye movements and severe progressive scoliosis. This disease is caused by homozygous or compound heterozygous mutations in the <em>ROBO3</em> gene encoding a transmembrane receptor that controls commissural axon guidance (<a href="#DDQ571C20" rid="DDQ571C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275786">20</a>). Brain imaging and neurophysiologic studies of HGPPS patients have revealed hindbrain anomalies and improper motor and sensory axonal projections (<a href="#DDQ571C21" rid="DDQ571C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275810">21</a>). We have noted with interest that Chl1 and Robo3 proteins belong to the same molecular (immunoglobulin transmembrane receptor) and functional (axon guidance and neurite outgrowth) classes. Thus, <em>CHL1</em> is a plausible candidate gene for AIS susceptibility. We selected 90 families having multiple members affected with AIS for analysis of the <em>CHL1</em> gene, with the rationale that such families could be more likely to harbor highly penetrant alleles (<a href="#DDQ571C22" rid="DDQ571C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275834">22</a>). Analysis of four markers in the region produced suggestive evidence for linkage (HLOD = 1.93, <em>P</em> = 0.001 at rs1400180) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). We re-sequenced coding exons and flanking intronic regions of the <em>CHL1</em> gene in 10 unrelated probands from AIS families with positive evidence of linkage to the region (LOD ≥ 1.0). We observed two coding changes that predict non-synonymous amino acid changes (rs2272522 and rs62230378) and are found in exons 3 and 17, respectively, of the <em>CHL1</em> gene. We noted with interest that SNP rs2272522 was previously associated with susceptibility to schizophrenia in separate studies of Japanese and Han Chinese populations (<a href="#DDQ571C23" rid="DDQ571C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275833">23</a>,<a href="#DDQ571C24" rid="DDQ571C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275824">24</a>). This SNP was actually present on the CNV370-quad beadchip and was informative in our population, but did not produce evidence for association with AIS. Further investigation of rs62230378 also did not yield evidence that this SNP was associated with AIS. We also observed nine non-coding variants, none of which predicted alterations of known functional elements, such as transcription factor binding sites or consensus splice sites (Materials and Methods, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>).</p><p id="__p20" class="p p-last">We observed additional associations in our discovery data with SNPs in axon guidance genes. Among the top results were several SNPs clustering in the <em>DSCAM</em> gene located on chromosome 21 within the Down syndrome critical region (OR = 0.56, 95% CI = 0.42–0.73; <em>P</em> = 2.26 × 10<sup>−5</sup> for rs2222973) (see top 100 SNP associations, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a>, and Figs&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2" co-legend-rid="lgnd_DDQ571F2"><span>2</span></a> and <a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3" co-legend-rid="lgnd_DDQ571F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3).</span></span><span>3</span></a>). <em>DSCAM</em> encodes <span class="underline">D</span>own <span class="underline">s</span>yndrome <span class="underline">c</span>ell <span class="underline">a</span>dhesion <span class="underline">m</span>olecule, an immunoglobulin-class neural cell adhesion molecule in the same molecular class with Chl1 and Robo3. Dscam likewise functions in axon guidance, including commissural axon pathfinding, as observed in both vertebrate and invertebrate model systems (<a href="#DDQ571C25" rid="DDQ571C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275812">25</a>,<a href="#DDQ571C26" rid="DDQ571C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275829">26</a>). We also noted rs11770843 in the <em>CNTNAP2</em> gene within our top-associated SNPs (OR = 1.75, 95% CI = 1.32–2.30; <em>P</em> = 6.20 × 10<sup>−5</sup>) and some evidence for association with nearby loci, although SNP coverage in this region was poor (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>). This could be a random effect, given the size of the <em>CNTNAP2</em> gene (2.3 Mb) and the number tests that we performed (297 SNPs from the <em>CNTNAP2</em> gene were genotyped). However, the evidence for association with rs11770843 remained significant after correction for the 297 tests. <em>CNTNAP2</em> is of interest, as this gene was previously linked with AIS (<a href="#DDQ571C27" rid="DDQ571C27" class=" bibr popnode">27</a>). The extent of overlap with our data is unclear, but we conclude that further study of <em>CNTNAP2</em> in AIS cohorts is warranted. <em>CNTNAP2</em> encodes contactin-associated protein 2, also called neurexin IV, that binds to the immunoglobulin-class neural cell adhesion molecule contactin-2 in <em>cis</em> and to the L1 family of neural cell adhesion molecules (possibly including Chl1) in <em>trans</em> (<a href="#DDQ571C28" rid="DDQ571C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275827">28</a>). In Drosophila, neurexin IV was recently shown to interact with Robo (<a href="#DDQ571C29" rid="DDQ571C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275795">29</a>). These data provide additional evidence for variation in <em>CNTNAP2</em> in AIS susceptibility. These and other top findings from our GWAS warrant further exploration in additional AIS cohorts.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p21" class="p p-first">AIS has been clinically recognized for centuries, yet the underlying molecular etiology and indeed the pathophysiology of the disease have remained elusive. We report here the first GWAS of AIS, which we conducted with the goal of identifying promising candidate loci for further analysis. We also anticipated that such loci could enlighten potential disease mechanisms. Our analyses yielded top results for a region of chromosome 3p26.3 associated with AIS risk in the proximity of the axon guidance gene <em>CHL1. CHL1</em> is a plausible candidate gene, given the prior evidence that compound heterozygous mutations in the structurally and functionally related gene <em>ROBO3</em> cause an extreme scoliosis phenotype, horizontal gaze palsy with progressive scoliosis (<a href="#DDQ571C20" rid="DDQ571C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275813">20</a>). We did not detect obvious causal variation within <em>CHL1</em> itself, but further re-sequencing in larger cohorts is needed to test this possibility. It is interesting to note that previous association studies have identified <em>CHL1</em> as a candidate gene for schizophrenia (<a href="#DDQ571C23" rid="DDQ571C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275784">23</a>,<a href="#DDQ571C24" rid="DDQ571C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275790">24</a>). The SNP specifically associated in these studies was not associated with AIS in our study, suggesting the possibility of distinct <em>CHL1</em>-mediated pathways in the two disorders. While we propose <em>CHL1</em> as a candidate gene in AIS susceptibility, we note that our experiments have shown <em>LOC642891</em> expression in the brain, and we cannot exclude a role for this gene in disease risk. The physical overlap of <em>CHL1</em> and <em>LOC642891</em> on chromosome 3p26.3 also suggests the interesting possibility of correlated expression (<a href="#DDQ571C30" rid="DDQ571C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303520007">30</a>).</p><p id="__p22">Our results suggest that variants in other immunoglobulin-class neural cell adhesion molecules and their interacting partners may contribute to AIS susceptibility. Specifically, candidate genes identified in this study underscore mechanisms of commissural axon guidance, i.e. axon crossing at the midline of the corticospinal tract. Under normal conditions, longitudinal axons are known to respond to attractive and repulsive cues at specified locations (<a href="#DDQ571C31" rid="DDQ571C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275788">31</a>). However, when such molecular cues are disrupted, as in the case of <em>ROBO3</em> mutations in HGPPS, axons may display aberrant growth and directionality. Clearly further study of the candidate genes and loci proposed in this study are needed to confirm causal roles. In this regard, we recently replicated association with SNPs in <em>DSCAM</em> in a second cohort of Texas AIS families (rs2222973 combined OR = 0.59, 95% CI = 0.48–0.74; <em>P</em> = 1.46 × 10<sup>−6</sup>). The role of Dscam in axon guidance and neurite outgrowth is well described, and on commissural axons it acts as a receptor for the attractive guidance molecule netrin (<a href="#DDQ571C25" rid="DDQ571C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275826">25</a>,<a href="#DDQ571C26" rid="DDQ571C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275817">26</a>). Dscam knockdowns in zebrafish embryos exhibit severe shortening of the anterior/posterior axis and, interestingly, partial knockdowns produce embryos with crooked tails (<a href="#DDQ571C32" rid="DDQ571C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275830">32</a>). Our data also provide additional evidence that variation in the <em>CNTNAP2</em> gene encoding neurexin IV contributes to AIS susceptibility. Neurexin IV is clearly involved in axon guidance pathways via its interactions with contactin and L1 neural cell adhesion molecules, and with Robo in a pathway of repulsive midline axon guidance (<a href="#DDQ571C29" rid="DDQ571C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275836">29</a>).</p><p id="__p23">The possibility of neuropathology underlying AIS has stemmed from several observations. Scoliosis is frequent in neurologic/neuromuscular diseases, including Duchenne's muscular dystrophy, spinal muscular atrophy etc. Spinal cord anomalies such as syrinx, a fluid-filled cavity in the spinal cord (or brainstem), is often detected in conjunction with scoliosis (<a href="#DDQ571C4" rid="DDQ571C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275809">4</a>,<a href="#DDQ571C5" rid="DDQ571C5" class=" bibr popnode">5</a>). Several investigations of AIS patients have suggested co-existing deficits in oculo-vestibular (visual/hearing) and proprioceptive function. As an example, one study measuring otolith vestibulo-ocular response in AIS found significant left–right asymmetry in horizontal eye movements in AIS patients compared with matched healthy control children; moreover, the average of horizontal, but not vertical, eye movements was significantly different (greater) in AIS patients than in controls (<a href="#DDQ571C33" rid="DDQ571C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275789">33</a>–<a href="#DDQ571C37" rid="DDQ571C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275835">37</a>). These data might suggest a disease continuum, with HGPPS at the extreme end of AIS clinical manifestations. How scoliosis develops in this disease is not clear, although secondary abrogation of muscle tone and locomotion has been hypothesized (<a href="#DDQ571C20" rid="DDQ571C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275808">20</a>).</p><p id="__p24">We also examined loci in our data set corresponding to previously published linkage peaks. Within the 9q31.2–34.2 interval, we observed modest associations clustering in four regions, with strongest results for the following SNPs (genes): rs4979321 (<em>ZNF618</em>; <em>P</em> = 0.0009), rs891725 (<em>AMBP</em>; <em>P</em> = 0.0003), rs1969944 (<em>PALM2</em>; <em>P</em> = 0.0005) and rs4836643 (intergenic; <em>P</em> = 0.0003). We also searched genomic intervals encoding the <em>ROBO3</em> and <em>CHD7</em> genes but did not observe significant associations, suggesting that common variation in these genes does not contribute significantly to overall disease risk in AIS populations. Complete re-sequencing of both genes in large AIS cohorts may be fruitful to test the possibility that they harbor rare disease-causing variation.</p><p id="__p25" class="p p-last">One conclusion from our GWAS is that no single locus contributes predominantly to AIS risk. This was not surprising, as such signals should have been detected in prior studies. Our data also suggested that genetic heterogeneity is a significant factor in AIS, illustrated in our replication studies of chromosome 3p candidate loci. Further studies in larger cohorts are needed to identify and confirm additional AIS associations with common variants. Our results suggest genomic regions where follow-up genotyping as well as deep re-sequencing to detect causal alleles in AIS cohorts may be fruitful.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study subjects</h3><p id="__p26" class="p p-first-last">All affected subjects considered for inclusion in the study met criteria for a positive diagnosis of idiopathic scoliosis: lateral deviation from the midline greater than 10° as measured by the Cobb angle method from standing spinal radiographs, axial rotation toward the side of the deviation and exclusion of relevant co-existing diagnoses. Control study subjects were parents (for TDT analyses) or unrelated individuals without history of scoliosis (for case/control analyses). Clinical details are summarized in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Ascertainment of Dallas, Texas study subjects</h3><p id="__p27" class="p p-first-last">All participating research subjects were recruited under a protocol approved by the University of Texas Southwestern Medical Center Institutional Review Board. Affected probands included in discovery and initial replication studies were ascertained in orthopedic clinics at Texas Scottish Rite Hospital for Children (TSRHC) as previously described (<a href="#DDQ571C14" rid="DDQ571C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275828">14</a>). Parents, and other affected family members where possible, were routinely ascertained as well. A pediatric orthopedic surgeon confirmed a positive diagnosis of AIS by exclusion of co-existing diagnoses and positive findings from standing spinal radiographs. Blood samples were obtained by venipuncture. In some cases, saliva samples were self-collected using the Oragene DNA kit (DNA Genotek, Inc.). Four hundred nineteen probands and parents of all self-reported ethnicities were included in the discovery phase as detailed below. This cohort, as reported for the proband, included 81% white non-Hispanic, 9% black non-Hispanic, 6% Hispanic/Latino, 1% Asian/Pacific Islander and 3% other. Three hundred seventy-six affected probands were similarly ascertained in orthopedic clinics at TSRHC and included in the first replication (Rep1 cohort).</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Ascertainment of St Louis, Missouri study subjects</h3><p id="__p28" class="p p-first-last">All study subjects were recruited under protocols approved by either Washington University or the St Louis Shriners Hospital for Children Institutional Review Board. Affected probands were ascertained by a single pediatric orthopedic surgeon (M.B.D.) at St Louis Shriners Hospital for Children. Non-hispanic white St Louis study subjects (<em>n</em> = 174) were included in the Rep2 cohort.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">USA study subjects</h3><p id="__p29" class="p p-first-last">Additional probands were ascertained by pediatric orthopedic surgeons at the time of treatment at Shriners Hospital for Children, Lexington, KY, USA (T. Milbrandt, V. Talwalkar, H. J. Iwinski); Hasbro Childrens' Hospital, Providence, RI, USA (C. P. Eberson); University of Massachusetts Memorial Medical Center, Worcester, MA, USA (A. Lapinsky); Childrens Hospital of Wisconsin, Milwaukee, WI, USA (J. C. Tassone, X. C. Liu) and Akron Children's Hospital, Akron, OH, USA (W. Schrader). All study subjects were recruited according to the protocols approved by local corresponding ethics boards or the University of Texas Southwestern Medical Center Institutional Review Board. USA study subjects (<em>n</em> = 13) were included in the Rep2 cohort (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>).</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Controls</h3><p id="__p30" class="p p-first">Control individuals were ascertained from within the local Texas population or non-orthopedic clinics at TSRHC. A diagnosis of scoliosis was excluded by questionnaire. Six hundred sixty-six of these were of self-reported white, non-Hispanic ethnicity. These individuals were randomly assigned as either Rep1 (444 individuals) or Rep2 (222 individuals) controls in proportion to the number of cases in each group.</p><p id="__p31" class="p p-last">We also obtained genotypes for three publicly available control data sets: (i) National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK) IBDGC Crohn's Disease Genome-Wide Association Study; (ii) National Institute of Neurological Disorders and Stroke (NINDS) Genome Wide Association in Familial Parkinson Disease; and (iii) European American controls from the New York Health project Intragen Population Genetics database (<a href="#DDQ571C38" rid="DDQ571C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_278432807">38</a>). For the purposes of this study, we refer to these as the ‘NIDDK', ‘NINDS' and ‘Intragen' controls. NIDDK controls consisted of 895 individuals of non-Jewish, European ancestry that were genotyped with the Illumina HumanHap300 Beadchip and were without history of inflammatory bowel disease. The NINDS controls consisted of 943 Caucasian, non-Hispanic individuals that were genotyped with the Illumina HumanCNV370 Beadchip and were negative for neurological disease by self-report. The data retrieved from IntraGen Population Genetics Database (IntraGenDB) consisted of 478 individuals between the ages of 30 and 60, of non-Jewish white ancestry, genotyped with the Illumina HumanHap300 genotyping platform.</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Discovery genotyping and quality assurance</h3><p id="__p32" class="p p-first-last">We genotyped 1128 samples from 422 families (probands and parents) in the discovery phase. The majority of DNA samples were derived from blood; 66 (5.4%) of DNA samples were derived from saliva. Each DNA sample was quantified in triplicate with the Quanti-iT PicoGreen dsDNA reagent (Invitrogen) and assessed for quality by gel electrophoresis prior to genotyping. Each DNA sample (250–375 ng) was subsequently genotyped on Illumina Human CNV370-Quad arrays (Illumina). Genotypes with quality metrics (‘GenCall scores') &gt; 0.15 were called. For the remaining SNPs, the average success rate for each SNP was 99.20%. To test reproducibility of SNP calls, we genotyped 42 samples (3.54% of the total) in duplicate on separate arrays. The average concordance for the 42 samples was &gt;99.99%. We then filtered individual samples and SNPs that failed the following quality control measures. Three families displayed mis-inheritances and their samples were excluded from analysis. For the remaining 419 families (1122 samples), the percent of SNPs called successfully was &gt;95% in all but 9 samples (&lt;1%). These nine samples were removed from further analysis, leaving 1113 samples with an average SNP success rate &gt;0.998. We note that the SNP success rate was 100% for both rs1400180 and rs10510181 after quality control. Individual SNPs were further pruned for low call rates (&lt;0.95), deviation from Hardy–Weinberg equilibrium (<em>P</em> &lt; 0.00001), and minor allele frequency (MAF) &lt; 0.01. In total, 326 498 autosomal SNPs were retained for further analyses.</p></div><div id="s4g" class="sec"><h3 id="s4gtitle">Statistical analyses in the discovery cohort</h3><p id="__p33" class="p p-first">We used the SNP genotypes in 1113 samples that remained after quality control (as described above) in statistical analyses. We performed statistical analyses using software within the BC/SNPmax database platform (Biocomputing Platforms Ltd.).</p><div id="s4g1" class="sec"><p></p><h4 id="s4g1title" class="inline">IBS and multidimensional scaling </h4><p id="__p34" class="p p-first-last">We searched for evidence of population stratification in self-reported non-Hispanic white probands of our discovery set using IBS analysis as implemented in PLINK (<a href="#DDQ571C15" rid="DDQ571C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275799">15</a>). For this purpose, we included autosomal SNPs present on the Illumina Human 370-quad chip and in HapMap release 23a for which we downloaded genotype data for 210 unrelated HapMap samples (60 CEU, 45 HCB, 45 JPT and 60 YRI) (<a href="http://www.hapmap.org">www.hapmap.org</a>) (<a href="#DDQ571C39" rid="DDQ571C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275803">39</a>). Of these, one of each SNP pair with pairwise <em>r</em><sup>2</sup> &gt; 0.5 was pruned, as were SNPs with call rate &lt;99%, MAF &lt; 0.01 and a possible disease association (<em>P</em> &lt; 0.05, observed in 419 Texas AIS family trios) that left 8297 SNPs. Dimensions based on IBS distances in non-Hispanic white probands from the discovery set and HapMap samples were calculated, and multidimensional scaling (MDS) plots were made using the first three dimensions. Outliers (1 out of the 340 families) were identified from the discovery set and removed from further analyses (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>).</p></div><div id="s4g2" class="sec"><p></p><h4 id="s4g2title" class="inline">IBD analyses and π<sub>hat</sub> estimation </h4><p id="__p35" class="p p-first-last">We calculated pairwise IBD coefficients using PLINK to assess cryptic relatedness between probands. SNPs were selected using the same criteria as for IBS analyses. We required <em>π</em><sub>hat</sub> &lt; 0.15 for each sample pair to exclude cryptic relationship (on the order of second cousins in an outbred population). All proband pairs passed the <em>π</em><sub>hat</sub> &lt; 0.15 threshold, with <em>π</em><sub>hat</sub> = 0.072 being the highest value observed.</p></div><div id="s4g3" class="sec"><p></p><h4 id="s4g3title" class="inline">TDT analyses in the discovery set </h4><p id="__p36" class="p p-first-last">TDT results, displayed as Manhattan plots in Figures&nbsp;<a href="/pmc/articles/PMC3049353/figure/DDQ571F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F2" rid-ob="ob-DDQ571F2" co-legend-rid="lgnd_DDQ571F2"><span>2</span></a> and <a href="/pmc/articles/PMC3049353/figure/DDQ571F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ571F3" rid-ob="ob-DDQ571F3" co-legend-rid="lgnd_DDQ571F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3,</span></span><span>3</span></a>, were computed using the methods implemented in the PLINK software (<a href="#DDQ571C15" rid="DDQ571C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275807">15</a>). We also computed TDT results using the TDTae statistic that is robust to genotyping errors and missing genotypes (<a href="#DDQ571C40" rid="DDQ571C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275818">40</a>). These results are given in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>. Pedigrees were subsequently assigned to one of two groups, ‘non-Hispanic white' (<em>n</em> = 339 after removal of outlier family), or ‘non-white' (<em>n</em> = 79), and TDT was computed for each in the same manner as above. We also performed a genome-wide TDT analysis in the non-Hispanic black trios, our largest non-white group. The resultant <em>P</em>-values for each of the two markers on chromosome 3 are presented in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>. None of the neighboring markers (within 500 kb) in these two groups showed TDT <em>P</em>-values less than 0.01 (data not shown).</p></div><div id="s4g4" class="sec"><p></p><h4 id="s4g4title" class="inline">Power analyses </h4><p id="__p37" class="p p-first-last">Power analyses were performed for the discovery cohort as described in the text. We also calculated power provided by our replication cohorts for <em>P</em> = 0.05 (<a href="#DDQ571C41" rid="DDQ571C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275797">41</a>) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2a and b</a>).</p></div><div id="s4g5" class="sec sec-last"><p></p><h4 id="s4g5title" class="inline">Imputation </h4><p id="__p38" class="p p-first-last">To predict individual genotypes at un-typed loci, we used statistics contained in the software application MaCH 1.0 (<a href="#DDQ571C42" rid="DDQ571C42" class=" bibr popnode">42</a>), applying HAPMAP reference haplotype release #22 data for population CEU. Genome-wide genotypes were imputed for the 339 non-Hispanic white families and analyzed using the TDT in PLINK as before (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S3 and S4</a> &nbsp;and <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Figs S3 and S4</a>).</p></div></div><div id="s4h" class="sec sec-last"><h3 id="s4htitle">Statistical analyses in the replication cohorts</h3><p id="__p39" class="p p-first">In our first replication study, we evaluated the possibility of biases due to population stratification using multidimensional scaling analysis of IBS distances as previously described. For this, we designed a panel of 384 ancestry-informative markers selected from Illumina databases and published literature (<a href="#DDQ571C43" rid="DDQ571C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275802">43</a>). We selected autosomal SNPs &gt;100 kb apart and present on the Illumina CNV370 beadchip (a full list of SNPs is available upon request). As a positive control, we performed IBS analyses in the discovery set utilizing these 384 SNP genotypes (obtained in discovery) and obtained essentially the same results as with the 8297 described above (data not shown). We genotyped these SNPs in the Rep1 cohort and controls using the Illumina GoldenGate platform. The average success rate for each SNP was 99.4%; five SNPs with call rates &lt;95% were excluded from analyses, leaving 379 SNPs. We included three duplicate samples and observed 100% concordant SNP calls. Subsequent IBS/MDS analysis identified one outlier individual in this data set that was removed, leaving 375 AIS individuals for further analyses (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>).</p><div id="s4h1" class="sec"><p></p><h4 id="s4h1title" class="inline">Individual genotyping </h4><p id="__p40" class="p p-first-last">We performed follow-up genotyping for SNPs rs1400180, rs965084, rs1400182, rs9754850, rs9754552 and rs10501081 using Taqman genotyping as previously described (<a href="#DDQ571C14" rid="DDQ571C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275793">14</a>). To check agreement across platforms, we included ∼30% of discovery samples in Taqman genotyping for rs1400180 and rs10510181 and found 100% concordance for the two methods. All genotyped SNPs displayed agreement with Hardy–Weinberg equilibrium (<em>P</em> &gt; 0.05) with the exception of rs1400180 (<em>P</em> = 0.04) which showed marginal deviation from HWE.</p></div><div id="s4h2" class="sec"><p></p><h4 id="s4h2title" class="inline">Case–control statistics </h4><p id="__p41" class="p p-first-last">We genotyped six SNPs: the original rs1400180 and rs10510181, and rs965084, rs1400182, rs9754850 and rs9754552, in the first replication study. The number of cases and controls included in this study (‘Rep1') was 375 and 444, respectively. We performed allelic tests of association in cases and controls. As above, the statistic was computed on all data using the method implemented in the PLINK software (<a href="#DDQ571C15" rid="DDQ571C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275801">15</a>). We genotyped four of the SNPs rs1400180, rs10510181, rs9754850, and rs9754552 in the second replication study. The number of cases and controls included in this study (‘Rep2') was 187 and 222, respectively. We performed allelic tests of association as in the Rep1 study.</p></div><div id="s4h3" class="sec"><p></p><h4 id="s4h3title" class="inline">Combined statistics </h4><p id="__p42" class="p p-first-last">We estimated combined ORs and CIs for TDT and case–control results using the method implemented in the Catmap software program (<a href="#DDQ571C44" rid="DDQ571C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275811">44</a>) We also estimated the combined <em>P</em>-values using Fisher's method as implemented in the PVALUES software program (<a href="#DDQ571C45" rid="DDQ571C45" class=" bibr popnode">45</a>). Results were as reported in the text.</p></div><div id="s4h4" class="sec"><p></p><h4 id="s4h4title" class="inline">LD structure </h4><p id="__p43" class="p p-first-last">Pairwise <em>r</em><sup>2</sup> values were computed using Haploview software Version 4.1 (<a href="#DDQ571C46" rid="DDQ571C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275792">46</a>). Only SNPs with MAF &gt; 0.01 were considered in this analysis.</p></div><div id="s4h5" class="sec"><p></p><h4 id="s4h5title" class="inline">RT–PCR </h4><p id="__p44" class="p p-first-last">RNA was isolated from frozen human lymphocytes using the RNeasy mini kit (Qiagen). Adult and fetal human brain poly A+ RNA (Clontech Laboratories) was procured. cDNA was prepared using the high-capacity cDNA Archive reagent (Applied Biosystems). Approximately 500 ng of each RNA was converted to cDNA using guidelines provided in the high-capacity cDNA archive reagent protocol (Applied Biosystems) in a 20 μl reaction. Primers were designed using Primer 3 software (<a href="#DDQ571C47" rid="DDQ571C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_373097290">47</a>). To amplify <em>CHL1</em> cDNA (derived from NM_006614.2), the forward primer was designed to span the splice junction of exons 6 and 7, and the reverse primer was designed within exon 7. For the hypothetical cDNA <em>LOC642891</em> (derived from XM_931148.3), the forward primer was designed to span the junction of exons 2 and 3, and the reverse primer was designed within exon 5. This design theoretically avoids amplifying unspliced RNA or genomic DNA. Primer sequences and amplified region are given in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S6</a>. Approximately 100 ng of each cDNA was amplified using 1 U of GoTaq DNA Polymerase (Promega) in a 12.5 μl reaction containing: 0.5 pmoles/μl of forward and reverse primer, 0.2 m<span class="small-caps">m</span> dNTPs, and 1.5 <span class="small-caps">m</span> MgCl<sub>2</sub>. Thirty-five cycles of PCR amplification were carried out at 94°–55°–72°, 30 s each cycle (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S7</a>).</p></div><div id="s4h6" class="sec"><p></p><h4 id="s4h6title" class="inline">Linkage analysis </h4><p id="__p45" class="p p-first-last">Each family selected for analysis was represented in our original discovery cohort and was selected to be informative in a linkage analysis but without regard to genotype at any particular locus. Maximum heterogeneity LOD scores (HLODs) were computed over dominant and recessive models with a phenocopy rate of 0.5 and a disease allele frequency of 0.001 to approximate the frequency of familial AIS. Results were adjusted for inflation by subtracting 0.3 as described in Hodge <em>et al</em>. (<a href="#DDQ571C48" rid="DDQ571C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275806">48</a>). The TDTae was computed to estimate evidence for association in the presence of linkage as previously described (<a href="#DDQ571C40" rid="DDQ571C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275815">40</a>).</p></div><div id="s4h7" class="sec sec-last"><p></p><h4 id="s4h7title" class="inline">Genomic re-sequencing </h4><p id="__p46" class="p p-first-last">Coding exons and splice junctions of <em>CHL1</em> (Gene ID: 10752; Ensembl ID:ENSG00000134121; reference sequence mRNA NM_006614.2) were extracted from available databases. Genomic re-sequencing and analysis was performed as previously described (<a href="#DDQ571C14" rid="DDQ571C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_213275794">14</a>) and by alignment with DNA Baser Sequence Assembler v2 (HeracleSoftware). Primer sequences and PCR conditions are available upon request. The SNP (rs2272522) that predicts a leucine-to-phenylalanine change was included in our original GWAS but did not show evidence of association with AIS in the total data set (OR = 0.92, 95% CI = 0.68–1.25; <em>P</em> = 0.59) nor in trios with family history of AIS (OR = 0.74, 95% CI = 0.42–1.32; <em>P</em> = 0.31). A second variant, SNP rs62230378 that predicts a glycine-to-alanine change, was observed in two unrelated affected individuals from families (91 families) with multiple members affected by AIS. Upon further sequencing, this variant was detected in 1/20 unrelated white, non-Hispanic controls and 1/29 unrelated white, non-Hispanic AIS cases, suggesting near-equal frequencies (3–5%) in cases and controls of this ethnic group.</p></div></div></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p48" class="p p-first-last">This project was funded by NIH <span>grant R01 HD052973</span>, the Crystal Charity Ball, the Scoliosis Research Society, the Cain Foundation and the TSRHC Research Fund (to C.A.W.).</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><!--/article/body/sec/--><p id="__p47" class="p p-first"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq571/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supplementary Data:</strong>
            
            <div class="sup-box half_rhythm" id="idm140469130912896"><a href="/pmc/articles/PMC3049353/bin/supp_20_7_1456__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
            
         </div></div><div id="idm140469130902080" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140469130902080title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p49">We thank the patients and families for their participation in these studies. We also particularly thank referring surgeons and associates: J. Birch, C. E. Johnston, L. A. Karol, K. E. Rathjen, B. S. Richards, D. J. Sucato, T. Milbrandt, V. Talwalkar, H. J. Iwinski, Jr., T. Cloar, C. P. Eberson, D. Adalio, K. Clark, A. Lapinsky, R. Merolli, J. C. Tassone, X. C. Liu, C. Blanchard, W. Schrader, M. Morsher; C. Horton, R. Wilson, and B. Crider in the McDermott Center core facility for help with genotyping and sequencing, B. Zhang in the UTSW microarray core facility for help with genotyping, R. Debaradinis for help with constructing questionnaires, J. Howell for administrative support and A. Bowcock for reviewing the manuscript. Two public data sets were made available through the Database of Genotypes and Phenotypes (dbGaP). The IBDGC Crohn's Disease Genome-Wide Association Study reported here was conducted by IBDGC Crohn's Disease Genome-Wide Association Study Investigators and supported by the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK). This manuscript was not prepared in collaboration with Investigators of the IBDGC Crohn's Disease Genome-Wide Association Study and does not necessarily reflect the opinions or views of the IBDGC Crohn's Disease Genome-Wide Association Study of the NIDDK. The Genome Wide Association Study in Familial Parkinson Disease reported here was conducted as part of PROGENI and GenePD and supported by the National Institute of Neurological Disorders and Stroke (NINDS).</p><p id="__p50"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140469130899008" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140469130899008title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDQ571C1">1. <span class="element-citation">Herring J.A., editor. <span class="ref-journal">Tachdjian's Pediatric Orthopaedics.</span> Vol. 1. Philedelphia: W.B. Saunders Company; 2002.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Tachdjian's+Pediatric+Orthopaedics&amp;publication_year=2002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C2">2. <span class="element-citation">Richards B.S., Vitale M.G. Screening for idiopathic scoliosis in adolescents. An information statement. <span><span class="ref-journal">J. Bone Joint Surg. Am. </span>2008;<span class="ref-vol">90</span>:195–198.</span>  <a href="http://dx.doi.org/10.2106/JBJS.G.01276" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2106/JBJS.G.01276</a>. [<a href="/pubmed/18171974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Joint+Surg.+Am.&amp;title=Screening+for+idiopathic+scoliosis+in+adolescents.+An+information+statement&amp;author=B.S.+Richards&amp;author=M.G.+Vitale&amp;volume=90&amp;publication_year=2008&amp;pages=195-198&amp;pmid=18171974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C3">3. <span class="element-citation">Katz D.E., Herring J.A., Browne R.H., Kelly D.M., Birch J.G. Brace wear control of curve progression in adolescent idiopathic scoliosis. <span><span class="ref-journal">J. Bone Joint Surg. Am. </span>2010;<span class="ref-vol">92</span>:1343–1352.</span>  <a href="http://dx.doi.org/10.2106/JBJS.I.01142" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2106/JBJS.I.01142</a>. [<a href="/pubmed/20516309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Bone+Joint+Surg.+Am.&amp;title=Brace+wear+control+of+curve+progression+in+adolescent+idiopathic+scoliosis&amp;author=D.E.+Katz&amp;author=J.A.+Herring&amp;author=R.H.+Browne&amp;author=D.M.+Kelly&amp;author=J.G.+Birch&amp;volume=92&amp;publication_year=2010&amp;pages=1343-1352&amp;pmid=20516309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C4">4. <span class="element-citation">Wise C.A., Gao X., Shoemaker S., Gordon D., Herring J.A. Understanding genetic factors in idiopathic scoliosis, a complex disease of childhood. <span><span class="ref-journal">Curr. Genomics. </span>2008;<span class="ref-vol">9</span>:51–59.</span>  <a href="http://dx.doi.org/10.2174/138920208783884874" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.2174/138920208783884874</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2674301/">PMC free article</a>]</span> [<a href="/pubmed/19424484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Genomics&amp;title=Understanding+genetic+factors+in+idiopathic+scoliosis,+a+complex+disease+of+childhood&amp;author=C.A.+Wise&amp;author=X.+Gao&amp;author=S.+Shoemaker&amp;author=D.+Gordon&amp;author=J.A.+Herring&amp;volume=9&amp;publication_year=2008&amp;pages=51-59&amp;pmid=19424484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C5">5. <span class="element-citation">Wise C.A., Sharma S.  In: <span class="ref-journal">The Genetics and Development of Scoliosis.</span> Kusumi K., Dunwoodie S.L., editors. Springer; 2009. pp. 167–190. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Genetics+and+Development+of+Scoliosis&amp;author=C.A.+Wise&amp;author=S.+Sharma&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C6">6. <span class="element-citation">Giampietro P.F., Raggio C.L., Blank R.D. Synteny-defined candidate genes for congenital and idiopathic scoliosis. <span><span class="ref-journal">Am. J. Med. Genet. </span>1999;<span class="ref-vol">83</span>:164–177.</span>  <a href="http://dx.doi.org/10.1002/(SICI)1096-8628(19990319)83:3&amp;lt;164::AID-AJMG5&amp;gt;3.0.CO;2-D" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/(SICI)1096-8628(19990319)83:3&amp;lt;164::AID-AJMG5&amp;gt;3.0.CO;2-D</a>. [<a href="/pubmed/10096591" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.&amp;title=Synteny-defined+candidate+genes+for+congenital+and+idiopathic+scoliosis&amp;author=P.F.+Giampietro&amp;author=C.L.+Raggio&amp;author=R.D.+Blank&amp;volume=83&amp;publication_year=1999&amp;pages=164-177&amp;pmid=10096591&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C7">7. <span class="element-citation">Semba K., Araki K., Li Z., Matsumoto K., Suzuki M., Nakagata N., Takagi K., Takeya M., Yoshinobu K., Araki M., et al.  A novel murine gene, Sickle tail, linked to the Danforth's short tail locus, is required for normal development of the intervertebral disc. <span><span class="ref-journal">Genetics. </span>2006;<span class="ref-vol">172</span>:445–456.</span>  <a href="http://dx.doi.org/10.1534/genetics.105.048934" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1534/genetics.105.048934</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1456172/">PMC free article</a>]</span> [<a href="/pubmed/16204209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=A+novel+murine+gene,+Sickle+tail,+linked+to+the+Danforth's+short+tail+locus,+is+required+for+normal+development+of+the+intervertebral+disc&amp;author=K.+Semba&amp;author=K.+Araki&amp;author=Z.+Li&amp;author=K.+Matsumoto&amp;author=M.+Suzuki&amp;volume=172&amp;publication_year=2006&amp;pages=445-456&amp;pmid=16204209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C8">8. <span class="element-citation">Kulkarni S., Nagarajan P., Wall J., Donovan D.J., Donell R.L., Ligon A.H., Venkatachalam S., Quade B.J. Disruption of chromodomain helicase DNA binding protein 2 (CHD2) causes scoliosis. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2008;<span class="ref-vol">146A</span>:1117–1127.</span>  <a href="http://dx.doi.org/10.1002/ajmg.a.32178" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/ajmg.a.32178</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2834558/">PMC free article</a>]</span> [<a href="/pubmed/18386809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=Disruption+of+chromodomain+helicase+DNA+binding+protein+2+(CHD2)+causes+scoliosis&amp;author=S.+Kulkarni&amp;author=P.+Nagarajan&amp;author=J.+Wall&amp;author=D.J.+Donovan&amp;author=R.L.+Donell&amp;volume=146A&amp;publication_year=2008&amp;pages=1117-1127&amp;pmid=18386809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C9">9. <span class="element-citation">Sullivan-Brown J., Schottenfeld J., Okabe N., Hostetter C.L., Serluca F.C., Thiberge S.Y., Burdine R.D. Zebrafish mutations affecting cilia motility share similar cystic phenotypes and suggest a mechanism of cyst formation that differs from pkd2 morphants. <span><span class="ref-journal">Dev. Biol. </span>2008;<span class="ref-vol">314</span>:261–275.</span>  <a href="http://dx.doi.org/10.1016/j.ydbio.2007.11.025" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ydbio.2007.11.025</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2453220/">PMC free article</a>]</span> [<a href="/pubmed/18178183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=Zebrafish+mutations+affecting+cilia+motility+share+similar+cystic+phenotypes+and+suggest+a+mechanism+of+cyst+formation+that+differs+from+pkd2+morphants&amp;author=J.+Sullivan-Brown&amp;author=J.+Schottenfeld&amp;author=N.+Okabe&amp;author=C.L.+Hostetter&amp;author=F.C.+Serluca&amp;volume=314&amp;publication_year=2008&amp;pages=261-275&amp;pmid=18178183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C10">10. <span class="element-citation">Wynne-Davies R. Genetic aspects of idiopathic scoliosis. <span><span class="ref-journal">Dev. Med. Child Neurol. </span>1973;<span class="ref-vol">15</span>:809–811.</span>  <a href="http://dx.doi.org/10.1111/j.1469-8749.1973.tb04919.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1469-8749.1973.tb04919.x</a>. [<a href="/pubmed/4767654" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Med.+Child+Neurol.&amp;title=Genetic+aspects+of+idiopathic+scoliosis&amp;author=R.+Wynne-Davies&amp;volume=15&amp;publication_year=1973&amp;pages=809-811&amp;pmid=4767654&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C11">11. <span class="element-citation">Miller N.H., Justice C.M., Marosy B., Doheny K.F., Pugh E., Zhang J., Dietz H.C., 3rd, Wilson A.F. Identification of candidate regions for familial idiopathic scoliosis. <span><span class="ref-journal">Spine. </span>2005;<span class="ref-vol">30</span>:1181–1187.</span>  <a href="http://dx.doi.org/10.1097/01.brs.0000162282.46160.0a" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1097/01.brs.0000162282.46160.0a</a>. [<a href="/pubmed/15897833" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Spine&amp;title=Identification+of+candidate+regions+for+familial+idiopathic+scoliosis&amp;author=N.H.+Miller&amp;author=C.M.+Justice&amp;author=B.+Marosy&amp;author=K.F.+Doheny&amp;author=E.+Pugh&amp;volume=30&amp;publication_year=2005&amp;pages=1181-1187&amp;pmid=15897833&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C12">12. <span class="element-citation">Ocaka L., Zhao C., Reed J.A., Ebenezer N.D., Brice G., Morley T., Mehta M., O'Dowd J., Weber J.L., Hardcastle A.J., et al.  Assignment of two loci for autosomal dominant adolescent idiopathic scoliosis to chromosomes 9q31.2-q34.2 and 17q25.3-qtel. <span><span class="ref-journal">J. Med. Genet. </span>2008;<span class="ref-vol">45</span>:87–92.</span>  <a href="http://dx.doi.org/10.1136/jmg.2007.051896" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1136/jmg.2007.051896</a>. [<a href="/pubmed/17932119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Assignment+of+two+loci+for+autosomal+dominant+adolescent+idiopathic+scoliosis+to+chromosomes+9q31.2-q34.2+and+17q25.3-qtel&amp;author=L.+Ocaka&amp;author=C.+Zhao&amp;author=J.A.+Reed&amp;author=N.D.+Ebenezer&amp;author=G.+Brice&amp;volume=45&amp;publication_year=2008&amp;pages=87-92&amp;pmid=17932119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C13">13. <span class="element-citation">Doyle C., Blake K. Scoliosis in CHARGE: a prospective survey and two case reports. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2005;<span class="ref-vol">133A</span>:340–343.</span>  <a href="http://dx.doi.org/10.1002/ajmg.a.30564" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1002/ajmg.a.30564</a>. [<a href="/pubmed/15688383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=Scoliosis+in+CHARGE:+a+prospective+survey+and+two+case+reports&amp;author=C.+Doyle&amp;author=K.+Blake&amp;volume=133A&amp;publication_year=2005&amp;pages=340-343&amp;pmid=15688383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C14">14. <span class="element-citation">Gao X., Gordon D., Zhang D., Browne R., Helms C., Gillum J., Weber S., Devroy S., Swaney S., Dobbs M., et al.  CHD7 gene polymorphisms are associated with susceptibility to idiopathic scoliosis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:957–965.</span>  <a href="http://dx.doi.org/10.1086/513571" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/513571</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1852746/">PMC free article</a>]</span> [<a href="/pubmed/17436250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=CHD7+gene+polymorphisms+are+associated+with+susceptibility+to+idiopathic+scoliosis&amp;author=X.+Gao&amp;author=D.+Gordon&amp;author=D.+Zhang&amp;author=R.+Browne&amp;author=C.+Helms&amp;volume=80&amp;publication_year=2007&amp;pages=957-965&amp;pmid=17436250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C15">15. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span>  <a href="http://dx.doi.org/10.1086/519795" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/519795</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C16">16. <span class="element-citation">Spielman R.S., McGinnis R.E., Ewens W.J. The transmission/disequilibrium test detects cosegregation and linkage. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1994;<span class="ref-vol">54</span>:559–560.</span> author reply 560-553) <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1918137/">PMC free article</a>]</span> [<a href="/pubmed/8116627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=The+transmission/disequilibrium+test+detects+cosegregation+and+linkage&amp;author=R.S.+Spielman&amp;author=R.E.+McGinnis&amp;author=W.J.+Ewens&amp;volume=54&amp;publication_year=1994&amp;pages=559-560&amp;pmid=8116627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C17">17. <span class="element-citation">Chen C., Yang G., Buyske S., Matise T., Finch S.J., Gordon D. Transmission disequilibrium test power and sample size in the presence of locus heterogeneity. <span><span class="ref-journal">Stat. Appl. Genet. Mol. Biol. </span>2009;<span class="ref-vol">8</span></span> Article 44. [<a href="/pubmed/19883370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Appl.+Genet.+Mol.+Biol.&amp;title=Transmission+disequilibrium+test+power+and+sample+size+in+the+presence+of+locus+heterogeneity&amp;author=C.+Chen&amp;author=G.+Yang&amp;author=S.+Buyske&amp;author=T.+Matise&amp;author=S.J.+Finch&amp;volume=8&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C18">18. <span class="element-citation">Holm J., Hillenbrand R., Steuber V., Bartsch U., Moos M., Lubbert H., Montag D., Schachner M. Structural features of a close homologue of L1 (CHL1) in the mouse: a new member of the L1 family of neural recognition molecules. <span><span class="ref-journal">Eur. J. Neurosci. </span>1996;<span class="ref-vol">8</span>:1613–1629.</span>  <a href="http://dx.doi.org/10.1111/j.1460-9568.1996.tb01306.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1460-9568.1996.tb01306.x</a>. [<a href="/pubmed/8921253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Neurosci.&amp;title=Structural+features+of+a+close+homologue+of+L1+(CHL1)+in+the+mouse:+a+new+member+of+the+L1+family+of+neural+recognition+molecules&amp;author=J.+Holm&amp;author=R.+Hillenbrand&amp;author=V.+Steuber&amp;author=U.+Bartsch&amp;author=M.+Moos&amp;volume=8&amp;publication_year=1996&amp;pages=1613-1629&amp;pmid=8921253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C19">19. <span class="element-citation">Wright A.G., Demyanenko G.P., Powell A., Schachner M., Enriquez-Barreto L., Tran T.S., Polleux F., Maness P.F. Close homolog of L1 and neuropilin 1 mediate guidance of thalamocortical axons at the ventral telencephalon. <span><span class="ref-journal">J. Neurosci. </span>2007;<span class="ref-vol">27</span>:13667–13679.</span>  <a href="http://dx.doi.org/10.1523/JNEUROSCI.2888-07.2007" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1523/JNEUROSCI.2888-07.2007</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6673613/">PMC free article</a>]</span> [<a href="/pubmed/18077678" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Close+homolog+of+L1+and+neuropilin+1+mediate+guidance+of+thalamocortical+axons+at+the+ventral+telencephalon&amp;author=A.G.+Wright&amp;author=G.P.+Demyanenko&amp;author=A.+Powell&amp;author=M.+Schachner&amp;author=L.+Enriquez-Barreto&amp;volume=27&amp;publication_year=2007&amp;pages=13667-13679&amp;pmid=18077678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C20">20. <span class="element-citation">Jen J.C., Chan W.M., Bosley T.M., Wan J., Carr J.R., Rub U., Shattuck D., Salamon G., Kudo L.C., Ou J., et al.  Mutations in a human ROBO gene disrupt hindbrain axon pathway crossing and morphogenesis. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">304</span>:1509–1513.</span>  <a href="http://dx.doi.org/10.1126/science.1096437" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1126/science.1096437</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1618874/">PMC free article</a>]</span> [<a href="/pubmed/15105459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutations+in+a+human+ROBO+gene+disrupt+hindbrain+axon+pathway+crossing+and+morphogenesis&amp;author=J.C.+Jen&amp;author=W.M.+Chan&amp;author=T.M.+Bosley&amp;author=J.+Wan&amp;author=J.R.+Carr&amp;volume=304&amp;publication_year=2004&amp;pages=1509-1513&amp;pmid=15105459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C21">21. <span class="element-citation">Sicotte N.L., Salamon G., Shattuck D.W., Hageman N., Rub U., Salamon N., Drain A.E., Demer J.L., Engle E.C., Alger J.R., et al.  Diffusion tensor MRI shows abnormal brainstem crossing fibers associated with ROBO3 mutations. <span><span class="ref-journal">Neurology. </span>2006;<span class="ref-vol">67</span>:519–521.</span>  <a href="http://dx.doi.org/10.1212/01.wnl.0000227960.38262.0c" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1212/01.wnl.0000227960.38262.0c</a>. [<a href="/pubmed/16894121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Diffusion+tensor+MRI+shows+abnormal+brainstem+crossing+fibers+associated+with+ROBO3+mutations&amp;author=N.L.+Sicotte&amp;author=G.+Salamon&amp;author=D.W.+Shattuck&amp;author=N.+Hageman&amp;author=U.+Rub&amp;volume=67&amp;publication_year=2006&amp;pages=519-521&amp;pmid=16894121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C22">22. <span class="element-citation">Goldstein D.B. Common genetic variation and human traits. <span><span class="ref-journal">N. Engl. J. Med. </span>2009;<span class="ref-vol">360</span>:1696–1698.</span>  <a href="http://dx.doi.org/10.1056/NEJMp0806284" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1056/NEJMp0806284</a>. [<a href="/pubmed/19369660" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Common+genetic+variation+and+human+traits&amp;author=D.B.+Goldstein&amp;volume=360&amp;publication_year=2009&amp;pages=1696-1698&amp;pmid=19369660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C23">23. <span class="element-citation">Sakurai K., Migita O., Toru M., Arinami T. An association between a missense polymorphism in the close homologue of L1 (CHL1, CALL) gene and schizophrenia. <span><span class="ref-journal">Mol. Psychiatry. </span>2002;<span class="ref-vol">7</span>:412–415.</span>  <a href="http://dx.doi.org/10.1038/sj.mp.4000973" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/sj.mp.4000973</a>. [<a href="/pubmed/11986985" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=An+association+between+a+missense+polymorphism+in+the+close+homologue+of+L1+(CHL1,+CALL)+gene+and+schizophrenia&amp;author=K.+Sakurai&amp;author=O.+Migita&amp;author=M.+Toru&amp;author=T.+Arinami&amp;volume=7&amp;publication_year=2002&amp;pages=412-415&amp;pmid=11986985&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C24">24. <span class="element-citation">Chen Q.Y., Chen Q., Feng G.Y., Lindpaintner K., Chen Y., Sun X., Chen Z., Gao Z., Tang J., He L. Case-control association study of the close homologue of L1 (CHL1) gene and schizophrenia in the Chinese population. <span><span class="ref-journal">Schizophr. Res. </span>2005;<span class="ref-vol">73</span>:269–274.</span>  <a href="http://dx.doi.org/10.1016/j.schres.2004.06.001" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.schres.2004.06.001</a>. [<a href="/pubmed/15653271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schizophr.+Res.&amp;title=Case-control+association+study+of+the+close+homologue+of+L1+(CHL1)+gene+and+schizophrenia+in+the+Chinese+population&amp;author=Q.Y.+Chen&amp;author=Q.+Chen&amp;author=G.Y.+Feng&amp;author=K.+Lindpaintner&amp;author=Y.+Chen&amp;volume=73&amp;publication_year=2005&amp;pages=269-274&amp;pmid=15653271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C25">25. <span class="element-citation">Andrews G.L., Tanglao S., Farmer W.T., Morin S., Brotman S., Berberoglu M.A., Price H., Fernandez G.C., Mastick G.S., Charron F., et al.  Dscam guides embryonic axons by Netrin-dependent and -independent functions. <span><span class="ref-journal">Development. </span>2008;<span class="ref-vol">135</span>:3839–3848.</span>  <a href="http://dx.doi.org/10.1242/dev.023739" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1242/dev.023739</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2712571/">PMC free article</a>]</span> [<a href="/pubmed/18948420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Dscam+guides+embryonic+axons+by+Netrin-dependent+and+-independent+functions&amp;author=G.L.+Andrews&amp;author=S.+Tanglao&amp;author=W.T.+Farmer&amp;author=S.+Morin&amp;author=S.+Brotman&amp;volume=135&amp;publication_year=2008&amp;pages=3839-3848&amp;pmid=18948420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C26">26. <span class="element-citation">Liu G., Li W., Wang L., Kar A., Guan K.L., Rao Y., Wu J.Y. DSCAM functions as a netrin receptor in commissural axon pathfinding. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:2951–2956.</span>  <a href="http://dx.doi.org/10.1073/pnas.0811083106" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1073/pnas.0811083106</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2650370/">PMC free article</a>]</span> [<a href="/pubmed/19196994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=DSCAM+functions+as+a+netrin+receptor+in+commissural+axon+pathfinding&amp;author=G.+Liu&amp;author=W.+Li&amp;author=L.+Wang&amp;author=A.+Kar&amp;author=K.L.+Guan&amp;volume=106&amp;publication_year=2009&amp;pages=2951-2956&amp;pmid=19196994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C27">27. <span class="element-citation">Nelson L.M., Kenneth W. <span>2008</span> Patent WO/2008/033813.</span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C28">28. <span class="element-citation">Schmid R.S., Maness P.F. L1 and NCAM adhesion molecules as signaling coreceptors in neuronal migration and process outgrowth. <span><span class="ref-journal">Curr. Opin. Neurobiol. </span>2008;<span class="ref-vol">18</span>:245–250.</span>  <a href="http://dx.doi.org/10.1016/j.conb.2008.07.015" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.conb.2008.07.015</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2633433/">PMC free article</a>]</span> [<a href="/pubmed/18760361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Neurobiol.&amp;title=L1+and+NCAM+adhesion+molecules+as+signaling+coreceptors+in+neuronal+migration+and+process+outgrowth&amp;author=R.S.+Schmid&amp;author=P.F.+Maness&amp;volume=18&amp;publication_year=2008&amp;pages=245-250&amp;pmid=18760361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C29">29. <span class="element-citation">Banerjee S., Blauth K., Peters K., Rogers S.L., Fanning A.S., Bhat M.A. Drosophila neurexin IV interacts with Roundabout and is required for repulsive midline axon guidance. <span><span class="ref-journal">J. Neurosci. </span>2010;<span class="ref-vol">30</span>:5653–5667.</span>  <a href="http://dx.doi.org/10.1523/JNEUROSCI.6187-09.2010" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1523/JNEUROSCI.6187-09.2010</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2869042/">PMC free article</a>]</span> [<a href="/pubmed/20410118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci.&amp;title=Drosophila+neurexin+IV+interacts+with+Roundabout+and+is+required+for+repulsive+midline+axon+guidance&amp;author=S.+Banerjee&amp;author=K.+Blauth&amp;author=K.+Peters&amp;author=S.L.+Rogers&amp;author=A.S.+Fanning&amp;volume=30&amp;publication_year=2010&amp;pages=5653-5667&amp;pmid=20410118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C30">30. <span class="element-citation">Buratowski S. Transcription. Gene expression—where to start? <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">322</span>:1804–1805.</span>  <a href="http://dx.doi.org/10.1126/science.1168805" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1126/science.1168805</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516868/">PMC free article</a>]</span> [<a href="/pubmed/19095933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Transcription.+Gene+expression—where+to+start?&amp;author=S.+Buratowski&amp;volume=322&amp;publication_year=2008&amp;pages=1804-1805&amp;pmid=19095933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C31">31. <span class="element-citation">Maness P.F., Schachner M. Neural recognition molecules of the immunoglobulin superfamily: signaling transducers of axon guidance and neuronal migration. <span><span class="ref-journal">Nat. Neurosci. </span>2007;<span class="ref-vol">10</span>:19–26.</span>  <a href="http://dx.doi.org/10.1038/nn1827" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nn1827</a>. [<a href="/pubmed/17189949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci.&amp;title=Neural+recognition+molecules+of+the+immunoglobulin+superfamily:+signaling+transducers+of+axon+guidance+and+neuronal+migration&amp;author=P.F.+Maness&amp;author=M.+Schachner&amp;volume=10&amp;publication_year=2007&amp;pages=19-26&amp;pmid=17189949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C32">32. <span class="element-citation">Yimlamai D., Konnikova L., Moss L.G., Jay D.G. The zebrafish down syndrome cell adhesion molecule is involved in cell movement during embryogenesis. <span><span class="ref-journal">Dev. Biol. </span>2005;<span class="ref-vol">279</span>:44–57.</span>  <a href="http://dx.doi.org/10.1016/j.ydbio.2004.12.001" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ydbio.2004.12.001</a>. [<a href="/pubmed/15708557" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=The+zebrafish+down+syndrome+cell+adhesion+molecule+is+involved+in+cell+movement+during+embryogenesis&amp;author=D.+Yimlamai&amp;author=L.+Konnikova&amp;author=L.G.+Moss&amp;author=D.G.+Jay&amp;volume=279&amp;publication_year=2005&amp;pages=44-57&amp;pmid=15708557&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C33">33. <span class="element-citation">Lonstein J.E. Adolescent idiopathic scoliosis. <span><span class="ref-journal">Lancet. </span>1994;<span class="ref-vol">344</span>:1407–1412.</span>  <a href="http://dx.doi.org/10.1016/S0140-6736(94)90572-X" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/S0140-6736(94)90572-X</a>. [<a href="/pubmed/7968079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Adolescent+idiopathic+scoliosis&amp;author=J.E.+Lonstein&amp;volume=344&amp;publication_year=1994&amp;pages=1407-1412&amp;pmid=7968079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C34">34. <span class="element-citation">Wiener-Vacher S.R., Mazda K. Asymmetric otolith vestibulo-ocular responses in children with idiopathic scoliosis. <span><span class="ref-journal">J. Pediatr. </span>1998;<span class="ref-vol">132</span>:1028–1032.</span>  <a href="http://dx.doi.org/10.1016/S0022-3476(98)70403-2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/S0022-3476(98)70403-2</a>. [<a href="/pubmed/9627598" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pediatr.&amp;title=Asymmetric+otolith+vestibulo-ocular+responses+in+children+with+idiopathic+scoliosis&amp;author=S.R.+Wiener-Vacher&amp;author=K.+Mazda&amp;volume=132&amp;publication_year=1998&amp;pages=1028-1032&amp;pmid=9627598&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C35">35. <span class="element-citation">Rousie D., Hache J.C., Pellerin P., Deroubaix J.P., Van Tichelen P., Berthoz A. Oculomotor, postural, and perceptual asymmetries associated with a common cause. Craniofacial asymmetries and asymmetries in vestibular organ anatomy. <span><span class="ref-journal">Ann. NY Acad. Sci. </span>1999;<span class="ref-vol">871</span>:439–446.</span>  <a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb09213.x" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1111/j.1749-6632.1999.tb09213.x</a>. [<a href="/pubmed/10372100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+NY+Acad.+Sci.&amp;title=Oculomotor,+postural,+and+perceptual+asymmetries+associated+with+a+common+cause.+Craniofacial+asymmetries+and+asymmetries+in+vestibular+organ+anatomy&amp;author=D.+Rousie&amp;author=J.C.+Hache&amp;author=P.+Pellerin&amp;author=J.P.+Deroubaix&amp;author=P.+Van+Tichelen&amp;volume=871&amp;publication_year=1999&amp;pages=439-446&amp;pmid=10372100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C36">36. <span class="element-citation">Guo X., Chau W.W., Hui-Chan C.W., Cheung C.S., Tsang W.W., Cheng J.C. Balance control in adolescents with idiopathic scoliosis and disturbed somatosensory function. <span><span class="ref-journal">Spine. </span>2006;<span class="ref-vol">31</span>:E437–E440.</span>  <a href="http://dx.doi.org/10.1097/01.brs.0000222048.47010.bf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1097/01.brs.0000222048.47010.bf</a>. [<a href="/pubmed/16778672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Spine&amp;title=Balance+control+in+adolescents+with+idiopathic+scoliosis+and+disturbed+somatosensory+function&amp;author=X.+Guo&amp;author=W.W.+Chau&amp;author=C.W.+Hui-Chan&amp;author=C.S.+Cheung&amp;author=W.W.+Tsang&amp;volume=31&amp;publication_year=2006&amp;pages=E437-E440&amp;pmid=16778672&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C37">37. <span class="element-citation">Mallau S., Bollini G., Jouve J.L., Assaiante C. Locomotor skills and balance strategies in adolescents idiopathic scoliosis. <span><span class="ref-journal">Spine. </span>2007;<span class="ref-vol">32</span>:E14–E22.</span>  <a href="http://dx.doi.org/10.1097/01.brs.0000251069.58498.eb" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1097/01.brs.0000251069.58498.eb</a>. [<a href="/pubmed/17202875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Spine&amp;title=Locomotor+skills+and+balance+strategies+in+adolescents+idiopathic+scoliosis&amp;author=S.+Mallau&amp;author=G.+Bollini&amp;author=J.L.+Jouve&amp;author=C.+Assaiante&amp;volume=32&amp;publication_year=2007&amp;pages=E14-E22&amp;pmid=17202875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C38">38. <span class="element-citation">Mitchell M.K., Gregersen P.K., Johnson S., Parsons R., Vlahov D. The New York Cancer Project: rationale, organization, design, and baseline characteristics. <span><span class="ref-journal">J. Urban Health. </span>2004;<span class="ref-vol">81</span>:301–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3456454/">PMC free article</a>]</span> [<a href="/pubmed/15136663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Urban+Health&amp;title=The+New+York+Cancer+Project:+rationale,+organization,+design,+and+baseline+characteristics&amp;author=M.K.+Mitchell&amp;author=P.K.+Gregersen&amp;author=S.+Johnson&amp;author=R.+Parsons&amp;author=D.+Vlahov&amp;volume=81&amp;publication_year=2004&amp;pages=301-310&amp;pmid=15136663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C39">39. <span class="element-citation">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span>  <a href="http://dx.doi.org/10.1038/nature06258" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nature06258</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C40">40. <span class="element-citation">Gordon D., Heath S.C., Liu X., Ott J. A transmission/disequilibrium test that allows for genotyping errors in the analysis of single-nucleotide polymorphism data. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2001;<span class="ref-vol">69</span>:371–380.</span>  <a href="http://dx.doi.org/10.1086/321981" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1086/321981</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235309/">PMC free article</a>]</span> [<a href="/pubmed/11443542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+transmission/disequilibrium+test+that+allows+for+genotyping+errors+in+the+analysis+of+single-nucleotide+polymorphism+data&amp;author=D.+Gordon&amp;author=S.C.+Heath&amp;author=X.+Liu&amp;author=J.+Ott&amp;volume=69&amp;publication_year=2001&amp;pages=371-380&amp;pmid=11443542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C41">41. <span class="element-citation">Gordon D., Haynes C., Blumenfeld J., Finch S.J. PAWE-3D: visualizing power for association with error in case-control genetic studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:3935–3937.</span>  <a href="http://dx.doi.org/10.1093/bioinformatics/bti643" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/bioinformatics/bti643</a>. [<a href="/pubmed/16123114" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PAWE-3D:+visualizing+power+for+association+with+error+in+case-control+genetic+studies+of+complex+traits&amp;author=D.+Gordon&amp;author=C.+Haynes&amp;author=J.+Blumenfeld&amp;author=S.J.+Finch&amp;volume=21&amp;publication_year=2005&amp;pages=3935-3937&amp;pmid=16123114&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C42">42. <span class="element-citation">Li Y., Abecasis G. Mach 1.0: rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mach+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C43">43. <span class="element-citation">Seldin M.F., Price A.L. Application of ancestry informative markers to association studies in European Americans. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e5.</span>  <a href="http://dx.doi.org/10.1371/journal.pgen.0040005" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1371/journal.pgen.0040005</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2211545/">PMC free article</a>]</span> [<a href="/pubmed/18208330" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Application+of+ancestry+informative+markers+to+association+studies+in+European+Americans&amp;author=M.F.+Seldin&amp;author=A.L.+Price&amp;volume=4&amp;publication_year=2008&amp;pages=e5&amp;pmid=18208330&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C44">44. <span class="element-citation">Nicodemus K.K. Catmap: case-control and TDT meta-analysis package. <span><span class="ref-journal">BMC Bioinformatics. </span>2008;<span class="ref-vol">9</span>:130.</span>  <a href="http://dx.doi.org/10.1186/1471-2105-9-130" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1186/1471-2105-9-130</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2291045/">PMC free article</a>]</span> [<a href="/pubmed/18307795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=Catmap:+case-control+and+TDT+meta-analysis+package&amp;author=K.K.+Nicodemus&amp;volume=9&amp;publication_year=2008&amp;pages=130&amp;pmid=18307795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C45">45. <span class="element-citation">Ott J.  <span class="ref-journal">Analysis of Human Genetic Linkage.</span> Baltimore: Johns Hopkins; 1999.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Analysis+of+Human+Genetic+Linkage&amp;author=J.+Ott&amp;publication_year=1999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C46">46. <span class="element-citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span>  <a href="http://dx.doi.org/10.1093/bioinformatics/bth457" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/bioinformatics/bth457</a>. [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C47">47. <span class="element-citation">Rozen S., Skaletsky H. Series.  <span class="ref-journal">Primer3 on the WWW for General Users and for Biologist Programmers.</span> Totowa: Humana Press; 2000.  [<a href="/pubmed/10547847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Primer3+on+the+WWW+for+General+Users+and+for+Biologist+Programmers&amp;author=S.+Rozen&amp;author=H.+Series+Skaletsky&amp;publication_year=2000&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C48">48. <span class="element-citation">Hodge S.E., Abreu P.C., Greenberg D.A. Magnitude of type I error when single-locus linkage analysis is maximized over models: a simulation study. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1997;<span class="ref-vol">60</span>:217–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1712558/">PMC free article</a>]</span> [<a href="/pubmed/8981965" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Magnitude+of+type+I+error+when+single-locus+linkage+analysis+is+maximized+over+models:+a+simulation+study&amp;author=S.E.+Hodge&amp;author=P.C.+Abreu&amp;author=D.A.+Greenberg&amp;volume=60&amp;publication_year=1997&amp;pages=217-227&amp;pmid=8981965&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ571C49">49. <span class="element-citation">Ge D., Zhang K., Need A.C., Martin O., Fellay J., Urban T.J., Telenti A., Goldstein D.B. WGAViewer: software for genomic annotation of whole genome association studies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:640–643.</span>  <a href="http://dx.doi.org/10.1101/gr.071571.107" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1101/gr.071571.107</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=WGAViewer:+software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D.+Ge&amp;author=K.+Zhang&amp;author=A.C.+Need&amp;author=O.+Martin&amp;author=J.+Fellay&amp;volume=18&amp;publication_year=2008&amp;pages=640-643&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3049353&amp;issue-id=194779&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140469162817120title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#idm140469130902080title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140469130899008title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3049353/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3049353/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3049353/pdf/ddq571.pdf">PDF (342K)</a></li> | <li><a href="#" data-citationid="PMC3049353" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3049353%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3049353%2F&amp;text=Genome-wide%20association%20studies%20of%20adolescent%20idiopathic%20scoliosis%20suggest%20candidate%20susceptibility%20genes" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3049353%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21216876" data-aiid="3049353" data-aid="3049353" data-iid="194779" data-domainid="740" data-domain="hmg" data-accid="PMC3049353" data-md5="e8f9bd72b411467431a94b6236e92c75"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:33:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE887915DD5153910000000009A40643&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3049353%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3049353/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>