<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="cts"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3521536/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>High Density GWAS for LDL Cholesterol in African Americans Using Electronic Medical Records Reveals a Strong Protective Variant in APOE
</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3521536/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Clinical and Translational Science"><meta name="citation_title" content="High Density GWAS for LDL Cholesterol in African Americans Using Electronic Medical Records Reveals a Strong Protective Variant in APOE
"><meta name="citation_authors" content="Laura J. Rasmussen‐Torvik, Jennifer A. Pacheco, Russell A. Wilke, William K. Thompson, Marylyn D. Ritchie, Abel N. Kho, Arun Muthalagu, M. Geoff Hayes, Loren L. Armstrong, Douglas A. Scheftner, John T. Wilkins, Rebecca L. Zuvich, David Crosslin, Dan M. Roden, Joshua C. Denny, Gail P. Jarvik, Christopher S. Carlson, Iftikhar J. Kullo, Suzette J. Bielinski, Catherine A. McCarty, Rongling Li, Teri A. Manolio, Dana C. Crawford, Rex L. Chisholm"><meta name="citation_date" content="October 2012"><meta name="citation_issue" content="5"><meta name="citation_volume" content="5"><meta name="citation_firstpage" content="394"><meta name="citation_doi" content="10.1111/j.1752-8062.2012.00446.x"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3521536/?report=abstract"><meta name="citation_pmid" content="23067351"><meta name="DC.Title" content="High Density GWAS for LDL Cholesterol in African Americans Using Electronic Medical Records Reveals a Strong Protective Variant in APOE
"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Wiley-Blackwell"><meta name="DC.Contributor" content="Laura J. Rasmussen‐Torvik"><meta name="DC.Contributor" content="Jennifer A. Pacheco"><meta name="DC.Contributor" content="Russell A. Wilke"><meta name="DC.Contributor" content="William K. Thompson"><meta name="DC.Contributor" content="Marylyn D. Ritchie"><meta name="DC.Contributor" content="Abel N. Kho"><meta name="DC.Contributor" content="Arun Muthalagu"><meta name="DC.Contributor" content="M. Geoff Hayes"><meta name="DC.Contributor" content="Loren L. Armstrong"><meta name="DC.Contributor" content="Douglas A. Scheftner"><meta name="DC.Contributor" content="John T. Wilkins"><meta name="DC.Contributor" content="Rebecca L. Zuvich"><meta name="DC.Contributor" content="David Crosslin"><meta name="DC.Contributor" content="Dan M. Roden"><meta name="DC.Contributor" content="Joshua C. Denny"><meta name="DC.Contributor" content="Gail P. Jarvik"><meta name="DC.Contributor" content="Christopher S. Carlson"><meta name="DC.Contributor" content="Iftikhar J. Kullo"><meta name="DC.Contributor" content="Suzette J. Bielinski"><meta name="DC.Contributor" content="Catherine A. McCarty"><meta name="DC.Contributor" content="Rongling Li"><meta name="DC.Contributor" content="Teri A. Manolio"><meta name="DC.Contributor" content="Dana C. Crawford"><meta name="DC.Contributor" content="Rex L. Chisholm"><meta name="DC.Date" content="2012 Oct"><meta name="DC.Identifier" content="10.1111/j.1752-8062.2012.00446.x"><meta name="DC.Language" content="en"><meta property="og:title" content="High Density GWAS for LDL Cholesterol in African Americans Using Electronic Medical Records Reveals a Strong Protective Variant in APOE
"><meta property="og:type" content="article"><meta property="og:description" content="Only one low‐density lipoprotein cholesterol (LDL‐C) genome‐wide association study (GWAS) has been previously reported in ‐African Americans. We performed a GWAS of LDL‐C in African Americans using data extracted ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3521536/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-cts.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3521536/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3521536/pdf/CTS-5-394.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8CC9CFDD54ACC100000000048102E7.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521536%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/2996/">Clin Transl Sci</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/292839/">v.5(5); 2012 Oct</a></li><li class="accid">PMC3521536</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-cts.gif" alt="Logo of cts" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://onlinelibrary.wiley.com/journal/10.1111/(ISSN)1752-8062" target="pmc_ext" ref="reftype=publisher&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Clin Transl Sci</a></span>. 2012 Oct; 5(5): 394–399. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Aug 23. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1111%2Fj.1752-8062.2012.00446.x" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1111/j.1752-8062.2012.00446.x</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3521536</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span><a class="int-reflink" href="/pmc/articles/mid/NIHMS424614/">NIHMS424614</a></span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23067351">23067351</a></div></div></div></div><h1 class="content-title">High Density GWAS for LDL Cholesterol in African Americans Using Electronic Medical Records Reveals a Strong Protective Variant in <em>APOE</em>
</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rasmussen%26%23x02010%3BTorvik%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr1" class="affpopup" co-rid="_co_idm139911992169888" co-class="co-affbox">Laura J. Rasmussen‐Torvik</a>, Ph.D., M.P.H.,<sup>
1
</sup> <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr2" class="affpopup" co-rid="_co_idm139911989325728" co-class="co-affbox">Jennifer A. Pacheco</a>, B.A.,<sup>
2
</sup> <a href="/pubmed/?term=Wilke%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr3" class="affpopup" co-rid="_co_idm139912024333680" co-class="co-affbox">Russell A. Wilke</a>, M.D., Ph.D.,<sup>
3
</sup> <a href="/pubmed/?term=Thompson%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr4" class="affpopup" co-rid="_co_idm139911995062192" co-class="co-affbox">William K. Thompson</a>, Ph.D.,<sup>
2
</sup> <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr5" class="affpopup" co-rid="_co_idm139911987293872" co-class="co-affbox">Marylyn D. Ritchie</a>, Ph.D., M.S.,<sup>
4
</sup> <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr6" class="affpopup" co-rid="_co_idm139911985527408" co-class="co-affbox">Abel N. Kho</a>, M.D., M.S.,<sup>
5
</sup> <a href="/pubmed/?term=Muthalagu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr7" class="affpopup" co-rid="_co_idm139911987714624" co-class="co-affbox">Arun Muthalagu</a>, M.S.,<sup>
2
</sup> <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr8" class="affpopup" co-rid="_co_idm139911993094704" co-class="co-affbox">M. Geoff Hayes</a>, Ph.D.,<sup>
6
</sup> <a href="/pubmed/?term=Armstrong%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr9" class="affpopup" co-rid="_co_idm139911990138896" co-class="co-affbox">Loren L. Armstrong</a>, M.Eng.,<sup>
6
</sup> <a href="/pubmed/?term=Scheftner%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr10" class="affpopup" co-rid="_co_idm139911983735424" co-class="co-affbox">Douglas A. Scheftner</a>, Ph.D.,<sup>
6
</sup> <a href="/pubmed/?term=Wilkins%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr11" class="affpopup" co-rid="_co_idm139912020147040" co-class="co-affbox">John T. Wilkins</a>, M.D., M.S.,<sup>
1
</sup> <a href="/pubmed/?term=Zuvich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr12" class="affpopup" co-rid="_co_idm139911983945264" co-class="co-affbox">Rebecca L. Zuvich</a>, Ph.D., M.S.,<sup>
7
</sup> <a href="/pubmed/?term=Crosslin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr13" class="affpopup" co-rid="_co_idm139912009420592" co-class="co-affbox">David Crosslin</a>, Ph.D.,<sup>
8
</sup> <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr14" class="affpopup" co-rid="_co_idm139912024262080" co-class="co-affbox">Dan M. Roden</a>, M.D.,<sup>
3
,</sup><sup>
9
</sup> <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr15" class="affpopup" co-rid="_co_idm139911982682384" co-class="co-affbox">Joshua C. Denny</a>, M.D., M.S.,<sup>
10
</sup> <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr16" class="affpopup" co-rid="_co_idm139911996508672" co-class="co-affbox">Gail P. Jarvik</a>, M.D., Ph.D.,<sup>
11
</sup> <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr17" class="affpopup" co-rid="_co_idm139911996209360" co-class="co-affbox">Christopher S. Carlson</a>, Ph.D.,<sup>
12
</sup> <a href="/pubmed/?term=Kullo%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr18" class="affpopup" co-rid="_co_idm139911991473440" co-class="co-affbox">Iftikhar J. Kullo</a>, M.D.,<sup>
13
</sup> <a href="/pubmed/?term=Bielinski%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr19" class="affpopup" co-rid="_co_idm139911981416160" co-class="co-affbox">Suzette J. Bielinski</a>, Ph.D., M.P.H.,<sup>
14
</sup> <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr20" class="affpopup" co-rid="_co_idm139911988086784" co-class="co-affbox">Catherine A. McCarty</a>, Ph.D., M.P.H.,<sup>
15
</sup> <a href="/pubmed/?term=Li%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr21" class="affpopup" co-rid="_co_idm139912009252416" co-class="co-affbox">Rongling Li</a>, M.D., Ph.D., M.P.H.,<sup>
16
</sup> <a href="/pubmed/?term=Manolio%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr22" class="affpopup" co-rid="_co_idm139912021278608" co-class="co-affbox">Teri A. Manolio</a>, M.D., Ph.D.,<sup>
16
</sup> <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr23" class="affpopup" co-rid="_co_idm139911980275824" co-class="co-affbox">Dana C. Crawford</a>, Ph.D.,<sup>
7
</sup> and  <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351" id="cr24" class="affpopup" co-rid="_co_idm139912009228768" co-class="co-affbox">Rex L. Chisholm</a>, Ph.D.<sup>
2
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139911992169888"><h3 class="no_margin">Laura J. Rasmussen‐Torvik</h3><p><sup>1</sup>Department of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Rasmussen%26%23x02010%3BTorvik%20LJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Laura J. Rasmussen‐Torvik</a></div></div><div id="_co_idm139911989325728"><h3 class="no_margin">Jennifer A. Pacheco</h3><p><sup>2</sup>Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Jennifer A. Pacheco</a></div></div><div id="_co_idm139912024333680"><h3 class="no_margin">Russell A. Wilke</h3><p><sup>3</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Wilke%20RA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Russell A. Wilke</a></div></div><div id="_co_idm139911995062192"><h3 class="no_margin">William K. Thompson</h3><p><sup>2</sup>Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Thompson%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">William K. Thompson</a></div></div><div id="_co_idm139911987293872"><h3 class="no_margin">Marylyn D. Ritchie</h3><p><sup>4</sup>Department of Biochemistry and Molecular Biology, Penn State University, University Park, Pennsylvania, USA</p><div>Find articles by <a href="/pubmed/?term=Ritchie%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Marylyn D. Ritchie</a></div></div><div id="_co_idm139911985527408"><h3 class="no_margin">Abel N. Kho</h3><p><sup>5</sup>Department of Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Abel N. Kho</a></div></div><div id="_co_idm139911987714624"><h3 class="no_margin">Arun Muthalagu</h3><p><sup>2</sup>Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Muthalagu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Arun Muthalagu</a></div></div><div id="_co_idm139911993094704"><h3 class="no_margin">M. Geoff Hayes</h3><p><sup>6</sup>Division of Endocrinology, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">M. Geoff Hayes</a></div></div><div id="_co_idm139911990138896"><h3 class="no_margin">Loren L. Armstrong</h3><p><sup>6</sup>Division of Endocrinology, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Armstrong%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Loren L. Armstrong</a></div></div><div id="_co_idm139911983735424"><h3 class="no_margin">Douglas A. Scheftner</h3><p><sup>6</sup>Division of Endocrinology, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Scheftner%20DA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Douglas A. Scheftner</a></div></div><div id="_co_idm139912020147040"><h3 class="no_margin">John T. Wilkins</h3><p><sup>1</sup>Department of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Wilkins%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">John T. Wilkins</a></div></div><div id="_co_idm139911983945264"><h3 class="no_margin">Rebecca L. Zuvich</h3><p><sup>7</sup>Center for Human Genetics Research, Department of Molecular Physiology and Biophysics, Vanderbilt University, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Zuvich%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Rebecca L. Zuvich</a></div></div><div id="_co_idm139912009420592"><h3 class="no_margin">David Crosslin</h3><p><sup>8</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Crosslin%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">David Crosslin</a></div></div><div id="_co_idm139912024262080"><h3 class="no_margin">Dan M. Roden</h3><p><sup>3</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, Tennessee, USA</p><p><sup>9</sup>Department of Pharmacology, Vanderbilt University, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Roden%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Dan M. Roden</a></div></div><div id="_co_idm139911982682384"><h3 class="no_margin">Joshua C. Denny</h3><p><sup>10</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Joshua C. Denny</a></div></div><div id="_co_idm139911996508672"><h3 class="no_margin">Gail P. Jarvik</h3><p><sup>11</sup>Departments of Medicine (Medical Genetics) and Genome Sciences, University of Washington, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Gail P. Jarvik</a></div></div><div id="_co_idm139911996209360"><h3 class="no_margin">Christopher S. Carlson</h3><p><sup>12</sup>Fred Hutchinson Cancer Research Center, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Christopher S. Carlson</a></div></div><div id="_co_idm139911991473440"><h3 class="no_margin">Iftikhar J. Kullo</h3><p><sup>13</sup>Division of Cardiovascular Diseases, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Kullo%20IJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Iftikhar J. Kullo</a></div></div><div id="_co_idm139911981416160"><h3 class="no_margin">Suzette J. Bielinski</h3><p><sup>14</sup>Division of Epidemiology, Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Bielinski%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Suzette J. Bielinski</a></div></div><div id="_co_idm139911988086784"><h3 class="no_margin">Catherine A. McCarty</h3><p><sup>15</sup>Essentia Institute of Rural Health, Duluth, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=McCarty%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Catherine A. McCarty</a></div></div><div id="_co_idm139912009252416"><h3 class="no_margin">Rongling Li</h3><p><sup>16</sup>Office of Population Genomics, National Human Genomics Research Institute, Bethesda, Maryland, USA.</p><div>Find articles by <a href="/pubmed/?term=Li%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Rongling Li</a></div></div><div id="_co_idm139912021278608"><h3 class="no_margin">Teri A. Manolio</h3><p><sup>16</sup>Office of Population Genomics, National Human Genomics Research Institute, Bethesda, Maryland, USA.</p><div>Find articles by <a href="/pubmed/?term=Manolio%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Teri A. Manolio</a></div></div><div id="_co_idm139911980275824"><h3 class="no_margin">Dana C. Crawford</h3><p><sup>7</sup>Center for Human Genetics Research, Department of Molecular Physiology and Biophysics, Vanderbilt University, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Crawford%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Dana C. Crawford</a></div></div><div id="_co_idm139912009228768"><h3 class="no_margin">Rex L. Chisholm</h3><p><sup>2</sup>Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23067351">Rex L. Chisholm</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139912023707920_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139912023707920_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139912023707920_ai" style="display:none"><div class="fm-affl" id="a1" lang="en"><sup>1</sup>Department of Preventive Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</div><div class="fm-affl" id="a2" lang="en"><sup>2</sup>Center for Genetic Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</div><div class="fm-affl" id="a3" lang="en"><sup>3</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, Tennessee, USA</div><div class="fm-affl" id="a4" lang="en"><sup>4</sup>Department of Biochemistry and Molecular Biology, Penn State University, University Park, Pennsylvania, USA</div><div class="fm-affl" id="a5" lang="en"><sup>5</sup>Department of Medicine, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</div><div class="fm-affl" id="a6" lang="en"><sup>6</sup>Division of Endocrinology, Northwestern University Feinberg School of Medicine, Chicago, Illinois, USA</div><div class="fm-affl" id="a7" lang="en"><sup>7</sup>Center for Human Genetics Research, Department of Molecular Physiology and Biophysics, Vanderbilt University, Nashville, Tennessee, USA</div><div class="fm-affl" id="a8" lang="en"><sup>8</sup>Department of Biostatistics, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="a9" lang="en"><sup>9</sup>Department of Pharmacology, Vanderbilt University, Nashville, Tennessee, USA</div><div class="fm-affl" id="a10" lang="en"><sup>10</sup>Department of Biomedical Informatics, Vanderbilt University, Nashville, Tennessee, USA</div><div class="fm-affl" id="a11" lang="en"><sup>11</sup>Departments of Medicine (Medical Genetics) and Genome Sciences, University of Washington, Seattle, Washington, USA</div><div class="fm-affl" id="a12" lang="en"><sup>12</sup>Fred Hutchinson Cancer Research Center, Seattle, Washington, USA</div><div class="fm-affl" id="a13" lang="en"><sup>13</sup>Division of Cardiovascular Diseases, Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="a14" lang="en"><sup>14</sup>Division of Epidemiology, Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="a15" lang="en"><sup>15</sup>Essentia Institute of Rural Health, Duluth, Minnesota, USA</div><div class="fm-affl" id="a16" lang="en"><sup>16</sup>Office of Population Genomics, National Human Genomics Research Institute, Bethesda, Maryland, USA.</div><div id="correspondenceTo">Laura Rasmussen‐Torvik (<a href="mailto:dev@null" data-email="ude.nretsewhtron@kivrotrjl" class="oemail">ude.nretsewhtron@kivrotrjl</a>)</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139912023707920_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012 Wiley Periodicals, Inc.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3521536/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><p>
<strong>Figure S1. (A) QQ plot for LDL GWAS in dataset with
exclusions.</strong> Genomic inflation factor for this analysis is equal
to 1.0. <strong>(B) QQ plot for LDL GWAS in dataset with no lab values
excluded.</strong> Genomic inflation factor for this analysis is equal to 1.0.</p><p>
<strong>Figure S2.</strong> LD&amp;sol;association plot of <em>APOE</em> region from LDL GWAS in dataset with exclusions.</p><p>
<strong>Table S1.</strong> Association of LDL‐C with three SNPs identified in a previous African American GWAS of LDL‐C for the eMERGE African American datasets</p><p>
<strong>Table S2.</strong> Details of SNPs from the <em>APOE</em> region previously detected in LDL GWAS</p></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Supporting info item</div><div><a href="/pmc/articles/PMC3521536/bin/CTS-5-394-s001.doc" data-ga-action="click_feat_suppl">CTS-5-394-s001.doc</a><span style="color:gray"> (186K)</span></div><div class="small guid">GUID:&nbsp;AA6D3614-1171-4CF5-9B49-42763242BF8D</div></div></dd></dl></div><div id="idm139911979462240" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139911979462240title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Only one low‐density lipoprotein cholesterol (LDL‐C) genome‐wide association study (GWAS) has been previously reported in ‐African Americans. We performed a GWAS of LDL‐C in African Americans using data extracted from electronic medical records (EMR) in the eMERGE network. African Americans were genotyped on the Illumina 1M chip. All LDL‐C measurements, prescriptions, and diagnoses of concomitant disease were extracted from EMR. We created two analytic datasets; one dataset having median LDL‐C calculated after the exclusion of some lab values based on comorbidities and medication (<em>n</em>= 618) and another dataset having median LDL‐C calculated without any exclusions (<em>n</em>= 1,249). SNP rs7412 in <em>APOE</em> was strongly associated with LDL‐C in both datasets (<em>p</em> &lt; 5 × 10<sup>−8</sup>). In the dataset with exclusions, a decrease of 20.0 mg/dL per minor allele was observed. The effect size was attenuated (12.3 mg/dL) in the dataset without any lab values excluded. Although other signals in <em>APOE</em> have been detected in previous GWAS, this large and important SNP association has not been well detected in large GWAS because rs7412 was not included on many genotyping arrays. Use of median LDL‐C extracted from EMR after exclusions for medications and comorbidities increased the percentage of trait variance explained by genetic variation. Clin Trans Sci 2012; Volume 5: 394–399</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">GWAS, LDL, electronic medical records</span></div></div><div id="ss1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss1title">Introduction</h2><p id="__p2" class="p p-first">Several genome‐wide association studies (GWAS) of low‐density lipoprotein cholesterol (LDL‐C) have been published in the past 3 years; the NHGRI Catalog of published GWAS<sup><a href="#b1" rid="b1" class=" ref">
1
</a></sup> currently lists at least 11 studies under the search term “LDL Cholesterol.” The largest LDL‐C GWAS, by Teslovich et al.,<sup><a href="#b2" rid="b2" class=" ref">
2
</a></sup> presents a GWAS meta‐analysis of 46 lipid GWAS including 95,454 individuals with LDL‐C measurements. This meta‐analysis detected 37 loci significantly (<em>p</em> &lt; 5 × 10<sup>−8</sup>) associated with LDL‐C which, combined together, accounted for 12.2% of LDL‐C trait variance in the Framingham Heart Study.<sup><a href="#b2" rid="b2" class=" ref">
2
</a></sup> However, the Teslovich meta‐analysis included primarily only individuals of European ancestry. Although Teslovich and others have examined SNPs detected in European‐ancestry GWAS in other ethnic groups including African Americans, only one LDL‐C GWAS study in African American adults has been published. In this analysis of 8,090 African Americans from the CARe consortium, eight signals from previous European‐ancestry LDL‐C GWAS were replicated and one novel LDL‐C association, for SNP rs13161895 on chromosome five, was reported.<sup><a href="#b3" rid="b3" class=" ref">
3
</a></sup>
</p><p id="__p3" class="p p-last">The eMERGE network has been established to link phenotypes from electronic medical records (EMR) with genetic information in biorepositories at the participating institutions. In the eMERGE network, we had the opportunity to conduct a GWAS of LDL‐C in African Americans using phenotypes extracted from EMR and a high‐density GWAS chip. We hypothesized that, despite a modest sample size, we might detect novel LDL‐C GWAS associations in our sample because of (a) our use of the Illumina 1M genotyping array, an array not used in the CARe consortium<sup><a href="#b3" rid="b3" class=" ref">3</a></sup> or in the participating studies of the Teslovich GWAS meta‐analysis<sup><a href="#b2" rid="b2" class=" ref">2</a></sup> and (b) our use of EMR‐derived phenotypes which permitted use of median LDL‐C and careful exclusion for medications and diseases which may alter LDL‐C levels. It has previously been demonstrated in eMERGE that traits constructed from longitudinal lipid data (right censored based on relevant comorbidity and medication history) can be leveraged to detect genotype–phenotype associations not previously recognized (such as <em>TRIB1</em> in an earlier HDL GWAS<sup><a href="#b4" rid="b4" class=" ref">4</a></sup>).</p></div><div id="ss2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss2title">Methods</h2><div id="ss2-1" class="sec sec-first"><h3 id="ss2-1title">The eMERGE network</h3><p id="__p4" class="p p-first-last">The eMERGE Network is a consortium of five US institutions (Northwestern University, Marshfield Clinic, Mayo Clinic, Vanderbilt University, and the Group Health Cooperative, University of Washington, and Fred Hutchinson Cancer Research Partnership) with DNA biorepositories linked to secure encrypted EMR data. The network is described in detail elsewhere.<sup><a href="#b5" rid="b5" class=" ref">5</a></sup> Institutional review boards at all participating institutions approved the project. As part of the consortium, each site conducted a GWAS on a specific phenotype derived from their EMR records. Members of the consortium then combined all genotyped samples across the network into one merged dataset<sup><a href="#b6" rid="b6" class=" ref">6</a></sup> and conducted further GWAS analyses<sup><a href="#b7" rid="b7" class=" ref">7</a></sup> by working across the Network to extract and harmonize phenotypes from all five sites’ databases. Here, we present analyses using a subset of African American individuals in eMERGE who were genotyped on the Illumina 1M‐Duo (1M) (Illumina, San Diego, CA, USA) genotyping platform, a chip which includes many SNPs targeting variation in individuals of recent African descent. These individuals were originally included in a case‐control study of diabetes that was a collaboration between Northwestern University and Vanderbilt<sup><a href="#b8" rid="b8" class=" ref">8</a></sup> and a study of QRS interval from normal electrocardiograms led by Vanderbilt University.<sup><a href="#b9" rid="b9" class=" ref">9</a></sup>
</p></div><div id="ss2-2" class="sec"><h3 id="ss2-2title">Genotyping and quality control</h3><p id="__p5" class="p p-first-last">DNA samples from subjects who were self‐reported (Northwestern) or observer‐reported (Vanderbilt University) to be African American were genotyped on the Illumina 1M platform. Genotyping was performed at the Broad Institute of MIT and Harvard. Genotyping calls made using BeadStudio version 3.3.7 (Illumina) and Gentrain version 1.0 (Illumina). SNP data were cleaned using the eMERGE quality control (QC) pipeline developed by the eMERGE genomics working group.<sup><a href="#b10" rid="b10" class=" ref">10</a></sup> The cleaning process included evaluation of sample and marker call rate, gender anomalies, duplicate and Hap Map concordance, batch effects, evaluation of Hardy–Weinberg equilibrium, Mendelian errors, sample relatedness, and population stratification. Cryptic relatedness was assessed for all pairs within and across sites. Any pairs estimated to be at the half‐sibling level or higher were resolved by dropping the member of the pair with the lower genotype call rate. Additional cleaning was completed when samples from Northwestern and Vanderbilt were merged.<sup><a href="#b6" rid="b6" class=" ref">6</a></sup> The 1M chip contains 1,199,887 SNPs. As part of initial cleaning, 44,496 intensity‐only SNPs were removed. After completing the QC pipeline and removing all SNPs with a minor allele frequency less than 0.05, 910,341 SNPs remained in the analytic sample. Principal components to attempt to account for ancestry and to avoid population stratification were estimated for the all eMERGE participants typed on the 1M chip using Eigenstrat (Eigensoft v3.0, Smartpca 8000) at Northwestern University.<sup><a href="#b11" rid="b11" class=" ref">11</a></sup>, <sup><a href="#b12" rid="b12" class=" ref">12</a></sup>
</p></div><div id="ss2-3" class="sec"><h3 id="ss2-3title">Phenotype extraction</h3><p id="__p6" class="p p-first">eMERGE investigators have worked collaboratively to develop algorithms for extracting phenotypes from EMRs for use in genetic research.<sup><a href="#b5" rid="b5" class=" ref">5</a></sup>, <sup><a href="#b13" rid="b13" class=" ref">13</a></sup>, <sup><a href="#b14" rid="b14" class=" ref">14</a></sup> The algorithm for LDL‐C values and exclusion dates was a modification of an earlier HDL‐C eMERGE algorithm.<sup><a href="#b4" rid="b4" class=" ref">4</a></sup> All LDL‐C measurements were extracted from longitudinal lipid panel data obtained during the course of clinical care. Dates of prescriptions for medications that may have altered LDL‐C level (antilipid medications, estrogen hormone replacement therapy, and exogenous androgens) were also extracted. Date of earliest cancer diagnosis was also extracted from the EMR, as was date of first diagnosis of diabetes, hyperthyroidism, or hypothyroidism, using diabetes<sup><a href="#b8" rid="b8" class=" ref">8</a></sup> and hyper/hypothyroidism<sup><a href="#b15" rid="b15" class=" ref">15</a></sup> algorithms developed for other eMERGE projects. The complete algorithm for extracting this information from the EMR is available at <a href="https://www.mc.vanderbilt.edu/victr/dcc/projects/acc/index.php/Library_of_Phenotype_Algorithms#Lipids" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.mc.vanderbilt.edu/victr/dcc/projects/acc/index.php/Library_of_Phenotype_Algorithms#Lipids</a>. Once all LDL‐C values, measurement dates, and exclusion dates were extracted from the EMR, the data were transferred to Northwestern University for analysis.</p><p id="__p7">For this project, two datasets were created from the sample of eMERGE African Americans for analysis. For the first dataset, the median LDL‐C for each individual was calculated without the removal of any LDL‐C values from the medical record (called the <em>dataset without lab values excluded</em>). For the second dataset, LDL‐C values that were recorded for an individual after the earliest known exclusion date (date of drug prescription, diabetes, hyperthyroidism, hypothyroidism, or cancer diagnosis) were removed from the dataset and then the median LDL‐C calculated from remaining LDL‐C values (we refer to this dataset as the <em>dataset with exclusions</em>). These two datasets included different numbers of individuals as some individuals had all LDL‐C values removed before the calculation of the median in the second dataset. It is important to note that individuals included in both datasets did not necessarily have the same median LDL‐C value in the two dataset. A diagram depicting the algorithm for extracting LDL‐C lab values and exclusion dates from the EMR and the creation of the two datasets is presented in <a href="/pmc/articles/PMC3521536/figure/f1/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1" rid-ob="ob-f1" co-legend-rid="lgnd_f1"><span>
<em>Figure 1</em>
</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f1" co-legend-rid="lgnd_f1"><a href="/pmc/articles/PMC3521536/figure/f1/" target="figure" rid-figpopup="f1" rid-ob="ob-f1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139911989894752" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is CTS-5-394-g001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is CTS-5-394-g001.jpg" src="/pmc/articles/PMC3521536/bin/CTS-5-394-g001.jpg"></div></a><div id="largeobj_idm139911989894752" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3521536/figure/f1/" target="figure" rid-figpopup="f1" rid-ob="ob-f1"></a><a target="object" href="/pmc/articles/PMC3521536/figure/f1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f1"><div><a class="figpopup" href="/pmc/articles/PMC3521536/figure/f1/" target="figure" rid-figpopup="f1" rid-ob="ob-f1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p8">EMR extraction of LDL‐C values and exclusion dates and creation of the analytic datasets. Rx = prescription, Dx = prevalent disease. This flowchart depicts the information extracted from the EMR, and the phenotypes and analytic dataset created from the extracted EMR information. The start of the process is depicted at the top of the figure. Diamonds represent questions asked to extract information or restrict the analytic dataset, while rectangles represent data collected. The top third of the figure depicts how the dataset with no lab values excluded was assembled, while the bottom two thirds show the additional steps required to assemble the more restrictive dataset after exclusions. The figure demonstrates that median LDL‐C was calculated at two separate points for the two dataset; for this reason, an individual present in both datasets may have different LDL‐C values in each dataset.</p></div></div></div><p id="__p9">Individuals with only one measurement of LDL‐C were included in the datasets; this measurement was considered the median LDL‐C. For individuals with an even number of LDL‐C measurements, the median LDL‐C was chosen as whichever of the two central measurements was closest to the mean of LDL‐C measurements (so as to have an actual LDL measurement and associated age of measurement for analysis). In an effort to minimize inclusion of errant LDL‐C values due to lab or data entry errors, unusually low LDL‐C levels due to unrecognized concomitant medical illness, or unusually high LDL‐C due to unrecognized familial hyperlipidemia, the following <em>a priori</em> exclusion criteria for median LDL‐C were used: any record from either dataset with a median measurement less than 50 mg/dL and greater than 210 mg/dL was excluded. This exclusion removed approximately 3.5% of individuals from the first dataset, over 75% of whom were individuals with only a single measure of LDL‐C.</p><p id="__p10" class="p p-last">Once median LDL‐C values were determined, the date of measurement for this LDL‐C value was extracted from the EMR. For individuals who had identical LDL‐C measurements on different dates, the earliest date on which the value was measured was extracted (and associated age used in analysis). A similar process to that used for the extraction of LDL values was also used to extract total cholesterol values.</p></div><div id="ss2-4" class="sec sec-last"><h3 id="ss2-4title">Statistical methods</h3><p id="__p11" class="p p-first-last">Univariate statistical analyses were performed in SAS version 9.2 (Cary, NC, USA). GWAS analyses were performed in PLINK.<sup><a href="#b16" rid="b16" class=" ref">16</a></sup> Genotypes were modeled additively. GWAS analyses were controlled for age, age<sup><a href="#b2" rid="b2" class=" ref">2</a></sup>, sex, eMERGE recruitment site (Northwestern or Vanderbilt), genotyping batch (type 2 diabetes project or QRS project), and first three principal components to attempt to adjust for ancestry. A genomic inflation factor of 1 was calculated for the LDL‐C GWAS in both datasets, so genomic control was not applied. Estimates of <em>r</em>
<sup>2</sup> between SNPs were calculated using the Hap Map website.</p></div></div><div id="ss3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss3title">Results</h2><p id="__p12" class="p p-first">
<a href="/pmc/articles/PMC3521536/table/t1/" target="table" class="fig-table-link figpopup" rid-figpopup="t1" rid-ob="ob-t1" co-legend-rid=""><span>
<em>Table 1</em>
</span></a> lists the characteristics of the eMERGE African Americans included in each dataset for the two LDL‐C GWAS. On average, the participants in the GWAS with exclusions (<em>n</em>= 618) were younger and had a higher median LDL‐C than participants with the GWAS without any lab values excluded (<em>n</em>= 1,249). For both datasets, nearly 70% of participants were female and approximately 30% were recruited at Northwestern.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p13">Characteristics of the study sample.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139911983451616" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead valign="bottom"><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Dataset with exclusions (n = 618)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Dataset with no lab values excluded (n = 1,249)</th></tr></thead><tbody valign="top"><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years)*</td><td rowspan="1" colspan="1" valign="top" align="left">44.2 (13.1)</td><td rowspan="1" colspan="1" valign="top" align="left">50.1 (14.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age range (years)</td><td rowspan="1" colspan="1" valign="top" align="left">14–87</td><td rowspan="1" colspan="1" valign="top" align="left">14–90</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Median LDL‐C (mg/dL)*</td><td rowspan="1" colspan="1" valign="top" align="left">112.2 (31.5)</td><td rowspan="1" colspan="1" valign="top" align="left">103.7 (29.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Percentage of participants recruited from Northwestern</td><td rowspan="1" colspan="1" valign="top" align="left">32.4%</td><td rowspan="1" colspan="1" valign="top" align="left">26.6%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Percentage of female participants</td><td rowspan="1" colspan="1" valign="top" align="left">69.1%</td><td rowspan="1" colspan="1" valign="top" align="left">69.0%</td></tr></tbody></table></div><div id="largeobj_idm139911983451616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3521536/table/t1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="t1_note18"><p id="__p14" class="p p-first-last">*Mean (SD).</p></div></div></div><p id="__p15">
<a href="/pmc/articles/PMC3521536/figure/f2/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2" rid-ob="ob-f2" co-legend-rid="lgnd_f2"><span>
<em>Figures 2A</em> and <em>B</em>
</span></a> show Manhattan plots for the GWAS of LDL‐C in the eMERGE network with exclusions (A) and without any lab values excluded (B). QQ plots for both GWAS analyses are presented in <em>Figures S1</em>(A) and (B). In each dataset, only one association, of SNP rs7412 in <em>APOE</em> and median LDL‐C, reached genome‐wide significance (<em>p</em> &lt; 5 × 10 − 8). In the dataset with exclusions, the effect size (β) per minor allele of rs7412 was –20.0 mg/dL (95% CI =–25.9 mg/dL to –14.1 mg/dL, <em>p</em> value for association = 6.3 × 10 − 11). The estimated percentage trait variance (<em>r</em>
<sup>2</sup>) accounted for by this SNP in this dataset was 6.8%. In the dataset without any lab values excluded, the effect size (β) per minor allele of rs7412 was –12.3 mg/dL (95% CI =–16.3 mg/dL to –8.4 mg/dL, <em>p</em> value for association = 1.5 × 10 − 9). The estimated percentage trait variance (<em>r</em>
<sup>2</sup>) accounted for by this SNP in this dataset was 2.9%. The minor allele frequency of rs7412 in both datasets was 0.10. Forcing additional SNPs in the <em>APOE</em> region into the model as covariates did not substantially attenuate the effect size for the association of rs7412 with LDL‐C. A regional association plot for the LDL‐C analysis in the dataset without exclusions which includes information about linkage disequilibrium (LD) between rs7412 and nearby SNPs, is presented in <em>Figure S2</em>. SNP rs7412 was also strongly associated with total cholesterol levels in the dataset with exclusions (the effect size [β] per minor allele of rs7412 was –16.7 mg/dL, p value for association = 9.6 × 10 − 7).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f2" co-legend-rid="lgnd_f2"><a href="/pmc/articles/PMC3521536/figure/f2/" target="figure" rid-figpopup="f2" rid-ob="ob-f2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139912009333744" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is CTS-5-394-g002.jpg" title="An external file that holds a picture, illustration, etc.
Object name is CTS-5-394-g002.jpg" src="/pmc/articles/PMC3521536/bin/CTS-5-394-g002.jpg"></div></a><div id="largeobj_idm139912009333744" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC3521536/figure/f2/" target="figure" rid-figpopup="f2" rid-ob="ob-f2"></a><a target="object" href="/pmc/articles/PMC3521536/figure/f2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f2"><div><a class="figpopup" href="/pmc/articles/PMC3521536/figure/f2/" target="figure" rid-figpopup="f2" rid-ob="ob-f2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p16">(A) GWAS of LDL‐C in eMERGE African Americans with exclusions. (B) GWAS of LDL‐C in eMERGE African Americans without any lab values excluded. Manhattan plots display the <em>p</em> values for the association of approximately 1 million SNPs with LDL‐C; each circle represents a single SNP association plotted by chromosome position (<em>x</em> axis) and log(10) transformed p value (y axis). The p values are from regression equations with SNPs modeled additively, adjusted for age<sup><a href="#b2" rid="b2" class=" ref">2</a></sup>, sex, eMERGE site, genotyping batch, and the first three principal components. In both plots, only the association of LDL‐C and SNP rs7412 in <em>APOE</em> exceeded the genome‐wide threshold for significance.</p></div></div></div><p id="__p17" class="p p-last">
<em>Table S1</em> shows SNP/LDL‐C associations for three SNPs identified in a previous African American GWAS of LDL‐C.3 Given our small sample size, we were not sufficiently powered to replicate these associations, but the beta estimates for the eMERGE dataset with exclusions were consistent with (within 1 standard error of) the Lettre et al.<sup><a href="#b3" rid="b3" class=" ref">3</a></sup> estimates supporting the validity of LDL‐C measures in this dataset.</p></div><div id="ss4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss4title">Discussion</h2><p id="__p18" class="p p-first">In this GWAS of LDL‐C in African Americans using a high‐density GWAS chip and EMR, we detected a single strong and GWAS‐significant association with SNP rs7412 in <em>APOE</em>. This association was genome‐wide significant whether or not lab values were excluded using information on medications and concomitant illness from the EMR. However, the effect size of the association and the percentage of trait variance explained were larger in the analysis with exclusions.</p><p id="__p19">
<em>APOE</em> circulates on very low‐density lipoprotein and is a ligand for LDL receptors, which participate in the removal of LDL from circulation. The E2 allele (a haplotype characterized primarily by rs7214) is known to be associated with lower LDL‐C levels<sup><a href="#b17" rid="b17" class=" ref">17</a></sup> as well as with type III hyperlipoproteinemia.<sup><a href="#b18" rid="b18" class=" ref">18</a></sup> rs7412 is a coding SNP which changes the amino acid at position 158 in <em>APOE</em> from Arg to Cys. Individuals with this mutation are less efficient at making and transferring VLDLs and chylomicrons from the blood plasma to the liver. The E4 allele (another <em>APOE</em> haplotype, this one characterized primarily by rs429358) has been associated with increased LDL‐C levels and increased atherosclerosis.<sup><a href="#b19" rid="b19" class=" ref">19</a></sup> (A recent fine‐mapping study of several loci associated with LDL‐C in GWAS verified that SNPs rs7412 and rs429358 are independently associated with the phenotype.<sup><a href="#b20" rid="b20" class=" ref">20</a></sup>) Many candidate gene studies, including some in non‐Caucasian populations, have examined the association of rs7412 (alone or in concert with rs429358) with LDL and have detected strong associations. In a 2007 meta‐analysis of 82 studies, Bennet et al. reported that individuals with the E2/E2 allele (two minor alleles of rs7412 and no minor alleles of rs429358) had an LDL‐C level 44 mg/dL lower than individuals with the E4/E4 allele (no minor alleles of rs7412 and two minor alleles of rs429358).<sup><a href="#b17" rid="b17" class=" ref">17</a></sup> The SNP has been shown to be significantly associated with lipids in African Americans; in NHANES III, rs7412 was significantly associated with LDL‐C (after FDR adjustment) in non‐Hispanic blacks. The effect size for the association was –22.54 mg/dL per minor allele.<sup><a href="#b21" rid="b21" class=" ref">21</a></sup>
</p><p id="__p20">Despite the large number of candidate gene studies examining rs7412 and LDL‐C levels, this variant has not been included in most GWAS analyses of LDL‐C, as rs7412 was not typed, imputed, or covered by proxy well in most published LDL‐C GWAS. According to the SNAP browser,<sup><a href="#b22" rid="b22" class=" ref">22</a></sup> SNP rs7412 is only genotyped on the Illumina Human 1M single and dual, the Illumina CARe iSelect, and the Illumina Omni Quad and Express, among the common GWAS chips. The previous African American GWAS used the Affymetrix 6.0 chip (Affymetrix, Santa Clara, CA, USA);<sup><a href="#b3" rid="b3" class=" ref">3</a></sup> according to the SNAP browser proxy search, the best available proxy for SNP rs7412 in Yorubans on the Affymetrix 6.0 chip would be SNP rs930461 which has an <em>r</em>
<sup>2</sup> value of 0.13 with rs7412 in the Yoruba in Ibadan HapMap sample. The vast majority of studies included in the European‐ancestry GWAS mentioned above<sup><a href="#b23" rid="b23" class=" ref">23</a></sup>, <sup><a href="#b24" rid="b24" class=" ref">24</a></sup>, <sup><a href="#b25" rid="b25" class=" ref">25</a></sup>, <sup><a href="#b26" rid="b26" class=" ref">26</a></sup>, <sup><a href="#b27" rid="b27" class=" ref">27</a></sup> also did not use any of the chips including rs7412. The lack of coverage/imputation of SNP rs7412 in many GWAS has been highlighted previously in a GWAS analysis of Alzheimer's.<sup><a href="#b28" rid="b28" class=" ref">28</a></sup>
</p><p id="__p21">Significant signals in <em>APOE</em> have been detected in published GWAS. However, these signals are likely driven primarily by SNP rs429358 (rs4420638, the index SNP in the <em>APOE</em> region for the large Teslovich LDL‐C GWAS meta‐analysis,<sup><a href="#b2" rid="b2" class=" ref">2</a></sup> is in LD with SNP rs429358 in European‐ancestry populations [<em>r</em>
<sup>2</sup>= 0.62 in a recent UK sample<sup><a href="#b29" rid="b29" class=" ref">29</a></sup>]). <em>Table S2</em> shows index SNPs in the <em>APOE</em> region that have been found to be significantly associated with LDL‐C in previously published GWAS of whites and African Americans (and estimates of their LD with rs7412—all of which are low [<em>r</em>
<sup>2</sup> &lt; 0.10]). In the Teslovich GWAS, the effect of SNP rs7412 was likely detected somewhat in conditional analysis, when SNP rs445925 was found to be significantly associated with LDL after control for SNP rs4420638.<sup><a href="#b2" rid="b2" class=" ref">2</a></sup> However, inclusion of this SNP in the genetic risk score would not be expected to fully capture the effect of SNP rs7412 as the predicted <em>r</em>
<sup>2</sup> between these two SNPs in the 1,000 genomes CEPH population is only 0.59.</p><p id="__p22">The results of this study offer a commentary on the frequent criticism that GWAS has been unsuccessful in explaining a large percentage of trait variation. In the Teslovich paper, the largest per allele effect size for any index SNP associated with LDL‐C was +7.14 mg/dL (for SNP rs4420638).<sup><a href="#b2" rid="b2" class=" ref">2</a></sup> Our study and others suggest that the per allele effect size of rs7412 may well be two to three times this, and the percentage of LDL‐C variance explained by this SNP in our study was 6.7% (2.9% in the dataset without exclusions). Thus, the inclusion of rs7412 in an LDL‐C gene score would be expected to considerably increase the percentage of trait variance explained; a recent analysis which fine mapped five loci identified through LDL‐C GWAS saw the percentage variance in LDL‐C explained by genes increase from 3.1% to 6.5% with the addition of several variants not covered by recent GWAS, including rs7412.<sup><a href="#b20" rid="b20" class=" ref">20</a></sup> The absence of SNP rs7412 (or a suitable proxy) in previously published GWAS to date has not been highlighted prominently in the discussion of lipid GWAS which we focused primarily on novel SNP discovery. Current GWAS chips combined with imputation cover a great deal of the genome, but there are many untested common variants and these variants may be contributing considerably to trait variation. This issue likely explains some of the “missing heritability” and is of particular relevance in African Americans, where there are more SNPs and stretches of LD are shorter.</p><p id="__p23" class="p p-last">Our results for SNPs rs7412 (<em>APOE</em>) and rs503662 (<em>APOB</em>) (whose effect sizes mirror those in previously published studies<sup><a href="#b3" rid="b3" class=" ref">3</a></sup>, <sup><a href="#b21" rid="b21" class=" ref">21</a></sup>), replicate previous experience with other phenotypes in the eMERGE consortium demonstrating that variables derived from EMR can be used for GWAS and other epidemiologic studies.<sup><a href="#b4" rid="b4" class=" ref">4</a></sup>, <sup><a href="#b13" rid="b13" class=" ref">13</a></sup>, <sup><a href="#b30" rid="b30" class=" ref">30</a></sup>, <sup><a href="#b31" rid="b31" class=" ref">31</a></sup> We initially hypothesized that using LDL‐C from EMR might prove difficult because some individuals may not have followed instructions to fast before blood draws. However, we believe the use of the median LDL‐C measure helped to protect against the selection of incorrect measurements due to nonfasting status. If some nonfasting calculated LDL‐C measurements did enter the dataset, we would anticipate that this would not occur differentially by genotype, and thus any error introduced should have biased the estimate of effect to the null. The detection of the rs7412 signal in a relatively small population and the relatively large percentage variance explained by the SNP suggest that the extraction of the phenotype from EMR may actually be advantageous in some ways compared to previous analyses undertaken in traditional epidemiologic studies. We believe this is due to our ability both to use a median measure and to eliminate phenotypic noise due to multiple types of medication usage and concomitant illness reduced trait variance. Previously, it has been demonstrated that GWAS using average measures have increased power,<sup><a href="#b32" rid="b32" class=" ref">32</a></sup> and that an HDL‐C GWAS right censored based on relevant comorbidity and medication history detected novel loci.<sup><a href="#b4" rid="b4" class=" ref">4</a></sup> Furthermore, our results in the dataset with no lab values excluded compared to the dataset with exclusions show how much more trait variance is explained by a SNP in a sample with less phenotypic noise due to medications and illness. Earlier studies have typically not censored values based on such a wide variety of medications or concomitant illness; the Teslovich et al. GWAS meta‐analysis in European‐ancestry populations excluded individuals on lipid‐lowering medications, but most of the analyses in the meta‐analysis did not exclude participants based on prevalent disease,<sup><a href="#b2" rid="b2" class=" ref">2</a></sup> while the Lettre et al. GWAS meta‐analysis in African Americans used a correction to account for lipid‐lowering therapy and did not exclude individuals based on prevalent disease.<sup><a href="#b3" rid="b3" class=" ref">3</a></sup> An analysis of percentage of LDL‐C trait variance explained by a GWAS score <em>including rs7412</em> in a <em>large</em> sample with multiple LDL‐C values <em>appropriately censored for medications and prevalent illness</em> may demonstrate that common genetic variation explains a larger percentage of variance in the trait in healthy individuals.</p></div><div id="ss5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss5title">Conclusion</h2><p id="__p24" class="p p-first-last">We performed a GWAS study of LDL‐C in African Americans which showed a very strong association with SNP rs7412 in <em>APOE</em> using a phenotype derived from EMR. Although this association has been reported previously in candidate‐gene studies, it has not been reported in previous GWAS studies because the SNP was likely not typed or imputed in previous analyses. Using median LDL‐C values extracted from EMR after censoring for medication and concomitant illness resulted in a SNP association that explained a relatively large percentage of trait variance. These results suggest we should be cautious in interpreting the small percentage of trait variance detected in large GWAS studies and continue to underscore the value of phenotypes derived from EMRs in genetic and epidemiologic research.</p></div><div id="ss6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss6title">Conflict of Interest</h2><p id="__p25" class="p p-first-last">We have no conflicts to disclose.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supporting information</h2><!--/article/body/sec/--><div class="sec suppmat" id="idm139912013784672"><p id="__p26">
<strong>Figure S1. (A) QQ plot for LDL GWAS in dataset with
exclusions.</strong> Genomic inflation factor for this analysis is equal
to 1.0. <strong>(B) QQ plot for LDL GWAS in dataset with no lab values
excluded.</strong> Genomic inflation factor for this analysis is equal to 1.0.</p><p id="__p27">
<strong>Figure S2.</strong> LD&amp;sol;association plot of <em>APOE</em> region from LDL GWAS in dataset with exclusions.</p><p id="__p28">
<strong>Table S1.</strong> Association of LDL‐C with three SNPs identified in a previous African American GWAS of LDL‐C for the eMERGE African American datasets</p><p id="__p29">
<strong>Table S2.</strong> Details of SNPs from the <em>APOE</em> region previously detected in LDL GWAS</p></div><div class="sec suppmat" id="idm139911992272576"><p id="__p30">Supporting info item</p><div class="sup-box half_rhythm" id="idm139911995537664"><a href="/pmc/articles/PMC3521536/bin/CTS-5-394-s001.doc" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(186K, doc)</sup></div></div></div><div id="ss7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss7title">Acknowledgments</h2><div class="sec"><p id="__p32">The eMERGE Network was initiated and funded by NHGRI, with additional funding from NIGMS through the following grants: U01‐HG‐004610 (Group Health Cooperative); U01‐HG‐004608 (Marshfield Clinic); U01‐HG‐04599 (Mayo Clinic); U01HG004609 (Northwestern University); U01‐HG‐04603 (Vanderbilt University), and the State of Washington Life Sciences Discovery Fund award to the Northwest Institute of Medical Genetics.</p><p id="__p33">LJRT was supported by the National Center for Research Resources (NCRR) and the National Center for Advancing Translational Sciences (NCATS), National Institutes of Health (NIH) through grant number KL2 RR025740. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div></div><div id="ss8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ss8title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1">1. <span class="mixed-citation" id="cit1">
Hindorff LA, 
Junkins HA, 
hall PN, 
Mehta JP, 
Manolio TA. 
<span class="ref-title">A Catalog of Published Genome‐Wide Association Studies</span>. 2010.  Available at: <a href="http://www.genome.gov/gwastudies" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.gov/gwastudies</a>. Accessed on August 24, 2010.
</span></div><div class="ref-cit-blk half_rhythm" id="b2">2. <span class="mixed-citation" id="cit2">
Teslovich TM, 
Musunuru K, 
Smith AV, <em>et al</em>
<span class="ref-title">Biological, clinical and population relevance of 95 loci for blood lipids</span>. 
<span class="ref-journal">
Nature.
</span>
2010; <span class="ref-vol">466</span>(<span class="ref-iss">7307</span>): 707–713.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nature.
&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;volume=466&amp;issue=7307&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3">3. <span class="mixed-citation" id="cit3">
Lettre G, 
Palmer CD, 
Young T, <em>et al</em>
<span class="ref-title">Genome‐wide association study of coronary heart disease and its risk factors in 8,090 African Americans: the NHLBI CARe Project</span>. 
<span class="ref-journal">
PLoS Genet.
</span>
2011; <span class="ref-vol">7</span>(<span class="ref-iss">2</span>): e1001300.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037413/">PMC free article</a>]</span> [<a href="/pubmed/21347282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
PLoS+Genet.
&amp;title=Genome‐wide+association+study+of+coronary+heart+disease+and+its+risk+factors+in+8,090+African+Americans:+the+NHLBI+CARe+Project&amp;volume=7&amp;issue=2&amp;publication_year=2011&amp;pages=e1001300&amp;pmid=21347282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4">4. <span class="mixed-citation" id="cit4">
Turner SD, 
Berg RL, 
Linneman JG, <em>et al</em>
<span class="ref-title">Knowledge‐driven multi‐locus analysis reveals gene‐gene interactions influencing HDL cholesterol level in two independent EMR‐linked biobanks</span>. 
<span class="ref-journal">
PLoS One.
</span>
2011; <span class="ref-vol">6</span>(<span class="ref-iss">5</span>): e19586.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092760/">PMC free article</a>]</span> [<a href="/pubmed/21589926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
PLoS+One.
&amp;title=Knowledge‐driven+multi‐locus+analysis+reveals+gene‐gene+interactions+influencing+HDL+cholesterol+level+in+two+independent+EMR‐linked+biobanks&amp;volume=6&amp;issue=5&amp;publication_year=2011&amp;pages=e19586&amp;pmid=21589926&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5">5. <span class="mixed-citation" id="cit5">
McCarty CA, 
Chisholm RL, 
Chute CG, <em>et al</em>
<span class="ref-title">The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies</span>. 
<span class="ref-journal">
BMC Med Genomics.
</span>
2011; <span class="ref-vol">4</span>: 13.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038887/">PMC free article</a>]</span> [<a href="/pubmed/21269473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
BMC+Med+Genomics.
&amp;title=The+eMERGE+Network:+a+consortium+of+biorepositories+linked+to+electronic+medical+records+data+for+conducting+genomic+studies&amp;volume=4&amp;publication_year=2011&amp;pages=13&amp;pmid=21269473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6">6. <span class="mixed-citation" id="cit6">
Zuvich RL, 
Armstrong LL, 
Bielinski SJ, <em>et al</em>
<span class="ref-title">Pitfalls of merging GWAS data: lessons learned in the eMERGE network and quality control procedures to maintain high data quality</span>. 
<span class="ref-journal">
Genet Epidemiol.
</span>
2011; <span class="ref-vol">35</span>(<span class="ref-iss">8</span>): 887–898.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3592376/">PMC free article</a>]</span> [<a href="/pubmed/22125226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Genet+Epidemiol.
&amp;title=Pitfalls+of+merging+GWAS+data:+lessons+learned+in+the+eMERGE+network+and+quality+control+procedures+to+maintain+high+data+quality&amp;volume=35&amp;issue=8&amp;publication_year=2011&amp;pages=887-898&amp;pmid=22125226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7">7. <span class="mixed-citation" id="cit7">
Crosslin DR, 
McDavid A, 
Weston N, <em>et al</em>
<span class="ref-title">Genetic variants associated with the white blood cell count in 13,923 subjects in the eMERGE Network</span>. 
<span class="ref-journal">
Hum Genet.
</span>
2012; <span class="ref-vol">131</span>(<span class="ref-iss">4</span>): 639–652.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3640990/">PMC free article</a>]</span> [<a href="/pubmed/22037903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Hum+Genet.
&amp;title=Genetic+variants+associated+with+the+white+blood+cell+count+in+13,923+subjects+in+the+eMERGE+Network&amp;volume=131&amp;issue=4&amp;publication_year=2012&amp;pages=639-652&amp;pmid=22037903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8">8. <span class="mixed-citation" id="cit8">
Kho AN, 
Hayes MG, 
Rasmussen‐Torvik L, <em>et al</em>
<span class="ref-title">Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome‐wide association study</span>. 
<span class="ref-journal">
J Am Med Inform Assoc.
</span>
2012; <span class="ref-vol">19</span>(<span class="ref-iss">2</span>): 212–218.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3277617/">PMC free article</a>]</span> [<a href="/pubmed/22101970" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
J+Am+Med+Inform+Assoc.
&amp;title=Use+of+diverse+electronic+medical+record+systems+to+identify+genetic+risk+for+type+2+diabetes+within+a+genome‐wide+association+study&amp;volume=19&amp;issue=2&amp;publication_year=2012&amp;pages=212-218&amp;pmid=22101970&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9">9. <span class="mixed-citation" id="cit9">
Denny JC, 
Ritchie MD, 
Crawford DC, <em>et al</em>
<span class="ref-title">Identification of genomic predictors of atrioventricular conduction: using electronic medical records as a tool for genome science</span>. 
<span class="ref-journal">
Circulation.
</span>
2010; <span class="ref-vol">122</span>(<span class="ref-iss">20</span>): 2016–2021.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991609/">PMC free article</a>]</span> [<a href="/pubmed/21041692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Circulation.
&amp;title=Identification+of+genomic+predictors+of+atrioventricular+conduction:+using+electronic+medical+records+as+a+tool+for+genome+science&amp;volume=122&amp;issue=20&amp;publication_year=2010&amp;pages=2016-2021&amp;pmid=21041692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10">10. <span class="mixed-citation" id="cit10">
Turner S, 
Armstrong LL, 
Bradford Y, <em>et al</em>
<span class="ref-title">Quality control procedures for genome‐wide association studies</span>. 
<span class="ref-journal">
Curr Protoc Hum Genet.
</span>
2011; Chapter 1:Unit1 19.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3066182/">PMC free article</a>]</span> [<a href="/pubmed/21234875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Curr+Protoc+Hum+Genet.
&amp;title=Quality+control+procedures+for+genome‐wide+association+studies&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11">11. <span class="mixed-citation" id="cit11">
Patterson N, 
Price AL, 
Reich D. 
<span class="ref-title">Population structure and eigenanalysis</span>. 
<span class="ref-journal">
PLoS Genet.
</span>
2006; <span class="ref-vol">2</span>(<span class="ref-iss">12</span>): e190.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
PLoS+Genet.
&amp;title=Population+structure+and+eigenanalysis&amp;volume=2&amp;issue=12&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12">12. <span class="mixed-citation" id="cit12">
Price AL, 
Patterson NJ, 
Plenge RM, 
Weinblatt ME, 
Shadick NA, 
Reich D. 
<span class="ref-title">Principal components analysis corrects for stratification in genome‐wide association studies</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2006; <span class="ref-vol">38</span>(<span class="ref-iss">8</span>): 904–909.
 [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome‐wide+association+studies&amp;volume=38&amp;issue=8&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13">13. <span class="mixed-citation" id="cit13">
Kho AN, 
Pacheco JA, 
Peissig PL, <em>et al</em>
<span class="ref-title">Electronic medical records for genetic research: results of the eMERGE consortium</span>. 
<span class="ref-journal">
Sci Transl Med.
</span>
2011; <span class="ref-vol">3</span>(<span class="ref-iss">79</span>): 79re1.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3690272/">PMC free article</a>]</span> [<a href="/pubmed/21508311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Sci+Transl+Med.
&amp;title=Electronic+medical+records+for+genetic+research:+results+of+the+eMERGE+consortium&amp;volume=3&amp;issue=79&amp;publication_year=2011&amp;pages=79re1&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14">14. <span class="mixed-citation" id="cit14">
Kullo IJ, 
Ding K, 
Shameer K, <em>et al</em>
<span class="ref-title">Complement receptor 1 gene variants are associated with erythrocyte sedimentation rate</span>. 
<span class="ref-journal">
Am J Hum Genet.
</span>
2011; <span class="ref-vol">89</span>(<span class="ref-iss">1</span>): 131–138.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135803/">PMC free article</a>]</span> [<a href="/pubmed/21700265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Am+J+Hum+Genet.
&amp;title=Complement+receptor+1+gene+variants+are+associated+with+erythrocyte+sedimentation+rate&amp;volume=89&amp;issue=1&amp;publication_year=2011&amp;pages=131-138&amp;pmid=21700265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15">15. <span class="mixed-citation" id="cit15">
Denny JC, 
Crawford DC, 
Ritchie MD, <em>et al</em>
<span class="ref-title">Variants near FOXE1 are associated with hypothyroidism and other thyroid conditions: using electronic medical records for genome‐ and phenome‐wide studies</span>. 
<span class="ref-journal">
Am J Hum Genet.
</span>
2011; <span class="ref-vol">89</span>(<span class="ref-iss">4</span>): 529–542.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3188836/">PMC free article</a>]</span> [<a href="/pubmed/21981779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Am+J+Hum+Genet.
&amp;title=Variants+near+FOXE1+are+associated+with+hypothyroidism+and+other+thyroid+conditions:+using+electronic+medical+records+for+genome‐+and+phenome‐wide+studies&amp;volume=89&amp;issue=4&amp;publication_year=2011&amp;pages=529-542&amp;pmid=21981779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16">16. <span class="mixed-citation" id="cit16">
Purcell S, 
Neale B, 
Todd‐Brown K, <em>et al</em>
<span class="ref-title">PLINK: a tool set for whole‐genome association and population‐based linkage analyses</span>. 
<span class="ref-journal">
Am J Hum Genet.
</span>
2007; <span class="ref-vol">81</span>(<span class="ref-iss">3</span>): 559–575.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Am+J+Hum+Genet.
&amp;title=PLINK:+a+tool+set+for+whole‐genome+association+and+population‐based+linkage+analyses&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17">17. <span class="mixed-citation" id="cit17">
Bennet AM, 
Di Angelantonio E, 
Ye Z, <em>et al</em>
<span class="ref-title">Association of apolipoprotein E genotypes with lipid levels and coronary risk</span>. 
<span class="ref-journal">
JAMA.
</span>
2007; <span class="ref-vol">298</span>(<span class="ref-iss">11</span>): 1300–1311.
 [<a href="/pubmed/17878422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
JAMA.
&amp;title=Association+of+apolipoprotein+E+genotypes+with+lipid+levels+and+coronary+risk&amp;volume=298&amp;issue=11&amp;publication_year=2007&amp;pages=1300-1311&amp;pmid=17878422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18">18. <span class="mixed-citation" id="cit18">
Rall SC Jr., 
Mahley RW. 
<span class="ref-title">The role of apolipoprotein E genetic variants in lipoprotein disorders</span>. 
<span class="ref-journal">
J Intern Med.
</span>
1992; <span class="ref-vol">231</span>(<span class="ref-iss">6</span>): 653–659.
 [<a href="/pubmed/1619388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
J+Intern+Med.
&amp;title=The+role+of+apolipoprotein+E+genetic+variants+in+lipoprotein+disorders&amp;volume=231&amp;issue=6&amp;publication_year=1992&amp;pages=653-659&amp;pmid=1619388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19">19. <span class="mixed-citation" id="cit19">
Davignon J, 
Gregg RE, 
Sing CF. 
<span class="ref-title">Apolipoprotein E polymorphism and atherosclerosis</span>. 
<span class="ref-journal">
Arteriosclerosis.
</span>
1988; <span class="ref-vol">8</span>(<span class="ref-iss">1</span>): 1–21.
 [<a href="/pubmed/3277611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Arteriosclerosis.
&amp;title=Apolipoprotein+E+polymorphism+and+atherosclerosis&amp;volume=8&amp;issue=1&amp;publication_year=1988&amp;pages=1-21&amp;pmid=3277611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20">20. <span class="mixed-citation" id="cit20">
Sanna S, 
Li B, 
Mulas A, <em>et al</em>
<span class="ref-title">Fine mapping of five Loci associated with low‐density lipoprotein cholesterol detects variants that double the explained heritability</span>. 
<span class="ref-journal">
PLoS Genet.
</span>
2011; <span class="ref-vol">7</span>(<span class="ref-iss">7</span>): e1002198.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3145627/">PMC free article</a>]</span> [<a href="/pubmed/21829380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
PLoS+Genet.
&amp;title=Fine+mapping+of+five+Loci+associated+with+low‐density+lipoprotein+cholesterol+detects+variants+that+double+the+explained+heritability&amp;volume=7&amp;issue=7&amp;publication_year=2011&amp;pages=e1002198&amp;pmid=21829380&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21">21. <span class="mixed-citation" id="cit21">
Chang MH, 
Yesupriya A, 
Ned RM, 
Mueller PW, 
Dowling NF. 
<span class="ref-title">Genetic variants associated with fasting blood lipids in the U.S. population: Third National Health and Nutrition Examination Survey</span>. 
<span class="ref-journal">
BMC Med Genet.
</span>
2010; <span class="ref-vol">11</span>: 62.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2876148/">PMC free article</a>]</span> [<a href="/pubmed/20406466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
BMC+Med+Genet.
&amp;title=Genetic+variants+associated+with+fasting+blood+lipids+in+the+U.S.+population:+Third+National+Health+and+Nutrition+Examination+Survey&amp;volume=11&amp;publication_year=2010&amp;pages=62&amp;pmid=20406466&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22">22. <span class="mixed-citation" id="cit22">
Johnson AD, 
Handsaker RE, 
Pulit SL, 
Nizzari MM, 
O’Donnell CJ, 
de Bakker PI. 
<span class="ref-title">SNAP: a web‐based tool for identification and annotation of proxy SNPs using HapMap</span>. 
<span class="ref-journal">
Bioinformatics.
</span>
2008; <span class="ref-vol">24</span>(<span class="ref-iss">24</span>): 2938–2939.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Bioinformatics.
&amp;title=SNAP:+a+web‐based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;volume=24&amp;issue=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23">23. <span class="mixed-citation" id="cit23">
Willer CJ, 
Sanna S, 
Jackson AU, <em>et al</em>
<span class="ref-title">Newly identified loci that influence lipid concentrations and risk of coronary artery disease</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2008; <span class="ref-vol">40</span>(<span class="ref-iss">2</span>): 161–169.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5206900/">PMC free article</a>]</span> [<a href="/pubmed/18193043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Newly+identified+loci+that+influence+lipid+concentrations+and+risk+of+coronary+artery+disease&amp;volume=40&amp;issue=2&amp;publication_year=2008&amp;pages=161-169&amp;pmid=18193043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24">24. <span class="mixed-citation" id="cit24">
Kathiresan S, 
Melander O, 
Guiducci C, <em>et al</em>
<span class="ref-title">Six new loci associated with blood low‐density lipoprotein cholesterol, high‐density lipoprotein cholesterol or triglycerides in humans</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2008; <span class="ref-vol">40</span>(<span class="ref-iss">2</span>): 189–197.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2682493/">PMC free article</a>]</span> [<a href="/pubmed/18193044" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Six+new+loci+associated+with+blood+low‐density+lipoprotein+cholesterol,+high‐density+lipoprotein+cholesterol+or+triglycerides+in+humans&amp;volume=40&amp;issue=2&amp;publication_year=2008&amp;pages=189-197&amp;pmid=18193044&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25">25. <span class="mixed-citation" id="cit25">
Kathiresan S, 
Willer CJ, 
Peloso GM, <em>et al</em>
<span class="ref-title">Common variants at 30 loci contribute to polygenic dyslipidemia</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2009; <span class="ref-vol">41</span>(<span class="ref-iss">1</span>): 56–65.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2881676/">PMC free article</a>]</span> [<a href="/pubmed/19060906" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Common+variants+at+30+loci+contribute+to+polygenic+dyslipidemia&amp;volume=41&amp;issue=1&amp;publication_year=2009&amp;pages=56-65&amp;pmid=19060906&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26">26. <span class="mixed-citation" id="cit26">
Aulchenko YS, 
Ripatti S, 
Lindqvist I, <em>et al</em>
<span class="ref-title">Loci influencing lipid levels and coronary heart disease risk in 16 European population cohorts</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2009; <span class="ref-vol">41</span>(<span class="ref-iss">1</span>): 47–55.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687074/">PMC free article</a>]</span> [<a href="/pubmed/19060911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Loci+influencing+lipid+levels+and+coronary+heart+disease+risk+in+16+European+population+cohorts&amp;volume=41&amp;issue=1&amp;publication_year=2009&amp;pages=47-55&amp;pmid=19060911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b27">27. <span class="mixed-citation" id="cit27">
Sabatti C, 
Service SK, 
Hartikainen AL, <em>et al</em>
<span class="ref-title">Genome‐wide association analysis of metabolic traits in a birth cohort from a founder population</span>. 
<span class="ref-journal">
Nat Genet.
</span>
2009; <span class="ref-vol">41</span>(<span class="ref-iss">1</span>): 35–46.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687077/">PMC free article</a>]</span> [<a href="/pubmed/19060910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Nat+Genet.
&amp;title=Genome‐wide+association+analysis+of+metabolic+traits+in+a+birth+cohort+from+a+founder+population&amp;volume=41&amp;issue=1&amp;publication_year=2009&amp;pages=35-46&amp;pmid=19060910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b28">28. <span class="mixed-citation" id="cit28">
Beecham GW, 
Martin ER, 
Gilbert JR, 
Haines JL, 
Pericak‐Vance MA. 
<span class="ref-title">
<em>APOE</em> is not associated with Alzheimer disease: a cautionary tale of genotype imputation</span>. 
<span class="ref-journal">
Ann Hum Genet.
</span>
2010; <span class="ref-vol">74</span>(<span class="ref-iss">3</span>): 189–194.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2934779/">PMC free article</a>]</span> [<a href="/pubmed/20529013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Ann+Hum+Genet.
&amp;title=
APOE+is+not+associated+with+Alzheimer+disease:+a+cautionary+tale+of+genotype+imputation&amp;volume=74&amp;issue=3&amp;publication_year=2010&amp;pages=189-194&amp;pmid=20529013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b29">29. <span class="mixed-citation" id="cit29">
Ken‐Dror G, 
Talmud PJ, 
Humphries SE, 
Drenos F. 
<span class="ref-title">
<em>APOE</em>/C1/C4/C2 gene cluster genotypes, haplotypes and lipid levels in prospective coronary heart disease risk among UK healthy men</span>. 
<span class="ref-journal">
Mol Med.
</span>
2010; <span class="ref-vol">16</span>(<span class="ref-iss">9–10</span>): 389–399.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935949/">PMC free article</a>]</span> [<a href="/pubmed/20498921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Mol+Med.
&amp;title=
APOE/C1/C4/C2+gene+cluster+genotypes,+haplotypes+and+lipid+levels+in+prospective+coronary+heart+disease+risk+among+UK+healthy+men&amp;volume=16&amp;issue=9–10&amp;publication_year=2010&amp;pages=389-399&amp;pmid=20498921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b30">30. <span class="mixed-citation" id="cit30">
Ritchie MD, 
Denny JC, 
Crawford DC, <em>et al</em>
<span class="ref-title">Robust replication of genotype‐phenotype associations across multiple diseases in an electronic medical record</span>. 
<span class="ref-journal">
Am J Hum Genet.
</span>
2010; <span class="ref-vol">86</span>(<span class="ref-iss">4</span>): 560–572.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850440/">PMC free article</a>]</span> [<a href="/pubmed/20362271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Am+J+Hum+Genet.
&amp;title=Robust+replication+of+genotype‐phenotype+associations+across+multiple+diseases+in+an+electronic+medical+record&amp;volume=86&amp;issue=4&amp;publication_year=2010&amp;pages=560-572&amp;pmid=20362271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b31">31. <span class="mixed-citation" id="cit31">
Kullo IJ, 
Fan J, 
Pathak J, 
Savova GK, 
Ali Z, 
Chute CG. 
<span class="ref-title">Leveraging informatics for genetic studies: use of the electronic medical record to enable a genome‐wide association study of peripheral arterial disease</span>. 
<span class="ref-journal">
J Am Med Inform Assoc.
</span>
2010; <span class="ref-vol">17</span>(<span class="ref-iss">5</span>): 568–574.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2995686/">PMC free article</a>]</span> [<a href="/pubmed/20819866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
J+Am+Med+Inform+Assoc.
&amp;title=Leveraging+informatics+for+genetic+studies:+use+of+the+electronic+medical+record+to+enable+a+genome‐wide+association+study+of+peripheral+arterial+disease&amp;volume=17&amp;issue=5&amp;publication_year=2010&amp;pages=568-574&amp;pmid=20819866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b32">32. <span class="mixed-citation" id="cit32">
Rasmussen‐Torvik LJ, 
Alonso A, 
Li M, <em>et al</em>
<span class="ref-title">Impact of repeated measures and sample selection on genome‐wide association studies of fasting glucose</span>. 
<span class="ref-journal">
Genet Epidemiol.
</span>
2010; <span class="ref-vol">34</span>(<span class="ref-iss">7</span>): 665–673.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2964401/">PMC free article</a>]</span> [<a href="/pubmed/20839289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=
Genet+Epidemiol.
&amp;title=Impact+of+repeated+measures+and+sample+selection+on+genome‐wide+association+studies+of+fasting+glucose&amp;volume=34&amp;issue=7&amp;publication_year=2010&amp;pages=665-673&amp;pmid=20839289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3521536&amp;issue-id=292839&amp;journal-id=2996&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139911979462240title">Abstract</a></li><li><a href="#ss1title">Introduction</a></li><li><a href="#ss2title">Methods</a></li><li><a href="#ss3title">Results</a></li><li><a href="#ss4title">Discussion</a></li><li><a href="#ss5title">Conclusion</a></li><li><a href="#ss6title">Conflict of Interest</a></li><li><a href="#__sec1title">Supporting information</a></li><li><a href="#ss7title">Acknowledgments</a></li><li><a href="#ss8title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Clinical and Translational Science</span> are provided here courtesy of <strong>Wiley-Blackwell</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3521536/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3521536/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3521536/pdf/CTS-5-394.pdf">PDF (1.7M)</a></li> | <li><a href="#" data-citationid="PMC3521536" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521536%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521536%2F&amp;text=High%20Density%20GWAS%20for%20LDL%20Cholesterol%20in%20African%20Americans%20Using%20Electronic%20Medical%20Records%20Reveals%20a%20Strong%20Protective%20Variant%20in%20APOE%0A" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521536%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23067351" data-aiid="5439794" data-aid="3521536" data-iid="292839" data-domainid="2996" data-domain="cts" data-accid="PMC3521536" data-md5="449db054a4a48b13c8d03b97f3a893c0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T09:43:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8CC9CFDD54ACC100000000048102E7&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3521536%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=cts&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3521536/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>