<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3312985/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A large scale multivariate parallel ICA method reveals novel imaging-genetic relationships for Alzheimer’s Disease in the ADNI cohort</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312985/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Neuroimage"><meta name="citation_title" content="A large scale multivariate parallel ICA method reveals novel imaging-genetic relationships for Alzheimer’s Disease in the ADNI cohort"><meta name="citation_authors" content="Shashwath A Meda, Balaji Narayanan, Jingyu Liu, Nora I. Perrone-Bizzozero, Michael C. Stevens, Vince D. Calhoun, David C. Glahn, Li Shen, Shannon L. Risacher, Andrew J. Saykin, Godfrey D. Pearlson, Alzheimer’s Disease Neuroimaging Initiative"><meta name="citation_date" content="15 April 2012"><meta name="citation_issue" content="3"><meta name="citation_volume" content="60"><meta name="citation_firstpage" content="1608"><meta name="citation_doi" content="10.1016/j.neuroimage.2011.12.076"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3312985/?report=abstract"><meta name="citation_pmid" content="22245343"><meta name="DC.Title" content="A large scale multivariate parallel ICA method reveals novel imaging-genetic relationships for Alzheimer’s Disease in the ADNI cohort"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Shashwath A Meda"><meta name="DC.Contributor" content="Balaji Narayanan"><meta name="DC.Contributor" content="Jingyu Liu"><meta name="DC.Contributor" content="Nora I. Perrone-Bizzozero"><meta name="DC.Contributor" content="Michael C. Stevens"><meta name="DC.Contributor" content="Vince D. Calhoun"><meta name="DC.Contributor" content="David C. Glahn"><meta name="DC.Contributor" content="Li Shen"><meta name="DC.Contributor" content="Shannon L. Risacher"><meta name="DC.Contributor" content="Andrew J. Saykin"><meta name="DC.Contributor" content="Godfrey D. Pearlson"><meta name="DC.Contributor" content="Alzheimer’s Disease Neuroimaging Initiative"><meta name="DC.Date" content="2012 Apr 15"><meta name="DC.Identifier" content="10.1016/j.neuroimage.2011.12.076"><meta name="DC.Language" content="en"><meta property="og:title" content="A large scale multivariate parallel ICA method reveals novel imaging-genetic relationships for Alzheimer’s Disease in the ADNI cohort"><meta property="og:type" content="article"><meta property="og:description" content="The underlying genetic etiology of late onset Alzheimer’s disease (LOAD) remains largely unknown, likely due to its polygenic architecture and a lack of sophisticated analytic methods to evaluate complex genotype-phenotype models. The aim of the ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312985/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3312985/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3312985/pdf/nihms-348696.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89F704DD5313410000000008E00607.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312985%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3312985</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Neuroimage</a></span>. Author manuscript; available in PMC 2013 Apr 15.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22245343" target="pmc_ext" ref="reftype=publisher&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Neuroimage. 2012 Apr 15; 60(3): 1608–1621. </span></a></div><span class="fm-vol-iss-date">Published online 2012 Jan 8. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.neuroimage.2011.12.076" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.neuroimage.2011.12.076</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3312985</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS348696</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22245343">22245343</a></div></div></div></div><h1 class="content-title">A large scale multivariate parallel ICA method reveals novel imaging-genetic relationships for Alzheimer’s Disease in the ADNI cohort</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Meda%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338162189632" co-class="co-affbox">Shashwath A Meda</a>, MS.,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Narayanan%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338200990368" co-class="co-affbox">Balaji Narayanan</a>, PhD.,<sup>1</sup> <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338200503808" co-class="co-affbox">Jingyu Liu</a>, PhD,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Perrone-Bizzozero%20NI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338128923136" co-class="co-affbox">Nora I. Perrone-Bizzozero</a>, PhD,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Stevens%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338128399328" co-class="co-affbox">Michael C. Stevens</a>, PhD,<sup>1</sup> <a href="/pubmed/?term=Calhoun%20VD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338127030800" co-class="co-affbox">Vince D. Calhoun</a>, PhD,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Glahn%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338123210016" co-class="co-affbox">David C. Glahn</a>, Ph.D.,<sup>1,</sup><sup>7</sup> <a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338197088848" co-class="co-affbox">Li Shen</a>, Ph.D.,<sup>6</sup> <a href="/pubmed/?term=Risacher%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338197086368" co-class="co-affbox">Shannon L. Risacher</a>, PhD,<sup>6</sup> <a href="/pubmed/?term=Saykin%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338197083888" co-class="co-affbox">Andrew J. Saykin</a>, PhD,<sup>6</sup> <a href="/pubmed/?term=Pearlson%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343" class="affpopup" co-rid="_co_idm140338128425536" co-class="co-affbox">Godfrey D. Pearlson</a>, M.D.,<sup>1,</sup><sup>7</sup> and  Alzheimer’s Disease Neuroimaging Initiative<sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140338162189632"><h3 class="no_margin">Shashwath A Meda</h3><p><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </p><p><sup>2</sup>Center for Human Genetics and Research, Vanderbilt University, Nashville, TN 37203 </p><div>Find articles by <a href="/pubmed/?term=Meda%20SA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Shashwath A Meda</a></div></div><div id="_co_idm140338200990368"><h3 class="no_margin">Balaji Narayanan</h3><p><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </p><div>Find articles by <a href="/pubmed/?term=Narayanan%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Balaji Narayanan</a></div></div><div id="_co_idm140338200503808"><h3 class="no_margin">Jingyu Liu</h3><p><sup>3</sup>Department of ECE, University of New Mexico, Albuquerque, NM 87106 </p><p><sup>4</sup>Mind Research Network, Albuquerque, NM 87106 </p><div>Find articles by <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Jingyu Liu</a></div></div><div id="_co_idm140338128923136"><h3 class="no_margin">Nora I. Perrone-Bizzozero</h3><p><sup>4</sup>Mind Research Network, Albuquerque, NM 87106 </p><p><sup>5</sup>Department of Neurosciences, University of New Mexico, Albuquerque, NM 87106 </p><div>Find articles by <a href="/pubmed/?term=Perrone-Bizzozero%20NI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Nora I. Perrone-Bizzozero</a></div></div><div id="_co_idm140338128399328"><h3 class="no_margin">Michael C. Stevens</h3><p><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </p><div>Find articles by <a href="/pubmed/?term=Stevens%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Michael C. Stevens</a></div></div><div id="_co_idm140338127030800"><h3 class="no_margin">Vince D. Calhoun</h3><p><sup>3</sup>Department of ECE, University of New Mexico, Albuquerque, NM 87106 </p><p><sup>4</sup>Mind Research Network, Albuquerque, NM 87106 </p><div>Find articles by <a href="/pubmed/?term=Calhoun%20VD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Vince D. Calhoun</a></div></div><div id="_co_idm140338123210016"><h3 class="no_margin">David C. Glahn</h3><p><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </p><p><sup>7</sup>Departments of Psychiatry and Neurobiology, Yale University, New Haven, CT 06511 </p><div>Find articles by <a href="/pubmed/?term=Glahn%20DC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">David C. Glahn</a></div></div><div id="_co_idm140338197088848"><h3 class="no_margin">Li Shen</h3><p><sup>6</sup>Department of Radiology and Imaging Sciences, Indiana University School of Medicine, Indianapolis, IN 46202 </p><div>Find articles by <a href="/pubmed/?term=Shen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Li Shen</a></div></div><div id="_co_idm140338197086368"><h3 class="no_margin">Shannon L. Risacher</h3><p><sup>6</sup>Department of Radiology and Imaging Sciences, Indiana University School of Medicine, Indianapolis, IN 46202 </p><div>Find articles by <a href="/pubmed/?term=Risacher%20SL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Shannon L. Risacher</a></div></div><div id="_co_idm140338197083888"><h3 class="no_margin">Andrew J. Saykin</h3><p><sup>6</sup>Department of Radiology and Imaging Sciences, Indiana University School of Medicine, Indianapolis, IN 46202 </p><div>Find articles by <a href="/pubmed/?term=Saykin%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Andrew J. Saykin</a></div></div><div id="_co_idm140338128425536"><h3 class="no_margin">Godfrey D. Pearlson</h3><p><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </p><p><sup>7</sup>Departments of Psychiatry and Neurobiology, Yale University, New Haven, CT 06511 </p><div>Find articles by <a href="/pubmed/?term=Pearlson%20GD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22245343">Godfrey D. Pearlson</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140338129271872_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140338129271872_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140338129271872_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Olin Neuropsychiatric Research Center, Hartford Hospital/IOL, Hartford CT 06106 </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Center for Human Genetics and Research, Vanderbilt University, Nashville, TN 37203 </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of ECE, University of New Mexico, Albuquerque, NM 87106 </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Mind Research Network, Albuquerque, NM 87106 </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Neurosciences, University of New Mexico, Albuquerque, NM 87106 </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Radiology and Imaging Sciences, Indiana University School of Medicine, Indianapolis, IN 46202 </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Departments of Psychiatry and Neurobiology, Yale University, New Haven, CT 06511 </div><div id="CR1"><sup>*</sup><strong>Corresponding Author:</strong> Shashwath Meda, Center for Human Genetics and Research, 515-F Light Hall, Vanderbilt University Medical Center, Nashville, TN 37203, Phone: 615-875-3086, <a href="mailto:dev@null" data-email="ude.tlibrednav@adem.htawhsahS" class="oemail">ude.tlibrednav@adem.htawhsahS</a></div><div id="FN3"><sup>*</sup>Data used in preparation of this article were obtained from the Alzheimer’s Disease Neuroimaging Initiative (ADNI) database (<a href="http://adni.loni.ucla.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">adni.loni.ucla.edu</a>). As such, the investigators within the ADNI contributed to the design and implementation of ADNI and/or provided data but did not participate in analysis or writing of this report. A complete listing of ADNI investigators can be found at:<a href="http://adni.loni.ucla.edu/wpcontent/uploads/how_to_apply/ADNI_Acknowledgement_List.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://adni.loni.ucla.edu/wpcontent/uploads/how_to_apply/ADNI_Acknowledgement_List.pdf</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140338129271872_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN1">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=22245343" target="pmc_ext" ref="reftype=publisher&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Neuroimage</a></div><div><strong>This article has been corrected. </strong>See the correction in volume 62 on&nbsp;page&nbsp;2177.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3312985/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>01.</strong></div><div><a href="/pmc/articles/PMC3312985/bin/NIHMS348696-supplement-01.tif" data-ga-action="click_feat_suppl">NIHMS348696-supplement-01.tif</a><span style="color:gray"> (540K)</span></div><div class="small guid">GUID:&nbsp;22F9928B-6B90-4ADB-B478-1F78CA990C2C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>02.</strong></div><div><a href="/pmc/articles/PMC3312985/bin/NIHMS348696-supplement-02.tif" data-ga-action="click_feat_suppl">NIHMS348696-supplement-02.tif</a><span style="color:gray"> (2.3M)</span></div><div class="small guid">GUID:&nbsp;A907F4F9-E330-4E20-9D43-85D0F7E12046</div></div></dd></dl></div><div id="idm140338197075968" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140338197075968title">Abstract</h2><!--article-meta--><div><p id="P2" class="p p-first-last">The underlying genetic etiology of late onset Alzheimer’s disease (LOAD) remains largely unknown, likely due to its polygenic architecture and a lack of sophisticated analytic methods to evaluate complex genotype-phenotype models. The aim of the current study was to overcome these limitations in a bi-multivariate fashion by linking intermediate magnetic resonance imaging (MRI) phenotypes with a genome-wide sample of common single nucleotide polymorphism (SNP) variants. We compared associations between 94 different brain regions of interest derived from structural MRI scans and 533,872 genome-wide SNPs using a novel multivariate statistical procedure, parallel-independent component analysis, in a large, national multi-center subject cohort. The study included 209 elderly healthy controls, 367 subjects with amnestic mild cognitive impairment and 181 with mild, early-stage LOAD, Caucasian adults, from the Alzheimer’s Disease Neuroimaging Initiative cohort. Imaging was performed on comparable 1.5T scanners at over 50 sites in the USA/Canada. Four primary “genetic components” were associated significantly with a single structural network including all regions involved neuropathologically in LOAD. Pathway analysis suggested that each component included several genes already known to contribute to LOAD risk (e.g. APOE4) or involved in pathologic processes contributing to the disorder, including inflammation, diabetes, obesity and cardiovascular disease. In addition significant novel genes identified included ZNF673, VPS13, SLC9A7, ATP5G2 and SHROOM2. Unlike conventional analyses, this multivariate approach identified distinct groups of genes that are plausibly linked in physiologic pathways, perhaps epistatically. Further, the study exemplifies the value of this novel approach to explore large-scale data sets involving high-dimensional gene and endophenotype data.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">epistasis, multivariate, ICA, genotype-phenotype, pathway, enrichment</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P3" class="p p-first">Late onset Alzheimer’s disease (LOAD), the commonest cause of late-life dementia (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667133">Bekris et al., 2010</a>) has high heritability (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667145">Gatz et al., 2006a</a>; <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667143">Gatz et al., 2006b</a>). However, its etiopathology, pathogenesis and major risk genes are only partly known, mainly due to its genetic complexity and heterogeneity. The “amyloid hypothesis” seems insufficient to fully explain LOAD etiology and alternative hypotheses continue to be advanced (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_286334550">Pimplikar et al., 2010</a>).</p><p id="P4">To date only one gene of major effect, apolipoprotein E ε4 (APOE4), replicates as significantly influencing LOAD risk (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667159">Strittmatter et al., 1993</a>), but does not account for all genetic variability, suggesting the interplay of multiple, mostly unidentified susceptibility loci of smaller effect size acting multiplicatively under a common disease variant model (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667160">Eccles and Tapper, 2010</a>) and/or with environmental factors (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667120">Traynor and Singleton, 2010</a>). Recent high-throughput genome wide association studies (GWAS) (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667130">van Es and van den Berg, 2009</a>)(<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667179">Grupe et al., 2007</a>; <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667151">Harold et al., 2009</a>; <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667153">Seshadri et al., 2010</a>) have identified and replicated in addition to APOE4, other genes such as BIN1, CLU, ABCA7, CR1, PICALM, MS4A6A, CD33, MSA4E and CD2AP, all of which (apart from APOE) have modest effect sizes and cumulatively account for only 35% of the population attributable risk (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667116">Ku et al., 2011</a>; <a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667134">Naj et al., 2011</a>). However, if LOAD risk is mediated in part by common polymorphisms individually conferring low disease risk, acting in concert, typical univariate GWAS might not have enough power to consistently detect these effects unless they utilize very large sample sizes. This might be an inherent issue as obtaining such large sample sizes are usually quite difficult. Also more importantly univariate studies do not take into account the effect of multiple genes at once. This is important because major LOAD risk factors include obesity, cerebrovascular disease and diabetes, all disorders with significant genetic underpinnings (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667166">Profenno et al., 2010</a>), suggesting causative genes might belong to common biological pathways shared by these conditions. To circumvent some of these issues, multivariate analyses have been suggested as an approach to identify important genetic factors in LOAD (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667142">Gandhi and Wood, 2010</a>).</p><p id="P5">MRI captures robust phenotypic neuroanatomical LOAD biomarkers, most consistently implicating posterior cingulate and entorhinal cortices, hippocampus and other medial temporal structures (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667139">Jack et al., 2010a</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667201">Jack et al., 2010b</a>; <a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667199">Smith, 2010</a>; <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667131">Villain et al., 2010</a>) corresponding to sites of early, severe LOAD-related neuropathology. Imaging genetics attempts to bridge genetic variations with phenotypic trait markers, relating genotypic variations to underlying biological disease etiologies and increasing statistical power, thereby requiring smaller sample sizes (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667170">Potkin et al., 2009</a>). However, such strategies require tools to simultaneously accommodate thousands of data points per feature set (e.g. ~10<sup>5</sup> voxels from imaging data and up to 10<sup>6</sup> SNPs from genetic data), posing a major statistical challenge. Often, large scale studies are performed in a univariate fashion that significantly limits either one or both feature sets. However, these techniques can curtail the usefulness of multidimensional data to identify potentially informative relationships. Conventional voxel-wise analyses are computationally time consuming on a genome-wide scale and ineffectively capture cumulative effects spread over multiple genes. Prior analyses (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667164">Biffi et al., 2010</a>; <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667149">Potkin et al., 2009</a>; <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667196">Shen et al., 2010</a>) on the multi-site MRI/genetic ADNI dataset used massively univariate approaches: GWAS, that confirmed the risk status of APOE4 and identified TOMM40 (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667198">Shen et al., 2010</a>) and hypothesis-driven analyses using pre-selected known affected brain regions plus GWAS, that reinforced the status of promising individual genes of interest (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667141">Biffi et al., 2010</a>). However, no analyses have evaluated the premise that genetic determinants are not randomly distributed among relevant biological pathways but instead grouped together among specific biological processes, nor have they detected predicted groups of common, interactive risk polymorphisms.</p><p id="P6">Parallel independent component analysis (Para-ICA) a novel multivariate data-driven, hypothesis-free statistical technique, extends ICA to analyze multiple modalities simultaneously (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667148">Calhoun et al., 2009</a>). Para-ICA identifies simultaneously clusters of associated, likely interacting genes related to either: (a) functional brain networks, (b) related structural brain regions, or (c) physiologic processes e.g. EEG patterns or other potential endophenotypes and shows their relationships (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667147">Calhoun et al., 2009</a>). Beginning with two modalities (here, SNP’s and MRI-derived regional brain volume/thickness), we sought to discover underlying factors from both modalities and their connections. Similar to conventional ICA analyses, extracted structural MRI components are maximally independent within modality and loading coefficients represent variation among individuals. Networks or components extracted from genetic data are groups of interacting SNP loci, contributing with varying degrees to a genetic process affecting a downstream biological function, i.e. linear SNP combinations highly associated with related phenotypes. To date, this technique has been used mainly in schizophrenia and healthy controls to find genes responsible for brain structure and function using MRI and EEG patterns (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667192">Jagannathan et al., 2010</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667200">Liu et al., 2008</a>; <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_380292457">Meda et al., 2010</a>). However, subject and SNP numbers in those studies were typically small.</p><p id="P7">Genetic and structural MRI data from the ADNI sample provide an ideal test bed to explore LOAD and to validate application of Para-ICA to larger datasets. The subject number (&gt;800) and large genotypic dataset (&gt;600,000 SNPs) allow for examination of feasibility of scaling up this technique where some valid results are published in this dataset from conventional, hypothesis-driven analyses (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667132">Biffi et al., 2010</a>). Because many LOAD risk genes remain to be discovered, the technique can simultaneously be used to identify novel risk genes, as it identifies clusters of related, interacting SNPs.</p><p id="P8" class="p p-last">We had the following goals: 1) to evaluate whether Para-ICA could be scaled up to deal with larger populations and many more SNPs than previously analyzed; 2) to identify new risk genes for LOAD and their corresponding endophenotypes and 3) to explore the different LOAD-mediating biological interactive pathways in which the identified risk genes might participate. We hypothesized that the method might identify previously unknown LOAD risk genes, as well as known candidate genes. We hypothesized that identified genes would group into LOAD-associated physiologic pathways and processes.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and Methods</h2><p id="P9" class="p p-first">We evaluated associations between two data modalities, structural MRI (sMRI), (regional brain volumes and cortical thicknesses), and genome-wide genotypic data (SNPs), to reveal multivariate relationships between structural brain regions and SNP’s that differed between healthy controls, MCI and AD subjects.</p><div id="S3" class="sec"><h3 id="S3title">Alzheimer’s Disease Neuroimaging Initiative (ADNI) Study</h3><p id="P10" class="p p-first">Data used in the preparation of this article were obtained from the ADNI database (<a href="http://adni.loni.ucla.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">adni.loni.ucla.edu</a>). ADNI results from efforts of many co-investigators from a broad range of academic institutions and private corporations, with subjects recruited from over 50 sites across the U.S. and Canada. For up-to-date information, see <a href="http://www.adni-info.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.adni-info.org</a>. The Principal Investigator of this initiative is Michael W. Weiner, MD, VA Medical Center and University of California – San Francisco.</p><div id="S4" class="sec"><p></p><h4 id="S4title" class="inline">Study Participants </h4><p id="P11" class="p p-first-last">Data derived from the ADNI database on 818 subjects included baseline 1.5 T MRI scans, Illumina SNP genotyping data, APOE genotype status and demographic information. We limited analyses to European-American ADNI subjects (classified initially into respective ethnic groups based on self-report and validated subsequently using genetic markers) to prevent confluence of population stratification effects on data, yielding a total of 209 HC (Mean/SD Age = 76.05/4.94; 113 Males) with no past history of neurological or psychiatric disorder, 367 subjects with MCI (Mean/SD Age = 74.95/7.37; 239 Males) and 181 subjects with clinically-assessed AD (Mean/SD Age = 75.57/7.48; 100 Males) for analysis.</p></div><div id="S5" class="sec"><p></p><h4 id="S5title" class="inline">Single Nucleotide Polymorphisms (SNP’s) - Genotype </h4><p id="P12" class="p p-first">Sample collection and Single nucleotide polymorphism (SNP) genotyping for more than 620,000 target SNPs across the whole genome was completed on all ADNI participants as described in (<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667167">Saykin et al., 2010</a>; <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667126">Shen et al., 2010</a>).</p><div id="S6" class="sec sec-last"><p></p><h5 id="S6title" class="inline">Quality Control/Pre-processing </h5><p id="P13" class="p p-first">Prior to Para-ICA, genotyped SNP’s underwent two pre-processing stages. First, quality control parameters were employed to discard data unsuitable for further analysis. Samples (both subjects and SNP’s) were checked for missing data and those with missing call rates &gt;5% were excluded. Remaining samples were imputed for missing values (&lt;1%) by replacing data with the corresponding major genotype. Following this, all uninformative SNP’s (constant variance) were excluded. SNP’s were then checked for minor allele frequency (MAF); rare SNP variants with MAFs &lt;0.01 were excluded. Highly correlated SNP’s (r&gt;0.95) (in block sizes of 100kb) were removed. Finally, SNPs (in controls only) were checked for Hardy-Weinberg equilibrium set at a threshold of p&lt;1E-7. QC SNP’s (N=533,872) were then carried over to the next processing stage. The above analyses were performed using custom scripts in Matlab 7.0 (<a href="http://www.mathworks.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.mathworks.com</a>).</p><p id="P14" class="p p-last">The above pre-processed SNP’s were subjected to a univariate GWAS type case-control association analysis to identify those differing significantly across the three diagnostic groups. This 1) effectively restricted the core analysis to disease-related genetic data in the current sample, 2) reduced potential noise from interacting genes with little or no relationship to the disease model, providing hypothesis-free data-driven “enrichment,” and 3) improved accuracy and the linking coefficient of the Para-ICA algorithm (determined based on previous simulation results (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667140">Liu et al., 2008</a>)). All SNP’s (using an additive model) were entered into a mass univariate ANOVA design with diagnostic group as the independent factor, using Matlab 7.0. SNP’s surviving a liberal <em>p</em>&lt;0.05 uncorrected threshold were then advanced to the Para-ICA multivariate association analyses. As noted, at this stage, no multiple comparison correction was performed. All significant SNP’s with a <em>p</em>&lt;0.05 uncorrected threshold (N=27,150) were carried forward to Para-ICA to determine genetic associations (including weak effects spread across multiple SNPs) with brain structures.</p></div></div><div id="S7" class="sec sec-last"><p></p><h4 id="S7title" class="inline">MRI Structural Imaging - Phenotype </h4><p id="P15" class="p p-first-last">All subjects underwent a high-resolution 3D structural MRI scan (MPRAGE) as detailed in <a href="http://www.adni-info.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.adni-info.org</a>. We utilized recently published ADNI imaging data, analyzed in Freesurfer V4.1.0 (<a href="http://surfer.nmr.mgh.harvard.edu/fswiki" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://surfer.nmr.mgh.harvard.edu/fswiki</a>) (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667157">Shen et al., 2010</a>), thus, brain structure preparation and analysis methods are described only briefly. An automated Freesurfer Bayesian segmentation and parcellation routine extracted and labeled cortical and subcortical tissue classes (<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667163">Shen et al., 2010</a>), yielding target region volumes, cortical thicknesses and total intracranial volumes for pre-defined brain structure regions-of-interest (ROIs). Freesurfer values for two independently collected MPRAGE scans per subject were averaged to yield a single volume/cortical thickness value. <a href="/pmc/articles/PMC3312985/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> lists all imaging phenotypes (N=94; bilateral volumes of interest and cortical thickness values). All values were normalized by <em>Z</em>-score transformation before entry into Para-ICA.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p8">Details all Freesurfer-derived regions of interest used as the imaging phenotype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338122491056" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Rank order</th><th rowspan="1" colspan="1" valign="top" align="center">Region</th><th rowspan="1" colspan="1" valign="top" align="center">Abs Z-Score for<br>Network S1</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>1</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Superior Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>2</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Superior Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.80</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>3</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Entorhinal Cortex</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.74</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>4</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Middle Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.72</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>5</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Rostral Mid Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Caudal Mid Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.64</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>7</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Inferior Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.58</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>8</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Middle Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.58</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>9</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Hippocampus Volume</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.57</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>10</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Entorhinal Cortex</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.56</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>11</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Rostral Mid Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.54</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>12</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Hippocampus Volume</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.52</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>13</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Isthmus Cingulate</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.51</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>14</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Caudal Mid Frontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>15</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Isthmus Cingulate</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.43</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>16</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Temporal Pole</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.39</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>17</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Temporal Pole</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.37</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>18</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Inferior Parietal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>19</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Inferior Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>20</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Fusiform</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.29</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>21</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Superior Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.28</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>22</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Inferior Lateral Ventricle</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.27</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>23</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Superior marginal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.24</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>24</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Superior marginal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>25</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Inferior Parietal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.21</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>26</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Post Cingulate</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>27</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Precuneus</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>28</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Superior Temporal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>29</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Amygdala Volume</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>30</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Pars Operculum</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>31</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Precentral</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>32</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Pars Trianglaris</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>33</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Amygdala Volume</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>34</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left BanksSTS</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>35</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Pars Trianglaris</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>36</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Post Cingulate</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>37</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Lateral OrbFrontal</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.07</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>38</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Left Precuneus</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>39</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Parahippocampus</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.01</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>40</strong></td><td rowspan="1" colspan="1" valign="top" align="center">
<strong>Right Fusiform</strong>
<sup>**</sup>
</td><td rowspan="1" colspan="1" valign="top" align="center">2.00</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>41</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Lateral OrbFrontal</td><td rowspan="1" colspan="1" valign="top" align="center">1.97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>42</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Pars Orb</td><td rowspan="1" colspan="1" valign="top" align="center">1.97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>43</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Precentral</td><td rowspan="1" colspan="1" valign="top" align="center">1.97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>44</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Pars Operculum</td><td rowspan="1" colspan="1" valign="top" align="center">1.96</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>45</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Parahippocampus</td><td rowspan="1" colspan="1" valign="top" align="center">1.95</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>46</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right BanksSTS</td><td rowspan="1" colspan="1" valign="top" align="center">1.90</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>47</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Medial Orbito Frontal</td><td rowspan="1" colspan="1" valign="top" align="center">1.84</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>48</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Superior Parietal</td><td rowspan="1" colspan="1" valign="top" align="center">1.82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>49</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Medial Orbito Frontal</td><td rowspan="1" colspan="1" valign="top" align="center">1.82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>50</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Frontal Pole</td><td rowspan="1" colspan="1" valign="top" align="center">1.79</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>51</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Ventricle DC</td><td rowspan="1" colspan="1" valign="top" align="center">1.79</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>52</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Ventricle DC</td><td rowspan="1" colspan="1" valign="top" align="center">1.78</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>53</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Paracentral</td><td rowspan="1" colspan="1" valign="top" align="center">1.75</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>54</strong></td><td rowspan="1" colspan="1" valign="top" align="center">3rd Ventricle</td><td rowspan="1" colspan="1" valign="top" align="center">1.71</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>55</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Paracentral</td><td rowspan="1" colspan="1" valign="top" align="center">1.68</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>56</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Superior Parietal</td><td rowspan="1" colspan="1" valign="top" align="center">1.68</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>57</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Frontal Pole</td><td rowspan="1" colspan="1" valign="top" align="center">1.61</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>58</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Pars Orb</td><td rowspan="1" colspan="1" valign="top" align="center">1.56</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>59</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Frontal Pole</td><td rowspan="1" colspan="1" valign="top" align="center">1.52</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>60</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Pallidum Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>61</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Post central</td><td rowspan="1" colspan="1" valign="top" align="center">1.44</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>62</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Rostral Ant Cingulate</td><td rowspan="1" colspan="1" valign="top" align="center">1.41</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>63</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Post central</td><td rowspan="1" colspan="1" valign="top" align="center">1.38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>64</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC_Mid Anterior</td><td rowspan="1" colspan="1" valign="top" align="center">1.37</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>65</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Pallidum Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>66</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC_Central</td><td rowspan="1" colspan="1" valign="top" align="center">1.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>67</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Putamen Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.24</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>68</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Accumbens Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.20</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>69</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC_Mid Posterior</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>70</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Thalamus Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>71</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC_Posterior</td><td rowspan="1" colspan="1" valign="top" align="center">1.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>72</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Putamen Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>73</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Left ingual</td><td rowspan="1" colspan="1" valign="top" align="center">1.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>74</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Lateral Occipital</td><td rowspan="1" colspan="1" valign="top" align="center">1.11</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>75</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Lateral Occipital</td><td rowspan="1" colspan="1" valign="top" align="center">1.10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>76</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Accumbens Volume</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>77</strong></td><td rowspan="1" colspan="1" valign="top" align="center">CC_Anterior</td><td rowspan="1" colspan="1" valign="top" align="center">1.05</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>78</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Caud Ant Cingulate</td><td rowspan="1" colspan="1" valign="top" align="center">1.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>79</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Left ingual</td><td rowspan="1" colspan="1" valign="top" align="center">0.95</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>80</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left TransverseTemporal</td><td rowspan="1" colspan="1" valign="top" align="center">0.84</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>81</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Rostral Ant Cingulate</td><td rowspan="1" colspan="1" valign="top" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>82</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Caud Ant Cingulate</td><td rowspan="1" colspan="1" valign="top" align="center">0.69</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>83</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Thalamus Volume</td><td rowspan="1" colspan="1" valign="top" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>84</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Corpus Collosum</td><td rowspan="1" colspan="1" valign="top" align="center">0.66</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>85</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Corpus Collosum</td><td rowspan="1" colspan="1" valign="top" align="center">0.56</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>86</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Cuneus</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>87</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right TransverseTemporal</td><td rowspan="1" colspan="1" valign="top" align="center">0.51</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>88</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Pericalc</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>89</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Caudate Volume</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>90</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Caudate Volume</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>91</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Cuneus</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>92</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Cerebellar Cortex</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>93</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Right Pericalc</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>94</strong></td><td rowspan="1" colspan="1" valign="top" align="center">Left Cerebellar Cortex</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td></tr></tbody></table></div><div id="largeobj_idm140338122491056" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312985/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>**</sup>Denotes regions with significant loadings on imaging network S1, ranked in order of Z-score loads. S1 was the only imaging network that was significantly associated with any (in this case all four) genetic networks.</div></div></div></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">Genotype-Phenotype Associations (Para-ICA)</h3><p id="P16" class="p p-first">Para-ICA was implemented using the Fusion ICA Toolbox v2.0a; <a href="http://icatb.sourceforge.net" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://icatb.sourceforge.net</a> in Matlab 7.0 to compute independent genetic/imaging networks and simultaneously identify and quantify association between the two modalities/features. This variant of ICA was designed for multimodality processing that extracts components using an entropy term based on information theory to maximize independence and enhances the interconnection by maximizing the linkage function in a joint estimation process (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667158">Calhoun et al., 2009</a>; <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667144">Liu et al., 2008</a>). In addition, Para-ICA estimated loading parameters expressing the weight of the overall component for each subject. Overall correlation values between loading coefficients of the two sets of imaging and genetic component(s) were calculated component-wise for the aggregate sample to identify significantly associated feature sets. Comprehensive mathematical details of the algorithm and methodology are provided in (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667188">Liu et al., 2008</a>). Data values from all three diagnostic groups were organized as a matrix of subjects by SNP’s/imaging-ROI values. These genotype and phenotype data matrices were input to the Para-ICA algorithm (see diagram in <a href="/pmc/articles/PMC3312985/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The number of independent estimated components for both SNP (12 components) and imaging data (8 components) was separately estimated using Akaike information criteria (AIC) (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667138">Calhoun et al., 2001</a>). Resulting correlation values between the Para-ICA feature sets were appropriately corrected for multiple comparisons at this stage and a Bonferroni correction was applied based on 12×8=96 comparisons yielding a corrected <em>p</em> value threshold of 0.05/96 = 0.0005. Once significant feature set associations were identified, all contributing SNPs/imaging ROIs across each significant feature/network/component were thresholded at a supra level |<em>Z</em>|&gt;2.0 to specifically identify dominant loadings for each individual network. SNP’s or regions surpassing this threshold were deemed to be contributing significantly to the overall signal of the corresponding component/network. Subsequently, loading coefficients of significantly associated components were tested in a case-control fashion to test if they differed significantly among diagnostic groups.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3312985/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338161954240" class="figure"><a href="/pmc/articles/PMC3312985/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312985_nihms-348696-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-348696-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312985/bin/nihms-348696-f0001.jpg"></a></div><div id="largeobj_idm140338161954240" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312985/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3312985/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p3">Flow diagram illustrating the Para-ICA technique employed in this study.</p></div></div></div><p id="P17" class="p p-last">Significant SNP’s from each component were then batch queried against the dbSNP database <a href="http://www.ncbi.nlm.nih.gov/projects/SNP/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/SNP/</a> to extract corresponding known gene information; genes from this query (derived for each component) were entered into the functional annotation tool, DAVID (<a href="http://david.abcc.ncifcrf.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://david.abcc.ncifcrf.gov/</a>) to identify enriched biological themes and visualize these genes on functional pathways e.g. KEGG and/or BioCarta. The Ingenuity pathway analysis tool (IPA; <a href="http://www.ingenuity.com/products/pathways_analysis.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com/products/pathways_analysis.html</a>) modeled and analyzed the complex biology and genetic interactions as canonical pathway models within the identified significant genetic network(s). SNPs associated with known genes were mapped to the Ingenuity Pathways Knowledgebase to delineate biological networks. Genes were also input to Funcassociate v2.0 (<a href="http://llama.mshri.on.ca/funcassociate/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://llama.mshri.on.ca/funcassociate/</a>) to reveal significantly enriched functional attributes in each component compared against a gene ontology database. Finally, we performed standard chi-square association analyses on the top 10 <em>Z</em>-score-ranked genes from each Para-ICA-derived genetic network, to determine their relative association with the disease model.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><p id="P18" class="p p-first">Initial data pre-processing with a univariate “GWAS like” analysis (p&lt;0.05 uncorrected) revealed N=27,150 SNPs that differed significantly across groups. It confirmed SNPs from APOE (ε4; p=6.6E-16; ε3; p=3.6E-09) and TOMM40 (p=7.25E-08) as the top three candidate genes, whose genotypes differed significantly across diagnostic groups, as identified in prior GWAS of the same parent dataset (<a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667181">Potkin et al., 2009</a>).</p><div id="S10" class="sec"><h3 id="S10title">Genotype-Phenotype associations using Para-ICA</h3><p id="P19" class="p p-first-last">From the 12 SNP and 8 anatomic principal component networks identified by AIC, Para-ICA identified four different independent genetic networks significantly associated with a single structural network (following Bonferroni correction). These four networks were component/networks 1 (G1; consisting of 169 significant genes/332 SNPs), 2 (G2; 182 genes/377 SNPs), 3 (G3; 267 genes/482 SNPs) and 4 (G4; 169 genes/332 SNPs). G1 and G3 had significant loadings (Z&gt;2) from APOE (ε4). All four networks were significantly associated with only one structural brain network (S1) that encompassed 40 different unilateral regions (a combination of both volumes and cortical thickness surpassing a <em>Z</em>=2 threshold). Key structural regions loading heavily in this component were entorhinal cortex and middle temporal cortex thicknesses and amygdala and hippocampus volumes. A complete list of significant regions is highlighted with double asterisks in <a href="/pmc/articles/PMC3312985/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and illustrated in <a href="/pmc/articles/PMC3312985/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>. Significant genotype-phenotype correlations (between loading parameters of each modality or feature) were as follows: 1) G1-S1 (r=−0.53; p&lt;0.0001) 2) G2-S1 (r=0.32; p&lt;0.0001) 3) G4-S1 (r=0.24; p&lt;0.0001) and G3-S1 (r=−0.14; p=0.0001). <a href="/pmc/articles/PMC3312985/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> summarizes these data with the top 10 representative genes from each genetic network along with their corresponding biological functions. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary figure 1</a> (<a href="#SD1" rid="SD1" class=" supplementary-material">Fig S1</a>) shows correlation (scatter) plots for these associations. Testing loading coefficients of the above networks for between-group differences revealed that in addition to having significant associations they also significantly discriminated groups by baseline diagnosis (AD, MCI, or healthy control). Mean loading coefficients across each genetic/structural network are shown in <a href="#SD2" rid="SD2" class=" supplementary-material">supplementary Figure 2</a> (<a href="#SD2" rid="SD2" class=" supplementary-material">Fig S2</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3312985/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338161949984" class="figure"><a href="/pmc/articles/PMC3312985/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312985_nihms-348696-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-348696-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312985/bin/nihms-348696-f0002.jpg"></a></div><div id="largeobj_idm140338161949984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312985/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3312985/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p4">Significant genotype-phenotype associations identified using the hypothesis-free Para-ICA approach. Genetic factors combine effects from all risk loci, thus explaining more phenotypic variability than traditional analytic approaches. Figure shows the top ten genes, ranked by Z-scores, plus their known functions, from each of the four genetic networks that were significantly correlated with the MRI network. Numbers of significant genes and SNPs within each network is listed at top. <strong>*</strong> Indicates that the same gene appears among the top ten genes of another network. Genes in red text have been previously implicated in LOAD risk.</p></div></div></div></div><div id="S11" class="sec"><h3 id="S11title">Pathway analysis of genetic networks (components)</h3><p id="P20" class="p p-first">Ingenuity Pathway Analysis (IPA) software (<a href="http://www.ingenuity.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com/</a>) was used to detect, visualize, and explore relevant biological networks associated with each genetic component. Top networks for G1 were cellular assembly/organization, cell morphology and development. G2 was enriched with genes related to cardiovascular disease, neurologic disease and cardiac arteriopathy. Primary networks for G4 were cell cycle, cell death and inflammatory response. G3 had an over-representation of genes related to neurological and psychological disorders. All the above networks contained known Alzheimer’s-related proteins in their pathway interactions. Top dynamic networks from each genetic component are illustrated in <a href="/pmc/articles/PMC3312985/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>. Based on known gene functions, the top five IPA canonical pathways for each gene network (sorted in terms of genotype-phenotype linkage significance) were as follows:</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3312985/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338161946176" class="figure"><a href="/pmc/articles/PMC3312985/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312985_nihms-348696-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-348696-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312985/bin/nihms-348696-f0003.jpg"></a></div><div id="largeobj_idm140338161946176" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312985/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3312985/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p5">Representation of the top canonical pathways derived from Ingenuity pathway software for all four significant genetic networks. G1: Pathway enriched for cellular assembly/organization, cell morphology/development; G2: Pathway enriched for cardiac arteriopathy, cardiovascular and neurological disease; G4: Pathway enriched for cell cycle, cell death and inflammatory response; G3: Enriched for genetic, neurological and psychological disorders. Key: red = known AD implicated gene; blue = involved in long-term memory, green = involved in long-term potentiation of CA1 neurons.</p></div></div></div><p id="P21">G1: cAMP mediated signaling, sulfur metabolism, calcium signaling, vascular NO signaling and regulation of IL-2 expression in T lymphocytes. G2: Neuro-protective role of THOP1 in Alzheimer’s, NOS endothelial effects, Type 2 diabetes signaling, tyrosine metabolism, CYP450. G4: cAMP-mediated signaling, cardiac beta-adrenergic signaling, synaptic long-term potentiation, molecular cancer mechanisms, NOS endothelial effects. Significantly associated non-Neurologic disorders were type 2 diabetes (N=80), coronary artery disease (N=71), Crohn’s/inflammatory bowel disease (N=65). G3: Protein kinase A signaling, cardiac beta-adrenergic signaling, cAMP-mediated signaling, amino sugars metabolism, glycosaminoglycan degradation. Significantly associated non-neurologic disorders were Type 2 Diabetes (N=70), Crohn’s disease/IBD (N=57), coronary artery disease (N=57).</p><p id="P22" class="p p-last">The DAVID functional annotation tool revealed that the significant genes from all four genetic networks are involved in multiple biological pathways including Alzheimer’s disease, adherens junction, arrhythmogenic right ventricular and dilated cardiomyopathy, axon guidance, calcium signaling, cell adhesion, ECM receptor interaction, focal adhesion and tight junction and smooth muscle contraction. <a href="/pmc/articles/PMC3312985/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a> illustrates (marked with red stars) the significant genes in our study directly related to Alzheimer’s disease on a KEGG pathway map derived from the above tool. Genes analyzed using the Funcassociate v2.0 toolkit revealed several (N=60) significantly over-represented attributes compared against the gene ontology database. The results presented in <a href="/pmc/articles/PMC3312985/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> are rank ordered based on adjusted p value along with the number of genes in the query, number of genes in the overall attribute and their odds ratios.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3312985/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338122497712" class="figure"><a href="/pmc/articles/PMC3312985/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312985_nihms-348696-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-348696-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312985/bin/nihms-348696-f0004.jpg"></a></div><div id="largeobj_idm140338122497712" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312985/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3312985/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p6">Genes identified as part of the significant Para-ICA networks highlighted on a KEGG Alzheimer’s pathway diagram derived from the DAVID functional association tool. Genes identified using red stars occurred in 1 or more of our 4 gene networks. Genes indicated in red text are the primary genes identified to date responsible for the early form of AD (PSEN, APP) or account for a major proportion of LOAD risk (APOE).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">Results derived from the Funcassociate web tool showing significantly enriched functional attributes, queried against the gene ontology database. Genes fed into the analysis comprised all significant genes from all four genetic networks identified in the study</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338127221776" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="center">Rank</th><th rowspan="1" colspan="1" valign="top" align="center">N</th><th rowspan="1" colspan="1" valign="top" align="center">X</th><th rowspan="1" colspan="1" valign="top" align="center">LOD</th><th rowspan="1" colspan="1" valign="top" align="center">P</th><th rowspan="1" colspan="1" valign="top" align="center">P-adj</th><th rowspan="1" colspan="1" valign="top" align="left">Gene Ontology Attribute</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>1</strong></td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">100</td><td rowspan="1" colspan="1" valign="top" align="center">1.847</td><td rowspan="1" colspan="1" valign="top" align="center">3.50E-28</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">antigen processing and presentation of peptide antigen via MHC class I</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>2</strong></td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">103</td><td rowspan="1" colspan="1" valign="top" align="center">1.831</td><td rowspan="1" colspan="1" valign="top" align="center">6.70E-28</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">antigen processing and presentation of peptide antigen</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>3</strong></td><td rowspan="1" colspan="1" valign="top" align="center">23</td><td rowspan="1" colspan="1" valign="top" align="center">206</td><td rowspan="1" colspan="1" valign="top" align="center">1.56</td><td rowspan="1" colspan="1" valign="top" align="center">3.70E-26</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">MHC class I protein complex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>4</strong></td><td rowspan="1" colspan="1" valign="top" align="center">23</td><td rowspan="1" colspan="1" valign="top" align="center">335</td><td rowspan="1" colspan="1" valign="top" align="center">1.327</td><td rowspan="1" colspan="1" valign="top" align="center">2.90E-21</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">MHC protein complex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>5</strong></td><td rowspan="1" colspan="1" valign="top" align="center">23</td><td rowspan="1" colspan="1" valign="top" align="center">377</td><td rowspan="1" colspan="1" valign="top" align="center">1.271</td><td rowspan="1" colspan="1" valign="top" align="center">4.10E-20</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">antigen processing and presentation</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>6</strong></td><td rowspan="1" colspan="1" valign="top" align="center">121</td><td rowspan="1" colspan="1" valign="top" align="center">2180</td><td rowspan="1" colspan="1" valign="top" align="center">0.409</td><td rowspan="1" colspan="1" valign="top" align="center">6.60E-17</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">plasma membrane part</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>7</strong></td><td rowspan="1" colspan="1" valign="top" align="center">28</td><td rowspan="1" colspan="1" valign="top" align="center">1152</td><td rowspan="1" colspan="1" valign="top" align="center">0.864</td><td rowspan="1" colspan="1" valign="top" align="center">5.60E-14</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">immune response</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>8</strong></td><td rowspan="1" colspan="1" valign="top" align="center">28</td><td rowspan="1" colspan="1" valign="top" align="center">1348</td><td rowspan="1" colspan="1" valign="top" align="center">0.791</td><td rowspan="1" colspan="1" valign="top" align="center">2.50E-12</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">immune system process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>9</strong></td><td rowspan="1" colspan="1" valign="top" align="center">160</td><td rowspan="1" colspan="1" valign="top" align="center">3765</td><td rowspan="1" colspan="1" valign="top" align="center">0.289</td><td rowspan="1" colspan="1" valign="top" align="center">5.90E-12</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">plasma membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>10</strong></td><td rowspan="1" colspan="1" valign="top" align="center">298</td><td rowspan="1" colspan="1" valign="top" align="center">9033</td><td rowspan="1" colspan="1" valign="top" align="center">0.234</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-11</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>11</strong></td><td rowspan="1" colspan="1" valign="top" align="center">254</td><td rowspan="1" colspan="1" valign="top" align="center">7427</td><td rowspan="1" colspan="1" valign="top" align="center">0.237</td><td rowspan="1" colspan="1" valign="top" align="center">3.40E-11</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">membrane part</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>12</strong></td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">33</td><td rowspan="1" colspan="1" valign="top" align="center">1.841</td><td rowspan="1" colspan="1" valign="top" align="center">8.80E-11</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">MHC class I receptor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>13</strong></td><td rowspan="1" colspan="1" valign="top" align="center">9</td><td rowspan="1" colspan="1" valign="top" align="center">150</td><td rowspan="1" colspan="1" valign="top" align="center">1.364</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-09</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">thiolester hydrolase activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>14</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">115</td><td rowspan="1" colspan="1" valign="top" align="center">1.43</td><td rowspan="1" colspan="1" valign="top" align="center">3.40E-09</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">ubiquitin thiolesterase activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>15</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">118</td><td rowspan="1" colspan="1" valign="top" align="center">1.418</td><td rowspan="1" colspan="1" valign="top" align="center">4.20E-09</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">ubiquitin-specific protease activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>16</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">119</td><td rowspan="1" colspan="1" valign="top" align="center">1.414</td><td rowspan="1" colspan="1" valign="top" align="center">4.50E-09</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">small conjugating protein-specific protease activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>17</strong></td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">171</td><td rowspan="1" colspan="1" valign="top" align="center">0.776</td><td rowspan="1" colspan="1" valign="top" align="center">5.90E-09</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;0.001</td><td rowspan="1" colspan="1" valign="top" align="left">Rho protein signal transduction</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>18</strong></td><td rowspan="1" colspan="1" valign="top" align="center">100</td><td rowspan="1" colspan="1" valign="top" align="center">6558</td><td rowspan="1" colspan="1" valign="top" align="center">0.316</td><td rowspan="1" colspan="1" valign="top" align="center">3.30E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="left">intrinsic to membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>19</strong></td><td rowspan="1" colspan="1" valign="top" align="center">16</td><td rowspan="1" colspan="1" valign="top" align="center">131</td><td rowspan="1" colspan="1" valign="top" align="center">0.802</td><td rowspan="1" colspan="1" valign="top" align="center">4.50E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.002</td><td rowspan="1" colspan="1" valign="top" align="left">Rho guanyl-nucleotide exchange factor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>20</strong></td><td rowspan="1" colspan="1" valign="top" align="center">275</td><td rowspan="1" colspan="1" valign="top" align="center">8319</td><td rowspan="1" colspan="1" valign="top" align="center">0.183</td><td rowspan="1" colspan="1" valign="top" align="center">8.10E-08</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">protein binding</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>21</strong></td><td rowspan="1" colspan="1" valign="top" align="center">35</td><td rowspan="1" colspan="1" valign="top" align="center">558</td><td rowspan="1" colspan="1" valign="top" align="center">0.475</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">cytoskeletal protein binding</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>22</strong></td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">140</td><td rowspan="1" colspan="1" valign="top" align="center">0.802</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">regulation of Rho protein signal transduction</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>23</strong></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">192</td><td rowspan="1" colspan="1" valign="top" align="center">0.979</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">ATPase activity, coupled to transmembrane movement of substances</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>24</strong></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">193</td><td rowspan="1" colspan="1" valign="top" align="center">0.976</td><td rowspan="1" colspan="1" valign="top" align="center">1.20E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">ATPase activity, coupled to movement of substances</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>25</strong></td><td rowspan="1" colspan="1" valign="top" align="center">521</td><td rowspan="1" colspan="1" valign="top" align="center">19305</td><td rowspan="1" colspan="1" valign="top" align="center">0.204</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">cell part</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>26</strong></td><td rowspan="1" colspan="1" valign="top" align="center">521</td><td rowspan="1" colspan="1" valign="top" align="center">19306</td><td rowspan="1" colspan="1" valign="top" align="center">0.204</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">cell</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>27</strong></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">197</td><td rowspan="1" colspan="1" valign="top" align="center">0.967</td><td rowspan="1" colspan="1" valign="top" align="center">1.40E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">hydrolase activity, acting on acid anhydrides, catalyzing transmembrane movement of<br>substances</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>28</strong></td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">143</td><td rowspan="1" colspan="1" valign="top" align="center">0.792</td><td rowspan="1" colspan="1" valign="top" align="center">1.50E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">Ras guanyl-nucleotide exchange factor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>29</strong></td><td rowspan="1" colspan="1" valign="top" align="center">31</td><td rowspan="1" colspan="1" valign="top" align="center">2607</td><td rowspan="1" colspan="1" valign="top" align="center">0.533</td><td rowspan="1" colspan="1" valign="top" align="center">1.50E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">protein complex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>30</strong></td><td rowspan="1" colspan="1" valign="top" align="center">50</td><td rowspan="1" colspan="1" valign="top" align="center">909</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">1.80E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">cell adhesion</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>31</strong></td><td rowspan="1" colspan="1" valign="top" align="center">50</td><td rowspan="1" colspan="1" valign="top" align="center">909</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">1.80E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">biological adhesion</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>32</strong></td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">1.551</td><td rowspan="1" colspan="1" valign="top" align="center">2.00E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">prostaglandin E receptor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>33</strong></td><td rowspan="1" colspan="1" valign="top" align="center">64</td><td rowspan="1" colspan="1" valign="top" align="center">1301</td><td rowspan="1" colspan="1" valign="top" align="center">0.328</td><td rowspan="1" colspan="1" valign="top" align="center">2.40E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.003</td><td rowspan="1" colspan="1" valign="top" align="left">intrinsic to plasma membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>34</strong></td><td rowspan="1" colspan="1" valign="top" align="center">96</td><td rowspan="1" colspan="1" valign="top" align="center">6449</td><td rowspan="1" colspan="1" valign="top" align="center">0.297</td><td rowspan="1" colspan="1" valign="top" align="center">2.50E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">integral to membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>35</strong></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">209</td><td rowspan="1" colspan="1" valign="top" align="center">0.939</td><td rowspan="1" colspan="1" valign="top" align="center">2.60E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">primary active transmembrane transporter activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>36</strong></td><td rowspan="1" colspan="1" valign="top" align="center">11</td><td rowspan="1" colspan="1" valign="top" align="center">209</td><td rowspan="1" colspan="1" valign="top" align="center">0.939</td><td rowspan="1" colspan="1" valign="top" align="center">2.60E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.004</td><td rowspan="1" colspan="1" valign="top" align="left">P-P-bond-hydrolysis-driven transmembrane transporter activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>37</strong></td><td rowspan="1" colspan="1" valign="top" align="center">63</td><td rowspan="1" colspan="1" valign="top" align="center">1285</td><td rowspan="1" colspan="1" valign="top" align="center">0.326</td><td rowspan="1" colspan="1" valign="top" align="center">3.30E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.005</td><td rowspan="1" colspan="1" valign="top" align="left">integral to plasma membrane</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>38</strong></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">2.561</td><td rowspan="1" colspan="1" valign="top" align="center">4.30E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.007</td><td rowspan="1" colspan="1" valign="top" align="left">sulfonylurea receptor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>39</strong></td><td rowspan="1" colspan="1" valign="top" align="center">27</td><td rowspan="1" colspan="1" valign="top" align="center">390</td><td rowspan="1" colspan="1" valign="top" align="center">0.511</td><td rowspan="1" colspan="1" valign="top" align="center">6.50E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="left">proteinaceous extracellular matrix</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>40</strong></td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">1.457</td><td rowspan="1" colspan="1" valign="top" align="center">6.90E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="left">cell adhesion molecule binding</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>41</strong></td><td rowspan="1" colspan="1" valign="top" align="center">32</td><td rowspan="1" colspan="1" valign="top" align="center">1350</td><td rowspan="1" colspan="1" valign="top" align="center">0.468</td><td rowspan="1" colspan="1" valign="top" align="center">6.90E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="left">transmembrane transporter activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>42</strong></td><td rowspan="1" colspan="1" valign="top" align="center">17</td><td rowspan="1" colspan="1" valign="top" align="center">577</td><td rowspan="1" colspan="1" valign="top" align="center">0.677</td><td rowspan="1" colspan="1" valign="top" align="center">7.00E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.012</td><td rowspan="1" colspan="1" valign="top" align="left">active transmembrane transporter activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>43</strong></td><td rowspan="1" colspan="1" valign="top" align="center">3</td><td rowspan="1" colspan="1" valign="top" align="center">4</td><td rowspan="1" colspan="1" valign="top" align="center">2.603</td><td rowspan="1" colspan="1" valign="top" align="center">7.90E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">cGMP-dependent protein kinase activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>44</strong></td><td rowspan="1" colspan="1" valign="top" align="center">58</td><td rowspan="1" colspan="1" valign="top" align="center">1262</td><td rowspan="1" colspan="1" valign="top" align="center">0.329</td><td rowspan="1" colspan="1" valign="top" align="center">7.90E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.017</td><td rowspan="1" colspan="1" valign="top" align="left">calcium ion binding</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>45</strong></td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">258</td><td rowspan="1" colspan="1" valign="top" align="center">0.622</td><td rowspan="1" colspan="1" valign="top" align="center">9.10E-07</td><td rowspan="1" colspan="1" valign="top" align="center">0.018</td><td rowspan="1" colspan="1" valign="top" align="left">guanyl-nucleotide exchange factor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>46</strong></td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="center">19</td><td rowspan="1" colspan="1" valign="top" align="center">1.398</td><td rowspan="1" colspan="1" valign="top" align="center">1.00E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="left">prostaglandin receptor activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>47</strong></td><td rowspan="1" colspan="1" valign="top" align="center">27</td><td rowspan="1" colspan="1" valign="top" align="center">400</td><td rowspan="1" colspan="1" valign="top" align="center">0.499</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="left">extracellular matrix</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>48</strong></td><td rowspan="1" colspan="1" valign="top" align="center">41</td><td rowspan="1" colspan="1" valign="top" align="center">3215</td><td rowspan="1" colspan="1" valign="top" align="center">0.416</td><td rowspan="1" colspan="1" valign="top" align="center">1.10E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.019</td><td rowspan="1" colspan="1" valign="top" align="left">macromolecular complex</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>49</strong></td><td rowspan="1" colspan="1" valign="top" align="center">34</td><td rowspan="1" colspan="1" valign="top" align="center">589</td><td rowspan="1" colspan="1" valign="top" align="center">0.433</td><td rowspan="1" colspan="1" valign="top" align="center">1.20E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.022</td><td rowspan="1" colspan="1" valign="top" align="left">metal ion transport</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>50</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">246</td><td rowspan="1" colspan="1" valign="top" align="center">1.082</td><td rowspan="1" colspan="1" valign="top" align="center">1.20E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.022</td><td rowspan="1" colspan="1" valign="top" align="left">cysteine-type peptidase activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>51</strong></td><td rowspan="1" colspan="1" valign="top" align="center">136</td><td rowspan="1" colspan="1" valign="top" align="center">3975</td><td rowspan="1" colspan="1" valign="top" align="center">0.212</td><td rowspan="1" colspan="1" valign="top" align="center">1.30E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.023</td><td rowspan="1" colspan="1" valign="top" align="left">localization</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>52</strong></td><td rowspan="1" colspan="1" valign="top" align="center">14</td><td rowspan="1" colspan="1" valign="top" align="center">409</td><td rowspan="1" colspan="1" valign="top" align="center">0.733</td><td rowspan="1" colspan="1" valign="top" align="center">1.60E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.027</td><td rowspan="1" colspan="1" valign="top" align="left">ATPase activity, coupled</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>53</strong></td><td rowspan="1" colspan="1" valign="top" align="center">45</td><td rowspan="1" colspan="1" valign="top" align="center">1158</td><td rowspan="1" colspan="1" valign="top" align="center">0.366</td><td rowspan="1" colspan="1" valign="top" align="center">1.60E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="left">ion transmembrane transporter activity</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>54</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">258</td><td rowspan="1" colspan="1" valign="top" align="center">1.06</td><td rowspan="1" colspan="1" valign="top" align="center">1.80E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="left">ubiquitin-dependent protein catabolic process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>55</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">259</td><td rowspan="1" colspan="1" valign="top" align="center">1.059</td><td rowspan="1" colspan="1" valign="top" align="center">1.80E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="left">modification-dependent protein catabolic process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>56</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">259</td><td rowspan="1" colspan="1" valign="top" align="center">1.059</td><td rowspan="1" colspan="1" valign="top" align="center">1.80E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="left">modification-dependent macromolecule catabolic process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>57</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">260</td><td rowspan="1" colspan="1" valign="top" align="center">1.057</td><td rowspan="1" colspan="1" valign="top" align="center">1.90E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.029</td><td rowspan="1" colspan="1" valign="top" align="left">proteolysis involved in cellular protein catabolic process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>58</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">262</td><td rowspan="1" colspan="1" valign="top" align="center">1.053</td><td rowspan="1" colspan="1" valign="top" align="center">2.00E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="left">cellular protein catabolic process</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>59</strong></td><td rowspan="1" colspan="1" valign="top" align="center">15</td><td rowspan="1" colspan="1" valign="top" align="center">152</td><td rowspan="1" colspan="1" valign="top" align="center">0.688</td><td rowspan="1" colspan="1" valign="top" align="center">2.50E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.046</td><td rowspan="1" colspan="1" valign="top" align="left">calcium ion transport</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center"><strong>60</strong></td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">42</td><td rowspan="1" colspan="1" valign="top" align="center">1.057</td><td rowspan="1" colspan="1" valign="top" align="center">2.60E-06</td><td rowspan="1" colspan="1" valign="top" align="center">0.047</td><td rowspan="1" colspan="1" valign="top" align="left">voltage-gated calcium channel activity</td></tr></tbody></table></div><div id="largeobj_idm140338127221776" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312985/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p10"><strong>Key</strong>: Rank: Position in the attribute list ranked by significant of association with the query, N: Number of genes that went into the query with this attribute, X: Number of genes overall in the ontology database with this attribute, LOD: logarithm (base 10) of the odds ratio, p-val: Single hypothesis p-value of the association between attribute and query (based on Fisher’s exact test), p-adj: Adjusted p-value; fraction of 1000 null-hypothesis simulations having attributes with this single hypothesis p-value or smaller</p></div></div></div><div id="S12" class="sec sec-last"><h3 id="S12title">Supplementary Association analysis of top genes from Para-ICA</h3><p id="P23" class="p p-first-last">Association analysis (to illustrate their relative disease association) of allelic frequencies for the top ten genes (based on ranked Z-scores) from the four genetic components revealed that the genes <strong>SLC9A7, SHROOM2, ZNF673, APOE (</strong>ε3, ε4), <strong>VPS13C</strong> and <strong>ATP5G2</strong> showed stronger effects of disease associations compared to other genes within the component. <a href="/pmc/articles/PMC3312985/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> details the allelic frequency and the associated statistical value for each of the top 10 genes from all four genetic networks; due to partial overlap (1 gene could appear in more than 1 component) totaling 32 unique genes. A weighting score was derived by normalizing the chi-square value of each gene to the chi-square of the gene most associated with clinical disease status within each component.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p11">Results of association analysis performed on the top ten genes from each genetic network sorted in descending order of their weighting scores. Weighting scores represent the relative effect of each SNPs association with disease status within a particular component, ranging from 0-1. A &amp; B refer to the different allelic frequencies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338127402608" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left">Gene</th><th rowspan="1" colspan="1" valign="middle" align="left">Locati<br>on</th><th rowspan="1" colspan="1" valign="middle" align="left">Functio<br>n</th><th rowspan="1" colspan="1" valign="middle" align="left">rs #</th><th colspan="2" rowspan="1" valign="middle" align="right">AD<br>(#<br>Alleles)</th><th colspan="2" rowspan="1" valign="middle" align="right">MCI<br>(#<br>Alleles)</th><th colspan="2" rowspan="1" valign="middle" align="right">HC<br>(#<br>Alleles)</th><th rowspan="1" colspan="1" valign="middle" align="right">- <em>N</em><br>square</th><th rowspan="1" colspan="1" valign="middle" align="center">p-<br>value</th><th rowspan="1" colspan="1" valign="middle" align="center">Weighti<br>ng Score</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">G1</th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right">A</th><th rowspan="1" colspan="1" valign="middle" align="right">B</th><th rowspan="1" colspan="1" valign="middle" align="right">A</th><th rowspan="1" colspan="1" valign="middle" align="right">B</th><th rowspan="1" colspan="1" valign="middle" align="right">A</th><th rowspan="1" colspan="1" valign="middle" align="right">B</th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>APOE3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>19q13.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>N/A</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>E3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">162</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">269</td><td rowspan="1" colspan="1" valign="middle" align="right">46<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">39</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">88</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00E-<br>16</td><td rowspan="1" colspan="1" valign="middle" align="center">1.000</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>APOE4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>19q13.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>N/A</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>E4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">209</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">521</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">358</td><td rowspan="1" colspan="1" valign="middle" align="right">60</td><td rowspan="1" colspan="1" valign="middle" align="right">80</td><td rowspan="1" colspan="1" valign="middle" align="center">1.00E-<br>15</td><td rowspan="1" colspan="1" valign="middle" align="center">0.909</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SHROO</em></strong><br><strong><em>M2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xp22.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs240594</em></strong><br><strong><em>0</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">209</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">503</td><td rowspan="1" colspan="1" valign="middle" align="right">23<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">294</td><td rowspan="1" colspan="1" valign="middle" align="right">12<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">16.58</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0003<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.188</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ZNF673</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xp11.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs787630</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">103</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">250</td><td rowspan="1" colspan="1" valign="middle" align="right">48<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">172</td><td rowspan="1" colspan="1" valign="middle" align="right">24<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">14.04</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0009<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.160</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC9A7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xp11.2</em></strong><br><strong><em>3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs188325</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">99</td><td rowspan="1" colspan="1" valign="middle" align="right">26<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">241</td><td rowspan="1" colspan="1" valign="middle" align="right">49<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">165</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">13</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0015<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.148</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC16A</em></strong><br><strong><em>2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xq13.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs479640</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">89</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">252</td><td rowspan="1" colspan="1" valign="middle" align="right">48<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">138</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">11.15</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0038</td><td rowspan="1" colspan="1" valign="middle" align="center">0.127</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>MAGI3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>1p13.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs755295</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">154</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">373</td><td rowspan="1" colspan="1" valign="middle" align="right">36<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">226</td><td rowspan="1" colspan="1" valign="middle" align="right">19<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">10.97</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0041</td><td rowspan="1" colspan="1" valign="middle" align="center">0.125</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>COBL</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>7p12.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs143749</em></strong><br><strong><em>0</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">195</td><td rowspan="1" colspan="1" valign="middle" align="right">16<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">374</td><td rowspan="1" colspan="1" valign="middle" align="right">36<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">255</td><td rowspan="1" colspan="1" valign="middle" align="right">16<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">10.9</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0043</td><td rowspan="1" colspan="1" valign="middle" align="center">0.124</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>PLXDC2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>10p12.3</em></strong><br><strong><em>1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs235883</em></strong><br><strong><em>9</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">102</td><td rowspan="1" colspan="1" valign="middle" align="right">26<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">265</td><td rowspan="1" colspan="1" valign="middle" align="right">46<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">150</td><td rowspan="1" colspan="1" valign="middle" align="right">26<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">7.54</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0229</td><td rowspan="1" colspan="1" valign="middle" align="center">0.086</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CTNND2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>5p15.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs757459</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">215</td><td rowspan="1" colspan="1" valign="middle" align="right">14<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">388</td><td rowspan="1" colspan="1" valign="middle" align="right">34<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">210</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">6.93</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0311</td><td rowspan="1" colspan="1" valign="middle" align="center">0.079</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><strong>G2</strong></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ZNF673</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xp11.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs787630</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">103</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">250</td><td rowspan="1" colspan="1" valign="middle" align="right">48<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">172</td><td rowspan="1" colspan="1" valign="middle" align="right">24<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">14.04</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0009<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.000</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC9A7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Xp11.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs188325</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">99</td><td rowspan="1" colspan="1" valign="middle" align="right">26<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">241</td><td rowspan="1" colspan="1" valign="middle" align="right">49<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">165</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">13</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0015<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.926</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>TNXB</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>inron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs223968</em></strong><br><strong><em>9</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">133</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">213</td><td rowspan="1" colspan="1" valign="middle" align="right">52<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">114</td><td rowspan="1" colspan="1" valign="middle" align="right">30<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">9.47</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0088</td><td rowspan="1" colspan="1" valign="middle" align="center">0.675</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ZSCAN1</em></strong><br><strong><em>2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p22.1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>Mis-</em></strong><br><strong><em>sense</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs136138</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">260</td><td rowspan="1" colspan="1" valign="middle" align="right">10<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">497</td><td rowspan="1" colspan="1" valign="middle" align="right">23<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">258</td><td rowspan="1" colspan="1" valign="middle" align="right">16<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">9.24</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0098</td><td rowspan="1" colspan="1" valign="middle" align="center">0.658</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>EHMT2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs284445</em></strong><br><strong><em>7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">146</td><td rowspan="1" colspan="1" valign="middle" align="right">21<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">236</td><td rowspan="1" colspan="1" valign="middle" align="right">49<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">131</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">8.9</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0116</td><td rowspan="1" colspan="1" valign="middle" align="center">0.634</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ZKSCAN</em></strong><br><strong><em>3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p22.1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs213228</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">258</td><td rowspan="1" colspan="1" valign="middle" align="right">10<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">491</td><td rowspan="1" colspan="1" valign="middle" align="right">24<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">258</td><td rowspan="1" colspan="1" valign="middle" align="right">16<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">8.03</td><td rowspan="1" colspan="1" valign="middle" align="center">0.018</td><td rowspan="1" colspan="1" valign="middle" align="center">0.572</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC44A</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs273642</em></strong><br><strong><em>8</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">139</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">227</td><td rowspan="1" colspan="1" valign="middle" align="right">50<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">127</td><td rowspan="1" colspan="1" valign="middle" align="right">29<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">7.41</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0246</td><td rowspan="1" colspan="1" valign="middle" align="center">0.528</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CFB</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs415165</em></strong><br><strong><em>7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">223</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">509</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">286</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">6.96</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0307</td><td rowspan="1" colspan="1" valign="middle" align="center">0.496</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>C6orf15</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs313097</em></strong><br><strong><em>7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">259</td><td rowspan="1" colspan="1" valign="middle" align="right">10<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">504</td><td rowspan="1" colspan="1" valign="middle" align="right">23<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">264</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">6.7</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0349</td><td rowspan="1" colspan="1" valign="middle" align="center">0.477</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CCHCR1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs746647</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">256</td><td rowspan="1" colspan="1" valign="middle" align="right">10<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">490</td><td rowspan="1" colspan="1" valign="middle" align="right">24<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">261</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">6</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0496</td><td rowspan="1" colspan="1" valign="middle" align="center">0.427</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><strong>G</strong><br><strong>3</strong></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ATP5G2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>12q13.1</em></strong><br><strong><em>3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs180063</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">111</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">166</td><td rowspan="1" colspan="1" valign="middle" align="right">56<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">134</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">14.97</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0006<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.000</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>VPS13C</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>15q22.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs198191</em></strong><br><strong><em>6</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">190</td><td rowspan="1" colspan="1" valign="middle" align="right">17<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">449</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">216</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">12.84</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0016<br><sup>**</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.858</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ATF7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>12q13.1</em></strong><br><strong><em>3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs784568</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">189</td><td rowspan="1" colspan="1" valign="middle" align="right">17<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">337</td><td rowspan="1" colspan="1" valign="middle" align="right">39<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">230</td><td rowspan="1" colspan="1" valign="middle" align="right">18<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">9.82</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0073</td><td rowspan="1" colspan="1" valign="middle" align="center">0.656</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CEP57</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>11q21</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs301775</em></strong><br><strong><em>6</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">154</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">316</td><td rowspan="1" colspan="1" valign="middle" align="right">41<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">215</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">8.95</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0113</td><td rowspan="1" colspan="1" valign="middle" align="center">0.598</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>EHMT2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs284445</em></strong><br><strong><em>7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">146</td><td rowspan="1" colspan="1" valign="middle" align="right">21<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">236</td><td rowspan="1" colspan="1" valign="middle" align="right">49<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">131</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">8.9</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0116</td><td rowspan="1" colspan="1" valign="middle" align="center">0.595</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>MTMR2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>11q21</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rS193364</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">225</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">437</td><td rowspan="1" colspan="1" valign="middle" align="right">29<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">219</td><td rowspan="1" colspan="1" valign="middle" align="right">19<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">8.66</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0132</td><td rowspan="1" colspan="1" valign="middle" align="center">0.578</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>PTCHD2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>1p36.2 2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs237913</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">160</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">259</td><td rowspan="1" colspan="1" valign="middle" align="right">47<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">165</td><td rowspan="1" colspan="1" valign="middle" align="right">25<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">8.32</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0156</td><td rowspan="1" colspan="1" valign="middle" align="center">0.556</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CNTN5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>11q22.1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs108929</em></strong><br><strong><em>01</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">134</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">314</td><td rowspan="1" colspan="1" valign="middle" align="right">42<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">146</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">7.86</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0196</td><td rowspan="1" colspan="1" valign="middle" align="center">0.525</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC44A</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3 2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs273642</em></strong><br><strong><em>8</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">139</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">227</td><td rowspan="1" colspan="1" valign="middle" align="right">50<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">127</td><td rowspan="1" colspan="1" valign="middle" align="right">29<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">7.41</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0246</td><td rowspan="1" colspan="1" valign="middle" align="center">0.495</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>C2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs273433</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">204</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">355</td><td rowspan="1" colspan="1" valign="middle" align="right">37<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">211</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">6.22</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0445</td><td rowspan="1" colspan="1" valign="middle" align="center">0.415</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"><strong>G4</strong></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="left"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="right"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SKIV2L</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs592229</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">227</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">388</td><td rowspan="1" colspan="1" valign="middle" align="right">34<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">239</td><td rowspan="1" colspan="1" valign="middle" align="right">17<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">9.69</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0078</td><td rowspan="1" colspan="1" valign="middle" align="center">0.999</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>TNXB</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6P21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs223968</em></strong><br><strong><em>9</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">133</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">213</td><td rowspan="1" colspan="1" valign="middle" align="right">52<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">114</td><td rowspan="1" colspan="1" valign="middle" align="right">30<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">9.47</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0088</td><td rowspan="1" colspan="1" valign="middle" align="center">0.976</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>EHMT2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rS159445</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">268</td><td rowspan="1" colspan="1" valign="middle" align="right">94</td><td rowspan="1" colspan="1" valign="middle" align="right">486</td><td rowspan="1" colspan="1" valign="middle" align="right">24<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">277</td><td rowspan="1" colspan="1" valign="middle" align="right">14<br>1</td><td rowspan="1" colspan="1" valign="middle" align="right">7.71</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0211</td><td rowspan="1" colspan="1" valign="middle" align="center">0.795</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>AKAP9</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>7q21.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs756647</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">234</td><td rowspan="1" colspan="1" valign="middle" align="right">12<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">461</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">234</td><td rowspan="1" colspan="1" valign="middle" align="right">18<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">7.39</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0248</td><td rowspan="1" colspan="1" valign="middle" align="center">0.762</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>SLC44A</em></strong><br><strong><em>4</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs224266</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">227</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">397</td><td rowspan="1" colspan="1" valign="middle" align="right">33<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">238</td><td rowspan="1" colspan="1" valign="middle" align="right">18<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">7.34</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0254</td><td rowspan="1" colspan="1" valign="middle" align="center">0.757</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CFB</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs415165</em></strong><br><strong><em>7</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">223</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">509</td><td rowspan="1" colspan="1" valign="middle" align="right">22<br>5</td><td rowspan="1" colspan="1" valign="middle" align="right">286</td><td rowspan="1" colspan="1" valign="middle" align="right">13<br>2</td><td rowspan="1" colspan="1" valign="middle" align="right">6.96</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0307</td><td rowspan="1" colspan="1" valign="middle" align="center">0.718</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>ATF6B</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p12.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs222862</em></strong><br><strong><em>8</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">242</td><td rowspan="1" colspan="1" valign="middle" align="right">12<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">526</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">315</td><td rowspan="1" colspan="1" valign="middle" align="right">10<br>3</td><td rowspan="1" colspan="1" valign="middle" align="right">6.9</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0316</td><td rowspan="1" colspan="1" valign="middle" align="center">0.711</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>C6orf10</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs485774</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">203</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">455</td><td rowspan="1" colspan="1" valign="middle" align="right">27<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">272</td><td rowspan="1" colspan="1" valign="middle" align="right">14<br>6</td><td rowspan="1" colspan="1" valign="middle" align="right">6.81</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0331</td><td rowspan="1" colspan="1" valign="middle" align="center">0.702</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>CYP51A</em></strong><br><strong><em>1</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>7q21.2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs230155</em></strong><br><strong><em>9</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">233</td><td rowspan="1" colspan="1" valign="middle" align="right">12<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">454</td><td rowspan="1" colspan="1" valign="middle" align="right">28<br>0</td><td rowspan="1" colspan="1" valign="middle" align="right">234</td><td rowspan="1" colspan="1" valign="middle" align="right">18<br>4</td><td rowspan="1" colspan="1" valign="middle" align="right">6.34</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0419</td><td rowspan="1" colspan="1" valign="middle" align="center">0.654</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>C2</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>6p21.3</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>intron</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="left"><strong><em>rs273433</em></strong><br><strong><em>5</em></strong></td><td rowspan="1" colspan="1" valign="middle" align="right">204</td><td rowspan="1" colspan="1" valign="middle" align="right">15<br>8</td><td rowspan="1" colspan="1" valign="middle" align="right">355</td><td rowspan="1" colspan="1" valign="middle" align="right">37<br>9</td><td rowspan="1" colspan="1" valign="middle" align="right">211</td><td rowspan="1" colspan="1" valign="middle" align="right">20<br>7</td><td rowspan="1" colspan="1" valign="middle" align="right">6.22</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0445</td><td rowspan="1" colspan="1" valign="middle" align="center">0.641</td></tr></tbody></table></div><div id="largeobj_idm140338127402608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312985/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><sup>**</sup>Genes that significantly discriminate groups after correction for multiple comparisons (Bonferroni-corrected across 40 chi-square tests).</div></div></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Discussion</h2><p id="P24" class="p p-first">As hypothesized, we validated a scaled-up Para-ICA approach to reveal novel interactive genes and pathways for LOAD, thus highlighting one of the primary advantages of Para-ICA which is the use of modest sample sizes compared to conventional GWAS analyses to effectively capture genotype-phenotype relationships. Dominant loading coefficients were contained in all major regions affected by LOAD pathology in the single structural component significantly associated with four different SNP/genetic networks. Seven other structural components were unassociated with other gene components. The genetic components identified included SNPs from APOE4 plus multiple other risk genes (and putatively protective SNPs, e.g. APOE2) either previously identified in LOAD risk (e.g. ATF7 in G3 (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667161">Lin et al., 2006</a>)) or involved in one or more biological processes thought to contribute to LOAD pathology. <a href="/pmc/articles/PMC3312985/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5</span></a> summarizes the involvement of these four genetic risk networks on a LOAD physiologic pathway diagram.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC3312985/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140338122494272" class="figure"><a href="/pmc/articles/PMC3312985/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312985_nihms-348696-f0005.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-348696-f0005.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312985/bin/nihms-348696-f0005.jpg"></a></div><div id="largeobj_idm140338122494272" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312985/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC3312985/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p7">A summary of different genetic components (circled in red) identified in the current study projected onto an Alzheimer’s disease functional interaction pathway (modified from Sleegers K et al., TIGS 2009)</p></div></div></div><p id="P25">The most significant association was between G1-S1. This genetic network had significant loading contributions from a total of 169 different genes (332 SNPs) and correlated negatively with brain network S1, implying increased genetic load is related to decreased brain volume/thickness within the network. This association was notable as the S1 had high loadings from APOE4 (in the top 10 gene loadings) and S1 included regions known to be affected early and severely in LOAD, including entorhinal, middle temporal and prefrontal cortices and hippocampus. More importantly, this genetic network had high loadings from several other genes (SLC9A7/NHE7, ZNF673, SHROOM2) previously unidentified in LOAD pathology. Given that they were part of the same independent genetic component as APOE, this finding both confirms APOE’s established role as an important LOAD risk gene and suggests that these additional SNPs may interact with APOE to influence disease risk, supporting APOE’s role as a LOAD risk factor rather than a direct cause (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304766315">Guerreiro et al., 2010</a>). The protein encoded by SLC9A7 mediates Na+/H+ exchange across cell surface plasma membranes (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667183">Kagami et al., 2008</a>) cycling between the cell surface and intracellular trans-Golgi network and recycling endosomes, which are vital to APP processing (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667171">Marks and Berg, 2010</a>). SLC9A7 co-localizes with actin, implicated in tau formation (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667178">Gallo, 2007</a>). LOAD lymphoblasts show abnormalities modulated by sodium/hydrogen exchanger blockers (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667135">Urcelay et al., 2001</a>). Overall, this genetic network was enriched with genes dominant in cell signaling pathways. Other strongly contributing genes are involved in lipid transport and tau formation (via actin/myosin binding). ZNF673 is associated with X-linked mental retardation, (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667136">Lugtenberg et al., 2006</a>; <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667155">Ramaswamy et al., 2010</a>) and is close (~0.2 MB) to SCL9A7 on Xp11.3.</p><p id="P26">The second most significant association in genotype/phenotype correlation was G2-S1. This correlation was positive. G2 comprised 182 unique genes (377 SNPs). Some top-ranked genes from this component overlapped with those from G1, including ZNF673 and SLC9A7. These genes had a significant differential distribution among diagnostic groups, suggesting a role of actin localization and transcriptional regulation in LOAD. Other top genes from this network involved in important AD-related processes included the complement system, involved in amyloid-beta formation and inflammatory damage (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667150">van Es and van den Berg, 2009</a>).</p><p id="P27">Network G4 correlated positively with S1. G4 contained several genes associated with risk for non-neurologic disorders, including diabetes and cardiovascular disease, both LOAD risk factors (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667193">Profenno et al., 2010</a>). Top genes from this network, previously unidentified in the context of LOAD, belonged to the complement factor/inhibition pathway related to amyloid-beta clearance (35) or are associated with major histocompatibility class III. Additionally, AKAP9, a top 10 gene in this network, maintains neuronal Golgi integrity and is involved in LOAD pathogenesis (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667168">Stieber et al., 1996</a>). Regarding association analysis, no top 10 gene from this network was significantly differentially distributed in the disease groups, suggesting that G4 comprises multiple SNPs of low effect acting together through diverse biological risk pathways, especially inflammation, (see Eikelenboom et al. 2006) to significantly affect LOAD-related neuropathology.</p><p id="P28">The final genotype-phenotype association was a negative correlation between G3 and S1. G3 included ATP5G2, a subunit of mitochondrial ATP-synthase, which was over-represented in the disease group. Mitochondrial ATP-synthase in entorhinal cortex is a target of oxidative stress in LOAD (<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667185">Terni et al., 2010</a>) and part of LOAD apoptosis pathways. Several other G3 genes included dominant signaling from CNTN5, recently associated with multiple AD MRI characteristics (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667189">Biffi et al., 2010</a>), CEP57, a microtubular/centrosomal localizer (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667186">Meunier et al., 2009</a>), MTMR2, an endosomal regulator (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667175">Lee et al., 2010</a>), and ATF7, associated with LOAD in Lin et al. (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667190">Lin et al., 2006</a>). The loading of previously identified LOAD genes and associated pathobiological pathways further supports the relevance of this genetic network.</p><p id="P29">Analyzing significant genes from all four components using DAVID and visualizing related processes on KEGG pathways revealed that genes grouped in multiple LOAD-relevant biological processes (see <a href="/pmc/articles/PMC3312985/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). Additional prominent processes not shown in figure included cellular communication, cardiovascular diseases, signal transduction, calcium signaling, cell adhesion and neuronal developmental processes (e.g. axon guidance). Many such processes are implicated in LOAD pathology (e.g. neuronal calcium signaling (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667137">Kostiuk et al., 2010</a>; <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667125">LaFerla, 2002</a>; <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667124">Mattson and Chan, 2003</a>)). Semaphorin 3A, an axon-guiding membrane protein, accumulates in hippocampus in AD (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667182">Koncina et al., 2007</a>).</p><p id="P30">Major themes deriving from the top 32 <em>Z</em> score-defined genes in the 4 SNP components suggest several major pathophysiological LOAD pathways, especially when such genes co-occurred within a component. From G1, APOE may relate to LOAD risk through pathways not directly linked to amyloid-beta, including actin-related mechanisms. Actin cytoskeletal changes as a path to tau formation (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667180">Gallo, 2007</a>) are implicated across all components by SCLC987/NHE7 (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667146">Kagami et al., 2008</a>; <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667191">Ohgaki et al., 2008</a>), SHROOM2 and COBL (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667156">Dominguez, 2009</a>) and microtubule-related genes including MTMR2, CEP57 and CTNND2 (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667128">Bamburg and Bloom, 2009</a>; <a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667119">Meunier et al., 2009</a>). Three such genes were present in component 1. Immune function, especially the complement system, related to amyloid-beta clearance (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304766314">Guerreiro et al., 2010</a>; <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667169">Kolev et al., 2009</a>) and expressed in cerebrovascular smooth muscle (<a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667176">Walker et al., 2008</a>), is suggested by ATF7, CFB, C2, SKIV2L, C6orf10 and C6orf15 (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667174">Li et al., 2006</a>; <a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304766313">Veerhuis, 2010</a>). These genes support the known role of the complement system in LOAD pathogenesis (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667118">van Es and van den Berg, 2009</a>), while adding new gene candidates, e.g. C2. Complement is present in dystrophic LOAD neurites, involved in immune response and linked to synaptic pruning (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304766312">Hollingworth et al., 2010</a>). Five immune related/complement genes are present in G4.</p><p id="P31">CTNND2/Delta Catenin/NPRAP is associated with GSK3-beta, hence BAP and tau (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_380292458">Bareiss et al., 2010</a>). CNTN5 encodes contactin5; other contactins participate in LOAD risk, (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667173">Biffi et al., 2010</a>; <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667121">Osterfield et al., 2008</a>). The prominence of SCLC987/NHE7(and MTMR2) suggests the importance of the trans-Golgi network and recycling endosome (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667129">Lee et al., 2010</a>). Endosomal processing of APP involving SorLA is of importance in LOAD (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667123">Lin et al., 2005</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667187">Marks and Berg, 2010</a>; <a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667154">Ohgaki et al., 2008</a>). VPS proteins are related to this process (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667197">He et al., 2005</a>; <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667177">Marks and Berg, 2010</a>), although VPS13C has yet to be implicated. VPS13C is associated with maintenance of plasma glucose levels (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667162">Saxena et al., 2010</a>); the related VPS26 is linked with BACE/memapsin2 (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667172">He et al., 2005</a>). CL44A4 is involved in choline uptake (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667127">Jurgensen and Ferreira, 2010</a>). MTMR2 has relevance to excitatory synapses (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667115">Lee et al., 2010</a>). ZKSCAN3/ZNF263 is associated with vascular endothelial growth factor (<a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667194">Yang et al., 2008</a>).</p><p id="P32">The above data suggest involvement of multiple genes influencing varied, complex pathways that might interact mutually to contribute to LOAD. Output from Para-ICA lends itself readily to functional pathway analysis and ultimately systems biology. We also identified novel putative LOAD risk genes, confirmed via testing allelic frequency distributions among disgnostic groups in standard case-control association analyses. It is notable that while none of these genes survived a standard GWAS study, they have high impact when their effect is evaluated in the context of other SNPs. In addition, the SNP components detected several genes previously unknown in the context of LOAD risk, having high Z scores, exceeding those for APOE. Several of these (e.g. SLC9A7, ZNF673, VPS13) were: (a) identified by multiple (up to 17) SNPs, (b) mediate processes plausibly associated with LOAD risk from pathway analyses and prior publications, (c) had SNPs differentially distributed among diagnostic groups and (d) are prominently expressed in brain. These results suggest validity of these novel loci as candidate LOAD risk genes.</p><p id="P33">Examining loading coefficients of the gene and structural networks revealed a stepped response pattern (see <a href="#SD2" rid="SD2" class=" supplementary-material">Fig S2</a>), with MCI values falling between those of healthy control and AD, except for in G3, where they were elevated in MCI compared to AD, suggesting that this gene component may act to either protect against or hasten regional brain deterioration in MCI to influence progression rate to AD.</p><p id="P34">Our study has limitations. Although the Para-ICA method is data driven, we restricted the genetic dataset to a disease-related subset. This focused analysis might fail to uncover genes affecting LOAD pathology via other interactive pathways that may not straightforwardly show group differences. However, since we employed a liberal statistical threshold to limit the genetic dataset to disease-related genes, we were able to include numerous SNPs discarded by conventional univariate studies. Our AD/MCI-focused gene set analysis may not have detected other genetic associations to brain structure. The analysis was carried out only in European-Americans, by far the most numerous ethnicity in the dataset. Future studies could include larger mixed populations. Also, since para-ICA identified multivariate relationships at the gene network level (comprised of linear combinations of SNPs), the directionality and effect magnitude of individual SNPs is not immediately transparent. Our supplementary association analysis to derive the top SNPs might be slightly biased, as they were already pre-selected at a liberal cut-off to be included in the multivariate analysis. Given these limitations and the novelty of our study, our results require further validation and replication in more diverse and larger independent datasets.</p><p id="P35" class="p p-last">In conclusion, we met our major study goals by 1) confirming the feasibility of a hypothesis-blind, multivariate approach to corroborate LOAD genes associated with known pathologic mechanisms and to discover new putative disease-relevant genes that interact but fail individually to reach genome-wide significance. These data thus extend existing GWAS and hypothesis-driven analyses on the same ADNI data set (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667152">Biffi et al., 2010</a>; <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667195">Saykin et al., 2010</a>; <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255667165">Shen et al., 2010</a>). 2) The Para-ICA approach identifies genes in relatively modest sized samples that are plausibly linked collectively in known physiologic pathways, perhaps epistatically and suggests itself as a novel method for exploring other large-scale data sets involving gene and endophenotype information such as BSNIP or COGS (Calkins et al., 2007; Thaker, 2008), in psychotic disorders where the neuropathology and genetic basis are less well-defined than LOAD. Finally, 3) we identified plausible new biological pathways associated with AD neuropathology. Possible therapies resulting from our findings might include agents targeted to the complement and/or immune systems.</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>01</h4><div class="sup-box half_rhythm" id="N0x179f2ca0N0x189c66c0"><a href="/pmc/articles/PMC3312985/bin/NIHMS348696-supplement-01.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(540K, tif)</sup></div></div><div class="sec suppmat" id="SD2"><h4>02</h4><div class="sup-box half_rhythm" id="N0x179f2ca0N0x189c6780"><a href="/pmc/articles/PMC3312985/bin/NIHMS348696-supplement-02.tif" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.3M, tif)</sup></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Acknowledgements</h2><div class="sec"><p id="__p1">The study was supported by the following grants and research support to Dr. Andrew Saykin from Eli Lilly and Company, Siemens AG, Welch Allyn Inc., the NIH (R01 CA101318 [PI], R01 AG19771 [PI], RC2 AG036535 [Core Leader], P30 AG10133-18S1 [Core Leader], and U01 AG032984 [Site PI and Chair, Genetics Working Group]), the Indiana Economic Development Corporation (IEDC #87884), and the Foundation for the NIH, Dr. Vince Calhoun NIH ROIEB005846. We would also like to thank Mrs. Joanna Mounce for assistance with the biological pathways and network analyses used in these studies.</p><p id="__p2">Data collection and sharing for this project was funded by the Alzheimer’s Disease Neuroimaging Initiative (ADNI) (National Institutes of Health Grant U01 AG024904). ADNI is funded by the National Institute on Aging, the National Institute of Biomedical Imaging and Bioengineering, and through generous contributions from the following: Abbott; Alzheimer’s Association; Alzheimer’s Drug Discovery Foundation; Amorfix Life Sciences Ltd.; AstraZeneca; Bayer HealthCare; BioClinica, Inc.; Biogen Idec Inc.; Bristol-Myers Squibb Company; Eisai Inc.; Elan Pharmaceuticals Inc.; Eli Lilly and Company; F. Hoffmann-La Roche Ltd and its affiliated company Genentech, Inc.; GE Healthcare; Innogenetics, N.V.;Janssen Alzheimer Immunotherapy Research &amp; Development, LLC.; Johnson &amp; Johnson Pharmaceutical Research &amp; Development LLC.; Medpace, Inc.; Merck &amp; Co., Inc.; Meso Scale Diagnostics, LLC.; Novartis Pharmaceuticals Corporation; Pfizer Inc.; Servier; Synarc Inc.; and Takeda Pharmaceutical Company. The Canadian Institutes of Health Research is providing funds to support ADNI clinical sites in Canada. Private sector contributions are facilitated by the Foundation for the National Institutes of Health (<a href="http://www.fnih.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.fnih.org</a>). The grantee organization is the Northern California Institute for Research and Education, and the study is coordinated by the Alzheimer’s Disease Cooperative Study at the University of California,San Diego. ADNI data are disseminated by the Laboratory for Neuro Imaging at the University of California, Los Angeles. This research was also supported by NIH grants P30 AG010129, K01 AG030514, and the Dana Foundation.</p></div></div><div id="idm140338127353184" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140338127353184title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="P36" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final citable form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.</p><p></p><p class="fn sec" id="FN2"></p><p id="P37" class="p p-first"><strong>Conflict of Interest:</strong></p><p id="P38" class="p p-last">Dr. Andrew Saykin receives research support from Eli Lilly and Company, Siemens AG, Welch Allyn Inc.</p><p></p></div></div><div id="idm140338127349856" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140338127349856title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Bamburg JR, Bloom GS. Cytoskeletal pathologies of Alzheimer disease. <span><span class="ref-journal">Cell Motil Cytoskeleton. </span>2009;<span class="ref-vol">66</span>:635–649.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2754410/">PMC free article</a>]</span> [<a href="/pubmed/19479823" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Motil+Cytoskeleton&amp;title=Cytoskeletal+pathologies+of+Alzheimer+disease&amp;author=JR+Bamburg&amp;author=GS+Bloom&amp;volume=66&amp;publication_year=2009&amp;pages=635-649&amp;pmid=19479823&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Bareiss S, Kim K, Lu Q. Delta-catenin/NPRAP: A new member of the glycogen synthase kinase-3beta signaling complex that promotes beta-catenin turnover in neurons. <span><span class="ref-journal">J Neurosci Res. </span>2010;<span class="ref-vol">88</span>:2350–2363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3813950/">PMC free article</a>]</span> [<a href="/pubmed/20623542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Res&amp;title=Delta-catenin/NPRAP:+A+new+member+of+the+glycogen+synthase+kinase-3beta+signaling+complex+that+promotes+beta-catenin+turnover+in+neurons&amp;author=S+Bareiss&amp;author=K+Kim&amp;author=Q+Lu&amp;volume=88&amp;publication_year=2010&amp;pages=2350-2363&amp;pmid=20623542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Bekris LM, Yu CE, Bird TD, Tsuang DW. Genetics of Alzheimer disease. <span><span class="ref-journal">J Geriatr Psychiatry Neurol. </span>2010;<span class="ref-vol">23</span>:213–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3044597/">PMC free article</a>]</span> [<a href="/pubmed/21045163" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Geriatr+Psychiatry+Neurol&amp;title=Genetics+of+Alzheimer+disease&amp;author=LM+Bekris&amp;author=CE+Yu&amp;author=TD+Bird&amp;author=DW+Tsuang&amp;volume=23&amp;publication_year=2010&amp;pages=213-227&amp;pmid=21045163&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Biffi A, Anderson CD, Desikan RS, Sabuncu M, Cortellini L, Schmansky N, Salat D, Rosand J. Genetic variation and neuroimaging measures in Alzheimer disease. <span><span class="ref-journal">Arch Neurol. </span>2010;<span class="ref-vol">67</span>:677–685.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2956757/">PMC free article</a>]</span> [<a href="/pubmed/20558387" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=Genetic+variation+and+neuroimaging+measures+in+Alzheimer+disease&amp;author=A+Biffi&amp;author=CD+Anderson&amp;author=RS+Desikan&amp;author=M+Sabuncu&amp;author=L+Cortellini&amp;volume=67&amp;publication_year=2010&amp;pages=677-685&amp;pmid=20558387&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Calhoun VD, Adali T, Pearlson GD, Pekar JJ. A method for making group inferences from functional MRI data using independent component analysis. <span><span class="ref-journal">Hum Brain Mapp. </span>2001;<span class="ref-vol">14</span>:140–151.</span> [<a href="/pubmed/11559959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Brain+Mapp&amp;title=A+method+for+making+group+inferences+from+functional+MRI+data+using+independent+component+analysis&amp;author=VD+Calhoun&amp;author=T+Adali&amp;author=GD+Pearlson&amp;author=JJ+Pekar&amp;volume=14&amp;publication_year=2001&amp;pages=140-151&amp;pmid=11559959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Calhoun VD, Liu J, Adali T. A review of group ICA for fMRI data and ICA for joint inference of imaging, genetic, and ERP data. <span><span class="ref-journal">Neuroimage. </span>2009;<span class="ref-vol">45</span>:S163–172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2651152/">PMC free article</a>]</span> [<a href="/pubmed/19059344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=A+review+of+group+ICA+for+fMRI+data+and+ICA+for+joint+inference+of+imaging,+genetic,+and+ERP+data&amp;author=VD+Calhoun&amp;author=J+Liu&amp;author=T+Adali&amp;volume=45&amp;publication_year=2009&amp;pages=S163-172&amp;pmid=19059344&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Dominguez R. Actin filament nucleation and elongation factors--structure-function relationships. <span><span class="ref-journal">Crit Rev Biochem Mol Biol. </span>2009;<span class="ref-vol">44</span>:351–366.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2787778/">PMC free article</a>]</span> [<a href="/pubmed/19874150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Crit+Rev+Biochem+Mol+Biol&amp;title=Actin+filament+nucleation+and+elongation+factors--structure-function+relationships&amp;author=R+Dominguez&amp;volume=44&amp;publication_year=2009&amp;pages=351-366&amp;pmid=19874150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Eccles D, Tapper W. The influence of common polymorphisms on breast cancer. <span><span class="ref-journal">Cancer Treat Res. </span>2010;<span class="ref-vol">155</span>:15–32.</span> [<a href="/pubmed/20517686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Treat+Res&amp;title=The+influence+of+common+polymorphisms+on+breast+cancer&amp;author=D+Eccles&amp;author=W+Tapper&amp;volume=155&amp;publication_year=2010&amp;pages=15-32&amp;pmid=20517686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Gallo G. Tau is actin up in Alzheimer’s disease. <span><span class="ref-journal">Nat Cell Biol. </span>2007;<span class="ref-vol">9</span>:133–134.</span> [<a href="/pubmed/17268475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Cell+Biol&amp;title=Tau+is+actin+up+in+Alzheimer’s+disease&amp;author=G+Gallo&amp;volume=9&amp;publication_year=2007&amp;pages=133-134&amp;pmid=17268475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Gandhi S, Wood NW. Genome-wide association studies: the key to unlocking neurodegeneration? <span><span class="ref-journal">Nat Neurosci. </span>2010;<span class="ref-vol">13</span>:789–794.</span> [<a href="/pubmed/20581814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Genome-wide+association+studies:+the+key+to+unlocking+neurodegeneration?&amp;author=S+Gandhi&amp;author=NW+Wood&amp;volume=13&amp;publication_year=2010&amp;pages=789-794&amp;pmid=20581814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Gatz M, Mortimer JA, Fratiglioni L, Johansson B, Berg S, Reynolds CA, Pedersen NL. Potentially modifiable risk factors for dementia in identical twins. <span><span class="ref-journal">Alzheimers Dement. </span>2006a;<span class="ref-vol">2</span>:110–117.</span> [<a href="/pubmed/19595867" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Potentially+modifiable+risk+factors+for+dementia+in+identical+twins&amp;author=M+Gatz&amp;author=JA+Mortimer&amp;author=L+Fratiglioni&amp;author=B+Johansson&amp;author=S+Berg&amp;volume=2&amp;publication_year=2006a&amp;pages=110-117&amp;pmid=19595867&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Gatz M, Reynolds CA, Fratiglioni L, Johansson B, Mortimer JA, Berg S, Fiske A, Pedersen NL. Role of genes and environments for explaining Alzheimer disease. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2006b;<span class="ref-vol">63</span>:168–174.</span> [<a href="/pubmed/16461860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=Role+of+genes+and+environments+for+explaining+Alzheimer+disease&amp;author=M+Gatz&amp;author=CA+Reynolds&amp;author=L+Fratiglioni&amp;author=B+Johansson&amp;author=JA+Mortimer&amp;volume=63&amp;publication_year=2006b&amp;pages=168-174&amp;pmid=16461860&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Grupe A, Abraham R, Li Y, Rowland C, Hollingworth P, Morgan A, Jehu L, Segurado R, Stone D, Schadt E, Karnoub M, Nowotny P, Tacey K, Catanese J, Sninsky J, Brayne C, Rubinsztein D, Gill M, Lawlor B, Lovestone S, Holmans P, O’Donovan M, Morris JC, Thal L, Goate A, Owen MJ, Williams J. Evidence for novel susceptibility genes for late-onset Alzheimer’s disease from a genome-wide association study of putative functional variants. <span><span class="ref-journal">Hum Mol Genet. </span>2007;<span class="ref-vol">16</span>:865–873.</span> [<a href="/pubmed/17317784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Evidence+for+novel+susceptibility+genes+for+late-onset+Alzheimer’s+disease+from+a+genome-wide+association+study+of+putative+functional+variants&amp;author=A+Grupe&amp;author=R+Abraham&amp;author=Y+Li&amp;author=C+Rowland&amp;author=P+Hollingworth&amp;volume=16&amp;publication_year=2007&amp;pages=865-873&amp;pmid=17317784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Guerreiro RJ, Gustafson DR, Hardy J. The genetic architecture of Alzheimer’s disease: beyond APP, PSENs and APOE. <span><span class="ref-journal">Neurobiol Aging. </span>2010</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2980860/">PMC free article</a>]</span> [<a href="/pubmed/20594621" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Aging&amp;title=The+genetic+architecture+of+Alzheimer’s+disease:+beyond+APP,+PSENs+and+APOE&amp;author=RJ+Guerreiro&amp;author=DR+Gustafson&amp;author=J+Hardy&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Harold D, Abraham R, Hollingworth P, Sims R, Gerrish A, Hamshere ML, Pahwa JS, Moskvina V, Dowzell K, Williams A, Jones N, Thomas C, Stretton A, Morgan AR, Lovestone S, Powell J, Proitsi P, Lupton MK, Brayne C, Rubinsztein DC, Gill M, Lawlor B, Lynch A, Morgan K, Brown KS, Passmore PA, Craig D, McGuinness B, Todd S, Holmes C, Mann D, Smith AD, Love S, Kehoe PG, Hardy J, Mead S, Fox N, Rossor M, Collinge J, Maier W, Jessen F, Schurmann B, van den Bussche H, Heuser I, Kornhuber J, Wiltfang J, Dichgans M, Frolich L, Hampel H, Hull M, Rujescu D, Goate AM, Kauwe JS, Cruchaga C, Nowotny P, Morris JC, Mayo K, Sleegers K, Bettens K, Engelborghs S, De Deyn PP, Van Broeckhoven C, Livingston G, Bass NJ, Gurling H, McQuillin A, Gwilliam R, Deloukas P, Al-Chalabi A, Shaw CE, Tsolaki M, Singleton AB, Guerreiro R, Muhleisen TW, Nothen MM, Moebus S, Jockel KH, Klopp N, Wichmann HE, Carrasquillo MM, Pankratz VS, Younkin SG, Holmans PA, O’Donovan M, Owen MJ, Williams J. Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1088–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2845877/">PMC free article</a>]</span> [<a href="/pubmed/19734902" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CLU+and+PICALM+associated+with+Alzheimer’s+disease&amp;author=D+Harold&amp;author=R+Abraham&amp;author=P+Hollingworth&amp;author=R+Sims&amp;author=A+Gerrish&amp;volume=41&amp;publication_year=2009&amp;pages=1088-1093&amp;pmid=19734902&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">He X, Li F, Chang WP, Tang J. GGA proteins mediate the recycling pathway of memapsin 2 (BACE) <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:11696–11703.</span> [<a href="/pubmed/15615712" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=GGA+proteins+mediate+the+recycling+pathway+of+memapsin+2+(BACE)&amp;author=X+He&amp;author=F+Li&amp;author=WP+Chang&amp;author=J+Tang&amp;volume=280&amp;publication_year=2005&amp;pages=11696-11703&amp;pmid=15615712&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Hollingworth P, Harold D, Jones L, Owen MJ, Williams J. Alzheimer’s disease genetics: current knowledge and future challenges. <span><span class="ref-journal">Int J Geriatr Psychiatry. </span>2010</span> [<a href="/pubmed/20957767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Geriatr+Psychiatry&amp;title=Alzheimer’s+disease+genetics:+current+knowledge+and+future+challenges&amp;author=P+Hollingworth&amp;author=D+Harold&amp;author=L+Jones&amp;author=MJ+Owen&amp;author=J+Williams&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Jack CR, Jr., Bernstein MA, Borowski BJ, Gunter JL, Fox NC, Thompson PM, Schuff N, Krueger G, Killiany RJ, Decarli CS, Dale AM, Carmichael OW, Tosun D, Weiner MW. Update on the magnetic resonance imaging core of the Alzheimer’s disease neuroimaging initiative. <span><span class="ref-journal">Alzheimers Dement. </span>2010a;<span class="ref-vol">6</span>:212–220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2886577/">PMC free article</a>]</span> [<a href="/pubmed/20451869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Update+on+the+magnetic+resonance+imaging+core+of+the+Alzheimer’s+disease+neuroimaging+initiative&amp;author=CR+Jack&amp;author=MA+Bernstein&amp;author=BJ+Borowski&amp;author=JL+Gunter&amp;author=NC+Fox&amp;volume=6&amp;publication_year=2010a&amp;pages=212-220&amp;pmid=20451869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Jack CR, Jr., Wiste HJ, Vemuri P, Weigand SD, Senjem ML, Zeng G, Bernstein MA, Gunter JL, Pankratz VS, Aisen PS, Weiner MW, Petersen RC, Shaw LM, Trojanowski JQ, Knopman DS. Brain beta-amyloid measures and magnetic resonance imaging atrophy both predict time-to-progression from mild cognitive impairment to Alzheimer’s disease. <span><span class="ref-journal">Brain. </span>2010b;<span class="ref-vol">133</span>:3336–3348.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965425/">PMC free article</a>]</span> [<a href="/pubmed/20935035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Brain+beta-amyloid+measures+and+magnetic+resonance+imaging+atrophy+both+predict+time-to-progression+from+mild+cognitive+impairment+to+Alzheimer’s+disease&amp;author=CR+Jack&amp;author=HJ+Wiste&amp;author=P+Vemuri&amp;author=SD+Weigand&amp;author=ML+Senjem&amp;volume=133&amp;publication_year=2010b&amp;pages=3336-3348&amp;pmid=20935035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Jagannathan K, Calhoun VD, Gelernter J, Stevens MC, Liu J, Bolognani F, Windemuth A, Ruano G, Assaf M, Pearlson GD. Genetic associations of brain structural networks in schizophrenia: a preliminary study. <span><span class="ref-journal">Biol Psychiatry. </span>2010;<span class="ref-vol">68</span>:657–666.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2990476/">PMC free article</a>]</span> [<a href="/pubmed/20691427" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Genetic+associations+of+brain+structural+networks+in+schizophrenia:+a+preliminary+study&amp;author=K+Jagannathan&amp;author=VD+Calhoun&amp;author=J+Gelernter&amp;author=MC+Stevens&amp;author=J+Liu&amp;volume=68&amp;publication_year=2010&amp;pages=657-666&amp;pmid=20691427&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Jurgensen S, Ferreira ST. Nicotinic receptors, amyloid-beta, and synaptic failure in Alzheimer’s disease. <span><span class="ref-journal">J Mol Neurosci. </span>2010;<span class="ref-vol">40</span>:221–229.</span> [<a href="/pubmed/19690986" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Neurosci&amp;title=Nicotinic+receptors,+amyloid-beta,+and+synaptic+failure+in+Alzheimer’s+disease&amp;author=S+Jurgensen&amp;author=ST+Ferreira&amp;volume=40&amp;publication_year=2010&amp;pages=221-229&amp;pmid=19690986&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Kagami T, Chen S, Memar P, Choi M, Foster LJ, Numata M. Identification and biochemical characterization of the SLC9A7 interactome. <span><span class="ref-journal">Mol Membr Biol. </span>2008;<span class="ref-vol">25</span>:436–447.</span> [<a href="/pubmed/18654930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Membr+Biol&amp;title=Identification+and+biochemical+characterization+of+the+SLC9A7+interactome&amp;author=T+Kagami&amp;author=S+Chen&amp;author=P+Memar&amp;author=M+Choi&amp;author=LJ+Foster&amp;volume=25&amp;publication_year=2008&amp;pages=436-447&amp;pmid=18654930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Kolev MV, Ruseva MM, Harris CL, Morgan BP, Donev RM. Implication of complement system and its regulators in Alzheimer’s disease. <span><span class="ref-journal">Curr Neuropharmacol. </span>2009;<span class="ref-vol">7</span>:1–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2724661/">PMC free article</a>]</span> [<a href="/pubmed/19721814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Neuropharmacol&amp;title=Implication+of+complement+system+and+its+regulators+in+Alzheimer’s+disease&amp;author=MV+Kolev&amp;author=MM+Ruseva&amp;author=CL+Harris&amp;author=BP+Morgan&amp;author=RM+Donev&amp;volume=7&amp;publication_year=2009&amp;pages=1-8&amp;pmid=19721814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Koncina E, Roth L, Gonthier B, Bagnard D. Role of semaphorins during axon growth and guidance. <span><span class="ref-journal">Adv Exp Med Biol. </span>2007;<span class="ref-vol">621</span>:50–64.</span> [<a href="/pubmed/18269210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv+Exp+Med+Biol&amp;title=Role+of+semaphorins+during+axon+growth+and+guidance&amp;author=E+Koncina&amp;author=L+Roth&amp;author=B+Gonthier&amp;author=D+Bagnard&amp;volume=621&amp;publication_year=2007&amp;pages=50-64&amp;pmid=18269210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Kostiuk OP, Korol T, Korol SV, Romanenko SV, Pinchenko VO, Kostiuk PH. [Alteration of calcium signaling as one of the mechanisms of Alzheimer’s disease and diabetic polyneuropathy] <span><span class="ref-journal">Fiziol Zh. </span>2010;<span class="ref-vol">56</span>:130–138.</span> [<a href="/pubmed/20968046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fiziol+Zh&amp;title=[Alteration+of+calcium+signaling+as+one+of+the+mechanisms+of+Alzheimer’s+disease+and+diabetic+polyneuropathy]&amp;author=OP+Kostiuk&amp;author=T+Korol&amp;author=SV+Korol&amp;author=SV+Romanenko&amp;author=VO+Pinchenko&amp;volume=56&amp;publication_year=2010&amp;pages=130-138&amp;pmid=20968046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Ku CS, Loy EY, Pawitan Y, Chia KS. The pursuit of genome-wide association studies: where are we now? <span><span class="ref-journal">J Hum Genet. </span>2011;<span class="ref-vol">55</span>:195–206.</span> [<a href="/pubmed/20300123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=The+pursuit+of+genome-wide+association+studies:+where+are+we+now?&amp;author=CS+Ku&amp;author=EY+Loy&amp;author=Y+Pawitan&amp;author=KS+Chia&amp;volume=55&amp;publication_year=2011&amp;pages=195-206&amp;pmid=20300123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">LaFerla FM. Calcium dyshomeostasis and intracellular signalling in Alzheimer’s disease. <span><span class="ref-journal">Nat Rev Neurosci. </span>2002;<span class="ref-vol">3</span>:862–872.</span> [<a href="/pubmed/12415294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurosci&amp;title=Calcium+dyshomeostasis+and+intracellular+signalling+in+Alzheimer’s+disease&amp;author=FM+LaFerla&amp;volume=3&amp;publication_year=2002&amp;pages=862-872&amp;pmid=12415294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Lee HW, Kim Y, Han K, Kim H, Kim E. The phosphoinositide 3-phosphatase MTMR2 interacts with PSD-95 and maintains excitatory synapses by modulating endosomal traffic. <span><span class="ref-journal">J Neurosci. </span>2010;<span class="ref-vol">30</span>:5508–5518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6632333/">PMC free article</a>]</span> [<a href="/pubmed/20410104" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=The+phosphoinositide+3-phosphatase+MTMR2+interacts+with+PSD-95+and+maintains+excitatory+synapses+by+modulating+endosomal+traffic&amp;author=HW+Lee&amp;author=Y+Kim&amp;author=K+Han&amp;author=H+Kim&amp;author=E+Kim&amp;volume=30&amp;publication_year=2010&amp;pages=5508-5518&amp;pmid=20410104&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Li D, Parks SB, Kushner JD, Nauman D, Burgess D, Ludwigsen S, Partain J, Nixon RR, Allen CN, Irwin RP, Jakobs PM, Litt M, Hershberger RE. Mutations of presenilin genes in dilated cardiomyopathy and heart failure. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">79</span>:1030–1039.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1698711/">PMC free article</a>]</span> [<a href="/pubmed/17186461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Mutations+of+presenilin+genes+in+dilated+cardiomyopathy+and+heart+failure&amp;author=D+Li&amp;author=SB+Parks&amp;author=JD+Kushner&amp;author=D+Nauman&amp;author=D+Burgess&amp;volume=79&amp;publication_year=2006&amp;pages=1030-1039&amp;pmid=17186461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Lin PI, Martin ER, Browning-Large CA, Schmechel DE, Welsh-Bohmer KA, Doraiswamy PM, Gilbert JR, Haines JL, Pericak-Vance MA. Parsing the genetic heterogeneity of chromosome 12q susceptibility genes for Alzheimer disease by family-based association analysis. <span><span class="ref-journal">Neurogenetics. </span>2006;<span class="ref-vol">7</span>:157–165.</span> [<a href="/pubmed/16770605" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurogenetics&amp;title=Parsing+the+genetic+heterogeneity+of+chromosome+12q+susceptibility+genes+for+Alzheimer+disease+by+family-based+association+analysis&amp;author=PI+Lin&amp;author=ER+Martin&amp;author=CA+Browning-Large&amp;author=DE+Schmechel&amp;author=KA+Welsh-Bohmer&amp;volume=7&amp;publication_year=2006&amp;pages=157-165&amp;pmid=16770605&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Lin PJ, Williams WP, Luu Y, Molday RS, Orlowski J, Numata M. Secretory carrier membrane proteins interact and regulate trafficking of the organellar (Na+,K+)/H+ exchanger NHE7. <span><span class="ref-journal">J Cell Sci. </span>2005;<span class="ref-vol">118</span>:1885–1897.</span> [<a href="/pubmed/15840657" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Sci&amp;title=Secretory+carrier+membrane+proteins+interact+and+regulate+trafficking+of+the+organellar+(Na+,K+)/H++exchanger+NHE7&amp;author=PJ+Lin&amp;author=WP+Williams&amp;author=Y+Luu&amp;author=RS+Molday&amp;author=J+Orlowski&amp;volume=118&amp;publication_year=2005&amp;pages=1885-1897&amp;pmid=15840657&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Liu J, Demirci O, Calhoun VD. A Parallel Independent Component Analysis Approach to Investigate Genomic Influence on Brain Function. <span><span class="ref-journal">IEEE Signal Process Lett. </span>2008;<span class="ref-vol">15</span>:413–416.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2761666/">PMC free article</a>]</span> [<a href="/pubmed/19834575" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE+Signal+Process+Lett&amp;title=A+Parallel+Independent+Component+Analysis+Approach+to+Investigate+Genomic+Influence+on+Brain+Function&amp;author=J+Liu&amp;author=O+Demirci&amp;author=VD+Calhoun&amp;volume=15&amp;publication_year=2008&amp;pages=413-416&amp;pmid=19834575&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Lugtenberg D, Yntema HG, Banning MJ, Oudakker AR, Firth HV, Willatt L, Raynaud M, Kleefstra T, Fryns JP, Ropers HH, Chelly J, Moraine C, Gecz J, van Reeuwijk J, Nabuurs SB, de Vries BB, Hamel BC, de Brouwer AP, van Bokhoven H. ZNF674: a new kruppel-associated box-containing zinc-finger gene involved in nonsyndromic X-linked mental retardation. <span><span class="ref-journal">Am J Hum Genet. </span>2006;<span class="ref-vol">78</span>:265–278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1380234/">PMC free article</a>]</span> [<a href="/pubmed/16385466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=ZNF674:+a+new+kruppel-associated+box-containing+zinc-finger+gene+involved+in+nonsyndromic+X-linked+mental+retardation&amp;author=D+Lugtenberg&amp;author=HG+Yntema&amp;author=MJ+Banning&amp;author=AR+Oudakker&amp;author=HV+Firth&amp;volume=78&amp;publication_year=2006&amp;pages=265-278&amp;pmid=16385466&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Marks N, Berg MJ. BACE and gamma-secretase characterization and their sorting as therapeutic targets to reduce amyloidogenesis. <span><span class="ref-journal">Neurochem Res. </span>2010;<span class="ref-vol">35</span>:181–210.</span> [<a href="/pubmed/19760173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem+Res&amp;title=BACE+and+gamma-secretase+characterization+and+their+sorting+as+therapeutic+targets+to+reduce+amyloidogenesis&amp;author=N+Marks&amp;author=MJ+Berg&amp;volume=35&amp;publication_year=2010&amp;pages=181-210&amp;pmid=19760173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Mattson MP, Chan SL. Neuronal and glial calcium signaling in Alzheimer’s disease. <span><span class="ref-journal">Cell Calcium. </span>2003;<span class="ref-vol">34</span>:385–397.</span> [<a href="/pubmed/12909083" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Calcium&amp;title=Neuronal+and+glial+calcium+signaling+in+Alzheimer’s+disease&amp;author=MP+Mattson&amp;author=SL+Chan&amp;volume=34&amp;publication_year=2003&amp;pages=385-397&amp;pmid=12909083&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">Meda SA, Jagannathan K, Gelernter J, Calhoun VD, Liu J, Stevens MC, Pearlson GD. A pilot multivariate parallel ICA study to investigate differential linkage between neural networks and genetic profiles in schizophrenia. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:1007–1015.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3968678/">PMC free article</a>]</span> [<a href="/pubmed/19944766" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=A+pilot+multivariate+parallel+ICA+study+to+investigate+differential+linkage+between+neural+networks+and+genetic+profiles+in+schizophrenia&amp;author=SA+Meda&amp;author=K+Jagannathan&amp;author=J+Gelernter&amp;author=VD+Calhoun&amp;author=J+Liu&amp;volume=53&amp;publication_year=2010&amp;pages=1007-1015&amp;pmid=19944766&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Meunier S, Navarro MG, Bossard C, Laurell H, Touriol C, Lacazette E, Prats H. Pivotal role of translokin/CEP57 in the unconventional secretion versus nuclear translocation of FGF2. <span><span class="ref-journal">Traffic. </span>2009;<span class="ref-vol">10</span>:1765–1772.</span> [<a href="/pubmed/19804566" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Pivotal+role+of+translokin/CEP57+in+the+unconventional+secretion+versus+nuclear+translocation+of+FGF2&amp;author=S+Meunier&amp;author=MG+Navarro&amp;author=C+Bossard&amp;author=H+Laurell&amp;author=C+Touriol&amp;volume=10&amp;publication_year=2009&amp;pages=1765-1772&amp;pmid=19804566&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Naj AC, Jun G, Beecham GW, Wang LS, Vardarajan BN, Buros J, Gallins PJ, Buxbaum JD, Jarvik GP, Crane PK, Larson EB, Bird TD, Boeve BF, Graff-Radford NR, De Jager PL, Evans D, Schneider JA, Carrasquillo MM, Ertekin-Taner N, Younkin SG, Cruchaga C, Kauwe JS, Nowotny P, Kramer P, Hardy J, Huentelman MJ, Myers AJ, Barmada MM, Demirci FY, Baldwin CT, Green RC, Rogaeva E, St George-Hyslop P, Arnold SE, Barber R, Beach T, Bigio EH, Bowen JD, Boxer A, Burke JR, Cairns NJ, Carlson CS, Carney RM, Carroll SL, Chui HC, Clark DG, Corneveaux J, Cotman CW, Cummings JL, DeCarli C, DeKosky ST, Diaz-Arrastia R, Dick M, Dickson DW, Ellis WG, Faber KM, Fallon KB, Farlow MR, Ferris S, Frosch MP, Galasko DR, Ganguli M, Gearing M, Geschwind DH, Ghetti B, Gilbert JR, Gilman S, Giordani B, Glass JD, Growdon JH, Hamilton RL, Harrell LE, Head E, Honig LS, Hulette CM, Hyman BT, Jicha GA, Jin LW, Johnson N, Karlawish J, Karydas A, Kaye JA, Kim R, Koo EH, Kowall NW, Lah JJ, Levey AI, Lieberman AP, Lopez OL, Mack WJ, Marson DC, Martiniuk F, Mash DC, Masliah E, McCormick WC, McCurry SM, McDavid AN, McKee AC, Mesulam M, Miller BL, Miller CA, Miller JW, Parisi JE, Perl DP, Peskind E, Petersen RC, Poon WW, Quinn JF, Rajbhandary RA, Raskind M, Reisberg B, Ringman JM, Roberson ED, Rosenberg RN, Sano M, Schneider LS, Seeley W, Shelanski ML, Slifer MA, Smith CD, Sonnen JA, Spina S, Stern RA, Tanzi RE, Trojanowski JQ, Troncoso JC, Van Deerlin VM, Vinters HV, Vonsattel JP, Weintraub S, Welsh-Bohmer KA, Williamson J, Woltjer RL, Cantwell LB, Dombroski BA, Beekly D, Lunetta KL, Martin ER, Kamboh MI, Saykin AJ, Reiman EM, Bennett DA, Morris JC, Montine TJ, Goate AM, Blacker D, Tsuang DW, Hakonarson H, Kukull WA, Foroud TM, Haines JL, Mayeux R, Pericak-Vance MA, Farrer LA, Schellenberg GD. Common variants at MS4A4/MS4A6E, CD2AP, CD33 and EPHA1 are associated with late-onset Alzheimer’s disease. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:436–441.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3090745/">PMC free article</a>]</span> [<a href="/pubmed/21460841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+at+MS4A4/MS4A6E,+CD2AP,+CD33+and+EPHA1+are+associated+with+late-onset+Alzheimer’s+disease&amp;author=AC+Naj&amp;author=G+Jun&amp;author=GW+Beecham&amp;author=LS+Wang&amp;author=BN+Vardarajan&amp;volume=43&amp;publication_year=2011&amp;pages=436-441&amp;pmid=21460841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Ohgaki R, Fukura N, Matsushita M, Mitsui K, Kanazawa H. Cell surface levels of organellar Na+/H+ exchanger isoform 6 are regulated by interaction with RACK1. <span><span class="ref-journal">J Biol Chem. </span>2008;<span class="ref-vol">283</span>:4417–4429.</span> [<a href="/pubmed/18057008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Cell+surface+levels+of+organellar+Na+/H++exchanger+isoform+6+are+regulated+by+interaction+with+RACK1&amp;author=R+Ohgaki&amp;author=N+Fukura&amp;author=M+Matsushita&amp;author=K+Mitsui&amp;author=H+Kanazawa&amp;volume=283&amp;publication_year=2008&amp;pages=4417-4429&amp;pmid=18057008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Osterfield M, Egelund R, Young LM, Flanagan JG. Interaction of amyloid precursor protein with contactins and NgCAM in the retinotectal system. <span><span class="ref-journal">Development. </span>2008;<span class="ref-vol">135</span>:1189–1199.</span> [<a href="/pubmed/18272596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Interaction+of+amyloid+precursor+protein+with+contactins+and+NgCAM+in+the+retinotectal+system&amp;author=M+Osterfield&amp;author=R+Egelund&amp;author=LM+Young&amp;author=JG+Flanagan&amp;volume=135&amp;publication_year=2008&amp;pages=1189-1199&amp;pmid=18272596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Pimplikar SW, Nixon RA, Robakis NK, Shen J, Tsai LH. Amyloid-independent mechanisms in Alzheimer’s disease pathogenesis. <span><span class="ref-journal">J Neurosci. </span>2010;<span class="ref-vol">30</span>:14946–14954.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3426835/">PMC free article</a>]</span> [<a href="/pubmed/21068297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci&amp;title=Amyloid-independent+mechanisms+in+Alzheimer’s+disease+pathogenesis&amp;author=SW+Pimplikar&amp;author=RA+Nixon&amp;author=NK+Robakis&amp;author=J+Shen&amp;author=LH+Tsai&amp;volume=30&amp;publication_year=2010&amp;pages=14946-14954&amp;pmid=21068297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Potkin SG, Guffanti G, Lakatos A, Turner JA, Kruggel F, Fallon JH, Saykin AJ, Orro A, Lupoli S, Salvi E, Weiner M, Macciardi F. Hippocampal atrophy as a quantitative trait in a genome-wide association study identifying novel susceptibility genes for Alzheimer’s disease. <span><span class="ref-journal">PLoS One. </span>2009;<span class="ref-vol">4</span>:e6501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719581/">PMC free article</a>]</span> [<a href="/pubmed/19668339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Hippocampal+atrophy+as+a+quantitative+trait+in+a+genome-wide+association+study+identifying+novel+susceptibility+genes+for+Alzheimer’s+disease&amp;author=SG+Potkin&amp;author=G+Guffanti&amp;author=A+Lakatos&amp;author=JA+Turner&amp;author=F+Kruggel&amp;volume=4&amp;publication_year=2009&amp;pages=e6501&amp;pmid=19668339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Profenno LA, Porsteinsson AP, Faraone SV. Meta-analysis of Alzheimer’s disease risk with obesity, diabetes, and related disorders. <span><span class="ref-journal">Biol Psychiatry. </span>2010;<span class="ref-vol">67</span>:505–512.</span> [<a href="/pubmed/19358976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Meta-analysis+of+Alzheimer’s+disease+risk+with+obesity,+diabetes,+and+related+disorders&amp;author=LA+Profenno&amp;author=AP+Porsteinsson&amp;author=SV+Faraone&amp;volume=67&amp;publication_year=2010&amp;pages=505-512&amp;pmid=19358976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Ramaswamy V, Castillo M, Bolduc FV. Developmental disability: duplication of zinc finger transcription factors 673 and 674. <span><span class="ref-journal">Pediatr Neurol. </span>2010;<span class="ref-vol">43</span>:209–212.</span> [<a href="/pubmed/20691945" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Neurol&amp;title=Developmental+disability:+duplication+of+zinc+finger+transcription+factors+673+and+674&amp;author=V+Ramaswamy&amp;author=M+Castillo&amp;author=FV+Bolduc&amp;volume=43&amp;publication_year=2010&amp;pages=209-212&amp;pmid=20691945&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R45"><span class="element-citation">Saxena R, Hivert MF, Langenberg C, Tanaka T, Pankow JS, Vollenweider P, Lyssenko V, Bouatia-Naji N, Dupuis J, Jackson AU, Kao WH, Li M, Glazer NL, Manning AK, Luan J, Stringham HM, Prokopenko I, Johnson T, Grarup N, Boesgaard TW, Lecoeur C, Shrader P, O’Connell J, Ingelsson E, Couper DJ, Rice K, Song K, Andreasen CH, Dina C, Kottgen A, Le Bacquer O, Pattou F, Taneera J, Steinthorsdottir V, Rybin D, Ardlie K, Sampson M, Qi L, van Hoek M, Weedon MN, Aulchenko YS, Voight BF, Grallert H, Balkau B, Bergman RN, Bielinski SJ, Bonnefond A, Bonnycastle LL, Borch-Johnsen K, Bottcher Y, Brunner E, Buchanan TA, Bumpstead SJ, Cavalcanti-Proenca C, Charpentier G, Chen YD, Chines PS, Collins FS, Cornelis M, G JC, Delplanque J, Doney A, Egan JM, Erdos MR, Firmann M, Forouhi NG, Fox CS, Goodarzi MO, Graessler J, Hingorani A, Isomaa B, Jorgensen T, Kivimaki M, Kovacs P, Krohn K, Kumari M, Lauritzen T, Levy-Marchal C, Mayor V, McAteer JB, Meyre D, Mitchell BD, Mohlke KL, Morken MA, Narisu N, Palmer CN, Pakyz R, Pascoe L, Payne F, Pearson D, Rathmann W, Sandbaek A, Sayer AA, Scott LJ, Sharp SJ, Sijbrands E, Singleton A, Siscovick DS, Smith NL, Sparso T, Swift AJ, Syddall H, Thorleifsson G, Tonjes A, Tuomi T, Tuomilehto J, Valle TT, Waeber G, Walley A, Waterworth DM, Zeggini E, Zhao JH, Illig T, Wichmann HE, Wilson JF, van Duijn C, Hu FB, Morris AD, Frayling TM, Hattersley AT, Thorsteinsdottir U, Stefansson K, Nilsson P, Syvanen AC, Shuldiner AR, Walker M, Bornstein SR, Schwarz P, Williams GH, Nathan DM, Kuusisto J, Laakso M, Cooper C, Marmot M, Ferrucci L, Mooser V, Stumvoll M, Loos RJ, Altshuler D, Psaty BM, Rotter JI, Boerwinkle E, Hansen T, Pedersen O, Florez JC, McCarthy MI, Boehnke M, Barroso I, Sladek R, Froguel P, Meigs JB, Groop L, Wareham NJ, Watanabe RM. Genetic variation in GIPR influences the glucose and insulin responses to an oral glucose challenge. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:142–148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922003/">PMC free article</a>]</span> [<a href="/pubmed/20081857" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genetic+variation+in+GIPR+influences+the+glucose+and+insulin+responses+to+an+oral+glucose+challenge&amp;author=R+Saxena&amp;author=MF+Hivert&amp;author=C+Langenberg&amp;author=T+Tanaka&amp;author=JS+Pankow&amp;volume=42&amp;publication_year=2010&amp;pages=142-148&amp;pmid=20081857&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">Saykin AJ, Shen L, Foroud TM, Potkin SG, Swaminathan S, Kim S, Risacher SL, Nho K, Huentelman MJ, Craig DW, Thompson PM, Stein JL, Moore JH, Farrer LA, Green RC, Bertram L, Jack CR, Jr., Weiner MW. Alzheimer’s Disease Neuroimaging Initiative biomarkers as quantitative phenotypes: Genetics core aims, progress, and plans. <span><span class="ref-journal">Alzheimers Dement. </span>2010;<span class="ref-vol">6</span>:265–273.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2868595/">PMC free article</a>]</span> [<a href="/pubmed/20451875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alzheimers+Dement&amp;title=Alzheimer’s+Disease+Neuroimaging+Initiative+biomarkers+as+quantitative+phenotypes:+Genetics+core+aims,+progress,+and+plans&amp;author=AJ+Saykin&amp;author=L+Shen&amp;author=TM+Foroud&amp;author=SG+Potkin&amp;author=S+Swaminathan&amp;volume=6&amp;publication_year=2010&amp;pages=265-273&amp;pmid=20451875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Seshadri S, Fitzpatrick AL, Ikram MA, DeStefano AL, Gudnason V, Boada M, Bis JC, Smith AV, Carassquillo MM, Lambert JC, Harold D, Schrijvers EM, Ramirez-Lorca R, Debette S, Longstreth WT, Jr., Janssens AC, Pankratz VS, Dartigues JF, Hollingworth P, Aspelund T, Hernandez I, Beiser A, Kuller LH, Koudstaal PJ, Dickson DW, Tzourio C, Abraham R, Antunez C, Du Y, Rotter JI, Aulchenko YS, Harris TB, Petersen RC, Berr C, Owen MJ, Lopez-Arrieta J, Varadarajan BN, Becker JT, Rivadeneira F, Nalls MA, Graff-Radford NR, Campion D, Auerbach S, Rice K, Hofman A, Jonsson PV, Schmidt H, Lathrop M, Mosley TH, Au R, Psaty BM, Uitterlinden AG, Farrer LA, Lumley T, Ruiz A, Williams J, Amouyel P, Younkin SG, Wolf PA, Launer LJ, Lopez OL, van Duijn CM, Breteler MM. Genome-wide analysis of genetic loci associated with Alzheimer disease. <span><span class="ref-journal">JAMA. </span>2010;<span class="ref-vol">303</span>:1832–1840.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2989531/">PMC free article</a>]</span> [<a href="/pubmed/20460622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Genome-wide+analysis+of+genetic+loci+associated+with+Alzheimer+disease&amp;author=S+Seshadri&amp;author=AL+Fitzpatrick&amp;author=MA+Ikram&amp;author=AL+DeStefano&amp;author=V+Gudnason&amp;volume=303&amp;publication_year=2010&amp;pages=1832-1840&amp;pmid=20460622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">Shen L, Kim S, Risacher SL, Nho K, Swaminathan S, West JD, Foroud T, Pankratz N, Moore JH, Sloan CD, Huentelman MJ, Craig DW, Dechairo BM, Potkin SG, Jack CR, Jr., Weiner MW, Saykin AJ. Whole genome association study of brain-wide imaging phenotypes for identifying quantitative trait loci in MCI and AD: A study of the ADNI cohort. <span><span class="ref-journal">Neuroimage. </span>2010;<span class="ref-vol">53</span>:1051–1063.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2892122/">PMC free article</a>]</span> [<a href="/pubmed/20100581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroimage&amp;title=Whole+genome+association+study+of+brain-wide+imaging+phenotypes+for+identifying+quantitative+trait+loci+in+MCI+and+AD:+A+study+of+the+ADNI+cohort&amp;author=L+Shen&amp;author=S+Kim&amp;author=SL+Risacher&amp;author=K+Nho&amp;author=S+Swaminathan&amp;volume=53&amp;publication_year=2010&amp;pages=1051-1063&amp;pmid=20100581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">Smith CD. Neuroimaging through the course of Alzheimer’s disease. <span><span class="ref-journal">J Alzheimers Dis. </span>2010;<span class="ref-vol">19</span>:273–290.</span> [<a href="/pubmed/20061644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Alzheimers+Dis&amp;title=Neuroimaging+through+the+course+of+Alzheimer’s+disease&amp;author=CD+Smith&amp;volume=19&amp;publication_year=2010&amp;pages=273-290&amp;pmid=20061644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Stieber A, Mourelatos Z, Gonatas NK. In Alzheimer’s disease the Golgi apparatus of a population of neurons without neurofibrillary tangles is fragmented and atrophic. <span><span class="ref-journal">Am J Pathol. </span>1996;<span class="ref-vol">148</span>:415–426.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1861703/">PMC free article</a>]</span> [<a href="/pubmed/8579105" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pathol&amp;title=In+Alzheimer’s+disease+the+Golgi+apparatus+of+a+population+of+neurons+without+neurofibrillary+tangles+is+fragmented+and+atrophic&amp;author=A+Stieber&amp;author=Z+Mourelatos&amp;author=NK+Gonatas&amp;volume=148&amp;publication_year=1996&amp;pages=415-426&amp;pmid=8579105&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="element-citation">Strittmatter WJ, Saunders AM, Schmechel D, Pericak-Vance M, Enghild J, Salvesen GS, Roses AD. Apolipoprotein E: high-avidity binding to beta-amyloid and increased frequency of type 4 allele in late-onset familial Alzheimer disease. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1993;<span class="ref-vol">90</span>:1977–1981.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC46003/">PMC free article</a>]</span> [<a href="/pubmed/8446617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Apolipoprotein+E:+high-avidity+binding+to+beta-amyloid+and+increased+frequency+of+type+4+allele+in+late-onset+familial+Alzheimer+disease&amp;author=WJ+Strittmatter&amp;author=AM+Saunders&amp;author=D+Schmechel&amp;author=M+Pericak-Vance&amp;author=J+Enghild&amp;volume=90&amp;publication_year=1993&amp;pages=1977-1981&amp;pmid=8446617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="element-citation">Terni B, Boada J, Portero-Otin M, Pamplona R, Ferrer I. Mitochondrial ATP-synthase in the entorhinal cortex is a target of oxidative stress at stages I/II of Alzheimer’s disease pathology. <span><span class="ref-journal">Brain Pathol. </span>2010;<span class="ref-vol">20</span>:222–233.</span> [<a href="/pubmed/19298596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Pathol&amp;title=Mitochondrial+ATP-synthase+in+the+entorhinal+cortex+is+a+target+of+oxidative+stress+at+stages+I/II+of+Alzheimer’s+disease+pathology&amp;author=B+Terni&amp;author=J+Boada&amp;author=M+Portero-Otin&amp;author=R+Pamplona&amp;author=I+Ferrer&amp;volume=20&amp;publication_year=2010&amp;pages=222-233&amp;pmid=19298596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R53"><span class="element-citation">Traynor BJ, Singleton AB. Nature versus nurture: death of a dogma, and the road ahead. <span><span class="ref-journal">Neuron. </span>2010;<span class="ref-vol">68</span>:196–200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2974322/">PMC free article</a>]</span> [<a href="/pubmed/20955927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Nature+versus+nurture:+death+of+a+dogma,+and+the+road+ahead&amp;author=BJ+Traynor&amp;author=AB+Singleton&amp;volume=68&amp;publication_year=2010&amp;pages=196-200&amp;pmid=20955927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R54"><span class="element-citation">Urcelay E, Ibarreta D, Parrilla R, Ayuso MS, Martin-Requero A. Enhanced proliferation of lymphoblasts from patients with Alzheimer dementia associated with calmodulin-dependent activation of the na+/H+ exchanger. <span><span class="ref-journal">Neurobiol Dis. </span>2001;<span class="ref-vol">8</span>:289–298.</span> [<a href="/pubmed/11300724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol+Dis&amp;title=Enhanced+proliferation+of+lymphoblasts+from+patients+with+Alzheimer+dementia+associated+with+calmodulin-dependent+activation+of+the+na+/H++exchanger&amp;author=E+Urcelay&amp;author=D+Ibarreta&amp;author=R+Parrilla&amp;author=MS+Ayuso&amp;author=A+Martin-Requero&amp;volume=8&amp;publication_year=2001&amp;pages=289-298&amp;pmid=11300724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R55"><span class="element-citation">van Es MA, van den Berg LH. Alzheimer’s disease beyond APOE. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1047–1048.</span> [<a href="/pubmed/19786950" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Alzheimer’s+disease+beyond+APOE&amp;author=MA+van+Es&amp;author=LH+van+den+Berg&amp;volume=41&amp;publication_year=2009&amp;pages=1047-1048&amp;pmid=19786950&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R56"><span class="element-citation">Veerhuis R. Histological and Direct Evidence for the Role of Complement in the Neuroinflammation of AD. <span><span class="ref-journal">Curr Alzheimer Res. </span>2010</span> [<a href="/pubmed/21143154" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Alzheimer+Res&amp;title=Histological+and+Direct+Evidence+for+the+Role+of+Complement+in+the+Neuroinflammation+of+AD&amp;author=R+Veerhuis&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R57"><span class="element-citation">Villain N, Chetelat G, Desgranges B, Eustache F. [Neuroimaging in Alzheimer’s disease: a synthesis and a contribution to the understanding of physiopathological mechanisms] <span><span class="ref-journal">Biol Aujourdhui. </span>2010;<span class="ref-vol">204</span>:145–158.</span> [<a href="/pubmed/20950559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Aujourdhui&amp;title=[Neuroimaging+in+Alzheimer’s+disease:+a+synthesis+and+a+contribution+to+the+understanding+of+physiopathological+mechanisms]&amp;author=N+Villain&amp;author=G+Chetelat&amp;author=B+Desgranges&amp;author=F+Eustache&amp;volume=204&amp;publication_year=2010&amp;pages=145-158&amp;pmid=20950559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R58"><span class="element-citation">Walker DG, Dalsing-Hernandez JE, Lue LF. Human postmortem brain-derived cerebrovascular smooth muscle cells express all genes of the classical complement pathway: a potential mechanism for vascular damage in cerebral amyloid angiopathy and Alzheimer’s disease. <span><span class="ref-journal">Microvasc Res. </span>2008;<span class="ref-vol">75</span>:411–419.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2774213/">PMC free article</a>]</span> [<a href="/pubmed/18048067" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microvasc+Res&amp;title=Human+postmortem+brain-derived+cerebrovascular+smooth+muscle+cells+express+all+genes+of+the+classical+complement+pathway:+a+potential+mechanism+for+vascular+damage+in+cerebral+amyloid+angiopathy+and+Alzheimer’s+disease&amp;author=DG+Walker&amp;author=JE+Dalsing-Hernandez&amp;author=LF+Lue&amp;volume=75&amp;publication_year=2008&amp;pages=411-419&amp;pmid=18048067&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R59"><span class="element-citation">Yang L, Zhang L, Wu Q, Boyd DD. Unbiased screening for transcriptional targets of ZKSCAN3 identifies integrin beta 4 and vascular endothelial growth factor as downstream targets. <span><span class="ref-journal">J Biol Chem. </span>2008;<span class="ref-vol">283</span>:35295–35304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2596387/">PMC free article</a>]</span> [<a href="/pubmed/18940803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Unbiased+screening+for+transcriptional+targets+of+ZKSCAN3+identifies+integrin+beta+4+and+vascular+endothelial+growth+factor+as+downstream+targets&amp;author=L+Yang&amp;author=L+Zhang&amp;author=Q+Wu&amp;author=DD+Boyd&amp;volume=283&amp;publication_year=2008&amp;pages=35295-35304&amp;pmid=18940803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312985&amp;issue-id=207174&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140338197075968title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and Methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S13title">Discussion</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S14title">Acknowledgements</a></li><li><a href="#idm140338127353184title">Footnotes</a></li><li><a href="#idm140338127349856title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3312985/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3312985/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3312985/pdf/nihms-348696.pdf">PDF (1.9M)</a></li> | <li><a href="#" data-citationid="PMC3312985" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312985%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312985%2F&amp;text=A%20large%20scale%20multivariate%20parallel%20ICA%20method%20reveals%20novel%20imaging-genetic%20relationships%20for%20Alzheimer%E2%80%99s%20Disease%20in%20the%20ADNI%20cohort" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312985%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22245343" data-aiid="3312985" data-aid="3312985" data-iid="207174" data-domainid="319" data-domain="nihpa" data-accid="PMC3312985" data-md5="711876f0be38a99f52b123d5cf520f4e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:25:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89F704DD5313410000000008E00607&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312985%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3312985/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>