<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jnci"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3691938/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Novel Susceptibility Variants at 10p12.31-12.2 for Childhood Acute Lymphoblastic Leukemia in Ethnically Diverse Populations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3691938/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="JNCI Journal of the National Cancer Institute"><meta name="citation_title" content="Novel Susceptibility Variants at 10p12.31-12.2 for Childhood Acute Lymphoblastic Leukemia in Ethnically Diverse Populations"><meta name="citation_authors" content="Heng Xu, Wenjian Yang, Virginia Perez-Andreu, Meenakshi Devidas, Yiping Fan, Cheng Cheng, Deqing Pei, Paul Scheet, Esteban González Burchard, Celeste Eng, Scott Huntsman, Dara G. Torgerson, Michael Dean, Naomi J. Winick, Paul L. Martin, Bruce M. Camitta, W. Paul Bowman, Cheryl L. Willman, William L. Carroll, Charles G. Mullighan, Deepa Bhojwani, Stephen P. Hunger, Ching-Hon Pui, William E. Evans, Mary V. Relling, Mignon L. Loh, Jun J. Yang"><meta name="citation_date" content="15 May 2013"><meta name="citation_issue" content="10"><meta name="citation_volume" content="105"><meta name="citation_firstpage" content="733"><meta name="citation_doi" content="10.1093/jnci/djt042"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3691938/?report=abstract"><meta name="citation_pmid" content="23512250"><meta name="DC.Title" content="Novel Susceptibility Variants at 10p12.31-12.2 for Childhood Acute Lymphoblastic Leukemia in Ethnically Diverse Populations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Heng Xu"><meta name="DC.Contributor" content="Wenjian Yang"><meta name="DC.Contributor" content="Virginia Perez-Andreu"><meta name="DC.Contributor" content="Meenakshi Devidas"><meta name="DC.Contributor" content="Yiping Fan"><meta name="DC.Contributor" content="Cheng Cheng"><meta name="DC.Contributor" content="Deqing Pei"><meta name="DC.Contributor" content="Paul Scheet"><meta name="DC.Contributor" content="Esteban González Burchard"><meta name="DC.Contributor" content="Celeste Eng"><meta name="DC.Contributor" content="Scott Huntsman"><meta name="DC.Contributor" content="Dara G. Torgerson"><meta name="DC.Contributor" content="Michael Dean"><meta name="DC.Contributor" content="Naomi J. Winick"><meta name="DC.Contributor" content="Paul L. Martin"><meta name="DC.Contributor" content="Bruce M. Camitta"><meta name="DC.Contributor" content="W. Paul Bowman"><meta name="DC.Contributor" content="Cheryl L. Willman"><meta name="DC.Contributor" content="William L. Carroll"><meta name="DC.Contributor" content="Charles G. Mullighan"><meta name="DC.Contributor" content="Deepa Bhojwani"><meta name="DC.Contributor" content="Stephen P. Hunger"><meta name="DC.Contributor" content="Ching-Hon Pui"><meta name="DC.Contributor" content="William E. Evans"><meta name="DC.Contributor" content="Mary V. Relling"><meta name="DC.Contributor" content="Mignon L. Loh"><meta name="DC.Contributor" content="Jun J. Yang"><meta name="DC.Date" content="2013 May 15"><meta name="DC.Identifier" content="10.1093/jnci/djt042"><meta name="DC.Language" content="en"><meta property="og:title" content="Novel Susceptibility Variants at 10p12.31-12.2 for Childhood Acute Lymphoblastic Leukemia in Ethnically Diverse Populations"><meta property="og:type" content="article"><meta property="og:description" content="Acute lymphoblastic leukemia (ALL) is the most common cancer in children and the incidence of ALL varies by ethnicity. Although accumulating evidence indicates inherited predisposition to ALL, the genetic basis of ALL susceptibility in diverse ancestry ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3691938/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jnci.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3691938/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3691938/pdf/djt042.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8828B0DD551BD100000000078A0508.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3691938%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="nuccore">Nucleotide</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22J Natl Cancer Inst%22[journal]" class="navlink">J Natl Cancer Inst</a></li><li class="accid">PMC3691938</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jnci.png" alt="Logo of jnci" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/jnci" target="pmc_ext" ref="reftype=publisher&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Natl Cancer Inst</a></span>. 2013 May 15; 105(10): 733–742. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Mar 19. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fjnci%2Fdjt042" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/jnci/djt042</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3691938</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23512250">23512250</a></div></div></div></div><h1 class="content-title">Novel Susceptibility Variants at 10p12.31-12.2 for Childhood Acute Lymphoblastic Leukemia in Ethnically Diverse Populations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Xu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Heng Xu</a>, <a href="/pubmed/?term=Yang%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Wenjian Yang</a>, <a href="/pubmed/?term=Perez-Andreu%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Virginia Perez-Andreu</a>, <a href="/pubmed/?term=Devidas%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Meenakshi Devidas</a>, <a href="/pubmed/?term=Fan%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Yiping Fan</a>, <a href="/pubmed/?term=Cheng%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Cheng Cheng</a>, <a href="/pubmed/?term=Pei%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Deqing Pei</a>, <a href="/pubmed/?term=Scheet%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Paul Scheet</a>, <a href="/pubmed/?term=Burchard%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Esteban González Burchard</a>, <a href="/pubmed/?term=Eng%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Celeste Eng</a>, <a href="/pubmed/?term=Huntsman%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Scott Huntsman</a>, <a href="/pubmed/?term=Torgerson%20DG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Dara G. Torgerson</a>, <a href="/pubmed/?term=Dean%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Michael Dean</a>, <a href="/pubmed/?term=Winick%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Naomi J. Winick</a>, <a href="/pubmed/?term=Martin%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Paul L. Martin</a>, <a href="/pubmed/?term=Camitta%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Bruce M. Camitta</a>, <a href="/pubmed/?term=Bowman%20WP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">W. Paul Bowman</a>, <a href="/pubmed/?term=Willman%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Cheryl L. Willman</a>, <a href="/pubmed/?term=Carroll%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">William L. Carroll</a>, <a href="/pubmed/?term=Mullighan%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Charles G. Mullighan</a>, <a href="/pubmed/?term=Bhojwani%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Deepa Bhojwani</a>, <a href="/pubmed/?term=Hunger%20SP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Stephen P. Hunger</a>, <a href="/pubmed/?term=Pui%20CH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Ching-Hon Pui</a>, <a href="/pubmed/?term=Evans%20WE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">William E. Evans</a>, <a href="/pubmed/?term=Relling%20MV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Mary V. Relling</a>, <a href="/pubmed/?term=Loh%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Mignon L. Loh</a>, and  <a href="/pubmed/?term=Yang%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23512250">Jun J. Yang</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140296001559712_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140296001559712_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140296001559712_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140296001559712_ai" style="display:none"><div class="fm-affl" id="AF0001" lang="en"><strong>Affiliations of authors:</strong>Department of Pharmaceutical Sciences (HX, WY, VP-A, WEE, MVR, JJY), Department of Computational Biology (YF), Department of Biostatistics (CC, DP), Department of Pathology (CGM), and Department of Oncology (DB, C-HP), St. Jude Children’s Research Hospital, Memphis, TN; Department of Epidemiology and Health Policy Research, University of Florida, Gainesville, FL (MDev); Department of Epidemiology, The University of Texas MD Anderson Cancer Center, Houston, TX (PS); Department of Bioengineering &amp; Therapeutic Science and Medicine (EGB, CE, SH, DGT) and Department of Pediatrics (MLL), University of California−San Francisco, San Francisco, CA; Laboratory of Experimental Immunology, National Cancer Institute, Frederick, MD (MDea); Pediatric Hematology/Oncology, University of Texas Southwestern Medical Center, Dallas, TX, (NJW); Department of Pediatrics, Duke University, Durham, NC (PLM); Department of Pediatrics, Medical College of Wisconsin, Milwaukee, WI (BMC); Hematology &amp; Oncology, Cook Children’s Medical Center, Ft. Worth, TX (WPB); University of New Mexico Cancer Center, Albuquerque, NM (CLW); New York University Cancer Institute, New York, NY (WLC); University of Colorado School of Medicine and The Children’s Hospital, Aurora, CO (SPH).</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="c1"><strong>Correspondence to:</strong> Jun J. Yang, PhD, Pharmaceutical Sciences, MS 313, St Jude Children’s Research Hospital, 262 Danny Thomas Pl, Memphis, TN 38105-3678 (e-mail: <a href="mailto:dev@null" data-email="gro.edujts@gnay.nuj" class="oemail">gro.edujts@gnay.nuj</a>).</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140296001559712_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2012 Jul 13; Revised 2013 Jan 31; Accepted 2013 Feb 4.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140296001559712_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press. All rights reserved. For Permissions, please e-mail: journals.permissions@oup.com.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3691938/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140296034648176" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140296034648176title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Background</h3><p id="__p1" class="p p-first-last">Acute lymphoblastic leukemia (ALL) is the most common cancer in children and the incidence of ALL varies by ethnicity. Although accumulating evidence indicates inherited predisposition to ALL, the genetic basis of ALL susceptibility in diverse ancestry has not been comprehensively examined.</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Methods</h3><p id="__p2" class="p p-first-last">We performed a multiethnic genome-wide association study in 1605 children with ALL and 6661 control subjects after adjusting for population structure, with validation in three replication series of 845 case subjects and 4316 control subjects. Association was tested by two-sided logistic regression.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Results</h3><p id="__p3" class="p p-first-last">A novel ALL susceptibility locus at 10p12.31-12.2 (<em>BMI1-PIP4K2A</em>, rs7088318, <em>P</em> = 1.1×10<sup>−11</sup>) was identified in the genome-wide association study, with independent replication in European Americans, African Americans, and Hispanic Americans (<em>P</em> = .001, .009, and .04, respectively). Association was also validated at four known ALL susceptibility loci: <em>ARID5B</em>, <em>IKZF1</em>, <em>CEBPE</em>, and <em>CDKN2A/2B</em>. Associations at <em>ARID5B, IKZF1,</em> and <em>BMI1</em>-<em>PIP4K2A</em> variants were consistent across ethnicity, with multiple independent signals at <em>IKZF1</em> and <em>BMI1-PIP4K2A</em> loci. The frequency of <em>ARID5B</em> and <em>BMI1-PIP4K2A</em> variants differed by ethnicity, in parallel with ethnic differences in ALL incidence. Suggestive evidence for modifying effects of age on genetic predisposition to ALL was also observed. <em>ARID5B</em>, <em>IKZF1</em>, <em>CEBPE</em>, and <em>BMI1</em>-<em>PIP4K2A</em> variants cumulatively conferred strong predisposition to ALL, with children carrying six to eight copies of risk alleles at a ninefold (95% confidence interval = 6.9 to 11.8) higher ALL risk relative to those carrying zero to one risk allele at these four single nucleotide polymorphisms.</p></div><div id="__sec4" class="sec sec-last"><h3 id="__sec4title">Conclusions</h3><p id="__p4" class="p p-first-last">These findings indicate strong associations between inherited genetic variation and ALL susceptibility in children and shed new light on ALL molecular etiology in diverse ancestry.</p></div></div></div><div id="idm140295955401744" class="tsec sec headless whole_rhythm"><p id="__p5" class="p p-first">Acute lymphoblastic leukemia (ALL) is the most common childhood malignancy and a leading cause of death due to disease in children (<a href="#CIT0001" rid="CIT0001" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107623">1</a>,<a href="#CIT0002" rid="CIT0002" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107614">2</a>). A genetic basis of ALL susceptibility is supported by its association with certain congenital abnormalities (<a href="#CIT0003" rid="CIT0003" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107627">3</a>,<a href="#CIT0004" rid="CIT0004" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107657">4</a>) and, more recently, by genome-wide association studies (GWASs) identifying common variants at <em>ARID5B</em> (10q21.2), <em>IKZF1</em> (7p12.2), and <em>CEBPE</em> (14q11.2) influencing ALL risk in children of European descent (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107613">5–9</a>). In fact, the disease risk associated with these common variants are among the strongest in cancer susceptibility variants identified through GWASs (<a href="#CIT0010" rid="CIT0010" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107619">10</a>), consistent with a relatively large impact of inherited genetic factors on the pathogenesis of this childhood malignancy (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107631">11</a>). However, the loci reported in ALL GWASs thus far cumulatively accounted for only 8% of genetic variation in ALL risk (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107628">11</a>), suggesting additional susceptibility variants yet to be discovered in larger studies.</p><p id="__p6">There is an extreme lack of population diversity in GWASs such that 96% of subjects studied in GWASs so far are individuals of European descent (<a href="#CIT0012" rid="CIT0012" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352915433">12</a>,<a href="#CIT0013" rid="CIT0013" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107636">13</a>). This exclusive focus on selected few ethnic groups raises a number of critical questions: For example, are findings from European-only GWASs transferable to other populations (<a href="#CIT0014" rid="CIT0014" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107626">14</a>, <a href="#CIT0015" rid="CIT0015" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107662">15</a>)? Can disease etiology be different among populations and thus characterized by distinct genetic risk factors (<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107678">16</a>)? What is the contribution of ancestry-related genetic variation to ethnic differences in cancer prevalence (<a href="#CIT0017" rid="CIT0017" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107658">17</a>,<a href="#CIT0018" rid="CIT0018" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107612">18</a>)? These issues are of particularly relevance to childhood ALL because the incidence of ALL varies substantially by ethnicity (14.8 per million person-years in African Americans, 35.6 per million person-years in European Americans, and 40.9 per million person-years in Hispanic Americans) (<a href="#CIT0019" rid="CIT0019" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107635">19</a>,<a href="#CIT0020" rid="CIT0020" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107611">20</a>), at least partly attributable to population differences in inherited genetic variations [eg, <em>ARID5B</em> (<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107673">21</a>,<a href="#CIT0022" rid="CIT0022" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_500512076">22</a>)].</p><p id="__p7" class="p">To identify novel ALL susceptibility loci and also to evaluate the associations of known susceptibility variants in diverse populations, we examined 709059 single nucleotide polymorphisms (SNPs) for association with childhood ALL in a multiethnic GWAS of 1605 case subjects and 6661 control subjects, followed by three independent replications in 845 case subjects and 4316 control subjects of European American, African American, and Hispanic American ethnicity.</p></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">Methods</h2><div id="s2" class="sec sec-first"><h3 id="s2title">Subjects and Genotyping</h3><p id="__p8" class="p p-first">Two nonoverlapping series of childhood ALL case subjects and control subjects were included: the GWAS series and the replication series. In the GWAS series, we included 1605 B-precursor childhood ALL case subjects enrolled on the Children’s Oncology Group (COG) P9904/P9905 protocols (<a href="#CIT0023" rid="CIT0023" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107622">23</a>), and 6661 unrelated subjects from the Multi-Ethnic Study of Atherosclerosis (MESA) (dbGAP phs000209.v9) were considered as non-ALL control subjects because the prevalence of adult survivors of childhood ALL is less than 1 in 10 000 in the United States (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107665">5</a>). The replication study included three case–control series separately by ethnicity (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a>, available online): European Americans: 574 case subjects and 2601 control subjects (<a href="#CIT0024" rid="CIT0024" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107654">24</a>,<a href="#CIT0025" rid="CIT0025" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352915434">25</a>); African Americans: 128 case subjects and 1075 control subjects (<a href="#CIT0026" rid="CIT0026" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107617">26</a>); Hispanic Americans: 143 case subjects and 640 control subjects (<a href="#CIT0027" rid="CIT0027" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107660">27</a>). ALL case subjects in the replication series were from the St. Jude Total Therapy XIIIB/XV and the COG P9906 protocol (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107641">5</a>). Ethnicity was defined by genetic ancestry as described below. ALL molecular subtypes included <em>MLL</em> rearrangements, <em>ETV6-RUNX1</em>, <em>TCF3-PBX1</em>, or <em>BCR-ABL1</em>, and hyperdiploid. Patients include in the genetic association analyses represented 85.3% (n = 1605 of 1882) of total enrolled participants on the COG P9904/9905 treatment protocols and 83.1% (n = 854 of 1017) of participants of the COG P9906 and St. Jude Total Therapy XIIIB/XV protocols (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 1</a>, available online).</p><p id="__p9">Genotyping of ALL case subjects was performed by using the Affymetrix (Santa Clara, CA) Human SNP Array 6.0 (COG P9904/P9905, the GWAS series) or the Affymetrix GeneChip Human Mapping 500K Array (St. Jude Total Therapy XIIIB/XV and COG P9906, the replication series). Non-ALL control subjects in both the GWAS and replication series were genotyped using Affymetrix Human SNP Array 6.0. Genotype calls (coded as 0, 1, and 2 for AA, AB, and BB genotypes) were determined by the Birdseed (Affymetrix Human SNP Array 6.0) (<a href="#CIT0028" rid="CIT0028" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107676">28</a>) or BRLMM (Affymetrix GeneChip Human Mapping 500K Array) algorithms (<a href="#CIT0029" rid="CIT0029" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107606">29</a>). Samples for which genotype was ascertained at less than 95% of SNPs on the array were deemed to have failed and were excluded from the analyses (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 1</a>, available online). SNP quality control procedures were performed on the basis of call rate, minor allele frequency, and Hardy–Weinberg equilibrium, and 197541 of 906600 SNPs were excluded during GWAS quality control (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 2</a> and <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a>, available online).</p><p id="__p10" class="p p-last">This study was approved by the respective institutional review boards, and informed consent was obtained from parents, guardians, or patients, as appropriate.</p></div><div id="s3" class="sec"><h3 id="s3title">Ethnicity Classification</h3><p id="__p11" class="p p-first-last">European, African, Asian, and Native American genetic ancestry was determined by using STRUCTURE (version 2.2.3) (<a href="#CIT0030" rid="CIT0030" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107618">30</a>,<a href="#CIT0031" rid="CIT0031" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107609">31</a>) with HapMap CEU, YRI, CHB/JPT, and indigenous Native Americans (<a href="#CIT0032" rid="CIT0032" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107634">32</a>) as reference populations, respectively. European Americans, African Americans, and Asian Americans were defined as having more than 95% European genetic ancestry, more than 70% African ancestry, and more than 90% Asian ancestry, respectively. Hispanic Americans were individuals for whom Native American ancestry was more than 10% and greater than African ancestry; the remaining subjects were grouped as “Others” (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 3</a>, available online).</p></div><div id="s4" class="sec sec-last"><h3 id="s4title">Statistical Analyses</h3><p id="__p12" class="p p-first">In the GWAS, the association between genotypes at each of 709059 SNPs and ALL susceptibility was tested by comparing the genotype frequency between ALL case subjects and control subjects in the logistic regression model, after adjusting for the top four principal components inferred by EIGENSTRAT (<a href="#CIT0033" rid="CIT0033" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107647">33</a>) to control for population stratification (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 4</a>, available online). To validate associations at four susceptibility loci previously identified in populations of European descent [<em>ARID5B</em> (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107630">5</a>,<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544244955">6</a>), <em>IKZF1</em> (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107632">5</a>,<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544244959">6</a>), <em>CEBPE</em> (<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544244958">6</a>), and <em>CDKN2A/2B</em> (<a href="#CIT0008" rid="CIT0008" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107637">8</a>)], we focused on variants within 600kb of the top SNP at each locus and applied statistical significance threshold that corrected for the number of SNPs tested at each locus (n = 174, 241, 104, and 145, respectively). To agnostically search for novel susceptibility variants by GWAS, we applied a genome-wide statistical significance cutoff of <em>P</em> less than or equal to 5×10<sup>−8</sup> and sought to verify SNPs meeting this threshold in independent replication series.</p><p id="__p13">In the replication studies, we tested six SNPs at the <em>BMI1-PIP4K2A</em> locus separately in European Americans, African Americans, and Hispanic Americans by logistic regression test with genetic ancestries as covariates. Those with <em>P</em> less than .05 in replication series were considered as validated.</p><p id="__p14">Logistic regression model was also used to determine the independent association of multiple SNPs within the same locus, to examine the cumulative effects of multiple susceptibility variants, and to evaluate the effects of susceptibility variants in different age groups. Association between <em>PIP4K2A</em> SNP genotype and gene expression was assessed by a linear regression model in HapMap CEU lymphoblastoid cell lines [<a id="__tag_321107680" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7851" ref="/geo/query/acc.cgi?acc=GSE7851"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7851","term_id":"7851"}}</span>GSE7851</a> (<a href="#CIT0034" rid="CIT0034" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107659">34</a>)] and in diagnostic blasts from European American children with ALL from St. Jude Total Therapy XIIIB/XV protocols (<a href="#CIT0035" rid="CIT0035" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107646">35</a>,<a href="#CIT0036" rid="CIT0036" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107644">36</a>).</p><p id="__p15" class="p p-last">R (version 2.15.1) statistical software was used for all analyses unless indicated otherwise, and a detailed description of statistical procedures is provided in the <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> (available online). All statistical tests were two-sided.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">Results</h2><p id="__p16" class="p p-first">To comprehensively examine germline ALL susceptibility variants, we performed GWAS in 1605 children with newly diagnosed B-precursor ALL and 6661 unrelated non-ALL control subjects of diverse ancestry (ie, European, African, Asian, and Native American genetic ancestry) (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figures 3</a> and <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">4</a><strong>, available online</strong>). Controlling for population structures, we evaluated 709059 germ line SNPs for differences in genotype frequency between ALL case subjects and control subjects.</p><p id="__p17">We first focused on three susceptibility loci previously identified by GWAS in populations of European descent (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107639">5</a>,<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544244957">6</a>)—<em>ARID5B</em> at 10q21.2, <em>IKZF1</em> at 7p12.2, and <em>CEBPE</em> at 14q11.2—to compare the association signals among populations, particularly in those of non-European descent (<a href="/pmc/articles/PMC3691938/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>, available online). At the <em>ARID5B</em> locus (ie, rs10821936), the association with ALL was consistent in all three genetically defined ethnicities: European Americans (<em>P</em> = 6.9×10<sup>−30</sup>; n = 972 case subjects and 1386 control subjects); African Americans (<em>P</em> = .004; n = 89 case subjects and 1363 control subjects); Hispanics (<em>P</em> = 3.8×10<sup>−11</sup>; n = 305 case subjects and 1008 control subjects); and the multiethnic group (<em>P</em> = 5.9×10<sup>−46</sup>; n = 1605 case subjects and 6661 control subjects). The frequency of the ALL risk allele (C) at rs10821936 increased in the order of African Americans, European Americans, and Hispanic Americans, consistent with the ethnic differences in ALL incidence (<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107671">21</a>). Multivariable analyses adjusting for rs10821936 did not identify any additional independent association signal at this locus (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 5</a>, available online). The top SNP in <em>IKZF1</em> (ie, rs11978267; <em>P</em> = 5.3×10<sup>−24</sup> in the multiethnic group) was also associated with ALL risk across ethnicities. Interestingly, another cluster of SNPs further upstream of rs11978267 were statistically significant even after controlling for rs11978267 (ie, rs10235226; <em>P</em> = 1.4×10<sup>−5</sup> in the multiethnic group) (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 5</a> and <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a>, available online). Association at <em>CEBPE</em> SNPs was validated in the multiethnic group (ie, rs4982731; <em>P</em> = 9×10<sup>−12</sup>) (<a href="/pmc/articles/PMC3691938/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), but multivariable model conditioning on the top SNP (rs4982731) did not support additional independent associations in this region (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 5</a>, available online). Another previously reported ALL risk locus at 9p21.3 (<a href="#CIT0008" rid="CIT0008" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107648">8</a>) was also validated in our GWAS series (ie, rs1775631 at <em>CDKN2A/2B</em>; <em>P</em> = 1.4×10<sup>−5</sup> in the multiethnic group). In total, of 664 SNPs at these four loci, 79 remained statistically significant after correcting for multiple testing (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>, available online).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1. </h3><!--caption a7--><div class="caption"><p id="__p18">Associations at four known acute lymphoblastic leukemia susceptibility loci: 7p12.2, 9p21.3, 10q21.2, and 14q11.2<sup>*</sup>
</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295995570112" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="2" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="2" colspan="1" valign="bottom" align="center">Position<sup>†</sup>
</th><th rowspan="2" colspan="1" valign="bottom" align="center">Alleles<sup>‡</sup>
</th><th rowspan="2" colspan="1" valign="bottom" align="center">Gene(s)</th><th colspan="3" rowspan="1" valign="bottom" align="center">European American (n = 972/n = 1386)<sup>§</sup>
</th><th colspan="3" rowspan="1" valign="bottom" align="center">African American (n = 89/n = 1363)<sup> §</sup>
</th><th colspan="3" rowspan="1" valign="bottom" align="center">Hispanic American (n = 305/n = 1008)<sup>§</sup>
</th><th colspan="2" rowspan="1" valign="bottom" align="center">All ethnicities <br>(n = 1605/ n = 6661)<sup>§</sup>
</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (case/control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (case/control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (case/control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs11978267</td><td rowspan="1" colspan="1" align="center"> 7</td><td rowspan="1" colspan="1" align="center">50433798</td><td rowspan="1" colspan="1" align="center">A/<strong>G</strong></td><td rowspan="1" colspan="1" align="center"><em>IKZF1</em></td><td rowspan="1" colspan="1" align="center">0.39/0.28</td><td rowspan="1" colspan="1" align="center">8.36×10<sup>−19</sup>
</td><td rowspan="1" colspan="1" align="center">1.67 (1.49 to 1.87)</td><td rowspan="1" colspan="1" align="center">0.27/0.19</td><td rowspan="1" colspan="1" align="center">.005</td><td rowspan="1" colspan="1" align="center">1.59 (1.15 to 2.18)</td><td rowspan="1" colspan="1" align="center">0.31/0.26</td><td rowspan="1" colspan="1" align="center">.01</td><td rowspan="1" colspan="1" align="center">1.31 (1.07 to 1.61)</td><td rowspan="1" colspan="1" align="center">5.29×10<sup>−24</sup>
</td><td rowspan="1" colspan="1" align="center">1.59 (1.45 to 1.74)</td></tr><tr><td rowspan="1" colspan="1">rs17756311</td><td rowspan="1" colspan="1" align="center"> 9</td><td rowspan="1" colspan="1" align="center">22043895</td><td rowspan="1" colspan="1" align="center">C/<strong>T</strong>
</td><td rowspan="1" colspan="1" align="center">
<em>CDKN2A/B</em>
</td><td rowspan="1" colspan="1" align="center">0.13/0.09</td><td rowspan="1" colspan="1" align="center">3.25×10<sup>−5</sup>
</td><td rowspan="1" colspan="1" align="center">1.43 (1.21 to 1.69)</td><td rowspan="1" colspan="1" align="center">0.11/0.1</td><td rowspan="1" colspan="1" align="center">.62</td><td rowspan="1" colspan="1" align="center">1.12 (0.71 to 1.76)</td><td rowspan="1" colspan="1" align="center">0.08/0.06</td><td rowspan="1" colspan="1" align="center">.10</td><td rowspan="1" colspan="1" align="center">1.36 (0.94 to 1.97)</td><td rowspan="1" colspan="1" align="center">1.37×10<sup>−5</sup>
</td><td rowspan="1" colspan="1" align="center">1.36 (1.18 to 1.56)</td></tr><tr><td rowspan="1" colspan="1">rs10821936</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">63393583</td><td rowspan="1" colspan="1" align="center">
<strong>C</strong>/T</td><td rowspan="1" colspan="1" align="center">
<em>ARID5B</em>
</td><td rowspan="1" colspan="1" align="center">0.48/0.33</td><td rowspan="1" colspan="1" align="center">6.93×10<sup>−30</sup>
</td><td rowspan="1" colspan="1" align="center">1.88 (1.68 to 2.10)</td><td rowspan="1" colspan="1" align="center">0.33/0.24</td><td rowspan="1" colspan="1" align="center">.004</td><td rowspan="1" colspan="1" align="center">1.52 (1.14 to 2.02)</td><td rowspan="1" colspan="1" align="center">0.63/0.47</td><td rowspan="1" colspan="1" align="center">3.78×10<sup>−11</sup>
</td><td rowspan="1" colspan="1" align="center">1.95 (1.60 to 2.38)</td><td rowspan="1" colspan="1" align="center">5.88×10<sup>−46</sup>
</td><td rowspan="1" colspan="1" align="center">1.86 (1.71 to 2.03)</td></tr><tr><td rowspan="1" colspan="1">rs4982731</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">22655173</td><td rowspan="1" colspan="1" align="center">
<strong>C</strong>/T</td><td rowspan="1" colspan="1" align="center">
<em>CEBPE</em>
</td><td rowspan="1" colspan="1" align="center">0.34/0.28</td><td rowspan="1" colspan="1" align="center">9.05×10<sup>−6</sup>
</td><td rowspan="1" colspan="1" align="center">1.29 (1.15 to 1.45)</td><td rowspan="1" colspan="1" align="center">0.41/0.38</td><td rowspan="1" colspan="1" align="center">.41</td><td rowspan="1" colspan="1" align="center">1.13 (0.85 to 1.50)</td><td rowspan="1" colspan="1" align="center">0.5/0.39</td><td rowspan="1" colspan="1" align="center">2.32×10<sup>−6</sup>
</td><td rowspan="1" colspan="1" align="center">1.58 (1.31 to 1.91)</td><td rowspan="1" colspan="1" align="center">8.97×10<sup>−12</sup>
</td><td rowspan="1" colspan="1" align="center">1.36 (1.24 to 1.48)</td></tr></tbody></table></div><div id="largeobj_idm140295995570112" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3691938/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140295959590592"><p id="__p19" class="p p-first-last">
<sup>*</sup> Association of variants at these four loci was tested in the genome-wide association study series and shown are the top single nucleotide polymorphisms at each locus. Chr = chromosome; CI = confidence interval; OR = odds ratio; RAF = risk allele frequency.</p></div><div id="idm140295959589072"><p id="__p20" class="p p-first-last">
<sup><strong>†</strong></sup> Chromosomal locations are based on hg18.</p></div><div id="idm140295959587696"><p id="__p21" class="p p-first-last">
<sup><strong>‡</strong></sup> Bold denotes the allele that had a statistically significantly higher frequency in children with acute lymphoblastic leukemia than in the non–acute lymphoblastic leukemia control subjects (ie, risk allele for acute lymphoblastic leukemia).</p></div><div id="idm140295949527120"><p id="__p22" class="p p-first-last">
<sup><strong>§</strong></sup> Ethnicity was defined by single nucleotide polymorphism genotype-based European, African, East Asian, and Native American genetic ancestry (see Methods), and numbers of acute lymphoblastic leukemia patients vs non–acute lymphoblastic leukemia control subjects are indicated.</p></div><div id="idm140295949525440"><p id="__p23" class="p p-first-last">
<sup>ǁ</sup> Odds ratio represents the increase in the risk of developing acute lymphoblastic leukemia for each copy of the risk allele compared with subjects who do not carry the risk allele. <em>P</em> values and odds ratios were estimated by the logistic regression test (two-sided).</p></div></div></div><p id="__p24">Importantly, our multiethnic GWAS also discovered a novel ALL susceptibility locus at 10p12.31-12.2 that was not identified by previous studies in populations of European descent. As shown in <a href="/pmc/articles/PMC3691938/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>, <a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>, and <a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, six variants in the <em>BMI1</em>-<em>PIP4K2A</em> region exhibited genome-wide statistically significant associations with ALL. Four SNPs were clustered within the intronic region of the <em>PIP4K2A</em> gene; the other two were upstream of the <em>COMMD3</em> and <em>BMI1</em> genes and further distal to the centromere (<a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The SNPs with the strongest association in each region were rs7088318 (<em>PIP4K2A</em>; <em>P</em> = 1.1×10<sup>−11</sup>) and rs4748793 (<em>COMMD3/BMI1</em>; <em>P</em> = 8.4×10<sup>−9</sup>), respectively (<a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Although both SNPs conferred a similar degree of increase in ALL risk (odds ratio [OR] = 1.4), they were independently associated with disease susceptibility (<em>P</em> &lt; .0001) in a multivariable model adjusting for each other (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 5</a>, available online) and were separated by distinct linkage disequilibrium (LD) blocks in all ethnic groups (<a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Also, the frequency of the ALL risk allele at rs7088318 was highest in Hispanic Americans, followed by European Americans, and lowest in African Americans, in parallel with ALL incidence in these populations (<a href="#CIT0020" rid="CIT0020" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107625">20</a>) (<a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). To explore possible functional consequences of this <em>PIP4K2A</em> variation, we investigated the relationship between rs7088318 genotype and <em>PIP4K2A</em> mRNA expression. In lymphoblastoid cell lines derived from the HapMap CEU samples, the ALL risk allele (A) at rs7088318 was linked to higher <em>PIP4K2A</em> expression (<em>P</em> = .001; n = 55) (<a href="/pmc/articles/PMC3691938/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Consistently, the number of A allele at this SNP was also positively associated with <em>PIP4K2A</em> expression in diagnostic blasts from children with ALL (<em>P</em> = .02; n = 228) (<a href="/pmc/articles/PMC3691938/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>), indicative of a cis-acting expression quantitative trait locus.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2. </h3><!--caption a7--><div class="caption"><p id="__p25">Genome-wide statistically significant association and replication of novel acute lymphoblastic leukemia susceptibility variants at 10p12.31-12.2 </p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295960693696" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="3" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="3" colspan="1" valign="bottom" align="center">Position*</th><th rowspan="3" colspan="1" valign="bottom" align="center">Alleles<sup>†</sup>
</th><th rowspan="3" colspan="1" valign="bottom" align="center">Gene(s)</th><th colspan="10" rowspan="1" valign="bottom" align="center">GWAS series (case/control)</th><th colspan="7" rowspan="1" valign="bottom" align="center">Replication series (case/control)</th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="center">European American (n = 972/n = 1386)<sup> ‡</sup>
</th><th colspan="3" rowspan="1" valign="bottom" align="center">African American <br> (n = 89/n = 1363)<sup>‡</sup>
</th><th colspan="3" rowspan="1" valign="bottom" align="center">Hispanic American <br>(n = 305/n = 1008)<sup>‡</sup>
</th><th colspan="2" rowspan="1" valign="bottom" align="center">All ethnicities (n = 1605/n = 6661)<sup>‡</sup>
</th><th colspan="2" rowspan="1" valign="bottom" align="center">European American (n = 574/n = 2601)<sup>‡</sup>
</th><th colspan="2" rowspan="1" valign="bottom" align="center">African American (n = 128/n = 1075)<sup>‡</sup>
</th><th colspan="2" rowspan="1" valign="bottom" align="center">Hispanic American (n = 143/n = 640)<sup>‡</sup>
</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (Case/Control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (Case/Control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">RAF (Case/Control)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">
<em>P</em>
</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<sup>ǁ</sup> (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs4266962</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22381580</td><td rowspan="1" colspan="1" align="center">C/<strong>G</strong>
</td><td rowspan="1" colspan="1" align="center">
<em>COMMD3/BMI1</em>
</td><td rowspan="1" colspan="1" align="center">0.83/0.76</td><td rowspan="1" colspan="1" align="center">4.35×10<sup>−8</sup>
</td><td rowspan="1" colspan="1" align="center">1.41 (1.20 to 1.65)</td><td rowspan="1" colspan="1" align="center">0.95/0.94</td><td rowspan="1" colspan="1" align="center">.97</td><td rowspan="1" colspan="1" align="center">1.08 (0.49 to 2.38)</td><td rowspan="1" colspan="1" align="center">0.8/0.78</td><td rowspan="1" colspan="1" align="center">.48</td><td rowspan="1" colspan="1" align="center">1.03 (0.82 to 1.30)</td><td rowspan="1" colspan="1" align="center">2.39×10<sup>−8</sup>
</td><td rowspan="1" colspan="1" align="center">1.39 (1.25 to 1.56)</td><td rowspan="1" colspan="1" align="center">.0003</td><td rowspan="1" colspan="1" align="center">1.33 (1.13 to 1.57)</td><td rowspan="1" colspan="1" align="center">NA<sup><strong>§</strong></sup>
</td><td rowspan="1" colspan="1" align="center">NA<sup><strong>§</strong></sup>
</td><td rowspan="1" colspan="1" align="center">.17</td><td rowspan="1" colspan="1" align="center">1.20 (0.82 to 1.76)</td></tr><tr><td rowspan="1" colspan="1">rs4748793</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22523017</td><td rowspan="1" colspan="1" align="center">C/<strong>T</strong>
</td><td rowspan="1" colspan="1" align="center">
<em>COMMD3/BMI1</em>
</td><td rowspan="1" colspan="1" align="center">0.83/0.78</td><td rowspan="1" colspan="1" align="center">4.60×10<sup>−6</sup>
</td><td rowspan="1" colspan="1" align="center">1.35 (1.15 to 1.58)</td><td rowspan="1" colspan="1" align="center">0.94/0.89</td><td rowspan="1" colspan="1" align="center">.04</td><td rowspan="1" colspan="1" align="center">1.83 (0.95 to 3.53)</td><td rowspan="1" colspan="1" align="center">0.83/0.78</td><td rowspan="1" colspan="1" align="center">.04</td><td rowspan="1" colspan="1" align="center">1.23 (0.97 to 1.57)</td><td rowspan="1" colspan="1" align="center">8.40×10<sup>−9</sup>
</td><td rowspan="1" colspan="1" align="center">1.40 (1.26 to 1.57)</td><td rowspan="1" colspan="1" align="center">.0004</td><td rowspan="1" colspan="1" align="center">1.33 (1.13 to 1.58)</td><td rowspan="1" colspan="1" align="center">.46</td><td rowspan="1" colspan="1" align="center">1.03 (0.65 to 1.63)</td><td rowspan="1" colspan="1" align="center">.11</td><td rowspan="1" colspan="1" align="center">1.26 (0.86 to 1.83)</td></tr><tr><td rowspan="1" colspan="1">rs7901152</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22873159</td><td rowspan="1" colspan="1" align="center">C/<strong>T</strong>
</td><td rowspan="1" colspan="1" align="center">
<em>PIP4K2A</em>
</td><td rowspan="1" colspan="1" align="center">0.67/0.61</td><td rowspan="1" colspan="1" align="center">5.84×10<sup>−5</sup>
</td><td rowspan="1" colspan="1" align="center">1.22 (1.07 to 1.39)</td><td rowspan="1" colspan="1" align="center">0.8/0.7</td><td rowspan="1" colspan="1" align="center">.006</td><td rowspan="1" colspan="1" align="center">1.61 (1.1 to 2.34)</td><td rowspan="1" colspan="1" align="center">0.83/0.78</td><td rowspan="1" colspan="1" align="center">.02</td><td rowspan="1" colspan="1" align="center">1.33 (1.04 to 1.70)</td><td rowspan="1" colspan="1" align="center">1.89×10<sup>−8</sup>
</td><td rowspan="1" colspan="1" align="center">1.33 (1.21 to 1.45)</td><td rowspan="1" colspan="1" align="center">.002</td><td rowspan="1" colspan="1" align="center">1.22 (1.06 to 1.40)</td><td rowspan="1" colspan="1" align="center">.14</td><td rowspan="1" colspan="1" align="center">1.18 (0.88 to 1.57)</td><td rowspan="1" colspan="1" align="center">.09</td><td rowspan="1" colspan="1" align="center">1.31 (0.80 to 1.94)</td></tr><tr><td rowspan="1" colspan="1">rs11013046</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22880589</td><td rowspan="1" colspan="1" align="center">
<strong>A</strong>/G</td><td rowspan="1" colspan="1" align="center">
<em>PIP4K2A</em>
</td><td rowspan="1" colspan="1" align="center">0.66/0.6</td><td rowspan="1" colspan="1" align="center">2.58×10<sup>−5</sup>
</td><td rowspan="1" colspan="1" align="center">1.23 (1.08 to 1.40)</td><td rowspan="1" colspan="1" align="center">0.48/0.39</td><td rowspan="1" colspan="1" align="center">.01</td><td rowspan="1" colspan="1" align="center">1.47 (1.07 to 2.01)</td><td rowspan="1" colspan="1" align="center">0.66/0.6</td><td rowspan="1" colspan="1" align="center">.009</td><td rowspan="1" colspan="1" align="center">1.31 (1.07 to 1.59)</td><td rowspan="1" colspan="1" align="center">2.92×10<sup>−9</sup>
</td><td rowspan="1" colspan="1" align="center">1.32 (1.21 to 1.43)</td><td rowspan="1" colspan="1" align="center">.001</td><td rowspan="1" colspan="1" align="center">1.23 (1.08 to 1.42)</td><td rowspan="1" colspan="1" align="center">.03</td><td rowspan="1" colspan="1" align="center">1.28 (0.98 to 1.68)</td><td rowspan="1" colspan="1" align="center">.04</td><td rowspan="1" colspan="1" align="center">1.34 (0.97 to 1.86)</td></tr><tr><td rowspan="1" colspan="1">rs7088318</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22892954</td><td rowspan="1" colspan="1" align="center">
<strong>A</strong>/C</td><td rowspan="1" colspan="1" align="center">
<em>PIP4K2A</em>
</td><td rowspan="1" colspan="1" align="center">0.65/0.59</td><td rowspan="1" colspan="1" align="center">5.25×10<sup>−6</sup>
</td><td rowspan="1" colspan="1" align="center">1.25 (1.10 to 1.42)</td><td rowspan="1" colspan="1" align="center">0.5/0.39</td><td rowspan="1" colspan="1" align="center">.001</td><td rowspan="1" colspan="1" align="center">1.65 (1.21 to 2.26)</td><td rowspan="1" colspan="1" align="center">0.81/0.75</td><td rowspan="1" colspan="1" align="center">.009</td><td rowspan="1" colspan="1" align="center">1.42 (1.12 to 1.80)</td><td rowspan="1" colspan="1" align="center">1.13×10<sup>−11</sup>
</td><td rowspan="1" colspan="1" align="center">1.40 (1.28 to 1.53)</td><td rowspan="1" colspan="1" align="center">.001</td><td rowspan="1" colspan="1" align="center">1.23 (1.07 to 1.41)</td><td rowspan="1" colspan="1" align="center">.009</td><td rowspan="1" colspan="1" align="center">1.38 (1.05 to 1.81)</td><td rowspan="1" colspan="1" align="center">.04</td><td rowspan="1" colspan="1" align="center">1.40 (0.95 to 2.05)</td></tr><tr><td rowspan="1" colspan="1">rs7075634</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">22893108</td><td rowspan="1" colspan="1" align="center">
<strong>C</strong>/T</td><td rowspan="1" colspan="1" align="center">
<em>PIP4K2A</em>
</td><td rowspan="1" colspan="1" align="center">0.66/0.6</td><td rowspan="1" colspan="1" align="center">1.78×10<sup>−5</sup>
</td><td rowspan="1" colspan="1" align="center">1.23 (1.08 to 1.40)</td><td rowspan="1" colspan="1" align="center">0.51/0.4</td><td rowspan="1" colspan="1" align="center">.003</td><td rowspan="1" colspan="1" align="center">1.62 (1.19 to 2.22)</td><td rowspan="1" colspan="1" align="center">0.82/0.75</td><td rowspan="1" colspan="1" align="center">.008</td><td rowspan="1" colspan="1" align="center">1.43 (1.12 to 1.82)</td><td rowspan="1" colspan="1" align="center">2.06×10<sup>−10</sup>
</td><td rowspan="1" colspan="1" align="center">1.38 (1.26 to 1.50)</td><td rowspan="1" colspan="1" align="center">.0001</td><td rowspan="1" colspan="1" align="center">1.29 (1.12 to 1.48)</td><td rowspan="1" colspan="1" align="center">.006</td><td rowspan="1" colspan="1" align="center">1.41 (1.08 to 1.84)</td><td rowspan="1" colspan="1" align="center">.07</td><td rowspan="1" colspan="1" align="center">1.34 (0.91 to 1.97)</td></tr></tbody></table></div><div id="largeobj_idm140295960693696" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3691938/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140296035269792"><p id="__p26" class="p p-first-last">* Chromosomal locations are based on hg18. Chr = chromosome; CI = confidence interval; NA = not applicable; OR = odds ratio; RAF = risk allele frequency.</p></div><div id="idm140296035268752"><p id="__p27" class="p p-first-last">† Bold denotes the allele that had a statistically significantly higher frequency in children with acute lymphoblastic leukemia than in the non–acute lymphoblastic leukemia control subjects (ie, risk allele for acute lymphoblastic leukemia).</p></div><div id="idm140296035267616"><p id="__p28" class="p p-first-last">‡ Ethnicity was defined by single nucleotide polymorphism genotype-based European, African, East Asian, and Native American genetic ancestry (see Methods), and numbers of acute lymphoblastic leukemia patients vs non–acute lymphoblastic leukemia control subjects are indicated.</p></div><div id="idm140296035266448"><p id="__p29" class="p p-first-last">§ Not applicable (single nucleotide polymorphism call rate &lt;95% in the respective replication series).</p></div><div id="idm140295999058400"><p id="__p30" class="p p-first-last">ǁ Odds ratio represents the increase in the risk of developing acute lymphoblastic leukemia for each copy of the risk allele compared with subjects who do not carry the risk allele. <em>P</em> values and odds ratios were estimated by the logistic regression test (two-sided).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3691938/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295997392672" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0001.jpg" title="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0001.jpg" src="/pmc/articles/PMC3691938/bin/jnci.j_djt042_f0001.jpg"></div></a><div id="largeobj_idm140295997392672" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3691938/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3691938/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3691938/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1. </a></div><!--caption a7--><div class="caption"><p id="__p31">Genome-wide association study (GWAS) results of acute lymphoblastic leukemia (ALL) susceptibility in multiethnic populations. Association between genotype and ALL was evaluated using a logistic regression model (two-sided) for 709509 single nucleotide polymorphisms (SNPs) in 1605 ALL case subjects and 6661 non-ALL control subjects. <em>P</em> values (-log10 <em>P</em>, <em>y</em> axis) were plotted against respective chromosomal position of each SNP (<em>x</em> axis). Gene symbols were indicated for four loci achieving genome-wide significance threshold (<em>P</em> &lt; 5×10<sup>−8</sup>; <strong>dashed blue line</strong>): <em>ARID5B</em> (10q21.2), <em>IKZF1</em> (7p12.2), <em>CEBPE</em> (14q11.2), and <em>BMI1-PIP4K2A</em> (10p12.31-12.2).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140296002314096" class="figure"><a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3691938_jnci.j_djt042_f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3691938/bin/jnci.j_djt042_f0002.jpg"></a></div><div id="largeobj_idm140296002314096" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3691938/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3691938/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2. </a></div><!--caption a7--><div class="caption"><p id="__p32">Association results and linkage disequilibrium (LD) at the 10p12.31-10p12.2 locus. <strong>A</strong>) The negative logarithm of the GWAS <em>P</em> values is plotted for each single nucleotide polymorphism (SNP) in a 600-kb window. <strong>Highlighted in red</strong> are six SNPs associated with ALL susceptibility with <em>P</em> &lt; 5×10<sup>–8</sup> (<strong>dashed orange line</strong>). Refseq genes are indicated below chromosomal position, which is based on hg18. <strong>B</strong>) LD at this locus is depicted based on <em>r</em>
<sup>2</sup> in HapMap CEU (European), YRI (African), and MEX (Hispanic) samples, and the plots were constructed using the HaploView software. Of six genome-wide statistically significant SNPs at this locus, four are in the intronic region of the <em>PIP4K2A</em> gene and are within a single LD block across ethnic groups. The other two SNPs are located in a region upstream of the <em>COMMD3</em> and <em>BMI1</em> genes. <em>P</em> value was calculated by two-sided logistic regression test.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3691938/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295950006608" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0003.jpg" title="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0003.jpg" src="/pmc/articles/PMC3691938/bin/jnci.j_djt042_f0003.jpg"></div></a><div id="largeobj_idm140295950006608" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3691938/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC3691938/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3691938/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3. </a></div><!--caption a7--><div class="caption"><p id="__p33">Expression quantitative loci (eQTL) analysis of <em>PIP4K2A</em> single nucleotide polymorphism rs7088318 in lymphoblastoid cell lines and primary acute lymphoblastic leukemia (ALL) blasts. <em>PIP4K2A</em> expression was determined in HapMap CEU (European) cell lines (<strong>A</strong>) and diagnostic blasts from children with ALL enrolled on St. Jude Total Therapy XIIIB/XV protocols (<strong>B</strong>). Genotype-expression association was evaluated using a linear regression model. AU = arbitrary unit.</p></div></div></div><p id="__p34">We next sought to validate the association at the novel susceptibility locus <em>BM1</em>-<em>PIP4K2A</em> in three independent case–control series in an ethnicity-specific manner: European Americans (n = 574 case subjects and 2601 control subjects), African Americans (n = 128 case subjects and 1075 control subjects), and Hispanic Americans (n = 143 case subjects and 640 control subjects). The top <em>PIP4K2A</em> SNP, rs7088318, was statistically significantly associated with ALL susceptibility in all three ethnic groups: European Americans (<em>P</em> = .001); African Americans, (<em>P</em> = .009); and Hispanic Americans, (<em>P</em> = .04) (<a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The remaining five SNPs at this locus were all replicated in at least one ethnic group (<a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>) and so was the independent association at rs4748793 (<em>P</em>
<sub>rs4748793</sub> = 3.5×10<sup>−4</sup>, after adjusting for rs7088318 and genetic ancestry in replication series).</p><p id="__p35">The genetic underpinning of childhood ALL susceptibility is likely to be complex, and current evidence strongly favors a polygenic model of ALL risk (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107679">11</a>). We next examined the combined effects of four genome-wide statistically significant loci (<a href="/pmc/articles/PMC3691938/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) on ALL susceptibility by multimarker analyses on the basis of genotype at top SNPs at each locus: rs10821936 at <em>ARID5B</em>, rs11978267 at <em>IKZF1</em>, rs7088318 at <em>PIP4K2A</em>, and rs4982731 at <em>CEBPE</em>. In the combined GWAS and replication series (n = 2450 case subjects and 10 977 control subjects), there was a positive correlation (<em>P</em> = 1.6×10<sup>−5</sup>; correlation coefficient = 0.39, 95% confidence interval [CI] = 0.33 to 0.45) between the number of risk alleles at these four SNPs and relative ALL risk (ie, odds ratio, relative to subjects carrying 0–1 copy of the risk alleles) (<a href="/pmc/articles/PMC3691938/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>). For example, subjects with six to eight copies of risk alleles (n = 252 case subjects and 314 control subjects) were at ninefold (95% CI = 6.9 to 11.8) higher risk of developing ALL than those with zero to one copy of the risk alleles (n = 153 case subjects and 1753 control subjects). Cumulative effects of these variants were also estimated separately in the GWAS and replication series (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 6</a>, available online).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC3691938/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295958523984" class="figure"><a href="/pmc/articles/PMC3691938/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3691938_jnci.j_djt042_f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is jnci.j_djt042_f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3691938/bin/jnci.j_djt042_f0004.jpg"></a></div><div id="largeobj_idm140295958523984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3691938/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC3691938/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4. </a></div><!--caption a7--><div class="caption"><p id="__p36">Cumulative effects of top variants at 7p12.2, 10p12.31-12.2, 10q21.2, and 14q11.2 on acute lymphoblastic leukemia (ALL) susceptibility. Odds ratio for ALL is plotted against the number of risk alleles at rs10821936 (<em>ARID5B</em>), rs11978267 (<em>IKZF1</em>), rs7088318 (<em>PIP4K2A</em>), and rs4982731 (<em>CEBPE</em>). <strong>Bars</strong> indicate 95% confidence intervals, and <strong>dotted line</strong> is odds ratio of 1. Odds ratio was estimated by logistical regression test combining genome-wide association study and replication series (n = 2450 ALL case subjects and n = 10 977 control subjects) after adjusting for genetic ancestry.</p></div></div></div><p id="__p37" class="p">Finally, because the incidence of ALL is highly related to age with the majority of cases occurring in children aged 2 to 5 years (<a href="#CIT0002" rid="CIT0002" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107668">2</a>), we examined the effects of ALL susceptibility variants by age. Combining GWAS and replication series, risk allele frequency at rs10821936 was higher in children who developed ALL before 10 years of age than in those diagnosed with ALL at ages older than 10 years (<em>P</em> = .02, .18, .007 in European Americans, African Americans, and Hispanic Americans, respectively) (<a href="/pmc/articles/PMC3691938/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), most evidently in hyperdiploid ALL (<a href="/pmc/articles/PMC3691938/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Consistently, when we further classified children into age groups of those aged less than 5 years, those aged 5 to 10 years, and those aged greater than 10 years, there was a trend for decreasing allelic odds ratio (ie, relative risk of ALL conferred by each copy of the C allele at rs10821936) as age increased: 2.01 (95% CI = 1.85 to 2.19), 1.8 (95% CI = 1.6 to 2.02), and 1.48 (95% CI = 1.3 to 1.68), respectively (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 7</a>, available online). Similar results were observed when we restricted the analysis to hyperdiploid ALL (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 7</a>, available online). In contrast, the effects of <em>IKZF1</em>, <em>CEBPE</em>, and <em>PIP4K2A</em> variants did not differ between age groups (data not shown). Together, these results suggest possible modifying effects of age on genetic predisposition to ALL.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3. </h3><!--caption a7--><div class="caption"><p id="__p38">The frequency of acute lymphoblastic leukemia risk (ALL) allele at rs10821936 (<em>ARID5B</em>) by ALL status, age at diagnosis, ALL molecular subtype, and ethnicity </p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140295953239728" class="xtable"><table frame="vsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="center">
</th><th colspan="4" rowspan="1" valign="bottom" align="center">European American<sup>*</sup> (RAF/No.)<sup>†</sup>
</th><th colspan="4" rowspan="1" valign="bottom" align="center">African American<sup>*</sup> (RAF/No.)<sup>†</sup>
</th><th colspan="4" rowspan="1" valign="bottom" align="center">Hispanic American<sup>*</sup> (RAF/No.)<sup>†</sup>
</th></tr><tr><th rowspan="2" colspan="1" valign="bottom" align="center">Non-ALL control</th><th colspan="3" rowspan="1" valign="bottom" align="center">Age at ALL diagnosis, years</th><th rowspan="2" colspan="1" valign="bottom" align="center">Non-ALL control</th><th colspan="3" rowspan="1" valign="bottom" align="center">Age at ALL diagnosis, years</th><th rowspan="2" colspan="1" valign="bottom" align="center">Non-ALL control</th><th colspan="3" rowspan="1" valign="bottom" align="center">Age at ALL diagnosis, years</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">&lt;10</th><th rowspan="1" colspan="1" valign="bottom" align="center">≥10</th><th rowspan="1" colspan="1" valign="bottom" align="center">All ages</th><th rowspan="1" colspan="1" valign="bottom" align="center">&lt;10</th><th rowspan="1" colspan="1" valign="bottom" align="center">≥10</th><th rowspan="1" colspan="1" valign="bottom" align="center">All ages</th><th rowspan="1" colspan="1" valign="bottom" align="center">&lt;10</th><th rowspan="1" colspan="1" valign="bottom" align="center">≥10</th><th rowspan="1" colspan="1" valign="bottom" align="center">All ages</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">ALL (all subtypes)</td><td rowspan="2" colspan="1" align="center">0.33/3987</td><td rowspan="1" colspan="1" align="center">0.49/1,219<sup>‡</sup>
</td><td rowspan="1" colspan="1" align="center">0.44/327<sup>‡</sup>
</td><td rowspan="1" colspan="1" align="center">0.48/1546</td><td rowspan="2" colspan="1" align="center">0.23/2438</td><td rowspan="1" colspan="1" align="center">0.34/156</td><td rowspan="1" colspan="1" align="center">0.27/61</td><td rowspan="1" colspan="1" align="center">0.32/217</td><td rowspan="2" colspan="1" align="center">0.46/1648</td><td rowspan="1" colspan="1" align="center">0.62/355<sup>‡</sup>
</td><td rowspan="1" colspan="1" align="center">0.5/93<sup>‡</sup>
</td><td rowspan="1" colspan="1" align="center">0.59/448</td></tr><tr><td rowspan="2" colspan="1">Hyperdiploid</td><td rowspan="2" colspan="1" align="center">0.57/351<sup>‡</sup>
</td><td rowspan="2" colspan="1" align="center">0.45/65<sup>‡</sup>
</td><td rowspan="2" colspan="1" align="center">0.55/416</td><td rowspan="2" colspan="1" align="center">0.4/29<sup>‡</sup>
</td><td rowspan="2" colspan="1" align="center">0.13/8<sup>‡</sup>
</td><td rowspan="2" colspan="1" align="center">0.34/37</td><td rowspan="2" colspan="1" align="center">0.67/98</td><td rowspan="2" colspan="1" align="center">0.6/15</td><td rowspan="2" colspan="1" align="center">0.66/113</td></tr><tr><td rowspan="2" colspan="1">
</td><td rowspan="2" colspan="1" align="center">
</td><td rowspan="2" colspan="1" align="center">
</td></tr><tr><td rowspan="2" colspan="1">
<em>ETV6-RUNX1</em>
</td><td rowspan="2" colspan="1" align="center">0.42/339</td><td rowspan="2" colspan="1" align="center">0.37/23</td><td rowspan="2" colspan="1" align="center">0.41/362</td><td rowspan="2" colspan="1" align="center">0.35/50</td><td rowspan="2" colspan="1" align="center">0.25/2</td><td rowspan="2" colspan="1" align="center">0.34/52</td><td rowspan="2" colspan="1" align="center">0.56/85</td><td rowspan="2" colspan="1" align="center">0.25/4</td><td rowspan="2" colspan="1" align="center">0.54/89</td></tr><tr><td rowspan="1" colspan="1">
</td><td rowspan="1" colspan="1" align="center">
</td><td rowspan="1" colspan="1" align="center">
</td></tr></tbody></table></div><div id="largeobj_idm140295953239728" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3691938/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140295955746096"><p id="__p39" class="p p-first-last">
<sup>*</sup> Ethnicity was defined by single nucleotide polymorphims genotype-based European, African, East Asian, and Native American genetic ancestry (see Methods). RAF = risk allele frequency.</p></div><div id="idm140295955744528"><p id="__p40" class="p p-first-last">
<sup>†</sup> Numbers indicate the C allele frequency at rs10821936 and total number of subjects in each category (eg, in 3987 EA non-ALL control subjects, the C allele frequency is 33%), combining the genome-wide association study and replication series.</p></div><div id="idm140295952586544"><p id="__p41" class="p p-first-last">
<sup>‡</sup> Differences in allele frequency were statistically significant between two age groups (ie, <em>P</em> &lt; .05 as determined by logistical regression test after adjusting genetic ancestry).</p></div></div></div></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">Discussion</h2><p id="__p42" class="p p-first">Non-European populations are indisputably underrepresented in GWASs (<a href="#CIT0012" rid="CIT0012" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_352915435">12</a>,<a href="#CIT0013" rid="CIT0013" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107652">13</a>). Recent GWASs in diverse populations reveal both similarities and differences in genetic architecture of disease susceptibility among ethnic groups. We reported here the first GWAS of ALL in multiethnic populations (including African Americans and Hispanic Americans), in which we discovered novel susceptibility variants at <em>BMI1</em>-<em>PIP4K2A</em> locus and comprehensively compared associations at known susceptibility loci (<em>ARID5B</em>, <em>IKZF1</em>, <em>CEBPE</em>, and <em>CDKN2A/2B</em>) in different ethnic groups.</p><p id="__p43">The discovery of the <em>BMI1</em>-<em>PIP4K2A</em> susceptibility variants that were not detected by previous European-only GWASs of ALL (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107633">5–7</a>) raises the question of improved power as a result of population diversity. When the disease variant is substantially more common in non-European populations, GWASs in these ethnic groups obviously heighten the power to discover such loci compared with GWASs in Europeans with the same sample size, as illustrated in the case of the <em>KCNQ1</em> locus in type 2 diabetes (<a href="#CIT0037" rid="CIT0037" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107649">37</a>) and by simulation using the 1000 Genome data (<a href="#CIT0038" rid="CIT0038" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107669">38</a>). However, this is probably unlikely to explain the <em>BMI1</em>-<em>PIP4K2A</em> variants that are actually less frequent in African Americans than European Americans, although they are modestly more common in Hispanic Americans. Another plausible explanation is population differences in LD around the <em>BMI1</em>-<em>PIP4K2A</em> variants: if the causal variant is better tagged in African populations, including African Americans in the GWAS is likely to improve the sensitivity to detect the signal at the genome-wide threshold. At this locus, LD pattern is similar between European Americans and Hispanic Americans, but is much less extensive in African Americans, as expected (<a href="/pmc/articles/PMC3691938/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). Lastly, population heterogeneity in effect size of the risk allele can also influence the sample size required in GWAS. In type 2 diabetes, the allelic risk at multiple disease variants is statistically significantly greater in the Japanese population than in the European population, although these variants are statistically significant in both ethnic groups (<a href="#CIT0039" rid="CIT0039" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107643">39</a>). Interestingly, the per-allele odds ratio at rs7088318 was greater in African Americans and Hispanic Americans relative to European Americans (<a href="/pmc/articles/PMC3691938/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), consistent with possibly improved power when these non-European populations are included in GWASs of ALL.</p><p id="__p44">The population diversity in our GWAS also offered a unique opportunity to examine the genetic basis of ethnic differences in ALL incidence (<a href="#CIT0020" rid="CIT0020" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107666">20</a>). Variants at the <em>ARID5B</em>, <em>IKZF1</em>, and <em>BMI1</em>-<em>PIP4K2A</em> loci were associated with ALL susceptibility across ethnic groups (ie, the SNP with the strongest association at each locus was statistically significant in all three populations), suggesting common causal variants across ancestral backgrounds. In contrast, <em>CEBPE</em> SNPs were strongly related to ALL risk in European Americans, with variable effects in non-European populations. Such disparities might reflect existence of true population-specific disease variants but can also arise from population differences in genomic structure at these loci (differences in LD between tagging SNPs and causal variants). Further, the frequency of ALL risk variants at the <em>ARID5B</em> and <em>PIP4K2A</em> loci vary substantially by ethnicity in a pattern consistent with their possible contribution to ethnic differences in ALL incidence (<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107664">21</a>) (<a href="/pmc/articles/PMC3691938/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Tables 1</span></a> and 2).</p><p id="__p45">The genetic basis of ALL is most likely to be polygenic (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107616">11</a>). However, it should be noted that carrying ALL risk variants at merely four SNPs (<em>ARID5B</em>, <em>IKZF1</em>, <em>CEBPE</em>, and <em>PIP4K2A</em>) conferred a ninefold increase in disease susceptibility (<a href="/pmc/articles/PMC3691938/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4</span></a>) and these GWAS signals are concentrated to genes directly related to hematopoietic differentiation and development [<em>ARID5B</em> (<a href="#CIT0040" rid="CIT0040" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107638">40</a>), <em>IKZF1</em> (<a href="#CIT0041" rid="CIT0041" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107621">41</a>), and <em>CEBPE</em> (<a href="#CIT0042" rid="CIT0042" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107653">42</a>)]. We hypothesize that genetic predisposition to ALL might be largely mediated by robust effects of a modest number of key genes rather than cumulative effects of tens of thousands of variants with small effects (OR = 1.1–1.2), as seen in GWASs of other common diseases (<a href="#CIT0043" rid="CIT0043" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107620">43</a>,<a href="#CIT0044" rid="CIT0044" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107670">44</a>). In fact, it is estimated that variants in <em>ARID5B</em>, <em>IKZF1</em>, <em>CEBPE</em>, and <em>CDKN2A/2B</em> account for approximately one-third of ALL risk conferred by common genetic polymorphisms (<a href="#CIT0011" rid="CIT0011" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107663">11</a>). The effect of ALL susceptibility variants was particularly strong in younger children (<a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 7</a>, available online), suggesting possible variation in ALL genetic predisposition at different developmental stages. Interestingly, several of the GWAS hits are also frequently targeted by somatic aberrations in ALL cells [<em>IKZF1</em> (<a href="#CIT0045" rid="CIT0045" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107675">45</a>) and <em>CEBPE</em> (<a href="#CIT0046" rid="CIT0046" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107661">46</a>)]. Susceptibility variants in <em>ARID5B</em> are also related to gross cytogenetic abnormalities in ALL blasts (ie, hyperdiploidy) (<a href="/pmc/articles/PMC3691938/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>), consistent with prior reports from us and others (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107624">5</a>,<a href="#CIT0006" rid="CIT0006" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544244956">6</a>,<a href="#CIT0021" rid="CIT0021" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107608">21</a>,<a href="#CIT0047" rid="CIT0047" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107655">47</a>). The C allele at rs10821936 confers a greater disease risk for this subtype of ALL (<a href="#CIT0005" rid="CIT0005" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107605">5</a>), although the molecular mechanisms linking <em>ARID5B</em> to aneuploidy remain unclear. Nevertheless, these observations raise the possibility of interactions between inherited (germline) and acquired (somatic) genetic variations in the pathogenesis of ALL.</p><p id="__p46">
<em>PIP4K2A</em> is a member of the family of enzymes that catalyze phosphorylation of phosphatidylinositol-5-phosphate to form phosphatidylinositol-5,4-bisphosphate (PIP2), a precursor of the important second messenger molecule, PIP3. Upon B-cell receptor activation, <em>PIP4K2A</em> is directly recruited by BTK to the plasma membrane as a means of stimulating local PIP2 synthesis (<a href="#CIT0048" rid="CIT0048" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107607">48</a>). Similarly, PIP5K enzymes also interact with the Rho-family small GTP-binding proteins (eg, Rac1) to regulate membrane PIP2 synthesis and PI3K and PLC signaling in B cells (<a href="#CIT0049" rid="CIT0049" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107677">49</a>). Although these observations point to <em>PIP4K2A</em> as a plausible regulator of lymphoid cell differentiation, functional studies are warranted to determine the mechanisms linking <em>PIP4K2A</em> to leukemogenesis.</p><p id="__p47" class="p p-last">Our study was not without limitations. Further fine-mapping and/or resequencing of the causal variants will be required to completely characterize the contribution of <em>BMI1-PIP4K2A</em> variants to ALL etiology in the context of ethnicity. Future GWASs and/or admixture mapping with even larger samples of non-European populations are needed to comprehensively characterize genetic variants that predispose children to this most common childhood cancer and to fully understand the genetic basis of ethnic disparity in ALL. Nonetheless, we argue that a GWAS approach that includes multiethnic subjects is likely to be more effective in discovering ALL risk loci than analyses selectively procuring large samples in a single population, as suggested by observations from GWASs of other diseases (<a href="#CIT0013" rid="CIT0013" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107674">13</a>,<a href="#CIT0015" rid="CIT0015" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107645">15</a>,<a href="#CIT0016" rid="CIT0016" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107604">16</a>,<a href="#CIT0050" rid="CIT0050" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_321107615">50</a>).</p></div><div id="__sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec5title">Funding</h2><p id="__p48" class="p p-first">This work was supported by the National Institutes of Health (grant numbers <span>CA156449</span>;, <span>CA21765</span>;, <span>CA36401</span>;, <span>CA98543</span>;, <span>CA114766</span>;, <span>CA98413</span>;, <span>CA140729</span>;, and <span>U01GM92666</span>), in part by the Intramural Program of the National Cancer Institute, and by the American Lebanese Syrian Associated Charities (ALSAC).</p><p id="__p49" class="p p-last">Genome-wide genotyping of COG P9904/P9905 samples was performed by the Center for Molecular Medicine with the generous financial support from the Jeffrey Pride Foundation and the National Childhood Cancer Foundation. S.P. Hunger is the Ergen Family Chair in Pediatric Cancer, and J.J. Yang is supported by the American Society of Hematology Scholar Award, Alex Lemonade Stand Foundation for Childhood Cancer Young Investigator Grant, and by the Order of St. Francis Foundation.</p></div><div id="idm140295952830112" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140295952830112title">Notes</h2><p id="__p50">H. Xu and W. Yang contributed equally to this work.</p><p id="__p51">The study sponsors were not directly involved in the design of the study, the collection, analysis, and interpretation of the data, the writing of the manuscript, or the decision to submit the manuscript.</p><p id="__p52">We thank the patients and parents who participated in the St. Jude and COG protocols included in this study, the clinicians and research staff at the St. Jude and COG institutions, and J. Pullen (University of Mississippi at Jackson) for assistance in classification of patients with ALL. We thank M. Shriver (Pennsylvania State University) for sharing SNP genotype data of the Native American references. Full acknowledgements of the dbGAP datasets are provided in the <a href="http://jnci.oxfordjournals.org/lookup/suppl/doi:10.1093/jnci/djt042/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Data</a> (available online).</p></div><div id="idm140295960385216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140295960385216title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="CIT0001">1. <span class="mixed-citation">
Pui CH, Evans WE.
<span class="ref-title">Treatment of acute lymphoblastic leukemia</span>.
<span class="ref-journal">N Engl J Med</span>.
2006;
<span class="ref-vol">354</span>(<span class="ref-iss">2</span>):166–178
 [<a href="/pubmed/16407512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Treatment+of+acute+lymphoblastic+leukemia&amp;author=CH+Pui&amp;author=WE+Evans&amp;volume=354&amp;issue=2&amp;publication_year=2006&amp;pages=166-178&amp;pmid=16407512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0002">2. <span class="mixed-citation">
Greaves M.
<span class="ref-title">Infection, immune responses and the aetiology of childhood leukaemia</span>.
<span class="ref-journal">Nat Rev Cancer</span>.
2006;
<span class="ref-vol">6</span>(<span class="ref-iss">3</span>):193–203
 [<a href="/pubmed/16467884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Infection,+immune+responses+and+the+aetiology+of+childhood+leukaemia&amp;author=M+Greaves&amp;volume=6&amp;issue=3&amp;publication_year=2006&amp;pages=193-203&amp;pmid=16467884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0003">3. <span class="mixed-citation">
Hasle H, Clemmensen IH, Mikkelsen M.
<span class="ref-title">Risks of leukaemia and solid tumours in individuals with Down’s syndrome</span>.
<span class="ref-journal">Lancet</span>.
2000;
<span class="ref-vol">355</span>(<span class="ref-iss">9199</span>):165–169
 [<a href="/pubmed/10675114" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Risks+of+leukaemia+and+solid+tumours+in+individuals+with+Down’s+syndrome&amp;author=H+Hasle&amp;author=IH+Clemmensen&amp;author=M+Mikkelsen&amp;volume=355&amp;issue=9199&amp;publication_year=2000&amp;pages=165-169&amp;pmid=10675114&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0004">4. <span class="mixed-citation">
Morrell D, Cromartie E, Swift M.
<span class="ref-title">Mortality and cancer incidence in 263 patients with ataxia-telangiectasia</span>.
<span class="ref-journal">J Natl Cancer Inst</span>.
1986;
<span class="ref-vol">77</span>(<span class="ref-iss">1</span>):89–92
 [<a href="/pubmed/3459930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Mortality+and+cancer+incidence+in+263+patients+with+ataxia-telangiectasia&amp;author=D+Morrell&amp;author=E+Cromartie&amp;author=M+Swift&amp;volume=77&amp;issue=1&amp;publication_year=1986&amp;pages=89-92&amp;pmid=3459930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0005">5. <span class="mixed-citation">
Trevino LR, Yang W, French D, et al. 
<span class="ref-title">Germline genomic variants associated with childhood acute lymphoblastic leukemia</span>.
<span class="ref-journal">Nat Genet</span>.
2009;
<span class="ref-vol">41</span>(<span class="ref-iss">9</span>):1001–1005
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762391/">PMC free article</a>]</span> [<a href="/pubmed/19684603" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Germline+genomic+variants+associated+with+childhood+acute+lymphoblastic+leukemia&amp;author=LR+Trevino&amp;author=W+Yang&amp;author=D+French&amp;volume=41&amp;issue=9&amp;publication_year=2009&amp;pages=1001-1005&amp;pmid=19684603&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0006">6. <span class="mixed-citation">
Papaemmanuil E, Hosking FJ, Vijayakrishnan J, et al. 
<span class="ref-title">Loci on 7p12.2, 10q21.2 and 14q11.2 are associated with risk of childhood acute lymphoblastic leukemia</span>.
<span class="ref-journal">Nat Genet</span>.
2009;
<span class="ref-vol">41</span>(<span class="ref-iss">9</span>):1006–1010
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4915548/">PMC free article</a>]</span> [<a href="/pubmed/19684604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Loci+on+7p12.2,+10q21.2+and+14q11.2+are+associated+with+risk+of+childhood+acute+lymphoblastic+leukemia&amp;author=E+Papaemmanuil&amp;author=FJ+Hosking&amp;author=J+Vijayakrishnan&amp;volume=41&amp;issue=9&amp;publication_year=2009&amp;pages=1006-1010&amp;pmid=19684604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0007">7. <span class="mixed-citation">
Ellinghaus E, Stanulla M, Richter G, et al. 
<span class="ref-title">Identification of germline susceptibility loci in ETV6-RUNX1-rearranged childhood acute lymphoblastic leukemia</span>.
<span class="ref-journal">Leukemia</span>.
2012;
<span class="ref-vol">26</span>(<span class="ref-iss">5</span>):902–909
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3356560/">PMC free article</a>]</span> [<a href="/pubmed/22076464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Identification+of+germline+susceptibility+loci+in+ETV6-RUNX1-rearranged+childhood+acute+lymphoblastic+leukemia&amp;author=E+Ellinghaus&amp;author=M+Stanulla&amp;author=G+Richter&amp;volume=26&amp;issue=5&amp;publication_year=2012&amp;pages=902-909&amp;pmid=22076464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0008">8. <span class="mixed-citation">
Sherborne AL, Hosking FJ, Prasad RB, et al. 
<span class="ref-title">Variation in CDKN2A at 9p21.3 influences childhood acute lymphoblastic leukemia risk</span>.
<span class="ref-journal">Nat Genet</span>.
2010;
<span class="ref-vol">42</span>(<span class="ref-iss">6</span>):492–494
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3434228/">PMC free article</a>]</span> [<a href="/pubmed/20453839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variation+in+CDKN2A+at+9p21.3+influences+childhood+acute+lymphoblastic+leukemia+risk&amp;author=AL+Sherborne&amp;author=FJ+Hosking&amp;author=RB+Prasad&amp;volume=42&amp;issue=6&amp;publication_year=2010&amp;pages=492-494&amp;pmid=20453839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0009">9. <span class="mixed-citation">
Orsi L, Rudant J, Bonaventure A, et al. 
<span class="ref-title">Genetic polymorphisms and childhood acute lymphoblastic leukemia: GWAS of the ESCALE study (SFCE)</span>.
<span class="ref-journal">Leukemia</span>.
. 2012
<span class="ref-vol">26</span>(<span class="ref-iss">12</span>):2561–2564
 [<a href="/pubmed/22660188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Genetic+polymorphisms+and+childhood+acute+lymphoblastic+leukemia:+GWAS+of+the+ESCALE+study+(SFCE)&amp;author=L+Orsi&amp;author=J+Rudant&amp;author=A+Bonaventure&amp;volume=26&amp;issue=12&amp;pages=2561-2564&amp;pmid=22660188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0010">10. <span class="mixed-citation">
Fletcher O, Houlston RS.
<span class="ref-title">Architecture of inherited susceptibility to common cancer</span>.
<span class="ref-journal">Nat Rev Cancer</span>.
2010;
<span class="ref-vol">10</span>(<span class="ref-iss">5</span>):353–361
 [<a href="/pubmed/20414203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Architecture+of+inherited+susceptibility+to+common+cancer&amp;author=O+Fletcher&amp;author=RS+Houlston&amp;volume=10&amp;issue=5&amp;publication_year=2010&amp;pages=353-361&amp;pmid=20414203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0011">11. <span class="mixed-citation">
Enciso-Mora V, Hosking FJ, Sheridan E, et al. 
<span class="ref-title">Common genetic variation contributes significantly to the risk of childhood B-cell precursor acute lymphoblastic leukemia</span>.
<span class="ref-journal">Leukemia</span>.
2012;
<span class="ref-vol">26</span>(<span class="ref-iss">10</span>):2212–2215
 [<a href="/pubmed/22456626" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Common+genetic+variation+contributes+significantly+to+the+risk+of+childhood+B-cell+precursor+acute+lymphoblastic+leukemia&amp;author=V+Enciso-Mora&amp;author=FJ+Hosking&amp;author=E+Sheridan&amp;volume=26&amp;issue=10&amp;publication_year=2012&amp;pages=2212-2215&amp;pmid=22456626&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0012">12. <span class="mixed-citation">
Bustamante CD, Burchard EG, De la Vega FM.
<span class="ref-title">Genomics for the world</span>.
<span class="ref-journal">Nature</span>.
2011;
<span class="ref-vol">475</span>(<span class="ref-iss">7355</span>):163–165
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3708540/">PMC free article</a>]</span> [<a href="/pubmed/21753830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genomics+for+the+world&amp;author=CD+Bustamante&amp;author=EG+Burchard&amp;author=FM+De+la+Vega&amp;volume=475&amp;issue=7355&amp;publication_year=2011&amp;pages=163-165&amp;pmid=21753830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0013">13. <span class="mixed-citation">
Rosenberg NA, Huang L, Jewett EM, et al. 
<span class="ref-title">Genome-wide association studies in diverse populations</span>.
<span class="ref-journal">Nat Rev Genet</span>.
2010;
<span class="ref-vol">11</span>(<span class="ref-iss">5</span>):356–366
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3079573/">PMC free article</a>]</span> [<a href="/pubmed/20395969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genome-wide+association+studies+in+diverse+populations&amp;author=NA+Rosenberg&amp;author=L+Huang&amp;author=EM+Jewett&amp;volume=11&amp;issue=5&amp;publication_year=2010&amp;pages=356-366&amp;pmid=20395969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0014">14. <span class="mixed-citation">
Ioannidis JP.
<span class="ref-title">Population-wide generalizability of genome-wide discovered associations</span>.
<span class="ref-journal">J Natl Cancer Inst</span>.
2009;
<span class="ref-vol">101</span>(<span class="ref-iss">19</span>):1297–1299
 [<a href="/pubmed/19726754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=Population-wide+generalizability+of+genome-wide+discovered+associations&amp;author=JP+Ioannidis&amp;volume=101&amp;issue=19&amp;publication_year=2009&amp;pages=1297-1299&amp;pmid=19726754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0015">15. <span class="mixed-citation">
Ntzani EE, Liberopoulos G, Manolio TA, et al. 
<span class="ref-title">Consistency of genome-wide associations across major ancestral groups</span>.
<span class="ref-journal">Hum Genet</span>.
2012;
<span class="ref-vol">131</span>(<span class="ref-iss">7</span>):1057–1071
 [<a href="/pubmed/22183176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Consistency+of+genome-wide+associations+across+major+ancestral+groups&amp;author=EE+Ntzani&amp;author=G+Liberopoulos&amp;author=TA+Manolio&amp;volume=131&amp;issue=7&amp;publication_year=2012&amp;pages=1057-1071&amp;pmid=22183176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0016">16. <span class="mixed-citation">
Torgerson DG, Ampleford EJ, Chiu GY, et al. 
<span class="ref-title">Meta-analysis of genome-wide association studies of asthma in ethnically diverse North American populations</span>.
<span class="ref-journal">Nat Genet</span>.
2011;
<span class="ref-vol">43</span>(<span class="ref-iss">9</span>):887–892
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3445408/">PMC free article</a>]</span> [<a href="/pubmed/21804549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+of+asthma+in+ethnically+diverse+North+American+populations&amp;author=DG+Torgerson&amp;author=EJ+Ampleford&amp;author=GY+Chiu&amp;volume=43&amp;issue=9&amp;publication_year=2011&amp;pages=887-892&amp;pmid=21804549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0017">17. <span class="mixed-citation">
Postel-Vinay S, Veron AS, Tirode F, et al. 
<span class="ref-title">Common variants near TARDBP and EGR2 are associated with susceptibility to Ewing sarcoma</span>.
<span class="ref-journal">Nat Genet</span>.
2012;
<span class="ref-vol">44</span>(<span class="ref-iss">3</span>):323–327
 [<a href="/pubmed/22327514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+near+TARDBP+and+EGR2+are+associated+with+susceptibility+to+Ewing+sarcoma&amp;author=S+Postel-Vinay&amp;author=AS+Veron&amp;author=F+Tirode&amp;volume=44&amp;issue=3&amp;publication_year=2012&amp;pages=323-327&amp;pmid=22327514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0018">18. <span class="mixed-citation">
Moonesinghe R, Ioannidis JP, Flanders WD, et al. 
<span class="ref-title">Estimating the contribution of genetic variants to difference in incidence of disease between population groups</span>.
<span class="ref-journal">Eur J Hum Genet</span>.
2012;
<span class="ref-vol">20</span>(<span class="ref-iss">8</span>):831–836
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3400729/">PMC free article</a>]</span> [<a href="/pubmed/22333905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Estimating+the+contribution+of+genetic+variants+to+difference+in+incidence+of+disease+between+population+groups&amp;author=R+Moonesinghe&amp;author=JP+Ioannidis&amp;author=WD+Flanders&amp;volume=20&amp;issue=8&amp;publication_year=2012&amp;pages=831-836&amp;pmid=22333905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0019">19. <span class="mixed-citation">
Dores GM, Devesa SS, Curtis RE, et al. 
<span class="ref-title">Acute leukemia incidence and patient survival among children and adults in the United States</span>.
2001–2007
<span class="ref-journal">Blood</span>.
2012;
<span class="ref-vol">119</span>(<span class="ref-iss">1</span>): 34–43
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3251235/">PMC free article</a>]</span> [<a href="/pubmed/22086414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Acute+leukemia+incidence+and+patient+survival+among+children+and+adults+in+the+United+States&amp;author=GM+Dores&amp;author=SS+Devesa&amp;author=RE+Curtis&amp;volume=119&amp;issue=1&amp;publication_year=2012&amp;pages=2001-2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0020">20. <span class="mixed-citation">
Linabery AM, Ross JA.
<span class="ref-title">Trends in childhood cancer incidence in the U.S</span>. (1992–2004). <span class="ref-journal">Cancer</span>.
2008;
<span class="ref-vol">112</span>(<span class="ref-iss">2</span>): 416–432
 [<a href="/pubmed/18074355" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Trends+in+childhood+cancer+incidence+in+the+U.S&amp;author=AM+Linabery&amp;author=JA+Ross&amp;volume=112&amp;issue=2&amp;publication_year=2008&amp;pages=1992-2004&amp;pmid=18311781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0021">21. <span class="mixed-citation">
Xu H, Cheng C, Devidas M, et al. 
<span class="ref-title">ARID5B genetic polymorphisms contribute to racial disparities in the incidence and treatment outcome of childhood acute lymphoblastic leukemia</span>.
<span class="ref-journal">J Clin Oncol</span>.
2012;
<span class="ref-vol">30</span>(<span class="ref-iss">7</span>):751–757
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3295551/">PMC free article</a>]</span> [<a href="/pubmed/22291082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Oncol&amp;title=ARID5B+genetic+polymorphisms+contribute+to+racial+disparities+in+the+incidence+and+treatment+outcome+of+childhood+acute+lymphoblastic+leukemia&amp;author=H+Xu&amp;author=C+Cheng&amp;author=M+Devidas&amp;volume=30&amp;issue=7&amp;publication_year=2012&amp;pages=751-757&amp;pmid=22291082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0022">22. <span class="mixed-citation">
Yang W, Trevino LR, Yang JJ, et al. 
<span class="ref-title">ARID5B SNP rs10821936 is associated with risk of childhood acute lymphoblastic leukemia in blacks and contributes to racial differences in leukemia incidence</span>.
<span class="ref-journal">Leukemia</span>.
2010;
<span class="ref-vol">24</span>(<span class="ref-iss">4</span>):894–896
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4607263/">PMC free article</a>]</span> [<a href="/pubmed/20054350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=ARID5B+SNP+rs10821936+is+associated+with+risk+of+childhood+acute+lymphoblastic+leukemia+in+blacks+and+contributes+to+racial+differences+in+leukemia+incidence&amp;author=W+Yang&amp;author=LR+Trevino&amp;author=JJ+Yang&amp;volume=24&amp;issue=4&amp;publication_year=2010&amp;pages=894-896&amp;pmid=20054350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0023">23. <span class="mixed-citation">
Borowitz MJ, Devidas M, Hunger SP, et al. 
<span class="ref-title">Clinical significance of minimal residual disease in childhood acute lymphoblastic leukemia and its relationship to other prognostic factors: a Children’s Oncology Group study</span>.
<span class="ref-journal">Blood</span>.
2008;
<span class="ref-vol">111</span>(<span class="ref-iss">12</span>):5477–5485
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2424148/">PMC free article</a>]</span> [<a href="/pubmed/18388178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Clinical+significance+of+minimal+residual+disease+in+childhood+acute+lymphoblastic+leukemia+and+its+relationship+to+other+prognostic+factors:+a+Children’s+Oncology+Group+study&amp;author=MJ+Borowitz&amp;author=M+Devidas&amp;author=SP+Hunger&amp;volume=111&amp;issue=12&amp;publication_year=2008&amp;pages=5477-5485&amp;pmid=18388178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0024">24. <span class="mixed-citation">
Shi J, Levinson DF, Duan J, et al. 
<span class="ref-title">Common variants on chromosome 6p22.1 are associated with schizophrenia</span>.
<span class="ref-journal">Nature</span>.
2009;
<span class="ref-vol">460</span>(<span class="ref-iss">7256</span>):753–757
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775422/">PMC free article</a>]</span> [<a href="/pubmed/19571809" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+on+chromosome+6p22.1+are+associated+with+schizophrenia&amp;author=J+Shi&amp;author=DF+Levinson&amp;author=J+Duan&amp;volume=460&amp;issue=7256&amp;publication_year=2009&amp;pages=753-757&amp;pmid=19571809&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0025">25. <span class="mixed-citation">
Purcell SM, Wray NR, Stone JL, et al. 
<span class="ref-title">Common polygenic variation contributes to risk of schizophrenia and bipolar disorder</span>.
<span class="ref-journal">Nature</span>.
2009;
<span class="ref-vol">460</span>(<span class="ref-iss">7256</span>):748–752
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;volume=460&amp;issue=7256&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0026">26. <span class="mixed-citation">
Troyer JL, Nelson GW, Lautenberger JA, et al. 
<span class="ref-title">Genome-wide association study implicates PARD3B-based AIDS restriction</span>.
<span class="ref-journal">J Infect Dis</span>.
2011;
<span class="ref-vol">203</span>(<span class="ref-iss">10</span>):1491–1502
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080910/">PMC free article</a>]</span> [<a href="/pubmed/21502085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Genome-wide+association+study+implicates+PARD3B-based+AIDS+restriction&amp;author=JL+Troyer&amp;author=GW+Nelson&amp;author=JA+Lautenberger&amp;volume=203&amp;issue=10&amp;publication_year=2011&amp;pages=1491-1502&amp;pmid=21502085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0027">27. <span class="mixed-citation">
Burchard EG, Avila PC, Nazario S, et al. 
<span class="ref-title">Lower bronchodilator responsiveness in Puerto Rican than in Mexican subjects with asthma</span>.
<span class="ref-journal">Am J Respir Crit Care Med</span>.
2004;
<span class="ref-vol">169</span>(<span class="ref-iss">3</span>):386–392
 [<a href="/pubmed/14617512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=Lower+bronchodilator+responsiveness+in+Puerto+Rican+than+in+Mexican+subjects+with+asthma&amp;author=EG+Burchard&amp;author=PC+Avila&amp;author=S+Nazario&amp;volume=169&amp;issue=3&amp;publication_year=2004&amp;pages=386-392&amp;pmid=14617512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0028">28. <span class="mixed-citation">
Korn JM, Kuruvilla FG, McCarroll SA, et al. 
<span class="ref-title">Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs</span>.
<span class="ref-journal">Nat Genet</span>.
2008;
<span class="ref-vol">40</span>(<span class="ref-iss">10</span>):1253–1260
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=JM+Korn&amp;author=FG+Kuruvilla&amp;author=SA+McCarroll&amp;volume=40&amp;issue=10&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0029">29. <span class="mixed-citation">
Rabbee N, Speed TP.
<span class="ref-title">A genotype calling algorithm for affymetrix SNP arrays</span>.
<span class="ref-journal">Bioinformatics</span>.
2006;
<span class="ref-vol">22</span>(<span class="ref-iss">1</span>):7–12
 [<a href="/pubmed/16267090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A+genotype+calling+algorithm+for+affymetrix+SNP+arrays&amp;author=N+Rabbee&amp;author=TP+Speed&amp;volume=22&amp;issue=1&amp;publication_year=2006&amp;pages=7-12&amp;pmid=16267090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0030">30. <span class="mixed-citation">
Pritchard JK, Stephens M, Donnelly P.
<span class="ref-title">Inference of population structure using multilocus genotype data</span>.
<span class="ref-journal">Genetics</span>.
2000;
<span class="ref-vol">155</span>(<span class="ref-iss">2</span>):945–959
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;issue=2&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0031">31. <span class="mixed-citation">
Yang JJ, Cheng C, Devidas M, et al. 
<span class="ref-title">Ancestry and pharmacogenomics of relapse in acute lymphoblastic leukemia</span>.
<span class="ref-journal">Nat Genet</span>.
2011;
<span class="ref-vol">43</span>(<span class="ref-iss">3</span>):237–241
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3104508/">PMC free article</a>]</span> [<a href="/pubmed/21297632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Ancestry+and+pharmacogenomics+of+relapse+in+acute+lymphoblastic+leukemia&amp;author=JJ+Yang&amp;author=C+Cheng&amp;author=M+Devidas&amp;volume=43&amp;issue=3&amp;publication_year=2011&amp;pages=237-241&amp;pmid=21297632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0032">32. <span class="mixed-citation">
Mao X, Bigham AW, Mei R, et al. 
<span class="ref-title">A genomewide admixture mapping panel for Hispanic/Latino populations</span>.
<span class="ref-journal">Am J Hum Genet</span>.
2007;
<span class="ref-vol">80</span>(<span class="ref-iss">6</span>): 1171–1178
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1867104/">PMC free article</a>]</span> [<a href="/pubmed/17503334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genomewide+admixture+mapping+panel+for+Hispanic/Latino+populations&amp;author=X+Mao&amp;author=AW+Bigham&amp;author=R+Mei&amp;volume=80&amp;issue=6&amp;publication_year=2007&amp;pages=1171-1178&amp;pmid=17503334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0033">33. <span class="mixed-citation">
Patterson N, Price AL, Reich D.
<span class="ref-title">Population structure and eigenanalysis</span>.
<span class="ref-journal">PLoS Genet</span>.
2006;
<span class="ref-vol">2</span>(<span class="ref-iss">12</span>):e190
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;issue=12&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0034">34. <span class="mixed-citation">
Zhang W, Duan S, Kistner EO, et al. 
<span class="ref-title">Evaluation of genetic variation contributing to differences in gene expression between populations</span>.
<span class="ref-journal">Am J Hum Genet</span>.
2008;
<span class="ref-vol">82</span>(<span class="ref-iss">3</span>):631–640
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2427223/">PMC free article</a>]</span> [<a href="/pubmed/18313023" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Evaluation+of+genetic+variation+contributing+to+differences+in+gene+expression+between+populations&amp;author=W+Zhang&amp;author=S+Duan&amp;author=EO+Kistner&amp;volume=82&amp;issue=3&amp;publication_year=2008&amp;pages=631-640&amp;pmid=18313023&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0035">35. <span class="mixed-citation">
French D, Yang W, Cheng C, et al. 
<span class="ref-title">Acquired variation outweighs inherited variation in whole genome analysis of methotrexate polyglutamate accumulation in leukemia</span>.
<span class="ref-journal">Blood</span>.
2009;
<span class="ref-vol">113</span>(<span class="ref-iss">19</span>):4512–4520
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2680361/">PMC free article</a>]</span> [<a href="/pubmed/19066393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Acquired+variation+outweighs+inherited+variation+in+whole+genome+analysis+of+methotrexate+polyglutamate+accumulation+in+leukemia&amp;author=D+French&amp;author=W+Yang&amp;author=C+Cheng&amp;volume=113&amp;issue=19&amp;publication_year=2009&amp;pages=4512-4520&amp;pmid=19066393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0036">36. <span class="mixed-citation">
Pottier N, Yang W, Assem M, et al. 
<span class="ref-title">The SWI/SNF chromatin-remodeling complex and glucocorticoid resistance in acute lymphoblastic leukemia</span>.
<span class="ref-journal">J Natl Cancer Inst</span>.
2008;
<span class="ref-vol">100</span>(<span class="ref-iss">24</span>):1792–1803
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2639326/">PMC free article</a>]</span> [<a href="/pubmed/19066270" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Natl+Cancer+Inst&amp;title=The+SWI/SNF+chromatin-remodeling+complex+and+glucocorticoid+resistance+in+acute+lymphoblastic+leukemia&amp;author=N+Pottier&amp;author=W+Yang&amp;author=M+Assem&amp;volume=100&amp;issue=24&amp;publication_year=2008&amp;pages=1792-1803&amp;pmid=19066270&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0037">37. <span class="mixed-citation">
Unoki H, Takahashi A, Kawaguchi T, et al. 
<span class="ref-title">SNPs in KCNQ1 are associated with susceptibility to type 2 diabetes in East Asian and European populations</span>.
<span class="ref-journal">Nat Genet</span>.
2008;
<span class="ref-vol">40</span>(<span class="ref-iss">9</span>):1098–1102
 [<a href="/pubmed/18711366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=SNPs+in+KCNQ1+are+associated+with+susceptibility+to+type+2+diabetes+in+East+Asian+and+European+populations&amp;author=H+Unoki&amp;author=A+Takahashi&amp;author=T+Kawaguchi&amp;volume=40&amp;issue=9&amp;publication_year=2008&amp;pages=1098-1102&amp;pmid=18711366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0038">38. <span class="mixed-citation">
Pulit SL, Voight BF, de Bakker PI.
<span class="ref-title">Multiethnic genetic association studies improve power for locus discovery</span>.
<span class="ref-journal">PLoS One</span>.
2010;
<span class="ref-vol">5</span>(<span class="ref-iss">9</span>):e12600
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935880/">PMC free article</a>]</span> [<a href="/pubmed/20838612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Multiethnic+genetic+association+studies+improve+power+for+locus+discovery&amp;author=SL+Pulit&amp;author=BF+Voight&amp;author=PI+de+Bakker&amp;volume=5&amp;issue=9&amp;publication_year=2010&amp;pages=e12600&amp;pmid=20838612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0039">39. <span class="mixed-citation">
Waters KM, Stram DO, Hassanein MT, et al. 
<span class="ref-title">Consistent association of type 2 diabetes risk variants found in Europeans in diverse racial and ethnic groups</span>.
<span class="ref-journal">PLoS Genet</span>.
2010;
<span class="ref-vol">6</span>(<span class="ref-iss">8</span>):e1001078
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2928808/">PMC free article</a>]</span> [<a href="/pubmed/20865176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Consistent+association+of+type+2+diabetes+risk+variants+found+in+Europeans+in+diverse+racial+and+ethnic+groups&amp;author=KM+Waters&amp;author=DO+Stram&amp;author=MT+Hassanein&amp;volume=6&amp;issue=8&amp;publication_year=2010&amp;pages=e1001078&amp;pmid=20865176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0040">40. <span class="mixed-citation">
Lahoud MH, Ristevski S, Venter DJ, et al. 
<span class="ref-title">Gene targeting of Desrt, a novel ARID class DNA-binding protein, causes growth retardation and abnormal development of reproductive organs</span>.
<span class="ref-journal">Genome Res</span>.
2001;
<span class="ref-vol">11</span>(<span class="ref-iss">8</span>):1327–1334
 [<a href="/pubmed/11483573" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Gene+targeting+of+Desrt,+a+novel+ARID+class+DNA-binding+protein,+causes+growth+retardation+and+abnormal+development+of+reproductive+organs&amp;author=MH+Lahoud&amp;author=S+Ristevski&amp;author=DJ+Venter&amp;volume=11&amp;issue=8&amp;publication_year=2001&amp;pages=1327-1334&amp;pmid=11483573&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0041">41. <span class="mixed-citation">
Schmitt C, Tonnelle C, Dalloul A, et al. 
<span class="ref-title">Aiolos and Ikaros: regulators of lymphocyte development, homeostasis and lymphoproliferation</span>.
<span class="ref-journal">Apoptosis</span>.
2002;
<span class="ref-vol">7</span>(<span class="ref-iss">3</span>):277–284
 [<a href="/pubmed/11997672" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Apoptosis&amp;title=Aiolos+and+Ikaros:+regulators+of+lymphocyte+development,+homeostasis+and+lymphoproliferation&amp;author=C+Schmitt&amp;author=C+Tonnelle&amp;author=A+Dalloul&amp;volume=7&amp;issue=3&amp;publication_year=2002&amp;pages=277-284&amp;pmid=11997672&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0042">42. <span class="mixed-citation">
Bedi R, Du J, Sharma AK, et al. 
<span class="ref-title">Human C/EBP-epsilon activator and repressor isoforms differentially reprogram myeloid lineage commitment and differentiation</span>.
<span class="ref-journal">Blood</span>.
2009;
<span class="ref-vol">113</span>(<span class="ref-iss">2</span>):317–327
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2615649/">PMC free article</a>]</span> [<a href="/pubmed/18832658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Human+C/EBP-epsilon+activator+and+repressor+isoforms+differentially+reprogram+myeloid+lineage+commitment+and+differentiation&amp;author=R+Bedi&amp;author=J+Du&amp;author=AK+Sharma&amp;volume=113&amp;issue=2&amp;publication_year=2009&amp;pages=317-327&amp;pmid=18832658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0043">43. <span class="mixed-citation">
Eichler EE, Flint J, Gibson G, et al. 
<span class="ref-title">Missing heritability and strategies for finding the underlying causes of complex disease</span>.
<span class="ref-journal">Nat Rev Genet</span>.
2010;
<span class="ref-vol">11</span>(<span class="ref-iss">6</span>): 446–450
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2942068/">PMC free article</a>]</span> [<a href="/pubmed/20479774" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Missing+heritability+and+strategies+for+finding+the+underlying+causes+of+complex+disease&amp;author=EE+Eichler&amp;author=J+Flint&amp;author=G+Gibson&amp;volume=11&amp;issue=6&amp;publication_year=2010&amp;pages=+446-450&amp;pmid=20479774&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0044">44. <span class="mixed-citation">
Manolio TA, Collins FS, Cox NJ, et al. 
<span class="ref-title">Finding the missing heritability of complex diseases</span>.
<span class="ref-journal">Nature</span>.
2009;
<span class="ref-vol">461</span>(<span class="ref-iss">7265</span>):747–753
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;author=FS+Collins&amp;author=NJ+Cox&amp;volume=461&amp;issue=7265&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0045">45. <span class="mixed-citation">
Mullighan CG, Su X, Zhang J, et al. 
<span class="ref-title">Deletion of IKZF1 and prognosis in acute lymphoblastic leukemia</span>.
<span class="ref-journal">N Engl J Med</span>.
2009;
<span class="ref-vol">360</span>(<span class="ref-iss">5</span>):470–480
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2674612/">PMC free article</a>]</span> [<a href="/pubmed/19129520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Deletion+of+IKZF1+and+prognosis+in+acute+lymphoblastic+leukemia&amp;author=CG+Mullighan&amp;author=X+Su&amp;author=J+Zhang&amp;volume=360&amp;issue=5&amp;publication_year=2009&amp;pages=470-480&amp;pmid=19129520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0046">46. <span class="mixed-citation">
Akasaka T, Balasas T, Russell LJ, et al. 
<span class="ref-title">Five members of the CEBP transcription factor family are targeted by recurrent IGH translocations in B-cell precursor acute lymphoblastic leukemia (BCP-ALL)</span>.
<span class="ref-journal">Blood</span>.
2007;
<span class="ref-vol">109</span>(<span class="ref-iss">8</span>):3451–3461
 [<a href="/pubmed/17170124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Five+members+of+the+CEBP+transcription+factor+family+are+targeted+by+recurrent+IGH+translocations+in+B-cell+precursor+acute+lymphoblastic+leukemia+(BCP-ALL)&amp;author=T+Akasaka&amp;author=T+Balasas&amp;author=LJ+Russell&amp;volume=109&amp;issue=8&amp;publication_year=2007&amp;pages=3451-3461&amp;pmid=17170124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0047">47. <span class="mixed-citation">
Paulsson K, Forestier E, Lilljebjorn H, et al. 
<span class="ref-title">Genetic landscape of high hyperdiploid childhood acute lymphoblastic leukemia</span>.
<span class="ref-journal">Proc Natl Acad Sci U S A</span>.
2010;
<span class="ref-vol">107</span>(<span class="ref-iss">50</span>):21719–21724
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3003126/">PMC free article</a>]</span> [<a href="/pubmed/21098271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Genetic+landscape+of+high+hyperdiploid+childhood+acute+lymphoblastic+leukemia&amp;author=K+Paulsson&amp;author=E+Forestier&amp;author=H+Lilljebjorn&amp;volume=107&amp;issue=50&amp;publication_year=2010&amp;pages=21719-21724&amp;pmid=21098271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0048">48. <span class="mixed-citation">
Saito K, Tolias KF, Saci A, et al. 
<span class="ref-title">BTK regulates PtdIns-4,5-P2 synthesis: importance for calcium signaling and PI3K activity</span>.
<span class="ref-journal">Immunity</span>.
2003;
<span class="ref-vol">19</span>(<span class="ref-iss">5</span>):669–678
 [<a href="/pubmed/14614854" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=BTK+regulates+PtdIns-4,5-P2+synthesis:+importance+for+calcium+signaling+and+PI3K+activity&amp;author=K+Saito&amp;author=KF+Tolias&amp;author=A+Saci&amp;volume=19&amp;issue=5&amp;publication_year=2003&amp;pages=669-678&amp;pmid=14614854&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0049">49. <span class="mixed-citation">
O’Rourke LM, Tooze R, Turner M, et al. 
<span class="ref-title">CD19 as a membrane-anchored adaptor protein of B lymphocytes: costimulation of lipid and protein kinases by recruitment of Vav</span>.
<span class="ref-journal">Immunity</span>.
1998;
<span class="ref-vol">8</span>(<span class="ref-iss">5</span>):635–645
 [<a href="/pubmed/9620684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=CD19+as+a+membrane-anchored+adaptor+protein+of+B+lymphocytes:+costimulation+of+lipid+and+protein+kinases+by+recruitment+of+Vav&amp;author=LM+O’Rourke&amp;author=R+Tooze&amp;author=M+Turner&amp;volume=8&amp;issue=5&amp;publication_year=1998&amp;pages=635-645&amp;pmid=9620684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="CIT0050">50. <span class="mixed-citation">
Kurreeman FA, Stahl EA, Okada Y, et al. 
<span class="ref-title">Use of a multiethnic approach to identify rheumatoid- arthritis-susceptibility loci, 1p36 and 17q12</span>.
<span class="ref-journal">Am J Hum Genet</span>.
2012;
<span class="ref-vol">90</span>(<span class="ref-iss">3</span>):524–532
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3309197/">PMC free article</a>]</span> [<a href="/pubmed/22365150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Use+of+a+multiethnic+approach+to+identify+rheumatoid-+arthritis-susceptibility+loci,+1p36+and+17q12&amp;author=FA+Kurreeman&amp;author=EA+Stahl&amp;author=Y+Okada&amp;volume=90&amp;issue=3&amp;publication_year=2012&amp;pages=524-532&amp;pmid=22365150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3691938&amp;issue-id=222913&amp;journal-id=753&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140296034648176title">Abstract</a></li><li><a href="#s1title">Methods</a></li><li><a href="#s5title">Results</a></li><li><a href="#s6title">Discussion</a></li><li><a href="#__sec5title">Funding</a></li><li><a href="#idm140295952830112title">Notes</a></li><li><a href="#idm140295960385216title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">JNCI Journal of the National Cancer Institute</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3691938/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3691938/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3691938/pdf/djt042.pdf">PDF (8.3M)</a></li> | <li><a href="#" data-citationid="PMC3691938" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3691938%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3691938%2F&amp;text=Novel%20Susceptibility%20Variants%20at%2010p12.31-12.2%20for%20Childhood%20Acute%20Lymphoblastic%20Leukemia%20in%20Ethnically%20Diverse%20Populations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3691938%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23512250" data-aiid="3691938" data-aid="3691938" data-iid="222913" data-domainid="753" data-domain="jnci" data-accid="PMC3691938" data-md5="a09eb986c498211da5ee84f54857959e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:30:20-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8828B0DD551BD100000000078A0508&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3691938%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jnci&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3691938/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>