<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3608855/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies genetic variants in GOT1 determining serum aspartate aminotransferase levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3608855/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of human genetics"><meta name="citation_title" content="Genome-wide association study identifies genetic variants in GOT1 determining serum aspartate aminotransferase levels"><meta name="citation_authors" content="Haiqing Shen, Coleen Damcott, Scott R Shuldiner, Sumbul Chai, Rongze Yang, Hong Hu, Quince Gibson, Kathleen A Ryan, Braxton D Mitchell, Da-Wei Gong"><meta name="citation_date" content="November 2011"><meta name="citation_issue" content="11"><meta name="citation_volume" content="56"><meta name="citation_firstpage" content="801"><meta name="citation_doi" content="10.1038/jhg.2011.105"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3608855/?report=abstract"><meta name="citation_pmid" content="21900944"><meta name="DC.Title" content="Genome-wide association study identifies genetic variants in GOT1 determining serum aspartate aminotransferase levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Haiqing Shen"><meta name="DC.Contributor" content="Coleen Damcott"><meta name="DC.Contributor" content="Scott R Shuldiner"><meta name="DC.Contributor" content="Sumbul Chai"><meta name="DC.Contributor" content="Rongze Yang"><meta name="DC.Contributor" content="Hong Hu"><meta name="DC.Contributor" content="Quince Gibson"><meta name="DC.Contributor" content="Kathleen A Ryan"><meta name="DC.Contributor" content="Braxton D Mitchell"><meta name="DC.Contributor" content="Da-Wei Gong"><meta name="DC.Date" content="2011 Nov"><meta name="DC.Identifier" content="10.1038/jhg.2011.105"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies genetic variants in GOT1 determining serum aspartate aminotransferase levels"><meta property="og:type" content="article"><meta property="og:description" content="We carried out a genome-wide association study of serum aspartate aminotransferase (AST) activity in 866 Amish participants of the Heredity and Phenotype Intervention Heart Study and identified significant association of AST activity with a cluster of ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3608855/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3608855/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3608855/pdf/nihms447117.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D8975DD52B7B1000000000582039B.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3608855%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3608855</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Hum Genet</a></span>. Author manuscript; available in PMC 2013 Mar 27.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21900944" target="pmc_ext" ref="reftype=publisher&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">J Hum Genet. 2011 Nov; 56(11): 801–805. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Sep 8. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fjhg.2011.105" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/jhg.2011.105</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3608855</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS447117</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21900944">21900944</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies genetic variants in <em>GOT1</em> determining serum aspartate aminotransferase levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Shen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971326861664" co-class="co-affbox">Haiqing Shen</a>,<sup>1</sup> <a href="/pubmed/?term=Damcott%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971258209712" co-class="co-affbox">Coleen Damcott</a>,<sup>1</sup> <a href="/pubmed/?term=Shuldiner%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971326859168" co-class="co-affbox">Scott R Shuldiner</a>,<sup>1</sup> <a href="/pubmed/?term=Chai%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971293924736" co-class="co-affbox">Sumbul Chai</a>,<sup>1</sup> <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971293922752" co-class="co-affbox">Rongze Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Hu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971292801808" co-class="co-affbox">Hong Hu</a>,<sup>1</sup> <a href="/pubmed/?term=Gibson%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971292799824" co-class="co-affbox">Quince Gibson</a>,<sup>1</sup> <a href="/pubmed/?term=Ryan%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971292797840" co-class="co-affbox">Kathleen A Ryan</a>,<sup>1</sup> <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971255210064" co-class="co-affbox">Braxton D Mitchell</a>,<sup>1</sup> and  <a href="/pubmed/?term=Gong%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944" class="affpopup" co-rid="_co_idm139971255208080" co-class="co-affbox">Da-Wei Gong</a><sup>1,</sup><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139971326861664"><h3 class="no_margin">Haiqing Shen</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Shen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Haiqing Shen</a></div></div><div id="_co_idm139971258209712"><h3 class="no_margin">Coleen Damcott</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Damcott%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Coleen Damcott</a></div></div><div id="_co_idm139971326859168"><h3 class="no_margin">Scott R Shuldiner</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Shuldiner%20SR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Scott R Shuldiner</a></div></div><div id="_co_idm139971293924736"><h3 class="no_margin">Sumbul Chai</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Chai%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Sumbul Chai</a></div></div><div id="_co_idm139971293922752"><h3 class="no_margin">Rongze Yang</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Rongze Yang</a></div></div><div id="_co_idm139971292801808"><h3 class="no_margin">Hong Hu</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Hu%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Hong Hu</a></div></div><div id="_co_idm139971292799824"><h3 class="no_margin">Quince Gibson</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Gibson%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Quince Gibson</a></div></div><div id="_co_idm139971292797840"><h3 class="no_margin">Kathleen A Ryan</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Ryan%20KA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Kathleen A Ryan</a></div></div><div id="_co_idm139971255210064"><h3 class="no_margin">Braxton D Mitchell</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Braxton D Mitchell</a></div></div><div id="_co_idm139971255208080"><h3 class="no_margin">Da-Wei Gong</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</p><p><sup>2</sup>Geriatric Research and Education Clinical Center, Veterans Administration Medical Center, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Gong%20DW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21900944">Da-Wei Gong</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139971328623216_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139971328623216_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139971328623216_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Geriatric Research and Education Clinical Center, Veterans Administration Medical Center, Baltimore, MD, USA</div><div id="FN1">Correspondence: Dr H Shen, Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, 660 West Redwood Street, Baltimore, MD 21201, USA. <a href="mailto:dev@null" data-email="ude.dnalyramu.enicidem@nehsh" class="oemail">ude.dnalyramu.enicidem@nehsh</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139971328623216_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21900944" target="pmc_ext" ref="reftype=publisher&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">J Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3608855/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="idm139971331420400" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139971331420400title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">We carried out a genome-wide association study of serum aspartate aminotransferase (AST) activity in 866 Amish participants of the Heredity and Phenotype Intervention Heart Study and identified significant association of AST activity with a cluster of single nucleotide polymorphisms located on chromosome 10q24.1 (peak association was rs17109512; <em>P</em>=2.80E-14), in the vicinity of <em>GOT1</em>, the gene encoding cytosolic AST (cAST). Sequencing of <em>GOT1</em> revealed an in-frame deletion of three nucleic acids encoding asparagine at position 389 c.1165_1167delAAC (p.Asn389del) in the gene. Deletion carriers had significantly lower AST activity levels compared with homozygotes for the common allele (mean±s.d.: 10.0±2.8 versus 18.8±5.2 U l<sup>−1</sup>; <em>P</em>=2.80E-14). Further genotyping of the deletion in other Amish samples (<em>n</em>=1932) identified an additional 20 carriers (minor allele frequency (MAF)=0.0052). The deletion was not detected in 647 outbred Caucasians. Asn at codon 389 is conserved among known mammalian cASTs. <em>In vitro</em> transient transfection of wild-type and mutant cAST indicated that mutant cAST protein was barely detectable in the cells. Furthermore, even after correction for cAST expression, mutant cAST had markedly diminished enzymatic activity. Remarkably, we did not find any association between the deletion and metabolic traits including serum fasting glucose or insulin, fasting and post-meal lipids, inflammatory markers, or sub-clinical markers of cardiovascular disease. In conclusion, we discovered a rare in-frame deletion in <em>GOT1</em> gene, which inactivates cAST enzyme in the Old Order Amish. This finding will help us to understand structure and function of the enzyme and would be useful for predicting serum AST levels.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Amish, AST, gene, GOT1, mutation</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">INTRODUCTION</h2><p id="P2" class="p p-first">Aspartate aminotransferase (AST; EC 2.6.1.1) catalyzes the reversible conversion between aspartate/α-ketoglutarate and oxaloacetate/glutamate, and has an important role in amino acid metabolism and in the urea and tricarboxylic acid cycles.<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> AST is conserved from all prokaryotic to eukaryotic organisms and is expressed in many types of cells/tissues including liver, red blood cells and cardiac/skeletal muscles. Two highly similar isoenzymes are present in mammals, one located in cytoplasm (cAST), and the other in mitochondria (mAST). In serum, the enzyme activity is largely of cytosolic origin.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_335234251">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933774">3</a></sup> Serum AST activity elevation is considered a surrogate marker of liver, heart and muscle injury.</p><p id="P3" class="p p-last">The locus for soluble AST (<em>GOT1</em>) is on chromosome 10q24, whereas the mitochondrial form of AST (<em>GOT2</em>) is on 16q12. Twin studies<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933763">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933768">5</a></sup> suggest that serum AST activity is influenced by both genetic and environmental factors. However, to date, specific gene variants that influence serum AST activity and/or metabolic or other phenotypes that may be associated with AST activity have yet to be identified. We sought to determine the genetic determinants of serum AST levels by carrying out a genome-wide association study (GWAS) of serum AST in 866 Amish subjects from the Heredity and Phenotype Intervention (HAPI) Heart Study.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">MATERIALS AND METHODS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Population</h3><p id="P4" class="p p-first">The HAPI Heart Study was initiated in 2003 to identify genes that interact with short-term environmental exposures to modify risk factors for cardiovascular disease. Details of recruitment and phenotyping have been reported previously.<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933767">6</a></sup> Briefly, subjects were members of the Old Order Amish (OOA) community aged 20 years or older and considered to be relatively healthy, based on exclusion criteria of severe hypertension (blood pressure greater than 180/105 mm Hg), malignancy, and kidney, liver or untreated thyroid disease.</p><p id="P5">HAPI Heart Study participants underwent measurement of cardiovascular disease risk factors and questioning about their prior history of cardiovascular disease. Physical examinations, including anthropometry were conducted at the Amish Research Clinic in Strasburg, PA, USA. Blood samples were obtained following an overnight fast. Those taking lipid-lowering medications at enrollment discontinued usage 7 days before examination. Serum lipid levels and high-density lipoprotein cholesterol were assayed by Quest Diagnostics (Horsham, PA, USA). All subjects had triglyceride levels less than 400 mg dl<sup>−1</sup>, and low-density lipoprotein cholesterol levels were calculated according to the Friedewald formula.<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933771">7</a></sup> Hypertension was defined as a systolic blood pressure of 140 mm Hg or greater, a diastolic blood pressure of 90 mm Hg or greater, or use of prescription blood pressure-lowering medications. Diabetes was defined as a fasting glucose of 126 mg dl<sup>−1</sup> or greater, or current use of prescription diabetes medications. Smoking habits were recorded; current smoking status included use of cigarettes, pipes or cigars. Serum AST and alanine aminotransferase were assayed using spectrophotometry by Quest Diagnostics.</p><p id="P6">Additional Amish subjects who had DNA samples stored in our Biobank (<em>n</em>=1932) were screened for the mutation identified. Six hundred and forty seven non-Amish Caucasians were screened, who were participants in the type 2 diabetes case–control study conducted by Joslin Diabetes Center (Boston, MA, USA) and University of Maryland (Baltimore, MD, USA).</p><p id="P7" class="p p-last">The study protocol was approved by the Institutional Review Board at the University of Maryland, and other participating institutions. Informed consent was obtained from each of the study participants.</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping</h3><p id="P8" class="p p-first">Genotyping was performed in HAPI Heart Study participants with the Affymetrix GeneChip Human Mapping 500 K Array set (Affymetrix, Santa Clara, CA, USA), including a total of 500 568 single nucleotide polymorphisms (SNPs). The Affymetrix GeneChip Genotyping Analysis Software and BRLMM genotype-calling algorithm (Affymetrix) were used to generate SNP data files. Mean sample call rate in 866 samples was 98.3% after filtered by relationship check and call rate (&gt;95%). A total of 373 825 SNPs that passed quality control (SNP call rate &gt;95%, minor allele frequency (MAF) &gt;0) and Hardy–Weinberg Equilibrium checks (at <em>P</em>&gt;0.0001) were retained for analysis.</p><p id="P9" class="p p-last">We confirmed the deletion c.1165_1167delAAC (p.Asn389del) by dideoxy sequence analysis (ABI 3730; Applied Biosystems, Foster City, CA, USA) in the exons of <em>GOT1</em> gene, followed by genotyping using Custom TaqMan SNP Genotyping Assays (Applied Biosystems) in HAPI Heart Study participants, 1932 additional Amish DNA samples from our Biobank, and 647 non-Amish Caucasians.</p></div><div id="S5" class="sec"><h3 id="S5title">Functional studies of recombinant cAST</h3><div id="S6" class="sec sec-first"><p></p><h4 id="S6title" class="inline">Construction of expression vector of wild-type and mutant cAST </h4><p id="P10" class="p p-first-last">Full-length human cAST cDNA plasmid (IMAGE clone ID: 44420) was purchased from Open Biosystems (Huntsville, AL, USA). The plasmid served as template for PCR amplification of the full cAST protein-coding region with restriction enzyme HindIII-anchored upstream primer p2254 5′-cgatgacgacaagcttcatatgg-cacctccgtcagtctttgcc-3′ and downstream XbaI-anchored downstream primer p2255 5′-cccgggatcctctagtcactggattttggtgactgc-3′ with high-fidelity Phusion polymerase (New England Biolabs, Ipswich, MA, USA). The resultant PCR product was subcloned into pFlag-CMV2 precut with HindIII and XbaI by In-Fusion cloning (Clontech, Mountain View, CA, USA) to make wild-type expression vector pFlag-CMV2-cASTwt. For construction of cASTmut (c.1165_1167delAAC), three fragment In-Fusion cloning was carried out with mutant 5′-end fragment (PCR amplified with upstream primer p2254 and downstream primer p2323 5′-ccactcacgattcgaccacttggcagcag-3′ on wild-type cAST cDNA template), mutant 3′-end fragment (PCR amplified with upstream primer p2313 5′-ggtcgaatcgtgagtggcttaaccaccaaa-3′ and downstream primer p2255 on wtAST template), and fragment of HindIII and Xba-precut pFlag-CMV2, to make the mutant cAST expression vector pFlag-CMV2-cASTmut. Both cASTwt and cASTmut DNA inserts were verified by sequence analysis.</p></div><div id="S7" class="sec sec-last"><p></p><h4 id="S7title" class="inline">Expression and enzymatic assay of recombinant cASTwt and cASTmut proteins </h4><p id="P11" class="p p-first">HEK293 cells were grown in Dulbecco’s modified Eagle medium containing 10% fetal bovine serum and antibiotics ampicillin and streptomycin. The cells were plated into six-well plates. At ~95% confluency, the cells were transfected with empty vector pFlag-CMV2, pFlag-CMV2-cASTwt and/or pFlag-CMV2-cASTmut with LipoD293 (SignaGene, Gaithersburg, MD, USA) according to the manufacturer’s instruction. At 48 h post transfection, the cells were collected in 10 mM Tris-HCl buffer (pH 7.7) and lysed by sonication on ice. The lysates were centrifuged and supernatants were collected for protein quantification using the Bradford method, AST activity assay and western blot analysis.</p><p id="P12">AST activity was determined by using the AST kit (Catachem, Oxford, CT, USA) according to the manufacturer’s instruction. Briefly, 5 μl of cell lysate was incubated with a 0.2 ml mixture of reagent A and B containing L-aspartate, NADH, malate dehydrogenase and 2-oxoglutarate at 25 °C. Absorbance at 340 nm was recorded for 5 min at every 30 s interval after addition of the protein fraction. The slope of absorbance decrease is proportional to AST activity. Final AST activities were corrected by protein concentration of cell lysates. One unit of AST activity was defined as the amount of enzyme, which catalyzes the formation of 1 μmol l<sup>−1</sup> of NAD per minute under conditions of the assay at 25 °C.</p><p id="P13" class="p p-last">For western analysis, 50 μg of total proteins from reduced cell lysate were separated by electrophoresis on 7.5% polyacrylamide gels. Following electrophoresis, proteins were transferred onto polyvinylidene fluoride membranes and bound proteins were probed with primary anti-Flag antibody (Sigma, St Louis, MO, USA) or GAPDH (Millipore, Billerica, MA, USA), and developed with an alkaline phosphatase-conjugated second antibody.</p></div></div><div id="S8" class="sec sec-last"><h3 id="S8title">Statistical analysis</h3><p id="P14" class="p p-first-last">Association analyses of quantitative traits were performed under a variance component model that assesses the effect of genotype, as an additive effect, on the quantitative trait, whereas simultaneously estimating the effects of age, age,<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_335234250">2</a></sup> sex, and a polygenic component to account for phenotypic correlation due to relatedness. The polygenic component was modeled using the relationship matrix derived from the complete pedigree structure as all the subjects are related. Association analysis using the complete pedigree structure was carried out using mixed model analysis for pedigree (MMAP) software developed by one of our colleague (JR O’Connell). AST, alanine aminotransferase, fasting glucose and fasting insulin were naturally log-transformed. Student’s <em>t</em>-test was used to examine differences between cASTwt and cASTmut activities.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">RESULTS</h2><p id="P15" class="p p-first">We carried out a GWAS of serum AST activity levels in 866 Amish subjects from the HAPI Heart Study. Characteristics of study participants are summarized in <a href="/pmc/articles/PMC3608855/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>. In the GWAS, the strongest SNP associations with serum AST levels were observed for a group of SNPs located on chromosome 10q24.1. The peak association was with rs17109512 (<em>P</em>-value=2.80E-14), in the vicinity of <em>GOT1</em> (~1 Mb downstream), the gene encoding cAST (<a href="/pmc/articles/PMC3608855/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). The frequency of the associated SNP is rare (MAF=0.006), with only 10 heterozygotes for rs17109512 found in the HAPI Heart Study. The 10 heterozygotes had significantly lower AST levels compared with the 856 homozygotes of the wide-type allele (mean±s.d. of AST: 10.0±2.8 versus 18.8±5.2 U l<sup>−1</sup>, <em>P</em>=2.80E-14).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3608855/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139971258310528" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms447117f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms447117f1.jpg" src="/pmc/articles/PMC3608855/bin/nihms447117f1.jpg"></div></a><div id="largeobj_idm139971258310528" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3608855/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC3608855/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3608855/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p4">Genome-wide association analysis plot for serum AST levels in 866 Old Order Amish subjects. Arrow indicates that SNP rs17109512 on chromosome 10q24.1 is highly associated with serum AST activity (<em>P</em>=2.80E-14). A full color version of this figure is available at the <em>Journal of Human Genetics</em> journal online.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p7">Clinical characteristics of the HAPI Heart Study participants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139971255106208" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Men</th><th rowspan="1" colspan="1" valign="middle" align="center">Women</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>n</em></td><td rowspan="1" colspan="1" valign="top" align="center">460</td><td rowspan="1" colspan="1" valign="top" align="center">406</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age, mean (s.d.), year</td><td rowspan="1" colspan="1" valign="top" align="center">42.2 (13.6)</td><td rowspan="1" colspan="1" valign="top" align="center">45.5 (14.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI, mean (s.d.), kg m<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">25.6 (5.2)</td><td rowspan="1" colspan="1" valign="top" align="center">27.7 (5.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Systolic blood pressure, mean (s.d.), mm Hg</td><td rowspan="1" colspan="1" valign="top" align="center">121.2 (12.5)</td><td rowspan="1" colspan="1" valign="top" align="center">121.0 (16.8)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diastolic blood pressure, mean (s.d.), mm Hg</td><td rowspan="1" colspan="1" valign="top" align="center">77.6 (9.0)</td><td rowspan="1" colspan="1" valign="top" align="center">75.6 (8.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total cholesterol, mean (s.d.), mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.25 (1.14)</td><td rowspan="1" colspan="1" valign="top" align="center">5.59 (1.27)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL-cholesterol, mean (s.d.), mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.54 (1.06)</td><td rowspan="1" colspan="1" valign="top" align="center">3.65 (1.19)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL-cholesterol, mean (s.d.), mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.37 (0.34)</td><td rowspan="1" colspan="1" valign="top" align="center">1.53 (0.39)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglyceride, median (s.d.), mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.72 (0.42)</td><td rowspan="1" colspan="1" valign="top" align="center">0.84 (0.51)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">AST, mean (s.d.), U l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">19.7 (5.1)</td><td rowspan="1" colspan="1" valign="top" align="center">17.6 (5.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ALT, mean (s.d.), U l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">20.9 (7.5)</td><td rowspan="1" colspan="1" valign="top" align="center">16.9 (6.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Hypertension, <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="center">52 (11.3)</td><td rowspan="1" colspan="1" valign="top" align="center">66 (16.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diabetes, <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="center">3 (0.7)</td><td rowspan="1" colspan="1" valign="top" align="center">2 (0.5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Current smoker, <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="center">91 (19.8)</td><td rowspan="1" colspan="1" valign="top" align="center">0 (0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Lipid medication, <em>n</em> (%)</td><td rowspan="1" colspan="1" valign="top" align="center">5 (1.1)</td><td rowspan="1" colspan="1" valign="top" align="center">4 (1.0)</td></tr></tbody></table></div><div id="largeobj_idm139971255106208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3608855/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p8" class="p p-first-last">Abbreviations: BMI, body mass index; HAPI, Heredity and Phenotype Intervention; HDL, high-density lipoprotein; LDL, low-density lipoprotein.</p></div></div></div><p id="P16">Given the low frequency of the associated SNP, its proximity to <em>GOT1</em>, and the very large effect size of the association, we hypothesized that rs17109512 might tag a functional SNP, most likely a mutation changing the protein sequence of cAST, in <em>GOT1</em>. We subsequently sequenced the gene and identified an in-frame deletion of three nucleic acids encoding asparagine at position 389 in <em>GOT1</em> gene (<a href="/pmc/articles/PMC3608855/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>). All of 10 heterozygotes for rs17109512 were heterozygotes for c.1165_1167delAAC. The deletion c.1165_1167delAAC (p.Asn389del) was in complete linkage with rs17109512. By constructing the pedigree, all the deletion carriers can be traced back to one most likely founder who was born in mid-18th century, suggesting a founder effect. Genotyping of 647 non-Amish Caucasians from Baltimore area did not identify carriers of this deletion.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3608855/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139971252422720" class="figure"><a href="/pmc/articles/PMC3608855/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3608855_nihms447117f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms447117f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3608855/bin/nihms447117f2.jpg"></a></div><div id="largeobj_idm139971252422720" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3608855/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3608855/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p5">Detection of <em>GOT1</em> (c.1165_1167delAAC) mutation. (<strong>a</strong>) Sequencing identified an in-frame deletion of three nucleic acids encoding asparagine at position 389 in <em>GOT1</em> gene. (<strong>b</strong>) Asparagine (<strong>N</strong>, indicated by arrow) is conserved in cAST protein sequences among human, chimpanzee, mouse, xenopus, zebrafish and <em>C. elegans</em>. Peptide sequences are aligned using ClustalW2<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933770">12</a></sup> with fully conserved amino acid marked with asterisk. Partial alignment of cAST is shown and amino acid positions are numbered to the left.</p></div></div></div><p id="P17">Glutamine at amino acid at 389 is conserved among known cAST of vertebrate species, including the human, chimpanzee, mouse and zebrafish (<a href="/pmc/articles/PMC3608855/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2b</span></a>). Further genotyping of deletion c.1165_1167delAAC (p.Asn389del) in an additional 1,932 Amish samples identified 20 more carriers (MAF=0.0052), among which 14 had phenotype measurements. No homozygotes for the deletion were found. Phenotype comparison between the 24 (10 from the HAPI Heart Study plus 14 from the additional screening) c.1165_1167delAAC (p.Asn389del) carriers and 856 non-carriers (from the HAPI Heart Study) is shown in <a href="/pmc/articles/PMC3608855/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. The deletion carriers had serum AST activity levels that were approximately one-half that of normal homozygotes, suggesting that the deletion resulted in a complete loss of enzymatic function. Despite this large difference in AST activity, we did not find any association between the mutation and metabolic traits, including serum fasting glucose or insulin, fasting lipids (<a href="/pmc/articles/PMC3608855/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Neither did we find any association with post-meal lipids, inflammatory markers or sub-clinical markers of cardiovascular disease (data not shown).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p9">Clinical characteristics of the <em>GOT1</em> c.1165_1167delAAC carriers and non-carriers</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139971334810224" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center">Carrier</th><th rowspan="1" colspan="1" valign="middle" align="center">Non-carrier</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>n</em></td><td rowspan="1" colspan="1" valign="top" align="center">24</td><td rowspan="1" colspan="1" valign="top" align="center">856</td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age, year</td><td rowspan="1" colspan="1" valign="top" align="center">50.6 (16.8)</td><td rowspan="1" colspan="1" valign="top" align="center">43.7 (14.0)</td><td rowspan="1" colspan="1" valign="top" align="left">0.03</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI, kg m<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">27.1 (4.8)</td><td rowspan="1" colspan="1" valign="top" align="center">26.6 (4.5)</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Waist circumference, cm</td><td rowspan="1" colspan="1" valign="top" align="center">91.9 (11.2)</td><td rowspan="1" colspan="1" valign="top" align="center">87.3 (11.0)</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">AST, U l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">10.0 (2.8)</td><td rowspan="1" colspan="1" valign="top" align="center">18.8 (5.2)</td><td rowspan="1" colspan="1" valign="top" align="left">2.80E-14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ALT, U l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">17.7 (3.9)</td><td rowspan="1" colspan="1" valign="top" align="center">19.1 (7.5)</td><td rowspan="1" colspan="1" valign="top" align="left">0.77</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Systolic blood pressure, mm Hg</td><td rowspan="1" colspan="1" valign="top" align="center">124.6 (17.6)</td><td rowspan="1" colspan="1" valign="top" align="center">121.0 (14.6)</td><td rowspan="1" colspan="1" valign="top" align="left">0.65</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diastolic blood pressure, mm Hg</td><td rowspan="1" colspan="1" valign="top" align="center">78.2 (7.6)</td><td rowspan="1" colspan="1" valign="top" align="center">76.6 (8.8)</td><td rowspan="1" colspan="1" valign="top" align="left">0.25</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total cholesterol, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.51 (1.01)</td><td rowspan="1" colspan="1" valign="top" align="center">5.40 (1.22)</td><td rowspan="1" colspan="1" valign="top" align="left">0.87</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LDL-cholesterol, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.75(0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">3.59 (1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">0.82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HDL-cholesterol, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.29 (0.34)</td><td rowspan="1" colspan="1" valign="top" align="center">1.45 (0.39)</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Triglyceride, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.05 (0.59)</td><td rowspan="1" colspan="1" valign="top" align="center">0.77 (0.47)</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fasting glucose, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">5.43 (1.59)</td><td rowspan="1" colspan="1" valign="top" align="center">4.80 (0.72)</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Fasting insulin, mmol l<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.67 (0.17)</td><td rowspan="1" colspan="1" valign="top" align="center">0.63 (0.31)</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td></tr></tbody></table></div><div id="largeobj_idm139971334810224" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3608855/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p10" class="p p-first-last">Abbreviations: ALT, alanine aminotransferase; AST, aspartate aminotransferase; BMI, body mass index; HAPI, Heredity and Phenotype Intervention; HDL, high-density lipoprotein; LDL, low-density lipoprotein.</p></div><div id="TFN3"><p id="__p11" class="p p-first-last">Mean (s.d.), Adjusted for age, age-square, sex, and family structure for phenotypes other than age.</p></div><div id="TFN4"><p id="__p12" class="p p-first-last">AST and ALT measurements were based on 10 <em>GOT1</em> deletion carriers from HAPI Heart study. The rest of the phenotype measurements were based on 10 <em>GOT1</em> deletion carriers from HAPI Heart Study and 14 additional identified <em>GOT1</em> deletion carriers in Amish; 856 non-carriers were from HAPI Heart Study.</p></div><div id="TFN5"><p id="__p13" class="p p-first-last"><em>P</em>-values for fasting glucose, fasting insulin and waist circumference were based on non-diabetics.</p></div></div></div><p id="P18" class="p p-last">To further examine the functional consequences of p.Asn389del, we conducted <em>in vitro</em> functional studies by expressing the recombinant wild-type and mutant cAST (p.Asn389del) in the mammalian kidney cell line HEK293 and measuring AST activity. As shown in <a href="/pmc/articles/PMC3608855/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3a</span></a>, the basal activity of the cells transfected with empty vector (lane 1) was barely detectable. AST activity levels were increased by &gt;100 fold in a dose-dependent manner when the cells were expressing wild-type cAST (lanes 2 and 3). By contrast, lysates from cells transfected with mutant cAST (lanes 5 and 6) had almost no AST activity, suggesting that the mutant cAST is enzymatically inactive. Western analysis was conducted to determine the levels of recombinant protein expression in the cells. As shown in <a href="/pmc/articles/PMC3608855/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3b</span></a>, mutant cAST only was barely detectable, compared with the wild type (lanes 5 and 6 versus 2 and 3). Interestingly, the activities from the cells transfected with wild-type and mutant cAST at a 1:1 ratio were less than half of that of wild-type AST (lane 4 versus 3; <em>P</em>&lt;0.001), suggesting a dominant negative effect. Western analysis revealed that less mutant cAST protein was expressed after correction for GAPDH, suggesting a decreased expression, probably due to decreased RNA stability and/or increased protein degradation (<a href="/pmc/articles/PMC3608855/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3b</span></a>). However, even after correcting for AST protein, mutant cAST had markedly decreased activity relative to wild-type cAST (lane 2 versus 5, <em>P</em>&lt;0.001).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3608855/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139971255021888" class="figure"><a href="/pmc/articles/PMC3608855/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3608855_nihms447117f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms447117f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3608855/bin/nihms447117f3.jpg"></a></div><div id="largeobj_idm139971255021888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3608855/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3608855/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p6">Functional study of recombinant mutant and wild-type cAST proteins. (<strong>a</strong>) AST activities were measured in lysates of HEK293 cells transfected with empty vector (Vector), wild-type (cASTwt), or p.Asn389del mutant (cASTmut) at the indicated amount in a six-well plate dish. Data are expressed as mean±s.e. (<em>n</em>=4). (<strong>b</strong>) Representative western blot of flag-tagged wild-type and mutant cAST protein levels in HEK293 cells (same lane order as (<strong>a</strong>)) detected with anti-flag antibody. Anti-GAPDH was used as protein loading control.</p></div></div></div></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">DISCUSSION</h2><p id="P19" class="p p-first">Serum AST activity levels are determined by both genetic and environmental factors.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933766">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933769">5</a></sup> Variation in serum AST activity has a moderate heritable basis (0.32–0.40) estimated from twin studies.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933762">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933775">5</a></sup> A GWAS of AST protein levels, as one of the group of 42 proteins measured in 1200 in CHIANTI study participants, didn’t find any <em>cis</em>-effect associations with SNPs near or in <em>GOT1</em> or <em>trans</em>-effect associations with SNPs elsewhere in the genome.<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933772">8</a></sup> In another GWAS, no SNP was associated with serum levels of AST at or exceeding genome-wide significance.<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933765">9</a></sup> Through a GWAS of serum AST activity in the OOA, we identified a rare in-frame 3 bp deletion in <em>GOT1</em> that was strongly associated with low serum AST activity in the OOA. Subjects heterozygous for the <em>GOT1</em> c.1165_1167delAAC had serum AST activity levels that were about one-half that of wild-type homozygotes. Consistent with this observation, functional studies found that AST activity of the mutant AST recombinant protein was totally abolished. This deletion was not present in 647 outbred Caucasians, consistent with negative GWAS findings of others. Also consistent with negative GWAS findings of others, our GWAS did not detect any other regions with genome-wide significant association to AST activity levels.</p><p id="P20">Elevated serum AST activity is a clinical marker for damage to hepatocytes, myocardial cells or skeletal muscle. By contrast, decreased serum AST and its clinical significance are less known. This loss-of-function mutation in AST provides a unique opportunity to determine if decreased AST is associated with a clinically significant phenotype. Although AST has a pivotal role in intermediary metabolism, deficiency might be expected to lead to abnormalities in glucose and/or lipid metabolism. Interestingly, we did not find any significant differences in a wide variety of metabolic traits in <em>GOT1</em> c.1165_1167delAAC carriers compared with wild-type homozygotes. These findings suggest that in humans, AST is not a rate-limiting step in intermediary metabolism, and that a single functional allele is sufficient for normal metabolic function. Nevertheless, a total loss of this enzyme could be lethal, as we did not find any of the mutant homozygotes. Of possible relevance, fasting glucose levels tended to be higher in deletion carriers than that in non-carriers (<em>P</em>=0.15, after adjusting for age, sex and family structure). Recruitment and phenotyping of additional c.1165_1167delAAC carriers will improve the statistical power and help us understand this potential relationship more clearly.</p><p id="P21" class="p p-last">Rare mutations or genetic diseases have been reported with higher frequency in founder populations, including the OOA. Through genetic drift, rare mutations may increase in frequency in founder populations. Advances in genetic technology have enabled the identification of many of these rare mutations and has provided the opportunity to glean insights into mechanisms underlying both rare and common phenotypes, which would otherwise be difficult in the general population.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933773">10</a>,<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_303933764">11</a></sup> In conclusion, we discovered a rare inactivation mutation in <em>GOT1</em> encoding cytosolic AST in the OOA. As AST is a widely used liver function marker, these mutant carriers will have disguisedly lower or near-normal AST levels in pathogenic liver conditions. The discovery of the <em>GOT1</em> mutation will help to explain and alert physician about lower AST levels under actual liver injury in those carriers. It will also be interesting and clinically and diagnostically significant to examine if such or similar mutations exist in other populations.</p></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Acknowledgments</h2><div class="sec"><p id="__p1">The study was supported by grants of Maryland Clinical Nutrition Research Unit (P30DK072488), the Baltimore Diabetes Research and Training Center (P60DK079637) from the National Institutes of Health, and AHA Scientist Development Grant 0830146N. We gratefully acknowledge our Amish liaisons and field workers and the extraordinary cooperation and support of the Amish community without which these studies would not have been possible.</p></div></div><div id="idm139971333030512" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139971333030512title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"> </p><p id="__p2" class="p p-first"><strong>CONFLICT OF INTEREST</strong></p><p id="__p3" class="p p-last">The authors declare no conflict of interest.</p><p></p></div></div><div id="idm139971333028656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139971333028656title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Rosenthal MD, Glew RH.  <span class="ref-journal">Medical Biochemistry: Human Metabolism in Health and Disease.</span> 1. Wiley; NJ, USA: 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Medical+Biochemistry:+Human+Metabolism+in+Health+and+Disease&amp;author=MD+Rosenthal&amp;author=RH+Glew&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Rej R. A spartate aminotransferase activity and isoenzyme proportions in human liver tissues. <span><span class="ref-journal">Clin Chem. </span>1978;<span class="ref-vol">24</span>:197–199.</span> [<a href="/pubmed/213206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=A+spartate+aminotransferase+activity+and+isoenzyme+proportions+in+human+liver+tissues&amp;author=R+Rej&amp;volume=24&amp;publication_year=1978&amp;pages=197-199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Rej R. Measurement of aspartate aminotransferase activity: effects of oxamate. <span><span class="ref-journal">Clin Chem. </span>1979;<span class="ref-vol">25</span>:555–559.</span> [<a href="/pubmed/466765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Measurement+of+aspartate+aminotransferase+activity:+effects+of+oxamate&amp;author=R+Rej&amp;volume=25&amp;publication_year=1979&amp;pages=555-559&amp;pmid=466765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Rahmioglu N, Andrew T, Cherkas L, Surdulescu G, Swaminathan R, Spector T, et al.  Epidemiology and genetic epidemiology of the liver function test proteins. <span><span class="ref-journal">PLoS One. </span>2009;<span class="ref-vol">4</span>:e4435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2636884/">PMC free article</a>]</span> [<a href="/pubmed/19209234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Epidemiology+and+genetic+epidemiology+of+the+liver+function+test+proteins&amp;author=N+Rahmioglu&amp;author=T+Andrew&amp;author=L+Cherkas&amp;author=G+Surdulescu&amp;author=R+Swaminathan&amp;volume=4&amp;publication_year=2009&amp;pages=e4435&amp;pmid=19209234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Whitfield JB, Zhu G, Nestler JE, Heath AC, Martin NG. Genetic covariation between serum gamma-glutamyltransferase activity and cardiovascular risk factors. <span><span class="ref-journal">Clin Chem. </span>2002;<span class="ref-vol">48</span>:1426–1431.</span> [<a href="/pubmed/12194918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Genetic+covariation+between+serum+gamma-glutamyltransferase+activity+and+cardiovascular+risk+factors&amp;author=JB+Whitfield&amp;author=G+Zhu&amp;author=JE+Nestler&amp;author=AC+Heath&amp;author=NG+Martin&amp;volume=48&amp;publication_year=2002&amp;pages=1426-1431&amp;pmid=12194918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Mitchell BD, McArdle PF, Shen H, Rampersaud E, Pollin TI, Bielak LF, et al.  The genetic response to short-term interventions affecting cardiovascular function: Rationale and design of the heredity and phenotype intervention (HAPI) heart study. <span><span class="ref-journal">Am Heart J. </span>2008;<span class="ref-vol">155</span>:823–828.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443415/">PMC free article</a>]</span> [<a href="/pubmed/18440328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+Heart+J&amp;title=The+genetic+response+to+short-term+interventions+affecting+cardiovascular+function:+Rationale+and+design+of+the+heredity+and+phenotype+intervention+(HAPI)+heart+study&amp;author=BD+Mitchell&amp;author=PF+McArdle&amp;author=H+Shen&amp;author=E+Rampersaud&amp;author=TI+Pollin&amp;volume=155&amp;publication_year=2008&amp;pages=823-828&amp;pmid=18440328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Friedewald WT, Levy RI, Fredrickson DS. Estimation of the concentration of low-density lipoprotein cholesterol in plasma, without use of the preparative ultracentrifuge. <span><span class="ref-journal">Clin Chem. </span>1972;<span class="ref-vol">18</span>:499–502.</span> [<a href="/pubmed/4337382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Chem&amp;title=Estimation+of+the+concentration+of+low-density+lipoprotein+cholesterol+in+plasma,+without+use+of+the+preparative+ultracentrifuge&amp;author=WT+Friedewald&amp;author=RI+Levy&amp;author=DS+Fredrickson&amp;volume=18&amp;publication_year=1972&amp;pages=499-502&amp;pmid=4337382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Melzer D, Perry JR, Hernandez D, Corsi AM, Stevens K, Rafferty I, et al.  genome-wide association study identifies protein quantitative trait loci (pQTLs) <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000072.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2362067/">PMC free article</a>]</span> [<a href="/pubmed/18464913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=genome-wide+association+study+identifies+protein+quantitative+trait+loci+(pQTLs)&amp;author=D+Melzer&amp;author=JR+Perry&amp;author=D+Hernandez&amp;author=AM+Corsi&amp;author=K+Stevens&amp;volume=4&amp;publication_year=2008&amp;pages=e1000072&amp;pmid=18464913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Yuan X, Waterworth D, Perry JR, Lim N, Song K, Chambers JC, et al.  Population-based genome-wide association studies reveal six loci influencing plasma levels of liver enzymes. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:520–528.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561937/">PMC free article</a>]</span> [<a href="/pubmed/18940312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Population-based+genome-wide+association+studies+reveal+six+loci+influencing+plasma+levels+of+liver+enzymes&amp;author=X+Yuan&amp;author=D+Waterworth&amp;author=JR+Perry&amp;author=N+Lim&amp;author=K+Song&amp;volume=83&amp;publication_year=2008&amp;pages=520-528&amp;pmid=18940312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Pollin TI, Damcott CM, Shen H, Ott SH, Shelton J, Horenstein RB, et al.  A null mutation in human APOC3 confers a favorable plasma lipid profile and apparent cardioprotection. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">322</span>:1702–1705.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673993/">PMC free article</a>]</span> [<a href="/pubmed/19074352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+null+mutation+in+human+APOC3+confers+a+favorable+plasma+lipid+profile+and+apparent+cardioprotection&amp;author=TI+Pollin&amp;author=CM+Damcott&amp;author=H+Shen&amp;author=SH+Ott&amp;author=J+Shelton&amp;volume=322&amp;publication_year=2008&amp;pages=1702-1705&amp;pmid=19074352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Shen H, Damcott CM, Rampersaud E, Pollin TI, O’Connell JR, Horenstein RB, et al.  Familial defective apolipoprotein B-100 and increased low-density lipoprotein cholesterol and coronary artery calcification in the old order amish. <span><span class="ref-journal">Arch Intern Med. </span>2010;<span class="ref-vol">170</span>:1850–1855.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3587042/">PMC free article</a>]</span> [<a href="/pubmed/21059979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=Familial+defective+apolipoprotein+B-100+and+increased+low-density+lipoprotein+cholesterol+and+coronary+artery+calcification+in+the+old+order+amish&amp;author=H+Shen&amp;author=CM+Damcott&amp;author=E+Rampersaud&amp;author=TI+Pollin&amp;author=JR+O’Connell&amp;volume=170&amp;publication_year=2010&amp;pages=1850-1855&amp;pmid=21059979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Corpet F. Multiple sequence alignment with hierarchical clustering. <span><span class="ref-journal">Nucleic Acids Res. </span>1988;<span class="ref-vol">16</span>:10881–10890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC338945/">PMC free article</a>]</span> [<a href="/pubmed/2849754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Multiple+sequence+alignment+with+hierarchical+clustering&amp;author=F+Corpet&amp;volume=16&amp;publication_year=1988&amp;pages=10881-10890&amp;pmid=2849754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3608855&amp;issue-id=221098&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm139971331420400title">Abstract</a></li><li><a href="#S1title">INTRODUCTION</a></li><li><a href="#S2title">MATERIALS AND METHODS</a></li><li><a href="#S9title">RESULTS</a></li><li><a href="#S10title">DISCUSSION</a></li><li><a href="#S11title">Acknowledgments</a></li><li><a href="#idm139971333030512title">Footnotes</a></li><li><a href="#idm139971333028656title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3608855/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3608855/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3608855/pdf/nihms447117.pdf">PDF (1.4M)</a></li> | <li><a href="#" data-citationid="PMC3608855" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3608855%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3608855%2F&amp;text=Genome-wide%20association%20study%20identifies%20genetic%20variants%20in%20GOT1%20determining%20serum%20aspartate%20aminotransferase%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3608855%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21900944" data-aiid="3608855" data-aid="3608855" data-iid="221098" data-domainid="319" data-domain="nihpa" data-accid="PMC3608855" data-md5="eef2f6dd989e02ce3d0d4c2a6191161a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:40:13-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D8975DD52B7B1000000000582039B&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3608855%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3608855/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>