<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3136364/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3136364/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2"><meta name="citation_authors" content="Eva Ellinghaus, David Ellinghaus, Philip E. Stuart, Rajan P. Nair, Sophie Debrus, John V. Raelson, Majid Belouchi, Hélène Fournier, Claudia Reinhard, Jun Ding, Yun Li, Trilokraj Tejasvi, Johann Gudjonsson, Stefan W. Stoll, John J. Voorhees, Sylviane Lambert, Stephan Weidinger, Bernadette Eberlein, Manfred Kunz, Proton Rahman, Dafna D. Gladman, Christian Gieger, H. Erich Wichmann, Tom H. Karlsen, Gabriele Mayr, Mario Albrecht, Dieter Kabelitz, Ulrich Mrowietz, Gonçalo R. Abecasis, James T. Elder, Stefan Schreiber, Michael Weichenthal, Andre Franke"><meta name="citation_date" content="November 2010"><meta name="citation_issue" content="11"><meta name="citation_volume" content="42"><meta name="citation_firstpage" content="991"><meta name="citation_doi" content="10.1038/ng.689"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3136364/?report=abstract"><meta name="citation_pmid" content="20953188"><meta name="DC.Title" content="Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Eva Ellinghaus"><meta name="DC.Contributor" content="David Ellinghaus"><meta name="DC.Contributor" content="Philip E. Stuart"><meta name="DC.Contributor" content="Rajan P. Nair"><meta name="DC.Contributor" content="Sophie Debrus"><meta name="DC.Contributor" content="John V. Raelson"><meta name="DC.Contributor" content="Majid Belouchi"><meta name="DC.Contributor" content="Hélène Fournier"><meta name="DC.Contributor" content="Claudia Reinhard"><meta name="DC.Contributor" content="Jun Ding"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Trilokraj Tejasvi"><meta name="DC.Contributor" content="Johann Gudjonsson"><meta name="DC.Contributor" content="Stefan W. Stoll"><meta name="DC.Contributor" content="John J. Voorhees"><meta name="DC.Contributor" content="Sylviane Lambert"><meta name="DC.Contributor" content="Stephan Weidinger"><meta name="DC.Contributor" content="Bernadette Eberlein"><meta name="DC.Contributor" content="Manfred Kunz"><meta name="DC.Contributor" content="Proton Rahman"><meta name="DC.Contributor" content="Dafna D. Gladman"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="H. Erich Wichmann"><meta name="DC.Contributor" content="Tom H. Karlsen"><meta name="DC.Contributor" content="Gabriele Mayr"><meta name="DC.Contributor" content="Mario Albrecht"><meta name="DC.Contributor" content="Dieter Kabelitz"><meta name="DC.Contributor" content="Ulrich Mrowietz"><meta name="DC.Contributor" content="Gonçalo R. Abecasis"><meta name="DC.Contributor" content="James T. Elder"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="Michael Weichenthal"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Date" content="2010 Nov"><meta name="DC.Identifier" content="10.1038/ng.689"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2"><meta property="og:type" content="article"><meta property="og:description" content="Psoriasis is a multifactorial skin disease characterized by epidermal hyperproliferation and chronic inflammation, the most common form of which is psoriasis vulgaris (PsV). We present a genome-wide association analysis of 2,339,118 SNPs in 472 psoriasis ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3136364/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3136364/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3136364/pdf/nihms302957.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D0BDBDD517DF10000000003D10298.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3136364%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3136364</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2011 Jul 14.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20953188" target="pmc_ext" ref="reftype=publisher&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2010 Nov; 42(11): 991–995. </span></a></div><span class="fm-vol-iss-date">Published online 2010 Oct 17. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.689" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.689</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3136364</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS302957</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/20953188">20953188</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies a psoriasis susceptibility locus at <em>TRAF3IP2</em></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623256226304" co-class="co-affbox">Eva Ellinghaus</a>,<sup>1</sup> <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623292126336" co-class="co-affbox">David Ellinghaus</a>,<sup>1</sup> <a href="/pubmed/?term=Stuart%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623249537408" co-class="co-affbox">Philip E. Stuart</a>,<sup>2</sup> <a href="/pubmed/?term=Nair%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623256222832" co-class="co-affbox">Rajan P. Nair</a>,<sup>2</sup> <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623245707520" co-class="co-affbox">Sophie Debrus</a>,<sup>3</sup> <a href="/pubmed/?term=Raelson%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623245705296" co-class="co-affbox">John V. Raelson</a>,<sup>3</sup> <a href="/pubmed/?term=Belouchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623249098672" co-class="co-affbox">Majid Belouchi</a>,<sup>3</sup> <a href="/pubmed/?term=Fournier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623249096448" co-class="co-affbox">Hélène Fournier</a>,<sup>3</sup> <a href="/pubmed/?term=Reinhard%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623254476848" co-class="co-affbox">Claudia Reinhard</a>,<sup>3</sup> <a href="/pubmed/?term=Ding%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623254474624" co-class="co-affbox">Jun Ding</a>,<sup>4</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247952512" co-class="co-affbox">Yun Li</a>,<sup>4</sup> <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247950288" co-class="co-affbox">Trilokraj Tejasvi</a>,<sup>2</sup> <a href="/pubmed/?term=Gudjonsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623259898048" co-class="co-affbox">Johann Gudjonsson</a>,<sup>2</sup> <a href="/pubmed/?term=Stoll%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623259895824" co-class="co-affbox">Stefan W. Stoll</a>,<sup>2</sup> <a href="/pubmed/?term=Voorhees%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247698864" co-class="co-affbox">John J. Voorhees</a>,<sup>2</sup> <a href="/pubmed/?term=Lambert%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247696640" co-class="co-affbox">Sylviane Lambert</a>,<sup>2</sup> <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623251513104" co-class="co-affbox">Stephan Weidinger</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Eberlein%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623251510144" co-class="co-affbox">Bernadette Eberlein</a>,<sup>5</sup> <a href="/pubmed/?term=Kunz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260123232" co-class="co-affbox">Manfred Kunz</a>,<sup>7</sup> <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260121008" co-class="co-affbox">Proton Rahman</a>,<sup>8</sup> <a href="/pubmed/?term=Gladman%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247443920" co-class="co-affbox">Dafna D. Gladman</a>,<sup>9</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247441696" co-class="co-affbox">Christian Gieger</a>,<sup>10</sup> <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260874192" co-class="co-affbox">H. Erich Wichmann</a>,<sup>10,</sup><sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Karlsen%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260870496" co-class="co-affbox">Tom H. Karlsen</a>,<sup>13</sup> <a href="/pubmed/?term=Mayr%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623258579904" co-class="co-affbox">Gabriele Mayr</a>,<sup>14</sup> <a href="/pubmed/?term=Albrecht%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623258577680" co-class="co-affbox">Mario Albrecht</a>,<sup>14</sup> <a href="/pubmed/?term=Kabelitz%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247850976" co-class="co-affbox">Dieter Kabelitz</a>,<sup>15</sup> <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623247848752" co-class="co-affbox">Ulrich Mrowietz</a>,<sup>6</sup> <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260098096" co-class="co-affbox">Gonçalo R. Abecasis</a>,<sup>4</sup> <a href="/pubmed/?term=Elder%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623260095872" co-class="co-affbox">James T. Elder</a>,<sup>2,</sup><sup>16</sup> <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623248208544" co-class="co-affbox">Stefan Schreiber</a>,<sup>1</sup> <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623248206320" co-class="co-affbox">Michael Weichenthal</a>,<sup>6,</sup><sup>17</sup> and  <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188" class="affpopup" co-rid="_co_idm140623249374608" co-class="co-affbox">Andre Franke</a><sup>1,</sup><sup>17</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140623256226304"><h3 class="no_margin">Eva Ellinghaus</h3><p><sup>1</sup> Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Eva Ellinghaus</a></div></div><div id="_co_idm140623292126336"><h3 class="no_margin">David Ellinghaus</h3><p><sup>1</sup> Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">David Ellinghaus</a></div></div><div id="_co_idm140623249537408"><h3 class="no_margin">Philip E. Stuart</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Stuart%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Philip E. Stuart</a></div></div><div id="_co_idm140623256222832"><h3 class="no_margin">Rajan P. Nair</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Nair%20RP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Rajan P. Nair</a></div></div><div id="_co_idm140623245707520"><h3 class="no_margin">Sophie Debrus</h3><p><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</p><div>Find articles by <a href="/pubmed/?term=Debrus%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Sophie Debrus</a></div></div><div id="_co_idm140623245705296"><h3 class="no_margin">John V. Raelson</h3><p><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</p><div>Find articles by <a href="/pubmed/?term=Raelson%20JV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">John V. Raelson</a></div></div><div id="_co_idm140623249098672"><h3 class="no_margin">Majid Belouchi</h3><p><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</p><div>Find articles by <a href="/pubmed/?term=Belouchi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Majid Belouchi</a></div></div><div id="_co_idm140623249096448"><h3 class="no_margin">Hélène Fournier</h3><p><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</p><div>Find articles by <a href="/pubmed/?term=Fournier%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Hélène Fournier</a></div></div><div id="_co_idm140623254476848"><h3 class="no_margin">Claudia Reinhard</h3><p><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</p><div>Find articles by <a href="/pubmed/?term=Reinhard%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Claudia Reinhard</a></div></div><div id="_co_idm140623254474624"><h3 class="no_margin">Jun Ding</h3><p><sup>4</sup> Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Ding%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Jun Ding</a></div></div><div id="_co_idm140623247952512"><h3 class="no_margin">Yun Li</h3><p><sup>4</sup> Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Yun Li</a></div></div><div id="_co_idm140623247950288"><h3 class="no_margin">Trilokraj Tejasvi</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Trilokraj Tejasvi</a></div></div><div id="_co_idm140623259898048"><h3 class="no_margin">Johann Gudjonsson</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Gudjonsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Johann Gudjonsson</a></div></div><div id="_co_idm140623259895824"><h3 class="no_margin">Stefan W. Stoll</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Stoll%20SW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Stefan W. Stoll</a></div></div><div id="_co_idm140623247698864"><h3 class="no_margin">John J. Voorhees</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Voorhees%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">John J. Voorhees</a></div></div><div id="_co_idm140623247696640"><h3 class="no_margin">Sylviane Lambert</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Lambert%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Sylviane Lambert</a></div></div><div id="_co_idm140623251513104"><h3 class="no_margin">Stephan Weidinger</h3><p><sup>5</sup> Department of Dermatology and Allergy, Technische Universität München, 80802 Munich, Germany</p><p><sup>6</sup> Department of Dermatology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Stephan Weidinger</a></div></div><div id="_co_idm140623251510144"><h3 class="no_margin">Bernadette Eberlein</h3><p><sup>5</sup> Department of Dermatology and Allergy, Technische Universität München, 80802 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Eberlein%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Bernadette Eberlein</a></div></div><div id="_co_idm140623260123232"><h3 class="no_margin">Manfred Kunz</h3><p><sup>7</sup> Comprehensive Center for Inflammation Medicine, University of Lübeck, 23538 Lübeck, Germany</p><div>Find articles by <a href="/pubmed/?term=Kunz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Manfred Kunz</a></div></div><div id="_co_idm140623260121008"><h3 class="no_margin">Proton Rahman</h3><p><sup>8</sup> Department of Medicine, Memorial University, St. John’s, Newfoundland A1C 5B8, Canada</p><div>Find articles by <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Proton Rahman</a></div></div><div id="_co_idm140623247443920"><h3 class="no_margin">Dafna D. Gladman</h3><p><sup>9</sup> Division of Rheumatology University Toronto, Psoriatic Arthritis Program, University Health Network, Toronto, Ontario M5T 2S8, Canada</p><div>Find articles by <a href="/pubmed/?term=Gladman%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Dafna D. Gladman</a></div></div><div id="_co_idm140623247441696"><h3 class="no_margin">Christian Gieger</h3><p><sup>10</sup> Institute of Epidemiology, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Christian Gieger</a></div></div><div id="_co_idm140623260874192"><h3 class="no_margin">H. Erich Wichmann</h3><p><sup>10</sup> Institute of Epidemiology, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>11</sup> Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</p><p><sup>12</sup> Klinikum Grosshadern, 81377 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20HE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">H. Erich Wichmann</a></div></div><div id="_co_idm140623260870496"><h3 class="no_margin">Tom H. Karlsen</h3><p><sup>13</sup> Medical Department, Rikshospitalet University Hospital, 0027 Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Karlsen%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Tom H. Karlsen</a></div></div><div id="_co_idm140623258579904"><h3 class="no_margin">Gabriele Mayr</h3><p><sup>14</sup> Max Planck Institute for Informatics, 66123 Saarbrücken, Germany</p><div>Find articles by <a href="/pubmed/?term=Mayr%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Gabriele Mayr</a></div></div><div id="_co_idm140623258577680"><h3 class="no_margin">Mario Albrecht</h3><p><sup>14</sup> Max Planck Institute for Informatics, 66123 Saarbrücken, Germany</p><div>Find articles by <a href="/pubmed/?term=Albrecht%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Mario Albrecht</a></div></div><div id="_co_idm140623247850976"><h3 class="no_margin">Dieter Kabelitz</h3><p><sup>15</sup> Institute for Immunology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Kabelitz%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Dieter Kabelitz</a></div></div><div id="_co_idm140623247848752"><h3 class="no_margin">Ulrich Mrowietz</h3><p><sup>6</sup> Department of Dermatology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Ulrich Mrowietz</a></div></div><div id="_co_idm140623260098096"><h3 class="no_margin">Gonçalo R. Abecasis</h3><p><sup>4</sup> Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20GR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Gonçalo R. Abecasis</a></div></div><div id="_co_idm140623260095872"><h3 class="no_margin">James T. Elder</h3><p><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</p><p><sup>16</sup> Ann Arbor Veterans Affairs Hopital, Ann Arbor, MI 48105, USA</p><div>Find articles by <a href="/pubmed/?term=Elder%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">James T. Elder</a></div></div><div id="_co_idm140623248208544"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>1</sup> Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Stefan Schreiber</a></div></div><div id="_co_idm140623248206320"><h3 class="no_margin">Michael Weichenthal</h3><p><sup>6</sup> Department of Dermatology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Michael Weichenthal</a></div></div><div id="_co_idm140623249374608"><h3 class="no_margin">Andre Franke</h3><p><sup>1</sup> Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=20953188">Andre Franke</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140623247831968_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140623247831968_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140623247831968_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup> Institute of Clinical Molecular Biology, Christian-Albrechts-University, 24105 Kiel, Germany</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup> Department of Dermatology, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup> Genizon BioSciences, Inc., St. Laurent, QC, Canada H4T 2C7</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup> Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup> Department of Dermatology and Allergy, Technische Universität München, 80802 Munich, Germany</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup> Department of Dermatology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup> Comprehensive Center for Inflammation Medicine, University of Lübeck, 23538 Lübeck, Germany</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup> Department of Medicine, Memorial University, St. John’s, Newfoundland A1C 5B8, Canada</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup> Division of Rheumatology University Toronto, Psoriatic Arthritis Program, University Health Network, Toronto, Ontario M5T 2S8, Canada</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup> Institute of Epidemiology, Helmholtz Centre Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup> Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup> Klinikum Grosshadern, 81377 Munich, Germany</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup> Medical Department, Rikshospitalet University Hospital, 0027 Oslo, Norway</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup> Max Planck Institute for Informatics, 66123 Saarbrücken, Germany</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup> Institute for Immunology, University Hospital Schleswig-Holstein, Christian-Albrechts-University, 24105 Kiel, Germany</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup> Ann Arbor Veterans Affairs Hopital, Ann Arbor, MI 48105, USA</div><div id="FN1"><sup>‡</sup>  To whom correspondence should be addressed: Prof. Dr. rer. nat. Andre Franke (<a href="mailto:dev@null" data-email="ed.asocum@eknarf.a" class="oemail">ed.asocum@eknarf.a</a>), Institute of Clinical Molecular Biology, Christian-Albrechts-University Kiel, Schittenhelmstr. 12, D-24105 Kiel, Germany, Tel.: +49-431-597-4138, Fax.: +49-431-597-2196. Prof. Dr. med. Michael Weichenthal (<a href="mailto:dev@null" data-email="ed.leik-inu.ygolotamred@lahtnehcieWM" class="oemail">ed.leik-inu.ygolotamred@lahtnehcieWM</a>), Department of Dermatology, University Hospital Schleswig-Holstein, Schittenhelmstr. 7, D-24105 Kiel, Germany, Tel.: +49-431-597-1537, Fax.: +49-431-597-1606</div><div id="FN2"><sup>17</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140623247831968_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=20953188" target="pmc_ext" ref="reftype=publisher&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3136364/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supple Data.</strong></div><div><a href="/pmc/articles/PMC3136364/bin/NIHMS302957-supplement-Supple_Data.pdf" data-ga-action="click_feat_suppl">NIHMS302957-supplement-Supple_Data.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;09EE7843-D7BE-4720-99C4-084C7AFFCE1A</div></div></dd></dl></div><div id="idm140623248703072" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623248703072title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Psoriasis is a multifactorial skin disease characterized by epidermal hyperproliferation and chronic inflammation, the most common form of which is psoriasis vulgaris (PsV). We present a genome-wide association analysis of 2,339,118 SNPs in 472 psoriasis patients and 1,146 controls from Germany, with follow-up of the 147 most significant SNPs in 2,746 PsV cases and 4,140 controls from three independent replication panels. We identified an association at <em>TRAF3IP2</em> on 6q21 and genotyped two SNPs at this locus in two additional replication panels (the combined discovery and replication panels consisted of 6,487 cases and 8,037 controls; combinded <em>P</em> = 2.36×10<sup>−10</sup> for rs13210247 and combined <em>P</em> = 1.24×10<sup>−16</sup> for rs33980500). About 15% of psoriasis cases develope psoriatic arthritis (PsA). A stratified analysis of our datasets including only PsA cases (1,922 cases compared to 8,037 controls, <em>P</em>=4.57×10<sup>−12</sup> for rs33980500) suggested that <em>TRAF3IP2</em> represents a shared susceptibility for PsV and PsA. <em>TRAF3IP2</em> encodes a protein involved in IL-17 signaling and which interacts with memebers of the Rel/NF-κB transcription factor family.</p></div></div><div id="idm140623256216304" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Psoriasis (MIM 177900) is a chronic immune-mediated and hyperproliferative disorder of the skin that affects up to 3% of individual in populations of European ancestry<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448198">1</a>–<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448220">3</a></sup>. The most common form, PsV, is characterized by red, raised, scaly plaques that commonly occur on the elbows, knees, scalp and lower back<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448214">4</a></sup>.</p><p id="P3">Disease concordance in monozygotic twin pairs amounts to at most 70%<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448192">5</a></sup> and the sibling recurrence risk, λ<sub>s</sub>, of PsV has been estimated to range between 4 and 11<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448228">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448194">6</a></sup>. Part of the genetic susceptibility can be explained by the established susceptibility locus at <em>PSORS1</em> in the human leucocyte antigen (HLA) complex on chromosome 6p21.3 (<em>HLA-C</em>) as well as polymorphisms at the <em>IL12B</em><sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448210">7</a></sup>, <em>IL23R</em><sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448215">8</a>–<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448203">11</a></sup>, <em>IL4/IL13</em><sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448204">12</a></sup>, <em>IL23A, TNIP1</em> and <em>TNFAIP3</em><sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448212">13</a></sup> loci. Because these susceptibility loci only account for a λ<sub>s</sub> of less than 1.35<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448200">13</a></sup>, a large fraction of the heritability for PsV remains unexplained. In addition, a deletion of two <em>LCE</em> cluster genes has been proposed to be a risk factor for the development of psoriasis, implying that structural variants could also contribute to the overall disease susceptibility<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448193">14</a></sup>.</p><p id="P4">To identify additional PsV susceptibility loci, we performed genome-wide SNP genotyping of 487 German PsV cases and 1161 controls (screening panel A, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) using the Illumina HumanHap 550k array. For genotype imputation with phased HapMap data as a reference and subsequent statistical analysis, we used a data set that passed stringent quality control filters. This cleaned data set consisted of 504,742 autosomal SNPs genotyped in 472 cases and 1146 controls. Imputation served to considerably increase the genomic coverage of our study to a total number of 2,339,118 SNPs. Conservatively accounting for multiple testing by Bonferroni correction, the threshold for genome-wide significance was <em>P</em>≤2.14×10<sup>−8</sup> in the imputed data set. A moderate genomic control value of λ<sub>GC</sub>=1.065 indicated a minimal overall inflation of the test statistics due to population stratification. Furthermore, the multidimensional scaling analysis showed genuine European ancestry of panel A and identity-by-state analysis revealed neither non-European “outliers” nor cryptically related individuals after quality control (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). Our screening panel A had 80% power to detect a variant with an odds ratio of 1.38 or higher at the 5% significance level, assuming a frequency of the disease-associated allele of at least 30% in controls.</p><p id="P5">The initial comparison of case-control frequencies confirmed the association of PsV with the established susceptibility loci at <em>HLA-C</em> (rs12191877, <em>P</em>=4.21×10<sup>−32</sup>, OR=2.79, 95% CI=2.35–3.33) and <em>IL12B</em> (rs2546890, <em>P</em>=8.83×10<sup>−8</sup>, OR=0.65, 95% CI=0.56–0.76). Suggestive evidence for association was found for <em>IL23R</em> (rs1004819, <em>P</em>=6.17×10<sup>−4</sup>, OR=1.33, 95% CI=1.13–1.57) and on 1q21, 22 kb upstream of <em>LCE3C</em> (rs4112788, <em>P</em>=6.36×10<sup>−4</sup>, OR=0.76, 95% CI=0.64–0.89) (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2</a>). The lead SNPs rs2546890 and rs1004819 were in moderate linkage disequilibrium (LD) (r<sup>2</sup>&gt;0.50) with the previously described associated variants at the <em>IL12B</em> (rs7709212)<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448231">9</a>,<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448221">15</a></sup> and <em>IL23R</em> (rs2201841)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448227">13</a></sup> loci, respectively. To identify novel susceptibility loci, we excluded all SNPs within the extended HLA complex (chromosome 6p21 at 25–34 Mb) and visually inspected the cluster plots of all index SNPs with a <em>P</em> value less than 0.001 after the clumping procedure. The 180 most strongly associated SNPs were subsequently selected for replication analysis and genotyped in the German replication panel B (681 cases, 1824 controls). After quality control, we excluded 33 follow-up SNPs.</p><p id="P6">In addition, an <em>in silico</em> replication was performed using available genome-wide association study (GWAS) datasets for panel C from the United States (1,303 cases and 1,322 controls; Collaborative Association Study of Psoriasis (CASP))<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448199">13</a></sup> and for panel D form Canada (762 cases and 994 controls; S.D., J.V.R., M.B., H.F. and C.R., unpublished data). We imputed both datasets, that is, the CASP dataset, which was genotyped by Perlegen Sciences, and the Canadian dataset, which was generated using the Illumina Human 1M BeadChip, were imputed. Of the 147 SNPs genotyped in panel B, we obtained genotypes for 126 SNPs in panel C and for 144 SNPs in panel D. Detailed association results including genotype counts of the 147 SNPs are given in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>.</p><p id="P7">Two SNPs upstream of <em>IL12B</em>, included as a positive control for our experiment, achieved genome-wide significance in the combined replication panels B through D (rs2546890: replication stage <em>P</em>=9.07×10<sup>−15</sup>; rs953861: replication stage <em>P</em>=6.50×10<sup>−14</sup>). A previously unidentified association in the combined analysis of the three replication panels was obtained for SNP rs13210247 (replication stage <em>P</em>=6.25×10<sup>−6</sup>, risk allele frequency for cases = 0.09, risk allele frequency for controls = 0.07) with association robust to Bonferroni correction (corrected significance threshold of α=3.4×10<sup>−4</sup>, calculated as 0.05/147). This intronic SNP is located in the <em>TRAF3IP2</em> gene, which encodes the TRAF3 interacting protein 2, and which is also known as <em>ACT1</em>, the NF- κB activator 1 protein. In the combined panels A to D, consisting of a total of 3218 PsV cases and 5286 controls, this SNP achieved genome-wide significance (<em>P</em><sub>comb</sub>.=7.31×10<sup>−9</sup>) (<a href="/pmc/articles/PMC3136364/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). We observed suggestive evidence for association for four other SNPs; however, these associations did not remain significant after Bonferroni correcting for multiple testing. These four SNPs are located in <em>RYR2</em> (rs2485558, replication stage <em>P</em>=0.01, combined <em>P</em>=1.53×10<sup>−5</sup>), in <em>DPP6</em> (rs916514, replication stage <em>P</em>=0.0496, combined <em>P</em>=6.03×10<sup>−4</sup>), in <em>MMP27</em> (rs1939015, replication stage <em>P</em> =0.026, combined <em>P</em>=7.09×10<sup>−4</sup>) and 31 kb downstream of <em>NFκBIA</em> (rs2145623, replication stage <em>P</em>=2.29×10<sup>−3</sup>, combined <em>P</em> =5.01×10<sup>−6</sup>), respectively (<a href="/pmc/articles/PMC3136364/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3136364/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623292781648" class="figure"><a href="/pmc/articles/PMC3136364/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3136364_nihms302957f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms302957f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3136364/bin/nihms302957f1.jpg"></a></div><div id="largeobj_idm140623292781648" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3136364/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3136364/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional plot of <em>TRAF3IP2</em></strong><p id="__p11">Regional plot of the negative decadic logarithm of the combined <em>P</em> values from the imputed panels A, C and D in a ~700 kb window around the intronic lead SNP rs13210247 (blue filled diamond) and the missense SNP rs33980500 (r<sup>2</sup>=0.63). Panels A, C and D were imputed with CEU haplotypes generated by the 1000 Genomes Project (August 2009 release) as a reference. The intronic SNP rs13210247 and the missense SNP rs33980500 were genotyped in the replication panels, and the combined <em>P</em> values of panels A through F are indicated for both SNPs (<a href="/pmc/articles/PMC3136364/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). The magnitude of linkage disequilibrium (LD) with the central SNP rs13210247, measured by r<sup>2</sup>, is reflected by the color of each SNP symbol (for color coding, see upper right corner of each plot). Recombination activity (cM/Mb) is depicted by a blue line.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of association results</strong><p id="__p12">We analyzed the top 147 SNPs of the GWAS (including typed and imputed genotypes) in three independent PsV case-control panels (B through D). Data is shown for the seven SNPs that were nominal significant in the combined replication analysis of panels B through D (replication stage <em>P</em>&lt;0.05, highlighted in bold) with consistent direction of effects in panel A and B. The number of cases and controls of each panel is shown in the top column. Results of all 147 SNPs are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>. SNPs are ranked according to their <em>P</em> value obtained in the GWAS. Nucleotide positions refer to NCBI build 36. <strong>Chr.</strong>, chromosome. <strong>A1</strong> denotes the minor allele and <strong>A2</strong> is the common allele. Allele frequencies of A1 are shown (<strong>AF<sub>A1</sub></strong>). The <strong><em>P</em> value,</strong> the odds ratio (<strong>OR</strong>) and the <strong>95% CI</strong> for carriership of the minor allele A1 are shown for the genome-wide association analysis (panel A). The combined <em>P</em> value of the meta-analysis is shown for the independent replication panels B through D (<strong><em>P</em><sub>repl.</sub></strong>) as well as for the GWAS panel together with the independent replication panels (panels A through D; <strong><em>P</em><sub>comb.</sub></strong>). rs13210247 was additionally tested in replication panels E and F. Results for this SNP in panels B through F (replication; 6,015 cases and 6,891 controls) and panels A through F (GWAS and replication; 6,487 cases and 8,037 controls) are provided in the main text and in <a href="/pmc/articles/PMC3136364/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623291457680" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="4" rowspan="1" valign="bottom" align="center">Panel A – GWAS<br>Germany<br>472 cases<br>1146 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel B<br>Germany<br>681cases<br>1824 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel C<br>CASP<br>1303 cases<br>1322 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel D<br>Genizon<br>762 cases<br>994 controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Panel B – D<br>(replication only)<br>Combined analysis<br>2746 cases<br>4140 controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Panel A – D<br>(GWAS &amp; replication)<br>Combined analysis<br>3218 cases<br>5286 controls</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Chr<br>Pos. (bp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">dbSNP ID</th><th rowspan="1" colspan="1" valign="bottom" align="center">Nearby genes<br>(relative position)</th><th rowspan="1" colspan="1" valign="bottom" align="center">A1<br>A2</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P-value</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>repl.</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>comb.</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">5<br>158,692,478</td><td rowspan="1" colspan="1" valign="top" align="center">rs2546890</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL12B</em> (−2.5 kb)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.36</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">8.83×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.65 (0.56–0.76)</td><td rowspan="1" colspan="1" valign="top" align="center">0.38</td><td rowspan="1" colspan="1" valign="top" align="center">0.44</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">0.45</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>9.07×10<sup>−15</sup></em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.29×10<strong><em><sup>−</sup></em></strong><sup><a href="#R20" rid="R20" class=" bibr popnode">20</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">1<br>235,345,148</td><td rowspan="1" colspan="1" valign="top" align="center">rs2485558</td><td rowspan="1" colspan="1" valign="top" align="center"><em>RYR2</em> (intronic)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>C</td><td rowspan="1" colspan="1" valign="top" align="center">0.31</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">2.81×10<strong><em><sup>−</sup></em></strong><sup>06</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.49 (1.26–1.76)</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>0.01</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">1.53×10<strong><em><sup>−</sup></em></strong><sup>05</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">5<br>158,705,160</td><td rowspan="1" colspan="1" valign="top" align="center">rs953861</td><td rowspan="1" colspan="1" valign="top" align="center"><em>IL12B</em> (−15 kb)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">1.60×10<strong><em><sup>−</sup></em></strong><sup>05</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.51 (1.25–1.82)</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.21</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>6.50×10<sup>−14</sup></em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">6.21×10<strong><em><sup>−</sup></em></strong><sup><a href="#R18" rid="R18" class=" bibr popnode">18</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">14<br>34,908,987</td><td rowspan="1" colspan="1" valign="top" align="center">rs2145623</td><td rowspan="1" colspan="1" valign="top" align="center"><em>NFKBIA</em> (+31 kb)</td><td rowspan="1" colspan="1" valign="top" align="center">C<br>G</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">0.27</td><td rowspan="1" colspan="1" valign="top" align="center">3.01×10<strong><em><sup>−</sup></em></strong><sup>05</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.41 (1.2–1.66)</td><td rowspan="1" colspan="1" valign="top" align="center">0.33</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.26</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>2.29×10<sup>−03</sup></em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">5.01×10<strong><em><sup>−</sup></em></strong><sup>06</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6<br>112,029,413</td><td rowspan="1" colspan="1" valign="top" align="center">rs13210247</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TRAF3IP2</em> (intronic)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">8.17×10<strong><em><sup>−</sup></em></strong><sup>05</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.7 (1.3–2.22)</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>6.25×10<sup>−06</sup></em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">7.31×10<strong><em><sup>−</sup></em></strong><sup>09</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">7<br>154,099,909</td><td rowspan="1" colspan="1" valign="top" align="center">rs916514</td><td rowspan="1" colspan="1" valign="top" align="center"><em>DPP6</em> (intronic)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">1.37×10<strong><em><sup>−</sup></em></strong><sup>04</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.61 (0.47–0.79)</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>0.0496</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">6.03×10<strong><em><sup>−</sup></em></strong><sup>04</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">11<br>102,081,585</td><td rowspan="1" colspan="1" valign="top" align="center">rs1939015</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MMP27</em> (missense)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">1.56×10<strong><em><sup>−</sup></em></strong><sup>03</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69 (0.55–0.87)</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center"><strong><em>0.026</em></strong></td><td rowspan="1" colspan="1" valign="top" align="center">7.09×10<strong><em><sup>−</sup></em></strong><sup>04</sup></td></tr></tbody></table></div><div id="largeobj_idm140623291457680" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3136364/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P8">We also confirmed the previously reported association of PsV and the <em>LCE3C-LCE3B</em> deletion<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448209">14</a></sup> at <em>P</em>=3.80×10<sup>−5</sup> (for details on genotyping and copy number analyses see <strong>Online Methods</strong>).</p><p id="P9">To further validate the newly associated PsV susceptibility gene <em>TRAF3IP2</em>, we genotyped the lead SNP, rs13210247, in the two additional case-control panels E (1,987 cases and 1,661 controls from Michigan) and F (1,282 cases and 1,090 controls from Canada). The combined replication panels B through F comprised 6,015 PsV cases and 6,891 healthy controls, yielding a replication <em>P</em> value of 1.00×10<sup>−7</sup> with the same direction of effect for all five panels (combined <em>P</em> value for panels A through F=2.36×10<sup>−10</sup>).</p><p id="P10">We additionally selected all validated coding SNPs within the <em>TRAF3IP2</em> gene from NCBI’s dbSNP<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448226">16</a></sup> build 130 (rs1043730, rs13190932 and rs33980500) for genotyping in replication panel B. Of the existing three missense SNPs, only rs33980500, located in exon 2, showed a significant association (<em>P</em>=0.0265, OR=1.28, 95% CI=1.03–1.58). Therefore, this SNP was also genotyped in the replication panels C through F. Association analysis of the obtained replication cohort (panel B through F) yielded a highly significant <em>P</em> value of 8.00×10<sup>−14</sup> (combined <em>P</em> value for panels A through F=1.24×10<sup>−16</sup>) (<a href="/pmc/articles/PMC3136364/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). Because the missense SNP was not available from the HapMap II CEU imputation reference, we performed imputation based on CEU haplotypes generated by the 1000 Genomes project for panels A, C and D in order to fine map a region of 700 kb around <em>TRAF3IP2</em> containing 1,935 SNPs (<a href="/pmc/articles/PMC3136364/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). The intronic <em>TRAF3IP2</em> SNP rs13210247 and the missense SNP rs33980500 are about 9.5 kb apart and are in moderate LD (LD in panels A to F; r<sup>2</sup>=0.63). We performed a logistic regression analysis to test for the independence of these two markers. When conditioning for the missense SNP rs33980500, the intronic SNP rs13210247 was not significantly associated with disease (<em>P</em>=0.709). In the reverse comparison, the missense SNP rs33980500 remained significantly associated (<em>P</em>=3.77×10<sup>−7</sup>), indicating that it can account for the association at the <em>TRAF3IP2</em> gene locus. In a subset of German cases and controls (993 cases and 2,277 controls from panels A and B) for which HLA-Cw6 status was available, we tested for the presence of a statistical interaction between HLA-Cw6 carriership and rs33980500 genotypes but found no statistical support for interaction (<em>P</em>=0.77).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of association results for the two <em>TRAF3IP2</em> SNPs genotyped in additional replication panels</strong><p id="__p13">The lead SNP rs13210247 and the missense SNP rs33980500 were genotyped in five independent PsV case-control panels (B through F, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). The number of cases and controls of each panel is shown in the top column. Nucleotide positions refer to NCBI build 36. <strong>Chr.,</strong> chromosome. <strong>A1</strong> denotes the rare allele and <strong>A2</strong> is the common allele. Allele frequencies of A1 are shown (<strong>AF<sub>A1</sub></strong>). The combined <em>P</em> value of the meta-analysis is shown for the independent replication panels B to F (<strong><em>P</em><sub>repl.</sub></strong>) as well as for the GWAS panel together with the independent replication panels (panels A through F; <strong><em>P</em><sub>comb.</sub></strong>).</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140623252708016" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th rowspan="1" colspan="1" valign="bottom" align="center"></th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel A<br>Germany<br>472cases<br>1146 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel B<br>Germany<br>681cases<br>1824 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel C<br>CASP<br>1303 cases<br>1322 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel D<br>Genizon<br>762 cases<br>994 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel E<br>Michigan<br>1987 cases<br>1661 controls</th><th colspan="2" rowspan="1" valign="bottom" align="center">Panel F<br>Canada<br>1282 cases<br>1090 controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Panel B – F<br>(replication only)<br>Combined analysis<br>6015 cases<br>6891 controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Panel A – F<br>(GWAS &amp; replication)<br>Combined analysis<br>6487 cases<br>8037 controls</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">Chr<br>Pos. (bp)</th><th rowspan="1" colspan="1" valign="bottom" align="center">dbSNP ID</th><th rowspan="1" colspan="1" valign="bottom" align="center">Nearby genes<br>(relative<br>position)</th><th rowspan="1" colspan="1" valign="bottom" align="center">A1<br>A2</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Case</th><th rowspan="1" colspan="1" valign="bottom" align="center">AF<sub>A1</sub><br>Control</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>repl.</sub></th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em><sub>comb.</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="center">6<br>112,029,413</td><td rowspan="1" colspan="1" valign="top" align="center">rs13210247</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TRAF3IP2</em> (intronic)</td><td rowspan="1" colspan="1" valign="top" align="center">G<br>A</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">1.00×10<strong><em><sup>−</sup></em></strong><sup>07</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.36×10<strong><em><sup>−</sup></em></strong><sup><a href="#R10" rid="R10" class=" bibr popnode">10</a></sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="center">6<br>112,019,955</td><td rowspan="1" colspan="1" valign="top" align="center">rs33980500</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TRAF3IP2</em> (missense)</td><td rowspan="1" colspan="1" valign="top" align="center">T<br>C</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">8.00×10<strong><em><sup>−</sup></em></strong><sup><a href="#R14" rid="R14" class=" bibr popnode">14</a></sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.24×10<strong><em><sup>−</sup></em></strong><sup><a href="#R16" rid="R16" class=" bibr popnode">16</a></sup></td></tr></tbody></table></div><div id="largeobj_idm140623252708016" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3136364/table/T2/?report=objectonly">Open in a separate window</a></div></div><p id="P11">The missense SNP rs33980500 (Asp19Asn) causes a mutation from aspartic acid to asparagine in the protein sequence and the resulting change in charge (a negative electric charge to nonpolar) might have an impact on the protein structure and, hence, its function. Additionally, the mutation is located in a region that is more than 90% conserved among different species (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 3</a>), which further implies a possible functional consequence of this change.</p><p id="P12">Furthermore, we performed <em>in silico</em> analyses and found a putative TRAF6-binding peptide (amino acids 13 to 21) containing the SNP Asp19Asn. This peptide is similar to a known TRAF6-binding motif of CD40, a member of the tumor necrosis factor receptor superfamily<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448216">17</a></sup>. Notably, the mutation of the accordant seventh peptide residue (asparagine to aspartate) has previously been observed to change the binding affinity of TRAF6 to CD40 significantly<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448205">18</a></sup>. Therefore, rs33980500 may also affect the interaction of TRAF3IP2 to TRAFs and, thus, the involved inflammatory pathways.</p><p id="P13">About 15% of individuals with psoriasis develop psoriatic arthritis (PsA), which is an inflammatory, disabling arthrits<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448229">2</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448217">19</a></sup>. This suggests that PsV and PsA share common susceptibility factors. To determine the association of the <em>TRAF3IP2</em> missense SNP with PsA, we performed a PsA stratified analysis of panel A through F. Association analysis of all PsA patients and controls within our combined sample (1,919 cases and 8,037 controls) yielded a <em>P</em> value of 4.57×10<sup>−12</sup> for rs33980500 (OR=1.57, CI=1.38–1.78). In an equally sized subgroup of randomly extracted PsV cases without PsA, we obtained a <em>P</em> value of 2.04×10<sup>−6</sup> (OR=1.37, CI=1.20–1.57). The overlap in confidence intervals for the two associations, along with the fact that no significant difference was obtained when comparing the 1,919 PsA cases versus the randomly selected 1,919 PsV cases suggests that the <em>TRAF3IP2</em> gene locus represents a shared susceptibility locus for PsA and PsV.</p><p id="P14">Gene expression of psoriatic and uninvolved skin differs significantly for hundreds of genes that are involved in immune response or in the regulation of cellular differentiation and proliferation<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448196">20</a></sup>. To check if altered gene activity might trigger disease progression, we examined expression levels of several loci in biopsies from 57 healthy controls and compared these to expression levels of biopsies of involved and uninvolved skin from 53 PsV cases (subgroup of panel C). We analyzed the <em>TRAF3IP2</em> locus and some genes that potentially act downstream of <em>TRAF3IP2</em>. For the <em>TRAF3IP2</em> locus, the expression level was slightly altered between involved and uninvolved skin (<em>P</em>=2.2×10<sup>−3</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 4</a>). However, the dosage of risk alleles at the according SNPs did not correlate with transcript levels for the genes in involved, uninvolved or normal skin (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P15">Genetic variants in the <em>TRAF3IP2</em> locus are implicated in PsV and PsA susceptibility for the first time, to our knowledge, by our study. The gene product of <em>TRAF3IP2</em> is a positive signaling adaptor required for IL-17-mediated T-cell immune responses<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448191">21</a></sup>. The TRAF3IP2 protein interacts with tumor necrosis factor receptor-associated factor (TRAF) proteins and either I-κB kinase or mitogen-activated protein kinase to activate either NF-κB or Jun kinase. Upon recruitment to CD40 and the BAFF receptor in B-cells, TRAF3IP2 also negatively regulates B-cell survival through its interaction with TRAF3<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448224">22</a></sup>. The negative regulation of CD40-BAFF-mediated B cell functions results in ‘hyper-T cell-dependent’ and T cell-independent immune responses in TRAF3IP2-deficient mice. Researchers in a previous study also found that TRAF3IP2 is a key component of IL-17-mediated signaling and that IL-17-mediated gene expression is completely abolished in TRAF3IP2-deficient mouse embryonic fibroblasts. In addition, epithelial cells could be identified as the critical cell type in which TRAF3IP2 mediates IL-17-dependent inflammatory disease<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448222">21</a></sup>. In PsV, a subset of T-cells expressing IL-17 plays a major role<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448195">23</a></sup>. The IL-17 expressing T-cells that are abundant in the epidermis of psoriatic lesions but which are absent in healthy donor epidermis are CD8+ cells, whereas in lesional psoriatic dermis, mainly IL-17 expressing CD4+ (but also CD8+) T-cells are strongly increased relative to healthy skin. Epidermal hyperplasia and the production of innate immune peptides such as human β-defensin 2 (hBD-2) are associated with these IL-17+ T-cells<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448225">24</a></sup>. The crucial role of IL-17 is also supported by the observation that in psoriasis treatment response to TNF inhibitors correlates with suppressed IL-17 signaling<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448211">25</a></sup>. Moreover, we showed that in immortalized N-TERT keratinocytes (N-TERT KCs) and in normal human keratinocytes (NHKs) IL-17A and TNF-α together induce the expression of <em>DEFB4</em> (encoding hBD-2) which is absent in N-TERT KCs and NHKs with silenced <em>TRAF3IP2</em> (see <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 5</a>). In epithelial cells, the binding of IL-17A and/or IL-17F to the heterodimeric IL-17R leads to the recruitment of TRAF3IP2 through homotypic interactions between conserved (SEFIR) domains. This allows the incorporation of TRAF6 into the signaling complex and then downstream activation of the NF-κB and mitogen-activated protein kinase pathways<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448201">26</a></sup>. We thus speculate that a dysregulation of TRAF3IP2 might have a major impact on IL-17 signaling and, hence, on the activation of NFκB-pathways, leading to the upregulation of pro-inflammatory factors.</p><p id="P16" class="p">For the <em>TRAF3IP2</em> locus newly identified here, fine mapping and resequencing efforts together with extensive functional studies are required to detect all potential causal variants and thus to specify the contribution of the locus to overall disease susceptibility.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Online Methods</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Recruitment of patients and healthy controls</h3><p id="P19" class="p p-first">Samples were organized in panels that corresponded to the successive steps of the present study. All individual panels (A–F) were independent from each other. All German patients of panels A and B were recruited either at the Department of Dermatology of the Christian-Albrechts-University Kiel or the Department of Dermatology and Allergy of the Technical University Munich through local outpatient services. Individuals were considered to be affected if chronic plaque or guttate psoriasis lesions covered more than 1% of the total body surface area or if at least two skin, scalp, nail or joint lesions were clinically diagnosed of psoriasis. Psoriatic arthritis (PsA) was diagnosed by clinical finding of joint complaints and radiologic and rheumatologic confirmation by criteria according to Moll and Wright<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448207">27</a></sup>, or more recently to the Classification Criteria for Psoriatic Arthritis (CASPAR)<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448202">28</a></sup>.</p><p id="P20">2510 German healthy control individuals of panel A and B were obtained from the popgen biobank<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448219">29</a></sup>. 483 German healthy controls were selected from the KORA S4 survey, an independent population-based sample from the general population living in the region of Augsburg, southern Germany<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448197">30</a></sup>.</p><p id="P21">The American study population (panel C) consisted of 1303 psoriasis cases and 1322 controls after quality control. The data sets used for the analyses described in this manuscript were obtained from the database of Genotype and Phenotype (dbGaP). The genotyping of samples was provided through the Genetic Association Information Network (GAIN). Samples and associated phenotype data were provided by the “Collaborative Association Study of Psoriasis” (CASP). Funding support for CASP was provided by the National Institutes of Health, the Foundation for NIH’s Genetic Association Information Network and the National Psoriasis Foundation.</p><p id="P22">The Canadian population sample used for panel D consisted of 762 psoriasis patients and 994 controls, sampled from the Québec founder population (QFP). Membership in the QFP was defined as having four grandparents with French-Canadian family names who were born in the Province of Québec, Canada, or in adjacent areas of the Provinces of New Brunswick and Ontario or in New England or New York State. This criterion assures that all subjects were descendant from French-Canadians living before the 1960s, after which time admixture with non-French-Canadians became more common.</p><p id="P23">For replication of the two <em>TRAF3IP2</em> SNPs (rs13210247 and rs33980500), panel E was tested in addition for an association. This panel comprised 1987 psoriasis cases and 1661 controls of European ancestry. The sample was collected at the University of Michigan and selection criteria for cases included a requirement of at least two psoriatic plaques or a single plaque occupying &gt;1% of total body surface area outside scalp. Individuals that presented only palmoplantar psoriasis, inverse psoriasis or sebopsoriasis were excluded. Individuals used as controls were older than 18 years, had no history of psoriasis and no family history of psoriasis.</p><p id="P24">Panel F, also used only for replication of the two <em>TRAF3IP2</em> SNPs, consisted of 1282 psoriasis cases and 1090 controls from Toronto and Newfoundland, Canada. Psoriasis was diagnosed by a dermatologist. Control individuals showed no evidence of psoriasis, rheumatoid arthritis or other auto-immune disorders.</p><p id="P25" class="p p-last">Written, informed consent was obtained from all study participants and all protocols were approved by the respective institutional ethical review committees of the participating centres.</p></div><div id="S7" class="sec"><h3 id="S7title">SNP genotyping for genome-wide screen</h3><p id="P26" class="p p-first">The genotyping for the GWAS - which was part of the German GWAS initiative funded by the National Genome Research Network (NGFN) - was performed by Illumina’s service facility (San Diego, CA, USA) using the Illumina HumanHap 550k v1 with 561,466 SNP markers (San Diego, CA, USA). All experimental steps were carried out according to standard protocols.</p><p id="P27" class="p p-last">We excluded 5 samples with more than 10% missing genotypes (i.e. call rate &lt;90%). Individuals who showed statistically relevant genetic dissimilarity to the other subjects (population outliers), or who showed evidence for cryptic relatedness to other study participants (unexpected duplicates, first- or second-degree relatives) were removed (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1</a>). These quality control measures left 472 psoriasis samples and 1146 control samples for inclusion in screening panel A. All gender assignments could be verified by reference to the proportion of heterozygous SNPs on the X chromosome. Before analysis we excluded 56,724 markers (10% of all SNPs) that had a low genotype call rate (&lt;95% in cases or controls; n=4,254), were monomorphic or rare (minor allele frequency &lt;2% in cases or controls; n=30,862), deviated from Hardy-Weinberg equilibrium (HWE) in the control sample (<em>P</em><sub>HWE</sub>&lt;0.01; n=7,600), or that were non-autosomal SNPs (n=14,008).</p></div><div id="S8" class="sec"><h3 id="S8title">Imputation</h3><p id="P28" class="p p-first">Genotype imputation was performed using a hidden Markov model algorithm implemented in the software program MACH v.1.0.16<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448230">31</a></sup> to infer missing genotypes <em>in silico</em>. As a reference, HapMap II CEU phased haplotypes<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448218">32</a></sup> were used.</p><p id="P29" class="p p-last">As input for the imputation only genotyped SNPs that passed quality control were used. Of the imputed SNPs we analyzed only these SNPs which could be imputed with a relatively high confidence (estimated r<sup>2</sup> between imputed SNP and true genotypes ≥0.8), had a minor allele frequency &gt;2% in cases or controls and a HWE <em>P</em>-value &lt;0.01 in the control sample. To take imputation uncertainty into account, we used allelic dosage association as implemented in the program MACH2DAT<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448206">31</a></sup>. The allelic dosage is the weighted sum of the genotype class probabilities.</p></div><div id="S9" class="sec"><h3 id="S9title">SNP selection for replication</h3><p id="P30" class="p p-first-last">SNPs of the genome-wide scan that passed quality control were analyzed using gPLINK v2.049 in combination with PLINK v1.05<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448213">33</a></sup>. The SNP list (including imputed and genotyped SNPs) was pruned for redundancy due to linkage disequilibrium by using the --clump command of PLINK. For all genotyped index SNPs with a <em>P</em>-value &lt;10<sup>−3</sup>, a visual inspection of the cluster plots was performed. SNPs that did not pass the visual inspection were excluded from further analyses. The 180 most strongly associated index SNPs of the clumps (<em>P</em>&lt;2.6×10<sup>−4</sup>) were ordered as genotyping assays. If the genotyping assay design was not possible for a SNP, the next best SNP from the clump was chosen for the assay design.</p></div><div id="S10" class="sec"><h3 id="S10title">SNPlex and TaqMan genotyping</h3><p id="P31" class="p p-first">The ligation-based SNPlex<sup>™</sup> genotyping system and functionally tested TaqMan<sup>®</sup> SNP Genotyping Assays (Applied Biosystems, Foster City, CA, USA) were used to genotype variants in replication panel B. For technical replication of the results of the two <em>TRAF3IP2</em> SNPs, TaqMan<sup>®</sup> SNP Genotyping Assays were used in panels A-E while the Sequenom Platform was used for panel F.</p><p id="P32" class="p p-last">Of 180 selected SNPs 147 SNPs passed quality control. These SNPs had a high call rate (&gt;90% in cases <span class="underline">or</span> controls; 25 SNPs were removed), were not monomorphic (minor allele frequency &gt;1% in cases <span class="underline">or</span> controls; 2 SNPs were removed), and did not deviate from Hardy-Weinberg equilibrium (HWE) in the control population (<em>P</em><sub>HWE</sub>&gt;0.0001; 5 SNPs were removed). One SNP failed genotyping and was excluded from the analysis.</p></div><div id="S11" class="sec"><h3 id="S11title">CNV genotyping and quality control</h3><p id="P33" class="p p-first">The functionally tested TaqMan<sup>®</sup> Copy Number Assays Hs 02550639_cn (in <em>LCE3C</em>) and Hs 02878369_cn (in <em>LCE3B</em>) (Applied Biosystems) were used to genotype copy number variation within the late cornified envelope (<em>LCE</em>) gene cluster in samples of panel A and B (754 cases and 1052 controls). Genotyping was carried out according to Applied Biosystems standard protocols with 5ng of dried DNA per well.</p><p id="P34" class="p p-last">All cases were genotyped thrice and all control samples were genotyped four times. The generated data was analyzed with the analysis software CopyCaller<sup>™</sup> v1.0. The analysis settings were selected as recommended by the <em>CopyCaller<sup>™</sup> Software User Guide</em>. The chosen confidence threshold of the associated predicted copy number was ≥95%. Control samples were removed when more than one of the four measurements did not pass quality control whereas cases were removed when one of the three measurements did not pass quality control. Altogether, 736 cases and 932 controls remained for association analysis (94% of all samples).</p></div><div id="S12" class="sec"><h3 id="S12title">Statistical analyses</h3><p id="P35" class="p p-first-last">Power calculations were carried out using PS Power and Sample Size v3.0.12<sup><a href="#R34" rid="R34" class=" bibr popnode">34</a></sup>. GWAS data were analyzed using R statistical environment version 2.10.0 and gPLINK v2.049 in combination with PLINK v1.05<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448208">33</a></sup>. The --clump command was used to reduce the number of SNPs for follow-up by removing correlated hit SNPs. The meta-analysis of the different panels was performed with METAL.</p></div><div id="S13" class="sec sec-last"><h3 id="S13title">TRAF3IP2 silencing in keratinocytes</h3><p id="P36" class="p p-first-last">Small hairpin RNA (shRNA) targeting enhanced green fluorescent protein (EGFP) was cloned into the pLenti4/Block-iT-DEST (Invitrogen, Carlsbad, CA) as previously described<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_380293360">35</a></sup>. ShRNA lentiviral constructs directed against <em>TRAF3IP2</em> were obtained from Sigma (St. Louis, MO). Lentiviral particles were produced in 293FT cells and used to infect immortalized N-TERT keratinocytes (kindly provided by Dr. James Rheinwald) and normal human keratinocytes (NHK) at 10–20% percent confluence as previously described<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_227448223">36</a></sup>. After 72 h of infection the cells were incubated in basal medium and stimulated for 30 h with IL17 (10 ng/ml), IL22 (10 ng/ml) and/or TNF-α (20 ng/ml). Total RNA was isolated using RNeasy Mini Kits with on-column DNAse digestion according to the manufacturer’s instructions (Qiagen, Valencia, CA) and reversed transcribed using the High Capacity cDNA Reverse Transcription Kit (Applied Biosystems, Foster City, CA). Real-time PCR analysis for <em>DEFB4</em>, <em>TRAF3IP2</em> and the control gene <em>RPLP0</em> (ribosomal protein P0) was performed using pre-validated TaqMan gene expression assays from Applied Biosystems according to the manufacturer’s instructions. Target gene expression was normalized to the control gene <em>RPLP0</em> and mRNA levels were expressed as percent of <em>RPLP0</em>.</p></div></div><div id="S14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supple Data</h4><div class="sup-box half_rhythm" id="N0x9b81040N0xaaa89b0"><a href="/pmc/articles/PMC3136364/bin/NIHMS302957-supplement-Supple_Data.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.3M, pdf)</sup></div></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank all individuals with psoriasis, their families and physicians for their cooperation. We acknowledge the cooperation of Genizon Biosciences. We wish to thank Tanja Wesse, Tanja Henke, Catharina Fürstenau, Susan Ehlers, Meike Davids and Rainer Vogler for expert technical help. This study was supported by the German Ministry of Education and Research (BMBF) through the National Genome Research Network (NGFN), the Popgen biobank, and the KORA research platform (KORA: Cooperative Research in the Region of Augsburg) KORA was initiated and financed by the Helmholtz Zentrum München-National Research Center for Environmental Health, which is funded by the German Federal Ministry of Education, Science, Research and Technology and by the State of Bavaria and the Munich Center of Health Sciences (MC Health) as part of LMUinnovativ. The project received infrastructure support through the DFG cluster of excellence “Inflammation at Interfaces”. This research was also supported by grants R01-AR42742, R01-AR050511 and R01-AR054966 from the National Institutes of Health.</p></div></div><div id="idm140623261247872" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623261247872title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><div><p id="__p3" class="p p-first">Author contributions</p>E.E. performed SNP selection, genotyping, data analysis and prepared figures and tables. A.F. helped with data analysis. D.E. performed the imputation and generated the regional association plots. P.E.S., J.G., J.D. and Y.L. performed the expression and eQTL analyses. S.W.S. and S.L performed shRNA experiments. G.A. helped with statistical analyses and interpretation of the results. M.A. and G.M. performed <em>in silico</em> protein analyses. M.W., U.M., S.W., B.E. and M.K. coordinated the recruitment and collected phenotype data of panels A and B.C.G. and H.E.W. provided the KORA control samples. J.T.E., J.J.V., R.P.N., T.T., S.D., J.V.R., M.B., H.F., C.R., P.R. and D.D.G. provided the replication samples C to F and respective genotypes and phenotypes. T.H.K. R.P.N. and D.K. helped with genotyping. D.E., M.W., J.T.E., T.H.K. and S.S. edited the manuscript and A.F. planned and supervised the study. E.E. and A.F. drafted the manuscript and all authors approved the final draft.</div><p class="fn sec" id="FN4"> </p><p id="__p5" class="p p-first">Competing financial interests</p><p id="__p6" class="p p-last">The authors declare no competing financial interests.</p><p></p><p class="fn sec" id="FN5"></p><p id="__p7" class="p p-first">Accession codes</p><p id="__p8" class="p p-last">The CASP GWAS data set (panel C) is deposited in dbGaP under the accession code phs000019.v1.p1.</p><p></p><p class="fn sec" id="FN6"></p><p id="__p9" class="p p-first">URLs</p><p id="__p10" class="p">METAL software: <a href="http://www.sph.umich.edu/csg/abecasis/Metal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/Metal/</a></p><p id="P37" class="p">dbGAP: <a href="http://www.ncbi.nlm.nih.gov/gap/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap/</a></p><p id="P38" class="p">NGFN Funding, press release 04-26-2007: <a href="http://www.ngfn.de/englisch/index_368.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ngfn.de/englisch/index_368.htm</a></p><p id="P39" class="p">MACH software: <a href="http://www.sph.umich.edu/csg/abecasis/mach/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/mach/</a></p><p id="P40" class="p">HapMap data: <a href="http://hapmap.ncbi.nlm.nih.gov/downloads/phasing/2006-07_phaseII/phased/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hapmap.ncbi.nlm.nih.gov/downloads/phasing/2006-07_phaseII/phased/</a></p><p id="P41" class="p">MACH2DAT software: <a href="http://www.sph.umich.edu/csg/abecasis/mach/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/mach/</a>)</p><p id="P42" class="p">PLINK v1.05: <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a></p><p id="P43" class="p p-last">R statistical environment version 2.10.0: <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a></p><p></p></div></div><div id="idm140623250018960" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140623250018960title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Griffiths CE, Barker JN. Pathogenesis and clinical features of psoriasis. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">370</span>:263–71.</span> [<a href="/pubmed/17658397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pathogenesis+and+clinical+features+of+psoriasis&amp;author=CE+Griffiths&amp;author=JN+Barker&amp;volume=370&amp;publication_year=2007&amp;pages=263-71&amp;pmid=17658397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Bowcock AM, Barker JN. Genetics of psoriasis: the potential impact on new therapies. <span><span class="ref-journal">J Am Acad Dermatol. </span>2003;<span class="ref-vol">49</span>:S51–6.</span> [<a href="/pubmed/12894126" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Dermatol&amp;title=Genetics+of+psoriasis:+the+potential+impact+on+new+therapies&amp;author=AM+Bowcock&amp;author=JN+Barker&amp;volume=49&amp;publication_year=2003&amp;pages=S51-6&amp;pmid=12894126&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Gottlieb AB. Psoriasis: emerging therapeutic strategies. <span><span class="ref-journal">Nat Rev Drug Discov. </span>2005;<span class="ref-vol">4</span>:19–34.</span> [<a href="/pubmed/15688070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Drug+Discov&amp;title=Psoriasis:+emerging+therapeutic+strategies&amp;author=AB+Gottlieb&amp;volume=4&amp;publication_year=2005&amp;pages=19-34&amp;pmid=15688070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Oudot T, et al.  An association study of 22 candidate genes in psoriasis families reveals shared genetic factors with other autoimmune and skin disorders. <span><span class="ref-journal">J Invest Dermatol. </span>2009;<span class="ref-vol">129</span>:2637–45.</span> [<a href="/pubmed/19554025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=An+association+study+of+22+candidate+genes+in+psoriasis+families+reveals+shared+genetic+factors+with+other+autoimmune+and+skin+disorders&amp;author=T+Oudot&amp;volume=129&amp;publication_year=2009&amp;pages=2637-45&amp;pmid=19554025&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Bhalerao J, Bowcock AM. The genetics of psoriasis: a complex disorder of the skin and immune system. <span><span class="ref-journal">Hum Mol Genet. </span>1998;<span class="ref-vol">7</span>:1537–45.</span> [<a href="/pubmed/9735374" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+genetics+of+psoriasis:+a+complex+disorder+of+the+skin+and+immune+system&amp;author=J+Bhalerao&amp;author=AM+Bowcock&amp;volume=7&amp;publication_year=1998&amp;pages=1537-45&amp;pmid=9735374&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Elder JT, et al.  The genetics of psoriasis. <span><span class="ref-journal">Arch Dermatol. </span>1994;<span class="ref-vol">130</span>:216–24.</span> [<a href="/pubmed/8304761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Dermatol&amp;title=The+genetics+of+psoriasis&amp;author=JT+Elder&amp;volume=130&amp;publication_year=1994&amp;pages=216-24&amp;pmid=8304761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Tsunemi Y, et al.  Interleukin-12 p40 gene (IL12B) 3[prime]-untranslated region polymorphism is associated with susceptibility to atopic dermatitis and psoriasis vulgaris. <span><span class="ref-journal">J Dermatol Sci. </span>2002;<span class="ref-vol">30</span>:161–166.</span> [<a href="/pubmed/12413772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dermatol+Sci&amp;title=Interleukin-12+p40+gene+(IL12B)+3[prime]-untranslated+region+polymorphism+is+associated+with+susceptibility+to+atopic+dermatitis+and+psoriasis+vulgaris&amp;author=Y+Tsunemi&amp;volume=30&amp;publication_year=2002&amp;pages=161-166&amp;pmid=12413772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Cargill M, et al.  A large-scale genetic association study confirms IL12B and leads to the identification of IL23R as psoriasis-risk genes. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">80</span>:273–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M+Cargill&amp;volume=80&amp;publication_year=2007&amp;pages=273-90&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Capon F, et al.  Sequence variants in the genes for the interleukin-23 receptor (IL23R) and its ligand (IL12B) confer protection against psoriasis. <span><span class="ref-journal">Hum Genet. </span>2007;<span class="ref-vol">122</span>:201–206.</span> [<a href="/pubmed/17587057" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Genet&amp;title=Sequence+variants+in+the+genes+for+the+interleukin-23+receptor+(IL23R)+and+its+ligand+(IL12B)+confer+protection+against+psoriasis&amp;author=F+Capon&amp;volume=122&amp;publication_year=2007&amp;pages=201-206&amp;pmid=17587057&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Nair RP, et al.  Polymorphisms of the IL12B and IL23R genes are associated with psoriasis. <span><span class="ref-journal">J Invest Dermatol. </span>2008;<span class="ref-vol">128</span>:1653–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2739284/">PMC free article</a>]</span> [<a href="/pubmed/18219280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=Polymorphisms+of+the+IL12B+and+IL23R+genes+are+associated+with+psoriasis&amp;author=RP+Nair&amp;volume=128&amp;publication_year=2008&amp;pages=1653-61&amp;pmid=18219280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Liu Y, et al.  A genome-wide association study of psoriasis and psoriatic arthritis identifies new disease loci. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000041.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2274885/">PMC free article</a>]</span> [<a href="/pubmed/18369459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+of+psoriasis+and+psoriatic+arthritis+identifies+new+disease+loci&amp;author=Y+Liu&amp;volume=4&amp;publication_year=2008&amp;pages=e1000041&amp;pmid=18369459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Chang M, et al.  Variants in the 5q31 cytokine gene cluster are associated with psoriasis. <span><span class="ref-journal">Genes Immun. </span>2008;<span class="ref-vol">9</span>:176–81.</span> [<a href="/pubmed/18075513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Variants+in+the+5q31+cytokine+gene+cluster+are+associated+with+psoriasis&amp;author=M+Chang&amp;volume=9&amp;publication_year=2008&amp;pages=176-81&amp;pmid=18075513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Nair RP, et al.  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=RP+Nair&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">de Cid R, et al.  Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:211–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128734/">PMC free article</a>]</span> [<a href="/pubmed/19169253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Deletion+of+the+late+cornified+envelope+LCE3B+and+LCE3C+genes+as+a+susceptibility+factor+for+psoriasis&amp;author=R+de+Cid&amp;volume=41&amp;publication_year=2009&amp;pages=211-5&amp;pmid=19169253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Zhang XJ, et al.  Polymorphisms in interleukin-15 gene on chromosome 4q31.2 are associated with psoriasis vulgaris in Chinese population. <span><span class="ref-journal">J Invest Dermatol. </span>2007;<span class="ref-vol">127</span>:2544–51.</span> [<a href="/pubmed/17554368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=Polymorphisms+in+interleukin-15+gene+on+chromosome+4q31.2+are+associated+with+psoriasis+vulgaris+in+Chinese+population&amp;author=XJ+Zhang&amp;volume=127&amp;publication_year=2007&amp;pages=2544-51&amp;pmid=17554368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Sherry ST, et al.  dbSNP: the NCBI database of genetic variation. <span><span class="ref-journal">Nucleic Acids Res. </span>2001;<span class="ref-vol">29</span>:308–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC29783/">PMC free article</a>]</span> [<a href="/pubmed/11125122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=dbSNP:+the+NCBI+database+of+genetic+variation&amp;author=ST+Sherry&amp;volume=29&amp;publication_year=2001&amp;pages=308-11&amp;pmid=11125122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Ye H, et al.  Distinct molecular mechanism for initiating TRAF6 signalling. <span><span class="ref-journal">Nature. </span>2002;<span class="ref-vol">418</span>:443–7.</span> [<a href="/pubmed/12140561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Distinct+molecular+mechanism+for+initiating+TRAF6+signalling&amp;author=H+Ye&amp;volume=418&amp;publication_year=2002&amp;pages=443-7&amp;pmid=12140561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Darnay BG, Ni J, Moore PA, Aggarwal BB. Activation of NF-kappaB by RANK requires tumor necrosis factor receptor-associated factor (TRAF) 6 and NF-kappaB-inducing kinase. Identification of a novel TRAF6 interaction motif. <span><span class="ref-journal">J Biol Chem. </span>1999;<span class="ref-vol">274</span>:7724–31.</span> [<a href="/pubmed/10075662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Activation+of+NF-kappaB+by+RANK+requires+tumor+necrosis+factor+receptor-associated+factor+(TRAF)+6+and+NF-kappaB-inducing+kinase.+Identification+of+a+novel+TRAF6+interaction+motif&amp;author=BG+Darnay&amp;author=J+Ni&amp;author=PA+Moore&amp;author=BB+Aggarwal&amp;volume=274&amp;publication_year=1999&amp;pages=7724-31&amp;pmid=10075662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Gelfand JM, et al.  Epidemiology of psoriatic arthritis in the population of the United States. <span><span class="ref-journal">J Am Acad Dermatol. </span>2005;<span class="ref-vol">53</span>:573.</span> [<a href="/pubmed/16198775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Dermatol&amp;title=Epidemiology+of+psoriatic+arthritis+in+the+population+of+the+United+States&amp;author=JM+Gelfand&amp;volume=53&amp;publication_year=2005&amp;pages=573&amp;pmid=16198775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Zhou X, et al.  Novel mechanisms of T-cell and dendritic cell activation revealed by profiling of psoriasis on the 63,100-element oligonucleotide array. <span><span class="ref-journal">Physiol Genomics. </span>2003;<span class="ref-vol">13</span>:69–78.</span> [<a href="/pubmed/12644634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Novel+mechanisms+of+T-cell+and+dendritic+cell+activation+revealed+by+profiling+of+psoriasis+on+the+63,100-element+oligonucleotide+array&amp;author=X+Zhou&amp;volume=13&amp;publication_year=2003&amp;pages=69-78&amp;pmid=12644634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Qian Y, et al.  The adaptor Act1 is required for interleukin 17-dependent signaling associated with autoimmune and inflammatory disease. <span><span class="ref-journal">Nat Immunol. </span>2007;<span class="ref-vol">8</span>:247–56.</span> [<a href="/pubmed/17277779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=The+adaptor+Act1+is+required+for+interleukin+17-dependent+signaling+associated+with+autoimmune+and+inflammatory+disease&amp;author=Y+Qian&amp;volume=8&amp;publication_year=2007&amp;pages=247-56&amp;pmid=17277779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Qian Y, et al.  Act1, a negative regulator in CD40- and BAFF-mediated B cell survival. <span><span class="ref-journal">Immunity. </span>2004;<span class="ref-vol">21</span>:575–87.</span> [<a href="/pubmed/15485634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Act1,+a+negative+regulator+in+CD40-+and+BAFF-mediated+B+cell+survival&amp;author=Y+Qian&amp;volume=21&amp;publication_year=2004&amp;pages=575-87&amp;pmid=15485634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Lowes MA, et al.  Psoriasis vulgaris lesions contain discrete populations of Th1 and Th17 T cells. <span><span class="ref-journal">J Invest Dermatol. </span>2008;<span class="ref-vol">128</span>:1207–11.</span> [<a href="/pubmed/18200064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=Psoriasis+vulgaris+lesions+contain+discrete+populations+of+Th1+and+Th17+T+cells&amp;author=MA+Lowes&amp;volume=128&amp;publication_year=2008&amp;pages=1207-11&amp;pmid=18200064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Kryczek I, et al.  Induction of IL-17+ T cell trafficking and development by IFN-gamma: mechanism and pathological relevance in psoriasis. <span><span class="ref-journal">J Immunol. </span>2008;<span class="ref-vol">181</span>:4733–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2677162/">PMC free article</a>]</span> [<a href="/pubmed/18802076" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=Induction+of+IL-17++T+cell+trafficking+and+development+by+IFN-gamma:+mechanism+and+pathological+relevance+in+psoriasis&amp;author=I+Kryczek&amp;volume=181&amp;publication_year=2008&amp;pages=4733-41&amp;pmid=18802076&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Zaba LC, et al.  Effective treatment of psoriasis with etanercept is linked to suppression of IL-17 signaling, not immediate response TNF genes. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2009;<span class="ref-vol">124</span>:1022–10. e1–395.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852188/">PMC free article</a>]</span> [<a href="/pubmed/19895991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Effective+treatment+of+psoriasis+with+etanercept+is+linked+to+suppression+of+IL-17+signaling,+not+immediate+response+TNF+genes&amp;author=LC+Zaba&amp;volume=124&amp;publication_year=2009&amp;pages=1022-10&amp;pmid=19895991&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Hunter CA. Act1-ivating IL-17 inflammation. <span><span class="ref-journal">Nat Immunol. </span>2007;<span class="ref-vol">8</span>:232–4.</span> [<a href="/pubmed/17304231" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Immunol&amp;title=Act1-ivating+IL-17+inflammation&amp;author=CA+Hunter&amp;volume=8&amp;publication_year=2007&amp;pages=232-4&amp;pmid=17304231&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Moll JM, Wright V. Psoriatic arthritis. <span><span class="ref-journal">Semin Arthritis Rheum. </span>1973;<span class="ref-vol">3</span>:55–78.</span> [<a href="/pubmed/4581554" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin+Arthritis+Rheum&amp;title=Psoriatic+arthritis&amp;author=JM+Moll&amp;author=V+Wright&amp;volume=3&amp;publication_year=1973&amp;pages=55-78&amp;pmid=4581554&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Taylor W, et al.  Classification criteria for psoriatic arthritis: development of new criteria from a large international study. <span><span class="ref-journal">Arthritis Rheum. </span>2006;<span class="ref-vol">54</span>:2665–73.</span> [<a href="/pubmed/16871531" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Classification+criteria+for+psoriatic+arthritis:+development+of+new+criteria+from+a+large+international+study&amp;author=W+Taylor&amp;volume=54&amp;publication_year=2006&amp;pages=2665-73&amp;pmid=16871531&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Krawczak M, et al.  PopGen: population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. <span><span class="ref-journal">Community Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> [<a href="/pubmed/16490960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Community+Genet&amp;title=PopGen:+population-based+recruitment+of+patients+and+controls+for+the+analysis+of+complex+genotype-phenotype+relationships&amp;author=M+Krawczak&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;pmid=16490960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Wichmann HE, Gieger C, Illig T. KORA-gen--resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67 </span>(Suppl 1):S26–30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen--resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=HE+Wichmann&amp;author=C+Gieger&amp;author=T+Illig&amp;volume=67+&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S26-30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu Rev Genomics Hum Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Genomics+Hum+Genet&amp;title=Genotype+imputation&amp;author=Y+Li&amp;author=C+Willer&amp;author=S+Sanna&amp;author=G+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">The International HapMap Consortium. A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–61.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;volume=449&amp;publication_year=2007&amp;pages=851-61&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Dupont WD, Plummer WD. PS power and sample size program available for free on the Internet. <span><span class="ref-journal">Controlled Clin Trials. </span>1997;<span class="ref-vol">18</span></span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Controlled+Clin+Trials&amp;title=PS+power+and+sample+size+program+available+for+free+on+the+Internet&amp;author=WD+Dupont&amp;author=WD+Plummer&amp;volume=18&amp;publication_year=1997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Stoll SW, Johnson JL, Li Y, Rittie L, Elder JT.  <span class="ref-journal">J Invest Dermatol.</span> Amphiregulin Carboxy-Terminal Domain Is Required for Autocrine Keratinocyte Growth. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3072808/">PMC free article</a>]</span> [<a href="/pubmed/20428186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=J+Invest+Dermatol&amp;author=SW+Stoll&amp;author=JL+Johnson&amp;author=Y+Li&amp;author=L+Rittie&amp;author=JT+Elder&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Stoll SW, et al.  Metalloproteinase-mediated, context-dependent function of amphiregulin and HB-EGF in human keratinocytes and skin. <span><span class="ref-journal">J Invest Dermatol. </span><span class="ref-vol">130</span>:295–304.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2795126/">PMC free article</a>]</span> [<a href="/pubmed/19609315" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Invest+Dermatol&amp;title=Metalloproteinase-mediated,+context-dependent+function+of+amphiregulin+and+HB-EGF+in+human+keratinocytes+and+skin&amp;author=SW+Stoll&amp;volume=130&amp;pages=295-304&amp;pmid=19609315&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3136364&amp;issue-id=198491&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140623248703072title">Abstract</a></li><li><a href="#S5title">Online Methods</a></li><li><a href="#S14title">Supplementary Material</a></li><li><a href="#S15title">Acknowledgments</a></li><li><a href="#idm140623261247872title">Footnotes</a></li><li><a href="#idm140623250018960title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3136364/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3136364/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3136364/pdf/nihms302957.pdf">PDF (829K)</a></li> | <li><a href="#" data-citationid="PMC3136364" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3136364%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3136364%2F&amp;text=Genome-wide%20association%20study%20identifies%20a%20psoriasis%20susceptibility%20locus%20at%20TRAF3IP2" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3136364%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="20953188" data-aiid="3136364" data-aid="3136364" data-iid="198491" data-domainid="319" data-domain="nihpa" data-accid="PMC3136364" data-md5="605523f08fbd91c65675887d17d0d8e2"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:05:08-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D0BDBDD517DF10000000003D10298&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3136364%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3136364/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>