<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3257356/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Meta-analysis of New Genome-wide Association Studies of Colorectal Cancer Risk</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257356/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Genetics"><meta name="citation_title" content="Meta-analysis of New Genome-wide Association Studies of Colorectal Cancer Risk"><meta name="citation_authors" content="Ulrike Peters, Carolyn M. Hutter, Li Hsu, Fredrick R. Schumacher, David V. Conti, Christopher S. Carlson, Christopher K. Edlund, Robert W. Haile, Steven Gallinger, Brent W. Zanke, Mathieu Lemire, Jagadish Rangrej, Raakhee Vijayaraghavan, Andrew T. Chan, Aditi Hazra, David J. Hunter, Jing Ma, Charles S. Fuchs, Edward L. Giovannucci, Peter Kraft, Yan Liu, Lin Chen, Shuo Jiao, Karen W. Makar, Darin Taverna, Stephen B. Gruber, Gad Rennert, Victor Moreno, Cornelia M. Ulrich, Michael O. Woods, Roger C. Green, Patrick S. Parfrey, Ross L. Prentice, Charles Kooperberg, Rebecca D. Jackson, Andrea Z. LaCroix, Bette J. Caan, Richard B. Hayes, Sonja I. Berndt, Stephen J. Chanock, Robert E. Schoen, Jenny Chang-Claude, Michael Hoffmeister, Hermann Brenner, Bernd Frank, Stéphane Bézieau, Sébastien Küry, Martha L. Slattery, John L. Hopper, Mark A. Jenkins, Loic Le Marchand, Noralane M. Lindor, Polly A. Newcomb, Daniela Seminara, Thomas J. Hudson, David J. Duggan, John D. Potter, Graham Casey"><meta name="citation_date" content="February 2012"><meta name="citation_issue" content="2"><meta name="citation_volume" content="131"><meta name="citation_firstpage" content="217"><meta name="citation_doi" content="10.1007/s00439-011-1055-0"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3257356/?report=abstract"><meta name="citation_pmid" content="21761138"><meta name="DC.Title" content="Meta-analysis of New Genome-wide Association Studies of Colorectal Cancer Risk"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Ulrike Peters"><meta name="DC.Contributor" content="Carolyn M. Hutter"><meta name="DC.Contributor" content="Li Hsu"><meta name="DC.Contributor" content="Fredrick R. Schumacher"><meta name="DC.Contributor" content="David V. Conti"><meta name="DC.Contributor" content="Christopher S. Carlson"><meta name="DC.Contributor" content="Christopher K. Edlund"><meta name="DC.Contributor" content="Robert W. Haile"><meta name="DC.Contributor" content="Steven Gallinger"><meta name="DC.Contributor" content="Brent W. Zanke"><meta name="DC.Contributor" content="Mathieu Lemire"><meta name="DC.Contributor" content="Jagadish Rangrej"><meta name="DC.Contributor" content="Raakhee Vijayaraghavan"><meta name="DC.Contributor" content="Andrew T. Chan"><meta name="DC.Contributor" content="Aditi Hazra"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Jing Ma"><meta name="DC.Contributor" content="Charles S. Fuchs"><meta name="DC.Contributor" content="Edward L. Giovannucci"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Yan Liu"><meta name="DC.Contributor" content="Lin Chen"><meta name="DC.Contributor" content="Shuo Jiao"><meta name="DC.Contributor" content="Karen W. Makar"><meta name="DC.Contributor" content="Darin Taverna"><meta name="DC.Contributor" content="Stephen B. Gruber"><meta name="DC.Contributor" content="Gad Rennert"><meta name="DC.Contributor" content="Victor Moreno"><meta name="DC.Contributor" content="Cornelia M. Ulrich"><meta name="DC.Contributor" content="Michael O. Woods"><meta name="DC.Contributor" content="Roger C. Green"><meta name="DC.Contributor" content="Patrick S. Parfrey"><meta name="DC.Contributor" content="Ross L. Prentice"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Rebecca D. Jackson"><meta name="DC.Contributor" content="Andrea Z. LaCroix"><meta name="DC.Contributor" content="Bette J. Caan"><meta name="DC.Contributor" content="Richard B. Hayes"><meta name="DC.Contributor" content="Sonja I. Berndt"><meta name="DC.Contributor" content="Stephen J. Chanock"><meta name="DC.Contributor" content="Robert E. Schoen"><meta name="DC.Contributor" content="Jenny Chang-Claude"><meta name="DC.Contributor" content="Michael Hoffmeister"><meta name="DC.Contributor" content="Hermann Brenner"><meta name="DC.Contributor" content="Bernd Frank"><meta name="DC.Contributor" content="Stéphane Bézieau"><meta name="DC.Contributor" content="Sébastien Küry"><meta name="DC.Contributor" content="Martha L. Slattery"><meta name="DC.Contributor" content="John L. Hopper"><meta name="DC.Contributor" content="Mark A. Jenkins"><meta name="DC.Contributor" content="Loic Le Marchand"><meta name="DC.Contributor" content="Noralane M. Lindor"><meta name="DC.Contributor" content="Polly A. Newcomb"><meta name="DC.Contributor" content="Daniela Seminara"><meta name="DC.Contributor" content="Thomas J. Hudson"><meta name="DC.Contributor" content="David J. Duggan"><meta name="DC.Contributor" content="John D. Potter"><meta name="DC.Contributor" content="Graham Casey"><meta name="DC.Date" content="2012 Feb"><meta name="DC.Identifier" content="10.1007/s00439-011-1055-0"><meta name="DC.Language" content="en"><meta property="og:title" content="Meta-analysis of New Genome-wide Association Studies of Colorectal Cancer Risk"><meta property="og:type" content="article"><meta property="og:description" content="Colorectal cancer is the second leading cause of cancer death in developed countries. Genome-wide association studies (GWAS) have successfully identified novel susceptibility loci for colorectal cancer. To follow-up on these findings, and try to identify ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3257356/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3257356/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3257356/pdf/nihms324653.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6573DD52E1210000000001A60110.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3257356%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3257356</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Genet</a></span>. Author manuscript; available in PMC 2013 Feb 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21761138" target="pmc_ext" ref="reftype=publisher&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hum Genet. 2012 Feb; 131(2): 217–234. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Jul 15. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00439-011-1055-0" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00439-011-1055-0</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3257356</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS324653</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21761138">21761138</a></div></div></div></div><h1 class="content-title">Meta-analysis of New Genome-wide Association Studies of Colorectal Cancer Risk</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441689744" co-class="co-affbox">Ulrike Peters</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Hutter%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093437061280" co-class="co-affbox">Carolyn M. Hutter</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Hsu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093442220512" co-class="co-affbox">Li Hsu</a>,<sup>3</sup> <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093442218288" co-class="co-affbox">Fredrick R. Schumacher</a>,<sup>4</sup> <a href="/pubmed/?term=Conti%20DV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093440946272" co-class="co-affbox">David V. Conti</a>,<sup>4</sup> <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093440944048" co-class="co-affbox">Christopher S. Carlson</a>,<sup>1</sup> <a href="/pubmed/?term=Edlund%20CK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093427958656" co-class="co-affbox">Christopher K. Edlund</a>,<sup>5</sup> <a href="/pubmed/?term=Haile%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093427956432" co-class="co-affbox">Robert W. Haile</a>,<sup>4</sup> <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093510375632" co-class="co-affbox">Steven Gallinger</a>,<sup>6</sup> <a href="/pubmed/?term=Zanke%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434555856" co-class="co-affbox">Brent W. Zanke</a>,<sup>7</sup> <a href="/pubmed/?term=Lemire%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434553632" co-class="co-affbox">Mathieu Lemire</a>,<sup>8</sup> <a href="/pubmed/?term=Rangrej%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434584832" co-class="co-affbox">Jagadish Rangrej</a>,<sup>8</sup> <a href="/pubmed/?term=Vijayaraghavan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434582608" co-class="co-affbox">Raakhee Vijayaraghavan</a>,<sup>9</sup> <a href="/pubmed/?term=Chan%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435608464" co-class="co-affbox">Andrew T. Chan</a>,<sup>10,</sup><sup>11</sup> <a href="/pubmed/?term=Hazra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093433053712" co-class="co-affbox">Aditi Hazra</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093433050752" co-class="co-affbox">David J. Hunter</a>,<sup>12</sup> <a href="/pubmed/?term=Ma%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093433589872" co-class="co-affbox">Jing Ma</a>,<sup>11</sup> <a href="/pubmed/?term=Fuchs%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093474570448" co-class="co-affbox">Charles S. Fuchs</a>,<sup>11,</sup><sup>13</sup> <a href="/pubmed/?term=Giovannucci%20EL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093474567488" co-class="co-affbox">Edward L. Giovannucci</a>,<sup>11,</sup><sup>14</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093432633536" co-class="co-affbox">Peter Kraft</a>,<sup>12</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441882512" co-class="co-affbox">Yan Liu</a>,<sup>15</sup> <a href="/pubmed/?term=Chen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441880288" co-class="co-affbox">Lin Chen</a>,<sup>16</sup> <a href="/pubmed/?term=Jiao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093511402848" co-class="co-affbox">Shuo Jiao</a>,<sup>1</sup> <a href="/pubmed/?term=Makar%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093511400624" co-class="co-affbox">Karen W. Makar</a>,<sup>1</sup> <a href="/pubmed/?term=Taverna%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435015712" co-class="co-affbox">Darin Taverna</a>,<sup>9</sup> <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435867552" co-class="co-affbox">Stephen B. Gruber</a>,<sup>17</sup> <a href="/pubmed/?term=Rennert%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435865440" co-class="co-affbox">Gad Rennert</a>,<sup>18</sup> <a href="/pubmed/?term=Moreno%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093431713920" co-class="co-affbox">Victor Moreno</a>,<sup>19</sup> <a href="/pubmed/?term=Ulrich%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093431711696" co-class="co-affbox">Cornelia M. Ulrich</a>,<sup>1,</sup><sup>2,</sup><sup>20</sup> <a href="/pubmed/?term=Woods%20MO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093436844272" co-class="co-affbox">Michael O. Woods</a>,<sup>21</sup> <a href="/pubmed/?term=Green%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441202032" co-class="co-affbox">Roger C. Green</a>,<sup>21</sup> <a href="/pubmed/?term=Parfrey%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441199808" co-class="co-affbox">Patrick S. Parfrey</a>,<sup>22</sup> <a href="/pubmed/?term=Prentice%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093513935792" co-class="co-affbox">Ross L. Prentice</a>,<sup>23</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438814752" co-class="co-affbox">Charles Kooperberg</a>,<sup>23</sup> <a href="/pubmed/?term=Jackson%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438812528" co-class="co-affbox">Rebecca D. Jackson</a>,<sup>24</sup> <a href="/pubmed/?term=LaCroix%20AZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093433340256" co-class="co-affbox">Andrea Z. LaCroix</a>,<sup>1</sup> <a href="/pubmed/?term=Caan%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093433338032" co-class="co-affbox">Bette J. Caan</a>,<sup>25</sup> <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093436362480" co-class="co-affbox">Richard B. Hayes</a>,<sup>26</sup> <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093436360256" co-class="co-affbox">Sonja I. Berndt</a>,<sup>27</sup> <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438973840" co-class="co-affbox">Stephen J. Chanock</a>,<sup>27</sup> <a href="/pubmed/?term=Schoen%20RE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438971616" co-class="co-affbox">Robert E. Schoen</a>,<sup>28</sup> <a href="/pubmed/?term=Chang-Claude%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435412976" co-class="co-affbox">Jenny Chang-Claude</a>,<sup>29</sup> <a href="/pubmed/?term=Hoffmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438224768" co-class="co-affbox">Michael Hoffmeister</a>,<sup>30</sup> <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093438222544" co-class="co-affbox">Hermann Brenner</a>,<sup>30</sup> <a href="/pubmed/?term=Frank%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435830384" co-class="co-affbox">Bernd Frank</a>,<sup>30</sup> <a href="/pubmed/?term=B%26%23x000e9%3Bzieau%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435827936" co-class="co-affbox">Stéphane Bézieau</a>,<sup>31</sup> <a href="/pubmed/?term=K%26%23x000fc%3Bry%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093512847968" co-class="co-affbox">Sébastien Küry</a>,<sup>31</sup> <a href="/pubmed/?term=Slattery%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093512871040" co-class="co-affbox">Martha L. Slattery</a>,<sup>32</sup> <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093512868528" co-class="co-affbox">John L. Hopper</a>,<sup>33</sup> <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093432983744" co-class="co-affbox">Mark A. Jenkins</a>,<sup>33</sup> <a href="/pubmed/?term=Le%20Marchand%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093432981296" co-class="co-affbox">Loic Le Marchand</a>,<sup>34</sup> <a href="/pubmed/?term=Lindor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434495936" co-class="co-affbox">Noralane M. Lindor</a>,<sup>35</sup> <a href="/pubmed/?term=Newcomb%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441543504" co-class="co-affbox">Polly A. Newcomb</a>,<sup>1</sup> <a href="/pubmed/?term=Seminara%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093441541056" co-class="co-affbox">Daniela Seminara</a>,<sup>36</sup> <a href="/pubmed/?term=Hudson%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093434517216" co-class="co-affbox">Thomas J. Hudson</a>,<sup>8,</sup><sup>37</sup> <a href="/pubmed/?term=Duggan%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435602848" co-class="co-affbox">David J. Duggan</a>,<sup>9</sup> <a href="/pubmed/?term=Potter%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093435600400" co-class="co-affbox">John D. Potter</a>,<sup>2,</sup><sup>23</sup> and  <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138" class="affpopup" co-rid="_co_idm140093436926304" co-class="co-affbox">Graham Casey</a><sup>4</sup>, on behalf of CCFR<sup>38</sup>, GECCO<sup>39</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140093441689744"><h3 class="no_margin">Ulrike Peters</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><p><sup>2</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Peters%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Ulrike Peters</a></div></div><div id="_co_idm140093437061280"><h3 class="no_margin">Carolyn M. Hutter</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Hutter%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Carolyn M. Hutter</a></div></div><div id="_co_idm140093442220512"><h3 class="no_margin">Li Hsu</h3><p><sup>3</sup>Biostatistics and Biomathematics, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Li Hsu</a></div></div><div id="_co_idm140093442218288"><h3 class="no_margin">Fredrick R. Schumacher</h3><p><sup>4</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, USA</p><div>Find articles by <a href="/pubmed/?term=Schumacher%20FR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Fredrick R. Schumacher</a></div></div><div id="_co_idm140093440946272"><h3 class="no_margin">David V. Conti</h3><p><sup>4</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, USA</p><div>Find articles by <a href="/pubmed/?term=Conti%20DV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">David V. Conti</a></div></div><div id="_co_idm140093440944048"><h3 class="no_margin">Christopher S. Carlson</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Christopher S. Carlson</a></div></div><div id="_co_idm140093427958656"><h3 class="no_margin">Christopher K. Edlund</h3><p><sup>5</sup>Keck School of Medicine, University of Southern California, Los Angeles, USA</p><div>Find articles by <a href="/pubmed/?term=Edlund%20CK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Christopher K. Edlund</a></div></div><div id="_co_idm140093427956432"><h3 class="no_margin">Robert W. Haile</h3><p><sup>4</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, USA</p><div>Find articles by <a href="/pubmed/?term=Haile%20RW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Robert W. Haile</a></div></div><div id="_co_idm140093510375632"><h3 class="no_margin">Steven Gallinger</h3><p><sup>6</sup>Department of Surgery, University Health Network, Toronto General Hospital, Toronto, Canada</p><div>Find articles by <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Steven Gallinger</a></div></div><div id="_co_idm140093434555856"><h3 class="no_margin">Brent W. Zanke</h3><p><sup>7</sup>Clinical Epidemiology Program, Ottawa Hospital Research Institute, Ottawa, Canada</p><div>Find articles by <a href="/pubmed/?term=Zanke%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Brent W. Zanke</a></div></div><div id="_co_idm140093434553632"><h3 class="no_margin">Mathieu Lemire</h3><p><sup>8</sup>Ontario Institute for Cancer Research, Toronto, Canada</p><div>Find articles by <a href="/pubmed/?term=Lemire%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Mathieu Lemire</a></div></div><div id="_co_idm140093434584832"><h3 class="no_margin">Jagadish Rangrej</h3><p><sup>8</sup>Ontario Institute for Cancer Research, Toronto, Canada</p><div>Find articles by <a href="/pubmed/?term=Rangrej%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Jagadish Rangrej</a></div></div><div id="_co_idm140093434582608"><h3 class="no_margin">Raakhee Vijayaraghavan</h3><p><sup>9</sup>Translational Genomics Research Institute, Phoenix, USA</p><div>Find articles by <a href="/pubmed/?term=Vijayaraghavan%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Raakhee Vijayaraghavan</a></div></div><div id="_co_idm140093435608464"><h3 class="no_margin">Andrew T. Chan</h3><p><sup>10</sup>Division of Gastroenterology, Massachusetts General Hospital and Harvard Medical School, Boston, USA</p><p><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Chan%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Andrew T. Chan</a></div></div><div id="_co_idm140093433053712"><h3 class="no_margin">Aditi Hazra</h3><p><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</p><p><sup>12</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Hazra%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Aditi Hazra</a></div></div><div id="_co_idm140093433050752"><h3 class="no_margin">David J. Hunter</h3><p><sup>12</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">David J. Hunter</a></div></div><div id="_co_idm140093433589872"><h3 class="no_margin">Jing Ma</h3><p><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Ma%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Jing Ma</a></div></div><div id="_co_idm140093474570448"><h3 class="no_margin">Charles S. Fuchs</h3><p><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</p><p><sup>13</sup>Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Fuchs%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Charles S. Fuchs</a></div></div><div id="_co_idm140093474567488"><h3 class="no_margin">Edward L. Giovannucci</h3><p><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</p><p><sup>14</sup>Departments of Epidemiology and Nutrition, Harvard School of Public Health, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Giovannucci%20EL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Edward L. Giovannucci</a></div></div><div id="_co_idm140093432633536"><h3 class="no_margin">Peter Kraft</h3><p><sup>12</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, USA</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Peter Kraft</a></div></div><div id="_co_idm140093441882512"><h3 class="no_margin">Yan Liu</h3><p><sup>15</sup>Quantitative Services, Baylor Health Care System, Dallas, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Yan Liu</a></div></div><div id="_co_idm140093441880288"><h3 class="no_margin">Lin Chen</h3><p><sup>16</sup>Department of Health Studies, University of Chicago, Chicago, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Lin Chen</a></div></div><div id="_co_idm140093511402848"><h3 class="no_margin">Shuo Jiao</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Jiao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Shuo Jiao</a></div></div><div id="_co_idm140093511400624"><h3 class="no_margin">Karen W. Makar</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Makar%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Karen W. Makar</a></div></div><div id="_co_idm140093435015712"><h3 class="no_margin">Darin Taverna</h3><p><sup>9</sup>Translational Genomics Research Institute, Phoenix, USA</p><div>Find articles by <a href="/pubmed/?term=Taverna%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Darin Taverna</a></div></div><div id="_co_idm140093435867552"><h3 class="no_margin">Stephen B. Gruber</h3><p><sup>17</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, USA</p><div>Find articles by <a href="/pubmed/?term=Gruber%20SB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Stephen B. Gruber</a></div></div><div id="_co_idm140093435865440"><h3 class="no_margin">Gad Rennert</h3><p><sup>18</sup>Department of Community Medicine and Epidemiology, CarmelMedical Center and Technion Faculty of Medicine, Haifa, Israel</p><div>Find articles by <a href="/pubmed/?term=Rennert%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Gad Rennert</a></div></div><div id="_co_idm140093431713920"><h3 class="no_margin">Victor Moreno</h3><p><sup>19</sup>Biostatistics and Bioinformatics Unit, Catalan Institute of Oncology-IDIBELL, Barcelona, Spain</p><div>Find articles by <a href="/pubmed/?term=Moreno%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Victor Moreno</a></div></div><div id="_co_idm140093431711696"><h3 class="no_margin">Cornelia M. Ulrich</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><p><sup>2</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, USA</p><p><sup>20</sup>Division of Preventive Oncology, German Cancer Research Center, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Ulrich%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Cornelia M. Ulrich</a></div></div><div id="_co_idm140093436844272"><h3 class="no_margin">Michael O. Woods</h3><p><sup>21</sup>Discipline of Genetics, Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Canada</p><div>Find articles by <a href="/pubmed/?term=Woods%20MO%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Michael O. Woods</a></div></div><div id="_co_idm140093441202032"><h3 class="no_margin">Roger C. Green</h3><p><sup>21</sup>Discipline of Genetics, Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Canada</p><div>Find articles by <a href="/pubmed/?term=Green%20RC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Roger C. Green</a></div></div><div id="_co_idm140093441199808"><h3 class="no_margin">Patrick S. Parfrey</h3><p><sup>22</sup>Discipline of Medicine, Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Canada</p><div>Find articles by <a href="/pubmed/?term=Parfrey%20PS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Patrick S. Parfrey</a></div></div><div id="_co_idm140093513935792"><h3 class="no_margin">Ross L. Prentice</h3><p><sup>23</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Prentice%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Ross L. Prentice</a></div></div><div id="_co_idm140093438814752"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>23</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Charles Kooperberg</a></div></div><div id="_co_idm140093438812528"><h3 class="no_margin">Rebecca D. Jackson</h3><p><sup>24</sup>Division of Endocrinology, Diabetes and Metabolism, Ohio State University, Columbus, USA</p><div>Find articles by <a href="/pubmed/?term=Jackson%20RD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Rebecca D. Jackson</a></div></div><div id="_co_idm140093433340256"><h3 class="no_margin">Andrea Z. LaCroix</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=LaCroix%20AZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Andrea Z. LaCroix</a></div></div><div id="_co_idm140093433338032"><h3 class="no_margin">Bette J. Caan</h3><p><sup>25</sup>Division of Research, Kaiser Permanente Medical Care Program, Oakland, USA</p><div>Find articles by <a href="/pubmed/?term=Caan%20BJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Bette J. Caan</a></div></div><div id="_co_idm140093436362480"><h3 class="no_margin">Richard B. Hayes</h3><p><sup>26</sup>Division of Epidemiology, Department of Environmental Medicine, New YorkUniversity School of Medicine, New York City, USA</p><div>Find articles by <a href="/pubmed/?term=Hayes%20RB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Richard B. Hayes</a></div></div><div id="_co_idm140093436360256"><h3 class="no_margin">Sonja I. Berndt</h3><p><sup>27</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, USA</p><div>Find articles by <a href="/pubmed/?term=Berndt%20SI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Sonja I. Berndt</a></div></div><div id="_co_idm140093438973840"><h3 class="no_margin">Stephen J. Chanock</h3><p><sup>27</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, USA</p><div>Find articles by <a href="/pubmed/?term=Chanock%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Stephen J. Chanock</a></div></div><div id="_co_idm140093438971616"><h3 class="no_margin">Robert E. Schoen</h3><p><sup>28</sup>Department of Epidemiology, University of Pittsburgh Medical Center, Pittsburgh, USA</p><div>Find articles by <a href="/pubmed/?term=Schoen%20RE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Robert E. Schoen</a></div></div><div id="_co_idm140093435412976"><h3 class="no_margin">Jenny Chang-Claude</h3><p><sup>29</sup>Division of Cancer Epidemiology, German Cancer Research Center, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Chang-Claude%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Jenny Chang-Claude</a></div></div><div id="_co_idm140093438224768"><h3 class="no_margin">Michael Hoffmeister</h3><p><sup>30</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Hoffmeister%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Michael Hoffmeister</a></div></div><div id="_co_idm140093438222544"><h3 class="no_margin">Hermann Brenner</h3><p><sup>30</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Brenner%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Hermann Brenner</a></div></div><div id="_co_idm140093435830384"><h3 class="no_margin">Bernd Frank</h3><p><sup>30</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Frank%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Bernd Frank</a></div></div><div id="_co_idm140093435827936"><h3 class="no_margin">Stéphane Bézieau</h3><p><sup>31</sup>Centre Hospitalier Universitaire (CHU) de Nantes, Pôle de Biologie, Service de Génétique Médicale, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=B%26%23x000e9%3Bzieau%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Stéphane Bézieau</a></div></div><div id="_co_idm140093512847968"><h3 class="no_margin">Sébastien Küry</h3><p><sup>31</sup>Centre Hospitalier Universitaire (CHU) de Nantes, Pôle de Biologie, Service de Génétique Médicale, Nantes, France</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000fc%3Bry%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Sébastien Küry</a></div></div><div id="_co_idm140093512871040"><h3 class="no_margin">Martha L. Slattery</h3><p><sup>32</sup>Department of Internal Medicine, University of Utah Health Sciences Center, Salt Lake City, USA</p><div>Find articles by <a href="/pubmed/?term=Slattery%20ML%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Martha L. Slattery</a></div></div><div id="_co_idm140093512868528"><h3 class="no_margin">John L. Hopper</h3><p><sup>33</sup>Centre for Molecular, Environmental, Genetic, and Analytical Epidemiology, University of Melbourne, Melbourne, Australia</p><div>Find articles by <a href="/pubmed/?term=Hopper%20JL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">John L. Hopper</a></div></div><div id="_co_idm140093432983744"><h3 class="no_margin">Mark A. Jenkins</h3><p><sup>33</sup>Centre for Molecular, Environmental, Genetic, and Analytical Epidemiology, University of Melbourne, Melbourne, Australia</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Mark A. Jenkins</a></div></div><div id="_co_idm140093432981296"><h3 class="no_margin">Loic Le Marchand</h3><p><sup>34</sup>Epidemiology Program, Cancer Research Center of Hawai’i, University of Hawai’i at Manoa, Honolulu, USA</p><div>Find articles by <a href="/pubmed/?term=Le%20Marchand%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Loic Le Marchand</a></div></div><div id="_co_idm140093434495936"><h3 class="no_margin">Noralane M. Lindor</h3><p><sup>35</sup>Department of Medical Genetics, Mayo Clinic, Rochester, USA</p><div>Find articles by <a href="/pubmed/?term=Lindor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Noralane M. Lindor</a></div></div><div id="_co_idm140093441543504"><h3 class="no_margin">Polly A. Newcomb</h3><p><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Newcomb%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Polly A. Newcomb</a></div></div><div id="_co_idm140093441541056"><h3 class="no_margin">Daniela Seminara</h3><p><sup>36</sup>Division of Cancer Control and Population Sciences, National Cancer Institute, Bethesda, USA</p><div>Find articles by <a href="/pubmed/?term=Seminara%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Daniela Seminara</a></div></div><div id="_co_idm140093434517216"><h3 class="no_margin">Thomas J. Hudson</h3><p><sup>8</sup>Ontario Institute for Cancer Research, Toronto, Canada</p><p><sup>37</sup>Departments of Medical Biophysics and Molecular Genetics, University of Toronto, Toronto, Canada</p><div>Find articles by <a href="/pubmed/?term=Hudson%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Thomas J. Hudson</a></div></div><div id="_co_idm140093435602848"><h3 class="no_margin">David J. Duggan</h3><p><sup>9</sup>Translational Genomics Research Institute, Phoenix, USA</p><div>Find articles by <a href="/pubmed/?term=Duggan%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">David J. Duggan</a></div></div><div id="_co_idm140093435600400"><h3 class="no_margin">John D. Potter</h3><p><sup>2</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, USA</p><p><sup>23</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, USA</p><div>Find articles by <a href="/pubmed/?term=Potter%20JD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">John D. Potter</a></div></div><div id="_co_idm140093436926304"><h3 class="no_margin">Graham Casey</h3><p><sup>4</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, USA</p><div>Find articles by <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21761138">Graham Casey</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140093512351376_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140093512351376_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140093512351376_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Epidemiology, School of Public Health, University of Washington, Seattle, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Biostatistics and Biomathematics, Fred Hutchinson Cancer Research Center, Seattle, USA</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Preventive Medicine, Keck School of Medicine, University of Southern California, Los Angeles, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Keck School of Medicine, University of Southern California, Los Angeles, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Surgery, University Health Network, Toronto General Hospital, Toronto, Canada</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Clinical Epidemiology Program, Ottawa Hospital Research Institute, Ottawa, Canada</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Ontario Institute for Cancer Research, Toronto, Canada</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Translational Genomics Research Institute, Phoenix, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Division of Gastroenterology, Massachusetts General Hospital and Harvard Medical School, Boston, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Channing Laboratory, Brigham and Women’s Hospital and Harvard Medical School, Boston, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Program in Molecular and Genetic Epidemiology, Department of Epidemiology, Harvard School of Public Health, Boston, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Medical Oncology, Dana-Farber Cancer Institute and Harvard Medical School, Boston, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Departments of Epidemiology and Nutrition, Harvard School of Public Health, Boston, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Quantitative Services, Baylor Health Care System, Dallas, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Health Studies, University of Chicago, Chicago, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Internal Medicine, University of Michigan, Ann Arbor, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Community Medicine and Epidemiology, CarmelMedical Center and Technion Faculty of Medicine, Haifa, Israel</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Biostatistics and Bioinformatics Unit, Catalan Institute of Oncology-IDIBELL, Barcelona, Spain</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Division of Preventive Oncology, German Cancer Research Center, Heidelberg, Germany</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Discipline of Genetics, Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Canada</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Discipline of Medicine, Faculty of Medicine, Memorial University of Newfoundland, St. John’s, Canada</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, USA</div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Division of Endocrinology, Diabetes and Metabolism, Ohio State University, Columbus, USA</div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Division of Research, Kaiser Permanente Medical Care Program, Oakland, USA</div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Division of Epidemiology, Department of Environmental Medicine, New YorkUniversity School of Medicine, New York City, USA</div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Division of Cancer Epidemiology and Genetics, National Cancer Institute, National Institutes of Health, Department of Health and Human Services, Bethesda, USA</div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Department of Epidemiology, University of Pittsburgh Medical Center, Pittsburgh, USA</div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Division of Cancer Epidemiology, German Cancer Research Center, Heidelberg, Germany</div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Division of Clinical Epidemiology and Aging Research, German Cancer Research Center, Heidelberg, Germany</div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Centre Hospitalier Universitaire (CHU) de Nantes, Pôle de Biologie, Service de Génétique Médicale, Nantes, France</div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Department of Internal Medicine, University of Utah Health Sciences Center, Salt Lake City, USA</div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Centre for Molecular, Environmental, Genetic, and Analytical Epidemiology, University of Melbourne, Melbourne, Australia</div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Epidemiology Program, Cancer Research Center of Hawai’i, University of Hawai’i at Manoa, Honolulu, USA</div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Department of Medical Genetics, Mayo Clinic, Rochester, USA</div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Division of Cancer Control and Population Sciences, National Cancer Institute, Bethesda, USA</div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>Departments of Medical Biophysics and Molecular Genetics, University of Toronto, Toronto, Canada</div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>CCFR: Colon Cancer Family Registry</div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>GECCO: Genetics and Epidemiology of Colorectal Cancer Consortium</div><div id="FN1">Correspondence should be addressed to both: Ulrike Peters, <a href="mailto:dev@null" data-email="gro.crchf@sretepu" class="oemail">gro.crchf@sretepu</a>, Tel: +1 (206) 667-2450. Graham Casey, <a href="mailto:dev@null" data-email="ude.csu@yesacg" class="oemail">ude.csu@yesacg</a>, Tel: +1 (323) 442-7865</div><div id="FN2"><sup>*</sup>The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140093512351376_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21761138" target="pmc_ext" ref="reftype=publisher&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC3257356/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC3257356/bin/NIHMS324653-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">NIHMS324653-supplement-supplement_1.pdf</a><span style="color:gray"> (610K)</span></div><div class="small guid">GUID:&nbsp;BE155332-10B9-4749-9501-2D6F98CE2561</div></div></dd></dl></div><div id="idm140093441399680" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140093441399680title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Colorectal cancer is the second leading cause of cancer death in developed countries. Genome-wide association studies (GWAS) have successfully identified novel susceptibility loci for colorectal cancer. To follow-up on these findings, and try to identify novel colorectal cancer susceptibility loci, we present results for genome-wide association studies (GWAS) of colorectal cancer (2,906 cases, 3,416 controls) that have not previously published main associations. Specifically, we calculated odds ratios (ORs) and 95% confidence intervals (CIs) using log-additive models for each study. In order to improve our power to detect novel colorectal cancer susceptibility loci, we performed a meta-analysis combining the results across studies. We selected the most statistically significant single nucleotide polymorphisms (SNPs) for replication using 10 independent studies (8,161 cases and 9,101 controls). We again used a meta-analysis to summarize results for the replication studies alone, and for a combined analysis of GWAS and replication studies. We measured 10 SNPs previously identified in colorectal cancer susceptibility loci and found eight to be associated with colorectal cancer (p-value range: 0.02 to 1.8 × 10<sup>−8</sup>). When we excluded studies that have previously published on these SNPs, five SNPs remained significant at p&lt;0.05 in the combined analysis. No novel susceptibility loci were significant in the replication study after adjustment for multiple testing, and none reached genome-wide significance from a combined analysis of GWAS and replication. We observed marginally significant evidence for a second independent SNP in the <em>BMP2</em> region at chromosomal location 20p12 (rs4813802; replication p-value 0.03; combined p-value 7.3 × 10<sup>−5</sup>). In a region on 5p33.15, which includes the coding regions of the <em>TERT-CLPTM1L</em> genes and has been identified in GWAS to be associated with susceptibility to at least seven other cancers, we observed a marginally significant association with rs2853668 (replication p-value 0.03; combined p-value 1.9 × 10<sup>−4</sup>). Our study suggests a complex nature of the contribution of common genetic variants to risk for colorectal cancer.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Colorectal cancer is the second leading cause of cancer death in developed countries, with the lifetime risk estimated to be 5% to 6% (<a href="#R48" rid="R48" class=" bibr popnode">Ries et al., 2007</a>). Linkage studies have identified important rare germline mutations, such as those in the <em>APC</em> gene and DNA mismatch repair genes, leading to severe syndromes, e.g. familial adenomatous polyposis and Lynch syndrome (also called hereditary non-polyposis colorectal cancer) (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467635">de la Chapelle, 2004</a>). However, these high-penetrance mutations explain only a small fraction of the genetic risk. To date, genome-wide association studies (GWAS) have identified fourteen low-penetrance genetic variants that, together, explain approximately 8% of the familial association of this disease (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467576">Broderick et al., 2007</a>;<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314494546">Gruber et al., 2007</a>;<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467646">Houlston et al., 2008</a>;<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_600192535">Houlston et al., 2010</a>;<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467618">Tenesa et al., 2008</a>;<a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467615">Tomlinson et al., 2007</a>;<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467611">Tomlinson et al., 2008</a>;<a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467569">Zanke et al., 2007</a>). Based on a recent method by Chatterjee and Park (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_489033064">Park et al., 2010</a>) that estimates the amount of familial association explained by common genetic variants, we estimate that about 60 to 70 common variants (95% confidence interval: 31–173) would explain approximately 17% (95% confidence interval: 11.6–35.8%) of the familial association in colorectal cancer. Accordingly, we hypothesize that additional common colorectal cancer susceptibility loci exist that yet have to be identified, and that these loci can be identified through a genome-wide analysis of single nucleotide polymorphism (SNP) data.</p><p id="P3" class="p p-last">As has been demonstrated in studies of other common complex diseases, power to detect novel loci is enhanced by performing meta-analysis that combine GWAS results (<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467641">Zeggini and Ioannidis, 2009</a>). Therefore, we conducted a combined analysis of two recently completed scans that have not previously published main associations, followed by a replication study of the most significant findings using ten independent studies (<a href="/pmc/articles/PMC3257356/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Note</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 1</a>) to follow up on the currently established colorectal cancer susceptibility loci and to try to identify additional susceptibility loci. The GWAS meta-analysis included a total of 2,906 cases and 3,416 controls recruited as part of the Colon Cancer Family Registry (CCFR), the Diet, Activity and Lifestyle Study (DALS), the Prostate, Lung, Colorectal and Ovarian Screening Trial (PLCO), and the Women’s Health Initiative (WHI). The replication included a total of 8,161 cases and 9,101 controls from the Nurses’ Health Study (NHS), the Health Professionals Follow-up Study (HPFS), the Physicians’ Health Study (PHS), the Assessment of Risk in Colorectal Tumors In Canada (ARCTIC), additional samples from DALS and CCFR, and case-control studies from Germany, France, Israel, and Newfoundland. Most of these studies are part of the Genetics and Epidemiology Colorectal Cancer Consortium (GECCO; details in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Note</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p22">Studies participating in the genome-wide association study (GWAS) and replication meta-analyses.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093428929280" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Study Name</th><th rowspan="1" colspan="1" valign="bottom" align="center">Abbreviation</th><th rowspan="1" colspan="1" valign="bottom" align="center">Cases</th><th rowspan="1" colspan="1" valign="bottom" align="center">Controls</th><th rowspan="1" colspan="1" valign="bottom" align="center">Total</th></tr></thead><tbody><tr><td colspan="5" rowspan="1" valign="top" align="left"><strong>GWAS</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Colon Cancer Family Registry</td><td rowspan="1" colspan="1" valign="top" align="center">CCFR</td><td rowspan="1" colspan="1" valign="top" align="center">1191</td><td rowspan="1" colspan="1" valign="top" align="center">999</td><td rowspan="1" colspan="1" valign="top" align="center">2208</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Women’s Health Initiative</td><td rowspan="1" colspan="1" valign="top" align="center">WHI</td><td rowspan="1" colspan="1" valign="top" align="center">483</td><td rowspan="1" colspan="1" valign="top" align="center">530</td><td rowspan="1" colspan="1" valign="top" align="center">1013</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Prostate, Lung, Colorectal and Ovarian Cancer Screening Trial</td><td rowspan="1" colspan="1" valign="top" align="center">PLCO</td><td rowspan="1" colspan="1" valign="top" align="center">534</td><td rowspan="1" colspan="1" valign="top" align="center">1168</td><td rowspan="1" colspan="1" valign="top" align="center">1702</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diet, Activity and Lifestyle Survey</td><td rowspan="1" colspan="1" valign="top" align="center">DALS</td><td rowspan="1" colspan="1" valign="top" align="center">698</td><td rowspan="1" colspan="1" valign="top" align="center">719</td><td rowspan="1" colspan="1" valign="top" align="center">1417</td></tr><tr><td colspan="5" rowspan="1" valign="top" align="left"><strong>Replication</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Assessment of Risk in Colorectal Tumors In Canada</td><td rowspan="1" colspan="1" valign="top" align="center">ARCTIC</td><td rowspan="1" colspan="1" valign="top" align="center">769</td><td rowspan="1" colspan="1" valign="top" align="center">665</td><td rowspan="1" colspan="1" valign="top" align="center">1434</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Colon Cancer Family Registry Set II</td><td rowspan="1" colspan="1" valign="top" align="center">CCFR-II</td><td rowspan="1" colspan="1" valign="top" align="center">780</td><td rowspan="1" colspan="1" valign="top" align="center">780</td><td rowspan="1" colspan="1" valign="top" align="center">1560</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Darmkrebs: Chancen der Verhütung durch Screening</td><td rowspan="1" colspan="1" valign="top" align="center">DACHS</td><td rowspan="1" colspan="1" valign="top" align="center">1731</td><td rowspan="1" colspan="1" valign="top" align="center">1742</td><td rowspan="1" colspan="1" valign="top" align="center">3473</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diet, Activity and Lifestyle Survey Set II</td><td rowspan="1" colspan="1" valign="top" align="center">DALS-II</td><td rowspan="1" colspan="1" valign="top" align="center">691</td><td rowspan="1" colspan="1" valign="top" align="center">720</td><td rowspan="1" colspan="1" valign="top" align="center">1411</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">French Case-Control Study</td><td rowspan="1" colspan="1" valign="top" align="center">FRENCH</td><td rowspan="1" colspan="1" valign="top" align="center">954</td><td rowspan="1" colspan="1" valign="top" align="center">1060</td><td rowspan="1" colspan="1" valign="top" align="center">2014</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Health Professionals Follow-up Study</td><td rowspan="1" colspan="1" valign="top" align="center">HPFS</td><td rowspan="1" colspan="1" valign="top" align="center">333</td><td rowspan="1" colspan="1" valign="top" align="center">595</td><td rowspan="1" colspan="1" valign="top" align="center">928</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Molecular Epidemiology of Colorectal Cancer Study</td><td rowspan="1" colspan="1" valign="top" align="center">MECC</td><td rowspan="1" colspan="1" valign="top" align="center">1686</td><td rowspan="1" colspan="1" valign="top" align="center">1779</td><td rowspan="1" colspan="1" valign="top" align="center">3465</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Newfoundland Familial Colon Cancer Registry</td><td rowspan="1" colspan="1" valign="top" align="center">NFCCR</td><td rowspan="1" colspan="1" valign="top" align="center">409</td><td rowspan="1" colspan="1" valign="top" align="center">321</td><td rowspan="1" colspan="1" valign="top" align="center">730</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Nurses’ Health Study</td><td rowspan="1" colspan="1" valign="top" align="center">NHS</td><td rowspan="1" colspan="1" valign="top" align="center">432</td><td rowspan="1" colspan="1" valign="top" align="center">946</td><td rowspan="1" colspan="1" valign="top" align="center">1378</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Physician’s Health Study</td><td rowspan="1" colspan="1" valign="top" align="center">PHS</td><td rowspan="1" colspan="1" valign="top" align="center">376</td><td rowspan="1" colspan="1" valign="top" align="center">493</td><td rowspan="1" colspan="1" valign="top" align="center">869</td></tr></tbody></table></div><div id="largeobj_idm140093428929280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3257356/table/T1/?report=objectonly">Open in a separate window</a></div></div></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Results</h2><p id="P4" class="p p-first">From the fixed-effects meta-analysis of GWAS scans, the inflation factor λ was 1.008, indicating little evidence of residual population substructure, cryptic relatedness, or differential genotyping between cases and controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>). When analyzed separately, λ was similarly low for each scan (range: 1.005 to 1.01).</p><p id="P5">Initially, we attempted to validate the ten established susceptibility SNPs (p-value 5 × 10<sup>−8</sup>) that had been published at the time we selected SNPs for replication (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467584">Broderick et al., 2007</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314494547">Gruber et al., 2007</a>; <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467565">Houlston et al., 2008</a>; <a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467585">Tenesa et al., 2008</a>; <a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467617">Tomlinson et al., 2007</a>; <a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467578">Tomlinson et al., 2008</a>; <a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467597">Zanke et al., 2007</a>). We found nominal evidence for association in the same direction with p&lt;0.05 from combined analyses of GWAS and replication for eight of these ten loci (rs4939827/<em>SMAD7,</em> rs4779584/<em>GREM1</em>, rs16892766/<em>EIF3H</em>, rs3802842/11q23, rs961253/<em>BMP2</em>, rs4444235/<em>BMP4</em>, rs9929218/<em>CDH1</em>, rs6983267/<em>MYC</em>; <a href="/pmc/articles/PMC3257356/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). When we excluded results from studies that had been previously published (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 2</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2</a>), we found evidence for replication at p&lt;0.05 for five out of nine SNPs. This latter analysis did not include rs6983267, since that SNP has already been published on by a majority of the studies.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p23">Risk estimates for colorectal cancer for previously reported genome-wide association studies (GWAS) hits.<sup>a</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093429668432" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="3" colspan="1" valign="bottom" align="left">Chr</th><th rowspan="3" colspan="1" valign="bottom" align="left">Position (bp)</th><th rowspan="3" colspan="1" valign="bottom" align="left">Gene/locus<sup>b</sup></th><th rowspan="3" colspan="1" valign="bottom" align="left">Ref. allele</th><th rowspan="3" colspan="1" valign="bottom" align="left">MAF<sup>c</sup></th><th rowspan="3" colspan="1" valign="bottom" align="left"># Case<sup>d</sup></th><th rowspan="3" colspan="1" valign="bottom" align="left"># Control<sup>d</sup></th><th colspan="2" rowspan="1" valign="bottom" align="center">GWAS</th><th colspan="2" rowspan="1" valign="bottom" align="center">Replication</th><th colspan="2" rowspan="1" valign="bottom" align="center">GWAS + Replication</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">P<sup>e</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">P<sup>e</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">P<sup>e</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4939827</td><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="left">44707461</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SMAD7</em></td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">7675</td><td rowspan="1" colspan="1" valign="top" align="left">8970</td><td rowspan="1" colspan="1" valign="top" align="left">0.86 (0.79–0.93)</td><td rowspan="1" colspan="1" valign="top" align="left">8.7×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.90 (0.85–0.95)</td><td rowspan="1" colspan="1" valign="top" align="left">2.1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.88 (0.85–0.93)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4779584</td><td rowspan="1" colspan="1" valign="top" align="left">15</td><td rowspan="1" colspan="1" valign="top" align="left">30782048</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SCG5, GREM1</em></td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="left">7669</td><td rowspan="1" colspan="1" valign="top" align="left">8795</td><td rowspan="1" colspan="1" valign="top" align="left">1.15 (1.04–1.26)</td><td rowspan="1" colspan="1" valign="top" align="left">5.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.19 (1.11–1.28)</td><td rowspan="1" colspan="1" valign="top" align="left">8.1×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.18 (1.11–1.24)</td><td rowspan="1" colspan="1" valign="top" align="left">1.8×10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16892766</td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">117699864</td><td rowspan="1" colspan="1" valign="top" align="left"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.08</td><td rowspan="1" colspan="1" valign="top" align="left">7686</td><td rowspan="1" colspan="1" valign="top" align="left">8977</td><td rowspan="1" colspan="1" valign="top" align="left">1.41 (1.23–1.62)</td><td rowspan="1" colspan="1" valign="top" align="left">6.8×10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.15 (1.03–1.27)</td><td rowspan="1" colspan="1" valign="top" align="left">9.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.24 (1.14–1.34)</td><td rowspan="1" colspan="1" valign="top" align="left">4.0×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3802842</td><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">110676919</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC120 376</em></td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.28</td><td rowspan="1" colspan="1" valign="top" align="left">7677</td><td rowspan="1" colspan="1" valign="top" align="left">8782</td><td rowspan="1" colspan="1" valign="top" align="left">1.13 (1.04–1.23)</td><td rowspan="1" colspan="1" valign="top" align="left">4.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.14 (1.07–1.22)</td><td rowspan="1" colspan="1" valign="top" align="left">2.7×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.14 (1.08–1.20)</td><td rowspan="1" colspan="1" valign="top" align="left">3.8×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs961253</td><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">6352281</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC643 503, BMP2</em></td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.35</td><td rowspan="1" colspan="1" valign="top" align="left">7683</td><td rowspan="1" colspan="1" valign="top" align="left">8790</td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.01–1.19)</td><td rowspan="1" colspan="1" valign="top" align="left">0.020</td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.04–1.17)</td><td rowspan="1" colspan="1" valign="top" align="left">9.0×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.10 (1.05–1.16)</td><td rowspan="1" colspan="1" valign="top" align="left">5.2×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4444235</td><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">53480669</td><td rowspan="1" colspan="1" valign="top" align="left"><em>BMP4</em></td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0.47</td><td rowspan="1" colspan="1" valign="top" align="left">7678</td><td rowspan="1" colspan="1" valign="top" align="left">8792</td><td rowspan="1" colspan="1" valign="top" align="left">1.07 (0.99–1.15)</td><td rowspan="1" colspan="1" valign="top" align="left">0.091</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 (1–1.12)</td><td rowspan="1" colspan="1" valign="top" align="left">0.055</td><td rowspan="1" colspan="1" valign="top" align="left">1.06 (1.01–1.11)</td><td rowspan="1" colspan="1" valign="top" align="left">0.011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9929218</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">67378447</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDH1</em></td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.30</td><td rowspan="1" colspan="1" valign="top" align="left">7668</td><td rowspan="1" colspan="1" valign="top" align="left">8770</td><td rowspan="1" colspan="1" valign="top" align="left">0.95 (0.87–1.03)</td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">0.94 (0.88–1)</td><td rowspan="1" colspan="1" valign="top" align="left">0.042</td><td rowspan="1" colspan="1" valign="top" align="left">0.94 (0.90–0.99)</td><td rowspan="1" colspan="1" valign="top" align="left">0.020</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10795668</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">8741225</td><td rowspan="1" colspan="1" valign="top" align="left"><em>LOC338 591</em></td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.32</td><td rowspan="1" colspan="1" valign="top" align="left">7681</td><td rowspan="1" colspan="1" valign="top" align="left">8790</td><td rowspan="1" colspan="1" valign="top" align="left">0.94 (0.87–1.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td><td rowspan="1" colspan="1" valign="top" align="left">0.98 (0.92–1.04)</td><td rowspan="1" colspan="1" valign="top" align="left">0.50</td><td rowspan="1" colspan="1" valign="top" align="left">0.97 (0.92–1.01)</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10411210</td><td rowspan="1" colspan="1" valign="top" align="left">19</td><td rowspan="1" colspan="1" valign="top" align="left">38224140</td><td rowspan="1" colspan="1" valign="top" align="left"><em>RHPN2</em></td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0.10</td><td rowspan="1" colspan="1" valign="top" align="left">7685</td><td rowspan="1" colspan="1" valign="top" align="left">8795</td><td rowspan="1" colspan="1" valign="top" align="left">0.88 (0.77–1.01)</td><td rowspan="1" colspan="1" valign="top" align="left">6.4×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.98 (0.89–1.07)</td><td rowspan="1" colspan="1" valign="top" align="left">0.62</td><td rowspan="1" colspan="1" valign="top" align="left">0.95 (0.88–1.02)</td><td rowspan="1" colspan="1" valign="top" align="left">0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6983267</td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">128482487</td><td rowspan="1" colspan="1" valign="top" align="left"><em>POU5F1 P1, MYC</em></td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">4166</td><td rowspan="1" colspan="1" valign="top" align="left">4990</td><td rowspan="1" colspan="1" valign="top" align="left">0.88 (0.81–0.95)</td><td rowspan="1" colspan="1" valign="top" align="left">7.5×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.80 (0.69–0.93)</td><td rowspan="1" colspan="1" valign="top" align="left">3.3×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0.86 (0.80–0.92)</td><td rowspan="1" colspan="1" valign="top" align="left">1.4×10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140093429668432" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3257356/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>GWAS hits defined as variants with p &lt; 5 × 10<sup>−8</sup>.</div><div id="TFN2"><sup>b</sup>Nearest gene or locus for each SNP. For SNPs where two genes/loci are listed the second gene represents a nearby gene thought to be a strong functional candidate.</div><div id="TFN3"><sup>c</sup>MAF: minor allele frequency based on controls of GWAS studies.</div><div id="TFN4"><sup>d</sup># of cases and controls differ for each SNP depending on genotyping platform used in replication and exclusions applied based on quality control measurements.</div><div id="TFN5"><sup>e</sup>p-value from fixed-effects meta-analysis.</div></div></div><p id="P6">The most significant novel SNP in both the replication study, and in the combined analysis of the GWAS and replication, was rs7315438 located on chromosome 12q24 near <em>MED13L</em> (replication OR=0.92; replication p-value 1.0 × 10<sup>−3</sup>; combined OR=0.92; combined fixed-effects p-value [p<sub>fixed</sub>] 5.6 × 10<sup>−6</sup>; combined random-effects p-value [p<sub>random</sub>] 1.5 × 2=10<sup>−3</sup>; <a href="/pmc/articles/PMC3257356/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>). For two other SNPs, their association with colorectal cancer was nominally significant within the replication study: rs4925386 located on chromosome 20q13 near <em>LAMA5</em> (replication p-value 2.5 × 10<sup>−3</sup>; p<sub>fixed</sub> 2.1 × 10<sup>−4</sup>; p<sub>random</sub> 0.015) and rs16888522 located on 8q23.3 near <em>EIF3H</em> (replication p-value 4.1 × 10<sup>−3</sup>; p<sub>fixed</sub> 1.7 × 10<sup>−5</sup>; p<sub>random</sub> 4.5 × 10<sup>−4</sup>). We note that the rs4925386 SNP did not have strong evidence for association in the random effects model. Since we selected rs4925386 for our replication study, it has been identified as being associated with colorectal cancer by a published GWAS (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_600192533">Houlston et al., 2010</a>). The variant rs16888522 is in the region of rs16892766/<em>EIF3H</em>, previously identified to be associated with colorectal cancer by a GWAS (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467605">Tomlinson et al., 2008</a>). The variant rs16888522/<em>EIF3H</em> was in weak linkage disequilibrium (LD) with rs16892766/<em>EIF3H</em> (D′=0.255; r<sup>2</sup>=0.043; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 4</a>). Conditional analysis, including both variants in the same model, resulted in less significant results for both variants (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>) and showed weak correlation between the beta coefficients (r = −0.269), which suggests that these variants may not be independently associated SNPs.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p29">Risk estimates for colorectal cancer for loci with p&lt; 5 × 10<sup>−4</sup> in combined genome-wide association studies (GWAS) and replication analysis.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093427902512" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="bottom" align="left">SNP</th><th rowspan="3" colspan="1" valign="bottom" align="center">Chr</th><th rowspan="3" colspan="1" valign="bottom" align="center">Position (bp)</th><th rowspan="3" colspan="1" valign="bottom" align="center">Gene/locus<sup>a</sup></th><th rowspan="3" colspan="1" valign="bottom" align="center">Ref. allele</th><th rowspan="3" colspan="1" valign="bottom" align="center">MAF<sup>b</sup></th><th rowspan="3" colspan="1" valign="bottom" align="center"># Case<sup>c</sup></th><th rowspan="3" colspan="1" valign="bottom" align="center"># Control<sup>c</sup></th><th colspan="2" rowspan="1" valign="bottom" align="center">GWAS (2,906 cases 3,416 controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">Replication (8,161 cases 9,101 controls)</th><th colspan="2" rowspan="1" valign="bottom" align="center">GWAS + Replication</th></tr><tr><th colspan="6" rowspan="1" valign="bottom" align="center"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sup>d</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sup>d</sup></th></tr></thead><tbody><tr><td colspan="14" rowspan="1" valign="top" align="left"><strong>SNPs in regions of prior GWAS hits for colorectal cancer</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16888522</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="center">117643094</td><td rowspan="1" colspan="1" valign="top" align="center"><em>EIF3H</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">9308</td><td rowspan="1" colspan="1" valign="top" align="center">10708</td><td rowspan="1" colspan="1" valign="top" align="center">1.32 (1.13–1.54)</td><td rowspan="1" colspan="1" valign="top" align="center">5.0×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.05–1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">4.1×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.11–1.31)</td><td rowspan="1" colspan="1" valign="top" align="center">1.7×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4813802</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">6647595</td><td rowspan="1" colspan="1" valign="top" align="center"><em>BMP2</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">11498</td><td rowspan="1" colspan="1" valign="top" align="center">13075</td><td rowspan="1" colspan="1" valign="top" align="center">1.18 (1.09–1.28)</td><td rowspan="1" colspan="1" valign="top" align="center">5.5×10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.05 (1.01–1.10)</td><td rowspan="1" colspan="1" valign="top" align="center">0.025</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.04–1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">7.3×10<sup>−5</sup></td></tr><tr><td colspan="14" rowspan="1" valign="top" align="left"><strong>SNPs in new regions</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7315438</td><td rowspan="1" colspan="1" valign="top" align="center">12</td><td rowspan="1" colspan="1" valign="top" align="center">114375786</td><td rowspan="1" colspan="1" valign="top" align="center"><em>MED13L</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">9255</td><td rowspan="1" colspan="1" valign="top" align="center">10657</td><td rowspan="1" colspan="1" valign="top" align="center">0.88 (0.81–0.95)</td><td rowspan="1" colspan="1" valign="top" align="center">1.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.92 (0.87–0.97)</td><td rowspan="1" colspan="1" valign="top" align="center">0.001</td><td rowspan="1" colspan="1" valign="top" align="center">0.90 (0.87–0.94)</td><td rowspan="1" colspan="1" valign="top" align="center">5.6×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs275454</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">6869013</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC72943 4/POLS</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">11348</td><td rowspan="1" colspan="1" valign="top" align="center">12925</td><td rowspan="1" colspan="1" valign="top" align="center">1.16 (1.07–1.25)</td><td rowspan="1" colspan="1" valign="top" align="center">2.9×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.06 (1.01–1.11)</td><td rowspan="1" colspan="1" valign="top" align="center">0.014</td><td rowspan="1" colspan="1" valign="top" align="center">1.08 (1.04–1.13)</td><td rowspan="1" colspan="1" valign="top" align="center">8.7×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2373859</td><td rowspan="1" colspan="1" valign="top" align="center">2</td><td rowspan="1" colspan="1" valign="top" align="center">40471324</td><td rowspan="1" colspan="1" valign="top" align="center"><em>SLC8A1</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">10999</td><td rowspan="1" colspan="1" valign="top" align="center">12483</td><td rowspan="1" colspan="1" valign="top" align="center">0.87 (0.8–0.95)</td><td rowspan="1" colspan="1" valign="top" align="center">8.8×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.95 (0.91–0.99)</td><td rowspan="1" colspan="1" valign="top" align="center">0.010</td><td rowspan="1" colspan="1" valign="top" align="center">0.93 (0.9–0.97)</td><td rowspan="1" colspan="1" valign="top" align="center">1.4×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2853668</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="center">1353025</td><td rowspan="1" colspan="1" valign="top" align="center"><em>TERT- CLPTM1L</em></td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.25</td><td rowspan="1" colspan="1" valign="top" align="center">9019</td><td rowspan="1" colspan="1" valign="top" align="center">10413</td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.78–0.93)</td><td rowspan="1" colspan="1" valign="top" align="center">4.5×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.94 (0.88–0.99)</td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="center">0.91 (0.87–0.96)</td><td rowspan="1" colspan="1" valign="top" align="center">1.9×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4925386</td><td rowspan="1" colspan="1" valign="top" align="center">20</td><td rowspan="1" colspan="1" valign="top" align="center">60354439</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LAMA5</em></td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.30</td><td rowspan="1" colspan="1" valign="top" align="center">10485</td><td rowspan="1" colspan="1" valign="top" align="center">11629</td><td rowspan="1" colspan="1" valign="top" align="center">0.91 (0.84–0.99)</td><td rowspan="1" colspan="1" valign="top" align="center">0.032</td><td rowspan="1" colspan="1" valign="top" align="center">0.93 (0.88–0.97)</td><td rowspan="1" colspan="1" valign="top" align="center">2.5×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.92 (0.88–0.96)</td><td rowspan="1" colspan="1" valign="top" align="center">2.1×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1525461</td><td rowspan="1" colspan="1" valign="top" align="center">7</td><td rowspan="1" colspan="1" valign="top" align="center">144359224</td><td rowspan="1" colspan="1" valign="top" align="center"><em>LOC64330 8/TPK1</em></td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">10978</td><td rowspan="1" colspan="1" valign="top" align="center">12483</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 (1.08–1.32)</td><td rowspan="1" colspan="1" valign="top" align="center">3.6×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.06 (1.0–1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">0.037</td><td rowspan="1" colspan="1" valign="top" align="center">1.09 (1.04–1.14)</td><td rowspan="1" colspan="1" valign="top" align="center">3.6×10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm140093427902512" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3257356/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><sup>a</sup>Nearest gene or locus for each SNP. For SNPs where two genes/loci are listed the second gene represents a nearby gene thought to be a strong functional candidate.</div><div id="TFN7"><sup>b</sup>MAF: minor allele frequency based on controls of GWAS studies</div><div id="TFN8"><sup>c</sup># of cases and controls differ for each single-nucleotide polymorphism (SNP) depending on genotyping platform used in replication and exclusions applied based on quality control measurements.</div><div id="TFN9"><sup>d</sup>p-value from fixed-effects meta-analysis</div></div></div><p id="P7">We identified five other loci with p&lt;0.05 in our replication study and combined p-value &lt;10<sup>−4</sup>, (<a href="/pmc/articles/PMC3257356/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). The associated SNPs were located near <em>BMP2, POLS, SLC8A1, TERT-CLPTM1L</em> and <em>TPK1</em>. One was in a region previously identified to be a colorectal cancer susceptibility locus by a GWAS (rs4813802/<em>BMP2</em>): replication p-value 0.03; p<sub>fixed</sub> 7.3 × 10<sup>−5</sup>; p<sub>random</sub> 0.014; <a href="/pmc/articles/PMC3257356/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 3</a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 3</a>). The higher p-value for the random effects for this SNP reflects the fact that we observed evidence of heterogeneity among GWAS for rs4813802 (<em>I<sup>2</sup></em>=76.1% and p = 0.06) (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467619">Ioannidis et al., 2007</a>); however, this was less pronounced among the replication studies (<em>I<sup>2</sup></em> =40.4% and p = 0.08) suggesting that the result is consistent among studies after accounting for those that may be subject to the “winner’s curse” (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467589">Garner, 2007</a>). rs4813802/<em>BMP2</em> was not in LD with the known colorectal cancer susceptibility SNP in the region rs961253/<em>BMP2</em> (D′=0.02, r<sup>2</sup> &lt;0.001) (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467595">Houlston et al., 2008</a>), and the joint conditional analysis demonstrates the independent association of both variants with colorectal cancer risk (correlation of beta coefficients = 0.018; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 4</a>; <a href="/pmc/articles/PMC3257356/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3257356/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093511767184" class="figure"><a href="/pmc/articles/PMC3257356/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3257356_nihms324653f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms324653f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3257356/bin/nihms324653f1.jpg"></a></div><div id="largeobj_idm140093511767184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3257356/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3257356/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association results and LD structure for the associated region on chromosome 20p12.3/<em>BMP2</em> locus</strong><p id="__p20">The top half of the figure has physical position along the x axis, and the −log<sub>10</sub> of the meta-analysis p-value on the y-axis. Each dot on the plot represents the result for one SNP. The bottom half of the figure shows pairwise linkage disequilibrium (LD) for the genotyped SNPs across the region. LD was measured as r<sup>2</sup> and calculated using the control individuals from the WHI, PLCO and DALS samples recruited from 53 centers across the USA. Darker shading indicates higher levels of LD. The lines between the top and bottom half of the figure connecting the same SNPs.</p></div></div></div><p id="P8">For all SNPs in <a href="/pmc/articles/PMC3257356/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, we tested if the risk estimates of these variants may vary by mode of inheritance or sex. While for some variants (rs4813802/<em>BMP2</em>, rs275454/<em>POLS</em>, rs2373859/<em>SLC8A1</em>, and rs2853668/<em>CLPTM1L</em>), the recessive model tended to provide stronger risk estimates and slightly lower p-values than the log-additive or dominant model, the AIC value was &gt;2 in all cases, indicating no statistical evidence for improvement over the log-additive model (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 5</a>). We also explored if results vary by sex and found that for rs16888522/<em>EIF3H</em> the statistical evidence for association was stronger in men (OR=1.25, p-value=0.002) than in women (OR=1.10; p-value=0.25), although the effect estimates were in the same direction and similar in magnitude for both men and women (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 6</a>).</p><p id="P9" class="p p-last">As a sensitivity analysis, we reran the combined fixed-effects meta-analysis leaving out one study at a time for all SNPs in <a href="/pmc/articles/PMC3257356/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>. In no case did the point estimate change &gt;3%. Further, all p<sub>fixed</sub> remained &lt;5×10<sup>−3</sup> except for when we removed the French study from the analysis of rs4925386. In that case the OR remained similar (OR=0.94) but the p-value was slightly attenuated p<sub>fixed</sub>=8.2×10<sup>−3</sup>.</p></div><div id="S3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S3title">Discussion</h2><p id="P10" class="p p-first">From the analysis of GWAS and replication, including a total of up to 11,067 cases and 12,517 controls, we found that SNPs in eight out of ten previously identified colorectal cancer susceptibility loci were associated with the disease in our replication study at p&lt;0.05. We found evidence that a second SNP (rs4813802) near the <em>BMP2</em> gene could be associated with colorectal cancer, independently of the association with the previously identified susceptibility SNP in that region (rs961253). Furthermore, our study reports for the first time a potential new association of a variant in the <em>TERT-CLPTM1L</em> region with colorectal cancer risk.</p><p id="P11">Our results provide further support for eight of ten previously identified GWAS hits. When excluding studies that have previously published results on these known loci, five loci showed evidence of replication in this independent subsample. The 8q24 SNP rs6983267 has already been heavily studied, including published reports for many of the studies included in this paper (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467590">Figueiredo et al., 2011</a>;<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467630">Hutter et al., 2010</a>), so we were not able to examine independent replication of this SNP in this study. Among the remaining four loci that did not show a significant association at p&lt;0.05, three showed a trend towards replication (with p&lt;0.2 and an odds ratio in the same direction as the original GWAS report). However, one SNP, rs10795668, did not show any evidence for association with disease (OR=1.00; 95% CI: 0.93–1.08; p=0.96; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 2</a>). Several papers have reviewed potential reasons for the lack of replication of GWAS findings (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467644">Chanock et al., 2007</a>;<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467580">Kraft et al., 2009</a>). As in any observational study, it is possible this represents either a false positive in the initial report, or a false negative in this replication; although that seems unlikely since both the discovery GWAS and this report are based on large, well-powered studies. We used the same genetic model and similar trait definitions as the discovery GWAS. Further, all studies were restricted to non-Hispanic Whites, limiting the possibility of differences in linkage disequilibrium patterns. It is possible that there may be differences in the distribution of a key effect modifier between the studies used to identify rs10795668 and the studies presented in this paper. A full exploration of underlying gene-gene or gene-environment interactions is beyond the scope of the current paper, but we did explore if the effect of rs10795668 varied by sex. Although the results were not significant for either sex, and the 95% CIs overlap, we do note an interesting pattern where the odds ratios are in opposite directions for women and men, with men showing a trend in the direction of the discovery GWAS. Specifically, we found OR<sub>women</sub> =1.07 (95% CI: 0.92–1.26; p<sub>fixed</sub>=0.38) and OR<sub>men</sub> =0.95 (95% CI: 0.84–1.07; p<sub>fixed</sub>=0.40).</p><p id="P12">The rs4925386/<em>LAMA5</em> SNP was also recently identified in another GWAS meta-analysis (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_600192536">Houlston et al., 2010</a>). Although it was not a known colorectal cancer susceptibility loci at the time that we selected SNPs for replication, this SNP met our criteria for selection, and showed evidence for association in our replication sample. The rs4925386 variant lies in the intron of the large laminin A5 protein encoding gene. As previously reported the variant is in LD (r<sup>2</sup>&gt;0.5) with four nonsynonymous SNPs in <em>LAMA5</em> (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_600192534">Houlston et al., 2010</a>). However, the prediction of each of these amino acid changes is proposed to be benign. Overall, our finding provides additional independent support that this variant is associated with susceptibility to colorectal cancer.</p><p id="P13">None of the loci were significantly associated with colorectal cancer in our replication study after adjusting for multiple testing (0.05/321=1.6 × 10<sup>−4</sup>), and none of the loci reached “genome-wide significance” at the suggested p-values of 1.6 × 10<sup>−7</sup> after accounting for the two-stage design (details see method section) (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467616">Dudbridge and Gusnanto, 2008</a>;<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467638">Hoggart et al., 2008</a>;<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467598">International HapMap Consortium, 2005</a>;<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467608">Pe’er et al., 2008</a>;<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467603">Risch and Merikangas, 1996</a>;<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467622">Wellcome Trust Case Control Consortium, 2007</a>). However, for some of the variants with p&lt;0.05 in our replication and combined p-value &lt;10<sup>−4</sup>, additional lines of evidence provide support for the hypothesis that we may have identified genomic regions harboring causal variants for colorectal cancer susceptibility. The variant rs4813802 is about 295.3kb centromeric to the previously identified rs961253/<em>BMP2</em> GWAS hit (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467563">Houlston et al., 2008</a>); both statistical models and LD data support the idea that these are independent signals. The closest gene is bone morphogenetic protein 2 (<em>BMP2</em>). The new variant of interest, rs4813802, is closer to <em>BMP2</em> (49.2kb upstream) than the previously identified SNP rs961253 (344.5kb upstream of <em>BMP2</em>). Interestingly, rs4813802 lies within an ENCODE Digital DNAseI Hypersensitivity Cluster; it is also within an ENCODE region showing H3K4Me1 enhancer associated histone marks (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467623">Rosenbloom et al., 2010</a>), and the flanking 15 bp show strong placental mammal conservation by PhastCons (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467647">Siepel et al., 2005</a>). While not conclusive, all of these are consistent with the region flanking the SNP acting as a long-range enhancer element, plausibly for <em>BMP2</em>. The <em>BMP2</em> gene belongs to the transforming growth factor-β (TGFβ) superfamily, which plays an important role in cell proliferation, differentiation, and apoptosis (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467624">Massague, 2000</a>). SNPs in five out of the 10 known colorectal cancer SNPs have chromosomal locations in or near TGFβ superfamily genes (<a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467594">Tenesa and Dunlop, 2009</a>). Furthermore, loss in BMP signaling has been reported at the transition from advanced adenoma to early cancer stage, compatible with a role in tumor progression (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467645">Hardwick et al., 2008</a>). Support for a role for BMP signaling in colorectal cancer comes from the identification of mutations in the bone morphogenetic protein receptor, type IA protein (<em>BMPR1A)</em> in juvenile polyposis (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467643">Howe et al., 2001</a>). Individuals with familial juvenile polyposis have a 20% risk of colon cancer by age 35 and 68% by age 60 (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467600">Schreibman et al., 2005</a>). Our finding supports the possibility of allelic heterogeneity at the <em>BMP2</em> locus, which is consistent with findings for the 8q24 cancer locus (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_761311875">Al Olama et al., 2009</a>;<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467602">Witte, 2007</a>) and recent findings for height showing evidence for allelic heterogeneity at as many as 19 loci (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467639">Lango et al., 2010</a>). Similar to our finding, these 19 secondary signals in height were rather distant (on average 177kb) from the initial index SNP that was found to be associated through GWAS (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467634">Lango et al., 2010</a>). Accordingly, a comprehensive exploration of already discovered colorectal cancer loci may uncover additional independent variants. However, this example demonstrates that defining the boundaries of a susceptibility locus may be challenging, because the SNP we identified (rs4813802) would not have been included if we had defined the region around the initial index SNP (rs961253) by LD.</p><p id="P14">The 8q24 region has been shown to have multiple independent variants that are associated with cancers. Several of these variants are associated with more than one cancer, and some cancers are associated with multiple variants in this region (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_761311876">Al Olama et al., 2009</a>;<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467637">Witte, 2007</a>). Similarly, multiple variants associated with various cancer sites, including cancers of lung, pancreas, testes, and bladder, as well as glioma, basal cell carcinoma, and melanoma are found in the <em>TERT</em>-<em>CLPTM1L</em> region (<a href="/pmc/articles/PMC3257356/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>) (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314494545">Hsiung et al., 2010</a>;<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467582">Landi et al., 2009</a>;<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467627">McKay et al., 2008</a>;<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467579">Miki et al., 2010</a>;<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467599">Petersen et al., 2010</a>;<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_489033066">Rafnar et al., 2009</a>;<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_489033065">Shete et al., 2009</a>;<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467640">Stacey et al., 2009</a>;<a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_347774084">Turnbull et al., 2010</a>;<a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467566">Wang et al., 2008</a>). Ours is the first report suggesting that a variant in the <em>TERT</em>-<em>CLPTM1L</em> region could be associated with colorectal cancer. The variant rs2853668 is 4.9kb upstream of telomerase reverse transcriptase (<em>TERT</em>) and 18.0kb downstream of cleft lip and palate transmembrane protein 1-like protein (<em>CLPTM1L</em>). Both genes have been implicated in cancer: <em>CLPTM1L</em> has been shown to be altered in cisplatin-resistant cell lines and potentially impacts apoptosis (<a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467604">Yamamoto et al., 2001</a>); <em>TERT</em> encodes for the telomerase catalytic subunit that is important for the replication and stabilization of telomere ends, and subsequently impacts chromosome replication and suppression of cell senescence. Malfunction of telomerase can result in chromosomal abnormality and subsequent tumor formation (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_489033063">Rafnar et al., 2009</a>). Our finding provides further evidence that <em>TERT-CLPTM1L</em> is a general cancer susceptibility locus that impacts critical function for cancer development, similar to the 8q24 region. The candidate gene in the 8q24 loci is <em>MYC</em>, and, as noted by Johnatty and colleagues (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467612">Johnatty et al., 2010</a>), these two loci could act in concert. Specifically, the <em>TERT</em> promoter has several <em>MYC</em> (the nearest gene to the 8q24 locus) binding sites (<a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467572">Wu et al., 1999</a>); however, we did not observe a statistically significant interaction between rs2853668/<em>TERT-CLPTM1L</em> and rs6983267/8q24, <em>MYC</em> (p for interaction term=0.8).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3257356/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140093428932592" class="figure"><a href="/pmc/articles/PMC3257356/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3257356_nihms324653f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms324653f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3257356/bin/nihms324653f2.jpg"></a></div><div id="largeobj_idm140093428932592" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3257356/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3257356/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Genetic variants associated with different cancer sites in the <em>TERT-CLPTM1L</em> region, including the new finding for colorectal cancer for rs2853668</strong><p id="__p21">This figure shows the genomic region on chromosome 5.p.15.33 including the two genes <em>TERT</em> and <em>CLPTM1L</em>. The top of the figure shows the genes with each exon represented by a short vertical line. The line below the genes shows the location of the different SNPs that have been associated with various cancer sites relative to the position of the genes (for instance, the first two SNPs are located in the last intron of <em>TERT</em>). The triangle shows the pairwise linkage disequilibrium (LD) of the SNPs. LD was measured as r<sup>2</sup> and calculated using the control individuals from the WHI, PLCO and DALS samples recruited from 53 centers across the USA. Darker shading indicates higher levels of LD.</p></div></div></div><p id="P15">The SNP rs7315438, which showed the most statistically significant association in both the replication study alone, as well as in the combined meta-analysis of GWAS and replication studies, is located on chromosome 12q24 about 76.9kb upstream of the T-box 3 protein (<em>TXB3</em>). The SNP is also located 50.4kb downstream of <em>MED13L</em>, which encodes for a subunit of the mediator complex, a large complex of proteins that functions as a transcriptional coactivator for most RNA polymerase II-transcribed genes. Since it has been implicated in transcription, this gene is a plausible candidate for further study. However, this SNP is in a large LD region containing numerous other potential candidate genes, including the kinase suppressor of RAS2 (<em>KSR2</em>).</p><p id="P16">Other SNPs identified as potentially associated with colorectal cancer in this study are rs27545 (<em>POLS</em>), rs2373859 (<em>SLC8A1</em>) and rs1525461 (LOC643308/<em>TPK1</em>). The gene closest to rs27545 is <em>POLS</em> (59 kb downstream) a DNA polymerase that is likely involved in DNA repair and, hence, provides a potentially interesting candidate gene (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467596">Hubscher et al., 2002</a>). Other genes close to rs27545 are <em>SRD5A1</em> (146 kb upstream), which converts testosterone into the more potent dihydrotestosterone, and the methyltransferase <em>NSUN2</em> (183 kb downstream), which methylates tRNA (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467593">Brzezicha et al., 2006</a>). The SNP rs2373859 resides in the intronic region of <em>SLC8A1</em> also known as <em>NCX1</em>, which is a cell membrane protein that is involved in the rapid Ca(2+) transports (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467620">Annunziato et al., 2004</a>). It is in a gene rich region including other interesting candidates, such as <em>MAP4K3</em> (954 kb upstream) a member of the mitogen-activated protein kinases, which is involved in regulating both cell growth and death and has altered gene expression in many cancer types (<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467642">Cuadrado and Nebreda, 2010</a>) and <em>SOS1</em>, which may act as a positive regulator of RAS (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467636">Freedman et al., 2006</a>). The closest gene to rs1525461 is <em>TPK1</em> (195kb upstream). <em>TPK1</em> is involved in the regulation of thiamine metabolism (<a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467625">Timm et al., 2001</a>). <em>TPK1</em> flanks a gene rich region, including several olfactory receptors but none of the genes has an obvious link to colorectal cancer development. However, the assignment of SNPs to candidate genes should be done with caution, as recently shown by additional fine mapping and <em>in silico</em> analysis of the previously identified colorectal cancer loci 8q23.3 (<em>EIF3H</em>), 16q22.1 (<em>CDH1/CDH3</em>), which suggested functional variation in unexpected candidate target genes (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_314494542">Carvajal-Carmona et al., 2011</a>).</p><p id="P17">Overall, our study suggests a complex nature of the contribution of common genetic variants to risk for colorectal cancer, and suggests the need for additional studies to identify variants with marginal effects, as well as studies to examine potential sources and role of heterogeneity, including gene-gene and gene-environment interactions. We note that this study focused on the log-additive model. Although we present results for other genetic models for our top findings, our results may have been biased for SNPs that do not follow this assumed log-additive model (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467628">Minelli et al., 2005</a>). Further, this study was not set up to investigate less frequent (allele frequency 1–5%) and rare variants (allele frequency &lt;1%), which have the potential to contribute substantially to the genetic susceptibility of colorectal cancer (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467632">Bodmer and Bonilla, 2008</a>;<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467567">Cirulli and Goldstein, 2010</a>;<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467629">Manolio et al., 2009</a>).</p><p id="P18" class="p p-last">In summary, we replicated the majority of SNPs that have previously been found to be associated with CRC in GWAS studies. We also report suggestive evidence for an additional independent signal for colorectal cancer risk in the <em>BMP2</em> locus and a possible new association of colorectal cancer with a variant in the multi-cancer susceptibility locus around <em>TERT-CLPTM1L</em>. Future studies are needed to try to replicate these findings, and if successful, to identify the underlying variants directly responsible for the association, and to study the underlying molecular mechanisms.</p></div><div id="S4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S4title">Materials and Methods</h2><div id="S5" class="sec sec-first"><h3 id="S5title">Study Participants</h3><p id="P19" class="p p-first-last">The studies and their abbreviations are listed in <a href="/pmc/articles/PMC3257356/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, and each study is described in detail in the <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Note</a>. In brief, all cases were defined as colorectal adenocarcinoma (International Classification of Disease Code 153–154) and confirmed by medical records, pathologic reports, or death certificate. All cases and controls were self-reported as White, which was confirmed in GWAS samples based on genotype data. All participants gave written informed consent and studies were approved by the Institutional Review Board.</p></div><div id="S6" class="sec"><h3 id="S6title">Study Design</h3><p id="P20" class="p p-first">The GWAS meta-analysis results are based on two scans. One GWAS was conducted within the CCFR, including population-based cases and unrelated population-based controls from three sites: USA, Canada, and Australia (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467626">Figueiredo et al., 2011</a>). In total, 1,191 cases and 999 controls were successfully genotyped on the Illumina 1M/1M Duo platform and passed all quality-control (QC) steps. The second scan was conducted across three US study populations: the WHI and PLCO cohorts and the DALS population-based case-control study. A total of 1,715 colon cancer cases and 2,417 controls were successfully genotyped on the Illumina HumanHap 550K, 610K or combined Illumina 300K and 240K platforms and passed all QC steps. After applying rigorous genotyping QC filters (see below), a total of 378,739 directly genotyped single-nucleotide polymorphisms (SNPs) commonly shared among the scans were included in the GWAS meta-analysis. To further boost the power and inform the ranking of SNPs we included summary statistics from a previously published colorectal cancer GWAS (Colorectal Tumour Gene Identification Consortium, CORGI) in the meta-analysis (<a href="#R58" rid="R58" class=" bibr popnode">The Institute of Cancer Research, 2008</a>;<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467570">Tomlinson et al., 2008</a>). However, to ensure independence of results from prior published scans we did not include any CORGI results in any of the presented odds ratios or p-values.</p><p id="P21">Fixed effects p-values from the GWAS meta-analysis were used to select SNPs for replication. We rank ordered the top SNPs. We used linkage disequilibrium (LD) information in our controls to prune out “redundant” signals (defined as r<sup>2</sup>&gt;0.5 for SNPs ≥100kb apart and r<sup>2</sup>&gt;0.1 for SNPs &lt;100kb apart). For the top five SNPs, with p&lt;10<sup>−5</sup>, we selected two other SNPs with r<sup>2</sup>&gt;0.9 to ensure against potential genotyping failure. We then went down the ranked list until we filled our SNP platform (total number of SNPs selected for this project=343). SNPs were excluded based on p-value for heterogeneity &lt;0.001 (n=1) and poor clustering in visual inspection of cluster plots (n=3). If SNPs had a low design score, we replaced them with an alternative SNP with r<sup>2</sup>&gt;0.9. The lowest ranked SNP had p-value 1.2 × 10<sup>−3</sup>. Our platform also included SNPs for the ten known colorectal cancer susceptibility loci published in previous GWAS at the time we designed the platform. These 343 SNPs were genotyped in samples from DACHS, DALS, French, HPFS, NHS and PHS studies (N=4,062 cases and 4,718 controls) (<a href="/pmc/articles/PMC3257356/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Note</a>) and 306 SNPs were successfully genotyped in all studies (see details below). After we selected SNPs for replication, the ARCTIC genome-wide scan became available (769 cases and 665 controls), and we used imputed data from that study for analysis of the 343 SNPs (12 SNPs were not included due to low imputation quality or low HWE p-values).</p><p id="P22" class="p p-last">As of April 2010, we had genotyped and analyzed the GWAS data and replication data from ARCTIC, DACHS, DALS and the French case control study. We selected 32 SNPs with p&lt;0.1 in this replication set and/or a p<sub>fixed</sub>&lt;10<sup>−4</sup> in the combined replication and GWAS for further genotyping in 2,550 cases and 3,539 controls, including additional samples from NHS, PHS, and HPFS, and samples from MECC and NFCCR. The top SNPs were also analyzed in a second set of data from the CCFR (780 cases and 780 controls). We present results for the total replication sample of 8,161 cases and 9,101 controls.</p></div><div id="S7" class="sec"><h3 id="S7title">Genotyping</h3><p id="P23" class="p p-first">Genomic DNA was extracted from blood samples or, in the case of a subset of PLCO samples, from buccal cells using conventional methods.</p><div id="S8" class="sec"><p></p><h4 id="S8title" class="inline">GWAS for CCFR </h4><p id="P24" class="p p-first">Genotyping was completed on the Illumina Human1M and Human1M-Duo Bead Array in accordance with the manufacturer’s protocol.</p><div id="S9" class="sec"><p></p><h5 id="S9title" class="inline">Sample exclusions </h5><p id="P25" class="p p-first-last">The following sample exclusion criteria were applied: call rate &lt;95% (n=75), any stripe (physical/analytical location on BeadChip) call rate &lt;80% (n=9), discordance with prior genotyping (n=3), non-White (n=29), samples that showed admixture identified using the program STRUCTURE (n=33) (<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467614">Falush et al., 2003</a>; <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467606">Pritchard et al., 2000</a>), high identity by descent using PLINK (n=2), and mismatch between called and phenotypic sex (n=4). The final analysis was based on 1,191 cases and 999 controls.</p></div><div id="S10" class="sec sec-last"><p></p><h5 id="S10title" class="inline">SNP exclusions </h5><p id="P26" class="p p-first">SNPs were excluded if they did not overlap between the Illumina Human1M and Human1M-Duo (n=190,301), were annotated as “Intensity Only” (n=8,263), had call rates &lt;90% on either the Illumina Human1M or Human1M-Duo (n=9,229), or by study center or case-control status (n=12,695). When further restricting analysis to SNPs with Hardy-Weinberg-Equilibrium (HWE) p&gt;0.0001, MAF &gt;0.05, and SNP call rate &gt;0.98, a total of 739,733 SNPs remained in the analysis.</p><p id="P27" class="p p-last">Average sample call rate was equal to 98.6% with &gt;94% of samples having a call rate &gt;98%. Intra- and interplate replicate concordance rates were equal to 99.97% and 98.7%, respectively.</p></div></div><div id="S11" class="sec"><p></p><h4 id="S11title" class="inline">GWAS for DALS, PLCO and WHI </h4><p id="P28" class="p p-first">Genotyping was completed using Illumina HumanHap300 and HumanHap240S (PLCO), 550k (WHI, DALS) and 610k (DALS, PLCO) BeadChip Array System on the Infinium platform in accordance with the manufacturer’s protocol or as previously described for HumanHap300 and HumanHap240S (<a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467581">Yeager et al., 2007</a>).</p><div id="S12" class="sec"><p></p><h5 id="S12title" class="inline">Sample exclusions </h5><p id="P29" class="p p-first-last">Samples were excluded if the average call rate was &lt;97% (DALS n=110, PLCO n=63, WHI n=66) or there was a mismatch between called and phenotypic sex (DALS n=6, PLCO n=1). To search for unexpected duplicates and closely related individuals we calculated identity-by-state values. We excluded unexpected duplicates (DALS n=2). Additionally, we excluded samples based on low concordance with prior genotyping (DALS: n=10, WHI: n=1) as well as samples that did not cluster with the CEU samples in principal component analysis including the three HapMap populations as a reference (DALS n=20, PLCO n=2, WHI n=6). The final analysis was based on 698 cases and 719 controls in DALS, 534 cases and 1,168 controls in PLCO, and 483 cases and 530 controls in WHI.</p></div><div id="S13" class="sec sec-last"><p></p><h5 id="S13title" class="inline">SNP exclusions </h5><p id="P30" class="p p-first">Because we combined data from different platforms, we took precautions to exclude SNPs that do not perform consistently across platforms. This included SNPs reported by Illumina as not performing consistently across platforms (n=78), SNPs found to have more than one discordant call across the 550K and 610K platforms in HapMap Data or our interplatform duplicates (n=185); and SNPs with different MAF calls on the two platforms in our control populations (n=9). We further filtered SNPs within each study (DALS, PLCO, WHI) based on MAF &lt;0.05% or HWE in controls &lt;0.0001. We applied a call rate per chip type per study of &gt;98%. A total of 392,361 SNPs passed all QC checks for all three studies.</p><p id="P31">The average sample call rate was ≥98.8% in any of the three studies, and the concordance rate of blinded duplicates (n=98 pairs) was &gt;97%.</p><p id="P32" class="p p-last">When we combined data across all scans a total of 378,739 autosomal SNPs were successfully genotyped across all studies and used in our final GWAS meta-analysis of 2,906 cases and 3,416 controls.</p></div></div><div id="S14" class="sec sec-last"><p></p><h4 id="S14title" class="inline">Replication </h4><p id="P33" class="p p-first">Genotyping of 343 SNPs in DACHS, DALS, French, and the first sub-sets of HPFS, NHS and PHS were carried out using BeadXpress technology according to the manufacturer’s protocol. Problematic genotype clusters were visually inspected by lot number and the calling algorithm was adjusted, if indicated. 35 SNPs were excluded from the analysis due to poor cluster quality and 2 SNPs were excluded for being out of HWE (p&lt;0.0001) in controls of at least one study. The 306 SNPs in the replication had call rates &gt;92% across studies (average call rate per SNP per study 97.8%). MECC and NFCCR samples were genotyped using Matrix-assisted Laser Desorption/Ionization Time-of-Flight on the Sequenom® MassARRAY 7K platform (Sequenom, Inc., San Diego, CA). A total of 23 and 30 SNPs were successfully genotyped in MECC and NFCCR, respectively. Additional samples from NHS, HPFS and PHS were genotyped on 29 SNPs using the TaqMan® OpenArray® Genotyping Instrument Platform Assays (Applied Biosystems, Carlsbad, CA). Overall, 32 SNPs had call rates &gt;98% across studies (average call rate per SNP per study 99.5%; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Table 7</a>), indicating excellent quality.</p><p id="P34" class="p p-last">Two GWAS data sets (ARCTIC and CCFR II) became available after the GWAS meta-analysis and were used only for replication as described above. ARCTIC has been previously published (<a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467573">Zanke et al., 2007</a>). Because ARCTIC was genotyped on the Affymetrix platform with limited overlap of SNPs with the Illumina platforms, we made use of imputed data for this study. Imputation was done with BEAGLE, using the phased HapMap release 22 as the reference sample (<a href="http://ftp.hapmap.org/phasing/2007-08_rel22/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://ftp.hapmap.org/phasing/2007-08_rel22/</a>) (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467592">Browning and Browning, 2009</a>). SNPs were removed if they were out of HWE (p&lt;0.0001) in the controls (n=1) or had an imputation r<sup>2</sup>&lt;0.3 (n=11). For CCFR phase II, samples were genotyped using the Illumina 1M Omni. Inclusion/exclusion criteria for cases in phase II were consistent to those described for phase I.</p></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Statistical Analysis</h3><div id="S16" class="sec sec-first"><p></p><h4 id="S16title" class="inline">Study-specific analysis of GWAS data </h4><p id="P35" class="p p-first">To estimate the association between each genetic marker and risk for colorectal cancer we calculated odds ratios (ORs) and 95% confidence intervals (CIs) using log-additive genetic models relating the genotype dose (0, 1 or 2 copies of the minor allele) to risk of colorectal cancer. We adjusted for age, sex (when appropriate), center and the first three principal components from EIGENSTRAT to account for population substructure. The CCFR calculated these estimates with Cochran-Mantel-Haenzsel analysis with strata defined by age, sex, and center.</p><p id="P36" class="p p-last">Quantile-quantile (Q-Q) plots were assessed to determine whether the distribution of the p-values in each study population was consistent with the null distribution (except for the extreme tail; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Figure 1</a>). To quantify the data in the QQ plots, we calculated the inflation factor (λ) to measure the over-dispersion of the test-statistics from association tests by dividing the mean of the test statistics by the mean of the expected values from a Chi-Square distribution with 1 degree of freedom.</p></div><div id="S17" class="sec"><p></p><h4 id="S17title" class="inline">Combined analysis of GWAS </h4><p id="P37" class="p p-first-last">We conducted inverse-variance weighted fixed-effects meta-analysis to combine odds ratio estimates from log-additive models or multiplicative methods across individual study populations as described above. In this approach, we weighted the beta estimates of each study by their inverse variance and calculated a combined estimate by summing the weighted betas and dividing by the summed weights. We chose to focus on fixed-effects because we only had a small number of studies. When the number of studies is small, the between study variance may be poorly estimated, resulting in deflated test statistics for association. As such, fixed-effects analysis is better powered for discovery of novel variants (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467564">Kraft et al., 2009</a>). We calculated I<sup>2</sup>, which is a measure of the percentage of total variation across studies due to heterogeneity beyond chance, and obtained the heterogeneity p-values based on Cochran’s Q statistic (<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467575">Ioannidis et al., 2007</a>).</p></div><div id="S18" class="sec"><p></p><h4 id="S18title" class="inline">Study-specific analysis of replication data </h4><p id="P38" class="p p-first-last">To estimate the association between each genetic marker and risk for colorectal cancer we calculated odds ratios (ORs) and 95% confidence intervals using a log-additive genetic model relating the genotype dose (0, 1 or 2 copies of the minor allele) to risk of colorectal cancer and adjusting for age, sex, and study center (as appropriate) in logistic regression analysis.</p></div><div id="S19" class="sec"><p></p><h4 id="S19title" class="inline">Combined analysis of replication data </h4><p id="P39" class="p p-first-last">We conducted inverse-variance weighted fixed-effects meta-analysis to combine odds ratio estimates from log-additive models across individual study populations and measured heterogeneity using I<sup>2</sup> and Cochran’s Q statistic, as discussed above.</p></div><div id="S20" class="sec"><p></p><h4 id="S20title" class="inline">Analysis of combined GWAS and replication data </h4><p id="P40" class="p p-first-last">We again combined across studies using inverse-variance weighted fixed effects meta-analysis. For novel SNPs with p &lt; 5 × 10<sup>−4</sup> based on combined analysis of GWAS and replication, we also report random-effects that incorporate potential heterogeneity into the effect estimate. For these SNPs we also examined dominant, recessive and unrestricted genetic models and compared models by calculating the Akaike information criterion (AIC). We performed stratified analyses and evaluated whether the effects differed by sex. For novel SNPs in regions identified by previous GWAS, we also performed a conditional analysis including both the newly and previously identified SNPs in the region in one model to examine whether the effect of the newly identified SNP can be explained by the existing one. To quantify the independence of the novel SNPs from prior GWAS hits in the same region we calculated the variance-covariance matrix and reported the correlation between the two betas. Finally, we performed a sensitivity analysis where we removed the studies one at a time and examined the results from the fixed-effect meta-analysis. We report any situations where removing one study resulted in a &gt;5% change in the OR point estimate and/or reduced the p-value of the combined fixed-effects meta-analysis to be &lt;5×10<sup>−3</sup>, since that would indicate the results might be being driven by only one study.</p></div><div id="S21" class="sec sec-last"><p></p><h4 id="S21title" class="inline">Criterion for genome-wide significance </h4><p id="P41" class="p p-first">Based on an increasing number of papers (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467607">Dudbridge and Gusnanto, 2008</a>;<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467649">Hoggart et al., 2008</a>;<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467571">International HapMap Consortium, 2005</a>;<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467613">Pe’er et al., 2008</a>;<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467586">Risch and Merikangas, 1996</a>;<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467574">Wellcome Trust Case Control Consortium, 2007</a>) providing a detailed discussion on the appropriate genome-wide significance threshold, which all arrive at similar values in the range of 5 × 10<sup>−7</sup> to 5 × 10<sup>−8</sup> for White populations, we decided to use a p-value of 5 × 10<sup>−8</sup> as genome-wide significance threshold. To account for the two-stage approach (GWAS and replication) we calculated that an overall p-value of 5 × 10<sup>−8</sup> is equal to a combined two-stage p-value of 1.6 × 10<sup>−7</sup> given our sample sizes in the GWAS and replication and a threshold for selecting SNPs from the GWAS of 1.2 × 10<sup>−3</sup> as used here.</p><p id="P42" class="p p-last">We used PLINK (<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467587">Purcell et al., 2007</a>; <a href="#R44" rid="R44" class=" bibr popnode">Purcell, 2011</a>) and R (R <a href="#R46" rid="R46" class=" bibr popnode">Development Core Team, 2011</a>) to conduct the statistical analysis and summarized results graphically using STATA (<a href="#R55" rid="R55" class=" bibr popnode">StataCorp, 2009</a>), snp.plotter (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467631">Luna and Nicodemus, 2007</a>), and LocusZOOM (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467621">Pruim et al., 2010</a>).</p></div></div></div><div id="S25" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S25title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="N0xd2323b0N0xdf94fd0"><a href="/pmc/articles/PMC3257356/bin/NIHMS324653-supplement-supplement_1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(610K, pdf)</sup></div></div></div><div id="S22" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S22title">Acknowledgments</h2><div class="sec"><p id="__p2">The authors thank Dr. Ian Tomlinson at the Wellcome Trust Centre for Human Genetics, Oxford, UK, Dr. Richard Houlston at the Section of Cancer Genetics, Institute of Cancer Research, Sutton, UK and Dr. Malcolm Dunlop at Colon Cancer Genetics Group, Institute of Genetics and Molecular Medicine, University of Edinburgh and Human Genetics Unit, Medical Research Council, Edinburgh, UK for providing access to GWAS summary statistics of the Colorectal Tumour Gene Identification Consortium (CORGI) and allow us to use these results to inform the ranking of the SNP selection for the replication.</p><p id="__p3"><strong>ARCTIC</strong>: This work was supported by the Cancer Risk Evaluation (CaRE) Program grant from the Canadian Cancer Society Research Institute. TJH and BWZ are recipients of Senior Investigator Awards from the Ontario Institute for Cancer Research, through generous support from the Ontario Ministry of Research.</p><p id="__p4"><strong>CCFR</strong>: This work was supported by the National Cancer Institute, National Institutes of Health under RFA # CA-95-011 and through cooperative agreements with members of the Colon Cancer Family Registry and P.I.s. This genome-wide scan was supported by the National Cancer Institute, National Institutes of Health by U01 <a id="__tag_814140746" class="tag_hotlink" href="/nuccore/CA122839" ref="/nuccore/CA122839"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA122839","term_id":"34976147","term_text":"CA122839"}}</span>CA122839</a>. The content of this manuscript does not necessarily reflect the views or policies of the National Cancer Institute or any of the collaborating centers in the CFRs, nor does mention of trade names, commercial products, or organizations imply endorsement by the US Government or the CFR. The following Colon CFR centers contributed data to this manuscript and were supported by the following sources: Australasian Colorectal Cancer Family Registry (U01 <a id="__tag_814140745" class="tag_hotlink" href="/nuccore/CA097735" ref="/nuccore/CA097735"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA097735","term_id":"34951042","term_text":"CA097735"}}</span>CA097735</a>), Familial Colorectal Neoplasia Collaborative Group (U01 <a id="__tag_814140748" class="tag_hotlink" href="/nuccore/CA074799" ref="/nuccore/CA074799"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074799","term_id":"34927072","term_text":"CA074799"}}</span>CA074799</a>), Mayo Clinic Cooperative Family Registry for Colon Cancer Studies (U01 <a id="__tag_814140747" class="tag_hotlink" href="/nuccore/CA074800" ref="/nuccore/CA074800"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074800","term_id":"34927073","term_text":"CA074800"}}</span>CA074800</a>), Ontario Registry for Studies of Familial Colorectal Cancer (U01 <a id="__tag_814140743" class="tag_hotlink" href="/nuccore/CA074783" ref="/nuccore/CA074783"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074783","term_id":"34927056","term_text":"CA074783"}}</span>CA074783</a>), Seattle Colorectal Cancer Family Registry (U01 <a id="__tag_814140744" class="tag_hotlink" href="/nuccore/CA074794" ref="/nuccore/CA074794"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074794","term_id":"34927067","term_text":"CA074794"}}</span>CA074794</a>), University of Hawaii Colorectal Cancer Family Registry (U01 <a id="__tag_814140742" class="tag_hotlink" href="/nuccore/CA074806" ref="/nuccore/CA074806"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074806","term_id":"34927079","term_text":"CA074806"}}</span>CA074806</a>).</p><p id="__p5"><strong>DACHS</strong>: This work was supported by grants from the German Research Council (Deutsche Forschungsgemeinschaft, BR 1704/6-1, BR 1704/6-3, BR 1704/6-4 and CH 117/1-1), and the German Federal Ministry of Education and Research (01KH0404 and 01ER0814). We thank all participants and cooperating clinicians, and Ute Handte-Daub, Belinda-Su Kaspereit and Ursula Eilber for excellent technical assistance.</p><p id="__p6"><strong>DALS</strong>: This work was supported by the National Cancer Institute, National Institutes of Health, U.S. Department of Health and Human Services (R01 CA48998 to MLS).</p><p id="__p7"><strong>DALS, PLCO and WHI GWAS:</strong> Funding for the genome-wide scan of DALS, PLCO, and DALS was provided by the National Cancer Institute, Institutes of Health, U.S. Department of Health and Human Services (R01 CA059045 to UP). CMH was supported by a training grant from the National Cancer Institute, Institutes of Health, U.S. Department of Health and Human Services (R25 CA094880).</p><p id="__p8"><strong>FRENCH</strong>: This work was funded by a regional Hospital Clinical Research Program (PHRC) and supported by the Regional Council of Pays de la Loire, the Groupement des Entreprises Françaises dans la LUtte contre le Cancer (GEFLUC), the Association Anne de Bretagne Génétique and the Ligue Régionale Contre le Cancer (LRCC).</p><p id="__p9"><strong>GECCO:</strong> Funding for GECCO infrastructure is supported by National Cancer Institute, Institutes of Health, U.S. Department of Health and Human Services (U01 <a id="__tag_814140741" class="tag_hotlink" href="/nuccore/CA137088" ref="/nuccore/CA137088"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA137088","term_id":"35026730","term_text":"CA137088"}}</span>CA137088</a> to UP).</p><p id="__p10"><strong>HPFS</strong>: This work was supported by the National Institutes of Health (P01 CA 055075 to C.S.F., R01 137178 to A.T.C, and P50 CA 127003 to C.S.F.). We acknowledge Patrice Soule and Hardeep Ranu for genotyping at the Dana-Farber Harvard Cancer Center High Throughput Polymorphism Core under the supervision of David J. Hunter, and Carolyn Guo for programming assistance.</p><p id="__p11"><strong>MECC</strong>: This work was supported by the National Institutes of Health, U.S. Department of Health and Human Services (R01 CA81488 to SBG and GR).</p><p id="__p12"><strong>NFCCR</strong>: This work was supported by an Interdisciplinary Health Research Team award from the Canadian Institutes of Health Research (CRT 43821); the National Institutes of Health, U.S. Department of Health and Human Serivces (U01 CA74783); and National Cancer Institute of Canada grants (18223 and 18226). The authors wish to acknowledge the contribution of Alexandre Belisle and the genotyping team of the McGill University and Génome Québec Innovation Centre, Montréal, Canada, for genotyping the Sequenom panel in the NFCCR samples.</p><p id="__p13"><strong>NHS</strong>: This work was supported by the National Institutes of Health (P01 CA 087969 to ELG, R01 137178 to ATC, and P50 CA 127003 to CSF). We acknowledge Patrice Soule and Hardeep Ranu for genotyping at the Dana-Farber Harvard Cancer Center High Throughput Polymorphism Core under the supervision of David J. Hunter, and Carolyn Guo for programming assistance.</p><p id="__p14"><strong>PHS</strong>: We acknowledge Patrice Soule and Hardeep Ranu for genotyping at the Dana-Farber Harvard Cancer Center High Throughput Polymorphism Core under the supervision of David J. Hunter, and Haiyan Zhang for programming assistance.</p><p id="__p15"><strong>PLCO:</strong> This research was supported by the Intramural Research Program of the Division of Cancer Epidemiology and Genetics and supported by contracts from the Division of Cancer Prevention, National Cancer Institute, National Institutes of Health, U.S. Department of Health and Human Services. The authors thank Drs. Christine Berg and Philip Prorok at the Division of Cancer Prevention at the National Cancer Institute, and investigators and staff from the screening centers of the Prostate, Lung, Colorectal, and Ovarian (PLCO) Cancer Screening Trial, Mr. Thomas Riley and staff at Information Management Services, Inc., Ms. Barbara O’Brien and staff at Westat, Inc., and Mr. Tim Sheehy and staff at SAIC-Frederick. Most importantly, we acknowledge the study participants for their contributions to making this study possible.</p><p id="__p16">Control samples were genotyped as part of the Cancer Genetic Markers of Susceptibility (CGEMS) prostate cancer scan were supported by the Intramural Research Program of the National Cancer Institute. The datasets used in this analysis were accessed with appropriate approval through the dbGaP online resource (<a href="http://www.cgems.cancer.gov/data" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cgems.cancer.gov/data</a>) through dbGaP accession number 000207v.1p1.c1.(2009;<a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_247467591">Yeager et al., 2007</a>) Control samples were also genotyped as part of the GWAS of Lung Cancer and Smoking. Funding for this work was provided through the National Institutes of Health, Genes, Environment and Health Initiative [NIH GEI] (Z01 CP 010200). The human subjects participating in the GWAS are derived from the Prostate, Lung, Colon and Ovarian Screening Trial and the study is supported by intramural resources of the National Cancer Institute. Assistance with genotype cleaning, as well as with general study coordination, was provided by the Gene Environment Association Studies, GENEVA Coordinating Center (U01 <a id="__tag_814140740" class="tag_hotlink" href="/nuccore/HG004446" ref="/nuccore/HG004446"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HG004446","term_id":"585865293","term_text":"HG004446"}}</span>HG004446</a>). Assistance with data cleaning was provided by the National Center for Biotechnology Information. Funding support for genotyping, which was performed at the Johns Hopkins University Center for Inherited Disease Research, was provided by the NHI GEI (U01 HG 004438). The datasets used for the analyses described in this manuscript were obtained from dbGaP at <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a> through dbGaP accession number ph000093.v2.p2.c1.</p><p id="__p17"><strong>WHI:</strong> The WHI program is funded by the National Heart, Lung, and Blood Institute, National Institutes of Health, U.S. Department of Health and Human Services through contracts N01WH22110, 24152, 32100-2, 32105-6, 32108-9, 32111-13, 32115, 32118-32119, 32122, 42107-26, 42129-32, 44221, and 268200764316C.</p><p id="__p18">The authors wish to acknowledge Jacques Rossouw, Shari Ludlam, Joan McGowan, Leslie Ford, and Nancy Geller at the (National Heart, Lung, and Blood Institute, Bethesda, Maryland); the following Clinical Coordinating Center investigators: Kooperberg (Fred Hutchinson Cancer Research Center, Seattle, WA) Ross Prentice, Garnet Anderson, Andrea LaCroix, Charles Kooperberg, (Medical Research Labs, Highland Heights, KY) Evan Stein, and (University of California at San Francisco, San Francisco, CA) Steven Cummings; and (Wake Forest University School of Medicine, Winston-Salem, NC) Sally Shumaker with the Women’s Health Initiative Memory Study.</p><p id="__p19">In addition, we wish to acknowledge the following Clinical Center investigators: (Albert Einstein College of Medicine, Bronx, NY) Sylvia Wassertheil-Smoller; (Baylor College of Medicine, Houston, TX) Haleh Sangi-Haghpeykar; (Brigham and Women’s Hospital, Harvard Medical School, Boston, MA) JoAnn E. Manson; (Brown University, Providence, RI) Charles B. Eaton; (Emory University, Atlanta, GA) Lawrence S. Phillips; (Fred Hutchinson Cancer Research Center, Seattle, WA) Shirley Beresford; (George Washington University Medical Center, Washington, DC) Lisa Martin; (Los Angeles Biomedical Research Institute at Harbor- UCLA Medical Center, Torrance, CA) Rowan Chlebowski; (Kaiser Permanente Center for Health Research, Portland, OR) Erin LeBlanc; (Kaiser Permanente Division of Research, Oakland, CA) Bette Caan; (Medical College of Wisconsin, Milwaukee, WI) Jane Morley Kotchen; (MedStar Research Institute/Howard University, Washington, DC) Barbara V. Howard; (Northwestern University, Chicago/Evanston, IL) Linda Van Horn; (Rush Medical Center, Chicago, IL) Henry Black; (Stanford Prevention Research Center, Stanford, CA) Marcia L. Stefanick; (State University of New York at Stony Brook, Stony Brook, NY) Dorothy Lane; (The Ohio State University, Columbus, OH) Rebecca Jackson; (University of Alabama at Birmingham, Birmingham, AL) Cora E. Lewis; (University of Arizona, Tucson/Phoenix, AZ) Cynthia A. Thomson; (University at Buffalo, Buffalo, NY) Jean Wactawski-Wende; (University of California at Davis, Sacramento, CA) John Robbins; (University of California at Irvine, CA) F. Allan Hubbell; (University of California at Los Angeles, Los Angeles, CA) Lauren Nathan; (University of California at San Diego, LaJolla/Chula Vista, CA) Robert D. Langer; (University of Cincinnati, Cincinnati, OH) Margery Gass; (University of Florida, Gainesville/Jacksonville, FL) Marian Limacher; (University of Hawaii, Honolulu, HI) J. David Curb; (University of Iowa, Iowa City/Davenport, IA) Robert Wallace; (University of Massachusetts/Fallon Clinic, Worcester, MA) Judith Ockene; (University of Medicine and Dentistry of New Jersey, Newark, NJ) Norman Lasser; (University of Miami, Miami, FL) Mary Jo O’Sullivan; (University of Minnesota, Minneapolis, MN) Karen Margolis; (University of Nevada, Reno, NV) Robert Brunner; (University of North Carolina, Chapel Hill, NC) Gerardo Heiss; (University of Pittsburgh, Pittsburgh, PA) Lewis Kuller; (University of Tennessee Health Science Center, Memphis, TN) Karen C. Johnson; (University of Texas Health Science Center, San Antonio, TX) Robert Brzyski; (University of Wisconsin, Madison, WI) Gloria E. Sarto; (Wake Forest University School of Medicine, Winston-Salem, NC) Mara Vitolins; (Wayne State University School of Medicine/Hutzel Hospital, Detroit, MI) Michael S. Simon.</p></div></div><div id="idm140093438132240" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140093438132240title">Reference List</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation"><span><span class="ref-journal">Cancer Genetic Markers of Susceptibility (CGEMS) Data Website. </span>2009</span>  <a href="http://cgemscancer.gov/data/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://cgemscancer.gov/data/</a></span></li><li id="R2"><span class="element-citation">Al Olama AA, Kote-Jarai Z, Giles GG, Guy M, Morrison J, Severi G, Leongamornlert DA, Tymrakiewicz M, Jhavar S, Saunders E, Hopper JL, Southey MC, Muir KR, English DR, Dearnaley DP, rdern-Jones AT, Hall AL, O’Brien LT, Wilkinson RA, Sawyer E, Lophatananon A, Horwich A, Huddart RA, Khoo VS, Parker CC, Woodhouse CJ, Thompson A, Christmas T, Ogden C, Cooper C, Donovan JL, Hamdy FC, Neal DE, Eeles RA, Easton DF. Multiple loci on 8q24 associated with prostate cancer susceptibility. <span><span class="ref-journal">Nat Genet </span>2009</span> [<a href="/pubmed/19767752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+loci+on+8q24+associated+with+prostate+cancer+susceptibility&amp;author=AA+Al+Olama&amp;author=Z+Kote-Jarai&amp;author=GG+Giles&amp;author=M+Guy&amp;author=J+Morrison&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Annunziato L, Pignataro G, Di Renzo GF. Pharmacology of brain Na+/Ca2+ exchanger: from molecular biology to therapeutic perspectives. <span><span class="ref-journal">Pharmacol Rev. </span>2004;<span class="ref-vol">56</span>:633–654.</span> [<a href="/pubmed/15602012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol+Rev&amp;title=Pharmacology+of+brain+Na+/Ca2++exchanger:+from+molecular+biology+to+therapeutic+perspectives&amp;author=L+Annunziato&amp;author=G+Pignataro&amp;author=GF+Di+Renzo&amp;volume=56&amp;publication_year=2004&amp;pages=633-654&amp;pmid=15602012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Bodmer W, Bonilla C. Common and rare variants in multifactorial susceptibility to common diseases. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:695–701.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2527050/">PMC free article</a>]</span> [<a href="/pubmed/18509313" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Common+and+rare+variants+in+multifactorial+susceptibility+to+common+diseases&amp;author=W+Bodmer&amp;author=C+Bonilla&amp;volume=40&amp;publication_year=2008&amp;pages=695-701&amp;pmid=18509313&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Broderick P, Carvajal-Carmona L, Pittman AM, Webb E, Howarth K, Rowan A, Lubbe S, Spain S, Sullivan K, Fielding S, Jaeger E, Vijayakrishnan J, Kemp Z, Gorman M, Chandler I, Papaemmanuil E, Penegar S, Wood W, Sellick G, Qureshi M, Teixeira A, Domingo E, Barclay E, Martin L, Sieber O, Kerr D, Gray R, Peto J, Cazier JB, Tomlinson I, Houlston RS. A genome-wide association study shows that common alleles of SMAD7 influence colorectal cancer risk. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1315–1317.</span> [<a href="/pubmed/17934461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+shows+that+common+alleles+of+SMAD7+influence+colorectal+cancer+risk&amp;author=P+Broderick&amp;author=L+Carvajal-Carmona&amp;author=AM+Pittman&amp;author=E+Webb&amp;author=K+Howarth&amp;volume=39&amp;publication_year=2007&amp;pages=1315-1317&amp;pmid=17934461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:210–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-223&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Brzezicha B, Schmidt M, Makalowska I, Jarmolowski A, Pienkowska J, Szweykowska-Kulinska Z. Identification of human tRNA:m5C methyltransferase catalysing intron-dependent m5C formation in the first position of the anticodon of the pre-tRNA Leu (CAA) <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:6034–6043.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1635329/">PMC free article</a>]</span> [<a href="/pubmed/17071714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Identification+of+human+tRNA:m5C+methyltransferase+catalysing+intron-dependent+m5C+formation+in+the+first+position+of+the+anticodon+of+the+pre-tRNA+Leu+(CAA)&amp;author=B+Brzezicha&amp;author=M+Schmidt&amp;author=I+Makalowska&amp;author=A+Jarmolowski&amp;author=J+Pienkowska&amp;volume=34&amp;publication_year=2006&amp;pages=6034-6043&amp;pmid=17071714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Carvajal-Carmona LG, Cazier JB, Jones AM, Howarth K, Broderick P, Pittman A, Dobbins S, Tenesa A, Farrington S, Prendergast J, Theodoratou E, Barnetson R, Conti D, Newcomb P, Hopper JL, Jenkins MA, Gallinger S, Duggan DJ, Campbell H, Kerr D, Casey G, Houlston R, Dunlop M, Tomlinson I. Fine-mapping of colorectal cancer susceptibility loci at 8q23.3, 16q22.1 and 19q13.11: refinement of association signals and use of in silico analysis to suggest functional variation and unexpected candidate target genes. <span><span class="ref-journal">Hum Mol Genet </span>2011</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3118761/">PMC free article</a>]</span> [<a href="/pubmed/21531788" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Fine-mapping+of+colorectal+cancer+susceptibility+loci+at+8q23.3,+16q22.1+and+19q13.11:+refinement+of+association+signals+and+use+of+in+silico+analysis+to+suggest+functional+variation+and+unexpected+candidate+target+genes&amp;author=LG+Carvajal-Carmona&amp;author=JB+Cazier&amp;author=AM+Jones&amp;author=K+Howarth&amp;author=P+Broderick&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Chanock SJ, Manolio T, Boehnke M, Boerwinkle E, Hunter DJ, Thomas G, Hirschhorn JN, Abecasis G, Altshuler D, Bailey-Wilson JE, Brooks LD, Cardon LR, Daly M, Donnelly P, Fraumeni JF, Jr, Freimer NB, Gerhard DS, Gunter C, Guttmacher AE, Guyer MS, Harris EL, Hoh J, Hoover R, Kong CA, Merikangas KR, Morton CC, Palmer LJ, Phimister EG, Rice JP, Roberts J, Rotimi C, Tucker MA, Vogan KJ, Wacholder S, Wijsman EM, Winn DM, Collins FS. Replicating genotype-phenotype associations. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:655–660.</span> [<a href="/pubmed/17554299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Replicating+genotype-phenotype+associations&amp;author=SJ+Chanock&amp;author=T+Manolio&amp;author=M+Boehnke&amp;author=E+Boerwinkle&amp;author=DJ+Hunter&amp;volume=447&amp;publication_year=2007&amp;pages=655-660&amp;pmid=17554299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Cirulli ET, Goldstein DB. Uncovering the roles of rare variants in common disease through whole-genome sequencing. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:415–425.</span> [<a href="/pubmed/20479773" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Uncovering+the+roles+of+rare+variants+in+common+disease+through+whole-genome+sequencing&amp;author=ET+Cirulli&amp;author=DB+Goldstein&amp;volume=11&amp;publication_year=2010&amp;pages=415-425&amp;pmid=20479773&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Cuadrado A, Nebreda AR. Mechanisms and functions of p38 MAPK signalling. <span><span class="ref-journal">Biochem J. </span>2010;<span class="ref-vol">429</span>:403–417.</span> [<a href="/pubmed/20626350" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+J&amp;title=Mechanisms+and+functions+of+p38+MAPK+signalling&amp;author=A+Cuadrado&amp;author=AR+Nebreda&amp;volume=429&amp;publication_year=2010&amp;pages=403-417&amp;pmid=20626350&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">de la Chapelle A. Genetic predisposition to colorectal cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2004;<span class="ref-vol">4</span>:769–780.</span> [<a href="/pubmed/15510158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Genetic+predisposition+to+colorectal+cancer&amp;author=A+de+la+Chapelle&amp;volume=4&amp;publication_year=2004&amp;pages=769-780&amp;pmid=15510158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Dudbridge F, Gusnanto A. Estimation of significance thresholds for genomewide association scans. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:227–234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2573032/">PMC free article</a>]</span> [<a href="/pubmed/18300295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Estimation+of+significance+thresholds+for+genomewide+association+scans&amp;author=F+Dudbridge&amp;author=A+Gusnanto&amp;volume=32&amp;publication_year=2008&amp;pages=227-234&amp;pmid=18300295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>:1567–1587.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462648/">PMC free article</a>]</span> [<a href="/pubmed/12930761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data:+linked+loci+and+correlated+allele+frequencies&amp;author=D+Falush&amp;author=M+Stephens&amp;author=JK+Pritchard&amp;volume=164&amp;publication_year=2003&amp;pages=1567-1587&amp;pmid=12930761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Figueiredo JC, Lewinger JP, Song C, Campbell PT, Conti DV, Edlund CK, Duggan DJ, Rangrej J, Lemire M, Hudson T, Zanke B, Cotterchio M, Gallinger S, Jenkins M, Hopper J, Haile R, Newcomb P, Potter J, Baron JA, Marchand LL, Casey G. Genotype-environment interactions in microsatellite stable/microsatellite instability-low colorectal cancer: results from a genome-wide association study. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2011;<span class="ref-vol">20</span>:758–766.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3089660/">PMC free article</a>]</span> [<a href="/pubmed/21357381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Genotype-environment+interactions+in+microsatellite+stable/microsatellite+instability-low+colorectal+cancer:+results+from+a+genome-wide+association+study&amp;author=JC+Figueiredo&amp;author=JP+Lewinger&amp;author=C+Song&amp;author=PT+Campbell&amp;author=DV+Conti&amp;volume=20&amp;publication_year=2011&amp;pages=758-766&amp;pmid=21357381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Freedman ML, Haiman CA, Patterson N, McDonald GJ, Tandon A, Waliszewska A, Penney K, Steen RG, Ardlie K, John EM, Oakley-Girvan I, Whittemore AS, Cooney KA, Ingles SA, Altshuler D, Henderson BE, Reich D. Admixture mapping identifies 8q24 as a prostate cancer risk locus in African-American men. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2006;<span class="ref-vol">103</span>:14068–14073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1599913/">PMC free article</a>]</span> [<a href="/pubmed/16945910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Admixture+mapping+identifies+8q24+as+a+prostate+cancer+risk+locus+in+African-American+men&amp;author=ML+Freedman&amp;author=CA+Haiman&amp;author=N+Patterson&amp;author=GJ+McDonald&amp;author=A+Tandon&amp;volume=103&amp;publication_year=2006&amp;pages=14068-14073&amp;pmid=16945910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Garner C. Upward bias in odds ratio estimates from genome-wide association studies. <span><span class="ref-journal">Genet Epidemiol. </span>2007;<span class="ref-vol">31</span>:288–295.</span> [<a href="/pubmed/17266119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Upward+bias+in+odds+ratio+estimates+from+genome-wide+association+studies&amp;author=C+Garner&amp;volume=31&amp;publication_year=2007&amp;pages=288-295&amp;pmid=17266119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Gruber SB, Moreno V, Rozek LS, Rennert HS, Lejbkowicz F, Bonner JD, Greenson JK, Giordano TJ, Fearon ER, Rennert G. Genetic Variation in 8q24 Associated with Risk of Colorectal Cancer. <span><span class="ref-journal">Cancer Biol Ther. </span>2007:6.</span> [<a href="/pubmed/17630503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Biol+Ther&amp;title=Genetic+Variation+in+8q24+Associated+with+Risk+of+Colorectal+Cancer&amp;author=SB+Gruber&amp;author=V+Moreno&amp;author=LS+Rozek&amp;author=HS+Rennert&amp;author=F+Lejbkowicz&amp;publication_year=2007&amp;pages=6&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Hardwick JC, Kodach LL, Offerhaus GJ, Van Den Brink GR. Bone morphogenetic protein signalling in colorectal cancer. <span><span class="ref-journal">Nat Rev Cancer. </span>2008;<span class="ref-vol">8</span>:806–812.</span> [<a href="/pubmed/18756288" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Cancer&amp;title=Bone+morphogenetic+protein+signalling+in+colorectal+cancer&amp;author=JC+Hardwick&amp;author=LL+Kodach&amp;author=GJ+Offerhaus&amp;author=GR+Van+Den+Brink&amp;volume=8&amp;publication_year=2008&amp;pages=806-812&amp;pmid=18756288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Hoggart CJ, Clark TG, De IM, Whittaker JC, Balding DJ. Genome-wide significance for dense SNP and resequencing data. <span><span class="ref-journal">Genet Epidemiol. </span>2008;<span class="ref-vol">32</span>:179–185.</span> [<a href="/pubmed/18200594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Genome-wide+significance+for+dense+SNP+and+resequencing+data&amp;author=CJ+Hoggart&amp;author=TG+Clark&amp;author=IM+De&amp;author=JC+Whittaker&amp;author=DJ+Balding&amp;volume=32&amp;publication_year=2008&amp;pages=179-185&amp;pmid=18200594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">Houlston RS, Cheadle J, Dobbins SE, Tenesa A, Jones AM, Howarth K, Spain SL, Broderick P, Domingo E, Farrington S, Prendergast JG, Pittman AM, Theodoratou E, Smith CG, Olver B, Walther A, Barnetson RA, Churchman M, Jaeger EE, Penegar S, Barclay E, Martin L, Gorman M, Mager R, Johnstone E, Midgley R, Niittymaki I, Tuupanen S, Colley J, Idziaszczyk S, Thomas HJ, Lucassen AM, Evans DG, Maher ER, Maughan T, Dimas A, Dermitzakis E, Cazier JB, Aaltonen LA, Pharoah P, Kerr DJ, Carvajal-Carmona LG, Campbell H, Dunlop MG, Tomlinson IP. Meta-analysis of three genome-wide association studies identifies susceptibility loci for colorectal cancer at 1q41, 3q26.2, 12q13.13 and 20q13.33. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:973–977.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5098601/">PMC free article</a>]</span> [<a href="/pubmed/20972440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+three+genome-wide+association+studies+identifies+susceptibility+loci+for+colorectal+cancer+at+1q41,+3q26.2,+12q13.13+and+20q13.33&amp;author=RS+Houlston&amp;author=J+Cheadle&amp;author=SE+Dobbins&amp;author=A+Tenesa&amp;author=AM+Jones&amp;volume=42&amp;publication_year=2010&amp;pages=973-977&amp;pmid=20972440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Houlston RS, Webb E, Broderick P, Pittman AM, Di Bernardo MC, Lubbe S, Chandler I, Vijayakrishnan J, Sullivan K, Penegar S, Carvajal-Carmona L, Howarth K, Jaeger E, Spain SL, Walther A, Barclay E, Martin L, Gorman M, Domingo E, Teixeira AS, Kerr D, Cazier JB, Niittymaki I, Tuupanen S, Karhu A, Aaltonen LA, Tomlinson IPM, Farrington SM, Tenesa A, Prendergast JGD, Barnetson RA, Cetnarskyj R, Porteous ME, Pharoah PDP, Koessler T, Hampe J, Buch S, Schafmayer C, Tepel J, Schreiber S, Volzke H, Chang-Claude J, Hoffmeister M, Brenner H, Zanke BW, Montpetit A, Hudson TJ, Gallinger S, Campbell H, Dunlop MG. Meta-analysis of genome-wide association data identifies four new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nature Genetics. </span>2008;<span class="ref-vol">40</span>:1426–1435.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2836775/">PMC free article</a>]</span> [<a href="/pubmed/19011631" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+Genetics&amp;title=Meta-analysis+of+genome-wide+association+data+identifies+four+new+susceptibility+loci+for+colorectal+cancer&amp;author=RS+Houlston&amp;author=E+Webb&amp;author=P+Broderick&amp;author=AM+Pittman&amp;author=MC+Di+Bernardo&amp;volume=40&amp;publication_year=2008&amp;pages=1426-1435&amp;pmid=19011631&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Howe JR, Bair JL, Sayed MG, Anderson ME, Mitros FA, Petersen GM, Velculescu VE, Traverso G, Vogelstein B. Germline mutations of the gene encoding bone morphogenetic protein receptor 1A in juvenile polyposis. <span><span class="ref-journal">Nat Genet. </span>2001;<span class="ref-vol">28</span>:184–187.</span> [<a href="/pubmed/11381269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Germline+mutations+of+the+gene+encoding+bone+morphogenetic+protein+receptor+1A+in+juvenile+polyposis&amp;author=JR+Howe&amp;author=JL+Bair&amp;author=MG+Sayed&amp;author=ME+Anderson&amp;author=FA+Mitros&amp;volume=28&amp;publication_year=2001&amp;pages=184-187&amp;pmid=11381269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Hsiung CA, Lan Q, Hong YC, Chen CJ, Hosgood HD, Chang IS, Chatterjee N, Brennan P, Wu C, Zheng W, Chang GC, Wu T, Park JY, Hsiao CF, Kim YH, Shen H, Seow A, Yeager M, Tsai YH, Kim YT, Chow WH, Guo H, Wang WC, Sung SW, Hu Z, Chen KY, Kim JH, Chen Y, Huang L, Lee KM, Lo YL, Gao YT, Kim JH, Liu L, Huang MS, Jung TH, Jin G, Caporaso N, Yu D, Kim CH, Su WC, Shu XO, Xu P, Kim IS, Chen YM, Ma H, Shen M, Cha SI, Tan W, Chang CH, Sung JS, Zhang M, Yang TY, Park KH, Yuenger J, Wang CL, Ryu JS, Xiang Y, Deng Q, Hutchinson A, Kim JS, Cai Q, Landi MT, Yu CJ, Park JY, Tucker M, Hung JY, Lin CC, Perng RP, Boffetta P, Chen CY, Chen KC, Yang SY, Hu CY, Chang CK, Fraumeni JF, Jr, Chanock S, Yang PC, Rothman N, Lin D. The 5p15.33 locus is associated with risk of lung adenocarcinoma in never-smoking females in Asia. <span><span class="ref-journal">PLoS Genet. </span>2010:6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2916850/">PMC free article</a>]</span> [<a href="/pubmed/20700438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=The+5p15.33+locus+is+associated+with+risk+of+lung+adenocarcinoma+in+never-smoking+females+in+Asia&amp;author=CA+Hsiung&amp;author=Q+Lan&amp;author=YC+Hong&amp;author=CJ+Chen&amp;author=HD+Hosgood&amp;publication_year=2010&amp;pages=6&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Hubscher U, Maga G, Spadari S. Eukaryotic DNA polymerases. <span><span class="ref-journal">Annu Rev Biochem. </span>2002;<span class="ref-vol">71</span>:133–163.</span> [<a href="/pubmed/12045093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Biochem&amp;title=Eukaryotic+DNA+polymerases&amp;author=U+Hubscher&amp;author=G+Maga&amp;author=S+Spadari&amp;volume=71&amp;publication_year=2002&amp;pages=133-163&amp;pmid=12045093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Hutter CM, Slattery ML, Duggan DJ, Muehling J, Curtin K, Hsu L, Beresford SA, Rajkovic A, Sarto GE, Marshall JR, Hammad N, Wallace R, Makar KW, Prentice RL, Caan BJ, Potter JD, Peters U. Characterization of the association between 8q24 and colon cancer: gene-environment exploration and meta-analysis. <span><span class="ref-journal">BMC Cancer. </span>2010;<span class="ref-vol">10</span>:670.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017062/">PMC free article</a>]</span> [<a href="/pubmed/21129217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Cancer&amp;title=Characterization+of+the+association+between+8q24+and+colon+cancer:+gene-environment+exploration+and+meta-analysis&amp;author=CM+Hutter&amp;author=ML+Slattery&amp;author=DJ+Duggan&amp;author=J+Muehling&amp;author=K+Curtin&amp;volume=10&amp;publication_year=2010&amp;pages=670&amp;pmid=21129217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">International HapMap Consortium. A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">Ioannidis JP, Patsopoulos NA, Evangelou E. Heterogeneity in meta-analyses of genome-wide association investigations. <span><span class="ref-journal">PLoS ONE. </span>2007;<span class="ref-vol">2</span>:e841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950790/">PMC free article</a>]</span> [<a href="/pubmed/17786212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Heterogeneity+in+meta-analyses+of+genome-wide+association+investigations&amp;author=JP+Ioannidis&amp;author=NA+Patsopoulos&amp;author=E+Evangelou&amp;volume=2&amp;publication_year=2007&amp;pages=e841&amp;pmid=17786212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Johnatty SE, Beesley J, Chen X, Macgregor S, Duffy DL, Spurdle AB, deFazio A, Gava N, Webb PM, Rossing MA, Doherty JA, Goodman MT, Lurie G, Thompson PJ, Wilkens LR, Ness RB, Moysich KB, Chang-Claude J, Wang-Gohrke S, Cramer DW, Terry KL, Hankinson SE, Tworoger SS, Garcia-Closas M, Yang H, Lissowska J, Chanock SJ, Pharoah PD, Song H, Whitemore AS, Pearce CL, Stram DO, Wu AH, Pike MC, Gayther SA, Ramus SJ, Menon U, Gentry-Maharaj A, Anton-Culver H, Ziogas A, Hogdall E, Kjaer SK, Hogdall C, Berchuck A, Schildkraut JM, Iversen ES, Moorman PG, Phelan CM, Sellers TA, Cunningham JM, Vierkant RA, Rider DN, Goode EL, Haviv I, Chenevix-Trench G. Evaluation of candidate stromal epithelial cross-talk genes identifies association between risk of serous ovarian cancer and TERT, a cancer susceptibility “hot-spot” <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1001016.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900295/">PMC free article</a>]</span> [<a href="/pubmed/20628624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Evaluation+of+candidate+stromal+epithelial+cross-talk+genes+identifies+association+between+risk+of+serous+ovarian+cancer+and+TERT,+a+cancer+susceptibility+“hot-spot”&amp;author=SE+Johnatty&amp;author=J+Beesley&amp;author=X+Chen&amp;author=S+Macgregor&amp;author=DL+Duffy&amp;volume=6&amp;publication_year=2010&amp;pages=e1001016&amp;pmid=20628624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Kraft P, Zeggini E, Ioannidis JP. Replication in genome-wide association studies. <span><span class="ref-journal">Stat Sci. </span>2009;<span class="ref-vol">24</span>:561–573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865141/">PMC free article</a>]</span> [<a href="/pubmed/20454541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Sci&amp;title=Replication+in+genome-wide+association+studies&amp;author=P+Kraft&amp;author=E+Zeggini&amp;author=JP+Ioannidis&amp;volume=24&amp;publication_year=2009&amp;pages=561-573&amp;pmid=20454541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R31"><span class="element-citation">Landi MT, Chatterjee N, Yu K, Goldin LR, Goldstein AM, Rotunno M, Mirabello L, Jacobs K, Wheeler W, Yeager M, Bergen AW, Li Q, Consonni D, Pesatori AC, Wacholder S, Thun M, Diver R, Oken M, Virtamo J, Albanes D, Wang Z, Burdette L, Doheny KF, Pugh EW, Laurie C, Brennan P, Hung R, Gaborieau V, McKay JD, Lathrop M, McLaughlin J, Wang Y, Tsao MS, Spitz MR, Wang Y, Krokan H, Vatten L, Skorpen F, Arnesen E, Benhamou S, Bouchard C, Metsapalu A, Vooder T, Nelis M, Valk K, Field JK, Chen C, Goodman G, Sulem P, Thorleifsson G, Rafnar T, Eisen T, Sauter W, Rosenberger A, Bickeboller H, Risch A, Chang-Claude J, Wichmann HE, Stefansson K, Houlston R, Amos CI, Fraumeni JF, Jr, Savage SA, Bertazzi PA, Tucker MA, Chanock S, Caporaso NE. A genome-wide association study of lung cancer identifies a region of chromosome 5p15 associated with risk for adenocarcinoma. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">85</span>:679–691.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775843/">PMC free article</a>]</span> [<a href="/pubmed/19836008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+genome-wide+association+study+of+lung+cancer+identifies+a+region+of+chromosome+5p15+associated+with+risk+for+adenocarcinoma&amp;author=MT+Landi&amp;author=N+Chatterjee&amp;author=K+Yu&amp;author=LR+Goldin&amp;author=AM+Goldstein&amp;volume=85&amp;publication_year=2009&amp;pages=679-691&amp;pmid=19836008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R32"><span class="element-citation">Lango AH, Estrada K, Lettre G, Berndt SI, Weedon MN, Rivadeneira F, Willer CJ, Jackson AU, Vedantam S, Raychaudhuri S, Ferreira T, Wood AR, Weyant RJ, Segre AV, Speliotes EK, Wheeler E, Soranzo N, Park JH, Yang J, Gudbjartsson D, Heard-Costa NL, Randall JC, Qi L, Vernon SA, Magi R, Pastinen T, Liang L, Heid IM, Luan J, Thorleifsson G, Winkler TW, Goddard ME, Sin LK, Palmer C, Workalemahu T, Aulchenko YS, Johansson A, Carola ZM, Feitosa MF, Esko T, Johnson T, Ketkar S, Kraft P, Mangino M, Prokopenko I, Absher D, Albrecht E, Ernst F, Glazer NL, Hayward C, Hottenga JJ, Jacobs KB, Knowles JW, Kutalik Z, Monda KL, Polasek O, Preuss M, Rayner NW, Robertson NR, Steinthorsdottir V, Tyrer JP, Voight BF, Wiklund F, Xu J, Hua ZJ, Nyholt DR, Pellikka N, Perola M, Perry JR, Surakka I, Tammesoo ML, Altmaier EL, Amin N, Aspelund T, Bhangale T, Boucher G, Chasman DI, Chen C, Coin L, Cooper MN, Dixon AL, Gibson Q, Grundberg E, Hao K, Juhani JM, Kaplan LM, Kettunen J, Konig IR, Kwan T, Lawrence RW, Levinson DF, Lorentzon M, McKnight B, Morris AP, Muller M, Suh NJ, Purcell S, Rafelt S, Salem RM, Salvi E, Sanna S, Shi J, Sovio U, Thompson JR, Turchin MC, Vandenput L, Verlaan DJ, Vitart V, White CC, Ziegler A, Almgren P, Balmforth AJ, Campbell H, Citterio L, De GA, Dominiczak A, Duan J, Elliott P, Elosua R, Eriksson JG, Freimer NB, Geus EJ, Glorioso N, Haiqing S, Hartikainen AL, Havulinna AS, Hicks AA, Hui J, Igl W, Illig T, Jula A, Kajantie E, Kilpelainen TO, Koiranen M, Kolcic I, Koskinen S, Kovacs P, Laitinen J, Liu J, Lokki ML, Marusic A, Maschio A, Meitinger T, Mulas A, Pare G, Parker AN, Peden JF, Petersmann A, Pichler I, Pietilainen KH, Pouta A, Ridderstrale M, Rotter JI, Sambrook JG, Sanders AR, Oliver SC, Sinisalo J, Smit JH, Stringham HM, Bragi WG, Widen E, Wild SH, Willemsen G, Zagato L, Zgaga L, Zitting P, Alavere H, Farrall M, McArdle WL, Nelis M, Peters MJ, Ripatti S, van Meurs JB, Aben KK, Ardlie KG, Beckmann JS, Beilby JP, Bergman RN, Bergmann S, Collins FS, Cusi D, den HM, Eiriksdottir G, Gejman PV, Hall AS, Hamsten A, Huikuri HV, Iribarren C, Kahonen M, Kaprio J, Kathiresan S, Kiemeney L, Kocher T, Launer LJ, Lehtimaki T, Melander O, Mosley TH, Jr, Musk AW, Nieminen MS, O’Donnell CJ, Ohlsson C, Oostra B, Palmer LJ, Raitakari O, Ridker PM, Rioux JD, Rissanen A, Rivolta C, Schunkert H, Shuldiner AR, Siscovick DS, Stumvoll M, Tonjes A, Tuomilehto J, van Ommen GJ, Viikari J, Heath AC, Martin NG, Montgomery GW, Province MA, Kayser M, Arnold AM, Atwood LD, Boerwinkle E, Chanock SJ, Deloukas P, Gieger C, Gronberg H, Hall P, Hattersley AT, Hengstenberg C, Hoffman W, Mark LG, Salomaa V, Schreiber S, Uda M, Waterworth D, Wright AF, Assimes TL, Barroso I, Hofman A, Mohlke KL, Boomsma DI, Caulfield MJ, Adrienne CL, Erdmann J, Fox CS, Gudnason V, Gyllensten U, Harris TB, Hayes RB, Jarvelin MR, Mooser V, Munroe PB, Ouwehand WH. Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=AH+Lango&amp;author=K+Estrada&amp;author=G+Lettre&amp;author=SI+Berndt&amp;author=MN+Weedon&amp;volume=467&amp;publication_year=2010&amp;pages=832-838&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R33"><span class="element-citation">Luna A, Nicodemus KK. snp.plotter: an R-based SNP/haplotype association and linkage disequilibrium plotting package. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:774–776.</span> [<a href="/pubmed/17234637" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=snp.plotter:+an+R-based+SNP/haplotype+association+and+linkage+disequilibrium+plotting+package&amp;author=A+Luna&amp;author=KK+Nicodemus&amp;volume=23&amp;publication_year=2007&amp;pages=774-776&amp;pmid=17234637&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R34"><span class="element-citation">Manolio TA, Collins FS, Cox NJ, Goldstein DB, Hindorff LA, Hunter DJ, McCarthy MI, Ramos EM, Cardon LR, Chakravarti A, Cho JH, Guttmacher AE, Kong A, Kruglyak L, Mardis E, Rotimi CN, Slatkin M, Valle D, Whittemore AS, Boehnke M, Clark AG, Eichler EE, Gibson G, Haines JL, Mackay TF, McCarroll SA, Visscher PM. Finding the missing heritability of complex diseases. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:747–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2831613/">PMC free article</a>]</span> [<a href="/pubmed/19812666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Finding+the+missing+heritability+of+complex+diseases&amp;author=TA+Manolio&amp;author=FS+Collins&amp;author=NJ+Cox&amp;author=DB+Goldstein&amp;author=LA+Hindorff&amp;volume=461&amp;publication_year=2009&amp;pages=747-753&amp;pmid=19812666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R35"><span class="element-citation">Massague J. How cells read TGF-beta signals. <span><span class="ref-journal">Nat Rev Mol Cell Biol. </span>2000;<span class="ref-vol">1</span>:169–178.</span> [<a href="/pubmed/11252892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Mol+Cell+Biol&amp;title=How+cells+read+TGF-beta+signals&amp;author=J+Massague&amp;volume=1&amp;publication_year=2000&amp;pages=169-178&amp;pmid=11252892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R36"><span class="element-citation">McKay JD, Hung RJ, Gaborieau V, Boffetta P, Chabrier A, Byrnes G, Zaridze D, Mukeria A, Szeszenia-Dabrowska N, Lissowska J, Rudnai P, Fabianova E, Mates D, Bencko V, Foretova L, Janout V, McLaughlin J, Shepherd F, Montpetit A, Narod S, Krokan HE, Skorpen F, Elvestad MB, Vatten L, Njolstad I, Axelsson T, Chen C, Goodman G, Barnett M, Loomis MM, Lubinski J, Matyjasik J, Lener M, Oszutowska D, Field J, Liloglou T, Xinarianos G, Cassidy A, Vineis P, Clavel-Chapelon F, Palli D, Tumino R, Krogh V, Panico S, Gonzalez CA, Ramon QJ, Martinez C, Navarro C, Ardanaz E, Larranaga N, Kham KT, Key T, Bueno-de-Mesquita HB, Peeters PH, Trichopoulou A, Linseisen J, Boeing H, Hallmans G, Overvad K, Tjonneland A, Kumle M, Riboli E, Zelenika D, Boland A, Delepine M, Foglio M, Lechner D, Matsuda F, Blanche H, Gut I, Heath S, Lathrop M, Brennan P. Lung cancer susceptibility locus at 5p15.33. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1404–1406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2748187/">PMC free article</a>]</span> [<a href="/pubmed/18978790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Lung+cancer+susceptibility+locus+at+5p15.33&amp;author=JD+McKay&amp;author=RJ+Hung&amp;author=V+Gaborieau&amp;author=P+Boffetta&amp;author=A+Chabrier&amp;volume=40&amp;publication_year=2008&amp;pages=1404-1406&amp;pmid=18978790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R37"><span class="element-citation">Miki D, Kubo M, Takahashi A, Yoon KA, Kim J, Lee GK, Zo JI, Lee JS, Hosono N, Morizono T, Tsunoda T, Kamatani N, Chayama K, Takahashi T, Inazawa J, Nakamura Y, Daigo Y. Variation in TP63 is associated with lung adenocarcinoma susceptibility in Japanese and Korean populations. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:893–896.</span> [<a href="/pubmed/20871597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variation+in+TP63+is+associated+with+lung+adenocarcinoma+susceptibility+in+Japanese+and+Korean+populations&amp;author=D+Miki&amp;author=M+Kubo&amp;author=A+Takahashi&amp;author=KA+Yoon&amp;author=J+Kim&amp;volume=42&amp;publication_year=2010&amp;pages=893-896&amp;pmid=20871597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R38"><span class="element-citation">Minelli C, Thompson JR, Abrams KR, Thakkinstian A, Attia J. The choice of a genetic model in the meta-analysis of molecular association studies. <span><span class="ref-journal">Int J Epidemiol. </span>2005;<span class="ref-vol">34</span>:1319–1328.</span> [<a href="/pubmed/16115824" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=The+choice+of+a+genetic+model+in+the+meta-analysis+of+molecular+association+studies&amp;author=C+Minelli&amp;author=JR+Thompson&amp;author=KR+Abrams&amp;author=A+Thakkinstian&amp;author=J+Attia&amp;volume=34&amp;publication_year=2005&amp;pages=1319-1328&amp;pmid=16115824&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R39"><span class="element-citation">Park JH, Wacholder S, Gail MH, Peters U, Jacobs KB, Chanock SJ, Chatterjee N. Estimation of effect size distribution from genome-wide association studies and implications for future discoveries. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:570–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4615599/">PMC free article</a>]</span> [<a href="/pubmed/20562874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Estimation+of+effect+size+distribution+from+genome-wide+association+studies+and+implications+for+future+discoveries&amp;author=JH+Park&amp;author=S+Wacholder&amp;author=MH+Gail&amp;author=U+Peters&amp;author=KB+Jacobs&amp;volume=42&amp;publication_year=2010&amp;pages=570-575&amp;pmid=20562874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R40"><span class="element-citation">Pe’er I, Yelensk R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genetic Epidemiology. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetic+Epidemiology&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensk&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R41"><span class="element-citation">Petersen GM, Amundadottir L, Fuchs CS, Kraft P, Stolzenberg-Solomon RZ, Jacobs KB, Arslan AA, Bueno-de-Mesquita HB, Gallinger S, Gross M, Helzlsouer K, Holly EA, Jacobs EJ, Klein AP, Lacroix A, Li D, Mandelson MT, Olson SH, Risch HA, Zheng W, Albanes D, Bamlet WR, Berg CD, Boutron-Ruault MC, Buring JE, Bracci PM, Canzian F, Clipp S, Cotterchio M, de AM, Duell EJ, Gaziano JM, Giovannucci EL, Goggins M, Hallmans G, Hankinson SE, Hassan M, Howard B, Hunter DJ, Hutchinson A, Jenab M, Kaaks R, Kooperberg C, Krogh V, Kurtz RC, Lynch SM, McWilliams RR, Mendelsohn JB, Michaud DS, Parikh H, Patel AV, Peeters PH, Rajkovic A, Riboli E, Rodriguez L, Seminara D, Shu XO, Thomas G, Tjonneland A, Tobias GS, Trichopoulos D, Van den Eeden SK, Virtamo J, Wactawski-Wende J, Wang Z, Wolpin BM, Yu H, Yu K, Zeleniuch-Jacquotte A, Fraumeni JF, Jr, Hoover RN, Hartge P, Chanock SJ. A genome-wide association study identifies pancreatic cancer susceptibility loci on chromosomes 13q22.1, 1q32.1 and 5p15.33. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:224–228.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2853179/">PMC free article</a>]</span> [<a href="/pubmed/20101243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+pancreatic+cancer+susceptibility+loci+on+chromosomes+13q22.1,+1q32.1+and+5p15.33&amp;author=GM+Petersen&amp;author=L+Amundadottir&amp;author=CS+Fuchs&amp;author=P+Kraft&amp;author=RZ+Stolzenberg-Solomon&amp;volume=42&amp;publication_year=2010&amp;pages=224-228&amp;pmid=20101243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R42"><span class="element-citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R43"><span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R44"><span class="element-citation">Purcell. <span><span class="ref-journal">PLINK. </span>2011</span>  <a href="http://pngu.mgh.harvard.edu/purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/purcell/plink/</a></span></li><li id="R45"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R46"><span class="element-citation">R Development Core Team. <span class="ref-journal">R: A language and environment for statistical computing.</span> R Foundation for Statistical Computing; Vienna, Austria: 2011.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+language+and+environment+for+statistical+computing&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R47"><span class="element-citation">Rafnar T, Sulem P, Stacey SN, Geller F, Gudmundsson J, Sigurdsson A, Jakobsdottir M, Helgadottir H, Thorlacius S, Aben KK, Blondal T, Thorgeirsson TE, Thorleifsson G, Kristjansson K, Thorisdottir K, Ragnarsson R, Sigurgeirsson B, Skuladottir H, Gudbjartsson T, Isaksson HJ, Einarsson GV, Benediktsdottir KR, Agnarsson BA, Olafsson K, Salvarsdottir A, Bjarnason H, Asgeirsdottir M, Kristinsson KT, Matthiasdottir S, Sveinsdottir SG, Polidoro S, Hoiom V, Botella-Estrada R, Hemminki K, Rudnai P, Bishop DT, Campagna M, Kellen E, Zeegers MP, de VP, Ferrer A, Isla D, Vidal MJ, Andres R, Saez B, Juberias P, Banzo J, Navarrete S, Tres A, Kan D, Lindblom A, Gurzau E, Koppova K, de VF, Schalken JA, van der Heijden HF, Smit HJ, Termeer RA, Oosterwijk E, van HO, Nagore E, Porru S, Steineck G, Hansson J, Buntinx F, Catalona WJ, Matullo G, Vineis P, Kiltie AE, Mayordomo JI, Kumar R, Kiemeney LA, Frigge ML, Jonsson T, Saemundsson H, Barkardottir RB, Jonsson E, Jonsson S, Olafsson JH, Gulcher JR, Masson G, Gudbjartsson DF, Kong A, Thorsteinsdottir U, Stefansson K. Sequence variants at the TERT-CLPTM1L locus associate with many cancer types. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:221–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4525478/">PMC free article</a>]</span> [<a href="/pubmed/19151717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+at+the+TERT-CLPTM1L+locus+associate+with+many+cancer+types&amp;author=T+Rafnar&amp;author=P+Sulem&amp;author=SN+Stacey&amp;author=F+Geller&amp;author=J+Gudmundsson&amp;volume=41&amp;publication_year=2009&amp;pages=221-227&amp;pmid=19151717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R48"><span class="element-citation">Ries Melbert D, Krapcho M, Mariotto A, Miller BA, Feuer EJ, Clegg L, Horner MJ, Howlader N, Eisner MP, Reichman M, Edwards BK.  <span class="ref-journal">SEER Cancer Statistics Review, 1975–2004.</span> National Cancer Institute; Bethesda, MD: 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=SEER+Cancer+Statistics+Review,+1975–2004&amp;author=Melbert+D+Ries&amp;author=M+Krapcho&amp;author=A+Mariotto&amp;author=BA+Miller&amp;author=EJ+Feuer&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R49"><span class="element-citation">Risch N, Merikangas K. The future of genetic studies of complex human diseases. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">273</span>:1516–1517.</span> [<a href="/pubmed/8801636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+future+of+genetic+studies+of+complex+human+diseases&amp;author=N+Risch&amp;author=K+Merikangas&amp;volume=273&amp;publication_year=1996&amp;pages=1516-1517&amp;pmid=8801636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R50"><span class="element-citation">Rosenbloom KR, Dreszer TR, Pheasant M, Barber GP, Meyer LR, Pohl A, Raney BJ, Wang T, Hinrichs AS, Zweig AS, Fujita PA, Learned K, Rhead B, Smith KE, Kuhn RM, Karolchik D, Haussler D, Kent WJ. ENCODE whole-genome data in the UCSC Genome Browser. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:D620–D625.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2808953/">PMC free article</a>]</span> [<a href="/pubmed/19920125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ENCODE+whole-genome+data+in+the+UCSC+Genome+Browser&amp;author=KR+Rosenbloom&amp;author=TR+Dreszer&amp;author=M+Pheasant&amp;author=GP+Barber&amp;author=LR+Meyer&amp;volume=38&amp;publication_year=2010&amp;pages=D620-D625&amp;pmid=19920125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R51"><span class="element-citation">Schreibman IR, Baker M, Amos C, McGarrity TJ. The hamartomatous polyposis syndromes: a clinical and molecular review. <span><span class="ref-journal">Am J Gastroenterol. </span>2005;<span class="ref-vol">100</span>:476–490.</span> [<a href="/pubmed/15667510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Gastroenterol&amp;title=The+hamartomatous+polyposis+syndromes:+a+clinical+and+molecular+review&amp;author=IR+Schreibman&amp;author=M+Baker&amp;author=C+Amos&amp;author=TJ+McGarrity&amp;volume=100&amp;publication_year=2005&amp;pages=476-490&amp;pmid=15667510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R52"><span class="element-citation">Shete S, Hosking FJ, Robertson LB, Dobbins SE, Sanson M, Malmer B, Simon M, Marie Y, Boisselier B, Delattre JY, Hoang-Xuan K, El HS, Idbaih A, Zelenika D, Andersson U, Henriksson R, Bergenheim AT, Feychting M, Lonn S, Ahlbom A, Schramm J, Linnebank M, Hemminki K, Kumar R, Hepworth SJ, Price A, Armstrong G, Liu Y, Gu X, Yu R, Lau C, Schoemaker M, Muir K, Swerdlow A, Lathrop M, Bondy M, Houlston RS. Genome-wide association study identifies five susceptibility loci for glioma. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:899–904.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4501476/">PMC free article</a>]</span> [<a href="/pubmed/19578367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+susceptibility+loci+for+glioma&amp;author=S+Shete&amp;author=FJ+Hosking&amp;author=LB+Robertson&amp;author=SE+Dobbins&amp;author=M+Sanson&amp;volume=41&amp;publication_year=2009&amp;pages=899-904&amp;pmid=19578367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R53"><span class="element-citation">Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier LW, Richards S, Weinstock GM, Wilson RK, Gibbs RA, Kent WJ, Miller W, Haussler D. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:1034–1050.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182216/">PMC free article</a>]</span> [<a href="/pubmed/16024819" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Evolutionarily+conserved+elements+in+vertebrate,+insect,+worm,+and+yeast+genomes&amp;author=A+Siepel&amp;author=G+Bejerano&amp;author=JS+Pedersen&amp;author=AS+Hinrichs&amp;author=M+Hou&amp;volume=15&amp;publication_year=2005&amp;pages=1034-1050&amp;pmid=16024819&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R54"><span class="element-citation">Stacey SN, Sulem P, Masson G, Gudjonsson SA, Thorleifsson G, Jakobsdottir M, Sigurdsson A, Gudbjartsson DF, Sigurgeirsson B, Benediktsdottir KR, Thorisdottir K, Ragnarsson R, Scherer D, Hemminki K, Rudnai P, Gurzau E, Koppova K, Botella-Estrada R, Soriano V, Juberias P, Saez B, Gilaberte Y, Fuentelsaz V, Corredera C, Grasa M, Hoiom V, Lindblom A, Bonenkamp JJ, van Rossum MM, Aben KK, de VE, Santinami M, Di Mauro MG, Maurichi A, Wendt J, Hochleitner P, Pehamberger H, Gudmundsson J, Magnusdottir DN, Gretarsdottir S, Holm H, Steinthorsdottir V, Frigge ML, Blondal T, Saemundsdottir J, Bjarnason H, Kristjansson K, Bjornsdottir G, Okamoto I, Rivoltini L, Rodolfo M, Kiemeney LA, Hansson J, Nagore E, Mayordomo JI, Kumar R, Karagas MR, Nelson HH, Gulcher JR, Rafnar T, Thorsteinsdottir U, Olafsson JH, Kong A, Stefansson K. New common variants affecting susceptibility to basal cell carcinoma. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:909–914.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2973331/">PMC free article</a>]</span> [<a href="/pubmed/19578363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=New+common+variants+affecting+susceptibility+to+basal+cell+carcinoma&amp;author=SN+Stacey&amp;author=P+Sulem&amp;author=G+Masson&amp;author=SA+Gudjonsson&amp;author=G+Thorleifsson&amp;volume=41&amp;publication_year=2009&amp;pages=909-914&amp;pmid=19578363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R55"><span class="element-citation">StataCorp. <span class="ref-journal">Stata Statistical Software: Release 11.</span> StataCorp LP: College Station, TX; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Stata+Statistical+Software:+Release+11&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R56"><span class="element-citation">Tenesa A, Dunlop MG. New insights into the aetiology of colorectal cancer from genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2009;<span class="ref-vol">10</span>:353–358.</span> [<a href="/pubmed/19434079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=New+insights+into+the+aetiology+of+colorectal+cancer+from+genome-wide+association+studies&amp;author=A+Tenesa&amp;author=MG+Dunlop&amp;volume=10&amp;publication_year=2009&amp;pages=353-358&amp;pmid=19434079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R57"><span class="element-citation">Tenesa A, Farrington SM, Prendergast JG, Porteous ME, Walker M, Haq N, Barnetson RA, Theodoratou E, Cetnarskyj R, Cartwright N, Semple C, Clark AJ, Reid FJ, Smith LA, Kavoussanakis K, Koessler T, Pharoah PD, Buch S, Schafmayer C, Tepel J, Schreiber S, Volzke H, Schmidt CO, Hampe J, Chang-Claude J, Hoffmeister M, Brenner H, Wilkening S, Canzian F, Capella G, Moreno V, Deary IJ, Starr JM, Tomlinson IP, Kemp Z, Howarth K, Carvajal-Carmona L, Webb E, Broderick P, Vijayakrishnan J, Houlston RS, Rennert G, Ballinger D, Rozek L, Gruber SB, Matsuda K, Kidokoro T, Nakamura Y, Zanke BW, Greenwood CM, Rangrej J, Kustra R, Montpetit A, Hudson TJ, Gallinger S, Campbell H, Dunlop MG. Genome-wide association scan identifies a colorectal cancer susceptibility locus on 11q23 and replicates risk loci at 8q24 and 18q21. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:631–637.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2778004/">PMC free article</a>]</span> [<a href="/pubmed/18372901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+11q23+and+replicates+risk+loci+at+8q24+and+18q21&amp;author=A+Tenesa&amp;author=SM+Farrington&amp;author=JG+Prendergast&amp;author=ME+Porteous&amp;author=M+Walker&amp;volume=40&amp;publication_year=2008&amp;pages=631-637&amp;pmid=18372901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R58"><span class="element-citation">The Institute of Cancer Research. <span class="ref-journal">Software and Databases, Molecular and Population Genetics Team, Section of Cancer Genetics Phase 1 GWAS data.</span> 2008.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Software+and+Databases,+Molecular+and+Population+Genetics+Team,+Section+of+Cancer+Genetics+Phase+1+GWAS+data&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R59"><span class="element-citation">Timm DE, Liu J, Baker LJ, Harris RA. Crystal structure of thiamin pyrophosphokinase. <span><span class="ref-journal">J Mol Biol. </span>2001;<span class="ref-vol">310</span>:195–204.</span> [<a href="/pubmed/11419946" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Biol&amp;title=Crystal+structure+of+thiamin+pyrophosphokinase&amp;author=DE+Timm&amp;author=J+Liu&amp;author=LJ+Baker&amp;author=RA+Harris&amp;volume=310&amp;publication_year=2001&amp;pages=195-204&amp;pmid=11419946&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R60"><span class="element-citation">Tomlinson I, Webb E, Carvajal-Carmona L, Broderick P, Kemp Z, Spain S, Penegar S, Chandler I, Gorman M, Wood W, Barclay E, Lubbe S, Martin L, Sellick G, Jaeger E, Hubner R, Wild R, Rowan A, Fielding S, Howarth K, Silver A, Atkin W, Muir K, Logan R, Kerr D, Johnstone E, Sieber O, Gray R, Thomas H, Peto J, Cazier JB, Houlston R. A genome-wide association scan of tag SNPs identifies a susceptibility variant for colorectal cancer at 8q24.21. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:984–988.</span> [<a href="/pubmed/17618284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+variant+for+colorectal+cancer+at+8q24.21&amp;author=I+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;author=P+Broderick&amp;author=Z+Kemp&amp;volume=39&amp;publication_year=2007&amp;pages=984-988&amp;pmid=17618284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R61"><span class="element-citation">Tomlinson IP, Webb E, Carvajal-Carmona L, Broderick P, Howarth K, Pittman AM, Spain S, Lubbe S, Walther A, Sullivan K, Jaeger E, Fielding S, Rowan A, Vijayakrishnan J, Domingo E, Chandler I, Kemp Z, Qureshi M, Farrington SM, Tenesa A, Prendergast JG, Barnetson RA, Penegar S, Barclay E, Wood W, Martin L, Gorman M, Thomas H, Peto J, Bishop DT, Gray R, Maher ER, Lucassen A, Kerr D, Evans DG, Schafmayer C, Buch S, Volzke H, Hampe J, Schreiber S, John U, Koessler T, Pharoah P, van WT, Morreau H, Wijnen JT, Hopper JL, Southey MC, Giles GG, Severi G, Castellvi-Bel S, Ruiz-Ponte C, Carracedo A, Castells A, Forsti A, Hemminki K, Vodicka P, Naccarati A, Lipton L, Ho JW, Cheng KK, Sham PC, Luk J, Agundez JA, Ladero JM, de la HM, Caldes T, Niittymaki I, Tuupanen S, Karhu A, Aaltonen L, Cazier JB, Campbell H, Dunlop MG, Houlston RS. A genome-wide association study identifies colorectal cancer susceptibility loci on chromosomes 10p14 and 8q23.3. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:623–630.</span> [<a href="/pubmed/18372905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+colorectal+cancer+susceptibility+loci+on+chromosomes+10p14+and+8q23.3&amp;author=IP+Tomlinson&amp;author=E+Webb&amp;author=L+Carvajal-Carmona&amp;author=P+Broderick&amp;author=K+Howarth&amp;volume=40&amp;publication_year=2008&amp;pages=623-630&amp;pmid=18372905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R62"><span class="element-citation">Turnbull C, Rapley EA, Seal S, Pernet D, Renwick A, Hughes D, Ricketts M, Linger R, Nsengimana J, Deloukas P, Huddart RA, Bishop DT, Easton DF, Stratton MR, Rahman N. Variants near DMRT1, TERT and ATF7IP are associated with testicular germ cell cancer. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:604–607.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773909/">PMC free article</a>]</span> [<a href="/pubmed/20543847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variants+near+DMRT1,+TERT+and+ATF7IP+are+associated+with+testicular+germ+cell+cancer&amp;author=C+Turnbull&amp;author=EA+Rapley&amp;author=S+Seal&amp;author=D+Pernet&amp;author=A+Renwick&amp;volume=42&amp;publication_year=2010&amp;pages=604-607&amp;pmid=20543847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R63"><span class="element-citation">Wang Y, Broderick P, Webb E, Wu X, Vijayakrishnan J, Matakidou A, Qureshi M, Dong Q, Gu X, Chen WV, Spitz MR, Eisen T, Amos CI, Houlston RS. Common 5p15.33 and 6p21.33 variants influence lung cancer risk. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1407–1409.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695928/">PMC free article</a>]</span> [<a href="/pubmed/18978787" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+5p15.33+and+6p21.33+variants+influence+lung+cancer+risk&amp;author=Y+Wang&amp;author=P+Broderick&amp;author=E+Webb&amp;author=X+Wu&amp;author=J+Vijayakrishnan&amp;volume=40&amp;publication_year=2008&amp;pages=1407-1409&amp;pmid=18978787&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R64"><span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R65"><span class="element-citation">Witte JS. Multiple prostate cancer risk variants on 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:579–580.</span> [<a href="/pubmed/17460686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Multiple+prostate+cancer+risk+variants+on+8q24&amp;author=JS+Witte&amp;volume=39&amp;publication_year=2007&amp;pages=579-580&amp;pmid=17460686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R66"><span class="element-citation">Wu KJ, Grandori C, Amacker M, Simon-Vermot N, Polack A, Lingner J, Dalla-Favera R. Direct activation of TERT transcription by c-MYC. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">21</span>:220–224.</span> [<a href="/pubmed/9988278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Direct+activation+of+TERT+transcription+by+c-MYC&amp;author=KJ+Wu&amp;author=C+Grandori&amp;author=M+Amacker&amp;author=N+Simon-Vermot&amp;author=A+Polack&amp;volume=21&amp;publication_year=1999&amp;pages=220-224&amp;pmid=9988278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R67"><span class="element-citation">Yamamoto K, Okamoto A, Isonishi S, Ochiai K, Ohtake Y. A novel gene, CRR9, which was up-regulated in CDDP-resistant ovarian tumor cell line, was associated with apoptosis. <span><span class="ref-journal">Biochem Biophys Res Commun. </span>2001;<span class="ref-vol">280</span>:1148–1154.</span> [<a href="/pubmed/11162647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem+Biophys+Res+Commun&amp;title=A+novel+gene,+CRR9,+which+was+up-regulated+in+CDDP-resistant+ovarian+tumor+cell+line,+was+associated+with+apoptosis&amp;author=K+Yamamoto&amp;author=A+Okamoto&amp;author=S+Isonishi&amp;author=K+Ochiai&amp;author=Y+Ohtake&amp;volume=280&amp;publication_year=2001&amp;pages=1148-1154&amp;pmid=11162647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R68"><span class="element-citation">Yeager M, Orr N, Hayes RB, Jacobs KB, Kraft P, Wacholder S, Minichiello MJ, Fearnhead P, Yu K, Chatterjee N, Wang Z, Welch R, Staats BJ, Calle EE, Feigelson HS, Thun MJ, Rodriguez C, Albanes D, Virtamo J, Weinstein S, Schumacher FR, Giovannucci E, Willett WC, Cancel-Tassin G, Cussenot O, Valeri A, Andriole GL, Gelmann EP, Tucker M, Gerhard DS, Fraumeni JF, Jr, Hoover R, Hunter DJ, Chanock SJ, Thomas G. Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M+Yeager&amp;author=N+Orr&amp;author=RB+Hayes&amp;author=KB+Jacobs&amp;author=P+Kraft&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R69"><span class="element-citation">Zanke BW, Greenwood CM, Rangrej J, Kustra R, Tenesa A, Farrington SM, Prendergast J, Olschwang S, Chiang T, Crowdy E, Ferretti V, Laflamme P, Sundararajan S, Roumy S, Olivier JF, Robidoux F, Sladek R, Montpetit A, Campbell P, Bezieau S, O’shea AM, Zogopoulos G, Cotterchio M, Newcomb P, McLaughlin J, Younghusband B, Green R, Green J, Porteous ME, Campbell H, Blanche H, Sahbatou M, Tubacher E, Bonaiti-Pellie C, Buecher B, Riboli E, Kury S, Chanock SJ, Potter J, Thomas G, Gallinger S, Hudson TJ, Dunlop MG. Genome-wide association scan identifies a colorectal cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:989–994.</span> [<a href="/pubmed/17618283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+scan+identifies+a+colorectal+cancer+susceptibility+locus+on+chromosome+8q24&amp;author=BW+Zanke&amp;author=CM+Greenwood&amp;author=J+Rangrej&amp;author=R+Kustra&amp;author=A+Tenesa&amp;volume=39&amp;publication_year=2007&amp;pages=989-994&amp;pmid=17618283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R70"><span class="element-citation">Zeggini E, Ioannidis JP. Meta-analysis in genome-wide association studies. <span><span class="ref-journal">Pharmacogenomics. </span>2009;<span class="ref-vol">10</span>:191–201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695132/">PMC free article</a>]</span> [<a href="/pubmed/19207020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Meta-analysis+in+genome-wide+association+studies&amp;author=E+Zeggini&amp;author=JP+Ioannidis&amp;volume=10&amp;publication_year=2009&amp;pages=191-201&amp;pmid=19207020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3257356&amp;issue-id=204409&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140093441399680title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Results</a></li><li><a href="#S3title">Discussion</a></li><li><a href="#S4title">Materials and Methods</a></li><li><a href="#S25title">Supplementary Material</a></li><li><a href="#S22title">Acknowledgments</a></li><li><a href="#idm140093438132240title">Reference List</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3257356/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3257356/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3257356/pdf/nihms324653.pdf">PDF (448K)</a></li> | <li><a href="#" data-citationid="PMC3257356" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3257356%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3257356%2F&amp;text=Meta-analysis%20of%20New%20Genome-wide%20Association%20Studies%20of%20Colorectal%20Cancer%20Risk" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3257356%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21761138" data-aiid="3257356" data-aid="3257356" data-iid="204409" data-domainid="319" data-domain="nihpa" data-accid="PMC3257356" data-md5="c5d170bc6adfade73ca5cec633052629"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:25:04-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6573DD52E1210000000001A60110&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3257356%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3257356/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>