<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3110001/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies nidogen 1 (NID1) as a susceptibility locus to cutaneous nevi and melanoma risk</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3110001/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study identifies nidogen 1 (NID1) as a susceptibility locus to cutaneous nevi and melanoma risk"><meta name="citation_authors" content="Hongmei Nan, Mousheng Xu, Jiangwen Zhang, Mingfeng Zhang, Peter Kraft, Abrar A. Qureshi, Constance Chen, Qun Guo, Frank B. Hu, Eric B. Rimm, Gary Curhan, Yiqing Song, Christopher I. Amos, Li-E Wang, Jeffrey E. Lee, Qingyi Wei, David J. Hunter, Jiali Han"><meta name="citation_date" content="1 July 2011"><meta name="citation_issue" content="13"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="2673"><meta name="citation_doi" content="10.1093/hmg/ddr154"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3110001/?report=abstract"><meta name="citation_pmid" content="21478494"><meta name="DC.Title" content="Genome-wide association study identifies nidogen 1 (NID1) as a susceptibility locus to cutaneous nevi and melanoma risk"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Hongmei Nan"><meta name="DC.Contributor" content="Mousheng Xu"><meta name="DC.Contributor" content="Jiangwen Zhang"><meta name="DC.Contributor" content="Mingfeng Zhang"><meta name="DC.Contributor" content="Peter Kraft"><meta name="DC.Contributor" content="Abrar A. Qureshi"><meta name="DC.Contributor" content="Constance Chen"><meta name="DC.Contributor" content="Qun Guo"><meta name="DC.Contributor" content="Frank B. Hu"><meta name="DC.Contributor" content="Eric B. Rimm"><meta name="DC.Contributor" content="Gary Curhan"><meta name="DC.Contributor" content="Yiqing Song"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Li-E Wang"><meta name="DC.Contributor" content="Jeffrey E. Lee"><meta name="DC.Contributor" content="Qingyi Wei"><meta name="DC.Contributor" content="David J. Hunter"><meta name="DC.Contributor" content="Jiali Han"><meta name="DC.Date" content="2011 Jul 1"><meta name="DC.Identifier" content="10.1093/hmg/ddr154"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies nidogen 1 (NID1) as a susceptibility locus to cutaneous nevi and melanoma risk"><meta property="og:type" content="article"><meta property="og:description" content="We conducted a genome-wide association study on the number of melanocytic nevi reported by 9136 individuals of European ancestry, with follow-up replication in 3581 individuals. We identified the nidogen 1 (NID1) gene on 1q42 associated with nevus count ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3110001/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3110001/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3110001/pdf/ddr154.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BB877DD5254510000000002F001C3.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3110001%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3110001</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2011 Jul 1; 20(13): 2673–2679. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Apr 9. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddr154" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddr154</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3110001</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21478494">21478494</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies nidogen 1 (<em>NID1</em>) as a susceptibility locus to cutaneous nevi and melanoma risk</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Nan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501722313104" co-class="co-affbox">Hongmei Nan</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Xu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501798583072" co-class="co-affbox">Mousheng Xu</a>,<sup>4,</sup><sup>†</sup> <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501723359504" co-class="co-affbox">Jiangwen Zhang</a>,<sup>6</sup> <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501715878224" co-class="co-affbox">Mingfeng Zhang</a>,<sup>2</sup> <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501715876000" co-class="co-affbox">Peter Kraft</a>,<sup>4</sup> <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501725006192" co-class="co-affbox">Abrar A. Qureshi</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Chen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501725003232" co-class="co-affbox">Constance Chen</a>,<sup>4</sup> <a href="/pubmed/?term=Guo%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501761208384" co-class="co-affbox">Qun Guo</a>,<sup>1</sup> <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501761206160" co-class="co-affbox">Frank B. Hu</a>,<sup>1,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Rimm%20EB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501725057568" co-class="co-affbox">Eric B. Rimm</a>,<sup>1,</sup><sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Curhan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501722516720" co-class="co-affbox">Gary Curhan</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Song%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501723958368" co-class="co-affbox">Yiqing Song</a>,<sup>3</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501723956144" co-class="co-affbox">Christopher I. Amos</a>,<sup>7</sup> <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501759615856" co-class="co-affbox">Li-E Wang</a>,<sup>7</sup> <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501759613632" co-class="co-affbox">Jeffrey E. Lee</a>,<sup>8</sup> <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501717889872" co-class="co-affbox">Qingyi Wei</a>,<sup>7</sup> <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501717887648" co-class="co-affbox">David J. Hunter</a>,<sup>1,</sup><sup>4,</sup><sup>5</sup> and  <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494" class="affpopup" co-rid="_co_idm140501724946896" co-class="co-affbox">Jiali Han</a><sup>1,</sup><sup>2,</sup><sup>4,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140501722313104"><h3 class="no_margin">Hongmei Nan</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><div>Find articles by <a href="/pubmed/?term=Nan%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Hongmei Nan</a></div></div><div id="_co_idm140501798583072"><h3 class="no_margin">Mousheng Xu</h3><p><sup>4</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Xu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Mousheng Xu</a></div></div><div id="_co_idm140501723359504"><h3 class="no_margin">Jiangwen Zhang</h3><p><sup>6</sup>FAS Research Computing, Harvard University, Cambridge, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Zhang%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Jiangwen Zhang</a></div></div><div id="_co_idm140501715878224"><h3 class="no_margin">Mingfeng Zhang</h3><p><sup>2</sup>Clinical Research Program, Department of Dermatology,  and</p><div>Find articles by <a href="/pubmed/?term=Zhang%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Mingfeng Zhang</a></div></div><div id="_co_idm140501715876000"><h3 class="no_margin">Peter Kraft</h3><p><sup>4</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Kraft%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Peter Kraft</a></div></div><div id="_co_idm140501725006192"><h3 class="no_margin">Abrar A. Qureshi</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>2</sup>Clinical Research Program, Department of Dermatology,  and</p><div>Find articles by <a href="/pubmed/?term=Qureshi%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Abrar A. Qureshi</a></div></div><div id="_co_idm140501725003232"><h3 class="no_margin">Constance Chen</h3><p><sup>4</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Chen%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Constance Chen</a></div></div><div id="_co_idm140501761208384"><h3 class="no_margin">Qun Guo</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><div>Find articles by <a href="/pubmed/?term=Guo%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Qun Guo</a></div></div><div id="_co_idm140501761206160"><h3 class="no_margin">Frank B. Hu</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>4</sup>Department of Epidemiology,  and</p><p><sup>5</sup>Department of Nutrition, Harvard School of Public Health, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Hu%20FB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Frank B. Hu</a></div></div><div id="_co_idm140501725057568"><h3 class="no_margin">Eric B. Rimm</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>4</sup>Department of Epidemiology,  and</p><p><sup>5</sup>Department of Nutrition, Harvard School of Public Health, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Rimm%20EB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Eric B. Rimm</a></div></div><div id="_co_idm140501722516720"><h3 class="no_margin">Gary Curhan</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>4</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Curhan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Gary Curhan</a></div></div><div id="_co_idm140501723958368"><h3 class="no_margin">Yiqing Song</h3><p><sup>3</sup>Division of Preventive Medicine, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Song%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Yiqing Song</a></div></div><div id="_co_idm140501723956144"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>7</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Christopher I. Amos</a></div></div><div id="_co_idm140501759615856"><h3 class="no_margin">Li-E Wang</h3><p><sup>7</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Wang%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Li-E Wang</a></div></div><div id="_co_idm140501759613632"><h3 class="no_margin">Jeffrey E. Lee</h3><p><sup>8</sup>Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Lee%20JE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Jeffrey E. Lee</a></div></div><div id="_co_idm140501717889872"><h3 class="no_margin">Qingyi Wei</h3><p><sup>7</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Wei%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Qingyi Wei</a></div></div><div id="_co_idm140501717887648"><h3 class="no_margin">David J. Hunter</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>4</sup>Department of Epidemiology,  and</p><p><sup>5</sup>Department of Nutrition, Harvard School of Public Health, Boston, MA, USA,</p><div>Find articles by <a href="/pubmed/?term=Hunter%20DJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">David J. Hunter</a></div></div><div id="_co_idm140501724946896"><h3 class="no_margin">Jiali Han</h3><p><sup>1</sup>Channing Laboratory, Department of Medicine,</p><p><sup>2</sup>Clinical Research Program, Department of Dermatology,  and</p><p><sup>4</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Han%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21478494">Jiali Han</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140501717616832_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140501717616832_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140501717616832_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140501717616832_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Channing Laboratory, Department of Medicine,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Clinical Research Program, Department of Dermatology,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Division of Preventive Medicine, Department of Medicine, Brigham and Women's Hospital and Harvard Medical School, Boston, MA, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Epidemiology,  and</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Nutrition, Harvard School of Public Health, Boston, MA, USA,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>FAS Research Computing, Harvard University, Cambridge, MA, USA,</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Epidemiology,  and</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Surgical Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: 181 Longwood Avenue, Boston, MA 02115, USA. Tel: +1 6175252098; Email: <a href="mailto:dev@null" data-email="ude.dravrah.gninnahc@nah.ilaij" class="oemail">ude.dravrah.gninnahc@nah.ilaij</a></div><div>Contributed by <sup>†</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140501717616832_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Dec 17; Revised 2011 Mar 23; Accepted 2011 Apr 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140501717616832_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3110001/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supplementary Data
            </div>
            
            <div><a href="/pmc/articles/PMC3110001/bin/supp_20_13_2673__index.html" data-ga-action="click_feat_suppl">supp_20_13_2673__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;B19CC052-D120-4B53-8414-8F385821F2C2</div>
            <div><a href="/pmc/articles/PMC3110001/bin/supp_ddr154_ddr154supp.doc" data-ga-action="click_feat_suppl">supp_ddr154_ddr154supp.doc</a><span style="color:gray"> (136K)</span></div><div class="small guid">GUID:&nbsp;58AF8686-57BE-48CC-AB83-B6B1DBDC9805</div>
         </div></dd></dl></div><div id="idm140501724671552" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140501724671552title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">We conducted a genome-wide association study on the number of melanocytic nevi reported by 9136 individuals of European ancestry, with follow-up replication in 3581 individuals. We identified the nidogen 1 (<em>NID1</em>) gene on 1q42 associated with nevus count (two linked single nucleotide polymorphisms with <em>r</em><sup>2</sup> &gt; 0.9: rs3768080 A allele associated with reduced count, <em>P</em> = 6.5 × 10<sup>−8</sup>; and rs10754833 T allele associated with reduced count, <em>P</em> = 1.5 × 10<sup>−7</sup>). We further determined that the rs10754833 [T] was associated with a decreased melanoma risk in 2368 melanoma cases and 7432 controls [for CT genotype: odds ratio (OR) = 0.86, 95% confidence interval (CI) = 0.75–0.99, <em>P</em> = 0.04; for TT genotype: OR = 0.84, 95% CI = 0.71–0.98, <em>P</em> = 0.03]. Expression level of the <em>NID1</em> locus was 2-fold higher for the rs10754833 T allele carriers than that with the CC genotype (<em>P</em> = 0.017) in the 87 HapMap CEU cell lines. The <em>NID1</em> gene is a biologically plausible locus for nevogenesis and melanoma development, with decreased expression levels of <em>NID1</em> in benign nevi (<em>P</em> = 3.5 × 10<sup>−6</sup>) and in primary melanoma (<em>P</em> = 4.6 × 10<sup>−4</sup>) compared with the normal skin.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">The number of melanocytic nevi (moles) occurring in sun-exposed sites (e.g. on the extensor forearms) is a well-known risk factor for melanoma (<a href="#DDR154C1" rid="DDR154C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272663">1</a>,<a href="#DDR154C2" rid="DDR154C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272664">2</a>). As a presumed precursor to melanoma, nevi occur when melanocytes cluster, forming nests at the dermo-epidermal junction. Both benign and dysplastic nevi are characterized by disruption of the epidermal melanin transfer system through which each melanocyte transfers melanin-containing melanosomes to the suprabasal keratinocytes via dendrites (<a href="#DDR154C3" rid="DDR154C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272665">3</a>). Among different types of nevi (most of which are benign), melanocytic dysplastic nevi are believed to confer a higher risk for melanoma (<a href="#DDR154C4" rid="DDR154C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272669">4</a>). The variation in nevus counts between individuals is primarily determined by genetic factors, with twin studies suggesting that 60% of the variance is explained by additive genetic variance (<a href="#DDR154C5" rid="DDR154C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272667">5</a>,<a href="#DDR154C6" rid="DDR154C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272668">6</a>).</p><p id="__p4" class="p p-last">A recent genome-wide association study (GWAS) using 1524 healthy adult female twins from the TwinsUK registry identified genetic variants at 9p21 and 22q13 associated with the development of melanocytic nevi (<a href="#DDR154C7" rid="DDR154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272675">7</a>). To identify additional genetic variants conferring development of melanocytic nevi, we performed a multistage GWAS of the number of melanocytic nevi. First, as the discovery set, we conducted a GWAS of melanocytic nevi in 9136 men and women of European ancestry who lived in the USA from the following five case–control studies nested within the Nurses’ Health Study (NHS) and Health Professionals Follow-up Study (HPFS): a type 2 diabetes case–control study nested within the NHS (T2D_NHS, <em>n</em> = 2900); a type 2 diabetes case–control study nested within the HPFS (T2D_HPFS, <em>n</em> = 2068); a coronary heart disease case–control study nested within the NHS (CHD_NHS, <em>n</em> = 1068); a coronary heart disease case–control study nested within the HPFS (CHD_HPFS, <em>n</em> = 978); and a postmenopausal invasive breast cancer case–control study nested within the NHS (BC_NHS, <em>n</em> = 2122). We performed a GWAS on mole count in each set separately and then combined the data using meta-analysis. Second, we conducted a fast-track replication of eight promising single nucleotide polymorphisms (SNPs) in the replication set of 3581 participants, including individuals with and without a history of kidney stones within the NHS and HPFS (KS_NHS_HPFS, <em>n</em> = 961) and individuals being studied for renal function within the NHS study (RF_NHS, <em>n</em> = 2620). Detailed descriptions of the study population for each study of discovery set and replication set are presented in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>. There was no sample overlap among the five studies of the discovery set and the two studies of the replication set, nor between discovery and replication sets. The study protocol was approved by the Institutional Review Board of Brigham and Women's Hospital and the Harvard School of Public Health.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p5" class="p p-first">In the discovery set, 2 318 094 SNPs were available for meta-analysis. The quantile–quantile (QQ) plots based on the five individual GWASs and combined meta-analysis in the discovery set are presented in Figure&nbsp;<a href="/pmc/articles/PMC3110001/figure/DDR154F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR154F1" rid-ob="ob-DDR154F1" co-legend-rid="lgnd_DDR154F1"><span>1</span></a>. The QQ plots did not demonstrate a systematic deviation from the expected distribution, consistent with a minimal likelihood of systematic genotype error or bias due to the underlying population substructure. The overall genomic control inflation factor was <em>λ</em><sub>GC</sub> = 1.006.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR154F1" co-legend-rid="lgnd_DDR154F1"><a href="/pmc/articles/PMC3110001/figure/DDR154F1/" target="figure" rid-figpopup="DDR154F1" rid-ob="ob-DDR154F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501760744128" class="figure"><a href="/pmc/articles/PMC3110001/figure/DDR154F1/" target="figure" rid-figpopup="DDR154F1" rid-ob="ob-DDR154F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3110001_ddr15401.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr15401.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3110001/bin/ddr15401.jpg"></a></div><div id="largeobj_idm140501760744128" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/figure/DDR154F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR154F1"><div><a class="figpopup" href="/pmc/articles/PMC3110001/figure/DDR154F1/" target="figure" rid-figpopup="DDR154F1" rid-ob="ob-DDR154F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p6">The QQ plot of observed versus expected −log<sub>10</sub>(<em>P</em>) values for each study of the discovery set and combined set.</p></div></div></div><p id="__p7">We selected top four regions (chromosomes 1, 9, 11 and 22) for a fast-track replication (Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB1" rid-ob="ob-DDR154TB1" co-legend-rid=""><span>1</span></a>). To ensure the validity of genotyping, in each of these four regions, we selected two SNPs in complete linkage disequilibrium (LD) as mutual surrogates (<em>r</em><sup>2</sup> = 1.0 in HapMap CEU). Although there were other SNPs in LD, we selected those two SNPs showing the strongest associations with nevus count. We excluded SNPs with <em>P</em>-value for heterogeneity test &lt;0.05. All eight SNPs were at <em>P</em>-value &lt;1.5 × 10<sup>−6</sup> and ranked among the lowest 25 <em>P</em>-values in the discovery set (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR154TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p8">Association results of selected eight SNPs in the discovery set, replication set and combined set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501760573744" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="left">Discovery set<hr></th><th colspan="2" rowspan="1" align="left">Replication set<hr></th><th colspan="2" rowspan="1" align="left">Combined set<hr></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP (allele)<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Chr.</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em> for heterogeneity</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs3768080 (A)</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>NID1</em></td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.2E − 06</td><td rowspan="1" colspan="1">−0.05 (0.02)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">6.5E − 08</td><td rowspan="1" colspan="1">0.80</td></tr><tr><td rowspan="1" colspan="1">rs10754833 (T)</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>NID1</em></td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.1E − 06</td><td rowspan="1" colspan="1">−0.05 (0.02)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">−0.06 (0.01)</td><td rowspan="1" colspan="1">1.5E − 07</td><td rowspan="1" colspan="1">0.67</td></tr><tr><td rowspan="1" colspan="1">rs12379423 (G)</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1"><em>KLF4</em></td><td rowspan="1" colspan="1">−0.14 (0.03)</td><td rowspan="1" colspan="1">8.5E − 07</td><td rowspan="1" colspan="1">0.07 (0.04)</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">−0.07 (0.02)</td><td rowspan="1" colspan="1">3.5E − 03</td><td rowspan="1" colspan="1">0.00006</td></tr><tr><td rowspan="1" colspan="1">rs10979103 (G)</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1"><em>KLF4</em></td><td rowspan="1" colspan="1">−0.14 (0.03)</td><td rowspan="1" colspan="1">8.2E − 07</td><td rowspan="1" colspan="1">0.06 (0.04)</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">−0.06 (0.02)</td><td rowspan="1" colspan="1">4.6E − 03</td><td rowspan="1" colspan="1">0.00007</td></tr><tr><td rowspan="1" colspan="1">rs16930685 (C)</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1"><em>C11orf74</em></td><td rowspan="1" colspan="1">0.12 (0.03)</td><td rowspan="1" colspan="1">1.3E − 06</td><td rowspan="1" colspan="1">0.05 (0.04)</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">0.10 (0.02)</td><td rowspan="1" colspan="1">2.3E − 06</td><td rowspan="1" colspan="1">0.26</td></tr><tr><td rowspan="1" colspan="1">rs167125 (C)</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1"><em>C11orf74</em></td><td rowspan="1" colspan="1">0.12 (0.03)</td><td rowspan="1" colspan="1">1.3E − 06</td><td rowspan="1" colspan="1">0.03 (0.04)</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1">0.09 (0.02)</td><td rowspan="1" colspan="1">9.2E − 06</td><td rowspan="1" colspan="1">0.09</td></tr><tr><td rowspan="1" colspan="1">rs738322 (G)</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1"><em>PLA2G6</em></td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.3E − 06</td><td rowspan="1" colspan="1">−0.03 (0.02)</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">−0.06 (0.01)</td><td rowspan="1" colspan="1">1.0E − 06</td><td rowspan="1" colspan="1">0.16</td></tr><tr><td rowspan="1" colspan="1">rs2413503 (T)</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1"><em>PLA2G6</em></td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.4E − 06</td><td rowspan="1" colspan="1">−0.03 (0.02)</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">−0.06 (0.01)</td><td rowspan="1" colspan="1">1.5E − 06</td><td rowspan="1" colspan="1">0.11</td></tr></tbody></table></div><div id="largeobj_idm140501760573744" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/table/DDR154TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140501721857616"><p id="__p9" class="p p-first-last"><sup>a</sup>The allele corresponding to the regression parameter beta is listed in the parentheses.</p></div></div></div><p id="__p10">We genotyped the selected eight SNPs in the replication set, and only the signal from the nidogen 1 (<em>NID1</em>) on chromosome 1 showed nominal significance (Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB1" rid-ob="ob-DDR154TB1" co-legend-rid=""><span>1</span></a>). The rs3768080 was replicated with the smallest <em>P</em>-value in the combined analysis of discovery set and replication set (<a href="#DDR154C8" rid="DDR154C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272681">8</a>). The regression parameter beta (<em>P</em>-value) for replication set was −0.05 (0.01) of the A allele, i.e. the A allele of the SNP rs3768080 was associated with fewer melanocytic nevi. The other SNP (rs10754833) selected in this region showed a similar association pattern (T allele: beta = −0.05, <em>P</em> = 0.03) (<em>r</em><sup>2</sup>= 0.92 between the two SNPs in our discovery set). Replication results for other three regions, including chromosomes 9, 11 and 22, are presented in Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB1" rid-ob="ob-DDR154TB1" co-legend-rid=""><span>1</span></a>. When combining the discovery set and replication set, the <em>P</em>-value was 6.5 × 10<sup>−8</sup> for the rs3768080 and 1.5 × 10<sup>−7</sup> for the rs10754833 (Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB2" rid-ob="ob-DDR154TB2" co-legend-rid=""><span>2</span></a>). The regional association plot for the <em>NID1</em> region in the discovery set is presented in Figure&nbsp;<a href="/pmc/articles/PMC3110001/figure/DDR154F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR154F2" rid-ob="ob-DDR154F2" co-legend-rid="lgnd_DDR154F2"><span>2</span></a>. After adjusting for rs10754833 or rs3768080 in the discovery set, none of the remaining 318 SNPs in the <em>NID1</em> region was significant at <em>P</em> &lt; 0.001 before correction for multiple comparisons.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR154TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p11">Association results of rs3768080 and rs10754833 in the <em>NID1</em> gene in the discovery set, replication set and combined set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501722754272" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Participants number</th><th rowspan="1" colspan="1" align="left">Mole count 6+ (%)</th><th colspan="3" rowspan="1" align="left">rs3768080 (‘A’ allele)<hr></th><th colspan="3" rowspan="1" align="left">rs10754833 (‘T’ allele)<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left"> Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em> for heterogeneity</th><th rowspan="1" colspan="1" align="left">Beta (SE)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em> for heterogeneity</th></tr></thead><tbody><tr><td colspan="9" rowspan="1">Discovery set</td></tr><tr><td rowspan="1" colspan="1"> BC_NHS (female)</td><td rowspan="1" colspan="1">2122</td><td rowspan="1" colspan="1">4.5</td><td rowspan="1" colspan="1">−0.06 (0.03)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.06 (0.03)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_NHS (female)</td><td rowspan="1" colspan="1">2900</td><td rowspan="1" colspan="1">6.0</td><td rowspan="1" colspan="1">−0.11 (0.03)</td><td rowspan="1" colspan="1">2.4E − 05</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.11 (0.03)</td><td rowspan="1" colspan="1">2.8E − 05</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> T2D_HPFS (male)</td><td rowspan="1" colspan="1">2068</td><td rowspan="1" colspan="1">6.2</td><td rowspan="1" colspan="1">−0.07 (0.03)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.07 (0.03)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_NHS (female)</td><td rowspan="1" colspan="1">1068</td><td rowspan="1" colspan="1">3.8</td><td rowspan="1" colspan="1">−0.01 (0.04)</td><td rowspan="1" colspan="1">0.85</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.01 (0.04)</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> CHD_HPFS (male)</td><td rowspan="1" colspan="1">978</td><td rowspan="1" colspan="1">6.4</td><td rowspan="1" colspan="1">−0.05 (0.05)</td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.06 (0.05)</td><td rowspan="1" colspan="1">0.21</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">9136</td><td rowspan="1" colspan="1">5.5</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.2E − 06</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">1.1E − 06</td><td rowspan="1" colspan="1">0.29</td></tr><tr><td colspan="9" rowspan="1">Replication set</td></tr><tr><td rowspan="1" colspan="1"> KS_NHS_HPFS (female and male)</td><td rowspan="1" colspan="1">961</td><td rowspan="1" colspan="1">4.7</td><td rowspan="1" colspan="1">−0.04 (0.04)</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.04 (0.04)</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> RF_NHS (female)</td><td rowspan="1" colspan="1">2620</td><td rowspan="1" colspan="1">4.6</td><td rowspan="1" colspan="1">−0.06 (0.03)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.05 (0.03)</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1"> All (meta-analysis)</td><td rowspan="1" colspan="1">3581</td><td rowspan="1" colspan="1">4.6</td><td rowspan="1" colspan="1">−0.05 (0.02)</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">−0.05 (0.02)</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.00</td></tr><tr><td rowspan="1" colspan="1">Combined set (meta-analysis)</td><td rowspan="1" colspan="1">12 717</td><td rowspan="1" colspan="1">5.3</td><td rowspan="1" colspan="1">−0.07 (0.01)</td><td rowspan="1" colspan="1">6.5E − 08</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">−0.06 (0.01)</td><td rowspan="1" colspan="1">1.5E − 07</td><td rowspan="1" colspan="1">0.67</td></tr></tbody></table></div><div id="largeobj_idm140501722754272" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3110001/table/DDR154TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140501759716800"><p id="__p12" class="p p-first-last">The regression parameter beta (of the rs3768080 A allele and rs10754833 T allele) for each GWAS of the discovery set was calculated based on the linear regression adjusted for the age and top three principal components of genetic variance. The regression parameter beta (of the rs3768080 A allele and rs10754833 T allele) for the KS_NHS_HPFS of replication set was calculated based on the linear regression adjusted for the age, gender and top three principal components of genetic variance. The regression parameter beta (of the rs3768080 A allele and rs10754833 T allele) for the RF_NHS of replication set was calculated based on the linear regression adjusted for the age. BC_NHS: postmenopausal invasive breast cancer case–control study nested within the NHS. T2D_NHS: type 2 diabetes case–control study nested within the NHS. T2D_HPFS: type 2 diabetes case–control study nested within the HPFS. CHD_NHS: coronary heart disease case–control study nested within the NHS. CHD_HPFS: coronary heart disease case–control study nested within the HPFS. KS_NHS_HPFS: kidney stone study nested within the NHS and HPFS. RF_NHS: renal function study nested within the NHS.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR154F2" co-legend-rid="lgnd_DDR154F2"><a href="/pmc/articles/PMC3110001/figure/DDR154F2/" target="figure" rid-figpopup="DDR154F2" rid-ob="ob-DDR154F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501721929344" class="figure"><a href="/pmc/articles/PMC3110001/figure/DDR154F2/" target="figure" rid-figpopup="DDR154F2" rid-ob="ob-DDR154F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3110001_ddr15402.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr15402.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3110001/bin/ddr15402.jpg"></a></div><div id="largeobj_idm140501721929344" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/figure/DDR154F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR154F2"><div><a class="figpopup" href="/pmc/articles/PMC3110001/figure/DDR154F2/" target="figure" rid-figpopup="DDR154F2" rid-ob="ob-DDR154F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p13">Regional association plot in the 400 kb neighborhood of <em>NID1</em>. The left-hand <em>Y</em>-axis shows the association <em>P</em>-value of individual SNPs in the discovery set, which is plotted as −log<sub>10</sub>(<em>P</em>) against chromosomal base-pair position. The right-hand <em>Y</em>-axis shows the recombination rate estimated from the HapMap CEU population. Genotyped SNPs are plotted as diamonds, and imputed as circles in grey. Blue highlights the SNPs of rs3768080 and rs10754833; bright red indicates high LD (<em>r</em><sup>2</sup> ≥ 0.8) with rs3768080; orange moderate LD (<em>r</em><sup>2</sup> ≥ 0.5 but &lt;0.8); yellow weak LD (<em>r</em><sup>2</sup> ≥ 0.2 but &lt;0.5); and white no LD (<em>r</em><sup>2</sup> &lt; 0.2). The genomic coordinate is in NCBI35/hg17.</p></div></div></div><p id="__p14">We further evaluated the association of rs3768080 and rs10754833 with melanoma risk among 2389 cases and 7527 controls, including 585 melanoma cases and 6500 controls nested within the NHS and HPFS cohorts as set 1, and a case–control study of 1804 melanoma cases and 1027 controls from the MD Anderson Cancer Center as set 2. Details of the study population are described in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>. The meta-analysis was used to combine the results from the two sets. As shown in Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB3" rid-ob="ob-DDR154TB3" co-legend-rid=""><span>3</span></a>, the T allele of rs10754833, which was associated with a reduced number of melanocytic nevi in our study, was associated with a decreased risk of melanoma [additive model: odds ratio (OR) = 0.92, 95% confidence interval (CI) = 0.84–0.99, <em>P</em>= 0.03]. This effect was significant in the co-dominant models (for CT genotype: OR = 0.86, 95% CI = 0.75–0.99, <em>P</em> = 0.04; for TT genotype: OR = 0.84, 95% CI = 0.71–0.98, <em>P</em> = 0.03) and dominant model (OR = 0.85, 95% CI = 0.74–0.98, <em>P</em> = 0.02). The A allele of rs3768080, which had an effect similar to the T allele of rs10754833 in our mole count GWAS, was also associated with a decreased risk of melanoma (Table&nbsp;<a href="/pmc/articles/PMC3110001/table/DDR154TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR154TB3" rid-ob="ob-DDR154TB3" co-legend-rid=""><span>3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR154TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p15">Associations of rs3768080 and rs10754833 in the <em>NID1</em> gene with melanoma risk</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501759103856" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1" align="left">Set 1<hr></th><th colspan="3" rowspan="1" align="left">Set 2<hr></th><th colspan="2" rowspan="1" align="left">Joint analysis<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">rs3768080</th><th rowspan="1" colspan="1" align="left">Genotype</th><th rowspan="1" colspan="1" align="left">Cases (%) (<em>n</em> = 556)</th><th rowspan="1" colspan="1" align="left">Controls (%) (<em>n</em> = 6397)</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">Cases (%) (<em>n</em> = 1798)</th><th rowspan="1" colspan="1" align="left">Controls (%) (<em>n</em> = 1026)</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left">OR (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th></tr></thead><tbody><tr><td rowspan="3" colspan="1">Co-dominant model</td><td rowspan="1" colspan="1">GG</td><td rowspan="1" colspan="1">159 (28.6)</td><td rowspan="1" colspan="1">1591 (24.9)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">420 (23.4)</td><td rowspan="1" colspan="1">217 (21.2)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">GA</td><td rowspan="1" colspan="1">237 (42.6)</td><td rowspan="1" colspan="1">3087 (48.2)</td><td rowspan="1" colspan="1">0.77 (0.62–0.95)</td><td rowspan="1" colspan="1">936 (52.0)</td><td rowspan="1" colspan="1">532 (51.8)</td><td rowspan="1" colspan="1">0.91 (0.75–1.10)</td><td rowspan="1" colspan="1">0.84 (0.73–0.97)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">AA</td><td rowspan="1" colspan="1">160 (28.8)</td><td rowspan="1" colspan="1">1719 (26.9)</td><td rowspan="1" colspan="1">0.93 (0.74–1.17)</td><td rowspan="1" colspan="1">442 (24.6)</td><td rowspan="1" colspan="1">277 (27.0)</td><td rowspan="1" colspan="1">0.82 (0.66–1.03)</td><td rowspan="1" colspan="1">0.87 (0.74–1.02)</td><td rowspan="1" colspan="1">0.10</td></tr><tr><td rowspan="2" colspan="1">Dominant model</td><td rowspan="1" colspan="1">GG</td><td rowspan="1" colspan="1">159 (28.6)</td><td rowspan="1" colspan="1">1591 (24.9)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">420 (23.4)</td><td rowspan="1" colspan="1">217 (21.2)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">GA or AA</td><td rowspan="1" colspan="1">397 (71.4)</td><td rowspan="1" colspan="1">4806 (75.1)</td><td rowspan="1" colspan="1">0.83 (0.68–1.00)</td><td rowspan="1" colspan="1">1378 (76.6)</td><td rowspan="1" colspan="1">809 (78.8)</td><td rowspan="1" colspan="1">0.88 (0.73–1.06)</td><td rowspan="1" colspan="1">0.86 (0.75–0.98)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">Additive model</td><td rowspan="1" colspan="1">‘A’ allele</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.96 (0.86–1.09)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.91 (0.81–1.01)</td><td rowspan="1" colspan="1">0.93 (0.86–1.01)</td><td rowspan="1" colspan="1">0.09</td></tr><tr><td rowspan="1" colspan="1">rs10754833</td><td rowspan="1" colspan="1">Genotype</td><td rowspan="1" colspan="1">Cases (%) (<em>n</em> = 564)</td><td rowspan="1" colspan="1">Controls (%) (<em>n</em> = 6405)</td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1">Cases (%) (<em>n</em> = 1804)</td><td rowspan="1" colspan="1">Controls (%) (<em>n</em> = 1027)</td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1"><em>P-</em>value</td></tr><tr><td rowspan="3" colspan="1">Co-dominant model</td><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">154 (27.3)</td><td rowspan="1" colspan="1">1513 (23.6)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">422 (23.4)</td><td rowspan="1" colspan="1">217 (21.1)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">CT</td><td rowspan="1" colspan="1">259 (45.9)</td><td rowspan="1" colspan="1">3122 (48.8)</td><td rowspan="1" colspan="1">0.82 (0.67–1.01)</td><td rowspan="1" colspan="1">937 (51.9)</td><td rowspan="1" colspan="1">535 (52.1)</td><td rowspan="1" colspan="1">0.90 (0.74–1.09)</td><td rowspan="1" colspan="1">0.86 (0.75–0.99)</td><td rowspan="1" colspan="1">0.04</td></tr><tr><td rowspan="1" colspan="1">TT</td><td rowspan="1" colspan="1">151 (26.8)</td><td rowspan="1" colspan="1">1770 (27.6)</td><td rowspan="1" colspan="1">0.84 (0.67–1.06)</td><td rowspan="1" colspan="1">445 (24.7)</td><td rowspan="1" colspan="1">275 (26.8)</td><td rowspan="1" colspan="1">0.83 (0.66–1.04)</td><td rowspan="1" colspan="1">0.84 (0.71–0.98)</td><td rowspan="1" colspan="1">0.03</td></tr><tr><td rowspan="2" colspan="1">Dominant model</td><td rowspan="1" colspan="1">CC</td><td rowspan="1" colspan="1">154 (27.3)</td><td rowspan="1" colspan="1">1513 (23.6)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">422 (23.4)</td><td rowspan="1" colspan="1">217 (21.1)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1">1 (ref)</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">CT or TT</td><td rowspan="1" colspan="1">410 (72.7)</td><td rowspan="1" colspan="1">4892 (76.4)</td><td rowspan="1" colspan="1">0.83 (0.68–1.01)</td><td rowspan="1" colspan="1">1382 (76.6)</td><td rowspan="1" colspan="1">810 (78.9)</td><td rowspan="1" colspan="1">0.87 (0.73–1.05)</td><td rowspan="1" colspan="1">0.85 (0.74–0.98)</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">Additive model</td><td rowspan="1" colspan="1">‘T’ allele</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.92 (0.81–1.03)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.91 (0.82–1.02)</td><td rowspan="1" colspan="1">0.92 (0.84–0.99)</td><td rowspan="1" colspan="1">0.03</td></tr></tbody></table></div><div id="largeobj_idm140501759103856" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3110001/table/DDR154TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140501760755920"><p id="__p16" class="p p-first-last">The OR was adjusted for the age and gender. Set 1: a case–control study nested within the NHS and HPFS. Set 2: a case–control study from the MD Anderson Cancer Center.</p></div></div></div><p id="__p17">In addition, we evaluated the expression of <em>NID1</em> in the normal skin, benign nevi and primary malignant melanoma, and found that decreased expression of <em>NID1</em> is associated with benign nevi and primary malignant melanoma, with a 3.1-fold lower expression in benign nevi (<em>P</em> = 3.5 × 10<sup>−6</sup>) and 1.5-fold lower expression in primary melanoma (<em>P</em> = 4.6 × 10<sup>−4</sup>) compared with the normal skin (Fig.&nbsp;<a href="/pmc/articles/PMC3110001/figure/DDR154F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR154F3" rid-ob="ob-DDR154F3" co-legend-rid="lgnd_DDR154F3"><span>3</span></a>; the data set was downloaded from NCBI GEO, accession number <a id="__tag_222272687" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE3189" ref="/geo/query/acc.cgi?acc=GSE3189"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE3189","term_id":"3189"}}</span>GSE3189</a>; <a href="#DDR154C9" rid="DDR154C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272672">9</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR154F3" co-legend-rid="lgnd_DDR154F3"><a href="/pmc/articles/PMC3110001/figure/DDR154F3/" target="figure" rid-figpopup="DDR154F3" rid-ob="ob-DDR154F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501725614480" class="figure"><a href="/pmc/articles/PMC3110001/figure/DDR154F3/" target="figure" rid-figpopup="DDR154F3" rid-ob="ob-DDR154F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3110001_ddr15403.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr15403.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3110001/bin/ddr15403.jpg"></a></div><div id="largeobj_idm140501725614480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/figure/DDR154F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR154F3"><div><a class="figpopup" href="/pmc/articles/PMC3110001/figure/DDR154F3/" target="figure" rid-figpopup="DDR154F3" rid-ob="ob-DDR154F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p18">The <em>NID1</em> expression in normal skin, benign nevi and primary melanoma. Total RNA isolated from 45 primary melanomas, 18 benign skin nevi and 7 normal skin tissue specimens was analyzed on an Affymetrix Hu133A microarray. The data set was downloaded from NCBI GEO, accession number <a id="__tag_222272688" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE3189" ref="/geo/query/acc.cgi?acc=GSE3189"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE3189","term_id":"3189"}}</span>GSE3189</a> (<a href="#DDR154C9" rid="DDR154C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272666">9</a>). The box part represents the central 50% of the data or the interquartile range. The lower edge of the box plot is the first quartile or 25th percentile. The upper edge of the box plot is the third quartile or 75th percentile. The line in the box is the median value. The ends of the vertical lines extend to a maximum of 1.5 times the inter-quartile range. The points plot the outliers.</p></div></div></div><p id="__p19">Moreover, we evaluated the potential function of these linked two SNPs (rs3768080 and rs10754833) in the <em>NID1</em> gene on the chromosome 1. The rs3768080 is located in intron 10 of <em>NID1</em>, and the rs10754833 is located in intron 9 of <em>NID1</em>. The mapping of enhancer and promoter histone marker H3K4Me1 in the ENCODE project (<a href="#DDR154C10" rid="DDR154C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272678">10</a>) suggested that they are located in the enhancer regions. Furthermore, the analysis of potential transcriptional factor-binding sites around these two SNPs using the Biobase MATCH tool (<a href="#DDR154C11" rid="DDR154C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272674">11</a>) shows that an extra binding site for the transcriptional factor AP1 was introduced by the rs3768080 A allele (cTGACCca, with a core match score of 0.957, and a matrix match score of 0.959). The AP1 is a well-accepted potent transcriptional activator. Hence, these two linked SNPs, rs3768080 A allele and the rs10754833 T allele, lead to an increased expression of <em>NID1</em> with the transcription activator AP1. Based on the transcript expression profiling data of 87 HapMap CEU cell lines (NCBI GEO database, accession <a id="__tag_222272686" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7792" ref="/geo/query/acc.cgi?acc=GSE7792"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7792","term_id":"7792"}}</span>GSE7792</a>) (<a href="#DDR154C12" rid="DDR154C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272677">12</a>), the expression level of the <em>NID1</em> locus was 2-fold higher among the rs10754833 T allele carriers than those of the CC genotype (<em>P</em> = 0.017, Fig.&nbsp;<a href="/pmc/articles/PMC3110001/figure/DDR154F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR154F4" rid-ob="ob-DDR154F4" co-legend-rid="lgnd_DDR154F4"><span>4</span></a>) (data not available for rs3768080).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR154F4" co-legend-rid="lgnd_DDR154F4"><a href="/pmc/articles/PMC3110001/figure/DDR154F4/" target="figure" rid-figpopup="DDR154F4" rid-ob="ob-DDR154F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501719170480" class="figure"><a href="/pmc/articles/PMC3110001/figure/DDR154F4/" target="figure" rid-figpopup="DDR154F4" rid-ob="ob-DDR154F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3110001_ddr15404.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr15404.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3110001/bin/ddr15404.jpg"></a></div><div id="largeobj_idm140501719170480" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/figure/DDR154F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR154F4"><div><a class="figpopup" href="/pmc/articles/PMC3110001/figure/DDR154F4/" target="figure" rid-figpopup="DDR154F4" rid-ob="ob-DDR154F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p20"><em>NID1</em> expression by the rs10754833 genotype. Based on the transcript expression profiling data of 87 HapMap CEU cell lines (NCBI GEO database, accession <a id="__tag_222272685" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE7792" ref="/geo/query/acc.cgi?acc=GSE7792"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE7792","term_id":"7792"}}</span>GSE7792</a>) (<a href="#DDR154C12" rid="DDR154C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272676">12</a>), the expression level of <em>NID1</em> locus was 2-fold higher for the rs10754833 T allele carriers compared with that with the CC genotype (<em>P</em> = 0.017) (data not available for rs3768080). The <em>Y</em>-axis is the expression intensity of <em>NID1</em>. The box represents the central 50% of the data or the interquartile range. The lower edge of the box plot is the first quartile or 25th percentile. The upper edge of the box plot is the third quartile or 75th percentile. The line in the box is the median value. The ends of the vertical lines extend to a maximum of 1.5 times the inter-quartile range. The points plot the outliers.</p></div></div></div><p id="__p21" class="p p-last">In this study, among the selected four regions for a fast-track replication, the PLA2G6 gene on 22q13.1 (rs738322 and rs2413503) was previously identified as a susceptibility locus to melanocytic nevi in the previous GWAS on nevus count conducted by Falchi <em>et al</em>. (rs2284063, <em>P</em> = 3.4 × 10<sup>−8</sup>) (<a href="#DDR154C7" rid="DDR154C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272684">7</a>). These two SNPs, rs2284063 and rs738322 (or rs2413503), are in modest LD (<em>r</em><sup>2</sup> = 0.5 in HapMap CEU). The <em>P</em>-value for the SNP rs2284063 and the SNP rs738322 (or rs2413503) in the discovery set of our study was 6.3 × 10<sup>−4</sup> and 1.3 × 10<sup>−6</sup> (or 1.4 × 10<sup>−6</sup>), respectively.</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p22" class="p p-first">In this study, we identified a novel susceptibility locus to mole count, NID1. The NID1, a 150 kDa sulfated glycoprotein, is a member of the nidogen family of basement membrane (BM) proteins. As a major component of BM in the skin, nidogen plays an active role in BM assembly, i.e. it interacts with many other BM molecules, promoting those components as integrating elements for BM assembly (<a href="#DDR154C13" rid="DDR154C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272662">13</a>). Hence, the <em>NID1</em> gene is a biologically plausible candidate locus for nevogenesis and melanoma development. Supporting this mechanism, we found that the rs3768080 A allele and rs10754833 T allele in the <em>NID1</em> gene were associated with lower mole count and reduced melanoma risk. These data are consistent with their association with increased expression of the <em>NID1</em> gene, considering that the expression levels of <em>NID1</em> are decreased in benign nevi and primary melanoma compared with the normal skin. Although the association of the <em>NID1</em> gene with mole count did not quite reach the genome-wide significance threshold, our additional functional data and melanoma results for this locus altogether substantiated the <em>NID1</em> gene as a novel susceptibility locus to cutaneous nevi.</p><p id="__p23">One limitation of this study is that nevus count was self-reported; however, the majority of studies on nevus counts have shown a substantial agreement between nevus self-counts and dermatologist counts (<a href="#DDR154C14" rid="DDR154C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272682">14</a>,<a href="#DDR154C15" rid="DDR154C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272670">15</a>), and the Spearman correlation coefficient between nevus counts by patients and physicians was 0.91 (<a href="#DDR154C16" rid="DDR154C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272671">16</a>). We have used the number of moles on arms as a proxy for the total body mole counts and systematic melanoma risk (<a href="#DDR154C17" rid="DDR154C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272679">17</a>).</p><p id="__p24" class="p p-last">In this GWAS of individuals of European ancestry, we identified a novel locus, the <em>NID1</em> gene on 1q42, which was associated with the number of melanocytic nevi on arms and melanoma risk. Identification of genetic susceptibility loci to nevi as melanoma precursors and intermediate phenotypes could contribute to improvements in risk assessment and stratification, early detection and prevention of melanoma, a highly lethal cancer.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Description of study populations</h3><div id="s4a1" class="sec sec-first"><p></p><h4 id="s4a1title" class="inline">Nurses’ health study (NHS) </h4><p id="__p25" class="p p-first-last">The NHS was established in 1976, when 121 700 female US-registered nurses between the ages of 30 and 55 residing in 11 larger US states completed and returned an initial self-administered questionnaire on their medical histories and baseline health-related exposures, forming the basis for the NHS cohort. Biennial questionnaires with collection of exposure information on risk factors have been collected prospectively. Along with exposures every 2 years, outcome data with appropriate follow-up of reported disease events are collected. Overall, follow-up has been very high; after more than 20 years ∼90% of participants continue to complete questionnaires. From May 1989 through September 1990, we collected blood samples from 32 826 participants in the NHS cohort. The information on the number of melanocytic nevi on arms was collected in the 1986 questionnaire.</p></div><div id="s4a2" class="sec sec-last"><p></p><h4 id="s4a2title" class="inline">Health professionals follow-up study (HPFS) </h4><p id="__p26" class="p p-first-last">In 1986, 51 529 men from all 50 US states in health professions (dentists, pharmacists, optometrists, osteopath physicians, podiatrists and veterinarians) aged 40–75 answered a detailed mailed questionnaire, forming the basis of the study. The average follow-up rate for this cohort over 10 years is &gt;90%. On each biennial questionnaire, we obtained disease- and health-related information. Between 1993 and 1994, 18 159 study participants provided blood samples by overnight courier. The information on mole count was collected in the 1987 questionnaire using wording similar to that used in the NHS.</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Laboratory assays</h3><div id="s4b1" class="sec sec-first"><p></p><h4 id="s4b1title" class="inline">Genotyping in five GWASs of the discovery set </h4><p id="__p27" class="p p-first-last">We performed genotyping in BC_NHS using the Illumina HumanHap550 array, as part of the National Cancer Institute's Cancer Genetic Markers of Susceptibility (CGEMS) Project (<a href="#DDR154C18" rid="DDR154C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_289072254">18</a>). For the other four GWASs of the discovery set, we performed genotyping using the Affymetrix 6.0 array. Based on the genotyped SNPs and haplotype information in the NCBI build 35 of phase II Hapmap CEU data, we imputed genotypes for &gt;2.5 million SNPs using the program MACH. The quality control procedures for five GWAS sets of the discovery phase are described in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>. The number of genotyped SNPs passed quality control procedures and the imputed SNPs with minor allele frequency (MAF) &gt; 2.5% and imputation <em>R</em><sup>2</sup> &gt; 0.3 in each study of the discovery set are presented as follows:
<!--table ft1--><!--table-wrap mode="anchored" t5--></p><div class="table-wrap table anchored whole_rhythm" id="d32e1683"><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140501794159872" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Study</th><th rowspan="1" colspan="1" align="left">Genotyped</th><th rowspan="1" colspan="1" align="left">Imputed</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">BC_NHS</td><td rowspan="1" colspan="1">546 646</td><td rowspan="1" colspan="1">2 352 569</td></tr><tr><td rowspan="1" colspan="1">T2D_NHS</td><td rowspan="1" colspan="1">704 409</td><td rowspan="1" colspan="1">2 351 699</td></tr><tr><td rowspan="1" colspan="1">T2D_HPFS</td><td rowspan="1" colspan="1">706 040</td><td rowspan="1" colspan="1">2 356 842</td></tr><tr><td rowspan="1" colspan="1">CHD_NHS</td><td rowspan="1" colspan="1">721 316</td><td rowspan="1" colspan="1">2 350 863</td></tr><tr><td rowspan="1" colspan="1">CHD_HPFS</td><td rowspan="1" colspan="1">724 881</td><td rowspan="1" colspan="1">2 356 504</td></tr></tbody></table></div><div id="largeobj_idm140501794159872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3110001/table/d32e1683/?report=objectonly">Open in a separate window</a></div></div><p></p></div><div id="s4b2" class="sec sec-last"><p></p><h4 id="s4b2title" class="inline">Genotyping in replication set </h4><p id="__p28" class="p p-first-last">Eight promising SNPs from the discovery set were selected for further replication in the replication set. (i) The genotyping for the KS_NHS_HPFS was performed using the Illumina HumanHap610 Quad, and the imputation was performed in the same fashion as in the discovery set. The genotype data we extracted for those eight SNPs and their imputation quality data are presented in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>. (ii) The genotyping for the RF_NHS was performed using Open Array assays at the Dana Farber/Harvard Cancer Center Polymorphism Detection Core.</p></div></div><div id="s4c" class="sec sec-last"><h3 id="s4ctitle">Statistical methods</h3><p id="__p29" class="p p-first">Both the NHS and HPFS prospectively collected self-reported information on the number of melanocytic nevi on the arms (larger than 3 mm in diameter) using similar wording; this variable is a highly significant predictor of melanoma risk in these cohorts (<a href="#DDR154C19" rid="DDR154C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272680">19</a>). In each GWAS of the discovery set and each study of the replication set, we regressed an ordinal coding for the number of melanocytic nevi on arms (1 = none; 2 = 1–2; 3 = 3–5; 4 = 6–9; 5 = 10–14; and 6 = 15+) on an ordinal coding for genotype (0, 1 or 2 copies of the SNP minor allele) individually for each SNP that passed quality control filters. We adjusted for three largest principal components of genetic variation in the regression model of each GWAS of the discovery set and the replication set KS_NHS_HPFS. Among the 10 largest principal components, none was significantly associated with nevus count at two-sided alpha of 0.0l, after adjusting for age in each GWAS of the discovery set in this study. These principal components were calculated for all individuals on the basis of ∼10 000 unlinked markers using the EIGENSTRAT software (<a href="#DDR154C20" rid="DDR154C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_222272683">20</a>).</p><p id="__p30" class="p p-last">In each study of the discovery set, those SNPs with MAF &gt; 2.5% and imputation <em>R</em><sup>2</sup> &gt; 0.3 were included in further meta-analysis. Betas from each study of the discovery set were combined by a meta-analysis with weights proportional to the inverse variance of the beta in each study. The same meta-analysis method was used to combine the results from the discovery set and replication set.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p31" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr154/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p32" class="p p-first-last">This work is supported by NIH grants <span>CA122838</span>;, <span>CA87969</span>;, <span>CA055075</span>;, <span>CA49449</span>;, <span>CA100264</span>; and <span>CA093459</span>.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supplementary Data:</strong>
            
            <div class="sup-box half_rhythm" id="idm140501798507408"><a href="/pmc/articles/PMC3110001/bin/supp_20_13_2673__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140501798503504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140501798503504title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p33">We thank Dr Wei V. Chen for assistance in performing analyses in the MD Anderson Cancer Center melanoma case–control study. We thank Pati Soule and Dr Hardeep Ranu of the Dana Farber/Harvard Cancer Center High-Throughput Polymorphism Detection Core for sample handling and genotyping of the NHS and HPFS samples. We are grateful to Merck Research Laboratories for funding of the GWAS of coronary heart disease. We are also indebted to the participants in all of these studies. We thank the following state cancer registries for their help: AL, AZ, AR, CA, CO, CT, DE, FL, GA, ID, IL, IN, IA, KY, LA, ME, MD, MA, MI, NE, NH, NJ, NY, NC, ND, OH, OK, OR, PA, RI, SC, TN, TX, VA, WA, WY.</p><p id="__p34"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140501721238864" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140501721238864title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDR154C1">1. <span class="element-citation">Gandini S., Sera F., Cattaruzza M.S., Pasquini P., Abeni D., Boyle P., Melchi C.F. Meta-analysis of risk factors for cutaneous melanoma: I. Common and atypical naevi. <span><span class="ref-journal">Eur. J. Cancer. </span>2005;<span class="ref-vol">41</span>:28–44.</span>  <a href="http://dx.doi.org/10.1016/j.ejca.2004.10.015" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1016/j.ejca.2004.10.015</a>. [<a href="/pubmed/15617989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Cancer&amp;title=Meta-analysis+of+risk+factors+for+cutaneous+melanoma:+I.+Common+and+atypical+naevi&amp;author=S.+Gandini&amp;author=F.+Sera&amp;author=M.S.+Cattaruzza&amp;author=P.+Pasquini&amp;author=D.+Abeni&amp;volume=41&amp;publication_year=2005&amp;pages=28-44&amp;pmid=15617989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C2">2. <span class="element-citation">Whiteman D.C., Whiteman C.A., Green A.C. Childhood sun exposure as a risk factor for melanoma: a systematic review of epidemiologic studies. <span><span class="ref-journal">Cancer Causes Control. </span>2001;<span class="ref-vol">12</span>:69–82.</span>  <a href="http://dx.doi.org/10.1023/A:1008980919928" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1023/A:1008980919928</a>. [<a href="/pubmed/11227927" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Causes+Control&amp;title=Childhood+sun+exposure+as+a+risk+factor+for+melanoma:+a+systematic+review+of+epidemiologic+studies&amp;author=D.C.+Whiteman&amp;author=C.A.+Whiteman&amp;author=A.C.+Green&amp;volume=12&amp;publication_year=2001&amp;pages=69-82&amp;pmid=11227927&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C3">3. <span class="element-citation">Chin L. The genetics of malignant melanoma: lessons from mouse and man. <span><span class="ref-journal">Nat. Rev. Cancer. </span>2003;<span class="ref-vol">3</span>:559–570.</span>  <a href="http://dx.doi.org/10.1038/nrc1145" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nrc1145</a>. [<a href="/pubmed/12894244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cancer&amp;title=The+genetics+of+malignant+melanoma:+lessons+from+mouse+and+man&amp;author=L.+Chin&amp;volume=3&amp;publication_year=2003&amp;pages=559-570&amp;pmid=12894244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C4">4. <span class="element-citation">Greene M.H., Clark W.H., Jr, Tucker M.A., Kraemer K.H., Elder D.E., Fraser M.C. High risk of malignant melanoma in melanoma-prone families with dysplastic nevi. <span><span class="ref-journal">Ann. Intern. Med. </span>1985;<span class="ref-vol">102</span>:458–465.</span> [<a href="/pubmed/3977193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Intern.+Med.&amp;title=High+risk+of+malignant+melanoma+in+melanoma-prone+families+with+dysplastic+nevi&amp;author=M.H.+Greene&amp;author=W.H.+Clark&amp;author=M.A.+Tucker&amp;author=K.H.+Kraemer&amp;author=D.E.+Elder&amp;volume=102&amp;publication_year=1985&amp;pages=458-465&amp;pmid=3977193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C5">5. <span class="element-citation">Bataille V., Snieder H., MacGregor A.J., Sasieni P., Spector T.D. Genetics of risk factors for melanoma: an adult twin study of nevi and freckles. <span><span class="ref-journal">J. Natl Cancer Inst. </span>2000;<span class="ref-vol">92</span>:457–463.</span>  <a href="http://dx.doi.org/10.1093/jnci/92.6.457" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/jnci/92.6.457</a>. [<a href="/pubmed/10716963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Genetics+of+risk+factors+for+melanoma:+an+adult+twin+study+of+nevi+and+freckles&amp;author=V.+Bataille&amp;author=H.+Snieder&amp;author=A.J.+MacGregor&amp;author=P.+Sasieni&amp;author=T.D.+Spector&amp;volume=92&amp;publication_year=2000&amp;pages=457-463&amp;pmid=10716963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C6">6. <span class="element-citation">Zhu G., Montgomery G.W., James M.R., Trent J.M., Hayward N.K., Martin N.G., Duffy D.L. A genome-wide scan for naevus count: linkage to CDKN2A and to other chromosome regions. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2007;<span class="ref-vol">15</span>:94–102.</span>  <a href="http://dx.doi.org/10.1038/sj.ejhg.5201729" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/sj.ejhg.5201729</a>. [<a href="/pubmed/17063143" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=A+genome-wide+scan+for+naevus+count:+linkage+to+CDKN2A+and+to+other+chromosome+regions&amp;author=G.+Zhu&amp;author=G.W.+Montgomery&amp;author=M.R.+James&amp;author=J.M.+Trent&amp;author=N.K.+Hayward&amp;volume=15&amp;publication_year=2007&amp;pages=94-102&amp;pmid=17063143&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C7">7. <span class="element-citation">Falchi M., Bataille V., Hayward N.K., Duffy D.L., Bishop J.A., Pastinen T., Cervino A., Zhao Z.Z., Deloukas P., Soranzo N., et al.  Genome-wide association study identifies variants at 9p21 and 22q13 associated with development of cutaneous nevi. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:915–919.</span>  <a href="http://dx.doi.org/10.1038/ng.410" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng.410</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3080738/">PMC free article</a>]</span> [<a href="/pubmed/19578365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+variants+at+9p21+and+22q13+associated+with+development+of+cutaneous+nevi&amp;author=M.+Falchi&amp;author=V.+Bataille&amp;author=N.K.+Hayward&amp;author=D.L.+Duffy&amp;author=J.A.+Bishop&amp;volume=41&amp;publication_year=2009&amp;pages=915-919&amp;pmid=19578365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C8">8. <span class="element-citation">Skol A.D., Scott L.J., Abecasis G.R., Boehnke M. Joint analysis is more efficient than replication-based analysis for two-stage genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:209–213.</span>  <a href="http://dx.doi.org/10.1038/ng1706" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1706</a>. [<a href="/pubmed/16415888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Joint+analysis+is+more+efficient+than+replication-based+analysis+for+two-stage+genome-wide+association+studies&amp;author=A.D.+Skol&amp;author=L.J.+Scott&amp;author=G.R.+Abecasis&amp;author=M.+Boehnke&amp;volume=38&amp;publication_year=2006&amp;pages=209-213&amp;pmid=16415888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C9">9. <span class="element-citation">Talantov D., Mazumder A., Yu J.X., Briggs T., Jiang Y., Backus J., Atkins D., Wang Y. Novel genes associated with malignant melanoma but not benign melanocytic lesions. <span><span class="ref-journal">Clin. Cancer Res. </span>2005;<span class="ref-vol">11</span>:7234–7242.</span>  <a href="http://dx.doi.org/10.1158/1078-0432.CCR-05-0683" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1158/1078-0432.CCR-05-0683</a>. [<a href="/pubmed/16243793" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Novel+genes+associated+with+malignant+melanoma+but+not+benign+melanocytic+lesions&amp;author=D.+Talantov&amp;author=A.+Mazumder&amp;author=J.X.+Yu&amp;author=T.+Briggs&amp;author=Y.+Jiang&amp;volume=11&amp;publication_year=2005&amp;pages=7234-7242&amp;pmid=16243793&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C10">10. <span class="element-citation">Birney E., Stamatoyannopoulos J.A., Dutta A., Guigo R., Gingeras T.R., Margulies E.H., Weng Z., Snyder M., Dermitzakis E.T., Thurman R.E., et al.  Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:799–816.</span>  <a href="http://dx.doi.org/10.1038/nature05874" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/nature05874</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2212820/">PMC free article</a>]</span> [<a href="/pubmed/17571346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification+and+analysis+of+functional+elements+in+1%+of+the+human+genome+by+the+ENCODE+pilot+project&amp;author=E.+Birney&amp;author=J.A.+Stamatoyannopoulos&amp;author=A.+Dutta&amp;author=R.+Guigo&amp;author=T.R.+Gingeras&amp;volume=447&amp;publication_year=2007&amp;pages=799-816&amp;pmid=17571346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C11">11. <span class="element-citation">Kel A.E., Gossling E., Reuter I., Cheremushkin E., Kel-Margoulis O.V., Wingender E. MATCH: a tool for searching transcription factor binding sites in DNA sequences. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3576–3579.</span>  <a href="http://dx.doi.org/10.1093/nar/gkg585" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/nar/gkg585</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC169193/">PMC free article</a>]</span> [<a href="/pubmed/12824369" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=MATCH:+a+tool+for+searching+transcription+factor+binding+sites+in+DNA+sequences&amp;author=A.E.+Kel&amp;author=E.+Gossling&amp;author=I.+Reuter&amp;author=E.+Cheremushkin&amp;author=O.V.+Kel-Margoulis&amp;volume=31&amp;publication_year=2003&amp;pages=3576-3579&amp;pmid=12824369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C12">12. <span class="element-citation">Huang R.S., Duan S., Bleibel W.K., Kistner E.O., Zhang W., Clark T.A., Chen T.X., Schweitzer A.C., Blume J.E., Cox N.J., et al.  A genome-wide approach to identify genetic variants that contribute to etoposide-induced cytotoxicity. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2007;<span class="ref-vol">104</span>:9758–9763.</span>  <a href="http://dx.doi.org/10.1073/pnas.0703736104" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1073/pnas.0703736104</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1887589/">PMC free article</a>]</span> [<a href="/pubmed/17537913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=A+genome-wide+approach+to+identify+genetic+variants+that+contribute+to+etoposide-induced+cytotoxicity&amp;author=R.S.+Huang&amp;author=S.+Duan&amp;author=W.K.+Bleibel&amp;author=E.O.+Kistner&amp;author=W.+Zhang&amp;volume=104&amp;publication_year=2007&amp;pages=9758-9763&amp;pmid=17537913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C13">13. <span class="element-citation">Breitkreutz D., Mirancea N., Nischt R. Basement membranes in skin: unique matrix structures with diverse functions? <span><span class="ref-journal">Histochem. Cell Biol. </span>2009;<span class="ref-vol">132</span>:1–10.</span> [<a href="/pubmed/19333614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Histochem.+Cell+Biol.&amp;title=Basement+membranes+in+skin:+unique+matrix+structures+with+diverse+functions?&amp;author=D.+Breitkreutz&amp;author=N.+Mirancea&amp;author=R.+Nischt&amp;volume=132&amp;publication_year=2009&amp;pages=1-10&amp;pmid=19333614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C14">14. <span class="element-citation">Buettner P.G., Garbe C. Agreement between self-assessment of melanocytic nevi by patients and dermatologic examination. <span><span class="ref-journal">Am. J. Epidemiol. </span>2000;<span class="ref-vol">151</span>:72–77.</span> [<a href="/pubmed/10625176" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Agreement+between+self-assessment+of+melanocytic+nevi+by+patients+and+dermatologic+examination&amp;author=P.G.+Buettner&amp;author=C.+Garbe&amp;volume=151&amp;publication_year=2000&amp;pages=72-77&amp;pmid=10625176&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C15">15. <span class="element-citation">Little P., Keefe M., White J. Self screening for risk of melanoma: validity of self mole counting by patients in a single general practice. <span><span class="ref-journal">BMJ. </span>1995;<span class="ref-vol">310</span>:912–916.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2549294/">PMC free article</a>]</span> [<a href="/pubmed/7719184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Self+screening+for+risk+of+melanoma:+validity+of+self+mole+counting+by+patients+in+a+single+general+practice&amp;author=P.+Little&amp;author=M.+Keefe&amp;author=J.+White&amp;volume=310&amp;publication_year=1995&amp;pages=912-916&amp;pmid=7719184&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C16">16. <span class="element-citation">Mikkilineni R., Weinstock M.A. Is the self-counting of moles a valid method of assessing melanoma risk? <span><span class="ref-journal">Arch. Dermatol. </span>2000;<span class="ref-vol">136</span>:1550–1551.</span>  <a href="http://dx.doi.org/10.1001/archderm.136.12.1550" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1001/archderm.136.12.1550</a>. [<a href="/pubmed/11115171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.&amp;title=Is+the+self-counting+of+moles+a+valid+method+of+assessing+melanoma+risk?&amp;author=R.+Mikkilineni&amp;author=M.A.+Weinstock&amp;volume=136&amp;publication_year=2000&amp;pages=1550-1551&amp;pmid=11115171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C17">17. <span class="element-citation">Weinstock M.A., Colditz G.A., Willett W.C., Stampfer M.J., Bronstein B.R., Mihm M.C., Jr., Speizer F.E. Moles and site-specific risk of nonfamilial cutaneous malignant melanoma in women. <span><span class="ref-journal">J. Natl Cancer Inst. </span>1989;<span class="ref-vol">81</span>:948–952.</span>  <a href="http://dx.doi.org/10.1093/jnci/81.12.948" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/jnci/81.12.948</a>. [<a href="/pubmed/2733040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Moles+and+site-specific+risk+of+nonfamilial+cutaneous+malignant+melanoma+in+women&amp;author=M.A.+Weinstock&amp;author=G.A.+Colditz&amp;author=W.C.+Willett&amp;author=M.J.+Stampfer&amp;author=B.R.+Bronstein&amp;volume=81&amp;publication_year=1989&amp;pages=948-952&amp;pmid=2733040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C18">18. <span class="element-citation">Hunter D.J., Kraft P., Jacobs K.B., Cox D.G., Yeager M., Hankinson S.E., Wacholder S., Wang Z., Welch R., Hutchinson A., et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span>  <a href="http://dx.doi.org/10.1038/ng2075" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng2075</a>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=D.J.+Hunter&amp;author=P.+Kraft&amp;author=K.B.+Jacobs&amp;author=D.G.+Cox&amp;author=M.+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C19">19. <span class="element-citation">Cho E., Rosner B.A., Feskanich D., Colditz G.A. Risk factors and individual probabilities of melanoma for whites. <span><span class="ref-journal">J. Clin. Oncol. </span>2005;<span class="ref-vol">23</span>:2669–2675.</span>  <a href="http://dx.doi.org/10.1200/JCO.2005.11.108" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1200/JCO.2005.11.108</a>. [<a href="/pubmed/15837981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Risk+factors+and+individual+probabilities+of+melanoma+for+whites&amp;author=E.+Cho&amp;author=B.A.+Rosner&amp;author=D.+Feskanich&amp;author=G.A.+Colditz&amp;volume=23&amp;publication_year=2005&amp;pages=2669-2675&amp;pmid=15837981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR154C20">20. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span>  <a href="http://dx.doi.org/10.1038/ng1847" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1038/ng1847</a>. [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3110001&amp;issue-id=197329&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140501724671552title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140501798503504title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140501721238864title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3110001/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3110001/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3110001/pdf/ddr154.pdf">PDF (207K)</a></li> | <li><a href="#" data-citationid="PMC3110001" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3110001%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3110001%2F&amp;text=Genome-wide%20association%20study%20identifies%20nidogen%201%20(NID1)%20as%20a%20susceptibility%20locus%20to%20cutaneous%20nevi%20and%20melanoma%20risk" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3110001%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21478494" data-aiid="3110001" data-aid="3110001" data-iid="197329" data-domainid="740" data-domain="hmg" data-accid="PMC3110001" data-md5="94dae042d62f3d0cc1358c64ac85e7e0"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:54:11-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BB877DD5254510000000002F001C3&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3110001%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3110001/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>