<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3888262/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of ancestry-specific TB risk in the South African Coloured population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3888262/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study of ancestry-specific TB risk in the South African Coloured population"><meta name="citation_authors" content="Emile R. Chimusa, Noah Zaitlen, Michelle Daya, Marlo Möller, Paul D. van Helden, Nicola J. Mulder, Alkes L. Price, Eileen G. Hoal"><meta name="citation_date" content="1 February 2014"><meta name="citation_issue" content="3"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="796"><meta name="citation_doi" content="10.1093/hmg/ddt462"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3888262/?report=abstract"><meta name="citation_pmid" content="24057671"><meta name="DC.Title" content="Genome-wide association study of ancestry-specific TB risk in the South African Coloured population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Emile R. Chimusa"><meta name="DC.Contributor" content="Noah Zaitlen"><meta name="DC.Contributor" content="Michelle Daya"><meta name="DC.Contributor" content="Marlo Möller"><meta name="DC.Contributor" content="Paul D. van Helden"><meta name="DC.Contributor" content="Nicola J. Mulder"><meta name="DC.Contributor" content="Alkes L. Price"><meta name="DC.Contributor" content="Eileen G. Hoal"><meta name="DC.Date" content="2014 Feb 1"><meta name="DC.Identifier" content="10.1093/hmg/ddt462"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of ancestry-specific TB risk in the South African Coloured population"><meta property="og:type" content="article"><meta property="og:description" content="The worldwide burden of tuberculosis (TB) remains an enormous problem, and is particularly severe in the admixed South African Coloured (SAC) population residing in the Western Cape. Despite evidence from twin studies suggesting a strong genetic component ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3888262/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3888262/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3888262/pdf/ddt462.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DB949DD55EF010000000007740474.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3888262%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3888262</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2014 Feb 1; 23(3): 796–809. </span></div><div><span class="fm-vol-iss-date">Published online 2013 Sep 20. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddt462" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddt462</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3888262</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24057671">24057671</a></div></div></div></div><h1 class="content-title">Genome-wide association study of ancestry-specific TB risk in the South African Coloured population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chimusa%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013673442640" co-class="co-affbox">Emile R. Chimusa</a>,<sup>1,</sup><sup>†</sup><sup>*</sup> <a href="/pubmed/?term=Zaitlen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013683645120" co-class="co-affbox">Noah Zaitlen</a>,<sup>7,</sup><sup>†</sup> <a href="/pubmed/?term=Daya%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013679291888" co-class="co-affbox">Michelle Daya</a>,<sup>6</sup> <a href="/pubmed/?term=M%26%23x000f6%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013679289616" co-class="co-affbox">Marlo Möller</a>,<sup>6</sup> <a href="/pubmed/?term=van%20Helden%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013679026976" co-class="co-affbox">Paul D. van Helden</a>,<sup>6</sup> <a href="/pubmed/?term=Mulder%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013679024800" co-class="co-affbox">Nicola J. Mulder</a>,<sup>1</sup> <a href="/pubmed/?term=Price%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013678655440" co-class="co-affbox">Alkes L. Price</a>,<sup>2,</sup><sup>3,</sup><sup>5,</sup><sup>4,</sup><sup>‡</sup> and  <a href="/pubmed/?term=Hoal%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671" class="affpopup" co-rid="_co_idm140013679887184" co-class="co-affbox">Eileen G. Hoal</a><sup>6,</sup><sup>‡</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140013673442640"><h3 class="no_margin">Emile R. Chimusa</h3><p><sup>1</sup>Department of Clinical Laboratory Sciences, Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa</p><div>Find articles by <a href="/pubmed/?term=Chimusa%20ER%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Emile R. Chimusa</a></div></div><div id="_co_idm140013683645120"><h3 class="no_margin">Noah Zaitlen</h3><p><sup>7</sup>Department of Medicine, University of California San Francisco, San Francisco, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Zaitlen%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Noah Zaitlen</a></div></div><div id="_co_idm140013679291888"><h3 class="no_margin">Michelle Daya</h3><p><sup>6</sup>Molecular Biology and Human Genetics, MRC Centre for Molecular and Cellular Biology, DST/NRF Centre of Excellence for Biomedical TB Research, Faculty of Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=Daya%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Michelle Daya</a></div></div><div id="_co_idm140013679289616"><h3 class="no_margin">Marlo Möller</h3><p><sup>6</sup>Molecular Biology and Human Genetics, MRC Centre for Molecular and Cellular Biology, DST/NRF Centre of Excellence for Biomedical TB Research, Faculty of Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=M%26%23x000f6%3Bller%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Marlo Möller</a></div></div><div id="_co_idm140013679026976"><h3 class="no_margin">Paul D. van Helden</h3><p><sup>6</sup>Molecular Biology and Human Genetics, MRC Centre for Molecular and Cellular Biology, DST/NRF Centre of Excellence for Biomedical TB Research, Faculty of Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=van%20Helden%20PD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Paul D. van Helden</a></div></div><div id="_co_idm140013679024800"><h3 class="no_margin">Nicola J. Mulder</h3><p><sup>1</sup>Department of Clinical Laboratory Sciences, Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa</p><div>Find articles by <a href="/pubmed/?term=Mulder%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Nicola J. Mulder</a></div></div><div id="_co_idm140013678655440"><h3 class="no_margin">Alkes L. Price</h3><p><sup>2</sup>Department of Epidemiology</p><p><sup>3</sup>Department of Biostatistics</p><p><sup>4</sup>Program in Molecular and Genetic Epidemiology, Harvard School of Public Health, Boston, MA, USA</p><p><sup>5</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Price%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Alkes L. Price</a></div></div><div id="_co_idm140013679887184"><h3 class="no_margin">Eileen G. Hoal</h3><p><sup>6</sup>Molecular Biology and Human Genetics, MRC Centre for Molecular and Cellular Biology, DST/NRF Centre of Excellence for Biomedical TB Research, Faculty of Health Sciences, Stellenbosch University, Tygerberg, South Africa</p><div>Find articles by <a href="/pubmed/?term=Hoal%20EG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24057671">Eileen G. Hoal</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140013683471968_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140013683471968_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140013683471968_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140013683471968_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Clinical Laboratory Sciences, Institute of Infectious Disease and Molecular Medicine, University of Cape Town, Cape Town, South Africa</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Epidemiology</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Biostatistics</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Program in Molecular and Genetic Epidemiology, Harvard School of Public Health, Boston, MA, USA</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Program in Medical and Population Genetics, Broad Institute, Cambridge, MA, USA</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Molecular Biology and Human Genetics, MRC Centre for Molecular and Cellular Biology, DST/NRF Centre of Excellence for Biomedical TB Research, Faculty of Health Sciences, Stellenbosch University, Tygerberg, South Africa</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Medicine, University of California San Francisco, San Francisco, CA, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Email: <a href="mailto:dev@null" data-email="az.ca.tcu.oibc@elime" class="oemail">az.ca.tcu.oibc@elime</a></div><div id="AN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors.</div><div id="AN2"><sup>‡</sup>The authors wish it to be known that, in their opinion, the final two authors should be regarded as joint Senior Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140013683471968_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Jul 30; Revised 2013 Jul 30; Accepted 2013 Sep 17.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140013683471968_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2013. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3888262/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3888262/bin/supp_23_3_796__index.html" data-ga-action="click_feat_suppl">supp_23_3_796__index.html</a><span style="color:gray"> (1.5K)</span></div><div class="small guid">GUID:&nbsp;F0CC8148-F716-42FC-B088-43EFFB20D236</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp.doc" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp.doc</a><span style="color:gray"> (360K)</span></div><div class="small guid">GUID:&nbsp;3E5DC4C8-8C7A-4644-A345-E64872303218</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig1.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig1.png</a><span style="color:gray"> (61K)</span></div><div class="small guid">GUID:&nbsp;7B9A5751-9355-45AE-A290-3DF4EBE65554</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig2.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig2.png</a><span style="color:gray"> (7.8K)</span></div><div class="small guid">GUID:&nbsp;5D1C5019-5B1D-4B62-861E-C181CB4B3C65</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig3.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig3.png</a><span style="color:gray"> (161K)</span></div><div class="small guid">GUID:&nbsp;37813EE4-682F-4A5B-A3FE-FA9630DD5C0A</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig4.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig4.png</a><span style="color:gray"> (310K)</span></div><div class="small guid">GUID:&nbsp;0EAE35CD-7F94-4C12-B27F-34CDF7C92884</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig5.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig5.png</a><span style="color:gray"> (162K)</span></div><div class="small guid">GUID:&nbsp;D37B0B6A-486D-4A7B-BFC1-5F98483997F1</div>
<div><a href="/pmc/articles/PMC3888262/bin/supp_ddt462_ddt462supp_fig6.png" data-ga-action="click_feat_suppl">supp_ddt462_ddt462supp_fig6.png</a><span style="color:gray"> (2.7M)</span></div><div class="small guid">GUID:&nbsp;C3B7722F-D1F4-4C10-A4F6-DFC50C8CD7C3</div>
</div></dd></dl></div><div id="idm140013683032176" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140013683032176title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">The worldwide burden of tuberculosis (TB) remains an enormous problem, and is particularly severe in the admixed South African Coloured (SAC) population residing in the Western Cape. Despite evidence from twin studies suggesting a strong genetic component to TB resistance, only a few loci have been identified to date. In this work, we conduct a genome-wide association study (GWAS), meta-analysis and trans-ethnic fine mapping to attempt the replication of previously identified TB susceptibility loci. Our GWAS results confirm the <em>WT1</em> chr11 susceptibility locus (rs2057178: odds ratio = 0.62, <em>P</em> = 2.71e<sup>−06</sup>) previously identified by Thye <em>et al</em>., but fail to replicate previously identified polymorphisms in the <em>TLR8</em> gene and locus 18q11.2. Our study demonstrates that the genetic contribution to TB risk varies between continental populations, and illustrates the value of including admixed populations in studies of TB risk and other complex phenotypes. Our evaluation of local ancestry based on the real and simulated data demonstrates that case-only admixture mapping is currently impractical in multi-way admixed populations, such as the SAC, due to spurious deviations in average local ancestry generated by current local ancestry inference methods. This study provides insights into identifying disease genes and ancestry-specific disease risk in multi-way admixed populations.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p4" class="p p-first">Tuberculosis (TB) is a significant source of morbidity and mortality worldwide, particularly in developing countries. It is a leading cause of human immunodeficiency virus (HIV)-related deaths, as almost one in four deaths among people with HIV infection is due to TB (<a href="#DDT462C1" rid="DDT462C1" class=" bibr popnode">1</a>,<a href="#DDT462C2" rid="DDT462C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522838">2</a>). In 2010, there were 8.8 million new cases of TB, of which 1.1 million were among people living with HIV (<a href="#DDT462C3" rid="DDT462C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140963">3</a>,<a href="#DDT462C4" rid="DDT462C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140945">4</a>). TB susceptibility is known to be a complex trait influenced by both genetic and environmental factors (<a href="#DDT462C5" rid="DDT462C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140980">5</a>). The environmental factors that influence TB susceptibility include socio-economic conditions, smoking and acute infection. One-third of the world's individuals are infected with TB, but only 10% go on to develop active TB during their lifetime (<a href="#DDT462C6" rid="DDT462C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522834">6</a>,<a href="#DDT462C7" rid="DDT462C7" class=" bibr popnode">7</a>). In addition, twin studies in humans and animal models also demonstrate a strong genetic influence on TB susceptibility (<a href="#DDT462C5" rid="DDT462C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140962">5</a>,<a href="#DDT462C8" rid="DDT462C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140929">8</a>,<a href="#DDT462C9" rid="DDT462C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140950">9</a>). The rate of concordance of TB among monozygotic twins (18 of 55, 32.7%) was more than twice (odds ratio of concordance: 2.4; 95% CI: 1.4–4.0) that observed among dizygotic twins (21 of 150, 14.0%) (<a href="#DDT462C8" rid="DDT462C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140946">8</a>,<a href="#DDT462C9" rid="DDT462C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140932">9</a>). These estimates suggest that genetic factors play an important role in TB susceptibility in both understanding the host response and determining the outcome of infection (<a href="#DDT462C2" rid="DDT462C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522841">2</a>,<a href="#DDT462C8" rid="DDT462C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140994">8</a>).</p><p id="__p5">GWAS identification of genetic associations of host susceptibility to infectious disease, such as TB, has been slow and limited by small sample sizes (<a href="#DDT462C10" rid="DDT462C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140939">10</a>,<a href="#DDT462C11" rid="DDT462C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140943">11</a>). Few GWAS papers that identify the signal of association with pulmonary TB were published until recently. Davila <em>et al</em>. identified four polymorphisms in the <em>TLR8</em> gene on chromosome X, in an Indonesian population and replicated the result in a Russian cohort (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140991">12</a>). Thye <em>et al</em>. conducted a combined GWAS in African populations in Ghana and Gambia to investigate the host susceptibility to TB, and identified the 18q11.2 locus (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137470">13</a>). Recently, Thye <em>et al</em>. reported a new TB susceptibility locus on chromosome 11p13 in a Ghanaian population near the <em>WT1</em> gene. This finding was replicated in samples from Gambia, Indonesia and Russia (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522860">14</a>). A case–control study in a Chinese population failed to replicate the 18q11.2 locus (<a href="#DDT462C15" rid="DDT462C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140990">15</a>). However, it was replicated in a separate Han Chinese TB cohort (<a href="#DDT462C16" rid="DDT462C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141008">16</a>).</p><p id="__p6">These studies and work showing that rates of TB vary considerably between populations and regions suggest that association effects may vary across ethnic groups (<a href="#DDT462C4" rid="DDT462C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140941">4</a>,<a href="#DDT462C17" rid="DDT462C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141003">17</a>). Analysis of the correlation between genetic ancestry and phenotype in recently admixed populations can improve disease prediction and provide crucial insights into medical genetics (<a href="#DDT462C18" rid="DDT462C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140961">18</a>). For example, Kumar <em>et al</em>. showed that genetic ancestry exerts a major influence on improving lung-function estimates and categorizing asthma severity in an African American population (<a href="#DDT462C18" rid="DDT462C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140956">18</a>).</p><p id="__p7">The second highest incidence of TB in the world is in the Western Cape in South Africa, particularly in the admixed South African Coloured (SAC) population (<a href="#DDT462C3" rid="DDT462C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140984">3</a>,<a href="#DDT462C19" rid="DDT462C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140995">19</a>). Investigations based on candidate gene studies and genome-wide linkage scans on the SAC population were previously conducted (<a href="#DDT462C3" rid="DDT462C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140975">3</a>,<a href="#DDT462C10" rid="DDT462C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140968">10</a>,<a href="#DDT462C11" rid="DDT462C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141006">11</a>,<a href="#DDT462C19" rid="DDT462C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140951">19</a>,<a href="#DDT462C20" rid="DDT462C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140934">20</a>). Many of the candidate gene studies failed to observe statistical associations with the markers that were examined and were often inconclusive (<a href="#DDT462C11" rid="DDT462C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140972">11</a>,<a href="#DDT462C20" rid="DDT462C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140928">20</a>). Although a few genetic association studies have identified candidate genes for TB susceptibility using data from the SAC population (<a href="#DDT462C11" rid="DDT462C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141007">11</a>,<a href="#DDT462C20" rid="DDT462C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140964">20</a>), neither ancestry-specific TB risk nor GWAS for TB has been considered in this population. In addition, no study on other African ethnic groups has been attempted to replicate the susceptibility loci on chromosomes 18q11.2 and 11p13, reported in Thye <em>et al</em>. (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137462">13</a>,<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522836">14</a>), respectively. It is not known whether the predisposition due to these SNPs applies to other African populations.</p><p id="__p8">Here, we conducted a GWAS with correction for genome-wide ancestry, computed the power to replicate each published odds ratio and specified whether the published odds ratio lies within the 95% confidence interval (CI) estimated in the SAC dataset. Our study replicated the polymorphisms in <em>WT1</em> gene previously identified in Thye <em>et al</em>. (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522849">14</a>). Both imputation-based meta-analysis and trans-ethnic fine mapping of our study with recent African TB studies confirm that genetic variants in the <em>WT1</em> gene confer risk of TB. However, our study did not validate previously identified polymorphisms in the <em>TLR8</em> gene on chromosome X (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141013">12</a>) or the 18q11.2 locus (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137464">13</a>).</p><p id="__p9" class="p p-last">The SAC have a mixed ancestry traced back over 350 years from various populations including European Caucasoid, South and East Asians, and SAN and Bantu Africans (<a href="#DDT462C21" rid="DDT462C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140973">21</a>–<a href="#DDT462C24" rid="DDT462C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141011">24</a>). Our results show that ancestry-specific TB risk is not due to confounding by socio-economic status (SES) as is the case for type-2 diabetes in Latino populations from Mexico and Colombia (<a href="#DDT462C25" rid="DDT462C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140987">25</a>). Furthermore, our findings indicated that the SAN ancestral component confers risk while non-African ancestral components confer protection against TB in the SAC.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">GWAS with correction for genome-wide ancestry</h3><p id="__p10" class="p p-first">We analyzed data from 642 TB cases and 91 controls (after QC) genotyped on the Affymetrix 500 K chip (see Materials and Methods). Using impute2 (<a href="#DDT462C26" rid="DDT462C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140960">26</a>), we imputed the untyped genotypes using HapMap3 release 2 (<a href="#DDT462C27" rid="DDT462C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140936">27</a>) and 1000 Genome project populations (<a href="#DDT462C28" rid="DDT462C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140925">28</a>). There were 1 453 294 and 4 467 279 genetic variants retained from each imputation panel respectively. As expected, we observed substantial population substructure in the SAC (see Materials and Methods). To account for both population stratification and hidden relatedness, we applied the mixed model approach EMMAX (<a href="#DDT462C29" rid="DDT462C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140971">29</a>) on these datasets; the Q–Q plots of genomic control factors effects are shown in Figure&nbsp;<a href="/pmc/articles/PMC3888262/figure/DDT462F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT462F1" rid-ob="ob-DDT462F1" co-legend-rid="lgnd_DDT462F1"><span>1</span></a>. The genomic control lambda (<a href="#DDT462C30" rid="DDT462C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141000">30</a>) was <em>λ</em><sub>GC</sub> = 1.05 for the typed dataset, <em>λ</em><sub>GC</sub> = 1.05 for the HapMap3 imputed dataset, <em>λ</em><sub>GC</sub> = 1.09 for the 1000G imputed dataset and <em>λ</em><sub>GC</sub> = 1.08 for the combined (typed, HapMap3 imputed and 1000G imputed) dataset. These genomic control lambdas are acceptable, and suggest little departure from the null expectation, except at the right end tail of the distribution. To control for cryptic relatedness, we used EMMAX (<a href="#DDT462C29" rid="DDT462C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140955">29</a>), a variance component-based statistical test shown to control for both structure and relatedness.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT462F1" co-legend-rid="lgnd_DDT462F1"><a href="/pmc/articles/PMC3888262/figure/DDT462F1/" target="figure" rid-figpopup="DDT462F1" rid-ob="ob-DDT462F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013676072064" class="figure"><a href="/pmc/articles/PMC3888262/figure/DDT462F1/" target="figure" rid-figpopup="DDT462F1" rid-ob="ob-DDT462F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3888262_ddt46201.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt46201.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3888262/bin/ddt46201.jpg"></a></div><div id="largeobj_idm140013676072064" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3888262/figure/DDT462F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT462F1"><div><a class="figpopup" href="/pmc/articles/PMC3888262/figure/DDT462F1/" target="figure" rid-figpopup="DDT462F1" rid-ob="ob-DDT462F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p11">Q–Q Plot of population stratification effects to compare the distribution of observed <em>P</em>-values with the expected distribution: The <em>λ</em><sub>GC</sub> values indicate the residual population stratification effects (after correction) which are minimal. Plots A–D are from typed genotype, imputed genotype from both HapMap3 and the 1000 Genomes project and combined datasets, respectively.</p></div></div></div><p id="__p12">As shown in Figure&nbsp;<a href="/pmc/articles/PMC3888262/figure/DDT462F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT462F2" rid-ob="ob-DDT462F2" co-legend-rid="lgnd_DDT462F2"><span>2</span></a>, a SNP on chromosome 14 q24.2, <em>rs17175227</em> (<em>P</em> = 8.99e<sup>−09</sup> and OR = 0.141) appears to be a genome-wide significant association signal. The SNP <em>rs17175227</em> has a low minor allele frequency of 0.01642. We performed a well-calibrated test for rare SNPs, Fisher's exact test, to see whether the specific SNP would still be genome-wide significant. The result suggested that <em>rs17175227</em> was not genome-wide significant (<em>P</em> = 2.77e<sup>−06</sup>, OR = 0.141) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). This highlights an important challenge in association analysis of low-frequency (1–5%) variants, which may often attain genome-wide significance in standard tests such as mixed model association or logistic regression due to the imperfect asymptotic distribution of those tests in the case of low-frequency variants (Materials and Methods).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT462F2" co-legend-rid="lgnd_DDT462F2"><a href="/pmc/articles/PMC3888262/figure/DDT462F2/" target="figure" rid-figpopup="DDT462F2" rid-ob="ob-DDT462F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013681893376" class="figure"><a href="/pmc/articles/PMC3888262/figure/DDT462F2/" target="figure" rid-figpopup="DDT462F2" rid-ob="ob-DDT462F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3888262_ddt46202.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt46202.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3888262/bin/ddt46202.jpg"></a></div><div id="largeobj_idm140013681893376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3888262/figure/DDT462F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT462F2"><div><a class="figpopup" href="/pmc/articles/PMC3888262/figure/DDT462F2/" target="figure" rid-figpopup="DDT462F2" rid-ob="ob-DDT462F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p13">Manhattan plot of genome-wide association analyses of TB in the SAC (<strong>A</strong>) from typed dataset only, and (<strong>B</strong> and <strong>C</strong>) both imputed datasets based on HapMap3 and 1000 Genomes reference populations, and (<strong>D</strong>) the combined dataset.</p></div></div></div><p id="__p14" class="p p-last">The HapMap3 imputed SNP <em>rs12294076</em> (<em>P</em> = 9.56e<sup>−08</sup>) on chromosome 11q21–q22.1 narrowly misses the threshold of genome-wide significance, which we define as <em>P</em> = 6.4e<sup>−08</sup>, 1.7e<sup>−08</sup> and 5.5e<sup>−09</sup> based on 390 887; 1 453 294 and 4 467 279 SNPs tested (Fig.&nbsp;<a href="/pmc/articles/PMC3888262/figure/DDT462F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT462F2" rid-ob="ob-DDT462F2" co-legend-rid="lgnd_DDT462F2"><span>2</span></a>) from typed genotype data and imputed data using HapMap3 and 1000 Genomes data, respectively (see Materials and Methods). The genetic variant <em>rs12294076</em> has a minor allele frequency of 0.16 in the SAC, 0.22 in Yoruba and 0.00 in other HapMap populations, and is likely to be an African-specific SNP. The closest gene to this SNP is <em>DYNC2H1</em>. This gene encodes a large cytoplasmic dynein protein known to be involved in retrograde transport in the cilium with a major role in intraflagellar transport (<a href="#DDT462C31" rid="DDT462C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140966">31</a>). Mutations in <em>DYNC2H1</em> cause a heterogeneous spectrum of conditions related to altered primary cilium function. The sub-cellular distribution of dynein shows specific association with elements of the late endocytic pathway (<a href="#DDT462C31" rid="DDT462C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140977">31</a>)<em>.</em> Additional 36 typed and 62 imputed genetic markers with suggestive <em>P</em>-values (10<sup>−05</sup>–10<sup>−06</sup>) that did not pass our genome-wide significance thresholds are listed in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>, respectively.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Replication of SNPs reported in previous studies</h3><p id="__p15" class="p p-first">We attempted to replicate the chr11, chr18 and chrX associations with TB previously reported by Thye <em>et al</em>. (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137471">13</a>, <a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522867">14</a>) and Davila <em>et al</em>. (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140924">12</a>), respectively. In each case, we computed our power to detect an association at the previously reported odds ratio, and evaluated whether the previously reported odds ratio was within our 95% CI (see Materials and Methods). We found that the associated SNP, <em>rs2057178</em> (<em>P</em> = 2.63e<sup>−09</sup>, OR = 0.77 and MAF = 0.33) on 11p13 reported by Thye <em>et al</em>. (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522850">14</a>), is close to genome-wide significance (<em>P</em> = 2.71e<sup>−06</sup>, OR = 0.62 and MAF = 0.08) in the SAC-TB imputation GWAS (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>). A second reported significant SNP in the Ghanaian study group <em>rs11031728</em> (<em>P</em> = 5.25e<sup>−09</sup>, MAF = 0.32 and OR = 0.77) was also associated in our imputation GWAS study (<em>P</em> = 2.86e<sup>−06</sup>, MAF = 0.08 and OR = 0.61). The third most significant SNP in their study, <em>rs11031731</em> (<em>P</em> = 7.01e<sup>−09</sup>, MAF = 0.31 and OR = 0.78), was not accurately imputed in our study (CALL ≈ 0.8), and could therefore not be examined with confidence (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). The <em>rs2057178</em>, <em>rs11031728</em> and <em>rs11031731</em> 1000G imputed SNPs in the SAC are not genotyped in GIH and SAN data. Nevertheless, we computed the <em>r</em><sup>2</sup> LD between these three SNPs and other SNPs in the <em>WT1</em> locus using the SAC data and the Yoruba in Ibadan (YRI), CEU, Japanese in Toyko (JPT) + CHB data from the 1000 Genomes project. Previous results reported that the <em>rs2057178</em>, <em>rs11031728</em> and <em>rs11031731</em> SNPs are in strong LD in the Ghanaian population (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522831">14</a>). We obtained <em>r</em><sup>2</sup>(<em>rs2057178</em>, <em>rs11031728</em>) = 0.90, 0.90, 1.00 and 0.8; <em>r</em><sup>2</sup>(<em>rs2057178</em>, <em>rs11031731</em>) = 0.70, 0.90, 1 and 1 and <em>r</em><sup>2</sup>(<em>rs11031728</em>, <em>rs11031731</em>) = 0.70, 1.00, 1.00 and 0.90 in SAC, CEU, YRI and JPT + CHB, respectively. These three variants are also found to be in LD in the SAC data. Additional loci in LD with the <em>WT1</em> locus are provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>. The <em>rs2057178</em>, <em>rs11031728</em> and <em>rs11031731</em> SNPs and others listed in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a> are associated with <em>WT1</em>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT462TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p16">Replication of SNPs reported in previous studies</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013675120320" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="5" rowspan="1" align="left">SAC TB Study<hr></th><th colspan="3" rowspan="1" align="left">Thye <em>et al</em>. (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522868">14</a>)</th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">CHR</th><th rowspan="1" colspan="1" align="left">POS</th><th rowspan="1" colspan="1" align="left">A1/A2</th><th rowspan="1" colspan="1" align="left">Call</th><th rowspan="1" colspan="1" align="left">INFO</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">OR(95%CI)</th><th rowspan="1" colspan="1" align="left">Power</th><th rowspan="1" colspan="1" align="left">SNP type</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">OR(95%CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>rs2057178</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32364187</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.8</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">2.71e<sup>−06</sup></td><td rowspan="1" colspan="1">0.62(0.50–0.75)</td><td rowspan="1" colspan="1">0.1564</td><td rowspan="1" colspan="1">1000G Imputed</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">2.63e<sup>−09</sup></td><td rowspan="1" colspan="1">0.77 (0.71–0.84)</td></tr><tr><td rowspan="1" colspan="1"><em>rs11031728</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32363616</td><td rowspan="1" colspan="1">C/G</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">2.86e<sup>−06</sup></td><td rowspan="1" colspan="1">0.61(0.50–0.75)</td><td rowspan="1" colspan="1">0.1454</td><td rowspan="1" colspan="1">1000G Imputed</td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">7.01e<sup>−09</sup></td><td rowspan="1" colspan="1">0.78 (0.71–0.8)</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1">Thye <em>et al</em>. 2010 (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137472">13</a>)</td></tr><tr><td rowspan="1" colspan="1"><em>rs4331426</em></td><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">19676176</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.9</td><td rowspan="1" colspan="1">0.19</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">1.00(0.95–1.04)</td><td rowspan="1" colspan="1">0.1185</td><td rowspan="1" colspan="1">HapMap3 Imputed</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">6.8e<sup>−09</sup></td><td rowspan="1" colspan="1">1.19 [1.1–1.3]</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1">Davila <em>et al</em>. 2008 (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140978">12</a>)</td></tr><tr><td rowspan="1" colspan="1"><em>rs3788935</em></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">12922659</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.386</td><td rowspan="1" colspan="1">0.1465</td><td rowspan="1" colspan="1">1.30(0.91–1.85)</td><td rowspan="1" colspan="1">0.5532</td><td rowspan="1" colspan="1">1000G Imputed</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.014</td><td rowspan="1" colspan="1">1.4 (1.07–1.8)</td></tr><tr><td rowspan="1" colspan="1"><em>rs3761624</em></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">12923681</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.382</td><td rowspan="1" colspan="1">0.1844</td><td rowspan="1" colspan="1">1.27(0.89–1.81)</td><td rowspan="1" colspan="1">0.5521</td><td rowspan="1" colspan="1">1000G Imputed</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.016</td><td rowspan="1" colspan="1">1.4 (1.06–1.8)</td></tr><tr><td rowspan="1" colspan="1"><em>rs3764879</em></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">12924697</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.386</td><td rowspan="1" colspan="1">0.2854</td><td rowspan="1" colspan="1">1.23(0.87–1.80)</td><td rowspan="1" colspan="1">0.5532</td><td rowspan="1" colspan="1">1000G Imputed</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.4 (1.06–1.8)</td></tr></tbody></table></div><div id="largeobj_idm140013675120320" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3888262/table/DDT462TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140013682855072"><p id="__p17" class="p p-first-last">Assuming a <em>P</em>-value threshold of 0.05, we computed the power to replicate each published odds ratio in the SAC dataset (Power). All published odds ratio from Thye <em>et al</em>. (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137469">13</a>,<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522856">14</a>) do not lie within the 95% CI estimated in the current SAC dataset.</p></div></div></div><p id="__p18">Looking at Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>, the effect of the susceptibility loci at <em>WT1</em> in the SAC is in the same direction as those published in Thye <em>et al</em>. (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522857">14</a>), suggesting resistance to TB. However, the published odds ratio in Thye <em>et al</em>. 2012 (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522852">14</a>) does not lie within the 95% CI estimated in the SAC dataset (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>). This may be due to different LD patterns between the SAC and WTCCC-TB data, since the SAC is admixed. SNPs can be associated with the same phenotype in different ethnicity groups, but the effect sizes may differ substantially, because the tag SNP may tag different sets of causal variants.</p><p id="__p19"><em>WT1</em> is a suppressor gene located on chromosome 11p13 and provides instructions for making a protein that is involved in the development of the kidneys and gonads (ovaries in females and testes in males) before birth (<a href="#DDT462C32" rid="DDT462C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140920">32</a>). Furthermore, it is also known as a transcription factor since it regulates the activity of other genes by binding to specific regions of DNA. Querying a comprehensive human Protein–Protein Interaction network (<a href="http://cbg.garvan.unsw.edu.au/pina/">http://cbg.garvan.unsw.edu.au/pina/</a>), <em>WT1</em> has known direct interactions (<a href="#DDT462C32" rid="DDT462C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140927">32</a>) with <em>UBE2I, AREG, WTAP, AREGB, U2AF2, TP73, SDGF, PRKACA</em> and the <em>P53</em> gene. In particular, this gene is unusually expressed in lung and certain types of prostate cancer, and has been seen in some cancers of blood-forming cells (leukemias), such as acute lymphoblastic leukemia, chronic myeloid leukemia and childhood acute myeloid leukemia (<a href="#DDT462C32" rid="DDT462C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140935">32</a>).</p><p id="__p20">The identified susceptibility locus <em>rs4331426</em> on chromosome 18q11.2 in Thye <em>et al</em>. 2010 (MAF = 0.48, Gambia: <em>P</em> = 0.003 and OR = 1.18, Ghana: <em>P</em> = 0.004 and OR = 1.19 and Combined data: <em>P</em> = 6.8e<sup>−09</sup> and OR = 1.19), for TB in the study of combined Gambia and Ghanaian populations (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137465">13</a>), did not yield an association with TB in our study (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>). We obtained <em>P</em> = 0.83, MAF = 0.19 and OR = 1.00, and no suggestive signals in the SAC data located near the variant at SNP <em>rs4331426</em> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>). As above, we computed <em>r</em><sup>2</sup> in the region of 18q11.2 in the data of the SAC, CEU, YRI, JPT + CHB, GIH and SAN. Four SNPs <em>rs4264496</em>, <em>rs4331426</em>, <em>rs4239431</em> and <em>rs4239432</em> in the entire region of 18q11.2 have <em>r</em><sup>2</sup>&gt; = 0.5, but all have weak <em>P</em>-values from the association study with TB (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>) in the SAC data. In addition, the <em>rs4331426</em> SNP is not in LD with any SNPs in <em>WT1</em> locus in the data of the SAC, CEU, YRI and JPT + CHB. One possible explanation for our failure to replicate this locus could be our low power to detect the reported odds ratio (power = 0.12). However, the published odds ratio of <em>rs4331426</em> is not included in the 95% CI (CI) estimated in the SAC dataset (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>), suggesting that our non-replication was not due solely to low power. Different LD patterns between the SAC and Gambia/Ghana/Malawi populations provide an alternative explanation.</p><p id="__p21" class="p p-last">To compare our study with previous findings of association with TB susceptibility at four polymorphisms in the <em>TLR8</em> gene on the X chromosome from Davila <em>et al</em>. 2008 (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140985">12</a>), we conducted an additional imputation GWAS on the non-pseudoautosomal region (nonPAR) and two pseudoautosomal regions (PAR1 and PAR2) of the X chromosome in the SAC. The results displayed in Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a> compare our results with those from Davila <em>et al</em>. (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140958">12</a>). Our imputation GWAS suggests a non-significant trend towards association of these four polymorphisms in the <em>TLR8</em> gene on the X chromosome, including <em>rs3764880</em>, <em>rs3764879</em>, <em>rs3761624</em> and <em>rs3788935</em> (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>). These four SNPs are in LD with one another (<em>r</em><sup>2</sup>&gt; = 0.5) in the data of the SAC, additional SNPs in LD with <em>rs3788935</em> (the reported TB associated SNPs in Davila <em>et al</em>. 2008 (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140926">12</a>)) in the SAC, CEU, YRI, JPT + CHB, GIH and SAN data and their related <em>P</em>-values from the SAC association study are provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>. Because we had incomplete power to replicate the published odds ratio (power = 0.55), and the published odds ratio in Davila <em>et al</em>. (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140993">12</a>) lies within the 95% CI estimated in the SAC dataset (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>), the lack of a statistically significant association in our data may be due to incomplete power.</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">Meta-analysis</h3><p id="__p22" class="p p-first">Identifying common variants of modest effect is a challenge, and large sample size is a way to increase power. The sample sizes of TB cases and controls in this study do not provide sufficient power to obtain associations at a stringent level of statistical significance. To increase the power to detect common variants, we combined the current dataset with existing data in order to fine-map existing signals by performing imputation on the WTCCC-TB (Gambia/Ghana/Malawi) data and including the imputed SNPs in fine-mapping analyses. Merging the two genome-wide imputation GWAS datasets (SAC-TB and WTCCC-TB), a total of 1 009 364 autosomal SNPs were meta-analyzed across the two studies (SAC-TB and WTCCC-TB). Since admixture LD in the SAC and possible different LD patterns between the SAC and Gambia/Ghana/Malawi samples can cause heterogeneity, we assessed the degree of heterogeneity in the combined studies in order to choose an appropriated meta-analysis method (fixed or random effect). We obtained 84.80% of variability due to heterogeneity between studies (SAC-WTCCC) rather than sampling error and a total amount of heterogeneity (Tau-square heterogeneity estimator of DerSimonian–Laird) was 0.0288 (SE = 0.0226), indicating a high level of heterogeneity in our combined studies.</p><p id="__p23">To account for the between-study heterogeneity, we applied both random and binary-effects methods implemented in the MetaSoft program (<a href="#DDT462C33" rid="DDT462C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141010">33</a>) (see Materials and Methods). We report results of random and binary-effect meta-analyses (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB2" rid-ob="ob-DDT462TB2" co-legend-rid=""><span>2</span></a>). We also obtained reasonable inflation rates from the random-effect (<em>λ</em><sub>GC</sub> = 1.062), binary-effect (<em>λ</em><sub>GC</sub> = 1.05) and from each individual study, SAC-TB (<em>λ</em><sub>GC</sub> = 1.094) and WTCCC-TB (<em>λ</em><sub>GC</sub> = 1.0495), respectively (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). In addition to standard <em>P</em>-values, we also examined the posterior probability (<em>m</em>-value) that the effect exists in each study (<a href="#DDT462C33" rid="DDT462C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140952">33</a>). Using a threshold <em>m</em> &gt; 0.7, we observed two genetic variants, <em>rs2057178</em> and <em>rs11031728</em> (Fig.&nbsp;<a href="/pmc/articles/PMC3888262/figure/DDT462F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT462F3" rid-ob="ob-DDT462F3" co-legend-rid="lgnd_DDT462F3"><span>3</span></a> and Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB2" rid-ob="ob-DDT462TB2" co-legend-rid=""><span>2</span></a>) with similar <em>P</em>-values to those from the standard GWAS (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>), that resulted in a significant risk of TB and had effects in our study and the Thye <em>et al</em>. study (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522858">14</a>). These SNPs are both on chromosome region 11p13 and replicate the recent findings of Thye <em>et al</em>. (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522846">14</a>) in our imputation GWAS.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT462TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p24">Imputation genome-wide meta-analysis of two TB case–control studies, SAC-TB and WTCCC-TB</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013674755616" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="left">SAC TB<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">CHR</th><th rowspan="1" colspan="1" align="left">P_RAN</th><th rowspan="1" colspan="1" align="left">P_BE</th><th rowspan="1" colspan="1" align="left">ST1</th><th rowspan="1" colspan="1" align="left">ST2</th><th rowspan="1" colspan="1" align="left"><em>P-</em>value</th><th rowspan="1" colspan="1" align="left"><em>M-</em>value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>rs2057178</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">3.26e<sup>−3</sup></td><td rowspan="1" colspan="1">9.83e<sup>−13</sup></td><td rowspan="1" colspan="1">53.05</td><td rowspan="1" colspan="1">2.91</td><td rowspan="1" colspan="1">2.75e<sup>−06</sup></td><td rowspan="1" colspan="1">1.0</td></tr><tr><td rowspan="1" colspan="1"><em>rs11031728</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">4.73e<sup>−07</sup></td><td rowspan="1" colspan="1">4.08e<sup>−10</sup></td><td rowspan="1" colspan="1">41.19</td><td rowspan="1" colspan="1">0.0</td><td rowspan="1" colspan="1">2.98e<sup>−06</sup></td><td rowspan="1" colspan="1">0.988</td></tr><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td colspan="2" rowspan="1">SAC TB</td></tr><tr><td rowspan="1" colspan="1"><em>rs4331426</em></td><td rowspan="1" colspan="1">18</td><td rowspan="1" colspan="1">0.282554</td><td rowspan="1" colspan="1">1.8950e<sup>−08</sup></td><td rowspan="1" colspan="1">1.15</td><td rowspan="1" colspan="1">32.6</td><td rowspan="1" colspan="1">0.002</td><td rowspan="1" colspan="1">0.0</td></tr></tbody></table></div><div id="largeobj_idm140013674755616" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3888262/table/DDT462TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140013676493056"><p id="__p25" class="p p-first-last">P_RAN is the <em>P</em>-value of random effect, P_BE is the <em>P</em>-value of binary effect, ST1 and ST2 are the statistic mean effect and heterogeneity, respectively. <em>M-</em>value is the posterior probability that the effect exists in each study. The <em>P</em>-value of rs4331426 differs from that in Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a> due to Metasoft fitting it together with the Thye <em>et al</em>. studies.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT462F3" co-legend-rid="lgnd_DDT462F3"><a href="/pmc/articles/PMC3888262/figure/DDT462F3/" target="figure" rid-figpopup="DDT462F3" rid-ob="ob-DDT462F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013676599904" class="figure"><a href="/pmc/articles/PMC3888262/figure/DDT462F3/" target="figure" rid-figpopup="DDT462F3" rid-ob="ob-DDT462F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3888262_ddt46203.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddt46203.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3888262/bin/ddt46203.jpg"></a></div><div id="largeobj_idm140013676599904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3888262/figure/DDT462F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT462F3"><div><a class="figpopup" href="/pmc/articles/PMC3888262/figure/DDT462F3/" target="figure" rid-figpopup="DDT462F3" rid-ob="ob-DDT462F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p26">Forest plot of combined TB studies, including our TB study, Thye <em>et al</em>. (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137463">13</a>,<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522866">14</a>).</p></div></div></div><p id="__p27">We evaluated whether trans-ethnic meta-analysis can refine or confirm the replicated association signals by narrowing the genomic regions at locus <em>WT1</em> where functional variants might be expected to reside. To address this, we re-imputed SNPs at chromosome region 11p13 (32409321–32457176 bp) from both WTCCC-TB and SAC-TB. <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a> displays GWAS results, the patterns of linkage disequilibrium in region in both ethnic groups (WTCCC-TB and SAC-TB). Although the imputations were moderately accurate (CALL ∼0.8, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>), all the imputed variants at locus <em>WT1</em> were in LD with the lead SNP <em>rs2057178</em> in the SAC.</p><p id="__p28">Our trans-ethnic fine mapping (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S6</a>) based on the imputation meta-analysis at chromosome region 11p13 confirmed the association signals at locus <em>WT1</em> and facilitated the localization of neighbouring variants in LD that may influence trait variability at loci of <em>WT1</em>. The between-study heterogeneity due to different patterns of LD and imputation accuracy are major limitations for fine mapping of GWAS signals (<a href="#DDT462C26" rid="DDT462C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140998">26</a>,<a href="#DDT462C34" rid="DDT462C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140923">34</a>), nevertheless this current result (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S3 and S6</a>) demonstrated that fine mapping is a powerful approach to better characterize TB susceptible loci risk in diverse populations.</p><p id="__p29" class="p p-last">Additional meta-analysis of our TB study and four polymorphisms in the <em>TLR8</em> gene on chromosome X previously identified in Davila <em>et al</em>. (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140997">12</a>) are found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S2</a>. Although Metasoft provides a slightly different <em>P</em>-value (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB2" rid-ob="ob-DDT462TB2" co-legend-rid=""><span>2</span></a>) for SNP <em>rs4331426</em> than the one from the standard GWAS (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB1" rid-ob="ob-DDT462TB1" co-legend-rid=""><span>1</span></a>), which may be due to high heterogeneity (ST2 = 32.6, see Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB2" rid-ob="ob-DDT462TB2" co-legend-rid=""><span>2</span></a>), this susceptibility locus reported in Thye <em>et al</em>. 2010 (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137467">13</a>) does not survive genome-wide significance in the TB meta-analysis of the SAC and Thye <em>et al</em>. 2010 (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137466">13</a>). Moreover, the combined SAC TB study and TB susceptibility at four polymorphisms in the <em>TLR8</em> gene on X chromosome in the Indonesian population reported in Davila <em>et al</em>. 2008 by meta-analysis do not yield any convincing association evidence. The failure may be mainly due to low power (56.2 and 12.3%) to detect the four polymorphisms in the <em>TLR8</em> gene on chrX and locus at 18q11.2 associations using the meta-analysis sample size, respectively. In addition, variability of patterns of LD across distinct ethnic groups (SAC, Gambia/Ghana/Malawi and Indonesian) might explain our replication failure.</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">Relationship between TB risk and genetic ancestry</h3><p id="__p30" class="p p-first">We estimated the proportion of genome-wide ancestry in 733 unrelated SAC individuals (642 cases and 91 controls) from these five proxy ancestral populations (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>), including European (CEU), Bantu Africans (YRI), SAN Africans, Gujarati Indian (GIH) and Chinese/Japanese (CHB + JPT). Our choice of the proxy ancestral populations of the SAC is based on previous findings (<a href="#DDT462C21" rid="DDT462C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140953">21</a>–<a href="#DDT462C23" rid="DDT462C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522832">23</a>) (Materials and Methods). In particular, Patterson <em>et al</em>. (<a href="#DDT462C22" rid="DDT462C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141009">22</a>) demonstrated that the admixed SAC population includes an east Asian related component. They inferred European, south Asian and Indonesian components of non-African ancestry. It is appropriate to model the Indonesian component using an east Asian population, because Patterson <em>et al</em>. (<a href="#DDT462C22" rid="DDT462C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140942">22</a>) reported an <em>F</em><sub>ST</sub> value of 0.02 between Indonesians and east Asians, which is much lower than the <em>F</em><sub>ST</sub> value of 0.07 between Indonesians and south Asians or the <em>F</em><sub>ST</sub> value of 0.10 between Indonesians and Europeans. Similarly, Patterson <em>et al</em>. (<a href="#DDT462C22" rid="DDT462C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140983">22</a>) and De Wit <em>et al</em>. (<a href="#DDT462C21" rid="DDT462C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140944">21</a>) demonstrated that the SAC population includes both SAN African and Bantu African components.</p><p id="__p31">To examine the relationship between genetic ancestry and TB status in this SAC dataset, we regressed case–control status against the estimated fraction of YRI, SAN, GIH, CHB + JPT and CEU ancestry, respectively, in 733 unrelated SAC individuals (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB3" rid-ob="ob-DDT462TB3" co-legend-rid=""><span>3</span></a>). We observed a statistically significant positive correlation (<em>r</em> 95%CI = 0.16 [0.09, 0.23], <em>P</em> = 1.58e<sup>−05</sup>) between SAN ancestry and TB status. The CEU (<em>r</em> 95% = −0.12 [−0.19, −0.05], <em>P</em> = 0.0007), CHB + JPT (<em>r</em> 95%CI = −0.13 [−0.20, −0.06], <em>P</em> = 0.0005) and GIH (<em>r</em> 95% = −0.11 [−0.18, −0.04], <em>P</em> = 0.002) ancestry in the SAC were negatively correlated with TB status. YRI ancestry proportion was not significantly correlated (r 95%CI = 0.06 [−0.01, 0.13], <em>P</em> = 0.109) with TB. Furthermore, we observed a statistically significant correlation of age (<em>P</em> = 1.01e<sup>−05</sup>, mean age 37 in cases and 31 in controls) with risk of TB, but no evidence of correlation between sex and TB (<em>P</em> = 0.597).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT462TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p32">Association of genetic ancestry with TB in the SAC, with nominal <em>P</em>-values before correcting for hypotheses tested</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013675626336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th colspan="2" rowspan="1" align="left">Association of genetic ancestry with TB<hr></th><th colspan="3" rowspan="1" align="left">Genome-wide ancestry</th></tr><tr><th rowspan="1" colspan="1" align="left">POP</th><th rowspan="1" colspan="1" align="left">Correlation 95% CI, <em>P</em>-value (TB-ancestry)</th><th rowspan="1" colspan="1" align="left">Cases</th><th rowspan="1" colspan="1" align="left">Controls</th><th rowspan="1" colspan="1" align="left">All samples</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">SAN</td><td rowspan="1" colspan="1">0.16 [0.09,0.23],1.58e − 05</td><td rowspan="1" colspan="1">0.342 ± 0.135</td><td rowspan="1" colspan="1">0.279 ± 0.114</td><td rowspan="1" colspan="1">0.334 ± 0.134</td></tr><tr><td rowspan="1" colspan="1">YRI</td><td rowspan="1" colspan="1">0.06 [−0.01, 0.13], 0.109</td><td rowspan="1" colspan="1">0.279 ± 0.163</td><td rowspan="1" colspan="1">0.250 ± 0.143</td><td rowspan="1" colspan="1">0.275 ± 0.161</td></tr><tr><td rowspan="1" colspan="1">CEU</td><td rowspan="1" colspan="1">−0.12 [−0.19, −0.05], 0.0007</td><td rowspan="1" colspan="1">0.184 ± 0.116</td><td rowspan="1" colspan="1">0.228 ± 0.124</td><td rowspan="1" colspan="1">0.190 ± 0.118</td></tr><tr><td rowspan="1" colspan="1">GIH</td><td rowspan="1" colspan="1">−0.11 [−0.18, −0.04], 0.002</td><td rowspan="1" colspan="1">0.127 ± 0.086</td><td rowspan="1" colspan="1">0.156 ± 0.079</td><td rowspan="1" colspan="1">0.130 ± 0.086</td></tr><tr><td rowspan="1" colspan="1">CHB + JPT</td><td rowspan="1" colspan="1">−0.13 [−0.20, −0.06], 0.0005</td><td rowspan="1" colspan="1">0.069 ± 0.047</td><td rowspan="1" colspan="1">0.087 ± 0.054</td><td rowspan="1" colspan="1">0.071 ± 0.048</td></tr></tbody></table></div><div id="largeobj_idm140013675626336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3888262/table/DDT462TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140013683002352"><p id="__p33" class="p p-first-last">Mean and standard error of ancestry proportion from each of five populations contributing to the admixture in the SAC (from cases, controls and all samples of the SAC.).</p></div></div></div><p id="__p34">We observed a correlation between ancestry proportions of the ancestral populations (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). We checked to see if the tests above could be confounded by this correlation by performing conditional risk tests between pairs of the ancestral populations (see Materials and Methods). Our results demonstrate that African ancestry (SAN, YRI) TB risk in the SAC is not significant conditioned on non-African ancestry (CEU and JPT–CHB) risk. With the exception of Indian (GIH) ancestry, non-African ancestry (CEU and JPT–CHB) risk is significant conditioned on African ancestry risk. This shows that YRI and SAN are differently correlated with risk than CEU, JPT–CHB and GIH (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). JPT–CHB, GIH and CEU are not significantly conditioned on each other and all are correlated with TB risk (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). We see that SAN confers risk, CEU, JPT–CHB and GIH confer protection, and YRI shows no evidence of correlation (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>).</p><p id="__p35" class="p p-last">Another potential concern was that the observed relationship between genetic ancestry and TB status could be a consequence of confounding due to SES, as described in a recent study of type 2 diabetes in Latinos (<a href="#DDT462C25" rid="DDT462C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140922">25</a>). We investigated this possibility by studying two SES variables (see Materials and Method), household and individual incomes (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). Although these variables were available in only a subset of 82 SAC cases, it was possible to draw a conclusion about whether or not there exists both a hypothetical difference in SES in cases versus controls (which we cannot quantify) and a correlation between ancestry and SES (which we can quantify), as both effects would need to be present in order for the ancestry analysis to be confounded by SES. When testing for correlations between each of the two SES variables and each of the five ancestries, none of the results were statistically significant after correcting for 10 hypotheses tested (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). In particular, SAN ancestry had a non-significant trend towards positive correlation (<em>r</em> = 0.05, 95% CI = [−0.17, 0.27], <em>P</em> = 0.65) and (<em>r</em> = 0.09, 95% CI = [−0.13, 0.31], <em>P</em> = 0.45) with SES self and household incomes, respectively. Although TB status is known to be associated with low SES (<a href="#DDT462C35" rid="DDT462C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140965">35</a>–<a href="#DDT462C38" rid="DDT462C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522839">38</a>), even considering the low end of the 95% CI of −0.17 (for SES self income) and −0.13 (for SES household income) this still could not explain the correlation (<em>r</em> = 0.16, 95% CI = [0.09, 0.23], <em>P</em> = 1.58e<sup>−05</sup>) between SAN ancestry and TB status unless there is a virtually perfect negative correlation between SES and TB status, which is highly unlikely. Thus, the observed association between SAN ancestry and TB status (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB3" rid-ob="ob-DDT462TB3" co-legend-rid=""><span>3</span></a>) is not entirely a consequence of SES.</p></div><div id="s2e" class="sec"><h3 id="s2etitle">Accuracy of multi-way local ancestry inference in the SAC</h3><p id="__p36" class="p p-first-last">Recently introduced locus-specific ancestry methods for multi-way admixed populations, including ALLOY (<a href="#DDT462C39" rid="DDT462C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140930">39</a>), PCAdmix (<a href="#DDT462C40" rid="DDT462C40" class=" bibr popnode">40</a>), MULTIMIX (<a href="#DDT462C41" rid="DDT462C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140992">41</a>) and ChromoPainter (<a href="#DDT462C42" rid="DDT462C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140947">42</a>) achieved equivalent accuracy to WinPOP (<a href="#DDT462C43" rid="DDT462C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522853">43</a>) or Lamp-LD (<a href="#DDT462C44" rid="DDT462C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140999">44</a>). Therefore, we assessed the accuracy of two methods for inferring local ancestry in multi-way admixed populations, Lamp-LD (<a href="#DDT462C44" rid="DDT462C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140954">44</a>) and WinPOP. We simulated 750 individuals of mixed European (CEU), Chinese and Japanese (CHB + JPT), Bantu (YRI) and SAN ancestry on chromosome 1 (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods Text 1</a>). We assessed the accuracy from the inferred local ancestry from both Lamp-LD and WinPOP by comparing an estimate of correlation between the inferred local ancestry from Lamp-LD (CEU: 86%, YRI: 78%, GIH: 84%, CHB + JPT: 90% and SAN: 80%) and WinPOP (CEU: 58%, YRI: 69%, GIH: 51%, CHB + JPT: 48% and SAN: 47%) and the true ancestry information by computing the accuracy (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods Text 1</a>). The results suggest that Lamp-LD provides greater accuracy than WinPOP (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figs S4 and S5</a>). Given this, we then assessed the ability of Lamp-LD to correctly infer the local ancestry in a multi-way admixed population by computing the rate of calling true ancestral and the error of calling other ancestral populations instead of the true ones (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods Text 1</a>). <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a> demonstrates that even Lamp-LD does not correctly estimate the true local ancestry in a multi-way admixed population. <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a> shows that the true CEU ancestry in the admixed population (simulation data) is miscalled as GIH ancestry (17%) more often than true GIH ancestry is miscalled as CEU ancestry (9.4%). The true SAN ancestry in the admixed population (simulation data) is miscalled as YRI ancestry (15.8%) slightly more often than true YRI ancestry is miscalled as SAN ancestry (14.2%). Due to the limitation in accuracy of these methods in inferring local ancestry in multi-way admixed populations, we conclude that case-only admixture mapping is impractical in the multi-way admixed populations such the SAC.</p></div><div id="s2f" class="sec sec-last"><h3 id="s2ftitle">Deviation in local ancestry in the SAC</h3><p id="__p37" class="p p-first-last">A major limitation of admixture mapping and admixture association is that the inference of locus-specific ancestry in complex multi-way admixed populations such as the SAC may suffer from spurious deviations in average local ancestry at particular chromosomal regions from cases and controls, resulting in spurious case-only admixture associations (<a href="#DDT462C45" rid="DDT462C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140940">45</a>,<a href="#DDT462C46" rid="DDT462C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522854">46</a>). Simulation may not measure systematic inaccuracies arising from the violations of model assumptions such as random mating, number of generations since admixture, and recombination rate (<a href="#DDT462C45" rid="DDT462C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140979">45</a>,<a href="#DDT462C46" rid="DDT462C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522842">46</a>) in estimating local ancestry. Here, we conducted a real data-based evaluation of local ancestry using the SAC population. We evaluated the deviations in local ancestry inferred from Lamp-LD (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Materials and Methods Text 2</a>). Several SNPs in a total of 69 chromosomal regions (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>) showed strong deviation in local ancestry (4 standard deviations above or below the genome-wide average) in the SAC. These inferred local ancestries regions with large deviation are consistent between cases and controls (Fig.&nbsp;<a href="/pmc/articles/PMC3888262/figure/DDT462F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDT462F4" rid-ob="ob-DDT462F4" co-legend-rid="lgnd_DDT462F4"><span>4</span></a>), indicating that the deviations in cases are not true disease associations and could lead to spurious admixture associations if case-only admixture mapping is employed (<a href="#DDT462C45" rid="DDT462C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140989">45</a>,<a href="#DDT462C46" rid="DDT462C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522840">46</a>). We conclude that case-only admixture mapping is currently impractical in multi-way admixed populations such the SAC.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDT462F4" co-legend-rid="lgnd_DDT462F4"><a href="/pmc/articles/PMC3888262/figure/DDT462F4/" target="figure" rid-figpopup="DDT462F4" rid-ob="ob-DDT462F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013672640736" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddt46204.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddt46204.jpg" src="/pmc/articles/PMC3888262/bin/ddt46204.jpg"></div></a><div id="largeobj_idm140013672640736" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC3888262/figure/DDT462F4/" target="figure" rid-figpopup="DDT462F4" rid-ob="ob-DDT462F4"></a><a target="object" href="/pmc/articles/PMC3888262/figure/DDT462F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDT462F4"><div><a class="figpopup" href="/pmc/articles/PMC3888262/figure/DDT462F4/" target="figure" rid-figpopup="DDT462F4" rid-ob="ob-DDT462F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p38">Deviations in the average local ancestry (DLA) in the admixed South African Coloured population (4 standard deviations above or below the genome-wide average). We plot regions of strong deviation in average local ancestry in case (<em>x</em>-axis) versus control (<em>y</em>-axis) samples of the SAC. The plot shows spurious deviations in average local ancestry (e.g. regions in which the modelled ancestral population is unusually different from the true ancestral population due to the historical action of natural selection).</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p39" class="p p-first">We conducted a genome-wide association analysis of TB case–controls from the admixed SAC population, resulting in the identification of a low-frequency variant at the <em>rs17175227</em> SNP. After imputation we also identified a rare variant at <em>rs12294076</em> SNP at the borderline of genome-wide significance and we moderately replicated a recently reported susceptibility locus, <em>rs2057178</em>. Similar results were obtained when including age and gender as covariates in the analysis (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). Power to detect association is a function of allele frequency and rare variants are underpowered when sample sizes are limited. Because of the imperfect asymptotic distribution of mixed model association or logistic regression in the specific case of low-frequency variants, which may often reach genome-wide significance; we computed Fisher's exact test values for variants that achieved the most significant mixed model association <em>P</em>-values. This resulted in <em>rs17175227</em> not reaching the genome-wide cut-off. Importantly, Fisher's exact test allowed us to demonstrate that a rare variant is not genome-wide significant although it achieved significant mixed model association <em>P</em>-values. In general, the analysis of rare or low-frequency variants poses challenges at any dataset of smaller to larger sample size. Fisher's exact test, as its name implies, may be a well-calibrated test for rare SNPs and works fine with small sample sizes. However, Fisher's exact test lacks flexibility to handle multiple predictors, does not account for the contribution of the sample structure to the phenotype and may inflated. Our study addressed these issues. Until such time as GWAS is able to account for rare variants, our current recommendation is to apply Fisher's exact test to the few low-frequency variants that are shown to be significant from the mixed model in GWAS. For combined typed and imputation GWAS, Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB4" rid-ob="ob-DDT462TB4" co-legend-rid=""><span>4</span></a> displays the top 24 genetic markers with suggestive <em>P</em>-values (&lt; or equal to 10<sup>−06</sup>) obtained from the association analysis with the TB phenotype in the SAC, including some previously reported TB SNPs that we discuss in the next section. An additional 36 typed and 62 imputed SNPs attained suggestive levels of significance (10<sup>−05</sup>–10<sup>−06</sup>), and can be found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S1 and S2</a>, respectively.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDT462TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p40">Top 24 genetic markers obtained from the association analysis with the TB phenotype (typed and imputed datasets)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140013670345984" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">CHR</th><th rowspan="1" colspan="1" align="left">POS</th><th rowspan="1" colspan="1" align="left">A1/A2</th><th rowspan="1" colspan="1" align="left">Call</th><th rowspan="1" colspan="1" align="left">INFO</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">OR</th><th rowspan="1" colspan="1" align="left">Closest Gene</th><th rowspan="1" colspan="1" align="left">Type</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>rs10916338</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">228715923</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">4.74e<sup>−06</sup></td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">RNF187</td><td rowspan="1" colspan="1">Typed</td></tr><tr><td rowspan="1" colspan="1"><em>rs1925714</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">226769912</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1.80e<sup>−07</sup></td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">RNF187</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs6676375</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">240941508</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">7.58e<sup>−07</sup></td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">PLD5</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs1075309</em></td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">5253598</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">7.07e<sup>−07</sup></td><td rowspan="1" colspan="1">0.35</td><td rowspan="1" colspan="1">SOX11</td><td rowspan="1" colspan="1">Typed</td></tr><tr><td rowspan="1" colspan="1"><em>rs2202157</em></td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">26518424</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">0.65</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">7.80e<sup>−07</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">Unknown</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs958617</em></td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">78824393</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.74</td><td rowspan="1" colspan="1">0.50</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">2.90e<sup>−07</sup></td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">CNOT6L</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs2505675</em></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">2300674</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.87</td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">3.87e<sup>−06</sup></td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">LOC100508120</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs17217757</em></td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">106613321</td><td rowspan="1" colspan="1">C/G</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.18</td><td rowspan="1" colspan="1">3.09e<sup>−06</sup></td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">ZFPM2</td><td rowspan="1" colspan="1">Typed</td></tr><tr><td rowspan="1" colspan="1"><em>rs1934954</em></td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">96792202</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">2.81e<sup>−06</sup></td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">CYP2C8</td><td rowspan="1" colspan="1">Typed</td></tr><tr><td rowspan="1" colspan="1"><em>rs12283022</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">102485804</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.76</td><td rowspan="1" colspan="1">0.48</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">1.88e<sup>−06</sup></td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">DYNC2H1</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs12294076</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">102632275</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.85</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">9.57e<sup>−08</sup></td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">DYNC2H1</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs2057178</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32364187</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">0.84</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">2.71e<sup>−06</sup></td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">WT1</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs11031728</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">32363616</td><td rowspan="1" colspan="1">C/G</td><td rowspan="1" colspan="1">0.843</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">2.86e<sup>−06</sup></td><td rowspan="1" colspan="1">0.61</td><td rowspan="1" colspan="1">WT1</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs7105967</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">102434653</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.45</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">3.51e<sup>−06</sup></td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">DCUN1D5</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs7947821</em></td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">102452675</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.47</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">1.95e<sup>−06</sup></td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">DCUN1D5</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs6538140</em></td><td rowspan="1" colspan="1">12</td><td rowspan="1" colspan="1">76262136</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.81</td><td rowspan="1" colspan="1">0.63</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">4.46e<sup>−06</sup></td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">E2F7</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs1900442</em></td><td rowspan="1" colspan="1">13</td><td rowspan="1" colspan="1">41403674</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">4.72e<sup>−06</sup></td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">VWA8</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs17175227</em></td><td rowspan="1" colspan="1">14</td><td rowspan="1" colspan="1">70502050</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">8.99e<sup>−09</sup></td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">SMOC1</td><td rowspan="1" colspan="1">Typed</td></tr><tr><td rowspan="1" colspan="1"><em>rs40363</em></td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">3449057</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">0.51</td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">3.13e<sup>−06</sup></td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">NAA60</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs2837857</em></td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">41138825</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.65</td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">2.40e<sup>−06</sup></td><td rowspan="1" colspan="1">0.30</td><td rowspan="1" colspan="1">DSCAM</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs451390</em></td><td rowspan="1" colspan="1">21</td><td rowspan="1" colspan="1">42216579</td><td rowspan="1" colspan="1">C/G</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">1.58e<sup>−07</sup></td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">C2CD2</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs3218255</em></td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">35874432</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1">0.31</td><td rowspan="1" colspan="1">2.32e<sup>−07</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">IL2RB</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs5928363</em></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">33784063</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">3.72e<sup>−06</sup></td><td rowspan="1" colspan="1">0.12</td><td rowspan="1" colspan="1">Unknown</td><td rowspan="1" colspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1"><em>rs142513793</em></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">47906480</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.84e<sup>−07</sup></td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">Unknown</td><td rowspan="1" colspan="1">Imputed</td></tr></tbody></table></div><div id="largeobj_idm140013670345984" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3888262/table/DDT462TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140013747332064"><p id="__p41" class="p p-first-last">A1/A2 are reference/derived alleles.</p></div></div></div><p id="__p42">Imputation-based meta-analysis, combining data from the GWAS conducted at different locations, was successful in a recent study of severe malaria in three African populations (<a href="#DDT462C47" rid="DDT462C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140981">47</a>). To achieve sufficient power and identify shared risk loci with a previously reported African TB case–control studies (<a href="#DDT462C13" rid="DDT462C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_568137468">13</a>,<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522863">14</a>), a GWAS meta-analysis was performed under random-effect and binary-effecs models. In combining GWAS data across these studies, two loci (<em>rs2057178</em> and <em>rs11031728</em>) had an association result with genome-wide significance, and showed strong effects in both our study and the previous African TB case–control study (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522865">14</a>).</p><p id="__p43">We used a combination of two complementary methods to examine whether the genetic ancestry contribution can increase TB risk, and evaluated the contribution of SES to the ancestry-TB relationship in the SAC. Our results demonstrated significant evidence of an association between SAN ancestry and TB status that is not confounded by SES. This is an important epidemiological result and illustrates the value of the inclusion of admixture association methods in the set of methods used to conduct TB association studies in this population. When the extremely high incidence of TB in the SAC population is considered, together with our finding that a significant percentage of their ancestry is derived from the SAN and other African populations, it appears possible that there is an element of population level genetic susceptibility to this disease. Although, our study had only 82 cases with SES information, it was possible to draw a conclusion about the extent to which both a hypothetical difference in SES in cases versus controls (which we could not quantify) and a correlation between ancestry and SES (which we quantified) could lead to the ancestry analysis being confounded by SES. Specifically, if ancestry and SES were correlated, this correlation would exist irrespective of TB status and would exist in both cases and controls, regardless of whether or not there is a mean SES difference between cases and controls. We justified our conclusion by the results we presented in Florez <em>et al</em>. (<a href="#DDT462C25" rid="DDT462C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140982">25</a>), in which they observed a similar correlation between ancestry and SES in both cases and controls.</p><p id="__p44">Our study is the first ancestry-specific GWAS of TB risk in the complex admixed SAC population. Our subsequent imputation-based meta-analysis and trans-ethnic fine mapping including other GWAS on African populations confirmed loci identified previously. The results (Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB2" rid-ob="ob-DDT462TB2" co-legend-rid=""><span>2</span></a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S3 and S6</a>) demonstrated that trans-ethnic fine mapping is a powerful approach to better characterize TB susceptible loci in diverse populations.</p><p id="__p45">Overall, some limitations should be noted in our current study. First, the present study is underpowered to detect risk variants of small effect size, because of our modest sample size. However, we addressed these issues by computing the power to replicate TB published odds ratio and for each published association we specified whether or not the published odds ratio lies within the 95% CI estimated in our current dataset. This allows us to interpret the failure to replicate previous TB findings in our study. We detected an association at the previously identified <em>WT1</em> locus that is almost genome-wide significant in our data despite the limited sample size. Second, imputing missing genotype data of a complex admixed population is an important challenge with several issues related to the choice and size of existing reference panels. In particular, the imputation of missing genotype data of the admixed SAC population was suboptimal. Nonetheless, the increased number of SNPs generated by imputation analyses was useful in this study, yielding the replication of TB susceptibility loci (<a href="#DDT462C14" rid="DDT462C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522855">14</a>). Third, despite applying Fisher's exact test to address the case of rare variants, the implementation of newer sequencing technologies is still required to search for rare risk variants. This may potentially provide crucial insights in identifying TB susceptibility genes and, therefore, inform the development of novel interventions. Fourth, the between-study heterogeneity due to divergent evolutionary and migratory histories, variability of patterns of linkage disequilibrium across distinct ethnic groups, and the differing imputation accuracy are major limitations for fine mapping of GWAS signals in multi-way admixed populations such as the SAC. Lastly, a major limitation of admixture mapping and case-only admixture association analysis is currently due to errors in inference of local ancestry in complex multi-way admixed populations such as the SAC. Our simulation result demonstrated that existing methods may attain high accuracy on average but may suffer from spurious deviations in average local ancestry at particular regions. These deviations would be present in both cases and controls, and would lead to spurious case-only admixture associations.</p><p id="__p46" class="p p-last">Overall, our GWAS provides promising results and suggests that more power can be obtained by designing an extension-association model that can combine both SNP case–control and admixture in the SAC. Priorities for future work will be to examine an accurate, unbiased estimation of the ancestry at every SNP in this multi-way admixed population to potentially provide crucial insights into identifying disease genes. This will provide a method to account for a combined genome-wide SNP case–control and admixture analysis in a multi-way admixed population such as the SAC.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Samples and genotype quality control</h3><p id="__p47" class="p p-first">The SAC population under study is located in the metropolitan area of Cape Town in the Western Cape Province in South Africa (<a href="#DDT462C12" rid="DDT462C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140949">12</a>). Since the ethnicity, SES and HIV infection may be confounders in TB association studies (<a href="#DDT462C2" rid="DDT462C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522847">2</a>–<a href="#DDT462C4" rid="DDT462C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140933">4</a>), this area was selected due to the high incidence of TB as well as the uniform ethnicity, SES and low prevalence of HIV (<a href="#DDT462C1" rid="DDT462C1" class=" bibr popnode">1</a>). This is due to the following reasons:
</p><ol class="enumerated" style="list-style-type:decimal"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><p id="__p48">Uniform ethnicity and SES status are important in disease association studies, as it removes some of the confounding variables.</p></li><li><p id="__p49">Low prevalence of HIV is important because in the presence of HIV infection, an individual has a greatly increased chance of progressing to TB disease once infected, simply because of their impaired immune system, and not necessarily because of their genetic susceptibility.</p></li></ol><p></p><p id="__p50">The definition for TB diagnosis and recruitment of appropriate controls for infectious diseases such as TB has been shown to be important in the interpretation of GWAS results (<a href="#DDT462C48" rid="DDT462C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141002">48</a>). Therefore, the TB patients in this study were identified through bacteriological confirmation (smear positive and/or culture positive). Controls were selected from the same community living under the same conditions including SES status and availability of health facilities. These healthy individuals had no previous history of TB disease or treatment. Approval from the Ethics Committee of the Faculty of Health Sciences, Stellenbosch University (project number 95/072) was obtained before blood samples were collected with informed consent, and known HIV-positive individuals were excluded from the study.</p><p id="__p51" class="p p-last">The study samples were genotyped on the Affymetrix 500 K chip and SNP calling was done as described by De Wit <em>et al</em>. (<a href="#DDT462C21" rid="DDT462C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140988">21</a>). Quality-control filters were applied to 797 cases and 91 controls. A total of 6450 SNPs failed the minor allele frequency (MAF &lt; 1%) and missingness test (GENO &gt; 0.05), as well as the HWE test in controls (alpha level 0.0001). We retained 390 887 SNPs for 888 individuals (381 558 autosomal SNPs; 797 cases and 91controls; 489 males of which 444 are cases and 45 controls). Further relatedness analysis using PLINK (<a href="#DDT462C49" rid="DDT462C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140921">49</a>) was conducted and resulted in the removal of 155 related individuals, producing a dataset suitable for methods that assume independent samples. It has 390 887 SNPs for 733 individuals (381 558 autosomal SNPs, 642 cases and 91 controls; 406 males of which 361 are cases and 45 controls). The resulting cases and controls can provide the same amount of power in detecting the association signal as 159 cases and 159 controls. To further check the homogeneity of the samples, we additionally performed the identity-by-state (IBS) permutation test, where case–control labels were permuted, and then recalculated between group metrics based on the average IBS (fixed 10 000 permutations).</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Imputation of missing genotypes</h3><p id="__p52" class="p p-first-last">To account for the population structure in the admixed SAC in imputing the untyped genotypes, we considered the imputation model based on population genetic parameters in the coalescent framework implemented in IMPUTE2 (<a href="#DDT462C26" rid="DDT462C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140959">26</a>). Exploring the advantage of the model in IMPUTE2, we combined all available reference-phased haplotype data from both release 2 of the HapMap 3 dataset (NCBI Build 36) (includes Utah residents (CEPH) with Northern and Western European ancestry (CEU), Japanese in Toyko (JPT), Chinese in Denver, Maasai in Kinyawa, Toscani in Italia (TSI), GIH, African Ancestry in Southwest (ASW), Luhya in Webuye (LWK), Mexican Ancestry in Los Angeles (MEX), Han Chinese in Beijing (CHB) and Yoruba in Ibadan (YRI); and 1000 Genomes Project (<a href="#DDT462C28" rid="DDT462C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140986">28</a>) includes CEU, YRI, British from England and Scotland (GBR), Finnish from Finland (FIN), Han Chinese South (CHS), Puerto Rican (PUR), Chinese in Denver (CHB), JPT, LWK, MXL, ASW, TSI, Colombian in Medellin (CLM) and Iberian populations in Spain (IBS). We decided to impute SNPs by splitting each chromosome into 5 Mb regions for analysis by IMPUTE2. On both resulting imputed datasets, post-imputation quality controls were similarly conducted in order to account for imputation uncertainty.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Admixture estimation and principal component analysis</h3><p id="__p53" class="p p-first-last">For admixture analysis in the SAC, we included a total of 25 unrelated SAN (5 samples of Ju|'hoan from HGDP) and 20 Ju|'hoan samples obtained from Schlebusch <em>et al</em>. (<a href="#DDT462C50" rid="DDT462C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141012">50</a>), and genome-wide SNP data of four populations from the International HapMap 3 project (<a href="#DDT462C27" rid="DDT462C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140974">27</a>), including European (CEU), Yoruba (YRI), Gujarati Indians (GIH) and Chinese and Japanese (CHB + JPT) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). We performed the quality-control filters on each population separately. After the quality-control filter checks, the resulting merged dataset of the SAC within its five putative ancestral populations had 272 798 autosomal SNPs. We additionally removed 37 556 A/T and C/G and 6176 sex SNPs from the merged dataset that contained 272 798 autosomal SNPs, resulting in a dataset of 229 076 SNPs. We additionally remove each SNP that has an LD <em>r</em><sup>2</sup> value of &gt;0.1 with any other SNP within a 50-SNP sliding window (advanced by 10 SNPs at a time). We finally obtained the subset of unlinked SNPs (<em>n</em> = 49930). We then applied the algorithm implemented in ADMIXTURE (<a href="#DDT462C51" rid="DDT462C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141005">51</a>) to determine the individual admixture proportion in the SAC data and in both case–control only data of the SAC. <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S6 (A and B) and Table S3</a> show that genome-wide ancestry varies widely across SAC individuals, SAN (33 ± 1%), YRI (28 ± 2%), CEU (19 ± 1%), GIH (13 ± 0.9%) and CHB + JPT (7 ± 0.5%). To evaluate the extent of substructure in the SAC (separating cases and controls as distinct groups) and examine whether stratification can be accounted for in the GWAS, the smartpca programme in the EIGENSOFT package (<a href="#DDT462C52" rid="DDT462C52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140967">52</a>,<a href="#DDT462C53" rid="DDT462C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141004">53</a>) was applied to the merged data of the 888 SAC samples (also case-only and control-only) and its five ancestral populations (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6C</a>) on autosomal SNPs and SNPs on the X chromosome (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6D</a>). We additionally regressed case–control status from autosomal SNPs against the two first principal components in order to quantify the difference in genetic ancestry between the SAC's case–control individuals. We obtained significant <em>P</em>-values = 3.7e<sup>−06</sup> and 0.002 respectively. Insignificant <em>P</em>-values of 0.332 and 0.317 were obtained from the same analysis applied to the X chromosome. The greatest genetic differentiation between the five proxy ancestral groups and the SAC is in the convex hull of the three non-African populations (GIH, CEU and JPT–CHB) and the two African populations (SAN, YRI). The pair-wise genetic distance between the SAC and each of its reference ancestral populations is also provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>. Of note, the first principal component differentiates the SAC cases and controls. Most of the TB cases are pooled towards African ancestry and controls towards non-African ancestry. This provides evidence of significant differences in genetic ancestry between cases and controls in the SAC data consistent with the result in Table&nbsp;<a href="/pmc/articles/PMC3888262/table/DDT462TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDT462TB3" rid-ob="ob-DDT462TB3" co-legend-rid=""><span>3</span></a>. In addition, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S6D</a> (on X chromosome) shows a strong genetic differentiation between SAN and other ancestral components in the SAC, including non-Africans and African Bantu. This indicates a high genetic contribution from SAN-related ancestral populations on the X chromosome.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Association test and meta-analyses</h3><p id="__p54" class="p p-first">The association testing was performed on the full dataset of 888 individuals, which contained related individuals. To account for both population stratification and hidden relatedness that can result from the genealogy, we applied EMMAX (<a href="#DDT462C29" rid="DDT462C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140969">29</a>), which corrects for these relationships during the association mapping. We first applied EMMAX-kin to compute a pair-wise relatedness matrix from our dataset, which represents the structure of our samples. EMMAX estimated the contribution of the sample structure to the TB phenotype using a variance component model, resulting in an estimated covariance matrix of phenotype that models the effect of genetic relatedness on the TB phenotype. We ran EMMAX on TB phenotype data using the estimated covariance matrix to detect possible association. To account for rare variants that EMMAX could not address adequately, we separately performed Fisher's exact test, which is known to be appropriate for rare SNPs. To cautiously account the lack of flexibility to handle multiple predictors, the contribution of the sample structure to the phenotype and possible inflation in using Fisher's exact test, we only apply this test to few low-frequency variants that show to be significant from the mixed models in GWAS, as this will aid in identifying artefactual associations. This is due to the imperfect asymptotic distribution of those mixed models in the case of low-frequency variants. Given <em>m</em> SNPs (from typed or imputed datasets) for association with TB, we expected around <em>m</em> × 0.05 to have <em>P</em>-value &lt;0.05 in each dataset, respectively. We therefore used a genome-wide significance level of <em>α</em> = 0.05/2 × <em>m</em>.</p><p id="__p55" class="p p-last">To identify associations with small effect sizes, which the standard single GWAS could not identify, we combined two African TB genome-wide association studies (GWASs) including our GWAS, and the recently combined TB study of Ghanaian, Gambian and Malawian populations in a single GWAS analysis. A fixed-effects model (<a href="#DDT462C33" rid="DDT462C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349141001">33</a>) based on the inverse-variance-weighted effect size was used to combine the results (log-odds ratio and standard error) from typed GWAS and two imputation GWASs (imputation based on both the data from HapMap 3 and the 1000 Genomes Project, including the non-pseudoautosomal region (nonPAR) and two pseudoautosomal regions (PAR1 and PAR2) of X chromosome) from the SAC and WTCCC-TB results (SAC-TB and Davila <em>et al</em>. are found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>). We additionally applied random- and binary-effects models described in the MetaSoft program (<a href="#DDT462C33" rid="DDT462C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140976">33</a>), and we used the study <em>P</em>-values, the <em>M</em>-values (the posterior probability that the effect exists in the study), the mean effect and heterogeneity statistics to interpret the association results showing high heterogeneity (<a href="#DDT462C33" rid="DDT462C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140931">33</a>).</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Power to replicate previous findings</h3><p id="__p56" class="p p-first-last">We calculated the power to replicate each published TB SNP in the current case–control study based on the published odds ratios and study minor allele frequencies. The power calculation was carried out using the model implemented in Purcell <em>et al</em>. (<a href="#DDT462C54" rid="DDT462C54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140970">54</a>). For each published association, we specified whether or not the published odds ratio lies within the 95% CI estimated in the current SAC dataset. Power analysis was conducted on the basis of a disease prevalence of 0.01%, the published effect size of each SNP, the minor allelic frequency (MAF) of each SNP observed in the SAC dataset, and the sample size in the SAC dataset. Under a one-sided test, we assumed a significance level of 0.05. We set error rates at 5% for cases, based on general evidence that misdiagnosis rates are often over 5%. To detect flip-flop associations due to different LD patterns and explain the failure of replicate previous TB studies in the current SAC dataset, we specified whether or not the published odds ratio lies within the 95% CI estimated in the current SAC dataset.</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Genetic ancestry and TB risk</h3><p id="__p57" class="p p-first">Socio-economic questionnaires were available for 82 cases and included information on two categories of income, per week self and per week household. These incomes were estimated based on the South African Rand (R) currency. These incomes were coded as follows: 0 = &lt;R50, 1 = R50 to R150, 2 = R150 to R300, 3 = R300 to R500, 4 = R500–1000, 5 ≥ R1000 and 9 = missing. We first computed the fraction of ancestry for each individual from five putative ancestral populations using the program ADMIXTURE (<a href="#DDT462C51" rid="DDT462C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140996">51</a>). To have the correlation and linear regression the same, we normalized all variables to have mean 0 and variance 1, since there exists a relationship between correlation and linear regression. To address this, we subtracted each variable by mean and divided by standard deviation. We separately regressed TB status against genetic ancestry proportion from each ancestral population. We evaluate the correlation between pairs of ancestral populations. To control the correlation between genetic ancestry in the SAC which can be potentially confounded, we test for the difference in TB risk (conditional risk) between pairs of ancestral populations.</p><p id="__p58">Suppose <em>β</em><sub>k</sub> and <em>ɛ</em><sub>k</sub> are the effect size and standard error from the regression model of the fraction of ancestry <em>k</em> in the admixed population against TB binary trait, respectively. To test for the difference in TB risk between a pair of ancestral populations <em>k</em> and <em>l</em>, we had to adapt the normal test statistic under the null hypothesis of no difference in risk between the two ancestral populations. Thus, we computed the <em>Z</em>-score of difference in risk, <em>Z</em><sub>kl</sub> = (<em>β</em><sub>k</sub>−<em>β</em><sub>l</sub>)/sqrt(+), (<em>k</em>≠<em>l</em>) which has a standard normal distribution <em>Z</em><sub>kl</sub> ∼ <em>N</em>(0,1). We computed the probability, (two-sided <em>P</em> = 2*(1 − <em>P</em>(&lt;|<em>Z</em><sub>kl</sub>|)) that the value may be less than the <em>Z</em>-score. To account for the correlation among ancestry proportions in the admixed population, we first conducted a permutation test, whereby the above distribution of the test statistic under the null hypothesis is re-sampled 10 000 times under rearrangements of the case–control status. In addition, we adjusted for the covariance by computing cov<sub>kl</sub> = <em>ρ</em><sub>kl</sub>.<em>ɛ</em><sub>k .</sub><em>ɛ</em><sub>l</sub> where <em>ρ</em><sub>kl</sub> is the correlation of the fraction of ancestry from ancestral population <em>k</em> and <em>l</em>. We derived the corrected test statistic by subtracting out 2*cov<sub>kl</sub>. Thus, the above test is applied between pairs (and triplets) of ancestral populations, each African/non-African ancestral groups conditional on non-African/African ancestral groups.</p><p id="__p59" class="p p-last">We additionally computed the correlation between TB-ancestry and ancestry-SES, and because we have socio-economic data only for TB cases, we regressed SES against genetic ancestry. Naturally this sample size may not provide sufficient power to identify correlations. Fortunately, because of the uniform ethnicity, SES status where the SAC's case/control sampling were conducted (Materials and Methods), we derived the relationship between TB status and SES based on the correlations (a 95% CI on the correlation of ancestry and SES) obtained from TB-ancestry and ancestry-SES models.</p></div><div id="s4g" class="sec sec-last"><h3 id="s4gtitle">Assessing locus-specific ancestry in multi-way admixed population</h3><p id="__p60" class="p p-first">GWAS of admixed populations, including admixture mapping was recently proposed to be informative for diseases for which the risk differs depending on the ancestry (<a href="#DDT462C1" rid="DDT462C1" class=" bibr popnode">1</a>,<a href="#DDT462C3" rid="DDT462C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140938">3</a>). These recent methods involve joint modeling of the admixture (accounting for locus-specific ancestry) and SNP association signals. From the simulation framework described in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods</a>, we first used the simulated five-way admixed population data to assess the accuracy of the local ancestry algorithms WinPOP and Lamp-LD. We computed the correlation between inferred local ancestry <em>Y</em> and the true ancestry <em>Z</em>. To estimate this correlation, we computed an estimate of the expected squared correlation between <em>Y</em> and <em>Z</em> as in (<a href="#DDT462C55" rid="DDT462C55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140937">55</a>). Given an ancestral population <em>k</em>, the expected squared correlation between <em>Y</em> and <em>Z</em> is a ratio of the expected covariance of <em>Y</em> and <em>Z</em> and the product of the expected variance of both <em>Y</em> and <em>Z</em> taken over loci and individuals:</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/instance/3888262/equ/M1" class="equation va_middle inline_block" alt="equation M1" title="equation M1"></div></div><p>
</p><p id="__p61">In addition, based on the true <em>Z</em> and inferred Y local ancestry, we compute the rate of calling true ancestry among different populations. Given a true ancestral segment of length <img src="/pmc/articles/instance/3888262/equ/M2" class="equation va_middle inline_block" alt="equation M2" title="equation M2"> (<img src="/pmc/articles/instance/3888262/equ/M3" class="equation va_middle inline_block" alt="equation M3" title="equation M3"> is the total number of true ancestral alleles) derived from population <img src="/pmc/articles/instance/3888262/equ/M4" class="equation va_middle inline_block" alt="equation M4" title="equation M4"> along the simulated genome of the admixed population, we computed the distribution of the rate of calling true ancestry <img src="/pmc/articles/instance/3888262/equ/M5" class="equation va_middle inline_block" alt="equation M5" title="equation M5"> and the error rate of calling <img src="/pmc/articles/instance/3888262/equ/M6" class="equation va_middle inline_block" alt="equation M6" title="equation M6">ancestral population instead of <em>k</em> as</p><div class="disp-formula"><div class="f"><img src="/pmc/articles/instance/3888262/equ/M7" class="equation va_middle inline_block" alt="equation M7" title="equation M7"></div></div><p>
where <img src="/pmc/articles/instance/3888262/equ/M8" class="equation va_middle inline_block" alt="equation M8" title="equation M8"> is the number of inferred ancestral alleles from ancestral population<img src="/pmc/articles/instance/3888262/equ/M9" class="equation va_middle inline_block" alt="equation M9" title="equation M9">. We obtain the rate of calling true ancestry by summing over all loci and averaging over all individuals.</p><p id="__p62" class="p p-last">Mendelian inconsistencies in local ancestry vary substantially across the genome and are mostly consistent with genomic regions with large deviations in local ancestry (<a href="#DDT462C45" rid="DDT462C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140957">45</a>,<a href="#DDT462C46" rid="DDT462C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522848">46</a>). Such genomic regions may lead to false-positive associations in scans of local ancestry across the genome (e.g. case-only admixture mapping). Assessing local ancestry using simulation may not measure systematic inaccuracies arising from the violations of model assumptions such as modelling of random mating, number of generations since admixed occurred and recombination rate (<a href="#DDT462C45" rid="DDT462C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_349140948">45</a>,<a href="#DDT462C46" rid="DDT462C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_399522862">46</a>). We conducted a real data-based evaluation of local ancestry using the SAC population. We evaluated the extent of spurious deviations in average local ancestry from case and control samples of the SAC. To address this, we subtracted the inferred average locus-specific ancestry by the corresponding genome-wide average ancestry; this later is approximated as a normal distribution under neutral drift with mean 0 and empirical variance. We fitted a chi-square on the obtained normal distribution (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Methods Text 2</a>) to test for spurious deviations in average local ancestry. We compare these deviations in case and control samples of the SAC.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p63" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddt462/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p64" class="p p-first-last">This project was supported by a Carnegie Corporation Grant, by the Department of Clinical Laboratory Sciences, University of Cape Town and by NIH grant <span>R01 HG006399</span> (N.Z. and A.L.P.).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140013714464064"><a href="/pmc/articles/PMC3888262/bin/supp_23_3_796__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>







</div></div><div id="idm140013673980928" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140013673980928title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p65">We are grateful to all the participants in the study. We thank C. Seioghe, N. Patterson and B. Pasaniuc for helpful discussions.</p><p id="__p66"><em>Conflict of Interest statement</em>. The authors declare no conflict of interest.</p></div></div><div id="idm140013673978752" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140013673978752title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDT462C1">1. <span class="element-citation">World Health Organization.  <a href="http://www.who.int/gtb/publications/globrep00/PDF/GTBR2000annex4.pdf">http://www.who.int/gtb/publications/globrep00/PDF/GTBR2000annex4.pdf</a> .</span></div><div class="ref-cit-blk half_rhythm" id="DDT462C2">2. <span class="element-citation">Kaufmann S., McMichael A. Annulling a dangerous liaison: vaccination strategies against AIDS and tuberculosis. <span><span class="ref-journal">Nat. Med. </span>2005;<span class="ref-vol">11</span>:33–44.</span> [<a href="/pubmed/15812488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Med.&amp;title=Annulling+a+dangerous+liaison:+vaccination+strategies+against+AIDS+and+tuberculosis&amp;author=S.+Kaufmann&amp;author=A.+McMichael&amp;volume=11&amp;publication_year=2005&amp;pages=33-44&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C3">3. <span class="element-citation">Dye C., Garnett G.P., Sleeman K., Williams B.G. Prospects for worldwide tuberculosis control under the WHO DOTS strategy. Directly observed short-course therapy. <span><span class="ref-journal">Lancet. </span>1998;<span class="ref-vol">352</span>:1886–1891.</span> [<a href="/pubmed/9863786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Prospects+for+worldwide+tuberculosis+control+under+the+WHO+DOTS+strategy.+Directly+observed+short-course+therapy&amp;author=C.+Dye&amp;author=G.P.+Garnett&amp;author=K.+Sleeman&amp;author=B.G.+Williams&amp;volume=352&amp;publication_year=1998&amp;pages=1886-1891&amp;pmid=9863786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C4">4. <span class="element-citation">Dye C., Scheele S., Dolin P., Pathania V., Raviglione M.C. Global burden of tuberculosis. Estimated incidence, prevalence, and mortality by country. <span><span class="ref-journal">JAMA. </span>1999;<span class="ref-vol">282</span>:677–686.</span> [<a href="/pubmed/10517722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Global+burden+of+tuberculosis.+Estimated+incidence,+prevalence,+and+mortality+by+country&amp;author=C.+Dye&amp;author=S.+Scheele&amp;author=P.+Dolin&amp;author=V.+Pathania&amp;author=M.C.+Raviglione&amp;volume=282&amp;publication_year=1999&amp;pages=677-686&amp;pmid=10517722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C5">5. <span class="element-citation">Comstock G.W. Tuberculosis in twins, a re-analysis of the Prophit survey. <span><span class="ref-journal">Am. Rev. Respir. Dis. </span>1978;<span class="ref-vol">117</span>:621–624.</span> [<a href="/pubmed/565607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Rev.+Respir.+Dis.&amp;title=Tuberculosis+in+twins,+a+re-analysis+of+the+Prophit+survey&amp;author=G.W.+Comstock&amp;volume=117&amp;publication_year=1978&amp;pages=621-624&amp;pmid=565607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C6">6. <span class="element-citation">Ducati R.G., Ruffino-Netto A., Basso L.A., Santos D.S. The resumption of consumption a review on tuberculosis. <span><span class="ref-journal">Mem. Inst. </span>2006;<span class="ref-vol">101</span>:697–714.</span> [<a href="/pubmed/17160276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mem.+Inst&amp;title=The+resumption+of+consumption+a+review+on+tuberculosis&amp;author=R.G.+Ducati&amp;author=A.+Ruffino-Netto&amp;author=L.A.+Basso&amp;author=D.S.+Santos&amp;volume=101&amp;publication_year=2006&amp;pages=697-714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C7">7. <span class="element-citation">Daniel T.M.  <span class="ref-journal">Captain of death, the story of tuberculosis.</span> Rochester, NY: University of Rochester Press; 1997.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Captain+of+death,+the+story+of+tuberculosis&amp;author=T.M.+Daniel&amp;publication_year=1997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C8">8. <span class="element-citation">Sorensen T.I., Nielsen G.G., Andersen P.K., Teasdale T.W. Genetic and environmental influences on premature death in adult adoptees. <span><span class="ref-journal">New Engl. J. Med. </span>1988;<span class="ref-vol">318</span>:727–732.</span> [<a href="/pubmed/3347221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+Engl.+J.+Med.&amp;title=Genetic+and+environmental+influences+on+premature+death+in+adult+adoptees&amp;author=T.I.+Sorensen&amp;author=G.G.+Nielsen&amp;author=P.K.+Andersen&amp;author=T.W.+Teasdale&amp;volume=318&amp;publication_year=1988&amp;pages=727-732&amp;pmid=3347221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C9">9. <span class="element-citation">Flynn J. Lessons from experimental Mycobacterium tuberculosis infections. <span><span class="ref-journal">Microbes Infect. </span>2006;<span class="ref-vol">8</span>:1179–1188.</span> [<a href="/pubmed/16513383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbes+Infect.&amp;title=Lessons+from+experimental+Mycobacterium+tuberculosis+infections&amp;author=J.+Flynn&amp;volume=8&amp;publication_year=2006&amp;pages=1179-1188&amp;pmid=16513383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C10">10. <span class="element-citation">Moller M., Nebel A., Valentonyte R., van Helden P.D., Schreiber S., Hoal E.G. Investigation of chromosome 17 candidate genes in susceptibility to TB in a South African population. <span><span class="ref-journal">Tuberculosis. </span>2009;<span class="ref-vol">89</span>:189–194.</span> [<a href="/pubmed/19147409" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis&amp;title=Investigation+of+chromosome+17+candidate+genes+in+susceptibility+to+TB+in+a+South+African+population&amp;author=M.+Moller&amp;author=A.+Nebel&amp;author=R.+Valentonyte&amp;author=P.D.+van+Helden&amp;author=S.+Schreiber&amp;volume=89&amp;publication_year=2009&amp;pages=189-194&amp;pmid=19147409&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C11">11. <span class="element-citation">Moller M., Hoal E.G. Current findings, challenges and novel approaches in human genetic susceptibility to tuberculosis. <span><span class="ref-journal">Tuberculosis. </span>2010a;<span class="ref-vol">90</span>:71–83.</span> [<a href="/pubmed/20206579" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis&amp;title=Current+findings,+challenges+and+novel+approaches+in+human+genetic+susceptibility+to+tuberculosis&amp;author=M.+Moller&amp;author=E.G.+Hoal&amp;volume=90&amp;publication_year=2010a&amp;pages=71-83&amp;pmid=20206579&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C12">12. <span class="element-citation">Davila S., Hibberd M.L., Hari Dass R., Wong H.E.E., Sahiratmadja E., Bonnard C., Alisjahbana B., Szeszko J.S., Balabanova Y., Drobniewski F., van Crevel R., van de Vosse E., Nejentsev S., Ottenhoff T.H.M., Seielstad M. Genetic association and expression studies indicate a role of Toll-like receptor 8 in pulmonary tuberculosis. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2568981/">PMC free article</a>]</span> [<a href="/pubmed/18927625" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+association+and+expression+studies+indicate+a+role+of+Toll-like+receptor+8+in+pulmonary+tuberculosis&amp;author=S.+Davila&amp;author=M.L.+Hibberd&amp;author=R.+Hari+Dass&amp;author=H.E.E.+Wong&amp;author=E.+Sahiratmadja&amp;volume=4&amp;publication_year=2008&amp;pages=e1000218&amp;pmid=18927625&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C13">13. <span class="element-citation">Thye T., Vannberg F.O., Wong S.H., Owusu-Dabo E., Osei I., Gyapong J., Sirugo G., SisayJoof F., Enimil A., Chinbuah M.A., Floyd S., Warndorff D.K., Sichali L., Malema S., Crampin A.C., Ngwira B., Teo Y.Y., Small K., Rockett K., Kwiatkowski D., Fine P.E., Hill P.C., Newport M., Lienhardt C., Adegbola R.A., Corrah T., Ziegler A., Morris A.P., Meyer C.G., Horstmann R.D., Hill A.V.S. African TB Genetics Consortium, The Wellcome Trust Case Control Consortium. Genome-wide association analyses identifies a susceptibility locus for tuberculosis on chromosome 18q11.2. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4975513/">PMC free article</a>]</span> [<a href="/pubmed/20694014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analyses+identifies+a+susceptibility+locus+for+tuberculosis+on+chromosome+18q11.2&amp;author=T.+Thye&amp;author=F.O.+Vannberg&amp;author=S.H.+Wong&amp;author=E.+Owusu-Dabo&amp;author=I.+Osei&amp;volume=42&amp;publication_year=2010&amp;pages=9&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C14">14. <span class="element-citation">Thye T., Owusu-Dabo E., Vannberg F.O., Crevel R.V., Curtis J., Sahiratmadja E., Balabanova Y., Ehmen C., Muntau B., Ruge G., Sievertsen J., Gyapong J., Nikolayevskyy V., Hill P.C., Sirugo G., Drobniewsk F., de Vosse E.V., Newport M., Alisjahbana B., Nejentsev S., Ottenhoff T.H.M., Hill A.V.S., Horstmann R.D., Meyer C.G. Common variants at 11p13 are associated with susceptibility to tuberculosis. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3427019/">PMC free article</a>]</span> [<a href="/pubmed/22306650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+at+11p13+are+associated+with+susceptibility+to+tuberculosis&amp;author=T.+Thye&amp;author=E.+Owusu-Dabo&amp;author=F.O.+Vannberg&amp;author=R.V.+Crevel&amp;author=J.+Curtis&amp;volume=44&amp;publication_year=2012&amp;pages=3&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C15">15. <span class="element-citation">Dai Y., Zhang X., Pan H., Tang S., Shen H., Wang J. Fine mapping of genetic polymorphisms of pulmonary tuberculosis within chromosome 18q11.2 in the Chinese population: a case–control study. <span><span class="ref-journal">BMC Infect. Dis. </span>2011;<span class="ref-vol">11</span>:282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3248069/">PMC free article</a>]</span> [<a href="/pubmed/22018224" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Infect.+Dis.&amp;title=Fine+mapping+of+genetic+polymorphisms+of+pulmonary+tuberculosis+within+chromosome+18q11.2+in+the+Chinese+population:+a+case–control+study&amp;author=Y.+Dai&amp;author=X.+Zhang&amp;author=H.+Pan&amp;author=S.+Tang&amp;author=H.+Shen&amp;volume=11&amp;publication_year=2011&amp;pages=282&amp;pmid=22018224&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C16">16. <span class="element-citation">Wang X., Leung-Sang N.T., Chiu C.L., Kam K.M., Yew W.W., Ming C.T., Chan C.Y. Association of polymorphisms in the Chr18q11.2 locus with tuberculosis in Chinese population. <span><span class="ref-journal">Hum. Genet. </span>2013;<span class="ref-vol">132</span>:691–695.</span> [<a href="/pubmed/23456169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Association+of+polymorphisms+in+the+Chr18q11.2+locus+with+tuberculosis+in+Chinese+population&amp;author=X.+Wang&amp;author=N.T.+Leung-Sang&amp;author=C.L.+Chiu&amp;author=K.M.+Kam&amp;author=W.W.+Yew&amp;volume=132&amp;publication_year=2013&amp;pages=691-695&amp;pmid=23456169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C17">17. <span class="element-citation">Small P.M. Tuberculosis research. Balancing the portfolio. <span><span class="ref-journal">JAMA. </span>1996;<span class="ref-vol">276</span>:1512–1513.</span> [<a href="/pubmed/8903265" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Tuberculosis+research.+Balancing+the+portfolio&amp;author=P.M.+Small&amp;volume=276&amp;publication_year=1996&amp;pages=1512-1513&amp;pmid=8903265&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C18">18. <span class="element-citation">Kumar R., Seibold M.A., Aldrich M.C., Keoki Williams L., Reiner A.P., Colangelo L., Galanter J., Gignoux C., Hu D., Sen S., Choudhry S., Peterson E.L., Rodriguez-Santana J., Rodriguez-Cintron W., Nalls M.A., Leak T.S., O'Meara E., Meibohm B., Kritchevsky S.B., Li R., Harris T.B., Nickerson D.A., Fornage M., Enright P., Ziv E., Smith L.J., Liu K., Burchard E.G. Genetic ancestry in lung-function predictions. <span><span class="ref-journal">N Engl. J. Med. </span>2010;<span class="ref-vol">363</span>:321–330.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922981/">PMC free article</a>]</span> [<a href="/pubmed/20647190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl.+J.+Med.&amp;title=Genetic+ancestry+in+lung-function+predictions&amp;author=R.+Kumar&amp;author=M.A.+Seibold&amp;author=M.C.+Aldrich&amp;author=L.+Keoki+Williams&amp;author=A.P.+Reiner&amp;volume=363&amp;publication_year=2010&amp;pages=321-330&amp;pmid=20647190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C19">19. <span class="element-citation">Hoal E.G., Lewis L.A., Jamieson S.E., Tanzer F., Rossouw M., et al.  SLC11A1 (NRAMP1) but not SLC11A2 (NRAMP2) polymorphisms are associated with susceptibility to tuberculosis in a high-incidence community in South Africa. <span><span class="ref-journal">Int. J. Tuberc. Lung Dis. </span>2004;<span class="ref-vol">8</span>:1464–1471.</span> [<a href="/pubmed/15636493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Tuberc.+Lung+Dis.&amp;title=SLC11A1+(NRAMP1)+but+not+SLC11A2+(NRAMP2)+polymorphisms+are+associated+with+susceptibility+to+tuberculosis+in+a+high-incidence+community+in+South+Africa&amp;author=E.G.+Hoal&amp;author=L.A.+Lewis&amp;author=S.E.+Jamieson&amp;author=F.+Tanzer&amp;author=M.+Rossouw&amp;volume=8&amp;publication_year=2004&amp;pages=1464-1471&amp;pmid=15636493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C20">20. <span class="element-citation">Moller M., de Wit E., Hoal E.G. Past, present and future directions in human genetic susceptibility to tuberculosis. <span><span class="ref-journal">FEMS Immunol. Med. Microbiol. </span>2010b;<span class="ref-vol">58</span>:3–26.</span> [<a href="/pubmed/19780822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEMS+Immunol.+Med.+Microbiol.&amp;title=Past,+present+and+future+directions+in+human+genetic+susceptibility+to+tuberculosis&amp;author=M.+Moller&amp;author=E.+de+Wit&amp;author=E.G.+Hoal&amp;volume=58&amp;publication_year=2010b&amp;pages=3-26&amp;pmid=19780822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C21">21. <span class="element-citation">de Wit E., Delport W., Chimusa R.E., Meintjes A., Moller M., et al.  Genome-wide analysis of the structure of the South African Coloured population in the Western Cape. <span><span class="ref-journal">Hum. Genet. </span>2010a;<span class="ref-vol">128</span>:145–153.</span> [<a href="/pubmed/20490549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genome-wide+analysis+of+the+structure+of+the+South+African+Coloured+population+in+the+Western+Cape&amp;author=E.+de+Wit&amp;author=W.+Delport&amp;author=R.E.+Chimusa&amp;author=A.+Meintjes&amp;author=M.+Moller&amp;volume=128&amp;publication_year=2010a&amp;pages=145-153&amp;pmid=20490549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C22">22. <span class="element-citation">Quintana-Murci L., Harmant C., Quach H., Balanovsky O., Zaporozhchenko V., et al.  Strong maternal Khoisan contribution to the South African Coloured population: a case of gender-biased admixture. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:611–620.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850426/">PMC free article</a>]</span> [<a href="/pubmed/20346436" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Strong+maternal+Khoisan+contribution+to+the+South+African+Coloured+population:+a+case+of+gender-biased+admixture&amp;author=L.+Quintana-Murci&amp;author=C.+Harmant&amp;author=H.+Quach&amp;author=O.+Balanovsky&amp;author=V.+Zaporozhchenko&amp;volume=86&amp;publication_year=2010&amp;pages=611-620&amp;pmid=20346436&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C23">23. <span class="element-citation">Patterson N., Desiree Petersen C., Richard van der Ross E., Sudoyo H., Glashoff R.H., Marzuki S., Reich D., Hayes V.M.  <span class="ref-journal">Genetic structure of a unique admixed population: implications for medical research. Hum. Mol. Genet.</span> 2009.   <a href="http://dx.doi.org/10.1093/hmg/ddp505" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">doi:10.1093/hmg/ddp505</a>. [<a href="/pubmed/19892779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Genetic+structure+of+a+unique+admixed+population:+implications+for+medical+research.+Hum.+Mol.+Genet&amp;author=N.+Patterson&amp;author=C.+Desiree+Petersen&amp;author=E.+Richard+van+der+Ross&amp;author=H.+Sudoyo&amp;author=R.H.+Glashoff&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C24">24. <span class="element-citation">Tishkoff S.A., Reed F.A., Friedlaender F.R., Ehret C., Ranciaro A., et al.  The Genetic Structure and History of Africans and African Americans. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">324</span>:1035–1044.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947357/">PMC free article</a>]</span> [<a href="/pubmed/19407144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+Genetic+Structure+and+History+of+Africans+and+African+Americans&amp;author=S.A.+Tishkoff&amp;author=F.A.+Reed&amp;author=F.R.+Friedlaender&amp;author=C.+Ehret&amp;author=A.+Ranciaro&amp;volume=324&amp;publication_year=2009&amp;pages=1035-1044&amp;pmid=19407144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C25">25. <span class="element-citation">Florez J.C., Price A., Campbell D., Riba L., Parra M.V., Yu F., Duque C., Saxena R., Gallego N., Tello-Ruiz M., Franco L., Rodríguez-Torres M., illegas V.A., Bedoya G., Aguilar-Salinas C.A., Tusié-Luna M.T., Ruiz-Linares A., Reich D. Strong association of socioeconomic status with genetic ancestry in Latinos: implications for admixture studies of type 2 diabetes. <span><span class="ref-journal">Diabetologia. </span>2009;<span class="ref-vol">52</span>:1528–1536.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3113605/">PMC free article</a>]</span> [<a href="/pubmed/19526211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Strong+association+of+socioeconomic+status+with+genetic+ancestry+in+Latinos:+implications+for+admixture+studies+of+type+2+diabetes&amp;author=J.C.+Florez&amp;author=A.+Price&amp;author=D.+Campbell&amp;author=L.+Riba&amp;author=M.V.+Parra&amp;volume=52&amp;publication_year=2009&amp;pages=1528-1536&amp;pmid=19526211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C26">26. <span class="element-citation">Marchini J., Howie B. Comparing algorithms for genotype imputation. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">83</span>:535–540.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2561929/">PMC free article</a>]</span> [<a href="/pubmed/18940314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Comparing+algorithms+for+genotype+imputation&amp;author=J.+Marchini&amp;author=B.+Howie&amp;volume=83&amp;publication_year=2008&amp;pages=535-540&amp;pmid=18940314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C27">27. <span class="element-citation">The International HapMap 3 Consortium. Integrating common and rare genetic variation in diverse human populations. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:52–58.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3173859/">PMC free article</a>]</span> [<a href="/pubmed/20811451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrating+common+and+rare+genetic+variation+in+diverse+human+populations&amp;volume=467&amp;publication_year=2010&amp;pages=52-58&amp;pmid=20811451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C28">28. <span class="element-citation">McVean G.A., Altshuler D.M., Durbin R.M., Abecasis G.A., Bentley D.R., Chakravarti A., Clark A.G., Donnelly P., et al.  An integrated map of genetic variation from 1092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1092+human+genomes&amp;author=G.A.+McVean&amp;author=D.M.+Altshuler&amp;author=R.M.+Durbin&amp;author=G.A.+Abecasis&amp;author=D.R.+Bentley&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C29">29. <span class="element-citation">Kang H.M., Sul J.H., Service S.K., Zaitlen N.A., Kong S.-y., Freimer N.B., Sabatti C., Eskin E. Variance component model to account for sample structure in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:348–354.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3092069/">PMC free article</a>]</span> [<a href="/pubmed/20208533" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Variance+component+model+to+account+for+sample+structure+in+genome-wide+association+studies&amp;author=H.M.+Kang&amp;author=J.H.+Sul&amp;author=S.K.+Service&amp;author=N.A.+Zaitlen&amp;author=S.-y.+Kong&amp;volume=42&amp;publication_year=2010&amp;pages=348-354&amp;pmid=20208533&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C30">30. <span class="element-citation">Devlin B., Roeder K. Genomic Control for Association Studies. <span><span class="ref-journal">Biometrics. </span>1999;<span class="ref-vol">55</span>:997–1004.</span> [<a href="/pubmed/11315092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Genomic+Control+for+Association+Studies&amp;author=B.+Devlin&amp;author=K.+Roeder&amp;volume=55&amp;publication_year=1999&amp;pages=997-1004&amp;pmid=11315092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C31">31. <span class="element-citation">Hokayem J.E., Huber C., Couvé A., Aziza J., Baujat G., Bouvier R., Cavalcanti D.P., Collins F.A., Cordier M.P., Delezoide A.L., Gonzales M., Johnson D., Le Merrer M., Levy-Mozziconacci A., Loget P., Martin-Coignard D., Martinovic J., Mortier G.R., Marie-José P., Roume J., Scarano G., Munnich A., Cormier-Daire V. NEK1 and DYNC2H1 are both involved in short rib polydactyly Majewski type but not in Beemer Langer cases. <span><span class="ref-journal">J. Med. Genet. </span>2012;<span class="ref-vol">49</span>:227–233.</span> [<a href="/pubmed/22499340" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=NEK1+and+DYNC2H1+are+both+involved+in+short+rib+polydactyly+Majewski+type+but+not+in+Beemer+Langer+cases&amp;author=J.E.+Hokayem&amp;author=C.+Huber&amp;author=A.+Couvé&amp;author=J.+Aziza&amp;author=G.+Baujat&amp;volume=49&amp;publication_year=2012&amp;pages=227-233&amp;pmid=22499340&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C32">32. <span class="element-citation">Rauscher F.J. The WT1 Wilms tumor gene product: a developmentally regulated transcription factor in the kidney that functions as a tumor suppressor. <span><span class="ref-journal">FASEB J. </span>1993;<span class="ref-vol">7</span>:896–903.</span> [<a href="/pubmed/8393820" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J.&amp;title=The+WT1+Wilms+tumor+gene+product:+a+developmentally+regulated+transcription+factor+in+the+kidney+that+functions+as+a+tumor+suppressor&amp;author=F.J.+Rauscher&amp;volume=7&amp;publication_year=1993&amp;pages=896-903&amp;pmid=8393820&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C33">33. <span class="element-citation">Buhm H., Eskin E. Random-effects model aimed at discovering associations in meta-analysis of genome-wide Association Studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:586–598.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146723/">PMC free article</a>]</span> [<a href="/pubmed/21565292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Random-effects+model+aimed+at+discovering+associations+in+meta-analysis+of+genome-wide+Association+Studies&amp;author=H.+Buhm&amp;author=E.+Eskin&amp;volume=88&amp;publication_year=2011&amp;pages=586-598&amp;pmid=21565292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C34">34. <span class="element-citation">Wu Y., Waite L.L., Jackson A.U., Sheu W.H.-H., Buyske S., et al.  Trans-ethnic fine-mapping of lipid loci identifies population-specific signals and allelic heterogeneity that increases the trait variance explained. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003379.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605054/">PMC free article</a>]</span> [<a href="/pubmed/23555291" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Trans-ethnic+fine-mapping+of+lipid+loci+identifies+population-specific+signals+and+allelic+heterogeneity+that+increases+the+trait+variance+explained&amp;author=Y.+Wu&amp;author=L.L.+Waite&amp;author=A.U.+Jackson&amp;author=W.H.-H.+Sheu&amp;author=S.+Buyske&amp;volume=9&amp;publication_year=2013&amp;pages=e1003379&amp;pmid=23555291&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C35">35. <span class="element-citation">Jimenez-Corona ME., Garcia-Garcia L., Deriemer K., Ferreyra-Reyes L., Bobadilla-del-Valle M., Cano-Arellano B., Canizales-Quintero S., Martinez-Gamboa A., Small PM., Sifuentes-Osornio J., Ponce-de-Leon A. Gender differentials of pulmonary tuberculosis transmission and reactivation in an endemic area. <span><span class="ref-journal">Thorax. </span>2006;<span class="ref-vol">61</span>:348–353.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2104608/">PMC free article</a>]</span> [<a href="/pubmed/16449260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Thorax&amp;title=Gender+differentials+of+pulmonary+tuberculosis+transmission+and+reactivation+in+an+endemic+area&amp;author=ME.+Jimenez-Corona&amp;author=L.+Garcia-Garcia&amp;author=K.+Deriemer&amp;author=L.+Ferreyra-Reyes&amp;author=M.+Bobadilla-del-Valle&amp;volume=61&amp;publication_year=2006&amp;pages=348-353&amp;pmid=16449260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C36">36. <span class="element-citation"><span class="ref-journal">Reaching the poor: challenges for tuberculosis programmes in the Western Pacific Region.</span> WHO Regional Office for the Western Pacific; 2004.  2004. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Reaching+the+poor:+challenges+for+tuberculosis+programmes+in+the+Western+Pacific+Region&amp;publication_year=2004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C37">37. <span class="element-citation">2005.  30 Gender-based analysis of tuberculosis-related data and other information <em>Office of the WHO Representative in Vietnam</em>.</span></div><div class="ref-cit-blk half_rhythm" id="DDT462C38">38. <span class="element-citation">de Wit E., van der Merwe L., van Helden P.D., Hoal E.G.  <span class="ref-journal">Gene-gene interaction between tuberculosis candidate genes in a South African population.</span> Springer Science, Business Media; 2010b.  LLC 2010, DOI 10.1007/s00335-010-9280-8. [<a href="/pubmed/20799037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Gene-gene+interaction+between+tuberculosis+candidate+genes+in+a+South+African+population&amp;author=E.+de+Wit&amp;author=L.+van+der+Merwe&amp;author=P.D.+van+Helden&amp;author=E.G.+Hoal&amp;publication_year=2010b&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C39">39. <span class="element-citation">Rodriguez J., Bercovici S., Elmore M., Batzoglou S. Ancestry inference in complex admixtures via variable-length Markov chain linkage models. <span><span class="ref-journal">J. Comput. Biol. </span>2012;<span class="ref-vol">20</span>:199–211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3590892/">PMC free article</a>]</span> [<a href="/pubmed/23421795" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Biol.&amp;title=Ancestry+inference+in+complex+admixtures+via+variable-length+Markov+chain+linkage+models&amp;author=J.+Rodriguez&amp;author=S.+Bercovici&amp;author=M.+Elmore&amp;author=S.+Batzoglou&amp;volume=20&amp;publication_year=2012&amp;pages=199-211&amp;pmid=23421795&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C40">40. <span class="element-citation">Brisbin A.  <span class="ref-journal">Linkage Analysis for Categorical Traits and Ancestry Assignment in Admixed Individuals.</span> Ithaca, New York: Doctoral Dissertation, Cornell University; 2010.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Linkage+Analysis+for+Categorical+Traits+and+Ancestry+Assignment+in+Admixed+Individuals&amp;author=A.+Brisbin&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C41">41. <span class="element-citation">Churchhouse C., Marchini J. Multiway admixture deconvolution using phased or unphased ancestral panels. <span><span class="ref-journal">Genet. Epidemiol. </span>2012;<span class="ref-vol">37</span>:1–12.</span> [<a href="/pubmed/23136122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Multiway+admixture+deconvolution+using+phased+or+unphased+ancestral+panels&amp;author=C.+Churchhouse&amp;author=J.+Marchini&amp;volume=37&amp;publication_year=2012&amp;pages=1-12&amp;pmid=23136122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C42">42. <span class="element-citation">Lawson D., Hellenthal G., Myers S., Falush D. Inference of population structure using dense haplotype data. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">8</span>:e1002453.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3266881/">PMC free article</a>]</span> [<a href="/pubmed/22291602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Inference+of+population+structure+using+dense+haplotype+data&amp;author=D.+Lawson&amp;author=G.+Hellenthal&amp;author=S.+Myers&amp;author=D.+Falush&amp;volume=8&amp;publication_year=2010&amp;pages=e1002453&amp;pmid=22291602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C43">43. <span class="element-citation">Pasaniuc B., Sankararaman S., Kimmel G., Halperin E. Inference of locus-specific ancestry in closely related population. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:213–221.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687951/">PMC free article</a>]</span> [<a href="/pubmed/19477991" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Inference+of+locus-specific+ancestry+in+closely+related+population&amp;author=B.+Pasaniuc&amp;author=S.+Sankararaman&amp;author=G.+Kimmel&amp;author=E.+Halperin&amp;volume=25&amp;publication_year=2009&amp;pages=213-221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C44">44. <span class="element-citation">Baran Y., Pasaniuc B., Sankararaman S., Dara G., Gignoux C., Celeste C., Torgerson W., Chapela R., JeanFord G., Pedro Avila C., Rodriguez-Santana J., Gonzalez Burchard E., Eran E. Fast and accurate inference of local ancestry in Latino populations. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:1359–1367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3348558/">PMC free article</a>]</span> [<a href="/pubmed/22495753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast+and+accurate+inference+of+local+ancestry+in+Latino+populations&amp;author=Y.+Baran&amp;author=B.+Pasaniuc&amp;author=S.+Sankararaman&amp;author=G.+Dara&amp;author=C.+Gignoux&amp;volume=28&amp;publication_year=2012&amp;pages=1359-1367&amp;pmid=22495753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C45">45. <span class="element-citation">Pasaniuc B., Sankararaman S., Torgerson D.G., Gignoux C., Zaitlen N., Eng C., Rodriguez-Cintron W., Chapela R., Ford J.G., Avila P.C., Rodriguez-Santana J., Chen G.K., Le Larchand L., Henderson B., Reich D., Haiman C.A., Burchard E.G., Halperin E. Analysis of Latino populations from GALA and MEC studies reveals genomic loci with biased local ancestry estimation. <span><span class="ref-journal">Bioinformatics. </span>2013;<span class="ref-vol">29</span>:1407–1415.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3661056/">PMC free article</a>]</span> [<a href="/pubmed/23572411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Analysis+of+Latino+populations+from+GALA+and+MEC+studies+reveals+genomic+loci+with+biased+local+ancestry+estimation&amp;author=B.+Pasaniuc&amp;author=S.+Sankararaman&amp;author=D.G.+Torgerson&amp;author=C.+Gignoux&amp;author=N.+Zaitlen&amp;volume=29&amp;publication_year=2013&amp;pages=1407-1415&amp;pmid=23572411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C46">46. <span class="element-citation">Seldin M.F., Pasaniuc B., Price A. New approaches to disease mapping in admixed populations. <span><span class="ref-journal">Nat. Rev. Genet. </span>2011;<span class="ref-vol">36</span>:21–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142784/">PMC free article</a>]</span> [<a href="/pubmed/21709689" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=New+approaches+to+disease+mapping+in+admixed+populations&amp;author=M.F.+Seldin&amp;author=B.+Pasaniuc&amp;author=A.+Price&amp;volume=36&amp;publication_year=2011&amp;pages=21-27&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C47">47. <span class="element-citation">Band G., Le Q.S., Jostins L., Pirinen M., Kivinen K., Jallow M., Sisay-Joof F., Bojang K., Pinder M., Sirugo G., Conway D.J., Nyirongo V., Kachala D., Molyneux M., Taylor T., Ndila C., Peshu N., Marsh K., Williams T.N., Alcock D., Andrews R., Edkins S., Gray E., Hubbart C., Jeffreys A., Rowlands K., Schuldt K., Clark T.G., Small K.S., Teo Y.Y., Kwiatkowski D.P., Rockett K.A., Barrett J.C., Spencer C.C. Imputation-based meta-analysis of severe malaria in three African populations. <span><span class="ref-journal">Plos Genet. </span>2013;<span class="ref-vol">9</span>:e1003509.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3662650/">PMC free article</a>]</span> [<a href="/pubmed/23717212" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Plos+Genet.&amp;title=Imputation-based+meta-analysis+of+severe+malaria+in+three+African+populations&amp;author=G.+Band&amp;author=Q.S.+Le&amp;author=L.+Jostins&amp;author=M.+Pirinen&amp;author=K.+Kivinen&amp;volume=9&amp;publication_year=2013&amp;pages=e1003509&amp;pmid=23717212&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C48">48. <span class="element-citation">Stein C.M. Genetic epidemiology of tuberculosis susceptibility: impact of study design. <span><span class="ref-journal">PLoS Pathog. </span>2011;<span class="ref-vol">7</span>:e1001189.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3024264/">PMC free article</a>]</span> [<a href="/pubmed/21283783" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog.&amp;title=Genetic+epidemiology+of+tuberculosis+susceptibility:+impact+of+study+design&amp;author=C.M.+Stein&amp;volume=7&amp;publication_year=2011&amp;pages=e1001189&amp;pmid=21283783&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C49">49. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C50">50. <span class="element-citation">Schlebusch C., Skoglund P., Sjodin P., Gattepaille L., Hernandez D., et al.  Genomic variation in seven Khoe-San groups reveals adaptation and complex African history. <span><span class="ref-journal">Science. </span>2012;<span class="ref-vol">338</span>:374–379.</span> [<a href="/pubmed/22997136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genomic+variation+in+seven+Khoe-San+groups+reveals+adaptation+and+complex+African+history&amp;author=C.+Schlebusch&amp;author=P.+Skoglund&amp;author=P.+Sjodin&amp;author=L.+Gattepaille&amp;author=D.+Hernandez&amp;volume=338&amp;publication_year=2012&amp;pages=374-379&amp;pmid=22997136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C51">51. <span class="element-citation">Alexander D.H., Novembre J., Lange K. Fast model-based estimation of ancestry in unrelated individuals. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:1655–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;author=D.H.+Alexander&amp;author=J.+Novembre&amp;author=K.+Lange&amp;volume=19&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C52">52. <span class="element-citation">Price A., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C53">53. <span class="element-citation">Patterson N., Price A.L., Reich D. Population structure and eigen analysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Population+structure+and+eigen+analysis&amp;author=N.+Patterson&amp;author=A.L.+Price&amp;author=D.+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C54">54. <span class="element-citation">Purcell S., Cherny S.S., Sham P.C. Genetic power calculator: design of linkage and association genetic mapping studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>:149–150.</span> [<a href="/pubmed/12499305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genetic+power+calculator:+design+of+linkage+and+association+genetic+mapping+studies+of+complex+traits&amp;author=S.+Purcell&amp;author=S.S.+Cherny&amp;author=P.C.+Sham&amp;volume=19&amp;publication_year=2003&amp;pages=149-150&amp;pmid=12499305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDT462C55">55. <span class="element-citation">Price A., Tandon A., Patterson N., Barnes K., Rafaels N., et al.  Sensitive detection of chromosomal segments of distinct ancestry in admixed populations. <span><span class="ref-journal">PLos Genet. </span>2009:e1000519.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689842/">PMC free article</a>]</span> [<a href="/pubmed/19543370" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+Genet.&amp;title=Sensitive+detection+of+chromosomal+segments+of+distinct+ancestry+in+admixed+populations&amp;author=A.+Price&amp;author=A.+Tandon&amp;author=N.+Patterson&amp;author=K.+Barnes&amp;author=N.+Rafaels&amp;publication_year=2009&amp;pages=e1000519&amp;pmid=19543370&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3888262&amp;issue-id=232154&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140013683032176title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140013673980928title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140013673978752title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3888262/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3888262/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3888262/pdf/ddt462.pdf">PDF (560K)</a></li> | <li><a href="#" data-citationid="PMC3888262" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3888262%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3888262%2F&amp;text=Genome-wide%20association%20study%20of%20ancestry-specific%20TB%20risk%20in%20the%20South%20African%20Coloured%20population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3888262%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24057671" data-aiid="3888262" data-aid="3888262" data-iid="232154" data-domainid="740" data-domain="hmg" data-accid="PMC3888262" data-md5="a2f93bfa0b410698dcdb4fca2c2d22be"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T11:24:10-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DB949DD55EF010000000007740474&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3888262%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3888262/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>