<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3159549/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies two loci strongly affecting transferrin glycosylation</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3159549/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study identifies two loci strongly affecting transferrin glycosylation"><meta name="citation_authors" content="Zoltán Kutalik, Beben Benyamin, Sven Bergmann, Vincent Mooser, Gérard Waeber, Grant W. Montgomery, Nicholas G. Martin, Pamela A.F. Madden, Andrew C. Heath, Jacques S. Beckmann, Peter Vollenweider, Pedro Marques-Vidal, John B. Whitfield"><meta name="citation_date" content="15 September 2011"><meta name="citation_issue" content="18"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="3710"><meta name="citation_doi" content="10.1093/hmg/ddr272"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3159549/?report=abstract"><meta name="citation_pmid" content="21665994"><meta name="DC.Title" content="Genome-wide association study identifies two loci strongly affecting transferrin glycosylation"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Zoltán Kutalik"><meta name="DC.Contributor" content="Beben Benyamin"><meta name="DC.Contributor" content="Sven Bergmann"><meta name="DC.Contributor" content="Vincent Mooser"><meta name="DC.Contributor" content="Gérard Waeber"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="Pamela A.F. Madden"><meta name="DC.Contributor" content="Andrew C. Heath"><meta name="DC.Contributor" content="Jacques S. Beckmann"><meta name="DC.Contributor" content="Peter Vollenweider"><meta name="DC.Contributor" content="Pedro Marques-Vidal"><meta name="DC.Contributor" content="John B. Whitfield"><meta name="DC.Date" content="2011 Sep 15"><meta name="DC.Identifier" content="10.1093/hmg/ddr272"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies two loci strongly affecting transferrin glycosylation"><meta property="og:type" content="article"><meta property="og:description" content="Polysaccharide sidechains attached to proteins play important roles in cell–cell and receptor–ligand interactions. Variation in the carbohydrate component has been extensively studied for the iron transport protein transferrin, because ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3159549/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3159549/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3159549/pdf/ddr272.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B3DB0DD51ED710000000009A30625.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3159549%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3159549</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2011 Sep 15; 20(18): 3710–3717. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Jun 10. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddr272" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddr272</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3159549</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21665994">21665994</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies two loci strongly affecting transferrin glycosylation</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kutalik%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130781264720" co-class="co-affbox">Zoltán Kutalik</a>,<sup>1,</sup><sup>2,</sup><sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Benyamin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130781261760" co-class="co-affbox">Beben Benyamin</a>,<sup>3,</sup><sup>†</sup> <a href="/pubmed/?term=Bergmann%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130741641568" co-class="co-affbox">Sven Bergmann</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Mooser%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130741638608" co-class="co-affbox">Vincent Mooser</a>,<sup>4</sup> <a href="/pubmed/?term=Waeber%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130741935104" co-class="co-affbox">Gérard Waeber</a>,<sup>5</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130741932880" co-class="co-affbox">Grant W. Montgomery</a>,<sup>6</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130741930656" co-class="co-affbox">Nicholas G. Martin</a>,<sup>7</sup> <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130774771792" co-class="co-affbox">Pamela A.F. Madden</a>,<sup>8</sup> <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130774769568" co-class="co-affbox">Andrew C. Heath</a>,<sup>8</sup> <a href="/pubmed/?term=Beckmann%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130774767344" co-class="co-affbox">Jacques S. Beckmann</a>,<sup>1,</sup><sup>9</sup> <a href="/pubmed/?term=Vollenweider%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130816122768" co-class="co-affbox">Peter Vollenweider</a>,<sup>5</sup> <a href="/pubmed/?term=Marques-Vidal%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130816120544" co-class="co-affbox">Pedro Marques-Vidal</a>,<sup>10,</sup><sup>‡</sup> and  <a href="/pubmed/?term=Whitfield%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994" class="affpopup" co-rid="_co_idm140130781317184" co-class="co-affbox">John B. Whitfield</a><sup>7,</sup><sup>‡</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140130781264720"><h3 class="no_margin">Zoltán Kutalik</h3><p><sup>1</sup>Department of Medical Genetics, University of Lausanne, Lausanne, 1005, Switzerland,</p><p><sup>2</sup>Swiss Institute of Bioinformatics, Lausanne, 1005, Switzerland,</p><div>Find articles by <a href="/pubmed/?term=Kutalik%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Zoltán Kutalik</a></div></div><div id="_co_idm140130781261760"><h3 class="no_margin">Beben Benyamin</h3><p><sup>3</sup>Queensland Statistical Genetics Laboratory, Queensland Institute of Medical Research, Brisbane 4006, Australia,</p><div>Find articles by <a href="/pubmed/?term=Benyamin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Beben Benyamin</a></div></div><div id="_co_idm140130741641568"><h3 class="no_margin">Sven Bergmann</h3><p><sup>1</sup>Department of Medical Genetics, University of Lausanne, Lausanne, 1005, Switzerland,</p><p><sup>2</sup>Swiss Institute of Bioinformatics, Lausanne, 1005, Switzerland,</p><div>Find articles by <a href="/pubmed/?term=Bergmann%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Sven Bergmann</a></div></div><div id="_co_idm140130741638608"><h3 class="no_margin">Vincent Mooser</h3><p><sup>4</sup>GlaxoSmithKline, King of Prussia, PA 19406, USA,</p><div>Find articles by <a href="/pubmed/?term=Mooser%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Vincent Mooser</a></div></div><div id="_co_idm140130741935104"><h3 class="no_margin">Gérard Waeber</h3><p><sup>5</sup>Department of Medicine, Internal Medicine, CHUV, Lausanne 1011, Switzerland,</p><div>Find articles by <a href="/pubmed/?term=Waeber%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Gérard Waeber</a></div></div><div id="_co_idm140130741932880"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>6</sup>Molecular Epidemiology Laboratory,  and</p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Grant W. Montgomery</a></div></div><div id="_co_idm140130741930656"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>7</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane 4006, Australia,</p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Nicholas G. Martin</a></div></div><div id="_co_idm140130774771792"><h3 class="no_margin">Pamela A.F. Madden</h3><p><sup>8</sup>Department of Psychiatry, Washington University, St Louis 63110, USA,</p><div>Find articles by <a href="/pubmed/?term=Madden%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Pamela A.F. Madden</a></div></div><div id="_co_idm140130774769568"><h3 class="no_margin">Andrew C. Heath</h3><p><sup>8</sup>Department of Psychiatry, Washington University, St Louis 63110, USA,</p><div>Find articles by <a href="/pubmed/?term=Heath%20AC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Andrew C. Heath</a></div></div><div id="_co_idm140130774767344"><h3 class="no_margin">Jacques S. Beckmann</h3><p><sup>1</sup>Department of Medical Genetics, University of Lausanne, Lausanne, 1005, Switzerland,</p><p><sup>9</sup>Service of Medical Genetics, Centre Hospitalier Universitaire Vaudois (CHUV) University Hospital, 1011 Lausanne, Switzerland, and</p><div>Find articles by <a href="/pubmed/?term=Beckmann%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Jacques S. Beckmann</a></div></div><div id="_co_idm140130816122768"><h3 class="no_margin">Peter Vollenweider</h3><p><sup>5</sup>Department of Medicine, Internal Medicine, CHUV, Lausanne 1011, Switzerland,</p><div>Find articles by <a href="/pubmed/?term=Vollenweider%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Peter Vollenweider</a></div></div><div id="_co_idm140130816120544"><h3 class="no_margin">Pedro Marques-Vidal</h3><p><sup>10</sup>Institute of Social and Preventive Medicine (IUMSP), University Hospital Center (CHUV) and University of Lausanne, Lausanne 1005, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Marques-Vidal%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">Pedro Marques-Vidal</a></div></div><div id="_co_idm140130781317184"><h3 class="no_margin">John B. Whitfield</h3><p><sup>7</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane 4006, Australia,</p><div>Find articles by <a href="/pubmed/?term=Whitfield%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21665994">John B. Whitfield</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140130774463664_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140130774463664_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140130774463664_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140130774463664_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Medical Genetics, University of Lausanne, Lausanne, 1005, Switzerland,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Swiss Institute of Bioinformatics, Lausanne, 1005, Switzerland,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Queensland Statistical Genetics Laboratory, Queensland Institute of Medical Research, Brisbane 4006, Australia,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>GlaxoSmithKline, King of Prussia, PA 19406, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Medicine, Internal Medicine, CHUV, Lausanne 1011, Switzerland,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Molecular Epidemiology Laboratory,  and</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Genetic Epidemiology Laboratory, Queensland Institute of Medical Research, Brisbane 4006, Australia,</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Psychiatry, Washington University, St Louis 63110, USA,</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Service of Medical Genetics, Centre Hospitalier Universitaire Vaudois (CHUV) University Hospital, 1011 Lausanne, Switzerland, and</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Institute of Social and Preventive Medicine (IUMSP), University Hospital Center (CHUV) and University of Lausanne, Lausanne 1005, Switzerland</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed: Tel: +41 21 692 54 63; Email: <a href="mailto:dev@null" data-email="hc.linu@kilatuk.natloz" class="oemail">hc.linu@kilatuk.natloz</a></div><div id="AN1"><sup>†</sup>The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors.</div><div id="AN2"><sup>‡</sup>The authors wish it to be known that, in their opinion, the last two authors should be regarded as joint Last Authors.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140130774463664_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Apr 27; Accepted 2011 Jun 7.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140130774463664_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2011. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3159549/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supplementary Data
            </div>
            
            <div><a href="/pmc/articles/PMC3159549/bin/supp_20_18_3710__index.html" data-ga-action="click_feat_suppl">supp_20_18_3710__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;D6D17A47-0220-4908-87AF-07E6C61FA904</div>
            <div><a href="/pmc/articles/PMC3159549/bin/supp_ddr272_ddr272supp.doc" data-ga-action="click_feat_suppl">supp_ddr272_ddr272supp.doc</a><span style="color:gray"> (853K)</span></div><div class="small guid">GUID:&nbsp;8E2DCF6D-8D1D-4247-9530-BF41892E1C23</div>
         </div></dd></dl></div><div id="idm140130779621152" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140130779621152title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Polysaccharide sidechains attached to proteins play important roles in cell–cell and receptor–ligand interactions. Variation in the carbohydrate component has been extensively studied for the iron transport protein transferrin, because serum levels of the transferrin isoforms asialotransferrin + disialotransferrin (carbohydrate-deficient transferrin, CDT) are used as biomarkers of excessive alcohol intake. We conducted a genome-wide association study to assess whether genetic factors affect CDT concentration in serum. CDT was measured in three population-based studies: one in Switzerland (CoLaus study, <em>n</em> = 5181) and two in Australia (<em>n</em> = 1509, <em>n</em> = 775). The first cohort was used as the discovery panel and the latter ones served as replication. Genome-wide single-nucleotide polymorphism (SNP) typing data were used to identify loci with significant associations with CDT as a percentage of total transferrin (CDT%). The top three SNPs in the discovery panel (rs2749097 near <em>PGM1</em> on chromosome 1, and missense polymorphisms rs1049296, rs1799899 in <em>TF</em> on chromosome 3) were successfully replicated , yielding genome-wide significant combined association with CDT% (<em>P</em> = 1.9 × 10<sup>−9</sup>, 4 × 10<sup>−39</sup>, 5.5 × 10<sup>−43</sup>, respectively) and explain 5.8% of the variation in CDT%. These allelic effects are postulated to be caused by variation in availability of glucose-1-phosphate as a precursor of the glycan (<em>PGM1</em>), and variation in transferrin (<em>TF</em>) structure.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p4" class="p p-first">Many proteins undergo N- or O-glycosylation during or immediately after peptide synthesis. Glycosylation of proteins modifies their physicochemical properties and plays an important role in receptor–ligand and cell–cell interactions (<a href="#DDR272C1" rid="DDR272C1" class=" bibr popnode">1</a>). It has been estimated that half of all proteins are glycoproteins (<a href="#DDR272C2" rid="DDR272C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294421">2</a>). Some proteins show variation in their glycosylation; the glycosylation sites may or may not be occupied (<a href="#DDR272C3" rid="DDR272C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294418">3</a>), the carbohydrate structure can vary, and this variation can affect function. Conditions leading to variation in protein glycosylation include congenital disorders of glycosylation (<a href="#DDR272C4" rid="DDR272C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294414">4</a>), infection, inflammation and cancers. In principle, variation in protein glycosylation could reflect genetic or acquired variation in (i) the enzymes which carry out the multiple steps of attachment, extension and modification of the glycan, (ii) the target or substrate protein which is to be glycosylated, (iii) the ‘quality control’ mechanisms which check and if necessary recycle proteins, or (iv) the turnover or repair of glycosylated proteins after their initial synthesis.</p><p id="__p5">Variation in the glycosylation of transferrin has been extensively studied because of the role of plasma transferrin in iron transport, and because the concentration or proportion of less-glycosylated transferrin isoforms (asialotransferrin and disialotransferrin, collectively known as carbohydrate-deficient transferrin, CDT) can serve as a biomarker of excessive alcohol consumption. Circulating transferrin can vary between the asialo- and octasialo- isoforms, with tetrasialotransferrin the most abundant (<a href="#DDR272C5" rid="DDR272C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294419">5</a>).</p><p id="__p6">Up to 40% of genetic variation in total transferrin levels is due to variants in the transferrin <em>(TF</em>) and hemochromatosis (<em>HFE</em>) genes (<a href="#DDR272C6" rid="DDR272C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294417">6</a>), but whether transferrin glycosylation is also gene dependent has never been assessed. In this study, we first tested whether <em>TF</em> gene polymorphisms have significant effects on CDT concentration or CDT as a percentage of total transferrin (CDT%). We then conducted a genome-wide search for polymorphisms affecting CDT, tested replication of significant or suggestive findings with independent cohorts and meta-analysed the combined dataset. In this work, we used data from three studies measuring CDT in different ways, which allows assessment of the robustness of allelic associations across methods.</p><p id="__p7" class="p p-last">Identification of significant allelic associations for transferrin glycosylation has the potential to improve clinical interpretation of CDT% as a diagnostic aid in the management of alcohol use disorders, through estimation of genotype-specific reference ranges. This would also provide proof of principle for potential allelic effects on other diagnostic glycoprotein measurements, such as tumour markers; and it has theoretical implications for genetic variation in the control of post-translational modification of proteins.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p8" class="p p-first">Relevant subject characteristics, for both the Swiss (CoLaus) and Australian participants, are summarized in Table&nbsp;<a href="/pmc/articles/PMC3159549/table/DDR272TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR272TB1" rid-ob="ob-DDR272TB1" co-legend-rid=""><span>1</span></a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR272TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p9">Studies and participants. All studies were population-based, but in the second Australian study subjects reporting high alcohol intake were prioritised for CDT measurement. In the table <em>R</em> stands for Pearson correlation, <em>P</em> for <em>P</em> value, <em>N</em> for sample size, and <em>SD</em> for standard deviation</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140130774520864" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">CDT method</th><th rowspan="1" colspan="1" align="left">CoLaus</th><th rowspan="1" colspan="1" align="left">Australia, 1993–96</th><th rowspan="1" colspan="1" align="left">Australia, 2004–06</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Capillary electrophoresis</th><th rowspan="1" colspan="1" align="left">Mini-column (CDTect)</th><th rowspan="1" colspan="1" align="left">Immunoassay (N-Latex CDT)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>n</em> (male, female) with CDT in units/l</td><td rowspan="1" colspan="1">690 (338, 352)</td><td rowspan="1" colspan="1">1400 (539, 861)</td><td rowspan="1" colspan="1">2087 (1000, 1087)</td></tr><tr><td rowspan="1" colspan="1"><em>n</em> (male, female) with CDT in percent</td><td rowspan="1" colspan="1">5909 (2815, 3094)</td><td rowspan="1" colspan="1">1353 (514, 839)</td><td rowspan="1" colspan="1">2087 (1000, 1087)</td></tr><tr><td rowspan="1" colspan="1"><em>n</em> (male, female) with CDT% and genome-wide SNP data</td><td rowspan="1" colspan="1">5181 (2454, 2727)</td><td rowspan="1" colspan="1">745 (169, 576)</td><td rowspan="1" colspan="1">1509 (764, 745)</td></tr><tr><td rowspan="1" colspan="1">Drinks per week (Mean, SD, Range)</td><td rowspan="1" colspan="1">7.0 ± 9.1 (0–80)</td><td rowspan="1" colspan="1">6.8 ± 9.0 (0–70)</td><td rowspan="1" colspan="1">24.4 ± 23.1 (0–315)</td></tr><tr><td rowspan="1" colspan="1">CDT, percent (Mean, SD)</td><td rowspan="1" colspan="1">0.947 ± 0.783</td><td rowspan="1" colspan="1">0.614 ± 0.303</td><td rowspan="1" colspan="1">1.766 ± 0.649</td></tr><tr><td rowspan="1" colspan="1">CDT, units (Mean, SD)</td><td rowspan="1" colspan="1">31.2 ± 25.2</td><td rowspan="1" colspan="1">17.4 ± 8.9</td><td rowspan="1" colspan="1">48.9 ± 17.2</td></tr><tr><td rowspan="1" colspan="1">Total transferrin, g/l (Mean, SD)</td><td rowspan="1" colspan="1">3.80 ± 0.44</td><td rowspan="1" colspan="1">2.85 ± 0.52</td><td rowspan="1" colspan="1">2.80 ± 0.41</td></tr><tr><td rowspan="1" colspan="1">Correlation, drinks per week versus CDT percent (R, p, N)</td><td rowspan="1" colspan="1">0.407, 1.09 × 10<sup>−234</sup>, 5401</td><td rowspan="1" colspan="1">0.241, 2.52 × 10<sup>−19</sup>, 1353</td><td rowspan="1" colspan="1">0.373, 1.07 × 10<sup>−67</sup>, 2020</td></tr><tr><td rowspan="1" colspan="1">Correlation, drinks per week versus CDT units (R, p, N)</td><td rowspan="1" colspan="1">0.367, 3.71 × 10<sup>−25</sup>, 690</td><td rowspan="1" colspan="1">0.200, 4.15 × 10<sup>−14</sup>, 1400</td><td rowspan="1" colspan="1">0.352, 8.07 × 10<sup>−60</sup>, 2020</td></tr></tbody></table></div><div id="largeobj_idm140130774520864" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3159549/table/DDR272TB1/?report=objectonly">Open in a separate window</a></div></div><p id="__p10" class="p">Correlations between CDT results for two occasions 10 years apart were estimated from data for 66 Australian participants with both CDTect and N-Latex results. The repeatability of this measure was 0.59 for CDT concentration and 0.63 for CDT%, and these correlations were only slightly reduced by adjustment for sex and reported alcohol consumption. Heritability estimates, derived from sibling correlations in the second Australian (N-Latex) study, were 0.59 for CDT concentration and 0.60 for CDT%.</p><div id="s2a" class="sec"><h3 id="s2atitle">Genome-wide association results for CDT percentage</h3><p id="__p11" class="p p-first">In our genome-wide association study (GWAS), two regions showed significant associations with CDT% (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr272/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). These were on chromosomes 1 and 3, within or close to the <em>PGM1</em> and <em>TF</em> genes, respectively. Both loci fulfilled the selection criteria listed in Materials and Methods, with lead single-nucleotide polymorphisms (SNPs) rs2749097 (<em>P</em> = 3.15 × 10<sup>−6</sup>) and rs1534166 (<em>P</em> = 1 × 10<sup>−38</sup>) in the discovery set. In the latter region, both the association and the linkage disequilibrium (LD) were so strong that we performed step-wise model selection in order not to miss imperfect tagging or allelic heterogeneity in the region. This analysis led to the most plausible association model, which included three non-synonymous SNPs: rs1799899 (<em>TF</em> Gly277Ser, <em>P</em> = 3.35 × 10<sup>−35</sup>), rs1049296 (<em>TF</em> Pro589Ser, <em>P</em> = 2.13 × 10<sup>−32</sup>), rs8177318 (<em>TF</em> Ser55Arg, <em>P</em> = 3.58 × 10<sup>−4</sup>). This result indicates that the original strongest association with rs1534166 was tagging multiple missense variants in the transferrin gene. Therefore, we selected rs2749097 (<em>PGM1</em>, chromosome 1) and the three non-synonymous SNPs in <em>TF</em> for <em>in silico</em> replication. The hit on chromosome 1 (rs2749097) was confirmed in both replication cohorts (<em>P</em> = 0.0068 and <em>P</em> = 0.0018) in a direction-consistent manner, yielding a combined <em>P</em>-value of 1.9 × 10<sup>−9</sup> (Fig.&nbsp;<a href="/pmc/articles/PMC3159549/figure/DDR272F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR272F1" rid-ob="ob-DDR272F1" co-legend-rid="lgnd_DDR272F1"><span>1</span></a>A).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDR272F1" co-legend-rid="lgnd_DDR272F1"><a href="/pmc/articles/PMC3159549/figure/DDR272F1/" target="figure" rid-figpopup="DDR272F1" rid-ob="ob-DDR272F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140130744778240" class="figure"><a href="/pmc/articles/PMC3159549/figure/DDR272F1/" target="figure" rid-figpopup="DDR272F1" rid-ob="ob-DDR272F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3159549_ddr27201.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddr27201.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3159549/bin/ddr27201.jpg"></a></div><div id="largeobj_idm140130744778240" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3159549/figure/DDR272F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDR272F1"><div><a class="figpopup" href="/pmc/articles/PMC3159549/figure/DDR272F1/" target="figure" rid-figpopup="DDR272F1" rid-ob="ob-DDR272F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p12">Local plots for the two regions associated with CDT%. The <em>y</em>-axis shows significance of the association as –log<sub>10</sub>(P), <em>x</em>-axis is physical distance (on Build 36). Shading of the data points indicates the strength of LD with the most significant SNP in the region. (<strong>A</strong>) SNP associations with CDT (as a percentage of total transferrin) near the <em>PGM1</em> locus on chromosome 1. The plotted combined <em>P</em>-values are from a meta-analysis of the discovery (CoLaus) and both (Australian) replication cohorts. (<strong>B</strong>) Association plot as described above, but for the region near the <em>TF</em> and <em>SRPRB</em> loci on chromosome 3. The plot shows combined <em>P-</em>values from a meta-analysis of the discovery (CoLaus) and the N-Latex-based (Australian) replication cohorts, showing univariate association results for all SNPs (red or uncoloured symbols). The three TF SNPs obtained by the multivariate model selection procedure (rs1049296, rs1799899 and rs8177318) are marked with green boundary circle.</p></div></div></div><p id="__p13">Two of the three non-synonymous markers (rs1799899 and rs1049296) successfully replicated in the N-Latex-based Australian cohort (<em>P</em> = 9.7 × 10<sup>−6</sup>, <em>P</em> = 7.1 × 10<sup>−13</sup>, respectively), giving rise to very significant meta <em>P-</em>values (<em>P</em> = 4 × 10<sup>−39</sup>, <em>P</em> = 5.5 × 10<sup>−43</sup>). The third SNP (rs8177318) showed a direction-consistent replication signal, but—consistent with the weaker initial association and the smaller replication sample size—could not reach nominally significant replication <em>P</em>-value.</p><p id="__p14">The association results for the significant chromosome 3 region are shown in Figure&nbsp;<a href="/pmc/articles/PMC3159549/figure/DDR272F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDR272F1" rid-ob="ob-DDR272F1" co-legend-rid="lgnd_DDR272F1"><span>1</span></a>B, where initial GWAS-significant associations were found across a range of genes and intergenic regions between <em>BFSP2</em> and <em>RAB6B</em>, with the most significant SNP (rs1534166, <em>P</em> = 4.3 × 10<sup>−46</sup>) being in <em>SRPRB</em>. In addition, the association results from the conditional analysis (on rs1799899 and rs1049296) clarified that the associations previously seen in the <em>BFSP2-CDV3-TOPBP1</em> and <em>RAB6B</em> regions could be ascribed to LD with the <em>TF</em> SNPs. Results are summarized in Table&nbsp;<a href="/pmc/articles/PMC3159549/table/DDR272TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR272TB2" rid-ob="ob-DDR272TB2" co-legend-rid=""><span>2</span></a> and shown in more detail in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr272/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDR272TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p15">Summary of the most significant allelic associations with CDT and total transferrin</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140130742287104" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">bp</th><th rowspan="1" colspan="1" align="left">Closest gene</th><th colspan="2" rowspan="1" align="left">Alleles<hr></th><th colspan="2" rowspan="1" align="left">MAF<hr></th><th colspan="2" rowspan="1" align="left">Discovery<hr></th><th colspan="2" rowspan="1" align="left">Replication I<hr></th><th colspan="2" rowspan="1" align="left">Replication II<hr></th><th colspan="3" rowspan="1" align="left">Combined<hr></th><th rowspan="1" colspan="1" align="left"><em>P</em><sub>het</sub></th><th rowspan="1" colspan="1" align="left">Comments</th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Effect</th><th rowspan="1" colspan="1" align="left">Other</th><th rowspan="1" colspan="1" align="left">CH</th><th rowspan="1" colspan="1" align="left">AU</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="left">CH<hr></th><th colspan="2" rowspan="1" align="left">AU<hr></th><th colspan="2" rowspan="1" align="left">AU<hr></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th colspan="2" rowspan="1" align="left">CE<hr></th><th colspan="2" rowspan="1" align="left">Latex<hr></th><th colspan="2" rowspan="1" align="left">CDTect<hr></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1"></th></tr></thead><tbody><tr><td colspan="19" rowspan="1">CDT percent</td></tr><tr><td rowspan="1" colspan="1"> rs2749097</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">63 839 489</td><td rowspan="1" colspan="1"><em>PGM1</em></td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.187</td><td rowspan="1" colspan="1">0.184</td><td rowspan="1" colspan="1">−0.105</td><td rowspan="1" colspan="1">3.15E−06</td><td rowspan="1" colspan="1">−0.134</td><td rowspan="1" colspan="1">0.0068</td><td rowspan="1" colspan="1">−0.242</td><td rowspan="1" colspan="1">0.0018</td><td rowspan="1" colspan="1">−0.119</td><td rowspan="1" colspan="1">0.020</td><td rowspan="1" colspan="1">1.89E−09</td><td rowspan="1" colspan="1">2.24E−01</td><td rowspan="1" colspan="1">(a)</td></tr><tr><td rowspan="1" colspan="1"> rs8177318</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 950 075</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.030</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">−0.200</td><td rowspan="1" colspan="1">3.58E−04</td><td rowspan="1" colspan="1">−0.141</td><td rowspan="1" colspan="1">0.290</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.187</td><td rowspan="1" colspan="1">0.051</td><td rowspan="1" colspan="1">2.10E−04</td><td rowspan="1" colspan="1">6.92E−01</td><td rowspan="1" colspan="1">(b, c)</td></tr><tr><td rowspan="1" colspan="1"> rs1799899</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 958 510</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.058</td><td rowspan="1" colspan="1">0.054</td><td rowspan="1" colspan="1">−0.480</td><td rowspan="1" colspan="1">3.35E−35</td><td rowspan="1" colspan="1">−0.364</td><td rowspan="1" colspan="1">9.70E−06</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.461</td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">3.96E−39</td><td rowspan="1" colspan="1">1.88E−01</td><td rowspan="1" colspan="1">(b, c)</td></tr><tr><td rowspan="1" colspan="1"> rs1049296</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 977 052</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.170</td><td rowspan="1" colspan="1">0.169</td><td rowspan="1" colspan="1">−0.270</td><td rowspan="1" colspan="1">2.13E−32</td><td rowspan="1" colspan="1">−0.365</td><td rowspan="1" colspan="1">7.10E−13</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.288</td><td rowspan="1" colspan="1">0.021</td><td rowspan="1" colspan="1">5.45E−43</td><td rowspan="1" colspan="1">9.82E−02</td><td rowspan="1" colspan="1">(b, c)</td></tr><tr><td colspan="19" rowspan="1">CDT concentration</td></tr><tr><td rowspan="1" colspan="1"> rs1799899</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 958 510</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.058</td><td rowspan="1" colspan="1">0.054</td><td rowspan="1" colspan="1">−0.551</td><td rowspan="1" colspan="1">5.66E−07</td><td rowspan="1" colspan="1">−0.319</td><td rowspan="1" colspan="1">9.90E−05</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.402</td><td rowspan="1" colspan="1">0.066</td><td rowspan="1" colspan="1">1.01E−09</td><td rowspan="1" colspan="1">9.12E−02</td><td rowspan="1" colspan="1">(b, d)</td></tr><tr><td rowspan="1" colspan="1"> rs1534166</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">135 019 765</td><td rowspan="1" colspan="1"><em>SRPRB</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.302</td><td rowspan="1" colspan="1">0.284</td><td rowspan="1" colspan="1">−0.282</td><td rowspan="1" colspan="1">3.03E−07</td><td rowspan="1" colspan="1">−0.293</td><td rowspan="1" colspan="1">5.80E−12</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">−0.289</td><td rowspan="1" colspan="1">0.034</td><td rowspan="1" colspan="1">1.56E−17</td><td rowspan="1" colspan="1">8.77E−01</td><td rowspan="1" colspan="1">(b)</td></tr><tr><td colspan="19" rowspan="1">Total transferrin</td></tr><tr><td rowspan="1" colspan="1"> rs6439434</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 933 069</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.491</td><td rowspan="1" colspan="1">0.473</td><td rowspan="1" colspan="1">0.199</td><td rowspan="1" colspan="1">9.13E−04</td><td rowspan="1" colspan="1">0.199</td><td rowspan="1" colspan="1">1.63E−08</td><td rowspan="1" colspan="1">0.133</td><td rowspan="1" colspan="1">9.98E−03</td><td rowspan="1" colspan="1">0.182</td><td rowspan="1" colspan="1">0.026</td><td rowspan="1" colspan="1">3.05E−12</td><td rowspan="1" colspan="1">5.47E−01</td><td rowspan="1" colspan="1">(a, f)</td></tr><tr><td rowspan="1" colspan="1"> rs3811647</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">134 966 727</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.318</td><td rowspan="1" colspan="1">0.338</td><td rowspan="1" colspan="1">0.362</td><td rowspan="1" colspan="1">1.27E−10</td><td rowspan="1" colspan="1">0.358</td><td rowspan="1" colspan="1">5.25E−19</td><td rowspan="1" colspan="1">0.353</td><td rowspan="1" colspan="1">8.80E−09</td><td rowspan="1" colspan="1">0.358</td><td rowspan="1" colspan="1">0.029</td><td rowspan="1" colspan="1">1.45E−35</td><td rowspan="1" colspan="1">9.94E−01</td><td rowspan="1" colspan="1">(a)</td></tr><tr><td rowspan="1" colspan="1"> rs1800562</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">26 201 120</td><td rowspan="1" colspan="1"><em>HFE</em></td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">0.078</td><td rowspan="1" colspan="1">−0.466</td><td rowspan="1" colspan="1">0.00078</td><td rowspan="1" colspan="1">−0.647</td><td rowspan="1" colspan="1">8.67E−22</td><td rowspan="1" colspan="1">−0.687</td><td rowspan="1" colspan="1">9.46E−10</td><td rowspan="1" colspan="1">−0.629</td><td rowspan="1" colspan="1">0.053</td><td rowspan="1" colspan="1">2.13E−32</td><td rowspan="1" colspan="1">4.21E−01</td><td rowspan="1" colspan="1">(a, e)</td></tr></tbody></table></div><div id="largeobj_idm140130742287104" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3159549/table/DDR272TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140130816014304"><p id="__p16" class="p p-first-last">(a) Meta-analysis based on all three methods.</p></div><div id="idm140130816013680"><p id="__p17" class="p p-first-last">(b) Meta-analysis based on CE and Latex methods only.</p></div><div id="idm140130816013056"><p id="__p18" class="p p-first-last">(c) Multivariate analysis of all three markers in TF exons (rs8177318, rs1799899, and rs1049296).</p></div><div id="idm140130816012272"><p id="__p19" class="p p-first-last">(d) Conditioned on the most significant SNP in the initial analysis (rs1534166).</p></div><div id="idm140130816011600"><p id="__p20" class="p p-first-last">(e) Conditioned on the most significant SNP in the initial analysis (rs3811647).</p></div><div id="idm140130816010928"><p id="__p21" class="p p-first-last">(f) Conditioned on the two most significant SNPs in the initial analysis (rs3811647 and rs1800562).</p></div><div id="idm140130743195216"><p id="__p22" class="p p-first-last">CH, CoLaus; AU, Australia. <em>P</em><sub>het</sub> values result from test of heterogeneity of allelic effects between the two or three datasets (CE, capillary electrophoresis, N-Latex and CDTect). A fuller list of SNP associations is given in <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddr272/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1.</a></p></div></div></div><p id="__p23">Since CDT% is correlated with transferrin levels, we investigated whether SNPs associated with CDT% also show association when CDT% is conditioned on transferrin levels. Although associations became slightly less significant, effect sizes remained unchanged (<em>t</em>-test <em>P</em>-value &gt; 0.75 for all SNPs). Thus, we consider that these genetic associations hold independently of the total transferrin level.</p><p id="__p24" class="p p-last">We also checked association <em>P</em>-values for these two <em>TF</em> SNPs (rs1534166 and rs1799899) in the other Australian replication cohort, but the results from the CDTect method showed an opposite direction of the allelic effects compared with the other assessment methods (<em>P</em><sub>het</sub> = 2.64 × 10<sup>−9</sup>). This arises from the analytical principle of the CDTect method, and because changes in the amino acid sequence of transferrin can alter the isoelectric point (pI) of the isoforms. A more detailed explanation can be found in Discussion. For this reason, we excluded the CDTect replication sample and used only the N-Latex-based replication data for the chromosome 3 region.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Genome-wide association results for CDT concentration</h3><p id="__p25" class="p p-first-last">For CDT concentration, the only significant SNPs were in the <em>TF-SRPRB</em> region of chromosome 3 (see Table&nbsp;<a href="/pmc/articles/PMC3159549/table/DDR272TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR272TB2" rid-ob="ob-DDR272TB2" co-legend-rid=""><span>2</span></a>). As for CDT%, the most significant SNP was rs1534166 in <em>SRPRB</em> (<em>P</em> = 1.6 × 10<sup>−17</sup>). Further, rs1799899 in <em>TF</em> was independently significant (<em>P</em> = 1.0 × 10<sup>−9</sup>) when the data were reanalysed after adjustment for the effects of rs1534166. These <em>P</em>-values were less significant than the equivalent ones for CDT%, because the number of Swiss participants who had total transferrin assessed (from which CDT concentration could be calculated) was only 690, but the effect sizes (betas in Table&nbsp;<a href="/pmc/articles/PMC3159549/table/DDR272TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR272TB2" rid-ob="ob-DDR272TB2" co-legend-rid=""><span>2</span></a>) were similar for CDT% and CDT concentration. Total CDT concentration was more weakly associated with the CDT%-associated <em>PGM1</em> polymorphism (rs2749097) yielding <em>P</em> = 3.99 × 10<sup>−4</sup>.</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">Genome-wide association results for total transferrin</h3><p id="__p26" class="p p-first-last">As previously reported, SNPs in <em>TF</em> and <em>HFE</em> showed significant effects on total transferrin concentration (see Table&nbsp;<a href="/pmc/articles/PMC3159549/table/DDR272TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDR272TB2" rid-ob="ob-DDR272TB2" co-legend-rid=""><span>2</span></a>). However, the <em>HFE</em> locus had no significant effects on either CDT% or CDT concentration. Even though the <em>TF</em> SNPs most strongly affecting total transferrin concentration showed only low-to-moderate LD with rs1799899 (<em>r</em><sup>2</sup> = 0.002 for rs6439434 and 0.026 for rs3811647), the combined association <em>P</em>-value for this SNP was still highly significant (<em>P</em> = 5.1 × 10<sup>−12</sup>).</p></div><div id="__sec1" class="sec sec-last"><h3 id="__sec1title">Genome-wide association results for total transferrin</h3><p id="__p27" class="p p-first-last">The discovered variants explained 5.8% of the variance of CDT%, 4.9% for CDT levels and 11.2% for transferrin levels.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p28" class="p p-first">We have found that both the absolute and relative concentrations of ‘carbohydrate-deficient’ isoforms of transferrin are subject to genetic variation. Indeed, comparison of the heritability and repeatability estimates suggests that all or nearly all of the repeatable variation is genetic. As explained in Materials and Methods section, we tried to correct for all relevant environmental factors (first two principal components (PCs), sex, smoking status and alcohol consumption) to maximize our chances to tease out the most important genetic components determining CDT%. Other factors, such as pregnancy, haemodialysis, liver disease are either uncommon in the general population (in the CoLaus sample we had &lt;11 pregnant women, 7 participants with liver disease and 9 participants with alcohol problems, &lt;0.5% of the total) or have negligible effect on CDT levels, thus could not have considerably influenced CDT levels.</p><p id="__p29">In terms of specific SNPs and genes, there is evidence that variation near to the <em>PGM1</em> gene on chromosome 1, and in the <em>TF</em> gene on chromosome 3, contributes to the overall genetic effect. This leads us to consider how these genes fit into the processes of synthesis, release or circulation of the isoforms of transferrin.</p><p id="__p30">The association with <em>PGM1</em> may be related to synthesis of the carbohydrate sidechains, which are attached to the protein and then modified by removal and addition of carbohydrate groups (<a href="#DDR272C7" rid="DDR272C7" class=" bibr popnode">7</a>). The phosphoglucomutase enzyme converts glucose-6-phosphate into glucose-1-phosphate needed for synthesis of UDP-glucose, a precursor of the carbohydrate which is eventually transferred to the protein (<a href="#DDR272C8" rid="DDR272C8" class=" bibr popnode">8</a>). However, phosphoglucomutase catalyses a very early step in this pathway and it is by no means clear how variation in the <em>PGM1</em> gene would affect the carbohydrate structure of the glycoprotein. Major defects in <em>PGM1</em> are known (<a href="http://www.ncbi.nlm.nih.gov/omim/171900">http://www.ncbi.nlm.nih.gov/omim/171900</a>), but they lead to a form of glycogen storage disease rather than a carbohydrate-deficient glycoprotein syndrome. However, <em>PGM1</em> expression in human liver (<a href="#DDR272C9" rid="DDR272C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294424">9</a>) is significantly affected by rs4643 at 63 898 026 bp (expression <em>P</em> = 1.0 × 10<sup>−6</sup>), and this SNP is one of a group of three affecting CDT% at <em>P</em> &lt; 10<sup>−8</sup>. It is reasonable to accept that the SNPs on chromosome 1 showing significant association with CDT affect expression of <em>PGM1</em> in the liver (the site of transferrin synthesis), but an understanding of how phosphoglucomutase activity affects transferrin isoforms will require experiments which are beyond the scope of this study.</p><p id="__p31">The associations on chromosome 3 are more readily related to relevant genes, but the details are complex. Although rs1534166 in <em>SRPRB</em> showed the strongest association (<em>P</em> = 4.3 × 10<sup>−46</sup>) with CDT% in the initial combined analysis, more detailed examination of this region showed two independent effects at neighbouring loci in different exons of <em>TF</em>, rs1799899 (Gly277Ser, <em>P</em> = 3.96 × 10<sup>−39</sup>), and rs1049296 (Pro589Ser, <em>P</em> = 5.45 × 10<sup>−43</sup>). These associations were identified using step-wise model selection; hence, each variant is associated when conditioned on the other one. The SNP rs1799899 had a substantially greater allelic effect than rs1049296, but was less common [minor allele frequency (MAF) = 5.8%]. A third non-synonymous SNP in <em>TF</em>, rs8177318 or Ser55Arg, may also contribute, but this could not be shown at the genome-wide significance level. Through this example we illustrate how computational methods can help in tracking down potentially causal variants, revealing allelic heterogeneity and substantially (from 3.6 to 5.8%) increasing the explained variance compared with univariate analysis.</p><p id="__p32">The effects of these <em>TF</em> polymorphisms remain unchanged even in association with transferrin-corrected CDT%, so the significant effects on CDT% cannot be explained as being secondary to changes in total transferrin concentration. They may be due to changes in tertiary structure of transferrin because of the amino acid differences (Pro589Ser from rs1049296 and/or Gly277Ser from rs1799899), even though these changes are not at the N-glycosylation sites (which are asparagine residues at positions 413 and 611).</p><p id="__p33">The heterogeneity of allelic effects across methods at the <em>TF</em> locus can be ascribed to these non-synonymous coding SNPs and the difference in analytical principles between the methods for measurement of CDT. The CDTect method relied on elution of isoforms with a pI &gt;5.7 from an anion-exchange column followed by measurement of transferrin in the eluate. Any changes in the pI of transferrin isoforms resulting from changes in the amino acid sequence will either increase or decrease the amount of transferrin eluting from the column. The effects of the Pro589Ser (or C1/C2) polymorphism in the transferrin protein on the pI of transferrin and estimated CDT concentrations were examined by Stibler <em>et al.</em> (<a href="#DDR272C10" rid="DDR272C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294427">10</a>). Although the effect was not significant with their limited number of samples, they found mean CDT concentrations of 51, 53 and 58 mg/l for C1, C1/C2 and C2 samples, respectively. The effects of this variant on the pI of disialotransferrin were illustrated by Helander <em>et al.</em> (<a href="#DDR272C11" rid="DDR272C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294415">11</a>). They showed a shift in disialotransferrin towards the pI of the more usual forms of mono- and asialotransferrin in participants with the C2 variant. Because the CDTect assay measured the sum of asialo- monosialo- and part of disialotransferrin, the difference in pI leads to a greater proportion of the disialotransferrin being measured as CDT and an increase in the apparent CDT concentration. However, both the capillary electrophoresis and the N-Latex methods showed the effects of rs1049296 and rs1799899 on CDT which cannot be explained by alteration in isoelectric point. The N-Latex method is based on a monoclonal antibody, which specifically recognizes transferrin glycoforms that lack one or both of the complete N-glycans (disialo-, monosialo- and asialotransferrins) (<a href="#DDR272C12" rid="DDR272C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294429">12</a>) and does not rely on a charge-based separation of the isoforms.</p><p id="__p34">Apart from questions of how these polymorphisms affect serum CDT% or concentration, there may be practical implications for the diagnostic use of CDT measurement. CDT% is confirmed to be a robust marker of various aspects of alcohol consumption in our cohorts: the consumption of beer, wine and sprits strongly influence CDT% (<em>P</em> &lt; 10<sup>−58</sup>, <em>P</em> &lt; 10<sup>−57</sup>, <em>P</em> &lt; 10<sup>−6</sup>, respectively), so does alcohol consumption frequency (<em>P</em> &lt; 10<sup>−67</sup>) and total weekly alcohol consumption (<em>P</em> &lt; 10<sup>−134</sup>). As for any clinical test, it is necessary to establish a reference range against which patients' results can be compared. Problems with CDT as a marker for alcohol consumption have been reported, thus a definition of genotype-dependent reference intervals may improve the sensitivity or specificity of this test; this aspect will be examined in a further paper.</p><p id="__p35" class="p p-last">In summary, we have identified two loci (and three independent SNPs) affecting the degree of glycosylation of circulating transferrin through a genome-wide association approach. They account for 5.8% of phenotypic variation in CDT% or CDT, which is a high proportion compared with most GWAS outcomes. These loci require further studies to investigate the detailed mechanisms. The types of variation, affecting the process of oligosaccharide synthesis and showing the importance of protein structure, which we illustrate for transferrin, may well be relevant to other biologically important glycoproteins.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Subjects and methods: CoLaus</h3><p id="__p36" class="p p-first">The design of the CoLaus study has been described previously (<a href="#DDR272C13" rid="DDR272C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294426">13</a>). Briefly, it is a population-based study conducted between 2003 and 2006 which recruited over 6000 subjects aged 35–75 years in Lausanne, Switzerland. The following inclusion criteria were applied: (a) voluntary participation in the examination, including blood sample, (b) aged 35–75 years and (c) Caucasian origin defined as having both parents and grand-parents Caucasian (determined by birth place). The Institutional Review Board of the Centre Hospitalier Universitaire Vaudois (CHUV) in Lausanne and the Cantonal Ethics Committee approved the study protocol and signed informed consent was obtained from participants. Participants were asked to attend the outpatient clinic at the CHUV, Lausanne, in the morning after an overnight fast. Data were collected by trained field interviewers during a single visit lasting ∼60 min. Overall participation rate was 41%.</p><p id="__p37">Most biological assays were performed by the CHUV Clinical Laboratory on fresh blood samples within 2 h of blood collection, and additional aliquots were stored at –80°C. Transferrin was measured in 690 participants by immunoassay with maximum inter- and intra-batch CVs of 1.8 and 1.0%, respectively. CDT measurement was done in a single batch for 5401 participants by capillary electrophoresis using the Ceofix-CDT reagent kits for the quantification of CDT in human serum (Analis, Belgium, kits #10-004760). Separation was performed on a Beckman P/ACE 5510 System (Beckman Coulter Instruments, Switzerland) equipped with an ultraviolet detector set at 200 nm. Maximum intra-batch CV was between 3.8% (highest CDT levels) and 15.3% (lowest CDT levels).</p><p id="__p38">Nuclear DNA was extracted from whole blood for whole-genome scan analysis using Nucleon® genomic DNA extraction kit (Tepnel life sciences, Manchester, UK) according to the manufacturer's recommendations. Genotyping was performed using the Affymetrix GeneChip® Human Mapping 500K array set, as recommended by the manufacturer. Genotypes were called using BRLMM (<a href="http://www.affymetrix.com/support/technical/whitepapers/brlmm_whitepaper.pdf">http://www.affymetrix.com/support/technical/whitepapers/brlmm_whitepaper.pdf</a>) (<a href="#DDR272C14" rid="DDR272C14" class=" bibr popnode">14</a>). Duplicate individuals, and first and second degree relatives, were identified by computing genomic identity-by-descent coefficients, using PLINK (<a href="#DDR272C15" rid="DDR272C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294431">15</a>). The younger individual from each duplicate or relative pair was removed. SNPs with call rate &lt;70% and individuals with call rate &lt;90% were excluded from further analysis.</p><p id="__p39">A set of unmeasured HapMap SNPs was imputed. For imputation, only autosomal SNPs that were present in HapMap release 21 were used; the dataset used for imputation was 5435 unrelated CoLaus individuals and 390 631 measured SNPs (with Hardy–Weinberg <em>P</em>-value &gt;10<sup>−7</sup> and MAF &gt;1%). Imputation was performed using IMPUTE (<a href="#DDR272C16" rid="DDR272C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294413">16</a>) version 0.2.0, and CEU haplotypes from HapMap release 21, all downloaded from IMPUTE (<a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/impute.html">http://www.stats.ox.ac.uk/~marchini/software/gwas/impute.html</a>). For any given SNP, IMPUTE computed genotype probabilities using information from all other measured SNPs except the focal SNP. Therefore, after running IMPUTE, measured SNP genotypes were used to replace the imputed genotype probabilities. Expected allele dosages were computed and plugged into the subsequent association analysis. SNPs with imputation quality (<em>r</em><sup>2</sup>-hat) &lt;0.5 or MAF &lt;1% were excluded from the analysis.</p><p id="__p40" class="p p-last">To account for possible population stratification, we computed PCs using the SMARTPCA (<a href="http://genepath.med.harvard.edu/~reich/">http://genepath.med.harvard.edu/~reich/</a>) implementation (<a href="#DDR272C17" rid="DDR272C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294428">17</a>), with default options except that no outlier removal iterations were performed. The first two PCs, sex, smoking status and alcohol consumption were used in all adjusted analyses.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Subjects and methods: Australia</h3><p id="__p41" class="p p-first">Australian subjects participated in one or both of two studies evaluating the relationship between serum CDT concentration and alcohol consumption in the general population. All study participants gave informed consent, and the protocols were approved by appropriate institutional ethics review committees.</p><p id="__p42">The first of these (<a href="#DDR272C18" rid="DDR272C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294423">18</a>) was a twin study designed to assess the heritability of alcohol consumption and alcohol dependence, and to evaluate biochemical effects of excessive alcohol intake. Telephone interviews were conducted with a total of 7764 participants and blood was collected from 3375 of them (1134 men and 2241 women) between 1993 and 1996. Secondly, a twin-family study conducted between 2004 and 2006 (<a href="#DDR272C19" rid="DDR272C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294420">19</a>) focused on identification of genetic loci contributing to susceptibility to alcohol or nicotine dependence, and to liver damage and biochemical abnormalities among excessive drinkers. Blood was collected from 9031 participants (3998 men and 5033 women) aged 18–92 years. In both studies, participants provided information about their alcohol use and symptoms associated with alcohol dependence as part of a telephone interview, and at the time of blood collection they completed a 7-day retrospective diary of alcohol use.</p><p id="__p43">In the first study, CDT was measured on 1400 people aged 29–92 years (539 men and 861 women; the mean ages 44.8 and 46.5, respectively) using the Pharmacia CDTect method, in which CDT isoforms are eluted from an anion-exchange column and measured by immunoassay of transferrin. The CDTect method may be subject to bias due to polymorphisms in the <em>TF</em> gene that affect the pI of transferrin (for details, see Discussion), thus it can be unreliable for associations with <em>TF</em> SNPs. Therefore, we ignored association results for this cohort for the TF region. In the second study, CDT was measured in all available samples from participants who reported alcohol intake greater than recommended limits of 40 g/day for men and 20 g/day for women (<em>n</em> = 1173; 634 men and 539 women) and in 915 participants (367 men and 548 women) who reported lower or no alcohol intake. Serum CDT concentration was measured with a direct immunoassay (<a href="#DDR272C12" rid="DDR272C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294432">12</a>) (N-Latex CDT method, Siemens Healthcare Diagnostics) on a Dade BN-II nephelometric analyser. CDT was also calculated as a percentage of total transferrin (CDT%). One result with very low total transferrin was excluded as a probable error or outlier.</p><p id="__p44">Genome-wide SNP genotyping on DNA extracted from blood samples was performed using Illumina Human610-Quadv1 chips (∼582 000 SNPs) or HumanCNV370-Quadv3 chips (∼351 000 SNPs). SNPs were included in the analyses if they met the following conditions: Hardy–Weinberg equilibrium test <em>P</em> ≥ 10<sup>−6</sup>, MAF ≥ 1%, call-rate ≥ 0.95 and the mean value of GenCall score ≥0.7. Subjects found to be of non-European ancestry by PC analysis (EIGENSTRAT) (<a href="#DDR272C20" rid="DDR272C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294430">20</a>) of the genotyping data were also excluded. The PC analysis used a set of 276 891 autosomal SNPs that were common to Australian samples, HapMap 3 (11 global populations) and 5 Northern European (Denmark, Finland, the Netherlands, the UK and Sweden) populations from the GenomEUtwin Consortium. We excluded 277 individuals who were &gt;6 SD from the mean of PCs 1 and 2 derived from the European populations.</p><p id="__p45">Imputation of non-genotyped HapMap SNPs was performed. Because imputation is sensitive to both missingness and SNP density, to avoid introducing bias to the imputed data, a set of SNPs common to the genotyping chips (<em>n</em> = 269 840) was used for imputation. Imputation was undertaken for 17 862 individuals participating in multiple twin and twin-family studies conducted at the Queensland Institute of Medical Research, after population outliers had been removed, as a single population. The imputation was run in two stages using Mach (<a href="#DDR272C21" rid="DDR272C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294422">21</a>) (<a href="http://www.sph.umich.edu/csg/abecasis/MACH/index.html">http://www.sph.umich.edu/csg/abecasis/MACH/index.html</a>). Following imputation, data for 2.54 million Hapmap SNPs were available. The complete description of the initial quality controls of the GWAS data and imputation was given by Medland <em>et al</em>. (<a href="#DDR272C22" rid="DDR272C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294425">22</a>). The best guess genotypes were used for the association analyses if they met the same conditions as mentioned for the genotyped SNPs as well as an imputation quality, <em>r</em><sup>2</sup>-hat of 0.3 or greater.</p><p id="__p46" class="p p-last">Association analyses taking account of family relationships were conducted using MERLIN (<a href="http://www.sph.umich.edu/csg/abecasis/Merlin/">http://www.sph.umich.edu/csg/abecasis/Merlin/</a>). CDT measurements and genome-wide SNP data were both available for 2284 subjects (775 with CDT measured by CDTect and 1509 by N-Latex). Prior to association analyses, the phenotypes were log-transformed, normalized to have zero mean and unit variance and adjusted for the effects of sex, age, self-reported smoking status, alcohol intake (number of drinks in the seven days preceding blood collection) and PCs 1 and 2 from the PC analysis. Visualization and annotation of the GWAS results were performed in WGAViewer (<a href="http://people.genome.duke.edu/~dg48/WGAViewer/download.php">http://people.genome.duke.edu/~dg48/WGAViewer/download.php</a>) (<a href="#DDR272C23" rid="DDR272C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_230294416">23</a>).</p></div><div id="s4c" class="sec sec-last"><h3 id="s4ctitle">Replication and meta-analysis</h3><p id="__p47" class="p p-first">SNPs with association <em>P</em>-value &lt;5 × 10<sup>−6</sup> (in the Swiss discovery cohort), imputation quality (<em>r</em><sup>2</sup>-hat) &gt;0.5 and MAF &gt; 1% were selected, and evaluated in the replication cohorts. The selection procedure excluded all variants that had a stronger association signal in their 0.2 cM neighbourhood.</p><p id="__p48">To dissect multiple independent association signals in regions with tight LD and strong association signal (e.g. the <em>TF</em> and <em>HFE</em> genes), we applied a step-wise model selection. The forward–backward step-wise model selection procedure started from the empty model (including non-genetic covariates only). In each step of the model selection, we tried to add (forward) or remove (backward) a SNP to maximize the Bayesian Information Criterion (BIC) (<a href="#DDR272C24" rid="DDR272C24" class=" bibr popnode">24</a>) in a greedy fashion until (local) maximum is reached. If, in the procedure, only forward steps occurred, we called it <em>conditional analysis</em>, as simply the most associated variant(s) was/were included as covariate(s) in a conditional regression model. For associations with CDT and transferrin concentrations only forward steps were needed, while for CDT percentage it was a truly forward–backward model selection.</p><p id="__p49" class="p p-last">The results from the discovery and replication cohorts were combined into a fixed-effects meta-analysis using inverse variance weighting. Tests for heterogeneity were assessed using Cochran's <em>Q</em> statistic. As explained above, association results for the CDTect cohort for the <em>TF</em> region were ignored, thus the heterogeneity <em>P</em> values for that locus were computed based on two cohorts only. The details of the technical explanation for this artefact can be found in Discussion. Using only two cohorts for this region is not a limitation: the discovery <em>P</em> values for this locus were &lt;2 × 10<sup>−32</sup>, so showing successful replication in only one replication cohort (N-Latex) is sufficiently convincing.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">FUNDING</h2><p id="__p50" class="p p-first-last">The CoLaus study was supported by research grants from GlaxoSmithKline. This work was supported by research grants from GlaxoSmithKline; Faculty of Biology and Medicine of Lausanne, Switzerland; Swiss National Science Foundation (<span>33CSCO-122661</span>); Australian National Health and Medical Research Council; EU 5th and 7th Framework Programmes (<span>GenomEUtwin Project QLG2-CT-2002-01254</span>); ENGAGE Consortium (<span>HEALTH-F4-2007-201413</span>) and U.S. National Institutes of Health (<span>AA07535, AA10248, AA11998, AA13320, AA13321, AA13326, AA14041, AA17688, DA12854, MH66206</span>). Swiss National Science Foundation (<span>33CSCO-122661</span>; to S.B., <span>3100AO-116323/1</span>; to S.B., <span>310000-112552</span> to J.S.B.); Giorgi-Cavaglieri Foundation to S.B.; Swiss Institute of Bioinformatics (Service Grant to S.B.); European Framework Project 6 (AnEuploidy and EuroDia projects to S.B.); National Health and Medical Research Council (NHMRC) Fellowships (<span>552498, 339446</span>; and <span>619667</span> to B.B. and G.W.M.).</p></div><div id="__sec2" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supplementary Data:</strong>
            
            <div class="sup-box half_rhythm" id="idm140130820054080"><a href="/pmc/articles/PMC3159549/bin/supp_20_18_3710__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm140130744151648" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140130744151648title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p51">For the Australian samples, sample processing and biobank management was carried out under the leadership of Anjali Henders; genotype QC and imputation were conducted by the QIMR GWAS Group including Scott Gordon, Brian McEvoy, Sarah Medland, Dale Nyholt and Naomi Wray; and CDT measurements were performed by Veronica Dy (for N-Latex data) and Linda Fletcher and Theresa Murphy (for CDTect data).</p><p id="__p52"><em>Conflict of Interest statement.</em> V.M. is a full-time employee of GlaxoSmithKline.</p></div></div><div id="idm140130744149600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140130744149600title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDR272C1">1. <span class="element-citation">Varki A., Lowe J.B.  Biological roles of glycans. In: Varki A., Cummings R.D., Esko J.D., Freeze H.H., Stanley P., Bertozzi C.R., Hart G.W., Etzler M.E., editors. <span class="ref-journal">Essentials of Glycobiology.</span> Cold Spring Harbor: Cold Spring Harbor Laboratory Press; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Essentials+of+Glycobiology&amp;author=A.+Varki&amp;author=J.B.+Lowe&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C2">2. <span class="element-citation">Apweiler R., Hermjakob H., Sharon N. On the frequency of protein glycosylation, as deduced from analysis of the SWISS-PROT database. <span><span class="ref-journal">Biochim. Biophys. Acta. </span>1999;<span class="ref-vol">1473</span>:4–8.</span> [<a href="/pubmed/10580125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=On+the+frequency+of+protein+glycosylation,+as+deduced+from+analysis+of+the+SWISS-PROT+database&amp;author=R.+Apweiler&amp;author=H.+Hermjakob&amp;author=N.+Sharon&amp;volume=1473&amp;publication_year=1999&amp;pages=4-8&amp;pmid=10580125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C3">3. <span class="element-citation">Jones J., Krag S.S., Betenbaugh M.J. Controlling N-linked glycan site occupancy. <span><span class="ref-journal">Biochim. Biophys. Acta. </span>2005;<span class="ref-vol">1726</span>:121–137.</span> [<a href="/pubmed/16126345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=Controlling+N-linked+glycan+site+occupancy&amp;author=J.+Jones&amp;author=S.S.+Krag&amp;author=M.J.+Betenbaugh&amp;volume=1726&amp;publication_year=2005&amp;pages=121-137&amp;pmid=16126345&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C4">4. <span class="element-citation">Marquardt T., Freeze H. Congenital disorders of glycosylation: glycosylation defects in man and biological models for their study. <span><span class="ref-journal">Biol. Chem. </span>2001;<span class="ref-vol">382</span>:161–177.</span> [<a href="/pubmed/11308015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Chem.&amp;title=Congenital+disorders+of+glycosylation:+glycosylation+defects+in+man+and+biological+models+for+their+study&amp;author=T.+Marquardt&amp;author=H.+Freeze&amp;volume=382&amp;publication_year=2001&amp;pages=161-177&amp;pmid=11308015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C5">5. <span class="element-citation">Martensson O., Harlin A., Brandt R., Seppa K., Sillanaukee P. Transferrin isoform distribution: gender and alcohol consumption. <span><span class="ref-journal">Alcohol Clin. Exp. Res. </span>1997;<span class="ref-vol">21</span>:1710–1715.</span> [<a href="/pubmed/9438535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Clin.+Exp.+Res.&amp;title=Transferrin+isoform+distribution:+gender+and+alcohol+consumption&amp;author=O.+Martensson&amp;author=A.+Harlin&amp;author=R.+Brandt&amp;author=K.+Seppa&amp;author=P.+Sillanaukee&amp;volume=21&amp;publication_year=1997&amp;pages=1710-1715&amp;pmid=9438535&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C6">6. <span class="element-citation">Benyamin B., McRae A.F., Zhu G., Gordon S., Henders A.K., Palotie A., Peltonen L., Martin N.G., Montgomery G.W., Whitfield J.B., et al.  Variants in TF and HFE explain approximately 40% of genetic variation in serum-transferrin levels. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">84</span>:60–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668053/">PMC free article</a>]</span> [<a href="/pubmed/19084217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Variants+in+TF+and+HFE+explain+approximately+40%+of+genetic+variation+in+serum-transferrin+levels&amp;author=B.+Benyamin&amp;author=A.F.+McRae&amp;author=G.+Zhu&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=84&amp;publication_year=2009&amp;pages=60-65&amp;pmid=19084217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C7">7. <span class="element-citation">Stanley P., Schachter H., Taniguchi N.  N-Glycans. In: Varki A., Cummings R.D., Esko J.D., Freeze H., Stanley P., Bertozzi C.R., Hart G.W., Etzler M.E., editors. <span class="ref-journal">Essentials of Glycobiology.</span> Cold Spring Harbor: Cold Spring Harbor Laboratory Press; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Essentials+of+Glycobiology&amp;author=P.+Stanley&amp;author=H.+Schachter&amp;author=N.+Taniguchi&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C8">8. <span class="element-citation">Freeze H., Elbein A.D.  Glycosylation precursors. In: Varki A., Cummings R.D., Esko J.D., Freeze H.H., Stanley P., Bertozzi C.R., Hart G.W., Etzler M.E., editors. <span class="ref-journal">Essentials of Glycobiology.</span> Cold Spring Harbor: Cold Spring Harbor Laboratory Press; 2009.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Essentials+of+Glycobiology&amp;author=H.+Freeze&amp;author=A.D.+Elbein&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C9">9. <span class="element-citation">Schadt E.E., Molony C., Chudin E., Hao K., Yang X., Lum P.Y., Kasarskis A., Zhang B., Wang S., Suver C., et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=E.E.+Schadt&amp;author=C.+Molony&amp;author=E.+Chudin&amp;author=K.+Hao&amp;author=X.+Yang&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C10">10. <span class="element-citation">Stibler H., Borg S., Beckman G. Transferrin phenotype and level of carbohydrate-deficient transferrin in healthy individuals. <span><span class="ref-journal">Alcohol Clin. Exp. Res. </span>1988;<span class="ref-vol">12</span>:450–453.</span> [<a href="/pubmed/3044178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Alcohol+Clin.+Exp.+Res.&amp;title=Transferrin+phenotype+and+level+of+carbohydrate-deficient+transferrin+in+healthy+individuals&amp;author=H.+Stibler&amp;author=S.+Borg&amp;author=G.+Beckman&amp;volume=12&amp;publication_year=1988&amp;pages=450-453&amp;pmid=3044178&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C11">11. <span class="element-citation">Helander A., Eriksson G., Stibler H., Jeppsson J.O. Interference of transferrin isoform types with carbohydrate-deficient transferrin quantification in the identification of alcohol abuse. <span><span class="ref-journal">Clin. Chem. </span>2001;<span class="ref-vol">47</span>:1225–1233.</span> [<a href="/pubmed/11427453" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Interference+of+transferrin+isoform+types+with+carbohydrate-deficient+transferrin+quantification+in+the+identification+of+alcohol+abuse&amp;author=A.+Helander&amp;author=G.+Eriksson&amp;author=H.+Stibler&amp;author=J.O.+Jeppsson&amp;volume=47&amp;publication_year=2001&amp;pages=1225-1233&amp;pmid=11427453&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C12">12. <span class="element-citation">Delanghe J.R., Helander A., Wielders J.P., Pekelharing J.M., Roth H.J., Schellenberg F., Born C., Yagmur E., Gentzer W., Althaus H. Development and multicenter evaluation of the N latex CDT direct immunonephelometric assay for serum carbohydrate-deficient transferrin. <span><span class="ref-journal">Clin. Chem. </span>2007;<span class="ref-vol">53</span>:1115–1121.</span> [<a href="/pubmed/17412797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Development+and+multicenter+evaluation+of+the+N+latex+CDT+direct+immunonephelometric+assay+for+serum+carbohydrate-deficient+transferrin&amp;author=J.R.+Delanghe&amp;author=A.+Helander&amp;author=J.P.+Wielders&amp;author=J.M.+Pekelharing&amp;author=H.J.+Roth&amp;volume=53&amp;publication_year=2007&amp;pages=1115-1121&amp;pmid=17412797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C13">13. <span class="element-citation">Firmann M., Mayor V., Marques-Vidal P., Bochud M., Pécoud A., Hayoz D., Paccaud F., Preisig M., Song K.S., Yuan X., et al.  The CoLaus study: a population-based study to investigate the epidemiology and genetic determinants of cardiovascular risk factors and metabolic syndrome. <span><span class="ref-journal">BMC Cardiovasc. Disord. </span>2008;<span class="ref-vol">8</span>:6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2311269/">PMC free article</a>]</span> [<a href="/pubmed/18366642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Cardiovasc.+Disord.&amp;title=The+CoLaus+study:+a+population-based+study+to+investigate+the+epidemiology+and+genetic+determinants+of+cardiovascular+risk+factors+and+metabolic+syndrome&amp;author=M.+Firmann&amp;author=V.+Mayor&amp;author=P.+Marques-Vidal&amp;author=M.+Bochud&amp;author=A.+Pécoud&amp;volume=8&amp;publication_year=2008&amp;pages=6&amp;pmid=18366642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C14">14. <span class="element-citation">Affymetrix. 2006. pp. 1–18. BRLMM: an improved genotype calling method for the GeneChip© Human Mapping 500K array set.</span></div><div class="ref-cit-blk half_rhythm" id="DDR272C15">15. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C16">16. <span class="element-citation">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C17">17. <span class="element-citation">Patterson N., Price A.L., Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Population+structure+and+eigenanalysis&amp;author=N.+Patterson&amp;author=A.L.+Price&amp;author=D.+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C18">18. <span class="element-citation">Whitfield J.B., Fletcher L.M., Murphy T.L., Powell L.W., Halliday J., Heath A.C., Martin N.G. Smoking, obesity, and hypertension alter the dose-response curve and test sensitivity of carbohydrate-deficient transferrin as a marker of alcohol intake. <span><span class="ref-journal">Clin. Chem. </span>1998;<span class="ref-vol">44</span>:2480–2489.</span> [<a href="/pubmed/9836715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Smoking,+obesity,+and+hypertension+alter+the+dose-response+curve+and+test+sensitivity+of+carbohydrate-deficient+transferrin+as+a+marker+of+alcohol+intake&amp;author=J.B.+Whitfield&amp;author=L.M.+Fletcher&amp;author=T.L.+Murphy&amp;author=L.W.+Powell&amp;author=J.+Halliday&amp;volume=44&amp;publication_year=1998&amp;pages=2480-2489&amp;pmid=9836715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C19">19. <span class="element-citation">Whitfield J.B., Dy V., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W. Measuring carbohydrate-deficient transferrin by direct immunoassay: factors affecting diagnostic sensitivity for excessive alcohol intake. <span><span class="ref-journal">Clin. Chem. </span>2008;<span class="ref-vol">54</span>:1158–1165.</span> [<a href="/pubmed/18487284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Measuring+carbohydrate-deficient+transferrin+by+direct+immunoassay:+factors+affecting+diagnostic+sensitivity+for+excessive+alcohol+intake&amp;author=J.B.+Whitfield&amp;author=V.+Dy&amp;author=P.A.+Madden&amp;author=A.C.+Heath&amp;author=N.G.+Martin&amp;volume=54&amp;publication_year=2008&amp;pages=1158-1165&amp;pmid=18487284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C20">20. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C21">21. <span class="element-citation">Li Y., Willer C., Sanna S., Abecasis G. Genotype imputation. <span><span class="ref-journal">Annu. Rev. Genomics Hum. Genet. </span>2009;<span class="ref-vol">10</span>:387–406.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925172/">PMC free article</a>]</span> [<a href="/pubmed/19715440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Genomics+Hum.+Genet.&amp;title=Genotype+imputation&amp;author=Y.+Li&amp;author=C.+Willer&amp;author=S.+Sanna&amp;author=G.+Abecasis&amp;volume=10&amp;publication_year=2009&amp;pages=387-406&amp;pmid=19715440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C22">22. <span class="element-citation">Medland S.E., Nyholt D.R., Painter J.N., McEvoy B.P., McRae A.F., Zhu G., Gordon S.D., Ferreira M.A., Wright M.J., Henders A.K., et al.  Common variants in the trichohyalin gene are associated with straight hair in Europeans. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">85</span>:750–755.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775823/">PMC free article</a>]</span> [<a href="/pubmed/19896111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Common+variants+in+the+trichohyalin+gene+are+associated+with+straight+hair+in+Europeans&amp;author=S.E.+Medland&amp;author=D.R.+Nyholt&amp;author=J.N.+Painter&amp;author=B.P.+McEvoy&amp;author=A.F.+McRae&amp;volume=85&amp;publication_year=2009&amp;pages=750-755&amp;pmid=19896111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C23">23. <span class="element-citation">Ge D., Zhang K., Need A.C., Martin O., Fellay J., Urban T.J., Telenti A., Goldstein D.B. WGAViewer: software for genomic annotation of whole genome association studies. <span><span class="ref-journal">Genome Res. </span>2008;<span class="ref-vol">18</span>:640–643.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2279251/">PMC free article</a>]</span> [<a href="/pubmed/18256235" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=WGAViewer:+software+for+genomic+annotation+of+whole+genome+association+studies&amp;author=D.+Ge&amp;author=K.+Zhang&amp;author=A.C.+Need&amp;author=O.+Martin&amp;author=J.+Fellay&amp;volume=18&amp;publication_year=2008&amp;pages=640-643&amp;pmid=18256235&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDR272C24">24. <span class="element-citation">Balding D.J., Bishop M.J., Cannings C.  <span class="ref-journal">Handbook of Statistical Genetics.</span> Chichester: John Wiley and Sons Ltd; 2007.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Handbook+of+Statistical+Genetics&amp;author=D.J.+Balding&amp;author=M.J.+Bishop&amp;author=C.+Cannings&amp;publication_year=2007&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3159549&amp;issue-id=199512&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140130779621152title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">FUNDING</a></li><li><a href="#__sec2title">Supplementary Material</a></li><li><a href="#idm140130744151648title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140130744149600title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3159549/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3159549/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3159549/pdf/ddr272.pdf">PDF (182K)</a></li> | <li><a href="#" data-citationid="PMC3159549" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3159549%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3159549%2F&amp;text=Genome-wide%20association%20study%20identifies%20two%20loci%20strongly%20affecting%20transferrin%20glycosylation" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3159549%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21665994" data-aiid="3159549" data-aid="3159549" data-iid="199512" data-domainid="740" data-domain="hmg" data-accid="PMC3159549" data-md5="a90290530df273d0ad6bb2a912f8e9ae"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:11:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B3DB0DD51ED710000000009A30625&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3159549%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3159549/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>