<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3024041/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3024041/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity"><meta name="citation_authors" content="Jian Gu, Yuanqing Ye, Margaret R. Spitz, Jie Lin, Lambertus A. Kiemeney, Jingliang Xing, Michelle A. T. Hildebrandt, Waun Ki Hong, Christopher I. Amos, Xifeng Wu"><meta name="citation_date" content="15 February 2011"><meta name="citation_issue" content="4"><meta name="citation_volume" content="20"><meta name="citation_firstpage" content="820"><meta name="citation_doi" content="10.1093/hmg/ddq509"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3024041/?report=abstract"><meta name="citation_pmid" content="21106707"><meta name="DC.Title" content="A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Jian Gu"><meta name="DC.Contributor" content="Yuanqing Ye"><meta name="DC.Contributor" content="Margaret R. Spitz"><meta name="DC.Contributor" content="Jie Lin"><meta name="DC.Contributor" content="Lambertus A. Kiemeney"><meta name="DC.Contributor" content="Jingliang Xing"><meta name="DC.Contributor" content="Michelle A. T. Hildebrandt"><meta name="DC.Contributor" content="Waun Ki Hong"><meta name="DC.Contributor" content="Christopher I. Amos"><meta name="DC.Contributor" content="Xifeng Wu"><meta name="DC.Date" content="2011 Feb 15"><meta name="DC.Identifier" content="10.1093/hmg/ddq509"><meta name="DC.Language" content="en"><meta property="og:title" content="A genetic variant near the PMAIP1/Noxa gene is associated with increased bleomycin sensitivity"><meta property="og:type" content="article"><meta property="og:description" content="Mutagen sensitivity, a measurement of chromatid breaks induced by various mutagens in short-term cultures of peripheral blood lymphocytes, is an established risk factor for a number of cancers and is highly heritable. The purpose of this study is to identify ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3024041/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3024041/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3024041/pdf/ddq509.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE885992DD520C610000000000200014.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3024041%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gquery">All Databases</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC3024041</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2011 Feb 15; 20(4): 820–826. </span></div><div><span class="fm-vol-iss-date">Published online 2010 Nov 24. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddq509" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddq509</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3024041</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21106707">21106707</a></div></div></div></div><h1 class="content-title">A genetic variant near the <em>PMAIP1/Noxa</em> gene is associated with increased bleomycin sensitivity</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714197365024" co-class="co-affbox">Jian Gu</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Ye%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714197225888" co-class="co-affbox">Yuanqing Ye</a>,<sup>1,</sup><sup>†</sup> <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714188444592" co-class="co-affbox">Margaret R. Spitz</a>,<sup>1</sup> <a href="/pubmed/?term=Lin%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714196965440" co-class="co-affbox">Jie Lin</a>,<sup>1</sup> <a href="/pubmed/?term=Kiemeney%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714194954416" co-class="co-affbox">Lambertus A. Kiemeney</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Xing%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714187236752" co-class="co-affbox">Jingliang Xing</a>,<sup>1</sup> <a href="/pubmed/?term=Hildebrandt%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714197224544" co-class="co-affbox">Michelle A. T. Hildebrandt</a>,<sup>1</sup> <a href="/pubmed/?term=Ki%20Hong%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714186270064" co-class="co-affbox">Waun Ki Hong</a>,<sup>2</sup> <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714186267840" co-class="co-affbox">Christopher I. Amos</a>,<sup>1</sup> and  <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707" class="affpopup" co-rid="_co_idm139714184559264" co-class="co-affbox">Xifeng Wu</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139714197365024"><h3 class="no_margin">Jian Gu</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Gu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Jian Gu</a></div></div><div id="_co_idm139714197225888"><h3 class="no_margin">Yuanqing Ye</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Ye%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Yuanqing Ye</a></div></div><div id="_co_idm139714188444592"><h3 class="no_margin">Margaret R. Spitz</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Spitz%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Margaret R. Spitz</a></div></div><div id="_co_idm139714196965440"><h3 class="no_margin">Jie Lin</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Lin%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Jie Lin</a></div></div><div id="_co_idm139714194954416"><h3 class="no_margin">Lambertus A. Kiemeney</h3><p><sup>3</sup>Department of Epidemiology, Biostatistics and Health Technology Assessment,  and</p><p><sup>4</sup>Department of Urology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Kiemeney%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Lambertus A. Kiemeney</a></div></div><div id="_co_idm139714187236752"><h3 class="no_margin">Jingliang Xing</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Xing%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Jingliang Xing</a></div></div><div id="_co_idm139714197224544"><h3 class="no_margin">Michelle A. T. Hildebrandt</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Hildebrandt%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Michelle A. T. Hildebrandt</a></div></div><div id="_co_idm139714186270064"><h3 class="no_margin">Waun Ki Hong</h3><p><sup>2</sup>Department of Thoracic/Head and Neck Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA,</p><div>Find articles by <a href="/pubmed/?term=Ki%20Hong%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Waun Ki Hong</a></div></div><div id="_co_idm139714186267840"><h3 class="no_margin">Christopher I. Amos</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Amos%20CI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Christopher I. Amos</a></div></div><div id="_co_idm139714184559264"><h3 class="no_margin">Xifeng Wu</h3><p><sup>1</sup>Department of Epidemiology,  and</p><div>Find articles by <a href="/pubmed/?term=Wu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21106707">Xifeng Wu</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139714188094368_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139714188094368_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139714188094368_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139714188094368_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Epidemiology,  and</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Thoracic/Head and Neck Oncology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Epidemiology, Biostatistics and Health Technology Assessment,  and</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Urology, Radboud University Nijmegen Medical Centre, Nijmegen, The Netherlands</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Epidemiology, Unit 1340, The University of Texas MD Anderson Cancer Center, 1155 Pressler Blvd., Houston, TX 77030, USA. Tel: +1 7137452485; Fax: +1 7137924657; Email: <a href="mailto:dev@null" data-email="gro.nosrednadm@uwx" class="oemail">gro.nosrednadm@uwx</a></div><div id="AN1"><sup>†</sup>J.G. and Y.Y. contributed equally to this work.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139714188094368_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Jul 27; Revised 2010 Oct 12; Accepted 2010 Nov 18.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139714188094368_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2010. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3024041/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
               Supplementary Data
            </div>
            
            <div><a href="/pmc/articles/PMC3024041/bin/supp_20_4_820__index.html" data-ga-action="click_feat_suppl">supp_20_4_820__index.html</a><span style="color:gray"> (961 bytes)</span></div><div class="small guid">GUID:&nbsp;C64CCC3E-1977-46F2-9C34-7ACCB7170E33</div>
            <div><a href="/pmc/articles/PMC3024041/bin/supp_ddq509_ddq509supp.pdf" data-ga-action="click_feat_suppl">supp_ddq509_ddq509supp.pdf</a><span style="color:gray"> (35K)</span></div><div class="small guid">GUID:&nbsp;583E6AA3-31D0-4F30-A173-E83C5D63F829</div>
         </div></dd></dl></div><div id="idm139714191970832" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139714191970832title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Mutagen sensitivity, a measurement of chromatid breaks induced by various mutagens in short-term cultures of peripheral blood lymphocytes, is an established risk factor for a number of cancers and is highly heritable. The purpose of this study is to identify genetic predictors of mutagen sensitivity. Therefore, we conducted a multi-stage genome-wide association study. The primary scan analyzed 539 437 autosomal SNPs in 673 healthy individuals, followed by validations in two independent sets of 575 and 259 healthy individuals, respectively. One SNP, rs8093763, on chromosome 18q21 showed significant association with bleomycin (BLM) sensitivity (combined <em>P</em> = 2.64 × 10<sup>−8</sup>). We observed significantly lower BLM-induced chromotid breaks for genotypes containing wild-type allele compared with the homozygous variant genotype in the discovery set (0.71 versus 0.90, <em>P</em>= 3.77 × 10<sup>−5</sup>) and in replication phase 1 (0.61 versus 0.84, <em>P</em>= 7.00 × 10<sup>−5</sup>). The result of replication phase 2 was not statistically significant (0.65 versus 0.68, <em>P</em>= 0.44). This SNP is approximately 64 kb from <em>PMAIP1/Noxa</em>, which is a radiation-inducible gene and exhibits higher expression in BLM-sensitive lymphoblastoid cell lines than insensitive cell lines upon BLM treatment. In conclusion, we identified a biologically plausible genetic variant on 18q21 near the <em>PMAIP1/Noxa</em> gene that is associated with BLM sensitivity.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Mutagen sensitivity, a measurement of chromatid breaks induced by various mutagens in short-term cultures of peripheral blood lymphocytes, is used as an indicator of DNA repair capacity (<a href="#DDQ509C1" rid="DDQ509C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176019">1</a>–<a href="#DDQ509C3" rid="DDQ509C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176009">3</a>). Numerous retrospective and prospective epidemiologic studies have shown that higher mutagen sensitivity predisposes individuals to a variety of epithelial cancers (<a href="#DDQ509C4" rid="DDQ509C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176031">4</a>–<a href="#DDQ509C9" rid="DDQ509C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208175999">9</a>). Furthermore, compelling evidence suggests that mutagen sensitivity is an inherited phenotype. Family studies have shown that the first-degree relatives of mutagen-sensitive individuals have a significantly higher rate of the mutagen-sensitive phenotype than relatives of individuals with normal sensitivity (<a href="#DDQ509C10" rid="DDQ509C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176012">10</a>). Bleomycin (BLM), a radiomimetic chemotherapeutic drug, was the first applied challenge mutagen (<a href="#DDQ509C1" rid="DDQ509C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176014">1</a>–<a href="#DDQ509C3" rid="DDQ509C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176016">3</a>). BLM induces oxidative base damage and DNA-strand breaks that require two major DNA repair pathways, base excision repair and double-strand break repair. Three independent twin studies that compared similarities between monozygotic twins and dizygotic twins demonstrated that BLM sensitivity is a genetic trait with heritability estimates of 41, 58 and 75%, respectively (<a href="#DDQ509C11" rid="DDQ509C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208175997">11</a>,<a href="#DDQ509C12" rid="DDQ509C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176006">12</a>), indicating high genetic heritability for this intermediate cancer phenotype. However, the genetic factors that determine BLM sensitivity remain unknown. Cloos <em>et al</em>. (<a href="#DDQ509C13" rid="DDQ509C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176002">13</a>) performed a microarray analysis in lymphoblastoid cells from BLM sensitive and insensitive individuals to search for genes and pathways responsible for BLM sensitivity and observed differential expression of genes in multiple biological pathways, but no specific pathway could be singled out to explain the difference in BLM sensitivity. These results suggest that there may not be a single major gene or a major pathway determining BLM sensitivity. More likely, and similar to genetic susceptibility to common cancers, there are a number of low-penetrance genes that determine BLM sensitivity.</p><p id="__p4" class="p p-last">Genome-wide association study (GWAS) have successfully identified genetic determinants for intermediate biomarkers, such as plasma levels of lipoprotein and vitamin B12 (<a href="#DDQ509C14" rid="DDQ509C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176004">14</a>,<a href="#DDQ509C15" rid="DDQ509C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176005">15</a>). However, to our knowledge, there have not been any reports of GWAS for intermediate cancer susceptibility biomarkers. In this study, we conducted a multi-stage GWAS to identify genetic variations associated with BLM sensitivity in healthy individuals of European ancestry.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p5" class="p p-first">Table&nbsp;<a href="/pmc/articles/PMC3024041/table/DDQ509TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ509TB1" rid-ob="ob-DDQ509TB1" co-legend-rid=""><span>1</span></a> shows the selected characteristics of the study participants in each phase.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ509TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p6">Characteristics of the study populations</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139714229713904" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Discovery</th><th rowspan="1" colspan="1" align="left">Replication 1</th><th rowspan="1" colspan="1" align="left">Replication 2</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Number of individuals<sup>a</sup></td><td rowspan="1" colspan="1">673</td><td rowspan="1" colspan="1">575</td><td rowspan="1" colspan="1">259</td></tr><tr><td rowspan="1" colspan="1">Age, median(range)</td><td rowspan="1" colspan="1">64 (24–90)</td><td rowspan="1" colspan="1">62 (31–86)</td><td rowspan="1" colspan="1">61 (33–86)</td></tr><tr><td rowspan="1" colspan="1">Packyr, median (range)</td><td rowspan="1" colspan="1">22.5 (0.05–165)</td><td rowspan="1" colspan="1">37 (0.05–184)</td><td rowspan="1" colspan="1">40 (0.6–189)</td></tr><tr><td colspan="4" rowspan="1">Gender, <em>n</em> (%)</td></tr><tr><td rowspan="1" colspan="1"> Male</td><td rowspan="1" colspan="1">519 (77)</td><td rowspan="1" colspan="1">317 (55)</td><td rowspan="1" colspan="1">96 (37)</td></tr><tr><td rowspan="1" colspan="1"> Female</td><td rowspan="1" colspan="1">154 (23)</td><td rowspan="1" colspan="1">258 (45)</td><td rowspan="1" colspan="1">163 (63)</td></tr><tr><td colspan="4" rowspan="1">Smoking status, <em>n</em> (%)</td></tr><tr><td rowspan="1" colspan="1"> Never</td><td rowspan="1" colspan="1">298 (44)</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">152 (59)</td></tr><tr><td rowspan="1" colspan="1"> Former</td><td rowspan="1" colspan="1">325 (48)</td><td rowspan="1" colspan="1">347 (60)</td><td rowspan="1" colspan="1">69 (27)</td></tr><tr><td rowspan="1" colspan="1"> Current</td><td rowspan="1" colspan="1">50 (7)</td><td rowspan="1" colspan="1">228 (40)</td><td rowspan="1" colspan="1">38 (15)</td></tr></tbody></table></div><div id="largeobj_idm139714229713904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3024041/table/DDQ509TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139714186273856"><p id="__p7" class="p p-first-last"><sup>a</sup>All study participants were of European ancestry.</p></div></div></div><p id="__p8">In the discovery phase, we tested the association of 539 437 SNPs that passed strict quality control measures in 673 healthy participants. These individuals were from the control population in a recently published GWAS of bladder cancer risk for whom the BLM sensitivity measurement was available (<a href="#DDQ509C16" rid="DDQ509C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_257934534">16</a>). The quantile–quantile plot for the multiplicity adjusted <em>P-</em>values showed no evidence for a systematic inflation of <em>P</em>-values (genomic inflation factor λ = 0.969) (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq509/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). We further determined the potential effect of population substructure by using two approaches: one including further adjustment for strata identified by PLINK and the other including further adjustment for residual population structure using the top four principal components by EIGENSTRAT. These two approaches showed similar strength of associations with no evidence for the inflation of the multiplicity adjusted <em>P-</em>values (λ = 0.969 for PLINK and λ = 0.968 for EIGENSTRAT, <a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq509/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>), indicating that the effect of population substructure was negligible.</p><p id="__p9">In the first replication phase, we selected SNPs with multiplicity adjusted <em>P</em> &lt; 10<sup>−3</sup> level in the discovery phase to be validated in an independent set of 575 individuals who had BLM sensitivity data from the control population of a recent GWAS (using Illumina's Humanhap300 chip) of lung cancer risk (<a href="#DDQ509C17" rid="DDQ509C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176020">17</a>). A total of 554 SNPs from discovery set were significantly associated with BLM sensitivity satisfying these criteria (Fig.&nbsp;<a href="/pmc/articles/PMC3024041/figure/DDQ509F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ509F1" rid-ob="ob-DDQ509F1" co-legend-rid="lgnd_DDQ509F1"><span>1</span></a>). Among these 554 SNPs, 287 were covered by the HumanHap300 chip, 245 were imputed from HapMap release 22, build 36, and 22 SNPs were not HapMap SNPs and hence not included in the analysis. All the imputed SNPs showed high quality with an average predicted <em>r</em><sup>2</sup> of 0.93 between imputed genotypes and the true underlying genotypes.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ509F1" co-legend-rid="lgnd_DDQ509F1"><a href="/pmc/articles/PMC3024041/figure/DDQ509F1/" target="figure" rid-figpopup="DDQ509F1" rid-ob="ob-DDQ509F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139714269162912" class="figure"><a href="/pmc/articles/PMC3024041/figure/DDQ509F1/" target="figure" rid-figpopup="DDQ509F1" rid-ob="ob-DDQ509F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3024041_ddq50901.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq50901.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3024041/bin/ddq50901.jpg"></a></div><div id="largeobj_idm139714269162912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3024041/figure/DDQ509F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ509F1"><div><a class="figpopup" href="/pmc/articles/PMC3024041/figure/DDQ509F1/" target="figure" rid-figpopup="DDQ509F1" rid-ob="ob-DDQ509F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p10">Manhattan plot presenting genome-wide association results for 539 437 autosomal SNPs that passed the strict quality control measures and had &gt;20 subjects carrying at least one variant allele.</p></div></div></div><p id="__p11">We then performed a second validation using TaqMan assay in 259 healthy individuals from additional controls of the ongoing bladder and lung case control studies for the 13 SNPs that were significantly associated with BLM sensitivity at <em>P</em> &lt; 0.05 in the first replication phase (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq509/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplement Material, Table S1</a>). There was no significant heterogeneity between the discovery and first replication phase (<em>P</em> for heterogeneity &gt; 0.1) for these 13 SNPs. Since SNP rs10093667 failed the TaqMan design, only 12 SNPs were genotyped in this second replication phase. Out of the 12 SNPs, 4 SNPs were imputed in the first replication phase. Therefore, we also performed direct genotyping using TaqMan assays for these four SNPs in the first replication phase (<a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq509/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplement Material, Table S2</a>).</p><p id="__p12">Overall, SNP rs8093763 showed the strongest evidence of association with BLM sensitivity in a recessive model (combined <em>P</em> = 2.64 × 10<sup>−8</sup>, Table&nbsp;<a href="/pmc/articles/PMC3024041/table/DDQ509TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ509TB2" rid-ob="ob-DDQ509TB2" co-legend-rid=""><span>2</span></a>). We observed significantly lower BLM-induced chromatid breaks (BICB) for genotypes containing wild-type allele compared with the homozygous variant genotype in the discovery set (0.71 versus 0.90, <em>P</em> = 3.77 × 10<sup>−5</sup>) and in the first replication phase (0.61 versus 0.84, <em>P</em> = 7.00 × 10<sup>−5</sup>). The result of replication phase 2 was not statistically significant (0.65 versus 0.68, <em>P</em> = 0.44). There was no significant evidence for heterogeneity among the three phases (<em>P</em> for heterogeneity = 0.23). The second strongest association was observed for rs708547 on chromosome 4q12 using the recessive model (combined <em>P</em> = 8.79 × 10<sup>−7</sup>, <em>P</em> for heterogeneity = 0.93, Table&nbsp;<a href="/pmc/articles/PMC3024041/table/DDQ509TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ509TB2" rid-ob="ob-DDQ509TB2" co-legend-rid=""><span>2</span></a>). The genotypes with at least one wild-type allele exhibited consistently higher BICB than the homozygous variant genotype in the discovery set (0.74 versus 0.56, <em>P</em> = 2.94 × 10<sup>−4</sup>), in replication phase 1 (0.63 versus 0.47, <em>P</em> = 1.12 × 10<sup>−2</sup>) and in replication phase 2 (0.66 versus 0.51, <em>P</em> = 3.29 × 10<sup>−2</sup>). We also observed a consistent association for SNP rs4662834 on chromosome 2q21 using the dominant model (combined <em>P</em> = 4.89 × 10<sup>−6</sup>, <em>P</em> for heterogeneity = 0.72, Table&nbsp;<a href="/pmc/articles/PMC3024041/table/DDQ509TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDQ509TB2" rid-ob="ob-DDQ509TB2" co-legend-rid=""><span>2</span></a>). The common homozygous genotype had higher BICB than variant allele carrying genotypes in the discovery set (0.79 versus 0.70, <em>P</em> = 1.54 × 10<sup>−4</sup>), in replication phase 1 (0.67 versus 0.59, <em>P</em> = 2.37 × 10<sup>−2</sup>) and in replication phase 2 (0.69 versus 0.63, <em>P</em> = 1.68 × 10<sup>−1</sup>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDQ509TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p13">SNPs with the strongest associations with BLM sensitivity in combined analysis of three phases</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139714195686768" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th colspan="3" rowspan="1" align="left">rs8093763 (G/A)<hr></th><th colspan="3" rowspan="1" align="left">rs708547 (A/G)<hr></th><th colspan="3" rowspan="1" align="left">rs4662834 (G/A)<hr></th></tr><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">MAF<sup>a</sup></th><th rowspan="1" colspan="1" align="left">Effect size (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Effect size (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">MAF</th><th rowspan="1" colspan="1" align="left">Effect size (95% CI)</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Model</td><td rowspan="1" colspan="1"></td><td colspan="2" rowspan="1">REC</td><td rowspan="1" colspan="1"></td><td colspan="2" rowspan="1">REC</td><td rowspan="1" colspan="1"></td><td colspan="2" rowspan="1">DOM</td></tr><tr><td rowspan="1" colspan="1">Discovery<sup>b</sup></td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">&nbsp;0.17 (0.09, 0.26)</td><td rowspan="1" colspan="1">3.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">−0.19 (−0.29, −0.09)</td><td rowspan="1" colspan="1">2.94 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">−0.09 (−0.14, −0.04)</td><td rowspan="1" colspan="1">1.54 × 10<sup>−4</sup></td></tr><tr><td colspan="10" rowspan="1">Replication</td></tr><tr><td rowspan="1" colspan="1"><strong> </strong>phase 1<sup>b</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.25 (0.13, 0.37)</td><td rowspan="1" colspan="1">7.00 × 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">−0.16 (−0.28, −0.04)</td><td rowspan="1" colspan="1">1.12 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">0.40</td><td rowspan="1" colspan="1">−0.06 (−0.12, −0.01)</td><td rowspan="1" colspan="1">2.37 × 10<sup>−2</sup></td></tr><tr><td rowspan="1" colspan="1"><strong> </strong>phase 2<sup>b</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.07 (−0.10, 0.24)</td><td rowspan="1" colspan="1">4.40 × 10<sup>−1</sup></td><td rowspan="1" colspan="1">0.27</td><td rowspan="1" colspan="1">−0.17 (−0.33, −0.01)</td><td rowspan="1" colspan="1">3.29 × 10<sup>−2</sup></td><td rowspan="1" colspan="1">0.43</td><td rowspan="1" colspan="1">−0.06 (−0.14, 0.02)</td><td rowspan="1" colspan="1">1.68 × 10<sup>−1</sup></td></tr><tr><td rowspan="1" colspan="1">Combined replication<sup>c</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">&nbsp;0.19 (0.09, 0.29)</td><td rowspan="1" colspan="1">2.05 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">−0.16 (−0.26, −0.07)</td><td rowspan="1" colspan="1">8.80 × 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">−0.06 (−0.11, −0.02)</td><td rowspan="1" colspan="1">7.95 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1">All studies<sup>c</sup></td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">&nbsp;0.18 (0.12, 0.24)</td><td rowspan="1" colspan="1">2.64 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">−0.18 (−0.24, −0.11)</td><td rowspan="1" colspan="1">8.79 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.41</td><td rowspan="1" colspan="1">−0.08 (−0.11, −0.04)</td><td rowspan="1" colspan="1">4.89 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1"><em>P</em> for heterogeneity</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.235</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.927</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">0.716</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm139714195686768" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3024041/table/DDQ509TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139714197385616"><p id="__p14" class="p p-first-last"><sup>a</sup>MAF, minor allele frequency. The minor alleles were ‘A’ for rs8093763 and rs4662834, and ‘G’ for rs70854.</p></div><div id="idm139714197384576"><p id="__p15" class="p p-first-last"><sup>b</sup>Effect size calculated by linear regression analysis while adjusting for age, gender, pack-year, slide reader and blood collection year.</p></div><div id="idm139714197383536"><p id="__p16" class="p p-first-last"><sup>c</sup>Effect size estimated by fixed effect meta-analysis for combined replication study and all studies combined.</p></div></div></div><p id="__p17">Rs8093763 is located on chromosome 18q21. Linkage disequilibrium (LD) analysis of the HapMap SNPs in its vicinity showed that it maps to a region of low LD structure, and no SNPs in the HapMap database are in strong LD (<em>r</em><sup>2</sup>&gt; 0.6) with rs8093763. We performed imputations for SNPs in the HapMap database within 1Mb of rs8093763 and computed the combined <em>P</em>-value for the discovery set and replication phase 1 (Fig.&nbsp;<a href="/pmc/articles/PMC3024041/figure/DDQ509F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ509F2" rid-ob="ob-DDQ509F2" co-legend-rid="lgnd_DDQ509F2"><span>2</span></a>). Rs8093763 remained the most significant SNP in the region.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDQ509F2" co-legend-rid="lgnd_DDQ509F2"><a href="/pmc/articles/PMC3024041/figure/DDQ509F2/" target="figure" rid-figpopup="DDQ509F2" rid-ob="ob-DDQ509F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139714190465456" class="figure"><a href="/pmc/articles/PMC3024041/figure/DDQ509F2/" target="figure" rid-figpopup="DDQ509F2" rid-ob="ob-DDQ509F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3024041_ddq50902.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddq50902.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3024041/bin/ddq50902.jpg"></a></div><div id="largeobj_idm139714190465456" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3024041/figure/DDQ509F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDQ509F2"><div><a class="figpopup" href="/pmc/articles/PMC3024041/figure/DDQ509F2/" target="figure" rid-figpopup="DDQ509F2" rid-ob="ob-DDQ509F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p18">The locus of 18q21 encompassing rs8093763. (<strong>A</strong>) Results of SNP association from combined discovery set and replication phase 1: observed results from genotyped SNPs are in red and imputed results are in blue. Screen shot of all known genes in this region are also obtained from UCSU genome browser. (<strong>B</strong>) LD structure of this region. Shown in each box is the square of the correlation coefficient (<em>r</em><sup>2</sup>) derived from Haploview software (v 4.1). Darker shading box indicates greater extent of LD between two SNPs.</p></div></div></div><p id="__p19" class="p p-last">There are a few genes in this region, the nearest being <em>PMAIP1/Noxa</em>, about 64 kb from rs8093763. To support the biological plausibility of our observation, we queried the Gene Expression Database (GEO; <a href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) for data sets obtained from BLM-treated samples. We identified one data set (GDS1714) that compared gene expression differences of 14 lymphoblastoid cell lines (7 BLM-sensitive and 7 BLM-insensitive based on mutagen sensitivity assays) at baseline and after a 4 h exposure to BLM (<a href="#DDQ509C13" rid="DDQ509C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176025">13</a>). The mean expression of <em>PMAIP1/Noxa</em> was higher in BLM sensitive (mean ± SD: 0.99 ± 0.58) than in insensitive cell lines (0.64 ± 0.60) at baseline, although the difference was not statistically significant (<em>P</em>= 0.28) possibly due to small sample size. In BLM-insensitive cells, the expression of <em>PMAIP1/Noxa</em> was little changed after BLM treatment (before versus after: 0.64 ± 0.60 versus 0.70 ± 0.41) (<em>P</em>= 0.80); however, in BLM-sensitive cells, the expression of <em>PMAIP1/Noxa</em> was apparently increased (before versus after: 0.99 ± 0.58 versus 1.37 ± 0.59) (<em>P</em>= 0.24), although again the data did not reach statistical significance possibly due to small sample size. When we compared the expression of <em>PMAIP1/Noxa</em> after BLM treatment, BLM-sensitive cells exhibited significantly higher expression (1.37 ± 0.59) than in BLM-insensitive cells (0.70 ± 0.41) (<em>P</em>= 0.03). We also performed same analyses of other genes (Fig.&nbsp;<a href="/pmc/articles/PMC3024041/figure/DDQ509F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDQ509F2" rid-ob="ob-DDQ509F2" co-legend-rid="lgnd_DDQ509F2"><span>2</span></a>) in this region, but none of them showed similar trend as <em>PMAIP1/Noxa</em> (data not shown).</p></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p20" class="p p-first">In this GWAS study, we identified a novel genetic determinant of BLM sensitivity that reached genome-wide significance in combined analysis. More importantly, rs8093763 is located on chromosome 18q21 and its closest gene is <em>PMAIP1/Noxa</em>, about 64 kb apart. <em>PMAIP1/Noxa</em> was originally isolated as a phorbol 12-myristate 13-acetate (PMA) induced protein 1 (PMAIP1) potentially involved in leukemogenesis (<a href="#DDQ509C18" rid="DDQ509C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208175998">18</a>). Its physiologic function was not known until a decade later when Oda <em>et al</em>. (<a href="#DDQ509C19" rid="DDQ509C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208175996">19</a>) discovered a mouse gene that was differentially expressed between γ-irradiated p53-wild-type and mutant mouse embryonic fibroblasts and named it <em>Noxa</em> (Latin for damage). <em>PMAIP1</em> was then revealed as <em>Noxa</em>'s human homologue. PMAIP1/Noxa is a Bcl2 homology domain 3 (BH3)-only protein that plays an important role in apoptosis. Extensive <em>in vitro</em> and <em>in vivo</em> studies have found that <em>PMAIP1/Noxa</em> is induced by multiple stimuli, including DNA damage (e.g. &nbsp;γ-irradiation and chemotherapeutic drugs), hypoxia and mitogenic stimulation (<a href="#DDQ509C20" rid="DDQ509C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_257934535">20</a>). Since BLM is a radiomimetic drug that induces DNA-strand breaks, it is tempting to speculate that <em>PMAIP1/Noxa</em> is also inducible by BLM treatment. Preliminary pilot data from a GEO data set suggest that <em>PMAIP1/Noxa</em> is a BLM-inducible gene in BLM-sensitive cells, but the results did not reach statistical significance and need to be confirmed in larger studies. Another data set (GDS1492) compared the expression of <em>PMAIP1/Noxa in vivo</em> in lung tissues from mouse strains with different susceptibility to BLM-induced pulmonary fibrosis 3 and 6 weeks after BLM treatment (<a href="#DDQ509C21" rid="DDQ509C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176022">21</a>). It appeared that BLM increased the expression of <em>PMAIP1/Noxa</em> in BLM-sensitive strain (C57BL/6J). But the results were difficult to interpret since it was an <em>in vivo</em> mice study and there were only two mice in each strain (one male and one female). Future <em>in vitro</em> and <em>in vivo</em> functional studies of larger sample sizes are needed to confirm the inducibility of <em>PMAIP1/Noxa</em> by BLM. Nevertheless, these results provide biological plausibility that <em>PMAIP1/Noxa</em> is a susceptibility locus for BLM sensitivity. It also remains to be studied whether rs8093763 affects <em>PMAIP1/Noxa</em> expression and whether there are other causal functional SNP(s) in <em>PMAIP1/Noxa</em> that are in LD with rs8093763. Interestingly, the <em>MC4R</em> gene that plays an important role in energy balance and obesity is located ∼2.4Mb from rs8093763. Two recent GWAS identified several genetic variants between <em>PMAIP1/Noxa</em> and <em>MC4R</em> genes that were associated with waist circumference and BMI (<a href="#DDQ509C22" rid="DDQ509C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176018">22</a>,<a href="#DDQ509C23" rid="DDQ509C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176024">23</a>). Those SNPs were not in strong LD with rs8093763 and were not associated with BLM sensitivity (data not shown).</p><p id="__p21">We also identified two other potential susceptibility loci for BLM sensitivity. Rs708547 is located in a region of high LD on chromosome 4q12. The two closest genes are C4orf14 (∼8.2 kb apart) and RE1-silencing transcription (REST) factor (∼22.9 kb apart). C4orf14 encode a putative mammalian mitochondrial nitric oxide synthase that may regulate mitochondrial functions and apoptosis (<a href="#DDQ509C24" rid="DDQ509C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176035">24</a>). REST is a zinc finger transcriptional repressor regulating many genes and playing critical roles in embryonic development and a number of cellular responses in neurons and other cell types (<a href="#DDQ509C25" rid="DDQ509C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176030">25</a>). Of particular interest, REST suppresses expression of <em>Mad2</em>, an essential component of the mitotic checkpoint /spindle-assembly checkpoint. <em>REST</em> aberrations resulted in shortened mitosis, premature sister-chromatid separation, chromosome bridges and other mitosis defects (<a href="#DDQ509C26" rid="DDQ509C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176010">26</a>), providing biological plausibility for this gene in influencing BICB. There were no apparent expression differences of <em>REST</em> between BLM-sensitive and insensitive lymphoblastoid cell lines and before and after BLM treatment in GDS1714 data set (data not shown). Rs4662834 maps to a gene-poor region on chromosome 2q21. The nearest gene is heparan sulfate 6-O-sulfotransferase 1 (<em>HS6ST1</em>), approximately 402 kb from the SNP. Further validation with more samples is warranted to confirm the associations of these two SNPs with BLM sensitivity, identify causal SNPs and genes, and investigate the biological mechanisms underlying the validated associations.</p><p id="__p22">Previous candidate gene studies have suggested correlations between a few potential functional polymorphisms in DNA repair and xenobiotic-metabolizing enzyme genes (e.g. GSTT1, GSTM1, XPD, XRCC1 and XRCC3) with BICB or micronuclei formation, but the sample sizes were small and the results were inconsistent (<a href="#DDQ509C27" rid="DDQ509C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176003">27</a>–<a href="#DDQ509C30" rid="DDQ509C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176015">30</a>). These potential functional polymorphisms were not covered in current SNP arrays; however, there are tagging SNPs from these genes. When we extracted tagging SNPs from 10 kb upstream and 10 kb downstream of these genes, none of the SNPs was among the top hits (<em>P</em>&lt; 10<sup>−3</sup>) in our discovery phase. Many more samples are needed to validate or refute the associations of SNPs in these genes with BLM sensitivity.</p><p id="__p23" class="p p-last">In conclusion, in this multi-stage GWAS, we identified a biologically plausible genetic variant on 18q21 near the <em>PMAIP1/Noxa</em> gene that is associated with BLM sensitivity. We also found two additional potential genetic variants on chromosomes 4q12 and 2q21 that may be associated with BLM sensitivity and warrant further validation.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study population</h3><p id="__p24" class="p p-first-last">All the participants in this study were healthy controls of European ancestry from Kelsey Seybold Clinic, the largest multi-specialty physician group in the Houston metropolitan area. For the discovery phase, participants were enrolled as controls in a case control study of bladder cancer and involved in the primary scanning of a recently published GWAS of bladder cancer (<a href="#DDQ509C16" rid="DDQ509C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_257934533">16</a>). Details of control ascertainment have been described previously (<a href="#DDQ509C16" rid="DDQ509C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_257934532">16</a>). For the first validation phase, participants were enrolled as controls in a case control study of lung cancer and involved in the primary scanning of a recently published GWAS of lung cancer (<a href="#DDQ509C17" rid="DDQ509C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176028">17</a>). The control recruitment of these two case control studies used the same infrastructure and procedure. For the second validation phase, participants were additional recent controls from these two ongoing case control studies who were not part of the published GWAS. After quality control, mutagen sensitivity data and complete demographic information were available for 673 individuals in the discovery set, 575 individuals in replication phase 1 and 259 individuals in replication phase 2. Informed consent was obtained from all study participants before the collection of epidemiological data and blood samples by trained MD Anderson staff interviewers.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping</h3><p id="__p25" class="p p-first-last">Genotyping for the discovery set was performed using the Illumina HumanHap610 chip. We retained genotype data for 542 879 autosomal SNPs from 957 subjects that passed strict quality control measures for the SNPs and subjects (<a href="#DDQ509C16" rid="DDQ509C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_257934536">16</a>). Genotyping for the replication phase 1 series was obtained from GWAS of lung cancer risk using Illumina's HumanHap300 BeadChip (<a href="#DDQ509C17" rid="DDQ509C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176032">17</a>). We retained 303 669 autosomal SNPs after performing quality control measures for 1137 subjects using PLINK to exclude SNPs with call rate &lt;95%, with minor allele frequency &lt; 0.01, and with deviations from Hardy–Weinberg equilibrium (<em>P</em> &lt; 0.0001). We removed 64 subjects that were either duplicated samples of the discovery set or genetically related samples with those in the discovery set identified by PLINK software (<a href="#DDQ509C31" rid="DDQ509C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176007">31</a>). The resulting replication phase 1 contained 1073 subjects. We used TaqMan assays to perform genotyping for replication phase 2 using the 7900HT sequence detection system (Applied BioSystems, Foster City, CA, USA). We also performed direct genotyping of the four imputed SNPs in the first replication phase that showed significant association with BLM sensitivity.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Genotype imputation</h3><p id="__p26" class="p p-first-last">Genotyping for the discovery phase and the first replication phase used two different genotyping platforms (HumanHap610 and HumanHap300, respectively). Nearly half of the top SNPs (<em>P</em> &lt; 10<sup>−3</sup>) to be validated from the discovery phase were not covered by the HumanHap300 in the first validation phase. We imputed genotypes for SNPs that were genotyped in the HapMap project but were not on the HumanHap300 chip. The imputation was done using a Hidden Markov Model programmed in MACH (<a href="#DDQ509C32" rid="DDQ509C32" class=" bibr popnode">32</a>). Phased haplotypes from HapMap CEU samples (HapMap release 22, build 36) were provided as the reference haplotypes to impute the unobserved genotypes based on the local haplotype information.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Mutagen sensitivity assay</h3><p id="__p27" class="p p-first-last">The standard procedure for the mutagen sensitivity assay using BLM as a challenge mutagen has been optimized and kept constant across different laboratories for over two decades (<a href="#DDQ509C1" rid="DDQ509C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176026">1</a>–<a href="#DDQ509C4" rid="DDQ509C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176013">4</a>,<a href="#DDQ509C33" rid="DDQ509C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176029">33</a>). Briefly, 1 ml of each fresh whole blood sample was added to 9 ml of RPMI 1640 supplemented with 20% fetal bovine serum and 1% (volume for volume) phytohemagglutinin. After incubation at 37°C for 72 h, BLM was added to each culture for 5 h with a final concentration of 0.03 U/ml. During the last hour of BLM treatment, 0.04 µg/ml colcemid was added to the cells to arrest them in mitosis. Then the cells were harvested and stained with Giesma. The number of BICB was counted under microscope in 50 metaphases per sample and was recorded as the mean number of breaks per cell. The assays are set up on a daily basis when the fresh blood sample is delivered to the lab and the readings have been highly consistent. A previous multicenter study (<a href="#DDQ509C4" rid="DDQ509C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176011">4</a>) compared the mean number of BICB per cell between two major US institutions and one European institution under the above assay conditions and found that the readings were highly consistent across different institutions.</p></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Statistical analysis</h3><p id="__p28" class="p p-first">BICB was analyzed as a continuous variable. In the primary scan, we used linear regression when adjusting for age, gender, pack-years of smoking, slide reader and blood collection year to compare the BICB by genotype of each autosomal SNP. The statistical significance of each SNP was determined by the lower <em>P</em>-value of the dominant model (comparing the BICB between common homozygous genotypes and heterozygous/rare homozygous genotypes) and the recessive model (comparing the BICB between common homozygous/heterozygous genotypes and rare homozygous genotypes). We chose the combination of dominant and recessive models since this yields optimal power to detect disease susceptibility loci under all of the three inheritance models, i.e. additive, dominant, or recessive, for quantitative traits (<a href="#DDQ509C34" rid="DDQ509C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176008">34</a>). The genome-wide association was computed for the 539 437 autosomal SNPs with at least 20 subjects carrying one variant allele. If the number of homozygous variant genotype was ≤ 20, we only computed results under the dominant model. For SNPs tested for both the dominant and recessive models, the multiplicity adjusted <em>P</em>-value was computed by the Šidák procedure as 1−(1−<em>P</em><sub>min</sub>)<sup>2</sup>. For SNPs tested only for the dominant model, the multiplicity adjusted <em>P</em>-value was the <em>P</em>-value for the dominant model. For replication phases 1 and 2, we performed linear regression while adjusting for age, gender, pack-year, slide reader and blood collection year using the most significant tests identified in the discovery set. To summarize results for the discovery set and the two replication sets, we performed meta-analysis for the most significant tests identified in the discovery set. All analyses were done with STATA software (version 10.1, STATA Corporation). Data manipulations were done using the R software package.</p><p id="__p29" class="p p-last">To assess the effect of population substructure, we used linear regression to perform two separate analyses for the discovery data, replication phase 1 and pooled data: one including strata identified by PLINK as covariates; the other including four eigenvectors from EIGENSTRAT (<a href="#DDQ509C35" rid="DDQ509C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176000">35</a>). PLINK clusters individuals into homogeneous groups using genetic similarity based on pairwise identity-by-state distance. EIGENSTRAT analysis uses a principle component method to detect subpopulation. We used the eigenvectors associated with the top four eigenvalues determined by the scree plot of all the available eigenvalues in descending order obtained from the principle component analysis. The LD structure of the neighboring region containing loci associated with BICB was inferred by Haploview software (v4.1) (<a href="#DDQ509C36" rid="DDQ509C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176017">36</a>). The screen shot of all the known genes in the region were obtained from UCSC genome browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>) (<a href="#DDQ509C37" rid="DDQ509C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_208176023">37</a>).</p></div></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p31" class="p p-first-last">This work was supported by the National Institutes of Health (<span>CA127615</span>; and <span>CA74880</span>; to X.W., <span>CA133996</span>; to C.I.A. and <span>CA055769</span> to M.R.S.).</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><!--/article/body/sec/--><p id="__p30" class="p p-first"><a href="http://hmg.oxfordjournals.org/cgi/content/full/ddq509/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p><div class="sec suppmat" id="PMC_1">
            
               <!--caption a9--><strong>Supplementary Data:</strong>
            
            <div class="sup-box half_rhythm" id="idm139714190302832"><a href="/pmc/articles/PMC3024041/bin/supp_20_4_820__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>
            
         </div></div><div id="idm139714191709552" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139714191709552title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p32">We thank the genotyping personnel, study coordinators and interviewers for performing experiments and recruiting participants. We are especially thankful for all the study participants, who make population based research possible.</p><p id="__p33"><em>Conflict of Interest statement.</em> The authors declare no conflict of interest.</p></div></div><div id="idm139714191707680" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139714191707680title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDQ509C1">1. <span class="element-citation">Hsu T.C. Genetic predisposition to cancer with special reference to mutagen sensitivity. <span><span class="ref-journal">In Vitro Cell Dev. Biol. </span>1987;<span class="ref-vol">23</span>:591–603.</span> [<a href="/pubmed/3308827" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=In+Vitro+Cell+Dev.+Biol.&amp;title=Genetic+predisposition+to+cancer+with+special+reference+to+mutagen+sensitivity&amp;author=T.C.+Hsu&amp;volume=23&amp;publication_year=1987&amp;pages=591-603&amp;pmid=3308827&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C2">2. <span class="element-citation">Hsu T.C., Johnston D.A., Cherry L.M., Ramkissoon D., Schantz S.P., Jessup J.M., Winn R.J., Shirley L., Furlong C. Sensitivity to genotoxic effects of bleomycin in humans: possible relationship to environmental carcinogenesis. <span><span class="ref-journal">Int. J. Cancer. </span>1989;<span class="ref-vol">43</span>:403–409.</span> [<a href="/pubmed/2466800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Sensitivity+to+genotoxic+effects+of+bleomycin+in+humans:+possible+relationship+to+environmental+carcinogenesis&amp;author=T.C.+Hsu&amp;author=D.A.+Johnston&amp;author=L.M.+Cherry&amp;author=D.+Ramkissoon&amp;author=S.P.+Schantz&amp;volume=43&amp;publication_year=1989&amp;pages=403-409&amp;pmid=2466800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C3">3. <span class="element-citation">Cherry L.M., Hsu T.C. Bleomycin-induced chromosome damage in lymphocytes of medullary thyroid carcinoma patients and their family members. <span><span class="ref-journal">Anticancer Res. </span>1983;<span class="ref-vol">3</span>:367–372.</span> [<a href="/pubmed/6200055" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anticancer+Res.&amp;title=Bleomycin-induced+chromosome+damage+in+lymphocytes+of+medullary+thyroid+carcinoma+patients+and+their+family+members&amp;author=L.M.+Cherry&amp;author=T.C.+Hsu&amp;volume=3&amp;publication_year=1983&amp;pages=367-372&amp;pmid=6200055&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C4">4. <span class="element-citation">Cloos J., Spitz M.R., Schantz S.P., Hsu T.C., Zhang Z.F., Tobi H., Braakhuis B.J., Snow G.B. Genetic susceptibility to head and neck squamous cell carcinoma. <span><span class="ref-journal">J. Natl Cancer Inst. </span>1996;<span class="ref-vol">88</span>:530–535.</span> [<a href="/pubmed/8606381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Genetic+susceptibility+to+head+and+neck+squamous+cell+carcinoma&amp;author=J.+Cloos&amp;author=M.R.+Spitz&amp;author=S.P.+Schantz&amp;author=T.C.+Hsu&amp;author=Z.F.+Zhang&amp;volume=88&amp;publication_year=1996&amp;pages=530-535&amp;pmid=8606381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C5">5. <span class="element-citation">Wu X., Gu J., Hong W.K., Lee J.J., Amos C.I., Jiang H., Winn R.J., Fu K.K., Cooper J., Spitz M.R. Benzo[a]pyrene diol epoxide and bleomycin sensitivity and susceptibility to cancer of upper aerodigestive tract. <span><span class="ref-journal">J. Natl Cancer Inst. </span>1998;<span class="ref-vol">90</span>:1393–1399.</span> [<a href="/pubmed/9747870" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Benzo[a]pyrene+diol+epoxide+and+bleomycin+sensitivity+and+susceptibility+to+cancer+of+upper+aerodigestive+tract&amp;author=X.+Wu&amp;author=J.+Gu&amp;author=W.K.+Hong&amp;author=J.J.+Lee&amp;author=C.I.+Amos&amp;volume=90&amp;publication_year=1998&amp;pages=1393-1399&amp;pmid=9747870&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C6">6. <span class="element-citation">Aben K.K., Cloos J., Koper N.P., Braakhuis B.J., Witjes J.A., Kiemeney L.A. Mutagen sensitivity in patients with familial and non-familial urothelial cell carcinoma. <span><span class="ref-journal">Int. J. Cancer. </span>2000;<span class="ref-vol">88</span>:493–496.</span> [<a href="/pubmed/11054683" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=Mutagen+sensitivity+in+patients+with+familial+and+non-familial+urothelial+cell+carcinoma&amp;author=K.K.+Aben&amp;author=J.+Cloos&amp;author=N.P.+Koper&amp;author=B.J.+Braakhuis&amp;author=J.A.+Witjes&amp;volume=88&amp;publication_year=2000&amp;pages=493-496&amp;pmid=11054683&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C7">7. <span class="element-citation">Chao D.L., Maley C.C., Wu X., Farrow D.C., Galipeau P.C., Sanchez C.A., Paulson T.G., Rabinovitch P.S., Reid B.J., Spitz M.R., et al.  Mutagen sensitivity and neoplastic progression in patients with Barrett's esophagus: a prospective analysis. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2006;<span class="ref-vol">15</span>:1935–1940.</span> [<a href="/pubmed/17035402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Mutagen+sensitivity+and+neoplastic+progression+in+patients+with+Barrett's+esophagus:+a+prospective+analysis&amp;author=D.L.+Chao&amp;author=C.C.+Maley&amp;author=X.+Wu&amp;author=D.C.+Farrow&amp;author=P.C.+Galipeau&amp;volume=15&amp;publication_year=2006&amp;pages=1935-1940&amp;pmid=17035402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C8">8. <span class="element-citation">Wu X., Gu J., Dong Q., Huang M., Do K.A., Hong W.K., Spitz M.R. Joint effect of mutagen sensitivity and insulin-like growth factors in predicting the risk of developing secondary primary tumors and tumor recurrence in patients with head and neck cancer. <span><span class="ref-journal">Clin. Cancer Res. </span>2006;<span class="ref-vol">12</span>:7194–7201.</span> [<a href="/pubmed/17145846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Joint+effect+of+mutagen+sensitivity+and+insulin-like+growth+factors+in+predicting+the+risk+of+developing+secondary+primary+tumors+and+tumor+recurrence+in+patients+with+head+and+neck+cancer&amp;author=X.+Wu&amp;author=J.+Gu&amp;author=Q.+Dong&amp;author=M.+Huang&amp;author=K.A.+Do&amp;volume=12&amp;publication_year=2006&amp;pages=7194-7201&amp;pmid=17145846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C9">9. <span class="element-citation">Wu X., Gu J., Spitz M.R. Mutagen sensitivity: a genetic predisposition factor for cancer. <span><span class="ref-journal">Cancer Res. </span>2007;<span class="ref-vol">67</span>:3493–3495.</span> [<a href="/pubmed/17440053" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Mutagen+sensitivity:+a+genetic+predisposition+factor+for+cancer&amp;author=X.+Wu&amp;author=J.+Gu&amp;author=M.R.+Spitz&amp;volume=67&amp;publication_year=2007&amp;pages=3493-3495&amp;pmid=17440053&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C10">10. <span class="element-citation">Roberts S.A., Spreadborough A.R., Bulman B., Barber J.B., Evans D.G., Scott D. Heritability of cellular radiosensitivity: a marker of low-penetrance predisposition genes in breast cancer? <span><span class="ref-journal">Am. J. Hum. Genet. </span>1999;<span class="ref-vol">65</span>:784–794.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377987/">PMC free article</a>]</span> [<a href="/pubmed/10441587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Heritability+of+cellular+radiosensitivity:+a+marker+of+low-penetrance+predisposition+genes+in+breast+cancer?&amp;author=S.A.+Roberts&amp;author=A.R.+Spreadborough&amp;author=B.+Bulman&amp;author=J.B.+Barber&amp;author=D.G.+Evans&amp;volume=65&amp;publication_year=1999&amp;pages=784-794&amp;pmid=10441587&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C11">11. <span class="element-citation">Tedeschi B., Cicchetti R., Argentin G., Caporossi D., Pittaluga M., Parisi P., Vernole P. Aphidicolin and bleomycin induced chromosome damage as biomarker of mutagen sensitivity: a twin study. <span><span class="ref-journal">Mutat. Res. </span>2004;<span class="ref-vol">546</span>:55–64.</span> [<a href="/pubmed/14757193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.+Res.&amp;title=Aphidicolin+and+bleomycin+induced+chromosome+damage+as+biomarker+of+mutagen+sensitivity:+a+twin+study&amp;author=B.+Tedeschi&amp;author=R.+Cicchetti&amp;author=G.+Argentin&amp;author=D.+Caporossi&amp;author=M.+Pittaluga&amp;volume=546&amp;publication_year=2004&amp;pages=55-64&amp;pmid=14757193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C12">12. <span class="element-citation">Cloos J., Nieuwenhuis E.J., Boomsma D.I., Kuik D.J., van der Sterre M.L., Arwert F., Snow G.B., Braakhuis B.J. Inherited susceptibility to bleomycin-induced chromatid breaks in cultured peripheral blood lymphocytes. <span><span class="ref-journal">J. Natl Cancer Inst. </span>1999;<span class="ref-vol">91</span>:1125–1130.</span> [<a href="/pubmed/10393720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Inherited+susceptibility+to+bleomycin-induced+chromatid+breaks+in+cultured+peripheral+blood+lymphocytes&amp;author=J.+Cloos&amp;author=E.J.+Nieuwenhuis&amp;author=D.I.+Boomsma&amp;author=D.J.+Kuik&amp;author=M.L.+van+der+Sterre&amp;volume=91&amp;publication_year=1999&amp;pages=1125-1130&amp;pmid=10393720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C13">13. <span class="element-citation">Cloos J., de Boer W.P., Snel M.H., van den Ijssel P., Ylstra B., Leemans C.R., Brakenhoff R.H., Braakhuis B.J. Microarray analysis of bleomycin-exposed lymphoblastoid cells for identifying cancer susceptibility genes. <span><span class="ref-journal">Mol. Cancer Res. </span>2006;<span class="ref-vol">4</span>:71–77.</span> [<a href="/pubmed/16513838" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cancer+Res.&amp;title=Microarray+analysis+of+bleomycin-exposed+lymphoblastoid+cells+for+identifying+cancer+susceptibility+genes&amp;author=J.+Cloos&amp;author=W.P.+de+Boer&amp;author=M.H.+Snel&amp;author=P.+van+den+Ijssel&amp;author=B.+Ylstra&amp;volume=4&amp;publication_year=2006&amp;pages=71-77&amp;pmid=16513838&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C14">14. <span class="element-citation">Clarke R., Peden J.F., Hopewell J.C., Kyriakou T., Goel A., Heath S.C., Parish S., Barlera S., Franzosi M.G., Rust S., et al.  Genetic variants associated with Lp(a) lipoprotein level and coronary disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2009;<span class="ref-vol">361</span>:2518–2528.</span> [<a href="/pubmed/20032323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Genetic+variants+associated+with+Lp(a)+lipoprotein+level+and+coronary+disease&amp;author=R.+Clarke&amp;author=J.F.+Peden&amp;author=J.C.+Hopewell&amp;author=T.+Kyriakou&amp;author=A.+Goel&amp;volume=361&amp;publication_year=2009&amp;pages=2518-2528&amp;pmid=20032323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C15">15. <span class="element-citation">Hazra A., Kraft P., Selhub J., Giovannucci E.L., Thomas G., Hoover R.N., Chanock S.J., Hunter D.J. Common variants of FUT2 are associated with plasma vitamin B12 levels. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1160–1162.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673801/">PMC free article</a>]</span> [<a href="/pubmed/18776911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+of+FUT2+are+associated+with+plasma+vitamin+B12+levels&amp;author=A.+Hazra&amp;author=P.+Kraft&amp;author=J.+Selhub&amp;author=E.L.+Giovannucci&amp;author=G.+Thomas&amp;volume=40&amp;publication_year=2008&amp;pages=1160-1162&amp;pmid=18776911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C16">16. <span class="element-citation">Wu X., Ye Y., Kiemeney L.A., Sulem P., Rafnar T., Matullo G., Seminara D., Yoshida T., Saeki N., Andrew A.S., et al.  Genetic variation in the prostate stem cell antigen gene PSCA confers susceptibility to urinary bladder cancer. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:991–995.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3313685/">PMC free article</a>]</span> [<a href="/pubmed/19648920" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+variation+in+the+prostate+stem+cell+antigen+gene+PSCA+confers+susceptibility+to+urinary+bladder+cancer&amp;author=X.+Wu&amp;author=Y.+Ye&amp;author=L.A.+Kiemeney&amp;author=P.+Sulem&amp;author=T.+Rafnar&amp;volume=41&amp;publication_year=2009&amp;pages=991-995&amp;pmid=19648920&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C17">17. <span class="element-citation">Amos C.I., Wu X., Broderick P., Gorlov I.P., Gu J., Eisen T., Dong Q., Zhang Q., Gu X., Vijayakrishnan J., et al.  Genome-wide association scan of tag SNPs identifies a susceptibility locus for lung cancer at 15q25.1. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:616–622.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2713680/">PMC free article</a>]</span> [<a href="/pubmed/18385676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+locus+for+lung+cancer+at+15q25.1&amp;author=C.I.+Amos&amp;author=X.+Wu&amp;author=P.+Broderick&amp;author=I.P.+Gorlov&amp;author=J.+Gu&amp;volume=40&amp;publication_year=2008&amp;pages=616-622&amp;pmid=18385676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C18">18. <span class="element-citation">Hijikata M., Kato N., Sato T., Kagami Y., Shimotohno K. Molecular cloning and characterization of a cDNA for a novel phorbol-12-myristate-13-acetate-responsive gene that is highly expressed in an adult T-cell leukemia cell line. <span><span class="ref-journal">J. Virol. </span>1990;<span class="ref-vol">64</span>:4632–4639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC247947/">PMC free article</a>]</span> [<a href="/pubmed/2398525" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Virol.&amp;title=Molecular+cloning+and+characterization+of+a+cDNA+for+a+novel+phorbol-12-myristate-13-acetate-responsive+gene+that+is+highly+expressed+in+an+adult+T-cell+leukemia+cell+line&amp;author=M.+Hijikata&amp;author=N.+Kato&amp;author=T.+Sato&amp;author=Y.+Kagami&amp;author=K.+Shimotohno&amp;volume=64&amp;publication_year=1990&amp;pages=4632-4639&amp;pmid=2398525&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C19">19. <span class="element-citation">Oda E., Ohki R., Murasawa H., Nemoto J., Shibue T., Yamashita T., Tokino T., Taniguchi T., Tanaka N. Noxa, a BH3-only member of the Bcl-2 family and candidate mediator of p53-induced apoptosis. <span><span class="ref-journal">Science. </span>2000;<span class="ref-vol">288</span>:1053–1058.</span> [<a href="/pubmed/10807576" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Noxa,+a+BH3-only+member+of+the+Bcl-2+family+and+candidate+mediator+of+p53-induced+apoptosis&amp;author=E.+Oda&amp;author=R.+Ohki&amp;author=H.+Murasawa&amp;author=J.+Nemoto&amp;author=T.+Shibue&amp;volume=288&amp;publication_year=2000&amp;pages=1053-1058&amp;pmid=10807576&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C20">20. <span class="element-citation">Ploner C., Kofler R., Villunger A. Noxa: at the tip of the balance between life and death. <span><span class="ref-journal">Oncogene. </span>2008;<span class="ref-vol">27</span>(Suppl. 1):S84–S92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3272398/">PMC free article</a>]</span> [<a href="/pubmed/19641509" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Noxa:+at+the+tip+of+the+balance+between+life+and+death&amp;author=C.+Ploner&amp;author=R.+Kofler&amp;author=A.+Villunger&amp;volume=27&amp;issue=Suppl.+1&amp;publication_year=2008&amp;pages=S84-S92&amp;pmid=19641509&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C21">21. <span class="element-citation">Haston C.K., Tomko T.G., Godin N., Kerckhoff L., Hallett M.T. Murine candidate bleomycin induced pulmonary fibrosis susceptibility genes identified by gene expression and sequence analysis of linkage regions. <span><span class="ref-journal">J. Med. Genet. </span>2005;<span class="ref-vol">42</span>:464–473.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1736068/">PMC free article</a>]</span> [<a href="/pubmed/15937080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Murine+candidate+bleomycin+induced+pulmonary+fibrosis+susceptibility+genes+identified+by+gene+expression+and+sequence+analysis+of+linkage+regions&amp;author=C.K.+Haston&amp;author=T.G.+Tomko&amp;author=N.+Godin&amp;author=L.+Kerckhoff&amp;author=M.T.+Hallett&amp;volume=42&amp;publication_year=2005&amp;pages=464-473&amp;pmid=15937080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C22">22. <span class="element-citation">Chambers J.C., Elliott P., Zabaneh D., Zhang W., Li Y., Froguel P., Balding D., Scott J., Kooner J.S. Common genetic variation near MC4R is associated with waist circumference and insulin resistance. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:716–718.</span> [<a href="/pubmed/18454146" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+genetic+variation+near+MC4R+is+associated+with+waist+circumference+and+insulin+resistance&amp;author=J.C.+Chambers&amp;author=P.+Elliott&amp;author=D.+Zabaneh&amp;author=W.+Zhang&amp;author=Y.+Li&amp;volume=40&amp;publication_year=2008&amp;pages=716-718&amp;pmid=18454146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C23">23. <span class="element-citation">Loos R.J., Lindgren C.M., Li S., Wheeler E., Zhao J.H., Prokopenko I., Inouye M., Freathy R.M., Attwood A.P., Beckmann J.S., et al.  Common variants near MC4R are associated with fat mass, weight and risk of obesity. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:768–775.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669167/">PMC free article</a>]</span> [<a href="/pubmed/18454148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+near+MC4R+are+associated+with+fat+mass,+weight+and+risk+of+obesity&amp;author=R.J.+Loos&amp;author=C.M.+Lindgren&amp;author=S.+Li&amp;author=E.+Wheeler&amp;author=J.H.+Zhao&amp;volume=40&amp;publication_year=2008&amp;pages=768-775&amp;pmid=18454148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C24">24. <span class="element-citation">Parihar M.S., Parihar A., Chen Z., Nazarewicz R., Ghafourifar P. mAtNOS1 regulates mitochondrial functions and apoptosis of human neuroblastoma cells. <span><span class="ref-journal">Biochim. Biophys. Acta. </span>2008;<span class="ref-vol">1780</span>:921–926.</span> [<a href="/pubmed/18359297" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.+Biophys.+Acta&amp;title=mAtNOS1+regulates+mitochondrial+functions+and+apoptosis+of+human+neuroblastoma+cells&amp;author=M.S.+Parihar&amp;author=A.+Parihar&amp;author=Z.+Chen&amp;author=R.+Nazarewicz&amp;author=P.+Ghafourifar&amp;volume=1780&amp;publication_year=2008&amp;pages=921-926&amp;pmid=18359297&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C25">25. <span class="element-citation">Weissman A.M. How much REST is enough. <span><span class="ref-journal">Cancer Cell. </span>2008;<span class="ref-vol">13</span>:381–383.</span> [<a href="/pubmed/18455121" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Cell&amp;title=How+much+REST+is+enough&amp;author=A.M.+Weissman&amp;volume=13&amp;publication_year=2008&amp;pages=381-383&amp;pmid=18455121&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C26">26. <span class="element-citation">Guardavaccaro D., Frescas D., Dorrello N.V., Peschiaroli A., Multani A.S., Cardozo T., Lasorella A., Iavarone A., Chang S., Hernando E., et al.  Control of chromosome stability by the beta-TrCP-REST-Mad2 axis. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:365–369.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2707768/">PMC free article</a>]</span> [<a href="/pubmed/18354482" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Control+of+chromosome+stability+by+the+beta-TrCP-REST-Mad2+axis&amp;author=D.+Guardavaccaro&amp;author=D.+Frescas&amp;author=N.V.+Dorrello&amp;author=A.+Peschiaroli&amp;author=A.S.+Multani&amp;volume=452&amp;publication_year=2008&amp;pages=365-369&amp;pmid=18354482&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C27">27. <span class="element-citation">Tuimala J., Szekely G., Gundy S., Hirvonen A., Norppa H. Genetic polymorphisms of DNA repair and xenobiotic-metabolizing enzymes: role in mutagen sensitivity. <span><span class="ref-journal">Carcinogenesis. </span>2002;<span class="ref-vol">23</span>:1003–1008.</span> [<a href="/pubmed/12082022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Genetic+polymorphisms+of+DNA+repair+and+xenobiotic-metabolizing+enzymes:+role+in+mutagen+sensitivity&amp;author=J.+Tuimala&amp;author=G.+Szekely&amp;author=S.+Gundy&amp;author=A.+Hirvonen&amp;author=H.+Norppa&amp;volume=23&amp;publication_year=2002&amp;pages=1003-1008&amp;pmid=12082022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C28">28. <span class="element-citation">Wang Y., Spitz M.R., Zhu Y., Dong Q., Shete S., Wu X. From genotype to phenotype: correlating XRCC1 polymorphisms with mutagen sensitivity. <span><span class="ref-journal">DNA Repair (Amst.) </span>2003;<span class="ref-vol">2</span>:901–908.</span> [<a href="/pubmed/12893086" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair+(Amst.)&amp;title=From+genotype+to+phenotype:+correlating+XRCC1+polymorphisms+with+mutagen+sensitivity&amp;author=Y.+Wang&amp;author=M.R.+Spitz&amp;author=Y.+Zhu&amp;author=Q.+Dong&amp;author=S.+Shete&amp;volume=2&amp;publication_year=2003&amp;pages=901-908&amp;pmid=12893086&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C29">29. <span class="element-citation">Maffei F., Carbone F., Angelini S., Forti G.C., Norppa H., Hrelia P. Micronuclei frequency induced by bleomycin in human peripheral lymphocytes: correlating BLHX polymorphism with mutagen sensitivity. <span><span class="ref-journal">Mutat. Res. </span>2008;<span class="ref-vol">639</span>:20–26.</span> [<a href="/pubmed/18082847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.+Res.&amp;title=Micronuclei+frequency+induced+by+bleomycin+in+human+peripheral+lymphocytes:+correlating+BLHX+polymorphism+with+mutagen+sensitivity&amp;author=F.+Maffei&amp;author=F.+Carbone&amp;author=S.+Angelini&amp;author=G.C.+Forti&amp;author=H.+Norppa&amp;volume=639&amp;publication_year=2008&amp;pages=20-26&amp;pmid=18082847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C30">30. <span class="element-citation">Angelini S., Kumar R., Carbone F., Bermejo J.L., Maffei F., Cantelli-Forti G., Hemminki K., Hrelia P. Inherited susceptibility to bleomycin-induced micronuclei: correlating polymorphisms in GSTT1, GSTM1 and DNA repair genes with mutagen sensitivity. <span><span class="ref-journal">Mutat. Res. </span>2008;<span class="ref-vol">638</span>:90–97.</span> [<a href="/pubmed/17953974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mutat.+Res.&amp;title=Inherited+susceptibility+to+bleomycin-induced+micronuclei:+correlating+polymorphisms+in+GSTT1,+GSTM1+and+DNA+repair+genes+with+mutagen+sensitivity&amp;author=S.+Angelini&amp;author=R.+Kumar&amp;author=F.+Carbone&amp;author=J.L.+Bermejo&amp;author=F.+Maffei&amp;volume=638&amp;publication_year=2008&amp;pages=90-97&amp;pmid=17953974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C31">31. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C32">32. <span class="element-citation">Li Y., Abecasis G. Mach 1.0: rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Mach+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C33">33. <span class="element-citation">Hsu T.C., Cherry L.M. Calculation of bleomycin doses. <span><span class="ref-journal">Cancer Genet. Cytogenet. </span>1991;<span class="ref-vol">53</span>:281.</span> [<a href="/pubmed/1712243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Genet.+Cytogenet.&amp;title=Calculation+of+bleomycin+doses&amp;author=T.C.+Hsu&amp;author=L.M.+Cherry&amp;volume=53&amp;publication_year=1991&amp;pages=281&amp;pmid=1712243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C34">34. <span class="element-citation">Lettre G., Lange C., Hirschhorn J.N. Genetic model testing and statistical power in population-based association studies of quantitative traits. <span><span class="ref-journal">Genet. Epidemiol. </span>2007;<span class="ref-vol">31</span>:358–362.</span> [<a href="/pubmed/17352422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Genetic+model+testing+and+statistical+power+in+population-based+association+studies+of+quantitative+traits&amp;author=G.+Lettre&amp;author=C.+Lange&amp;author=J.N.+Hirschhorn&amp;volume=31&amp;publication_year=2007&amp;pages=358-362&amp;pmid=17352422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C35">35. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C36">36. <span class="element-citation">Barrett J.C., Fry B., Maller J., Daly M.J. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=J.C.+Barrett&amp;author=B.+Fry&amp;author=J.+Maller&amp;author=M.J.+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDQ509C37">37. <span class="element-citation">Kent W.J., Sugnet C.W., Furey T.S., Roskin K.M., Pringle T.H., Zahler A.M., Haussler D. The human genome browser at UCSC. <span><span class="ref-journal">Genome Res. </span>2002;<span class="ref-vol">12</span>:996–1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC186604/">PMC free article</a>]</span> [<a href="/pubmed/12045153" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=The+human+genome+browser+at+UCSC&amp;author=W.J.+Kent&amp;author=C.W.+Sugnet&amp;author=T.S.+Furey&amp;author=K.M.+Roskin&amp;author=T.H.+Pringle&amp;volume=12&amp;publication_year=2002&amp;pages=996-1006&amp;pmid=12045153&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3024041&amp;issue-id=193535&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139714191970832title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#idm139714191709552title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139714191707680title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3024041/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3024041/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3024041/pdf/ddq509.pdf">PDF (204K)</a></li> | <li><a href="#" data-citationid="PMC3024041" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3024041%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3024041%2F&amp;text=A%20genetic%20variant%20near%20the%20PMAIP1%2FNoxa%20gene%20is%20associated%20with%20increased%20bleomycin%20sensitivity" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3024041%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21106707" data-aiid="3024041" data-aid="3024041" data-iid="193535" data-domainid="740" data-domain="hmg" data-accid="PMC3024041" data-md5="47b87dad6f180d985f748aeb7c31ca16"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T06:18:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE885992DD520C610000000000200014&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3024041%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3024041/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>