<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jbmr"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3312758/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of Homogeneous Genetic Architecture of Multiple Genetically Correlated Traits by Block Clustering of Genome-Wide Associations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312758/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Bone and Mineral Research"><meta name="citation_title" content="Identification of Homogeneous Genetic Architecture of Multiple Genetically Correlated Traits by Block Clustering of Genome-Wide Associations"><meta name="citation_authors" content="Mayetri Gupta, Ching-Lung Cheung, Yi-Hsiang Hsu, Serkalem Demissie, L Adrienne Cupples, Douglas P Kiel, David Karasik"><meta name="citation_date" content="June 2011"><meta name="citation_issue" content="6"><meta name="citation_volume" content="26"><meta name="citation_firstpage" content="1261"><meta name="citation_doi" content="10.1002/jbmr.333"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3312758/?report=abstract"><meta name="citation_pmid" content="21611967"><meta name="DC.Title" content="Identification of Homogeneous Genetic Architecture of Multiple Genetically Correlated Traits by Block Clustering of Genome-Wide Associations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society for Bone and Mineral Research"><meta name="DC.Contributor" content="Mayetri Gupta"><meta name="DC.Contributor" content="Ching-Lung Cheung"><meta name="DC.Contributor" content="Yi-Hsiang Hsu"><meta name="DC.Contributor" content="Serkalem Demissie"><meta name="DC.Contributor" content="L Adrienne Cupples"><meta name="DC.Contributor" content="Douglas P Kiel"><meta name="DC.Contributor" content="David Karasik"><meta name="DC.Date" content="2011 Jun"><meta name="DC.Identifier" content="10.1002/jbmr.333"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of Homogeneous Genetic Architecture of Multiple Genetically Correlated Traits by Block Clustering of Genome-Wide Associations"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) using high-density genotyping platforms offer an unbiased strategy to identify new candidate genes for osteoporosis. It is imperative to be able to clearly distinguish signal from noise by focusing on the best phenotype ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312758/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jbmr.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3312758/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3312758/pdf/jbmr-26-6-1261.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89AE52DD52028100000000086E0560.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312758%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22J Bone Miner Res%22[journal]" class="navlink">J Bone Miner Res</a></li><li class="accid">PMC3312758</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jbmr.gif" alt="Logo of jbmr" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="24,55,134,71" alt="JBMR Instructions for Authors" title="JBMR Instructions for Authors" href="http://www.jbmr.org/view/0/authorGuide.html" ref="reftype=publisher&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="152,55,212,71" alt="Subscribe to JBMR" title="Subscribe to JBMR" href="http://www.jbmr.org/view/0/SubscriberInfo.html" ref="reftype=publisher&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="227,55,267,71" alt="About JBMR" title="About JBMR" href="http://www.jbmr.org" ref="reftype=publisher&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="282,55,362,71" alt="JBMR Editorial Board" title="JBMR Editorial Board" href="http://www.jbmr.org/view/0/editorialBoard.html" ref="reftype=publisher&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,500,50" alt="JBMR" title="JBMR" href="http://www.jbmr.org" ref="reftype=publisher&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Bone Miner Res</a></span>. 2011 Jun; 26(6): 1261–1271. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Jan 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fjbmr.333" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/jbmr.333</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3312758</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21611967">21611967</a></div></div></div></div><h1 class="content-title">Identification of Homogeneous Genetic Architecture of Multiple Genetically Correlated Traits by Block Clustering of Genome-Wide Associations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Gupta%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140238022676480" co-class="co-affbox">Mayetri Gupta</a>,<sup>1</sup> <a href="/pubmed/?term=Cheung%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140237985909728" co-class="co-affbox">Ching-Lung Cheung</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140238022673248" co-class="co-affbox">Yi-Hsiang Hsu</a>,<sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Demissie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140237986396640" co-class="co-affbox">Serkalem Demissie</a>,<sup>1</sup> <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140237986394416" co-class="co-affbox">L Adrienne Cupples</a>,<sup>1</sup> <a href="/pubmed/?term=Kiel%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140237986392192" co-class="co-affbox">Douglas P Kiel</a>,<sup>2</sup> and  <a href="/pubmed/?term=Karasik%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967" class="affpopup" co-rid="_co_idm140237986389968" co-class="co-affbox">David Karasik</a><sup>2</sup><sup></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140238022676480"><h3 class="no_margin">Mayetri Gupta</h3><p><sup>1</sup>Department of Biostatistics, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Gupta%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">Mayetri Gupta</a></div></div><div id="_co_idm140237985909728"><h3 class="no_margin">Ching-Lung Cheung</h3><p><sup>2</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, and Harvard Medical School, Boston, MA, USA</p><p><sup>3</sup>Department of Medicine, University of Hong Kong, Hong-Kong, China</p><div>Find articles by <a href="/pubmed/?term=Cheung%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">Ching-Lung Cheung</a></div></div><div id="_co_idm140238022673248"><h3 class="no_margin">Yi-Hsiang Hsu</h3><p><sup>2</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, and Harvard Medical School, Boston, MA, USA</p><p><sup>4</sup>Molecular and Integrative Physiological Sciences Program, Harvard School of Public Health, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">Yi-Hsiang Hsu</a></div></div><div id="_co_idm140237986396640"><h3 class="no_margin">Serkalem Demissie</h3><p><sup>1</sup>Department of Biostatistics, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Demissie%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">Serkalem Demissie</a></div></div><div id="_co_idm140237986394416"><h3 class="no_margin">L Adrienne Cupples</h3><p><sup>1</sup>Department of Biostatistics, Boston University, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">L Adrienne Cupples</a></div></div><div id="_co_idm140237986392192"><h3 class="no_margin">Douglas P Kiel</h3><p><sup>2</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, and Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Kiel%20DP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">Douglas P Kiel</a></div></div><div id="_co_idm140237986389968"><h3 class="no_margin">David Karasik</h3><p><sup>2</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, and Harvard Medical School, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Karasik%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21611967">David Karasik</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140237984048336_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140237984048336_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140237984048336_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140237984048336_ai" style="display:none"><div class="fm-affl" id="af1-jbmr-26-6-1261" lang="en"><sup>1</sup>Department of Biostatistics, Boston University, Boston, MA, USA</div><div class="fm-affl" id="af2-jbmr-26-6-1261" lang="en"><sup>2</sup>Institute for Aging Research, Hebrew SeniorLife, Boston, and Harvard Medical School, Boston, MA, USA</div><div class="fm-affl" id="af3-jbmr-26-6-1261" lang="en"><sup>3</sup>Department of Medicine, University of Hong Kong, Hong-Kong, China</div><div class="fm-affl" id="af4-jbmr-26-6-1261" lang="en"><sup>4</sup>Molecular and Integrative Physiological Sciences Program, Harvard School of Public Health, Boston, MA, USA</div><div id="c1-jbmr-26-6-1261">Address correspondence to: David Karasik, PhD, Institute for Aging Research, Hebrew SeniorLife, 1200 Center Street, Boston, MA 02131, USA., E-mail: <a href="mailto:dev@null" data-email="ude.dravrah.acrh@kisarak" class="oemail">ude.dravrah.acrh@kisarak</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140237984048336_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2010 Jul 15; Revised 2010 Nov 17; Accepted 2010 Dec 23.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140237984048336_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2011 American Society for Bone and Mineral Research</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3312758/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261-s1.doc" data-ga-action="click_feat_suppl">jbmr-26-6-1261-s1.doc</a><span style="color:gray"> (48K)</span></div><div class="small guid">GUID:&nbsp;0825C6B4-831D-4F60-8B5D-007B875DE76B</div></div><div class="half_rhythm"><div><a href="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261-s2.doc" data-ga-action="click_feat_suppl">jbmr-26-6-1261-s2.doc</a><span style="color:gray"> (561K)</span></div><div class="small guid">GUID:&nbsp;D66906A5-FA45-4B5C-A008-BBFAC93A57FF</div></div></dd></dl></div><div id="idm140237986253920" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237986253920title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Genome-wide association studies (GWAS) using high-density genotyping platforms offer an unbiased strategy to identify new candidate genes for osteoporosis. It is imperative to be able to clearly distinguish signal from noise by focusing on the best phenotype in a genetic study. We performed GWAS of multiple phenotypes associated with fractures [bone mineral density (BMD), bone quantitative ultrasound (QUS), bone geometry, and muscle mass] with approximately 433,000 single-nucleotide polymorphisms (SNPs) and created a database of resulting associations. We performed analysis of GWAS data from 23 phenotypes by a novel modification of a block clustering algorithm followed by gene-set enrichment analysis. A data matrix of standardized regression coefficients was partitioned along both axes—SNPs and phenotypes. Each partition represents a distinct cluster of SNPs that have similar effects over a particular set of phenotypes. Application of this method to our data shows several SNP-phenotype connections. We found a strong cluster of association coefficients of high magnitude for 10 traits (BMD at several skeletal sites, ultrasound measures, cross-sectional bone area, and section modulus of femoral neck and shaft). These clustered traits were highly genetically correlated. Gene-set enrichment analyses indicated the augmentation of genes that cluster with the 10 osteoporosis-related traits in pathways such as aldosterone signaling in epithelial cells, role of osteoblasts, osteoclasts, and chondrocytes in rheumatoid arthritis, and Parkinson signaling. In addition to several known candidate genes, we also identified <em>PRKCH</em> and <em>SCNN1B</em> as potential candidate genes for multiple bone traits. In conclusion, our mining of GWAS results revealed the similarity of association results between bone strength phenotypes that may be attributed to pleiotropic effects of genes. This knowledge may prove helpful in identifying novel genes and pathways that underlie several correlated phenotypes, as well as in deciphering genetic and phenotypic modularity underlying osteoporosis risk.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">BLOCK CLUSTERING, GENOME-WIDE ASSOCIATION STUDY, OSTEOPOROSIS, PHENOMICS, PLEIOTROPY</span></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Genome-wide association studies (GWAS) using high-density genotyping platforms offer an unbiased strategy to identify new candidate genes for different traits and diseases. A number of candidate genes have been identified for traits associated with fracture, such as bone mineral density (BMD),<sup>(<a href="#b1-jbmr-26-6-1261" rid="b1-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612696">1</a>–<a href="#b3-jbmr-26-6-1261" rid="b3-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612716">3</a>)</sup> bone geometry,<sup>(<a href="#b4-jbmr-26-6-1261" rid="b4-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612679">4</a>,<a href="#b5-jbmr-26-6-1261" rid="b5-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612713">5</a>)</sup> and lean mass.<sup>(<a href="#b6-jbmr-26-6-1261" rid="b6-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612721">6</a>)</sup> Although this is a powerful approach for disease gene discovery, many GWAS focus only on identification of candidate genes and provide no clues on gene networks that may underlie the trait variations. Genes do not work alone, especially in the genetics of complex diseases. More often, they work in an interactive network synergistically and coregulate the expression of proteins that are implicated in a trait variation. In the study of disease networks or interactomes, it has been suggested that disease genes usually are located centrally (“hub”) and are highly connected with other genes in the interactome such that perturbations of candidate genes in the same network are more likely to cause either the same or similar disease phenotypes.<sup>(<a href="#b7-jbmr-26-6-1261" rid="b7-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612671">7</a>,<a href="#b8-jbmr-26-6-1261" rid="b8-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_525591594">8</a>)</sup> Given that many traits are correlated genetically, it is possible that common gene network(s) underlie multiple correlated trait variations. In fact, using a plethora of genetic information from high-throughput technologies such as mRNA microarray and GWAS, it is now feasible to “reverse phenotype” (whereby associations with genetic markers are used to define phenotype groupings)<sup>(<a href="#b9-jbmr-26-6-1261" rid="b9-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612667">9</a>,<a href="#b10-jbmr-26-6-1261" rid="b10-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612680">10</a>)</sup> and classify multiple traits based on the similarity of their genetic organization. There is a need to develop innovative computational and informatics tools and high-throughput discovery technologies for both gene network and phenomic research<sup>(<a href="#b11-jbmr-26-6-1261" rid="b11-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612685">11</a>)</sup> given the large numbers of multiple correlated traits that are common in cohorts performing GWAS.</p><p id="__p3">Age-related osteoporotic fractures are common in the United States and represent a major public health threat that is likely to increase in importance as the population ages.<sup>(<a href="#b12-jbmr-26-6-1261" rid="b12-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612688">12</a>)</sup> Although osteoporotic fracture per se is a heritable trait, the fracture phenotype presents a challenge for a genetic study: Nontraumatic fractures typically do not occur until later in life. Therefore, quantitative risk factors for fracture [such as BMD, quantitative ultrasound (QUS), and bone geometry] are used traditionally,<sup>(<a href="#b13-jbmr-26-6-1261" rid="b13-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612682">13</a>–<a href="#b19-jbmr-26-6-1261" rid="b19-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612717">19</a>)</sup> especially because these surrogate traits may be measured consistently at any age. Combinations of BMD assessments in more than one region,<sup>(<a href="#b20-jbmr-26-6-1261" rid="b20-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612687">20</a>)</sup> as well as composite use of BMD with QUS<sup>(<a href="#b17-jbmr-26-6-1261" rid="b17-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612729">17</a>)</sup> or BMD with hip geometry,<sup>(<a href="#b21-jbmr-26-6-1261" rid="b21-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612652">21</a>)</sup> have been suggested to improve risk assessment in clinical practice. In addition, it seems worthwhile to use information on muscle mass and strength together with bone mineralization and its spatial arrangement to assess the risk of fragility because loads that are applied to the skeleton are attributable to both overall body weight and muscle contractions. These musculoskeletal anatomic structures are a conglomeration of multiple traits; some of them are phenotypically and genetically correlated. Therefore, it is important to understand how the relationship among traits is regulated<sup>(<a href="#b22-jbmr-26-6-1261" rid="b22-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_305062482">22</a>)</sup> so as to gain more insights into complex disease etiologies.</p><p id="__p4">In this study we present a biclustering approach for identifying common genetic architectures of different musculoskeletal traits using GWAS data. Statistical model-based clustering is useful in many applications, including gene expression analysis, but one of its drawbacks is the necessity of allocating all samples (or genes) to at least one cluster, which, owing to noise in the data, may lead to artificial clusters that have no real biologic meaning. One heuristic method that has been proposed to overcome this problem is tight clustering,<sup>(<a href="#b23-jbmr-26-6-1261" rid="b23-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612678">23</a>)</sup> in which only samples (or variables) that are similar enough, according to some predefined criterion, are clustered together, whereas the rest are left ungrouped as outliers. However, this method works only in one dimension, considering either samples or variables, and not if both directions are to be considered simultaneously, as we wish to do here. On the other hand, biclustering or block clustering<sup>(<a href="#b24-jbmr-26-6-1261" rid="b24-jbmr-26-6-1261" class=" bibr popnode">24</a>–<a href="#b28-jbmr-26-6-1261" rid="b28-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612712">28</a>)</sup> is an approach to simultaneously cluster row and column variables, deriving local subgroups within a data matrix. Many biclustering methods also may be adapted so that not all samples and variables need to be grouped (only ones that are “similar enough” according to some predefined measure). These approaches have been used previously in discovering local patterns in gene expression data.<sup>(<a href="#b25-jbmr-26-6-1261" rid="b25-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612689">25</a>,<a href="#b26-jbmr-26-6-1261" rid="b26-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612708">26</a>)</sup> Based on a Bayesian statistical model framework proposed by Gu and Liu,<sup>(<a href="#b28-jbmr-26-6-1261" rid="b28-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612728">28</a>)</sup> but with modifications to increase the robustness and efficiency of the model for dealing with the more massive GWAS data, we developed a fast and accurate algorithm to identify block clusters of single-nucleotide polymorphisms (SNP) and traits in data arising from GWAS.<sup>(<a href="#b29-jbmr-26-6-1261" rid="b29-jbmr-26-6-1261" class=" bibr popnode">29</a>)</sup></p><p id="__p5" class="p p-last">We applied this method to results from bone and muscle traits from the Framingham Osteoporosis Study. We studied GWAS data on 23 traits and approximately 433,000 SNPs by a modification of our block-clustering algorithm.<sup>(<a href="#b29-jbmr-26-6-1261" rid="b29-jbmr-26-6-1261" class=" bibr popnode">29</a>)</sup> The primary aim of this study was to identify homogeneous genetic architectures among different musculoskeletal traits and to classify these traits based on a reverse-phenotyping approach.<sup>(<a href="#b9-jbmr-26-6-1261" rid="b9-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612698">9</a>)</sup> We hypothesized that correlated musculoskeletal phenotypes would cluster together based on their SNP associations (thus representing “phenotypic moduli”<sup>(<a href="#b9-jbmr-26-6-1261" rid="b9-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612704">9</a>)</sup>) and that the shared SNPs would indicate a gene network that contributes to the development and/or maintenance of the musculoskeletal traits (thus reflecting a “genetic modularity”<sup>(<a href="#b30-jbmr-26-6-1261" rid="b30-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612724">30</a>)</sup>).</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Sample</h3><p id="__p6" class="p p-first-last">The sample used for our analyses was derived from two cohorts of the population-based Framingham Heart Study. Details of the Framingham Osteoporosis Study (FOS), an ancillary study of the Framingham Heart Study, have been reported.<sup>(<a href="#b31-jbmr-26-6-1261" rid="b31-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612670">31</a>,<a href="#b32-jbmr-26-6-1261" rid="b32-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612656">32</a>)</sup> In brief, the FOS original and the offspring cohorts represent members of two-generational (mostly nuclear) families recruited at different times. Descriptions of the family samples with musculoskeletal phenotypes available for analyses in the FOS are provided elsewhere,<sup>(<a href="#b32-jbmr-26-6-1261" rid="b32-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612690">32</a>,<a href="#b33-jbmr-26-6-1261" rid="b33-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612727">33</a>)</sup> as well as being available publicly through the dbGaP at <a href="http://view.ncbi.nlm.nih.gov/dbgap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://view.ncbi.nlm.nih.gov/dbgap</a>. For this study, a sample of 2211 women and 1633 men who had phenotypic measurements and who consented to genetic analyses was available. Details on the FOS sample also are available elsewhere.<sup>(<a href="#b34-jbmr-26-6-1261" rid="b34-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612725">34</a>,<a href="#b35-jbmr-26-6-1261" rid="b35-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612692">35</a>)</sup> The study was approved by the institutional review boards for human subjects research of Boston University and Hebrew SeniorLife.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Musculoskeletal phenotypes</h3><p id="__p7" class="p p-first">The following measures were available in members of both cohorts of the FOS:</p><div id="__sec5" class="sec"><p></p><h4 id="__sec5title" class="inline">Bone densitometry </h4><p id="__p8" class="p p-first-last">The participants underwent dual-energy X-ray absorptiometry (DXA) with a Lunar DPX-L device (Lunar Corp., Madison, WI, USA) between 1996 and 2001 to assess BMD at the lumbar spine (LS), femoral neck (FN), trochanter (TR), and whole body (TOT). The coefficients of variation (CVs) in normal subjects for the DPX-L have been reported previously as 0.9% for LS, 1.7% for FN, and 2.5% for TR.<sup>(<a href="#b31-jbmr-26-6-1261" rid="b31-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612674">31</a>)</sup></p></div><div id="__sec6" class="sec"><p></p><h4 id="__sec6title" class="inline">Quantitative ultrasound (QUS) </h4><p id="__p9" class="p p-first-last">QUS of the right heel was performed to obtain calcaneal broadband ultrasound attenuation (BUA) and speed of sound (SOS) with a Sahara bone sonometer (Hologic, Inc., Waltham, MA, USA) between 1996 and 2001. Based on duplicate same-day measurements on 29 subjects, CVs for BUA and SOS were 5.3% and 0.4%, respectively.<sup>(<a href="#b36-jbmr-26-6-1261" rid="b36-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612651">36</a>)</sup></p></div><div id="__sec7" class="sec"><p></p><h4 id="__sec7title" class="inline">Hip geometry </h4><p id="__p10" class="p p-first-last">DXA scans were measured with an interactive computer program<sup>(<a href="#b37-jbmr-26-6-1261" rid="b37-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612718">37</a>)</sup> that derived a number of proximal femoral structural variables, including gross anatomic [femoral neck length (FNL, cm) and neck-shaft angle (NSA, degrees)] and cross-sectional indices (outer, or subperiosteal, width, cm), cross-sectional bone area (CSA, cm<sup>2</sup>), and section modulus (Z, cm<sup>3</sup>) at the two femoral regions (narrow neck, NN, and femoral shaft, S). CVs were reported previously to range from 3.3% (NN outer diameter) to 9.1% (FNL).<sup>(<a href="#b37-jbmr-26-6-1261" rid="b37-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612697">37</a>)</sup></p></div><div id="__sec8" class="sec"><p></p><h4 id="__sec8title" class="inline">Body composition </h4><p id="__p11" class="p p-first-last">We obtained whole-body DXA scans from the study participants with the same Lunar DPX-L machine used for BMD and at the same visit. The scans were collected at medium speed for all subjects regardless of weight or body thickness. Regions of interest were analyzed using the extended analysis of the Lunar software for body composition. Lean mass was derived by subtracting regional bone mineral content from the fat-free mass of the whole body and lower extremities.</p></div><div id="__sec9" class="sec"><p></p><h4 id="__sec9title" class="inline">Metacarpal measurements by radiogrammetry </h4><p id="__p12" class="p p-first-last">In the Framingham original cohort, right-hand radiographs were obtained in 1967–1969, and the subsample of the offspring cohort was radiographed using the same techniques in 1993–1995<sup>(<a href="#b38-jbmr-26-6-1261" rid="b38-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612714">38</a>,<a href="#b39-jbmr-26-6-1261" rid="b39-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612672">39</a>)</sup> as part of the osteoporosis study. The digitized radiographic images of the second, third, and fourth metacarpals were measured semiautomatically (as described in detail elsewhere<sup>(<a href="#b40-jbmr-26-6-1261" rid="b40-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612686">40</a>)</sup>). CVs, calculated as root-mean-square average,<sup>(<a href="#b41-jbmr-26-6-1261" rid="b41-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612693">41</a>)</sup> ranged from 1.0% (bone length) to 7.0% (midshaft endosteal diameter). Metacarpal length and midshaft width were measured directly, whereas cortical cross-sectional indices, such as metacarpal cortical thickness (MCT, mm), cortical index (MCI, %), and section modulus (MZ, mm<sup>3</sup>), were calculated in the middle of bone diaphysis<sup>(<a href="#b42-jbmr-26-6-1261" rid="b42-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612715">42</a>–<a href="#b44-jbmr-26-6-1261" rid="b44-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612653">44</a>)</sup>; values of metacarpals 2, 3, and 4 were averaged.</p></div><div id="__sec10" class="sec sec-last"><p></p><h4 id="__sec10title" class="inline">Other measurements (covariates) </h4><p id="__p13" class="p p-first-last">Information on age, sex, height, body mass index (BMI), and for women, menopausal status and estrogen use were obtained at the time of bone measurement. Details for these measurements are available elsewhere.<sup>(<a href="#b31-jbmr-26-6-1261" rid="b31-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612711">31</a>,<a href="#b45-jbmr-26-6-1261" rid="b45-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612676">45</a>)</sup> Women were assigned to one of two estrogenic status groups: (1) premenopausal or postmenopausal on estrogen (estrogen replete) or (2) postmenopausal not on estrogen (estrogen depleted).</p></div></div><div id="__sec11" class="sec"><h3 id="__sec11title">Genotyping, quality control, and population structure</h3><p id="__p14" class="p p-first">Genotyping was conducted through the FHS SHARe (SNP Health Association Resource) project initiated in 2007 on all Framingham Study participants with DNA available using the Affymetrix 500K (250K Sty and 250K Nsp) mapping array with addition of the Affymetrix 50K MIP (gene-centric) supplemental array (Santa Clara, CA, USA). Sample-level exclusions were a participant call rate of less than 97%, a per-subject heterozygosity of ±5 SD from the mean, or a per-subject number of Mendelian errors greater than 165 (99th quantile). Genotyping from 433,510 SNPs in 8481 individuals passed these quality-control measures. Eigenstrat principal-components analysis<sup>(<a href="#b46-jbmr-26-6-1261" rid="b46-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612695">46</a>)</sup> was applied to evaluate population structure using a subset of 425,173 SNPs (minor allele frequency [MAF] ≥0.01, Hardy–Weinberg equilibrium [HWE] <em>p</em> ≥10<sup>−6</sup>, and call rate ≥0.95). Ten principal components (PCs) were calculated. Since the first 4 of the 10 top PCs were significantly associated with some musculoskeletal traits ( <em>p</em> &lt;.01), we consistently adjusted for PCs 1 through 4 in the SNP association analyses.</p><div id="__sec12" class="sec"><p></p><h4 id="__sec12title" class="inline">Statistical analysis </h4><p id="__p15" class="p p-first-last">Multivariable regression analysis was performed separately in men and women from each cohort (original and offspring) to obtain normalized residual phenotypes adjusted for concurrent covariates. Cohort- and gender-specific residuals were combined in ensuing analyses. Musculoskeletal traits were adjusted for age, age<sup>2</sup>, height, PC1 through PC4, and estrogenic status. Bone geometry traits additionally were adjusted for BMI.</p></div><div id="__sec13" class="sec"><p></p><h4 id="__sec13title" class="inline">Genome-wide association study </h4><p id="__p16" class="p p-first-last">We performed GWAS analyses using population-based additive linear mixed-effects (LME) models<sup>(<a href="#b47-jbmr-26-6-1261" rid="b47-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612720">47</a>)</sup> with 433,510 SNPs in men and women, separately. LME regression models adjust for within-family correlations in pedigrees of arbitrary sizes and varying degrees of relationship. Additive genetic associations were modeled using R-Kinship software (<a href="http://cran.r-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org</a>).</p></div><div id="__sec14" class="sec sec-last"><p></p><h4 id="__sec14title" class="inline">GWAS database mining </h4><p id="__p17" class="p p-first-last">We extracted association statistics for SNPs nominally significantly associated at α level less than 10<sup>−4</sup> with <em>any</em> trait from the list of 23 fracture-related phenotypes in LME analyses. There were 1109 autosomal SNPs in the initial data set. To avoid artifacts of clustering SNPs owing to patterns of linkage disequilibrium (LD), we filtered SNPs using the Tagger software (<a href="http://www.broadinstitute.org/mpg/tagger" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/tagger</a>)<sup>(<a href="#b48-jbmr-26-6-1261" rid="b48-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255612705">48</a>)</sup> to exclude SNPs in higher LD (pairwise genotypic <em>r</em><sup>2</sup> ≥0.5); we also used two additional thresholds: a liberal (<em>r</em><sup>2</sup> ≥0.2) and a conservative, with <em>r</em><sup>2</sup> ≥0.8. Since we randomly selected tag SNPs among a group of SNPs in LD at each threshold, there were several replicates for each data set. Randomly selected tag SNPs that are representative of a group at each LD threshold (<em>r</em><sup>2</sup> ≥0.2, 0.5, or 0.8) were included in the biclustering procedure to reduce the effect of correlation between the markers.</p></div></div><div id="__sec15" class="sec"><h3 id="__sec15title">Block-clustering model and fitting algorithm</h3><p id="__p18" class="p p-first-last">We used a Bayesian model framework for our biclustering (block-clustering) algorithm. The details of the statistical model and fitting algorithm are provided in the <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary materials</a>. We use a modified version of the evolutionary Monte Carlo (EMC) algorithm<sup>(<a href="#b49-jbmr-26-6-1261" rid="b49-jbmr-26-6-1261" class=" bibr popnode">49</a>)</sup> to fit the model. After monitoring the algorithm until it reaches convergence, we ultimately obtained the best estimate of the block clusters given the observed data. In the general case, when we can have more than one cluster in the data, we iteratively apply the algorithm repeatedly, adding block clusters one at a time after one cluster is found in the data until no more clusters can be found or at least one of the axes has been completely allocated to block clusters (no more rows or columns could be added to a cluster).</p></div><div id="__sec16" class="sec sec-last"><h3 id="__sec16title">Bioinformatic analysis</h3><div id="__sec17" class="sec sec-first"><p></p><h4 id="__sec17title" class="inline">Annotation </h4><p id="__p19" class="p p-first-last">The gene annotation was done based on the UCSC table browser for all RefSeq (hg18) genes. If the SNP was located outside known genic region, the nearest RefSeq gene was assigned as the gene annotation. The distance between nongenic SNPs and the nearest gene ranged from 32 to 1757 kbp.</p></div><div id="__sec18" class="sec"><p></p><h4 id="__sec18title" class="inline">Identification of enriched canonical pathway in clustered genes </h4><p id="__p20" class="p p-first-last">To determine whether any canonical pathway was enriched in the clustered SNPs, the clustered genes were imported into Ingenuity Pathways Analysis (IPA) software (Ingenuity Systems, Redwood City, CA, USA) to obtain networks for further analysis. We used IPA to identify the top enriched canonical pathways. The enriched canonical pathways were ranked by the <em>p</em> values of the Fisher’s exact test, which indicated the probabilities that the input genes (from the biclustered gene set) were associated with genes in the canonical pathways by chance.</p></div><div id="__sec19" class="sec sec-last"><p></p><h4 id="__sec19title" class="inline">Gene network inference via knowledge-based data mining </h4><p id="__p21" class="p p-first">We next analyzed biologic interactions among clustered genes using the IPA tool. The gene annotations from the biclustered SNPs were entered into the IPA analysis tool to construct the biologic networks of the clustered genes. Networks are generated from the biclustered gene set by maximizing the specific connectivity of the input genes, which represents their interconnectedness with each other relative to other molecules with which they are connected in Ingenuity’s knowledge database. Networks were limited to 35 molecules each to keep them at a functional size. The <em>p</em> value of probability for the genes forming a network was calculated with the right-tailed Fisher’s exact test based on the hypergeometric distribution. To gain biologic insights on whether this novel gene network is associated with any known canonical pathways, we further overlaid the novel gene network with canonical pathways using IPA.</p><p id="__p22" class="p p-last">Because musculoskeletal traits are sexually dimorphic and the genetic mechanisms are likely to be sex-specific,<sup>(<a href="#b50-jbmr-26-6-1261" rid="b50-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860798">50</a>)</sup> the analyses were performed separately in men and women. We focused on results from the 2038 women because osteoporotic fractures are more common in women, genetic regulation has been shown to differ by gender,<sup>(<a href="#b50-jbmr-26-6-1261" rid="b50-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860811">50</a>,<a href="#b51-jbmr-26-6-1261" rid="b51-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860799">51</a>)</sup> and our sample size for women was greater than for men.</p></div></div></div><div id="__sec20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec20title">Results</h2><div id="__sec21" class="sec sec-first"><h3 id="__sec21title">GWAS data set</h3><p id="__p23" class="p p-first-last">GWAS analysis by linear mixed-effects models resulted in 1109 unique SNPs associated with the 23 studied traits at the lowest <em>p</em> values ranging from <em>p</em> =1.45 ×10<sup>−5</sup> to <em>p</em> =4.15 ×10<sup>−8</sup> (<a href="/pmc/articles/PMC3312758/table/t1-jbmr-26-6-1261/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-jbmr-26-6-1261" rid-ob="ob-t1-jbmr-26-6-1261" co-legend-rid=""><span>Table 1</span></a>). The number of SNPs associated with each trait at a <em>p</em> value of less than 10<sup>−4</sup> ranged from 34 SNPs for femoral shaft width to 81 SNPs for metacarpal length. When filtered for LD, there were 668 SNPs at threshold <em>r</em><sup>2</sup> ≥0.5 (at thresholds <em>r</em><sup>2</sup> ≥0.8 and <em>r</em><sup>2</sup> ≥0.2, there were 722 and 589 SNPs, respectively).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1-jbmr-26-6-1261"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p46">Number of Significant Associated SNPs for the Individual Phenotypes (at α &lt;10<sup>−4</sup>)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237982324048" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">Trait</th><th rowspan="1" colspan="1" valign="bottom" align="center">No. of SNPs at α &lt;10<sup>−4</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest <em>p</em> value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">TOT BMD</td><td rowspan="1" colspan="1" valign="top" align="center">56</td><td rowspan="1" colspan="1" valign="top" align="center">3.84 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">FN BMD</td><td rowspan="1" colspan="1" valign="top" align="center">53</td><td rowspan="1" colspan="1" valign="top" align="center">3.86 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">TR BMD</td><td rowspan="1" colspan="1" valign="top" align="center">42</td><td rowspan="1" colspan="1" valign="top" align="center">1.95 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">LS BMD</td><td rowspan="1" colspan="1" valign="top" align="center">46</td><td rowspan="1" colspan="1" valign="top" align="center">1.45 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">BUA</td><td rowspan="1" colspan="1" valign="top" align="center">49</td><td rowspan="1" colspan="1" valign="top" align="center">5.25 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">SOS</td><td rowspan="1" colspan="1" valign="top" align="center">51</td><td rowspan="1" colspan="1" valign="top" align="center">3.43 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">7</td><td rowspan="1" colspan="1" valign="top" align="left">NSA</td><td rowspan="1" colspan="1" valign="top" align="center">43</td><td rowspan="1" colspan="1" valign="top" align="center">7.98 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">FNL</td><td rowspan="1" colspan="1" valign="top" align="center">44</td><td rowspan="1" colspan="1" valign="top" align="center">5.57 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">9</td><td rowspan="1" colspan="1" valign="top" align="left">NN WID</td><td rowspan="1" colspan="1" valign="top" align="center">58</td><td rowspan="1" colspan="1" valign="top" align="center">4.15 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">NN CSA</td><td rowspan="1" colspan="1" valign="top" align="center">55</td><td rowspan="1" colspan="1" valign="top" align="center">2.73 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">11</td><td rowspan="1" colspan="1" valign="top" align="left">NN BR</td><td rowspan="1" colspan="1" valign="top" align="center">49</td><td rowspan="1" colspan="1" valign="top" align="center">4.70 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">NN Z</td><td rowspan="1" colspan="1" valign="top" align="center">50</td><td rowspan="1" colspan="1" valign="top" align="center">8.31 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">S WID</td><td rowspan="1" colspan="1" valign="top" align="center">34</td><td rowspan="1" colspan="1" valign="top" align="center">9.71 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">14</td><td rowspan="1" colspan="1" valign="top" align="left">S Z</td><td rowspan="1" colspan="1" valign="top" align="center">45</td><td rowspan="1" colspan="1" valign="top" align="center">1.87 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">15</td><td rowspan="1" colspan="1" valign="top" align="left">S CSA</td><td rowspan="1" colspan="1" valign="top" align="center">70</td><td rowspan="1" colspan="1" valign="top" align="center">3.54 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">S BR</td><td rowspan="1" colspan="1" valign="top" align="center">56</td><td rowspan="1" colspan="1" valign="top" align="center">3.02 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">17</td><td rowspan="1" colspan="1" valign="top" align="left">Metacarpal length</td><td rowspan="1" colspan="1" valign="top" align="center">81</td><td rowspan="1" colspan="1" valign="top" align="center">4.54 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">18</td><td rowspan="1" colspan="1" valign="top" align="left">Metacarpal WID</td><td rowspan="1" colspan="1" valign="top" align="center">53</td><td rowspan="1" colspan="1" valign="top" align="center">3.14 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">19</td><td rowspan="1" colspan="1" valign="top" align="left">MCI</td><td rowspan="1" colspan="1" valign="top" align="center">65</td><td rowspan="1" colspan="1" valign="top" align="center">2.61 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">20</td><td rowspan="1" colspan="1" valign="top" align="left">MCT</td><td rowspan="1" colspan="1" valign="top" align="center">50</td><td rowspan="1" colspan="1" valign="top" align="center">3.47 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">21</td><td rowspan="1" colspan="1" valign="top" align="left">MZ</td><td rowspan="1" colspan="1" valign="top" align="center">62</td><td rowspan="1" colspan="1" valign="top" align="center">1.37 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">WBLM</td><td rowspan="1" colspan="1" valign="top" align="center">61</td><td rowspan="1" colspan="1" valign="top" align="center">2.18 × 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">23</td><td rowspan="1" colspan="1" valign="top" align="left">LLM</td><td rowspan="1" colspan="1" valign="top" align="center">65</td><td rowspan="1" colspan="1" valign="top" align="center">5.88 × 10<sup>−7</sup></td></tr></tbody></table></div><div id="largeobj_idm140237982324048" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312758/table/t1-jbmr-26-6-1261/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec22" class="sec"><h3 id="__sec22title">Block clustering of SNP data</h3><p id="__p24" class="p p-first">We first applied the block-clustering algorithm to the data set that consisted of regression coefficients from a single-SNP, single-trait association test with an LD cutoff at <em>r</em><sup>2</sup> ≥0.5 to avoid the artifact of clustering SNPs in linkage disequilibrium. The SNPs were chosen randomly among the three replicates for the specified LD cutoff. From <a href="/pmc/articles/PMC3312758/figure/f1-jbmr-26-6-1261/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1-jbmr-26-6-1261" rid-ob="ob-f1-jbmr-26-6-1261" co-legend-rid="lgnd_f1-jbmr-26-6-1261"><span>Fig. 1</span></a>, a clear cluster of traits and SNPs is observed, the SNPs being negatively correlated with a group of 10 traits: BMD (LS, FN, TR, and TOT), both ultrasound measures, and cross-sectional area (CSA) and section moduli (Z) of the femoral neck and shaft.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f1-jbmr-26-6-1261" co-legend-rid="lgnd_f1-jbmr-26-6-1261"><a href="/pmc/articles/PMC3312758/figure/f1-jbmr-26-6-1261/" target="figure" rid-figpopup="f1-jbmr-26-6-1261" rid-ob="ob-f1-jbmr-26-6-1261"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237982331264" class="figure"><a href="/pmc/articles/PMC3312758/figure/f1-jbmr-26-6-1261/" target="figure" rid-figpopup="f1-jbmr-26-6-1261" rid-ob="ob-f1-jbmr-26-6-1261"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312758_jbmr-26-6-1261f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is jbmr-26-6-1261f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261f1.jpg"></a></div><div id="largeobj_idm140237982331264" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312758/figure/f1-jbmr-26-6-1261/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f1-jbmr-26-6-1261"><div><a class="figpopup" href="/pmc/articles/PMC3312758/figure/f1-jbmr-26-6-1261/" target="figure" rid-figpopup="f1-jbmr-26-6-1261" rid-ob="ob-f1-jbmr-26-6-1261">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p44">Biclustering results. Heat map: yellow corresponds to the positive association; red, the negative. The resulting cluster is outlined in blue.</p></div></div></div><p id="__p25" class="p p-last">Next, in order to test the consistency and robustness of our algorithm to (1) initial choices of SNPs under the same LD cutoff and (2) different LD cutoffs, we applied the block-clustering algorithm to all the separate data sets under different LD cutoffs (at thresholds <em>r</em><sup>2</sup> ≥0.8 and <em>r</em><sup>2</sup> ≥0.2). The same 10 traits, namely, LS, FN, TR, and TOT BMD, both ultrasound measures, and CSA, and Z of femoral neck and shaft, were clustered in each data set. Similarly, the same SNPs were selected to be in the bicluster, almost irrespective of LD thresholds (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Fig. S1</a>). There was a substantial overlap between the sets of SNPs in clusters: From 64.1% to 78.5% of SNPs were shared between the SNP clusters.</p></div><div id="__sec23" class="sec"><h3 id="__sec23title">Phenomic analysis</h3><p id="__p26" class="p p-first-last">We further carried out phenomic analyses and gene network inference procedures on the results of block clustering. Since 10 bone traits were clustered with a distinct set of SNPs, this may imply that these 10 traits share homogeneous genetic architecture to a certain extent. To evaluate this hypothesis, we performed pairwise genetic correlation analyses of all tested musculoskeletal traits and examined whether these 10 clustered traits had stronger genetic correlations than the unclustered traits (see “<a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental Methods</a>” for details). Pairwise genetic correlations of all traits are provided in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table S1</a>. <a href="/pmc/articles/PMC3312758/table/t2-jbmr-26-6-1261/" target="table" class="fig-table-link figpopup" rid-figpopup="t2-jbmr-26-6-1261" rid-ob="ob-t2-jbmr-26-6-1261" co-legend-rid=""><span>Table 2</span></a> shows the mean magnitude of the genetic correlation of the analyses. As follows from the table, the mean magnitudes of the genetic correlation were significantly higher in tested clustered traits with other clustered traits (except for femoral neck Z) than with nonclustered traits. This observation confirmed that the block-clustering algorithm identified a cluster of genes that might underlie similarities between correlated traits.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t2-jbmr-26-6-1261"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p47">Difference in Mean Magnitude of Genetic Correlation (ρ<em><sub>g</sub></em>) Between (<em>A</em>) One Tested Clustered Trait and Other Clustered Traits (<em>n</em> =9) and (<em>B</em>) a Tested Clustered Trait and All Nonclustered Traits (<em>n</em> =13)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237980942640" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">ρ<em><sub>g</sub></em> with other clustered traits (<em>n</em> =9)</th><th rowspan="1" colspan="1" valign="bottom" align="center">ρ<sub>g</sub> with non-clustered<sup>a</sup> traits (<em>n</em> =13)</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"><hr></th><th rowspan="1" colspan="1" valign="bottom" align="left"><hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Clustered trait</th><th rowspan="1" colspan="1" valign="bottom" align="center">Mean SD</th><th rowspan="1" colspan="1" valign="bottom" align="center">Mean SD</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">BUA</td><td rowspan="1" colspan="1" valign="top" align="center">0.40 ± 0.25</td><td rowspan="1" colspan="1" valign="top" align="center">0.17 ± 0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FNBMD</td><td rowspan="1" colspan="1" valign="top" align="center">0.58 ± 0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.31 ± 0.16</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">LSBMD</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 ± 0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.18 ± 0.12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NNCSA</td><td rowspan="1" colspan="1" valign="top" align="center">0.59 ± 0.22</td><td rowspan="1" colspan="1" valign="top" align="center">0.17 ± 0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">NNZ</td><td rowspan="1" colspan="1" valign="top" align="center">0.37 ± 0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.25 ± 0.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SCSA</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 ± 0.23</td><td rowspan="1" colspan="1" valign="top" align="center">0.18 ± 0.18</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SOS</td><td rowspan="1" colspan="1" valign="top" align="center">0.42 ± 0.26</td><td rowspan="1" colspan="1" valign="top" align="center">0.20 ± 0.13</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SZ</td><td rowspan="1" colspan="1" valign="top" align="center">0.42 ± 0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.21 ± 0.14</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">TOTBMD</td><td rowspan="1" colspan="1" valign="top" align="center">0.61 ± 0.28</td><td rowspan="1" colspan="1" valign="top" align="center">0.31 ± 0.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">TRBMD</td><td rowspan="1" colspan="1" valign="top" align="center">0.55 ± 0.24</td><td rowspan="1" colspan="1" valign="top" align="center">0.28 ± 0.19</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Overall mean<sup>b</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.52 ± 0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.23 ± 0.06<sup>c</sup></td></tr></tbody></table></div><div id="largeobj_idm140237980942640" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312758/table/t2-jbmr-26-6-1261/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn1-jbmr-26-6-1261"><sup>a</sup>Nonclustered traits include NSA, FNL, NN WID, NN BR, S WID, S BR, metacarpal length, metacarpal WID, MCI, MCT, MZ, WBLM, and LLM.</div><div id="tfn2-jbmr-26-6-1261"><sup>b</sup>Mean was calculated using absolute value of ρ<em><sub>g</sub></em> regardless of the direction of association.</div><div id="tfn3-jbmr-26-6-1261"><sup>c</sup>Paired <em>t</em> test <em>p</em> &lt;.001.</div></div></div></div><div id="__sec24" class="sec"><h3 id="__sec24title">Canonical pathway identification and novel gene network inference</h3><p id="__p27" class="p p-first">To evaluate the potential associated canonical pathway in the SNPs cluster, we annotated the clustered SNP sets using the IPA method (hereafter, we refer to the gene annotation of the clustered SNPs as <em>clustered genes</em>). The top three enriched canonical pathways are listed in <a href="/pmc/articles/PMC3312758/table/t3-jbmr-26-6-1261/" target="table" class="fig-table-link figpopup" rid-figpopup="t3-jbmr-26-6-1261" rid-ob="ob-t3-jbmr-26-6-1261" co-legend-rid=""><span>Table 3</span></a> (the full list of the top enriched canonical pathways in clustered data sets is provided in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table S2</a>). The aldosterone signaling in epithelial cells; role of osteoblasts, osteoclasts, and chondrocytes in rheumatoid arthritis (RA); and Parkinson’s signaling pathways are the top three canonical pathways implicated in the analyzed data set (Fisher exact <em>p</em> ≤.01). Other significant pathways that associated with the data set are provided in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Table S3</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t3-jbmr-26-6-1261"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p51">Top Three Enriched Canonical Pathways in Clustered Data Set 05_01</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237986561632" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Canonical pathways</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>p</em> Value<sup>a</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Ratio<sup>b</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Molecules</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Aldosterone signaling in epithelial cells</td><td rowspan="1" colspan="1" valign="top" align="center">.003</td><td rowspan="1" colspan="1" valign="top" align="left">0.04 (4/95)</td><td rowspan="1" colspan="1" valign="top" align="left">PIK3R1, SCNN1B, NR3C2, PRKCH</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Role of osteoblasts, osteoclasts, and chondrocytes in rheumatoid arthritis</td><td rowspan="1" colspan="1" valign="top" align="center">.006</td><td rowspan="1" colspan="1" valign="top" align="left">0.03 (6/228)</td><td rowspan="1" colspan="1" valign="top" align="left">SFRP4, PIK3R1, WNT16, DKK2, BMP7, DKK1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Parkinson’s signaling</td><td rowspan="1" colspan="1" valign="top" align="center">.008</td><td rowspan="1" colspan="1" valign="top" align="left">0.12 (2/17)</td><td rowspan="1" colspan="1" valign="top" align="left">GPR37, PARK2</td></tr></tbody></table></div><div id="largeobj_idm140237986561632" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3312758/table/t3-jbmr-26-6-1261/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn4-jbmr-26-6-1261"><sup>a</sup>Fisher exact <em>p</em> value.</div><div id="tfn5-jbmr-26-6-1261"><sup>b</sup>Ratio =number of molecules in a given pathway that meet cutoff criteria/total number of molecules in that pathway (numbers shown in parentheses).</div></div></div><p id="__p28" class="p p-last">In addition to the fact that the clustered genes contained enriched canonical pathways, they also may represent a novel functional gene network. Therefore, we generated functional gene networks from the clustered genes using IPA. The most significant gene network [<em>26S proteasome, AGAP1, ALCAM, BMP7, COL1A2, collagen α1, collagen(s), cyclin E, DAB1, DKK1, ERK1/2, focal adhesion kinase, GPR37, Hsp70, LY96, MAP2K1/2, NFkB (complex), NKX3-1, PARK2, PI3K, PLAUR, POLR2A, PRKCH, PTPRZ1, PVR, RIOK3, RNA polymerase II, RORA, SCNN1B, SIRPA, SMYD3, SPRY2, TBX3, TNFAIP2, ubiquitin</em>) had <em>p</em> value of 1 × 10<sup>−44</sup> (<a href="/pmc/articles/PMC3312758/figure/f2-jbmr-26-6-1261/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-jbmr-26-6-1261" rid-ob="ob-f2-jbmr-26-6-1261" co-legend-rid="lgnd_f2-jbmr-26-6-1261"><span>Fig. 2</span></a>). This gene network is predicted by the IPA to participate in cellular assembly and organization, embryonic development, and cellular movement functions.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f2-jbmr-26-6-1261" co-legend-rid="lgnd_f2-jbmr-26-6-1261"><a href="/pmc/articles/PMC3312758/figure/f2-jbmr-26-6-1261/" target="figure" rid-figpopup="f2-jbmr-26-6-1261" rid-ob="ob-f2-jbmr-26-6-1261"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237982327792" class="figure"><a href="/pmc/articles/PMC3312758/figure/f2-jbmr-26-6-1261/" target="figure" rid-figpopup="f2-jbmr-26-6-1261" rid-ob="ob-f2-jbmr-26-6-1261"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3312758_jbmr-26-6-1261f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is jbmr-26-6-1261f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261f2.jpg"></a></div><div id="largeobj_idm140237982327792" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3312758/figure/f2-jbmr-26-6-1261/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f2-jbmr-26-6-1261"><div><a class="figpopup" href="/pmc/articles/PMC3312758/figure/f2-jbmr-26-6-1261/" target="figure" rid-figpopup="f2-jbmr-26-6-1261" rid-ob="ob-f2-jbmr-26-6-1261">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p45">Gene network predicted by the Ingenuity Pathways Analysis based on the clustered genes. Shaded genes are clustered genes identified using the biclustering algorithm.</p></div></div></div></div><div id="__sec25" class="sec sec-last"><h3 id="__sec25title">Association of bone-related canonical pathways with the functional gene network</h3><p id="__p29" class="p p-first-last">To gain insight on whether the functional networks were associated with any known bone-related canonical pathways, we overlaid the preceding functional network with canonical pathways and focused on the top canonical pathways that showed the greatest number of molecules within the gene network that also participated in the pathway. In the gene network presented in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplemental Fig. S2</a>, eight network molecules showed association with two canonical pathways. The top associated canonical pathways in this gene network are role of osteoblasts, osteoclasts, and chondrocytes in RA and molecular mechanisms of cancer.</p></div></div><div id="__sec26" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec26title">Discussion</h2><p id="__p30" class="p p-first">Osteoporosis is a complex trait with a relatively late age of onset, making it a challenging disease to study. In terms of quantitative genetics of complex traits, in the absence of a single valid endophenotype (an internal feature of a disease<sup>(<a href="#b52-jbmr-26-6-1261" rid="b52-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860807">52</a>)</sup>), pleiotropy among the traits is expected to assist phenotypic prioritization for a study. For example, to decide which is the best phenotype for a planned genetic study, it is imperative to be able to clearly distinguish signal from noise. This approach also results in increased power when mapping correlated phenotypes if they share pleiotropic genetic effects.<sup>(<a href="#b53-jbmr-26-6-1261" rid="b53-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860809">53</a>)</sup> Our novel Bayesian block-clustering algorithm adapted the technique of evolutionary Monte Carlo<sup>(<a href="#b49-jbmr-26-6-1261" rid="b49-jbmr-26-6-1261" class=" bibr popnode">49</a>)</sup> for high-dimensional SNP-phenotype data using genetic information from GWAS. Each partition represented a distinct cluster of SNPs that had similar effects over a particular set of traits. Application of this method to our data showed several SNP-trait connections. Thus we identified (1) 10 traits showing homogeneous genetic architecture and (2) the pleiotropic gene networks implicated in these clustered traits that were connected with several bone-related canonical pathways.</p><p id="__p31">In this study, we analyzed many musculoskeletal phenotypes because neither is a perfect proxy for osteoporotic fracture. For example, among the hip geometry indices, in addition to Z and CSA, there were femoral neck length, neck shaft angles, and estimates of mean cortical thickness, cross-sectional moment of inertia, and an index of bone structural instability (buckling ratio) that play a role in local susceptibility to fracture.<sup>(<a href="#b54-jbmr-26-6-1261" rid="b54-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860828">54</a>)</sup> Metacarpal bone geometry measures also had been used in epidemiologic research as well as in primary care to identify patients with osteoporosis.<sup>(<a href="#b55-jbmr-26-6-1261" rid="b55-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860832">55</a>–<a href="#b57-jbmr-26-6-1261" rid="b57-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860801">57</a>)</sup> We also included the lean mass measures because the loss of muscle mass with age (“sarcopenia”) is accompanied by a decrease in muscle strength and reduced loading of the skeleton.<sup>(<a href="#b58-jbmr-26-6-1261" rid="b58-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337475135">58</a>)</sup> Using the biclustering approach, we found a strong cluster of high-magnitude coefficients and/or lower <em>p</em> values for some bone traits associated with a subset of SNPs. These 10 traits included BMD of several skeletal locations, heel ultrasound measures, and cross-section area (CSA) and section moduli (Z) of the femoral neck and shaft. Biologically, these 10 traits represent a functional unit of “bone strength.” Indeed, BMD and heel ultrasound measures are among best predictors of fracture risk.<sup>(<a href="#b17-jbmr-26-6-1261" rid="b17-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860810">17</a>,<a href="#b20-jbmr-26-6-1261" rid="b20-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860804">20</a>)</sup> For long bones, Z is an index of bone bending strength, and CSA is an indicator of bone axial compression strength; both are important indicators of structural capacity to resist bending or compressive forces, respectively. Identification of this clustering of bone strength traits can be interpreted as a potential measure of success of our reverse-phenotyping efforts. Because high values of the coefficients were achieved for 10 traits clustering with a group of SNPs, we hypothesized that these traits should share a strong homogeneous genetic architecture. In agreement with the hypothesis, the mean magnitudes of genetic correlations among these 10 traits were higher than with the nonclustered traits, implying that the block-biclustering algorithm identified a cluster of genes that form a gene network (or genetic module) that underlies these traits.</p><p id="__p32">Our clustering was independent of LD threshold and random reshuffling of the associated SNPs; indeed, to reduce the effect of correlation between the markers on the cluster analysis, we constructed data sets with three thresholds of <em>r</em><sup>2</sup> ≥0.2, 0.5, and 0.8. Analyses of multiple replicates at the same LD threshold demonstrated the consistency of our algorithm and its relative robustness to LD threshold differences. The biclustering approach also highlighted some aspects of how the SNPs were selected. Although there was the expected variation in the SNPs selected to be in a given bicluster based on the different LD-based data sets, overall, the cluster was highly consistent with regard to random reshuffling. Whether the selected genes are truly associated with osteoporosis can only be shown empirically,<sup>(<a href="#b59-jbmr-26-6-1261" rid="b59-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860808">59</a>)</sup> and it is not possible to validate our approach to selection of “pleiotropic” candidate genes without further functional investigation.</p><p id="__p33">A closer investigation of the top connected networks generated by IPA provided the following interesting biologic insights: The most significant canonical pathway associated with the gene clusters was aldosterone signaling in epithelial cells. It has been well reported that aldosteronism affects bone metabolism; for instance, bone mineral and bone strength are lost in the presence of a high level of aldosterone.<sup>(<a href="#b60-jbmr-26-6-1261" rid="b60-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860806">60</a>,<a href="#b61-jbmr-26-6-1261" rid="b61-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860824">61</a>)</sup> Interestingly, two (<em>NR3C2</em> and <em>PIK3R1</em>) of four pathway genes from the cluster have been demonstrated to be involved in bone metabolism. <em>NR3C2</em> encodes mineralcortical receptor (MR) that is expressed in osteocytes, osteoblasts, and osteoclasts,<sup>(<a href="#b62-jbmr-26-6-1261" rid="b62-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860819">62</a>)</sup> and aldosterone activation of MR promotes osteoblastic differentiation.<sup>(<a href="#b63-jbmr-26-6-1261" rid="b63-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860818">63</a>)</sup>
<em>PIK3R1</em> encodes a subunit of phosphatidylinositol 3 kinase, which is an important mediator in both osteoblast and osteoclast differentiation.<sup>(<a href="#b64-jbmr-26-6-1261" rid="b64-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860816">64</a>–<a href="#b67-jbmr-26-6-1261" rid="b67-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860829">67</a>)</sup> Although no known function of <em>PRKCH</em> (protein kinase C, eta) and <em>SCNN1B</em> (epithelial sodium channel) is reported in any bone cell or bone diseases, gene expression of <em>PRKCH</em> in osteoblasts is correlated with other bone markers (ie, osteocalcin, bone sialoprotein, and alkaline phosphatase) at various differentiation and maturation stages,<sup>(<a href="#b68-jbmr-26-6-1261" rid="b68-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860815">68</a>)</sup> suggesting that <em>PRKCH</em> is a potential candidate gene that affects bone metabolism.</p><p id="__p34">The second most significant canonical pathway associated with the gene clusters was the pathway responsible for rheumatoid arthritis (RA). RA is a chronic inflammatory disease that leads to strucutural destruction of both bones and joints. The pathogenesis of RA in bone is an imbalance of the osteoblast-osteoclast axis driven by inflammatory processes that leads to excessive bone resorption by osteoclasts. Six genes (<em>SFRP4, WNT16, DKK2, BMP7, DKK1,</em> and <em>PIK3R1</em>) from the cluster are associated with this disease pathway. In fact, four among these six genes also belong to the Wnt/β-catenin signaling pathway,<sup>(<a href="#b69-jbmr-26-6-1261" rid="b69-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860830">69</a>–<a href="#b71-jbmr-26-6-1261" rid="b71-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860822">71</a>)</sup> except <em>BMP7</em>. BMP-7 plays a role in calcium regulation and bone homeostasis by inducing cartilage and bone formation. It is a Food and Drug Administration–approved ectopic bone inducer for clinical use in long bone open fractures, nonunion fractures, and spine fusion. Therefore, our findings confirmed two important pathways that underlie the etiology of osteoporosis and additionally identified <em>PRKCH</em> and <em>SCNN1B</em> as the novel potential candidate genes that affect bone metabolism.</p><p id="__p35">The role of the third most significant canonical pathway, the Parkinson’s signaling pathway, in bone strength is more puzzling. Recent epidemiologic studies pointed out this connection too. Thus, in community-dwelling older women with Parkinson’s disease (PD), compared with those without it, women with PD had 7.3% lower BMD.<sup>(<a href="#b72-jbmr-26-6-1261" rid="b72-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267447881">72</a>)</sup> Very similarly, older men with PD had a significantly greater rate of annualized total hip bone loss than those without parkinsonism.<sup>(<a href="#b73-jbmr-26-6-1261" rid="b73-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_267447880">73</a>)</sup> Dopamine transporter (Dat)–deficient mice manifest a low-bone-mass phenotype.<sup>(<a href="#b74-jbmr-26-6-1261" rid="b74-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860823">74</a>)</sup> We therefore believe that this tantalizing finding should be explored further, whether or not there is a dopaminergic-related connection between the diseases.</p><p id="__p36">To further understand signaling cascades engaged by the most significant gene networks, we overlaid the canonical pathway genes with the gene network generated from our data set. It <em>was</em> shown clearly that the top canonical pathways associated with this new network are related to the skeletal system (canonical pathway role of osteoblasts, osteoclasts, and chondrocytes in rheumatoid arthritis). This observation suggested that the gene network underlying bone strength is governed by multiple signaling pathways and therefore that any perturbation of these pathways may affect the gene network and hence bone strength. Future experimentation will be required to validate these bioinformatic observations.</p><p id="__p37">This study is one of the first forays into phenomics of the musculoskeletal system. Phenomics has a chance of changing how we view heritable diseases first by defining latent phenotypes that underlie genetically similar phenotype categories and second by revealing unexpected genetic links among disease entities.<sup>(<a href="#b75-jbmr-26-6-1261" rid="b75-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860812">75</a>)</sup> The incorporation of genetic data clearly improves the quality of the predictions over those derived solely from trait correlations,<sup>(<a href="#b76-jbmr-26-6-1261" rid="b76-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860825">76</a>)</sup> although phenotypic overlap is often a very good predictor of functional relatedness of the underlying genes.<sup>(<a href="#b77-jbmr-26-6-1261" rid="b77-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860797">77</a>)</sup> Thus van Driel and colleagues<sup>(<a href="#b78-jbmr-26-6-1261" rid="b78-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860814">78</a>)</sup> found that similarity between phenotypes did correlate positively with several measures of gene function, including relatedness at the level of protein sequence, protein motifs, functional annotation, and direct protein-protein interaction. We similarly hypothesized that the correlated phenotypes would share multiple associated genetic variants. Obviously, there is also an effect of the shared environmental factors or a confounding by an unmeasured—intermediate—variable.<sup>(<a href="#b79-jbmr-26-6-1261" rid="b79-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860800">79</a>)</sup> However, we recently found that the phenotypic and genetic correlations between a pair of traits rather than environmental correlation predict how many SNPs are shared between the two traits.<sup>(<a href="#b35-jbmr-26-6-1261" rid="b35-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860827">35</a>)</sup></p><p id="__p38">This study has several limitations. We used SNPs rather than genes as a source of GWAS mining and later for the gene-enrichment search. In rare cases where one SNP was located on two overlapping genes, we mapped the SNP to both genes; by this, we diminished a bias toward genes that were physically close. Otherwise, genes with better annotation would be more likely to be selected, which could result in a bias toward assortment of well-investigated genes rather than novel predicted or less well-characterized genes.<sup>(<a href="#b59-jbmr-26-6-1261" rid="b59-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860803">59</a>,<a href="#b80-jbmr-26-6-1261" rid="b80-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860802">80</a>)</sup> We focused on SNP-based statistics for clustering because there is not a widely agreed on and accepted theory for how to combine test statistics on multiple SNPs in a gene into one single <em>p</em> value (to obtain a maximum statistic to summarize association signals in that gene). When multiple distinct variants in the gene contribute to the overall association signal, the maximum statistic no longer may be the best statistic to capture such information.<sup>(<a href="#b81-jbmr-26-6-1261" rid="b81-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860826">81</a>)</sup> As noted, our model was restricted to non-overlapping clusters. We are cognizant that gene pathways overlap, as well as phenotypic modules, so that many gene networks and pathways unavoidably will share the same genes.<sup>(<a href="#b81-jbmr-26-6-1261" rid="b81-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860805">81</a>)</sup> However, the restriction of clusters to not overlap not only leads to a more rigorous model specification but also is more likely to pick out nonredundant clusters, thus overcoming one of the major drawbacks of other existing block-clustering algorithms. Because of the inclusion and exclusion criteria that, owing to redundant SNPs, might influence algorithm performance, we checked the consistency of our results by analyzing the data sets with several exclusion criteria based on LD between the SNPs. We did not apply a similar threshold to the phenotypes, whose selection was based on availability and epidemiologic evidence of their associations with fracture rather than on correlations among them. Because musculoskeletal traits are sexually dimorphic and likely to be influenced by sex-specific genetic mechanisms,<sup>(<a href="#b50-jbmr-26-6-1261" rid="b50-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860821">50</a>)</sup> the analyses were performed separately in men and women; however, we presented results from the women because the sample size was greater, and women experience more fractures than men. Results in men appear similar; however, our study has insufficient power for a formal comparison of the clustering results between the sexes. Finally, in this study we adjusted musculoskeletal traits for age, height (bone geometry for BMI), and estrogenic status; however, some potentially important factors, such as exercise, diet, and shared household among family members, were not accounted for.</p><p id="__p39" class="p p-last">In conclusion, we used GWAS results to examine pleiotropic relationships between osteoporosis-related traits. We found a distinct SNP cluster of negative coefficients of high magnitude associated with 10 bone-strength phenotypes (mainly BMD, ultrasound, and hip strength indices). The results of biclustering can be used for finding modules within and between the genome and the phenome,<sup>(<a href="#b9-jbmr-26-6-1261" rid="b9-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860817">9</a>,<a href="#b30-jbmr-26-6-1261" rid="b30-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860820">30</a>)</sup> data reduction, and choosing the best phenotypes in the pathway to fracture. Accurate understanding of disease phenotype can lead to success in identifying genetic determinants for complex disease.<sup>(<a href="#b59-jbmr-26-6-1261" rid="b59-jbmr-26-6-1261" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_255860831">59</a>)</sup> Similarly, the SNP clusters pointed out the presence of potentially pleiotropic genetic pathways involved in bone strength. Targeting a gene network common for many aspects of a disease may be a favorable approach for therapeutic agent development or predicting the adverse effects of a drug. This study thus presents a novel approach to the analysis of GWAS data and provides directions for further refining of the osteoporosis phenotype and genetic pathways on which to focus in functional studies of osteoporosis.</p></div><div id="__sec27" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec27title">Supplementary Data</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><div class="sup-box half_rhythm" id="d33e1028"><a href="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261-s1.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(48K, doc)</sup></div></div><div class="sec suppmat" id="SD2"><div class="sup-box half_rhythm" id="d33e1030"><a href="/pmc/articles/PMC3312758/bin/jbmr-26-6-1261-s2.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(561K, doc)</sup></div></div></div><div id="idm140237981616480" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237981616480title">Acknowledgments</h2><div class="sec"><p id="__p40">The study was funded by grants from the US National Institute for Arthritis, Musculoskeletal and Skin Diseases and National Institute on Aging (R01 AR/AG 41398, R01 AR 050066, and R01 AR 057118), as well as from the National Human Genome Research Institute (R03 HG004946-01). The Framingham Heart Study of the National Institutes of Health and Boston University School of Medicine was supported by the National Heart, Lung, and Blood Institute’s Framingham Heart Study (N01-HC-25195) and its contract with Affymetrix, Inc., for genotyping services (N02-HL-6-4278). A portion of this research was conducted using the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at Boston University School of Medicine and Boston Medical Center.</p></div></div><div id="idm140237981614720" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237981614720title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn1-jbmr-26-6-1261"></p><p id="__p41" class="p p-first"><strong>Disclosures</strong></p><p id="__p42" class="p p-last">All the authors state that they have no conflicts of interest.</p><p></p><p class="fn sec" id="fn2-jbmr-26-6-1261"></p><p id="__p43" class="p p-first-last">Additional Supporting Information may be found in the online version of this article.</p><p></p></div></div><div id="idm140237981612240" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237981612240title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1-jbmr-26-6-1261">1. <span class="element-citation">Rivadeneira F, Styrkarsdottir U, Estrada K, et al.  Twenty bone-mineral-density loci identified by large-scale meta-analysis of genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1199–1206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2783489/">PMC free article</a>]</span> [<a href="/pubmed/19801982" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Twenty+bone-mineral-density+loci+identified+by+large-scale+meta-analysis+of+genome-wide+association+studies&amp;author=F+Rivadeneira&amp;author=U+Styrkarsdottir&amp;author=K+Estrada&amp;volume=41&amp;publication_year=2009&amp;pages=1199-1206&amp;pmid=19801982&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2-jbmr-26-6-1261">2. <span class="element-citation">Xiong DH, Liu XG, Guo YF, et al.  Genome-wide association and follow-up replication studies identified <em>ADAMTS18</em> and <em>TGFBR3</em> as bone mass candidate genes in different ethnic groups. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:388–398.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2667986/">PMC free article</a>]</span> [<a href="/pubmed/19249006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+and+follow-up+replication+studies+identified+ADAMTS18+and+TGFBR3+as+bone+mass+candidate+genes+in+different+ethnic+groups&amp;author=DH+Xiong&amp;author=XG+Liu&amp;author=YF+Guo&amp;volume=84&amp;publication_year=2009&amp;pages=388-398&amp;pmid=19249006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3-jbmr-26-6-1261">3. <span class="element-citation">Kung AW, Xiao SM, Cherny S, et al.  Association of <em>JAG1</em> with bone mineral density and osteoporotic fractures: a genome-wide association study and follow-up replication studies. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">86</span>:229–239.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820171/">PMC free article</a>]</span> [<a href="/pubmed/20096396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Association+of+JAG1+with+bone+mineral+density+and+osteoporotic+fractures:+a+genome-wide+association+study+and+follow-up+replication+studies&amp;author=AW+Kung&amp;author=SM+Xiao&amp;author=S+Cherny&amp;volume=86&amp;publication_year=2010&amp;pages=229-239&amp;pmid=20096396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4-jbmr-26-6-1261">4. <span class="element-citation">Kiel DP, Demissie S, Dupuis J, Lunetta KL, Murabito JM, Karasik D. Genome-wide association with bone mass and geometry in the Framingham Heart Study. <span><span class="ref-journal">BMC Med Genet. </span>2007;<span class="ref-vol">8</span>:S14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995606/">PMC free article</a>]</span> [<a href="/pubmed/17903296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genet&amp;title=Genome-wide+association+with+bone+mass+and+geometry+in+the+Framingham+Heart+Study&amp;author=DP+Kiel&amp;author=S+Demissie&amp;author=J+Dupuis&amp;author=KL+Lunetta&amp;author=JM+Murabito&amp;volume=8&amp;publication_year=2007&amp;pages=S14&amp;pmid=17903296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5-jbmr-26-6-1261">5. <span class="element-citation">Liu YZ, Wilson SG, Wang L, et al.  Identification of <em>PLCL1</em> gene for hip bone size variation in females in a genome-wide association study. <span><span class="ref-journal">PLoS One. </span>2008;<span class="ref-vol">3</span>:e3160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2522269/">PMC free article</a>]</span> [<a href="/pubmed/18776929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Identification+of+PLCL1+gene+for+hip+bone+size+variation+in+females+in+a+genome-wide+association+study&amp;author=YZ+Liu&amp;author=SG+Wilson&amp;author=L+Wang&amp;volume=3&amp;publication_year=2008&amp;pages=e3160&amp;pmid=18776929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6-jbmr-26-6-1261">6. <span class="element-citation">Liu XG, Tan LJ, Lei SF, et al.  Genome-wide association and replication studies identified TRHR as an important gene for lean body mass. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:418–423.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668008/">PMC free article</a>]</span> [<a href="/pubmed/19268274" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genome-wide+association+and+replication+studies+identified+TRHR+as+an+important+gene+for+lean+body+mass&amp;author=XG+Liu&amp;author=LJ+Tan&amp;author=SF+Lei&amp;volume=84&amp;publication_year=2009&amp;pages=418-423&amp;pmid=19268274&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7-jbmr-26-6-1261">7. <span class="element-citation">Goh KI, Cusick ME, Valle D, Childs B, Vidal M, Barabasi AL. The human disease network. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2007;<span class="ref-vol">104</span>:8685–8690.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1885563/">PMC free article</a>]</span> [<a href="/pubmed/17502601" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+human+disease+network&amp;author=KI+Goh&amp;author=ME+Cusick&amp;author=D+Valle&amp;author=B+Childs&amp;author=M+Vidal&amp;volume=104&amp;publication_year=2007&amp;pages=8685-8690&amp;pmid=17502601&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8-jbmr-26-6-1261">8. <span class="element-citation">Wachi S, Yoneda K, Wu R. Interactome-transcriptome analysis reveals the high centrality of genes differentially expressed in lung cancer tissues. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:4205–4208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4631381/">PMC free article</a>]</span> [<a href="/pubmed/16188928" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Interactome-transcriptome+analysis+reveals+the+high+centrality+of+genes+differentially+expressed+in+lung+cancer+tissues&amp;author=S+Wachi&amp;author=K+Yoneda&amp;author=R+Wu&amp;volume=21&amp;publication_year=2005&amp;pages=4205-4208&amp;pmid=16188928&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9-jbmr-26-6-1261">9. <span class="element-citation">Schulze TG, McMahon FJ. Defining the phenotype in human genetic studies: forward genetics and reverse phenotyping. <span><span class="ref-journal">Hum Hered. </span>2004;<span class="ref-vol">58</span>:131–138.</span> [<a href="/pubmed/15812169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Hered&amp;title=Defining+the+phenotype+in+human+genetic+studies:+forward+genetics+and+reverse+phenotyping&amp;author=TG+Schulze&amp;author=FJ+McMahon&amp;volume=58&amp;publication_year=2004&amp;pages=131-138&amp;pmid=15812169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10-jbmr-26-6-1261">10. <span class="element-citation">Harris TB, Launer LJ, Eiriksdottir G, et al.  Age, Gene/Environment Susceptibility-Reykjavik Study: multidisciplinary applied phenomics. <span><span class="ref-journal">Am J Epidemiol. </span>2007;<span class="ref-vol">165</span>:1076–1087.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2723948/">PMC free article</a>]</span> [<a href="/pubmed/17351290" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Age,+Gene/Environment+Susceptibility-Reykjavik+Study:+multidisciplinary+applied+phenomics&amp;author=TB+Harris&amp;author=LJ+Launer&amp;author=G+Eiriksdottir&amp;volume=165&amp;publication_year=2007&amp;pages=1076-1087&amp;pmid=17351290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11-jbmr-26-6-1261">11. <span class="element-citation">Lussier YA, Liu Y. Computational approaches to phenotyping: high-throughput phenomics. <span><span class="ref-journal">Proc Am Thorac Soc. </span>2007;<span class="ref-vol">4</span>:18–25.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2647609/">PMC free article</a>]</span> [<a href="/pubmed/17202287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Am+Thorac+Soc&amp;title=Computational+approaches+to+phenotyping:+high-throughput+phenomics&amp;author=YA+Lussier&amp;author=Y+Liu&amp;volume=4&amp;publication_year=2007&amp;pages=18-25&amp;pmid=17202287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12-jbmr-26-6-1261">12. <span class="element-citation">Johnell O, Kanis JA. An estimate of the worldwide prevalence and disability associated with osteoporotic fractures. <span><span class="ref-journal">Osteoporos Int. </span>2006;<span class="ref-vol">17</span>:1726–1733.</span> [<a href="/pubmed/16983459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=An+estimate+of+the+worldwide+prevalence+and+disability+associated+with+osteoporotic+fractures&amp;author=O+Johnell&amp;author=JA+Kanis&amp;volume=17&amp;publication_year=2006&amp;pages=1726-1733&amp;pmid=16983459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13-jbmr-26-6-1261">13. <span class="element-citation">Deng HW, Mahaney MC, Williams JT, et al.  Relevance of the genes for bone mass variation to susceptibility to osteoporotic fractures and its implications to gene search for complex human diseases. <span><span class="ref-journal">Genet Epidemiol. </span>2002;<span class="ref-vol">22</span>:12–25.</span> [<a href="/pubmed/11754470" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Relevance+of+the+genes+for+bone+mass+variation+to+susceptibility+to+osteoporotic+fractures+and+its+implications+to+gene+search+for+complex+human+diseases&amp;author=HW+Deng&amp;author=MC+Mahaney&amp;author=JT+Williams&amp;volume=22&amp;publication_year=2002&amp;pages=12-25&amp;pmid=11754470&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14-jbmr-26-6-1261">14. <span class="element-citation">Andrew T, Antioniades L, Scurrah KJ, Macgregor AJ, Spector TD. Risk of wrist fracture in women is heritable and is influenced by genes that are largely independent of those influencing BMD. <span><span class="ref-journal">J Bone Miner Res. </span>2005;<span class="ref-vol">20</span>:67–74.</span> [<a href="/pubmed/15619671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Risk+of+wrist+fracture+in+women+is+heritable+and+is+influenced+by+genes+that+are+largely+independent+of+those+influencing+BMD&amp;author=T+Andrew&amp;author=L+Antioniades&amp;author=KJ+Scurrah&amp;author=AJ+Macgregor&amp;author=TD+Spector&amp;volume=20&amp;publication_year=2005&amp;pages=67-74&amp;pmid=15619671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15-jbmr-26-6-1261">15. <span class="element-citation">van Meurs JB, Schuit SC, Weel AE, et al.  Association of 5’ estrogen receptor alpha gene polymorphisms with bone mineral density, vertebral bone area and fracture risk. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:1745–1754.</span> [<a href="/pubmed/12837697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Association+of+5’+estrogen+receptor+alpha+gene+polymorphisms+with+bone+mineral+density,+vertebral+bone+area+and+fracture+risk&amp;author=JB+van+Meurs&amp;author=SC+Schuit&amp;author=AE+Weel&amp;volume=12&amp;publication_year=2003&amp;pages=1745-1754&amp;pmid=12837697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16-jbmr-26-6-1261">16. <span class="element-citation">Faulkner KG, Wacker WK, Barden HS, et al.  Femur strength index predicts hip fracture independent of bone density and hip axis length. <span><span class="ref-journal">Osteoporos Int. </span>2006;<span class="ref-vol">17</span>:593–599.</span> [<a href="/pubmed/16447009" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=Femur+strength+index+predicts+hip+fracture+independent+of+bone+density+and+hip+axis+length&amp;author=KG+Faulkner&amp;author=WK+Wacker&amp;author=HS+Barden&amp;volume=17&amp;publication_year=2006&amp;pages=593-599&amp;pmid=16447009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17-jbmr-26-6-1261">17. <span class="element-citation">Gluer CC, Hans D. How to use ultrasound for risk assessment: a need for defining strategies. <span><span class="ref-journal">Osteoporos Int. </span>1999;<span class="ref-vol">9</span>:193–195.</span> [<a href="/pubmed/10450405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=How+to+use+ultrasound+for+risk+assessment:+a+need+for+defining+strategies&amp;author=CC+Gluer&amp;author=D+Hans&amp;volume=9&amp;publication_year=1999&amp;pages=193-195&amp;pmid=10450405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18-jbmr-26-6-1261">18. <span class="element-citation">Marshall D, Johnell O, Wedel H. Meta-analysis of how well measures of bone mineral density predict occurrence of osteoporotic fractures [see comments] <span><span class="ref-journal">BMJ. </span>1996;<span class="ref-vol">312</span>:1254–1259.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2351094/">PMC free article</a>]</span> [<a href="/pubmed/8634613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ&amp;title=Meta-analysis+of+how+well+measures+of+bone+mineral+density+predict+occurrence+of+osteoporotic+fractures+[see+comments]&amp;author=D+Marshall&amp;author=O+Johnell&amp;author=H+Wedel&amp;volume=312&amp;publication_year=1996&amp;pages=1254-1259&amp;pmid=8634613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19-jbmr-26-6-1261">19. <span class="element-citation">Kanis JA, Johnell O, Oden A, et al.  The use of multiple sites for the diagnosis of osteoporosis. <span><span class="ref-journal">Osteoporos Int. </span>2006;<span class="ref-vol">17</span>:527–534.</span> [<a href="/pubmed/16402164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=The+use+of+multiple+sites+for+the+diagnosis+of+osteoporosis&amp;author=JA+Kanis&amp;author=O+Johnell&amp;author=A+Oden&amp;volume=17&amp;publication_year=2006&amp;pages=527-534&amp;pmid=16402164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20-jbmr-26-6-1261">20. <span class="element-citation">Lu Y, Genant HK, Shepherd J, et al.  Classification of osteoporosis based on bone mineral densities. <span><span class="ref-journal">J Bone Miner Res. </span>2001;<span class="ref-vol">16</span>:901–910.</span> [<a href="/pubmed/11341335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Classification+of+osteoporosis+based+on+bone+mineral+densities&amp;author=Y+Lu&amp;author=HK+Genant&amp;author=J+Shepherd&amp;volume=16&amp;publication_year=2001&amp;pages=901-910&amp;pmid=11341335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21-jbmr-26-6-1261">21. <span class="element-citation">Kaptoge S, Beck TJ, Reeve J, et al.  Prediction of incident hip fracture risk by femur geometry variables measured by hip structural analysis in the study of osteoporotic fractures. <span><span class="ref-journal">J Bone Miner Res. </span>2008;<span class="ref-vol">23</span>:1892–1904.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2686919/">PMC free article</a>]</span> [<a href="/pubmed/18684092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Prediction+of+incident+hip+fracture+risk+by+femur+geometry+variables+measured+by+hip+structural+analysis+in+the+study+of+osteoporotic+fractures&amp;author=S+Kaptoge&amp;author=TJ+Beck&amp;author=J+Reeve&amp;volume=23&amp;publication_year=2008&amp;pages=1892-1904&amp;pmid=18684092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22-jbmr-26-6-1261">22. <span class="element-citation">Jepsen KJ. Systems analysis of bone. <span><span class="ref-journal">WIREs Syst Biol Med. </span>2009;<span class="ref-vol">14</span>:73–88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2790199/">PMC free article</a>]</span> [<a href="/pubmed/20046860" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=WIREs+Syst+Biol+Med&amp;title=Systems+analysis+of+bone&amp;author=KJ+Jepsen&amp;volume=14&amp;publication_year=2009&amp;pages=73-88&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23-jbmr-26-6-1261">23. <span class="element-citation">Tseng GC, Wong WH. Tight clustering: a resampling-based approach for identifying stable and tight patterns in data. <span><span class="ref-journal">Biometrics. </span>2005;<span class="ref-vol">61</span>:10–16.</span> [<a href="/pubmed/15737073" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Tight+clustering:+a+resampling-based+approach+for+identifying+stable+and+tight+patterns+in+data&amp;author=GC+Tseng&amp;author=WH+Wong&amp;volume=61&amp;publication_year=2005&amp;pages=10-16&amp;pmid=15737073&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24-jbmr-26-6-1261">24. <span class="element-citation">Hartigan JA. Direct clustering of a data matrix. <span><span class="ref-journal">J Am Stat Assoc. </span>1972;<span class="ref-vol">67</span>:123–129.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Stat+Assoc&amp;title=Direct+clustering+of+a+data+matrix&amp;author=JA+Hartigan&amp;volume=67&amp;publication_year=1972&amp;pages=123-129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25-jbmr-26-6-1261">25. <span class="element-citation">Cheng Y, Church GM. Biclustering of expression data. <span><span class="ref-journal">Proc Int Conf Intell Syst Mol Biol. </span>2000;<span class="ref-vol">8</span>:93–103.</span> [<a href="/pubmed/10977070" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Int+Conf+Intell+Syst+Mol+Biol&amp;title=Biclustering+of+expression+data&amp;author=Y+Cheng&amp;author=GM+Church&amp;volume=8&amp;publication_year=2000&amp;pages=93-103&amp;pmid=10977070&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26-jbmr-26-6-1261">26. <span class="element-citation">Kluger Y, Basri R, Chang JT, Gerstein M. Spectral biclustering of microarray data: coclustering genes and conditions. <span><span class="ref-journal">Genome Res. </span>2003;<span class="ref-vol">13</span>:703–716.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC430175/">PMC free article</a>]</span> [<a href="/pubmed/12671006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Spectral+biclustering+of+microarray+data:+coclustering+genes+and+conditions&amp;author=Y+Kluger&amp;author=R+Basri&amp;author=JT+Chang&amp;author=M+Gerstein&amp;volume=13&amp;publication_year=2003&amp;pages=703-716&amp;pmid=12671006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b27-jbmr-26-6-1261">27. <span class="element-citation">Madeira SC, Oliveira AL. Biclustering algorithms for biological data analysis: a survey. <span><span class="ref-journal">IEEE/ACM Trans Comput Biol Bioinform. </span>2004;<span class="ref-vol">1</span>:24–45.</span> [<a href="/pubmed/17048406" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=IEEE/ACM+Trans+Comput+Biol+Bioinform&amp;title=Biclustering+algorithms+for+biological+data+analysis:+a+survey&amp;author=SC+Madeira&amp;author=AL+Oliveira&amp;volume=1&amp;publication_year=2004&amp;pages=24-45&amp;pmid=17048406&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b28-jbmr-26-6-1261">28. <span class="element-citation">Gu J, Liu J. Bayesian biclustering of gene expression data. <span><span class="ref-journal">BMC Genomics. </span>2008;<span class="ref-vol">9</span>( Suppl 1):S4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2386069/">PMC free article</a>]</span> [<a href="/pubmed/18366617" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics&amp;title=Bayesian+biclustering+of+gene+expression+data&amp;author=J+Gu&amp;author=J+Liu&amp;volume=9&amp;issue=+Suppl+1&amp;publication_year=2008&amp;pages=S4&amp;pmid=18366617&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b29-jbmr-26-6-1261">29. <span class="element-citation">Gupta M.  <span class="ref-journal">Two-way clustering of data matrices using a hierarchical Bayesian model Technical report Dept of Biostatistics.</span> Boston, MA: Boston University; 2010.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Two-way+clustering+of+data+matrices+using+a+hierarchical+Bayesian+model+Technical+report+Dept+of+Biostatistics&amp;author=M+Gupta&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b30-jbmr-26-6-1261">30. <span class="element-citation">Segal E, Kim SK. The modular era of functional genomics. <span><span class="ref-journal">Genome Biol. </span>2003;<span class="ref-vol">4</span>:317.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC156581/">PMC free article</a>]</span> [<a href="/pubmed/12734004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=The+modular+era+of+functional+genomics&amp;author=E+Segal&amp;author=SK+Kim&amp;volume=4&amp;publication_year=2003&amp;pages=317&amp;pmid=12734004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b31-jbmr-26-6-1261">31. <span class="element-citation">Hannan MT, Felson DT, Dawson-Hughes B, et al.  Risk factors for longitudinal bone loss in elderly men and women: the Framingham Osteoporosis Study. <span><span class="ref-journal">J Bone Miner Res. </span>2000;<span class="ref-vol">15</span>:710–720.</span> [<a href="/pubmed/10780863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Risk+factors+for+longitudinal+bone+loss+in+elderly+men+and+women:+the+Framingham+Osteoporosis+Study&amp;author=MT+Hannan&amp;author=DT+Felson&amp;author=B+Dawson-Hughes&amp;volume=15&amp;publication_year=2000&amp;pages=710-720&amp;pmid=10780863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b32-jbmr-26-6-1261">32. <span class="element-citation">Karasik D, Myers RH, Cupples LA, et al.  Genome screen for quantitative trait loci contributing to normal variation in bone mineral density: the Framingham Study. <span><span class="ref-journal">J Bone Miner Res. </span>2002;<span class="ref-vol">17</span>:1718–1727.</span> [<a href="/pubmed/12211443" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Genome+screen+for+quantitative+trait+loci+contributing+to+normal+variation+in+bone+mineral+density:+the+Framingham+Study&amp;author=D+Karasik&amp;author=RH+Myers&amp;author=LA+Cupples&amp;volume=17&amp;publication_year=2002&amp;pages=1718-1727&amp;pmid=12211443&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b33-jbmr-26-6-1261">33. <span class="element-citation">Demissie S, Dupuis J, Cupples LA, Beck TJ, Kiel DP, Karasik D. Proximal hip geometry is linked to several chromosomal regions: Genome-wide linkage results from the Framingham Osteoporosis Study. <span><span class="ref-journal">Bone. </span>2007;<span class="ref-vol">40</span>:743–750.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1952180/">PMC free article</a>]</span> [<a href="/pubmed/17079199" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Proximal+hip+geometry+is+linked+to+several+chromosomal+regions:+Genome-wide+linkage+results+from+the+Framingham+Osteoporosis+Study&amp;author=S+Demissie&amp;author=J+Dupuis&amp;author=LA+Cupples&amp;author=TJ+Beck&amp;author=DP+Kiel&amp;volume=40&amp;publication_year=2007&amp;pages=743-750&amp;pmid=17079199&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b34-jbmr-26-6-1261">34. <span class="element-citation">Karasik D, Zhou Y, Cupples LA, Hannan MT, Kiel DP, Demissie S. Bivariate genome-wide linkage analysis of femoral bone traits and leg lean mass: Framingham study. <span><span class="ref-journal">J Bone Miner Res. </span>2009;<span class="ref-vol">24</span>:710–718.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2659513/">PMC free article</a>]</span> [<a href="/pubmed/19063671" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Bivariate+genome-wide+linkage+analysis+of+femoral+bone+traits+and+leg+lean+mass:+Framingham+study&amp;author=D+Karasik&amp;author=Y+Zhou&amp;author=LA+Cupples&amp;author=MT+Hannan&amp;author=DP+Kiel&amp;volume=24&amp;publication_year=2009&amp;pages=710-718&amp;pmid=19063671&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b35-jbmr-26-6-1261">35. <span class="element-citation">Karasik D, Hsu YH, Zhou Y, Cupples LA, Kiel DP, Demissie S. Genome-wide pleiotropy of osteoporosis-related phenotypes: The Framingham Study. <span><span class="ref-journal">J Bone Miner Res. </span>2010;<span class="ref-vol">25</span>:1555–1563.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3153998/">PMC free article</a>]</span> [<a href="/pubmed/20200953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Genome-wide+pleiotropy+of+osteoporosis-related+phenotypes:+The+Framingham+Study&amp;author=D+Karasik&amp;author=YH+Hsu&amp;author=Y+Zhou&amp;author=LA+Cupples&amp;author=DP+Kiel&amp;volume=25&amp;publication_year=2010&amp;pages=1555-1563&amp;pmid=20200953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b36-jbmr-26-6-1261">36. <span class="element-citation">McLean RR, Hannan MT, Epstein BE, et al.  Elderly cohort study subjects unable to return for follow-up have lower bone mass than those who can return. <span><span class="ref-journal">Am J Epidemiol. </span>2000;<span class="ref-vol">151</span>:689–692.</span> [<a href="/pubmed/10752796" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Elderly+cohort+study+subjects+unable+to+return+for+follow-up+have+lower+bone+mass+than+those+who+can+return&amp;author=RR+McLean&amp;author=MT+Hannan&amp;author=BE+Epstein&amp;volume=151&amp;publication_year=2000&amp;pages=689-692&amp;pmid=10752796&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b37-jbmr-26-6-1261">37. <span class="element-citation">Khoo BC, Beck TJ, Qiao QH, et al.  In vivo short-term precision of hip structure analysis variables in comparison with bone mineral density using paired dual-energy X-ray absorptiometry scans from multi-center clinical trials. <span><span class="ref-journal">Bone. </span>2005;<span class="ref-vol">37</span>:112–121.</span> [<a href="/pubmed/15869917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=In+vivo+short-term+precision+of+hip+structure+analysis+variables+in+comparison+with+bone+mineral+density+using+paired+dual-energy+X-ray+absorptiometry+scans+from+multi-center+clinical+trials&amp;author=BC+Khoo&amp;author=TJ+Beck&amp;author=QH+Qiao&amp;volume=37&amp;publication_year=2005&amp;pages=112-121&amp;pmid=15869917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b38-jbmr-26-6-1261">38. <span class="element-citation">Felson D, Couropmitree N, Chaisson C, et al.  Evidence for Mendelian gene in a segregation analysis of generalized radiographic osteoarthritis. <span><span class="ref-journal">Arthritis Rheum. </span>1998;<span class="ref-vol">41</span>:1064–1071.</span> [<a href="/pubmed/9627016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Evidence+for+Mendelian+gene+in+a+segregation+analysis+of+generalized+radiographic+osteoarthritis&amp;author=D+Felson&amp;author=N+Couropmitree&amp;author=C+Chaisson&amp;volume=41&amp;publication_year=1998&amp;pages=1064-1071&amp;pmid=9627016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b39-jbmr-26-6-1261">39. <span class="element-citation">Demissie S, Cupples L, Myers R, Aliabadi P, Levy D, Felson D. Genome Scan for Quantity of Hand Osteoarthritis: The Framingham Study. <span><span class="ref-journal">Arthritis Rheum. </span>2002;<span class="ref-vol">46</span>:946–952.</span> [<a href="/pubmed/11953971" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Genome+Scan+for+Quantity+of+Hand+Osteoarthritis:+The+Framingham+Study&amp;author=S+Demissie&amp;author=L+Cupples&amp;author=R+Myers&amp;author=P+Aliabadi&amp;author=D+Levy&amp;volume=46&amp;publication_year=2002&amp;pages=946-952&amp;pmid=11953971&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b40-jbmr-26-6-1261">40. <span class="element-citation">Karasik D, Shimabuku NA, Zhou Y, et al.  A genome wide linkage scan of metacarpal size and geometry in the Framingham Study. <span><span class="ref-journal">Am J Hum Biol. </span>2008;<span class="ref-vol">20</span>:663–670.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2574919/">PMC free article</a>]</span> [<a href="/pubmed/18449921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Biol&amp;title=A+genome+wide+linkage+scan+of+metacarpal+size+and+geometry+in+the+Framingham+Study&amp;author=D+Karasik&amp;author=NA+Shimabuku&amp;author=Y+Zhou&amp;volume=20&amp;publication_year=2008&amp;pages=663-670&amp;pmid=18449921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b41-jbmr-26-6-1261">41. <span class="element-citation">Gluer CC, Blake G, Lu Y, Blunt BA, Jergas M, Genant HK. Accurate assessment of precision errors: how to measure the reproducibility of bone densitometry techniques. <span><span class="ref-journal">Osteoporos Int. </span>1995;<span class="ref-vol">5</span>:262–270.</span> [<a href="/pubmed/7492865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=Accurate+assessment+of+precision+errors:+how+to+measure+the+reproducibility+of+bone+densitometry+techniques&amp;author=CC+Gluer&amp;author=G+Blake&amp;author=Y+Lu&amp;author=BA+Blunt&amp;author=M+Jergas&amp;volume=5&amp;publication_year=1995&amp;pages=262-270&amp;pmid=7492865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b42-jbmr-26-6-1261">42. <span class="element-citation">Garn S, Rohmann C, Wagner B, Sascoli W. Continuing bone growth throughout life: a general phenomenon. <span><span class="ref-journal">Am J Phys Anthropol. </span>1967;<span class="ref-vol">26</span>:313–317.</span> [<a href="/pubmed/6035855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Continuing+bone+growth+throughout+life:+a+general+phenomenon&amp;author=S+Garn&amp;author=C+Rohmann&amp;author=B+Wagner&amp;author=W+Sascoli&amp;volume=26&amp;publication_year=1967&amp;pages=313-317&amp;pmid=6035855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b43-jbmr-26-6-1261">43. <span class="element-citation">Karasik D, Ginsburg E, Livshits G, Pavlovsky O, Kobyliansky E. Evidence on major gene control of cortical bone loss in human population. <span><span class="ref-journal">Genet Epidemiol. </span>2000;<span class="ref-vol">19</span>:410–421.</span> [<a href="/pubmed/11108649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Evidence+on+major+gene+control+of+cortical+bone+loss+in+human+population&amp;author=D+Karasik&amp;author=E+Ginsburg&amp;author=G+Livshits&amp;author=O+Pavlovsky&amp;author=E+Kobyliansky&amp;volume=19&amp;publication_year=2000&amp;pages=410-421&amp;pmid=11108649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b44-jbmr-26-6-1261">44. <span class="element-citation">Plato CC, Garruto RM, Yanagihara RT, et al.  Cortical bone loss and measurements of the second metacarpal bone. I. Comparisons between adult Guamanian Chamorros and American Caucasians. <span><span class="ref-journal">Am J Phys Anthropol. </span>1982;<span class="ref-vol">59</span>:461–465.</span> [<a href="/pubmed/7165042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Phys+Anthropol&amp;title=Cortical+bone+loss+and+measurements+of+the+second+metacarpal+bone.+I.+Comparisons+between+adult+Guamanian+Chamorros+and+American+Caucasians&amp;author=CC+Plato&amp;author=RM+Garruto&amp;author=RT+Yanagihara&amp;volume=59&amp;publication_year=1982&amp;pages=461-465&amp;pmid=7165042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b45-jbmr-26-6-1261">45. <span class="element-citation">Kannel WB, Sorlie P. Some health benefits of physical activity. The Framingham Study. <span><span class="ref-journal">Arch Intern Med. </span>1979;<span class="ref-vol">139</span>:857–861.</span> [<a href="/pubmed/464698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=Some+health+benefits+of+physical+activity.+The+Framingham+Study&amp;author=WB+Kannel&amp;author=P+Sorlie&amp;volume=139&amp;publication_year=1979&amp;pages=857-861&amp;pmid=464698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b46-jbmr-26-6-1261">46. <span class="element-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b47-jbmr-26-6-1261">47. <span class="element-citation">Boerwinkle E, Chakraborty R, Sing CF. The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods. <span><span class="ref-journal">Ann Hum Genet. </span>1986;<span class="ref-vol">50</span>( Pt 2):181–194.</span> [<a href="/pubmed/3435047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=The+use+of+measured+genotype+information+in+the+analysis+of+quantitative+phenotypes+in+man.+I.+Models+and+analytical+methods&amp;author=E+Boerwinkle&amp;author=R+Chakraborty&amp;author=CF+Sing&amp;volume=50&amp;issue=+Pt+2&amp;publication_year=1986&amp;pages=181-194&amp;pmid=3435047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b48-jbmr-26-6-1261">48. <span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b49-jbmr-26-6-1261">49. <span class="element-citation">Liang F, Wong WH. Evolutionary Monte Carlo: applications to model sampling and change point problem. <span><span class="ref-journal">Statistica Sinica. </span>2000;<span class="ref-vol">10</span>:317–342.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Statistica+Sinica&amp;title=Evolutionary+Monte+Carlo:+applications+to+model+sampling+and+change+point+problem&amp;author=F+Liang&amp;author=WH+Wong&amp;volume=10&amp;publication_year=2000&amp;pages=317-342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b50-jbmr-26-6-1261">50. <span class="element-citation">Karasik D, Ferrari SL. Contribution of gender-specific genetic factors to osteoporosis risk. <span><span class="ref-journal">Ann Hum Genet. </span>2008;<span class="ref-vol">72</span>( Pt 5):696–714.</span> [<a href="/pubmed/18485052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Contribution+of+gender-specific+genetic+factors+to+osteoporosis+risk&amp;author=D+Karasik&amp;author=SL+Ferrari&amp;volume=72&amp;issue=+Pt+5&amp;publication_year=2008&amp;pages=696-714&amp;pmid=18485052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b51-jbmr-26-6-1261">51. <span class="element-citation">Karasik D, Cupples LA, Hannan MT, Kiel DP. Age, gender, and body mass effects on quantitative trait loci for bone mineral density: the Framingham study. <span><span class="ref-journal">Bone. </span>2003;<span class="ref-vol">33</span>:308–316.</span> [<a href="/pubmed/13678771" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Age,+gender,+and+body+mass+effects+on+quantitative+trait+loci+for+bone+mineral+density:+the+Framingham+study&amp;author=D+Karasik&amp;author=LA+Cupples&amp;author=MT+Hannan&amp;author=DP+Kiel&amp;volume=33&amp;publication_year=2003&amp;pages=308-316&amp;pmid=13678771&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b52-jbmr-26-6-1261">52. <span class="element-citation">Almasy L, Blangero J. Endophenotypes as quantitative risk factors for psychiatric disease: rationale and study design. <span><span class="ref-journal">Am J Med Genet. </span>2001;<span class="ref-vol">105</span>:42–44.</span> [<a href="/pubmed/11424994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Endophenotypes+as+quantitative+risk+factors+for+psychiatric+disease:+rationale+and+study+design&amp;author=L+Almasy&amp;author=J+Blangero&amp;volume=105&amp;publication_year=2001&amp;pages=42-44&amp;pmid=11424994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b53-jbmr-26-6-1261">53. <span class="element-citation">Allison DB, Thiel B, St Jean P, Elston RC, Infante MC, Schork NJ. Multiple phenotype modeling in gene-mapping studies of quantitative traits: power advantages. <span><span class="ref-journal">Am J Hum Genet. </span>1998;<span class="ref-vol">63</span>:1190–1201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377471/">PMC free article</a>]</span> [<a href="/pubmed/9758596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Multiple+phenotype+modeling+in+gene-mapping+studies+of+quantitative+traits:+power+advantages&amp;author=DB+Allison&amp;author=B+Thiel&amp;author=P+St+Jean&amp;author=RC+Elston&amp;author=MC+Infante&amp;volume=63&amp;publication_year=1998&amp;pages=1190-1201&amp;pmid=9758596&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b54-jbmr-26-6-1261">54. <span class="element-citation">Rivadeneira F, Zillikens MC, De Laet CE, et al.  Femoral neck BMD is a strong predictor of hip fracture susceptibility in elderly men and women because it detects cortical bone instability: the Rotterdam Study. <span><span class="ref-journal">J Bone Miner Res. </span>2007;<span class="ref-vol">22</span>:1781–1790.</span> [<a href="/pubmed/17638578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Femoral+neck+BMD+is+a+strong+predictor+of+hip+fracture+susceptibility+in+elderly+men+and+women+because+it+detects+cortical+bone+instability:+the+Rotterdam+Study&amp;author=F+Rivadeneira&amp;author=MC+Zillikens&amp;author=CE+De+Laet&amp;volume=22&amp;publication_year=2007&amp;pages=1781-1790&amp;pmid=17638578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b55-jbmr-26-6-1261">55. <span class="element-citation">Adami S, Zamberlan N, Gatti D, et al.  Computed radiographic absorptiometry and morphometry in the assessment of postmenopausal bone loss. <span><span class="ref-journal">Osteoporos Int. </span>1996;<span class="ref-vol">6</span>:8–13.</span> [<a href="/pubmed/8845604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=Computed+radiographic+absorptiometry+and+morphometry+in+the+assessment+of+postmenopausal+bone+loss&amp;author=S+Adami&amp;author=N+Zamberlan&amp;author=D+Gatti&amp;volume=6&amp;publication_year=1996&amp;pages=8-13&amp;pmid=8845604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b56-jbmr-26-6-1261">56. <span class="element-citation">Wishart JM, Horowitz M, Bochner M, Need AG, Nordin BE. Relationships between metacarpal morphometry, forearm and vertebral bone density and fractures in postmenopausal women. <span><span class="ref-journal">Br J Radiol. </span>1993;<span class="ref-vol">66</span>:435–440.</span> [<a href="/pubmed/8319065" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Radiol&amp;title=Relationships+between+metacarpal+morphometry,+forearm+and+vertebral+bone+density+and+fractures+in+postmenopausal+women&amp;author=JM+Wishart&amp;author=M+Horowitz&amp;author=M+Bochner&amp;author=AG+Need&amp;author=BE+Nordin&amp;volume=66&amp;publication_year=1993&amp;pages=435-440&amp;pmid=8319065&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b57-jbmr-26-6-1261">57. <span class="element-citation">Kiel DP, Hannan MT, Broe KE, Felson DT, Cupples LA. Can metacarpal cortical area predict the occurrence of hip fracture in women and men over 3 decades of follow-up? Results from the Framingham Osteoporosis Study. <span><span class="ref-journal">J Bone Miner Res. </span>2001;<span class="ref-vol">16</span>:2260–2266.</span> [<a href="/pubmed/11760840" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Can+metacarpal+cortical+area+predict+the+occurrence+of+hip+fracture+in+women+and+men+over+3+decades+of+follow-up?+Results+from+the+Framingham+Osteoporosis+Study&amp;author=DP+Kiel&amp;author=MT+Hannan&amp;author=KE+Broe&amp;author=DT+Felson&amp;author=LA+Cupples&amp;volume=16&amp;publication_year=2001&amp;pages=2260-2266&amp;pmid=11760840&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b58-jbmr-26-6-1261">58. <span class="element-citation">Rubin J, Rubin C, Jacobs CR. Molecular pathways mediating mechanical signaling in bone. <span><span class="ref-journal">Gene. </span>2006;<span class="ref-vol">367</span>:1–16.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3687520/">PMC free article</a>]</span> [<a href="/pubmed/16361069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Molecular+pathways+mediating+mechanical+signaling+in+bone&amp;author=J+Rubin&amp;author=C+Rubin&amp;author=CR+Jacobs&amp;volume=367&amp;publication_year=2006&amp;pages=1-16&amp;pmid=16361069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b59-jbmr-26-6-1261">59. <span class="element-citation">Tiffin N, Okpechi I, Perez-Iratxeta C, Andrade-Navarro MA, Ramesar R. Prioritization of candidate disease genes for metabolic syndrome by computational analysis of its defining phenotypes. <span><span class="ref-journal">Physiol Genomics. </span>2008;<span class="ref-vol">35</span>:55–64.</span> [<a href="/pubmed/18612082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Prioritization+of+candidate+disease+genes+for+metabolic+syndrome+by+computational+analysis+of+its+defining+phenotypes&amp;author=N+Tiffin&amp;author=I+Okpechi&amp;author=C+Perez-Iratxeta&amp;author=MA+Andrade-Navarro&amp;author=R+Ramesar&amp;volume=35&amp;publication_year=2008&amp;pages=55-64&amp;pmid=18612082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b60-jbmr-26-6-1261">60. <span class="element-citation">Law PH, Sun Y, Bhattacharya SK, Chhokar VS, Weber KT. Diuretics and bone loss in rats with aldosteronism. <span><span class="ref-journal">J Am Coll Cardiol. </span>2005;<span class="ref-vol">46</span>:142–146.</span> [<a href="/pubmed/15992648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Coll+Cardiol&amp;title=Diuretics+and+bone+loss+in+rats+with+aldosteronism&amp;author=PH+Law&amp;author=Y+Sun&amp;author=SK+Bhattacharya&amp;author=VS+Chhokar&amp;author=KT+Weber&amp;volume=46&amp;publication_year=2005&amp;pages=142-146&amp;pmid=15992648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b61-jbmr-26-6-1261">61. <span class="element-citation">Chhokar VS, Sun Y, Bhattacharya SK, et al.  Loss of bone minerals and strength in rats with aldosteronism. <span><span class="ref-journal">Am J Physiol Heart Circ Physiol. </span>2004;<span class="ref-vol">287</span>:H2023–2026.</span> [<a href="/pubmed/15475529" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Physiol+Heart+Circ+Physiol&amp;title=Loss+of+bone+minerals+and+strength+in+rats+with+aldosteronism&amp;author=VS+Chhokar&amp;author=Y+Sun&amp;author=SK+Bhattacharya&amp;volume=287&amp;publication_year=2004&amp;pages=H2023-2026&amp;pmid=15475529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b62-jbmr-26-6-1261">62. <span class="element-citation">Beavan S, Horner A, Bord S, Ireland D, Compston J. Colocalization of glucocorticoid and mineralocorticoid receptors in human bone. <span><span class="ref-journal">J Bone Miner Res. </span>2001;<span class="ref-vol">16</span>:1496–1504.</span> [<a href="/pubmed/11499872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Colocalization+of+glucocorticoid+and+mineralocorticoid+receptors+in+human+bone&amp;author=S+Beavan&amp;author=A+Horner&amp;author=S+Bord&amp;author=D+Ireland&amp;author=J+Compston&amp;volume=16&amp;publication_year=2001&amp;pages=1496-1504&amp;pmid=11499872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b63-jbmr-26-6-1261">63. <span class="element-citation">Jaffe IZ, Tintut Y, Newfell BG, Demer LL, Mendelsohn ME. Mineralocorticoid receptor activation promotes vascular cell calcification. <span><span class="ref-journal">Arterioscler Thromb Vasc Biol. </span>2007;<span class="ref-vol">27</span>:799–805.</span> [<a href="/pubmed/17234727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb+Vasc+Biol&amp;title=Mineralocorticoid+receptor+activation+promotes+vascular+cell+calcification&amp;author=IZ+Jaffe&amp;author=Y+Tintut&amp;author=BG+Newfell&amp;author=LL+Demer&amp;author=ME+Mendelsohn&amp;volume=27&amp;publication_year=2007&amp;pages=799-805&amp;pmid=17234727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b64-jbmr-26-6-1261">64. <span class="element-citation">Mandal CC, Ghosh Choudhury G, Ghosh-Choudhury N. Phosphatidylinositol 3 kinase/Akt signal relay cooperates with smad in bone morphogenetic protein-2-induced colony stimulating factor-1 (CSF-1) expression and osteoclast differentiation. <span><span class="ref-journal">Endocrinology. </span>2009;<span class="ref-vol">150</span>:4989–4998.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2775973/">PMC free article</a>]</span> [<a href="/pubmed/19819979" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=Phosphatidylinositol+3+kinase/Akt+signal+relay+cooperates+with+smad+in+bone+morphogenetic+protein-2-induced+colony+stimulating+factor-1+(CSF-1)+expression+and+osteoclast+differentiation&amp;author=CC+Mandal&amp;author=G+Ghosh+Choudhury&amp;author=N+Ghosh-Choudhury&amp;volume=150&amp;publication_year=2009&amp;pages=4989-4998&amp;pmid=19819979&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b65-jbmr-26-6-1261">65. <span class="element-citation">Adapala NS, Barbe MF, Langdon WY, Tsygankov AY, Sanjay A. Cbl-phosphatidylinositol 3 kinase interaction differentially regulates macrophage colony-stimulating factor-mediated osteoclast survival and cytoskeletal reorganization. <span><span class="ref-journal">Ann N Y Acad Sci. </span>2010;<span class="ref-vol">1192</span>:376–384.</span> [<a href="/pubmed/20392263" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+N+Y+Acad+Sci&amp;title=Cbl-phosphatidylinositol+3+kinase+interaction+differentially+regulates+macrophage+colony-stimulating+factor-mediated+osteoclast+survival+and+cytoskeletal+reorganization&amp;author=NS+Adapala&amp;author=MF+Barbe&amp;author=WY+Langdon&amp;author=AY+Tsygankov&amp;author=A+Sanjay&amp;volume=1192&amp;publication_year=2010&amp;pages=376-384&amp;pmid=20392263&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b66-jbmr-26-6-1261">66. <span class="element-citation">Ling L, Dombrowski C, Foong KM, et al.  Synergism between Wnt3a and heparin enhances osteogenesis via a phosphoinositide 3-kinase/Akt/RUNX2 pathway. <span><span class="ref-journal">J Biol Chem. </span>2010;<span class="ref-vol">285</span>:26233–26244.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2924036/">PMC free article</a>]</span> [<a href="/pubmed/20547765" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Synergism+between+Wnt3a+and+heparin+enhances+osteogenesis+via+a+phosphoinositide+3-kinase/Akt/RUNX2+pathway&amp;author=L+Ling&amp;author=C+Dombrowski&amp;author=KM+Foong&amp;volume=285&amp;publication_year=2010&amp;pages=26233-26244&amp;pmid=20547765&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b67-jbmr-26-6-1261">67. <span class="element-citation">Miraoui H, Oudina K, Petite H, Tanimoto Y, Moriyama K, Marie PJ. Fibroblast growth factor receptor 2 promotes osteogenic differentiation in mesenchymal cells via ERK1/2 and protein kinase C signaling. <span><span class="ref-journal">J Biol Chem. </span>2009;<span class="ref-vol">284</span>:4897–4904.</span> [<a href="/pubmed/19117954" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Fibroblast+growth+factor+receptor+2+promotes+osteogenic+differentiation+in+mesenchymal+cells+via+ERK1/2+and+protein+kinase+C+signaling&amp;author=H+Miraoui&amp;author=K+Oudina&amp;author=H+Petite&amp;author=Y+Tanimoto&amp;author=K+Moriyama&amp;volume=284&amp;publication_year=2009&amp;pages=4897-4904&amp;pmid=19117954&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b68-jbmr-26-6-1261">68. <span class="element-citation">Lampasso JD, Chen W, Marzec N. The expression profile of PKC isoforms during MC3T3-E1 differentiation. <span><span class="ref-journal">Int J Mol Med. </span>2006;<span class="ref-vol">17</span>:1125–1131.</span> [<a href="/pubmed/16685425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Mol+Med&amp;title=The+expression+profile+of+PKC+isoforms+during+MC3T3-E1+differentiation&amp;author=JD+Lampasso&amp;author=W+Chen&amp;author=N+Marzec&amp;volume=17&amp;publication_year=2006&amp;pages=1125-1131&amp;pmid=16685425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b69-jbmr-26-6-1261">69. <span class="element-citation">Kamiya N, Kobayashi T, Mochida Y, et al.  Wnt Inhibitors Dkk1 and Sost are downstream targets of BMP signaling through the type IA receptor (BMPRIA) in osteoblasts. <span><span class="ref-journal">J Bone Miner Res. </span>2010;<span class="ref-vol">25</span>:200–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3153381/">PMC free article</a>]</span> [<a href="/pubmed/19874086" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Wnt+Inhibitors+Dkk1+and+Sost+are+downstream+targets+of+BMP+signaling+through+the+type+IA+receptor+(BMPRIA)+in+osteoblasts&amp;author=N+Kamiya&amp;author=T+Kobayashi&amp;author=Y+Mochida&amp;volume=25&amp;publication_year=2010&amp;pages=200-210&amp;pmid=19874086&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b70-jbmr-26-6-1261">70. <span class="element-citation">van der Horst G, van der Werf SM, Farih-Sips H, van Bezooijen RL, Lowik CW, Karperien M. Downregulation of Wnt signaling by increased expression of Dickkopf-1 and -2 is a prerequisite for late-stage osteoblast differentiation of KS483cells. <span><span class="ref-journal">J Bone Miner Res. </span>2005;<span class="ref-vol">20</span>:1867–1877.</span> [<a href="/pubmed/16160745" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Downregulation+of+Wnt+signaling+by+increased+expression+of+Dickkopf-1+and+-2+is+a+prerequisite+for+late-stage+osteoblast+differentiation+of+KS483cells&amp;author=G+van+der+Horst&amp;author=SM+van+der+Werf&amp;author=H+Farih-Sips&amp;author=RL+van+Bezooijen&amp;author=CW+Lowik&amp;volume=20&amp;publication_year=2005&amp;pages=1867-1877&amp;pmid=16160745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b71-jbmr-26-6-1261">71. <span class="element-citation">Li X, Zhang Y, Kang H, et al.  Sclerostin binds to LRP5/6 and antagonizes canonical Wnt signaling. <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:19883–19887.</span> [<a href="/pubmed/15778503" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Sclerostin+binds+to+LRP5/6+and+antagonizes+canonical+Wnt+signaling&amp;author=X+Li&amp;author=Y+Zhang&amp;author=H+Kang&amp;volume=280&amp;publication_year=2005&amp;pages=19883-19887&amp;pmid=15778503&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b72-jbmr-26-6-1261">72. <span class="element-citation">Schneider J, Fink H, Ewing S, Ensrud K, Cummings S. The association of Parkinson’s disease with bone mineral density and fracture in older women. <span><span class="ref-journal">Osteoporosis Int. </span>2008;<span class="ref-vol">19</span>:1093–1097.</span> [<a href="/pubmed/18301855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporosis+Int&amp;title=The+association+of+Parkinson’s+disease+with+bone+mineral+density+and+fracture+in+older+women&amp;author=J+Schneider&amp;author=H+Fink&amp;author=S+Ewing&amp;author=K+Ensrud&amp;author=S+Cummings&amp;volume=19&amp;publication_year=2008&amp;pages=1093-1097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b73-jbmr-26-6-1261">73. <span class="element-citation">Fink H, Kuskowski M, Taylor B, Schousboe J, Orwoll E, Ensrud K. Association of Parkinson’s disease with accelerated bone loss, fractures and mortality in older men: the Osteoporotic Fractures in Men (MrOS) study. <span><span class="ref-journal">Osteoporosis Int. </span>2008;<span class="ref-vol">19</span>:1277–1282.</span> [<a href="/pubmed/18301854" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporosis+Int&amp;title=Association+of+Parkinson’s+disease+with+accelerated+bone+loss,+fractures+and+mortality+in+older+men:+the+Osteoporotic+Fractures+in+Men+(MrOS)+study&amp;author=H+Fink&amp;author=M+Kuskowski&amp;author=B+Taylor&amp;author=J+Schousboe&amp;author=E+Orwoll&amp;volume=19&amp;publication_year=2008&amp;pages=1277-1282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b74-jbmr-26-6-1261">74. <span class="element-citation">Bliziotes MM, Eshleman AJ, Zhang XW, Wiren KM. Neurotransmitter action in osteoblasts: expression of a functional system for serotonin receptor activation and reuptake. <span><span class="ref-journal">Bone. </span>2001;<span class="ref-vol">29</span>:477–486.</span> [<a href="/pubmed/11704501" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Neurotransmitter+action+in+osteoblasts:+expression+of+a+functional+system+for+serotonin+receptor+activation+and+reuptake&amp;author=MM+Bliziotes&amp;author=AJ+Eshleman&amp;author=XW+Zhang&amp;author=KM+Wiren&amp;volume=29&amp;publication_year=2001&amp;pages=477-486&amp;pmid=11704501&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b75-jbmr-26-6-1261">75. <span class="element-citation">Oti M, Huynen MA, Brunner HG. Phenome connections. <span><span class="ref-journal">Trends Genet. </span>2008;<span class="ref-vol">24</span>:103–106.</span> [<a href="/pubmed/18243400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Phenome+connections&amp;author=M+Oti&amp;author=MA+Huynen&amp;author=HG+Brunner&amp;volume=24&amp;publication_year=2008&amp;pages=103-106&amp;pmid=18243400&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b76-jbmr-26-6-1261">76. <span class="element-citation">Zhu J, Wiener MC, Zhang C, et al.  Increasing the power to detect causal associations by combining genotypic and expression data in segregating populations. <span><span class="ref-journal">PLoS Comput Biol. </span>2007;<span class="ref-vol">3</span>:e69.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1851982/">PMC free article</a>]</span> [<a href="/pubmed/17432931" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput+Biol&amp;title=Increasing+the+power+to+detect+causal+associations+by+combining+genotypic+and+expression+data+in+segregating+populations&amp;author=J+Zhu&amp;author=MC+Wiener&amp;author=C+Zhang&amp;volume=3&amp;publication_year=2007&amp;pages=e69&amp;pmid=17432931&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b77-jbmr-26-6-1261">77. <span class="element-citation">Brunner HG, van Driel MA. From syndrome families to functional genomics. <span><span class="ref-journal">Nat Rev Genet. </span>2004;<span class="ref-vol">5</span>:545–551.</span> [<a href="/pubmed/15211356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=From+syndrome+families+to+functional+genomics&amp;author=HG+Brunner&amp;author=MA+van+Driel&amp;volume=5&amp;publication_year=2004&amp;pages=545-551&amp;pmid=15211356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b78-jbmr-26-6-1261">78. <span class="element-citation">van Driel MA, Bruggeman J, Vriend G, Brunner HG, Leunissen JA. A text-mining analysis of the human phenome. <span><span class="ref-journal">Eur J Hum Genet. </span>2006;<span class="ref-vol">14</span>:535–542.</span> [<a href="/pubmed/16493445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=A+text-mining+analysis+of+the+human+phenome&amp;author=MA+van+Driel&amp;author=J+Bruggeman&amp;author=G+Vriend&amp;author=HG+Brunner&amp;author=JA+Leunissen&amp;volume=14&amp;publication_year=2006&amp;pages=535-542&amp;pmid=16493445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b79-jbmr-26-6-1261">79. <span class="element-citation">Drenos F, Talmud PJ, Casas JP, et al.  Integrated associations of genotypes with multiple blood biomarkers linked to coronary heart disease risk. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:2305–2316.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2685759/">PMC free article</a>]</span> [<a href="/pubmed/19336475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Integrated+associations+of+genotypes+with+multiple+blood+biomarkers+linked+to+coronary+heart+disease+risk&amp;author=F+Drenos&amp;author=PJ+Talmud&amp;author=JP+Casas&amp;volume=18&amp;publication_year=2009&amp;pages=2305-2316&amp;pmid=19336475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b80-jbmr-26-6-1261">80. <span class="element-citation">Elbers CC, van Eijk KR, Franke L, et al.  Using genome-wide pathway analysis to unravel the etiology of complex diseases. <span><span class="ref-journal">Genet Epidemiol. </span>2009;<span class="ref-vol">33</span>:419–431.</span> [<a href="/pubmed/19235186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Using+genome-wide+pathway+analysis+to+unravel+the+etiology+of+complex+diseases&amp;author=CC+Elbers&amp;author=KR+van+Eijk&amp;author=L+Franke&amp;volume=33&amp;publication_year=2009&amp;pages=419-431&amp;pmid=19235186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b81-jbmr-26-6-1261">81. <span class="element-citation">Wang K, Li M, Bucan M. Pathway-Based Approaches for Analysis of Genomewide Association Studies. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:1278–1283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2276352/">PMC free article</a>]</span> [<a href="/pubmed/17966091" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Pathway-Based+Approaches+for+Analysis+of+Genomewide+Association+Studies&amp;author=K+Wang&amp;author=M+Li&amp;author=M+Bucan&amp;volume=81&amp;publication_year=2007&amp;pages=1278-1283&amp;pmid=17966091&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3312758&amp;issue-id=200324&amp;journal-id=656&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140237986253920title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Methods</a></li><li><a href="#__sec20title">Results</a></li><li><a href="#__sec26title">Discussion</a></li><li><a href="#__sec27title">Supplementary Data</a></li><li><a href="#idm140237981616480title">Acknowledgments</a></li><li><a href="#idm140237981614720title">Footnotes</a></li><li><a href="#idm140237981612240title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of Bone and Mineral Research</span> are provided here courtesy of <strong>American Society for Bone and Mineral Research</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3312758/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3312758/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3312758/pdf/jbmr-26-6-1261.pdf">PDF (11M)</a></li> | <li><a href="#" data-citationid="PMC3312758" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312758%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312758%2F&amp;text=Identification%20of%20Homogeneous%20Genetic%20Architecture%20of%20Multiple%20Genetically%20Correlated%20Traits%20by%20Block%20Clustering%20of%20Genome-Wide%20Associations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312758%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21611967" data-aiid="3312758" data-aid="3312758" data-iid="200324" data-domainid="656" data-domain="jbmr" data-accid="PMC3312758" data-md5="e2bae8cc3ddc5509e8cc76522e449d0a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:06:01-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89AE52DD52028100000000086E0560&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3312758%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jbmr&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3312758/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>