<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="tmh"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4280456/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A Genome-Wide Screen for Large-Effect Alloimmunization Susceptibility Loci among Red Blood Cell Transfusion Recipients with Sickle Cell Disease</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280456/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Transfusion Medicine and Hemotherapy"><meta name="citation_title" content="A Genome-Wide Screen for Large-Effect Alloimmunization Susceptibility Loci among Red Blood Cell Transfusion Recipients with Sickle Cell Disease"><meta name="citation_authors" content="Neil A. Hanchard, Joann M. Moulds, John W. Belmont, Alice Chen"><meta name="citation_date" content="November 2014"><meta name="citation_issue" content="6"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="453"><meta name="citation_doi" content="10.1159/000369079"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4280456/?report=abstract"><meta name="citation_pmid" content="25670933"><meta name="DC.Title" content="A Genome-Wide Screen for Large-Effect Alloimmunization Susceptibility Loci among Red Blood Cell Transfusion Recipients with Sickle Cell Disease"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Karger Publishers"><meta name="DC.Contributor" content="Neil A. Hanchard"><meta name="DC.Contributor" content="Joann M. Moulds"><meta name="DC.Contributor" content="John W. Belmont"><meta name="DC.Contributor" content="Alice Chen"><meta name="DC.Date" content="2014 Nov"><meta name="DC.Identifier" content="10.1159/000369079"><meta name="DC.Language" content="en"><meta property="og:title" content="A Genome-Wide Screen for Large-Effect Alloimmunization Susceptibility Loci among Red Blood Cell Transfusion Recipients with Sickle Cell Disease"><meta property="og:type" content="article"><meta property="og:description" content="A selective susceptibility of certain individuals to form multiple alloantibodies in response to red cell transfusion is well-recognized in clinical practice, and is a particular problem in persons with sickle cell disease (SCD). The reason for this differential ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4280456/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-tmh.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4280456/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4280456/pdf/tmh-0041-0453.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B3522DD57642100000000093F0595.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-3"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280456%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="clinvar" class="last">ClinVar</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/1263/">Transfus Med Hemother</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/247087/">v.41(6); 2014 Nov</a></li><li class="accid">PMC4280456</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-tmh.gif" alt="Logo of tmh" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,100,75" alt="Karger" title="Karger" href="http://www.karger.com/" ref="reftype=publisher&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="poly" coords="210,0,500,0,500,59,295,59,295,75,59,75" alt="Home" title="Home" href="http://www.karger.com/TMH" ref="reftype=publisher&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="306,60,380,75" alt="Alerts" title="Alerts" href="http://content.karger.com/services/alert.asp" ref="reftype=publisher&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="395,60,458,75" alt="Resources" title="Resources" href="http://content.karger.com/resources/index.asp" ref="reftype=publisher&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Transfus Med Hemother</a></span>. 2014 Nov; 41(6): 453–461. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Nov 7. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1159%2F000369079" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1159/000369079</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4280456</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25670933">25670933</a></div></div></div></div><h1 class="content-title">A Genome-Wide Screen for Large-Effect Alloimmunization Susceptibility Loci among Red Blood Cell Transfusion Recipients with Sickle Cell Disease</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Hanchard%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933" class="affpopup" co-rid="_co_idm140091864379344" co-class="co-affbox">Neil A. Hanchard</a>,<sup>a,</sup><sup>b</sup> <a href="/pubmed/?term=Moulds%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933" class="affpopup" co-rid="_co_idm140091939494752" co-class="co-affbox">Joann M. Moulds</a>,<sup>c</sup> <a href="/pubmed/?term=Belmont%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933" class="affpopup" co-rid="_co_idm140091898514560" co-class="co-affbox">John W. Belmont</a>,<sup>a,</sup><sup>b</sup> and  <a href="/pubmed/?term=Chen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933" class="affpopup" co-rid="_co_idm140091936588816" co-class="co-affbox">Alice Chen</a><sup>d,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140091864379344"><h3 class="no_margin">Neil A. Hanchard</h3><p><sup>a</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX, USA</p><p><sup>b</sup>ARS/USDA/Children's Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Hanchard%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933">Neil A. Hanchard</a></div></div><div id="_co_idm140091939494752"><h3 class="no_margin">Joann M. Moulds</h3><p><sup>c</sup>Scientific Support Services, LifeShare Blood Centers, Shreveport, LA, USA</p><div>Find articles by <a href="/pubmed/?term=Moulds%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933">Joann M. Moulds</a></div></div><div id="_co_idm140091898514560"><h3 class="no_margin">John W. Belmont</h3><p><sup>a</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX, USA</p><p><sup>b</sup>ARS/USDA/Children's Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Belmont%20JW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933">John W. Belmont</a></div></div><div id="_co_idm140091936588816"><h3 class="no_margin">Alice Chen</h3><p><sup>d</sup>Department of Pathology, Texas Heart Institute, Baylor St. Luke's Medical Center, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25670933">Alice Chen</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140091898506448_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140091898506448_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140091898506448_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140091898506448_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>a</sup>Department of Molecular and Human Genetics, Baylor College of Medicine, Houston, TX, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>b</sup>ARS/USDA/Children's Nutrition Research Center, Department of Pediatrics, Baylor College of Medicine, Houston, TX, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>c</sup>Scientific Support Services, LifeShare Blood Centers, Shreveport, LA, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>d</sup>Department of Pathology, Texas Heart Institute, Baylor St. Luke's Medical Center, Houston, TX, USA</div><div id="cor1">*Dr. Alice Chen, Department of Pathology, Texas Heart Institute, Baylor St. Luke's Medical Center, 6720 Bertner Avenue MC 4-265, Houston, TX 77030, USA, <a href="mailto:dev@null" data-email="gro.htlaehsekults@nehca" class="oemail">gro.htlaehsekults@nehca</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140091898506448_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Jul 3; Accepted 2014 Oct 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140091898506448_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2014 by S. Karger GmbH, Freiburg</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4280456/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="idm140091861957696" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140091861957696title">Summary</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Background</h3><p id="__p1" class="p p-first-last">A selective susceptibility of certain individuals to form multiple alloantibodies in response to red cell transfusion is well-recognized in clinical practice, and is a particular problem in persons with sickle cell disease (SCD). The reason for this differential susceptibility is unclear, but inter-individual genetic differences are likely to contribute.</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Methods</h3><p id="__p2" class="p p-first-last">We conducted a pilot case-control genome-wide association study using 1,000,000 SNPs in 94 alloimmune responders (cases) and non-responders (controls) with SCD in order to identify loci of large effect size associated with alloimmunization.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Results</h3><p id="__p3" class="p p-first-last">No loci showed evidence of association at a genome-wide significance cut-off (p &lt; 0.5 × 10<sup>–8</sup>). SNPs in the <em>ARAP1/STARD10</em> region showed suggestive association (p &lt; 1 × 10<sup>–6</sup>), but no association was observed at previously implicated loci <em>TRIM21</em> or <em>HLA.</em> In analyses of the number of accumulated antibodies, a modest association was found with SNPs in the Toll-like receptor gene <em>TLR10</em> (p &lt; 1 × 10<sup>–4</sup>).</p></div><div id="__sec4" class="sec sec-last"><h3 id="__sec4title">Conclusions</h3><p id="__p4" class="p p-first-last">Alloimmunization in persons with SCD is unlikely to be mediated by loci of very large effect size; however, larger and more comprehensive studies are required to fully evaluate loci with more moderate effects. This study provides a working approach to such future studies in SCD.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Genome-wide association studies, GWAS, African American, Responders, Genomics</span></div></div><div id="sec1_1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_1title">Introduction</h2><p id="__p5" class="p p-first">Transfusion of allogenic blood products is an essential part of modern medicine; however, this process is also an immunologic challenge to the recipient, who is exposed to multiple foreign antigens as a consequence. In the setting of red cell transfusions, this exposure sets the stage for the development of alloantibodies, the presence of which creates the considerable challenge of providing compatible blood products while avoiding the attendant problem of hemolytic transfusion reactions [<a href="#B1" rid="B1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412426">1</a>]. Given that some 5 million patients receive blood products every year in the USA, including those who receive multiple transfusions as a consequence of inherited or acquired hematological disorders, post-transfusion alloimmunization is a problem of major public health significance [<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412451">2</a>].</p><p id="__p6">In clinical practice, it is well established that some individuals will form multiple antibodies with almost each transfusion; such ‘responders’ appear immunologically distinct from ‘nonresponders’ for whom the development of antibodies is a rare and unusual occurrence. This phenotypic dichotomy of ‘responders’ and ‘non-responders’ is commonly observed in persons with sickle cell disease (SCD), who often require red cell transfusions both in the acute setting of sickle crises and for their long-term disease management. The rate of red cell alloimmunization in SCD patients is empirically quoted in the range of 30–50% [<a href="#B3" rid="B3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412433">3</a>, <a href="#B4" rid="B4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412466">4</a>] – several orders of magnitude higher than the ˜5% figure observed in the general population [<a href="#B2" rid="B2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412438">2</a>, <a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412441">5</a>]. Alloimmune responders with SCD are also at increased risk of hyperhemolysis [<a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412439">6</a>], an uncommon but potentially devastating type of delayed transfusion reaction.</p><p id="__p7">The reason some individuals become responders while others do not is not well-understood, but is likely to be multifactorial, involving donor, recipient, and external factors [<a href="#B5" rid="B5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412442">5</a>, <a href="#B6" rid="B6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412453">6</a>, <a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412468">7</a>]. For instance, there is support for genetic ancestry contributing to alloimmunization in SCD patients – the genetic ancestry of individuals with SCD is predominantly African American, whereas blood donors of similar ethnicity are underrepresented in the national donor pool [<a href="#B7" rid="B7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412476">7</a>, <a href="#B8" rid="B8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412428">8</a>, <a href="#B9" rid="B9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412436">9</a>, <a href="#B10" rid="B10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412458">10</a>]. This potentiates the mismatches between donor and recipient red cell antigens that are necessary to stimulate an alloimmune response. Accordingly, expanded red cell antigen phenotyping and closer matching of common antigens between donor and recipient has decreased the incidence of alloimmunization in susceptible groups [<a href="#B11" rid="B11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412465">11</a>, <a href="#B12" rid="B12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412425">12</a>], but a significant rate of alloimmunization remains, strongly implicating other mechanisms in the pathogenesis [<a href="#B13" rid="B13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412454">13</a>].</p><p id="__p8">Among the potential additional factors is the well-known genetic variation that exists between individuals; for instance, possession of human leukocyte antigen (HLA) types HLA-B35 and HLA-Cw4 alleles have been linked to ‘responder’ status in persons with SCD [<a href="#B14" rid="B14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412427">14</a>, <a href="#B15" rid="B15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412473">15</a>], and a single nucleotide polymorphism (SNP) in the erythrocyte antigen <em>Ro52 (TRIM21)</em> has been associated with the induction and efficiency of alloimmunization in both a transfusion-dependent and age-dependent fashion in patients with SCD [<a href="#B16" rid="B16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412435">16</a>]. These and other similar reports [<a href="#B17" rid="B17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412430">17</a>] validate a genetic underpinning to the alloimmune response, but they do so in a disparate, ad hoc fashion, and the primary results have not been replicated in other cohorts.</p><p id="__p9">Advances in microarray technologies designed to interrogate SNPs across the genome have made it possible to perform genome-wide association studies (GWAS). The GWAS approach has emerged as a powerful way of agnostically identifying disease susceptibility loci and has facilitated the identification of biologically important loci in multiple diseases across a range of disciplines [<a href="#B18" rid="B18" class=" bibr popnode">18</a>, <a href="#B19" rid="B19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412452">19</a>] including disease severity in SCD [<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_706087813">20</a>, <a href="#B21" rid="B21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412449">21</a>]. Many of the associations with the greatest magnitudes of effect have occurred in diseases with an immune or autoimmune basis [<a href="#B22" rid="B22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412474">22</a>, <a href="#B23" rid="B23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412463">23</a>, <a href="#B24" rid="B24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412455">24</a>, <a href="#B25" rid="B25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412443">25</a>] or in disorders with a strong gene-environment component [<a href="#B26" rid="B26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412456">26</a>, <a href="#B27" rid="B27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412460">27</a>, <a href="#B28" rid="B28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412467">28</a>]. These associations have fundamentally changed our understanding of the underlying disease processes [<a href="#B29" rid="B29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412457">29</a>] and in some cases have facilitated clinically useful genetic testing [<a href="#B30" rid="B30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412431">30</a>, <a href="#B31" rid="B31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412447">31</a>]. Therefore, GWAS are an attractive and robust way of identifying biologically relevant genes and gene pathways contributing to the alloimmune responder phenotype in individuals with SCD; this in turn holds the ultimate promise of new therapeutic targets for those at higher risk. Further, because the GWAS approach includes the identification of both susceptibility <em>and</em> protection alleles, relevant loci could directly impact the ability of blood centers to preemptively identify both susceptible recipients for whom extended phenotyping is especially indicated as well as likely non-responders to whom random units can be provided, thereby increasing the number of donors who are available to supply phenotype-matched blood to SCD patients.</p><p id="__p10" class="p p-last">As a first foray into alloimmunization genomics, we undertook a pilot case-control GWAS of alloimmune responder status in a cohort of multiply transfused participants with SCD, with the express goal of identifying large-effect susceptibility loci. The genome-wide coverage achieved also allowed us to evaluate previously associated loci and to perform subgroup analyses of allo-antibody accumulation.</p></div><div id="sec1_2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_2title">Material and Methods</h2><div id="sec2_1" class="sec sec-first"><h3 id="sec2_1title">Subjects</h3><p id="__p11" class="p p-first-last">The study protocol was approved by the Institutional Review Board of St Luke's Episcopal Hospital and of Baylor College of Medicine. DNA samples used were those taken from patients with a diagnosis of SCD referred to LifeShare for blood group genotyping prior to receiving a red cell transfusion. DNA remaining after completion of the clinical test was retained and de-identified. As these clinical refuse samples were fully de-identified, only limited clinical data was available and the total (or lifetime) number of transfusion units for each individual is unknown; however, all patients had to have received a minimum of two transfusions in the past to be included. Each patient's hemoglobinopathy status was confirmed by genotyping. Samples were obtained from within the LifeShare service area which includes Southeast Texas, the northern half of Louisiana, and Southern Arkansas. Patients were also screened for antibodies using routine serologic techniques. This included incubations at room temperature and 37 ° C using Lo-Ion (Immucor, Norcross, GA, USA) and an AHG phase using polyspecific antiglobulin reagent. Patients were characterized as ‘responders’ (N = 198) if they developed clinically significant alloantibodies after two or more packed red cell transfusions, where clinically significant was defined as ‘an antibody that is frequently associated with HDFN (Hemolytic Disease of the Fetus or Newborn), hemolytic transfusion reactions, or a notable decrease in the survival of transfused RBCs’ [<a href="#B32" rid="B32" class=" bibr popnode">32</a>]. ‘Non-responders’ (N = 186) were those with no alloantibodies after a similar exposure (table <a href="/pmc/articles/PMC4280456/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">(table1).</span></span><span>1</span></a>). We obtained funding to analyze 109 of these DNA samples in the first phase pilot GWAS – 55 ‘responders’ and 54 ‘non-responders’ homozygous for the sickle cell (HbS) mutation. The responder group also included 17 persons with a history of hyperhemolysis syndrome, defined as a post-transfusion drop in hemoglobin below pre-transfusion levels without evidence of bleeding.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p36">Cohort (n = 384) and pilot study demographics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091855500128" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><tbody><tr><td rowspan="1" colspan="1" align="left"><em>Cohort</em></td><td rowspan="1" colspan="1" align="left"><em>384</em></td></tr><tr><td rowspan="1" colspan="1" align="left">Alloimmune category</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left"> Non-responder</td><td rowspan="1" colspan="1" align="left">186</td></tr><tr><td rowspan="1" colspan="1" align="left"> Responder</td><td rowspan="1" colspan="1" align="left">198</td></tr><tr><td rowspan="1" colspan="1" align="left">Alloantibodies, median (min, max, IQR) Gender</td><td rowspan="1" colspan="1" align="left">2 (1, 11, 1–4)</td></tr><tr><td rowspan="1" colspan="1" align="left"> Male</td><td rowspan="1" colspan="1" align="left">165</td></tr><tr><td rowspan="1" colspan="1" align="left"> Female</td><td rowspan="1" colspan="1" align="left">211</td></tr><tr><td rowspan="1" colspan="1" align="left"> Unknown</td><td rowspan="1" colspan="1" align="left">8</td></tr><tr><td rowspan="1" colspan="1" align="left">Sickle genotype</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left"> HbSS</td><td rowspan="1" colspan="1" align="left">373</td></tr><tr><td rowspan="1" colspan="1" align="left"> HbSC</td><td rowspan="1" colspan="1" align="left">11</td></tr><tr><td rowspan="1" colspan="1" align="left">Age group</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left"> Adult (&gt;18 years)</td><td rowspan="1" colspan="1" align="left">346</td></tr><tr><td rowspan="1" colspan="1" align="left"> Child (&lt;18 years)</td><td rowspan="1" colspan="1" align="left">36</td></tr><tr><td rowspan="1" colspan="1" align="left"> Unknown</td><td rowspan="1" colspan="1" align="left">2</td></tr><tr><td colspan="2" rowspan="1" align="left"><hr></td></tr><tr><td rowspan="1" colspan="1" align="left"><em>Genotyped individuals</em></td><td rowspan="1" colspan="1" align="left"><em>104</em></td></tr><tr><td rowspan="1" colspan="1" align="left">Alloimmune category</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1" align="left"> Non-responder (males)</td><td rowspan="1" colspan="1" align="left">54 (23)</td></tr><tr><td rowspan="1" colspan="1" align="left"> Responder (males)</td><td rowspan="1" colspan="1" align="left">50 (20)</td></tr><tr><td rowspan="1" colspan="1" align="left">Alloantibodies, median (min, max, IQR) Sickle genotype</td><td rowspan="1" colspan="1" align="left">4 (1, 11, 2–5)</td></tr><tr><td rowspan="1" colspan="1" align="left"> HbSS</td><td rowspan="1" colspan="1" align="left">104</td></tr></tbody></table></div><div id="largeobj_idm140091855500128" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280456/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140091900435632"><p id="__p37" class="p p-first-last">IQR = Interquartile range.</p></div></div></div></div><div id="sec2_2" class="sec"><h3 id="sec2_2title">DNA Extraction and Genotyping</h3><p id="__p12" class="p p-first">DNA was extracted from leukocyte buffy coats or whole blood using either a manual method (PureGene<sup>®</sup>; Qiagen, Venlo, the Netherlands) or the QIACube<sup>®</sup> (Qiagen). The quantity of double-stranded DNA was assessed in each sample using the PicoGreen (Invitrogen, Carlsbad, CA, USA) assay. All assays were performed according to manufacturer's specifications. Five samples had a DNA concentration below the 50 vg/μl concentration recommended for optimal microarray genotyping and were omitted from further study.</p><p id="__p13" class="p p-last">Genotyping was performed according to manufacturer's instructions using the Bead-Station system and HumanOmni1-Quad Bead Chips (Illumina, San Diego, CA, USA), which was specifically designed to provide improved SNP coverage in people of West African heritage, as might be expected for participants with SCO. The GenomeStudio software (Illumina) was used to interpret normalized fluorescent intensities as genotypes. HumanOmni1-Quad BeadChips contain approximately 1,140,419 SNPs. SNPs with genotyping efficiency &lt; 90% or out of Hardy-Weinberg equilibrium in controls (at the p &lt; 10<sup>–5</sup> level) were removed. The final analysis incorporated genotype data from 1,008,655 SNPs.</p></div><div id="sec2_3" class="sec"><h3 id="sec2_3title">Sample Quality Control</h3><p id="__p14" class="p p-first-last">Known siblings or other related samples were excluded from the initial sample selection. In addition, samples with more than 5% missing data (genotyping efficiency &lt; 0.95) or with evidence of excessive inbreeding (inbreeding coefficient F &gt; 0.1) were removed. Pairs of samples with excess allele sharing suggestive of close familial relationships (parent-offspring, siblings, or 1st degree relatives; PI_HAT &gt; 0.1) were identified by identity-by-descent (IBD); in such cases one member of the pair was removed from analysis (fig. <a href="/pmc/articles/PMC4280456/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.1</span></span><span>1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4280456/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091858850832" class="figure"><a href="/pmc/articles/PMC4280456/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280456_tmh-0041-0453-g01.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is tmh-0041-0453-g01.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280456/bin/tmh-0041-0453-g01.jpg"></a></div><div id="largeobj_idm140091858850832" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280456/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4280456/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p31">Sample quality control (QC); Samples were removed from analysis if they i) failed to meet the minimum DNA concentration for the assay according to manufacturer's recommendations (DNA conc); ii) were successfully genotyped at less than 95% of included SNPs (Genotyping); iii) had an inbreeding coefficient (F) suggestive of an excess of homozygotes; iv) had evidence of close-familial pairing (identify by-decent (IBD) or were ancestry population outliers on principal components analysis (see fig. <a href="/pmc/articles/PMC4280456/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2em;">fig.2</span></span><span>2</span></a>).</p></div></div></div></div><div id="sec2_4" class="sec"><h3 id="sec2_4title">Identification of Population Stratification</h3><p id="__p15" class="p p-first-last">Prior to analysis, the remaining individuals were evaluated for shared ancestry using multidimensional scaling (MDS). The principle of MDS is to cluster individuals of similar ancestry by repeatedly adjusting individual genotypes by the amounts attributable to ancestry. The resulting dimensions can be plotted to visualize individuals whose inferred ancestry is outside that of the sample. Clustering was performed with a subset of SNPs pruned using a sliding window to exclude SNPs with strong nonrandom association (linkage disequilibrium; LD) with surrounding SNPs (r<sup>2</sup> &gt; 0.2). Five outliers were identified using this approach (fig. <a href="/pmc/articles/PMC4280456/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.2).</span></span><span>2</span></a>). Exclusion of these outliers resulted in a final cohort of 48 ‘responder’ cases and 46 ‘non-responder’ controls.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4280456/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091860295536" class="figure"><a href="/pmc/articles/PMC4280456/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280456_tmh-0041-0453-g02.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is tmh-0041-0453-g02.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280456/bin/tmh-0041-0453-g02.jpg"></a></div><div id="largeobj_idm140091860295536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4280456/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4280456/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p32">Multidimensional scaling (MDS) cluster plot of study participants for principal components C1 and C2. <strong>A</strong> Clustering of SCD pilot cohort with HapMap African Americans from Oklahoma and in relation to major continental HapMap groups (YRI = Yoruba from Nigeria; CEU = Ceph from Utah). <strong>B</strong> MDS clustering of cases (black circles) and controls (red triangles) in the pilot cohort. PCA study outliers are circled.</p></div></div></div></div><div id="sec2_5" class="sec sec-last"><h3 id="sec2_5title">Statistical Analyses</h3><p id="__p16" class="p p-first">MDS clustering, IBD analysis, and F statistics (inbreeding) were carried out using PLINK [<a href="#B33" rid="B33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412471">33</a>], as were basic statistical analyses (χ<sup>2</sup> and permutation). After primary analysis, label swapping adaptive permutation was performed for each SNP. In label swapping, the case/control label is swapped for each individual, and the association statistic is derived multiple times to create a new null distribution with which to compare with the observed data. The ‘adaptive’ mode essentially curtails the run length by removing SNPs that are highly unlikely to attain statistically significance at each permutation, until a user-defined maximum number of permutations (in this case 1,000,000) is reached for SNPs that continue to be statistically significant. Genome-wide inflation was estimated using the genABEL package [<a href="#B34" rid="B34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412462">34</a>] in R. Manhattan and MDS plots were generated using a webbased R source code (<em><a href="http://GettingGeneticsDone.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://GettingGeneticsDone.com</a></em>) and LocusZoom [<a href="#B35" rid="B35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412444">35</a>]. Shapeit2 [<a href="#B36" rid="B36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412470">36</a>] and Impute-2 [<a href="#B37" rid="B37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412464">37</a>] were used to impute genotype data from the HapMap [<a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412440">38</a>] and the 1000 Genomes Project Consortium [<a href="#B39" rid="B39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412475">39</a>]. Imputation uses derived patterns of population-specific LD between SNPs to infer genotypes at sites that have not directly been genotyped. This can be a particularly useful approach in populations of African ancestry, where genotyped SNPs may not provide true ‘genome-wide’ coverage.</p><p id="__p17">In subgroup analyses of the cumulative number of antibodies developed among responders, the distribution of alloantibodies was slightly skewed to the right. A power transformation (x<sup>1/4</sup>) was performed to better fit a normal distribution. Unadjusted linear regression models were fit to the data to derive beta statistics and p values.</p><p id="__p18" class="p p-last">Quanto 1.2.4 [<a href="#B40" rid="B40" class=" bibr popnode">40</a>] was used to estimate study power. The study sample size employed had 80% power to determine genetic risk ratios (GRRs) greater than 10 at genome-wide significance (p &lt; 0.5 × 10<sup>–8</sup>) using SNPs with minor allele frequency (MAF) of 0.15 or greater under an additive disease model.</p></div></div><div id="sec1_3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_3title">Results</h2><div id="sec2_6" class="sec sec-first"><h3 id="sec2_6title">No Large-Effect Responder Loci, but Suggestive Association in ARAP1/STARD10</h3><p id="__p19" class="p p-first">For our primary analysis, each SNP with MAF &gt; 0.10 was tested for association (χ<sup>2</sup>) under an allelic trend model using 48 responders as cases and 46 non-responders as controls. The genome-wide inflation factor in this analysis was 0.985, suggesting little residual inflation of the test statistic from genotyping error, cryptic relatedness of individuals, or population stratification. We did not observe any SNPs or clusters of SNPs at or below the accepted threshold for statistical significance in genome-wide association studies (p &lt; 0.5 × 10<sup>–8</sup>) (fig. <a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.3a</span></span><span>3a</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091857405328" class="figure"><a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280456_tmh-0041-0453-g03.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is tmh-0041-0453-g03.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280456/bin/tmh-0041-0453-g03.jpg"></a></div><div id="largeobj_idm140091857405328" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4280456/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4280456/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p33">Case control GWAS of responder status. <strong>A</strong> Manhattan plot showing no SNPs below genome-wide significance (red line, p = 0.5 × 10<sup>–8</sup>), but suggestive SNPs (blue line, p = 1 × 10<sup>–4</sup>) in the <em>ARAP1/STARD10</em> gene region (green dots). <strong>B</strong> Local Manhattan plot of 250 kb region around <em>ARAP1/ STARD10</em> including genotyped and imputed SNPs. Pairwise LD (r<sup>2</sup>) is derived from the HapMap YRI population. <strong>C</strong> GWAS results for genotyped and imputed (shaded) SNPs in the <em>ARAP1/STARD10</em> region. OR = Odds ratio; Perm P = permuted p value.</p></div></div></div><p id="__p20">Next, we sought to evaluate SNPs with p values that were below the ‘background’ association of most SNPs, but did not meet genome-wide significance. We focused on regions of the genome that had more than one SNP within 100 kb with p &lt; 1 × 10<sup>–4</sup>; this threshold was deemed as ‘suggestive’ association. One region met this criteria (fig. <a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.3a)</span></span><span>3a</span></a>) – a cluster of three SNPs (lowest p = 1.2 × 10<sup>–5</sup>) on the long arm of chromosome 11 at the 3 end of <em>ARAP1</em>, extending 5 of <em>STARD10</em> (fig. <a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.3c).</span></span><span>3c</span></a>). This modest association persisted after permutation testing (lowest p = 1.4 × 10<sup>–4</sup>). Manual inspection of the genotyping clusters and the relative local LD pattern in the HapMap YRI from Yoruba, Nigeria [<a href="#B38" rid="B38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412472">38</a>], suggested that technical (genotyping) error was unlikely to be a significant contributing factor.</p><p id="__p21" class="p p-last">Having observed moderately suggestive association with SNPs across this region, we then performed statistical imputation of ˜17,983 variants within in a 5 megabase (MB) region centered upon the SNP with the best association in the original genotyping. We then filtered for common variants (MAF &gt; 0.05) with imputation certainty greater than 95%. This resulted in a final test set of 3,795 SNPs. Using a χ<sup>2</sup> test for allelic trend under the same case-control model, five additional SNPs showed similar and, in some cases, stronger association than the genotyped SNPs at this candidate locus (fig. <a href="/pmc/articles/PMC4280456/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>3b, c</span></a>), although none reached genome-wide significance.</p></div><div id="sec2_7" class="sec"><h3 id="sec2_7title">No Association with Responder Status at Previously Reported Loci</h3><p id="__p22" class="p p-first-last">Next, we focused on two regions that have been previously associated with the development of alloantibodies in other studies – <em>TRIM21</em> on chromosome 11 and the major histocompatibility complex (MHC) on chromosome 6. At <em>TRIM21</em>, none of the genotyped SNPs had a p value less than 10<sup>–3</sup> (fig. <a href="/pmc/articles/PMC4280456/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.4a).</span></span><span>4a</span></a>). A similar pattern was observed across the4 MB of the MHC; most SNPs had p values greater than 1 × 10<sup>–3</sup> (fig. <a href="/pmc/articles/PMC4280456/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.4b),</span></span><span>4b</span></a>), with the strongest SNP association (p &lt; 0.0004) being found in <em>c6orf10</em>, upstream of the MHC class II region some distance from the major HLA types previously associated with the phenotype.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4280456/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091900061616" class="figure"><a href="/pmc/articles/PMC4280456/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280456_tmh-0041-0453-g04.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is tmh-0041-0453-g04.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280456/bin/tmh-0041-0453-g04.jpg"></a></div><div id="largeobj_idm140091900061616" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280456/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4280456/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Fig. 4</a></div><!--caption a7--><div class="caption"><p id="__p34">Local Manhattan plots for previously associated loci <em>TRIM21</em> (<strong>A</strong>) and the <em>MHC</em> (<strong>B</strong>). In each figure the highlighted SNP (purple diamond) is the strongest association in the region. In <strong>A</strong> pairwise LD (r<sup>2</sup>) is derived from the Hapmap YRI population. In <strong>B</strong> the gene list is restricted for space considerations.</p></div></div></div></div><div id="sec2_9" class="sec sec-last"><h3 id="sec2_9title">Subgroup Analysis of Alloantibody Accumulation Reveals an Additional Suggestive Locus</h3><p id="__p23" class="p p-first-last">As an exploratory subgroup analysis, we incorporated a quantitative approach that focused on SNPs influencing the number of antibodies formed by each responder (N = 48) using unadjusted linear regression models. In this analysis there were no SNPs with evidence of association at or below genome-wide significance; however, using the same criteria for suggestive loci employed in our primary analysis, three regions had more than two SNPs with p &lt; 1 × 10<sup>–4</sup> within 100 kb. The most convincing of these included a cluster of eight SNPs in an 11 kb region on chromosome 4p involving the Toll-receptor 10 (<em>TLR10</em>) gene (fig. <a href="/pmc/articles/PMC4280456/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">(fig.5</span></span><span>5</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC4280456/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140091857667504" class="figure"><a href="/pmc/articles/PMC4280456/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4280456_tmh-0041-0453-g05.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is tmh-0041-0453-g05.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4280456/bin/tmh-0041-0453-g05.jpg"></a></div><div id="largeobj_idm140091857667504" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4280456/figure/F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC4280456/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Fig. 5</a></div><!--caption a7--><div class="caption"><p id="__p35">Suggestive association between SNPs across <em>TLR10</em> and number of alloantibodies formed. <strong>A</strong> Local Manhattan plot of SNPs within 250 kb of <em>TLR10.</em>
<strong>B</strong> SNPs showing suggestive association.</p></div></div></div></div></div><div id="sec1_4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_4title">Discussion</h2><p id="__p24" class="p p-first">We did not observe any SNP associations with responder status at or below genome-wide significance in our cohort. In the small sample size of this pilot study, genome-wide significance represents a conservative cut-off p value below which false-positive results are highly unlikely. Thus, we can conclude that in this SCD cohort, becoming an alloimmune responder is not influenced by a locus with very large effect size (GRR &gt; 10). Like many other complex traits, the contribution of common genetic variation to responder status is more likely to be the result of loci with more moderate effects. This class of effects is difficult to confidently discern in a pilot GWAS such as this one, and is further compounded by the African ancestry of the cohort. African populations have greater genetic diversity and less LD between SNPs. Because genome-wide genotyping will rarely, if ever, genotype the true causal variant, GWAS studies are dependent upon LD for genotyped SNPs to act as ‘proxies’ for ungenotyped SNPs. The relative lack of LD in African and African-derived populations thus makes finding more moderate association signals using conventional microarray genotyping platforms more difficult [<a href="#B41" rid="B41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412469">41</a>], despite the availability of statistical imputation. Recent interest in the development of Afro-centric SNP genotyping microarray chips should make future GWAS in the African-American SCD population more robust.</p><p id="__p25">The use of a less conservative cut-off allowed us to evaluate evidence for loci with more moderate effects on the phenotype, although such SNPs have a higher possibility of being false-positives. A cluster of SNPs spanning <em>ARAP1</em> and the adjacent <em>STARD10</em> genes showed suggestive association with responder status in our cohort. <em>ARAP1</em> (MIM# 606646) encodes a Rho/Arf GTPase-activating protein (GAP) postulated to play a role in regulating endocytic trafficking of epidermal growth factor receptors (EGFRs) to the cell membrane. <em>STARD10</em> (Gene ID:10809) encodes one of a group of proteins characterized by incorporation of a steroidogenic acute regulatory protein(StAR)-related lipid transfer (START) domain that binds specific lipid motifs and implicated in the control of lipid biology and cell trafficking. The related STARD11 protein interacts with the Goodpasture antigen and is known to play a role in the autoimmune response seen in Goodpasture syndrome [<a href="#B42" rid="B42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412437">42</a>]; there is thus a potential, albeit speculative, biological role for <em>STARD10</em> in the processing and presentation of red cell membrane antigens. Larger, well-powered replication studies, utilizing more comprehensive genotyping platforms that are better suited to the African American population, are necessary to validate these findings.</p><p id="__p26">In keeping with recent murine studies [<a href="#B43" rid="B43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412446">43</a>], our findings suggest that there is little evidence to support a major role for variants in <em>TRIM21</em> in responder status. Our sample size was larger than that employed in previous studies; however, our study was not designed to look for more subtle determinants or known covariates of alloimmunization (e.g. age or transfusion dependency) and thus we could not confirm the postulated role of the rs660 SNP in this cohort. It should be noted that rs660 is not represented on the HumanOmni1-quad chip, but also maps to several sites in the current version of the human genome reference sequence (<em><a href="http://www.ensembl.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.ensembl.org</a></em>). The fact that rs660-flanking sequences do not map uniquely in the current genome assemblies raises the possibility that technical factors may underlie the previously reported association. Similarly, we did not find strong evidence for association with SNPs across the MHC in our analyses of this cohort. This would be in line with earlier studies suggesting that restricted HLA allelotypes may not play a significant role in susceptibility to alloimmune transfusion tolerance [<a href="#B44" rid="B44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412445">44</a>]. However, because our pilot study did not include extended haplotype analyses of the MHC and current genotyping microarrays do not necessarily fully capture African diversity at the MHC, this result does not completely preclude a role for the MHC in the responder phenotype.</p><p id="__p27">In subgroup analysis we employed a quantitative approach to look at potential genetic factors influencing the development of alloantibodies. A quantitative approach for continuous and ordinal traits has increased power to detect association in genetic studies and consequently generally requires a smaller sample size to find similar effect sizes. This approach has been successfully employed in other GWAS [<a href="#B45" rid="B45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412432">45</a>, <a href="#B46" rid="B46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412448">46</a>], including the association of SNPs in <em>BCL11A</em> with fetal hemoglobin levels in SCD patients, which was accomplished with a few hundred cases and controls [<a href="#B20" rid="B20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_706087812">20</a>]. We did not find evidence for large-effect loci controlling the development of multiple antibodies; however, several SNPs around <em>TLR10</em> showed modest association with increasing alloantibodies. Toll-like receptors (TLRs) have a well-established role in the early immune response to invading pathogens, although a specific role for the TLR10 protein has not yet been identified. Recently, murine models of cardiac and skin transplantation have implicated TLRs in the alloimmunity of graft rejection [<a href="#B47" rid="B47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412429">47</a>], and polymorphisms in <em>TLR4</em> have been associated with lung transplant rejection in humans [<a href="#B48" rid="B48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412434">48</a>]. There is thus growing interest in the role of innate immunity and TLRs in alloimmunization. This again makes it imperative that larger and more comprehensive genomics studies are undertaken to replicate the preliminary results presented here.</p><p id="__p28" class="p p-last">This pilot GWAS of alloimmune responder status and alloantibody number did not reveal evidence of a large-effect locus in this SCD cohort, including previously identified loci <em>TRIM21</em> and the MHC. However, suggestive association at the putative candidate loci <em>ARAP1/STARD10</em> and <em>TLR10</em> advocate for future genomics studies that will not only evaluate these preliminary results but also engender novel approaches to understanding differential alloantibody susceptibility in SCD. A genomics approach to the problem has recently been endorsed as a viable framework in which to study various aspects of the SCD responder phenotype [<a href="#B49" rid="B49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_419412459">49</a>]. The challenge demonstrated by this study is that success in such a venture will require larger cohorts built around a broad expertise and strong collaboration.</p></div><div id="sec1_5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1_5title">Disclosure Statement</h2><p id="__p29" class="p p-first-last">JMM serves is currently an employee of Grifols Inc. and previously served as the principal investigator for grants and research agreements between LifeShare and Novartis Diagnostics or BioArray/Immucor. No other authors have conflicts to declare.</p></div><div id="idm140091855818256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140091855818256title">Acknowledgements</h2><div class="sec"><p id="__p30">This work was funded in part by a grant from Gulf Coast Regional Blood Center to AC. NH is supported by a Clinical Scientist Development Award from the Doris Duke Charitable Foundation. We would also like to thank Kristine Bucasas for help with implementing the statistical imputation.</p></div></div><div id="idm140091855817072" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140091855817072title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1">1. <span class="element-citation">Schonewille H, Haak HL, van Zijl AM. Alloimmunization after blood transfusion in patients with hematologic and oncologic diseases. <span><span class="ref-journal">Transfusion. </span>1999;<span class="ref-vol">39</span>:763–771.</span> [<a href="/pubmed/10413286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Alloimmunization+after+blood+transfusion+in+patients+with+hematologic+and+oncologic+diseases&amp;author=H+Schonewille&amp;author=HL+Haak&amp;author=AM+van+Zijl&amp;volume=39&amp;publication_year=1999&amp;pages=763-771&amp;pmid=10413286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2">2. <span class="element-citation">Heddle NM, Soutar RL, O'Hoski PL, Singer J, McBride JA, Ali MA, Kelton JG. A prospective study to determine the frequency and clinical significance of alloimmunization post-transfusion. <span><span class="ref-journal">Br J Haematol. </span>1995;<span class="ref-vol">91</span>:1000–1005.</span> [<a href="/pubmed/8547111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=A+prospective+study+to+determine+the+frequency+and+clinical+significance+of+alloimmunization+post-transfusion&amp;author=NM+Heddle&amp;author=RL+Soutar&amp;author=PL+O'Hoski&amp;author=J+Singer&amp;author=JA+McBride&amp;volume=91&amp;publication_year=1995&amp;pages=1000-1005&amp;pmid=8547111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3">3. <span class="element-citation">Rosse WF, Gallagher D, Kinney TR, Castro O, Dosik H, Moohr J, Wang W, Levy PS. Transfusion and alloimmunization in sickle cell disease. The Cooperative Study of Sickle Cell Disease. <span><span class="ref-journal">Blood. </span>1990;<span class="ref-vol">76</span>:1431–1437.</span> [<a href="/pubmed/2207318" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Transfusion+and+alloimmunization+in+sickle+cell+disease.+The+Cooperative+Study+of+Sickle+Cell+Disease&amp;author=WF+Rosse&amp;author=D+Gallagher&amp;author=TR+Kinney&amp;author=O+Castro&amp;author=H+Dosik&amp;volume=76&amp;publication_year=1990&amp;pages=1431-1437&amp;pmid=2207318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4">4. <span class="element-citation">Aygun B, Padmanabhan S, Paley C, Chandrasekaran V. Clinical significance of RBC alloantibodies and autoantibodies in sickle cell patients who received transfusions. <span><span class="ref-journal">Transfusion. </span>2002;<span class="ref-vol">42</span>:37–43.</span> [<a href="/pubmed/11896310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Clinical+significance+of+RBC+alloantibodies+and+autoantibodies+in+sickle+cell+patients+who+received+transfusions&amp;author=B+Aygun&amp;author=S+Padmanabhan&amp;author=C+Paley&amp;author=V+Chandrasekaran&amp;volume=42&amp;publication_year=2002&amp;pages=37-43&amp;pmid=11896310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5">5. <span class="element-citation">Bauer MP, Wiersum-Osselton J, Schipperus M, Vandenbroucke JP, Briet E. Clinical predictors of alloimmunization after red blood cell transfusion. <span><span class="ref-journal">Transfusion. </span>2007;<span class="ref-vol">47</span>:2066–2071.</span> [<a href="/pubmed/17958536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Clinical+predictors+of+alloimmunization+after+red+blood+cell+transfusion&amp;author=MP+Bauer&amp;author=J+Wiersum-Osselton&amp;author=M+Schipperus&amp;author=JP+Vandenbroucke&amp;author=E+Briet&amp;volume=47&amp;publication_year=2007&amp;pages=2066-2071&amp;pmid=17958536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6">6. <span class="element-citation">Talano JA, Hillery CA, Gottschall JL, Baylerian DM, Scott JP. Delayed hemolytic transfusion reaction/hyperhemolysis syndrome in children with sickle cell disease. <span><span class="ref-journal">Pediatrics. </span>2003;<span class="ref-vol">111</span>:e661–665.</span> [<a href="/pubmed/12777582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Delayed+hemolytic+transfusion+reaction/hyperhemolysis+syndrome+in+children+with+sickle+cell+disease&amp;author=JA+Talano&amp;author=CA+Hillery&amp;author=JL+Gottschall&amp;author=DM+Baylerian&amp;author=JP+Scott&amp;volume=111&amp;publication_year=2003&amp;pages=e661-665&amp;pmid=12777582&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7">7. <span class="element-citation">Luban NL. Variability in rates of alloimmunization in different groups of children with sickle cell disease: effect of ethnic background. <span><span class="ref-journal">Am J Pediatr Hematol Oncol. </span>1989;<span class="ref-vol">11</span>:314–319.</span> [<a href="/pubmed/2782559" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Pediatr+Hematol+Oncol&amp;title=Variability+in+rates+of+alloimmunization+in+different+groups+of+children+with+sickle+cell+disease:+effect+of+ethnic+background&amp;author=NL+Luban&amp;volume=11&amp;publication_year=1989&amp;pages=314-319&amp;pmid=2782559&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8">8. <span class="element-citation">Vichinsky EP, Earles A, Johnson RA, Hoag MS, Williams A, Lubin B. Alloimmunization in sickle cell anemia and transfusion of racially unmatched blood. <span><span class="ref-journal">N Engl J Med. </span>1990;<span class="ref-vol">322</span>:1617–1621.</span> [<a href="/pubmed/2342522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Alloimmunization+in+sickle+cell+anemia+and+transfusion+of+racially+unmatched+blood&amp;author=EP+Vichinsky&amp;author=A+Earles&amp;author=RA+Johnson&amp;author=MS+Hoag&amp;author=A+Williams&amp;volume=322&amp;publication_year=1990&amp;pages=1617-1621&amp;pmid=2342522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9">9. <span class="element-citation">Olujohungbe A, Hambleton I, Stephens L, Serjeant B, Serjeant G. Red cell antibodies in patients with homozygous sickle cell disease: a comparison of patients in Jamaica and the United Kingdom. <span><span class="ref-journal">Br J Haematol. </span>2001;<span class="ref-vol">113</span>:661–665.</span> [<a href="/pubmed/11380455" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Red+cell+antibodies+in+patients+with+homozygous+sickle+cell+disease:+a+comparison+of+patients+in+Jamaica+and+the+United+Kingdom&amp;author=A+Olujohungbe&amp;author=I+Hambleton&amp;author=L+Stephens&amp;author=B+Serjeant&amp;author=G+Serjeant&amp;volume=113&amp;publication_year=2001&amp;pages=661-665&amp;pmid=11380455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10">10. <span class="element-citation">Murphy EL, Shaz B, Hillyer CD, Carey P, Custer BS, Hirschler N, Fang J, Schreiber GB. Minority and foreign-born representation among us blood donors: Demographics and donation frequency for 2006. <span><span class="ref-journal">Transfusion. </span>2009;<span class="ref-vol">49</span>:2221–2228.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2784257/">PMC free article</a>]</span> [<a href="/pubmed/19555415" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Minority+and+foreign-born+representation+among+us+blood+donors:+Demographics+and+donation+frequency+for+2006&amp;author=EL+Murphy&amp;author=B+Shaz&amp;author=CD+Hillyer&amp;author=P+Carey&amp;author=BS+Custer&amp;volume=49&amp;publication_year=2009&amp;pages=2221-2228&amp;pmid=19555415&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11">11. <span class="element-citation">Ness PM. To match or not to match: the question for chronically transfused patients with sickle cell anemia. <span><span class="ref-journal">Transfusion. </span>1994;<span class="ref-vol">34</span>:558–560.</span> [<a href="/pubmed/8053035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=To+match+or+not+to+match:+the+question+for+chronically+transfused+patients+with+sickle+cell+anemia&amp;author=PM+Ness&amp;volume=34&amp;publication_year=1994&amp;pages=558-560&amp;pmid=8053035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12">12. <span class="element-citation">Vichinsky EP, Luban NL, Wright E, Olivieri N, Driscoll C, Pegelow CH, Adams RJ. Prospective RBC phenotype matching in a stroke-prevention trial in sickle cell anemia: a multicenter transfusion trial. <span><span class="ref-journal">Transfusion. </span>2001;<span class="ref-vol">41</span>:1086–1092.</span> [<a href="/pubmed/11552063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Prospective+RBC+phenotype+matching+in+a+stroke-prevention+trial+in+sickle+cell+anemia:+a+multicenter+transfusion+trial&amp;author=EP+Vichinsky&amp;author=NL+Luban&amp;author=E+Wright&amp;author=N+Olivieri&amp;author=C+Driscoll&amp;volume=41&amp;publication_year=2001&amp;pages=1086-1092&amp;pmid=11552063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13">13. <span class="element-citation">Chou ST, Jackson T, Vege S, Smith-Whitley K, Friedman DF, Westhoff CM. High prevalence of red blood cell alloimmunization in sickle cell disease despite transfusion from Rh-matched minority donors. <span><span class="ref-journal">Blood. </span>2013;<span class="ref-vol">122</span>:1062–1071.</span> [<a href="/pubmed/23723452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=High+prevalence+of+red+blood+cell+alloimmunization+in+sickle+cell+disease+despite+transfusion+from+Rh-matched+minority+donors&amp;author=ST+Chou&amp;author=T+Jackson&amp;author=S+Vege&amp;author=K+Smith-Whitley&amp;author=DF+Friedman&amp;volume=122&amp;publication_year=2013&amp;pages=1062-1071&amp;pmid=23723452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14">14. <span class="element-citation">Alarif L, Castro O, Ofosu M, Dunston G, Scott RB. <em>HLA-B35</em> is associated with red cell alloimmunization in sickle cell disease. <span><span class="ref-journal">Clin Immunol Immunopathol. </span>1986;<span class="ref-vol">38</span>:178–183.</span> [<a href="/pubmed/3484440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Immunol+Immunopathol&amp;title=HLA-B35+is+associated+with+red+cell+alloimmunization+in+sickle+cell+disease&amp;author=L+Alarif&amp;author=O+Castro&amp;author=M+Ofosu&amp;author=G+Dunston&amp;author=RB+Scott&amp;volume=38&amp;publication_year=1986&amp;pages=178-183&amp;pmid=3484440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15">15. <span class="element-citation">Noizat-Pirenne F, Tournamille C, Bierling P, Roudot-Thoraval F, Le Pennec PY, Rouger P, Ansart-Pirenne H. Relative immunogenicity of Fya and K antigens in a Caucasian population, based on HLA class II restriction analysis. <span><span class="ref-journal">Transfusion. </span>2006;<span class="ref-vol">46</span>:1328–1333.</span> [<a href="/pubmed/16934068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Relative+immunogenicity+of+Fya+and+K+antigens+in+a+Caucasian+population,+based+on+HLA+class+II+restriction+analysis&amp;author=F+Noizat-Pirenne&amp;author=C+Tournamille&amp;author=P+Bierling&amp;author=F+Roudot-Thoraval&amp;author=PY+Le+Pennec&amp;volume=46&amp;publication_year=2006&amp;pages=1328-1333&amp;pmid=16934068&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16">16. <span class="element-citation">Tatari-Calderone Z, Minniti CP, Kratovil T, Stojakovic M, Vollmer A, Barjaktarevic I, Zhang E, Hoang A, Luban NL, Vukmanovic S. Rs660 polymorphism in Ro52 (<em>SSA1 TRIM21</em>) is a marker for age-dependent tolerance induction and efficiency of alloimmunization in sickle cell disease. <span><span class="ref-journal">Mol Immunol. </span>2009;<span class="ref-vol">47</span>:64–70.</span> [<a href="/pubmed/19201475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Immunol&amp;title=Rs660+polymorphism+in+Ro52+(SSA1+TRIM21)+is+a+marker+for+age-dependent+tolerance+induction+and+efficiency+of+alloimmunization+in+sickle+cell+disease&amp;author=Z+Tatari-Calderone&amp;author=CP+Minniti&amp;author=T+Kratovil&amp;author=M+Stojakovic&amp;author=A+Vollmer&amp;volume=47&amp;publication_year=2009&amp;pages=64-70&amp;pmid=19201475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17">17. <span class="element-citation">Tatari-Calderone Z, Tamouza R, Le Bouder GP, Dewan R, Luban NL, Lasserre J, Maury J, Lionnet F, Krishnamoorthy R, Girot R, Vukmanovic S. The association of <em>CD81</em> polymorphisms with alloimmunization in sickle cell disease. <span><span class="ref-journal">Clin Dev Immunol. </span>2013;<span class="ref-vol">2013</span>:937846.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3674646/">PMC free article</a>]</span> [<a href="/pubmed/23762099" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Dev+Immunol&amp;title=The+association+of+CD81+polymorphisms+with+alloimmunization+in+sickle+cell+disease&amp;author=Z+Tatari-Calderone&amp;author=R+Tamouza&amp;author=GP+Le+Bouder&amp;author=R+Dewan&amp;author=NL+Luban&amp;volume=2013&amp;publication_year=2013&amp;pages=937846&amp;pmid=23762099&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18">18. <span class="element-citation">Hindorff LA, MacArthur J, Morales J, Junkins HA, Hall PN, Klemm AK, Manolio TA. <span class="ref-journal">A Catalog of Published Genome-Wide Association Studies.</span> 2011.  <em><a href="http://www.genome.gov/gwastudies/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">www.genome.gov/gwastudies/</a></em> Accessed May 27, 2011.</span></div><div class="ref-cit-blk half_rhythm" id="B19">19. <span class="element-citation">Wellcome Trust Case Control Consortium. Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–678.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-678&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20">20. <span class="element-citation">Bhatnagar P, Purvis S, Barron-Casella E, Debaun MR, Casella JF, Arking DE, Keefer JR. Genome-wide association study identifies genetic variants influencing F-cell levels in sickle-cell patients. <span><span class="ref-journal">J Hum Genet. </span>2010;<span class="ref-vol">56</span>:316–323.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5825003/">PMC free article</a>]</span> [<a href="/pubmed/21326311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Hum+Genet&amp;title=Genome-wide+association+study+identifies+genetic+variants+influencing+F-cell+levels+in+sickle-cell+patients&amp;author=P+Bhatnagar&amp;author=S+Purvis&amp;author=E+Barron-Casella&amp;author=MR+Debaun&amp;author=JF+Casella&amp;volume=56&amp;publication_year=2010&amp;pages=316-323&amp;pmid=21326311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21">21. <span class="element-citation">Uda M, Galanello R, Sanna S, Lettre G, Sankaran VG, Chen W, Usala G, Busonero F, Maschio A, Albai G, Piras MG, Sestu N, Lai S, Dei M, Mulas A, Crisponi L, Naitza S, Asunis I, Deiana M, Nagaraja R, Perseu L, Satta S, Cipollina MD, Sollaino C, Moi P, Hirschhorn JN, Orkin SH, Abecasis GR, Schlessinger D, Cao A. Genome-wide association study shows <em>BCL11A</em> associated with persistent fetal hemoglobin and amelioration of the phenotype of beta-thalassemia. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">105</span>:1620–1625.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2234194/">PMC free article</a>]</span> [<a href="/pubmed/18245381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Genome-wide+association+study+shows+BCL11A+associated+with+persistent+fetal+hemoglobin+and+amelioration+of+the+phenotype+of+beta-thalassemia&amp;author=M+Uda&amp;author=R+Galanello&amp;author=S+Sanna&amp;author=G+Lettre&amp;author=VG+Sankaran&amp;volume=105&amp;publication_year=2008&amp;pages=1620-1625&amp;pmid=18245381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22">22. <span class="element-citation">Hunt KA, Zhernakova A, Turner G, Heap GA, Franke L, Bruinenberg M, Romanos J, Dinesen LC, Ryan AW, Panesar D, Gwilliam R, Takeuchi F, McLaren WM, Holmes GK, Howdle PD, Walters JR, Sanders DS, Playford RJ, Trynka G, Mulder CJ, Mearin ML, Verbeek WH, Trimble V, Stevens FM, O'Morain C, Kennedy NP, Kelleher D, Pennington DJ, Strachan DP, McArdle WL, Mein CA, Wapenaar MC, Deloukas P, McGinnis R, McManus R, Wijmenga C, van Heel DA. Newly identified genetic risk variants for celiac disease related to the immune response. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:395–402.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673512/">PMC free article</a>]</span> [<a href="/pubmed/18311140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Newly+identified+genetic+risk+variants+for+celiac+disease+related+to+the+immune+response&amp;author=KA+Hunt&amp;author=A+Zhernakova&amp;author=G+Turner&amp;author=GA+Heap&amp;author=L+Franke&amp;volume=40&amp;publication_year=2008&amp;pages=395-402&amp;pmid=18311140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23">23. <span class="element-citation">Fellay J, Shianna KV, Ge D, Colombo S, Ledergerber B, Weale M, Zhang K, Gumbs C, Castagna A, Cossarizza A, Cozzi-Lepri A, De Luca A, Easterbrook P, Francioli P, Mallal S, Martinez-Picado J, Miro JM, Obel N, Smith JP, Wyniger J, Descombes P, Antonarakis SE, Letvin NL, McMichael AJ, Haynes BF, Telenti A, Goldstein DB. A whole-genome association study of major determinants for host control of HIV-1. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">317</span>:944–947.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1991296/">PMC free article</a>]</span> [<a href="/pubmed/17641165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+whole-genome+association+study+of+major+determinants+for+host+control+of+HIV-1&amp;author=J+Fellay&amp;author=KV+Shianna&amp;author=D+Ge&amp;author=S+Colombo&amp;author=B+Ledergerber&amp;volume=317&amp;publication_year=2007&amp;pages=944-947&amp;pmid=17641165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24">24. <span class="element-citation">Behrens EM, Finkel TH, Bradfield JP, Kim CE, Linton L, Casalunovo T, Frackelton EC, Santa E, Otieno FG, Glessner JT, Chiavacci RM, Grant SF, Hakonarson H. Association of the <em>TRAF1-C5</em> locus on chromosome 9 with juvenile idiopathic arthritis. <span><span class="ref-journal">Arthritis Rheum. </span>2008;<span class="ref-vol">58</span>:2206–2207.</span> [<a href="/pubmed/18576341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Association+of+the+TRAF1-C5+locus+on+chromosome+9+with+juvenile+idiopathic+arthritis&amp;author=EM+Behrens&amp;author=TH+Finkel&amp;author=JP+Bradfield&amp;author=CE+Kim&amp;author=L+Linton&amp;volume=58&amp;publication_year=2008&amp;pages=2206-2207&amp;pmid=18576341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25">25. <span class="element-citation">Hakonarson H, Grant SF, Bradfield JP, Marchand L, Kim CE, Glessner JT, Grabs R, Casalunovo T, Taback SP, Frackelton EC, Lawson ML, Robinson LJ, Skraban R, Lu Y, Chiavacci RM, Stanley CA, Kirsch SE, Rappaport EF, Orange JS, Monos DS, Devoto M, Qu HQ, Polychronakos C. A genome-wide association study identifies <em>KIAA0350</em> as a type 1 diabetes gene. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:591–594.</span> [<a href="/pubmed/17632545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+KIAA0350+as+a+type+1+diabetes+gene&amp;author=H+Hakonarson&amp;author=SF+Grant&amp;author=JP+Bradfield&amp;author=L+Marchand&amp;author=CE+Kim&amp;volume=448&amp;publication_year=2007&amp;pages=591-594&amp;pmid=17632545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26">26. <span class="element-citation">Tekola Ayele F, Adeyemo A, Finan C, Hailu E, Sinnott P, Burlinson ND, Aseffa A, Rotimi CN, Newport MJ, Davey G. HLA class II locus and susceptibility to podoconiosis. <span><span class="ref-journal">N Engl J Med. </span>2012;<span class="ref-vol">366</span>:1200–1208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3350841/">PMC free article</a>]</span> [<a href="/pubmed/22455414" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=HLA+class+II+locus+and+susceptibility+to+podoconiosis&amp;author=F+Tekola+Ayele&amp;author=A+Adeyemo&amp;author=C+Finan&amp;author=E+Hailu&amp;author=P+Sinnott&amp;volume=366&amp;publication_year=2012&amp;pages=1200-1208&amp;pmid=22455414&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27">27. <span class="element-citation">Daly AK, Donaldson PT, Bhatnagar P, Shen Y, Pe'er I, Floratos A, Daly MJ, Goldstein DB, John S, Nelson MR, Graham J, Park BK, Dillon JF, Bernal W, Cordell HJ, Pirmohamed M, Aithal GP, Day CP, Study D International SAEC. HLA-B*5701 genotype is a major determinant of druginduced liver injury due to flucloxacillin. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:816–819.</span> [<a href="/pubmed/19483685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=HLA-B*5701+genotype+is+a+major+determinant+of+druginduced+liver+injury+due+to+flucloxacillin&amp;author=AK+Daly&amp;author=PT+Donaldson&amp;author=P+Bhatnagar&amp;author=Y+Shen&amp;author=I+Pe'er&amp;volume=41&amp;publication_year=2009&amp;pages=816-819&amp;pmid=19483685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28">28. <span class="element-citation">Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, Nakagawa M, Korenaga M, Hino K, Hige S, Ito Y, Mita E, Tanaka E, Mochida S, Murawaki Y, Honda M, Sakai A, Hiasa Y, Nishiguchi S, Koike A, Sakaida I, Imamura M, Ito K, Yano K, Masaki N, Sugauchi F, Izumi N, Tokunaga K, Mizokami M. Genome-wide association of <em>IL28B</em> with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:1105–1109.</span> [<a href="/pubmed/19749757" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+of+IL28B+with+response+to+pegylated+interferon-alpha+and+ribavirin+therapy+for+chronic+hepatitis+C&amp;author=Y+Tanaka&amp;author=N+Nishida&amp;author=M+Sugiyama&amp;author=M+Kurosaki&amp;author=K+Matsuura&amp;volume=41&amp;publication_year=2009&amp;pages=1105-1109&amp;pmid=19749757&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29">29. <span class="element-citation">Visscher PM, Brown MA, McCarthy MI, Yang J. Five years of GWAS discovery. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">90</span>:7–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3257326/">PMC free article</a>]</span> [<a href="/pubmed/22243964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Five+years+of+GWAS+discovery&amp;author=PM+Visscher&amp;author=MA+Brown&amp;author=MI+McCarthy&amp;author=J+Yang&amp;volume=90&amp;publication_year=2012&amp;pages=7-24&amp;pmid=22243964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30">30. <span class="element-citation">Hetherington S, Hughes AR, Mosteller M, Shortino D, Baker KL, Spreen W, Lai E, Davies K, Handley A, Dow DJ, Fling ME, Stocum M, Bowman C, Thurmond LM, Roses AD. Genetic variations in <em>HLA-B</em> region and hypersensitivity reactions to abacavir. <span><span class="ref-journal">Lancet. </span>2002;<span class="ref-vol">359</span>:1121–1122.</span> [<a href="/pubmed/11943262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Genetic+variations+in+HLA-B+region+and+hypersensitivity+reactions+to+abacavir&amp;author=S+Hetherington&amp;author=AR+Hughes&amp;author=M+Mosteller&amp;author=D+Shortino&amp;author=KL+Baker&amp;volume=359&amp;publication_year=2002&amp;pages=1121-1122&amp;pmid=11943262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31">31. <span class="element-citation">Mallal S, Phillips E, Carosi G, Molina JM, Workman C, Tomazic J, Jagel-Guedes E, Rugina S, Kozyrev O, Cid JF, Hay P, Nolan D, Hughes S, Hughes A, Ryan S, Fitch N, Thorborn D, Benbow A, Team P-S. <em>HLA-B*5701</em> screening for hypersensitivity to abacavir. <span><span class="ref-journal">N Engl J Med. </span>2008;<span class="ref-vol">358</span>:568–579.</span> [<a href="/pubmed/18256392" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=HLA-B*5701+screening+for+hypersensitivity+to+abacavir&amp;author=S+Mallal&amp;author=E+Phillips&amp;author=G+Carosi&amp;author=JM+Molina&amp;author=C+Workman&amp;volume=358&amp;publication_year=2008&amp;pages=568-579&amp;pmid=18256392&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32">32. <span class="element-citation">American Association of Blood Banks. <span class="ref-journal">AABB Technical Manual.</span> 17th ed. Bethesda: American Association of Blood Banks (AABB); 2011.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=AABB+Technical+Manual&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33">33. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. Plink: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Plink:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34">34. <span class="element-citation">Aulchenko YS, Ripke S, Isaacs A, van Duijn CM. Genabel: An R library for genome-wide association analysis. <span><span class="ref-journal">Bioinformatics. </span>2007;<span class="ref-vol">23</span>:1294–1296.</span> [<a href="/pubmed/17384015" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genabel:+An+R+library+for+genome-wide+association+analysis&amp;author=YS+Aulchenko&amp;author=S+Ripke&amp;author=A+Isaacs&amp;author=CM+van+Duijn&amp;volume=23&amp;publication_year=2007&amp;pages=1294-1296&amp;pmid=17384015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35">35. <span class="element-citation">Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, Boehnke M, Abecasis GR, Willer CJ. Locuszoom: Regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Locuszoom:+Regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;author=RP+Welch&amp;author=S+Sanna&amp;author=TM+Teslovich&amp;author=PS+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36">36. <span class="element-citation">Delaneau O, Zagury JF, Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. <span><span class="ref-journal">Nat Methods. </span>2013;<span class="ref-vol">10</span>:5–6.</span> [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies&amp;author=O+Delaneau&amp;author=JF+Zagury&amp;author=J+Marchini&amp;volume=10&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37">37. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38">38. <span class="element-citation">Altshuler D, Brooks LD, Chakravarti A, Collins FS, Daly MJ, Donnelly P. A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;author=D+Altshuler&amp;author=LD+Brooks&amp;author=A+Chakravarti&amp;author=FS+Collins&amp;author=MJ+Daly&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39">39. <span class="element-citation">Genomes Project C, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;author=C+Genomes+Project&amp;author=GR+Abecasis&amp;author=D+Altshuler&amp;author=A+Auton&amp;author=LD+Brooks&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40">40. <span class="element-citation">Gauderman W, Morrison J. <span class="ref-journal">Quanto 1.1: A Computer Program for Power and Sample Size Calculations for Genetic-Epidemiology Studies.</span> 2006.  <em><a href="http://hydra.Usc.Edu/gxe" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://hydra.Usc.Edu/gxe</a></em>.</span></div><div class="ref-cit-blk half_rhythm" id="B41">41. <span class="element-citation">Jallow M, Teo YY, Small KS, Rockett KA, Deloukas P, Clark TG, Kivinen K, Bojang KA, Conway DJ, Pinder M, Sirugo G, Sisay-Joof F, Usen S, Auburn S, Bumpstead SJ, Campino S, Coffey A, Dunham A, Fry AE, Green A, Gwilliam R, Hunt SE, Inouye M, Jeffreys AE, Mendy A, Palotie A, Potter S, Ragoussis J, Rogers J, Rowlands K, Somaskantharajah E, Whittaker P, Widden C, Donnelly P, Howie B, Marchini J, Morris A, SanJoaquin M, Achidi EA, Agbenyega T, Allen A, Amodu O, Corran P, Djimde A, Dolo A, Doumbo OK, Drakeley C, Dunstan S, Evans J, Farrar J, Fernando D, Hien TT, Horstmann RD, Ibrahim M, Karunaweera N, Kokwaro G, Koram KA, Lemnge M, Makani J, Marsh K, Michon P, Modiano D, Molyneux ME, Mueller I, Parker M, Peshu N, Plowe CV, Puijalon O, Reeder J, Reyburn H, Riley EM, Sakuntabhai A, Singhasivanon P, Sirima S, Tall A, Taylor TE, Thera M, Troye-Blomberg M, Williams TN, Wilson M, Kwiatkowski DP. Genome-wide and fine-resolution association analysis of malaria in West Africa. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:657–665.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889040/">PMC free article</a>]</span> [<a href="/pubmed/19465909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+and+fine-resolution+association+analysis+of+malaria+in+West+Africa&amp;author=M+Jallow&amp;author=YY+Teo&amp;author=KS+Small&amp;author=KA+Rockett&amp;author=P+Deloukas&amp;volume=41&amp;publication_year=2009&amp;pages=657-665&amp;pmid=19465909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42">42. <span class="element-citation">Alpy F, Tomasetto C. Give lipids a start: The star-related lipid transfer (start) domain in mammals. <span><span class="ref-journal">J Cell Sci. </span>2005;<span class="ref-vol">118</span>:2791–2801.</span> [<a href="/pubmed/15976441" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Cell+Sci&amp;title=Give+lipids+a+start:+The+star-related+lipid+transfer+(start)+domain+in+mammals&amp;author=F+Alpy&amp;author=C+Tomasetto&amp;volume=118&amp;publication_year=2005&amp;pages=2791-2801&amp;pmid=15976441&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43">43. <span class="element-citation">Patel SR, Hendrickson JE, Smith NH, Cadwell CM, Ozato K, Morse HC, 3rd, Yoshimi R, Zimring JC. Alloimmunization against RBC or PLT antigens is independent of <em>TRIM21</em> expression in a murine model. <span><span class="ref-journal">Mol Immunol. </span>2011;<span class="ref-vol">48</span>:909–913.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3090620/">PMC free article</a>]</span> [<a href="/pubmed/21269695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Immunol&amp;title=Alloimmunization+against+RBC+or+PLT+antigens+is+independent+of+TRIM21+expression+in+a+murine+model&amp;author=SR+Patel&amp;author=JE+Hendrickson&amp;author=NH+Smith&amp;author=CM+Cadwell&amp;author=K+Ozato&amp;volume=48&amp;publication_year=2011&amp;pages=909-913&amp;pmid=21269695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44">44. <span class="element-citation">Brantley SG, Ramsey G. Red cell alloimmunization in multitransfused HLA-typed patients. <span><span class="ref-journal">Transfusion. </span>1988;<span class="ref-vol">28</span>:463–466.</span> [<a href="/pubmed/3420675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transfusion&amp;title=Red+cell+alloimmunization+in+multitransfused+HLA-typed+patients&amp;author=SG+Brantley&amp;author=G+Ramsey&amp;volume=28&amp;publication_year=1988&amp;pages=463-466&amp;pmid=3420675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45">45. <span class="element-citation">Sanna S, Busonero F, Maschio A, McArdle PF, Usala G, Dei M, Lai S, Mulas A, Piras MG, Perseu L, Masala M, Marongiu M, Crisponi L, Naitza S, Galanello R, Abecasis GR, Shuldiner AR, Schlessinger D, Cao A, Uda M. Common variants in the <em>SLCO1B3</em> locus are associated with bilirubin levels and unconjugated hyperbilirubinemia. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:2711–2718.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2701337/">PMC free article</a>]</span> [<a href="/pubmed/19419973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Common+variants+in+the+SLCO1B3+locus+are+associated+with+bilirubin+levels+and+unconjugated+hyperbilirubinemia&amp;author=S+Sanna&amp;author=F+Busonero&amp;author=A+Maschio&amp;author=PF+McArdle&amp;author=G+Usala&amp;volume=18&amp;publication_year=2009&amp;pages=2711-2718&amp;pmid=19419973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46">46. <span class="element-citation">Hazra A, Kraft P, Lazarus R, Chen C, Chanock SJ, Jacques P, Selhub J, Hunter DJ. Genome-wide significant predictors of metabolites in the one-carbon metabolism pathway. <span><span class="ref-journal">Hum Mol Genet. </span>2009;<span class="ref-vol">18</span>:4677–4687.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2773275/">PMC free article</a>]</span> [<a href="/pubmed/19744961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+significant+predictors+of+metabolites+in+the+one-carbon+metabolism+pathway&amp;author=A+Hazra&amp;author=P+Kraft&amp;author=R+Lazarus&amp;author=C+Chen&amp;author=SJ+Chanock&amp;volume=18&amp;publication_year=2009&amp;pages=4677-4687&amp;pmid=19744961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47">47. <span class="element-citation">Goldstein DR. Toll-like receptors and other links between innate and acquired alloimmunity. <span><span class="ref-journal">Curr Opin Immunol. </span>2004;<span class="ref-vol">16</span>:538–544.</span> [<a href="/pubmed/15341996" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Opin+Immunol&amp;title=Toll-like+receptors+and+other+links+between+innate+and+acquired+alloimmunity&amp;author=DR+Goldstein&amp;volume=16&amp;publication_year=2004&amp;pages=538-544&amp;pmid=15341996&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48">48. <span class="element-citation">Palmer SM, Burch LH, Davis RD, Herczyk WF, Howell DN, Reinsmoen NL, Schwartz DA. The role of innate immunity in acute allograft rejection after lung transplantation. <span><span class="ref-journal">Am J Respir Crit Care Med. </span>2003;<span class="ref-vol">168</span>:628–632.</span> [<a href="/pubmed/12773319" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Respir+Crit+Care+Med&amp;title=The+role+of+innate+immunity+in+acute+allograft+rejection+after+lung+transplantation&amp;author=SM+Palmer&amp;author=LH+Burch&amp;author=RD+Davis&amp;author=WF+Herczyk&amp;author=DN+Howell&amp;volume=168&amp;publication_year=2003&amp;pages=628-632&amp;pmid=12773319&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49">49. <span class="element-citation">Yazdanbakhsh K, Ware RE, Noizat-Pirenne F. Red blood cell alloimmunization in sickle cell disease: Pathophysiology, risk factors, and transfusion management. <span><span class="ref-journal">Blood. </span>2012;<span class="ref-vol">120</span>:528–537.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3401213/">PMC free article</a>]</span> [<a href="/pubmed/22563085" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Red+blood+cell+alloimmunization+in+sickle+cell+disease:+Pathophysiology,+risk+factors,+and+transfusion+management&amp;author=K+Yazdanbakhsh&amp;author=RE+Ware&amp;author=F+Noizat-Pirenne&amp;volume=120&amp;publication_year=2012&amp;pages=528-537&amp;pmid=22563085&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4280456&amp;issue-id=247087&amp;journal-id=1263&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#idm140091861957696title">Summary</a></li><li><a href="#sec1_1title">Introduction</a></li><li><a href="#sec1_2title">Material and Methods</a></li><li><a href="#sec1_3title">Results</a></li><li><a href="#sec1_4title">Discussion</a></li><li><a href="#sec1_5title">Disclosure Statement</a></li><li><a href="#idm140091855818256title">Acknowledgements</a></li><li><a href="#idm140091855817072title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Transfusion Medicine and Hemotherapy</span> are provided here courtesy of <strong>Karger Publishers</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4280456/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4280456/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4280456/pdf/tmh-0041-0453.pdf">PDF (770K)</a></li> | <li><a href="#" data-citationid="PMC4280456" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280456%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280456%2F&amp;text=A%20Genome-Wide%20Screen%20for%20Large-Effect%20Alloimmunization%20Susceptibility%20Loci%20among%20Red%20Blood%20Cell%20Transfusion%20Recipients%20with%20Sickle%20Cell%20Disease" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280456%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25670933" data-aiid="4280456" data-aid="4280456" data-iid="247087" data-domainid="1263" data-domain="tmh" data-accid="PMC4280456" data-md5="0c901d5a3bcb1a5e5aaae576d7d2183e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:22:38-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B3522DD57642100000000093F0595&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4280456%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=tmh&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4280456/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>