<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4757735/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>The impact of low-frequency and rare variants on lipid levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4757735/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="The impact of low-frequency and rare variants on lipid levels"><meta name="citation_authors" content="Ida Surakka, Momoko Horikoshi, Reedik Mägi, Antti-Pekka Sarin, Anubha Mahajan, Vasiliki Lagou, Letizia Marullo, Teresa Ferreira, Benjamin Miraglio, Sanna Timonen, Johannes Kettunen, Matti Pirinen, Juha Karjalainen, Gudmar Thorleifsson, Sara Hägg, Jouke-Jan Hottenga, Aaron Isaacs, Claes Ladenvall, Marian Beekman, Tõnu Esko, Janina S Ried, Christopher P Nelson, Christina Willenborg, Stefan Gustafsson, Harm-Jan Westra, Matthew Blades, Anton JM de Craen, Eco J de Geus, Joris Deelen, Harald Grallert, Anders Hamsten, Aki S. Havulinna, Christian Hengstenberg, Jeanine J Houwing-Duistermaat, Elina Hyppönen, Lennart C Karssen, Terho Lehtimäki, Valeriya Lyssenko, Patrik KE Magnusson, Evelin Mihailov, Martina Müller-Nurasyid, John-Patrick Mpindi, Nancy L Pedersen, Brenda WJH Penninx, Markus Perola, Tune H Pers, Annette Peters, Johan Rung, Johannes H Smit, Valgerdur Steinthorsdottir, Martin D Tobin, Natalia Tsernikova, Elisabeth M van Leeuwen, Jorma S Viikari, Sara M Willems, Gonneke Willemsen, Heribert Schunkert, Jeanette Erdmann, Nilesh J Samani, Jaakko Kaprio, Lars Lind, Christian Gieger, Andres Metspalu, P Eline Slagboom, Leif Groop, Cornelia M van Duijn, Johan G Eriksson, Antti Jula, Veikko Salomaa, Dorret I Boomsma, Christine Power, Olli T Raitakari, Erik Ingelsson, Marjo-Riitta Järvelin, Kari Stefansson, Lude Franke, Elina Ikonen, Olli Kallioniemi, Vilja Pietiäinen, Cecilia M Lindgren, Unnur Thorsteinsdottir, Aarno Palotie, Mark I McCarthy, Andrew P Morris, Inga Prokopenko, Samuli Ripatti"><meta name="citation_date" content="June 2015"><meta name="citation_issue" content="6"><meta name="citation_volume" content="47"><meta name="citation_firstpage" content="589"><meta name="citation_doi" content="10.1038/ng.3300"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4757735/?report=abstract"><meta name="citation_pmid" content="25961943"><meta name="DC.Title" content="The impact of low-frequency and rare variants on lipid levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Ida Surakka"><meta name="DC.Contributor" content="Momoko Horikoshi"><meta name="DC.Contributor" content="Reedik Mägi"><meta name="DC.Contributor" content="Antti-Pekka Sarin"><meta name="DC.Contributor" content="Anubha Mahajan"><meta name="DC.Contributor" content="Vasiliki Lagou"><meta name="DC.Contributor" content="Letizia Marullo"><meta name="DC.Contributor" content="Teresa Ferreira"><meta name="DC.Contributor" content="Benjamin Miraglio"><meta name="DC.Contributor" content="Sanna Timonen"><meta name="DC.Contributor" content="Johannes Kettunen"><meta name="DC.Contributor" content="Matti Pirinen"><meta name="DC.Contributor" content="Juha Karjalainen"><meta name="DC.Contributor" content="Gudmar Thorleifsson"><meta name="DC.Contributor" content="Sara Hägg"><meta name="DC.Contributor" content="Jouke-Jan Hottenga"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Claes Ladenvall"><meta name="DC.Contributor" content="Marian Beekman"><meta name="DC.Contributor" content="Tõnu Esko"><meta name="DC.Contributor" content="Janina S Ried"><meta name="DC.Contributor" content="Christopher P Nelson"><meta name="DC.Contributor" content="Christina Willenborg"><meta name="DC.Contributor" content="Stefan Gustafsson"><meta name="DC.Contributor" content="Harm-Jan Westra"><meta name="DC.Contributor" content="Matthew Blades"><meta name="DC.Contributor" content="Anton JM de Craen"><meta name="DC.Contributor" content="Eco J de Geus"><meta name="DC.Contributor" content="Joris Deelen"><meta name="DC.Contributor" content="Harald Grallert"><meta name="DC.Contributor" content="Anders Hamsten"><meta name="DC.Contributor" content="Aki S. Havulinna"><meta name="DC.Contributor" content="Christian Hengstenberg"><meta name="DC.Contributor" content="Jeanine J Houwing-Duistermaat"><meta name="DC.Contributor" content="Elina Hyppönen"><meta name="DC.Contributor" content="Lennart C Karssen"><meta name="DC.Contributor" content="Terho Lehtimäki"><meta name="DC.Contributor" content="Valeriya Lyssenko"><meta name="DC.Contributor" content="Patrik KE Magnusson"><meta name="DC.Contributor" content="Evelin Mihailov"><meta name="DC.Contributor" content="Martina Müller-Nurasyid"><meta name="DC.Contributor" content="John-Patrick Mpindi"><meta name="DC.Contributor" content="Nancy L Pedersen"><meta name="DC.Contributor" content="Brenda WJH Penninx"><meta name="DC.Contributor" content="Markus Perola"><meta name="DC.Contributor" content="Tune H Pers"><meta name="DC.Contributor" content="Annette Peters"><meta name="DC.Contributor" content="Johan Rung"><meta name="DC.Contributor" content="Johannes H Smit"><meta name="DC.Contributor" content="Valgerdur Steinthorsdottir"><meta name="DC.Contributor" content="Martin D Tobin"><meta name="DC.Contributor" content="Natalia Tsernikova"><meta name="DC.Contributor" content="Elisabeth M van Leeuwen"><meta name="DC.Contributor" content="Jorma S Viikari"><meta name="DC.Contributor" content="Sara M Willems"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Heribert Schunkert"><meta name="DC.Contributor" content="Jeanette Erdmann"><meta name="DC.Contributor" content="Nilesh J Samani"><meta name="DC.Contributor" content="Jaakko Kaprio"><meta name="DC.Contributor" content="Lars Lind"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Andres Metspalu"><meta name="DC.Contributor" content="P Eline Slagboom"><meta name="DC.Contributor" content="Leif Groop"><meta name="DC.Contributor" content="Cornelia M van Duijn"><meta name="DC.Contributor" content="Johan G Eriksson"><meta name="DC.Contributor" content="Antti Jula"><meta name="DC.Contributor" content="Veikko Salomaa"><meta name="DC.Contributor" content="Dorret I Boomsma"><meta name="DC.Contributor" content="Christine Power"><meta name="DC.Contributor" content="Olli T Raitakari"><meta name="DC.Contributor" content="Erik Ingelsson"><meta name="DC.Contributor" content="Marjo-Riitta Järvelin"><meta name="DC.Contributor" content="Kari Stefansson"><meta name="DC.Contributor" content="Lude Franke"><meta name="DC.Contributor" content="Elina Ikonen"><meta name="DC.Contributor" content="Olli Kallioniemi"><meta name="DC.Contributor" content="Vilja Pietiäinen"><meta name="DC.Contributor" content="Cecilia M Lindgren"><meta name="DC.Contributor" content="Unnur Thorsteinsdottir"><meta name="DC.Contributor" content="Aarno Palotie"><meta name="DC.Contributor" content="Mark I McCarthy"><meta name="DC.Contributor" content="Andrew P Morris"><meta name="DC.Contributor" content="Inga Prokopenko"><meta name="DC.Contributor" content="Samuli Ripatti"><meta name="DC.Date" content="2015 Jun"><meta name="DC.Identifier" content="10.1038/ng.3300"><meta name="DC.Language" content="en"><meta property="og:title" content="The impact of low-frequency and rare variants on lipid levels"><meta property="og:type" content="article"><meta property="og:description" content="Using a genome-wide screen of 9.6 million genetic variants achieved through 1000 Genomes imputation in 62,166 samples, we identify association to lipids in 93 loci including 79 previously identified loci with new lead-SNPs, 10 new loci, 15 loci with a ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4757735/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4757735/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4757735/pdf/emss-67068.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B7924DD57971100000000099305A1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4757735%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC4757735</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2016 Feb 18.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25961943" target="pmc_ext" ref="reftype=publisher&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2015 Jun; 47(6): 589–597. </span></a></div><span class="fm-vol-iss-date">Published online 2015 May 11. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.3300" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.3300</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4757735</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS67068</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25961943">25961943</a></div></div></div></div><h1 class="content-title">The impact of low-frequency and rare variants on lipid levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Surakka%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947219770464" co-class="co-affbox">Ida Surakka</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Horikoshi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213451904" co-class="co-affbox">Momoko Horikoshi</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=M%26%23x000e4%3Bgi%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218553536" co-class="co-affbox">Reedik Mägi</a>,<sup>5</sup> <a href="/pubmed/?term=Sarin%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218551552" co-class="co-affbox">Antti-Pekka Sarin</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Mahajan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211916864" co-class="co-affbox">Anubha Mahajan</a>,<sup>3</sup> <a href="/pubmed/?term=Lagou%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211914880" co-class="co-affbox">Vasiliki Lagou</a>,<sup>3,</sup><sup>4</sup> <a href="/pubmed/?term=Marullo%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211235808" co-class="co-affbox">Letizia Marullo</a>,<sup>6</sup> <a href="/pubmed/?term=Ferreira%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213897056" co-class="co-affbox">Teresa Ferreira</a>,<sup>3</sup> <a href="/pubmed/?term=Miraglio%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213895072" co-class="co-affbox">Benjamin Miraglio</a>,<sup>1</sup> <a href="/pubmed/?term=Timonen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947254095776" co-class="co-affbox">Sanna Timonen</a>,<sup>1</sup> <a href="/pubmed/?term=Kettunen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947254093792" co-class="co-affbox">Johannes Kettunen</a>,<sup>1</sup> <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947214960272" co-class="co-affbox">Matti Pirinen</a>,<sup>1</sup> <a href="/pubmed/?term=Karjalainen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947210630272" co-class="co-affbox">Juha Karjalainen</a>,<sup>7</sup> <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947210628288" co-class="co-affbox">Gudmar Thorleifsson</a>,<sup>8</sup> <a href="/pubmed/?term=H%26%23x000e4%3Bgg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211923056" co-class="co-affbox">Sara Hägg</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211920336" co-class="co-affbox">Jouke-Jan Hottenga</a>,<sup>11</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215380832" co-class="co-affbox">Aaron Isaacs</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Ladenvall%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217458048" co-class="co-affbox">Claes Ladenvall</a>,<sup>14</sup> <a href="/pubmed/?term=Beekman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217456064" co-class="co-affbox">Marian Beekman</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947214814496" co-class="co-affbox">Tõnu Esko</a>,<sup>5,</sup><sup>17,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Ried%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947207650640" co-class="co-affbox">Janina S Ried</a>,<sup>20</sup> <a href="/pubmed/?term=Nelson%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216102576" co-class="co-affbox">Christopher P Nelson</a>,<sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Willenborg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215991536" co-class="co-affbox">Christina Willenborg</a>,<sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Gustafsson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215988816" co-class="co-affbox">Stefan Gustafsson</a>,<sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Westra%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947219854352" co-class="co-affbox">Harm-Jan Westra</a>,<sup>7</sup> <a href="/pubmed/?term=Blades%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216813216" co-class="co-affbox">Matthew Blades</a>,<sup>25</sup> <a href="/pubmed/?term=de%20Craen%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216811232" co-class="co-affbox">Anton JM de Craen</a>,<sup>15,</sup><sup>26</sup> <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215995984" co-class="co-affbox">Eco J de Geus</a>,<sup>11</sup> <a href="/pubmed/?term=Deelen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215809632" co-class="co-affbox">Joris Deelen</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Grallert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215806912" co-class="co-affbox">Harald Grallert</a>,<sup>27,</sup><sup>28,</sup><sup>29</sup> <a href="/pubmed/?term=Hamsten%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211910784" co-class="co-affbox">Anders Hamsten</a>,<sup>30</sup> <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947209400144" co-class="co-affbox">Aki S. Havulinna</a>,<sup>31</sup> <a href="/pubmed/?term=Hengstenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218677040" co-class="co-affbox">Christian Hengstenberg</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Houwing-Duistermaat%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218674320" co-class="co-affbox">Jeanine J Houwing-Duistermaat</a>,<sup>34</sup> <a href="/pubmed/?term=Hypp%26%23x000f6%3Bnen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947212448592" co-class="co-affbox">Elina Hyppönen</a>,<sup>35,</sup><sup>36,</sup><sup>37</sup> <a href="/pubmed/?term=Karssen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213659120" co-class="co-affbox">Lennart C Karssen</a>,<sup>12</sup> <a href="/pubmed/?term=Lehtim%26%23x000e4%3Bki%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213657136" co-class="co-affbox">Terho Lehtimäki</a>,<sup>38</sup> <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213721392" co-class="co-affbox">Valeriya Lyssenko</a>,<sup>14,</sup><sup>39</sup> <a href="/pubmed/?term=Magnusson%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215204048" co-class="co-affbox">Patrik KE Magnusson</a>,<sup>9</sup> <a href="/pubmed/?term=Mihailov%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215202064" co-class="co-affbox">Evelin Mihailov</a>,<sup>5</sup> <a href="/pubmed/?term=M%26%23x000fc%3Bller-Nurasyid%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215597424" co-class="co-affbox">Martina Müller-Nurasyid</a>,<sup>20,</sup><sup>33,</sup><sup>40,</sup><sup>41</sup> <a href="/pubmed/?term=Mpindi%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218692912" co-class="co-affbox">John-Patrick Mpindi</a>,<sup>1</sup> <a href="/pubmed/?term=Pedersen%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218690928" co-class="co-affbox">Nancy L Pedersen</a>,<sup>9</sup> <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947291267056" co-class="co-affbox">Brenda WJH Penninx</a>,<sup>42</sup> <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217710672" co-class="co-affbox">Markus Perola</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Pers%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217707952" co-class="co-affbox">Tune H Pers</a>,<sup>17,</sup><sup>18,</sup><sup>43</sup> <a href="/pubmed/?term=Peters%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947209254928" co-class="co-affbox">Annette Peters</a>,<sup>27,</sup><sup>29,</sup><sup>32</sup> <a href="/pubmed/?term=Rung%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947290342592" co-class="co-affbox">Johan Rung</a>,<sup>44</sup> <a href="/pubmed/?term=Smit%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218650032" co-class="co-affbox">Johannes H Smit</a>,<sup>42</sup> <a href="/pubmed/?term=Steinthorsdottir%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218648048" co-class="co-affbox">Valgerdur Steinthorsdottir</a>,<sup>8</sup> <a href="/pubmed/?term=Tobin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216656576" co-class="co-affbox">Martin D Tobin</a>,<sup>45</sup> <a href="/pubmed/?term=Tsernikova%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216654592" co-class="co-affbox">Natalia Tsernikova</a>,<sup>5</sup> <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213458320" co-class="co-affbox">Elisabeth M van Leeuwen</a>,<sup>12</sup> <a href="/pubmed/?term=Viikari%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215816000" co-class="co-affbox">Jorma S Viikari</a>,<sup>46</sup> <a href="/pubmed/?term=Willems%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215814016" co-class="co-affbox">Sara M Willems</a>,<sup>12</sup> <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215609264" co-class="co-affbox">Gonneke Willemsen</a>,<sup>11</sup> <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215607280" co-class="co-affbox">Heribert Schunkert</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218684432" co-class="co-affbox">Jeanette Erdmann</a>,<sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216078576" co-class="co-affbox">Nilesh J Samani</a>,<sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217478736" co-class="co-affbox">Jaakko Kaprio</a>,<sup>1,</sup><sup>47,</sup><sup>48</sup> <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217475280" co-class="co-affbox">Lars Lind</a>,<sup>49</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216781664" co-class="co-affbox">Christian Gieger</a>,<sup>20</sup> <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218931248" co-class="co-affbox">Andres Metspalu</a>,<sup>5,</sup><sup>50</sup> <a href="/pubmed/?term=Slagboom%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947218928528" co-class="co-affbox">P Eline Slagboom</a>,<sup>15,</sup><sup>16</sup> <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947212098064" co-class="co-affbox">Leif Groop</a>,<sup>1,</sup><sup>14</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947217737056" co-class="co-affbox">Cornelia M van Duijn</a>,<sup>12,</sup><sup>13</sup> <a href="/pubmed/?term=Eriksson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947214579664" co-class="co-affbox">Johan G Eriksson</a>,<sup>51,</sup><sup>52,</sup><sup>53,</sup><sup>54</sup> <a href="/pubmed/?term=Jula%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213558624" co-class="co-affbox">Antti Jula</a>,<sup>55</sup> <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213556640" co-class="co-affbox">Veikko Salomaa</a>,<sup>31</sup> <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947209393696" co-class="co-affbox">Dorret I Boomsma</a>,<sup>11</sup> <a href="/pubmed/?term=Power%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947209391712" co-class="co-affbox">Christine Power</a>,<sup>35</sup> <a href="/pubmed/?term=Raitakari%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947207725200" co-class="co-affbox">Olli T Raitakari</a>,<sup>56,</sup><sup>57</sup> <a href="/pubmed/?term=Ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213753392" co-class="co-affbox">Erik Ingelsson</a>,<sup>3,</sup><sup>10</sup> <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213750672" co-class="co-affbox">Marjo-Riitta Järvelin</a>,<sup>58,</sup><sup>59,</sup><sup>60,</sup><sup>61,</sup><sup>62</sup> <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215410512" co-class="co-affbox">Kari Stefansson</a>,<sup>8,</sup><sup>63</sup> <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947215407792" co-class="co-affbox">Lude Franke</a>,<sup>7</sup> <a href="/pubmed/?term=Ikonen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947214375792" co-class="co-affbox">Elina Ikonen</a>,<sup>64,</sup><sup>65</sup> <a href="/pubmed/?term=Kallioniemi%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947212357552" co-class="co-affbox">Olli Kallioniemi</a>,<sup>1</sup> <a href="/pubmed/?term=Pieti%26%23x000e4%3Binen%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947212355568" co-class="co-affbox">Vilja Pietiäinen</a>,<sup>1</sup> <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211452720" co-class="co-affbox">Cecilia M Lindgren</a>,<sup>3,</sup><sup>18</sup> <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947211450000" co-class="co-affbox">Unnur Thorsteinsdottir</a>,<sup>8,</sup><sup>63</sup> <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947210813568" co-class="co-affbox">Aarno Palotie</a>,<sup>1,</sup><sup>2,</sup><sup>18,</sup><sup>66</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216138064" co-class="co-affbox">Mark I McCarthy</a>,<sup>3,</sup><sup>4,</sup><sup>67</sup> <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947216786144" co-class="co-affbox">Andrew P Morris</a>,<sup>3,</sup><sup>5,</sup><sup>68</sup> <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947214509792" co-class="co-affbox">Inga Prokopenko</a>,<sup>69</sup> and  <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943" class="affpopup" co-rid="_co_idm139947213835792" co-class="co-affbox">Samuli Ripatti</a><sup>1,</sup><sup>47,</sup><sup>70</sup>, for the ENGAGE Consortium</div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139947219770464"><h3 class="no_margin">Ida Surakka</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>2</sup>Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Surakka%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Ida Surakka</a></div></div><div id="_co_idm139947213451904"><h3 class="no_margin">Momoko Horikoshi</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Horikoshi%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Momoko Horikoshi</a></div></div><div id="_co_idm139947218553536"><h3 class="no_margin">Reedik Mägi</h3><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><div>Find articles by <a href="/pubmed/?term=M%26%23x000e4%3Bgi%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Reedik Mägi</a></div></div><div id="_co_idm139947218551552"><h3 class="no_margin">Antti-Pekka Sarin</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>2</sup>Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Sarin%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Antti-Pekka Sarin</a></div></div><div id="_co_idm139947211916864"><h3 class="no_margin">Anubha Mahajan</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Mahajan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Anubha Mahajan</a></div></div><div id="_co_idm139947211914880"><h3 class="no_margin">Vasiliki Lagou</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Lagou%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Vasiliki Lagou</a></div></div><div id="_co_idm139947211235808"><h3 class="no_margin">Letizia Marullo</h3><p><sup>6</sup>Department of Life Sciences and Biotechnology, Genetic Section, University of Ferrara, Ferrara, Italy </p><div>Find articles by <a href="/pubmed/?term=Marullo%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Letizia Marullo</a></div></div><div id="_co_idm139947213897056"><h3 class="no_margin">Teresa Ferreira</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=Ferreira%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Teresa Ferreira</a></div></div><div id="_co_idm139947213895072"><h3 class="no_margin">Benjamin Miraglio</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Miraglio%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Benjamin Miraglio</a></div></div><div id="_co_idm139947254095776"><h3 class="no_margin">Sanna Timonen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Timonen%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Sanna Timonen</a></div></div><div id="_co_idm139947254093792"><h3 class="no_margin">Johannes Kettunen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Kettunen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Johannes Kettunen</a></div></div><div id="_co_idm139947214960272"><h3 class="no_margin">Matti Pirinen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Pirinen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Matti Pirinen</a></div></div><div id="_co_idm139947210630272"><h3 class="no_margin">Juha Karjalainen</h3><p><sup>7</sup>University of Croningen, University Medical Center Groningen, Department of Genetics, Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Karjalainen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Juha Karjalainen</a></div></div><div id="_co_idm139947210628288"><h3 class="no_margin">Gudmar Thorleifsson</h3><p><sup>8</sup>deCODE Genetics/Amgen inc., Reykjavik, Iceland </p><div>Find articles by <a href="/pubmed/?term=Thorleifsson%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Gudmar Thorleifsson</a></div></div><div id="_co_idm139947211923056"><h3 class="no_margin">Sara Hägg</h3><p><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><p><sup>10</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=H%26%23x000e4%3Bgg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Sara Hägg</a></div></div><div id="_co_idm139947211920336"><h3 class="no_margin">Jouke-Jan Hottenga</h3><p><sup>11</sup>Department of Biological Psychology, EMGO institute for Health and Care research, VU University &amp; VU medical center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Jouke-Jan Hottenga</a></div></div><div id="_co_idm139947215380832"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </p><p><sup>13</sup>Centre for Medical Systems Biology, Leiden, the Netherlands </p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Aaron Isaacs</a></div></div><div id="_co_idm139947217458048"><h3 class="no_margin">Claes Ladenvall</h3><p><sup>14</sup>Department of Clinical Sciences, Diabetes and Endocrinology, Lund University Diabetes Centre, Skåne University Hospital, Malmö, Sweden </p><div>Find articles by <a href="/pubmed/?term=Ladenvall%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Claes Ladenvall</a></div></div><div id="_co_idm139947217456064"><h3 class="no_margin">Marian Beekman</h3><p><sup>15</sup>Department of Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands </p><p><sup>16</sup>Netherlands Consortium for Healthy Ageing, Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Beekman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Marian Beekman</a></div></div><div id="_co_idm139947214814496"><h3 class="no_margin">Tõnu Esko</h3><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><p><sup>17</sup>Divisions of Endocrinology and Center for Basic and Translational Obesity Research, Children's Hospital, Boston, Massachusetts, USA </p><p><sup>18</sup>The Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, USA </p><p><sup>19</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Tõnu Esko</a></div></div><div id="_co_idm139947207650640"><h3 class="no_margin">Janina S Ried</h3><p><sup>20</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Ried%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Janina S Ried</a></div></div><div id="_co_idm139947216102576"><h3 class="no_margin">Christopher P Nelson</h3><p><sup>21</sup>Department of Cardiovascular Sciences, University of Leicester, Leicester, LE3 9QP, UK </p><p><sup>22</sup>National Institute for Health Research (NIHR) Leicester Cardiovascular Disease Biomedical Research Unit, Glenfield Hospital, Leicester, UK </p><div>Find articles by <a href="/pubmed/?term=Nelson%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Christopher P Nelson</a></div></div><div id="_co_idm139947215991536"><h3 class="no_margin">Christina Willenborg</h3><p><sup>23</sup>Institut für Integrative und Experimentelle Genomik, Universität zu Lübeck, Lübeck, Germany </p><p><sup>24</sup>Deutsches Zentrum für Herz-Kreislauf-Forschung e. V. (DZHK), Partnersite Hamburg, Lübeck, Kiel, Germany </p><div>Find articles by <a href="/pubmed/?term=Willenborg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Christina Willenborg</a></div></div><div id="_co_idm139947215988816"><h3 class="no_margin">Stefan Gustafsson</h3><p><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><p><sup>10</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Gustafsson%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Stefan Gustafsson</a></div></div><div id="_co_idm139947219854352"><h3 class="no_margin">Harm-Jan Westra</h3><p><sup>7</sup>University of Croningen, University Medical Center Groningen, Department of Genetics, Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Westra%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Harm-Jan Westra</a></div></div><div id="_co_idm139947216813216"><h3 class="no_margin">Matthew Blades</h3><p><sup>25</sup>Bioinformatics and Biostatistics Support Hub (B/BASH), University of Leicester, University Road, Leicester, UK </p><div>Find articles by <a href="/pubmed/?term=Blades%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Matthew Blades</a></div></div><div id="_co_idm139947216811232"><h3 class="no_margin">Anton JM de Craen</h3><p><sup>15</sup>Department of Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands </p><p><sup>26</sup>Department of Gerontology and Geriatrics, Leiden University Medical Center, Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=de%20Craen%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Anton JM de Craen</a></div></div><div id="_co_idm139947215995984"><h3 class="no_margin">Eco J de Geus</h3><p><sup>11</sup>Department of Biological Psychology, EMGO institute for Health and Care research, VU University &amp; VU medical center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Eco J de Geus</a></div></div><div id="_co_idm139947215809632"><h3 class="no_margin">Joris Deelen</h3><p><sup>15</sup>Department of Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands </p><p><sup>16</sup>Netherlands Consortium for Healthy Ageing, Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Deelen%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Joris Deelen</a></div></div><div id="_co_idm139947215806912"><h3 class="no_margin">Harald Grallert</h3><p><sup>27</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>28</sup>German Center for Diabetes Research (DZD), Neuherberg, Germany </p><p><sup>29</sup>Institute of Epidemiology II, Helmholtz Zentrum München - Germany Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Grallert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Harald Grallert</a></div></div><div id="_co_idm139947211910784"><h3 class="no_margin">Anders Hamsten</h3><p><sup>30</sup>Cardiovascular Genetics and Genomics Group, Atherosclerosis Research Unit, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Hamsten%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Anders Hamsten</a></div></div><div id="_co_idm139947209400144"><h3 class="no_margin">Aki S. Havulinna</h3><p><sup>31</sup>Unit of Chronic Disease Epidemiology and Prevention, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Havulinna%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Aki S. Havulinna</a></div></div><div id="_co_idm139947218677040"><h3 class="no_margin">Christian Hengstenberg</h3><p><sup>32</sup>Deutsches Herzzentrum München, Technische Universität München, München, Germany </p><p><sup>33</sup>DZHK (German Centre for Cardiovascular Research), partner site Munich Heart Alliance, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Hengstenberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Christian Hengstenberg</a></div></div><div id="_co_idm139947218674320"><h3 class="no_margin">Jeanine J Houwing-Duistermaat</h3><p><sup>34</sup>Department of Medical Statistics and Bioinformatices, Leiden University Medical Center, Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Houwing-Duistermaat%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Jeanine J Houwing-Duistermaat</a></div></div><div id="_co_idm139947212448592"><h3 class="no_margin">Elina Hyppönen</h3><p><sup>35</sup>Centre for Paediatric Epidemiology and Biostatistics, UCL Institute of Child Health, London, UK </p><p><sup>36</sup>South Australian Health and Medical Research Institute, Adelaide, Australia </p><p><sup>37</sup>School of population Health and Sansom Institute, University of South Australia, Adelaide, Australia </p><div>Find articles by <a href="/pubmed/?term=Hypp%26%23x000f6%3Bnen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Elina Hyppönen</a></div></div><div id="_co_idm139947213659120"><h3 class="no_margin">Lennart C Karssen</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </p><div>Find articles by <a href="/pubmed/?term=Karssen%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Lennart C Karssen</a></div></div><div id="_co_idm139947213657136"><h3 class="no_margin">Terho Lehtimäki</h3><p><sup>38</sup>Department of Clinical Chemistry, Fimlab Laboratories and School of Medicine, University of Tampere, Tampere, Finland </p><div>Find articles by <a href="/pubmed/?term=Lehtim%26%23x000e4%3Bki%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Terho Lehtimäki</a></div></div><div id="_co_idm139947213721392"><h3 class="no_margin">Valeriya Lyssenko</h3><p><sup>14</sup>Department of Clinical Sciences, Diabetes and Endocrinology, Lund University Diabetes Centre, Skåne University Hospital, Malmö, Sweden </p><p><sup>39</sup>Steno Diabetes Center A/S, Gentofte, Denmark </p><div>Find articles by <a href="/pubmed/?term=Lyssenko%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Valeriya Lyssenko</a></div></div><div id="_co_idm139947215204048"><h3 class="no_margin">Patrik KE Magnusson</h3><p><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Magnusson%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Patrik KE Magnusson</a></div></div><div id="_co_idm139947215202064"><h3 class="no_margin">Evelin Mihailov</h3><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><div>Find articles by <a href="/pubmed/?term=Mihailov%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Evelin Mihailov</a></div></div><div id="_co_idm139947215597424"><h3 class="no_margin">Martina Müller-Nurasyid</h3><p><sup>20</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>33</sup>DZHK (German Centre for Cardiovascular Research), partner site Munich Heart Alliance, Munich, Germany </p><p><sup>40</sup>Department of Medicine I, University Hospital Grosshadern, Ludwig-Maximilians-Universität, Munich, Germany </p><p><sup>41</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=M%26%23x000fc%3Bller-Nurasyid%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Martina Müller-Nurasyid</a></div></div><div id="_co_idm139947218692912"><h3 class="no_margin">John-Patrick Mpindi</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Mpindi%20JP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">John-Patrick Mpindi</a></div></div><div id="_co_idm139947218690928"><h3 class="no_margin">Nancy L Pedersen</h3><p><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </p><div>Find articles by <a href="/pubmed/?term=Pedersen%20NL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Nancy L Pedersen</a></div></div><div id="_co_idm139947291267056"><h3 class="no_margin">Brenda WJH Penninx</h3><p><sup>42</sup>Department of Psychiatry, VU University Medical Center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Brenda WJH Penninx</a></div></div><div id="_co_idm139947217710672"><h3 class="no_margin">Markus Perola</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>2</sup>Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Perola%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Markus Perola</a></div></div><div id="_co_idm139947217707952"><h3 class="no_margin">Tune H Pers</h3><p><sup>17</sup>Divisions of Endocrinology and Center for Basic and Translational Obesity Research, Children's Hospital, Boston, Massachusetts, USA </p><p><sup>18</sup>The Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, USA </p><p><sup>43</sup>Department of Systems Biology, Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark </p><div>Find articles by <a href="/pubmed/?term=Pers%20TH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Tune H Pers</a></div></div><div id="_co_idm139947209254928"><h3 class="no_margin">Annette Peters</h3><p><sup>27</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>29</sup>Institute of Epidemiology II, Helmholtz Zentrum München - Germany Research Center for Environmental Health, Neuherberg, Germany </p><p><sup>32</sup>Deutsches Herzzentrum München, Technische Universität München, München, Germany </p><div>Find articles by <a href="/pubmed/?term=Peters%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Annette Peters</a></div></div><div id="_co_idm139947290342592"><h3 class="no_margin">Johan Rung</h3><p><sup>44</sup>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, UK </p><div>Find articles by <a href="/pubmed/?term=Rung%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Johan Rung</a></div></div><div id="_co_idm139947218650032"><h3 class="no_margin">Johannes H Smit</h3><p><sup>42</sup>Department of Psychiatry, VU University Medical Center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Smit%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Johannes H Smit</a></div></div><div id="_co_idm139947218648048"><h3 class="no_margin">Valgerdur Steinthorsdottir</h3><p><sup>8</sup>deCODE Genetics/Amgen inc., Reykjavik, Iceland </p><div>Find articles by <a href="/pubmed/?term=Steinthorsdottir%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Valgerdur Steinthorsdottir</a></div></div><div id="_co_idm139947216656576"><h3 class="no_margin">Martin D Tobin</h3><p><sup>45</sup>Genetic Epidemiology Group, Department of Health Sciences, University of Leicester, University Road, Leicester, UK </p><div>Find articles by <a href="/pubmed/?term=Tobin%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Martin D Tobin</a></div></div><div id="_co_idm139947216654592"><h3 class="no_margin">Natalia Tsernikova</h3><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><div>Find articles by <a href="/pubmed/?term=Tsernikova%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Natalia Tsernikova</a></div></div><div id="_co_idm139947213458320"><h3 class="no_margin">Elisabeth M van Leeuwen</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </p><div>Find articles by <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Elisabeth M van Leeuwen</a></div></div><div id="_co_idm139947215816000"><h3 class="no_margin">Jorma S Viikari</h3><p><sup>46</sup>Department of Medicine, University of Turku and Division of Medicine, Turku University Hospital, Turku, Finland </p><div>Find articles by <a href="/pubmed/?term=Viikari%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Jorma S Viikari</a></div></div><div id="_co_idm139947215814016"><h3 class="no_margin">Sara M Willems</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </p><div>Find articles by <a href="/pubmed/?term=Willems%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Sara M Willems</a></div></div><div id="_co_idm139947215609264"><h3 class="no_margin">Gonneke Willemsen</h3><p><sup>11</sup>Department of Biological Psychology, EMGO institute for Health and Care research, VU University &amp; VU medical center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Gonneke Willemsen</a></div></div><div id="_co_idm139947215607280"><h3 class="no_margin">Heribert Schunkert</h3><p><sup>32</sup>Deutsches Herzzentrum München, Technische Universität München, München, Germany </p><p><sup>33</sup>DZHK (German Centre for Cardiovascular Research), partner site Munich Heart Alliance, Munich, Germany </p><div>Find articles by <a href="/pubmed/?term=Schunkert%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Heribert Schunkert</a></div></div><div id="_co_idm139947218684432"><h3 class="no_margin">Jeanette Erdmann</h3><p><sup>23</sup>Institut für Integrative und Experimentelle Genomik, Universität zu Lübeck, Lübeck, Germany </p><p><sup>24</sup>Deutsches Zentrum für Herz-Kreislauf-Forschung e. V. (DZHK), Partnersite Hamburg, Lübeck, Kiel, Germany </p><div>Find articles by <a href="/pubmed/?term=Erdmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Jeanette Erdmann</a></div></div><div id="_co_idm139947216078576"><h3 class="no_margin">Nilesh J Samani</h3><p><sup>21</sup>Department of Cardiovascular Sciences, University of Leicester, Leicester, LE3 9QP, UK </p><p><sup>22</sup>National Institute for Health Research (NIHR) Leicester Cardiovascular Disease Biomedical Research Unit, Glenfield Hospital, Leicester, UK </p><div>Find articles by <a href="/pubmed/?term=Samani%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Nilesh J Samani</a></div></div><div id="_co_idm139947217478736"><h3 class="no_margin">Jaakko Kaprio</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>47</sup>Hjelt Institute, University of Helsinki, Helsinki, Finland </p><p><sup>48</sup>Department of Mental Health and Substance Abuse Services, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Kaprio%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Jaakko Kaprio</a></div></div><div id="_co_idm139947217475280"><h3 class="no_margin">Lars Lind</h3><p><sup>49</sup>Department of Medical Sciences, Uppsala University, Akademiska Sjukhuset, Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Lind%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Lars Lind</a></div></div><div id="_co_idm139947216781664"><h3 class="no_margin">Christian Gieger</h3><p><sup>20</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Christian Gieger</a></div></div><div id="_co_idm139947218931248"><h3 class="no_margin">Andres Metspalu</h3><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><p><sup>50</sup>The Institute of Molecular and Cell Biology of the University of Tartu, Tartu, Estonia </p><div>Find articles by <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Andres Metspalu</a></div></div><div id="_co_idm139947218928528"><h3 class="no_margin">P Eline Slagboom</h3><p><sup>15</sup>Department of Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands </p><p><sup>16</sup>Netherlands Consortium for Healthy Ageing, Leiden, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Slagboom%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">P Eline Slagboom</a></div></div><div id="_co_idm139947212098064"><h3 class="no_margin">Leif Groop</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>14</sup>Department of Clinical Sciences, Diabetes and Endocrinology, Lund University Diabetes Centre, Skåne University Hospital, Malmö, Sweden </p><div>Find articles by <a href="/pubmed/?term=Groop%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Leif Groop</a></div></div><div id="_co_idm139947217737056"><h3 class="no_margin">Cornelia M van Duijn</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </p><p><sup>13</sup>Centre for Medical Systems Biology, Leiden, the Netherlands </p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Cornelia M van Duijn</a></div></div><div id="_co_idm139947214579664"><h3 class="no_margin">Johan G Eriksson</h3><p><sup>51</sup>Department of General Practice and Primary Health Care, University of Helsinki, Helsinki, Finland </p><p><sup>52</sup>Folkhälsan Research Centre, Helsinki, Finland </p><p><sup>53</sup>Helsinki University Hospital, Unit of Primary Health Care, Helsinki, Finland </p><p><sup>54</sup>Department of Health Promotion and Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Eriksson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Johan G Eriksson</a></div></div><div id="_co_idm139947213558624"><h3 class="no_margin">Antti Jula</h3><p><sup>55</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Turku, Finland </p><div>Find articles by <a href="/pubmed/?term=Jula%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Antti Jula</a></div></div><div id="_co_idm139947213556640"><h3 class="no_margin">Veikko Salomaa</h3><p><sup>31</sup>Unit of Chronic Disease Epidemiology and Prevention, National Institute for Health and Welfare, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Salomaa%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Veikko Salomaa</a></div></div><div id="_co_idm139947209393696"><h3 class="no_margin">Dorret I Boomsma</h3><p><sup>11</sup>Department of Biological Psychology, EMGO institute for Health and Care research, VU University &amp; VU medical center, Amsterdam, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Dorret I Boomsma</a></div></div><div id="_co_idm139947209391712"><h3 class="no_margin">Christine Power</h3><p><sup>35</sup>Centre for Paediatric Epidemiology and Biostatistics, UCL Institute of Child Health, London, UK </p><div>Find articles by <a href="/pubmed/?term=Power%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Christine Power</a></div></div><div id="_co_idm139947207725200"><h3 class="no_margin">Olli T Raitakari</h3><p><sup>56</sup>Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Finland </p><p><sup>57</sup>Department of Clinical Physiology and Nuclear Medicine, University of Turku and Turku University Hospital, Finland </p><div>Find articles by <a href="/pubmed/?term=Raitakari%20OT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Olli T Raitakari</a></div></div><div id="_co_idm139947213753392"><h3 class="no_margin">Erik Ingelsson</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>10</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden </p><div>Find articles by <a href="/pubmed/?term=Ingelsson%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Erik Ingelsson</a></div></div><div id="_co_idm139947213750672"><h3 class="no_margin">Marjo-Riitta Järvelin</h3><p><sup>58</sup>Department of Epidemiology and Biostatistics, MRC Health Protection Agency (HPE), Centre for Environment and Health, School of Public Health, Imperial College London, UK </p><p><sup>59</sup>Institute of Health Sciences, University of Oulu, Oulu, Finland </p><p><sup>60</sup>Biocenter Oulu, University of Oulu, Oulu, Finland </p><p><sup>61</sup>Unit of Primary Care, Oulu University Hospital, Oulu, Finland </p><p><sup>62</sup>Department of Children and Young People and Families, National Institute for Health Welfare, Oulu, Finland </p><div>Find articles by <a href="/pubmed/?term=J%26%23x000e4%3Brvelin%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Marjo-Riitta Järvelin</a></div></div><div id="_co_idm139947215410512"><h3 class="no_margin">Kari Stefansson</h3><p><sup>8</sup>deCODE Genetics/Amgen inc., Reykjavik, Iceland </p><p><sup>63</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland </p><div>Find articles by <a href="/pubmed/?term=Stefansson%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Kari Stefansson</a></div></div><div id="_co_idm139947215407792"><h3 class="no_margin">Lude Franke</h3><p><sup>7</sup>University of Croningen, University Medical Center Groningen, Department of Genetics, Groningen, The Netherlands </p><div>Find articles by <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Lude Franke</a></div></div><div id="_co_idm139947214375792"><h3 class="no_margin">Elina Ikonen</h3><p><sup>64</sup>Institute of Biomedicine, Anatomy, University of Helsinki, Helsinki, Finland </p><p><sup>65</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Ikonen%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Elina Ikonen</a></div></div><div id="_co_idm139947212357552"><h3 class="no_margin">Olli Kallioniemi</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Kallioniemi%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Olli Kallioniemi</a></div></div><div id="_co_idm139947212355568"><h3 class="no_margin">Vilja Pietiäinen</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Pieti%26%23x000e4%3Binen%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Vilja Pietiäinen</a></div></div><div id="_co_idm139947211452720"><h3 class="no_margin">Cecilia M Lindgren</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>18</sup>The Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, USA </p><div>Find articles by <a href="/pubmed/?term=Lindgren%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Cecilia M Lindgren</a></div></div><div id="_co_idm139947211450000"><h3 class="no_margin">Unnur Thorsteinsdottir</h3><p><sup>8</sup>deCODE Genetics/Amgen inc., Reykjavik, Iceland </p><p><sup>63</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland </p><div>Find articles by <a href="/pubmed/?term=Thorsteinsdottir%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Unnur Thorsteinsdottir</a></div></div><div id="_co_idm139947210813568"><h3 class="no_margin">Aarno Palotie</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>2</sup>Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland </p><p><sup>18</sup>The Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, USA </p><p><sup>66</sup>Haartman Institute, University of Helsinki and Helsinki University Central Hospital, Helsinki, Finland </p><div>Find articles by <a href="/pubmed/?term=Palotie%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Aarno Palotie</a></div></div><div id="_co_idm139947216138064"><h3 class="no_margin">Mark I McCarthy</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford, UK </p><p><sup>67</sup>Oxford National Institute for Health Research (NIHR) Biomedical Research Centre, Churchill Hospital, Oxford, UK </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Mark I McCarthy</a></div></div><div id="_co_idm139947216786144"><h3 class="no_margin">Andrew P Morris</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </p><p><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </p><p><sup>68</sup>Department of Biostatistics, University of Liverpool, Liverpool, UK </p><div>Find articles by <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Andrew P Morris</a></div></div><div id="_co_idm139947214509792"><h3 class="no_margin">Inga Prokopenko</h3><p><sup>69</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, Hammersmith Hospital, London, UK </p><div>Find articles by <a href="/pubmed/?term=Prokopenko%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Inga Prokopenko</a></div></div><div id="_co_idm139947213835792"><h3 class="no_margin">Samuli Ripatti</h3><p><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </p><p><sup>47</sup>Hjelt Institute, University of Helsinki, Helsinki, Finland </p><p><sup>70</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </p><div>Find articles by <a href="/pubmed/?term=Ripatti%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25961943">Samuli Ripatti</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139947253916768_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139947253916768_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139947253916768_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Institute for Molecular Medicine Finland FIMM, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Public Health Genomics Unit, National Institute for Health and Welfare, Helsinki, Finland </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Oxford, UK </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Estonian Genome Center, University of Tartu, Tartu, Estonia </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Life Sciences and Biotechnology, Genetic Section, University of Ferrara, Ferrara, Italy </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>University of Croningen, University Medical Center Groningen, Department of Genetics, Groningen, The Netherlands </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>deCODE Genetics/Amgen inc., Reykjavik, Iceland </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Medical Sciences, Molecular Epidemiology and Science for Life Laboratory, Uppsala University, Uppsala, Sweden </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Biological Psychology, EMGO institute for Health and Care research, VU University &amp; VU medical center, Amsterdam, The Netherlands </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus University Medical Center, Rotterdam, the Netherlands </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Centre for Medical Systems Biology, Leiden, the Netherlands </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Department of Clinical Sciences, Diabetes and Endocrinology, Lund University Diabetes Centre, Skåne University Hospital, Malmö, Sweden </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Molecular Epidemiology, Leiden University Medical Center, Leiden, The Netherlands </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Netherlands Consortium for Healthy Ageing, Leiden, The Netherlands </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Divisions of Endocrinology and Center for Basic and Translational Obesity Research, Children's Hospital, Boston, Massachusetts, USA </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>The Broad Institute of Massachusetts Institute of Technology and Harvard University, Cambridge, Massachusetts, USA </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Department of Genetics, Harvard Medical School, Boston, Massachusetts 02115, USA </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Cardiovascular Sciences, University of Leicester, Leicester, LE3 9QP, UK </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>National Institute for Health Research (NIHR) Leicester Cardiovascular Disease Biomedical Research Unit, Glenfield Hospital, Leicester, UK </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Institut für Integrative und Experimentelle Genomik, Universität zu Lübeck, Lübeck, Germany </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Deutsches Zentrum für Herz-Kreislauf-Forschung e. V. (DZHK), Partnersite Hamburg, Lübeck, Kiel, Germany </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Bioinformatics and Biostatistics Support Hub (B/BASH), University of Leicester, University Road, Leicester, UK </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Gerontology and Geriatrics, Leiden University Medical Center, Leiden, The Netherlands </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München - German Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>German Center for Diabetes Research (DZD), Neuherberg, Germany </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Institute of Epidemiology II, Helmholtz Zentrum München - Germany Research Center for Environmental Health, Neuherberg, Germany </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Cardiovascular Genetics and Genomics Group, Atherosclerosis Research Unit, Department of Medicine Solna, Karolinska Institutet, Stockholm, Sweden </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Unit of Chronic Disease Epidemiology and Prevention, National Institute for Health and Welfare, Helsinki, Finland </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Deutsches Herzzentrum München, Technische Universität München, München, Germany </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>DZHK (German Centre for Cardiovascular Research), partner site Munich Heart Alliance, Munich, Germany </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Department of Medical Statistics and Bioinformatices, Leiden University Medical Center, Leiden, The Netherlands </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Centre for Paediatric Epidemiology and Biostatistics, UCL Institute of Child Health, London, UK </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>South Australian Health and Medical Research Institute, Adelaide, Australia </div><div class="fm-affl" id="A37" lang="en"><sup>37</sup>School of population Health and Sansom Institute, University of South Australia, Adelaide, Australia </div><div class="fm-affl" id="A38" lang="en"><sup>38</sup>Department of Clinical Chemistry, Fimlab Laboratories and School of Medicine, University of Tampere, Tampere, Finland </div><div class="fm-affl" id="A39" lang="en"><sup>39</sup>Steno Diabetes Center A/S, Gentofte, Denmark </div><div class="fm-affl" id="A40" lang="en"><sup>40</sup>Department of Medicine I, University Hospital Grosshadern, Ludwig-Maximilians-Universität, Munich, Germany </div><div class="fm-affl" id="A41" lang="en"><sup>41</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich, Germany </div><div class="fm-affl" id="A42" lang="en"><sup>42</sup>Department of Psychiatry, VU University Medical Center, Amsterdam, The Netherlands </div><div class="fm-affl" id="A43" lang="en"><sup>43</sup>Department of Systems Biology, Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark </div><div class="fm-affl" id="A44" lang="en"><sup>44</sup>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, UK </div><div class="fm-affl" id="A45" lang="en"><sup>45</sup>Genetic Epidemiology Group, Department of Health Sciences, University of Leicester, University Road, Leicester, UK </div><div class="fm-affl" id="A46" lang="en"><sup>46</sup>Department of Medicine, University of Turku and Division of Medicine, Turku University Hospital, Turku, Finland </div><div class="fm-affl" id="A47" lang="en"><sup>47</sup>Hjelt Institute, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A48" lang="en"><sup>48</sup>Department of Mental Health and Substance Abuse Services, National Institute for Health and Welfare, Helsinki, Finland </div><div class="fm-affl" id="A49" lang="en"><sup>49</sup>Department of Medical Sciences, Uppsala University, Akademiska Sjukhuset, Uppsala, Sweden </div><div class="fm-affl" id="A50" lang="en"><sup>50</sup>The Institute of Molecular and Cell Biology of the University of Tartu, Tartu, Estonia </div><div class="fm-affl" id="A51" lang="en"><sup>51</sup>Department of General Practice and Primary Health Care, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A52" lang="en"><sup>52</sup>Folkhälsan Research Centre, Helsinki, Finland </div><div class="fm-affl" id="A53" lang="en"><sup>53</sup>Helsinki University Hospital, Unit of Primary Health Care, Helsinki, Finland </div><div class="fm-affl" id="A54" lang="en"><sup>54</sup>Department of Health Promotion and Chronic Disease Prevention, National Institute for Health and Welfare, Helsinki, Finland </div><div class="fm-affl" id="A55" lang="en"><sup>55</sup>Department of Chronic Disease Prevention, National Institute for Health and Welfare, Turku, Finland </div><div class="fm-affl" id="A56" lang="en"><sup>56</sup>Research Centre of Applied and Preventive Cardiovascular Medicine, University of Turku, Finland </div><div class="fm-affl" id="A57" lang="en"><sup>57</sup>Department of Clinical Physiology and Nuclear Medicine, University of Turku and Turku University Hospital, Finland </div><div class="fm-affl" id="A58" lang="en"><sup>58</sup>Department of Epidemiology and Biostatistics, MRC Health Protection Agency (HPE), Centre for Environment and Health, School of Public Health, Imperial College London, UK </div><div class="fm-affl" id="A59" lang="en"><sup>59</sup>Institute of Health Sciences, University of Oulu, Oulu, Finland </div><div class="fm-affl" id="A60" lang="en"><sup>60</sup>Biocenter Oulu, University of Oulu, Oulu, Finland </div><div class="fm-affl" id="A61" lang="en"><sup>61</sup>Unit of Primary Care, Oulu University Hospital, Oulu, Finland </div><div class="fm-affl" id="A62" lang="en"><sup>62</sup>Department of Children and Young People and Families, National Institute for Health Welfare, Oulu, Finland </div><div class="fm-affl" id="A63" lang="en"><sup>63</sup>Faculty of Medicine, University of Iceland, Reykjavik, Iceland </div><div class="fm-affl" id="A64" lang="en"><sup>64</sup>Institute of Biomedicine, Anatomy, University of Helsinki, Helsinki, Finland </div><div class="fm-affl" id="A65" lang="en"><sup>65</sup>Minerva Foundation Institute for Medical Research, Helsinki, Finland </div><div class="fm-affl" id="A66" lang="en"><sup>66</sup>Haartman Institute, University of Helsinki and Helsinki University Central Hospital, Helsinki, Finland </div><div class="fm-affl" id="A67" lang="en"><sup>67</sup>Oxford National Institute for Health Research (NIHR) Biomedical Research Centre, Churchill Hospital, Oxford, UK </div><div class="fm-affl" id="A68" lang="en"><sup>68</sup>Department of Biostatistics, University of Liverpool, Liverpool, UK </div><div class="fm-affl" id="A69" lang="en"><sup>69</sup>Department of Genomics of Common Disease, School of Public Health, Imperial College London, Hammersmith Hospital, London, UK </div><div class="fm-affl" id="A70" lang="en"><sup>70</sup>Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK </div><div id="CR1"><strong>Corresponding author</strong> Prof. Samuli Ripatti Institute for Molecular Medicine Finland, FIMM P.O. Box 20, 00014 University of Helsinki, Finland <a href="mailto:dev@null" data-email="if.mmif@ittapir.ilumas" class="oemail">if.mmif@ittapir.ilumas</a></div><div id="FN1"><strong>Contributions</strong>
<p id="P2" class="p p-last">IS, MH, RM, APS, AnuM, VasL, LM, TF, EI, OK, VP, CML, UT, AaP, MIM, APM, IP, and SR designed and performed experiments, analysed data and wrote the paper. BM, ST, JoK, MatP, JuK, HJW, EI, JPM, THP and LF performed follow-up experiments and analysed the data. GT, SH, JJH, AI, CL, MarB, TE, JSR, CPN, CW and SG analysed cohort specific data. HS, JE, NJD, JaK, LL, CG, AndM, PES, LG, CMvanD, JGE, AJ, VeS, DIB, CP, OTR, MRJ, and KS designed cohort specific experiments. MatB, AJMdeC, EJdeG, JD, HG, AH, ASH, CH, JJHD, EH, MK, LCK, TL, ValL, PKEM, EM, MMN, NLP, BWJHP, MarP, AnP, JR, JHS, VaS, MDT, NT, EMvanL, JSV, SMW, and GW performed cohort specific experiments and analysed cohort specific data. All authors contributed to the research and reviewed the manuscript.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139947253916768_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25961943" target="pmc_ext" ref="reftype=publisher&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4757735/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary figures.</strong></div><div><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_figures.docx" data-ga-action="click_feat_suppl">NIHMS67068-supplement-Supplementary_figures.docx</a><span style="color:gray"> (17M)</span></div><div class="small guid">GUID:&nbsp;089A993B-BC13-4F71-8A16-692CC7ECFEA1</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary note.</strong></div><div><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_note.docx" data-ga-action="click_feat_suppl">NIHMS67068-supplement-Supplementary_note.docx</a><span style="color:gray"> (37K)</span></div><div class="small guid">GUID:&nbsp;9543DDE5-A7B2-4FF1-AEDF-A399EBFDC81A</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary table 8.</strong></div><div><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_table_8.xlsx" data-ga-action="click_feat_suppl">NIHMS67068-supplement-Supplementary_table_8.xlsx</a><span style="color:gray"> (202K)</span></div><div class="small guid">GUID:&nbsp;6380D165-FAB9-4DB5-B36C-16C977CB410D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary tables 1-7, 9-14.</strong></div><div><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_tables_1-7__9-14.docx" data-ga-action="click_feat_suppl">NIHMS67068-supplement-Supplementary_tables_1-7__9-14.docx</a><span style="color:gray"> (383K)</span></div><div class="small guid">GUID:&nbsp;16050383-08DF-4BD3-BE8A-6912FD40169C</div></div></dd></dl></div><div id="idm139947215143648" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139947215143648title">Abstract</h2><!--article-meta--><div><p id="P3" class="p p-first-last">Using a genome-wide screen of 9.6 million genetic variants achieved through 1000 Genomes imputation in 62,166 samples, we identify association to lipids in 93 loci including 79 previously identified loci with new lead-SNPs, 10 new loci, 15 loci with a low-frequency and 10 loci with missense lead-SNPs, and, 2 loci with an accumulation of rare variants. In six loci, SNPs with established function in lipid genetics (<em>CELSR2, GCKR, LIPC</em>, and <em>APOE</em>), or candidate missense mutations with predicted damaging function (<em>CD300LG</em> and <em>TM6SF2</em>), explained the locus associations. The low-frequency variants increased the proportion of variance explained, particularly for LDL-C and TC. Altogether, our results highlight the impact of low-frequency variants in complex traits and show that imputation offers a cost-effective alternative to re-sequencing.</p></div></div><div id="idm139947211679168" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Genome-wide association (GWA) studies have been successful in identifying genetic loci associated with complex diseases and traits. Due to the design of genotyping arrays, most of the associated variants have been common in population samples. While thousands of loci have been associated with complex diseases and traits, they so far typically explain only a fraction of the heritability<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305215">1</a></sup>.</p><p id="P5">It has now become possible to search for associations with variants that are less frequent than in previous GWA studies by the analysis of large numbers of samples using whole genome or exome sequencing approaches. However, costs have so far limited the possibility for sequencing of tens of thousands of samples likely needed to detect significant associations for low-frequency variants.</p><p id="P6">Stochastic imputation to individuals genotyped using genotyping arrays in large enough samples offers an alternative and cost-effective design to study associations of low-frequency and rare variants at a genome-wide level. GWA studies of circulating lipids have been highly successful in identifying loci with common variants with small effects<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305214">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305223">3</a></sup>. In previous large scale GWA studies, 157 loci have been shown to associate with lipids<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305192">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305205">3</a></sup>, but the strongest associations have almost exclusively been reported with common variants (minor allele frequency, MAF &gt; 5 %) in European datasets due to the study designs.</p><p id="P7">In contrast, previously published variants known to cause Mendelian forms of dyslipidemic syndromes and, more broadly, variants with known functional impact on lipids (FL SNPs) typically have low MAF (≤ 5%). While there are almost 40 loci where both FL SNPs and common SNPs, implicated in GWA studies, reside, it is often not known if these associations are driven by the same underlying haplotypes and if the Mendelian variants explain the association in population samples.</p><p id="P8" class="p">We sought to evaluate the impact of common (MAF &gt; 5%), low-frequency (0.5% &lt; MAF ≤ 5%) and rare (MAF ≤ 0.5%) genetic variants on circulating blood lipids in up to 62,166 European samples by imputing variants into the GWA cohorts using the sequence-based 1000 Genomes reference panel<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305225">4</a></sup> (Phase I interim release, June 2011). We aimed to answer the following questions: 1) what is the role of low-frequency and the burden of rare variants in the established lipid loci, 2) can a dense set of markers from 1000 Genomes-based imputation help to identify additional loci undetected in previous studies focused largely on common variants imputed up to less dense reference panels from the HapMap-project, 3) how do low-frequency and FL variants contribute to the overall trait variance compared to common variants.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">RESULTS</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Study Overview</h3><p id="P9" class="p p-first-last">To understand the contribution of low-frequency and rare genetic variation to circulating lipid concentrations, we undertook genome-wide imputation and association analysis in up to 62,166 individuals across 22 GWA cohorts of European ancestry. Within each cohort, we performed sex-stratified inverse-rank normalisation of high-density lipoprotein cholesterol (HDL-C), low-density lipoprotein cholesterol (LDL-C), triglycerides (TG) and total cholesterol (TC), after adjustment of each trait for age, age<sup>2</sup>, and study-specific covariates, including principal components to account for population structure. Case-control studies were further sub-divided according to original data selection disease status. Each cohort GWA genotype scaffold was imputed at up to ~37.4 million autosomal variants from the 1000 Genomes Project multi-ethnic reference panel<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305203">4</a></sup> (Phase I interim release, June 2011). Across a subset of studies, ~98% and ~95% of variants present in the reference panel with 1% &lt; MAF ≤ 5% and 0.5% &lt; MAF ≤ 1%, respectively, were well imputed, defined here by an IMPUTE<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305248">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305197">6</a></sup> info score of at least 0.4 (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 1</a>). However, as expected, imputation of rare variants (MAF ≤ 0.5%) proved more difficult, although ~65% of the rare variants polymorphic in the reference dataset were well imputed across the same subset of studies.</p></div><div id="S3" class="sec"><h3 id="S3title">Genome-wide screen for single variant associations</h3><p id="P10" class="p p-first">We first tested for association of over 9.6 million genotyped or successfully imputed SNPs, enabled by the 1000 Genomes imputation, with circulating HDL-C, LDL-C, TG and TC levels. Overall, we detected 93 loci with genome-wide significant association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>) to one or more lipid traits (p-value &lt; 5×10<sup>−8</sup>), of which 10 loci have not been associated to lipids before (<a href="/pmc/articles/PMC4757735/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 2A-J</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 3A-J</a>). Out of the 83 previously established lipid loci, 79 had a novel lead-SNP for at least one lipid trait in our analysis (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 2</a>). In 34 out of the 79 loci the linkage disequilibrium (LD) is <em>r</em><sup>2</sup> ≤ 40% (15 loci with <em>r</em><sup>2</sup> ≤ 5%) and in 56 loci the newly identified variant has not been present in HapMap2 imputation reference set used in previous studies. In 11 loci the novel lead-SNP had MAF ≤ 5% and an average effect size of 0.18 (in standard deviation, s.d., units) compared to the average effect size of 0.05 for the previously established common lead-SNP estimated in a cohort independent of the discovery scan to avoid bias due to the winner’s curse (<em>N</em> = 5,119, <a href="/pmc/articles/PMC4757735/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1A-B</span></a>). These include well-known lipid gene <em>LPA</em> for LDL-C (rs186696265, MAF = 0.8%, effect size = 0.26, p-value = 4.4×10<sup>−14</sup>, <em>r</em><sup>2</sup> = 0.1%). In addition we observed high effect lead-SNPs in <em>PCSK9</em> for LDL-C (rs11591147, MAF = 1.9%, effect size = 0.53, p-value = 2.2×10<sup>−92</sup> and <em>r</em><sup>2</sup> = 0.9%) and <em>APOE</em> for TC (rs7412, MAF = 7.1%, effect size = 0.41, p-value = 7.5×10<sup>−239</sup> and <em>r</em><sup>2</sup> = 1.6%) that were highlighted already in the Global Lipids Genetics Consortium fine-mapping analyses<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305221">3</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4757735/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947211289888" class="figure"><a href="/pmc/articles/PMC4757735/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4757735_emss-67068-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-67068-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4757735/bin/emss-67068-f0001.jpg"></a></div><div id="largeobj_idm139947211289888" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4757735/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4757735/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figures 1A-B</a></div><!--caption a7--><div class="caption"><p id="__p2">Change in p-value after analysis conditional on the new lead-SNP and comparison of new and previously reported lead-SNP effect sizes and allele frequencies per locus. In both figures, each of the arrows represent one locus and trait, where significant association was found in our screening and in one of the previously published large-scale screening studies<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305216">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305199">3</a></sup> and the colouring is based on the linkage disequilibrium (LD) between the old and new lead-SNP. The red ‘*’ represents for the new low-frequency lead-SNPs. In the Figure 1A, on the Y-axis are the −log10 p-values, arrows starting from the p-value seen in the unconditional analysis in Finnish subset (<em>N</em> = 12,834) and pointing to the p-value in analysis conditional on the new lead-SNP. In Figure 1B, each arrow starts from established lead-SNP effect and minor allele frequency (MAF) and points to the corresponding values for the new lead-SNP. The effects have been estimated in the FRCoreExome9702 sample set (<em>N</em> = 5,119), independent of the discovery set. Only results for loci with <em>r</em><sup>2</sup>&lt;0.4 have been presented for clarity.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Newly identified loci associated to HDL-C, LDL-C, TC and/or TG</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947218604144" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="center">
<em>Locus</em>
</th><th rowspan="2" colspan="1" valign="middle" align="center">CHR</th><th rowspan="2" colspan="1" valign="middle" align="center">Position<br>B37</th><th rowspan="2" colspan="1" valign="middle" align="center">rsID</th><th rowspan="2" colspan="1" valign="middle" align="center">Annotation</th><th rowspan="2" colspan="1" valign="middle" align="center">Primary<br>associated<br>trait</th><th rowspan="2" colspan="1" valign="middle" align="center">Secondary<br>associated<br>trait</th><th rowspan="2" colspan="1" valign="middle" align="center">Alleles<br>(effect/<br>other)</th><th rowspan="2" colspan="1" valign="middle" align="center">EAF</th><th colspan="4" rowspan="1" valign="middle" align="center">Meta-analysis</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Effect</th><th rowspan="1" colspan="1" valign="middle" align="center">SE</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value</th><th rowspan="1" colspan="1" valign="middle" align="center">
<em>N</em>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PROX1</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">214161820</td><td rowspan="1" colspan="1" valign="middle" align="center">rs340839<sup>*</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">UTR5</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.47</td><td rowspan="1" colspan="1" valign="middle" align="center">0.039</td><td rowspan="1" colspan="1" valign="middle" align="center">0.006</td><td rowspan="1" colspan="1" valign="middle" align="center">4.4×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">54836</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>CEP68</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">65284623</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2540948</td><td rowspan="1" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">C/T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.35</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.036</td><td rowspan="1" colspan="1" valign="middle" align="center">0.006</td><td rowspan="1" colspan="1" valign="middle" align="center">6.6×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">59939</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>PRKAG3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">219699999</td><td rowspan="1" colspan="1" valign="middle" align="center">rs78058190</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">HDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.141</td><td rowspan="1" colspan="1" valign="middle" align="center">0.020</td><td rowspan="1" colspan="1" valign="middle" align="center">5.7×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">52934</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>ADAMTS3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">4</td><td rowspan="1" colspan="1" valign="middle" align="center">73696709</td><td rowspan="1" colspan="1" valign="middle" align="center">rs117087731</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T/A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.308</td><td rowspan="1" colspan="1" valign="middle" align="center">0.051</td><td rowspan="1" colspan="1" valign="middle" align="center">2.310<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">23641</td></tr><tr><td rowspan="2" colspan="1" valign="middle" align="center">
<em>MTHFD2L</em>
</td><td rowspan="2" colspan="1" valign="middle" align="center">4</td><td rowspan="2" colspan="1" valign="middle" align="center">75084732</td><td rowspan="2" colspan="1" valign="middle" align="center">rs182616603</td><td rowspan="2" colspan="1" valign="middle" align="center">intronic</td><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.374</td><td rowspan="1" colspan="1" valign="middle" align="center">0.044</td><td rowspan="1" colspan="1" valign="middle" align="center">1.8×10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">42905</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">0.314</td><td rowspan="1" colspan="1" valign="middle" align="center">0.045</td><td rowspan="1" colspan="1" valign="middle" align="center">2.1×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">38420</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GPR85</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">7</td><td rowspan="1" colspan="1" valign="middle" align="center">112722196</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2255811</td><td rowspan="1" colspan="1" valign="middle" align="center">UTR3</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">G/A</td><td rowspan="1" colspan="1" valign="middle" align="center">0.25</td><td rowspan="1" colspan="1" valign="middle" align="center">0.041</td><td rowspan="1" colspan="1" valign="middle" align="center">0.007</td><td rowspan="1" colspan="1" valign="middle" align="center">2.3×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">59962</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>RMI2</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center">11454650</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7188861</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">HDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">A/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.20</td><td rowspan="1" colspan="1" valign="middle" align="center">0.044</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">6.9×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">60578</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>TM4SF5</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">17</td><td rowspan="1" colspan="1" valign="middle" align="center">4667984</td><td rowspan="1" colspan="1" valign="middle" align="center">rs193042029</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">G/T</td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.170</td><td rowspan="1" colspan="1" valign="middle" align="center">0.029</td><td rowspan="1" colspan="1" valign="middle" align="center">8.1×10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">50105</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GATA6</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">19907770</td><td rowspan="1" colspan="1" valign="middle" align="center">rs79588679</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T/C</td><td rowspan="1" colspan="1" valign="middle" align="center">0.17</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.049</td><td rowspan="1" colspan="1" valign="middle" align="center">0.009</td><td rowspan="1" colspan="1" valign="middle" align="center">3.6×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">53108</td></tr><tr><td rowspan="2" colspan="1" valign="middle" align="center">
<em>ZNF274</em>
</td><td rowspan="2" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="center">58681861</td><td rowspan="1" colspan="1" valign="middle" align="center">rs117492019</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">T/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.19</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.047</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">1.2×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">55371</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">58671267</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12983728</td><td rowspan="1" colspan="1" valign="middle" align="center">intergenic</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center">A/G</td><td rowspan="1" colspan="1" valign="middle" align="center">0.16</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.046</td><td rowspan="1" colspan="1" valign="middle" align="center">0.008</td><td rowspan="1" colspan="1" valign="middle" align="center">4.9×10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">58904</td></tr></tbody></table></div><div id="largeobj_idm139947218604144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4757735/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>*</sup>Present in the HapMap 2 reference panel.</div><div id="TFN2"><p id="P53" class="p p-first-last">Table presents the association meta-analysis results for the newly identified loci for the four tested lipid traits. Effect sizes are presented in s.d units. CHR: Chromosome, EAF: Effect allele frequency, SE: standard error of the effect, <em>N</em>: number of samples, HDL-C: high density lipoprotein</p></div></div></div><p id="P11">Using a formal conditional analysis, in <em>MAFB</em> locus, the new low-frequency lead-SNP with large effect size (effect size &gt; 0.2 and MAF ≤ 5%) explained the association of the previously identified lead-SNP in seven population cohorts (<em>N</em> = 12,834) though the linkage disequilibrium between the variants was less than <em>r</em><sup>2</sup>=5% (<a href="/pmc/articles/PMC4757735/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1A-B</span></a>). Additionally, there were 7 loci with two or more association lead-SNPs over 1Mb apart and with <em>r<sup>2</sup>&lt;5%</em>, but in all cases the individual level formal conditional analyses showed that the associations were completely explained by the known lipid SNPs in the regions (<em>ZCCHC11, TMEM48</em> and <em>PPAP2B</em> associations explained by rs11591147 in <em>PCSK9</em> locus, <em>OR</em>-cluster association by rs7395581 in <em>LRP4-MADD</em> locus, <em>CCDC79</em> association by rs73591976 in <em>LCAT-RANBP10</em> locus, and <em>PSG9</em> and <em>IRF2BP1</em> associations by rs7412 in <em>APOE</em> locus).</p><p id="P12">In five of the 79 loci, the lead-SNP was a missense variant pointing to either a well-established causal gene (<em>ANGPTL4, APOE, PCSK9</em> and <em>CILP2)</em> or to a new candidate gene <em>(ABCA6/8</em>). The <em>APOE</em> lead-SNP for TC, rs7412 (Arg176Cys, MAF = 7.1%, <em>r</em><sup>2</sup> = 0.7%) has been shown to associate with recessive familial type III hyperlipoproteinemia<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305241">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305218">8</a></sup> and the <em>PCSK9</em> lead-SNP for LDL, rs11591147 (Arg46Leu, MAF = 1.9%, <em>r</em><sup>2</sup> = 0.9%), with extreme LDL-C values<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305211">9</a></sup>. In the <em>ANGPTL4</em> locus, the lead-SNP in our GWA data is a predicted damaging missense variant, rs116843064 (Glu40Lys, MAF = 3.0%) with <em>r</em><sup>2</sup> = 1.8% with the previously associated common lead-SNP. The missense variant is associated with TG and HDL-C, and has previously been associated with extreme TG values<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305246">10</a></sup>. The <em>CILP2</em> lead-SNP, rs58542926 (Glu167Lys in <em>TM6SF2</em> gene, MAF = 7.8%, <em>r</em><sup>2</sup> = 98%), was associated to TC, myocardial infarction risk, and nonalcoholic fatty liver disease in two papers appearing while revising this manuscript<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305258">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305213">12</a></sup>. Our new lead-SNP in <em>ABCA6/8</em> locus, rs77542162 (Cys1359Arg in <em>ABCA6</em> gene, MAF = 2.0%, <em>r</em><sup>2</sup> = 0.6%) associates with LDL-C and TC (p-value = 1.6×10<sup>−18</sup> and p-value = 1.9×10<sup>−13</sup>, respectively).</p><p id="P13" class="p p-last">In the genome-wide screening we identified 10 loci that have not previously been associated to lipids (near PROX1, CEP68, PRKAG3, ADAMTS3, MTHFD2L, GPR85, RMI2, TM4SF5, GATA6 and <em>ZNF274</em>), with 4 having a low-frequency variant (MAF &lt; 5%) as the lead-SNP (lowest MAF = 0.7% rs182616603 in <em>MTHFD2L</em> locus; <a href="/pmc/articles/PMC4757735/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). All except one of the lead-SNPs have not been surveyed in the previous GWA studies based on HapMap 2 imputation. The one lead-SNP that has been present in the HapMap 2 imputation references is in the <em>PROX1</em> 5′UTR (rs340839 associated with TG, p-value = 4.4×10<sup>−12</sup>) and is correlated with a marker previously associated with fasting glucose and type 2 diabetes<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305207">13</a></sup> (rs340874, <em>r</em><sup>2</sup> = 74.7%). The lead-SNP in the HDL associated <em>PRKAG3</em> locus is located upstream to the gene, close to a transcription factor binding site. <em>PRKAG3</em> is a regulatory subunit of the AMP-activated protein kinase (AMPK), which has previously been shown to regulate lipid homeostasis<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305206">14</a></sup>.</p></div><div id="S4" class="sec"><h3 id="S4title">The role of variants with known functional impact on lipids in the general population</h3><p id="P14" class="p p-first">In 8 loci (<em>PCSK9, CELSR2-SORT1, GCKR, HLA</em>-region, <em>LPL, LIPC, CETP</em> and <em>APOE</em>), we tested if the variants known to cause Mendelian forms of dyslipidemic syndromes and, more broadly, with known functional impact on lipids, also explained the associations of the common lipid SNPs. These FL SNPs were identified through the Online Mendelian Inheritance in Man database search (OMIM; <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.omim.org</a>) and confirmed through literature, and SNPs previously reported to affect gene transcription or translation in cellular and/or animal models were taken forward into conditional analyses in seven population cohorts (<em>N</em> = 12,834; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a>, <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 3</a> and <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 3</a>).</p><p id="P15" class="p p-last">The FL SNPs explained the lead-SNP association (with p-value &lt; 5×10<sup>−8</sup> and conditional p-value &gt; 0.01 for the lead-SNP) in four of the 8 loci (<em>CELSR2-SORT1, GCKR, APOE</em> and <em>LIPC</em>; <a href="/pmc/articles/PMC4757735/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5A-G</a>). In <em>GCKR</em> and <em>APOE</em> loci, the lead-SNPs of our GWA screen were FL SNPs (rs1260326<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305236">15</a></sup>; Pro446Leu and rs7412<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305198">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305219">8</a></sup>; Arg158Cys, for <em>GCKR</em> and <em>APOE</em>, respectively). In the <em>GCKR</em> locus, rs1260326 explained the population-level association. Similarly, in <em>APOE</em> locus, the two FL SNPs rs7412 and rs429358<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305237">16</a></sup> (Cys112Arg) defining the APOE isoforms ε2, ε3 and ε4<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305209">17</a></sup> explained the association (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5D and 5E</a>). The <em>LIPC</em> association was explained by rs1800588<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305239">18</a></sup> (−514C-T, MAF = 25.1%) and rs113298164<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305244">19</a></sup> (Thr383Met, MAF = 1.4%) for TC and TG (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5F and 5G</a>) but not for HDL-C (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 5H</a>). All results for the conditional analyses are presented in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Tables 5A-D</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p6">Association results of unconditional analysis and analysis conditional on known Mendelian and functional lipid SNPs in loci where the functional SNPs explain the genome-wide association.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947211271024" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="middle" align="center">Locus</th><th rowspan="3" colspan="1" valign="middle" align="center">CHR</th><th rowspan="3" colspan="1" valign="middle" align="center">Trait</th><th colspan="7" rowspan="1" valign="middle" align="center">Lead-SNP in the unconditional analysis<hr></th></tr><tr><th rowspan="2" colspan="1" valign="middle" align="center">rsID</th><th rowspan="2" colspan="1" valign="middle" align="center">MAF</th><th colspan="2" rowspan="1" valign="middle" align="center">Unconditional</th><th colspan="3" rowspan="1" valign="middle" align="center">Conditional</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Effect<br>(SE)</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br><em>N</em></th><th rowspan="1" colspan="1" valign="middle" align="center">Covariate SNPs in the<br>model<br>(MAF)</th><th rowspan="1" colspan="1" valign="middle" align="center">Effect<br>(SE)</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br><em>N</em></th></tr></thead><tbody><tr><td rowspan="2" colspan="1" valign="middle" align="center">
<em>CELSR2-SORT1</em>
</td><td rowspan="2" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center">rs646776</td><td rowspan="1" colspan="1" valign="middle" align="center">0.216</td><td rowspan="1" colspan="1" valign="middle" align="center">0.159 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.31×10<sup>−25</sup><br>12739</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12740374 (21.6 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.001 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.958<br>12739</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center">rs646776</td><td rowspan="1" colspan="1" valign="middle" align="center">0.216</td><td rowspan="1" colspan="1" valign="middle" align="center">0.123 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.06×10<sup>−16</sup><br>12834</td><td rowspan="1" colspan="1" valign="middle" align="center">rs12740374 (21.6 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.001 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.959<br>12834</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>GCKR</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1260326</td><td rowspan="1" colspan="1" valign="middle" align="center">0.353</td><td rowspan="1" colspan="1" valign="middle" align="center">0.128 (0.013)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.44×10<sup>−23</sup><br>12815</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1260326 (35.3 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td></tr><tr><td rowspan="2" colspan="1" valign="middle" align="center">
<em>LIPC</em>
</td><td rowspan="2" colspan="1" valign="middle" align="center">15</td><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center">rs1800588</td><td rowspan="1" colspan="1" valign="middle" align="center">0.251</td><td rowspan="1" colspan="1" valign="middle" align="center">0.090 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.23×10<sup>−10</sup><br>12825</td><td rowspan="1" colspan="1" valign="middle" align="center">rs113298164 (1.4 %)<br>rs1800588 (25.1 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">−2×10<sup>−6</sup> (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.000<br>11893</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center">rs686958</td><td rowspan="1" colspan="1" valign="middle" align="center">0.252</td><td rowspan="1" colspan="1" valign="middle" align="center">0.085 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.86×10<sup>−9</sup><br>12801</td><td rowspan="1" colspan="1" valign="middle" align="center">rs113298164 (1.4 %)<br>rs1800588 (25.1 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.022 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.152<br>11873</td></tr><tr><td rowspan="3" colspan="1" valign="middle" align="center">
<em>APOE</em>
</td><td rowspan="3" colspan="1" valign="middle" align="center">19</td><td rowspan="1" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7412</td><td rowspan="1" colspan="1" valign="middle" align="center">0.048</td><td rowspan="1" colspan="1" valign="middle" align="center">0.648 (0.031)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.93×10<sup>−95</sup><br>12730</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7412 (4.8 %)<br>rs429358 (18.1 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">TC</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7412</td><td rowspan="1" colspan="1" valign="middle" align="center">0.048</td><td rowspan="1" colspan="1" valign="middle" align="center">0.456 (0.031)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.10×10<sup>−49</sup><br>12827</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7412 (4.8 %)<br>rs429358 (18.1 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center">rs483082</td><td rowspan="1" colspan="1" valign="middle" align="center">0.229</td><td rowspan="1" colspan="1" valign="middle" align="center">0.089 (0.015)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.74×10<sup>−9</sup><br>12799</td><td rowspan="1" colspan="1" valign="middle" align="center">rs7412 (4.8 %)<br>rs429358 (18.1 %)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td></tr></tbody></table></div><div id="largeobj_idm139947211271024" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4757735/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><p id="P54" class="p p-first-last">The table shows results for unconditional association analysis and analysis conditional on variants known to cause Mendelian forms of dyslipidemic syndromes and, more broadly, variants with known functional impact on lipids (FL SNPs). In case multiple candidate variants were observed in a locus, they were all included in the same model. Results for the lead-SNP from the unconditional analysis are presented from the meta-analysis of Finnish subset (<em>N</em> = 12,834). Effect sizes are presented in s.d units. CHR: Chromosome, MAF: Minor allele frequency, SE: Standard error of effect estimate, <em>N</em>: Number of samples, HDL-C: High-density lipoprotein cholesterol, LDL-C: Low-density lipoprotein cholesterol, TC: Total cholesterol, TG: Triglycerides.</p></div></div></div></div><div id="S5" class="sec"><h3 id="S5title">Search for novel functional candidate SNPs</h3><p id="P16" class="p p-first">We then searched for potential candidate causal SNPs in the lipid-associated (157 established and 10 novel) loci with a similar predicted function to well-characterized FL SNPs. We identified possible functional variants in four loci without known functional variants at the time of analysis (<em>MLXILP, LRP4-MADD, SOST-DUSP3</em> and <em>CILP2)</em>, and tested whether the identified variants explained the significant association seen in the locus (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 6</a>). The results of the conditional regression analyses for these four loci are presented in <a href="/pmc/articles/PMC4757735/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 6A-F</a>. In the <em>SOST-DUSP3</em> and <em>CILP2</em> loci, the functional candidates explained the genome-wide associations of the lead-SNPs in the region in the test set (in both loci, conditional p-value &gt; 0.01). In the <em>SOST-DUSP3</em> locus (<a href="/pmc/articles/PMC4757735/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>), a single low-frequency deleterious missense variant, rs72836561 (Arg82Cys, MAF = 2.7%, p-value = 1.36×10<sup>−8</sup>, effect size = 0.23) in the <em>CD300LG</em> gene, explained the whole regional association indicating <em>CD300LG</em> as a likely candidate gene for TG in the locus. The same variant has also recently been shown to associate with HDL-C and with fasting serum triacylglycerol in exome-wide association studies<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305191">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305224">21</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4757735/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947213576208" class="figure"><a href="/pmc/articles/PMC4757735/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4757735_emss-67068-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-67068-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4757735/bin/emss-67068-f0002.jpg"></a></div><div id="largeobj_idm139947213576208" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4757735/figure/F2/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947213575360" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4757735_emss-67068-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-67068-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4757735/bin/emss-67068-f0003.jpg"></a></div><div id="largeobj_idm139947213575360" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4757735/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4757735/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figures 2A-B</a></div><!--caption a7--><div class="caption"><p id="__p3">Regional association plots of the conditional analysis in loci where the new functional candidate SNPs explain the genome-wide association. Figure 2A illustrates the results in <em>SOST-DUSP3</em> locus for TG and Figure 2B results in <em>CILP2</em> locus for TC in Finnish subset (<em>N</em> = 12,834). In these figures, the first panel shows the −log10 p-value of each variant as a dot whose size reflects the effect size. The second panel shows the recombination rate in the area and the third panel shows the positions of genes. X-axis is the physical position in the genome. In grey are the association results from the unconditional analysis with green dots representing the new functional candidate SNPs. Black dots are the results from the conditional analysis.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p7">Association results of unconditional analysis and analysis conditional on the functional candidate SNPs in four loci with genome-wide significant p-value and new functional candidate SNPs.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947217548960" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="4" colspan="1" valign="middle" align="center">Locus</th><th rowspan="4" colspan="1" valign="middle" align="center">CHR</th><th rowspan="4" colspan="1" valign="middle" align="center">Trait</th><th colspan="10" rowspan="1" valign="middle" align="center">Testing the lead-SNP effects<hr></th></tr><tr><th rowspan="3" colspan="1" valign="middle" align="center">Tested lead-SNP<br>rsID</th><th rowspan="3" colspan="1" valign="middle" align="center">MAF</th><th colspan="2" rowspan="1" valign="middle" align="center">Unconditional</th><th colspan="6" rowspan="1" valign="middle" align="center">Conditional</th></tr><tr><th colspan="8" rowspan="1" valign="middle" align="center">
<hr>
</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">Effect<br>(SE)</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br><em>N</em></th><th rowspan="1" colspan="1" valign="middle" align="center">Candidate SNP used as a<br>single covariate in the model<br>(MAF)</th><th rowspan="1" colspan="1" valign="middle" align="center">Lead-SNP<br>effect when<br>adjusting for<br>single SNP<br>(SE)</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br><em>N</em></th><th rowspan="1" colspan="1" valign="middle" align="center">Candidate SNPs used<br>jointly as covariates in<br>the model<br>(MAF)</th><th rowspan="1" colspan="1" valign="middle" align="center">Lead-SNP<br>effect when<br>adjusting for<br>multiple<br>SNPs (SE)</th><th rowspan="1" colspan="1" valign="middle" align="center">p-value<br><em>N</em></th></tr></thead><tbody><tr><td rowspan="3" colspan="1" valign="middle" align="center">
<em>MLXIPL</em>
</td><td rowspan="3" colspan="1" valign="middle" align="center">7</td><td rowspan="3" colspan="1" valign="middle" align="center">TG</td><td rowspan="3" colspan="1" valign="middle" align="center">rs35797675</td><td rowspan="3" colspan="1" valign="middle" align="center">0.174</td><td rowspan="3" colspan="1" valign="middle" align="center">−0.119 (0.017)</td><td rowspan="3" colspan="1" valign="middle" align="center">7.89×10<sup>−13</sup><br>12810</td><td rowspan="1" colspan="1" valign="middle" align="center">rs35332062 (12%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.044 (0.017)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0073<br>12810</td><td rowspan="3" colspan="1" valign="middle" align="center">rs35332062 (12%)<br>rs3812316 (12%)</td><td rowspan="3" colspan="1" valign="middle" align="center">−0.044 (0.017)</td><td rowspan="3" colspan="1" valign="middle" align="center">0.0074<br>12810</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs3812316 (12%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.044 (0.017)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0073<br>12810</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="11" colspan="1" valign="middle" align="center">
<em>LRP4-MADD</em>
</td><td rowspan="11" colspan="1" valign="middle" align="center">11</td><td rowspan="11" colspan="1" valign="middle" align="center">HDL-C</td><td rowspan="11" colspan="1" valign="middle" align="center">rs2596401</td><td rowspan="11" colspan="1" valign="middle" align="center">0.426</td><td rowspan="11" colspan="1" valign="middle" align="center">−0.092 (0.013)</td><td rowspan="11" colspan="1" valign="middle" align="center">3.55×10<sup>−12</sup><br>11894</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2279238 (27%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.039 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0033<br>11894</td><td rowspan="11" colspan="1" valign="middle" align="center">rs2279238 (27%)<br>rs2290148 (27%)<br>rs34312154 (20%)<br>rs75352463 (3%)<br>rs1064608 (36%)<br>rs5896 (23%)</td><td rowspan="11" colspan="1" valign="middle" align="center">−0.031 (0.013)</td><td rowspan="11" colspan="1" valign="middle" align="center">0.020<br>11894</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2290148 (27%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.040 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">0.0028<br>11894</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs34312154 (20%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.060 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">6.0×10<sup>−6</sup><br>11894</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs75352463 (3%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.077 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">6.4×10<sup>−9</sup><br>11894</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1064608 (36%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.060 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">7.1×10<sup>−6</sup><br>11894</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs5896 (23%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.055 0.013</td><td rowspan="1" colspan="1" valign="middle" align="center">3.0×10<sup>−5</sup><br>11894</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">
<em>SOST-DUSP3</em>
</td><td rowspan="1" colspan="1" valign="middle" align="center">17</td><td rowspan="1" colspan="1" valign="middle" align="center">TG</td><td rowspan="1" colspan="1" valign="middle" align="center">rs72836561</td><td rowspan="1" colspan="1" valign="middle" align="center">0.027</td><td rowspan="1" colspan="1" valign="middle" align="center">0.234 (0.041)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.36×10<sup>−8</sup><br>12806</td><td rowspan="1" colspan="1" valign="middle" align="center">rs72836561 (3%)</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center">NA</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center"></td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="5" colspan="1" valign="middle" align="center">
<em>CILP2</em>
</td><td rowspan="5" colspan="1" valign="middle" align="center">19</td><td rowspan="5" colspan="1" valign="middle" align="center">LDL-C</td><td rowspan="5" colspan="1" valign="middle" align="center">rs8100204</td><td rowspan="5" colspan="1" valign="middle" align="center">0.137</td><td rowspan="5" colspan="1" valign="middle" align="center">−0.124 (0.019)</td><td rowspan="5" colspan="1" valign="middle" align="center">1.75×10<sup>−10</sup><br>12723</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2228603 (7%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.067 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.7×10<sup>−4</sup><br>12723</td><td rowspan="5" colspan="1" valign="middle" align="center">rs58542926 (6%)<br>rs187429064 (4%)</td><td rowspan="5" colspan="1" valign="middle" align="center">−0.006 (0.019)</td><td rowspan="5" colspan="1" valign="middle" align="center">0.744<br>12723</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs58542926 (6%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.064 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">9.9×10<sup>−4</sup><br>12723</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs187429064 (4%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.078 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.4×10<sup>−5</sup><br>12723</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="9" colspan="1" valign="middle" align="center">
<em>CILP2</em>
</td><td rowspan="9" colspan="1" valign="middle" align="center">19</td><td rowspan="9" colspan="1" valign="middle" align="center">TC</td><td rowspan="9" colspan="1" valign="middle" align="center">rs8100204</td><td rowspan="9" colspan="1" valign="middle" align="center">0.137</td><td rowspan="9" colspan="1" valign="middle" align="center">−0.149 (0.019)</td><td rowspan="9" colspan="1" valign="middle" align="center">1.12×10<sup>−14</sup><br>12819</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2228603 (7%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.079 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.8×10<sup>−5</sup><br>12819</td><td rowspan="9" colspan="1" valign="middle" align="center">rs58542926 (6%)<br>rs187429064 (4%)</td><td rowspan="9" colspan="1" valign="middle" align="center">−0.011 (0.019)</td><td rowspan="9" colspan="1" valign="middle" align="center">0.565<br>12819</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs58542926 (6%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.079 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.2×10<sup>−5</sup><br>12819</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs187429064 (4%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.093 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.3×10<sup>−6</sup><br>12819</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs2074300 (14%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.099 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.2×10<sup>−7</sup><br>12819</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12151060 (14%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.125 (0.019)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.0×10<sup>−10</sup><br>12819</td></tr><tr><td colspan="13" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="7" colspan="1" valign="middle" align="center">
<em>CILP2</em>
</td><td rowspan="7" colspan="1" valign="middle" align="center">19</td><td rowspan="7" colspan="1" valign="middle" align="center">TG</td><td rowspan="7" colspan="1" valign="middle" align="center">rs58434384</td><td rowspan="7" colspan="1" valign="middle" align="center">0.056</td><td rowspan="7" colspan="1" valign="middle" align="center">−0.158 (0.026)</td><td rowspan="7" colspan="1" valign="middle" align="center">2.06×10<sup>−9</sup><br>12807</td><td rowspan="1" colspan="1" valign="middle" align="center">rs2228603 (7%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.064 (0.026)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.015<br>12809</td><td rowspan="7" colspan="1" valign="middle" align="center">rs58542926 (6%)<br>rs187429064 (4%)</td><td rowspan="7" colspan="1" valign="middle" align="center">−0.047 (0.026)</td><td rowspan="7" colspan="1" valign="middle" align="center">0.075<br>12809</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs58542926 (6%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.048 (0.026)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.066<br>12809</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs187429064 (4%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.147 (0.026)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.5×10<sup>−8</sup><br>12809</td></tr><tr><td colspan="3" rowspan="1" valign="middle" align="center">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12151060 (14%)</td><td rowspan="1" colspan="1" valign="middle" align="center">−0.127 (0.026)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5×10<sup>−6</sup><br>12809</td></tr></tbody></table></div><div id="largeobj_idm139947217548960" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4757735/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN4"><p id="P55" class="p p-first-last">The table shows results for the lead-SNP (column rsID) before and after conditional analysis on the functional candidate variants. In case multiple candidate variants were observed in a locus, multiple linear regression models were fitted to explore the effect of each individual SNP and of all SNPs together. For CILP2 locus, result of the final model with two SNPs is represented. Results for the lead-SNP in the unconditional analysis are presented from the meta-analysis of Finnish subset (<em>N</em> = 12,834). Effect sizes are presented in s.d units. CHR: Chromosome, MAF: Minor allele frequency, SE: Standard error of effect estimate, <em>N</em>: Number of samples, HDL-C: High-density lipoprotein cholesterol, LDL-C: Low-density lipoprotein cholesterol, TC: Total cholesterol, TG: Triglycerides.</p></div></div></div><p id="P17" class="p p-last">In the <em>CILP2</em> locus for LDL-C, TC and TG, two independent missense variants (<em>r<sup>2</sup></em> = 0) in the <em>TM6SF2</em> gene, a deleterious missense variant rs187429064 (MAF = 3.6%, Leu156Pro; for TC effect size = −0.25 and p-value = 2.03×10<sup>−11</sup>) and a probably damaging missense variant rs58542926 (MAF = 6.3%, Glu167Lys; for TC effect size = −0.18 and p-value = 6.47×10<sup>−12</sup>), explained the lead-SNP association for LDL-C, TC and TG (<a href="/pmc/articles/PMC4757735/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 6D-F</a>, and <a href="/pmc/articles/PMC4757735/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 7</a> illustrates the result of the conditional analysis for TC).</p></div><div id="S6" class="sec"><h3 id="S6title">Biological profiling of <em>CD300LG</em> and <em>TM6SF2</em> genes in lipid metabolism</h3><p id="P18" class="p p-first">CD300LG (CD300 Molecule-Like Family Member G; also called nepmucin) is a type I cell surface glycoprotein that contains a single immunoglobulin (Ig) V-like domain<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305204">22</a></sup> and plays a role in lymphocyte binding and transmigration<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305251">23</a></sup>. The predicted damaging mutation (Arg82Cys) in our TG/HDL-C -associated variant rs72836561 is located in the Ig domain of <em>CD300LG</em>, which binds to lymphocytes. <em>CD300LG</em> is expressed in the vascular endothelial cells of various tissues, and is located both at the plasma membrane and intracellular vesicles<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305220">23</a>,<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305227">24</a></sup>. While CD300 family members have been demonstrated to bind lipids<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305233">25</a></sup>, the function of <em>CD300LG</em> in lipid metabolism has not been studied. TM6SF2 (Transmembrane 6 Superfamily Member 2)<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305256">26</a></sup>, is a multi-pass membrane protein, in which the predicted deleterious missense mutation (rs1874290064; Leu156Pro) locates to the predicted 5<sup>th</sup> transmembrane domain, and the probably damaging missense mutation (rs58542926; Glu167Lys) in the exposed non-transmembrane domain. <em>TM6SF2</em> gene has been shown to localize to endoplasmic reticulum (ER) compartment/ER-Golgi intermediate compartment (ERGIC) and influence TG secretion in liver cells<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305226">27</a></sup>. Additionally, the Glu167Lys missense mutation was shown to alter serum lipid profiles in humans and the knockdown of TM6SF2 in mice was shown to lead to increased liver triglyceride content and decreased very-low-density lipoprotein (VLDL) secretion<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305229">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305255">12</a></sup>.</p><p id="P19">We further characterized the two genes by using the Gene-Network database<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305196">28</a></sup> (<a href="http://genenetwork.nl/genenetwork" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genenetwork.nl/genenetwork</a>, see <strong>online Methods</strong> for details) for tissue specific expression, pathway analysis, and prediction of mice knockout phenotype, based on Mouse Genome Informatics (MGI; <a href="http://www.informatics.jax.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.informatics.jax.org</a>)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305247">29</a></sup>. We found that <em>CD300LG</em> gene is co-expressed with genes where knockout increases circulating very low-density lipoprotein (VLDL) particle levels in mice (prediction p-value = 1.4×10<sup>−9</sup>), in line with our phenotype of higher TG levels in humans carrying the deleterious missense variant of <em>CD300LG</em>. For <em>TM6SF2</em>, the MGI-based predictions, using co-expression of genes, show abnormal lipid levels (decreased LDL-C: prediction p-value = 8.6×10<sup>−19</sup>, decreased VLDL: prediction p-value = 2.5×10<sup>−29</sup> and decreased TC: prediction p-value = 6.3×10<sup>−24</sup>) amongst the most highly significant predictions, in line with the recent publications and our association results. All associated MGI-based knockout predictions (p-value &lt; 1e-6) are shown in the <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Tables 7A-B</a> and lists of genes with same and stronger MGI-based predictions can be found in the <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Table 8</a>.</p><p id="P20" class="p p-last">Both genes were found to be amongst the most highly expressed genes in tissues important for lipid absorption and/or metabolism based on the analysis using the Gene Network database (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Tables 9A-B</a>). <em>CD300LG</em> is highly expressed in muscles, plasma, and adipose tissue and <em>TM6SF2</em> in liver, plasma, and intestines. Furthermore, based on the gene expression network analysis, <em>TM6SF2</em> likely interacts with proteins involved in intestinal absorption (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 10</a>), and it is most highly predicted to function as lipid transporter (p-value = 1.05×10<sup>−14</sup>, prediction is based on co-expressed genes, <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 11</a>).</p></div><div id="S7" class="sec"><h3 id="S7title">Contribution of low-frequency variants to population lipid variation</h3><p id="P21" class="p p-first">We estimated the proportion of the variance of lipid traits explained by variants in the 157 previously established and 10 novel loci in an additional cohort of 5,119 individuals from Finrisk cohort (FRCoreExome9702), not included in our discovery meta-analysis. The lead-SNPs from all three GWA screens (Teslovich et al.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305252">2</a></sup>, Willer et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305201">3</a></sup> and this study) together with the FL SNPs and new functional candidate SNPs were divided into two groups based on their allele frequency in the FRCoreExome9702 dataset. Common SNPs explained 8.2% (TG), 11.9% (HDL-C), 16.3% (LDL-C), and 16.2% (TC) of the variance in lipid levels (<a href="/pmc/articles/PMC4757735/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Together with the low-frequency variants we now explain 9.3%, 12.8%, 19.5% and 18.8% of the variance in TG, HDL-C, LDL-C and TC, respectively.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4757735/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139947218606720" class="figure"><a href="/pmc/articles/PMC4757735/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4757735_emss-67068-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-67068-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4757735/bin/emss-67068-f0004.jpg"></a></div><div id="largeobj_idm139947218606720" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4757735/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4757735/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Proportion of total trait variance explained by the lead-SNPs and functional SNPs. The proportion of the trait variance explained by different SNP-sets has been estimated in independent FRCoreExome9702 sample set (<em>N</em> = 5,119). All lead-SNPs from the three association screens (Teslovich et al.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305202">2</a></sup>, Willer et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305188">3</a></sup> and our screen) together with the known functional lipid SNPs (FL SNPs) and new functional candidate SNPs were grouped based on their allele frequency in the FRCoreExome9702 dataset to common SNPs (allele frequency &gt; 5%) and to low-frequency SNPs (allele frequency ≤ 5%). The variance explained by these two groups is presented with blue bars. The proportion of variance explained by the FL SNPs and functional candidates is presented with the red bar.</p></div></div></div><p id="P22" class="p p-last">We also compared the contribution of our SNPs to the additive genetic variance estimated by a linear mixed model (LMM) applied to 10,472 individuals from six Finnish GWA cohorts (<strong>Online Methods</strong>) with those obtained from a large twin study<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305234">30</a></sup>. The narrow sense heritability estimates from the twin study were 40%, 51%, 51% and 33% and the mixed linear model estimates derived from the Finnish subset were 26%, 29%, 27% and 19% (for HDL-C, LDL-C, TC and TG, respectively. See <strong>Online Methods</strong> for details). We estimate that the SNP set explain at least 28.1%, 32.0%, 38.2% and 36.7% (narrow sense heritability) and at most 48.9%, 49.2%, 67.2% and 69.6% (LMM heritability estimate) of the additive genetic variance of TG, HDL-C, LDL-C and TC, respectively.</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Gene-based association analysis</h3><p id="P23" class="p p-first">To complement the single-variant tests for low-frequency variation we used GRANVIL<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305195">31</a></sup> to test for association of each lipid trait with accumulations of minor alleles (“mutational load”) at well imputed rare variants within genes in a subset of 30,463 individuals from 15 cohorts (Online Methods, <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 1</a>).</p><p id="P24">We observed genome-wide significant evidence of association (p-value &lt; 1.7×10<sup>−6</sup>, Bonferroni correction for 30,000 genes) of HDL-C with the mutational load of rare non-synonymous variants in <em>LIPC</em> (p-value = 2.1×10<sup>−7</sup>, mean MAF = 0.26%, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 8</a>). To further investigate the relationship between gene-based and single SNP association signals at this locus, we performed conditional analysis, adjusting the effect of the mutational load for the lead-SNP in our study (rs261291). The association of HDL-C with rare non-synonymous variants in <em>LIPC</em> remains relatively unchanged (conditional p-value = 3.6×10<sup>−6</sup>), suggesting that the mutational load of the gene is independent of the GWA signal at this locus.</p><p id="P25" class="p p-last">We identified two genes for which the mutational load of rare variants (irrespective of annotation) was associated with TG at genome-wide significance, both mapping to the APO-cluster: <em>ZNF259</em> (p-value = 1.5×10<sup>−11</sup>, mean MAF = 0.25%) and <em>APOA5</em> (p-value = 5.0×10<sup>−8</sup>, mean MAF = 0.24%). Conditional analyses, adjusting for the association lead-SNP (rs964184) at the APO-cluster, reduced the strength of association of rare variants in both <em>ZNF259</em> and <em>APOA5</em> with TG, but could not fully explain the effect of the mutational load of these genes (<a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 12</a>). As <em>ZNF259</em> and <em>APOA5</em> map within 2kb of each other, we further investigated the impact of LD on the association signal at the region with conditional analyses adjusting for the mutational load of each gene for that at the other (<strong>Online Methods</strong>). The strength of association of both genes was reduced, but not fully attenuated, after adjusting for the effect of the other (<em>ZNF259</em> conditional p-value = 1.4×10<sup>−5</sup>; <em>APOA5</em> conditional p-value = 6.3×10<sup>−4</sup>), suggesting the effects of rare variants in these two genes to be only partially correlated with each other.</p></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">DISCUSSION</h2><p id="P26" class="p p-first">Using 1000 Genomes imputed data with a dense SNP set, we were able to impute 9.6M common and low-frequency SNPs with good quality in 62,166 European samples. With GWA meta-analysis on these data, we identified 10 novel loci associated to blood lipids and new lead-SNPs in 79 previously known lipid loci. In 11 previously known loci, the new lead-SNP had a minor allele frequency ≤ 5% and, on average the newly identified low-frequency variants showed 3.6 times larger effect size compared to the corresponding lead-SNP in previous meta-analysis studies. Moreover, in four of the ten novel loci, the lead-SNPs were low-frequency variants.</p><p id="P27">Our association results reveal that low-frequency variants have a much larger contribution to lipid variation in the general population than has previously been shown<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305257">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305243">3</a></sup>. In several cases, the association that has previously been tagged by common variants is now led by variants with 0.5 – 5% allele frequency and larger effect sizes. The large effect sizes also show in the population lipid variance explained, where low-frequency variants add 3.2% to the LDL-C variance explained when adding on top of the common variants identified in previous reports or in our study, even though there are relatively few carriers of low-frequency variants in the general population.</p><p id="P28">While GWA studies have typically identified associations to lipid levels in cohorts with normal population variation, the known functional variants, some causing Mendelian forms of lipid syndromes and others changing the protein structure or disturbing the gene transcription, often been identified in patients and families with extreme lipid values. We found four regions where the population-level association was explained by known Mendelian and/or functional SNPs, suggesting that the effects of FL SNPs seem to generalize to European samples with normal lipid variation. Taken together, the successfully imputed and tested functional SNPs together with the new functional candidate variants explained 2.2 – 6.7% of the lipid variation in the population level.</p><p id="P29">As the FL SNPs explained the population-level association in four of the studied eight loci through LD-structure, we reversed this connection to identify potential functional candidate genes through SNPs with similar functional profile to the FL SNPs in lipid loci with no previous strong functional candidates. Using this strategy we identified two loci where missense variants with predicted damaging or deleterious functions explained the lead-SNP associations from the GWA meta-analysis, thus, together with previous evidence, supporting the role of <em>CD300LG</em> (TGs) and <em>TM6SF2</em> (TC, LDL-C, TGs) in lipid metabolism together with evidence from gene network analysis, gene expression correlations, predicted functions in mice, and expression patterns across organs each suggest potential links to lipid metabolism. <em>TM6SF2</em> was recently listed among genes potentially affecting LDL-C uptake in a recent siRNA screen focused on cellular lipid phenotypes within previously published blood lipid-associated GWA-loci.<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305194">32</a></sup> Additionally, two reports showing strong evidence for one of the two <em>TM6SF2</em> missense variants, Glu167Lys, on VLDL and TG metabolism were published during this study<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305238">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305189">12</a></sup>. However, in our data, this mutation alone does not explain the whole regional association, but together with a second missense mutation, with lower MAF and larger effect, the association was explained. Overall, our results reinforce the importance of <em>CD300LG</em> and <em>TM6SF2</em> for blood lipid levels in the general population.</p><p id="P30">In two established GWAS loci with common lead SNPs, our analyses revealed associations of the mutational load of rare variants with lipids. The association of HDL-C with rare variants in <em>LIPC</em> has been previously reported<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305235">33</a></sup>, and we also demonstrate that this signal is independent of the common lead GWAS SNP at this locus. We identified association on TG for the accumulation of <em>APOA5</em> rare variants as significant, but conditional analysis on the GWAs lead-SNP suggested that the single variant and gene-based associations are partially correlated. However, the GWA lead-SNP alone was not sufficient to fully explain the gene-based signal. An excess of minor alleles in <em>APOA5</em> has previously been associated with hypertriglyceridemia<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305254">34</a></sup>, but we report here an impact of this gene on TG at a population level. Although imputation enables recovery of ~65% of rare variants that are present in the 1000 Genomes haplotypes, many will not be represented in the reference panel. Re-sequencing in large sample sizes will be required to fully elucidate the role of rare variation at these GWA loci on HDL-C and TG and to inform functional studies to determine the underlying mechanisms mediated through these genes for the regulation of lipids.</p><p id="P31">In addition to the 93 loci identified, there were seven loci showing two or more association signals that were more than 1Mb distance from each other and the linkage disequilibrium between the lead SNPs were small (<em>r</em><sup>2</sup> ≤ 0.05). However, in formal conditional analyses of these loci using individual level data the most strongly associated SNPs in the locus explained also the other associations, even over a physical distance of 1Mb or more, or low level of LD. As these observations were only revealed after careful conditional testing of individual level data, they also highlight how challenging it is to interpret the association patterns using only summary level results on single SNP analyses.</p><p id="P32">There are some potential limitations to our genetic study. Although we used a dense sequence-based global imputation panel, it does not cover all low-frequency and rare variants in Europe. Similarly, although the imputation reference set included a large number of low-frequency SNPs and other variants with known functional impact on lipids, some were either missing from the panel or they were not polymorphic in our test sets of seven Finnish cohorts. Therefore we are likely missing some additional effects in our data. As more individuals are being sequenced and made available as imputation reference panels, more variants can also be imputed with high confidence and tested for associations.</p><p id="P33" class="p p-last">In conclusion, our study shows that low-frequency variants contribute significantly to population variance in lipid levels. The variants known to cause Mendelian forms of lipid syndromes and variants with known functional effects on lipid levels explain the common variant association in overlapping loci revealing a similar role of these variants in extreme patient series and in general populations. In addition, we found 10 new lipid loci for further investigations and for two previously known lipid loci we identified new candidate missense variants with predicted damaging function. When combining all the accumulated genetic evidence, we could explain up to 19.5% of the trait lipid variation. By considering the aggregate effects of rare variants within genes, we identified three transcripts associated with lipids in already established GWA loci that could not be fully explained by the common lead-SNPs reported in this study. Together, these observations show the important role of low-frequency functional SNPs in lipid level variation in the general population and present new therapeutic opportunities for treating dyslipidemias and preventing cardiovascular diseases. They also highlight that imputation is a cost-effective approach to assessing association with low-frequency and rare variants, without the need for costly re-sequencing experiments.</p></div><div id="S10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S10title">ONLINE METHODS</h2><div id="S11" class="sec sec-first"><h3 id="S11title">Genotype quality control and imputation</h3><p id="P34" class="p p-first-last">Before imputation all cohorts (see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note</a> for cohort information) went through a quality control (QC) pipeline with the following criteria: samples with genotype call rate &lt; 95%, sex discrepancies, excess heterozygosity and cryptic relatedness were removed. Additionally, ethnic outliers and MDS outliers were excluded. SNPs with minor allele frequency (MAF) &lt; 1%, call rate &lt; 95% (or &lt; 99% if the SNP has MAF &lt; 5%), failure of the Hardy-Weinberg Equilibrium (HWE) exact test (precise threshold depending on study) and sex chromosome SNPs were removed. Genotyping platforms, study-specific QC criteria and other details are presented in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 13</a>. The imputation of the datasets was performed using IMPUTE v2.0<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305222">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305231">6</a></sup> (unless stated otherwise) with 1000 Genomes June 2011 imputation reference panel with 2,188 haplotypes<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305208">4</a></sup> (<a href="http://www.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.1000genomes.org</a>).</p></div><div id="S12" class="sec"><h3 id="S12title">Phenotype measures</h3><p id="P35" class="p p-first-last">All four lipids, HDL-C, LDL-C, TC and TG were measured using basic enzymatic methods. Summary statistics of phenotypes in each cohort are presented in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 14</a>. Individuals with lipid-lowering medication were excluded and measures deviating more than 5 s.d. were set to missing. All four phenotypes were adjusted for age, age<sup>2</sup> and the first three genetic principal components. Principal components were derived from the GWA data using principal component analysis for the IBS sharing matrix for each study separately<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305212">35</a></sup>. Both the removal of outliers and the adjustments were done for males and females separately in each of the studies for all four traits. The residuals resulting from the adjustments where then inverse normal transformed to the <em>N</em>(0,1) distribution. The GenMets and DGI cohorts were additionally stratified by the Metabolic syndrome and Type 2 Diabetes case status, respectively. Only men were available in GerMIFS I and II and ULSAM. As NTR has related samples, males and females were analysed together in order to account for the relatedness.</p></div><div id="S13" class="sec"><h3 id="S13title">Single variant association- and Meta-analysis methods</h3><p id="P36" class="p p-first">A genome-wide association analysis was run in each of the cohorts separately (see <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 13</a> for software details). The association results were quality controlled centrally to have as harmonized dataset as possible. In the procedure, the following SNPs were removed: SNPs with minor allele count &lt; 3; SNPs with imputation quality Proper_INFO &lt; 0.4; duplicates; genotyped SNPs with HWE p-value &lt; 1×10<sup>−4</sup>. The meta-analysis was run using the GWAMA software tool<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305200">36</a>,<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305240">37</a></sup>, which uses fixed-effects inverse-variance weighted meta-analysis. Genomic control was applied to each of the cohorts in the meta-analysis. SNPs with &lt; 50% of the cohorts contributing or SNPs showing between-study heterogeneity of effect size (Cochran ’s Q test statistics, <em>I<sup>2</sup></em>&lt;50%) were discarded from the meta-analysis results. After these QC steps, the maximum number of SNPs in the analysis was 9,657,952.</p><p id="P37" class="p p-last">SNP associations with p-value &lt; 5×10<sup>−8</sup> were considered genome-wide significant and lead-SNPs were inquired to be at least 1Mb away from adjacent lead-SNPs. In areas with long-spanning linkage-disequilibrium, formal conditional analysis was performed in a subset of 12,834 Finnish samples to ensure the independence of the lead-SNPs.</p></div><div id="S14" class="sec"><h3 id="S14title">Search for known functional lipid SNPs</h3><p id="P38" class="p p-first-last">We searched the Online Mendelian Inheritance of Man (OMIM; <a href="http://www.omim.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.omim.org</a>) database for information on 167 loci, which had been found to associate with one of the studied traits (HDL-C, LDL-C, TC and TG) in either the two previously published GWA studies<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305217">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305193">3</a></sup> or in our genome-wide screening. In each of these 167 loci, every gene in a 2Mb area around the published lead variant was looked up for in the OMIM database and the variants associated with lipid related syndromes or population extreme lipid values were collected. Out of the 167 loci, 38 had OMIM-listed lipid SNP variants within the searched window. As our genotype data only includes SNP variants, deletions, insertions and other copy number variations could not be studied. Each of the OMIM-listed lipid SNPs was subsequently mapped to genome build 37 using dbSNP database for the rsID identification. Of the 38 loci, 18 had at least one polymorphic OMIM SNP in the imputed Finnish test set of seven cohorts (Corogene controls, FTC, GenMets, HBCS, NFBC1966, YFS and PredictCVD, combined <em>N</em> = 12,834). To be sure about the functionality of these SNPs, additional literature search was performed to find evidence of the effect on gene transcription or translation. Out of the 18 loci, 8 showed genome-wide significant association in the Finnish meta-analysis and had at least one variant with evidence for functional impact on lipid levels in cell or animal models.</p></div><div id="S15" class="sec"><h3 id="S15title">Formal conditional association analysis in loci containing known functional lipid SNPs</h3><p id="P39" class="p p-first">Formal conditional analyses were run using the Finnish test set of 7 cohorts (<em>N</em> = 12,834). Each of the cohorts was analysed separately with linear regression analysis implemented by SNPTest software (<a href="http://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/genetics_software/snptest/snptest.html</a>). In each cohort, imputation quality threshold of Proper_INFO &gt; 0.4 was applied. Each locus was analysed only for the trait(s) it had been previously reported to associate with in already published GWA studies. In the conditional analysis on a particular SNP(s), the phenotype was first adjusted with the SNP(s) and then a linear regression model was fitted for the remaining residuals. When we performed iterative conditional analyses in a locus, the signal was first conditioned on the most significant variant followed by conditioning on the top variant from the initial conditional analysis and so forth. Loci where the initial lead-SNP association in the conditional analyses was conditional p-value &lt; 0.01 and no further significant associations (conditional p-value &lt; 5×10<sup>−8</sup>) were found within the 2Mb window were considered to be explained.</p><p id="P40" class="p p-last">The results from the seven Finnish cohorts were combined using GWAMA. Because the conditional analyses were run for the pre-selected 2Mb windows only, genomic inflation factor (λ) correction could not be applied. However, we did not see substantial inflation in the genome-wide association analysis of all four traits in the seven Finnish cohorts (λ range 0.992 – 1.029 depending of the trait and cohort).</p></div><div id="S16" class="sec"><h3 id="S16title">Search for functional candidate SNPs</h3><p id="P41" class="p p-first-last">In order to explore suggestive functional variants causing association signals, which do not have lipid related OMIM listed variants in the locus, we selected 9 loci: <em>GALNT2</em>, <em>MLXILP</em>, <em>PPP1R3B</em>, <em>TRIB1</em>, <em>ADAMTS3</em>, <em>LRP4-MADD</em>, <em>SOST-DUSP3</em>, <em>CILP2</em> and <em>HNF4A</em>. These loci had been significantly associated with lipid traits in either previously published GWA studies<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305190">2</a>,<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305230">3</a></sup> or in our genome-wide screening as well as in the meta-analysis using 7 Finnish cohorts (<em>N</em> = 12,834). In each of these loci, 2Mb windows were searched for functional variants that had association p-value &lt; 5×10<sup>−4</sup>. Candidate SNPs were annotated using the Ensembl database and functional effects were predicted using the Provean<sup><a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305210">38</a></sup>, SIFT<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305249">39</a></sup> and PolyPhen<sup><a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305232">40</a></sup> databases. If a variant was annotated as a missense mutation with damaging prediction in at least one of the prediction databases, it was treated as a FL variant and formal conditional analysis was performed to investigate if it explains the association.</p></div><div id="S17" class="sec"><h3 id="S17title">Gene-network analysis</h3><p id="P42" class="p p-first">We used 2,206 principal components that had been derived from 77,840 Affymetrix microarrays (54,736 human, 17,081 mouse and 6,023 rat). Since gene-set enrichment analysis showed that each of these components are enriched for at least one biological pathway we used these components to developed a gene function prediction algorithm. To do so we first determine whether each of the components are enriched for a given gene-set, by performing a T-test (contrasting genes, known to be part of this pathways with all other genes), and transform the T-statistics into Z-scores. Subsequently we can the eigenvector coefficients of the 2,206 components for individual genes of interest with the Z-score profile of this gene-set, to predict the gene’s involvement in a specific pathway (details provided in Fehrmann et al.<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305228">28</a></sup>, see Cvejic <em>et al</em>.<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305253">41</a></sup> and Wood <em>et al</em>.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305250">29</a></sup> for a short description). We used a permutation strategy to determine significance of the predictions, controlling the false discovery rate at 5%. See <a href="http://genenetwork.nl/genenetwork" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genenetwork.nl/genenetwork</a> for the predictions. Based on the Mouse Genome Informatics (MGI; <a href="http://www.informatics.jax.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.informatics.jax.org</a>) mouse knockout database, we predicted <em>CD300LG</em> to increase circulating VLDL cholesterol levels. For <em>TM6SF2</em>, the most significantly predicted biological process was intestinal absorption. Only highly significant predictions (permuted p-value &lt; 1×10<sup>−6</sup>) were taken into account when profiling the two genes.</p><p id="P43" class="p p-last">We text-mined the sample descriptions provided by the experimenters who uploaded the microarray data to GEO. This text mining allowed us to determine the tissue or cell type for the majority of the samples. We subsequently used Wilcoxon-Mann-Whitney tests in the human samples from the Affymetrix U133 Plus 2.0 platform to ascertain how highly each gene was expressed in samples of a certain tissue or cell type as compared to samples in other tissues and cell types. We found that <em>CD300LG</em> is highly expressed in adipose tissue, heart, muscle and plasma and that <em>TM6SF2</em> is highly expressed in ileum and intestinal mucosa. See <a href="http://genenetwork.nl/genenetwork" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genenetwork.nl/genenetwork</a> for the expression of genes in different tissues and cell types.</p></div><div id="S18" class="sec"><h3 id="S18title">Modelling proportion of variance explained</h3><p id="P44" class="p p-first-last">To estimate the phenotypic variance explained by different types of SNPs we ran multiple linear regression models in R<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup> using the FRCoreExome9702 dataset (<em>N</em> = 5,119), an independent sample set from the Finrisk cohort. For the models all lead-SNPs (Teslovich et al.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305187">2</a></sup>, Willer et al.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305186">3</a></sup> and our study) together with FL SNPs and new functional candidates were divided into two groups based on the MAF of the variants in the FRCoreExome9702 dataset. The tested SNP sets were:
</p><ol class="enumerated" style="list-style-type:decimal" id="L1"><!--
list-behavior=enumerated
prefix-word=
mark-type=decimal
max-label-size=0
--><li><p id="P45">Common (MAF &gt; 5%) lead-SNPs and functional SNPs</p></li><li><p id="P46">Adding low-frequency (MAF ≤ 5%) lead-SNPs and functional SNPs to SNP-set 1.</p></li><li><p id="P47">FL SNPs and the three identified functional candidates.</p></li></ol><p>
These SNP-sets were used to explain the variation of the sex, age, age<sup>2</sup> and population stratification adjusted trait residuals. In order to apply linear models, TG was log-transformed before adjustments.</p></div><div id="S19" class="sec"><h3 id="S19title">Linear mixed model estimate of the variance explained by common SNPs</h3><p id="P48" class="p p-first-last">We estimated how much phenotypic variance a panel of 319,445 directly genotyped SNPs with MAF &gt; 1% in the autosomes explain using the linear mixed model approach implemented in GCTA<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305242">43</a></sup> (v.1.13). This estimate is a lower bound of the total additive genetic variance, because it only includes the contribution of the variants tagged by the panel of common SNPs that was used in the analysis. The analysis included samples from six Finnish cohorts (NFBC1966, Corogene controls, GenMets, YFS, HBCS and PredictCVD) for which we had access to the individual genotype data. All mixed model analyses excluded individuals in such a way that none of the remaining pairs of individuals had an estimated relatedness coefficient r &gt; 0.05 and the same trait values were used as with the individual SNP analyses. The sample sizes for the traits were 10,466 for HDL-C, 10,383 for LDL-C, 10,472 for TC and 10,451 for TG.</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Gene-based association analysis</h3><p id="P49" class="p p-first-last">Transcript boundaries were defined according to the UCSC human genome database. Within each study, GRANVIL<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_514305245">31</a></sup> was used to test for association of each trait with accumulations of minor alleles (“mutational load”) at successfully imputed rare variants (MAF ≤ 1% and info ≥ 0.4, <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Table 1</a>) within genes in a linear regression framework: (i) irrespective of annotation; and (ii) restricted to non-synonymous changes. Fixed-effects meta-analysis was performed by combining directed Z-scores from the regression analysis across studies, weighted by sample size. The significance threshold was set to p-value &lt; 1.7×10<sup>−6</sup> corresponding to a Bonferroni correction for 30,000 genes. Conditional analyses were performed to assess the evidence of association of traits the mutational load of a gene after accounting for the lead-SNP by including the genotype (under an additive model) of this variant as a covariate in the regression model. Conditional analyses were also performed to assess the independence of effects of rare variants in two genes by including the mutational load of one as a covariate in the regression model for the trait association with the other.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary figures</h4><div class="sup-box half_rhythm" id="d37e2306"><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_figures.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(17M, docx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supplementary note</h4><div class="sup-box half_rhythm" id="d37e2310"><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_note.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(37K, docx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Supplementary table 8</h4><div class="sup-box half_rhythm" id="d37e2314"><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_table_8.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(202K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Supplementary tables 1-7, 9-14</h4><div class="sup-box half_rhythm" id="d37e2318"><a href="/pmc/articles/PMC4757735/bin/NIHMS67068-supplement-Supplementary_tables_1-7__9-14.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(383K, docx)</sup></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Acknowledgements</h2><div class="sec"><p id="__p1">IS was partly funded by the Helsinki University Doctoral Programme in Biomedicine (DPBM). MH was funded by Manpei Suzuki Diabetes Foundation Grant-in-Aid for the young scientists working abroad. APM and AnuM acknowledged funding from the Wellcome Trust under awards WT098017, WT090532 and WT064890. VasL, LM, SH and IP were funded in part through the European Community's Seventh Framework Programme (FP7/2007-2013), ENGAGE project, grant agreement HEALTH-F4-2007- 201413. LM was in part sponsored by “5 per mille” contribution assigned to the University of Ferrara, income tax return year 2009 and in part from the ENGAGE Exchange and Mobility Program for ENGAGE training funds Program for ENGAGE training funds. MDT holds a Medical Research Council Senior Clinical Fellowship (G0902313). ST is supported by Sigrid Juselius foundation. JSR and CG have received funding by a grant of the RFBR (Russian Foundation for Basic Research)-Helmholtz Joint Research Group (12-04-91322). CG received funding from the European Union's Seventh Framework Program (FP7-Health-F5-2012) under grant agreement N<span class="underline">o</span> 305280 (MIMOmics). NJS holds a BHF Chair funded by the British Heart Foundation. CPN was funded by the NIHR Leicester Cardiovascular Biomedical Research Unit. VeS was supported by Finnish Foundation for Cardiovascular research, and the Finnish Academy (grant number 139635). EI was supported by the Academy of Finland Centre of Excellence in Biomembrane Research (272130), Academy of Finland (263841) and Sigrid Juselius Foundation. VP was supported by the University of Helsinki Postdoctoral researcher grant, Magnus Ehrnrooth foundation and the Kymenlaakso Cultural foundation. OK, JP and VP have received funding from the European Union’s Seventh Framework Programme (FP7/2007–2013) under grant agreement n°258068; EU-FP7-Systems Microscopy NoE. SR was supported by the Academy of Finland Center of Excellence in Complex Disease Genetics (213506 and 129680), Academy of Finland (251217), the Finnish foundation for Cardiovascular Research and the Sigrid Juselius Foundation. The High Throughput Biomedicine Unit of Institute for Molecular Medicine Finland, Ruusu Kovanen and Anna Uro are acknowledged for technical expertise. Jauhiainen M. is acknowledged for sharing his expertise in the article writing process. Cohort specific acknowledgements are in the <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Note</a>.</p></div></div><div id="idm139947213746112" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139947213746112title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="P50" class="p p-first"><strong>Competing financial interests</strong>
</p><p id="P51" class="p p-last">UT, GT, VaS and KS are employed by deCODE Genetics/Amgen inc.</p><p></p></div></div><div id="idm139947210283728" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139947210283728title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Gibson G. Rare and common variants: twenty arguments. <span><span class="ref-journal">Nat Rev Genet. </span>2012;<span class="ref-vol">13</span>:135–145.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4408201/">PMC free article</a>]</span> [<a href="/pubmed/22251874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Rare+and+common+variants:+twenty+arguments&amp;author=G+Gibson&amp;volume=13&amp;publication_year=2012&amp;pages=135-145&amp;pmid=22251874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Teslovich T, et al.  Biological, clinical and population relevance of 95 loci for blood lipids. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:707–713.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3039276/">PMC free article</a>]</span> [<a href="/pubmed/20686565" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological,+clinical+and+population+relevance+of+95+loci+for+blood+lipids&amp;author=T+Teslovich&amp;volume=466&amp;publication_year=2010&amp;pages=707-713&amp;pmid=20686565&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Willer C, et al.  Discovery and Refinement of Loci Associated with Lipid Levels. <span><span class="ref-journal">Nat. Genet. </span>2013 doi: 10.1038/ng.2797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3838666/">PMC free article</a>]</span> [<a href="/pubmed/24097068" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2797" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Discovery+and+Refinement+of+Loci+Associated+with+Lipid+Levels&amp;author=C+Willer&amp;publication_year=2013&amp;doi=10.1038/ng.2797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">1000 Genomes Project Consortium et al.  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">38</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=38&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Howie B, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=B+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Rall SJ, Weisgraber K, Innerarity T, Mahley R. Identical structural and receptor binding defects in apolipoprotein E2 in hypo-, normo-, and hypercholesterolemic dysbetalipoproteinemia. <span><span class="ref-journal">J. Clin. Invest. </span>1983;<span class="ref-vol">71</span>:1023–1031.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC436959/">PMC free article</a>]</span> [<a href="/pubmed/6300187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Invest&amp;title=Identical+structural+and+receptor+binding+defects+in+apolipoprotein+E2+in+hypo-,+normo-,+and+hypercholesterolemic+dysbetalipoproteinemia&amp;author=SJ+Rall&amp;author=K+Weisgraber&amp;author=T+Innerarity&amp;author=R+Mahley&amp;volume=71&amp;publication_year=1983&amp;pages=1023-1031&amp;pmid=6300187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Rall SJ, Weisgraber K, Innerarity T, Mahley R. Structural basis for receptor binding heterogeneity of apolipoprotein F from type III hyperlipoproteinemic subjects. <span><span class="ref-journal">Proc. Natl. Acad. Sci. U S A. </span>1982;<span class="ref-vol">79</span>:4696–4700.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC346743/">PMC free article</a>]</span> [<a href="/pubmed/6289314" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U+S+A&amp;title=Structural+basis+for+receptor+binding+heterogeneity+of+apolipoprotein+F+from+type+III+hyperlipoproteinemic+subjects&amp;author=SJ+Rall&amp;author=K+Weisgraber&amp;author=T+Innerarity&amp;author=R+Mahley&amp;volume=79&amp;publication_year=1982&amp;pages=4696-4700&amp;pmid=6289314&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Cohen J, Boerwinkle E, Mosley TJ, Hobbs H. Sequence variations in PCSK9, low LDL, and protection against coronary heart disease. <span><span class="ref-journal">N. Engl. J. Med. </span>2006;<span class="ref-vol">354</span>:1264–1272.</span> [<a href="/pubmed/16554528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med&amp;title=Sequence+variations+in+PCSK9,+low+LDL,+and+protection+against+coronary+heart+disease&amp;author=J+Cohen&amp;author=E+Boerwinkle&amp;author=TJ+Mosley&amp;author=H+Hobbs&amp;volume=354&amp;publication_year=2006&amp;pages=1264-1272&amp;pmid=16554528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Romeo S, et al.  Population-based resequencing of ANGPTL4 uncovers variations that reduce triglycerides and increase HDL. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:513–516.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2762948/">PMC free article</a>]</span> [<a href="/pubmed/17322881" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Population-based+resequencing+of+ANGPTL4+uncovers+variations+that+reduce+triglycerides+and+increase+HDL&amp;author=S+Romeo&amp;volume=39&amp;publication_year=2007&amp;pages=513-516&amp;pmid=17322881&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Holmen OL, et al.  Systematic evaluation of coding variation identifies a candidate causal variant in TM6SF2 influencing total cholesterol and myocardial infarction risk. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:345–351.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4169222/">PMC free article</a>]</span> [<a href="/pubmed/24633158" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Systematic+evaluation+of+coding+variation+identifies+a+candidate+causal+variant+in+TM6SF2+influencing+total+cholesterol+and+myocardial+infarction+risk&amp;author=OL+Holmen&amp;volume=46&amp;publication_year=2014&amp;pages=345-351&amp;pmid=24633158&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Kozlitina J, Smargris E, Stender S, Nordestgaard BG, Zhou HH, Tybjærg-Hansen A, Vogt TF, Hobbs HH, Cohen JC. Exome-wide association study identifies a TM6SF2 variant that confers susceptibility to nonalcoholic fatty liver disease. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:352–356.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3969786/">PMC free article</a>]</span> [<a href="/pubmed/24531328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Exome-wide+association+study+identifies+a+TM6SF2+variant+that+confers+susceptibility+to+nonalcoholic+fatty+liver+disease&amp;author=J+Kozlitina&amp;author=E+Smargris&amp;author=S+Stender&amp;author=BG+Nordestgaard&amp;author=HH+Zhou&amp;volume=46&amp;publication_year=2014&amp;pages=352-356&amp;pmid=24531328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Dupuis J, et al.  New genetic loci implicated in fasting glucose homeostasis and their impact on type 2 diabetes risk. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:105–116.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3018764/">PMC free article</a>]</span> [<a href="/pubmed/20081858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=New+genetic+loci+implicated+in+fasting+glucose+homeostasis+and+their+impact+on+type+2+diabetes+risk&amp;author=J+Dupuis&amp;volume=42&amp;publication_year=2010&amp;pages=105-116&amp;pmid=20081858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Kahn B, Alquier T, Carling D, Hardie D. AMP-activated protein kinase: ancient energy gauge provides clues to modern understanding of metabolism. <span><span class="ref-journal">Cell. Metab. </span>2005;<span class="ref-vol">1</span>:15–25.</span> [<a href="/pubmed/16054041" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Metab&amp;title=AMP-activated+protein+kinase:+ancient+energy+gauge+provides+clues+to+modern+understanding+of+metabolism&amp;author=B+Kahn&amp;author=T+Alquier&amp;author=D+Carling&amp;author=D+Hardie&amp;volume=1&amp;publication_year=2005&amp;pages=15-25&amp;pmid=16054041&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Beer N, et al.  The P446L variant in GCKR associated with fasting plasma glucose and triglyceride levels exerts its effect through increased glucokinase activity in liver. <span><span class="ref-journal">Hum. Mol. Genet. </span>2009;<span class="ref-vol">18</span>:4081–4088.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2758140/">PMC free article</a>]</span> [<a href="/pubmed/19643913" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=The+P446L+variant+in+GCKR+associated+with+fasting+plasma+glucose+and+triglyceride+levels+exerts+its+effect+through+increased+glucokinase+activity+in+liver&amp;author=N+Beer&amp;volume=18&amp;publication_year=2009&amp;pages=4081-4088&amp;pmid=19643913&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Weisgraber K, Rall SJ, Mahley R. Human E apoprotein heterogeneity. Cysteine-arginine interchanges in the amino acid sequence of the apo-E isoforms. <span><span class="ref-journal">J. Biol. Chem. </span>1981;<span class="ref-vol">256</span>:9077–9083.</span> [<a href="/pubmed/7263700" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=Human+E+apoprotein+heterogeneity.+Cysteine-arginine+interchanges+in+the+amino+acid+sequence+of+the+apo-E+isoforms&amp;author=K+Weisgraber&amp;author=SJ+Rall&amp;author=R+Mahley&amp;volume=256&amp;publication_year=1981&amp;pages=9077-9083&amp;pmid=7263700&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Ghebranious N, Ivacic L, Mallum J, Dokken C. Detection of ApoE E2, E3 and E4 alleles using MALDI-TOF mass spectrometry and the homogeneous mass-extend technology. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:e149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1243648/">PMC free article</a>]</span> [<a href="/pubmed/16204452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=Detection+of+ApoE+E2,+E3+and+E4+alleles+using+MALDI-TOF+mass+spectrometry+and+the+homogeneous+mass-extend+technology&amp;author=N+Ghebranious&amp;author=L+Ivacic&amp;author=J+Mallum&amp;author=C+Dokken&amp;volume=33&amp;publication_year=2005&amp;pages=e149&amp;pmid=16204452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Deeb S, Peng R. The C-514T polymorphism in the human hepatic lipase gene promoter diminishes its activity. <span><span class="ref-journal">J. Lipid. Res. </span>2000;<span class="ref-vol">41</span>:155–158.</span> [<a href="/pubmed/10627514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid.+Res&amp;title=The+C-514T+polymorphism+in+the+human+hepatic+lipase+gene+promoter+diminishes+its+activity&amp;author=S+Deeb&amp;author=R+Peng&amp;volume=41&amp;publication_year=2000&amp;pages=155-158&amp;pmid=10627514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Durstenfeld A, Ben-Zeev O, Reue K, Stahnke G, Doolittle M. Molecular characterization of human hepatic lipase deficiency. In vitro expression of two naturally occurring mutations. <span><span class="ref-journal">Arterioscler Thromb. </span>1994;<span class="ref-vol">14</span>:381–385.</span> [<a href="/pubmed/8123642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler+Thromb&amp;title=Molecular+characterization+of+human+hepatic+lipase+deficiency.+In+vitro+expression+of+two+naturally+occurring+mutations&amp;author=A+Durstenfeld&amp;author=O+Ben-Zeev&amp;author=K+Reue&amp;author=G+Stahnke&amp;author=M+Doolittle&amp;volume=14&amp;publication_year=1994&amp;pages=381-385&amp;pmid=8123642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Liu DJ, et al.  Meta-analysis of gene-level tests for rare variant association. <span><span class="ref-journal">Nat. Genet. </span>2013 doi: 10.1038/ng.2852.</span> epub ahead of print:Dec 15. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3939031/">PMC free article</a>]</span> [<a href="/pubmed/24336170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.2852" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Meta-analysis+of+gene-level+tests+for+rare+variant+association&amp;author=DJ+Liu&amp;publication_year=2013&amp;doi=10.1038/ng.2852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Albrechtsen A, et al.  Exome sequencing-driven discovery of coding polymorphisms associated with common metabolic phenotypes. <span><span class="ref-journal">Diabetologia. </span>2013;<span class="ref-vol">56</span>:298–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3536959/">PMC free article</a>]</span> [<a href="/pubmed/23160641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=Exome+sequencing-driven+discovery+of+coding+polymorphisms+associated+with+common+metabolic+phenotypes&amp;author=A+Albrechtsen&amp;volume=56&amp;publication_year=2013&amp;pages=298-310&amp;pmid=23160641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Takatsu H, Hase K, Ohmae M, Ohshima S, Hashimoto K, Taniura N, Yamamoto A, Ohno H. CD300 antigen like family member G: A novel Ig receptor like protein exclusively expressed on capillary endothelium. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>2006;<span class="ref-vol">348</span>:183–191.</span> [<a href="/pubmed/16876123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun&amp;title=CD300+antigen+like+family+member+G:+A+novel+Ig+receptor+like+protein+exclusively+expressed+on+capillary+endothelium&amp;author=H+Takatsu&amp;author=K+Hase&amp;author=M+Ohmae&amp;author=S+Ohshima&amp;author=K+Hashimoto&amp;volume=348&amp;publication_year=2006&amp;pages=183-191&amp;pmid=16876123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Umemoto E, et al.  Nepmucin, a novel HEV sialomucin, mediates L-selectin-dependent lymphocyte rolling and promotes lymphocyte adhesion under flow. <span><span class="ref-journal">J. Exp. Med. </span>2006;<span class="ref-vol">203</span>:1603–1614.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118321/">PMC free article</a>]</span> [<a href="/pubmed/16754720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Med&amp;title=Nepmucin,+a+novel+HEV+sialomucin,+mediates+L-selectin-dependent+lymphocyte+rolling+and+promotes+lymphocyte+adhesion+under+flow&amp;author=E+Umemoto&amp;volume=203&amp;publication_year=2006&amp;pages=1603-1614&amp;pmid=16754720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Jin S, Umemoto E, Tanaka T, Shimomura Y, Tohya K, Yang BG, Jang MH, Hirata T, Miyasaka M. Nepmucin/CLM-9, an Ig domain-containing sialomucin in vascular endothelial cells, promotes lymphocyte transendothelial migration in vitro. <span><span class="ref-journal">FEBS Lett. </span>2008;<span class="ref-vol">582</span>:3018–3024.</span> [<a href="/pubmed/18675811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett&amp;title=Nepmucin/CLM-9,+an+Ig+domain-containing+sialomucin+in+vascular+endothelial+cells,+promotes+lymphocyte+transendothelial+migration+in+vitro&amp;author=S+Jin&amp;author=E+Umemoto&amp;author=T+Tanaka&amp;author=Y+Shimomura&amp;author=K+Tohya&amp;volume=582&amp;publication_year=2008&amp;pages=3018-3024&amp;pmid=18675811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Cannon JP, O’Driscoll M, Litman GW. Specific lipid recognition is a general feature of CD300 and TREM molecules. <span><span class="ref-journal">Immunogenetics. </span>2012;<span class="ref-vol">64</span>:39–47.</span> [<a href="/pubmed/21800138" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunogenetics&amp;title=Specific+lipid+recognition+is+a+general+feature+of+CD300+and+TREM+molecules&amp;author=JP+Cannon&amp;author=M+O’Driscoll&amp;author=GW+Litman&amp;volume=64&amp;publication_year=2012&amp;pages=39-47&amp;pmid=21800138&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Carim-Todd L, Escarceller M, Estivill X, Sumoy L. Cloning of the novel gene TM6SF1 reveals conservation of clusters of paralogous genes between human chromosomes 15q24--&gt;q26 and 19p13.3--&gt;p12. <span><span class="ref-journal">Cytogenet. Cell Genet. </span>2000;<span class="ref-vol">90</span>:255–260.</span> [<a href="/pubmed/11124529" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytogenet.+Cell+Genet&amp;title=Cloning+of+the+novel+gene+TM6SF1+reveals+conservation+of+clusters+of+paralogous+genes+between+human+chromosomes+15q24-->q26+and+19p13.3-->p12&amp;author=L+Carim-Todd&amp;author=M+Escarceller&amp;author=X+Estivill&amp;author=L+Sumoy&amp;volume=90&amp;publication_year=2000&amp;pages=255-260&amp;pmid=11124529&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Mahdessian H, Taxiarchis A, Popov S, Silveira A, Franco-Cereceda A, Hamsten A, Eriksson P, Van’t Hooft F. TM6SF2 is a regulator of liver fat metabolism influencing triglyceride secretion and hepatic lipid droplet content. <span><span class="ref-journal">Proc Natl Sci U S A. </span>2014</span> epub ahead of print June 4<sup>th</sup>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4066487/">PMC free article</a>]</span> [<a href="/pubmed/24927523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Sci+U+S+A&amp;title=TM6SF2+is+a+regulator+of+liver+fat+metabolism+influencing+triglyceride+secretion+and+hepatic+lipid+droplet+content&amp;author=H+Mahdessian&amp;author=A+Taxiarchis&amp;author=S+Popov&amp;author=A+Silveira&amp;author=A+Franco-Cereceda&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Fehrmann SN, et al.  Gene expression analysis identifies global gene dosage sensitivity in cancer. <span><span class="ref-journal">Nat Genet. </span>2015</span> epub ahead of print Jan 12<sup>th</sup>. [<a href="/pubmed/25581432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Gene+expression+analysis+identifies+global+gene+dosage+sensitivity+in+cancer&amp;author=SN+Fehrmann&amp;publication_year=2015&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Wood AR, et al.  Defining the role of common variation in the genomic and biological architecture of adult human height. <span><span class="ref-journal">Nat Gen. </span>2014 doi: 10.1038/ng.3097.</span> Epub Oct 5. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4250049/">PMC free article</a>]</span> [<a href="/pubmed/25282103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3097" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Gen&amp;title=Defining+the+role+of+common+variation+in+the+genomic+and+biological+architecture+of+adult+human+height&amp;author=AR+Wood&amp;publication_year=2014&amp;doi=10.1038/ng.3097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">van Dongen J, Willemsen G, Chen WM, de Geus EJC, Boomsma DI. Heritability of metabolic syndrome traits in a large population-based sample. <span><span class="ref-journal">J. Lipid Res. </span>2013;<span class="ref-vol">54</span>:2914–2923.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3770104/">PMC free article</a>]</span> [<a href="/pubmed/23918046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res&amp;title=Heritability+of+metabolic+syndrome+traits+in+a+large+population-based+sample&amp;author=J+van+Dongen&amp;author=G+Willemsen&amp;author=WM+Chen&amp;author=EJC+de+Geus&amp;author=DI+Boomsma&amp;volume=54&amp;publication_year=2013&amp;pages=2914-2923&amp;pmid=23918046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Mägi R, et al.  Genome-wide association analysis of imputed rare variants: application to seven common complex diseases. <span><span class="ref-journal">Genet Epidemiol. </span>2012;<span class="ref-vol">36</span>:785–796.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3569874/">PMC free article</a>]</span> [<a href="/pubmed/22951892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Genome-wide+association+analysis+of+imputed+rare+variants:+application+to+seven+common+complex+diseases&amp;author=R+Mägi&amp;volume=36&amp;publication_year=2012&amp;pages=785-796&amp;pmid=22951892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Blattman P, Schubert C, Pepperkok R, Runz H. RNAi-based functional profiling of loci from blood lipid genome-wide association studies indentifies genes with cholesterol-regulatory function. <span><span class="ref-journal">PLoS Genet. </span>2013;<span class="ref-vol">9</span>:e1003338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3585126/">PMC free article</a>]</span> [<a href="/pubmed/23468663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=RNAi-based+functional+profiling+of+loci+from+blood+lipid+genome-wide+association+studies+indentifies+genes+with+cholesterol-regulatory+function&amp;author=P+Blattman&amp;author=C+Schubert&amp;author=R+Pepperkok&amp;author=H+Runz&amp;volume=9&amp;publication_year=2013&amp;pages=e1003338&amp;pmid=23468663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Service SK, et al.  Re-sequencing expands our understanding of the phenotypic impact of variants at GWAS loci. <span><span class="ref-journal">PLos Genet. </span>2014;<span class="ref-vol">10</span>:e1004147.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3907339/">PMC free article</a>]</span> [<a href="/pubmed/24497850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLos+Genet&amp;title=Re-sequencing+expands+our+understanding+of+the+phenotypic+impact+of+variants+at+GWAS+loci&amp;author=SK+Service&amp;volume=10&amp;publication_year=2014&amp;pages=e1004147&amp;pmid=24497850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Johansen CT, et al.  Excess of rare variants in genes identified by genome-wide association study of hypertriglyceridemia. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:682–687.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3017369/">PMC free article</a>]</span> [<a href="/pubmed/20657596" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Excess+of+rare+variants+in+genes+identified+by+genome-wide+association+study+of+hypertriglyceridemia&amp;author=CT+Johansen&amp;volume=42&amp;publication_year=2010&amp;pages=682-687&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Mägi R, Morris A. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288. doi: 10.1186/1471-2105-11-288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1186%2F1471-2105-11-288" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R+Mägi&amp;author=A+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;doi=10.1186/1471-2105-11-288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Mägi R, Lindgren C, Morris A. Meta-analysis of sex-specific genome-wide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:846–853.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3410525/">PMC free article</a>]</span> [<a href="/pubmed/21104887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Meta-analysis+of+sex-specific+genome-wide+association+studies&amp;author=R+Mägi&amp;author=C+Lindgren&amp;author=A+Morris&amp;volume=34&amp;publication_year=2010&amp;pages=846-853&amp;pmid=21104887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Choi Y, Sims G, Murphy S, Miller J, Chan A. Predicting the Functional Effect of Amino Acid Substitutions and Indels. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e46688.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3466303/">PMC free article</a>]</span> [<a href="/pubmed/23056405" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Predicting+the+Functional+Effect+of+Amino+Acid+Substitutions+and+Indels&amp;author=Y+Choi&amp;author=G+Sims&amp;author=S+Murphy&amp;author=J+Miller&amp;author=A+Chan&amp;volume=7&amp;publication_year=2012&amp;pages=e46688&amp;pmid=23056405&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Ng P, Henikoff S. Predicting deleterious amino acid substitutions. <span><span class="ref-journal">Genome Res. </span>2001;<span class="ref-vol">11</span>:863–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC311071/">PMC free article</a>]</span> [<a href="/pubmed/11337480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Predicting+deleterious+amino+acid+substitutions&amp;author=P+Ng&amp;author=S+Henikoff&amp;volume=11&amp;publication_year=2001&amp;pages=863-874&amp;pmid=11337480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Adzhubei I, et al.  A method and server for predicting damaging missense mutations. <span><span class="ref-journal">Nat. Methods. </span>2010;<span class="ref-vol">7</span>:248–249.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="/pubmed/20354512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations&amp;author=I+Adzhubei&amp;volume=7&amp;publication_year=2010&amp;pages=248-249&amp;pmid=20354512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Cvejic A, et al.  SMIM1 underlies the Vel blood group and influences red blood cell traits. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:542–545.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4179282/">PMC free article</a>]</span> [<a href="/pubmed/23563608" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=SMIM1+underlies+the+Vel+blood+group+and+influences+red+blood+cell+traits&amp;author=A+Cvejic&amp;volume=45&amp;publication_year=2013&amp;pages=542-545&amp;pmid=23563608&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">R Development Core Team  <span><span class="ref-journal">R: A language and environment for statistical computing. </span>2008</span> &lt; <a href="http://R-project.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://R-project.org</a>&gt;.</span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4757735&amp;issue-id=265410&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139947215143648title">Abstract</a></li><li><a href="#S1title">RESULTS</a></li><li><a href="#S9title">DISCUSSION</a></li><li><a href="#S10title">ONLINE METHODS</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S21title">Acknowledgements</a></li><li><a href="#idm139947213746112title">Footnotes</a></li><li><a href="#idm139947210283728title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4757735/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4757735/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4757735/pdf/emss-67068.pdf">PDF (1.2M)</a></li> | <li><a href="#" data-citationid="PMC4757735" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4757735%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4757735%2F&amp;text=The%20impact%20of%20low-frequency%20and%20rare%20variants%20on%20lipid%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4757735%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25961943" data-aiid="4757735" data-aid="4757735" data-iid="265410" data-domainid="358" data-domain="wtpa" data-accid="PMC4757735" data-md5="79ef6ee9ec14c8e3610eac05463dd74c"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:38:05-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B7924DD57971100000000099305A1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4757735%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4757735/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>