<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4133584/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4133584/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis"><meta name="citation_authors" content="Nicola Whiffin, Fay J. Hosking, Susan M. Farrington, Claire Palles, Sara E. Dobbins, Lina Zgaga, Amy Lloyd, Ben Kinnersley, Maggie Gorman, Albert Tenesa, Peter Broderick, Yufei Wang, Ella Barclay, Caroline Hayward, Lynn Martin, Daniel D. Buchanan, Aung Ko Win, John Hopper, Mark Jenkins, Noralane M. Lindor, Polly A. Newcomb, Steve Gallinger, David Conti, Fred Schumacher, Graham Casey, Tao Liu, The Swedish Low-Risk Colorectal Cancer Study Group, Harry Campbell, Annika Lindblom, Richard S. Houlston, Ian P. Tomlinson, Malcolm G. Dunlop"><meta name="citation_date" content="1 September 2014"><meta name="citation_issue" content="17"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="4729"><meta name="citation_doi" content="10.1093/hmg/ddu177"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4133584/?report=abstract"><meta name="citation_pmid" content="24737748"><meta name="DC.Title" content="Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Nicola Whiffin"><meta name="DC.Contributor" content="Fay J. Hosking"><meta name="DC.Contributor" content="Susan M. Farrington"><meta name="DC.Contributor" content="Claire Palles"><meta name="DC.Contributor" content="Sara E. Dobbins"><meta name="DC.Contributor" content="Lina Zgaga"><meta name="DC.Contributor" content="Amy Lloyd"><meta name="DC.Contributor" content="Ben Kinnersley"><meta name="DC.Contributor" content="Maggie Gorman"><meta name="DC.Contributor" content="Albert Tenesa"><meta name="DC.Contributor" content="Peter Broderick"><meta name="DC.Contributor" content="Yufei Wang"><meta name="DC.Contributor" content="Ella Barclay"><meta name="DC.Contributor" content="Caroline Hayward"><meta name="DC.Contributor" content="Lynn Martin"><meta name="DC.Contributor" content="Daniel D. Buchanan"><meta name="DC.Contributor" content="Aung Ko Win"><meta name="DC.Contributor" content="John Hopper"><meta name="DC.Contributor" content="Mark Jenkins"><meta name="DC.Contributor" content="Noralane M. Lindor"><meta name="DC.Contributor" content="Polly A. Newcomb"><meta name="DC.Contributor" content="Steve Gallinger"><meta name="DC.Contributor" content="David Conti"><meta name="DC.Contributor" content="Fred Schumacher"><meta name="DC.Contributor" content="Graham Casey"><meta name="DC.Contributor" content="Tao Liu"><meta name="DC.Contributor" content="The Swedish Low-Risk Colorectal Cancer Study Group"><meta name="DC.Contributor" content="Harry Campbell"><meta name="DC.Contributor" content="Annika Lindblom"><meta name="DC.Contributor" content="Richard S. Houlston"><meta name="DC.Contributor" content="Ian P. Tomlinson"><meta name="DC.Contributor" content="Malcolm G. Dunlop"><meta name="DC.Date" content="2014 Sep 1"><meta name="DC.Identifier" content="10.1093/hmg/ddu177"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis"><meta property="og:type" content="article"><meta property="og:description" content="To identify common variants influencing colorectal cancer (CRC) risk, we performed a meta-analysis of five genome-wide association studies, comprising 5626 cases and 7817 controls of European descent. We conducted replication of top ranked single nucleotide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4133584/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4133584/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4133584/pdf/ddu177.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8DAFF1DD569FD10000000005E2039D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4133584%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC4133584</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2014 Sep 1; 23(17): 4729–4737. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Apr 15. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu177" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddu177</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4133584</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24737748">24737748</a></div></div></div></div><h1 class="content-title">Identification of susceptibility loci for colorectal cancer in a genome-wide meta-analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Whiffin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901966490144" co-class="co-affbox">Nicola Whiffin</a>,<sup>1</sup> <a href="/pubmed/?term=Hosking%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901971053616" co-class="co-affbox">Fay J. Hosking</a>,<sup>1</sup> <a href="/pubmed/?term=Farrington%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902042244432" co-class="co-affbox">Susan M. Farrington</a>,<sup>2</sup> <a href="/pubmed/?term=Palles%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901966487424" co-class="co-affbox">Claire Palles</a>,<sup>3</sup> <a href="/pubmed/?term=Dobbins%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901962010160" co-class="co-affbox">Sara E. Dobbins</a>,<sup>1</sup> <a href="/pubmed/?term=Zgaga%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902043203792" co-class="co-affbox">Lina Zgaga</a>,<sup>2</sup> <a href="/pubmed/?term=Lloyd%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902043201808" co-class="co-affbox">Amy Lloyd</a>,<sup>1</sup> <a href="/pubmed/?term=Kinnersley%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901972459920" co-class="co-affbox">Ben Kinnersley</a>,<sup>1</sup> <a href="/pubmed/?term=Gorman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901972457936" co-class="co-affbox">Maggie Gorman</a>,<sup>3</sup> <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901969918432" co-class="co-affbox">Albert Tenesa</a>,<sup>4</sup> <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901968442032" co-class="co-affbox">Peter Broderick</a>,<sup>1</sup> <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901968440048" co-class="co-affbox">Yufei Wang</a>,<sup>1</sup> <a href="/pubmed/?term=Barclay%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901970041904" co-class="co-affbox">Ella Barclay</a>,<sup>3</sup> <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901970039920" co-class="co-affbox">Caroline Hayward</a>,<sup>5</sup> <a href="/pubmed/?term=Martin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901974901632" co-class="co-affbox">Lynn Martin</a>,<sup>3</sup> <a href="/pubmed/?term=Buchanan%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901974899648" co-class="co-affbox">Daniel D. Buchanan</a>,<sup>6</sup> <a href="/pubmed/?term=Win%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901975212336" co-class="co-affbox">Aung Ko Win</a>,<sup>7</sup> <a href="/pubmed/?term=Hopper%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902009749968" co-class="co-affbox">John Hopper</a>,<sup>7</sup> <a href="/pubmed/?term=Jenkins%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902009747984" co-class="co-affbox">Mark Jenkins</a>,<sup>7</sup> <a href="/pubmed/?term=Lindor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901970225536" co-class="co-affbox">Noralane M. Lindor</a>,<sup>8</sup> <a href="/pubmed/?term=Newcomb%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901970223552" co-class="co-affbox">Polly A. Newcomb</a>,<sup>9</sup> <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901972827264" co-class="co-affbox">Steve Gallinger</a>,<sup>10</sup> <a href="/pubmed/?term=Conti%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902009656688" co-class="co-affbox">David Conti</a>,<sup>11</sup> <a href="/pubmed/?term=Schumacher%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902009654704" co-class="co-affbox">Fred Schumacher</a>,<sup>11</sup> <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901965465824" co-class="co-affbox">Graham Casey</a>,<sup>11</sup> <a href="/pubmed/?term=Liu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901965463840" co-class="co-affbox">Tao Liu</a>,<sup>12</sup> The Swedish Low-Risk Colorectal Cancer Study Group, <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902008035520" co-class="co-affbox">Harry Campbell</a>,<sup>13</sup> <a href="/pubmed/?term=Lindblom%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901968375792" co-class="co-affbox">Annika Lindblom</a>,<sup>12</sup> <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901968373808" co-class="co-affbox">Richard S. Houlston</a>,<sup>1,</sup><sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Tomlinson%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139901970068464" co-class="co-affbox">Ian P. Tomlinson</a>,<sup>4,</sup><sup>*</sup><sup>†</sup> and  <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748" class="affpopup" co-rid="_co_idm139902009560656" co-class="co-affbox">Malcolm G. Dunlop</a><sup>2,</sup><sup>*</sup><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139901966490144"><h3 class="no_margin">Nicola Whiffin</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Whiffin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Nicola Whiffin</a></div></div><div id="_co_idm139901971053616"><h3 class="no_margin">Fay J. Hosking</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Hosking%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Fay J. Hosking</a></div></div><div id="_co_idm139902042244432"><h3 class="no_margin">Susan M. Farrington</h3><p><sup>2</sup>Colon Cancer Genetics Group,, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Farrington%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Susan M. Farrington</a></div></div><div id="_co_idm139901966487424"><h3 class="no_margin">Claire Palles</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Palles%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Claire Palles</a></div></div><div id="_co_idm139901962010160"><h3 class="no_margin">Sara E. Dobbins</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Dobbins%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Sara E. Dobbins</a></div></div><div id="_co_idm139902043203792"><h3 class="no_margin">Lina Zgaga</h3><p><sup>2</sup>Colon Cancer Genetics Group,, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Zgaga%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Lina Zgaga</a></div></div><div id="_co_idm139902043201808"><h3 class="no_margin">Amy Lloyd</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Lloyd%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Amy Lloyd</a></div></div><div id="_co_idm139901972459920"><h3 class="no_margin">Ben Kinnersley</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Kinnersley%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Ben Kinnersley</a></div></div><div id="_co_idm139901972457936"><h3 class="no_margin">Maggie Gorman</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Gorman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Maggie Gorman</a></div></div><div id="_co_idm139901969918432"><h3 class="no_margin">Albert Tenesa</h3><p><sup>4</sup>The Roslin Institute, University of Edinburgh, Easter Bush, Roslin, EH25 9RG, UK</p><div>Find articles by <a href="/pubmed/?term=Tenesa%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Albert Tenesa</a></div></div><div id="_co_idm139901968442032"><h3 class="no_margin">Peter Broderick</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Broderick%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Peter Broderick</a></div></div><div id="_co_idm139901968440048"><h3 class="no_margin">Yufei Wang</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Wang%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Yufei Wang</a></div></div><div id="_co_idm139901970041904"><h3 class="no_margin">Ella Barclay</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Barclay%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Ella Barclay</a></div></div><div id="_co_idm139901970039920"><h3 class="no_margin">Caroline Hayward</h3><p><sup>5</sup>Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Hayward%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Caroline Hayward</a></div></div><div id="_co_idm139901974901632"><h3 class="no_margin">Lynn Martin</h3><p><sup>3</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK</p><div>Find articles by <a href="/pubmed/?term=Martin%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Lynn Martin</a></div></div><div id="_co_idm139901974899648"><h3 class="no_margin">Daniel D. Buchanan</h3><p><sup>6</sup>Cancer and Population Studies Group, Queensland Institute of Medical Research, Queensland, Australia</p><div>Find articles by <a href="/pubmed/?term=Buchanan%20DD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Daniel D. Buchanan</a></div></div><div id="_co_idm139901975212336"><h3 class="no_margin">Aung Ko Win</h3><p><sup>7</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Win%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Aung Ko Win</a></div></div><div id="_co_idm139902009749968"><h3 class="no_margin">John Hopper</h3><p><sup>7</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Hopper%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">John Hopper</a></div></div><div id="_co_idm139902009747984"><h3 class="no_margin">Mark Jenkins</h3><p><sup>7</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Victoria, Australia</p><div>Find articles by <a href="/pubmed/?term=Jenkins%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Mark Jenkins</a></div></div><div id="_co_idm139901970225536"><h3 class="no_margin">Noralane M. Lindor</h3><p><sup>8</sup>Department of Health Sciences Research, Mayo Clinic, Scottsdale, AZ, USA</p><div>Find articles by <a href="/pubmed/?term=Lindor%20NM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Noralane M. Lindor</a></div></div><div id="_co_idm139901970223552"><h3 class="no_margin">Polly A. Newcomb</h3><p><sup>9</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Newcomb%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Polly A. Newcomb</a></div></div><div id="_co_idm139901972827264"><h3 class="no_margin">Steve Gallinger</h3><p><sup>10</sup>Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Gallinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Steve Gallinger</a></div></div><div id="_co_idm139902009656688"><h3 class="no_margin">David Conti</h3><p><sup>11</sup>Department of Preventive Medicine, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Conti%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">David Conti</a></div></div><div id="_co_idm139902009654704"><h3 class="no_margin">Fred Schumacher</h3><p><sup>11</sup>Department of Preventive Medicine, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Schumacher%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Fred Schumacher</a></div></div><div id="_co_idm139901965465824"><h3 class="no_margin">Graham Casey</h3><p><sup>11</sup>Department of Preventive Medicine, University of Southern California, Los Angeles, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Casey%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Graham Casey</a></div></div><div id="_co_idm139901965463840"><h3 class="no_margin">Tao Liu</h3><p><sup>12</sup>Department of Molecular Medicine and Surgery, Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Liu%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Tao Liu</a></div></div><div id="_co_idm139902008035520"><h3 class="no_margin">Harry Campbell</h3><p><sup>13</sup>Centre for Population Health Sciences, University of Edinburgh, Teviot Place, Edinburgh, EH8 9AG, UK</p><div>Find articles by <a href="/pubmed/?term=Campbell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Harry Campbell</a></div></div><div id="_co_idm139901968375792"><h3 class="no_margin">Annika Lindblom</h3><p><sup>12</sup>Department of Molecular Medicine and Surgery, Karolinska Institute, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Lindblom%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Annika Lindblom</a></div></div><div id="_co_idm139901968373808"><h3 class="no_margin">Richard S. Houlston</h3><p><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</p><div>Find articles by <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Richard S. Houlston</a></div></div><div id="_co_idm139901970068464"><h3 class="no_margin">Ian P. Tomlinson</h3><p><sup>4</sup>The Roslin Institute, University of Edinburgh, Easter Bush, Roslin, EH25 9RG, UK</p><div>Find articles by <a href="/pubmed/?term=Tomlinson%20IP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Ian P. Tomlinson</a></div></div><div id="_co_idm139902009560656"><h3 class="no_margin">Malcolm G. Dunlop</h3><p><sup>2</sup>Colon Cancer Genetics Group,, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</p><div>Find articles by <a href="/pubmed/?term=Dunlop%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24737748">Malcolm G. Dunlop</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139902042624016_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139902042624016_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139902042624016_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139902042624016_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Molecular and Population Genetics, Division of Genetics and Epidemiology, Institute of Cancer Research, Sutton, Surrey SM2 5NG, UK</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Colon Cancer Genetics Group,, Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Wellcome Trust Centre for Human Genetics, Oxford, UK</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>The Roslin Institute, University of Edinburgh, Easter Bush, Roslin, EH25 9RG, UK</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Institute of Genetics and Molecular Medicine, University of Edinburgh and MRC Human Genetics Unit, Western General Hospital Edinburgh, Crewe Road, Edinburgh, EH4 2XU, UK</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Cancer and Population Studies Group, Queensland Institute of Medical Research, Queensland, Australia</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Centre for Molecular, Environmental, Genetic and Analytic Epidemiology, The University of Melbourne, Victoria, Australia</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Health Sciences Research, Mayo Clinic, Scottsdale, AZ, USA</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Cancer Prevention Program, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Samuel Lunenfeld Research Institute, Mount Sinai Hospital, Toronto, ON, Canada</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Department of Preventive Medicine, University of Southern California, Los Angeles, CA, USA</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Department of Molecular Medicine and Surgery, Karolinska Institute, Stockholm, Sweden</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Centre for Population Health Sciences, University of Edinburgh, Teviot Place, Edinburgh, EH8 9AG, UK</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed: E-mails: <a href="mailto:dev@null" data-email="ku.ca.rci@notsluoh.drahcir" class="oemail">ku.ca.rci@notsluoh.drahcir</a> (R.H.); <a href="mailto:dev@null" data-email="ku.ca.xo.llew@tnai" class="oemail">ku.ca.xo.llew@tnai</a> (I.T.); <a href="mailto:dev@null" data-email="ku.ca.de.mmgi@polnud.mloclam" class="oemail">ku.ca.de.mmgi@polnud.mloclam</a> (M.D.)</div><div id="AN1"><sup>†</sup>These authors should be considered to have equal status.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139902042624016_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Jan 7; Revised 2014 Apr 3; Accepted 2014 Apr 10.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139902042624016_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2014. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4133584/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4133584/bin/supp_23_17_4729__index.html" data-ga-action="click_feat_suppl">supp_23_17_4729__index.html</a><span style="color:gray"> (1.9K)</span></div><div class="small guid">GUID:&nbsp;85D711C7-72C5-401F-828C-A255E9C5685B</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp.pdf" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp.pdf</a><span style="color:gray"> (542K)</span></div><div class="small guid">GUID:&nbsp;83636B03-7034-4B91-982D-40144E459DEF</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table1.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table1.docx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID:&nbsp;81253CF0-68D1-4B81-9B12-EF19AC8C6429</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table2.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table2.docx</a><span style="color:gray"> (23K)</span></div><div class="small guid">GUID:&nbsp;23A4B54F-6CCC-441B-AA58-B76C5AB22294</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table3.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table3.docx</a><span style="color:gray"> (22K)</span></div><div class="small guid">GUID:&nbsp;678EBF62-8F29-47EE-8A24-EDB8F1516DC6</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table4.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table4.docx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;F4E03554-2E5F-42A3-AE87-CA4FAABE6C58</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table5.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table5.docx</a><span style="color:gray"> (19K)</span></div><div class="small guid">GUID:&nbsp;3B430BEF-F7B9-4C05-A48C-BF13428B2597</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table6.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table6.docx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID:&nbsp;64C69180-8A95-475C-8885-4157E74254CD</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table7.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table7.docx</a><span style="color:gray"> (17K)</span></div><div class="small guid">GUID:&nbsp;0000ED28-B98B-4728-B9E8-8AAA8AFDDF38</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table8.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table8.docx</a><span style="color:gray"> (31K)</span></div><div class="small guid">GUID:&nbsp;4AD23700-C115-4F46-B115-B3529BC3D5F3</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table9.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table9.docx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;55C91230-635B-4EB3-BF0E-2FDA7C65070F</div>
<div><a href="/pmc/articles/PMC4133584/bin/supp_ddu177_ddu177supp_table10.docx" data-ga-action="click_feat_suppl">supp_ddu177_ddu177supp_table10.docx</a><span style="color:gray"> (11K)</span></div><div class="small guid">GUID:&nbsp;C368C43F-0A9F-4A9C-BB51-421DB801E03E</div>
</div></dd></dl></div><div id="idm139901971920480" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139901971920480title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">To identify common variants influencing colorectal cancer (CRC) risk, we performed a meta-analysis of five genome-wide association studies, comprising 5626 cases and 7817 controls of European descent. We conducted replication of top ranked single nucleotide polymorphisms (SNPs) in additional series totalling 14 037 cases and 15 937 controls, identifying a new CRC risk locus at 10q24.2 [rs1035209; odds ratio (OR) = 1.13, <em>P</em> = 4.54 × 10<sup>−11</sup>]. We also performed meta-analysis of our studies, with previously published data, of several recently purported CRC risk loci. We failed to find convincing evidence for a previously reported genome-wide association at rs11903757 (2q32.3). Of the three additional loci for which evidence of an association in Europeans has been previously described we failed to show an association between rs59336 (12q24.21) and CRC risk. However, for the other two SNPs, our analyses demonstrated new, formally significant associations with CRC. These are rs3217810 intronic in <em>CCND2</em> (12p13.32; OR = 1.19, <em>P</em> = 2.16 × 10<sup>−10</sup>) and rs10911251 near <em>LAMC1</em> (1q25.3; OR = 1.09, <em>P</em> = 1.75 × 10<sup>−8</sup>). Additionally, we found some evidence to support a relationship between, rs647161, rs2423297 and rs10774214 and CRC risk originally identified in East Asians in our European datasets. Our findings provide further insights into the genetic and biological basis of inherited genetic susceptibility to CRC.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p3" class="p p-first">Many colorectal cancers (CRC) develop in genetically susceptible individuals, most of whom are not carriers of germ-line mismatch repair or <em>APC</em> mutations (<a href="#DDU177C1" rid="DDU177C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093546">1</a>–<a href="#DDU177C3" rid="DDU177C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093560">3</a>). Genome-wide association studies (GWAS) conducted over recent years have demonstrated that an appreciable part of the heritable risk of CRC is attributable to common, low-risk variants. These GWAS have also provided insights into the biological basis of CRC development, highlighting the potential role of genes within the bone morphogenetic protein signalling pathway (<em>BMP2, BMP4, GREM1</em> and SMAD7) (<a href="#DDU177C4" rid="DDU177C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093595">4</a>,<a href="#DDU177C5" rid="DDU177C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093552">5</a>) and some candidate genes (<em>e.g. CDH1/CDH3</em>), as well as genes not previously implicated in CRC (<em>e.g. POLD3, TERC, CDKN1A</em> and <em>SHROOM2</em>) (<a href="#DDU177C6" rid="DDU177C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539783655">6</a>,<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606272">7</a>).</p><p id="__p4" class="p p-last">The statistical power of individual GWAS has been limited by the modest effect sizes of genetic variants, the need to establish stringent thresholds of statistical significance and economic constraints on the number of variants that can be followed up. Meta-analysis of existing GWAS data therefore offer the opportunity to discover additional CRC risk loci and provide further insights into disease aetiology. In this study, we conducted a discovery meta-analysis of five European GWAS datasets, followed by validation in three independent case–control series, enabling us to identify a novel susceptibility locus for CRC. We also tested a set of recently reported risk SNPs, some with statistically significant associations and others with more limited evidence of a relationship with CRC. This analysis refutes two of these associations and demonstrates two new genome-wide genetic associations with CRC risk.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p5" class="p p-first">The discovery phase comprised analysis of five non-overlapping GWAS case–control series of Northern European ancestry, which have been previously reported (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>): UK1, 890 familial colorectal tumour cases and 900 cancer-free controls from the COloRectal Gene Identification (CORGI) consortium (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606278">7</a>); Scotland1, 972 early-onset CRC cases (aged &lt;55 years at diagnosis) and 998 population controls(<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606273">7</a>); VQ58, 1794 UK cases with stage B/C CRC from the VICTOR and QUASAR2 trials, together with publicly available data from 2686 population controls from the UK 1958 Birth Cohort (<a href="#DDU177C8" rid="DDU177C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093587">8</a>); CCFR1, 1175 familial CRC cases and 999 controls from the Colon Cancer Family Registry (CCFR) (<a href="#DDU177C9" rid="DDU177C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093551">9</a>); and CCFR2, 795 CRC cases from CCFR and 2234 controls from the Cancer Genetic Markers of Susceptibility studies of breast and prostate cancer (<a href="#DDU177C10" rid="DDU177C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093581">10</a>,<a href="#DDU177C11" rid="DDU177C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093567">11</a>). Samples were genotyped using proprietary Illumina SNP arrays: UK1 on Hap550; Scotland1 on Hap300 + Hap240S; VQ58 on Hap300, Hap370, Hap660 or Hap1M; CGEMS on Hap300 + Hap240 or Hap550; and CCFR samples using Hap1M, Human1M-Duo or Omni-express arrays. Quality control of genotyping was assessed as previously described and all SNPs presented in this study passed the pre-determined thresholds (<a href="#DDU177C6" rid="DDU177C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_539783654">6</a>).</p><p id="__p6" class="p">The adequacy of the case–control matching and possibility of differential genotyping of cases and controls was assessed using <em>Q</em>–<em>Q</em> plots of test statistics. <em>λ</em><sub>GC</sub> values (<a href="#DDU177C12" rid="DDU177C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093556">12</a>) for the UK1, Scotland1, VQ58, CCFR1 and CCFR2 studies were 1.02, 1.01, 1.01, 1.02 and 1.03, respectively, thereby excluding significant differential genotyping or cryptic population substructure (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). Any ethnic outliers or individuals identified as related were excluded (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a>).</p><div id="s2a" class="sec"><h3 id="s2atitle">Discovery of a new CRC susceptibility SNP</h3><p id="__p7" class="p p-first">Using data from the above five GWAS, we derived for each directly genotyped SNP joint odds ratios (ORs) and confidence intervals (CIs) under a fixed-effects model, and the associated <em>P</em>-values. In this meta-analysis, associations for all 20 established CRC risk SNPs showed a direction of effect consistent with previously reported studies, with eight of those loci having a <em>P</em>-value of &lt;5.0 × 10<sup>−8</sup> (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). We also identified 31 SNPs that showed good evidence of an association (i.e. <em>P</em> &lt; 1.5 × 10<sup>−4</sup>) and mapped to distinct loci that had not previously been associated with CRC risk. This threshold for follow-up does not exclude the possibility that other SNPs represented genuine association signals, but was simply a pragmatic strategy for prioritizing replication.</p><p id="__p8">For these 31 SNPs we conducted a replication study using a custom array-based approach. We utilized three additional case–control series comprising a total of 14 037 cases and 15 937 controls: UK NSCCG (National Study of Colorectal Cancer Genetics) replication (UK3); Edinburgh replication (Scotland3); and Swedish replication (Sweden). Genotyping was successful for 29 of the 31 SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). In the combined analysis of discovery and replication phases, one SNP, rs1035209, showed an association with CRC which was genome-wide significant (<em>P</em> = 4.54 × 10<sup>−11</sup>; Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F1" rid-ob="ob-DDU177F1" co-legend-rid="lgnd_DDU177F1"><span>1</span></a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU177F1" co-legend-rid="lgnd_DDU177F1"><a href="/pmc/articles/PMC4133584/figure/DDU177F1/" target="figure" rid-figpopup="DDU177F1" rid-ob="ob-DDU177F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139901962306944" class="figure"><a href="/pmc/articles/PMC4133584/figure/DDU177F1/" target="figure" rid-figpopup="DDU177F1" rid-ob="ob-DDU177F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4133584_ddu17701.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu17701.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4133584/bin/ddu17701.jpg"></a></div><div id="largeobj_idm139901962306944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4133584/figure/DDU177F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU177F1"><div><a class="figpopup" href="/pmc/articles/PMC4133584/figure/DDU177F1/" target="figure" rid-figpopup="DDU177F1" rid-ob="ob-DDU177F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p9">Forest plot of the ORs for the association between CRC and rs1035209. Studies were weighted according to the inverse of the variance of the log of the OR calculated by unconditional logistic regression. Horizontal lines: 95% CI<em>.</em> Box: OR point estimate; its area is proportional to the weight of the study. Diamond (and broken line): overall summary estimate, with CI given by its width. Unbroken vertical line: null value (OR = 1.0).</p></div></div></div><p id="__p10" class="p p-last">rs1035209 localizes to chromosome 10q24.2 (101 345 366 bp; Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2" co-legend-rid="lgnd_DDU177F2"><span>2</span></a>A). To comprehensively analyse the 10q24.2 association, we imputed unobserved genotypes in the region in GWAS cases and controls using 1000genomes data (Phase 1 integrated version 3, March 2012). We did not find substantive evidence of stronger associations than that provided by rs1035209 (Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2" co-legend-rid="lgnd_DDU177F2"><span>2</span></a>A). The 320 kb region of linkage disequilibrium (LD) to which rs1035209 maps encompasses five transcripts, including <em>ABCC2/MRP2</em> (multidrug resistant protein 2). We examined for relationships between rs1035209 and gender, age at diagnosis, family history of CRC and clinico-pathological features (tumour site, stage or microsatellite instability) through case-only analysis, but no significant association was found (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU177F2" co-legend-rid="lgnd_DDU177F2"><a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139901973387392" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddu17702.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddu17702.jpg" src="/pmc/articles/PMC4133584/bin/ddu17702.jpg"></div></a><div id="largeobj_idm139901973387392" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2"></a><a target="object" href="/pmc/articles/PMC4133584/figure/DDU177F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU177F2"><div><a class="figpopup" href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p11">Regional plots of association results, recombination rates and chromatin state segmentation track for (A) 10q24.2, (B) 12p13.32 and (C) 1q25.3 susceptibility loci. Association results of both genotyped (triangles) and imputed (circles) SNPs in the GWAS samples and recombination rates for rates. −log<sub>10</sub>
<em>P-</em>values (<em>y</em> axis) of the SNPs are shown according to their chromosomal positions (<em>x</em> axis). The top genotyped SNP in each combined analysis is shown as a large triangle and is labelled by its rsID. Colour intensity of each symbol reflects the extent of LD with the top genotyped SNP; white (<em>r</em><sup>2</sup> = 0) through to dark red (<em>r</em><sup>2</sup> = 1.0) Genetic recombination rates, estimated using HapMap Utah residents of Western and Northern European ancestry (CEU) samples, are shown with a light blue line. Physical positions are based on NCBI Build 37 of the human genome. Also shown are the relative positions of genes and transcripts mapping to the region of association. Genes have been redrawn to show the relative positions; therefore, maps are not to physical scale. The lower panel shows the chromatin state segmentation track (ChromHMM).</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Evaluation of CRC SNPs reported in Asians</h3><p id="__p12" class="p p-first-last">Jia <em>et al.</em> (<a href="#DDU177C13" rid="DDU177C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093558">13</a>) have recently identified three SNPs; rs647161 (5q31.1), rs2423279 (20p12.3) and rs10774214 (12p13.32, near <em>CCND2</em>) that had genome-wide significant associations with CRC in East Asian populations. The same study evaluated these SNPs in a limited number of European CRC cases and controls, including the CCFR datasets, and found evidence of associations for all SNPs, albeit at relatively modest levels of significance (<em>P</em> = 0.040, <em>P</em> = 0.002 and <em>P</em> = 0.001 respectively). To test for association of these SNPs in our data we imputed all five GWAS datasets using the 1000 Genomes project as a reference panel and performed a meta-analysis. We found similar modest levels of significance (<em>P</em> = 0.02, <em>P</em> = 4.57 × 10<sup>−4</sup> and <em>P</em> = 0.02 respectively; <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>) to the original study. The nominal level of association at these loci may be attributed to environmental differences or differences in risk allele frequency, specifically of rs647161, between Asians and Europeans (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>).</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">Evaluation of SNP rs11903757 previously found to be associated with CRC in europeans</h3><p id="__p13" class="p p-first-last">Making use of imputed data, Peters <em>et al</em> (<a href="#DDU177C14" rid="DDU177C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093600">14</a>) have recently reported a genome-wide significant association between rs11903757 at 2q32.3 and CRC risk in a combined analysis of European and Asian case–control series (<em>P</em> = 3.7 × 10<sup>−8</sup>; <em>P</em> = 1.4 × 10<sup>−6</sup> in Europeans). Despite our overlapping datasets, with both studies including CCFR, we did not find any evidence in our samples to support a relationship between rs11903757 and CRC (<em>P</em> = 0.90). In a combined analysis of data from both our study and that of Peters <em>et al</em> (<a href="#DDU177C14" rid="DDU177C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093555">14</a>), this SNP association was not significant at the genome-wide threshold (<em>P</em> = 1.89 × 10<sup>−4</sup>
<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>).</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">Identification of two further CRC predisposition SNPs in combined analysis of our data and published data</h3><p id="__p14" class="p p-first-last">The study by Peters <em>et al.</em> (<a href="#DDU177C14" rid="DDU177C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093594">14</a>) also reported non-significant, but promising, associations (5.0 × 10<sup>−7</sup> &gt; <em>P</em> &gt; 5.0 × 10<sup>−8</sup>) at rs3217810 (12p13.32), rs59336 (12q24.21) and rs10911251 (1q25.3). Intriguingly, rs3217810 is intronic in the <em>CCND2</em> gene close to the Asian CRC SNP rs10774214, but these two SNPs are essentially uncorrelated (<em>r</em><sup>2</sup> = 0.002, <em>D</em>′ = 0.092). We examined the robustness of the three promising associations in a meta-analysis of our data and the published data. This analysis showed rs3217810 and rs10911251 to be CRC risk SNPs, with both associations now attaining genome-wide significance (<em>P</em> = 2.16 × 10<sup>−10</sup> and <em>P</em> = 1.75 × 10<sup>−8</sup> respectively; Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2" co-legend-rid="lgnd_DDU177F2"><span>2</span></a>B and C and Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F3" rid-ob="ob-DDU177F3" co-legend-rid="lgnd_DDU177F3"><span>3</span></a>). In contrast we did not find any evidence to support a relationship between rs59336 and CRC (<em>P</em> = 0.17; <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S5</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU177F3" co-legend-rid="lgnd_DDU177F3"><a href="/pmc/articles/PMC4133584/figure/DDU177F3/" target="figure" rid-figpopup="DDU177F3" rid-ob="ob-DDU177F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139902042294288" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddu17703.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddu17703.jpg" src="/pmc/articles/PMC4133584/bin/ddu17703.jpg"></div></a><div id="largeobj_idm139902042294288" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4133584/figure/DDU177F3/" target="figure" rid-figpopup="DDU177F3" rid-ob="ob-DDU177F3"></a><a target="object" href="/pmc/articles/PMC4133584/figure/DDU177F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU177F3"><div><a class="figpopup" href="/pmc/articles/PMC4133584/figure/DDU177F3/" target="figure" rid-figpopup="DDU177F3" rid-ob="ob-DDU177F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p15">Forest plot of the ORs for associations between CRC and (A) rs3217810 and (B) rs10911251. Studies were weighted according to the inverse of the variance of the log of the OR calculated by unconditional logistic regression. Horizontal lines: 95% CI. Box: OR point estimate; its area is proportional to the weight of the study. Diamond (and broken line): overall summary estimate, with CI given by its width. Unbroken vertical line: null value (OR = 1.0). Summary estimates are shown for studies from Peters <em>et al</em>., from the three GWAS not included in that study and a combined estimate.</p></div></div></div></div><div id="s2e" class="sec sec-last"><h3 id="s2etitle">eQTL analysis of the three new CRC SNPs</h3><p id="__p16" class="p p-first">To gain insight into the biological basis of the associations at rs1035209, rs3217810 and rs10911251, we analyzed publicly available mRNA expression data from fibroblasts, lymphoblastoid cell lines (LCLs), T cells, adipose tissue and skin cells (<a href="#DDU177C15" rid="DDU177C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093559">15</a>,<a href="#DDU177C16" rid="DDU177C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093582">16</a>). Additionally we interrogated The Cancer Genome Atlas (TCGA) RNA-seq expression and Affymetrix 6.0 SNP data (dbGaP accession number: phs000178.v7.p6) on 223 colonic and 75 rectal cancers using rs7075305 (<em>r</em><sup>2</sup> = 0.30, <em>D</em>′ = 0.77), rs3217805 (<em>r</em><sup>2</sup> = 0.19, <em>D</em>′ = 0.87) and rs3768617 (<em>r</em><sup>2</sup> = 0.90, <em>D</em>′ = 1.00) as proxies for rs1035209, rs3217810 and rs10911251, respectively. After adjustment for multiple testing, no significant associations were seen between SNP genotype and expression of genes mapping to any of the three risk loci (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S6 and S7</a>).</p><p id="__p17">Using HaploReg (<a href="#DDU177C17" rid="DDU177C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093596">17</a>) and RegulomeDB (<a href="#DDU177C18" rid="DDU177C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093584">18</a>), we examined whether any of the SNPs or their proxies (i.e. <em>r</em><sup>2</sup> &gt; 0.8 in 1000 Genomes CEU reference panel) lie at putative transcription factor binding/enhancer elements and derived GERP (Genomic Evolutionary Rate Profiling) scores to asses sequence conservation at these positions (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). rs3217810 is evolutionarily conserved (GERP score 2.97) with the motif being shown to have GATA6 binding in the CRC cell line CaCo2. Intriguingly, GATA6 expression is elevated in CRC and has been linked to invasiveness (<a href="#DDU177C19" rid="DDU177C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093588">19</a>,<a href="#DDU177C20" rid="DDU177C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093574">20</a>). While rs10911251 is not evolutionarily conserved, the correlated SNPs rs10911205 (<em>r</em><sup>2</sup> = 0.81) and rs10911211 (<em>r</em><sup>2</sup> = 0.83) are conserved (GREP scores 2.95 and 3.17, respectively), and are associated with transcription factor binding, albeit weakly. In addition, rs10911205 lies within an enhancer region predicted by ChromHMM (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>).</p><p id="__p18" class="p p-last">Finally, we conducted pathway analysis to determine whether any genes mapping to the three newly identified regions act in pathways already over-represented in GWAS regions. All genes within the LD block containing each tagSNP, or linked to the SNP through functional experiments, were uploaded to the NCI pathway interaction database. Pathways containing three or more genes are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a>. While this analysis identifies the BMP signalling pathway as expected, the analysis also reveals multiple pathways involving the LAMC1 and LAMC2 genes near rs10911251 and the regulation of nuclear beta catenin signalling pathway implicating <em>CCND2</em> to which rs3217810 maps.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p19" class="p p-first">Here, we have identified a novel CRC susceptibility SNP rs1035209 at 10q24.2. By performing a combined analyses with published data, we have also been able to show that two promising, but formally non-significant SNP associations, rs3217810 (12p13.32) and rs10911251 (1q25.3), are indeed associated with CRC risk in Europeans at genome-wide levels of significance. Additionally, we have provided evidence that a previously reported association between CRC and SNP rs11903757 does not hold in Europeans. Finally our data lend some support to the association of three other SNPs, originally identified in East Asians, with CRC in European populations.</p><p id="__p20">The relationship between genetic variation near <em>CCND2</em> (12p13.32) and risk of CRC was first reported in East Asians (<a href="#DDU177C13" rid="DDU177C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093563">13</a>), with low levels of support in Europeans. Hence, the observation that variation at this locus also impacts on risk in Europeans at genome-wide significance thresholds provides evidence for the generalizability of this association. Additional studies are required to decipher the number of independent signals of association and the functional basis of the 12p13.32 associations in different ethnic groups. We note that rs3217810 is located within an intron of <em>CCND2</em> (Fig.&nbsp;<a href="/pmc/articles/PMC4133584/figure/DDU177F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU177F2" rid-ob="ob-DDU177F2" co-legend-rid="lgnd_DDU177F2"><span>2</span></a>)<em>.</em> Cyclin D2 is a member of the D-type cyclin family which plays a critical role in cell cycle control through activation of cyclin-dependent kinases (CDK), primarily CDK4 and CDK6 (<a href="#DDU177C21" rid="DDU177C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093585">21</a>). Additionally CCND2 has been shown to represses the transcriptional activity of <em>SMAD3</em>. While less well studied than <em>CCND1</em>, over-expression of CCND2 typifies a subset of CRC and has been reported to be an independent predictor of survival (<a href="#DDU177C22" rid="DDU177C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093575">22</a>–<a href="#DDU177C24" rid="DDU177C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093577">24</a>).</p><p id="__p21">The 1q25.3 association implicates variation in the gene encoding the extracellular matrix protein laminin gamma 1 (<em>LAMC1</em>) which is involved in the maintenance of cell adhesion, migration and signalling (<a href="#DDU177C25" rid="DDU177C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093553">25</a>–<a href="#DDU177C27" rid="DDU177C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093550">27</a>). <em>LAMC1</em> is mutated in ∼8% of CRC and differential expression linked to development of metastatic CRC (<a href="http://www.cbioportal.org/public-portal/">http://www.cbioportal.org/public-portal/</a>). We have previously shown that variation in <em>LAMA5</em> is a determinant of CRC risk (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606274">7</a>) and our new observation implicates laminin genes more generally in the aetiology of CRC. Pathway analysis reveals integrin signalling pathways containing both LAMC1 and LAMA5 genes as over-represented in GWAS regions.</p><p id="__p22">The susceptibility locus at 10q24.2 marked by rs1035209 encompasses a number of genes with a possible role in the development of CRC including <em>ABCC2/MRP2</em> a multi-drug resistance gene influencing responsiveness to anticancer drugs (<a href="#DDU177C28" rid="DDU177C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093598">28</a>) which is mutated in ∼7% of CRC (<a href="http://www.cbioportal.org/public-portal/">http://www.cbioportal.org/public-portal/</a>).</p><p id="__p23">Since the marker SNPs at each of the risk loci are not strongly correlated with nsSNPs, it is likely that the functional variants at each site take the form of non-coding changes influencing gene expression. Accepting the caveat that our eQTL analysis of CRC was, for rs1035209 and rs3217810, dependent upon weak proxies we found no evidence for allele-specific <em>cis</em>-regulation of neighbouring genes. It is however likely that the potential impact of common alleles on gene expression will be modest and could occur at any time before diagnosis of CRC. Moreover, expression differences may only be relevant to a subpopulation of cells that provide ‘targets’ for tumour mutations.</p><p id="__p24">Our failure to find any evidence to support associations marked by rs11903757 and rs59336, or any proxy SNPs, was despite the CCFR GWAS datasets, which we included in our meta-analysis, previously contributing to the analysis by Peters <em>et al.</em> (<a href="#DDU177C14" rid="DDU177C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093571">14</a>). Both our study and that of Peters relied on imputation to recover rs11903757 and rs59336 genotypes. Hence, the imputation fidelity is of paramount importance. Through sequencing, we were able to establish that imputed genotypes were accurately recovered in our study. The study of Peters did not include this quality control step, moreover, imputation used as reference HapMap data derived from 30 trios, rather than the &gt;1000 individuals catalogued by the 1000 Genomes Project. Overall, the data suggest that the putative association with CRC at this locus may be spurious.</p><p id="__p25">The power of our study to detect the major common loci conferring risks of 1.2 or greater (such as the 8q24 variant rs6983267) was high. Hence, there are unlikely to be many additional CRC SNPs with similar effects for alleles with frequencies &gt;0.2 in populations of European ancestry. However, estimates of the missing heritability suggest that further variants of smaller effect sizes remain to be discovered. To discover these additional common CRC risk variants, there is a requirement for even larger scale GWAS meta-analyses in terms of both sample size and SNP coverage, as well as an increase in the number of SNPs taken forward to large-scale replication.</p><p id="__p26" class="p p-last">In conclusion, in this large study, we have identified a novel susceptibility locus associated with the risk of colorectal at 10q24.2 and performed combined analyses with published data to identify further associations at 12p13.32 (<em>CCND2</em>) and 1q25.3 (<em>LAMC1</em>). These findings bring to 23 the number of independent loci which influence CRC risk in Europeans.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Ethics statement</h3><p id="__p27" class="p p-first-last">Collection of blood samples and clinico-pathological information from subjects was undertaken with informed consent and ethical review board approval at all sites in accordance with the tenets of the Declaration of Helsinki. Specifically: for CORGI REC 06/Q1702/99; for SOCCS REC 11/SS/0109; for NSCCG REC 02/0/97.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Subjects and datasets</h3><p id="__p28" class="p p-first">UK1 (CORGI) (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606277">7</a>) comprised 940 cases with colorectal neoplasia (47% male) ascertained through the Colorectal Tumour Gene Identification (CoRGI) consortium. All had at least one first-degree relative affected by CRC and one or more of the following phenotypes: CRC at age 75 or less; any colorectal adenoma (CRAd) at age 45 or less; ≥3 colorectal adenomas at age 75 or less; or a large (&gt;1 cm diameter) or aggressive (villous and/or severely dysplastic) adenoma at age 75 or less. The 965 controls (45% males) were spouses or partners unaffected by cancer and without a personal family history (to second degree relative level) of colorectal neoplasia. Known dominant polyposis syndromes, HNPCC/Lynch syndrome or bi-allelic MUTYH mutation carriers were excluded. All cases and controls were of white UK ethnic origin.</p><p id="__p29">Scotland1 (COGS) (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606271">7</a>) included 1012 CRC cases (51% male; mean age at diagnosis 49.6 years, SD ± 6.1) and 1012 cancer-free population controls (51% male; mean age 51.0 years; SD ± 5.9). Cases were selected for early age at onset (age ≤55 years). Known dominant polyposis syndromes, HNPCC/Lynch syndrome or bi-allelic MUTYH mutation carriers were excluded. Control subjects were sampled from the Scottish population NHS registers, matched by age (±5 years), gender and area of residence within Scotland.</p><p id="__p30">VQ58 comprised 1800 CRC cases (1099 males, mean age of diagnosis 62.5 years; SD ± 10.9) from the VICTOR (<a href="#DDU177C29" rid="DDU177C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093583">29</a>) and QUASAR2 (<a href="http://www.octo-oxford.org.uk/alltrials/trials/q2.html">www.octo-oxford.org.uk/alltrials/trials/q2.html</a>) trials. There were 2690 population control genotypes (1391 males) from the Wellcome Trust Case–Control Consortium 2 (WTCCC2) 1958 birth cohort (<a href="#DDU177C8" rid="DDU177C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093548">8</a>) (also known as the National Child Development Study), which included all births in England, Wales and Scotland during a single week in 1958.</p><p id="__p31">The CCFR1 data set comprised 1290 familial CRC cases and 1055 controls from the Colon Cancer Family Registry (<a href="http://epi.grants.cancer.gov/CFR/about_colon.html)">http://epi.grants.cancer.gov/CFR/about_colon.html)</a> (<a href="#DDU177C9" rid="DDU177C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093592">9</a>). The cases were recently diagnosed CRC cases reported to population complete cancer registries in the USA (Puget Sound, Washington State) who were recruited by the Seattle Familial Colorectal Cancer Registry; in Canada (Ontario) who were recruited by the Ontario Familial Cancer Registry; and in Australia (Melbourne, Victoria) who were recruited by the Australasian Colorectal Cancer Family Study. Controls were population-based and for this analysis were restricted to those without a family history of colorectal cancer. The CCFR2 data set comprised a further 796 cases from the Colon Cancer Family Registry and 2236 controls from the Cancer Genetic Markers of Susceptibility studies of breast (<em>n</em> = 1142) and prostate (<em>n</em> = 1094) cancer (<a href="#DDU177C10" rid="DDU177C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093576">10</a>,<a href="#DDU177C11" rid="DDU177C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093579">11</a>).</p><p id="__p32">UK3 (NSCCG) (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606275">7</a>) comprised 7562 CRC cases (59% male; mean age at diagnosis 58 years, SD ± 8.4) and 7430 controls (39% male; mean age 58 years, SD ± 10.8) ascertained through NSCCG (National Study of Colorectal Cancer Genetics) post-2005 (<a href="#DDU177C30" rid="DDU177C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093569">30</a>).</p><p id="__p33">Scotland3 comprised 3969 CRC cases recruited as part of the SOCCS/COGS (<a href="#DDU177C7" rid="DDU177C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_598606276">7</a>) studies and 6791 population controls. Controls comprised unrelated cancer-free participants from Generation Scotland (<a href="http://www.generationscotland.org)">http://www.generationscotland.org)</a> (<a href="#DDU177C31" rid="DDU177C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093566">31</a>).</p><p id="__p34">The Swedish study comprised 2506 CRC patients were recruited within a Swedish national study conducted by the Swedish Low-Risk Colorectal Cancer Study Group. Samples were obtained during 2004–2009 from 14 different surgical clinics in central Sweden. All CRC patients during the study period were eligible for recruitment and were invited to participate. Only those too ill or too frail to consent were excluded. Controls (<em>n</em> = 1716) comprised blood donors from Stockholm and Uppsala. Fully informed consent was obtained in accordance with the Swedish law concerning ethical approval of research on human subjects (refs: 2002:489,2003:198,2010:1213-31/4).</p><p id="__p35" class="p p-last">In all cases CRC was defined according to the ninth revision of the International Classification of Diseases (ICD) by codes 153–154.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Sample preparation and genotyping</h3><p id="__p36" class="p p-first-last">DNA was extracted from samples using conventional methods and quantified using PicoGreen (Invitrogen). The VQ, UK1 and Scotland1 GWA cohorts were genotyped using Illumina Hap300, Hap240S, Hap370 or Hap550 arrays. 1958BC genotyping was performed as part of the WTCCC2 study on Hap1.2M-Duo Custom arrays. The CCFR samples were genotyped using Illumina Hap1M, Hap1M-Duo or Omni-express arrays. CGEMS samples were genotyped using Illumina Hap300 and Hap240 or Hap550 arrays. Genotyping quality and validity was very high in GWA datasets, as demonstrated by genotyping duplicate samples on orthogonal platforms which revealed 99.9% genotype concordance for all samples tested (data not shown). The replication samples were typed using custom 32 SNP genotyping plates Openarray (TaqMan<sup>®</sup> OpenArray<sup>®</sup>). The SNP content was designed based on SNP associations from analysis of GWA data, including imputed data. After technical assay failures, the final replication array comprised 29 SNPs (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>). Additional genotyping was conducted using competitive allele-specific PCR KASPar chemistry (LCG, Hertfordshire, UK). All primers, probes and conditions used are available on request. Genotypes for the Scottish control replication dataset were SNPs genotyped on the Illumina OmniExpressExome array (only SNPs that genotyped were included, imputed data were not used). Genotyping quality control was tested using duplicate DNA samples within studies and SNP assays, together with direct sequencing of subsets of samples to confirm genotyping accuracy. For all SNPs, &gt;99% concordant results were obtained.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">Quality control and sample exclusion</h3><p id="__p37" class="p p-first">We excluded SNPs from analysis if they failed one or more of the following thresholds: GenCall scores &lt;0.25; overall call rates &lt;95%; minor allele frequency (MAF) &lt; 0.01; departure from Hardy–Weinberg equilibrium (HWE) in controls at <em>P</em> &lt; 10<sup>−4</sup> or in cases at <em>P</em> &lt; 10<sup>−6</sup>; outlying in terms of signal intensity or <em>X</em>:<em>Y</em> ratio; discordance between duplicate samples; and, for SNPs with evidence of association, poor clustering on inspection of <em>X</em>:<em>Y</em> plots. We excluded samples from analysis if they failed one or more of the following thresholds: duplication or cryptic relatedness to estimated identity by descent (IBD) &gt;6.25%; overall successfully genotyped SNPs &lt; 95%; mismatch between predicted and reported gender; outliers in a plot of heterozygosity versus missingness; and evidence of non-white European ancestry by principal component analysis (PCA)-based analysis in comparison with HapMap samples (<a href="http://hapmap.ncbi.nlm.nih.gov">http://hapmap.ncbi.nlm.nih.gov</a>). Details of all sample exclusions are provided in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S4</a>.</p><p id="__p38">To identify individuals who might have non-northern European ancestry, we merged our case and control data from all sample sets with the 60 European (CEU), 60 Nigerian (YRI), 90 Japanese (JPT) and 90 Han Chinese (CHB) individuals from the International HapMap Project. For each pair of individuals, we calculated genome-wide identity-by-state distances based on markers shared between HapMap2 and our SNP panel, and used these as dissimilarity measures upon which to perform PCA. PCA was performed in R. The first two principal components for each individual were plotted and any individual not present in the main CEU cluster (i.e. &gt;5% of the PC distance from HapMap CEU cluster centroid) was excluded from subsequent analyses.</p><p id="__p39" class="p p-last">We have previously shown the adequacy of the case–control matching and possibility of differential genotyping of cases and controls using <em>Q</em>–<em>Q</em> plots of test statistics. The inflation factor <em>λ</em><sub>GC</sub> was calculated before and after imputation by dividing the mean of the lower 90% of the test statistics by the mean of the lower 90% of the expected values from a <em>χ</em><sup>2</sup> distribution with 1 d.f. Only SNPs with minor allele frequency &gt;0.05, imputation INFO &gt; 0.4, <em>P</em>-heterogeneity &gt; 0.01 and <em>P</em>-HWE &gt; 0.01 were considered. In addition, calculations were made using the median values of the observed and expected <em>χ</em><sup>2</sup> distribution but no significant differences were seen. Deviation of the genotype frequencies in the controls from those expected under HWE was assessed by <em>χ</em><sup>2</sup> test (1 d.f.), or Fisher's exact test where an expected cell count was &lt;5.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Statistical and bioinformatic analysis</h3><p id="__p40" class="p p-first">Main analyses were undertaken using R (v2.6), Stata v.11 (College Station, TX, US) and PLINK (v1.06) software (<a href="#DDU177C32" rid="DDU177C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093586">32</a>). The association between each SNP and risk of CRC was assessed by the Cochran–Armitage trend test. ORs and associated 95% CIs were calculated by unconditional logistic regression. Meta-analysis was conducted using standard methods (<a href="#DDU177C33" rid="DDU177C33" class=" bibr popnode">33</a>). GWAS meta-analysis data is available from EGA (accession number EGAS00001000759). Cochran's <em>Q</em> statistic to test for heterogeneity (<a href="#DDU177C33" rid="DDU177C33" class=" bibr popnode">33</a>) and the <em>I<sup>2</sup></em> statistic to quantify the proportion of the total variation due to heterogeneity were calculated (<a href="#DDU177C34" rid="DDU177C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093602">34</a>). <em>I<sup>2</sup></em> values ≥75% are considered characteristic of large heterogeneity (<a href="#DDU177C34" rid="DDU177C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093599">34</a>–<a href="#DDU177C36" rid="DDU177C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093590">36</a>). Associations by sex, age and clinic-pathological phenotypes were examined by logistic regression in case-only analyses.</p><p id="__p41">Prediction of the untyped SNPs was carried out using IMPUTEv2, based on 1000genomes Phase 1 integrated version 3 reference data. Imputed data were analyzed using SNPTESTv2.3.0 to account for uncertainties in SNP prediction and meta-analysis was performed using METAv1.4 with an information score threshold of 0.4. To filter poorly imputed SNPs, as previously recommended, we excluded variants having overall information scores from SNPTESTv2.3.0 of &lt;0.4. LD metrics were calculated from 1000genomes pilot release I data and viewed using SNAP. Where SNPs had not been catalogued, LD metrics were calculated using in house Perl scripts using the 1000genomes Phase 1 data. Regional association plots of LD metrics were then plotted using SNAP. LD blocks were defined on the basis of HapMap recombination rate and were viewed using the Haploview software (v4.2).</p><p id="__p42">Accuracy of imputation was assessed using sequence data of 200 CRC cases. The genotypes of the SNPs which passed QC in the UK1 dataset were extracted and used for imputation of each sample. The concordance between imputed SNPs and those obtained from sequencing was calculated for all individuals and for those individuals that are heterozygous or homozygous for the rare allele in the sequencing data. rs3217810, rs10911251, rs59336 and rs11903757 showed strong correlation between imputation and sequencing with <em>r</em><sup>2</sup> of 1.00, 0.99, 0.95 and 1.00 over all SNPs, respectively (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S10</a>).</p><p id="__p43">To explore epigenetic profiles of association signals, we used ChromHMM (<a href="#DDU177C37" rid="DDU177C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093578">37</a>). States were inferred from ENCODE Histone Modification data on the CRC cell line Hct116 (DNAse, H3K4me3, H3K4me1, H3K27ac, Pol2 and CTCF) binarized using a multivariate Hidden Markov Model.</p><p id="__p44" class="p p-last">We made use of HaploReg (<a href="#DDU177C17" rid="DDU177C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093580">17</a>) and RegulomeDB (<a href="#DDU177C18" rid="DDU177C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093557">18</a>) to examine whether any of the SNPs or their proxies (i.e. <em>r</em><sup>2</sup> &gt; 0.8 in 1000genomes CEU reference panel) annotate putative transcription factor binding/enhancer elements. We assessed sequence conservation using Genomic Evolutionary Rate Profiling (GERP). GERP scores (−12 to 6, with 6 being indicative of complete conservation) reflect the proportion of substitutions at that site rejected by selection compared with observed substitutions expected under a neutral evolutionary model, based on sequence alignment of 34 mammalian species (<a href="#DDU177C38" rid="DDU177C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093593">38</a>). We also analyzed expression data generated from (i) fibroblasts, lymphoblastoid cell lines (LCLs) and T cells derived from the umbilical cords of 75 Geneva GenCord individuals (<a href="#DDU177C15" rid="DDU177C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093554">15</a>); (ii) 166 adipose, 156 LCL and 160 skin samples derived from a subset of healthy female twins of the MuTHER resource (<a href="#DDU177C16" rid="DDU177C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093565">16</a>) using Sentrix Human-6 Expression BeadChips (Illumina) (<a href="#DDU177C39" rid="DDU177C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093564">39</a>,<a href="#DDU177C40" rid="DDU177C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093589">40</a>).</p></div><div id="s4f" class="sec"><h3 id="s4ftitle">Relationship between SNP genotype and mRNA expression</h3><p id="__p45" class="p p-first-last">To examine for a relationship between SNP genotype and mRNA expression we made use of Tumor Cancer Genome Atlas (TCGA) RNA-seq expression and Affymetrix 6.0 SNP data (dbGaP accession number: phs000178.v7.p6) on 223 colorectal adenocarcinoma (COAD) and 75 rectal adenocarcinoma tumour samples using a best proxy where SNPs were not represented directly. Association between normalized RNA counts per-gene and SNP genotype was quantified using the Kruskal–Wallis trend test.</p></div><div id="s4g" class="sec"><h3 id="s4gtitle">Pathway analysis</h3><p id="__p46" class="p p-first-last">To determine whether any genes mapping to the three newly identified regions act in pathways already over-represented in GWAS regions we utilized the NCI pathway interaction database (<a href="http://pid.nci.nih.gov/index.shtml">http://pid.nci.nih.gov/index.shtml</a>). All genes within the LD block containing each tagSNP, or linked to the SNP through functional experiments (MYC) were submitted as a Batch query using the NCI-Nature curated data source.</p></div><div id="s4h" class="sec sec-last"><h3 id="s4htitle">Assignment of microsatellite instability (MSI) in colorectal cancers</h3><p id="__p47" class="p p-first-last">Tumour MSI status in CRCs was determined using the mononucleotide microsatellite loci BAT25 and BAT26, which are highly sensitive MSI markers. Briefly, 10 μm sections were cut from formalin-fixed paraffin-embedded CRC tumours, lightly stained with toluidine blue and regions containing at least 60% tumour microdissected. Tumour DNA was extracted using the QIAamp DNA Mini kit (Qiagen, Crawley, UK) according to the manufacturer's instructions and genotyped for the BAT25 and BAT26 loci using <sup>32</sup>P-labelled oligonucleotide primers. Samples showing more than or equal to five novel alleles, when compared with normal DNA, at either or both markers were assigned as MSI-H (corresponding to MSI-high) (<a href="#DDU177C41" rid="DDU177C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_395093601">41</a>).</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">URLS</h2><p id="__p48" class="p p-first">The R suite can be found at <a href="http://www.r-project.org">http://www.r-project.org</a>.</p><p id="__p49">Detailed information on the tag SNP panel can be found at <a href="http://www.illumina.com">http://www.illumina.com</a>.</p><p id="__p50">dbSNP: <a href="http://www.ncbi.nlm.nih.gov/projects/SNP">http://www.ncbi.nlm.nih.gov/projects/SNP</a>.</p><p id="__p51">HapMap: <a href="http://www.hapmap.org">http://www.hapmap.org</a>.</p><p id="__p52">1000Genomes: <a href="http://www.1000genomes.org">http://www.1000genomes.org</a>.</p><p id="__p53">SNAP <a href="http://www.broadinstitute.org/mpg/snap">http://www.broadinstitute.org/mpg/snap</a>.</p><p id="__p54">IMPUTE: <a href="https://mathgen.stats.ox.ac.uk/impute/impute.html">https://mathgen.stats.ox.ac.uk/impute/impute.html</a>.</p><p id="__p55">SNPTEST: <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html">http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html</a>.</p><p id="__p56">Cancer Genome Atlas project: <a href="http://cancergenome.nih.gov">http://cancergenome.nih.gov</a>.</p><p id="__p57">The ENCODE Project: ENCyclopedia Of DNA Elements: <a href="http://www.genome.gov">http://www.genome.gov</a>.</p><p id="__p58">HaploReg: <a href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</a>.</p><p id="__p59" class="p p-last">RegulomeDB: <a href="http://regulome.stanford.edu/">http://regulome.stanford.edu/</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">AUTHORS’ CONTRIBUTIONS</h2><p id="__p60" class="p p-first-last">The study was designed and financial support was obtained by R.S.H., I.P.M.T. and M.G.D. The manuscript was drafted by R.S.H., N.W., I.P.M.T and M.G.D. All authors had access to data, analysis and had opportunity to contribute to drafting the manuscript. Statistical and bioinformatic analyses were conducted by N.W. and F.H., with contributions from S.D., Y.W. and B.K. ICR and local collaborators: coordination of sample preparation and genotyping was performed by P.B. Sample preparation and genotyping were performed by A.L. and N.W. Oxford and local collaborators: subject recruitment and sample acquisition were done by E.B., M.G., L.M. and members of the CORGI Consortium. Sample preparation and genotyping were performed by C.P. Colon Cancer Genetics Group, Edinburgh and local collaborators: subject recruitment and sample acquisition were performed by S.M.F., C.H., H.C., I.D. and M.G.D., as well as members of SOCCS and COGS recruitment teams. Sample preparation was coordinated by S.M.F.. Genotyping was performed and coordinated by S.M.F., C.H. and M.G.D. Data curation and analysis was conducted by L.Z., S.M.F. and A.T. Recruitment sample preps, wet lab expression analysis and genotyping was performed by C.H. and S.M.F. M.T. L.Z. performed the bioinformatic analyses.</p></div><div id="s7" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s7title">SUPPLEMENTARY MATERIAL</h2><!--/article/body/sec/--><p id="__p61" class="p p-first"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu177/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139901974481680"><a href="/pmc/articles/PMC4133584/bin/supp_23_17_4729__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>











</div></div><div id="idm139902043209472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139902043209472title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p62">Cancer Research UK provided principal funding for this study to R.S.H., I.P.M.T. and M.G.D. In addition, this work was supported by the European Union (FP7/207-2013) under grant 258236, FP7 collaborative project SYSCOL and COST Action BM1206.</p><p id="__p63">At the Institute of Cancer Research, the work was supported by a Programme Grant from Cancer Research UK (C1298/A8362—Bobby Moore Fund for Cancer Research UK). Additional support was provided by the National Cancer Research Network and the NHS via the Biological Research Centre of the National Institute for Health Research at the Royal Marsden Hospital NHS Trust. N.W. and B.K. were in receipt of PhD studentships from the ICR. B.K. additionally receives funding from the Sir John Fisher Foundation.</p><p id="__p64">In Edinburgh the work was supported by Programme Grant funding from Cancer Research UK (C348/A12076). We are most grateful to all participants of the SOCCS/COGS studies and Generation Scotland, without whom our research could not be conducted. We acknowledge the excellent technical support from CCGG technicians and the COGS/SOCCS operational team: study coordinators, recruitment nurses, data entry and collation. Generation Scotland is a collaboration between the University Medical Schools and NHS in Aberdeen, Dundee, Edinburgh and Glasgow. The Generation Scotland Family Health Study (<a href="http://www.generationscotland.org">http://www.generationscotland.org</a>) was funded by a grant from the Scottish Government Health Department, Chief Scientist Office, CZD/16/6. Genotyping was funded by the UK Medical Research Council. We are grateful to GPs and Scottish School of Primary Care for help with recruitment to GS:SFHS. We acknowledge the excellent work of the whole GS team: academic researchers, clinic staff, laboratory technicians, clerical workers, statisticians and research managers. We acknowledge the expert support on sample preparation and genotyping by the Genetics Core of the Edinburgh University Wellcome Trust Clinical Research Facility.</p><p id="__p65">In Oxford additional funding was provided by the Oxford Comprehensive Biomedical Research Centre (C.P. and I.P.M.T.) and the EU FP7 CHIBCHA grant (I.P.M.T.). Core infrastructure support to the Wellcome Trust Centre for Human Genetics, Oxford was provided by grant (090532/Z/09/Z). We are grateful to many colleagues within UK Clinical Genetics Departments (for CORGI) and to many collaborators who participated in the VICTOR and QUASAR2 trials. We also thank colleagues from the UK National Cancer Research Network (for NSCCG).</p><p id="__p66">The Swedish sample and data resource was funded by the Swedish Cancer Society, the Swedish Scientific Research Council and the Stockholm Cancer Foundation. We acknowledge the contribution to recruitment and data collection of the Swedish Low-Risk Colorectal Cancer Study Group.</p><p id="__p67">The work of the Colon Cancer Family Registry CFR was supported by the National Cancer Institute, National Institutes of Health under RFA #CA-95-011 and through cooperative agreements with members of the Colon CFR and Principal Investigators. Collaborating centres include the Australasian Colorectal Cancer Family Registry (U01 <a id="__tag_787947571" class="tag_hotlink" href="/nuccore/CA097735" ref="/nuccore/CA097735"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA097735","term_id":"34951042","term_text":"CA097735"}}</span>CA097735</a>), the USC Familial Colorectal Neoplasia Collaborative Group (U01 <a id="__tag_787947570" class="tag_hotlink" href="/nuccore/CA074799" ref="/nuccore/CA074799"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074799","term_id":"34927072","term_text":"CA074799"}}</span>CA074799</a>), Mayo Clinic Cooperative Familial Registry for Colon Cancer Studies (U01 <a id="__tag_787947569" class="tag_hotlink" href="/nuccore/CA074800" ref="/nuccore/CA074800"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074800","term_id":"34927073","term_text":"CA074800"}}</span>CA074800</a>), Ontario Registry for Studies of Familial Colorectal Cancer (U01 <a id="__tag_787947567" class="tag_hotlink" href="/nuccore/CA074783" ref="/nuccore/CA074783"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074783","term_id":"34927056","term_text":"CA074783"}}</span>CA074783</a>) and the Seattle Colorectal Cancer Family Registry (U01 <a id="__tag_787947568" class="tag_hotlink" href="/nuccore/CA074794" ref="/nuccore/CA074794"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA074794","term_id":"34927067","term_text":"CA074794"}}</span>CA074794</a>). The Colon CFR GWAS was supported by funding from the National Cancer Institute, National Institutes of Health (U01CA122839 to GC).</p><p id="__p68">This study made use of genotyping data from the 1958 Birth Cohort, kindly made available by the Wellcome Trust Case Control Consortium 2. A full list of the investigators who contributed to the generation of the data is available at <a href="http://www.wtccc.org.uk/">http://www.wtccc.org.uk/</a>. Finally, we would like to thank all individuals who participated in the study. The results published here are in whole or part based upon data generated by The Cancer Genome Atlas pilot project established by the NCI and NHGRI. Information about TCGA and the investigators and institutions who constitute the TCGA research network can be found at <a href="http://cancergenome.nih.gov/">http://cancergenome.nih.gov/</a>.</p><p id="__p69"><em>Conflict of Interest statement</em>. The authors declare no competing financial interests.</p></div></div><div id="idm139901966249280" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139901966249280title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDU177C1">1. <span class="element-citation">Lichtenstein P., Holm N.V., Verkasalo P.K., Iliadou A., Kaprio J., Koskenvuo M., Pukkala E., Skytthe A., Hemminki K. Environmental and heritable factors in the causation of cancer—analyses of cohorts of twins from Sweden, Denmark, and Finland. <span><span class="ref-journal">N. Engl. J. Med. </span>2000;<span class="ref-vol">343</span>:78–85.</span> [<a href="/pubmed/10891514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Environmental+and+heritable+factors+in+the+causation+of+cancer—analyses+of+cohorts+of+twins+from+Sweden,+Denmark,+and+Finland&amp;author=P.+Lichtenstein&amp;author=N.V.+Holm&amp;author=P.K.+Verkasalo&amp;author=A.+Iliadou&amp;author=J.+Kaprio&amp;volume=343&amp;publication_year=2000&amp;pages=78-85&amp;pmid=10891514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C2">2. <span class="element-citation">Aaltonen L., Johns L., Jarvinen H., Mecklin J.P., Houlston R. Explaining the familial colorectal cancer risk associated with mismatch repair (MMR)-deficient and MMR-stable tumors. <span><span class="ref-journal">Clin. Cancer Res. </span>2007;<span class="ref-vol">13</span>:356–361.</span> [<a href="/pubmed/17200375" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Explaining+the+familial+colorectal+cancer+risk+associated+with+mismatch+repair+(MMR)-deficient+and+MMR-stable+tumors&amp;author=L.+Aaltonen&amp;author=L.+Johns&amp;author=H.+Jarvinen&amp;author=J.P.+Mecklin&amp;author=R.+Houlston&amp;volume=13&amp;publication_year=2007&amp;pages=356-361&amp;pmid=17200375&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C3">3. <span class="element-citation">Lubbe S.J., Webb E.L., Chandler I.P., Houlston R.S. Implications of familial colorectal cancer risk profiles and microsatellite instability status. <span><span class="ref-journal">J. Clin. Oncol. </span>2009;<span class="ref-vol">27</span>:2238–2244.</span> [<a href="/pubmed/19307499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Implications+of+familial+colorectal+cancer+risk+profiles+and+microsatellite+instability+status&amp;author=S.J.+Lubbe&amp;author=E.L.+Webb&amp;author=I.P.+Chandler&amp;author=R.S.+Houlston&amp;volume=27&amp;publication_year=2009&amp;pages=2238-2244&amp;pmid=19307499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C4">4. <span class="element-citation">Tomlinson I.P., Carvajal-Carmona L.G., Dobbins S.E., Tenesa A., Jones A.M., Howarth K., Palles C., Broderick P., Jaeger E.E., Farrington S., et al.  Multiple common susceptibility variants near BMP pathway loci GREM1, BMP4, and BMP2 explain part of the missing heritability of colorectal cancer. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002105.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3107194/">PMC free article</a>]</span> [<a href="/pubmed/21655089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Multiple+common+susceptibility+variants+near+BMP+pathway+loci+GREM1,+BMP4,+and+BMP2+explain+part+of+the+missing+heritability+of+colorectal+cancer&amp;author=I.P.+Tomlinson&amp;author=L.G.+Carvajal-Carmona&amp;author=S.E.+Dobbins&amp;author=A.+Tenesa&amp;author=A.M.+Jones&amp;volume=7&amp;publication_year=2011&amp;pages=e1002105&amp;pmid=21655089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C5">5. <span class="element-citation">Broderick P., Carvajal-Carmona L., Pittman A.M., Webb E., Howarth K., Rowan A., Lubbe S., Spain S., Sullivan K., Fielding S., et al.  A genome-wide association study shows that common alleles of SMAD7 influence colorectal cancer risk. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1315–1317.</span> [<a href="/pubmed/17934461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+shows+that+common+alleles+of+SMAD7+influence+colorectal+cancer+risk&amp;author=P.+Broderick&amp;author=L.+Carvajal-Carmona&amp;author=A.M.+Pittman&amp;author=E.+Webb&amp;author=K.+Howarth&amp;volume=39&amp;publication_year=2007&amp;pages=1315-1317&amp;pmid=17934461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C6">6. <span class="element-citation">Dunlop M.G., Dobbins S.E., Farrington S.M., Jones A.M., Palles C., Whiffin N., Tenesa A., Spain S., Broderick P., Ooi L.Y., et al.  Common variation near CDKN1A, POLD3 and SHROOM2 influences colorectal cancer risk. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:770–776.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4747430/">PMC free article</a>]</span> [<a href="/pubmed/22634755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variation+near+CDKN1A,+POLD3+and+SHROOM2+influences+colorectal+cancer+risk&amp;author=M.G.+Dunlop&amp;author=S.E.+Dobbins&amp;author=S.M.+Farrington&amp;author=A.M.+Jones&amp;author=C.+Palles&amp;volume=44&amp;publication_year=2012&amp;pages=770-776&amp;pmid=22634755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C7">7. <span class="element-citation">Houlston R.S., Cheadle J., Dobbins S.E., Tenesa A., Jones A.M., Howarth K., Spain S.L., Broderick P., Domingo E., Farrington S., et al.  Meta-analysis of three genome-wide association studies identifies susceptibility loci for colorectal cancer at 1q41, 3q26.2, 12q13.13 and 20q13.33. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:973–977.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5098601/">PMC free article</a>]</span> [<a href="/pubmed/20972440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+of+three+genome-wide+association+studies+identifies+susceptibility+loci+for+colorectal+cancer+at+1q41,+3q26.2,+12q13.13+and+20q13.33&amp;author=R.S.+Houlston&amp;author=J.+Cheadle&amp;author=S.E.+Dobbins&amp;author=A.+Tenesa&amp;author=A.M.+Jones&amp;volume=42&amp;publication_year=2010&amp;pages=973-977&amp;pmid=20972440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C8">8. <span class="element-citation">Power C., Elliott J. Cohort profile: 1958 British birth cohort (National Child Development Study) <span><span class="ref-journal">Int. J. Epidemiol. </span>2006;<span class="ref-vol">35</span>:34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Cohort+profile:+1958+British+birth+cohort+(National+Child+Development+Study)&amp;author=C.+Power&amp;author=J.+Elliott&amp;volume=35&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C9">9. <span class="element-citation">Newcomb P.A., Baron J., Cotterchio M., Gallinger S., Grove J., Haile R., Hall D., Hopper J.L., Jass J., Le Marchand L., et al.  Colon Cancer Family Registry: an international resource for studies of the genetic epidemiology of colon cancer. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2007;<span class="ref-vol">16</span>:2331–2343.</span> [<a href="/pubmed/17982118" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Colon+Cancer+Family+Registry:+an+international+resource+for+studies+of+the+genetic+epidemiology+of+colon+cancer&amp;author=P.A.+Newcomb&amp;author=J.+Baron&amp;author=M.+Cotterchio&amp;author=S.+Gallinger&amp;author=J.+Grove&amp;volume=16&amp;publication_year=2007&amp;pages=2331-2343&amp;pmid=17982118&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C10">10. <span class="element-citation">Hunter D.J., Kraft P., Jacobs K.B., Cox D.G., Yeager M., Hankinson S.E., Wacholder S., Wang Z., Welch R., Hutchinson A., et al.  A genome-wide association study identifies alleles in FGFR2 associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=D.J.+Hunter&amp;author=P.+Kraft&amp;author=K.B.+Jacobs&amp;author=D.G.+Cox&amp;author=M.+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C11">11. <span class="element-citation">Yeager M., Orr N., Hayes R.B., Jacobs K.B., Kraft P., Wacholder S., Minichiello M.J., Fearnhead P., Yu K., Chatterjee N., et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M.+Yeager&amp;author=N.+Orr&amp;author=R.B.+Hayes&amp;author=K.B.+Jacobs&amp;author=P.+Kraft&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C12">12. <span class="element-citation">Clayton D.G., Walker N.M., Smyth D.J., Pask R., Cooper J.D., Maier L.M., Smink L.J., Lam A.C., Ovington N.R., Stevens H.E., et al.  Population structure, differential bias and genomic control in a large-scale, case–control association study. <span><span class="ref-journal">Nat. Genet. </span>2005;<span class="ref-vol">37</span>:1243–1246.</span> [<a href="/pubmed/16228001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Population+structure,+differential+bias+and+genomic+control+in+a+large-scale,+case–control+association+study&amp;author=D.G.+Clayton&amp;author=N.M.+Walker&amp;author=D.J.+Smyth&amp;author=R.+Pask&amp;author=J.D.+Cooper&amp;volume=37&amp;publication_year=2005&amp;pages=1243-1246&amp;pmid=16228001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C13">13. <span class="element-citation">Jia W.H., Zhang B., Matsuo K., Shin A., Xiang Y.B., Jee S.H., Kim D.H., Ren Z., Cai Q., Long J., et al.  Genome-wide association analyses in East Asians identify new susceptibility loci for colorectal cancer. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:191–196.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679924/">PMC free article</a>]</span> [<a href="/pubmed/23263487" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analyses+in+East+Asians+identify+new+susceptibility+loci+for+colorectal+cancer&amp;author=W.H.+Jia&amp;author=B.+Zhang&amp;author=K.+Matsuo&amp;author=A.+Shin&amp;author=Y.B.+Xiang&amp;volume=45&amp;publication_year=2013&amp;pages=191-196&amp;pmid=23263487&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C14">14. <span class="element-citation">Peters U., Jiao S., Schumacher F.R., Hutter C.M., Aragaki A.K., Baron J.A., Berndt S.I., Bezieau S., Brenner H., Butterbach K., et al.  Identification of genetic susceptibility loci for colorectal tumors in a genome-wide meta-analysis. <span><span class="ref-journal">Gastroenterology. </span>2013;<span class="ref-vol">144</span>:799–807.</span> . e724. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3636812/">PMC free article</a>]</span> [<a href="/pubmed/23266556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Identification+of+genetic+susceptibility+loci+for+colorectal+tumors+in+a+genome-wide+meta-analysis&amp;author=U.+Peters&amp;author=S.+Jiao&amp;author=F.R.+Schumacher&amp;author=C.M.+Hutter&amp;author=A.K.+Aragaki&amp;volume=144&amp;publication_year=2013&amp;pages=799-807&amp;pmid=23266556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C15">15. <span class="element-citation">Dimas A.S., Deutsch S., Stranger B.E., Montgomery S.B., Borel C., Attar-Cohen H., Ingle C., Beazley C., Gutierrez Arcelus M., Sekowska M., et al.  Common regulatory variation impacts gene expression in a cell type-dependent manner. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">325</span>:1246–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867218/">PMC free article</a>]</span> [<a href="/pubmed/19644074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+regulatory+variation+impacts+gene+expression+in+a+cell+type-dependent+manner&amp;author=A.S.+Dimas&amp;author=S.+Deutsch&amp;author=B.E.+Stranger&amp;author=S.B.+Montgomery&amp;author=C.+Borel&amp;volume=325&amp;publication_year=2009&amp;pages=1246-1250&amp;pmid=19644074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C16">16. <span class="element-citation">Nica A.C., Parts L., Glass D., Nisbet J., Barrett A., Sekowska M., Travers M., Potter S., Grundberg E., Small K., et al.  The architecture of gene regulatory variation across multiple human tissues: the MuTHER study. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002003.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3033383/">PMC free article</a>]</span> [<a href="/pubmed/21304890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=The+architecture+of+gene+regulatory+variation+across+multiple+human+tissues:+the+MuTHER+study&amp;author=A.C.+Nica&amp;author=L.+Parts&amp;author=D.+Glass&amp;author=J.+Nisbet&amp;author=A.+Barrett&amp;volume=7&amp;publication_year=2011&amp;pages=e1002003&amp;pmid=21304890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C17">17. <span class="element-citation">Ward L.D., Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:D930–D934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=40&amp;publication_year=2012&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C18">18. <span class="element-citation">Boyle A.P., Hong E.L., Hariharan M., Cheng Y., Schaub M.A., Kasowski M., Karczewski K.J., Park J., Hitz B.C., Weng S., et al.  Annotation of functional variation in personal genomes using RegulomeDB. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:1790–1797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=A.P.+Boyle&amp;author=E.L.+Hong&amp;author=M.+Hariharan&amp;author=Y.+Cheng&amp;author=M.A.+Schaub&amp;volume=22&amp;publication_year=2012&amp;pages=1790-1797&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C19">19. <span class="element-citation">Belaguli N.S., Aftab M., Rigi M., Zhang M., Albo D., Berger D.H. GATA6 promotes colon cancer cell invasion by regulating urokinase plasminogen activator gene expression. <span><span class="ref-journal">Neoplasia. </span>2010;<span class="ref-vol">12</span>:856–865.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2978909/">PMC free article</a>]</span> [<a href="/pubmed/21076612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neoplasia&amp;title=GATA6+promotes+colon+cancer+cell+invasion+by+regulating+urokinase+plasminogen+activator+gene+expression&amp;author=N.S.+Belaguli&amp;author=M.+Aftab&amp;author=M.+Rigi&amp;author=M.+Zhang&amp;author=D.+Albo&amp;volume=12&amp;publication_year=2010&amp;pages=856-865&amp;pmid=21076612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C20">20. <span class="element-citation">Shureiqi I., Zuo X., Broaddus R., Wu Y., Guan B., Morris J.S., Lippman S.M. The transcription factor GATA-6 is overexpressed in vivo and contributes to silencing 15-LOX-1 <em>in vitro</em> in human colon cancer. <span><span class="ref-journal">FASEB J. </span>2007;<span class="ref-vol">21</span>:743–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1847772/">PMC free article</a>]</span> [<a href="/pubmed/17167069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J.&amp;title=The+transcription+factor+GATA-6+is+overexpressed+in+vivo+and+contributes+to+silencing+15-LOX-1+in+vitro+in+human+colon+cancer&amp;author=I.+Shureiqi&amp;author=X.+Zuo&amp;author=R.+Broaddus&amp;author=Y.+Wu&amp;author=B.+Guan&amp;volume=21&amp;publication_year=2007&amp;pages=743-753&amp;pmid=17167069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C21">21. <span class="element-citation">Matsushime H., Quelle D.E., Shurtleff S.A., Shibuya M., Sherr C.J., Kato J.Y. D-type cyclin-dependent kinase activity in mammalian cells. <span><span class="ref-journal">Mol. Cell. Biol. </span>1994;<span class="ref-vol">14</span>:2066–2076.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC358567/">PMC free article</a>]</span> [<a href="/pubmed/8114738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Cell.+Biol.&amp;title=D-type+cyclin-dependent+kinase+activity+in+mammalian+cells&amp;author=H.+Matsushime&amp;author=D.E.+Quelle&amp;author=S.A.+Shurtleff&amp;author=M.+Shibuya&amp;author=C.J.+Sherr&amp;volume=14&amp;publication_year=1994&amp;pages=2066-2076&amp;pmid=8114738&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C22">22. <span class="element-citation">Musgrove E.A., Caldon C.E., Barraclough J., Stone A., Sutherland R.L. Cyclin D as a therapeutic target in cancer. <span><span class="ref-journal">Nat. Rev. Cancer. </span>2011;<span class="ref-vol">11</span>:558–572.</span> [<a href="/pubmed/21734724" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cancer&amp;title=Cyclin+D+as+a+therapeutic+target+in+cancer&amp;author=E.A.+Musgrove&amp;author=C.E.+Caldon&amp;author=J.+Barraclough&amp;author=A.+Stone&amp;author=R.L.+Sutherland&amp;volume=11&amp;publication_year=2011&amp;pages=558-572&amp;pmid=21734724&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C23">23. <span class="element-citation">Mermelshtein A., Gerson A., Walfisch S., Delgado B., Shechter-Maor G., Delgado J., Fich A., Gheber L. Expression of D-type cyclins in colon cancer and in cell lines from colon carcinomas. <span><span class="ref-journal">Br. J. Cancer. </span>2005;<span class="ref-vol">93</span>:338–345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2361572/">PMC free article</a>]</span> [<a href="/pubmed/16012517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Expression+of+D-type+cyclins+in+colon+cancer+and+in+cell+lines+from+colon+carcinomas&amp;author=A.+Mermelshtein&amp;author=A.+Gerson&amp;author=S.+Walfisch&amp;author=B.+Delgado&amp;author=G.+Shechter-Maor&amp;volume=93&amp;publication_year=2005&amp;pages=338-345&amp;pmid=16012517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C24">24. <span class="element-citation">Sarkar R., Hunter I.A., Rajaganeshan R., Perry S.L., Guillou P., Jayne D.G. Expression of cyclin D2 is an independent predictor of the development of hepatic metastasis in colorectal cancer. <span><span class="ref-journal">Colorectal Dis. </span>2010;<span class="ref-vol">12</span>:316–323.</span> [<a href="/pubmed/19508551" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Colorectal+Dis.&amp;title=Expression+of+cyclin+D2+is+an+independent+predictor+of+the+development+of+hepatic+metastasis+in+colorectal+cancer&amp;author=R.+Sarkar&amp;author=I.A.+Hunter&amp;author=R.+Rajaganeshan&amp;author=S.L.+Perry&amp;author=P.+Guillou&amp;volume=12&amp;publication_year=2010&amp;pages=316-323&amp;pmid=19508551&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C25">25. <span class="element-citation">Turck N., Gross I., Gendry P., Stutzmann J., Freund J.N., Kedinger M., Simon-Assmann P., Launay J.F. Laminin isoforms: biological roles and effects on the intracellular distribution of nuclear proteins in intestinal epithelial cells. <span><span class="ref-journal">Exp. Cell Res. </span>2005;<span class="ref-vol">303</span>:494–503.</span> [<a href="/pubmed/15652360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res.&amp;title=Laminin+isoforms:+biological+roles+and+effects+on+the+intracellular+distribution+of+nuclear+proteins+in+intestinal+epithelial+cells&amp;author=N.+Turck&amp;author=I.+Gross&amp;author=P.+Gendry&amp;author=J.+Stutzmann&amp;author=J.N.+Freund&amp;volume=303&amp;publication_year=2005&amp;pages=494-503&amp;pmid=15652360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C26">26. <span class="element-citation">Pouliot N., Saunders N.A., Kaur P. Laminin 10/11: an alternative adhesive ligand for epidermal keratinocytes with a functional role in promoting proliferation and migration. <span><span class="ref-journal">Exp. Dermatol. </span>2002;<span class="ref-vol">11</span>:387–397.</span> [<a href="/pubmed/12366691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Dermatol.&amp;title=Laminin+10/11:+an+alternative+adhesive+ligand+for+epidermal+keratinocytes+with+a+functional+role+in+promoting+proliferation+and+migration&amp;author=N.+Pouliot&amp;author=N.A.+Saunders&amp;author=P.+Kaur&amp;volume=11&amp;publication_year=2002&amp;pages=387-397&amp;pmid=12366691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C27">27. <span class="element-citation">Patarroyo M., Tryggvason K., Virtanen I. Laminin isoforms in tumor invasion, angiogenesis and metastasis. <span><span class="ref-journal">Semin. Cancer Biol. </span>2002;<span class="ref-vol">12</span>:197–207.</span> [<a href="/pubmed/12083850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin.+Cancer+Biol.&amp;title=Laminin+isoforms+in+tumor+invasion,+angiogenesis+and+metastasis&amp;author=M.+Patarroyo&amp;author=K.+Tryggvason&amp;author=I.+Virtanen&amp;volume=12&amp;publication_year=2002&amp;pages=197-207&amp;pmid=12083850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C28">28. <span class="element-citation">Taniguchi K., Wada M., Kohno K., Nakamura T., Kawabe T., Kawakami M., Kagotani K., Okumura K., Akiyama S., Kuwano M. A human canalicular multispecific organic anion transporter (cMOAT) gene is overexpressed in cisplatin-resistant human cancer cell lines with decreased drug accumulation. <span><span class="ref-journal">Cancer Res. </span>1996;<span class="ref-vol">56</span>:4124–4129.</span> [<a href="/pubmed/8797578" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=A+human+canalicular+multispecific+organic+anion+transporter+(cMOAT)+gene+is+overexpressed+in+cisplatin-resistant+human+cancer+cell+lines+with+decreased+drug+accumulation&amp;author=K.+Taniguchi&amp;author=M.+Wada&amp;author=K.+Kohno&amp;author=T.+Nakamura&amp;author=T.+Kawabe&amp;volume=56&amp;publication_year=1996&amp;pages=4124-4129&amp;pmid=8797578&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C29">29. <span class="element-citation">Midgley R.S., McConkey C.C., Johnstone E.C., Dunn J.A., Smith J.L., Grumett S.A., Julier P., Iveson C., Yanagisawa Y., Warren B., et al.  Phase III randomized trial assessing rofecoxib in the adjuvant setting of colorectal cancer: final results of the VICTOR trial. <span><span class="ref-journal">J. Clin. Oncol. </span>2010;<span class="ref-vol">28</span>:4575–4580.</span> [<a href="/pubmed/20837956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Phase+III+randomized+trial+assessing+rofecoxib+in+the+adjuvant+setting+of+colorectal+cancer:+final+results+of+the+VICTOR+trial&amp;author=R.S.+Midgley&amp;author=C.C.+McConkey&amp;author=E.C.+Johnstone&amp;author=J.A.+Dunn&amp;author=J.L.+Smith&amp;volume=28&amp;publication_year=2010&amp;pages=4575-4580&amp;pmid=20837956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C30">30. <span class="element-citation">Penegar S., Wood W., Lubbe S., Chandler I., Broderick P., Papaemmanuil E., Sellick G., Gray R., Peto J., Houlston R. National study of colorectal cancer genetics. <span><span class="ref-journal">Br. J. Cancer. </span>2007;<span class="ref-vol">97</span>:1305–1309.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2360468/">PMC free article</a>]</span> [<a href="/pubmed/17895893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=National+study+of+colorectal+cancer+genetics&amp;author=S.+Penegar&amp;author=W.+Wood&amp;author=S.+Lubbe&amp;author=I.+Chandler&amp;author=P.+Broderick&amp;volume=97&amp;publication_year=2007&amp;pages=1305-1309&amp;pmid=17895893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C31">31. <span class="element-citation">Smith B.H., Campbell A., Linksted P., Fitzpatrick B., Jackson C., Kerr S.M., Deary I.J., Macintyre D.J., Campbell H., McGilchrist M., et al.  Cohort Profile: Generation Scotland: Scottish Family Health Study (GS:SFHS). The study, its participants and their potential for genetic research on health and illness. <span><span class="ref-journal">Int. J. Epidemiol. </span>2013;<span class="ref-vol">42</span>:689–700.</span> [<a href="/pubmed/22786799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Cohort+Profile:+Generation+Scotland:+Scottish+Family+Health+Study+(GS:SFHS).+The+study,+its+participants+and+their+potential+for+genetic+research+on+health+and+illness&amp;author=B.H.+Smith&amp;author=A.+Campbell&amp;author=P.+Linksted&amp;author=B.+Fitzpatrick&amp;author=C.+Jackson&amp;volume=42&amp;publication_year=2013&amp;pages=689-700&amp;pmid=22786799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C32">32. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C33">33. <span class="element-citation">Petitti DB.  <span class="ref-journal">Meta-Analysis Decision Analysis and Cost-Effectiveness Analysis.</span> New York: Oxford University Press; 1994.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Meta-Analysis+Decision+Analysis+and+Cost-Effectiveness+Analysis&amp;author=DB.+Petitti&amp;publication_year=1994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C34">34. <span class="element-citation">Higgins J.P., Thompson S.G. Quantifying heterogeneity in a meta-analysis. <span><span class="ref-journal">Stat. Med. </span>2002;<span class="ref-vol">21</span>:1539–1558.</span> [<a href="/pubmed/12111919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med.&amp;title=Quantifying+heterogeneity+in+a+meta-analysis&amp;author=J.P.+Higgins&amp;author=S.G.+Thompson&amp;volume=21&amp;publication_year=2002&amp;pages=1539-1558&amp;pmid=12111919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C35">35. <span class="element-citation">Ioannidis J.P., Ntzani E.E., Trikalinos T.A. ‘Racial’ differences in genetic effects for complex diseases. <span><span class="ref-journal">Nat. Genet. </span>2004;<span class="ref-vol">36</span>:1312–1318.</span> [<a href="/pubmed/15543147" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=‘Racial’+differences+in+genetic+effects+for+complex+diseases&amp;author=J.P.+Ioannidis&amp;author=E.E.+Ntzani&amp;author=T.A.+Trikalinos&amp;volume=36&amp;publication_year=2004&amp;pages=1312-1318&amp;pmid=15543147&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C36">36. <span class="element-citation">Gabriel S.B., Schaffner S.F., Nguyen H., Moore J.M., Roy J., Blumenstiel B., Higgins J., DeFelice M., Lochner A., Faggart M., et al.  The structure of haplotype blocks in the human genome. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:2225–2229.</span> [<a href="/pubmed/12029063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+structure+of+haplotype+blocks+in+the+human+genome&amp;author=S.B.+Gabriel&amp;author=S.F.+Schaffner&amp;author=H.+Nguyen&amp;author=J.M.+Moore&amp;author=J.+Roy&amp;volume=296&amp;publication_year=2002&amp;pages=2225-2229&amp;pmid=12029063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C37">37. <span class="element-citation">Ernst J., Kellis M. Discovery and characterization of chromatin states for systematic annotation of the human genome. <span><span class="ref-journal">Nat. Biotechnol. </span>2010;<span class="ref-vol">28</span>:817–825.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2919626/">PMC free article</a>]</span> [<a href="/pubmed/20657582" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Discovery+and+characterization+of+chromatin+states+for+systematic+annotation+of+the+human+genome&amp;author=J.+Ernst&amp;author=M.+Kellis&amp;volume=28&amp;publication_year=2010&amp;pages=817-825&amp;pmid=20657582&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C38">38. <span class="element-citation">Cooper G.M., Stone E.A., Asimenos G., Green E.D., Batzoglou S., Sidow A. Distribution and intensity of constraint in mammalian genomic sequence. <span><span class="ref-journal">Genome Res. </span>2005;<span class="ref-vol">15</span>:901–913.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1172034/">PMC free article</a>]</span> [<a href="/pubmed/15965027" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Distribution+and+intensity+of+constraint+in+mammalian+genomic+sequence&amp;author=G.M.+Cooper&amp;author=E.A.+Stone&amp;author=G.+Asimenos&amp;author=E.D.+Green&amp;author=S.+Batzoglou&amp;volume=15&amp;publication_year=2005&amp;pages=901-913&amp;pmid=15965027&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C39">39. <span class="element-citation">Stranger B.E., Forrest M.S., Clark A.G., Minichiello M.J., Deutsch S., Lyle R., Hunt S., Kahl B., Antonarakis S.E., Tavare S., et al.  Genome-wide associations of gene expression variation in humans. <span><span class="ref-journal">PLoS Genet. </span>2005;<span class="ref-vol">1</span>:e78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1315281/">PMC free article</a>]</span> [<a href="/pubmed/16362079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+associations+of+gene+expression+variation+in+humans&amp;author=B.E.+Stranger&amp;author=M.S.+Forrest&amp;author=A.G.+Clark&amp;author=M.J.+Minichiello&amp;author=S.+Deutsch&amp;volume=1&amp;publication_year=2005&amp;pages=e78&amp;pmid=16362079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C40">40. <span class="element-citation">Stranger B.E., Forrest M.S., Dunning M., Ingle C.E., Beazley C., Thorne N., Redon R., Bird C.P., de Grassi A., Lee C., et al.  Relative impact of nucleotide and copy number variation on gene expression phenotypes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:848–853.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665772/">PMC free article</a>]</span> [<a href="/pubmed/17289997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Relative+impact+of+nucleotide+and+copy+number+variation+on+gene+expression+phenotypes&amp;author=B.E.+Stranger&amp;author=M.S.+Forrest&amp;author=M.+Dunning&amp;author=C.E.+Ingle&amp;author=C.+Beazley&amp;volume=315&amp;publication_year=2007&amp;pages=848-853&amp;pmid=17289997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C41">41. <span class="element-citation">Boland C.R., Thibodeau S.N., Hamilton S.R., Sidransky D., Eshleman J.R., Burt R.W., Meltzer S.J., Rodriguez-Bigas M.A., Fodde R., Ranzani G.N., et al.  A National Cancer Institute Workshop on Microsatellite Instability for cancer detection and familial predisposition: development of international criteria for the determination of microsatellite instability in colorectal cancer. <span><span class="ref-journal">Cancer Res. </span>1998;<span class="ref-vol">58</span>:5248–5257.</span> [<a href="/pubmed/9823339" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=A+National+Cancer+Institute+Workshop+on+Microsatellite+Instability+for+cancer+detection+and+familial+predisposition:+development+of+international+criteria+for+the+determination+of+microsatellite+instability+in+colorectal+cancer&amp;author=C.R.+Boland&amp;author=S.N.+Thibodeau&amp;author=S.R.+Hamilton&amp;author=D.+Sidransky&amp;author=J.R.+Eshleman&amp;volume=58&amp;publication_year=1998&amp;pages=5248-5257&amp;pmid=9823339&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU177C42">42. <span class="element-citation">Tomlinson I.P., Dunlop M., Campbell H., Zanke B., Gallinger S., Hudson T., Koessler T., Pharoah P.D., Niittymaki I., Tuupanen S., et al.  COGENT (COlorectal cancer GENeTics): an international consortium to study the role of polymorphic variation on the risk of colorectal cancer. <span><span class="ref-journal">Br. J. Cancer. </span>2010;<span class="ref-vol">102</span>:447–454.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2816642/">PMC free article</a>]</span> [<a href="/pubmed/19920828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=COGENT+(COlorectal+cancer+GENeTics):+an+international+consortium+to+study+the+role+of+polymorphic+variation+on+the+risk+of+colorectal+cancer&amp;author=I.P.+Tomlinson&amp;author=M.+Dunlop&amp;author=H.+Campbell&amp;author=B.+Zanke&amp;author=S.+Gallinger&amp;volume=102&amp;publication_year=2010&amp;pages=447-454&amp;pmid=19920828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4133584&amp;issue-id=241183&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139901971920480title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">URLS</a></li><li><a href="#s6title">AUTHORS’ CONTRIBUTIONS</a></li><li><a href="#s7title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#idm139902043209472title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139901966249280title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4133584/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4133584/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4133584/pdf/ddu177.pdf">PDF (460K)</a></li> | <li><a href="#" data-citationid="PMC4133584" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4133584%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4133584%2F&amp;text=Identification%20of%20susceptibility%20loci%20for%20colorectal%20cancer%20in%20a%20genome-wide%20meta-analysis" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4133584%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24737748" data-aiid="4133584" data-aid="4133584" data-iid="241183" data-domainid="740" data-domain="hmg" data-accid="PMC4133584" data-md5="c24125aa37c7dc0f4f0403203d5784ba"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:04:21-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8DAFF1DD569FD10000000005E2039D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4133584%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4133584/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>