<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4570286/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Analysis of Body Proportion Classifies Height-Associated Variants by Mechanism of Action and Implicates Genes Important for Skeletal Development</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4570286/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Analysis of Body Proportion Classifies Height-Associated Variants by Mechanism of Action and Implicates Genes Important for Skeletal Development"><meta name="citation_authors" content="Yingleong Chan, Rany&nbsp;M. Salem, Yu-Han&nbsp;H. Hsu, George McMahon, Tune&nbsp;H. Pers, Sailaja Vedantam, Tonu Esko, Michael&nbsp;H. Guo, Elaine&nbsp;T. Lim, GIANT Consortium, Lude Franke, George&nbsp;Davey Smith, David&nbsp;P. Strachan, Joel&nbsp;N. Hirschhorn"><meta name="citation_date" content="7 May 2015"><meta name="citation_issue" content="5"><meta name="citation_volume" content="96"><meta name="citation_firstpage" content="695"><meta name="citation_doi" content="10.1016/j.ajhg.2015.02.018"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4570286/?report=abstract"><meta name="citation_pmid" content="25865494"><meta name="DC.Title" content="Genome-wide Analysis of Body Proportion Classifies Height-Associated Variants by Mechanism of Action and Implicates Genes Important for Skeletal Development"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Yingleong Chan"><meta name="DC.Contributor" content="Rany&nbsp;M. Salem"><meta name="DC.Contributor" content="Yu-Han&nbsp;H. Hsu"><meta name="DC.Contributor" content="George McMahon"><meta name="DC.Contributor" content="Tune&nbsp;H. Pers"><meta name="DC.Contributor" content="Sailaja Vedantam"><meta name="DC.Contributor" content="Tonu Esko"><meta name="DC.Contributor" content="Michael&nbsp;H. Guo"><meta name="DC.Contributor" content="Elaine&nbsp;T. Lim"><meta name="DC.Contributor" content="GIANT Consortium"><meta name="DC.Contributor" content="Lude Franke"><meta name="DC.Contributor" content="George&nbsp;Davey Smith"><meta name="DC.Contributor" content="David&nbsp;P. Strachan"><meta name="DC.Contributor" content="Joel&nbsp;N. Hirschhorn"><meta name="DC.Date" content="2015 May 7"><meta name="DC.Identifier" content="10.1016/j.ajhg.2015.02.018"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Analysis of Body Proportion Classifies Height-Associated Variants by Mechanism of Action and Implicates Genes Important for Skeletal Development"><meta property="og:type" content="article"><meta property="og:description" content="Human height is a composite measurement, reflecting the sum of leg, spine, and head lengths. Many common variants influence total height, but the effects of these or other variants on the components of height (body proportion) remain largely unknown. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4570286/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4570286/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4570286/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6880DD578921000000000952058E.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4570286%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/257845/">v.96(5); 2015 May 7</a></li><li class="accid">PMC4570286</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2015 May 7; 96(5): 695–708. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2015.02.018" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2015.02.018</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4570286</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25865494">25865494</a></div></div></div></div><h1 class="content-title">Genome-wide Analysis of Body Proportion Classifies Height-Associated Variants by Mechanism of Action and Implicates Genes Important for Skeletal Development</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chan%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711132961424" co-class="co-affbox">Yingleong Chan</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>∗</sup> <a href="/pubmed/?term=Salem%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711134043984" co-class="co-affbox">Rany&nbsp;M. Salem</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711131595776" co-class="co-affbox">Yu-Han&nbsp;H. Hsu</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=McMahon%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711127466544" co-class="co-affbox">George McMahon</a>,<sup>5</sup> <a href="/pubmed/?term=Pers%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711127464560" co-class="co-affbox">Tune&nbsp;H. Pers</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Vedantam%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711126324976" co-class="co-affbox">Sailaja Vedantam</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711130999104" co-class="co-affbox">Tonu Esko</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>6</sup> <a href="/pubmed/?term=Guo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711127009456" co-class="co-affbox">Michael&nbsp;H. Guo</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Lim%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711203075344" co-class="co-affbox">Elaine&nbsp;T. Lim</a>,<sup>1,</sup><sup>2,</sup><sup>3</sup> GIANT Consortium, <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711127342048" co-class="co-affbox">Lude Franke</a>,<sup>7</sup> <a href="/pubmed/?term=Smith%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711131887728" co-class="co-affbox">George&nbsp;Davey Smith</a>,<sup>5,</sup><sup>8</sup> <a href="/pubmed/?term=Strachan%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711133001200" co-class="co-affbox">David&nbsp;P. Strachan</a>,<sup>9</sup> and  <a href="/pubmed/?term=Hirschhorn%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494" class="affpopup" co-rid="_co_idm140711132999216" co-class="co-affbox">Joel&nbsp;N. Hirschhorn</a><sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>∗∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140711132961424"><h3 class="no_margin">Yingleong Chan</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><p><sup>4</sup>Wyss Institute for Biologically Inspired Engineering, Harvard Medical School, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Chan%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Yingleong Chan</a></div></div><div id="_co_idm140711134043984"><h3 class="no_margin">Rany&nbsp;M. Salem</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Salem%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Rany&nbsp;M. Salem</a></div></div><div id="_co_idm140711131595776"><h3 class="no_margin">Yu-Han&nbsp;H. Hsu</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20YH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Yu-Han&nbsp;H. Hsu</a></div></div><div id="_co_idm140711127466544"><h3 class="no_margin">George McMahon</h3><p><sup>5</sup>School of Social and Community Medicine, University of Bristol, Oakfield House, Oakfield Grove, Bristol BS8 2BN, UK</p><div>Find articles by <a href="/pubmed/?term=McMahon%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">George McMahon</a></div></div><div id="_co_idm140711127464560"><h3 class="no_margin">Tune&nbsp;H. Pers</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Pers%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Tune&nbsp;H. Pers</a></div></div><div id="_co_idm140711126324976"><h3 class="no_margin">Sailaja Vedantam</h3><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Vedantam%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Sailaja Vedantam</a></div></div><div id="_co_idm140711130999104"><h3 class="no_margin">Tonu Esko</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><p><sup>6</sup>Estonian Genome Center, University of Tartu, Tartu 51010, Estonia</p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Tonu Esko</a></div></div><div id="_co_idm140711127009456"><h3 class="no_margin">Michael&nbsp;H. Guo</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Guo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Michael&nbsp;H. Guo</a></div></div><div id="_co_idm140711203075344"><h3 class="no_margin">Elaine&nbsp;T. Lim</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Lim%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Elaine&nbsp;T. Lim</a></div></div><div id="_co_idm140711127342048"><h3 class="no_margin">Lude Franke</h3><p><sup>7</sup>Department of Genetics, University of Groningen, University Medical Centre Groningen, Groningen 9700 AB, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Lude Franke</a></div></div><div id="_co_idm140711131887728"><h3 class="no_margin">George&nbsp;Davey Smith</h3><p><sup>5</sup>School of Social and Community Medicine, University of Bristol, Oakfield House, Oakfield Grove, Bristol BS8 2BN, UK</p><p><sup>8</sup>MRC Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2PR, UK</p><div>Find articles by <a href="/pubmed/?term=Smith%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">George&nbsp;Davey Smith</a></div></div><div id="_co_idm140711133001200"><h3 class="no_margin">David&nbsp;P. Strachan</h3><p><sup>9</sup>Population Health Research Institute, St George’s, University of London, London SW17 0RE, UK</p><div>Find articles by <a href="/pubmed/?term=Strachan%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">David&nbsp;P. Strachan</a></div></div><div id="_co_idm140711132999216"><h3 class="no_margin">Joel&nbsp;N. Hirschhorn</h3><p><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p><p><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</p><p><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</p><div>Find articles by <a href="/pubmed/?term=Hirschhorn%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25865494">Joel&nbsp;N. Hirschhorn</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140711127482656_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140711127482656_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140711127482656_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140711127482656_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Broad Institute of the Massachusetts Institute of Technology and Harvard University, Cambridge, MA 02142, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, Boston, MA 02115, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Wyss Institute for Biologically Inspired Engineering, Harvard Medical School, Boston, MA 02115, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>School of Social and Community Medicine, University of Bristol, Oakfield House, Oakfield Grove, Bristol BS8 2BN, UK</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Estonian Genome Center, University of Tartu, Tartu 51010, Estonia</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Genetics, University of Groningen, University Medical Centre Groningen, Groningen 9700 AB, the Netherlands</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>MRC Integrative Epidemiology Unit, University of Bristol, Bristol BS8 2PR, UK</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Population Health Research Institute, St George’s, University of London, London SW17 0RE, UK</div><div><span class="contrib-email" id="idm140711132961424">Yingleong Chan: <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@legir" class="oemail">gro.etutitsnidaorb@legir</a>; </span> <span class="contrib-email" id="idm140711132999216">Joel&nbsp;N. Hirschhorn: <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@hleoj" class="oemail">gro.etutitsnidaorb@hleoj</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@legir" class="oemail">gro.etutitsnidaorb@legir</a></div><div id="cor2"><sup>∗∗</sup>Corresponding author <a href="mailto:dev@null" data-email="gro.etutitsnidaorb@hleoj" class="oemail">gro.etutitsnidaorb@hleoj</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140711127482656_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Jan 8; Accepted 2015 Feb 27.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140711127482656_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2015 The American Society of Human Genetics. Published by Elsevier Ltd. All right reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4570286/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. GIANT Consortium Information, Cohort-Specific Acknowledgments, and Figures S1–S4</div><div><a href="/pmc/articles/PMC4570286/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (670K)</span></div><div class="small guid">GUID:&nbsp;052C0951-3B28-4C39-ABC9-1F541DCDC98F</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Data S1. Tables S1–S19</div><div><a href="/pmc/articles/PMC4570286/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">mmc2.xlsx</a><span style="color:gray"> (2.8M)</span></div><div class="small guid">GUID:&nbsp;32021BF6-01C8-4941-82A6-2FF0601978A4</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S3. Article plus Supplemental Data</div><div><a href="/pmc/articles/PMC4570286/bin/mmc3.pdf" data-ga-action="click_feat_suppl">mmc3.pdf</a><span style="color:gray"> (3.2M)</span></div><div class="small guid">GUID:&nbsp;BE41D35A-9426-44D1-AF73-01C1D58F5D5D</div></div></dd></dl></div><div id="idm140711133935616" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711133935616title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Human height is a composite measurement, reflecting the sum of leg, spine, and head lengths. Many common variants influence total height, but the effects of these or other variants on the components of height (body proportion) remain largely unknown. We studied sitting height ratio (SHR), the ratio of sitting height to total height, to identify such effects in 3,545 African Americans and 21,590 individuals of European ancestry. We found that SHR is heritable: 26% and 39% of the total variance of SHR can be explained by common variants in European and African Americans, respectively, and global European admixture is negatively correlated with SHR in African Americans (<em>r</em><sup>2</sup> ≈ 0.03). Six regions reached genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>) for association with SHR and overlapped biological candidate genes, including TBX2 and IGFBP3. We found that 130 of 670 height-associated variants are nominally associated (p &lt; 0.05) with SHR, more than expected by chance (p = 5&nbsp;× 10<sup>−40</sup>). At these 130 loci, the height-increasing alleles are associated with either a decrease (71 loci) or increase (59 loci) in SHR, suggesting that different height loci disproportionally affect either leg length or spine/head length. Pathway analyses via DEPICT revealed that height loci affecting SHR, and especially those affecting leg length, show enrichment of different biological pathways (e.g., bone/cartilage/growth plate pathways) than do loci with no effect on SHR (e.g., embryonic development). These results highlight the value of using a pair of related but orthogonal phenotypes, in this case SHR with height, as a prism to dissect the biology underlying genetic associations in polygenic traits and diseases.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p2" class="p p-first">Human height has been used extensively as a model quantitative trait because it is highly heritable and polygenic, influenced by variants across many different loci.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> Recently, we and others identified 697 height-associated variants in a genome-wide association study (GWAS) of&nbsp;more than 300,000 individuals that together implicated many genes and pathways important for skeletal growth.<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> However, human height is a composite phenotype, reflecting the sum of head, trunk, and leg lengths. Although the genetics of height has been extensively studied, little is known about the genetics of the relationship between these individual components with each other (body proportion). We hypothesize that studying the genetics of body proportion could provide complementary insights to those obtained by studying overall height.</p><p id="__p3">Sitting height, the distance from the top of the head to the seated surface while a person is sitting upright, is a clinical measurement and sometimes measured in population-based studies.<sup><a href="#bib3" rid="bib3" class=" bibr popnode">3</a></sup> As such, it is an available measure that&nbsp;can provide insights into body proportion. Specifically, sitting height ratio (SHR), the ratio of sitting height to total height, can be used as a measure of body proportion: higher SHR corresponds to a higher spine/head to leg ratio, i.e., shorter leg length. Previous studies characterizing the SHR of individuals from childhood to adulthood show that SHR decreases rapidly from birth to teenage years (from 0.65 to 0.51 between ages 1 and 12 years) and increases slightly during puberty (from 0.51 to 0.52 between ages 12 and 18 years).<sup><a href="#bib3" rid="bib3 bib4" class=" bibr popnode">3,4</a></sup> Combined consideration of height and SHR can identify effects on height that act disproportionately on one body segment. For example, individuals with achondroplasia (MIM <a href="/omim/100800" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">100800</a>) have short stature due to shortened limbs, reflected in an average SHR of 0.66 (for comparison, the 95% confidence interval of normal adult SHR of European-ancestry individuals is 0.51 to 0.55).<sup><a href="#bib5" rid="bib5 bib6" class=" bibr popnode">5,6</a></sup> On the other hand, individuals with spondyloepiphyseal (MIM <a href="/omim/608361" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608361</a>) and spondylometaepiphyseal (MIM <a href="/omim/271665" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">271665</a>) dysplasias can have short stature but normal or decreased SHR values.<sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup> SHR can also be indicative of future health outcomes. Previous studies indicate that SHR can be predictive of several traits and diseases including blood pressure, cholesterol levels, body mass index, intimal-medial thickness, type 2 diabetes, and insulin resistance.<sup><a href="#bib8" rid="bib8 bib9 bib10" class=" bibr popnode">8–10</a></sup></p><p id="__p4">Although little is known about the heritability of SHR, this ratio is known to differ significantly across different ancestries. Adult individuals of African ancestry have lower SHR (SHR ∼ 0.51) than individuals of European ancestry (SHR ∼ 0.52) and individuals of Asian ancestry (SHR ∼ 0.53).<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> The basis of these significant differences in SHR across populations is unclear.</p><p id="__p5" class="p p-last">This lack of knowledge about the genetics of SHR and the potential to gain new insights into the genetics of skeletal growth prompted us to investigate the heritability and the genetic architecture of this trait. We studied a total of 3,545 African Americans as well as 21,590 individuals of European ancestry that had both whole genome genotype data as well as the phenotypes needed to calculate SHR. We investigated the heritability and genetic architecture of SHR by performing several analyses including GCTA-REML<sup><a href="#bib12" rid="bib12" class=" bibr popnode">12</a></sup> to estimate heritability and admixture analysis and GWAS to identify common variants associated with SHR. Furthermore, given that we and others have recently identified many height-associated loci from GWASs,<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> we used SHR like a prism to separate these loci into those that have an association with SHR and those that do not. The loci that have an association with SHR can be further separated into those that disproportionately affect the head/spine and those that disproportionately affect the long bones by testing whether the height-increasing allele increases or decreases SHR, respectively. We tested for enrichment of biological pathways for the various groups of loci and found that the loci associated with SHR, especially those that disproportionately affect the long bones, are significantly more enriched for genes in well-known pathways relevant to skeletal growth. On the other hand, the loci that are least associated with SHR tend to be more enriched for pathways involved in growth during embryogenesis. In general, our results highlight the capability of using an orthogonal phenotype like SHR to separate genetic loci known to be associated with height into their biological relevant components.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Material and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Subjects</h3><p id="__p6" class="p p-first">Individuals were recruited from various cohorts that have recording of sitting height, height, gender, age when measured, and body mass index (BMI). If there exists duplicated or closely related samples (PI_HAT &gt; 0.05), one was randomly filtered out.</p><div id="sec2.1.1" class="sec"><p></p><h4 id="sec2.1.1title" class="inline">Atherosclerosis Risk in Communities </h4><p id="__p7" class="p p-first-last">The participants of Atherosclerosis Risk in Communities (ARIC) study<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> were recruited from four communities beginning in 1987: Forsyth County, NC; Jackson, MS; the northwest suburbs of Minneapolis, MN; and Washington County, MD (dbGaP phs000280.v3.p1). Whole genome genotyping performed with the Affymetrix 6.0 platform and genotype calling was performed with Birdseed calling algorithm.<sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup> 2,354 African Americans and 7,257 European Americans were used for the analyses.</p></div><div id="sec2.1.2" class="sec"><p></p><h4 id="sec2.1.2title" class="inline">Cardiovascular Health Study </h4><p id="__p8" class="p p-first-last">The participants of Cardiovascular Health Study (CHS)<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> were recruited from Forsyth County, NC; Sacramento County, CA;&nbsp;Washington County, MD; and Pittsburgh, PA (dbGaP phs000287.v4.p1). Whole genome genotyping was performed with the Illumina HumanCNV370v1 platform and genotype calling was performed with Illumina GenomeStudio software. 476 African Americans and 2,926 European Americans were used for the analyses.</p></div><div id="sec2.1.3" class="sec"><p></p><h4 id="sec2.1.3title" class="inline">Coronary Artery Risk Development in Young Adults </h4><p id="__p9" class="p p-first-last">The participants of Coronary Artery Risk Development in Young Adults (CARDIA) study<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> were recruited from four communities in the United States: Birmingham, AL; Chicago, IL; Minneapolis, MN; and Oakland, CA (dbGaP phs000285.v3.p2). Whole genome genotyping performed with the Affymetrix 6.0 platform and genotype calling was performed with Birdseed calling algorithm.<sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup> 715 African Americans and 1,047 European Americans were used for the analyses.</p></div><div id="sec2.1.4" class="sec"><p></p><h4 id="sec2.1.4title" class="inline">Framingham Heart Study </h4><p id="__p10" class="p p-first">The participants of Framingham Heart Study (FHS)<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> were recruited from town of Framingham, MA, starting from 1948 (dbGaP phs000007.v25.p9). Whole genome genotyping was performed by the Affymetrix 500K platform and genotype calling was performed with Birdseed calling algorithm.<sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup> 713 European Americans were used for the analyses.</p><p id="__p11" class="p p-last">The data for ARIC, CHS, CARDIA, and FHS cohorts were obtained from dbGaP (see Acknowledgments).</p></div><div id="sec2.1.5" class="sec"><p></p><h4 id="sec2.1.5title" class="inline">1958 British Birth cohort </h4><p id="__p12" class="p p-first-last">The participants of 1958 British Birth Cohort (B58C)<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> were born in England, Scotland, and Wales during a single week of 1958. Whole genome genotyping was performed with the Illumina HumanHap 550K/610K platform and genotype calling was performed with Illumina GenomeStudio software. Genome-wide typing and imputations were available for 6,296 cohort members with measurements of standing and sitting height at age 44–45 years.</p></div><div id="sec2.1.6" class="sec sec-last"><p></p><h4 id="sec2.1.6title" class="inline">Avon Longitudinal Study of Parents and Children </h4><p id="__p13" class="p p-first-last">We used only the adult mothers from Avon Longitudinal Study of Parents and Children (ALSPAC) study<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup> for the analyses. Whole genome genotyping was performed with Illumina human660W-quad array platform and genotype calling was performed with Illumina GenomeStudio software. A total of 3,351 British adult females were used for analyses. A more complete description of ALSPAC can be found in the ALSPAC section below.</p></div></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Ethical Approval</h3><p id="__p14" class="p p-first-last">Ethical approval for the use of cohorts from dbGaP was obtained from the Institutional Review Board (IRB) of both Boston Children’s Hospital and the Broad Institute of Harvard and MIT. Ethical approval for the use of the 1958 British Birth Cohort was obtained from the South East Multicentre Research Ethics Committee (SE-MREC). Ethical approval for the use of ALSPAC was obtained from the ALSPAC Ethics and Law Committee and the Local Research Ethics Committees.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Quality Control</h3><p id="__p15" class="p p-first-last">We filtered out samples that have ambiguous or incorrect gender, SNPs that have &gt;5% missing rate, samples that have &gt;2% missing SNPs, SNPs that have minor allele frequencies &lt; 1%, and samples that have extreme heterozygosity (±4 SD). The SNP annotations for chromosome and base-pair positions were set to the coordinates of hg19 (GRCh37) via the liftOver tool.<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> Pairwise IBD/IBS were calculated and individuals that have excessive matching with other individuals were removed. A filter of PI_HAT &gt; 0.125 was used for ALSPAC. For all other cohorts, the threshold was set at PI_HAT &gt; 0.05. Principal components analysis was performed with SMARTPCA<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> and each study was projected onto HapMap v.3<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> space. Samples that did not cluster with the expected HapMap population were removed. SNPs that have excessive plate effects (p &lt; 1&nbsp;× 10<sup>−7</sup>) were dropped. For samples that are of European ancestry, SNPs that had excessive deviation from Hardy-Weinberg equilibrium (p &lt; 1&nbsp;× 10<sup>−7</sup>) were also dropped. Individuals who had no records of sitting height, height, age, or BMI or had very extreme sitting height ratios (SHR <em>Z</em> score &gt; 4 SD or SHR <em>Z</em> score &lt; −4 SD) were also removed. The QC protocol was performed with PLINK<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup> (v.1.07) and custom R and PERL scripts.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Genotype Phasing</h3><p id="__p16" class="p p-first-last">For ARIC, CHS, CARDIA, FHS, and ALSPAC, the genotypes were phased with SHAPEIT2 (v.2.644)<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> and imputed with IMPUTE2 (v.2.3).<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> For B58C, the genotypes were phased with MACH<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> and imputed with Minimac.<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup></p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Genotype Imputation</h3><p id="__p17" class="p p-first-last">The imputation for all cohorts was performed with a 1000 Genomes version 1 phase 3 reference panel<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> containing 379 Europeans, 246 Africans and African Americans, 286 Asians, and 181 Latin Americans. The imputation panel consists of approximately 22 million variants (SNPs and indels). For the X chromosome, only the non-pseudo autosomal region was imputed. The phasing and imputation were done separately for males and females for the X chromosome.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">ALSPAC</h3><div id="sec2.6.1" class="sec sec-first"><p></p><h4 id="sec2.6.1title" class="inline">Description of Study Numbers </h4><p id="__p18" class="p p-first-last">ALSPAC recruited 14,541 pregnant women resident in Avon, UK with expected dates of delivery 1st April 1991 to 31st December 1992. 14,541 is the initial number of pregnancies for which the mother enrolled in the ALSPAC study and had either returned at least one questionnaire or attended a “Children in Focus” clinic by 19/07/99. Of these initial pregnancies, there were a total of 14,676 fetuses, resulting in 14,062 live births and 13,988 children who were alive at 1 year of age. We used only the enrolled adult mothers that had data on sitting height, height, and body mass index. After filtering away individuals that were not genotyped or were removed after going through the quality-control pipeline, there was a final tally of 3,351 individuals.</p></div><div id="sec2.6.2" class="sec"><p></p><h4 id="sec2.6.2title" class="inline">Quality Control and Preparation </h4><p id="__p19" class="p p-first-last">Centre National de Génotypage (CNG) carried out DNA genotyping on the Illumina human660W-quad array and genotypes were called with Illumina GenomeStudio. PLINK (v.1.07) was used to carry out quality-control measures on an initial set of 10,015 subjects and 557,124 directly genotyped SNPs. SNPs were removed if they displayed more than 5% missingness or a Hardy-Weinberg equilibrium p value of less than 10<sup>−6</sup>. Additionally, SNPs with a minor allele frequency of less than 1% were removed. Samples were excluded if they displayed more than 5% missingness or had indeterminate X chromosome heterozygosity or extreme autosomal heterozygosity. Samples showing evidence of population stratification were identified by multidimensional scaling of genome-wide identity by state pairwise distances with the four HapMap populations as a reference, and then excluded. Cryptic relatedness was assessed with a PI_HAT of more than 0.125, which is expected to correspond to roughly 12.5% alleles shared IBD or a relatedness at the first-cousin level. Related subjects that passed all other quality-control thresholds were retained during subsequent phasing and imputation. 9,048 subjects and 526,688 SNPs passed these quality-control filters. We combined 477,482 SNP genotypes of these subjects with genotype data of a sample of 9,115 children, removed SNPs with a missingness above 1% due to poor quality (n&nbsp;= 11,396), and removed a further 321 subjects due to potential ID mismatches. This resulted in a dataset of 17,842 subjects containing 6,305 duos and 465,740 SNPs (112 were removed during liftover and 234 were out of HWE after combination). We estimated haplotypes with ShapeIT (v.2.r644), which utilizes relatedness during phasing. We imputed SNPs against the 1000 Genomes phase I version 3 integrated set via IMPUTE2 (v.2.3). Analysis was carried out in SNPtest (v.2.5) including the ten principal components of population ancestry from EIGENSTRAT.</p></div><div id="sec2.6.3" class="sec sec-last"><p></p><h4 id="sec2.6.3title" class="inline">Data Dictionary </h4><p id="__p20" class="p p-first-last">The ALSPAC study website contains details of all the data that are available through a fully searchable data dictionary (see <a href="#app3" rid="app3" class=" sec">Web Resources</a>).</p></div></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Overall European Ancestry in African-American Individuals</h3><p id="__p21" class="p p-first-last">Overall European admixture (ARIC, CARDIA, and CHS) was calculated by SMARTPCA from the CEU and YRI samples of HapMap v.3.<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> CEU individuals were used as proxies of European ancestry and YRI individuals were used as proxies of African ancestry. The principal components were calculated using only the CEU and YRI individuals while projecting them onto the ARIC, CARDIA, and CHS African Americans. The first principal component (PC1) was used to calculate the degree of overall European admixture for each individual. The percentage overall European admixture was calculated by (Individual<sub>PC1</sub> – YRI<sub>PC1</sub>)/(CEU<sub>PC1</sub> – YRI<sub>PC1</sub>) × 100%, where Individual<sub>PC1</sub> is PC1 for each individual and CEU<sub>PC1</sub> and YRI<sub>PC1</sub> are the mean PC1 values for the CEU and YRI individuals, respectively.</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">SHR Phenotype for Comparison between European and African Americans</h3><p id="__p22" class="p p-first-last">Both European-American and African-American individuals from ARIC, CARDIA, and CHS were pooled together and the SHR was calculated by taking the sitting height divided by overall height. SHR residuals were obtained by performing linear regression of SHR against overall height, age, gender, BMI, and cohort as covariates. The SHR <em>Z</em> scores were then obtained by taking the resulting residuals divided by the estimated standard deviation of the residuals. This was performed with custom R scripts.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">SHR Phenotype for GWAS and GCTA-REML Analysis</h3><p id="__p23" class="p p-first-last">SHR was calculated by taking the sitting height divided by overall height. The SHR residuals were obtained by performing linear regression of SHR against overall height, age, gender, and BMI as covariates separately for each cohort. For the ARIC, CARDIA, and CHS cohorts, this was performed separately for the African Americans and European Americans. For the African Americans, SHR was also adjusted for overall European ancestry by including it as a covariate. For ALSPAC, the top ten principal components were included as covariates to adjust for population stratification. For the other cohorts, only the components that were nominally significant (p &lt; 0.05) with SHR were included as covariates to adjust for population stratification. The SHR <em>Z</em> scores were then obtained by taking the resulting residuals divided by the estimated standard deviation of the residuals. The final phenotype for each cohort/group was then obtained by rank-based inversed normalization. This was performed with custom R scripts.</p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">Determining Overall Height Phenotype</h3><p id="__p24" class="p p-first-last">The height residuals were obtained by performing linear regression of height against age and gender and BMI as covariates separately for each of the cohorts. Additionally, the data were analyzed separately for African Americans and European Americans. For the&nbsp;African-American cohorts, we also adjusted for overall European ancestry by including it as a covariate. Among the top ten principal components, the components that were nominally significant (p &lt; 0.05) were also included as covariates to adjust for population stratification. The height <em>Z</em> scores were then obtained by taking the resulting residuals divided by the estimated standard deviation of the residuals. The final phenotype for each cohort group was then obtained by rank-based inversed normalization. This was performed with custom R scripts.</p></div><div id="sec2.11" class="sec"><h3 id="sec2.11title">GCTA-REML Analysis</h3><p id="__p25" class="p p-first-last">We used the GCTA software<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> to calculate the genetic relationship matrix separately for each chromosome by combining the 3,545 African-American individuals from the ARIC, CARDIA, and CHS cohorts. We also performed the same calculations by combining the 11,943 European-American individuals from the ARIC, CARDIA, and CHS cohorts. We used genome-wide autosomal SNPs that were well imputed (INFO &gt; 0.8) in all the African-American and European-American cohorts resulting in a total of 10,250,422 and 6,655,203 SNPs for each ancestry group, respectively. We then merged the genetic relationship matrix calculated for each chromosome to obtain a global genetic relationship matrix for both the African-American and European-American groups. We then performed GCTA-GREML analysis to estimate the variance explained by the SNPs for both ancestry groups with SHR as well as overall height.</p></div><div id="sec2.12" class="sec"><h3 id="sec2.12title">Genome-wide Association Study</h3><p id="__p26" class="p p-first-last">The associations were performed with the SHR phenotype as the outcome variable (see above). Only unrelated individuals were used, i.e., no pair of individuals has PI_HAT &gt; 0.05. The association for the imputed variants with SHR was performed by a linear regression with either PLINK<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup> (v.1.07) or SNPtest<sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup> (v.2.5). SNPs that were imputed poorly were discarded (INFO &lt; 0.6). The association results for each cohort were meta-analyzed together via METAL<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> with genomic control correction. Variants on the X chromosome were analyzed separately between males and females and meta-analyzed together via METAL.<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup> For sex-specific analysis, males and females were separately analyzed via METAL.<sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup></p></div><div id="sec2.13" class="sec"><h3 id="sec2.13title">Enhancer Enrichment Test</h3><p id="__p27" class="p p-first-last">The SNPs (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S2</a>) were mapped from hg18 to hg19 coordinates via the UCSC liftOver tool.<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> For each SNP set, 2,000 sets of matched SNPs were obtained via SNPsnap with its default settings. These matched SNPs were matched for frequency, LD, and gene proximity. Proxy SNPs for each SNP set were also identified with PLINK<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup> based on LD (<em>r</em><sup>2</sup> &gt; 0.5) between SNPs (≤500 kb) in the European (CEU, GBR, TSI) populations in the 1000 Genomes Phase 1 data.<sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup> Next, the set of permissive enhancers defined by the FANTOM5 project<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> were used to screen for enhancer regions that overlap at least one SNP or its proxies in each SNP set. The enhancer enrichment p value was then calculated by taking the proportion of matched SNP sets that contain equal or higher number of enhancers compared to the observed SNP sets.</p></div><div id="sec2.14" class="sec"><h3 id="sec2.14title">Comparison with Known Height-Associated Loci</h3><p id="__p28" class="p p-first-last">We obtained the 697 SNPs shown to be robustly associated with overall height<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> (p &lt; 5&nbsp;× 10<sup>−8</sup>) and analyzed the summary statistics of SHR from a GWAS of 21,590 individuals of European ancestry. We filtered out SNPs that were either not present in the GWAS or failed imputation (INFO &lt; 0.6) in any of the individual cohorts, which resulted in 670 SNPs. We performed conditional and joint GWAS<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup> for the 670 SNPs from the GWAS summary statistics of SHR from linkage-disequilibrium (LD) estimates calculated from 174 CEU individuals from HapMap v.2.<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> We then tabulated the effect sizes and p values for the conditional and joint GWAS for both height and SHR with respect to the height-increasing allele (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>). The QQ plots and 95% confidence intervals were generated with custom R scripts. For the symmetrical analysis, we obtained 725&nbsp;SNPs that were significantly associated (p &lt; 5&nbsp;× 10<sup>−5</sup>) with&nbsp;SHR from a GWAS of 21,590 individuals of European ancestry.&nbsp;We pruned away SNPs that were in strong LD (<em>r</em><sup>2</sup> &gt; 0.1) by using 174 CEU individuals from HapMap v.2, which resulted in 70 SNPs.</p></div><div id="sec2.15" class="sec sec-last"><h3 id="sec2.15title">Pathway/Gene-set Enrichment Analysis via DEPICT</h3><p id="__p29" class="p p-first-last">A comprehensive pathway/gene-set enrichment and tissue/cell type analysis was performed by applying a new method, Data-Driven Expression Prioritized Integration for Complex Traits&nbsp;(DEPICT),<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> which has been recently applied to several studies.<sup><a href="#bib2" rid="bib2 bib33 bib34 bib35 bib36" class=" bibr popnode">2,33–36</a></sup> DEPICT performs gene set enrichment analysis by testing whether genes in GWAS-associated loci are enriched for reconstituted versions of known molecular pathways (jointly referred to as reconstituted gene sets). The reconstitution is accomplished by identifying genes that are co-regulated with other genes in a given gene set based on a panel of 77,840 gene expression microarrays.<sup><a href="#bib37" rid="bib37" class=" bibr popnode">37</a></sup> Genes that are found to be transcriptionally co-regulated with genes from the original gene set are added to the gene set and thus the reconstitution. Several types of gene sets were reconstituted: 5,984 protein molecular pathways derived from 169,810 high-confidence experimentally derived protein-protein interactions,<sup><a href="#bib38" rid="bib38" class=" bibr popnode">38</a></sup> 2,473 phenotypic gene sets derived from 211,882 gene-phenotype pairs from the Mouse Genetics Initiative,<sup><a href="#bib39" rid="bib39" class=" bibr popnode">39</a></sup> 737 Reactome database pathways,<sup><a href="#bib40" rid="bib40" class=" bibr popnode">40</a></sup> 184 Kyoto Encyclopedia of Genes and Genomes (KEGG) database pathways,<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup> and 5,083 Gene Ontology database terms.<sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup> In total, 14,461 gene sets were assessed for enrichment in genes in associated regions. DEPICT also facilitated tissue and cell type enrichment analysis by testing whether the genes in associated regions were highly expressed in any of 209 MeSH annotations for 37,427 microarrays on the Affymetrix U133 Plus 2.0 Array platform.</p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">GCTA-GREML Analysis to Determine Heritability of&nbsp;SHR</h3><p id="__p30" class="p p-first">We performed GCTA-GREML analysis<sup><a href="#bib12" rid="bib12" class=" bibr popnode">12</a></sup> to estimate the variance explained by the autosomal SNPs from 3,545 African Americans and 11,943 European Americans by using unrelated individuals from the ARIC,<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> CHS,<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> CARDIA,<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> and FHS<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> cohorts. The results show that SHR is highly heritable; the phenotypic variance explained by the autosomal SNPs is estimated to be 0.38 (95% CI: 0.22–0.55; p&nbsp;= 4.96&nbsp;× 10<sup>−6</sup>) and 0.26 (95% CI: 0.21–0.31; p = 1.92&nbsp;× 10<sup>−24</sup>) for African Americans and European Americans, respectively (<a href="/pmc/articles/PMC4570286/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). We also tested overall height: the phenotypic variance is estimated to be 0.77 (95% CI: 0.62–0.92; p = 1.66&nbsp;× 10<sup>−20</sup>) and 0.45 (95% CI: 0.4–0.5; p = 1.74&nbsp;× 10<sup>−72</sup>) for African Americans and European Americans, respectively (<a href="/pmc/articles/PMC4570286/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). The variance explained for European Americans on overall height is similar to what was previously estimated.<sup><a href="#bib2" rid="bib2 bib43" class=" bibr popnode">2,43</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p83">Estimation of Phenotypic Variance of SHR Explained by Genetic Relationships from Autosomal SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711132387152" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Phenotype</strong></th><th rowspan="1" colspan="1"><strong>No. of SNPs</strong></th><th rowspan="1" colspan="1"><strong>h</strong><sup><strong>2</strong></sup><strong>(SE)</strong></th><th rowspan="1" colspan="1"><strong>LRT</strong></th><th rowspan="1" colspan="1"><strong>p Value</strong></th></tr></thead><tbody><tr><td colspan="5" rowspan="1"><strong>African Americans (n = 3,545)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">SHR</td><td rowspan="1" colspan="1">10,250,422</td><td rowspan="1" colspan="1" align="center">0.387 (0.0845)</td><td rowspan="1" colspan="1" align="center">19.53</td><td rowspan="1" colspan="1" align="center">4.96&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">10,250,422</td><td rowspan="1" colspan="1" align="center">0.768 (0.0780)</td><td rowspan="1" colspan="1" align="center">84.79</td><td rowspan="1" colspan="1" align="center">1.66&nbsp;× 10<sup>−20</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>European Americans (n = 11,943)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">SHR</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.26 (0.0267)</td><td rowspan="1" colspan="1" align="center">102.73</td><td rowspan="1" colspan="1" align="center">1.92&nbsp;× 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.449 (0.0265)</td><td rowspan="1" colspan="1" align="center">322.84</td><td rowspan="1" colspan="1" align="center">1.74&nbsp;× 10<sup>−72</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>European-American Males (n = 5,477)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">SHR</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.366 (0.0552)</td><td rowspan="1" colspan="1" align="center">46.81</td><td rowspan="1" colspan="1" align="center">3.92&nbsp;× 10<sup>−12</sup></td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.485 (0.0550)</td><td rowspan="1" colspan="1" align="center">83.54</td><td rowspan="1" colspan="1" align="center">3.11&nbsp;× 10<sup>−20</sup></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td colspan="5" rowspan="1"><strong>European-American Females (n = 6,466)</strong></td></tr><tr><td colspan="5" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">SHR</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.188 (0.0454)</td><td rowspan="1" colspan="1" align="center">19.15</td><td rowspan="1" colspan="1" align="center">6.05&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">Height</td><td rowspan="1" colspan="1">6,655,203</td><td rowspan="1" colspan="1" align="center">0.454 (0.0465)</td><td rowspan="1" colspan="1" align="center">100.85</td><td rowspan="1" colspan="1" align="center">4.96&nbsp;× 10<sup>−24</sup></td></tr></tbody></table></div><div id="largeobj_idm140711132387152" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140711166696288"><p id="__p84" class="p p-first-last">The results from GCTA-REML analysis performed on both African- and European-American individuals for SHR as well as overall height using autosomal variants successfully imputed (INFO &gt; 0.8) in all cohorts.</p></div></div></div><p id="__p31" class="p p-last">We further investigated gender-specific heritability of SHR by performing GCTA-GREML analysis separately on European-American males (n = 5,477) and European-American females (n = 6,466). The results show that SHR is significantly more heritable in males than in females; the phenotypic variance explained for males is estimated to be 0.37 (95% CI: 0.26–0.47; p = 3.92&nbsp;× 10<sup>−12</sup>) whereas for females, it is estimated to be 0.188 (95% CI: 0.10–0.28; p = 6.05&nbsp;× 10<sup>−6</sup>) (<a href="/pmc/articles/PMC4570286/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>). Overall height showed no significant gender-specific heritability.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Comparing SHR between African and European Americans</h3><p id="__p32" class="p p-first">We tested whether SHR differs across populations as previously reported.<sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup> We used the ARIC,<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> CARDIA,<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> and CHS<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> cohorts because they include both African Americans and European Americans, resulting in a total sample size of 11,230 European Americans and 3,545 African Americans. We observed that African Americans have lower sitting heights than European Americans (<a href="/pmc/articles/PMC4570286/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figures 1</span></a>A–1C): SHR is 1.09 SD lower in African Americans than European Americans (<a href="/pmc/articles/PMC4570286/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>D), consistent with earlier studies.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4570286/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711202130064" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr1.jpg"></a></div><div id="largeobj_idm140711202130064" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p61">Comparing the Distribution of&nbsp;SHR between European Americans and African Americans</p><p id="__p62">Plots of sitting height and height of individuals, both European Americans and African Americans, from combining the ARIC, CARDIA, and CHS cohorts. Red points indicate individuals of African ancestry; blue points indicate individuals of European ancestry.</p><p id="__p63">(A) Values of sitting height and height in centimeters.</p><p id="__p64">(B) Values of SHR and height in centimeters.</p><p id="__p65">(C) Values of SHR <em>Z</em> scores (adjusted for height, sex, BMI, age, and cohort) and height in centimeters.</p><p id="__p66">(D) The histograms of the SHR <em>Z</em> scores of African Americans (red line) and European Americans (blue line). The dotted horizontal line indicates the mean difference (in standard deviation units) between the two distributions.</p></div></div></div><p id="__p33" class="p p-last">To address whether genetic factors could explain the basis for the SHR difference between populations, we tested whether the degree of European admixture in African Americans is correlated with SHR because African Americans have variable amount of European admixture, with a mean of about 20%.<sup><a href="#bib44" rid="bib44 bib45" class=" bibr popnode">44,45</a></sup> If genetic factors contribute to the difference between populations, the degree of overall European admixture within African Americans should be positively correlated with SHR. We observed a&nbsp;very strong correlation between overall European admixture and SHR in the African-American individuals (<a href="/pmc/articles/PMC4570286/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>); on average, a 1% increase in European admixture is correlated with a 0.0166 SD increase in SHR (p = 8.23&nbsp;× 10<sup>−27</sup>). This result can explain most if not all of the SHR difference between African Americans and European Americans, suggesting that the basis for this difference is mainly genetic.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4570286/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711128544672" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr2.jpg"></a></div><div id="largeobj_idm140711128544672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4570286/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p67">Association of Global European Admixture with SHR in African-American Individuals</p><p id="__p68">The association of global European admixture (<em>x</em> axis) with SHR (<em>y</em>&nbsp;axis) for African-American individuals in the ARIC, CARDIA, and CHS cohorts (n = 3,545). The correlation (red line) obtained from linear regression is shown with the corresponding effect size (β), standard deviation, p value, and <em>r</em><sup>2</sup> (inset).</p></div></div></div></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">GWASs of SHR in African Americans and European Americans</h3><p id="__p34" class="p p-first">To test for specific genetic loci associated with SHR, we performed genome-wide association of SHR on imputed genotypes of the 3,545 African-American individuals from ARIC,<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> CARDIA,<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> and CHS<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> cohorts. A single locus, rs10736877, reached genome-wide significance (p = 2.03&nbsp;× 10<sup>−8</sup>) (<a href="/pmc/articles/PMC4570286/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>; <a href="/pmc/articles/PMC4570286/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figures 3</span></a> and <a href="#app2" rid="app2" class=" sec">S1</a>). rs10736877 is a common variant that is directly genotyped (allele frequency = 51%, effect size = −0.13 SD) and is not in strong LD with its neighbors and thus none of its neighbors show significant associations with SHR. rs10736877 is upstream of <em>C10orf90</em>, the function of which is still unknown. Thus, more samples are required to confirm whether rs10736877 is truly associated with SHR in African Americans.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4570286/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711168779232" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr3.jpg"></a></div><div id="largeobj_idm140711168779232" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p69">GWAS of SHR of African Americans</p><p id="__p70">The GWAS results of SHR of African-American individuals (n = 3,545) shown as a Manhattan plot. The <em>x</em> axis marks the chromosomes and their relative positions of each variant tested and the <em>y</em> axis indicates the –log<sub>10</sub> p value. The candidate gene within the locus that&nbsp;reached genome-wide significance is also shown. The horizontal dashed line represents the threshold of genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p85">Genome-wide Significant SNPs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711202094928" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Variant</strong></th><th rowspan="1" colspan="1"><strong>Location (chr-pos)</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Alleles</strong></th><th rowspan="1" colspan="1"><strong>Freq</strong></th><th colspan="2" rowspan="1"><strong>SHR</strong><hr></th><th rowspan="1" colspan="1"><strong>Gene</strong><sup>b</sup></th></tr><tr><th rowspan="1" colspan="1"><strong>Beta (SE)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td colspan="7" rowspan="1"><strong>African Americans</strong></td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">rs10736877</td><td rowspan="1" colspan="1">chr10: 128,373,707</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.51</td><td rowspan="1" colspan="1" align="center">−0.1311 (0.0234)</td><td rowspan="1" colspan="1" align="center">2.03&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>C10orf90</em></td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td colspan="7" rowspan="1"><strong>European Ancestry</strong></td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">rs6931421</td><td rowspan="1" colspan="1">chr6: 80,880,138</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1" align="center">0.68</td><td rowspan="1" colspan="1" align="center">0.0586 (0.0103)</td><td rowspan="1" colspan="1" align="center">1.45&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>BCKDHB</em></td></tr><tr><td rowspan="1" colspan="1">rs1722141</td><td rowspan="1" colspan="1">chr7: 46,016,662</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.22</td><td rowspan="1" colspan="1" align="center">−0.0667 (0.0121)</td><td rowspan="1" colspan="1" align="center">3.31&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>IGFBP3</em></td></tr><tr><td rowspan="1" colspan="1">rs882367</td><td rowspan="1" colspan="1">chr17: 59,494,574</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.33</td><td rowspan="1" colspan="1" align="center">−0.0778 (0.0104)</td><td rowspan="1" colspan="1" align="center">8.42&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1"><em>TBX2</em></td></tr><tr><td rowspan="1" colspan="1">rs140449984</td><td rowspan="1" colspan="1">chr18: 8,321,896</td><td rowspan="1" colspan="1">T/TAGA…</td><td rowspan="1" colspan="1" align="center">0.0747</td><td rowspan="1" colspan="1" align="center">−0.1043 (0.0188)</td><td rowspan="1" colspan="1" align="center">2.91&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>PTPRM</em></td></tr><tr><td rowspan="1" colspan="1">rs228836</td><td rowspan="1" colspan="1">chr20: 50,065,648</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">0.0608 (0.0099)</td><td rowspan="1" colspan="1" align="center">8.48&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1"><em>NFATC2</em></td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td colspan="7" rowspan="1"><strong>Male-Specific</strong></td></tr><tr><td colspan="7" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">rs35832626</td><td rowspan="1" colspan="1">chr2: 24,054,647</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.41</td><td rowspan="1" colspan="1" align="center">0.0902 (0.0159)</td><td rowspan="1" colspan="1" align="center">1.27&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>ATAD2B</em></td></tr><tr><td rowspan="1" colspan="1">rs34638952</td><td rowspan="1" colspan="1">chr17: 27,537,410</td><td rowspan="1" colspan="1">A/T</td><td rowspan="1" colspan="1" align="center">0.64</td><td rowspan="1" colspan="1" align="center">−0.0906 (0.0159)</td><td rowspan="1" colspan="1" align="center">1.12&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1"><em>MYO18A</em></td></tr></tbody></table></div><div id="largeobj_idm140711202094928" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140711132415376"><p id="__p86" class="p p-first-last">The table shows the variants that reached genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>) when tested for their association with SHR.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>hg19 annotation.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Candidate gene within the locus.</div></div></div><p id="__p35">We also performed genome-wide association of sitting height ratio (SHR) in a total of 21,590 individuals of European ancestry (ARIC,<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup> CARDIA,<sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> CHS,<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> FHS,<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> B58C,<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup> and ALSPAC<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup>). We identified five independent loci reaching genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>) (<a href="/pmc/articles/PMC4570286/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>; <a href="/pmc/articles/PMC4570286/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures 4</span></a> and <a href="#app2" rid="app2" class=" sec">S2</a>). Of the five lead SNPs, two (rs6931421 and rs882367) are in strong LD with variants known to be significantly associated with overall height.<sup><a href="#bib1" rid="bib1 bib2" class=" bibr popnode">1,2</a></sup> Interestingly, rs1722141 is in the <em>IGFBP3</em> (MIM <a href="/omim/146732" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">146732</a>) locus, which has different SNPs significantly associated with overall height (<a href="/pmc/articles/PMC4570286/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>); rs1722141 is not itself associated with height and is not in LD with the height-associated SNPs. The remaining two loci (rs140449984 and rs228836) have not been associated with overall height.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC4570286/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711166875376" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr4.jpg"></a></div><div id="largeobj_idm140711166875376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p71">GWAS of SHR of Individuals with Ancestry from Europe</p><p id="__p72">The GWAS results of SHR of individuals with ancestry from Europe (n = 21,590) shown as a Manhattan plot. The <em>x</em> axis marks the chromosomes and their relative positions of each variant tested and the <em>y</em> axis indicates the –log<sub>10</sub> p value. Candidate genes within loci that reached genome-wide significance are also shown. The horizontal dashed line represents the threshold of genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>).</p></div></div></div><p id="__p36" class="p p-last">Because SHR is significantly different between men and&nbsp;women and their overall heritability were shown to be significantly different, we reanalyzed the European ancestry genome-wide association data for SHR separately for females (n = 12,965) and males (n = 8,625). Although we did not find any additional significant loci for the female-specific analysis (<a href="/pmc/articles/PMC4570286/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figures 5</span></a> and <a href="#app2" rid="app2" class=" sec">S3</a>), two new loci (rs34638952, rs35832626) reached genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>) in the male-specific analysis (<a href="/pmc/articles/PMC4570286/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>; <a href="/pmc/articles/PMC4570286/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Figures 6</span></a> and <a href="#app2" rid="app2" class=" sec">S4</a>). One of these, rs35832626, has also been shown to be significantly associated with overall height, but not in any gender-specific manner<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> (<a href="/pmc/articles/PMC4570286/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC4570286/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711128324768" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr5.jpg"></a></div><div id="largeobj_idm140711128324768" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure&nbsp;5</a></div><!--caption a7--><div class="caption"><p id="__p73">Female-Specific GWAS of SHR of Individuals with Ancestry from Europe</p><p id="__p74">The GWAS results of SHR using only the female individuals with ancestry from Europe (n = 12,965) shown as a Manhattan plot. The <em>x</em> axis marks the chromosomes and their relative positions of each variant tested and the <em>y</em> axis indicates the –log<sub>10</sub> p value. The candidate gene within the locus that reached genome-wide significance is also shown. The horizontal dashed line represents the threshold of genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC4570286/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711130000496" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr6.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr6.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr6.jpg"></a></div><div id="largeobj_idm140711130000496" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Figure&nbsp;6</a></div><!--caption a7--><div class="caption"><p id="__p75">Male-Specific GWAS of SHR of Individuals with Ancestry from Europe</p><p id="__p76">The GWAS results of SHR using only the male individuals with ancestry from Europe (n = 8,625) shown as a Manhattan plot. The <em>x</em> axis marks the chromosomes and their relative positions of each variant tested and the <em>y</em> axis indicates the –log<sub>10</sub> p value. Candidate genes within loci that reached genome-wide significance are also shown. The horizontal dashed line represents the threshold of genome-wide significance (p &lt; 5&nbsp;× 10<sup>−8</sup>).</p></div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Comparison with Known Height-Associated Loci</h3><p id="__p37" class="p p-first">We examined the association statistics of SHR for the SNPs that were reported to be significantly associated with overall height. Among the 697 height-associated SNPs, only 670 SNPs were robustly imputed (INFO &gt; 0.6) in all of our cohorts. We performed conditional and joint GWASs<sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup> for the 670 SNPs from the GWAS summary statistics of SHR. We tabulated the effect sizes and p values for both height and SHR with respect to the height-increasing allele (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S1</a>). We observed a significant enrichment of height-associated SNPs that were also nominally significant with SHR (p &lt; 0.05). A total of 130 SNPs reached nominal significance whereas only 33.5 were expected under the null hypothesis (p = 1.16&nbsp;× 10<sup>−60</sup>) (<a href="/pmc/articles/PMC4570286/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Figure&nbsp;7</span></a>; <a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S2</a>). This result indicates that many of these 130 SNPs have a disproportionate effect on the long bones or spine/head lengths to cause a change not only in SHR, but also in overall height. Among these 130 SNPs, 71 of them had the height-increasing allele predicted to decrease SHR (SNP<sub>longbones</sub>), which indicate that they act disproportionately on the long bones, whereas 59 had the height-increasing allele predicted to increase SHR (SNP<sub>headspine</sub>), which indicate that they act disproportionately to affect head or spine length.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC4570286/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711134046336" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr7.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr7.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr7.jpg"></a></div><div id="largeobj_idm140711134046336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4570286/figure/fig7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Figure&nbsp;7</a></div><!--caption a7--><div class="caption"><p id="__p77">QQ Plot of the 670 Height-Associated SNPs with SHR</p><p id="__p78">The plot shows the observed –log<sub>10</sub> p values of SHR (<em>y</em> axis) for the 670 height-associated SNPs and compare them with the expected –log<sub>10</sub> p values (<em>x</em> axis) under the null hypothesis. The red line is the trend line under the null hypothesis and the dashed lines indicate the 95% confidence interval under the null hypothesis.</p></div></div></div><p id="__p38" class="p p-last">We further performed a symmetric analysis and investigated the SNPs that were associated with SHR (p &lt; 5&nbsp;× 10<sup>−5</sup>) for their association with overall height. After pruning for LD, we obtained 70 such SNPs. Of these 70 SNPs, 32 were nominally significant (p &lt; 0.05) for overall height whereas only 3.5 were expected under the null hypothesis (p = 3.07&nbsp;× 10<sup>−23</sup>). Of these 32 SHR-increasing alleles, 14 were associated with increased height and 18 were associated with decreased height (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S3</a>).</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Enhancer Enrichment Analysis</h3><p id="__p39" class="p p-first">We analyzed the genome-wide significant SNPs (p &lt; 5&nbsp;× 10<sup>−8</sup>) to determine whether there are any overlaps with known enhancer regions. We used the set of permissive and robust enhancers as defined by the FANTOM5 project.<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> We obtained the genomic region surrounding the lead SNPs by taking the other SNPs significant for SHR (p&nbsp;&lt; 5&nbsp;× 10<sup>−6</sup>) spanning the region (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S4</a>). The SNP rs10736877 found to be genome-wide significant from African Americans had no other SNPs in LD spanning the region and was dropped from this analysis. Among the seven remaining genome-wide significant loci, six overlapped with at least one known permissive or robust enhancer region (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S5</a>).</p><p id="__p40" class="p p-last">Next, we investigated the 130 height-associated SNPs that were nominally significant (p &lt; 0.05) for SHR. Using a randomized permutation procedure, we tested whether the genomic regions surrounding these 130 SNPs have significant enrichment with the set of permissive enhancers as defined by the FANTOM5 project.<sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup> We also tested the remaining 540 height-associated SNPs that were not nominally significant for SHR (p &gt; 0.05) in the same way. We observed that although both groups of SNPs had significant enrichment for enhancer regions (p &lt; 0.05), the 130 SNPs are more significantly associated (p = 0.0005) with enhancer regions than the 540 SNPs (p = 0.003) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S6</a>). We further divided the 130 SNPs into the 71 SNP<sub>longbones</sub> and 59 SNP<sub>headspine</sub> SNPs and tested them separately for enrichment of enhancer regions. Again, although we observed that both the SNP<sub>longbones</sub> and SNP<sub>headspine</sub> SNPs were significantly enriched for enhancer regions (p &lt; 0.05), the SNP<sub>longbones</sub> showed more significant enrichment (p = 0.0045) than the SNP<sub>headspine</sub> SNPs (p = 0.0465) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S6</a>). These results suggest that there is more overlap with annotated regulatory regions for height-associated SNPs that are also associated with SHR, especially those that were predicted to act disproportionately on the long bones (SNP<sub>longbones</sub>).</p></div><div id="sec3.6" class="sec sec-last"><h3 id="sec3.6title">Pathway and Tissue Enrichment Analysis by DEPICT</h3><p id="__p41" class="p p-first">We used a recently available method, Data-Driven Expression Prioritized Integration for Complex Traits (DEPICT),<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> to analyze the various groups of SNPs. We first used the list of SNPs that were significant from the earlier GWAS of 21,950 individuals of European ancestry. Because there were only five genome-wide significant SHR loci, which is too few loci to utilize DEPICT, we used a more liberal significance threshold (p &lt; 5&nbsp;× 10<sup>−6</sup>), resulting in 21 distinct genomic loci (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S7</a>). Interestingly, although there was no significant tissue enrichment (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S9</a>), a single pathway/gene set, labeled as “short tibia” (MGI MP: 0002764) was significantly enriched (p = 5.46&nbsp;× 10<sup>−8</sup>; false discovery rate [FDR] &lt; 0.05) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S8</a>). With just these 21 loci, DEPICT also successfully prioritized (FDR &lt; 0.05) a candidate causal gene in three of these loci (<em>NSD1</em> [MIM <a href="/omim/606681" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606681</a>], <em>DMP1</em> [MIM <a href="/omim/600980" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600980</a>], and <em>IBSP</em> [MIM <a href="/omim/147563" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147563</a>]) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S10</a>).</p><p id="__p42">We next analyzed the set of 130 height-associated SNPs that were nominally associated with SHR (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S2</a>). We also separately analyzed a set of 130 height-associated SNPs that have the least significant SHR p values (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S11</a>). We used both sets of SNPs as inputs and analyzed them with DEPICT. For the 130 SNPs that were nominally associated with SHR, DEPICT reported a substantial number (&gt;500) of pathway/gene sets that were significantly enriched (FDR &lt; 0.05) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S12</a>). Many of these gene sets had belonged to pathways involved in bone development, e.g., “abnormal rib morphology,” “chondrocyte differentiation,” “abnormal skeleton morphology,” etc. The “short tibia” gene set found previously was also significant. There were also a number of cell/tissue types (ten types) that showed significant enrichment (FDR &lt; 0.05) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S13</a>). These cell/tissue types included the tissues that are known to be broadly involved in skeletal growth, like cartilage, chondrocytes, and spine. In contrast, the 130 SNPs least associated with SHR had much fewer significant pathways or gene sets that are directly associated with skeletal or bone development and appear to&nbsp;be more strongly associated with pathways involved in early&nbsp;development, e.g., “embryonic growth retardation,” “decreased embryo size,” “decreased cell proliferation,” etc. (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S14</a>). These 130 SNPs also showed significant enrichment in some cell/tissue types (three types) (FDR&nbsp;&lt; 0.05), but none of those directly involved in skeletal growth, i.e., cartilage, chondrocytes, and spine, showed significant enrichment (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S15</a>). By comparing the top 30 pathways or gene sets discovered from either set of SNPs, we demonstrate that the height-associated SNPs nominally associated with SHR are tied more with pathways involved in skeletal, bone, or cartilage development whereas the height-associated SNPs least associated with SHR are tied more to pathways involved in embryonic development (<a href="/pmc/articles/PMC4570286/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Figure&nbsp;8</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig8" co-legend-rid="lgnd_fig8"><a href="/pmc/articles/PMC4570286/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711170093536" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr8.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr8.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr8.jpg"></a></div><div id="largeobj_idm140711170093536" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig8/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig8"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8">Figure&nbsp;8</a></div><!--caption a7--><div class="caption"><p id="__p79">Pathway Analysis by DEPICT of Height-Associated SNPs</p><p id="__p80">Comparing the top 30 gene sets obtained by DEPICT for the 130 known height-associated SNPs also nominally associated with SHR (SHR-positive gene set) as well as the 130 known height-associated SNPs least associated with SHR (SHR-negative gene set). A line connecting two gene sets indicates that the gene sets have a Pearson correlation coefficient &gt; 0.6.</p></div></div></div><p id="__p43" class="p p-last">Finally, we analyzed the 71 SNP<sub>longbones</sub> and 59 SNP<sub>headspine</sub> SNPs separately with DEPICT. The SNP<sub>longbones</sub> SNPs yielded a number of pathways/gene sets (&gt;30) that were significant (FDR &lt; 0.05) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S16</a>). Although the number of significant pathways/gene sets were much smaller than those obtained from using all the 130 SNPs, the results were similar, i.e., the significant gene sets generally belonged to pathways involved in bone development like “abnormal rib morphology,” “skeletal system development,” and “short ribs.” There were also three cell/tissue types that were enriched, i.e., “cartilage,” “mesenchymal stem cells,” and “osteoblasts” (FDR &lt; 0.05) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S17</a>), implicating the tissues involved in cartilage and bone development. In contrast, there were no significant results (FDR &lt; 0.05) from analyzing the SHR<sub>headspine</sub> SNPs (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Tables S18</a> and <a href="#mmc2" rid="mmc2" class=" supplementary-material">S19</a>). This suggests that the genes disproportionately affecting the long bones are more tied to known skeletal growth pathways than are genes disproportionately affecting the head/spine, although this could be due to having fewer SNP<sub>headspine</sub> loci than SNP<sub>longbones</sub> loci. Nonetheless, these results demonstrate that the height-associated SNPs can be stratified into different biological groups by examining their respective effects on SHR.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p44" class="p p-first">Unlike overall height, body proportion or sitting height ratio (SHR) has not been well studied and less is known about its underlying genetics. Here, we have provided evidence that SHR is heritable and we have identified loci that are significantly associated with SHR. We also showed that the lower SHR in African-ancestry populations is consistent with an effect of genetic factors. However, the GWAS performed on African-American samples did not identify loci that either explains much of the heritability of SHR nor much of the difference between European- and African-ancestry populations. These results suggest that the difference in SHR is polygenic, i.e., due to many loci.</p><p id="__p45">The results from the GWAS of individuals of European ancestry yielded more genome-wide significant results, probably because of the larger number of samples. The most significant locus (rs882367) has been shown to be significantly associated with overall height.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> Although the most plausible candidate gene in this locus (<em>TBX2</em> [MIM <a href="/omim/600747" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600747</a>]) has been demonstrated to affect heart development,<sup><a href="#bib46" rid="bib46 bib47" class=" bibr popnode">46,47</a></sup> it has also been suggested to cause limb abnormalities and skeletal malformations.<sup><a href="#bib48" rid="bib48 bib49" class=" bibr popnode">48,49</a></sup> Another significant locus (rs1722141), although not significantly associated with overall height, contains multiple SNPs, not correlated to rs1722141, that are strongly associated with height. The likely gene underlying this locus (<em>IGFBP3</em> [MIM <a href="/omim/146732" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">146732</a>]) is known to bind to and modulate IGF by increasing IGF half-life and inhibiting or potentiating IGF binding to the IGF receptor,<sup><a href="#bib50" rid="bib50" class=" bibr popnode">50</a></sup> which has been directly tied to skeletal growth.<sup><a href="#bib51" rid="bib51" class=" bibr popnode">51</a></sup> The SNPs that are associated with SHR, but not overall height, have also been shown to be significantly correlated with circulating levels of IGFBP3,<sup><a href="#bib52" rid="bib52" class=" bibr popnode">52</a></sup> suggesting that circulating IGFBP3 levels might influence body proportion.</p><p id="__p46">Some of the SNPs that showed suggestive significance (p &lt; 5&nbsp;× 10<sup>−6</sup>) are in loci with genes underlying Mendelian causes of growth disorders and/or skeletal dysplasias that are known to affect body proportion. One example, rs2011077 (p = 2.52&nbsp;× 10<sup>−7</sup>), is near <em>NSD1</em>, a known cause of Sotos syndrome (MIM <a href="/omim/117550" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">117550</a>), which is characterized by extreme overgrowth in childhood, large head size, and advanced bone age. Individuals affected with Sotos syndrome have a decreased upper to lower body segment ratio as compared to age- and gender-matched controls.<sup><a href="#bib53" rid="bib53" class=" bibr popnode">53</a></sup></p><p id="__p47">We also demonstrated that SHR is a useful tool to understand the underlying biology of height-associated loci. Although height and SHR are orthogonal traits, we showed that there are significant overlaps between the loci affecting SHR and the loci affecting overall height. Height-increasing alleles that cause a reduction in SHR have to have disproportionate effect such that the increase in long bone length is relatively larger than the increase in&nbsp;the spine and/or head length. In contrast, height-increasing alleles that cause an increase in SHR have a relatively larger effect on the spine and/or head. From the pathway analysis by DEPICT, we have shown that the loci affecting height disproportionately through the long bones (SNP<sub>longbones</sub>) are involved in fundamental processes of bone and cartilage development whereas loci in the other sets are less clearly connected to these processes.</p><p id="__p48">Recent work has raised the possibility that adjusting one trait for a highly correlated covariate can lead to false-positive “mirror” associations at variants that are strongly associated with the covariate.<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup> This phenomenon will cause such variants to be associated with the trait but have a direction of effect that is opposite of the covariate. This phenomenon does not explain our result because our SHR associations are divided between SNPs that are associated with increased and decreased height. Furthermore, SHR is only weakly correlated with height and not all height-associated SNPs at genome-wide significance are associated with SHR.</p><p id="__p49" class="p p-last">Overall, this study is a large-scale whole-genome experiment to address the underlying genetic basis for differences in body proportion by SHR. We uncovered several loci that are significantly associated with SHR and showed that many loci associated with height also alter SHR. These&nbsp;results suggest that SHR is also polygenic and that further studies of larger sample sizes are required to explain&nbsp;the full genetic spectrum of SHR. Finally, we demonstrated that SHR can be used as a prism (<a href="/pmc/articles/PMC4570286/figure/fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig9" rid-ob="ob-fig9" co-legend-rid="lgnd_fig9"><span>Figure&nbsp;9</span></a>) to partition the&nbsp;height-associated loci into those that affect the head/spine, the long bones, or have no effect on SHR and showed that different groups of loci are enriched for different pathways and tissue types. This suggest that in general, the use of orthogonal but related phenotypes can be used as a tool to dissect the variants into different functional classes so as to better understand the biological mechanism underlying the trait or disease.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig9" co-legend-rid="lgnd_fig9"><a href="/pmc/articles/PMC4570286/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140711168437600" class="figure"><a href="/pmc/articles/PMC4570286/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4570286_gr9.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr9.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4570286/bin/gr9.jpg"></a></div><div id="largeobj_idm140711168437600" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4570286/figure/fig9/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig9"><div><a class="figpopup" href="/pmc/articles/PMC4570286/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9">Figure&nbsp;9</a></div><!--caption a7--><div class="caption"><p id="__p81">SHR as a Prism to Dissect the Height-Associated SNPs into Functional Classes</p><p id="__p82">This diagram illustrates the use of an orthogonal phenotype like SHR to partition the known variants associated with overall height into various functional classes (head/spine, long bones, and balanced).</p></div></div></div></div><div id="ack0010" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="__p59">This publication is the work of the authors and Y.C. and J.N.H. will serve as guarantors for the contents of this paper. This research was specifically funded by NIH grant R01DK075787 and March of Dimes grant 6-FY12-393 to J.N.H. R.M.S. was supported by an NIH-NHLBI K99 award (#1K99HL122515-01A1) and advanced postdoctoral fellowship award from the Juvenile Diabetes Research Foundation (JDRF # 3-APF-2014-111-A-N). Acknowledgments for each of the cohorts used in this study can be found in the <a href="#app2" rid="app2" class=" sec">Supplemental Data</a>.</p></div></div><div id="idm140711132496864" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711132496864title">Notes</h2><p id="misc0010">Published: April 9, 2015</p></div><div id="idm140711134080704" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711134080704title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="app1"></p><p id="__p60" class="p p-first-last">Supplemental Data include GIANT consortium information, cohort-specific acknowledgments, 4 figures, and 19 tables, and can be found with this article online at <a href="//dx.doi.org/10.1016/j.ajhg.2015.02.018" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">http://dx.doi.org/10.1016/j.ajhg.2015.02.018</a>.</p><p></p></div></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p50" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p51">1000 Genomes, <a href="http://browser.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://browser.1000genomes.org</a></p></li><li id="u0015"><p id="__p52">ALSPAC data dictionary, <a href="http://www.bris.ac.uk/alspac/researchers/data-access/data-dictionary/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.bris.ac.uk/alspac/researchers/data-access/data-dictionary/</a></p></li><li id="u0020"><p id="__p53">dbGaP, <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a></p></li><li id="u0025"><p id="__p54">DEPICT, <a href="http://www.broadinstitute.org/mpg/depict" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/depict</a></p></li><li id="u0030"><p id="__p55">GENE PRISM, <a href="http://www.broadinstitute.org/%7Erigel/prism/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/∼rigel/prism/</a></p></li><li id="u0035"><p id="__p56">OMIM, <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a></p></li><li id="u0040"><p id="__p57">SNPsnap, <a href="http://www.broadinstitute.org/mpg/snpsnap/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mpg/snpsnap/index.html</a></p></li></ul><p></p></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. GIANT Consortium Information, Cohort-Specific Acknowledgments, and Figures S1–S4:</strong><div class="sup-box half_rhythm" id="idm140711129142800"><a href="/pmc/articles/PMC4570286/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(670K, pdf)</sup></div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Data S1. Tables S1–S19:</strong><div class="sup-box half_rhythm" id="idm140711129140480"><a href="/pmc/articles/PMC4570286/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.8M, xlsx)</sup></div></div><div class="sec suppmat" id="mmc3"><!--caption a9--><strong>Document S3. Article plus Supplemental Data:</strong><div class="sup-box half_rhythm" id="idm140711134084048"><a href="/pmc/articles/PMC4570286/bin/mmc3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.2M, pdf)</sup></div></div><p></p></div><div id="idm140711123986256" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140711123986256title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Lango Allen H., Estrada K., Lettre G., Berndt S.I., Weedon M.N., Rivadeneira F., Willer C.J., Jackson A.U., Vedantam S., Raychaudhuri S. Hundreds of variants clustered in genomic loci and biological pathways affect human height. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:832–838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955183/">PMC free article</a>]</span> [<a href="/pubmed/20881960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Hundreds+of+variants+clustered+in+genomic+loci+and+biological+pathways+affect+human+height&amp;author=H.+Lango+Allen&amp;author=K.+Estrada&amp;author=G.+Lettre&amp;author=S.I.+Berndt&amp;author=M.N.+Weedon&amp;volume=467&amp;publication_year=2010&amp;pages=832-838&amp;pmid=20881960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Wood A.R., Esko T., Yang J., Vedantam S., Pers T.H., Gustafsson S., Chu A.Y., Estrada K., Luan J., Kutalik Z., Electronic Medical Records and Genomics (eMEMERGEGE) Consortium. MIGen Consortium. PAGEGE Consortium. LifeLines Cohort Study  Defining the role of common variation in the genomic and biological architecture of adult human height. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:1173–1186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4250049/">PMC free article</a>]</span> [<a href="/pubmed/25282103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Defining+the+role+of+common+variation+in+the+genomic+and+biological+architecture+of+adult+human+height&amp;author=A.R.+Wood&amp;author=T.+Esko&amp;author=J.+Yang&amp;author=S.+Vedantam&amp;author=T.H.+Pers&amp;volume=46&amp;publication_year=2014&amp;pages=1173-1186&amp;pmid=25282103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Fredriks A.M., van Buuren S., van Heel W.J.M., Dijkman-Neerincx R.H., Verloove-Vanhorick S.P., Wit J.M. Nationwide age references for sitting height, leg length, and sitting height/height ratio, and their diagnostic value for disproportionate growth disorders. <span><span class="ref-journal">Arch. Dis. Child. </span>2005;<span class="ref-vol">90</span>:807–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1720514/">PMC free article</a>]</span> [<a href="/pubmed/15863466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dis.+Child.&amp;title=Nationwide+age+references+for+sitting+height,+leg+length,+and+sitting+height/height+ratio,+and+their+diagnostic+value+for+disproportionate+growth+disorders&amp;author=A.M.+Fredriks&amp;author=S.+van+Buuren&amp;author=W.J.M.+van+Heel&amp;author=R.H.+Dijkman-Neerincx&amp;author=S.P.+Verloove-Vanhorick&amp;volume=90&amp;publication_year=2005&amp;pages=807-812&amp;pmid=15863466&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">de Arriba Muñoz A., Domínguez Cajal M., Rueda Caballero C., Labarta Aizpún J.I., Mayayo Dehesa E., Ferrández Longás Á. Sitting height/standing height ratio in a Spanish population from birth to adulthood. <span><span class="ref-journal">Arch. Argent. Pediatr. </span>2013;<span class="ref-vol">111</span>:309–314.</span> [<a href="/pubmed/23912288" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Argent.+Pediatr.&amp;title=Sitting+height/standing+height+ratio+in+a+Spanish+population+from+birth+to+adulthood&amp;author=A.+de+Arriba+Muñoz&amp;author=M.+Domínguez+Cajal&amp;author=C.+Rueda+Caballero&amp;author=J.I.+Labarta+Aizpún&amp;author=E.+Mayayo+Dehesa&amp;volume=111&amp;publication_year=2013&amp;pages=309-314&amp;pmid=23912288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Krakow D., Rimoin D.L. The skeletal dysplasias. <span><span class="ref-journal">Genet. Med. </span>2010;<span class="ref-vol">12</span>:327–341.</span> [<a href="/pubmed/20556869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Med.&amp;title=The+skeletal+dysplasias&amp;author=D.+Krakow&amp;author=D.L.+Rimoin&amp;volume=12&amp;publication_year=2010&amp;pages=327-341&amp;pmid=20556869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Stokes D.C., Pyeritz R.E., Wise R.A., Fairclough D., Murphy E.A. Spirometry and chest wall dimensions in achondroplasia. <span><span class="ref-journal">Chest. </span>1988;<span class="ref-vol">93</span>:364–369.</span> [<a href="/pubmed/3338305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chest&amp;title=Spirometry+and+chest+wall+dimensions+in+achondroplasia&amp;author=D.C.+Stokes&amp;author=R.E.+Pyeritz&amp;author=R.A.+Wise&amp;author=D.+Fairclough&amp;author=E.A.+Murphy&amp;volume=93&amp;publication_year=1988&amp;pages=364-369&amp;pmid=3338305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Hertel N.T., Müller J. Anthropometry in skeletal dysplasia. <span><span class="ref-journal">J.&nbsp;Pediatr. Endocrinol. Metab. </span>1994;<span class="ref-vol">7</span>:155–161.</span> [<a href="/pubmed/8061761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Pediatr.+Endocrinol.+Metab.&amp;title=Anthropometry+in+skeletal+dysplasia&amp;author=N.T.+Hertel&amp;author=J.+Müller&amp;volume=7&amp;publication_year=1994&amp;pages=155-161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Gunnell D., Whitley E., Upton M.N., McConnachie A., Smith G.D., Watt G.C.M. Associations of height, leg length, and lung function with cardiovascular risk factors in the Midspan Family Study. <span><span class="ref-journal">J.&nbsp;Epidemiol. Community Health. </span>2003;<span class="ref-vol">57</span>:141–146.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1732388/">PMC free article</a>]</span> [<a href="/pubmed/12540691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Epidemiol.+Community+Health&amp;title=Associations+of+height,+leg+length,+and+lung+function+with+cardiovascular+risk+factors+in+the+Midspan+Family+Study&amp;author=D.+Gunnell&amp;author=E.+Whitley&amp;author=M.N.+Upton&amp;author=A.+McConnachie&amp;author=G.D.+Smith&amp;volume=57&amp;publication_year=2003&amp;pages=141-146&amp;pmid=12540691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Tilling K., Lawlor D.A., Davey Smith G., Chambless L., Szklo M. The relation between components of adult height and intimal-medial thickness in middle age: the Atherosclerosis Risk in Communities Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>2006;<span class="ref-vol">164</span>:136–142.</span> [<a href="/pubmed/16707651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=The+relation+between+components+of+adult+height+and+intimal-medial+thickness+in+middle+age:+the+Atherosclerosis+Risk+in+Communities+Study&amp;author=K.+Tilling&amp;author=D.A.+Lawlor&amp;author=G.+Davey+Smith&amp;author=L.+Chambless&amp;author=M.+Szklo&amp;volume=164&amp;publication_year=2006&amp;pages=136-142&amp;pmid=16707651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Lawlor D.A., Ebrahim S., Davey Smith G. The association between components of adult height and type II diabetes and insulin resistance: British Women’s Heart and Health Study. <span><span class="ref-journal">Diabetologia. </span>2002;<span class="ref-vol">45</span>:1097–1106.</span> [<a href="/pubmed/12189439" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia&amp;title=The+association+between+components+of+adult+height+and+type+II+diabetes+and+insulin+resistance:+British+Women’s+Heart+and+Health+Study&amp;author=D.A.+Lawlor&amp;author=S.+Ebrahim&amp;author=G.+Davey+Smith&amp;volume=45&amp;publication_year=2002&amp;pages=1097-1106&amp;pmid=12189439&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Bogin B., Varela-Silva M.I. Leg length, body proportion, and health: a review with a note on beauty. <span><span class="ref-journal">Int. J. Environ. Res. Public Health. </span>2010;<span class="ref-vol">7</span>:1047–1075.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872302/">PMC free article</a>]</span> [<a href="/pubmed/20617018" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Environ.+Res.+Public+Health&amp;title=Leg+length,+body+proportion,+and+health:+a+review+with+a+note+on+beauty&amp;author=B.+Bogin&amp;author=M.I.+Varela-Silva&amp;volume=7&amp;publication_year=2010&amp;pages=1047-1075&amp;pmid=20617018&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Yang J., Lee S.H., Goddard M.E., Visscher P.M. Genome-wide complex trait analysis (GCTA): methods, data analyses, and interpretations. <span><span class="ref-journal">Methods Mol. Biol. </span>2013;<span class="ref-vol">1019</span>:215–236.</span> [<a href="/pubmed/23756893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Genome-wide+complex+trait+analysis+(GCTA):+methods,+data+analyses,+and+interpretations&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=1019&amp;publication_year=2013&amp;pages=215-236&amp;pmid=23756893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">The Atherosclerosis Risk in Communities (ARIC) Study: design and objectives. The ARIC investigators. <span><span class="ref-journal">Am. J. Epidemiol. </span>1989;<span class="ref-vol">129</span>:687–702.</span> [<a href="/pubmed/2646917" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=The+Atherosclerosis+Risk+in+Communities+(ARIC)+Study:+design+and+objectives.+The+ARIC+investigators&amp;volume=129&amp;publication_year=1989&amp;pages=687-702&amp;pmid=2646917&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Korn J.M., Kuruvilla F.G., McCarroll S.A., Wysoker A., Nemesh J., Cawley S., Hubbell E., Veitch J., Collins P.J., Darvishi K. Integrated genotype calling and association analysis of SNPs, common copy number polymorphisms and rare CNVs. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1253–1260.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2756534/">PMC free article</a>]</span> [<a href="/pubmed/18776909" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Integrated+genotype+calling+and+association+analysis+of+SNPs,+common+copy+number+polymorphisms+and+rare+CNVs&amp;author=J.M.+Korn&amp;author=F.G.+Kuruvilla&amp;author=S.A.+McCarroll&amp;author=A.+Wysoker&amp;author=J.+Nemesh&amp;volume=40&amp;publication_year=2008&amp;pages=1253-1260&amp;pmid=18776909&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Fried L.P., Kronmal R.A., Newman A.B., Bild D.E., Mittelmark M.B., Polak J.F., Robbins J.A., Gardin J.M. Risk factors for 5-year mortality in older adults: the Cardiovascular Health Study. <span><span class="ref-journal">J.&nbsp;Am. Med. Assoc. </span>1998;<span class="ref-vol">279</span>:585–592.</span> [<a href="/pubmed/9486752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Med.+Assoc.&amp;title=Risk+factors+for+5-year+mortality+in+older+adults:+the+Cardiovascular+Health+Study&amp;author=L.P.+Fried&amp;author=R.A.+Kronmal&amp;author=A.B.+Newman&amp;author=D.E.+Bild&amp;author=M.B.+Mittelmark&amp;volume=279&amp;publication_year=1998&amp;pages=585-592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Gardin J.M., Wagenknecht L.E., Anton-Culver H., Flack J., Gidding S., Kurosaki T., Wong N.D., Manolio T.A. Relationship of cardiovascular risk factors to echocardiographic left ventricular mass in healthy young black and white adult men and women. The CARDIA study. Coronary Artery Risk Development in Young Adults. <span><span class="ref-journal">Circulation. </span>1995;<span class="ref-vol">92</span>:380–387.</span> [<a href="/pubmed/7634452" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Relationship+of+cardiovascular+risk+factors+to+echocardiographic+left+ventricular+mass+in+healthy+young+black+and+white+adult+men+and+women.+The+CARDIA+study.+Coronary+Artery+Risk+Development+in+Young+Adults&amp;author=J.M.+Gardin&amp;author=L.E.+Wagenknecht&amp;author=H.+Anton-Culver&amp;author=J.+Flack&amp;author=S.+Gidding&amp;volume=92&amp;publication_year=1995&amp;pages=380-387&amp;pmid=7634452&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Wilson P.W.F., Castelli W.P., Kannel W.B. Coronary risk prediction in adults (the Framingham Heart Study) <span><span class="ref-journal">Am. J. Cardiol. </span>1987;<span class="ref-vol">59</span>:91G–94G.</span> [<a href="/pubmed/3591674" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Cardiol.&amp;title=Coronary+risk+prediction+in+adults+(the+Framingham+Heart+Study)&amp;author=P.W.F.+Wilson&amp;author=W.P.+Castelli&amp;author=W.B.+Kannel&amp;volume=59&amp;publication_year=1987&amp;pages=91G-94G&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Strachan D.P., Rudnicka A.R., Power C., Shepherd P., Fuller E., Davis A., Gibb I., Kumari M., Rumley A., Macfarlane G.J. Lifecourse influences on health among British adults: effects of region of residence in childhood and adulthood. <span><span class="ref-journal">Int. J. Epidemiol. </span>2007;<span class="ref-vol">36</span>:522–531.</span> [<a href="/pubmed/17255346" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Lifecourse+influences+on+health+among+British+adults:+effects+of+region+of+residence+in+childhood+and+adulthood&amp;author=D.P.+Strachan&amp;author=A.R.+Rudnicka&amp;author=C.+Power&amp;author=P.+Shepherd&amp;author=E.+Fuller&amp;volume=36&amp;publication_year=2007&amp;pages=522-531&amp;pmid=17255346&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Fraser A., Macdonald-Wallis C., Tilling K., Boyd A., Golding J., Davey Smith G., Henderson J., Macleod J., Molloy L., Ness A. Cohort Profile: the Avon Longitudinal Study of Parents and Children: ALSPAC mothers cohort. <span><span class="ref-journal">Int. J. Epidemiol. </span>2013;<span class="ref-vol">42</span>:97–110.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3600619/">PMC free article</a>]</span> [<a href="/pubmed/22507742" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Epidemiol.&amp;title=Cohort+Profile:+the+Avon+Longitudinal+Study+of+Parents+and+Children:+ALSPAC+mothers+cohort&amp;author=A.+Fraser&amp;author=C.+Macdonald-Wallis&amp;author=K.+Tilling&amp;author=A.+Boyd&amp;author=J.+Golding&amp;volume=42&amp;publication_year=2013&amp;pages=97-110&amp;pmid=22507742&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Fujita P.A., Rhead B., Zweig A.S., Hinrichs A.S., Karolchik D., Cline M.S., Goldman M., Barber G.P., Clawson H., Coelho A. The UCSC Genome Browser database: update 2011. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D876–D882.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3242726/">PMC free article</a>]</span> [<a href="/pubmed/20959295" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+UCSC+Genome+Browser+database:+update+2011&amp;author=P.A.+Fujita&amp;author=B.+Rhead&amp;author=A.S.+Zweig&amp;author=A.S.+Hinrichs&amp;author=D.+Karolchik&amp;volume=39&amp;publication_year=2011&amp;pages=D876-D882&amp;pmid=20959295&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">International HapMap Consortium  A haplotype map of the human genome. <span><span class="ref-journal">Nature. </span>2005;<span class="ref-vol">437</span>:1299–1320.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1880871/">PMC free article</a>]</span> [<a href="/pubmed/16255080" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+haplotype+map+of+the+human+genome&amp;volume=437&amp;publication_year=2005&amp;pages=1299-1320&amp;pmid=16255080&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A.R., Bender D., Maller J., Sklar P., de Bakker P.I.W., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.R.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Delaneau O., Zagury J.-F., Marchini J. Improved whole-chromosome phasing for disease and population genetic studies. <span><span class="ref-journal">Nat. Methods. </span>2013;<span class="ref-vol">10</span>:5–6.</span> [<a href="/pubmed/23269371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Improved+whole-chromosome+phasing+for+disease+and+population+genetic+studies&amp;author=O.+Delaneau&amp;author=J.-F.+Zagury&amp;author=J.+Marchini&amp;volume=10&amp;publication_year=2013&amp;pages=5-6&amp;pmid=23269371&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Andersson R., Gebhard C., Miguel-Escalada I., Hoof I., Bornholdt J., Boyd M., Chen Y., Zhao X., Schmidl C., Suzuki T., FANTOM Consortium  An atlas of active enhancers across human cell types and tissues. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">507</span>:455–461.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5215096/">PMC free article</a>]</span> [<a href="/pubmed/24670763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+atlas+of+active+enhancers+across+human+cell+types+and+tissues&amp;author=R.+Andersson&amp;author=C.+Gebhard&amp;author=I.+Miguel-Escalada&amp;author=I.+Hoof&amp;author=J.+Bornholdt&amp;volume=507&amp;publication_year=2014&amp;pages=455-461&amp;pmid=24670763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Yang J., Ferreira T., Morris A.P., Medland S.E., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W., Weedon M.N., Loos R.J., Genetic Investigation of ANthropometric Traits (GIANT) Consortium. DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium  Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:369–375.</span> S1–S3. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J.+Yang&amp;author=T.+Ferreira&amp;author=A.P.+Morris&amp;author=S.E.+Medland&amp;author=P.A.+Madden&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Pers T.H., Karjalainen J.M., Chan Y., Westra H.-J., Wood A.R., Yang J., Lui J.C., Vedantam S., Gustafsson S., Esko T., Genetic Investigation of ANthropometric Traits (GIANT) Consortium  Biological interpretation of genome-wide association studies using predicted gene functions. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:5890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4420238/">PMC free article</a>]</span> [<a href="/pubmed/25597830" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Biological+interpretation+of+genome-wide+association+studies+using+predicted+gene+functions&amp;author=T.H.+Pers&amp;author=J.M.+Karjalainen&amp;author=Y.+Chan&amp;author=H.-J.+Westra&amp;author=A.R.+Wood&amp;volume=6&amp;publication_year=2015&amp;pages=5890&amp;pmid=25597830&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Geller F., Feenstra B., Carstensen L., Pers T.H., van Rooij I.A.L.M., Körberg I.B., Choudhry S., Karjalainen J.M., Schnack T.H., Hollegaard M.V. Genome-wide association analyses identify variants in developmental genes associated with hypospadias. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:957–963.</span> [<a href="/pubmed/25108383" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analyses+identify+variants+in+developmental+genes+associated+with+hypospadias&amp;author=F.+Geller&amp;author=B.+Feenstra&amp;author=L.+Carstensen&amp;author=T.H.+Pers&amp;author=I.A.L.M.+van+Rooij&amp;volume=46&amp;publication_year=2014&amp;pages=957-963&amp;pmid=25108383&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Van der Valk R.J.P., Kreiner-Møller E., Kooijman M.N., Guxens M., Stergiakouli E., Sääf A., Bradfield J.P., Geller F., Hayes M.G., Cousminer D.L. A novel common variant in DCST2 is associated with length in early life and height in adulthood. <span><span class="ref-journal">Hum. Mol. Genet. </span>2015;<span class="ref-vol">24</span>:1155–1168.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4447786/">PMC free article</a>]</span> [<a href="/pubmed/25281659" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+novel+common+variant+in+DCST2+is+associated+with+length+in+early+life+and+height+in+adulthood&amp;author=R.J.P.+Van+der+Valk&amp;author=E.+Kreiner-Møller&amp;author=M.N.+Kooijman&amp;author=M.+Guxens&amp;author=E.+Stergiakouli&amp;volume=24&amp;publication_year=2015&amp;pages=1155-1168&amp;pmid=25281659&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Locke A.E., Kahali B., Berndt S.I., Justice A.E., Pers T.H., Day F.R., Powell C., Vedantam S., Buchkovich M.L., Yang J., LifeLines Cohort Study. ADIPOGen Consortium. AGEN-BMI Working Group. CARDIOGRAMplusC4D Consortium. CKDGen Consortium. GLGC. ICBP. MAGIC Investigators. MuTHER Consortium. MIGen Consortium. PAGE Consortium. ReproGen Consortium. GENIE Consortium. International Endogene Consortium  Genetic studies of body mass index yield new insights for obesity biology. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:197–206.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4382211/">PMC free article</a>]</span> [<a href="/pubmed/25673413" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic+studies+of+body+mass+index+yield+new+insights+for+obesity+biology&amp;author=A.E.+Locke&amp;author=B.+Kahali&amp;author=S.I.+Berndt&amp;author=A.E.+Justice&amp;author=T.H.+Pers&amp;volume=518&amp;publication_year=2015&amp;pages=197-206&amp;pmid=25673413&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Shungin D., Winkler T.W., Croteau-Chonka D.C., Ferreira T., Locke A.E., Mägi R., Strawbridge R.J., Pers T.H., Fischer K., Justice A.E., ADIPOGen Consortium. CARDIOGRAMplusC4D Consortium. CKDGen Consortium. GEFOS Consortium. GENIE Consortium. GLGC. ICBP. International Endogene Consortium. LifeLines Cohort Study. MAGIC Investigators. MuTHER Consortium. PAGE Consortium. ReproGen Consortium  New genetic loci link adipose and insulin biology to body fat distribution. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:187–196.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4338562/">PMC free article</a>]</span> [<a href="/pubmed/25673412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=New+genetic+loci+link+adipose+and+insulin+biology+to+body+fat+distribution&amp;author=D.+Shungin&amp;author=T.W.+Winkler&amp;author=D.C.+Croteau-Chonka&amp;author=T.+Ferreira&amp;author=A.E.+Locke&amp;volume=518&amp;publication_year=2015&amp;pages=187-196&amp;pmid=25673412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Fehrmann R.S.N., Karjalainen J.M., Krajewska M., Westra H.-J., Maloney D., Simeonov A., Pers T.H., Hirschhorn J.N., Jansen R.C., Schultes E.A. Gene expression analysis identifies global gene dosage sensitivity in cancer. <span><span class="ref-journal">Nat. Genet. </span>2015;<span class="ref-vol">47</span>:115–125.</span> [<a href="/pubmed/25581432" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Gene+expression+analysis+identifies+global+gene+dosage+sensitivity+in+cancer&amp;author=R.S.N.+Fehrmann&amp;author=J.M.+Karjalainen&amp;author=M.+Krajewska&amp;author=H.-J.+Westra&amp;author=D.+Maloney&amp;volume=47&amp;publication_year=2015&amp;pages=115-125&amp;pmid=25581432&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Lage K., Karlberg E.O., Størling Z.M., Olason P.I., Pedersen A.G., Rigina O., Hinsby A.M., Tümer Z., Pociot F., Tommerup N. A human phenome-interactome network of protein complexes implicated in genetic disorders. <span><span class="ref-journal">Nat. Biotechnol. </span>2007;<span class="ref-vol">25</span>:309–316.</span> [<a href="/pubmed/17344885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=A+human+phenome-interactome+network+of+protein+complexes+implicated+in+genetic+disorders&amp;author=K.+Lage&amp;author=E.O.+Karlberg&amp;author=Z.M.+Størling&amp;author=P.I.+Olason&amp;author=A.G.+Pedersen&amp;volume=25&amp;publication_year=2007&amp;pages=309-316&amp;pmid=17344885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Blake J.A., Bult C.J., Eppig J.T., Kadin J.A., Richardson J.E., Mouse Genome Database Group  The Mouse Genome Database: integration of and access to knowledge about the laboratory mouse. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D810–D817.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3964950/">PMC free article</a>]</span> [<a href="/pubmed/24285300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+Mouse+Genome+Database:+integration+of+and+access+to+knowledge+about+the+laboratory+mouse&amp;author=J.A.+Blake&amp;author=C.J.+Bult&amp;author=J.T.+Eppig&amp;author=J.A.+Kadin&amp;author=J.E.+Richardson&amp;volume=42&amp;publication_year=2014&amp;pages=D810-D817&amp;pmid=24285300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Croft D., O’Kelly G., Wu G., Haw R., Gillespie M., Matthews L., Caudy M., Garapati P., Gopinath G., Jassal B. Reactome: a database of reactions, pathways and biological processes. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D691–D697.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013646/">PMC free article</a>]</span> [<a href="/pubmed/21067998" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Reactome:+a+database+of+reactions,+pathways+and+biological+processes&amp;author=D.+Croft&amp;author=G.+O’Kelly&amp;author=G.+Wu&amp;author=R.+Haw&amp;author=M.+Gillespie&amp;volume=39&amp;publication_year=2011&amp;pages=D691-D697&amp;pmid=21067998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Kanehisa M., Goto S., Sato Y., Furumichi M., Tanabe M. KEGG for integration and interpretation of large-scale molecular data sets. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:D109–D114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245020/">PMC free article</a>]</span> [<a href="/pubmed/22080510" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=KEGG+for+integration+and+interpretation+of+large-scale+molecular+data+sets&amp;author=M.+Kanehisa&amp;author=S.+Goto&amp;author=Y.+Sato&amp;author=M.+Furumichi&amp;author=M.+Tanabe&amp;volume=40&amp;publication_year=2012&amp;pages=D109-D114&amp;pmid=22080510&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Ashburner M., Ball C.A., Blake J.A., Botstein D., Butler H., Cherry J.M., Davis A.P., Dolinski K., Dwight S.S., Eppig J.T., The Gene Ontology Consortium  Gene ontology: tool for the unification of biology. <span><span class="ref-journal">Nat. Genet. </span>2000;<span class="ref-vol">25</span>:25–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037419/">PMC free article</a>]</span> [<a href="/pubmed/10802651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Gene+ontology:+tool+for+the+unification+of+biology&amp;author=M.+Ashburner&amp;author=C.A.+Ball&amp;author=J.A.+Blake&amp;author=D.+Botstein&amp;author=H.+Butler&amp;volume=25&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Yang J., Benyamin B., McEvoy B.P., Gordon S., Henders A.K., Nyholt D.R., Madden P.A., Heath A.C., Martin N.G., Montgomery G.W. Common SNPs explain a large proportion of the heritability for human height. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:565–569.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3232052/">PMC free article</a>]</span> [<a href="/pubmed/20562875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+SNPs+explain+a+large+proportion+of+the+heritability+for+human+height&amp;author=J.+Yang&amp;author=B.+Benyamin&amp;author=B.P.+McEvoy&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=42&amp;publication_year=2010&amp;pages=565-569&amp;pmid=20562875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Tishkoff S.A., Reed F.A., Friedlaender F.R., Ehret C., Ranciaro A., Froment A., Hirbo J.B., Awomoyi A.A., Bodo J.-M., Doumbo O. The genetic structure and history of Africans and African Americans. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">324</span>:1035–1044.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947357/">PMC free article</a>]</span> [<a href="/pubmed/19407144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+genetic+structure+and+history+of+Africans+and+African+Americans&amp;author=S.A.+Tishkoff&amp;author=F.A.+Reed&amp;author=F.R.+Friedlaender&amp;author=C.+Ehret&amp;author=A.+Ranciaro&amp;volume=324&amp;publication_year=2009&amp;pages=1035-1044&amp;pmid=19407144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Bryc K., Durand E.Y., Macpherson J.M., Reich D., Mountain J.L. The genetic ancestry of African Americans, Latinos, and European Americans across the United States. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2015;<span class="ref-vol">96</span>:37–53.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4289685/">PMC free article</a>]</span> [<a href="/pubmed/25529636" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=The+genetic+ancestry+of+African+Americans,+Latinos,+and+European+Americans+across+the+United+States&amp;author=K.+Bryc&amp;author=E.Y.+Durand&amp;author=J.M.+Macpherson&amp;author=D.+Reich&amp;author=J.L.+Mountain&amp;volume=96&amp;publication_year=2015&amp;pages=37-53&amp;pmid=25529636&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Habets P.E.M.H., Moorman A.F.M., Clout D.E.W., van Roon M.A., Lingbeek M., van Lohuizen M., Campione M., Christoffels V.M. Cooperative action of Tbx2 and Nkx2.5 inhibits ANF expression in the atrioventricular canal: implications for cardiac chamber formation. <span><span class="ref-journal">Genes Dev. </span>2002;<span class="ref-vol">16</span>:1234–1246.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC186286/">PMC free article</a>]</span> [<a href="/pubmed/12023302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev.&amp;title=Cooperative+action+of+Tbx2+and+Nkx2.5+inhibits+ANF+expression+in+the+atrioventricular+canal:+implications+for+cardiac+chamber+formation&amp;author=P.E.M.H.+Habets&amp;author=A.F.M.+Moorman&amp;author=D.E.W.+Clout&amp;author=M.A.+van+Roon&amp;author=M.+Lingbeek&amp;volume=16&amp;publication_year=2002&amp;pages=1234-1246&amp;pmid=12023302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Harrelson Z., Kelly R.G., Goldin S.N., Gibson-Brown J.J., Bollag R.J., Silver L.M., Papaioannou V.E. Tbx2 is essential for patterning the atrioventricular canal and for morphogenesis of the outflow tract during heart development. <span><span class="ref-journal">Development. </span>2004;<span class="ref-vol">131</span>:5041–5052.</span> [<a href="/pubmed/15459098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Tbx2+is+essential+for+patterning+the+atrioventricular+canal+and+for+morphogenesis+of+the+outflow+tract+during+heart+development&amp;author=Z.+Harrelson&amp;author=R.G.+Kelly&amp;author=S.N.+Goldin&amp;author=J.J.+Gibson-Brown&amp;author=R.J.+Bollag&amp;volume=131&amp;publication_year=2004&amp;pages=5041-5052&amp;pmid=15459098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Ballif B.C., Theisen A., Rosenfeld J.A., Traylor R.N., Gastier-Foster J., Thrush D.L., Astbury C., Bartholomew D., McBride K.L., Pyatt R.E. Identification of a recurrent microdeletion at 17q23.1q23.2 flanked by segmental duplications associated with heart defects and limb abnormalities. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2010;<span class="ref-vol">86</span>:454–461.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2833380/">PMC free article</a>]</span> [<a href="/pubmed/20206336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Identification+of+a+recurrent+microdeletion+at+17q23.1q23.2+flanked+by+segmental+duplications+associated+with+heart+defects+and+limb+abnormalities&amp;author=B.C.+Ballif&amp;author=A.+Theisen&amp;author=J.A.+Rosenfeld&amp;author=R.N.+Traylor&amp;author=J.+Gastier-Foster&amp;volume=86&amp;publication_year=2010&amp;pages=454-461&amp;pmid=20206336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Radio F.C., Bernardini L., Loddo S., Bottillo I., Novelli A., Mingarelli R., Dallapiccola B. TBX2 gene duplication associated with complex heart defect and skeletal malformations. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2010;<span class="ref-vol">152A</span>:2061–2066.</span> [<a href="/pubmed/20635360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A.&amp;title=TBX2+gene+duplication+associated+with+complex+heart+defect+and+skeletal+malformations&amp;author=F.C.+Radio&amp;author=L.+Bernardini&amp;author=S.+Loddo&amp;author=I.+Bottillo&amp;author=A.+Novelli&amp;volume=152A&amp;publication_year=2010&amp;pages=2061-2066&amp;pmid=20635360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Weinzimer S.A., Gibson T.B., Collett-Solberg P.F., Khare A., Liu B., Cohen P. Transferrin is an insulin-like growth factor-binding protein-3 binding protein. <span><span class="ref-journal">J.&nbsp;Clin. Endocrinol. Metab. </span>2001;<span class="ref-vol">86</span>:1806–1813.</span> [<a href="/pubmed/11297622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Clin.+Endocrinol.+Metab.&amp;title=Transferrin+is+an+insulin-like+growth+factor-binding+protein-3+binding+protein&amp;author=S.A.+Weinzimer&amp;author=T.B.+Gibson&amp;author=P.F.+Collett-Solberg&amp;author=A.+Khare&amp;author=B.+Liu&amp;volume=86&amp;publication_year=2001&amp;pages=1806-1813&amp;pmid=11297622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Yakar S., Rosen C.J., Beamer W.G., Ackert-Bicknell C.L., Wu Y., Liu J.-L., Ooi G.T., Setser J., Frystyk J., Boisclair Y.R., LeRoith D. Circulating levels of IGF-1 directly regulate bone growth and density. <span><span class="ref-journal">J.&nbsp;Clin. Invest. </span>2002;<span class="ref-vol">110</span>:771–781.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC151128/">PMC free article</a>]</span> [<a href="/pubmed/12235108" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Clin.+Invest.&amp;title=Circulating+levels+of+IGF-1+directly+regulate+bone+growth+and+density&amp;author=S.+Yakar&amp;author=C.J.+Rosen&amp;author=W.G.+Beamer&amp;author=C.L.+Ackert-Bicknell&amp;author=Y.+Wu&amp;volume=110&amp;publication_year=2002&amp;pages=771-781&amp;pmid=12235108&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Cheng I., DeLellis Henderson K., Haiman C.A., Kolonel L.N., Henderson B.E., Freedman M.L., Le Marchand L. Genetic determinants of circulating insulin-like growth factor (IGF)-I, IGF binding protein (BP)-1, and IGFBP-3 levels in a multiethnic population. <span><span class="ref-journal">J.&nbsp;Clin. Endocrinol. Metab. </span>2007;<span class="ref-vol">92</span>:3660–3666.</span> [<a href="/pubmed/17566087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Clin.+Endocrinol.+Metab.&amp;title=Genetic+determinants+of+circulating+insulin-like+growth+factor+(IGF)-I,+IGF+binding+protein+(BP)-1,+and+IGFBP-3+levels+in+a+multiethnic+population&amp;author=I.+Cheng&amp;author=K.+DeLellis+Henderson&amp;author=C.A.+Haiman&amp;author=L.N.+Kolonel&amp;author=B.E.+Henderson&amp;volume=92&amp;publication_year=2007&amp;pages=3660-3666&amp;pmid=17566087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Agwu J.C., Shaw N.J., Kirk J., Chapman S., Ravine D., Cole T.R. Growth in Sotos syndrome. <span><span class="ref-journal">Arch. Dis. Child. </span>1999;<span class="ref-vol">80</span>:339–342.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1717905/">PMC free article</a>]</span> [<a href="/pubmed/10086939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dis.+Child.&amp;title=Growth+in+Sotos+syndrome&amp;author=J.C.+Agwu&amp;author=N.J.+Shaw&amp;author=J.+Kirk&amp;author=S.+Chapman&amp;author=D.+Ravine&amp;volume=80&amp;publication_year=1999&amp;pages=339-342&amp;pmid=10086939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Aschard H., Vilhjálmsson B.J., Joshi A.D., Price A.L., Kraft P. Adjusting for heritable covariates can bias effect estimates in genome-wide association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2015;<span class="ref-vol">96</span>:329–339.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4320269/">PMC free article</a>]</span> [<a href="/pubmed/25640676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Adjusting+for+heritable+covariates+can+bias+effect+estimates+in+genome-wide+association+studies&amp;author=H.+Aschard&amp;author=B.J.+Vilhjálmsson&amp;author=A.D.+Joshi&amp;author=A.L.+Price&amp;author=P.+Kraft&amp;volume=96&amp;publication_year=2015&amp;pages=329-339&amp;pmid=25640676&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4570286&amp;issue-id=257845&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140711133935616title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Material and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#ack0010title">Acknowledgments</a></li><li><a href="#idm140711132496864title">Notes</a></li><li><a href="#idm140711134080704title">Footnotes</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#idm140711123986256title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4570286/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4570286/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4570286/pdf/main.pdf">PDF (2.6M)</a></li> | <li><a href="#" data-citationid="PMC4570286" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4570286%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4570286%2F&amp;text=Genome-wide%20Analysis%20of%20Body%20Proportion%20Classifies%20Height-Associated%20Variants%20by%20Mechanism%20of%20Action%20and%20Implicates%20Genes%20Important%20for%20Skeletal%20Development" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4570286%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25865494" data-aiid="4570286" data-aid="4570286" data-iid="257845" data-domainid="203" data-domain="ajhg" data-accid="PMC4570286" data-md5="f596af73ad9425ffa9143e119eb58082"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:33:03-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6880DD578921000000000952058E&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4570286%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4570286/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>