<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4106235/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4106235/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity"><meta name="citation_authors" content="Eleanor Wheeler, Ni Huang, Elena Bochukova, Julia M. Keogh, Sarah Lindsay, Sumedha Garg, Elana Henning, Hannah Blackburn, Ruth J.F. Loos, Nick J Wareham, Stephen O’Rahilly, Matthew E. Hurles, Inês Barroso, I. Sadaf Farooqi"><meta name="citation_date" content="May 2013"><meta name="citation_issue" content="5"><meta name="citation_volume" content="45"><meta name="citation_firstpage" content="513"><meta name="citation_doi" content="10.1038/ng.2607"><meta name="citation_pmid" content="23563609"><meta name="DC.Title" content="Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Eleanor Wheeler"><meta name="DC.Contributor" content="Ni Huang"><meta name="DC.Contributor" content="Elena Bochukova"><meta name="DC.Contributor" content="Julia M. Keogh"><meta name="DC.Contributor" content="Sarah Lindsay"><meta name="DC.Contributor" content="Sumedha Garg"><meta name="DC.Contributor" content="Elana Henning"><meta name="DC.Contributor" content="Hannah Blackburn"><meta name="DC.Contributor" content="Ruth J.F. Loos"><meta name="DC.Contributor" content="Nick J Wareham"><meta name="DC.Contributor" content="Stephen O’Rahilly"><meta name="DC.Contributor" content="Matthew E. Hurles"><meta name="DC.Contributor" content="Inês Barroso"><meta name="DC.Contributor" content="I. Sadaf Farooqi"><meta name="DC.Date" content="2013 May"><meta name="DC.Identifier" content="10.1038/ng.2607"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity"><meta property="og:type" content="article"><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4106235/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4106235/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4106235/pdf/nihms-612416.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D26AADD5589710000000005030310.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4106235%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4106235</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2014 Jul 22.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23563609" target="pmc_ext" ref="reftype=publisher&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2013 May; 45(5): 513–517. </span></a></div><span class="fm-vol-iss-date">Published online 2013 Apr 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.2607" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.2607</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4106235</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS612416</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/23563609">23563609</a></div></div></div></div><h1 class="content-title">Genome-wide SNP and CNV analysis identifies common and low-frequency variants associated with severe early-onset obesity</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Wheeler%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341535967584" co-class="co-affbox">Eleanor Wheeler</a>,<sup>1</sup> <a href="/pubmed/?term=Huang%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341538336352" co-class="co-affbox">Ni Huang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Bochukova%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341576758144" co-class="co-affbox">Elena Bochukova</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Keogh%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341576755424" co-class="co-affbox">Julia M. Keogh</a>,<sup>2</sup> <a href="/pubmed/?term=Lindsay%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341543501648" co-class="co-affbox">Sarah Lindsay</a>,<sup>1</sup> <a href="/pubmed/?term=Garg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341543499664" co-class="co-affbox">Sumedha Garg</a>,<sup>2</sup> <a href="/pubmed/?term=Henning%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341543497680" co-class="co-affbox">Elana Henning</a>,<sup>2</sup> <a href="/pubmed/?term=Blackburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341541538496" co-class="co-affbox">Hannah Blackburn</a>,<sup>1</sup> <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341541536512" co-class="co-affbox">Ruth J.F. Loos</a>,<sup>3</sup> <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341532313104" co-class="co-affbox">Nick J Wareham</a>,<sup>3</sup> <a href="/pubmed/?term=O%26%23x02019%3BRahilly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341532311120" co-class="co-affbox">Stephen O’Rahilly</a>,<sup>2</sup> <a href="/pubmed/?term=Hurles%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341532309136" co-class="co-affbox">Matthew E. Hurles</a>,<sup>1</sup> <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341614600192" co-class="co-affbox">Inês Barroso</a>,<sup>1,</sup><sup>2</sup> and  <a href="/pubmed/?term=Farooqi%20IS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609" class="affpopup" co-rid="_co_idm140341614597472" co-class="co-affbox">I. Sadaf Farooqi</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140341535967584"><h3 class="no_margin">Eleanor Wheeler</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Wheeler%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Eleanor Wheeler</a></div></div><div id="_co_idm140341538336352"><h3 class="no_margin">Ni Huang</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Huang%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Ni Huang</a></div></div><div id="_co_idm140341576758144"><h3 class="no_margin">Elena Bochukova</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Bochukova%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Elena Bochukova</a></div></div><div id="_co_idm140341576755424"><h3 class="no_margin">Julia M. Keogh</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Keogh%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Julia M. Keogh</a></div></div><div id="_co_idm140341543501648"><h3 class="no_margin">Sarah Lindsay</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Lindsay%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Sarah Lindsay</a></div></div><div id="_co_idm140341543499664"><h3 class="no_margin">Sumedha Garg</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Garg%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Sumedha Garg</a></div></div><div id="_co_idm140341543497680"><h3 class="no_margin">Elana Henning</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Henning%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Elana Henning</a></div></div><div id="_co_idm140341541538496"><h3 class="no_margin">Hannah Blackburn</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Blackburn%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Hannah Blackburn</a></div></div><div id="_co_idm140341541536512"><h3 class="no_margin">Ruth J.F. Loos</h3><p><sup>3</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Loos%20RJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Ruth J.F. Loos</a></div></div><div id="_co_idm140341532313104"><h3 class="no_margin">Nick J Wareham</h3><p><sup>3</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Wareham%20NJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Nick J Wareham</a></div></div><div id="_co_idm140341532311120"><h3 class="no_margin">Stephen O’Rahilly</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BRahilly%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Stephen O’Rahilly</a></div></div><div id="_co_idm140341532309136"><h3 class="no_margin">Matthew E. Hurles</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Hurles%20ME%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Matthew E. Hurles</a></div></div><div id="_co_idm140341614600192"><h3 class="no_margin">Inês Barroso</h3><p><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</p><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Barroso%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">Inês Barroso</a></div></div><div id="_co_idm140341614597472"><h3 class="no_margin">I. Sadaf Farooqi</h3><p><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</p><div>Find articles by <a href="/pubmed/?term=Farooqi%20IS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=23563609">I. Sadaf Farooqi</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140341538029776_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140341538029776_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140341538029776_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Wellcome Trust Sanger Institute, Cambridge, UK</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>University of Cambridge Metabolic Research Laboratories, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>MRC Epidemiology Unit, Institute of Metabolic Science, Addenbrooke's Hospital, Cambridge, UK</div><div id="CR1">Correspondence should be addressed to: I. Sadaf Farooqi (<span class="underline"><a href="mailto:dev@null" data-email="ku.ca.mac@02fsi" class="oemail">ku.ca.mac@02fsi</a></span>) and Inês Barroso (<span class="underline"><a href="mailto:dev@null" data-email="ku.ca.regnas@1bi" class="oemail">ku.ca.regnas@1bi</a></span>).</div><div id="FN1"><sup>*</sup>These authors contributed equally.</div><div id="FN2"><strong>AUTHOR CONTRIBUTIONS</strong><p id="P3" class="p p-last">I.S.F., I.B., M.E.H., E.W., N.H. and E.G.B. designed the study, analysed the data and wrote the paper. All the authors contributed to data collection and/or processing, as well as the current version of the paper. The authors have no competing financial interests.</p></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140341538029776_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=23563609" target="pmc_ext" ref="reftype=publisher&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4106235/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary Material.</strong></div><div><a href="/pmc/articles/PMC4106235/bin/NIHMS612416-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">NIHMS612416-supplement-Supplementary_Material.pdf</a><span style="color:gray"> (1.3M)</span></div><div class="small guid">GUID:&nbsp;998EF434-E9D9-4C26-B32A-403B8258C2D7</div></div></dd></dl></div><div id="idm140341533886848" class="tsec sec headless whole_rhythm"><p id="P4" class="p p-first">Common and rare variants associated with body mass index (BMI) and obesity account for &lt;5% of the variance in BMI. We performed SNP and CNV association analyses in 1,509 children with obesity at the extreme tail (&gt;3 standard deviations) of the BMI distribution and 5380 controls. Evaluation of 29 SNPs (<em>p</em>&lt;10<sup>−5</sup>) in an additional 971 severely obese children and 1990 controls identified four new loci associated with severe obesity (<em>LEPR</em>, <em>PRKCH</em>, <em>PACS1</em>, <em>RMST</em>). A previously reported 43kb deletion at the <em>NEGR1</em> locus was significantly associated with severe obesity (<em>p</em>=6.6×10<sup>−7</sup>). However, this signal was entirely driven by a flanking 8kb deletion; absence of this deletion increased risk for obesity (<em>p</em>=6.1×10<sup>−11</sup>). We found a significant burden of rare, single CNVs in severely obese cases (<em>p</em>&lt;0.0001). Integrative gene network pathway analysis of rare deletions indicated enrichment of genes affecting GPCRs involved in the neuronal regulation of energy homeostasis.</p><p id="P5">Within a population that shares the same environment, a significant proportion of the variance in body mass index (BMI) is genetically determined<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655856">1</a></sup>. Meta-analyses of genome-wide association studies (GWAS) for obesity-related traits (including BMI, waist-hip ratio and obesity) have led to the discovery of at least 52 loci to date<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655857">2</a></sup>. However, the fraction of BMI variance that can be attributed to common SNPs is at the most 2%<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655864">2</a></sup>. Candidate gene studies in severe obesity have led to the identification of several rare, highly penetrant mutations involving the hypothalamic leptin-melanocortin pathway, with variants in the melanocortin 4 receptor (<em>MC4R</em>) being the most common, with a population prevalence of 1/1000<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655868">1</a></sup>. Although epistatic and gene-environment interactions may contribute to the unexplained heritability of obesity, it seems likely that a significant fraction is due to missing loci or established loci that have not yet been fully characterised<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655867">3</a></sup>.</p><p id="P6">As studying individuals at the extremes of a quantitative phenotype distribution has established utility<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655846">4</a></sup> we performed a genome-wide association study (GWAS) in 1509 UK Caucasian children with severe early onset obesity (SCOOP-UK) and 5380 WTCCC2 UK controls<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655854">5</a></sup> to identify novel associations of common and low-frequency SNPs and CNVs with obesity (<strong>Methods</strong>). Analysis of ~2 Million directly genotyped or imputed SNPs, under a log-additive model, revealed 29 loci with evidence of association with obesity (<em>p</em>&lt;1×10<sup>−5</sup>) (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a></strong>). These 29 SNPs were taken forward for validation in additional 971 severely obese children and 1990 controls (<strong>Methods</strong>). Joint analysis of discovery and validation stages, including up to 2480 cases and 7370 controls identified nine genome-wide significant signals in eight loci (<em>p</em>&lt;5×10<sup>−8</sup>; <strong><a href="/pmc/articles/PMC4106235/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></strong>; <strong><a href="/pmc/articles/PMC4106235/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a></strong>). Three mapped in or near <em>PRKCH</em>, <em>PACS1</em> and <em>RMST</em> and have not been previously associated with obesity <strong>(<a href="/pmc/articles/PMC4106235/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> and <a href="/pmc/articles/PMC4106235/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>)</strong>. These genes are ubiquitously expressed (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a></strong>) and to date their role in energy homeostasis is unknown. A significant finding was that of a novel association with an intermediate frequency allele (rs11208659, 6% MAF in CEU HapMap) in the gene encoding the leptin receptor (<em>LEPR)</em>. Homozygous loss of function <em>LEPR</em> variants can lead to monogenic obesity<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655843">1</a></sup>. To test whether this signal was driven by rare causal mutations, we sequenced all cases carrying at least one predisposing (C) allele on whom DNA was available (286/295); none had homozygous <em>LEPR</em> mutations and the three rare non-synonymous changes identified are unlikely to be sufficient to drive the association signal (data not shown), arguing against the <em>LEPR</em> signal being driven by causal mutations and suggests that a more common <em>LEPR</em> allele can influence predisposition to severe obesity. An analysis collapsing directly genotyped low-frequency SNPs (MAF &lt;5%) across the discovery sample set did not find a statistically significant accumulation of rare SNPs in any loci apart from <em>LEPR</em>. However, the rare variant association signal at the LEPR locus was driven by the lead SNP (rs11208659); adjusting for this SNP abolished the association signal (data not shown). We also found that the risk allele (C) at the <em>LEPR</em> locus is associated with lower LEPR expression in monocytes<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655863">6</a></sup> (<em>p</em> = 0.0321, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a></strong>). Given the established importance of LEPR-mediated signalling in energy homeostasis, this might suggest that the association with obesity may be mediated by a reduction in levels of LEPR expression.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4106235/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341537310416" class="figure"><a href="/pmc/articles/PMC4106235/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4106235_nihms-612416-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-612416-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4106235/bin/nihms-612416-f0001.jpg"></a></div><div id="largeobj_idm140341537310416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4106235/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4106235/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Genome wide SNP association study in severe early onset obesity. Manhattan plot showing the significance (−log<sub>10</sub>(<em>p</em>)) of all SNPs in the discovery analysis. SNPs taken for replication (circled), established loci (dark grey) and novel loci (green) are shown. Horizontal dashed line indicates <em>p</em>=1×10<sup>−6</sup>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4106235/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341535936704" class="figure"><a href="/pmc/articles/PMC4106235/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4106235_nihms-612416-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-612416-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4106235/bin/nihms-612416-f0002.jpg"></a></div><div id="largeobj_idm140341535936704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4106235/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4106235/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Regional association plots of the novel loci generated using LocusZoom<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655859">23</a></sup>.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">SNPs associated with severe, early-onset obesity at genome-wide significance levels</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341539675952" class="xtable"><table frame="void" rules="none" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="4" rowspan="1" valign="middle" align="center">Discovery <br>1509 cases, 5380 controls</th><th colspan="4" rowspan="1" valign="middle" align="center">Follow-up <br>971 cases, 1990 controls</th><th colspan="2" rowspan="1" valign="middle" align="center">Combined <br>2480 cases, 7370 controls</th></tr><tr><th colspan="17" rowspan="1" valign="middle" align="center">
<hr>
</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="center">SNP</th><th rowspan="1" colspan="1" valign="middle" align="left">Nearest gene</th><th rowspan="1" colspan="1" valign="middle" align="center">Novel</th><th rowspan="1" colspan="1" valign="middle" align="center">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="center">Position <br>(bp)<sup><sup>a</sup></sup></th><th rowspan="1" colspan="1" valign="middle" align="center">Effect <br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">Other <br>allele</th><th rowspan="1" colspan="1" valign="middle" align="center">OR <br>(95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P-value</th><th rowspan="1" colspan="1" valign="middle" align="center">EAF <br>cases</th><th rowspan="1" colspan="1" valign="middle" align="center">EAF <br>controls</th><th rowspan="1" colspan="1" valign="middle" align="center">OR <br>(95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P-value</th><th rowspan="1" colspan="1" valign="middle" align="center">EAF <br>cases</th><th rowspan="1" colspan="1" valign="middle" align="center">EAF <br>controls</th><th rowspan="1" colspan="1" valign="middle" align="center">OR <br>(95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center">P-value</th></tr><tr><th colspan="17" rowspan="1" valign="middle" align="center">
<hr>
</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1421085</td><td rowspan="1" colspan="1" valign="middle" align="left">FTO</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">16</td><td rowspan="1" colspan="1" valign="middle" align="center">52358455</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.49 <br>(1.37 - 1.61)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.38×10<sup>−22</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.5</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.36 <br>(1.21 - 1.51)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.66×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.48</td><td rowspan="1" colspan="1" valign="middle" align="center">0.4</td><td rowspan="1" colspan="1" valign="middle" align="center">1.44 <br>(1.35 - 1.54)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>3.07×10<sup>−28</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs476828<sup><sup>b</sup></sup></td><td rowspan="1" colspan="1" valign="middle" align="left">MC4R</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">18</td><td rowspan="1" colspan="1" valign="middle" align="center">56003567</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33 <br>(1.21 - 1.46)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.91×10<sup>−09</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.29</td><td rowspan="1" colspan="1" valign="middle" align="center">0.24</td><td rowspan="1" colspan="1" valign="middle" align="center">1.32 <br>(1.16 - 1.50)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.64×10<sup>−05</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.27</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33 <br>(1.23 - 1.43)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>9.38×10<sup>−14</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs12463617</td><td rowspan="1" colspan="1" valign="middle" align="left">TMEM18</td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">2</td><td rowspan="1" colspan="1" valign="middle" align="center">619244</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.43 <br>(1.27 - 1.60)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.70×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.89</td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="center">1.4 <br>(1.19 - 1.65)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.80×10<sup>−05</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="center">0.83</td><td rowspan="1" colspan="1" valign="middle" align="center">1.42 <br>(1.29 - 1.56)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>1.70×10<sup>−13</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1993709</td><td rowspan="1" colspan="1" valign="middle" align="left">NEGR1 <em>(8kb</em>
<br><em>deletion)</em></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">72611117</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.46 <br>(1.32 - 1.63)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.98×10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.87</td><td rowspan="1" colspan="1" valign="middle" align="center">0.81</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 <br>(1.05 - 1.41)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.011</td><td rowspan="1" colspan="1" valign="middle" align="center">0.85</td><td rowspan="1" colspan="1" valign="middle" align="center">0.82</td><td rowspan="1" colspan="1" valign="middle" align="center">1.38 <br>(1.26 - 1.50)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>5.09×10<sup>−13</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs1957894</td><td rowspan="1" colspan="1" valign="middle" align="left">PRKCH</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">14</td><td rowspan="1" colspan="1" valign="middle" align="center">60977864</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.64 <br>(1.39 - 1.95)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">0.06</td><td rowspan="1" colspan="1" valign="middle" align="center">1.34 <br>(1.11 - 1.61)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.002</td><td rowspan="1" colspan="1" valign="middle" align="center">0.1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5 <br>(1.32 - 1.70)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>3.35×10<sup>−10</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs11208659</td><td rowspan="1" colspan="1" valign="middle" align="left">LEPR</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">65751868</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.63 <br>(1.40 - 1.88)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.16×10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.11</td><td rowspan="1" colspan="1" valign="middle" align="center">0.08</td><td rowspan="1" colspan="1" valign="middle" align="center">1.15 <br>(0.96 - 1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.118</td><td rowspan="1" colspan="1" valign="middle" align="center">0.1</td><td rowspan="1" colspan="1" valign="middle" align="center">0.09</td><td rowspan="1" colspan="1" valign="middle" align="center">1.42 <br>(1.27 - 1.59)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>1.84×10<sup>−09</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs564343</td><td rowspan="1" colspan="1" valign="middle" align="left">PACS1</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">11</td><td rowspan="1" colspan="1" valign="middle" align="center">65651742</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25 <br>(1.16 - 1.36)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.81×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.47</td><td rowspan="1" colspan="1" valign="middle" align="center">0.41</td><td rowspan="1" colspan="1" valign="middle" align="center">1.17 <br>(1.05 - 1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.006</td><td rowspan="1" colspan="1" valign="middle" align="center">0.44</td><td rowspan="1" colspan="1" valign="middle" align="center">0.4</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 <br>(1.15 - 1.31)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>2.03×10<sup>−09</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs11109072</td><td rowspan="1" colspan="1" valign="middle" align="left">RMST</td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">12</td><td rowspan="1" colspan="1" valign="middle" align="center">96425401</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.79 <br>(1.44 - 2.22)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.48×10<sup>−07</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td><td rowspan="1" colspan="1" valign="middle" align="center">0.03</td><td rowspan="1" colspan="1" valign="middle" align="center">1.5 <br>(1.13 - 1.99)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.004</td><td rowspan="1" colspan="1" valign="middle" align="center">0.04</td><td rowspan="1" colspan="1" valign="middle" align="center">0.03</td><td rowspan="1" colspan="1" valign="middle" align="center">1.67 <br>(1.41 - 1.99)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>4.21×10<sup>−09</sup></strong>
</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="center">rs3101336</td><td rowspan="1" colspan="1" valign="middle" align="left">NEGRI (<em>43kb</em>
<br><em>deletion)</em></td><td rowspan="1" colspan="1" valign="middle" align="center"></td><td rowspan="1" colspan="1" valign="middle" align="center">1</td><td rowspan="1" colspan="1" valign="middle" align="center">72523773</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26 <br>(1.16 - 1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.28×10<sup>−08</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.66</td><td rowspan="1" colspan="1" valign="middle" align="center">0.6</td><td rowspan="1" colspan="1" valign="middle" align="center">1.12 <br>(1.00 - 1.25)</td><td rowspan="1" colspan="1" valign="middle" align="center">0.047</td><td rowspan="1" colspan="1" valign="middle" align="center">0.64</td><td rowspan="1" colspan="1" valign="middle" align="center">0.61</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21 <br>(1.13 - 1.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">
<strong>2.21×10<sup>−08</sup></strong>
</td></tr></tbody></table></div><div id="largeobj_idm140341539675952" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4106235/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p7">Genome-wide loci for association with severe, early-onset obesity aligned to the risk-increasing (effect) allele. SNPs rs12463617, rs1993709, rs11208659, rs11109072 and rs3101336 were imputed. OR = Odds ratio; 95% CI = 95% confidence interval for the odds ratio; EAF = effect allele frequency.</p><div id="TFN1"><sup>a</sup>Positions according to Build 36.</div><div id="TFN2"><sup>b</sup>rs2168711 used as a proxy in the follow-up stage.</div></div></div><p id="P7">The remaining four obesity associated loci (in or near, <em>FTO</em>, <em>MC4R</em>, <em>TMEM18</em> and <em>NEGR1</em>) have been reported previously<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655844">2</a></sup>. As our cases lie at the very extreme end of the BMI distribution, there are very few other cohorts of sufficient size and comparable severity of obesity for replication. Consistent with this, on comparing our genome-wide significant results to other GWAS for childhood obesity<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655851">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655850">8</a></sup> and for varying degrees of obesity in adults and children<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655858">9</a>-<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655853">16</a></sup>
<strong>(<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>),</strong> we found there was relatively little overlap, other than for <em>FTO</em>, <em>TMEM18</em>
<em>and</em>
<em>MC4R</em>. To explore the differences between our results and those from large-scale meta-analysis of BMI in population-based cohorts, we compared our results with those obtained from the GIANT consortium (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 3</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">4</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 3</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">4</a>). For several GIANT loci we did not obtain genome-wide significant support for association in our cohort with severe early onset obesity although 14 loci had directionally consistent associations at nominal significance levels (p&lt;0.05), more than expected by chance p<sub>binomial</sub>= 6.10×10<sup>−5</sup> (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a></strong>, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a>.</strong> Some of these differences are accounted for by differences in power as our sample size is much smaller than that of GIANT; power calculations showed that we had &lt; 80% power to detect signals other than <em>FTO</em>, <em>TMEM18</em> and <em>MC4R</em> at genome-wide levels of significance (<strong>Methods</strong>). This observation may also reflect differences in susceptibility to early versus adult onset obesity or in alleles associated with BMI in the obese vs severely obese range <strong>(<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>).</strong> Indeed the data suggest that while there is significant overlap between the loci influencing BMI and common obesity and those influencing severe obesity, this overlap is incomplete and the relative contribution of each locus to common and severe forms of obesity also differs. For example, while <em>FTO</em> is the locus with the largest reported effect size in several population studies, our results show that there are other loci with comparable or greater effect sizes on severe early onset obesity (<strong><a href="/pmc/articles/PMC4106235/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></strong>). Also, while <em>NEGR1</em> (tagging the 40kb deletion) and <em>SH2B1</em>, both have similar effect sizes in GIANT, the effect of <em>SH2B1</em> is much smaller in SCOOP (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 4</a></strong>). Construction of a risk score with the SNPs from GIANT<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655840">17</a></sup> available in our data also suggests that the severely obese cases seen in SCOOP do not have an increased burden of established BMI loci (mean risk score = 27.2), suggesting they may be due to different risk alleles. Similarly, data from GIANT do not support the association of the new SCOOP loci with BMI in population-derived cohorts (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a></strong>). However, in the recent GIANT discovery sample (N=123,865), only 167 individuals would have a comparable BMI to SCOOP cases, so the lack of replication does not preclude an effect of these variants on severe obesity.</p><p id="P8">We analysed the SNP data to examine whether there was an excess in cases of homozygosity by descent (HBD), either genome-wide, or in specific genomic locations (<strong>Methods</strong>). We did not observe any significant genome-wide homozygosity burden in cases (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 5</a></strong>). When the distribution of regions of HBD across individual genes was considered, there were a number of overlapping regions of HBD in cases, across genes in regions of chromosomes 2, 8, 10, with nominal <em>p</em>&lt;0.01, although these did not survive correction for multiple testing (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 5</a>)</strong>. These findings suggest that individually, and cumulatively, homozygously-acting genes are unlikely to play a major role in the etiology of severe obesity in our outbred UK cohort.</p><p id="P9">We assessed the role of common and rare CNVs in severe early onset obesity. Our strongest association signal (<em>p</em> = 6.1×10<sup>−11</sup>), also supported by the SNP data, was of a protective ~8kb deletion upstream of <em>NEGR1</em>, with a weaker positive association signal (<em>p</em>=6.6×10<sup>−7</sup>) from a larger non-overlapping ~43kb deletion (72,541-72,584 kb) flanking the ~8kb deletion (<strong><a href="/pmc/articles/PMC4106235/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a></strong>). These deletions are known to segregate on distinct haplotypes, and as the three alleles (43kb deletion, 8kb deletion and undeleted) are mutually exclusive, we tested whether the two deletion alleles were independently associated with severe obesity. The frequency of the undeleted allele is approximately the same in cases and controls and is not associated with the phenotype (OR (95% CI) =1 (0.90-1.1); <em>p</em> = 0.93, two-sided Fisher’s exact test). When conditioned on the smaller deletion allele, the association of the larger deletion allele was completely abolished (OR (95% CI) = (0.97-1.22); <em>p</em>=0.16). However, when conditioned on the larger allele, the association of the smaller deletion remained significant (OR (95% CI) =0.70 (0.60-0.82); <em>p</em>=6.93×10<sup>−6</sup>), suggesting that the association signal at the <em>NEGR1</em> locus in our cohort is largely driven by the protective effect of the ~8kb deletion allele. In keeping with these findings, we found that the SNP with the strongest association signal for obesity at this locus (rs1993709) tags the 8Kb deletion (<strong><a href="/pmc/articles/PMC4106235/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a></strong>) and conditional analyses performed at the SNP level are consistent with these findings (data not shown), highlighting the strength of the combined SNP and CNV analysis.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4106235/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341535932416" class="figure"><a href="/pmc/articles/PMC4106235/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4106235_nihms-612416-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-612416-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4106235/bin/nihms-612416-f0003.jpg"></a></div><div id="largeobj_idm140341535932416" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4106235/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4106235/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">A quantile-quantile plot of –log<sub>10</sub>(p) of 481 common CNVs. Concentration band represents 95% confidence intervals. The data generally conform to the –log<sub>10</sub> transformed uniform distribution expected under the null hypothesis of no association, with the exception of strong associations with the two deletions upstream of <em>NEGR1</em>.</p></div></div></div><p id="P10">Although the 8kb deletion does not disrupt the coding sequence of any gene, it encompasses a single conserved transcription factor binding site for NKX6.1<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655847">18</a></sup> (<strong><a href="/pmc/articles/PMC4106235/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4a</span></a> and <a href="/pmc/articles/PMC4106235/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4b</span></span><span>4b</span></a></strong>), which is known to be involved in neuronal development in the mid and hindbrain. We used Electrophoretic Mobility Shift Assay (EMSA) to confirm binding of NKX6.1 to its predicted binding site within the 8kb deletion (<strong><a href="/pmc/articles/PMC4106235/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4c</span></a></strong>). NKX6.1 is a potent transcriptional repressor, so loss of binding would be predicted to lead to increased gene expression. In existing eQTL data, the small deletion appears to be associated with increased expression of <em>NEGR1</em> in lymphoblastoid cell lines (<em>p</em>=0.0427, <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 6</a></strong>). Further studies will be needed to establish whether <em>NEGR1</em> is transcriptionally regulated by NKX6.1 in relevant human tissues. As NEGR1 is involved in neurite outgrowth<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655842">19</a></sup>, a process known to be involved in human obesity<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655862">1</a></sup>, these observations could form the basis for a molecular explanation for this association, which needs to be formally tested.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC4106235/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341539678528" class="figure"><a href="/pmc/articles/PMC4106235/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4106235_nihms-612416-f0004.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms-612416-f0004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4106235/bin/nihms-612416-f0004.jpg"></a></div><div id="largeobj_idm140341539678528" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4106235/figure/F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC4106235/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p5">NEGR1 locus on chromosome 1p31.1<strong>. a.</strong> Schematic of the locus with the positions of the two deletions (8-kb and 43-kb) and neighbouring genes. <strong>b.</strong> Sequence conservation of the small 8-kb deletion sequence. Nucleotide Percent Identity Plot (PIP) for the region is shown. Orthologous sequences from multiple species were aligned to the human sequence using MultiPIP maker, and sequence identity &gt;50% (light grey) and &gt;75% (dark grey) is highlighted, with the details shown below. The predicted NKX6.1 binding site is located in the most conserved region (arrow). <strong>c.</strong> Transcription factor NKX6.1 binding was confirmed using EMSA assay, electromobility shift (arrow) was specific to the sequence contained in the 8-kb deletion. n/s (non-specific band).</p></div></div></div><p id="P11">We previously demonstrated a significant burden of large rare deletions in patients with severe-early onset obesity and developmental delay<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655852">20</a></sup>. Here we used a permutation-based method to assess the statistical significance of genome-wide CNV burden <strong>(Methods)</strong>. A significant enrichment in cases was observed for all CNVs &gt;100kb in size and &lt; 1% in frequency (<strong><a href="/pmc/articles/PMC4106235/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>)</strong>. The most significant enrichment was observed in duplications in the range of 100-200kb and deletions &gt;100kb in size. The enrichment of singleton CNVs is stronger than that of rare recurrent CNVs (1.8-fold increase, <em>p</em>=0.0001). We focused on the large number of obesity-specific deletions that were not detected in controls (765).-Whilst CNVs may contribute to disease by affecting regulatory regions of the genome, given the large number of rare CNVs identified in this study, we focused on CNVs that disrupt the exon sequence of 490 genes (in 336 patients) where we can more directly infer pathogenicity due to dosage sensitivity. To examine whether these 490 genes converged on specific biological pathways, we conducted pathway enrichment analysis using Ingenuity Pathways Analysis (IPA); genes involved in nervous system development and/or function were significantly overrepresented (<em>p</em>=9.83×10 <sup>−5</sup>) (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 6</a></strong>). The most significant enrichment involved a network of 13 genes encoding G-protein coupled receptors (GPCRs) (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 7</a></strong>); these CNVs were validated by an independent method, MLPA (data not shown). Several of the genes disrupted by these CNVs play an important role in energy homeostasis in animals, the mu-opioid receptor 1 (OPRM1), the follicle-stimulating hormone receptor (FSHR) and the oxytocin receptor (OXTR). In addition, genetic variation in the dopamine receptor D2 (DRD2) has been associated with eating behaviour and brain activation responses to images of food in humans<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655866">21</a></sup>. We independently analysed the same dataset using another method which corrects for CNV size<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655849">22</a></sup> and replicated the observation that rare CNVs deleting genes encoding GPCRs were significantly associated with severe obesity (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a></strong>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Global CNV burden analysis of rare CNVs &gt;100kb.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140341535144576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Type</th><th rowspan="1" colspan="1" valign="bottom" align="left">Frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center">Case rate</th><th rowspan="1" colspan="1" valign="bottom" align="center">Control <br>rate</th><th rowspan="1" colspan="1" valign="bottom" align="center">Case/control <br>ratio</th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sub><em>MAD</em></sub></th><th rowspan="1" colspan="1" valign="bottom" align="center">P<sub><em>NCPS</em></sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">All &lt; 1%</td><td rowspan="1" colspan="1" valign="top" align="center">1.7439</td><td rowspan="1" colspan="1" valign="top" align="center">1.5271</td><td rowspan="1" colspan="1" valign="top" align="center">1.1419</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Losses and gains</td><td rowspan="1" colspan="1" valign="top" align="left">Single occurrence</td><td rowspan="1" colspan="1" valign="top" align="center">0.4002</td><td rowspan="1" colspan="1" valign="top" align="center">0.2849</td><td rowspan="1" colspan="1" valign="top" align="center">1.4048</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Recurrent &lt; 0.1%</td><td rowspan="1" colspan="1" valign="top" align="center">0.4627</td><td rowspan="1" colspan="1" valign="top" align="center">0.3968</td><td rowspan="1" colspan="1" valign="top" align="center">1.1659</td><td rowspan="1" colspan="1" valign="top" align="center">1.7×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2×10<sup>−4</sup></td></tr><tr><td colspan="7" rowspan="1" valign="top" align="left">
<hr>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">All &lt; 1%</td><td rowspan="1" colspan="1" valign="top" align="center">0.5972</td><td rowspan="1" colspan="1" valign="top" align="center">0.5438</td><td rowspan="1" colspan="1" valign="top" align="center">1.0983</td><td rowspan="1" colspan="1" valign="top" align="center">1.23×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">3.5×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Losses</td><td rowspan="1" colspan="1" valign="top" align="left">Single occurrence</td><td rowspan="1" colspan="1" valign="top" align="center">0.1259</td><td rowspan="1" colspan="1" valign="top" align="center">0.1001</td><td rowspan="1" colspan="1" valign="top" align="center">1.2574</td><td rowspan="1" colspan="1" valign="top" align="center">7.2×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">7.0×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Recurrent &lt; 0.1%</td><td rowspan="1" colspan="1" valign="top" align="center">0.1710</td><td rowspan="1" colspan="1" valign="top" align="center">0.1415</td><td rowspan="1" colspan="1" valign="top" align="center">1.2085</td><td rowspan="1" colspan="1" valign="top" align="center">9.4×10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.3×10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">All &lt; 1%</td><td rowspan="1" colspan="1" valign="top" align="center">1.1467</td><td rowspan="1" colspan="1" valign="top" align="center">0.9834</td><td rowspan="1" colspan="1" valign="top" align="center">1.1661</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Gains</td><td rowspan="1" colspan="1" valign="top" align="left">Single occurrence</td><td rowspan="1" colspan="1" valign="top" align="center">0.2743</td><td rowspan="1" colspan="1" valign="top" align="center">0.1848</td><td rowspan="1" colspan="1" valign="top" align="center">1.4846</td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">&lt;1×10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Recurrent &lt; 0.1%</td><td rowspan="1" colspan="1" valign="top" align="center">0.2917</td><td rowspan="1" colspan="1" valign="top" align="center">0.2553</td><td rowspan="1" colspan="1" valign="top" align="center">1.1423</td><td rowspan="1" colspan="1" valign="top" align="center">3.17×10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">8.0×10<sup>−3</sup></td></tr></tbody></table></div><div id="largeobj_idm140341535144576" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4106235/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><p id="__p9">P<sub>MAD</sub>, P<sub>NCPS</sub>: P values derived from permutation conditioned on MAD of samples log2 ratio or number of calls per sample</p></div></div><p id="P12" class="p">In conclusion, by combining SNP and CNV analysis and focussing on severe obesity, we have identified four novel obesity susceptibility loci, including an intermediate frequency variant in <em>LEPR</em>. These findings add to the body of evidence suggesting that both common and rare variants around specific genes/loci (<em>LEPR,</em>
<em>POMC,</em>
<em>MC4R,</em>
<em>BDNF,</em>
<em>SH2B1</em>) are involved in the pathogenesis of obesity. We show that there is an incomplete overlap between loci influencing risk of severe obesity and those that influence more common obesity, as detected by studies in population-based cohorts, and that the relative contribution of each locus to severe versus common obesity also differs. Furthermore, we provide evidence that severe obesity without developmental delay is associated with a significantly increased burden of rare, typically singleton CNVs, in parallel with findings in intellectual disability. Using pathway analysis, we found that rare CNVs that delete genes involved in the neuronal regulation of energy homeostasis contribute to severe obesity; looking for rare coding variants in these genes may be fruitful. As we observed a significant enrichment for CNVs that deleted GPCRs, which are key targets for drug development, these findings may have potential therapeutic implications.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">URLs</h2><p id="P13" class="p p-first-last">Wellcome Trust Case-Control Consortium , <span class="underline"><a href="http://www.wtccc.org.uk/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk/</a></span>; Wellcome Trust Case Control Consortium 2, <span class="underline"><a href="http://www.wtccc.org.uk/ccc2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk/ccc2/</a></span>; Stata 11, <span class="underline"><a href="http://www.stata.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stata.com/</a></span>; Quanto v 1.2.3, <span class="underline"><a href="http://hydra.usc.edu/gxe/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://hydra.usc.edu/gxe/</a></span>; CCRaVAT, Case-Control Rare Variant Analysis Tool, <a href="http://www.sanger.ac.uk/resources/software/rarevariant/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sanger.ac.uk/resources/software/rarevariant/</a>; Database for Annotation, Visualization and Integrated Discovery (DAVID), <span class="underline"><a href="http://david.abcc.ncifcrf.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://david.abcc.ncifcrf.gov/</a></span>; PANTHER classification system, <span class="underline"><a href="http://www.pantherdb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.pantherdb.org/</a></span>; Ingenuity Pathway Analysis (IPA, version 9.0, Ingenuity Systems), <span class="underline"><a href="http://www.ingenuity.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com/</a></span>.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Online Methods</h2><div id="S3" class="sec sec-first-last"><h3 id="S3title">Cohort information</h3><div id="S4" class="sec sec-first"><p></p><h4 id="S4title" class="inline">The Severe Childhood Onset Obesity Project (SCOOP) </h4><p id="P14" class="p p-first-last">The SCOOP cohort is a UK Caucasian (self-reported) subset of the Genetics of Obesity Study (GOOS), which includes individuals with severe, early-onset obesity (BMI standard deviation score (SDS) &gt; 3; onset of obesity before the age of 10 years.</p></div><div id="S5" class="sec"><p></p><h4 id="S5title" class="inline">WTCCC2 UK controls </h4><p id="P15" class="p p-first-last">Publicly available Affymetrix Human SNP Array 6.0 was available for 6,000 individuals (3,000 from the 1958 British Birth Cohort and 3,000 from the UK Blood Service Collection) recruited for the Wellcome Trust Case Control Consortium 2 (<span class="underline">see URLs</span>).</p></div><div id="S6" class="sec sec-last"><p></p><h4 id="S6title" class="inline">EPIC Norfolk controls </h4><p id="P16" class="p p-first-last">Population-based controls from the EPIC-Norfolk (European Prospective Investigation into Cancer and Nutrition - Norfolk) study<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655848">24</a></sup> were genotyped in the follow-up stage. Samples, not previously genotyped as part of the EPIC-Obesity Study<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655855">25</a></sup> were selected to have BMI greater than or equal to 20 and less than or equal to 25.</p></div></div></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">SNP Analysis</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Discovery genotyping and quality control</h3><p id="P17" class="p p-first">Genome-wide genotyping on 1386 patients from the SCOOP cohort was carried out using the Affymetrix Human SNP Array 6.0 chip at the Wellcome Trust Sanger Institute (WTSI). An additional 334 UK Caucasian GOOS samples described elsewhere<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655860">20</a></sup> previously genotyped using the Affymetrix Human SNP Array 6.0 chip at Aros, Inc. were re-called at the WTSI; both sets of cases were used in the discovery stage.</p><p id="P18" class="p p-last">For the common SNP association analysis the following criteria were applied for exclusion of SNPs prior to imputation: (i) minor allele frequency (MAF)&lt;0.01, (ii) Hardy-Weinberg equilibrium (HWE)<em>p</em>&lt;10<sup>−4</sup>, (iii) call-rate&lt;0.95 if MAF&gt;0.05, call-rate&lt;0.97 if 0.02&lt;=MAF&lt;=0.05 and call-rate&lt;0.99 if 0.01&lt;=MAF&lt;0.02. Samples were excluded prior to imputation based on the following criteria: (i) call-rate&lt;0.95, (ii) heterozygosity outside the population-specific bounds ((iii) relatedness – 500 SNPs with 0.45≤ minor allele frequency&lt;0.5 and call-rate&gt;0.95 and 1000000 bp apart were used to estimate the concordance between pairs of samples. If the genotype concordance for two individuals is &gt;0.7 and &lt;0.97 (related) or ≥0.97 (duplicate) then the one with the lowest call rate was excluded. (iv) gender checks: gender was inferred from the intensity of the A-allele probes on the X chromosome normalised against the autosomal intensities. Individuals for which gender could not be inferred were excluded. (v) intensity failure: individuals were excluded if the mean of their A and B allele intensities from 10000 SNPs on chromosome 22 was outlying when compared to the sample at large. (vi) identity: prior to being genotyped, ~30 SNPs were typed using Sequenom<sup>®</sup> at the WTSI. If the concordance between the genome-wide &amp; Sequenom<sup>®</sup> genotypes was &lt;90% then the individual is excluded on the basis of unknown identity. (vii) ethnic outliers: cases with a distance to CEU &gt; 30 based on a principal components analysis were excluded.</p></div><div id="S9" class="sec"><h3 id="S9title">Imputation</h3><p id="P19" class="p p-first-last">After sample and SNP quality control, data was available for 1509 patients (333 with developmental delay) and 5380 controls from the 1958BC and UKBS collections used in WTCCC2 and up to 862,722 autosomal SNPs. These were separately imputed to the HapMap Phase 2 reference panel (release 22) in IMPUTE v1.0.0<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655845">26</a></sup>. After imputation, SNPs were excluded based on the same MAF, HWE &amp; call-rate thresholds used prior to imputation.</p></div><div id="S10" class="sec"><h3 id="S10title">Principal components analysis</h3><p id="P20" class="p p-first-last">The genotypes of 10k SNPs at least 20kb apart along the genome were taken to calculate the pairwise distance matrix, which was used as the input for classical multidimensional scaling. Individuals were projected using the first two dimensions that correspond to the largest eigenvalues, and the ‘genetic distance’ to Europeans was calculated as the distance in the projected space between individual and the center (median) of the CEU cluster. An empirical distance threshold was adopted, above which individuals were regarded as ethnic outliers (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 8</a></strong>).</p></div><div id="S11" class="sec"><h3 id="S11title">Follow-up genotyping</h3><p id="P21" class="p p-first-last">In the validation stage, 980 SCOOP cases and 2,000 EPIC-Norfolk controls were genotyped at the WTSIby Sequenom<sup>®</sup> or Taqman<sup>®</sup> (Applied Biosystems). Sequenom genotyping was performed using the iPLEX<sup>™</sup> Gold Assay (Sequenom<sup>®</sup> Inc.) according to manufacturer's instructions. Variants that failed or could not be designed on the Sequenom<sup>®</sup> platform were genotyped individually as TaqMan<sup>®</sup> assays (Applied Biosystems, California USA) on 8ng DNA per sample following the standard protocol. Replication samples were excluded on the basis of (i) call-rate&lt;80% or mismatch between inferred and supplied gender. SNPs all had call-rate&gt;90%, Hardy-Weinberg equilibrium <em>P</em>&gt;10<sup>−4</sup> and MAF≥0.01 in cases and controls apart from rs17025867 and rs7255638 indicated in <strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a></strong>.</p></div><div id="S12" class="sec"><h3 id="S12title">Statistical analysis</h3><p id="P22" class="p p-first-last">Case control analyses under a log-additive model were performed using SNPTEST v1.1.5 adjusting for gender. A list of independent loci was generated using the clumping procedure implemented in PLINK, whereby loci were considered to be independent if the pair-wise linkage disequilibrium (LD, <em>r</em><sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655865">2</a></sup>) was less than 0.1 and if they were at least 500kb from the index SNP. Manual inspection of cluster plots was carried out to eliminate spurious associations due to poor genotyping prior to selecting variants for follow-up genotyping. Replication analysis was performed in Stata 11<span class="underline"> (see URLs)</span>. Power calculations were performed using Quanto v 1.2.3 (<span class="underline">see URLs</span>) using the effect size estimate and control effect allele frequency from the validation stage.</p></div><div id="S13" class="sec"><h3 id="S13title">Rare variant analysis</h3><p id="P23" class="p p-first-last">A rare variant analysis method implemented in CCRaVAT (Case-Control Rare Variant Analysis Tool, see URLs) was used to test for association with rare variants (MAF&lt;0.05 in cases), collapsing within genes. Only directly genotyped SNPs for the discovery sample set were used, removing those that failed quality control. Cluster plots for SNPs contributing to any potential signals were manually re-called and checked.</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">Analysis of HBD burden</h3><p id="P24" class="p p-first-last">In order to remove non-random missingness (SNP failures correlating with individual datasets and experimental error rather than random failures) a chi-squared test was carried out on missingness per SNP between datasets. The SNPs with the top 5% of the resultant <em>p</em>-values were removed. In addition, in order to remove possible inflation of HBD from samples with non-European ancestry, only samples that clustered tightly with other European samples in the principle components analysis were included (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 8</a></strong>). Consequently, 1472 case and 5380 controls were used in the Beagle analysis. Beagle was run using the default settings, and memory requirements dictated that the control cohorts were run as separate datasets. HBD burden was obtained by calculating the proportion of individuals that had one or more regions of HBD above a number of different size thresholds in each dataset. <em>P</em>-values for these thresholds were obtained by carrying out a Fishers Exact Test.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">CNV analysis</h2><p id="P25" class="p p-first">Case and control CNVs were called by plate using Birdsuite-1.53 with CEU parameters. Autosomal calls made by Birdseye and Canary with LOD&gt;=10, number of probes&gt;=5, size&gt;=1kb, probe density&gt;=1 per 10kb were kept. Normalized probe intensities produced by apt-probe set-summarize were converted into log<sub>2</sub>ratios using the plate median as the reference. For each sample, the median-of-absolute-deviation (MAD) and the sum of autocorrelation (SAC) between pairs of probe 1-5 probes apart were calculated from log<sub>2</sub>ratios, which reflect the level of noise and long range waviness in the data, respectively. Samples yielding excessive CNV calls were filtered out by fitting the number of kept CNV calls per sample as a linear function of the sample’s MAD in samples with a SAC in the lower 90% and removing outliers of the fitted linear model. Adjacent CNVs in the same sample were merged if (i) they were called with the same genotype, (ii) the distance between them was covered by &lt;100 probes with a density &gt; 0.2 probe per kb, (iii) the ratio of the number of probes between them to the number of probes of the merged call&lt;0.1 and (iv) the difference in average log<sub>2</sub>ratio &lt;0.15. Affy6 SNPs of all cases, controls, the HapMap1 populations (CEU, CHB+JPT, YRI) and an Indian population in HapMap3 (GIH) were called using Birdseed.</p><div id="S16" class="sec"><h3 id="S16title">Common CNV analysis</h3><p id="P26" class="p p-first-last">Case and control CNVs were pooled and clustered into CNV events (CNVEs) as previously described <sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655861">20</a></sup>. 587 CNVEs with a carrier frequency &gt;1% were defined as common CNVEs. Each CNVE was analyzed using the R package CNV tools which implements a likelihood ratio test that models the distribution of per sample CNV measurements as a Gaussian mixture and compares the goodness of fit with or without association to affected status. Test p values could be obtained for 481 of the 587 common CNVEs, whereas the rest failed to run through CNV tools due to poor genotype clustering. For further analysis of NEGR1 deletions, association tests were performed using Fisher’s exact test based on discrete genotype calls, as the distribution of CNV measures of both cases and controls were in distinct and non-overlapping copy number clusters.</p></div><div id="S17" class="sec sec-last"><h3 id="S17title">Rare CNV burden</h3><p id="P27" class="p p-first-last">Conditional permutation test<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_390655841">20</a></sup> was performed to test the hypothesis that cases exhibit a greater burden of rare (carrier frequency&lt;1%) and large (&gt;100kb) CNVs relative to controls. Statistical significance was established by 10,000 rounds of permutation.</p></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Biological Pathway Enrichment Analysis</h2><p id="P28" class="p p-first-last">Using the Database for Annotation, Visualization and Integrated Discovery (DAVID, see URLs), gene lists were mapped onto curated biological processes gene categories using Gene Ontology (GO, The Gene Ontology Consortium). Gene lists were also mapped to pathways and biological processes using the PANTHER classification system (see URLs) and Ingenuity Pathway Analysis (IPA, version 9.0, Ingenuity Systems, see URLs). The most significantly enriched categories of molecules and pathways (smallest p-values, p&lt;0.05) were identified using these tools.</p></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Electrophoretic Mobility Shift Assay (EMSA)</h2><p id="P29" class="p p-first-last">Perfect match (wild type, WT) or mutated (MUT) oligonucleotides to the predicted NKX6.1 binding site within the 8-kb deletion were designed (<strong><a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 8</a></strong>) and used in an EMSA assay. Non-radioactive chemiluminescent detection was employed, and specifically the Light Shift Chemiluminescent EMSA Kit (Fisher-Thermo Scientific, UK). Briefly, labelled or unlabeled oligonucleotides were incubated with cell lysates of HEK293 cells expressing the human NKX6.1 protein (Novus Biologicals, EU), in the presence or absence of anα-NKX6.1 antibody (Abcam, UK). The DNA-protein complexes were run on 4% DNA retardation polyacrylamide gels (Invitrogen, UK), transferred to a nylon membrane using iBlot DNA transfer stacks and iBlot transfer system (Invitrogen, UK), and the resulting chemiluminesce detected by exposure to an X-ray film (Kodak, UK).</p></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplementary Material</h4><div class="sup-box half_rhythm" id="d36e918"><a href="/pmc/articles/PMC4106235/bin/NIHMS612416-supplement-Supplementary_Material.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.3M, pdf)</sup></div></div></div><div id="S20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S20title">Acknowledgements</h2><div class="sec"><p id="__p1">We are indebted to the patients and their families for their participation and to the physicians involved in the Genetics of Obesity Study (GOOS). This work was supported by the Wellcome Trust (085475; WT098051), Medical Research Council and NIHR Cambridge Biomedical Research Centre. This study makes use of data generated by the Wellcome Trust Case-Control Consortium 2. A full list of the investigators who contributed to the generation of the data is available from the Wellcome Trust Case-Control Consortium website (see URLs). The authors would like to thank Sarah Edkins, Emma Gray, Douglas Simpkin, Sarah Hunt, Sue Bumpstead and Serge Dronov of the WTSI Sample Logistics, Genotyping and Variation Informatics Facilities; Felicity Payne &amp; Emma Gayton for Taqman genotyping and Eva Gonçalves Serra for imputation of the <em>GPR120</em> region.</p></div></div><div id="idm140341546310768" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140341546310768title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Farooqi IS, O'Rahilly S. Genetic factors in human obesity. <span><span class="ref-journal">Obes Rev. </span>2007;<span class="ref-vol">8</span>(Suppl 1):37–40.</span> [<a href="/pubmed/17316299" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obes+Rev&amp;title=Genetic+factors+in+human+obesity&amp;author=IS+Farooqi&amp;author=S+O'Rahilly&amp;volume=8&amp;issue=Suppl+1&amp;publication_year=2007&amp;pages=37-40&amp;pmid=17316299&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Loos RJ. Genetic determinants of common obesity and their value in prediction. <span><span class="ref-journal">Best Pract Res Clin Endocrinol Metab. </span>2012;<span class="ref-vol">26</span>:211–26.</span> [<a href="/pubmed/22498250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Best+Pract+Res+Clin+Endocrinol+Metab&amp;title=Genetic+determinants+of+common+obesity+and+their+value+in+prediction&amp;author=RJ+Loos&amp;volume=26&amp;publication_year=2012&amp;pages=211-26&amp;pmid=22498250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Zuk O, Hechter E, Sunyaev SR, Lander ES. The mystery of missing heritability: Genetic interactions create phantom heritability. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2012;<span class="ref-vol">109</span>:1193–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3268279/">PMC free article</a>]</span> [<a href="/pubmed/22223662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+mystery+of+missing+heritability:+Genetic+interactions+create+phantom+heritability&amp;author=O+Zuk&amp;author=E+Hechter&amp;author=SR+Sunyaev&amp;author=ES+Lander&amp;volume=109&amp;publication_year=2012&amp;pages=1193-8&amp;pmid=22223662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Cohen JC, et al.  Multiple rare alleles contribute to low plasma levels of HDL cholesterol. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">305</span>:869–72.</span> [<a href="/pubmed/15297675" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Multiple+rare+alleles+contribute+to+low+plasma+levels+of+HDL+cholesterol&amp;author=JC+Cohen&amp;volume=305&amp;publication_year=2004&amp;pages=869-72&amp;pmid=15297675&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:661–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Stranger BE, et al.  Patterns of cis regulatory variation in diverse human populations. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002639.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3330104/">PMC free article</a>]</span> [<a href="/pubmed/22532805" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Patterns+of+cis+regulatory+variation+in+diverse+human+populations&amp;author=BE+Stranger&amp;volume=8&amp;publication_year=2012&amp;pages=e1002639&amp;pmid=22532805&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Bradfield JP, et al.  A genome-wide association meta-analysis identifies new childhood obesity loci. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:526–31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3370100/">PMC free article</a>]</span> [<a href="/pubmed/22484627" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+meta-analysis+identifies+new+childhood+obesity+loci&amp;author=JP+Bradfield&amp;volume=44&amp;publication_year=2012&amp;pages=526-31&amp;pmid=22484627&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Dina C, et al.  Variation in FTO contributes to childhood obesity and severe adult obesity. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:724–6.</span> [<a href="/pubmed/17496892" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Variation+in+FTO+contributes+to+childhood+obesity+and+severe+adult+obesity&amp;author=C+Dina&amp;volume=39&amp;publication_year=2007&amp;pages=724-6&amp;pmid=17496892&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Frayling TM, et al.  A common variant in the FTO gene is associated with body mass index and predisposes to childhood and adult obesity. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:889–94.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646098/">PMC free article</a>]</span> [<a href="/pubmed/17434869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+variant+in+the+FTO+gene+is+associated+with+body+mass+index+and+predisposes+to+childhood+and+adult+obesity&amp;author=TM+Frayling&amp;volume=316&amp;publication_year=2007&amp;pages=889-94&amp;pmid=17434869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Scherag A, et al.  Two new Loci for body-weight regulation identified in a joint analysis of genome-wide association studies for early-onset extreme obesity in French and german study groups. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span>:e1000916.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2858696/">PMC free article</a>]</span> [<a href="/pubmed/20421936" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Two+new+Loci+for+body-weight+regulation+identified+in+a+joint+analysis+of+genome-wide+association+studies+for+early-onset+extreme+obesity+in+French+and+german+study+groups&amp;author=A+Scherag&amp;volume=6&amp;publication_year=2010&amp;pages=e1000916&amp;pmid=20421936&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Loos RJ, et al.  Common variants near MC4R are associated with fat mass, weight and risk of obesity. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:768–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669167/">PMC free article</a>]</span> [<a href="/pubmed/18454148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+near+MC4R+are+associated+with+fat+mass,+weight+and+risk+of+obesity&amp;author=RJ+Loos&amp;volume=40&amp;publication_year=2008&amp;pages=768-75&amp;pmid=18454148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Jiao H, et al.  Genome wide association study identifies KCNMA1 contributing to human obesity. <span><span class="ref-journal">BMC Med Genomics. </span>2011;<span class="ref-vol">4</span>:51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3148553/">PMC free article</a>]</span> [<a href="/pubmed/21708048" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genomics&amp;title=Genome+wide+association+study+identifies+KCNMA1+contributing+to+human+obesity&amp;author=H+Jiao&amp;volume=4&amp;publication_year=2011&amp;pages=51&amp;pmid=21708048&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Benzinou M, et al.  Common nonsynonymous variants in PCSK1 confer risk of obesity. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:943–5.</span> [<a href="/pubmed/18604207" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+nonsynonymous+variants+in+PCSK1+confer+risk+of+obesity&amp;author=M+Benzinou&amp;volume=40&amp;publication_year=2008&amp;pages=943-5&amp;pmid=18604207&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Meyre D, et al.  Genome-wide association study for early-onset and morbid adult obesity identifies three new risk loci in European populations. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:157–9.</span> [<a href="/pubmed/19151714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+for+early-onset+and+morbid+adult+obesity+identifies+three+new+risk+loci+in+European+populations&amp;author=D+Meyre&amp;volume=41&amp;publication_year=2009&amp;pages=157-9&amp;pmid=19151714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Ichimura A, et al.  Dysfunction of lipid sensor GPR120 leads to obesity in both mouse and human. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">483</span>:350–4.</span> [<a href="/pubmed/22343897" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dysfunction+of+lipid+sensor+GPR120+leads+to+obesity+in+both+mouse+and+human&amp;author=A+Ichimura&amp;volume=483&amp;publication_year=2012&amp;pages=350-4&amp;pmid=22343897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Paternoster L, et al.  Genome-wide population-based association study of extremely overweight young adults--the GOYA study. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e24303.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3174168/">PMC free article</a>]</span> [<a href="/pubmed/21935397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Genome-wide+population-based+association+study+of+extremely+overweight+young+adults--the+GOYA+study&amp;author=L+Paternoster&amp;volume=6&amp;publication_year=2011&amp;pages=e24303&amp;pmid=21935397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Speliotes EK, et al.  Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:937–48.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014648/">PMC free article</a>]</span> [<a href="/pubmed/20935630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Association+analyses+of+249,796+individuals+reveal+18+new+loci+associated+with+body+mass+index&amp;author=EK+Speliotes&amp;volume=42&amp;publication_year=2010&amp;pages=937-48&amp;pmid=20935630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Hafler BP, Choi MY, Shivdasani RA, Rowitch DH. Expression and function of Nkx6.3 in vertebrate hindbrain. <span><span class="ref-journal">Brain Res. </span>2008;<span class="ref-vol">1222</span>:42–50.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2555971/">PMC free article</a>]</span> [<a href="/pubmed/18586225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Expression+and+function+of+Nkx6.3+in+vertebrate+hindbrain&amp;author=BP+Hafler&amp;author=MY+Choi&amp;author=RA+Shivdasani&amp;author=DH+Rowitch&amp;volume=1222&amp;publication_year=2008&amp;pages=42-50&amp;pmid=18586225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Schafer M, Brauer AU, Savaskan NE, Rathjen FG, Brummendorf T. Neurotractin/kilon promotes neurite outgrowth and is expressed on reactive astrocytes after entorhinal cortex lesion. <span><span class="ref-journal">Mol Cell Neurosci. </span>2005;<span class="ref-vol">29</span>:580–90.</span> [<a href="/pubmed/15946856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Neurosci&amp;title=Neurotractin/kilon+promotes+neurite+outgrowth+and+is+expressed+on+reactive+astrocytes+after+entorhinal+cortex+lesion&amp;author=M+Schafer&amp;author=AU+Brauer&amp;author=NE+Savaskan&amp;author=FG+Rathjen&amp;author=T+Brummendorf&amp;volume=29&amp;publication_year=2005&amp;pages=580-90&amp;pmid=15946856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Bochukova EG, et al.  Large, rare chromosomal deletions associated with severe early-onset obesity. <span><span class="ref-journal">Nature. </span><span class="ref-vol">463</span>:666–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3108883/">PMC free article</a>]</span> [<a href="/pubmed/19966786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Large,+rare+chromosomal+deletions+associated+with+severe+early-onset+obesity&amp;author=EG+Bochukova&amp;volume=463&amp;pages=666-70&amp;pmid=19966786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Stice E, Spoor S, Bohon C, Small DM. Relation between obesity and blunted striatal response to food is moderated by TaqIA A1 allele. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">322</span>:449–52.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2681095/">PMC free article</a>]</span> [<a href="/pubmed/18927395" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Relation+between+obesity+and+blunted+striatal+response+to+food+is+moderated+by+TaqIA+A1+allele&amp;author=E+Stice&amp;author=S+Spoor&amp;author=C+Bohon&amp;author=DM+Small&amp;volume=322&amp;publication_year=2008&amp;pages=449-52&amp;pmid=18927395&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Raychaudhuri S, et al.  Accurately assessing the risk of schizophrenia conferred by rare copy-number variation affecting genes with brain function. <span><span class="ref-journal">PLoS Genet. </span>2010;<span class="ref-vol">6</span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2936523/">PMC free article</a>]</span> [<a href="/pubmed/20838587" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Accurately+assessing+the+risk+of+schizophrenia+conferred+by+rare+copy-number+variation+affecting+genes+with+brain+function&amp;author=S+Raychaudhuri&amp;volume=6&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Pruim RJ, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Day N, et al.  EPIC-Norfolk: study design and characteristics of the cohort. European Prospective Investigation of Cancer. <span><span class="ref-journal">Br J Cancer. </span>1999;<span class="ref-vol">80</span>(Suppl 1):95–103.</span> [<a href="/pubmed/10466767" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=EPIC-Norfolk:+study+design+and+characteristics+of+the+cohort.+European+Prospective+Investigation+of+Cancer&amp;author=N+Day&amp;volume=80&amp;issue=Suppl+1&amp;publication_year=1999&amp;pages=95-103&amp;pmid=10466767&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Loos RJ, et al.  Common variants near MC4R are associated with fat mass, weight and risk of obesity. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:768–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2669167/">PMC free article</a>]</span> [<a href="/pubmed/18454148" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+near+MC4R+are+associated+with+fat+mass,+weight+and+risk+of+obesity&amp;author=RJ+Loos&amp;volume=40&amp;publication_year=2008&amp;pages=768-75&amp;pmid=18454148&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–13.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=P+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-13&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4106235&amp;issue-id=240607&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#S1title">URLs</a></li><li><a href="#S2title">Online Methods</a></li><li><a href="#S7title">SNP Analysis</a></li><li><a href="#S15title">CNV analysis</a></li><li><a href="#S18title">Biological Pathway Enrichment Analysis</a></li><li><a href="#S19title">Electrophoretic Mobility Shift Assay (EMSA)</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S20title">Acknowledgements</a></li><li><a href="#idm140341546310768title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4106235/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4106235/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4106235/pdf/nihms-612416.pdf">PDF (756K)</a></li> | <li><a href="#" data-citationid="PMC4106235" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4106235%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4106235%2F&amp;text=Genome-wide%20SNP%20and%20CNV%20analysis%20identifies%20common%20and%20low-frequency%20variants%20associated%20with%20severe%20early-onset%20obesity" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4106235%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="23563609" data-aiid="4106235" data-aid="4106235" data-iid="240607" data-domainid="319" data-domain="nihpa" data-accid="PMC4106235" data-md5="6d596ce8e2142b543e7fa003c3fe330a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T10:42:16-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D26AADD5589710000000005030310&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4106235%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4106235/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>