<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4245044/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Trans-ethnic meta-analysis of white blood cell phenotypes</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4245044/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Trans-ethnic meta-analysis of white blood cell phenotypes"><meta name="citation_authors" content="Margaux F. Keller, Alexander P. Reiner, Yukinori Okada, Frank J.A. van Rooij, Andrew D. Johnson, Ming-Huei Chen, Albert V. Smith, Andrew P. Morris, Toshiko Tanaka, Luigi Ferrucci, Alan B. Zonderman, Guillaume Lettre, Tamara Harris, Melissa Garcia, Stefania Bandinelli, Rehan Qayyum, Lisa R. Yanek, Diane M. Becker, Lewis C. Becker, Charles Kooperberg, Brendan Keating, Jared Reis, Hua Tang, Eric Boerwinkle, Yoichiro Kamatani, Koichi Matsuda, Naoyuki Kamatani, Yusuke Nakamura, Michiaki Kubo, Simin Liu, Abbas Dehghan, Janine F. Felix, Albert Hofman, André G. Uitterlinden, Cornelia M. van Duijn, Oscar H. Franco, Dan L. Longo, Andrew B. Singleton, Bruce M. Psaty, Michelle K. Evans, L. Adrienne Cupples, Jerome I. Rotter, Christopher J. O'Donnell, Atsushi Takahashi, James G. Wilson, Santhi K. Ganesh, Mike A. Nalls"><meta name="citation_date" content="20 December 2014"><meta name="citation_issue" content="25"><meta name="citation_volume" content="23"><meta name="citation_firstpage" content="6944"><meta name="citation_doi" content="10.1093/hmg/ddu401"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4245044/?report=abstract"><meta name="citation_pmid" content="25096241"><meta name="DC.Title" content="Trans-ethnic meta-analysis of white blood cell phenotypes"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Margaux F. Keller"><meta name="DC.Contributor" content="Alexander P. Reiner"><meta name="DC.Contributor" content="Yukinori Okada"><meta name="DC.Contributor" content="Frank J.A. van Rooij"><meta name="DC.Contributor" content="Andrew D. Johnson"><meta name="DC.Contributor" content="Ming-Huei Chen"><meta name="DC.Contributor" content="Albert V. Smith"><meta name="DC.Contributor" content="Andrew P. Morris"><meta name="DC.Contributor" content="Toshiko Tanaka"><meta name="DC.Contributor" content="Luigi Ferrucci"><meta name="DC.Contributor" content="Alan B. Zonderman"><meta name="DC.Contributor" content="Guillaume Lettre"><meta name="DC.Contributor" content="Tamara Harris"><meta name="DC.Contributor" content="Melissa Garcia"><meta name="DC.Contributor" content="Stefania Bandinelli"><meta name="DC.Contributor" content="Rehan Qayyum"><meta name="DC.Contributor" content="Lisa R. Yanek"><meta name="DC.Contributor" content="Diane M. Becker"><meta name="DC.Contributor" content="Lewis C. Becker"><meta name="DC.Contributor" content="Charles Kooperberg"><meta name="DC.Contributor" content="Brendan Keating"><meta name="DC.Contributor" content="Jared Reis"><meta name="DC.Contributor" content="Hua Tang"><meta name="DC.Contributor" content="Eric Boerwinkle"><meta name="DC.Contributor" content="Yoichiro Kamatani"><meta name="DC.Contributor" content="Koichi Matsuda"><meta name="DC.Contributor" content="Naoyuki Kamatani"><meta name="DC.Contributor" content="Yusuke Nakamura"><meta name="DC.Contributor" content="Michiaki Kubo"><meta name="DC.Contributor" content="Simin Liu"><meta name="DC.Contributor" content="Abbas Dehghan"><meta name="DC.Contributor" content="Janine F. Felix"><meta name="DC.Contributor" content="Albert Hofman"><meta name="DC.Contributor" content="André G. Uitterlinden"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Oscar H. Franco"><meta name="DC.Contributor" content="Dan L. Longo"><meta name="DC.Contributor" content="Andrew B. Singleton"><meta name="DC.Contributor" content="Bruce M. Psaty"><meta name="DC.Contributor" content="Michelle K. Evans"><meta name="DC.Contributor" content="L. Adrienne Cupples"><meta name="DC.Contributor" content="Jerome I. Rotter"><meta name="DC.Contributor" content="Christopher J. O'Donnell"><meta name="DC.Contributor" content="Atsushi Takahashi"><meta name="DC.Contributor" content="James G. Wilson"><meta name="DC.Contributor" content="Santhi K. Ganesh"><meta name="DC.Contributor" content="Mike A. Nalls"><meta name="DC.Date" content="2014 Dec 20"><meta name="DC.Identifier" content="10.1093/hmg/ddu401"><meta name="DC.Language" content="en"><meta property="og:title" content="Trans-ethnic meta-analysis of white blood cell phenotypes"><meta property="og:type" content="article"><meta property="og:description" content="White blood cell (WBC) count is a common clinical measure used as a predictor of certain aspects of human health, including immunity and infection status. WBC count is also a complex trait that varies among individuals and ancestry groups. Differences ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4245044/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4245044/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4245044/pdf/ddu401.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8848C2DD56CF510000000008EE054F.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4245044%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC4245044</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2014 Dec 20; 23(25): 6944–6960. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Aug 5. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu401" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddu401</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4245044</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25096241">25096241</a></div></div></div></div><h1 class="content-title">Trans-ethnic meta-analysis of white blood cell phenotypes</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Keller%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167485407408" co-class="co-affbox">Margaux F. Keller</a>,<sup>1,</sup><sup>7,</sup><sup>†</sup> <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167557059984" co-class="co-affbox">Alexander P. Reiner</a>,<sup>8,</sup><sup>12,</sup><sup>†</sup> <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482382416" co-class="co-affbox">Yukinori Okada</a>,<sup>13,</sup><sup>15,</sup><sup>†</sup> <a href="/pubmed/?term=van%20Rooij%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167475425664" co-class="co-affbox">Frank J.A. van Rooij</a>,<sup>16,</sup><sup>19,</sup><sup>†</sup> <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167480095232" co-class="co-affbox">Andrew D. Johnson</a>,<sup>20,</sup><sup>22</sup> <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167485282976" co-class="co-affbox">Ming-Huei Chen</a>,<sup>21,</sup><sup>22</sup> <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167479443904" co-class="co-affbox">Albert V. Smith</a>,<sup>23,</sup><sup>24</sup> <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167557252432" co-class="co-affbox">Andrew P. Morris</a>,<sup>25,</sup><sup>26</sup> <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167476409536" co-class="co-affbox">Toshiko Tanaka</a>,<sup>3</sup> <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167473213280" co-class="co-affbox">Luigi Ferrucci</a>,<sup>3</sup> <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167476622624" co-class="co-affbox">Alan B. Zonderman</a>,<sup>4</sup> <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167477780512" co-class="co-affbox">Guillaume Lettre</a>,<sup>27,</sup><sup>28</sup> <a href="/pubmed/?term=Harris%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482422912" co-class="co-affbox">Tamara Harris</a>,<sup>2</sup> <a href="/pubmed/?term=Garcia%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482420928" co-class="co-affbox">Melissa Garcia</a>,<sup>2</sup> <a href="/pubmed/?term=Bandinelli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167476770400" co-class="co-affbox">Stefania Bandinelli</a>,<sup>29</sup> <a href="/pubmed/?term=Qayyum%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167484587584" co-class="co-affbox">Rehan Qayyum</a>,<sup>30</sup> <a href="/pubmed/?term=Yanek%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482290352" co-class="co-affbox">Lisa R. Yanek</a>,<sup>30</sup> <a href="/pubmed/?term=Becker%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482288480" co-class="co-affbox">Diane M. Becker</a>,<sup>30</sup> <a href="/pubmed/?term=Becker%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167556145216" co-class="co-affbox">Lewis C. Becker</a>,<sup>30,</sup><sup>31</sup> <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167480802688" co-class="co-affbox">Charles Kooperberg</a>,<sup>12</sup> <a href="/pubmed/?term=Keating%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167486890352" co-class="co-affbox">Brendan Keating</a>,<sup>32,</sup><sup>33</sup> <a href="/pubmed/?term=Reis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167480866384" co-class="co-affbox">Jared Reis</a>,<sup>34</sup> <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167480864400" co-class="co-affbox">Hua Tang</a>,<sup>35</sup> <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167559526048" co-class="co-affbox">Eric Boerwinkle</a>,<sup>36</sup> <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167523290288" co-class="co-affbox">Yoichiro Kamatani</a>,<sup>13</sup> <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167479788016" co-class="co-affbox">Koichi Matsuda</a>,<sup>37</sup> <a href="/pubmed/?term=Kamatani%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167479786032" co-class="co-affbox">Naoyuki Kamatani</a>,<sup>13</sup> <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167478687712" co-class="co-affbox">Yusuke Nakamura</a>,<sup>37,</sup><sup>38,</sup><sup>39</sup> <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167481568576" co-class="co-affbox">Michiaki Kubo</a>,<sup>14</sup> <a href="/pubmed/?term=Liu%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167477724464" co-class="co-affbox">Simin Liu</a>,<sup>40,</sup><sup>41</sup> <a href="/pubmed/?term=Dehghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167486454864" co-class="co-affbox">Abbas Dehghan</a>,<sup>16,</sup><sup>19</sup> <a href="/pubmed/?term=Felix%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167480292928" co-class="co-affbox">Janine F. Felix</a>,<sup>16,</sup><sup>19</sup> <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167473563472" co-class="co-affbox">Albert Hofman</a>,<sup>16,</sup><sup>19</sup> <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167478567920" co-class="co-affbox">André G. Uitterlinden</a>,<sup>16,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167486064816" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>16,</sup><sup>19</sup> <a href="/pubmed/?term=Franco%20OH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167475174592" co-class="co-affbox">Oscar H. Franco</a>,<sup>16,</sup><sup>17,</sup><sup>19</sup> <a href="/pubmed/?term=Longo%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167481585904" co-class="co-affbox">Dan L. Longo</a>,<sup>5</sup> <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167481583920" co-class="co-affbox">Andrew B. Singleton</a>,<sup>1</sup> <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167479381696" co-class="co-affbox">Bruce M. Psaty</a>,<sup>9,</sup><sup>10,</sup><sup>11,</sup><sup>42</sup> <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167520865312" co-class="co-affbox">Michelle K. Evans</a>,<sup>6</sup> <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167478031936" co-class="co-affbox">L. Adrienne Cupples</a>,<sup>22,</sup><sup>43</sup> <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167482231600" co-class="co-affbox">Jerome I. Rotter</a>,<sup>44,</sup><sup>45</sup> <a href="/pubmed/?term=O'Donnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167521001904" co-class="co-affbox">Christopher J. O'Donnell</a>,<sup>20,</sup><sup>22</sup> <a href="/pubmed/?term=Takahashi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167485680480" co-class="co-affbox">Atsushi Takahashi</a>,<sup>13,</sup><sup>‡</sup> <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167478419744" co-class="co-affbox">James G. Wilson</a>,<sup>46,</sup><sup>‡</sup> <a href="/pubmed/?term=Ganesh%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167475705872" co-class="co-affbox">Santhi K. Ganesh</a>,<sup>47,</sup><sup>48,</sup><sup>‡</sup><sup>*</sup> and  <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241" class="affpopup" co-rid="_co_idm140167486976752" co-class="co-affbox">Mike A. Nalls</a><sup>1,</sup><sup>‡</sup><sup>*</sup>, for the CHARGE Hematology, COGENT, and BioBank Japan Project (RIKEN) Working Groups<sup>¶</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140167485407408"><h3 class="no_margin">Margaux F. Keller</h3><p><sup>1</sup>Laboratory of Neurogenetics</p><p><sup>7</sup>Department of Biological Anthropology, Temple University, Philadelphia, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Keller%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Margaux F. Keller</a></div></div><div id="_co_idm140167557059984"><h3 class="no_margin">Alexander P. Reiner</h3><p><sup>8</sup>Department of Epidemiology</p><p><sup>12</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Reiner%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Alexander P. Reiner</a></div></div><div id="_co_idm140167482382416"><h3 class="no_margin">Yukinori Okada</h3><p><sup>13</sup>Laboratory for Statistical Analysis</p><p><sup>15</sup>Department of Human Genetics and Disease Diversity, Graduate School of Medical and Dental Sciences, Tokyo Medical and Dental University, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Okada%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Yukinori Okada</a></div></div><div id="_co_idm140167475425664"><h3 class="no_margin">Frank J.A. van Rooij</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Rooij%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Frank J.A. van Rooij</a></div></div><div id="_co_idm140167480095232"><h3 class="no_margin">Andrew D. Johnson</h3><p><sup>20</sup>Cardiovascular Epidemiology and Human Genomics Branch, NHLBI Division of Intramural Research, Bethesda, MD, USA</p><p><sup>22</sup>NHLBI Framingham Heart Study, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20AD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Andrew D. Johnson</a></div></div><div id="_co_idm140167485282976"><h3 class="no_margin">Ming-Huei Chen</h3><p><sup>21</sup>Department of Neurology, Boston University School of Medicine, Boston, MA, USA</p><p><sup>22</sup>NHLBI Framingham Heart Study, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Chen%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Ming-Huei Chen</a></div></div><div id="_co_idm140167479443904"><h3 class="no_margin">Albert V. Smith</h3><p><sup>23</sup>Icelandic Heart Association, Kopavogur, Iceland</p><p><sup>24</sup>University of Iceland, Reykjavik, Iceland</p><div>Find articles by <a href="/pubmed/?term=Smith%20AV%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Albert V. Smith</a></div></div><div id="_co_idm140167557252432"><h3 class="no_margin">Andrew P. Morris</h3><p><sup>25</sup>Genetic and Genomic Epidemiology Unit, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</p><p><sup>26</sup>Department of Biostatistics, University of Liverpool, Liverpool, UK</p><div>Find articles by <a href="/pubmed/?term=Morris%20AP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Andrew P. Morris</a></div></div><div id="_co_idm140167476409536"><h3 class="no_margin">Toshiko Tanaka</h3><p><sup>3</sup>Longitudinal Studies Section, Clinical Research Branch</p><div>Find articles by <a href="/pubmed/?term=Tanaka%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Toshiko Tanaka</a></div></div><div id="_co_idm140167473213280"><h3 class="no_margin">Luigi Ferrucci</h3><p><sup>3</sup>Longitudinal Studies Section, Clinical Research Branch</p><div>Find articles by <a href="/pubmed/?term=Ferrucci%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Luigi Ferrucci</a></div></div><div id="_co_idm140167476622624"><h3 class="no_margin">Alan B. Zonderman</h3><p><sup>4</sup>Behavioral Epidemiology Section, Laboratory of Epidemiology &amp; Population Sciences, National Institute on Aging Intramural Research Program, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Alan B. Zonderman</a></div></div><div id="_co_idm140167477780512"><h3 class="no_margin">Guillaume Lettre</h3><p><sup>27</sup>Montreal Heart Institute, Montréal, Canada</p><p><sup>28</sup>Département de Médecine, Université de Montréal, Montréal, Canada</p><div>Find articles by <a href="/pubmed/?term=Lettre%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Guillaume Lettre</a></div></div><div id="_co_idm140167482422912"><h3 class="no_margin">Tamara Harris</h3><p><sup>2</sup>Laboratory of Epidemiology and Population Sciences, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Harris%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Tamara Harris</a></div></div><div id="_co_idm140167482420928"><h3 class="no_margin">Melissa Garcia</h3><p><sup>2</sup>Laboratory of Epidemiology and Population Sciences, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Garcia%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Melissa Garcia</a></div></div><div id="_co_idm140167476770400"><h3 class="no_margin">Stefania Bandinelli</h3><p><sup>29</sup>Geriatric Rehabilitation Unit, Azienda Sanitaria Firenze (ASF), Florence, Italy</p><div>Find articles by <a href="/pubmed/?term=Bandinelli%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Stefania Bandinelli</a></div></div><div id="_co_idm140167484587584"><h3 class="no_margin">Rehan Qayyum</h3><p><sup>30</sup>GeneSTAR Research Program, Division of General Internal Medicine</p><div>Find articles by <a href="/pubmed/?term=Qayyum%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Rehan Qayyum</a></div></div><div id="_co_idm140167482290352"><h3 class="no_margin">Lisa R. Yanek</h3><p><sup>30</sup>GeneSTAR Research Program, Division of General Internal Medicine</p><div>Find articles by <a href="/pubmed/?term=Yanek%20LR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Lisa R. Yanek</a></div></div><div id="_co_idm140167482288480"><h3 class="no_margin">Diane M. Becker</h3><p><sup>30</sup>GeneSTAR Research Program, Division of General Internal Medicine</p><div>Find articles by <a href="/pubmed/?term=Becker%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Diane M. Becker</a></div></div><div id="_co_idm140167556145216"><h3 class="no_margin">Lewis C. Becker</h3><p><sup>30</sup>GeneSTAR Research Program, Division of General Internal Medicine</p><p><sup>31</sup>Division of Cardiology, Johns Hopkins School of Medicine, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Becker%20LC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Lewis C. Becker</a></div></div><div id="_co_idm140167480802688"><h3 class="no_margin">Charles Kooperberg</h3><p><sup>12</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Kooperberg%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Charles Kooperberg</a></div></div><div id="_co_idm140167486890352"><h3 class="no_margin">Brendan Keating</h3><p><sup>32</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, PA, USA</p><p><sup>33</sup>Department of Pediatrics, University of Pennsylvania, PA, USA</p><div>Find articles by <a href="/pubmed/?term=Keating%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Brendan Keating</a></div></div><div id="_co_idm140167480866384"><h3 class="no_margin">Jared Reis</h3><p><sup>34</sup>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Reis%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Jared Reis</a></div></div><div id="_co_idm140167480864400"><h3 class="no_margin">Hua Tang</h3><p><sup>35</sup>Stanford University School of Medicine, Stanford, CA 94305, USA</p><div>Find articles by <a href="/pubmed/?term=Tang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Hua Tang</a></div></div><div id="_co_idm140167559526048"><h3 class="no_margin">Eric Boerwinkle</h3><p><sup>36</sup>The Brown Foundation, Institute of Molecular Medicine for the Prevention of Human Diseases, University of Texas, Houston, TX, USA</p><div>Find articles by <a href="/pubmed/?term=Boerwinkle%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Eric Boerwinkle</a></div></div><div id="_co_idm140167523290288"><h3 class="no_margin">Yoichiro Kamatani</h3><p><sup>13</sup>Laboratory for Statistical Analysis</p><div>Find articles by <a href="/pubmed/?term=Kamatani%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Yoichiro Kamatani</a></div></div><div id="_co_idm140167479788016"><h3 class="no_margin">Koichi Matsuda</h3><p><sup>37</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, Tokyo, Japan</p><div>Find articles by <a href="/pubmed/?term=Matsuda%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Koichi Matsuda</a></div></div><div id="_co_idm140167479786032"><h3 class="no_margin">Naoyuki Kamatani</h3><p><sup>13</sup>Laboratory for Statistical Analysis</p><div>Find articles by <a href="/pubmed/?term=Kamatani%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Naoyuki Kamatani</a></div></div><div id="_co_idm140167478687712"><h3 class="no_margin">Yusuke Nakamura</h3><p><sup>37</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, Tokyo, Japan</p><p><sup>38</sup>Department of Medicine</p><p><sup>39</sup>Department of Surgery, Center for Personalized Therapeutics, The University of Chicago, Chicago, IL, USA</p><div>Find articles by <a href="/pubmed/?term=Nakamura%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Yusuke Nakamura</a></div></div><div id="_co_idm140167481568576"><h3 class="no_margin">Michiaki Kubo</h3><p><sup>14</sup>Laboratory for Genotyping Development, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</p><div>Find articles by <a href="/pubmed/?term=Kubo%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Michiaki Kubo</a></div></div><div id="_co_idm140167477724464"><h3 class="no_margin">Simin Liu</h3><p><sup>40</sup>Department of Epidemiology</p><p><sup>41</sup>Department of Medicine, Brown University, Providence, RI, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Simin Liu</a></div></div><div id="_co_idm140167486454864"><h3 class="no_margin">Abbas Dehghan</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Dehghan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Abbas Dehghan</a></div></div><div id="_co_idm140167480292928"><h3 class="no_margin">Janine F. Felix</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Felix%20JF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Janine F. Felix</a></div></div><div id="_co_idm140167473563472"><h3 class="no_margin">Albert Hofman</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Hofman%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Albert Hofman</a></div></div><div id="_co_idm140167478567920"><h3 class="no_margin">André G. Uitterlinden</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>18</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, The Netherlands</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Uitterlinden%20AG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">André G. Uitterlinden</a></div></div><div id="_co_idm140167486064816"><h3 class="no_margin">Cornelia M. van Duijn</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Cornelia M. van Duijn</a></div></div><div id="_co_idm140167475174592"><h3 class="no_margin">Oscar H. Franco</h3><p><sup>16</sup>Department of Epidemiology</p><p><sup>17</sup>ErasmusAGE, Department of Epidemiology</p><p><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</p><div>Find articles by <a href="/pubmed/?term=Franco%20OH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Oscar H. Franco</a></div></div><div id="_co_idm140167481585904"><h3 class="no_margin">Dan L. Longo</h3><p><sup>5</sup>Laboratory of Genetics</p><div>Find articles by <a href="/pubmed/?term=Longo%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Dan L. Longo</a></div></div><div id="_co_idm140167481583920"><h3 class="no_margin">Andrew B. Singleton</h3><p><sup>1</sup>Laboratory of Neurogenetics</p><div>Find articles by <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Andrew B. Singleton</a></div></div><div id="_co_idm140167479381696"><h3 class="no_margin">Bruce M. Psaty</h3><p><sup>9</sup>Cardiovascular Health Research Unit</p><p><sup>10</sup>Department of Medicine</p><p><sup>11</sup>Department of Epidemiology and Health Services, University of Washington, Seattle, WA, USA</p><p><sup>42</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Psaty%20BM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Bruce M. Psaty</a></div></div><div id="_co_idm140167520865312"><h3 class="no_margin">Michelle K. Evans</h3><p><sup>6</sup>Health Disparities Research Section, Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Michelle K. Evans</a></div></div><div id="_co_idm140167478031936"><h3 class="no_margin">L. Adrienne Cupples</h3><p><sup>22</sup>NHLBI Framingham Heart Study, Bethesda, MD, USA</p><p><sup>43</sup>Boston University Department of Statistics, Boston, MA, USA</p><div>Find articles by <a href="/pubmed/?term=Cupples%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">L. Adrienne Cupples</a></div></div><div id="_co_idm140167482231600"><h3 class="no_margin">Jerome I. Rotter</h3><p><sup>44</sup>Institute for Translational Genomics and Population Sciences, Los Angeles BioMedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA, USA</p><p><sup>45</sup>Division of Genetic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Rotter%20JI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Jerome I. Rotter</a></div></div><div id="_co_idm140167521001904"><h3 class="no_margin">Christopher J. O'Donnell</h3><p><sup>20</sup>Cardiovascular Epidemiology and Human Genomics Branch, NHLBI Division of Intramural Research, Bethesda, MD, USA</p><p><sup>22</sup>NHLBI Framingham Heart Study, Bethesda, MD, USA</p><div>Find articles by <a href="/pubmed/?term=O'Donnell%20CJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Christopher J. O'Donnell</a></div></div><div id="_co_idm140167485680480"><h3 class="no_margin">Atsushi Takahashi</h3><p><sup>13</sup>Laboratory for Statistical Analysis</p><div>Find articles by <a href="/pubmed/?term=Takahashi%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Atsushi Takahashi</a></div></div><div id="_co_idm140167478419744"><h3 class="no_margin">James G. Wilson</h3><p><sup>46</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">James G. Wilson</a></div></div><div id="_co_idm140167475705872"><h3 class="no_margin">Santhi K. Ganesh</h3><p><sup>47</sup>Division of Cardiovascular Medicine, Department of Internal Medicine, University of Michigan, Ann Arbor, MI, USA</p><p><sup>48</sup>Department of Human Genetics, University of Michigan, Ann Arbor, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Ganesh%20SK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Santhi K. Ganesh</a></div></div><div id="_co_idm140167486976752"><h3 class="no_margin">Mike A. Nalls</h3><p><sup>1</sup>Laboratory of Neurogenetics</p><div>Find articles by <a href="/pubmed/?term=Nalls%20MA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25096241">Mike A. Nalls</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140167484722320_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140167484722320_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140167484722320_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140167484722320_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Laboratory of Neurogenetics</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Laboratory of Epidemiology and Population Sciences, National Institute on Aging, National Institutes of Health, Bethesda, MD, USA</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Longitudinal Studies Section, Clinical Research Branch</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Behavioral Epidemiology Section, Laboratory of Epidemiology &amp; Population Sciences, National Institute on Aging Intramural Research Program, National Institutes of Health, Baltimore, MD, USA</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Laboratory of Genetics</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Health Disparities Research Section, Clinical Research Branch, National Institute on Aging, National Institutes of Health, Baltimore, MD, USA</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Biological Anthropology, Temple University, Philadelphia, PA, USA</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Epidemiology</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>Cardiovascular Health Research Unit</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Department of Medicine</div><div class="fm-affl" id="af11" lang="en"><sup>11</sup>Department of Epidemiology and Health Services, University of Washington, Seattle, WA, USA</div><div class="fm-affl" id="af12" lang="en"><sup>12</sup>Division of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA, USA</div><div class="fm-affl" id="af13" lang="en"><sup>13</sup>Laboratory for Statistical Analysis</div><div class="fm-affl" id="af14" lang="en"><sup>14</sup>Laboratory for Genotyping Development, RIKEN Center for Integrative Medical Sciences, Yokohama, Japan</div><div class="fm-affl" id="af15" lang="en"><sup>15</sup>Department of Human Genetics and Disease Diversity, Graduate School of Medical and Dental Sciences, Tokyo Medical and Dental University, Tokyo, Japan</div><div class="fm-affl" id="af16" lang="en"><sup>16</sup>Department of Epidemiology</div><div class="fm-affl" id="af17" lang="en"><sup>17</sup>ErasmusAGE, Department of Epidemiology</div><div class="fm-affl" id="af18" lang="en"><sup>18</sup>Department of Internal Medicine, Erasmus Medical Center, Rotterdam, The Netherlands</div><div class="fm-affl" id="af19" lang="en"><sup>19</sup>Consortium for Healthy Aging (NGI-NCHA), The Netherlands Genomics Initiative, Leiden, The Netherlands</div><div class="fm-affl" id="af20" lang="en"><sup>20</sup>Cardiovascular Epidemiology and Human Genomics Branch, NHLBI Division of Intramural Research, Bethesda, MD, USA</div><div class="fm-affl" id="af21" lang="en"><sup>21</sup>Department of Neurology, Boston University School of Medicine, Boston, MA, USA</div><div class="fm-affl" id="af22" lang="en"><sup>22</sup>NHLBI Framingham Heart Study, Bethesda, MD, USA</div><div class="fm-affl" id="af23" lang="en"><sup>23</sup>Icelandic Heart Association, Kopavogur, Iceland</div><div class="fm-affl" id="af24" lang="en"><sup>24</sup>University of Iceland, Reykjavik, Iceland</div><div class="fm-affl" id="af25" lang="en"><sup>25</sup>Genetic and Genomic Epidemiology Unit, Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford, UK</div><div class="fm-affl" id="af26" lang="en"><sup>26</sup>Department of Biostatistics, University of Liverpool, Liverpool, UK</div><div class="fm-affl" id="af27" lang="en"><sup>27</sup>Montreal Heart Institute, Montréal, Canada</div><div class="fm-affl" id="af28" lang="en"><sup>28</sup>Département de Médecine, Université de Montréal, Montréal, Canada</div><div class="fm-affl" id="af29" lang="en"><sup>29</sup>Geriatric Rehabilitation Unit, Azienda Sanitaria Firenze (ASF), Florence, Italy</div><div class="fm-affl" id="af30" lang="en"><sup>30</sup>GeneSTAR Research Program, Division of General Internal Medicine</div><div class="fm-affl" id="af31" lang="en"><sup>31</sup>Division of Cardiology, Johns Hopkins School of Medicine, Baltimore, MD, USA</div><div class="fm-affl" id="af32" lang="en"><sup>32</sup>Center for Applied Genomics, Children's Hospital of Philadelphia, PA, USA</div><div class="fm-affl" id="af33" lang="en"><sup>33</sup>Department of Pediatrics, University of Pennsylvania, PA, USA</div><div class="fm-affl" id="af34" lang="en"><sup>34</sup>Division of Cardiovascular Sciences, National Heart, Lung, and Blood Institute, Bethesda, MD, USA</div><div class="fm-affl" id="af35" lang="en"><sup>35</sup>Stanford University School of Medicine, Stanford, CA 94305, USA</div><div class="fm-affl" id="af36" lang="en"><sup>36</sup>The Brown Foundation, Institute of Molecular Medicine for the Prevention of Human Diseases, University of Texas, Houston, TX, USA</div><div class="fm-affl" id="af37" lang="en"><sup>37</sup>Laboratory of Molecular Medicine, Human Genome Center, Institute of Medical Science, The University of Tokyo, Tokyo, Japan</div><div class="fm-affl" id="af38" lang="en"><sup>38</sup>Department of Medicine</div><div class="fm-affl" id="af39" lang="en"><sup>39</sup>Department of Surgery, Center for Personalized Therapeutics, The University of Chicago, Chicago, IL, USA</div><div class="fm-affl" id="af40" lang="en"><sup>40</sup>Department of Epidemiology</div><div class="fm-affl" id="af41" lang="en"><sup>41</sup>Department of Medicine, Brown University, Providence, RI, USA</div><div class="fm-affl" id="af42" lang="en"><sup>42</sup>Group Health Research Institute, Group Health Cooperative, Seattle, WA, USA</div><div class="fm-affl" id="af43" lang="en"><sup>43</sup>Boston University Department of Statistics, Boston, MA, USA</div><div class="fm-affl" id="af44" lang="en"><sup>44</sup>Institute for Translational Genomics and Population Sciences, Los Angeles BioMedical Research Institute at Harbor-UCLA Medical Center, Torrance, CA, USA</div><div class="fm-affl" id="af45" lang="en"><sup>45</sup>Division of Genetic Outcomes, Department of Pediatrics, Harbor-UCLA Medical Center, Torrance, CA, USA</div><div class="fm-affl" id="af46" lang="en"><sup>46</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS, USA</div><div class="fm-affl" id="af47" lang="en"><sup>47</sup>Division of Cardiovascular Medicine, Department of Internal Medicine, University of Michigan, Ann Arbor, MI, USA</div><div class="fm-affl" id="af48" lang="en"><sup>48</sup>Department of Human Genetics, University of Michigan, Ann Arbor, MI, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Division of Cardiovascular Medicine, Department of Internal Medicine, University of Michigan, 1500 E. Medical Center Drive, Ann Arbor, MI 48109, USA. Tel: +1 7347644500; Fax: +1 7349368266; Email: <a href="mailto:dev@null" data-email="ude.hcimu.dem@hsenags" class="oemail">ude.hcimu.dem@hsenags</a>; Laboratory of Neurogenetics, Building 35, 1A-1014, 35 Convent Drive Bethesda, MD 20892, USA. Tel: 301 451 3831; E-mail: <a href="mailto:dev@null" data-email="vog.hin.liam@msllan" class="oemail">vog.hin.liam@msllan</a></div><div>Contributed by <sup>†</sup>These authors contributed equally to this work.</div><div>Contributed by <sup>‡</sup>These authors contributed equally to this work.</div><div>Contributed by <sup>¶</sup>A full list of collaborators from CHARGE, COGENT and RIKEN can be found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material</a>.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140167484722320_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Apr 25; Accepted 2014 Jul 31.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140167484722320_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> Published by Oxford University Press 2014. This work is written by (a) US Government employee(s) and is in the public domain in the US.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4245044/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4245044/bin/supp_23_25_6944__index.html" data-ga-action="click_feat_suppl">supp_23_25_6944__index.html</a><span style="color:gray"> (923 bytes)</span></div><div class="small guid">GUID:&nbsp;A450A528-44FC-4650-BA7D-9C4C340CB33B</div>
<div><a href="/pmc/articles/PMC4245044/bin/supp_ddu401_ddu401supp_data.pdf" data-ga-action="click_feat_suppl">supp_ddu401_ddu401supp_data.pdf</a><span style="color:gray"> (2.0M)</span></div><div class="small guid">GUID:&nbsp;46C45165-3AD9-4E2E-AB13-361B275B4AAB</div>
<div><a href="/pmc/articles/PMC4245044/bin/supp_ddu401_ddu401supp_data1.zip" data-ga-action="click_feat_suppl">supp_ddu401_ddu401supp_data1.zip</a><span style="color:gray"> (50M)</span></div><div class="small guid">GUID:&nbsp;0E7235A9-EA2D-416B-9616-BCC9C13634AB</div>
</div></dd></dl></div><div id="idm140167473436032" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140167473436032title">Abstract</h2><!--article-meta--><div><p id="__p4" class="p p-first-last">White blood cell (WBC) count is a common clinical measure used as a predictor of certain aspects of human health, including immunity and infection status. WBC count is also a complex trait that varies among individuals and ancestry groups. Differences in linkage disequilibrium structure and heterogeneity in allelic effects are expected to play a role in the associations observed between populations. Prior genome-wide association study (GWAS) meta-analyses have identified genomic loci associated with WBC and its subtypes, but much of the heritability of these phenotypes remains unexplained. Using GWAS summary statistics for over 50 000 individuals from three diverse populations (Japanese, African-American and European ancestry), a Bayesian model methodology was employed to account for heterogeneity between ancestry groups. This approach was used to perform a trans-ethnic meta-analysis of total WBC, neutrophil and monocyte counts. Ten previously known associations were replicated and six new loci were identified, including several regions harboring genes related to inflammation and immune cell function. Ninety-five percent credible interval regions were calculated to narrow the association signals and fine-map the putatively causal variants within loci. Finally, a conditional analysis was performed on the most significant SNPs identified by the trans-ethnic meta-analysis (MA), and nine secondary signals within loci previously associated with WBC or its subtypes were identified. This work illustrates the potential of trans-ethnic analysis and ascribes a critical role to multi-ethnic cohorts and consortia in exploring complex phenotypes with respect to variants that lie outside the European-biased GWAS pool.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p5" class="p p-first">White blood cells (WBCs) are critically involved in the body's immune system, serving as a primary defense mechanism against foreign pathogens. WBC count is used as a clinical marker of inflammation status, and higher WBC count has been associated with a risk of cardiovascular disease, cancer mortality and all-cause mortality (<a href="#DDU401C1" rid="DDU401C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793854">1</a>–<a href="#DDU401C3" rid="DDU401C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793810">3</a>). Elevated WBC count is also associated with disease risk factors including increasing age, high blood pressure, cigarette smoking, adiposity and increasing plasma inflammatory markers (<a href="#DDU401C4" rid="DDU401C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793779">4</a>).</p><p id="__p6">WBCs are classified into five subtypes according to their morphology and functions, including neutrophils, basophils, eosinophils, lymphocytes and monocytes. Total WBC count is highly variable even among healthy individuals of the same population (<a href="#DDU401C5" rid="DDU401C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793786">5</a>). WBC count is a moderately heritable phenotype, with <em>h</em><sup>2</sup> estimates ranging from 0.14 to 0.40 across the WBC subtypes (<a href="#DDU401C6" rid="DDU401C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793849">6</a>). Additionally, between 25 and 50% of individuals of African descent exhibit benign ethnic neutropenia, characterized by low neutrophil counts, due to a regulatory variant in the Duffy antigen receptor for chemokines (<em>DARC</em>) gene (<a href="#DDU401C5" rid="DDU401C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793766">5</a>,<a href="#DDU401C7" rid="DDU401C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793792">7</a>,<a href="#DDU401C8" rid="DDU401C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793824">8</a>). Given the importance of WBC in both host defense and, potentially, pathologic inflammation, elucidation of additional genetic mechanisms responsible for regulating while blood cell count could have a substantial medical impact.</p><p id="__p7">Admixture mapping and genome-wide association studies (GWAS) performed on cohorts of differing continental ancestry, including European, Japanese and African-American, have been successful in identifying multiple loci associated with WBC phenotypes (<a href="#DDU401C5" rid="DDU401C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793853">5</a>,<a href="#DDU401C8" rid="DDU401C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793837">8</a>–<a href="#DDU401C14" rid="DDU401C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793772">14</a>). The joint effects of these loci generally explain only a small portion of the overall heritability of either total WBC or WBC subtypes. Some prior GWAS have not defined the subtypes of WBC that are driving their observed associations; however, neutrophils in particular are often implicated (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793770">9</a>,<a href="#DDU401C11" rid="DDU401C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793834">11</a>). Furthermore, the loci identified by GWAS generally encompass large genomic regions, often containing many genes and variants with comparable association signals. Thus, fine-mapping methods aimed at pinpointing association signals more precisely are needed (<a href="#DDU401C15" rid="DDU401C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793797">15</a>).</p><p id="__p8">Recently, the 1000 Genomes Project and phase three of the HapMap project released comprehensive reference panels for a number of ethnic groups, including African, Asian and additional European populations (<a href="#DDU401C16" rid="DDU401C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793847">16</a>,<a href="#DDU401C17" rid="DDU401C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793851">17</a>). Imputation using these higher density reference panels allows inference of genotypes not captured by genotyping arrays, markedly increasing the breadth of genetic variation that can be included in association tests. This has provided new opportunities both to detect novel loci and to refine the localization of association signals for a number of phenotypes, including WBC phenotypes.</p><p id="__p9">Trans-ethnic meta-analysis (MA) potentially offers a more comprehensive view of the genetic variation that is associated with a trait, but traditional fixed-effects MA methods do not adequately address heterogeneity in allelic effects, allele frequencies or differences in linkage disequilibrium between ethnicities (<a href="#DDU401C18" rid="DDU401C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793821">18</a>). For example, in a previous fixed-effects analysis of the cohorts included in the current study, only 152 of 161 single nucleotide polymorphisms (SNPs) that had been associated with WBC phenotypes in earlier analyses were replicated at a Bonferroni-corrected significance threshold of <em>P</em> &lt; 3.57E−3, and no novel associations were observed (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793788">9</a>). Whereas random-effects-based methods of MA do account for inter-study heterogeneity, they lose statistical power in the setting of high levels of heterogeneity that may result from experimental or statistical differences in study design (<a href="#DDU401C15" rid="DDU401C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793833">15</a>,<a href="#DDU401C19" rid="DDU401C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793798">19</a>).</p><p id="__p10" class="p p-last">These shortcomings have been addressed in the software package, MANTRA (Meta-Analysis of Trans-ethnic Association studies), which allows for heterogeneity between diverse ethnic groups and provides increased power and mapping resolution compared with random-effects-based methods (<a href="#DDU401C15" rid="DDU401C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793809">15</a>). In the current study, we used MANTRA to combine summary results of ancestry-specific GWAS of WBC traits in three distinct populations. We identify novel loci associated with WBC count, assess heterogeneity in allelic effects between ancestry groups and improve fine-mapping resolution of some previously identified regions.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p11" class="p p-first">Descriptive statistics for each cohort are found in Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB1" rid-ob="ob-DDU401TB1" co-legend-rid=""><span>1</span></a>. In the trans-ethnic MANTRA analyses, we observed strong evidence of association, defined by a log<sub>10</sub> Bayes factor (BF) of &gt;6, at 10 previously identified loci and six novel loci, and detected nine secondary signals within 500 kb of a previously identified locus. The population-specific and trans-ethnic results for the established and novel loci associated with each WBC phenotype (total WBC, neutrophil and monocyte count) are summarized in Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB2" rid-ob="ob-DDU401TB2" co-legend-rid=""><span>2</span></a>. Cohort-level Manhattan plots are shown in Figure&nbsp;<a href="/pmc/articles/PMC4245044/figure/DDU401F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU401F1" rid-ob="ob-DDU401F1" co-legend-rid="lgnd_DDU401F1"><span>1</span></a>. Of the 15 previously identified variants (at 10 loci), six of the six monocyte associations, two of the four neutrophil associations and two of the five WBC count associations were initially identified in the original GWAS papers from which the data employed by this analysis are drawn.


</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU401TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p12">Descriptive statistics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167557049376" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Study</th><th rowspan="1" colspan="1" align="left">RIKEN</th><th rowspan="1" colspan="1" align="left">CHARGE</th><th rowspan="1" colspan="1" align="left">COGENT</th></tr><tr><th rowspan="1" colspan="1" align="left">Population ancestry</th><th rowspan="1" colspan="1" align="left">Japanese</th><th rowspan="1" colspan="1" align="left">European</th><th rowspan="1" colspan="1" align="left">African-American</th></tr></thead><tbody><tr><td colspan="4" rowspan="1">Total WBC</td></tr><tr><td rowspan="1" colspan="1"> WBC count: mean (SD) in cells × 10<sup>3</sup>/ml</td><td rowspan="1" colspan="1">6.20 (1.67), <em>n</em> = 16 843</td><td rowspan="1" colspan="1">5.72 (1.24), <em>n</em> = 19 509</td><td rowspan="1" colspan="1">5.94 (1.88), <em>n</em> ∼16 388</td></tr><tr><td rowspan="1" colspan="1"> Neutrophils: mean (SD) in cells × 10<sup>3</sup>/ml</td><td rowspan="1" colspan="1">3.80 (1.40), <em>n</em> = 9802</td><td rowspan="1" colspan="1">3.52 (1.06), <em>n</em> = 16 550</td><td rowspan="1" colspan="1">3.57 (1.57), <em>n</em> ∼7391</td></tr><tr><td rowspan="1" colspan="1"> Monocytes: mean (SD) in cells × 10<sup>3</sup>/ml</td><td rowspan="1" colspan="1">0.36 (0.22), <em>n</em> = 9810</td><td rowspan="1" colspan="1">0.43 (0.14), <em>n</em> = 16 550</td><td rowspan="1" colspan="1">0.36 (0.17), <em>n</em> ∼7369</td></tr><tr><td colspan="4" rowspan="1">Covariates</td></tr><tr><td rowspan="1" colspan="1"> Age in years: mean (SD)</td><td rowspan="1" colspan="1">63.5 (10.6)</td><td rowspan="1" colspan="1">63.8 (8.9)</td><td rowspan="1" colspan="1">50.9 (7.6)</td></tr><tr><td rowspan="1" colspan="1"> BMI: mean (SD) kg/m<sup>2</sup></td><td rowspan="1" colspan="1">23.1 (3.6)</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1"> % Female</td><td rowspan="1" colspan="1">39.1</td><td rowspan="1" colspan="1">53.5</td><td rowspan="1" colspan="1">65.6</td></tr><tr><td rowspan="1" colspan="1"> % Current smoker</td><td rowspan="1" colspan="1">52.6</td><td rowspan="1" colspan="1">17.6</td><td rowspan="1" colspan="1">NA</td></tr><tr><td colspan="4" rowspan="1">Sample size</td></tr><tr><td rowspan="1" colspan="1"> Total <em>N</em></td><td rowspan="1" colspan="1">17 218</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">16 388</td></tr></tbody></table></div><div id="largeobj_idm140167557049376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4245044/table/DDU401TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><p id="__p13" class="p p-first-last">Counts are reported in thousands of cells per milliliter of blood (cells × 10<sup>9</sup>/l).</p></div><div id="TN2"><p id="__p14" class="p p-first-last">SD, standard deviation; NA, not available.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU401TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p15">Loci defined in MANTRA analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167480962976" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Lead SNP</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">base pair (BP) (bd 37)</th><th rowspan="2" colspan="1" align="left">Effect</th><th rowspan="2" colspan="1" align="left">Other</th><th colspan="4" rowspan="1" align="center">MANTRA trans-ethnic meta-analysis<hr></th><th colspan="6" rowspan="1" align="center">European ancestry GWAS meta-analysis<hr></th><th colspan="6" rowspan="1" align="center">Japanese ancestry GWAS meta-analysis<hr></th><th colspan="6" rowspan="1" align="center">African-American ancestry GWAS meta-analysis<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">log<sub>10</sub> BF (bayes factor) assocation</th><th rowspan="1" colspan="1" align="left">PPA</th><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">Gene(s)</th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left"><em>R</em><sup>2</sup><sup>a</sup></th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left"><em>R</em><sup>2</sup></th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left"><em>R</em><sup>2</sup><sup>a</sup></th></tr></thead><tbody><tr><td colspan="27" rowspan="1">WBC count—established loci</td></tr><tr><td rowspan="1" colspan="1"> rs2518564*</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159 062 436</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">34 1.59</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">51 768</td><td rowspan="1" colspan="1">DARC</td><td rowspan="1" colspan="1">0.293</td><td rowspan="1" colspan="1">0.007</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">2.81E−02</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">0.883</td><td rowspan="1" colspan="1">0.7159</td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">0.9936</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.991</td><td rowspan="1" colspan="1">0.1725</td><td rowspan="1" colspan="1">0.1905</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">1.39E−149</td><td rowspan="1" colspan="1">15 416</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs1371799</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 977 837</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">13.5</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">CXCL2</td><td rowspan="1" colspan="1">0.572</td><td rowspan="1" colspan="1">−0.012</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">2.14E−05</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">0.985</td><td rowspan="1" colspan="1">0.461</td><td rowspan="1" colspan="1">−0.012</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">3.63E−05</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.921</td><td rowspan="1" colspan="1">0.761</td><td rowspan="1" colspan="1">0.023</td><td rowspan="1" colspan="1">0.0041</td><td rowspan="1" colspan="1">5.21E−04</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs4895441</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">135 426 573</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">9.64</td><td rowspan="1" colspan="1">0.165</td><td rowspan="1" colspan="1">52 703</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.347</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">3.42E−04</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.628</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">8.99E−11</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.902</td><td rowspan="1" colspan="1">0.0002</td><td rowspan="1" colspan="1">0.0059</td><td rowspan="1" colspan="1">9.90E−01</td><td rowspan="1" colspan="1">16 351</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">16.8</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">CDK6</td><td rowspan="1" colspan="1">0.776</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">4.51E−04</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.685</td><td rowspan="1" colspan="1">0.019</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">8.99E−11</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.193</td><td rowspan="1" colspan="1">−0.0249</td><td rowspan="1" colspan="1">0.0049</td><td rowspan="1" colspan="1">9.90E−01</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs4794822</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 156 712</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">29.64</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">PSMD3</td><td rowspan="1" colspan="1">0.586</td><td rowspan="1" colspan="1">−0.028</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">3.23E−23</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.481</td><td rowspan="1" colspan="1">−0.019</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">2.90E−12</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.325</td><td rowspan="1" colspan="1">0.0128</td><td rowspan="1" colspan="1">0.0038</td><td rowspan="1" colspan="1">4.33E−02</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">0.918</td></tr><tr><td colspan="27" rowspan="1">WBC count—novel loci</td></tr><tr><td rowspan="1" colspan="1"> rs10932765</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">219 099 484</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">6.95</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">ARPC2</td><td rowspan="1" colspan="1">0.503</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">3.36E−05</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">1.69E−05</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.986</td><td rowspan="1" colspan="1">0.877</td><td rowspan="1" colspan="1">0.0039</td><td rowspan="1" colspan="1">0.0056</td><td rowspan="1" colspan="1">7.26E−01</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs6734238</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">113 841 030</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">6.62</td><td rowspan="1" colspan="1">0.028</td><td rowspan="1" colspan="1">52 691</td><td rowspan="1" colspan="1">IL1F10</td><td rowspan="1" colspan="1">0.451</td><td rowspan="1" colspan="1">−0.009</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">1.46E−03</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.967</td><td rowspan="1" colspan="1">−0.018</td><td rowspan="1" colspan="1">0.008</td><td rowspan="1" colspan="1">2.49E−02</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.907</td><td rowspan="1" colspan="1">0.553</td><td rowspan="1" colspan="1">−0.0158</td><td rowspan="1" colspan="1">0.0035</td><td rowspan="1" colspan="1">5.92E−03</td><td rowspan="1" colspan="1">16 339</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs2853946</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">31 247 203</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">12.16</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">HLA-B</td><td rowspan="1" colspan="1">0.348</td><td rowspan="1" colspan="1">−0.017</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">1.28E−08</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.717</td><td rowspan="1" colspan="1">−0.013</td><td rowspan="1" colspan="1">0.003</td><td rowspan="1" colspan="1">8.19E−06</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.976</td><td rowspan="1" colspan="1">0.512</td><td rowspan="1" colspan="1">−0.0106</td><td rowspan="1" colspan="1">0.0035</td><td rowspan="1" colspan="1">7.65E−02</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs2163950</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 597 585</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">7.75</td><td rowspan="1" colspan="1">0.065</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.054</td><td rowspan="1" colspan="1">−0.017</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">5.91E−03</td><td rowspan="1" colspan="1">19 509</td><td rowspan="1" colspan="1">0.861</td><td rowspan="1" colspan="1">0.174</td><td rowspan="1" colspan="1">−0.021</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">5.36E−09</td><td rowspan="1" colspan="1">16 843</td><td rowspan="1" colspan="1">0.995</td><td rowspan="1" colspan="1">0.124</td><td rowspan="1" colspan="1">−0.0053</td><td rowspan="1" colspan="1">0.0058</td><td rowspan="1" colspan="1">6.44E−01</td><td rowspan="1" colspan="1">16 388</td><td rowspan="1" colspan="1">0.788</td></tr><tr><td colspan="27" rowspan="1">Monocyte count—established loci</td></tr><tr><td rowspan="1" colspan="1"> rs1449263</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">182 319 301</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">19.24</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 729</td><td rowspan="1" colspan="1">ITGA4</td><td rowspan="1" colspan="1">0.541</td><td rowspan="1" colspan="1">−0.036</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">6.70E−14</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.942</td><td rowspan="1" colspan="1">0.626</td><td rowspan="1" colspan="1">−0.033</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">8.32E−08</td><td rowspan="1" colspan="1">9,810</td><td rowspan="1" colspan="1">0.994</td><td rowspan="1" colspan="1">0.58</td><td rowspan="1" colspan="1">0.0087</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">4.68E−03</td><td rowspan="1" colspan="1">7369</td><td rowspan="1" colspan="1">0.965</td></tr><tr><td rowspan="1" colspan="1"> rs12988934</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">182 323 665</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">7.75</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">26 360</td><td rowspan="1" colspan="1">ITGA4</td><td rowspan="1" colspan="1">0.789</td><td rowspan="1" colspan="1">−0.034</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">1.01E−03</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.511</td><td rowspan="1" colspan="1">0.732</td><td rowspan="1" colspan="1">−0.041</td><td rowspan="1" colspan="1">0.008</td><td rowspan="1" colspan="1">1.42E−07</td><td rowspan="1" colspan="1">9810</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1">−0.0165</td><td rowspan="1" colspan="1">0.0854</td><td rowspan="1" colspan="1">8.47E−01</td><td rowspan="1" colspan="1">1803</td><td rowspan="1" colspan="1">0.418</td></tr><tr><td rowspan="1" colspan="1"> rs9880192</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">128 297 569</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">8.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 745</td><td rowspan="1" colspan="1">C3orf27</td><td rowspan="1" colspan="1">0.441</td><td rowspan="1" colspan="1">−0.028</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.35E−08</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.867</td><td rowspan="1" colspan="1">0.091</td><td rowspan="1" colspan="1">−0.037</td><td rowspan="1" colspan="1">0.012</td><td rowspan="1" colspan="1">1.21E−03</td><td rowspan="1" colspan="1">9810</td><td rowspan="1" colspan="1">0.804</td><td rowspan="1" colspan="1">0.188</td><td rowspan="1" colspan="1">0.0037</td><td rowspan="1" colspan="1">0.0047</td><td rowspan="1" colspan="1">4.31E−01</td><td rowspan="1" colspan="1">7385</td><td rowspan="1" colspan="1">0.674</td></tr><tr><td rowspan="1" colspan="1"> rs3095254</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">31 221 668</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">6.81</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 745</td><td rowspan="1" colspan="1">MHC</td><td rowspan="1" colspan="1">0.377</td><td rowspan="1" colspan="1">0.008</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.19E−01</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.976</td><td rowspan="1" colspan="1">0.461</td><td rowspan="1" colspan="1">0.035</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">8.27E−09</td><td rowspan="1" colspan="1">9810</td><td rowspan="1" colspan="1">0.896</td><td rowspan="1" colspan="1">0.562</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">3.31E−01</td><td rowspan="1" colspan="1">7385</td><td rowspan="1" colspan="1">0.935</td></tr><tr><td rowspan="1" colspan="1"> rs1991866</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 624 105</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">15.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 598</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.451</td><td rowspan="1" colspan="1">−0.032</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">4.58E−11</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">0.598</td><td rowspan="1" colspan="1">−0.034</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">7.38E−08</td><td rowspan="1" colspan="1">9810</td><td rowspan="1" colspan="1">0.892</td><td rowspan="1" colspan="1">0.52</td><td rowspan="1" colspan="1">−0.0046</td><td rowspan="1" colspan="1">0.0031</td><td rowspan="1" colspan="1">1.32E−01</td><td rowspan="1" colspan="1">7238</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs10980800</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">113 915 905</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">11.63</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 692</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.769</td><td rowspan="1" colspan="1">0.044</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">1.10E−14</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.988</td><td rowspan="1" colspan="1">0.056</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">0.013</td><td rowspan="1" colspan="1">7.61E−01</td><td rowspan="1" colspan="1">9810</td><td rowspan="1" colspan="1">0.941</td><td rowspan="1" colspan="1">0.787</td><td rowspan="1" colspan="1">−0.0049</td><td rowspan="1" colspan="1">0.0037</td><td rowspan="1" colspan="1">1.85E−01</td><td rowspan="1" colspan="1">7332</td><td rowspan="1" colspan="1">1</td></tr><tr><td colspan="27" rowspan="1">Monocyte count—novel loci</td></tr><tr><td rowspan="1" colspan="1"> rs2047076</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">76 058 509</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">6.03</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 729</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">0.714</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">0.006</td><td rowspan="1" colspan="1">3.80E−01</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.9998</td><td rowspan="1" colspan="1">2.678</td><td rowspan="1" colspan="1">0.474</td><td rowspan="1" colspan="1">1.64E−08</td><td rowspan="1" colspan="1">9,810</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.037</td><td rowspan="1" colspan="1">0.0023</td><td rowspan="1" colspan="1">0.0091</td><td rowspan="1" colspan="1">7.97E−01</td><td rowspan="1" colspan="1">7369</td><td rowspan="1" colspan="1">0.789</td></tr><tr><td colspan="27" rowspan="1">Neutrophil count—established loci</td></tr><tr><td rowspan="1" colspan="1"> rs7667376</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 967 890</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">11.13</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 743</td><td rowspan="1" colspan="1">CXCL2</td><td rowspan="1" colspan="1">0.596</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">1.47E−04</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.972</td><td rowspan="1" colspan="1">0.542</td><td rowspan="1" colspan="1">0.017</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">5.96E−04</td><td rowspan="1" colspan="1">9802</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.221</td><td rowspan="1" colspan="1">−0.0428</td><td rowspan="1" colspan="1">0.008</td><td rowspan="1" colspan="1">8.50E−04</td><td rowspan="1" colspan="1">7391</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">10.52</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 744</td><td rowspan="1" colspan="1">CDK6</td><td rowspan="1" colspan="1">0.776</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">1.21E−02</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.685</td><td rowspan="1" colspan="1">0.024</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">8.96E−06</td><td rowspan="1" colspan="1">9802</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.203</td><td rowspan="1" colspan="1">−0.0479</td><td rowspan="1" colspan="1">0.0092</td><td rowspan="1" colspan="1">1.15E−03</td><td rowspan="1" colspan="1">7392</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs8078723</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 166 879</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">28.06</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 693</td><td rowspan="1" colspan="1">CSF3, MED24</td><td rowspan="1" colspan="1">0.626</td><td rowspan="1" colspan="1">0.043</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">2.84E−23</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.997</td><td rowspan="1" colspan="1">0.519</td><td rowspan="1" colspan="1">0.032</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">1.39E−10</td><td rowspan="1" colspan="1">9802</td><td rowspan="1" colspan="1">0.995</td><td rowspan="1" colspan="1">0.678</td><td rowspan="1" colspan="1">−0.0071</td><td rowspan="1" colspan="1">0.0073</td><td rowspan="1" colspan="1">6.19E−01</td><td rowspan="1" colspan="1">7341</td><td rowspan="1" colspan="1">1</td></tr><tr><td rowspan="1" colspan="1"> rs4794822</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 156 712</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">28.92</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 753</td><td rowspan="1" colspan="1">PSMD3</td><td rowspan="1" colspan="1">0.586</td><td rowspan="1" colspan="1">−0.043</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">3.64E−23</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.481</td><td rowspan="1" colspan="1">−0.032</td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">7.09E−11</td><td rowspan="1" colspan="1">9802</td><td rowspan="1" colspan="1">0.999</td><td rowspan="1" colspan="1">0.325</td><td rowspan="1" colspan="1">0.0048</td><td rowspan="1" colspan="1">0.0073</td><td rowspan="1" colspan="1">7.41E−01</td><td rowspan="1" colspan="1">7401</td><td rowspan="1" colspan="1">0.918</td></tr><tr><td colspan="27" rowspan="1">Neutrophil count—novel loci</td></tr><tr><td rowspan="1" colspan="1"> rs6936204</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 217 092</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">6.45</td><td rowspan="1" colspan="1">0.048</td><td rowspan="1" colspan="1">33 706</td><td rowspan="1" colspan="1"><a id="__tag_705775135" class="tag_hotlink" href="/nuccore/AK123889" ref="/nuccore/AK123889"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK123889","term_id":"34529536","term_text":"AK123889"}}</span>AK123889</a></td><td rowspan="1" colspan="1">0.585</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.004</td><td rowspan="1" colspan="1">5.83E−06</td><td rowspan="1" colspan="1">16 550</td><td rowspan="1" colspan="1">0.991</td><td rowspan="1" colspan="1">0.912</td><td rowspan="1" colspan="1">0.032</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">3.97E−04</td><td rowspan="1" colspan="1">9802</td><td rowspan="1" colspan="1">0.983</td><td rowspan="1" colspan="1">0.279</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.0074</td><td rowspan="1" colspan="1">4.68E−01</td><td rowspan="1" colspan="1">7354</td><td rowspan="1" colspan="1">1</td></tr></tbody></table></div><div id="largeobj_idm140167480962976" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/table/DDU401TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN3"><p id="__p16" class="p p-first-last">Strong evidence for association is defined as a BF of &gt;6. Differences in effect alleles between cohorts were corrected before the MANTRA analysis.</p></div><div id="TN4"><p id="__p17" class="p p-first-last">PPA, posterior probability of association; SE, standard error; EAF, effect allele frequency.</p></div><div id="TN5"><p id="__p18" class="p p-first-last"><sup>a</sup>Consortia summary of imputation qualtiy score.</p></div><div id="TN6"><p id="__p19" class="p p-first-last">*Proxy for rs2814778.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU401F1" co-legend-rid="lgnd_DDU401F1"><a href="/pmc/articles/PMC4245044/figure/DDU401F1/" target="figure" rid-figpopup="DDU401F1" rid-ob="ob-DDU401F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167479991376" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F1/" target="figure" rid-figpopup="DDU401F1" rid-ob="ob-DDU401F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40101.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40101.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40101.jpg"></a></div><div id="largeobj_idm140167479991376" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F1"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F1/" target="figure" rid-figpopup="DDU401F1" rid-ob="ob-DDU401F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p20">Manhattan plots subset by WBC subtype and cohort. Horizontal axis indicates the chromosomal position. Vertical axis for the blue Manhattan plots indicates −log<sub>10</sub>
<em>P</em>-values from fixed-effects meta-analysis, and vertical axis for the black Manhattan plots indicates BFs from trans-ethnic meta-analysis. Significance values are truncated at 30 on the <em>y</em>-axis for clarity and scaling of images. A 50 Mb region of apparent significance surrounds the centromere of chromosome 1 and suggests two distinct peaks; however, this results from the lack of genotyped or imputed SNPs in the region and is a spatial inflation of a truly causal variant in the nearby <em>DARC</em> gene [Nalls <em>et al</em>. (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793807">9</a>) and Reiner <em>et al</em>. (<a href="#DDU401C12" rid="DDU401C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793848">12</a>)].</p></div></div></div><p id="__p21" class="p">Regions previously identified by single-ethnicity GWAS reappeared in the MANTRA trans-ethnic analyses, but in some instances, the index SNP from the original publication was not the most significant. These include rs4065321 and rs17609240 on 17q21.1 (WBC count and neutrophil count), rs2517524 on 6p21.33 (WBC count) and rs10956483 on 8q24.21 (monocyte count). Additionally, rs2814778, a marker identifying the Duffy null blood group antigen and located on the <em>DARC</em> gene of chromosome 1 at position (b37) 159 174 683, was available only in the COGENT data. This marker accounts for 20% of population variance in the WBC of African ancestry populations and is monomorphic in non-African populations.</p><div id="s2a" class="sec"><h3 id="s2atitle">Novel associations</h3><p id="__p22" class="p p-first">In addition to replicating known variants, the trans-ethnic analysis identified six novel trait–locus associations (Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB2" rid-ob="ob-DDU401TB2" co-legend-rid=""><span>2</span></a>). For neutrophil counts, novel findings include rs6936204 in region 6p21.32, located nearest to <em><a id="__tag_705775134" class="tag_hotlink" href="/nuccore/AK123889" ref="/nuccore/AK123889"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK123889","term_id":"34529536","term_text":"AK123889"}}</span>AK123889</a></em>. This region is very near known locus 6p21.33, which was previously associated with WBC and lymphocyte counts (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793790">9</a>). These loci are near the <em>HLA</em> region; thus, it is possible that population stratification is driving this association (<a href="#DDU401C20" rid="DDU401C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793839">20</a>). These variants are not in linkage disequilibrium (LD) with any of the known <em>HLA</em> markers, but as meiotic crossovers are known to cluster around <em>HLA</em>, it is possible that these variants are separated from this region by a recombination hotspot. Novel association for WBC count includes rs10932765 in region 2q35, located near <em>ARPC2</em>, which has previously been associated with monocyte count and inflammatory bowel disease (<a href="#DDU401C12" rid="DDU401C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793767">12</a>,<a href="#DDU401C21" rid="DDU401C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793800">21</a>). This region is notable because of its proximity to <em>IL8RA</em>, which encodes <em>CXCR1. CXCR1</em>, the receptor for the chemokine <em>IL-8</em>, is a mediator of inflammatory responses; interestingly, the Duffy antigen is also a receptor for <em>IL-8</em>. Additional novel associations for WBC include rs2163950 at 8q24.21, located in an intergenic region, and rs6734238 in region 2q13, located near <em>IL1F10</em> and <em>IL1RN.</em> Top SNP rs6734238 tags the <em>IL-1</em> gene family locus near 2q13 and has also been associated with C-reactive protein levels in a European ancestry population (<a href="#DDU401C22" rid="DDU401C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793803">22</a>). Notably, a large region of the implicated chromosome 6 associations for WBC, tagged by rs2853946, contains an apparently bimodal signal of association, between 310.00 and 313.00 Mb (Fig.&nbsp;<a href="/pmc/articles/PMC4245044/figure/DDU401F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU401F2" rid-ob="ob-DDU401F2" co-legend-rid="lgnd_DDU401F2"><span>2</span></a>). A single novel association for monocyte count was identified by rs2047076 in region 5q13.3, also within an intergenic region of the genome. This region falls between <em>F2R</em>, associated with platelet count (<a href="#DDU401C23" rid="DDU401C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793763">23</a>), and <em>F2RL1</em> which encodes PAR-2, a monocyte receptor (<a href="#DDU401C24" rid="DDU401C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793799">24</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU401F2" co-legend-rid="lgnd_DDU401F2"><a href="/pmc/articles/PMC4245044/figure/DDU401F2/" target="figure" rid-figpopup="DDU401F2" rid-ob="ob-DDU401F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167486885120" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F2/" target="figure" rid-figpopup="DDU401F2" rid-ob="ob-DDU401F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40102.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40102.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40102.jpg"></a></div><div id="largeobj_idm140167486885120" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F2"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F2/" target="figure" rid-figpopup="DDU401F2" rid-ob="ob-DDU401F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p23">Locus plot for Monocyte association on chromosome 5. Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p id="__p24">Variants showing strong evidence for association were examined for heterogeneity in their allelic effects across ancestries, indicated by a posterior probability of heterogeneity (PPH) of &gt;0.5 (Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB3" rid-ob="ob-DDU401TB3" co-legend-rid=""><span>3</span></a>). For the novel monocyte-associated locus on chromosome 5, the RIKEN cohort has a large posterior mean allelic effect (PMAE), whereas the COGENT and CHARGE cohorts have PMAE that are much smaller. This suggests that the association may be specific to the Japanese population, or that the variant tagging this region, rs2047076, may not be a good proxy among European or African-American individuals because of differing LD structure between these populations. The associations reported on chromosome 4 for both WBC and neutrophils exhibit allelic effects in opposite directions between the ancestry groups, which could reflect multiple risk variants, or differing LD structure (<a href="#DDU401C1" rid="DDU401C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793812">1</a>). The novel associations reported on chromosome 2 for WBC have a posterior probability of association (PPA) of ∼0.03, and the PMAE of these variants are similar across the three ancestries (∼0.01). PPA is estimated from the weighted average of the alternative models and accounts for the differences in likelihood and statistical power between tests (<a href="#DDU401C25" rid="DDU401C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793819">25</a>). Locus plots for novel associations are shown in Figures&nbsp;<a href="/pmc/articles/PMC4245044/figure/DDU401F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU401F2" rid-ob="ob-DDU401F2" co-legend-rid="lgnd_DDU401F2"><span>2</span></a>–<a href="/pmc/articles/PMC4245044/figure/DDU401F7/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU401F7" rid-ob="ob-DDU401F7" co-legend-rid="lgnd_DDU401F7"><span>7</span></a>, and locus plots for known associations are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figures S1–S11</a>.

<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm clearfix" id="DDU401F3" co-legend-rid="lgnd_DDU401F3"><a href="/pmc/articles/PMC4245044/figure/DDU401F3/" target="figure" rid-figpopup="DDU401F3" rid-ob="ob-DDU401F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167483203184" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F3/" target="figure" rid-figpopup="DDU401F3" rid-ob="ob-DDU401F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40103.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40103.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40103.jpg"></a></div><div id="largeobj_idm140167483203184" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F3"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F3/" target="figure" rid-figpopup="DDU401F3" rid-ob="ob-DDU401F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p29">Locus plot for Neutrophil association on chromosome 6. Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p>
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm clearfix" id="DDU401F4" co-legend-rid="lgnd_DDU401F4"><a href="/pmc/articles/PMC4245044/figure/DDU401F4/" target="figure" rid-figpopup="DDU401F4" rid-ob="ob-DDU401F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167520385712" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F4/" target="figure" rid-figpopup="DDU401F4" rid-ob="ob-DDU401F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40104.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40104.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40104.jpg"></a></div><div id="largeobj_idm140167520385712" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F4"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F4/" target="figure" rid-figpopup="DDU401F4" rid-ob="ob-DDU401F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p30">Locus plot for White Blood Cell count association on chromosome 2 (first region). Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p>
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm clearfix" id="DDU401F5" co-legend-rid="lgnd_DDU401F5"><a href="/pmc/articles/PMC4245044/figure/DDU401F5/" target="figure" rid-figpopup="DDU401F5" rid-ob="ob-DDU401F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167520382480" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F5/" target="figure" rid-figpopup="DDU401F5" rid-ob="ob-DDU401F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40105.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40105.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40105.jpg"></a></div><div id="largeobj_idm140167520382480" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F5"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F5/" target="figure" rid-figpopup="DDU401F5" rid-ob="ob-DDU401F5">Figure&nbsp;5.</a></div><!--caption a7--><div class="caption"><p id="__p31">Locus plot for White Blood Cell count association on chromosome 2 (second region). Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p>
<!--fig ft0--><!--fig mode=article f1--></p><div class="fig iconblock whole_rhythm clearfix" id="DDU401F6" co-legend-rid="lgnd_DDU401F6"><a href="/pmc/articles/PMC4245044/figure/DDU401F6/" target="figure" rid-figpopup="DDU401F6" rid-ob="ob-DDU401F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167484602864" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F6/" target="figure" rid-figpopup="DDU401F6" rid-ob="ob-DDU401F6"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40106.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40106.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40106.jpg"></a></div><div id="largeobj_idm140167484602864" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F6/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F6"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F6/" target="figure" rid-figpopup="DDU401F6" rid-ob="ob-DDU401F6">Figure&nbsp;6.</a></div><!--caption a7--><div class="caption"><p id="__p32">Locus plot for White Blood Cell count association on chromosome 6. Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p>
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU401TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p25">Heterogeneity of allelic effects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167468888560" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Subtype</th><th colspan="3" rowspan="1" align="center">Variant details<hr></th><th colspan="2" rowspan="1" align="center">MANTRA MA<hr></th><th colspan="2" rowspan="1" align="center">CHARGE<hr></th><th colspan="2" rowspan="1" align="center">RIKEN<hr></th><th colspan="2" rowspan="1" align="center">COGENT<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position (b37)</th><th rowspan="1" colspan="1" align="left">log<sub>10</sub> (BF)</th><th rowspan="1" colspan="1" align="left">PPH</th><th rowspan="1" colspan="1" align="left">PMAE</th><th rowspan="1" colspan="1" align="left">PSD</th><th rowspan="1" colspan="1" align="left">PMAE</th><th rowspan="1" colspan="1" align="left">PSD</th><th rowspan="1" colspan="1" align="left">PMAE</th><th rowspan="1" colspan="1" align="left">PSD</th></tr></thead><tbody><tr><td colspan="12" rowspan="1">Novel</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs10932765</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">219 099 484</td><td rowspan="1" colspan="1">6.95</td><td rowspan="1" colspan="1">0.025</td><td rowspan="1" colspan="1">0.01068</td><td rowspan="1" colspan="1">0.00182</td><td rowspan="1" colspan="1">0.01068</td><td rowspan="1" colspan="1">0.00182</td><td rowspan="1" colspan="1">0.01057</td><td rowspan="1" colspan="1">0.00205</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs6734238</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">113 841 030</td><td rowspan="1" colspan="1">6.62</td><td rowspan="1" colspan="1">0.028</td><td rowspan="1" colspan="1">−0.01193</td><td rowspan="1" colspan="1">0.00244</td><td rowspan="1" colspan="1">−0.01181</td><td rowspan="1" colspan="1">0.00213</td><td rowspan="1" colspan="1">−0.01195</td><td rowspan="1" colspan="1">0.00227</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs1371799</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 977 837</td><td rowspan="1" colspan="1">13.5</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.01164</td><td rowspan="1" colspan="1">0.00207</td><td rowspan="1" colspan="1">−0.01163</td><td rowspan="1" colspan="1">0.00207</td><td rowspan="1" colspan="1">0.02273</td><td rowspan="1" colspan="1">0.00408</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs2853946</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">31 247 203</td><td rowspan="1" colspan="1">12.16</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">−0.01379</td><td rowspan="1" colspan="1">0.00179</td><td rowspan="1" colspan="1">−0.01379</td><td rowspan="1" colspan="1">0.00179</td><td rowspan="1" colspan="1">−0.01378</td><td rowspan="1" colspan="1">0.00181</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs2163950</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 597 585</td><td rowspan="1" colspan="1">7.75</td><td rowspan="1" colspan="1">0.065</td><td rowspan="1" colspan="1">−0.01688</td><td rowspan="1" colspan="1">0.00287</td><td rowspan="1" colspan="1">−0.01655</td><td rowspan="1" colspan="1">0.00304</td><td rowspan="1" colspan="1">−0.01614</td><td rowspan="1" colspan="1">0.00346</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs2047076</td><td rowspan="1" colspan="1">5</td><td rowspan="1" colspan="1">76 058 509</td><td rowspan="1" colspan="1">6.03</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">2.50643</td><td rowspan="1" colspan="1">0.47953</td><td rowspan="1" colspan="1">0.00458</td><td rowspan="1" colspan="1">0.00498</td><td rowspan="1" colspan="1">0.00457</td><td rowspan="1" colspan="1">0.00495</td></tr><tr><td rowspan="1" colspan="1"> NEU</td><td rowspan="1" colspan="1">rs7667376</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 967 890</td><td rowspan="1" colspan="1">11.13</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.011</td><td rowspan="1" colspan="1">0.00199</td><td rowspan="1" colspan="1">0.01104</td><td rowspan="1" colspan="1">0.00199</td><td rowspan="1" colspan="1">−0.02275</td><td rowspan="1" colspan="1">0.00405</td></tr><tr><td rowspan="1" colspan="1"> NEU</td><td rowspan="1" colspan="1">rs6936204</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">32 217 092</td><td rowspan="1" colspan="1">6.45</td><td rowspan="1" colspan="1">0.048</td><td rowspan="1" colspan="1">0.01028</td><td rowspan="1" colspan="1">0.00356</td><td rowspan="1" colspan="1">0.0096</td><td rowspan="1" colspan="1">0.00259</td><td rowspan="1" colspan="1">0.00681</td><td rowspan="1" colspan="1">0.00477</td></tr><tr><td colspan="12" rowspan="1">Known</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs2518564*</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">159 062 436</td><td rowspan="1" colspan="1">341.59</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.00318</td><td rowspan="1" colspan="1">0.00241</td><td rowspan="1" colspan="1">0.00363</td><td rowspan="1" colspan="1">0.00258</td><td rowspan="1" colspan="1">0.19017</td><td rowspan="1" colspan="1">0.00469</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs4895441</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">135 426 573</td><td rowspan="1" colspan="1">9.64</td><td rowspan="1" colspan="1">0.165</td><td rowspan="1" colspan="1">0.01368</td><td rowspan="1" colspan="1">0.00248</td><td rowspan="1" colspan="1">0.01282</td><td rowspan="1" colspan="1">0.00239</td><td rowspan="1" colspan="1">0.01169</td><td rowspan="1" colspan="1">0.00446</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">16.8</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.01905</td><td rowspan="1" colspan="1">0.00254</td><td rowspan="1" colspan="1">0.01899</td><td rowspan="1" colspan="1">0.00297</td><td rowspan="1" colspan="1">−0.02476</td><td rowspan="1" colspan="1">0.00476</td></tr><tr><td rowspan="1" colspan="1"> WBC</td><td rowspan="1" colspan="1">rs4794822</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 156 712</td><td rowspan="1" colspan="1">29.64</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.02088</td><td rowspan="1" colspan="1">0.00309</td><td rowspan="1" colspan="1">−0.02491</td><td rowspan="1" colspan="1">0.00337</td><td rowspan="1" colspan="1">0.01249</td><td rowspan="1" colspan="1">0.00391</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs1449263</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">182 319 301</td><td rowspan="1" colspan="1">19.24</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.03449</td><td rowspan="1" colspan="1">0.00399</td><td rowspan="1" colspan="1">−0.03484</td><td rowspan="1" colspan="1">0.00378</td><td rowspan="1" colspan="1">0.00857</td><td rowspan="1" colspan="1">0.00316</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs12988934</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">182 323 665</td><td rowspan="1" colspan="1">7.75</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">−0.03872</td><td rowspan="1" colspan="1">0.00621</td><td rowspan="1" colspan="1">−0.03853</td><td rowspan="1" colspan="1">0.00627</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs9880192</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">128 297 569</td><td rowspan="1" colspan="1">8.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.03049</td><td rowspan="1" colspan="1">0.00687</td><td rowspan="1" colspan="1">−0.02917</td><td rowspan="1" colspan="1">0.0046</td><td rowspan="1" colspan="1">0.00315</td><td rowspan="1" colspan="1">0.00483</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs3095254</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">31 221 668</td><td rowspan="1" colspan="1">6.81</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.03449</td><td rowspan="1" colspan="1">0.00622</td><td rowspan="1" colspan="1">0.00473</td><td rowspan="1" colspan="1">0.00306</td><td rowspan="1" colspan="1">0.00439</td><td rowspan="1" colspan="1">0.00267</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs1991866</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 624 105</td><td rowspan="1" colspan="1">15.3</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.0322</td><td rowspan="1" colspan="1">0.00418</td><td rowspan="1" colspan="1">−0.03217</td><td rowspan="1" colspan="1">0.00392</td><td rowspan="1" colspan="1">−0.00466</td><td rowspan="1" colspan="1">0.00316</td></tr><tr><td rowspan="1" colspan="1"> MONO</td><td rowspan="1" colspan="1">rs10980800</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">113 915 905</td><td rowspan="1" colspan="1">11.63</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.0087</td><td rowspan="1" colspan="1">0.01581</td><td rowspan="1" colspan="1">0.043</td><td rowspan="1" colspan="1">0.00581</td><td rowspan="1" colspan="1">−0.00475</td><td rowspan="1" colspan="1">0.00379</td></tr><tr><td rowspan="1" colspan="1"> NEU</td><td rowspan="1" colspan="1">rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">10.52</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.01905</td><td rowspan="1" colspan="1">0.00254</td><td rowspan="1" colspan="1">0.01899</td><td rowspan="1" colspan="1">0.00297</td><td rowspan="1" colspan="1">−0.02476</td><td rowspan="1" colspan="1">0.00476</td></tr><tr><td rowspan="1" colspan="1"> NEU</td><td rowspan="1" colspan="1">rs8078723</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 166 879</td><td rowspan="1" colspan="1">28.06</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">0.01851</td><td rowspan="1" colspan="1">0.00278</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">−0.01105</td><td rowspan="1" colspan="1">0.00378</td></tr><tr><td rowspan="1" colspan="1"> NEU</td><td rowspan="1" colspan="1">rs4794822</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 156 712</td><td rowspan="1" colspan="1">28.92</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">−0.02088</td><td rowspan="1" colspan="1">0.00309</td><td rowspan="1" colspan="1">−0.02491</td><td rowspan="1" colspan="1">0.00337</td><td rowspan="1" colspan="1">0.01249</td><td rowspan="1" colspan="1">0.00391</td></tr></tbody></table></div><div id="largeobj_idm140167468888560" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/table/DDU401TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN7"><p id="__p26" class="p p-first-last">Variants encompassing the 95% credible region of an associated region, as identified using the top hits from the MANTRA analysis, are presented for each subtype. Ancestry-specific posterior mean allelic effects (PMAE) are reported. PSD is the Bayesian equivalent of standard error and characterizes the variance of the effect. PMAE is the posterior mean allelic effect; when these values are similar between ancestry cohorts, it suggests that simiar variants are responsible for the effect. When values are in opposite directions, it suggests multiple risk variants, or differing LD structure.</p></div><div id="TN8"><p id="__p27" class="p p-first-last">PPH, a posterior probability of heterogeneity; PPA, posterior probability of association; the probability that an SNP is truly associated with a phenotype.</p></div><div id="TN9"><p id="__p28" class="p p-first-last">*Proxy for rs2814778.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU401F7" co-legend-rid="lgnd_DDU401F7"><a href="/pmc/articles/PMC4245044/figure/DDU401F7/" target="figure" rid-figpopup="DDU401F7" rid-ob="ob-DDU401F7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167484599648" class="figure"><a href="/pmc/articles/PMC4245044/figure/DDU401F7/" target="figure" rid-figpopup="DDU401F7" rid-ob="ob-DDU401F7"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4245044_ddu40107.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu40107.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4245044/bin/ddu40107.jpg"></a></div><div id="largeobj_idm140167484599648" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/figure/DDU401F7/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU401F7"><div><a class="figpopup" href="/pmc/articles/PMC4245044/figure/DDU401F7/" target="figure" rid-figpopup="DDU401F7" rid-ob="ob-DDU401F7">Figure&nbsp;7.</a></div><!--caption a7--><div class="caption"><p id="__p33">Locus plot for White Blood Cell count association on chromosome 8. Vertical axis indicates BF, and horizontal axes indicate both chromosomal position and gene location.</p></div></div></div><p id="__p34">To determine how consistent the MANTRA results are across studies, a random-effects MA was also performed using METAL. The results are largely supportive of the known and novel loci reported here, with the exception of those variants with high levels of heterogeneity between populations, such as the novel variant associated with monocytes and located on chromosome 5, rs2047076. The results of this additional MA are found in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>.</p><p id="__p35" class="p p-last">Cohort-level data were imputed before the trans-ethnic analysis was performed. In instances where a novel variant was both imputed across multiple cohorts and exhibits low allele frequencies within those cohorts, additional replication is needed to validate the associations made here. In particular, these include rs2047076, associated with monocytes, and rs2163950, associated with WBC count (Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB2" rid-ob="ob-DDU401TB2" co-legend-rid=""><span>2</span></a>). Without replication, these novel results should be viewed cautiously. We are optimistic that in time, a similarly sized, ancestry-matched cohort will be available to replicate these analyses. In the meantime, we have provided genome-wide summary statistics for the primary MANTRA analyses of monocyte, neutrophil and WBC count traits in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Fine mapping</h3><p id="__p36" class="p p-first">Credible sets were defined to assess the extent to which the trans-ethnic analysis improved fine-mapping resolution of known associations. Credible region summary data for associated loci are presented in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>.</p><p id="__p37">The major locus affecting WBC levels in African-Americans is located in the <em>DARC</em> gene of chromosome 1, spanning 900 000 bp between 158 724 683 and 159 624 683 (<a href="#DDU401C3" rid="DDU401C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793813">3</a>). The main variant associated with this signal is rs2814778, located at position (b37) 159 174 683. Our analyses replicated this finding, identifying a number of significant hits surrounding rs2814778 (chr1: 159 174 683). The surrounding variants are the product of a well-established selective sweep, and proximity to the Duffy null mutations predisposed them to association (<a href="#DDU401C26" rid="DDU401C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793777">26</a>). While rs2814778 was not included in all three cohorts due to the removal of monomorphic SNPs during quality control, credible region analyses of this region identify a single nearby variant, rs2518564 (chr1: 159 062 436), as encompassing 99% of the signal<em>.</em> As the functional variant in the <em>DARC</em> locus is present exclusively in the African-American population consortium, we did not expect the this region to show meaningful fine mapping. The use of a proxy variant in high LD with the other two cohorts provides additional evidence of what is already known, this region is highly associated with WBC.</p><p id="__p38" class="p p-last">Other previously identified associations with WBC subtypes, however, were substantially narrowed. For example, the 7q21.2 region tagged soley by rs445 in both WBC and neutrophil counts was found to encompass 99% of the association signal for these traits. This variant is located within an intronic section of <em>CDK6</em>, a gene in the cyclin-dependent protein kinase family. At another previously identified locus for WBC and neutrophil count, located on chromosome 17, the association signal could be limited with 99% confidence to seven variants across a ∼15 kb region associated with WBC count and two variants across a ∼10 kb region associated with neutrophil count. The previously identified variant tagging this region, rs4794822, is within both of these SNP sets, but individually reaches only 72% confidence in WBC count and 88% confidence in neutrophil count.</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">Conditional analysis</h3><p id="__p39" class="p p-first-last">Conditional analysis adjusting for the effect of the most significantly associated SNP at each locus was performed to assess the independence of possible novel variants and to detect the presence of any secondary association signals within known regions. Secondary signals were defined as additional associated variants within 500 kb of previously known loci. Of the primary (known and novel) loci, approximately half contained secondary signals; these include four signals associated with monocytes, three associated with neutrophils and six associated with WBC count. In some instances, the top signal identified by the secondary analysis was stronger than that observed in the primary analysis. This could occur, for example, when the allele frequencies of the initial, index SNP are similar across ancestries, but the conditional signal(s) more accurately tag a functional variant in the respective populations. In order to verify the authenticity of our conditional analysis results, we performed reverse conditioning on our secondary signals and found the signals reported here to remain significant, suggesting an independent effect on WBC subtypes<em>.</em> The top association signals from these conditional analyses are found in Table&nbsp;<a href="/pmc/articles/PMC4245044/table/DDU401TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU401TB4" rid-ob="ob-DDU401TB4" co-legend-rid=""><span>4</span></a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU401TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p40">Loci identified by conditional analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140167477090816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Subtype</th><th colspan="3" rowspan="1" align="center">Primary top hits<sup>a</sup><hr></th><th colspan="9" rowspan="1" align="center">Secondary top hits<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Chr</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">SNP</th><th rowspan="1" colspan="1" align="left">Position</th><th rowspan="1" colspan="1" align="left">Effect allele</th><th rowspan="1" colspan="1" align="left">Other allele</th><th rowspan="1" colspan="1" align="left"><em>N</em>— studies</th><th rowspan="1" colspan="1" align="left">log<sub>10</sub> BF</th><th rowspan="1" colspan="1" align="left">PPH</th><th rowspan="1" colspan="1" align="left">Sample size</th><th rowspan="1" colspan="1" align="left">Effect direction</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">MONO</td><td rowspan="1" colspan="1">rs1449263</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">182 319 301</td><td rowspan="1" colspan="1">rs711801</td><td rowspan="1" colspan="1">182 334 873</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">10.46982</td><td rowspan="1" colspan="1">0.579</td><td rowspan="1" colspan="1">33 711</td><td rowspan="1" colspan="1">++−</td></tr><tr><td rowspan="1" colspan="1">MONO</td><td rowspan="1" colspan="1">rs3095254</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">31 221 668</td><td rowspan="1" colspan="1">rs2517774</td><td rowspan="1" colspan="1">29 893 982</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">72.9753</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">26 360</td><td rowspan="1" colspan="1">?−−</td></tr><tr><td rowspan="1" colspan="1">MONO</td><td rowspan="1" colspan="1">rs2163952</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 610 389</td><td rowspan="1" colspan="1">rs1457475</td><td rowspan="1" colspan="1">134 988 329</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">6.48727</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">28 317</td><td rowspan="1" colspan="1">−−+</td></tr><tr><td rowspan="1" colspan="1">MONO</td><td rowspan="1" colspan="1">rs12350763</td><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">113 923 723</td><td rowspan="1" colspan="1">rs4401938</td><td rowspan="1" colspan="1">121 179 496</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">68.41786</td><td rowspan="1" colspan="1">0.99</td><td rowspan="1" colspan="1">33 729</td><td rowspan="1" colspan="1">+++</td></tr><tr><td rowspan="1" colspan="1">NEU</td><td rowspan="1" colspan="1">rs7667376</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 967 890</td><td rowspan="1" colspan="1">rs1440404</td><td rowspan="1" colspan="1">74 944 449</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">54.50984</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 713</td><td rowspan="1" colspan="1">++−</td></tr><tr><td rowspan="1" colspan="1">NEU</td><td rowspan="1" colspan="1">rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">rs3731326</td><td rowspan="1" colspan="1">92 327 026</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">6.86032</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 753</td><td rowspan="1" colspan="1">+−−</td></tr><tr><td rowspan="1" colspan="1">NEU</td><td rowspan="1" colspan="1">rs4794822</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 156 712</td><td rowspan="1" colspan="1">rs4794321</td><td rowspan="1" colspan="1">46 028 844</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">7.89538</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 753</td><td rowspan="1" colspan="1">−−+</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs6734238</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">113 841 030</td><td rowspan="1" colspan="1">rs11899198</td><td rowspan="1" colspan="1">113 840 539</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">7.90504</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 694</td><td rowspan="1" colspan="1">−++</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs1371799</td><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">74 977 837</td><td rowspan="1" colspan="1">rs1440404</td><td rowspan="1" colspan="1">74 944 449</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">53.18811</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 697</td><td rowspan="1" colspan="1">+−−</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs9402686</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">135 427 817</td><td rowspan="1" colspan="1">rs1890428</td><td rowspan="1" colspan="1">140 253 023</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">22.53076</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">22 472</td><td rowspan="1" colspan="1">+?−</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs445</td><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">92 408 370</td><td rowspan="1" colspan="1">rs42626</td><td rowspan="1" colspan="1">89 961 237</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">11.88853</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 740</td><td rowspan="1" colspan="1">−++</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs2163950</td><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">130 597 585</td><td rowspan="1" colspan="1">rs10505542</td><td rowspan="1" colspan="1">130 547 253</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">19.59631</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">52 686</td><td rowspan="1" colspan="1">−−+</td></tr><tr><td rowspan="1" colspan="1">WBC</td><td rowspan="1" colspan="1">rs2241245</td><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">38 151 014</td><td rowspan="1" colspan="1">rs8070454</td><td rowspan="1" colspan="1">38 160 754</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">22.03367</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">33 231</td><td rowspan="1" colspan="1">+?−</td></tr></tbody></table></div><div id="largeobj_idm140167477090816" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4245044/table/DDU401TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN10"><p id="__p41" class="p p-first-last">Top hits from the GCTA conditional analyses are reported.</p></div><div id="TN11"><p id="__p42" class="p p-first-last">PPH, posterior probability of association; or, the probability that an SNP is truly associated with a phenotype. Effect direction order of studies: COGENT, CHARGE, RIKEN.</p></div><div id="TN12"><p id="__p43" class="p p-first-last"><sup>a</sup>Used as covariates in conditional analysis.</p></div></div></div></div><div id="s2d" class="sec sec-last"><h3 id="s2dtitle">Expression quantitative trait loci analysis</h3><p id="__p44" class="p p-first-last">All known, conditional and novel loci were assessed as potential expression quantitative trait loci (eQTLs) in leukocyte-derived tissues in order to identify any correlations between association signals and gene expression, as such correlations may account for functional relationships that are not captured by LD. Two known loci associated with monocyte count, and one novel locus newly associated with both WBC and neutrophil count, represented significant (<em>P</em> &lt; 5E−05) eQTLs when assessing either the European ancestry sentinel SNPs or their proxies in the YRI (African) and ASN (Asian) populations (<a href="#DDU401C1" rid="DDU401C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793845">1</a>). Three of the four index SNPs at these loci are located within chromosome 6p21.3; each has been associated with a different blood cell trait. The monocyte eQTLs on chromosome six are defined by the transcription factor gene <em>TCF19</em>; the neutrophil eQTLs relate primarily to the <em>HLA</em> transcripts, but depending on the tissue type are also associated with expression of <em>ATP6V1G2</em>, which encodes an enzyme involved in eukaryotic cell compartment acidification. The WBC count eQTLs also relate to various transcripts of this region. Additionally, a previously described locus associated with monocyte count on chromosome 2 is an eQTL for the <em>ITGA4</em> transcript at 2q31.3, which encodes the integrin alpha-4 subunit of the very late antigen-4 receptor on monocytes and other mononuclear cells (<a href="#DDU401C27" rid="DDU401C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793805">27</a>,<a href="#DDU401C28" rid="DDU401C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793785">28</a>). In instances where a proxy eQTL SNP has been used to represent a WBC or subtype-associated SNP, variants with greater concordance to the index SNP are viewed with more confidence than those with lower <em>R</em><sup>2</sup> values. Only two eQTL SNPs were found to be concordant or in very high LD: rs3130320 (<em>R</em><sup>2</sup> = 1) for neutrophils and rs6740847 (<em>R</em><sup>2</sup> = 0.983) for monocytes. We have reported other SNPs as potential eQTLs if their <em>R</em><sup>2</sup> value with the proxy eQTL SNP is &gt;0.5; these SNPs are of course likely to be near the blood cell SNPs' physical location. All the proxy eQTL variants identified here are located in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S4</a>.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p45" class="p p-first">We applied trans-ethnic MA to summary data from Japanese, African-American and European-Americans populations and identified six new regions that contain biologically plausible genetic loci associated with WBC traits. Many of the novel and secondary association signals we observed involve genomic regions that contain several inflammatory and immune cell-related genes.</p><p id="__p46">Particularly interesting, novel regions include the two loci on chromosome 2 associated with WBC count. The first, identified by rs6734238, falls within an inflammatory gene region of the interleukin-1 cytokine gene family (<a href="#DDU401C29" rid="DDU401C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793822">29</a>). This region has been associated with several inflammation-related biomarkers, including C-reactive protein and fibrinogen. rs6734238 is located downstream of <em>IL1F10</em> and upstream of <em>IL1RN. IL1RN</em> encodes IL-1 receptor antagonist (IL-1RA), which regulates a variety of interleukin-1-related immune and inflammatory responses, including inhibition of interleukin 1, alpha (IL1A) and interleukin 1, beta (IL1B). <em>IL1F10</em> encodes IL-38, which regulates Th17 immune responses and stimulates IL-6 cytokine production from dendritic cells <em>in vitro</em> (<a href="#DDU401C30" rid="DDU401C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793781">30</a>). The second chromosome 2 region, identified by rs10932765, is near <em>APRC2</em> and <em>CXCR1</em> (also known as <em>IL8RA</em>). <em>CXCR1</em> is of particular interest as this is a chemokine receptor involved in leukocyte chemotaxis and trafficking (<a href="#DDU401C31" rid="DDU401C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793789">31</a>).</p><p id="__p47">The novel chromosome 5q13 region associated with monocyte count lies within a family of protease-activated receptor genes, <em>F2RL2-F2RL1-F2R</em>. The <em>F2RL1</em> gene (protease-activated receptor-2 or PAR-2) has previously been related to some inflammatory and autoimmune diseases, and is a known receptor on monocytes (<a href="#DDU401C32" rid="DDU401C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793806">32</a>). PAR-2 is a G protein-coupled receptor on monocyte/macrophages and other cell types that appear to have a direct role in the regulation of innate immune function. Specifically, PAR-2 can be activated by a number of endogenous inflammation-associated proteinases (e.g. mast cell tryptase, trypsin and neutrophil proteinase 3) or exogenous pathogen-derived proteinases (<a href="#DDU401C33" rid="DDU401C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793804">33</a>). Notably, another member of the PAR gene family located on 5q13, <em>F2R</em>, encodes PAR-1, the platelet thrombin receptor. Common variants of <em>F2R</em> were recently associated with circulating platelet count in a European GWAS (<a href="#DDU401C23" rid="DDU401C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793791">23</a>).</p><p id="__p48">A single region of chromosome 6, associated with neutrophil and WBC count, is located near the <em>HBS1L</em> and <em>MYB</em> genes, which are known to be associated with fetal hemoglobin levels and monocyte counts. This locus has also been reported to be associated with red cell and platelet traits, but not previously with white cell traits. Although there is a single signal for neutrophils at this locus, the signal for WBC count appears bimodal. One of these two regions is captured by a single SNP with 99% confidence, while the second requires a regional span of nearly 1 Mb to reach the same level of confidence. This locus was also significant in our eQTL analysis across all ancestry types for both neutrophil and WBC count, in lymphoblastoid cell lines (LCLs) and whole blood.</p><p id="__p49">The region of chromosome 8 newly associated with WBC count lies near a gene of unknown function, <em>GSDMC</em>, which encodes gasdermin C<em>.</em> Other genes belonging to the gasdermin family have been associated with immune-mediated phenotypes such as asthma and alopecia (<a href="#DDU401C34" rid="DDU401C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793773">34</a>,<a href="#DDU401C35" rid="DDU401C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793852">35</a>), suggesting a role for this gene family in inflammatory disorders (<a href="#DDU401C36" rid="DDU401C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793818">36</a>).</p><p id="__p50">A signal was observed on chromosome 16 for WBC count and neutrophils; however, the top associated variant is located within the intronic region of <em>HYDIN</em>, an mRNA transcript sequence involved in cilia motility. As previously reported, it is a likely homolog to the <em>DARC</em> region of chromosome 1 and represents a spurious signal (<a href="#DDU401C12" rid="DDU401C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793787">12</a>).</p><p id="__p51">The replicated region on chromosome 4, associated with both WBC and neutrophil counts, is located near a chemokine family gene cluster, <em>CXCL5-CXCL3-CXCL2. CXCL2</em> interacts with another chemokine receptor, <em>CXCR2</em>, to control migration of leukocytes from the bone marrow (<a href="#DDU401C37" rid="DDU401C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793815">37</a>). Taken together with the <em>DARC</em> locus, these findings extend the importance of common genetic variants of chemokine ligands and receptors in the regulation of WBC counts.</p><p id="__p52">Using the proper reference panel is critical to the dependability and accuracy of this analysis. For this reason, ancestry-matched subsets of the 1000 Genomes were used as the reference panel from which LD was calculated, which are presumed to be drawn from the same general populations as those used here. However, the relatively small sample sizes available through 1000 Genomes increase the possibility of error in LD estimations (<a href="#DDU401C38" rid="DDU401C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793802">38</a>). While the localization and resolution of functional variants may improve with the additional genomic variation measured in newer reference panels, we show that even without these newer panels, the associations identified using MANTRA provide plausible candidates for functionality.</p><p id="__p53"><em>In conclusion</em>, trans-ethnic meta-analyses allow for an examination of disease traits within a large population of individuals and provide the opportunity to localize previously known regions and detect novel ones, while considering the heterogeneity of allelic effects that may exist between contentially distinct populations. Additionally, our results illustrate the utility of trans-ethnic fine mapping for narrowing regions of association. Well-established loci replicated in the present study show credible intervals that flank the known index variant. For example, the previously known monocyte associations on chromosomes 8 and 9, shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S3</a>, have 99% credible intervals of only a few thousand base-pairs, located in close proximity to the originally identified variants. Our analysis of the <em>DARC</em> region is complicated by the causal SNP being monomorphic in two of the three populations employed. When the credible interval analysis is applied to this region, the proxy variant (rs2518564) tagging the known functional variant (rs2814778) (<a href="#DDU401C5" rid="DDU401C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793765">5</a>,<a href="#DDU401C8" rid="DDU401C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793778">8</a>) is identified as accounting for the entire signal. However, prior association studies and evidence of biological function allow confident identification of rs2814778 as the functional variant. By calculating credible intervals across test statistics from analyses of combined ancestries, we were able to narrow expansive loci to smaller regions. Further work is necessary to identify what functional variants may lie within these regions.</p><p id="__p54" class="p p-last">The increasing availability of GWAS summary data for many phenotypic traits of interest, from many ethnically diverse populations, suggests that the trans-ethnic GWAS MA approach can yield additional association signals, thereby explaining some of the missing heritability and genetic architecture for other complex traits. In addition, this work is relevant for future targeted sequencing follow-up studies, as we have narrowed the scope of follow-up sequencing efforts for functional variants. By increasing the mapping resolution of the causal variants within these loci, we hope that these results guide next-generation targeted deep sequencing studies, which may disentangle the heterogeneity of effect across ethnicities (<a href="#DDU401C39" rid="DDU401C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793841">39</a>). Future work will discern which functional variants are the same across ethnicities and which tag nearby regions, through LD, that harbor the true functional variant or variants.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><p id="__p55" class="p p-first">The trans-ethnic GWA strategy was applied to three consortia containing WBC phenotypes. These include a Japanese population represented by RIKEN (Rikagaku Kenkyusho, Institute of Physical and Chemical Research, Japan), a European ancestry population represented by the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium and an African-American population represented by the Continental Origins and Genetic Epidemiology Network (COGENT) Consortium (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793794">9</a>,<a href="#DDU401C11" rid="DDU401C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793846">11</a>,<a href="#DDU401C12" rid="DDU401C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793835">12</a>). All three consortia contain measurements for total WBC count, neutrophil count and monocyte count, using the same scale of transformation and similar analytic paradigms. Clinical information of the subjects includes age, gender and smoking history, and was collected by self-report. Subject BMI was also collected as a measure. The laboratory data include total WBC count and subtypes, as determined using automated hematology cell counters according to the standardized protocol. The WBC phenotypes were natural log-transformed prior to analysis to provide a normal distribution. Samples &gt;2 SD outside of the ethnicity specific mean for the given phenotype were excluded. This was done to ensure normality of the included samples, and to exclude any subclinical inflammation such as the common cold. If a sample was identified as an outlier for one or more subtype, it was excluded entirely from the study.</p><p id="__p56">The RIKEN study comprises over 17 000 individuals from The BioBank Japan Project, which is made of up over 300 000 subjects (<a href="http://biobankjp.org">http://biobankjp.org</a>) (<a href="#DDU401C11" rid="DDU401C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793826">11</a>). Samples determined to be of non-Japanese origin by either self-report or by principal components analyses (PCAs) in GWAS were excluded from further analyses. For the GWAS, 592 232 SNPs were genotyped using Illumina HumanHap610-Quad Genotyping BeadChip. Subjects with call rates &lt;0.98 were removed, as were SNPs with call rates &lt;0.99. First- and second-degree relatives were excluded based on identity-by-descent analyses, as were SNPs with minor allele frequency (MAF) &lt;0.01 or with Hardy–Weinberg equilibrium (HWE) <em>P</em>-values &lt;1.00E−7. After quality control, genotypes were imputed using MACH 1.0 in a two-step procedure, described in detail elsewhere (<a href="#DDU401C40" rid="DDU401C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793820">40</a>,<a href="#DDU401C41" rid="DDU401C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793830">41</a>). HapMap Phase II Japanese individuals from Tokyo (JPT) and Han Chinese individiuals from Beijing (CHB) individuals were adopted as references. SNPs with imputation qualities &lt;0.30 were excluded prior to analyses, and genomic control was applied to the cohort-level data. All participants provided written informed consent as approved by the ethical committees of the Center for Genomic Medicine, RIKEN and the Institute of Medical Science, the University of Tokyo (<a href="#DDU401C11" rid="DDU401C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793796">11</a>).</p><p id="__p57">The CHARGE consortium dataset is comprised of over 19 000 individuals from seven discovery cohorts, including: the Rotterdam Study (RS), Framingham Heart Study (FHS), the NHLBI's Atherosclerosis Risk in Communities (ARIC) Study, the Age, Gene/Environment Susceptibility—Reykjavik Study (AGES), Health Aging and Body Composition study (HABC), the Baltimore Longitudinal Study of Aging (BLSA) and the Invecchaire in Chianti Study (inChianti) (<a href="#DDU401C9" rid="DDU401C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793793">9</a>). Each of these studies, with the exception of the Framingham Heart Study, is comprised of unrelated individuals of confirmed European ancestry, based on PCAs. Prior to MA, SNPs with MAF &lt;0.01, missingness&nbsp;&gt;5% or HWE &lt;1.00E−7 were excluded. Individuals with call rates &lt;95% were also excluded. After quality control, genotypes were imputed using the CEU reference panel of the HapMap Phase II haplotype data. The CHARGE consortium is comprised of MA data resulting from the summary statistics of these individual studies. Prior to the meta- analyses, study results were adjusted for genomic inflation factors, and SNPs with imputation quality&lt;0.30 were excluded. Meta-analyses were performed using a fixed-effects model in METAL (<a href="#DDU401C42" rid="DDU401C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793827">42</a>).</p><p id="__p58" class="p">The COGENT consortium is comprised of over 16 000 self-identified African-Americans from seven discovery cohorts, including: Atherosclerosis Risk in Communities (ARIC), Coronary Artery Risk Development in Young Adults (CARDIA), Johns Hopkins Genetic Study of Atherosclerosis Risk (GeneSTAR), HealthyAging in Neighborhoods of Diversity across the Life Span (HANDLS), Health, Aging, and Body Composition (Health ABC), Jackson Heart Study (JHS) and the Women's Health Initiative (WHI) (<a href="#DDU401C12" rid="DDU401C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793776">12</a>). SNPs were excluded from cohort-level GWAS if MAF &lt;1% or missingness&gt;5%. Monomorphic SNPs and ambiguously mapped SNPs were also removed. Individual samples exhibiting gender mismatch or genotype missingness &gt;10% were excluded. After quality control, genotypes were imputed with HapMap Phase II, using a 1 : 1 mixture of the CEU and YRI reference populations. Prior to MA, SNPs with imputation quality &lt;0.30 were excluded. Study-specific GWA results were corrected for genomic inflation factors, and MA was performed using a fixed-effects model in METAL (<a href="#DDU401C42" rid="DDU401C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793831">42</a>).</p><div id="s4a" class="sec sec-last"><h3 id="s4atitle">Statistical analysis</h3><p id="__p59" class="p p-first">Summary statistics for the RIKEN GWAS and the CHARGE and COGENT meta-analyses were collected and stratified by ethnicity and WBC subtype availability. Data were input into the trans-ethnic MA software package, MANTRA (<a href="#DDU401C15" rid="DDU401C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793808">15</a>), which makes use of a prior model of relatedness between studies corresponding to Fst, or mean effect allele frequency differences between populations. Relatedness is determined by differences in allele frequency between studies. MANTRA estimates the BF in favor of association for each SNP using a Markov chain Monte Carlo (MCMC) algorithm. Results are reported as log<sub>10</sub> (BF), and associations of 6 or greater have the highest posterior odds of being truly present (<a href="#DDU401C43" rid="DDU401C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793795">43</a>,<a href="#DDU401C44" rid="DDU401C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793825">44</a>). Posterior probability of heterogeneity is also reported to examine levels of variation in allelic effects across the populations used in the analysis. Combining results across studies using a Bayesian approach is advantageous, as the evidence produced by this study is directly comparable to future studies performed in the same way. Simulations of distinct ancestry populations show that when MANTRA is compared with random-effects and fixed-effects meta-analyses, MANTRA shows increased performance and produces the highest-powered results in the detection of novel associations (<a href="#DDU401C15" rid="DDU401C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793783">15</a>,<a href="#DDU401C22" rid="DDU401C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793814">22</a>,<a href="#DDU401C45" rid="DDU401C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793855">45</a>–<a href="#DDU401C47" rid="DDU401C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_468732529">47</a>).</p><p id="__p60">Since the initial imputation of the datasets used here, a number of more comprehensive reference panels have been released including the latest HapMap release and the samples available through the 1000 Genomes Project. As these reference panels contain more individuals and greater genome coverage, more genotypes are predicted with greater confidence than when using prior HapMap releases. Imputing raw data to the latest release of 1000 Genomes would be ideal; however, due to the data-sharing requirements of the cohorts included in this analysis, only summary statistics were available for these datasets. However, MANTRA is still expected to outperform a traditional MA in this case, as MANTRA accounts for heterogeneity while making no assumptions about differences or similarities in allelic effect.</p><p id="__p61">To quantify uncertainty surrounding the top hits from the trans-ethnic MANTRA analysis, we calculated 95 and 99% credible regions (<a href="#DDU401C48" rid="DDU401C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793823">48</a>). We estimated credible sets of SNPs by first defining a 1 Mb genomic region surrounding lead SNPs (±500 kb), then ranking the regional SNPs within this region according to their BF and then combining the cumulative posterior probabilities of these ranked SNPs until 95 and 99% confidence was reached.</p><p id="__p62">In addition, the top hits from MANTRA were input into a conditional analysis in order to identify additional association signals at nearby susceptibility loci and to determine independence of these secondary signals from the index SNP association. As complex diseases are assumed to be influenced by two or more genes acting in concert, it is possible that prior GWAS aimed at identifying single loci have not detected secondary signals. Thus, when large sample sizes are available, conditional approaches can be useful in detecting secondary association signals with loci that initially appear to contribute a negligible risk to disease susceptibility. In addition, as evidence of association is predicated on a given conditioned SNP, it is possible that, in some instances, the secondary associations are stronger than what was observed in the primary single-ethnicity GWAS analysis. This is possible when the allele frequencies of a given SNP are similar across ancestries, as conditional hits may be closer to a functional variant than the original, single-population hit.</p><p id="__p63">We used the software program Genome-wide Complex Trait Analysis (GCTA) v1.13 to perform conditional association analysis for each ancestry-specific set of summary results (<a href="#DDU401C49" rid="DDU401C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793836">49</a>). As individual genotype data were unavailable, this was performed separately for each cohort using summary statistics and incorporating LD information from ancestry-matched reference samples containing individual-level genotype data. When original genotype data are not available, it is essential that the reference samples be from the same population as the original data, so that the LD structure estimated from the reference population is not biased. It is also critical that the reference sample is not affected by cryptic relatedness or population stratification. This is particularly relevant to admixed populations, such as COGENT, which is comprised of African-American individuals. In order to avoid confounding the genetic relationship matrix (GRM) produced by GCTA, eigenvectors are included in the model as covariates, to capture and account for any variance that is present due to population structure (<a href="#DDU401C49" rid="DDU401C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793784">49</a>). In this analysis, we used ancestry-matched subsets from the 1000 Genomes Project to estimate LD structure within our samples (<a href="#DDU401C17" rid="DDU401C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793780">17</a>). Using these samples as LD proxies, the GCTA association analyses were conditioned on the top hits from MANTRA, specific to each locus of interest. Independently associated SNPs were selected using a stepwise model selection procedure. Analyses were performed separately for each ancestry cohort. The results from each cohort were then meta-analyzed across ethnicities.</p><p id="__p64" class="p p-last">In addition to the trans-ethnic association analysis, a secondary analysis exploring eQTLs was performed using SNAP (<a href="#DDU401C50" rid="DDU401C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793838">50</a>). SNAP is a web server that identifies and annotates nearby proxy SNPs in LD (according to HapMap) to those queried. Using ancestry-specific tissues, SNAP was identified alias SNPs for significant index SNPs, and proxy SNPs in high linkage disequilibrium (<em>R</em><sup>2</sup> &gt; 0.5). Sentinel, alias and proxy SNPs were queried within a collected database of expression SNP (eSNP) results, drawn from the following leukocyte-derived tissues: fresh lymphocytes (<a href="#DDU401C51" rid="DDU401C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793840">51</a>), fresh leukocytes (<a href="#DDU401C52" rid="DDU401C52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793768">52</a>), leukocyte samples in individuals with celiac disease (<a href="#DDU401C53" rid="DDU401C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793817">53</a>), whole blood samples (<a href="#DDU401C54" rid="DDU401C54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793775">54</a>–<a href="#DDU401C56" rid="DDU401C56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793771">56</a>), LCLs derived from asthmatic children (<a href="#DDU401C57" rid="DDU401C57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793801">57</a>,<a href="#DDU401C58" rid="DDU401C58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793829">58</a>), HapMap LCL from three populations (<a href="#DDU401C59" rid="DDU401C59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793844">59</a>), a separate study on HapMap CEU LCL (<a href="#DDU401C60" rid="DDU401C60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793782">60</a>) and additional LCL population samples [(<a href="#DDU401C61" rid="DDU401C61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793764">61</a>–<a href="#DDU401C63" rid="DDU401C63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793816">63</a>); Mangravite <em>et al</em>., unpublished], CD19+ B cells (<a href="#DDU401C64" rid="DDU401C64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793850">64</a>), primary PHA (phytohaemagglutinin)-stimulated T cells (<a href="#DDU401C61" rid="DDU401C61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793832">61</a>), CD4+ T cells (<a href="#DDU401C65" rid="DDU401C65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793828">65</a>), peripheral blood monocytes (<a href="#DDU401C27" rid="DDU401C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793843">27</a>,<a href="#DDU401C64" rid="DDU401C64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793769">64</a>,<a href="#DDU401C66" rid="DDU401C66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793842">66</a>), CD11+ dendritic cells before and after <em>Mycobacterium tuberculosis</em> infection (<a href="#DDU401C67" rid="DDU401C67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793774">67</a>) and micro-RNA QTLs queried for LCL (<a href="#DDU401C68" rid="DDU401C68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_414793811">68</a>). The collected eSNP results met criteria for statistical thresholds of association with gene transcript levels, as described in the original cited papers. In cases where an SNP was associated with a transcript, we further examined the strongest eSNP for the transcript within that dataset and the LD between the strongest eSNP and blood count-selected eSNPs. This was done to assess the concordance of the blood count and expression signals.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p65" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu401/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p66" class="p p-first-last">Y.O. is supported by a grant from the Japan Society of the Promotion of Science (JSPS). The development of the software package MANTRA was supported by Wellcome Trust (grant nos <span id="cn1">WT098017, WT090532 and WT064890</span>). S.K.G. was supported in part by <span id="cn2">P30HL101290</span>. Funding for CHARGE was made possible by NIA/NIH contract <span id="cn3">AG000932-2</span> (2009) Characterization of Normal Genomic Variability. The Age, Gene/Environment Susceptibility Reykjavik Study is funded by NIH contract <span id="cn4">N01-AG-12100</span>, the NIA Intramural Research Program, Hjartavernd (the Icelandic Heart Association) and the Althingi (the Icelandic Parliament). The Atherosclerosis Risk in the Communities Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (<span id="cn5">HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C and HHSN268201100012C), R01HL087641, R01HL59367 and R01HL086694</span>; National Human Genome Research Institute contract (<span id="cn6">U01HG004402</span>) and National Institutes of Health contract (<span id="cn7">HHSN268200625226C</span>;). The authors thank the staff and participants of the ARIC study for their important contributions. Infrastructure was partly supported by grant number <span id="cn8">UL1RR025005</span>, a component of the National Institutes of Health and NIH Roadmap for Medical Research. The National Heart, Lung, and Blood Institute's Framingham Heart Study is a joint project of the National Institutes of Health and Boston University School of Medicine and was supported by the National Heart, Lung, and Blood Institute's Framingham Heart Study (contract No. N01-HC-25195) and its contract with Affymetrix for genotyping services (contract No. N02-HL-6-4278). Analyses reflect the efforts and resource development from the Framingham Heart Study investigators participating in the SNP Health Association Resource (SHARe) project. A portion of this research was conducted using the Linux Cluster for Genetic Analysis (LinGA-II) funded by the Robert Dawson Evans Endowment of the Department of Medicine at the Boston University School of Medicine and Boston Medical Center. The Health ABC Study was supported in part by the Intramural Research Program of the NIH, National Institute on Aging, NIA contracts N01AG62101, N01AG62103 and N01AG62106. The GWAS was funded by NIA grant <span id="cn9">1R01AG032098-01A1</span> to Wake Forest University Health Sciences and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University (contract number <span id="cn10">HHSN268200782096C</span>). The InChianti Study was supported as a ‘targeted project’ (ICS 110.1RS97.71) by the Italian Ministry of Health, by the US National Institute on Aging (contracts <span id="cn11">N01-AG-916413, N01-AG-821336, 263 MD 9164 13 and 263 MD 821336</span>) and in part by the Intramural Research Program, National Institute on Aging, National Institutes of Health, USA. The generation and management of GWAS genotype data for the Rotterdam Study is supported by the Netherlands Organization of Scientific Research NWO Investments (no. <span id="cn12">175.010.2005.011, 911-03-012</span>), the Research Institute for Diseases in the Elderly (<span id="cn13">014-93-015</span>; RIDE2), EUROSPAN (European Special Populations Research Network; <span id="cn14">LSHG-CT-2006-01947</span>), the Netherlands Organization for Scientific Research (Pionier, <span id="cn15">047.016.009, 047.017.043;050-060-810</span>), Erasmus Medical Center and the Centre for Medical Systems Biology (CMSB I and II and Grand; National Genomics Initiative) of the Netherlands Genomics Initiative (NGI); The Rotterdam Study is further funded by Erasmus Medical Center and Erasmus University, Rotterdam, Netherlands Organization for the Health Research and Development (ZonMw), the Research Institute for Diseases in the Elderly (RIDE), the Ministry of Education, Culture and Science, the Ministry for Health, Welfare and Sports, the European Commission (DG XII) and the Municipality of Rotterdam. Funding for COGENT was obtained through the Broad Institute (<span id="cn16">N01-HC-65226</span>) to create this genotype/phenotype database for wide dissemination to the biomedical research community. The Atherosclerosis Risk in Communities Study is carried out as a collaborative study supported by National Heart, Lung, and Blood Institute contracts (<span id="cn17">HHSN268201100005C, HHSN268201100006C, HHSN268201100007C, HHSN268201100008C, HHSN268201100009C, HHSN268201100010C, HHSN268201100011C and HHSN268201100012C), R01HL087641, R01HL59367 and R01HL086694</span>; National Human Genome Research Institute (contract <span id="cn18">U01HG004402</span>) and National Institutes of Health (contract <span id="cn19">HHSN268200625226C</span>;). The authors thank the staff and participants of the ARIC study for their important contributions. Infrastructure was partly supported by grant number <span id="cn20">UL1RR025005</span>, a component of the National Institutes of Health and NIH Roadmap for Medical Research. ARIC is also supported by University of North Carolina at Chapel Hill (funded by <span id="cn21">N01-HC-55015</span>), Baylor Medical College (<span id="cn22">N01-HC-55016</span>), University of Mississippi Medical Center (<span id="cn23">N01-HC-55021</span>), University of Minnesota (<span id="cn24">N01-HC-55019</span>), Johns Hopkins University (<span id="cn25">N01-HC-55020</span>), University of Texas Houston (<span id="cn26">N01-HC-55017</span>) and University of North Carolina (<span id="cn27">N01-HC-55018</span>). Coronary Artery Risk in Young Adults (CARDIA): University of Alabama at Birmingham (<span id="cn28">N01-HC-48047</span>), University of Minnesota (<span id="cn29">N01-HC-48048</span>), Northwestern University (<span id="cn30">N01-HC-48049</span>), Kaiser Foundation Research Institute (<span id="cn31">N01-HC-48050</span>), University of Alabama at Birmingham (<span id="cn32">N01-HC-95095</span>), Tufts-New England Medical Center (<span id="cn33">N01-HC-45204</span>), Wake Forest University (<span id="cn34">N01-HC-45205</span>), Harbor-UCLA Research and Education Institute (<span id="cn35">N01-HC-05187</span>), University of California, Irvine (<span id="cn36">N01-HC-45134 and N01-HC-95100</span>). Jackson Heart Study (JHS): Jackson State University (<span id="cn37">N01-HC-95170</span>), University of Mississippi (<span id="cn38">N01-HC-95171</span>), Tougaloo College (<span id="cn39">N01-HC-95172</span>). Healthy Aging in Neighborhoods of Diversity across the Life Span Study (HANDLS): This research was supported by the Intramural Research Program of the NIH, National Institute on Aging and the National Center on Minority Health and Health Disparities (intramural project # Z01-AG000513 and human subjects protocol # 2009-149). Health ABC: This research was supported by NIA contracts N01AG62101, N01AG62103 and N01AG62106. The GWAS was funded by NIA grant 1R01AG032098-01A1 to Wake Forest University Health Sciences and genotyping services were provided by the Center for Inherited Disease Research (CIDR). CIDR is fully funded through a federal contract from the National Institutes of Health to The Johns Hopkins University (contract number <span id="cn40">HHSN268200782096C</span>). This research was supported in part by the Intramural Research Program of the NIH, National Institute on Aging. GeneSTAR Acknowledgement: This research was supported by the National Heart, Lung, and Blood Institute (NHLBI) through the PROGENI (<span id="cn41">U01 HL72518</span>) and STAMPEED (<span id="cn42">R01 HL087698-01</span>) consortia. Additional support was provided by grants from the NIH/National Institute of Nursing Research (<span id="cn43">R01 NR08153</span>) and the NIH/National Center for Research Resources (<span id="cn44">M01-RR000052</span>) to the Johns Hopkins General Clinical Research Center. WHI: The WHI program is funded by the National Heart, Lung, and Blood Institute, National Institutes of Health, US Department of Health and Human Services, through contracts <span id="cn45">N01WH22110, 24152, 32100-2, 32105-6, 32108-9, 32111-13, 32115, 32118-32119, 32122, 42107-26, 42129-32 and 44221</span>. The Cardiovascular Health Study (CHS) is supported from contracts <span id="cn46">N01-HC-35129, N01-HC-45133, N01-HC-75150, N01-HC-85079 through N01-HC-85086, N01 HC-15103, N01 HC-55222 and U01 HL080295</span> from the National Heart, Lung, and Blood Institute, with additional contribution from the National Institute of Neurological Disorders and Stroke. Additional support for this work was provided by NIH R01 HL71862-06 and ARRA N000949304 (to A.P.R.). Some of the results of this paper were obtained by using the program package S.A.G.E., which is supported by a US Public Health Service Resource Grant (RR03655) from the National Center for Research Resources. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript. Funding for RIKEN and the BioBank Japan Project was supported by Ministry of Education, Culture, Sports, Science and Technology, Japan. The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140167481047136"><a href="/pmc/articles/PMC4245044/bin/supp_23_25_6944__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>


</div></div><div id="idm140167481041488" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140167481041488title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p67">This study utilized the high-performance computational capabilities of the Biowulf Linux cluster at the National Institutes of Health, Bethesda, MD (<a href="http://biowulf.nih.gov">http://biowulf.nih.gov</a>).</p><p id="__p68"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140167488288800" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140167488288800title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDU401C1">1. <span class="element-citation">Ruggiero C., Metter E.J., Cherubini A., Maggio M., Sen R., Najjar S.S., Windham G.B., Ble A., Senin U., Ferrucci L. White blood cell count and mortality in the Baltimore Longitudinal Study of Aging. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2007;<span class="ref-vol">49</span>:1841–1850.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2646088/">PMC free article</a>]</span> [<a href="/pubmed/17481443" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=White+blood+cell+count+and+mortality+in+the+Baltimore+Longitudinal+Study+of+Aging&amp;author=C.+Ruggiero&amp;author=E.J.+Metter&amp;author=A.+Cherubini&amp;author=M.+Maggio&amp;author=R.+Sen&amp;volume=49&amp;publication_year=2007&amp;pages=1841-1850&amp;pmid=17481443&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C2">2. <span class="element-citation">Danesh J., Collins R., Appleby P., Peto R. Association of fibrinogen, C-reactive protein, albumin, or leukocyte count with coronary heart disease: meta-analyses of prospective studies. <span><span class="ref-journal">JAMA. </span>1998;<span class="ref-vol">279</span>:1477–1482.</span> [<a href="/pubmed/9600484" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+of+fibrinogen,+C-reactive+protein,+albumin,+or+leukocyte+count+with+coronary+heart+disease:+meta-analyses+of+prospective+studies&amp;author=J.+Danesh&amp;author=R.+Collins&amp;author=P.+Appleby&amp;author=R.+Peto&amp;volume=279&amp;publication_year=1998&amp;pages=1477-1482&amp;pmid=9600484&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C3">3. <span class="element-citation">Shankar A., Wang J.J., Rochtchina E., Yu M.C., Kefford R., Mitchell P. Association between circulating white blood cell count and cancer mortality: a population-based cohort study. <span><span class="ref-journal">Arch. Intern. Med. </span>2006;<span class="ref-vol">166</span>:188–194.</span> [<a href="/pubmed/16432087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Intern.+Med.&amp;title=Association+between+circulating+white+blood+cell+count+and+cancer+mortality:+a+population-based+cohort+study&amp;author=A.+Shankar&amp;author=J.J.+Wang&amp;author=E.+Rochtchina&amp;author=M.C.+Yu&amp;author=R.+Kefford&amp;volume=166&amp;publication_year=2006&amp;pages=188-194&amp;pmid=16432087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C4">4. <span class="element-citation">Nieto F.J., Szklo M., Folsom A.R., Rock R., Mercuri M. Leukocyte count correlates in middle-aged adults: the Atherosclerosis Risk in Communities (ARIC) Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>1992;<span class="ref-vol">136</span>:525–537.</span> [<a href="/pubmed/1442716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Leukocyte+count+correlates+in+middle-aged+adults:+the+Atherosclerosis+Risk+in+Communities+(ARIC)+Study&amp;author=F.J.+Nieto&amp;author=M.+Szklo&amp;author=A.R.+Folsom&amp;author=R.+Rock&amp;author=M.+Mercuri&amp;volume=136&amp;publication_year=1992&amp;pages=525-537&amp;pmid=1442716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C5">5. <span class="element-citation">Nalls M.A., Wilson J.G., Patterson N.J., Tandon A., Zmuda J.M., Huntsman S., Garcia M., Hu D., Li R., Beamer B.A., et al.  Admixture mapping of white cell count: genetic locus responsible for lower white blood cell count in the Health ABC and Jackson Heart studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:81–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253985/">PMC free article</a>]</span> [<a href="/pubmed/18179887" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Admixture+mapping+of+white+cell+count:+genetic+locus+responsible+for+lower+white+blood+cell+count+in+the+Health+ABC+and+Jackson+Heart+studies&amp;author=M.A.+Nalls&amp;author=J.G.+Wilson&amp;author=N.J.+Patterson&amp;author=A.+Tandon&amp;author=J.M.+Zmuda&amp;volume=82&amp;publication_year=2008&amp;pages=81-87&amp;pmid=18179887&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C6">6. <span class="element-citation">Pilia G., Chen W.-M., Scuteri A., Orrú M., Albai G., Dei M., Lai S., Usala G., Lai M., Loi P., et al.  Heritability of cardiovascular and personality traits in 6,148 Sardinians. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1557782/">PMC free article</a>]</span> [<a href="/pubmed/16934002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Heritability+of+cardiovascular+and+personality+traits+in+6,148+Sardinians&amp;author=G.+Pilia&amp;author=W.-M.+Chen&amp;author=A.+Scuteri&amp;author=M.+Orrú&amp;author=G.+Albai&amp;volume=2&amp;publication_year=2006&amp;pages=e132&amp;pmid=16934002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C7">7. <span class="element-citation">Haddy T.B., Rana S.R., Castro O. Benign ethnic neutropenia: what is a normal absolute neutrophil count? <span><span class="ref-journal">J. Lab. Clin. Med. </span>1999;<span class="ref-vol">133</span>:15–22.</span> [<a href="/pubmed/10385477" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lab.+Clin.+Med.&amp;title=Benign+ethnic+neutropenia:+what+is+a+normal+absolute+neutrophil+count?&amp;author=T.B.+Haddy&amp;author=S.R.+Rana&amp;author=O.+Castro&amp;volume=133&amp;publication_year=1999&amp;pages=15-22&amp;pmid=10385477&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C8">8. <span class="element-citation">Reich D., Nalls M.A., Kao W.H.L., Akylbekova E.L., Tandon A., Patterson N., Mullikin J., Hsueh W.-C., Cheng C.-Y., Coresh J., et al.  Reduced neutrophil count in people of African descent is due to a regulatory variant in the Duffy antigen receptor for chemokines gene. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628742/">PMC free article</a>]</span> [<a href="/pubmed/19180233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Reduced+neutrophil+count+in+people+of+African+descent+is+due+to+a+regulatory+variant+in+the+Duffy+antigen+receptor+for+chemokines+gene&amp;author=D.+Reich&amp;author=M.A.+Nalls&amp;author=W.H.L.+Kao&amp;author=E.L.+Akylbekova&amp;author=A.+Tandon&amp;volume=5&amp;publication_year=2009&amp;pages=e1000360&amp;pmid=19180233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C9">9. <span class="element-citation">Nalls M.A., Couper D.J., Tanaka T., van Rooij F.J.A., Chen M.-H., Smith A.V., Toniolo D., Zakai N.A., Yang Q., Greinacher A., et al.  Multiple loci are associated with white blood cell phenotypes. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002113.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128114/">PMC free article</a>]</span> [<a href="/pubmed/21738480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Multiple+loci+are+associated+with+white+blood+cell+phenotypes&amp;author=M.A.+Nalls&amp;author=D.J.+Couper&amp;author=T.+Tanaka&amp;author=F.J.A.+van+Rooij&amp;author=M.-H.+Chen&amp;volume=7&amp;publication_year=2011&amp;pages=e1002113&amp;pmid=21738480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C10">10. <span class="element-citation">Okada Y., Kamatani Y. Common genetic factors for hematological traits in humans. <span><span class="ref-journal">J. Hum. Genet. </span>2012;<span class="ref-vol">57</span>:161–169.</span> [<a href="/pubmed/22277899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet.&amp;title=Common+genetic+factors+for+hematological+traits+in+humans&amp;author=Y.+Okada&amp;author=Y.+Kamatani&amp;volume=57&amp;publication_year=2012&amp;pages=161-169&amp;pmid=22277899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C11">11. <span class="element-citation">Okada Y., Hirota T., Kamatani Y., Takahashi A., Ohmiya H., Kumasaka N., Higasa K., Yamaguchi-Kabata Y., Hosono N., Nalls M.A., et al.  Identification of nine novel loci associated with white blood cell subtypes in a Japanese population. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128095/">PMC free article</a>]</span> [<a href="/pubmed/21738478" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Identification+of+nine+novel+loci+associated+with+white+blood+cell+subtypes+in+a+Japanese+population&amp;author=Y.+Okada&amp;author=T.+Hirota&amp;author=Y.+Kamatani&amp;author=A.+Takahashi&amp;author=H.+Ohmiya&amp;volume=7&amp;publication_year=2011&amp;pages=e1002067&amp;pmid=21738478&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C12">12. <span class="element-citation">Reiner A.P., Lettre G., Nalls M.A., Ganesh S.K., Mathias R., Austin M.A., Dean E., Arepalli S., Britton A., Chen Z., et al.  Genome-wide association study of white blood cell count in 16,388 African Americans: the continental origins and genetic epidemiology network (COGENT) <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128101/">PMC free article</a>]</span> [<a href="/pubmed/21738479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+study+of+white+blood+cell+count+in+16,388+African+Americans:+the+continental+origins+and+genetic+epidemiology+network+(COGENT)&amp;author=A.P.+Reiner&amp;author=G.+Lettre&amp;author=M.A.+Nalls&amp;author=S.K.+Ganesh&amp;author=R.+Mathias&amp;volume=7&amp;publication_year=2011&amp;pages=e1002108&amp;pmid=21738479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C13">13. <span class="element-citation">Crosslin D.R., McDavid A., Weston N., Nelson S.C., Zheng X., Hart E., de Andrade M., Kullo I.J., McCarty C.A., Doheny K.F., et al.  Genetic variants associated with the white blood cell count in 13,923 subjects in the eMERGE Network. <span><span class="ref-journal">Hum. Genet. </span>2012;<span class="ref-vol">131</span>:639–652.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3640990/">PMC free article</a>]</span> [<a href="/pubmed/22037903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genetic+variants+associated+with+the+white+blood+cell+count+in+13,923+subjects+in+the+eMERGE+Network&amp;author=D.R.+Crosslin&amp;author=A.+McDavid&amp;author=N.+Weston&amp;author=S.C.+Nelson&amp;author=X.+Zheng&amp;volume=131&amp;publication_year=2012&amp;pages=639-652&amp;pmid=22037903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C14">14. <span class="element-citation">Li J., Glessner J.T., Zhang H., Hou C., Wei Z., Bradfield J.P., Mentch F.D., Guo Y., Kim C., Xia Q., et al.  GWAS of blood cell traits identifies novel associated loci and epistatic interactions in Caucasian and African-American children. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:1457–1464.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3657475/">PMC free article</a>]</span> [<a href="/pubmed/23263863" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=GWAS+of+blood+cell+traits+identifies+novel+associated+loci+and+epistatic+interactions+in+Caucasian+and+African-American+children&amp;author=J.+Li&amp;author=J.T.+Glessner&amp;author=H.+Zhang&amp;author=C.+Hou&amp;author=Z.+Wei&amp;volume=22&amp;publication_year=2013&amp;pages=1457-1464&amp;pmid=23263863&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C15">15. <span class="element-citation">Morris A.P. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C16">16. <span class="element-citation">International HapMap Consortium. The International HapMap Project. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">426</span>:789–796.</span> [<a href="/pubmed/14685227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+International+HapMap+Project&amp;volume=426&amp;publication_year=2003&amp;pages=789-796&amp;pmid=14685227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C17">17. <span class="element-citation">Abecasis G.R., Altshuler D., Auton A., Brooks L.D., Durbin R.M., Gibbs R.A., Hurles M.E., McVean G.A. 1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">467</span>:1061–1073.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3042601/">PMC free article</a>]</span> [<a href="/pubmed/20981092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+map+of+human+genome+variation+from+population-scale+sequencing&amp;author=G.R.+Abecasis&amp;author=D.+Altshuler&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=R.M.+Durbin&amp;volume=467&amp;publication_year=2010&amp;pages=1061-1073&amp;pmid=20981092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C18">18. <span class="element-citation">Mägi R., Morris A.P. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R.+Mägi&amp;author=A.P.+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C19">19. <span class="element-citation">Han B., Eskin E. Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:586–598.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146723/">PMC free article</a>]</span> [<a href="/pubmed/21565292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Random-effects+model+aimed+at+discovering+associations+in+meta-analysis+of+genome-wide+association+studies&amp;author=B.+Han&amp;author=E.+Eskin&amp;volume=88&amp;publication_year=2011&amp;pages=586-598&amp;pmid=21565292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C20">20. <span class="element-citation">Tian C., Gregersen P.K., Seldin M.F. Accounting for ancestry: population substructure and genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:R143–e150.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2782357/">PMC free article</a>]</span> [<a href="/pubmed/18852203" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Accounting+for+ancestry:+population+substructure+and+genome-wide+association+studies&amp;author=C.+Tian&amp;author=P.K.+Gregersen&amp;author=M.F.+Seldin&amp;volume=17&amp;publication_year=2008&amp;pages=R143-e150&amp;pmid=18852203&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C21">21. <span class="element-citation">Franke A., Balschun T., Sina C., Ellinghaus D., Häsler R., Mayr G., Albrecht M., Wittig M., Buchert E., Nikolaus S., et al.  Genome-wide association study for ulcerative colitis identifies risk loci at 7q22 and 22q13 (IL17REL) <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:292–294.</span> [<a href="/pubmed/20228798" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+for+ulcerative+colitis+identifies+risk+loci+at+7q22+and+22q13+(IL17REL)&amp;author=A.+Franke&amp;author=T.+Balschun&amp;author=C.+Sina&amp;author=D.+Ellinghaus&amp;author=R.+Häsler&amp;volume=42&amp;publication_year=2010&amp;pages=292-294&amp;pmid=20228798&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C22">22. <span class="element-citation">Dastani Z., Hivert M.-F., Timpson N., Perry J.R.B., Yuan X., Scott R.A., Henneman P., Heid I.M., Kizer J.R., Lyytikäinen L.-P., et al.  Novel loci for adiponectin levels and their influence on type 2 diabetes and metabolic traits: a multi-ethnic meta-analysis of 45,891 individuals. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002607.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315470/">PMC free article</a>]</span> [<a href="/pubmed/22479202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Novel+loci+for+adiponectin+levels+and+their+influence+on+type+2+diabetes+and+metabolic+traits:+a+multi-ethnic+meta-analysis+of+45,891+individuals&amp;author=Z.+Dastani&amp;author=M.-F.+Hivert&amp;author=N.+Timpson&amp;author=J.R.B.+Perry&amp;author=X.+Yuan&amp;volume=8&amp;publication_year=2012&amp;pages=e1002607&amp;pmid=22479202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C23">23. <span class="element-citation">Gieger C., Radhakrishnan A., Cvejic A., Tang W., Porcu E., Pistis G., Serbanovic-Canic J., Elling U., Goodall A.H., Labrune Y., et al.  New gene functions in megakaryopoiesis and platelet formation. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">480</span>:201–208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3335296/">PMC free article</a>]</span> [<a href="/pubmed/22139419" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=New+gene+functions+in+megakaryopoiesis+and+platelet+formation&amp;author=C.+Gieger&amp;author=A.+Radhakrishnan&amp;author=A.+Cvejic&amp;author=W.+Tang&amp;author=E.+Porcu&amp;volume=480&amp;publication_year=2011&amp;pages=201-208&amp;pmid=22139419&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C24">24. <span class="element-citation">Johansson U., Lawson C., Dabare M., Syndercombe-Court D., Newland A.C., Howells G.L., Macey M.G. Human peripheral blood monocytes express protease receptor-2 and respond to receptor activation by production of IL-6, IL-8, and IL-1{beta} <span><span class="ref-journal">J. Leukoc. Biol. </span>2005;<span class="ref-vol">78</span>:967–975.</span> [<a href="/pubmed/16000389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Leukoc.+Biol.&amp;title=Human+peripheral+blood+monocytes+express+protease+receptor-2+and+respond+to+receptor+activation+by+production+of+IL-6,+IL-8,+and+IL-1{beta}&amp;author=U.+Johansson&amp;author=C.+Lawson&amp;author=M.+Dabare&amp;author=D.+Syndercombe-Court&amp;author=A.C.+Newland&amp;volume=78&amp;publication_year=2005&amp;pages=967-975&amp;pmid=16000389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C25">25. <span class="element-citation">Stephens M., Balding D.J. Bayesian statistical methods for genetic association studies. <span><span class="ref-journal">Nat. Rev. Genet. </span>2009;<span class="ref-vol">10</span>:681–690.</span> [<a href="/pubmed/19763151" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Bayesian+statistical+methods+for+genetic+association+studies&amp;author=M.+Stephens&amp;author=D.J.+Balding&amp;volume=10&amp;publication_year=2009&amp;pages=681-690&amp;pmid=19763151&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C26">26. <span class="element-citation">Hamblin M.T., Di Rienzo A. Detection of the signature of natural selection in humans: evidence from the Duffy blood group locus. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2000;<span class="ref-vol">66</span>:1669–1679.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378024/">PMC free article</a>]</span> [<a href="/pubmed/10762551" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Detection+of+the+signature+of+natural+selection+in+humans:+evidence+from+the+Duffy+blood+group+locus&amp;author=M.T.+Hamblin&amp;author=A.+Di+Rienzo&amp;volume=66&amp;publication_year=2000&amp;pages=1669-1679&amp;pmid=10762551&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C27">27. <span class="element-citation">Zeller T., Wild P., Szymczak S., Rotival M., Schillert A., Castagne R., Maouche S., Germain M., Lackner K., Rossmann H., et al.  Genetics and beyond—the transcriptome of human monocytes and disease susceptibility. <span><span class="ref-journal">PLoS ONE. </span>2010;<span class="ref-vol">5</span>:e10693.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872668/">PMC free article</a>]</span> [<a href="/pubmed/20502693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Genetics+and+beyond—the+transcriptome+of+human+monocytes+and+disease+susceptibility&amp;author=T.+Zeller&amp;author=P.+Wild&amp;author=S.+Szymczak&amp;author=M.+Rotival&amp;author=A.+Schillert&amp;volume=5&amp;publication_year=2010&amp;pages=e10693&amp;pmid=20502693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C28">28. <span class="element-citation">Maugeri N., Powell J.E., ‘t Hoen P.A.C., de Geus E.J.C., Willemsen G., Kattenberg M., Henders A.K., Wallace L., Penninx B., Hottenga J.-J., et al.  LPAR1 and ITGA4 regulate peripheral blood monocyte counts. <span><span class="ref-journal">Hum. Mutat. </span>2011;<span class="ref-vol">32</span>:873–876.</span> [<a href="/pubmed/21598361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat.&amp;title=LPAR1+and+ITGA4+regulate+peripheral+blood+monocyte+counts&amp;author=N.+Maugeri&amp;author=J.E.+Powell&amp;author=P.A.C.+‘t+Hoen&amp;author=E.J.C.+de+Geus&amp;author=G.+Willemsen&amp;volume=32&amp;publication_year=2011&amp;pages=873-876&amp;pmid=21598361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C29">29. <span class="element-citation">Van de Veerdonk F.L., Netea M.G. New insights in the immunobiology of IL-1 family members. <span><span class="ref-journal">Front. Immunol. </span>2013;<span class="ref-vol">4</span>:167.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3703542/">PMC free article</a>]</span> [<a href="/pubmed/23847614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Immunol.&amp;title=New+insights+in+the+immunobiology+of+IL-1+family+members&amp;author=F.L.+Van+de+Veerdonk&amp;author=M.G.+Netea&amp;volume=4&amp;publication_year=2013&amp;pages=167&amp;pmid=23847614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C30">30. <span class="element-citation">Van de Veerdonk F.L., Stoeckman A.K., Wu G., Boeckermann A.N., Azam T., Netea M.G., Joosten L.A.B., van der Meer J.W.M., Hao R., Kalabokis V., et al.  IL-38 binds to the IL-36 receptor and has biological effects on immune cells similar to IL-36 receptor antagonist. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2012;<span class="ref-vol">109</span>:3001–3005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3286950/">PMC free article</a>]</span> [<a href="/pubmed/22315422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=IL-38+binds+to+the+IL-36+receptor+and+has+biological+effects+on+immune+cells+similar+to+IL-36+receptor+antagonist&amp;author=F.L.+Van+de+Veerdonk&amp;author=A.K.+Stoeckman&amp;author=G.+Wu&amp;author=A.N.+Boeckermann&amp;author=T.+Azam&amp;volume=109&amp;publication_year=2012&amp;pages=3001-3005&amp;pmid=22315422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C31">31. <span class="element-citation">Raghuwanshi S.K., Su Y., Singh V., Haynes K., Richmond A., Richardson R.M. The chemokine receptors CXCR1 and CXCR2 couple to distinct G protein-coupled receptor kinases to mediate and regulate leukocyte functions. <span><span class="ref-journal">J. Immunol. </span>2012;<span class="ref-vol">189</span>:2824–2832.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3436986/">PMC free article</a>]</span> [<a href="/pubmed/22869904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Immunol.&amp;title=The+chemokine+receptors+CXCR1+and+CXCR2+couple+to+distinct+G+protein-coupled+receptor+kinases+to+mediate+and+regulate+leukocyte+functions&amp;author=S.K.+Raghuwanshi&amp;author=Y.+Su&amp;author=V.+Singh&amp;author=K.+Haynes&amp;author=A.+Richmond&amp;volume=189&amp;publication_year=2012&amp;pages=2824-2832&amp;pmid=22869904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C32">32. <span class="element-citation">Rothmeier A.S., Ruf W. Protease-activated receptor 2 signaling in inflammation. <span><span class="ref-journal">Semin. Immunopathol. </span>2012;<span class="ref-vol">34</span>:133–149.</span> [<a href="/pubmed/21971685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Semin.+Immunopathol.&amp;title=Protease-activated+receptor+2+signaling+in+inflammation&amp;author=A.S.+Rothmeier&amp;author=W.+Ruf&amp;volume=34&amp;publication_year=2012&amp;pages=133-149&amp;pmid=21971685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C33">33. <span class="element-citation">Shpacovitch V., Feld M., Bunnett N.W., Steinhoff M. Protease-activated receptors: novel PARtners in innate immunity. <span><span class="ref-journal">Trends Immunol. </span>2007;<span class="ref-vol">28</span>:541–550.</span> [<a href="/pubmed/17977790" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Immunol.&amp;title=Protease-activated+receptors:+novel+PARtners+in+innate+immunity&amp;author=V.+Shpacovitch&amp;author=M.+Feld&amp;author=N.W.+Bunnett&amp;author=M.+Steinhoff&amp;volume=28&amp;publication_year=2007&amp;pages=541-550&amp;pmid=17977790&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C34">34. <span class="element-citation">Li X., Ampleford E.J., Howard T.D., Moore W.C., Torgerson D.G., Li H., Busse W.W., Castro M., Erzurum S.C., Israel E., et al.  Genome-wide association studies of asthma indicate opposite immunopathogenesis direction from autoimmune diseases. <span><span class="ref-journal">J. Allergy Clin. Immunol. </span>2012;<span class="ref-vol">130</span>:861–868.</span> e7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3579216/">PMC free article</a>]</span> [<a href="/pubmed/22694930" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Allergy+Clin.+Immunol.&amp;title=Genome-wide+association+studies+of+asthma+indicate+opposite+immunopathogenesis+direction+from+autoimmune+diseases&amp;author=X.+Li&amp;author=E.J.+Ampleford&amp;author=T.D.+Howard&amp;author=W.C.+Moore&amp;author=D.G.+Torgerson&amp;volume=130&amp;publication_year=2012&amp;pages=861-868&amp;pmid=22694930&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C35">35. <span class="element-citation">Zhou Y., Jiang X., Gu P., Chen W., Zeng X., Gao X. Gsdma3 mutation causes bulge stem cell depletion and alopecia mediated by skin inflammation. <span><span class="ref-journal">Am. J. Pathol. </span>2012;<span class="ref-vol">180</span>:763–774.</span> [<a href="/pubmed/22155111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Pathol.&amp;title=Gsdma3+mutation+causes+bulge+stem+cell+depletion+and+alopecia+mediated+by+skin+inflammation&amp;author=Y.+Zhou&amp;author=X.+Jiang&amp;author=P.+Gu&amp;author=W.+Chen&amp;author=X.+Zeng&amp;volume=180&amp;publication_year=2012&amp;pages=763-774&amp;pmid=22155111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C36">36. <span class="element-citation">Saeki N., Usui T., Aoyagi K., Kim D.H., Sato M., Mabuchi T., Yanagihara K., Ogawa K., Sakamoto H., Yoshida T., et al.  Distinctive expression and function of four GSDM family genes (GSDMA-D) in normal and malignant upper gastrointestinal epithelium. <span><span class="ref-journal">Genes Chromosomes Cancer. </span>2009;<span class="ref-vol">48</span>:261–271.</span> [<a href="/pubmed/19051310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Chromosomes+Cancer&amp;title=Distinctive+expression+and+function+of+four+GSDM+family+genes+(GSDMA-D)+in+normal+and+malignant+upper+gastrointestinal+epithelium&amp;author=N.+Saeki&amp;author=T.+Usui&amp;author=K.+Aoyagi&amp;author=D.H.+Kim&amp;author=M.+Sato&amp;volume=48&amp;publication_year=2009&amp;pages=261-271&amp;pmid=19051310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C37">37. <span class="element-citation">Day R.B., Link D.C. Regulation of neutrophil trafficking from the bone marrow. <span><span class="ref-journal">Cell. Mol. Life Sci. </span>2012;<span class="ref-vol">69</span>:1415–1423.</span> [<a href="/pubmed/22045556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell.+Mol.+Life+Sci.&amp;title=Regulation+of+neutrophil+trafficking+from+the+bone+marrow&amp;author=R.B.+Day&amp;author=D.C.+Link&amp;volume=69&amp;publication_year=2012&amp;pages=1415-1423&amp;pmid=22045556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C38">38. <span class="element-citation">Yang J., Ferreira T., Morris A.P., Medland S.E., Madden P.A.F., Heath A.C., Martin N.G., Montgomery G.W., Weedon M.N., Loos R.J., et al.  Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:369–375.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593158/">PMC free article</a>]</span> [<a href="/pubmed/22426310" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Conditional+and+joint+multiple-SNP+analysis+of+GWAS+summary+statistics+identifies+additional+variants+influencing+complex+traits&amp;author=J.+Yang&amp;author=T.+Ferreira&amp;author=A.P.+Morris&amp;author=S.E.+Medland&amp;author=P.A.F.+Madden&amp;volume=44&amp;publication_year=2012&amp;pages=369-375&amp;pmid=22426310&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C39">39. <span class="element-citation">Goldstein D.B. The importance of synthetic associations will only be resolved empirically. <span><span class="ref-journal">PLoS Biol. </span>2011;<span class="ref-vol">9</span>:e1001008.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3022532/">PMC free article</a>]</span> [<a href="/pubmed/21267066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=The+importance+of+synthetic+associations+will+only+be+resolved+empirically&amp;author=D.B.+Goldstein&amp;volume=9&amp;publication_year=2011&amp;pages=e1001008&amp;pmid=21267066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C40">40. <span class="element-citation">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C41">41. <span class="element-citation">Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C42">42. <span class="element-citation">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C43">43. <span class="element-citation">Lander E., Kruglyak L. Genetic dissection of complex traits: guidelines for interpreting and reporting linkage results. <span><span class="ref-journal">Nat. Genet. </span>1995;<span class="ref-vol">11</span>:241–247.</span> [<a href="/pubmed/7581446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+dissection+of+complex+traits:+guidelines+for+interpreting+and+reporting+linkage+results&amp;author=E.+Lander&amp;author=L.+Kruglyak&amp;volume=11&amp;publication_year=1995&amp;pages=241-247&amp;pmid=7581446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C44">44. <span class="element-citation">Sawcer S. Bayes factors in complex genetics. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2010;<span class="ref-vol">18</span>:746–750.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987358/">PMC free article</a>]</span> [<a href="/pubmed/20179745" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Bayes+factors+in+complex+genetics&amp;author=S.+Sawcer&amp;volume=18&amp;publication_year=2010&amp;pages=746-750&amp;pmid=20179745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C45">45. <span class="element-citation">Wang X., Chua H.-X., Chen P., Ong R.T.-H., Sim X., Zhang W., Takeuchi F., Liu X., Khor C.-C., Tay W.-T., et al.  Comparing methods for performing trans-ethnic meta-analysis of genome-wide association studies. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:2303–2311.</span> [<a href="/pubmed/23406875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Comparing+methods+for+performing+trans-ethnic+meta-analysis+of+genome-wide+association+studies&amp;author=X.+Wang&amp;author=H.-X.+Chua&amp;author=P.+Chen&amp;author=R.T.-H.+Ong&amp;author=X.+Sim&amp;volume=22&amp;publication_year=2013&amp;pages=2303-2311&amp;pmid=23406875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C46">46. <span class="element-citation">Urbanek M., Hayes M.G., Armstrong L.L., Morrison J., Lowe L.P., Badon S.E., Scheftner D., Pluzhnikov A., Levine D., Laurie C.C., et al.  The chromosome 3q25 genomic region is associated with measures of adiposity in newborns in a multi-ethnic genome-wide association study. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:3583–3596.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3736865/">PMC free article</a>]</span> [<a href="/pubmed/23575227" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=The+chromosome+3q25+genomic+region+is+associated+with+measures+of+adiposity+in+newborns+in+a+multi-ethnic+genome-wide+association+study&amp;author=M.+Urbanek&amp;author=M.G.+Hayes&amp;author=L.L.+Armstrong&amp;author=J.+Morrison&amp;author=L.P.+Lowe&amp;volume=22&amp;publication_year=2013&amp;pages=3583-3596&amp;pmid=23575227&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C47">47. <span class="element-citation">Franceschini N., Fox E., Zhang Z., Edwards T.L., Nalls M.A., Sung Y.J., Tayo B.O., Sun Y.V., Gottesman O., Adeyemo A., et al.  Genome-wide association analysis of blood-pressure traits in African-Ancestry individuals reveals common associated genes in African and non-African populations. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2013</span> 10.1016/j.ajhg.2013.07.010. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3769920/">PMC free article</a>]</span> [<a href="/pubmed/23972371" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=Genome-wide+association+analysis+of+blood-pressure+traits+in+African-Ancestry+individuals+reveals+common+associated+genes+in+African+and+non-African+populations&amp;author=N.+Franceschini&amp;author=E.+Fox&amp;author=Z.+Zhang&amp;author=T.L.+Edwards&amp;author=M.A.+Nalls&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C48">48. <span class="element-citation">Franceschini N., van Rooij F.J.A., Prins B.P., Feitosa M.F., Karakas M., Eckfeldt J.H., Folsom A.R., Kopp J., Vaez A., Andrews J.S., et al.  Discovery and fine mapping of serum protein loci through transethnic meta-analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:744–753.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3484648/">PMC free article</a>]</span> [<a href="/pubmed/23022100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Discovery+and+fine+mapping+of+serum+protein+loci+through+transethnic+meta-analysis&amp;author=N.+Franceschini&amp;author=F.J.A.+van+Rooij&amp;author=B.P.+Prins&amp;author=M.F.+Feitosa&amp;author=M.+Karakas&amp;volume=91&amp;publication_year=2012&amp;pages=744-753&amp;pmid=23022100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C49">49. <span class="element-citation">Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J.+Yang&amp;author=S.H.+Lee&amp;author=M.E.+Goddard&amp;author=P.M.+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C50">50. <span class="element-citation">Johnson A.D., Handsaker R.E., Pulit S.L., Nizzari M.M., O'Donnell C.J., de Bakker P.I.W. SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>:2938–2939.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=A.D.+Johnson&amp;author=R.E.+Handsaker&amp;author=S.L.+Pulit&amp;author=M.M.+Nizzari&amp;author=C.J.+O'Donnell&amp;volume=24&amp;publication_year=2008&amp;pages=2938-2939&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C51">51. <span class="element-citation">Göring H.H.H., Curran J.E., Johnson M.P., Dyer T.D., Charlesworth J., Cole S.A., Jowett J.B.M., Abraham L.J., Rainwater D.L., Comuzzie A.G., et al.  Discovery of expression QTLs using large-scale transcriptional profiling in human lymphocytes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1208–1216.</span> [<a href="/pubmed/17873875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Discovery+of+expression+QTLs+using+large-scale+transcriptional+profiling+in+human+lymphocytes&amp;author=H.H.H.+Göring&amp;author=J.E.+Curran&amp;author=M.P.+Johnson&amp;author=T.D.+Dyer&amp;author=J.+Charlesworth&amp;volume=39&amp;publication_year=2007&amp;pages=1208-1216&amp;pmid=17873875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C52">52. <span class="element-citation">Idaghdour Y., Czika W., Shianna K.V., Lee S.H., Visscher P.M., Martin H.C., Miclaus K., Jadallah S.J., Goldstein D.B., Wolfinger R.D., et al.  Geographical genomics of human leukocyte gene expression variation in southern Morocco. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:62–67.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2798927/">PMC free article</a>]</span> [<a href="/pubmed/19966804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Geographical+genomics+of+human+leukocyte+gene+expression+variation+in+southern+Morocco&amp;author=Y.+Idaghdour&amp;author=W.+Czika&amp;author=K.V.+Shianna&amp;author=S.H.+Lee&amp;author=P.M.+Visscher&amp;volume=42&amp;publication_year=2010&amp;pages=62-67&amp;pmid=19966804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C53">53. <span class="element-citation">Heap G.A., Trynka G., Jansen R.C., Bruinenberg M., Swertz M.A., Dinesen L.C., Hunt K.A., Wijmenga C., Vanheel D.A., Franke L. Complex nature of SNP genotype effects on gene expression in primary human leucocytes. <span><span class="ref-journal">BMC Med. Genomics. </span>2009;<span class="ref-vol">2</span>:1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628677/">PMC free article</a>]</span> [<a href="/pubmed/19128478" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genomics&amp;title=Complex+nature+of+SNP+genotype+effects+on+gene+expression+in+primary+human+leucocytes&amp;author=G.A.+Heap&amp;author=G.+Trynka&amp;author=R.C.+Jansen&amp;author=M.+Bruinenberg&amp;author=M.A.+Swertz&amp;volume=2&amp;publication_year=2009&amp;pages=1&amp;pmid=19128478&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C54">54. <span class="element-citation">Fehrmann R.S.N., Jansen R.C., Veldink J.H., Westra H.-J., Arends D., Bonder M.J., Fu J., Deelen P., Groen H.J.M., Smolonska A., et al.  Trans-eQTLs reveal that independent genetic variants associated with a complex phenotype converge on intermediate genes, with a major role for the HLA. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002197.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3150446/">PMC free article</a>]</span> [<a href="/pubmed/21829388" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Trans-eQTLs+reveal+that+independent+genetic+variants+associated+with+a+complex+phenotype+converge+on+intermediate+genes,+with+a+major+role+for+the+HLA&amp;author=R.S.N.+Fehrmann&amp;author=R.C.+Jansen&amp;author=J.H.+Veldink&amp;author=H.-J.+Westra&amp;author=D.+Arends&amp;volume=7&amp;publication_year=2011&amp;pages=e1002197&amp;pmid=21829388&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C55">55. <span class="element-citation">Emilsson V., Thorleifsson G., Zhang B., Leonardson A.S., Zink F., Zhu J., Carlson S., Helgason A., Walters G.B., Gunnarsdottir S., et al.  Genetics of gene expression and its effect on disease. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">452</span>:423–428.</span> [<a href="/pubmed/18344981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics+of+gene+expression+and+its+effect+on+disease&amp;author=V.+Emilsson&amp;author=G.+Thorleifsson&amp;author=B.+Zhang&amp;author=A.S.+Leonardson&amp;author=F.+Zink&amp;volume=452&amp;publication_year=2008&amp;pages=423-428&amp;pmid=18344981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C56">56. <span class="element-citation">Mehta D., Heim K., Herder C., Carstensen M., Eckstein G., Schurmann C., Homuth G., Nauck M., Völker U., Roden M., et al.  Impact of common regulatory single-nucleotide variants on gene expression profiles in whole blood. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2013;<span class="ref-vol">21</span>:48–54.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3522194/">PMC free article</a>]</span> [<a href="/pubmed/22692066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Impact+of+common+regulatory+single-nucleotide+variants+on+gene+expression+profiles+in+whole+blood&amp;author=D.+Mehta&amp;author=K.+Heim&amp;author=C.+Herder&amp;author=M.+Carstensen&amp;author=G.+Eckstein&amp;volume=21&amp;publication_year=2013&amp;pages=48-54&amp;pmid=22692066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C57">57. <span class="element-citation">Dixon A.L., Liang L., Moffatt M.F., Chen W., Heath S., Wong K.C.C., Taylor J., Burnett E., Gut I., Farrall M., et al.  A genome-wide association study of global gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1202–1207.</span> [<a href="/pubmed/17873877" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+of+global+gene+expression&amp;author=A.L.+Dixon&amp;author=L.+Liang&amp;author=M.F.+Moffatt&amp;author=W.+Chen&amp;author=S.+Heath&amp;volume=39&amp;publication_year=2007&amp;pages=1202-1207&amp;pmid=17873877&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C58">58. <span class="element-citation">Liang L., Morar N., Dixon A.L., Lathrop G.M., Abecasis G.R., Moffatt M.F., Cookson W.O.C. A cross-platform analysis of 14,177 expression quantitative trait loci derived from lymphoblastoid cell lines. <span><span class="ref-journal">Genome Res. </span>2013;<span class="ref-vol">23</span>:716–726.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3613588/">PMC free article</a>]</span> [<a href="/pubmed/23345460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=A+cross-platform+analysis+of+14,177+expression+quantitative+trait+loci+derived+from+lymphoblastoid+cell+lines&amp;author=L.+Liang&amp;author=N.+Morar&amp;author=A.L.+Dixon&amp;author=G.M.+Lathrop&amp;author=G.R.+Abecasis&amp;volume=23&amp;publication_year=2013&amp;pages=716-726&amp;pmid=23345460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C59">59. <span class="element-citation">Stranger B.E., Nica A.C., Forrest M.S., Dimas A., Bird C.P., Beazley C., Ingle C.E., Dunning M., Flicek P., Koller D., et al.  Population genomics of human gene expression. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:1217–1224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683249/">PMC free article</a>]</span> [<a href="/pubmed/17873874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Population+genomics+of+human+gene+expression&amp;author=B.E.+Stranger&amp;author=A.C.+Nica&amp;author=M.S.+Forrest&amp;author=A.+Dimas&amp;author=C.P.+Bird&amp;volume=39&amp;publication_year=2007&amp;pages=1217-1224&amp;pmid=17873874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C60">60. <span class="element-citation">Kwan T., Benovoy D., Dias C., Gurd S., Provencher C., Beaulieu P., Hudson T.J., Sladek R., Majewski J. Genome-wide analysis of transcript isoform variation in humans. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:225–231.</span> [<a href="/pubmed/18193047" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+analysis+of+transcript+isoform+variation+in+humans&amp;author=T.+Kwan&amp;author=D.+Benovoy&amp;author=C.+Dias&amp;author=S.+Gurd&amp;author=C.+Provencher&amp;volume=40&amp;publication_year=2008&amp;pages=225-231&amp;pmid=18193047&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C61">61. <span class="element-citation">Dimas A.S., Deutsch S., Stranger B.E., Montgomery S.B., Borel C., Attar-Cohen H., Ingle C., Beazley C., Gutierrez Arcelus M., Sekowska M., et al.  Common regulatory variation impacts gene expression in a cell type-dependent manner. <span><span class="ref-journal">Science. </span>2009;<span class="ref-vol">325</span>:1246–1250.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2867218/">PMC free article</a>]</span> [<a href="/pubmed/19644074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common+regulatory+variation+impacts+gene+expression+in+a+cell+type-dependent+manner&amp;author=A.S.+Dimas&amp;author=S.+Deutsch&amp;author=B.E.+Stranger&amp;author=S.B.+Montgomery&amp;author=C.+Borel&amp;volume=325&amp;publication_year=2009&amp;pages=1246-1250&amp;pmid=19644074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C62">62. <span class="element-citation">Cusanovich D.A., Billstrand C., Zhou X., Chavarria C., De Leon S., Michelini K., Pai A.A., Ober C., Gilad Y. The combination of a genome-wide association study of lymphocyte count and analysis of gene expression data reveals novel asthma candidate genes. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:2111–2123.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3315207/">PMC free article</a>]</span> [<a href="/pubmed/22286170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=The+combination+of+a+genome-wide+association+study+of+lymphocyte+count+and+analysis+of+gene+expression+data+reveals+novel+asthma+candidate+genes&amp;author=D.A.+Cusanovich&amp;author=C.+Billstrand&amp;author=X.+Zhou&amp;author=C.+Chavarria&amp;author=S.+De+Leon&amp;volume=21&amp;publication_year=2012&amp;pages=2111-2123&amp;pmid=22286170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C63">63. <span class="element-citation">Grundberg E., Small K.S., Hedman Å.K., Nica A.C., Buil A., Keildson S., Bell J.T., Yang T.-P., Meduri E., Barrett A., et al.  Mapping cis- and trans-regulatory effects across multiple tissues in twins. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1084–1089.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784328/">PMC free article</a>]</span> [<a href="/pubmed/22941192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Mapping+cis-+and+trans-regulatory+effects+across+multiple+tissues+in+twins&amp;author=E.+Grundberg&amp;author=K.S.+Small&amp;author=Å.K.+Hedman&amp;author=A.C.+Nica&amp;author=A.+Buil&amp;volume=44&amp;publication_year=2012&amp;pages=1084-1089&amp;pmid=22941192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C64">64. <span class="element-citation">Fairfax B.P., Makino S., Radhakrishnan J., Plant K., Leslie S., Dilthey A., Ellis P., Langford C., Vannberg F.O., Knight J.C. Genetics of gene expression in primary immune cells identifies cell type-specific master regulators and roles of HLA alleles. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:502–510.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3437404/">PMC free article</a>]</span> [<a href="/pubmed/22446964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetics+of+gene+expression+in+primary+immune+cells+identifies+cell+type-specific+master+regulators+and+roles+of+HLA+alleles&amp;author=B.P.+Fairfax&amp;author=S.+Makino&amp;author=J.+Radhakrishnan&amp;author=K.+Plant&amp;author=S.+Leslie&amp;volume=44&amp;publication_year=2012&amp;pages=502-510&amp;pmid=22446964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C65">65. <span class="element-citation">Murphy A., Chu J.-H., Xu M., Carey V.J., Lazarus R., Liu A., Szefler S.J., Strunk R., Demuth K., Castro M., et al.  Mapping of numerous disease-associated expression polymorphisms in primary peripheral blood CD4+ lymphocytes. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:4745–4757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2972694/">PMC free article</a>]</span> [<a href="/pubmed/20833654" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Mapping+of+numerous+disease-associated+expression+polymorphisms+in+primary+peripheral+blood+CD4++lymphocytes&amp;author=A.+Murphy&amp;author=J.-H.+Chu&amp;author=M.+Xu&amp;author=V.J.+Carey&amp;author=R.+Lazarus&amp;volume=19&amp;publication_year=2010&amp;pages=4745-4757&amp;pmid=20833654&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C66">66. <span class="element-citation">Heinzen E.L., Ge D., Cronin K.D., Maia J.M., Shianna K.V., Gabriel W.N., Welsh-Bohmer K.A., Hulette C.M., Denny T.N., Goldstein D.B. Tissue-specific genetic control of splicing: implications for the study of complex traits. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2605930/">PMC free article</a>]</span> [<a href="/pubmed/19222302" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol.&amp;title=Tissue-specific+genetic+control+of+splicing:+implications+for+the+study+of+complex+traits&amp;author=E.L.+Heinzen&amp;author=D.+Ge&amp;author=K.D.+Cronin&amp;author=J.M.+Maia&amp;author=K.V.+Shianna&amp;volume=6&amp;publication_year=2008&amp;pages=e1&amp;pmid=19222302&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C67">67. <span class="element-citation">Barreiro L.B., Tailleux L., Pai A.A., Gicquel B., Marioni J.C., Gilad Y. Deciphering the genetic architecture of variation in the immune response to <em>Mycobacterium tuberculosis</em> infection. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2012;<span class="ref-vol">109</span>:1204–1209.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3268270/">PMC free article</a>]</span> [<a href="/pubmed/22233810" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Deciphering+the+genetic+architecture+of+variation+in+the+immune+response+to+Mycobacterium+tuberculosis+infection&amp;author=L.B.+Barreiro&amp;author=L.+Tailleux&amp;author=A.A.+Pai&amp;author=B.+Gicquel&amp;author=J.C.+Marioni&amp;volume=109&amp;publication_year=2012&amp;pages=1204-1209&amp;pmid=22233810&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU401C68">68. <span class="element-citation">Huang R.S., Gamazon E.R., Ziliak D., Wen Y., Im H.K., Zhang W., Wing C., Duan S., Bleibel W.K., Cox N.J., et al.  Population differences in microRNA expression and biological implications. <span><span class="ref-journal">RNA Biol. </span>2011;<span class="ref-vol">8</span>:692–701.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3225983/">PMC free article</a>]</span> [<a href="/pubmed/21691150" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=RNA+Biol.&amp;title=Population+differences+in+microRNA+expression+and+biological+implications&amp;author=R.S.+Huang&amp;author=E.R.+Gamazon&amp;author=D.+Ziliak&amp;author=Y.+Wen&amp;author=H.K.+Im&amp;volume=8&amp;publication_year=2011&amp;pages=692-701&amp;pmid=21691150&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4245044&amp;issue-id=245747&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140167473436032title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140167481041488title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140167488288800title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4245044/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4245044/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4245044/pdf/ddu401.pdf">PDF (906K)</a></li> | <li><a href="#" data-citationid="PMC4245044" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4245044%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4245044%2F&amp;text=Trans-ethnic%20meta-analysis%20of%20white%20blood%20cell%20phenotypes" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4245044%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25096241" data-aiid="4245044" data-aid="4245044" data-iid="245747" data-domainid="740" data-domain="hmg" data-accid="PMC4245044" data-md5="9acbe78c955d4a4f1dd7cf4322b2a3ee"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:34:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal101&amp;ncbi_phid=CE8848C2DD56CF510000000008EE054F&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4245044%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4245044/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal101 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>