<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4450081/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies HMGN3 locus for spine bone size variation in Chinese</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4450081/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human genetics"><meta name="citation_title" content="Genome-wide association study identifies HMGN3 locus for spine bone size variation in Chinese"><meta name="citation_authors" content="Shu-Feng Lei, Hui Shen, Tie-Lin Yang, Yan Guo, Shan-Shan Dong, Xiang-Hong Xu, Fei-Yan Deng, Qing Tian, Yong-Jun Liu, Yao-Zhong Liu, Jian Li, Hong-Wen Deng"><meta name="citation_date" content="March 2012"><meta name="citation_issue" content="3"><meta name="citation_volume" content="131"><meta name="citation_firstpage" content="463"><meta name="citation_doi" content="10.1007/s00439-011-1093-7"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4450081/?report=abstract"><meta name="citation_pmid" content="21947420"><meta name="DC.Title" content="Genome-wide association study identifies HMGN3 locus for spine bone size variation in Chinese"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Shu-Feng Lei"><meta name="DC.Contributor" content="Hui Shen"><meta name="DC.Contributor" content="Tie-Lin Yang"><meta name="DC.Contributor" content="Yan Guo"><meta name="DC.Contributor" content="Shan-Shan Dong"><meta name="DC.Contributor" content="Xiang-Hong Xu"><meta name="DC.Contributor" content="Fei-Yan Deng"><meta name="DC.Contributor" content="Qing Tian"><meta name="DC.Contributor" content="Yong-Jun Liu"><meta name="DC.Contributor" content="Yao-Zhong Liu"><meta name="DC.Contributor" content="Jian Li"><meta name="DC.Contributor" content="Hong-Wen Deng"><meta name="DC.Date" content="2012 Mar"><meta name="DC.Identifier" content="10.1007/s00439-011-1093-7"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies HMGN3 locus for spine bone size variation in Chinese"><meta property="og:type" content="article"><meta property="og:description" content="Bone size (BS) is one of the major risk factors for osteoporotic fractures. BS variation is genetically determined to a substantial degree with heritability over 50%, but specific genes underlying variation of BS are still largely unknown. To identify ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4450081/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4450081/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4450081/pdf/nihms386049.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D3E7ADD5349E10000000000E7009A.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4450081%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4450081</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Genet</a></span>. Author manuscript; available in PMC 2015 May 31.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21947420" target="pmc_ext" ref="reftype=publisher&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Hum Genet. 2012 Mar; 131(3): 463–469. </span></a></div><span class="fm-vol-iss-date">Published online 2011 Sep 25. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1007%2Fs00439-011-1093-7" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1007/s00439-011-1093-7</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4450081</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS386049</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/21947420">21947420</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies HMGN3 locus for spine bone size variation in Chinese</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696394762640" co-class="co-affbox">Shu-Feng Lei</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup> <a href="/pubmed/?term=Shen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696395740096" co-class="co-affbox">Hui Shen</a>, <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696396429312" co-class="co-affbox">Tie-Lin Yang</a>, <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696359421520" co-class="co-affbox">Yan Guo</a>, <a href="/pubmed/?term=Dong%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696359419616" co-class="co-affbox">Shan-Shan Dong</a>, <a href="/pubmed/?term=Xu%20XH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696396553616" co-class="co-affbox">Xiang-Hong Xu</a>, <a href="/pubmed/?term=Deng%20FY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696357064688" co-class="co-affbox">Fei-Yan Deng</a>, <a href="/pubmed/?term=Tian%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696350681136" co-class="co-affbox">Qing Tian</a>, <a href="/pubmed/?term=Liu%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696350679232" co-class="co-affbox">Yong-Jun Liu</a>, <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696359472864" co-class="co-affbox">Yao-Zhong Liu</a>, <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696357294272" co-class="co-affbox">Jian Li</a>, and  <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420" class="affpopup" co-rid="_co_idm139696356467392" co-class="co-affbox">Hong-Wen Deng</a><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139696394762640"><h3 class="no_margin">Shu-Feng Lei</h3><p>Laboratory of Molecular and Statistical Genetics and the Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, Changsha 410081, Hunan, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Lei%20SF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Shu-Feng Lei</a></div></div><div id="_co_idm139696395740096"><h3 class="no_margin">Hui Shen</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Shen%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Hui Shen</a></div></div><div id="_co_idm139696396429312"><h3 class="no_margin">Tie-Lin Yang</h3><p>School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Yang%20TL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Tie-Lin Yang</a></div></div><div id="_co_idm139696359421520"><h3 class="no_margin">Yan Guo</h3><p>School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Guo%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Yan Guo</a></div></div><div id="_co_idm139696359419616"><h3 class="no_margin">Shan-Shan Dong</h3><p>School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Dong%20SS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Shan-Shan Dong</a></div></div><div id="_co_idm139696396553616"><h3 class="no_margin">Xiang-Hong Xu</h3><p>School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Xu%20XH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Xiang-Hong Xu</a></div></div><div id="_co_idm139696357064688"><h3 class="no_margin">Fei-Yan Deng</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Deng%20FY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Fei-Yan Deng</a></div></div><div id="_co_idm139696350681136"><h3 class="no_margin">Qing Tian</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Tian%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Qing Tian</a></div></div><div id="_co_idm139696350679232"><h3 class="no_margin">Yong-Jun Liu</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20YJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Yong-Jun Liu</a></div></div><div id="_co_idm139696359472864"><h3 class="no_margin">Yao-Zhong Liu</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20YZ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Yao-Zhong Liu</a></div></div><div id="_co_idm139696357294272"><h3 class="no_margin">Jian Li</h3><p>Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Jian Li</a></div></div><div id="_co_idm139696356467392"><h3 class="no_margin">Hong-Wen Deng</h3><p>Laboratory of Molecular and Statistical Genetics and the Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, Changsha 410081, Hunan, People’s Republic of China. Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA. Center of Systematic Biomedical Research, University of Shanghai for Science and Technology, Shanghai 200093, People’s Republic of China</p><div>Find articles by <a href="/pubmed/?term=Deng%20HW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=21947420">Hong-Wen Deng</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139696357367440_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139696357367440_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139696357367440_ai" style="display:none"><div><span class="fm-affl">Shu-Feng Lei, </span><span class="fm-affl" id="A1" lang="en"> Laboratory of Molecular and Statistical Genetics and the Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, Changsha 410081, Hunan, People’s Republic of China;</span></div><div><a href="#idm139696357367440aff-info">Contributor Information</a>.</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div><span class="contrib-email" id="idm139696394762640">Shu-Feng Lei: <a href="mailto:dev@null" data-email="nc.ude.unnuh@iel" class="oemail">nc.ude.unnuh@iel</a>; </span> <span class="contrib-email" id="idm139696356467392">Hong-Wen Deng: <a href="mailto:dev@null" data-email="ude.enalut@2gnedh" class="oemail">ude.enalut@2gnedh</a></span> </div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139696357367440_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=21947420" target="pmc_ext" ref="reftype=publisher&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Hum Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4450081/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S1.</strong></div><div><a href="/pmc/articles/PMC4450081/bin/NIHMS386049-supplement-S1.pdf" data-ga-action="click_feat_suppl">NIHMS386049-supplement-S1.pdf</a><span style="color:gray"> (44K)</span></div><div class="small guid">GUID:&nbsp;2B299D53-354C-42A0-ADD7-1F4E27181985</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>S2.</strong></div><div><a href="/pmc/articles/PMC4450081/bin/NIHMS386049-supplement-S2.pdf" data-ga-action="click_feat_suppl">NIHMS386049-supplement-S2.pdf</a><span style="color:gray"> (90K)</span></div><div class="small guid">GUID:&nbsp;92F42E7E-D153-4E14-A20F-B52E28A4ECC7</div></div></dd></dl></div><div id="idm139696396376160" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139696396376160title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Bone size (BS) is one of the major risk factors for osteoporotic fractures. BS variation is genetically determined to a substantial degree with heritability over 50%, but specific genes underlying variation of BS are still largely unknown. To identify specific genes for BS in Chinese, initial genome-wide association scan (GWAS) study and follow-up replication study were performed. In initial GWAS study, a group of 12 contiguous single-nucleotide polymorphism (SNP)s, which span a region of ~ 25 kb and locate at the upstream of HMGN3 gene (high-mobility group nucleosomal binding domain 3), achieved moderate association signals for spine BS, with <em>P</em> values ranging from 6.2E–05 to 1.8E–06. In the follow-up replication study, eight of the 12 SNPs were detected suggestive replicate associations with BS in 1,728 unrelated female Caucasians, which have well-known differences from Chinese in ethnic genetic background. The SNPs in the region of HMGN3 gene formed a tightly combined haplotype block in both Chinese and Caucasians. The results suggest that the genomic region containing HMGN3 gene may be associated with spine BS in Chinese.</p></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">Osteoporosis is a skeletal disorder characterized by excessive skeletal fragility leading to an increased risk of fracture. Bone size (BS) is one of the major parameters in determining bone quality and strength (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044982">Ahlborg et al. 2003</a>; <a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044975">Augat et al. 1996</a>; <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044967">Fox et al. 1998</a>; <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044973">McCreadie and Goldstein 2000</a>). Abnormal BS significantly contributes to the pathology of osteoporotic fracture (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044995">Deng et al. 2002c</a>; <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044974">Seeman et al. 2001</a>), the most serious clinical complication of osteoporosis. BS variation is genetically determined to a substantial degree (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044998">Deng et al. 2002a</a>; <a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044970">Tan et al. 2008</a>) with heritability over 50%, but specific genes underlying variation of BS are still largely unknown.</p><p id="P3">Genetic studies via candidate gene association analyses were performed to identify genes contributing to variation of BS (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045001">Fang et al. 2007</a>; <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044966">Lei et al. 2005a</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044977">b</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044986">c</a>, <a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044983">2005</a>; <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045003">Malkin et al. 2005</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044984">van der Sluis et al. 2003</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044980">Zmuda et al. 2001</a>). However, the number of candidate genes tested in previous candidate gene association analyses is very limited, partly because the selection of candidate genes was based on known significance in bone biology.</p><p id="P4">Recently developing genome-wide association scan (GWAS) studies are powerful tools to rapidly and systematically identify genes for complex traits/diseases. In Caucasian, our group performed the first GWAS study for BS by interrogating ~380,000 SNPs on the Affymetrix platform in 1,000 homogeneous unrelated Caucasian subjects, and identified that PLCL1 (phospholipase c-like 1) is a novel gene associated with variation of BS (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044988">Liu et al. 2008</a>). However, in Chinese, which is the largest population in the world, has well-known difference from Caucasian in ethnic genetic background, no such GWAS study on BS has been performed to dissect the genetic basis of BS at the genome-wide scale.</p><p id="P5" class="p p-last">Here, we performed an initial GWAS study in 1,627 unrelated Chinese subjects using highly dense Affymetrix Genome-Wide Human SNP Arrays that examine ~909,622 SNPs and a subsequent replication study to identify genes important to BS.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Materials and methods</h2><p id="P6" class="p p-first">The study was approved by the necessary Institutional Review Board or Research Administration of the involved institutions. Signed informed consent documents were obtained from all study participants before entering the study. For the initial GWAS and replicate samples, areal BS values of lumbar spine (L1–L4) were measured using dual energy X-ray absorptiometry (DXA) machines (Hologic Inc., Bedford, MA, USA) that were calibrated daily.</p><p id="P7" class="p">For the initial GWAS sample, a total of 1,627 Han adult unrelated subjects (age 34.5 ± 13.2 years) including 802 males and 825 females were recruited from two cities (Xi’an and Changsha) and their surrounding areas. All study subjects belong to Chinese Han ethnic group. Strict exclusion criteria (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045002">Deng et al. 2002b</a>) were adopted to minimize any known and potential confounding effects on variation of bone phenotype. Briefly, patients with chronic diseases/conditions that may potentially affect bone mass or bone metabolism were excluded. These diseases/conditions included chronic disorders involving vital organs (heart, lung, liver, kidney, and brain), serious metabolic diseases (diabetes, hypo- or hyperparathyroidism, and hyperthyroidism), other skeletal diseases (Paget’s disease, osteogenesis imperfecta, and rheumatoid arthritis), chronic use of drugs affecting bone metabolism (corticosteroid therapy, and anticonvulsant drugs), and malnutrition conditions (chronic diarrhea and chronic ulcerative colitis). For the replication study sample, a total of 1,728 random samples (age 51.6 ± 12.9 years) were recruited in Midwestern US in Kansas City, Missouri and Omaha, Nebraska. All identified subjects were American Caucasians. Basic characteristics of study subjects were summarized in <a href="/pmc/articles/PMC4450081/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p6">Basic characteristics of study subjects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139696355881184" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left">Trait</th><th rowspan="1" colspan="1" valign="middle" align="left">Initial discovery</th><th rowspan="1" colspan="1" valign="middle" align="left">Replication</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Sample size</td><td rowspan="1" colspan="1" valign="top" align="left">1,627</td><td rowspan="1" colspan="1" valign="top" align="left">1,728</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years)</td><td rowspan="1" colspan="1" valign="top" align="left">34.5 (13.2)</td><td rowspan="1" colspan="1" valign="top" align="left">51.6 (12.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Height (m)</td><td rowspan="1" colspan="1" valign="top" align="left">1.64 (0.08)</td><td rowspan="1" colspan="1" valign="top" align="left">1.63 (0.06)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Weight (kg)</td><td rowspan="1" colspan="1" valign="top" align="left">60.1 (10.5)</td><td rowspan="1" colspan="1" valign="top" align="left">71.4 (16.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Spine BS (cm<sup>2</sup>)</td><td rowspan="1" colspan="1" valign="top" align="left">60.4 (7.0)</td><td rowspan="1" colspan="1" valign="top" align="left">58.6 (5.7)</td></tr></tbody></table></div><div id="largeobj_idm139696355881184" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4450081/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p7" class="p p-first-last">Mean (SD)</p></div></div></div><div id="S3" class="sec"><h3 id="S3title">Genotyping</h3><p id="P8" class="p p-first-last">Genomic DNA was extracted from whole human blood using a commercial isolation kit (Gentra systems, Minneapolis, MN, USA) following the protocol of the kit. For the initial GWAS sample, genotyping with the Affymetrix Genome-Wide Human SNP Array 6.0 was performed using the standard protocol recommended by the manufacturer. Fluorescence intensities were quantified using an Affymetrix array scanner 3000 7G. Data management and analyses were performed using the Affymetrix<sup>®</sup> Gene-Chip<sup>®</sup> Command Console<sup>®</sup> Software (AGCC). Contrast QC threshold, which measures how well experiments resolve SNP signals into three genotype clusters, was set at the default value of &gt;0.4 for quality control. Subjects with contrast QC of &lt;0.4 were subject to re-genotyping. The final average contrast QC across the entire sample reached a high level of 2.62. The average missing call rate for all SNPs is 1.67%. Out of the initial full-set of 909,622 SNPs, there are 17,888 SNPs with allele frequencies extremely deviating from Hardy–Weinberg equilibrium (<em>P</em> &lt; 0.01), 202,984 SNPs with minor allele frequency (MAF) &lt;1% and 618 with both. Therefore, the final SNP set maintained in the subsequent analyses contained 689,368 SNPs, yielding an average marker spacing of ~4 kb throughout the human genome. For the replication study sample, genotyping of SNPs was performed with the same Affymetrix Genome-Wide Human SNP Array 6.0 using the standard protocol. The final average contrast QC across the entire sample also reached a high level of 2.32. The average missing call rate for all SNPs is 1.27%.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Statistical analysis</h3><div id="S5" class="sec sec-first"><p></p><h4 id="S5title" class="inline">GWAS analysis </h4><p id="P9" class="p p-first-last">Gender and age, two significant covariates, were used to adjust the raw spine BS values for subsequent analyses by using a liner regression method. Spine BS values and gender &amp; age were used as dependent and independent variables, respectively, and the output residuals of spine BS were used as phenotype value in association analysis. PLINK (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044981">Purcell et al. 2007</a>) was used to perform genotypic association analyses with the command “–assoc”. Genotypic association analyses were used to compare the difference of mean BS values among three genotypic groups for each SNP. The adjusted spine BS values were in agreement with the asymptotic multivariate normal distribution.</p></div><div id="S6" class="sec"><p></p><h4 id="S6title" class="inline">Haplotype analysis </h4><p id="P10" class="p p-first-last">The linkage disequilibrium (LD) [standardized <em>D′</em> (<em>D</em>/<em>D</em><sub>max</sub>)] patterns for the detected SNPs were analyzed and plotted using the Haploview program (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044969">Barrett et al. 2005</a>) (<a href="http://www.broad.mit.edu/mpg/haploview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broad.mit.edu/mpg/haploview/</a>). Haplotype-specific association analysis in a quantitative trait association model was performed with the command “—hap-assoc” in the PLINK.</p></div><div id="S7" class="sec"><p></p><h4 id="S7title" class="inline">Population stratification </h4><p id="P11" class="p p-first-last">EIGENSTRAT was employed to perform principal component analysis to correct for stratification in GWAS studies. We used ~700,000 SNPs to calculate the principal components and the ten default main eigenvectors were used as covariates to adjust BS.</p></div><div id="S8" class="sec sec-last"><p></p><h4 id="S8title" class="inline">Meta-analysis </h4><p id="P12" class="p p-first-last">Meta-analysis for initial discovery and replication studies for BS was performed with the command “–meta-analysis” in the PLINK. The PLINK calculated four parameters for each SNP including <em>P</em>(F): fixed-effects meta-analysis <em>P</em> value; <em>P</em>(R): random-effects meta-analysis <em>P</em> value; <em>Q: P</em> value for Cochran’s <em>Q</em> statistics to test presence (<em>P</em> &lt; 0.05) versus absence of heterogeneity; <em>I: I</em><sup>2</sup> heterogeneity index to test the extent of heterogeneity (0–100).</p></div></div></div><div id="S9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S9title">Results</h2><div id="S10" class="sec sec-first-last"><h3 id="S10title">Initial GWAS study</h3><p id="P13" class="p p-first">As shown in a quantile–quantile (Q–Q) plot (<a href="/pmc/articles/PMC4450081/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>) that demonstrate the distribution of <em>P</em> values involving 689,368 eligible SNPs in the initial sample, the observed <em>P</em> values matched the expected <em>P</em> values over the range of 1 &lt; −log<sub>10</sub>(<em>P</em>) &lt; 4.5, but little departed at the extreme tail (−log<sub>10</sub>(<em>P</em>) &gt; 4.5), probably suggesting that the suggestive associations identified are likely due to true variants rather than potential biases such as genotyping error, potential population stratification, and cryptic family relatedness.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4450081/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139696359811280" class="figure"><a href="/pmc/articles/PMC4450081/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4450081_nihms386049f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms386049f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4450081/bin/nihms386049f1.jpg"></a></div><div id="largeobj_idm139696359811280" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4450081/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4450081/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p3">Quantile–quantile (Q–Q) plot for BS in Chinese. From the Q–Q plot, the observed <em>P</em> values for BS match the expected <em>P</em> values under the null distributions over the range of (1 &lt; −log<sub>10</sub>(<em>P</em>) &lt; 4.5)</p></div></div></div><p id="P14"><a href="/pmc/articles/PMC4450081/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> shows the distribution of <em>P</em> values in association analyses across the genome. We detected no significant associations according to the conservative genome-wide threshold of Bonferroni correction (<em>P</em> &lt; 7.3E–08) (0.05/689,368). However, we detected 75 SNPs with moderate association signals (<em>P</em> &lt; 1.0E–04). The most significant association was detected at rs17743190 (<em>P</em> = 1.8E–06), which is located at the upstream of high-mobility group nucleosomal binding domain 3 (HMGN3 gene). Moreover, at the HMGN3 locus, another 11 contiguous SNPs (<a href="/pmc/articles/PMC4450081/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>), which span a region of ~25 kb, also achieved moderate association signals for spine BS, with <em>P</em> values ranging from 6.2E–05 to 6.1E–06. In the follow-up replication study, eight of the 12 SNPs were detected suggestive replicate association signals with BS in 1,728 unrelated female Caucasians, which have well-known difference from Chinese in ethnic genetic background. We also identified other two distinct groups of contiguous SNPs with moderate consistent association signals to BS, one in the upstream region of ALCAM gene (activated leukocyte cell adhesion molecule) and the other in the intron of COMMD7 gene (COMM domain containing 7) (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>), but no significant or suggestive replicate association signals were detected for these two distinct groups in the 1,728 unrelated female Caucasians. Due to high phenotypic correlation between BS and height, we also performed association analysis between the 75 SNPs and height. Among the 75 SNPs, 48 SNPs were associated with height (<em>P</em> &lt; 0.05) with the strongest association at rs16878913 (<em>P</em> = 9.5E–07). After we adjusted BS value with height, most of the association <em>P</em> values for the 75 SNPs were much weaker than those without adjustment of height (data not shown). These results taken together suggest that some of the detected SNPs have evident pleiotropic effects on both BS and height.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4450081/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139696395613872" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms386049f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms386049f2.jpg" src="/pmc/articles/PMC4450081/bin/nihms386049f2.jpg"></div></a><div id="largeobj_idm139696395613872" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4450081/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC4450081/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4450081/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p4">Distribution of <em>P</em> values across the genome in association analyses. The −log10 <em>P</em> values from 689,368 SNPs in 1,627 Chinese are plotted according to its physical position on successive chromosomes. “<em>P</em> = 7.3E–08” is the conservative genome-wide threshold of Bonferroni correction in this study</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p8">Associations of 12 SNPs in HMGN3 locus with BS and height</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139696395857632" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1" valign="top" align="left">SNP ID</th><th rowspan="3" colspan="1" valign="top" align="left">Position</th><th rowspan="3" colspan="1" valign="top" align="left">Allele<sup>a</sup></th><th colspan="6" rowspan="1" valign="bottom" align="left">BS
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">Height
<hr></th><th colspan="4" rowspan="1" valign="top" align="left">Meta-analysis<sup>d</sup></th></tr><tr><th colspan="3" rowspan="1" valign="bottom" align="left">Initial discovery
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="left">Replication
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">Chinese
<hr></th><th colspan="4" rowspan="1" valign="bottom" align="left">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">MAF</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>β</em></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em><sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="left">MAF</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>β</em><sup>c</sup></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>β</em></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em>(F)</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>P</em>(R)</th><th rowspan="1" colspan="1" valign="middle" align="left"><em>Q</em></th><th rowspan="1" colspan="1" valign="middle" align="left"><em>I</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17743190</td><td rowspan="1" colspan="1" valign="top" align="left">80164398</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.83</td><td rowspan="1" colspan="1" valign="top" align="left">1.8E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">0.40</td><td rowspan="1" colspan="1" valign="top" align="left">0.27</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">4.2E–05</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">6.6E–03</td><td rowspan="1" colspan="1" valign="top" align="left">86.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3857454</td><td rowspan="1" colspan="1" valign="top" align="left">80166252</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.57</td><td rowspan="1" colspan="1" valign="top" align="left">6.1E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.56</td><td rowspan="1" colspan="1" valign="top" align="left">6.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.70</td><td rowspan="1" colspan="1" valign="top" align="left">4.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">1.1E–05</td><td rowspan="1" colspan="1" valign="top" align="left">3.6E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.9E–02</td><td rowspan="1" colspan="1" valign="top" align="left">79.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3907327</td><td rowspan="1" colspan="1" valign="top" align="left">80164301</td><td rowspan="1" colspan="1" valign="top" align="left">C/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.54</td><td rowspan="1" colspan="1" valign="top" align="left">7.8E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">8.5E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.68</td><td rowspan="1" colspan="1" valign="top" align="left">4.8E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.2E–05</td><td rowspan="1" colspan="1" valign="top" align="left">4.7E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.5E–02</td><td rowspan="1" colspan="1" valign="top" align="left">80.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17799762</td><td rowspan="1" colspan="1" valign="top" align="left">80165971</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.53</td><td rowspan="1" colspan="1" valign="top" align="left">8.3E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.51</td><td rowspan="1" colspan="1" valign="top" align="left">8.5E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">5.1E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.4E–05</td><td rowspan="1" colspan="1" valign="top" align="left">4.9E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.4E–02</td><td rowspan="1" colspan="1" valign="top" align="left">80.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17799805</td><td rowspan="1" colspan="1" valign="top" align="left">80166050</td><td rowspan="1" colspan="1" valign="top" align="left">C/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.53</td><td rowspan="1" colspan="1" valign="top" align="left">8.4E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.55</td><td rowspan="1" colspan="1" valign="top" align="left">6.7E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">5.0E–02</td><td rowspan="1" colspan="1" valign="top" align="left">1.6E–05</td><td rowspan="1" colspan="1" valign="top" align="left">3.7E–02</td><td rowspan="1" colspan="1" valign="top" align="left">3.0E–02</td><td rowspan="1" colspan="1" valign="top" align="left">78.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1385530</td><td rowspan="1" colspan="1" valign="top" align="left">80164684</td><td rowspan="1" colspan="1" valign="top" align="left">G/A</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.53</td><td rowspan="1" colspan="1" valign="top" align="left">8.8E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.55</td><td rowspan="1" colspan="1" valign="top" align="left">6.4E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.63</td><td rowspan="1" colspan="1" valign="top" align="left">6.8E–02</td><td rowspan="1" colspan="1" valign="top" align="left">1.5E–05</td><td rowspan="1" colspan="1" valign="top" align="left">3.6E–02</td><td rowspan="1" colspan="1" valign="top" align="left">3.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">78.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs17799462</td><td rowspan="1" colspan="1" valign="top" align="left">80163839</td><td rowspan="1" colspan="1" valign="top" align="left">C/T</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">1.52</td><td rowspan="1" colspan="1" valign="top" align="left">9.3E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.09</td><td rowspan="1" colspan="1" valign="top" align="left">0.17</td><td rowspan="1" colspan="1" valign="top" align="left">0.60</td><td rowspan="1" colspan="1" valign="top" align="left">0.68</td><td rowspan="1" colspan="1" valign="top" align="left">5.0E–02</td><td rowspan="1" colspan="1" valign="top" align="left">5.2E–04</td><td rowspan="1" colspan="1" valign="top" align="left">0.21</td><td rowspan="1" colspan="1" valign="top" align="left">4.7E–03</td><td rowspan="1" colspan="1" valign="top" align="left">87.5</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3857453</td><td rowspan="1" colspan="1" valign="top" align="left">80166130</td><td rowspan="1" colspan="1" valign="top" align="left">G/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.52</td><td rowspan="1" colspan="1" valign="top" align="left">9.9E–06</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">7.8E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.68</td><td rowspan="1" colspan="1" valign="top" align="left">5.0E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.2E–05</td><td rowspan="1" colspan="1" valign="top" align="left">4.3E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.9E–02</td><td rowspan="1" colspan="1" valign="top" align="left">79.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3857449</td><td rowspan="1" colspan="1" valign="top" align="left">80163957</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.48</td><td rowspan="1" colspan="1" valign="top" align="left">1.7E–05</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.56</td><td rowspan="1" colspan="1" valign="top" align="left">6.3E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.67</td><td rowspan="1" colspan="1" valign="top" align="left">5.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.2E–05</td><td rowspan="1" colspan="1" valign="top" align="left">3.0E–02</td><td rowspan="1" colspan="1" valign="top" align="left">4.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">75.7</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10484943</td><td rowspan="1" colspan="1" valign="top" align="left">80181477</td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">0.11</td><td rowspan="1" colspan="1" valign="top" align="left">1.50</td><td rowspan="1" colspan="1" valign="top" align="left">1.9E–05</td><td rowspan="1" colspan="1" valign="top" align="left">0.12</td><td rowspan="1" colspan="1" valign="top" align="left">0.52</td><td rowspan="1" colspan="1" valign="top" align="left">8.5E–02</td><td rowspan="1" colspan="1" valign="top" align="left">0.61</td><td rowspan="1" colspan="1" valign="top" align="left">8.6E–02</td><td rowspan="1" colspan="1" valign="top" align="left">4.1E–05</td><td rowspan="1" colspan="1" valign="top" align="left">4.4E–02</td><td rowspan="1" colspan="1" valign="top" align="left">3.2E–02</td><td rowspan="1" colspan="1" valign="top" align="left">78.2</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs196670</td><td rowspan="1" colspan="1" valign="top" align="left">80170314</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="left">1.43</td><td rowspan="1" colspan="1" valign="top" align="left">3.8E–05</td><td rowspan="1" colspan="1" valign="top" align="left">0.19</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">0.59</td><td rowspan="1" colspan="1" valign="top" align="left">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">0.13</td><td rowspan="1" colspan="1" valign="top" align="left">4.7E–03</td><td rowspan="1" colspan="1" valign="top" align="left">0.24</td><td rowspan="1" colspan="1" valign="top" align="left">2.2E–03</td><td rowspan="1" colspan="1" valign="top" align="left">89.4</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs16890720</td><td rowspan="1" colspan="1" valign="top" align="left">80189011</td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">0.15</td><td rowspan="1" colspan="1" valign="top" align="left">1.40</td><td rowspan="1" colspan="1" valign="top" align="left">6.2E–05</td><td rowspan="1" colspan="1" valign="top" align="left">0.16</td><td rowspan="1" colspan="1" valign="top" align="left">0.22</td><td rowspan="1" colspan="1" valign="top" align="left">0.40</td><td rowspan="1" colspan="1" valign="top" align="left">0.59</td><td rowspan="1" colspan="1" valign="top" align="left">9.1E–02</td><td rowspan="1" colspan="1" valign="top" align="left">2.0E–03</td><td rowspan="1" colspan="1" valign="top" align="left">0.18</td><td rowspan="1" colspan="1" valign="top" align="left">6.9E–03</td><td rowspan="1" colspan="1" valign="top" align="left">86.3</td></tr></tbody></table></div><div id="largeobj_idm139696395857632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4450081/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p9" class="p p-first-last">HMGN3 gene is located at 6q14</p></div><div id="TFN3"><sup>a</sup>The first allele represents the minor allele of each SNP</div><div id="TFN4"><sup>b</sup><em>P</em> values after correction of population stratification: EIGENSTRAT was employed to calculate the principal components and the ten default main eigenvectors were used as covariate to adjust BS</div><div id="TFN5"><sup>c</sup>The direction of effect indicated by <em>β</em> is calculated for minor allele</div><div id="TFN6"><sup>d</sup>Meta-analysis for initial discovery and replication studies for BS was performed with the command “–meta-analysis” in PLINK</div><div id="TFN7"><p id="__p14" class="p p-first-last"><em>P(F)</em> fixed-effects meta-analysis <em>P</em> value, <em>P(R)</em> random-effects meta-analysis <em>P</em> value, <em>Q P</em> value for Cochran’s <em>Q</em> statistic, <em>I I</em><sup>2</sup> heterogeneity index</p></div></div></div><p id="P15">As shown in <a href="/pmc/articles/PMC4450081/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>, the fixed-effects meta-analysis <em>P</em> values (<em>P</em>(F)) for most of the 75 SNPs are little weaker than in initial study, but much stronger than in replicate study. However, for the 75 SNPs except for five SNPS (rs2955948, rs16960741, rs16999576, rs16999576 and rs926598), <em>P</em> values for Cochran’s <em>Q</em> statistic are significant and <em>I</em><sup>2</sup> heterogeneity indexes are high, suggesting high heterogeneity in effect of SNP across Chinese and Caucasians. In this situation, random-effects meta-analysis <em>P</em> values (<em>P</em>(R)) are more appropriate to evaluate the combined effects than <em>P</em>(F). The <em>P</em>(R) values for most of the 75 SNPs are much weaker than in initial study, but little stronger than in replicate study.</p><p id="P16"><a href="/pmc/articles/PMC4450081/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> shows the LD pattern and the haplotype block structure for the SNP group of HMGN3 locus in initial Chinese sample. The SNPs in the region of HMGN3 gene formed a tightly combined haplotype block in initial Chinese sample (<a href="/pmc/articles/PMC4450081/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>) and also in replicate Caucasian sample (data not shown). As shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table 2</a>, the PLINK detected two haplotypes for the 12 SNPs in the initial Chinese sample, and the associations for the two haplotypes are significant (<em>P</em> = 6.8E–05 at HAPL-1 and <em>P</em> = 9.3E–06 at HAPL-1). However, in the replicate Caucasian sample, six haplotypes were detected, but only one haplotype (HAPL-6) is significant.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4450081/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139696430274656" class="figure"><a href="/pmc/articles/PMC4450081/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4450081_nihms386049f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms386049f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4450081/bin/nihms386049f3.jpg"></a></div><div id="largeobj_idm139696430274656" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4450081/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4450081/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p5">Haplotype block for 12 SNPs detected in HMGN3 gene locus in Chinese. The detected SNPs are located at the upstream of HMGN3 (~200 kb). The LD patterns for the 12 SNPs were analyzed and plotted using the Haploview program</p></div></div></div><p id="P17" class="p p-last">Previous genetic studies on BS are relatively few. Together, about seven genes (ANKH, PLCL1, COL1A1, COL1A2, ESR1, VDR, and CYP17) (<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044979">Lei et al. 2005a</a>; <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044972">Liu et al. 2008</a>; <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044989">Long et al. 2004</a>; <a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044985">Malkin et al. 2005</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044991">van der Sluis et al. 2003</a>; <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044976">Zmuda et al. 2001</a>) were studied and found significant associations with BS at various skeletal sites (e.g., spine, femoral neck, hip, and hand). We selected 353 SNPs, which were located in the seven genes above and genotyped with Affymetrix Human Genome SNP 6.0, for association analysis with spine BS in initial Chinese sample. Only six SNPs from two genes (estrogen receptor-alpha and vitamin receptor) have <em>P</em> values of &lt;0.05, but these <em>P</em> values will be not significant if adapting multiple testing corrections (FDR_BH).</p></div></div><div id="S11" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S11title">Discussion</h2><p id="P18" class="p p-first">In Caucasian, a lot of genetic studies using genome-wide linkage scan (GWLS) (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044968">Shen et al. 2006</a>; <a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045000">Xu et al. 2004</a>) or candidate gene association analyses (<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044978">Long et al. 2004</a>; <a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044994">van der Sluis et al. 2003</a>) identified quite a few chromosomal regions potentially harboring genes underlying variation of spine BS, or some variants in candidate genes associated with spine BS variation. Recently developing GWAS studies are powerful tools to rapidly and systematically identify genes for complex traits/diseases. Our group performed the first GWAS study of BS in 1,000 homogeneous unrelated Caucasian subjects (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044996">Liu et al. 2008</a>). No such GWAS study on BS was performed to dissect the genetic basis of BS at the genome-wide scale in Chinese, which has well-known difference in genetic background from Caucasian. Therefore, it is necessary to perform independent genetic studies to rapidly and systematically identify genes for BS variation in Chinese. This study represents the first GWAS study for BS in Chinese.</p><p id="P19">HMGN3 is a nuclear protein that binds to nucleosomes, promotes chromatin unfolding, enhances access to nucleosomes, and modulates transcription from chromatin templates. This gene is a new, but important regulator of glucose homeostasis and demonstrates the relevance to glucose-stimulated insulin secretion (<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044993">Ueda et al. 2009</a>). Insulin acts directly on osteoclast activity, and thiazolidinediones (TZD) may have a negative effect by switching mesenchymal progenitor cells to adipose rather than bone tissue (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044987">Montagnani et al. 2011</a>). The data from expression database demonstrated that this gene is expressed in tissues of bone and bone marrow (<a href="http://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?UGID=140116&amp;TAXID=9606&amp;SEARCH=HMGN3" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?UGID=140116&amp;TAXID=9606&amp;SEARCH=HMGN3</a>). The current detected association between the HMGN3 gene and spine BS added further evidence on supporting the link between HMGN3 locus and bone. Such association results should be considered exploratory and will require more replications in larger cohorts to better understand their role on bone. Focused molecular functional studies deserve to be pursued to clarify the regulating mechanisms possibly from the aspect of insulin signaling and glucose metabolism.</p><p id="P20">In a previous GWAS study of BS in Caucasian subjects, we identified that PLCL1 (phospholipase c-like 1) is a novel gene associated with variation of BS at hip in Caucasian (<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045005">Liu et al. 2008</a>). However, we have not found any replicate association evidence in the current study at spine in Chinese, possibly suggesting the differences at skeletal sites in genetic determination of BS.</p><p id="P21">Some bone phenotypes are clearly under ethnic-specific genetic determination (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044999">Dvornyk et al. 2005</a>, <a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044990">2003</a>; <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044965">Lei et al. 2006</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044997">2003</a>). For example, five candidate genes (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452045004">Dvornyk et al. 2005</a>) were tested for their contribution of ethnicity to bone mineral density variation in Caucasian and Chinese. The frequencies and distribution patterns of SNPs of some prominent bone candidate genes were different between Caucasian and Chinese (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044971">Lei et al. 2003</a>). In this study, all association signals for the 75 detected SNPs (<a href="/pmc/articles/PMC4450081/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>) in the initial study are stronger than in replication study, and some of them even have reverse direction of effect (<em>β</em>). Moreover, all the studied SNPs except for five SNPS (rs2955948, rs16960741, rs16999576, rs16999576, and rs926598), <em>P</em> values for Cochran’s <em>Q</em> statistic are significant and <em>I</em><sup>2</sup> heterogeneity indexes are high, suggesting high heterogeneity in effect of SNP across Chinese and Caucasians. The above observations combined together suggest evident differences in genetic regulators of BS in Chinese versus Caucasians.</p><p id="P22">This study has several limitations. First, the age between two cohorts are quite different (34.5 in Chinese versus 51.6 in Caucasian). Second, the sex composition is different, with both men and women in Chinese, but only women in Caucasian. These two above-mentioned differences may be the two important factors underlying discrepancy of the findings between Chinese and Caucasian. Second, the statistical power is relatively low. Calculated with a GWA-Power (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_452044992">Feng et al. 2011</a>), the current sample size (<em>n</em> = 1,627) in initial study has only over 11% of power to detect the studied SNPs with strong LD (<em>D′</em> = 0.9) with QTLs that explain over 1% of spine BS variation according to the conservative genome-wide threshold of Bonferroni correction (<em>P</em> &lt; 7.3E–08) (0.05/689,368).</p><p id="P23" class="p p-last">In conclusion, this study reported the first GWAS for BS in Chinese population. The results of this study suggest that HMGN3 locus may be associated with spine BS.</p></div><div id="S13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>S1</h4><div class="sup-box half_rhythm" id="d36e587"><a href="/pmc/articles/PMC4450081/bin/NIHMS386049-supplement-S1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(44K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>S2</h4><div class="sup-box half_rhythm" id="d36e591"><a href="/pmc/articles/PMC4450081/bin/NIHMS386049-supplement-S2.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(90K, pdf)</sup></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Acknowledgments</h2><div class="sec"><p id="__p1">The study was partially supported by Natural Science Foundation of China (NSFC) (31071097, 30600364, 30771222, and 30900810), NSFC-CIHR (Canadian Institutes of Health Research) Joint Health Research Initiative Proposal (30811120436), NSFC/RGC (Research Grants Council) Joint Research Scheme (30731160618), and Shanghai Leading Academic Discipline Project (<a id="__tag_782175424" class="tag_hotlink" href="/protein/S30501" ref="/protein/S30501"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"S30501","term_id":"478847","term_text":"pir||S30501"}}</span>S30501</a>) and startup fund from Shanghai University of Science and Technology. HWD was partially supported by grants from NIH (R03TW008221, R01AR050496, R01AG026564, R01AR057049, and P50AR055081) and Edward G. Schlieder Endowed Chair.</p></div></div><div id="idm139696356755536" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139696356755536title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"></p><p id="__p2" class="p p-first-last"><strong>Electronic supplementary material</strong> The online version of this article (doi:10.1007/s00439-011-1093-7) contains <a href="#SD1" rid="SD1" class=" supplementary-material">supplementary material</a>, which is available to authorized users.</p><p></p></div></div><div id="idm139696357367440aff-info" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139696357367440aff-infotitle">Contributor Information</h2><p><span class="fm-affl">Shu-Feng Lei, </span><span class="fm-affl" id="A1" lang="en"> Laboratory of Molecular and Statistical Genetics and the Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, Changsha 410081, Hunan, People’s Republic of China.</span></p><p><span class="fm-affl">Hui Shen, </span><span class="fm-affl" id="A2" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Tie-Lin Yang, </span><span class="fm-affl" id="A3" lang="en"> School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China.</span></p><p><span class="fm-affl">Yan Guo, </span><span class="fm-affl" id="A4" lang="en"> School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China.</span></p><p><span class="fm-affl">Shan-Shan Dong, </span><span class="fm-affl" id="A5" lang="en"> School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China.</span></p><p><span class="fm-affl">Xiang-Hong Xu, </span><span class="fm-affl" id="A6" lang="en"> School of Life Science and Technology, Xi’an Jiaotong University, Xi’an 710049, Shanxi, People’s Republic of China.</span></p><p><span class="fm-affl">Fei-Yan Deng, </span><span class="fm-affl" id="A7" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Qing Tian, </span><span class="fm-affl" id="A8" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Yong-Jun Liu, </span><span class="fm-affl" id="A9" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Yao-Zhong Liu, </span><span class="fm-affl" id="A10" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Jian Li, </span><span class="fm-affl" id="A11" lang="en"> Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA.</span></p><p><span class="fm-affl">Hong-Wen Deng, </span><span class="fm-affl" id="A12" lang="en"> Laboratory of Molecular and Statistical Genetics and the Key Laboratory of Protein Chemistry and Developmental Biology of Ministry of Education, College of Life Sciences, Hunan Normal University, Changsha 410081, Hunan, People’s Republic of China. Department of Biostatistics and Bioinformatics, School of Public Health and Tropical Medicine, Tulane University, New Orleans, LA 70112, USA. Center of Systematic Biomedical Research, University of Shanghai for Science and Technology, Shanghai 200093, People’s Republic of China.</span></p></div><div id="idm139696429979216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139696429979216title">References</h2><div class="ref-list-sec sec" id="reference-list"><ul class="back-ref-list" style="list-style-type:decimal;"><li id="R1"><span class="element-citation">Ahlborg HG, Johnell O, Turner CH, Rannevik G, Karlsson MK. Bone loss and bone size after menopause. <span><span class="ref-journal">N Engl J Med. </span>2003;<span class="ref-vol">349</span>:327–334.</span> [<a href="/pubmed/12878739" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Bone+loss+and+bone+size+after+menopause&amp;author=HG+Ahlborg&amp;author=O+Johnell&amp;author=CH+Turner&amp;author=G+Rannevik&amp;author=MK+Karlsson&amp;volume=349&amp;publication_year=2003&amp;pages=327-334&amp;pmid=12878739&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R2"><span class="element-citation">Augat P, Reeb H, Claes LE. Prediction of fracture load at different skeletal sites by geometric properties of the cortical shell. <span><span class="ref-journal">J Bone Miner Res. </span>1996;<span class="ref-vol">11</span>:1356–1363.</span> [<a href="/pubmed/8864911" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Prediction+of+fracture+load+at+different+skeletal+sites+by+geometric+properties+of+the+cortical+shell&amp;author=P+Augat&amp;author=H+Reeb&amp;author=LE+Claes&amp;volume=11&amp;publication_year=1996&amp;pages=1356-1363&amp;pmid=8864911&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R3"><span class="element-citation">Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>:263–265.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;author=B+Fry&amp;author=J+Maller&amp;author=MJ+Daly&amp;volume=21&amp;publication_year=2005&amp;pages=263-265&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R4"><span class="element-citation">Deng HW, Deng XT, Conway T, Xu FH, Heaney R, Recker RR. Determination of bone size of hip, spine, and wrist in human pedigrees by genetic and lifestyle factors. <span><span class="ref-journal">J Clin Densitom. </span>2002a;<span class="ref-vol">5</span>:45–56.</span> [<a href="/pubmed/11940728" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Densitom&amp;title=Determination+of+bone+size+of+hip,+spine,+and+wrist+in+human+pedigrees+by+genetic+and+lifestyle+factors&amp;author=HW+Deng&amp;author=XT+Deng&amp;author=T+Conway&amp;author=FH+Xu&amp;author=R+Heaney&amp;volume=5&amp;publication_year=2002a&amp;pages=45-56&amp;pmid=11940728&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R5"><span class="element-citation">Deng HW, Shen H, Xu FH, Deng HY, Conway T, Zhang HT, Recker RR. Tests of linkage and/or association of genes for vitamin D receptor, osteocalcin, and parathyroid hormone with bone mineral density. <span><span class="ref-journal">J Bone Miner Res. </span>2002b;<span class="ref-vol">17</span>:678–686.</span> [<a href="/pubmed/11918225" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Tests+of+linkage+and/or+association+of+genes+for+vitamin+D+receptor,+osteocalcin,+and+parathyroid+hormone+with+bone+mineral+density&amp;author=HW+Deng&amp;author=H+Shen&amp;author=FH+Xu&amp;author=HY+Deng&amp;author=T+Conway&amp;volume=17&amp;publication_year=2002b&amp;pages=678-686&amp;pmid=11918225&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R6"><span class="element-citation">Deng HW, Xu FH, Davies KM, Heaney R, Recker RR. Differences in bone mineral density, bone mineral content, and bone areal size in fracturing and non-fracturing women, and their interrelationships at the spine and hip. <span><span class="ref-journal">J Bone Miner Metab. </span>2002c;<span class="ref-vol">20</span>:358–366.</span> [<a href="/pubmed/12434164" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=Differences+in+bone+mineral+density,+bone+mineral+content,+and+bone+areal+size+in+fracturing+and+non-fracturing+women,+and+their+interrelationships+at+the+spine+and+hip&amp;author=HW+Deng&amp;author=FH+Xu&amp;author=KM+Davies&amp;author=R+Heaney&amp;author=RR+Recker&amp;volume=20&amp;publication_year=2002c&amp;pages=358-366&amp;pmid=12434164&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R7"><span class="element-citation">Dvornyk V, Liu XH, Shen H, Lei SF, Zhao LJ, Huang QR, Qin YJ, Jiang DK, Long JR, Zhang YY, Gong G, Recker RR, Deng HW. Differentiation of Caucasians and Chinese at bone mass candidate genes: implication for ethnic difference of bone mass. <span><span class="ref-journal">Ann Hum Genet. </span>2003;<span class="ref-vol">67</span>:216–227.</span> [<a href="/pubmed/12914574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Hum+Genet&amp;title=Differentiation+of+Caucasians+and+Chinese+at+bone+mass+candidate+genes:+implication+for+ethnic+difference+of+bone+mass&amp;author=V+Dvornyk&amp;author=XH+Liu&amp;author=H+Shen&amp;author=SF+Lei&amp;author=LJ+Zhao&amp;volume=67&amp;publication_year=2003&amp;pages=216-227&amp;pmid=12914574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R8"><span class="element-citation">Dvornyk V, Liu PY, Long JR, Zhang YY, Lei SF, Recker RR, Deng HW. Contribution of genotype and ethnicity to bone mineral density variation in Caucasians and Chinese: a test for five candidate genes for bone mass. <span><span class="ref-journal">Chin Med J (Engl) </span>2005;<span class="ref-vol">118</span>:1235–1244.</span> [<a href="/pubmed/16117875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chin+Med+J+(Engl)&amp;title=Contribution+of+genotype+and+ethnicity+to+bone+mineral+density+variation+in+Caucasians+and+Chinese:+a+test+for+five+candidate+genes+for+bone+mass&amp;author=V+Dvornyk&amp;author=PY+Liu&amp;author=JR+Long&amp;author=YY+Zhang&amp;author=SF+Lei&amp;volume=118&amp;publication_year=2005&amp;pages=1235-1244&amp;pmid=16117875&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R9"><span class="element-citation">Fang Y, van Meurs JB, Rivadeneira F, van Schoor NM, van Leeuwen JP, Lips P, Pols HA, Uitterlinden AG. Vitamin D receptor gene haplotype is associated with body height and bone size. <span><span class="ref-journal">J Clin Endocrinol Metab. </span>2007;<span class="ref-vol">92</span>:1491–1501.</span> [<a href="/pubmed/17213281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Endocrinol+Metab&amp;title=Vitamin+D+receptor+gene+haplotype+is+associated+with+body+height+and+bone+size&amp;author=Y+Fang&amp;author=JB+van+Meurs&amp;author=F+Rivadeneira&amp;author=NM+van+Schoor&amp;author=JP+van+Leeuwen&amp;volume=92&amp;publication_year=2007&amp;pages=1491-1501&amp;pmid=17213281&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R10"><span class="element-citation">Feng S, Wang S, Chen CC, Lan L. GWAPower: a statistical power calculation software for genome-wide association studies with quantitative traits. <span><span class="ref-journal">BMC Genet. </span>2011;<span class="ref-vol">12</span>:12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3036643/">PMC free article</a>]</span> [<a href="/pubmed/21255436" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet&amp;title=GWAPower:+a+statistical+power+calculation+software+for+genome-wide+association+studies+with+quantitative+traits&amp;author=S+Feng&amp;author=S+Wang&amp;author=CC+Chen&amp;author=L+Lan&amp;volume=12&amp;publication_year=2011&amp;pages=12&amp;pmid=21255436&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R11"><span class="element-citation">Fox KM, Cummings SR, Powell-Threets K, Stone K. Family history and risk of osteoporotic fracture Study of Osteoporotic Fractures Research Group. <span><span class="ref-journal">Osteoporos Int. </span>1998;<span class="ref-vol">8</span>:557–562.</span> [<a href="/pubmed/10326061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoporos+Int&amp;title=Family+history+and+risk+of+osteoporotic+fracture+Study+of+Osteoporotic+Fractures+Research+Group&amp;author=KM+Fox&amp;author=SR+Cummings&amp;author=K+Powell-Threets&amp;author=K+Stone&amp;volume=8&amp;publication_year=1998&amp;pages=557-562&amp;pmid=10326061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R12"><span class="element-citation">Lei SF, Deng FY, Liu XH, Huang QR, Qin Y, Zhou Q, Jiang DK, Li YM, Mo XY, Liu MY, Chen XD, Wu XS, Shen H, Dvornyk V, Zhao L, Recker RR, Deng HW. Polymorphisms of four bone mineral density candidate genes in Chinese populations and comparison with other populations of different ethnicity. <span><span class="ref-journal">J Bone Miner Metab. </span>2003;<span class="ref-vol">21</span>:34–42.</span> [<a href="/pubmed/12491092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=Polymorphisms+of+four+bone+mineral+density+candidate+genes+in+Chinese+populations+and+comparison+with+other+populations+of+different+ethnicity&amp;author=SF+Lei&amp;author=FY+Deng&amp;author=XH+Liu&amp;author=QR+Huang&amp;author=Y+Qin&amp;volume=21&amp;publication_year=2003&amp;pages=34-42&amp;pmid=12491092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R13"><span class="element-citation">Lei SF, Deng FY, Xiao SM, Chen XD, Deng HW. Association and haplotype analyses of the COL1A2 and ER-alpha gene polymorphisms with bone size and height in Chinese. <span><span class="ref-journal">Bone. </span>2005a;<span class="ref-vol">36</span>:533–541.</span> [<a href="/pubmed/15777651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Association+and+haplotype+analyses+of+the+COL1A2+and+ER-alpha+gene+polymorphisms+with+bone+size+and+height+in+Chinese&amp;author=SF+Lei&amp;author=FY+Deng&amp;author=SM+Xiao&amp;author=XD+Chen&amp;author=HW+Deng&amp;volume=36&amp;publication_year=2005a&amp;pages=533-541&amp;pmid=15777651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R14"><span class="element-citation">Lei SF, Liu YZ, Deng FY, Li YM, Li MX, Deng HW. Association and linkage analyses of interleukin-6 gene 634C/G polymorphism and bone phenotypes in Chinese. <span><span class="ref-journal">J Bone Miner Metab. </span>2005b;<span class="ref-vol">23</span>:323–328.</span> [<a href="/pubmed/15981030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=Association+and+linkage+analyses+of+interleukin-6+gene+634C/G+polymorphism+and+bone+phenotypes+in+Chinese&amp;author=SF+Lei&amp;author=YZ+Liu&amp;author=FY+Deng&amp;author=YM+Li&amp;author=MX+Li&amp;volume=23&amp;publication_year=2005b&amp;pages=323-328&amp;pmid=15981030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R15"><span class="element-citation">Lei SF, Wang YB, Liu MY, Mo XY, Deng HW. The VDR, COL1A1, PTH, and PTHR1 gene polymorphisms are not associated with bone size and height in Chinese nuclear families. <span><span class="ref-journal">J Bone Miner Metab. </span>2005c;<span class="ref-vol">23</span>:501–505.</span> [<a href="/pubmed/16261459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Metab&amp;title=The+VDR,+COL1A1,+PTH,+and+PTHR1+gene+polymorphisms+are+not+associated+with+bone+size+and+height+in+Chinese+nuclear+families&amp;author=SF+Lei&amp;author=YB+Wang&amp;author=MY+Liu&amp;author=XY+Mo&amp;author=HW+Deng&amp;volume=23&amp;publication_year=2005c&amp;pages=501-505&amp;pmid=16261459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R16"><span class="element-citation">Lei SF, Chen Y, Xiong DH, Li LM, Deng HW. Ethnic difference in osteoporosis-related phenotypes and its potential underlying genetic determination. <span><span class="ref-journal">J Musculoskelet Neuronal Interact. </span>2006;<span class="ref-vol">6</span>:36–46.</span> [<a href="/pubmed/16675888" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Musculoskelet+Neuronal+Interact&amp;title=Ethnic+difference+in+osteoporosis-related+phenotypes+and+its+potential+underlying+genetic+determination&amp;author=SF+Lei&amp;author=Y+Chen&amp;author=DH+Xiong&amp;author=LM+Li&amp;author=HW+Deng&amp;volume=6&amp;publication_year=2006&amp;pages=36-46&amp;pmid=16675888&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R17"><span class="element-citation">Liu YJ, Liu XH, Lei SF, Li MX, Deng HW. Alpha2-HS glycoprotein gene is associated with bone size at the hip in Chinese. <span><span class="ref-journal">Yi Chuan Xue Bao. </span>2005;<span class="ref-vol">32</span>:1128–1135.</span> [<a href="/pubmed/16318278" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Yi+Chuan+Xue+Bao&amp;title=Alpha2-HS+glycoprotein+gene+is+associated+with+bone+size+at+the+hip+in+Chinese&amp;author=YJ+Liu&amp;author=XH+Liu&amp;author=SF+Lei&amp;author=MX+Li&amp;author=HW+Deng&amp;volume=32&amp;publication_year=2005&amp;pages=1128-1135&amp;pmid=16318278&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R18"><span class="element-citation">Liu YZ, Wilson SG, Wang L, Liu XG, Guo YF, Li J, Yan H, Deloukas P, Soranzo N, Chinappen-Horsley U, Cervino A, Williams FM, Xiong DH, Zhang YP, Jin TB, Levy S, Papasian CJ, Drees BM, Hamilton JJ, Recker RR, Spector TD, Deng HW. Identification of PLCL1 gene for hip bone size variation in females in a genome-wide association study. <span><span class="ref-journal">PLoS One. </span>2008;<span class="ref-vol">3</span>:e3160.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2522269/">PMC free article</a>]</span> [<a href="/pubmed/18776929" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Identification+of+PLCL1+gene+for+hip+bone+size+variation+in+females+in+a+genome-wide+association+study&amp;author=YZ+Liu&amp;author=SG+Wilson&amp;author=L+Wang&amp;author=XG+Liu&amp;author=YF+Guo&amp;volume=3&amp;publication_year=2008&amp;pages=e3160&amp;pmid=18776929&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R19"><span class="element-citation">Long JR, Liu PY, Lu Y, Xiong DH, Zhao LJ, Zhang YY, Elze L, Recker RR, Deng HW. Association between COL1A1 gene polymorphisms and bone size in Caucasians. <span><span class="ref-journal">Eur J Hum Genet. </span>2004;<span class="ref-vol">12</span>:383–388.</span> [<a href="/pubmed/14722584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Hum+Genet&amp;title=Association+between+COL1A1+gene+polymorphisms+and+bone+size+in+Caucasians&amp;author=JR+Long&amp;author=PY+Liu&amp;author=Y+Lu&amp;author=DH+Xiong&amp;author=LJ+Zhao&amp;volume=12&amp;publication_year=2004&amp;pages=383-388&amp;pmid=14722584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R20"><span class="element-citation">Malkin I, Dahm S, Suk A, Kobyliansky E, Toliat M, Ruf N, Livshits G, Nurnberg P. Association of ANKH gene polymorphisms with radiographic hand bone size and geometry in a Chuvasha population. <span><span class="ref-journal">Bone. </span>2005;<span class="ref-vol">36</span>:365–373.</span> [<a href="/pubmed/15780964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=Association+of+ANKH+gene+polymorphisms+with+radiographic+hand+bone+size+and+geometry+in+a+Chuvasha+population&amp;author=I+Malkin&amp;author=S+Dahm&amp;author=A+Suk&amp;author=E+Kobyliansky&amp;author=M+Toliat&amp;volume=36&amp;publication_year=2005&amp;pages=365-373&amp;pmid=15780964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R21"><span class="element-citation">McCreadie BR, Goldstein SA. Biomechanics of fracture: is bone mineral density sufficient to assess risk? <span><span class="ref-journal">J Bone Miner Res. </span>2000;<span class="ref-vol">15</span>:2305–2308.</span> [<a href="/pubmed/11127195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Biomechanics+of+fracture:+is+bone+mineral+density+sufficient+to+assess+risk?&amp;author=BR+McCreadie&amp;author=SA+Goldstein&amp;volume=15&amp;publication_year=2000&amp;pages=2305-2308&amp;pmid=11127195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R22"><span class="element-citation">Montagnani A, Gonnelli S, Alessandri M, Nuti R. Osteoporosis and risk of fracture in patients with diabetes: an update. <span><span class="ref-journal">Aging Clin Exp Res. </span>2011;<span class="ref-vol">23</span>:84–90.</span> [<a href="/pubmed/21743287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aging+Clin+Exp+Res&amp;title=Osteoporosis+and+risk+of+fracture+in+patients+with+diabetes:+an+update&amp;author=A+Montagnani&amp;author=S+Gonnelli&amp;author=M+Alessandri&amp;author=R+Nuti&amp;volume=23&amp;publication_year=2011&amp;pages=84-90&amp;pmid=21743287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R23"><span class="element-citation">Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, Maller J, Sklar P, de Bakker PI, Daly MJ, Sham PC. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;author=L+Thomas&amp;author=MA+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R24"><span class="element-citation">Seeman E, Duan Y, Fong C, Edmonds J. Fracture site-specific deficits in bone size and volumetric density in men with spine or hip fractures. <span><span class="ref-journal">J Bone Miner Res. </span>2001;<span class="ref-vol">16</span>:120–127.</span> [<a href="/pubmed/11149475" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=Fracture+site-specific+deficits+in+bone+size+and+volumetric+density+in+men+with+spine+or+hip+fractures&amp;author=E+Seeman&amp;author=Y+Duan&amp;author=C+Fong&amp;author=J+Edmonds&amp;volume=16&amp;publication_year=2001&amp;pages=120-127&amp;pmid=11149475&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R25"><span class="element-citation">Shen H, Long JR, Xiong DH, Guo YF, Xiao P, Liu YZ, Zhao LJ, Liu YJ, Deng HY, Li JL, Recker RR, Deng HW. A genomewide scan for quantitative trait loci underlying areal bone size variation in 451 Caucasian families. <span><span class="ref-journal">J Med Genet. </span>2006;<span class="ref-vol">43</span>:873–880.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2563191/">PMC free article</a>]</span> [<a href="/pubmed/16825438" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=A+genomewide+scan+for+quantitative+trait+loci+underlying+areal+bone+size+variation+in+451+Caucasian+families&amp;author=H+Shen&amp;author=JR+Long&amp;author=DH+Xiong&amp;author=YF+Guo&amp;author=P+Xiao&amp;volume=43&amp;publication_year=2006&amp;pages=873-880&amp;pmid=16825438&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R26"><span class="element-citation">Tan LJ, Liu YZ, Xiao P, Yang F, Tang ZH, Liu PY, Recker RR, Deng HW. Evidence for major pleiotropic effects on bone size variation from a principal component analysis of 451 Caucasian families. <span><span class="ref-journal">Acta Pharmacol Sin. </span>2008;<span class="ref-vol">29</span>:745–751.</span> [<a href="/pubmed/18501122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Pharmacol+Sin&amp;title=Evidence+for+major+pleiotropic+effects+on+bone+size+variation+from+a+principal+component+analysis+of+451+Caucasian+families&amp;author=LJ+Tan&amp;author=YZ+Liu&amp;author=P+Xiao&amp;author=F+Yang&amp;author=ZH+Tang&amp;volume=29&amp;publication_year=2008&amp;pages=745-751&amp;pmid=18501122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R27"><span class="element-citation">Ueda T, Furusawa T, Kurahashi T, Tessarollo L, Bustin M. The nucleosome binding protein HMGN3 modulates the transcription profile of pancreatic beta cells and affects insulin secretion. <span><span class="ref-journal">Mol Cell Biol. </span>2009;<span class="ref-vol">29</span>:5264–5276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2747976/">PMC free article</a>]</span> [<a href="/pubmed/19651901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Cell+Biol&amp;title=The+nucleosome+binding+protein+HMGN3+modulates+the+transcription+profile+of+pancreatic+beta+cells+and+affects+insulin+secretion&amp;author=T+Ueda&amp;author=T+Furusawa&amp;author=T+Kurahashi&amp;author=L+Tessarollo&amp;author=M+Bustin&amp;volume=29&amp;publication_year=2009&amp;pages=5264-5276&amp;pmid=19651901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R28"><span class="element-citation">van der Sluis IM, de Muinck Keizer-Schrama SM, Krenning EP, Pols HA, Uitterlinden AG. Vitamin D receptor gene polymorphism predicts height and bone size, rather than bone density in children and young adults. <span><span class="ref-journal">Calcif Tissue Int. </span>2003;<span class="ref-vol">73</span>:332–338.</span> [<a href="/pubmed/12874698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Calcif+Tissue+Int&amp;title=Vitamin+D+receptor+gene+polymorphism+predicts+height+and+bone+size,+rather+than+bone+density+in+children+and+young+adults&amp;author=IM+van+der+Sluis&amp;author=SM+de+Muinck+Keizer-Schrama&amp;author=EP+Krenning&amp;author=HA+Pols&amp;author=AG+Uitterlinden&amp;volume=73&amp;publication_year=2003&amp;pages=332-338&amp;pmid=12874698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R29"><span class="element-citation">Xu FH, Liu YJ, Deng H, Huang QY, Zhao LJ, Shen H, Liu YZ, Dvornyk V, Conway T, Li JL, Davies KM, Recker RR, Deng HW. A follow-up linkage study for bone size variation in an extended sample. <span><span class="ref-journal">Bone. </span>2004;<span class="ref-vol">35</span>:777–784.</span> [<a href="/pubmed/15336616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bone&amp;title=A+follow-up+linkage+study+for+bone+size+variation+in+an+extended+sample&amp;author=FH+Xu&amp;author=YJ+Liu&amp;author=H+Deng&amp;author=QY+Huang&amp;author=LJ+Zhao&amp;volume=35&amp;publication_year=2004&amp;pages=777-784&amp;pmid=15336616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li><li id="R30"><span class="element-citation">Zmuda JM, Cauley JA, Kuller LH, Ferrell RE. A common promotor variant in the cytochrome P450c17alpha (CYP17) gene is associated with bioavailability testosterone levels and bone size in men. <span><span class="ref-journal">J Bone Miner Res. </span>2001;<span class="ref-vol">16</span>:911–917.</span> [<a href="/pubmed/11341336" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bone+Miner+Res&amp;title=A+common+promotor+variant+in+the+cytochrome+P450c17alpha+(CYP17)+gene+is+associated+with+bioavailability+testosterone+levels+and+bone+size+in+men&amp;author=JM+Zmuda&amp;author=JA+Cauley&amp;author=LH+Kuller&amp;author=RE+Ferrell&amp;volume=16&amp;publication_year=2001&amp;pages=911-917&amp;pmid=11341336&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4450081&amp;issue-id=253579&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></li></ul></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139696396376160title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Materials and methods</a></li><li><a href="#S9title">Results</a></li><li><a href="#S11title">Discussion</a></li><li><a href="#S13title">Supplementary Material</a></li><li><a href="#S12title">Acknowledgments</a></li><li><a href="#idm139696356755536title">Footnotes</a></li><li><a href="#idm139696357367440aff-infotitle">Contributor Information</a></li><li><a href="#idm139696429979216title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4450081/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4450081/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4450081/pdf/nihms386049.pdf">PDF (458K)</a></li> | <li><a href="#" data-citationid="PMC4450081" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4450081%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4450081%2F&amp;text=Genome-wide%20association%20study%20identifies%20HMGN3%20locus%20for%20spine%20bone%20size%20variation%20in%20Chinese" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4450081%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="21947420" data-aiid="4450081" data-aid="4450081" data-iid="253579" data-domainid="319" data-domain="nihpa" data-accid="PMC4450081" data-md5="6189a9b2f4501e004987fe90c2d3575d"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T07:48:23-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D3E7ADD5349E10000000000E7009A&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4450081%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4450081/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>