<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4379124/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genomewide association study of autism using the Simons Simplex
Collection: Does reducing phenotypic heterogeneity in autism increase genetic
homogeneity?</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379124/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Biological psychiatry"><meta name="citation_title" content="A genomewide association study of autism using the Simons Simplex
Collection: Does reducing phenotypic heterogeneity in autism increase genetic
homogeneity?"><meta name="citation_authors" content="Pauline Chaste, Lambertus Klei, Stephan J. Sanders, Vanessa Hus, Michael T. Murtha, Jennifer K. Lowe, A. Jeremy Willsey, Daniel Moreno-De-Luca, Timothy W. Yu, Eric Fombonne, Daniel Geschwind, Dorothy E. Grice, David H. Ledbetter, Shrikant M. Mane, Donna M. Martin, Eric M. Morrow, Christopher A. Walsh, James S. Sutcliffe, Christa Lese Martin, Arthur L. Beaudet, Catherine Lord, Matthew W. State, Edwin H. Cook, Jr., Bernie Devlin"><meta name="citation_date" content="1 May 2015"><meta name="citation_issue" content="9"><meta name="citation_volume" content="77"><meta name="citation_firstpage" content="775"><meta name="citation_doi" content="10.1016/j.biopsych.2014.09.017"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4379124/?report=abstract"><meta name="citation_pmid" content="25534755"><meta name="DC.Title" content="A genomewide association study of autism using the Simons Simplex
Collection: Does reducing phenotypic heterogeneity in autism increase genetic
homogeneity?"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Pauline Chaste"><meta name="DC.Contributor" content="Lambertus Klei"><meta name="DC.Contributor" content="Stephan J. Sanders"><meta name="DC.Contributor" content="Vanessa Hus"><meta name="DC.Contributor" content="Michael T. Murtha"><meta name="DC.Contributor" content="Jennifer K. Lowe"><meta name="DC.Contributor" content="A. Jeremy Willsey"><meta name="DC.Contributor" content="Daniel Moreno-De-Luca"><meta name="DC.Contributor" content="Timothy W. Yu"><meta name="DC.Contributor" content="Eric Fombonne"><meta name="DC.Contributor" content="Daniel Geschwind"><meta name="DC.Contributor" content="Dorothy E. Grice"><meta name="DC.Contributor" content="David H. Ledbetter"><meta name="DC.Contributor" content="Shrikant M. Mane"><meta name="DC.Contributor" content="Donna M. Martin"><meta name="DC.Contributor" content="Eric M. Morrow"><meta name="DC.Contributor" content="Christopher A. Walsh"><meta name="DC.Contributor" content="James S. Sutcliffe"><meta name="DC.Contributor" content="Christa Lese Martin"><meta name="DC.Contributor" content="Arthur L. Beaudet"><meta name="DC.Contributor" content="Catherine Lord"><meta name="DC.Contributor" content="Matthew W. State"><meta name="DC.Contributor" content="Edwin H. Cook, Jr."><meta name="DC.Contributor" content="Bernie Devlin"><meta name="DC.Date" content="2015 May 1"><meta name="DC.Identifier" content="10.1016/j.biopsych.2014.09.017"><meta name="DC.Language" content="en"><meta property="og:title" content="A genomewide association study of autism using the Simons Simplex
Collection: Does reducing phenotypic heterogeneity in autism increase genetic
homogeneity?"><meta property="og:type" content="article"><meta property="og:description" content="Phenotypic heterogeneity in autism has long been conjectured to be a
major hindrance to the discovery of genetic risk factors, leading to
numerous attempts to stratify children based on phenotype to increase power
of discovery studies. This approach, ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379124/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4379124/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4379124/pdf/nihms631893.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8918EFDD5711C1000000000ABA069D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4379124%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4379124</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Biol Psychiatry</a></span>. Author manuscript; available in PMC 2016 May 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25534755" target="pmc_ext" ref="reftype=publisher&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Biol Psychiatry. 2015 May 1; 77(9): 775–784. </span></a></div><span class="fm-vol-iss-date">Published online 2014 Sep 30. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.biopsych.2014.09.017" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.biopsych.2014.09.017</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4379124</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS631893</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25534755">25534755</a></div></div></div></div><h1 class="content-title">A genomewide association study of autism using the Simons Simplex
Collection: Does reducing phenotypic heterogeneity in autism increase genetic
homogeneity?</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Chaste%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324010162336" co-class="co-affbox">Pauline Chaste</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>*</sup> <a href="/pubmed/?term=Klei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324010158976" co-class="co-affbox">Lambertus Klei</a>,<sup>1</sup> <a href="/pubmed/?term=Sanders%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324010156992" co-class="co-affbox">Stephan J. Sanders</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Hus%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324013118144" co-class="co-affbox">Vanessa Hus</a>,<sup>6</sup> <a href="/pubmed/?term=Murtha%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324013116160" co-class="co-affbox">Michael T. Murtha</a>,<sup>7</sup> <a href="/pubmed/?term=Lowe%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324013114176" co-class="co-affbox">Jennifer K. Lowe</a>,<sup>8</sup> <a href="/pubmed/?term=Willsey%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324013112192" co-class="co-affbox">A. Jeremy Willsey</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Moreno-De-Luca%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324004943888" co-class="co-affbox">Daniel Moreno-De-Luca</a>,<sup>7,</sup><sup>9</sup> <a href="/pubmed/?term=Yu%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324004941168" co-class="co-affbox">Timothy W. Yu</a>,<sup>10</sup> <a href="/pubmed/?term=Fombonne%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324004939184" co-class="co-affbox">Eric Fombonne</a>,<sup>11</sup> <a href="/pubmed/?term=Geschwind%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324004937200" co-class="co-affbox">Daniel Geschwind</a>,<sup>12</sup> <a href="/pubmed/?term=Grice%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324004935216" co-class="co-affbox">Dorothy E. Grice</a>,<sup>13</sup> <a href="/pubmed/?term=Ledbetter%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006356688" co-class="co-affbox">David H. Ledbetter</a>,<sup>14</sup> <a href="/pubmed/?term=Mane%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006354816" co-class="co-affbox">Shrikant M. Mane</a>,<sup>15</sup> <a href="/pubmed/?term=Martin%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006352832" co-class="co-affbox">Donna M. Martin</a>,<sup>16</sup> <a href="/pubmed/?term=Morrow%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006350848" co-class="co-affbox">Eric M. Morrow</a>,<sup>17,</sup><sup>18</sup> <a href="/pubmed/?term=Walsh%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006348128" co-class="co-affbox">Christopher A. Walsh</a>,<sup>19</sup> <a href="/pubmed/?term=Sutcliffe%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324006346144" co-class="co-affbox">James S. Sutcliffe</a>,<sup>20</sup> <a href="/pubmed/?term=Martin%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007576896" co-class="co-affbox">Christa Lese Martin</a>,<sup>14</sup> <a href="/pubmed/?term=Beaudet%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007574912" co-class="co-affbox">Arthur L. Beaudet</a>,<sup>21</sup> <a href="/pubmed/?term=Lord%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007572928" co-class="co-affbox">Catherine Lord</a>,<sup>22</sup> <a href="/pubmed/?term=State%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007570944" co-class="co-affbox">Matthew W. State</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Cook%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007568224" co-class="co-affbox">Edwin H. Cook, Jr.</a>,<sup>23</sup> and  <a href="/pubmed/?term=Devlin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755" class="affpopup" co-rid="_co_idm140324007286576" co-class="co-affbox">Bernie Devlin</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140324010162336"><h3 class="no_margin">Pauline Chaste</h3><p><sup>1</sup>Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, Pennsylvania, USA</p><p><sup>2</sup>FondaMental Foundation, Créteil, France</p><p><sup>3</sup>Centre hospitalier Sainte Anne, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Chaste%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Pauline Chaste</a></div></div><div id="_co_idm140324010158976"><h3 class="no_margin">Lambertus Klei</h3><p><sup>1</sup>Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, Pennsylvania, USA</p><div>Find articles by <a href="/pubmed/?term=Klei%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Lambertus Klei</a></div></div><div id="_co_idm140324010156992"><h3 class="no_margin">Stephan J. Sanders</h3><p><sup>4</sup>Department of Genetics, Yale University School of Medicine, New Haven, Connecticut, USA</p><p><sup>5</sup>Department of Psychiatry, University of California at San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Sanders%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Stephan J. Sanders</a></div></div><div id="_co_idm140324013118144"><h3 class="no_margin">Vanessa Hus</h3><p><sup>6</sup>Department of Psychology, University of Michigan, Ann Arbor, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Hus%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Vanessa Hus</a></div></div><div id="_co_idm140324013116160"><h3 class="no_margin">Michael T. Murtha</h3><p><sup>7</sup>Program on Neurogenetics, Yale University School of Medicine, New Haven, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=Murtha%20MT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Michael T. Murtha</a></div></div><div id="_co_idm140324013114176"><h3 class="no_margin">Jennifer K. Lowe</h3><p><sup>8</sup>Neurogenetics Program, Department of Neurology and Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Lowe%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Jennifer K. Lowe</a></div></div><div id="_co_idm140324013112192"><h3 class="no_margin">A. Jeremy Willsey</h3><p><sup>4</sup>Department of Genetics, Yale University School of Medicine, New Haven, Connecticut, USA</p><p><sup>5</sup>Department of Psychiatry, University of California at San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=Willsey%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">A. Jeremy Willsey</a></div></div><div id="_co_idm140324004943888"><h3 class="no_margin">Daniel Moreno-De-Luca</h3><p><sup>7</sup>Program on Neurogenetics, Yale University School of Medicine, New Haven, Connecticut, USA</p><p><sup>9</sup>Department of Psychiatry, Yale University School of Medicine, New Haven, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=Moreno-De-Luca%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Daniel Moreno-De-Luca</a></div></div><div id="_co_idm140324004941168"><h3 class="no_margin">Timothy W. Yu</h3><p><sup>10</sup>Division of Genetics, Children's Hospital Boston, Harvard Medical School, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Yu%20TW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Timothy W. Yu</a></div></div><div id="_co_idm140324004939184"><h3 class="no_margin">Eric Fombonne</h3><p><sup>11</sup>Department of Psychiatry and Institute for Development and disability, Oregon Health &amp; Science University, Portland, Oregon, USA</p><div>Find articles by <a href="/pubmed/?term=Fombonne%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Eric Fombonne</a></div></div><div id="_co_idm140324004937200"><h3 class="no_margin">Daniel Geschwind</h3><p><sup>12</sup>Neurogenetics Program, Department of Neurology and Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, USA</p><div>Find articles by <a href="/pubmed/?term=Geschwind%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Daniel Geschwind</a></div></div><div id="_co_idm140324004935216"><h3 class="no_margin">Dorothy E. Grice</h3><p><sup>13</sup>Department of Psychiatry, Mount Sinai School of Medicine, New York, New York, USA</p><div>Find articles by <a href="/pubmed/?term=Grice%20DE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Dorothy E. Grice</a></div></div><div id="_co_idm140324006356688"><h3 class="no_margin">David H. Ledbetter</h3><p><sup>14</sup>Autism and Developmental Medicine Institute, Geisinger Health System, Danville, Pennsylvania, USA</p><div>Find articles by <a href="/pubmed/?term=Ledbetter%20DH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">David H. Ledbetter</a></div></div><div id="_co_idm140324006354816"><h3 class="no_margin">Shrikant M. Mane</h3><p><sup>15</sup>Yale Center for Genome Analysis, Orange, Connecticut, USA</p><div>Find articles by <a href="/pubmed/?term=Mane%20SM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Shrikant M. Mane</a></div></div><div id="_co_idm140324006352832"><h3 class="no_margin">Donna M. Martin</h3><p><sup>16</sup>Departments of Pediatrics and Human Genetics, University of Michigan Medical Center, Ann Arbor, Michigan, USA</p><div>Find articles by <a href="/pubmed/?term=Martin%20DM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Donna M. Martin</a></div></div><div id="_co_idm140324006350848"><h3 class="no_margin">Eric M. Morrow</h3><p><sup>17</sup>Department of Molecular Biology, Cell Biology and Biochemistry, Brown University, Providence, Rhode Island, USA</p><p><sup>18</sup>Department of Psychiatry and Human Behavior, Brown University, Providence, Rhode Island, USA</p><div>Find articles by <a href="/pubmed/?term=Morrow%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Eric M. Morrow</a></div></div><div id="_co_idm140324006348128"><h3 class="no_margin">Christopher A. Walsh</h3><p><sup>19</sup>Howard Hughes Medical Institute and Division of Genetics, Children's Hospital Boston, and Neurology and Pediatrics, Harvard Medical School Center for Life Sciences, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Walsh%20CA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Christopher A. Walsh</a></div></div><div id="_co_idm140324006346144"><h3 class="no_margin">James S. Sutcliffe</h3><p><sup>20</sup>Departments of Molecular Physiology &amp; Biophysics and Psychiatry, Vanderbilt Brain Institute, Vanderbilt University, Nashville, TN, USA</p><div>Find articles by <a href="/pubmed/?term=Sutcliffe%20JS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">James S. Sutcliffe</a></div></div><div id="_co_idm140324007576896"><h3 class="no_margin">Christa Lese Martin</h3><p><sup>14</sup>Autism and Developmental Medicine Institute, Geisinger Health System, Danville, Pennsylvania, USA</p><div>Find articles by <a href="/pubmed/?term=Martin%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Christa Lese Martin</a></div></div><div id="_co_idm140324007574912"><h3 class="no_margin">Arthur L. Beaudet</h3><p><sup>21</sup>Department of Human and Molecular Genetics, Baylor College of Medicine, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Beaudet%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Arthur L. Beaudet</a></div></div><div id="_co_idm140324007572928"><h3 class="no_margin">Catherine Lord</h3><p><sup>22</sup>Center for Autism and the Developing Brain, Weill Cornell Medical College, White Plains, New York, USA</p><div>Find articles by <a href="/pubmed/?term=Lord%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Catherine Lord</a></div></div><div id="_co_idm140324007570944"><h3 class="no_margin">Matthew W. State</h3><p><sup>4</sup>Department of Genetics, Yale University School of Medicine, New Haven, Connecticut, USA</p><p><sup>5</sup>Department of Psychiatry, University of California at San Francisco, California, USA</p><div>Find articles by <a href="/pubmed/?term=State%20MW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Matthew W. State</a></div></div><div id="_co_idm140324007568224"><h3 class="no_margin">Edwin H. Cook, Jr.</h3><p><sup>23</sup>Institute for Juvenile Research, Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Cook%20EH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Edwin H. Cook, Jr.</a></div></div><div id="_co_idm140324007286576"><h3 class="no_margin">Bernie Devlin</h3><p><sup>1</sup>Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, Pennsylvania, USA</p><div>Find articles by <a href="/pubmed/?term=Devlin%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25534755">Bernie Devlin</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140324004728560_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140324004728560_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140324004728560_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Psychiatry, University of Pittsburgh School of Medicine, Pittsburgh, Pennsylvania, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>FondaMental Foundation, Créteil, France</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Centre hospitalier Sainte Anne, Paris, France</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Genetics, Yale University School of Medicine, New Haven, Connecticut, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Psychiatry, University of California at San Francisco, California, USA</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Psychology, University of Michigan, Ann Arbor, MI, USA</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Program on Neurogenetics, Yale University School of Medicine, New Haven, Connecticut, USA</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Neurogenetics Program, Department of Neurology and Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Psychiatry, Yale University School of Medicine, New Haven, Connecticut, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Division of Genetics, Children's Hospital Boston, Harvard Medical School, Boston, Massachusetts, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Psychiatry and Institute for Development and disability, Oregon Health &amp; Science University, Portland, Oregon, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Neurogenetics Program, Department of Neurology and Center for Autism Research and Treatment, Semel Institute, David Geffen School of Medicine, University of California Los Angeles, Los Angeles, California, USA</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Department of Psychiatry, Mount Sinai School of Medicine, New York, New York, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Autism and Developmental Medicine Institute, Geisinger Health System, Danville, Pennsylvania, USA</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Yale Center for Genome Analysis, Orange, Connecticut, USA</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Departments of Pediatrics and Human Genetics, University of Michigan Medical Center, Ann Arbor, Michigan, USA</div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Molecular Biology, Cell Biology and Biochemistry, Brown University, Providence, Rhode Island, USA</div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Department of Psychiatry and Human Behavior, Brown University, Providence, Rhode Island, USA</div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Howard Hughes Medical Institute and Division of Genetics, Children's Hospital Boston, and Neurology and Pediatrics, Harvard Medical School Center for Life Sciences, Boston, Massachusetts, USA</div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>Departments of Molecular Physiology &amp; Biophysics and Psychiatry, Vanderbilt Brain Institute, Vanderbilt University, Nashville, TN, USA</div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Department of Human and Molecular Genetics, Baylor College of Medicine, Houston, Texas, USA</div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Center for Autism and the Developing Brain, Weill Cornell Medical College, White Plains, New York, USA</div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Institute for Juvenile Research, Department of Psychiatry, University of Illinois at Chicago, Chicago, Illinois, USA</div><div id="FN1"><sup>*</sup>Corresponding author: Pauline Chaste, Centre hospitalier Sainte Anne, 1 rue Cabanis, 75014 Paris, France, Phone: 33-1-45-65-61-05, <a href="mailto:dev@null" data-email="rf.enna-etnias-hc@etsahc.p" class="oemail">rf.enna-etnias-hc@etsahc.p</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140324004728560_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div><a href="#FN4">Publisher's Disclaimer</a></div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=25534755" target="pmc_ext" ref="reftype=publisher&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Biol Psychiatry</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4379124/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>supplement.</strong></div><div><a href="/pmc/articles/PMC4379124/bin/NIHMS631893-supplement.pdf" data-ga-action="click_feat_suppl">NIHMS631893-supplement.pdf</a><span style="color:gray"> (97K)</span></div><div class="small guid">GUID:&nbsp;D0C7280B-AF05-477A-A600-95407A205D0E</div></div></dd></dl></div><div id="idm140324007628736" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140324007628736title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P1" class="p p-first-last">Phenotypic heterogeneity in autism has long been conjectured to be a
major hindrance to the discovery of genetic risk factors, leading to
numerous attempts to stratify children based on phenotype to increase power
of discovery studies. This approach, however, is based on the hypothesis
that phenotypic heterogeneity closely maps to genetic variation, which has
not been tested. Our study examines the impact of sub-phenotyping of a
well-characterized ASD sample on genetic homogeneity and the ability to
discover common genetic variants conferring liability to ASD.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">Genome-wide genotypic data of 2576 families from the Simons Simplex
Collection (SSC) were analyzed in the overall sample and phenotypic
subgroups defined on the basis of diagnosis, IQ, and symptom profiles. We
conducted a family-based association study as well as estimating
heritability and evaluating allele scores for each phenotypic subgroup.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">Association analyses revealed no genome-wide significant association
signal. Sub-phenotyping did not increase power substantially. Moreover,
allele scores built from the most associated SNPs, based on the odds ratio
in the full sample, predicted case status in subsets of the sample equally
well and heritability estimates were very similar for all subgroups.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusions</h3><p id="P4" class="p p-first-last">In genome-wide association analysis of the SSC sample, reducing
phenotypic heterogeneity had at most a modest impact on genetic homogeneity.
Our results are based on a relatively small sample, one with greater
homogeneity than the entire population; if they apply more broadly, they
imply that analysis of sub-phenotypes is not a productive path forward for
discovering genetic risk variants in ASD.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">genetics, gwas, autism, phenotype, heterogeneity, power</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P5" class="p p-first">Beyond its core diagnostic features, the presentation of autism spectrum
disorder (ASD) often varies widely. Indeed many clinicians and researchers have
remarked on its phenotypic heterogeneity and speculated that this heterogeneity is
related to genetic variation (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037188">1</a>). These
observations have led to the conjecture that a means of discovering novel genetic
variation underlying liability for ASD is to amass a large sample and subset the
sample on phenotypic dimensions (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037119">2</a>, <a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037191">3</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037179">4</a>). The
success of this approach relies on a relatively simple relationship between genetic
and phenotypic variation. On the other hand, if the relationship were quite complex,
so that myriad combinations of genetic variants could lead to similar phenotypic
presentation, then this approach to discovery would be unlikely to be more
successful than simply analyzing the total, albeit heterogeneous sample.</p><p id="P6">Such phenotypic sub-setting has successfully led to the identification of
genetic variants in neurodevelopmental disorders. The severe developmental
regression and stereotypical hand movements that typify Rett syndrome led to the
identification of causal mutations in <em>MECP2</em> (<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037123">5</a>) and <em>FOXG1</em> (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037110">6</a>). Conversely, identical 3Mb deletions at the 22q11.2
locus lead to extensive phenotypic heterogeneity across a range of neuropsychiatric
disorders (<a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037168">7</a>).</p><p id="P7">The results of several linkage studies in autism support the conjecture that
stratifying children based on their phenotype yields genetically more homogeneous
subgroups. Notably, for several studies the signal from linkage analysis increased
substantially when the phenotype was narrowly defined, either using endophenotypes
(<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037157">8</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037108">9</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037122">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037174">11</a>) or dimensions stemming from factor analyses (<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037112">12</a>, <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037130">13</a>,
<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037132">14</a>). Nonetheless the limited congruence
of results from larger samples (<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559204800">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037143">16</a>) and from the smaller samples suggests that
sub-phenotyping has limited impact on reducing genetic heterogeneity.</p><p id="P8">In contrast to linkage studies of psychiatric disorders, replicable
association findings for common genetic variants are now becoming commonplace. This
is especially true for genetic studies of schizophrenia, presumably due in large
part to the tens of thousands of samples amassed for study (<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037184">17</a>, <a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037180">18</a>, <a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037140">19</a>). Indeed the number of discoveries and their
replication are tightly correlated with sample size and comports with statistical
theory of power for factors of small effect – as indeed all common variants
affecting liability for schizophrenia appear to be. Aside from sample size, which
lags for ASD, the genetics of ASD and schizophrenia have strong parallels. Liability
for both appears to trace to common and rare variants, with the balance of liability
– at the population level – tilted toward common variants (<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037138">20</a>, <a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037160">21</a>).</p><p id="P9">For ASD there is now sound evidence that common variants, en masse, exert a
substantial effect on ASD liability. Heritability of autism attributable to common
variants is estimated to be 40-60% (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037175">22</a>) and the variance of the phenotype explained by allele scores derived
from GWAS results has been shown to be significant (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037154">23</a>). Thus, common variants play a major role in the pathophysiology of
autism. Yet identifying even a small portion of those variants will be a major
challenge for the next few years. Two paths for progress in this venture seem
apparent: (1) use phenotypes to define clinical subgroups and thereby identify
genetic variants accounting for more of the risk in the subgroup; or (2) genotype
more affected and unaffected subjects or trios to increase sample size. While the
results of schizophrenia studies suggest the second option is a successful path, the
impact of reducing phenotypic heterogeneity on the ability to detect associated
common variants remains to be determined.</p><p id="P10">A number of studies have attempted to identify informative clinical
sub-phenotypes to reduce genetic heterogeneity and results from several studies
suggest that certain clinical variables do improve statistical power of genetic
analyses. Of these phenotypes, overall severity (<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037142">4</a>), Insistence on Sameness (IS) scores (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037150">24</a>, <a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037141">3</a>), and IQ (<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037195">10</a>, <a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037197">25</a>) have been
proposed as defining clinical subgroups. Moreover, a recent study has built on the
results of factor analyzes of autistic symptoms. Indeed most factor studies suggest
that the structure of ASD phenotype is best conceptualized as a dyad of symptoms
with distinct social communication and repetitive behavior dimensions (<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037118">26</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037126">27</a>,
<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037162">28</a>). Georgiades et al. (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037158">2</a>), using a mixture model, find distinct profiles of
relative severity of social-communicative impairment and repetitive behaviors.
Finally, while not beyond reasonable doubt (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037145">29</a>, <a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037146">30</a>), some studies suggest
that DSM-IV categorical diagnosis could relate to more specific genetic liability
(<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037135">23</a>, <a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_471605083">31</a>).</p><p id="P11" class="p p-last">While all of the aforementioned studies contain promising results, it is
difficult to judge whether using more homogeneous presentations of ASD, as these
studies do, truly garner more power or not. Indeed, although this approach is
relatively common across the fields within psychiatric genetics, insofar as we are
aware there has been no rigorous analysis of whether these approaches generate
greater genetic homogeneity and greater power to detect risk loci. Here we take a
first step in that direction. We examine how phenotypic subgrouping impacts genetic
heterogeneity and the ability to discover common genetics variants affecting risk
for ASD. To do so we use the genome-wide genotypic data from the SSC (<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037186">32</a>) the largest cohort of autism simplex
families amassed to date. Phenotypic subgroups are defined on the basis of ASD
diagnostic category, IQ, ASD severity, insistence on sameness, and symptom profiles.
We then conduct a family-based association study, as well as estimating heritability
and evaluating allele scores for each phenotypic subgroup. The results from this
study suggest that, in ASD, greater phenotypic homogeneity has only a modest impact
on genetic homogeneity and in only a few instances does it appear likely to be a
productive path for discovering genetic risk variants.</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Method</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Genotyping</h3><p id="P12" class="p p-first">Individuals from 2576 SSC families were genotyped for a million or more
single nucleotide polymorphisms (SNPs) on one of three array versions - Illumina
1Mv1 (330 families), Illumina 1Mv3 Duo (1185 families), or Illumina
HumanOmni2.5M (1061 families) - at the Yale Center for Genomic Analysis. Members
of each family were analyzed on the same array version.</p><p id="P13" class="p p-last">All individuals had call rates &gt; 96.9% and thus all
individuals were retained for analysis. Call rates for SNPs were determined for
the 3,045,301 genotyped SNPs on the autosomes and chromosome X, in subsets of
SNPs based on which platform produced the genotypes (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>). A total of 47,582 SNPs
had a non-call rate &gt; 2.5% and were subsequently removed from
the analysis. Data were checked for reported sex, familial relationships and
duplicates.</p></div><div id="S8" class="sec"><h3 id="S8title">Phenotypes</h3><p id="P14" class="p p-first">Phenotypes were selected based on the previous research aimed at
defining informative subgroups. Eleven groups (<a href="/pmc/articles/PMC4379124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) were defined based on diagnosis, IQ, age at first words,
ASD severity, insistence sameness, and symptom profiles. Regression and epilepsy
were considered for analysis (<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037183">33</a>, <a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037148">34</a>), but were not included because they
were relatively uncommon in the SSC cohort.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary of phenotypic subsets</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324080855312" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="left">Abbreviated name</th><th rowspan="1" colspan="1" valign="top" align="left">Criteria</th><th rowspan="1" colspan="1" valign="top" align="left">N cases</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Whole sample</td><td rowspan="1" colspan="1" valign="top" align="left">All ASD</td><td rowspan="1" colspan="1" valign="top" align="left">ASD on both ADI-R and ADOS</td><td rowspan="1" colspan="1" valign="top" align="left">2,575</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Restricted autism diagnosis</td><td rowspan="1" colspan="1" valign="top" align="left">Autism</td><td rowspan="1" colspan="1" valign="top" align="left">ADI-R “autism” &amp; ADOS
“autism”</td><td rowspan="1" colspan="1" valign="top" align="left">2,088</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Verbal IQ&gt; 60</td><td rowspan="1" colspan="1" valign="top" align="left">High vIQ</td><td rowspan="1" colspan="1" valign="top" align="left">Verbal IQ &gt; 60</td><td rowspan="1" colspan="1" valign="top" align="left">1,880</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS ASD symptoms</td><td rowspan="1" colspan="1" valign="top" align="left">Severe ASD</td><td rowspan="1" colspan="1" valign="top" align="left">Overall ADOS Calibrated Severity Score (CSS)
≥ 8</td><td rowspan="1" colspan="1" valign="top" align="left">1,200</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS RRB symptoms</td><td rowspan="1" colspan="1" valign="top" align="left">Severe RBB</td><td rowspan="1" colspan="1" valign="top" align="left">ADOS Repetitive and Restrictive Behavior (RRB)
CSS ≥ 8</td><td rowspan="1" colspan="1" valign="top" align="left">1,589</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS social affect impairment</td><td rowspan="1" colspan="1" valign="top" align="left">Severe SA</td><td rowspan="1" colspan="1" valign="top" align="left">ADOS Social AffectCSS ≥ 8</td><td rowspan="1" colspan="1" valign="top" align="left">1,091</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R circumscribed interests</td><td rowspan="1" colspan="1" valign="top" align="left">Sameness1</td><td rowspan="1" colspan="1" valign="top" align="left">ADI-R Circumscribed interests ≥ 2</td><td rowspan="1" colspan="1" valign="top" align="left">1,474</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R difficulty with change</td><td rowspan="1" colspan="1" valign="top" align="left">Sameness2</td><td rowspan="1" colspan="1" valign="top" align="left">ADI-R Difficulty with change≥ 2</td><td rowspan="1" colspan="1" valign="top" align="left">1,313</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Higher sensitivity to noise</td><td rowspan="1" colspan="1" valign="top" align="left">Noise</td><td rowspan="1" colspan="1" valign="top" align="left">ADI-R Sensitivity to noise ≥ 2</td><td rowspan="1" colspan="1" valign="top" align="left">1,676</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ADOS CSS RRB higher than SA</td><td rowspan="1" colspan="1" valign="top" align="left">RBB &gt; SA</td><td rowspan="1" colspan="1" valign="top" align="left">ADOS RRB CSS ≥ Social Affect CSS</td><td rowspan="1" colspan="1" valign="top" align="left">1,805</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ADOS CSS RRB much higher than SA</td><td rowspan="1" colspan="1" valign="top" align="left">RBB ≫ SA</td><td rowspan="1" colspan="1" valign="top" align="left">ADOS RRB CSS ≥ Social Affect
CSS+2</td><td rowspan="1" colspan="1" valign="top" align="left">880</td></tr></tbody></table></div><div id="largeobj_idm140324080855312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4379124/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P15">The first subgroup of the full data was formed by considering only
probands with a categorical diagnosis of “autism” based on ADI-R
and ADOS (N = 2,088). The second subgroup (N=1,851) included
subjects with relatively high verbal IQ (vIQ&gt;61, see <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a>). Three more
groups targeted inclusion of the most severely impaired children, based on high
severity score on i) the total ADOS calibrated severity score (<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037156">35</a>), ii) the Social Affect (SA) calibrated severity
score iii) the Restricted and Repetitive Behavior (RRB) calibrated severity
score (<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037193">36</a>). High severity was defined as
a score of 8 or more for each subgroup.</p><p id="P16">The distribution of the insistence on sameness (IS) score (<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037171">24</a>, see <a href="#SD1" rid="SD1" class=" supplementary-material">supplemental methods</a>) was
quasi-continuous and relatively symmetric. Three subscores (difficulty with
change, sensitivity to noise and circumscribed interests) were used to define
three distinct subgroups, whereas the full IS score was used for quantitative
trait analysis. A second quantitative trait analysis was performed with age at
first words treated as a continuous variable.</p><p id="P17" class="p p-last">Lastly, two subgroups emphasized repetitive behaviors, a common feature
of subjects with ASD. The first subgroup included all individuals who had a
calibrated ADOS RRB score equal to or superior to SA calibrated ADOS score. A
second subgroup was restricted to individuals whose RRB calibrated ADOS score
exceeded SA calibrated ADOS score by ≥2.</p></div><div id="S9" class="sec"><h3 id="S9title">Genetic association analysis</h3><p id="P18" class="p p-first-last">For family-based association analysis we analyzed only trios from the
family (mother, father and proband diagnosed with ASD). Association analyses
were performed using an in-house program (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_530567009">37</a>) that implements the conditional logistic regression form of
family-based association and by FBAT (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037192">38</a>)
for analysis of quantitative traits. In all, 20 genome-wide association analyses
were performed. Among these 18 included families of all ancestry groups:
analysis of the whole sample; 10 subgroups (<a href="/pmc/articles/PMC4379124/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>); two quantitative trait locus (QTL) analysis of insistence
on sameness and age at first words; and parent-of-origin analyses for the entire
sample, those subjects with strict autism diagnosis and subjects with higher IQ;
and one was based on the European ancestry subset of the full data. A complete
listing of the 20 analyses can be found in <a href="/pmc/articles/PMC4379124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table
2</span></a>. Note that results from SNPs showing strong association, yet with
minor allele frequency MAF &lt; 2.5% in parents and
over-transmission of the common allele, were not displayed because they most
likely represent false positive results.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Association analyses models</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324005090912" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">N</th><th colspan="3" rowspan="1" valign="top" align="left">Genetic models</th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left"></th><th rowspan="1" colspan="1" valign="top" align="left">additive</th><th rowspan="1" colspan="1" valign="top" align="left">paternal</th><th rowspan="1" colspan="1" valign="top" align="left">maternal</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Whole sample</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">2575</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">European Ancestry</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">1633</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Subsets of the whole sample</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Restricted autism diagnosis</td><td rowspan="1" colspan="1" valign="top" align="left">2088</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Verbal IQ&gt;60</td><td rowspan="1" colspan="1" valign="top" align="left">1880</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left">√</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS ASD symptoms</td><td rowspan="1" colspan="1" valign="top" align="left">1200</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS RRB symptoms</td><td rowspan="1" colspan="1" valign="top" align="left">1589</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS social affect impairment</td><td rowspan="1" colspan="1" valign="top" align="left">1091</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R circumscribed interests</td><td rowspan="1" colspan="1" valign="top" align="left">1474</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R difficulty with change</td><td rowspan="1" colspan="1" valign="top" align="left">1313</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Higher sensitivity to noise</td><td rowspan="1" colspan="1" valign="top" align="left">1676</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">ADOS CSS RRB higher than SA</td><td rowspan="1" colspan="1" valign="top" align="left">1805</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">ADOS CSS RRB much higher than SA</td><td rowspan="1" colspan="1" valign="top" align="left">880</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">IS, QTL (whole sample)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">2575</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age at first words QTL (whole sample)</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">2575</td><td rowspan="1" colspan="1" valign="top" align="left">√</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140324005090912" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4379124/table/T2/?report=objectonly">Open in a separate window</a></div></div></div><div id="S10" class="sec"><h3 id="S10title">Heritability estimates</h3><p id="P19" class="p p-first-last">Heritability was estimated in each subgroup as described elsewhere
(<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037165">22</a>). In brief it was estimated from
comparing ASD subjects with 1,663 individuals from HealthABC (<a href="#R39" rid="R39" class=" bibr popnode">39</a>) using the GCTA software (<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037176">40</a>). The largest ASD sample analyzed comprised of
subjects of European Ancestry (N=1633). All other subgroups were subsets
of the European sample. To allow direct comparisons we used the same prevalence
for ASD (0.01) for each subgroup.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Allele scores</h3><p id="P20" class="p p-first-last">We built on association results to compute a polygenic allele score and
estimate the predictive value of this score for case control status, using the
method developed by Purcell et al (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037163">41</a>).
Allele scores were defined based on the GWA signals from the association
analysis in the full sample (N = 2,576 families). All SNPs associated
with a p-value&lt;0.001 were included in the score computation. Of note,
the p-value threshold was lower than in the previous analyses to limit the
number of predictors (<a href="#R42" rid="R42" class=" bibr popnode">42</a>). For each
autosomal SNP, the SNP-specific component of the score was calculated as the
dosage of the risk allele multiplied by the corresponding log of the odds ratio.
The allele-score approach requires controls as well as cases. Thus in each
family a pseudo-control was generated by comparing the genotype of the parents
to that of the proband and assigning the un-transmitted allele of each parent as
the alleles for the pseudo-control's genotype. After generating a
pseudo-control for each SNP and family, the individual-specific score was
calculated by summing over the SNP-specific component values for each individual
from the whole sample, using PLINK version 1.07 (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037185">41</a>). Last, a logistic regression estimated the predictive value of
the allele score for the case/control status.</p></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Results</h2><div id="S13" class="sec sec-first"><h3 id="S13title">Association analyses</h3><p id="P21" class="p p-first">The association results are split into four categories: genome-wide
significant (p &lt;5 × 10<sup>-8</sup>); suggestive (p between 5
× 10<sup>-7</sup> and &gt; 5 × 10<sup>-8</sup>); unreliable
(p &lt; 5 × 10<sup>-7</sup>, but with all the hallmarks of
genotyping error (<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037134">43</a>), including MAF
&lt; 0.05 and over-transmission of the major allele); and non-significant
(p &gt; 5 × 10<sup>-7</sup>). Only the genome-wide significant and
suggestive findings will be reported in this manuscript. P-values are
uncorrected for the 20 association tests performed.</p><p id="P22" class="p p-last">No genome-wide significant SNPs were identified in a GWAS analysis of
the complete dataset, or in any of the 18 phenotypic subsets of the data (<a href="/pmc/articles/PMC4379124/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). However, seven SNPs, meeting the
threshold for suggestive association (p &lt;5 × 10<sup>-7</sup>)
look promising (<a href="/pmc/articles/PMC4379124/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Table 3</span></a>). Among these, 6
SNPs are either located in, or have been shown to affect the expression of four
genes involved in DNA integrity maintenance: <em>CTU2</em>,
<em>CUEDC2</em>, <em>ZNF365</em>, <em>TOPBP1</em>
(<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037129">44</a>, <a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037178">45</a>, <a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037106">46</a>, <a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037131">47</a>, <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037116">48</a>).
Moreover, one of these SNPs (rs1409313) was shown to regulate cerebellum
expression of <em>C10orf26</em> (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037121">49</a>), a gene associated with schizophrenia (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037147">50</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Most associated SNPs<sup>a</sup> with case
status in primary analyses</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324005876016" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Chr</th><th rowspan="1" colspan="1" valign="top" align="left">Clinical subgroup</th><th rowspan="1" colspan="1" valign="top" align="left">P.LRT</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left">L95.OR</th><th rowspan="1" colspan="1" valign="top" align="left">U95.OR</th><th rowspan="1" colspan="1" valign="top" align="left">GENE</th><th rowspan="1" colspan="1" valign="top" align="left">eQTLs<sup>b</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11641365</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">ADOS CSS RRB much higher than SA</td><td rowspan="1" colspan="1" valign="top" align="left">3.19e-07</td><td rowspan="1" colspan="1" valign="top" align="left">2.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.54</td><td rowspan="1" colspan="1" valign="top" align="left">2.76</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CTU2</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2393895</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS social affect impairment</td><td rowspan="1" colspan="1" valign="top" align="left">2.23e-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.91</td><td rowspan="1" colspan="1" valign="top" align="left">1.48</td><td rowspan="1" colspan="1" valign="top" align="left">2.47</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF365</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs868914</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS social affect impairment</td><td rowspan="1" colspan="1" valign="top" align="left">2.48e-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.68</td><td rowspan="1" colspan="1" valign="top" align="left">1.37</td><td rowspan="1" colspan="1" valign="top" align="left">2.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF365</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1867503</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">ASD paternal model</td><td rowspan="1" colspan="1" valign="top" align="left">4.18e-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.55</td><td rowspan="1" colspan="1" valign="top" align="left">1.30</td><td rowspan="1" colspan="1" valign="top" align="left">1.84</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Regulates <em>TOPBP1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1867504</td><td rowspan="1" colspan="1" valign="top" align="left">3</td><td rowspan="1" colspan="1" valign="top" align="left">ASD paternal model</td><td rowspan="1" colspan="1" valign="top" align="left">4.18e-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.55</td><td rowspan="1" colspan="1" valign="top" align="left">1.30</td><td rowspan="1" colspan="1" valign="top" align="left">1.84</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">Regulates <em>TOPBP1</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1409313</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">ASD paternal model</td><td rowspan="1" colspan="1" valign="top" align="left">4.01e-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.75</td><td rowspan="1" colspan="1" valign="top" align="left">1.40</td><td rowspan="1" colspan="1" valign="top" align="left">2.18</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CUEDC2</em></td><td rowspan="1" colspan="1" valign="top" align="left">Regulates <em>C10orf26</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4773054</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">ASD paternal model</td><td rowspan="1" colspan="1" valign="top" align="left">5.25e-08</td><td rowspan="1" colspan="1" valign="top" align="left">2.66</td><td rowspan="1" colspan="1" valign="top" align="left">1.83</td><td rowspan="1" colspan="1" valign="top" align="left">3.86</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4773054</td><td rowspan="1" colspan="1" valign="top" align="left">13</td><td rowspan="1" colspan="1" valign="top" align="left">Autism paternal model</td><td rowspan="1" colspan="1" valign="top" align="left">7.28e-08</td><td rowspan="1" colspan="1" valign="top" align="left">2.9</td><td rowspan="1" colspan="1" valign="top" align="left">1.91</td><td rowspan="1" colspan="1" valign="top" align="left">4.39</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140324005876016" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4379124/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>p&lt;5×10<sup>-7</sup> and MAF&gt;0.05 and positive effect
of the rare allele</div><div id="TFN2"><sup>b</sup>In cerebellum or parietal cortex according to scan database (<a href="http://www.scandb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.scandb.org</a>, (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037113">49</a>))</div></div></div></div><div id="S14" class="sec"><h3 id="S14title">Analysis of the impact of sub-phenotyping on association results</h3><p id="P23" class="p p-first">Over all SNPs an obvious and anticipated impact of phenotypic
subgrouping is to reduce the size of association statistic (one sided z
decreases and p-values increase). In general, if subsamples are merely a random
subsample of the larger sample, test statistics diminish in a very predictable
fashion, specifically as the square root of the sample size for each subsample.
In this study, association statistics (one-sided-z) on average diminish as a
function of the square root of the sample size (<a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1a</span></a>). The pattern itself is unremarkable because statistical
theory shows power has this functional relationship, but what is remarkable is
how closely the points adhere to the linear relationship.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324080864448" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms631893f1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms631893f1.jpg" src="/pmc/articles/PMC4379124/bin/nihms631893f1.jpg"></div></a><div id="largeobj_idm140324080864448" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a target="object" href="/pmc/articles/PMC4379124/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4379124/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Fig. 1</a></div><!--caption a7--><div class="caption"><p id="__p6">A) Relationship between the number of informative families and the average
difference in z statistics for selected SNPs. For SNPs with p-value &lt;
0.01 for association in the full sample, the difference in average z-statistics
is calculated as the absolute value of the z-statistic for the full sample minus
the absolute value of the z-statistic achieved for the subsample and the average
is taken over all qualifying SNPs. The straight line shows the expected
relationship for the difference if the samples were drawn at random from the
full sample. B) Box plot of difference of one sided z in each phenotypic
subset of the whole sample from one sided z score in all ASD. The difference was
standardized. The vertical line in the plot is drawn at the one-sided Z -score
of 4.58, corresponding to p=0.05/21,351.</p></div></div></div><p id="P24" class="p p-last">Another way to view the influence of phenotypic subgrouping on
association between SNP and case status is to examine how the p-value changes
from the full sample to that for the subgroup. To examine this effect we
conducted secondary analyses and selected a subset of 21,351 SNPs based on the
following association results from the whole sample: p-value &lt; 0.01 and
OR &gt; 1 if an allele with MAF &lt; 0.025. To compare all subgroups,
p-values in each subgroup were transformed into absolute values of the
associated Z-scores. The difference between the one sided z score in the
subgroup and the one sided z score in the full sample, corresponding to the
shift toward association in the subgroup, was standardized (<a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1B</span></a>). We did not observe a significant shift
toward association (i.e. a standardized difference &gt;4.58, corresponding
to a p-value = 0.05/21351) for any SNP in any subgroup. However, the
results of these secondary analyses were suggestive for 11 SNPs. Indeed, these
SNPs were associated with case status with a p value &lt; 2×
10<sup>-6</sup> (i.e. significant when correcting for 21351 markers) in at
least one of the subgroups analyses and standardized differences among the
largest (&gt;2) (<a href="/pmc/articles/PMC4379124/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span>Table 4</span></a>). Among
those, 6 SNPs were located in 4 genes (<em>FBXW7, ZNF365, CTU2,
PTCHD1</em>), and one (rs3856064 on chromosome 1) was reported
(<a href="http://www.scandb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.scandb.org</a> (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037187">49</a>)) to
regulate the expression of <em>STX6</em> gene in parietal cortex. FBXW7
is involved in genome stability, and more specifically checkpoint signaling,
through the regulation of cyclin E (<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037115">51</a>).
Although the exact function of PTCHD1 is not known, it is thought to be involved
in Hedgehog signaling which regulates the formation of the neural tube and brain
(<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037189">52</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T4"><h3>Table 4</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>SNPs associated with a z-score&gt;4.58 in one of the phenotypic
subgroups in secondary analyses<sup>*</sup></strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324010090512" class="xtable"><table frame="box" rules="all" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Chr<sup>a</sup></th><th rowspan="1" colspan="1" valign="top" align="left">Subgroup</th><th rowspan="1" colspan="1" valign="top" align="left">P<sup>b</sup></th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left">L95%<sup>c</sup></th><th rowspan="1" colspan="1" valign="top" align="left">U95%<sup>d</sup></th><th rowspan="1" colspan="1" valign="top" align="left">GENE</th><th rowspan="1" colspan="1" valign="top" align="left">eQTLs<sup>k</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs3856064</td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R SN<sup>e</sup></td><td rowspan="1" colspan="1" valign="top" align="left">2.25E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.37</td><td rowspan="1" colspan="1" valign="top" align="left">1.2</td><td rowspan="1" colspan="1" valign="top" align="left">1.56</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"><em>STX6</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10021011</td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS SA<sup>f</sup> impairment</td><td rowspan="1" colspan="1" valign="top" align="left">1.85E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.62</td><td rowspan="1" colspan="1" valign="top" align="left">1.32</td><td rowspan="1" colspan="1" valign="top" align="left">1.99</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FBXW7</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2393895</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS SA<sup>f</sup> impairment</td><td rowspan="1" colspan="1" valign="top" align="left">2.23E-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.91</td><td rowspan="1" colspan="1" valign="top" align="left">1.48</td><td rowspan="1" colspan="1" valign="top" align="left">2.47</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF365</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs868913</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS SA<sup>f</sup> impairment</td><td rowspan="1" colspan="1" valign="top" align="left">5.11E-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.65</td><td rowspan="1" colspan="1" valign="top" align="left">1.35</td><td rowspan="1" colspan="1" valign="top" align="left">2.02</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF365</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs868914</td><td rowspan="1" colspan="1" valign="top" align="left">10</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADOS SA<sup>f</sup> impairment</td><td rowspan="1" colspan="1" valign="top" align="left">2.48E-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.68</td><td rowspan="1" colspan="1" valign="top" align="left">1.37</td><td rowspan="1" colspan="1" valign="top" align="left">2.07</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF365</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SNP12-76144695</td><td rowspan="1" colspan="1" valign="top" align="left">12</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R DC<sup>g</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.41E-06</td><td rowspan="1" colspan="1" valign="top" align="left">9.33</td><td rowspan="1" colspan="1" valign="top" align="left">2.84</td><td rowspan="1" colspan="1" valign="top" align="left">30.70</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11641365</td><td rowspan="1" colspan="1" valign="top" align="left">16</td><td rowspan="1" colspan="1" valign="top" align="left">Extreme RRB<sup>h</sup> predominance</td><td rowspan="1" colspan="1" valign="top" align="left">3.19E-07</td><td rowspan="1" colspan="1" valign="top" align="left">2.06</td><td rowspan="1" colspan="1" valign="top" align="left">1.54</td><td rowspan="1" colspan="1" valign="top" align="left">2.76</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CTU2</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs5751278</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R CI<sup>i</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.75E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.57</td><td rowspan="1" colspan="1" valign="top" align="left">1.3</td><td rowspan="1" colspan="1" valign="top" align="left">1.90</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOBEC3D</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs5758742</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R CI<sup>i</sup></td><td rowspan="1" colspan="1" valign="top" align="left">5.70E-07</td><td rowspan="1" colspan="1" valign="top" align="left">1.56</td><td rowspan="1" colspan="1" valign="top" align="left">1.3</td><td rowspan="1" colspan="1" valign="top" align="left">1.86</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOBEC3D</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs5751281</td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">Higher ADI-R CI<sup>i</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.50E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.56</td><td rowspan="1" colspan="1" valign="top" align="left">1.31</td><td rowspan="1" colspan="1" valign="top" align="left">1.93</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"><em>APOBEC3D</em></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs5971115</td><td rowspan="1" colspan="1" valign="top" align="left">X</td><td rowspan="1" colspan="1" valign="top" align="left">Restricted autism dx<sup>j</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1.67E-06</td><td rowspan="1" colspan="1" valign="top" align="left">1.6</td><td rowspan="1" colspan="1" valign="top" align="left">1.31</td><td rowspan="1" colspan="1" valign="top" align="left">1.94</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PTCHD1</em></td><td rowspan="1" colspan="1" valign="top" align="left"></td></tr></tbody></table></div><div id="largeobj_idm140324010090512" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4379124/table/T4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN3"><sup>*</sup>Choice of SNPs is based, in part, on the difference between the one sided z
score in the subgroup and the one sided z score in the full sample,
corresponding to the shift toward association in the subgroup.</div><div id="TFN4"><sup>a</sup>Chromosome</div><div id="TFN5"><sup>b</sup>p-value for association in the subgroup</div><div id="TFN6"><sup>c</sup>Lower bound 95%CI OR</div><div id="TFN7"><sup>d</sup>Upperbound 95%CI OR</div><div id="TFN8"><sup>e</sup>Sensitivity to noise.</div><div id="TFN9"><sup>f</sup>Socio-affective standardized score</div><div id="TFN10"><sup>g</sup>Difficulty with change</div><div id="TFN11"><sup>h</sup>Repetitive and ritualized behavior</div><div id="TFN12"><sup>i</sup>Circumscribed interests</div><div id="TFN13"><sup>j</sup>Diagnosis</div><div id="TFN14"><sup>k</sup>In cerebellum or parietal cortex, according to scan database (<a href="http://www.scandb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.scandb.org</a>, (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037120">49</a>)</div></div></div></div><div id="S15" class="sec sec-last"><h3 id="S15title">Impact of sub-phenotyping on the genetic architecture of ASD</h3><p id="P25" class="p p-first">After examining the impact of sub-phenotyping on association statistics
for each SNP, we evaluated its impact on genetic features of the sample as
measured through a large number of SNPs, assessed jointly. In particular we
investigated how heritability and allele scores changed across subgroups.</p><p id="P26">Heritability of this sample was previously evaluated by Klei et al.
(<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037166">22</a>). In that study heritability was
estimated from samples of European ancestry genotyped on the Illumina 1Mv1 or
the Illumina 1Mv3 array. To be consistent with that study, we used the same
approach and sample, after removing a small set of families according to the
updated SSC participants list (these families being excluded mostly because of
withdrawal of consent or additional cases in the simplex family, thereby
altering its simplex status). For the ASD sample the estimated heritability was
0.401 (<a href="/pmc/articles/PMC4379124/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">Table S2</a>) and subgroups of this
sample were not statistically different from the whole sample or other subsets.
Estimated heritability, however, showed a trend toward a higher heritability
estimate in the three subgroups defined by an excess of repetitive
behaviors.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4379124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324080861344" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms631893f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms631893f2.jpg" src="/pmc/articles/PMC4379124/bin/nihms631893f2.jpg"></div></a><div id="largeobj_idm140324080861344" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4379124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC4379124/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4379124/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Fig. 2</a></div><!--caption a7--><div class="caption"><p id="__p7">Heritability estimates in each phenotypic subset of the whole sample. All
ASD=Whole sample, Autism=Restricted autism diagnosis, High
vIQ=Verbal IQ&gt; 60, Severe ASD=Higher ADOS ASD symptoms,
Severe RBB=Higher ADOS RRB symptoms, Severe SA=Higher ADOS
social affect impairment, Sameness1=Higher ADI-R circumscribed
interests, Sameness2=Higher ADI-R difficulty with change,
Noise=Higher sensitivity to noise, RBB &gt; SA=ADOS CSS RRB
higher than SA, RBB ≫ SA=ADOS CSS RRB much higher than SA.</p></div></div></div><p id="P27" class="p p-last">A total of 2438 SNPs with a MAF &gt; 0.025 showed an association
p-value &lt; 0.001 in the full ASD data. The allele score based on this set
of SNPs explained around 0.454 the variance in case control status
(pseudoR<sup>2</sup>) based on the whole sample and performed similarly for
each subgroup (<a href="/pmc/articles/PMC4379124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>), indicating that the
predictive value of the score does not differ materially amongst these samples.
One can take a different approach in which one generates an allele score for
each of the samples in <a href="/pmc/articles/PMC4379124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>, yet such
an approach yields results indistinguishable from those in <a href="/pmc/articles/PMC4379124/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a> (data not shown).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4379124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140324080857888" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms631893f3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms631893f3.jpg" src="/pmc/articles/PMC4379124/bin/nihms631893f3.jpg"></div></a><div id="largeobj_idm140324080857888" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4379124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a target="object" href="/pmc/articles/PMC4379124/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4379124/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Fig. 3</a></div><!--caption a7--><div class="caption"><p id="__p8">Portion of the variance of each phenotypic subset from the whole sample explained
by the allele score computed from the whole sample analysis odds ratios. All
ASD=Whole sample, Autism=Restricted autism diagnosis, High
vIQ=Verbal IQ&gt; 60, Severe ASD=Higher ADOS ASD symptoms,
Severe RBB=Higher ADOS RRB symptoms, Severe SA=Higher ADOS
social affect impairment, Sameness1=Higher ADI-R circumscribed
interests, Sameness2=Higher ADI-R difficulty with change,
Noise=Higher sensitivity to noise, RBB &gt; SA=ADOS CSS RRB
higher than SA, RBB ≫ SA=ADOS CSS RRB much higher than SA.</p></div></div></div></div></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P28" class="p p-first">A longstanding conjecture regarding the genetics of ASD is that
ASD's great phenotypic heterogeneity is a key hindrance to the discovery of
genetic factors conferring liability. Consequently, disentangling this heterogeneity
has been deemed an essential step toward identification of genetic variants
affecting risk (<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037153">2</a>, <a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037111">13</a>, <a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037173">4</a>). The results
of this study, however, suggest that this approach cannot ensure success. To the
contrary the results suggest that observable clinical variation phenotypic
variability does not map closely onto common genetic variation.</p><p id="P29">The paucity of SNPs that emerge with certain sub-phenotypes suggests that
subgrouping based on phenotype does not substantially increase power to detect
associated common variants (<a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>). One could
argue that this is because sub-phenotyping loses power by reducing the size of the
sample and that recruiting larger samples with homogeneous phenotype would lead to
the identification of some associated SNPs. However, test statistics closely follow
the expected linear relationship with square root of sample size. (<a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1A</span></a>), suggesting that sub-phenotypes are not highly
informative for genetic discovery. (Indeed, if sub-phenotypes were highly
informative for genetic discovery we would expect some noteworthy deviation from the
linear fit observed in <a href="/pmc/articles/PMC4379124/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). Alleles
scores, based on a small subset of SNPs, predict case status in the full sample and
in subsamples based on various phenotypes (e.g., meeting ASD diagnostic criteria and
manifesting repetitive behaviors) equally well. The similarity of the proportion of
variance explained among subgroups suggests that subgrouping does not contribute
much to our understanding of the genetics of risk for ASD in its various clinical
forms. Taken together with the similarity of heritability estimates across
subsamples this strongly suggests that subgrouping based on clinical phenotypes does
little to modify the genetic architecture of autism risk in each subgroup and does
not substantially increase genetic homogeneity. Consistent with these results,
sub-phenotyping does not significantly shift one sided z scores toward higher values
for any SNP. Remarkably, heritability estimates in the higher IQ group and the full
sample show very little difference, even ignoring sampling error suggesting that the
architecture of autism risk is similar in high IQ and low IQ populations.
Intriguingly, this result appears at odds with the observation of a significant
enrichment for de novo loss of function mutations in individuals with low IQ (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037149">53</a>). We can not exclude that the relatively
small size of the sample of people with low IQ, due to the large fraction of
individuals with high IQ in the SSC, could obscur the difference in heritability.
However, the results are also consistent with common variants making similar
contribution to the ASD phenotype for both groups, while de novo mutations, in
general, have a modest impact on risk for ASD and a greater impact on IQ. This
interpretation is consistent with the conjecture of Samocha et al. (<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037161">53</a>). In this regard, when we excluded individuals with a
clinically relevant CNV, which are likely to have a substantial impact on IQ (<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037155">54</a>), it did not substantially change
association results (unpublished data).</p><p id="P30">These results, however, cannot be taken to mean that clinical phenotypes
have no impact on power for discovering genetic associations and no underlying
genetic basis. At some level they must, but the mapping of genotype and phenotype
appears to be almost as complex as that between genotype and ASD itself. Still,
examining the results of each subgroup separately raises the possibility that
probands with high repetitive behaviors are a slightly more genetically homogeneous
group and one worth further exploration. It is noteworthy that several lines of
evidence support the existence of a different genetic etiology for RRB and SA
symptoms (<a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037194">55</a>, <a href="#R56" rid="R56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037125">56</a>). Moreover, while some studies have suggested that the genetic
architecture of autism risk slightly differs between sex (<a href="#R57" rid="R57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037136">57</a>, <a href="#R58" rid="R58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037170">58</a>, <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037167">59</a>), RRB have been found to be the main
symptoms to be differently distributed between sex (<a href="#R60" rid="R60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037114">60</a>, <a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037181">27</a>, <a href="#R59" rid="R59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037139">59</a>). The SNP which was found to be significantly shifted
toward association in the high RRB subgroup, rs11641365, is located in
<em>CTU2</em>.</p><p id="P31">Although no locus reached statistical significance after correcting for
genome wide multiple testing in primary analyses, some of the genes that emerge from
primary and secondary association analyses (<em>CTU2</em>,
<em>CUEDC2</em>, <em>ZNF365</em>, <em>TOPBP1,
FBXW7</em>; <a href="/pmc/articles/PMC4379124/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>Tables 3</span></a> and <a href="/pmc/articles/PMC4379124/table/T4/" target="table" class="fig-table-link figpopup" rid-figpopup="T4" rid-ob="ob-T4" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and4)</span></span><span>4</span></a>) raise the possibility of a role for maintenance of
genomic integrity in the pathophysiology of autism, a role highlighted in recent
sequencing analyses of the 15q13.3 region (<a href="#R61" rid="R61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037159">61</a>). In addition <em>FBXW7</em>, which targets mTOR for
ubiquitination and degradation and cooperates with <em>PTEN</em> (<a href="#R62" rid="R62" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037164">62</a>), plays a key role in neural stem cell
differentiation (<a href="#R63" rid="R63" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037137">63</a>), and is required for
cerebellar development (<a href="#R64" rid="R64" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037151">64</a>). ZNF365 isoform D
regulates the expression of <em>DISC-1</em>, a gene implicated in
schizophrenia risk (<a href="#R65" rid="R65" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037152">65</a>, <a href="#R66" rid="R66" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037104">66</a>). <em>STX6</em> is a SNARE component (<a href="#R67" rid="R67" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037133">67</a>) and thus may play a role in synaptic
transmission. Finally <em>PTCHD1</em> has been previously implicated in
autism and intellectual disability (<a href="#R68" rid="R68" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037117">68</a>, <a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037127">52</a>, <a href="#R69" rid="R69" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037169">69</a>,
<a href="#R70" rid="R70" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037109">70</a>, <a href="#R71" rid="R71" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037144">71</a>). Interestingly the SNP rs5971115 was found to be substantially
shifted toward association in the group with strict autistic disorder, which is
characterized by a low IQ. However, it is important to underscore that all these
association results are exploratory, none are statistically significant, and only
additional independent data can determine what role, if any, they play in
liability.</p><p id="P32" class="p p-last">While we acknowledge that these results are based on a limited selection of
phenotypes - and there still could be phenotypes, including biomarkers, which could
be informative - they are in line with the body of work that has led to the single
ASD category in DSM-5 classification (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037124">29</a> .
They support ASD categorical diagnosis as a valid entity for the search of common
variants increasing risk, which is consistent with high heritability estimates for
ASD found in most studies (<a href="#R73" rid="R73" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037182">73</a>, <a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037196">22</a>, <a href="#R74" rid="R74" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037177">74</a>).
There are other limitations to our study, including its sample size, which could
always be larger; perhaps the strict simplex nature of the families, which could
limit heritable variation (<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037172">22</a>); somewhat
reduced variability of phenotypes like IQ and ASD severity; and use of ASD/ID
related behavioral phenotypes as opposed to medical and psychiatric comorbidity. On
the other hand, taking the results presented here together with the emerging picture
of autism risk architecture regarding for common variants, they suggest that
phenotype is the result of the complex combination of a large number of modest
allelic effects on each trait. If, as some authors have proposed, we can assume that
there is no common variant of substantial effect on the risk of ASD (odds
ratio&gt; 1.2), but a great number of common variations with a more modest
impact (<a href="#R75" rid="R75" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_442037105">75</a>), the identification of common
risk variants is hindered mainly by sample size and only marginally by clinical
heterogeneity even in samples carefully phenotyped to confirm an ASD diagnosis. Thus
the results of this study advocate for recruitment of larger cohorts of ASD subjects
to identify common variation affecting risk for ASD. Indeed, while GWAS studies in
ASD are large relative to earlier candidate gene studies, their sample sizes are
still small relative to the majority of the studies in other complex diseases.</p></div><div id="S17" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S17title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>supplement</h4><div class="sup-box half_rhythm" id="d36e955"><a href="/pmc/articles/PMC4379124/bin/NIHMS631893-supplement.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(97K, pdf)</sup></div></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Acknowledgments</h2><div class="sec"><p id="__p1">We are grateful to all of the families at the participating Simons Simplex Collection
(SSC) sites, as well as the principal investigators (A. Beaudet, R. Bernier, J.
Constantino, E. Cook, E. Fombonne, D. Geschwind, R. Goin-Kochel, E. Hanson, D.
Grice, A. Klin, D. Ledbetter, C. Lord, C. Martin, D. Martin, R. Maxim, J. Miles, O.
Ousley, K. Pelphrey, B. Peterson, J. Piggot, C. Saulnier, M. State, W. Stone, J.
Sutcliffe, C. Walsh, Z. Warren, E. Wijsman). We appreciate obtaining access to
phenotypic data on the Simons Foundation Autism Research Initiative (SFARI) Base.
Approved researchers can obtain the SSC population dataset described in this study
(<a href="http://sfari.org/resources/sfari-base" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://sfari.org/resources/sfari-base</a>), by applying at <a href="https://base.sfari.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://base.sfari.org</a>. This work was supported by a grant from the
Simons Foundation (SFARI 124827 to the investigators of the SSC Genetic Consortium);
NIH R01HD065272 (CL), NIH R01MH089390 (CL), NIH R37 MH057881 (BD), NIMH
5R25MH071584-07 Malison (DMDL), NIMH 5 T32 MH19961-14 Malison (DMDL) and the
Fondamental Foundation (PC).</p></div></div><div id="idm140324006222848" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140324006222848title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p2" class="p p-first-last"><a href="#SD1" rid="SD1" class=" supplementary-material">Supplemental
information</a>: Yes, with 2 tables and 1 figure</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last"><strong>Financial Disclosures</strong>: Dr. Cathy Lord receives royalties for the
scales of diagnostic instruments for which she's a coauthor; she donates
all royalties to autism charities. All other authors report no biomedical
financial interests or potential conflict of interest.</p><p></p><p class="fn sec" id="FN4"></p><p id="__p4" class="p p-first-last"><strong>Publisher's Disclaimer: </strong>This is a PDF file of an unedited manuscript
that has been accepted for publication. As a service to our customers we are
providing this early version of the manuscript. The manuscript will undergo
copyediting, typesetting, and review of the resulting proof before it is
published in its final citable form. Please note that during the production
process errors may be discovered which could affect the content, and all legal
disclaimers that apply to the journal pertain.</p><p></p></div></div><div id="idm140324013517808" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140324013517808title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Jeste SS, Geschwind DH. Disentangling the heterogeneity of autism spectrum disorder
through genetic findings. <span><span class="ref-journal">Nat Rev Neurol. </span>2014;<span class="ref-vol">10</span>:74–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4125617/">PMC free article</a>]</span> [<a href="/pubmed/24468882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Neurol&amp;title=Disentangling+the+heterogeneity+of+autism+spectrum+disorder
through+genetic+findings&amp;author=SS+Jeste&amp;author=DH+Geschwind&amp;volume=10&amp;publication_year=2014&amp;pages=74-81&amp;pmid=24468882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Georgiades S, Szatmari P, Boyle M, Hanna S, Duku E, Zwaigenbaum L, et al.  Investigating phenotypic heterogeneity in children with autism
spectrum disorder: a factor mixture modeling approach. <span><span class="ref-journal">J Child Psychol Psychiatry. </span>2013;<span class="ref-vol">54</span>:206–215.</span> [<a href="/pubmed/22862778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Child+Psychol+Psychiatry&amp;title=Investigating+phenotypic+heterogeneity+in+children+with+autism
spectrum+disorder:+a+factor+mixture+modeling+approach&amp;author=S+Georgiades&amp;author=P+Szatmari&amp;author=M+Boyle&amp;author=S+Hanna&amp;author=E+Duku&amp;volume=54&amp;publication_year=2013&amp;pages=206-215&amp;pmid=22862778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Hus V, Pickles A, Cook EH, Jr, Risi S, Lord C. Using the autism diagnostic interview--revised to increase
phenotypic homogeneity in genetic studies of autism. <span><span class="ref-journal">Biol Psychiatry. </span>2007;<span class="ref-vol">61</span>:438–448.</span> [<a href="/pubmed/17276746" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Using+the+autism+diagnostic+interview--revised+to+increase
phenotypic+homogeneity+in+genetic+studies+of+autism&amp;author=V+Hus&amp;author=A+Pickles&amp;author=EH+Cook&amp;author=S+Risi&amp;author=C+Lord&amp;volume=61&amp;publication_year=2007&amp;pages=438-448&amp;pmid=17276746&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Veatch OJ, Veenstra-Vanderweele J, Potter M, Pericak-Vance MA, Haines JL. Genetically meaningful phenotypic subgroups in autism spectrum
disorders. <span><span class="ref-journal">Genes Brain Behav. </span>2013;<span class="ref-vol">13</span>:276–285.</span> [<a href="/pubmed/24373520" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Brain+Behav&amp;title=Genetically+meaningful+phenotypic+subgroups+in+autism+spectrum
disorders&amp;author=OJ+Veatch&amp;author=J+Veenstra-Vanderweele&amp;author=M+Potter&amp;author=MA+Pericak-Vance&amp;author=JL+Haines&amp;volume=13&amp;publication_year=2013&amp;pages=276-285&amp;pmid=24373520&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Amir RE, Van den Veyver IB, Wan M, Tran CQ, Francke U, Zoghbi HY. Rett syndrome is caused by mutations in X-linked MECP2, encoding
methyl-CpG-binding protein 2. <span><span class="ref-journal">Nat Genet. </span>1999;<span class="ref-vol">23</span>:185–188.</span> [<a href="/pubmed/10508514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Rett+syndrome+is+caused+by+mutations+in+X-linked+MECP2,+encoding
methyl-CpG-binding+protein+2&amp;author=RE+Amir&amp;author=IB+Van+den+Veyver&amp;author=M+Wan&amp;author=CQ+Tran&amp;author=U+Francke&amp;volume=23&amp;publication_year=1999&amp;pages=185-188&amp;pmid=10508514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Ariani F, Hayek G, Rondinella D, Artuso R, Mencarelli MA, Spanhol-Rosseto A, et al.  FOXG1 is responsible for the congenital variant of Rett
syndrome. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:89–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443837/">PMC free article</a>]</span> [<a href="/pubmed/18571142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=FOXG1+is+responsible+for+the+congenital+variant+of+Rett
syndrome&amp;author=F+Ariani&amp;author=G+Hayek&amp;author=D+Rondinella&amp;author=R+Artuso&amp;author=MA+Mencarelli&amp;volume=83&amp;publication_year=2008&amp;pages=89-93&amp;pmid=18571142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Schneider M, Debbane M, Bassett AS, Chow EW, Fung WL, van den Bree MB, et al.  Psychiatric Disorders From Childhood to Adulthood in 22q11.2
Deletion Syndrome: Results From the International Consortium on Brain and
Behavior in 22q11.2 Deletion Syndrome. <span><span class="ref-journal">Am J Psychiatry </span>2014</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4285461/">PMC free article</a>]</span> [<a href="/pubmed/24577245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Psychiatry&amp;title=Psychiatric+Disorders+From+Childhood+to+Adulthood+in+22q11.2
Deletion+Syndrome:+Results+From+the+International+Consortium+on+Brain+and
Behavior+in+22q11.2+Deletion+Syndrome&amp;author=M+Schneider&amp;author=M+Debbane&amp;author=AS+Bassett&amp;author=EW+Chow&amp;author=WL+Fung&amp;publication_year=2014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Buxbaum JD, Silverman J, Keddache M, Smith CJ, Hollander E, Ramoz N, et al.  Linkage analysis for autism in a subset families with
obsessive-compulsive behaviors: evidence for an autism susceptibility gene
on chromosome 1 and further support for susceptibility genes on chromosome 6
and 19. <span><span class="ref-journal">Mol Psychiatry. </span>2004;<span class="ref-vol">9</span>:144–150.</span> [<a href="/pubmed/14699429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=Linkage+analysis+for+autism+in+a+subset+families+with
obsessive-compulsive+behaviors:+evidence+for+an+autism+susceptibility+gene
on+chromosome+1+and+further+support+for+susceptibility+genes+on+chromosome+6
and+19&amp;author=JD+Buxbaum&amp;author=J+Silverman&amp;author=M+Keddache&amp;author=CJ+Smith&amp;author=E+Hollander&amp;volume=9&amp;publication_year=2004&amp;pages=144-150&amp;pmid=14699429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Buxbaum JD, Silverman JM, Smith CJ, Kilifarski M, Reichert J, Hollander E, et al.  Evidence for a susceptibility gene for autism on chromosome 2 and
for genetic heterogeneity. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">68</span>:1514–1520.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1226139/">PMC free article</a>]</span> [<a href="/pubmed/11353400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Evidence+for+a+susceptibility+gene+for+autism+on+chromosome+2+and
for+genetic+heterogeneity&amp;author=JD+Buxbaum&amp;author=JM+Silverman&amp;author=CJ+Smith&amp;author=M+Kilifarski&amp;author=J+Reichert&amp;volume=68&amp;publication_year=2001&amp;pages=1514-1520&amp;pmid=11353400&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Liu XQ, Paterson AD, Szatmari P. Genome-wide linkage analyses of quantitative and categorical
autism subphenotypes. <span><span class="ref-journal">Biol Psychiatry. </span>2008;<span class="ref-vol">64</span>:561–570.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670970/">PMC free article</a>]</span> [<a href="/pubmed/18632090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Psychiatry&amp;title=Genome-wide+linkage+analyses+of+quantitative+and+categorical
autism+subphenotypes&amp;author=XQ+Liu&amp;author=AD+Paterson&amp;author=P+Szatmari&amp;volume=64&amp;publication_year=2008&amp;pages=561-570&amp;pmid=18632090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Shao Y, Wolpert CM, Raiford KL, Menold MM, Donnelly SL, Ravan SA, et al.  Genomic screen and follow-up analysis for autistic
disorder. <span><span class="ref-journal">Am J Med Genet. </span>2002;<span class="ref-vol">114</span>:99–105.</span> [<a href="/pubmed/11840513" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet&amp;title=Genomic+screen+and+follow-up+analysis+for+autistic
disorder&amp;author=Y+Shao&amp;author=CM+Wolpert&amp;author=KL+Raiford&amp;author=MM+Menold&amp;author=SL+Donnelly&amp;volume=114&amp;publication_year=2002&amp;pages=99-105&amp;pmid=11840513&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Cannon DS, Miller JS, Robison RJ, Villalobos ME, Wahmhoff NK, Allen-Brady K, et al.  Genome-wide linkage analyses of two repetitive behavior
phenotypes in Utah pedigrees with autism spectrum disorders. <span><span class="ref-journal">Mol Autism. </span>2010;<span class="ref-vol">1</span>:3.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2907569/">PMC free article</a>]</span> [<a href="/pubmed/20678246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Autism&amp;title=Genome-wide+linkage+analyses+of+two+repetitive+behavior
phenotypes+in+Utah+pedigrees+with+autism+spectrum+disorders&amp;author=DS+Cannon&amp;author=JS+Miller&amp;author=RJ+Robison&amp;author=ME+Villalobos&amp;author=NK+Wahmhoff&amp;volume=1&amp;publication_year=2010&amp;pages=3&amp;pmid=20678246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Liu XQ, Georgiades S, Duku E, Thompson A, Devlin B, Cook EH, et al.  Identification of genetic loci underlying the phenotypic
constructs of autism spectrum disorders. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2011;<span class="ref-vol">50</span>:687–696 e613.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3593812/">PMC free article</a>]</span> [<a href="/pubmed/21703496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Identification+of+genetic+loci+underlying+the+phenotypic
constructs+of+autism+spectrum+disorders&amp;author=XQ+Liu&amp;author=S+Georgiades&amp;author=E+Duku&amp;author=A+Thompson&amp;author=B+Devlin&amp;volume=50&amp;publication_year=2011&amp;pages=687-696+e613&amp;pmid=21703496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Shao Y, Cuccaro ML, Hauser ER, Raiford KL, Menold MM, Wolpert CM, et al.  Fine mapping of autistic disorder to chromosome 15q11-q13 by use
of phenotypic subtypes. <span><span class="ref-journal">Am J Hum Genet. </span>2003;<span class="ref-vol">72</span>:539–548.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1180230/">PMC free article</a>]</span> [<a href="/pubmed/12567325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fine+mapping+of+autistic+disorder+to+chromosome+15q11-q13+by+use
of+phenotypic+subtypes&amp;author=Y+Shao&amp;author=ML+Cuccaro&amp;author=ER+Hauser&amp;author=KL+Raiford&amp;author=MM+Menold&amp;volume=72&amp;publication_year=2003&amp;pages=539-548&amp;pmid=12567325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Szatmari P, Paterson AD, Zwaigenbaum L, Roberts W, Brian J, Liu XQ, et al.  Mapping autism risk loci using genetic linkage and chromosomal
rearrangements. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:319–328.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4867008/">PMC free article</a>]</span> [<a href="/pubmed/17322880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mapping+autism+risk+loci+using+genetic+linkage+and+chromosomal
rearrangements&amp;author=P+Szatmari&amp;author=AD+Paterson&amp;author=L+Zwaigenbaum&amp;author=W+Roberts&amp;author=J+Brian&amp;volume=39&amp;publication_year=2007&amp;pages=319-328&amp;pmid=17322880&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Weiss LA, Arking DE, Daly MJ, Chakravarti A. A genome-wide linkage and association scan reveals novel loci for
autism. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">461</span>:802–808.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772655/">PMC free article</a>]</span> [<a href="/pubmed/19812673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+linkage+and+association+scan+reveals+novel+loci+for
autism&amp;author=LA+Weiss&amp;author=DE+Arking&amp;author=MJ+Daly&amp;author=A+Chakravarti&amp;volume=461&amp;publication_year=2009&amp;pages=802-808&amp;pmid=19812673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Schizophrenia working group of the Psychiatric Genomics
Consortium. Biological insights from 108 schizophrenia-associated genetic
loci. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">511</span>:421–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4112379/">PMC free article</a>]</span> [<a href="/pubmed/25056061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biological+insights+from+108+schizophrenia-associated+genetic
loci&amp;volume=511&amp;publication_year=2014&amp;pages=421-7&amp;pmid=25056061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Ripke S, O'Dushlaine C, Chambert K, Moran JL, Kahler AK, Akterin S, et al.  Genome-wide association analysis identifies 13 new risk loci for
schizophrenia. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:1150–1159.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3827979/">PMC free article</a>]</span> [<a href="/pubmed/23974872" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+analysis+identifies+13+new+risk+loci+for
schizophrenia&amp;author=S+Ripke&amp;author=C+O'Dushlaine&amp;author=K+Chambert&amp;author=JL+Moran&amp;author=AK+Kahler&amp;volume=45&amp;publication_year=2013&amp;pages=1150-1159&amp;pmid=23974872&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Stefansson H, Ophoff RA, Steinberg S, Andreassen OA, Cichon S, Rujescu D, et al.  Common variants conferring risk of schizophrenia. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:744–747.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3077530/">PMC free article</a>]</span> [<a href="/pubmed/19571808" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+variants+conferring+risk+of+schizophrenia&amp;author=H+Stefansson&amp;author=RA+Ophoff&amp;author=S+Steinberg&amp;author=OA+Andreassen&amp;author=S+Cichon&amp;volume=460&amp;publication_year=2009&amp;pages=744-747&amp;pmid=19571808&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Purcell SM, Moran JL, Fromer M, Ruderfer D, Solovieff N, Roussos P, et al.  A polygenic burden of rare disruptive mutations in
schizophrenia. <span><span class="ref-journal">Nature. </span>2014;<span class="ref-vol">506</span>:185–190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4136494/">PMC free article</a>]</span> [<a href="/pubmed/24463508" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+polygenic+burden+of+rare+disruptive+mutations+in
schizophrenia&amp;author=SM+Purcell&amp;author=JL+Moran&amp;author=M+Fromer&amp;author=D+Ruderfer&amp;author=N+Solovieff&amp;volume=506&amp;publication_year=2014&amp;pages=185-190&amp;pmid=24463508&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Gaugler T, Klei L, Sanders S, Bodea C, Goldberg AP, Lee AB, Mahajan M, Manaa D, Pawitan Y, Reichert J, Ripke S, Sandin S, Sklar P, Svantesson O, Reichenberg A, Hultman CM, Devlin B, Roeder K, Buxbaum JD. Most genetic risk for autism resides with common
variation. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:881–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4137411/">PMC free article</a>]</span> [<a href="/pubmed/25038753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Most+genetic+risk+for+autism+resides+with+common
variation&amp;author=T+Gaugler&amp;author=L+Klei&amp;author=S+Sanders&amp;author=C+Bodea&amp;author=AP+Goldberg&amp;volume=46&amp;publication_year=2014&amp;pages=881-5&amp;pmid=25038753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Klei L, Sanders SJ, Murtha MT, Hus V, Lowe JK, Willsey AJ, et al.  Common genetic variants, acting additively, are a major source of
risk for autism. <span><span class="ref-journal">Mol Autism. </span>2012;<span class="ref-vol">3</span>:9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3579743/">PMC free article</a>]</span> [<a href="/pubmed/23067556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Autism&amp;title=Common+genetic+variants,+acting+additively,+are+a+major+source+of
risk+for+autism&amp;author=L+Klei&amp;author=SJ+Sanders&amp;author=MT+Murtha&amp;author=V+Hus&amp;author=JK+Lowe&amp;volume=3&amp;publication_year=2012&amp;pages=9&amp;pmid=23067556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Anney R, Klei L, Pinto D, Almeida J, Bacchelli E, Baird G, et al.  Individual common variants exert weak effects on the risk for
autism spectrum disorder. <span><span class="ref-journal">Hum Mol Genet. </span>2012;<span class="ref-vol">21</span>:4781–4792.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3471395/">PMC free article</a>]</span> [<a href="/pubmed/22843504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Individual+common+variants+exert+weak+effects+on+the+risk+for
autism+spectrum+disorder&amp;author=R+Anney&amp;author=L+Klei&amp;author=D+Pinto&amp;author=J+Almeida&amp;author=E+Bacchelli&amp;volume=21&amp;publication_year=2012&amp;pages=4781-4792&amp;pmid=22843504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Bishop SL, Hus V, Duncan A, Huerta M, Gotham K, Pickles A, et al.  Subcategories of restricted and repetitive behaviors in children
with autism spectrum disorders. <span><span class="ref-journal">J Autism Dev Disord. </span>2013;<span class="ref-vol">43</span>:1287–1297.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3579001/">PMC free article</a>]</span> [<a href="/pubmed/23065116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Subcategories+of+restricted+and+repetitive+behaviors+in+children
with+autism+spectrum+disorders&amp;author=SL+Bishop&amp;author=V+Hus&amp;author=A+Duncan&amp;author=M+Huerta&amp;author=K+Gotham&amp;volume=43&amp;publication_year=2013&amp;pages=1287-1297&amp;pmid=23065116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Vieland VJ, Hallmayer J, Huang Y, Pagnamenta AT, Pinto D, Khan H, et al.  Novel method for combined linkage and genome-wide association
analysis finds evidence of distinct genetic architecture for two subtypes of
autism. <span><span class="ref-journal">J Neurodev Disord. </span>2011;<span class="ref-vol">3</span>:113–123.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3105232/">PMC free article</a>]</span> [<a href="/pubmed/21484201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurodev+Disord&amp;title=Novel+method+for+combined+linkage+and+genome-wide+association
analysis+finds+evidence+of+distinct+genetic+architecture+for+two+subtypes+of
autism&amp;author=VJ+Vieland&amp;author=J+Hallmayer&amp;author=Y+Huang&amp;author=AT+Pagnamenta&amp;author=D+Pinto&amp;volume=3&amp;publication_year=2011&amp;pages=113-123&amp;pmid=21484201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Frazier TW, Youngstrom EA, Kubu CS, Sinclair L, Rezai A. Exploratory and confirmatory factor analysis of the autism
diagnostic interview-revised. <span><span class="ref-journal">J Autism Dev Disord. </span>2008;<span class="ref-vol">38</span>:474–480.</span> [<a href="/pubmed/17619129" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Exploratory+and+confirmatory+factor+analysis+of+the+autism
diagnostic+interview-revised&amp;author=TW+Frazier&amp;author=EA+Youngstrom&amp;author=CS+Kubu&amp;author=L+Sinclair&amp;author=A+Rezai&amp;volume=38&amp;publication_year=2008&amp;pages=474-480&amp;pmid=17619129&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Mandy WP, Charman T, Skuse DH. Testing the construct validity of proposed criteria for DSM-5
autism spectrum disorder. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2012;<span class="ref-vol">51</span>:41–50.</span> [<a href="/pubmed/22176938" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Testing+the+construct+validity+of+proposed+criteria+for+DSM-5
autism+spectrum+disorder&amp;author=WP+Mandy&amp;author=T+Charman&amp;author=DH+Skuse&amp;volume=51&amp;publication_year=2012&amp;pages=41-50&amp;pmid=22176938&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Snow AV, Lecavalier L. Comparing autism, PDD-NOS, and other developmental disabilities
on parent-reported behavior problems: little evidence for ASD subtype
validity. <span><span class="ref-journal">J Autism Dev Disord. </span>2011;<span class="ref-vol">41</span>:302–310.</span> [<a href="/pubmed/20556500" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Comparing+autism,+PDD-NOS,+and+other+developmental+disabilities
on+parent-reported+behavior+problems:+little+evidence+for+ASD+subtype
validity&amp;author=AV+Snow&amp;author=L+Lecavalier&amp;volume=41&amp;publication_year=2011&amp;pages=302-310&amp;pmid=20556500&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Lord C, Petkova E, Hus V, Gan W, Lu F, Martin DM, et al.  A multisite study of the clinical diagnosis of different autism
spectrum disorders. <span><span class="ref-journal">Arch Gen Psychiatry. </span>2012;<span class="ref-vol">69</span>:306–313.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3626112/">PMC free article</a>]</span> [<a href="/pubmed/22065253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Gen+Psychiatry&amp;title=A+multisite+study+of+the+clinical+diagnosis+of+different+autism
spectrum+disorders&amp;author=C+Lord&amp;author=E+Petkova&amp;author=V+Hus&amp;author=W+Gan&amp;author=F+Lu&amp;volume=69&amp;publication_year=2012&amp;pages=306-313&amp;pmid=22065253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Walker DR, Thompson A, Zwaigenbaum L, Goldberg J, Bryson SE, Mahoney WJ, et al.  Specifying PDD-NOS: a comparison of PDD-NOS, Asperger syndrome,
and autism. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2004;<span class="ref-vol">43</span>:172–180.</span> [<a href="/pubmed/14726723" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Specifying+PDD-NOS:+a+comparison+of+PDD-NOS,+Asperger+syndrome,
and+autism&amp;author=DR+Walker&amp;author=A+Thompson&amp;author=L+Zwaigenbaum&amp;author=J+Goldberg&amp;author=SE+Bryson&amp;volume=43&amp;publication_year=2004&amp;pages=172-180&amp;pmid=14726723&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Salyakina D, Ma DQ, Jaworski JM, Konidari I, Whitehead PL, Henson R, et al.  Variants in several genomic regions associated with asperger
disorder. <span><span class="ref-journal">Autism Res. </span>2010;<span class="ref-vol">3</span>:303–310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4435556/">PMC free article</a>]</span> [<a href="/pubmed/21182207" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autism+Res&amp;title=Variants+in+several+genomic+regions+associated+with+asperger
disorder&amp;author=D+Salyakina&amp;author=DQ+Ma&amp;author=JM+Jaworski&amp;author=I+Konidari&amp;author=PL+Whitehead&amp;volume=3&amp;publication_year=2010&amp;pages=303-310&amp;pmid=21182207&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Fischbach GD, Lord C. The Simons Simplex Collection: a resource for identification of
autism genetic risk factors. <span><span class="ref-journal">Neuron. </span>2010;<span class="ref-vol">68</span>:192–195.</span> [<a href="/pubmed/20955926" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=The+Simons+Simplex+Collection:+a+resource+for+identification+of
autism+genetic+risk+factors&amp;author=GD+Fischbach&amp;author=C+Lord&amp;volume=68&amp;publication_year=2010&amp;pages=192-195&amp;pmid=20955926&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Nordahl CW, Lange N, Li DD, Barnett LA, Lee A, Buonocore MH, et al.  Brain enlargement is associated with regression in preschool-age
boys with autism spectrum disorders. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2011;<span class="ref-vol">108</span>:20195–20200.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3250128/">PMC free article</a>]</span> [<a href="/pubmed/22123952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Brain+enlargement+is+associated+with+regression+in+preschool-age
boys+with+autism+spectrum+disorders&amp;author=CW+Nordahl&amp;author=N+Lange&amp;author=DD+Li&amp;author=LA+Barnett&amp;author=A+Lee&amp;volume=108&amp;publication_year=2011&amp;pages=20195-20200&amp;pmid=22123952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Tuchman R, Hirtz D, Mamounas LA. NINDS epilepsy and autism spectrum disorders workshop
report. <span><span class="ref-journal">Neurology. </span>2013;<span class="ref-vol">81</span>:1630–1636.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3806917/">PMC free article</a>]</span> [<a href="/pubmed/24089385" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=NINDS+epilepsy+and+autism+spectrum+disorders+workshop
report&amp;author=R+Tuchman&amp;author=D+Hirtz&amp;author=LA+Mamounas&amp;volume=81&amp;publication_year=2013&amp;pages=1630-1636&amp;pmid=24089385&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Gotham K, Pickles A, Lord C. Standardizing ADOS scores for a measure of severity in autism
spectrum disorders. <span><span class="ref-journal">J Autism Dev Disord. </span>2009;<span class="ref-vol">39</span>:693–705.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922918/">PMC free article</a>]</span> [<a href="/pubmed/19082876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Standardizing+ADOS+scores+for+a+measure+of+severity+in+autism
spectrum+disorders&amp;author=K+Gotham&amp;author=A+Pickles&amp;author=C+Lord&amp;volume=39&amp;publication_year=2009&amp;pages=693-705&amp;pmid=19082876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Hus V, Gotham K, Lord C. Standardizing ADOS Domain Scores: Separating Severity of Social
Affect and Restricted and Repetitive Behaviors. <span><span class="ref-journal">J Autism Dev Disord </span>2012</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612387/">PMC free article</a>]</span> [<a href="/pubmed/23143131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Standardizing+ADOS+Domain+Scores:+Separating+Severity+of+Social
Affect+and+Restricted+and+Repetitive+Behaviors&amp;author=V+Hus&amp;author=K+Gotham&amp;author=C+Lord&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Crossett A, Kent BP, Klei L, Ringquist S, Trucco M, Roeder K, et al.  Using ancestry matching to combine family-based and unrelated
samples for genome-wide association studies. <span><span class="ref-journal">Stat Med. </span>2010;<span class="ref-vol">29</span>:2932–2945.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4629477/">PMC free article</a>]</span> [<a href="/pubmed/20862653" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Med&amp;title=Using+ancestry+matching+to+combine+family-based+and+unrelated
samples+for+genome-wide+association+studies&amp;author=A+Crossett&amp;author=BP+Kent&amp;author=L+Klei&amp;author=S+Ringquist&amp;author=M+Trucco&amp;volume=29&amp;publication_year=2010&amp;pages=2932-2945&amp;pmid=20862653&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Laird NM, Horvath S, Xu X. Implementing a unified approach to family-based tests of
association. <span><span class="ref-journal">Genet Epidemiol. </span>2000;<span class="ref-vol">19 Suppl 1</span>:S36–42.</span> [<a href="/pubmed/11055368" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Implementing+a+unified+approach+to+family-based+tests+of
association&amp;author=NM+Laird&amp;author=S+Horvath&amp;author=X+Xu&amp;volume=19+Suppl+1&amp;publication_year=2000&amp;pages=S36-42&amp;pmid=11055368&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span>Health-ABC  <a href="http://www.ncbi.nlm.nih.gov/projects/gap/cgibin/study.cgi?studyid=phs000169.v1.p1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/cgibin/study.cgi?studyid=phs000169.v1.p1</a></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait
analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait
analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2010&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Purcell SM, Wray NR, Stone JL, Visscher PM, O'Donovan MC, Sullivan PF, et al.  Common polygenic variation contributes to risk of schizophrenia
and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia
and+bipolar+disorder&amp;author=SM+Purcell&amp;author=NR+Wray&amp;author=JL+Stone&amp;author=PM+Visscher&amp;author=MC+O'Donovan&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Tibshirani R. Regression shrinkage and selection via the lasso. <span><span class="ref-journal">J Royal Statist Soc B. </span>1996;<span class="ref-vol">58</span>:267–288.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Royal+Statist+Soc+B&amp;title=Regression+shrinkage+and+selection+via+the+lasso&amp;author=R+Tibshirani&amp;volume=58&amp;publication_year=1996&amp;pages=267-288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Gordon D, Heath SC, Liu X, Ott J. A transmission/disequilibrium test that allows for genotyping
errors in the analysis of single-nucleotide polymorphism
data. <span><span class="ref-journal">Am J Hum Genet. </span>2001;<span class="ref-vol">69</span>:371–380.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1235309/">PMC free article</a>]</span> [<a href="/pubmed/11443542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+transmission/disequilibrium+test+that+allows+for+genotyping
errors+in+the+analysis+of+single-nucleotide+polymorphism
data&amp;author=D+Gordon&amp;author=SC+Heath&amp;author=X+Liu&amp;author=J+Ott&amp;volume=69&amp;publication_year=2001&amp;pages=371-380&amp;pmid=11443542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Dewez M, Bauer F, Dieu M, Raes M, Vandenhaute J, Hermand D. The conserved Wobble uridine tRNA thiolase Ctu1-Ctu2 is required
to maintain genome integrity. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2008;<span class="ref-vol">105</span>:5459–5464.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2291126/">PMC free article</a>]</span> [<a href="/pubmed/18391219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=The+conserved+Wobble+uridine+tRNA+thiolase+Ctu1-Ctu2+is+required
to+maintain+genome+integrity&amp;author=M+Dewez&amp;author=F+Bauer&amp;author=M+Dieu&amp;author=M+Raes&amp;author=J+Vandenhaute&amp;volume=105&amp;publication_year=2008&amp;pages=5459-5464&amp;pmid=18391219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Kumagai A, Lee J, Yoo HY, Dunphy WG. TopBP1 activates the ATR-ATRIP complex. <span><span class="ref-journal">Cell. </span>2006;<span class="ref-vol">124</span>:943–955.</span> [<a href="/pubmed/16530042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=TopBP1+activates+the+ATR-ATRIP+complex&amp;author=A+Kumagai&amp;author=J+Lee&amp;author=HY+Yoo&amp;author=WG+Dunphy&amp;volume=124&amp;publication_year=2006&amp;pages=943-955&amp;pmid=16530042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Zhang WN, Zhou J, Zhou T, Li AL, Wang N, Xu JJ, et al.  Phosphorylation-triggered CUEDC2 degradation promotes UV-induced
G1 arrest through APC/C(Cdh1) regulation. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2013a;<span class="ref-vol">110</span>:11017–11022.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3703968/">PMC free article</a>]</span> [<a href="/pubmed/23776205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Phosphorylation-triggered+CUEDC2+degradation+promotes+UV-induced
G1+arrest+through+APC/C(Cdh1)+regulation&amp;author=WN+Zhang&amp;author=J+Zhou&amp;author=T+Zhou&amp;author=AL+Li&amp;author=N+Wang&amp;volume=110&amp;publication_year=2013a&amp;pages=11017-11022&amp;pmid=23776205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Zhang Y, Park E, Kim CS, Paik JH. ZNF365 promotes stalled replication forks recovery to maintain
genome stability. <span><span class="ref-journal">Cell Cycle. </span>2013b;<span class="ref-vol">12</span>:2817–2828.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3899195/">PMC free article</a>]</span> [<a href="/pubmed/23966166" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=ZNF365+promotes+stalled+replication+forks+recovery+to+maintain
genome+stability&amp;author=Y+Zhang&amp;author=E+Park&amp;author=CS+Kim&amp;author=JH+Paik&amp;volume=12&amp;publication_year=2013b&amp;pages=2817-2828&amp;pmid=23966166&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Zhang Y, Shin SJ, Liu D, Ivanova E, Foerster F, Ying H, et al.  ZNF365 promotes stability of fragile sites and
telomeres. <span><span class="ref-journal">Cancer Discov. </span>2013c;<span class="ref-vol">3</span>:798–811.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3710545/">PMC free article</a>]</span> [<a href="/pubmed/23776040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Discov&amp;title=ZNF365+promotes+stability+of+fragile+sites+and
telomeres&amp;author=Y+Zhang&amp;author=SJ+Shin&amp;author=D+Liu&amp;author=E+Ivanova&amp;author=F+Foerster&amp;volume=3&amp;publication_year=2013c&amp;pages=798-811&amp;pmid=23776040&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Gamazon ER, Zhang W, Konkashbaev A, Duan S, Kistner EO, Nicolae DL, et al.  SCAN: SNP and copy number annotation. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:259–262.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2852202/">PMC free article</a>]</span> [<a href="/pubmed/19933162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SCAN:+SNP+and+copy+number+annotation&amp;author=ER+Gamazon&amp;author=W+Zhang&amp;author=A+Konkashbaev&amp;author=S+Duan&amp;author=EO+Kistner&amp;volume=26&amp;publication_year=2010&amp;pages=259-262&amp;pmid=19933162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Consortium TSPG-WASG. Genome-wide association study identifies five new schizophrenia
loci. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">43</span>:969–976.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3303194/">PMC free article</a>]</span> [<a href="/pubmed/21926974" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+five+new+schizophrenia
loci&amp;volume=43&amp;publication_year=2011&amp;pages=969-976&amp;pmid=21926974&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Strohmaier H, Spruck CH, Kaiser P, Won KA, Sangfelt O, Reed SI. Human F-box protein hCdc4 targets cyclin E for proteolysis and is
mutated in a breast cancer cell line. <span><span class="ref-journal">Nature. </span>2001;<span class="ref-vol">413</span>:316–322.</span> [<a href="/pubmed/11565034" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+F-box+protein+hCdc4+targets+cyclin+E+for+proteolysis+and+is
mutated+in+a+breast+cancer+cell+line&amp;author=H+Strohmaier&amp;author=CH+Spruck&amp;author=P+Kaiser&amp;author=KA+Won&amp;author=O+Sangfelt&amp;volume=413&amp;publication_year=2001&amp;pages=316-322&amp;pmid=11565034&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Noor A, Whibley A, Marshall CR, Gianakopoulos PJ, Piton A, Carson AR, et al.  Disruption at the PTCHD1 Locus on Xp22.11 in Autism spectrum
disorder and intellectual disability. <span><span class="ref-journal">Sci Transl Med. </span>2010;<span class="ref-vol">2</span>:49ra68.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2987731/">PMC free article</a>]</span> [<a href="/pubmed/20844286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Disruption+at+the+PTCHD1+Locus+on+Xp22.11+in+Autism+spectrum
disorder+and+intellectual+disability&amp;author=A+Noor&amp;author=A+Whibley&amp;author=CR+Marshall&amp;author=PJ+Gianakopoulos&amp;author=A+Piton&amp;volume=2&amp;publication_year=2010&amp;pages=49ra68&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="element-citation">Samocha KE, Robinson EB, Sanders SJ, Stevens C, Sabo A, McGrath LM, et al.  A framework for the interpretation of <em>de novo</em>
mutation in human disease. <span>2014 doi: 10.1038/ng.3050.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4222185/">PMC free article</a>]</span> [<a href="/pubmed/25086666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//dx.doi.org/10.1038%2Fng.3050" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Samocha+KE+Robinson+EB+Sanders+SJ+Stevens+C+Sabo+A+McGrath+LM+2014+A+framework+for+the+interpretation+of++de+novo+
mutation+in+human+disease+10.1038/ng.3050+" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="element-citation">Stefansson H, Meyer-Lindenberg A, Steinberg S, Magnusdottir B, Morgen K, Arnarsdottir S, et al.  CNVs conferring risk of autism or schizophrenia affect cognition
in controls <span>2014</span> [<a href="/pubmed/24352232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Stefansson+H+Meyer-Lindenberg+A+Steinberg+S+Magnusdottir+B+Morgen+K+Arnarsdottir+S+2014+CNVs+conferring+risk+of+autism+or+schizophrenia+affect+cognition
in+controls+" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="element-citation">Dworzynski K, Happe F, Bolton P, Ronald A. Relationship between symptom domains in autism spectrum
disorders: a population based twin study. <span><span class="ref-journal">J Autism Dev Disord. </span>2009;<span class="ref-vol">39</span>:1197–1210.</span> [<a href="/pubmed/19373549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Relationship+between+symptom+domains+in+autism+spectrum
disorders:+a+population+based+twin+study&amp;author=K+Dworzynski&amp;author=F+Happe&amp;author=P+Bolton&amp;author=A+Ronald&amp;volume=39&amp;publication_year=2009&amp;pages=1197-1210&amp;pmid=19373549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="element-citation">Ronald A, Larsson H, Anckarsater H, Lichtenstein P. A twin study of autism symptoms in Sweden. <span><span class="ref-journal">Mol Psychiatry. </span>2011;<span class="ref-vol">16</span>:1039–1047.</span> [<a href="/pubmed/20644553" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+twin+study+of+autism+symptoms+in+Sweden&amp;author=A+Ronald&amp;author=H+Larsson&amp;author=H+Anckarsater&amp;author=P+Lichtenstein&amp;volume=16&amp;publication_year=2011&amp;pages=1039-1047&amp;pmid=20644553&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="element-citation">Iossifov I, Ronemus M, Levy D, Wang Z, Hakker I, Rosenbaum J, et al.  De novo gene disruptions in children on the autistic
spectrum. <span><span class="ref-journal">Neuron. </span>2012;<span class="ref-vol">74</span>:285–299.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3619976/">PMC free article</a>]</span> [<a href="/pubmed/22542183" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=De+novo+gene+disruptions+in+children+on+the+autistic
spectrum&amp;author=I+Iossifov&amp;author=M+Ronemus&amp;author=D+Levy&amp;author=Z+Wang&amp;author=I+Hakker&amp;volume=74&amp;publication_year=2012&amp;pages=285-299&amp;pmid=22542183&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="element-citation">Robinson EB, Lichtenstein P, Anckarsater H, Happe F, Ronald A. Examining and interpreting the female protective effect against
autistic behavior. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2013;<span class="ref-vol">110</span>:5258–5262.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3612665/">PMC free article</a>]</span> [<a href="/pubmed/23431162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Examining+and+interpreting+the+female+protective+effect+against
autistic+behavior&amp;author=EB+Robinson&amp;author=P+Lichtenstein&amp;author=H+Anckarsater&amp;author=F+Happe&amp;author=A+Ronald&amp;volume=110&amp;publication_year=2013&amp;pages=5258-5262&amp;pmid=23431162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="element-citation">Szatmari P, Liu XQ, Goldberg J, Zwaigenbaum L, Paterson AD, Woodbury-Smith M, et al.  Sex differences in repetitive stereotyped behaviors in autism:
implications for genetic liability. <span><span class="ref-journal">Am J Med Genet B Neuropsychiatr Genet. </span>2012;<span class="ref-vol">159B</span>:5–12.</span> [<a href="/pubmed/22095612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Med+Genet+B+Neuropsychiatr+Genet&amp;title=Sex+differences+in+repetitive+stereotyped+behaviors+in+autism:
implications+for+genetic+liability&amp;author=P+Szatmari&amp;author=XQ+Liu&amp;author=J+Goldberg&amp;author=L+Zwaigenbaum&amp;author=AD+Paterson&amp;volume=159B&amp;publication_year=2012&amp;pages=5-12&amp;pmid=22095612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="element-citation">Lord C, Schopler E, Revicki D. Sex differences in autism. <span><span class="ref-journal">J Autism Dev Disord. </span>1982;<span class="ref-vol">12</span>:317–330.</span> [<a href="/pubmed/7161234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autism+Dev+Disord&amp;title=Sex+differences+in+autism&amp;author=C+Lord&amp;author=E+Schopler&amp;author=D+Revicki&amp;volume=12&amp;publication_year=1982&amp;pages=317-330&amp;pmid=7161234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="element-citation">Ionita-Laza I, Xu B, Makarov V, Buxbaum JD, Roos JL, Gogos JA, et al.  Scan statistic-based analysis of exome sequencing data identifies
FAN1 at 15q13.3 as a susceptibility gene for schizophrenia and
autism. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2014;<span class="ref-vol">111</span>:343–348.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3890869/">PMC free article</a>]</span> [<a href="/pubmed/24344280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Scan+statistic-based+analysis+of+exome+sequencing+data+identifies
FAN1+at+15q13.3+as+a+susceptibility+gene+for+schizophrenia+and
autism&amp;author=I+Ionita-Laza&amp;author=B+Xu&amp;author=V+Makarov&amp;author=JD+Buxbaum&amp;author=JL+Roos&amp;volume=111&amp;publication_year=2014&amp;pages=343-348&amp;pmid=24344280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="element-citation">Mao JH, Kim IJ, Wu D, Climent J, Kang HC, DelRosario R, et al.  FBXW7 targets mTOR for degradation and cooperates with PTEN in
tumor suppression. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">321</span>:1499–1502.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2849753/">PMC free article</a>]</span> [<a href="/pubmed/18787170" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=FBXW7+targets+mTOR+for+degradation+and+cooperates+with+PTEN+in
tumor+suppression&amp;author=JH+Mao&amp;author=IJ+Kim&amp;author=D+Wu&amp;author=J+Climent&amp;author=HC+Kang&amp;volume=321&amp;publication_year=2008&amp;pages=1499-1502&amp;pmid=18787170&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="element-citation">Hoeck JD, Jandke A, Blake SM, Nye E, Spencer-Dene B, Brandner S, et al.  Fbw7 controls neural stem cell differentiation and progenitor
apoptosis via Notch and c-Jun. <span><span class="ref-journal">Nat Neurosci. </span>2010;<span class="ref-vol">13</span>:1365–1372.</span> [<a href="/pubmed/20935640" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Fbw7+controls+neural+stem+cell+differentiation+and+progenitor
apoptosis+via+Notch+and+c-Jun&amp;author=JD+Hoeck&amp;author=A+Jandke&amp;author=SM+Blake&amp;author=E+Nye&amp;author=B+Spencer-Dene&amp;volume=13&amp;publication_year=2010&amp;pages=1365-1372&amp;pmid=20935640&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="element-citation">Jandke A, Da Costa C, Sancho R, Nye E, Spencer-Dene B, Behrens A. The F-box protein Fbw7 is required for cerebellar
development. <span><span class="ref-journal">Dev Biol. </span>2011;<span class="ref-vol">358</span>:201–212.</span> [<a href="/pubmed/21827743" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=The+F-box+protein+Fbw7+is+required+for+cerebellar
development&amp;author=A+Jandke&amp;author=C+Da+Costa&amp;author=R+Sancho&amp;author=E+Nye&amp;author=B+Spencer-Dene&amp;volume=358&amp;publication_year=2011&amp;pages=201-212&amp;pmid=21827743&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="element-citation">Hattori T, Baba K, Matsuzaki S, Honda A, Miyoshi K, Inoue K, et al.  A novel DISC1-interacting partner DISC1-Binding Zinc-finger
protein: implication in the modulation of DISC1-dependent neurite
outgrowth. <span><span class="ref-journal">Mol Psychiatry. </span>2007;<span class="ref-vol">12</span>:398–407.</span> [<a href="/pubmed/17389905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Psychiatry&amp;title=A+novel+DISC1-interacting+partner+DISC1-Binding+Zinc-finger
protein:+implication+in+the+modulation+of+DISC1-dependent+neurite
outgrowth&amp;author=T+Hattori&amp;author=K+Baba&amp;author=S+Matsuzaki&amp;author=A+Honda&amp;author=K+Miyoshi&amp;volume=12&amp;publication_year=2007&amp;pages=398-407&amp;pmid=17389905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="element-citation">Hennah W, Varilo T, Kestila M, Paunio T, Arajarvi R, Haukka J, et al.  Haplotype transmission analysis provides evidence of association
for DISC1 to schizophrenia and suggests sex-dependent
effects. <span><span class="ref-journal">Hum Mol Genet. </span>2003;<span class="ref-vol">12</span>:3151–3159.</span> [<a href="/pubmed/14532331" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Haplotype+transmission+analysis+provides+evidence+of+association
for+DISC1+to+schizophrenia+and+suggests+sex-dependent
effects&amp;author=W+Hennah&amp;author=T+Varilo&amp;author=M+Kestila&amp;author=T+Paunio&amp;author=R+Arajarvi&amp;volume=12&amp;publication_year=2003&amp;pages=3151-3159&amp;pmid=14532331&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="element-citation">Murray RZ, Wylie FG, Khromykh T, Hume DA, Stow JL. Syntaxin 6 and Vti1b form a novel SNARE complex, which is
up-regulated in activated macrophages to facilitate exocytosis of tumor
necrosis Factor-alpha. <span><span class="ref-journal">J Biol Chem. </span>2005;<span class="ref-vol">280</span>:10478–10483.</span> [<a href="/pubmed/15640147" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Syntaxin+6+and+Vti1b+form+a+novel+SNARE+complex,+which+is
up-regulated+in+activated+macrophages+to+facilitate+exocytosis+of+tumor
necrosis+Factor-alpha&amp;author=RZ+Murray&amp;author=FG+Wylie&amp;author=T+Khromykh&amp;author=DA+Hume&amp;author=JL+Stow&amp;volume=280&amp;publication_year=2005&amp;pages=10478-10483&amp;pmid=15640147&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="element-citation">Filges I, Rothlisberger B, Blattner A, Boesch N, Demougin P, Wenzel F, et al.  Deletion in Xp22.11: PTCHD1 is a candidate gene for X-linked
intellectual disability with or without autism. <span><span class="ref-journal">Clin Genet. </span>2011;<span class="ref-vol">79</span>:79–85.</span> [<a href="/pubmed/21091464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Deletion+in+Xp22.11:+PTCHD1+is+a+candidate+gene+for+X-linked
intellectual+disability+with+or+without+autism&amp;author=I+Filges&amp;author=B+Rothlisberger&amp;author=A+Blattner&amp;author=N+Boesch&amp;author=P+Demougin&amp;volume=79&amp;publication_year=2011&amp;pages=79-85&amp;pmid=21091464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">69. <span class="element-citation">Pinto D, Pagnamenta AT, Klei L, Anney R, Merico D, Regan R, et al.  Functional impact of global rare copy number variation in autism
spectrum disorders. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>:368–372.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3021798/">PMC free article</a>]</span> [<a href="/pubmed/20531469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Functional+impact+of+global+rare+copy+number+variation+in+autism
spectrum+disorders&amp;author=D+Pinto&amp;author=AT+Pagnamenta&amp;author=L+Klei&amp;author=R+Anney&amp;author=D+Merico&amp;volume=466&amp;publication_year=2010&amp;pages=368-372&amp;pmid=20531469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">70. <span class="element-citation">Sanders SJ, Ercan-Sencicek AG, Hus V, Luo R, Murtha MT, Moreno-De-Luca D, et al.  Multiple recurrent de novo CNVs, including duplications of the
7q11.23 Williams syndrome region, are strongly associated with
autism. <span><span class="ref-journal">Neuron. </span>2011;<span class="ref-vol">70</span>:863–885.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3939065/">PMC free article</a>]</span> [<a href="/pubmed/21658581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Multiple+recurrent+de+novo+CNVs,+including+duplications+of+the
7q11.23+Williams+syndrome+region,+are+strongly+associated+with
autism&amp;author=SJ+Sanders&amp;author=AG+Ercan-Sencicek&amp;author=V+Hus&amp;author=R+Luo&amp;author=MT+Murtha&amp;volume=70&amp;publication_year=2011&amp;pages=863-885&amp;pmid=21658581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">71. <span class="element-citation">Whibley AC, Plagnol V, Tarpey PS, Abidi F, Fullston T, Choma MK, et al.  Fine-scale survey of X chromosome copy number variants and indels
underlying intellectual disability. <span><span class="ref-journal">Am J Hum Genet. </span>2010;<span class="ref-vol">87</span>:173–188.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2917707/">PMC free article</a>]</span> [<a href="/pubmed/20655035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Fine-scale+survey+of+X+chromosome+copy+number+variants+and+indels
underlying+intellectual+disability&amp;author=AC+Whibley&amp;author=V+Plagnol&amp;author=PS+Tarpey&amp;author=F+Abidi&amp;author=T+Fullston&amp;volume=87&amp;publication_year=2010&amp;pages=173-188&amp;pmid=20655035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">72. <span class="element-citation">Snow AV, Lecavalier L, Houts C. The structure of the Autism Diagnostic Interview-Revised:
diagnostic and phenotypic implications. <span><span class="ref-journal">J Child Psychol Psychiatry. </span>2009;<span class="ref-vol">50</span>:734–742.</span> [<a href="/pubmed/19207624" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Child+Psychol+Psychiatry&amp;title=The+structure+of+the+Autism+Diagnostic+Interview-Revised:
diagnostic+and+phenotypic+implications&amp;author=AV+Snow&amp;author=L+Lecavalier&amp;author=C+Houts&amp;volume=50&amp;publication_year=2009&amp;pages=734-742&amp;pmid=19207624&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">73. <span class="element-citation">Bailey A, Le Couteur A, Gottesman I, Bolton P, Simonoff E, Yuzda E, et al.  Autism as a strongly genetic disorder: evidence from a British
twin study. <span><span class="ref-journal">Psychol Med. </span>1995;<span class="ref-vol">25</span>:63–77.</span> [<a href="/pubmed/7792363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Psychol+Med&amp;title=Autism+as+a+strongly+genetic+disorder:+evidence+from+a+British
twin+study&amp;author=A+Bailey&amp;author=A+Le+Couteur&amp;author=I+Gottesman&amp;author=P+Bolton&amp;author=E+Simonoff&amp;volume=25&amp;publication_year=1995&amp;pages=63-77&amp;pmid=7792363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">74. <span class="element-citation">Stilp RL, Gernsbacher MA, Schweigert EK, Arneson CL, Goldsmith HH. Genetic variance for autism screening items in an unselected
sample of toddler-age twins. <span><span class="ref-journal">J Am Acad Child Adolesc Psychiatry. </span>2010;<span class="ref-vol">49</span>:267–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2947446/">PMC free article</a>]</span> [<a href="/pubmed/20410716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Acad+Child+Adolesc+Psychiatry&amp;title=Genetic+variance+for+autism+screening+items+in+an+unselected
sample+of+toddler-age+twins&amp;author=RL+Stilp&amp;author=MA+Gernsbacher&amp;author=EK+Schweigert&amp;author=CL+Arneson&amp;author=HH+Goldsmith&amp;volume=49&amp;publication_year=2010&amp;pages=267-276&amp;pmid=20410716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">75. <span class="element-citation">Devlin B, Melhem N, Roeder K. Do common variants play a role in risk for autism? Evidence and
theoretical musings. <span><span class="ref-journal">Brain Res. </span>2010;<span class="ref-vol">1380</span>:78–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3638767/">PMC free article</a>]</span> [<a href="/pubmed/21078308" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Res&amp;title=Do+common+variants+play+a+role+in+risk+for+autism?+Evidence+and
theoretical+musings&amp;author=B+Devlin&amp;author=N+Melhem&amp;author=K+Roeder&amp;volume=1380&amp;publication_year=2010&amp;pages=78-84&amp;pmid=21078308&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4379124&amp;issue-id=251138&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140324007628736title">Abstract</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Method</a></li><li><a href="#S12title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#S17title">Supplementary Material</a></li><li><a href="#S18title">Acknowledgments</a></li><li><a href="#idm140324006222848title">Footnotes</a></li><li><a href="#idm140324013517808title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4379124/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4379124/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4379124/pdf/nihms631893.pdf">PDF (357K)</a></li> | <li><a href="#" data-citationid="PMC4379124" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4379124%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4379124%2F&amp;text=A%20genomewide%20association%20study%20of%20autism%20using%20the%20Simons%20Simplex%0ACollection%3A%20Does%20reducing%20phenotypic%20heterogeneity%20in%20autism%20increase%20genetic%0Ahomogeneity%3F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4379124%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25534755" data-aiid="4379124" data-aid="4379124" data-iid="251138" data-domainid="319" data-domain="nihpa" data-accid="PMC4379124" data-md5="48def296142fad65a47f68367693dd0a"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:08:53-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8918EFDD5711C1000000000ABA069D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4379124%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4379124/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>