<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jlr"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4076081/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of the plasma triglyceride response to an n-3 polyunsaturated fatty acid supplementation</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4076081/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Lipid Research"><meta name="citation_title" content="Genome-wide association study of the plasma triglyceride response to an n-3 polyunsaturated fatty acid supplementation"><meta name="citation_authors" content="Iwona Rudkowska, Frédéric Guénard, Pierre Julien, Patrick Couture, Simone Lemieux, Olivier Barbier, Philip C. Calder, Anne Marie Minihane, Marie-Claude Vohl"><meta name="citation_date" content="July 2014"><meta name="citation_issue" content="7"><meta name="citation_volume" content="55"><meta name="citation_firstpage" content="1245"><meta name="citation_doi" content="10.1194/jlr.M045898"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4076081/?report=abstract"><meta name="citation_pmid" content="24847101"><meta name="DC.Title" content="Genome-wide association study of the plasma triglyceride response to an n-3 polyunsaturated fatty acid supplementation"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society for Biochemistry and Molecular Biology"><meta name="DC.Contributor" content="Iwona Rudkowska"><meta name="DC.Contributor" content="Frédéric Guénard"><meta name="DC.Contributor" content="Pierre Julien"><meta name="DC.Contributor" content="Patrick Couture"><meta name="DC.Contributor" content="Simone Lemieux"><meta name="DC.Contributor" content="Olivier Barbier"><meta name="DC.Contributor" content="Philip C. Calder"><meta name="DC.Contributor" content="Anne Marie Minihane"><meta name="DC.Contributor" content="Marie-Claude Vohl"><meta name="DC.Date" content="2014 Jul"><meta name="DC.Identifier" content="10.1194/jlr.M045898"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of the plasma triglyceride response to an n-3 polyunsaturated fatty acid supplementation"><meta property="og:type" content="article"><meta property="og:description" content="Studies have shown a large interindividual variability in plasma TG response to long-chain n-3 PUFA supplementation, which may likely be attributable to genetic variability within the populations studied. The objective is to compare the frequency of SNPs ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4076081/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jlr.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4076081/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4076081/pdf/1245.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BB144DD56FFF100000000030001B6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4076081%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/568/">J Lipid Res</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/239622/">v.55(7); 2014 Jul</a></li><li class="accid">PMC4076081</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jlr.png" alt="Logo of jlr" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,500,75" alt="Journal of Lipid Research" title="Journal of Lipid Research" href="http://www.jlr.org" ref="reftype=publisher&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Lipid Res</a></span>. 2014 Jul; 55(7): 1245–1253. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1194%2Fjlr.M045898" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1194/jlr.M045898</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4076081</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/24847101">24847101</a></div></div></div></div><h1 class="content-title">Genome-wide association study of the plasma triglyceride response to an n-3 polyunsaturated fatty acid supplementation<sup><a href="#fn3" rid="fn3" class=" fn">[S]</a></sup></h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Rudkowska%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979260532560" co-class="co-affbox">Iwona Rudkowska</a>,<sup>1,</sup><sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Gu%26%23x000e9%3Bnard%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979263882256" co-class="co-affbox">Frédéric Guénard</a>,<sup>1,</sup><sup>*</sup><sup>†</sup> <a href="/pubmed/?term=Julien%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979263878672" co-class="co-affbox">Pierre Julien</a>,<sup>†</sup> <a href="/pubmed/?term=Couture%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979265254912" co-class="co-affbox">Patrick Couture</a>,<sup>*</sup> <a href="/pubmed/?term=Lemieux%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979265252928" co-class="co-affbox">Simone Lemieux</a>,<sup>*</sup> <a href="/pubmed/?term=Barbier%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979265250944" co-class="co-affbox">Olivier Barbier</a>,<sup>†</sup><sup>§</sup><sup>**</sup> <a href="/pubmed/?term=Calder%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979265247488" co-class="co-affbox">Philip C. Calder</a>,<sup>††</sup><sup>§§</sup><sup>***</sup> <a href="/pubmed/?term=Minihane%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979334569072" co-class="co-affbox">Anne Marie Minihane</a>,<sup>†††</sup> and  <a href="/pubmed/?term=Vohl%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101" class="affpopup" co-rid="_co_idm139979334567088" co-class="co-affbox">Marie-Claude Vohl</a><sup>*</sup><sup>†</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139979260532560"><h3 class="no_margin">Iwona Rudkowska</h3><p><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</p><p><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Rudkowska%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Iwona Rudkowska</a></div></div><div id="_co_idm139979263882256"><h3 class="no_margin">Frédéric Guénard</h3><p><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</p><p><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Gu%26%23x000e9%3Bnard%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Frédéric Guénard</a></div></div><div id="_co_idm139979263878672"><h3 class="no_margin">Pierre Julien</h3><p><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Julien%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Pierre Julien</a></div></div><div id="_co_idm139979265254912"><h3 class="no_margin">Patrick Couture</h3><p><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Couture%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Patrick Couture</a></div></div><div id="_co_idm139979265252928"><h3 class="no_margin">Simone Lemieux</h3><p><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Lemieux%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Simone Lemieux</a></div></div><div id="_co_idm139979265250944"><h3 class="no_margin">Olivier Barbier</h3><p><sup>**</sup>Faculty of Pharmacy, Laval University, Quebec, Canada</p><p><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</p><p><sup>§</sup>Laboratory of Molecular Pharmacology, CHU de Québec Research Center, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Barbier%20O%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Olivier Barbier</a></div></div><div id="_co_idm139979265247488"><h3 class="no_margin">Philip C. Calder</h3><p><sup>††</sup>Human Development and Health Academic Unit, Faculty of Medicine, University of Southampton, Southampton, United Kingdom</p><p><sup>§§</sup>NIHR Southampton Biomedical Research Centre, University Hospital Southampton NHS Foundation Trust and University of Southampton, Southampton, United Kingdom</p><p><sup>***</sup>Department of Biological Sciences, Faculty of Science, King Abdulaziz University, Jeddah, Saudi Arabia,  Department of Nutrition</p><div>Find articles by <a href="/pubmed/?term=Calder%20PC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Philip C. Calder</a></div></div><div id="_co_idm139979334569072"><h3 class="no_margin">Anne Marie Minihane</h3><p><sup>†††</sup> Norwich Medical School, University of East Anglia, Norwich, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Minihane%20AM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Anne Marie Minihane</a></div></div><div id="_co_idm139979334567088"><h3 class="no_margin">Marie-Claude Vohl</h3><p><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</p><p><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</p><div>Find articles by <a href="/pubmed/?term=Vohl%20MC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=24847101">Marie-Claude Vohl</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139979258263424_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139979258263424_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139979258263424_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139979258263424_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>*</sup>Institute of Nutrition and Functional Foods and Laval University, Quebec, Canada</div><div class="fm-affl" id="aff2" lang="en"><sup>**</sup>Faculty of Pharmacy, Laval University, Quebec, Canada</div><div class="fm-affl" id="aff3" lang="en"><sup>†</sup>Endocrinology and Nephrology and CHU de Québec Research Center, Quebec, Canada</div><div class="fm-affl" id="aff4" lang="en"><sup>§</sup>Laboratory of Molecular Pharmacology, CHU de Québec Research Center, Quebec, Canada</div><div class="fm-affl" id="aff5" lang="en"><sup>††</sup>Human Development and Health Academic Unit, Faculty of Medicine, University of Southampton, Southampton, United Kingdom</div><div class="fm-affl" id="aff6" lang="en"><sup>§§</sup>NIHR Southampton Biomedical Research Centre, University Hospital Southampton NHS Foundation Trust and University of Southampton, Southampton, United Kingdom</div><div class="fm-affl" id="aff7" lang="en"><sup>***</sup>Department of Biological Sciences, Faculty of Science, King Abdulaziz University, Jeddah, Saudi Arabia,  Department of Nutrition</div><div class="fm-affl" id="aff8" lang="en"><sup>†††</sup> Norwich Medical School, University of East Anglia, Norwich, United Kingdom</div><div id="fn1"><sup>1</sup>I. Rudkowska and F. Guénard contributed equally to this work.</div><div id="cor1"><sup>2</sup>To whom correspondence should be addressed. e-mail: <a href="mailto:dev@null" data-email="ac.lavalu.aasf@lhov.edualc-eiram" class="oemail">ac.lavalu.aasf@lhov.edualc-eiram</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139979258263424_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2013 Nov 20; Revised 2014 Apr 16</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139979258263424_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  © 2014 by the American Society for Biochemistry and Molecular Biology, Inc.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4076081/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplemental Data
</div>

<div><a href="/pmc/articles/PMC4076081/bin/supp_55_7_1245__index.html" data-ga-action="click_feat_suppl">supp_55_7_1245__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;EB0E28CB-754D-48CD-84FC-209EEC673C59</div>
<div><a href="/pmc/articles/PMC4076081/bin/supp_M045898_jlr.M045898-1.xlsx" data-ga-action="click_feat_suppl">supp_M045898_jlr.M045898-1.xlsx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID:&nbsp;23929E63-4157-4FF2-8ADF-5FC0DE7DE921</div>
</div></dd></dl></div><div id="idm139979298995024" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139979298995024title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Studies have shown a large interindividual variability in plasma TG response to long-chain n-3 PUFA supplementation, which may likely be attributable to genetic variability within the populations studied. The objective is to compare the frequency of SNPs in a genome-wide association study between responders (reduction in plasma TG levels ≥0.01 mM) and nonresponders (increase in plasma TG of ≥0 mM) to supplementation. Genomic DNA from 141 subjects who completed a 2-week run-in period followed by 6-week supplementation with 5 g of fish oil daily (1.9–2.2 g EPA and 1.1 g DHA daily) were genotyped on Illumina HumanOmni-5-QuadBeadChip. Thirteen loci had frequency differences between responders and nonresponders (<em>P</em> &lt; 1 × 10<sup>−5</sup>), including SNPs in or near <em>IQCJ-SCHIP1</em>, <em>MYB</em>,<em> NELL1</em>, <em>NXPH1</em>, <em>PHF17</em>, and <em>SLIT2</em> genes. A genetic risk score (GRS) was constructed by summing the number of risk alleles. This GRS explained 21.53% of the variation in TG response to n-3 PUFA supplementation when adjusted for age, sex, and BMI (<em>P</em> = 0.0002). Using Fish Oil Intervention and Genotype as a replication cohort, the GRS was able to explain 2% of variation in TG response when adjusted. In conclusion, subjects who decrease their plasma TG levels following n-3 PUFA supplementation may have a different genetic profile than individuals who do not respond.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">nutrigenetics, single nucleotide polymorphism, responders, fish oil, genetic risk score</span></div></div><div id="idm139979264232960" class="tsec sec headless whole_rhythm"><p id="__p3" class="p p-first">Numerous meta-analyses have demonstrated that the long-chain n-3 PUFAs (LC n-3 PUFAs; EPA and DHA) significantly reduce plasma TG levels in a dose-dependent manner, with the TG lowering being proportional to baseline levels (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943989">1</a>, <a href="#bib2" rid="bib2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943957">2</a>). In trials of subjects with high TG levels, n-3 PUFAs in dosages of 3.4–4.0 g/day decreased TG levels by 16% to 45% (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943996">1</a>). However, there is well-recognized heterogeneity in the plasma TG response to LC n-3 PUFA supplementation (<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943965">3</a>): for example, 31% of all volunteers from the Fish Oil Intervention and Genotype (FINGEN) Study showed no reduction in TG after 1.8 g EPA and DHA per day for 8 weeks (<a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943983">4</a>). Similarly, our research group has reported that 29% of all subjects showed no reduction in plasma TG after a 6-week supplementation with 5 g of fish oil daily (1.9–2.2 g EPA and 1.1 g DHA daily) (<a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943980">5</a>, <a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943973">6</a>).</p><p id="__p4">This large heterogeneity in the TG response is likely to be partly attributable to genetic variability within the study populations. Studies have examined whether the <em>APOE</em> genotype contributes to the lipid response to n-3 PUFAs; these studies have yielded inconsistent results (<a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943997">4</a>, <a href="#bib7" rid="bib7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943971">7</a>–<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943961">9</a>). Further, research has demonstrated that the effect of the <em>PPAR α</em> L162V polymorphism (rs1800206) on plasma TG and apoC-III concentrations depends on dietary PUFAs, with a high intake triggering lower TG in carriers of the V162 allele (<a href="#bib10" rid="bib10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943962">10</a>). Likewise, Lindi et al. (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943986">11</a>), investigated the influence of the Pro12Ala (rs1801282) polymorphism of the <em>PPAR γ</em> gene: carriers of the Ala12 allele presented a greater decrease in plasma TG concentrations in response to LC n-3 PUFA supplementation than did homozygotes for Pro12 when the total dietary fat or saturated fat intake was low (<a href="#bib11" rid="bib11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943976">11</a>). Recently, studies have also supported the notion that the fatty acid desaturase 1 and 2 (<em>FADS1</em> and <em>2</em>) gene cluster is a major determinant of plasma TG levels (<a href="#bib5" rid="bib5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943958">5</a>). Thus, these genetic variations may influence the response to LC n-3 PUFA supplementation; however, other unknown genetic variations may also exist.</p><p id="__p5" class="p">Genome-wide association studies (GWASs) provide a more comprehensive approach unconstrained by existing knowledge to test common genetic variants across the genome. Therefore, a GWAS approach is ideal to identify SNPs that have not been discovered previously. The objective of this study was to compare the frequency of alleles in individuals with and without a decrease in plasma TG levels after LC n-3 PUFA supplementation, in order to determine whether an association exists between the responsiveness to LC n-3 PUFA supplementation and specific SNPs. In addition, a genetic risk model was developed with the specific SNPs identified to be able to predict the response to LC n-3 PUFA supplementation. To validate the results, we genotyped the DNA of participants in the FINGEN Study (<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943995">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943981">4</a>) to demonstrate whether the genetic risk score (GRS) may explain, at least partly, the variability of the TG-lowering effect of LC n-3 PUFA supplementation.</p></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">SUBJECTS AND METHODS</h2><div id="__sec2" class="sec sec-first"><h3 id="__sec2title">Study population</h3><p id="__p6" class="p p-first">A total of 254 subjects from the greater Quebec City metropolitan area were recruited to participate in the study. In total, 210 subjects completed the intervention protocol. Further, subjects who completed the study were separated into subgroups: responders and nonresponders. Responders are defined as having reduction in plasma TG (change in plasma TG levels ≥0.01 mM; TG postsupplementation with n-3 PUFAs minus TG presupplementation with n-3 PUFAs), and nonresponders are defined by no reduction in plasma TG concentrations after the LC n-3 PUFA supplementation (change in plasma TG of ≥0 mM; TG postsupplementation with n-3 PUFAs minus TG presupplementation with n-3 PUFAs). From this classification, a total of 141 subjects with the most extreme response to LC n-3 PUFA supplementation were selected including 81 responders and 60 nonresponders.</p><p id="__p7" class="p p-last">Participants had a BMI between 25 and 40 kg/m<sup>2</sup> and were not currently taking any lipid-lowering medications. Subjects were excluded from the study if they had taken LC n-3 PUFA supplements for at least 6 months prior, used oral hypolipidemic therapy, or had been diagnosed with diabetes, hypertension, hypothyroidism, or other known metabolic disorders such as severe dyslipidemia or coronary heart disease. The experimental protocol was approved by the ethics committees of Centre Hospitalier Universitaire de Québec Research Center and Laval University. This trial was registered at ClinicalTrials.gov as <a id="__tag_562264361" class="tag_hotlink" href="https://clinicaltrials.gov/ct2/show/NCT01343342" ref="reftype=extlink-clinical-trial&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CBody&amp;TO=External%7CResource%7CClinical%20Trials%7CClinicalTrials.gov"><span class="tag-json" style="display:none">{"type":"clinical-trial","attrs":{"text":"NCT01343342","term_id":"NCT01343342"}}</span>NCT01343342</a>.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Study design and diets</h3><p id="__p8" class="p p-first">First, subjects followed a run-in period of 2 weeks. Individual dietary instructions were given by a trained dietician to achieve the recommendation from <em>Canada’s Food Guide</em> (<a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/index-eng.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/index-eng.php</a>). Subjects were asked to follow these dietary recommendations and maintain a stable body weight throughout the study period. Some specifications were given regarding LC n-3 PUFA dietary intake: not to exceed two fish or seafood servings per week (maximum 150 g), preferably white flesh fish instead of fatty fish (examples were given), and to avoid enriched LC n-3 PUFA dietary products such as some milks, juices, breads, and eggs. Subjects were also asked to limit their alcohol consumption during the protocol: two regular drinks per week (∼28 g of alcohol per week) were allowed. In addition, subjects were not allowed to take n-3 PUFA supplements (such as flaxseed or fish oils), vitamins, or natural health products during the study period.</p><p id="__p9" class="p p-last">Second, after the run-in period, each participant received a bottle containing the fish oil capsules to be consumed in the following 6 weeks. They were asked to take five (each 1 g of fish oil concentrate) capsules per day (Ocean Nutrition, Nova Scotia, Canada), providing a total of 3 g of LC n-3 PUFAs (including 1.9–2.2 g EPA and 1.1 g DHA) per day. To aid digestion, we recommended taking the fish oil capsules while eating. Compliance was assessed from the counting of returned capsules. Subjects were asked to report any deviation from the protocol, to write down their alcohol and fish consumption, and to record any side effects. Before each phase, subjects received detailed written and oral instructions on their diet.</p></div><div id="__sec4" class="sec sec-last"><h3 id="__sec4title">Laboratory methods</h3><div id="__sec5" class="sec sec-first"><p></p><h4 id="__sec5title" class="inline">Plasma lipids. </h4><p id="__p10" class="p p-first-last">Blood samples were collected from an antecubital vein into vacutainer tubes containing EDTA after a 12 h overnight fast and 48 h alcohol abstinence. Blood samples were taken to identify and exclude individuals with any metabolic disorders. Afterward, selected participants had blood samples taken prior to and after the LC n-3 PUFA supplementation period. Plasma was separated by centrifugation (2,500 <em>g</em> for 10 min at 4°C), and samples were aliquoted and frozen for subsequent analyses. Plasma total cholesterol (TC) and TG concentrations were measured using enzymatic assays (<a href="#bib12" rid="bib12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385944000">12</a>). The HDL cholesterol (HDL-C) fraction was obtained after precipitation of VLDL and LDL particles in the infranatant with heparin manganese chloride (<a href="#bib13" rid="bib13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943998">13</a>). LDL cholesterol was calculated with the Friedewald formula (<a href="#bib14" rid="bib14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943992">14</a>). Fasting insulin concentrations were measured by radioimmunoassay with polyethylene glycol separation (<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943964">15</a>). Fasting glucose concentrations were enzymatically measured (<a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943979">16</a>). ApoB100 concentrations were measured in plasma by the rocket immunoelectrophoretic method of Laurell (<a href="#bib17" rid="bib17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943970">17</a>), as previously described. Plasma C-reactive protein (CRP) concentration was measured by nephelometry (Prospec equipment, Behring) using a sensitive assay, as described previously (<a href="#bib18" rid="bib18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943975">18</a>).</p></div><div id="__sec6" class="sec"><p></p><h4 id="__sec6title" class="inline">FA composition of plasma phospholipids. </h4><p id="__p11" class="p p-first-last">Plasma lipids were extracted with chloroform-methanol (2:1, by volume) according to a modified Folch method (<a href="#bib19" rid="bib19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943974">19</a>). Total plasma phospholipids (PLs) were separated by thin layer chromatography using a combination of isopropyl ether and acetic acid, and FAs of isolated PLs were then methylated. Capillary gas chromatography was then used to obtain FA profiles. The technique used for plasma analyses has been previously validated (<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943991">20</a>). Values of FA concentrations are expressed as percent of total FA in plasma PL.</p></div><div id="__sec7" class="sec"><p></p><h4 id="__sec7title" class="inline">DNA extraction and genotyping. </h4><p id="__p12" class="p p-first">The GenElute Gel Extraction Kit (Sigma-Aldrich Co., St. Louis, MO) was used to extract genomic DNA (gDNA) from whole blood. After spectrophotometric quantification and verification of DNA quality via an Agilent 2100 Bioanalyser (Agilent Technologies, Palo Alto, CA), samples (n = 141) were sent for genotyping. This was performed at the McGill University and Génome Québec Innovation Centre (Montreal, Canada) using Illumina HumanOmni-5-Quad BeadChip (Illumina, San Diego, CA), according to the manufacturer’s instructions. Each HumanOmni-5-Quad BeadChip contained 4,301,331 markers.</p><p id="__p13" class="p p-last">Sample quality was assessed by signal intensity using R and overall call rate using PLINK (version 1.07; http://pngu.mgh.harvard.edu/purcell/plink/). From the 141 samples analyzed, no subjects were excluded due to low signal intensity exhibited across all chromosomes. Furthermore, no samples were excluded due to overall call rate lower than 90% (default value in PLINK), with subjects having mean call rates of 99.75%. Therefore, all 141 subjects met eligibility criteria for further analysis.</p></div><div id="__sec8" class="sec"><p></p><h4 id="__sec8title" class="inline">SNP analysis. </h4><p id="__p14" class="p p-first">Calculations of allele frequencies and tests of SNP data for Hardy-Weinberg equilibrium (HWE) were performed using PLINK software. The quality control inclusion criteria for the SNPs were the following: minor allele frequency &gt;1%, SNP call rate &gt;95%, and HWE <em>P</em> &gt; 1.87 × 10<sup>−8</sup>. A total of 1,632,526 SNPs were excluded, leaving 2,668,805 SNPs for statistical analyses.</p><p id="__p15" class="p p-last">For each of these SNPs, we investigated whether the allele frequency was significantly different between nonresponders and responders. The odds ratio reports the ratio between two proportions, which in the context of GWAS is the proportion of individuals in the nonresponders group having a specific allele and the proportion of individuals in the responders group having the same allele. Additionally, a <em>P</em> value for the significance of the odds ratio was calculated using a Chi-square test. After quality control and exclusion of SNPs, the calculated threshold for statistical significance was <em>P</em> &lt; 1.87 × 10<sup>−8</sup> (as calculated with 0.05/2,668,805 SNPs). One critical, but often overlooked, assumption is that all such tests are independent. However, SNPs in close proximity are not independent, and therefore the traditional method to adjust significance thresholds for multiple testing overcorrects when used in GWASs (<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943969">21</a>). Thus, a statistically suggestive <em>P</em> value was defined as <em>P</em> &lt; 10<sup>−5</sup> (<a href="#bib22" rid="bib22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943963">22</a>) and was used in order to avoid discounting true positive associations. ANOVA was used to test for the effects of each of the revealed SNPs on metabolic variables when adjusted for age, sex, and BMI.</p></div><div id="__sec9" class="sec"><p></p><h4 id="__sec9title" class="inline">Identifications of genes. </h4><p id="__p16" class="p p-first-last">A list of suggestive SNPs was imported into the Ingenuity Pathway Analysis (IPA) system (Ingenuity® Systems; <a href="http://www.ingenuity.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ingenuity.com</a>). IPA allowed determination of whether these SNPs were associated with known genes and their functions and therefore allowed addition of structure to the vast amount of data generated by GWAS.</p></div><div id="__sec10" class="sec"><p></p><h4 id="__sec10title" class="inline">Replication study. </h4><p id="__p17" class="p p-first-last">The FINGEN Study is a four-center trial that was conducted at the Universities of Glasgow, Newcastle, Reading, and Southampton in the United Kingdom between June 2003 and Sept 2005. The study was a double-blind, placebo-controlled, dose-response crossover study consisting of three intervention arms each of 8 weeks duration and separated by 12 week washout intervals. During the intervention periods, participants consumed either <em>A</em>) the control, <em>B</em>) 0.7 g EPA+DHA/day, or <em>C</em>) 1.8 g EPA+DHA/day in random order. The study subjects and recruitment are described in full detail elsewhere (<a href="#bib3" rid="bib3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943956">3</a>, <a href="#bib4" rid="bib4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943999">4</a>). In the context of the current study, we used the TG results from 310 subjects who completed the 1.8 g EPA+DHA/day intervention phase to classify individuals as responders and nonresponders. In the same matter as the original study, responders are defined as demonstrating a reduction in plasma TG (change in plasma TG levels ≥0.01 mM; TG postsupplementation with n-3 PUFAs minus TG presupplementation with n-3 PUFAs), and nonresponders are defined by no reduction in plasma TG concentrations after the LC n-3 PUFA supplementation (change in plasma TG of ≥0 mM; TG postsupplementation with n-3 PUFAs minus TG presupplementation with n-3 PUFAs).</p></div><div id="__sec11" class="sec"><p></p><h4 id="__sec11title" class="inline">Genotyping in the replication study. </h4><p id="__p18" class="p p-first-last">The 10 SNPs revealed by the GWAS were run on the QuantStudio™12K Flex Real-Time PCR System (Life Technologies, Carlsbad, CA). A total of 310 DNA samples were normalized to 20 ng/uL. Two microliters of TaqMan OpenArray Genotyper Master Mix (Life Technologies) and 2 μl of each gDNA sample were premixed in a 384-well plate and loaded onto the genotyping plates using the QuantStudio™ 12K Flex OpenArray® AccuFill™ System. The results were analyzed in TaqMan Genotyper v1.3 (Life Technologies).</p></div><div id="__sec12" class="sec sec-last"><p></p><h4 id="__sec12title" class="inline">Statistical analyses. </h4><p id="__p19" class="p p-first">Results are presented as mean ± SD. Variables not normally distributed were transformed by natural logarithm to normalize their distribution. Data were analyzed using a paired <em>t</em>-test to determine significant changes between pre- and postsupplementation periods for each of the subgroups: responders and nonresponders. An unpaired <em>t</em>-test was used to establish differences between predata between the subgroups.</p><p id="__p20" class="p p-last">A genetic risk model was derived for each subject from the sum of risk alleles. GRS was computed with SNPs that are uncorrelated [linkage disequilibrium (LD) &lt;80%] and in turn contributed to the effect of LC n-3 PUFAs on plasma TG concentration in an additive way. Based on this genetic risk model, subjects were dichotomized with the median GRS. Sensitivity and specificity of the genetic risk model to classify individuals into responder and nonresponder categories were calculated. The general linear model was used to examine the effect of the GRS on TG response to LC n-3 PUFAs taking into account the confounding effects of age, sex, and BMI. Statistical analyses were performed with SAS statistical software, version 9.2 (SAS Institute Inc, Cary, NC) and statistical significance defined as <em>P</em> ≤ 0.05.</p></div></div></div><div id="__sec13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">RESULTS</h2><div id="__sec14" class="sec sec-first"><h3 id="__sec14title">Subjects’ characteristics</h3><p id="__p21" class="p p-first">Subjects with extreme responses were used to determine whether SNPs were of different frequency between the 81 responders [including 38 men (47%) and 43 women (53%)] and 60 nonresponders [including 30 men (50%) and 30 women (50%)]. <a href="/pmc/articles/PMC4076081/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span><strong>Table 1</strong></span></a> describes the characteristics of the responders and nonresponders presupplementation. The average population age was 31 years, and the subjects were slightly overweight with a mean BMI of 28.4 kg/m<sup>2</sup>. The characteristics of responder and nonresponder subgroups are comparable to those previously reported (<a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943985">6</a>). The presupplementation characteristics of nonresponders were similar to those of responders, except that higher plasma TG (<em>P</em> &lt; 0.0001), glucose (<em>P</em> = 0.0144), insulin (<em>P</em> = 0.0199), and TC/HDL-C ratio (<em>P</em> = 0.0018) together with lower HDL-C levels (<em>P</em> = 0.0142) were observed in responders. <a href="/pmc/articles/PMC4076081/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span><strong>Table 2</strong></span></a> demonstrates the response of each group to LC n-3 PUFA supplementation. Clearly, the plasma TG decreased in responders (<em>P</em> &lt; 0.0001) and increased in nonresponders (<em>P</em> &lt; 0.0001). Further, responders showed decreased plasma TC (<em>P</em> = 0.0234) and increased plasma HDL-C levels (<em>P</em> &lt; 0.0001), as well as decreased TC/HDL-C ratio (<em>P</em> &lt; 0.0001) after LC n-3 PUFA supplementation. In contrast, nonresponders showed increased TC/HDL-C ratio (<em>P</em> = 0.0054) and apoB levels (<em>P</em> = 0.0015). Fasting glucose levels tended to increase in both the responders (<em>P</em> = 0.0629) and nonresponders (<em>P</em> = 0.0576). Plasma insulin levels decreased in responders (<em>P</em> = 0.0177) and increased in nonresponders (<em>P</em> = 0.0250).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>TABLE 1.</h3><!--caption a7--><div class="caption"><p id="__p22">Presupplementation subject characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979260766736" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Characteristics</td><td rowspan="1" colspan="1" align="center">Responders (n = 81)</td><td rowspan="1" colspan="1" align="center">Nonresponders (n = 60)</td><td rowspan="1" colspan="1" align="center"><em>P</em></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Sex</td><td rowspan="1" colspan="1" align="center">38 men (47%) and 43 women (53%)</td><td rowspan="1" colspan="1" align="center">30 men (50%) and 30 women (50%)</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1" align="center">31.9 ± 8.78</td><td rowspan="1" colspan="1" align="center">31.1 ± 8.98</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">28.9 ± 3.64</td><td rowspan="1" colspan="1" align="center">27.8 ± 3.91</td><td rowspan="1" colspan="1" align="center">0.0612</td></tr><tr><td rowspan="1" colspan="1">TGs<sup><em><sup>a</sup></em></sup> (mM)</td><td rowspan="1" colspan="1" align="center">1.53 ± 0.74</td><td rowspan="1" colspan="1" align="center">1.03 ± 0.48</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Cholesterol (mM)</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1"> Total</td><td rowspan="1" colspan="1" align="center">5.00 ± 0.93</td><td rowspan="1" colspan="1" align="center">4.77 ± 1.01</td><td rowspan="1" colspan="1" align="center">0.2619</td></tr><tr><td rowspan="1" colspan="1"> LDL</td><td rowspan="1" colspan="1" align="center">2.94 ± 0.85</td><td rowspan="1" colspan="1" align="center">2.79 ± 0.91</td><td rowspan="1" colspan="1" align="center">0.5598</td></tr><tr><td rowspan="1" colspan="1"> HDL<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">1.36 ± 0.37</td><td rowspan="1" colspan="1" align="center">1.50 ± 0.39</td><td rowspan="1" colspan="1" align="center">0.0142</td></tr><tr><td rowspan="1" colspan="1">Ratio TC/HDL-C</td><td rowspan="1" colspan="1" align="center">3.91 ± 1.15</td><td rowspan="1" colspan="1" align="center">3.33 ± 0.95</td><td rowspan="1" colspan="1" align="center">0.0018</td></tr><tr><td rowspan="1" colspan="1">ApoB (g/l)</td><td rowspan="1" colspan="1" align="center">0.90 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.83 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.1687</td></tr><tr><td rowspan="1" colspan="1">Ratio LDL/apoB</td><td rowspan="1" colspan="1" align="center">3.28 ± 0.46</td><td rowspan="1" colspan="1" align="center">3.36 ± 0.43</td><td rowspan="1" colspan="1" align="center">0.3479</td></tr><tr><td rowspan="1" colspan="1">Glucose (mM)<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">5.05 ± 0.50</td><td rowspan="1" colspan="1" align="center">4.94 ± 0.39</td><td rowspan="1" colspan="1" align="center">0.0144</td></tr><tr><td rowspan="1" colspan="1">Insulin (pM)<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">102 ± 110</td><td rowspan="1" colspan="1" align="center">80.2 ± 36.8</td><td rowspan="1" colspan="1" align="center">0.0199</td></tr><tr><td rowspan="1" colspan="1">CRP<sup><em><sup>a</sup></em></sup> (mg/l)</td><td rowspan="1" colspan="1" align="center">2.89 ± 3.71</td><td rowspan="1" colspan="1" align="center">2.87 ± 5.20</td><td rowspan="1" colspan="1" align="center">0.3798</td></tr></tbody></table></div><div id="largeobj_idm139979260766736" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139979262574992"><p id="__p23" class="p p-first-last">Data are shown as mean ± SD.</p></div><div id="tblfn1"><sup>a</sup><em>P</em> value derived from transformed data.</div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>TABLE 2.</h3><!--caption a7--><div class="caption"><p id="__p25">Characteristics of responders and nonresponders pre- and postsupplementation with n-3 PUFAs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979264876928" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td colspan="4" rowspan="1" align="center">Responders (n = 81)</td><td colspan="4" rowspan="1" align="center">Nonresponders (n = 60)</td></tr><tr><td rowspan="1" colspan="1">Characteristics</td><td rowspan="1" colspan="1" align="center">Pre n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Post n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Change Pre to Post</td><td rowspan="1" colspan="1" align="center"><em>P</em> Pre to Post</td><td rowspan="1" colspan="1" align="center">Pre n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Post n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Change Pre to Post</td><td rowspan="1" colspan="1" align="center"><em>P</em> Pre to Post</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">28.9 ± 3.64</td><td rowspan="1" colspan="1" align="center">28.9 ± 3.79</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.7006</td><td rowspan="1" colspan="1" align="center">27.8 ± 3.91</td><td rowspan="1" colspan="1" align="center">28.0 ± 4.01</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">0.0007</td></tr><tr><td rowspan="1" colspan="1">TGs<sup><em><sup>a</sup></em></sup>  (mM)</td><td rowspan="1" colspan="1" align="center">1.53 ± 0.74</td><td rowspan="1" colspan="1" align="center">1.03 ± 0.56</td><td rowspan="1" colspan="1" align="center">−0.50 ± 0.36</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">1.03 ± 0.48</td><td rowspan="1" colspan="1" align="center">1.20 ± 0.54</td><td rowspan="1" colspan="1" align="center">0.17 ± 0.17</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Cholesterol (mM)</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1"> Total</td><td rowspan="1" colspan="1" align="center">5.00 ± 0.93</td><td rowspan="1" colspan="1" align="center">4.85 ± 1.00</td><td rowspan="1" colspan="1" align="center">−0.14 ± 0.56</td><td rowspan="1" colspan="1" align="center">0.0234</td><td rowspan="1" colspan="1" align="center">4.77 ± 1.01</td><td rowspan="1" colspan="1" align="center">4.87 ± 1.07</td><td rowspan="1" colspan="1" align="center">0.10 ± 0.53</td><td rowspan="1" colspan="1" align="center">0.1196</td></tr><tr><td rowspan="1" colspan="1"> LDL</td><td rowspan="1" colspan="1" align="center">2.94 ± 0.85</td><td rowspan="1" colspan="1" align="center">2.93 ± 0.91</td><td rowspan="1" colspan="1" align="center">−0.01 ± 0.51</td><td rowspan="1" colspan="1" align="center">0.9603</td><td rowspan="1" colspan="1" align="center">2.79 ± 0.91</td><td rowspan="1" colspan="1" align="center">2.84 ± 0.97</td><td rowspan="1" colspan="1" align="center">0.05 ± 0.45</td><td rowspan="1" colspan="1" align="center">0.4044</td></tr><tr><td rowspan="1" colspan="1"> HDL<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">1.36 ± 0.37</td><td rowspan="1" colspan="1" align="center">1.45 ± 0.43</td><td rowspan="1" colspan="1" align="center">0.09 ± 0.18</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">1.50 ± 0.39</td><td rowspan="1" colspan="1" align="center">1.48 ± 0.41</td><td rowspan="1" colspan="1" align="center">−0.02 ± 0.17</td><td rowspan="1" colspan="1" align="center">0.3467</td></tr><tr><td rowspan="1" colspan="1">Ratio TC/HDL-C</td><td rowspan="1" colspan="1" align="center">3.91 ± 1.15</td><td rowspan="1" colspan="1" align="center">3.62 ± 1.22</td><td rowspan="1" colspan="1" align="center">−0.29 ± 0.44</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">3.33 ± 0.95</td><td rowspan="1" colspan="1" align="center">3.48 ± 1.05</td><td rowspan="1" colspan="1" align="center">0.15 ± 0.39</td><td rowspan="1" colspan="1" align="center">0.0054</td></tr><tr><td rowspan="1" colspan="1">ApoB (g/l)</td><td rowspan="1" colspan="1" align="center">0.90 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.91 ± 0.24</td><td rowspan="1" colspan="1" align="center">0.01 ± 0.13</td><td rowspan="1" colspan="1" align="center">0.8054</td><td rowspan="1" colspan="1" align="center">0.83 ± 0.26</td><td rowspan="1" colspan="1" align="center">0.88 ± 0.25</td><td rowspan="1" colspan="1" align="center">0.05 ± 0.12</td><td rowspan="1" colspan="1" align="center">0.0015</td></tr><tr><td rowspan="1" colspan="1">Ratio LDL/apoB</td><td rowspan="1" colspan="1" align="center">3.28 ± 0.46</td><td rowspan="1" colspan="1" align="center">3.22 ± 0.51</td><td rowspan="1" colspan="1" align="center">−0.06 ± 0.54</td><td rowspan="1" colspan="1" align="center">0.3254</td><td rowspan="1" colspan="1" align="center">3.36 ± 0.43</td><td rowspan="1" colspan="1" align="center">3.20 ± 0.51</td><td rowspan="1" colspan="1" align="center">−0.14 ± 0.42</td><td rowspan="1" colspan="1" align="center">0.0098</td></tr><tr><td rowspan="1" colspan="1">Glucose (mM)<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">5.05 ± 0.50</td><td rowspan="1" colspan="1" align="center">5.13 ± 0.45</td><td rowspan="1" colspan="1" align="center">0.08 ± 0.41</td><td rowspan="1" colspan="1" align="center">0.0629</td><td rowspan="1" colspan="1" align="center">4.94 ± 0.39</td><td rowspan="1" colspan="1" align="center">5.04 ± 0.52</td><td rowspan="1" colspan="1" align="center">0.10 ± 0.41</td><td rowspan="1" colspan="1" align="center">0.0576</td></tr><tr><td rowspan="1" colspan="1">Insulin (pM)<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">102 ± 110</td><td rowspan="1" colspan="1" align="center">85.8 ± 43.1</td><td rowspan="1" colspan="1" align="center">−16.2 ± 88.2</td><td rowspan="1" colspan="1" align="center">0.0177</td><td rowspan="1" colspan="1" align="center">80.2 ± 36.8</td><td rowspan="1" colspan="1" align="center">89.7 ± 44.6</td><td rowspan="1" colspan="1" align="center">9.5 ± 27.7</td><td rowspan="1" colspan="1" align="center">0.0250</td></tr><tr><td rowspan="1" colspan="1">CRP<sup><em><sup>a</sup></em></sup> (mg/l)</td><td rowspan="1" colspan="1" align="center">2.89 ± 3.71</td><td rowspan="1" colspan="1" align="center">2.64 ± 3.17</td><td rowspan="1" colspan="1" align="center">−0.25 ± 2.46</td><td rowspan="1" colspan="1" align="center">0.6162</td><td rowspan="1" colspan="1" align="center">2.87 ± 5.20</td><td rowspan="1" colspan="1" align="center">3.06 ± 6.43</td><td rowspan="1" colspan="1" align="center">0.16 ± 5.45</td><td rowspan="1" colspan="1" align="center">0.5972</td></tr></tbody></table></div><div id="largeobj_idm139979264876928" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4076081/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139979259055216"><p id="__p26" class="p p-first-last">Data are shown as mean ± SD.</p></div><div id="tblfn2"><sup>a</sup><em>P</em> value derived from transformed data.</div></div></div><p id="__p28" class="p">There were no differences in FA composition of plasma PL between the responders and the nonresponders presupplementation with n-3 PUFAs. As expected, the 6 weeks of LC n-3 PUFA supplementation increased the concentration of total n-3 PUFAs and decreased total n-6 PUFAs in PLs in both subgroups (<a href="/pmc/articles/PMC4076081/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span><strong>Table 3</strong></span></a>). In brief, the LC n-3 PUFA supplementation was associated with a similar increase in total n-3 PUFAs in both subgroups, from 5.95% to 10.48% of total FA for responders and from 5.86% to 10.65% of total FA for nonresponders. Similar increases were also observed for percentages of EPA and DHA and the ratio of total n-3/n-6 for the responders and nonresponders.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>TABLE 3.</h3><!--caption a7--><div class="caption"><p id="__p29">PL FA profiles of responders and nonresponders pre- and postsupplementation with n-3 PUFAs</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979259048944" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td colspan="3" rowspan="1" align="center">Responders (n = 81)</td><td colspan="3" rowspan="1" align="center">Nonresponders (n = 60)</td><td rowspan="1" colspan="1" align="center"></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">(% of FA)</td><td rowspan="1" colspan="1" align="center">Pre n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Post n-3 PUFAs</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center">Pre n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">Post n-3 PUFAs</td><td rowspan="1" colspan="1" align="center"><em>P</em></td><td rowspan="1" colspan="1" align="center"><em>P</em> between Responders and Nonresponders</td></tr><tr><td rowspan="1" colspan="1">EPA (20:5n-3)</td><td rowspan="1" colspan="1" align="center">1.15 ± 0.46</td><td rowspan="1" colspan="1" align="center">3.91 ± 1.21</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">1.12 ± 0.59</td><td rowspan="1" colspan="1" align="center">3.94 ± 1.32</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.6442</td></tr><tr><td rowspan="1" colspan="1">DHA (22:6n-3)</td><td rowspan="1" colspan="1" align="center">3.50 ± 0.78</td><td rowspan="1" colspan="1" align="center">5.02 ± 0.83</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">3.54 ± 0.83</td><td rowspan="1" colspan="1" align="center">5.07 ± 1.06</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.9362</td></tr><tr><td rowspan="1" colspan="1">Total n-6 PUFAs</td><td rowspan="1" colspan="1" align="center">34.80 ± 1.86</td><td rowspan="1" colspan="1" align="center">30.68 ± 2.56</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">35.06 ± 1.56</td><td rowspan="1" colspan="1" align="center">30.13 ± 3.04</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.1145</td></tr><tr><td rowspan="1" colspan="1">Total n-3 PUFAs</td><td rowspan="1" colspan="1" align="center">5.95 ± 1.06</td><td rowspan="1" colspan="1" align="center">10.48 ± 1.83</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">5.86 ± 1.25</td><td rowspan="1" colspan="1" align="center">10.65 ± 2.31</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.4364</td></tr><tr><td rowspan="1" colspan="1">Ratio total n-3/total n-6</td><td rowspan="1" colspan="1" align="center">0.17 ± 0.04</td><td rowspan="1" colspan="1" align="center">0.35 ± 0.09</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.17 ± 0.04</td><td rowspan="1" colspan="1" align="center">0.36 ± 0.10</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td><td rowspan="1" colspan="1" align="center">0.3156</td></tr></tbody></table></div><div id="largeobj_idm139979259048944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139979262905136"><p id="__p30" class="p p-first-last">Data are shown as mean ± SD; supplementation is the effect of n-3 PUFA supplementation on parameters.</p></div></div></div></div><div id="__sec15" class="sec"><h3 id="__sec15title">GWAS</h3><p id="__p31" class="p p-first"><a href="/pmc/articles/PMC4076081/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span><strong>Figure 1</strong></span></a> demonstrates the GWAS Manhattan plot showing the association of the TG response to LC n-3 PUFA supplementation. Thirteen loci showed <em>P</em> values less than the suggestive threshold (<em>P</em> &lt; 1 × 10<sup>−5</sup>) as described in detail in <a href="/pmc/articles/PMC4076081/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span><strong>Table 4</strong></span></a>. Suggestive SNPs included rs2952724/rs2629715 intergenic SNPs upstream of the slit homolog 2 (<em>Drosophila</em>) (<em>SLIT2</em>) gene, rs1216352/rs1216365/rs931681 intergenic upstream of the PHD finger protein 17 (<em>PHF17</em>) gene, rs2621308/rs1449009<sup>/</sup>rs61332355/rs2621309 intron variants of IQ motif containing J-schwannomin interacting protein 1 fusion protein (<em>IQCJ-SCHIP1</em>) gene, rs6463808 intron variant of neurexophilin-1 (<em>NXPH1</em>), rs752088 intron variant of NEL-like 1 (chicken) (<em>NELL1</em>) gene, and rs6920829 intron variant of V-myb myeloblastosis viral oncogene homolog (<em>MYB</em>) gene. Additional analysis showed that individually most of the SNPs were associated with the change in TG levels as well as with other metabolic variables when adjusted for age, sex, and BMI (see supplementary data).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4076081/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979260374928" class="figure"><a href="/pmc/articles/PMC4076081/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4076081_1245fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1245fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4076081/bin/1245fig1.jpg"></a></div><div id="largeobj_idm139979260374928" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4076081/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Fig. 1.</a></div><!--caption a7--><div class="caption"><p id="__p32">GWAS Manhattan plot showing differences in allele frequency between responders and nonresponders to n-3 PUFA supplementation. The odds ratios were calculated for each SNP in GWAS data. The figure demonstrates that 13 SNPs (depicted by dots) were of significantly different (<em>P</em> &lt; 10<sup>−5</sup> depicted by a line) frequency between responders and nonresponders to n-3 PUFA supplementation.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>TABLE 4.</h3><!--caption a7--><div class="caption"><p id="__p33">Significant differences in frequency of alleles of SNPs in individuals with and without a decrease in TG levels following LC n-3 PUFA supplementation</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979334153472" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center"></td><td colspan="2" rowspan="1" align="center">Location</td><td rowspan="1" colspan="1" align="center"></td><td colspan="2" rowspan="1" align="center">Frequency of Allele</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">SNP</td><td rowspan="1" colspan="1" align="center">CHR</td><td rowspan="1" colspan="1" align="center">Base pairs</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">Allele</td><td rowspan="1" colspan="1" align="center">Responders</td><td rowspan="1" colspan="1" align="center">Nonresponders</td><td rowspan="1" colspan="1" align="center"><em>P</em> between Responders and Nonresponders</td><td rowspan="1" colspan="1" align="center">Odds Ratio between Responders and Nonresponders</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs2621308<sup><em><sup>a</sup></em></sup> </td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">158,980,014</td><td rowspan="1" colspan="1" align="center">Intron variant <em>IQCJ-SCHIP1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.15</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center">3.21 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.65</td></tr><tr><td rowspan="1" colspan="1">rs1449009<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">158,977,883</td><td rowspan="1" colspan="1" align="center">Intron variant <em>IQCJ-SCHIP1</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.19</td><td rowspan="1" colspan="1" align="center">0.43</td><td rowspan="1" colspan="1" align="center">5.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.36</td></tr><tr><td rowspan="1" colspan="1">rs61332355</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">158,921,902</td><td rowspan="1" colspan="1" align="center">Intron variant <em>IQCJ-SCHIP1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">0.29</td><td rowspan="1" colspan="1" align="center">6.88 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">4.35</td></tr><tr><td rowspan="1" colspan="1">rs2621309</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">158,980,745</td><td rowspan="1" colspan="1" align="center">Intron variant <em>IQCJ-SCHIP1</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">0.43</td><td rowspan="1" colspan="1" align="center">7.90 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.34</td></tr><tr><td rowspan="1" colspan="1">rs2952724<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">20,024,979</td><td rowspan="1" colspan="1" align="center">Intergenic (300 kb upstream of <em>SLIT2</em>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">3.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.25</td></tr><tr><td rowspan="1" colspan="1"><sup><em><sup>a</sup></em></sup> </td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">20,010,995</td><td rowspan="1" colspan="1" align="center">Intergenic (300 kb upstream of <em>SLIT2</em>)</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">3.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.25</td></tr><tr><td rowspan="1" colspan="1">rs2629715<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">20,020,240</td><td rowspan="1" colspan="1" align="center">Intergenic (300 kb upstream of <em>SLIT2</em>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">0.18</td><td rowspan="1" colspan="1" align="center">3.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">0.25</td></tr><tr><td rowspan="1" colspan="1">rs1216352<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">129,559,799</td><td rowspan="1" colspan="1" align="center">Intergenic (300 kb upstream of <em>PHF17</em>)</td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">0.51</td><td rowspan="1" colspan="1" align="center">1.06 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.49</td></tr><tr><td rowspan="1" colspan="1">rs1216365</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">129,569,808</td><td rowspan="1" colspan="1" align="center">Intergenic (500 kb upstream of <em>PHF17</em>)</td><td rowspan="1" colspan="1" align="center">C</td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">0.53</td><td rowspan="1" colspan="1" align="center">5.67 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.16</td></tr><tr><td rowspan="1" colspan="1">rs931681<sup><em><sup>a</sup></em></sup></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">129,578,480</td><td rowspan="1" colspan="1" align="center">Intergenic (300 kb upstream of <em>PHF17</em>)</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">0.53</td><td rowspan="1" colspan="1" align="center">7.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.06</td></tr><tr><td rowspan="1" colspan="1">rs6920829</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">135,519,500</td><td rowspan="1" colspan="1" align="center">Intron variant <em>MYB</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.05</td><td rowspan="1" colspan="1" align="center">0.23</td><td rowspan="1" colspan="1" align="center">9.76 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">5.59</td></tr><tr><td rowspan="1" colspan="1">rs6463808</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">8,476,787</td><td rowspan="1" colspan="1" align="center">Intron variant <em>NXPH1</em></td><td rowspan="1" colspan="1" align="center">A</td><td rowspan="1" colspan="1" align="center">0.09</td><td rowspan="1" colspan="1" align="center">0.30</td><td rowspan="1" colspan="1" align="center">3.43 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">4.53</td></tr><tr><td rowspan="1" colspan="1">rs752088</td><td rowspan="1" colspan="1" align="center">11</td><td rowspan="1" colspan="1" align="center">20,781,004</td><td rowspan="1" colspan="1" align="center">Intron variant <em>NELL1</em></td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">3.73 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">3.23</td></tr></tbody></table></div><div id="largeobj_idm139979334153472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4076081/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn3"><sup>a</sup>The same subscript letter within the same gene determines an LD between these SNPs (100%) CHR, Chromosome.</div></div></div><p id="__p35" class="p">A GRS was computed using 10 out of the 13 SNPs showing suggestive associations because 3 SNPs were in LD ≥80% (<a href="/pmc/articles/PMC4076081/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a>). Based on this genetic risk model, subjects were categorized into two genetic risk groups using the median score: lower genetic risk (scores from −1 to 3) and higher genetic risk (scores from 4 to 8) (<a href="/pmc/articles/PMC4076081/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span><strong>Fig. 2</strong></span></a>). The sensitivity and specificity of the GRS to correctly identify nonresponders and responders were calculated as 83.95% [95% confidence interval (CI): 74.12% to 91.16%] and 68.33% (95% CI: 55.04% to 79.74%), respectively (<a href="/pmc/articles/PMC4076081/table/tbl5/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl5" rid-ob="ob-tbl5" co-legend-rid=""><span><strong>Table 5</strong></span></a>). In the general linear model, the GRS was significantly associated with TG response when adjusted for age, sex, and BMI (<em>P</em> = 0.0002, respectively). The GRS explained 21.53% of variation in TG during the LC n-3 PUFA supplementation protocol when adjusted for age, sex, and BMI (<em>P =</em> 0.0002).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4076081/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979263722944" class="figure"><a href="/pmc/articles/PMC4076081/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4076081_1245fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1245fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4076081/bin/1245fig2.jpg"></a></div><div id="largeobj_idm139979263722944" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4076081/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Fig. 2.</a></div><!--caption a7--><div class="caption"><p id="__p36">GRS distribution in study population (n = 141). If a GRS is positive, then the subject carries more at-risk alleles from the SNPs identified in the GWAS. If a GRS is negative, then the subject carries more beneficial alleles from the SNPs identified in the GWAS.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl5"><h3>TABLE 5.</h3><!--caption a7--><div class="caption"><p id="__p37">Sensitivity and specificity of GRS in study population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979263720368" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">Responders</td><td rowspan="1" colspan="1" align="center">Nonresponders</td><td rowspan="1" colspan="1" align="center">Total</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Low GRS</td><td rowspan="1" colspan="1" align="center">68</td><td rowspan="1" colspan="1" align="center">19</td><td rowspan="1" colspan="1" align="center">87</td></tr><tr><td rowspan="1" colspan="1">High GRS</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">41</td><td rowspan="1" colspan="1" align="center">54</td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1" align="center">81</td><td rowspan="1" colspan="1" align="center">60</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">Sensitivity</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">83.95% (95% CI: 74.12% to 91.16%)</td></tr><tr><td rowspan="1" colspan="1">Specificity</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">68.33% (95% CI: 55.04% to 79.74%)</td></tr></tbody></table></div><div id="largeobj_idm139979263720368" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/table/tbl5/?report=objectonly">Open in a separate window</a></div></div></div><div id="__sec16" class="sec"><h3 id="__sec16title">Pathway analysis results</h3><p id="__p38" class="p p-first-last">Using IPA, the top three pathways associated with the suggestive SNPs were <em>PPAR</em> signaling, tight junction signaling pathway, and ceramide signaling (data not shown).</p></div><div id="__sec17" class="sec sec-last"><h3 id="__sec17title">FINGEN replication cohort</h3><p id="__p39" class="p p-first">Briefly, subjects were identified as 188 responders [including 95 men (51%) and 93 women (49%)] and 122 nonresponders [including 53 men (43%) and 69 women (57%)]. <a href="/pmc/articles/PMC4076081/table/tbl6/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl6" rid-ob="ob-tbl6" co-legend-rid=""><span><strong>Table 6</strong></span></a> describes the presupplementation characteristics of the responders and nonresponders. Clearly, the plasma TG decreased in responders from 1.45 to 1.05 mM (change of −0.40 mM; <em>P</em> &lt; 0.0001) and increased in nonresponders from 1.02 to 1.26 mM (change of 0.24 mM; <em>P</em> &lt; 0.0001) after the daily intake of 1.8 g EPA+DHA.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl6"><h3>TABLE 6.</h3><!--caption a7--><div class="caption"><p id="__p40">Presupplementation subject characteristics in FINGEN replication population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979264489600" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Characteristics</td><td rowspan="1" colspan="1" align="center">Responders (n = 188)</td><td rowspan="1" colspan="1" align="center">Nonresponders (n = 122)</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">Sex</td><td rowspan="1" colspan="1" align="center">95 men (51%) and 93 women (49%)</td><td rowspan="1" colspan="1" align="center">53 men (43%) and 69 women (57%)</td><td rowspan="1" colspan="1" align="center"><em>P</em></td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1" align="center">46.2 ± 12.3</td><td rowspan="1" colspan="1" align="center">43.9 ± 13.9</td><td rowspan="1" colspan="1" align="center">0.0672</td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1" align="center">25.3 ± 3.27</td><td rowspan="1" colspan="1" align="center">25.0 ± 3.55</td><td rowspan="1" colspan="1" align="center">0.0669</td></tr><tr><td rowspan="1" colspan="1">TGs<em><sup>a</sup></em> (mM)</td><td rowspan="1" colspan="1" align="center">1.21 ± 0.57</td><td rowspan="1" colspan="1" align="center">1.15 ± 0.50</td><td rowspan="1" colspan="1" align="center">0.0052</td></tr><tr><td rowspan="1" colspan="1">Cholesterol (mM)</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1"> Total</td><td rowspan="1" colspan="1" align="center">5.16 ± 0.96</td><td rowspan="1" colspan="1" align="center">5.15 ± 1.05</td><td rowspan="1" colspan="1" align="center">0.2289</td></tr><tr><td rowspan="1" colspan="1"> HDL<em><sup>a</sup></em></td><td rowspan="1" colspan="1" align="center">1.40 ± 0.37</td><td rowspan="1" colspan="1" align="center">1.50 ± 0.37</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Ratio TC/HDL-C</td><td rowspan="1" colspan="1" align="center">3.96 ± 1.27</td><td rowspan="1" colspan="1" align="center">3.63 ± 1.14</td><td rowspan="1" colspan="1" align="center">&lt;0.0001</td></tr><tr><td rowspan="1" colspan="1">Glucose (mM)<em><sup>a</sup></em></td><td rowspan="1" colspan="1" align="center">4.98 ± 0.64</td><td rowspan="1" colspan="1" align="center">4.90 ± 0.67</td><td rowspan="1" colspan="1" align="center">0.1022</td></tr></tbody></table></div><div id="largeobj_idm139979264489600" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/table/tbl6/?report=objectonly">Open in a separate window</a></div></div><p id="__p41" class="p">Risk allele frequency and GRS were determined in the FINGEN population using the 10 newly discovered SNPs (<a href="/pmc/articles/PMC4076081/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span><strong>Fig. 3</strong></span></a>). No differences in allele frequencies were observed between the two populations (data not shown). Further, the subjects were categorized into two genetic risk groups using the median score of the genetic risk model developed: lower genetic risk (scores from −1 to 3) and higher genetic risk (scores from 4 to 9). The sensitivity and specificity of the GRS to correctly identify nonresponders and responders were calculated as 69.68% (95% CI: 62.57% to 76.16%) and 24.59% (95% CI: 17.25% to 33.21%), respectively (<a href="/pmc/articles/PMC4076081/table/tbl7/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl7" rid-ob="ob-tbl7" co-legend-rid=""><span><strong>Table 7</strong></span></a>). Although not significant, the GRS explained 2% of variation in TG response to LC n-3 PUFA supplementation in the replication cohort from the regression analyses when adjusted for age, sex, and BMI (<em>P</em> = not significant).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4076081/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979265100832" class="figure"><a href="/pmc/articles/PMC4076081/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4076081_1245fig3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 1245fig3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4076081/bin/1245fig3.jpg"></a></div><div id="largeobj_idm139979265100832" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4076081/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Fig. 3.</a></div><!--caption a7--><div class="caption"><p id="__p42">GRS distribution in FINGEN replication population (n = 310). If a GRS is positive, then the subject carries more at-risk alleles from the SNPs identified in the GWAS. If a GRS is negative, then the subject carries more beneficial alleles from the SNPs identified in the GWAS.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl7"><h3>TABLE 7.</h3><!--caption a7--><div class="caption"><p id="__p43">Sensitivity and specificity of GRS in FINGEN replication population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139979265098256" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">Responders</td><td rowspan="1" colspan="1" align="center">Nonresponders</td><td rowspan="1" colspan="1" align="center">Total</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Low GRS</td><td rowspan="1" colspan="1" align="center">131</td><td rowspan="1" colspan="1" align="center">92</td><td rowspan="1" colspan="1" align="center">223</td></tr><tr><td rowspan="1" colspan="1">High GRS</td><td rowspan="1" colspan="1" align="center">57</td><td rowspan="1" colspan="1" align="center">30</td><td rowspan="1" colspan="1" align="center">87</td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1" align="center">188</td><td rowspan="1" colspan="1" align="center">122</td><td rowspan="1" colspan="1" align="center"></td></tr><tr><td rowspan="1" colspan="1">Sensitivity</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">69.68% (95% CI: 62.57% to 76.16%)</td></tr><tr><td rowspan="1" colspan="1">Specificity</td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center"></td><td rowspan="1" colspan="1" align="center">24.59% (95% CI: 17.25% to 33.21%)</td></tr></tbody></table></div><div id="largeobj_idm139979265098256" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4076081/table/tbl7/?report=objectonly">Open in a separate window</a></div></div></div></div><div id="__sec18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec18title">DISCUSSION</h2><p id="__p44" class="p p-first">This study compares the frequency of alleles of SNPs in individuals with and without a decrease in plasma TG levels following LC n-3 PUFA supplementation. TG levels were significantly different between responders and nonresponders postsupplementation with n-3 PUFAs, representing the interindividual variability in response to LC n-3 PUFA supplementation. However, no significant locus (<em>P</em> &lt; 1.87 × 10<sup>−8)</sup> was identified by the GWAS, most likely because of the small sample size of the cohort. However, a suggestive association threshold (&lt;10<sup>−5</sup>) should be used to identify SNPs for consideration in follow-up studies. Results suggested that 13 SNPs had different frequencies between responders and nonresponders to LC n-3 PUFA supplementation, including SNPs in or near <em>IQCJ-SCHIP1</em>, <em>MYB</em>,<em> NELL1</em>, <em>NXPH1</em>, <em>PHF17</em>, and <em>SLIT2</em>. Further, a GRS based on these identified SNPs together with traditional risk factors can potentially differentiate subjects who may or may not respond to LC n-3 PUFAs for TG lowering.</p><p id="__p45">In the current study, the rs752088 in the intron of the <em>NELL1</em> gene was more frequent in nonresponders compared with responders. The <em>NELL1</em> gene encodes a heterotrimeric protein that is involved in cell growth regulation and differentiation. Specifically, <em>NELL1</em> plays a role in the promotion of osteogenic differentiation and bone formation. A recent study has suggested that NELL1 is also a potent antiadipogenic agent (<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943978">23</a>). Previously, a GWAS examined the use of hydrochlorothia­zide, an antihypertensive medication that is known to be associated with hyperglycemia and hypertriglyceridemia. Results indicate that two SNPs (rs12279250 and rs4319515) located at the <em>NELL1</em> gene locus achieved genome-wide significance for association with change in fasting plasma TG levels in African Americans, whereby each variant allele was associated with a 0.32 mM increase in the change in TG (<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943960">23</a>). The authors postulated that hydrochlorothiazide could modulate adipocyte differentiation through NELL1, leading to accumulation of plasma TG in susceptible subjects (<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943977">23</a>). Additionally, a recent study, namely the Genetics of Lipid Lowering Drugs and Diet Network study cohort, investigated which genetic variants may explain the observed heterogeneity in response to lipid-lowering therapy with fenofibrate (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943993">24</a>). The results demonstrate that seven proxy genes showed pleiotropic effects on nuclear magnetic resonance spectroscopy lipoprotein measurements including the <em>NELL1</em> gene on HDL and VLDL size (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943982">24</a>). In the same study, SNPs near the <em>IQCJ-SCHIP1</em> gene were also associated with VLDL particle clearance (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943972">24</a>). In the current study, SNPs of the intron of <em>IQCJ-SCHIP1</em> were more frequent in the nonresponders compared with the responders. It is well known in the literature that hypertriglyceridemia is characterized by elevated TG content in both VLDL particles and chylomicrons. Furthermore, recent studies suggested that the TG-lowering effect of LC n-3 PUFAs is associated with the decreased hepatic VLDL-TG secretion rate (<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943990">25</a>, <a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943988">26</a>). Thus, the functional impact of these genetic variations on plasma lipid metabolism and LC n-3 PUFA response should be further investigated.</p><p id="__p46">Results demonstrate that SNPs in two genes, <em>MYB</em> and <em>SLIT2</em>, were of different frequency between responders and nonresponders. These two genes have been previously related to features of obesity. First, rs6920829 in the intron of the <em>MYB</em> gene had higher frequency of G allele in nonresponders than in responders. This gene encodes a transcription factor that plays an essential role in the regulation of hematopoiesis and tumorigenesis. Recently, a study showed that rs9494145 in <em>MYB</em> was associated with both asthma and BMI, although associations with BMI were modest and not significant after correction for multiple comparisons (<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943968">27</a>). Further, the rs2952724/rs2629715 upstream of the <em>SLIT2</em> gene, which is known to suppress tumor growth, had a lower frequency in nonresponders compared with responders. Previously, researchers have shown that <em>SLIT2</em> was overexpressed in abdominal subcutaneous adipocytes from obese compared with nonobese Pima Indians (<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943966">28</a>). It is recognized that obese subjects frequently have atherogenic dyslipidemia, including elevated TG, VLDL, and apoB, as well as decreased HDL-C levels. The role of these specific genes and their SNPs in obesity and the plasma lipid response to LC n-3 PUFA supplementation still needs to be determined.</p><p id="__p47">Further, two genes identified here have been previously been associated with cancer: <em>PHF17</em> and <em>NXPH1</em>. However, very few studies have been published relating to the functions of the proteins encoded by these genes. Overall, further confirmation of these findings is needed to determine whether the signals from these SNPs are indirectly associated due to LD or are directly related to a functional impact of these genes.</p><p id="__p48">Pathway analysis reveals that these suggestive SNPs are related to the following pathways: PPAR signaling, tight junction signaling, and ceramide production. PPARs act as ligand activated transcriptional regulators. Their ligands include LC n-3 PUFAs, which makes the PPARs closely linked to intracellular lipid levels (<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943994">29</a>). Tight junctions are the most apical cell-cell junctions of epithelial cells, and evidence indicates that they participate in the suppression of cell proliferation and stimulation of differentiation. Ceramides are a class of sphingolipids that contribute to membrane structure and functions as an intracellular signaling molecule (<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943959">30</a>). Plasma ceramides have been shown to be elevated in obese subjects with type 2 diabetes and correlated with the severity of insulin resistance (<a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943984">31</a>). Overall, these pathways are closely linked to the suggestive SNPs relating to lipid and cellular metabolism in disease.</p><p id="__p49">Based on 10 SNPs from GWAS results, we propose a GRS that can effectively differentiate subjects with high or low genetic risk who may or may not respond to LC n-3 PUFA supplementation. The replication study suggests that subjects with a low GRS are more likely to be responders to LC n-3 PUFAs. Previous studies have found that a GRS from 32 BMI-associated variants explained 1.5–1.8% of variation in BMI (<a href="#bib32" rid="bib32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943987">32</a>, <a href="#bib33" rid="bib33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_385943967">33</a>). Thus, in the same way, our GRS from 10 SNPs related to LC n-3 PUFA supplementation explained 2% of TG response in the replication study, but this effect did not reach statistical significance. Differences may be attributable to, among other factors, differences in the study design including the length of the supplementation period, the dose of LC n-3 PUFAs, habitual diet and lifestyle of the participants, or inclusion/exclusion criteria. The addition of other genetic variations and risk factors could strengthen the predictive power of this model. Other risk factors may also have an additive or interactive effect with this GRS to modify TG variation after LC n-3 PUFA supplementation. In sum, the addition of a GRS to traditional risk factors could be an essential part of therapeutic stratification for preventive LC n-3 PUFA treatment.</p><p id="__p50" class="p p-last">To our knowledge, this is the first GWAS to examine TG response to LC n-3 PUFA supplementation. In the current study, genes of interest were identified that explain interindividual variability in lipid metabolism. Functional studies are needed to explore these novel biological pathways that may explain variability in the plasma lipid response to LC n-3 PUFA supplementation. Further studies need to confirm this GRS classification scheme together with other known SNPs or risk factors in other demographic and ethnic groups. Overall, this study shows that subjects who decrease their plasma TG levels following LC n-3 PUFA supplementation have a different genetic profile than individuals who do not respond to the LC n-3 PUFA supplementation.</p></div><div id="__sec19" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec19title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplemental Data:</strong>

<div class="sup-box half_rhythm" id="idm139979334961872"><a href="/pmc/articles/PMC4076081/bin/supp_55_7_1245__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm139979334958336" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139979334958336title">Acknowledgments</h2><div class="sec"><p id="__p51">The authors thank the subjects for their excellent collabora­tion. We thank Catherine Raymond for contributing to the labora­tory work. We would also like to thank Line Berthiaume who performed gas chromatographic analyses to determine FA profiles in PLs.</p></div></div><div id="idm139979334957424" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139979334957424title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="gloss1"></p><div class="def-list"><h4>Abbreviations:</h4><dl class="labeled-list"><dt>CI</dt><dd>confidence interval</dd><dt>CRP</dt><dd>C-reactive protein</dd><dt>FINGEN</dt><dd>Fish Oil Intervention and Genotype</dd><dt>GRS</dt><dd>genetic risk score</dd><dt>GWAS</dt><dd>genome-wide association study</dd><dt>HDL-C</dt><dd>HDL cholesterol</dd><dt>IPA</dt><dd>Ingenuity Pathway Analysis</dd><dt>IQCJ-SCHIP1</dt><dd>IQ motif containing J-schwannomin interacting protein 1 fusion protein</dd><dt>LC n-3 PUFA</dt><dd>long-chain n-3 PUFA</dd><dt>LD</dt><dd>linkage disequilibrium</dd><dt>MYB</dt><dd>V-myb myeloblastosis viral oncogene homolog</dd><dt>NELL1</dt><dd>NEL-like 1 (chicken)</dd><dt>NXPH1</dt><dd>neurexophilin-1</dd><dt>PHF17</dt><dd>PHD finger protein 17</dd><dt>PL</dt><dd>plasma phospholipid</dd><dt>SLIT2</dt><dd>slit homolog 2</dd><dt>TC</dt><dd>total cholesterol</dd></dl></div><p></p><p></p><p class="fn sec" id="idm139979260448736"></p><p id="__p70" class="p p-first-last">This work was supported by an operating grant from Canadian Institutes of Health Research (CIHR) (MOP229488). F. Guénard received a research fellowship award from the Heart and Stroke Foundation of Canada. O. Barbier received a scholarship from the CIHR (New Investigator Award, MSH95330). P. Couture received a scholarship from the Fonds de recherche du Québec - Santé (FRQS). M-C. Vohl holds a Tier 1 Canada Research Chair in Genomics Applied to Nutrition and Health. The Fish Oil Intervention and Genotype Study was supported by the Biotechnology and Biological Sciences Research Council. The authors do not declare any conflicts of interest.</p><p></p><p class="fn sec" id="fn3"><sup>[S]</sup>The online version of this article (available at <a href="http://www.jlr.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.jlr.org</a>) contains supplementary data in the form of one data file.</p></div></div><div id="idm139979260444960" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139979260444960title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="mixed-citation">Harris W. S.
1997. 
<span class="ref-title">n-3 fatty acids and serum lipoproteins: human studies</span>. <span class="ref-journal">Am. J. Clin. Nutr.</span>
<span class="ref-vol">65</span>: 1645S–1654S [<a href="/pubmed/9129504" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=n-3+fatty+acids+and+serum+lipoproteins:+human+studies&amp;author=W.+S.+Harris&amp;volume=65&amp;publication_year=1997&amp;pages=1645S-1654S&amp;pmid=9129504&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="mixed-citation">Hartweg J., Perera R., Montori V., Dinneen S., Neil H. A., Farmer A.
2008. 
<span class="ref-title">Omega-3 polyunsaturated fatty acids (PUFA) for type 2 diabetes mellitus</span>. <span class="ref-journal">Cochrane Database Syst. Rev.</span>
CD003205. [<a href="/pubmed/18254017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cochrane+Database+Syst.+Rev.&amp;title=Omega-3+polyunsaturated+fatty+acids+(PUFA)+for+type+2+diabetes+mellitus&amp;author=J.+Hartweg&amp;author=R.+Perera&amp;author=V.+Montori&amp;author=S.+Dinneen&amp;author=H.+A.+Neil&amp;publication_year=2008&amp;pages=CD003205&amp;pmid=18254017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="mixed-citation">Madden J., Williams C. M., Calder P. C., Lietz G., Miles E. A., Cordell H., Mathers J. C., Minihane A. M.
2011. 
<span class="ref-title">The impact of common gene variants on the response of biomarkers of cardiovascular disease (CVD) risk to increased fish oil fatty acids intakes</span>. <span class="ref-journal">Annu. Rev. Nutr.</span>
<span class="ref-vol">31</span>: 203–234 [<a href="/pubmed/21568708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Nutr.&amp;title=The+impact+of+common+gene+variants+on+the+response+of+biomarkers+of+cardiovascular+disease+(CVD)+risk+to+increased+fish+oil+fatty+acids+intakes&amp;author=J.+Madden&amp;author=C.+M.+Williams&amp;author=P.+C.+Calder&amp;author=G.+Lietz&amp;author=E.+A.+Miles&amp;volume=31&amp;publication_year=2011&amp;pages=203-234&amp;pmid=21568708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="mixed-citation">Caslake M. J., Miles E. A., Kofler B. M., Lietz G., Curtis P., Armah C. K., Kimber A. C., Grew J. P., Farrell L., Stannard J., et al. 
2008. 
<span class="ref-title">Effect of sex and genotype on cardiovascular biomarker response to fish oils: the FINGEN Study</span>. <span class="ref-journal">Am. J. Clin. Nutr.</span>
<span class="ref-vol">88</span>: 618–629 [<a href="/pubmed/18779276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=Effect+of+sex+and+genotype+on+cardiovascular+biomarker+response+to+fish+oils:+the+FINGEN+Study&amp;author=M.+J.+Caslake&amp;author=E.+A.+Miles&amp;author=B.+M.+Kofler&amp;author=G.+Lietz&amp;author=P.+Curtis&amp;volume=88&amp;publication_year=2008&amp;pages=618-629&amp;pmid=18779276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="mixed-citation">Cormier H., Rudkowska I., Paradis A. M., Thifault E., Garneau V., Lemieux S., Couture P., Vohl M. C.
2012. 
<span class="ref-title">Association between polymorphisms in the fatty acid desaturase gene cluster and the plasma triacylglycerol response to an n-3 PUFA supplementation</span>. <span class="ref-journal">Nutrients.</span>
<span class="ref-vol">4</span>: 1026–1041 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3448085/">PMC free article</a>]</span> [<a href="/pubmed/23016130" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients.&amp;title=Association+between+polymorphisms+in+the+fatty+acid+desaturase+gene+cluster+and+the+plasma+triacylglycerol+response+to+an+n-3+PUFA+supplementation&amp;author=H.+Cormier&amp;author=I.+Rudkowska&amp;author=A.+M.+Paradis&amp;author=E.+Thifault&amp;author=V.+Garneau&amp;volume=4&amp;publication_year=2012&amp;pages=1026-1041&amp;pmid=23016130&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="mixed-citation">Rudkowska I., Paradis A. M., Thifault E., Julien P., Barbier O., Couture P., Lemieux S., Vohl M. C.
2013. 
<span class="ref-title">Differences in metabolomic and transcriptomic profiles between responders and non-responders to an n-3 polyunsaturated fatty acids (PUFAs) supplementation</span>. <span class="ref-journal">Genes Nutr.</span>
<span class="ref-vol">8</span>: 411–423 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3689890/">PMC free article</a>]</span> [<a href="/pubmed/23250786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Nutr.&amp;title=Differences+in+metabolomic+and+transcriptomic+profiles+between+responders+and+non-responders+to+an+n-3+polyunsaturated+fatty+acids+(PUFAs)+supplementation&amp;author=I.+Rudkowska&amp;author=A.+M.+Paradis&amp;author=E.+Thifault&amp;author=P.+Julien&amp;author=O.+Barbier&amp;volume=8&amp;publication_year=2013&amp;pages=411-423&amp;pmid=23250786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="mixed-citation">Minihane A. M., Khan S., Leigh-Firbank E. C., Talmud P., Wright J. W., Murphy M. C., Griffin B. A., Williams C. M.
2000. 
<span class="ref-title">ApoE polymorphism and fish oil supplementation in subjects with an atherogenic lipoprotein phenotype</span>. <span class="ref-journal">Arterioscler. Thromb. Vasc. Biol.</span>
<span class="ref-vol">20</span>: 1990–1997 [<a href="/pubmed/10938022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol.&amp;title=ApoE+polymorphism+and+fish+oil+supplementation+in+subjects+with+an+atherogenic+lipoprotein+phenotype&amp;author=A.+M.+Minihane&amp;author=S.+Khan&amp;author=E.+C.+Leigh-Firbank&amp;author=P.+Talmud&amp;author=J.+W.+Wright&amp;volume=20&amp;publication_year=2000&amp;pages=1990-1997&amp;pmid=10938022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="mixed-citation">Olano-Martin E., Anil E., Caslake M. J., Packard C. J., Bedford D., Stewart G., Peiris D., Williams C. M., Minihane A. M.
2010. 
<span class="ref-title">Contribution of apolipoprotein E genotype and docosahexaenoic acid to the LDL-cholesterol response to fish oil</span>. <span class="ref-journal">Atherosclerosis.</span>
<span class="ref-vol">209</span>: 104–110 [<a href="/pubmed/19748619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis.&amp;title=Contribution+of+apolipoprotein+E+genotype+and+docosahexaenoic+acid+to+the+LDL-cholesterol+response+to+fish+oil&amp;author=E.+Olano-Martin&amp;author=E.+Anil&amp;author=M.+J.+Caslake&amp;author=C.+J.+Packard&amp;author=D.+Bedford&amp;volume=209&amp;publication_year=2010&amp;pages=104-110&amp;pmid=19748619&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="mixed-citation">Thifault E., Cormier H., Bouchard-Mercier A., Rudkowska I., Paradis A. M., Garneau V., Ouellette C., Lemieux S., Couture P., Vohl M. C.
2013. 
<span class="ref-title">Effects of age, sex, body mass index and APOE genotype on cardiovascular biomarker response to an n-3 polyunsaturated fatty acid supplementation</span>. <span class="ref-journal">J. Nutrigenet. Nutrigenomics.</span>
<span class="ref-vol">6</span>: 73–82 [<a href="/pubmed/23689286" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Nutrigenet.+Nutrigenomics.&amp;title=Effects+of+age,+sex,+body+mass+index+and+APOE+genotype+on+cardiovascular+biomarker+response+to+an+n-3+polyunsaturated+fatty+acid+supplementation&amp;author=E.+Thifault&amp;author=H.+Cormier&amp;author=A.+Bouchard-Mercier&amp;author=I.+Rudkowska&amp;author=A.+M.+Paradis&amp;volume=6&amp;publication_year=2013&amp;pages=73-82&amp;pmid=23689286&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="mixed-citation">Tai E. S., Corella D., Demissie S., Cupples L. A., Coltell O., Schaefer E. J., Tucker K. L., Ordovas J. M.
2005. 
<span class="ref-title">Polyunsaturated fatty acids interact with the PPARA-L162V polymorphism to affect plasma triglyceride and apolipoprotein C–III concentrations in the Framingham Heart Study</span>. <span class="ref-journal">J. Nutr.</span>
<span class="ref-vol">135</span>: 397–403 [<a href="/pubmed/15735069" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Nutr.&amp;title=Polyunsaturated+fatty+acids+interact+with+the+PPARA-L162V+polymorphism+to+affect+plasma+triglyceride+and+apolipoprotein+C–III+concentrations+in+the+Framingham+Heart+Study&amp;author=E.+S.+Tai&amp;author=D.+Corella&amp;author=S.+Demissie&amp;author=L.+A.+Cupples&amp;author=O.+Coltell&amp;volume=135&amp;publication_year=2005&amp;pages=397-403&amp;pmid=15735069&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="mixed-citation">Lindi V., Schwab U., Louheranta A., Laakso M., Vessby B., Hermansen K., Storlien L., Riccardi G., Rivellese A.
2003. 
<span class="ref-title">Impact of the Pro12Ala polymorphism of the PPAR-gamma2 gene on serum triacylglycerol response to n-3 fatty acid supplementation</span>. <span class="ref-journal">Mol. Genet. Metab.</span>
<span class="ref-vol">79</span>: 52–60 [<a href="/pubmed/12765846" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Genet.+Metab.&amp;title=Impact+of+the+Pro12Ala+polymorphism+of+the+PPAR-gamma2+gene+on+serum+triacylglycerol+response+to+n-3+fatty+acid+supplementation&amp;author=V.+Lindi&amp;author=U.+Schwab&amp;author=A.+Louheranta&amp;author=M.+Laakso&amp;author=B.+Vessby&amp;volume=79&amp;publication_year=2003&amp;pages=52-60&amp;pmid=12765846&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="mixed-citation">Burstein M., Samaille J.
1960. 
<span class="ref-title">On a rapid determination of the cholesterol bound to the serum alpha- and beta-lipoproteins</span>. [Article in French] <span class="ref-journal">Clin. Chim. Acta.</span>
<span class="ref-vol">5</span>: 609. [<a href="/pubmed/13806309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chim.+Acta.&amp;title=On+a+rapid+determination+of+the+cholesterol+bound+to+the+serum+alpha-+and+beta-lipoproteins&amp;author=M.+Burstein&amp;author=J.+Samaille&amp;volume=5&amp;publication_year=1960&amp;pages=609&amp;pmid=13806309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="mixed-citation">Albers J. J., Warnick G. R., Wiebe D., King P., Steiner P., Smith L., Breckenridge C., Chow A., Kuba K., Weidman S., et al. 
1978. 
<span class="ref-title">Multi-laboratory comparison of three heparin-Mn2+ precipitation procedures for estimating cholesterol in high-density lipoprotein</span>. <span class="ref-journal">Clin. Chem.</span>
<span class="ref-vol">24</span>: 853–856 [<a href="/pubmed/207461" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Multi-laboratory+comparison+of+three+heparin-Mn2++precipitation+procedures+for+estimating+cholesterol+in+high-density+lipoprotein&amp;author=J.+J.+Albers&amp;author=G.+R.+Warnick&amp;author=D.+Wiebe&amp;author=P.+King&amp;author=P.+Steiner&amp;volume=24&amp;publication_year=1978&amp;pages=853-856&amp;pmid=207461&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="mixed-citation">Friedewald W. T., Levy R. I., Fredrickson D. S.
1972. 
<span class="ref-title">Estima­tion of the concentration of low-density lipoprotein cholesterol in plasma, without use of the preparative ultracentrifuge</span>. <span class="ref-journal">Clin. Chem.</span>
<span class="ref-vol">18</span>: 499–502 [<a href="/pubmed/4337382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Estima­tion+of+the+concentration+of+low-density+lipoprotein+cholesterol+in+plasma,+without+use+of+the+preparative+ultracentrifuge&amp;author=W.+T.+Friedewald&amp;author=R.+I.+Levy&amp;author=D.+S.+Fredrickson&amp;volume=18&amp;publication_year=1972&amp;pages=499-502&amp;pmid=4337382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="mixed-citation">Desbuquois B., Aurbach G. D.
1971. 
<span class="ref-title">Use of polyethylene glycol to separate free and antibody-bound peptide hormones in radioimmunoassays</span>. <span class="ref-journal">J. Clin. Endocrinol. Metab.</span>
<span class="ref-vol">33</span>: 732–738 [<a href="/pubmed/5166455" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Endocrinol.+Metab.&amp;title=Use+of+polyethylene+glycol+to+separate+free+and+antibody-bound+peptide+hormones+in+radioimmunoassays&amp;author=B.+Desbuquois&amp;author=G.+D.+Aurbach&amp;volume=33&amp;publication_year=1971&amp;pages=732-738&amp;pmid=5166455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="mixed-citation">Richterich R., Dauwalder H.
1971. 
<span class="ref-title">Determination of plasma glucose by hexokinase-glucose-6-phosphate dehydrogenase method</span>. <span class="ref-journal">Schweiz. Med. Wochenschr.</span>
<span class="ref-vol">101</span>: 615–618 [<a href="/pubmed/5576952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Schweiz.+Med.+Wochenschr.&amp;title=Determination+of+plasma+glucose+by+hexokinase-glucose-6-phosphate+dehydrogenase+method&amp;author=R.+Richterich&amp;author=H.+Dauwalder&amp;volume=101&amp;publication_year=1971&amp;pages=615-618&amp;pmid=5576952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="mixed-citation">Laurell C. B.
1966. 
<span class="ref-title">Quantitative estimation of proteins by electrophoresis in agarose gel containing antibodies</span>. <span class="ref-journal">Anal. Biochem.</span>
<span class="ref-vol">15</span>: 45–52 [<a href="/pubmed/5959431" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Anal.+Biochem.&amp;title=Quantitative+estimation+of+proteins+by+electrophoresis+in+agarose+gel+containing+antibodies&amp;author=C.+B.+Laurell&amp;volume=15&amp;publication_year=1966&amp;pages=45-52&amp;pmid=5959431&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="mixed-citation">Pirro M., Bergeron J., Dagenais G. R., Bernard P. M., Cantin B., Despres J. P., Lamarche B.
2001. 
<span class="ref-title">Age and duration of follow-up as modulators of the risk for ischemic heart disease associated with high plasma C-reactive protein levels in men</span>. <span class="ref-journal">Arch. Intern. Med.</span>
<span class="ref-vol">161</span>: 2474–2480 [<a href="/pubmed/11700160" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Intern.+Med.&amp;title=Age+and+duration+of+follow-up+as+modulators+of+the+risk+for+ischemic+heart+disease+associated+with+high+plasma+C-reactive+protein+levels+in+men&amp;author=M.+Pirro&amp;author=J.+Bergeron&amp;author=G.+R.+Dagenais&amp;author=P.+M.+Bernard&amp;author=B.+Cantin&amp;volume=161&amp;publication_year=2001&amp;pages=2474-2480&amp;pmid=11700160&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="mixed-citation">Shaikh N. A., Downar E.
1981. 
<span class="ref-title">Time course of changes in porcine myocardial phospholipid levels during ischemia. A reassessment of the lysolipid hypothesis</span>. <span class="ref-journal">Circ. Res.</span>
<span class="ref-vol">49</span>: 316–325 [<a href="/pubmed/7249269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Res.&amp;title=Time+course+of+changes+in+porcine+myocardial+phospholipid+levels+during+ischemia.+A+reassessment+of+the+lysolipid+hypothesis&amp;author=N.+A.+Shaikh&amp;author=E.+Downar&amp;volume=49&amp;publication_year=1981&amp;pages=316-325&amp;pmid=7249269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="mixed-citation">Kröger E., Verreault R., Carmichael P. H., Lindsay J., Julien P., Dewailly E., Ayotte P., Laurin D.
2009. 
<span class="ref-title">Omega-3 fatty acids and risk of dementia: the Canadian Study of Health and Aging</span>. <span class="ref-journal">Am. J. Clin. Nutr.</span>
<span class="ref-vol">90</span>: 184–192 [<a href="/pubmed/19474137" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=Omega-3+fatty+acids+and+risk+of+dementia:+the+Canadian+Study+of+Health+and+Aging&amp;author=E.+Kröger&amp;author=R.+Verreault&amp;author=P.+H.+Carmichael&amp;author=J.+Lindsay&amp;author=P.+Julien&amp;volume=90&amp;publication_year=2009&amp;pages=184-192&amp;pmid=19474137&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="mixed-citation">Nicodemus K. K., Liu W., Chase G. A., Tsai Y. Y., Fallin M. D.
2005. 
<span class="ref-title">Comparison of type I error for multiple test corrections in large single-nucleotide polymorphism studies using principal components versus haplotype blocking algorithms</span>. <span class="ref-journal">BMC Genet.</span>
<span class="ref-vol">6</span> (<span class="ref-iss">Suppl 1</span>): S78. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1866703/">PMC free article</a>]</span> [<a href="/pubmed/16451692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genet.&amp;title=Comparison+of+type+I+error+for+multiple+test+corrections+in+large+single-nucleotide+polymorphism+studies+using+principal+components+versus+haplotype+blocking+algorithms&amp;author=K.+K.+Nicodemus&amp;author=W.+Liu&amp;author=G.+A.+Chase&amp;author=Y.+Y.+Tsai&amp;author=M.+D.+Fallin&amp;volume=6&amp;issue=Suppl+1&amp;publication_year=2005&amp;pages=S78&amp;pmid=16451692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="mixed-citation">Duggal P., Gillanders E. M., Holmes T. N., Bailey-Wilson J. E.
2008. 
<span class="ref-title">Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies</span>. <span class="ref-journal">BMC Genomics.</span>
<span class="ref-vol">9</span>: 516. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2621212/">PMC free article</a>]</span> [<a href="/pubmed/18976480" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genomics.&amp;title=Establishing+an+adjusted+p-value+threshold+to+control+the+family-wide+type+1+error+in+genome+wide+association+studies&amp;author=P.+Duggal&amp;author=E.+M.+Gillanders&amp;author=T.+N.+Holmes&amp;author=J.+E.+Bailey-Wilson&amp;volume=9&amp;publication_year=2008&amp;pages=516&amp;pmid=18976480&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="mixed-citation">Del-Aguila J. L., Beitelshees A. L., Cooper-Dehoff R. M., Chapman A. B., Gums J. G., Bailey K., Gong Y., Turner S. T., Johnson J. A., Boerwinkle E.
2014. 
<span class="ref-title">Genome-wide association ana­lyses suggest NELL1 influences adverse metabolic response to HCTZ in African Americans</span>. <span class="ref-journal">Pharmacogenomics J.</span>
<span class="ref-vol">14</span>: 35–40 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3812324/">PMC free article</a>]</span> [<a href="/pubmed/23400010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics+J.&amp;title=Genome-wide+association+ana­lyses+suggest+NELL1+influences+adverse+metabolic+response+to+HCTZ+in+African+Americans&amp;author=J.+L.+Del-Aguila&amp;author=A.+L.+Beitelshees&amp;author=R.+M.+Cooper-Dehoff&amp;author=A.+B.+Chapman&amp;author=J.+G.+Gums&amp;volume=14&amp;publication_year=2014&amp;pages=35-40&amp;pmid=23400010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="mixed-citation">Kraja A. T., Borecki I. B., Tsai M. Y., Ordovas J. M., Hopkins P. N., Lai C. Q., Frazier-Wood A. C., Straka R. J., Hixson J. E., Province M. A., et al. 
2013. 
<span class="ref-title">Genetic analysis of 16 NMR-lipoprotein fractions in humans, the GOLDN study</span>. <span class="ref-journal">Lipids.</span>
<span class="ref-vol">48</span>: 155–165 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3601742/">PMC free article</a>]</span> [<a href="/pubmed/23192668" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lipids.&amp;title=Genetic+analysis+of+16+NMR-lipoprotein+fractions+in+humans,+the+GOLDN+study&amp;author=A.+T.+Kraja&amp;author=I.+B.+Borecki&amp;author=M.+Y.+Tsai&amp;author=J.+M.+Ordovas&amp;author=P.+N.+Hopkins&amp;volume=48&amp;publication_year=2013&amp;pages=155-165&amp;pmid=23192668&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="mixed-citation">Qi K., Fan C., Jiang J., Zhu H., Jiao H., Meng Q., Deckelbaum R. J.
2008. 
<span class="ref-title">Omega-3 fatty acid containing diets decrease plasma triglyceride concentrations in mice by reducing endogenous triglyceride synthesis and enhancing the blood clearance of triglyceride-rich particles</span>. <span class="ref-journal">Clin. Nutr.</span>
<span class="ref-vol">27</span>: 424–430 [<a href="/pubmed/18362042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Nutr.&amp;title=Omega-3+fatty+acid+containing+diets+decrease+plasma+triglyceride+concentrations+in+mice+by+reducing+endogenous+triglyceride+synthesis+and+enhancing+the+blood+clearance+of+triglyceride-rich+particles&amp;author=K.+Qi&amp;author=C.+Fan&amp;author=J.+Jiang&amp;author=H.+Zhu&amp;author=H.+Jiao&amp;volume=27&amp;publication_year=2008&amp;pages=424-430&amp;pmid=18362042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="mixed-citation">Wong A. T., Chan D. C., Ooi E. M., Ng T. W., Watts G. F., Barrett P. H.
2013. 
<span class="ref-title">Omega-3 fatty acid ethyl ester supplementation decreases very-low-density lipoprotein triacylglycerol secretion in obese men</span>. <span class="ref-journal">Clin. Sci. (Lond.).</span>
<span class="ref-vol">125</span>: 45–51 [<a href="/pubmed/23356247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Sci.+(Lond.).&amp;title=Omega-3+fatty+acid+ethyl+ester+supplementation+decreases+very-low-density+lipoprotein+triacylglycerol+secretion+in+obese+men&amp;author=A.+T.+Wong&amp;author=D.+C.+Chan&amp;author=E.+M.+Ooi&amp;author=T.+W.+Ng&amp;author=G.+F.+Watts&amp;volume=125&amp;publication_year=2013&amp;pages=45-51&amp;pmid=23356247&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="mixed-citation">Melén E., Himes B. E., Brehm J. M., Boutaoui N., Klanderman B. J., Sylvia J. S., Lasky-Su J.
2010. 
<span class="ref-title">Analyses of shared genetic factors between asthma and obesity in children</span>. <span class="ref-journal">J. Allergy Clin. Immunol.</span>
<span class="ref-vol">126</span>: 631–637 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2941152/">PMC free article</a>]</span> [<a href="/pubmed/20816195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Allergy+Clin.+Immunol.&amp;title=Analyses+of+shared+genetic+factors+between+asthma+and+obesity+in+children&amp;author=E.+Melén&amp;author=B.+E.+Himes&amp;author=J.+M.+Brehm&amp;author=N.+Boutaoui&amp;author=B.+J.+Klanderman&amp;volume=126&amp;publication_year=2010&amp;pages=631-637&amp;pmid=20816195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="mixed-citation">Lee Y. H., Nair S., Rousseau E., Allison D. B., Page G. P., Tataranni P. A., Bogardus C., Permana P. A.
2005. 
<span class="ref-title">Microarray profiling of isolated abdominal subcutaneous adipocytes from obese vs non-obese Pima Indians: increased expression of inflammation-related genes</span>. <span class="ref-journal">Diabetologia.</span>
<span class="ref-vol">48</span>: 1776–1783 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1409820/">PMC free article</a>]</span> [<a href="/pubmed/16059715" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetologia.&amp;title=Microarray+profiling+of+isolated+abdominal+subcutaneous+adipocytes+from+obese+vs+non-obese+Pima+Indians:+increased+expression+of+inflammation-related+genes&amp;author=Y.+H.+Lee&amp;author=S.+Nair&amp;author=E.+Rousseau&amp;author=D.+B.+Allison&amp;author=G.+P.+Page&amp;volume=48&amp;publication_year=2005&amp;pages=1776-1783&amp;pmid=16059715&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="mixed-citation">Rudkowska I., Verreault M., Barbier O., Vohl M. C.
2009. 
<span class="ref-title">Differences in transcriptional activation by the two allelic (L162V Polymorphic) variants of PPARalpha after omega-3 fatty acids treatment</span>. <span class="ref-journal">PPAR Res.</span>
<span class="ref-vol">2009</span>: 369602. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2649533/">PMC free article</a>]</span> [<a href="/pubmed/19266045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PPAR+Res.&amp;title=Differences+in+transcriptional+activation+by+the+two+allelic+(L162V+Polymorphic)+variants+of+PPARalpha+after+omega-3+fatty+acids+treatment&amp;author=I.+Rudkowska&amp;author=M.+Verreault&amp;author=O.+Barbier&amp;author=M.+C.+Vohl&amp;volume=2009&amp;publication_year=2009&amp;pages=369602&amp;pmid=19266045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="mixed-citation">Hannun Y. A., Obeid L. M.
2002. 
<span class="ref-title">The ceramide-centric universe of lipid-mediated cell regulation: stress encounters of the lipid kind</span>. <span class="ref-journal">J. Biol. Chem.</span>
<span class="ref-vol">277</span>: 25847–25850 [<a href="/pubmed/12011103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=The+ceramide-centric+universe+of+lipid-mediated+cell+regulation:+stress+encounters+of+the+lipid+kind&amp;author=Y.+A.+Hannun&amp;author=L.+M.+Obeid&amp;volume=277&amp;publication_year=2002&amp;pages=25847-25850&amp;pmid=12011103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="mixed-citation">Haus J. M., Kashyap S. R., Kasumov T., Zhang R., Kelly K. R., Defronzo R. A., Kirwan J. P.
2009. 
<span class="ref-title">Plasma ceramides are elevated in obese subjects with type 2 diabetes and correlate with the severity of insulin resistance</span>. <span class="ref-journal">Diabetes.</span>
<span class="ref-vol">58</span>: 337–343 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628606/">PMC free article</a>]</span> [<a href="/pubmed/19008343" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes.&amp;title=Plasma+ceramides+are+elevated+in+obese+subjects+with+type+2+diabetes+and+correlate+with+the+severity+of+insulin+resistance&amp;author=J.+M.+Haus&amp;author=S.+R.+Kashyap&amp;author=T.+Kasumov&amp;author=R.+Zhang&amp;author=K.+R.+Kelly&amp;volume=58&amp;publication_year=2009&amp;pages=337-343&amp;pmid=19008343&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="mixed-citation">Qi Q., Chu A. Y., Kang J. H., Huang J., Rose L. M., Jensen M. K., Liang L., Curhan G. C., Pasquale L. R., Wiggs J. L., et al. 
2014. 
<span class="ref-title">Fried food consumption, genetic risk, and body mass index: gene-diet interaction analysis in three US cohort studies</span>. <span class="ref-journal">BMJ.</span>
<span class="ref-vol">348</span>: g1610. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3959253/">PMC free article</a>]</span> [<a href="/pubmed/24646652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMJ.&amp;title=Fried+food+consumption,+genetic+risk,+and+body+mass+index:+gene-diet+interaction+analysis+in+three+US+cohort+studies&amp;author=Q.+Qi&amp;author=A.+Y.+Chu&amp;author=J.+H.+Kang&amp;author=J.+Huang&amp;author=L.+M.+Rose&amp;volume=348&amp;publication_year=2014&amp;pages=g1610&amp;pmid=24646652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="mixed-citation">Speliotes E. K., Willer C. J., Berndt S. I., Monda K. L., Thorleifsson G., Jackson A. U., Lango A. H., Lindgren C. M., Luan J., Mägi R., et al. 
2010. 
<span class="ref-title">Association analyses of 249,796 individuals reveal 18 new loci associated with body mass index</span>. <span class="ref-journal">Nat. Genet.</span>
<span class="ref-vol">42</span>: 937–948 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014648/">PMC free article</a>]</span> [<a href="/pubmed/20935630" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Association+analyses+of+249,796+individuals+reveal+18+new+loci+associated+with+body+mass+index&amp;author=E.+K.+Speliotes&amp;author=C.+J.+Willer&amp;author=S.+I.+Berndt&amp;author=K.+L.+Monda&amp;author=G.+Thorleifsson&amp;volume=42&amp;publication_year=2010&amp;pages=937-948&amp;pmid=20935630&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4076081&amp;issue-id=239622&amp;journal-id=568&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139979298995024title">Abstract</a></li><li><a href="#__sec1title">SUBJECTS AND METHODS</a></li><li><a href="#__sec13title">RESULTS</a></li><li><a href="#__sec18title">DISCUSSION</a></li><li><a href="#__sec19title">Supplementary Material</a></li><li><a href="#idm139979334958336title">Acknowledgments</a></li><li><a href="#idm139979334957424title">Footnotes</a></li><li><a href="#idm139979260444960title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of Lipid Research</span> are provided here courtesy of <strong>American Society for Biochemistry and Molecular Biology</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4076081/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4076081/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4076081/pdf/1245.pdf">PDF (679K)</a></li> | <li><a href="#" data-citationid="PMC4076081" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4076081%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4076081%2F&amp;text=Genome-wide%20association%20study%20of%20the%20plasma%20triglyceride%20response%20to%20an%20n-3%20polyunsaturated%20fatty%20acid%20supplementation" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4076081%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="24847101" data-aiid="4076081" data-aid="4076081" data-iid="239622" data-domainid="568" data-domain="jlr" data-accid="PMC4076081" data-md5="10d708daef219410c5023d1adf682016"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:13:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BB144DD56FFF100000000030001B6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4076081%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jlr&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4076081/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>