<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4289690/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289690/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms"><meta name="citation_authors" content="Hansjörg Baurecht, Melanie Hotze, Stephan Brand, Carsten Büning, Paul Cormican, Aiden Corvin, David Ellinghaus, Eva Ellinghaus, Jorge Esparza-Gordillo, Regina Fölster-Holst, Andre Franke, Christian Gieger, Norbert Hubner, Thomas Illig, Alan&nbsp;D. Irvine, Michael Kabesch, Young&nbsp;A.E. Lee, Wolfgang Lieb, Ingo Marenholz, W.H.&nbsp;Irwin McLean, Derek&nbsp;W. Morris, Ulrich Mrowietz, Rajan Nair, Markus&nbsp;M. Nöthen, Natalija Novak, Grainne&nbsp;M. O’Regan, the PAGE Consortium, Stefan Schreiber, Catherine Smith, Konstantin Strauch, Philip&nbsp;E. Stuart, Richard Trembath, Lam&nbsp;C. Tsoi, Michael Weichenthal, Jonathan Barker, James&nbsp;T. Elder, Stephan Weidinger, Heather&nbsp;J. Cordell, Sara&nbsp;J. Brown"><meta name="citation_date" content="8 January 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="96"><meta name="citation_firstpage" content="104"><meta name="citation_doi" content="10.1016/j.ajhg.2014.12.004"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4289690/?report=abstract"><meta name="citation_pmid" content="25574825"><meta name="DC.Title" content="Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Hansjörg Baurecht"><meta name="DC.Contributor" content="Melanie Hotze"><meta name="DC.Contributor" content="Stephan Brand"><meta name="DC.Contributor" content="Carsten Büning"><meta name="DC.Contributor" content="Paul Cormican"><meta name="DC.Contributor" content="Aiden Corvin"><meta name="DC.Contributor" content="David Ellinghaus"><meta name="DC.Contributor" content="Eva Ellinghaus"><meta name="DC.Contributor" content="Jorge Esparza-Gordillo"><meta name="DC.Contributor" content="Regina Fölster-Holst"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Norbert Hubner"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Alan&nbsp;D. Irvine"><meta name="DC.Contributor" content="Michael Kabesch"><meta name="DC.Contributor" content="Young&nbsp;A.E. Lee"><meta name="DC.Contributor" content="Wolfgang Lieb"><meta name="DC.Contributor" content="Ingo Marenholz"><meta name="DC.Contributor" content="W.H.&nbsp;Irwin McLean"><meta name="DC.Contributor" content="Derek&nbsp;W. Morris"><meta name="DC.Contributor" content="Ulrich Mrowietz"><meta name="DC.Contributor" content="Rajan Nair"><meta name="DC.Contributor" content="Markus&nbsp;M. Nöthen"><meta name="DC.Contributor" content="Natalija Novak"><meta name="DC.Contributor" content="Grainne&nbsp;M. O’Regan"><meta name="DC.Contributor" content="the PAGE Consortium"><meta name="DC.Contributor" content="Stefan Schreiber"><meta name="DC.Contributor" content="Catherine Smith"><meta name="DC.Contributor" content="Konstantin Strauch"><meta name="DC.Contributor" content="Philip&nbsp;E. Stuart"><meta name="DC.Contributor" content="Richard Trembath"><meta name="DC.Contributor" content="Lam&nbsp;C. Tsoi"><meta name="DC.Contributor" content="Michael Weichenthal"><meta name="DC.Contributor" content="Jonathan Barker"><meta name="DC.Contributor" content="James&nbsp;T. Elder"><meta name="DC.Contributor" content="Stephan Weidinger"><meta name="DC.Contributor" content="Heather&nbsp;J. Cordell"><meta name="DC.Contributor" content="Sara&nbsp;J. Brown"><meta name="DC.Date" content="2015 Jan 8"><meta name="DC.Identifier" content="10.1016/j.ajhg.2014.12.004"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms"><meta property="og:type" content="article"><meta property="og:description" content="Atopic dermatitis and psoriasis are the two most common immune-mediated inflammatory disorders affecting the skin. Genome-wide studies demonstrate a high degree of genetic overlap, but these diseases have mutually exclusive clinical phenotypes and opposing ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4289690/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4289690/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4289690/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B30A6DD5760710000000007D40496.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289690%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/247455/">v.96(1); 2015 Jan 8</a></li><li class="accid">PMC4289690</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2015 Jan 8; 96(1): 104–120. </span></div><div>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2014.12.004" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2014.12.004</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4289690</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25574825">25574825</a></div></div></div></div><h1 class="content-title">Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Baurecht%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812602429232" co-class="co-affbox">Hansjörg Baurecht</a>,<sup>1,</sup><sup>31</sup> <a href="/pubmed/?term=Hotze%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812643519648" co-class="co-affbox">Melanie Hotze</a>,<sup>1,</sup><sup>31</sup> <a href="/pubmed/?term=Brand%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606628224" co-class="co-affbox">Stephan Brand</a>,<sup>2</sup> <a href="/pubmed/?term=B%26%23x000fc%3Bning%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606626240" co-class="co-affbox">Carsten Büning</a>,<sup>3</sup> <a href="/pubmed/?term=Cormican%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812643670128" co-class="co-affbox">Paul Cormican</a>,<sup>4</sup> <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812586276064" co-class="co-affbox">Aiden Corvin</a>,<sup>4</sup> <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812608381616" co-class="co-affbox">David Ellinghaus</a>,<sup>5</sup> <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812608379632" co-class="co-affbox">Eva Ellinghaus</a>,<sup>5</sup> <a href="/pubmed/?term=Esparza-Gordillo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812605379216" co-class="co-affbox">Jorge Esparza-Gordillo</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=F%26%23x000f6%3Blster-Holst%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812602466496" co-class="co-affbox">Regina Fölster-Holst</a>,<sup>1</sup> <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812603050272" co-class="co-affbox">Andre Franke</a>,<sup>5</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812603048288" co-class="co-affbox">Christian Gieger</a>,<sup>8,</sup><sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Hubner%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606378544" co-class="co-affbox">Norbert Hubner</a>,<sup>6</sup> <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606376560" co-class="co-affbox">Thomas Illig</a>,<sup>9,</sup><sup>11</sup> <a href="/pubmed/?term=Irvine%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812644274432" co-class="co-affbox">Alan&nbsp;D. Irvine</a>,<sup>12,</sup><sup>13,</sup><sup>14</sup> <a href="/pubmed/?term=Kabesch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812679068688" co-class="co-affbox">Michael Kabesch</a>,<sup>15</sup> <a href="/pubmed/?term=Lee%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812679066704" co-class="co-affbox">Young&nbsp;A.E. Lee</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Lieb%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812604640960" co-class="co-affbox">Wolfgang Lieb</a>,<sup>16</sup> <a href="/pubmed/?term=Marenholz%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812601306976" co-class="co-affbox">Ingo Marenholz</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=McLean%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812603231088" co-class="co-affbox">W.H.&nbsp;Irwin McLean</a>,<sup>17</sup> <a href="/pubmed/?term=Morris%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812599818432" co-class="co-affbox">Derek&nbsp;W. Morris</a>,<sup>4,</sup><sup>18</sup> <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812604083440" co-class="co-affbox">Ulrich Mrowietz</a>,<sup>1</sup> <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812607140592" co-class="co-affbox">Rajan Nair</a>,<sup>19</sup> <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812607138608" co-class="co-affbox">Markus&nbsp;M. Nöthen</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Novak%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812676451472" co-class="co-affbox">Natalija Novak</a>,<sup>22</sup> <a href="/pubmed/?term=O%26%23x02019%3BRegan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812604940256" co-class="co-affbox">Grainne&nbsp;M. O’Regan</a>,<sup>12</sup> the PAGE Consortium, <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812603404080" co-class="co-affbox">Stefan Schreiber</a>,<sup>5,</sup><sup>23</sup> <a href="/pubmed/?term=Smith%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812610480096" co-class="co-affbox">Catherine Smith</a>,<sup>24</sup> <a href="/pubmed/?term=Strauch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812598815648" co-class="co-affbox">Konstantin Strauch</a>,<sup>8,</sup><sup>25</sup> <a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812608816464" co-class="co-affbox">Philip&nbsp;E. Stuart</a>,<sup>19</sup> <a href="/pubmed/?term=Trembath%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812608814480" co-class="co-affbox">Richard Trembath</a>,<sup>26</sup> <a href="/pubmed/?term=Tsoi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812585495728" co-class="co-affbox">Lam&nbsp;C. Tsoi</a>,<sup>27</sup> <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606617424" co-class="co-affbox">Michael Weichenthal</a>,<sup>1</sup> <a href="/pubmed/?term=Barker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812599969344" co-class="co-affbox">Jonathan Barker</a>,<sup>24</sup> <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812599967360" co-class="co-affbox">James&nbsp;T. Elder</a>,<sup>19,</sup><sup>28</sup> <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812606089008" co-class="co-affbox">Stephan Weidinger</a>,<sup>1,</sup><sup>32,</sup><sup>∗</sup> <a href="/pubmed/?term=Cordell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812607912432" co-class="co-affbox">Heather&nbsp;J. Cordell</a>,<sup>29,</sup><sup>32</sup> and  <a href="/pubmed/?term=Brown%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825" class="affpopup" co-rid="_co_idm139812605659968" co-class="co-affbox">Sara&nbsp;J. Brown</a><sup>30,</sup><sup>32,</sup><sup>∗∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139812602429232"><h3 class="no_margin">Hansjörg Baurecht</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Baurecht%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Hansjörg Baurecht</a></div></div><div id="_co_idm139812643519648"><h3 class="no_margin">Melanie Hotze</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Hotze%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Melanie Hotze</a></div></div><div id="_co_idm139812606628224"><h3 class="no_margin">Stephan Brand</h3><p><sup>2</sup>Department of Medicine II - Grosshadern, Ludwig-Maximilians-University (LMU), 81377 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Brand%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Stephan Brand</a></div></div><div id="_co_idm139812606626240"><h3 class="no_margin">Carsten Büning</h3><p><sup>3</sup>Department of Gastroenterology, Hepatology and Endocrinology, Charité, Campus Mitte, 10117 Berlin, Germany</p><div>Find articles by <a href="/pubmed/?term=B%26%23x000fc%3Bning%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Carsten Büning</a></div></div><div id="_co_idm139812643670128"><h3 class="no_margin">Paul Cormican</h3><p><sup>4</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry and Institute of Molecular Medicine, Trinity College Dublin, Dublin 2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Cormican%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Paul Cormican</a></div></div><div id="_co_idm139812586276064"><h3 class="no_margin">Aiden Corvin</h3><p><sup>4</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry and Institute of Molecular Medicine, Trinity College Dublin, Dublin 2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Corvin%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Aiden Corvin</a></div></div><div id="_co_idm139812608381616"><h3 class="no_margin">David Ellinghaus</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University of Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">David Ellinghaus</a></div></div><div id="_co_idm139812608379632"><h3 class="no_margin">Eva Ellinghaus</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University of Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Eva Ellinghaus</a></div></div><div id="_co_idm139812605379216"><h3 class="no_margin">Jorge Esparza-Gordillo</h3><p><sup>6</sup>Max-Delbrück-Centrum (MDC) for Molecular Medicine, Berlin-Buch, 13092 Berlin, Germany</p><p><sup>7</sup>Pediatric Allergy, Experimental and Clinical Research Center, Charité Universitätsmedizin Berlin, 10117 Berlin, Germany</p><div>Find articles by <a href="/pubmed/?term=Esparza-Gordillo%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Jorge Esparza-Gordillo</a></div></div><div id="_co_idm139812602466496"><h3 class="no_margin">Regina Fölster-Holst</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=F%26%23x000f6%3Blster-Holst%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Regina Fölster-Holst</a></div></div><div id="_co_idm139812603050272"><h3 class="no_margin">Andre Franke</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University of Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Andre Franke</a></div></div><div id="_co_idm139812603048288"><h3 class="no_margin">Christian Gieger</h3><p><sup>8</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>9</sup>Research Unit of Molecular Epidemiology, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>10</sup>Institute of Epidemiology II, Helmholtz Zentrum München - German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Christian Gieger</a></div></div><div id="_co_idm139812606378544"><h3 class="no_margin">Norbert Hubner</h3><p><sup>6</sup>Max-Delbrück-Centrum (MDC) for Molecular Medicine, Berlin-Buch, 13092 Berlin, Germany</p><div>Find articles by <a href="/pubmed/?term=Hubner%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Norbert Hubner</a></div></div><div id="_co_idm139812606376560"><h3 class="no_margin">Thomas Illig</h3><p><sup>9</sup>Research Unit of Molecular Epidemiology, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>11</sup>Hannover Unified Biobank, Hannover Medical School, 30625 Hannover, Germany</p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Thomas Illig</a></div></div><div id="_co_idm139812644274432"><h3 class="no_margin">Alan&nbsp;D. Irvine</h3><p><sup>12</sup>Department of Paediatric Dermatology, Our Lady’s Children’s Hospital, Crumlin, Dublin 12, Ireland</p><p><sup>13</sup>National Children’s Research Centre, Dublin 12, Ireland</p><p><sup>14</sup>Department of Clinical Medicine, Trinity College Dublin, Dublin 2, Ireland</p><div>Find articles by <a href="/pubmed/?term=Irvine%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Alan&nbsp;D. Irvine</a></div></div><div id="_co_idm139812679068688"><h3 class="no_margin">Michael Kabesch</h3><p><sup>15</sup>Department of Pediatric Pneumology and Allergy, University Children's Hospital Regensburg (KUNO), 93042 Regensburg, Germany</p><div>Find articles by <a href="/pubmed/?term=Kabesch%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Michael Kabesch</a></div></div><div id="_co_idm139812679066704"><h3 class="no_margin">Young&nbsp;A.E. Lee</h3><p><sup>6</sup>Max-Delbrück-Centrum (MDC) for Molecular Medicine, Berlin-Buch, 13092 Berlin, Germany</p><p><sup>7</sup>Pediatric Allergy, Experimental and Clinical Research Center, Charité Universitätsmedizin Berlin, 10117 Berlin, Germany</p><div>Find articles by <a href="/pubmed/?term=Lee%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Young&nbsp;A.E. Lee</a></div></div><div id="_co_idm139812604640960"><h3 class="no_margin">Wolfgang Lieb</h3><p><sup>16</sup>Institute of Epidemiology and PopGen Biobank, University Hospital Schleswig-Holstein, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Lieb%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Wolfgang Lieb</a></div></div><div id="_co_idm139812601306976"><h3 class="no_margin">Ingo Marenholz</h3><p><sup>6</sup>Max-Delbrück-Centrum (MDC) for Molecular Medicine, Berlin-Buch, 13092 Berlin, Germany</p><p><sup>7</sup>Pediatric Allergy, Experimental and Clinical Research Center, Charité Universitätsmedizin Berlin, 10117 Berlin, Germany</p><div>Find articles by <a href="/pubmed/?term=Marenholz%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Ingo Marenholz</a></div></div><div id="_co_idm139812603231088"><h3 class="no_margin">W.H.&nbsp;Irwin McLean</h3><p><sup>17</sup>Dermatology and Genetic Medicine, College of Life Sciences and College of Medicine, Dentistry &amp; Nursing, University of Dundee, Dundee DD1 5EH, UK</p><div>Find articles by <a href="/pubmed/?term=McLean%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">W.H.&nbsp;Irwin McLean</a></div></div><div id="_co_idm139812599818432"><h3 class="no_margin">Derek&nbsp;W. Morris</h3><p><sup>4</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry and Institute of Molecular Medicine, Trinity College Dublin, Dublin 2, Ireland</p><p><sup>18</sup>Cognitive Genetics and Therapy Group, School of Psychology and Discipline of Biochemistry, National University of Ireland, Galway, Ireland</p><div>Find articles by <a href="/pubmed/?term=Morris%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Derek&nbsp;W. Morris</a></div></div><div id="_co_idm139812604083440"><h3 class="no_margin">Ulrich Mrowietz</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Ulrich Mrowietz</a></div></div><div id="_co_idm139812607140592"><h3 class="no_margin">Rajan Nair</h3><p><sup>19</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109-5675, USA</p><div>Find articles by <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Rajan Nair</a></div></div><div id="_co_idm139812607138608"><h3 class="no_margin">Markus&nbsp;M. Nöthen</h3><p><sup>20</sup>Institute of Human Genetics, University of Bonn, 53127 Bonn, Germany</p><p><sup>21</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, 53127 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Markus&nbsp;M. Nöthen</a></div></div><div id="_co_idm139812676451472"><h3 class="no_margin">Natalija Novak</h3><p><sup>22</sup>Department of Dermatology and Allergy, University of Bonn Medical Center, 53105 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Novak%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Natalija Novak</a></div></div><div id="_co_idm139812604940256"><h3 class="no_margin">Grainne&nbsp;M. O’Regan</h3><p><sup>12</sup>Department of Paediatric Dermatology, Our Lady’s Children’s Hospital, Crumlin, Dublin 12, Ireland</p><div>Find articles by <a href="/pubmed/?term=O%26%23x02019%3BRegan%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Grainne&nbsp;M. O’Regan</a></div></div><div id="_co_idm139812603404080"><h3 class="no_margin">Stefan Schreiber</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University of Kiel, 24105 Kiel, Germany</p><p><sup>23</sup>Department of Internal Medicine, University Medical Center Schleswig-Holstein, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Schreiber%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Stefan Schreiber</a></div></div><div id="_co_idm139812610480096"><h3 class="no_margin">Catherine Smith</h3><p><sup>24</sup>Division of Genetics and Molecular Medicine, St John’s Institute of Dermatology, Kings College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Smith%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Catherine Smith</a></div></div><div id="_co_idm139812598815648"><h3 class="no_margin">Konstantin Strauch</h3><p><sup>8</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>25</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, 80539 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Strauch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Konstantin Strauch</a></div></div><div id="_co_idm139812608816464"><h3 class="no_margin">Philip&nbsp;E. Stuart</h3><p><sup>19</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109-5675, USA</p><div>Find articles by <a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Philip&nbsp;E. Stuart</a></div></div><div id="_co_idm139812608814480"><h3 class="no_margin">Richard Trembath</h3><p><sup>26</sup>Queen Mary University of London, Barts and the London School of Medicine and Dentistry, London E1 4NS, UK</p><div>Find articles by <a href="/pubmed/?term=Trembath%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Richard Trembath</a></div></div><div id="_co_idm139812585495728"><h3 class="no_margin">Lam&nbsp;C. Tsoi</h3><p><sup>27</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan School of Public Health, Ann Arbor, MI 48109-5314, USA</p><div>Find articles by <a href="/pubmed/?term=Tsoi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Lam&nbsp;C. Tsoi</a></div></div><div id="_co_idm139812606617424"><h3 class="no_margin">Michael Weichenthal</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Michael Weichenthal</a></div></div><div id="_co_idm139812599969344"><h3 class="no_margin">Jonathan Barker</h3><p><sup>24</sup>Division of Genetics and Molecular Medicine, St John’s Institute of Dermatology, Kings College London, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Barker%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Jonathan Barker</a></div></div><div id="_co_idm139812599967360"><h3 class="no_margin">James&nbsp;T. Elder</h3><p><sup>19</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109-5675, USA</p><p><sup>28</sup>Ann Arbor Veterans Affairs Hospital, Ann Arbor, MI 48105, USA</p><div>Find articles by <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">James&nbsp;T. Elder</a></div></div><div id="_co_idm139812606089008"><h3 class="no_margin">Stephan Weidinger</h3><p><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Stephan Weidinger</a></div></div><div id="_co_idm139812607912432"><h3 class="no_margin">Heather&nbsp;J. Cordell</h3><p><sup>29</sup>Institute of Genetic Medicine, Newcastle University, Newcastle upon Tyne NE1 3BZ, UK</p><div>Find articles by <a href="/pubmed/?term=Cordell%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Heather&nbsp;J. Cordell</a></div></div><div id="_co_idm139812605659968"><h3 class="no_margin">Sara&nbsp;J. Brown</h3><p><sup>30</sup>Dermatology and Genetic Medicine, Medical Research Institute, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK</p><div>Find articles by <a href="/pubmed/?term=Brown%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25574825">Sara&nbsp;J. Brown</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139812598840496_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139812598840496_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139812598840496_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139812598840496_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Dermatology, Allergology, and Venereology, University Hospital Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Medicine II - Grosshadern, Ludwig-Maximilians-University (LMU), 81377 Munich, Germany</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Gastroenterology, Hepatology and Endocrinology, Charité, Campus Mitte, 10117 Berlin, Germany</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Neuropsychiatric Genetics Research Group, Department of Psychiatry and Institute of Molecular Medicine, Trinity College Dublin, Dublin 2, Ireland</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University of Kiel, 24105 Kiel, Germany</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Max-Delbrück-Centrum (MDC) for Molecular Medicine, Berlin-Buch, 13092 Berlin, Germany</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Pediatric Allergy, Experimental and Clinical Research Center, Charité Universitätsmedizin Berlin, 10117 Berlin, Germany</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Research Unit of Molecular Epidemiology, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Institute of Epidemiology II, Helmholtz Zentrum München - German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Hannover Unified Biobank, Hannover Medical School, 30625 Hannover, Germany</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Paediatric Dermatology, Our Lady’s Children’s Hospital, Crumlin, Dublin 12, Ireland</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>National Children’s Research Centre, Dublin 12, Ireland</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Department of Clinical Medicine, Trinity College Dublin, Dublin 2, Ireland</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Department of Pediatric Pneumology and Allergy, University Children's Hospital Regensburg (KUNO), 93042 Regensburg, Germany</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Institute of Epidemiology and PopGen Biobank, University Hospital Schleswig-Holstein, 24105 Kiel, Germany</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Dermatology and Genetic Medicine, College of Life Sciences and College of Medicine, Dentistry &amp; Nursing, University of Dundee, Dundee DD1 5EH, UK</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Cognitive Genetics and Therapy Group, School of Psychology and Discipline of Biochemistry, National University of Ireland, Galway, Ireland</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Department of Dermatology, University of Michigan, Ann Arbor, MI 48109-5675, USA</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Institute of Human Genetics, University of Bonn, 53127 Bonn, Germany</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, 53127 Bonn, Germany</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Department of Dermatology and Allergy, University of Bonn Medical Center, 53105 Bonn, Germany</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Department of Internal Medicine, University Medical Center Schleswig-Holstein, 24105 Kiel, Germany</div><div class="fm-affl" id="aff24" lang="en"><sup>24</sup>Division of Genetics and Molecular Medicine, St John’s Institute of Dermatology, Kings College London, London SE1 9RT, UK</div><div class="fm-affl" id="aff25" lang="en"><sup>25</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, 80539 Munich, Germany</div><div class="fm-affl" id="aff26" lang="en"><sup>26</sup>Queen Mary University of London, Barts and the London School of Medicine and Dentistry, London E1 4NS, UK</div><div class="fm-affl" id="aff27" lang="en"><sup>27</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan School of Public Health, Ann Arbor, MI 48109-5314, USA</div><div class="fm-affl" id="aff28" lang="en"><sup>28</sup>Ann Arbor Veterans Affairs Hospital, Ann Arbor, MI 48105, USA</div><div class="fm-affl" id="aff29" lang="en"><sup>29</sup>Institute of Genetic Medicine, Newcastle University, Newcastle upon Tyne NE1 3BZ, UK</div><div class="fm-affl" id="aff30" lang="en"><sup>30</sup>Dermatology and Genetic Medicine, Medical Research Institute, Ninewells Hospital and Medical School, University of Dundee, Dundee DD1 9SY, UK</div><div><span class="contrib-email" id="idm139812606089008">Stephan Weidinger: <a href="mailto:dev@null" data-email="ed.leik-inu.ygolotamred@regnidiews" class="oemail">ed.leik-inu.ygolotamred@regnidiews</a>; </span> <span class="contrib-email" id="idm139812605659968">Sara&nbsp;J. Brown: <a href="mailto:dev@null" data-email="ku.ca.eednud@nworb.j.s" class="oemail">ku.ca.eednud@nworb.j.s</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ed.leik-inu.ygolotamred@regnidiews" class="oemail">ed.leik-inu.ygolotamred@regnidiews</a></div><div id="cor2"><sup>∗∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ku.ca.eednud@nworb.j.s" class="oemail">ku.ca.eednud@nworb.j.s</a></div><div id="fn1"><sup>31</sup>These authors contributed equally to this work</div><div id="fn2"><sup>32</sup>These authors contributed equally to this work</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139812598840496_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Sep 5; Accepted 2014 Dec 5.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139812598840496_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2015 The Authors</div><div class="license half_rhythm">This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/3.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div><strong>This article has been corrected. </strong>See <a href="/pmc/articles/PMC4678432/">Am J Hum Genet. 2015 December 03; 97(6): 933</a>.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC4289690/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="4" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Supplemental Consortia Information, Figures S1–S6, and Tables S1–S4 and S6</div><div><a href="/pmc/articles/PMC4289690/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;677F3AF1-958B-46FB-8AF4-525FC0582459</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S5. Results of CCMA and MNM Analyses Showing Significance Levels and Comparison with Known Susceptibility Loci for AD and Psoriasis Lead SNP defined by clumping procedure (Purcell, S. et&nbsp;al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81, 559-75 2007); genomewide significant results defined as Tmax&gt;6, BFDP&lt;0.05 with PO=1/999 or PMNM&lt;10-8 are marked in bold; genes and transcripts identified from UCSC Genome Browser Human Feb. 2009 (GRCh37/hg19) Assembly accessed 21 March 2014.</div><div><a href="/pmc/articles/PMC4289690/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">mmc2.xlsx</a><span style="color:gray"> (79K)</span></div><div class="small guid">GUID:&nbsp;5C5FB052-1EC0-4491-978A-19D22E7192FA</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S7. Gene Expression Data in Psoriasis and AD Relating to Candidate Genes Identified by Comparative Analysis Gene expression data available from published and publically available data sets are summarized for candidate genes.</div><div><a href="/pmc/articles/PMC4289690/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">mmc3.xlsx</a><span style="color:gray"> (65K)</span></div><div class="small guid">GUID:&nbsp;D7797294-3B5C-4809-8E54-5FD6CD0C4867</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S2. Article plus Supplemental Data</div><div><a href="/pmc/articles/PMC4289690/bin/mmc4.pdf" data-ga-action="click_feat_suppl">mmc4.pdf</a><span style="color:gray"> (4.2M)</span></div><div class="small guid">GUID:&nbsp;081CD3A3-0A90-441F-9624-593FB65F3814</div></div></dd></dl></div><div id="idm139812601480432" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139812601480432title">Abstract</h2><!--article-meta--><div><p id="__p3" class="p p-first-last">Atopic dermatitis and psoriasis are the two most common immune-mediated inflammatory disorders affecting the skin. Genome-wide studies demonstrate a high degree of genetic overlap, but these diseases have mutually exclusive clinical phenotypes and opposing immune mechanisms. Despite their prevalence, atopic dermatitis and psoriasis very rarely co-occur within one individual. By utilizing genome-wide association study and ImmunoChip data from &gt;19,000 individuals and methodologies developed from meta-analysis, we have identified opposing risk alleles at shared loci as well as independent disease-specific loci within the epidermal differentiation complex (chromosome 1q21.3), the Th2 locus control region (chromosome 5q31.1), and the major histocompatibility complex (chromosome 6p21–22). We further identified previously unreported pleiotropic alleles with opposing effects on atopic dermatitis and psoriasis risk in <em>PRKRA</em> and <em>ANXA6/TNIP1</em>. In contrast, there was no evidence for shared loci with effects operating in the same direction on both diseases. Our results show that atopic dermatitis and psoriasis have distinct genetic mechanisms with opposing effects in shared pathways influencing epidermal differentiation and immune response. The statistical analysis methods developed in the conduct of this study have produced additional insight from previously published data sets. The approach is likely to be applicable to the investigation of the genetic basis of other complex traits with overlapping and distinct clinical features.</p></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p4" class="p p-first">Atopic dermatitis (AD, synonymous with eczema [MIM <a href="/omim/603165" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603165</a>]) and psoriasis (psoriasis vulgaris [MIM <a href="/omim/177900" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">177900</a>]) are the two most common chronic inflammatory skin conditions. They are associated with a significantly reduced quality of life and multiple comorbidities.<sup><a href="#bib1" rid="bib1 bib2" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">1,2</a></sup> Both diseases result from the interaction of genetic and environmental factors and are characterized by epidermal defects as well as local and systemic immunological abnormalities. Despite a lifetime prevalence of ∼2% for psoriasis and 10%–20% for AD,<sup><a href="#bib3" rid="bib3 bib4" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">3,4</a></sup> these diseases rarely co-occur within an individual<sup><a href="#bib5" rid="bib5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">5</a></sup>—an observation attributed to opposing immune response patterns.<sup><a href="#bib6" rid="bib6" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">6</a></sup> However, it has been reported that both Th1-cell-dominated autoimmune and Th2-cell-dominated allergic diseases aggregate within families<sup><a href="#bib7" rid="bib7" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">7</a></sup> and that parental psoriasis might increase the risk of AD in offspring.<sup><a href="#bib8" rid="bib8" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">8</a></sup> Furthermore, genome-wide linkage and association studies have shown genetic risk loci in each disease that map to similar regions of the genome. The epidermal differentiation complex (EDC) on chromosome 1q21.3 includes AD and psoriasis risk loci in close proximity.<sup><a href="#bib9" rid="bib9 bib10 bib11 bib12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">9–12</a></sup> Null mutations in the gene encoding filaggrin (<em>FLG</em> [MIM <a href="/omim/135940" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">135940</a>]) represent the strongest known risk factor for AD<sup><a href="#bib13" rid="bib13 bib14" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">13,14</a></sup> and account for at least a proportion of AD risk within the EDC, but <em>FLG</em>-null mutations are not associated with psoriasis.<sup><a href="#bib15" rid="bib15 bib16" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">15,16</a></sup> A deletion of the late cornified envelope genes <em>LCE3B</em>-<em>LCE3C</em> (MIM <a href="/omim/612614" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612614</a>, <a href="/omim/612615" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612615</a>) represents a genetic substrate for psoriasis within the EDC,<sup><a href="#bib17" rid="bib17 bib18" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">17,18</a></sup> but this deletion is not associated with AD.<sup><a href="#bib19" rid="bib19" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">19</a></sup> The cytokine cluster encoded at 5q23.1–5q31.1 includes variants showing association with both diseases,<sup><a href="#bib10" rid="bib10 bib20 bib21" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">10,20,21</a></sup> and an intergenic region of chromosome 20q13.2 has also shown association with both AD and psoriasis.<sup><a href="#bib22" rid="bib22 bib23" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">22,23</a></sup> Finally, a recent genome-wide association study (GWAS) on AD identified a strong association within the margins of the major histocompatibility complex (MHC)<sup><a href="#bib20" rid="bib20" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">20</a></sup> on chromosome 6p21.3, less than 2.4 kb from a variant associated with HLA-Cw6 (MIM <a href="/omim/142840" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142840</a>),<sup><a href="#bib24" rid="bib24" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">24</a></sup> the strongest known psoriasis-risk locus.</p><p id="__p5" class="p p-last">In order to gain insight into overlapping and specific genetic mechanisms, we systematically compared and contrasted AD and psoriasis via analytical techniques developed from meta-analysis.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Subjects and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Study Subjects</h3><p id="__p6" class="p p-first">Genome-wide genotype data were obtained on samples from six case-control cohorts (three each of AD and psoriasis), totaling 2,262 AD and 4,489 psoriasis case subjects and 12,333 control subjects (<a href="#app2" rid="app2" class=" sec">Table S1</a> available online).</p><p id="__p7">The German AD case subjects were recruited from tertiary dermatology clinics at Munich, as part of the GENEVA study, University of Kiel, University of Bonn, and the University Children’s Hospital of Charité Universitätsmedizin Berlin. AD was diagnosed by experienced dermatologists and/or pediatricians according to the UK Diagnostic Criteria.<sup><a href="#bib25" rid="bib25" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">25</a></sup> German control subjects were obtained from the PopGen biorepository,<sup><a href="#bib26" rid="bib26" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">26</a></sup> the population-based KORA study in southern Germany,<sup><a href="#bib27" rid="bib27" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">27</a></sup> and the German part of ISAAC II to assess the prevalence of asthma and allergies in schoolchildren.<sup><a href="#bib28" rid="bib28" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">28</a></sup> The Irish AD case collection was recruited from the secondary and tertiary pediatric dermatology clinic at Our Lady’s Children’s Hospital, Crumlin, Dublin. Irish control individuals were obtained from healthy adult blood donors as part of the Trinity Biobank, Dublin.<sup><a href="#bib29" rid="bib29" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">29</a></sup></p><p id="__p8">The German psoriasis case subjects were recruited from the tertiary dermatology clinic at the University of Kiel and German controls were again obtained from the PopGen biorepository and the KORA study (independent from those used as controls for AD). The British psoriasis case-control study is part of the Welcome Trust Case Control Consortium 2<sup><a href="#bib24" rid="bib24" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">24</a></sup> and the US psoriasis study has been described elsewhere.<sup><a href="#bib21" rid="bib21" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">21</a></sup></p><p id="__p9">ImmunoChip data on 2,425 AD case subjects, 3,580 psoriasis case subjects, and 9,061 control subjects were obtained from previous studies,<sup><a href="#bib11" rid="bib11 bib12" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11,12</a></sup> including data on a subset of case and control individuals also analyzed by GWAS. Results of analysis of the four most prevalent <em>FLG</em> (RefSeq accession number <a id="__tag_589567605" class="tag_hotlink" href="/nuccore/NM_002016.1" ref="/nuccore/NM_002016.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"NM_002016.1","term_id":"60097901","term_text":"NM_002016.1"}}</span>NM_002016.1</a>) loss-of-function mutations were obtained for a total of 2,865 case subjects and 5,540 control subjects as data generated for previous studies;<sup><a href="#bib11" rid="bib11 bib20" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">11,20</a></sup> the <em>FLG</em> mutations in these analyses are as follows: p.Arg501<sup>∗</sup> (c.1501C&gt;T), p.Ser761Cysfs<sup>∗</sup>36 (c.2282_2285del), p.Arg2447<sup>∗</sup> (c.7339C&gt;T), and p.Ser3247<sup>∗</sup> (c.9740C&gt;A) (R501X, 2282del4, R2447X, and S3247X, respectively).</p><p id="__p10" class="p p-last">The institutional review board in each contributing center approved these studies. All participants (or their parents or guardians) gave written informed consent.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Study Design</h3><p id="__p11" class="p p-first-last">The study design is summarized in <a href="/pmc/articles/PMC4289690/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4289690/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812604849760" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" src="/pmc/articles/PMC4289690/bin/gr1.jpg"></div></a><div id="largeobj_idm139812604849760" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4289690/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a target="object" href="/pmc/articles/PMC4289690/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4289690/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p63">Study Design</p><p id="__p64">Abbreviations are as follows: CCMA, case control meta-analysis; MANTRA, meta-analysis of trans-ethnic association studies; BFD, Bayesian false discovery; PO, prior odds; <sup>∗</sup>conditional analysis for the MHC was also carried out with imputed classical HLA-allele (detailed in the <a href="#sec2" rid="sec2" class=" sec">Subjects and Methods</a>).</p></div></div></div></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Quality Control</h3><p id="__p12" class="p p-first-last">Quality control and standard GWAS analysis of genotyped single-nucleotide variants (SNVs) was carried out with PLINK<sup><a href="#bib30" rid="bib30" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">30</a></sup> and R. Samples with extensive missing data (rate &gt;5%), excess of heterozygosity or homozygosity, and discrepant gender determined on the basis of average X-chromosomal heterozygosity compared to the gender recorded in the database were excluded. We then examined identity-by-state (IBS) sharing and estimated identity-by-decent (IBD) on a pruned SNV set between all pairs of individuals and deleted resulting duplicates or closely related samples with PI_HAT &gt; 0.1875 (halfway between expected IBD for third- and second-degree relatives). Multidimensional scaling (MDS) of the pairwise IBS matrix was carried out to identify and delete outliers of unusual ancestry and to calculate genome-wide principal-component scores for each individual. We excluded 894 samples because of SNVs showing a missing rate of &gt;5%, deviation of Hardy-Weinberg equilibrium p<sub>HWE</sub> &lt; 10<sup>−8</sup>, or minor allele frequency (MAF) &lt;5% (summarized in <a href="#app2" rid="app2" class=" sec">Table S2</a>). After quality control, the resulting SNVs and samples were analyzed for association via logistic regression with age, sex, and principal-component scores as covariates. Results from each panel were investigated to determine whether established GWAS loci were identified for the respective trait of interest, and genomic control inflation factors were calculated.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Imputation of SNVs and Classical HLA Alleles</h3><p id="__p13" class="p p-first">Any SNVs showing significant association were checked (e.g., by visual inspection of the intensity cluster plots and investigation of consistency of LD with surrounding markers) and those SNVs deemed unreliable were removed. The final data sets of high-quality SNVs were prephased with SHAPEIT<sup><a href="#bib31" rid="bib31" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">31</a></sup> and subsequently used to perform imputation with IMPUTE2,<sup><a href="#bib32" rid="bib32" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">32</a></sup> the 1000 Genomes reference panel (integrated variant set, release March 2012).<sup><a href="#bib33" rid="bib33" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">33</a></sup> In the Irish AD collection (<a href="#app2" rid="app2" class=" sec">Table S1</a>), case and control subjects were genotyped on different platforms, and therefore only the 131,692 SNVs in common between the platforms were used to inform imputation.</p><p id="__p14">Postimputation SNVs with low imputation quality (info score &lt; 0.4), call rate &lt;95%, deviation from p<sub>HWE</sub> &lt; 10<sup>−8</sup>, or MAF &lt; 5% were excluded. A final data set of approximately 5.2 million SNVs in 2,079 AD case subjects, 3,867 control subjects, 4,212 psoriasis case subjects, and 8,032 control subjects were eligible for subsequent analysis (<a href="#app2" rid="app2" class=" sec">Table S3</a>).</p><p id="__p15" class="p p-last">Classical alleles for <em>HLA-A</em>, <em>HLA-B</em>, and <em>HLA-C</em> were imputed for each case-control cohort separately by HLA<sup>∗</sup>IMP<sup><a href="#bib34" rid="bib34 bib35" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">34,35</a></sup> and best guess genotypes with probability &gt;0.9. Additional classical <em>HLA-DQA1</em>, <em>HLA-DQB1</em>, and <em>HLA-DRB1</em> alleles were imputed in each case-control cohort with the exception of the Irish samples, in which there were insufficient informative SNPs. Alleles with a frequency &gt;1% were put forward for analysis. For each individual, alleles were coded as having no, one, or two copies of the respective allele via allele probability &gt;0.9. We obtained high-quality data at the four-digit level with call rates of 92%–100% and accuracy of 92%–98%.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Statistical Analysis</h3><p id="__p16" class="p p-first">Meta-GWAS was performed on each disease, via standard methodologies. To analyze these findings further, we developed two different meta-analysis-based approaches to filter SNVs and model the contrasting effects in each disease. The first was a compare and contrast meta-analysis (CCMA) approach inspired by a subset-based method.<sup><a href="#bib36" rid="bib36" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">36</a></sup> The second used transethnic meta-analysis implemented in the MANTRA software,<sup><a href="#bib37" rid="bib37" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">37</a></sup> combining all six studies by using prior clustering to reflect the ethnic difference and the disease type. The MHC region was reserved for separate analysis because of its unique and complex variability and patterns of strong linkage disequilibrium (LD).</p><p id="__p17">The CCMA approach is based on an adaptation of an idea of Bhattacharjee et&nbsp;al.,<sup><a href="#bib36" rid="bib36" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">36</a></sup> who modeled association with heterogeneous traits. With METAL,<sup><a href="#bib38" rid="bib38" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">38</a></sup> we calculated z-scores signed positive or negative with respect to the same reference allele for two meta-analyses, <em>T</em><sub>1</sub>, combining AD studies only, and <em>T</em><sub>2</sub>, combining psoriasis studies only. We then calculated the overall test statistic <em>T</em><sub><em>max</em></sub> with the formula <em>T</em><sub><em>m</em><em>a</em><em>x</em></sub> =&nbsp;max(|<em>T</em><sub>1</sub>|,&nbsp;|<em>T</em><sub>2</sub>|,&nbsp;|<em>T</em><sub>12<em>s</em><em>h</em><em>a</em><em>r</em><em>e</em><em>d</em></sub>|,&nbsp;|<em>T</em><sub>12<em>o</em><em>p</em><em>p</em><em>o</em><em>s</em><em>i</em><em>n</em><em>g</em></sub>|), where <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" altimg="si2.gif" overflow="scroll"><mrow><msub><mi>T</mi><mrow><mn>12</mn><mi>s</mi><mi>h</mi><mi>a</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>T</mi><mn>1</mn></msub><mo>+</mo><msub><mi>T</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="M2" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="msub" id="MathJax-Span-4"><span style="display: inline-block; position: relative; width: 3.497em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mrow" id="MathJax-Span-6"><span class="mn" id="MathJax-Span-7" style="font-size: 70.7%; font-family: MathJax_Main;">12</span><span class="mi" id="MathJax-Span-8" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-9" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">h</span><span class="mi" id="MathJax-Span-10" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">a</span><span class="mi" id="MathJax-Span-11" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span class="mi" id="MathJax-Span-12" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">e</span><span class="mi" id="MathJax-Span-13" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-14" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mrow" id="MathJax-Span-15" style="padding-left: 0.278em;"><span class="mo" id="MathJax-Span-16" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-17"><span class="msub" id="MathJax-Span-18"><span style="display: inline-block; position: relative; width: 1.013em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-19" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mn" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-21" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-22" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.013em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-23" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mn" id="MathJax-Span-24" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-25" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-26" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="msqrt" id="MathJax-Span-27"><span style="display: inline-block; position: relative; width: 1.333em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.45em, 4.132em, -1000em); top: -3.977em; left: 0.833em;"><span class="mrow" id="MathJax-Span-28"><span class="mn" id="MathJax-Span-29" style="font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.93em, 1000.5em, 1.304em, -1000em); top: -1.929em; left: 0.833em;"><span style="display: inline-block; overflow: hidden; vertical-align: -0.065em; border-top: 1.3px solid; width: 0.5em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span><span style="position: absolute; clip: rect(3.022em, 1000.85em, 4.332em, -1000em); top: -4.022em; left: 0em;"><span style="font-family: MathJax_Main;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" altimg="si2.gif" overflow="scroll"><mrow><msub><mi>T</mi><mrow><mn>12</mn><mi>s</mi><mi>h</mi><mi>a</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>T</mi><mn>1</mn></msub><mo>+</mo><msub><mi>T</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></script> and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" altimg="si3.gif" overflow="scroll"><mrow><msub><mi>T</mi><mrow><mn>12</mn><mi>o</mi><mi>p</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>T</mi><mn>1</mn></msub><mo>−</mo><msub><mi>T</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="M3" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-31"><span class="mrow" id="MathJax-Span-32"><span class="msub" id="MathJax-Span-33"><span style="display: inline-block; position: relative; width: 4.103em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-34" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mrow" id="MathJax-Span-35"><span class="mn" id="MathJax-Span-36" style="font-size: 70.7%; font-family: MathJax_Main;">12</span><span class="mi" id="MathJax-Span-37" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">o</span><span class="mi" id="MathJax-Span-38" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">p</span><span class="mi" id="MathJax-Span-39" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">p</span><span class="mi" id="MathJax-Span-40" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">o</span><span class="mi" id="MathJax-Span-41" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-42" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span><span class="mi" id="MathJax-Span-44" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mrow" id="MathJax-Span-46" style="padding-left: 0.278em;"><span class="mo" id="MathJax-Span-47" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-48"><span class="msub" id="MathJax-Span-49"><span style="display: inline-block; position: relative; width: 1.013em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-50" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mn" id="MathJax-Span-51" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-52" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-53" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.013em; height: 0px;"><span style="position: absolute; clip: rect(3.145em, 1000.7em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-54" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.584em;"><span class="mn" id="MathJax-Span-55" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-56" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-57" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="msqrt" id="MathJax-Span-58"><span style="display: inline-block; position: relative; width: 1.333em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.45em, 4.132em, -1000em); top: -3.977em; left: 0.833em;"><span class="mrow" id="MathJax-Span-59"><span class="mn" id="MathJax-Span-60" style="font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.93em, 1000.5em, 1.304em, -1000em); top: -1.929em; left: 0.833em;"><span style="display: inline-block; overflow: hidden; vertical-align: -0.065em; border-top: 1.3px solid; width: 0.5em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span><span style="position: absolute; clip: rect(3.022em, 1000.85em, 4.332em, -1000em); top: -4.022em; left: 0em;"><span style="font-family: MathJax_Main;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M3" altimg="si3.gif" overflow="scroll"><mrow><msub><mi>T</mi><mrow><mn>12</mn><mi>o</mi><mi>p</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>T</mi><mn>1</mn></msub><mo>−</mo><msub><mi>T</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></script>. We categorized the effect of each SNV as corresponding to an effect on AD only, to an effect on psoriasis only, to a shared effect (in the same direction on AD and psoriasis), or to opposing effects, according to which of the four test statistics (|<em>T</em><sub>1</sub>|,&nbsp;|<em>T</em><sub>2</sub>|,&nbsp;|<em>T</em><sub>12<em>s</em><em>h</em><em>a</em><em>r</em><em>e</em><em>d</em></sub>|,&nbsp;|<em>T</em><sub>12<em>o</em><em>p</em><em>p</em><em>o</em><em>s</em><em>i</em><em>n</em><em>g</em></sub>|) was the largest. In order to derive a p value for <em>T</em><sub><em>max</em></sub>, we worked out an empirical null distribution by simulating 10,000,000 realizations of two normally distributed random variables, <em>Z</em><sub>1</sub> and <em>Z</em><sub>2</sub>. Then we calculated <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" altimg="si5.gif" overflow="scroll"><mrow><msub><mi>Z</mi><mrow><mn>12</mn><mi>s</mi><mi>h</mi><mi>a</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>Z</mi><mn>1</mn></msub><mo>+</mo><msub><mi>Z</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="M5" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-62"><span class="mrow" id="MathJax-Span-63"><span class="msub" id="MathJax-Span-64"><span style="display: inline-block; position: relative; width: 3.596em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-65" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mrow" id="MathJax-Span-66"><span class="mn" id="MathJax-Span-67" style="font-size: 70.7%; font-family: MathJax_Main;">12</span><span class="mi" id="MathJax-Span-68" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-69" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">h</span><span class="mi" id="MathJax-Span-70" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">a</span><span class="mi" id="MathJax-Span-71" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">r</span><span class="mi" id="MathJax-Span-72" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">e</span><span class="mi" id="MathJax-Span-73" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-74" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mrow" id="MathJax-Span-75" style="padding-left: 0.278em;"><span class="mo" id="MathJax-Span-76" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-77"><span class="msub" id="MathJax-Span-78"><span style="display: inline-block; position: relative; width: 1.112em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-79" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mn" id="MathJax-Span-80" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-81" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-82" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.112em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-83" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mn" id="MathJax-Span-84" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-85" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-86" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="msqrt" id="MathJax-Span-87"><span style="display: inline-block; position: relative; width: 1.333em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.45em, 4.132em, -1000em); top: -3.977em; left: 0.833em;"><span class="mrow" id="MathJax-Span-88"><span class="mn" id="MathJax-Span-89" style="font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.93em, 1000.5em, 1.304em, -1000em); top: -1.929em; left: 0.833em;"><span style="display: inline-block; overflow: hidden; vertical-align: -0.065em; border-top: 1.3px solid; width: 0.5em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span><span style="position: absolute; clip: rect(3.022em, 1000.85em, 4.332em, -1000em); top: -4.022em; left: 0em;"><span style="font-family: MathJax_Main;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M5" altimg="si5.gif" overflow="scroll"><mrow><msub><mi>Z</mi><mrow><mn>12</mn><mi>s</mi><mi>h</mi><mi>a</mi><mi>r</mi><mi>e</mi><mi>d</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>Z</mi><mn>1</mn></msub><mo>+</mo><msub><mi>Z</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></script>, <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" altimg="si6.gif" overflow="scroll"><mrow><msub><mi>Z</mi><mrow><mn>12</mn><mi>o</mi><mi>p</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>Z</mi><mn>1</mn></msub><mo>−</mo><msub><mi>Z</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="M6" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-91"><span class="mrow" id="MathJax-Span-92"><span class="msub" id="MathJax-Span-93"><span style="display: inline-block; position: relative; width: 4.202em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-94" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mrow" id="MathJax-Span-95"><span class="mn" id="MathJax-Span-96" style="font-size: 70.7%; font-family: MathJax_Main;">12</span><span class="mi" id="MathJax-Span-97" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">o</span><span class="mi" id="MathJax-Span-98" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">p</span><span class="mi" id="MathJax-Span-99" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">p</span><span class="mi" id="MathJax-Span-100" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">o</span><span class="mi" id="MathJax-Span-101" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-103" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span><span class="mi" id="MathJax-Span-104" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-105" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mrow" id="MathJax-Span-106" style="padding-left: 0.278em;"><span class="mo" id="MathJax-Span-107" style="font-family: MathJax_Main;">(</span><span class="mrow" id="MathJax-Span-108"><span class="msub" id="MathJax-Span-109"><span style="display: inline-block; position: relative; width: 1.112em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-110" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mn" id="MathJax-Span-111" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-112" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-113" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.112em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.72em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-114" style="font-family: MathJax_Math; font-style: italic;">Z<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.04em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.683em;"><span class="mn" id="MathJax-Span-115" style="font-size: 70.7%; font-family: MathJax_Main;">2</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-116" style="font-family: MathJax_Main;">)</span></span><span class="mo" id="MathJax-Span-117" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="msqrt" id="MathJax-Span-118"><span style="display: inline-block; position: relative; width: 1.333em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.45em, 4.132em, -1000em); top: -3.977em; left: 0.833em;"><span class="mrow" id="MathJax-Span-119"><span class="mn" id="MathJax-Span-120" style="font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.93em, 1000.5em, 1.304em, -1000em); top: -1.929em; left: 0.833em;"><span style="display: inline-block; overflow: hidden; vertical-align: -0.065em; border-top: 1.3px solid; width: 0.5em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span><span style="position: absolute; clip: rect(3.022em, 1000.85em, 4.332em, -1000em); top: -4.022em; left: 0em;"><span style="font-family: MathJax_Main;">√</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M6" altimg="si6.gif" overflow="scroll"><mrow><msub><mi>Z</mi><mrow><mn>12</mn><mi>o</mi><mi>p</mi><mi>p</mi><mi>o</mi><mi>s</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub><mo>=</mo><mrow><mo>(</mo><mrow><msub><mi>Z</mi><mn>1</mn></msub><mo>−</mo><msub><mi>Z</mi><mn>2</mn></msub></mrow><mo>)</mo></mrow><mo>/</mo><msqrt><mn>2</mn></msqrt></mrow></math></script>, and <em>Z</em><sub><em>m</em><em>a</em><em>x</em></sub> =&nbsp;max(|<em>Z</em><sub>1</sub>|,&nbsp;|<em>Z</em><sub>2</sub>|,&nbsp;|<em>Z</em><sub>12<em>s</em><em>h</em><em>a</em><em>r</em><em>e</em><em>d</em></sub>|,&nbsp;|<em>Z</em><sub>12<em>o</em><em>p</em><em>p</em><em>o</em><em>s</em><em>i</em><em>n</em><em>g</em></sub>|). The emprical p values can be derived as <em>P</em><sub><em>e</em><em>m</em><em>p</em></sub> =&nbsp;(#(<em>Z</em><sub><em>m</em><em>a</em><em>x</em></sub>&nbsp;&gt;&nbsp;<em>T</em><sub><em>m</em><em>a</em><em>x</em></sub>) +&nbsp;1)/(# <em>s</em><em>i</em><em>m</em><em>u</em><em>l</em><em>a</em><em>t</em><em>i</em><em>o</em><em>n</em><em>s</em> +&nbsp;1).</p><p id="__p18">In a separate simulation of 1,000,000,000 replicates, we derived a calibration curve for the p values and found it suitable up to a p value of 10<sup>−9</sup>. Hence with the calibration curve we can derive <em>Z</em><sub><em>max</em></sub> thresholds corresponding to standard genome-wide “suggestive” (10<sup>−5</sup>) and genome-wide “significant” (10<sup>−8</sup>) thresholds, corresponding to <em>T</em><sub><em>max</em></sub> values of approximately 4.7 and 6.0, respectively (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S1</a>).</p><p id="__p19">In the second approach we used the MANTRA software<sup><a href="#bib37" rid="bib37" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">37</a></sup> developed for transethnic meta-analysis. MANTRA uses a Bayesian partition model for grouping studies according to their ethnicity. We adopted this idea and worked out a prior distribution to cluster studies according to both our phenotypes of interest and the genetic distance between the studies derived from our MDS analysis based on the pairwise IBS matrix: <em>D</em><sub><em>Total</em></sub> = <em>D</em><sub><em>Disease</em></sub> + <em>D</em><sub><em>Ethnicity</em></sub>, where <em>D</em><sub><em>Ethnicity</em></sub> is a diagonal matrix of Euclidean distances between study centers. To distinguish the two diseases (psoriasis and AD), we&nbsp;set&nbsp;the corresponding cells of the <em>D</em><sub><em>Disease</em></sub> matrix to <em>D</em><sub><em>ij</em></sub> = 2&nbsp;× max(<em>D</em><sub><em>Ethnicity</em></sub>) and to account for the different subphenotype in AD (AD in general versus childhood AD), we set the corresponding cells of the <em>D</em><sub><em>Disease</em></sub> matrix to <em>D</em><sub><em>ij</em></sub> = max(<em>D</em><sub><em>Ethnicity</em></sub>), resulting in the prior components shown in <a href="#app2" rid="app2" class=" sec">Table S4</a>.</p><p id="__p20">We calibrated the resulting log<sub>10</sub>BF<sub>MANTRA</sub> = log<sub>10</sub>(Bayes Factors) from the MANTRA software in order to find a threshold for filtering SNVs, which were compared with the CCMA top SNVs and subsequently carried forward to multinomial regression modeling. To perform this calibration, we calculated the Bayesian False Discovery Probability proposed by Wakefield<sup><a href="#bib39" rid="bib39" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">39</a></sup> with diverse prior odds (PO) in favor of H<sub>0</sub>:</p><div class="disp-formula" id="ufd1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M9" altimg="si9.gif" overflow="scroll"><mrow><mtext>BFDP</mtext><mo>=</mo><mfrac><mrow><msub><mrow><mtext>BF</mtext></mrow><mrow><mtext>MANTRA</mtext></mrow></msub><mo>×</mo><mtext>PO</mtext></mrow><mrow><mn>1</mn><mo>+</mo><msub><mrow><mtext>BF</mtext></mrow><mrow><mtext>MANTRA</mtext></mrow></msub><mo>×</mo><mtext>PO</mtext></mrow></mfrac><mo>.</mo></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style=""><nobr><span class="math" id="M9" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-122"><span class="mrow" id="MathJax-Span-123"><span class="mtext" id="MathJax-Span-124" style="font-family: MathJax_Main;">BFDP</span><span class="mo" id="MathJax-Span-125" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-126" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 9.229em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.117em, 1007.33em, 4.31em, -1000em); top: -4.654em; left: 50%; margin-left: -3.694em;"><span class="mrow" id="MathJax-Span-127"><span class="msub" id="MathJax-Span-128"><span style="display: inline-block; position: relative; width: 4.706em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1001.32em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-129"><span class="mtext" id="MathJax-Span-130" style="font-family: MathJax_Main;">BF</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 1.361em;"><span class="mrow" id="MathJax-Span-131"><span class="mtext" id="MathJax-Span-132" style="font-size: 70.7%; font-family: MathJax_Main;">MANTRA</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-133" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mtext" id="MathJax-Span-134" style="font-family: MathJax_Main; padding-left: 0.222em;">PO</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.117em, 1009.05em, 4.31em, -1000em); top: -3.291em; left: 50%; margin-left: -4.555em;"><span class="mrow" id="MathJax-Span-135"><span class="mn" id="MathJax-Span-136" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-137" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-138" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 4.706em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1001.32em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mrow" id="MathJax-Span-139"><span class="mtext" id="MathJax-Span-140" style="font-family: MathJax_Main;">BF</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 1.361em;"><span class="mrow" id="MathJax-Span-141"><span class="mtext" id="MathJax-Span-142" style="font-size: 70.7%; font-family: MathJax_Main;">MANTRA</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-143" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mtext" id="MathJax-Span-144" style="font-family: MathJax_Main; padding-left: 0.222em;">PO</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(0.865em, 1009.23em, 1.24em, -1000em); top: -1.305em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 9.229em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.085em;"></span></span></span></span><span class="mo" id="MathJax-Span-145" style="font-family: MathJax_Main;">.</span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-5"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M9" altimg="si9.gif" overflow="scroll"><mrow><mtext>BFDP</mtext><mo>=</mo><mfrac><mrow><msub><mrow><mtext>BF</mtext></mrow><mrow><mtext>MANTRA</mtext></mrow></msub><mo>×</mo><mtext>PO</mtext></mrow><mrow><mn>1</mn><mo>+</mo><msub><mrow><mtext>BF</mtext></mrow><mrow><mtext>MANTRA</mtext></mrow></msub><mo>×</mo><mtext>PO</mtext></mrow></mfrac><mo>.</mo></mrow></math></script></div></div><p>Sensitivity analysis was performed with only the <em>D</em><sub><em>Ethnicity</em></sub> as prior matrix and we observe high correlation (<em>r</em><sup>2</sup> &gt; 0.99) of the top-ranked SNVs (BFDP &lt; 0.05; PO = 99) with our analysis (data not shown).</p><p id="__p21">Finally, we carried forward a filtered set of SNVs from CCMA and MANTRA for modeling via a multinomial regression model, adjusted for sex and the first four genome-wide principal-component scores. The multinomial model involves three outcome categories: the “baseline” category into which all controls are categorized, a “psoriasis” case category, and an “AD” case category (modeled through regression coefficients β<sub>PSO</sub> and β<sub>AD</sub>, respectively). This analysis makes use of individual-level genotypes and is thus more computationally intensive (although arguably more powerful and more statistically satisfactory) than CCMA and MANTRA. We calculated p values for tests that were designed to be sensitive to the following situations: an overall SNV effect (on either or both diseases, in either direction), an individual SNV effect on one disease (but not on the other), a shared SNV effect (operating in the same direction for both diseases), and a contrasting SNV effect (operating in opposing directions between both diseases), by performing Wald tests of the following linear hypotheses:</p><div class="disp-formula" id="ufd2"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M10" altimg="si10.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Overall</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><mtable><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr></mtable><mo>=</mo><mtable><mtr><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><mtable><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr></mtable><mo>≠</mo><mtable><mtr><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style=""><nobr><span class="math" id="M10" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-147"><span class="mrow" id="MathJax-Span-148"><span class="mtable" id="MathJax-Span-149" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 27.421em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.98em, 4.154em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 6.059em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.98em, 4.154em, -1000em); top: -3.977em; left: 0em;"><span class="mtd" id="MathJax-Span-150"><span class="mrow" id="MathJax-Span-151"><span class="mrow" id="MathJax-Span-152"><span class="mtext" id="MathJax-Span-153" style="font-family: MathJax_Main;">Overall</span><span class="mspace" id="MathJax-Span-154" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-155" style="font-family: MathJax_Main;">effect:</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(2.357em, 1009.95em, 5.098em, -1000em); top: -3.977em; left: 6.859em;"><span style="display: inline-block; position: relative; width: 10.02em; height: 0px;"><span style="position: absolute; clip: rect(2.357em, 1009.95em, 5.098em, -1000em); top: -3.977em; left: 0em;"><span class="mtd" id="MathJax-Span-156"><span class="mrow" id="MathJax-Span-157"><span class="mrow" id="MathJax-Span-158"><span class="msub" id="MathJax-Span-159"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-160" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-161" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-162" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="mtable" id="MathJax-Span-163" style="padding-right: 0.167em; padding-left: 0.445em;"><span style="display: inline-block; position: relative; width: 5.146em; height: 0px;"><span style="position: absolute; clip: rect(2.452em, 1005.15em, 5.092em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 5.146em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.15em, 4.326em, -1000em); top: -4.643em; left: 50%; margin-left: -2.573em;"><span class="mtd" id="MathJax-Span-164"><span class="mrow" id="MathJax-Span-165"><span class="mrow" id="MathJax-Span-166"><span class="msub" id="MathJax-Span-167"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-168" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-169"><span class="mi" id="MathJax-Span-170" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-171" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-172" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-173" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-174" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-175" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-176"><span class="mi" id="MathJax-Span-177" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-178" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.117em, 1005.15em, 4.326em, -1000em); top: -3.212em; left: 50%; margin-left: -2.573em;"><span class="mtd" id="MathJax-Span-179"><span class="mrow" id="MathJax-Span-180"><span class="mrow" id="MathJax-Span-181"><span class="msub" id="MathJax-Span-182"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-183" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-184"><span class="mi" id="MathJax-Span-185" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-186" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-187" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-188" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-189" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-190" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-191"><span class="mi" id="MathJax-Span-192" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-193" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-194" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mtable" id="MathJax-Span-195" style="padding-right: 0.167em; padding-left: 0.445em;"><span style="display: inline-block; position: relative; width: 0.5em; height: 0px;"><span style="position: absolute; clip: rect(2.491em, 1000.46em, 4.92em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 0.5em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.46em, 4.154em, -1000em); top: -4.643em; left: 50%; margin-left: -0.25em;"><span class="mtd" id="MathJax-Span-196"><span class="mrow" id="MathJax-Span-197"><span class="mn" id="MathJax-Span-198" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.156em, 1000.46em, 4.154em, -1000em); top: -3.212em; left: 50%; margin-left: -0.25em;"><span class="mtd" id="MathJax-Span-199"><span class="mrow" id="MathJax-Span-200"><span class="mn" id="MathJax-Span-201" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-202" style="font-family: MathJax_Main;">,</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(2.357em, 1009.54em, 5.098em, -1000em); top: -3.977em; left: 17.679em;"><span style="display: inline-block; position: relative; width: 9.742em; height: 0px;"><span style="position: absolute; clip: rect(2.357em, 1009.54em, 5.098em, -1000em); top: -3.977em; left: 0em;"><span class="mtd" id="MathJax-Span-203"><span class="mrow" id="MathJax-Span-204"><span class="mrow" id="MathJax-Span-205"><span class="msub" id="MathJax-Span-206"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-207" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-208" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-209" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="mtable" id="MathJax-Span-210" style="padding-right: 0.167em; padding-left: 0.445em;"><span style="display: inline-block; position: relative; width: 5.146em; height: 0px;"><span style="position: absolute; clip: rect(2.452em, 1005.15em, 5.092em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 5.146em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.15em, 4.326em, -1000em); top: -4.643em; left: 50%; margin-left: -2.573em;"><span class="mtd" id="MathJax-Span-211"><span class="mrow" id="MathJax-Span-212"><span class="mrow" id="MathJax-Span-213"><span class="msub" id="MathJax-Span-214"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-215" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-216"><span class="mi" id="MathJax-Span-217" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-218" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-219" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-220" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-221" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-222" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-223"><span class="mi" id="MathJax-Span-224" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-225" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.117em, 1005.15em, 4.326em, -1000em); top: -3.212em; left: 50%; margin-left: -2.573em;"><span class="mtd" id="MathJax-Span-226"><span class="mrow" id="MathJax-Span-227"><span class="mrow" id="MathJax-Span-228"><span class="msub" id="MathJax-Span-229"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-230" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-231"><span class="mi" id="MathJax-Span-232" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-233" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-234" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-235" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-236" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-237" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-238"><span class="mi" id="MathJax-Span-239" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-240" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-241" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="mtable" id="MathJax-Span-242" style="padding-right: 0.167em; padding-left: 0.445em;"><span style="display: inline-block; position: relative; width: 0.5em; height: 0px;"><span style="position: absolute; clip: rect(2.491em, 1000.46em, 4.92em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 0.5em; height: 0px;"><span style="position: absolute; clip: rect(3.156em, 1000.46em, 4.154em, -1000em); top: -4.643em; left: 50%; margin-left: -0.25em;"><span class="mtd" id="MathJax-Span-243"><span class="mrow" id="MathJax-Span-244"><span class="mn" id="MathJax-Span-245" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.156em, 1000.46em, 4.154em, -1000em); top: -3.212em; left: 50%; margin-left: -0.25em;"><span class="mtd" id="MathJax-Span-246"><span class="mrow" id="MathJax-Span-247"><span class="mn" id="MathJax-Span-248" style="font-family: MathJax_Main;">0</span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-6"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M10" altimg="si10.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Overall</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><mtable><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr></mtable><mo>=</mo><mtable><mtr><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><mtable><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr><mtr><mtd><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub></mrow></mtd></mtr></mtable><mo>≠</mo><mtable><mtr><mtd><mn>0</mn></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable></mrow></mtd></mtr></mtable></mrow></math></script></div></div><div class="disp-formula" id="ufd3"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M11" altimg="si11.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Psoriasis</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0" style=""><nobr><span class="math" id="M11" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-250"><span class="mrow" id="MathJax-Span-251"><span class="mtable" id="MathJax-Span-252" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 20.74em; height: 0px;"><span style="position: absolute; clip: rect(3.16em, 1006.59em, 4.186em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 6.672em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1006.59em, 4.143em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-253"><span class="mrow" id="MathJax-Span-254"><span class="mrow" id="MathJax-Span-255"><span class="mtext" id="MathJax-Span-256" style="font-family: MathJax_Main;">Psoriasis</span><span class="mspace" id="MathJax-Span-257" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-258" style="font-family: MathJax_Main;">effect:</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.16em, 1006.31em, 4.369em, -1000em); top: -3.977em; left: 7.472em;"><span style="display: inline-block; position: relative; width: 6.373em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1006.31em, 4.326em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-259"><span class="mrow" id="MathJax-Span-260"><span class="mrow" id="MathJax-Span-261"><span class="msub" id="MathJax-Span-262"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-263" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-264" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-265" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-266" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-267" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-268"><span class="mi" id="MathJax-Span-269" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-270" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-271" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-272" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-273" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo" id="MathJax-Span-274" style="font-family: MathJax_Main;">,</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.149em, 1006.06em, 4.39em, -1000em); top: -3.977em; left: 14.645em;"><span style="display: inline-block; position: relative; width: 6.095em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1006.06em, 4.347em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-275"><span class="mrow" id="MathJax-Span-276"><span class="mrow" id="MathJax-Span-277"><span class="msub" id="MathJax-Span-278"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-279" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-280" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-281" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-282" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-283" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-284"><span class="mi" id="MathJax-Span-285" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-286" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-287" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-288" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="mn" id="MathJax-Span-289" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-7"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M11" altimg="si11.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Psoriasis</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></script></div></div><div class="disp-formula" id="ufd4"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M12" altimg="si12.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>AD</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0" style=""><nobr><span class="math" id="M12" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-291"><span class="mrow" id="MathJax-Span-292"><span class="mtable" id="MathJax-Span-293" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 17.622em; height: 0px;"><span style="position: absolute; clip: rect(3.149em, 1004.3em, 4.186em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 4.375em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1004.3em, 4.143em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-294"><span class="mrow" id="MathJax-Span-295"><span class="mrow" id="MathJax-Span-296"><span class="mtext" id="MathJax-Span-297" style="font-family: MathJax_Main;">AD</span><span class="mspace" id="MathJax-Span-298" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-299" style="font-family: MathJax_Main;">effect:</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.16em, 1005.89em, 4.369em, -1000em); top: -3.977em; left: 5.175em;"><span style="display: inline-block; position: relative; width: 5.962em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.89em, 4.326em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-300"><span class="mrow" id="MathJax-Span-301"><span class="mrow" id="MathJax-Span-302"><span class="msub" id="MathJax-Span-303"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-304" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-305" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-306" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-307" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-308" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-309"><span class="mi" id="MathJax-Span-310" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-311" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-312" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-313" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo" id="MathJax-Span-314" style="font-family: MathJax_Main;">,</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.149em, 1005.64em, 4.39em, -1000em); top: -3.977em; left: 11.937em;"><span style="display: inline-block; position: relative; width: 5.684em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1005.64em, 4.347em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-315"><span class="mrow" id="MathJax-Span-316"><span class="mrow" id="MathJax-Span-317"><span class="msub" id="MathJax-Span-318"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-319" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-320" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-321" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-322" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-323" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-324"><span class="mi" id="MathJax-Span-325" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-326" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-327" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="mn" id="MathJax-Span-328" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-8"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M12" altimg="si12.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>AD</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></script></div></div><div class="disp-formula" id="ufd5"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M13" altimg="si13.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Shared</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0" style=""><nobr><span class="math" id="M13" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-330"><span class="mrow" id="MathJax-Span-331"><span class="mtable" id="MathJax-Span-332" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 25.891em; height: 0px;"><span style="position: absolute; clip: rect(3.16em, 1005.79em, 4.197em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 5.865em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1005.79em, 4.154em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-333"><span class="mrow" id="MathJax-Span-334"><span class="mrow" id="MathJax-Span-335"><span class="mtext" id="MathJax-Span-336" style="font-family: MathJax_Main;">Shared</span><span class="mspace" id="MathJax-Span-337" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-338" style="font-family: MathJax_Main;">effect:</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.16em, 1009.28em, 4.369em, -1000em); top: -3.977em; left: 6.665em;"><span style="display: inline-block; position: relative; width: 9.352em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1009.28em, 4.326em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-339"><span class="mrow" id="MathJax-Span-340"><span class="mrow" id="MathJax-Span-341"><span class="msub" id="MathJax-Span-342"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-343" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-344" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-345" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-346" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-347" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-348"><span class="mi" id="MathJax-Span-349" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-350" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-351" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-352" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-353" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-354" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-355"><span class="mi" id="MathJax-Span-356" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-357" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-358" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-359" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo" id="MathJax-Span-360" style="font-family: MathJax_Main;">,</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.149em, 1009.03em, 4.39em, -1000em); top: -3.977em; left: 16.817em;"><span style="display: inline-block; position: relative; width: 9.074em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1009.03em, 4.347em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-361"><span class="mrow" id="MathJax-Span-362"><span class="mrow" id="MathJax-Span-363"><span class="msub" id="MathJax-Span-364"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-365" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-366" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-367" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-368" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-369" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-370"><span class="mi" id="MathJax-Span-371" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-372" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-373" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-374" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-375" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-376" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-377"><span class="mi" id="MathJax-Span-378" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-379" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-380" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="mn" id="MathJax-Span-381" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-9"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M13" altimg="si13.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Shared</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></script></div></div><div class="disp-formula" id="ufd6"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M14" altimg="si14.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Opposing</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></span><div class="MathJax_Display MathJax_Processed"><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0" style=""><nobr><span class="math" id="M14" overflow="scroll" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-383"><span class="mrow" id="MathJax-Span-384"><span class="mtable" id="MathJax-Span-385" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 27.005em; height: 0px;"><span style="position: absolute; clip: rect(3.16em, 1006.9em, 4.381em, -1000em); top: -3.977em; left: 0em;"><span style="display: inline-block; position: relative; width: 6.979em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1006.9em, 4.338em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-386"><span class="mrow" id="MathJax-Span-387"><span class="mrow" id="MathJax-Span-388"><span class="mtext" id="MathJax-Span-389" style="font-family: MathJax_Main;">Opposing</span><span class="mspace" id="MathJax-Span-390" style="height: 0em; vertical-align: 0em; width: 0.25em; display: inline-block; overflow: hidden;"></span><span class="mtext" id="MathJax-Span-391" style="font-family: MathJax_Main;">effect:</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.16em, 1009.28em, 4.369em, -1000em); top: -3.977em; left: 7.779em;"><span style="display: inline-block; position: relative; width: 9.352em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1009.28em, 4.326em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-392"><span class="mrow" id="MathJax-Span-393"><span class="mrow" id="MathJax-Span-394"><span class="msub" id="MathJax-Span-395"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-396" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-397" style="font-size: 70.7%; font-family: MathJax_Main;">0</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-398" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-399" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-400" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-401"><span class="mi" id="MathJax-Span-402" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-403" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-404" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-405" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-406" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-407" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-408"><span class="mi" id="MathJax-Span-409" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-410" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-411" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-412" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo" id="MathJax-Span-413" style="font-family: MathJax_Main;">,</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.149em, 1009.03em, 4.39em, -1000em); top: -3.977em; left: 17.931em;"><span style="display: inline-block; position: relative; width: 9.074em; height: 0px;"><span style="position: absolute; clip: rect(3.106em, 1009.03em, 4.347em, -1000em); top: -3.935em; left: 0em;"><span class="mtd" id="MathJax-Span-414"><span class="mrow" id="MathJax-Span-415"><span class="mrow" id="MathJax-Span-416"><span class="msub" id="MathJax-Span-417"><span style="display: inline-block; position: relative; width: 1.26em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.89em, 4.132em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-418" style="font-family: MathJax_Math; font-style: italic;">H<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.057em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.831em;"><span class="mn" id="MathJax-Span-419" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-420" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="msub" id="MathJax-Span-421" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.168em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-422" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.823em; left: 0.566em;"><span class="mrow" id="MathJax-Span-423"><span class="mi" id="MathJax-Span-424" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-425" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.023em;"></span></span><span class="mi" id="MathJax-Span-426" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">O</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-427" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-428" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.757em; height: 0px;"><span style="position: absolute; clip: rect(3.117em, 1000.57em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-429" style="font-family: MathJax_Math; font-style: italic;">β<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.007em;"></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; top: -3.815em; left: 0.566em;"><span class="mrow" id="MathJax-Span-430"><span class="mi" id="MathJax-Span-431" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">A</span><span class="mi" id="MathJax-Span-432" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">D</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span><span class="mo" id="MathJax-Span-433" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="mn" id="MathJax-Span-434" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></span></nobr></span></div><script type="math/mml" id="MathJax-Element-10"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M14" altimg="si14.gif" overflow="scroll"><mrow><mtable columnalign="left"><mtr columnalign="left"><mtd columnalign="left"><mrow><mtext>Opposing</mtext><mspace width="0.25em"></mspace><mtext>effect:</mtext></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>0</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>=</mo><mn>0</mn><mo>,</mo></mrow></mtd><mtd columnalign="left"><mrow><msub><mi>H</mi><mn>1</mn></msub><mo>:</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>S</mi><mi>O</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>A</mi><mi>D</mi></mrow></msub><mo>≠</mo><mn>0</mn></mrow></mtd></mtr></mtable></mrow></math></script></div></div><p></p><p id="__p22">The overall significance of the SNV was assessed through the 2 degree of freedom (df) test of overall effect, which compares the null hypothesis that the SNV has no effect on either psoriasis or AD with the alternative hypothesis that it has an effect on one or both diseases. The other four 1 df tests were used to categorize the effect (in analogy to CCMA) in four categories—AD only, psoriasis only, shared effect, and opposing effects—by categorizing according to the minimum of the p values: p<sub>MNM</sub> = min(p<sub>AD</sub>, p<sub>PSO</sub>, p<sub>SHARED</sub>, p<sub>OPPOSING</sub>). The rationale for the use of the minimum of these 1 df tests for categorization is as follows: if a SNV is associated with one disease but not the other, the test of a nonzero regression coefficient for that disease (even while unnecessarily also allowing for a nonzero coefficient for the other disease, as is done in the psoriasis effect and AD effect tests), should be more powerful than a test that erroneously groups together the coefficients of the associated and the nonassociated disease (as is done in the shared and opposing effect tests). This is on account of the fact that grouping together these coefficients will incur a penalty in terms of increasing the variance, while not incurring any greater expected magnitude of effect since the expected value of the regression coefficient for the nonassociated disease is zero. If, on the other hand, the SNV has effects that operate in the same direction on both diseases, then a test based on adding together these effects (as is done in the shared effect test) should be more powerful than considering each effect on its own, or subtracting one effect from the other (as is done in the opposing effect test), because adding together the coefficients induces the greatest magnitude of effect. Finally, if the SNV has effects that operate in opposite directions in the two diseases, then a test based on subtracting one effect from the other (as is done in the opposing effect test) should be most powerful because it induces the greatest magnitude of effect.</p><p id="__p23" class="p p-last">All analyses if not explicitly stated were carried out with R. For the purposes of this analysis, we distinguished between a shared genetic “region” and a shared genetic “locus.” We arbitrarily designated a shared region as a block of genomic DNA spanning 2 Mb with association signals for both traits. We defined a genetic locus as the lead SNV and all SNVs with <em>r</em><sup>2</sup> &gt; 0.5.</p></div><div id="sec2.6" class="sec sec-last"><h3 id="sec2.6title">Predicted Protein Network Analysis and Gene Ontology Analysis</h3><p id="__p24" class="p p-first-last">Functional protein association networks were investigated in&nbsp;silico and gene ontology analyses were performed with STRING<sub>9.1.</sub></p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">Filtering Variants to Define Risk Effects</h3><p id="__p25" class="p p-first">Quality control and imputation generated 5.2 million SNVs with a minor allele frequency &gt;0.01 for further analysis (<a href="/pmc/articles/PMC4289690/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>). GWASs within each cohort resulted in genomic inflation factors λ between 1.03 and 1.08. Meta-GWAS performed on each disease confirmed previously reported risk loci in AD and psoriasis and illustrated areas of colocalization on chromosomes 1, 5, and 6 (<a href="/pmc/articles/PMC4289690/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>A).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4289690/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812600391088" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" src="/pmc/articles/PMC4289690/bin/gr2.jpg"></div></a><div id="largeobj_idm139812600391088" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4289690/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a target="object" href="/pmc/articles/PMC4289690/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4289690/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p65">Genome-wide Comparison of AD and Psoriasis</p><p id="__p66">(A) Mirrored Manhattan plots showing results of AD meta-GWAS (top) and psoriasis meta-GWAS (bottom).</p><p id="__p67">(B) Comparative analysis of AD and psoriasis in which SNVs are color coded to show AD-specific effect (black), psoriasis-specific effect (red), shared effects defined as alleles operating in the same direction (green), and opposing effects (blue). The genome-wide significance level is marked at p = 0.5&nbsp;× 10<sup>−8</sup> (<em>T<sub>max</sub></em> = 6.0).</p></div></div></div><p id="__p26">Excluding the MHC, 2,210 SNVs were identified with shared (by which we mean alleles having effects operating in the same direction in both diseases), opposing, and disease-specific SNVs with CCMA test statistic <em>T</em><sub><em>max</em></sub> &gt; 4.7. This threshold was defined to correspond to a suggestive significance of p &lt; 10<sup>−5</sup> in order to reduce the probability of false negatives. The 2,210 SNVs were condensed to 142 distinct loci after an LD-based clumping procedure<sup><a href="#bib30" rid="bib30" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">30</a></sup> with the following parameters: distance ≤ 250 kb and <em>r</em><sup>2</sup> ≥ 0.5.</p><p id="__p27" class="p p-last">Analysis with MANTRA revealed 3,304 SNVs with Bayesian false discovery probability (BFDP) &lt; 0.05 with prior odds (PO) 1/99 resulting in 76 distinct loci after clumping. The overlap of CCMA and MANTRA gave 2,183 SNVs and the union of both methods resulted in 3,331 SNVs that were carried forward for multinomial regression modeling (MNM), which was adjusted for sex and the first four genome-wide principal-component scores. The results are displayed in <a href="/pmc/articles/PMC4289690/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure&nbsp;2</span></a>B, in which disease-specific, shared, and opposing loci are coded by color. SNVs showing genome-wide significance in at least one of the three methods of analysis (<em>T</em><sub><em>max</em></sub> &gt; 6, BFDP &lt; 0.05 with PO = 1/999, or p<sub>MNM</sub> &lt; 10<sup>−8</sup>) map to 144 distinct loci (<sup><a href="#bib5" rid="bib5" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">Table S5</a></sup>). Comparison of effect classification (AD, psoriasis, shared, opposing) in CCMA and MNM (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S2</a>) showed an agreement of 94.8% when excluding the MHC region (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S3</a>). For further investigation, we considered only loci containing more than one SNV and an effect classified in the same direction by CCMA and MNM.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Validation of Previously Reported AD- and Psoriasis-Risk Loci</h3><p id="__p28" class="p p-first-last">15 European and 9 Asian loci have previously been reported in GWASs on AD, and 44 European and 9 Asian loci have been reported in association with psoriasis (<a href="#app2" rid="app2" class=" sec">Table S6</a>). In our disease-specific meta-analysis individuals of white European descent, 14 of the European AD loci as well as 43 of the psoriasis loci are replicated. Furthermore, 4 AD and 4 psoriasis loci so far reported only in Asians showed evidence for association in European populations (p &lt; 10<sup>−3</sup>): <em>CCDC80</em> (MIM <a href="/omim/608298" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608298</a>)<em>/CD200R1L</em> at 3q13.2, <em>CARD11</em> (MIM <a href="/omim/607210" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607210</a>) at 7p22.2, <em>ZNF365</em> (MIM <a href="/omim/607818" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607818</a>) at 10q21.2, and <em>BCAS1</em> (MIM <a href="/omim/602968" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602968</a>) at 20q13.2 in AD; <em>CSMD1</em> (MIM <a href="/omim/608397" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608397</a>) at 8p23.2, <em>SERPINB8</em> (MIM <a href="/omim/601697" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601697</a>) at 18q22.1, <em>MAMSTR</em> (MIM <a href="/omim/610349" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">610349</a>)<em>/RASIP1</em> (MIM <a href="/omim/609623" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609623</a>) at 19q13.33, and <em>ZNF816A</em> at 19q13.41 in psoriasis (<a href="#app2" rid="app2" class=" sec">Table S6</a>).</p></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">New Opposing-Effect Loci Identified by Genome-wide Comparative Analysis</h3><p id="__p29" class="p p-first">Excluding the MHC, 25 loci showed a genome-wide significant association with either skin disorder, defined by&nbsp;all three methods of analysis (CCMA <em>T</em><sub><em>max</em></sub> &gt; 6 and MANTRA BFDP &lt; 0.05 with PO = 1/999, and p<sub>MNM</sub> &lt; 10<sup>−8</sup>) including six loci that were coassociated with both AD and psoriasis. Each coassociated locus displayed opposing effects and two of these loci (2q31.2, 5q33.1) have not previously been reported as showing coassociation with AD and psoriasis (<a href="/pmc/articles/PMC4289690/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p76">Loci Showing Genome-wide Significant Association with Either AD or Psoriasis Defined by All Three Methods of Comparative Analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812598462400" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Chr Band</strong></th><th rowspan="2" colspan="1"><strong>Reference SNV Number(s)</strong></th><th rowspan="2" colspan="1"><strong>Position (hg19)</strong></th><th rowspan="2" colspan="1"><strong>Nearest Gene(s) or Transcript(s)</strong></th><th rowspan="2" colspan="1"><strong>Effect Observed in GW Analyses</strong></th><th colspan="2" rowspan="1"><strong>Estimated Odds Ratio (95% CI)</strong><hr></th><th rowspan="2" colspan="1"><strong>Previous Report(s) of Association at This Locus</strong></th></tr><tr><th rowspan="1" colspan="1"><strong>AD</strong></th><th rowspan="1" colspan="1"><strong>Psoriasis</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">1p31.3</td><td rowspan="1" colspan="1">rs77614545 (del)</td><td rowspan="1" colspan="1">67749581</td><td rowspan="1" colspan="1">retro-DNAJB6 and <em>IL23R</em> (MIM <a href="/omim/607562" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607562</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.99 (0.92–1.07)</td><td rowspan="1" colspan="1" align="center">1.21 (1.15–1.28)</td><td rowspan="1" colspan="1">psoriasis: 1p31.3 locus, <em>IL28RA</em><sup><a href="#bib21" rid="bib21 bib24 bib40" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">21,24,40</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="8" colspan="1">1q21.3<sup>a</sup><hr></td><td rowspan="1" colspan="1">rs55879323</td><td rowspan="1" colspan="1">152168740</td><td rowspan="1" colspan="1">within <em>FLG-AS1</em></td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1" align="center">0.76 (0.70–0.82)</td><td rowspan="1" colspan="1" align="center">1.05 (1.00–1.12)</td><td rowspan="1" colspan="1">AD and psoriasis: 1q21.3 locus, <em>HRNR</em>, <em>FLG</em>;<sup><a href="#bib41" rid="bib41" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">41</a></sup><em>FLG</em>;<sup><a href="#bib20" rid="bib20 bib42" class=" bibr popnode" role="button" aria-expanded="false" aria-haspopup="true">20,42</a></sup><em>LCE3B</em>, <em>LCE3C</em>,<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup><em>LCE</em> gene cluster;<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup><em>LCE3D</em><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs11205006, rs12144049</td><td rowspan="1" colspan="1">152440176, 152440910</td><td rowspan="1" colspan="1">RP1-91G5.3, <em>CRNN</em> (MIM <a href="/omim/611312" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">611312</a>), <em>LCE5A</em> (MIM <a href="/omim/612619" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612619</a>)</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1" align="center">1.52 (1.41–1.64), 1.53 (1.42–1.64)</td><td rowspan="1" colspan="1" align="center">0.97 (0.92–1.03), 0.98 (0.92–1.03)</td><td rowspan="1" colspan="1">AD: 1q21.3 locus, <em>HRNR, FLG</em>;<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup><em>FLG</em><sup><a href="#bib20" rid="bib20 bib42" class=" bibr popnode">20,42</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs471144</td><td rowspan="1" colspan="1">152454255</td><td rowspan="1" colspan="1"><em>LCE5A</em> (MIM <a href="/omim/612619" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612619</a>)</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1" align="center">1.54 (1.37–1.73)</td><td rowspan="1" colspan="1" align="center">1.03 (0.94–1.14)</td><td rowspan="1" colspan="1">AD: 1q21.3 locus, <em>HRNR</em>, <em>FLG</em>;<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup><em>FLG</em><sup><a href="#bib20" rid="bib20 bib42" class=" bibr popnode">20,42</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs10888499</td><td rowspan="1" colspan="1">152532742</td><td rowspan="1" colspan="1"><em>LCE3E</em> (MIM <a href="/omim/612617" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612617</a>)</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1" align="center">1.49 (1.38–1.61)</td><td rowspan="1" colspan="1" align="center">0.98 (0.93–1.04)</td><td rowspan="1" colspan="1">AD: 1q21.3 locus, <em>HRNR</em>, <em>FLG</em>;<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup><em>FLG</em><sup><a href="#bib20" rid="bib20 bib42" class=" bibr popnode">20,42</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs4112788</td><td rowspan="1" colspan="1">152551276</td><td rowspan="1" colspan="1"><em>LCE3D</em> (MIM <a href="/omim/612616" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612616</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.97 (0.90–1.05)</td><td rowspan="1" colspan="1" align="center">1.22 (1.15–1.28)</td><td rowspan="1" colspan="1">psoriasis: LCE gene cluster;<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup><em>LCE3D</em><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs1581803</td><td rowspan="1" colspan="1">152592281</td><td rowspan="1" colspan="1"><em>LCE3A</em> (MIM <a href="/omim/612613" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612613</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.97 (0.90–1.04)</td><td rowspan="1" colspan="1" align="center">1.22 (1.15–1.30)</td><td rowspan="1" colspan="1">psoriasis: LCE gene cluster<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs77199844 (del)</td><td rowspan="1" colspan="1">152757094</td><td rowspan="1" colspan="1"><em>LCE1E</em> (MIM <a href="/omim/612607" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612607</a>)</td><td rowspan="1" colspan="1">[AD]</td><td rowspan="1" colspan="1" align="center">2.01 (1.72–2.35)</td><td rowspan="1" colspan="1" align="center">1.16 (1.01–1.33)</td><td rowspan="1" colspan="1">AD: 1q21.3 locus, <em>HRNR, FLG</em>;<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup><em>FLG</em><sup><a href="#bib20" rid="bib20 bib42" class=" bibr popnode">20,42</a></sup><br>psoriasis: LCE gene cluster<sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs4363385</td><td rowspan="1" colspan="1">152989321</td><td rowspan="1" colspan="1">SNORA31, <em>SPRR3</em> (MIM <a href="/omim/182271" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">182271</a>), <em>SPRR1B</em> (MIM <a href="/omim/182266" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">182266</a>)</td><td rowspan="1" colspan="1">[opposing]</td><td rowspan="1" colspan="1" align="center">1.23 (1.15–1.32)</td><td rowspan="1" colspan="1" align="center">0.89 (0.85–0.94)</td><td rowspan="1" colspan="1">AD: <em>SPRR3</em> repeat number variant<sup><a href="#bib43" rid="bib43" class=" bibr popnode">43</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">2p16.1</td><td rowspan="1" colspan="1">rs35741374</td><td rowspan="1" colspan="1">61072567</td><td rowspan="1" colspan="1">within lincRNA <a id="__tag_589567604" class="tag_hotlink" href="/nuccore/AC010733.4" ref="/nuccore/AC010733.4"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC010733.4","term_id":"9858984","term_text":"AC010733.4"}}</span>AC010733.4</a></td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.09 (1.01–1.63)</td><td rowspan="1" colspan="1" align="center">1.20 (1.15–1.27)</td><td rowspan="1" colspan="1">psoriasis: <em>REL</em>;<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup><em>NR</em><sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">2q31.2</td><td rowspan="1" colspan="1">rs62176107</td><td rowspan="1" colspan="1">179300971</td><td rowspan="1" colspan="1">exonic <em>PRKRA</em> and within miRNA548n and AC009948.5</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1" align="center">0.55 (0.46–0.65)</td><td rowspan="1" colspan="1" align="center">1.42 (1.32–1.53)</td><td rowspan="1" colspan="1">–</td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="2" colspan="1">5q31.1<sup>a</sup><hr></td><td rowspan="1" colspan="1">rs1295686</td><td rowspan="1" colspan="1">131952222</td><td rowspan="1" colspan="1">intronic <em>IL13</em> (MIM <a href="/omim/147683" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147683</a>) and within AC004041.2</td><td rowspan="1" colspan="1">[opposing]</td><td rowspan="1" colspan="1" align="center">1.27 (1.17–1.38)</td><td rowspan="1" colspan="1" align="center">0.88 (0.82–0.94)</td><td rowspan="1" colspan="1">AD and psoriasis: <em>IL13</em>;<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><em>KIF3A, IL13</em>;<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup><em>KIF3A</em>, <em>IL4</em>, <em>IL13-RAD50</em>;<sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup> multiple effect locus <em>RAD50/IL13</em>;<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup><em>C5orf56</em></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">rs6596086</td><td rowspan="1" colspan="1">131995843</td><td rowspan="1" colspan="1">intronic <em>RAD50</em> (MIM <a href="/omim/604040" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604040</a>)</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1" align="center">1.30 (1.20–1.41)</td><td rowspan="1" colspan="1" align="center">0.85 (0.8–0.91)</td><td rowspan="1" colspan="1">AD and psoriasis: <em>IL13-RAD50</em>;<sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup> multiple effect locus <em>RAD50/IL13</em><sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">5q33.1</td><td rowspan="1" colspan="1">rs17728338</td><td rowspan="1" colspan="1">150478318</td><td rowspan="1" colspan="1"><em>ANXA6</em> (MIM <a href="/omim/114070" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">114070</a>)</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1" align="center">0.70 (0.59–0.84)</td><td rowspan="1" colspan="1" align="center">1.77 (1.61–1.95)</td><td rowspan="1" colspan="1">psoriasis: <em>TNIP1</em><sup><a href="#bib21" rid="bib21 bib42" class=" bibr popnode">21,42</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="4" colspan="1">5q33.3<hr></td><td rowspan="1" colspan="1">rs10515778, rs7715173, rs7719425<sup>b</sup></td><td rowspan="1" colspan="1">158658012, 158664631, 158670938</td><td rowspan="1" colspan="1">within CTB-11I22.1</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.07 (0.98–1.17)</td><td rowspan="1" colspan="1" align="center">1.29 (1.21–1.38)</td><td rowspan="4" colspan="1">psoriasis: 5q33.3 locus, <em>IL12B</em><sup><a href="#bib17" rid="bib17 bib21 bib24 bib44" class=" bibr popnode">17,21,24,44</a></sup>; AD: <em>PTTG1</em><sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup><hr></td></tr><tr><td rowspan="1" colspan="1">rs11135056, rs4921442<sup>b</sup></td><td rowspan="1" colspan="1">158687281, 158694100</td><td rowspan="1" colspan="1">intronic <em>UBLCP1</em> (MIM <a href="/omim/609867" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609867</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.97 (0.89–1.05)</td><td rowspan="1" colspan="1" align="center">1.45 (1.35–1.56)</td></tr><tr><td rowspan="1" colspan="1">rs2546890</td><td rowspan="1" colspan="1">158759900</td><td rowspan="1" colspan="1">within <a id="__tag_589567607" class="tag_hotlink" href="/nuccore/AC008697.1" ref="/nuccore/AC008697.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC008697.1","term_id":"5686374","term_text":"AC008697.1"}}</span>AC008697.1</a></td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.01 (0.94–1.06)</td><td rowspan="1" colspan="1" align="center">1.39 (1.32–1.47)</td></tr><tr><td rowspan="1" colspan="1">rs5872599 (indel)</td><td rowspan="1" colspan="1">158859989</td><td rowspan="1" colspan="1">lincRNA <a id="__tag_589567606" class="tag_hotlink" href="/nuccore/AC008703.1" ref="/nuccore/AC008703.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC008703.1","term_id":"5686368","term_text":"AC008703.1"}}</span>AC008703.1</a>, <em>IL12B</em> (MIM <a href="/omim/161561" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">161561</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.82 (0.73–0.93)</td><td rowspan="1" colspan="1" align="center">1.54 (1.45–1.64)</td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="3" colspan="1">6q21<hr></td><td rowspan="1" colspan="1">rs9487605</td><td rowspan="1" colspan="1">111582885</td><td rowspan="1" colspan="1">intronic <em>KIAA1919</em></td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.06 (0.98–1.14)</td><td rowspan="1" colspan="1" align="center">1.27 (1.20–1.35)</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">rs240993</td><td rowspan="1" colspan="1">111673714</td><td rowspan="1" colspan="1">intronic <em>REV3L</em> (MIM <a href="/omim/602776" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602776</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.05 (0.97–1.13)</td><td rowspan="1" colspan="1" align="center">1.29 (1.22–1.36)</td><td rowspan="1" colspan="1">–</td></tr><tr><td rowspan="1" colspan="1">rs9481169</td><td rowspan="1" colspan="1">111929862</td><td rowspan="1" colspan="1"><em>TRAF3IP2</em> (MIM <a href="/omim/607043" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607043</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.98 (0.86–1.11)</td><td rowspan="1" colspan="1" align="center">1.58 (1.45–1.72)</td><td rowspan="1" colspan="1">psoriasis and psoriatic arthritis: <em>TRAF3IP2</em><sup><a href="#bib24" rid="bib24 bib44 bib45" class=" bibr popnode">24,44,45</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">6q23.2</td><td rowspan="1" colspan="1">rs643177, rs582757<sup>b</sup></td><td rowspan="1" colspan="1">138195693, 138197824</td><td rowspan="1" colspan="1"><em>TNFAIP3</em> (MIM <a href="/omim/610669" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">610669</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">1.05 (0.97–1.14)</td><td rowspan="1" colspan="1" align="center">1.27 (1.20–1.34)</td><td rowspan="1" colspan="1">psoriasis: <em>TNFAIP3</em><sup><a href="#bib21" rid="bib21 bib24" class=" bibr popnode">21,24</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">11q13.5</td><td rowspan="1" colspan="1">rs2212434, rs7126418<sup>b</sup></td><td rowspan="1" colspan="1">76281593, 76292573</td><td rowspan="1" colspan="1"><em>c11orf30</em> (MIM <a href="/omim/608574" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608574</a>)</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1" align="center">1.29 (1.21–1.39)</td><td rowspan="1" colspan="1" align="center">1.05 (1.00–1.11)</td><td rowspan="1" colspan="1">AD: <em>c11orf30-LRRC32</em>;<sup><a href="#bib10" rid="bib10 bib41" class=" bibr popnode">10,41</a></sup><em>c11orf30</em>;<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup> 11q13 locus<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup></td></tr><tr><td colspan="8" rowspan="1"><hr></td></tr><tr><td rowspan="2" colspan="1">12q13.3</td><td rowspan="1" colspan="1">rs36207871 (del)</td><td rowspan="1" colspan="1">56684496</td><td rowspan="1" colspan="1">intronic <em>CS</em> (MIM <a href="/omim/118950" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">118950</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.94 (0.83–1.06)</td><td rowspan="1" colspan="1" align="center">1.47 (1.33–1.67)</td><td rowspan="1" colspan="1">psoriasis: 12q13.3 locus, <em>IL23A, STAT2</em>;<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><em>IL23A</em><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup></td></tr><tr><td rowspan="1" colspan="1">rs11575234</td><td rowspan="1" colspan="1">56744276</td><td rowspan="1" colspan="1">intronic <em>STAT2</em> (MIM <a href="/omim/600556" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600556</a>)</td><td rowspan="1" colspan="1">psoriasis</td><td rowspan="1" colspan="1" align="center">0.90 (0.79–1.02)</td><td rowspan="1" colspan="1" align="center">1.47 (1.32–1.64)</td><td rowspan="1" colspan="1">psoriasis: <em>STAT2</em><sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup></td></tr></tbody></table></div><div id="largeobj_idm139812598462400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4289690/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139812600879872"><p id="__p77" class="p p-first-last">Genome-wide significance is defined as CCMA <em>T</em><sub><em>max</em></sub> &gt; 6 and MANTRA BFDP &lt; 0.05 with PO = 1/999 and multinomial model p &lt; 10<sup>−8</sup>; genes and transcripts identified from UCSC Genome Browser Human Feb. 2009 (GRCh37/hg19) Assembly accessed 21 March 2014; this variant of <em>FLG-AS1</em> extends across <em>HRHR</em> and <em>FLG</em>; RP1-91G5.3 extends across <em>CRNN</em>; AC004041.2 extends across <em>RAD50</em> and <em>IL13</em>; CTB-11I22.1 overlaps <em>RNF145</em>.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>1q21.3 and 5q31.1 were further investigated via stepwise conditional analysis (the results are shown in Table 2); square brackets indicate results from the univariate analysis that were subsequently accounted for by other nearby variants when examined by stepwise conditional analysis.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Multiple SNVs are assigned to the same LD block but odds ratios and 95% CI are presented only for the first SNV.</div></div></div><p id="__p30">2q31.2 demonstrates an opposing effect at rs62176107 (MNM p = 1.08&nbsp;× 10<sup>−34</sup>; <sup><a href="#bib5" rid="bib5" class=" bibr popnode">Table S5</a></sup>); this variant is within exon 6 of <em>PRKRA</em> (MIM <a href="/omim/603424" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603424</a>) and also within microRNA&nbsp;548n (MIR548N) and a noncoding transcript, AC009948.5. <em>PRKRA</em> encodes protein kinase interferon-inducible double-stranded RNA-dependent activator (PACT), a cellular dsRNA-binding protein originally identified as a binding partner and activator of PKR in response to extracellular stress.<sup><a href="#bib46" rid="bib46" class=" bibr popnode">46</a></sup> More recently, it has been shown to be an essential factor in the PKR-independent initiation of RIG-I-induced antiviral response.<sup><a href="#bib47" rid="bib47" class=" bibr popnode">47</a></sup> Of note, individuals with AD are known to be susceptible to viral skin infections, but cutaneous infections&nbsp;rarely occur in psoriasis.<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> MicroRNAs play a role in posttranscriptional regulation of gene expression by affecting the stability and translation of mRNAs, but the specific role of miRNA548n has not been defined. The most significantly associated (“lead”) SNV at 2q31.2 (rs62176107, G&gt;A, having the smallest p value from MNM) is a synonymous SNV with predicted effects on 12 transcripts, including <em>PRKRA</em> splice variants’ UTR and intronic regions and a variant predicted to undergo nonsense-mediated decay (Ensembl release 75). Gene expression profiling data show downregulation of both <em>PRKRA</em> mRNA and miRNA548n in psoriatic lesions compared to nonlesional skin, but no significant differences in AD (<a href="#app2" rid="app2" class=" sec">Table S7</a>).</p><p id="__p31">The most highly significant variant at 5q33.1 (rs17728338) shows opposing effects on AD and psoriasis (MNM p = 3.96&nbsp;× 10<sup>−38</sup>; <sup><a href="#bib5" rid="bib5" class=" bibr popnode">Table S5</a></sup>) and lies 2 kb upstream of <em>ANXA6</em> (MIM <a href="/omim/114070" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">114070</a>) and 8 kb downstream of <em>TNIP1</em> (MIM <a href="/omim/607714" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607714</a>). LD analysis in 1000 Genomes (release August 2009) via LocusZoom<sup><a href="#bib49" rid="bib49" class=" bibr popnode">49</a></sup> showed that rs17728338 is located within a 25-kb block containing both <em>TNIP1</em> and <em>ANXA6</em>. The locus has previously been associated with psoriasis in European and Chinese populations but&nbsp;has not been implicated in AD. <em>TNIP1</em> is involved in TNF signaling and regulation of the transcription factor NF-κB;<sup><a href="#bib21" rid="bib21 bib50" class=" bibr popnode">21,50</a></sup> it shows increased expression both in AD and psoriatic lesions compared to control skin (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). In contrast, <em>ANXA6</em>, which encodes a calcium-dependent membrane and phospholipid binding protein, shows significant upregulation of expression in atopic skin compared to control skin (fold change 1.3, FDR p = 0.016) and lesional to nonlesional AD skin (fold change 2.4, p = 0.027), whereas expression is decreased in psoriatic versus healthy skin (fold change 0.7, p = 6.38&nbsp;× 10<sup>−13</sup>) (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). Clearly, further fine mapping is necessary to identify the causal variant that exerts opposing effects on AD and psoriasis, but we speculate that <em>ANXA6</em> might be a switch-point differentiating AD from psoriasis that reflects the importance of calcium-dependent effects in keratinocyte differentiation.</p><p id="__p32" class="p p-last">Opposing effect loci were also identified within regions characterized by complex patterns of LD within the EDC (<a href="#app2" rid="app2" class=" sec">Figure&nbsp;S4</a>), the cytokine cluster on 5q31.1 (<a href="#app2" rid="app2" class=" sec">Figures S4</a> and <a href="#app2" rid="app2" class=" sec">S5</a>), and the MHC. These regions were therefore investigated further via conditional analysis.</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Stepwise Conditional Analysis within 1q21.3 and 5q31.1 Identifies Opposing and Disease-Specific Risk Variants</h3><p id="__p33" class="p p-first">Coverage of the EDC was achieved via GWAS data (<a href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>), whereas ImmunoChip data provided better coverage for the cytokine cluster on 5q31.1 (<a href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures 4</span></a> and <a href="#app2" rid="app2" class=" sec">S5</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812598313792" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" src="/pmc/articles/PMC4289690/bin/gr3.jpg"></div></a><div id="largeobj_idm139812598313792" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a target="object" href="/pmc/articles/PMC4289690/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p68">Regional Association within the Epidermal Differentiation Complex at 1q21.3</p><p id="__p69">(A) Multinomial regression model with GWAS and ImmunoChip data. Seven blocks of linkage disequilibrium are indicated by curly brackets; black circles indicate AD-specific association, red circles indicate a psoriasis-specific association, blue circles represent opposing effects in AD and psoriasis, and green circles indicate shared effects. Vertical lines have been drawn to mark the positions of known genes and transcripts (identified from UCSC Genome Browser, GRCh37/hg19 accessed Feb. 2009) and the horizontal dotted lines indicate thresholds of suggestive and genome-wide significance (p = 10<sup>−5</sup> and 10<sup>−8</sup>). The horizontal gray bands at the bottom indicate the coverage of the region by GWAS SNVs (upper row) and ImmunoChip SNVs (lower row).</p><p id="__p70">(B) Conditional regional association plot of stepwise logistic regression using GWAS and ImmunoChip data. The different colored symbols indicate association results after each step of analysis, as follows. Unconditioned results are shown by black dots to indicate association with AD and red dots to indicate association with psoriasis; blue triangles and blue crosses represent results after conditioning on the known disease-associated variants, <em>FLG</em> in AD and <em>LCE3B-LCE3C</em> deletion in psoriasis; SNVs indicated by the same symbol are in LD with the lead SNV of each stepwise conditional analysis (defined as <em>r</em><sup>2</sup> ≥ 0.5). Vertical lines are drawn to mark the positions of known genes and transcripts (identified from the UCSC Genome Browser GRCh37/hg19 accessed Feb. 2009), and horizontal dotted lines indicate significance thresholds of p = 0.005, 10<sup>−5</sup>, and 10<sup>−8</sup>. The horizontal gray bands at the bottom indicate the coverage of the region by GWAS SNVs (upper row) and ImmunoChip SNVs (lower row).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812644145520" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" src="/pmc/articles/PMC4289690/bin/gr4.jpg"></div></a><div id="largeobj_idm139812644145520" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a target="object" href="/pmc/articles/PMC4289690/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p71">Regional Association within the Cytokine Cluster at 5q31.1</p><p id="__p72">(A) Multinomial regression model with GWAS and ImmunoChip data. Black circles indicate AD-specific association, red circles indicate psoriasis-specific association, blue circles represent opposing effects in AD and psoriasis, and green circles indicate shared effects. Vertical gray shading marks the positions of known genes (identified from the UCSC Genome Browser GRCh37/hg19 accessed Feb. 2009), and horizontal dotted lines indicate suggestive and genome-wide significance thresholds (p = 10<sup>−5</sup> and 10<sup>−8</sup>, respectively); results are shown for SNVs in LD with the lead SNV (defined as <em>r</em><sup>2</sup> ≥ 0.5). The horizontal bands at the bottom indicate the coverage of the region by GWAS SNVs (upper row) and ImmunoChip SNVs (lower row).</p><p id="__p73">(B) Conditional regional association plot of the EDC by multinomial regression of GWAS and ImmunoChip data. Different symbols indicate association results after each step of analysis, as follows. Unconditioned results are shown by blue circles representing opposing effects in AD and psoriasis; black dots show AD-specific association results after conditioning on the lead SNV in <em>RAD50</em> (a gene reported to be associated with AD and psoriasis); black squares indicate the residual AD-specific association after conditioning on the lead SNVs in <em>RAD50</em> and <em>IL13</em> (genes reported to be associated with AD); and black triangles indicate the residual AD-specific association after additionally conditioning on the lead SNV in <em>KIF3A</em> (a gene reported to be associated with AD). SNVs indicated by the same symbol are in LD with the lead SNV of each stepwise conditional analysis (defined as <em>r</em><sup>2</sup> ≥ 0.5). Vertical gray shading marks the positions of known genes (identified from the UCSC Genome Browser GRCh37/hg19 accessed Feb. 2009), and horizontal dotted lines indicate significance thresholds of p = 0.005, 10<sup>−5</sup>, and 10<sup>−8</sup>; results are shown for SNVs in LD with the lead SNV (defined as <em>r</em><sup>2</sup> ≥ 0.5). The horizontal bands at the bottom indicate the coverage of the region by GWAS SNVs (upper row) and ImmunoChip SNVs (lower row).</p></div></div></div><p id="__p34">Within 1q21.3 we identified seven LD blocks with disease-specific or opposing signals (<a href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>A). Stepwise conditional analysis on the four most prevalent <em>FLG</em>-null mutations and variants tagging the <em>LCE3B-LCE3C</em> deletion identified one AD-specific locus mapping to <em>FLG</em>, a psoriasis-specific locus mapping to <em>LCE3B-LCE3C</em>, and a locus with opposing effects on both diseases mapping to <em>RPTN</em> (MIM <a href="/omim/613259" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">613259</a>)<em>/HRNR</em>/<em>FLG-AS1</em> (<a href="/pmc/articles/PMC4289690/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figure&nbsp;3</span></a>B and <a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). After conditioning on the four <em>FLG</em>-null mutations and the <em>LCE3B-LCE3C</em> deletion, the G allele of the lead MNM SNV rs12130219 decreases the risk for AD (OR<sub>ADcond</sub> = 0.812, p<sub>ADcond</sub> = 0.0018) and increases the risk for psoriasis (OR<sub>PSOcond</sub> = 1.119, p<sub>PSOcond</sub> = 3.68&nbsp;× 10<sup>−4</sup>) (<a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). Filaggrin, repetin, and hornerin are all members&nbsp;of the S100 fused-type protein family and each contribute to the cornified cell envelope, a functional component of the epidermal barrier. Both <em>FLG</em> and <em>HRNR</em> show reduced expression in AD<sup><a href="#bib51" rid="bib51 bib52 bib53" class=" bibr popnode">51–53</a></sup> whereas <em>RPTN</em> shows no significant difference (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). In psoriasis <em>HRNR</em> expression can be&nbsp;downregulated<sup><a href="#bib53" rid="bib53" class=" bibr popnode">53</a></sup> or upregulated,<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup>
<em>RPTN</em> expression might be upregulated, and <em>FLG</em> expression might be downregulated<sup><a href="#bib55" rid="bib55" class=" bibr popnode">55</a></sup> or dysregulated<sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). The function of <em>FLG-AS1</em> (<em>FLG</em> antisense RNA1) is currently undefined, but its proximity to <em>FLG</em> and <em>HRNR</em> suggests a role in coordinating keratinocyte terminal differentiation. <em>FLG-AS1</em> expression is increased in&nbsp;psoriasis lesional compared with nonlesional skin, whereas in AD lesional skin, expression is reduced (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). Together, our results confirm the role of the <em>LCE3B-LCE3C</em> deletion in psoriasis and support the presence of genetic risk mechanisms for AD within the EDC in addition to the predominant effect of <em>FLG</em>-null mutations, with opposing effects on psoriasis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p80">Conditional Analysis of the 1q21.3 and 5q31.1 Regions Showing Disease-Specific and Opposing Risk Effects in AD and Psoriasis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812598631664" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1"><strong>Data Source</strong></th><th rowspan="2" colspan="1"><strong>Effect</strong></th><th rowspan="2" colspan="1"><strong>SNV</strong></th><th rowspan="2" colspan="1"><strong>Pos (hg19)</strong></th><th rowspan="2" colspan="1"><strong>Allele</strong></th><th rowspan="2" colspan="1"><strong>Candidate Genes</strong></th><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>p</strong><sub><strong>overall</strong></sub><sup>a</sup></th><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>P</strong><sub><strong>overall</strong></sub><sup>a</sup></th></tr><tr><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td colspan="16" rowspan="1"><strong>Chromosome 1q21.3</strong></td></tr><tr><td colspan="16" rowspan="1"><hr></td></tr><tr><td colspan="6" rowspan="1"></td><td colspan="4" rowspan="1"><strong>Unconditioned Analysis</strong></td><td rowspan="1" colspan="1"></td><td colspan="4" rowspan="1"><strong>Conditional Analysis</strong><sup>b</sup></td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="16" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">Opposing</td><td rowspan="1" colspan="1">rs12130219</td><td rowspan="1" colspan="1" align="center">152162106</td><td rowspan="1" colspan="1"><span class="underline">G</span>/A</td><td rowspan="1" colspan="1"><em>FLG-AS1/RPTN/HRNR</em></td><td rowspan="1" colspan="1" align="center">0.66 (0.60–0.73)</td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1">1.15 (1.09–1.224)</td><td rowspan="1" colspan="1">4.0&nbsp;× 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.2&nbsp;× 10<sup>−23</sup></td><td rowspan="1" colspan="1" align="center">0.812 (0.71–0.93)</td><td rowspan="1" colspan="1">0.0018</td><td rowspan="1" colspan="1" align="center">1.119 (1.05–1.19)</td><td rowspan="1" colspan="1">3.68&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">2.4&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1">rs12144049</td><td rowspan="1" colspan="1" align="center">152440910</td><td rowspan="1" colspan="1"><span class="underline">C</span>/T</td><td rowspan="1" colspan="1"><em>FLG</em></td><td rowspan="1" colspan="1" align="center">1.53 (1.42–1.64)</td><td rowspan="1" colspan="1" align="center">2.7&nbsp;× 10<sup>−30</sup></td><td rowspan="1" colspan="1">0.98 (0.92–1.03)</td><td rowspan="1" colspan="1">0.4140</td><td rowspan="1" colspan="1" align="center">3.0&nbsp;× 10<sup>−30</sup></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">Psoriasis</td><td rowspan="1" colspan="1">rs1581803<sup>c</sup></td><td rowspan="1" colspan="1" align="center">152592281</td><td rowspan="1" colspan="1"><span class="underline">G</span>/T</td><td rowspan="1" colspan="1"><em>LCE3B/LCE3D</em></td><td rowspan="1" colspan="1" align="center">0.97 (0.90–1.04)</td><td rowspan="1" colspan="1">0.4396</td><td rowspan="1" colspan="1">1.22 (1.16–1.30)</td><td rowspan="1" colspan="1">1.5&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.6&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">Opposing</td><td rowspan="1" colspan="1">rs35722864</td><td rowspan="1" colspan="1" align="center">153040505</td><td rowspan="1" colspan="1">G/GA</td><td rowspan="1" colspan="1"><em>SPRR</em> cluster</td><td rowspan="1" colspan="1" align="center">0.81 (0.75–0.88)</td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.129 (1.07–1.20)</td><td rowspan="1" colspan="1">2.1&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">4.8&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">0.851 (0.71–0.93)</td><td rowspan="1" colspan="1">0.0019</td><td rowspan="1" colspan="1" align="center">1.074 (1.01–1.14)</td><td rowspan="1" colspan="1" align="center">0.0211</td><td rowspan="1" colspan="1">1.3&nbsp;× 10<sup>−4</sup></td></tr><tr><td colspan="16" rowspan="1"><hr></td></tr><tr><td colspan="16" rowspan="1"><strong>Chromosome 5q31.1</strong></td></tr><tr><td colspan="16" rowspan="1"><hr></td></tr><tr><td colspan="6" rowspan="1"></td><td colspan="5" rowspan="1"><strong>Conditional Models</strong><sup>d</sup></td><td colspan="4" rowspan="1"><strong>Full Model</strong></td><td rowspan="1" colspan="1"></td></tr><tr><td colspan="16" rowspan="1"><hr></td></tr><tr><td rowspan="1" colspan="1">Ichip</td><td rowspan="1" colspan="1">Opposing</td><td rowspan="1" colspan="1">rs6596086</td><td rowspan="1" colspan="1" align="center">131952222</td><td rowspan="1" colspan="1"><span class="underline">C</span>/T</td><td rowspan="1" colspan="1"><em>RAD50</em></td><td rowspan="1" colspan="1" align="center">1.31 (1.22–1.41)</td><td rowspan="1" colspan="1" align="center">1.7&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1">0.86 (0.80–0.92)</td><td rowspan="1" colspan="1">1.7&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">5.7&nbsp;× 10<sup>−21</sup></td><td rowspan="1" colspan="1">1.17 (1.07–1.28)</td><td rowspan="1" colspan="1">4.04&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.88 (0.81–0.96)</td><td rowspan="1" colspan="1" align="center">0.0023</td><td rowspan="1" colspan="1">6.3&nbsp;× 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">Ichip</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1">rs848</td><td rowspan="1" colspan="1" align="center">131996500</td><td rowspan="1" colspan="1"><span class="underline">A/</span>C</td><td rowspan="1" colspan="1"><em>IL13</em></td><td rowspan="1" colspan="1" align="center">1.20 (1.10–1.30)</td><td rowspan="1" colspan="1" align="center">5.6&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">0.96 (0.89–1.04)</td><td rowspan="1" colspan="1">0.3375</td><td rowspan="1" colspan="1" align="center">4.14&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.12 (1.02–1.23)</td><td rowspan="1" colspan="1">0.0197</td><td rowspan="1" colspan="1">0.96 (0.88–1.05)</td><td rowspan="1" colspan="1" align="center">0.3515</td><td rowspan="1" colspan="1">0.0204</td></tr><tr><td rowspan="1" colspan="1">Ichip</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1">rs2299009</td><td rowspan="1" colspan="1" align="center">132042813</td><td rowspan="1" colspan="1"><span class="underline">G</span>/T</td><td rowspan="1" colspan="1"><em>IL4/KIF3A</em></td><td rowspan="1" colspan="1" align="center">1.14 (1.06–1.23)</td><td rowspan="1" colspan="1" align="center">7.9&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.99 (0.92–1.06)</td><td rowspan="1" colspan="1">0.7392</td><td rowspan="1" colspan="1">0.0018</td><td rowspan="1" colspan="1">1.16 (1.07–1.25)</td><td rowspan="1" colspan="1">2.03&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">0.99 (0.92–1.06)</td><td rowspan="1" colspan="1" align="center">0.6657</td><td rowspan="1" colspan="1">4.1&nbsp;× 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1">Ichip</td><td rowspan="1" colspan="1">AD</td><td rowspan="1" colspan="1">rs74458173</td><td rowspan="1" colspan="1" align="center">131621731</td><td rowspan="1" colspan="1"><span class="underline">A</span>/C</td><td rowspan="1" colspan="1"><em>SLC22A4</em></td><td rowspan="1" colspan="1" align="center">1.57 (1.26–1.96)</td><td rowspan="1" colspan="1" align="center">6.1&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.02 (0.80–1.30)</td><td rowspan="1" colspan="1">0.8590</td><td rowspan="1" colspan="1" align="center">2.14&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1">1.57 (1.26–1.96)</td><td rowspan="1" colspan="1">5.71&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1">1.02 (0.80–1.30)</td><td rowspan="1" colspan="1" align="center">0.8683</td><td rowspan="1" colspan="1">2.0&nbsp;× 10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm139812598631664" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4289690/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139812601287952"><p id="__p81" class="p p-first-last">Full model incorporates the combined effects of independent SNVs identified by stepwise analyses.</p></div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>a</sup>p<sub>overall</sub> represents the overall opposing signal calculated using the T<sub>12opposing</sub> statistic and derive the p value from the normal distribution.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>b</sup>Conditional analysis of chr1q21.3 was conditioned on <em>FLG</em> for AD and <em>LCE3B/LCE3D</em> for psoriasis.</div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>c</sup>rs1581803 tags the previously reported psoriasis SNV rs4112788 (<em>r</em><sup>2</sup> = 0.995).</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>d</sup>Stepwise conditional analysis at chr5q31.1 was carried out using multinomial regression models and resulted in three additional signals for AD; this table shows only independent loci (<em>r</em><sup>2</sup> &lt; 0.5) and the SNV with the strongest association; the effect allele is underlined.</div></div></div><p id="__p35" class="p p-last">Conditional analysis at 5q31.1 revealed three independent loci specifically contributing to AD risk: <em>IL13</em> (MIM <a href="/omim/147683" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147683</a>, rs848, OR<sub>ADfull</sub> = 1.12, p = 0.0204), <em>KIF3A</em> (MIM <a href="/omim/604683" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604683</a>, rs 2299009, OR<sub>ADfull</sub> = 1.16, p = 4.1&nbsp;× 10<sup>−4</sup>), and <em>SLC22A4</em> (MIM <a href="/omim/604190" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604190</a>)/<em>C5orf56</em> (rs74458173, OR<sub>ADfull</sub> = 1.57, p = 2.0&nbsp;× 10<sup>−4</sup>) (<a href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>A, <a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). None of these loci showed significant effects on psoriasis. However, a fourth independent locus has opposing effects on AD and psoriasis. The most highly significant variant maps to <em>RAD50</em> (MIM <a href="/omim/604040" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604040</a>, rs6596086, OR<sub>ADfull</sub> = 1.17, OR<sub>PSOfull</sub> = 0.88, p = 6.3&nbsp;× 10<sup>−7</sup>); this variant is associated with increased risk of AD but is protective against psoriasis (<a href="/pmc/articles/PMC4289690/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a>B, <a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>).</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Analysis of the MHC Confirms Multiple Psoriasis-Risk Loci and Identifies Opposing Effects</h3><p id="__p36" class="p p-first">In the extended HLA region, we took forward 23,479 SNVs with <em>T</em><sub><em>max</em></sub> &gt; 4.7 or BFDP &lt; 0.05 (PO = 1/99) and 75 variables representing the classical HLA alleles obtained from HLA imputation for multinomial modeling, of which 18,515 SNVs were classified as specific to psoriasis by CCMA and MNM. To reduce the data set for post hoc analysis, we considered only SNVs with effect classification in the same direction by CCMA and MNM, meeting the p &lt; 10<sup>−5</sup> threshold in MNM. Within the psoriasis-specific markers, we excluded all tagging SNVs (<em>r</em><sup>2</sup> &gt; 0.8 with the lead SNV), resulting in 1,503 SNVs, including those previously reported for AD.<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup></p><p id="__p37" class="p p-last">The strongest and most significant association was observed for psoriasis, a variant (rs111576655 OR<sub>PSOfull</sub> = 3.32, p = 3.2&nbsp;× 10<sup>−65</sup>) tagging the well-known psoriasis-risk allele HLA-C<sup>∗</sup>06:02 (OR<sub>PSOfull</sub> = 3.59, p = 8.7&nbsp;× 10<sup>−154</sup>). Conditional analysis revealed two additional independent loci contributing to psoriasis risk at <em>MICA</em> (MIM <a href="/omim/600169" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600169</a>, rs201374403, OR<sub>PSOfull</sub> = 1.65, p = 1.0&nbsp;× 10<sup>−26</sup>) and <em>HLA-A</em> (MIM <a href="/omim/142800" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142800</a>, rs113573479, OR<sub>PSOfull</sub> = 1.41, p = 2.7&nbsp;× 10<sup>−17</sup>), as well as two loci with opposing effects at <em>HLA-C</em> (MIM <a href="/omim/142840" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142840</a>, rs1793889, OR<sub>ADfull</sub> = 0.6, OR<sub>PSOfull</sub> = 1.18, p = 1.1&nbsp;× 10<sup>−9</sup>) and <em>HLA-DRB1</em> (MIM <a href="/omim/142857" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142857</a>, rs28383201, OR<sub>ADfull</sub> = 0.61, OR<sub>PSOfull</sub> = 1.18, p = 6.5&nbsp;× 10<sup>−9</sup>) (<a href="/pmc/articles/PMC4289690/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Figure&nbsp;5</span></a>, <a href="/pmc/articles/PMC4289690/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>). Conditional analysis with imputed classical alleles identified five independent HLA-class I alleles contributing to psoriasis risk in addition to HLA-C<sup>∗</sup>06:02 and two alleles with opposing effects: HLA-C<sup>∗</sup>03:03 (OR<sub>ADfull</sub> = 0.71, OR<sub>PSOfull</sub> = 1.27, p = 2.3&nbsp;× 10<sup>−5</sup>) and HLA-DQA1<sup>∗</sup>02:01 (OR<sub>ADfull</sub> = 0.64, OR<sub>PSOfull</sub> = 1.09, p = 6.0&nbsp;× 10<sup>−8</sup>; r<sup>2</sup> = 0.405 with rs28383201) (<a href="/pmc/articles/PMC4289690/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table&nbsp;3</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC4289690/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812593431232" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" title="An external file that holds a picture, illustration, etc.
Object name is gr5.jpg" src="/pmc/articles/PMC4289690/bin/gr5.jpg"></div></a><div id="largeobj_idm139812593431232" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4289690/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"></a><a target="object" href="/pmc/articles/PMC4289690/figure/fig5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC4289690/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Figure&nbsp;5</a></div><!--caption a7--><div class="caption"><p id="__p74">Conditional Regional Association within the Major Histocompatibility Complex at 6p21–22 via GWAS and ImmunoChip Data</p><p id="__p75">Symbols indicate association results after each step of analysis, as follows. Unconditioned psoriasis-specific results are shown by red dots; red triangles show psoriasis-specific association results after conditioning on C<sup>∗</sup>06:02 (known to be strongly associated with psoriasis); red ×s indicate psoriasis-specific association after conditioning on C<sup>∗</sup>06:02 and <em>MICA</em>; blue +s indicate the association after conditioning on C<sup>∗</sup>06:02, <em>MICA</em>, and <em>HLA-A</em> with opposing effects on AD and psoriasis; and blue squares indicate the residual association after conditioning on C<sup>∗</sup>06:02, <em>MICA</em>, <em>HLA-A</em>, and <em>HLA-DRB1</em> with opposing effects on AD and psoriasis. SNVs indicated by the same symbol are in LD with the lead SNV of each stepwise conditional analysis (defined as <em>r</em><sup>2</sup> ≥ 0.5). Vertical shading marks the positions of known genes (identified from the UCSC Genome Browser GRCh37/hg19 accessed Feb. 2009) and HLA classes; horizontal dotted lines indicate significance thresholds of p = 10<sup>−5</sup> and p = 10<sup>−8</sup>; results are shown for SNVs in LD with the lead SNV (defined as <em>r</em><sup>2</sup> ≥ 0.5). The horizontal bands at the bottom indicate the coverage of the region by GWAS SNVs (upper row) and ImmunoChip SNVs (lower row).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p86">Conditional Analysis of the MHC Region on 6p21–22 Showing Psoriasis-Specific and Opposing Risk Effects in AD and Psoriasis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812600165568" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1"><strong>Data Source</strong></th><th rowspan="3" colspan="1"><strong>Effect</strong></th><th rowspan="3" colspan="1"><strong>SNV</strong></th><th rowspan="3" colspan="1"><strong>Pos (hg19)</strong></th><th rowspan="3" colspan="1"><strong>Allele</strong></th><th rowspan="3" colspan="1"><strong>HLA Allele/Candidate Genes</strong></th><th colspan="5" rowspan="1"><strong>Conditional Models</strong><sup><strong>a</strong></sup><hr></th><th colspan="5" rowspan="1"><strong>Full Model</strong><hr></th></tr><tr><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>p</strong><sub><strong>overall</strong></sub></th><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>p</strong><sub><strong>overall</strong></sub></th></tr><tr><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">rs111576655</td><td rowspan="1" colspan="1">31242731</td><td rowspan="1" colspan="1"><span class="underline">A</span>/T</td><td rowspan="1" colspan="1">C<sup>∗</sup>06:02</td><td rowspan="1" colspan="1" align="center">0.84 (0.74–0.95)</td><td rowspan="1" colspan="1">0.0053</td><td rowspan="1" colspan="1" align="center">4.41 (4.10–4.74)</td><td rowspan="1" colspan="1" align="center">3.1&nbsp;× 10<sup>−376</sup></td><td rowspan="1" colspan="1" align="center">9.8&nbsp;× 10<sup>−380</sup></td><td rowspan="1" colspan="1" align="center">1.12 (0.81–1.54)</td><td rowspan="1" colspan="1">0.5071</td><td rowspan="1" colspan="1" align="center">3.32 (2.90–3.81)</td><td rowspan="1" colspan="1" align="center">2.3&nbsp;× 10<sup>−69</sup></td><td rowspan="1" colspan="1" align="center">3.2&nbsp;× 10<sup>−65</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">rs201374403</td><td rowspan="1" colspan="1">31383754</td><td rowspan="1" colspan="1"><span class="underline">T</span>/TAG</td><td rowspan="1" colspan="1">MICA</td><td rowspan="1" colspan="1" align="center">0.78 (0.7–0.88)</td><td rowspan="1" colspan="1">6.4&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.68 (1.56–1.8)</td><td rowspan="1" colspan="1" align="center">7.9&nbsp;× 10<sup>−48</sup></td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−53</sup></td><td rowspan="1" colspan="1" align="center">0.81 (0.67–0.96)</td><td rowspan="1" colspan="1">0.0174</td><td rowspan="1" colspan="1" align="center">1.65 (1.50–1.81)</td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−25</sup></td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−26</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">rs113573479</td><td rowspan="1" colspan="1">29842444</td><td rowspan="1" colspan="1"><span class="underline">A</span>/G</td><td rowspan="1" colspan="1">HLA-A</td><td rowspan="1" colspan="1" align="center">0.89 (0.81–0.97)</td><td rowspan="1" colspan="1">0.0109</td><td rowspan="1" colspan="1" align="center">1.39 (1.30–1.49)</td><td rowspan="1" colspan="1" align="center">6.6&nbsp;× 10<sup>−25</sup></td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−26</sup></td><td rowspan="1" colspan="1" align="center">0.92 (0.81–1.04)</td><td rowspan="1" colspan="1">0.1948</td><td rowspan="1" colspan="1" align="center">1.41 (1.30–1.52)</td><td rowspan="1" colspan="1" align="center">2.8&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">2.7&nbsp;× 10<sup>−17</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1">rs28383201</td><td rowspan="1" colspan="1">32574869</td><td rowspan="1" colspan="1"><span class="underline">C</span>/G</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1" align="center">0.59 (0.51–0.68)</td><td rowspan="1" colspan="1">4.6&nbsp;× 10<sup>−13</sup></td><td rowspan="1" colspan="1" align="center">1.15 (1.06–1.24)</td><td rowspan="1" colspan="1" align="center">4.5&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">3.3&nbsp;× 10<sup>−16</sup></td><td rowspan="1" colspan="1" align="center">0.61 (0.52–0.71)</td><td rowspan="1" colspan="1">3.4&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">1.18 (1.08–1.28)</td><td rowspan="1" colspan="1" align="center">1.0&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">6.5&nbsp;× 10<sup>−14</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1">rs1793889</td><td rowspan="1" colspan="1">31222181</td><td rowspan="1" colspan="1"><span class="underline">A</span>/G</td><td rowspan="1" colspan="1">HLA-C</td><td rowspan="1" colspan="1" align="center">0.60 (0.50–0.73)</td><td rowspan="1" colspan="1">2.5&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.18 (1.07–1.31)</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">0.60 (0.50–0.73)</td><td rowspan="1" colspan="1">2.5&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.18 (1.07–1.31)</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−9</sup></td></tr></tbody></table></div><div id="largeobj_idm139812600165568" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4289690/table/tbl3/?report=objectonly">Open in a separate window</a></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139812599645472" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="3" colspan="1"><strong>Data Source</strong></th><th rowspan="3" colspan="1"><strong>Effect</strong></th><th rowspan="3" colspan="1"><strong>HLA Allele</strong></th><th rowspan="3" colspan="1"><strong>HLA Allele Frequency in Ps/AD/Controls</strong></th><th colspan="5" rowspan="1"><strong>Conditional Models</strong><sup>a</sup><hr></th><th colspan="5" rowspan="1"><strong>Full Model</strong><hr></th></tr><tr><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>p</strong><sub><strong>overall</strong></sub></th><th colspan="2" rowspan="1"><strong>AD</strong><hr></th><th colspan="2" rowspan="1"><strong>Psoriasis</strong><hr></th><th rowspan="2" colspan="1"><strong>p</strong><sub><strong>overall</strong></sub></th></tr><tr><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>OR (95% CI)</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">C<sup>∗</sup>06:02</td><td rowspan="1" colspan="1">0.271/0.075/0.089</td><td rowspan="1" colspan="1" align="center">0.81 (0.71–0.91)</td><td rowspan="1" colspan="1">8.48&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">4.28 (3.98–4.61)</td><td rowspan="1" colspan="1" align="center">2.9&nbsp;× 10<sup>−362</sup></td><td rowspan="1" colspan="1" align="center">1.30&nbsp;× 10<sup>−368</sup></td><td rowspan="1" colspan="1" align="center">0.97 (0.82–1.15)</td><td rowspan="1" colspan="1" align="center">0.7475</td><td rowspan="1" colspan="1" align="center">3.59 (3.26–3.95)</td><td rowspan="1" colspan="1" align="center">2.1&nbsp;× 10<sup>−159</sup></td><td rowspan="1" colspan="1" align="center">8.7&nbsp;× 10<sup>−154</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">A<sup>∗</sup>02:01</td><td rowspan="1" colspan="1">0.28/0.227/0.239</td><td rowspan="1" colspan="1" align="center">0.95 (0.88–1.03)</td><td rowspan="1" colspan="1">0.1793</td><td rowspan="1" colspan="1" align="center">1.32 (1.24–1.40)</td><td rowspan="1" colspan="1" align="center">4.1&nbsp;× 10<sup>−20</sup></td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−20</sup></td><td rowspan="1" colspan="1" align="center">0.99 (0.90–1.08)</td><td rowspan="1" colspan="1" align="center">0.7739</td><td rowspan="1" colspan="1" align="center">1.32 (1.24–1.41)</td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−17</sup></td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−16</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">B<sup>∗</sup>57:01</td><td rowspan="1" colspan="1">0.13/0.017/0.032</td><td rowspan="1" colspan="1" align="center">0.61 (0.46–0.80)</td><td rowspan="1" colspan="1">3.83&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.63 (1.44–1.84)</td><td rowspan="1" colspan="1" align="center">8.0&nbsp;× 10<sup>−15</sup></td><td rowspan="1" colspan="1" align="center">5.8&nbsp;× 10<sup>−20</sup></td><td rowspan="1" colspan="1" align="center">0.60 (0.42–0.85)</td><td rowspan="1" colspan="1" align="center">0.0039</td><td rowspan="1" colspan="1" align="center">1.58 (1.38–1.81)</td><td rowspan="1" colspan="1" align="center">3.8&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">1.1&nbsp;× 10<sup>−13</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">C<sup>∗</sup>12:03</td><td rowspan="1" colspan="1">0.048/0.044/0.038</td><td rowspan="1" colspan="1" align="center">0.87 (0.74–1.03)</td><td rowspan="1" colspan="1">0.1169</td><td rowspan="1" colspan="1" align="center">1.74 (1.54–2.00)</td><td rowspan="1" colspan="1" align="center">5.2&nbsp;× 10<sup>−18</sup></td><td rowspan="1" colspan="1" align="center">3.5&nbsp;× 10<sup>−19</sup></td><td rowspan="1" colspan="1" align="center">0.86 (0.72–1.04)</td><td rowspan="1" colspan="1" align="center">0.1295</td><td rowspan="1" colspan="1" align="center">1.85 (1.62–2.11)</td><td rowspan="1" colspan="1" align="center">7.5&nbsp;× 10<sup>−20</sup></td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−20</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">B<sup>∗</sup>27:05</td><td rowspan="1" colspan="1">0.032/0.02/0.025</td><td rowspan="1" colspan="1" align="center">0.80 (0.63–1.01)</td><td rowspan="1" colspan="1">0.0619</td><td rowspan="1" colspan="1" align="center">1.59 (1.37–1.86)</td><td rowspan="1" colspan="1" align="center">2.9&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">3.2&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">0.74 (0.55–1.00)</td><td rowspan="1" colspan="1" align="center">0.0499</td><td rowspan="1" colspan="1" align="center">1.50 (1.28–1.76)</td><td rowspan="1" colspan="1" align="center">7.3&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">PSO</td><td rowspan="1" colspan="1">A<sup>∗</sup>01:01</td><td rowspan="1" colspan="1">0.221/0.154/0.166</td><td rowspan="1" colspan="1" align="center">1.03 (0.94–1.13)</td><td rowspan="1" colspan="1">0.5475</td><td rowspan="1" colspan="1" align="center">1.25 (1.17–1.35)</td><td rowspan="1" colspan="1" align="center">2.3&nbsp;× 10<sup>−10</sup></td><td rowspan="1" colspan="1" align="center">1.7&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">1.08 (0.96–1.21)</td><td rowspan="1" colspan="1" align="center">0.2013</td><td rowspan="1" colspan="1" align="center">1.23 (1.14–1.33)</td><td rowspan="1" colspan="1" align="center">6.8&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">4.0&nbsp;× 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1">C<sup>∗</sup>03:03</td><td rowspan="1" colspan="1">0.037/0.025/0.038</td><td rowspan="1" colspan="1" align="center">0.66 (0.54–0.81)</td><td rowspan="1" colspan="1">9.71&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.30 (1.13–1.49)</td><td rowspan="1" colspan="1" align="center">2.5&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">5.9&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.71 (0.56–0.90)</td><td rowspan="1" colspan="1" align="center">0.0040</td><td rowspan="1" colspan="1" align="center">1.27 (1.10–1.47)</td><td rowspan="1" colspan="1">0.0011</td><td rowspan="1" colspan="1" align="center">2.3&nbsp;× 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">GWAS</td><td rowspan="1" colspan="1">opposing</td><td rowspan="1" colspan="1">DQA1<sup>∗</sup>02:01</td><td rowspan="1" colspan="1">0.186/0.055/0.098</td><td rowspan="1" colspan="1" align="center">0.64 (0.54–0.76)</td><td rowspan="1" colspan="1">3.44&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.09 (1.01–1.19)</td><td rowspan="1" colspan="1">0.0385</td><td rowspan="1" colspan="1" align="center">6.0&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.64 (0.54–0.76)</td><td rowspan="1" colspan="1">3.4&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.09 (1.01–1.19)</td><td rowspan="1" colspan="1">0.0385</td><td rowspan="1" colspan="1" align="center">6.0&nbsp;× 10<sup>−8</sup></td></tr></tbody></table></div><div id="largeobj_idm139812599645472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4289690/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139812598278080"><p id="__p87" class="p p-first-last">Effect allele is underlined. Abbreviations are as follows: PSO, psoriasis; AD, atopic dermatitis. Table shows only independent loci (<em>r</em><sup>2</sup> &lt; 0.5) and the SNV with the strongest association.</p></div></div><div class="tblwrap-foot"><div id="tblfn7"><sup>a</sup>Stepwise conditional analysis was carried out with multinomial regression models and resulted in three psoriasis-specific and two opposing signals.</div></div></div></div><div id="sec3.6" class="sec sec-last"><h3 id="sec3.6title">Ontology and Network Analysis of Genes Indicate Effects in the Skin Barrier and Immune Response</h3><p id="__p38" class="p p-first-last">Genes implicated from genome-wide and conditional analyses (identified from <a href="/pmc/articles/PMC4289690/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Tables 1</span></a>, <a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,2,</span></span><span>2</span></a>, and <a href="/pmc/articles/PMC4289690/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3)</span></span><span>3</span></a>) were investigated via predicted protein network and gene ontology (GO) analysis. The results are summarized in <a href="#app2" rid="app2" class=" sec">Figure&nbsp;S6</a>. The GO term “keratinocyte differentiation” (GO:0030216) is enriched in genes implicated in AD and psoriasis risk (FDR p = 4.3&nbsp;× 10<sup>−4</sup> in AD; p = 6.9&nbsp;× 10<sup>−4</sup> in psoriasis; and p = 2.7&nbsp;× 10<sup>−3</sup> in opposing effects). The GO term “response to interferon-gamma” (GO:0034341) is also significantly enriched in psoriasis (FDR p = 1.9&nbsp;× 10<sup>−3</sup>).</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p39" class="p p-first">This genome-wide comparative analysis confirms a high degree of genomic coincidence between AD and psoriasis, suggesting that common molecular mechanisms are involved. This agrees with the central role of epidermal barrier defects and T-cell-dominated inflammation in both diseases.<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> Within the six regions of colocalization, we demonstrate coassociated and independent disease-specific loci. Of note, all coassociated loci display opposing (antagonistic) effects on AD and psoriasis, in agreement with the epidemiological observations of lower-than-expected&nbsp;coincidence between these diseases in the population.<sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup> Within these loci, specific variants including chromosome 2q31.2 (rs62176107), chromosome 5q33.1 (rs17728338), and within <em>RAD50</em> on chromosome 5q33.3 (rs6596086) demonstrate opposing effects on risk of AD and psoriasis. This raises the intriguing possibility that the same biological mechanisms might act differentially on AD versus psoriasis. However, our current data cannot distinguish this specific opposing mechanism from the possibility that each lead variant is in LD with other variants having opposing effects in each disease.</p><p id="__p40">The majority of the opposing effect loci are implicated in pathways related to adaptive immunological functions, which potentially mirrors the polarized immune mechanisms.<sup><a href="#bib6" rid="bib6" class=" bibr popnode">6</a></sup> It might further be speculated whether the presence of multiple opposing alleles reflects balancing selection as a response to heterogeneity in environmental pressures. Balancing selection is particularly common within the extended MHC region and has been proposed as a potential explanation for antagonistic effects at multiple loci in different autoimmune diseases.<sup><a href="#bib56" rid="bib56" class=" bibr popnode">56</a></sup></p><p id="__p41">Two of the loci displaying opposing effects (<em>ANXA6/TNIP1</em> and <em>PRKRA</em>) have not previously been reported in association with psoriasis and/or AD. Formal external validation is limited by the requirement for additional independent, population-matched GWAS data for AD and psoriasis, but data available from RNA sequencing and microarray analyses provide some support for the differential expression of <em>ANXA6/TNIP1</em> and <em>PRKRA</em> in AD and psoriasis, relative to normal or uninvolved skin. The lead variant within <em>PRKRA</em> might mediate opposing effects in AD and psoriasis via miRNA processing and/or cellular response to environmental stress, and we hypothesize that this reflects the striking differential susceptibility to viral and bacterial&nbsp;skin infections observed in AD and psoriasis. The opposing effect of variation in <em>ANXA6</em> suggests a role for calcium-dependent effects in defining patterns of skin inflammation.</p><p id="__p42">On chromosome 1q21.3, apart from well-established AD-associated <em>FLG</em> mutations and psoriasis-associated deletion of <em>LCE3B</em>-<em>LCE3C</em>, <em>FLG-AS1</em> is a plausible candidate to mediate differential AD/psoriasis risk via the network of regulatory elements coordinating gene expression.<sup><a href="#bib57" rid="bib57" class=" bibr popnode">57</a></sup> Natural antisense transcripts contribute to gene regulation via a variety of transcriptional and posttranscriptional mechanisms<sup><a href="#bib58" rid="bib58" class=" bibr popnode">58</a></sup> and include effects on human epidermal differentiation.<sup><a href="#bib59" rid="bib59" class=" bibr popnode">59</a></sup> The proximity of <em>FLG-AS1</em> to <em>FLG</em> and <em>HRNR</em>, combined with data showing coordinated differential expression of these genes, supports a role in control of keratinocyte terminal differentiation.</p><p id="__p43">On chromosome 5q31.1, antagonistic signals for AD and psoriasis have previously been attributed to <em>IL13</em>.<sup><a href="#bib10" rid="bib10 bib11 bib21" class=" bibr popnode">10,11,21</a></sup> We here show that <em>IL13</em> polymorphisms specifically influence AD risk, whereas opposing signals map to <em>RAD50</em>. The Rad50 protein, a component of the MRN complex (Mre11, Rad50, and Nbs1), is involved in DNA double-strand break repair but has no known function directly related to AD or psoriasis. However, <em>RAD50</em> mRNA shows significantly increased expression in psoriasis lesional skin and a trend to reduced expression in AD lesional skin (<sup><a href="#bib7" rid="bib7" class=" bibr popnode">Table S7</a></sup>). Of note, <em>RAD50</em> is located in the center of the Th2-cytokine cluster and its 3′ end is part of a locus control region regulating expression of these cytokine genes.<sup><a href="#bib60" rid="bib60" class=" bibr popnode">60</a></sup> AD and psoriasis represent opposing extremes of Th2 cell dysregulation, and therefore we hypothesize that <em>RAD50</em> polymorphisms might exert opposing effects on AD and psoriasis through variation in DNA repair resulting in a differential skew in Th2 cell response.</p><p id="__p44">Our dissection of the MHC locus confirms the presence of multiple independent psoriasis-risk loci. Markers tagging HLA-Cw<sup>∗</sup>0602 generate the strongest effects, which is in line with previous reports.<sup><a href="#bib17" rid="bib17 bib21 bib24 bib44 bib61" class=" bibr popnode">17,21,24,44,61</a></sup> CD8<sup>+</sup> T&nbsp;cells are increased in the epidermis of lesional psoriatic skin, and the association of psoriasis susceptibility primarily with class I HLA alleles might reflect the critical role of psoriasis-associated (auto-)antigen presentation to pathogenic CD8<sup>+</sup> T&nbsp;cells.<sup><a href="#bib62" rid="bib62" class=" bibr popnode">62</a></sup> CD8<sup>+</sup> T&nbsp;cells are also increased in the epidermis of AD skin, but with strikingly different cytokine profiles compared to psoriasis.<sup><a href="#bib63" rid="bib63" class=" bibr popnode">63</a></sup> The opposing effects of class II HLA alleles in AD and psoriasis might represent the differential responses to pathogenic and allergenic peptides presented to CD4<sup>+</sup> T&nbsp;cells.<sup><a href="#bib64" rid="bib64" class=" bibr popnode">64</a></sup> GWASs in AD by univariate and multivariate models have reported association signals in the MHC class I and II regions<sup><a href="#bib22" rid="bib22 bib65" class=" bibr popnode">22,65</a></sup> and two specific HLA class II haplotypes, HLA-DRB1<sup>∗</sup>0701 (a protective effect) and HLA-B<sup>∗</sup>4402 (a risk effect).<sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup> Our analysis confirms the association of classical HLA class II alleles with AD, but in the conditional analysis, only HLA-DQA1<sup>∗</sup>02:01 remained, showing a significant protective effect on AD and a significant opposing effect on psoriasis. A further opposing locus mapped to HLA-C<sup>∗</sup>03:03 (<a href="/pmc/articles/PMC4289690/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>).</p><p id="__p45">The reported observation of AD occurring within the offspring of parents with psoriasis<sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup> is not supported by our findings, and the observation that both Th1-cell-dominated autoimmune and Th2-cell-dominated allergic diseases can show aggregation within families<sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup> also presents a discrepancy with our analyses. It is possible that there are shared risk loci for AD and psoriasis that were not detected in our current study because of lack of power, if the shared effect is not strong; alternatively, there might be hereditary risk factors associated with predisposition to any chronic inflammatory (auto-)immune disease. It is also possible that diagnostic misclassification occurs, particularly in pediatric cases, where the clinical signs of psoriasis are more difficult to distinguish from AD than is the case in adult disease,<sup><a href="#bib66" rid="bib66" class=" bibr popnode">66</a></sup> or by recall bias for disease in parents.</p><p id="__p46">It is interesting to estimate the extent to which our findings can explain the mutual exclusivity of AD and psoriasis, but an accurate assessment is hindered by the lack of published data on the proportion of AD and psoriasis cases where the diseases do and do not co-occur. Henseler et&nbsp;al. report a 25-fold lower prevalence of AD occurring in psoriasis cases<sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup> and assuming a prevalence of 10% and 2% for AD and psoriasis, respectively,<sup><a href="#bib3" rid="bib3 bib4" class=" bibr popnode">3,4</a></sup> we estimate that the effects at the six opposing loci listed in <a href="/pmc/articles/PMC4289690/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Tables 1</span></a>, <a href="/pmc/articles/PMC4289690/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -0.5em;">,2,</span></span><span>2</span></a>, and <a href="/pmc/articles/PMC4289690/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> would result in a reduction in prevalence of AD from 10% to 8% within the group of individuals with psoriasis. This 2% reduction contrasts with the 25-fold reduction reported by Henseler et&nbsp;al.,<sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup> which is equivalent to a reduction of 9.6%, from 10% in the population to 0.4%. Our results have therefore explained approximately 21% (2/9.6&nbsp;× 100) of the mutual exclusivity of AD and psoriasis.</p><p id="__p47">Taken together, our comparative analyses of AD and psoriasis support a paradigm in which genetic factors determining keratinocyte differentiation and cutaneous barrier function have particularly strong effects on AD risk, whereas in psoriasis genetic factors influencing (auto-)antigen recognition are of paramount importance. Furthermore, multiple pleiotropic loci with antagonistic effects contribute to opposing mechanisms of adaptive immunity in both AD and psoriasis.</p><p id="__p48" class="p p-last">The meta-analysis-inspired methodology developed in the course of this study has demonstrated the power to leverage additional information from GWAS and high-density SNV data and to dissect cross-phenotype associations. AD and psoriasis are particularly well suited to the compare/contrast approach, but this methodology will be applicable to many other complex traits with overlapping and disease-specific phenotypic features. Characterizing shared and opposing molecular mechanisms across complex phenotypes will expand our understanding of biology and disease and will have implications for treatment and drug discovery.</p></div><div id="sec5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec5title">Consortia</h2><p id="__p49" class="p p-first-last">Membership of the PAGE (Population Architecture using Genomics and Epidemiology) consortium is as follows: Trilokraj Tejasvi, Johann E. Gudjonsson, John J. Voorhees, Jun Ding, Yanming Li, Hyun M. Kang, Goncalo R. Abecasis, Dafna D. Gladman, Fawnda J. Pellett, Vinod Chandran, Cheryl F. Rosen, Proton Rahman, Sulev Koks, Külli Kingo, Tonu Esko, Andres Metspalu, Peter Gregersen, Andrew Henschel, Marin Aurand, Bruce Bebo, and Henry W. Lim.</p></div><div id="ack0010" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="__p61">We are grateful to the individuals who provided clinical data and DNA for research. This study makes use of genome-wide analysis data generated by the Wellcome Trust Case-Control Consortium (WTCCC); a full list of investigators is available at <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk</a>. Funding for the WTCCC was provided by Wellcome Trust awards 076113 and 085475. This work was additionally supported by the Wellcome Trust through an Intermediate Clinical Fellowship (WT086398MA) to S.J.B., Senior Research Fellowship in Basic Biomedical Science (087436/Z/08/Z and 102858/Z/13/Z) to H.J.C., Programme Grant (092530/Z/10/Z) to W.H.I.M. and A.D.I., and Strategic Award (098439/Z/12/Z) to W.H.I.M. The KORA research platform was initiated and financed by the Helmholtz Zentrum München - German Research Center for Environmental Health, funded by the German Federal Ministry of Education and Research and by the State of Bavaria. KORA research was supported within the Munich Center of Health Sciences (MC Health), Ludwig-Maximilians-Universität, as part of LMUinnovativ. The project received infrastructure support through DFG Clusters of Excellence “Inflammation at Interfaces” (grants EXC306 and EXC306/2) and the German Federal Ministry of Education and Research within the framework of e:Med research and funding concept (sysINFLAME, 01ZX1306A). M.M.N. is a member of the Cluster of Excellence “ImmunoSensation.” The NCRC-ADC is supported by the National Children’s Research Centre, Dublin. We also acknowledge use of Trinity Biobank samples from the Irish Blood Transfusion Service. This work was supported by the NIH (R01AR042742, R01AR050511, R01AR054966, R01AR062886-01, R01AR062382) and by the Babcock Memorial Trust. J.T.E. is supported by the Ann Arbor Veterans Affairs Hospital.</p></div></div><div id="idm139812676599472" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139812676599472title">Notes</h2><p id="misc0010">Published: January 8, 2015</p></div><div id="idm139812598682160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139812598682160title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="d32e2347"></p><p id="__p62" class="p p-first-last">This is an open access article under the CC BY license (<a href="http://creativecommons.org/licenses/by/3.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/3.0/</a>).</p><p></p></div></div><div id="app2" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app2title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Supplemental Consortia Information, Figures S1–S6, and Tables S1–S4 and S6:</strong><div class="sup-box half_rhythm" id="idm139812586984976"><a href="/pmc/articles/PMC4289690/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.2M, pdf)</sup></div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Table S5. Results of CCMA and MNM Analyses Showing Significance Levels and Comparison with Known Susceptibility Loci for AD and Psoriasis: </strong><p id="__p51">Lead SNP defined by clumping procedure (Purcell, S. et&nbsp;al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81, 559-75 2007); genomewide significant results defined as Tmax&gt;6, BFDP&lt;0.05 with PO=1/999 or PMNM&lt;10-8 are marked in bold; genes and transcripts identified from UCSC Genome Browser Human Feb. 2009 (GRCh37/hg19) Assembly accessed 21 March 2014.</p><div class="sup-box half_rhythm" id="idm139812600744464"><a href="/pmc/articles/PMC4289690/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(79K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc3"><!--caption a9--><strong>Table S7. Gene Expression Data in Psoriasis and AD Relating to Candidate Genes Identified by Comparative Analysis: </strong><p id="__p52">Gene expression data available from published and publically available data sets are summarized for candidate genes.</p><div class="sup-box half_rhythm" id="idm139812599689888"><a href="/pmc/articles/PMC4289690/bin/mmc3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(65K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc4"><!--caption a9--><strong>Document S2. Article plus Supplemental Data:</strong><div class="sup-box half_rhythm" id="idm139812599687344"><a href="/pmc/articles/PMC4289690/bin/mmc4.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(4.2M, pdf)</sup></div></div><p></p></div><div id="app3" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Web Resources</h2><!--/article/back/sec/--><p id="__p53" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p54">1000 Genomes, <a href="http://browser.1000genomes.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://browser.1000genomes.org</a></p></li><li id="u0015"><p id="__p55">Ensembl Genome Browser, <a href="http://www.ensembl.org/index.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ensembl.org/index.html</a></p></li><li id="u0020"><p id="__p56">Online Mendelian Inheritance in Man (OMIM), <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a></p></li><li id="u0025"><p id="__p57">PLINK, <a href="http://pngu.mgh.harvard.edu/%7Epurcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a></p></li><li id="u0030"><p id="__p58">R statistical software, <a href="http://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.r-project.org/</a></p></li><li id="u0035"><p id="__p59">RefSeq, <a href="http://www.ncbi.nlm.nih.gov/RefSeq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/RefSeq</a></p></li><li id="u0040"><p id="__p60">STRING 9.1, <a href="http://www.string-db.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.string-db.org/</a></p></li></ul><p></p></div><div id="idm139812603575856" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139812603575856title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Deckert S., Kopkow C., Schmitt J. Nonallergic comorbidities of atopic eczema: an overview of systematic reviews. <span><span class="ref-journal">Allergy. </span>2014;<span class="ref-vol">69</span>:37–45.</span> [<a href="/pubmed/24053642" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Allergy&amp;title=Nonallergic+comorbidities+of+atopic+eczema:+an+overview+of+systematic+reviews&amp;author=S.+Deckert&amp;author=C.+Kopkow&amp;author=J.+Schmitt&amp;volume=69&amp;publication_year=2014&amp;pages=37-45&amp;pmid=24053642&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Griffiths C.E., Barker J.N. Pathogenesis and clinical features of psoriasis. <span><span class="ref-journal">Lancet. </span>2007;<span class="ref-vol">370</span>:263–271.</span> [<a href="/pubmed/17658397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Pathogenesis+and+clinical+features+of+psoriasis&amp;author=C.E.+Griffiths&amp;author=J.N.+Barker&amp;volume=370&amp;publication_year=2007&amp;pages=263-271&amp;pmid=17658397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Parisi R., Symmons D.P., Griffiths C.E., Ashcroft D.M., Identification and Management of Psoriasis and Associated ComorbidiTy (IMPACT) project team  Global epidemiology of psoriasis: a systematic review of incidence and prevalence. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2013;<span class="ref-vol">133</span>:377–385.</span> [<a href="/pubmed/23014338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Global+epidemiology+of+psoriasis:+a+systematic+review+of+incidence+and+prevalence&amp;author=R.+Parisi&amp;author=D.P.+Symmons&amp;author=C.E.+Griffiths&amp;author=D.M.+Ashcroft&amp;volume=133&amp;publication_year=2013&amp;pages=377-385&amp;pmid=23014338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Deckers I.A., McLean S., Linssen S., Mommers M., van Schayck C.P., Sheikh A. Investigating international time trends in the incidence and prevalence of atopic eczema 1990-2010: a systematic review of epidemiological studies. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e39803.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3394782/">PMC free article</a>]</span> [<a href="/pubmed/22808063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Investigating+international+time+trends+in+the+incidence+and+prevalence+of+atopic+eczema+1990-2010:+a+systematic+review+of+epidemiological+studies&amp;author=I.A.+Deckers&amp;author=S.+McLean&amp;author=S.+Linssen&amp;author=M.+Mommers&amp;author=C.P.+van+Schayck&amp;volume=7&amp;publication_year=2012&amp;pages=e39803&amp;pmid=22808063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Henseler T., Christophers E. Disease concomitance in psoriasis. <span><span class="ref-journal">J.&nbsp;Am. Acad. Dermatol. </span>1995;<span class="ref-vol">32</span>:982–986.</span> [<a href="/pubmed/7751469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Acad.+Dermatol.&amp;title=Disease+concomitance+in+psoriasis&amp;author=T.+Henseler&amp;author=E.+Christophers&amp;volume=32&amp;publication_year=1995&amp;pages=982-986&amp;pmid=7751469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Eyerich S., Onken A.T., Weidinger S., Franke A., Nasorri F., Pennino D., Grosber M., Pfab F., Schmidt-Weber C.B., Mempel M. Mutual antagonism of T&nbsp;cells causing psoriasis and atopic eczema. <span><span class="ref-journal">N.&nbsp;Engl. J. Med. </span>2011;<span class="ref-vol">365</span>:231–238.</span> [<a href="/pubmed/21774711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.&nbsp;Engl.+J.+Med.&amp;title=Mutual+antagonism+of+T&nbsp;cells+causing+psoriasis+and+atopic+eczema&amp;author=S.+Eyerich&amp;author=A.T.+Onken&amp;author=S.+Weidinger&amp;author=A.+Franke&amp;author=F.+Nasorri&amp;volume=365&amp;publication_year=2011&amp;pages=231-238&amp;pmid=21774711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Hemminki K., Li X., Sundquist K., Sundquist J. Shared familial aggregation of susceptibility to autoimmune diseases. <span><span class="ref-journal">Arthritis Rheum. </span>2009;<span class="ref-vol">60</span>:2845–2847.</span> [<a href="/pubmed/19714648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Shared+familial+aggregation+of+susceptibility+to+autoimmune+diseases&amp;author=K.+Hemminki&amp;author=X.+Li&amp;author=K.+Sundquist&amp;author=J.+Sundquist&amp;volume=60&amp;publication_year=2009&amp;pages=2845-2847&amp;pmid=19714648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Maas T., Nieuwhof C., Passos V.L., Robertson C., Boonen A., Landewé R.B., Voncken J.W., Knottnerus J.A., Damoiseaux J.G. Transgenerational occurrence of allergic disease and autoimmunity: general practice-based epidemiological research. <span><span class="ref-journal">Prim. Care Respir. J. </span>2014;<span class="ref-vol">23</span>:14–21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6442276/">PMC free article</a>]</span> [<a href="/pubmed/24449016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prim.+Care+Respir.+J.&amp;title=Transgenerational+occurrence+of+allergic+disease+and+autoimmunity:+general+practice-based+epidemiological+research&amp;author=T.+Maas&amp;author=C.+Nieuwhof&amp;author=V.L.+Passos&amp;author=C.+Robertson&amp;author=A.+Boonen&amp;volume=23&amp;publication_year=2014&amp;pages=14-21&amp;pmid=24449016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Cookson W.O., Ubhi B., Lawrence R., Abecasis G.R., Walley A.J., Cox H.E., Coleman R., Leaves N.I., Trembath R.C., Moffatt M.F., Harper J.I. Genetic linkage of childhood atopic dermatitis to psoriasis susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2001;<span class="ref-vol">27</span>:372–373.</span> [<a href="/pubmed/11279517" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genetic+linkage+of+childhood+atopic+dermatitis+to+psoriasis+susceptibility+loci&amp;author=W.O.+Cookson&amp;author=B.+Ubhi&amp;author=R.+Lawrence&amp;author=G.R.+Abecasis&amp;author=A.J.+Walley&amp;volume=27&amp;publication_year=2001&amp;pages=372-373&amp;pmid=11279517&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Paternoster L., Standl M., Chen C.M., Ramasamy A., Bønnelykke K., Duijts L., Ferreira M.A., Alves A.C., Thyssen J.P., Albrecht E., Australian Asthma Genetics Consortium (AAGC) Genetics of Overweight Young Adults (GOYA) Consortium. EArly Genetics &amp; Lifecourse Epidemiology (EAGLE) Consortium  Meta-analysis of genome-wide association studies identifies three new risk loci for atopic dermatitis. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:187–192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3272375/">PMC free article</a>]</span> [<a href="/pubmed/22197932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+three+new+risk+loci+for+atopic+dermatitis&amp;author=L.+Paternoster&amp;author=M.+Standl&amp;author=C.M.+Chen&amp;author=A.+Ramasamy&amp;author=K.+Bønnelykke&amp;volume=44&amp;publication_year=2012&amp;pages=187-192&amp;pmid=22197932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Ellinghaus D., Baurecht H., Esparza-Gordillo J., Rodríguez E., Matanovic A., Marenholz I., Hübner N., Schaarschmidt H., Novak N., Michel S. High-density genotyping study identifies four new susceptibility loci for atopic dermatitis. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:808–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3797441/">PMC free article</a>]</span> [<a href="/pubmed/23727859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=High-density+genotyping+study+identifies+four+new+susceptibility+loci+for+atopic+dermatitis&amp;author=D.+Ellinghaus&amp;author=H.+Baurecht&amp;author=J.+Esparza-Gordillo&amp;author=E.+Rodríguez&amp;author=A.+Matanovic&amp;volume=45&amp;publication_year=2013&amp;pages=808-812&amp;pmid=23727859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Tsoi L.C., Spain S.L., Knight J., Ellinghaus E., Stuart P.E., Capon F., Ding J., Li Y., Tejasvi T., Gudjonsson J.E., Collaborative Association Study of Psoriasis (CASP) Genetic Analysis of Psoriasis Consortium. Psoriasis Association Genetics Extension. Wellcome Trust Case Control Consortium 2  Identification of 15 new psoriasis susceptibility loci highlights the role of innate immunity. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1341–1348.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510312/">PMC free article</a>]</span> [<a href="/pubmed/23143594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+15+new+psoriasis+susceptibility+loci+highlights+the+role+of+innate+immunity&amp;author=L.C.+Tsoi&amp;author=S.L.+Spain&amp;author=J.+Knight&amp;author=E.+Ellinghaus&amp;author=P.E.+Stuart&amp;volume=44&amp;publication_year=2012&amp;pages=1341-1348&amp;pmid=23143594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Palmer C.N., Irvine A.D., Terron-Kwiatkowski A., Zhao Y., Liao H., Lee S.P., Goudie D.R., Sandilands A., Campbell L.E., Smith F.J. Common loss-of-function variants of the epidermal barrier protein filaggrin are a major predisposing factor for atopic dermatitis. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:441–446.</span> [<a href="/pubmed/16550169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+loss-of-function+variants+of+the+epidermal+barrier+protein+filaggrin+are+a+major+predisposing+factor+for+atopic+dermatitis&amp;author=C.N.+Palmer&amp;author=A.D.+Irvine&amp;author=A.+Terron-Kwiatkowski&amp;author=Y.+Zhao&amp;author=H.+Liao&amp;volume=38&amp;publication_year=2006&amp;pages=441-446&amp;pmid=16550169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Rodríguez E., Baurecht H., Herberich E., Wagenpfeil S., Brown S.J., Cordell H.J., Irvine A.D., Weidinger S. Meta-analysis of filaggrin polymorphisms in eczema and asthma: robust risk factors in atopic disease. <span><span class="ref-journal">J.&nbsp;Allergy Clin. Immunol. </span>2009;<span class="ref-vol">123</span>:1361–1370.e7.</span> [<a href="/pubmed/19501237" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Allergy+Clin.+Immunol.&amp;title=Meta-analysis+of+filaggrin+polymorphisms+in+eczema+and+asthma:+robust+risk+factors+in+atopic+disease&amp;author=E.+Rodríguez&amp;author=H.+Baurecht&amp;author=E.+Herberich&amp;author=S.+Wagenpfeil&amp;author=S.J.+Brown&amp;volume=123&amp;publication_year=2009&amp;pages=1361-1370.e7&amp;pmid=19501237&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Hüffmeier U., Traupe H., Oji V., Lascorz J., Ständer M., Lohmann J., Wendler J., Burkhardt H., Reis A. Loss-of-function variants of the filaggrin gene are not major susceptibility factors for psoriasis vulgaris or psoriatic arthritis in German patients. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2007;<span class="ref-vol">127</span>:1367–1370.</span> [<a href="/pubmed/17255953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Loss-of-function+variants+of+the+filaggrin+gene+are+not+major+susceptibility+factors+for+psoriasis+vulgaris+or+psoriatic+arthritis+in+German+patients&amp;author=U.+Hüffmeier&amp;author=H.+Traupe&amp;author=V.+Oji&amp;author=J.+Lascorz&amp;author=M.+Ständer&amp;volume=127&amp;publication_year=2007&amp;pages=1367-1370&amp;pmid=17255953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Zhao Y., Terron-Kwiatkowski A., Liao H., Lee S.P., Allen M.H., Hull P.R., Campbell L.E., Trembath R.C., Capon F., Griffiths C.E. Filaggrin null alleles are not associated with psoriasis. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2007;<span class="ref-vol">127</span>:1878–1882.</span> [<a href="/pubmed/17410197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Filaggrin+null+alleles+are+not+associated+with+psoriasis&amp;author=Y.+Zhao&amp;author=A.+Terron-Kwiatkowski&amp;author=H.+Liao&amp;author=S.P.+Lee&amp;author=M.H.+Allen&amp;volume=127&amp;publication_year=2007&amp;pages=1878-1882&amp;pmid=17410197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Zhang X.J., Huang W., Yang S., Sun L.D., Zhang F.Y., Zhu Q.X., Zhang F.R., Zhang C., Du W.H., Pu X.M. Psoriasis genome-wide association study identifies susceptibility variants within LCE gene cluster at 1q21. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:205–210.</span> [<a href="/pubmed/19169255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Psoriasis+genome-wide+association+study+identifies+susceptibility+variants+within+LCE+gene+cluster+at+1q21&amp;author=X.J.+Zhang&amp;author=W.+Huang&amp;author=S.+Yang&amp;author=L.D.+Sun&amp;author=F.Y.+Zhang&amp;volume=41&amp;publication_year=2009&amp;pages=205-210&amp;pmid=19169255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">de Cid R., Riveira-Munoz E., Zeeuwen P.L., Robarge J., Liao W., Dannhauser E.N., Giardina E., Stuart P.E., Nair R., Helms C. Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:211–215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128734/">PMC free article</a>]</span> [<a href="/pubmed/19169253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Deletion+of+the+late+cornified+envelope+LCE3B+and+LCE3C+genes+as+a+susceptibility+factor+for+psoriasis&amp;author=R.+de+Cid&amp;author=E.+Riveira-Munoz&amp;author=P.L.+Zeeuwen&amp;author=J.+Robarge&amp;author=W.+Liao&amp;volume=41&amp;publication_year=2009&amp;pages=211-215&amp;pmid=19169253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Bergboer J.G., Zeeuwen P.L., Irvine A.D., Weidinger S., Giardina E., Novelli G., Den Heijer M., Rodriguez E., Illig T., Riveira-Munoz E. Deletion of Late Cornified Envelope 3B and 3C genes is not associated with atopic dermatitis. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2010;<span class="ref-vol">130</span>:2057–2061.</span> [<a href="/pubmed/20376060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Deletion+of+Late+Cornified+Envelope+3B+and+3C+genes+is+not+associated+with+atopic+dermatitis&amp;author=J.G.+Bergboer&amp;author=P.L.+Zeeuwen&amp;author=A.D.+Irvine&amp;author=S.+Weidinger&amp;author=E.+Giardina&amp;volume=130&amp;publication_year=2010&amp;pages=2057-2061&amp;pmid=20376060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Weidinger S., Willis-Owen S.A., Kamatani Y., Baurecht H., Morar N., Liang L., Edser P., Street T., Rodriguez E., O’Regan G.M. A genome-wide association study of atopic dermatitis identifies loci with overlapping effects on asthma and psoriasis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:4841–4856.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3820131/">PMC free article</a>]</span> [<a href="/pubmed/23886662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+atopic+dermatitis+identifies+loci+with+overlapping+effects+on+asthma+and+psoriasis&amp;author=S.+Weidinger&amp;author=S.A.+Willis-Owen&amp;author=Y.+Kamatani&amp;author=H.+Baurecht&amp;author=N.+Morar&amp;volume=22&amp;publication_year=2013&amp;pages=4841-4856&amp;pmid=23886662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Nair R.P., Duffin K.C., Helms C., Ding J., Stuart P.E., Goldgar D., Gudjonsson J.E., Li Y., Tejasvi T., Feng B.J., Collaborative Association Study of Psoriasis  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=R.P.+Nair&amp;author=K.C.+Duffin&amp;author=C.+Helms&amp;author=J.+Ding&amp;author=P.E.+Stuart&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Hirota T., Takahashi A., Kubo M., Tsunoda T., Tomita K., Sakashita M., Yamada T., Fujieda S., Tanaka S., Doi S. Genome-wide association study identifies eight new susceptibility loci for atopic dermatitis in the Japanese population. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1222–1226.</span> [<a href="/pubmed/23042114" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+eight+new+susceptibility+loci+for+atopic+dermatitis+in+the+Japanese+population&amp;author=T.+Hirota&amp;author=A.+Takahashi&amp;author=M.+Kubo&amp;author=T.+Tsunoda&amp;author=K.+Tomita&amp;volume=44&amp;publication_year=2012&amp;pages=1222-1226&amp;pmid=23042114&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Stuart P.E., Nair R.P., Ellinghaus E., Ding J., Tejasvi T., Gudjonsson J.E., Li Y., Weidinger S., Eberlein B., Gieger C. Genome-wide association analysis identifies three psoriasis susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1000–1004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965799/">PMC free article</a>]</span> [<a href="/pubmed/20953189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+three+psoriasis+susceptibility+loci&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=E.+Ellinghaus&amp;author=J.+Ding&amp;author=T.+Tejasvi&amp;volume=42&amp;publication_year=2010&amp;pages=1000-1004&amp;pmid=20953189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Strange A., Capon F., Spencer C.C., Knight J., Weale M.E., Allen M.H., Barton A., Band G., Bellenguez C., Bergboer J.G., Genetic Analysis of Psoriasis Consortium &amp; the Wellcome Trust Case Control Consortium 2  A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:985–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749730/">PMC free article</a>]</span> [<a href="/pubmed/20953190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+new+psoriasis+susceptibility+loci+and+an+interaction+between+HLA-C+and+ERAP1&amp;author=A.+Strange&amp;author=F.+Capon&amp;author=C.C.+Spencer&amp;author=J.+Knight&amp;author=M.E.+Weale&amp;volume=42&amp;publication_year=2010&amp;pages=985-990&amp;pmid=20953190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Williams H.C., Burney P.G., Pembroke A.C., Hay R.J. The U.K. Working Party’s Diagnostic Criteria for Atopic Dermatitis. III. Independent hospital validation. <span><span class="ref-journal">Br. J. Dermatol. </span>1994;<span class="ref-vol">131</span>:406–416.</span> [<a href="/pubmed/7918017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=The+U.K.+Working+Party’s+Diagnostic+Criteria+for+Atopic+Dermatitis.+III.+Independent+hospital+validation&amp;author=H.C.+Williams&amp;author=P.G.+Burney&amp;author=A.C.+Pembroke&amp;author=R.J.+Hay&amp;volume=131&amp;publication_year=1994&amp;pages=406-416&amp;pmid=7918017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Krawczak M., Nikolaus S., von Eberstein H., Croucher P.J., El Mokhtari N.E., Schreiber S. PopGen: population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. <span><span class="ref-journal">Community Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> [<a href="/pubmed/16490960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Community+Genet.&amp;title=PopGen:+population-based+recruitment+of+patients+and+controls+for+the+analysis+of+complex+genotype-phenotype+relationships&amp;author=M.+Krawczak&amp;author=S.+Nikolaus&amp;author=H.+von+Eberstein&amp;author=P.J.+Croucher&amp;author=N.E.+El+Mokhtari&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;pmid=16490960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Wichmann H.E., Gieger C., Illig T., MONICA/KORA Study Group  KORA-gen—resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen—resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=H.E.+Wichmann&amp;author=C.+Gieger&amp;author=T.+Illig&amp;volume=67&amp;issue=1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Weiland S.K., von Mutius E., Hirsch T., Duhme H., Fritzsch C., Werner B., Hüsing A., Stender M., Renz H., Leupold W., Keil U. Prevalence of respiratory and atopic disorders among children in the East and West of Germany five years after unification. <span><span class="ref-journal">Eur. Respir. J. </span>1999;<span class="ref-vol">14</span>:862–870.</span> [<a href="/pubmed/10573234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Respir.+J.&amp;title=Prevalence+of+respiratory+and+atopic+disorders+among+children+in+the+East+and+West+of+Germany+five+years+after+unification&amp;author=S.K.+Weiland&amp;author=E.+von+Mutius&amp;author=T.+Hirsch&amp;author=H.+Duhme&amp;author=C.+Fritzsch&amp;volume=14&amp;publication_year=1999&amp;pages=862-870&amp;pmid=10573234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">O’Donovan M.C., Craddock N., Norton N., Williams H., Peirce T., Moskvina V., Nikolov I., Hamshere M., Carroll L., Georgieva L., Molecular Genetics of Schizophrenia Collaboration  Identification of loci associated with schizophrenia by genome-wide association and follow-up. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1053–1055.</span> [<a href="/pubmed/18677311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+loci+associated+with+schizophrenia+by+genome-wide+association+and+follow-up&amp;author=M.C.+O’Donovan&amp;author=N.+Craddock&amp;author=N.+Norton&amp;author=H.+Williams&amp;author=T.+Peirce&amp;volume=40&amp;publication_year=2008&amp;pages=1053-1055&amp;pmid=18677311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref31">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref32">Delaneau O., Marchini J., Zagury J.F. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat. Methods. </span>2012;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O.+Delaneau&amp;author=J.+Marchini&amp;author=J.F.+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref33">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref34">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref35">Dilthey A.T., Moutsianas L., Leslie S., McVean G. HLA<sup>∗</sup>IMP—an integrated framework for imputing classical HLA alleles from SNP genotypes. <span><span class="ref-journal">Bioinformatics. </span>2011;<span class="ref-vol">27</span>:968–972.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3065693/">PMC free article</a>]</span> [<a href="/pubmed/21300701" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=HLA∗IMP—an+integrated+framework+for+imputing+classical+HLA+alleles+from+SNP+genotypes&amp;author=A.T.+Dilthey&amp;author=L.+Moutsianas&amp;author=S.+Leslie&amp;author=G.+McVean&amp;volume=27&amp;publication_year=2011&amp;pages=968-972&amp;pmid=21300701&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref36">Leslie S., Donnelly P., McVean G. A statistical method for predicting classical HLA alleles from SNP data. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:48–56.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2253983/">PMC free article</a>]</span> [<a href="/pubmed/18179884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+statistical+method+for+predicting+classical+HLA+alleles+from+SNP+data&amp;author=S.+Leslie&amp;author=P.+Donnelly&amp;author=G.+McVean&amp;volume=82&amp;publication_year=2008&amp;pages=48-56&amp;pmid=18179884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref37">Bhattacharjee S., Rajaraman P., Jacobs K.B., Wheeler W.A., Melin B.S., Hartge P., Yeager M., Chung C.C., Chanock S.J., Chatterjee N., GliomaScan Consortium  A subset-based approach improves power and interpretation for the combined analysis of genetic association studies of heterogeneous traits. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">90</span>:821–835.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3376551/">PMC free article</a>]</span> [<a href="/pubmed/22560090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+subset-based+approach+improves+power+and+interpretation+for+the+combined+analysis+of+genetic+association+studies+of+heterogeneous+traits&amp;author=S.+Bhattacharjee&amp;author=P.+Rajaraman&amp;author=K.B.+Jacobs&amp;author=W.A.+Wheeler&amp;author=B.S.+Melin&amp;volume=90&amp;publication_year=2012&amp;pages=821-835&amp;pmid=22560090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref38">Morris A.P. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref39">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref40">Wakefield J. Bayes factors for genome-wide association studies: comparison with P-values. <span><span class="ref-journal">Genet. Epidemiol. </span>2009;<span class="ref-vol">33</span>:79–86.</span> [<a href="/pubmed/18642345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Bayes+factors+for+genome-wide+association+studies:+comparison+with+P-values&amp;author=J.+Wakefield&amp;volume=33&amp;publication_year=2009&amp;pages=79-86&amp;pmid=18642345&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref41">Liu Y., Helms C., Liao W., Zaba L.C., Duan S., Gardner J., Wise C., Miner A., Malloy M.J., Pullinger C.R. A genome-wide association study of psoriasis and psoriatic arthritis identifies new disease loci. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000041.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2274885/">PMC free article</a>]</span> [<a href="/pubmed/18369459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+of+psoriasis+and+psoriatic+arthritis+identifies+new+disease+loci&amp;author=Y.+Liu&amp;author=C.+Helms&amp;author=W.+Liao&amp;author=L.C.+Zaba&amp;author=S.+Duan&amp;volume=4&amp;publication_year=2008&amp;pages=e1000041&amp;pmid=18369459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref42">Esparza-Gordillo J., Weidinger S., Fölster-Holst R., Bauerfeind A., Ruschendorf F., Patone G., Rohde K., Marenholz I., Schulz F., Kerscher T. A common variant on chromosome 11q13 is associated with atopic dermatitis. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:596–601.</span> [<a href="/pubmed/19349984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+on+chromosome+11q13+is+associated+with+atopic+dermatitis&amp;author=J.+Esparza-Gordillo&amp;author=S.+Weidinger&amp;author=R.+Fölster-Holst&amp;author=A.+Bauerfeind&amp;author=F.+Ruschendorf&amp;volume=41&amp;publication_year=2009&amp;pages=596-601&amp;pmid=19349984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref43">Sun L.D., Xiao F.L., Li Y., Zhou W.M., Tang H.Y., Tang X.F., Zhang H., Schaarschmidt H., Zuo X.B., Foelster-Holst R. Genome-wide association study identifies two new susceptibility loci for atopic dermatitis in the Chinese Han population. <span><span class="ref-journal">Nat. Genet. </span>2011;<span class="ref-vol">43</span>:690–694.</span> [<a href="/pubmed/21666691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+two+new+susceptibility+loci+for+atopic+dermatitis+in+the+Chinese+Han+population&amp;author=L.D.+Sun&amp;author=F.L.+Xiao&amp;author=Y.+Li&amp;author=W.M.+Zhou&amp;author=H.Y.+Tang&amp;volume=43&amp;publication_year=2011&amp;pages=690-694&amp;pmid=21666691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref44">Marenholz I., Rivera V.A., Esparza-Gordillo J., Bauerfeind A., Lee-Kirsch M.A., Ciechanowicz A., Kurek M., Piskackova T., Macek M., Lee Y.A. Association screening in the Epidermal Differentiation Complex (EDC) identifies an SPRR3 repeat number variant as a risk factor for eczema. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2011;<span class="ref-vol">131</span>:1644–1649.</span> [<a href="/pubmed/21490620" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Association+screening+in+the+Epidermal+Differentiation+Complex+(EDC)+identifies+an+SPRR3+repeat+number+variant+as+a+risk+factor+for+eczema&amp;author=I.+Marenholz&amp;author=V.A.+Rivera&amp;author=J.+Esparza-Gordillo&amp;author=A.+Bauerfeind&amp;author=M.A.+Lee-Kirsch&amp;volume=131&amp;publication_year=2011&amp;pages=1644-1649&amp;pmid=21490620&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref45">Ellinghaus E., Ellinghaus D., Stuart P.E., Nair R.P., Debrus S., Raelson J.V., Belouchi M., Fournier H., Reinhard C., Ding J. Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:991–995.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3136364/">PMC free article</a>]</span> [<a href="/pubmed/20953188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+a+psoriasis+susceptibility+locus+at+TRAF3IP2&amp;author=E.+Ellinghaus&amp;author=D.+Ellinghaus&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=S.+Debrus&amp;volume=42&amp;publication_year=2010&amp;pages=991-995&amp;pmid=20953188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref46">Hüffmeier U., Uebe S., Ekici A.B., Bowes J., Giardina E., Korendowych E., Juneblad K., Apel M., McManus R., Ho P. Common variants at TRAF3IP2 are associated with susceptibility to psoriatic arthritis and psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:996–999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2981079/">PMC free article</a>]</span> [<a href="/pubmed/20953186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+at+TRAF3IP2+are+associated+with+susceptibility+to+psoriatic+arthritis+and+psoriasis&amp;author=U.+Hüffmeier&amp;author=S.+Uebe&amp;author=A.B.+Ekici&amp;author=J.+Bowes&amp;author=E.+Giardina&amp;volume=42&amp;publication_year=2010&amp;pages=996-999&amp;pmid=20953186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref47">Patel C.V., Handy I., Goldsmith T., Patel R.C. PACT, a stress-modulated cellular activator of interferon-induced double-stranded RNA-activated protein kinase, PKR. <span><span class="ref-journal">J.&nbsp;Biol. Chem. </span>2000;<span class="ref-vol">275</span>:37993–37998.</span> [<a href="/pubmed/10988289" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Biol.+Chem.&amp;title=PACT,+a+stress-modulated+cellular+activator+of+interferon-induced+double-stranded+RNA-activated+protein+kinase,+PKR&amp;author=C.V.+Patel&amp;author=I.+Handy&amp;author=T.+Goldsmith&amp;author=R.C.+Patel&amp;volume=275&amp;publication_year=2000&amp;pages=37993-37998&amp;pmid=10988289&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref48">Kok K.H., Lui P.Y., Ng M.H., Siu K.L., Au S.W., Jin D.Y. The double-stranded RNA-binding protein PACT functions as a cellular activator of RIG-I to facilitate innate antiviral response. <span><span class="ref-journal">Cell Host Microbe. </span>2011;<span class="ref-vol">9</span>:299–309.</span> [<a href="/pubmed/21501829" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Host+Microbe&amp;title=The+double-stranded+RNA-binding+protein+PACT+functions+as+a+cellular+activator+of+RIG-I+to+facilitate+innate+antiviral+response&amp;author=K.H.+Kok&amp;author=P.Y.+Lui&amp;author=M.H.+Ng&amp;author=K.L.+Siu&amp;author=S.W.+Au&amp;volume=9&amp;publication_year=2011&amp;pages=299-309&amp;pmid=21501829&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref49">Guttman-Yassky E., Nograles K.E., Krueger J.G. Contrasting pathogenesis of atopic dermatitis and psoriasis—part I: clinical and pathologic concepts. <span><span class="ref-journal">J.&nbsp;Allergy Clin. Immunol. </span>2011;<span class="ref-vol">127</span>:1110–1118.</span> [<a href="/pubmed/21388665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Allergy+Clin.+Immunol.&amp;title=Contrasting+pathogenesis+of+atopic+dermatitis+and+psoriasis—part+I:+clinical+and+pathologic+concepts&amp;author=E.+Guttman-Yassky&amp;author=K.E.+Nograles&amp;author=J.G.+Krueger&amp;volume=127&amp;publication_year=2011&amp;pages=1110-1118&amp;pmid=21388665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref50">Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P., Boehnke M., Abecasis G.R., Willer C.J. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=R.J.+Pruim&amp;author=R.P.+Welch&amp;author=S.+Sanna&amp;author=T.M.+Teslovich&amp;author=P.S.+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref51">Elder J.T., Bruce A.T., Gudjonsson J.E., Johnston A., Stuart P.E., Tejasvi T., Voorhees J.J., Abecasis G.R., Nair R.P. Molecular dissection of psoriasis: integrating genetics and biology. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2010;<span class="ref-vol">130</span>:1213–1226.</span> [<a href="/pubmed/19812592" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Molecular+dissection+of+psoriasis:+integrating+genetics+and+biology&amp;author=J.T.+Elder&amp;author=A.T.+Bruce&amp;author=J.E.+Gudjonsson&amp;author=A.+Johnston&amp;author=P.E.+Stuart&amp;volume=130&amp;publication_year=2010&amp;pages=1213-1226&amp;pmid=19812592&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref52">Henry J., Hsu C.Y., Haftek M., Nachat R., de Koning H.D., Gardinal-Galera I., Hitomi K., Balica S., Jean-Decoster C., Schmitt A.M. Hornerin is a component of the epidermal cornified cell envelopes. <span><span class="ref-journal">FASEB J. </span>2011;<span class="ref-vol">25</span>:1567–1576.</span> [<a href="/pubmed/21282207" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J.&amp;title=Hornerin+is+a+component+of+the+epidermal+cornified+cell+envelopes&amp;author=J.+Henry&amp;author=C.Y.+Hsu&amp;author=M.+Haftek&amp;author=R.+Nachat&amp;author=H.D.+de+Koning&amp;volume=25&amp;publication_year=2011&amp;pages=1567-1576&amp;pmid=21282207&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref53">Guttman-Yassky E., Suárez-Fariñas M., Chiricozzi A., Nograles K.E., Shemer A., Fuentes-Duculan J., Cardinale I., Lin P., Bergman R., Bowcock A.M., Krueger J.G. Broad defects in epidermal cornification in atopic dermatitis identified through genomic analysis. <span><span class="ref-journal">J.&nbsp;Allergy Clin. Immunol. </span>2009;<span class="ref-vol">124</span>:1235–1244.e58.</span> [<a href="/pubmed/20004782" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Allergy+Clin.+Immunol.&amp;title=Broad+defects+in+epidermal+cornification+in+atopic+dermatitis+identified+through+genomic+analysis&amp;author=E.+Guttman-Yassky&amp;author=M.+Suárez-Fariñas&amp;author=A.+Chiricozzi&amp;author=K.E.+Nograles&amp;author=A.+Shemer&amp;volume=124&amp;publication_year=2009&amp;pages=1235-1244.e58&amp;pmid=20004782&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref54">Wu Z., Hansmann B., Meyer-Hoffert U., Gläser R., Schröder J.M. Molecular identification and expression analysis of filaggrin-2, a member of the S100 fused-type protein family. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e5227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668185/">PMC free article</a>]</span> [<a href="/pubmed/19384417" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Molecular+identification+and+expression+analysis+of+filaggrin-2,+a+member+of+the+S100+fused-type+protein+family&amp;author=Z.+Wu&amp;author=B.+Hansmann&amp;author=U.+Meyer-Hoffert&amp;author=R.+Gläser&amp;author=J.M.+Schröder&amp;volume=4&amp;publication_year=2009&amp;pages=e5227&amp;pmid=19384417&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref55">Takaishi M., Makino T., Morohashi M., Huh N.H. Identification of human hornerin and its expression in regenerating and psoriatic skin. <span><span class="ref-journal">J.&nbsp;Biol. Chem. </span>2005;<span class="ref-vol">280</span>:4696–4703.</span> [<a href="/pubmed/15507446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Biol.+Chem.&amp;title=Identification+of+human+hornerin+and+its+expression+in+regenerating+and+psoriatic+skin&amp;author=M.+Takaishi&amp;author=T.+Makino&amp;author=M.+Morohashi&amp;author=N.H.+Huh&amp;volume=280&amp;publication_year=2005&amp;pages=4696-4703&amp;pmid=15507446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref56">Kim B.E., Howell M.D., Guttman-Yassky E., Gilleaudeau P.M., Cardinale I.R., Boguniewicz M., Krueger J.G., Leung D.Y. TNF-α downregulates filaggrin and loricrin through c-Jun N-terminal kinase: role for TNF-α antagonists to improve skin barrier. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2011;<span class="ref-vol">131</span>:1272–1279.</span> [<a href="/pubmed/21346775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=TNF-α+downregulates+filaggrin+and+loricrin+through+c-Jun+N-terminal+kinase:+role+for+TNF-α+antagonists+to+improve+skin+barrier&amp;author=B.E.+Kim&amp;author=M.D.+Howell&amp;author=E.+Guttman-Yassky&amp;author=P.M.+Gilleaudeau&amp;author=I.R.+Cardinale&amp;volume=131&amp;publication_year=2011&amp;pages=1272-1279&amp;pmid=21346775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation" id="sref57">Wang K., Baldassano R., Zhang H., Qu H.Q., Imielinski M., Kugathasan S., Annese V., Dubinsky M., Rotter J.I., Russell R.K. Comparative genetic analysis of inflammatory bowel disease and type 1 diabetes implicates multiple loci with opposite effects. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:2059–2067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2860894/">PMC free article</a>]</span> [<a href="/pubmed/20176734" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Comparative+genetic+analysis+of+inflammatory+bowel+disease+and+type+1+diabetes+implicates+multiple+loci+with+opposite+effects&amp;author=K.+Wang&amp;author=R.+Baldassano&amp;author=H.+Zhang&amp;author=H.Q.+Qu&amp;author=M.+Imielinski&amp;volume=19&amp;publication_year=2010&amp;pages=2059-2067&amp;pmid=20176734&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation" id="sref58">de Guzman Strong C., Conlan S., Deming C.B., Cheng J., Sears K.E., Segre J.A. A milieu of regulatory elements in the epidermal differentiation complex syntenic block: implications for atopic dermatitis and psoriasis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:1453–1460.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846160/">PMC free article</a>]</span> [<a href="/pubmed/20089530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+milieu+of+regulatory+elements+in+the+epidermal+differentiation+complex+syntenic+block:+implications+for+atopic+dermatitis+and+psoriasis&amp;author=C.+de+Guzman+Strong&amp;author=S.+Conlan&amp;author=C.B.+Deming&amp;author=J.+Cheng&amp;author=K.E.+Sears&amp;volume=19&amp;publication_year=2010&amp;pages=1453-1460&amp;pmid=20089530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation" id="sref59">Faghihi M.A., Wahlestedt C. Regulatory roles of natural antisense transcripts. <span><span class="ref-journal">Nat. Rev. Mol. Cell Biol. </span>2009;<span class="ref-vol">10</span>:637–643.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850559/">PMC free article</a>]</span> [<a href="/pubmed/19638999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Mol.+Cell+Biol.&amp;title=Regulatory+roles+of+natural+antisense+transcripts&amp;author=M.A.+Faghihi&amp;author=C.+Wahlestedt&amp;volume=10&amp;publication_year=2009&amp;pages=637-643&amp;pmid=19638999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">59. <span class="element-citation" id="sref60">Kretz M., Siprashvili Z., Chu C., Webster D.E., Zehnder A., Qu K., Lee C.S., Flockhart R.J., Groff A.F., Chow J. Control of somatic tissue differentiation by the long non-coding RNA TINCR. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">493</span>:231–235.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3674581/">PMC free article</a>]</span> [<a href="/pubmed/23201690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Control+of+somatic+tissue+differentiation+by+the+long+non-coding+RNA+TINCR&amp;author=M.+Kretz&amp;author=Z.+Siprashvili&amp;author=C.+Chu&amp;author=D.E.+Webster&amp;author=A.+Zehnder&amp;volume=493&amp;publication_year=2013&amp;pages=231-235&amp;pmid=23201690&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib60">60. <span class="element-citation" id="sref61">Lee G.R., Spilianakis C.G., Flavell R.A. Hypersensitive site 7 of the TH2 locus control region is essential for expressing TH2 cytokine genes and for long-range intrachromosomal interactions. <span><span class="ref-journal">Nat. Immunol. </span>2005;<span class="ref-vol">6</span>:42–48.</span> [<a href="/pubmed/15608641" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol.&amp;title=Hypersensitive+site+7+of+the+TH2+locus+control+region+is+essential+for+expressing+TH2+cytokine+genes+and+for+long-range+intrachromosomal+interactions&amp;author=G.R.+Lee&amp;author=C.G.+Spilianakis&amp;author=R.A.+Flavell&amp;volume=6&amp;publication_year=2005&amp;pages=42-48&amp;pmid=15608641&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib61">61. <span class="element-citation" id="sref62">Knight J., Spain S.L., Capon F., Hayday A., Nestle F.O., Clop A., Barker J.N., Weale M.E., Trembath R.C., Wellcome Trust Case Control Consortium. Genetic Analysis of Psoriasis Consortium. I-chip for Psoriasis Consortium  Conditional analysis identifies three novel major histocompatibility complex loci associated with psoriasis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:5185–5192.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3490509/">PMC free article</a>]</span> [<a href="/pubmed/22914738" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Conditional+analysis+identifies+three+novel+major+histocompatibility+complex+loci+associated+with+psoriasis&amp;author=J.+Knight&amp;author=S.L.+Spain&amp;author=F.+Capon&amp;author=A.+Hayday&amp;author=F.O.+Nestle&amp;volume=21&amp;publication_year=2012&amp;pages=5185-5192&amp;pmid=22914738&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib62">62. <span class="element-citation" id="sref63">Gunderson A.J., Mohammed J., Horvath F.J., Podolsky M.A., Anderson C.R., Glick A.B. CD8(+) T&nbsp;cells mediate RAS-induced psoriasis-like skin inflammation through IFN-γ <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2013;<span class="ref-vol">133</span>:955–963.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3577939/">PMC free article</a>]</span> [<a href="/pubmed/23151849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=CD8(+)+T&nbsp;cells+mediate+RAS-induced+psoriasis-like+skin+inflammation+through+IFN-γ&amp;author=A.J.+Gunderson&amp;author=J.+Mohammed&amp;author=F.J.+Horvath&amp;author=M.A.+Podolsky&amp;author=C.R.+Anderson&amp;volume=133&amp;publication_year=2013&amp;pages=955-963&amp;pmid=23151849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib63">63. <span class="element-citation" id="sref64">Hijnen D., Knol E.F., Gent Y.Y., Giovannone B., Beijn S.J., Kupper T.S., Bruijnzeel-Koomen C.A., Clark R.A. CD8(+) T&nbsp;cells in the lesional skin of atopic dermatitis and psoriasis patients are an important source of IFN-γ, IL-13, IL-17, and IL-22. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2013;<span class="ref-vol">133</span>:973–979.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3835628/">PMC free article</a>]</span> [<a href="/pubmed/23223131" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=CD8(+)+T&nbsp;cells+in+the+lesional+skin+of+atopic+dermatitis+and+psoriasis+patients+are+an+important+source+of+IFN-γ,+IL-13,+IL-17,+and+IL-22&amp;author=D.+Hijnen&amp;author=E.F.+Knol&amp;author=Y.Y.+Gent&amp;author=B.+Giovannone&amp;author=S.J.+Beijn&amp;volume=133&amp;publication_year=2013&amp;pages=973-979&amp;pmid=23223131&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib64">64. <span class="element-citation" id="sref65">Mangalam A.K., Taneja V., David C.S. HLA class II molecules influence susceptibility versus protection in inflammatory diseases by determining the cytokine profile. <span><span class="ref-journal">J.&nbsp;Immunol. </span>2013;<span class="ref-vol">190</span>:513–518.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3545203/">PMC free article</a>]</span> [<a href="/pubmed/23293357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Immunol.&amp;title=HLA+class+II+molecules+influence+susceptibility+versus+protection+in+inflammatory+diseases+by+determining+the+cytokine+profile&amp;author=A.K.+Mangalam&amp;author=V.+Taneja&amp;author=C.S.+David&amp;volume=190&amp;publication_year=2013&amp;pages=513-518&amp;pmid=23293357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib65">65. <span class="element-citation" id="sref66">Okada Y., Han B., Tsoi L.C., Stuart P.E., Ellinghaus E., Tejasvi T., Chandran V., Pellett F., Pollock R., Bowcock A.M. Fine mapping major histocompatibility complex associations in psoriasis and its clinical subtypes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2014;<span class="ref-vol">95</span>:162–172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4129407/">PMC free article</a>]</span> [<a href="/pubmed/25087609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Fine+mapping+major+histocompatibility+complex+associations+in+psoriasis+and+its+clinical+subtypes&amp;author=Y.+Okada&amp;author=B.+Han&amp;author=L.C.+Tsoi&amp;author=P.E.+Stuart&amp;author=E.+Ellinghaus&amp;volume=95&amp;publication_year=2014&amp;pages=162-172&amp;pmid=25087609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib66">66. <span class="element-citation" id="sref67">Kapila S., Hong E., Fischer G. A comparative study of childhood psoriasis and atopic dermatitis and greater understanding of the overlapping condition, psoriasis-dermatitis. <span><span class="ref-journal">Australas. J. Dermatol. </span>2012;<span class="ref-vol">53</span>:98–105.</span> [<a href="/pubmed/22571556" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Australas.+J.+Dermatol.&amp;title=A+comparative+study+of+childhood+psoriasis+and+atopic+dermatitis+and+greater+understanding+of+the+overlapping+condition,+psoriasis-dermatitis&amp;author=S.+Kapila&amp;author=E.+Hong&amp;author=G.+Fischer&amp;volume=53&amp;publication_year=2012&amp;pages=98-105&amp;pmid=22571556&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139812601480432title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Subjects and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#sec5title">Consortia</a></li><li><a href="#ack0010title">Acknowledgments</a></li><li><a href="#idm139812676599472title">Notes</a></li><li><a href="#idm139812598682160title">Footnotes</a></li><li><a href="#app2title">Supplemental Data</a></li><li><a href="#app3title">Web Resources</a></li><li><a href="#idm139812603575856title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4289690/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4289690/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4289690/pdf/main.pdf">PDF (3.0M)</a></li> | <li><a href="#" data-citationid="PMC4289690" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289690%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289690%2F&amp;text=Genome-wide%20Comparative%20Analysis%20of%20Atopic%20Dermatitis%20and%20Psoriasis%20Gives%20Insight%20into%20Opposing%20Genetic%20Mechanisms" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289690%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25574825" data-aiid="4289690" data-aid="4289690" data-iid="247455" data-domainid="203" data-domain="ajhg" data-accid="PMC4289690" data-md5="1a6f704ad30e45f444813ee573fedc03">
<div class="portlet pubmed_favoritesad">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Save items</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter shutter_closed ui-ncbitoggler" title="Show/hide content" remembercollapsed="true" pgsec_name="pmfavad" toggles="ui-portlet_content-5" role="button" aria-expanded="false" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave" style="display: none;" id="ui-portlet_content-5" aria-hidden="true"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
<input name="absid" id="absid" value="4289690" type="hidden"><div id="pubmed_favoritesad" class="empty" data-db="pmc">
<span role="menubar"><div class="ui-ncbisetswitch ltd-hover" role="menuitem" aria-expanded="false" aria-haspopup="true"><a href="/myncbi/collections/" class="jig-ncbisetswitch blind collink ltd-hover ui-ncbisetswitch-hasStar" id="favList" data-jigconfig="destSelector:'#favUL',starExists:true,createCollectionUrl:'#',manageCollectionsUrl:'/myncbi/collections/'"><span class="star"></span>Add to Favorites</a><a class="ui-ncbisetswitch-button ltd-hover" title="View more options" href="#"><span class="ui-icon ui-icon-triangle-1-s ui-ncbisetswitch-down">View more options</span></a></div></span><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu ui-ncbisetswitch-popper"><ul id="favUL" class="ui-ncbisetswitch-options"></ul><ul class="ui-ncbisetswitch-actions"><li class="ui-ncbisetswitch-actions-create" style="display: block;"><a class="ui-ncbisetswitch-create-collection" href="#">Create collection...</a></li><li class="ui-ncbisetswitch-actions-manage" style="display: block;"><a class="ui-ncbisetswitch-manage-collections" href="/myncbi/collections/">Manage collections...</a></li></ul></div>
</div>
<div class="colloading">loading</div>
<span id="submenu_AddToCollections"><form method="post"><input class="button_apply" name="addtocollection" value="1" type="submit"></form></span>
</div></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Similar articles in PubMed</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PBooksDiscovery_RA" toggles="ui-portlet_content-6" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-6" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/23886662" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A genome-wide association study of atopic dermatitis identifies loci with overlapping effects on asthma and psoriasis.</a><span class="source">[Hum Mol Genet. 2013]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-7" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Weidinger S, Willis-Owen SA, Kamatani Y, Baurecht H, Morar N, Liang L, Edser P, Street T, Rodriguez E, O'Regan GM, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Mol Genet. 2013 Dec 1; 22(23):4841-56. Epub 2013 Jul 25.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/16912508" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Co-localization of susceptibility loci for psoriasis (PSORS4) and atopic dermatitis (ATOD2) on human chromosome 1q21.</a><span class="source">[Hum Hered. 2006]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-8" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Giardina E, Sinibaldi C, Chini L, Moschese V, Marulli G, Provini A, Rossi P, Paradisi M, Chimenti S, Galli E, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Hered. 2006; 61(4):229-36. Epub 2006 Aug 15.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/14996755" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">The genetics of psoriasis, psoriatic arthritis and atopic dermatitis.</a><span class="source">[Hum Mol Genet. 2004]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-9" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Bowcock AM, Cookson WO. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Hum Mol Genet. 2004 Apr 1; 13 Spec No 1:R43-55. </em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/21666691" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genome-wide association study identifies two new susceptibility loci for atopic dermatitis in the Chinese Han population.</a><span class="source">[Nat Genet. 2011]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-10" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Sun LD, Xiao FL, Li Y, Zhou WM, Tang HY, Tang XF, Zhang H, Schaarschmidt H, Zuo XB, Foelster-Holst R, et al. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Nat Genet. 2011 Jun 12; 43(7):690-4. Epub 2011 Jun 12.</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pubmed/17573887" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">On the role of the epidermal differentiation complex in ichthyosis vulgaris, atopic dermatitis and psoriasis.</a><span class="source">[Br J Dermatol. 2007]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-11" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Hoffjan S, Stemmler S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Br J Dermatol. 2007 Sep; 157(3):441-9. Epub 2007 Jun 15.</em>
</div>
</li>
</ul>
<a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed_reviews&amp;uid=25574825&amp;log%24=relatedreviews&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Reviews&amp;rendering-type=normal">See reviews...</a><a class="seemore" href="/sites/entrez?db=pubmed&amp;cmd=link&amp;linkname=pubmed_pubmed&amp;uid=25574825&amp;log%24=relatedarticles&amp;logdbfrom=pmc" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|RelatedArticles&amp;TO=Entrez|Pubmed|Related%20Records&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Cited by other articles in PMC</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-12" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-12" aria-hidden="false">
<ul>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6711380/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Atopic dermatitis is an IL-13 dominant disease with greater molecular heterogeneity compared to psoriasis</a><span class="source">[The Journal of investigative d...]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-13" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Tsoi LC, Rodriguez E, Degenhardt F, Baurecht H, Wehkamp U, Volks N, Szymczak S, Swindell WR, Sarkar MK, Raja K, Shao S, Patrick M, Gao Y, Uppala R, Perez White BE, Getsios S, Harms PW, Maverakis E, Elder JT, Franke A, Gudjonsson JE, Weidinger S. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">The Journal of investigative dermatology. 2019 Jan 11; 139(7): 1480-1489</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6706929/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Replicated methylation changes associated with eczema herpeticum and allergic response</a><span class="source">[Clinical Epigenetics. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-14" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Boorgula MP, Taub MA, Rafaels N, Daya M, Campbell M, Chavan S, Shetty A, Cheadle C, Barkataki S, Fan J, David G, Beaty TH, Ruczinski I, Hanifin J, Schneider LC, Gallo RL, Paller AS, Beck LA, Leung DY, Mathias RA, Barnes KC. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Clinical Epigenetics. 2019 Aug 23; 11: 122</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6546803/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Sensitive Skin: Lessons From Transcriptomic Studies</a><span class="source">[Frontiers in Medicine. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-15" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Bataille A, Le Gall-Ianotto C, Genin E, Misery L. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Frontiers in Medicine. 2019 May 28; 6: 115</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6370860/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">A phenome-wide association study to discover pleiotropic effects of PCSK9, APOB, and LDLR</a><span class="source">[NPJ Genomic Medicine. 2019]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-16" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Safarova MS, Satterfield BA, Fan X, Austin EE, Ye Z, Bastarache L, Zheng N, Ritchie MD, Borthwick KM, Williams MS, Larson EB, Scrol A, Jarvik GP, Crosslin DR, Leppig K, Rasmussen-Torvik LJ, Pendergrass SA, Sturm AC, Namjou B, Shah AS, Carroll RJ, Chung WK, Wei WQ, Feng Q, Stein CM, Roden DM, Manolio TA, Schaid DJ, Denny JC, Hebbring SJ, de Andrade M, Kullo IJ. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">NPJ Genomic Medicine. 2019 Feb 11; 4: 3</em>
</div>
</li>
<li class="brieflinkpopper two_line">
<a class="brieflinkpopperctrl" href="/pmc/articles/PMC6205839/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Genetic Architecture of Adaptive Immune System Identifies Key Immune Regulators</a><span class="source">[Cell Reports. 2018]</span><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-17" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">
<em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="author">Lagou V, Garcia-Perez JE, Smets I, Van Horebeek L, Vandebergh M, Chen L, Mallants K, Prezzemolo T, Hilven K, Humblet-Baron S, Moisse M, Van Damme P, Boeckxstaens G, Bowness P, Dubois B, Dooley J, Liston A, Goris A. </em><em xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="cit">Cell Reports. 2018 Oct 16; 25(3): 798-810.e6</em>
</div>
</li>
</ul>
<a class="seemore" href="citedby/" ref="reftype=relart&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|CitedByInPmc&amp;TO=PMC|Article|CitedBy&amp;rendering-type=normal">See all...</a>
</div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Links</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="PMCVCitedByPmcArticlesP" toggles="ui-portlet_content-18" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-18" aria-hidden="false"><ul>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=medgen&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_medgen&amp;IdsFromResult=4289690" ref="reftype=MedGen&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Unknown%20Link%20Name&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">MedGen</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-19" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">MedGen<div class="brieflinkpopdesc">Related information in MedGen</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pubmed/25574825/" ref="reftype=PubMed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">PubMed</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-20" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">PubMed<div class="brieflinkpopdesc">PubMed citations for these articles</div>
</div>
</li>
<li class="brieflinkpopper">
<a class="brieflinkpopperctrl" href="/pmc/?Db=taxonomy&amp;DbFrom=pmc&amp;Cmd=Link&amp;LinkName=pmc_taxonomy&amp;IdsFromResult=4289690" ref="reftype=Taxonomy&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article|DiscoveryLinks&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal" role="button" aria-expanded="false" aria-haspopup="true">Taxonomy</a><div class="brieflinkpop ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" id="ui-brieflinkpop-21" aria-live="assertive" aria-hidden="true" style="top: -100px; left: -100px; display: none;">Taxonomy<div class="brieflinkpopdesc">Related taxonomy entry</div>
</div>
</li>
</ul></div></div>
</div><div class="portlet">
<div class="portlet_head">
<div class="portlet_title"><h3><span>Recent Activity</span></h3></div>
<a name="Shutter" sid="1" href="#" class="portlet_shutter ui-ncbitoggler-open" title="Show/hide content" remembercollapsed="true" pgsec_name="recent_activity" toggles="ui-portlet_content-22" role="button" aria-expanded="true" style="position: absolute; padding: 0px;"></a>
</div>
<div class="ui-helper-reset" aria-live="assertive"><div class="portlet_content ui-ncbitoggler ui-ncbitoggler-slave-open" id="ui-portlet_content-22" aria-hidden="false"><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HTDisplay" class="">
<div class="action">
<a href="javascript:historyDisplayState('ClearHT')">Clear</a><a href="javascript:historyDisplayState('HTOff')" class="HTOn">Turn Off</a><a href="javascript:historyDisplayState('HTOn')" class="HTOff">Turn On</a>
</div>
<ul id="activity">
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=1" href="/portal/utils/pageresolver.fcgi?recordid=5dd581f1dde0893a8b3cbc73">Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insigh...</a><div class="ralinkpop offscreen_noflow">Genome-wide Comparative Analysis of Atopic Dermatitis and Psoriasis Gives Insight into Opposing Genetic Mechanisms<div class="brieflinkpopdesc">American Journal of Human Genetics. 2015 Jan 8; 96(1)104</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=2" href="/portal/utils/pageresolver.fcgi?recordid=5dd581c939a66eb03d23f4d7">Seasonality shows evidence for polygenic architecture and genetic correlation wi...</a><div class="ralinkpop offscreen_noflow">Seasonality shows evidence for polygenic architecture and genetic correlation with schizophrenia and bipolar disorder – a meta-analysis of genetic studies<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2015 Feb; 76(2)128</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=3" href="/portal/utils/pageresolver.fcgi?recordid=5dd581a2dd2142621d2563cb">Genome-Wide Association Study of Short-Acting
β2-Agonists. A Novel Genome-Wide S...</a><div class="ralinkpop offscreen_noflow">Genome-Wide Association Study of Short-Acting
β2-Agonists. A Novel Genome-Wide Significant Locus on Chromosome 2
near ASB3<div class="brieflinkpopdesc">American Journal of Respiratory and Critical Care Medicine. 2015 Mar 1; 191(5)530</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=4" href="/portal/utils/pageresolver.fcgi?recordid=5dd5817edd2142621d200136">Genome-Wide Association Analysis of Plasma B-Type Natriuretic Peptide in African...</a><div class="ralinkpop offscreen_noflow">Genome-Wide Association Analysis of Plasma B-Type Natriuretic Peptide in African Americans: The Jackson Heart Study<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2015 Feb; 8(1)122</div>
</div>
<div class="tertiary"></div>
</li>
<li class="ra_rcd ralinkpopper two_line">
<a class="htb ralinkpopperctrl" ref="log$=activity&amp;linkpos=5" href="/portal/utils/pageresolver.fcgi?recordid=5dd5815cdde0893a8b24d795">Genome-Wide Association Study for Endothelial Growth Factors</a><div class="ralinkpop offscreen_noflow">Genome-Wide Association Study for Endothelial Growth Factors<div class="brieflinkpopdesc">NIHPA Author Manuscripts. 2015 Apr; 8(2)389</div>
</div>
<div class="tertiary"></div>
</li>
</ul>
<p class="HTOn">Your browsing activity is empty.</p>
<p class="HTOff">Activity recording is turned off.</p>
<p id="turnOn" class="HTOff"><a href="javascript:historyDisplayState('HTOn')">Turn recording back on</a></p>
<a class="seemore" href="/sites/myncbi/recentactivity">See more...</a>
</div></div></div>
</div>
   
    </div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"><div id="body-link-popper-bib1 bib2" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib3 bib4" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib5" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">5. <span class="element-citation" id="sref5">Henseler T., Christophers E. Disease concomitance in psoriasis. <span><span class="ref-journal">J.&nbsp;Am. Acad. Dermatol. </span>1995;<span class="ref-vol">32</span>:982–986.</span> [<a href="/pubmed/7751469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Acad.+Dermatol.&amp;title=Disease+concomitance+in+psoriasis&amp;author=T.+Henseler&amp;author=E.+Christophers&amp;volume=32&amp;publication_year=1995&amp;pages=982-986&amp;pmid=7751469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib5">Ref list</a>]</div><div id="body-link-popper-bib6" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">6. <span class="element-citation" id="sref6">Eyerich S., Onken A.T., Weidinger S., Franke A., Nasorri F., Pennino D., Grosber M., Pfab F., Schmidt-Weber C.B., Mempel M. Mutual antagonism of T&nbsp;cells causing psoriasis and atopic eczema. <span><span class="ref-journal">N.&nbsp;Engl. J. Med. </span>2011;<span class="ref-vol">365</span>:231–238.</span> [<a href="/pubmed/21774711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.&nbsp;Engl.+J.+Med.&amp;title=Mutual+antagonism+of+T&nbsp;cells+causing+psoriasis+and+atopic+eczema&amp;author=S.+Eyerich&amp;author=A.T.+Onken&amp;author=S.+Weidinger&amp;author=A.+Franke&amp;author=F.+Nasorri&amp;volume=365&amp;publication_year=2011&amp;pages=231-238&amp;pmid=21774711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib6">Ref list</a>]</div><div id="body-link-popper-bib7" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">7. <span class="element-citation" id="sref7">Hemminki K., Li X., Sundquist K., Sundquist J. Shared familial aggregation of susceptibility to autoimmune diseases. <span><span class="ref-journal">Arthritis Rheum. </span>2009;<span class="ref-vol">60</span>:2845–2847.</span> [<a href="/pubmed/19714648" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Shared+familial+aggregation+of+susceptibility+to+autoimmune+diseases&amp;author=K.+Hemminki&amp;author=X.+Li&amp;author=K.+Sundquist&amp;author=J.+Sundquist&amp;volume=60&amp;publication_year=2009&amp;pages=2845-2847&amp;pmid=19714648&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib7">Ref list</a>]</div><div id="body-link-popper-bib8" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">8. <span class="element-citation" id="sref8">Maas T., Nieuwhof C., Passos V.L., Robertson C., Boonen A., Landewé R.B., Voncken J.W., Knottnerus J.A., Damoiseaux J.G. Transgenerational occurrence of allergic disease and autoimmunity: general practice-based epidemiological research. <span><span class="ref-journal">Prim. Care Respir. J. </span>2014;<span class="ref-vol">23</span>:14–21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6442276/">PMC free article</a>]</span> [<a href="/pubmed/24449016" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prim.+Care+Respir.+J.&amp;title=Transgenerational+occurrence+of+allergic+disease+and+autoimmunity:+general+practice-based+epidemiological+research&amp;author=T.+Maas&amp;author=C.+Nieuwhof&amp;author=V.L.+Passos&amp;author=C.+Robertson&amp;author=A.+Boonen&amp;volume=23&amp;publication_year=2014&amp;pages=14-21&amp;pmid=24449016&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib8">Ref list</a>]</div><div id="body-link-popper-bib9 bib10 bib11 bib12" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib13 bib14" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib15 bib16" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib17 bib18" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib19" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">19. <span class="element-citation" id="sref19">Bergboer J.G., Zeeuwen P.L., Irvine A.D., Weidinger S., Giardina E., Novelli G., Den Heijer M., Rodriguez E., Illig T., Riveira-Munoz E. Deletion of Late Cornified Envelope 3B and 3C genes is not associated with atopic dermatitis. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2010;<span class="ref-vol">130</span>:2057–2061.</span> [<a href="/pubmed/20376060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Deletion+of+Late+Cornified+Envelope+3B+and+3C+genes+is+not+associated+with+atopic+dermatitis&amp;author=J.G.+Bergboer&amp;author=P.L.+Zeeuwen&amp;author=A.D.+Irvine&amp;author=S.+Weidinger&amp;author=E.+Giardina&amp;volume=130&amp;publication_year=2010&amp;pages=2057-2061&amp;pmid=20376060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib19">Ref list</a>]</div><div id="body-link-popper-bib10 bib20 bib21" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib22 bib23" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib20" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">20. <span class="element-citation" id="sref20">Weidinger S., Willis-Owen S.A., Kamatani Y., Baurecht H., Morar N., Liang L., Edser P., Street T., Rodriguez E., O’Regan G.M. A genome-wide association study of atopic dermatitis identifies loci with overlapping effects on asthma and psoriasis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:4841–4856.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3820131/">PMC free article</a>]</span> [<a href="/pubmed/23886662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+genome-wide+association+study+of+atopic+dermatitis+identifies+loci+with+overlapping+effects+on+asthma+and+psoriasis&amp;author=S.+Weidinger&amp;author=S.A.+Willis-Owen&amp;author=Y.+Kamatani&amp;author=H.+Baurecht&amp;author=N.+Morar&amp;volume=22&amp;publication_year=2013&amp;pages=4841-4856&amp;pmid=23886662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib20">Ref list</a>]</div><div id="body-link-popper-bib24" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">24. <span class="element-citation" id="sref24">Strange A., Capon F., Spencer C.C., Knight J., Weale M.E., Allen M.H., Barton A., Band G., Bellenguez C., Bergboer J.G., Genetic Analysis of Psoriasis Consortium &amp; the Wellcome Trust Case Control Consortium 2  A genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:985–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749730/">PMC free article</a>]</span> [<a href="/pubmed/20953190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+new+psoriasis+susceptibility+loci+and+an+interaction+between+HLA-C+and+ERAP1&amp;author=A.+Strange&amp;author=F.+Capon&amp;author=C.C.+Spencer&amp;author=J.+Knight&amp;author=M.E.+Weale&amp;volume=42&amp;publication_year=2010&amp;pages=985-990&amp;pmid=20953190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib24">Ref list</a>]</div><div id="body-link-popper-bib25" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">25. <span class="element-citation" id="sref25">Williams H.C., Burney P.G., Pembroke A.C., Hay R.J. The U.K. Working Party’s Diagnostic Criteria for Atopic Dermatitis. III. Independent hospital validation. <span><span class="ref-journal">Br. J. Dermatol. </span>1994;<span class="ref-vol">131</span>:406–416.</span> [<a href="/pubmed/7918017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=The+U.K.+Working+Party’s+Diagnostic+Criteria+for+Atopic+Dermatitis.+III.+Independent+hospital+validation&amp;author=H.C.+Williams&amp;author=P.G.+Burney&amp;author=A.C.+Pembroke&amp;author=R.J.+Hay&amp;volume=131&amp;publication_year=1994&amp;pages=406-416&amp;pmid=7918017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib25">Ref list</a>]</div><div id="body-link-popper-bib26" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">26. <span class="element-citation" id="sref26">Krawczak M., Nikolaus S., von Eberstein H., Croucher P.J., El Mokhtari N.E., Schreiber S. PopGen: population-based recruitment of patients and controls for the analysis of complex genotype-phenotype relationships. <span><span class="ref-journal">Community Genet. </span>2006;<span class="ref-vol">9</span>:55–61.</span> [<a href="/pubmed/16490960" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Community+Genet.&amp;title=PopGen:+population-based+recruitment+of+patients+and+controls+for+the+analysis+of+complex+genotype-phenotype+relationships&amp;author=M.+Krawczak&amp;author=S.+Nikolaus&amp;author=H.+von+Eberstein&amp;author=P.J.+Croucher&amp;author=N.E.+El+Mokhtari&amp;volume=9&amp;publication_year=2006&amp;pages=55-61&amp;pmid=16490960&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib26">Ref list</a>]</div><div id="body-link-popper-bib27" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">27. <span class="element-citation" id="sref27">Wichmann H.E., Gieger C., Illig T., MONICA/KORA Study Group  KORA-gen—resource for population genetics, controls and a broad spectrum of disease phenotypes. <span><span class="ref-journal">Gesundheitswesen. </span>2005;<span class="ref-vol">67</span>(1):S26–S30.</span> [<a href="/pubmed/16032514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gesundheitswesen&amp;title=KORA-gen—resource+for+population+genetics,+controls+and+a+broad+spectrum+of+disease+phenotypes&amp;author=H.E.+Wichmann&amp;author=C.+Gieger&amp;author=T.+Illig&amp;volume=67&amp;issue=1&amp;publication_year=2005&amp;pages=S26-S30&amp;pmid=16032514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib27">Ref list</a>]</div><div id="body-link-popper-bib28" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">28. <span class="element-citation" id="sref28">Weiland S.K., von Mutius E., Hirsch T., Duhme H., Fritzsch C., Werner B., Hüsing A., Stender M., Renz H., Leupold W., Keil U. Prevalence of respiratory and atopic disorders among children in the East and West of Germany five years after unification. <span><span class="ref-journal">Eur. Respir. J. </span>1999;<span class="ref-vol">14</span>:862–870.</span> [<a href="/pubmed/10573234" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Respir.+J.&amp;title=Prevalence+of+respiratory+and+atopic+disorders+among+children+in+the+East+and+West+of+Germany+five+years+after+unification&amp;author=S.K.+Weiland&amp;author=E.+von+Mutius&amp;author=T.+Hirsch&amp;author=H.+Duhme&amp;author=C.+Fritzsch&amp;volume=14&amp;publication_year=1999&amp;pages=862-870&amp;pmid=10573234&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib28">Ref list</a>]</div><div id="body-link-popper-bib29" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">29. <span class="element-citation" id="sref29">O’Donovan M.C., Craddock N., Norton N., Williams H., Peirce T., Moskvina V., Nikolov I., Hamshere M., Carroll L., Georgieva L., Molecular Genetics of Schizophrenia Collaboration  Identification of loci associated with schizophrenia by genome-wide association and follow-up. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:1053–1055.</span> [<a href="/pubmed/18677311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+loci+associated+with+schizophrenia+by+genome-wide+association+and+follow-up&amp;author=M.C.+O’Donovan&amp;author=N.+Craddock&amp;author=N.+Norton&amp;author=H.+Williams&amp;author=T.+Peirce&amp;volume=40&amp;publication_year=2008&amp;pages=1053-1055&amp;pmid=18677311&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib29">Ref list</a>]</div><div id="body-link-popper-bib21" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">21. <span class="element-citation" id="sref21">Nair R.P., Duffin K.C., Helms C., Ding J., Stuart P.E., Goldgar D., Gudjonsson J.E., Li Y., Tejasvi T., Feng B.J., Collaborative Association Study of Psoriasis  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=R.P.+Nair&amp;author=K.C.+Duffin&amp;author=C.+Helms&amp;author=J.+Ding&amp;author=P.E.+Stuart&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib21">Ref list</a>]</div><div id="body-link-popper-bib11 bib12" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib11 bib20" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib30" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">30. <span class="element-citation" id="sref31">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib30">Ref list</a>]</div><div id="body-link-popper-bib31" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">31. <span class="element-citation" id="sref32">Delaneau O., Marchini J., Zagury J.F. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat. Methods. </span>2012;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O.+Delaneau&amp;author=J.+Marchini&amp;author=J.F.+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib31">Ref list</a>]</div><div id="body-link-popper-bib32" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">32. <span class="element-citation" id="sref33">Marchini J., Howie B., Myers S., McVean G., Donnelly P. A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J.+Marchini&amp;author=B.+Howie&amp;author=S.+Myers&amp;author=G.+McVean&amp;author=P.+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib32">Ref list</a>]</div><div id="body-link-popper-bib33" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">33. <span class="element-citation" id="sref34">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib33">Ref list</a>]</div><div id="body-link-popper-bib34 bib35" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib36" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">36. <span class="element-citation" id="sref37">Bhattacharjee S., Rajaraman P., Jacobs K.B., Wheeler W.A., Melin B.S., Hartge P., Yeager M., Chung C.C., Chanock S.J., Chatterjee N., GliomaScan Consortium  A subset-based approach improves power and interpretation for the combined analysis of genetic association studies of heterogeneous traits. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">90</span>:821–835.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3376551/">PMC free article</a>]</span> [<a href="/pubmed/22560090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+subset-based+approach+improves+power+and+interpretation+for+the+combined+analysis+of+genetic+association+studies+of+heterogeneous+traits&amp;author=S.+Bhattacharjee&amp;author=P.+Rajaraman&amp;author=K.B.+Jacobs&amp;author=W.A.+Wheeler&amp;author=B.S.+Melin&amp;volume=90&amp;publication_year=2012&amp;pages=821-835&amp;pmid=22560090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib36">Ref list</a>]</div><div id="body-link-popper-bib37" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">37. <span class="element-citation" id="sref38">Morris A.P. Transethnic meta-analysis of genomewide association studies. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:809–822.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3460225/">PMC free article</a>]</span> [<a href="/pubmed/22125221" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Transethnic+meta-analysis+of+genomewide+association+studies&amp;author=A.P.+Morris&amp;volume=35&amp;publication_year=2011&amp;pages=809-822&amp;pmid=22125221&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib37">Ref list</a>]</div><div id="body-link-popper-bib38" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">38. <span class="element-citation" id="sref39">Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=C.J.+Willer&amp;author=Y.+Li&amp;author=G.R.+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib38">Ref list</a>]</div><div id="body-link-popper-bib39" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">39. <span class="element-citation" id="sref40">Wakefield J. Bayes factors for genome-wide association studies: comparison with P-values. <span><span class="ref-journal">Genet. Epidemiol. </span>2009;<span class="ref-vol">33</span>:79–86.</span> [<a href="/pubmed/18642345" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Bayes+factors+for+genome-wide+association+studies:+comparison+with+P-values&amp;author=J.+Wakefield&amp;volume=33&amp;publication_year=2009&amp;pages=79-86&amp;pmid=18642345&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib39">Ref list</a>]</div><div id="body-link-popper-bib21 bib24 bib40" class="body-link-popper" style="display: none"></div><div id="body-link-popper-bib41" class="body-link-popper ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic" style="display: none; width: 30em; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true">41. <span class="element-citation" id="sref42">Esparza-Gordillo J., Weidinger S., Fölster-Holst R., Bauerfeind A., Ruschendorf F., Patone G., Rohde K., Marenholz I., Schulz F., Kerscher T. A common variant on chromosome 11q13 is associated with atopic dermatitis. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:596–601.</span> [<a href="/pubmed/19349984" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+on+chromosome+11q13+is+associated+with+atopic+dermatitis&amp;author=J.+Esparza-Gordillo&amp;author=S.+Weidinger&amp;author=R.+Fölster-Holst&amp;author=A.+Bauerfeind&amp;author=F.+Ruschendorf&amp;volume=41&amp;publication_year=2009&amp;pages=596-601&amp;pmid=19349984&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4289690&amp;issue-id=247455&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span> [<a href="#bib41">Ref list</a>]</div><div id="body-link-popper-bib20 bib42" class="body-link-popper" style="display: none"></div></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:12:01-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B30A6DD5760710000000007D40496&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4289690%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4289690/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>