<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jid"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4747615/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Polymorphisms in HLA Class II Genes Are Associated With Susceptibility to Staphylococcus aureus Infection in a White Population</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747615/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Journal of Infectious Diseases"><meta name="citation_title" content="Polymorphisms in HLA Class II Genes Are Associated With Susceptibility to Staphylococcus aureus Infection in a White Population"><meta name="citation_authors" content="Gerald N. DeLorenze, Charlotte L. Nelson, William K. Scott, Andrew S. Allen, G. Thomas Ray, Ai-Lin Tsai, Charles P. Quesenberry, Jr, Vance G. Fowler, Jr"><meta name="citation_date" content="1 March 2016"><meta name="citation_issue" content="5"><meta name="citation_volume" content="213"><meta name="citation_firstpage" content="816"><meta name="citation_doi" content="10.1093/infdis/jiv483"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4747615/?report=abstract"><meta name="citation_pmid" content="26450422"><meta name="DC.Title" content="Polymorphisms in HLA Class II Genes Are Associated With Susceptibility to Staphylococcus aureus Infection in a White Population"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Gerald N. DeLorenze"><meta name="DC.Contributor" content="Charlotte L. Nelson"><meta name="DC.Contributor" content="William K. Scott"><meta name="DC.Contributor" content="Andrew S. Allen"><meta name="DC.Contributor" content="G. Thomas Ray"><meta name="DC.Contributor" content="Ai-Lin Tsai"><meta name="DC.Contributor" content="Charles P. Quesenberry, Jr"><meta name="DC.Contributor" content="Vance G. Fowler, Jr"><meta name="DC.Date" content="2016 Mar 1"><meta name="DC.Identifier" content="10.1093/infdis/jiv483"><meta name="DC.Language" content="en"><meta property="og:title" content="Polymorphisms in HLA Class II Genes Are Associated With Susceptibility to Staphylococcus aureus Infection in a White Population"><meta property="og:type" content="article"><meta property="og:description" content="Background. Staphylococcus aureus can cause life-threatening infections. Human susceptibility to S. aureus infection may be influenced by host genetic variation.Methods. A genome-wide association study (GWAS) in a large health plan–based ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747615/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jid.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4747615/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4747615/pdf/jiv483.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8BCF28DD58A27100000000036901F8.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747615%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="protein">Protein</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22 J Infect Dis%22[journal]" class="navlink">J Infect Dis</a></li><li class="accid">PMC4747615</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jid.png" alt="Logo of jid" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://academic.oup.com/jid/" target="pmc_ext" ref="reftype=publisher&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Infect Dis</a></span>. 2016 Mar 1; 213(5): 816–823. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Oct 8. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Finfdis%2Fjiv483" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/infdis/jiv483</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4747615</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26450422">26450422</a></div></div></div></div><h1 class="content-title">Polymorphisms in HLA Class II Genes Are Associated With Susceptibility to <em>Staphylococcus aureus</em> Infection in a White Population</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=DeLorenze%20GN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318876570928" co-class="co-affbox">Gerald N. DeLorenze</a>,<sup>1</sup> <a href="/pubmed/?term=Nelson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318877882352" co-class="co-affbox">Charlotte L. Nelson</a>,<sup>2</sup> <a href="/pubmed/?term=Scott%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318839216912" co-class="co-affbox">William K. Scott</a>,<sup>5,</sup><sup>6</sup> <a href="/pubmed/?term=Allen%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318839214192" co-class="co-affbox">Andrew S. Allen</a>,<sup>4</sup> <a href="/pubmed/?term=Ray%20GT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318804398512" co-class="co-affbox">G. Thomas Ray</a>,<sup>1</sup> <a href="/pubmed/?term=Tsai%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318804396528" co-class="co-affbox">Ai-Lin Tsai</a>,<sup>1</sup> <a href="/pubmed/?term=Quesenberry%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318804394544" co-class="co-affbox">Charles P. Quesenberry, Jr</a>,<sup>1</sup> and  <a href="/pubmed/?term=Fowler%20VG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422" class="affpopup" co-rid="_co_idm140318839266416" co-class="co-affbox">Vance G. Fowler, Jr</a><sup>2,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140318876570928"><h3 class="no_margin">Gerald N. DeLorenze</h3><p><sup>1</sup>Division of Research, Kaiser Permanente Northern California, Oakland</p><div>Find articles by <a href="/pubmed/?term=DeLorenze%20GN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Gerald N. DeLorenze</a></div></div><div id="_co_idm140318877882352"><h3 class="no_margin">Charlotte L. Nelson</h3><p><sup>2</sup>Duke Clinical Research Institute</p><div>Find articles by <a href="/pubmed/?term=Nelson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Charlotte L. Nelson</a></div></div><div id="_co_idm140318839216912"><h3 class="no_margin">William K. Scott</h3><p><sup>5</sup>John P. Hussman Institute for Human Genomics</p><p><sup>6</sup>Dr. John T. Macdonald Foundation Department of Human Genetics, University of Miami Miller School of Medicine, Florida</p><div>Find articles by <a href="/pubmed/?term=Scott%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">William K. Scott</a></div></div><div id="_co_idm140318839214192"><h3 class="no_margin">Andrew S. Allen</h3><p><sup>4</sup>Department of Biostatistics and Bioinformatics, Duke University School of Medicine, Durham, North Carolina</p><div>Find articles by <a href="/pubmed/?term=Allen%20AS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Andrew S. Allen</a></div></div><div id="_co_idm140318804398512"><h3 class="no_margin">G. Thomas Ray</h3><p><sup>1</sup>Division of Research, Kaiser Permanente Northern California, Oakland</p><div>Find articles by <a href="/pubmed/?term=Ray%20GT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">G. Thomas Ray</a></div></div><div id="_co_idm140318804396528"><h3 class="no_margin">Ai-Lin Tsai</h3><p><sup>1</sup>Division of Research, Kaiser Permanente Northern California, Oakland</p><div>Find articles by <a href="/pubmed/?term=Tsai%20AL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Ai-Lin Tsai</a></div></div><div id="_co_idm140318804394544"><h3 class="no_margin">Charles P. Quesenberry, Jr</h3><p><sup>1</sup>Division of Research, Kaiser Permanente Northern California, Oakland</p><div>Find articles by <a href="/pubmed/?term=Quesenberry%20CP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Charles P. Quesenberry, Jr</a></div></div><div id="_co_idm140318839266416"><h3 class="no_margin">Vance G. Fowler, Jr</h3><p><sup>2</sup>Duke Clinical Research Institute</p><p><sup>3</sup>Division of Infectious Diseases, Duke University Medical Center</p><div>Find articles by <a href="/pubmed/?term=Fowler%20VG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26450422">Vance G. Fowler, Jr</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140318843584848_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140318843584848_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140318843584848_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140318843584848_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Division of Research, Kaiser Permanente Northern California, Oakland</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Duke Clinical Research Institute</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Division of Infectious Diseases, Duke University Medical Center</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Department of Biostatistics and Bioinformatics, Duke University School of Medicine, Durham, North Carolina</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>John P. Hussman Institute for Human Genomics</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Dr. John T. Macdonald Foundation Department of Human Genetics, University of Miami Miller School of Medicine, Florida</div><div><p id="__p1" class="p p-first-last">Presented in part: European Congress of Clinical Microbiology and Infectious Diseases, May 2014, Barcelona, Spain.</p></div><div id="idm140318803560352">Correspondence: G. N. DeLorenze, Division of Research, Kaiser Permanente Northern California, 2000 Broadway, Oakland, CA 94612 (<a href="mailto:dev@null" data-email="gro.pk@ezneroled.n.dlareg" class="oemail">gro.pk@ezneroled.n.dlareg</a>).</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140318843584848_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Jul 21; Accepted 2015 Sep 30.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140318843584848_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2015. Published by Oxford University Press for the Infectious Diseases Society of America. All rights reserved. For permissions, e-mail <a href="mailto:dev@null" data-email="moc.puo@snoissimrep.slanruoj" class="oemail">moc.puo@snoissimrep.slanruoj</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4747615/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4747615/bin/supp_213_5_816__index.html" data-ga-action="click_feat_suppl">supp_213_5_816__index.html</a><span style="color:gray"> (1.3K)</span></div><div class="small guid">GUID:&nbsp;25206EB2-6F0C-4127-866A-623D208810BD</div>
<div><a href="/pmc/articles/PMC4747615/bin/supp_jiv483_jiv483supp_figs.docx" data-ga-action="click_feat_suppl">supp_jiv483_jiv483supp_figs.docx</a><span style="color:gray"> (2.7M)</span></div><div class="small guid">GUID:&nbsp;648D411A-AF13-4585-9215-D3328B065E25</div>
<div><a href="/pmc/articles/PMC4747615/bin/supp_jiv483_jiv483supp_file1.docx" data-ga-action="click_feat_suppl">supp_jiv483_jiv483supp_file1.docx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;D90112F2-A6AC-4E4E-9AA4-6DADD64CE9F0</div>
<div><a href="/pmc/articles/PMC4747615/bin/supp_jiv483_jiv483supp_file2.docx" data-ga-action="click_feat_suppl">supp_jiv483_jiv483supp_file2.docx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;01C412A4-E589-46CA-A132-FE666494856E</div>
</div></dd></dl></div><div id="idm140318840833248" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140318840833248title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first"><strong><em>Background.</em></strong> <em>Staphylococcus aureus</em> can cause life-threatening infections. Human susceptibility to <em>S. aureus</em> infection may be influenced by host genetic variation.</p><p id="__p3"><strong><em>Methods.</em></strong> A genome-wide association study (GWAS) in a large health plan–based cohort included biologic specimens from 4701 culture-confirmed <em>S. aureus</em> cases and 45 344 matched controls; 584 535 single-nucleotide polymorphisms (SNPs) were genotyped on an array specific to individuals of European ancestry. Coverage was increased by imputation of &gt;25 million common SNPs, using the 1000 Genomes Reference panel. In addition, human leukocyte antigen (HLA) serotypes were also imputed.</p><p id="__p4"><strong><em>Results.</em></strong> Logistic regression analysis, performed under the assumption of an additive genetic model, revealed several imputed SNPs (eg, rs115231074: odds ratio [OR], 1.22 [<em>P</em> = 1.3 × 10<sup>−10</sup>]; rs35079132: OR, 1.24 [<em>P</em> = 3.8 × 10<sup>−8</sup>]) achieving genome-wide significance on chromosome 6 in the HLA class II region. One adjacent genotyped SNP was nearly genome-wide significant (rs4321864: OR, 1.13; <em>P</em> = 8.8 × 10<sup>−8</sup>). These polymorphisms are located near the genes encoding HLA-DRA and HLA-DRB1. Results of further logistic regression analysis, in which the most significant GWAS SNPs were conditioned on HLA-DRB1*04 serotype, showed additional support for the strength of association between HLA class II genetic variants and <em>S. aureus</em> infection.</p><p id="__p5" class="p p-last"><strong><em>Conclusions.</em></strong> Our study results are the first reported evidence of human genetic susceptibility to <em>S. aureus</em> infection.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text"><em>Staphylococcus aureus</em>, host genetics, HLA</span></div></div><div id="idm140318800323760" class="tsec sec headless whole_rhythm"><p id="__p6" class="p p-first"><em>Staphylococcus aureus</em> is both a harmless colonizer and a leading cause of life-threatening infections. A number of observations suggest a genetic basis for human susceptibility to <em>S. aureus</em>, including variable genetic susceptibility to <em>S. aureus</em> infection in inbred mice [<a href="#JIV483C1" rid="JIV483C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219236">1</a>, <a href="#JIV483C2" rid="JIV483C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219239">2</a>], cattle [<a href="#JIV483C3" rid="JIV483C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219242">3</a>], and sheep [<a href="#JIV483C4" rid="JIV483C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219227">4</a>]; familial clusters of <em>S. aureus</em> infection [<a href="#JIV483C5" rid="JIV483C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219228">5</a>]; and genetic conditions conferring susceptibility to <em>S. aureus</em> (eg, Job syndrome and Chediak-Higashi syndrome) [<a href="#JIV483C6" rid="JIV483C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219276">6</a>, <a href="#JIV483C7" rid="JIV483C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219250">7</a>]. Two previous investigations have used genome-wide association studies (GWAS) to evaluate human genetic susceptibility to <em>S. aureus</em> infection [<a href="#JIV483C8" rid="JIV483C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219256">8</a>, <a href="#JIV483C9" rid="JIV483C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219273">9</a>]. Both were likely underpowered to detect effects at genome-wide significance. As a result, the impact of host genetic variation on the susceptibility to <em>S. aureus</em> infection is largely unknown.</p><p id="__p7" class="p">Our goal for the current study was to determine which host genetic polymorphisms were associated with (1) all <em>S. aureus</em> infections and (2) a subset of community-acquired skin and soft tissue <em>S. aureus</em> infections (SSTIs), using genomic data from &gt;50 000 white individuals. We hypothesized that genetic variants that encode proteins with immune functions would be associated with an increased (or decreased) risk of <em>S. aureus</em> infection. A second objective was to determine whether including imputed genetic variants [<a href="#JIV483C10" rid="JIV483C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219258">10</a>] would reveal additional gene–<em>S. aureus</em> infection associations. A third objective was to evaluate the strength of potential significant associations from the HLA regions of chromosome 6 by conditioning on imputed HLA serotypes [<a href="#JIV483C11" rid="JIV483C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219234">11</a>].</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">MATERIALS AND METHODS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Study Sample</h3><p id="__p8" class="p p-first">The study population consists of participants in the Research Program on Genes, Environment, and Health (RPGEH). This cohort was recruited from 3.3 million patients in the Kaiser Permanente Northern California (KPNC) health plan. The RPGEH includes a cohort with baseline survey data obtained in 2007–2008 from 400 000 adult patients in KPNC who had ≥2 years membership prior to the survey (average time, 23.5 years). More than 200 000 patients who had completed surveys were selected to receive Oragene saliva collection kits. Genotyping was conducted using returned saliva specimens from 110 266 RPGEH subjects (of whom 89 341 self-identified as white). The RPGEH imputed &gt;25 million SNPs not covered in the original genotyping arrays, as well as HLA major histocompatibility complex (MHC) serotypes. Informed consent was obtained from all RPGEH subjects. Questionnaire data included demographic characteristics (eg, age and sex), as well as self-reports of disease history (eg, diabetes). The RPGEH study and the current study were approved by the KPNC institutional review board.</p><p id="__p9" class="p p-last">KPNC maintains electronic databases that include data on hospitalizations, clinic visits, laboratory testing results, and pharmacy dispensing records. These databases were used to create the case-control sample in the present study.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Identification of Cases</h3><p id="__p10" class="p p-first">The study's phenotypes included culture-confirmed diagnoses of <em>S. aureus</em> infections identified using standard methods in the clinical microbiology laboratory of KPNC [<a href="#JIV483C12" rid="JIV483C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219251">12</a>]. For repeated testing, we adapted the Clinical and Laboratory Standards Institute guideline and recommendations in the literature, including only the first isolate per person in a 365-day period [<a href="#JIV483C13" rid="JIV483C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219274">13</a>, <a href="#JIV483C14" rid="JIV483C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219230">14</a>]. To identify isolates likely to be related to clinically relevant infections, we restricted our analyses to blood, bone, cerebrospinal fluid, body fluid, urine, tissue, respiratory, and miscellaneous bacterial specimens (eg, abscesses). Screening tests (nares) and cultures of genital specimens, feces/stool specimens, catheter tips, and throat specimens were excluded. The primary study phenotype included all <em>S. aureus</em> infections diagnosed in white subjects of the RPGEH cohort between 1995 and 2011. The secondary phenotype was the subset of community-acquired SSTIs diagnosed during the same interval. SSTIs were ascertained via linking to diagnostic data in the electronic databases (<em>International Classification of Diseases, Ninth Revision, Clinical Modification</em> [ICD-9]). SSTI diagnostic codes included erysipelas (035), cellulitis and abscess (566, 67510–67514, 6810–6829, 6850), mastitis (67520–67524), carbuncle and furuncle (6800–6809), acute lymphadenitis (683), impetigo (684), other skin infections (6860–6861, 6868–6869), folliculitis (7048), and hidradenitis (70583). An infection that was diagnosed ≥48 hours after admission to the hospital was classified as a hospital-onset case; all other infections were classified as community acquired.</p><p id="__p11" class="p p-last">Study controls included RPGEH subjects with no evidence of culture-confirmed <em>S. aureus</em> infection. Controls lacking a positive isolate but having records of an ICD-9 code for <em>S. aureus</em> infection and subsequent treatment with antibiotics were excluded. Our study used a frequency matched case-control design. Cases were matched on age (5-year age groups) at the time of specimen collection and sex to approximately 10 controls.</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">DNA Isolation, Genotyping, and Quality Control</h3><p id="__p12" class="p p-first">DNA was extracted from saliva specimens, using an Agencourt AMPure XP kit (Beckman Coulter) in a high-throughput process. Genotyping was accomplished using the Affymetrix Axiom Genotyping Solution (available at: <a href="http://media.affymetrix.com/support/technical/datasheets/axiom_genotypingsolution_datasheet.pdf">http://media.affymetrix.com/support/technical/datasheets/axiom_genotypingsolution_datasheet.pdf</a>). The Axiom genotyping platform is a 2-color, ligation-based assay that uses 30-mer oligonucleotide probes synthesized in situ on a microarray substrate, with 96 samples per plate. A maximum of approximately 690 000 SNPs may be accommodated by this format. Performance of the array was assessed by assaying the white and Yoruban HapMap2 [<a href="#JIV483C15" rid="JIV483C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219272">15</a>] populations. Call rates, sample concordance, reproducibility, and Mendelian consistency were extremely high. A large majority of SNPs have overall call rates of ≥97%. Genotyping of 89 341 saliva samples from the European (EUR) ancestry study subjects was completed using 3 Affymetrix Gene Titan systems and 3 Beckman Biomek Systems. The Affymetrix Powertools Package, version 1.12.0 (available at: <a href="http://www.affymetrix.com/partners_programs/programs/developer/tools/powertools.affx?hightlight=true&amp;rootCategoryId=34002#13">http://www.affymetrix.com/partners_programs/programs/developer/tools/powertools.affx?hightlight=true&amp;rootCategoryId=34002#13</a>), was used to make genotype calls. The Affymetrix Axiom EUR array included 674 112 SNPs: 116 were mitochondrial, 289 were on the Y chromosome, 388 were in pseudo-autosomal regions of the X and Y chromosomes, and the remaining 660 989 SNPs were autosomal.</p><p id="__p13">Examination of graphics from principal components (PC) analysis [<a href="#JIV483C16" rid="JIV483C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219271">16</a>] (see the “GWAS Data Analysis” section, below) led to the identification of some individuals whose genetic ancestry appeared to be discordant from their self-report on the RPGEH survey (available at: <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetPdf.cgi?id=phd004309">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/GetPdf.cgi?id=phd004309</a>). Some individuals with data on the African (AFR) array were estimated to have 100% European ancestry. Investigation of so-called discordant individuals revealed a discrepancy between the survey form and the computerized records from optical scanning. About 2% of surveys had been mis-scanned for race/ethnicity/nationality. This led to the systematic reassignment of these individuals to their original survey responses, supplemented by race/ethnicity information in the KPNC databases.</p><p id="__p14" class="p p-last">Our quality control (QC) filtering excluded SNPs with genotyping call rates of &lt;95%, a minor allele frequency (MAF) of &lt;0.4% (primary analysis) and &lt;0.9% (secondary analysis), or a statistically significant (<em>P</em> &lt; 10<sup>−3</sup>) departure from Hardy–Weinberg equilibrium (HWE) in the controls. Subjects with missing genotype rates of &gt;5% or mismatch between reported and genetically determined sex were excluded. Samples were evaluated for cryptic relatedness through estimation of kinship coefficient, using KING [<a href="#JIV483C17" rid="JIV483C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219241">17</a>] software. One sample was randomly excluded from sample pairs exhibiting a kinship coefficient of ≥0.0625.</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">Genotype Imputation Method</h3><p id="__p15" class="p p-first-last">To increase coverage of common variants in the genotyped platform, particularly in the region where our most significant genotyped SNP GWAS results were found, imputation was done on the EUR array (for a comparison of SNP imputation vs other multiple imputation techniques, see <a href="http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jiv483/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary 1</a>). This approach is now a well-established method to increase marker density in GWAS [<a href="#JIV483C18" rid="JIV483C18" class=" bibr popnode">18</a>] and takes advantage of known multi-SNP haplotype structures that have been determined by sequencing large diversity panels, such as the 1000 Genomes Project [<a href="#JIV483C19" rid="JIV483C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219270">19</a>]. Therefore, untyped SNPs can be inferred from the genotyped markers included on a genome-wide array with high degrees of confidence and used in association analysis. The data were prephased (inferring haplotypes) with SHAPE-IT v2.r644 [<a href="#JIV483C20" rid="JIV483C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219235">20</a>], using the family structure of first-degree cryptic related individuals as available. With 1000 Genomes (phase I; March 2012) as a reference panel, SNP data were imputed using IMPUTE2 v2.2.2 [<a href="#JIV483C10" rid="JIV483C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219238">10</a>, <a href="#JIV483C21" rid="JIV483C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219244">21</a>].</p></div><div id="s2e" class="sec"><h3 id="s2etitle">HLA Serotype Imputation Method</h3><p id="__p16" class="p p-first-last">The HLA serotype imputation method used in our study was originally developed and described in detail by Jia et al [<a href="#JIV483C11" rid="JIV483C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219243">11</a>]. Briefly, the complex genetic structure of MHC makes it difficult to collect high-resolution haplotype data in large cohorts. Long-range linkage disequilibrium (LD) between HLA loci and SNPs across the MHC region offers an alternative approach through imputation to interrogate HLA. We used the SNP2HLA program to impute individual amino acid–changing polymorphisms and classical 4-digit haplotypes at class I and II loci. SNP2HLA was developed using the following reference panels: (1) HapMap-CEPH (individuals of European ancestry) [<a href="#JIV483C22" rid="JIV483C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219249">22</a>] and (2) Type 1 Diabetes Genetics Consortium reference panel (5225 unrelated individuals) [<a href="#JIV483C23" rid="JIV483C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219275">23</a>]. Beagle software [<a href="#JIV483C18" rid="JIV483C18" class=" bibr popnode">18</a>] was used to phase genotype data into individual haplotypes, taking into account familial relationships. Genotyped SNPs were extracted from within the MHC region (chromosome 6: 29–34 Mb on build 36/hg18); SNPs with a MAF of &lt;2.5% were removed. Beagle imputed all missing SNPs, classical HLA alleles, and amino acid polymorphisms. Output included posterior probabilities, allelic dosages, and phased haplotypes for each individual.</p></div><div id="s2f" class="sec"><h3 id="s2ftitle">Statistical Power</h3><p id="__p17" class="p p-first-last">The original statistical power calculations showed that, under an additive logistic model with a 2-sided test, an α of 5 × 10<sup>−8</sup>, and a MAF of &gt; 0.10, there was sufficient power to detect odds ratios (ORs) of ≥1.20, given a (pre-QC filtering) sample size of 56 100 subjects (5100 cases and 51 000 controls).</p></div><div id="s2g" class="sec sec-last"><h3 id="s2gtitle">GWAS Data Analysis</h3><p id="__p18" class="p p-first-last">Our study conducted case-control analyses for each <em>S. aureus</em> phenotype, using an additive genetic model of inheritance. Unconditional logistic regression was used for both primary outcome (all <em>S. aureus</em> infections) and secondary outcome (community-acquired SSTIs), testing the association with each genotyped and each imputed SNP separately. Logistic regression was also used in association testing of the imputed HLA serotype data in relation to the primary phenotype, as well as in further analysis, in which the 3 most significant SNPs were conditioned on an imputed HLA serotype variant. To account for population stratification and admixture, our regression models included adjustment for the first 10 eigenvectors from a PC analysis [<a href="#JIV483C24" rid="JIV483C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219248">24</a>], using Eigensoft 4.2. To reflect the study sampling scheme, our regression models included age (5-year intervals) at time of specimen collection and sex covariates. PLINK software [<a href="#JIV483C25" rid="JIV483C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219264">25</a>] was used in the association testing analyses of the genotyped data for both phenotypes [<a href="#JIV483C25" rid="JIV483C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219259">25</a>]. PLINK and R software were used to analyze the imputed SNP and imputed HLA serotype data.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">RESULTS</h2><p id="__p19" class="p p-first">The initial study sample for the primary phenotype analysis (ie, all culture-confirmed <em>S. aureus</em> infections as outcome) included 53 322 subjects (4997 <em>S. aureus</em> cases and 48 325 controls). During QC filtering, 16 513 SNPs with genotyping call rates of &lt;95%, 6958 with a MAF of &lt;0.4%, and 66 106 showing significant departure from HWE in controls were excluded, leaving 584 535 SNPs for data analysis. Removal from the data set prior to analysis occurred for 9 subjects whose monozygotic twin was also in the study sample, 3178 subjects whose sample pair had a kinship coefficient of ≥0.0625, 54 controls with ICD-9 diagnostic coding for <em>S. aureus</em> infection followed by coding for appropriate treatment, 24 subjects who had a mismatch between their reported and genetically determined sex, and 12 subjects (of South Asian ancestry) who did not have estimates for European PC. Thus, our final sample for the primary phenotype analysis consisted of 50 045 unique subjects (4701 <em>S. aureus</em> cases and 45 344 controls).</p><p id="__p20">Table <a href="/pmc/articles/PMC4747615/table/JIV483TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="JIV483TB1" rid-ob="ob-JIV483TB1" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table1</span></span><span>1</span></a> shows the distribution of study characteristics among 50 045 subjects. Approximately 51% of cases and 49% of controls were male; 73% of cases and 74% of controls were ≥60 years of age at the time of specimen collection. Fifteen percent of cases and 13% of controls had a history of diabetes. Of the 4701 <em>S. aureus</em> cases, 28% were methicillin-resistant <em>S. aureus</em> (MRSA) infections, 50% were SSTIs, and 96% were community-acquired infections. Laboratory order codes for isolates from the 4701 confirmed cases included blood culture for 3%, cerebral spinal fluid/other body fluid for 3%, and miscellaneous bacterial culture for 61%.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="JIV483TB1"><h3>Table 1.</h3><!--caption a7--><div class="caption"><p id="__p21">Distribution of Study Characteristics for 50 045 Subjects With (Cases) or Without (Controls) <em>Staphylococcus aureus</em> Infection in the Primary Phenotype Analysis Data Set</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140318801249616" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Characteristic</th><th rowspan="1" colspan="1" align="center">Cases, No. (%)</th><th rowspan="1" colspan="1" align="center">Controls, No. (%)</th></tr></thead><tbody><tr><td colspan="3" rowspan="1">Sex</td></tr><tr><td rowspan="1" colspan="1"> Female</td><td rowspan="1" colspan="1">2315 (49)</td><td rowspan="1" colspan="1">23 322 (51)</td></tr><tr><td rowspan="1" colspan="1"> Male</td><td rowspan="1" colspan="1">2386 (51)</td><td rowspan="1" colspan="1">22 022 (49)</td></tr><tr><td colspan="3" rowspan="1">Age at specimen collection date, y</td></tr><tr><td rowspan="1" colspan="1"> 20–24</td><td rowspan="1" colspan="1">16 (&lt;1)</td><td rowspan="1" colspan="1">171 (&lt;1)</td></tr><tr><td rowspan="1" colspan="1"> 25–29</td><td rowspan="1" colspan="1">19 (&lt;1)</td><td rowspan="1" colspan="1">214 (&lt;1)</td></tr><tr><td rowspan="1" colspan="1"> 30–34</td><td rowspan="1" colspan="1">44 (1)</td><td rowspan="1" colspan="1">446 (1)</td></tr><tr><td rowspan="1" colspan="1"> 35–39</td><td rowspan="1" colspan="1">82 (2)</td><td rowspan="1" colspan="1">822 (2)</td></tr><tr><td rowspan="1" colspan="1"> 40–44</td><td rowspan="1" colspan="1">112 (2)</td><td rowspan="1" colspan="1">1067 (2)</td></tr><tr><td rowspan="1" colspan="1"> 45–49</td><td rowspan="1" colspan="1">194 (4)</td><td rowspan="1" colspan="1">1892 (4)</td></tr><tr><td rowspan="1" colspan="1"> 50–54</td><td rowspan="1" colspan="1">308 (7)</td><td rowspan="1" colspan="1">3069 (7)</td></tr><tr><td rowspan="1" colspan="1"> 55–59</td><td rowspan="1" colspan="1">461 (10)</td><td rowspan="1" colspan="1">4625 (10)</td></tr><tr><td rowspan="1" colspan="1"> 60–64</td><td rowspan="1" colspan="1">715 (15)</td><td rowspan="1" colspan="1">7079 (16)</td></tr><tr><td rowspan="1" colspan="1"> 65–69</td><td rowspan="1" colspan="1">666 (14)</td><td rowspan="1" colspan="1">6654 (15)</td></tr><tr><td rowspan="1" colspan="1"> 70–74</td><td rowspan="1" colspan="1">607 (13)</td><td rowspan="1" colspan="1">6134 (14)</td></tr><tr><td rowspan="1" colspan="1"> 75–79</td><td rowspan="1" colspan="1">629 (13)</td><td rowspan="1" colspan="1">6133 (14)</td></tr><tr><td rowspan="1" colspan="1"> ≥80</td><td rowspan="1" colspan="1">848 (18)</td><td rowspan="1" colspan="1">7038 (15)</td></tr><tr><td colspan="3" rowspan="1">History of diabetes</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">705 (15)</td><td rowspan="1" colspan="1">5895 (13)</td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">3996 (85)</td><td rowspan="1" colspan="1">39 449 (87)</td></tr><tr><td colspan="3" rowspan="1">History of cancer</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">799 (17)</td><td rowspan="1" colspan="1">7255 (16)</td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">3902 (83)</td><td rowspan="1" colspan="1">38 089 (84)</td></tr><tr><td colspan="3" rowspan="1">HIV infected</td></tr><tr><td rowspan="1" colspan="1"> Yes</td><td rowspan="1" colspan="1">66 (1.4)</td><td rowspan="1" colspan="1">136 (0.3)</td></tr><tr><td rowspan="1" colspan="1"> No</td><td rowspan="1" colspan="1">4635 (98.6)</td><td rowspan="1" colspan="1">45 208 (99.7)</td></tr><tr><td colspan="3" rowspan="1">Source of <em>S. aureus</em></td></tr><tr><td rowspan="1" colspan="1"> Blood culture</td><td rowspan="1" colspan="1">141 (3)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1"> Cerebrospinal fluid or other fluid</td><td rowspan="1" colspan="1">141 (3)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1"> Urine culture</td><td rowspan="1" colspan="1">282 (6)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1"> Respiratory culture</td><td rowspan="1" colspan="1">329 (7)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1"> Tissue/biopsy culture</td><td rowspan="1" colspan="1">940 (20)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1"> Miscellaneous bacterial culture</td><td rowspan="1" colspan="1">2868 (61)</td><td rowspan="1" colspan="1">…</td></tr><tr><td rowspan="1" colspan="1">Overall</td><td rowspan="1" colspan="1">4701 (100)</td><td rowspan="1" colspan="1">45 344 (100)</td></tr></tbody></table></div><div id="largeobj_idm140318801249616" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4747615/table/JIV483TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn1"><p id="__p22" class="p p-first-last">Abbreviation: HIV, human immunodeficiency virus.</p></div></div></div><p id="__p23">Prior to QC filtering, 50 576 subjects (2251 cases and 48 325 controls) were available for analysis of the secondary phenotype, community-acquired SSTI; 16 499 SNPs with call rates of &lt;95%, 13 752 with a MAF of &lt;0.9%, and 46 730 showing departure from HWE in controls were excluded, leaving 597 131 SNPs for data analysis. Eight subjects whose monozygotic twin was also in the study sample, 2915 subjects whose sample pair had a kinship coefficient of ≥0.0625, 54 controls with an ICD-9 code for <em>S. aureus</em> infection, 23 subjects who had a mismatch between self-reported and genetically determined sex, and 12 subjects who did not have estimates for European PC were removed. Our final analysis sample for the secondary phenotype consisted of 47 564 subjects (2130 cases and 45 434 controls).</p><p id="__p24" class="p">A Q-Q plot of the expected distribution of association test statistics across all genotyped SNPs in the primary phenotype analysis in comparison to observed <em>P</em> values is presented in <a href="http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jiv483/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 1</a>; the genomic inflation factor was estimated to be 1.01, indicating adequate control of population stratification. Table <a href="/pmc/articles/PMC4747615/table/JIV483TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="JIV483TB2" rid-ob="ob-JIV483TB2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a> presents results of logistic regression analysis for both genotyped and imputed SNPs in relation to the primary and the secondary phenotype outcomes. The primary phenotype model revealed 1 genotyped SNP, rs4321864, that approached genome-wide significance (OR, 1.13; <em>P</em> = 8.85 × 10<sup>−8</sup>; <a href="http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jiv483/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Figure 2</a>). This variant is located in the HLA class II region of chromosome 6 (position: 32 399 187-32 399 187; band: 6p21.32). SNP rs4321864 appears to be at the 5′ terminus of the HLA-DRA gene. Given that rs4321864 approached the threshold (<em>P</em> ≤ 5.0 × 10<sup>−8</sup>) of genome-wide significance, we conducted additional regression analyses examining imputed SNPs on chromosome 6. Two imputed HLA class II variants were significantly associated with the primary phenotype (Table <a href="/pmc/articles/PMC4747615/table/JIV483TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="JIV483TB2" rid-ob="ob-JIV483TB2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">(Table2):</span></span><span>2</span></a>): imputed SNP rs115231074 (OR, 1.22; <em>P</em> = 1.3 × 10<sup>−10</sup>) and rs35079132 (OR, 1.24; <em>P</em> = 3.8 × 10<sup>−8</sup>); imputed SNP rs189516143 (OR, 1.21; <em>P</em> = 9.2 × 10<sup>−8</sup>) approached genome-wide significance. All 3 SNPs are intergenic near the HLA-DRB1 gene locus. Another imputed SNP, rs17210959 (OR, 1.23; <em>P</em> = 3.5 × 10<sup>−7</sup>), is located within the HLA-DRB1 gene. A focused view of the region in chromosome 6 where these SNPs are located is shown in Figure <a href="/pmc/articles/PMC4747615/figure/JIV483F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="JIV483F1" rid-ob="ob-JIV483F1" co-legend-rid="lgnd_JIV483F1"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -3em;">Figure1.</span></span><span>1</span></a>. All of these SNPs had a MAF of &gt;0.10. A subanalysis in which MRSA infection was the outcome did not show any SNPs with <em>P</em> values that approached genome-wide significance (not shown). Also, analyses stratified by 3 large age groups did not show differences in SNP effect estimates across the age strata.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="JIV483TB2"><h3>Table 2.</h3><!--caption a7--><div class="caption"><p id="__p25">Final Results of Logistic Regression Analysis of Primary and Secondary Phenotypes Versus Genotyped and Imputed Single-Nucleotide Polymorphisms (SNPs), and Primary Phenotype vs Imputed HLA Serotype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140318842507296" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Model, SNP</th><th rowspan="1" colspan="1" align="center">Chromosome</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">Location</th><th rowspan="1" colspan="1" align="center">A1</th><th rowspan="1" colspan="1" align="center">A2</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center"><em>P</em> Value<sup>a</sup></th></tr></thead><tbody><tr><td colspan="8" rowspan="1">Primary phenotype (all <em>S. aureus</em> infections) SNP association model</td></tr><tr><td colspan="8" rowspan="1"> Genotyped</td></tr><tr><td rowspan="1" colspan="1">  rs4321864</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRA</td><td rowspan="1" colspan="1">5′ end of gene</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1">8.8 × 10<sup>−8</sup></td></tr><tr><td colspan="8" rowspan="1"> Imputed</td></tr><tr><td rowspan="1" colspan="1">  rs115231074</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic: 3′ downstream of –DRB1</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.22</td><td rowspan="1" colspan="1">1.3 × 10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">  rs35079132</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.24</td><td rowspan="1" colspan="1">3.8 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">  rs189516143</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">1.21</td><td rowspan="1" colspan="1">9.2 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">  rs190073676</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">1.18</td><td rowspan="1" colspan="1">1.6 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">  rs184932624</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.23</td><td rowspan="1" colspan="1">2.5 × 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">  rs17210959</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Within locus</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">1.23</td><td rowspan="1" colspan="1">3.5 × 10<sup>−7</sup></td></tr><tr><td colspan="8" rowspan="1">Secondary phenotype (community-acquired SST<sup>c</sup>
<em>S. aureus</em> infections) SNP association model</td></tr><tr><td colspan="8" rowspan="1"> Genotyped</td></tr><tr><td rowspan="1" colspan="1">  rs4321864</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRA</td><td rowspan="1" colspan="1">5′ end of gene</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">8.1 × 10<sup>−5</sup></td></tr><tr><td colspan="8" rowspan="1"> Imputed</td></tr><tr><td rowspan="1" colspan="1">  rs115231074</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic: 3′ downstream of –DRB1</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">1.28</td><td rowspan="1" colspan="1">5.2 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">  rs12526396</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">4.3 × 10<sup>−7</sup></td></tr><tr><td colspan="8" rowspan="1">Primary phenotype (all <em>S. aureus</em> infections) imputed HLA serotype association model</td></tr><tr><td rowspan="1" colspan="1"> Imputed HLA serotype</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">  HLA_DRB1_04</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">Present</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">1.08</td><td rowspan="1" colspan="1">.01</td></tr><tr><td rowspan="1" colspan="1">  HLA_DRB1_0401</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">Present</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">1.08</td><td rowspan="1" colspan="1">.04</td></tr><tr><td rowspan="1" colspan="1">  HLA_DRB1_0402</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">Present</td><td rowspan="1" colspan="1">…</td><td rowspan="1" colspan="1">1.36</td><td rowspan="1" colspan="1">.002</td></tr></tbody></table></div><div id="largeobj_idm140318842507296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4747615/table/JIV483TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn2"><p id="__p26" class="p p-first-last">Analyses were adjusted for age, sex, and the first 10 eigenvectors from a principal components analysis.</p></div><div id="tblfn3"><p id="__p27" class="p p-first-last">Abbreviations: A1, minor allele; A2, major allele; OR, odds ratio; <em>S. aureus</em>, <em>Staphylococcus aureus</em>; SST, skin and soft tissue.</p></div><div id="tblfn4"><p id="__p28" class="p p-first-last"><sup>a</sup> Threshold for genome-wide significance: <em>P</em> ≤ 5 × 10<sup>−8</sup>.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="JIV483F1" co-legend-rid="lgnd_JIV483F1"><a href="/pmc/articles/PMC4747615/figure/JIV483F1/" target="figure" rid-figpopup="JIV483F1" rid-ob="ob-JIV483F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140318840323984" class="figure"><a href="/pmc/articles/PMC4747615/figure/JIV483F1/" target="figure" rid-figpopup="JIV483F1" rid-ob="ob-JIV483F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4747615_jiv48301.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is jiv48301.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4747615/bin/jiv48301.jpg"></a></div><div id="largeobj_idm140318840323984" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4747615/figure/JIV483F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_JIV483F1"><div><a class="figpopup" href="/pmc/articles/PMC4747615/figure/JIV483F1/" target="figure" rid-figpopup="JIV483F1" rid-ob="ob-JIV483F1">Figure 1.</a></div><!--caption a7--><div class="caption"><p id="__p29">Primary (all <em>Staphylococcus aureus</em> infections) phenotype: focused view of the locus of genotyped and imputed single-nucleotide polymorphisms (SNPs) between 32.3 and 32.5 Mb on chromosome 6. The plot also shows the SNP location in relation to known genes in that locus. Linkage disequilibrium of index SNP rs115231074 with other plotted SNPs is shown as a range of r<sup>2</sup> values, in which red = 0.81–1.0, orange = 0.61–0.80, green = 0.41–0.60, light blue = 0.21–0.40, and dark blue = 0.01–0.20.</p></div></div></div><div id="s3a" class="sec"><h3 id="s3atitle">Secondary Analysis: Community-acquired SSTI</h3><p id="__p30" class="p p-first">Results from the secondary phenotype analyses (community-acquired SSTIs) demonstrated that genotyped SNP rs4321864 did not approach genome-wide significance (<em>P</em> = 8.1 × 10<sup>−5</sup>). However, imputed SNP rs115231074 closely approached genome-wide significance (OR, 1.28; <em>P</em> = 5.2 × 10<sup>−8</sup>). No other genotyped or imputed SNPs approached genome-wide significance.</p><p id="__p31" class="p p-last">Sensitivity analysis was conducted for both the primary and secondary phenotypes by excluding subjects who had a history of diabetes diagnosis. Results were essentially the same as those for the analyses that included patients with diabetes.</p></div><div id="s3b" class="sec sec-last"><h3 id="s3btitle">HLA Serotype Association and Conditional Association Analysis</h3><p id="__p32" class="p p-first-last">Table <a href="/pmc/articles/PMC4747615/table/JIV483TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="JIV483TB2" rid-ob="ob-JIV483TB2" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table2</span></span><span>2</span></a> also presents selected significant results from association testing for the imputed HLA serotypes. Given the significant association from the genotyped and imputed SNP data analysis in the HLA class II region, we examined the association of imputed HLA-DR serotype variants to primary phenotype to determine whether the imputed SNP association results could be attributed to the effect of 1 or more classical HLA haplotypes. Among several HLA-DRB1 serotypes significantly associated with the primary phenotype, HLA-DRB1*04 variants showed the largest effect estimates (eg, HLA-DRB1*0402: OR, 1.36 [<em>P</em> = .002]). We then conducted a conditional association analysis by fitting separate logistic regression models based on the most significant results from the genotyped and imputed SNP GWAS analyses conditioning on imputed HLA-DRB1*04 serotype variant (eg, logit [primary phenotype] = β<sub>1</sub> rs4321864 + β<sub>2</sub> HLA-DRB1*04 + β<sub>3</sub> age-sex group + β<sub>4</sub> PC1+  …  + β<sub>14</sub> PC10; Table <a href="/pmc/articles/PMC4747615/table/JIV483TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="JIV483TB3" rid-ob="ob-JIV483TB3" co-legend-rid=""><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -2.5em;">Table3).</span></span><span>3</span></a>). Results showed that, while the <em>P</em> value for genotyped SNP rs4321864 had increased, it still remained significant even after conditioning on imputed HLA-DRB1*04 serotype; the effect estimate was unchanged (OR, 1.14; 95% confidence interval, 1.07–1.20). Moreover, the <em>P</em> value and effect estimate for imputed SNP rs115231074 were virtually unchanged after adjustment for HLA-DRB1*04. These results indicate that the individual SNP associations are not due to LD with HLA class II haplotypes and provide further support for the association between both SNPs and the primary phenotype.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="JIV483TB3"><h3>Table 3.</h3><!--caption a7--><div class="caption"><p id="__p33">Results of Logistic Regression Analysis of Primary Phenotype vs Genotyped and Imputed Single-Nucleotide Polymorphisms (SNP) Conditioned on Imputed HLA-DRB1*04 Serotype</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140318876221792" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">HLA SNP</th><th rowspan="1" colspan="1" align="center">Chromosome</th><th rowspan="1" colspan="1" align="center">Gene</th><th rowspan="1" colspan="1" align="center">Location</th><th rowspan="1" colspan="1" align="center">OR</th><th rowspan="1" colspan="1" align="center"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Genotyped</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">rs4321864</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRA</td><td rowspan="1" colspan="1">5′ end of gene</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1">1.38 × 10<sup>−5</sup></td></tr><tr><td colspan="6" rowspan="1">Imputed</td></tr><tr><td rowspan="1" colspan="1">rs115231074</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic: 3′ downstream of –DRB1</td><td rowspan="1" colspan="1">1.22</td><td rowspan="1" colspan="1">5.45 × 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1">rs35079132</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">HLA-DRB1</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">1.24</td><td rowspan="1" colspan="1">1.72 × 10<sup>−6</sup></td></tr></tbody></table></div><div id="largeobj_idm140318876221792" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4747615/table/JIV483TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tblfn5"><p id="__p34" class="p p-first-last">Analyses were adjusted for imputed serotype HLA-DRB1*04, age, sex, and the first 10 eigenvectors from a principal components analysis.</p></div><div id="tblfn6"><p id="__p35" class="p p-first-last">Abbreviation: OR, odds ratio.</p></div></div></div></div></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">DISCUSSION</h2><p id="__p36" class="p p-first">In the current study, we used a sample of &gt;50 000 unique white subjects to identify 2 imputed SNPs (rs115231074 and rs35079132) located in the HLA class II region of chromosome 6 that achieved genome-wide significance in the primary phenotype GWAS analysis. These results strengthened our finding of genotyped SNP rs4321864, located on chromosome 6 near the 5′ terminus of the HLA-DRA gene, in the primary phenotype association testing. The secondary phenotype analysis also revealed a significant finding for imputed SNP rs115231074.</p><p id="__p37">HLA-DRA encodes the sole α chain for the β chains encoded by HLA-DRB-1 and HLA-DRB-3, HLA-DRB-4, and HLA-DRB-5. Together, the proteins encoded by HLA-DRA and HLA-DRB genes form an antigen binding heterodimer that presents foreign peptides to trigger the immune response. A significant body of evidence supports the possibility that HLA class II haplotypes may influence human susceptibility to <em>S. aureus</em> infection. First, specific HLA haplotypes (HLAII DR14/DQ5) are associated with susceptibility to invasive <em>Streptococcus pyogenes</em> infection in patients [<a href="#JIV483C26" rid="JIV483C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219237">26</a>] and determine the severity of response to bacterial superantigens from both <em>S. pyogenes</em> [<a href="#JIV483C27" rid="JIV483C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219252">27</a>] and <em>S. aureus</em> [<a href="#JIV483C28" rid="JIV483C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219257">28</a>]. Second, <em>S. aureus</em> superantigens, including toxic shock syndrome toxin (TSST-1), bind to the HLA II DR1 molecule [<a href="#JIV483C29" rid="JIV483C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219268">29</a>, <a href="#JIV483C30" rid="JIV483C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219255">30</a>] and are critical in the development of <em>S. aureus</em> bacteremia and endocarditis [<a href="#JIV483C31" rid="JIV483C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550411490">31</a>, <a href="#JIV483C32" rid="JIV483C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219254">32</a>]. Third, nasal carriage of <em>S. aureus</em> is associated with the HLA-DR3 and HLA-DR7 class II serotypes [<a href="#JIV483C33" rid="JIV483C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219253">33</a>]. Finally, polymorphisms in HLA-DRB1 are strongly associated with rheumatoid arthritis, an inflammatory disease characterized by a high risk of <em>S. aureus</em> infection [<a href="#JIV483C34" rid="JIV483C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219263">34</a>–<a href="#JIV483C36" rid="JIV483C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219247">36</a>]. The HLA-DR α chain is a relatively nonpolymorphic gene containing 5 exons and spanning approximately 33–35 kDa. Because of its relative stability, the positions in the HLA-DR β chain (HLA-DRB) appear to play a major role in binding to and presenting different antigens for recognition by T cells. Recognition of staphylococcal toxins and the subsequent immunologic response may be critical elements in determining infection. The intergenic location of the most significant imputed SNPs (rs115231074 and rs35079132) suggests that functional elements (eg, regulatory element) may exist in the region. Such variants might change the quantity of a specific variable chain produced, rather than its sequence. Alternatively these SNPs could be in LD with multiple variants that collectively constitute a functional haplotype. Finally, it is also possible that there is LD with genic variation that was not genotyped or imputed. Further study is therefore warranted.</p><p id="__p38">Refining our primary end point to include only community-acquired SSTI did not result in more-significant findings. This may be due in part to the emergence of the highly virulent <em>S. aureus</em> USA300 clone [<a href="#JIV483C37" rid="JIV483C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219233">37</a>] during the case ascertainment period. Unfortunately, bacterial genotyping was not available in this study.</p><p id="__p39">Both our primary and secondary phenotype sample sizes were adequate to power GWAS testing that could detect relatively small risk-ratio estimates for common variants (MAF, &gt;1%) at genome-wide significance. Because a GWAS relies on LD between common genotyped markers and relatively common causative variants, it generally has inadequate power to detect significant associations with rare causative variants. Much of the genetic component to common infectious disease may be attributable to the cumulative effects of many rare mutations with limited penetrance [<a href="#JIV483C38" rid="JIV483C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219232">38</a>]. While rare mutations could account for much of the genetic component, our imputed SNP results support a consensus that common variants are a part of the genetic component for which there is evidence in other diseases/conditions [<a href="#JIV483C39" rid="JIV483C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219269">39</a>–<a href="#JIV483C41" rid="JIV483C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219267">41</a>].</p><p id="__p40">Previously, we reported a case-control study of 361 white individuals with a diagnosis of <em>S. aureus</em> bacteremia who were matched to 699 controls [<a href="#JIV483C8" rid="JIV483C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219262">8</a>]. That study did not find any genetic associations that reached genome-wide significance (<em>P</em> ≤ 5 × 10<sup>−8</sup>). Ye et al conducted a case-control design in which the outcome included all <em>S. aureus</em> infections (309 cases and 2925 controls) from a cohort of approximately 20 000 individuals of northern European ancestry [<a href="#JIV483C9" rid="JIV483C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219261">9</a>]. That study also failed to identify any SNPs reaching genome-wide significance. More-targeted approaches, including murine sepsis models, have identified candidate genes associated with susceptibility to <em>S. aureus</em> infection in murine chromosomes [<a href="#JIV483C1" rid="JIV483C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219229">1</a>, <a href="#JIV483C2" rid="JIV483C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219260">2</a>, <a href="#JIV483C42" rid="JIV483C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219245">42</a>]. Thus, the current study is the first to identify candidate polymorphisms associated with susceptibility to <em>S. aureus</em> infection at a genome-wide significant level.</p><p id="__p41">Our study had limitations. First, the genotype of the infecting <em>S. aureus</em> isolates is not known. Specific <em>S. aureus</em> clones possess different combinations of virulence genes (<em>fnbpA</em> and <em>fnbpB</em> [<a href="#JIV483C43" rid="JIV483C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219265">43</a>] and <em>sasX</em> [<a href="#JIV483C44" rid="JIV483C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219240">44</a>]), human immune evasion clusters (<em>scn</em> and <em>chp</em>) [<a href="#JIV483C45" rid="JIV483C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219231">45</a>], and enterotoxins that influence their ability to cause and continue infection in humans. In addition, specific nonsynonymous SNPs in key virulence genes have been shown to enhance [<a href="#JIV483C46" rid="JIV483C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219266">46</a>] and reduce [<a href="#JIV483C47" rid="JIV483C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512219246">47</a>] bacterial virulence. Second, we were unable to control for certain environmental factors (eg, nutrition) that may influence how <em>S. aureus</em> interacts with host gene variants. Third, it is possible that some of the <em>S. aureus</em> respiratory isolates may have reflected colonization instead of active infection<em>.</em> However, this possibility would have led to a reduction in the difference between cases and controls. Fourth, our study population was limited to white subjects (see <a href="http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jiv483/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary 2</a> for further discussion of this topic). Thus, our findings cannot be generalized to other racial groups. Finally, we were unable to perform additional subgroup analyses in potentially important populations, such as patients with recurrent <em>S. aureus</em> infections.</p><p id="__p42" class="p p-last">Despite these limitations, our fully powered GWAS identified both genotyped and imputed genetic variants in the HLA class II region that are associated with susceptibility to <em>S. aureus</em> infection. These findings are independent of classical haplotype associations. Future studies using whole-genome sequencing experiments in patients with complicated and uncomplicated <em>S. aureus</em> bacteremia and admixture mapping studies to evaluate susceptibility to <em>S. aureus</em> in African American patients are currently underway. Further knowledge of host genetic response to <em>S. aureus</em> infection will contribute to our understanding and eventually inform our management of this serious, common infection.</p></div><div id="s5" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">Supplementary Data</h2><!--/article/body/sec/--><p id="__p43" class="p p-first"><a href="http://jid.oxfordjournals.org/lookup/suppl/doi:10.1093/infdis/jiv483/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary materials</a> are available at <a href="http://jid.oxfordjournals.org">http://jid.oxfordjournals.org</a>. Consisting of data provided by the author to benefit the reader, the posted materials are not copyedited and are the sole responsibility of the author, so questions or comments should be addressed to the author.</p><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140318842530944"><a href="/pmc/articles/PMC4747615/bin/supp_213_5_816__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>



</div></div><div id="idm140318842853120" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140318842853120title">Notes</h2><div class="sec"><p id="__p44"><strong><em>Acknowledgments.</em></strong> We thank the Research Program on Genes, Environment, and Health research team (Cathy Schaefer and Neil Risch, codirectors) for providing the genotyped single-nucleotide polymorphisms (SNPs), imputed SNPs, imputed human leukocyte antigen serotypes, and survey data, as well as advice during our study activities.</p><p id="__p45"><strong><em>Disclaimer.</em></strong> The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health (NIH).</p><p id="__p46"><strong><em>Financial support.</em></strong> This work was supported by the NIH (grant <span>2R01-AI068804</span> to V. G. F.) and the National Center for Advancing Translational Sciences, NIH (award <span>UL1TR001117</span>).</p><p id="__p47"><strong><em>Potential conflicts of interest.</em></strong> V. G. F. was chair of Merck's V710 scientific advisory committee; received grant support and has grants pending from the NIH, MedImmune, Forest/Cerexa, Pfizer, Merck, Advanced Liquid Logics, Theravance, Novartis, and Cubist; was a paid consultant for Pfizer, Novartis, Galderma, Novadigm, Durata, Debiopharm, Genentech, Achaogen, Affinium, Medicines, Cerexa, Tetraphase, Trius, MedImmune, Bayer, Theravance, Cubist, Basilea, Affinergy, and Contrafect; received royalties from UpToDate; received payment for development of educational presentations from Green Cross, Cubist, Cerexa, Durata, and Theravance; and has a patent pending on bacterial diagnostics. All other authors report no potential conflicts. All authors have submitted the ICMJE Form for Disclosure of Potential Conflicts of Interest. Conflicts that the editors consider relevant to the content of the manuscript have been disclosed</p></div></div><div id="idm140318840879776" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140318840879776title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="JIV483C1">1. <span class="mixed-citation">Ahn SH, Deshmukh H, Johnson N et al. 
<span class="ref-title">Two genes on A/J chromosome 18 are associated with susceptibility to <em>Staphylococcus aureus</em> infection by combined microarray and QTL analyses</span>. <span class="ref-journal">PLoS Pathog</span>
2010; <span class="ref-vol">6</span>:e1001088. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2932726/">PMC free article</a>]</span> [<a href="/pubmed/20824097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Two+genes+on+A/J+chromosome+18+are+associated+with+susceptibility+to+Staphylococcus+aureus+infection+by+combined+microarray+and+QTL+analyses&amp;author=SH+Ahn&amp;author=H+Deshmukh&amp;author=N+Johnson&amp;volume=6&amp;publication_year=2010&amp;pages=e1001088&amp;pmid=20824097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C2">2. <span class="mixed-citation">Johnson NV, Ahn SH, Deshmukh H et al. 
<span class="ref-title">Haplotype association mapping identifies a candidate gene region in mice infected with <em>Staphylococcus aureus</em></span>. <span class="ref-journal">G3 (Bethesda)</span>
2012; <span class="ref-vol">2</span>:693–700. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3362298/">PMC free article</a>]</span> [<a href="/pubmed/22690378" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=G3+(Bethesda)&amp;title=Haplotype+association+mapping+identifies+a+candidate+gene+region+in+mice+infected+with+Staphylococcus+aureus&amp;author=NV+Johnson&amp;author=SH+Ahn&amp;author=H+Deshmukh&amp;volume=2&amp;publication_year=2012&amp;pages=693-700&amp;pmid=22690378&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C3">3. <span class="mixed-citation">Griesbeck-Zilch B, Osman M, Kuhn C et al. 
<span class="ref-title">Analysis of key molecules of the innate immune system in mammary epithelial cells isolated from marker-assisted and conventionally selected cattle</span>. <span class="ref-journal">J Dairy Sci</span>
2009; <span class="ref-vol">92</span>:4621–33. [<a href="/pubmed/19700725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Dairy+Sci&amp;title=Analysis+of+key+molecules+of+the+innate+immune+system+in+mammary+epithelial+cells+isolated+from+marker-assisted+and+conventionally+selected+cattle&amp;author=B+Griesbeck-Zilch&amp;author=M+Osman&amp;author=C+Kuhn&amp;volume=92&amp;publication_year=2009&amp;pages=4621-33&amp;pmid=19700725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C4">4. <span class="mixed-citation">Bonnefont CM, Rainard P, Cunha P et al. 
<span class="ref-title">Genetic susceptibility to S. aureus mastitis in sheep: differential expression of mammary epithelial cells in response to live bacteria or supernatant</span>. <span class="ref-journal">Physiol Genomics</span>
2012; <span class="ref-vol">44</span>:403–16. [<a href="/pubmed/22337903" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Physiol+Genomics&amp;title=Genetic+susceptibility+to+S.+aureus+mastitis+in+sheep:+differential+expression+of+mammary+epithelial+cells+in+response+to+live+bacteria+or+supernatant&amp;author=CM+Bonnefont&amp;author=P+Rainard&amp;author=P+Cunha&amp;volume=44&amp;publication_year=2012&amp;pages=403-16&amp;pmid=22337903&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C5">5. <span class="mixed-citation">Zimakoff J, Rosdahl VT, Petersen W, Scheibel J.
<span class="ref-title">Recurrent staphylococcal furunculosis in families</span>. <span class="ref-journal">Scand J Infect Dis</span>
1988; <span class="ref-vol">20</span>:403–5. [<a href="/pubmed/3194708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand+J+Infect+Dis&amp;title=Recurrent+staphylococcal+furunculosis+in+families&amp;author=J+Zimakoff&amp;author=VT+Rosdahl&amp;author=W+Petersen&amp;author=J+Scheibel&amp;volume=20&amp;publication_year=1988&amp;pages=403-5&amp;pmid=3194708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C6">6. <span class="mixed-citation">Holland SM, DeLeo FR, Elloumi HZ et al. 
<span class="ref-title">STAT3 mutations in the hyper-IgE syndrome</span>. <span class="ref-journal">N Engl J Med</span>
2007; <span class="ref-vol">357</span>:1608–19. [<a href="/pubmed/17881745" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=STAT3+mutations+in+the+hyper-IgE+syndrome&amp;author=SM+Holland&amp;author=FR+DeLeo&amp;author=HZ+Elloumi&amp;volume=357&amp;publication_year=2007&amp;pages=1608-19&amp;pmid=17881745&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C7">7. <span class="mixed-citation">Tanabe F, Kasai H, He L et al. 
<span class="ref-title">Improvement of deficient natural killer activity and delayed bactericidal activity by a thiol proteinase inhibitor, E-64-d, in leukocytes from Chediak-Higashi syndrome patients in vitro</span>. <span class="ref-journal">Int Immunopharmacol</span>
2009; <span class="ref-vol">9</span>:366–70. [<a href="/pubmed/19185618" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+Immunopharmacol&amp;title=Improvement+of+deficient+natural+killer+activity+and+delayed+bactericidal+activity+by+a+thiol+proteinase+inhibitor,+E-64-d,+in+leukocytes+from+Chediak-Higashi+syndrome+patients+in+vitro&amp;author=F+Tanabe&amp;author=H+Kasai&amp;author=L+He&amp;volume=9&amp;publication_year=2009&amp;pages=366-70&amp;pmid=19185618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C8">8. <span class="mixed-citation">Nelson CL, Pelak K, Podgoreanu MV et al. 
<span class="ref-title">A genome-wide association study of variants associated with acquisition of <em>Staphylococcus aureus</em> bacteremia in a healthcare setting</span>. <span class="ref-journal">BMC Infect Dis</span>
2014; <span class="ref-vol">14</span>:83. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3928605/">PMC free article</a>]</span> [<a href="/pubmed/24524581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Infect+Dis&amp;title=A+genome-wide+association+study+of+variants+associated+with+acquisition+of+Staphylococcus+aureus+bacteremia+in+a+healthcare+setting&amp;author=CL+Nelson&amp;author=K+Pelak&amp;author=MV+Podgoreanu&amp;volume=14&amp;publication_year=2014&amp;pages=83&amp;pmid=24524581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C9">9. <span class="mixed-citation">Ye Z, Vasco DA, Carter TC, Brilliant MH, Schrodi SJ, Shukla SK.
<span class="ref-title">Genome wide association study of SNP-, gene-, and pathway-based approaches to identify genes influencing susceptibility to <em>Staphylococcus aureus</em> infections</span>. <span class="ref-journal">Front Genet</span>
2014; <span class="ref-vol">5</span>:125. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4023021/">PMC free article</a>]</span> [<a href="/pubmed/24847357" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Genet&amp;title=Genome+wide+association+study+of+SNP-,+gene-,+and+pathway-based+approaches+to+identify+genes+influencing+susceptibility+to+Staphylococcus+aureus+infections&amp;author=Z+Ye&amp;author=DA+Vasco&amp;author=TC+Carter&amp;author=MH+Brilliant&amp;author=SJ+Schrodi&amp;volume=5&amp;publication_year=2014&amp;pages=125&amp;pmid=24847357&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C10">10. <span class="mixed-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR.
<span class="ref-title">Fast and accurate genotype imputation in genome-wide association studies through pre-phasing</span>. <span class="ref-journal">Nat Genet</span>
2012; <span class="ref-vol">44</span>:955–9. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-9&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C11">11. <span class="mixed-citation">Jia X, Han B, Onengut-Gumuscu S et al. 
<span class="ref-title">Imputing amino acid polymorphisms in human leukocyte antigens</span>. <span class="ref-journal">PLoS One</span>
2013; <span class="ref-vol">8</span>:e64683. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3675122/">PMC free article</a>]</span> [<a href="/pubmed/23762245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Imputing+amino+acid+polymorphisms+in+human+leukocyte+antigens&amp;author=X+Jia&amp;author=B+Han&amp;author=S+Onengut-Gumuscu&amp;volume=8&amp;publication_year=2013&amp;pages=e64683&amp;pmid=23762245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C12">12. <span class="mixed-citation">Ray GT, Suaya JA, Baxter R.
<span class="ref-title">Trends and characteristics of culture-confirmed <em>Staphylococcus aureus</em> infections in a large U.S. integrated health care organization</span>. <span class="ref-journal">J Clin Microbiol</span>
2012; <span class="ref-vol">50</span>:1950–7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3372166/">PMC free article</a>]</span> [<a href="/pubmed/22422853" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Microbiol&amp;title=Trends+and+characteristics+of+culture-confirmed+Staphylococcus+aureus+infections+in+a+large+U.S.+integrated+health+care+organization&amp;author=GT+Ray&amp;author=JA+Suaya&amp;author=R+Baxter&amp;volume=50&amp;publication_year=2012&amp;pages=1950-7&amp;pmid=22422853&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C13">13. <span class="mixed-citation">Shannon KP, French GL.
<span class="ref-title">Validation of the NCCLS proposal to use results only from the first isolate of a species per patient in the calculation of susceptibility frequencies</span>. <span class="ref-journal">J Antimicrob Chemother</span>
2002; <span class="ref-vol">50</span>:965–9. [<a href="/pubmed/12461018" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Antimicrob+Chemother&amp;title=Validation+of+the+NCCLS+proposal+to+use+results+only+from+the+first+isolate+of+a+species+per+patient+in+the+calculation+of+susceptibility+frequencies&amp;author=KP+Shannon&amp;author=GL+French&amp;volume=50&amp;publication_year=2002&amp;pages=965-9&amp;pmid=12461018&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C14">14. <span class="mixed-citation">Liu C, Graber CJ, Karr M et al. 
<span class="ref-title">A population-based study of the incidence and molecular epidemiology of methicillin-resistant <em>Staphylococcus aureus</em> disease in San Francisco, 2004–2005</span>. <span class="ref-journal">Clin Infect Dis</span>
2008; <span class="ref-vol">46</span>:1637–46. [<a href="/pubmed/18433335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Infect+Dis&amp;title=A+population-based+study+of+the+incidence+and+molecular+epidemiology+of+methicillin-resistant+Staphylococcus+aureus+disease+in+San+Francisco,+2004–2005&amp;author=C+Liu&amp;author=CJ+Graber&amp;author=M+Karr&amp;volume=46&amp;publication_year=2008&amp;pages=1637-46&amp;pmid=18433335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C15">15. <span class="mixed-citation">Frazer KA, Ballinger DG, Cox DR et al. 
<span class="ref-title">A second generation human haplotype map of over 3.1 million SNPs</span>. <span class="ref-journal">Nature</span>
2007; <span class="ref-vol">449</span>:851–61. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;author=DG+Ballinger&amp;author=DR+Cox&amp;volume=449&amp;publication_year=2007&amp;pages=851-61&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C16">16. <span class="mixed-citation">Banda Y, Kvale MN, Hoffmann TJ et al. 
<span class="ref-title">Characterizing race/ethnicity and genetic ancestry for 100,000 subjects in the genetic epidemiology research on adult health and aging (GERA) cohort</span>. <span class="ref-journal">Genetics</span>
2015; <span class="ref-vol">200</span>:1285–95. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4574246/">PMC free article</a>]</span> [<a href="/pubmed/26092716" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Characterizing+race/ethnicity+and+genetic+ancestry+for+100,000+subjects+in+the+genetic+epidemiology+research+on+adult+health+and+aging+(GERA)+cohort&amp;author=Y+Banda&amp;author=MN+Kvale&amp;author=TJ+Hoffmann&amp;volume=200&amp;publication_year=2015&amp;pages=1285-95&amp;pmid=26092716&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C17">17. <span class="mixed-citation">Manichaikul A, Mychaleckyj JC, Rich SS, Daly K, Sale M, Chen WM.
<span class="ref-title">Robust relationship inference in genome-wide association studies</span>. <span class="ref-journal">Bioinformatics</span>
2010; <span class="ref-vol">26</span>:2867–73. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3025716/">PMC free article</a>]</span> [<a href="/pubmed/20926424" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Robust+relationship+inference+in+genome-wide+association+studies&amp;author=A+Manichaikul&amp;author=JC+Mychaleckyj&amp;author=SS+Rich&amp;author=K+Daly&amp;author=M+Sale&amp;volume=26&amp;publication_year=2010&amp;pages=2867-73&amp;pmid=20926424&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C18">18. <span class="mixed-citation">Porcu E, Sanna S, Fuchsberger C, Fritsche LG.
<span class="ref-title">Genotype imputation in genome-wide association studies</span>. <span class="ref-journal">Curr Protoc Hum Genet</span>
2013; <span class="ref-vol">78</span>:1.25.1–1.25.14. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Hum+Genet&amp;title=Genotype+imputation+in+genome-wide+association+studies&amp;author=E+Porcu&amp;author=S+Sanna&amp;author=C+Fuchsberger&amp;author=LG+Fritsche&amp;volume=78&amp;publication_year=2013&amp;pages=1.25.1–1.25.14&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C19">19. <span class="mixed-citation">Delaneau O, Marchini J.
<span class="ref-title">Integrating sequence and array data to create an improved 1000 Genomes Project haplotype reference panel</span>. <span class="ref-journal">Nat Commun</span>
2014; <span class="ref-vol">5</span>:3934. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4338501/">PMC free article</a>]</span> [<a href="/pubmed/25653097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Commun&amp;title=Integrating+sequence+and+array+data+to+create+an+improved+1000+Genomes+Project+haplotype+reference+panel&amp;author=O+Delaneau&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2014&amp;pages=3934&amp;pmid=25653097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C20">20. <span class="mixed-citation">O'Connell J, Gurdasani D, Delaneau O et al. 
<span class="ref-title">A general approach for haplotype phasing across the full spectrum of relatedness</span>. <span class="ref-journal">PLoS Genet</span>
2014; <span class="ref-vol">10</span>:e1004234. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3990520/">PMC free article</a>]</span> [<a href="/pubmed/24743097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+general+approach+for+haplotype+phasing+across+the+full+spectrum+of+relatedness&amp;author=J+O'Connell&amp;author=D+Gurdasani&amp;author=O+Delaneau&amp;volume=10&amp;publication_year=2014&amp;pages=e1004234&amp;pmid=24743097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C21">21. <span class="mixed-citation">Howie BN, Donnelly P, Marchini J.
<span class="ref-title">A flexible and accurate genotype imputation method for the next generation of genome-wide association studies</span>. <span class="ref-journal">PLoS Genet</span>
2009; <span class="ref-vol">5</span>:e1000529. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C22">22. <span class="mixed-citation">de Bakker PI, McVean G, Sabeti PC et al. 
<span class="ref-title">A high-resolution HLA and SNP haplotype map for disease association studies in the extended human MHC</span>. <span class="ref-journal">Nat Genet</span>
2006; <span class="ref-vol">38</span>:1166–72. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2670196/">PMC free article</a>]</span> [<a href="/pubmed/16998491" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+high-resolution+HLA+and+SNP+haplotype+map+for+disease+association+studies+in+the+extended+human+MHC&amp;author=PI+de+Bakker&amp;author=G+McVean&amp;author=PC+Sabeti&amp;volume=38&amp;publication_year=2006&amp;pages=1166-72&amp;pmid=16998491&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C23">23. <span class="mixed-citation">Consortium WTCC.
<span class="ref-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <span class="ref-journal">Nature</span>
2007; <span class="ref-vol">447</span>:661–78. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;author=WTCC+Consortium&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C24">24. <span class="mixed-citation">Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D.
<span class="ref-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <span class="ref-journal">Nat Genet</span>
2006; <span class="ref-vol">38</span>:904–9. [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;author=NJ+Patterson&amp;author=RM+Plenge&amp;author=ME+Weinblatt&amp;author=NA+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C25">25. <span class="mixed-citation">Purcell S, Neale B, Todd-Brown K et al. 
<span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am J Hum Genet</span>
2007; <span class="ref-vol">81</span>:559–75. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C26">26. <span class="mixed-citation">Kotb M, Norrby-Teglund A, McGeer A et al. 
<span class="ref-title">An immunogenetic and molecular basis for differences in outcomes of invasive group A streptococcal infections</span>. <span class="ref-journal">Nat Med</span>
2002; <span class="ref-vol">8</span>:1398–404. [<a href="/pubmed/12436116" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=An+immunogenetic+and+molecular+basis+for+differences+in+outcomes+of+invasive+group+A+streptococcal+infections&amp;author=M+Kotb&amp;author=A+Norrby-Teglund&amp;author=A+McGeer&amp;volume=8&amp;publication_year=2002&amp;pages=1398-404&amp;pmid=12436116&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C27">27. <span class="mixed-citation">Nooh MM, El-Gengehi N, Kansal R, David CS, Kotb M.
<span class="ref-title">HLA transgenic mice provide evidence for a direct and dominant role of HLA class II variation in modulating the severity of streptococcal sepsis</span>. <span class="ref-journal">J Immunol</span>
2007; <span class="ref-vol">178</span>:3076–83. [<a href="/pubmed/17312154" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=HLA+transgenic+mice+provide+evidence+for+a+direct+and+dominant+role+of+HLA+class+II+variation+in+modulating+the+severity+of+streptococcal+sepsis&amp;author=MM+Nooh&amp;author=N+El-Gengehi&amp;author=R+Kansal&amp;author=CS+David&amp;author=M+Kotb&amp;volume=178&amp;publication_year=2007&amp;pages=3076-83&amp;pmid=17312154&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C28">28. <span class="mixed-citation">Llewelyn M, Sriskandan S, Peakman M et al. 
<span class="ref-title">HLA class II polymorphisms determine responses to bacterial superantigens</span>. <span class="ref-journal">J Immunol</span>
2004; <span class="ref-vol">172</span>:1719–26. [<a href="/pubmed/14734754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=HLA+class+II+polymorphisms+determine+responses+to+bacterial+superantigens&amp;author=M+Llewelyn&amp;author=S+Sriskandan&amp;author=M+Peakman&amp;volume=172&amp;publication_year=2004&amp;pages=1719-26&amp;pmid=14734754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C29">29. <span class="mixed-citation">Kim J, Urban RG, Strominger JL, Wiley DC.
<span class="ref-title">Toxic shock syndrome toxin-1 complexed with a class II major histocompatibility molecule HLA-DR1</span>. <span class="ref-journal">Science</span>
1994; <span class="ref-vol">266</span>:1870–4. [<a href="/pubmed/7997880" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Toxic+shock+syndrome+toxin-1+complexed+with+a+class+II+major+histocompatibility+molecule+HLA-DR1&amp;author=J+Kim&amp;author=RG+Urban&amp;author=JL+Strominger&amp;author=DC+Wiley&amp;volume=266&amp;publication_year=1994&amp;pages=1870-4&amp;pmid=7997880&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C30">30. <span class="mixed-citation">Lavoie PM, Thibodeau J, Cloutier I, Busch R, Sekaly RP.
<span class="ref-title">Selective binding of bacterial toxins to major histocompatibility complex class II-expressing cells is controlled by invariant chain and HLA-DM</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>
1997; <span class="ref-vol">94</span>:6892–7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC21255/">PMC free article</a>]</span> [<a href="/pubmed/9192662" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Selective+binding+of+bacterial+toxins+to+major+histocompatibility+complex+class+II-expressing+cells+is+controlled+by+invariant+chain+and+HLA-DM&amp;author=PM+Lavoie&amp;author=J+Thibodeau&amp;author=I+Cloutier&amp;author=R+Busch&amp;author=RP+Sekaly&amp;volume=94&amp;publication_year=1997&amp;pages=6892-7&amp;pmid=9192662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C31">31. <span class="mixed-citation">Salgado-Pabon W, Breshears L, Spaulding AR et al. 
<span class="ref-title">Superantigens are critical for <em>Staphylococcus aureus</em> Infective endocarditis, sepsis, and acute kidney injury</span>. <span class="ref-journal">MBio</span>
2013; <span class="ref-vol">4</span>. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3747586/">PMC free article</a>]</span> [<a href="/pubmed/23963178" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=MBio&amp;title=Superantigens+are+critical+for+Staphylococcus+aureus+Infective+endocarditis,+sepsis,+and+acute+kidney+injury&amp;author=W+Salgado-Pabon&amp;author=L+Breshears&amp;author=AR+Spaulding&amp;volume=4&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C32">32. <span class="mixed-citation">Pragman AA, Yarwood JM, Tripp TJ, Schlievert PM.
<span class="ref-title">Characterization of virulence factor regulation by SrrAB, a two-component system in <em>Staphylococcus aureus</em></span>. <span class="ref-journal">J Bacteriol</span>
2004; <span class="ref-vol">186</span>:2430–8. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC412142/">PMC free article</a>]</span> [<a href="/pubmed/15060046" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Bacteriol&amp;title=Characterization+of+virulence+factor+regulation+by+SrrAB,+a+two-component+system+in+Staphylococcus+aureus&amp;author=AA+Pragman&amp;author=JM+Yarwood&amp;author=TJ+Tripp&amp;author=PM+Schlievert&amp;volume=186&amp;publication_year=2004&amp;pages=2430-8&amp;pmid=15060046&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C33">33. <span class="mixed-citation">Kinsman OS, McKenna R, Noble WC.
<span class="ref-title">Association between histocompatability antigens (HLA) and nasal carriage of <em>Staphylococcus aureus</em></span>. <span class="ref-journal">J Med Microbiol</span>
1983; <span class="ref-vol">16</span>:215–20. [<a href="/pubmed/6573514" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Microbiol&amp;title=Association+between+histocompatability+antigens+(HLA)+and+nasal+carriage+of+Staphylococcus+aureus&amp;author=OS+Kinsman&amp;author=R+McKenna&amp;author=WC+Noble&amp;volume=16&amp;publication_year=1983&amp;pages=215-20&amp;pmid=6573514&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C34">34. <span class="mixed-citation">Suarez-Almazor ME, Tao S, Moustarah F, Russell AS, Maksymowych W.
<span class="ref-title">HLA-DR1, DR4, and DRB1 disease related subtypes in rheumatoid arthritis. Association with susceptibility but not severity in a city wide community based study</span>. <span class="ref-journal">J Rheumatol</span>
1995; <span class="ref-vol">22</span>:2027–33. [<a href="/pubmed/8596139" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Rheumatol&amp;title=HLA-DR1,+DR4,+and+DRB1+disease+related+subtypes+in+rheumatoid+arthritis.+Association+with+susceptibility+but+not+severity+in+a+city+wide+community+based+study&amp;author=ME+Suarez-Almazor&amp;author=S+Tao&amp;author=F+Moustarah&amp;author=AS+Russell&amp;author=W+Maksymowych&amp;volume=22&amp;publication_year=1995&amp;pages=2027-33&amp;pmid=8596139&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C35">35. <span class="mixed-citation">Thomson W, Harrison B, Ollier B et al. 
<span class="ref-title">Quantifying the exact role of HLA-DRB1 alleles in susceptibility to inflammatory polyarthritis: results from a large, population-based study</span>. <span class="ref-journal">Arthritis Rheum</span>
1999; <span class="ref-vol">42</span>:757–62. [<a href="/pubmed/10211891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum&amp;title=Quantifying+the+exact+role+of+HLA-DRB1+alleles+in+susceptibility+to+inflammatory+polyarthritis:+results+from+a+large,+population-based+study&amp;author=W+Thomson&amp;author=B+Harrison&amp;author=B+Ollier&amp;volume=42&amp;publication_year=1999&amp;pages=757-62&amp;pmid=10211891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C36">36. <span class="mixed-citation">Rau R, Herborn G, Zueger S, Fenner H.
<span class="ref-title">The effect of HLA-DRB1 genes, rheumatoid factor, and treatment on radiographic disease progression in rheumatoid arthritis over 6 years</span>. <span class="ref-journal">J Rheumatol</span>
2000; <span class="ref-vol">27</span>:2566–75. [<a href="/pubmed/11093435" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Rheumatol&amp;title=The+effect+of+HLA-DRB1+genes,+rheumatoid+factor,+and+treatment+on+radiographic+disease+progression+in+rheumatoid+arthritis+over+6+years&amp;author=R+Rau&amp;author=G+Herborn&amp;author=S+Zueger&amp;author=H+Fenner&amp;volume=27&amp;publication_year=2000&amp;pages=2566-75&amp;pmid=11093435&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C37">37. <span class="mixed-citation">Diep BA, Gill SR, Chang RF et al. 
<span class="ref-title">Complete genome sequence of USA300, an epidemic clone of community-acquired methicillin-resistant <em>Staphylococcus aureus</em></span>. <span class="ref-journal">Lancet</span>
2006; <span class="ref-vol">367</span>:731–9. [<a href="/pubmed/16517273" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Complete+genome+sequence+of+USA300,+an+epidemic+clone+of+community-acquired+methicillin-resistant+Staphylococcus+aureus&amp;author=BA+Diep&amp;author=SR+Gill&amp;author=RF+Chang&amp;volume=367&amp;publication_year=2006&amp;pages=731-9&amp;pmid=16517273&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C38">38. <span class="mixed-citation">Gorlov IP, Gorlova OY, Frazier ML, Spitz MR, Amos CI.
<span class="ref-title">Evolutionary evidence of the effect of rare variants on disease etiology</span>. <span class="ref-journal">Clin Genet</span>
2011; <span class="ref-vol">79</span>:199–206. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3652532/">PMC free article</a>]</span> [<a href="/pubmed/20831747" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Genet&amp;title=Evolutionary+evidence+of+the+effect+of+rare+variants+on+disease+etiology&amp;author=IP+Gorlov&amp;author=OY+Gorlova&amp;author=ML+Frazier&amp;author=MR+Spitz&amp;author=CI+Amos&amp;volume=79&amp;publication_year=2011&amp;pages=199-206&amp;pmid=20831747&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C39">39. <span class="mixed-citation">Imamura M, Maeda S.
<span class="ref-title">Genetics of type 2 diabetes: the GWAS era and future perspectives [Review]</span>. <span class="ref-journal">Endocr J</span>
2011; <span class="ref-vol">58</span>:723–39. [<a href="/pubmed/21778616" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocr+J&amp;title=Genetics+of+type+2+diabetes:+the+GWAS+era+and+future+perspectives+[Review]&amp;author=M+Imamura&amp;author=S+Maeda&amp;volume=58&amp;publication_year=2011&amp;pages=723-39&amp;pmid=21778616&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C40">40. <span class="mixed-citation">Ishak MB, Giri VN.
<span class="ref-title">A systematic review of replication studies of prostate cancer susceptibility genetic variants in high-risk men originally identified from genome-wide association studies</span>. <span class="ref-journal">Cancer Epidemiol Biomarkers Prev</span>
2011; <span class="ref-vol">20</span>:1599–610. [<a href="/pubmed/21715604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=A+systematic+review+of+replication+studies+of+prostate+cancer+susceptibility+genetic+variants+in+high-risk+men+originally+identified+from+genome-wide+association+studies&amp;author=MB+Ishak&amp;author=VN+Giri&amp;volume=20&amp;publication_year=2011&amp;pages=1599-610&amp;pmid=21715604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C41">41. <span class="mixed-citation">Rafiq S, Khan S, Tapper W et al. 
<span class="ref-title">A genome wide meta-analysis study for identification of common variation associated with breast cancer prognosis</span>. <span class="ref-journal">PLoS One</span>
2014; <span class="ref-vol">9</span>:e101488. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4272267/">PMC free article</a>]</span> [<a href="/pubmed/25526632" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=A+genome+wide+meta-analysis+study+for+identification+of+common+variation+associated+with+breast+cancer+prognosis&amp;author=S+Rafiq&amp;author=S+Khan&amp;author=W+Tapper&amp;volume=9&amp;publication_year=2014&amp;pages=e101488&amp;pmid=25526632&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C42">42. <span class="mixed-citation">Yan Q, Sharma-Kuinkel BK, Deshmukh H et al. 
<span class="ref-title">Dusp3 and Psme3 are associated with murine susceptibility to <em>Staphylococcus aureus</em> infection and human sepsis</span>. <span class="ref-journal">PLoS Pathog</span>
2014; <span class="ref-vol">10</span>:e1004149. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4047107/">PMC free article</a>]</span> [<a href="/pubmed/24901344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Pathog&amp;title=Dusp3+and+Psme3+are+associated+with+murine+susceptibility+to+Staphylococcus+aureus+infection+and+human+sepsis&amp;author=Q+Yan&amp;author=BK+Sharma-Kuinkel&amp;author=H+Deshmukh&amp;volume=10&amp;publication_year=2014&amp;pages=e1004149&amp;pmid=24901344&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C43">43. <span class="mixed-citation">Nienaber JJ, Sharma Kuinkel BK, Clarke-Pearson M et al. 
<span class="ref-title">Methicillin-susceptible <em>Staphylococcus aureus</em> endocarditis isolates are associated with clonal complex 30 genotype and a distinct repertoire of enterotoxins and adhesins</span>. <span class="ref-journal">J Infect Dis</span>
2011; <span class="ref-vol">204</span>:704–13. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3156104/">PMC free article</a>]</span> [<a href="/pubmed/21844296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Infect+Dis&amp;title=Methicillin-susceptible+Staphylococcus+aureus+endocarditis+isolates+are+associated+with+clonal+complex+30+genotype+and+a+distinct+repertoire+of+enterotoxins+and+adhesins&amp;author=JJ+Nienaber&amp;author=BK+Sharma+Kuinkel&amp;author=M+Clarke-Pearson&amp;volume=204&amp;publication_year=2011&amp;pages=704-13&amp;pmid=21844296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C44">44. <span class="mixed-citation">Li M, Du X, Villaruz AE et al. 
<span class="ref-title">MRSA epidemic linked to a quickly spreading colonization and virulence determinant</span>. <span class="ref-journal">Nat Med</span>
2012; <span class="ref-vol">18</span>:816–9. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3378817/">PMC free article</a>]</span> [<a href="/pubmed/22522561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Med&amp;title=MRSA+epidemic+linked+to+a+quickly+spreading+colonization+and+virulence+determinant&amp;author=M+Li&amp;author=X+Du&amp;author=AE+Villaruz&amp;volume=18&amp;publication_year=2012&amp;pages=816-9&amp;pmid=22522561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C45">45. <span class="mixed-citation">Spaan AN, Surewaard BG, Nijland R, van Strijp JA.
<span class="ref-title">Neutrophils versus <em>Staphylococcus aureus</em>: a biological tug of war</span>. <span class="ref-journal">Annu Rev Microbiol</span>
2013; <span class="ref-vol">67</span>:629–50. [<a href="/pubmed/23834243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu+Rev+Microbiol&amp;title=Neutrophils+versus+Staphylococcus+aureus:+a+biological+tug+of+war&amp;author=AN+Spaan&amp;author=BG+Surewaard&amp;author=R+Nijland&amp;author=JA+van+Strijp&amp;volume=67&amp;publication_year=2013&amp;pages=629-50&amp;pmid=23834243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C46">46. <span class="mixed-citation">Lower SK, Lamlertthon S, Casillas-Ituarte NN et al. 
<span class="ref-title">Polymorphisms in fibronectin binding protein A of <em>Staphylococcus aureus</em> are associated with infection of cardiovascular devices</span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>
2011; <span class="ref-vol">108</span>:18372–7. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3215016/">PMC free article</a>]</span> [<a href="/pubmed/22025727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Polymorphisms+in+fibronectin+binding+protein+A+of+Staphylococcus+aureus+are+associated+with+infection+of+cardiovascular+devices&amp;author=SK+Lower&amp;author=S+Lamlertthon&amp;author=NN+Casillas-Ituarte&amp;volume=108&amp;publication_year=2011&amp;pages=18372-7&amp;pmid=22025727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="JIV483C47">47. <span class="mixed-citation">DeLeo FR, Kennedy AD, Chen L et al. 
<span class="ref-title">Molecular differentiation of historic phage-type 80/81 and contemporary epidemic <em>Staphylococcus aureus</em></span>. <span class="ref-journal">Proc Natl Acad Sci U S A</span>
2011; <span class="ref-vol">108</span>:18091–6. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3207694/">PMC free article</a>]</span> [<a href="/pubmed/22025717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Molecular+differentiation+of+historic+phage-type+80/81+and+contemporary+epidemic+Staphylococcus+aureus&amp;author=FR+DeLeo&amp;author=AD+Kennedy&amp;author=L+Chen&amp;volume=108&amp;publication_year=2011&amp;pages=18091-6&amp;pmid=22025717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747615&amp;issue-id=264971&amp;journal-id=1410&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140318840833248title">Abstract</a></li><li><a href="#s2title">MATERIALS AND METHODS</a></li><li><a href="#s3title">RESULTS</a></li><li><a href="#s4title">DISCUSSION</a></li><li><a href="#s5title">Supplementary Data</a></li><li><a href="#idm140318842853120title">Notes</a></li><li><a href="#idm140318840879776title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">The Journal of Infectious Diseases</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4747615/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4747615/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4747615/pdf/jiv483.pdf">PDF (317K)</a></li> | <li><a href="#" data-citationid="PMC4747615" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747615%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747615%2F&amp;text=Polymorphisms%20in%20HLA%20Class%20II%20Genes%20Are%20Associated%20With%20Susceptibility%20to%20Staphylococcus%20aureus%20Infection%20in%20a%20White%20Population" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747615%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26450422" data-aiid="4747615" data-aid="4747615" data-iid="264971" data-domainid="1410" data-domain="jid" data-accid="PMC4747615" data-md5="5b39555a901fbf157d50805bd6bf231b"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:09:33-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8BCF28DD58A27100000000036901F8&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747615%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jid&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4747615/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>