<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4943223/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4943223/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="The Lancet. Neurology"><meta name="citation_title" content="Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies"><meta name="citation_authors" content="Neurology Working Group of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, the Stroke Genetics Network (SiGN), and the International Stroke Genetics Consortium (ISGC)"><meta name="citation_date" content="June 2016"><meta name="citation_issue" content="7"><meta name="citation_volume" content="15"><meta name="citation_firstpage" content="695"><meta name="citation_doi" content="10.1016/S1474-4422(16)00102-2"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4943223/?report=abstract"><meta name="citation_pmid" content="27068588"><meta name="DC.Title" content="Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Neurology Working Group of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, the Stroke Genetics Network (SiGN), and the International Stroke Genetics Consortium (ISGC)"><meta name="DC.Date" content="2016 Jun"><meta name="DC.Identifier" content="10.1016/S1474-4422(16)00102-2"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies"><meta property="og:type" content="article"><meta property="og:description" content="Genetic determinants of stroke, the leading neurological cause of death and disability, are poorly understood and have seldom been explored in the general population. Our aim was to identify additional loci for stroke by doing a meta-analysis of genome-wide ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4943223/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4943223/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4943223/pdf/nihms778629.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8900A6DD58893100000000089C0549.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4943223%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4943223</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Lancet Neurol</a></span>. Author manuscript; available in PMC 2017 Jun 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27068588" target="pmc_ext" ref="reftype=publisher&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Lancet Neurol. 2016 Jun; 15(7): 695–707. </span></a></div><span class="fm-vol-iss-date">Published online 2016 Apr 7. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2FS1474-4422(16)00102-2" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/S1474-4422(16)00102-2</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4943223</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS778629</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/27068588">27068588</a></div></div></div></div><h1 class="content-title">Identification of additional risk loci for stroke and small vessel disease: a meta-analysis of genome-wide association studies</h1><div class="half_rhythm"><div class="contrib-group fm-author">Neurology Working Group of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, the Stroke Genetics Network (SiGN), and the International Stroke Genetics Consortium (ISGC)<sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140271646215552_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140271646215552_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140271646215552_ai" style="display:none"><div id="FN1">Correspondence to: Stéphanie Debette, Inserm U1219, University of Bordeaux and University Hospital of Bordeaux, 33076 Bordeaux, France, <a href="mailto:dev@null" data-email="rf.xuaedrob-u@ettebed.einahpets" class="oemail">rf.xuaedrob-u@ettebed.einahpets</a></div><div id="FN2"><sup>*</sup><strong>The Neurology Working Group of the Cohorts for Heart and Aging Research in Genomic Epidemiology (CHARGE) Consortium, the Stroke Genetics Network (SiGN)</strong><sup><sup>*</sup></sup>
<strong>and the International Stroke Genetics Consortium (ISGC)</strong><p id="__p2" class="p p-last">Ganesh Chauhan<sup><sup>†</sup></sup>, Corey R Arnold<sup><sup>†</sup></sup>, Audrey Y Chu<sup><sup>†</sup></sup>, Myriam Fornage<sup><sup>†</sup></sup>, Azadeh Reyahi<sup><sup>†</sup></sup>, Joshua C Bis<sup><sup>†</sup></sup>, Aki S Havulinna<sup><sup>†</sup></sup>, Muralidharan Sargurupremraj, Albert Vernon Smith, Hieab H H Adams, Seung Hoan Choi, Sara L Pulit, Stella Trompet, Melissa E Garcia, Ani Manichaikul, Alexander Teumer, Stefan Gustafsson, Traci M Bartz, Céline Bellenguez, Jean Sebastien Vidal, Xueqiu Jian, Olafur Kjartansson, Kerri L Wiggins, Claudia L Satizabal, Flora Xue, Samuli Ripatti, Yongmei Liu, Joris Deelen, Marcel den Hoed, Steve Bevan, Jemma C Hopewell, Rainer Malik, Susan R Heckbert, Kenneth Rice, Nicholas L Smith, Christopher Levi, Pankaj Sharma, Cathie LM Sudlow, Ali Moussavi Nik, John W Cole, Reinhold Schmidt, James Meschia, Vincent Thijs, Arne Lindgren, Olle Melander, Raji P Grewal, Ralph L Sacco, Tatjana Rundek, Peter M Rothwell, Donna K Arnett, Christina Jern, Julie A Johnson, Oscar R Benavente, Sylvia Wassertheil-Smoller, Jin-Moo Lee, Quenna Wong, Hugo J Aparicio, Stefan T Engelter, Manja Kloss, Didier Leys, Alessandro Pezzini, Julie E Buring, Paul M Ridker, Claudine Berr, Jean-François Dartigues, Anders Hamsten, Patrik K Magnusson, Matthew Traylor, Nancy L Pedersen, Lars Lannfelt, Lars Lind, Cecilia M Lindgren, Andrew P Morris, Jordi Jimenez-Conde, Joan Montaner, Farid Radmanesh, Agnieszka Slowik, Daniel Woo, Albert Hofman, Peter J Koudstaal, Marileen L P Portegies, André G Uitterlinden, Anton J M de Craen, Ian Ford, J Wouter Jukema, David J Stott, Norrina B Allen, Michele M Sale, Andrew D Johnson, David A Bennett, Philip L De Jager, Charles C White, Hans Jörgen Grabe, Marcello Ricardo Paulista Markus, Ulf Schminke, Giorgio B Boncoraglio, Robert Clarke, Yoichiro Kamatani, and Jean Dallongeville for the Cervical Artery Dissection and Ischemic Stroke Patients (CADISP) study; Oscar L Lopez, Jerome I Rotter, Michael A Nalls, Rebecca F Gottesman, Michael E Griswold, David S Knopman, B Gwen Windham, Alexa Beiser, Hugh S Markus, Erkki Vartiainen, Curtis R French, and Martin Dichgans for the METASTROKE consortium; and Tomi Pastinen, Mark Lathrop, Vilmundur Gudnason, Tobias Kurth, Bruce M Psaty, Tamara B Harris, Stephen S Rich, Anita L deStefano, Carsten Oliver Schmidt, Bradford B Worrall, Jonathan Rosand, Veikko Salomaa, Thomas H Mosley, Erik Ingelsson, Cornelia M van Duijn, Christophe Tzourio, Kathryn M Rexrode, Ordan J Lehmann<sup><sup>‡</sup></sup>, Lenore J Launer<sup><sup>‡</sup></sup>, M Arfan Ikram<sup><sup>‡</sup></sup>, Peter Carlsson<sup><sup>‡</sup></sup>, Daniel I Chasman<sup><sup>‡</sup></sup>, Sarah J Childs<sup><sup>‡</sup></sup>, William T Longstreth Junior<sup><sup>‡</sup></sup>, Sudha Seshadri<sup><sup>‡</sup></sup>, and Stéphanie Debette<sup><sup>‡</sup></sup>.</p></div><div id="FN3"><sup>*</sup>See <a href="#SD1" rid="SD1" class=" supplementary-material">appendix</a> for more details of the study organisation.</div><div id="FN4"><sup>†</sup>Contributed equally.</div><div id="FN5"><sup>‡</sup>Co-directed this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140271646215552_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=27068588" target="pmc_ext" ref="reftype=publisher&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Lancet Neurol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4943223/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC4943223/bin/NIHMS778629-supplement-1.docx" data-ga-action="click_feat_suppl">NIHMS778629-supplement-1.docx</a><span style="color:gray"> (2.6M)</span></div><div class="small guid">GUID:&nbsp;CB085E85-131C-447D-9350-1A3136ED0B73</div></div></dd></dl></div><div id="idm140271718249888" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271718249888title">Summary</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Background</h3><p id="P2" class="p p-first-last">Genetic determinants of stroke, the leading neurological cause of death and disability, are poorly understood and have seldom been explored in the general population. Our aim was to identify additional loci for stroke by doing a meta-analysis of genome-wide association studies.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P3" class="p p-first-last">For the discovery sample, we did a genome-wide analysis of common genetic variants associated with incident stroke risk in 18 population-based cohorts comprising 84 961 participants, of whom 4348 had stroke. Stroke diagnosis was ascertained and validated by the study investigators. Mean age at stroke ranged from 45·8 years to 76·4 years, and data collection in the studies took place between 1948 and 2013. We did validation analyses for variants yielding a significant association (at p&lt;5 × 10<sup>−6</sup>) with all-stroke, ischaemic stroke, cardioembolic ischaemic stroke, or non-cardioembolic ischaemic stroke in the largest available cross-sectional studies (70 804 participants, of whom 19 816 had stroke). Summary-level results of discovery and follow-up stages were combined using inverse-variance weighted fixed-effects meta-analysis, and in-silico lookups were done in stroke subtypes. For genome-wide significant findings (at p&lt;5 × 10<sup>−8</sup>), we explored associations with additional cerebrovascular phenotypes and did functional experiments using conditional (inducible) deletion of the probable causal gene in mice. We also studied the expression of orthologs of this probable causal gene and its effects on cerebral vasculature in zebrafish mutants.</p></div><div id="S3" class="sec"><h3 id="S3title">Findings</h3><p id="P4" class="p p-first-last">We replicated seven of eight known loci associated with risk for ischaemic stroke, and identified a novel locus at chromosome 6p25 (rs12204590, near <em>FOXF2</em>) associated with risk of all-stroke (odds ratio [OR] 1·08, 95% CI 1·05–1·12, p=1·48 × 10<sup>−8</sup>; minor allele frequency 21%). The rs12204590 stroke risk allele was also associated with increased MRI-defined burden of white matter hyperintensity—a marker of cerebral small vessel disease—in stroke-free adults (n=21 079; p=0·0025). Consistently, young patients (aged 2–32 years) with segmental deletions of <em>FOXF2</em> showed an extensive burden of white matter hyperintensity. Deletion of <em>Foxf2</em> in adult mice resulted in cerebral infarction, reactive gliosis, and microhaemorrhage. The orthologs of <em>FOXF2</em> in zebrafish (<em>foxf2b</em> and <em>foxf2a</em>) are expressed in brain pericytes and mutant <em>foxf2b</em><sup>−/−</sup> cerebral vessels show decreased smooth muscle cell and pericyte coverage.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Interpretation</h3><p id="P5" class="p p-first-last">We identified common variants near <em>FOXF2</em> that are associated with increased stroke susceptibility. Epidemiological and experimental data suggest that <em>FOXF2</em> mediates this association, potentially via differentiation defects of cerebral vascular mural cells. Further expression studies in appropriate human tissues, and further functional experiments with long follow-up periods are needed to fully understand the underlying mechanisms.</p></div></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Introduction</h2><p id="P6" class="p p-first">Stroke is the leading neurological cause of death and disability worldwide.<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875859">1</a></sup> A substantial proportion of stroke risk remains unexplained, and contribution of genetic factors is supported by discoveries of common genetic variation associated with stroke risk, identified through large, collaborative, genome-wide association studies (GWAS).<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875862">2</a></sup> These studies have estimated the proportion of phenotype variance explained by the genome-wide genotypes to range between 16% and 40% for ischaemic stroke, and between 34% and 73% for intracerebral haemorrhage.<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875871">3</a></sup> Most associations so far have been specific to particular ischaemic or haemorrhagic stroke types, although a few risk loci for overall ischaemic stroke have also been reported.<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875864">2</a></sup> Overall, the search for stroke loci has been less successful than for other complex phenotypes.<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875873">4</a></sup> Potential explanations include heterogeneity of stroke and limited ability to detect genetic variants increasing both stroke risk and severity because of the cross-sectional design of most studies, with hospital-based case ascertainment and non-inclusion of severe strokes with early mortality. Population-based cohort studies, with blood samples drawn at recruitment and prospective incident stroke ascertainment offer the advantage of including participants with severe strokes leading to early death.</p><p id="P7" class="p">We did a genome-wide analysis for common genetic variants associated with an increased risk of incident stroke in population-based cohort studies and validated these results with analysis of association between the identified variants and stroke in the largest available cross-sectional studies. Detailed functional exploration of novel genome-wide significant association was done in zebrafish and mice.</p><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140271650506464title">Research in context</h3></div><div class="sec" id="S20"><h3 id="S20title">Evidence before this study</h3><p id="P35" class="p p-first-last">We searched PubMed with the search terms “stroke”, “cerebral small vessel disease”, “genetics”, “GWAS”, ”genomics”, and the GWAS catalogue for reports before June, 2015, with no language restrictions; we only included peer-reviewed reports in English. Most of the seven studies found only associations with cardioembolic or large-vessel ischaemic stroke, and no robust genetic association has been reported for other subtypes, especially the very common but poorly understood small-vessel ischaemic stroke. Genetic associations with overall stroke were also scarcely reported, with only a few genetic studies thoroughly investigating incident stroke in a population-based longitudinal setting. Although genome-wide association studies (GWAS) have successfully identified numerous genetic associations with complex diseases, including stroke, biological mechanisms underlying these associations are unknown for most of the variants, precluding clinical applications beyond risk prediction.</p></div><div class="sec" id="S21"><h3 id="S21title">Added value of this study</h3><p id="P37" class="p p-first-last">First, discovery analyses were done in 4348 cases and 80 613 controls and findings were validated in an additional 19 816 cases and 50 988 controls. We identified a novel risk locus for stroke that appears to be mediated by small vessel disease. Although small vessel disease is one of the major subtypes of stroke, GWAS have so far not discovered risk loci for small-vessel ischaemic stroke (except for an association with the <em>PRKCH</em> locus identified in a study in Japanese participants, which was not found in European populations). Second, we provide preliminary experimental evidence from zebrafish and mouse models that the recorded statistical association reflects an effect of the nearby transcription factor <em>FOXF2</em> (a gene predominantly expressed in fetal tissue) on the development of cerebral vasculature. Conditional deletion of <em>Foxf2</em> in adult mice led to cerebral infarction, reactive gliosis, and microhaemorrhage. In zebrafish, <em>foxf2b</em><sup>−/−</sup> mutants showed decreased smooth-muscle cell and pericyte coverage. Third, we show that patients with a rare monogenic ophthalmological condition due to segmental deletions encompassing <em>FOXF2</em> also exhibit features of cerebral small vessel disease, providing an example of how monogenic conditions can inform the mechanisms of complex diseases.</p></div><div class="sec" id="S22"><h3 id="S22title">Implications of all the available evidence</h3><p id="P39" class="p p-first-last">The present findings provide insight into the genetic underpinnings of stroke, especially of the small vessel subtype, with evidence from multiple approaches for a pivotal role of <em>FOXF2</em>, a neural crest expressed transcription factor involved in cerebral vessel development. Cerebral small vessel disease is a major, but poorly understood, cause of stroke in all ethnic groups, and subclinical small vessel disease (which was also associated with the stroke risk variants near <em>FOXF2</em> in our study) has been associated with progressive functional and cognitive decline, and increased risk of dementia. At present, no mechanism-based treatment is available for small vessel disease, other than management of risk factors. Our findings indicate a possible novel mechanism of stroke and small vessel disease. Further research is warranted to explore whether these findings can be applied to clinical practice.</p></div></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study population for discovery analysis</h3><p id="P8" class="p p-first">The GWAS discovery sample comprised 84 961 participants of European origin from 18 community-based prospective cohort studies participating in the Cohorts of Heart and Aging Research in Genomic Epidemiology (CHARGE) consortium. All participants did not have stroke at baseline and 4348 developed incident stroke during an average of 10 years (SD 3·6) of follow-up (<a href="/pmc/articles/PMC4943223/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 8–13</a>). Some, but not all, of the cohorts included in our analysis (representing &lt;1544 incident stroke cases) have been included in published HapMap-based stroke GWAS.<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875860">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875884">6</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p15">Population characteristics</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271646414880" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Country</th><th rowspan="1" colspan="1" valign="bottom" align="right">Total (n)</th><th rowspan="1" colspan="1" valign="bottom" align="right">All stroke (n)</th><th rowspan="1" colspan="1" valign="bottom" align="right">Ischaemic stroke (n)</th><th rowspan="1" colspan="1" valign="bottom" align="right">Cardioembolic-ischaemic stroke (n)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Non-cardioembolic ischaemic stroke (n)<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="right">Women (%)<sup>†</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Age at stroke (years)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Age of controls at DNA draw (years)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Follow-up (years)</th></tr></thead><tbody><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><strong>Discovery sample (longitudinal population-based cohort studies)</strong>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">AGES</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">2996</td><td rowspan="1" colspan="1" valign="top" align="right">114</td><td rowspan="1" colspan="1" valign="top" align="right">99</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">58%</td><td rowspan="1" colspan="1" valign="top" align="left">79·9 (5·5)</td><td rowspan="1" colspan="1" valign="top" align="left">76·4 (5·5)</td><td rowspan="1" colspan="1" valign="top" align="center">3·6 (1·1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ARIC</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">8939</td><td rowspan="1" colspan="1" valign="top" align="right">473</td><td rowspan="1" colspan="1" valign="top" align="right">416</td><td rowspan="1" colspan="1" valign="top" align="right">108</td><td rowspan="1" colspan="1" valign="top" align="center">305</td><td rowspan="1" colspan="1" valign="top" align="right">53%</td><td rowspan="1" colspan="1" valign="top" align="left">69·3 (7·4)</td><td rowspan="1" colspan="1" valign="top" align="left">54·2 (5·7)</td><td rowspan="1" colspan="1" valign="top" align="center">19·1 (4·6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CHS</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">3268</td><td rowspan="1" colspan="1" valign="top" align="right">563</td><td rowspan="1" colspan="1" valign="top" align="right">447</td><td rowspan="1" colspan="1" valign="top" align="right">139</td><td rowspan="1" colspan="1" valign="top" align="center">308</td><td rowspan="1" colspan="1" valign="top" align="right">61%</td><td rowspan="1" colspan="1" valign="top" align="left">82·7 (6·2)</td><td rowspan="1" colspan="1" valign="top" align="left">72·3 (5·4)</td><td rowspan="1" colspan="1" valign="top" align="center">13·5 (6·3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FHS</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">4369</td><td rowspan="1" colspan="1" valign="top" align="right">235</td><td rowspan="1" colspan="1" valign="top" align="right">198</td><td rowspan="1" colspan="1" valign="top" align="right">57</td><td rowspan="1" colspan="1" valign="top" align="center">127</td><td rowspan="1" colspan="1" valign="top" align="right">55%</td><td rowspan="1" colspan="1" valign="top" align="left">73·4 (11·7)</td><td rowspan="1" colspan="1" valign="top" align="left">65·5 (12·7)</td><td rowspan="1" colspan="1" valign="top" align="center">8·0 (3·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FINRISK CoreExome</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="right">5202</td><td rowspan="1" colspan="1" valign="top" align="right">94</td><td rowspan="1" colspan="1" valign="top" align="right">60</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">55%</td><td rowspan="1" colspan="1" valign="top" align="left">69·9 (9·4)</td><td rowspan="1" colspan="1" valign="top" align="left">45·8 (12·8)</td><td rowspan="1" colspan="1" valign="top" align="center">13·3 (2·7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FINRISK Corogene</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="right">1887</td><td rowspan="1" colspan="1" valign="top" align="right">60</td><td rowspan="1" colspan="1" valign="top" align="right">46</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">49%</td><td rowspan="1" colspan="1" valign="top" align="left">73·7 (9·5)</td><td rowspan="1" colspan="1" valign="top" align="left">55·2 (12·2)</td><td rowspan="1" colspan="1" valign="top" align="center">9·0 (4·0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">FINRISK Predict CVD</td><td rowspan="1" colspan="1" valign="top" align="left">Finland</td><td rowspan="1" colspan="1" valign="top" align="right">1309</td><td rowspan="1" colspan="1" valign="top" align="right">352</td><td rowspan="1" colspan="1" valign="top" align="right">294</td><td rowspan="1" colspan="1" valign="top" align="right">NA.</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">53%</td><td rowspan="1" colspan="1" valign="top" align="left">66·5 (9·8)</td><td rowspan="1" colspan="1" valign="top" align="left">46·5 (13·3)</td><td rowspan="1" colspan="1" valign="top" align="center">10·0 (5·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Health ABC</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">1661</td><td rowspan="1" colspan="1" valign="top" align="right">124</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">47%</td><td rowspan="1" colspan="1" valign="top" align="left">80·1 (4·2)</td><td rowspan="1" colspan="1" valign="top" align="left">73·8 (2·8)</td><td rowspan="1" colspan="1" valign="top" align="center">9·3 (2·9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">MESA</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">2364</td><td rowspan="1" colspan="1" valign="top" align="right">49</td><td rowspan="1" colspan="1" valign="top" align="right">43</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">35</td><td rowspan="1" colspan="1" valign="top" align="right">52%</td><td rowspan="1" colspan="1" valign="top" align="left">75·1 (8·9)</td><td rowspan="1" colspan="1" valign="top" align="left">62·7 (10·2)</td><td rowspan="1" colspan="1" valign="top" align="center">7·2 (1·4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">PROSPER</td><td rowspan="1" colspan="1" valign="top" align="left">Netherlands</td><td rowspan="1" colspan="1" valign="top" align="right">4658</td><td rowspan="1" colspan="1" valign="top" align="right">193</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">53%</td><td rowspan="1" colspan="1" valign="top" align="left">77·8 (3·6)</td><td rowspan="1" colspan="1" valign="top" align="left">75·2 (3·3)</td><td rowspan="1" colspan="1" valign="top" align="center">3·1 (0·7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rotterdam Study I</td><td rowspan="1" colspan="1" valign="top" align="left">Netherlands</td><td rowspan="1" colspan="1" valign="top" align="right">6066</td><td rowspan="1" colspan="1" valign="top" align="right">821</td><td rowspan="1" colspan="1" valign="top" align="right">448</td><td rowspan="1" colspan="1" valign="top" align="right">95</td><td rowspan="1" colspan="1" valign="top" align="center">353</td><td rowspan="1" colspan="1" valign="top" align="right">60%</td><td rowspan="1" colspan="1" valign="top" align="left">80·6 (8·1)</td><td rowspan="1" colspan="1" valign="top" align="left">69·2 (9·0)</td><td rowspan="1" colspan="1" valign="top" align="center">13·0 (6·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rotterdam Study II</td><td rowspan="1" colspan="1" valign="top" align="left">Netherlands</td><td rowspan="1" colspan="1" valign="top" align="right">2080</td><td rowspan="1" colspan="1" valign="top" align="right">125</td><td rowspan="1" colspan="1" valign="top" align="right">88</td><td rowspan="1" colspan="1" valign="top" align="right">17</td><td rowspan="1" colspan="1" valign="top" align="center">71</td><td rowspan="1" colspan="1" valign="top" align="right">54%</td><td rowspan="1" colspan="1" valign="top" align="left">75·9 (9·2)</td><td rowspan="1" colspan="1" valign="top" align="left">64·6 (7·9)</td><td rowspan="1" colspan="1" valign="top" align="center">9·7 (2·5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SHIP</td><td rowspan="1" colspan="1" valign="top" align="left">Germany</td><td rowspan="1" colspan="1" valign="top" align="right">3112</td><td rowspan="1" colspan="1" valign="top" align="right">75</td><td rowspan="1" colspan="1" valign="top" align="right">37</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">52%</td><td rowspan="1" colspan="1" valign="top" align="left">71·8 (10·6)</td><td rowspan="1" colspan="1" valign="top" align="left">48·7 (15·2)</td><td rowspan="1" colspan="1" valign="top" align="center">12·1 (2·5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">TWINGENE</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="right">6702</td><td rowspan="1" colspan="1" valign="top" align="right">116</td><td rowspan="1" colspan="1" valign="top" align="right">95</td><td rowspan="1" colspan="1" valign="top" align="right">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="right">52%</td><td rowspan="1" colspan="1" valign="top" align="left">75·6 (8·8)</td><td rowspan="1" colspan="1" valign="top" align="left">64·9 (8·1)</td><td rowspan="1" colspan="1" valign="top" align="center">3·2 (1·0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ULSAM</td><td rowspan="1" colspan="1" valign="top" align="left">Sweden</td><td rowspan="1" colspan="1" valign="top" align="right">1139</td><td rowspan="1" colspan="1" valign="top" align="right">216</td><td rowspan="1" colspan="1" valign="top" align="right">171</td><td rowspan="1" colspan="1" valign="top" align="right">56</td><td rowspan="1" colspan="1" valign="top" align="center">115</td><td rowspan="1" colspan="1" valign="top" align="right">0%</td><td rowspan="1" colspan="1" valign="top" align="left">79·9 (4·8)</td><td rowspan="1" colspan="1" valign="top" align="left">71·0 (0·6)</td><td rowspan="1" colspan="1" valign="top" align="center">12·8 (5·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">WGHS</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">23 294</td><td rowspan="1" colspan="1" valign="top" align="right">499</td><td rowspan="1" colspan="1" valign="top" align="right">402</td><td rowspan="1" colspan="1" valign="top" align="right">82</td><td rowspan="1" colspan="1" valign="top" align="center">320</td><td rowspan="1" colspan="1" valign="top" align="right">100%</td><td rowspan="1" colspan="1" valign="top" align="left">69·6 (9·3)</td><td rowspan="1" colspan="1" valign="top" align="left">54·7 (7·1)</td><td rowspan="1" colspan="1" valign="top" align="center">16·0 (3·2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3C Study Dijon</td><td rowspan="1" colspan="1" valign="top" align="left">France</td><td rowspan="1" colspan="1" valign="top" align="right">3762</td><td rowspan="1" colspan="1" valign="top" align="right">157</td><td rowspan="1" colspan="1" valign="top" align="right">125</td><td rowspan="1" colspan="1" valign="top" align="right">33</td><td rowspan="1" colspan="1" valign="top" align="center">92</td><td rowspan="1" colspan="1" valign="top" align="right">62%</td><td rowspan="1" colspan="1" valign="top" align="left">81·5 (6·1)</td><td rowspan="1" colspan="1" valign="top" align="left">72·4 (5·6)</td><td rowspan="1" colspan="1" valign="top" align="center">8·7 (3·1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">3C Study Bordeaux and Montpellier</td><td rowspan="1" colspan="1" valign="top" align="left">France</td><td rowspan="1" colspan="1" valign="top" align="right">2153</td><td rowspan="1" colspan="1" valign="top" align="right">82</td><td rowspan="1" colspan="1" valign="top" align="right">59</td><td rowspan="1" colspan="1" valign="top" align="right">15</td><td rowspan="1" colspan="1" valign="top" align="center">44</td><td rowspan="1" colspan="1" valign="top" align="right">60%</td><td rowspan="1" colspan="1" valign="top" align="left">81·7 (5·2)</td><td rowspan="1" colspan="1" valign="top" align="left">73·9 (5·1)</td><td rowspan="1" colspan="1" valign="top" align="center">7·8 (2·5)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="right">84 961</td><td rowspan="1" colspan="1" valign="top" align="right">4348</td><td rowspan="1" colspan="1" valign="top" align="right">3028</td><td rowspan="1" colspan="1" valign="top" align="right">602</td><td rowspan="1" colspan="1" valign="top" align="center">1770</td><td rowspan="1" colspan="1" valign="top" align="right">67%</td><td rowspan="1" colspan="1" valign="top" align="left">75·8 (8·0)</td><td rowspan="1" colspan="1" valign="top" align="left">63·7 (8·4)</td><td rowspan="1" colspan="1" valign="top" align="center">10·0 (3·6)</td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="11" rowspan="1" valign="bottom" align="left"><strong>Validation sample (cross-sectional case-control studies)</strong><sup>*</sup>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">SiGN</td><td rowspan="1" colspan="1" valign="top" align="left">USA and Europe</td><td rowspan="1" colspan="1" valign="top" align="right">49 324</td><td rowspan="1" colspan="1" valign="top" align="right">16 851</td><td rowspan="1" colspan="1" valign="top" align="right">16 851</td><td rowspan="1" colspan="1" valign="top" align="right">3427</td><td rowspan="1" colspan="1" valign="top" align="center">2346/3150<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="right">46%</td><td rowspan="1" colspan="1" valign="top" align="left">66·5 (14·8)</td><td rowspan="1" colspan="1" valign="top" align="left">NA<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">METASTROKE</td><td rowspan="1" colspan="1" valign="top" align="left">USA and Europe</td><td rowspan="1" colspan="1" valign="top" align="right">9654</td><td rowspan="1" colspan="1" valign="top" align="right">1729</td><td rowspan="1" colspan="1" valign="top" align="right">1729</td><td rowspan="1" colspan="1" valign="top" align="right">276</td><td rowspan="1" colspan="1" valign="top" align="center">206/159<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="right">36%</td><td rowspan="1" colspan="1" valign="top" align="left">67·0 (10·1)</td><td rowspan="1" colspan="1" valign="top" align="left">60·6 (11·9)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HVH1</td><td rowspan="1" colspan="1" valign="top" align="left">USA</td><td rowspan="1" colspan="1" valign="top" align="right">2012</td><td rowspan="1" colspan="1" valign="top" align="right">681</td><td rowspan="1" colspan="1" valign="top" align="right">577</td><td rowspan="1" colspan="1" valign="top" align="right">92</td><td rowspan="1" colspan="1" valign="top" align="center">62/175<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="right">57%</td><td rowspan="1" colspan="1" valign="top" align="left">68·8 (8·9)</td><td rowspan="1" colspan="1" valign="top" align="left">66·7 (9·1)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">CADISP</td><td rowspan="1" colspan="1" valign="top" align="left">Europe</td><td rowspan="1" colspan="1" valign="top" align="right">9814</td><td rowspan="1" colspan="1" valign="top" align="right">555</td><td rowspan="1" colspan="1" valign="top" align="right">555</td><td rowspan="1" colspan="1" valign="top" align="right">211</td><td rowspan="1" colspan="1" valign="top" align="center">67/31<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="right">61%</td><td rowspan="1" colspan="1" valign="top" align="left">43·7 (9·9)</td><td rowspan="1" colspan="1" valign="top" align="left">NA<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Total</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="right">70 804</td><td rowspan="1" colspan="1" valign="top" align="right">19 816</td><td rowspan="1" colspan="1" valign="top" align="right">19 712</td><td rowspan="1" colspan="1" valign="top" align="right">4006</td><td rowspan="1" colspan="1" valign="top" align="center">2681/3515<sup>‡</sup></td><td rowspan="1" colspan="1" valign="top" align="right">47%</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="left">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr></tbody></table></div><div id="largeobj_idm140271646414880" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4943223/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="__p16" class="p p-first-last">Data are mean (SD) or n (%). NA=not available. AGES=Age, Gene/Environment Susceptibility-Reykjavik Study. ARIC=Atherosclerosis Risk in Communities Study. CHS=Cardiovascular Health Study. FHS=Framingham Heart Study. ABC=Health, Aging, and Body Composition. MESA=Multi-Ethnic Study of Atherosclerosis. PROSPER=Prospective Study of Pravastatin in the Elderly at Risk. SHIP=Study of Health In Pomerania. ULSAM=Uppsala Longitudinal Study of Adult Men. WGHS=Women’s Genome Health Study. SiGN=Stroke Genetics Network. HVH1= Heart and Vascular Health. CADISP=Cervical Artery Dissections and Ischemic Stroke Patients.</p></div><div id="TFN2"><sup>*</sup>More detailed descriptions of composition of replication studies are shown in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix</a>.</div><div id="TFN3"><sup>†</sup>Percentage of women calculated from the total (n) column.</div><div id="TFN4"><sup>‡</sup>Samples are from large-vessel ischaemic stroke or small-vessel ischaemic stroke.</div><div id="TFN5"><sup>§</sup>Mean age of controls in SiGN and CADISP study is not available because they were obtained from anonymous genotype databases.</div></div></div><p id="P9" class="p p-last">This study was approved by the ethics committees of the participating studies and written informed consent was obtained from all study participants in the original cohort studies that allowed for data use in subsequent studies.</p></div><div id="S8" class="sec"><h3 id="S8title">Stroke definition and classification of subtypes</h3><p id="P10" class="p p-first-last">Stroke was defined as a focal neurological deficit of presumed vascular origin with sudden onset and lasting for at least 24 hours, or until death if the participant died less than 24 hours after onset of symptoms. Stroke diagnosis and classification was validated by an expert committee in participating studies (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 13–19</a>). Strokes were classified as ischaemic stroke (n=3028), intracerebral haemorrhage (n=277), or unknown type based on clinical and imaging criteria; for cohorts in which ischaemic stroke subtypes were available (<a href="/pmc/articles/PMC4943223/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>), ischaemic stroke was subdivided into cardioembolic (n=602) and non-cardioembolic (n=1770) subtypes. Numbers of events were too small to analyse large-vessel ischaemic stroke (n=117) and small-vessel ischaemic stroke (n=87) separately in this discovery dataset. Subarachnoid haemorrhage was not analysed because of its distinct mechanisms and very small number of events. Detailed definitions of stroke types and subtypes are given in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix</a>.</p></div><div id="S9" class="sec"><h3 id="S9title">Genotyping and imputation</h3><p id="P11" class="p p-first-last">Genotyping platforms and quality control filters are described in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix (pp 42–43)</a>. All but one study used imputed genotypes based on the 1000GpIv3 “All” reference panel (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 44–45</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Genome-wide association analyses</h3><p id="P12" class="p p-first-last">Using genome-wide multivariable Cox regression, we tested associations of genetic variants with incident stroke (all stroke, ischaemic stroke, cardioembolic ischaemic stroke, non-cardioembolic ischaemic stroke, and in secondary analyses intracerebral haemorrhage) under an additive genetic model, adjusting for sex and age and, when relevant, for principal components of population stratification, study site, or familial structure (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 19, 20, 46</a>). Meta-analysis of study-specific association statistics was done by GC and AYC at two sites (University of Bordeaux, Bordeaux, France, and Harvard Medical School, Boston, MA, USA) with inverse-variance weighted meta-analysis with METAL (A software designed to facilitate meta-analysis of large datasets). The quantile–quantile plots and values of the genomic inflation factor λ suggested no systematic inflation of test statistics due to population stratification, cryptic relatedness, or technical arteficts (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 33, 47</a>). Power of the discovery stage to detect association with various stroke subtypes is presented in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix (p 34)</a>.</p></div><div id="S11" class="sec"><h3 id="S11title">Validation analyses</h3><p id="P13" class="p p-first">We selected variants with high imputation accuracy (mean r<sup>2</sup>&gt;0·80) yielding an association at p&lt;5 × 10<sup>−6</sup> significance level with all stroke, ischaemic stroke, cardioembolic ischaemic stroke, or non-cardioembolic ischaemic stroke. 177 variants belonging to 21 loci (linkage disequilibrium r<sup>2</sup>&gt;0·7 within each locus) were selected. We did in-silico lookups of association results using data from four independent, previously published cross-sectional studies, with mostly hospital-based stroke ascertainment, totalling 19 816 stroke patients (<a href="/pmc/articles/PMC4943223/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>) and 50 988 control participants from the Stroke Genetics Network (SiGN),<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875878">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875865">8</a></sup> METASTROKE,<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875836">6</a></sup> Heart and Vascular Health 1 (HVH1),<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875838">9</a></sup> and Cervical Artery Dissections and Ischaemic Stroke Patients (CADISP) studies.<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_668357913">10</a></sup> Except for 4963 black and 3371 Hispanic participants (cases and controls) in SiGN, participants in the validation samples were of European ancestry. Validation analyses were done with the same, or most similar, stroke phenotype as in the discovery phase (<a href="/pmc/articles/PMC4943223/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>table 1</span></a>), with logistic regression under an additive genetic model, followed by inverse-variance weighted meta-analysis of study-specific association statistics (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 20–25</a>).</p><p id="P14" class="p p-last">After Bonferroni correction for the number of independent loci (r<sup>2</sup>&lt;0·01 reflecting absence of linkage disequilibrium), p&lt;2·38 × 10<sup>−3</sup> was considered significant evidence of replication. We did not correct for the number of stroke phenotypes (all stroke, ischaemic stroke, cardioembolic ischaemic stroke, and non-cardioembolic ischaemic stroke) because they are not independent (cardioembolic and non-cardioembolic ischaemic stroke being subtypes of ischaemic stroke, and ischaemic stroke a subtype of stroke). Only loci reaching genome-wide significance at p&lt;5 × 10<sup>−8</sup> in the combined meta-analysis of discovery and validation samples were given further consideration.</p></div><div id="S12" class="sec"><h3 id="S12title">Secondary analyses</h3><p id="P15" class="p p-first-last">We examined the association of novel, genome-wide significant (at p&lt;5×10<sup>−8</sup>), all-stroke risk variants with stroke subtypes in CHARGE and follow-up samples (using TOAST subtyping<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875872">11</a></sup> and, in sensitivity analyses, the Causative Classification System [CCS] implemented in SiGN<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875854">7</a>,<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875881">8</a></sup>). We examined whether the same variants were associated with burden of white matter hyperintensity, a quantitative MRI-marker of cerebral small vessel disease, in 21 079 participants,<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875886">12</a></sup> and with intracerebral haemorrhage in an independent study comprising 1576 patients (682 patients with lobar intracerebral haemorrhage and 894 with deep intracerebral haemorrhage) and 1303 controls of European ancestry.<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875857">13</a></sup> We tested whether risk variants with genome-wide or suggestive association for all-stroke or ischaemic stroke in the population-based discovery stage were associated with incident fatal and non-fatal stroke. We also looked for a significant association of these risk variants comparing fatal or non-fatal stroke, which would suggest different genetic influences in the two groups of cases, assuming systematic bias in follow-up does not influence the frequency of the candidate variant in either group.</p></div><div id="S13" class="sec"><h3 id="S13title">Functional exploration of novel stroke risk locus</h3><p id="P16" class="p p-first">Based on in-silico functional annotation (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 29–31</a>) and literature review of the novel genome-wide significant stroke risk locus identified by the aforementioned approach, we examined the effect of loss of function of the putative causal gene on brain vasculature and stroke-related phenotypes in humans, mice, and zebrafish (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 31, 32</a>). Studies into human participants and animal models were approved by the ethics committees of the participating institutions. To complement the genome-wide analysis, we used a rare cohort of patients with Axenfeld-Rieger syndrome with deletions of the novel stroke risk locus to directly determine if loss of this locus resulted in more severe cerebrovascular MRI phenotypes. By extracting data from individual MRI slices, we calculated the volume of white matter hyperintensity in two patients (aged 2 and 32 years) with large segmental deletions encompassing the putative causal gene and two patients (aged 15 and 17 years) with smaller deletions in whom this gene was intact.</p><p id="P17">The putative causal gene was deleted in adult (12 weeks) conditional (inducible) mouse mutants by Cre-ERT2, an inducible Cre recombinase<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875868">14</a></sup> and killed 6 weeks later. Brains of conditional knockout mice and controls were examined by histology (using haematoxylin and eosin or Richardson’s methylene blue–Azure 2 staining) and glial fibrillary acidic protein immuno fluorescence (DAKO Z0334) to search for features of vascular brain injury.</p><p id="P18" class="p p-last">Zebrafish allow live imaging of blood vessel and mural cell interactions to be done with exceptional clarity, using transgenic lines that permit in-vivo visualisation of endothelium and smooth muscle. Expression of the putative causal gene in the brain of zebrafish larvae was assessed by in-situ hybridisation and compared with that of established pericyte markers <em>notch3</em> and <em>pdgfrβ</em>.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875845">15</a></sup> Function of forkhead transcription factor 2 <em>(Foxf2)</em> was assessed by knockout with transcription activator-like effector nucleases (TALENs) to create targeted nonsense mutations in the DNA-binding domain (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 35</a>). Smooth muscle cell coverage of branches of brain vessels was examined in live transgenic mutant and wild-type zebrafish embryos; these smooth muscle cells in zebrafish were modified to express green fluorescent protein with the α-smooth muscle actin promoter (acta2:GFP) at 4–6 days postfertilisation. Pericyte density was assessed by <em>pdgfrβ</em> in-situ hybridisation.<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875848">15</a></sup></p></div><div id="S14" class="sec sec-last"><h3 id="S14title">Role of the funding source</h3><p id="P19" class="p p-first-last">All funders were involved in the study design but had no role in data collection. The corresponding author had full access to all the data in the study and had final responsibility for the decision to submit for publication.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">Results</h2><p id="P20" class="p p-first">In the population-based discovery stage (4348 stroke patients <em>vs</em> 80 613 controls), 177 genetic variants in 21 independent loci were associated with incident all-stroke, ischaemic stroke, cardioembolic ischaemic stroke, or non-cardioembolic ischaemic stroke at p&lt;5 × 10<sup>−6</sup> significance (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 48–54</a>). 11 loci showed suggestive association with incident all-stroke or ischaemic stroke at p&lt;5 × 10<sup>−6</sup>. Ten additional loci were associated with incident cardioembolic ischaemic stroke at p&lt;5 × 10<sup>−6</sup> significance, with one locus (the lead single nucleotide polymorphism [SNP] rs72794386 in <em>SLC12A2</em>) showing genome-wide significance (hazard ratio [HR] 1·67, 95% CI 1·39–2·00, p=4·37 × 10<sup>−8</sup>) and minor allele frequency (MAF) of 10% (<a href="/pmc/articles/PMC4943223/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>table 2</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 36, 37</a>)</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p21">Single nucleotide polymorphism associated with risk of stroke at p&lt;5 × 10<sup>−6</sup></p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271642661008" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Chromosome:<br>position<sup>*</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Function</th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th rowspan="1" colspan="1" valign="bottom" align="left">Number<br>of<br>variants<sup>†</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Minor<br>allele</th><th rowspan="1" colspan="1" valign="bottom" align="left">Minor<br>allele<br>frequency</th><th rowspan="1" colspan="1" valign="bottom" align="left">Hazard ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value<sup>‡</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Direction<sup>§</sup></th><th rowspan="1" colspan="1" valign="bottom" align="center">Heterogeneity,<br><em>I</em><sup>2¶</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Heterogeneity,<br>p value<sup>||</sup></th><th rowspan="1" colspan="1" valign="bottom" align="left">Imputation<br>quality<sup>**</sup></th></tr></thead><tbody><tr><td colspan="13" rowspan="1" valign="bottom" align="left"><strong>All stroke</strong>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6433905</td><td rowspan="1" colspan="1" valign="top" align="left">2:182138150</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>UBE2E3</em></td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·08</td><td rowspan="1" colspan="1" valign="top" align="left">1·21 (1·12–1·31)</td><td rowspan="1" colspan="1" valign="top" align="left">2·54 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++++++++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·54</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12204590</td><td rowspan="1" colspan="1" valign="top" align="left">6:1337393</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FOXF2</em></td><td rowspan="1" colspan="1" valign="top" align="left">6</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·21</td><td rowspan="1" colspan="1" valign="top" align="left">1·14 (1·08–1·20)</td><td rowspan="1" colspan="1" valign="top" align="left">2·17 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++++++++++++−+−++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·60</td><td rowspan="1" colspan="1" valign="top" align="left">1·00</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs790919</td><td rowspan="1" colspan="1" valign="top" align="left">6:154298875</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>OPRM1</em></td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·44</td><td rowspan="1" colspan="1" valign="top" align="left">1·12 (1·07–1·17)</td><td rowspan="1" colspan="1" valign="top" align="left">2·44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++++++++++−−++</td><td rowspan="1" colspan="1" valign="top" align="center">16·2</td><td rowspan="1" colspan="1" valign="top" align="left">0·26</td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11788316</td><td rowspan="1" colspan="1" valign="top" align="left">9:13445687</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MPDZ</em></td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·28</td><td rowspan="1" colspan="1" valign="top" align="left">1·13 (1·07–1·19)</td><td rowspan="1" colspan="1" valign="top" align="left">2·49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++−++++−−++++++−−+</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·84</td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11627959</td><td rowspan="1" colspan="1" valign="top" align="left">14:35160471</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CFL2</em></td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·44</td><td rowspan="1" colspan="1" valign="top" align="left">0·89 (0·85–0·93)</td><td rowspan="1" colspan="1" valign="top" align="left">2·23 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−−+−−−−−−+−−+−−−−</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·87</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4899120</td><td rowspan="1" colspan="1" valign="top" align="left">14:64335447</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SYNE2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·09</td><td rowspan="1" colspan="1" valign="top" align="left">1·19 (1·11–1·29)</td><td rowspan="1" colspan="1" valign="top" align="left">4·71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++−++++++++−−++</td><td rowspan="1" colspan="1" valign="top" align="center">24</td><td rowspan="1" colspan="1" valign="top" align="left">0·18</td><td rowspan="1" colspan="1" valign="top" align="left">0·98</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left"><strong>Ischaemic stroke</strong>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs62262077</td><td rowspan="1" colspan="1" valign="top" align="left">3:105014929</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ALCAM</em></td><td rowspan="1" colspan="1" valign="top" align="left">5</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·27</td><td rowspan="1" colspan="1" valign="top" align="left">1·17 (1·10–1·24)</td><td rowspan="1" colspan="1" valign="top" align="left">6·04 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−+++++++++++−+++</td><td rowspan="1" colspan="1" valign="top" align="center">19·2</td><td rowspan="1" colspan="1" valign="top" align="left">0·23</td><td rowspan="1" colspan="1" valign="top" align="left">0·94</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10037362</td><td rowspan="1" colspan="1" valign="top" align="left">5:31110857</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDH6</em></td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·07</td><td rowspan="1" colspan="1" valign="top" align="left">1·27 (1·15–1·40)</td><td rowspan="1" colspan="1" valign="top" align="left">4·41 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−+−++++++++</td><td rowspan="1" colspan="1" valign="top" align="center">25·7</td><td rowspan="1" colspan="1" valign="top" align="left">0·20</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4448595</td><td rowspan="1" colspan="1" valign="top" align="left">10:21666138</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>C10orf114</em></td><td rowspan="1" colspan="1" valign="top" align="left">8</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·16</td><td rowspan="1" colspan="1" valign="top" align="left">0·83 (0·77–0·90)</td><td rowspan="1" colspan="1" valign="top" align="left">2·50 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−−−−−−−−+−−−−−−−</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·80</td><td rowspan="1" colspan="1" valign="top" align="left">0·98</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11833579</td><td rowspan="1" colspan="1" valign="top" align="left">12:775199</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NINJ2</em></td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·24</td><td rowspan="1" colspan="1" valign="top" align="left">1·19 (1·12–1·27)</td><td rowspan="1" colspan="1" valign="top" align="left">5·74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−+++−++++++−++++</td><td rowspan="1" colspan="1" valign="top" align="center">18·5</td><td rowspan="1" colspan="1" valign="top" align="left">0·24</td><td rowspan="1" colspan="1" valign="top" align="left">0·92</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs77858481</td><td rowspan="1" colspan="1" valign="top" align="left">13:81142325</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPRY2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·06</td><td rowspan="1" colspan="1" valign="top" align="left">1·38 (1·22–1·55)</td><td rowspan="1" colspan="1" valign="top" align="left">2·32 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td><td rowspan="1" colspan="1" valign="top" align="left">0·83</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left"><strong>Cardioembolic ischaemic stroke</strong>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4284256</td><td rowspan="1" colspan="1" valign="top" align="left">1:157675273</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FCRL3</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·18</td><td rowspan="1" colspan="1" valign="top" align="left">1·41 (1·22–1·64)</td><td rowspan="1" colspan="1" valign="top" align="left">3·13 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">−+++++</td><td rowspan="1" colspan="1" valign="top" align="center">66·5</td><td rowspan="1" colspan="1" valign="top" align="left">0·01</td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12646447</td><td rowspan="1" colspan="1" valign="top" align="left">4:111699326</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>PITX2</em></td><td rowspan="1" colspan="1" valign="top" align="left">102</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·12</td><td rowspan="1" colspan="1" valign="top" align="left">1·53 (1·31–1·80)</td><td rowspan="1" colspan="1" valign="top" align="left">1·92 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·44</td><td rowspan="1" colspan="1" valign="top" align="left">0·99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs72184</td><td rowspan="1" colspan="1" valign="top" align="left">5:123754837</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF608</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·43</td><td rowspan="1" colspan="1" valign="top" align="left">1·30 (1·17–1·46)</td><td rowspan="1" colspan="1" valign="top" align="left">2·29 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++++++</td><td rowspan="1" colspan="1" valign="top" align="center">9·1</td><td rowspan="1" colspan="1" valign="top" align="left">0·6</td><td rowspan="1" colspan="1" valign="top" align="left">0·90</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs72794386</td><td rowspan="1" colspan="1" valign="top" align="left">5:127479278</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC12A2</em></td><td rowspan="1" colspan="1" valign="top" align="left">22</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·10</td><td rowspan="1" colspan="1" valign="top" align="left">1·67 (1·39–2·00)</td><td rowspan="1" colspan="1" valign="top" align="left">4·37 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·87</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1428155</td><td rowspan="1" colspan="1" valign="top" align="left">5:151281633</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GLRA1</em></td><td rowspan="1" colspan="1" valign="top" align="left">2</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·38</td><td rowspan="1" colspan="1" valign="top" align="left">1·28 (1·16–1·43)</td><td rowspan="1" colspan="1" valign="top" align="left">3·10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++++++</td><td rowspan="1" colspan="1" valign="top" align="center">36·2</td><td rowspan="1" colspan="1" valign="top" align="left">0·13</td><td rowspan="1" colspan="1" valign="top" align="left">1·00</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7771564</td><td rowspan="1" colspan="1" valign="top" align="left">6:22504092</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HDGFL1</em></td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·10</td><td rowspan="1" colspan="1" valign="top" align="left">1·53 (1·28–1·82)</td><td rowspan="1" colspan="1" valign="top" align="left">2·10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·67</td><td rowspan="1" colspan="1" valign="top" align="left">0·99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1495081</td><td rowspan="1" colspan="1" valign="top" align="left">8:15314955</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TUSC3</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·14</td><td rowspan="1" colspan="1" valign="top" align="left">1·48 (1·25–1·74)</td><td rowspan="1" colspan="1" valign="top" align="left">3·09 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++−+++</td><td rowspan="1" colspan="1" valign="top" align="center">49·9</td><td rowspan="1" colspan="1" valign="top" align="left">0·08</td><td rowspan="1" colspan="1" valign="top" align="left">0·88</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2393938</td><td rowspan="1" colspan="1" valign="top" align="left">10:44063812</td><td rowspan="1" colspan="1" valign="top" align="left">UTR5</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF239</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">C</td><td rowspan="1" colspan="1" valign="top" align="left">0·12</td><td rowspan="1" colspan="1" valign="top" align="left">1·45 (1·24–1·70)</td><td rowspan="1" colspan="1" valign="top" align="left">3·47 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·51</td><td rowspan="1" colspan="1" valign="top" align="left">0·99</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11021485</td><td rowspan="1" colspan="1" valign="top" align="left">11:95968208</td><td rowspan="1" colspan="1" valign="top" align="left">Intronic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MAML2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">A</td><td rowspan="1" colspan="1" valign="top" align="left">0·12</td><td rowspan="1" colspan="1" valign="top" align="left">1·60 (1·32–1·94)</td><td rowspan="1" colspan="1" valign="top" align="left">1·24 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++</td><td rowspan="1" colspan="1" valign="top" align="center">30·5</td><td rowspan="1" colspan="1" valign="top" align="left">0·22</td><td rowspan="1" colspan="1" valign="top" align="left">0·82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs710009</td><td rowspan="1" colspan="1" valign="top" align="left">14:59184500</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DACT1</em></td><td rowspan="1" colspan="1" valign="top" align="left">4</td><td rowspan="1" colspan="1" valign="top" align="left">G</td><td rowspan="1" colspan="1" valign="top" align="left">0·16</td><td rowspan="1" colspan="1" valign="top" align="left">1·41 (1·22–1·64)</td><td rowspan="1" colspan="1" valign="top" align="left">3·62 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++++</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·84</td><td rowspan="1" colspan="1" valign="top" align="left">0·98</td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="13" rowspan="1" valign="bottom" align="left"><strong>Non-cardioembolic ischaemic stroke</strong>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs77744591</td><td rowspan="1" colspan="1" valign="top" align="left">13:81142325</td><td rowspan="1" colspan="1" valign="top" align="left">Intergenic</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPRY2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1</td><td rowspan="1" colspan="1" valign="top" align="left">T</td><td rowspan="1" colspan="1" valign="top" align="left">0·08</td><td rowspan="1" colspan="1" valign="top" align="left">1·34 (1·18–1·51)</td><td rowspan="1" colspan="1" valign="top" align="left">3·44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">+++++−+</td><td rowspan="1" colspan="1" valign="top" align="center">0</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td></tr></tbody></table></div><div id="largeobj_idm140271642661008" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4943223/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN6"><p id="__p22" class="p p-first-last">Only associations with the lead single nucleotide polymorphism in each locus are shown, and full set of genetic associations at p&lt;5×10<sup>−6</sup> is presented in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix (pp 48–54)</a>. All results are presented with respect to the minor allele as coded allele. UTR57</p></div><div id="TFN7"><sup>*</sup>Chromosome positions with respect to National Center for Biotechnology Information Build 37 data.</div><div id="TFN8"><sup>†</sup>Number of variants reaching p&lt;5×10<sup>−6</sup> in the locus.</div><div id="TFN9"><sup>‡</sup>p value after genomic control.</div><div id="TFN10"><sup>§</sup>Direction refers to direction of effect size with respect to the minor allele in each study contributing to the meta-analysis (in alphabetic order), “+” sign refers to positive values of betas and “−” sign referes to negative values of betas with respect to the minor allele.</div><div id="TFN11"><sup>¶</sup>Heterogeneity (I<sup>2</sup>) ranges between 0 and 100, with higher values suggesting more heterogeneity.</div><div id="TFN12"><sup>||</sup>p value for heterogeneity was calculated with the Cochran’s Q test.</div><div id="TFN13"><sup>**</sup>Mean value of imputation quality across studies.</div></div></div><p id="P21">In the cross-sectional validation samples (19 816 stroke patients <em>vs</em> 50 988 control participants), associations for two loci were replicated at p&lt;2·38 × 10<sup>−3</sup> significance and reached genome-wide significance (p&lt;5 × 10<sup>−8</sup>) in the combined analysis (<a href="/pmc/articles/PMC4943223/table/T3/" target="table" class="fig-table-link figpopup" rid-figpopup="T3" rid-ob="ob-T3" co-legend-rid=""><span>table 3</span></a>). The first association we found was a previously unreported locus (chr6p25·3, lead SNP rs12204590), located between <em>FOXQ1</em> and <em>FOXF2</em>, showing association with risk of incident all-stroke (combined odds ratio [OR] 1·08 95% CI 1·05–1·12, p=1·48 × 10<sup>−8</sup>; MAF 21%; <a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>). Associations in each study are shown in the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix (p 38)</a>. The second locus (chr4q25, near <em>PITX2</em>) is a known risk locus for cardioembolic ischaemic stroke (combined OR 1·37, 1·29–1·46, p=4·72 × 10<sup>−23</sup> for incident cardioembolic ischaemic stroke; MAF 12%). The <em>SLC12A2</em> locus (genome-wide significant in the small cardioembolic ischaemic stroke discovery sample) did not show evidence of replication (p=0·27).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271718190672" class="figure"><a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4943223_nihms778629f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms778629f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4943223/bin/nihms778629f1.jpg"></a></div><div id="largeobj_idm140271718190672" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4943223/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4943223/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plot of rs12204590 in discovery stage</strong><p id="__p13">Association of rs12204590 and other genotyped or imputed SNPs (circles) in the region with incident all-stroke. Colour variation shows linkage disequilibrium between SNPs (r<sup>2</sup>) as calculated in 1000 Genomes Project (phase 1, version 3). Blue lines show estimated recombination rates. Coloured tracksseen at bottom of figure were added using the University of California, Santa Cruz genome browser and the RegulomeDB database. SNP track=SNPs encompassing the selected region and red dashed line shows position of top SNP (rs12204590). Regulome track=RegulomeDB scores and variants with lower scores have higher probability of acting as regulatory variants. DNase track=DNase hypersensitive regions assayed in 125 cell types (ENCODE project, Release 3,2014). TFbs track=regions where transcription factors, proteins responsible for modulating gene transcription, bind to DNA as assayed by ChIP-seq assay (ENCODE project, Release 3, 2013).</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p30">Validation of top loci in independent cross-sectional case-control studies associated with risk of stroke</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271641637760" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Gene</th><th colspan="2" rowspan="1" valign="bottom" align="left">Discovery
<hr></th><th colspan="10" rowspan="1" valign="bottom" align="left">Validation
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">Combined total
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="2" rowspan="1" valign="bottom" align="left">CHARGE
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">SiGN
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">METASTROKE
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">HVH1
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">CADISP
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">Validation<br>meta-analysis
<hr></th><th colspan="2" rowspan="1" valign="bottom" align="left">Meta-analysis
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="left">Hazard<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th><th rowspan="1" colspan="1" valign="bottom" align="left">Odds<br>ratio<br>(95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="left">p value</th></tr></thead><tbody><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><strong>All stroke</strong><sup>*</sup>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6433905</td><td rowspan="1" colspan="1" valign="top" align="left"><em>UBE2E3</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·21 (1·12–1·31)</td><td rowspan="1" colspan="1" valign="top" align="left">2·54 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·97–1·08)</td><td rowspan="1" colspan="1" valign="top" align="left">0·41</td><td rowspan="1" colspan="1" valign="top" align="left">1·11 (0·98–1·27)</td><td rowspan="1" colspan="1" valign="top" align="left">0·11</td><td rowspan="1" colspan="1" valign="top" align="left">1·09 (0·83–1·44)</td><td rowspan="1" colspan="1" valign="top" align="left">0·54</td><td rowspan="1" colspan="1" valign="top" align="left">0·72 (0·54–0·95)</td><td rowspan="1" colspan="1" valign="top" align="left">0·018</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·98–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·22</td><td rowspan="1" colspan="1" valign="top" align="left">1·07 (1·03–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">8·72 × 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12204590<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>FOXF2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·14 (1·08–1·20)</td><td rowspan="1" colspan="1" valign="top" align="left">2·17 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·07 (1·03–1·11)</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·07 (0·98–1·16)</td><td rowspan="1" colspan="1" valign="top" align="left">0·13</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·86–1·24)</td><td rowspan="1" colspan="1" valign="top" align="left">0·73</td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (0·92–1·26)</td><td rowspan="1" colspan="1" valign="top" align="left">0·36</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (1·03–1·09)</td><td rowspan="1" colspan="1" valign="top" align="left">2·15 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (1·05–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">1·48 × 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs790919</td><td rowspan="1" colspan="1" valign="top" align="left"><em>OPRM1</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·12 (1·07–1·17)</td><td rowspan="1" colspan="1" valign="top" align="left">2·44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·97–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·88</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·95–1·09)</td><td rowspan="1" colspan="1" valign="top" align="left">0·70</td><td rowspan="1" colspan="1" valign="top" align="left">0·88 (0·76–1·02)</td><td rowspan="1" colspan="1" valign="top" align="left">0·10</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (0·93–1·21)</td><td rowspan="1" colspan="1" valign="top" align="left">0·37</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·98–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·80</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (1·01–1·05)</td><td rowspan="1" colspan="1" valign="top" align="left">0·013</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11788316</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FLJ41200</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·13 (1·07–1·19)</td><td rowspan="1" colspan="1" valign="top" align="left">2·49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·96–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·73</td><td rowspan="1" colspan="1" valign="top" align="left">1·07 (0·99–1·16)</td><td rowspan="1" colspan="1" valign="top" align="left">0·074</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·84–1·21)</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·85–1·15)</td><td rowspan="1" colspan="1" valign="top" align="left">0·89</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·99–1·04)</td><td rowspan="1" colspan="1" valign="top" align="left">0·38</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (1·01–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">7·53 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11627959</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CFL2</em></td><td rowspan="1" colspan="1" valign="top" align="left">0·89 (0·85–0·93)</td><td rowspan="1" colspan="1" valign="top" align="left">2·23 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·96–1·02)</td><td rowspan="1" colspan="1" valign="top" align="left">0·70</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·93–1·08)</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·85–1·17)</td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·90–1·18)</td><td rowspan="1" colspan="1" valign="top" align="left">0·62</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·98–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·53</td><td rowspan="1" colspan="1" valign="top" align="left">0·97 (0·95–0·99)</td><td rowspan="1" colspan="1" valign="top" align="left">0·011</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4899120</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SYNE2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·19 (1·11–1·29)</td><td rowspan="1" colspan="1" valign="top" align="left">4·71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·97–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·50</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·88–1·14)</td><td rowspan="1" colspan="1" valign="top" align="left">0·98</td><td rowspan="1" colspan="1" valign="top" align="left">1·05 (0·80–1·37)</td><td rowspan="1" colspan="1" valign="top" align="left">0·73</td><td rowspan="1" colspan="1" valign="top" align="left">1·16 (0·92–1·46)</td><td rowspan="1" colspan="1" valign="top" align="left">0·21</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·99–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·23</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (1·02–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">2·00 × 10<sup>−3</sup></td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><strong>Ischaemic stroke</strong><sup>‡</sup>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs62262077</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ALCAM</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·17 (1·10–1·24)</td><td rowspan="1" colspan="1" valign="top" align="left">6·04 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·96–1·02)</td><td rowspan="1" colspan="1" valign="top" align="left">0·52</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·95–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">0·46</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·84–1·26)</td><td rowspan="1" colspan="1" valign="top" align="left">0·78</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·88–1·18)</td><td rowspan="1" colspan="1" valign="top" align="left">0·82</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·98–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·63</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (1·01–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">0·015</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs10037362</td><td rowspan="1" colspan="1" valign="top" align="left"><em>CDH6</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·27 (1·15–1·40)</td><td rowspan="1" colspan="1" valign="top" align="left">4·41 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·93–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·40</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·87–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">0·84</td><td rowspan="1" colspan="1" valign="top" align="left">0·93 (0·69–1·26)</td><td rowspan="1" colspan="1" valign="top" align="left">0·65</td><td rowspan="1" colspan="1" valign="top" align="left">0·86 (0·66–1·13)</td><td rowspan="1" colspan="1" valign="top" align="left">0·27</td><td rowspan="1" colspan="1" valign="top" align="left">0·97 (0·93–1·02)</td><td rowspan="1" colspan="1" valign="top" align="left">0·22</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·97–1·05)</td><td rowspan="1" colspan="1" valign="top" align="left">0·55</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4448595</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NEBL-AS1</em></td><td rowspan="1" colspan="1" valign="top" align="left">0·83 (0·77–0·90)</td><td rowspan="1" colspan="1" valign="top" align="left">2·50 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·97–1·05)</td><td rowspan="1" colspan="1" valign="top" align="left">0·72</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·93–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">0·69</td><td rowspan="1" colspan="1" valign="top" align="left">0·89 (0·72–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·28</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·81–1·14)</td><td rowspan="1" colspan="1" valign="top" align="left">0·65</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·97–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·95–1·00)</td><td rowspan="1" colspan="1" valign="top" align="left">0·09</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11833579</td><td rowspan="1" colspan="1" valign="top" align="left"><em>NINJ2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·19 (1·12–1·27)</td><td rowspan="1" colspan="1" valign="top" align="left">5·74 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·95–1·01)</td><td rowspan="1" colspan="1" valign="top" align="left">0·21</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·88–1·04)</td><td rowspan="1" colspan="1" valign="top" align="left">0·28</td><td rowspan="1" colspan="1" valign="top" align="left">1·04 (0·85–1·29)</td><td rowspan="1" colspan="1" valign="top" align="left">0·69</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·82–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">0·57</td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·95–1·01)</td><td rowspan="1" colspan="1" valign="top" align="left">0·14</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·98–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·47</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs77858481</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPRY2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·38 (1·22–1·55)</td><td rowspan="1" colspan="1" valign="top" align="left">2·32 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·93–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">0·75</td><td rowspan="1" colspan="1" valign="top" align="left">0·90 (0·77–1·05)</td><td rowspan="1" colspan="1" valign="top" align="left">0·17</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·72–1·44)</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td><td rowspan="1" colspan="1" valign="top" align="left">0·91 (0·69–1·20)</td><td rowspan="1" colspan="1" valign="top" align="left">0·48</td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·93–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·50</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·99–1·08)</td><td rowspan="1" colspan="1" valign="top" align="left">0·17</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><strong>Cardioembolic ischaemic stroke</strong><sup>§</sup>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4284256</td><td rowspan="1" colspan="1" valign="top" align="left"><em>FCRL3</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·41 (1·22–1·64)</td><td rowspan="1" colspan="1" valign="top" align="left">3·13 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·90–1·04)</td><td rowspan="1" colspan="1" valign="top" align="left">0·31</td><td rowspan="1" colspan="1" valign="top" align="left">0·93 (0·78–1·12)</td><td rowspan="1" colspan="1" valign="top" align="left">0·45</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·69–1·55)</td><td rowspan="1" colspan="1" valign="top" align="left">0·88</td><td rowspan="1" colspan="1" valign="top" align="left">0·98 (0·75–1·28)</td><td rowspan="1" colspan="1" valign="top" align="left">0·90</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·90–1·01)</td><td rowspan="1" colspan="1" valign="top" align="left">0·13</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·97–1·09)</td><td rowspan="1" colspan="1" valign="top" align="left">0·41</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12646447<sup>†</sup></td><td rowspan="1" colspan="1" valign="top" align="left"><em>PITX2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·53 (1·31–1·80)</td><td rowspan="1" colspan="1" valign="top" align="left">1·92 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·39 (1·29–1·50)</td><td rowspan="1" colspan="1" valign="top" align="left">3·15 × 10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·17 (0·98–1·41)</td><td rowspan="1" colspan="1" valign="top" align="left">0·083</td><td rowspan="1" colspan="1" valign="top" align="left">1·62 (1·09–2·42)</td><td rowspan="1" colspan="1" valign="top" align="left">0·018</td><td rowspan="1" colspan="1" valign="top" align="left">1·04 (0·78–1·37)</td><td rowspan="1" colspan="1" valign="top" align="left">0·80</td><td rowspan="1" colspan="1" valign="top" align="left">1·36 (1·28–1·44)</td><td rowspan="1" colspan="1" valign="top" align="left">1·89 × 10<sup>−23</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·37 (1·29–1·46)</td><td rowspan="1" colspan="1" valign="top" align="left">4·72 × 10<sup>−24</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs72184</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF608</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·30 (1·17–1·46)</td><td rowspan="1" colspan="1" valign="top" align="left">2·29 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·96–1·08)</td><td rowspan="1" colspan="1" valign="top" align="left">0·53</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·87–1·13)</td><td rowspan="1" colspan="1" valign="top" align="left">0·90</td><td rowspan="1" colspan="1" valign="top" align="left">0·92 (0·67–1·28)</td><td rowspan="1" colspan="1" valign="top" align="left">0·63</td><td rowspan="1" colspan="1" valign="top" align="left">1·03 (0·85–1·26)</td><td rowspan="1" colspan="1" valign="top" align="left">0·74</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·97–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">0·49</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (1·01–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·017</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs72794386</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SLC12A2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·67 (1·39–2·00)</td><td rowspan="1" colspan="1" valign="top" align="left">4·37 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·97 (0·89–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">0·51</td><td rowspan="1" colspan="1" valign="top" align="left">1·09 (0·87–1·37)</td><td rowspan="1" colspan="1" valign="top" align="left">0·46</td><td rowspan="1" colspan="1" valign="top" align="left">0·95 (0·56–1·62)</td><td rowspan="1" colspan="1" valign="top" align="left">0·85</td><td rowspan="1" colspan="1" valign="top" align="left">0·78 (0·54–1·14)</td><td rowspan="1" colspan="1" valign="top" align="left">0·18</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·90–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·27</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (0·99–1·14)</td><td rowspan="1" colspan="1" valign="top" align="left">0·12</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1428155</td><td rowspan="1" colspan="1" valign="top" align="left"><em>GLRA1</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·28 (1·16–1·43)</td><td rowspan="1" colspan="1" valign="top" align="left">3·10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·97 (0·92–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·33</td><td rowspan="1" colspan="1" valign="top" align="left">0·95 (0·83–1·08)</td><td rowspan="1" colspan="1" valign="top" align="left">0·44</td><td rowspan="1" colspan="1" valign="top" align="left">0·90 (0·66–1·24)</td><td rowspan="1" colspan="1" valign="top" align="left">0·51</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·81–1·21)</td><td rowspan="1" colspan="1" valign="top" align="left">0·94</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·95–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·64</td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·98–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·38</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs7771564</td><td rowspan="1" colspan="1" valign="top" align="left"><em>HDGFL1</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·53 (1·28–1·82)</td><td rowspan="1" colspan="1" valign="top" align="left">2·10 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·92–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·87</td><td rowspan="1" colspan="1" valign="top" align="left">0·88 (0·70–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·25</td><td rowspan="1" colspan="1" valign="top" align="left">1·23 (0·77–1·96)</td><td rowspan="1" colspan="1" valign="top" align="left">0·39</td><td rowspan="1" colspan="1" valign="top" align="left">1·20 (0·89–1·62)</td><td rowspan="1" colspan="1" valign="top" align="left">0·24</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·93–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·97</td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (1·01–1·17)</td><td rowspan="1" colspan="1" valign="top" align="left">0·031</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs1495081</td><td rowspan="1" colspan="1" valign="top" align="left"><em>TUSC3</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·48 (1·25–1·74)</td><td rowspan="1" colspan="1" valign="top" align="left">3·09 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·05 (0·98–1·14)</td><td rowspan="1" colspan="1" valign="top" align="left">0·18</td><td rowspan="1" colspan="1" valign="top" align="left">0·89 (0·72–1·09)</td><td rowspan="1" colspan="1" valign="top" align="left">0·25</td><td rowspan="1" colspan="1" valign="top" align="left">1·35 (0·84–2·16)</td><td rowspan="1" colspan="1" valign="top" align="left">0·21</td><td rowspan="1" colspan="1" valign="top" align="left">1·05 (0·79–1·40)</td><td rowspan="1" colspan="1" valign="top" align="left">0·73</td><td rowspan="1" colspan="1" valign="top" align="left">1·04 (0·98–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·24</td><td rowspan="1" colspan="1" valign="top" align="left">1·10 (1·03–1·17)</td><td rowspan="1" colspan="1" valign="top" align="left">5·07 × 10<sup>−3</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2393938</td><td rowspan="1" colspan="1" valign="top" align="left"><em>ZNF239</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·45 (1·24–1·70)</td><td rowspan="1" colspan="1" valign="top" align="left">3·47 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·02 (0·94–1·10)</td><td rowspan="1" colspan="1" valign="top" align="left">0·68</td><td rowspan="1" colspan="1" valign="top" align="left">1·01 (0·84–1·22)</td><td rowspan="1" colspan="1" valign="top" align="left">0·88</td><td rowspan="1" colspan="1" valign="top" align="left">0·95 (0·60–1·52)</td><td rowspan="1" colspan="1" valign="top" align="left">0·84</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·71–1·29)</td><td rowspan="1" colspan="1" valign="top" align="left">0·76</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·94–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·95</td><td rowspan="1" colspan="1" valign="top" align="left">1·07 (1·01–1·15)</td><td rowspan="1" colspan="1" valign="top" align="left">0·029</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs11021485</td><td rowspan="1" colspan="1" valign="top" align="left"><em>MAML2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·60 (1·32–1·94)</td><td rowspan="1" colspan="1" valign="top" align="left">1·24 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">0·94 (0·86–1·03)</td><td rowspan="1" colspan="1" valign="top" align="left">0·17</td><td rowspan="1" colspan="1" valign="top" align="left">0·77 (0·63–0·95)</td><td rowspan="1" colspan="1" valign="top" align="left">0·015</td><td rowspan="1" colspan="1" valign="top" align="left">0·51 (0·25–1·04)</td><td rowspan="1" colspan="1" valign="top" align="left">0·063</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (0·78–1·43)</td><td rowspan="1" colspan="1" valign="top" align="left">0·73</td><td rowspan="1" colspan="1" valign="top" align="left">0·92 (0·86–0·99)</td><td rowspan="1" colspan="1" valign="top" align="left">0·017</td><td rowspan="1" colspan="1" valign="top" align="left">0·99 (0·92–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·82</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs710009</td><td rowspan="1" colspan="1" valign="top" align="left"><em>DACT1</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·41 (1·22–1·64)</td><td rowspan="1" colspan="1" valign="top" align="left">3·62 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·92–1·07)</td><td rowspan="1" colspan="1" valign="top" align="left">0·93</td><td rowspan="1" colspan="1" valign="top" align="left">0·96 (0·80–1·15)</td><td rowspan="1" colspan="1" valign="top" align="left">0·68</td><td rowspan="1" colspan="1" valign="top" align="left">1·21 (0·80–1·83)</td><td rowspan="1" colspan="1" valign="top" align="left">0·37</td><td rowspan="1" colspan="1" valign="top" align="left">1·10 (0·84–1·43)</td><td rowspan="1" colspan="1" valign="top" align="left">0·50</td><td rowspan="1" colspan="1" valign="top" align="left">1·00 (0·94–1·06)</td><td rowspan="1" colspan="1" valign="top" align="left">0·96</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (1·00–1·13)</td><td rowspan="1" colspan="1" valign="top" align="left">0·048</td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="16" rowspan="1" valign="bottom" align="left"><strong>Non-cardioembolic ischaemic stroke</strong><sup>¶</sup>
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs77744591</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPRY2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·34 (1·18–1·51)</td><td rowspan="1" colspan="1" valign="top" align="left">3·44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (0·96–1·21)</td><td rowspan="1" colspan="1" valign="top" align="left">0·19</td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (0·84–1·40)</td><td rowspan="1" colspan="1" valign="top" align="left">0·53</td><td rowspan="1" colspan="1" valign="top" align="left">1·39 (0·74–2·59)</td><td rowspan="1" colspan="1" valign="top" align="left">0·30</td><td rowspan="1" colspan="1" valign="top" align="left">0·91 (0·47–1·77)</td><td rowspan="1" colspan="1" valign="top" align="left">0·78</td><td rowspan="1" colspan="1" valign="top" align="left">1·08 (0·99–1·18)</td><td rowspan="1" colspan="1" valign="top" align="left">0·80</td><td rowspan="1" colspan="1" valign="top" align="left">1·18 (1·09–1·28)</td><td rowspan="1" colspan="1" valign="top" align="left">3·22 × 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs77744591</td><td rowspan="1" colspan="1" valign="top" align="left"><em>SPRY2</em></td><td rowspan="1" colspan="1" valign="top" align="left">1·34 (1·18–1·51)</td><td rowspan="1" colspan="1" valign="top" align="left">3·44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">1·12 (1·02–1·24)</td><td rowspan="1" colspan="1" valign="top" align="left">0·023</td><td rowspan="1" colspan="1" valign="top" align="left">1·06 (0·80–1·40)</td><td rowspan="1" colspan="1" valign="top" align="left">0·68</td><td rowspan="1" colspan="1" valign="top" align="left">0·83 (0·52–1·31)</td><td rowspan="1" colspan="1" valign="top" align="left">0·42</td><td rowspan="1" colspan="1" valign="top" align="left">0·73 (0·24–2·22)</td><td rowspan="1" colspan="1" valign="top" align="left">0·56</td><td rowspan="1" colspan="1" valign="top" align="left">1·11 (1·02–1·20)</td><td rowspan="1" colspan="1" valign="top" align="left">0·16</td><td rowspan="1" colspan="1" valign="top" align="left">1·18 (1·10–1·27)</td><td rowspan="1" colspan="1" valign="top" align="left">1·08 × 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm140271641637760" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4943223/table/T3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN14"><sup>*</sup>Validationresults from association analyses of ischaemic stroke for SiGN, METASTROKE, and CADISP, and of all stroke for HVH1.</div><div id="TFN15"><sup>†</sup>Loci that reach genome-wide significance (p&lt;5 × 10<sup>−8</sup>) in the combined meta-analysis.</div><div id="TFN16"><sup>‡</sup>Follow-up results from association analyses of ischaemic stroke for SiGN, METASTROKE, HVH1, and CADISP.</div><div id="TFN17"><sup>§</sup>Validation results from association analyses of cardioembolic ischaemic stroke for SiGN, METASTROKE, HVH1, and CADISP (TOAST subtyping).</div><div id="TFN18"><sup>¶</sup>Validation results from association analyses of large-vessel ischaemic stroke (first line) and small-vessel ischaemic stroke (second line) for SiGN, METASTROKE, HVH1, and CADISP (TOAST subtyping).</div></div></div><p id="P22">We also explored association of the chr6p25·3 locus with stroke subtypes. In the discovery sample, lead SNP rs12204590 was associated with incident ischaemic stroke (HR 1·13, 95% CI 1·06–1·20), p=1·64 × 10<sup>−4</sup>; non-cardioembolic ischaemic stroke: HR 1·12, 1·04–1·22, p=4·35×10<sup>−3</sup>; and cardioembolic ischaemic stroke: HR 1·10 0·95–1·27, p=0·21). In validation samples, we detected an association with small-vessel ischaemic stroke (OR 1·08, 95% CI 1·02–1·14, p=0·0094) for rs12200309 (in complete linkage disequilibrium with rs12204590), using TOAST subtypes, whereas association with large-vessel and cardioembolic ischaemic stroke was not significant (p&gt;0·35). The association with small-vessel ischaemic stroke was also evident when using CCS-causative subtyping where available (SiGN; OR 1·11, 1·05–1·18, p=0·00029; <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 55</a>). chromosome 6p25·3 was also significantly associated with increasing burden of white matter hyperintensity in the general population (most significant p value was p=0·0025, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 56</a>). The HR for association of rs12204590 with incident fatal ischaemic stroke (n=271; HR 1·21, 0·99–1·50, p=0·0684) was higher than with non-fatal ischaemic stroke (n=2300; HR 1·14, 1·06–1·22, p=4·93×10<sup>−4</sup>), but the difference was not significant (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 57–59</a>). We did not record any heterogeneity by ethnicity for associations between the chr6p25 locus and stroke risk (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 60</a>).</p><p id="P23">The genomic region where the variant associated with increased stroke risk and adjacent linked variants (linkage disequilibrium r<sup>2</sup>&gt;0·50) are located seems to include enhancers (regions of DNA that activate transcription of nearby genes). This genomic region also includes DNase I hypersensitive regions, a marker of open chromatin associated with active cis-regulatory elements important for transcription of nearby genes (<a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 61, 62</a>). Two SNPs (rs7750826 and rs2006798, r<sup>2</sup>&gt;0·75 with rs12204590) had RegulomeDB scores of 2b, suggesting a probable role in regulating gene expression (combination of transcription factor binding site and DNase peak and footprint; <a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 63, 64</a>). The genomic region that includes the lead variant and variants in linkage disequilibrium also includes two protein-coding genes, <em>FOXQ1</em> and <em>FOXF2</em>. The same region also includes a microRNA (<em>MIR6720</em>). However, if we expand the regional plot to a 1 Mb region around the lead variant, the region also includes two other protein coding genes (<em>FOXC1</em> and <em>GMDS</em>) and a long non-coding RNA (LINC01622; <a href="/pmc/articles/PMC4943223/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>figure 1</span></a>). We did an extensive search of publicaly available expression quantitative trait loci (eQTL)<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875841">16</a></sup> and miRNA databases,<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875840">17</a></sup> examined mRNA expression of <em>FOXF2</em> and adjacent genes in the dorsolateral prefrontal cortex of 508 participants enrolled in the Religious Orders Study and the Rush Memory and Aging Project,<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875876">18</a></sup> and mined large sets of epigenomic data from the International Human Epigenome Consortium (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 65–67</a>). eQTL or methylation quantitative trait loci in this region were absent except for a long non-coding RNA (LOC285768) in the human brain (p=5·25×10<sup>−7</sup> for rs7746700, average in ten brain regions in the BRAINEAC database). However, the different distribution of histone modifications associated with active genes in cells expressing <em>FOXF2</em> or <em>FOXQ1</em> suggested that these variants are likely to lie within the regulatory region of <em>FOXF2</em> (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 39, 40</a>). Moreover, we noted the lowest CpG methylation levels (indicating highest activity) at this locus in the fetal brain compared with any other tissue.<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875882">19</a></sup></p><p id="P24">We have previously described that patients with Axenfeld-Rieger syndrome, a rare heterogeneous condition with maldevelopment of the ocular anterior segment attributable to mutation or copy number variation of <em>FOXC1</em> (adjacent to <em>FOXF2</em> on chr6p25), have increased burden of MRI markers of cerebral small vessel disease.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875844">20</a></sup> Within this cohort of people with <em>FOXC1</em>-attributable Axenfeld-Rieger syndrome, we identified two patients with 300 kb segmental deletions encompassing both <em>FOXC1</em> and <em>FOXF2</em> and found that they had extensive, confluent white matter hyperintensity, with more than ten-times larger volume of white matter hyperintensity than two patients with segmental deletions of <em>FOXC1</em> only (30 kb), although the small number of patients does not allow a formal statistical comparison to be made. All patients were younger than 35 years (range 2–32 years) and lacked vascular risk factors (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2A–E</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 68</a>). White matter hyperintensities are normally absent or negligible in this age range in the general population.<sup><a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875863">21</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140271646417456" class="figure"><a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4943223_nihms778629f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms778629f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4943223/bin/nihms778629f2.jpg"></a></div><div id="largeobj_idm140271646417456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4943223/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4943223/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Cerebrovascular phenotype of <em>FOXF2</em> and <em>FOXC1</em> deletions in humans and <em>FoxF2</em> knockout mice, and expression of <em>foxf2a</em> and <em>foxf2b</em> in zebrafish</strong><p id="__p14">(A–D) In two patients with segmental deletion encompassing <em>FOXC1</em>, white matter hyperintensities are noted in periventricular region (A,B) and subcortical regions (B). In two patients with segmental deletion of both <em>FOXC1</em> and <em>FOXF2</em> (C,D), mean white matter hyperintensities volume is increased by more than ten times (E), in subcortical and periventricular regions (see <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 68</a> for white matter hyperintensities volumes in each of the four patients). (F–I) Cerebral cortex of conditional <em>Foxf2</em> knockout mouse showing ischaemic infarction and haemorrhagic tissue. (F) Area with condensed eosinophilic cytoplasm and pyknotic nuclei (to the right dashed line) that indicates recent ischaemic infarction. (Fi) Normal tissue and (Fii) tissue with ischaemic infarction at higher magnification. (G) Glial fibrillary acidic protein immunofluorescence of area with reactive astrogliosis in cerebral cortex of <em>Foxf2</em> conditional knockout mouse. (H) Cerebral cortex from control mouse showing normal neuronal tissue and intact capillaries. (I) Haemorrhagic area of cerebral cortex from <em>Foxf2</em> conditional knockout mouse. Extravascular erythrocytes seen both as intact cells (homogeneous greenish blue) and lysed cells (black). (J–M) RNA in situ hybridisation (purple) of larval zebrafish brains shows expression of <em>foxf2a</em> (J) and <em>foxf2b</em> (K) in presumptive pericytes compared with known pericyte markers <em>pdgfrβ</em> (L) and <em>notch3</em> (M; purple) around capillaries in 1-month old larval zebrafish (brown). (N-Q) <em>foxf2</em><sup>−</sup><em><sup>/</sup></em><sup>−</sup> mutants have reduced expression of pericyte marker <em>pdgfrβ</em> in 4-day postfertilisation embryonic cerebral vasculature. (R–U) Loss of <em>foxf2b</em> results in reduction of the smooth-muscle marker acta2:GFP coverage of blood vessels in wild type (n=11), <em>foxf2<sup>+/</sup></em><sup>−</sup> (n=31), and <em>Foxf2<sup>−/−</sup></em> (n=20) embryonic cerebral vasculature. (R–T) Examples of high, medium, and low branch order coverage scored from 0 to the 4th order branch in vessel coverage presented as percentages of total embryo counts (U). Arrow heads in images J–Q refer to positions showing pericytes.</p></div></div></div><p id="P25">To understand the mechanism at the tissue level we used histology of the brains from six mice 6 weeks after <em>Foxf2</em> inactivation and found areas of neurons with pyknotic nuclei and eosinophilic cytoplasm (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2F–Fii</span></a>), suggestive of ischaemic infarction, in five of the brains. Patches with elevated levels of glial fibrillary acidic protein in astrocytes (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2G</span></a>) indicated reactive gliosis. Increased magnification revealed a few instances of microhaemorrhage with extravascular erythrocytes (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2I</span></a>). By contrast, brains from control mice contained only occasional and scattered neurons that showed signs of apoptosis or isolated astrocytes with increased glial fibrillary acidic protein immuonoreactivity and no visible haemorrhagic lesions (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2H</span></a>). Mice in which <em>Foxf2</em> had been deleted had significantly higher mortality than control mice. Usually the animals were found dead, but some had to be euthanised after showing behaviour indicative of brain damage, such as circling or lopsided gait.</p><p id="P26">In zebrafish, <em>foxf2a</em> and <em>foxf2b</em> (<em>FOXF2</em> orthologs) are expressed in the cerebral endothelium in pericytes, similar to expression in pericytes in mice (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2J,K</span></a>). Expression occurrs in a pattern similar to those of established pericyte markers <em>pdgfr</em>β and <em>notch3</em> (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2L,M</span></a>). We made two zebrafish knockout lines, <em>foxf2b<sup>ca22</sup></em> and <em>foxf2b<sup>ca23</sup>,</em> with nonsense mutations in the first exon of <em>foxf2b</em> that would result in a translation block before the essential DNA-binding domain (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 35</a>). Both alleles had identical phenotypes. <em>foxf2b</em> mutants had decreased expression of the brain pericyte marker <em>pdgfrβ</em> (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2N–Q</span></a>), which is indicative of pericyte maturation defects and decreased acta2-positive smooth muscle cell coverage on large cerebral vessels, which in turn is suggestive of smooth-muscle defects (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2R–U</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 41</a>). In homozygous mutants, acta2 was visible up to second order of cerebral vessel branching or less versus fourth or higher order branch in wild-type or heterozygous embryos.</p><p id="P27" class="p p-last">When considering risk loci for ischaemic stroke reported by previous cross-sectional studies,<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875858">2</a></sup> we found that seven of the eight published loci were associated with incident stroke in the discovery stage of the present population-based GWAS, predominantly in the same subtype as the original study (p value range=0·047–7·82 × 10<sup>−5</sup>, <a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 69</a>). One known risk locus for ischaemic stroke (<em>PITX2</em>) also showed association with incident intracerebral haemorrhage (p=0·0031), and one known risk locus for intracerebral haemorrhage (<em>PMF1-BGLAP</em>), also a known risk locus for increasing white matter hyperintensity burden,<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875850">12</a></sup> was associated with incident ischaemic stroke (p=0·00064), both in the same direction (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 31</a>).</p></div><div id="S16" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S16title">Discussion</h2><p id="P28" class="p p-first">In a large population-based GWAS meta-analysis of incident stroke, with validation in the largest available cross-sectional stroke GWAS, we identified a novel genome-wide significant association of common variants in the chr6p25·3 region, near <em>FOXF2</em>, with risk of stroke. Associations predominated with small-vessel ischaemic stroke, and significant association was noted with burden of white matter hyperintensity; in addition, patients with rare segmental deletions of <em>FOXF2</em> also showed extensive burden of white matter hyperintensity. These findings suggest an effect of this locus on cerebral small vessel disease; however, the mechanism by which this transcription factor results in cerebral small vessel disease and stroke is unclear. To investigate the possibility of a role of <em>FOXF2</em> in stroke<em>,</em> we did experimental studies in two animal species to examine its role in cerebral vessel development and stability. We showed areas of infarction and microhaemorrhages in brains of conditional <em>Foxf2</em> mutant mice. We showed in zebrafish that <em>foxf2b</em> was expressed in brain pericytes (as in mice<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875851">14</a></sup>) and that reduction in <em>foxf2b</em> function led to differentiation defects of both pericytes and smooth muscle cells in the developing cerebral vasculature.</p><p id="P29">Converging evidence from the present study and previous publications suggests an important role of <em>FOXF2</em> in cerebrovascular disease. In mice, we previously showed that <em>Foxf2</em> is required for brain pericyte differentiation and blood–brain barrier development, with <em>Foxf2</em><sup>−/−</sup> embryos showing thickened endothelium, perivascular oedema, thinning of the vascular basal lamina, and a leaky blood–brain barrier.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875853">14</a></sup> We had also described that <em>Foxf2</em> inactivation in adult mice results in endothelial thickening and blood–brain barrier breakdown,<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875885">14</a></sup> an important mediator of cerebral small vessel disease, and increased mortality.<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875843">22</a></sup> In the conditional <em>Foxf2</em> mutant, we analysed mice brains 6 weeks after <em>Foxf2</em> inactivation and we found signs of brain infarction, with reactive astrogliosis and microhaemorrhages. Of note in previous experiments, <em>Foxf2</em><sup>−/−</sup> mouse embryos developed intracranial haemorrhage,<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875870">14</a></sup> whereas areas of microhaemorrhage were scarce in conditional <em>Foxf2</em> mutant brains. We were also unable to show an association of the chr6p25.3 locus with intracerebral haemorrhage in the largest available collaborative genetic association study, although power might have been limited by the size of the study (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 70</a>). In zebrafish, <em>foxf2b</em> knockout led to disruption of cerebral vasculature with decreased pericyte density and smooth muscle cell coverage (<a href="/pmc/articles/PMC4943223/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>figure 2</span></a>). <em>Foxf2</em> is first expressed in the neural crest and in mice regulates pathways involved in mural cell (pericyte and vascular smooth muscle cell) differentiation, including the pdgfβ and serum response factor pathways.<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875849">14</a>,<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875834">23</a></sup> We did not see haemorrhage in the zebrafish <em>foxf2</em> mutant model during embryonic stages. We note that we knocked out only one of two <em>foxf2</em> genes in zebrafish, and even though mural cell markers have changed expression in <em>foxf2b</em> mutants, there might be genetic compensation from the <em>foxf2a</em> gene that could make the phenotype less severe. However, we have not found expression changes of <em>foxf2a</em> in zebrafish having mutant <em>foxf2b</em>. We cannot exclude haemorrhage occurring at juvenile or adult stages that we have not been able to examine. In summary, our data suggest that association of <em>FOXF2</em> and stroke might arise from differentiation defects of cerebral vascular mural cells. To demonstrate the cellular requirement for <em>Foxf2</em> by expression under a vascular mural cell promoter for prevention of stroke and mural cell phenotypes in mutants is an important future experiment, but is beyond the scope of this study.</p><p id="P30">Forkhead transcription factors are involved in various developmental and biological processes, and tend to be distributed in clusters on the genome.<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875880">24</a>,<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875867">25</a></sup> The evolutionarily conserved chr6p25 cluster comprises <em>FOXQ1, FOXF2</em>, and <em>FOXC1</em>. Our lead stroke risk variants lie between <em>FOXQ1</em> (22·4 kb) and <em>FOXF2</em> (52·7 kb). By contrast with the compelling experimental evidence for a central role of <em>FOXF2</em> in cerebrovascular disease, <em>FOXQ1</em> has not been implicated in cerebrovascular phenotypes; mutant mice for this gene show mainly altered hair differentiation and gastric mucin secretion.<sup><a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875888">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875883">27</a></sup> The third gene in the cluster, <em>FOXC1</em> (225 kb downstream of <em>FOXF2</em> and 273·3 kb downstream of stroke risk variants), is also expressed in the brain vasculature and influences vessel morphogenesis<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875874">28</a></sup> and arteriovenous specification.<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875861">29</a></sup> MRI analysis of patients with <em>FOXC1</em>-attributable Axenfeld-Rieger syndrome revealed MRI features of cerebral small vessel disease (including increased burden of white matter hyper intensity), and genetic variants downstream of <em>FOXC1</em> were associated with burden of white matter hyper intensity in the general population.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875856">20</a></sup> These previously described variants are independent (r<sup>2</sup>&lt;0·017) from the stroke risk variants near <em>FOXF2</em> described here.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875877">20</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875869">30</a></sup> These findings, together with the ten-times increased burden of white matter hyperintensity in patients with segmental deletion of both <em>FOXC1</em> and <em>FOXF2</em> versus <em>FOXC1</em> alone, suggest an independent role of <em>FOXF2</em> in cerebral small vessel disease. Differences in the roles of <em>foxf2</em> and <em>foxc1a</em> and <em>foxc1b</em> are also seen in the zebrafish model.<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875847">31</a></sup> Knockdown of zebrafish <em>foxc1a/b</em> leads to embryonic cerebral hemorrhage in embryos,<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875839">31</a></sup> whereas knockout of <em>foxf2b</em> at the same developmental stage does not. Thus, although the two genes are closely related, there are indications that their roles in vascular mural cells might be distinct.</p><p id="P31">Intriguingly, <em>PITX2</em> (chr4q25), a known risk locus for cardioembolic ischaemic stroke and atrial fibrillation<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875835">32</a>,<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875837">33</a></sup> as well as being genome-wide significant for cardioembolic ischaemic stroke in the present sample, encodes a neural crest–expressed transcription factor that physically interacts with FOXC1 and harbours causal mutations for Axenfeld-Rieger syndrome.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875842">20</a></sup> Variants near <em>PITX2</em> were associated with burden of white matter hyperintensity.<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875887">20</a>,<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875890">30</a></sup> In this study, common variants near <em>PITX2</em> were also associated with risk of intracerebral haemorrhage, of which the main mechanism in the general population is small vessel disease.<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875879">34</a></sup> These findings suggest that <em>FOXF2</em>, <em>FOXC1,</em> and <em>PITX2</em> could perhaps contribute to cerebrovascular disease via partly shared pathways, of relevance in cerebral small arteries.</p><p id="P32">The small number of incident strokes (n=4300), particularly stroke subtypes (n=602 for cardioembolic ischaemic stroke, whereas small-vessel occlusion, large-vessel ischaemic stroke, and other stroke subtypes had to be merged into a single category) might have hampered our ability to detect additional associations. The study was also underpowered (&lt;80% power) to detect associations with effect sizes smaller than OR 1·10 and allele frequency lower than 5% (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix p 34</a>). Although the results of the described animal studies suggest that <em>FOXF2</em> is the causal gene underlying the observed genetic association with stroke and small vessel disease at chr6p25, functional annotation of the identified risk variants is limited, with an absence of eQTL despite an extensive search of publicly available and other databases, possibly reflecting tissue specificity or primarily developmental effects. These effects are supported by a higher expression of <em>FOXF2</em> in fetal brain than adult brain, and lower methylation levels of the stroke risk locus in fetal tissues than adult tissues. Another limitation is that we only explored common variants and not rare variants in this region. In addition, we have used an additive genetic model only, which is the most powerful approach when the underlying genetic model is unknown, but we cannot exclude that associations with genetic risk loci following a recessive or dominant model might have been missed. Nevertheless, we confirmed and extended the range of associations for previously discovered stroke risk loci in a population-based sample. For the first time, to our knowledge, we describe shared genetic variation underlying both ischaemic stroke (chr4q25) and intracerebral haemorrhage (chr1q22), in agreement with some monogenic strokes having both ischaemic and haemorrhagic phenotypes, mostly with underlying small vessel disease.<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875855">35</a>–<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875866">37</a></sup> The association we previously described between chr12p13 and incident stroke in a smaller, overlapping sample was also the most significant association with ischaemic stroke in the present population-based GWAS,<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875852">5</a></sup> showing a stronger association with incident fatal stroke versus non-fatal stroke, which suggests an effect on stroke survival (<a href="#SD1" rid="SD1" class=" supplementary-material">appendix pp 58, 59</a>).</p><p id="P33" class="p p-last">In summary, we identified common variants near <em>FOXF2</em> associated with increased stroke susceptibility (especially of the small vessel subtype) and extensive subclinical small vessel disease. This association is particularly interesting because GWAS have not yet discovered risk loci for small-vessel ischaemic stroke (except for an association with the <em>PRKCH</em> locus identified in a study of Japanese participants, which was not found in European populations).<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875875">6</a>,<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_544875846">38</a></sup> Brain imaging data from patients with rare segmental deletions encompassing <em>FOXF2</em>, and functional experiments across evolutionarily separated species, suggest an important role of <em>FOXF2</em> in cerebrovascular disease—especially cerebral small vessel disease—possibly by affecting differentiation of cerebral vascular mural cells. Cerebral small vessel disease is a major, but poorly understood, cause of stroke in all ethnic groups, and subclinical small vessel disease has been associated with progressive functional and cognitive decline and increased risk of dementia. At present, no mechanism-based treatment for small vessel disease is available, other than management of vascular risk factors. Our findings provide promising grounds for follow-up, pointing to a possible mechanism of stroke and small vessel disease. Further research is warranted to explore whether these findings can have implications for clinical practice.</p></div><div id="S18" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d37e1133"><a href="/pmc/articles/PMC4943223/bin/NIHMS778629-supplement-1.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(2.6M, docx)</sup></div></div></div><div id="S19" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S19title">Acknowledgments</h2><div class="sec"><p id="__p6">Funding NIH, NINDS, NHMRC, CIHR, European national research institutions, Fondation Leducq.</p><p id="__p7">For information about the funding for each section, please see the <a href="#SD1" rid="SD1" class=" supplementary-material">appendix (pp 63–73)</a>.</p></div></div><div id="idm140271641188400" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271641188400title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN6"></p><p id="__p8" class="p p-first-last">See <a href="#SD1" rid="SD1" class=" supplementary-material">Online</a> for <a href="#SD1" rid="SD1" class=" supplementary-material">appendix</a></p><p></p><div><p id="__p9" class="p p-first"><strong>Contributors</strong></p>SD, SS, WTL, SJC, DIC, PC, MAI, and LJL jointly supervised research. GC, CRA, AYC, MF, AR, JCB, and ASH contributed equally. SD, SS, WTL, SJC, PC, JR, and BBW conceived and designed the experiments. GC, AYC, MF, JCB, ASH, MS, AVS, HHHA, SHC, SLP, ST, MEGa, AM, AT, SG, TMB, CeB, XJ, FX, YL, MdH, TP, CCW, SB, RM, QW, MT, and YK. did statistical analysis and other data analysis. MAI, LJL, KMR, CT, OJL, CMvD, EI, THM, VS, JR, BBW, COS, ALD, SSR, TBH, BMP, TK, VG, ML, TP, CCW, PLDJ, DAB, MD, CRF, EV, HSM, AB, BGW, DSK, MEGr, RFG, MAN, JIR, OLL, YK, JDa, RC, GBB, US, MRPM, HJG, ADJ, MMS, NBA, DJS, JWJ, IF, AJMdC, AGU, MLP, PJK, AHo, DW, AS, FR, JMo, JJ-C, APM, CML, LLi, LLa, NLP, MT, PKM, AHa, J-FD, ClB, PMRi, JEB, AP, DL, MK, STE, HJA, QW, J-ML, SW-S, ORB, JAJ, CJ, DKA, PMRo, TR, RLS, RPG, OM, AL, VT, JMe, RS, JWC, AMN, CLMS, PS, CL, NLS, KR, SRH, RM, JCH, SB, MdH, JDe, YL, SR, FX, CLS, KLW, OK, XJ, JSV, CeB, TMB, SG, AT, AM, MEGa, ST, SLP, SHC, HHHA, AVS, MS, ASH, JCB, AR, MF, AYC, and CRA revised the draft critically for important intellectual content. SJC, OJL, AMN, AR, and CRA. did the experiments. SD, SS, WTL, SJC, DIC, PC, and GC wrote the Article.</div><p class="fn sec" id="FN8"> </p><p id="__p11" class="p p-first"><strong>Declaration of interests</strong></p><p id="__p12" class="p p-last">We declare no competing interests.</p><p></p></div></div><div id="idm140271641181168" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140271641181168title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Johnston SC, Mendis S, Mathers CD. Global variation in stroke burden and mortality: estimates from monitoring, surveillance, and modelling. <span><span class="ref-journal">Lancet Neurol. </span>2009;<span class="ref-vol">8</span>:345–54.</span> [<a href="/pubmed/19233730" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Global+variation+in+stroke+burden+and+mortality:+estimates+from+monitoring,+surveillance,+and+modelling&amp;author=SC+Johnston&amp;author=S+Mendis&amp;author=CD+Mathers&amp;volume=8&amp;publication_year=2009&amp;pages=345-54&amp;pmid=19233730&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Falcone GJ, Malik R, Dichgans M, Rosand J. Current concepts and clinical applications of stroke genetics. <span><span class="ref-journal">Lancet Neurol. </span>2014;<span class="ref-vol">13</span>:405–18.</span> [<a href="/pubmed/24646874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Current+concepts+and+clinical+applications+of+stroke+genetics&amp;author=GJ+Falcone&amp;author=R+Malik&amp;author=M+Dichgans&amp;author=J+Rosand&amp;volume=13&amp;publication_year=2014&amp;pages=405-18&amp;pmid=24646874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Bevan S, Traylor M, Adib-Samii P, et al.  Genetic heritability of ischemic stroke and the contribution of previously reported candidate gene and genomewide associations. <span><span class="ref-journal">Stroke. </span>2012;<span class="ref-vol">43</span>:3161–67.</span> [<a href="/pubmed/23042660" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Genetic+heritability+of+ischemic+stroke+and+the+contribution+of+previously+reported+candidate+gene+and+genomewide+associations&amp;author=S+Bevan&amp;author=M+Traylor&amp;author=P+Adib-Samii&amp;volume=43&amp;publication_year=2012&amp;pages=3161-67&amp;pmid=23042660&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Manolio TA. Bringing genome-wide association findings into clinical use. <span><span class="ref-journal">Nat Rev Genet. </span>2013;<span class="ref-vol">14</span>:549–58.</span> [<a href="/pubmed/23835440" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Bringing+genome-wide+association+findings+into+clinical+use&amp;author=TA+Manolio&amp;volume=14&amp;publication_year=2013&amp;pages=549-58&amp;pmid=23835440&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Ikram MA, Seshadri S, Bis JC, et al.  Genomewide association studies of stroke. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">360</span>:1718–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2768348/">PMC free article</a>]</span> [<a href="/pubmed/19369658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomewide+association+studies+of+stroke&amp;author=MA+Ikram&amp;author=S+Seshadri&amp;author=JC+Bis&amp;volume=360&amp;publication_year=2009&amp;pages=1718-28&amp;pmid=19369658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Traylor M, Farrall M, Holliday EG, et al.  Genetic risk factors for ischaemic stroke and its subtypes (the METASTROKE collaboration): a meta-analysis of genome-wide association studies. <span><span class="ref-journal">Lancet Neurol. </span>2012;<span class="ref-vol">11</span>:951–62.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3490334/">PMC free article</a>]</span> [<a href="/pubmed/23041239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Genetic+risk+factors+for+ischaemic+stroke+and+its+subtypes+(the+METASTROKE+collaboration):+a+meta-analysis+of+genome-wide+association+studies&amp;author=M+Traylor&amp;author=M+Farrall&amp;author=EG+Holliday&amp;volume=11&amp;publication_year=2012&amp;pages=951-62&amp;pmid=23041239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Meschia JF, Arnett DK, Ay H, et al.  Stroke Genetics Network (SiGN) study: design and rationale for a genome-wide association study of ischemic stroke subtypes. <span><span class="ref-journal">Stroke. </span>2013;<span class="ref-vol">44</span>:2694–702.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4056331/">PMC free article</a>]</span> [<a href="/pubmed/24021684" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Stroke+Genetics+Network+(SiGN)+study:+design+and+rationale+for+a+genome-wide+association+study+of+ischemic+stroke+subtypes&amp;author=JF+Meschia&amp;author=DK+Arnett&amp;author=H+Ay&amp;volume=44&amp;publication_year=2013&amp;pages=2694-702&amp;pmid=24021684&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Pulit SL, McArdle PF, Wong Q, et al.  Loci associated with ischaemic stroke and its subtypes (SiGN): a genome-wide association study. <span><span class="ref-journal">Lancet Neurol. </span>2015;<span class="ref-vol">15</span>:174–84.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4912948/">PMC free article</a>]</span> [<a href="/pubmed/26708676" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Loci+associated+with+ischaemic+stroke+and+its+subtypes+(SiGN):+a+genome-wide+association+study&amp;author=SL+Pulit&amp;author=PF+McArdle&amp;author=Q+Wong&amp;volume=15&amp;publication_year=2015&amp;pages=174-84&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Klungel OH, Heckbert SR, Longstreth WT, Junior, et al.  Antihypertensive drug therapies and the risk of ischemic stroke. <span><span class="ref-journal">Arch Intern Med. </span>2001;<span class="ref-vol">161</span>:37–43.</span> [<a href="/pubmed/11146696" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=Antihypertensive+drug+therapies+and+the+risk+of+ischemic+stroke&amp;author=OH+Klungel&amp;author=SR+Heckbert&amp;author=WT+Longstreth&amp;volume=161&amp;publication_year=2001&amp;pages=37-43&amp;pmid=11146696&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Debette S, Kamatani Y, Metso TM, et al.  Common variation in PHACTR1 is associated with susceptibility to cervical artery dissection. <span><span class="ref-journal">Nat Genet. </span>2015;<span class="ref-vol">47</span>:78–83.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5824623/">PMC free article</a>]</span> [<a href="/pubmed/25420145" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variation+in+PHACTR1+is+associated+with+susceptibility+to+cervical+artery+dissection&amp;author=S+Debette&amp;author=Y+Kamatani&amp;author=TM+Metso&amp;volume=47&amp;publication_year=2015&amp;pages=78-83&amp;pmid=25420145&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Adams HP, Junior, Bendixen BH, Kappelle LJ, et al.  Classification of subtype of acute ischemic stroke. Definitions for use in a multicenter clinical trial. TOAST. Trial of Org 10172 in Acute Stroke Treatment. <span><span class="ref-journal">Stroke. </span>1993;<span class="ref-vol">24</span>:35–41.</span> [<a href="/pubmed/7678184" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=Classification+of+subtype+of+acute+ischemic+stroke.+Definitions+for+use+in+a+multicenter+clinical+trial.+TOAST.+Trial+of+Org+10172+in+Acute+Stroke+Treatment&amp;author=HP+Adams&amp;author=BH+Bendixen&amp;author=LJ+Kappelle&amp;volume=24&amp;publication_year=1993&amp;pages=35-41&amp;pmid=7678184&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Verhaaren BF, Debette S, Bis JC, et al.  Multi-ethnic genome-wide association study of cerebral white matter hyperintensities on MRI. <span><span class="ref-journal">Circ Cardiovasc Genet. </span>2015;<span class="ref-vol">8</span>:398–409.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4427240/">PMC free article</a>]</span> [<a href="/pubmed/25663218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ+Cardiovasc+Genet&amp;title=Multi-ethnic+genome-wide+association+study+of+cerebral+white+matter+hyperintensities+on+MRI&amp;author=BF+Verhaaren&amp;author=S+Debette&amp;author=JC+Bis&amp;volume=8&amp;publication_year=2015&amp;pages=398-409&amp;pmid=25663218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Woo D, Falcone GJ, Devan WJ, et al.  Meta-analysis of genome-wide association studies identifies 1q22 as a susceptibility locus for intracerebral hemorrhage. <span><span class="ref-journal">Am J Hum Genet. </span>2014;<span class="ref-vol">94</span>:511–21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3980413/">PMC free article</a>]</span> [<a href="/pubmed/24656865" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+1q22+as+a+susceptibility+locus+for+intracerebral+hemorrhage&amp;author=D+Woo&amp;author=GJ+Falcone&amp;author=WJ+Devan&amp;volume=94&amp;publication_year=2014&amp;pages=511-21&amp;pmid=24656865&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Reyahi A, Nik AM, Ghiami M, et al.  Foxf2 is required for brain pericyte differentiation and development and maintenance of the blood-brain barrier. <span><span class="ref-journal">Dev Cell. </span>2015;<span class="ref-vol">34</span>:19–32.</span> [<a href="/pubmed/26120030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Cell&amp;title=Foxf2+is+required+for+brain+pericyte+differentiation+and+development+and+maintenance+of+the+blood-brain+barrier&amp;author=A+Reyahi&amp;author=AM+Nik&amp;author=M+Ghiami&amp;volume=34&amp;publication_year=2015&amp;pages=19-32&amp;pmid=26120030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Arnold CR, Lamont RE, Walker JT, et al.  Comparative analysis of genes regulated by Dzip1/iguana and hedgehog in zebrafish. <span><span class="ref-journal">Dev Dyn. </span>2015;<span class="ref-vol">244</span>:211–23.</span> [<a href="/pubmed/25476803" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Dyn&amp;title=Comparative+analysis+of+genes+regulated+by+Dzip1/iguana+and+hedgehog+in+zebrafish&amp;author=CR+Arnold&amp;author=RE+Lamont&amp;author=JT+Walker&amp;volume=244&amp;publication_year=2015&amp;pages=211-23&amp;pmid=25476803&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Ramasamy A, Trabzuni D, Guelfi S, et al.  Genetic variability in the regulation of gene expression in ten regions of the human brain. <span><span class="ref-journal">Nat Neurosci. </span>2014;<span class="ref-vol">17</span>:1418–28.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4208299/">PMC free article</a>]</span> [<a href="/pubmed/25174004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Neurosci&amp;title=Genetic+variability+in+the+regulation+of+gene+expression+in+ten+regions+of+the+human+brain&amp;author=A+Ramasamy&amp;author=D+Trabzuni&amp;author=S+Guelfi&amp;volume=17&amp;publication_year=2014&amp;pages=1418-28&amp;pmid=25174004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Kozomara A, Griffiths-Jones S. miRBase: annotating high confidence microRNAs using deep sequencing data. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D68–73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965103/">PMC free article</a>]</span> [<a href="/pubmed/24275495" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=miRBase:+annotating+high+confidence+microRNAs+using+deep+sequencing+data&amp;author=A+Kozomara&amp;author=S+Griffiths-Jones&amp;volume=42&amp;publication_year=2014&amp;pages=D68-73&amp;pmid=24275495&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Yu L, Chibnik LB, Srivastava GP, et al.  Association of brain DNA methylation in SORL1, ABCA7, HLA-DRB5, SLC24A4, and BIN1 with pathological diagnosis of Alzheimer disease. <span><span class="ref-journal">JAMA Neurol. </span>2015;<span class="ref-vol">72</span>:15–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4344367/">PMC free article</a>]</span> [<a href="/pubmed/25365775" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=Association+of+brain+DNA+methylation+in+SORL1,+ABCA7,+HLA-DRB5,+SLC24A4,+and+BIN1+with+pathological+diagnosis+of+Alzheimer+disease&amp;author=L+Yu&amp;author=LB+Chibnik&amp;author=GP+Srivastava&amp;volume=72&amp;publication_year=2015&amp;pages=15-24&amp;pmid=25365775&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Ziller MJ, Gu H, Muller F, et al.  Charting a dynamic DNA methylation landscape of the human genome. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">500</span>:477–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3821869/">PMC free article</a>]</span> [<a href="/pubmed/23925113" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Charting+a+dynamic+DNA+methylation+landscape+of+the+human+genome&amp;author=MJ+Ziller&amp;author=H+Gu&amp;author=F+Muller&amp;volume=500&amp;publication_year=2013&amp;pages=477-81&amp;pmid=23925113&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">French CR, Seshadri S, Destefano AL, et al.  Mutation of FOXC1 and PITX2 induces cerebral small-vessel disease. <span><span class="ref-journal">J Clin Invest. </span>2014;<span class="ref-vol">124</span>:4877–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4347243/">PMC free article</a>]</span> [<a href="/pubmed/25250569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Mutation+of+FOXC1+and+PITX2+induces+cerebral+small-vessel+disease&amp;author=CR+French&amp;author=S+Seshadri&amp;author=AL+Destefano&amp;volume=124&amp;publication_year=2014&amp;pages=4877-81&amp;pmid=25250569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Kim BS, Illes J, Kaplan RT, et al.  Incidental findings on pediatric MR images of the brain. <span><span class="ref-journal">Am J Neuroradiol. </span>2002;<span class="ref-vol">23</span>:1674–77.</span> [<a href="/pubmed/12427622" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Neuroradiol&amp;title=Incidental+findings+on+pediatric+MR+images+of+the+brain&amp;author=BS+Kim&amp;author=J+Illes&amp;author=RT+Kaplan&amp;volume=23&amp;publication_year=2002&amp;pages=1674-77&amp;pmid=12427622&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Wardlaw JM, Smith C, Dichgans M. Mechanisms of sporadic cerebral small vessel disease: insights from neuroimaging. <span><span class="ref-journal">Lancet Neurol. </span>2013;<span class="ref-vol">12</span>:483–97.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3836247/">PMC free article</a>]</span> [<a href="/pubmed/23602162" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Mechanisms+of+sporadic+cerebral+small+vessel+disease:+insights+from+neuroimaging&amp;author=JM+Wardlaw&amp;author=C+Smith&amp;author=M+Dichgans&amp;volume=12&amp;publication_year=2013&amp;pages=483-97&amp;pmid=23602162&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Bolte C, Ren X, Tomley T, et al.  Forkhead box F2 regulation of platelet-derived growth factor and myocardin/serum response factor signaling is essential for intestinal development. <span><span class="ref-journal">J Biol Chem. </span>2015;<span class="ref-vol">290</span>:7563–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4367262/">PMC free article</a>]</span> [<a href="/pubmed/25631042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Forkhead+box+F2+regulation+of+platelet-derived+growth+factor+and+myocardin/serum+response+factor+signaling+is+essential+for+intestinal+development&amp;author=C+Bolte&amp;author=X+Ren&amp;author=T+Tomley&amp;volume=290&amp;publication_year=2015&amp;pages=7563-75&amp;pmid=25631042&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Lehmann OJ, Sowden JC, Carlsson P, Jordan T, Bhattacharya SS. Fox’s in development and disease. <span><span class="ref-journal">Trends Genet. </span>2003;<span class="ref-vol">19</span>:339–44.</span> [<a href="/pubmed/12801727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet&amp;title=Fox’s+in+development+and+disease&amp;author=OJ+Lehmann&amp;author=JC+Sowden&amp;author=P+Carlsson&amp;author=T+Jordan&amp;author=SS+Bhattacharya&amp;volume=19&amp;publication_year=2003&amp;pages=339-44&amp;pmid=12801727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Carlsson P, Mahlapuu M. Forkhead transcription factors: key players in development and metabolism. <span><span class="ref-journal">Dev Biol. </span>2002;<span class="ref-vol">250</span>:1–23.</span> [<a href="/pubmed/12297093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Biol&amp;title=Forkhead+transcription+factors:+key+players+in+development+and+metabolism&amp;author=P+Carlsson&amp;author=M+Mahlapuu&amp;volume=250&amp;publication_year=2002&amp;pages=1-23&amp;pmid=12297093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Hong HK, Noveroske JK, Headon DJ, et al.  The winged helix/forkhead transcription factor Foxq1 regulates differentiation of hair in satin mice. <span><span class="ref-journal">Genesis. </span>2001;<span class="ref-vol">29</span>:163–71.</span> [<a href="/pubmed/11309849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genesis&amp;title=The+winged+helix/forkhead+transcription+factor+Foxq1+regulates+differentiation+of+hair+in+satin+mice&amp;author=HK+Hong&amp;author=JK+Noveroske&amp;author=DJ+Headon&amp;volume=29&amp;publication_year=2001&amp;pages=163-71&amp;pmid=11309849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Verzi MP, Khan AH, Ito S, Shivdasani RA. Transcription factor foxq1 controls mucin gene expression and granule content in mouse stomach surface mucous cells. <span><span class="ref-journal">Gastroenterology. </span>2008;<span class="ref-vol">135</span>:591–600.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2955860/">PMC free article</a>]</span> [<a href="/pubmed/18558092" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Transcription+factor+foxq1+controls+mucin+gene+expression+and+granule+content+in+mouse+stomach+surface+mucous+cells&amp;author=MP+Verzi&amp;author=AH+Khan&amp;author=S+Ito&amp;author=RA+Shivdasani&amp;volume=135&amp;publication_year=2008&amp;pages=591-600&amp;pmid=18558092&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Siegenthaler JA, Choe Y, Patterson KP, et al.  Foxc1 is required by pericytes during fetal brain angiogenesis. <span><span class="ref-journal">Biol Open. </span>2013;<span class="ref-vol">2</span>:647–59.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3711032/">PMC free article</a>]</span> [<a href="/pubmed/23862012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Open&amp;title=Foxc1+is+required+by+pericytes+during+fetal+brain+angiogenesis&amp;author=JA+Siegenthaler&amp;author=Y+Choe&amp;author=KP+Patterson&amp;volume=2&amp;publication_year=2013&amp;pages=647-59&amp;pmid=23862012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Fish JE, Wythe JD. The molecular regulation of arteriovenous specification and maintenance. <span><span class="ref-journal">Dev Dyn. </span>2015;<span class="ref-vol">244</span>:391–409.</span> [<a href="/pubmed/25641373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev+Dyn&amp;title=The+molecular+regulation+of+arteriovenous+specification+and+maintenance&amp;author=JE+Fish&amp;author=JD+Wythe&amp;volume=244&amp;publication_year=2015&amp;pages=391-409&amp;pmid=25641373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Fornage M, Debette S, Bis JC, et al.  Genome-wide association studies of cerebral white matter lesion burden: the CHARGE consortium. <span><span class="ref-journal">Ann Neurol. </span>2011;<span class="ref-vol">69</span>:928–39.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3122147/">PMC free article</a>]</span> [<a href="/pubmed/21681796" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Genome-wide+association+studies+of+cerebral+white+matter+lesion+burden:+the+CHARGE+consortium&amp;author=M+Fornage&amp;author=S+Debette&amp;author=JC+Bis&amp;volume=69&amp;publication_year=2011&amp;pages=928-39&amp;pmid=21681796&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">French CR, Seshadri S, Destefano AL, et al.  Mutation of FOXC1 and PITX2 induces cerebral small-vessel disease. <span><span class="ref-journal">J Clin Invest. </span>2014;<span class="ref-vol">124</span>:4877–81.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4347243/">PMC free article</a>]</span> [<a href="/pubmed/25250569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Mutation+of+FOXC1+and+PITX2+induces+cerebral+small-vessel+disease&amp;author=CR+French&amp;author=S+Seshadri&amp;author=AL+Destefano&amp;volume=124&amp;publication_year=2014&amp;pages=4877-81&amp;pmid=25250569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Bellenguez C, Bevan S, Gschwendtner A, et al.  Genome-wide association study identifies a variant in HDAC9 associated with large vessel ischemic stroke. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:328–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3303115/">PMC free article</a>]</span> [<a href="/pubmed/22306652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+a+variant+in+HDAC9+associated+with+large+vessel+ischemic+stroke&amp;author=C+Bellenguez&amp;author=S+Bevan&amp;author=A+Gschwendtner&amp;volume=44&amp;publication_year=2012&amp;pages=328-33&amp;pmid=22306652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Gudbjartsson DF, Arnar DO, Helgadottir A, et al.  Variants conferring risk of atrial fibrillation on chromosome 4q25. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:353–57.</span> [<a href="/pubmed/17603472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Variants+conferring+risk+of+atrial+fibrillation+on+chromosome+4q25&amp;author=DF+Gudbjartsson&amp;author=DO+Arnar&amp;author=A+Helgadottir&amp;volume=448&amp;publication_year=2007&amp;pages=353-57&amp;pmid=17603472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Greenberg SM. Small vessels, big problems. <span><span class="ref-journal">N Engl J Med. </span>2006;<span class="ref-vol">354</span>:1451–53.</span> [<a href="/pubmed/16598043" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Small+vessels,+big+problems&amp;author=SM+Greenberg&amp;volume=354&amp;publication_year=2006&amp;pages=1451-53&amp;pmid=16598043&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Zhou Q, Yang D, Ombrello AK, et al.  Early-onset stroke and vasculopathy associated with mutations in ADA2. <span><span class="ref-journal">N Engl J Med. </span>2014;<span class="ref-vol">370</span>:911–20.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4193683/">PMC free article</a>]</span> [<a href="/pubmed/24552284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Early-onset+stroke+and+vasculopathy+associated+with+mutations+in+ADA2&amp;author=Q+Zhou&amp;author=D+Yang&amp;author=AK+Ombrello&amp;volume=370&amp;publication_year=2014&amp;pages=911-20&amp;pmid=24552284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Sibon I, Coupry I, Menegon P, et al.  COL4A1 mutation in Axenfeld-Rieger anomaly with leukoencephalopathy and stroke. <span><span class="ref-journal">Ann Neurol. </span>2007;<span class="ref-vol">62</span>:177–84.</span> [<a href="/pubmed/17696175" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=COL4A1+mutation+in+Axenfeld-Rieger+anomaly+with+leukoencephalopathy+and+stroke&amp;author=I+Sibon&amp;author=I+Coupry&amp;author=P+Menegon&amp;volume=62&amp;publication_year=2007&amp;pages=177-84&amp;pmid=17696175&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Lanfranconi S, Markus HS. COL4A1 mutations as a monogenic cause of cerebral small vessel disease: a systematic review. <span><span class="ref-journal">Stroke. </span>2010;<span class="ref-vol">41</span>:e513–18.</span> [<a href="/pubmed/20558831" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stroke&amp;title=COL4A1+mutations+as+a+monogenic+cause+of+cerebral+small+vessel+disease:+a+systematic+review&amp;author=S+Lanfranconi&amp;author=HS+Markus&amp;volume=41&amp;publication_year=2010&amp;pages=e513-18&amp;pmid=20558831&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Kubo M, Hata J, Ninomiya T, et al.  A nonsynonymous SNP in PRKCH (protein kinase C eta) increases the risk of cerebral infarction. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:212–17.</span> [<a href="/pubmed/17206144" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+nonsynonymous+SNP+in+PRKCH+(protein+kinase+C+eta)+increases+the+risk+of+cerebral+infarction&amp;author=M+Kubo&amp;author=J+Hata&amp;author=T+Ninomiya&amp;volume=39&amp;publication_year=2007&amp;pages=212-17&amp;pmid=17206144&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4943223&amp;issue-id=272449&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140271718249888title">Summary</a></li><li><a href="#S5title">Introduction</a></li><li><a href="#S6title">Methods</a></li><li><a href="#S15title">Results</a></li><li><a href="#S16title">Discussion</a></li><li><a href="#S18title">Supplementary Material</a></li><li><a href="#S19title">Acknowledgments</a></li><li><a href="#idm140271641188400title">Footnotes</a></li><li><a href="#idm140271641181168title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4943223/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4943223/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4943223/pdf/nihms778629.pdf">PDF (1.0M)</a></li> | <li><a href="#" data-citationid="PMC4943223" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4943223%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4943223%2F&amp;text=Identification%20of%20additional%20risk%20loci%20for%20stroke%20and%20small%20vessel%20disease%3A%20a%20meta-analysis%20of%20genome-wide%20association%20studies" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4943223%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="27068588" data-aiid="4943223" data-aid="4943223" data-iid="272449" data-domainid="319" data-domain="nihpa" data-accid="PMC4943223" data-md5="c55b8069239cfc361fdd92a95fd05183"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:38:02-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8900A6DD58893100000000089C0549&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4943223%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4943223/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>