<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="ajhg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4678416/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4678416/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture"><meta name="citation_authors" content="Philip&nbsp;E. Stuart, Rajan&nbsp;P. Nair, Lam&nbsp;C. Tsoi, Trilokraj Tejasvi, Sayantan Das, Hyun&nbsp;Min Kang, Eva Ellinghaus, Vinod Chandran, Kristina Callis-Duffin, Robert Ike, Yanming Li, Xiaoquan Wen, Charlotta Enerbäck, Johann&nbsp;E. Gudjonsson, Sulev Kõks, Külli Kingo, Tõnu Esko, Ulrich Mrowietz, Andre Reis, H.&nbsp;Erich Wichmann, Christian Gieger, Per Hoffmann, Markus&nbsp;M. Nöthen, Juliane Winkelmann, Manfred Kunz, Elvia&nbsp;G. Moreta, Philip&nbsp;J. Mease, Christopher&nbsp;T. Ritchlin, Anne&nbsp;M. Bowcock, Gerald&nbsp;G. Krueger, Henry&nbsp;W. Lim, Stephan Weidinger, Michael Weichenthal, John&nbsp;J. Voorhees, Proton Rahman, Peter&nbsp;K. Gregersen, Andre Franke, Dafna&nbsp;D. Gladman, Gonçalo&nbsp;R. Abecasis, James&nbsp;T. Elder"><meta name="citation_date" content="3 December 2015"><meta name="citation_issue" content="6"><meta name="citation_volume" content="97"><meta name="citation_firstpage" content="816"><meta name="citation_doi" content="10.1016/j.ajhg.2015.10.019"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4678416/?report=abstract"><meta name="citation_pmid" content="26626624"><meta name="DC.Title" content="Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Elsevier"><meta name="DC.Contributor" content="Philip&nbsp;E. Stuart"><meta name="DC.Contributor" content="Rajan&nbsp;P. Nair"><meta name="DC.Contributor" content="Lam&nbsp;C. Tsoi"><meta name="DC.Contributor" content="Trilokraj Tejasvi"><meta name="DC.Contributor" content="Sayantan Das"><meta name="DC.Contributor" content="Hyun&nbsp;Min Kang"><meta name="DC.Contributor" content="Eva Ellinghaus"><meta name="DC.Contributor" content="Vinod Chandran"><meta name="DC.Contributor" content="Kristina Callis-Duffin"><meta name="DC.Contributor" content="Robert Ike"><meta name="DC.Contributor" content="Yanming Li"><meta name="DC.Contributor" content="Xiaoquan Wen"><meta name="DC.Contributor" content="Charlotta Enerbäck"><meta name="DC.Contributor" content="Johann&nbsp;E. Gudjonsson"><meta name="DC.Contributor" content="Sulev Kõks"><meta name="DC.Contributor" content="Külli Kingo"><meta name="DC.Contributor" content="Tõnu Esko"><meta name="DC.Contributor" content="Ulrich Mrowietz"><meta name="DC.Contributor" content="Andre Reis"><meta name="DC.Contributor" content="H.&nbsp;Erich Wichmann"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Per Hoffmann"><meta name="DC.Contributor" content="Markus&nbsp;M. Nöthen"><meta name="DC.Contributor" content="Juliane Winkelmann"><meta name="DC.Contributor" content="Manfred Kunz"><meta name="DC.Contributor" content="Elvia&nbsp;G. Moreta"><meta name="DC.Contributor" content="Philip&nbsp;J. Mease"><meta name="DC.Contributor" content="Christopher&nbsp;T. Ritchlin"><meta name="DC.Contributor" content="Anne&nbsp;M. Bowcock"><meta name="DC.Contributor" content="Gerald&nbsp;G. Krueger"><meta name="DC.Contributor" content="Henry&nbsp;W. Lim"><meta name="DC.Contributor" content="Stephan Weidinger"><meta name="DC.Contributor" content="Michael Weichenthal"><meta name="DC.Contributor" content="John&nbsp;J. Voorhees"><meta name="DC.Contributor" content="Proton Rahman"><meta name="DC.Contributor" content="Peter&nbsp;K. Gregersen"><meta name="DC.Contributor" content="Andre Franke"><meta name="DC.Contributor" content="Dafna&nbsp;D. Gladman"><meta name="DC.Contributor" content="Gonçalo&nbsp;R. Abecasis"><meta name="DC.Contributor" content="James&nbsp;T. Elder"><meta name="DC.Date" content="2015 Dec 3"><meta name="DC.Identifier" content="10.1016/j.ajhg.2015.10.019"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture"><meta property="og:type" content="article"><meta property="og:description" content="Psoriasis vulgaris (PsV) is a common inflammatory and hyperproliferative skin disease. Up to 30% of people with PsV eventually develop psoriatic arthritis (PsA), an inflammatory musculoskeletal condition. To discern differences in genetic risk factors ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4678416/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-ajhg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4678416/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4678416/pdf/main.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D3762DD58C3C100000000035D0202.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4678416%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="mesh" class="last">MeSH</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/203/">Am J Hum Genet</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/261962/">v.97(6); 2015 Dec 3</a></li><li class="accid">PMC4678416</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,30,118,50" alt="Guide for Authors" title="Guide for Authors" href="http://www.elsevier.com/journals/The-American-Journal-of-Human-Genetics/0002-9297/guide-for-authors" ref="reftype=publisher&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="118,30,165,50" alt="About this journal" title="About this journal" href="http://www.journals.elsevier.com/The-American-Journal-of-Human-Genetics" ref="reftype=publisher&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,30,286,50" alt="Explore this journal" title="Explore this journal" href="http://www.sciencedirect.com/science/journal/00029297" ref="reftype=publisher&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="American Journal of Human Genetics" title="American Journal of Human Genetics" href="http://www.cell.com/AJHG/" ref="reftype=publisher&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hum Genet</a></span>. 2015 Dec 3; 97(6): 816–836. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Nov 28. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1016%2Fj.ajhg.2015.10.019" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.ajhg.2015.10.019</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4678416</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26626624">26626624</a></div></div></div></div><h1 class="content-title">Genome-wide Association Analysis of Psoriatic Arthritis and Cutaneous Psoriasis Reveals Differences in Their Genetic Architecture</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913883607952" co-class="co-affbox">Philip&nbsp;E. Stuart</a>,<sup>1,</sup><sup>34</sup> <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913883605824" co-class="co-affbox">Rajan&nbsp;P. Nair</a>,<sup>1,</sup><sup>34</sup> <a href="/pubmed/?term=Tsoi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926551808" co-class="co-affbox">Lam&nbsp;C. Tsoi</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>34</sup> <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926547616" co-class="co-affbox">Trilokraj Tejasvi</a>,<sup>1,</sup><sup>4</sup> <a href="/pubmed/?term=Das%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926736576" co-class="co-affbox">Sayantan Das</a>,<sup>2</sup> <a href="/pubmed/?term=Kang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926734592" co-class="co-affbox">Hyun&nbsp;Min Kang</a>,<sup>2</sup> <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964577392" co-class="co-affbox">Eva Ellinghaus</a>,<sup>5</sup> <a href="/pubmed/?term=Chandran%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964575408" co-class="co-affbox">Vinod Chandran</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Callis-Duffin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964572688" co-class="co-affbox">Kristina Callis-Duffin</a>,<sup>8</sup> <a href="/pubmed/?term=Ike%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963662384" co-class="co-affbox">Robert Ike</a>,<sup>9</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963660400" co-class="co-affbox">Yanming Li</a>,<sup>2</sup> <a href="/pubmed/?term=Wen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963658416" co-class="co-affbox">Xiaoquan Wen</a>,<sup>2</sup> <a href="/pubmed/?term=Enerb%26%23x000e4%3Bck%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886644992" co-class="co-affbox">Charlotta Enerbäck</a>,<sup>10</sup> <a href="/pubmed/?term=Gudjonsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886643008" co-class="co-affbox">Johann&nbsp;E. Gudjonsson</a>,<sup>1</sup> <a href="/pubmed/?term=K%26%23x000f5%3Bks%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886641024" co-class="co-affbox">Sulev Kõks</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Kingo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913888959952" co-class="co-affbox">Külli Kingo</a>,<sup>13</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913888957968" co-class="co-affbox">Tõnu Esko</a>,<sup>14</sup> <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913888955984" co-class="co-affbox">Ulrich Mrowietz</a>,<sup>15</sup> <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913961955312" co-class="co-affbox">Andre Reis</a>,<sup>16</sup> <a href="/pubmed/?term=Wichmann%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913961953328" co-class="co-affbox">H.&nbsp;Erich Wichmann</a>,<sup>17,</sup><sup>18,</sup><sup>19</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913927064368" co-class="co-affbox">Christian Gieger</a>,<sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=Hoffmann%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913927061648" co-class="co-affbox">Per Hoffmann</a>,<sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964614352" co-class="co-affbox">Markus&nbsp;M. Nöthen</a>,<sup>22,</sup><sup>23</sup> <a href="/pubmed/?term=Winkelmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964611632" co-class="co-affbox">Juliane Winkelmann</a>,<sup>24,</sup><sup>25</sup> <a href="/pubmed/?term=Kunz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913964608912" co-class="co-affbox">Manfred Kunz</a>,<sup>26</sup> <a href="/pubmed/?term=Moreta%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963993856" co-class="co-affbox">Elvia&nbsp;G. Moreta</a>,<sup>27</sup> <a href="/pubmed/?term=Mease%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963991872" co-class="co-affbox">Philip&nbsp;J. Mease</a>,<sup>28</sup> <a href="/pubmed/?term=Ritchlin%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913963989888" co-class="co-affbox">Christopher&nbsp;T. Ritchlin</a>,<sup>29</sup> <a href="/pubmed/?term=Bowcock%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926702288" co-class="co-affbox">Anne&nbsp;M. Bowcock</a>,<sup>30</sup> <a href="/pubmed/?term=Krueger%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926700304" co-class="co-affbox">Gerald&nbsp;G. Krueger</a>,<sup>8</sup> <a href="/pubmed/?term=Lim%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913926698320" co-class="co-affbox">Henry&nbsp;W. Lim</a>,<sup>31</sup> <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913885060000" co-class="co-affbox">Stephan Weidinger</a>,<sup>15</sup> <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913885058016" co-class="co-affbox">Michael Weichenthal</a>,<sup>15</sup> <a href="/pubmed/?term=Voorhees%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913885056032" co-class="co-affbox">John&nbsp;J. Voorhees</a>,<sup>1</sup> <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886652432" co-class="co-affbox">Proton Rahman</a>,<sup>32</sup> <a href="/pubmed/?term=Gregersen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886650448" co-class="co-affbox">Peter&nbsp;K. Gregersen</a>,<sup>33</sup> <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913886648464" co-class="co-affbox">Andre Franke</a>,<sup>5</sup> <a href="/pubmed/?term=Gladman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913891535616" co-class="co-affbox">Dafna&nbsp;D. Gladman</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Abecasis%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913891532896" co-class="co-affbox">Gonçalo&nbsp;R. Abecasis</a>,<sup>2</sup> and  <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624" class="affpopup" co-rid="_co_idm139913891530912" co-class="co-affbox">James&nbsp;T. Elder</a><sup>1,</sup><sup>4,</sup><sup>∗</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139913883607952"><h3 class="no_margin">Philip&nbsp;E. Stuart</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Stuart%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Philip&nbsp;E. Stuart</a></div></div><div id="_co_idm139913883605824"><h3 class="no_margin">Rajan&nbsp;P. Nair</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Nair%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Rajan&nbsp;P. Nair</a></div></div><div id="_co_idm139913926551808"><h3 class="no_margin">Lam&nbsp;C. Tsoi</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><p><sup>3</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Tsoi%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Lam&nbsp;C. Tsoi</a></div></div><div id="_co_idm139913926547616"><h3 class="no_margin">Trilokraj Tejasvi</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><p><sup>4</sup>Ann Arbor Veterans Affairs Hospital, Ann Arbor, MI 48105, USA</p><div>Find articles by <a href="/pubmed/?term=Tejasvi%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Trilokraj Tejasvi</a></div></div><div id="_co_idm139913926736576"><h3 class="no_margin">Sayantan Das</h3><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Das%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Sayantan Das</a></div></div><div id="_co_idm139913926734592"><h3 class="no_margin">Hyun&nbsp;Min Kang</h3><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Kang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Hyun&nbsp;Min Kang</a></div></div><div id="_co_idm139913964577392"><h3 class="no_margin">Eva Ellinghaus</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University&nbsp;of Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Ellinghaus%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Eva Ellinghaus</a></div></div><div id="_co_idm139913964575408"><h3 class="no_margin">Vinod Chandran</h3><p><sup>6</sup>Department of Medicine, Division of Rheumatology, University of Toronto, Toronto, ON M5T 2S8, Canada</p><p><sup>7</sup>Centre for Prognosis Studies in the Rheumatic Diseases, Toronto Western Research Institute, University of Toronto, Toronto, ON M5T 2S8, Canada</p><div>Find articles by <a href="/pubmed/?term=Chandran%20V%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Vinod Chandran</a></div></div><div id="_co_idm139913964572688"><h3 class="no_margin">Kristina Callis-Duffin</h3><p><sup>8</sup>Department of Dermatology, University of Utah, Salt Lake City, UT 84132, USA</p><div>Find articles by <a href="/pubmed/?term=Callis-Duffin%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Kristina Callis-Duffin</a></div></div><div id="_co_idm139913963662384"><h3 class="no_margin">Robert Ike</h3><p><sup>9</sup>Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Ike%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Robert Ike</a></div></div><div id="_co_idm139913963660400"><h3 class="no_margin">Yanming Li</h3><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Yanming Li</a></div></div><div id="_co_idm139913963658416"><h3 class="no_margin">Xiaoquan Wen</h3><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Wen%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Xiaoquan Wen</a></div></div><div id="_co_idm139913886644992"><h3 class="no_margin">Charlotta Enerbäck</h3><p><sup>10</sup>Department of Clinical and Experimental Medicine, Division of Cell Biology and Dermatology, Linköping University, 581 83 Linköping, Sweden</p><div>Find articles by <a href="/pubmed/?term=Enerb%26%23x000e4%3Bck%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Charlotta Enerbäck</a></div></div><div id="_co_idm139913886643008"><h3 class="no_margin">Johann&nbsp;E. Gudjonsson</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Gudjonsson%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Johann&nbsp;E. Gudjonsson</a></div></div><div id="_co_idm139913886641024"><h3 class="no_margin">Sulev Kõks</h3><p><sup>11</sup>Department of Pathophysiology, Centre of Translational Medicine and Centre for Translational Genomics, University of Tartu, 50411 Tartu, Estonia</p><p><sup>12</sup>Department of Reproductive Biology, Estonian University of Life Sciences, 51014 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=K%26%23x000f5%3Bks%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Sulev Kõks</a></div></div><div id="_co_idm139913888959952"><h3 class="no_margin">Külli Kingo</h3><p><sup>13</sup>Dermatology Clinic, Tartu University Hospital, Department of Dermatology and Venereology, University of Tartu, 50417 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Kingo%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Külli Kingo</a></div></div><div id="_co_idm139913888957968"><h3 class="no_margin">Tõnu Esko</h3><p><sup>14</sup>Estonian Genome Center, University of Tartu, 51010 Tartu, Estonia</p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Tõnu Esko</a></div></div><div id="_co_idm139913888955984"><h3 class="no_margin">Ulrich Mrowietz</h3><p><sup>15</sup>Department of Dermatology, University Medical Center Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Mrowietz%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Ulrich Mrowietz</a></div></div><div id="_co_idm139913961955312"><h3 class="no_margin">Andre Reis</h3><p><sup>16</sup>Institute of Human Genetics, University of Erlangen-Nuremberg, 91054 Erlangen, Germany</p><div>Find articles by <a href="/pubmed/?term=Reis%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Andre Reis</a></div></div><div id="_co_idm139913961953328"><h3 class="no_margin">H.&nbsp;Erich Wichmann</h3><p><sup>17</sup>Institute of Epidemiology I, Helmholtz Zentrum Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>18</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</p><p><sup>19</sup>Institute of Medical Statistics and Epidemiology, Technical University Munich, 80333 Munich, Germany</p><div>Find articles by <a href="/pubmed/?term=Wichmann%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">H.&nbsp;Erich Wichmann</a></div></div><div id="_co_idm139913927064368"><h3 class="no_margin">Christian Gieger</h3><p><sup>20</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><p><sup>21</sup>Institute of Epidemiology II, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Christian Gieger</a></div></div><div id="_co_idm139913927061648"><h3 class="no_margin">Per Hoffmann</h3><p><sup>22</sup>Institute of Human Genetics, University of Bonn, 53127 Bonn, Germany</p><p><sup>23</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, 53127 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Hoffmann%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Per Hoffmann</a></div></div><div id="_co_idm139913964614352"><h3 class="no_margin">Markus&nbsp;M. Nöthen</h3><p><sup>22</sup>Institute of Human Genetics, University of Bonn, 53127 Bonn, Germany</p><p><sup>23</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, 53127 Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=N%26%23x000f6%3Bthen%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Markus&nbsp;M. Nöthen</a></div></div><div id="_co_idm139913964611632"><h3 class="no_margin">Juliane Winkelmann</h3><p><sup>24</sup>Neurologische Klinik and Poliklinik, Klinikum rechts der Isar, Technische Universität München, 80333 Munich, Germany</p><p><sup>25</sup>Institute of Neurogenomics, Helmholtz Zentrum Munchen, German Research Center for Environmental Health, 85764 Neuherberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Winkelmann%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Juliane Winkelmann</a></div></div><div id="_co_idm139913964608912"><h3 class="no_margin">Manfred Kunz</h3><p><sup>26</sup>Department of Dermatology, Venereology and Allergology, University of Leipzig, 04103 Leipzig, Germany</p><div>Find articles by <a href="/pubmed/?term=Kunz%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Manfred Kunz</a></div></div><div id="_co_idm139913963993856"><h3 class="no_margin">Elvia&nbsp;G. Moreta</h3><p><sup>27</sup>Saint Paul Rheumatology, Eagan, MN 55121, USA</p><div>Find articles by <a href="/pubmed/?term=Moreta%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Elvia&nbsp;G. Moreta</a></div></div><div id="_co_idm139913963991872"><h3 class="no_margin">Philip&nbsp;J. Mease</h3><p><sup>28</sup>Seattle Rheumatology Associates, 601 Broadway, Suite 600, Seattle, WA 98122, USA</p><div>Find articles by <a href="/pubmed/?term=Mease%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Philip&nbsp;J. Mease</a></div></div><div id="_co_idm139913963989888"><h3 class="no_margin">Christopher&nbsp;T. Ritchlin</h3><p><sup>29</sup>Allergy, Immunology, and Rheumatology Division, University of Rochester Medical Center, Rochester, NY 14623, USA</p><div>Find articles by <a href="/pubmed/?term=Ritchlin%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Christopher&nbsp;T. Ritchlin</a></div></div><div id="_co_idm139913926702288"><h3 class="no_margin">Anne&nbsp;M. Bowcock</h3><p><sup>30</sup>National Heart and Lung Institute, Imperial College, London SW7 2AZ, UK</p><div>Find articles by <a href="/pubmed/?term=Bowcock%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Anne&nbsp;M. Bowcock</a></div></div><div id="_co_idm139913926700304"><h3 class="no_margin">Gerald&nbsp;G. Krueger</h3><p><sup>8</sup>Department of Dermatology, University of Utah, Salt Lake City, UT 84132, USA</p><div>Find articles by <a href="/pubmed/?term=Krueger%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Gerald&nbsp;G. Krueger</a></div></div><div id="_co_idm139913926698320"><h3 class="no_margin">Henry&nbsp;W. Lim</h3><p><sup>31</sup>Department of Dermatology, Henry Ford Hospital, Detroit, MI 48202, USA</p><div>Find articles by <a href="/pubmed/?term=Lim%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Henry&nbsp;W. Lim</a></div></div><div id="_co_idm139913885060000"><h3 class="no_margin">Stephan Weidinger</h3><p><sup>15</sup>Department of Dermatology, University Medical Center Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weidinger%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Stephan Weidinger</a></div></div><div id="_co_idm139913885058016"><h3 class="no_margin">Michael Weichenthal</h3><p><sup>15</sup>Department of Dermatology, University Medical Center Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Weichenthal%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Michael Weichenthal</a></div></div><div id="_co_idm139913885056032"><h3 class="no_margin">John&nbsp;J. Voorhees</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Voorhees%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">John&nbsp;J. Voorhees</a></div></div><div id="_co_idm139913886652432"><h3 class="no_margin">Proton Rahman</h3><p><sup>32</sup>Memorial University, St. John’s, NL A1C 5B8, Canada</p><div>Find articles by <a href="/pubmed/?term=Rahman%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Proton Rahman</a></div></div><div id="_co_idm139913886650448"><h3 class="no_margin">Peter&nbsp;K. Gregersen</h3><p><sup>33</sup>The Feinstein Institute for Medical Research, North Shore–Long Island Jewish Health System, Manhasset, NY 11030, USA</p><div>Find articles by <a href="/pubmed/?term=Gregersen%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Peter&nbsp;K. Gregersen</a></div></div><div id="_co_idm139913886648464"><h3 class="no_margin">Andre Franke</h3><p><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University&nbsp;of Kiel, 24105 Kiel, Germany</p><div>Find articles by <a href="/pubmed/?term=Franke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Andre Franke</a></div></div><div id="_co_idm139913891535616"><h3 class="no_margin">Dafna&nbsp;D. Gladman</h3><p><sup>6</sup>Department of Medicine, Division of Rheumatology, University of Toronto, Toronto, ON M5T 2S8, Canada</p><p><sup>7</sup>Centre for Prognosis Studies in the Rheumatic Diseases, Toronto Western Research Institute, University of Toronto, Toronto, ON M5T 2S8, Canada</p><div>Find articles by <a href="/pubmed/?term=Gladman%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Dafna&nbsp;D. Gladman</a></div></div><div id="_co_idm139913891532896"><h3 class="no_margin">Gonçalo&nbsp;R. Abecasis</h3><p><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</p><div>Find articles by <a href="/pubmed/?term=Abecasis%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">Gonçalo&nbsp;R. Abecasis</a></div></div><div id="_co_idm139913891530912"><h3 class="no_margin">James&nbsp;T. Elder</h3><p><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</p><p><sup>4</sup>Ann Arbor Veterans Affairs Hospital, Ann Arbor, MI 48105, USA</p><div>Find articles by <a href="/pubmed/?term=Elder%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26626624">James&nbsp;T. Elder</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139913883701072_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139913883701072_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139913883701072_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139913883701072_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Department of Dermatology, University of Michigan Medical School, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Department of Biostatistics, Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Ann Arbor Veterans Affairs Hospital, Ann Arbor, MI 48105, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Institute of Clinical Molecular Biology, Christian-Albrechts-University&nbsp;of Kiel, 24105 Kiel, Germany</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Medicine, Division of Rheumatology, University of Toronto, Toronto, ON M5T 2S8, Canada</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Centre for Prognosis Studies in the Rheumatic Diseases, Toronto Western Research Institute, University of Toronto, Toronto, ON M5T 2S8, Canada</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Department of Dermatology, University of Utah, Salt Lake City, UT 84132, USA</div><div class="fm-affl" id="aff9" lang="en"><sup>9</sup>Department of Internal Medicine, University of Michigan Medical School, Ann Arbor, MI 48109, USA</div><div class="fm-affl" id="aff10" lang="en"><sup>10</sup>Department of Clinical and Experimental Medicine, Division of Cell Biology and Dermatology, Linköping University, 581 83 Linköping, Sweden</div><div class="fm-affl" id="aff11" lang="en"><sup>11</sup>Department of Pathophysiology, Centre of Translational Medicine and Centre for Translational Genomics, University of Tartu, 50411 Tartu, Estonia</div><div class="fm-affl" id="aff12" lang="en"><sup>12</sup>Department of Reproductive Biology, Estonian University of Life Sciences, 51014 Tartu, Estonia</div><div class="fm-affl" id="aff13" lang="en"><sup>13</sup>Dermatology Clinic, Tartu University Hospital, Department of Dermatology and Venereology, University of Tartu, 50417 Tartu, Estonia</div><div class="fm-affl" id="aff14" lang="en"><sup>14</sup>Estonian Genome Center, University of Tartu, 51010 Tartu, Estonia</div><div class="fm-affl" id="aff15" lang="en"><sup>15</sup>Department of Dermatology, University Medical Center Schleswig-Holstein, Campus Kiel, 24105 Kiel, Germany</div><div class="fm-affl" id="aff16" lang="en"><sup>16</sup>Institute of Human Genetics, University of Erlangen-Nuremberg, 91054 Erlangen, Germany</div><div class="fm-affl" id="aff17" lang="en"><sup>17</sup>Institute of Epidemiology I, Helmholtz Zentrum Munich, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff18" lang="en"><sup>18</sup>Institute of Medical Informatics, Biometry and Epidemiology, Ludwig-Maximilians-University, 81377 Munich, Germany</div><div class="fm-affl" id="aff19" lang="en"><sup>19</sup>Institute of Medical Statistics and Epidemiology, Technical University Munich, 80333 Munich, Germany</div><div class="fm-affl" id="aff20" lang="en"><sup>20</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff21" lang="en"><sup>21</sup>Institute of Epidemiology II, Helmholtz Zentrum München, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff22" lang="en"><sup>22</sup>Institute of Human Genetics, University of Bonn, 53127 Bonn, Germany</div><div class="fm-affl" id="aff23" lang="en"><sup>23</sup>Department of Genomics, Life &amp; Brain Center, University of Bonn, 53127 Bonn, Germany</div><div class="fm-affl" id="aff24" lang="en"><sup>24</sup>Neurologische Klinik and Poliklinik, Klinikum rechts der Isar, Technische Universität München, 80333 Munich, Germany</div><div class="fm-affl" id="aff25" lang="en"><sup>25</sup>Institute of Neurogenomics, Helmholtz Zentrum Munchen, German Research Center for Environmental Health, 85764 Neuherberg, Germany</div><div class="fm-affl" id="aff26" lang="en"><sup>26</sup>Department of Dermatology, Venereology and Allergology, University of Leipzig, 04103 Leipzig, Germany</div><div class="fm-affl" id="aff27" lang="en"><sup>27</sup>Saint Paul Rheumatology, Eagan, MN 55121, USA</div><div class="fm-affl" id="aff28" lang="en"><sup>28</sup>Seattle Rheumatology Associates, 601 Broadway, Suite 600, Seattle, WA 98122, USA</div><div class="fm-affl" id="aff29" lang="en"><sup>29</sup>Allergy, Immunology, and Rheumatology Division, University of Rochester Medical Center, Rochester, NY 14623, USA</div><div class="fm-affl" id="aff30" lang="en"><sup>30</sup>National Heart and Lung Institute, Imperial College, London SW7 2AZ, UK</div><div class="fm-affl" id="aff31" lang="en"><sup>31</sup>Department of Dermatology, Henry Ford Hospital, Detroit, MI 48202, USA</div><div class="fm-affl" id="aff32" lang="en"><sup>32</sup>Memorial University, St. John’s, NL A1C 5B8, Canada</div><div class="fm-affl" id="aff33" lang="en"><sup>33</sup>The Feinstein Institute for Medical Research, North Shore–Long Island Jewish Health System, Manhasset, NY 11030, USA</div><div><span class="contrib-email" id="idm139913891530912">James&nbsp;T. Elder: <a href="mailto:dev@null" data-email="ude.hcimu@redlej" class="oemail">ude.hcimu@redlej</a></span> </div><div id="cor1"><sup>∗</sup>Corresponding author <a href="mailto:dev@null" data-email="ude.hcimu@redlej" class="oemail">ude.hcimu@redlej</a></div><div id="fn1"><sup>34</sup>These authors contributed equally to this study</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139913883701072_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Jul 9; Accepted 2015 Oct 29.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139913883701072_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> ©2015 by The American Society of Human Genetics. All rights reserved.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4678416/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="3" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S1. Figures S1–S18 and Tables S1–S7 and S9–S16</div><div><a href="/pmc/articles/PMC4678416/bin/mmc1.pdf" data-ga-action="click_feat_suppl">mmc1.pdf</a><span style="color:gray"> (7.1M)</span></div><div class="small guid">GUID:&nbsp;DFA00D9B-83DA-41F0-AB60-ECA6BE782D69</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Table S8. Discovery Study Association Statistics at Previously Reported Genome-wide Significant Risk Variants for PsV, PsA, and PsC</div><div><a href="/pmc/articles/PMC4678416/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">mmc2.xlsx</a><span style="color:gray"> (47K)</span></div><div class="small guid">GUID:&nbsp;73FEAF96-EB9E-45AF-83F8-BA3B200484C6</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">Document S2. Article plus Supplemental Data</div><div><a href="/pmc/articles/PMC4678416/bin/mmc3.pdf" data-ga-action="click_feat_suppl">mmc3.pdf</a><span style="color:gray"> (11M)</span></div><div class="small guid">GUID:&nbsp;BB1B2825-842C-436F-91FE-3469394A4A5A</div></div></dd></dl></div><div id="idm139913885105120" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913885105120title">Abstract</h2><!--article-meta--><div><p id="__p2" class="p p-first-last">Psoriasis vulgaris (PsV) is a common inflammatory and hyperproliferative skin disease. Up to 30% of people with PsV eventually develop psoriatic arthritis (PsA), an inflammatory musculoskeletal condition. To discern differences in genetic risk factors for PsA and cutaneous-only psoriasis (PsC), we carried out a genome-wide association study (GWAS) of 1,430 PsA case subjects and 1,417 unaffected control subjects. Meta-analysis of this study with three other GWASs and two targeted genotyping studies, encompassing a total of 9,293 PsV case subjects, 3,061 PsA case subjects, 3,110 PsC case subjects, and 13,670 unaffected control subjects of European descent, detected 10 regions associated with PsA and 11 with PsC at genome-wide (GW) significance. Several of these association signals (<em>IFNLR1</em>, <em>IFIH1</em>, <em>NFKBIA</em> for PsA; <em>TNFRSF9</em>, <em>LCE3C/B</em>, <em>TRAF3IP2</em>, <em>IL23A, NFKBIA</em> for PsC) have not previously achieved GW significance. After replication, we also identified a PsV-associated SNP near <em>CDKAL1</em> (rs4712528, odds ratio [OR] = 1.16, p = 8.4&nbsp;× 10<sup>−11</sup>). Among identified psoriasis risk variants, three were more strongly associated with PsC than PsA (rs12189871 near <em>HLA-C</em>, p = 5.0&nbsp;× 10<sup>−19</sup>; rs4908742 near <em>TNFRSF9</em>, p = 0.00020; rs10888503 near <em>LCE3A</em>, p = 0.0014), and two were more strongly associated with PsA than PsC (rs12044149 near <em>IL23R</em>, p = 0.00018; rs9321623 near <em>TNFAIP3</em>, p = 0.00022). The PsA-specific variants were independent of previously identified psoriasis variants near <em>IL23R</em> and <em>TNFAIP3</em>. We also found multiple independent susceptibility variants in the <em>IL12B</em>, <em>NOS2</em>, and <em>IFIH1</em> regions. These results provide insights into the pathogenetic similarities and differences between PsC and PsA.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">psoriasis, psoriatic arthritis, skin diseases, joint diseases, genome-wide association study</span></div></div><div id="sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec1title">Introduction</h2><p id="__p3" class="p p-first">Psoriasis vulgaris (PsV [MIM: <a href="/omim/177900" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">177900</a>]) is a chronic, immune-mediated disease of the skin characterized by inflammation and epidermal hyperplasia, with a prevalence of 0.2%−6.5%, depending upon the population of origin.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> Approximately 15%−30% of people with psoriasis will eventually develop psoriatic arthritis (PsA [MIM: <a href="/omim/607507" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607507</a>]), an often debilitating musculoskeletal condition that is usually seronegative for rheumatoid factor.<sup><a href="#bib2" rid="bib2" class=" bibr popnode">2</a></sup> PsV and PsA pose increased risk for cardiovascular, autoimmune, metabolic, and psychiatric conditions.<sup><a href="#bib3" rid="bib3" class=" bibr popnode">3</a></sup><sup>, </sup><sup><a href="#bib4" rid="bib4" class=" bibr popnode">4</a></sup><sup>, </sup><sup><a href="#bib5" rid="bib5" class=" bibr popnode">5</a></sup><sup>, </sup><sup><a href="#bib6" rid="bib6" class=" bibr popnode">6</a></sup></p><p id="__p4">Both PsV and PsA have a strong genetic component. Based on twin and family studies in populations of European descent, estimates of heritability for PsV range from 50% to 90%,<sup><a href="#bib7" rid="bib7" class=" bibr popnode">7</a></sup><sup>, </sup><sup><a href="#bib8" rid="bib8" class=" bibr popnode">8</a></sup><sup>, </sup><sup><a href="#bib9" rid="bib9" class=" bibr popnode">9</a></sup><sup>, </sup><sup><a href="#bib10" rid="bib10" class=" bibr popnode">10</a></sup><sup>, </sup><sup><a href="#bib11" rid="bib11" class=" bibr popnode">11</a></sup><sup>, </sup><sup><a href="#bib12" rid="bib12" class=" bibr popnode">12</a></sup> with recurrence rates in siblings (λ<sub>s</sub>) and first-degree relatives (λ<sub>1</sub>) of 4–19.<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup><sup>, </sup><sup><a href="#bib14" rid="bib14" class=" bibr popnode">14</a></sup><sup>, </sup><sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup><sup>, </sup><sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup> The genetic contribution to PsA appears even larger, with estimates of 80%–100% for heritability<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup><sup>, </sup><sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup> and of 30–49 for λ<sub>s</sub> and λ<sub>1</sub>.<sup><a href="#bib13" rid="bib13" class=" bibr popnode">13</a></sup><sup>, </sup><sup><a href="#bib15" rid="bib15" class=" bibr popnode">15</a></sup><sup>, </sup><sup><a href="#bib16" rid="bib16" class=" bibr popnode">16</a></sup><sup>, </sup><sup><a href="#bib17" rid="bib17" class=" bibr popnode">17</a></sup> A series of large-scale genome-wide association studies (GWASs) and genome-wide (GW) meta-analyses during the past decade have identified nearly 70 genetic loci harboring PsV risk variants.<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup><sup>, </sup><sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup><sup>, </sup><sup><a href="#bib20" rid="bib20" class=" bibr popnode">20</a></sup><sup>, </sup><sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup><sup>, </sup><sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup><sup>, </sup><sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup><sup>, </sup><sup><a href="#bib26" rid="bib26" class=" bibr popnode">26</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib28" rid="bib28" class=" bibr popnode">28</a></sup><sup>, </sup><sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup><sup>, </sup><sup><a href="#bib30" rid="bib30" class=" bibr popnode">30</a></sup><sup>, </sup><sup><a href="#bib31" rid="bib31" class=" bibr popnode">31</a></sup><sup>, </sup><sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> In contrast, only three small to modestly sized GWASs for PsA have been published,<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup> which, together with analyses of candidate loci in larger samples and recently published Immunochip array studies, have identified 13 regions associated with PsA at GW significance (p ≤ 5&nbsp;× 10<sup>−8</sup>): near <em>HLA-B</em> (MIM: <a href="/omim/142830" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">142830</a>) and <em>HLA-C</em> (MIM: <a href="/omim/143840" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">143840</a>) in the major histocompatibility complex (MHC)<sup><a href="#bib19" rid="bib19" class=" bibr popnode">19</a></sup><sup>, </sup><sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#bib35" rid="bib35" class=" bibr popnode">35</a></sup><sup>, </sup><sup><a href="#bib36" rid="bib36" class=" bibr popnode">36</a></sup><sup>, </sup><sup><a href="#bib37" rid="bib37" class=" bibr popnode">37</a></sup><sup>, </sup><sup><a href="#bib38" rid="bib38" class=" bibr popnode">38</a></sup><sup>, </sup><sup><a href="#bib39" rid="bib39" class=" bibr popnode">39</a></sup><sup>, </sup><sup><a href="#bib40" rid="bib40" class=" bibr popnode">40</a></sup><sup>, </sup><sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup><sup>, </sup><sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup><sup>, </sup><sup><a href="#bib43" rid="bib43" class=" bibr popnode">43</a></sup><sup>, </sup><sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup> and also near <em>IL12B</em> (MIM: <a href="/omim/161561" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">161561</a>),<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>IL23R</em> (MIM: <a href="/omim/607562" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607562</a>),<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>IL23A</em> (MIM: <a href="/omim/605580" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605580</a>),<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>TNIP1</em> (MIM: <a href="/omim/607714" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607714</a>),<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup><sup>, </sup><sup><a href="#bib46" rid="bib46" class=" bibr popnode">46</a></sup>
<em>TRAF3IP2</em> (MIM: <a href="/omim/607043" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607043</a>),<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#bib40" rid="bib40" class=" bibr popnode">40</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>CSF2</em> (MIM: <a href="/omim/138960" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">138960</a>),<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>FBXL19</em> (MIM: <a href="/omim/609085" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609085</a>),<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup>
<em>REL</em> (MIM: <a href="/omim/164910" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">164910</a>),<sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup>
<em>RUNX3</em> (MIM: <a href="/omim/600210" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600210</a>),<sup><a href="#bib47" rid="bib47" class=" bibr popnode">47</a></sup>
<em>TYK2</em> (MIM: <a href="/omim/176941" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">176941</a>),<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>NOS2</em> (MIM: <a href="/omim/163730" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">163730</a>),<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> and <em>PTPN22</em> (MIM: <a href="/omim/600716" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600716</a>).<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup></p><p id="__p5">There are no published GWASs of cutaneous-only psoriasis (PsC), which we define as the presence of skin symptoms without PsA for at least 10 years. Candidate studies of PsC have identified seven associated genetic regions achieving GW significance: near MHC genes <em>HLA-C</em>, <em>HLA-B</em>, <em>CCHCR1</em> (MIM: <a href="/omim/605310" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605310</a>), and <em>TNF</em> (MIM: <a href="/omim/191160" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">191160</a>),<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup><sup>, </sup><sup><a href="#bib37" rid="bib37" class=" bibr popnode">37</a></sup><sup>, </sup><sup><a href="#bib40" rid="bib40" class=" bibr popnode">40</a></sup><sup>, </sup><sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup><sup>, </sup><sup><a href="#bib49" rid="bib49" class=" bibr popnode">49</a></sup><sup>, </sup><sup><a href="#bib50" rid="bib50" class=" bibr popnode">50</a></sup><sup>, </sup><sup><a href="#bib51" rid="bib51" class=" bibr popnode">51</a></sup><sup>, </sup><sup><a href="#bib52" rid="bib52" class=" bibr popnode">52</a></sup><sup>, </sup><sup><a href="#bib53" rid="bib53" class=" bibr popnode">53</a></sup> and also near <em>IL12B</em>,<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup>
<em>TNIP1</em>,<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup>
<em>TNFAIP3</em> (MIM: <a href="/omim/191163" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">191163</a>),<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup>
<em>IL23R</em>,<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup>
<em>NOS2</em>,<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> and <em>IL13</em> (MIM: <a href="/omim/147683" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147683</a>).<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup></p><p id="__p6">It is reasonable to hypothesize that the observed differences in phenotype, prevalence, and heritability of PsA and PsC stem at least in part from differences in their underlying genetic architecture. Differences in strength of association with PsC and PsA have been repeatedly observed for the MHC, including a stronger association of <em>HLA-C<sup>∗</sup>06</em> with PsC<sup><a href="#bib42" rid="bib42" class=" bibr popnode">42</a></sup> and a stronger association of <em>HLA-B<sup>∗</sup>27</em> with PsA,<sup><a href="#bib41" rid="bib41" class=" bibr popnode">41</a></sup> both differences reaching GW significance. There is a paucity of well-powered studies formally comparing association for PsA versus PsC outside of the MHC region. A&nbsp;scan of the literature finds only ten non-MHC regions with at least a nominally significant (p &lt; 0.05) difference in the strength of association for these two subphenotypes of psoriasis—near <em>IL13</em>,<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup><sup>, </sup><sup><a href="#bib55" rid="bib55" class=" bibr popnode">55</a></sup><sup>, </sup><sup><a href="#bib56" rid="bib56" class=" bibr popnode">56</a></sup><sup>, </sup><sup><a href="#bib57" rid="bib57" class=" bibr popnode">57</a></sup>
<em>CSF2</em> (MIM: <a href="/omim/601267" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601267</a>),<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup>
<em>CCR2</em>,<sup><a href="#bib58" rid="bib58" class=" bibr popnode">58</a></sup>
<em>IFNG</em> (MIM: <a href="/omim/147570" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147570</a>),<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup>
<em>IL12B</em>,<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib46" rid="bib46" class=" bibr popnode">46</a></sup>
<em>ZNF816</em>,<sup><a href="#bib46" rid="bib46" class=" bibr popnode">46</a></sup>
<em>IL23R</em>,<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup>
<em>FBXL19</em>,<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup>
<em>PTPN22</em>,<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> and the <em>KIR</em> gene cluster.<sup><a href="#bib59" rid="bib59" class=" bibr popnode">59</a></sup></p><p id="__p7" class="p p-last">To address the need for more powerful and comprehensive genetic studies of PsA and PsC, we first carried out a large GWAS of PsA, involving 1,430 PsA case subjects and 1,417 unaffected control subjects. We then combined results of this PsA GWAS with five published studies of psoriasis association—three GWASs and two targeted studies, comprising a discovery meta-analysis of 3,061 PsA case subjects, 3,110 PsC case subjects, 9,293 PsV case subjects, and 13,670 unaffected control subjects. For markers with promising association signals in this discovery analysis, a large partially overlapping sample of up to 13,857 individuals was genotyped, allowing both independent replication and validation of the quality of imputed genotypes in the discovery sample. We used stepwise conditional regression to discover additional PsA, PsC, and PsV risk variants that were independent of the primary index signal in each region. We then devised a bootstrap procedure to compare the relative strength of PsA and PsC associations for all strongly associated primary and secondary variants detected by the discovery and replication analyses, as well as for most variants previously reported as differentially associated with PsA versus PsC. Finally, we investigated potential functional consequences of our findings.</p></div><div id="sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec2title">Material and Methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">Samples</h3><p id="__p8" class="p p-first">All PsV case subjects were diagnosed by a dermatologist; a large subset of these individuals was also evaluated for PsA by a rheumatologist. Individuals affected with PsV for 10 or more years without developing any signs of PsA were classified as PsC. For all studies, each PsA and PsC case subject was also classified as a PsV case subject; the remaining PsV case subjects were PsA-negative individuals affected with psoriasis for less than 10 years or people for whom a diagnosis of PsA was either unavailable or uncertain. Control subjects were 18 years of age or older and unaffected with PsV or PsA. All participating individuals provided written informed consent and were recruited according to the protocols approved by the institutional review board of each institution.</p><p id="__p9" class="p p-last">For the PsA GWAS, 1,526 PsA case subjects and 1,508 unaffected&nbsp;control subjects were recruited by four collection centers—Michigan, Toronto, Newfoundland, and the National Psoriasis Foundation. Samples for the other five studies in the discovery meta-analysis have been described previously. Three have GW coverage, referred to here as the Collaborative Association Study of Psoriasis (CASP),<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> Kiel,<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup> and Genizon<sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup><sup>, </sup><sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup> GWAS. The other two are targeted genotyping studies: a deep follow-up study of the CASP GWAS (CASP-DFU) designed to fine-map and replicate association signals from this earlier GWAS<sup><a href="#bib60" rid="bib60" class=" bibr popnode">60</a></sup> and the Psoriasis Association Genetics Extension (PAGE) Immunochip-based study.<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> For these five published studies, case and control subjects were recruited from local populations at each of 17 collection centers in 4 countries—the United States, Canada, Estonia, and Germany. The published PAGE Immunochip study was extended with an additional 395 unaffected control subjects from the United States and 568 PsA case subjects from the United States, Canada, New Zealand, Australia, and 11 European countries. For our replication/validation sample, we genotyped up to 13,857 individuals from six collection centers in the United States, Canada, and Sweden (7,004 PsV case subjects, 2,554 PsA case subjects, 1,921 PsC case subjects, and 6,830 unaffected control subjects) that partially overlapped the six studies in the discovery set. All samples used in this study were of self-reported white European ancestry.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">Genotyping and Quality Control</h3><p id="__p10" class="p p-first">Samples for the PsA GWAS were genotyped with the Illumina HumanOmni1-Quad BeadChip array (Illumina). Genotypes were obtained for 1,020,596 autosomal SNPs. We removed SNPs with a call rate &lt; 95%, a HWE p in controls &lt; 1&nbsp;× 10<sup>−7</sup>, or a minor allele frequency (MAF) &lt; 0.01. Samples were removed if they had substantial non-European admixture, were duplicates or first- or second-degree relatives of other samples, had a genotype call rate &lt; 98%, had outlier inbreeding coefficients (F &lt; −0.0225 or F &gt; 0.0285), or were part of unresolvable sample swaps determined by X chromosome genotypes.</p><p id="__p11">Population outliers for the PsA GWAS were determined by principal components analysis (PCA), using SNPs passing the aforementioned quality-control filters that were further filtered to exclude those with a MAF &lt; 0.05 and those occurring in known regions of psoriasis association. After linkage disequilibrium (LD)-based pruning with version 1.7 of Plink<sup><a href="#bib61" rid="bib61" class=" bibr popnode">61</a></sup> to quasi-independence (maximum pairwise LD-r<sup>2</sup> = 0.20), 103,699 SNPs remained. Genotypes for these SNPs were combined with HapMap2 genotypes for samples from the YRI, CEU, JPT, and CHB reference populations and subjected to PCA, and individuals lying farther than 1/10 of the distance between CEU and JPT/CHB/YRI when plotted on the first two PCA axes were considered to have substantial non-European admixture.</p><p id="__p12">Array-based genotyping and subsequent quality control measures for the CASP GWAS, Kiel GWAS, Genizon GWAS, CASP-DFU, and PAGE Immunochip were similar to those employed for the PSA GWAS and have been described in detail previously.<sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib34" rid="bib34" class=" bibr popnode">34</a></sup><sup>, </sup><sup><a href="#bib60" rid="bib60" class=" bibr popnode">60</a></sup> We used Plink<sup><a href="#bib61" rid="bib61" class=" bibr popnode">61</a></sup> and RelativeFinder<sup><a href="#bib62" rid="bib62" class=" bibr popnode">62</a></sup> to determine genetic relatedness of samples among all six discovery studies, removing all but one of each cluster of duplicated samples or first- or second-degree relatives, which ensured that meta-analysis was restricted to unrelated individuals.</p><p id="__p13" class="p p-last">We genotyped 25 markers to validate and replicate promising signals from the discovery analysis. Genotyping was performed by Taqman, Snapshot, or length polymorphisms assays (Life Technologies), or as part of a custom set on the Affymetrix Axiom Biobank Plus Array (Affymetrix), or both. We were unable to design successful genotyping assays for three of the 25 markers (those near <em>CAMTA1</em> [MIM: <a href="/omim/611501" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">611501</a>], <em>NFKBIZ</em> [MIM: <a href="/omim/608004" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608004</a>], and <em>LINC00701</em>); for <em>NFKBIZ</em> we instead genotyped the second most significantly associated marker (rs4683946) in the region. Subsequent validation analyses based on this genotyping found that 7 of the 23 follow-up markers had questionable imputation quality. For one of these (rs8018217), we substituted a marker that was more reliably imputed in the discovery analysis (rs3168891), which is in near-perfect LD (r<sup>2</sup> = 0.985) with the primary signal, and which also happened to be genotyped in the replication/validation set as part of the standard content of the exome array. Samples that failed to genotype for 3 or more of the 23 markers or that were not of self-reported European ancestry were excluded from downstream analysis. Samples were also excluded if they had two or more discrepant genotypes among the 11 markers typed twice by different methods. After sample filtering, genotyping success exceeded 95% and HWE test p values were &gt;0.05 for all 23 markers.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">Imputation</h3><p id="__p14" class="p p-first-last">Version 1.0.16.b of MaCH<sup><a href="#bib63" rid="bib63" class=" bibr popnode">63</a></sup> was used to pre-phase haplotypes based on genotype data for each of the six discovery studies. Release 2012-05-29 of minimac<sup><a href="#bib64" rid="bib64" class=" bibr popnode">64</a></sup> was used to impute SNP and indel genotypes based on phased chromosomal haplotypes from the 379 individuals in the EUR subpopulation of release 3, phase 1 of the 1000 Genomes Project.<sup><a href="#bib65" rid="bib65" class=" bibr popnode">65</a></sup> The following criteria were used to flag markers with poor imputation despite passing our predicted imputation quality threshold (r<sup>2</sup> ≥ 0.3): (1) a substantially (ratio&nbsp;&gt; 1.10 or &lt; 0.91) and significantly (adjusted p &lt; 0.05) different MAF for imputed and experimental genotypes, (2) empirical imputation quality substantially (≥0.1 r<sup>2</sup> units) worse than predicted, and (3) strength of association substantially (ratio of log(OR) values &gt; 1.1 or &lt; 0.91) and significantly (p &lt; 0.05) different when comparing association results from meta-analysis of imputed and assayed genotypes.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">Association Tests</h3><p id="__p15" class="p p-first">We used logistic regression to test for association of variants with the binary case-control phenotype. Variant genotypes were coded as imputed allele dosages. To control for population stratification in array-based studies, we used coordinates from the top ten dimensions of PCA or multidimensional scaling analysis as covariates in the regression model for each individual study. Geographic cohort covariates were also included for the PsA GWAS, CASP GWAS, and PAGE Immunochip studies, as well as when analyzing the replication and validation markers. To test for associations that were independent of the primary signal in a region, we utilized stepwise forward logistic regression. The regression model in this case included imputed allele dosages of the most significantly associated variant as an additional covariate. Dosages of the most significantly associated variant in the region when conditioning upon the primary signal were then included as a new covariate in the next round of stepwise analysis, providing its significance exceeded the threshold set to correct for multiple testing (p ≤ 1&nbsp;× 10<sup>−6</sup>, based on the number of LD-independent tested variants within all analyzed regions and phenotype comparisons combined). To test for epistasis between pairs of variants, imputed allele dosages of the two tested variants and a multiplicative dosage interaction term were included in the logistic regression model along with cohort and PCA covariates.</p><p id="__p16">Inverse variance-weighted fixed-effects meta-analysis was used to combine association results across datasets. For the initial six-study discovery analysis, all variants for which at least one study had good imputation quality (r<sup>2</sup> ≥ 0.3)<sup><a href="#bib63" rid="bib63" class=" bibr popnode">63</a></sup> were tested; no minimum MAF criterion was imposed. For meta-analysis of stepwise regression results and generation of association meta-p-values needed for construction of 95% Bayesian credible sets, the analyses for a region were restricted to those discovery studies with good coverage in the region, and only those markers well imputed in all included studies were tested. Heterogeneity of ORs across studies was tested with Cochran’s Q test.</p><p id="__p17">We utilized QQ plots and the genomic inflation factor (λ<sub>GC</sub>) to test for residual effects of population stratification not fully controlled for by the inclusion of PCA and cohort covariates in the regression model. All variants in regions of known association at the time of the analysis<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> were excluded, and variants were also LD-pruned with version 1.7 of Plink<sup><a href="#bib61" rid="bib61" class=" bibr popnode">61</a></sup> (window size = 50, step = 5, pairwise r<sup>2</sup> threshold = 0.2).</p><p id="__p18" class="p p-last">A posteriori power of detecting significant (α = 0.05) association for markers in the replication sample was determined with the Genetic Power Calculator.<sup><a href="#bib66" rid="bib66" class=" bibr popnode">66</a></sup> For power analysis, estimates of ORs and their 95% confidence intervals were based on results from the discovery meta-analysis that had been adjusted for ascertainment bias (winner’s curse).<sup><a href="#bib67" rid="bib67" class=" bibr popnode">67</a></sup></p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">Linkage Disequilibrium</h3><p id="__p19" class="p p-first-last">We computed D′ and r<sup>2</sup> measures of LD for pairs of variants using genotypes of 379 individuals of European ancestry (EUR subpopulation) in release 3 of phase 1 of the 1000 Genomes Project.<sup><a href="#bib65" rid="bib65" class=" bibr popnode">65</a></sup> Blocks of LD in regions of association were delineated by the&nbsp;D′&nbsp;confidence interval method<sup><a href="#bib68" rid="bib68" class=" bibr popnode">68</a></sup> based on all SNPs with MAF&nbsp;≥ 0.05 in the same EUR subpopulation. Version 1.7 of Plink was used for all LD calculations.<sup><a href="#bib61" rid="bib61" class=" bibr popnode">61</a></sup></p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">Phenotypic Variance Explained</h3><p id="__p20" class="p p-first-last">For the non-MHC risk variants detected by this study, which are all biallelic, we used a liability threshold model to compute separately for each variant the percent variance of liability for disease explained.<sup><a href="#bib69" rid="bib69" class=" bibr popnode">69</a></sup> Relative risk was approximated by the log-additive OR, frequency of the risk allele in the underlying population was estimated as a weighted average of its frequency in PsV case subjects and unaffected control subjects (weights = 0.02 and 0.98, respectively) of our combined discovery set, and population prevalences for PsV, PsA, and PsC were assumed to be 0.02, 0.003, and 0.017 based on published data.<sup><a href="#bib1" rid="bib1" class=" bibr popnode">1</a></sup> For regions with multiple risk variants, we adjusted values for any interdependencies due to LD by first computing the variance explained by the correlated SNPs as a group<sup><a href="#bib69" rid="bib69" class=" bibr popnode">69</a></sup> and then partitioning this total explained variance into non-overlapping variances for each SNP by the method of Pratt.<sup><a href="#bib70" rid="bib70" class=" bibr popnode">70</a></sup> For the MHC, which contains several correlated multiallelic polymorphisms that influence risk for psoriasis, we applied a multiallelic extension of the biallelic method<sup><a href="#bib69" rid="bib69" class=" bibr popnode">69</a></sup> to ORs derived from a full regression model containing all MHC risk variants and their multiple alleles that were identified by stepwise regression analysis of our six discovery studies.<sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup></p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">Comparison of Effect Sizes for PsA and PsC Associations</h3><p id="__p21" class="p p-first">We used both parametric and bootstrap methods to compare effect sizes from meta-analysis of PsA case versus control subjects and of PsC case versus control subjects. With the exception of the PsA GWAS, the unaffected control subjects are shared by all discovery studies in these two meta-analyses, so the two meta-ORs are dependent random variables. A test statistic for the difference in two dependent ORs can be formulated as</p><div class="disp-formula" id="ufd1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.gif" overflow="scroll"><mrow><mfrac><mrow><msup><mrow><mrow><mo>(</mo><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mn>2</mn></msup></mrow><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub><mo>+</mo><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>ρ</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi><mo>,</mo><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub><msqrt><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub></mrow></msqrt><msqrt><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub></mrow></msqrt></mrow></mfrac><mo>,</mo></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.gif" overflow="scroll"><mrow><mfrac><mrow><msup><mrow><mrow><mo>(</mo><mrow><msub><mi>β</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub><mo>−</mo><msub><mi>β</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub></mrow><mo>)</mo></mrow></mrow><mn>2</mn></msup></mrow><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub><mo>+</mo><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub><mo>−</mo><mn>2</mn><msub><mi>ρ</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi><mo>,</mo><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub><msqrt><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>A</mi></mrow></msub></mrow></msqrt><msqrt><mrow><msub><mi>V</mi><mrow><mi>P</mi><mi>s</mi><mi>C</mi></mrow></msub></mrow></msqrt></mrow></mfrac><mo>,</mo></mrow></math></script></div></div><p>where <em>β</em><sub><em>PsA</em></sub> and <em>β</em><sub><em>PsC</em></sub> are the log ORs for PsA-affected versus control individuals and PsC-affected versus control individuals, <em>V</em><sub><em>PsA</em></sub> and <em>V</em><sub><em>PsC</em></sub> are the variances of these log ORs, and <em>ρ</em><sub><em>PsA,PsC</em></sub> is the correlation of the two log ORs. Under the null hypothesis of no difference in the two effects, this test statistic follows a χ<sup>2</sup> distribution with 1 degree of freedom. Although the log ORs and their variances for the six discovery studies can be estimated by fixed-effects meta-analysis, there is no analytic approximation for the correlation of the ORs. As expected, simulations show that the correlation is nearly always positive but that its value varies substantially, especially in response to the relative and absolute magnitudes of the observed effect sizes. Setting the last term in the denominator of the test statistic to zero, equivalent to assuming independence of the two ORs, affords a rapid albeit conservative parametric test.</p><p id="__p22" class="p p-last">As an alternative, we devised a bootstrap procedure. For each of the six discovery studies, we separately sampled with replacement PsA case subjects, PsC case subjects, and unaffected control subjects, maintaining the same sample sizes as in the original studies. Association tests for PsA case versus control subjects and PsC case versus control subjects were then run for each individual study, followed by fixed-effects meta-analysis for each of the two phenotype comparisons, using the same logistic regression models applied to the observed data. Sampling and testing were repeated 50,000 times. A χ<sup>2</sup> test statistic was computed by dividing the square of the difference in the two observed log meta-ORs by the variance of their difference, where the latter is computed from the 50,000 bootstrap estimates of the difference. This standard bootstrap approach<sup><a href="#bib71" rid="bib71" class=" bibr popnode">71</a></sup> yielded very similar p values to other bootstrap methods (percentile, bias-corrected). To validate the bootstrap approach, we inspected all final and intermediate bootstrapped estimates for 100,000 iterations of 52 independently associated markers. For all markers, the bootstrapped distribution for the difference in log ORs was normal (Lilliefor’s test p &gt; 0.05) and bias was negligible (mean and median bias = 0.0001 and 0.0000, respectively). Furthermore, for all markers and each phenotype comparison, bootstrapped and parametric association results for the individual studies and the meta-analysis were very similar.</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">Bayesian Credible Sets</h3><p id="__p23" class="p p-first-last">Using a Bayesian approach, for selected association signals we identified the credible set of markers that were 95% likely, based on posterior probability, to contain the causal disease-associated variant. Although the concept was based on the approach of Maller et&nbsp;al.,<sup><a href="#bib72" rid="bib72" class=" bibr popnode">72</a></sup> our method used a Bayes factor derived from more recent work<sup><a href="#bib73" rid="bib73" class=" bibr popnode">73</a></sup> that explicitly considers meta-analysis. Input for a signal consisted of meta-ORs and their standard errors for all markers within 500 kb of the most significantly associated variant in the six-study discovery analysis.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">Analysis of RNA Expression</h3><p id="__p24" class="p p-first">We used the Wilcoxon rank-sum test to test for differences in RNA expression levels, derived by RNA-seq, between 92 punch biopsies of lesional skin from psoriatic individuals versus 82 biopsies of normal skin from unaffected control subjects, as described previously.<sup><a href="#bib74" rid="bib74" class=" bibr popnode">74</a></sup> To declare a gene as differentially expressed, we required a 2-fold or greater difference in median RPKM-normalized transcript levels, with a rank-sum test p&nbsp;&lt;&nbsp;10<sup>−6</sup> (corresponding to a family-wise error rate &lt; 0.025 for all genes with detectable expression in at least half of the lesional or normal skin biopsies).</p><p id="__p25">For analysis of possible eQTL variants, RNA levels were determined for all gene models in Ensembl v.74 for the same 92 lesional and 82 normal skin biopsies, using version 1.3.3 of Tophat<sup><a href="#bib75" rid="bib75" class=" bibr popnode">75</a></sup> for alignment and version 2.1.1 of Cufflink<sup><a href="#bib76" rid="bib76" class=" bibr popnode">76</a></sup> for ab initio transcript assembly, as described previously.<sup><a href="#bib77" rid="bib77" class=" bibr popnode">77</a></sup> All 92 lesional samples and all but two of the 82 normal samples were genotyped on the Affymetrix Axiom Biobank Plus Array, which contains a GWAS backbone panel of 260,000 SNPs in addition to exome and custom content. After quality control, high-quality genotypes for 586,178 SNPs were available for GW imputation using all populations of release 1 of phase 3 of the 1000 Genomes Project. All Cufflink-normalized RNA values &lt; 0.1 were treated as zero. Only those genes with RNA expression &gt; 0 in at least 20% of the normal or 20% of the lesional samples were analyzed. Before analysis, we first applied a square-root transformation for variance stabilization, then standardized separately within lesional and normal groups to a mean of zero and standard deviation of one. Merlin-offline, distributed as part of version 1.1.2 of Merlin,<sup><a href="#bib78" rid="bib78" class=" bibr popnode">78</a></sup> was used to test for significant correlation of transcript levels with imputed allele dosages of variants; inverse normalization was applied to the expression levels, and the top two principal components and an indicator variable for lesional/normal skin were included as covariates in the association model.</p><p id="__p26" class="p p-last">The RNA expression data used in these analyses are available at NCBI’s Gene Expression Omnibus (GEO accession number <a id="__tag_497670664" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE63979" ref="/geo/query/acc.cgi?acc=GSE63979"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE63979","term_id":"63979"}}</span>GSE63979</a>). Details of the generation of these data by RNA-seq, QC metrics, and validation of normalized expression values are presented elsewhere.<sup><a href="#bib74" rid="bib74" class=" bibr popnode">74</a></sup><sup>, </sup><sup><a href="#bib77" rid="bib77" class=" bibr popnode">77</a></sup></p></div><div id="sec2.10" class="sec sec-last"><h3 id="sec2.10title">Annotation Analysis</h3><p id="__p27" class="p p-first-last">For associated variants, the UCSC Table Browser<sup><a href="#bib79" rid="bib79" class=" bibr popnode">79</a></sup> was used to extract PhastCons, PhyloP, and Gerp++ conservation scores, as well as location within CpG islands, from hg19 reference assembly tracks phastCons100way, phyloP100wayAll, allHg19RS_BW, and cpgIslandExt, respectively. Version 2014-11-12 of ANNOVAR<sup><a href="#bib80" rid="bib80" class=" bibr popnode">80</a></sup> was used to determine positions of variants relative to RefSeq genes. Version 3 of HaploReg<sup><a href="#bib81" rid="bib81" class=" bibr popnode">81</a></sup> was used to determine how many cell types among the ENCODE and Roadmap reference epigenomes have promoter histone marks, enhancer histone marks, or DNase I hypersensitivite sites overlapping a variant, as well as ChIP-seq evidence of transcription factors that bind to the interval containing the variant, and which transcription factor binding sites motifs are changed by a variant. Version 1.1 of RegulomeDB<sup><a href="#bib82" rid="bib82" class=" bibr popnode">82</a></sup> was used to obtain scores of how likely it is that a variant has regulatory function, and version 1.2 of CADD<sup><a href="#bib83" rid="bib83" class=" bibr popnode">83</a></sup> provided scores of variant deleteriousness. The overlap of associated variants with chromatin states of 33 cell types relevant to psoriasis, which were derived from a 15-state segmentation model based on five chromatin marks, was based on results of analyses generated by the Roadmap Epigenomics Project.<sup><a href="#bib84" rid="bib84" class=" bibr popnode">84</a></sup></p></div></div><div id="sec3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec3title">Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">PsA GWAS</h3><p id="__p28" class="p p-first">Application of quality control filters to the PsA GWAS yielded high-quality genotypes at 791,217 SNPs for 1,430 PsA case subjects and 1,417 control subjects. Demographic details of the post-filtering sample collection are shown in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S1</a>. After imputation, genotypes for 11,532,644 SNPs and 918,976 indels were available for downstream analysis. QQ plots and genomic inflation factors (1.013−1.025) for four different MAF bins of the imputed variants indicated good control of population stratification (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S1</a>).</p><p id="__p29" class="p p-last">Five regions in the PsA GWAS were associated at GW significance (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S2</a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S2</a>). These were all known risk regions for PsA, and the strongest signals were near <em>HLA-B</em> (rs36058333, OR = 2.13, p = 1.3&nbsp;× 10<sup>−29</sup>), <em>IL12B</em> (rs918520, OR = 1.50, p = 9.3&nbsp;× 10<sup>−10</sup>), <em>TRAF3IP2</em> (rs33980500, OR = 1.74, p = 4.2&nbsp;× 10<sup>−9</sup>), <em>TNIP1</em> (rs8177833, OR = 1.75, p = 3.7&nbsp;× 10<sup>−8</sup>), and <em>TYK2</em> (rs35251378, OR = 1.41, p = 3.5&nbsp;× 10<sup>−8</sup>). Five additional regions surpassed a suggestive threshold of significance (p = 1&nbsp;× 10<sup>−6</sup>), four of which (near <em>TNS1</em> [MIM: <a href="/omim/600076" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">600076</a>], <em>ATP10D</em>, <em>PTPRN2</em> [MIM: <a href="/omim/601698" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601698</a>], <em>CTNNA3</em> [MIM: <a href="/omim/607667" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607667</a>]) have no previously reported risk variants for PsV or PsA.</p></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">Meta-analysis of Six Discovery Studies</h3><p id="__p30" class="p p-first">To increase power and extend our analysis to include PsC case subjects, we combined the results of the PsA GWAS with those of five published genetic studies of psoriasis in people of European descent. Three of these studies have GW coverage; the other two use targeted genotyping arrays. As shown in <a href="/pmc/articles/PMC4678416/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a>, the four GWASs combined provided a discovery set of 4,007 PsV, 1,946 PsA, and 1,363 PsC case subjects as well as 4,934 unaffected control subjects. The two targeted studies provided an additional 5,286 PsV, 1,115 PsA, and 1,747 PsC case subjects and 8,736 control subjects for a subset of the genome. The five additional datasets were imputed to the same reference panel as the PsA GWAS. Fixed-effects meta-analysis was used to test for association of the six discovery studies combined. The four phenotype comparisons tested were PsV case versus unaffected control subjects, PsA case versus unaffected control subjects, PsC case versus unaffected control subjects, and PsA case versus PsC case subjects. The PsA GWAS was excluded from the latter two meta-analyses because it did not include any PsC case subjects. For each of the four phenotype comparisons, QQ plots of p values from the meta-analyses indicated good control of population stratification (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S3</a>), as did genomic inflation factors for the six individual studies as well as for the meta-analyses (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S3</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p94">Characteristics of the Six Studies in the Discovery Meta-analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913888737712" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Study</strong></th><th colspan="5" rowspan="1"><strong>No. Individuals</strong><hr></th><th rowspan="1" colspan="1"><strong>No. Genotyped SNPs</strong><sup>a</sup></th><th colspan="2" rowspan="1"><strong>No. Imputed Variants</strong><sup>b</sup><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>PsV</strong></th><th rowspan="1" colspan="1"><strong>PsA</strong></th><th rowspan="1" colspan="1"><strong>PsC</strong></th><th rowspan="1" colspan="1"><strong>Controls</strong></th><th rowspan="1" colspan="1"><strong>Total</strong></th><th rowspan="1" colspan="1"><strong>SNPs</strong></th><th rowspan="1" colspan="1"><strong>Indels</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">PsA GWAS</td><td rowspan="1" colspan="1" align="center">1,430</td><td rowspan="1" colspan="1" align="center">1,430</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">1,417</td><td rowspan="1" colspan="1" align="center">2,847</td><td rowspan="1" colspan="1" align="center">791,217</td><td rowspan="1" colspan="1" align="center">11,532,644</td><td rowspan="1" colspan="1" align="center">918,976</td></tr><tr><td rowspan="1" colspan="1">CASP GWAS</td><td rowspan="1" colspan="1" align="center">1,352</td><td rowspan="1" colspan="1" align="center">344</td><td rowspan="1" colspan="1" align="center">695</td><td rowspan="1" colspan="1" align="center">1,389</td><td rowspan="1" colspan="1" align="center">2,741</td><td rowspan="1" colspan="1" align="center">438,609</td><td rowspan="1" colspan="1" align="center">10,708,247</td><td rowspan="1" colspan="1" align="center">865,325</td></tr><tr><td rowspan="1" colspan="1">Kiel GWAS</td><td rowspan="1" colspan="1" align="center">464</td><td rowspan="1" colspan="1" align="center">33</td><td rowspan="1" colspan="1" align="center">269</td><td rowspan="1" colspan="1" align="center">1,135</td><td rowspan="1" colspan="1" align="center">1,599</td><td rowspan="1" colspan="1" align="center">504,637</td><td rowspan="1" colspan="1" align="center">10,730,120</td><td rowspan="1" colspan="1" align="center">872,971</td></tr><tr><td rowspan="1" colspan="1">Genizon GWAS</td><td rowspan="1" colspan="1" align="center">761</td><td rowspan="1" colspan="1" align="center">139</td><td rowspan="1" colspan="1" align="center">399</td><td rowspan="1" colspan="1" align="center">993</td><td rowspan="1" colspan="1" align="center">1,754</td><td rowspan="1" colspan="1" align="center">489,501</td><td rowspan="1" colspan="1" align="center">10,525,123</td><td rowspan="1" colspan="1" align="center">863,572</td></tr><tr><td rowspan="1" colspan="1">CASP-DFU</td><td rowspan="1" colspan="1" align="center">1,563</td><td rowspan="1" colspan="1" align="center">133</td><td rowspan="1" colspan="1" align="center">575</td><td rowspan="1" colspan="1" align="center">1,141</td><td rowspan="1" colspan="1" align="center">2,704</td><td rowspan="1" colspan="1" align="center">7,592</td><td rowspan="1" colspan="1" align="center">207,911</td><td rowspan="1" colspan="1" align="center">18,001</td></tr><tr><td rowspan="1" colspan="1">PAGE Immunochip<hr></td><td rowspan="1" colspan="1" align="center">3,723<hr></td><td rowspan="1" colspan="1" align="center">982<hr></td><td rowspan="1" colspan="1" align="center">1,172<hr></td><td rowspan="1" colspan="1" align="center">7,595<hr></td><td rowspan="1" colspan="1" align="center">11,318<hr></td><td rowspan="1" colspan="1" align="center">168,360<hr></td><td rowspan="1" colspan="1" align="center">1,654,147<hr></td><td rowspan="1" colspan="1" align="center">141,025<hr></td></tr><tr><td rowspan="1" colspan="1">All</td><td rowspan="1" colspan="1" align="center">9,293</td><td rowspan="1" colspan="1" align="center">3,061</td><td rowspan="1" colspan="1" align="center">3,110</td><td rowspan="1" colspan="1" align="center">13,670</td><td rowspan="1" colspan="1" align="center">22,963</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td><td rowspan="1" colspan="1">–</td></tr></tbody></table></div><div id="largeobj_idm139913888737712" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4678416/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913889758112"><p id="__p95" class="p p-first-last">Abbreviations are as follows: PsV, psoriasis vulgaris; PsA, psoriatic arthritis; PsC, cutaneous psoriasis.</p></div></div><div class="tblwrap-foot"><div id="tblfn1"><sup>a</sup>After applying all quality control measures.</div></div><div class="tblwrap-foot"><div id="tblfn2"><sup>b</sup>Restricted to variants with minimum acceptable imputation quality (MaCH-r<sup>2</sup> ≥ 0.3).</div></div></div><p id="__p31">The Manhattan plots in <a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a> label all regions with GW significant association detected by the four meta-analyses (see <a href="#mmc1" rid="mmc1" class=" supplementary-material">Tables S4–S7</a> for detailed results). <a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>A shows 26 GW significant regions for the test of PsV case versus unaffected control subjects; all harbor known PsV susceptibility variants, including two near <em>CAMK2G</em> (MIM: <a href="/omim/602123" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602123</a>) and <em>NFKBIZ</em> that were recently published as psoriasis associated<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> based on a sample that has a large overlap with the discovery sample of this study. <a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>B shows ten regions with GW significant association for PsA. Seven of these were previously known (near <em>IL23R</em>, <em>REL</em>, <em>TNIP1</em>, <em>IL12B</em>, <em>HLA-C</em>, <em>TRAF3IP2</em>, and <em>TYK2</em>) and the other three were newly established at GW significance—<em>IFNLR1</em> ([MIM: <a href="/omim/607404" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607404</a>] rs7540214, OR = 1.40, p = 1.6&nbsp;× 10<sup>−8</sup>), <em>IFIH1</em> ([MIM: <a href="/omim/606951" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606951</a>] rs1990760, OR&nbsp;=&nbsp;1.22, p = 1.8&nbsp;× 10<sup>−8</sup>), and <em>NFKBIA</em> ([MIM: <a href="/omim/164008" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">164008</a>] rs12883343, OR = 1.22, p = 2.6&nbsp;× 10<sup>−9</sup>). Twelve regions with GW significant association were identified by meta-analysis of PsC cases (<a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>C), including six known PsC signals (near <em>IL13</em>, <em>TNIP1</em>, <em>IL12B</em>, <em>HLA-C</em>, <em>TNFAIP3</em>, and <em>NOS2</em>) and a candidate locus near <em>RGS6</em> ([MIM: <a href="/omim/603894" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">603894</a>] rs12895275, OR = 1.65, p = 2.0&nbsp;× 10<sup>−8</sup>). In addition, five known PsV signals were newly established as associated with PsC at GW significance—<em>TNFRSF9</em> ([MIM: <a href="/omim/602250" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">602250</a>] rs4908742, OR = 1.22, p = 2.2&nbsp;× 10<sup>−8</sup>), <em>LCE3C/B</em> ([MIM: <a href="/omim/612615" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612615</a>, <a href="/omim/612614" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">612614</a>] rs10888503, OR = 1.25, p = 1.8&nbsp;× 10<sup>−11</sup>), <em>TRAF3IP2</em> (rs9481169, OR = 1.39, p = 5.7&nbsp;× 10<sup>−11</sup>), <em>IL23A</em> (rs11575231, OR = 1.48, p = 7.9&nbsp;× 10<sup>−9</sup>), and <em>NFKBIA</em> (rs61251127, OR = 1.25, p = 1.3&nbsp;× 10<sup>−8</sup>). Meta-analysis of PsA case versus PsC case subjects identified only two GW significant peaks (<a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>D). One is a well-known signal near <em>HLA-C</em> and <em>HLA-B</em> (rs1050414, OR = 1.53, p&nbsp;= 7.4&nbsp;× 10<sup>−11</sup>). The second is a previously unreported association near <em>LOC100505817</em> on chromosome 18 (rs4891505, OR = 1.63, p = 6.7&nbsp;× 10<sup>−9</sup>), which is an uncharacterized long non-coding RNA.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913964692624" class="figure"><a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4678416_gr1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4678416/bin/gr1.jpg"></a></div><div id="largeobj_idm139913964692624" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4678416/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure&nbsp;1</a></div><!--caption a7--><div class="caption"><p id="__p86">Manhattan Plots of Association Results for Meta-analysis of Six Discovery Studies</p><p id="__p87">Each circle represents the −log<sub>10</sub>(p) of the imputed variants, including SNPs and indels. Thresholds of suggestive (p = 1&nbsp;× 10<sup>−6</sup>) and GW (p = 5&nbsp;× 10<sup>−8</sup>) significance are delineated with blue and red lines, respectively. All chromosomal regions achieving GW significance are highlighted in green and labeled with their leading candidate susceptibility gene. Font color of the candidate genes indicates published GW significant associations with psoriasis and its cutaneous and joint subphenotypes: purple indicates known PsA, PsC, and PsV association, red indicates known PsA and PsV association, cyan indicates known PsC and PsV association, black indicates known PsV association, and green indicates possible new psoriasis association. We tested association for four binary phenotypes: PsV case versus control subjects (A), PsA case versus control subjects (B), PsC case versus control subjects (C), and PsA case versus PsC case subjects (D).</p></div></div></div><p id="__p32" class="p p-last">The discovery meta-analysis found nominal (p ≤ 0.05) association at all previously reported GW-significant signals for PsV, PsA, and PsC in populations of European ancestry, and association at p ≤ 1&nbsp;× 10<sup>−5</sup> for most of them (76%, 77%, and 100%, respectively) (<a href="#mmc2" rid="mmc2" class=" supplementary-material">Table S8</a>). In contrast, just 23% of the PsV-associated regions detected only in Chinese populations reached even nominal association in our study.</p></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">Validation and Replication of Candidate Signals</h3><p id="__p33" class="p p-first">The discovery analyses uncovered 25 regions with no published psoriasis variants at the time of analysis that were at least suggestively associated (p &lt; 1&nbsp;× 10<sup>−6</sup>) with PsV, PsA, or PsC (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Tables S4–S7</a>). In order to replicate these promising association signals in an independent sample, and also to validate their imputed dosages in the discovery sample, we genotyped up to 13,857 individuals of European descent that partially overlapped the discovery set. Depending upon the phenotype comparison being tested, the imputation quality of a marker across the six discovery studies, and the method of assay, this yielded a set of 2,615−8,415 retyped individuals for validation of imputation quality and of 2,602−7,779 newly typed individuals for independent replication.</p><p id="__p34">Samples from four of the discovery studies (CASP GWAS, PsA GWAS, CASP-DFU, and PAGE Immunochip) were retyped for the 23 successfully assayed replication markers. We assessed reliability of imputation (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figures S4–S6</a>; see <a href="#sec2" rid="sec2" class=" sec">Material and Methods</a> for more details) and found that 7 of the 23 replication markers (near <em>BHLHE40</em> [MIM: <a href="/omim/604256" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604256</a>], <em>USP49</em>, <em>JAK2</em> [MIM: <a href="/omim/147796" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">147796</a>], <em>ZNF462</em>, <em>MBIP</em> [MIM: <a href="/omim/609431" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">609431</a>], <em>RGS6</em>, and <em>APOBEC3G</em> [MIM: <a href="/omim/607113" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">607113</a>]) might have unreliable imputation quality in at least one study of the discovery analysis (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S9</a>).</p><p id="__p35" class="p p-last">After exclusion of poorly imputed markers for which we could find no well-imputed substitute, the newly typed samples provided an independent replication of 17 association signals. Association statistics for these markers for the discovery set, the replication set, and the two sets combined are given in <a href="/pmc/articles/PMC4678416/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> (see <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S10</a> for full results for all 25 suggestively associated signals). Only two of the markers (rs4712528 near <em>CDKAL1</em> [MIM: <a href="/omim/611259" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">611259</a>] and rs2675669 near <em>CAMK2G</em>) were significantly associated in the replication set, which pushed the evidence of their association with PsV to GW significance in the combined analysis (OR = 1.16, p = 8.4&nbsp;× 10<sup>−11</sup> for <em>CDKAL1</em>; OR = 1.13, p = 3.9&nbsp;× 10<sup>−11</sup> for <em>CAMK2G</em>). Neither of these SNPs showed significant heterogeneity of association across studies in the combined analysis (Cochran’s Q test p = 0.54 and 0.42, respectively). The <em>CAMK2G</em> region has recently been shown to be associated with psoriasis based on a sample that has substantial overlap with the one used in this study,<sup><a href="#bib32" rid="bib32" class=" bibr popnode">32</a></sup> but to our knowledge GW-significant psoriasis association for a <em>CDKAL1</em> variant has not been previously described. The association peak for <em>CDKAL1</em> lies fully within the gene, near its 5′ end (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S7</a>). Two SNPs that were associated at GW significance in the discovery stage (near <em>NFKBIZ</em> and <em>LOC100505817</em>) were no longer associated at this level in the combined analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p98">Selected Results for Previously Unreported Regions of Psoriasis Association</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913889752000" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Marker</strong></th><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>hg19 Position</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Alleles (Risk/Nonrisk)</strong></th><th rowspan="1" colspan="1"><strong>Nearby Gene</strong></th><th rowspan="1" colspan="1"><strong>Phenotype Comparison</strong><sup>b</sup></th><th colspan="3" rowspan="1"><strong>Discovery (6-Study Meta)</strong><hr></th><th colspan="3" rowspan="1"><strong>Replication</strong><hr></th><th colspan="3" rowspan="1"><strong>Combined (Meta)</strong><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>No. Cases/Controls</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>No. Cases/Controls</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th><th rowspan="1" colspan="1"><strong>No. Cases/Controls</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs13396545</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1" align="center">215,672,833</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1"><em>BARD1</em></td><td rowspan="1" colspan="1">PsA-PsC</td><td rowspan="1" colspan="1" align="center">516/1,363</td><td rowspan="1" colspan="1">1.59</td><td rowspan="1" colspan="1" align="center">9&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,663/1,101</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">2,179/2,464</td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1">0.01</td></tr><tr><td rowspan="1" colspan="1">rs4685408</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">16,996,035</td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1"><em>PLCL2</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">9,293/13,670</td><td rowspan="1" colspan="1">1.12</td><td rowspan="1" colspan="1" align="center">4&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">799/1,438</td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">10,092/15,108</td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs4683946</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">101,615,826</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1"><em>NFKBIZ</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">7,730/12,529</td><td rowspan="1" colspan="1">1.19</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1,949/2,440</td><td rowspan="1" colspan="1" align="center">1.05</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">9,679/14,969</td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">7&nbsp;× 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs4683946</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1" align="center">101,615,826</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1"><em>NFKBIZ</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">2,928/12,529</td><td rowspan="1" colspan="1">1.27</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">162/2,440</td><td rowspan="1" colspan="1" align="center">1.21</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">3,090/14,969</td><td rowspan="1" colspan="1" align="center">1.27</td><td rowspan="1" colspan="1" align="center">9&nbsp;× 10<sup>−8</sup></td></tr><tr><td rowspan="1" colspan="1">rs72622783</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">32,577,723</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1"><em>—</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">1,946/4,934</td><td rowspan="1" colspan="1">2.78</td><td rowspan="1" colspan="1" align="center">3&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">893/4,789</td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">2,839/9,723</td><td rowspan="1" colspan="1" align="center">1.91</td><td rowspan="1" colspan="1" align="center">3&nbsp;× 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">rs113633694</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1" align="center">68,949,589</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1"><em>TMPRSS11F</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">1,602/3,545</td><td rowspan="1" colspan="1">7.13</td><td rowspan="1" colspan="1" align="center">5&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,168/5,443</td><td rowspan="1" colspan="1" align="center">0.88</td><td rowspan="1" colspan="1" align="center">0.6</td><td rowspan="1" colspan="1" align="center">2,770/8,988</td><td rowspan="1" colspan="1" align="center">1.63</td><td rowspan="1" colspan="1">0.02</td></tr><tr><td rowspan="1" colspan="1">rs4712528</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">20,678,430</td><td rowspan="1" colspan="1">C/G</td><td rowspan="1" colspan="1"><em>CDKAL1</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">9,293/13,670</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">2,223/3,185</td><td rowspan="1" colspan="1" align="center">1.22</td><td rowspan="1" colspan="1">6&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">11,516/16,855</td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1" align="center">8&nbsp;× 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs4725242</td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1" align="center">1,629,781</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1"><em>PSMG3</em></td><td rowspan="1" colspan="1">PsA-PsC</td><td rowspan="1" colspan="1" align="center">1,154/1,840</td><td rowspan="1" colspan="1">1.46</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,596/1,140</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">0.5</td><td rowspan="1" colspan="1" align="center">2,750/2,980</td><td rowspan="1" colspan="1" align="center">1.23</td><td rowspan="1" colspan="1" align="center">4&nbsp;× 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">rs56677333</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1" align="center">6,750,801</td><td rowspan="1" colspan="1">C/A</td><td rowspan="1" colspan="1"><em>DEFB1</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">1,946/4,934</td><td rowspan="1" colspan="1">1.59</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">343/2,996</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">1.0</td><td rowspan="1" colspan="1" align="center">2,289/7,930</td><td rowspan="1" colspan="1" align="center">1.45</td><td rowspan="1" colspan="1" align="center">3&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs7922314</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">64,538,279</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1"><em>ADO</em></td><td rowspan="1" colspan="1">PsC-ctl</td><td rowspan="1" colspan="1" align="center">3,110/12,253</td><td rowspan="1" colspan="1">1.42</td><td rowspan="1" colspan="1" align="center">9&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">342/2,889</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.6</td><td rowspan="1" colspan="1" align="center">3,452/15,142</td><td rowspan="1" colspan="1" align="center">1.33</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs12356475</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">67,761,720</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1"><em>CTNNA3</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">1,946/4,934</td><td rowspan="1" colspan="1">1.38</td><td rowspan="1" colspan="1" align="center">8&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">873/4,744</td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.3</td><td rowspan="1" colspan="1" align="center">2,819/9,678</td><td rowspan="1" colspan="1" align="center">1.25</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">rs2675669</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1" align="center">75,653,872</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1"><em>CAMK2G</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">9,293/13,670</td><td rowspan="1" colspan="1">1.13</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">2,223/3,191</td><td rowspan="1" colspan="1" align="center">1.16</td><td rowspan="1" colspan="1">3&nbsp;× 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">11,516/16,861</td><td rowspan="1" colspan="1" align="center">1.13</td><td rowspan="1" colspan="1" align="center">4&nbsp;× 10<sup>−11</sup></td></tr><tr><td rowspan="1" colspan="1">rs149014202</td><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1" align="center">29,436,518</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1"><em>SLC46A3</em></td><td rowspan="1" colspan="1">PsA-ctl</td><td rowspan="1" colspan="1" align="center">33/1,135</td><td rowspan="1" colspan="1" align="center">297</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,768/4,936</td><td rowspan="1" colspan="1" align="center">1.84</td><td rowspan="1" colspan="1" align="center">0.1</td><td rowspan="1" colspan="1" align="center">1,801/6,071</td><td rowspan="1" colspan="1" align="center">3.49</td><td rowspan="1" colspan="1">0.001</td></tr><tr><td rowspan="1" colspan="1">rs3168891</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">36,789,729</td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1"><em>MBIP</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">4,007/4,934</td><td rowspan="1" colspan="1">1.35</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">3,789/3,990</td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.4</td><td rowspan="1" colspan="1" align="center">7,796/8,924</td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1">0.009</td></tr><tr><td rowspan="1" colspan="1">rs35144987</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1" align="center">62,725,664</td><td rowspan="1" colspan="1">T/–</td><td rowspan="1" colspan="1"><em>LINC00643</em></td><td rowspan="1" colspan="1">PsC-ctl</td><td rowspan="1" colspan="1" align="center">2,535/11,112</td><td rowspan="1" colspan="1">1.24</td><td rowspan="1" colspan="1" align="center">8&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,222/5,648</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">3,757/16,760</td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">4&nbsp;× 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">rs17052344</td><td rowspan="1" colspan="1" align="center">17</td><td rowspan="1" colspan="1" align="center">21,717,543</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1"><em>FAM27L</em></td><td rowspan="1" colspan="1">PsC-ctl</td><td rowspan="1" colspan="1" align="center">2,535/11,112</td><td rowspan="1" colspan="1">1.24</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1,220/5,625</td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.2</td><td rowspan="1" colspan="1" align="center">3,755/16,737</td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">4&nbsp;× 10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">rs4891505</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1" align="center">71007958</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1"><em>LOC100505817</em></td><td rowspan="1" colspan="1">PsA-PsC</td><td rowspan="1" colspan="1" align="center">1,498/2,535</td><td rowspan="1" colspan="1">1.63</td><td rowspan="1" colspan="1" align="center">7&nbsp;× 10<sup>−9</sup></td><td rowspan="1" colspan="1" align="center">2,101/1,221</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">0.7</td><td rowspan="1" colspan="1" align="center">3,599/3,756</td><td rowspan="1" colspan="1" align="center">1.30</td><td rowspan="1" colspan="1" align="center">1&nbsp;× 10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">rs6088273</td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1" align="center">30045393</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1"><em>DEFB123</em></td><td rowspan="1" colspan="1">PsV-ctl</td><td rowspan="1" colspan="1" align="center">7,730/12,529</td><td rowspan="1" colspan="1">1.14</td><td rowspan="1" colspan="1" align="center">9&nbsp;× 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">3,234/3,804</td><td rowspan="1" colspan="1" align="center">1.02</td><td rowspan="1" colspan="1" align="center">0.6</td><td rowspan="1" colspan="1" align="center">10,964/16,333</td><td rowspan="1" colspan="1" align="center">1.09</td><td rowspan="1" colspan="1" align="center">2&nbsp;× 10<sup>−5</sup></td></tr></tbody></table></div><div id="largeobj_idm139913889752000" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4678416/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913890622784"><p id="__p99" class="p p-first-last">Regions in table had no published reports of GW significant association with PsV, PsA, or PsC at time of analysis (February 2014). Abbreviations are as follows: chr, chromosome; PsV, psoriasis vulgaris; PsA, psoriatic arthritis; PsC, cutaneous psoriasis; ctl, unaffected control; OR, odds ratio.</p></div></div><div class="tblwrap-foot"><div id="tblfn3"><sup>a</sup>For insertions or deletions of the reference sequence, position of first base before the insertion point or of first base of the deleted sequence is shown, respectively.</div></div><div class="tblwrap-foot"><div id="tblfn4"><sup>b</sup>Binary phenotype tested for association: PsA-ctl, PsA case versus unaffected control subjects; PsC-ctl, PsA case versus unaffected control subjects; PsV-ctl, PsV case versus unaffected control subjects; PsA-PsC, PsA case versus PsC case subjects; PsA-PsC+ctl, PsA case versus combination of PsC case and unaffected control subjects.</div></div></div></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">Conditional Association Analysis to Identify Secondary Susceptibility Variants</h3><p id="__p36" class="p p-first">We next investigated whether we could detect additional susceptibility variants independent of the best signal in&nbsp;any of the regions attaining suggestive significance (p&nbsp;= 1&nbsp;× 10<sup>−6</sup>) in our discovery analysis. To this end, for each of the six discovery studies, we utilized stepwise forward logistic regression. We excluded the MHC region from our analysis, because we have recently published a fine-mapping analysis of this region for PsV, PsA, and PsC associations based on the same six discovery sets of this study.<sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup> For each region, all markers within 500 kb of the primary signal were candidates for conditional analysis.</p><p id="__p37" class="p p-last">The results of meta-analysis of qualifying studies are shown in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S11</a> and <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figures S8–S13</a>. Variants independent of the primary signal were detected in four susceptibility regions, containing candidate genes <em>IL23R</em>, <em>IL12B</em>, <em>NOS2</em>, and <em>IFIH1</em>. Multiple secondary signals upstream of <em>IL12B</em> were associated with PsV, PsA, and PsC, yielding, respectively, a total of five, four, and three independent variants in this region for these three phenotypes. Two PsV-associated conditional signals were observed for <em>IL23R</em>, the first near the 5′ end and the second near the 3′ end of the gene. A single secondary signal near <em>NOS2</em> was associated with PsV, and another secondary signal upstream of <em>IFIH1</em> was associated with PsA. A comparison of unconditional and full model association results for these four regions reveals that the identified signals within a region are not fully independent of each other. Because of both positive and negative LD among the variants, several show either much weaker or much stronger association in the full model than when tested in isolation.</p></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">Epistasis of Psoriasis Susceptibility Variants</h3><p id="__p38" class="p p-first-last">We tested for statistical interaction among pairwise combinations of all known or newly established primary plus all secondary risk variants reaching adequate statistical significance in the discovery analysis (p ≤ 1&nbsp;× 10<sup>−7</sup> for primary signals; p ≤ 1&nbsp;× 10<sup>−6</sup> for secondary signals). For PsV, positive epistasis was observed between the risk alleles of the primary MHC variant (rs12189871) and the <em>ERAP1</em> (MIM: <a href="/omim/606832" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606832</a>) variant (rs30377) (interaction OR = 1.14; p&nbsp;= 4.6&nbsp;× 10<sup>−6</sup>) and the risk alleles of the MHC variant&nbsp;and the <em>LCE</em> variant (rs10888503) (OR = 1.26; p&nbsp;=&nbsp;6.9&nbsp;× 10<sup>−10</sup>), confirming previous reports of epistasis for these two pairs of loci.<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib85" rid="bib85" class=" bibr popnode">85</a></sup><sup>, </sup><sup><a href="#bib86" rid="bib86" class=" bibr popnode">86</a></sup> No pairs of PsA or PsC risk variants showed&nbsp;significant interaction after correction for multiple testing.</p></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">Comparison of Strength of Association with PsA and&nbsp;PsC</h3><p id="__p39" class="p p-first">Genetic association with PsA and PsC case subjects was directly compared in the discovery meta-analysis (<a href="/pmc/articles/PMC4678416/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Figure&nbsp;1</span></a>D). This analysis necessarily omitted nearly half of the PsA case subjects because they were from the PsA GWAS, which had no PsC case subjects. We thus turned to a conservative parametric test of the ratio of the ORs from the meta-analyses of PsA case versus control subjects and PsC case versus control subjects. As shown in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S14</a>, the increase in power due to inclusion of all PsA case subjects yielded a peak MHC signal (rs2523615, OR = 1.91, p = 9.9&nbsp;× 10<sup>−28</sup>) 17 orders of magnitude more significant than the best MHC signal from the PsA versus PsC meta-analysis. The signal in <em>LOC100505817</em> on chromosome 18, which was GW significant in the meta-analysis of PsA versus PsC case subjects but failed independent replication, does not reach suggestive significance in this analysis.</p><p id="__p40">We devised a bootstrap procedure to compare ORs from the PsA versus control and PsC versus control subject meta-analyses. Although the bootstrap approach remedies the modestly conservative nature of the parametric test (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S15</a>), it is too computationally demanding for GW analysis. We instead first applied the bootstrap test to the four non-MHC regions with p &lt; 1&nbsp;× 10<sup>−5</sup> from GW analysis using the parametric test. The significance of association increased for the best variant in each of&nbsp;the four regions, but still failed to reach suggestive (p&nbsp;= 1&nbsp;× 10<sup>−6</sup>) significance.</p><p id="__p41">We then applied the bootstrap test to all known and newly established psoriasis signals with strong evidence of association (p ≤ 1&nbsp;× 10<sup>−7</sup>) in the discovery analysis, as well as to all secondary signals uncovered in the conditional analysis. The most significantly associated variant for each of the four phenotype comparisons that met the aforementioned criteria was tested. Nine variants, comprising five LD-independent groups, had an FDR-adjusted p &lt; 0.05 for a test of the null hypothesis of equal strength of association with PsA and PsC (<a href="/pmc/articles/PMC4678416/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a>; <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S12</a> has results for all 59 tested variants). Three of these groups were variants whose PsV risk allele was more strongly associated with PsC than PsA—rs4908742 near <em>TNFRSF9</em> (OR = 1.19, p = 0.00020; <a href="/pmc/articles/PMC4678416/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figures 2</span></a>A–2C), rs10888503 near <em>LCE3C/B</em> (OR = 1.15, p = 0.0014; <a href="/pmc/articles/PMC4678416/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figures 2</span></a>D–2F), and four SNPs (rs1050414, rs12189871, rs13214872, and rs13210419) near <em>HLA-C</em> (OR = 1.34−1.80, p = 5.5&nbsp;× 10<sup>−9</sup>−1.7&nbsp;× 10<sup>−21</sup>). The other two groups were variants whose PsV risk allele was more strongly associated with PsA than PsC—two SNPs (rs12044149 and rs4655683) upstream of <em>IL23R</em> (OR = 1.20 and 1.14, p = 0.00018 and 0.0075, respectively) and SNP rs9321623 (OR = 1.17, p = 0.00022), which lies 230 kb upstream of <em>TNFAIP3</em>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC4678416/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913887975104" class="figure"><a href="/pmc/articles/PMC4678416/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4678416_gr2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4678416/bin/gr2.jpg"></a></div><div id="largeobj_idm139913887975104" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4678416/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC4678416/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure&nbsp;2</a></div><!--caption a7--><div class="caption"><p id="__p88">Comparison of PsC and PsA Associations for the <em>TNFRSF9</em> and <em>LCE3C/B</em> Regions</p><p id="__p89">Negative log10-transformed p values from meta-analyses of association for the six discovery datasets are shown for 800-kb regions encompassing genes <em>TNFRSF9</em> (A−C) and <em>LCE3C</em> and <em>LCE3B</em> (D−F). For each region the SNP with the most significant association for the comparison of PsC case subjects with unaffected control subjects is labeled. The color of each symbol denotes the strength of LD (r<sup>2</sup> coefficient) with the index SNP. (A) and (D) Shown are association results for PsA case versus control subjects (A, D), for PsC case versus control subjects (B, E), and for a conservative parametric test of the difference in log(OR) values from the PsA versus control and PsC versus control subjects (C, F) comparisons. The bottom two panels give the physical positions of known genes in each region.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p102">Selected Results for Comparative Strength of Association of PsA versus PsC for Established Psoriasis Variants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913890616336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"><strong>Candidate Gene</strong></th><th rowspan="1" colspan="1"><strong>Rnd</strong><sup>a</sup></th><th rowspan="1" colspan="1"><strong>Marker</strong></th><th rowspan="1" colspan="1"><strong>Chr</strong></th><th rowspan="1" colspan="1"><strong>Position (hg19)</strong><sup>b</sup></th><th colspan="4" rowspan="1"><strong>Index Marker for:</strong><sup>c</sup><hr></th><th rowspan="1" colspan="1"><strong>Alleles</strong><sup>d</sup></th><th rowspan="1" colspan="1"><strong>RAF</strong><sup>e</sup></th><th colspan="2" rowspan="1"><strong>PsA versus Control</strong><sup>f</sup><hr></th><th colspan="2" rowspan="1"><strong>PsC versus Control</strong><sup>f</sup><hr></th><th colspan="2" rowspan="1"><strong>PsA-ctl versus PsC-ctl</strong><sup>g</sup><hr></th></tr><tr><th rowspan="1" colspan="1"><strong>PsV ctl</strong></th><th rowspan="1" colspan="1"><strong>PsA ctl</strong></th><th rowspan="1" colspan="1"><strong>PsC ctl</strong></th><th rowspan="1" colspan="1"><strong>PsA PsC</strong></th><th rowspan="1" colspan="1"><strong>Meta-OR</strong></th><th rowspan="1" colspan="1"><strong>Meta-p</strong></th><th rowspan="1" colspan="1"><strong>Meta-OR</strong></th><th rowspan="1" colspan="1"><strong>Meta-p</strong></th><th rowspan="1" colspan="1"><strong>OR</strong></th><th rowspan="1" colspan="1"><strong>p</strong></th></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>TNFRSF9</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs4908742</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">8,245,030</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1" align="center">0.3487</td><td rowspan="1" colspan="1" align="center">1.024</td><td rowspan="1" colspan="1">0.53</td><td rowspan="1" colspan="1" align="center">1.223</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">0.837</td><td rowspan="1" colspan="1" align="center">2.0&nbsp;× 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1"><em>IL23R</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs12044149</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">67,600,686</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1" align="center">0.2525</td><td rowspan="1" colspan="1" align="center">1.296</td><td rowspan="1" colspan="1" align="center">2.5&nbsp;× 10<sup>−12</sup></td><td rowspan="1" colspan="1" align="center">1.084</td><td rowspan="1" colspan="1">0.023</td><td rowspan="1" colspan="1" align="center">1.196</td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−4</sup></td></tr><tr><td rowspan="1" colspan="1"><em>IL23R</em></td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1">rs4655683</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">67,611,613</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.3368</td><td rowspan="1" colspan="1" align="center">1.319</td><td rowspan="1" colspan="1" align="center">7.8&nbsp;× 10<sup>−14</sup></td><td rowspan="1" colspan="1" align="center">1.162</td><td rowspan="1" colspan="1" align="center">1.6&nbsp;× 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">1.135</td><td rowspan="1" colspan="1">0.0075</td></tr><tr><td rowspan="1" colspan="1"><em>LCE3C/B</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs10888503</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">152,593,549</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.6413</td><td rowspan="1" colspan="1" align="center">1.085</td><td rowspan="1" colspan="1">0.018</td><td rowspan="1" colspan="1" align="center">1.247</td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−11</sup></td><td rowspan="1" colspan="1" align="center">0.870</td><td rowspan="1" colspan="1">0.0014</td></tr><tr><td rowspan="1" colspan="1"><em>HLA-C</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs1050414</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31,239,506</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1" align="center">0.1150</td><td rowspan="1" colspan="1" align="center">2.333</td><td rowspan="1" colspan="1" align="center">1.5&nbsp;× 10<sup>−70</sup></td><td rowspan="1" colspan="1" align="center">4.020</td><td rowspan="1" colspan="1" align="center">6.5&nbsp;× 10<sup>−225</sup></td><td rowspan="1" colspan="1" align="center">0.580</td><td rowspan="1" colspan="1" align="center">1.7&nbsp;× 10<sup>−21</sup></td></tr><tr><td rowspan="1" colspan="1"><em>HLA-C</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs12189871</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31,251,924</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1" align="center">0.1429</td><td rowspan="1" colspan="1" align="center">2.631</td><td rowspan="1" colspan="1" align="center">5.6&nbsp;× 10<sup>−86</sup></td><td rowspan="1" colspan="1" align="center">4.462</td><td rowspan="1" colspan="1" align="center">6.6&nbsp;× 10<sup>−248</sup></td><td rowspan="1" colspan="1" align="center">0.590</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−19</sup></td></tr><tr><td rowspan="1" colspan="1"><em>HLA-C</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs13214872</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31,255,008</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">G/C</td><td rowspan="1" colspan="1" align="center">0.5621</td><td rowspan="1" colspan="1" align="center">2.416</td><td rowspan="1" colspan="1" align="center">2.3&nbsp;× 10<sup>−99</sup></td><td rowspan="1" colspan="1" align="center">3.231</td><td rowspan="1" colspan="1" align="center">1.2&nbsp;× 10<sup>−206</sup></td><td rowspan="1" colspan="1" align="center">0.748</td><td rowspan="1" colspan="1" align="center">5.5&nbsp;× 10<sup>−9</sup></td></tr><tr><td rowspan="1" colspan="1"><em>HLA-C</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs13210419</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">31,266,977</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1" align="center">0.0929</td><td rowspan="1" colspan="1" align="center">2.778</td><td rowspan="1" colspan="1" align="center">1.8&nbsp;× 10<sup>−84</sup></td><td rowspan="1" colspan="1" align="center">5.004</td><td rowspan="1" colspan="1" align="center">3.5&nbsp;× 10<sup>−249</sup></td><td rowspan="1" colspan="1" align="center">0.555</td><td rowspan="1" colspan="1" align="center">5.7&nbsp;× 10<sup>−21</sup></td></tr><tr><td rowspan="1" colspan="1"><em>TNFAIP3</em></td><td rowspan="1" colspan="1">0</td><td rowspan="1" colspan="1">rs9321623</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1" align="center">137,958,265</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1" align="center">0.5461</td><td rowspan="1" colspan="1" align="center">1.201</td><td rowspan="1" colspan="1" align="center">5.9&nbsp;× 10<sup>−8</sup></td><td rowspan="1" colspan="1" align="center">1.025</td><td rowspan="1" colspan="1">0.44</td><td rowspan="1" colspan="1" align="center">1.172</td><td rowspan="1" colspan="1" align="center">2.2&nbsp;× 10<sup>−4</sup></td></tr></tbody></table></div><div id="largeobj_idm139913890616336" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4678416/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139913889598528"><p id="__p103" class="p p-first-last">Abbreviations are as follows: Chr, chromosome; RAF, risk allele frequency; PsV, psoriasis vulgaris; PsA, psoriatic arthritis; PsC, cutaneous psoriasis; ctl, control; OR, odds ratio.</p></div></div><div class="tblwrap-foot"><div id="tblfn5"><sup>a</sup>Round of stepwise conditional association analysis: 0, unconditional analysis; 1, first round stepwise conditional; 2, second round stepwise conditional, etc.</div></div><div class="tblwrap-foot"><div id="tblfn6"><sup>b</sup>For insertions or deletions of the hg19 human reference sequence, position of first base before the insertion point or of first base of the deleted sequence is shown, respectively.</div></div><div class="tblwrap-foot"><div id="tblfn7"><sup>c</sup>An X indicates, for each of the four analyzed phenotype comparisons (PsV case versus control subjects, PsA case versus control subjects, PsC case versus control subjects, PsA case versus PsC case subjects), that the marker is either the most strongly associated variant in the region with a p ≤ 1&nbsp;× 10<sup>−7</sup> (unconditional meta-analysis) or the best associated independent locus with a p ≤ 1&nbsp;× 10<sup>−6</sup> at the specified round of the stepwise conditional meta-analysis.</div></div><div class="tblwrap-foot"><div id="tblfn8"><sup>d</sup>Risk and nonrisk alleles based on results of association analysis of PsV case subjects and unaffected control subjects.</div></div><div class="tblwrap-foot"><div id="tblfn9"><sup>e</sup>Risk allele frequency for unaffected controls.</div></div><div class="tblwrap-foot"><div id="tblfn10"><sup>f</sup>For regions with a single independently associated variant, OR and p value are for a fixed-effects meta-analysis of PsA or PsC case versus unaffected control subjects where no other loci are included as conditioning covariates; for regions with multiple independently associated variants, OR and p value are for fixed-effects meta-analysis where the logistic regression model includes the other independent risk variants in the region as conditioning covariates.</div></div><div class="tblwrap-foot"><div id="tblfn11"><sup>g</sup>OR and nominal p value for a bootstrap test of the equality of the log(OR)s from meta-analyses of PsA versus control and PsC versus control subjects for the six study discovery sample; p values ≤ 0.05 after FDR adjustment are indicated in the final column.</div></div></div><p id="__p42">The PsA-specific signals upstream of <em>IL23R</em> and <em>TNFAIP3</em> are of particular interest, as they are independent of all published PsV associations. <a href="/pmc/articles/PMC4678416/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figures 3</span></a>A–3C clearly show two LD-independent peaks of association for <em>IL23R</em>—the PsA-specific peak near the 5′ end of the gene, and a peak associated with both PsA and PsC near the 3′ end of the gene. Similarly, <a href="/pmc/articles/PMC4678416/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Figures 3</span></a>D–3F illustrate two independent peaks of association for the <em>TNFAIP3</em> region—the PsA-specific peak well upstream of the gene, and a peak within <em>TNFAIP3</em> that is associated with both subphenotypes. The published PsV associations for each of these two regions are in LD with the second but not first peak of association.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC4678416/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913887967360" class="figure"><a href="/pmc/articles/PMC4678416/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4678416_gr3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4678416/bin/gr3.jpg"></a></div><div id="largeobj_idm139913887967360" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4678416/figure/fig3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC4678416/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Figure&nbsp;3</a></div><!--caption a7--><div class="caption"><p id="__p90">Comparison of PsA and PsC Associations for the <em>IL23R</em> and <em>TNFAIP3</em> Regions</p><p id="__p91">Negative log10-transformed p values from meta-analyses of association for the six discovery datasets are shown for 800-kb regions encompassing genes <em>IL23R</em> (A−C) and <em>TNFAIP3</em> (D−F). For each region two reference SNPs are labeled—the red and blue diamonds denote the variant that was most significantly associated for a test of PsA case versus control subjects or PsC case versus control subjects, respectively. For all other plotted variants, the color and shape of the symbol denotes which index SNP the variant is in highest LD with (red circles for the PsA index, blue triangles for the PsC index), and the intensity of the color indicates the magnitude of the pairwise LD r<sup>2</sup> coefficient. Shown are association results for PsA case versus control subjects (A, D) and for PsC case versus control subjects (B, E). Results for a conservative parametric test of the difference in log(OR) values from the PsA versus control and PsC versus control comparisons shown in (C) and (F). The bottom two panels give the physical positions of known genes in each region.</p></div></div></div><p id="__p43" class="p p-last">Finally, we attempted to confirm previous reports of non-MHC variants exhibiting significantly different association with PsA versus PsC. As shown in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S13</a>, we applied the bootstrap test to 10 of 12 such variants (the other two, a copy-number variant affecting <em>KIR2DS2</em> [MIM: <a href="/omim/604953" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">604953</a>] and a microsatellite within <em>IFNG</em>, were not in our discovery set). We could confirm only two of the previous findings, for rs2201841 near <em>IL23R</em> (OR = 1.10, p = 0.035) and for rs2476601 in <em>PTPN22</em> (OR = 1.16, p = 0.041). The original report for rs2201841 was based on a subset of the discovery&nbsp;samples in the current study; this SNP is not in LD&nbsp;with&nbsp;either of the PsA-specific variants uncovered here (r<sup>2</sup>&nbsp;&lt; 0.1), but is in moderate LD (r<sup>2</sup> = 0.62) with rs116238258, the second conditional PsV signal we detected in the <em>IL23R</em> region. Variants mapping near <em>CCR2</em>, <em>CSF2</em>, <em>IL13</em>, <em>IL12B</em>, <em>FBXL19</em>, and <em>ZNF816</em> showed no evidence of differential association with PsA versus PsC. Non-confirmation of the PsA specificity of the <em>CSF2</em> association was unexpected given the robustness of the original report.<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup> Although we were able to confirm the existence of a PsA signal near <em>CSF2</em> that is independent of association with nearby <em>IL13</em>, we also detected comparable <em>IL13</em>-independent association with PsC (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S16</a>). Our inability to confirm putative PsA-specific signals at <em>IL13</em> itself (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S12</a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S16</a>) is also noteworthy given the multiple positive reports for three different <em>IL13</em> SNPs.<sup><a href="#bib54" rid="bib54" class=" bibr popnode">54</a></sup><sup>, </sup><sup><a href="#bib55" rid="bib55" class=" bibr popnode">55</a></sup><sup>, </sup><sup><a href="#bib56" rid="bib56" class=" bibr popnode">56</a></sup><sup>, </sup><sup><a href="#bib57" rid="bib57" class=" bibr popnode">57</a></sup></p></div><div id="sec3.7" class="sec"><h3 id="sec3.7title">Disease Liability for Regions with Multiple Risk Variants</h3><p id="__p44" class="p p-first"><a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S14</a> lists, for each independent non-MHC risk variant for PsV, PsA, or PsC identified by this study, an LD-adjusted percent of variance in disease liability explained. The total proportion of disease liability explained by all non-MHC variants combined summed to 7.80%, 4.49%, and 4.91% for PsV, PsA, and PsC, respectively. These figures increased to 14.39%, 9.22%, and 12.10% when the contribution of the MHC region<sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup> was also considered.</p><p id="__p45" class="p p-last"><a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S14</a> includes variants in 29 distinct genomic regions. Sixteen of these regions contained only one independent signal, eight regions had two signals that were in substantial pairwise LD (r<sup>2</sup> = 0.46−1.00), and five regions (<em>IL12B</em>, <em>IL23R</em>, <em>NOS2</em>, <em>IFIH1</em>, and <em>TNFAIP3</em>) had two or more signals, at least one pair of which were in low pairwise LD (r<sup>2</sup> &lt; 0.3). <a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a> displays disease liability plots for the five regions with at least two independent signals. Plotting disease liabilities allowed a more meaningful comparison of relative risk for multiple variants in a region than did association statistics such as p values or ORs. These plots also allowed unified display for primary and conditional signals of different disease phenotypes, as well as visual representation of LD and spatial relationships of the risk variants with each other and with candidate genes. Spatially, the risk variants in <a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figure&nbsp;4</span></a> aggregated into one (<em>NOS2</em>), two (<em>IL23R</em>, <em>TNFAIP3</em>, <em>IFIH1</em>), or four (<em>IL12B</em>) distinct clusters. The number of LD-independent clusters per region varied from two to four based on the D′ measure and from two to five based on the r<sup>2</sup> measure. Most of the 19 plotted variants fell outside of the leading candidate gene for the region, but six were intronic (<em>IL23R</em>, <em>TNFAIP3</em>, <em>NOS2</em>, and <em>IFIH1</em>) and two exonic (<em>IFIH1</em>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139913888740784" class="figure"><a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4678416_gr4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is gr4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4678416/bin/gr4.jpg"></a></div><div id="largeobj_idm139913888740784" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4678416/figure/fig4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Figure&nbsp;4</a></div><!--caption a7--><div class="caption"><p id="__p92">Liability Plots for Five Regions with Multiple Independent Psoriasis Variants</p><p id="__p93">Liability plots for five genomic regions—near <em>IL23R</em> (A), <em>IFIH1</em> (B), <em>IL12B</em> (C), <em>TNFAIP3</em> (D), and <em>NOS2</em> (E)—with multiple independent variants associated with psoriasis or its cutaneous and joint subphenotypes. Percent variance in disease liability explained by each variant was computed and adjusted for LD dependencies with other variants in the region by the method of Ho et&nbsp;al.<sup><a href="#bib69" rid="bib69" class=" bibr popnode">69</a></sup> The label above each plotted symbol indicates both the associated phenotype (P = PsV, A = PsA, C = PsC) and the round of conditional association analysis (0&nbsp;= unconditional, 1&nbsp;= first round conditional, 2&nbsp;= second round conditional, etc.). Grey fill for a plotted symbol indicates that the association ORs for the variant in the six-study meta-analysis of PsA case versus unaffected control subjects and PsC case versus unaffected control subjects were not significantly different; red fill indicates a significantly larger OR for the PsA versus control comparison. LD blocks, shown with two alternating shades of green, were delineated by the D′ confidence interval method<sup><a href="#bib68" rid="bib68" class=" bibr popnode">68</a></sup> using genotypes for all SNPs with MAF ≥ 0.05 in the 379 individuals of European ancestry in phase 1 of the 1000 Genomes Project.<sup><a href="#bib65" rid="bib65" class=" bibr popnode">65</a></sup> The bands for the r<sup>2</sup> clumps, shown in two shades of purple when necessary, encompass clusters of markers with an average pairwise r<sup>2</sup> LD coefficient ≥ 0.3.</p></div></div></div></div><div id="sec3.8" class="sec sec-last"><h3 id="sec3.8title">Functional Annotation</h3><p id="__p46" class="p p-first">We investigated potential functional consequences of associated variants for eight regions where this study has enhanced our knowledge of the genetics of psoriasis and PsA: <em>CDKAL1</em> (PsV variant), <em>TNFRSF9</em> and <em>LCE3C/B</em> (PsC-specific association), <em>IL23R</em> and <em>TNFAIP3</em> (PsA-specific associations independent of known PsV variants), and <em>IFIH1</em>, <em>IL12B</em>, and <em>NOS2</em> (secondary PsV, PsC, and PsA signals independent of previously reported risk variants).</p><p id="__p47">Dendrograms for LD-based hierarchical clustering (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S17</a>) show that 7 of the 22 associated variants detected by this study in these regions were not in strong LD (r<sup>2</sup> ≥ 0.7) with published variants for any genetic trait. These include the primary PsV signal near <em>TNFRSF9</em>, one of the secondary <em>IL23R</em> PsV signals, the primary PsV and secondary PsA signals in the <em>IFIH1</em> region, the primary PsV, PsA, and PsC as well as one of the secondary PsV signals near <em>IL12B</em>, and the secondary PsV signal near <em>NOS2</em>. Of the remaining 15 risk variants, all but the <em>CDKAL1</em> SNP and the PsA-specific signal upstream of <em>TNFAIP3</em> were in strong LD with previously reported variants for PsV or PsA; other diseases sharing these variants or variants in strong LD with them include Crohn disease (MIM: <a href="/omim/266600" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">266600</a>), ulcerative colitis (MIM: <a href="/omim/266600" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">266600</a>), inflammatory bowel disease (MIM: <a href="/omim/266600" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">266600</a>), ankylosing spondylitis (MIM: <a href="/omim/106300" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">106300</a>), multiple sclerosis (MIM: <a href="/omim/126200" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">126200</a>), type 1 diabetes (MIM: <a href="/omim/222100" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">222100</a>), selective IgA deficiency (MIM: <a href="/omim/137100" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">137100</a>), systemic lupus erythematosus (MIM: <a href="/omim/152700" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">152700</a>), and vitiligo (MIM: <a href="/omim/606579" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">606579</a>). All of these immune-mediated diseases have been reported to share susceptibility signals with psoriasis.<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib87" rid="bib87" class=" bibr popnode">87</a></sup><sup>, </sup><sup><a href="#bib88" rid="bib88" class=" bibr popnode">88</a></sup></p><p id="__p48">Overlap of association signals with various functional features is presented in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S15</a>. Because the most significantly associated variants are not necessarily causative, for each signal we computed a Bayesian credible set that had a 95% posterior probability (PP) of including the causal disease-associated variant.<sup><a href="#bib72" rid="bib72" class=" bibr popnode">72</a></sup><sup>, </sup><sup><a href="#bib73" rid="bib73" class=" bibr popnode">73</a></sup> To accommodate uncertainty about which variant drives an association signal, we annotated by computing weighted averages of each feature over all variants in the credible set, using PPs as weights. Most of the association signals in the eight regions were intronic or intergenic. Only the two independent PsA association signals in <em>IFIH1</em> had a PP &gt; 0.50 of being exonic; the index variants for these signals are missense SNPs with a high likelihood (PP = 0.79 and 0.97) of being causal. Another variant likely to influence protein expression is a 32-kb indel spanning <em>LCE3C</em> and <em>LCE3B</em>, which is in complete LD with the index SNP for this region and is known to be associated with psoriasis.<sup><a href="#bib85" rid="bib85" class=" bibr popnode">85</a></sup><sup>, </sup><sup><a href="#bib89" rid="bib89" class=" bibr popnode">89</a></sup> Many of the association signals fell within regions bearing evidence of regulatory function. The strongest such evidence was for the PsA-specific signal near <em>IL23R</em>, which was in a region with promoter histone marks, enhancer histone marks, and DNase I hypersensitive sites for a weighted average of 40, 11, and 12 different cell types, respectively, among 127 reference epigenomes. The evidence for the index variant of this signal, rs12044149, was even more striking: 67, 15, and 19 cell types, respectively, where the SNP was located within chromatin having promoter histone marks, enhancer histone marks, and DNase I hypersensitive sites; location within a CpG island; and a 2b (“likely to affect binding”) RegulomeDB score. Another association signal with multiple lines of evidence for regulatory function was the likely causative SNP for the best PsV and PsC association with <em>NOS2</em> (rs28998802), which alters binding motifs for seven transcription factors (TFs), is scored as likely to affect TF binding, and contains DNase I hypersensitive sites in 11 cell types.</p><p id="__p49"><a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S18</a> shows overlap of association signals in the eight regions of interest with chromatin states for 33 cell types relevant to psoriasis and PsA. As before, the PsA-specific signal near <em>IL23R</em> appears to be located within an important regulatory region, showing promoter-proximal activity in many relevant cell types, including monocytes, neutrophils, mononuclear cells, most T helper cells, CD8<sup>+</sup> T&nbsp;cells, fibroblasts, keratinocytes, melanocytes, mesenchymal stem cell-derived cells, and osteoblasts. The strongest secondary <em>IL12B</em> signal for PsV and PsA (P1/A1) shows enhancer activity in NK cells, in all three keratinocyte cell lines and in T regulatory cells, and also in activated T helper cells. The primary PsV signal for <em>IFIH1</em>, which is intronic, is in a region of transcriptional activity for most cell types, but exhibits enhancer activity in memory T helper and T regulatory cells. In contrast, both <em>NOS2</em> signals are located in regions that are silenced by Polycomb group proteins in the majority of relevant cell types.</p><p id="__p50" class="p p-last">Regulatory mechanisms often affect RNA expression. Accordingly, we examined differential expression in psoriasis lesions versus skin from unaffected individuals for all genes within 500 kb of the index signal in each of the eight regions of interest, as well as evidence that the associated regions contain <em>cis</em>-eQTLs for any of these genes (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S16</a>). Most of the plausible candidate genes in these eight regions showed a substantial (fold-change [FC] &gt; 2 or &lt; 0.5) and significant (p &lt; 1&nbsp;× 10<sup>−6</sup>) difference in RNA expression between lesional and normal skin. Expression of the inflammation-related genes <em>IFIH1</em>, <em>IL12B</em>, <em>IL23R</em>, <em>NOS2</em>, and <em>TNFRSF9</em> was higher in lesional compared to normal skin (FC = 2.8, ∞, 4.4, 50, and 6.0, respectively). <em>TNFAIP3</em> was the only inflammation-related candidate gene showing no sign of differential expression. The gene-dense <em>LCE3C/B</em> region teems with plausible candidate genes; most (25/37) were dramatically up- or downregulated in psoriatic skin. After correction for multiple testing, only the PsV signal in the <em>LCE3C/B</em> region provided evidence for being an eQTL in skin tissue. The risk alleles for all 27 variants in the 95% Bayesian credible set for this locus showed a highly significant negative correlation with mRNA levels of both <em>LCE3C</em> and <em>LCE3B</em> (p ≤ 1&nbsp;× 10<sup>−24</sup> and p ≤ 2&nbsp;× 10<sup>−12</sup>, respectively), and the peak eQTL variant for each gene (rs67488802 and rs10888503, respectively) was in the credible set. However, the risk alleles of the associated variants are all in very strong LD (r<sup>2</sup> ≥ 0.98) with a 32-kb deletion spanning <em>LCE3C</em> and <em>LCE3B</em> and hence are simply markers for the absence of these two genes. The associated variants in this region also showed a significant&nbsp;(p&nbsp;= 1.4&nbsp;× 10<sup>−5</sup>) positive correlation with <em>LCE3A</em> expression, but the peak eQTL for <em>LCE3A</em> (rs138035792, p&nbsp;= 1.6&nbsp;× 10<sup>−6</sup>) is not in the 95% Bayesian credible set for the association signal.</p></div></div><div id="sec4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="sec4title">Discussion</h2><p id="__p51" class="p p-first">This study is a well-powered and comprehensive examination of the genetic associations of PsA and PsC. We found five independent variants that are differentially associated with these two subphenotypes, all in regions of known association with psoriasis.</p><p id="__p52">A signal in the class I MHC between <em>HLA-C</em> and <em>HLA-B</em> exhibited by far the greatest degree of differential association in our analysis, with carriage of the PsV risk allele nearly halving the odds of developing PsA in addition to PsC. Previous work using the same six discovery samples of this study demonstrated that this differential association is driven by a glutamine residue at position 45 of HLA-B, which increased PsA versus PsC susceptibility more than any other classical HLA protein allele, amino acid, SNP, or indel in the region.<sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup> This amino acid is situated in the antigen binding groove of HLA-B, with potential influence on receptor cell-surface expression or antigen peptide binding or presentation.<sup><a href="#bib44" rid="bib44" class=" bibr popnode">44</a></sup><sup>, </sup><sup><a href="#bib90" rid="bib90" class=" bibr popnode">90</a></sup></p><p id="__p53">The PsA-specific association near <em>IL23R</em> was previously reported by us<sup><a href="#bib91" rid="bib91" class=" bibr popnode">91</a></sup> and has been recently confirmed.<sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup> The index variant in both studies is the same (rs12044149), increasing the likelihood that this SNP is causal. However, although the confirmatory study demonstrated that this association is independent of previously reported psoriasis associations, our study demonstrates that the SNP itself is significantly more strongly associated with PsA than PsC. SNP rs12044149 lies in a region with promoter-proximal activity in diverse cell types, including many relevant to psoriasis (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S15</a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S18</a>). We found no evidence that this SNP is a <em>cis</em>-eQTL for <em>IL23R</em> mRNA levels in skin tissue, but IL23R-expressing immunocytes likely to be of relevance to PsA (including IL17A-expressing CD4<sup>+</sup> and CD8<sup>+</sup> T&nbsp;cells<sup><a href="#bib92" rid="bib92" class=" bibr popnode">92</a></sup>) are relatively minor components of normal and psoriatic skin.<sup><a href="#bib93" rid="bib93" class=" bibr popnode">93</a></sup> This SNP is only 32&nbsp;bp from the transcription start site (TSS) of uncharacterized protein-coding gene <em>C1orf141</em> (versus 31.5 kb from the TSS of <em>IL23R</em>), and&nbsp;its risk allele also shows a nominally significant (p = 0.0036) positive correlation with <em>IL12RB2</em> (MIM: <a href="/omim/601642" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">601642</a>) mRNA levels in skin tissue, suggesting that its risk effect might be mediated via regulation of genes in this region other than <em>IL23R</em>. Our results also support the existence of at least three independent effects at the <em>IL23R</em> locus for psoriatic disease (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S11</a>, <a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures 4</span></a> and&nbsp;<a href="#mmc1" rid="mmc1" class=" supplementary-material">S17</a>).</p><p id="__p54">We also discovered a PsA-specific variant (rs9321623) in the <em>TNFAIP3</em> region, which is LD independent of the best PsV and PsC associations. This SNP is one of a spatial cluster of seven (rs9321623, rs7769192, rs13192841, rs2327832, rs10499194, rs17264332, and rs6920220) associated with a variety of immune-mediated diseases, which spans a 48-kb interval that is intergenic and situated more than 180 kb upstream of the TSS of <em>TNFAIP3</em>. However, only one of the risk variants for these other diseases (rs7769192, a secondary signal for multiple sclerosis<sup><a href="#bib94" rid="bib94" class=" bibr popnode">94</a></sup>) is allied with the PsA variant (LD-r<sup>2</sup> = 0.85). When aggregated over all variants in the 95% Bayesian credible set, there is no compelling evidence that this PsA association signal lies in a regulatory region (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S15</a>, <a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S18</a>). The index variant does, however, fall within an enhancer element in many cell types relevant to psoriasis and PsA, including CD14<sup>+</sup> monocytes, mesenchymal stem cells, fibroblasts, keratinocytes, and osteoblasts (data not shown). However, <em>TNFAIP3</em> is not differentially expressed in normal versus psoriatic skin, and the PsA-specific signal shows no evidence of being an eQTL for <em>TNFAIP3</em> in skin-derived transcriptomes (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S16</a>).</p><p id="__p55">We established that a known psoriasis signal within a cluster of 19 LCE genes in the epidermal differentiation complex on chromosome 1q21.3<sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup><sup>, </sup><sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> is more strongly associated with PsC than PsA. Published evidence for association of PsA with variants in the <em>LCE</em> region is mixed,<sup><a href="#bib95" rid="bib95" class=" bibr popnode">95</a></sup><sup>, </sup><sup><a href="#bib96" rid="bib96" class=" bibr popnode">96</a></sup><sup>, </sup><sup><a href="#bib97" rid="bib97" class=" bibr popnode">97</a></sup><sup>, </sup><sup><a href="#bib98" rid="bib98" class=" bibr popnode">98</a></sup><sup>, </sup><sup><a href="#bib99" rid="bib99" class=" bibr popnode">99</a></sup> but a meta-analysis of 2,400 PsA case and 7,779 control subjects from five European countries found&nbsp;a nominally significant association (OR = 1.14, p&nbsp;= 0.00041).<sup><a href="#bib95" rid="bib95" class=" bibr popnode">95</a></sup> We also found a nominally significant association of PsA with PsV index variant rs10888503 (OR = 1.085, p = 0.021), but the association with PsC was far stronger (OR = 1.247, p = 1.8&nbsp;× 10<sup>−11</sup>). The index SNP is in perfect LD with a 32 kb deletion spanning <em>LCE3C</em> and <em>LCE3B</em> that has been shown to be associated with psoriasis to a degree equal to or better than any SNP surrogate.<sup><a href="#bib85" rid="bib85" class=" bibr popnode">85</a></sup><sup>, </sup><sup><a href="#bib89" rid="bib89" class=" bibr popnode">89</a></sup> We found a highly significant positive correlation between copy number and mRNA levels of <em>LCE3C</em> and <em>LCE3B</em> in skin tissue (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S16</a>). Expression of these two genes was strongly upregulated in psoriasis lesions compared to normal skin (72.8- and 12.5-fold increase, respectively), and they encode late cornified envelope structural proteins thought to be involved in skin barrier repair after injury or inflammation.<sup><a href="#bib100" rid="bib100" class=" bibr popnode">100</a></sup> It has been theorized that lower levels of these LCE3 proteins due to the deletion impairs response of the skin barrier to exogenous agents, thereby facilitating the systemic skin inflammation characteristic of psoriasis.<sup><a href="#bib101" rid="bib101" class=" bibr popnode">101</a></sup> It has also been demonstrated that the 32 kb indel possesses epidermal-specific developmental in&nbsp;vivo enhancer activity,<sup><a href="#bib102" rid="bib102" class=" bibr popnode">102</a></sup> which might also play a role in the association with disease.</p><p id="__p56">The final differential association we detected is tagged by&nbsp;SNP rs4908742, which lies between <em>ERRFI1</em> (MIM: <a href="/omim/608069" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">608069</a>) and <em>SLC45A1</em> (MIM: <a href="/omim/605763" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605763</a>) on chromosome 1p36.23. This association signal appears to be specific to PsC (OR = 1.223, p = 2.2&nbsp;× 10<sup>−8</sup>), with no detectable association to PsA (OR = 1.028, p = 0.53). Although this region contains susceptibility variants for several other autoimmune diseases, none of them are in LD with rs4908742 or the published<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> psoriasis variant (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S17</a>). There are at least four plausible candidate genes in this region (<em>TNFRSF9</em>, <em>ERRFI1</em>, <em>SLC45A1</em>, and <em>RERE</em> [MIM: <a href="/omim/605226" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">605226</a>]),<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> all of which are differentially expressed in both normal and psoriatic skin, especially <em>TNFRS9</em>, which is 6-fold upregulated, and <em>SLC45A1</em>, which is 2.4-fold downregulated in psoriasis lesions. The 95% Bayesian credible set for this locus contains 59 variants, and the PP for the index variant is only 0.091. Thus, further refinement of this PsC association signal is needed before the target or mechanism of action of the risk variant in this region can be determined.</p><p id="__p57">Other than a nominally significant greater association with PsC than PsA for a known PsV variant within <em>IL23R</em><sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup> and a recently published PsA signal in <em>PTPN22</em>,<sup><a href="#bib48" rid="bib48" class=" bibr popnode">48</a></sup> we could not confirm any of the prior reports of differential association of these two subphenotypes with non-MHC genetic variants (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S13</a>) even though this study had greater power and used a more stringent definition of PsC than most prior reports.</p><p id="__p58">Given the marked distinction in phenotype, heritability, and incidence of PsA and PsC, it is remarkable that we can detect or confirm only five differentially associated signals, all within known regions of PsV association. Much of this might stem from the fact that this study, though based on a large sample, is still underpowered compared to recently published case-control studies of PsV and other genetic traits.<sup><a href="#bib103" rid="bib103" class=" bibr popnode">103</a></sup> The consequences of this disparity in power are also reflected by this study when comparing its greater effective sample size for PsV-control versus PsA-control and PsC-control comparisons (22,128 versus 10,604 and 9,996) with the resulting number of independent risk variants identified (40 versus 20 and 19, respectively) and the total percent of variance in phenotypic liability explained (14.4% versus 9.2% and 12.1%, respectively). Another limitation of this study is that two of the six discovery studies had targeted rather than GW coverage, which biases discovery to loci already implicated in psoriasis and other autoimmune diseases. It is likely that larger GWASs of PsA and PsC will uncover more variants specific to each subphenotype, aided by enhanced imputation resources such as the Haplotype Reference Consortium.</p><p id="__p59">In addition to uncovering differential associations with PsA and PsC, we discovered GW significant association with PsA for three regions (near <em>IFNLR1</em>, <em>IFIH1</em>, and <em>NFKBIA</em>) and with PsC for five regions (<em>TNFRSF9</em>, <em>LCE3C/B</em>, <em>TRAF3IP2</em>, <em>IL23A</em>, and <em>NFKBIA</em>). This increases the list of firmly established PsA and PsC risk regions to 16 and 12, respectively.</p><p id="__p60">We also substantiated a PsV susceptibility signal within the 5′ portion of the 700 kb <em>CDKAL1</em> gene. Nominal association of psoriasis to a SNP in this region has been noted previously.<sup><a href="#bib104" rid="bib104" class=" bibr popnode">104</a></sup><sup>, </sup><sup><a href="#bib105" rid="bib105" class=" bibr popnode">105</a></sup> The index SNP for this signal (rs4712528) is in strong LD with a risk variant for Crohn disease, but it is completely LD independent of a susceptibility variant for inflammatory bowel disease and a host of SNPs that increase risk for type 2 diabetes (MIM: <a href="/omim/125853" data-ga-action="click_feat_suppl" ref="reftype=extlink-omim&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7COmim" target="_blank">125853</a>) and an array of other metabolic traits (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Figure&nbsp;S17</a>). <em>CDKAL1</em> was expressed at significantly lower levels in psoriatic lesions than in normal skin (FC = 0.77, p = 3.7&nbsp;× 10<sup>−14</sup>), in keeping with an observation that <em>CDKAL1</em> transcription is downregulated when immune cells expressing the gene (especially CD4<sup>+</sup>, CD8<sup>+</sup>, and CD19<sup>+</sup> lymphocytes) are activated.<sup><a href="#bib105" rid="bib105" class=" bibr popnode">105</a></sup> We found no evidence that this signal is an eQTL for <em>CDKAL1</em> or that it maps to a regulatory feature, but the relatively low PP of the index variant (0.22) and the long interval spanned by the 95% Bayesian credible set (568 kb) indicate that deciphering the role of this locus in psoriasis will depend upon better refinement of the association signal.</p><p id="__p61">Only 2 of 17 previously unreported associations were replicated, including only one of three signals achieving GW significance in the discovery analysis (<a href="/pmc/articles/PMC4678416/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>). After adjusting discovery results for the well-known winner’s curse<sup><a href="#bib67" rid="bib67" class=" bibr popnode">67</a></sup> and correction for multiple testing, none of the 17 markers showed a significant difference between the&nbsp;ascertainment-adjusted OR of the discovery set and the&nbsp;observed OR of the replication set. Post hoc power tests indicated that only for rs4891505 in <em>LOC100505817</em> was there high confidence that power to replicate exceeds 80%, although the predicted power for the successfully replicated <em>CAMK2G</em> variant was respectable (43%). These results show that although effect sizes in the discovery and replication sets are similar for the candidate variants, the replication set is underpowered to replicate most of the true positives among them. Power to replicate <em>NFKBIZ</em> was also reduced by the need to substitute the second-best associated marker for the index variant (compare p for these two variants in <a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S10</a>). The PsA versus PsC signal in <em>LOC100505817</em> is probably a false positive, given its much reduced significance when including the PsA GWAS in the discovery analysis and its poor showing in the replication set.</p><p id="__p62">Conditional association analysis revealed multiple independent risk variants in regions of known psoriasis association, several of which have not been reported previously (<a href="#mmc1" rid="mmc1" class=" supplementary-material">Table S11</a>, <a href="/pmc/articles/PMC4678416/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Figures 4</span></a> and <a href="#mmc1" rid="mmc1" class=" supplementary-material">S17</a>). Besides signals near <em>IL23R</em> and <em>TNFAIP3</em> that have already been discussed, we detected multiple variants for <em>IFIH1</em>, <em>NOS2</em>, and <em>IL12B</em>. Three independent associations were found for the <em>IFIH1</em> region—a PsV signal tagged by rs17715343 in this study and rs17716942 in previous studies;<sup><a href="#bib23" rid="bib23" class=" bibr popnode">23</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> a signal associated with both PsA (rs1990760) and PsV (rs211485) that also poses risk for type 1 diabetes, vitiligo, systemic lupus erythematosus, ulcerative colitis, and irritable bowel syndrome; and a PsA signal at rs3747517. The PsA signals at rs1990760 and rs3747517 are only 4.7 kb apart, but their&nbsp;risk alleles usually occur on different haplotypes (r&nbsp;= −0.73). The <em>IL12B</em> region harbors at least five independent risk variants for psoriatic disease. One cluster contains most of the previously reported associations for PsV and PsA (rs4921482, rs2546890, rs4379175, rs3213094, rs2082412, rs7709212, rs6887695, rs3212227, rs228831),<sup><a href="#bib18" rid="bib18" class=" bibr popnode">18</a></sup><sup>, </sup><sup><a href="#bib21" rid="bib21" class=" bibr popnode">21</a></sup><sup>, </sup><sup><a href="#bib22" rid="bib22" class=" bibr popnode">22</a></sup><sup>, </sup><sup><a href="#bib25" rid="bib25" class=" bibr popnode">25</a></sup><sup>, </sup><sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup><sup>, </sup><sup><a href="#bib29" rid="bib29" class=" bibr popnode">29</a></sup><sup>, </sup><sup><a href="#bib45" rid="bib45" class=" bibr popnode">45</a></sup><sup>, </sup><sup><a href="#bib72" rid="bib72" class=" bibr popnode">72</a></sup> along with the strongest secondary PsV and PsA (rs62377586) and PsC (rs4921485) signals from this study, as well as risk SNPs for multiple sclerosis (rs2546890) and Crohn disease (rs6556412). A&nbsp;second group includes a secondary PsV, PsA, and PsC signal from this study, rs12188300, which is known to be associated with PsA<sup><a href="#bib33" rid="bib33" class=" bibr popnode">33</a></sup> and PsV.<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> Three additional clusters contain variants not previously implicated in psoriasis susceptibility: a group with the best signal for PsV, PsA, and PsC (rs918520), a secondary PsV signal (rs6870256), and a group containing secondary PsV signal rs953861, secondary PsA signal rs1592975, plus SNPs associated with ankylosing spondylitis and Crohn disease. The <em>NOS2</em> region contains three independent psoriasis associations—rs28998802, the best PsV and PsC signal of this study as well in a large meta-analysis of PsV;<sup><a href="#bib27" rid="bib27" class=" bibr popnode">27</a></sup> a known PsV signal tagged by rs4795067;<sup><a href="#bib24" rid="bib24" class=" bibr popnode">24</a></sup> and rs2301369, a newly implicated secondary PsV signal.</p><p id="__p63" class="p p-last">Data mining suggests plausible functional consequences for some of the variants in the eight regions where this study has added to our knowledge of genetic susceptibility to psoriasis and its subphenotypes. These include variants that might mediate their risk via protein alterations (PsV-associated deletion of <em>LCE3B</em> and <em>LCE3C</em> and two independent PsA-associated missense SNPs in <em>IFIH1</em>), promoter-proximal regulation (PsA-specific <em>IL23R</em> SNP), enhancer-mediated regulation (PsV signal in <em>IFIH1</em>, strongest secondary PsV and PsA signal in <em>IL12B</em> region), and reversal of Polycomb repression (two independent <em>NOS2</em> variants). Given the great cost and labor involved in experimental follow-up studies, it is crucial to identify the causative variants driving the observed associations. This will require larger samples of affected case subjects that are densely genotyped or deeply sequenced for the entire genome. Correlation of the resulting finely mapped genetic information with transcriptomes and epigenomes of relevant cell types will aid prediction and therapy of psoriasis and its cutaneous and joint manifestations.</p></div><div id="ack0010" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="ack0010title">Acknowledgments</h2><div class="sec"><p id="__p84">We thank Soumya Raychaudhuri, Aaron Deutsch, and Yuki Okada for sharing their implementation to calculate the percent variance explained for a multi-allelic variant under a disease liability threshold model. We also thank Fawnda Pellett and Remy Pollock from The Centre for Prognostic Studies in the Rheumatic Diseases in Toronto, Canada, for their expert technical assistance, and the Barbara and Neal Henschel Charitable Foundation for their support of the National Psoriasis Victor Henschel Biobank. This work was supported by awards from the NIH (R01AR042742, R01AR050511, R01AR054966, R01AR063611, and R01AR065183 to J.T.E. and R01AR050266 to A.M.B.) and the German Research Foundation. A.F. and E.E. received infrastructure support through the DFG Clusters of Excellence 306 “Inflammation at Interfaces” and are supported by the German Ministry of Education and Research (BMBF) through the e:Med sysINFLAME grant. J.E.G. was supported by the Doris Duke Foundation (grant #2013106) and the Taubman Medical Research Institute as the Frances and Kenneth Eisenberg Emerging Scholar. The Estonian Psoriasis cohort was supported by institutional research funding IUT20-46 of the Estonian Ministry of Education and Research, by the Centre of Translational Genomics of University of Tartu (SP1GVARENG) and by the European Regional Development Fund (Centre of Translational Medicine, University of Tartu). J.T.E. is supported by the Ann Arbor Veterans Affairs Hospital.</p></div></div><div id="idm139913887819808" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913887819808title">Notes</h2><p id="misc0010">Published: November 25, 2015</p></div><div id="idm139913964700160" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913964700160title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="app2"></p><p id="__p85" class="p p-first-last">Supplemental Data include 18 figures and 16 tables and can be found with this article online at <a href="//dx.doi.org/10.1016/j.ajhg.2015.10.019" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">http://dx.doi.org/10.1016/j.ajhg.2015.10.019</a>.</p><p></p></div></div><div id="app1" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app1title">Accession Numbers</h2><!--/article/back/sec/--><p id="__p64" class="p p-first-last">Genotypes for the PsA GWAS have been deposited in dbGaP under accession number phs000982.v1.p1.</p></div><div id="app4" class="tsec bk-sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app4title">Web Resources</h2><!--/article/back/sec/--><p id="__p65" class="p p-first-last">The URLs for data presented herein are as follows:</p><ul class="simple" style="list-style-type:none"><!--
list-behavior=simple
prefix-word=
mark-type=none
max-label-size=0
--><li id="u0010"><p id="__p66">1000 Genomes Project, <a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20110521/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/20110521/</a></p></li><li id="u0015"><p id="__p67">ANNOVAR, <a href="http://www.openbioinformatics.org/annovar/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.openbioinformatics.org/annovar/</a></p></li><li id="u0020"><p id="__p68">CADD, <a href="http://cadd.gs.washington.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cadd.gs.washington.edu/</a></p></li><li id="u0025"><p id="__p69">dbGaP, <a href="http://www.ncbi.nlm.nih.gov/gap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gap</a></p></li><li id="u0030"><p id="__p70">Gene Expression Omnibus (GEO), <a href="http://www.ncbi.nlm.nih.gov/geo/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/geo/</a></p></li><li id="u0035"><p id="__p71">GWAS Catalog (28 August 2015 release), <a href="http://www.ebi.ac.uk/gwas/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ebi.ac.uk/gwas/</a></p></li><li id="u0040"><p id="__p72">HaploReg, <a href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</a></p></li><li id="u0045"><p id="__p73">MACH, <a href="http://www.sph.umich.edu/csg/abecasis/MACH/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sph.umich.edu/csg/abecasis/MACH/</a></p></li><li id="u0050"><p id="__p74">MERLIN, <a href="http://csg.sph.umich.edu/abecasis/Merlin/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csg.sph.umich.edu/abecasis/Merlin/</a></p></li><li id="u0055"><p id="__p75">Minimac, <a href="http://genome.sph.umich.edu/wiki/Minimac" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/Minimac</a></p></li><li id="u0060"><p id="__p76">OMIM, <a href="http://www.omim.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.omim.org/</a></p></li><li id="u0065"><p id="__p77">PLINK, <a href="http://pngu.mgh.harvard.edu/%7Epurcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a></p></li><li id="u0070"><p id="__p78">RegulomeDB, <a href="http://RegulomeDB.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://RegulomeDB.org/</a></p></li><li id="u0075"><p id="__p79">RelativeFinder, <a href="http://genome.sph.umich.edu/wiki/RelativeFinder" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.sph.umich.edu/wiki/RelativeFinder</a></p></li><li id="u0080"><p id="__p80">Roadmap Epigenomics Project, <a href="http://egg2.wustl.edu/roadmap/web_portal/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://egg2.wustl.edu/roadmap/web_portal/</a></p></li><li id="u0085"><p id="__p81">The Haplotype Reference Consortium, <a href="http://www.haplotype-reference-consortium.org/home" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.haplotype-reference-consortium.org/home</a></p></li><li id="u0090"><p id="__p82">UCSC Genome Browser (February 2009 assembly, GRCh37/hg19), <a href="http://genome.ucsc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genome.ucsc.edu</a></p></li></ul><p></p></div><div id="app3" class="tsec bk-sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="app3title">Supplemental Data</h2><!--/article/back/sec/--><div class="sec suppmat" id="mmc1"><!--caption a9--><strong>Document S1. Figures S1–S18 and Tables S1–S7 and S9–S16:</strong><div class="sup-box half_rhythm" id="idm139913887010128"><a href="/pmc/articles/PMC4678416/bin/mmc1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(7.1M, pdf)</sup></div></div><div class="sec suppmat" id="mmc2"><!--caption a9--><strong>Table S8. Discovery Study Association Statistics at Previously Reported Genome-wide Significant Risk Variants for PsV, PsA, and PsC:</strong><div class="sup-box half_rhythm" id="idm139913887007408"><a href="/pmc/articles/PMC4678416/bin/mmc2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(47K, xlsx)</sup></div></div><div class="sec suppmat" id="mmc3"><!--caption a9--><strong>Document S2. Article plus Supplemental Data:</strong><div class="sup-box half_rhythm" id="idm139913887004832"><a href="/pmc/articles/PMC4678416/bin/mmc3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11M, pdf)</sup></div></div><p></p></div><div id="idm139913890396032" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139913890396032title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Chandran V., Raychaudhuri S.P. Geoepidemiology and environmental factors of psoriasis and psoriatic arthritis. <span><span class="ref-journal">J.&nbsp;Autoimmun. </span>2010;<span class="ref-vol">34</span>:J314–J321.</span> [<a href="/pubmed/20034760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Autoimmun.&amp;title=Geoepidemiology+and+environmental+factors+of+psoriasis+and+psoriatic+arthritis&amp;author=V.+Chandran&amp;author=S.P.+Raychaudhuri&amp;volume=34&amp;publication_year=2010&amp;pages=J314-J321&amp;pmid=20034760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">Gladman D.D.  Epidemiology. In: Gordon K.B., Ruderman E.M., editors. <span class="ref-journal">Psoriasis and Psoriatic Arthritis: An Integrated Approach.</span> Springer; Berlin, New York: 2005. pp. 57–65. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Psoriasis+and+Psoriatic+Arthritis:+An+Integrated+Approach&amp;author=D.D.+Gladman&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Husted J.A., Thavaneswaran A., Chandran V., Eder L., Rosen C.F., Cook R.J., Gladman D.D. Cardiovascular and other comorbidities in patients with psoriatic arthritis: a comparison with patients with psoriasis. <span><span class="ref-journal">Arthritis Care Res. (Hoboken) </span>2011;<span class="ref-vol">63</span>:1729–1735.</span> [<a href="/pubmed/21905258" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Care+Res.+(Hoboken)&amp;title=Cardiovascular+and+other+comorbidities+in+patients+with+psoriatic+arthritis:+a+comparison+with+patients+with+psoriasis&amp;author=J.A.+Husted&amp;author=A.+Thavaneswaran&amp;author=V.+Chandran&amp;author=L.+Eder&amp;author=C.F.+Rosen&amp;volume=63&amp;publication_year=2011&amp;pages=1729-1735&amp;pmid=21905258&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Jamnitski A., Symmons D., Peters M.J., Sattar N., McInnes I., Nurmohamed M.T. Cardiovascular comorbidities in patients with psoriatic arthritis: a systematic review. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2013;<span class="ref-vol">72</span>:211–216.</span> [<a href="/pubmed/22532629" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Cardiovascular+comorbidities+in+patients+with+psoriatic+arthritis:+a+systematic+review&amp;author=A.+Jamnitski&amp;author=D.+Symmons&amp;author=M.J.+Peters&amp;author=N.+Sattar&amp;author=I.+McInnes&amp;volume=72&amp;publication_year=2013&amp;pages=211-216&amp;pmid=22532629&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Yeung H., Takeshita J., Mehta N.N., Kimmel S.E., Ogdie A., Margolis D.J., Shin D.B., Attor R., Troxel A.B., Gelfand J.M. Psoriasis severity and the prevalence of major medical comorbidity: a population-based study. <span><span class="ref-journal">JAMA Dermatol. </span>2013;<span class="ref-vol">149</span>:1173–1179.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3800487/">PMC free article</a>]</span> [<a href="/pubmed/23925466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Dermatol.&amp;title=Psoriasis+severity+and+the+prevalence+of+major+medical+comorbidity:+a+population-based+study&amp;author=H.+Yeung&amp;author=J.+Takeshita&amp;author=N.N.+Mehta&amp;author=S.E.+Kimmel&amp;author=A.+Ogdie&amp;volume=149&amp;publication_year=2013&amp;pages=1173-1179&amp;pmid=23925466&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Ni C., Chiu M.W. Psoriasis and comorbidities: links and risks. <span><span class="ref-journal">Clin. Cosmet. Investig. Dermatol. </span>2014;<span class="ref-vol">7</span>:119–132.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4000177/">PMC free article</a>]</span> [<a href="/pubmed/24790463" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cosmet.+Investig.+Dermatol.&amp;title=Psoriasis+and+comorbidities:+links+and+risks&amp;author=C.+Ni&amp;author=M.W.+Chiu&amp;volume=7&amp;publication_year=2014&amp;pages=119-132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Ananthakrishnan R., Eckes L., Walter H. On the genetics of psoriasis. An analysis of Hellgren’s data for a model of multifactorial inheritance. <span><span class="ref-journal">Arch. Dermatol. Forsch. </span>1973;<span class="ref-vol">247</span>:53–58.</span> [<a href="/pubmed/4747538" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.+Forsch.&amp;title=On+the+genetics+of+psoriasis.+An+analysis+of+Hellgren’s+data+for+a+model+of+multifactorial+inheritance&amp;author=R.+Ananthakrishnan&amp;author=L.+Eckes&amp;author=H.+Walter&amp;volume=247&amp;publication_year=1973&amp;pages=53-58&amp;pmid=4747538&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Ananthakrishnan R., Eckes L., Walter H. On the genetics of psoriasis. 2. Analysis of Lomholt’s data from Faroer Islands for a multifactorial mode of inheritance. <span><span class="ref-journal">J.&nbsp;Genet. </span>1974;<span class="ref-vol">61</span>:142–146.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Genet.&amp;title=On+the+genetics+of+psoriasis.+2.+Analysis+of+Lomholt’s+data+from+Faroer+Islands+for+a+multifactorial+mode+of+inheritance&amp;author=R.+Ananthakrishnan&amp;author=L.+Eckes&amp;author=H.+Walter&amp;volume=61&amp;publication_year=1974&amp;pages=142-146&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">Brandrup F., Holm N., Grunnet N., Henningsen K., Hansen H.E. Psoriasis in monozygotic twins: variations in expression in individuals with identical genetic constitution. <span><span class="ref-journal">Acta Derm. Venereol. </span>1982;<span class="ref-vol">62</span>:229–236.</span> [<a href="/pubmed/6179364" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Derm.+Venereol.&amp;title=Psoriasis+in+monozygotic+twins:+variations+in+expression+in+individuals+with+identical+genetic+constitution&amp;author=F.+Brandrup&amp;author=N.+Holm&amp;author=N.+Grunnet&amp;author=K.+Henningsen&amp;author=H.E.+Hansen&amp;volume=62&amp;publication_year=1982&amp;pages=229-236&amp;pmid=6179364&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Duffy D.L., Spelman L.S., Martin N.G. Psoriasis in Australian twins. <span><span class="ref-journal">J.&nbsp;Am. Acad. Dermatol. </span>1993;<span class="ref-vol">29</span>:428–434.</span> [<a href="/pubmed/8349859" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Am.+Acad.+Dermatol.&amp;title=Psoriasis+in+Australian+twins&amp;author=D.L.+Duffy&amp;author=L.S.+Spelman&amp;author=N.G.+Martin&amp;volume=29&amp;publication_year=1993&amp;pages=428-434&amp;pmid=8349859&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">Grjibovski A.M., Olsen A.O., Magnus P., Harris J.R. Psoriasis in Norwegian twins: contribution of genetic and environmental effects. <span><span class="ref-journal">J.&nbsp;Eur. Acad. Dermatol. Venereol. </span>2007;<span class="ref-vol">21</span>:1337–1343.</span> [<a href="/pubmed/17958839" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Eur.+Acad.+Dermatol.+Venereol.&amp;title=Psoriasis+in+Norwegian+twins:+contribution+of+genetic+and+environmental+effects&amp;author=A.M.+Grjibovski&amp;author=A.O.+Olsen&amp;author=P.+Magnus&amp;author=J.R.+Harris&amp;volume=21&amp;publication_year=2007&amp;pages=1337-1343&amp;pmid=17958839&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Lønnberg A.S., Skov L., Skytthe A., Kyvik K.O., Pedersen O.B., Thomsen S.F. Heritability of psoriasis in a large twin sample. <span><span class="ref-journal">Br. J. Dermatol. </span>2013;<span class="ref-vol">169</span>:412–416.</span> [<a href="/pubmed/23574549" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=Heritability+of+psoriasis+in+a+large+twin+sample&amp;author=A.S.+Lønnberg&amp;author=L.+Skov&amp;author=A.+Skytthe&amp;author=K.O.+Kyvik&amp;author=O.B.+Pedersen&amp;volume=169&amp;publication_year=2013&amp;pages=412-416&amp;pmid=23574549&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Moll J.M., Wright V. Familial occurrence of psoriatic arthritis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>1973;<span class="ref-vol">32</span>:181–201.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1006078/">PMC free article</a>]</span> [<a href="/pubmed/4715537" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Familial+occurrence+of+psoriatic+arthritis&amp;author=J.M.+Moll&amp;author=V.+Wright&amp;volume=32&amp;publication_year=1973&amp;pages=181-201&amp;pmid=4715537&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Elder J.T., Nair R.P., Guo S.W., Henseler T., Christophers E., Voorhees J.J. The genetics of psoriasis. <span><span class="ref-journal">Arch. Dermatol. </span>1994;<span class="ref-vol">130</span>:216–224.</span> [<a href="/pubmed/8304761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Dermatol.&amp;title=The+genetics+of+psoriasis&amp;author=J.T.+Elder&amp;author=R.P.+Nair&amp;author=S.W.+Guo&amp;author=T.+Henseler&amp;author=E.+Christophers&amp;volume=130&amp;publication_year=1994&amp;pages=216-224&amp;pmid=8304761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Myers A., Kay L.J., Lynch S.A., Walker D.J. Recurrence risk for psoriasis and psoriatic arthritis within sibships. <span><span class="ref-journal">Rheumatology (Oxford) </span>2005;<span class="ref-vol">44</span>:773–776.</span> [<a href="/pubmed/15757963" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology+(Oxford)&amp;title=Recurrence+risk+for+psoriasis+and+psoriatic+arthritis+within+sibships&amp;author=A.+Myers&amp;author=L.J.+Kay&amp;author=S.A.+Lynch&amp;author=D.J.+Walker&amp;volume=44&amp;publication_year=2005&amp;pages=773-776&amp;pmid=15757963&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Chandran V., Schentag C.T., Brockbank J.E., Pellett F.J., Shanmugarajah S., Toloza S.M., Rahman P., Gladman D.D. Familial aggregation of psoriatic arthritis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2009;<span class="ref-vol">68</span>:664–667.</span> [<a href="/pubmed/18524791" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Familial+aggregation+of+psoriatic+arthritis&amp;author=V.+Chandran&amp;author=C.T.+Schentag&amp;author=J.E.+Brockbank&amp;author=F.J.+Pellett&amp;author=S.+Shanmugarajah&amp;volume=68&amp;publication_year=2009&amp;pages=664-667&amp;pmid=18524791&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Karason A., Love T.J., Gudbjornsson B. A strong heritability of psoriatic arthritis over four generations--the Reykjavik Psoriatic Arthritis Study. <span><span class="ref-journal">Rheumatology (Oxford) </span>2009;<span class="ref-vol">48</span>:1424–1428.</span> [<a href="/pubmed/19741010" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology+(Oxford)&amp;title=A+strong+heritability+of+psoriatic+arthritis+over+four+generations--the+Reykjavik+Psoriatic+Arthritis+Study&amp;author=A.+Karason&amp;author=T.J.+Love&amp;author=B.+Gudbjornsson&amp;volume=48&amp;publication_year=2009&amp;pages=1424-1428&amp;pmid=19741010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Cargill M., Schrodi S.J., Chang M., Garcia V.E., Brandon R., Callis K.P., Matsunami N., Ardlie K.G., Civello D., Catanese J.J. A large-scale genetic association study confirms IL12B and leads to the identification of IL23R as psoriasis-risk genes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">80</span>:273–290.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1785338/">PMC free article</a>]</span> [<a href="/pubmed/17236132" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+large-scale+genetic+association+study+confirms+IL12B+and+leads+to+the+identification+of+IL23R+as+psoriasis-risk+genes&amp;author=M.+Cargill&amp;author=S.J.+Schrodi&amp;author=M.+Chang&amp;author=V.E.+Garcia&amp;author=R.+Brandon&amp;volume=80&amp;publication_year=2007&amp;pages=273-290&amp;pmid=17236132&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Liu Y., Helms C., Liao W., Zaba L.C., Duan S., Gardner J., Wise C., Miner A., Malloy M.J., Pullinger C.R. A genome-wide association study of psoriasis and psoriatic arthritis identifies new disease loci. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000041.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2274885/">PMC free article</a>]</span> [<a href="/pubmed/18369459" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=A+genome-wide+association+study+of+psoriasis+and+psoriatic+arthritis+identifies+new+disease+loci&amp;author=Y.+Liu&amp;author=C.+Helms&amp;author=W.+Liao&amp;author=L.C.+Zaba&amp;author=S.+Duan&amp;volume=4&amp;publication_year=2008&amp;pages=e1000041&amp;pmid=18369459&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Capon F., Bijlmakers M.J., Wolf N., Quaranta M., Huffmeier U., Allen M., Timms K., Abkevich V., Gutin A., Smith R. Identification of ZNF313/RNF114 as a novel psoriasis susceptibility gene. <span><span class="ref-journal">Hum. Mol. Genet. </span>2008;<span class="ref-vol">17</span>:1938–1945.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2900900/">PMC free article</a>]</span> [<a href="/pubmed/18364390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Identification+of+ZNF313/RNF114+as+a+novel+psoriasis+susceptibility+gene&amp;author=F.+Capon&amp;author=M.J.+Bijlmakers&amp;author=N.+Wolf&amp;author=M.+Quaranta&amp;author=U.+Huffmeier&amp;volume=17&amp;publication_year=2008&amp;pages=1938-1945&amp;pmid=18364390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Nair R.P., Duffin K.C., Helms C., Ding J., Stuart P.E., Goldgar D., Gudjonsson J.E., Li Y., Tejasvi T., Feng B.J., Collaborative Association Study of Psoriasis  Genome-wide scan reveals association of psoriasis with IL-23 and NF-kappaB pathways. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:199–204.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745122/">PMC free article</a>]</span> [<a href="/pubmed/19169254" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+scan+reveals+association+of+psoriasis+with+IL-23+and+NF-kappaB+pathways&amp;author=R.P.+Nair&amp;author=K.C.+Duffin&amp;author=C.+Helms&amp;author=J.+Ding&amp;author=P.E.+Stuart&amp;volume=41&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19169254&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Zhang X.J., Huang W., Yang S., Sun L.D., Zhang F.Y., Zhu Q.X., Zhang F.R., Zhang C., Du W.H., Pu X.M. Psoriasis genome-wide association study identifies susceptibility variants within LCE gene cluster at 1q21. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:205–210.</span> [<a href="/pubmed/19169255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Psoriasis+genome-wide+association+study+identifies+susceptibility+variants+within+LCE+gene+cluster+at+1q21&amp;author=X.J.+Zhang&amp;author=W.+Huang&amp;author=S.+Yang&amp;author=L.D.+Sun&amp;author=F.Y.+Zhang&amp;volume=41&amp;publication_year=2009&amp;pages=205-210&amp;pmid=19169255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Strange A., Capon F., Spencer C.C., Knight J., Weale M.E., Allen M.H., Barton A., Band G., Bellenguez C., Bergboer J.G., Genetic Analysis of Psoriasis Consortium &amp; the&nbsp;Wellcome Trust Case Control Consortium 2  A&nbsp;genome-wide association study identifies new psoriasis susceptibility loci and an interaction between HLA-C and ERAP1. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:985–990.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3749730/">PMC free article</a>]</span> [<a href="/pubmed/20953190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A&nbsp;genome-wide+association+study+identifies+new+psoriasis+susceptibility+loci+and+an+interaction+between+HLA-C+and+ERAP1&amp;author=A.+Strange&amp;author=F.+Capon&amp;author=C.C.+Spencer&amp;author=J.+Knight&amp;author=M.E.+Weale&amp;volume=42&amp;publication_year=2010&amp;pages=985-990&amp;pmid=20953190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Stuart P.E., Nair R.P., Ellinghaus E., Ding J., Tejasvi T., Gudjonsson J.E., Li Y., Weidinger S., Eberlein B., Gieger C. Genome-wide association analysis identifies three psoriasis susceptibility loci. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1000–1004.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2965799/">PMC free article</a>]</span> [<a href="/pubmed/20953189" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+analysis+identifies+three+psoriasis+susceptibility+loci&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=E.+Ellinghaus&amp;author=J.+Ding&amp;author=T.+Tejasvi&amp;volume=42&amp;publication_year=2010&amp;pages=1000-1004&amp;pmid=20953189&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Ellinghaus E., Ellinghaus D., Stuart P.E., Nair R.P., Debrus S., Raelson J.V., Belouchi M., Fournier H., Reinhard C., Ding J. Genome-wide association study identifies a psoriasis susceptibility locus at TRAF3IP2. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:991–995.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3136364/">PMC free article</a>]</span> [<a href="/pubmed/20953188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+a+psoriasis+susceptibility+locus+at+TRAF3IP2&amp;author=E.+Ellinghaus&amp;author=D.+Ellinghaus&amp;author=P.E.+Stuart&amp;author=R.P.+Nair&amp;author=S.+Debrus&amp;volume=42&amp;publication_year=2010&amp;pages=991-995&amp;pmid=20953188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Sun L.D., Cheng H., Wang Z.X., Zhang A.P., Wang P.G., Xu J.H., Zhu Q.X., Zhou H.S., Ellinghaus E., Zhang F.R. Association analyses identify six new psoriasis susceptibility loci in the Chinese population. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:1005–1009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3140436/">PMC free article</a>]</span> [<a href="/pubmed/20953187" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Association+analyses+identify+six+new+psoriasis+susceptibility+loci+in+the+Chinese+population&amp;author=L.D.+Sun&amp;author=H.+Cheng&amp;author=Z.X.+Wang&amp;author=A.P.+Zhang&amp;author=P.G.+Wang&amp;volume=42&amp;publication_year=2010&amp;pages=1005-1009&amp;pmid=20953187&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref27">Tsoi L.C., Spain S.L., Knight J., Ellinghaus E., Stuart P.E., Capon F., Ding J., Li Y., Tejasvi T., Gudjonsson J.E., Collaborative Association Study of Psoriasis (CASP) Genetic Analysis of Psoriasis Consortium. Psoriasis Association Genetics Extension. Wellcome Trust Case Control Consortium 2  Identification of 15 new psoriasis susceptibility loci highlights the role of innate immunity. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1341–1348.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3510312/">PMC free article</a>]</span> [<a href="/pubmed/23143594" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+15+new+psoriasis+susceptibility+loci+highlights+the+role+of+innate+immunity&amp;author=L.C.+Tsoi&amp;author=S.L.+Spain&amp;author=J.+Knight&amp;author=E.+Ellinghaus&amp;author=P.E.+Stuart&amp;volume=44&amp;publication_year=2012&amp;pages=1341-1348&amp;pmid=23143594&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Ellinghaus D., Ellinghaus E., Nair R.P., Stuart P.E., Esko T., Metspalu A., Debrus S., Raelson J.V., Tejasvi T., Belouchi M. Combined analysis of genome-wide association studies for Crohn disease and psoriasis identifies seven shared susceptibility loci. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">90</span>:636–647.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3322238/">PMC free article</a>]</span> [<a href="/pubmed/22482804" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Combined+analysis+of+genome-wide+association+studies+for+Crohn+disease+and+psoriasis+identifies+seven+shared+susceptibility+loci&amp;author=D.+Ellinghaus&amp;author=E.+Ellinghaus&amp;author=R.P.+Nair&amp;author=P.E.+Stuart&amp;author=T.+Esko&amp;volume=90&amp;publication_year=2012&amp;pages=636-647&amp;pmid=22482804&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Sheng Y., Jin X., Xu J., Gao J., Du X., Duan D., Li B., Zhao J., Zhan W., Tang H. Sequencing-based approach identified three new susceptibility loci for psoriasis. <span><span class="ref-journal">Nat. Commun. </span>2014;<span class="ref-vol">5</span>:4331.</span> [<a href="/pubmed/25006012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Sequencing-based+approach+identified+three+new+susceptibility+loci+for+psoriasis&amp;author=Y.+Sheng&amp;author=X.+Jin&amp;author=J.+Xu&amp;author=J.+Gao&amp;author=X.+Du&amp;volume=5&amp;publication_year=2014&amp;pages=4331&amp;pmid=25006012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Yin X., Low H.Q., Wang L., Li Y., Ellinghaus E., Han J., Estivill X., Sun L., Zuo X., Shen C. Genome-wide meta-analysis identifies multiple novel associations and ethnic heterogeneity of psoriasis susceptibility. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:6916.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4423213/">PMC free article</a>]</span> [<a href="/pubmed/25903422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Genome-wide+meta-analysis+identifies+multiple+novel+associations+and+ethnic+heterogeneity+of+psoriasis+susceptibility&amp;author=X.+Yin&amp;author=H.Q.+Low&amp;author=L.+Wang&amp;author=Y.+Li&amp;author=E.+Ellinghaus&amp;volume=6&amp;publication_year=2015&amp;pages=6916&amp;pmid=25903422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Zuo X., Sun L., Yin X., Gao J., Sheng Y., Xu J., Zhang J., He C., Qiu Y., Wen G. Whole-exome SNP array identifies 15 new susceptibility loci for psoriasis. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:6793.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4403312/">PMC free article</a>]</span> [<a href="/pubmed/25854761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Whole-exome+SNP+array+identifies+15+new+susceptibility+loci+for+psoriasis&amp;author=X.+Zuo&amp;author=L.+Sun&amp;author=X.+Yin&amp;author=J.+Gao&amp;author=Y.+Sheng&amp;volume=6&amp;publication_year=2015&amp;pages=6793&amp;pmid=25854761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Tsoi L.C., Spain S.L., Ellinghaus E., Stuart P.E., Capon F., Knight J., Tejasvi T., Kang H.M., Allen M.H., Lambert S. Enhanced meta-analysis and replication studies identify five new psoriasis susceptibility loci. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:7001.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4422106/">PMC free article</a>]</span> [<a href="/pubmed/25939698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Enhanced+meta-analysis+and+replication+studies+identify+five+new+psoriasis+susceptibility+loci&amp;author=L.C.+Tsoi&amp;author=S.L.+Spain&amp;author=E.+Ellinghaus&amp;author=P.E.+Stuart&amp;author=F.+Capon&amp;volume=6&amp;publication_year=2015&amp;pages=7001&amp;pmid=25939698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Hüffmeier U., Uebe S., Ekici A.B., Bowes J., Giardina E., Korendowych E., Juneblad K., Apel M., McManus R., Ho P. Common variants at TRAF3IP2 are associated with susceptibility to psoriatic arthritis and psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:996–999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2981079/">PMC free article</a>]</span> [<a href="/pubmed/20953186" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+at+TRAF3IP2+are+associated+with+susceptibility+to+psoriatic+arthritis+and+psoriasis&amp;author=U.+Hüffmeier&amp;author=S.+Uebe&amp;author=A.B.+Ekici&amp;author=J.+Bowes&amp;author=E.+Giardina&amp;volume=42&amp;publication_year=2010&amp;pages=996-999&amp;pmid=20953186&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Ellinghaus E., Stuart P.E., Ellinghaus D., Nair R.P., Debrus S., Raelson J.V., Belouchi M., Tejasvi T., Li Y., Tsoi L.C. Genome-wide meta-analysis of psoriatic arthritis identifies susceptibility locus at REL. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2012;<span class="ref-vol">132</span>:1133–1140.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3305829/">PMC free article</a>]</span> [<a href="/pubmed/22170493" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Genome-wide+meta-analysis+of+psoriatic+arthritis+identifies+susceptibility+locus+at+REL&amp;author=E.+Ellinghaus&amp;author=P.E.+Stuart&amp;author=D.+Ellinghaus&amp;author=R.P.+Nair&amp;author=S.+Debrus&amp;volume=132&amp;publication_year=2012&amp;pages=1133-1140&amp;pmid=22170493&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Gonzalez S., Martinez-Borra J., Torre-Alonso J.C., Gonzalez-Roces S., Sanchez del Río J., Rodriguez Pérez A., Brautbar C., López-Larrea C. The MICA-A9 triplet repeat polymorphism in the transmembrane region confers additional susceptibility to the development of psoriatic arthritis and is independent of the association of Cw<sup>∗</sup>0602 in psoriasis. <span><span class="ref-journal">Arthritis Rheum. </span>1999;<span class="ref-vol">42</span>:1010–1016.</span> [<a href="/pubmed/10323458" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=The+MICA-A9+triplet+repeat+polymorphism+in+the+transmembrane+region+confers+additional+susceptibility+to+the+development+of+psoriatic+arthritis+and+is+independent+of+the+association+of+Cw∗0602+in+psoriasis&amp;author=S.+Gonzalez&amp;author=J.+Martinez-Borra&amp;author=J.C.+Torre-Alonso&amp;author=S.+Gonzalez-Roces&amp;author=J.+Sanchez+del+Río&amp;volume=42&amp;publication_year=1999&amp;pages=1010-1016&amp;pmid=10323458&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Queiro R., Gonzalez S., López-Larrea C., Alperi M., Sarasqueta C., Riestra J.L., Ballina J. HLA-C locus alleles may modulate the clinical expression of psoriatic arthritis. <span><span class="ref-journal">Arthritis Res. Ther. </span>2006;<span class="ref-vol">8</span>:R185.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1794531/">PMC free article</a>]</span> [<a href="/pubmed/17166285" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Res.+Ther.&amp;title=HLA-C+locus+alleles+may+modulate+the+clinical+expression+of+psoriatic+arthritis&amp;author=R.+Queiro&amp;author=S.+Gonzalez&amp;author=C.+López-Larrea&amp;author=M.+Alperi&amp;author=C.+Sarasqueta&amp;volume=8&amp;publication_year=2006&amp;pages=R185&amp;pmid=17166285&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Ho P.Y., Barton A., Worthington J., Plant D., Griffiths C.E., Young H.S., Bradburn P., Thomson W., Silman A.J., Bruce I.N. Investigating the role of the HLA-Cw<sup>∗</sup>06 and HLA-DRB1 genes in susceptibility to psoriatic arthritis: comparison with psoriasis and undifferentiated inflammatory arthritis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2008;<span class="ref-vol">67</span>:677–682.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2563264/">PMC free article</a>]</span> [<a href="/pubmed/17728335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Investigating+the+role+of+the+HLA-Cw∗06+and+HLA-DRB1+genes+in+susceptibility+to+psoriatic+arthritis:+comparison+with+psoriasis+and+undifferentiated+inflammatory+arthritis&amp;author=P.Y.+Ho&amp;author=A.+Barton&amp;author=J.+Worthington&amp;author=D.+Plant&amp;author=C.E.+Griffiths&amp;volume=67&amp;publication_year=2008&amp;pages=677-682&amp;pmid=17728335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Rahman P., Roslin N.M., Pellett F.J., Lemire M., Greenwood C.M., Beyene J., Pope A., Peddle L., Paterson A.D., Uddin M., Gladman D.D. High resolution mapping in the major histocompatibility complex region identifies multiple independent novel loci for psoriatic arthritis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2011;<span class="ref-vol">70</span>:690–694.</span> [<a href="/pubmed/21242233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=High+resolution+mapping+in+the+major+histocompatibility+complex+region+identifies+multiple+independent+novel+loci+for+psoriatic+arthritis&amp;author=P.+Rahman&amp;author=N.M.+Roslin&amp;author=F.J.+Pellett&amp;author=M.+Lemire&amp;author=C.M.+Greenwood&amp;volume=70&amp;publication_year=2011&amp;pages=690-694&amp;pmid=21242233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Giardina E., Hüffmeier U., Ravindran J., Behrens F., Lepre T., McHugh N.J., Korendowych E., Burkhardt H., Novelli G., Reis A. Tumor necrosis factor promoter polymorphism TNF<sup>∗</sup>-857 is a risk allele for psoriatic arthritis independent of the PSORS1 locus. <span><span class="ref-journal">Arthritis Rheum. </span>2011;<span class="ref-vol">63</span>:3801–3806.</span> [<a href="/pubmed/22127698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Tumor+necrosis+factor+promoter+polymorphism+TNF∗-857+is+a+risk+allele+for+psoriatic+arthritis+independent+of+the+PSORS1+locus&amp;author=E.+Giardina&amp;author=U.+Hüffmeier&amp;author=J.+Ravindran&amp;author=F.+Behrens&amp;author=T.+Lepre&amp;volume=63&amp;publication_year=2011&amp;pages=3801-3806&amp;pmid=22127698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Julià A., Tortosa R., Hernanz J.M., Cañete J.D., Fonseca E., Ferrándiz C., Unamuno P., Puig L., Fernández-Sueiro J.L., Sanmartí R. Risk variants for psoriasis vulgaris in a large case-control collection and association with clinical subphenotypes. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:4549–4557.</span> [<a href="/pubmed/22814393" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Risk+variants+for+psoriasis+vulgaris+in+a+large+case-control+collection+and+association+with+clinical+subphenotypes&amp;author=A.+Julià&amp;author=R.+Tortosa&amp;author=J.M.+Hernanz&amp;author=J.D.+Cañete&amp;author=E.+Fonseca&amp;volume=21&amp;publication_year=2012&amp;pages=4549-4557&amp;pmid=22814393&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Eder L., Chandran V., Pellet F., Shanmugarajah S., Rosen C.F., Bull S.B., Gladman D.D. Human leucocyte antigen risk alleles for psoriatic arthritis among patients with psoriasis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2012;<span class="ref-vol">71</span>:50–55.</span> [<a href="/pubmed/21900282" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Human+leucocyte+antigen+risk+alleles+for+psoriatic+arthritis+among+patients+with+psoriasis&amp;author=L.+Eder&amp;author=V.+Chandran&amp;author=F.+Pellet&amp;author=S.+Shanmugarajah&amp;author=C.F.+Rosen&amp;volume=71&amp;publication_year=2012&amp;pages=50-55&amp;pmid=21900282&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Winchester R., Minevich G., Steshenko V., Kirby B., Kane D., Greenberg D.A., FitzGerald O. HLA associations reveal genetic heterogeneity in psoriatic arthritis and in the psoriasis phenotype. <span><span class="ref-journal">Arthritis Rheum. </span>2012;<span class="ref-vol">64</span>:1134–1144.</span> [<a href="/pubmed/22006066" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=HLA+associations+reveal+genetic+heterogeneity+in+psoriatic+arthritis+and+in+the+psoriasis+phenotype&amp;author=R.+Winchester&amp;author=G.+Minevich&amp;author=V.+Steshenko&amp;author=B.+Kirby&amp;author=D.+Kane&amp;volume=64&amp;publication_year=2012&amp;pages=1134-1144&amp;pmid=22006066&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Chandran V., Bull S.B., Pellett F.J., Ayearst R., Rahman P., Gladman D.D. Human leukocyte antigen alleles and susceptibility to psoriatic arthritis. <span><span class="ref-journal">Hum. Immunol. </span>2013;<span class="ref-vol">74</span>:1333–1338.</span> [<a href="/pubmed/23916976" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Immunol.&amp;title=Human+leukocyte+antigen+alleles+and+susceptibility+to+psoriatic+arthritis&amp;author=V.+Chandran&amp;author=S.B.+Bull&amp;author=F.J.+Pellett&amp;author=R.+Ayearst&amp;author=P.+Rahman&amp;volume=74&amp;publication_year=2013&amp;pages=1333-1338&amp;pmid=23916976&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Okada Y., Han B., Tsoi L.C., Stuart P.E., Ellinghaus E., Tejasvi T., Chandran V., Pellett F., Pollock R., Bowcock A.M. Fine mapping major histocompatibility complex associations in psoriasis and its clinical subtypes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2014;<span class="ref-vol">95</span>:162–172.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4129407/">PMC free article</a>]</span> [<a href="/pubmed/25087609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Fine+mapping+major+histocompatibility+complex+associations+in+psoriasis+and+its+clinical+subtypes&amp;author=Y.+Okada&amp;author=B.+Han&amp;author=L.C.+Tsoi&amp;author=P.E.+Stuart&amp;author=E.+Ellinghaus&amp;volume=95&amp;publication_year=2014&amp;pages=162-172&amp;pmid=25087609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Bowes J., Budu-Aggrey A., Huffmeier U., Uebe S., Steel K., Hebert H.L., Wallace C., Massey J., Bruce I.N., Bluett J. Dense genotyping of immune-related susceptibility loci reveals new insights into the genetics of psoriatic arthritis. <span><span class="ref-journal">Nat. Commun. </span>2015;<span class="ref-vol">6</span>:6046.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4327416/">PMC free article</a>]</span> [<a href="/pubmed/25651891" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Dense+genotyping+of+immune-related+susceptibility+loci+reveals+new+insights+into+the+genetics+of+psoriatic+arthritis&amp;author=J.+Bowes&amp;author=A.+Budu-Aggrey&amp;author=U.+Huffmeier&amp;author=S.+Uebe&amp;author=K.+Steel&amp;volume=6&amp;publication_year=2015&amp;pages=6046&amp;pmid=25651891&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Yang Q., Liu H., Qu L., Fu X., Yu Y., Yu G., Tian H., Yu Y., Sun D., Peng J. Investigation of 20 non-HLA (human leucocyte antigen) psoriasis susceptibility loci in Chinese patients with psoriatic arthritis and psoriasis vulgaris. <span><span class="ref-journal">Br. J. Dermatol. </span>2013;<span class="ref-vol">168</span>:1060–1065.</span> [<a href="/pubmed/23252691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=Investigation+of+20+non-HLA+(human+leucocyte+antigen)+psoriasis+susceptibility+loci+in+Chinese+patients+with+psoriatic+arthritis+and+psoriasis+vulgaris&amp;author=Q.+Yang&amp;author=H.+Liu&amp;author=L.+Qu&amp;author=X.+Fu&amp;author=Y.+Yu&amp;volume=168&amp;publication_year=2013&amp;pages=1060-1065&amp;pmid=23252691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Apel M., Uebe S., Bowes J., Giardina E., Korendowych E., Juneblad K., Pasutto F., Ekici A.B., McManus R., Ho P. Variants in RUNX3 contribute to susceptibility to psoriatic arthritis, exhibiting further common ground with ankylosing spondylitis. <span><span class="ref-journal">Arthritis Rheum. </span>2013;<span class="ref-vol">65</span>:1224–1231.</span> [<a href="/pubmed/23401011" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Variants+in+RUNX3+contribute+to+susceptibility+to+psoriatic+arthritis,+exhibiting+further+common+ground+with+ankylosing+spondylitis&amp;author=M.+Apel&amp;author=S.+Uebe&amp;author=J.+Bowes&amp;author=E.+Giardina&amp;author=E.+Korendowych&amp;volume=65&amp;publication_year=2013&amp;pages=1224-1231&amp;pmid=23401011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Bowes J., Loehr S., Budu-Aggrey A., Uebe S., Bruce I.N., Feletar M., Marzo-Ortega H., Helliwell P., Ryan A.W., Kane D. PTPN22 is associated with susceptibility to psoriatic arthritis but not psoriasis: evidence for a further PsA-specific risk locus. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2015;<span class="ref-vol">74</span>:1882–1885.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4602265/">PMC free article</a>]</span> [<a href="/pubmed/25923216" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=PTPN22+is+associated+with+susceptibility+to+psoriatic+arthritis+but+not+psoriasis:+evidence+for+a+further+PsA-specific+risk+locus&amp;author=J.+Bowes&amp;author=S.+Loehr&amp;author=A.+Budu-Aggrey&amp;author=S.+Uebe&amp;author=I.N.+Bruce&amp;volume=74&amp;publication_year=2015&amp;pages=1882-1885&amp;pmid=25923216&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Woodrow J.C., Ilchysyn A. HLA antigens in psoriasis and psoriatic arthritis. <span><span class="ref-journal">J.&nbsp;Med. Genet. </span>1985;<span class="ref-vol">22</span>:492–495.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1049511/">PMC free article</a>]</span> [<a href="/pubmed/3866077" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Med.+Genet.&amp;title=HLA+antigens+in+psoriasis+and+psoriatic+arthritis&amp;author=J.C.+Woodrow&amp;author=A.+Ilchysyn&amp;volume=22&amp;publication_year=1985&amp;pages=492-495&amp;pmid=3866077&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Gladman D.D., Anhorn K.A., Schachter R.K., Mervart H. HLA antigens in psoriatic arthritis. <span><span class="ref-journal">J.&nbsp;Rheumatol. </span>1986;<span class="ref-vol">13</span>:586–592.</span> [<a href="/pubmed/3735281" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Rheumatol.&amp;title=HLA+antigens+in+psoriatic+arthritis&amp;author=D.D.+Gladman&amp;author=K.A.+Anhorn&amp;author=R.K.+Schachter&amp;author=H.+Mervart&amp;volume=13&amp;publication_year=1986&amp;pages=586-592&amp;pmid=3735281&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Höhler T., Grossmann S., Stradmann-Bellinghausen B., Kaluza W., Reuss E., de Vlam K., Veys E., Märker-Hermann E. Differential association of polymorphisms in the TNFalpha region with psoriatic arthritis but not psoriasis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2002;<span class="ref-vol">61</span>:213–218.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1754040/">PMC free article</a>]</span> [<a href="/pubmed/11830425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Differential+association+of+polymorphisms+in+the+TNFalpha+region+with+psoriatic+arthritis+but+not+psoriasis&amp;author=T.+Höhler&amp;author=S.+Grossmann&amp;author=B.+Stradmann-Bellinghausen&amp;author=W.+Kaluza&amp;author=E.+Reuss&amp;volume=61&amp;publication_year=2002&amp;pages=213-218&amp;pmid=11830425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Lascorz J., Burkhardt H., Hüffmeier U., Böhm B., Schürmeyer-Horst F., Lohmann J., Ständer M., Wendler J., Kelsch R., Baumann C. Lack of genetic association of the three more common polymorphisms of CARD15 with psoriatic arthritis and psoriasis in a German cohort. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2005;<span class="ref-vol">64</span>:951–954.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1755526/">PMC free article</a>]</span> [<a href="/pubmed/15539411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Lack+of+genetic+association+of+the+three+more+common+polymorphisms+of+CARD15+with+psoriatic+arthritis+and+psoriasis+in+a+German+cohort&amp;author=J.+Lascorz&amp;author=H.+Burkhardt&amp;author=U.+Hüffmeier&amp;author=B.+Böhm&amp;author=F.+Schürmeyer-Horst&amp;volume=64&amp;publication_year=2005&amp;pages=951-954&amp;pmid=15539411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Reich K., Hüffmeier U., König I.R., Lascorz J., Lohmann J., Wendler J., Traupe H., Mössner R., Reis A., Burkhardt H. TNF polymorphisms in psoriasis: association of psoriatic arthritis with the promoter polymorphism TNF<sup>∗</sup>-857 independent of the PSORS1 risk allele. <span><span class="ref-journal">Arthritis Rheum. </span>2007;<span class="ref-vol">56</span>:2056–2064.</span> [<a href="/pubmed/17530646" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=TNF+polymorphisms+in+psoriasis:+association+of+psoriatic+arthritis+with+the+promoter+polymorphism+TNF∗-857+independent+of+the+PSORS1+risk+allele&amp;author=K.+Reich&amp;author=U.+Hüffmeier&amp;author=I.R.+König&amp;author=J.+Lascorz&amp;author=J.+Lohmann&amp;volume=56&amp;publication_year=2007&amp;pages=2056-2064&amp;pmid=17530646&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Chang Y.T., Chou C.T., Yu C.W., Lin M.W., Shiao Y.M., Chen C.C., Huang C.H., Lee D.D., Liu H.N., Wang W.J., Tsai S.F. Cytokine gene polymorphisms in Chinese patients with psoriasis. <span><span class="ref-journal">Br. J. Dermatol. </span>2007;<span class="ref-vol">156</span>:899–905.</span> [<a href="/pubmed/17388919" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Dermatol.&amp;title=Cytokine+gene+polymorphisms+in+Chinese+patients+with+psoriasis&amp;author=Y.T.+Chang&amp;author=C.T.+Chou&amp;author=C.W.+Yu&amp;author=M.W.+Lin&amp;author=Y.M.+Shiao&amp;volume=156&amp;publication_year=2007&amp;pages=899-905&amp;pmid=17388919&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref55">Duffin K.C., Freeny I.C., Schrodi S.J., Wong B., Feng B.J., Soltani-Arabshahi R., Rakkhit T., Goldgar D.E., Krueger G.G. Association between IL13 polymorphisms and psoriatic arthritis is modified by smoking. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2009;<span class="ref-vol">129</span>:2777–2783.</span> [<a href="/pubmed/19554022" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Association+between+IL13+polymorphisms+and+psoriatic+arthritis+is+modified+by+smoking&amp;author=K.C.+Duffin&amp;author=I.C.+Freeny&amp;author=S.J.+Schrodi&amp;author=B.+Wong&amp;author=B.J.+Feng&amp;volume=129&amp;publication_year=2009&amp;pages=2777-2783&amp;pmid=19554022&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation" id="sref56">Eder L., Chandran V., Pellett F., Pollock R., Shanmugarajah S., Rosen C.F., Rahman P., Gladman D.D. IL13 gene polymorphism is a marker for psoriatic arthritis among psoriasis patients. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2011;<span class="ref-vol">70</span>:1594–1598.</span> [<a href="/pubmed/21613309" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=IL13+gene+polymorphism+is+a+marker+for+psoriatic+arthritis+among+psoriasis+patients&amp;author=L.+Eder&amp;author=V.+Chandran&amp;author=F.+Pellett&amp;author=R.+Pollock&amp;author=S.+Shanmugarajah&amp;volume=70&amp;publication_year=2011&amp;pages=1594-1598&amp;pmid=21613309&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation" id="sref57">Bowes J., Eyre S., Flynn E., Ho P., Salah S., Warren R.B., Marzo-Ortega H., Coates L., McManus R., Ryan A.W. Evidence to support IL-13 as a risk locus for psoriatic arthritis but not psoriasis vulgaris. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2011;<span class="ref-vol">70</span>:1016–1019.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3086035/">PMC free article</a>]</span> [<a href="/pubmed/21349879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Evidence+to+support+IL-13+as+a+risk+locus+for+psoriatic+arthritis+but+not+psoriasis+vulgaris&amp;author=J.+Bowes&amp;author=S.+Eyre&amp;author=E.+Flynn&amp;author=P.+Ho&amp;author=S.+Salah&amp;volume=70&amp;publication_year=2011&amp;pages=1016-1019&amp;pmid=21349879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation" id="sref58">Soto-Sánchez J., Santos-Juanes J., Coto-Segura P., Coto E., Díaz M., Rodríguez I., Alvarez V. Genetic variation at the CCR5/CCR2 gene cluster and risk of psoriasis and psoriatic arthritis. <span><span class="ref-journal">Cytokine. </span>2010;<span class="ref-vol">50</span>:114–116.</span> [<a href="/pubmed/20153665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine&amp;title=Genetic+variation+at+the+CCR5/CCR2+gene+cluster+and+risk+of+psoriasis+and+psoriatic+arthritis&amp;author=J.+Soto-Sánchez&amp;author=J.+Santos-Juanes&amp;author=P.+Coto-Segura&amp;author=E.+Coto&amp;author=M.+Díaz&amp;volume=50&amp;publication_year=2010&amp;pages=114-116&amp;pmid=20153665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">59. <span class="element-citation" id="sref59">Chandran V., Bull S.B., Pellett F.J., Ayearst R., Pollock R.A., Gladman D.D. Killer-cell immunoglobulin-like receptor gene polymorphisms and susceptibility to psoriatic arthritis. <span><span class="ref-journal">Rheumatology (Oxford) </span>2014;<span class="ref-vol">53</span>:233–239.</span> [<a href="/pubmed/24185760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology+(Oxford)&amp;title=Killer-cell+immunoglobulin-like+receptor+gene+polymorphisms+and+susceptibility+to+psoriatic+arthritis&amp;author=V.+Chandran&amp;author=S.B.+Bull&amp;author=F.J.+Pellett&amp;author=R.+Ayearst&amp;author=R.A.+Pollock&amp;volume=53&amp;publication_year=2014&amp;pages=233-239&amp;pmid=24185760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib60">60. <span class="element-citation" id="sref60">Das S., Stuart P.E., Ding J., Tejasvi T., Li Y., Tsoi L.C., Chandran V., Fischer J., Helms C., Duffin K.C. Fine mapping of eight psoriasis susceptibility loci. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2014;<span class="ref-vol">23</span>:844–853.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4795047/">PMC free article</a>]</span> [<a href="/pubmed/25182136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Fine+mapping+of+eight+psoriasis+susceptibility+loci&amp;author=S.+Das&amp;author=P.E.+Stuart&amp;author=J.+Ding&amp;author=T.+Tejasvi&amp;author=Y.+Li&amp;volume=23&amp;publication_year=2014&amp;pages=844-853&amp;pmid=25182136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib61">61. <span class="element-citation" id="sref61">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., Sham P.C. PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib62">62. <span class="element-citation" id="sref62">Abecasis G.R.  Department of Biostatistics, University of Michigan; Ann Arbor, Michigan: 2010. RelativeFinder Computer Program. <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Abecasis+G.R.+RelativeFinder+Computer+Program+2010+Department+of+Biostatistics,+University+of+Michigan+Ann+Arbor,+Michigan+" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib63">63. <span class="element-citation" id="sref63">Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib64">64. <span class="element-citation" id="sref64">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib65">65. <span class="element-citation" id="sref65">Abecasis G.R., Auton A., Brooks L.D., DePristo M.A., Durbin R.M., Handsaker R.E., Kang H.M., Marth G.T., McVean G.A., 1000 Genomes Project Consortium  An integrated map of genetic variation from 1,092 human genomes. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">491</span>:56–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3498066/">PMC free article</a>]</span> [<a href="/pubmed/23128226" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=An+integrated+map+of+genetic+variation+from+1,092+human+genomes&amp;author=G.R.+Abecasis&amp;author=A.+Auton&amp;author=L.D.+Brooks&amp;author=M.A.+DePristo&amp;author=R.M.+Durbin&amp;volume=491&amp;publication_year=2012&amp;pages=56-65&amp;pmid=23128226&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib66">66. <span class="element-citation" id="sref66">Purcell S., Cherny S.S., Sham P.C. Genetic Power Calculator: design of linkage and association genetic mapping studies of complex traits. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>:149–150.</span> [<a href="/pubmed/12499305" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genetic+Power+Calculator:+design+of+linkage+and+association+genetic+mapping+studies+of+complex+traits&amp;author=S.+Purcell&amp;author=S.S.+Cherny&amp;author=P.C.+Sham&amp;volume=19&amp;publication_year=2003&amp;pages=149-150&amp;pmid=12499305&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib67">67. <span class="element-citation" id="sref67">Ghosh A., Zou F., Wright F.A. Estimating odds ratios in genome scans: an approximate conditional likelihood approach. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2008;<span class="ref-vol">82</span>:1064–1074.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665019/">PMC free article</a>]</span> [<a href="/pubmed/18423522" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Estimating+odds+ratios+in+genome+scans:+an+approximate+conditional+likelihood+approach&amp;author=A.+Ghosh&amp;author=F.+Zou&amp;author=F.A.+Wright&amp;volume=82&amp;publication_year=2008&amp;pages=1064-1074&amp;pmid=18423522&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib68">68. <span class="element-citation" id="sref68">Gabriel S.B., Schaffner S.F., Nguyen H., Moore J.M., Roy J., Blumenstiel B., Higgins J., DeFelice M., Lochner A., Faggart M. The structure of haplotype blocks in the human genome. <span><span class="ref-journal">Science. </span>2002;<span class="ref-vol">296</span>:2225–2229.</span> [<a href="/pubmed/12029063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The+structure+of+haplotype+blocks+in+the+human+genome&amp;author=S.B.+Gabriel&amp;author=S.F.+Schaffner&amp;author=H.+Nguyen&amp;author=J.M.+Moore&amp;author=J.+Roy&amp;volume=296&amp;publication_year=2002&amp;pages=2225-2229&amp;pmid=12029063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib69">69. <span class="element-citation" id="sref69">So H.C., Gui A.H., Cherny S.S., Sham P.C. Evaluating the heritability explained by known susceptibility variants: a survey of ten complex diseases. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:310–317.</span> [<a href="/pubmed/21374718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Evaluating+the+heritability+explained+by+known+susceptibility+variants:+a+survey+of+ten+complex+diseases&amp;author=H.C.+So&amp;author=A.H.+Gui&amp;author=S.S.+Cherny&amp;author=P.C.+Sham&amp;volume=35&amp;publication_year=2011&amp;pages=310-317&amp;pmid=21374718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib70">70. <span class="element-citation" id="sref70">Pratt J.W.  Dividing the indivisible: Using simple symmetry to partition variance explained. In: Pukkila T., Puntanen S., editors. <span class="ref-journal">Proceedings of the Second International Tampere Conference in Statistics.</span> University of Tampere; Tampere, Finland: 1987. pp. 245–260. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Proceedings+of+the+Second+International+Tampere+Conference+in+Statistics&amp;author=J.W.+Pratt&amp;publication_year=1987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib71">71. <span class="element-citation" id="sref71">Efron B., Tibshriani R. Bootstrap methods for standard errors, confidence intervals, and other measures of statistical accuracy. <span><span class="ref-journal">Stat. Sci. </span>1986;<span class="ref-vol">1</span>:54–77.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Sci.&amp;title=Bootstrap+methods+for+standard+errors,+confidence+intervals,+and+other+measures+of+statistical+accuracy&amp;author=B.+Efron&amp;author=R.+Tibshriani&amp;volume=1&amp;publication_year=1986&amp;pages=54-77&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib72">72. <span class="element-citation" id="sref72">Maller J.B., McVean G., Byrnes J., Vukcevic D., Palin K., Su Z., Howson J.M., Auton A., Myers S., Morris A., Wellcome Trust Case Control Consortium  Bayesian refinement of association signals for 14 loci in 3 common diseases. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:1294–1301.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3791416/">PMC free article</a>]</span> [<a href="/pubmed/23104008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Bayesian+refinement+of+association+signals+for+14+loci+in+3+common+diseases&amp;author=J.B.+Maller&amp;author=G.+McVean&amp;author=J.+Byrnes&amp;author=D.+Vukcevic&amp;author=K.+Palin&amp;volume=44&amp;publication_year=2012&amp;pages=1294-1301&amp;pmid=23104008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib73">73. <span class="element-citation" id="sref73">Wen X., Stephens M. Bayesian methods for genetic association analysis with heterogeneous subgroups: from meta-analyses to gene-environment interactions. <span><span class="ref-journal">Ann. Appl. Stat. </span>2014;<span class="ref-vol">8</span>:176–203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4583155/">PMC free article</a>]</span> [<a href="/pubmed/26413181" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Appl.+Stat.&amp;title=Bayesian+methods+for+genetic+association+analysis+with+heterogeneous+subgroups:+from+meta-analyses+to+gene-environment+interactions&amp;author=X.+Wen&amp;author=M.+Stephens&amp;volume=8&amp;publication_year=2014&amp;pages=176-203&amp;pmid=26413181&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib74">74. <span class="element-citation" id="sref74">Li B., Tsoi L.C., Swindell W.R., Gudjonsson J.E., Tejasvi T., Johnston A., Ding J., Stuart P.E., Xing X., Kochkodan J.J. Transcriptome analysis of psoriasis in a large case-control sample: RNA-seq provides insights into disease mechanisms. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2014;<span class="ref-vol">134</span>:1828–1838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4057954/">PMC free article</a>]</span> [<a href="/pubmed/24441097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Transcriptome+analysis+of+psoriasis+in+a+large+case-control+sample:+RNA-seq+provides+insights+into+disease+mechanisms&amp;author=B.+Li&amp;author=L.C.+Tsoi&amp;author=W.R.+Swindell&amp;author=J.E.+Gudjonsson&amp;author=T.+Tejasvi&amp;volume=134&amp;publication_year=2014&amp;pages=1828-1838&amp;pmid=24441097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib75">75. <span class="element-citation" id="sref75">Trapnell C., Pachter L., Salzberg S.L. TopHat: discovering splice junctions with RNA-seq. <span><span class="ref-journal">Bioinformatics. </span>2009;<span class="ref-vol">25</span>:1105–1111.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2672628/">PMC free article</a>]</span> [<a href="/pubmed/19289445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=TopHat:+discovering+splice+junctions+with+RNA-seq&amp;author=C.+Trapnell&amp;author=L.+Pachter&amp;author=S.L.+Salzberg&amp;volume=25&amp;publication_year=2009&amp;pages=1105-1111&amp;pmid=19289445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib76">76. <span class="element-citation" id="sref76">Trapnell C., Williams B.A., Pertea G., Mortazavi A., Kwan G., van Baren M.J., Salzberg S.L., Wold B.J., Pachter L. Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation. <span><span class="ref-journal">Nat. Biotechnol. </span>2010;<span class="ref-vol">28</span>:511–515.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3146043/">PMC free article</a>]</span> [<a href="/pubmed/20436464" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol.&amp;title=Transcript+assembly+and+quantification+by+RNA-Seq+reveals+unannotated+transcripts+and+isoform+switching+during+cell+differentiation&amp;author=C.+Trapnell&amp;author=B.A.+Williams&amp;author=G.+Pertea&amp;author=A.+Mortazavi&amp;author=G.+Kwan&amp;volume=28&amp;publication_year=2010&amp;pages=511-515&amp;pmid=20436464&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib77">77. <span class="element-citation" id="sref77">Tsoi L.C., Iyer M.K., Stuart P.E., Swindell W.R., Gudjonsson J.E., Tejasvi T., Sarkar M.K., Li B., Ding J., Voorhees J.J. Analysis of long non-coding RNAs highlights tissue-specific expression patterns and epigenetic profiles in normal and psoriatic skin. <span><span class="ref-journal">Genome Biol. </span>2015;<span class="ref-vol">16</span>:24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4311508/">PMC free article</a>]</span> [<a href="/pubmed/25723451" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Analysis+of+long+non-coding+RNAs+highlights+tissue-specific+expression+patterns+and+epigenetic+profiles+in+normal+and+psoriatic+skin&amp;author=L.C.+Tsoi&amp;author=M.K.+Iyer&amp;author=P.E.+Stuart&amp;author=W.R.+Swindell&amp;author=J.E.+Gudjonsson&amp;volume=16&amp;publication_year=2015&amp;pages=24&amp;pmid=25723451&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib78">78. <span class="element-citation" id="sref78">Abecasis G.R., Cherny S.S., Cookson W.O., Cardon L.R. Merlin--rapid analysis of dense genetic maps using sparse gene flow trees. <span><span class="ref-journal">Nat. Genet. </span>2002;<span class="ref-vol">30</span>:97–101.</span> [<a href="/pubmed/11731797" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Merlin--rapid+analysis+of+dense+genetic+maps+using+sparse+gene+flow+trees&amp;author=G.R.+Abecasis&amp;author=S.S.+Cherny&amp;author=W.O.+Cookson&amp;author=L.R.+Cardon&amp;volume=30&amp;publication_year=2002&amp;pages=97-101&amp;pmid=11731797&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib79">79. <span class="element-citation" id="sref79">Karolchik D., Hinrichs A.S., Furey T.S., Roskin K.M., Sugnet C.W., Haussler D., Kent W.J. The UCSC Table Browser data retrieval tool. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:D493–D496.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC308837/">PMC free article</a>]</span> [<a href="/pubmed/14681465" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+UCSC+Table+Browser+data+retrieval+tool&amp;author=D.+Karolchik&amp;author=A.S.+Hinrichs&amp;author=T.S.+Furey&amp;author=K.M.+Roskin&amp;author=C.W.+Sugnet&amp;volume=32&amp;publication_year=2004&amp;pages=D493-D496&amp;pmid=14681465&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib80">80. <span class="element-citation" id="sref80">Wang K., Li M., Hakonarson H. ANNOVAR: functional annotation of genetic variants from high-throughput sequencing data. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:e164.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2938201/">PMC free article</a>]</span> [<a href="/pubmed/20601685" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=ANNOVAR:+functional+annotation+of+genetic+variants+from+high-throughput+sequencing+data&amp;author=K.+Wang&amp;author=M.+Li&amp;author=H.+Hakonarson&amp;volume=38&amp;publication_year=2010&amp;pages=e164&amp;pmid=20601685&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib81">81. <span class="element-citation" id="sref81">Ward L.D., Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. <span><span class="ref-journal">Nucleic Acids Res. </span>2012;<span class="ref-vol">40</span>:D930–D934.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3245002/">PMC free article</a>]</span> [<a href="/pubmed/22064851" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=HaploReg:+a+resource+for+exploring+chromatin+states,+conservation,+and+regulatory+motif+alterations+within+sets+of+genetically+linked+variants&amp;author=L.D.+Ward&amp;author=M.+Kellis&amp;volume=40&amp;publication_year=2012&amp;pages=D930-D934&amp;pmid=22064851&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib82">82. <span class="element-citation" id="sref82">Boyle A.P., Hong E.L., Hariharan M., Cheng Y., Schaub M.A., Kasowski M., Karczewski K.J., Park J., Hitz B.C., Weng S. Annotation of functional variation in personal genomes using RegulomeDB. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:1790–1797.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=A.P.+Boyle&amp;author=E.L.+Hong&amp;author=M.+Hariharan&amp;author=Y.+Cheng&amp;author=M.A.+Schaub&amp;volume=22&amp;publication_year=2012&amp;pages=1790-1797&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib83">83. <span class="element-citation" id="sref83">Kircher M., Witten D.M., Jain P., O’Roak B.J., Cooper G.M., Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:310–315.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3992975/">PMC free article</a>]</span> [<a href="/pubmed/24487276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+general+framework+for+estimating+the+relative+pathogenicity+of+human+genetic+variants&amp;author=M.+Kircher&amp;author=D.M.+Witten&amp;author=P.+Jain&amp;author=B.J.+O’Roak&amp;author=G.M.+Cooper&amp;volume=46&amp;publication_year=2014&amp;pages=310-315&amp;pmid=24487276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib84">84. <span class="element-citation" id="sref84">Kundaje A., Meuleman W., Ernst J., Bilenky M., Yen A., Heravi-Moussavi A., Kheradpour P., Zhang Z., Wang J., Ziller M.J., Roadmap Epigenomics Consortium  Integrative analysis of 111 reference human epigenomes. <span><span class="ref-journal">Nature. </span>2015;<span class="ref-vol">518</span>:317–330.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4530010/">PMC free article</a>]</span> [<a href="/pubmed/25693563" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Integrative+analysis+of+111+reference+human+epigenomes&amp;author=A.+Kundaje&amp;author=W.+Meuleman&amp;author=J.+Ernst&amp;author=M.+Bilenky&amp;author=A.+Yen&amp;volume=518&amp;publication_year=2015&amp;pages=317-330&amp;pmid=25693563&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib85">85. <span class="element-citation" id="sref85">Riveira-Munoz E., He S.M., Escaramís G., Stuart P.E., Hüffmeier U., Lee C., Kirby B., Oka A., Giardina E., Liao W. Meta-analysis confirms the LCE3C_LCE3B deletion as a risk factor for psoriasis in several ethnic groups and finds interaction with HLA-Cw6. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2011;<span class="ref-vol">131</span>:1105–1109.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3386316/">PMC free article</a>]</span> [<a href="/pubmed/21107349" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Meta-analysis+confirms+the+LCE3C_LCE3B+deletion+as+a+risk+factor+for+psoriasis+in+several+ethnic+groups+and+finds+interaction+with+HLA-Cw6&amp;author=E.+Riveira-Munoz&amp;author=S.M.+He&amp;author=G.+Escaramís&amp;author=P.E.+Stuart&amp;author=U.+Hüffmeier&amp;volume=131&amp;publication_year=2011&amp;pages=1105-1109&amp;pmid=21107349&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib86">86. <span class="element-citation" id="sref86">Zheng H.F., Zuo X.B., Lu W.S., Li Y., Cheng H., Zhu K.J., Yin X.Y., Zhang C., Ren Y.Q., Wang W.J. Variants in MHC, LCE and IL12B have epistatic effects on psoriasis risk in Chinese population. <span><span class="ref-journal">J.&nbsp;Dermatol. Sci. </span>2011;<span class="ref-vol">61</span>:124–128.</span> [<a href="/pubmed/21208785" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Dermatol.+Sci.&amp;title=Variants+in+MHC,+LCE+and+IL12B+have+epistatic+effects+on+psoriasis+risk+in+Chinese+population&amp;author=H.F.+Zheng&amp;author=X.B.+Zuo&amp;author=W.S.+Lu&amp;author=Y.+Li&amp;author=H.+Cheng&amp;volume=61&amp;publication_year=2011&amp;pages=124-128&amp;pmid=21208785&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib87">87. <span class="element-citation" id="sref87">Parkes M., Cortes A., van Heel D.A., Brown M.A. Genetic insights into common pathways and complex relationships among immune-mediated diseases. <span><span class="ref-journal">Nat. Rev. Genet. </span>2013;<span class="ref-vol">14</span>:661–673.</span> [<a href="/pubmed/23917628" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genetic+insights+into+common+pathways+and+complex+relationships+among+immune-mediated+diseases&amp;author=M.+Parkes&amp;author=A.+Cortes&amp;author=D.A.+van+Heel&amp;author=M.A.+Brown&amp;volume=14&amp;publication_year=2013&amp;pages=661-673&amp;pmid=23917628&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib88">88. <span class="element-citation" id="sref88">Zhu K.J., Lv Y.M., Yin X.Y., Wang Z.X., Sun L.D., He S.M., Cheng H., Hu D.Y., Zhang Z., Li Y. Psoriasis regression analysis of MHC loci identifies shared genetic variants with vitiligo. <span><span class="ref-journal">PLoS ONE. </span>2011;<span class="ref-vol">6</span>:e23089.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3220662/">PMC free article</a>]</span> [<a href="/pubmed/22125590" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Psoriasis+regression+analysis+of+MHC+loci+identifies+shared+genetic+variants+with+vitiligo&amp;author=K.J.+Zhu&amp;author=Y.M.+Lv&amp;author=X.Y.+Yin&amp;author=Z.X.+Wang&amp;author=L.D.+Sun&amp;volume=6&amp;publication_year=2011&amp;pages=e23089&amp;pmid=22125590&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib89">89. <span class="element-citation" id="sref89">de Cid R., Riveira-Munoz E., Zeeuwen P.L., Robarge J., Liao W., Dannhauser E.N., Giardina E., Stuart P.E., Nair R., Helms C. Deletion of the late cornified envelope LCE3B and LCE3C genes as a susceptibility factor for psoriasis. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:211–215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3128734/">PMC free article</a>]</span> [<a href="/pubmed/19169253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Deletion+of+the+late+cornified+envelope+LCE3B+and+LCE3C+genes+as+a+susceptibility+factor+for+psoriasis&amp;author=R.+de+Cid&amp;author=E.+Riveira-Munoz&amp;author=P.L.+Zeeuwen&amp;author=J.+Robarge&amp;author=W.+Liao&amp;volume=41&amp;publication_year=2009&amp;pages=211-215&amp;pmid=19169253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib90">90. <span class="element-citation" id="sref90">FitzGerald O., Haroon M., Giles J.T., Winchester R. Concepts of pathogenesis in psoriatic arthritis: genotype determines clinical phenotype. <span><span class="ref-journal">Arthritis Res. Ther. </span>2015;<span class="ref-vol">17</span>:115.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4422545/">PMC free article</a>]</span> [<a href="/pubmed/25948071" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Res.+Ther.&amp;title=Concepts+of+pathogenesis+in+psoriatic+arthritis:+genotype+determines+clinical+phenotype&amp;author=O.+FitzGerald&amp;author=M.+Haroon&amp;author=J.T.+Giles&amp;author=R.+Winchester&amp;volume=17&amp;publication_year=2015&amp;pages=115&amp;pmid=25948071&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib91">91. <span class="element-citation" id="sref91">Nair R.P., Tsoi L.C., Stuart P.E., Chandra V., Tejasvi T., Ellinghaus E., Voorhees J.J., Weichenthal M., Abecasis G.R., Rahman P. Meta-analysis of psoriasis and psoriatic arthritis identifies three new susceptibility loci. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2013;<span class="ref-vol">133</span>:S136.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Meta-analysis+of+psoriasis+and+psoriatic+arthritis+identifies+three+new+susceptibility+loci&amp;author=R.P.+Nair&amp;author=L.C.+Tsoi&amp;author=P.E.+Stuart&amp;author=V.+Chandra&amp;author=T.+Tejasvi&amp;volume=133&amp;publication_year=2013&amp;pages=S136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib92">92. <span class="element-citation" id="sref92">Menon B., Gullick N.J., Walter G.J., Rajasekhar M., Garrood T., Evans H.G., Taams L.S., Kirkham B.W. Interleukin-17+CD8+ T&nbsp;cells are enriched in the joints of patients with psoriatic arthritis and correlate with disease activity and joint damage progression. <span><span class="ref-journal">Arthritis Rheumatol. </span>2014;<span class="ref-vol">66</span>:1272–1281.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4158887/">PMC free article</a>]</span> [<a href="/pubmed/24470327" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheumatol.&amp;title=Interleukin-17+CD8++T&nbsp;cells+are+enriched+in+the+joints+of+patients+with+psoriatic+arthritis+and+correlate+with+disease+activity+and+joint+damage+progression&amp;author=B.+Menon&amp;author=N.J.+Gullick&amp;author=G.J.+Walter&amp;author=M.+Rajasekhar&amp;author=T.+Garrood&amp;volume=66&amp;publication_year=2014&amp;pages=1272-1281&amp;pmid=24470327&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib93">93. <span class="element-citation" id="sref93">Kryczek I., Bruce A.T., Gudjonsson J.E., Johnston A., Aphale A., Vatan L., Szeliga W., Wang Y., Liu Y., Welling T.H. Induction of IL-17+ T&nbsp;cell trafficking and development by IFN-gamma: mechanism and pathological relevance in psoriasis. <span><span class="ref-journal">J.&nbsp;Immunol. </span>2008;<span class="ref-vol">181</span>:4733–4741.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2677162/">PMC free article</a>]</span> [<a href="/pubmed/18802076" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Immunol.&amp;title=Induction+of+IL-17++T&nbsp;cell+trafficking+and+development+by+IFN-gamma:+mechanism+and+pathological+relevance+in+psoriasis&amp;author=I.+Kryczek&amp;author=A.T.+Bruce&amp;author=J.E.+Gudjonsson&amp;author=A.+Johnston&amp;author=A.+Aphale&amp;volume=181&amp;publication_year=2008&amp;pages=4733-4741&amp;pmid=18802076&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib94">94. <span class="element-citation" id="sref94">Beecham A.H., Patsopoulos N.A., Xifara D.K., Davis M.F., Kemppinen A., Cotsapas C., Shah T.S., Spencer C., Booth D., Goris A., International Multiple Sclerosis Genetics Consortium (IMSGC) Wellcome Trust Case Control Consortium 2 (WTCCC2) International IBD Genetics Consortium (IIBDGC)  Analysis of immune-related loci identifies 48 new susceptibility variants for multiple sclerosis. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:1353–1360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832895/">PMC free article</a>]</span> [<a href="/pubmed/24076602" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Analysis+of+immune-related+loci+identifies+48+new+susceptibility+variants+for+multiple+sclerosis&amp;author=A.H.+Beecham&amp;author=N.A.+Patsopoulos&amp;author=D.K.+Xifara&amp;author=M.F.+Davis&amp;author=A.+Kemppinen&amp;volume=45&amp;publication_year=2013&amp;pages=1353-1360&amp;pmid=24076602&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib95">95. <span class="element-citation" id="sref95">Docampo E., Giardina E., Riveira-Muñoz E., de Cid R., Escaramís G., Perricone C., Fernández-Sueiro J.L., Maymó J., González-Gay M.A., Blanco F.J. Deletion of LCE3C and LCE3B is a susceptibility factor for psoriatic arthritis: a study in Spanish and Italian populations and meta-analysis. <span><span class="ref-journal">Arthritis Rheum. </span>2011;<span class="ref-vol">63</span>:1860–1865.</span> [<a href="/pubmed/21400479" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+Rheum.&amp;title=Deletion+of+LCE3C+and+LCE3B+is+a+susceptibility+factor+for+psoriatic+arthritis:+a+study+in+Spanish+and+Italian+populations+and+meta-analysis&amp;author=E.+Docampo&amp;author=E.+Giardina&amp;author=E.+Riveira-Muñoz&amp;author=R.+de+Cid&amp;author=G.+Escaramís&amp;volume=63&amp;publication_year=2011&amp;pages=1860-1865&amp;pmid=21400479&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib96">96. <span class="element-citation" id="sref96">Chiraz B.S., Myriam A., Ines Z., Catherine J., Fatma B., Ilhem C., Raoudha T., Hela Z., Hela F., Elyes B. Deletion of late cornified envelope genes, LCE3C_LCE3B-del, is not associated with psoriatic arthritis in Tunisian patients. <span><span class="ref-journal">Mol. Biol. Rep. </span>2014;<span class="ref-vol">41</span>:4141–4146.</span> [<a href="/pubmed/24566688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Rep.&amp;title=Deletion+of+late+cornified+envelope+genes,+LCE3C_LCE3B-del,+is+not+associated+with+psoriatic+arthritis+in+Tunisian+patients&amp;author=B.S.+Chiraz&amp;author=A.+Myriam&amp;author=Z.+Ines&amp;author=J.+Catherine&amp;author=B.+Fatma&amp;volume=41&amp;publication_year=2014&amp;pages=4141-4146&amp;pmid=24566688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib97">97. <span class="element-citation" id="sref97">Coto E., Santos-Juanes J., Coto-Segura P., Díaz M., Soto J., Queiro R., Alvarez V. Mutation analysis of the LCE3B/LCE3C genes in psoriasis. <span><span class="ref-journal">BMC Med. Genet. </span>2010;<span class="ref-vol">11</span>:45.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2859809/">PMC free article</a>]</span> [<a href="/pubmed/20331852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=Mutation+analysis+of+the+LCE3B/LCE3C+genes+in+psoriasis&amp;author=E.+Coto&amp;author=J.+Santos-Juanes&amp;author=P.+Coto-Segura&amp;author=M.+Díaz&amp;author=J.+Soto&amp;volume=11&amp;publication_year=2010&amp;pages=45&amp;pmid=20331852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib98">98. <span class="element-citation" id="sref98">Hüffmeier U., Estivill X., Riveira-Munoz E., Traupe H., Wendler J., Lohmann J., Böhm B., Burkhardt H., Reis A. Deletion of LCE3C and LCE3B genes at PSORS4 does not contribute to susceptibility to psoriatic arthritis in German patients. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2010;<span class="ref-vol">69</span>:876–878.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925148/">PMC free article</a>]</span> [<a href="/pubmed/19439430" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Deletion+of+LCE3C+and+LCE3B+genes+at+PSORS4+does+not+contribute+to+susceptibility+to+psoriatic+arthritis+in+German+patients&amp;author=U.+Hüffmeier&amp;author=X.+Estivill&amp;author=E.+Riveira-Munoz&amp;author=H.+Traupe&amp;author=J.+Wendler&amp;volume=69&amp;publication_year=2010&amp;pages=876-878&amp;pmid=19439430&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib99">99. <span class="element-citation" id="sref99">Bowes J., Flynn E., Ho P., Aly B., Morgan A.W., Marzo-Ortega H., Coates L., McManus R., Ryan A.W., Kane D. Variants in linkage disequilibrium with the late cornified envelope gene cluster deletion are associated with susceptibility to psoriatic arthritis. <span><span class="ref-journal">Ann. Rheum. Dis. </span>2010;<span class="ref-vol">69</span>:2199–2203.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3002763/">PMC free article</a>]</span> [<a href="/pubmed/20643763" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Rheum.+Dis.&amp;title=Variants+in+linkage+disequilibrium+with+the+late+cornified+envelope+gene+cluster+deletion+are+associated+with+susceptibility+to+psoriatic+arthritis&amp;author=J.+Bowes&amp;author=E.+Flynn&amp;author=P.+Ho&amp;author=B.+Aly&amp;author=A.W.+Morgan&amp;volume=69&amp;publication_year=2010&amp;pages=2199-2203&amp;pmid=20643763&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib100">100. <span class="element-citation" id="sref100">Bergboer J.G., Tjabringa G.S., Kamsteeg M., van Vlijmen-Willems I.M., Rodijk-Olthuis D., Jansen P.A., Thuret J.Y., Narita M., Ishida-Yamamoto A., Zeeuwen P.L., Schalkwijk J. Psoriasis risk genes of the late cornified envelope-3 group are distinctly expressed compared with genes of other LCE groups. <span><span class="ref-journal">Am. J. Pathol. </span>2011;<span class="ref-vol">178</span>:1470–1477.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3078472/">PMC free article</a>]</span> [<a href="/pubmed/21435436" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Pathol.&amp;title=Psoriasis+risk+genes+of+the+late+cornified+envelope-3+group+are+distinctly+expressed+compared+with+genes+of+other+LCE+groups&amp;author=J.G.+Bergboer&amp;author=G.S.+Tjabringa&amp;author=M.+Kamsteeg&amp;author=I.M.+van+Vlijmen-Willems&amp;author=D.+Rodijk-Olthuis&amp;volume=178&amp;publication_year=2011&amp;pages=1470-1477&amp;pmid=21435436&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib101">101. <span class="element-citation" id="sref101">Coto E., Santos-Juanes J., Coto-Segura P., Alvarez V. New psoriasis susceptibility genes: momentum for skin-barrier disruption. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2011;<span class="ref-vol">131</span>:1003–1005.</span> [<a href="/pubmed/21494239" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=New+psoriasis+susceptibility+genes:+momentum+for+skin-barrier+disruption&amp;author=E.+Coto&amp;author=J.+Santos-Juanes&amp;author=P.+Coto-Segura&amp;author=V.+Alvarez&amp;volume=131&amp;publication_year=2011&amp;pages=1003-1005&amp;pmid=21494239&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib102">102. <span class="element-citation" id="sref102">de Guzman Strong C., Conlan S., Deming C.B., Cheng J., Sears K.E., Segre J.A. A milieu of regulatory elements in the epidermal differentiation complex syntenic block: implications for atopic dermatitis and psoriasis. <span><span class="ref-journal">Hum. Mol. Genet. </span>2010;<span class="ref-vol">19</span>:1453–1460.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846160/">PMC free article</a>]</span> [<a href="/pubmed/20089530" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=A+milieu+of+regulatory+elements+in+the+epidermal+differentiation+complex+syntenic+block:+implications+for+atopic+dermatitis+and+psoriasis&amp;author=C.+de+Guzman+Strong&amp;author=S.+Conlan&amp;author=C.B.+Deming&amp;author=J.+Cheng&amp;author=K.E.+Sears&amp;volume=19&amp;publication_year=2010&amp;pages=1453-1460&amp;pmid=20089530&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib103">103. <span class="element-citation" id="sref103">Welter D., MacArthur J., Morales J., Burdett T., Hall P., Junkins H., Klemm A., Flicek P., Manolio T., Hindorff L., Parkinson H. The NHGRI GWAS Catalog, a curated resource of SNP-trait associations. <span><span class="ref-journal">Nucleic Acids Res. </span>2014;<span class="ref-vol">42</span>:D1001–D1006.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965119/">PMC free article</a>]</span> [<a href="/pubmed/24316577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+NHGRI+GWAS+Catalog,+a+curated+resource+of+SNP-trait+associations&amp;author=D.+Welter&amp;author=J.+MacArthur&amp;author=J.+Morales&amp;author=T.+Burdett&amp;author=P.+Hall&amp;volume=42&amp;publication_year=2014&amp;pages=D1001-D1006&amp;pmid=24316577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib104">104. <span class="element-citation" id="sref104">Li Y., Liao W., Chang M., Schrodi S.J., Bui N., Catanese J.J., Poon A., Matsunami N., Callis-Duffin K.P., Leppert M.F. Further genetic evidence for three psoriasis-risk genes: ADAM33, CDKAL1, and PTPN22. <span><span class="ref-journal">J.&nbsp;Invest. Dermatol. </span>2009;<span class="ref-vol">129</span>:629–634.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4130997/">PMC free article</a>]</span> [<a href="/pubmed/18923449" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.&nbsp;Invest.+Dermatol.&amp;title=Further+genetic+evidence+for+three+psoriasis-risk+genes:+ADAM33,+CDKAL1,+and+PTPN22&amp;author=Y.+Li&amp;author=W.+Liao&amp;author=M.+Chang&amp;author=S.J.+Schrodi&amp;author=N.+Bui&amp;volume=129&amp;publication_year=2009&amp;pages=629-634&amp;pmid=18923449&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib105">105. <span class="element-citation" id="sref105">Quaranta M., Burden A.D., Griffiths C.E., Worthington J., Barker J.N., Trembath R.C., Capon F. Differential contribution of CDKAL1 variants to psoriasis, Crohn’s disease and type II diabetes. <span><span class="ref-journal">Genes Immun. </span>2009;<span class="ref-vol">10</span>:654–658.</span> [<a href="/pubmed/19587699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun.&amp;title=Differential+contribution+of+CDKAL1+variants+to+psoriasis,+Crohn’s+disease+and+type+II+diabetes&amp;author=M.+Quaranta&amp;author=A.D.+Burden&amp;author=C.E.+Griffiths&amp;author=J.+Worthington&amp;author=J.N.+Barker&amp;volume=10&amp;publication_year=2009&amp;pages=654-658&amp;pmid=19587699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4678416&amp;issue-id=261962&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139913885105120title">Abstract</a></li><li><a href="#sec1title">Introduction</a></li><li><a href="#sec2title">Material and Methods</a></li><li><a href="#sec3title">Results</a></li><li><a href="#sec4title">Discussion</a></li><li><a href="#ack0010title">Acknowledgments</a></li><li><a href="#idm139913887819808title">Notes</a></li><li><a href="#idm139913964700160title">Footnotes</a></li><li><a href="#app1title">Accession Numbers</a></li><li><a href="#app4title">Web Resources</a></li><li><a href="#app3title">Supplemental Data</a></li><li><a href="#idm139913890396032title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <strong>American Society of Human Genetics</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4678416/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4678416/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4678416/pdf/main.pdf">PDF (2.6M)</a></li> | <li><a href="#" data-citationid="PMC4678416" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4678416%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4678416%2F&amp;text=Genome-wide%20Association%20Analysis%20of%20Psoriatic%20Arthritis%20and%20Cutaneous%20Psoriasis%20Reveals%20Differences%20in%20Their%20Genetic%20Architecture" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4678416%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26626624" data-aiid="4678416" data-aid="4678416" data-iid="261962" data-domainid="203" data-domain="ajhg" data-accid="PMC4678416" data-md5="cd4c33014cfbbbfe233addaec8d8c37b"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:19:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D3762DD58C3C100000000035D0202&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4678416%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=ajhg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4678416/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>