<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4334839/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide admixture and association study of serum iron, ferritin, transferrin saturation and total iron binding capacity in African Americans</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4334839/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide admixture and association study of serum iron, ferritin, transferrin saturation and total iron binding capacity in African Americans"><meta name="citation_authors" content="Jin Li, Leslie A. Lange, Qing Duan, Yurong Lu, Andrew B. Singleton, Alan B. Zonderman, Michele K. Evans, Yun Li, Herman A. Taylor, Monte S. Willis, Mike Nalls, James G. Wilson, Ethan M. Lange"><meta name="citation_date" content="15 January 2015"><meta name="citation_issue" content="2"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="572"><meta name="citation_doi" content="10.1093/hmg/ddu454"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4334839/?report=abstract"><meta name="citation_pmid" content="25224454"><meta name="DC.Title" content="Genome-wide admixture and association study of serum iron, ferritin, transferrin saturation and total iron binding capacity in African Americans"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Jin Li"><meta name="DC.Contributor" content="Leslie A. Lange"><meta name="DC.Contributor" content="Qing Duan"><meta name="DC.Contributor" content="Yurong Lu"><meta name="DC.Contributor" content="Andrew B. Singleton"><meta name="DC.Contributor" content="Alan B. Zonderman"><meta name="DC.Contributor" content="Michele K. Evans"><meta name="DC.Contributor" content="Yun Li"><meta name="DC.Contributor" content="Herman A. Taylor"><meta name="DC.Contributor" content="Monte S. Willis"><meta name="DC.Contributor" content="Mike Nalls"><meta name="DC.Contributor" content="James G. Wilson"><meta name="DC.Contributor" content="Ethan M. Lange"><meta name="DC.Date" content="2015 Jan 15"><meta name="DC.Identifier" content="10.1093/hmg/ddu454"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide admixture and association study of serum iron, ferritin, transferrin saturation and total iron binding capacity in African Americans"><meta property="og:type" content="article"><meta property="og:description" content="Iron is an essential component of many important proteins and enzymes, including hemoglobin, which is responsible for carrying oxygen to the cells. African Americans (AAs) have a greater prevalence of iron deficiency compared with European Americans. ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4334839/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4334839/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4334839/pdf/ddu454.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8C5CD9DD576D510000000002CE01B1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4334839%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC4334839</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2015 Jan 15; 24(2): 572–581. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Sep 15. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu454" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddu454</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4334839</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25224454">25224454</a></div></div></div></div><h1 class="content-title">Genome-wide admixture and association study of serum iron, ferritin, transferrin saturation and total iron binding capacity in African Americans</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826556609088" co-class="co-affbox">Jin Li</a>,<sup>1</sup> <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826564081520" co-class="co-affbox">Leslie A. Lange</a>,<sup>1</sup> <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826593600592" co-class="co-affbox">Qing Duan</a>,<sup>1</sup> <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826558780352" co-class="co-affbox">Yurong Lu</a>,<sup>1</sup> <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826558778368" co-class="co-affbox">Andrew B. Singleton</a>,<sup>4</sup> <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826554824544" co-class="co-affbox">Alan B. Zonderman</a>,<sup>5</sup> <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826554822560" co-class="co-affbox">Michele K. Evans</a>,<sup>6</sup> <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826592617136" co-class="co-affbox">Yun Li</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Taylor%20HA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826558585056" co-class="co-affbox">Herman A. Taylor</a>,<sup>7,</sup><sup>9,</sup><sup>10</sup> <a href="/pubmed/?term=Willis%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826558581440" co-class="co-affbox">Monte S. Willis</a>,<sup>3</sup> <a href="/pubmed/?term=Nalls%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826560342240" co-class="co-affbox">Mike Nalls</a>,<sup>4</sup> <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826560340256" co-class="co-affbox">James G. Wilson</a>,<sup>8</sup> and  <a href="/pubmed/?term=Lange%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454" class="affpopup" co-rid="_co_idm139826555283600" co-class="co-affbox">Ethan M. Lange</a><sup>1,</sup><sup>2,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139826556609088"><h3 class="no_margin">Jin Li</h3><p><sup>1</sup>Department of Genetics</p><div>Find articles by <a href="/pubmed/?term=Li%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Jin Li</a></div></div><div id="_co_idm139826564081520"><h3 class="no_margin">Leslie A. Lange</h3><p><sup>1</sup>Department of Genetics</p><div>Find articles by <a href="/pubmed/?term=Lange%20LA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Leslie A. Lange</a></div></div><div id="_co_idm139826593600592"><h3 class="no_margin">Qing Duan</h3><p><sup>1</sup>Department of Genetics</p><div>Find articles by <a href="/pubmed/?term=Duan%20Q%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Qing Duan</a></div></div><div id="_co_idm139826558780352"><h3 class="no_margin">Yurong Lu</h3><p><sup>1</sup>Department of Genetics</p><div>Find articles by <a href="/pubmed/?term=Lu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Yurong Lu</a></div></div><div id="_co_idm139826558778368"><h3 class="no_margin">Andrew B. Singleton</h3><p><sup>4</sup>Laboratory of Neurogenetics</p><div>Find articles by <a href="/pubmed/?term=Singleton%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Andrew B. Singleton</a></div></div><div id="_co_idm139826554824544"><h3 class="no_margin">Alan B. Zonderman</h3><p><sup>5</sup>Laboratory of Personality and Cognition,  and</p><div>Find articles by <a href="/pubmed/?term=Zonderman%20AB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Alan B. Zonderman</a></div></div><div id="_co_idm139826554822560"><h3 class="no_margin">Michele K. Evans</h3><p><sup>6</sup>Health Disparities Unit, National Institute on Aging, National Institutes of Health, Bethesda, MD 21225, USA</p><div>Find articles by <a href="/pubmed/?term=Evans%20MK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Michele K. Evans</a></div></div><div id="_co_idm139826592617136"><h3 class="no_margin">Yun Li</h3><p><sup>1</sup>Department of Genetics</p><p><sup>2</sup>Department of Biostatistics,  and</p><div>Find articles by <a href="/pubmed/?term=Li%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Yun Li</a></div></div><div id="_co_idm139826558585056"><h3 class="no_margin">Herman A. Taylor</h3><p><sup>7</sup>Department of Medicine,  and</p><p><sup>9</sup>School of Health Sciences, Jackson State University, Jackson, MS 39217, USA,  and</p><p><sup>10</sup>Division of Natural Science, Tougaloo College, Tougaloo, MS 39174, USA</p><div>Find articles by <a href="/pubmed/?term=Taylor%20HA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Herman A. Taylor</a></div></div><div id="_co_idm139826558581440"><h3 class="no_margin">Monte S. Willis</h3><p><sup>3</sup>Department of Pathology and Laboratory Medicine, University of North Carolina, Chapel Hill, NC 27599, USA</p><div>Find articles by <a href="/pubmed/?term=Willis%20MS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Monte S. Willis</a></div></div><div id="_co_idm139826560342240"><h3 class="no_margin">Mike Nalls</h3><p><sup>4</sup>Laboratory of Neurogenetics</p><div>Find articles by <a href="/pubmed/?term=Nalls%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Mike Nalls</a></div></div><div id="_co_idm139826560340256"><h3 class="no_margin">James G. Wilson</h3><p><sup>8</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS 39216, USA</p><div>Find articles by <a href="/pubmed/?term=Wilson%20JG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">James G. Wilson</a></div></div><div id="_co_idm139826555283600"><h3 class="no_margin">Ethan M. Lange</h3><p><sup>1</sup>Department of Genetics</p><p><sup>2</sup>Department of Biostatistics,  and</p><div>Find articles by <a href="/pubmed/?term=Lange%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25224454">Ethan M. Lange</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139826556378160_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139826556378160_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139826556378160_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139826556378160_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Department of Genetics</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Biostatistics,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Department of Pathology and Laboratory Medicine, University of North Carolina, Chapel Hill, NC 27599, USA</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Laboratory of Neurogenetics</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Laboratory of Personality and Cognition,  and</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Health Disparities Unit, National Institute on Aging, National Institutes of Health, Bethesda, MD 21225, USA</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Department of Medicine,  and</div><div class="fm-affl" id="af8" lang="en"><sup>8</sup>Department of Physiology and Biophysics, University of Mississippi Medical Center, Jackson, MS 39216, USA</div><div class="fm-affl" id="af9" lang="en"><sup>9</sup>School of Health Sciences, Jackson State University, Jackson, MS 39217, USA,  and</div><div class="fm-affl" id="af10" lang="en"><sup>10</sup>Division of Natural Science, Tougaloo College, Tougaloo, MS 39174, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: Department of Genetics, University of North Carolina at Chapel Hill, 5111 Genetic Medicine Building, Chapel Hill, NC 27599-7064, USA. Tel: +1 9199663356; Fax: +1 9198434682; Email: <a href="mailto:dev@null" data-email="ude.cnu.dem@egnale" class="oemail">ude.cnu.dem@egnale</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139826556378160_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Apr 21; Revised 2014 Aug 11; Accepted 2014 Sep 1.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139826556378160_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2014. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4334839/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4334839/bin/supp_24_2_572__index.html" data-ga-action="click_feat_suppl">supp_24_2_572__index.html</a><span style="color:gray"> (1.2K)</span></div><div class="small guid">GUID:&nbsp;CA11C73B-4D60-4E03-874A-57F1FED473F3</div>
<div><a href="/pmc/articles/PMC4334839/bin/supp_ddu454_ddu454supp.docx" data-ga-action="click_feat_suppl">supp_ddu454_ddu454supp.docx</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;C5C5B690-AE83-4233-97F4-8B11C68632BC</div>
</div></dd></dl></div><div id="idm139826559335264" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139826559335264title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Iron is an essential component of many important proteins and enzymes, including hemoglobin, which is responsible for carrying oxygen to the cells. African Americans (AAs) have a greater prevalence of iron deficiency compared with European Americans. We conducted genome-wide admixture-mapping and association studies for serum iron, serum ferritin, transferrin saturation (SAT) and total iron binding capacity (TIBC) in 2347 AAs participating in the Jackson Heart Study (JHS). Follow-up replication analyses for JHS iron-trait associated SNPs were conducted in 329 AA participants in the Healthy Aging in Neighborhoods of Diversity across the Life Span study (HANDLS). Higher estimated proportions of global African ancestry were significantly associated with lower levels of iron (<em>P</em> = 2.4 × 10<sup>−5</sup>), SAT (<em>P</em> = 0.0019) and TIBC (<em>P</em> = 0.042). We observed significant associations (<em>P</em> &lt; 5 × 10<sup>−8</sup>) between serum TIBC levels and two independent SNPs around <em>TF</em> on chromosome 3<em>,</em> the first report of a genome-wide significant second independent signal in this region, and SNPs near two novel genes: <em>HDGFL1</em> on chromosome 6 and <em>MAF</em> on chromosome 16. We also observed significant associations between ferritin levels and SNPs near <em>GAB3</em> on chromosome X<em>.</em> We replicated our two independent associations at <em>TF</em> and our association at <em>GAB3</em> in HANDLS. Our study provides evidence for both shared and unique genetic risk factors that are associated with iron-related measures in AAs. The top two variants in <em>TF</em> explain 11.2% of the total variation in TIBC levels in AAs after accounting for age, gender, body mass index and background ancestry.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Iron is critical to an array of metabolic functions, such as oxygen transport and oxidative phosphorylation. Normally, small daily losses of iron in the feces and through menstruation are balanced by its regulated intestinal absorption and its recovery from heme after phagocytosis of senescent red blood cells (RBCs) (<a href="#DDU454C1" rid="DDU454C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981852">1</a>). Iron deficiency can cause anemia, while iron overload may lead to increased risk for cardiovascular disease, including cardiomyopathy, diabetes mellitus, arthritis and liver disease (<a href="#DDU454C2" rid="DDU454C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981869">2</a>). Laboratory tests typically used to assess iron transport and storage include serum iron, total iron binding capacity (TIBC), transferrin saturation (SAT) and serum ferritin. Ferritin, the predominant iron storage protein, reflects the cumulative iron stores in the bone marrow and tissues. Transferrin functions in iron transport, and the concentration of transferrin is proportional to the TIBC in serum. Transferrin SAT, calculated as (serum iron/TIBC) × 100, is affected by the rate of iron absorption in the small bowel as well as the sufficiency of tissue iron stores. Genetic factors could affect iron metabolism through gastrointestinal absorption, transport, tissue uptake, storage or remobilization from tissue stores, and thus could have a large impact on the variation of these iron-related measures (<a href="#DDU454C2" rid="DDU454C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981862">2</a>,<a href="#DDU454C3" rid="DDU454C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981890">3</a>).</p><p id="__p3" class="p p-last">Genome-wide association studies (GWAS) have identified seven loci associated with these measures in subjects of European descent (<em>P</em> &lt; 5 × 10<sup>−8</sup>) (<a href="#DDU454C4" rid="DDU454C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981874">4</a>–<a href="#DDU454C7" rid="DDU454C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981867">7</a>), including variants in or near <em>TF-SRPRB, SLC17A1, HFE, HIST1H2BJ, SIK3, PCSK7</em>&nbsp;and <em>TMPRSS6</em>. Here, we present a genome-wide admixture and association study of serum iron, TIBC, SAT and ferritin among African Americans (AAs) enrolled in the Jackson Heart Study (JHS) and Healthy Aging in Neighborhoods of Diversity across the Life Span study (HANDLS) cohorts.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><p id="__p4" class="p p-first">Descriptive statistics for the JHS and HANDLS participants included in this study are detailed in Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB1" rid-ob="ob-DDU454TB1" co-legend-rid=""><span>1</span></a>. The correlations between these four phenotypes in JHS are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU454TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p5">Descriptive statistics of JHS and HANDLS participants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826554813216" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left"></th><th colspan="3" rowspan="1" align="center">JHS<hr></th><th colspan="3" rowspan="1" align="center">HANDLS<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Total</th><th rowspan="1" colspan="1" align="left">Male</th><th rowspan="1" colspan="1" align="left">Female</th><th rowspan="1" colspan="1" align="left">Total</th><th rowspan="1" colspan="1" align="left">Male</th><th rowspan="1" colspan="1" align="left">Female</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Sample size</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">1012</td><td rowspan="1" colspan="1">1335</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">189</td><td rowspan="1" colspan="1">140</td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1">54.5 ± 12.6</td><td rowspan="1" colspan="1">53.0 ± 12.8</td><td rowspan="1" colspan="1">55.7 ± 12.4</td><td rowspan="1" colspan="1">49.4 ± 8.3</td><td rowspan="1" colspan="1">49.0 ± 8.7</td><td rowspan="1" colspan="1">49.9 ± 7.7</td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1">31.5 ± 6.9</td><td rowspan="1" colspan="1">29.9 ± 6.1</td><td rowspan="1" colspan="1">32.8 ± 7.3</td><td rowspan="1" colspan="1">28.8 ± 7.6</td><td rowspan="1" colspan="1">27.1 ± 5.4</td><td rowspan="1" colspan="1">31.2 ± 9.3</td></tr><tr><td rowspan="1" colspan="1">Ferritin (ng/ml)</td><td rowspan="1" colspan="1">134.0 (74.0, 232.0)</td><td rowspan="1" colspan="1">177.5 (110.8, 286.0)</td><td rowspan="1" colspan="1">105 (58.0, 185.5)</td><td rowspan="1" colspan="1">107.5 (54.0, 201.0)</td><td rowspan="1" colspan="1">137.0 (76.0, 257.0)</td><td rowspan="1" colspan="1">71.0 (35.0, 113.0)</td></tr><tr><td rowspan="1" colspan="1">Iron (μg/dl)</td><td rowspan="1" colspan="1">81.0 (67.0, 98.0)</td><td rowspan="1" colspan="1">86.0 (70.0, 104.0)</td><td rowspan="1" colspan="1">78.0 (64.0, 94.0)</td><td rowspan="1" colspan="1">83.0 (66.2, 103.8)</td><td rowspan="1" colspan="1">86.0 (70.0, 109.0)</td><td rowspan="1" colspan="1">81.0 (64.0, 98.0)</td></tr><tr><td rowspan="1" colspan="1">SAT (%)</td><td rowspan="1" colspan="1">28.0 (23.0, 35.0)</td><td rowspan="1" colspan="1">30.0 (25.0, 37.0)</td><td rowspan="1" colspan="1">27.0 (22.0, 33.0)</td><td rowspan="1" colspan="1">25.0 (20.2, 31.1)</td><td rowspan="1" colspan="1">26.4 (20.8, 33.5)</td><td rowspan="1" colspan="1">23.7 (19.3, 27.3)</td></tr><tr><td rowspan="1" colspan="1">TIBC (μg/dl)</td><td rowspan="1" colspan="1">284 (260.0, 314.0)</td><td rowspan="1" colspan="1">280.0 (256.0, 307.0)</td><td rowspan="1" colspan="1">288.0 (263.0, 319.0)</td><td rowspan="1" colspan="1">332.0 (306.0, 372.0)</td><td rowspan="1" colspan="1">326.0 (300.0, 368.0)</td><td rowspan="1" colspan="1">342.5 (316.8, 379.0)</td></tr></tbody></table></div><div id="largeobj_idm139826554813216" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4334839/table/DDU454TB1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><p id="__p6" class="p p-first-last">Note: data are mean ± SE, median (25th, 75th percentiles).</p></div><div id="TN2"><p id="__p7" class="p p-first-last">SAT, transferrin saturation; TIBC, total iron binding capacity.</p></div></div></div><div id="s2a" class="sec"><h3 id="s2atitle">Admixture analyses</h3><p id="__p8" class="p p-first-last">A higher level of estimated overall average African ancestry was significantly associated (<em>P</em> &lt; 0.05) with lower levels of TIBC, iron and SAT (Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB2" rid-ob="ob-DDU454TB2" co-legend-rid=""><span>2</span></a>). No individual region had local ancestry significantly associated with the iron measures. Three regions (<em>DZIP1L</em> on chr 3, <em>TRDN</em> on chr 6 and <em>TMCO5B-RYR3</em> on chr 15) had a LOD score &gt;3 for TIBC&nbsp;and one region (<em>DEFB129-DEFB132</em> on chr 20) had a LOD score &gt;3 for iron. The plots of the LOD scores across the whole genome are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figure S1a–d</a>.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU454TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p9">Association between global African ancestry estimate and phenotypes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826555242704" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Phenotypes</th><th rowspan="1" colspan="1" align="left"><em>β</em><sup>a</sup></th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">−13.90</td><td rowspan="1" colspan="1">6.80</td><td rowspan="1" colspan="1">0.04</td></tr><tr><td rowspan="1" colspan="1">log_ferritin</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1">0.32</td></tr><tr><td rowspan="1" colspan="1">log_iron</td><td rowspan="1" colspan="1">−0.20</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">2.4E-05</td></tr><tr><td rowspan="1" colspan="1">log_SAT</td><td rowspan="1" colspan="1">−0.15</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.0019</td></tr></tbody></table></div><div id="largeobj_idm139826555242704" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4334839/table/DDU454TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN3"><p id="__p10" class="p p-first-last"><sup>a</sup>The predicted change in the iron measure for each one-percentage point increase in estimated global African ancestry.</p></div></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Summary GWAS results in JHS</h3><p id="__p11" class="p p-first-last">One-hundred-fifty-seven SNPs reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) for TIBC (153 SNPs on chromosome 3, 3 SNPs on chromosome 6 and 1 SNP on chromosome 16); five SNPs were significant for ferritin (all on chromosome X) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>). No SNPs reached genome-wide significance for serum iron or SAT. Top results for all four traits are listed in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Tables S3–S6</a>. Manhattan plots and quantile–quantile (Q–Q) plots for the four traits are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Figures S2–S5</a>. Q–Q plots revealed no substantial evidence for inflated results, due to population stratification, residual relatedness among subjects or experimental outliers.</p></div><div id="s2c" class="sec"><h3 id="s2ctitle">TIBC GWAS results on chromosome 3</h3><p id="__p12" class="p p-first">All top SNPs on chromosome 3 clustered within a region spanning &lt;150 kb containing three genes: <em>TOPBP1</em>, <em>TF</em> and <em>SRPRB</em> (Fig.&nbsp;<a href="/pmc/articles/PMC4334839/figure/DDU454F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU454F1" rid-ob="ob-DDU454F1" co-legend-rid="lgnd_DDU454F1"><span>1</span></a>A). The strongest signal [rs8177253, <em>P</em> = 1.8 × 10<sup>−47</sup>, minor allele frequency (MAF) = 0.24] mapped to the <em>TF</em> gene, which encodes transferrin (Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB3" rid-ob="ob-DDU454TB3" co-legend-rid=""><span>3</span></a>). Rs8177253 was also nominally (defined as unadjusted <em>P</em> &lt; 0.05) associated with SAT (<em>P</em> = 3.0 × 10<sup>−7</sup>). Thirty-six <em>TF</em> region SNPs were genome-wide significant for TIBC levels after multi-SNP analyses, including rs8177253 as a covariate (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S7</a>). The top SNP in the conditional analysis was rs9872999 (<em>P</em> = 5.4 × 10<sup>−20</sup>, MAF = 0.38), which was not significant (<em>P</em> = 1.0 × 10<sup>−6</sup>) prior to the adjustment for rs8177253 (Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB3" rid-ob="ob-DDU454TB3" co-legend-rid=""><span>3</span></a> and Fig.&nbsp;<a href="/pmc/articles/PMC4334839/figure/DDU454F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU454F1" rid-ob="ob-DDU454F1" co-legend-rid="lgnd_DDU454F1"><span>1</span></a>B). There was no evidence of an interaction between rs8177253 and rs9872999 (<em>P</em> = 0.11). No SNPs remained genome-wide significant after covariate adjustment for both rs8177253 and rs9872999, although a large number of SNPs remained nominally significant (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S8</a>). SNPs rs8177253 and rs9872999 together explained an estimated 11.2% of the total variance of TIBC after accounting for age, gender, BMI and the first 10 PCs.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU454TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p13">Top SNPs that reached genome-wide significance (<em>P</em> &lt; 5 × 10<sup>−8</sup>) in JHS and replication results from HANDLS</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826557602752" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Trait</th><th rowspan="2" colspan="1" align="left">Chr</th><th rowspan="2" colspan="1" align="left">Nearest gene</th><th rowspan="2" colspan="1" align="left">SNP</th><th rowspan="2" colspan="1" align="left">Pos(hg19)</th><th rowspan="2" colspan="1" align="left">EA</th><th colspan="6" rowspan="1" align="center">JHS<hr></th><th colspan="6" rowspan="1" align="center">HANDLS<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left">N</th><th rowspan="1" colspan="1" align="left">R<sup>2</sup></th><th rowspan="1" colspan="1" align="left">β</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left">N</th><th rowspan="1" colspan="1" align="left">R<sup>2</sup></th><th rowspan="1" colspan="1" align="left">β</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">rs8177253</td><td rowspan="1" colspan="1">133480192</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">NA</td><td rowspan="1" colspan="1">19.86</td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">1.8E−47</td><td rowspan="1" colspan="1">0.23</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">26.72</td><td rowspan="1" colspan="1">4.91</td><td rowspan="1" colspan="1">1.1E−07</td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">rs9872999</td><td rowspan="1" colspan="1">133457514</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">−6.55</td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">1.1E−06</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">−14.19</td><td rowspan="1" colspan="1">4.33</td><td rowspan="1" colspan="1">0.0012</td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1"><em>TF</em></td><td rowspan="1" colspan="1">rs9872999*</td><td rowspan="1" colspan="1">133457514</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">−11.72</td><td rowspan="1" colspan="1">1.28</td><td rowspan="1" colspan="1">5.4E−20</td><td rowspan="1" colspan="1">0.60</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.93</td><td rowspan="1" colspan="1">−19.11</td><td rowspan="1" colspan="1">4.16</td><td rowspan="1" colspan="1">6.2E−06</td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1"><em>HDGFL1</em></td><td rowspan="1" colspan="1">rs115923437</td><td rowspan="1" colspan="1">22678302</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.06</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">14.84</td><td rowspan="1" colspan="1">2.6</td><td rowspan="1" colspan="1">1.1E−08</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.96</td><td rowspan="1" colspan="1">−5.28</td><td rowspan="1" colspan="1">9.93</td><td rowspan="1" colspan="1">0.60</td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1"><em>MAF-DYNLRB2</em></td><td rowspan="1" colspan="1">rs16951289</td><td rowspan="1" colspan="1">79790621</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.07</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1">13.38</td><td rowspan="1" colspan="1">2.38</td><td rowspan="1" colspan="1">2.0E−08</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.97</td><td rowspan="1" colspan="1">−15.95</td><td rowspan="1" colspan="1">7.92</td><td rowspan="1" colspan="1">0.04</td></tr><tr><td rowspan="1" colspan="1">Log_ferritin</td><td rowspan="1" colspan="1">X</td><td rowspan="1" colspan="1"><em>GAB3</em></td><td rowspan="1" colspan="1">rs141555380</td><td rowspan="1" colspan="1">153906012</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">2347</td><td rowspan="1" colspan="1">0.94</td><td rowspan="1" colspan="1">0.17</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">1.1E−08</td><td rowspan="1" colspan="1">0.l3</td><td rowspan="1" colspan="1">329</td><td rowspan="1" colspan="1">0.98</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.0057</td></tr></tbody></table></div><div id="largeobj_idm139826557602752" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4334839/table/DDU454TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN4"><p id="__p14" class="p p-first-last">*: <em>β</em>, SE, and <em>P</em> were reported for rs9872999 after adjusting for rs8177253.</p></div><div id="TN5"><p id="__p15" class="p p-first-last">Chr, chromosome; Pos(hg19), physical position of the SNP according to human genome build version 19; EA, effect allele; EAF, effect allele frequency; <em>β</em>, <em>β</em> coefficients representing the estimated change in the raw or transformed trait value associated with each additional copy of the effect allele; SE, standard error; <em>R</em><sup>2</sup>, <em>R</em><sup>2</sup> represents the imputation quality provided. “NA” indicates that the actual genotype data from the Affy 6.0 array were used in the analyses.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU454F1" co-legend-rid="lgnd_DDU454F1"><a href="/pmc/articles/PMC4334839/figure/DDU454F1/" target="figure" rid-figpopup="DDU454F1" rid-ob="ob-DDU454F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826554423872" class="figure"><a href="/pmc/articles/PMC4334839/figure/DDU454F1/" target="figure" rid-figpopup="DDU454F1" rid-ob="ob-DDU454F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4334839_ddu45401.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu45401.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4334839/bin/ddu45401.jpg"></a></div><div id="largeobj_idm139826554423872" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4334839/figure/DDU454F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU454F1"><div><a class="figpopup" href="/pmc/articles/PMC4334839/figure/DDU454F1/" target="figure" rid-figpopup="DDU454F1" rid-ob="ob-DDU454F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p16">(<strong>A</strong>) Regional plot of the −log<sub>10</sub>(P) values for the SNPs in the <em>TF</em> region for TIBC. The <em>X</em>-axis shows the human genome build 19 coordinates (Mb) and the genes in the region. The <em>Y</em>-axis shows the −log<sub>10</sub> transformed association <em>P</em>-values of SNPs on the left, and recombination rates in cM per Mb on the right. Different colors of shading indicate the strength of linkage disequilibrium (LD) (<em>r</em><sup>2</sup>) between the top SNP and the other SNPs tested in the region. (<strong>B</strong>) Regional plot of the −log<sub>10</sub>(<em>P</em>) values for the SNPs in the <em>TF</em> region for TIBC after adjusting for the top SNP rs8177253 in this region. The top SNP was rs9872999 after adjusting for rs8177253.</p></div></div></div><p id="__p17" class="p p-last">Higher local African ancestry at rs8177253 in the <em>TF</em> region was nominally associated with lower TIBC levels (<em>P</em> = 0.0012). The association between TIBC and rs8177253 remained robust after the adjustment for local African ancestry at rs8177253. When stratified by the estimated local number of European versus African chromosomes, the rs8177253–TIBC association was present among 1579 AA who were predicted to carry two African chromosomes (<em>β</em> = 19.64 ± 1.70; <em>P</em> = 8.5 × 10<sup>−30</sup>) as well as the 768 AA who were predicted to carry at least one European chromosome (<em>β</em> = 19.68 ± 2.26; <em>P</em> = 1.8 × 10<sup>−17</sup>). The rs9872999–TIBC association also remained significant after adjustment for both rs8177253 and local African ancestry at the genotyped marker nearest its location (data not shown).</p></div><div id="s2d" class="sec"><h3 id="s2dtitle">TIBC GWAS results on chromosome 6</h3><p id="__p18" class="p p-first">The three genome-wide significant SNPs on chromosome 6 mapped near the <em>HDGFL1</em> gene (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6</a>). The strongest signal (rs115923437, <em>P</em> = 1.1 × 10<sup>−8</sup>) mapped ∼100&nbsp;Kb distal to <em>HDGFL1</em>, which is a gene that encodes hepatoma-derived growth factor-like 1 and is associated with glycosylated hemoglobin level, and ∼3.5 Mb proximal to the known iron gene <em>HFE</em>. After conditioning on rs115923437, the remaining SNPs within 1 Mb of the SNP no longer showed strong evidence for association (all <em>P</em> &gt; 1 × 10<sup>−4</sup>). SNP rs115923437 explained 1.3% of the total variance of TIBC after accounting for the covariates.</p><p id="__p19" class="p p-last">Local African ancestry near rs115923437 was also nominally associated with TIBC, although this time higher local African ancestry was associated with higher TIBC levels (<em>P</em> = 0.0083). The associations between TIBC and rs115923437 remained significant after the adjustment for local African ancestry at the genotyped marker nearest its location. When stratified by the estimated local number of European versus African chromosomes, the rs115923437–TIBC association was present among the 1569 AA who were predicted to carry two African chromosomes (<em>β</em> = −14.11 ± 2.95; <em>P</em> = 2.0 × 10<sup>−6</sup>), as well as the 778 AA who were predicted to carry at least one European chromosome (<em>β</em> = −15.42 ± 5.78; <em>P</em> = 0.0070).</p></div><div id="s2e" class="sec"><h3 id="s2etitle">TIBC GWAS results on chromosome 16</h3><p id="__p20" class="p p-first-last">A single SNP (rs16951289, <em>P</em> = 2.0 × 10<sup>−8</sup>) on chromosome 16, an intronic variant in uncharacterized gene <em>LOC102467146</em> that is ∼150 Kb distal to the <em>MAF</em> gene (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S7</a>), reached genome-wide significance. <em>MAF</em> encodes v-maf musculoaponeurotic fibrosarcoma oncogene homolog. Local African ancestry in this region was not associated with TIBC (<em>P</em> = 0.97) and did not modify the evidence for association with rs16951289. Rs16951289 explained 1.2% of the total variance of TIBC.</p></div><div id="s2f" class="sec"><h3 id="s2ftitle">Ferritin GWAS results on chromosome X</h3><p id="__p21" class="p p-first-last">Five SNPs on chromosome X near the <em>GAB3</em> gene reached genome-wide significance for association with ferritin levels (Fig.&nbsp;<a href="/pmc/articles/PMC4334839/figure/DDU454F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU454F2" rid-ob="ob-DDU454F2" co-legend-rid="lgnd_DDU454F2"><span>2</span></a>). The strongest signal (rs141555380, <em>P</em> = 1.1 × 10<sup>−8</sup>) mapped to the UTR-3 region of the <em>GAB3</em> gene. <em>GAB3</em> encodes GRB2-associated binding protein 3, which is involved in several growth factor and cytokine signaling pathways. After conditioning on rs141555380, the remaining SNPs within 1 Mb of rs141555380 no longer showed significant evidence for association (all <em>P</em> &gt; 1 × 10<sup>−4</sup>). The effect estimates for carriers of the rs141555380 minor allele in stratified analysis were similar for hemizygous males and homozygous females (hemizygous males: <em>β</em> = 0.17 ± 0.04, <em>P</em> = 5.05 × 10<sup>−6</sup>; homozygous females: <em>β</em> = 0.14 ± 0.05, <em>P</em> = 0.002). SNP rs141555380 explained 1.2% of the total variance of ferritin after accounting for age, gender, BMI and the first 10 PCs.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU454F2" co-legend-rid="lgnd_DDU454F2"><a href="/pmc/articles/PMC4334839/figure/DDU454F2/" target="figure" rid-figpopup="DDU454F2" rid-ob="ob-DDU454F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826561361312" class="figure"><a href="/pmc/articles/PMC4334839/figure/DDU454F2/" target="figure" rid-figpopup="DDU454F2" rid-ob="ob-DDU454F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4334839_ddu45402.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu45402.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4334839/bin/ddu45402.jpg"></a></div><div id="largeobj_idm139826561361312" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4334839/figure/DDU454F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU454F2"><div><a class="figpopup" href="/pmc/articles/PMC4334839/figure/DDU454F2/" target="figure" rid-figpopup="DDU454F2" rid-ob="ob-DDU454F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p22">Regional plot of the −log<sub>10</sub>(<em>P</em>) values for the SNPs in the <em>GAB3</em> region for log_ferritin.</p></div></div></div></div><div id="s2g" class="sec"><h3 id="s2gtitle">Menopause status-adjusted analyses of association with ferritin</h3><p id="__p23" class="p p-first-last">Association analysis for serum ferritin was conducted by including menopausal status as an additional covariate. Menopausal status was significantly associated with serum ferritin levels <em>P</em> = 5.3 × 10<sup>−17</sup>; however, the effect size and significance of SNP-ferritin associations did not change considerably after adjusting for menopausal status. The top SNP rs141555380 (<em>P</em> = 1.1 × 10<sup>−8</sup>) continued to be the most significant SNP after adjusting for menopause status (<em>P</em> = 1.4 × 10<sup>−8</sup>). The effect sizes and <em>P</em>-values of top SNPs (<em>P</em> &lt; 10<sup>−7</sup>) before and after adjusting for menopause status are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S9</a>. This result is consistent with previous findings that the effects of variation in menstrual blood loss, although significant, were small when compared with the genetic effects that influence the iron reserves (<a href="#DDU454C8" rid="DDU454C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981875">8</a>).</p></div><div id="s2h" class="sec"><h3 id="s2htitle">Replication results in HANDLS</h3><p id="__p24" class="p p-first-last">Four regions contained SNPs that reached genome-wide significance in JHS, including one region, the chromosome 3 <em>TF</em> region, which contained a second significant SNP (rs9872999) after covariate adjustment for the top SNP (rs8177253). Five SNPs (rs8177253, rs9872999, chromosome 6 top SNP rs115923437 and chromosome 16 top SNP rs16951289 for TIBC; chromosome X top SNP rs141555380 for ferritin) were selected and tested for association in HANDLS to determine whether the associations could be replicated (Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB3" rid-ob="ob-DDU454TB3" co-legend-rid=""><span>3</span></a>). Rs9872999 was tested before and after adjustment for rs8177253. Both associations in the <em>TF</em> region with TIBC were replicated (<em>P</em> = 1.1 × 10<sup>−7</sup> for rs8177253; <em>P</em> = 0.0012 for rs9872999 before adjusting for rs8177253). As with JHS, the association between rs9872999 and TIBC became more significant after adjusting for the primary signal at rs8177253 (<em>P</em> = 6.2 × 10<sup>−6</sup>). The association between the <em>GAB3</em> region SNP rs141555380 and ferritin also replicated (<em>P</em> = 5.7 × 10<sup>−3</sup>). The associations between TIBC and SNPs rs115923437 and rs16951289 did not replicate in HANDLS; rs16951289 was nominally significant (<em>P</em> = 0.04), but the direction of the effect went in the opposite direction to that observed in JHS.</p></div><div id="s2i" class="sec sec-last"><h3 id="s2ititle">Prior European GWAS-established signals at <em>P</em> &lt; 5 × 10<sup>−8</sup> that are replicated in JHS at <em>P</em> &lt; 0.05</h3><p id="__p25" class="p p-first-last">GWAS have reported five regions to contain SNPs to be significantly associated (<em>P</em> &lt; 5 × 10<sup>−8</sup>) with at least one of the following iron-related traits: iron, ferritin, SAT and transferrin, including <em>TF-SRPRB</em> (rs3811647<em>,</em> rs1830084)<em>, SLC17A1</em> (rs17342717), <em>HFE</em> (rs1799945 [H63D] and rs1800562 [C282Y]), <em>HIST1H2BJ</em> (rs13194491) and <em>TMPRSS6</em> (rs855791 [V736A] and rs4820268) in subjects of European descent (Table&nbsp;<a href="/pmc/articles/PMC4334839/table/DDU454TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU454TB4" rid-ob="ob-DDU454TB4" co-legend-rid=""><span>4</span></a>)<em>.</em> Iron, ferritin and SAT are directly reported in the current study, while TIBC is proportional to transferrin [TIBC (μmol/L) = 25.1 × transferrin (g/L)]. All five regions contained a SNP, either the index SNP in the prior report or a SNP in strong linkage disequilibrium (defined as a SNP with estimated <em>r</em><sup>2</sup> &gt; 0.8 with the index SNP based on CEU 1000 Genomes subjects) with the index SNP, that was nominally associated with at least one corresponding iron measure in JHS at <em>P</em> &lt; 0.05. For <em>HFE</em> rs1800562, only the associations with ferritin and transferrin were replicated, but not the associations with iron and SAT. For <em>HIST1H2BJ</em> rs13194491 and <em>TMPRSS6</em> rs855791, the associations with the index SNPs were not replicated, but nearby SNPs, which are estimated to be in high LD with the index SNPs in European populations, did reach nominal significance in JHS, suggesting a narrowing of the candidate regions for the causal variants if the causal variants are the same for the two populations.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU454TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p26">Replication of signals established in prior GWAS including subjects of European descent</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139826633938144" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Trait</th><th rowspan="2" colspan="1" align="left">Chr</th><th colspan="7" rowspan="1" align="center">Index SNP result in JHS<sup>a</sup><hr></th><th colspan="8" rowspan="1" align="center">Most significant SNP in JHS<sup>b</sup><hr></th><th rowspan="2" colspan="1" align="left">Nearest gene</th></tr><tr><th rowspan="1" colspan="1" align="left">Index SNP</th><th rowspan="1" colspan="1" align="left">Pos(hg19)</th><th rowspan="1" colspan="1" align="left">EA</th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left"><em>β</em></th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">Most significant SNP</th><th rowspan="1" colspan="1" align="left">Pos(hg19)</th><th rowspan="1" colspan="1" align="left">EA</th><th rowspan="1" colspan="1" align="left">EAF</th><th rowspan="1" colspan="1" align="left"><em>β</em></th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em></th><th rowspan="1" colspan="1" align="left">LD <em>r</em><sup>2</sup></th></tr></thead><tbody><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs3811647*(7)</td><td rowspan="1" colspan="1">133484029</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.22</td><td rowspan="1" colspan="1">17.55</td><td rowspan="1" colspan="1">1.43</td><td rowspan="1" colspan="1">9.7E−35</td><td rowspan="1" colspan="1">rs8177253</td><td rowspan="1" colspan="1">133480192</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">19.86</td><td rowspan="1" colspan="1">1.34</td><td rowspan="1" colspan="1">1.8E−47</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>TF</em></td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs1830084(7)</td><td rowspan="1" colspan="1">133508464</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">16.84</td><td rowspan="1" colspan="1">1.60</td><td rowspan="1" colspan="1">2.2E−25</td><td rowspan="1" colspan="1">rs1830084</td><td rowspan="1" colspan="1">133508464</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.16</td><td rowspan="1" colspan="1">16.84</td><td rowspan="1" colspan="1">1.60</td><td rowspan="1" colspan="1">2.2E−25</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>TF</em></td></tr><tr><td rowspan="1" colspan="1">log_ferritin</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs17342717*(4)</td><td rowspan="1" colspan="1">25821770</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.24</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">rs78273613*</td><td rowspan="1" colspan="1">25866075</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.26</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.88</td><td rowspan="1" colspan="1"><em>SLC17A1</em></td></tr><tr><td rowspan="1" colspan="1">log_iron</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs1799945*(5)</td><td rowspan="1" colspan="1">26091179</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">7.8E−04</td><td rowspan="1" colspan="1">rs129128</td><td rowspan="1" colspan="1">26125342</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.09</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">1.0E−04</td><td rowspan="1" colspan="1">0.92</td><td rowspan="1" colspan="1"><em>HFE</em></td></tr><tr><td rowspan="1" colspan="1">log_iron</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs1800562(7)</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">rs1800562</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−0.03</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.37</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>HFE</em></td></tr><tr><td rowspan="1" colspan="1">log_ferritin</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs1800562(4)</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">rs1800562</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.20</td><td rowspan="1" colspan="1">0.10</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>HFE</em></td></tr><tr><td rowspan="1" colspan="1">TIBC</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs1800562(7)</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−24.63</td><td rowspan="1" colspan="1">5.13</td><td rowspan="1" colspan="1">1.7E−06</td><td rowspan="1" colspan="1">rs1800562</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">−24.63</td><td rowspan="1" colspan="1">5.13</td><td rowspan="1" colspan="1">1.7E−06</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>HFE</em></td></tr><tr><td rowspan="1" colspan="1">log_sat</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs1800562(7)</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">rs1800562</td><td rowspan="1" colspan="1">26093141</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.04</td><td rowspan="1" colspan="1">0.14</td><td rowspan="1" colspan="1">1</td><td rowspan="1" colspan="1"><em>HFE</em></td></tr><tr><td rowspan="1" colspan="1">log_sat</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs13194491*(7)</td><td rowspan="1" colspan="1">27037080</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.75</td><td rowspan="1" colspan="1">rs35657082*</td><td rowspan="1" colspan="1">27067657</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.25</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.91</td><td rowspan="1" colspan="1"><em>HIST1H2BJ</em></td></tr><tr><td rowspan="1" colspan="1">log_iron</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs855791*(11)</td><td rowspan="1" colspan="1">37462936</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.08</td><td rowspan="1" colspan="1">rs2072860*</td><td rowspan="1" colspan="1">37470604</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">7.7E−03</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1"><em>TMPRSS6</em></td></tr><tr><td rowspan="1" colspan="1">log_sat</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs855791*(11)</td><td rowspan="1" colspan="1">37462936</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.15</td><td rowspan="1" colspan="1">rs4820268*</td><td rowspan="1" colspan="1">37469591</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.90</td><td rowspan="1" colspan="1"><em>TMPRSS6</em></td></tr><tr><td rowspan="1" colspan="1">log_iron</td><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs4820268*(5)</td><td rowspan="1" colspan="1">37469591</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1">0.73</td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">0.01</td><td rowspan="1" colspan="1">8.4E−03</td><td rowspan="1" colspan="1">rs9610638*</td><td rowspan="1" colspan="1">35775614</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1">0.80</td><td rowspan="1" colspan="1">0.05</td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">3.5E−03</td><td rowspan="1" colspan="1">0.83</td><td rowspan="1" colspan="1"><em>TMPRSS6</em></td></tr></tbody></table></div><div id="largeobj_idm139826633938144" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4334839/table/DDU454TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN6"><p id="__p27" class="p p-first-last">*Imputed.</p></div><div id="TN7"><p id="__p28" class="p p-first-last">() besides “Index SNP” contains the citation to the initial association study for each individual variant.</p></div><div id="TN8"><p id="__p29" class="p p-first-last"><sup>a</sup>Index SNP: Index SNP that was reported to be significantly (<em>P</em> &lt; 5 × 10<sup>−8</sup>) associated with iron traits in prior GWAS in subjects of European descent.</p></div><div id="TN9"><p id="__p30" class="p p-first-last"><sup>b</sup>Most significant SNP: the proxy for index SNP (LD <em>r</em><sup>2</sup> &gt; 0.8 in CEU 1000G with the index SNP) with the smallest association <em>P</em>-value in JHS.</p></div><div id="TN10"><p id="__p31" class="p p-first-last">LD <em>r</em><sup>2</sup>: <em>r</em><sup>2</sup> with index SNP in CEU 1000G subjects.</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p32" class="p p-first">We conducted genome-wide admixture and association studies for iron-related phenotypes, including serum iron, serum ferritin, SAT and TIBC in 2347 AAs participating in the JHS. We observed significant associations between SNPs around <em>TF,</em> a well-established region on chromosome 3, and two novel regions: near <em>HDGFL1</em> on chromosome 6 and <em>MAF</em> on chromosome 16, and TIBC levels. Conditional analyses revealed a second significant SNP associated with TIBC in the <em>TF</em> region that was independent of the top SNP from the unconditional analyses. We also observed significant associations between SNPs around <em>GAB3</em>, a novel region on chromosome X, and ferritin levels. The two independent associations for TIBC at <em>TF</em> and the association for ferritin at <em>GAB3</em> were successfully replicated in HANDLS. The associations between the JHS index SNPs near <em>HDGFL1</em> and <em>MAF</em> were not replicated. The available sample size in HANDLS (<em>N</em> = 329) was considerably smaller than for JHS. If we assume the index variants at <em>HDGFL1</em> and <em>MAF</em> each explain an estimated 1.3% of the total variance of TIBC<em>,</em> then we only had power of 0.34 to observe these associations at <em>P</em> &lt; 0.01 in HANDLS. Thus, larger replication studies will be necessary to make more decisive statements regarding the overall evidence of these other regions. We did not observe any significant SNP associations for the other iron measures, although we were able to replicate associations from previous studies based on subjects of European descent using a less stringent significance threshold (<em>P</em> &lt; 0.05).</p><p id="__p33">The estimated average (“global”) proportion of African ancestry was significantly associated with lower levels of TIBC, serum iron and SAT—which are entirely consistent with previous findings reporting lower levels of these same measures, on average, in AAs compared with European Americans (<a href="#DDU454C9" rid="DDU454C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981887">9</a>,<a href="#DDU454C10" rid="DDU454C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981849">10</a>). Subjects with higher levels of global African ancestry were observed to have higher levels of ferritin, also consistent with prior reported differences between individuals of European and African ancestry (<a href="#DDU454C9" rid="DDU454C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981859">9</a>,<a href="#DDU454C10" rid="DDU454C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981861">10</a>), although this result was not statistically significant. These results implicate novel genetic risk factors in AAs and underscore the importance of studying this population for genetic risk factors that uniquely/disproportionately impact them. Local ancestry was not significantly associated with any iron measures, although a couple of regions containing our GWAS top results were nominally significant.</p><p id="__p34">Variants in and around <em>TF</em> have been observed to be associated with serum ferritin levels (<a href="#DDU454C7" rid="DDU454C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981876">7</a>), transferrin levels (<a href="#DDU454C5" rid="DDU454C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981885">5</a>–<a href="#DDU454C7" rid="DDU454C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981865">7</a>), serum transferrin saturation (<a href="#DDU454C11" rid="DDU454C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981882">11</a>) and serum levels of carbohydrate-deficient transferrin (<a href="#DDU454C12" rid="DDU454C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981889">12</a>) in subjects of European descent. In JHS, the top SNP at <em>TF</em>, rs8177253, was associated with TIBC (<em>P</em> = 1.8 × 10<sup>−47</sup>) and nominally associated with SAT (<em>P</em> = 3.0 × 10<sup>−7</sup>). SNP rs8177253 is located in an intronic region of <em>TF</em> and is in high LD (<em>r</em><sup>2</sup> = 1, <em>D</em>′ = 1 in 1000 Genomes CEU) with GWAS index SNP rs3811647 previously reported to be associated with transferrin in subjects of European and Australian descent (<a href="#DDU454C7" rid="DDU454C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981866">7</a>,<a href="#DDU454C9" rid="DDU454C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981854">9</a>) [rs3811647 is also associated with TIBC (<em>P</em> = 9.7 × 10<sup>−35</sup>) and nominally associated with SAT (<em>P</em> = 2.2 × 10<sup>−6</sup>) in JHS]. After conditioning on rs8177253, all SNPs, including rs3811647, reported to be significantly associated with TIBC in previous studies became non-significant in JHS.</p><p id="__p35">Our study is the first study to report a second significant independently associated SNP in the <em>TF</em> region for TIBC. Interestingly, this second signal (index SNP rs9872999 which maps to an intergenic region ∼10 Kb proximal to <em>TF</em>) only became significant at the genome-wide level after conditioning on our top <em>TF</em> region SNP rs8177253. Rs8177253 and rs9872999 are in modest LD in 1000 Genomes YRI subjects (<em>r</em><sup>2</sup> = 0.07, <em>D</em>′ = 0.58) and in stronger LD in 1000 Genomes CEU subjects (<em>r</em><sup>2</sup> = 0.27, <em>D</em>′ = 0.71). The allele associated with an increase in TIBC for rs8177253 is preferentially on the same haplotype with the allele associated with a decrease in TIBC for rs9872999. Thus, the mean effects for rs9872999 are shrunk toward the null when not factoring in genotype for rs8177253. There is no evidence for an interaction between these SNPs on TIBC levels; thus the results indicate a second independent signal in the <em>TF</em> region. The MAF for rs9872999 is 0.39 in 1000 Genomes YRI subjects and 0.50 in 1000 Genomes CEU subjects. It is unclear if this second signal is specific to AAs, as results from conditional analyses in other populations have not been reported. Conditional analyses would likely be especially important for detecting significant mean effects for rs9872999 in populations of European descent given the stronger LD between the two SNPs in this population.</p><p id="__p36">Variants in <em>HDGFL1</em> have been reported previously to be associated with glycosylated hemoglobin levels (<em>P</em> = 2.4 × 10<sup>−5</sup>) in European type 1 diabetic subjects (<a href="#DDU454C13" rid="DDU454C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981883">13</a>) and nominally associated (<em>P</em> &lt; 0.05) with levels of VLDL, LDL, Apolipoprotein C, HDL and carotid artery disease (<a href="#DDU454C14" rid="DDU454C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981864">14</a>–<a href="#DDU454C16" rid="DDU454C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981857">16</a>). In recent years, there has been considerable interest in the possibility that excessive tissue iron stores may contribute to the pathogenesis of both diabetes and ischemic heart disease (<a href="#DDU454C3" rid="DDU454C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981880">3</a>). <em>MAF</em>, which encodes v-maf musculoaponeurotic fibrosarcoma oncogene homolog, appears to be important in early development. Mutations in <em>MAF</em> have been reported to co-segregate with cerulean congenital cataracts (<a href="#DDU454C17" rid="DDU454C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981870">17</a>) and juvenile-onset pulverulent cataract (<a href="#DDU454C18" rid="DDU454C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981891">18</a>) in human pedigrees. We can find no evidence in the literature suggesting a direct connection between <em>MAF</em> and iron metabolism. However, there is some evidence of pleiotropy for iron metabolism and cataracts, namely hereditary hyperferritinemia cataract syndrome, which is an inherited syndrome caused by a mutation within the L-ferritin gene and characterized by early-onset cataracts and elevated serum ferritin (<a href="#DDU454C19" rid="DDU454C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981856">19</a>).</p><p id="__p37">A cluster of SNPs near <em>GAB3</em> on chromosome X were significantly associated with ferritin (top SNP rs141555380, MAF = 0.14, <em>P</em> = 1.1 × 10<sup>−8</sup>) and nominally associated with SAT (rs141555380, <em>P</em> = 0.037). Although no prior studies observed any connection between this gene and iron metabolism, another gene ∼0.2 Mb upstream of <em>GAB3</em>, <em>G6PD,</em> plays a critical role in iron metabolism. <em>G6PD</em> deficiency may cause acute hemolysis or severe chronic non-spherocytic hemolytic anemia. Increases in serum ferritin levels have been observed in <em>G6PD</em>-deficient patients (<a href="#DDU454C20" rid="DDU454C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981871">20</a>,<a href="#DDU454C21" rid="DDU454C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981848">21</a>), which is possibly due to both a shortened life span and increased break down of erythrocytes in <em>G6PD</em>-deficient patients. A functional missense variant in <em>G6PD</em>, rs1050828 (MAF = 0.13, leading to a Val68Met amino acid substitution), was also associated with ferritin but narrowly missed genome-wide significance (<em>P</em> = 9.1 × 10<sup>−8</sup>). Strong LD exists between this functional variant at <em>G6PD</em> and rs141555380 (<em>R</em><sup>2</sup> = 0.91, <em>D</em>′ = 1 in 1000 Genomes Project participants of African descent), and the association between rs141555380 and ferritin disappears after adjustment for rs1050828 (<em>P</em> = 0.55). Rs1050828 and nearby rs762516 (two SNPs in LD: <em>R</em><sup>2</sup> = 0.68, <em>D</em>′ = 1.0 in HapMap YRI) have been shown to be significantly associated with multiple erythrocyte traits in AAs, including hematocrit, hemoglobin, RBC count, mean corpuscular volume (MCV) and red cell distribution width (RDW) in previous GWAS or candidate gene studies (<a href="#DDU454C22" rid="DDU454C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981879">22</a>,<a href="#DDU454C23" rid="DDU454C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981872">23</a>). In JHS, rs1050828 is significantly associated with MCV (<em>P</em> = 1.7 × 10<sup>−8</sup>), RBC count (<em>P</em> = 9.9 × 10<sup>−16</sup>) and RDW (<em>P</em> = 8.9 × 10<sup>−21</sup>) and nominally associated with hematocrit (<em>P</em> = 8.7 × 10<sup>−7</sup>) and hemoglobin (7.1 × 10<sup>−8</sup>). Ferritin levels are correlated with levels of hematocrit (<em>r</em> = 0.25), hemoglobin (<em>r</em> = 0.27), MCV (<em>r</em> = 0.089), RBC (<em>r</em> = 0.15) and RDW (<em>r</em> = −0.13). The associations for both hematocrit (<em>P</em> = 2.9 × 10<sup>−8</sup>) and hemoglobin (<em>P</em> = 4.3 × 10<sup>−10</sup>) both became genome-wide significant after additional covariate adjustment for ferritin. Similarly, the association between rs1050828 and ferritin also became genome-wide significant after adjustment for hematocrit (7.2 × 10<sup>−9</sup>) and hemoglobin (1.3 × 10<sup>−9</sup>), but evidence for association with ferritin decreased after covariate adjustment for MCV (1.1 × 10<sup>−4</sup>), RBC (5.2 × 10<sup>−6</sup>) and RDW (7.2 × 10<sup>−3</sup>). Since <em>G6DP</em> has been reported to play an important role in hemolysis and affects the levels of erythrocyte traits, the signal we observed at this region may help explain the relationship between hemolysis and iron metabolism. This variant is also implicated in malaria resistance, and the A- form of G6PD deficiency in Africa is under strong natural selection from the preferential protection it provides to hemizygous males and homozygous females against life-threatening malaria (<a href="#DDU454C24" rid="DDU454C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981888">24</a>). This natural selection of G6PD deficiency in African descent may help explain some of the marked differences in iron measures among ethnic groups.</p><p id="__p38" class="p p-last">In summary, we report that global genetic admixture is an important predictor of iron measures in AAs, further implicating the importance of unique genetic effect alleles in the AA population. We observed SNPs in or near three genes, <em>TF</em>, <em>HDGFL1</em> and <em>MAF,</em> which were significantly associated with TIBC in JHS, and SNPs near <em>GAB3</em> that were significantly associated with ferritin. We identified a novel second independently associated SNP in the <em>TF</em> region for TIBC that was only identified after conditioning on the top SNP in the region. The two <em>TF</em> signals and the <em>GAB3</em> signal were replicated in a small independent AA sample from HANDLS. Larger replication samples will be necessary to draw firm conclusions regarding the associations for the other loci. The <em>TF</em> region is known to be associated with various serum iron-related measures in subjects of European descent; we now show similar associations in AA. While the <em>G6DP-GAB3</em> region is known to be associated with multiple erythrocyte traits in AA, this is the first time it has been reported to be significantly associated with ferritin, a specific iron-related measure. We have also nominally replicated four other established loci from other populations in our AA samples. Future fine-mapping studies, including rare and uncommon variants, and functional studies should be undertaken to better characterize these and other loci and to identify the functional variants directly influencing iron levels in AA.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Study subjects</h3><div id="s4a1" class="sec sec-first"><p></p><h4 id="s4a1title" class="inline">Discovery stage </h4><p id="__p39" class="p p-first-last">The JHS is a longitudinal, population-based cohort designed to identify risk factors for the development of cardiovascular disease, diabetes, obesity, chronic kidney disease and stroke in more than 5000 AAs from the Jackson, metropolitan area (<a href="#DDU454C25" rid="DDU454C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981881">25</a>). The design, recruitment and initial characterization of this study have been described previously (<a href="#DDU454C26" rid="DDU454C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981892">26</a>). The JHS participants for the current study included 1012 AA males and 1335 AA females with available iron-related measures and genome-wide genotype data.</p></div><div id="s4a2" class="sec sec-last"><p></p><h4 id="s4a2title" class="inline">Replication stage </h4><p id="__p40" class="p p-first">The HANDLS is a community-based, longitudinal epidemiological study that aims to examine the influences of race and socioeconomic status on the development of age-related diseases in African and European Americans from the city of Baltimore. The study consists of 2200 AAs and 1522 European Americans aged 30–64 years. The design, recruitment and initial characterization of this study have been described previously (<a href="#DDU454C27" rid="DDU454C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981868">27</a>). The HANDLS participants for the current study included 329 AAs with iron measures and genome-wide genotype data.</p><p id="__p41" class="p p-last">JHS and HANDLS participants provided written informed consent. The study protocols and consent forms for these studies were approved by the responsible research ethics committees and institutional research boards.</p></div></div><div id="s4b" class="sec"><h3 id="s4btitle">Phenotypes</h3><p id="__p42" class="p p-first">All phenotype measures came from blood samples that were collected from fasting blood during the baseline examination, which occurred during 2000–2004 for JHS and 2004–2009 for HANDLS. Iron measures included total levels of serum iron (μg/dl), serum ferritin (ng/ml), TIBC (μg/dl) and SAT (%). Serum iron in JHS participants was measured by the FerroZine colorimetric assay (Roche), standardized to NIST traceable iron standards and calibrated against control sera from the manufacturer. TIBC was determined by colorimetric (FerroZine) measurement of iron that remains unbound after addition of a known amount of iron to the serum. The assay was standardized and calibrated as for serum iron. Ferritin was measured by an immunoturbidimetric assay (Roche) based on agglutination of anti-ferritin-latex conjugates, standardized with human spleen ferritin and calibrated against a standard protein solution provided by the manufacturer. For HANDLS, serum iron and TIBC were measured using standard clinical laboratory spectrophotometric assays. Serum ferritin was measured using chemiluminescence assays. For both studies, SAT was calculated as: (serum iron/TIBC)*100%.</p><p id="__p43" class="p p-last">Participants were excluded if they were taking iron supplements, or not fasting at time of blood draw or if they had known chronic infectious or inflammatory disease, or residual cancer. Additional exclusions included hematocrit&nbsp;&lt;35%, hemoglobin &lt;11 g/dl, mean red cell volume &gt;100 fl, white blood cell count &gt;11 000 per mm<sup>3</sup>, platelet count &gt;400 000 per mm<sup>3</sup>, C-reactive protein &gt;3 standard deviations above the mean or transferrin saturation &lt;15% (indicates iron deficiency likely due to blood loss).</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Genotyping and imputation</h3><p id="__p44" class="p p-first">A total of 3030 JHS participants were genotyped using the Affymetrix Genome-Wide Human SNP Array 6.0. 874 712 SNPs, with a call rate &gt;0.95, MAF &gt;0.01 and genotype distributions consistent with the Hardy–Weinberg equilibrium (HWE, <em>P</em> &gt; 1 × 10<sup>−5</sup>) were included for further analysis. Following pre-phasing using MACH 1.0.18 software (<a href="#DDU454C28" rid="DDU454C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981846">28</a>), 38 million SNPs, excluding SNPs monomorphic in CEU/YRI, were imputed using minimac (<a href="#DDU454C29" rid="DDU454C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981878">29</a>) based on 1000 Genome Project phase I reference samples (November 2010, Version 3). Analyses were limited to the ∼17 million imputed SNPs with estimated imputation quality of <em>r</em><sup>2</sup> &gt;0.3.</p><p id="__p45" class="p p-last">A total of 1024 HANDLS participants were genotyped using the Illumina 1 <span class="small-caps">m</span> SNP array, including 329 AAs with iron measures. SNPs with HWE <em>P</em> &gt; 1 × 10<sup>−7</sup>, MAF &gt; 0.01 and call rate &gt; 95% were included for further analysis. 2 939 993 SNPs were imputed using MACH (<a href="#DDU454C28" rid="DDU454C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981858">28</a>) and Minimac (<a href="#DDU454C29" rid="DDU454C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981860">29</a>) software based on combined reference haplotype data from HapMap Phase 2 CEU + YRI samples that includes monomorphic SNPs in either of the two constituent populations (release 22, build 36.3). Chromosome X variants were imputed based on 1000 Genomes Project EUR + AFR + AMR + ASN reference samples. Only index variants demonstrating significant evidence for association in JHS (<em>P</em> &lt; 5 × 10<sup>−8</sup>) were subsequently analyzed for the relevant iron phenotype in HANDLS.</p></div><div id="s4d" class="sec sec-last"><h3 id="s4dtitle">Statistical analyses</h3><p id="__p46" class="p p-first">To assess the impact of genetic admixture on iron measures within the AA population, we first estimated the genome-wide average of African ancestry for each JHS participant (“global ancestry”). We used the software ADMIXTURE (<a href="#DDU454C30" rid="DDU454C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981886">30</a>) with <em>K</em> = 2 clusters and tested, using linear models implemented in R, whether this estimated global ancestry proportion was associated with each of our iron measures after covariate adjustment for age, sex and BMI. Values of serum ferritin, total iron and SAT were natural log-transformed to achieve approximate normality of residuals.</p><p id="__p47">ANCESTRYMAP (<a href="#DDU454C31" rid="DDU454C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981863">31</a>) was used to estimate local ancestry (probabilities of whether an individual has 0, 1 or 2 alleles of European ancestry) at 738 831 autosomal SNPs across the genome, for each participant in JHS, as previously described (<a href="#DDU454C32" rid="DDU454C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981845">32</a>). In brief, local ancestry was inferred using a hidden Markov model based on the genotypes from a panel of densely spaced markers differentiated in frequency between African and European populations. To assess whether there were any regions where local ancestry was associated with iron-related measures, we performed admixture mapping across the whole genome by regressing each of our iron measures on the local ancestry estimates at each SNP location, including covariate adjustment for age, sex, BMI and estimated global ancestry. The conventionally reported LOD score, defined as the log, base 10, ratio of the maximum likelihood of the data under a local-ancestry-associated model divided by the likelihood of the data under the null model (with no local ancestry predictor), was computed at each SNP location. For regions showing association of increased African ancestry with higher levels of iron measures, the LOD scores were assigned positive values, and for regions showing association of increased African ancestry with lower level of iron measures, the LOD scores were assigned negative values. The LOD scores were plotted across the genome, and a LOD score of 5 was assumed to be the threshold of statistical significance (<a href="#DDU454C31" rid="DDU454C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981850">31</a>).</p><p id="__p48">Individual genotyped and imputed SNPs were tested for association using multivariable linear regression models in PLINK (<a href="#DDU454C33" rid="DDU454C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981847">33</a>) and MACH2QTL v.1.08 (<a href="#DDU454C28" rid="DDU454C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981851">28</a>), respectively; adjusted for age, sex, BMI and 10 principal components that we constructed using the software EIGENSOFT (<a href="#DDU454C34" rid="DDU454C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981884">34</a>) to model background ancestry. A second level of covariate adjustment for log-ferritin additionally included self-reported menopausal status, which was available on a large subset of JHS subjects and has been shown previously to be a strong predictor of serum ferritin levels (<a href="#DDU454C8" rid="DDU454C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981853">8</a>,<a href="#DDU454C35" rid="DDU454C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981855">35</a>). We assumed an additive mode of inheritance and reported <em>β</em> coefficients representing the estimated change in the raw or transformed trait value associated with each additional copy of the effect allele. For chromosome X SNPs, hemizygous males were modeled so that males with the minor allele had the same value as females homozygous for the minor allele. We used a significance threshold of <em>P</em> = 5 × 10<sup>−8</sup> to maintain an overall type 1 error rate of ∼5% for each phenotype.</p><p id="__p49">Manhattan plots were made to illustrate the association results across the genome and Q–Q plots were made to assess any systematic inflation of the regression test statistics across the genome. In regions demonstrating significant evidence for association, we examined multivariable regression models that included the genotype data of the most strongly associated SNP as a covariate to assess whether there was any evidence for multiple independently associated SNPs in a particular region. If a second signal also reached genome-wide significance after conditioning on the top variant, multivariable regression models were repeated to include the genotypes of both SNPs as covariates. The relevant SNP–SNP interactions were also tested. Region-specific (“locus-zoom”) plots were made to show the magnitude of association between all SNPs and the relevant iron phenotype as well as the LD between each SNP in the region and the most strongly associated SNP (<a href="#DDU454C36" rid="DDU454C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981873">36</a>). Finally, to control for possible confounding between SNP genotype and local ancestry in any observed iron trait-SNP associations, we identified the genetic position of the most strongly associated SNP, selected the local ancestry estimate at the location closest to that SNP (either the SNP itself if genotyped or the closest genotyped SNP) and examined multivariable regression models as described above, but now including estimated local ancestry proportion as an additional covariate.</p><p id="__p50">SNPs that reached genome-wide significance in JHS were selected for testing in the replication study of HANDLS. At each associated region, only the single SNP with the most significant <em>P</em>-value (index SNP) was selected to avoid over adjustment for multiple testing. For the <em>TF</em> region, where conditional analyses revealed a second independently associated SNP with TIBC, the second independent SNP was also included in the replication analyses. A GWAS result was considered replicated if the effect in the replication was in the same direction as in the discovery stage, and if the association in the replication stage was statistically significant after Bonferroni correction adjusting for the number of SNPs tested.</p><p id="__p51" class="p p-last">Ethnic differences in iron-related measures have also been observed between subjects of European and African descent (<a href="#DDU454C9" rid="DDU454C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_427981877">9</a>). Thus, we compared our association results in JHS with established variants from GWASs in populations of European descent in order to assess the importance of these same variants in an AA population. For each prior GWAS-established SNP, we identified and tested all genotyped or imputed proxy SNPs in JHS that were estimated to be in high LD (<em>r</em><sup>2</sup> &gt; 0.8 in CEU based on 1000 Genomes data) with the GWAS index SNPs for association with the reported iron phenotype.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p52" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu454/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p53" class="p p-first-last">JHS is supported by the National Heart, Lung, and Blood Institute and the National Institute on Minority Health and Health Disparities, with additional support from the National Institute on Biomedical Imaging and Bioengineering (grant numbers <span id="cn1">HHSN268201300046C, HHSN268201300047C, HHSN268201300048C, HHSN268201300049C, HHSN268201300050C</span>). HANDLS is supported by the Intramural Research Program of the National Institute of Health, National Institute on Aging and the National Center on Minority Health and Health Disparities (project # Z01-AG000513 and human subjects protocol # 2009-149). E.M.L., Y.L. and Q.D. are partially supported by R01HG006703.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139826563534800"><a href="/pmc/articles/PMC4334839/bin/supp_24_2_572__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm139826594385840" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139826594385840title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p54">The authors express their gratitude to JHS and HANDLS participants who made this research possible.</p><p id="__p55"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm139826557194784" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139826557194784title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDU454C1">1. <span class="element-citation">Andrews N.C., Schmidt P.J. Iron homeostasis. <span><span class="ref-journal">Annu. Rev. Physiol. </span>2007;<span class="ref-vol">69</span>:69–85.</span> [<a href="/pubmed/17014365" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Physiol.&amp;title=Iron+homeostasis&amp;author=N.C.+Andrews&amp;author=P.J.+Schmidt&amp;volume=69&amp;publication_year=2007&amp;pages=69-85&amp;pmid=17014365&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C2">2. <span class="element-citation">Njajou O.T., Alizadeh B.Z., Aulchenko Y., Zillikens M.C., Pols H.A., Oostra B.A., Swinkels D.W., van Duijn C.M. Heritability of serum iron, ferritin and transferrin saturation in a genetically isolated population, the Erasmus Rucphen Family (ERF) Study. <span><span class="ref-journal">Hum. Hered. </span>2006;<span class="ref-vol">61</span>:222–228.</span> [<a href="/pubmed/16877869" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=Heritability+of+serum+iron,+ferritin+and+transferrin+saturation+in+a+genetically+isolated+population,+the+Erasmus+Rucphen+Family+(ERF)+Study&amp;author=O.T.+Njajou&amp;author=B.Z.+Alizadeh&amp;author=Y.+Aulchenko&amp;author=M.C.+Zillikens&amp;author=H.A.+Pols&amp;volume=61&amp;publication_year=2006&amp;pages=222-228&amp;pmid=16877869&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C3">3. <span class="element-citation">Wilson J.G., Lindquist J.H., Grambow S.C., Crook E.D., Maher J.F. Potential role of increased iron stores in diabetes. <span><span class="ref-journal">Am. J. Med. Sci. </span>2003;<span class="ref-vol">325</span>:332–339.</span> [<a href="/pubmed/12811229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Sci.&amp;title=Potential+role+of+increased+iron+stores+in+diabetes&amp;author=J.G.+Wilson&amp;author=J.H.+Lindquist&amp;author=S.C.+Grambow&amp;author=E.D.+Crook&amp;author=J.F.+Maher&amp;volume=325&amp;publication_year=2003&amp;pages=332-339&amp;pmid=12811229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C4">4. <span class="element-citation">Oexle K., Ried J.S., Hicks A.A., Tanaka T., Hayward C., Bruegel M., Gogele M., Lichtner P., Muller-Myhsok B., Doring A., et al.  Novel association to the proprotein convertase PCSK7 gene locus revealed by analysing soluble transferrin receptor (sTfR) levels. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:1042–1047.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3033185/">PMC free article</a>]</span> [<a href="/pubmed/21149283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Novel+association+to+the+proprotein+convertase+PCSK7+gene+locus+revealed+by+analysing+soluble+transferrin+receptor+(sTfR)+levels&amp;author=K.+Oexle&amp;author=J.S.+Ried&amp;author=A.A.+Hicks&amp;author=T.+Tanaka&amp;author=C.+Hayward&amp;volume=20&amp;publication_year=2011&amp;pages=1042-1047&amp;pmid=21149283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C5">5. <span class="element-citation">Pichler I., Minelli C., Sanna S., Tanaka T., Schwienbacher C., Naitza S., Porcu E., Pattaro C., Busonero F., Zanon A., et al.  Identification of a common variant in the TFR2 gene implicated in the physiological regulation of serum iron levels. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:1232–1240.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3043660/">PMC free article</a>]</span> [<a href="/pubmed/21208937" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Identification+of+a+common+variant+in+the+TFR2+gene+implicated+in+the+physiological+regulation+of+serum+iron+levels&amp;author=I.+Pichler&amp;author=C.+Minelli&amp;author=S.+Sanna&amp;author=T.+Tanaka&amp;author=C.+Schwienbacher&amp;volume=20&amp;publication_year=2011&amp;pages=1232-1240&amp;pmid=21208937&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C6">6. <span class="element-citation">Traglia M., Girelli D., Biino G., Campostrini N., Corbella M., Sala C., Masciullo C., Vigano F., Buetti I., Pistis G., et al.  Association of HFE and TMPRSS6 genetic variants with iron and erythrocyte parameters is only in part dependent on serum hepcidin concentrations. <span><span class="ref-journal">J. Med. Genet. </span>2011;<span class="ref-vol">48</span>:629–634.</span> [<a href="/pubmed/21785125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Genet.&amp;title=Association+of+HFE+and+TMPRSS6+genetic+variants+with+iron+and+erythrocyte+parameters+is+only+in+part+dependent+on+serum+hepcidin+concentrations&amp;author=M.+Traglia&amp;author=D.+Girelli&amp;author=G.+Biino&amp;author=N.+Campostrini&amp;author=M.+Corbella&amp;volume=48&amp;publication_year=2011&amp;pages=629-634&amp;pmid=21785125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C7">7. <span class="element-citation">Benyamin B., McRae A.F., Zhu G., Gordon S., Henders A.K., Palotie A., Peltonen L., Martin N.G., Montgomery G.W., Whitfield J.B., et al.  Variants in TF and HFE explain approximately 40% of genetic variation in serum-transferrin levels. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">84</span>:60–65.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668053/">PMC free article</a>]</span> [<a href="/pubmed/19084217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Variants+in+TF+and+HFE+explain+approximately+40%+of+genetic+variation+in+serum-transferrin+levels&amp;author=B.+Benyamin&amp;author=A.F.+McRae&amp;author=G.+Zhu&amp;author=S.+Gordon&amp;author=A.K.+Henders&amp;volume=84&amp;publication_year=2009&amp;pages=60-65&amp;pmid=19084217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C8">8. <span class="element-citation">Whitfield J.B., Treloar S., Zhu G., Powell L.W., Martin N.G. Relative importance of female-specific and non-female-specific effects on variation in iron stores between women. <span><span class="ref-journal">Br. J. Haematol. </span>2003;<span class="ref-vol">120</span>:860–866.</span> [<a href="/pubmed/12614223" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Haematol.&amp;title=Relative+importance+of+female-specific+and+non-female-specific+effects+on+variation+in+iron+stores+between+women&amp;author=J.B.+Whitfield&amp;author=S.+Treloar&amp;author=G.+Zhu&amp;author=L.W.+Powell&amp;author=N.G.+Martin&amp;volume=120&amp;publication_year=2003&amp;pages=860-866&amp;pmid=12614223&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C9">9. <span class="element-citation">McLaren C.E., McLachlan S., Garner C.P., Vulpe C.D., Gordeuk V.R., Eckfeldt J.H., Adams P.C., Acton R.T., Murray J.A., Leiendecker-Foster C., et al.  Associations between single nucleotide polymorphisms in iron-related genes and iron status in multiethnic populations. <span><span class="ref-journal">PLoS ONE. </span>2012;<span class="ref-vol">7</span>:e38339.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3382217/">PMC free article</a>]</span> [<a href="/pubmed/22761678" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Associations+between+single+nucleotide+polymorphisms+in+iron-related+genes+and+iron+status+in+multiethnic+populations&amp;author=C.E.+McLaren&amp;author=S.+McLachlan&amp;author=C.P.+Garner&amp;author=C.D.+Vulpe&amp;author=V.R.+Gordeuk&amp;volume=7&amp;publication_year=2012&amp;pages=e38339&amp;pmid=22761678&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C10">10. <span class="element-citation">Beutler E., West C. Hematologic differences between African-Americans and whites: the roles of iron deficiency and α-thalassemia on hemoglobin levels and mean corpuscular volume. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:740–745.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1895180/">PMC free article</a>]</span> [<a href="/pubmed/15790781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Hematologic+differences+between+African-Americans+and+whites:+the+roles+of+iron+deficiency+and+α-thalassemia+on+hemoglobin+levels+and+mean+corpuscular+volume&amp;author=E.+Beutler&amp;author=C.+West&amp;volume=106&amp;publication_year=2005&amp;pages=740-745&amp;pmid=15790781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C11">11. <span class="element-citation">Benyamin B., Ferreira M.A., Willemsen G., Gordon S., Middelberg R.P., McEvoy B.P., Hottenga J.J., Henders A.K., Campbell M.J., Wallace L., et al.  Common variants in TMPRSS6 are associated with iron status and erythrocyte volume. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1173–1175.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3135421/">PMC free article</a>]</span> [<a href="/pubmed/19820699" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+variants+in+TMPRSS6+are+associated+with+iron+status+and+erythrocyte+volume&amp;author=B.+Benyamin&amp;author=M.A.+Ferreira&amp;author=G.+Willemsen&amp;author=S.+Gordon&amp;author=R.P.+Middelberg&amp;volume=41&amp;publication_year=2009&amp;pages=1173-1175&amp;pmid=19820699&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C12">12. <span class="element-citation">Kutalik Z., Benyamin B., Bergmann S., Mooser V., Waeber G., Montgomery G.W., Martin N.G., Madden P.A., Heath A.C., Beckmann J.S., et al.  Genome-wide association study identifies two loci strongly affecting transferrin glycosylation. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:3710–3717.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3159549/">PMC free article</a>]</span> [<a href="/pubmed/21665994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+identifies+two+loci+strongly+affecting+transferrin+glycosylation&amp;author=Z.+Kutalik&amp;author=B.+Benyamin&amp;author=S.+Bergmann&amp;author=V.+Mooser&amp;author=G.+Waeber&amp;volume=20&amp;publication_year=2011&amp;pages=3710-3717&amp;pmid=21665994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C13">13. <span class="element-citation">Paterson A.D., Waggott D., Boright A.P., Hosseini S.M., Shen E., Sylvestre M.P., Wong I., Bharaj B., Cleary P.A., Lachin J.M., et al.  A genome-wide association study identifies a novel major locus for glycemic control in type 1 diabetes, as measured by both A1C and glucose. <span><span class="ref-journal">Diabetes. </span>2010;<span class="ref-vol">59</span>:539–549.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2809960/">PMC free article</a>]</span> [<a href="/pubmed/19875614" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=A+genome-wide+association+study+identifies+a+novel+major+locus+for+glycemic+control+in+type+1+diabetes,+as+measured+by+both+A1C+and+glucose&amp;author=A.D.+Paterson&amp;author=D.+Waggott&amp;author=A.P.+Boright&amp;author=S.M.+Hosseini&amp;author=E.+Shen&amp;volume=59&amp;publication_year=2010&amp;pages=539-549&amp;pmid=19875614&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C14">14. <span class="element-citation">Yang Q., Kathiresan S., Lin J.P., Tofler G.H., O'Donnell C.J. Genome-wide association and linkage analyses of hemostatic factors and hematological phenotypes in the Framingham Heart Study. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>(Suppl. 1):S12.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995619/">PMC free article</a>]</span> [<a href="/pubmed/17903294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=Genome-wide+association+and+linkage+analyses+of+hemostatic+factors+and+hematological+phenotypes+in+the+Framingham+Heart+Study&amp;author=Q.+Yang&amp;author=S.+Kathiresan&amp;author=J.P.+Lin&amp;author=G.H.+Tofler&amp;author=C.J.+O'Donnell&amp;volume=8&amp;issue=Suppl.+1&amp;publication_year=2007&amp;pages=S12&amp;pmid=17903294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C15">15. <span class="element-citation">O'Donnell C.J., Cupples L.A., D'Agostino R.B., Fox C.S., Hoffmann U., Hwang S.J., Ingellson E., Liu C., Murabito J.M., Polak J.F., et al.  Genome-wide association study for subclinical atherosclerosis in major arterial territories in the NHLBI's Framingham Heart Study. <span><span class="ref-journal">BMC Med. Genet. </span>2007;<span class="ref-vol">8</span>(Suppl. 1):S4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1995605/">PMC free article</a>]</span> [<a href="/pubmed/17903303" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med.+Genet.&amp;title=Genome-wide+association+study+for+subclinical+atherosclerosis+in+major+arterial+territories+in+the+NHLBI's+Framingham+Heart+Study&amp;author=C.J.+O'Donnell&amp;author=L.A.+Cupples&amp;author=R.B.+D'Agostino&amp;author=C.S.+Fox&amp;author=U.+Hoffmann&amp;volume=8&amp;issue=Suppl.+1&amp;publication_year=2007&amp;pages=S4&amp;pmid=17903303&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C16">16. <span class="element-citation">Lowe J.K., Maller J.B., Pe'er I., Neale B.M., Salit J., Kenny E.E., Shea J.L., Burkhardt R., Smith J.G., Ji W., et al.  Genome-wide association studies in an isolated founder population from the Pacific Island of Kosrae. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000365.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2628735/">PMC free article</a>]</span> [<a href="/pubmed/19197348" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genome-wide+association+studies+in+an+isolated+founder+population+from+the+Pacific+Island+of+Kosrae&amp;author=J.K.+Lowe&amp;author=J.B.+Maller&amp;author=I.+Pe'er&amp;author=B.M.+Neale&amp;author=J.+Salit&amp;volume=5&amp;publication_year=2009&amp;pages=e1000365&amp;pmid=19197348&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C17">17. <span class="element-citation">Vanita V., Singh D., Robinson P.N., Sperling K., Singh J.R. A novel mutation in the DNA-binding domain of MAF at 16q23.1 associated with autosomal dominant "cerulean cataract" in an Indian family. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2006;<span class="ref-vol">140</span>:558–566.</span> [<a href="/pubmed/16470690" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=A+novel+mutation+in+the+DNA-binding+domain+of+MAF+at+16q23.1+associated+with+autosomal+dominant+&quot;cerulean+cataract&quot;+in+an+Indian+family&amp;author=V.+Vanita&amp;author=D.+Singh&amp;author=P.N.+Robinson&amp;author=K.+Sperling&amp;author=J.R.+Singh&amp;volume=140&amp;publication_year=2006&amp;pages=558-566&amp;pmid=16470690&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C18">18. <span class="element-citation">Jamieson R.V., Perveen R., Kerr B., Carette M., Yardley J., Heon E., Wirth M.G., van Heyningen V., Donnai D., Munier F., et al.  Domain disruption and mutation of the bZIP transcription factor, MAF, associated with cataract, ocular anterior segment dysgenesis and coloboma. <span><span class="ref-journal">Hum. Mol. Genet. </span>2002;<span class="ref-vol">11</span>:33–42.</span> [<a href="/pubmed/11772997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Domain+disruption+and+mutation+of+the+bZIP+transcription+factor,+MAF,+associated+with+cataract,+ocular+anterior+segment+dysgenesis+and+coloboma&amp;author=R.V.+Jamieson&amp;author=R.+Perveen&amp;author=B.+Kerr&amp;author=M.+Carette&amp;author=J.+Yardley&amp;volume=11&amp;publication_year=2002&amp;pages=33-42&amp;pmid=11772997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C19">19. <span class="element-citation">Christiansen G., Mohney B.G. Hereditary hyperferritinemia-cataract syndrome. <span><span class="ref-journal">J. AAPOS. </span>2007;<span class="ref-vol">11</span>:294–296.</span> [<a href="/pubmed/17572344" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+AAPOS&amp;title=Hereditary+hyperferritinemia-cataract+syndrome&amp;author=G.+Christiansen&amp;author=B.G.+Mohney&amp;volume=11&amp;publication_year=2007&amp;pages=294-296&amp;pmid=17572344&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C20">20. <span class="element-citation">Suga Y., Nagita A., Takesako R., Tanaka I., Kobayashi K., Hirai M., Matsuoka H. A new glucose-6-phosphate dehydrogenase deficiency variant, G6PD Mizushima, showing increases in serum ferritin and cytosol leucine aminopeptidase levels. <span><span class="ref-journal">J. Pediatr. Hematol. Oncol. </span>2011;<span class="ref-vol">33</span>:15–17.</span> [<a href="/pubmed/21063220" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Pediatr.+Hematol.+Oncol.&amp;title=A+new+glucose-6-phosphate+dehydrogenase+deficiency+variant,+G6PD+Mizushima,+showing+increases+in+serum+ferritin+and+cytosol+leucine+aminopeptidase+levels&amp;author=Y.+Suga&amp;author=A.+Nagita&amp;author=R.+Takesako&amp;author=I.+Tanaka&amp;author=K.+Kobayashi&amp;volume=33&amp;publication_year=2011&amp;pages=15-17&amp;pmid=21063220&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C21">21. <span class="element-citation">Wong C.T., Saha N. Haemoglobin, serum iron, transferrin, ferritin concentrations and total iron-binding capacity in erythrocyte glucose-6-phosphate dehydrogenase deficiency. <span><span class="ref-journal">Trop. Geogr. Med. </span>1987;<span class="ref-vol">39</span>:350–353.</span> [<a href="/pubmed/3451411" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trop.+Geogr.+Med.&amp;title=Haemoglobin,+serum+iron,+transferrin,+ferritin+concentrations+and+total+iron-binding+capacity+in+erythrocyte+glucose-6-phosphate+dehydrogenase+deficiency&amp;author=C.T.+Wong&amp;author=N.+Saha&amp;volume=39&amp;publication_year=1987&amp;pages=350-353&amp;pmid=3451411&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C22">22. <span class="element-citation">Chen Z., Tang H., Qayyum R., Schick U.M., Nalls M.A., Handsaker R., Li J., Lu Y., Yanek L.R., Keating B., et al.  Genome-wide association analysis of red blood cell traits in African Americans: the COGENT Network. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:2529–2538.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3658166/">PMC free article</a>]</span> [<a href="/pubmed/23446634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+analysis+of+red+blood+cell+traits+in+African+Americans:+the+COGENT+Network&amp;author=Z.+Chen&amp;author=H.+Tang&amp;author=R.+Qayyum&amp;author=U.M.+Schick&amp;author=M.A.+Nalls&amp;volume=22&amp;publication_year=2013&amp;pages=2529-2538&amp;pmid=23446634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C23">23. <span class="element-citation">Lo K.S., Wilson J.G., Lange L.A., Folsom A.R., Galarneau G., Ganesh S.K., Grant S.F., Keating B.J., McCarroll S.A., Mohler E.R., 3rd, et al.  Genetic association analysis highlights new loci that modulate hematological trait variation in Caucasians and African Americans. <span><span class="ref-journal">Hum. Genet. </span>2011;<span class="ref-vol">129</span>:307–317.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3442357/">PMC free article</a>]</span> [<a href="/pubmed/21153663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=Genetic+association+analysis+highlights+new+loci+that+modulate+hematological+trait+variation+in+Caucasians+and+African+Americans&amp;author=K.S.+Lo&amp;author=J.G.+Wilson&amp;author=L.A.+Lange&amp;author=A.R.+Folsom&amp;author=G.+Galarneau&amp;volume=129&amp;publication_year=2011&amp;pages=307-317&amp;pmid=21153663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C24">24. <span class="element-citation">Guindo A., Fairhurst R.M., Doumbo O.K., Wellems T.E., Diallo D.A. X-linked G6PD deficiency protects hemizygous males but not heterozygous females against severe malaria. <span><span class="ref-journal">PLoS Med. </span>2007;<span class="ref-vol">4</span>:e66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1820604/">PMC free article</a>]</span> [<a href="/pubmed/17355169" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Med.&amp;title=X-linked+G6PD+deficiency+protects+hemizygous+males+but+not+heterozygous+females+against+severe+malaria&amp;author=A.+Guindo&amp;author=R.M.+Fairhurst&amp;author=O.K.+Doumbo&amp;author=T.E.+Wellems&amp;author=D.A.+Diallo&amp;volume=4&amp;publication_year=2007&amp;pages=e66&amp;pmid=17355169&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C25">25. <span class="element-citation">Taylor H.A., Jr The Jackson Heart Study: an overview. <span><span class="ref-journal">Ethn. Dis. </span>2005;<span class="ref-vol">15</span></span> S6-1– S6-3. [<a href="/pubmed/16317981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ethn.+Dis.&amp;title=The+Jackson+Heart+Study:+an+overview&amp;author=H.A.+Taylor&amp;volume=15&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C26">26. <span class="element-citation">Wilson J.G., Rotimi C.N., Ekunwe L., Royal C.D., Crump M.E., Wyatt S.B., Steffes M.W., Adeyemo A., Zhou J., Taylor H.A., Jr., et al.  Study design for genetic analysis in the Jackson Heart Study. <span><span class="ref-journal">Ethn. Dis. </span>2005;<span class="ref-vol">15</span></span> S6-30– S6-37. [<a href="/pubmed/16317983" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ethn.+Dis.&amp;title=Study+design+for+genetic+analysis+in+the+Jackson+Heart+Study&amp;author=J.G.+Wilson&amp;author=C.N.+Rotimi&amp;author=L.+Ekunwe&amp;author=C.D.+Royal&amp;author=M.E.+Crump&amp;volume=15&amp;publication_year=2005&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C27">27. <span class="element-citation">Evans M.K., Lepkowski J.M., Powe N.R., LaVeist T., Kuczmarski M.F., Zonderman A.B. Healthy aging in neighborhoods of diversity across the life span (HANDLS): overcoming barriers to implementing a longitudinal, epidemiologic, urban study of health, race, and socioeconomic status. <span><span class="ref-journal">Ethn. Dis. </span>2010;<span class="ref-vol">20</span>:267–275.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3040595/">PMC free article</a>]</span> [<a href="/pubmed/20828101" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ethn.+Dis.&amp;title=Healthy+aging+in+neighborhoods+of+diversity+across+the+life+span+(HANDLS):+overcoming+barriers+to+implementing+a+longitudinal,+epidemiologic,+urban+study+of+health,+race,+and+socioeconomic+status&amp;author=M.K.+Evans&amp;author=J.M.+Lepkowski&amp;author=N.R.+Powe&amp;author=T.+LaVeist&amp;author=M.F.+Kuczmarski&amp;volume=20&amp;publication_year=2010&amp;pages=267-275&amp;pmid=20828101&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C28">28. <span class="element-citation">Li Y., Willer C.J., Ding J., Scheet P., Abecasis G.R. MaCH: using sequence and genotype data to estimate haplotypes and unobserved genotypes. <span><span class="ref-journal">Genet. Epidemiol. </span>2010;<span class="ref-vol">34</span>:816–834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175618/">PMC free article</a>]</span> [<a href="/pubmed/21058334" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=MaCH:+using+sequence+and+genotype+data+to+estimate+haplotypes+and+unobserved+genotypes&amp;author=Y.+Li&amp;author=C.J.+Willer&amp;author=J.+Ding&amp;author=P.+Scheet&amp;author=G.R.+Abecasis&amp;volume=34&amp;publication_year=2010&amp;pages=816-834&amp;pmid=21058334&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C29">29. <span class="element-citation">Howie B., Fuchsberger C., Stephens M., Marchini J., Abecasis G.R. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B.+Howie&amp;author=C.+Fuchsberger&amp;author=M.+Stephens&amp;author=J.+Marchini&amp;author=G.R.+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C30">30. <span class="element-citation">Alexander D.H., Novembre J., Lange K. Fast model-based estimation of ancestry in unrelated individuals. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:1655–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;author=D.H.+Alexander&amp;author=J.+Novembre&amp;author=K.+Lange&amp;volume=19&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C31">31. <span class="element-citation">Patterson N., Hattangadi N., Lane B., Lohmueller K.E., Hafler D.A., Oksenberg J.R., Hauser S.L., Smith M.W., O'Brien S.J., Altshuler D., et al.  Methods for high-density admixture mapping of disease genes. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2004;<span class="ref-vol">74</span>:979–1000.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1181990/">PMC free article</a>]</span> [<a href="/pubmed/15088269" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Methods+for+high-density+admixture+mapping+of+disease+genes&amp;author=N.+Patterson&amp;author=N.+Hattangadi&amp;author=B.+Lane&amp;author=K.E.+Lohmueller&amp;author=D.A.+Hafler&amp;volume=74&amp;publication_year=2004&amp;pages=979-1000&amp;pmid=15088269&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C32">32. <span class="element-citation">Deo R.C., Reich D., Tandon A., Akylbekova E., Patterson N., Waliszewska A., Kathiresan S., Sarpong D., Taylor H.A., Jr, Wilson J.G. Genetic differences between the determinants of lipid profile phenotypes in African and European Americans: the Jackson Heart Study. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000342.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2613537/">PMC free article</a>]</span> [<a href="/pubmed/19148283" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Genetic+differences+between+the+determinants+of+lipid+profile+phenotypes+in+African+and+European+Americans:+the+Jackson+Heart+Study&amp;author=R.C.+Deo&amp;author=D.+Reich&amp;author=A.+Tandon&amp;author=E.+Akylbekova&amp;author=N.+Patterson&amp;volume=5&amp;publication_year=2009&amp;pages=e1000342&amp;pmid=19148283&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C33">33. <span class="element-citation">Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D., Maller J., Sklar P., de Bakker P.I., Daly M.J., et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S.+Purcell&amp;author=B.+Neale&amp;author=K.+Todd-Brown&amp;author=L.+Thomas&amp;author=M.A.+Ferreira&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C34">34. <span class="element-citation">Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=A.L.+Price&amp;author=N.J.+Patterson&amp;author=R.M.+Plenge&amp;author=M.E.+Weinblatt&amp;author=N.A.+Shadick&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C35">35. <span class="element-citation">Cade J.E., Moreton J.A., O'Hara B., Greenwood D.C., Moor J., Burley V.J., Kukalizch K., Bishop D.T., Worwood M. Diet and genetic factors associated with iron status in middle-aged women. <span><span class="ref-journal">Am. J. Clin. Nutr. </span>2005;<span class="ref-vol">82</span>:813–820.</span> [<a href="/pubmed/16210711" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Clin.+Nutr.&amp;title=Diet+and+genetic+factors+associated+with+iron+status+in+middle-aged+women&amp;author=J.E.+Cade&amp;author=J.A.+Moreton&amp;author=B.+O'Hara&amp;author=D.C.+Greenwood&amp;author=J.+Moor&amp;volume=82&amp;publication_year=2005&amp;pages=813-820&amp;pmid=16210711&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU454C36">36. <span class="element-citation">Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P., Boehnke M., Abecasis G.R., Willer C.J. LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=R.J.+Pruim&amp;author=R.P.+Welch&amp;author=S.+Sanna&amp;author=T.M.+Teslovich&amp;author=P.S.+Chines&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4334839&amp;issue-id=246880&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139826559335264title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm139826594385840title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm139826557194784title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4334839/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4334839/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4334839/pdf/ddu454.pdf">PDF (281K)</a></li> | <li><a href="#" data-citationid="PMC4334839" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4334839%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4334839%2F&amp;text=Genome-wide%20admixture%20and%20association%20study%20of%20serum%20iron%2C%20ferritin%2C%20transferrin%20saturation%20and%20total%20iron%20binding%20capacity%20in%20African%20Americans" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4334839%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25224454" data-aiid="4334839" data-aid="4334839" data-iid="246880" data-domainid="740" data-domain="hmg" data-accid="PMC4334839" data-md5="42b9126c664eb6d020a88758f063b1aa"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:42:19-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal105&amp;ncbi_phid=CE8C5CD9DD576D510000000002CE01B1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4334839%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4334839/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal105 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>