<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4501476/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies five susceptibility loci for glioma</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4501476/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature genetics"><meta name="citation_title" content="Genome-wide association study identifies five susceptibility loci for glioma"><meta name="citation_authors" content="Sanjay Shete, Fay J Hosking, Lindsay B Robertson, Sara E Dobbins, Marc Sanson, Beatrice Malmer, Matthias Simon, Yannick Marie, Blandine Boisselier, Jean-Yves Delattre, Khe Hoang-Xuan, Soufiane El Hallani, Ahmed Idbaih, Diana Zelenika, Ulrika Andersson, Roger Henriksson, A Tommy Bergenheim, Maria Feychting, Stefan Lönn, Anders Ahlbom, Johannes Schramm, Michael Linnebank, Kari Hemminki, Rajiv Kumar, Sarah J Hepworth, Amy Price, Georgina Armstrong, Yanhong Liu, Xiangjun Gu, Robert Yu, Ching Lau, Minouk Schoemaker, Kenneth Muir, Anthony Swerdlow, Mark Lathrop, Melissa Bondy, Richard S Houlston"><meta name="citation_date" content="August 2009"><meta name="citation_issue" content="8"><meta name="citation_volume" content="41"><meta name="citation_firstpage" content="899"><meta name="citation_doi" content="10.1038/ng.407"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4501476/?report=abstract"><meta name="citation_pmid" content="19578367"><meta name="DC.Title" content="Genome-wide association study identifies five susceptibility loci for glioma"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Sanjay Shete"><meta name="DC.Contributor" content="Fay J Hosking"><meta name="DC.Contributor" content="Lindsay B Robertson"><meta name="DC.Contributor" content="Sara E Dobbins"><meta name="DC.Contributor" content="Marc Sanson"><meta name="DC.Contributor" content="Beatrice Malmer"><meta name="DC.Contributor" content="Matthias Simon"><meta name="DC.Contributor" content="Yannick Marie"><meta name="DC.Contributor" content="Blandine Boisselier"><meta name="DC.Contributor" content="Jean-Yves Delattre"><meta name="DC.Contributor" content="Khe Hoang-Xuan"><meta name="DC.Contributor" content="Soufiane El Hallani"><meta name="DC.Contributor" content="Ahmed Idbaih"><meta name="DC.Contributor" content="Diana Zelenika"><meta name="DC.Contributor" content="Ulrika Andersson"><meta name="DC.Contributor" content="Roger Henriksson"><meta name="DC.Contributor" content="A Tommy Bergenheim"><meta name="DC.Contributor" content="Maria Feychting"><meta name="DC.Contributor" content="Stefan Lönn"><meta name="DC.Contributor" content="Anders Ahlbom"><meta name="DC.Contributor" content="Johannes Schramm"><meta name="DC.Contributor" content="Michael Linnebank"><meta name="DC.Contributor" content="Kari Hemminki"><meta name="DC.Contributor" content="Rajiv Kumar"><meta name="DC.Contributor" content="Sarah J Hepworth"><meta name="DC.Contributor" content="Amy Price"><meta name="DC.Contributor" content="Georgina Armstrong"><meta name="DC.Contributor" content="Yanhong Liu"><meta name="DC.Contributor" content="Xiangjun Gu"><meta name="DC.Contributor" content="Robert Yu"><meta name="DC.Contributor" content="Ching Lau"><meta name="DC.Contributor" content="Minouk Schoemaker"><meta name="DC.Contributor" content="Kenneth Muir"><meta name="DC.Contributor" content="Anthony Swerdlow"><meta name="DC.Contributor" content="Mark Lathrop"><meta name="DC.Contributor" content="Melissa Bondy"><meta name="DC.Contributor" content="Richard S Houlston"><meta name="DC.Date" content="2009 Aug"><meta name="DC.Identifier" content="10.1038/ng.407"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies five susceptibility loci for glioma"><meta property="og:type" content="article"><meta property="og:description" content="To identify risk variants for glioma, we conducted a meta-analysis of two genome-wide association studies by genotyping 550K tagging SNPs in a total of 1,878 cases and 3,670 controls, with validation in three additional independent series totaling 2,545 ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4501476/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4501476/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4501476/pdf/nihms705858.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE89A6DEDD4F5911000000000657041D.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4501476%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4501476</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Genet</a></span>. Author manuscript; available in PMC 2015 Jul 14.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19578367" target="pmc_ext" ref="reftype=publisher&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Genet. 2009 Aug; 41(8): 899–904. </span></a></div><span class="fm-vol-iss-date">Published online 2009 Jul 5. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fng.407" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ng.407</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4501476</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS705858</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/19578367">19578367</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies five susceptibility loci for glioma</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Shete%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778727325440" co-class="co-affbox">Sanjay Shete</a>,<sup>1,</sup><sup>17</sup> <a href="/pubmed/?term=Hosking%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778721695248" co-class="co-affbox">Fay J Hosking</a>,<sup>2,</sup><sup>17</sup> <a href="/pubmed/?term=Robertson%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778721692976" co-class="co-affbox">Lindsay B Robertson</a>,<sup>2</sup> <a href="/pubmed/?term=Dobbins%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725014880" co-class="co-affbox">Sara E Dobbins</a>,<sup>2</sup> <a href="/pubmed/?term=Sanson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725012896" co-class="co-affbox">Marc Sanson</a>,<sup>3</sup> <a href="/pubmed/?term=Malmer%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725010912" co-class="co-affbox">Beatrice Malmer</a>,<sup>4</sup> <a href="/pubmed/?term=Simon%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725008928" co-class="co-affbox">Matthias Simon</a>,<sup>5</sup> <a href="/pubmed/?term=Marie%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725006944" co-class="co-affbox">Yannick Marie</a>,<sup>3</sup> <a href="/pubmed/?term=Boisselier%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725004960" co-class="co-affbox">Blandine Boisselier</a>,<sup>3</sup> <a href="/pubmed/?term=Delattre%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725002976" co-class="co-affbox">Jean-Yves Delattre</a>,<sup>3</sup> <a href="/pubmed/?term=Hoang-Xuan%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778725000992" co-class="co-affbox">Khe Hoang-Xuan</a>,<sup>3</sup> <a href="/pubmed/?term=Hallani%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729272448" co-class="co-affbox">Soufiane El Hallani</a>,<sup>3</sup> <a href="/pubmed/?term=Idbaih%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729270464" co-class="co-affbox">Ahmed Idbaih</a>,<sup>3</sup> <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729268480" co-class="co-affbox">Diana Zelenika</a>,<sup>6</sup> <a href="/pubmed/?term=Andersson%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729266496" co-class="co-affbox">Ulrika Andersson</a>,<sup>4</sup> <a href="/pubmed/?term=Henriksson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729264512" co-class="co-affbox">Roger Henriksson</a>,<sup>4</sup> <a href="/pubmed/?term=Bergenheim%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729262528" co-class="co-affbox">A Tommy Bergenheim</a>,<sup>7</sup> <a href="/pubmed/?term=Feychting%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729260544" co-class="co-affbox">Maria Feychting</a>,<sup>8</sup> <a href="/pubmed/?term=L%26%23x000f6%3Bnn%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778729258560" co-class="co-affbox">Stefan Lönn</a>,<sup>9</sup> <a href="/pubmed/?term=Ahlbom%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723953104" co-class="co-affbox">Anders Ahlbom</a>,<sup>8</sup> <a href="/pubmed/?term=Schramm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723951120" co-class="co-affbox">Johannes Schramm</a>,<sup>5</sup> <a href="/pubmed/?term=Linnebank%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723949136" co-class="co-affbox">Michael Linnebank</a>,<sup>10</sup> <a href="/pubmed/?term=Hemminki%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723947152" co-class="co-affbox">Kari Hemminki</a>,<sup>11</sup> <a href="/pubmed/?term=Kumar%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723945168" co-class="co-affbox">Rajiv Kumar</a>,<sup>11</sup> <a href="/pubmed/?term=Hepworth%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723943184" co-class="co-affbox">Sarah J Hepworth</a>,<sup>12</sup> <a href="/pubmed/?term=Price%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723941200" co-class="co-affbox">Amy Price</a>,<sup>2</sup> <a href="/pubmed/?term=Armstrong%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778723939216" co-class="co-affbox">Georgina Armstrong</a>,<sup>1</sup> <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795334768" co-class="co-affbox">Yanhong Liu</a>,<sup>1</sup> <a href="/pubmed/?term=Gu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795332784" co-class="co-affbox">Xiangjun Gu</a>,<sup>1</sup> <a href="/pubmed/?term=Yu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795330800" co-class="co-affbox">Robert Yu</a>,<sup>1</sup> <a href="/pubmed/?term=Lau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795328816" co-class="co-affbox">Ching Lau</a>,<sup>13</sup> <a href="/pubmed/?term=Schoemaker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795326832" co-class="co-affbox">Minouk Schoemaker</a>,<sup>14</sup> <a href="/pubmed/?term=Muir%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795324848" co-class="co-affbox">Kenneth Muir</a>,<sup>15</sup> <a href="/pubmed/?term=Swerdlow%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795322864" co-class="co-affbox">Anthony Swerdlow</a>,<sup>14</sup> <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778795320880" co-class="co-affbox">Mark Lathrop</a>,<sup>6,</sup><sup>16</sup> <a href="/pubmed/?term=Bondy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778726910928" co-class="co-affbox">Melissa Bondy</a>,<sup>1</sup> and  <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367" class="affpopup" co-rid="_co_idm139778726908944" co-class="co-affbox">Richard S Houlston</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139778727325440"><h3 class="no_margin">Sanjay Shete</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Shete%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Sanjay Shete</a></div></div><div id="_co_idm139778721695248"><h3 class="no_margin">Fay J Hosking</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Hosking%20FJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Fay J Hosking</a></div></div><div id="_co_idm139778721692976"><h3 class="no_margin">Lindsay B Robertson</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Robertson%20LB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Lindsay B Robertson</a></div></div><div id="_co_idm139778725014880"><h3 class="no_margin">Sara E Dobbins</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Dobbins%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Sara E Dobbins</a></div></div><div id="_co_idm139778725012896"><h3 class="no_margin">Marc Sanson</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Sanson%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Marc Sanson</a></div></div><div id="_co_idm139778725010912"><h3 class="no_margin">Beatrice Malmer</h3><p><sup>4</sup>Department of Radiation Sciences, Oncology, Umeå University, Umeå, Sweden</p><div>Find articles by <a href="/pubmed/?term=Malmer%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Beatrice Malmer</a></div></div><div id="_co_idm139778725008928"><h3 class="no_margin">Matthias Simon</h3><p><sup>5</sup>Neurochirurgische Universitätsklinik, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Simon%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Matthias Simon</a></div></div><div id="_co_idm139778725006944"><h3 class="no_margin">Yannick Marie</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Marie%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Yannick Marie</a></div></div><div id="_co_idm139778725004960"><h3 class="no_margin">Blandine Boisselier</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Boisselier%20B%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Blandine Boisselier</a></div></div><div id="_co_idm139778725002976"><h3 class="no_margin">Jean-Yves Delattre</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Delattre%20JY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Jean-Yves Delattre</a></div></div><div id="_co_idm139778725000992"><h3 class="no_margin">Khe Hoang-Xuan</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Hoang-Xuan%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Khe Hoang-Xuan</a></div></div><div id="_co_idm139778729272448"><h3 class="no_margin">Soufiane El Hallani</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Hallani%20SE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Soufiane El Hallani</a></div></div><div id="_co_idm139778729270464"><h3 class="no_margin">Ahmed Idbaih</h3><p><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Idbaih%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Ahmed Idbaih</a></div></div><div id="_co_idm139778729268480"><h3 class="no_margin">Diana Zelenika</h3><p><sup>6</sup>Centre National de Génotypage, IG/CEA, Evry Cedex, France</p><div>Find articles by <a href="/pubmed/?term=Zelenika%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Diana Zelenika</a></div></div><div id="_co_idm139778729266496"><h3 class="no_margin">Ulrika Andersson</h3><p><sup>4</sup>Department of Radiation Sciences, Oncology, Umeå University, Umeå, Sweden</p><div>Find articles by <a href="/pubmed/?term=Andersson%20U%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Ulrika Andersson</a></div></div><div id="_co_idm139778729264512"><h3 class="no_margin">Roger Henriksson</h3><p><sup>4</sup>Department of Radiation Sciences, Oncology, Umeå University, Umeå, Sweden</p><div>Find articles by <a href="/pubmed/?term=Henriksson%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Roger Henriksson</a></div></div><div id="_co_idm139778729262528"><h3 class="no_margin">A Tommy Bergenheim</h3><p><sup>7</sup>Department of Clinical Neuroscience, Umeå University, Umeå, Sweden</p><div>Find articles by <a href="/pubmed/?term=Bergenheim%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">A Tommy Bergenheim</a></div></div><div id="_co_idm139778729260544"><h3 class="no_margin">Maria Feychting</h3><p><sup>8</sup>Institute of Environmental Medicine, Karolinska Institutet, Sweden</p><div>Find articles by <a href="/pubmed/?term=Feychting%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Maria Feychting</a></div></div><div id="_co_idm139778729258560"><h3 class="no_margin">Stefan Lönn</h3><p><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=L%26%23x000f6%3Bnn%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Stefan Lönn</a></div></div><div id="_co_idm139778723953104"><h3 class="no_margin">Anders Ahlbom</h3><p><sup>8</sup>Institute of Environmental Medicine, Karolinska Institutet, Sweden</p><div>Find articles by <a href="/pubmed/?term=Ahlbom%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Anders Ahlbom</a></div></div><div id="_co_idm139778723951120"><h3 class="no_margin">Johannes Schramm</h3><p><sup>5</sup>Neurochirurgische Universitätsklinik, Bonn, Germany</p><div>Find articles by <a href="/pubmed/?term=Schramm%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Johannes Schramm</a></div></div><div id="_co_idm139778723949136"><h3 class="no_margin">Michael Linnebank</h3><p><sup>10</sup>Department of Neurology, University Hospital Zurich, Zurich, Switzerland</p><div>Find articles by <a href="/pubmed/?term=Linnebank%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Michael Linnebank</a></div></div><div id="_co_idm139778723947152"><h3 class="no_margin">Kari Hemminki</h3><p><sup>11</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 580, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Hemminki%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Kari Hemminki</a></div></div><div id="_co_idm139778723945168"><h3 class="no_margin">Rajiv Kumar</h3><p><sup>11</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 580, Heidelberg, Germany</p><div>Find articles by <a href="/pubmed/?term=Kumar%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Rajiv Kumar</a></div></div><div id="_co_idm139778723943184"><h3 class="no_margin">Sarah J Hepworth</h3><p><sup>12</sup>Centre for Epidemiology and Biostatistics, Faculty of Medicine and Health, University of Leeds, Leeds, UK</p><div>Find articles by <a href="/pubmed/?term=Hepworth%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Sarah J Hepworth</a></div></div><div id="_co_idm139778723941200"><h3 class="no_margin">Amy Price</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Price%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Amy Price</a></div></div><div id="_co_idm139778723939216"><h3 class="no_margin">Georgina Armstrong</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Armstrong%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Georgina Armstrong</a></div></div><div id="_co_idm139778795334768"><h3 class="no_margin">Yanhong Liu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Liu%20Y%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Yanhong Liu</a></div></div><div id="_co_idm139778795332784"><h3 class="no_margin">Xiangjun Gu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Gu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Xiangjun Gu</a></div></div><div id="_co_idm139778795330800"><h3 class="no_margin">Robert Yu</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Yu%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Robert Yu</a></div></div><div id="_co_idm139778795328816"><h3 class="no_margin">Ching Lau</h3><p><sup>13</sup>Texas Children's Cancer Center, Baylor College of Medicine, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Lau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Ching Lau</a></div></div><div id="_co_idm139778795326832"><h3 class="no_margin">Minouk Schoemaker</h3><p><sup>14</sup>Section of Epidemiology, Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Schoemaker%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Minouk Schoemaker</a></div></div><div id="_co_idm139778795324848"><h3 class="no_margin">Kenneth Muir</h3><p><sup>15</sup>Division of Epidemiology and Public Health, University of Nottingham Medical School, Queen's Medical Centre, Nottingham, UK</p><div>Find articles by <a href="/pubmed/?term=Muir%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Kenneth Muir</a></div></div><div id="_co_idm139778795322864"><h3 class="no_margin">Anthony Swerdlow</h3><p><sup>14</sup>Section of Epidemiology, Institute of Cancer Research, Sutton, UK</p><div>Find articles by <a href="/pubmed/?term=Swerdlow%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Anthony Swerdlow</a></div></div><div id="_co_idm139778795320880"><h3 class="no_margin">Mark Lathrop</h3><p><sup>6</sup>Centre National de Génotypage, IG/CEA, Evry Cedex, France</p><p><sup>16</sup>Foundation Jean Dausett-CEPH, Paris, France</p><div>Find articles by <a href="/pubmed/?term=Lathrop%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Mark Lathrop</a></div></div><div id="_co_idm139778726910928"><h3 class="no_margin">Melissa Bondy</h3><p><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</p><div>Find articles by <a href="/pubmed/?term=Bondy%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Melissa Bondy</a></div></div><div id="_co_idm139778726908944"><h3 class="no_margin">Richard S Houlston</h3><p><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</p><div>Find articles by <a href="/pubmed/?term=Houlston%20RS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=19578367">Richard S Houlston</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139778726615216_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139778726615216_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139778726615216_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Epidemiology, The University of Texas M.D. Anderson Cancer Center, Houston, Texas, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Section of Cancer Genetics, Institute of Cancer Research, Sutton, Surrey, UK</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Service de Neurologie Mazarin and UMR 975 INSERM-UPMC, GH Pitié-Salpêtrière, APHP, Paris, France</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Department of Radiation Sciences, Oncology, Umeå University, Umeå, Sweden</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Neurochirurgische Universitätsklinik, Bonn, Germany</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Centre National de Génotypage, IG/CEA, Evry Cedex, France</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Clinical Neuroscience, Umeå University, Umeå, Sweden</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Institute of Environmental Medicine, Karolinska Institutet, Sweden</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institutet, Stockholm, Sweden</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Department of Neurology, University Hospital Zurich, Zurich, Switzerland</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Division of Molecular Genetic Epidemiology, German Cancer Research Center (DKFZ), Im Neuenheimer Feld 580, Heidelberg, Germany</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Centre for Epidemiology and Biostatistics, Faculty of Medicine and Health, University of Leeds, Leeds, UK</div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Texas Children's Cancer Center, Baylor College of Medicine, Houston, Texas, USA</div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Section of Epidemiology, Institute of Cancer Research, Sutton, UK</div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Division of Epidemiology and Public Health, University of Nottingham Medical School, Queen's Medical Centre, Nottingham, UK</div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Foundation Jean Dausett-CEPH, Paris, France</div><div id="FN1">Correspondence should be addressed to R.S.H. (<a href="mailto:dev@null" data-email="ku.ca.rci@notsluoh.drahciR" class="oemail">ku.ca.rci@notsluoh.drahciR</a>) or M.B. (<a href="mailto:dev@null" data-email="gro.nosrednadm@ydnobm" class="oemail">gro.nosrednadm@ydnobm</a>)</div><div id="FN2"><sup>17</sup>These authors contributed equally to this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm139778726615216_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div><div class="license half_rhythm">Reprints and permissions information is available online at <a href="http://npg.nature.com/reprintsandpermissions/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://npg.nature.com/reprintsandpermissions/</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=19578367" target="pmc_ext" ref="reftype=publisher&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Genet</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4501476/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplemenatry materials.</strong></div><div><a href="/pmc/articles/PMC4501476/bin/NIHMS705858-supplement-Supplemenatry_materials.pdf" data-ga-action="click_feat_suppl">NIHMS705858-supplement-Supplemenatry_materials.pdf</a><span style="color:gray"> (639K)</span></div><div class="small guid">GUID:&nbsp;C600567B-C4E2-4986-9850-E66E34DD4A7F</div></div></dd></dl></div><div id="idm139778727169872" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139778727169872title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">To identify risk variants for glioma, we conducted a meta-analysis of two genome-wide association studies by genotyping 550K tagging SNPs in a total of 1,878 cases and 3,670 controls, with validation in three additional independent series totaling 2,545 cases and 2,953 controls. We identified five risk loci for glioma at 5p15.33 (rs2736100, <em>TERT</em>; <em>P</em> = 1.50 × 10<sup>−17</sup>), 8q24.21 (rs4295627, <em>CCDC26</em>; <em>P</em> = 2.34 × 10<sup>−18</sup>), 9p21.3 (rs4977756, <em>CDKN2A-CDKN2B</em>; <em>P</em> = 7.24 × 10<sup>−15</sup>), 20q13.33 (rs6010620, <em>RTEL1</em>; <em>P</em> = 2.52 × 10<sup>−12</sup>) and 11q23.3 (rs498872, <em>PHLDB1</em>; <em>P</em> = 1.07 × 10<sup>−8</sup>). These data show that common low-penetrance susceptibility alleles contribute to the risk of developing glioma and provide insight into disease causation of this primary brain tumor.</p></div></div><div id="idm139778723539024" class="tsec sec headless whole_rhythm"><p id="P2" class="p p-first">Gliomas account for ∼80% of malignant primary brain tumors (PBT). In the United States, ∼21,000 individuals are diagnosed with glioma annually<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366106">1</a></sup> and for most the prognosis is dismal<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366107">1</a></sup>. No lifestyle exposure has consistently been linked to glioma risk except ionizing radiation, which accounts for few cases<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366111">1</a></sup>. Predicated on the hypothesis that part of the twofold increased glioma risk in relatives of individuals with PBT<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366120">2</a></sup> is a consequence of the co-inheritance of multiple low-risk variants, we conducted two genome-wide association (GWA) studies. Pooling data from both scans and following up the best-supported associations, we have identified five glioma risk loci.</p><p id="P3">The two GWA studies were conducted in the UK and US (UK-GWA and US-GWA studies, respectively; <a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). In both, genotyping of cases was conducted using Illumina Infinium HD Human610-Quad BeadChips. The UK-GWA study was based on genotyping 636 glioma cases ascertained through INTERPHONE<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366088">3</a></sup>. After application of quality control criteria, genotypes were available for 631 cases (<a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). For controls, we used Illumina Hap550K-BeadChip genotypes on 1,438 individuals from the British 1958 Birth Cohort<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366089">4</a></sup>. The US-GWA study was based on genotyping 1,281 glioma cases ascertained by The University of Texas M.D. Anderson Cancer Center. After applying quality control criteria, genotypes were available for 1,247 cases (<a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>). For controls, we used Illumina Hap550K-BeadChip genotypes on 2,243 individuals from the Cancer Genetic Markers of Susceptibility studies<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366108">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366090">6</a></sup>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139778728597344" class="figure"><a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4501476_nihms705858f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms705858f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4501476/bin/nihms705858f1.jpg"></a></div><div id="largeobj_idm139778728597344" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4501476/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4501476/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p5">Subjects and single-SNP exclusion schema for genome-wide association studies.</p></div></div></div><p id="P4">Across both case series, 572,571 SNPs were satisfactorily genotyped (99.4%), with mean individual sample call rates of 99.8%. We excluded 32 individuals because of non-Western European ancestry and two because of cryptic relatedness (<a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1a</a>). For the 1,878 cases and 3,670 controls in the combined data, 521,318 SNP genotypes were available. We considered only the 454,576 autosomal SNPs having call rates &gt;95% in all series, showing mimimal departure from Hardy-Weinberg equilibrium (HWE; <em>P</em> &gt; 10<sup>−5</sup> in controls) and minor allele frequencies (MAF) &gt; 1% in cases and controls (<a href="/pmc/articles/PMC4501476/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>).</p><p id="P5">Comparison of the observed and expected distributions showed little evidence for an inflation of the test statistics in the datasets (inflation factor<sup><a href="#R7" rid="R7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366100">7</a></sup>
<em>λ</em> = 1.02 and 1.05 in UK-GWA and US-GWA, respectively, based on the 90% least significant SNPs; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 1b</a>), thereby excluding significant hidden population substructure or differential genotype calling. Using standard methods and data from both studies, we derived joint odds ratios (ORs) and confidence intervals for each SNP and associated <em>P</em> values.</p><p id="P6">To identify true risk alleles among the 34 SNPs showing evidence of an association at <em>P</em> &lt; 10<sup>−5</sup>, we conducted replication in three case-control series involving a total of 5,498 individuals that passed quality control (French series: 1,392 cases, 1,602 controls; German series: 504 cases, 573 controls; Swedish series: 649 cases, 778 controls). We satisfactorily genotyped 31 of the 34 SNPs in all series. In the French series, rs9656979 and rs7257116 were not genotyped. However, an association between these SNPs and glioma risk was not supported in either the German or Swedish series (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Although rs7300686 was not typed in the German or Swedish series, an association with risk was not supported in the French series (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Hence, failure to genotype all additional series for the three SNPs has not affected study findings. Across the five data series, there was a significant association between rs7124728 and glioma risk, but this was not supported in the three replication series (<em>P</em> &gt; 0.1). rs4944840 maps 57 kb from rs7124728 and both SNPs are in strong linkage disequilibrium (LD; <em>D′</em> = 0.89, <em>r</em><sup>2</sup> = 0.71). No association between rs4944840 and glioma risk was seen in either GWA study (<em>P</em> = 0.13 UK-GWA, <em>P</em> = 0.03 US-GWA). There was a significant difference in MAF of rs7124728 between both GWA cases and replication cases but not between controls, suggesting a genotyping error with the Illumina610-Quad BeadChip.</p><p id="P7">Excluding rs7124728, in the combined analyses, 14 of the 31 SNPs representing five genomic regions satisfied an accepted threshold for genome-wide statistical significance (that is, <em>P</em> &lt; 5 × 10<sup>−7</sup>; <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a>). Under a fixed-effects model, the strongest signal was attained at rs4295627 (OR = 1.36, 95% CI: 1.29–1.43; <em>P</em> = 2.34 × 10<sup>−18</sup>), which localizes to 8q24.21 (130,754,639 bp; <a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). There was, however, evidence of between-study heterogeneity (<em>P</em><sub>het</sub> = 0.01) ascribable to the association being modest in the Swedish series. Under a random-effects model, the OR was 1.36 (95% CI: 1.19–1.58; <em>P</em> = 2.01 × 10<sup>−5</sup>) using all data and 1.45 (95% CI: 1.32–1.57; <em>P</em> = 1.02 × 10<sup>−8</sup>) excluding the Swedish study. rs4295627 maps to intron 3 of <em>CCDC26</em>, encoding a retinoic acid modulator of differentiation and death<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366092">8</a></sup>. Retinoic acid induces caspase-8 transcription through phosphorylation of CREB and increases apoptosis induced by death stimuli in neuroblastoma cells<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366124">9</a></sup> and in glioblastoma cells with downregulation of telomerase activity<sup><a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366091">10</a></sup>. Some evidence suggests a second 8q24.21 locus defined by rs891835, mapping to intron 1 of <em>CCDC26</em> 207 kb telomeric to rs4295627 (130,560,934 bp; OR = 1.24, 95% CI: 1.17–1.30; <em>P</em> = 7.54 × 10<sup>−11</sup>) and showing low LD with rs4295627 (<em>D′</em> = 0.59 and <em>r</em><sup>2</sup> = 0.19 and <em>D′</em> = 0.58, <em>r</em><sup>2</sup> = 0.26 in HapMap and GWA scan, respectively). The OR for rs4295627 with adjustment for rs891835 was 1.30 (95% CI: 1.20–1.41; <em>P</em> = 2.80 × 10<sup>−10</sup>). Similarly, adjusting rs891835 for rs4295627 provided evidence of an association, albeit at a nominal significance level (OR = 1.08, 95% CI: 1.00–1.17; <em>P</em> = 0.03). There was also an increasing trend in OR with an increasing number of risk alleles (<em>P</em> = 2.67 × 10<sup>−13</sup>) and the frequency of the two common haplotypes differed between cases and controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). When we imputed 8q24.21 genotypes using genome-wide data, rs1077236, mapping to 130,709,683 bp, provided a marginally better association signal than rs4295627 (<em>P</em> = 2.75 × 10<sup>−9</sup> and 1.60 × 10<sup>−8</sup>, respectively). Hence, the possibility remains that rs4295627 and rs891835, although only weakly associated, are in LD with one or more causal variants in this region.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139778728589760" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is nihms705858f2.jpg" title="An external file that holds a picture, illustration, etc.
Object name is nihms705858f2.jpg" src="/pmc/articles/PMC4501476/bin/nihms705858f2.jpg"></div></a><div id="largeobj_idm139778728589760" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a target="object" href="/pmc/articles/PMC4501476/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4501476/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p6">LD structure and association results for the five confirmed glioma-associated regions. (<strong>a</strong>) 8q24.21. (<strong>b</strong>) 5p15.33. (<strong>c</strong>) 9p21.3. (<strong>d</strong>) 20q13.33. (<strong>e</strong>) 11q23.3. Chromosomal positions and genes based on NCBI build 36 coordinates. Armitage trend test <em>P</em> values (as −log<sub>10</sub> values; left <em>y</em> axis) are shown for SNPs analyzed in the GWA studies in blue and for the combined analysis including the replication series in red. Recombination rates in HapMap CEU across the region are shown in black (right <em>y</em> axis). Also shown are the relative positions of genes mapping to each region of association. Exons of genes have been redrawn to show the relative positions in the gene, and therefore maps are not to physical scale.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Summary results for SNPs associated with glioma in GWA and replication samples</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139778728583472" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="middle" align="center">GWA studies</th><th colspan="2" rowspan="1" valign="middle" align="center">Replication studies</th><th colspan="3" rowspan="1" valign="middle" align="center">Combined</th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="left"></th><th rowspan="1" colspan="1" valign="middle" align="right"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th rowspan="1" colspan="1" valign="middle" align="center"></th><th colspan="2" rowspan="1" valign="bottom">
<hr></th><th colspan="2" rowspan="1" valign="bottom">
<hr></th><th colspan="3" rowspan="1" valign="bottom">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="middle" align="left">SNP</th><th rowspan="1" colspan="1" valign="middle" align="right">Chr.</th><th rowspan="1" colspan="1" valign="middle" align="left">Gene<sup>a</sup></th><th rowspan="1" colspan="1" valign="middle" align="right">Location (bp)</th><th rowspan="1" colspan="1" valign="middle" align="center">Ancestral allele frequency</th><th rowspan="1" colspan="1" valign="middle" align="center">Risk allele<sup>b</sup></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="middle" align="center"><em>P</em>
<sub>het</sub></th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2736100</td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>TERT</em></td><td rowspan="1" colspan="1" valign="middle" align="right">1,339,516</td><td rowspan="1" colspan="1" valign="middle" align="center">0.51</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.20 (1.10–1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.21 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.33 (1.20–1.49)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.87 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 (1.19–1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.50 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.18</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2853676</td><td rowspan="1" colspan="1" valign="middle" align="right">5</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>TERT</em></td><td rowspan="1" colspan="1" valign="middle" align="right">1,341,547</td><td rowspan="1" colspan="1" valign="middle" align="center">0.27</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.14–1.31)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.30 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.30 (1.21–1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.06 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.26 (1.20–1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.21 × 10<sup>−14</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs10464870</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CCDC26</em></td><td rowspan="1" colspan="1" valign="middle" align="right">130,547,005</td><td rowspan="1" colspan="1" valign="middle" align="center">0.21</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.15–1.34)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.90 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.13–1.31)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.77 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23 (1.17–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.04 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.05</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs891835</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CCDC26</em></td><td rowspan="1" colspan="1" valign="middle" align="right">130,560,934</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.15–1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">3.92 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.15–1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.43 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.17–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.54 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6470745</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CCDC26</em></td><td rowspan="1" colspan="1" valign="middle" align="right">130,711,103</td><td rowspan="1" colspan="1" valign="middle" align="center">0.20</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.30 (1.20–1.39)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.79 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.31 (1.22–1.41)</td><td rowspan="1" colspan="1" valign="middle" align="center">9.09 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.30 (1.24–1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.77 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs16904140</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CCDC26</em></td><td rowspan="1" colspan="1" valign="middle" align="right">130,734,825</td><td rowspan="1" colspan="1" valign="middle" align="center">0.21</td><td rowspan="1" colspan="1" valign="middle" align="center">A</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25 (1.16–1.35)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.28 (1.19–1.37)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.14 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 (1.20–1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.88 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4295627</td><td rowspan="1" colspan="1" valign="middle" align="right">8</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CCDC26</em></td><td rowspan="1" colspan="1" valign="middle" align="right">130,754,639</td><td rowspan="1" colspan="1" valign="middle" align="center">0.17</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.33 (1.23–1.42)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.47 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.39 (1.30–1.49)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.20 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.36 (1.29–1.43)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.34 × 10<sup>−18</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.01</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1063192</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CDKN2A/B</em></td><td rowspan="1" colspan="1" valign="middle" align="right">21,993,367</td><td rowspan="1" colspan="1" valign="middle" align="center">0.44</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.21 (1.13–1.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.21 (1.14–1.29)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.97 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.21 (1.16–1.27)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.61 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.81</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2157719</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CDKN2A/B</em></td><td rowspan="1" colspan="1" valign="middle" align="right">22,023,366</td><td rowspan="1" colspan="1" valign="middle" align="center">0.57</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>G</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.11–1.35)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.11–1.33)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.42 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.14–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.41 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.68</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs1412829</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CDKN2A/B</em></td><td rowspan="1" colspan="1" valign="middle" align="right">22,033,926</td><td rowspan="1" colspan="1" valign="middle" align="center">0.42</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.14–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.23 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.23 (1.15–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.80 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.22 (1.17–1.28)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.23 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.67</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs4977756</td><td rowspan="1" colspan="1" valign="middle" align="right">9</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>CDKN2A/B</em></td><td rowspan="1" colspan="1" valign="middle" align="right">22,058,652</td><td rowspan="1" colspan="1" valign="middle" align="center">0.40</td><td rowspan="1" colspan="1" valign="middle" align="center">G</td><td rowspan="1" colspan="1" valign="middle" align="center">1.25 (1.17–1.32)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.39 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.16–1.31)</td><td rowspan="1" colspan="1" valign="middle" align="center">5.90 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.24 (1.19–1.30)</td><td rowspan="1" colspan="1" valign="middle" align="center">7.24 × 10<sup>−15</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.94</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs498872</td><td rowspan="1" colspan="1" valign="middle" align="right">11</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>PHLDB1</em></td><td rowspan="1" colspan="1" valign="middle" align="right">117,982,577</td><td rowspan="1" colspan="1" valign="middle" align="center">0.31</td><td rowspan="1" colspan="1" valign="middle" align="center">T</td><td rowspan="1" colspan="1" valign="middle" align="center">1.26 (1.17–1.34)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.03 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.12 (1.04–1.20)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.18 (1.13–1.24)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.07 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.04</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs6010620</td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>RTEL1</em></td><td rowspan="1" colspan="1" valign="middle" align="right">61,780,283</td><td rowspan="1" colspan="1" valign="middle" align="center">0.23</td><td rowspan="1" colspan="1" valign="middle" align="center"><strong>G</strong></td><td rowspan="1" colspan="1" valign="middle" align="center">1.28 (1.18–1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">8.38 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.28 (1.18–1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">6.49 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.28 (1.21–1.35)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.52 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.38</td></tr><tr><td rowspan="1" colspan="1" valign="middle" align="left">rs2297440</td><td rowspan="1" colspan="1" valign="middle" align="right">20</td><td rowspan="1" colspan="1" valign="middle" align="left"><em>RTEL1</em></td><td rowspan="1" colspan="1" valign="middle" align="right">61,782,743</td><td rowspan="1" colspan="1" valign="middle" align="center">0.22</td><td rowspan="1" colspan="1" valign="middle" align="center">C</td><td rowspan="1" colspan="1" valign="middle" align="center">1.28 (1.18–1.38)</td><td rowspan="1" colspan="1" valign="middle" align="center">1.01 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.26 (1.16–1.35)</td><td rowspan="1" colspan="1" valign="middle" align="center">4.44 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">1.27 (1.20–1.34)</td><td rowspan="1" colspan="1" valign="middle" align="center">2.06 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="middle" align="center">0.40</td></tr></tbody></table></div><div id="largeobj_idm139778728583472" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4501476/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>a</sup>Gene(s) mapping within 20 kb of each SNP are listed.</div><div id="TFN2"><sup>b</sup>Ancestral allele marked in bold.</div></div></div><p id="P8">Variation at 8q24.21 has been implicated in colorectal<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366101">11</a></sup>, breast<sup><a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366109">12</a></sup>, prostate<sup><a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366097">6</a></sup> and bladder cancer risk<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_473907923">13</a></sup>. The genomic regions underlying these associations map to the 128- to 129-Mb interval, distinct from the glioma signal. Closer to the 8q24.21 glioma signal is rs987525, a risk factor for cleft lip<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366099">14</a></sup>. Notably, nonsyndromic cleft lip has been reported to be a risk factor for PBT<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366114">15</a></sup>. Although rs987525 maps 740 kb centromeric to rs4295627 and LD between rs987525 and rs4295627 is weak (<em>D′</em> = 0.46, <em>r</em><sup>2</sup> = 0.01), it is possible that these SNPs define a common disease locus.</p><p id="P9">The second strongest evidence for an association was for rs2736100 (OR = 1.27, 95% CI: 1.19–1.37; <em>P</em> = 1.50 × 10<sup>−17</sup>) mapping to 5p15.33, which localizes to intron 2 of <em>TERT</em> (1,339,516 bp; <a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). TERT is the reverse transcriptase component of telomerase, essential for telomerase activity in maintaining telomeres and cell immortalization. Although 5p15.33 copy number change is uncommon in glioblastoma multiforme (GBM; <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2a</a>), <em>TERT</em> expression correlates with glioma grade and prognosis<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366105">16</a></sup>. Several lines of evidence raise the possibility of a second locus defined by rs2853676, which also maps to intron 2 of <em>TERT</em> (1,341,547 bp; OR = 1.26, 95% CI: 1.20–1.32; <em>P</em> = 4.21 × 10<sup>−14</sup>) and is in weak LD with rs2736100 (<em>D′</em> = 0.67 and <em>r</em><sup>2</sup> = 0.15 and <em>D′</em> = 0.81 and <em>r</em><sup>2</sup> = 0.24 in HapMap and GWA scan, respectively). The OR for rs2736100 with adjustment for rs2853676 was 1.19 (95% CI: 1.12–1.27; <em>P</em> = 5.33 × 10<sup>−8</sup>) and adjusting rs2853676 for rs2736100 also provided evidence of an association, with little change in the risk estimate (OR = 1.15, 95% CI: 1.08–1.23; <em>P</em> = 6.90 × 10<sup>−5</sup>). There was an increasing trend in OR with an increasing number of risk alleles (<em>P</em> = 1.19 × 10<sup>−19</sup>). Comparison of haplotype frequencies provided evidence of two haplotypes differing between cases and controls (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). Recent data has implicated variation at 5p15.33 (<em>TERT-CLPTM1L</em>) defined by rs2736098 with risk of multiple tumor types<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_473907924">17</a></sup>. As LD between rs2736098 and both rs2736100 and rs2853676 is poor (<em>D′</em> = 0.48 and <em>r</em><sup>2</sup> = 0.11 and <em>D′</em> = 0.28 and <em>r</em><sup>2</sup> = 0.02 in HapMap and GWA scan, respectively), whether the 5p15.33 association with glioma is the same remains to be established.</p><p id="P10">The third strongest evidence for an association was for rs4977756 (OR = 1.24, 95% CI: 1.19–1.30; <em>P</em> = 7.24 × 10<sup>−15</sup>), mapping 59 kb telomeric to <em>CDKN2B</em> (22,058,652 bp; <a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) within a 122-kb region of LD at 9p21.3. This region encompasses the <em>CDKN2A-CDKN2B</em> tumor suppressor genes. <em>CDKN2A</em> encodes p16(INK4A) a negative regulator of cyclin-dependent kinases and p14(ARF1), an activator of p53. <em>CDKN2A-CDKN2B</em> has an established role in glioma, with homozygous deletion in <em>CDKN2A</em> detectable in ∼50% of tumors<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366127">18</a>,<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366084">19</a></sup> and loss of expression linked to poor prognosis<sup><a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366095">16</a></sup>. Furthermore, germline mutation of <em>CDKN2A-CDKN2B</em> causes the melanoma-astrocytoma syndrome<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366119">20</a>,<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366110">21</a></sup>. Regulation of p16/p14ARF is important for sensitivity to ionizing radiation, the only environmental factor strongly linked to gliomagenesis<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366125">22</a></sup>. These data provide evidence that common variation in addition to rare mutations in <em>CDKN2A-CDKN2B</em> contributes to glioma predisposition.</p><p id="P11">A 68-kb core region of 9p21.3 incorporating the association signal influences risk of type 2 diabetes (T2D)<sup><a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366087">23</a></sup> and coronary heart disease (CHD)<sup><a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366115">24</a></sup>. rs564398, a risk factor for T2D and CHD, is in LD with rs4977756 (<em>D′</em> = 0.77, <em>r</em><sup>2</sup> = 0.50). A possible explanation for a common etiology is that these variants affect determination of birth weight, linked as a risk factor to all three diseases<sup><a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366116">25</a>–<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_473907922">27</a></sup>.</p><p id="P12">The fourth strongest evidence for an association was for rs6010620 (OR = 1.28, 95% CI: 1.21–1.35; <em>P</em> = 2.52 × 10<sup>−12</sup>), localizing to intron 12 (61,780,283 bp) of the RAD3-like helicase gene <em>RTEL1</em> (<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366086">ref 28</a></sup>) and mapping within a 65-kb region of LD at 20q13.33 (<a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Amplification of 20q13.33 is seen in ∼30% of gliomas with copy number change correlating with <em>RTEL1</em> expression (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2a</a>). <em>RTEL1</em> maintains genomic stability directly by suppressing homologous recombination<sup><a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366093">28</a></sup>. These data, coupled with the role of <em>TERT</em> in glioma, make <em>RTEL1</em> an attractive candidate for the 20q13.33 association but do not preclude the association being through variation in other genes mapping to the region, such as <em>TNFRSF6B</em>, whose expression correlates with glioma grade<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366102">29</a></sup>.</p><p id="P13">The fifth-strongest signal was at rs498872 (OR = 1.18, 95% CI: 1.13–1.24; <em>P</em> = 1.07 × 10<sup>−8</sup>), which maps to the 5′-UTR of <em>PHLDB1</em> within a 101-kb LD block on 11q23.3 (117982577 bp; <a href="/pmc/articles/PMC4501476/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and <a href="/pmc/articles/PMC4501476/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). There was, however, evidence of between-study heterogeneity (<em>P</em><sub>het</sub> = 0.04). Under a random-effects model, the combined OR was 1.16 (95% CI: 1.05–1.28; <em>P</em> = 2.24 × 10<sup>−3</sup>). Most of this heterogeneity could be ascribed to the German series; excluding this, the OR was 1.22 (95% CI: 1.16–1.28; <em>P</em> = 2.56 × 10<sup>−10</sup>, <em>P</em><sub>het</sub> = 0.40). Although there is no direct evidence for a role of <em>PHLDB1</em>, 11q23.3 is commonly deleted in neuroblastoma<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366118">30</a></sup>.</p><p id="P14">We analyzed the association between SNP genotypes and risk of tumors of glial origin, GBM and astrocytic lineage. There was evidence that GBMs have a different risk profile, notably with respect to rs4295627, rs498872 and rs2736100 (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>). However, because of study heterogeneity, we cannot exclude the possibility that the variants do not influence the risk of developing a specific histological form of glioma without central histopathological review.</p><p id="P15">We only found evidence of interactive effects between 8q24.21 SNPs (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>). For all other pairwise combinations, each locus had an independent role in glioma (<em>P</em><sub>interaction</sub> &gt; 0.10). Glioma risk increases with increasing numbers of variant alleles for the five loci (OR<sub>per-allele</sub> = 1.31, 95% CI: 1.26–1.36; <em>P</em> = 1.39 × 10<sup>−74</sup>; <a href="/pmc/articles/PMC4501476/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 5</a>). Individuals with eight or more risk alleles have an approximately fourfold increase in glioma risk compared to those with a median number of risk alleles. These ORs may be underestimates because the additive model assumes equal weighting across SNPs. We estimate that the five loci we have identified account for ∼7–14% of the excess familial risk of glioma (assuming naïve multiplicative or additive models and an overall familial relative risk of 2.1). However, we acknowledge that the present data provide only crude estimates of the effect on susceptibility attributable to variation at the loci, as the effect of the actual common causal variant responsible for the association will typically be larger.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4501476/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139778728585904" class="figure"><a href="/pmc/articles/PMC4501476/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4501476_nihms705858f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms705858f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4501476/bin/nihms705858f3.jpg"></a></div><div id="largeobj_idm139778728585904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4501476/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4501476/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p7">Cumulative effects of glioma risk alleles. (<strong>a</strong>) Distribution of risk alleles in glioma cases (green bars) and controls (blue bars). (<strong>b</strong>) Plot of the increasing ORs for glioma with increasing number of risk alleles. The ORs are relative to the median number of four risk alleles; vertical bars correspond to 95% confidence intervals. Horizontal line marks the null value (OR = 1). The distribution of risk alleles follows a normal distribution in both case and controls, with a shift toward a higher number of risk alleles in cases.</p></div></div></div><p id="P16">To investigate the basis of causality for each of the five associations, we interrogated HapMap to identify nonsynonymous SNPs (nsSNPs) correlated with the most strongly associated SNP signals. The strongest LD was between rs6010620 and nsSNP rs3848668 (<em>D′</em> = 1.0, <em>r</em><sup>2</sup> = 0.05). Although HapMap is not comprehensive, these data suggest the associations identified are probably mediated through LD with sequence changes that influence expression rather than protein sequence or through LD with low-frequency variants not catalogued. We searched for <em>cis</em>-acting regulatory effects of SNPs on nearby genes in lymphoblastoid cell lines and normal brain tissue using publicly available data. Although we did not identify any significant genotype–expression relationships (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Fig. 2b</a>), this does not exclude subtle effects or the possibility that the variants exert effects through untested transactivation of genes.</p><p id="P17" class="p">Our findings show that common variants influence glioma risk and highlight the importance of variation in genes encoding components of the CDKN2A-CDK4 signaling pathway in glioma. Moreover, this pathway, elucidated through the extended interaction network of <em>CDKN2A</em>, incorporates <em>TERT</em> (through mutual interaction with HSP90) and other genes (including <em>CCDC26</em>) we have identified as risk factors.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Online Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Subjects</h3><div id="S4" class="sec sec-first"><p></p><h4 id="S4title" class="inline">UK-GWA study </h4><p id="P19" class="p p-first-last">The UK study was based on 636 cases (401 male, 235 female; mean age 46 years; s.d. = 12) ascertained through the INTERPHONE Study<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366083">3</a></sup>. Briefly, the INTERPHONE Study was an international multicenter case-control study of PBT coordinated by the International Agency for Research on Cancer (IARC), with material collected between September 2000 and February 2004. UK individuals with PBT were collected through neurosurgery, neuropathology, oncology and neurology centers in the Thames regions of Southeast England and the Northern UK including central Scotland, the West Midlands, West Yorkshire and the Trent area. Cases were individuals with glioma (International Classification of Diseases for oncology (ICD-O), 2nd ed., codes 9380-9384, 9390-9411, 9420-9451 and 9505; ICD10 code C71). Cases with previous brain tumors were excluded. To minimize population stratification, cases with self-reported non-western-European ancestry were excluded from the present study. Individuals from the 1958 Birth Cohort served as source of controls<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366117">4</a></sup>.</p></div><div id="S5" class="sec"><p></p><h4 id="S5title" class="inline">US-GWA study </h4><p id="P20" class="p p-first-last">The US study was based on 1,247 cases (768 male, 479 female; mean age 47 years; s.d. = 13) ascertained through The University of Texas M.D. Anderson Cancer Center, Texas, between 1990 and 2008. Cases were individuals with glioma (ICD10 code C71; ICD-O codes 9380-9384, 9390-9411, 9420-9451, and 9505). Individuals from CGEMS<sup><a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366094">5</a>,<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366112">6</a></sup> served as controls.</p></div><div id="S6" class="sec sec-last"><p></p><h4 id="S6title" class="inline">Replication series </h4><p id="P21" class="p p-first-last">The French Glioma Collection (FGC) is systematic series of 1,439 individuals with histologically proven glioma (WHO classification AI, AII, AIII, OII, OIII, OAII, OAIII, GBM-IV) ascertained through the Service de Neurologie Mazarin, Groupe Hospitalier Pitié-Salpêtrière Paris. The controls are taken from the SU.VI.MAX (SUpplementation en VItamines et Minraux-AntioXydants) study of 12,735 healthy subjects (aged 35 to 60 years) recruited from across France in 1994 (<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366085">ref. 31</a></sup>). The Swedish Glioma Collection (SGCCS) comprises 215 glioma cases ascertained as part of the INTERPHONE Study<sup><a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366122">3</a></sup> conducted between 2000 and 2002 in Sweden, 134 cases from Umeå University Hospital, 122 from the Northern Sweden Health and Disease Study (NSHDS)<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366096">32</a></sup> and 203 from neurosurgery university clinics in Sweden. 383 controls for the INTERPHONE study were randomly selected from the population register, frequency-matched to brain tumor cases on age, sex and geographical region. In addition, 400 controls were ascertained from NSHDS that were age, sex and geographically representative of the cohort. The German Glioma Collection (GGC) comprises 564 individuals who underwent surgery for a glioma at the Department of Neurosurgery, University of Bonn Medical Center, between 1996 and 2008. All histological diagnoses were made at the Institute for Neuropathology/German Brain Tumor Reference Center, University of Bonn Medical Center. Control subjects were 576 healthy individuals (288 men, 288 women; mean age 31, range 18–45) of German origin recruited in 2004 by the Institute of Transfusion Medicine and Immunology, Mannheim.</p></div></div><div id="S7" class="sec"><h3 id="S7title">Ethics</h3><p id="P22" class="p p-first-last">Collection of blood samples and clinico-pathological information from case and control subjects was undertaken with informed consent and relevant ethical review board approval in accordance with the tenets of the Declaration of Helsinki.</p></div><div id="S8" class="sec"><h3 id="S8title">Genotyping</h3><p id="P23" class="p p-first">DNA was extracted from samples using conventional methodologies and quantified using PicoGreen (Invitrogen). Genotyping of both GWA studies was conducted by Illumina Service laboratory using the Illumina Infinium Human610-Quad BeadChips according to Illumina protocols. DNA samples with GenCall scores &lt;0.25 at any locus were considered ‘no calls’. To ensure quality of genotyping, a series of duplicate samples was genotyped in the same batches.</p><p id="P24">Intensity data from arrays was imported into Illumina's BeadStudio clustering and calling software application. For the small subset of loci that were not clustered properly by the automated algorithm, the data were reviewed to identify loci that needed to be removed, manually edited or left unchanged. Clustered SNPs were evaluated using the metrics listed in the SNP Table of the BeadStudio software. These metrics are based on all samples for each locus and thus provide overall performance information for each locus. To identify loci potentially needing to be edited or removed, each quality metric column in the SNP table was sequentially sorted. Metrics used for identifying poorly or incorrectly clustered data included intensity, cluster separation, position of each cluster (AA, AB, BB), Hardy-Weinberg equilibrium, call frequency and variation of cluster width. The reproducibility of control samples on each plate as well as replicates were also used to identify misclustered loci. Although not all cluster plots were assessed, ∼10% of the lowest performing loci were examined. Of these 10%, ∼20% were edited or annotated (to indicate loci with nearby polymorphisms or hemizygous deletions) and ∼2% were excluded. Review of data was conducted by a second individual to determine if any metrics were missed or if further editing was required. Overall, this process provided substantially increased genotyping accuracy.</p><p id="P25" class="p p-last">Subsequent genotyping of SNPs was conducted using either competitive allele-specific PCR KASPar chemistry (KBiosciences) or single-base primer extension chemistry MALDI-TOF MS detection (Sequenom). All primers and probes used are available on request. Genotyping quality control was further evaluated through inclusion of duplicate DNA samples in SNP assays. For all SNP assays, &gt;99% concordant results were obtained. Samples having SNP call rates &lt;90% were excluded from the analysis.</p></div><div id="S9" class="sec"><h3 id="S9title">Statistical analysis</h3><p id="P26" class="p p-first">Statistical analyses were undertaken using R (v2.6), STATA (v8; State College, Texas, US) and PLINK (v1.05)<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366123">33</a></sup> software. All reported <em>P</em> values are two-sided. Genotype data were used to search for duplicates and closely related individuals among all samples in each of the GWA studies. Identity-by-state (IBS) values were calculated for each pair of individuals, and for any pair with allele sharing of &gt;80%, the sample generating the lowest call rate was removed from further analysis. To identify individuals who might have non-Western European ancestry, we merged our case and control data with the 60 western European (CEU), 60 Nigerian (YRI), 90 Japanese (JPT) and 90 Han Chinese (CHB) individuals from the HapMap Project. For each pair of individuals, we calculated genome-wide IBS distances on markers shared between HapMap and our SNP panel and used these as dissimilarity measures upon which to perform principal component analysis. The first two principal components for each individual were plotted and any individual not present in the main CEU cluster (that is, outside 5% from cluster centroids) was excluded from subsequent analyses.</p><p id="P27">In the UK-GWA study, genotyped samples were excluded from analyses due to non-CEU ancestry (<em>n</em> = 3). In the US-GWA study, genotyped samples were excluded from analyses for the following reasons: relatedness (<em>n</em> = 2) and non-CEU ancestry (<em>n</em> = 23).</p><p id="P28">The adequacy of the case-control matching and possibility of differential genotyping of cases and controls were formally evaluated using quantile-quantile plots of test statistics. Deviation of the genotype frequencies in the controls from those expected under Hardy-Weinberg equilibrium (HWE) was assessed by a <em>χ</em><sup>2</sup> test or Fisher's exact test where an expected cell count was &lt; 5.</p><p id="P29">The association between each SNP and risk of glioma was assessed by the Cochran-Armitage trend test. Odds ratios and associated 95% CIs were calculated by unconditional logistic regression. Relationships between multiple SNPs showing association with glioma risk in the same region were investigated using logistic regression analysis, and the impact of additional SNPs from the same region was assessed by a likelihood-ratio test.</p><p id="P30">The combined effect of each pair of risk locus was investigated by logistic regression modeling with evidence for interactive effects between SNPs assessed by a likelihood ratio test. The OR and trend test for increasing numbers of deleterious alleles was estimated by counting two for a homozygote and one for a heterozygote.</p><p id="P31">Meta-analysis was conducted using standard methods based on weighted average of study-specific estimates of the ORs, using inverse variance weights. Cochran's <em>Q</em> statistic to test for heterogeneity and the <em>I</em><sup>2</sup> statistic to quantify the proportion of the total variation due to heterogeneity were calculated. The sibling relative risk attributable to a given SNP was calculated using the following formula:</p><div class="disp-formula" id="FD1"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><msup><mi>λ</mi><mo>∗</mo></msup><mo>=</mo><mfrac><mrow><mi>p</mi><msup><mrow><mo stretchy="false">(</mo><mi>p</mi><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mi>q</mi><msub><mi>r</mi><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><mi>q</mi><msup><mrow><mo stretchy="false">(</mo><mi>p</mi><msub><mi>r</mi><mn>1</mn></msub><mo>+</mo><mi>q</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><msup><mrow><mo stretchy="false">(</mo><msup><mi>p</mi><mn>2</mn></msup><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><mi>p</mi><mi>q</mi><msub><mi>r</mi><mn>1</mn></msub><mo>+</mo><msup><mi>q</mi><mn>2</mn></msup><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mfrac></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M1" display="block" overflow="scroll"><msup><mi>λ</mi><mo>∗</mo></msup><mo>=</mo><mfrac><mrow><mi>p</mi><msup><mrow><mo stretchy="false">(</mo><mi>p</mi><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mi>q</mi><msub><mi>r</mi><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup><mo>+</mo><mi>q</mi><msup><mrow><mo stretchy="false">(</mo><mi>p</mi><msub><mi>r</mi><mn>1</mn></msub><mo>+</mo><mi>q</mi><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow><msup><mrow><mo stretchy="false">(</mo><msup><mi>p</mi><mn>2</mn></msup><msub><mi>r</mi><mn>2</mn></msub><mo>+</mo><mn>2</mn><mi>p</mi><mi>q</mi><msub><mi>r</mi><mn>1</mn></msub><mo>+</mo><msup><mi>q</mi><mn>2</mn></msup><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mfrac></math></script></div><div class="l">(1)</div></div><p id="P32" class="p">where <em>p</em> is the population frequency of the minor allele, <em>q</em> = 1 − <em>p</em>, and <em>r</em><sub>1</sub> and <em>r</em><sub>2</sub> are the relative risks (estimated as OR) for heterozygotes and rare homozygotes relative to common homozygotes. Assuming a multiplicative interaction the proportion of the familial risk attributable to a SNP was calculated as log(<em>λ</em>*)/log(<em>λ</em><sub>0</sub>), where <em>λ</em><sub>0</sub> is the overall familial relative risk estimated from epidemiological studies, assumed to be 2.1 (<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366104">ref. 2</a></sup>). A naïve estimation of the contribution of all of the loci identified to the excess familial risk of glioma under an additive model was calculated using the following formula:</p><div class="disp-formula" id="FD2"><div class="f"><span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mfrac><mrow><msub><mtext>OR</mtext><mtext>per allele</mtext></msub><mo>−</mo><mn>1</mn></mrow><mrow><mn>2</mn><mo stretchy="false">(</mo><msub><mi>λ</mi><mn>0</mn></msub><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="M2" display="block" overflow="scroll"><mfrac><mrow><msub><mtext>OR</mtext><mtext>per allele</mtext></msub><mo>−</mo><mn>1</mn></mrow><mrow><mn>2</mn><mo stretchy="false">(</mo><msub><mi>λ</mi><mn>0</mn></msub><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></math></script></div><div class="l">(2)</div></div></div><div id="S10" class="sec"><h3 id="S10title">Bioinformatics</h3><p id="P33" class="p p-first">LD metrics between SNPs reported in HapMap were based on Data Release 2/phaseIII Feb09 on NCBI B35 assembly, dbSNPb125, except for between rs2736098 and rs2736100 and rs2853676 which were only available in Data Release 23a/phaseII March08.</p><p id="P34">We used Haploview software (v3.2) to infer the LD structure of the genome in the regions containing loci associated with glioma risk. Prediction of the untyped SNP in the case-control datasets of GWA data was carried out using IMPUTE and SNPTEST on HapMap. In total, 894 HapMap SNPs were successfully imputed in the interval between 130,051,729 bp and 131,225,253 bp using available SNP genotype data from GWA scans (225 SNPs). We verified the integrity of imputed data, where possible, by crosschecking the concordance of imputed genotypes with that of available Illumina SNP genotype data. Directly genotyped SNPs are denoted in blue; those imputed are shown in green.</p><p id="P35">To examine whether there is a relationship between SNP genotype and mRNA expression in lymphocytes and normal human cortex, we used publicly available data. We analyzed 90 CEU-derived Epstein-Barr virus–transformed lymphoblastoid cell lines using Sentrix Human-6 Expression BeadChips (Illumina). Online recovery of data was carried out using WGAViewer v1.25 Software<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366113">34</a>,<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366082">35</a></sup>. We analyzed 193 normal human cortex cells using Illumina HumanRef-seq-8 Expression BeadChip arrays<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366126">36</a></sup>. Where SNPs genotyped in our study had not been typed, HapMap data were used to identify correlated SNPs in high LD (<em>r</em><sup>2</sup> &gt; 0.8). Differences in the distribution of levels of mRNA expression between SNP genotypes were compared using a Wilcoxon-type test for trend. Relationship between copy number change and mRNA expression at loci in glioma was investigated using data from The Cancer Genome Atlas (TCGA)<sup><a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366103">18</a></sup>. The Bioconductor module CGHcall was used to assign copy number status.</p><p id="P36" class="p p-last">We searched for interactions between proteins encoded by genes mapping to association signals using the program PINA<sup><a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_462366081">37</a></sup>.</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">URLs</h3><p id="P37" class="p p-first-last">Detailed information on the tagSNP panel, <a href="http://www.illumina.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.illumina.com/</a>; WGAViewer, <a href="http://www.genome.duke.edu/centers/pg2/downloads/wgaviewer.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.duke.edu/centers/pg2/downloads/wgaviewer.php</a>; 1958 Birth Cohort: <a href="http://www.cls.ioe.ac.uk/studies.asp?section=0001000200030012" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.cls.ioe.ac.uk/studies.asp?section=0001000200030012</a>; PLINK, <a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/∼purcell/plink/</a>; Cancer Genetic Markers of Susceptibility (CGEMS), <a href="http://cgems.cancer.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov/</a>; The Cancer Genomics Data Portal, <a href="http://cbio.mskcc.org/cancergenomics-dataportal" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cbio.mskcc.org/cancergenomics-dataportal</a>; The Cancer Genome Atlas (TCGA) data portal, <a href="http://tcga-data.nci.nih.gov/tcga/homepage.htm" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://tcga-data.nci.nih.gov/tcga/homepage.htm</a>; A Survey of Genetic Human Cortical Gene Expression, <a href="http://labs.med.miami.edu/myers/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://labs.med.miami.edu/myers/</a>; IMPUTE, <a href="http://mathgen.stats.ox.ac.uk/impute/impute.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://mathgen.stats.ox.ac.uk/impute/impute.html</a>; SNPTEST, <a href="http://www.stats.ox.ac.uk/~marchini/software/gwas/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.stats.ox.ac.uk/∼marchini/software/gwas/snptest.html</a>; PINA (Protein Interaction Network Analysis platform), <a href="http://csbi.ltdk.helsinki.fi/pina/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://csbi.ltdk.helsinki.fi/pina/</a>.</p></div></div><div id="S12" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supplemenatry materials</h4><div class="sup-box half_rhythm" id="d36e1462"><a href="/pmc/articles/PMC4501476/bin/NIHMS705858-supplement-Supplemenatry_materials.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(639K, pdf)</sup></div></div></div><div id="S13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Acknowledgments</h2><div class="sec"><p id="__p2">The Wellcome Trust provided principal funding for the study. In the UK, additional funding was provided by Cancer Research UK (C1298/A8362 supported by the Bobby Moore Fund) and the European Union (CPRB LSHCCT-2004-503465). The UK and Swedish INTERPHONE studies were supported by the European Union Fifth Framework Program “Quality of Life and Management of Living Resources” (contract number QLK4-CT-1999-01563) and the International Union against Cancer (UICC). The UICC received funds for this purpose from the Mobile Manufacturers' Forum and Groupe Speciale Mobile (GSM) Association. Provision of funds via the UICC was governed by agreements that guaranteed INTERPHONE's complete scientific independence. These agreements are publicly available at <a href="http://www.iarc.fr/en/research-groups/RAD/RCAd.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.iarc.fr/en/research-groups/RAD/RCAd.html</a>. The Swedish centre was also supported by the Swedish Research Council, the Cancer Foundation of Northern Sweden, the Swedish Cancer Society, and the Nordic Cancer Union and the UK centre by the Mobile Telecommunications and Health Research (MTHR) Programme and the Health and Safety Executive, Department of Health and Safety Executive and the UK Network Operators (O<sub>2</sub>, Orange, T-Mobile, Vodafone and ‘3’). The views expressed in the publication are those of the authors and not necessarily those of the funding bodies. In the United States, funding was provided by US National Institutes of Health grants 5R01 <a id="__tag_782378016" class="tag_hotlink" href="/nuccore/CA119215" ref="/nuccore/CA119215"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA119215","term_id":"34972523","term_text":"CA119215"}}</span>CA119215</a> and 5R01 <a id="__tag_782378015" class="tag_hotlink" href="/nuccore/CA070917" ref="/nuccore/CA070917"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"CA070917","term_id":"34922843","term_text":"CA070917"}}</span>CA070917</a>. Additional support was obtained from the American Brain Tumor Association and the National Brain Tumor Society. The University of Texas M.D. Anderson Cancer Center acknowledges the work of P. Adatto, F. Morice, H. Zhang, V. Levin, A. Yung, M. Gilbert, R. Sawaya, V. Puduvalli, C. Conrad, F. Lang and J. Weinberg from the Brain and Spine Center. In France, funding was provided by the Délégation à la Recherche Clinique (MUL03012), the Association pour la Recherche sur les Tumeurs Cérébrales (ARTC), the Institut National du Cancer (INCa; PL 046) and the French Ministry of Higher Education and Research. In Germany, funding was provided to M. Simon, J.S. and M. Linnebank by the Deutsche Forschungsgemeinschaft (Si 552, Schr 285), the Deutsche Krebshilfe (70-2385-Wi2, 70-3163-Wi3, 10-6262) and BONFOR. In Sweden, the collection of samples from the Northern Sweden Health of disease study were collected with support from Umeå University Hospital and NIH funded part of GLIOGENE collection (R01 CA119215). B.M. was supported by Acta Oncologica Foundation as a research fellow at Royal Swedish Academy of Science. In the UK we acknowledge NHS funding to the NIHR Biomedical Research Centre. The UK-GWA study made use of genotyping data on the 1958 Birth Cohort. Genotyping data on controls was generated and generously supplied to us by P. Deloukas (Wellcome Trust Sanger Institute). A full list of the investigators who contributed to the generation of the data are available from <a href="http://www.wtccc.org.uk" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.wtccc.org.uk</a>. The US-GWA study made use of control genotypes from the CGEMS prostate and breast cancer studies. A full list of the investigators who contributed to the generation of the data are available from <a href="http://cgems.cancer.gov/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cgems.cancer.gov/</a>. The results published here are in whole or part based upon data generated by The Cancer Genome Atlas pilot project established by the National Cancer Institute and National Human Genome Research Institute. Information about TCGA and the investigators and institutions that constitute the TCGA research network can be found at <a href="http://cancergenome.nih.gov" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cancergenome.nih.gov</a>. Finally, we are grateful to all the study subjects for their participation. We also thank the clinicians and other hospital staff, cancer registries and study staff who contributed to the blood sample and data collection for this study.</p></div></div><div id="idm139778722082736" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139778722082736title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first-last">Note: <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary information</a> is available on the Nature Genetics website.</p><p></p><div><p id="__p4" class="p p-first-last"><strong>Author Contributions</strong>: R.S.H. and M.B. designed the study. R.S.H. drafted the manuscript, with extensive contributions from F.J.H. and S.S. F.J.H. and S.S. performed statistical analyses. S.E.D., F.J.H. and S.S. performed bioinformatics analyses. L.B.R. performed laboratory management and oversaw genotyping of UK cases and with A.P. performed genotyping of German and Swedish case-control series. In the UK, A.S., M. Schoemaker, K.M., S.J.H. and R.S.H. developed patient recruitment, sample acquisition and performed sample collection of cases. In the US, M.B. and C.L. developed protocols, M.B. and G.A. developed patient recruitment, G.A., X.G. and R.Y. performed curation and organization of samples and data, and Y.L. performed management of genotyping. In Germany, M. Simon and J.S. developed patient recruitment and blood sample collection, M. Simon oversaw DNA isolation and storage, K.H. and M. Linnebank collected control samples, M. Simon and M. Linnebank performed case ascertainment and supervision of DNA extractions, and K.H. and R.K. procured German control samples. In France, M. Sanson, J.-Y.D., K.H.-X. and A.I. developed patient recruitment, and Y.M., B.B. and S. El-H. developed sample acquisition and performed sample collection of cases. M. Lathrop and D.Z. performed laboratory management and oversaw genotyping of the French samples. In Sweden, for the Swedish INTERPHONE Study, M.F., S.L. and A.A. developed study design and conducted patient recruitment and control selection, M.F., S.L., A.A., B.M. and R.H. organized sample acquisition and performed sample collection of case and controls, U.A. coordinated sample collection and complied information into data files of cases and controls for statistical analyses, and B.M. and R.H. performed laboratory management and oversaw DNA extraction. The NSHDS samples were collected by Umeå University (Principal Investigator Göran Hallmans), and the additional samples were collected at the neurosurgery department in Umeå from 2005 and onwards (A.T.B. and R.H.) and through the national GLIOGENE study (Principal Investigator B.M.). All authors contributed to the final paper.</p></div></div></div><div id="idm139778722076912" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139778722076912title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Bondy ML, et al.  Brain tumor epidemiology: consensus from the Brain Tumor Epidemiology Consortium. <span><span class="ref-journal">Cancer. </span>2008;<span class="ref-vol">113</span>:1953–1968.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2861559/">PMC free article</a>]</span> [<a href="/pubmed/18798534" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=Brain+tumor+epidemiology:+consensus+from+the+Brain+Tumor+Epidemiology+Consortium&amp;author=ML+Bondy&amp;volume=113&amp;publication_year=2008&amp;pages=1953-1968&amp;pmid=18798534&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Hemminki K, Li X. Familial risks in nervous system tumors. <span><span class="ref-journal">Cancer Epidemiol Biomarkers Prev. </span>2003;<span class="ref-vol">12</span>:1137–1142.</span> [<a href="/pubmed/14652272" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol+Biomarkers+Prev&amp;title=Familial+risks+in+nervous+system+tumors&amp;author=K+Hemminki&amp;author=X+Li&amp;volume=12&amp;publication_year=2003&amp;pages=1137-1142&amp;pmid=14652272&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Cardis E, et al.  The INTERPHONE study: design, epidemiological methods, and description of the study population. <span><span class="ref-journal">Eur J Epidemiol. </span>2007;<span class="ref-vol">22</span>:647–664.</span> [<a href="/pubmed/17636416" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Epidemiol&amp;title=The+INTERPHONE+study:+design,+epidemiological+methods,+and+description+of+the+study+population&amp;author=E+Cardis&amp;volume=22&amp;publication_year=2007&amp;pages=647-664&amp;pmid=17636416&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Power C, Elliott J. Cohort profile: 1958 British birth cohort (National Child Development Study) <span><span class="ref-journal">Int J Epidemiol. </span>2006;<span class="ref-vol">35</span>:34–41.</span> [<a href="/pubmed/16155052" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Cohort+profile:+1958+British+birth+cohort+(National+Child+Development+Study)&amp;author=C+Power&amp;author=J+Elliott&amp;volume=35&amp;publication_year=2006&amp;pages=34-41&amp;pmid=16155052&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Hunter DJ, et al.  A genome-wide association study identifies alleles in <em>FGFR2</em> associated with risk of sporadic postmenopausal breast cancer. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:870–874.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3493132/">PMC free article</a>]</span> [<a href="/pubmed/17529973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+study+identifies+alleles+in+FGFR2+associated+with+risk+of+sporadic+postmenopausal+breast+cancer&amp;author=DJ+Hunter&amp;volume=39&amp;publication_year=2007&amp;pages=870-874&amp;pmid=17529973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Yeager M, et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Clayton DG, et al.  Population structure, differential bias and genomic control in a large-scale, case-control association study. <span><span class="ref-journal">Nat Genet. </span>2005;<span class="ref-vol">37</span>:1243–1246.</span> [<a href="/pubmed/16228001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Population+structure,+differential+bias+and+genomic+control+in+a+large-scale,+case-control+association+study&amp;author=DG+Clayton&amp;volume=37&amp;publication_year=2005&amp;pages=1243-1246&amp;pmid=16228001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Yin W, Rossin A, Clifford JL, Gronemeyer H. Co-resistance to retinoic acid and TRAIL by insertion mutagenesis into RAM. <span><span class="ref-journal">Oncogene. </span>2006;<span class="ref-vol">25</span>:3735–3744.</span> [<a href="/pubmed/16449964" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Co-resistance+to+retinoic+acid+and+TRAIL+by+insertion+mutagenesis+into+RAM&amp;author=W+Yin&amp;author=A+Rossin&amp;author=JL+Clifford&amp;author=H+Gronemeyer&amp;volume=25&amp;publication_year=2006&amp;pages=3735-3744&amp;pmid=16449964&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Jiang M, Zhu K, Grenet J, Lahti JM. Retinoic acid induces caspase-8 transcription via phospho-CREB and increases apoptotic responses to death stimuli in neuroblastoma cells. <span><span class="ref-journal">Biochim Biophys Acta. </span>2008;<span class="ref-vol">1783</span>:1055–1067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2474548/">PMC free article</a>]</span> [<a href="/pubmed/18342014" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochim+Biophys+Acta&amp;title=Retinoic+acid+induces+caspase-8+transcription+via+phospho-CREB+and+increases+apoptotic+responses+to+death+stimuli+in+neuroblastoma+cells&amp;author=M+Jiang&amp;author=K+Zhu&amp;author=J+Grenet&amp;author=JM+Lahti&amp;volume=1783&amp;publication_year=2008&amp;pages=1055-1067&amp;pmid=18342014&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Das A, Banik NL, Ray SK. Differentiation decreased telomerase activity in rat glioblastoma C6 cells and increased sensitivity to IFN-γ and taxol for apoptosis. <span><span class="ref-journal">Neurochem Res. </span>2007;<span class="ref-vol">32</span>:2167–2183.</span> [<a href="/pubmed/17694433" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem+Res&amp;title=Differentiation+decreased+telomerase+activity+in+rat+glioblastoma+C6+cells+and+increased+sensitivity+to+IFN-γ+and+taxol+for+apoptosis&amp;author=A+Das&amp;author=NL+Banik&amp;author=SK+Ray&amp;volume=32&amp;publication_year=2007&amp;pages=2167-2183&amp;pmid=17694433&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Tomlinson I, et al.  A genome-wide association scan of tag SNPs identifies a susceptibility variant for colorectal cancer at 8q24.21. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:984–988.</span> [<a href="/pubmed/17618284" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+scan+of+tag+SNPs+identifies+a+susceptibility+variant+for+colorectal+cancer+at+8q24.21&amp;author=I+Tomlinson&amp;volume=39&amp;publication_year=2007&amp;pages=984-988&amp;pmid=17618284&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Easton DF, et al.  Genome-wide association study identifies novel breast cancer susceptibility loci. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">447</span>:1087–1093.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2714974/">PMC free article</a>]</span> [<a href="/pubmed/17529967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+identifies+novel+breast+cancer+susceptibility+loci&amp;author=DF+Easton&amp;volume=447&amp;publication_year=2007&amp;pages=1087-1093&amp;pmid=17529967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Kiemeney LA, et al.  Sequence variant on 8q24 confers susceptibility to urinary bladder cancer. <span><span class="ref-journal">Nat Genet. </span>2008;<span class="ref-vol">40</span>:1307–1312.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4539560/">PMC free article</a>]</span> [<a href="/pubmed/18794855" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variant+on+8q24+confers+susceptibility+to+urinary+bladder+cancer&amp;author=LA+Kiemeney&amp;volume=40&amp;publication_year=2008&amp;pages=1307-1312&amp;pmid=18794855&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Birnbaum S, et al.  Key susceptibility locus for nonsyndromic cleft lip with or without cleft palate on chromosome 8q24. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:473–477.</span> [<a href="/pubmed/19270707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Key+susceptibility+locus+for+nonsyndromic+cleft+lip+with+or+without+cleft+palate+on+chromosome+8q24&amp;author=S+Birnbaum&amp;volume=41&amp;publication_year=2009&amp;pages=473-477&amp;pmid=19270707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Bille C, et al.  Cancer risk in persons with oral cleft—a population-based study of 8,093 cases. <span><span class="ref-journal">Am J Epidemiol. </span>2005;<span class="ref-vol">161</span>:1047–1055.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2839121/">PMC free article</a>]</span> [<a href="/pubmed/15901625" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Cancer+risk+in+persons+with+oral+cleft—a+population-based+study+of+8,093+cases&amp;author=C+Bille&amp;volume=161&amp;publication_year=2005&amp;pages=1047-1055&amp;pmid=15901625&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Wager M, et al.  Prognostic molecular markers with no impact on decision-making: the paradox of gliomas based on a prospective study. <span><span class="ref-journal">Br J Cancer. </span>2008;<span class="ref-vol">98</span>:1830–1838.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2410116/">PMC free article</a>]</span> [<a href="/pubmed/18506188" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Cancer&amp;title=Prognostic+molecular+markers+with+no+impact+on+decision-making:+the+paradox+of+gliomas+based+on+a+prospective+study&amp;author=M+Wager&amp;volume=98&amp;publication_year=2008&amp;pages=1830-1838&amp;pmid=18506188&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Rafnar T, et al.  Sequence variants at the <em>TERT-CLPTM1L</em> locus associate with many cancer types. <span><span class="ref-journal">Nat Genet. </span>2009;<span class="ref-vol">41</span>:221–227.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4525478/">PMC free article</a>]</span> [<a href="/pubmed/19151717" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Sequence+variants+at+the+TERT-CLPTM1L+locus+associate+with+many+cancer+types&amp;author=T+Rafnar&amp;volume=41&amp;publication_year=2009&amp;pages=221-227&amp;pmid=19151717&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Cancer Genome Atlas Research Network. Comprehensive genomic characterization defines human glioblastoma genes and core pathways. <span><span class="ref-journal">Nature. </span>2008;<span class="ref-vol">455</span>:1061–1068.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2671642/">PMC free article</a>]</span> [<a href="/pubmed/18772890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive+genomic+characterization+defines+human+glioblastoma+genes+and+core+pathways&amp;volume=455&amp;publication_year=2008&amp;pages=1061-1068&amp;pmid=18772890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Parsons DW, et al.  An integrated genomic analysis of human glioblastoma multiforme. <span><span class="ref-journal">Science. </span>2008;<span class="ref-vol">321</span>:1807–1812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820389/">PMC free article</a>]</span> [<a href="/pubmed/18772396" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An+integrated+genomic+analysis+of+human+glioblastoma+multiforme&amp;author=DW+Parsons&amp;volume=321&amp;publication_year=2008&amp;pages=1807-1812&amp;pmid=18772396&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Randerson-Moor JA, et al.  A germline deletion of p14(<em>ARF</em>) but not <em>CDKN2A</em> in a melanoma-neural system tumor syndrome family. <span><span class="ref-journal">Hum Mol Genet. </span>2001;<span class="ref-vol">10</span>:55–62.</span> [<a href="/pubmed/11136714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=A+germline+deletion+of+p14(ARF)+but+not+CDKN2A+in+a+melanoma-neural+system+tumor+syndrome+family&amp;author=JA+Randerson-Moor&amp;volume=10&amp;publication_year=2001&amp;pages=55-62&amp;pmid=11136714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Bahuau M, et al.  Germ-line deletion involving the <em>INK4</em> locus in familial proneness to melanoma and nervous system tumors. <span><span class="ref-journal">Cancer Res. </span>1998;<span class="ref-vol">58</span>:2298–2303.</span> [<a href="/pubmed/9622062" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res&amp;title=Germ-line+deletion+involving+the+INK4+locus+in+familial+proneness+to+melanoma+and+nervous+system+tumors&amp;author=M+Bahuau&amp;volume=58&amp;publication_year=1998&amp;pages=2298-2303&amp;pmid=9622062&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Simon M, Voss D, Park-Simon TW, Mahlberg R, Koster G. Role of p16 and p14ARF in radio- and chemosensitivity of malignant gliomas. <span><span class="ref-journal">Oncol Rep. </span>2006;<span class="ref-vol">16</span>:127–132.</span> [<a href="/pubmed/16786135" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol+Rep&amp;title=Role+of+p16+and+p14ARF+in+radio-+and+chemosensitivity+of+malignant+gliomas&amp;author=M+Simon&amp;author=D+Voss&amp;author=TW+Park-Simon&amp;author=R+Mahlberg&amp;author=G+Koster&amp;volume=16&amp;publication_year=2006&amp;pages=127-132&amp;pmid=16786135&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Scott LJ, et al.  A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1341–1345.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;volume=316&amp;publication_year=2007&amp;pages=1341-1345&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">McPherson R, et al.  A common allele on chromosome 9 associated with coronary heart disease. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">316</span>:1488–1491.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2711874/">PMC free article</a>]</span> [<a href="/pubmed/17478681" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+common+allele+on+chromosome+9+associated+with+coronary+heart+disease&amp;author=R+McPherson&amp;volume=316&amp;publication_year=2007&amp;pages=1488-1491&amp;pmid=17478681&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Harder T, Plagemann A, Harder A. Birth weight and subsequent risk of childhood primary brain tumors: a meta-analysis. <span><span class="ref-journal">Am J Epidemiol. </span>2008;<span class="ref-vol">168</span>:366–373.</span> [<a href="/pubmed/18579539" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Epidemiol&amp;title=Birth+weight+and+subsequent+risk+of+childhood+primary+brain+tumors:+a+meta-analysis&amp;author=T+Harder&amp;author=A+Plagemann&amp;author=A+Harder&amp;volume=168&amp;publication_year=2008&amp;pages=366-373&amp;pmid=18579539&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Ferrie JE, Langenberg C, Shipley MJ, Marmot MG. Birth weight, components of height and coronary heart disease: evidence from the Whitehall II study. <span><span class="ref-journal">Int J Epidemiol. </span>2006;<span class="ref-vol">35</span>:1532–1542.</span> [<a href="/pubmed/16931524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int+J+Epidemiol&amp;title=Birth+weight,+components+of+height+and+coronary+heart+disease:+evidence+from+the+Whitehall+II+study&amp;author=JE+Ferrie&amp;author=C+Langenberg&amp;author=MJ+Shipley&amp;author=MG+Marmot&amp;volume=35&amp;publication_year=2006&amp;pages=1532-1542&amp;pmid=16931524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Whincup PH, et al.  Birth weight and risk of type 2 diabetes: a systematic review. <span><span class="ref-journal">J Am Med Assoc. </span>2008;<span class="ref-vol">300</span>:2886–2897.</span> [<a href="/pubmed/19109117" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Assoc&amp;title=Birth+weight+and+risk+of+type+2+diabetes:+a+systematic+review&amp;author=PH+Whincup&amp;volume=300&amp;publication_year=2008&amp;pages=2886-2897&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Barber LJ, et al.  RTEL1 maintains genomic stability by suppressing homologous recombination. <span><span class="ref-journal">Cell. </span>2008;<span class="ref-vol">135</span>:261–271.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3726190/">PMC free article</a>]</span> [<a href="/pubmed/18957201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=RTEL1+maintains+genomic+stability+by+suppressing+homologous+recombination&amp;author=LJ+Barber&amp;volume=135&amp;publication_year=2008&amp;pages=261-271&amp;pmid=18957201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Arakawa Y, et al.  Frequent gene amplification and overexpression of decoy receptor 3 in glioblastoma. <span><span class="ref-journal">Acta Neuropathol. </span>2005;<span class="ref-vol">109</span>:294–298.</span> [<a href="/pubmed/15627206" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Neuropathol&amp;title=Frequent+gene+amplification+and+overexpression+of+decoy+receptor+3+in+glioblastoma&amp;author=Y+Arakawa&amp;volume=109&amp;publication_year=2005&amp;pages=294-298&amp;pmid=15627206&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Guo C, et al.  Allelic deletion at 11q23 is common in <em>MYCN</em> single copy neuroblastomas. <span><span class="ref-journal">Oncogene. </span>1999;<span class="ref-vol">18</span>:4948–4957.</span> [<a href="/pubmed/10490829" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Allelic+deletion+at+11q23+is+common+in+MYCN+single+copy+neuroblastomas&amp;author=C+Guo&amp;volume=18&amp;publication_year=1999&amp;pages=4948-4957&amp;pmid=10490829&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Hercberg S, et al.  The SU.VI.MAX Study: a randomized, placebo-controlled trial of the health effects of antioxidant vitamins and minerals. <span><span class="ref-journal">Arch Intern Med. </span>2004;<span class="ref-vol">164</span>:2335–2342.</span> [<a href="/pubmed/15557412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=The+SU.VI.MAX+Study:+a+randomized,+placebo-controlled+trial+of+the+health+effects+of+antioxidant+vitamins+and+minerals&amp;author=S+Hercberg&amp;volume=164&amp;publication_year=2004&amp;pages=2335-2342&amp;pmid=15557412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Hallmans G, et al.  Cardiovascular disease and diabetes in the Northern Sweden Health and Disease Study Cohort—evaluation of risk factors and their interactions. <span><span class="ref-journal">Scand J Public Health Suppl. </span>2003;<span class="ref-vol">61</span>:18–24.</span> [<a href="/pubmed/14660243" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Scand+J+Public+Health+Suppl&amp;title=Cardiovascular+disease+and+diabetes+in+the+Northern+Sweden+Health+and+Disease+Study+Cohort—evaluation+of+risk+factors+and+their+interactions&amp;author=G+Hallmans&amp;volume=61&amp;publication_year=2003&amp;pages=18-24&amp;pmid=14660243&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Stranger BE, et al.  Genome-wide associations of gene expression variation in humans. <span><span class="ref-journal">PLoS Genet. </span>2005;<span class="ref-vol">1</span>:e78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1315281/">PMC free article</a>]</span> [<a href="/pubmed/16362079" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+associations+of+gene+expression+variation+in+humans&amp;author=BE+Stranger&amp;volume=1&amp;publication_year=2005&amp;pages=e78&amp;pmid=16362079&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Stranger BE, et al.  Relative impact of nucleotide and copy number variation on gene expression phenotypes. <span><span class="ref-journal">Science. </span>2007;<span class="ref-vol">315</span>:848–853.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2665772/">PMC free article</a>]</span> [<a href="/pubmed/17289997" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Relative+impact+of+nucleotide+and+copy+number+variation+on+gene+expression+phenotypes&amp;author=BE+Stranger&amp;volume=315&amp;publication_year=2007&amp;pages=848-853&amp;pmid=17289997&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Myers AJ, et al.  A survey of genetic human cortical gene expression. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1494–1499.</span> [<a href="/pubmed/17982457" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+survey+of+genetic+human+cortical+gene+expression&amp;author=AJ+Myers&amp;volume=39&amp;publication_year=2007&amp;pages=1494-1499&amp;pmid=17982457&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Wu J, et al.  Integrated network analysis platform for protein-protein interactions. <span><span class="ref-journal">Nat Methods. </span>2009;<span class="ref-vol">6</span>:75–77.</span> [<a href="/pubmed/19079255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=Integrated+network+analysis+platform+for+protein-protein+interactions&amp;author=J+Wu&amp;volume=6&amp;publication_year=2009&amp;pages=75-77&amp;pmid=19079255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4501476&amp;issue-id=255314&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139778727169872title">Abstract</a></li><li><a href="#S2title">Online Methods</a></li><li><a href="#S12title">Supplementary Material</a></li><li><a href="#S13title">Acknowledgments</a></li><li><a href="#idm139778722082736title">Footnotes</a></li><li><a href="#idm139778722076912title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4501476/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4501476/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4501476/pdf/nihms705858.pdf">PDF (684K)</a></li> | <li><a href="#" data-citationid="PMC4501476" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4501476%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4501476%2F&amp;text=Genome-wide%20association%20study%20identifies%20five%20susceptibility%20loci%20for%20glioma" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4501476%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="19578367" data-aiid="4501476" data-aid="4501476" data-iid="255314" data-domainid="319" data-domain="nihpa" data-accid="PMC4501476" data-md5="613c169a1edb813845cf0afd587f9bd4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T03:55:54-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE89A6DEDD4F5911000000000657041D&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4501476%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4501476/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>