<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4747422/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Identification of a New Susceptibility Locus for Systemic Lupus Erythematosus on Chromosome 12 in Individuals of European Ancestry</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747422/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Arthritis &amp; rheumatology (Hoboken, N.J.)"><meta name="citation_title" content="Identification of a New Susceptibility Locus for Systemic Lupus Erythematosus on Chromosome 12 in Individuals of European Ancestry"><meta name="citation_authors" content="F. Yesim Demirci, Xingbin Wang, Jennifer A. Kelly, David L. Morris, M. Michael Barmada, Eleanor Feingold, Amy H. Kao, Kathy L. Sivils, Sasha Bernatsky, Christian Pineau, Ann Clarke, Rosalind Ramsey-Goldman, Timothy J. Vyse, Patrick M. Gaffney, Susan Manzi, M. Ilyas Kamboh"><meta name="citation_date" content="January 2016"><meta name="citation_issue" content="1"><meta name="citation_volume" content="68"><meta name="citation_firstpage" content="174"><meta name="citation_doi" content="10.1002/art.39403"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4747422/?report=abstract"><meta name="citation_pmid" content="26316170"><meta name="DC.Title" content="Identification of a New Susceptibility Locus for Systemic Lupus Erythematosus on Chromosome 12 in Individuals of European Ancestry"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="F. Yesim Demirci"><meta name="DC.Contributor" content="Xingbin Wang"><meta name="DC.Contributor" content="Jennifer A. Kelly"><meta name="DC.Contributor" content="David L. Morris"><meta name="DC.Contributor" content="M. Michael Barmada"><meta name="DC.Contributor" content="Eleanor Feingold"><meta name="DC.Contributor" content="Amy H. Kao"><meta name="DC.Contributor" content="Kathy L. Sivils"><meta name="DC.Contributor" content="Sasha Bernatsky"><meta name="DC.Contributor" content="Christian Pineau"><meta name="DC.Contributor" content="Ann Clarke"><meta name="DC.Contributor" content="Rosalind Ramsey-Goldman"><meta name="DC.Contributor" content="Timothy J. Vyse"><meta name="DC.Contributor" content="Patrick M. Gaffney"><meta name="DC.Contributor" content="Susan Manzi"><meta name="DC.Contributor" content="M. Ilyas Kamboh"><meta name="DC.Date" content="2016 Jan"><meta name="DC.Identifier" content="10.1002/art.39403"><meta name="DC.Language" content="en"><meta property="og:title" content="Identification of a New Susceptibility Locus for Systemic Lupus Erythematosus on Chromosome 12 in Individuals of European Ancestry"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWASs) in individuals of European ancestry identified a number of systemic lupus erythematosus (SLE) susceptibility loci using earlier versions of high-density genotyping platforms. Follow-up studies on suggestive GWAS ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747422/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4747422/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4747422/pdf/nihms751533.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A18DEDD585D510000000005170307.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747422%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4747422</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Arthritis Rheumatol</a></span>. Author manuscript; available in PMC 2017 Jan 1.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26316170" target="pmc_ext" ref="reftype=publisher&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access"><span class="cit">Arthritis Rheumatol. 2016 Jan; 68(1): 174–183. </span></a></div> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fart.39403" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/art.39403</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4747422</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS751533</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26316170">26316170</a></div></div></div></div><h1 class="content-title">Identification of a New Susceptibility Locus for Systemic Lupus Erythematosus on Chromosome 12 in Individuals of European Ancestry</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Demirci%20FY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640788686368" co-class="co-affbox">F. Yesim Demirci</a>,<sup>1</sup> <a href="/pubmed/?term=Wang%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640785409568" co-class="co-affbox">Xingbin Wang</a>,<sup>1</sup> <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640784967104" co-class="co-affbox">Jennifer A. Kelly</a>,<sup>2</sup> <a href="/pubmed/?term=Morris%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640767857520" co-class="co-affbox">David L. Morris</a>,<sup>3</sup> <a href="/pubmed/?term=Barmada%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640771132448" co-class="co-affbox">M. Michael Barmada</a>,<sup>1</sup> <a href="/pubmed/?term=Feingold%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640771130576" co-class="co-affbox">Eleanor Feingold</a>,<sup>1</sup> <a href="/pubmed/?term=Kao%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640786948016" co-class="co-affbox">Amy H. Kao</a>,<sup>4</sup> <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640782627248" co-class="co-affbox">Kathy L. Sivils</a>,<sup>2</sup> <a href="/pubmed/?term=Bernatsky%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640782625264" co-class="co-affbox">Sasha Bernatsky</a>,<sup>5</sup> <a href="/pubmed/?term=Pineau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640784699328" co-class="co-affbox">Christian Pineau</a>,<sup>5</sup> <a href="/pubmed/?term=Clarke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640790463472" co-class="co-affbox">Ann Clarke</a>,<sup>6</sup> <a href="/pubmed/?term=Ramsey-Goldman%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640792194656" co-class="co-affbox">Rosalind Ramsey-Goldman</a>,<sup>7</sup> <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640792192672" co-class="co-affbox">Timothy J. Vyse</a>,<sup>3</sup> <a href="/pubmed/?term=Gaffney%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640771140880" co-class="co-affbox">Patrick M. Gaffney</a>,<sup>2</sup> <a href="/pubmed/?term=Manzi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640786065744" co-class="co-affbox">Susan Manzi</a>,<sup>4</sup> and  <a href="/pubmed/?term=Kamboh%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170" class="affpopup" co-rid="_co_idm140640782377120" co-class="co-affbox">M. Ilyas Kamboh</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140640788686368"><h3 class="no_margin">F. Yesim Demirci</h3><p><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Demirci%20FY%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">F. Yesim Demirci</a></div></div><div id="_co_idm140640785409568"><h3 class="no_margin">Xingbin Wang</h3><p><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Wang%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Xingbin Wang</a></div></div><div id="_co_idm140640784967104"><h3 class="no_margin">Jennifer A. Kelly</h3><p><sup>2</sup>Arthritis &amp; Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Kelly%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Jennifer A. Kelly</a></div></div><div id="_co_idm140640767857520"><h3 class="no_margin">David L. Morris</h3><p><sup>3</sup>Department of Medical &amp; Molecular Genetics, King's College London, Guy's Hospital, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Morris%20DL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">David L. Morris</a></div></div><div id="_co_idm140640771132448"><h3 class="no_margin">M. Michael Barmada</h3><p><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Barmada%20MM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">M. Michael Barmada</a></div></div><div id="_co_idm140640771130576"><h3 class="no_margin">Eleanor Feingold</h3><p><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Feingold%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Eleanor Feingold</a></div></div><div id="_co_idm140640786948016"><h3 class="no_margin">Amy H. Kao</h3><p><sup>4</sup>Lupus Center of Excellence, Department of Medicine, Allegheny Health Network, Pittsburgh, PA 15224, USA</p><div>Find articles by <a href="/pubmed/?term=Kao%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Amy H. Kao</a></div></div><div id="_co_idm140640782627248"><h3 class="no_margin">Kathy L. Sivils</h3><p><sup>2</sup>Arthritis &amp; Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Sivils%20KL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Kathy L. Sivils</a></div></div><div id="_co_idm140640782625264"><h3 class="no_margin">Sasha Bernatsky</h3><p><sup>5</sup>Division of Rheumatology, Department of Medicine, McGill University, Montreal, QC H3G 1A4, Canada</p><div>Find articles by <a href="/pubmed/?term=Bernatsky%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Sasha Bernatsky</a></div></div><div id="_co_idm140640784699328"><h3 class="no_margin">Christian Pineau</h3><p><sup>5</sup>Division of Rheumatology, Department of Medicine, McGill University, Montreal, QC H3G 1A4, Canada</p><div>Find articles by <a href="/pubmed/?term=Pineau%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Christian Pineau</a></div></div><div id="_co_idm140640790463472"><h3 class="no_margin">Ann Clarke</h3><p><sup>6</sup>Division of Rheumatology, Department of Medicine, University of Calgary, Calgary, AB T2N 4Z6, Canada</p><div>Find articles by <a href="/pubmed/?term=Clarke%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Ann Clarke</a></div></div><div id="_co_idm140640792194656"><h3 class="no_margin">Rosalind Ramsey-Goldman</h3><p><sup>7</sup>Division of Rheumatology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611, USA</p><div>Find articles by <a href="/pubmed/?term=Ramsey-Goldman%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Rosalind Ramsey-Goldman</a></div></div><div id="_co_idm140640792192672"><h3 class="no_margin">Timothy J. Vyse</h3><p><sup>3</sup>Department of Medical &amp; Molecular Genetics, King's College London, Guy's Hospital, London SE1 9RT, UK</p><div>Find articles by <a href="/pubmed/?term=Vyse%20TJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Timothy J. Vyse</a></div></div><div id="_co_idm140640771140880"><h3 class="no_margin">Patrick M. Gaffney</h3><p><sup>2</sup>Arthritis &amp; Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</p><div>Find articles by <a href="/pubmed/?term=Gaffney%20PM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Patrick M. Gaffney</a></div></div><div id="_co_idm140640786065744"><h3 class="no_margin">Susan Manzi</h3><p><sup>4</sup>Lupus Center of Excellence, Department of Medicine, Allegheny Health Network, Pittsburgh, PA 15224, USA</p><div>Find articles by <a href="/pubmed/?term=Manzi%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">Susan Manzi</a></div></div><div id="_co_idm140640782377120"><h3 class="no_margin">M. Ilyas Kamboh</h3><p><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</p><div>Find articles by <a href="/pubmed/?term=Kamboh%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26316170">M. Ilyas Kamboh</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140640823473184_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140640823473184_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140640823473184_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Arthritis &amp; Clinical Immunology Research Program, Oklahoma Medical Research Foundation, Oklahoma City, OK 73104, USA</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Department of Medical &amp; Molecular Genetics, King's College London, Guy's Hospital, London SE1 9RT, UK</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Lupus Center of Excellence, Department of Medicine, Allegheny Health Network, Pittsburgh, PA 15224, USA</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Division of Rheumatology, Department of Medicine, McGill University, Montreal, QC H3G 1A4, Canada</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Division of Rheumatology, Department of Medicine, University of Calgary, Calgary, AB T2N 4Z6, Canada</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Division of Rheumatology, Feinberg School of Medicine, Northwestern University, Chicago, IL 60611, USA</div><div id="cor1">CORRESPONDENCE: F. Yesim Demirci, MD (<a href="mailto:dev@null" data-email="ude.ttip@1dyf" class="oemail">ude.ttip@1dyf</a>, phone: 412-383-7193, fax: 412-383-7844) or M. Ilyas Kamboh, PhD (<a href="mailto:dev@null" data-email="ude.ttip@hobmak" class="oemail">ude.ttip@hobmak</a>, phone: 412-624-3066, fax: 412-624-3020), Department of Human Genetics, Graduate School of Public Health, University of Pittsburgh, Pittsburgh, PA 15261, USA</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140640823473184_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26316170" target="pmc_ext" ref="reftype=publisher&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Arthritis Rheumatol</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4747422/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Info.</strong></div><div><a href="/pmc/articles/PMC4747422/bin/NIHMS751533-supplement-Supp_Info.pdf" data-ga-action="click_feat_suppl">NIHMS751533-supplement-Supp_Info.pdf</a><span style="color:gray"> (3.3M)</span></div><div class="small guid">GUID:&nbsp;1C27558E-D512-451B-B978-FD8A1D25C1FE</div></div></dd></dl></div><div id="idm140640790238576" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140640790238576title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Objective</h3><p id="P1" class="p p-first-last">Genome-wide association studies (GWASs) in individuals of European ancestry identified a number of systemic lupus erythematosus (SLE) susceptibility loci using earlier versions of high-density genotyping platforms. Follow-up studies on suggestive GWAS regions using larger samples and more markers identified additional SLE loci in European-descent subjects. Here we report the results of a multi-stage study that we performed to identify novel SLE loci.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">In Stage 1, we conducted a new GWAS of SLE in a North American case-control sample of European ancestry (<em>n</em>=1,166) genotyped on Affymetrix Genome-Wide Human SNP Array 6.0. In Stage 2, we further investigated top new suggestive GWAS hits by <em>in silico</em> evaluation and meta-analysis using an additional dataset of European-descent subjects (&gt;2,500 individuals), followed by replication of top meta-analysis findings in another dataset of European-descent subjects (&gt;10,000 individuals) in Stage 3.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">As expected, our GWAS revealed most significant associations at the major histocompatibility complex locus (6p21), which easily surpassed genome-wide significance threshold (<em>P</em>&lt;5×10<sup>−8</sup>). Several other SLE signals/loci previously implicated in Caucasians and/or Asians were also supported in Stage 1 discovery sample and strongest signals were observed at 2q32/<em>STAT4</em> (<em>P</em>=3.6×10<sup>−7</sup>) and at 8p23/<em>BLK</em> (<em>P</em>=8.1×10<sup>−6</sup>). Stage 2 meta-analyses identified a new genome-wide significant SLE locus at 12q12 (meta <em>P</em>=3.1×10<sup>−8</sup>), which was replicated in Stage 3.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusion</h3><p id="P4" class="p p-first-last">Our multi-stage study identified and replicated a new SLE locus that warrants further follow-up in additional studies. Publicly available databases suggest that this new SLE signal falls within a functionally relevant genomic region and near biologically important genes.</p></div></div></div><div id="idm140640768039536" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Systemic lupus erythematosus (SLE) is a prototypic multisystem autoimmune disease that is influenced by a complex interplay of heritable, hormonal, and environmental factors. Genetic basis of SLE was initially suggested by heritability estimates (~66%), familial clustering (sibling recurrence risk ratio: ~30), and twin studies (~10 times higher concordance rate in monozygotic vs. dizygotic twins) and subsequently supported by a growing number of susceptibility loci identified by association studies (<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198775">1</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198790">5</a>).</p><p id="P6">SLE is clinically highly heterogeneous and variable in disease course and prognosis. Underlying immunopathologic mechanisms also appear to be highly heterogeneous, involving various abnormalities in both innate and adaptive immune responses. Therefore, a large number of genetic factors are expected to influence SLE risk and several susceptibility loci have been identified to date through candidate gene and/or genome-wide association (GWA) studies (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198776">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198773">5</a>).</p><p id="P7">GWA studies (GWASs) in European-descent subjects identified a number of new SLE loci and also confirmed some previously known loci, using earlier versions of high-density GWAS platforms. Three high-density GWASs of SLE were initially published in European-descent subjects (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198796">6</a>–<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198793">8</a>) using the following platforms: Illumina HumanHap550 BeadChip (1311 cases, 1783 controls, and public data from 1557 controls), Illumina HumanHap300 BeadChip (720 cases, 475 controls, and public data from 1862 controls) and Affymetrix Human SNP Array 5.0 (431 cases and public data from 2155 controls). Follow-up studies on suggestive GWAS regions using larger samples and more markers identified additional SLE loci in European-derived populations (<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198765">9</a>–<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198805">15</a>). Although the data from initial GWASs have been extensively used for follow-up studies and additional analyses, no other ‘new’ high-density GWAS of SLE in European-derived populations (using new or extended samples genotyped on new platforms) has appeared in PubMed since 2008; however, a number of them were published from Asian populations (<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198804">16</a>–<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198807">20</a>) and the publication of a large European GWAS is underway (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198767">21</a>, <a href="#R22" rid="R22" class=" bibr popnode">22</a>). Despite the tremendous progress, a substantial portion of genetic component of SLE still remains to be uncovered by additional studies/discoveries.</p><p id="P8" class="p">Given the heterogeneous nature of SLE and coverage differences between GWAS platforms, investigation of different and/or larger samples using different and/or higher density platforms is likely to increase the chance of discovering additional loci, as already exemplified by initial GWASs (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198772">6</a>–<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198774">8</a>). Here we report the results of a multi-stage study that involved (1) a new GWAS of SLE in a North American case-control discovery sample of European ancestry (<em>n</em>=1,166) genotyped on Affymetrix Genome-Wide Human SNP Array 6.0 (Affy 6.0), (2) follow-up evaluation and meta-analysis of top new findings using an additional dataset of European-descent subjects (&gt;2,500 individuals), and (3) replication of the most promising meta-analysis findings in another dataset of European-descent subjects (&gt;10,000 individuals). Several previously reported SLE loci were supported in Stage 1 discovery sample, and Stage 2 meta-analyses implicated a new genome-wide significant locus at 12q12 (meta <em>P</em>=3.1×10<sup>−8</sup>), which was replicated in Stage 3.</p></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">MATERIALS AND METHODS</h2><div id="S6" class="sec sec-first"><h3 id="S6title">Stage 1 discovery sample for GWAS</h3><p id="P9" class="p p-first-last">A total of 1,166 European-descent subjects (676 SLE cases and 490 controls) were included in GWAS, after excluding subjects with cryptic relationship (one from each pair of duplicates or close relatives) from ~1,200 individuals initially genotyped. Participants were recruited at three sites [Pittsburgh (<em>n</em>=750), Chicago (<em>n</em>=204), and Montreal (<em>n</em>=212)] and all patients met 1982 or revised 1997 American College of Rheumatology classification criteria for SLE (<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198800">23</a>, <a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198791">24</a>) as determined by treating rheumatologist at each site. SLE cases were 18 years or older (mean age 45.1 ± 12.5 years; 97.3% women) and the controls were 21 years or older (mean age 48.9 ± 10.6 years; 100% women). Detailed information on these samples can be found elsewhere (<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198792">25</a>–<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198803">28</a>). All subjects provided written informed consent for genetic studies that were approved by the Institutional Review Board at each participating center.</p></div><div id="S7" class="sec"><h3 id="S7title">GWAS genotyping</h3><p id="P10" class="p p-first-last">DNA isolation was performed in the same lab (University of Pittsburgh Human Genetics Department) for all samples utilized in discovery stage for genotyping (at Expression Analysis, Durham, NC) on Affy 6.0 (containing probes for 906,600 SNPs). After applying the marker map corresponding to human genome build 37 (GRCh37/hg19), a total of 905,420 successfully mapped SNPs (excluding duplicates) were advanced into quality control (QC) filtering process prior to GWA analysis.</p></div><div id="S8" class="sec"><h3 id="S8title">GWAS QC filters</h3><p id="P11" class="p p-first">Samples with poor performance (15 with &lt;95% average call rate across the array) and poor-quality markers (44,507 with &lt;95% call rate across all samples) were removed from analysis. Markers with significant deviation from Hardy-Weinberg equilibrium in controls (<em>P</em>≤1×10<sup>−6</sup>) and/or with low minor allele frequency (MAF&lt;0.01) were also excluded (<em>n</em>=133,396). Population stratification analysis was conducted using 691,565 markers and a multi-dimensional scaling method implemented in PLINK (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/~purcell/plink/</a>). SNPs falling within genomic regions exhibiting abnormal linkage disequilibrium (LD) patterns and structural variations (chr 6: 24–38 Mb, chr 8: 7–13 Mb, chr 11: 42–58 Mb, and chr 17: 41–45 Mb) and those located on Y chromosome were excluded from the calculation of principal components (PCs). First four components were determined to be relevant for controlling population substructure. Three samples were removed because they were extreme population outliers based on multi-dimensional scaling plots.</p><p id="P12" class="p p-last">After applying above QC filters, a final set of 1,148 subjects (661 cases and 487 controls) and 727,517 markers were advanced into association analysis for SLE. While the entire QC-passed autosomal marker data (<em>n</em>=700,598) were assessed for associations at previously reported SLE genes/loci, only the results from 627,076 common (MAF≥0.05) autosomal markers were used to identify potentially new SLE signals/loci.</p></div><div id="S9" class="sec"><h3 id="S9title">GWAS analysis</h3><p id="P13" class="p p-first-last">Logistic regression analysis of the effects of genotypes on SLE risk was conducted under an additive model that included recruitment site, sex, age, and first 4 PCs as covariates. All analyses were performed in PLINK. Pairwise LD plots were created for selected markers by analyzing discovery sample data using Haploview (<a href="http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/scientific-community/science/programs/medical-and-population-genetics/haploview/haploview</a>).</p></div><div id="S10" class="sec"><h3 id="S10title">Stage 2 dataset for follow-up investigation of suggestive GWAS hits (from Stage 1) and Meta-analyses of Stage 1 and 2 datasets for relevant markers</h3><p id="P14" class="p p-first-last">Data from an independent high-density GWAS of SLE in European-descent subjects (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198788">8</a>) was used for follow-up investigation of new suggestive autosomal GWAS signals (from Stage 1) located outside the extended major histocompatibility complex (xMHC) region. This Stage 2 dataset included 431 SLE cases (97% women) and 2,155 database controls (50% women) successfully genotyped for 311,238 SNPs on Affy 5.0 (details can be found in original publication) (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198778">8</a>). A total of 558 common markers were evaluated in this sample, which included index or proxy SNPs for new suggestive autosomal signals (<em>P</em>&lt;0.001) that were present in this dataset. The SNAP tool (<a href="https://www.broadinstitute.org/mpg/snap/ldsearch.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.broadinstitute.org/mpg/snap/ldsearch.php</a>) was utilized to determine proxy SNPs, using an <em>r<sup>2</sup></em> threshold of 0.8 and 1000 genomes pilot and HapMap 3 data from CEU population as references. Initial meta-analysis of Stage 1 and 2 datasets for 558 SNPs identified 26 potentially new SLE loci harboring SNPs with meta <em>P</em>&lt;0.001 and same direction of association in both datasets. Next, these 26 loci were more extensively examined in Stage 2 dataset by evaluating all relevant SNPs (567 common SNPs with <em>P</em>&lt;0.05 in Stage 1 sample falling within/near potentially new signals). Data was available for 306 of 567 SNPs in Stage 2 dataset and thus final meta-analysis of Stage 1 and 2 datasets included 306 SNPs from 26 loci. Meta-analyses were performed using the METAL software (<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409327">29</a>).</p></div><div id="S11" class="sec"><h3 id="S11title">Stage 3 dataset for replication of findings from meta-analyses of Stage 1 and 2 datasets</h3><p id="P15" class="p p-first-last">A genome-wide significant locus identified by meta-analyses of Stage 1 and 2 datasets was further tested for replication in the data from a large European high-density GWAS of SLE (Stage 3 dataset) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198780">21</a>, <a href="#R22" rid="R22" class=" bibr popnode">22</a>). A total of 8 SNPs from this locus (with meta <em>P</em>&lt;0.001 and same direction of association in Stage 1 and 2 samples) were evaluated in Stage 3 sample, which consisted of 4,036 SLE patients (90% women) and 6,959 controls (50% women; 5,699 from the NIH Health and Retirement Study) successfully genotyped for 644,674 markers on HumanOmni1-Quad BeadChip. This data was imputed to the density of 1000 genomes project using IMPUTE V2.2.3 (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409323">30</a>) and analyzed for association under an additive model computed by SNPTEST (<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409325">30</a>) with first 4 PCs used as covariates to account for European population substructure.</p></div><div id="S12" class="sec"><h3 id="S12title">In silico assessment of the regulatory potential of relevant variants</h3><p id="P16" class="p p-first-last">Relevant variants were evaluated for their regulatory potential by using RegulomeDB v1.1 (<a href="http://www.regulomedb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.regulomedb.org/</a>), a public database for human DNA features and regulatory elements in non-coding genomic regions. RegulomeDB is a powerful tool that uses various datasets (high-throughput experimental data from ENCODE project and other resources as well as computational and manual annotations) to score non-coding variants for their putative functional relevance. The RegulomeDB scoring scheme is provided in <a href="/pmc/articles/PMC4747422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a> footnote.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="P44">New genome-wide significant SLE locus that was identified by Stage 1 and 2 analyses and then replicated in Stage 3 analysis</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140640782574080" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" valign="middle" align="left">Chr</th><th rowspan="2" colspan="1" valign="middle" align="left">SNP</th><th rowspan="2" colspan="1" valign="middle" align="left">Position<br>in bp<br>(hg19)</th><th rowspan="2" colspan="1" valign="middle" align="center">Allele</th><th colspan="4" rowspan="1" valign="bottom" align="center">Stage 1 discovery set<br>661 cases &amp; 487 controls<hr></th><th colspan="4" rowspan="1" valign="bottom" align="center">Stage 2 data set<br>431 cases &amp; 2,155 controls<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">Stages 1&amp;2<br>Meta <em>P</em></th><th colspan="5" rowspan="1" valign="bottom" align="center">Stage 3 replication data set<br>4,036 cases &amp; 6,959 controls<hr></th><th rowspan="2" colspan="1" valign="middle" align="center">Nearest<br>Protein-<br>coding<br>Gene<sup>*</sup></th><th rowspan="2" colspan="1" valign="middle" align="center">RegulomeDB<br>score<sup>**</sup></th></tr><tr><th rowspan="1" colspan="1" valign="top" align="left">Case<br>Freq</th><th rowspan="1" colspan="1" valign="top" align="left">Contro<br>l Freq</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Case<br>Freq</th><th rowspan="1" colspan="1" valign="top" align="left">Contro<br>l Freq</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Case<br>Freq</th><th rowspan="1" colspan="1" valign="top" align="left">Contro<br>l Freq</th><th rowspan="1" colspan="1" valign="top" align="left">OR</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em></th><th rowspan="1" colspan="1" valign="top" align="left">Impute<br>d</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" align="left">12q12</td><td rowspan="1" colspan="1" align="left">rs10506216</td><td rowspan="1" colspan="1" align="left">43130885</td><td rowspan="1" colspan="1" align="center"><strong>T</strong></td><td rowspan="1" colspan="1" align="left">0.109</td><td rowspan="1" colspan="1" align="left">0.079</td><td rowspan="1" colspan="1" align="left">1.701</td><td rowspan="1" colspan="1" align="left">1.71E-03</td><td rowspan="1" colspan="1" align="left">0.124</td><td rowspan="1" colspan="1" align="left">0.080</td><td rowspan="1" colspan="1" align="left">1.666</td><td rowspan="1" colspan="1" align="left">5.04E-06</td><td rowspan="1" colspan="1" align="center"><strong>3.08E-08</strong></td><td rowspan="1" colspan="1" align="left">0.093</td><td rowspan="1" colspan="1" align="left">0.081</td><td rowspan="1" colspan="1" align="left">1.167</td><td rowspan="1" colspan="1" align="left"><strong>2.27E-03</strong></td><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="center"><em>PRICKLE1</em></td><td rowspan="1" colspan="1" align="center">6</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">rs1356422</td><td rowspan="1" colspan="1" align="left">43142996</td><td rowspan="1" colspan="1" align="center"><strong>G</strong></td><td rowspan="1" colspan="1" align="left">0.239</td><td rowspan="1" colspan="1" align="left">0.184</td><td rowspan="1" colspan="1" align="left">1.570</td><td rowspan="1" colspan="1" align="left">2.02E-04</td><td rowspan="1" colspan="1" align="left">0.215</td><td rowspan="1" colspan="1" align="left">0.182</td><td rowspan="1" colspan="1" align="left">1.247</td><td rowspan="1" colspan="1" align="left">1.22E-02</td><td rowspan="1" colspan="1" align="center"><strong>3.46E-05</strong></td><td rowspan="1" colspan="1" align="left">0.205</td><td rowspan="1" colspan="1" align="left">0.188</td><td rowspan="1" colspan="1" align="left">1.124</td><td rowspan="1" colspan="1" align="left"><strong>1.05E-03</strong></td><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="center"><em>PRICKLE1</em></td><td rowspan="1" colspan="1" align="center">5</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">rs11181677</td><td rowspan="1" colspan="1" align="left">43150285</td><td rowspan="1" colspan="1" align="center"><strong>C</strong></td><td rowspan="1" colspan="1" align="left">0.236</td><td rowspan="1" colspan="1" align="left">0.183</td><td rowspan="1" colspan="1" align="left">1.520</td><td rowspan="1" colspan="1" align="left">6.13E-04</td><td rowspan="1" colspan="1" align="left">0.209</td><td rowspan="1" colspan="1" align="left">0.181</td><td rowspan="1" colspan="1" align="left">1.204</td><td rowspan="1" colspan="1" align="left">4.00E-02</td><td rowspan="1" colspan="1" align="center"><strong>3.17E-04</strong></td><td rowspan="1" colspan="1" align="left">0.205</td><td rowspan="1" colspan="1" align="left">0.189</td><td rowspan="1" colspan="1" align="left">1.121</td><td rowspan="1" colspan="1" align="left"><strong>1.40E-03</strong></td><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="center"><em>PRICKLE1</em></td><td rowspan="1" colspan="1" align="center">ND</td></tr><tr><td rowspan="1" colspan="1" align="left"></td><td rowspan="1" colspan="1" align="left">rs1914490</td><td rowspan="1" colspan="1" align="left">43214484</td><td rowspan="1" colspan="1" align="center">G</td><td rowspan="1" colspan="1" align="left">0.300</td><td rowspan="1" colspan="1" align="left">0.251</td><td rowspan="1" colspan="1" align="left">1.427</td><td rowspan="1" colspan="1" align="left">1.49E-03</td><td rowspan="1" colspan="1" align="left">0.268</td><td rowspan="1" colspan="1" align="left">0.238</td><td rowspan="1" colspan="1" align="left">1.186</td><td rowspan="1" colspan="1" align="left">3.47E-02</td><td rowspan="1" colspan="1" align="center"><strong>4.33E-04</strong></td><td rowspan="1" colspan="1" align="left">0.270</td><td rowspan="1" colspan="1" align="left">0.257</td><td rowspan="1" colspan="1" align="left">1.072</td><td rowspan="1" colspan="1" align="left"><strong>3.29E-02</strong></td><td rowspan="1" colspan="1" align="left">Yes</td><td rowspan="1" colspan="1" align="center"><em>PRICKLE1</em></td><td rowspan="1" colspan="1" align="center">6</td></tr></tbody></table></div><div id="largeobj_idm140640782574080" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4747422/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P45" class="p p-first-last">Only the SNPs with meta <em>P</em>&lt;1.00E-03 and same direction of association in Stage 1 and 2 samples that had <em>P</em>&lt;0.05 in Stage 3 sample are shown. <strong>Bold</strong> alleles: reverse strand alleles</p></div><div id="TFN2"><sup>*</sup>The nearest protein-coding gene within ± 500 kb (<a href="http://www.genome.ucsc.edu/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.genome.ucsc.edu/</a>).</div><div id="TFN3"><sup>**</sup>Please see the methods section and <a href="http://www.regulomedb.org/help" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.regulomedb.org/help</a>;</div><div id="TFN4"><p id="P48" class="p p-first-last">ND: No Data.; RegulomeDB scoring scheme is divided into the following categories (1 through 6) based on available data resources: "1a: expression Quantitative Trait Locus (eQTL) + Transcription factor (TF) binding + matched TF motif + matched DNase Footprint + DNase peak, 1b: eQTL + TF binding + any motif + DNase Footprint + DNase peak, 1c: eQTL + TF binding + matched TF motif + DNase peak, 1d: eQTL + TF binding + any motif + DNase peak, 1e: eQTL + TF binding + matched TF motif, 1f: eQTL + TF binding / DNase peak, 2a: TF binding + matched TF motif + matched DNase Footprint + DNase peak, 2b: TF binding + any motif + DNase Footprint + DNase peak, 2c: TF binding + matched TF motif + DNase peak, 3a: TF binding + any motif + DNase peak, 3b: TF binding + matched TF motif, 4: TF binding + DNase peak, 5: TF binding or DNase peak, 6: other".</p></div></div></div></div><div id="S13" class="sec"><h3 id="S13title">In silico assessment of gene expression profiles and association patterns with expression quantitative trait loci (eQTLs) and methylation quantitative trait loci (mQTLs)</h3><p id="P17" class="p p-first-last"><em>In silico</em> assessment of expression profiles of genes of interest was performed using BioGPS gene annotation portal (<a href="http://biogps.org/#goto=welcome" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biogps.org/#goto=welcome</a>). The Genevar (GENe Expression VARiation) database v3.3.0 (<a href="http://www.sanger.ac.uk/resources/software/genevar/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sanger.ac.uk/resources/software/genevar/</a>) and data from Multiple Tissue Human Expression Resource (MuTHER) (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198798">31</a>–<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409317">33</a>) were utilized to visualize association patterns with eQTLs and mQTLs located within genomic region(s) of interest. In addition, an SQL database (ghs_probe_express030510) (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409321">34</a>) compiling the results of genome-wide SNP associations with human monocyte expression traits (<a href="http://genecanvas.ecgene.net/uploads/ForReview/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://genecanvas.ecgene.net/uploads/ForReview/</a>) was used to evaluate most relevant <em>cis</em> and <em>trans</em> eQTL effects of SNPs of interest.</p></div><div id="S14" class="sec sec-last"><h3 id="S14title">In silico enhancer enrichment analysis</h3><p id="P18" class="p p-first-last">HaploReg v2 (<a href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</a>) is a functional annotation tool for non-coding variants on haplotype blocks (based on LD calculations on 1000 Genomes Phase 1 individuals; for our analysis we used the default settings that included EUR population and LD threshold of <em>r<sup>2</sup></em>=0.8) and it includes an extensive library of SNPs, motif instances, enhancer annotations, and eQTLs. HaploReg v2 also conducts enhancer enrichment analysis by comparing the coverage of all enhancers and strongest enhancers calculated for queried variants to that of all variants. When the coverage exceeds that of selected background set (for our analysis we selected all SNPs in 1KG CEU pilot), the enrichment is reported if nominally significant based on a binomial test.</p></div></div><div id="S15" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S15title">RESULTS</h2><div id="S16" class="sec sec-first"><h3 id="S16title">GWAS of SLE in a North American discovery sample of European ancestry (Stage 1)</h3><p id="P19" class="p p-first">Quantile–quantile plot of observed vs. expected p-values under the null hypothesis is presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a> for the GWAS of Stage 1 discovery sample. Manhattan plot summarizing association results for QC-passed autosomal common markers is presented in <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>, including over 1,000 markers with <em>P</em>&lt;0.001. As expected, strongest associations were observed at the established MHC locus (6p21), which easily surpassed the stringent genome-wide significance threshold (<em>P</em>&lt;5×10<sup>−8</sup>) in our discovery sample. About 200 markers from the xMHC region at 6p22–21 had <em>P</em>&lt;0.001 and more than 600 had <em>P</em>&lt;0.05. The most significant MHC SNP was rs2187668 in <em>HLA-DQA1</em> (<em>P</em>=1.2×10<sup>−9</sup>), followed by rs3129716 near<em> HLA-DQB1</em> (<em>P</em>=4.2×10<sup>−9</sup>), rs1150753 in <em>TNXB</em> (<em>P</em>=5.6×10<sup>−9</sup>), rs3131379 in <em>MSH5</em> (<em>P</em>=7.6×10<sup>−8</sup>), and rs9267531 in <em>CSNK2B</em> (<em>P</em>=8.2×10<sup>−8</sup>). Three of these SNPs (rs2187668, rs3129716, and rs3131379) showed the RegulomeDB score of "1f", indicating a strong regulatory potential.</p><p id="P20" class="p p-last">Next we examined our Stage 1 data for 38 non-MHC autosomal SLE loci, where at least one common variant was previously shown to reach genome-wide significance in reported discovery and/or combined (discovery+replication) samples in GWASs or post-GWAS large studies of European-descent or Asian subjects (<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198764">6</a>–<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198797">20</a>, <a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198808">35</a>, <a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409318">36</a>) (see <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a> footnote). Twenty-nine of these loci showed nominal associations (<em>P</em>&lt;0.05) in our discovery sample, of which 10 were supported at the SNP level [by the same previously reported SNP and/or a proxy SNP (<em>r<sup>2</sup></em>≥0.8 in 1000 genomes pilot and/or HapMap 3 data from CEU population)] and 19 at the locus level [by other SNP(s) located within the gene(s) of interest ± 50 kb]. The top ones were the 2q32.3 locus (<em>STAT4</em>/rs7582694, <em>P</em>=3.6×10<sup>−7</sup>) and the 8p23.1 locus (<em>BLK</em>/rs1478897, <em>P</em>=8.1×10<sup>−6</sup>, proxy for rs2248932), both of which were previously implicated in more than one high-density GWAS of European-descent subjects. The full list of nominally associated SNPs from 29 non-MHC autosomal SLE loci supported in our discovery sample is provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>. At the remaining 9 loci, we observed borderline associations [<em>P</em>&lt;0.10 for some markers located within gene(s) of interest ± 50 kb] and/or nominal associations in extended regions [for some markers located within gene(s) of interest ± 100 kb] (data not shown).</p></div><div id="S17" class="sec"><h3 id="S17title">Follow-up investigation of new suggestive hits (from Stage 1) by using an additional dataset of European-descent subjects (Stage 2)</h3><p id="P21" class="p p-first-last">We next examined our GWAS data (SNPs with MAF≥0.05) for new suggestive autosomal signals and assessed them <em>in silico</em> in an independent GWAS data (Stage 2 dataset) (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198769">8</a>). A total of 558 common markers (index or proxy SNPs for non-xMHC autosomal associations with <em>P</em>&lt;0.001) had information available in Stage 2 dataset. Initial meta-analysis of Stage 1 and 2 datasets for 558 markers identified 26 new loci of interest, harboring SNPs with meta <em>P</em>&lt;0.001 and same direction of association in both datasets. These 26 loci were further examined in Stage 2 dataset by evaluating all relevant GWAS SNPs residing within/near implicated loci/genes (567 common SNPs with <em>P</em>&lt;0.05 in Stage 1 dataset). Information was available for 306 of 567 SNPs in Stage 2 dataset and final meta-analysis of Stage 1 and 2 datasets for these SNPs revealed a genome-wide significant new SLE locus at 12q12 (meta <em>P</em>=3.1×10<sup>−8</sup> for the lead SNP) (<a href="/pmc/articles/PMC4747422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>, <a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140640783103040" class="figure"><a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4747422_nihms751533f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms751533f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4747422/bin/nihms751533f1.jpg"></a></div><div id="largeobj_idm140640783103040" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4747422/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4747422/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Regional association plot on chromosome 12q12 (top panel) and the region overview of the corresponding chromosome segment in Ensembl genome browser (bottom panel)</strong><p id="P42">TOP PANEL: The associations observed in Stage 1 discovery sample are depicted as dark blue diamonds while the results from the meta-analysis of Stage 1 and Stage 2 samples are shown as red dots. The genes located in the region (based on the UCSC genome browser) and the recombination rates by position (light blue line) are also shown. The most relevant SNP with best meta <em>P</em> is labeled. BOTTOM PANEL: The red rectangle shows the large/long intergenic non-coding RNA (lincRNA) genes that reside adjacent to the SLE association signal in this region.</p></div></div></div></div><div id="S18" class="sec"><h3 id="S18title">Replication of new genome-wide significant locus (from Stage 2) in another dataset of European-descent subjects (Stage 3)</h3><p id="P22" class="p p-first-last">We next sought <em>in silico</em> replication of new SLE locus identified by meta-analyses of Stage 1 and 2 datasets in an independent imputed GWAS data (Stage 3 dataset) (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198766">21</a>, <a href="#R22" rid="R22" class=" bibr popnode">22</a>). A total of 8 SNPs from new 12q12 locus, with meta <em>P</em>&lt;0.001 and same direction of association in Stage 1 and 2 samples (<a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>-top panel), were further interrogated in Stage 3 sample. The genome-wide significant lead SNP (rs10506216/meta <em>P</em>=3.1×10<sup>−8</sup>) was replicated in Stage 3 dataset (rs10506216/<em>P</em>=2.3×10<sup>−3</sup>) and three suggestive SNPs (rs1356422/meta <em>P</em>=3.5×10<sup>−5</sup>, rs11181677/meta <em>P</em>=3.2×10<sup>−4</sup>, and rs1914490/meta <em>P</em>=4.3×10<sup>−4</sup>) also showed nominal significance in Stage 3 sample (rs1356422/<em>P</em>=1.1×10<sup>−3</sup>, rs11181677/<em>P</em>=1.4×10<sup>−3</sup>, and rs1914490/<em>P</em>=3.3×10<sup>−2</sup>) (<a href="/pmc/articles/PMC4747422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). The pairwise LD between these 4 SNPs in our discovery sample is illustrated in <a href="/pmc/articles/PMC4747422/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4747422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140640785279632" class="figure"><a href="/pmc/articles/PMC4747422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4747422_nihms751533f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms751533f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4747422/bin/nihms751533f2.jpg"></a></div><div id="largeobj_idm140640785279632" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4747422/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4747422/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>The extent of pairwise LD between the SNPs of interest at the 12q12 locus</strong><p id="P43">The LD plot shows the pairwise <em>r<sup>2</sup></em>×100 values between the SNPs of interest in Stage 1 discovery sample.</p></div></div></div></div><div id="S19" class="sec sec-last"><h3 id="S19title">Functional annotation of the new locus</h3><p id="P23" class="p p-first">The novel 12q12 association signal falls within an intergenic region, adjacent to some large/long intergenic non-coding RNA (lincRNA) genes (<a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>-bottom panel) including <em>LOC101927058</em> (<a href="http://www.ncbi.nlm.nih.gov/gene/101927058" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/101927058</a>). Although <em>PRICKLE1</em> is the nearest protein-coding gene (~147 kb from the lead SNP), this region also harbors an immunologically important gene (<em>IRAK4;</em> ~1 Mb from the lead SNP) (<a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>).</p><p id="P24">We first used RegulomeDB (<a href="http://www.regulomedb.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.regulomedb.org/</a>) to assess the regulatory potential of 4 SNPs of interest at this new locus, which revealed no functionally relevant low scores for these SNPs (<a href="/pmc/articles/PMC4747422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). Although no proxy with low score was identified for the lead SNP (rs10506216), the remaining 3 SNPs (rs1356422, rs11181677, and rs1914490) were in strong LD (<em>r<sup>2</sup></em>≥0.80) with 5 nearby potentially regulatory SNPs that were not analyzed in our study (rs868765, rs2897590, rs870972, rs12302566, and rs10785378; RegulomeDB scores ranging from "2b to 3a"). Two of these SNPs, rs870972 and rs12302566 (in strong LD with rs1356422 and rs11181677), were found to reside in a binding site or motif for STAT or IRF family of transcription factors (TFs) along with others.</p><p id="P25">We next used Genevar (<a href="http://www.sanger.ac.uk/resources/software/genevar/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.sanger.ac.uk/resources/software/genevar/</a>) to visualize eQTLs and mQTLs surrounding our SNPs of interest by examining the data from 856 Caucasian female twins with information on genetic variation, gene expression [tissues: adipose, lymphoblastoid cell lines (LCL) and skin], and DNA methylation (tissue: adipose) (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198787">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198785">32</a>). <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a> shows SNP-centric <em>cis</em>-eQTL analysis results for 4 SNPs of interest at 12q12, by displaying SNP-probe associations in a 2-Mb window (1-Mb on either side of each SNP) in 3 different tissues. Although the probes for <em>GXYLT1, YAF2, ZCRB1, PPHLN1, PRICKLE1, ADAMTS20</em>, and <em>PUS7L</em> were within targeted window for all 4 SNPs, those for <em>IRAK4</em> and downstream <em>PTK9</em> were included in analysis window for only rs1914490 due to their &gt;1-Mb of distance from other 3 SNPs (see <a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> for gene locations). In adipose tissue, the most significant eQTL gene was <em>PRICKLE1</em> for all 4 SNPs (2.5×10<sup>−4</sup>≤<em>P</em>≤5.0×10<sup>−3</sup>), followed by <em>ZCRB1</em> (for 3 SNPs) and <em>IRAK4 &amp; PPHLN1</em> (for rs1914490). In LCL, the most significant eQTL gene was <em>PPHLN1</em> (for rs1356422 and rs11181677, 3.4×10<sup>−3</sup>≤<em>P</em>≤5.6×10<sup>−3</sup>), followed by <em>ADAMTS20</em> (for rs10506216). Gene-centric <em>cis</em>-eQTL analysis (in a 2-Mb window around a given probe/gene) revealed that our SNPs and region of interest (GRCh37/hg19; chr12: 43130885–43214484, NCBI36/hg18; chr12: 41417152–41500751) overlapped with the most relevant cis-eQTL SNPs region for <em>PRICKLE1</em> in adipose tissue (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4</a>). In LCL, a similar overlap was observed for <em>PPHLN1</em> (for one probe) and our region of interest seemed to be also relevant for <em>IRAK4</em> expression (although it was only partly evaluated for this gene given its distant location) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5</a>). SNP-centric <em>cis</em>-mQTL analysis [performed only in adipose tissue for common variants (MAF&gt;0.05) located close to methylation sites/probes (probe ± 100 kb)] revealed strong associations between our SNPs of interest at 12q12 and surrounding methylation sites/probes (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>). Two probes (cg16758809 and cg22738642) were significantly associated with all 4 SNPs (2.0×10<sup>−10</sup>≤<em>P</em>≤2.1×10<sup>−4</sup> and 2.2×10<sup>−10</sup>≤<em>P</em>≤1.8×10<sup>−2</sup>, respectively).</p><p id="P26">Evaluation of 4 SNPs of interest at 12q12 in a database of genome-wide SNP associations (<em>cis</em> and <em>trans</em>) with monocyte expression in 1,490 European subjects (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409322">34</a>) not only confirmed a <em>cis</em> effect on <em>PRICKLE1</em> expression (rs1356422/<em>P</em>=3.5×10<sup>−6</sup> and rs11181677/<em>P</em>=1.3×10<sup>−6</sup>) but also revealed a <em>trans</em> effect of the lead SNP (rs10506216) on <em>PHRF1</em> expression at 11p15 (<em>P</em>=1.3×10<sup>−5</sup>).</p><p id="P27">Assessment of gene expression using BioGPS (<a href="http://biogps.org/#goto=welcome" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://biogps.org/#goto=welcome</a>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S7–S8</a>) revealed that although all genes of interest mentioned above are expressed in various cells/tissues including immune system, 3 of them (<em>ZCRB1, PPHLN1</em>, and <em>IRAK4)</em> exhibit a particularly high-level of expression in immune cells.</p><p id="P28" class="p p-last">Finally, we also performed enhancer enrichment analysis using HaploReg v2 (<a href="http://www.broadinstitute.org/mammals/haploreg/haploreg.php" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.broadinstitute.org/mammals/haploreg/haploreg.php</a>) for 4 SNPs of interest (rs10506216, rs1356422, rs11181677, rs1914490) and 3 proxy SNPs (rs868765, rs2897590, rs870972) falling into RegulomeDB categories 1 or 2 (see above and Methods section) and the results for these 7 SNPs are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S9</a>. The most significant enrichment of strongest enhancers was detected in human embryonic stem cells (H1; <em>P</em>=1.0×10<sup>−6</sup>), B lymphoblastocytes (GM12878; <em>P</em>=1.6×10<sup>−4</sup>) and epidermal keratinocytes (NHEK; <em>P</em>=3.7×10<sup>−4</sup>).</p></div></div><div id="S20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S20title">DISCUSSION</h2><p id="P29" class="p p-first">We conducted a new GWAS of SLE in a North American case-control sample of European ancestry and used the data from an independent GWAS of SLE in European-descent subjects (<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198801">8</a>) for follow-up evaluation of new suggestive autosomal signals, followed by meta-analyses that revealed a new genome-wide significant locus. This new locus was then replicated in imputed data from an independent European GWAS of SLE that is pending publication (<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198794">21</a>, <a href="#R22" rid="R22" class=" bibr popnode">22</a>).</p><div id="S21" class="sec"><h3 id="S21title">Replication of associations with previously reported autosomal SLE loci</h3><p id="P30" class="p p-first">Our GWAS further confirmed the MHC locus at 6p21 as the strongest SLE susceptibility locus (<em>P</em>=1.2×10<sup>−9</sup> in our discovery sample), followed by 2q32/<em>STAT4</em> (<em>P</em>=3.6×10<sup>−7</sup>) and 8p23/<em>BLK</em> (<em>P</em>=8.1×10<sup>−6</sup>) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2</a>). Overall, 30 of 39 previously reported autosomal SLE loci were supported in our discovery sample with <em>P</em>&lt;0.05 (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>); 11 (<em>MHC region, STAT4, XKR6/BLK, IRF5/TNPO3, ITGAM, OLIG3/TNFAIP3, IL10, SNRPC/UHRF1BP1, IRF8, WDFY4, CREBL2/CDKN1B</em>) at the SNP level and 19 at the locus level (see Results for details). Of note, we cross-referenced our findings with only ‘selected genome-wide significant common SNPs’ reported at these 39 loci (not with all published significant SNPs), therefore the number of associations that we observed at the SNP level may be an underestimate. Our findings are not surprising given that several SLE loci reached genome-wide significance only after increasing the sample size by adding more controls from public databases and/or after meta-analyses of discovery and replication samples and/or after performing follow-up studies by genotyping additional markers/samples (including the use of various ethnic groups for transancestral meta-analyses), all of which indicating small effect sizes for these loci. We considered nominal association(s) in our discovery sample as supporting evidence for previously reported loci. According to recommendations for publication of genetic associations with rheumatic diseases, it is acceptable to consider <em>P</em>&lt;0.05 as significant in follow-up studies of established loci (<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198786">37</a>).</p><p id="P31">Our GWAS supported 11 SLE loci initially reported in Asian GWASs or follow-up studies (<em>RASGRP3, TET3, AFF1, HIP1, WDFY4, ARID5B, ETS1, CREBL2/CDKN1B, SLC15A4, ELF1</em>, and <em>PRKCB</em>) in our North American discovery sample, at the SNP (<em>WDFY4 and CREBL2/CDKN1B</em>) or locus level (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>). Although some of these loci have already been replicated in Caucasians (<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198779">38</a>), to our knowledge we are the first to report supporting evidence for <em>TET3, AFF1, HIP1, ARID5B, CREBL2/CDKN1B, ELF1</em>, and <em>PRKCB</em> in European-descent subjects.</p><p id="P32">As part of this study, we used some recently developed public databases including RegulomeDB, to assess the regulatory potential of SLE-relevant SNPs. A number of SNPs from established SLE loci with <em>P</em>&lt;0.05 in our discovery sample showed low RegulomeDB scores, indicating a strong regulatory potential (<a href="#SD1" rid="SD1" class=" supplementary-material">Table S1</a>).</p><p id="P33" class="p p-last">In summary, our GWAS supported several, but not all, previously published SLE loci. Various factors may influence replication of reported loci, including GWAS platform coverage differences, study sample characteristics, study power, allele frequency/LD differences among various ethnic groups, allelic/genetic heterogeneity, effect size differences, gene-environment interactions and epigenetic factors.</p></div><div id="S22" class="sec sec-last"><h3 id="S22title">New SLE locus at 12q12</h3><p id="P34" class="p p-first">To follow-up on our new GWAS findings for SLE, we adopted a cost-effective multi-stage strategy by using available data from already performed studies; therefore, we could not evaluate all new suggestive loci and/or all relevant SNPs at these loci in follow-up analyses. Nevertheless, we identified and replicated a new genome-wide significant locus at <strong>12q12</strong> (<a href="/pmc/articles/PMC4747422/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). This new SLE signal falls within an intergenic region, located ~147 kb 5’-upstream of <strong>PRICKLE1</strong> [prickle homolog 1 (Drosophila)] and ~1 Mb 5’-upstream of <strong>IRAK4</strong> (interleukin-1 receptor-associated kinase 4) (<a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>). This region also harbors some <strong>lincRNA</strong> genes that reside in closer vicinity to this SLE signal (<a href="/pmc/articles/PMC4747422/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>-bottom panel). Non-coding RNAs are emerging as essential regulators of various cellular processes including immune response (<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409326">39</a>, <a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198783">40</a>). Although these recently described lincRNAs do not seem to be represented in public gene expression databases (Genevar and BioGPS), <em>in silico</em> assessment of eQTL and mQTL associations and expression of relevant protein-coding genes (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S3–S8</a>) suggest that this new locus probably functions as a distal regulatory element (i.e., enhancer) for the expression of neighboring genes in a tissue/cell type-specific manner, with indication of epigenetic modulation. The association of SLE-relevant SNPs with surrounding methylation sites is particularly strong at this new locus (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a>) and these sites may modulate chromatin accessibility and transcriptional potential of underlying DNA segment (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198802">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198789">32</a>). Recent studies suggest that inter-individual variation in DNA methylation is genetically regulated (more variable and heritable for gene-body and intergenic methylation sites) and these regulatory variants usually function in a tissue-specific manner (<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198781">31</a>, <a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198777">32</a>). Moreover, our ‘possible enhancer’ hypothesis for this new intergenic locus has also been supported by the results of enhancer enrichment analysis performed in HaploReg v2 (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S9</a>).</p><p id="P35">Based on available public data (Genevar and BioGPS), noteworthy genes that seem to be <em>cis</em>-regulated by this new locus include <strong>PRICKLE1, PPHLN1, and IRAK4</strong> (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S3–S5, S7–S8</a>). <em>PRICKLE1</em> (<a href="http://www.ncbi.nlm.nih.gov/gene/144165" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/144165</a>) encodes a nuclear receptor that regulates the Wnt/beta-catenin signaling pathway. This pathway was (a) implicated in renal disease pathogenesis including lupus nephritis in mice, (b) shown to be activated in most autoimmune diseases, and (c) reported to play a critical role in senescence of bone marrow-mesenchymal stem cells from SLE patients (<a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409319">41</a>–<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198795">43</a>). <em>PRICKLE1</em> mutations were linked to epilepsy and Autism Spectrum Disorders (<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198768">44</a>–<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198806">46</a>). <em>PPHLN1</em> (Periphilin 1; <a href="http://www.ncbi.nlm.nih.gov/gene/51535" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/51535</a>) appears to play a role in epithelial differentiation and epidermal integrity; however, it is most abundantly expressed in immune cells (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S7</a>), thus awaiting further functional characterization. <em>IRAK4</em> (<a href="http://www.ncbi.nlm.nih.gov/gene/51135" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene/51135</a>) is involved in NF-kappaB activation in Toll-like receptor and T-cell receptor signaling pathways and its mutations were reported to cause recurrent infections. <em>IRAK4</em> was shown to play an important role in Th17-mediated immunity and removal of autoreactive B cells (<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198771">47</a>, <a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409320">48</a>). IRAK1/4 inhibition was reported to affect plasmacytoid dendritic cell response to SLE serum (<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409316">49</a>) (<em>IRAK1</em> is an X-linked member of IRAK family previously implicated in SLE) (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198799">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198784">5</a>).</p><p id="P36">A database of genome-wide SNP associations with monocyte expression (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409328">34</a>) also suggested a possible <em>trans</em> effect of the lead SNP at 12q12 on <em>PHRF1</em> expression at 11p15. The <em>PHRF1-IRF7</em> region is among previously implicated SLE loci (<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198782">3</a>–<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512198770">5</a>) and its interaction with the 12q12 locus warrants further investigation. Interestingly, three genomic regions reported to be major <em>trans</em>-acting regulators of multiple expression traits in monocytes are all located at 12q (12q13, 12q15, 12q24) (<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409324">34</a>). Although studies investigating <em>trans</em> eQTLs are relatively limited, a similar <em>trans</em> effect was also reported for another SLE locus (<a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550409329">50</a>).</p><p id="P37">According to RegulomeDB, this 12q12 regulatory region binds various TFs, including STAT and IRF family TFs that are essential for SLE as indicated by their interactions with type I interferon system that plays a central role in SLE and by direct implication of some of their members in SLE susceptibility.</p><p id="P38">Although public databases are extremely useful for preliminary functional evaluation of disease-associated variants/loci, they suffer from some limitations. Major limitation includes data/analyses being restricted to selected SNPs, selected probes (not representing all genes or all splice forms of a given gene), selected tissue/cell types, and selected methods (our region of interest was partially evaluated for its effect on <em>IRAK4</em> expression in Genevar-MuTHER due to a selected maximum window size). Therefore, an extensive functional analysis is necessary to determine the role of this new SLE locus in regulation of various genes in various cells/tissues involved in SLE pathogenesis. Follow-up deep sequencing is also warranted to uncover all causative common/rare variants that may jointly contribute to SLE risk at this locus.</p><p id="P39" class="p p-last">In conclusion, several previously published SLE loci were supported in our discovery sample and our multi-stage approach identified and replicated a new genome-wide significant locus at 12q12. Although recently developed public tools/databases significantly helped us to gain valuable insights into the regulatory potential of our region/SNPs of interest, they also emphasized the need for comprehensive follow-up functional studies to determine tissue/cell type-specific roles of implicated loci. Thus, this new 12q12 locus warrants further characterization using deep sequencing and extensive functional evaluation and is likely to shed further light on underlying disease mechanisms. Additional large studies are also necessary to examine association of this new locus with SLE subphenotypes and its interaction with hormonal and environmental factors. These factors could influence the effect size of this locus (given its association with DNA methylation and its predicted role in epigenetic regulation), depending on their enrichment in a given study sample.</p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp Info</h4><div class="sup-box half_rhythm" id="d37e1278"><a href="/pmc/articles/PMC4747422/bin/NIHMS751533-supplement-Supp_Info.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(3.3M, pdf)</sup></div></div></div><div id="S23" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S23title">Acknowledgments</h2><div class="sec"><p id="P40"><span class="underline">FUNDING</span>: This work was supported by grants from the US National Institutes of Health [<a id="__tag_781612677" class="tag_hotlink" href="/nuccore/HL092397" ref="/nuccore/HL092397"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL092397","term_id":"1051662806","term_text":"HL092397"}}</span>HL092397</a>, <a id="__tag_781612676" class="tag_hotlink" href="/nuccore/HL088648" ref="/nuccore/HL088648"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"HL088648","term_id":"1051659057","term_text":"HL088648"}}</span>HL088648</a>, <a id="__tag_781612674" class="tag_hotlink" href="/nuccore/AR057028" ref="/nuccore/AR057028"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR057028","term_id":"5982605","term_text":"AR057028"}}</span>AR057028</a>, <a id="__tag_781612673" class="tag_hotlink" href="/nuccore/AR046588" ref="/nuccore/AR046588"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR046588","term_id":"5968053","term_text":"AR046588"}}</span>AR046588</a>, <a id="__tag_781612672" class="tag_hotlink" href="/nuccore/AR057338" ref="/nuccore/AR057338"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR057338","term_id":"5982915","term_text":"AR057338"}}</span>AR057338</a>, HD066139, AR02318, AR30492, <a id="__tag_781612675" class="tag_hotlink" href="/nuccore/AR064464" ref="/nuccore/AR064464"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AR064464","term_id":"5993772","term_text":"AR064464"}}</span>AR064464</a>, AR30692, AR43274, AI63274, AR56360, and TR000150]; by a grant from the Lupus Foundation of America; and by grants from Wellcome Trust (Ref 085492) and Arthritis Research UK (Ref 19289).</p></div></div><div id="idm140640782197936" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140640782197936title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN1"> </p><p id="P41" class="p p-first-last"><span class="underline">CONFLICT OF INTEREST STATEMENT</span>: The authors have no conflict of interest to disclose.</p><p></p></div></div><div id="idm140640789653600" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140640789653600title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Alarcon-Segovia D, Alarcon-Riquelme ME, Cardiel MH, Caeiro F, Massardo L, Villa AR, et al.  Familial aggregation of systemic lupus erythematosus, rheumatoid arthritis, and other autoimmune diseases in 1,177 lupus patients from the GLADEL cohort. <span><span class="ref-journal">Arthritis and rheumatism. </span>2005;<span class="ref-vol">52</span>(4):1138–1147.</span> [<a href="/pubmed/15818688" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Familial+aggregation+of+systemic+lupus+erythematosus,+rheumatoid+arthritis,+and+other+autoimmune+diseases+in+1,177+lupus+patients+from+the+GLADEL+cohort&amp;author=D+Alarcon-Segovia&amp;author=ME+Alarcon-Riquelme&amp;author=MH+Cardiel&amp;author=F+Caeiro&amp;author=L+Massardo&amp;volume=52&amp;issue=4&amp;publication_year=2005&amp;pages=1138-1147&amp;pmid=15818688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Deapen D, Escalante A, Weinrib L, Horwitz D, Bachman B, Roy-Burman P, et al.  A revised estimate of twin concordance in systemic lupus erythematosus. <span><span class="ref-journal">Arthritis and rheumatism. </span>1992;<span class="ref-vol">35</span>(3):311–318.</span> [<a href="/pubmed/1536669" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=A+revised+estimate+of+twin+concordance+in+systemic+lupus+erythematosus&amp;author=D+Deapen&amp;author=A+Escalante&amp;author=L+Weinrib&amp;author=D+Horwitz&amp;author=B+Bachman&amp;volume=35&amp;issue=3&amp;publication_year=1992&amp;pages=311-318&amp;pmid=1536669&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Guerra SG, Vyse TJ, Cunninghame Graham DS. The genetics of lupus: a functional perspective. <span><span class="ref-journal">Arthritis research &amp; therapy. </span>2012;<span class="ref-vol">14</span>(3):211.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3446495/">PMC free article</a>]</span> [<a href="/pubmed/22640752" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+research+&amp;+therapy&amp;title=The+genetics+of+lupus:+a+functional+perspective&amp;author=SG+Guerra&amp;author=TJ+Vyse&amp;author=DS+Cunninghame+Graham&amp;volume=14&amp;issue=3&amp;publication_year=2012&amp;pages=211&amp;pmid=22640752&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Vaughn SE, Kottyan LC, Munroe ME, Harley JB. Genetic susceptibility to lupus: the biological basis of genetic risk found in B cell signaling pathways. <span><span class="ref-journal">Journal of leukocyte biology. </span>2012;<span class="ref-vol">92</span>(3):577–591.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3748338/">PMC free article</a>]</span> [<a href="/pubmed/22753952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+leukocyte+biology&amp;title=Genetic+susceptibility+to+lupus:+the+biological+basis+of+genetic+risk+found+in+B+cell+signaling+pathways&amp;author=SE+Vaughn&amp;author=LC+Kottyan&amp;author=ME+Munroe&amp;author=JB+Harley&amp;volume=92&amp;issue=3&amp;publication_year=2012&amp;pages=577-591&amp;pmid=22753952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Dai C, Deng Y, Quinlan A, Gaskin F, Tsao BP, Fu SM. Genetics of systemic lupus erythematosus: immune responses and end organ resistance to damage. <span><span class="ref-journal">Current opinion in immunology. </span>2014;<span class="ref-vol">31</span>:87–96.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4274270/">PMC free article</a>]</span> [<a href="/pubmed/25458999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Current+opinion+in+immunology&amp;title=Genetics+of+systemic+lupus+erythematosus:+immune+responses+and+end+organ+resistance+to+damage&amp;author=C+Dai&amp;author=Y+Deng&amp;author=A+Quinlan&amp;author=F+Gaskin&amp;author=BP+Tsao&amp;volume=31&amp;publication_year=2014&amp;pages=87-96&amp;pmid=25458999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Hom G, Graham RR, Modrek B, Taylor KE, Ortmann W, Garnier S, et al.  Association of systemic lupus erythematosus with C8orf13-BLK and ITGAM-ITGAX. <span><span class="ref-journal">The New England journal of medicine. </span>2008;<span class="ref-vol">358</span>(9):900–909.</span> [<a href="/pubmed/18204098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+New+England+journal+of+medicine&amp;title=Association+of+systemic+lupus+erythematosus+with+C8orf13-BLK+and+ITGAM-ITGAX&amp;author=G+Hom&amp;author=RR+Graham&amp;author=B+Modrek&amp;author=KE+Taylor&amp;author=W+Ortmann&amp;volume=358&amp;issue=9&amp;publication_year=2008&amp;pages=900-909&amp;pmid=18204098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Harley JB, Alarcon-Riquelme ME, Criswell LA, Jacob CO, Kimberly RP, Moser KL, et al.  Genome-wide association scan in women with systemic lupus erythematosus identifies susceptibility variants in ITGAM, PXK, KIAA1542 and other loci. <span><span class="ref-journal">Nature genetics. </span>2008;<span class="ref-vol">40</span>(2):204–210.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3712260/">PMC free article</a>]</span> [<a href="/pubmed/18204446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Genome-wide+association+scan+in+women+with+systemic+lupus+erythematosus+identifies+susceptibility+variants+in+ITGAM,+PXK,+KIAA1542+and+other+loci&amp;author=JB+Harley&amp;author=ME+Alarcon-Riquelme&amp;author=LA+Criswell&amp;author=CO+Jacob&amp;author=RP+Kimberly&amp;volume=40&amp;issue=2&amp;publication_year=2008&amp;pages=204-210&amp;pmid=18204446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Graham RR, Cotsapas C, Davies L, Hackett R, Lessard CJ, Leon JM, et al.  Genetic variants near TNFAIP3 on 6q23 are associated with systemic lupus erythematosus. <span><span class="ref-journal">Nature genetics. </span>2008;<span class="ref-vol">40</span>(9):1059–1061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2772171/">PMC free article</a>]</span> [<a href="/pubmed/19165918" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Genetic+variants+near+TNFAIP3+on+6q23+are+associated+with+systemic+lupus+erythematosus&amp;author=RR+Graham&amp;author=C+Cotsapas&amp;author=L+Davies&amp;author=R+Hackett&amp;author=CJ+Lessard&amp;volume=40&amp;issue=9&amp;publication_year=2008&amp;pages=1059-1061&amp;pmid=19165918&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Cunninghame Graham DS, Morris DL, Bhangale TR, Criswell LA, Syvanen AC, Ronnblom L, et al.  Association of NCF2, IKZF1, IRF8, IFIH1, and TYK2 with systemic lupus erythematosus. <span><span class="ref-journal">PLoS genetics. </span>2011;<span class="ref-vol">7</span>(10):e1002341.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3203198/">PMC free article</a>]</span> [<a href="/pubmed/22046141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Association+of+NCF2,+IKZF1,+IRF8,+IFIH1,+and+TYK2+with+systemic+lupus+erythematosus&amp;author=DS+Cunninghame+Graham&amp;author=DL+Morris&amp;author=TR+Bhangale&amp;author=LA+Criswell&amp;author=AC+Syvanen&amp;volume=7&amp;issue=10&amp;publication_year=2011&amp;pages=e1002341&amp;pmid=22046141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Gateva V, Sandling JK, Hom G, Taylor KE, Chung SA, Sun X, et al.  A large-scale replication study identifies TNIP1, PRDM1, JAZF1, UHRF1BP1 and IL10 as risk loci for systemic lupus erythematosus. <span><span class="ref-journal">Nature genetics. </span>2009;<span class="ref-vol">41</span>(11):1228–1233.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2925843/">PMC free article</a>]</span> [<a href="/pubmed/19838195" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=A+large-scale+replication+study+identifies+TNIP1,+PRDM1,+JAZF1,+UHRF1BP1+and+IL10+as+risk+loci+for+systemic+lupus+erythematosus&amp;author=V+Gateva&amp;author=JK+Sandling&amp;author=G+Hom&amp;author=KE+Taylor&amp;author=SA+Chung&amp;volume=41&amp;issue=11&amp;publication_year=2009&amp;pages=1228-1233&amp;pmid=19838195&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Graham RR, Hom G, Ortmann W, Behrens TW. Review of recent genome-wide association scans in lupus. <span><span class="ref-journal">Journal of internal medicine. </span>2009;<span class="ref-vol">265</span>(6):680–688.</span> [<a href="/pubmed/19493061" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+internal+medicine&amp;title=Review+of+recent+genome-wide+association+scans+in+lupus&amp;author=RR+Graham&amp;author=G+Hom&amp;author=W+Ortmann&amp;author=TW+Behrens&amp;volume=265&amp;issue=6&amp;publication_year=2009&amp;pages=680-688&amp;pmid=19493061&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Jacob CO, Eisenstein M, Dinauer MC, Ming W, Liu Q, John S, et al.  Lupus-associated causal mutation in neutrophil cytosolic factor 2 (NCF2) brings unique insights to the structure and function of NADPH oxidase. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2012;<span class="ref-vol">109</span>(2):E59–E67.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3258621/">PMC free article</a>]</span> [<a href="/pubmed/22203994" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Lupus-associated+causal+mutation+in+neutrophil+cytosolic+factor+2+(NCF2)+brings+unique+insights+to+the+structure+and+function+of+NADPH+oxidase&amp;author=CO+Jacob&amp;author=M+Eisenstein&amp;author=MC+Dinauer&amp;author=W+Ming&amp;author=Q+Liu&amp;volume=109&amp;issue=2&amp;publication_year=2012&amp;pages=E59-E67&amp;pmid=22203994&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Lessard CJ, Adrianto I, Ice JA, Wiley GB, Kelly JA, Glenn SB, et al.  Identification of IRF8, TMEM39A, and IKZF3-ZPBP2 as susceptibility loci for systemic lupus erythematosus in a large-scale multiracial replication study. <span><span class="ref-journal">American journal of human genetics. </span>2012;<span class="ref-vol">90</span>(4):648–660.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3322228/">PMC free article</a>]</span> [<a href="/pubmed/22464253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Identification+of+IRF8,+TMEM39A,+and+IKZF3-ZPBP2+as+susceptibility+loci+for+systemic+lupus+erythematosus+in+a+large-scale+multiracial+replication+study&amp;author=CJ+Lessard&amp;author=I+Adrianto&amp;author=JA+Ice&amp;author=GB+Wiley&amp;author=JA+Kelly&amp;volume=90&amp;issue=4&amp;publication_year=2012&amp;pages=648-660&amp;pmid=22464253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Lessard CJ, Adrianto I, Kelly JA, Kaufman KM, Grundahl KM, Adler A, et al.  Identification of a systemic lupus erythematosus susceptibility locus at 11p13 between PDHX and CD44 in a multiethnic study. <span><span class="ref-journal">American journal of human genetics. </span>2011;<span class="ref-vol">88</span>(1):83–91.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014359/">PMC free article</a>]</span> [<a href="/pubmed/21194677" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Identification+of+a+systemic+lupus+erythematosus+susceptibility+locus+at+11p13+between+PDHX+and+CD44+in+a+multiethnic+study&amp;author=CJ+Lessard&amp;author=I+Adrianto&amp;author=JA+Kelly&amp;author=KM+Kaufman&amp;author=KM+Grundahl&amp;volume=88&amp;issue=1&amp;publication_year=2011&amp;pages=83-91&amp;pmid=21194677&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Taylor KE, Chung SA, Graham RR, Ortmann WA, Lee AT, Langefeld CD, et al.  Risk alleles for systemic lupus erythematosus in a large case-control collection and associations with clinical subphenotypes. <span><span class="ref-journal">PLoS genetics. </span>2011;<span class="ref-vol">7</span>(2):e1001311.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3040652/">PMC free article</a>]</span> [<a href="/pubmed/21379322" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Risk+alleles+for+systemic+lupus+erythematosus+in+a+large+case-control+collection+and+associations+with+clinical+subphenotypes&amp;author=KE+Taylor&amp;author=SA+Chung&amp;author=RR+Graham&amp;author=WA+Ortmann&amp;author=AT+Lee&amp;volume=7&amp;issue=2&amp;publication_year=2011&amp;pages=e1001311&amp;pmid=21379322&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Han JW, Zheng HF, Cui Y, Sun LD, Ye DQ, Hu Z, et al.  Genome-wide association study in a Chinese Han population identifies nine new susceptibility loci for systemic lupus erythematosus. <span><span class="ref-journal">Nature genetics. </span>2009;<span class="ref-vol">41</span>(11):1234–1237.</span> [<a href="/pubmed/19838193" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Genome-wide+association+study+in+a+Chinese+Han+population+identifies+nine+new+susceptibility+loci+for+systemic+lupus+erythematosus&amp;author=JW+Han&amp;author=HF+Zheng&amp;author=Y+Cui&amp;author=LD+Sun&amp;author=DQ+Ye&amp;volume=41&amp;issue=11&amp;publication_year=2009&amp;pages=1234-1237&amp;pmid=19838193&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Okada Y, Shimane K, Kochi Y, Tahira T, Suzuki A, Higasa K, et al.  A genome-wide association study identified AFF1 as a susceptibility locus for systemic lupus eyrthematosus in Japanese. <span><span class="ref-journal">PLoS genetics. </span>2012;<span class="ref-vol">8</span>(1):e1002455.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3266877/">PMC free article</a>]</span> [<a href="/pubmed/22291604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=A+genome-wide+association+study+identified+AFF1+as+a+susceptibility+locus+for+systemic+lupus+eyrthematosus+in+Japanese&amp;author=Y+Okada&amp;author=K+Shimane&amp;author=Y+Kochi&amp;author=T+Tahira&amp;author=A+Suzuki&amp;volume=8&amp;issue=1&amp;publication_year=2012&amp;pages=e1002455&amp;pmid=22291604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Yang J, Yang W, Hirankarn N, Ye DQ, Zhang Y, Pan HF, et al.  ELF1 is associated with systemic lupus erythematosus in Asian populations. <span><span class="ref-journal">Human molecular genetics. </span>2011;<span class="ref-vol">20</span>(3):601–607.</span> [<a href="/pubmed/21044949" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Human+molecular+genetics&amp;title=ELF1+is+associated+with+systemic+lupus+erythematosus+in+Asian+populations&amp;author=J+Yang&amp;author=W+Yang&amp;author=N+Hirankarn&amp;author=DQ+Ye&amp;author=Y+Zhang&amp;volume=20&amp;issue=3&amp;publication_year=2011&amp;pages=601-607&amp;pmid=21044949&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Yang W, Shen N, Ye DQ, Liu Q, Zhang Y, Qian XX, et al.  Genome-wide association study in Asian populations identifies variants in ETS1 and WDFY4 associated with systemic lupus erythematosus. <span><span class="ref-journal">PLoS genetics. </span>2010;<span class="ref-vol">6</span>(2):e1000841.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2820522/">PMC free article</a>]</span> [<a href="/pubmed/20169177" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+genetics&amp;title=Genome-wide+association+study+in+Asian+populations+identifies+variants+in+ETS1+and+WDFY4+associated+with+systemic+lupus+erythematosus&amp;author=W+Yang&amp;author=N+Shen&amp;author=DQ+Ye&amp;author=Q+Liu&amp;author=Y+Zhang&amp;volume=6&amp;issue=2&amp;publication_year=2010&amp;pages=e1000841&amp;pmid=20169177&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Yang W, Tang H, Zhang Y, Tang X, Zhang J, Sun L, et al.  Meta-analysis followed by replication identifies loci in or near CDKN1B, TET3, CD80, DRAM1, and ARID5B as associated with systemic lupus erythematosus in Asians. <span><span class="ref-journal">American journal of human genetics. </span>2013;<span class="ref-vol">92</span>(1):41–51.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3542470/">PMC free article</a>]</span> [<a href="/pubmed/23273568" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Meta-analysis+followed+by+replication+identifies+loci+in+or+near+CDKN1B,+TET3,+CD80,+DRAM1,+and+ARID5B+as+associated+with+systemic+lupus+erythematosus+in+Asians&amp;author=W+Yang&amp;author=H+Tang&amp;author=Y+Zhang&amp;author=X+Tang&amp;author=J+Zhang&amp;volume=92&amp;issue=1&amp;publication_year=2013&amp;pages=41-51&amp;pmid=23273568&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Bentham J, Vyse TJ. The development of genome-wide association studies and their application to complex diseases, including lupus. <span><span class="ref-journal">Lupus. </span>2013;<span class="ref-vol">22</span>(12):1205–1213.</span> [<a href="/pubmed/24097992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lupus&amp;title=The+development+of+genome-wide+association+studies+and+their+application+to+complex+diseases,+including+lupus&amp;author=J+Bentham&amp;author=TJ+Vyse&amp;volume=22&amp;issue=12&amp;publication_year=2013&amp;pages=1205-1213&amp;pmid=24097992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Bentham J, Morris DL, Alarcón-Riquelme ME, Anand V, Delgado-Vega AM, Fortin PR, et al.  Novel loci identified by the largest genome-wide association study of lupus to date; (Abstract). Presented at the 61st Annual Meeting of The American Society of Human Genetics; October 11-15, 2011; Montréal, Canada. 2011.  <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Bentham+J+Morris+DL+Alarcón-Riquelme+ME+Anand+V+Delgado-Vega+AM+Fortin+PR+Novel+loci+identified+by+the+largest+genome-wide+association+study+of+lupus+to+date;+(Abstract)+2011+Presented+at+the+61st+Annual+Meeting+of+The+American+Society+of+Human+Genetics+October+11-15,+2011+Montréal,+Canada+" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Tan EM, Cohen AS, Fries JF, Masi AT, McShane DJ, Rothfield NF, et al.  The 1982 revised criteria for the classification of systemic lupus erythematosus. <span><span class="ref-journal">Arthritis and rheumatism. </span>1982;<span class="ref-vol">25</span>(11):1271–1277.</span> [<a href="/pubmed/7138600" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=The+1982+revised+criteria+for+the+classification+of+systemic+lupus+erythematosus&amp;author=EM+Tan&amp;author=AS+Cohen&amp;author=JF+Fries&amp;author=AT+Masi&amp;author=DJ+McShane&amp;volume=25&amp;issue=11&amp;publication_year=1982&amp;pages=1271-1277&amp;pmid=7138600&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Hochberg MC. Updating the American College of Rheumatology revised criteria for the classification of systemic lupus erythematosus. <span><span class="ref-journal">Arthritis and rheumatism. </span>1997;<span class="ref-vol">40</span>(9):1725.</span> [<a href="/pubmed/9324032" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Updating+the+American+College+of+Rheumatology+revised+criteria+for+the+classification+of+systemic+lupus+erythematosus&amp;author=MC+Hochberg&amp;volume=40&amp;issue=9&amp;publication_year=1997&amp;pages=1725&amp;pmid=9324032&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Bernatsky S, Ramsey-Goldman R, Labrecque J, Joseph L, Boivin JF, Petri M, et al.  Cancer risk in systemic lupus: an updated international multi-centre cohort study. <span><span class="ref-journal">Journal of autoimmunity. </span>2013;<span class="ref-vol">42</span>:130–135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3646904/">PMC free article</a>]</span> [<a href="/pubmed/23410586" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+autoimmunity&amp;title=Cancer+risk+in+systemic+lupus:+an+updated+international+multi-centre+cohort+study&amp;author=S+Bernatsky&amp;author=R+Ramsey-Goldman&amp;author=J+Labrecque&amp;author=L+Joseph&amp;author=JF+Boivin&amp;volume=42&amp;publication_year=2013&amp;pages=130-135&amp;pmid=23410586&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Demirci FY, Dressen AS, Kammerer CM, Barmada MM, Kao AH, Ramsey-Goldman R, et al.  Functional polymorphisms of the coagulation factor II gene (F2) and susceptibility to systemic lupus erythematosus. <span><span class="ref-journal">The Journal of rheumatology. </span>2011;<span class="ref-vol">38</span>(4):652–657.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3073870/">PMC free article</a>]</span> [<a href="/pubmed/21239755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=The+Journal+of+rheumatology&amp;title=Functional+polymorphisms+of+the+coagulation+factor+II+gene+(F2)+and+susceptibility+to+systemic+lupus+erythematosus&amp;author=FY+Demirci&amp;author=AS+Dressen&amp;author=CM+Kammerer&amp;author=MM+Barmada&amp;author=AH+Kao&amp;volume=38&amp;issue=4&amp;publication_year=2011&amp;pages=652-657&amp;pmid=21239755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Pineau CA, Bernatsky S, Abrahamowicz M, Neville C, Karp I, Clarke AE. A comparison of damage accrual across different calendar periods in systemic lupus erythematosus patients. <span><span class="ref-journal">Lupus. </span>2006;<span class="ref-vol">15</span>(9):590–594.</span> [<a href="/pubmed/17080914" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lupus&amp;title=A+comparison+of+damage+accrual+across+different+calendar+periods+in+systemic+lupus+erythematosus+patients&amp;author=CA+Pineau&amp;author=S+Bernatsky&amp;author=M+Abrahamowicz&amp;author=C+Neville&amp;author=I+Karp&amp;volume=15&amp;issue=9&amp;publication_year=2006&amp;pages=590-594&amp;pmid=17080914&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Rhew EY, Manzi SM, Dyer AR, Kao AH, Danchenko N, Barinas-Mitchell E, et al.  Differences in subclinical cardiovascular disease between African American and Caucasian women with systemic lupus erythematosus. <span><span class="ref-journal">Translational research : the journal of laboratory and clinical medicine. </span>2009;<span class="ref-vol">153</span>(2):51–59.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2674850/">PMC free article</a>]</span> [<a href="/pubmed/19138649" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Translational+research+:+the+journal+of+laboratory+and+clinical+medicine&amp;title=Differences+in+subclinical+cardiovascular+disease+between+African+American+and+Caucasian+women+with+systemic+lupus+erythematosus&amp;author=EY+Rhew&amp;author=SM+Manzi&amp;author=AR+Dyer&amp;author=AH+Kao&amp;author=N+Danchenko&amp;volume=153&amp;issue=2&amp;publication_year=2009&amp;pages=51-59&amp;pmid=19138649&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics (Oxford, England) </span>2010;<span class="ref-vol">26</span>(17):2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+(Oxford,+England)&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;issue=17&amp;publication_year=2010&amp;pages=2190-2191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nature reviews Genetics. </span>2010;<span class="ref-vol">11</span>(7):499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+reviews+Genetics&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;issue=7&amp;publication_year=2010&amp;pages=499-511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Grundberg E, Meduri E, Sandling JK, Hedman AK, Keildson S, Buil A, et al.  Global analysis of DNA methylation variation in adipose tissue from twins reveals links to disease-associated variants in distal regulatory elements. <span><span class="ref-journal">American journal of human genetics. </span>2013;<span class="ref-vol">93</span>(5):876–890.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3824131/">PMC free article</a>]</span> [<a href="/pubmed/24183450" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Global+analysis+of+DNA+methylation+variation+in+adipose+tissue+from+twins+reveals+links+to+disease-associated+variants+in+distal+regulatory+elements&amp;author=E+Grundberg&amp;author=E+Meduri&amp;author=JK+Sandling&amp;author=AK+Hedman&amp;author=S+Keildson&amp;volume=93&amp;issue=5&amp;publication_year=2013&amp;pages=876-890&amp;pmid=24183450&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Grundberg E, Small KS, Hedman AK, Nica AC, Buil A, Keildson S, et al.  Mapping cis- and trans-regulatory effects across multiple tissues in twins. <span><span class="ref-journal">Nature genetics. </span>2012;<span class="ref-vol">44</span>(10):1084–1089.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3784328/">PMC free article</a>]</span> [<a href="/pubmed/22941192" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Mapping+cis-+and+trans-regulatory+effects+across+multiple+tissues+in+twins&amp;author=E+Grundberg&amp;author=KS+Small&amp;author=AK+Hedman&amp;author=AC+Nica&amp;author=A+Buil&amp;volume=44&amp;issue=10&amp;publication_year=2012&amp;pages=1084-1089&amp;pmid=22941192&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Yang TP, Beazley C, Montgomery SB, Dimas AS, Gutierrez-Arcelus M, Stranger BE, et al.  Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies. <span><span class="ref-journal">Bioinformatics (Oxford, England) </span>2010;<span class="ref-vol">26</span>(19):2474–2476.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2944204/">PMC free article</a>]</span> [<a href="/pubmed/20702402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics+(Oxford,+England)&amp;title=Genevar:+a+database+and+Java+application+for+the+analysis+and+visualization+of+SNP-gene+associations+in+eQTL+studies&amp;author=TP+Yang&amp;author=C+Beazley&amp;author=SB+Montgomery&amp;author=AS+Dimas&amp;author=M+Gutierrez-Arcelus&amp;volume=26&amp;issue=19&amp;publication_year=2010&amp;pages=2474-2476&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Rotival M, Zeller T Fau - Wild PS, Wild Ps Fau - Maouche S, Maouche S Fau - Szymczak S, Szymczak S Fau - Schillert A, Schillert A Fau - Castagne R, et al.  Integrating genome-wide genetic variations and monocyte expression data reveals trans-regulated gene modules in humans. (1553-7404 (Electronic)) <span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3228821/">PMC free article</a>]</span> [<a href="/pubmed/22144904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Rotival+M+Zeller+T+Fau+-+Wild+PS+Wild+Ps+Fau+-+Maouche+S+Maouche+S+Fau+-+Szymczak+S+Szymczak+S+Fau+-+Schillert+A+Schillert+A+Fau+-+Castagne+R+Integrating+genome-wide+genetic+variations+and+monocyte+expression+data+reveals+trans-regulated+gene+modules+in+humans.+(1553-7404+(Electronic))+" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Kozyrev SV, Abelson AK, Wojcik J, Zaghlool A, Linga Reddy MV, Sanchez E, et al.  Functional variants in the B-cell gene BANK1 are associated with systemic lupus erythematosus. <span><span class="ref-journal">Nature genetics. </span>2008;<span class="ref-vol">40</span>(2):211–216.</span> [<a href="/pubmed/18204447" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature+genetics&amp;title=Functional+variants+in+the+B-cell+gene+BANK1+are+associated+with+systemic+lupus+erythematosus&amp;author=SV+Kozyrev&amp;author=AK+Abelson&amp;author=J+Wojcik&amp;author=A+Zaghlool&amp;author=MV+Linga+Reddy&amp;volume=40&amp;issue=2&amp;publication_year=2008&amp;pages=211-216&amp;pmid=18204447&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Sheng YJ, Gao JP, Li J, Han JW, Xu Q, Hu WL, et al.  Follow-up study identifies two novel susceptibility loci PRKCB and 8p11.21 for systemic lupus erythematosus. <span><span class="ref-journal">Rheumatology (Oxford, England) </span>2011;<span class="ref-vol">50</span>(4):682–688.</span> [<a href="/pubmed/21134959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology+(Oxford,+England)&amp;title=Follow-up+study+identifies+two+novel+susceptibility+loci+PRKCB+and+8p11.21+for+systemic+lupus+erythematosus&amp;author=YJ+Sheng&amp;author=JP+Gao&amp;author=J+Li&amp;author=JW+Han&amp;author=Q+Xu&amp;volume=50&amp;issue=4&amp;publication_year=2011&amp;pages=682-688&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Plenge RM, Bridges SL, Jr, Huizinga TW, Criswell LA, Gregersen PK. Recommendations for publication of genetic association studies in Arthritis &amp; Rheumatism. <span><span class="ref-journal">Arthritis and rheumatism. </span>2011;<span class="ref-vol">63</span>(10):2839–2847.</span> [<a href="/pubmed/21702018" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Recommendations+for+publication+of+genetic+association+studies+in+Arthritis+&amp;+Rheumatism&amp;author=RM+Plenge&amp;author=SL+Bridges&amp;author=TW+Huizinga&amp;author=LA+Criswell&amp;author=PK+Gregersen&amp;volume=63&amp;issue=10&amp;publication_year=2011&amp;pages=2839-2847&amp;pmid=21702018&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Wang C, Ahlford A, Jarvinen TM, Nordmark G, Eloranta ML, Gunnarsson I, et al.  Genes identified in Asian SLE GWASs are also associated with SLE in Caucasian populations. <span><span class="ref-journal">European journal of human genetics : EJHG. </span>2013;<span class="ref-vol">21</span>(9):994–999.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3746253/">PMC free article</a>]</span> [<a href="/pubmed/23249952" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=European+journal+of+human+genetics+:+EJHG&amp;title=Genes+identified+in+Asian+SLE+GWASs+are+also+associated+with+SLE+in+Caucasian+populations&amp;author=C+Wang&amp;author=A+Ahlford&amp;author=TM+Jarvinen&amp;author=G+Nordmark&amp;author=ML+Eloranta&amp;volume=21&amp;issue=9&amp;publication_year=2013&amp;pages=994-999&amp;pmid=23249952&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Carpenter S, Aiello D, Atianand MK, Ricci EP, Gandhi P, Hall LL, et al.  A long noncoding RNA mediates both activation and repression of immune response genes. <span><span class="ref-journal">Science (New York, NY) </span>2013;<span class="ref-vol">341</span>(6147):789–792.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4376668/">PMC free article</a>]</span> [<a href="/pubmed/23907535" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science+(New+York,+NY)&amp;title=A+long+noncoding+RNA+mediates+both+activation+and+repression+of+immune+response+genes&amp;author=S+Carpenter&amp;author=D+Aiello&amp;author=MK+Atianand&amp;author=EP+Ricci&amp;author=P+Gandhi&amp;volume=341&amp;issue=6147&amp;publication_year=2013&amp;pages=789-792&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Pagani M, Rossetti G, Panzeri I, de Candia P, Bonnal RJ, Rossi RL, et al.  Role of microRNAs and long-non-coding RNAs in CD4(+) T-cell differentiation. <span><span class="ref-journal">Immunological reviews. </span>2013;<span class="ref-vol">253</span>(1):82–96.</span> [<a href="/pubmed/23550640" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunological+reviews&amp;title=Role+of+microRNAs+and+long-non-coding+RNAs+in+CD4(+)+T-cell+differentiation&amp;author=M+Pagani&amp;author=G+Rossetti&amp;author=I+Panzeri&amp;author=P+de+Candia&amp;author=RJ+Bonnal&amp;volume=253&amp;issue=1&amp;publication_year=2013&amp;pages=82-96&amp;pmid=23550640&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Gu Z, Tan W, Feng G, Meng Y, Shen B, Liu H, et al.  Wnt/beta-catenin signaling mediates the senescence of bone marrow-mesenchymal stem cells from systemic lupus erythematosus patients through the p53/p21 pathway. <span><span class="ref-journal">Molecular and cellular biochemistry. </span>2013</span> [<a href="/pubmed/24130040" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+and+cellular+biochemistry&amp;title=Wnt/beta-catenin+signaling+mediates+the+senescence+of+bone+marrow-mesenchymal+stem+cells+from+systemic+lupus+erythematosus+patients+through+the+p53/p21+pathway&amp;author=Z+Gu&amp;author=W+Tan&amp;author=G+Feng&amp;author=Y+Meng&amp;author=B+Shen&amp;publication_year=2013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Tuller T, Atar S, Ruppin E, Gurevich M, Achiron A. Common and specific signatures of gene expression and protein-protein interactions in autoimmune diseases. <span><span class="ref-journal">Genes and immunity. </span>2013;<span class="ref-vol">14</span>(2):67–82.</span> [<a href="/pubmed/23190644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+and+immunity&amp;title=Common+and+specific+signatures+of+gene+expression+and+protein-protein+interactions+in+autoimmune+diseases&amp;author=T+Tuller&amp;author=S+Atar&amp;author=E+Ruppin&amp;author=M+Gurevich&amp;author=A+Achiron&amp;volume=14&amp;issue=2&amp;publication_year=2013&amp;pages=67-82&amp;pmid=23190644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Tveita AA, Rekvig OP. Alterations in Wnt pathway activity in mouse serum and kidneys during lupus development. <span><span class="ref-journal">Arthritis and rheumatism. </span>2011;<span class="ref-vol">63</span>(2):513–522.</span> [<a href="/pubmed/21280006" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis+and+rheumatism&amp;title=Alterations+in+Wnt+pathway+activity+in+mouse+serum+and+kidneys+during+lupus+development&amp;author=AA+Tveita&amp;author=OP+Rekvig&amp;volume=63&amp;issue=2&amp;publication_year=2011&amp;pages=513-522&amp;pmid=21280006&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">Bassuk AG, Wallace RH, Buhr A, Buller AR, Afawi Z, Shimojo M, et al.  A homozygous mutation in human PRICKLE1 causes an autosomal-recessive progressive myoclonus epilepsy-ataxia syndrome. <span><span class="ref-journal">American journal of human genetics. </span>2008;<span class="ref-vol">83</span>(5):572–581.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668041/">PMC free article</a>]</span> [<a href="/pubmed/18976727" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=A+homozygous+mutation+in+human+PRICKLE1+causes+an+autosomal-recessive+progressive+myoclonus+epilepsy-ataxia+syndrome&amp;author=AG+Bassuk&amp;author=RH+Wallace&amp;author=A+Buhr&amp;author=AR+Buller&amp;author=Z+Afawi&amp;volume=83&amp;issue=5&amp;publication_year=2008&amp;pages=572-581&amp;pmid=18976727&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Cukier HN, Dueker ND, Slifer SH, Lee JM, Whitehead PL, Lalanne E, et al.  Exome sequencing of extended families with autism reveals genes shared across neurodevelopmental and neuropsychiatric disorders. <span><span class="ref-journal">Molecular autism. </span>2014;<span class="ref-vol">5</span>(1):1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3896704/">PMC free article</a>]</span> [<a href="/pubmed/24410847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+autism&amp;title=Exome+sequencing+of+extended+families+with+autism+reveals+genes+shared+across+neurodevelopmental+and+neuropsychiatric+disorders&amp;author=HN+Cukier&amp;author=ND+Dueker&amp;author=SH+Slifer&amp;author=JM+Lee&amp;author=PL+Whitehead&amp;volume=5&amp;issue=1&amp;publication_year=2014&amp;pages=1&amp;pmid=24410847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Tao H, Manak JR, Sowers L, Mei X, Kiyonari H, Abe T, et al.  Mutations in prickle orthologs cause seizures in flies, mice, and humans. <span><span class="ref-journal">American journal of human genetics. </span>2011;<span class="ref-vol">88</span>(2):138–149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3035715/">PMC free article</a>]</span> [<a href="/pubmed/21276947" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=American+journal+of+human+genetics&amp;title=Mutations+in+prickle+orthologs+cause+seizures+in+flies,+mice,+and+humans&amp;author=H+Tao&amp;author=JR+Manak&amp;author=L+Sowers&amp;author=X+Mei&amp;author=H+Kiyonari&amp;volume=88&amp;issue=2&amp;publication_year=2011&amp;pages=138-149&amp;pmid=21276947&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Isnardi I, Ng YS, Srdanovic I, Motaghedi R, Rudchenko S, von Bernuth H, et al.  IRAK-4- and MyD88-dependent pathways are essential for the removal of developing autoreactive B cells in humans. <span><span class="ref-journal">Immunity. </span>2008;<span class="ref-vol">29</span>(5):746–757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666307/">PMC free article</a>]</span> [<a href="/pubmed/19006693" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=IRAK-4-+and+MyD88-dependent+pathways+are+essential+for+the+removal+of+developing+autoreactive+B+cells+in+humans&amp;author=I+Isnardi&amp;author=YS+Ng&amp;author=I+Srdanovic&amp;author=R+Motaghedi&amp;author=S+Rudchenko&amp;volume=29&amp;issue=5&amp;publication_year=2008&amp;pages=746-757&amp;pmid=19006693&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Staschke KA, Dong S, Saha J, Zhao J, Brooks NA, Hepburn DL, et al.  IRAK4 kinase activity is required for Th17 differentiation and Th17-mediated disease. <span><span class="ref-journal">Journal of immunology (Baltimore, Md : 1950) </span>2009;<span class="ref-vol">183</span>(1):568–577.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3638260/">PMC free article</a>]</span> [<a href="/pubmed/19542468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+immunology+(Baltimore,+Md+:+1950)&amp;title=IRAK4+kinase+activity+is+required+for+Th17+differentiation+and+Th17-mediated+disease&amp;author=KA+Staschke&amp;author=S+Dong&amp;author=J+Saha&amp;author=J+Zhao&amp;author=NA+Brooks&amp;volume=183&amp;issue=1&amp;publication_year=2009&amp;pages=568-577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Chiang EY, Yu X, Grogan JL. Immune complex-mediated cell activation from systemic lupus erythematosus and rheumatoid arthritis patients elaborate different requirements for IRAK1/4 kinase activity across human cell types. <span><span class="ref-journal">Journal of immunology (Baltimore, Md : 1950) </span>2011;<span class="ref-vol">186</span>(2):1279–1288.</span> [<a href="/pubmed/21160042" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Journal+of+immunology+(Baltimore,+Md+:+1950)&amp;title=Immune+complex-mediated+cell+activation+from+systemic+lupus+erythematosus+and+rheumatoid+arthritis+patients+elaborate+different+requirements+for+IRAK1/4+kinase+activity+across+human+cell+types&amp;author=EY+Chiang&amp;author=X+Yu&amp;author=JL+Grogan&amp;volume=186&amp;issue=2&amp;publication_year=2011&amp;pages=1279-1288&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Westra HA-OX, Peters Mj Fau - Esko T, Esko T Fau - Yaghootkar H, Yaghootkar H Fau - Schurmann C, Schurmann C Fau - Kettunen J, Kettunen J Fau - Christiansen MW, et al.  Systematic identification of trans eQTLs as putative drivers of known disease associations. (1546-1718 (Electronic)) <span></span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3991562/">PMC free article</a>]</span> [<a href="/pubmed/24013639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Westra+HA-OX+Peters+Mj+Fau+-+Esko+T+Esko+T+Fau+-+Yaghootkar+H+Yaghootkar+H+Fau+-+Schurmann+C+Schurmann+C+Fau+-+Kettunen+J+Kettunen+J+Fau+-+Christiansen+MW+Systematic+identification+of+trans+eQTLs+as+putative+drivers+of+known+disease+associations.+(1546-1718+(Electronic))+" target="pmc_ext" ref="reftype=other&amp;article-id=4747422&amp;issue-id=264923&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140640790238576title">Abstract</a></li><li><a href="#S5title">MATERIALS AND METHODS</a></li><li><a href="#S15title">RESULTS</a></li><li><a href="#S20title">DISCUSSION</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S23title">Acknowledgments</a></li><li><a href="#idm140640782197936title">Footnotes</a></li><li><a href="#idm140640789653600title">REFERENCES</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4747422/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4747422/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4747422/pdf/nihms751533.pdf">PDF (515K)</a></li> | <li><a href="#" data-citationid="PMC4747422" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747422%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747422%2F&amp;text=Identification%20of%20a%20New%20Susceptibility%20Locus%20for%20Systemic%20Lupus%20Erythematosus%20on%20Chromosome%2012%20in%20Individuals%20of%20European%20Ancestry" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747422%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26316170" data-aiid="4747422" data-aid="4747422" data-iid="264923" data-domainid="319" data-domain="nihpa" data-accid="PMC4747422" data-md5="8c997444a0c35f729f088537ce49833e"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:01:39-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A18DEDD585D510000000005170307&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4747422%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4747422/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>