<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="wtpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4745136/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4745136/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Nature communications"><meta name="citation_title" content="Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels"><meta name="citation_authors" content="Harmen H.M. Draisma, René Pool, Michael Kobl, Rick Jansen, Ann-Kristin Petersen, Anika A.M. Vaarhorst, Idil Yet, Toomas Haller, Ayşe Demirkan, Tõnu Esko, Gu Zhu, Stefan Böhringer, Marian Beekman, Jan Bert van Klinken, Werner Römisch-Margl, Cornelia Prehn, Jerzy Adamski, Anton J.M. de Craen, Elisabeth M. van Leeuwen, Najaf Amin, Harish Dharuri, Harm-Jan Westra, Lude Franke, Eco J.C. de Geus, Jouke Jan Hottenga, Gonneke Willemsen, Anjali K. Henders, Grant W. Montgomery, Dale R. Nyholt, John B. Whitfield, Brenda W. Penninx, Tim D. Spector, Andres Metspalu, P. Eline Slagboom, Ko Willems van Dijk, Peter A.C. ‘t Hoen, Konstantin Strauch, Nicholas G. Martin, Gert-Jan B. van Ommen, Thomas Illig, Jordana T. Bell, Massimo Mangino, Karsten Suhre, Mark I. McCarthy, Christian Gieger, Aaron Isaacs, Cornelia M. van Duijn, Dorret I. Boomsma"><meta name="citation_date" content="2015"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="7208"><meta name="citation_doi" content="10.1038/ncomms8208"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4745136/?report=abstract"><meta name="citation_pmid" content="26068415"><meta name="DC.Title" content="Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Europe PMC Funders"><meta name="DC.Contributor" content="Harmen H.M. Draisma"><meta name="DC.Contributor" content="René Pool"><meta name="DC.Contributor" content="Michael Kobl"><meta name="DC.Contributor" content="Rick Jansen"><meta name="DC.Contributor" content="Ann-Kristin Petersen"><meta name="DC.Contributor" content="Anika A.M. Vaarhorst"><meta name="DC.Contributor" content="Idil Yet"><meta name="DC.Contributor" content="Toomas Haller"><meta name="DC.Contributor" content="Ayşe Demirkan"><meta name="DC.Contributor" content="Tõnu Esko"><meta name="DC.Contributor" content="Gu Zhu"><meta name="DC.Contributor" content="Stefan Böhringer"><meta name="DC.Contributor" content="Marian Beekman"><meta name="DC.Contributor" content="Jan Bert van Klinken"><meta name="DC.Contributor" content="Werner Römisch-Margl"><meta name="DC.Contributor" content="Cornelia Prehn"><meta name="DC.Contributor" content="Jerzy Adamski"><meta name="DC.Contributor" content="Anton J.M. de Craen"><meta name="DC.Contributor" content="Elisabeth M. van Leeuwen"><meta name="DC.Contributor" content="Najaf Amin"><meta name="DC.Contributor" content="Harish Dharuri"><meta name="DC.Contributor" content="Harm-Jan Westra"><meta name="DC.Contributor" content="Lude Franke"><meta name="DC.Contributor" content="Eco J.C. de Geus"><meta name="DC.Contributor" content="Jouke Jan Hottenga"><meta name="DC.Contributor" content="Gonneke Willemsen"><meta name="DC.Contributor" content="Anjali K. Henders"><meta name="DC.Contributor" content="Grant W. Montgomery"><meta name="DC.Contributor" content="Dale R. Nyholt"><meta name="DC.Contributor" content="John B. Whitfield"><meta name="DC.Contributor" content="Brenda W. Penninx"><meta name="DC.Contributor" content="Tim D. Spector"><meta name="DC.Contributor" content="Andres Metspalu"><meta name="DC.Contributor" content="P. Eline Slagboom"><meta name="DC.Contributor" content="Ko Willems van Dijk"><meta name="DC.Contributor" content="Peter A.C. ‘t Hoen"><meta name="DC.Contributor" content="Konstantin Strauch"><meta name="DC.Contributor" content="Nicholas G. Martin"><meta name="DC.Contributor" content="Gert-Jan B. van Ommen"><meta name="DC.Contributor" content="Thomas Illig"><meta name="DC.Contributor" content="Jordana T. Bell"><meta name="DC.Contributor" content="Massimo Mangino"><meta name="DC.Contributor" content="Karsten Suhre"><meta name="DC.Contributor" content="Mark I. McCarthy"><meta name="DC.Contributor" content="Christian Gieger"><meta name="DC.Contributor" content="Aaron Isaacs"><meta name="DC.Contributor" content="Cornelia M. van Duijn"><meta name="DC.Contributor" content="Dorret I. Boomsma"><meta name="DC.Date" content="2015"><meta name="DC.Identifier" content="10.1038/ncomms8208"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels"><meta property="og:type" content="article"><meta property="og:description" content="Metabolites are small molecules involved in cellular metabolism, which can be detected in biological samples using metabolomic techniques. Here we present the results of genome-wide association and meta-analyses for variation in the blood serum levels ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4745136/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-wtpa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4745136/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4745136/pdf/emss-66678.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D03C1DD57CD5100000000086904CE.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/element/mml/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/input/MathML/jax.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/jax/output/HTML-CSS/jax.js?rev=2.6.1"></script></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4745136%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=epmc%20author%20manuscript[filter]" class="navlink">Europe PMC Author Manuscripts</a></li><li class="accid">PMC4745136</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-wtpa2.gif" alt="Logo of wtpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="395,0,500,52" alt="Europe PMC" title="Europe PMC" href="http://europepmc.org/" ref="reftype=publisher&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,394,52" alt="Europe PMC Funders Group" title="Europe PMC Funders Group" href="http://europepmc.org/Funders/" ref="reftype=publisher&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="53,0,500,75" alt="Submit a Manuscript" title="Submit a Manuscript" href="https://plus.europepmc.org" ref="reftype=publisher&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Nat Commun</a></span>. Author manuscript; available in PMC 2016 Feb 8.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26068415" target="pmc_ext" ref="reftype=publisher&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Nat Commun. 2015; 6: 7208. </span></a></div><span class="fm-vol-iss-date">Published online 2015 Jun 12. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1038%2Fncomms8208" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1038/ncomms8208</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4745136</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">EMSID: </span><span>EMS66678</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26068415">26068415</a></div></div></div></div><h1 class="content-title">Genome-wide association study identifies novel genetic variants contributing to variation in blood metabolite levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Draisma%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236515379104" co-class="co-affbox">Harmen H.M. Draisma</a>,<sup>#</sup><sup>1,</sup><sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Pool%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236510959776" co-class="co-affbox">René Pool</a>,<sup>#</sup><sup>1,</sup><sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Kobl%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236510957088" co-class="co-affbox">Michael Kobl</a>,<sup>5</sup> <a href="/pubmed/?term=Jansen%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236510955328" co-class="co-affbox">Rick Jansen</a>,<sup>6</sup> <a href="/pubmed/?term=Petersen%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513452928" co-class="co-affbox">Ann-Kristin Petersen</a>,<sup>5</sup> <a href="/pubmed/?term=Vaarhorst%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513450944" co-class="co-affbox">Anika A.M. Vaarhorst</a>,<sup>4,</sup><sup>7,</sup><sup>8</sup> <a href="/pubmed/?term=Yet%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513829648" co-class="co-affbox">Idil Yet</a>,<sup>9</sup> <a href="/pubmed/?term=Haller%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513827664" co-class="co-affbox">Toomas Haller</a>,<sup>10</sup> <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516773744" co-class="co-affbox">Ayşe Demirkan</a>,<sup>11,</sup><sup>12</sup> <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516771136" co-class="co-affbox">Tõnu Esko</a>,<sup>10,</sup><sup>13,</sup><sup>14,</sup><sup>15</sup> <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513854752" co-class="co-affbox">Gu Zhu</a>,<sup>16</sup> <a href="/pubmed/?term=B%26%23x000f6%3Bhringer%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513852768" co-class="co-affbox">Stefan Böhringer</a>,<sup>17</sup> <a href="/pubmed/?term=Beekman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516760160" co-class="co-affbox">Marian Beekman</a>,<sup>7</sup> <a href="/pubmed/?term=van%20Klinken%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516758176" co-class="co-affbox">Jan Bert van Klinken</a>,<sup>11</sup> <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516756192" co-class="co-affbox">Werner Römisch-Margl</a>,<sup>18</sup> <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236517705728" co-class="co-affbox">Cornelia Prehn</a>,<sup>19</sup> <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236517703744" co-class="co-affbox">Jerzy Adamski</a>,<sup>19,</sup><sup>20,</sup><sup>21</sup> <a href="/pubmed/?term=de%20Craen%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516432992" co-class="co-affbox">Anton J.M. de Craen</a>,<sup>22</sup> <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516431008" co-class="co-affbox">Elisabeth M. van Leeuwen</a>,<sup>12</sup> <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516429024" co-class="co-affbox">Najaf Amin</a>,<sup>12</sup> <a href="/pubmed/?term=Dharuri%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499682992" co-class="co-affbox">Harish Dharuri</a>,<sup>11</sup> <a href="/pubmed/?term=Westra%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499681008" co-class="co-affbox">Harm-Jan Westra</a>,<sup>23</sup> <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499679024" co-class="co-affbox">Lude Franke</a>,<sup>23</sup> <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236510865920" co-class="co-affbox">Eco J.C. de Geus</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236510863200" co-class="co-affbox">Jouke Jan Hottenga</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236514757568" co-class="co-affbox">Gonneke Willemsen</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236514754848" co-class="co-affbox">Anjali K. Henders</a>,<sup>16</sup> <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513195440" co-class="co-affbox">Grant W. Montgomery</a>,<sup>16</sup> <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236513193456" co-class="co-affbox">Dale R. Nyholt</a>,<sup>16,</sup><sup>24</sup> <a href="/pubmed/?term=Whitfield%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236511074592" co-class="co-affbox">John B. Whitfield</a>,<sup>16</sup> <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236511072608" co-class="co-affbox">Brenda W. Penninx</a>,<sup>2,</sup><sup>25</sup> <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236511069888" co-class="co-affbox">Tim D. Spector</a>,<sup>9</sup> <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499688448" co-class="co-affbox">Andres Metspalu</a>,<sup>10</sup> <a href="/pubmed/?term=Slagboom%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499686464" co-class="co-affbox">P. Eline Slagboom</a>,<sup>4,</sup><sup>7</sup> <a href="/pubmed/?term=van%20Dijk%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236515540192" co-class="co-affbox">Ko Willems van Dijk</a>,<sup>11,</sup><sup>26</sup> <a href="/pubmed/?term=%26%23x02018%3Bt%20Hoen%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236515537472" co-class="co-affbox">Peter A.C. ‘t Hoen</a>,<sup>11</sup> <a href="/pubmed/?term=Strauch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236515535488" co-class="co-affbox">Konstantin Strauch</a>,<sup>5,</sup><sup>27</sup> <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236518960720" co-class="co-affbox">Nicholas G. Martin</a>,<sup>16</sup> <a href="/pubmed/?term=van%20Ommen%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236518958736" co-class="co-affbox">Gert-Jan B. van Ommen</a>,<sup>11</sup> <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499835696" co-class="co-affbox">Thomas Illig</a>,<sup>28,</sup><sup>29,</sup><sup>30</sup> <a href="/pubmed/?term=Bell%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499832240" co-class="co-affbox">Jordana T. Bell</a>,<sup>9</sup> <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236517353632" co-class="co-affbox">Massimo Mangino</a>,<sup>9</sup> <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236517351648" co-class="co-affbox">Karsten Suhre</a>,<sup>18,</sup><sup>31,</sup><sup>32</sup> <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236507457552" co-class="co-affbox">Mark I. McCarthy</a>,<sup>33,</sup><sup>34,</sup><sup>35</sup> <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236507454096" co-class="co-affbox">Christian Gieger</a>,<sup>5,</sup><sup>28,</sup><sup>36</sup> <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499109072" co-class="co-affbox">Aaron Isaacs</a>,<sup>12</sup> <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236499107088" co-class="co-affbox">Cornelia M. van Duijn</a>,<sup>4,</sup><sup>8,</sup><sup>12,</sup><sup>**</sup> and  <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415" class="affpopup" co-rid="_co_idm140236516214528" co-class="co-affbox">Dorret I. Boomsma</a><sup>1,</sup><sup>2,</sup><sup>4,</sup><sup>**</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140236515379104"><h3 class="no_margin">Harmen H.M. Draisma</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><p><sup>3</sup>Neuroscience Campus Amsterdam, De Boelelaan 1085, Amsterdam 1081 HV, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Draisma%20HH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Harmen H.M. Draisma</a></div></div><div id="_co_idm140236510959776"><h3 class="no_margin">René Pool</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><p><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Pool%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">René Pool</a></div></div><div id="_co_idm140236510957088"><h3 class="no_margin">Michael Kobl</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><div>Find articles by <a href="/pubmed/?term=Kobl%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Michael Kobl</a></div></div><div id="_co_idm140236510955328"><h3 class="no_margin">Rick Jansen</h3><p><sup>6</sup>Department of Psychiatry, VU University Medical Center, Neuroscience Campus Amsterdam, VUmc, A.J. Ernststraat 1187, Amsterdam 1081 HL, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Jansen%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Rick Jansen</a></div></div><div id="_co_idm140236513452928"><h3 class="no_margin">Ann-Kristin Petersen</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><div>Find articles by <a href="/pubmed/?term=Petersen%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Ann-Kristin Petersen</a></div></div><div id="_co_idm140236513450944"><h3 class="no_margin">Anika A.M. Vaarhorst</h3><p><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>7</sup>Department of Molecular Epidemiology, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>8</sup>Netherlands Consortium for Healthy Aging, Leiden University Medical Center, Leiden, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Vaarhorst%20AA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Anika A.M. Vaarhorst</a></div></div><div id="_co_idm140236513829648"><h3 class="no_margin">Idil Yet</h3><p><sup>9</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, Westminster Bridge Road, London SE1 7EH, UK. </p><div>Find articles by <a href="/pubmed/?term=Yet%20I%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Idil Yet</a></div></div><div id="_co_idm140236513827664"><h3 class="no_margin">Toomas Haller</h3><p><sup>10</sup>Estonian Genome Center, University of Tartu, 23b Riia Street, Tartu 51010, Estonia. </p><div>Find articles by <a href="/pubmed/?term=Haller%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Toomas Haller</a></div></div><div id="_co_idm140236516773744"><h3 class="no_margin">Ayşe Demirkan</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Demirkan%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Ayşe Demirkan</a></div></div><div id="_co_idm140236516771136"><h3 class="no_margin">Tõnu Esko</h3><p><sup>10</sup>Estonian Genome Center, University of Tartu, 23b Riia Street, Tartu 51010, Estonia. </p><p><sup>13</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, 300 Longwood Ave, Boston MA02115, Massachusetts, USA. </p><p><sup>14</sup>Medical and Population Genetics Program, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 2142, USA. </p><p><sup>15</sup>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, NRB 0330, Boston MA 02115, Massachusetts, USA. </p><div>Find articles by <a href="/pubmed/?term=Esko%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Tõnu Esko</a></div></div><div id="_co_idm140236513854752"><h3 class="no_margin">Gu Zhu</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><div>Find articles by <a href="/pubmed/?term=Zhu%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Gu Zhu</a></div></div><div id="_co_idm140236513852768"><h3 class="no_margin">Stefan Böhringer</h3><p><sup>17</sup>Department of Medical Statistics and Bioinformatics, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=B%26%23x000f6%3Bhringer%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Stefan Böhringer</a></div></div><div id="_co_idm140236516760160"><h3 class="no_margin">Marian Beekman</h3><p><sup>7</sup>Department of Molecular Epidemiology, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Beekman%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Marian Beekman</a></div></div><div id="_co_idm140236516758176"><h3 class="no_margin">Jan Bert van Klinken</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=van%20Klinken%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Jan Bert van Klinken</a></div></div><div id="_co_idm140236516756192"><h3 class="no_margin">Werner Römisch-Margl</h3><p><sup>18</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><div>Find articles by <a href="/pubmed/?term=R%26%23x000f6%3Bmisch-Margl%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Werner Römisch-Margl</a></div></div><div id="_co_idm140236517705728"><h3 class="no_margin">Cornelia Prehn</h3><p><sup>19</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, Ingolstädter Landstraße1, Neuherberg 85764, Germany. </p><div>Find articles by <a href="/pubmed/?term=Prehn%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Cornelia Prehn</a></div></div><div id="_co_idm140236517703744"><h3 class="no_margin">Jerzy Adamski</h3><p><sup>19</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, Ingolstädter Landstraße1, Neuherberg 85764, Germany. </p><p><sup>20</sup>German Center for Diabetes Research at Helmholtz Zentrum München, Ingolstädter Landstr. 1, Neuherberg 85764, Germany. </p><p><sup>21</sup>Lehrstuhl für Experimentelle Genetik, Technische Universität München, Freising-Weihenstephan 85350, Germany. </p><div>Find articles by <a href="/pubmed/?term=Adamski%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Jerzy Adamski</a></div></div><div id="_co_idm140236516432992"><h3 class="no_margin">Anton J.M. de Craen</h3><p><sup>22</sup>Department of Gerontology and Geriatrics, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=de%20Craen%20AJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Anton J.M. de Craen</a></div></div><div id="_co_idm140236516431008"><h3 class="no_margin">Elisabeth M. van Leeuwen</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=van%20Leeuwen%20EM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Elisabeth M. van Leeuwen</a></div></div><div id="_co_idm140236516429024"><h3 class="no_margin">Najaf Amin</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Amin%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Najaf Amin</a></div></div><div id="_co_idm140236499682992"><h3 class="no_margin">Harish Dharuri</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Dharuri%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Harish Dharuri</a></div></div><div id="_co_idm140236499681008"><h3 class="no_margin">Harm-Jan Westra</h3><p><sup>23</sup>Department of Genetics, CB50, UniversityMedical Center Groningen, University of Groningen, P.O. Box 30001, Groningen 9700 RB, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Westra%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Harm-Jan Westra</a></div></div><div id="_co_idm140236499679024"><h3 class="no_margin">Lude Franke</h3><p><sup>23</sup>Department of Genetics, CB50, UniversityMedical Center Groningen, University of Groningen, P.O. Box 30001, Groningen 9700 RB, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Franke%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Lude Franke</a></div></div><div id="_co_idm140236510865920"><h3 class="no_margin">Eco J.C. de Geus</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=de%20Geus%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Eco J.C. de Geus</a></div></div><div id="_co_idm140236510863200"><h3 class="no_margin">Jouke Jan Hottenga</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Hottenga%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Jouke Jan Hottenga</a></div></div><div id="_co_idm140236514757568"><h3 class="no_margin">Gonneke Willemsen</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Willemsen%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Gonneke Willemsen</a></div></div><div id="_co_idm140236514754848"><h3 class="no_margin">Anjali K. Henders</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><div>Find articles by <a href="/pubmed/?term=Henders%20AK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Anjali K. Henders</a></div></div><div id="_co_idm140236513195440"><h3 class="no_margin">Grant W. Montgomery</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><div>Find articles by <a href="/pubmed/?term=Montgomery%20GW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Grant W. Montgomery</a></div></div><div id="_co_idm140236513193456"><h3 class="no_margin">Dale R. Nyholt</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><p><sup>24</sup>Statistical and Genomic Epidemiology, Institute of Health and Biomedical Innovation, Queensland University of Technology, 60 Musk Ave, Kelvin Grove QLD 4059, Queensland, Australia. </p><div>Find articles by <a href="/pubmed/?term=Nyholt%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Dale R. Nyholt</a></div></div><div id="_co_idm140236511074592"><h3 class="no_margin">John B. Whitfield</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><div>Find articles by <a href="/pubmed/?term=Whitfield%20JB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">John B. Whitfield</a></div></div><div id="_co_idm140236511072608"><h3 class="no_margin">Brenda W. Penninx</h3><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><p><sup>25</sup>Department of Psychiatry, VU University Medical Center, A.J. Ernststraat 1187, Amsterdam 1081 HL, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Penninx%20BW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Brenda W. Penninx</a></div></div><div id="_co_idm140236511069888"><h3 class="no_margin">Tim D. Spector</h3><p><sup>9</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, Westminster Bridge Road, London SE1 7EH, UK. </p><div>Find articles by <a href="/pubmed/?term=Spector%20TD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Tim D. Spector</a></div></div><div id="_co_idm140236499688448"><h3 class="no_margin">Andres Metspalu</h3><p><sup>10</sup>Estonian Genome Center, University of Tartu, 23b Riia Street, Tartu 51010, Estonia. </p><div>Find articles by <a href="/pubmed/?term=Metspalu%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Andres Metspalu</a></div></div><div id="_co_idm140236499686464"><h3 class="no_margin">P. Eline Slagboom</h3><p><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>7</sup>Department of Molecular Epidemiology, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Slagboom%20PE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">P. Eline Slagboom</a></div></div><div id="_co_idm140236515540192"><h3 class="no_margin">Ko Willems van Dijk</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>26</sup>Department of Endocrinology, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=van%20Dijk%20KW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Ko Willems van Dijk</a></div></div><div id="_co_idm140236515537472"><h3 class="no_margin">Peter A.C. ‘t Hoen</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=%26%23x02018%3Bt%20Hoen%20PA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Peter A.C. ‘t Hoen</a></div></div><div id="_co_idm140236515535488"><h3 class="no_margin">Konstantin Strauch</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><p><sup>27</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich 81377, Germany. </p><div>Find articles by <a href="/pubmed/?term=Strauch%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Konstantin Strauch</a></div></div><div id="_co_idm140236518960720"><h3 class="no_margin">Nicholas G. Martin</h3><p><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </p><div>Find articles by <a href="/pubmed/?term=Martin%20NG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Nicholas G. Martin</a></div></div><div id="_co_idm140236518958736"><h3 class="no_margin">Gert-Jan B. van Ommen</h3><p><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=van%20Ommen%20GJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Gert-Jan B. van Ommen</a></div></div><div id="_co_idm140236499835696"><h3 class="no_margin">Thomas Illig</h3><p><sup>28</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><p><sup>29</sup>Hannover Unified Biobank, Hannover Medical School, Hannover 30625, Germany. </p><p><sup>30</sup>Institute for Human Genetics, Hannover Medical School, Carl-Neuberg-Strasse 1, Hannover 30625, Germany. </p><div>Find articles by <a href="/pubmed/?term=Illig%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Thomas Illig</a></div></div><div id="_co_idm140236499832240"><h3 class="no_margin">Jordana T. Bell</h3><p><sup>9</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, Westminster Bridge Road, London SE1 7EH, UK. </p><div>Find articles by <a href="/pubmed/?term=Bell%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Jordana T. Bell</a></div></div><div id="_co_idm140236517353632"><h3 class="no_margin">Massimo Mangino</h3><p><sup>9</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, Westminster Bridge Road, London SE1 7EH, UK. </p><div>Find articles by <a href="/pubmed/?term=Mangino%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Massimo Mangino</a></div></div><div id="_co_idm140236517351648"><h3 class="no_margin">Karsten Suhre</h3><p><sup>18</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><p><sup>31</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried 82152, Germany. </p><p><sup>32</sup>Department of Physiology and Biophysics, Weill Cornell Medical College in Qatar (WCMC-Q), PO Box 24144, Education City—Qatar Foundation, Doha, Qatar. </p><div>Find articles by <a href="/pubmed/?term=Suhre%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Karsten Suhre</a></div></div><div id="_co_idm140236507457552"><h3 class="no_margin">Mark I. McCarthy</h3><p><sup>33</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </p><p><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Headington OX3 7LE, UK. </p><p><sup>35</sup>Oxford National Institute for Health Research Biomedical Research Centre, The Joint Research Office, Block 60, Churchill Hospital, Old Road, Headington OX3 7LE, UK. </p><div>Find articles by <a href="/pubmed/?term=McCarthy%20MI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Mark I. McCarthy</a></div></div><div id="_co_idm140236507454096"><h3 class="no_margin">Christian Gieger</h3><p><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><p><sup>28</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><p><sup>36</sup>Institute of Epidemiology II, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </p><div>Find articles by <a href="/pubmed/?term=Gieger%20C%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Christian Gieger</a></div></div><div id="_co_idm140236499109072"><h3 class="no_margin">Aaron Isaacs</h3><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Isaacs%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Aaron Isaacs</a></div></div><div id="_co_idm140236499107088"><h3 class="no_margin">Cornelia M. van Duijn</h3><p><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </p><p><sup>8</sup>Netherlands Consortium for Healthy Aging, Leiden University Medical Center, Leiden, The Netherlands. </p><p><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=van%20Duijn%20CM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Cornelia M. van Duijn</a></div></div><div id="_co_idm140236516214528"><h3 class="no_margin">Dorret I. Boomsma</h3><p><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </p><p><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </p><p><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </p><div>Find articles by <a href="/pubmed/?term=Boomsma%20DI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26068415">Dorret I. Boomsma</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140236518639360_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140236518639360_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140236518639360_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Biological Psychology, VU University Amsterdam, van der Boechorststraat 1, Amsterdam 1081 BT, The Netherlands. </div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>The EMGO+ Institute for Health and Care Research, VU University Medical Center, Van der Boechorststraat 7, Amsterdam 1081 BT, The Netherlands. </div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Neuroscience Campus Amsterdam, De Boelelaan 1085, Amsterdam 1081 HV, The Netherlands. </div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>BBMRINL: Infrastructure for the Application of Metabolomics Technology in Epidemiology (RP4), S4-P, Postbus 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Institute of Genetic Epidemiology, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Psychiatry, VU University Medical Center, Neuroscience Campus Amsterdam, VUmc, A.J. Ernststraat 1187, Amsterdam 1081 HL, The Netherlands. </div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Molecular Epidemiology, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Netherlands Consortium for Healthy Aging, Leiden University Medical Center, Leiden, The Netherlands. </div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>Department of Twin Research and Genetic Epidemiology, King’s College London, Westminster Bridge Road, London SE1 7EH, UK. </div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Estonian Genome Center, University of Tartu, 23b Riia Street, Tartu 51010, Estonia. </div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department of Human Genetics, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Genetic Epidemiology Unit, Department of Epidemiology, Erasmus Medical Center, P.O. Box 2040, Rotterdam 3000 CA, The Netherlands. </div><div class="fm-affl" id="A13" lang="en"><sup>13</sup>Divisions of Endocrinology and Genetics and Center for Basic and Translational Obesity Research, Boston Children’s Hospital, 300 Longwood Ave, Boston MA02115, Massachusetts, USA. </div><div class="fm-affl" id="A14" lang="en"><sup>14</sup>Medical and Population Genetics Program, Broad Institute of MIT and Harvard, Cambridge, Massachusetts 2142, USA. </div><div class="fm-affl" id="A15" lang="en"><sup>15</sup>Department of Genetics, Harvard Medical School, 77 Avenue Louis Pasteur, NRB 0330, Boston MA 02115, Massachusetts, USA. </div><div class="fm-affl" id="A16" lang="en"><sup>16</sup>Department of Genetics and Computational Biology, QIMR Berghofer Medical Research Institute, 300 Herston Road, Brisbane 4006, Australia. </div><div class="fm-affl" id="A17" lang="en"><sup>17</sup>Department of Medical Statistics and Bioinformatics, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A18" lang="en"><sup>18</sup>Institute of Bioinformatics and Systems Biology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </div><div class="fm-affl" id="A19" lang="en"><sup>19</sup>Institute of Experimental Genetics, Genome Analysis Center, Helmholtz Zentrum München, Ingolstädter Landstraße1, Neuherberg 85764, Germany. </div><div class="fm-affl" id="A20" lang="en"><sup>20</sup>German Center for Diabetes Research at Helmholtz Zentrum München, Ingolstädter Landstr. 1, Neuherberg 85764, Germany. </div><div class="fm-affl" id="A21" lang="en"><sup>21</sup>Lehrstuhl für Experimentelle Genetik, Technische Universität München, Freising-Weihenstephan 85350, Germany. </div><div class="fm-affl" id="A22" lang="en"><sup>22</sup>Department of Gerontology and Geriatrics, Leiden University Medical Center, PO Box 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A23" lang="en"><sup>23</sup>Department of Genetics, CB50, UniversityMedical Center Groningen, University of Groningen, P.O. Box 30001, Groningen 9700 RB, The Netherlands. </div><div class="fm-affl" id="A24" lang="en"><sup>24</sup>Statistical and Genomic Epidemiology, Institute of Health and Biomedical Innovation, Queensland University of Technology, 60 Musk Ave, Kelvin Grove QLD 4059, Queensland, Australia. </div><div class="fm-affl" id="A25" lang="en"><sup>25</sup>Department of Psychiatry, VU University Medical Center, A.J. Ernststraat 1187, Amsterdam 1081 HL, The Netherlands. </div><div class="fm-affl" id="A26" lang="en"><sup>26</sup>Department of Endocrinology, Leiden University Medical Center, S4-P, PO Box 9600, Leiden 2300 RC, The Netherlands. </div><div class="fm-affl" id="A27" lang="en"><sup>27</sup>Institute of Medical Informatics, Biometry and Epidemiology, Chair of Genetic Epidemiology, Ludwig-Maximilians-Universität, Munich 81377, Germany. </div><div class="fm-affl" id="A28" lang="en"><sup>28</sup>Research Unit of Molecular Epidemiology, Helmholtz Zentrum München, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </div><div class="fm-affl" id="A29" lang="en"><sup>29</sup>Hannover Unified Biobank, Hannover Medical School, Hannover 30625, Germany. </div><div class="fm-affl" id="A30" lang="en"><sup>30</sup>Institute for Human Genetics, Hannover Medical School, Carl-Neuberg-Strasse 1, Hannover 30625, Germany. </div><div class="fm-affl" id="A31" lang="en"><sup>31</sup>Faculty of Biology, Ludwig-Maximilians-Universität, Planegg-Martinsried 82152, Germany. </div><div class="fm-affl" id="A32" lang="en"><sup>32</sup>Department of Physiology and Biophysics, Weill Cornell Medical College in Qatar (WCMC-Q), PO Box 24144, Education City—Qatar Foundation, Doha, Qatar. </div><div class="fm-affl" id="A33" lang="en"><sup>33</sup>Wellcome Trust Centre for Human Genetics, University of Oxford, Roosevelt Drive, Oxford OX3 7BN, UK. </div><div class="fm-affl" id="A34" lang="en"><sup>34</sup>Oxford Centre for Diabetes, Endocrinology and Metabolism, University of Oxford, Churchill Hospital, Headington OX3 7LE, UK. </div><div class="fm-affl" id="A35" lang="en"><sup>35</sup>Oxford National Institute for Health Research Biomedical Research Centre, The Joint Research Office, Block 60, Churchill Hospital, Old Road, Headington OX3 7LE, UK. </div><div class="fm-affl" id="A36" lang="en"><sup>36</sup>Institute of Epidemiology II, Helmholtz Zentrum München—German Research Center for Environmental Health, Ingolstädter Landstraße 1, Neuherberg 85764, Germany. </div><div><sup>#</sup>Contributed equally.</div><div id="CR1">Correspondence and requests for materials should be addressed to H.H.M.D. (<a href="mailto:dev@null" data-email="ln.uv@amsiard.m.h.h" class="oemail">ln.uv@amsiard.m.h.h</a>)</div><div id="FN1"><sup>**</sup>These authors jointly supervised this work.</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140236518639360_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26068415" target="pmc_ext" ref="reftype=publisher&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Nat Commun</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4745136/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="9" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData1.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData1.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData1.xlsx</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;81395D54-051C-40FE-B5C1-4C3C38570D8B</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData2.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData2.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData2.xlsx</a><span style="color:gray"> (25K)</span></div><div class="small guid">GUID:&nbsp;D34EA922-72BE-4A0B-8D49-F339F53AEA9D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData3.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData3.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData3.xlsx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID:&nbsp;05B82F18-367B-4694-8B27-9252C73C484E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData4.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData4.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData4.xlsx</a><span style="color:gray"> (9.4K)</span></div><div class="small guid">GUID:&nbsp;F2233A8E-7F12-4717-8882-24B7A927F057</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData5.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData5.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData5.xlsx</a><span style="color:gray"> (13K)</span></div><div class="small guid">GUID:&nbsp;FE3BC8DC-DF94-4BA9-80B7-A4E21CED5264</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData6.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData6.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData6.xlsx</a><span style="color:gray"> (7.2K)</span></div><div class="small guid">GUID:&nbsp;5E64EC1D-0234-4E2B-A647-521B64900A10</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryData7.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData7.xlsx" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryData7.xlsx</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;64375FF5-73EB-4AFB-8651-444D17037C50</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryFiguresTablesAndNotes.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryFiguresTablesAndNotes.pdf" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryFiguresTablesAndNotes.pdf</a><span style="color:gray"> (19M)</span></div><div class="small guid">GUID:&nbsp;6FD55E89-E00F-4EBF-AFB6-F6A36CDA5F8E</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>SupplementaryMovie.</strong></div><div><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryMovie.mp4" data-ga-action="click_feat_suppl">NIHMS66678-supplement-SupplementaryMovie.mp4</a><span style="color:gray"> (11M)</span></div><div class="small guid">GUID:&nbsp;3A165051-9C02-485D-82C4-637DEDF7C220</div></div></dd></dl></div><div id="idm140236519484736" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140236519484736title">Abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">Metabolites are small molecules involved in cellular metabolism, which can be detected in biological samples using metabolomic techniques. Here we present the results of genome-wide association and meta-analyses for variation in the blood serum levels of 129 metabolites as measured by the Biocrates metabolomic platform. In a discovery sample of 7,478 individuals of European descent, we find 4,068 genome- and metabolome-wide significant (<em>Z</em>-test, <em>P</em> &lt; 1.09 × 10<sup>−9</sup>) associations between single nucleotide polymorphisms (SNPs) and metabolites, involving 59 independent SNPs and 85 metabolites. Five of the fifty-nine independent SNPs are new for serum metabolite levels, and were followed-up for replication in an independent sample (<em>N</em>=1,182). The novel SNPs are located in or near genes encoding metabolite transporter proteins or enzymes (<em>SLC22A16, ARG1, AGPS</em> and <em>ACSL1</em>) that have demonstrated biomedical or pharmaceutical importance. The further characterization of genetic influences on metabolic phenotypes is important for progress in biological and medical research.</p></div></div><div id="idm140236589761472" class="tsec sec headless whole_rhythm"><p id="P5" class="p p-first">Metabolite levels in human blood reflect the physiological state of the body, and may differ between individuals due to variation in genetic makeup and environmental exposure<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789147">1</a></sup>. The study of the genetic contribution to variation in metabolite levels is an important basis for improved etiological understanding, prevention, diagnosis and treatment of complex disorders<sup><a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789131">1</a>,<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789107">2</a></sup>. Modern high-throughput metabolomics enables the cost-effective measurement of large metabolite panels in blood samples obtained from many individuals. The data generated by such metabolomic experiments have been combined with genotypic data in several recent genome-wide association (GWA) studies<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789102">2</a>-<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789115">12</a></sup>. Indeed, the combined investigation of large numbers of genetic variants and large numbers of metabolic traits is beginning to draw a systems-wide overview of genetic influences on human metabolism<sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789109">11</a></sup>. However, the heritability estimates from twin and family studies<sup><a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789130">9</a>-<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789118">11</a>,<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789116">13</a></sup> suggest that additional genetic variants influencing variation in serum metabolite levels remain to be found in GWA studies.</p><p id="P6">In the current study we set out to further characterize the genetic contribution to variation in human blood metabolite levels. We perform GWA and meta-analyses for the concentrations of 129 serum metabolites in seven independent cohorts, with replication analyses in one additional cohort. To functionally characterize the significant single-nucleotide polymorphism (SNP)-metabolite associations, we integrate the results of the GWA meta-analyses with those from gene expression analysis in whole blood and the liver. Finally, we compare the variance explained by significantly associated SNPs with heritability estimates for each metabolite.</p><p id="P7" class="p">We identify 4,068 significant SNP-metabolite associations, involving 59 independent SNPs and 85 different metabolites. Five of the fifty-nine independent SNPs are novel for serum metabolite levels. The newly found SNP-metabolite associations may lead to a better understanding of cardiovascular and metabolic disease, and may have implications for chemotherapy. Our findings contribute to the understanding of human metabolism.</p></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Results</h2><div id="S2" class="sec sec-first"><h3 id="S2title">Discovery meta-analysis of GWA scans</h3><p id="P8" class="p p-first">Primary genetic association analyses were carried out in seven cohorts (TwinsUK, KORA, EGCUT, LLS, QIMR, ERF and NTR) with a combined sample size of 7,478 individuals. Characteristics of the study participants included in the analyses (all of European descent) are given in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 1</a>. Within each cohort, SNP genotypes were imputed and analysed for association with the concentrations of each metabolite, assuming a linear model of association and correcting for population stratification (see Methods and <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 2</a>). <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Tables 3</a>–<a href="#SD8" rid="SD8" class=" supplementary-material">5</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Data 1</a> list the characteristics of the 129 metabolites (18 acylcarnitines, 14 amino acids, 82 glycerophospholipids, 14 sphingolipids and hexose) that were measured in the serum samples from all study participants using the Biocrates platform. The cohort-level GWA results were pooled in inverse variance-weighted, fixed-effects meta-analysis. The values of the genomic control lambda ( λ<sub>GC</sub>, applied to the individual cohortlevel results for each metabolite before meta-analysis) varied between 0.976 and 1.081 across all metabolites and cohorts (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 6</a>), suggesting little residual influence on the GWA results of population stratification and other potential confounders. A three-dimensional Manhattan plot providing an overview of the association <em>P</em> values in the discovery phase for all metabolites is given in <a href="/pmc/articles/PMC4745136/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Fig. 1</span></a>; two-dimensional Manhattan plots and quantile-quantile plots for each metabolite separately are given in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Figs 1</a> and <a href="#SD8" rid="SD8" class=" supplementary-material">2</a>, respectively. Overall, 4,068 SNP-metabolite associations reached genome- and metabolome-wide significance (<em>Z</em>-test, <em>P</em> &lt; 1.09 × 10<sup>−9</sup> ), which reduced to 123 associations involving 59 independent SNPs and 85 different metabolites. Of these 123 associations (listed in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Data 2</a>), 4 represented secondary association signals according to approximate conditional analysis. Regional association plots, showing the association signals in the regions surrounding the lead metabolomic SNPs, are given for all 123 associations in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Fig. 3</a>. SNPs representing independent association signals were aggregated into 31 genomic loci, which are listed in <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Data 3</a>. <a href="/pmc/articles/PMC4745136/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a> depicts all associations between loci and metabolites as detected in the discovery phase.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4745136/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140236509798624" class="figure"><a href="/pmc/articles/PMC4745136/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4745136_emss-66678-f0001.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-66678-f0001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4745136/bin/emss-66678-f0001.jpg"></a></div><div id="largeobj_idm140236509798624" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4745136/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4745136/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p2">Manhattan plots for all metabolites targeted by the Biocrates AbsoluteIDQ p150 kit (<em>N</em> = [1497, 7478]). These plots graphically display the <em>P</em> values for significant (<em>Z</em> test <em>P</em> &lt; 1.09 × 10<sup>−9</sup>) SNP metabolite associations in the discovery phase in the current study. Panel (<strong>a</strong>) provides a three-dimensional view; orthogonal projections are given in panels (<strong>b</strong>) and (<strong>c</strong>). SNPs are arranged according to genomic location along the 'chromosome' axes. The ordering of the metabolites along the 'metabolite index' axes is equal in both panels (<strong>a</strong>) and (<strong>c</strong>), and equal to that in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 3</a>. In panels (<strong>a</strong>) and (<strong>b</strong>), all data points are displayed semi-transparent and therefore opaque regions in the plot indicate clusters of significant associations. In panel (<strong>b</strong>), loci are identified by most plausible causal gene or, if no plausible genes found, by nearest gene. Where multiple plausible genes could be identified at the locus (possibly for different metabolites), the gene names are separated by an underscore (“_”) in the locus name. In panel (<strong>c</strong>), the size of the markers scales linearly with −log<sub>10</sub>(<em>P</em> value). This Figure is also supplied as a movie (see <a href="#SD9" rid="SD9" class=" supplementary-material">Supplementary Movie 1</a>).</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4745136/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140236499944400" class="figure"><a href="/pmc/articles/PMC4745136/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4745136_emss-66678-f0002.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-66678-f0002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4745136/bin/emss-66678-f0002.jpg"></a></div><div id="largeobj_idm140236499944400" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4745136/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4745136/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p3">Associations between loci and metabolites detected in stage 1 meta-analysis in the current study (<em>N</em> = [1588, 7478]). Loci significantly associated with at least one metabolite are depicted as grey circles. Biochemical classes (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 3</a>) of the metabolites (hexagons) are indicated by node colors: green, acylcarnitines; blue, amino acids; purple, glycerophospholipids; yellow, sphingolipids. Arrows point from each locus to the associated metabolite(s); arrow widths scale linearly with −log<sub>10</sub>(association <em>P</em> value). Grey arrows denote previously known associations; red arrows denote associations that were newly discovered on the basis of stage 1 meta-analysis in the current study (<em>i.e.</em>, either associations with new SNPs for serum metabolite levels, or an association of a known SNPs with a new metabolite with respect to 11 previous GWA studies for serum metabolite levels<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789146">2</a>-<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789132">12</a></sup>). Loci are identified by most plausible causal gene or, if no plausible genes found, by nearest gene. Where multiple plausible genes could be identified at the locus (possibly for different metabolites), the gene names are separated by an underscore (“_”) in the locus name. At this significance threshold (<em>P</em>=1.09 × 10<sup>−9</sup>), the locus-metabolite association network separates into 12 connected components or disconnected subnetworks, each including metabolites from maximally two chemical classes. This figure was created using Cytoscape<sup><a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789134">52</a></sup>.</p></div></div></div><p id="P9" class="p p-last">Five independent SNPs had not been associated with variation in serum metabolite levels in previous GWA studies (see <a href="/pmc/articles/PMC4745136/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>). To further interpret the association of the remaining 54 SNPs with serum metabolite concentrations, we compared our findings with those from 11 published GWA studies<sup><a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789103">2</a>–<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789141">12</a></sup> for which at least one of the included metabolites overlapped with the current study. The identified associations of known SNPs with metabolites that were significant in discovery stage meta-analysis in the current study and that had not been reported in those previous studies are highlighted in <a href="/pmc/articles/PMC4745136/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Fig. 2</span></a> and in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Data 2</a>.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p5">Novel SNPs for serum metabolite levels identified in the current study.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140236550173504" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Lead<br>metabolomic<br>SNP</th><th rowspan="1" colspan="1" valign="top" align="left">Lead<br>metabolite</th><th rowspan="1" colspan="1" valign="top" align="left">Cytoband</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value in<br>discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">EA/NEA</th><th rowspan="1" colspan="1" valign="top" align="left">EAF in<br>discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Beta in<br>discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Total <em>N</em> in<br>discovery<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">Nearest gene</th><th rowspan="1" colspan="1" valign="top" align="left">Beta in<br>replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left"><em>P</em> value in<br>replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">EAF in<br>replication<br>phase</th><th rowspan="1" colspan="1" valign="top" align="left">replicated</th><th rowspan="1" colspan="1" valign="top" align="left">Locus name</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs12210538</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">C18:2</td><td rowspan="1" colspan="1" valign="top" align="left">6q21</td><td rowspan="1" colspan="1" valign="top" align="left">5.03 × 10<sup>−21</sup></td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">80.8%</td><td rowspan="1" colspan="1" valign="top" align="right">0.086</td><td rowspan="1" colspan="1" valign="top" align="right">6,574</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SLC22A16</em>
</td><td rowspan="1" colspan="1" valign="top" align="right">0.099</td><td rowspan="1" colspan="1" valign="top" align="right">2.65 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="left">75.9%</td><td rowspan="1" colspan="1" valign="top" align="left">*</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>SLC22A16_</em><br><em>SLC16A10</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs17657817</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">Orn</td><td rowspan="1" colspan="1" valign="top" align="left">6q23.2</td><td rowspan="1" colspan="1" valign="top" align="left">1.32 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">98.0%</td><td rowspan="1" colspan="1" valign="top" align="right">−0.156</td><td rowspan="1" colspan="1" valign="top" align="right">2,991</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ARG1</em>
</td><td rowspan="1" colspan="1" valign="top" align="right">−0.123</td><td rowspan="1" colspan="1" valign="top" align="right">1.40 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">97.5%</td><td rowspan="1" colspan="1" valign="top" align="left">*</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ARG1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs2246012</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">Orn</td><td rowspan="1" colspan="1" valign="top" align="left">6q23.2</td><td rowspan="1" colspan="1" valign="top" align="left">6.43 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">83.9%</td><td rowspan="1" colspan="1" valign="top" align="right">0.045</td><td rowspan="1" colspan="1" valign="top" align="right">7,476</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ARG1</em>
</td><td rowspan="1" colspan="1" valign="top" align="right">0.044</td><td rowspan="1" colspan="1" valign="top" align="right">1.57 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">84.5%</td><td rowspan="1" colspan="1" valign="top" align="left">*</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ARG1</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs7582179</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">PC ae C44:5</td><td rowspan="1" colspan="1" valign="top" align="left">2q31.2</td><td rowspan="1" colspan="1" valign="top" align="left">4.07 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">A/G</td><td rowspan="1" colspan="1" valign="top" align="left">16.8%</td><td rowspan="1" colspan="1" valign="top" align="right">−0.048</td><td rowspan="1" colspan="1" valign="top" align="right">5,360</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>AGPS</em>
</td><td rowspan="1" colspan="1" valign="top" align="right">−0.021</td><td rowspan="1" colspan="1" valign="top" align="right">0.147</td><td rowspan="1" colspan="1" valign="top" align="left">16.5%</td><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left">
<em>AGPS</em>
</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">
<strong>rs7700133</strong>
</td><td rowspan="1" colspan="1" valign="top" align="left">PC ae C44:5</td><td rowspan="1" colspan="1" valign="top" align="left">4q35.1</td><td rowspan="1" colspan="1" valign="top" align="left">3.35 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="left">T/C</td><td rowspan="1" colspan="1" valign="top" align="left">30.5%</td><td rowspan="1" colspan="1" valign="top" align="right">0.036</td><td rowspan="1" colspan="1" valign="top" align="right">7,476</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>CENPU</em>
</td><td rowspan="1" colspan="1" valign="top" align="right">0.038</td><td rowspan="1" colspan="1" valign="top" align="right">1.12 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">30.9%</td><td rowspan="1" colspan="1" valign="top" align="left">*</td><td rowspan="1" colspan="1" valign="top" align="left">
<em>ACSL1</em>
</td></tr></tbody></table></div><div id="largeobj_idm140236550173504" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4745136/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><p id="P32" class="p p-first-last">Lead metabolite, metabolite displaying strongest association with SNP in discovery phase GWA meta-analysis in current study. EA/NEA, effect allele / non-effect allele. EAF, frequency of EA. <em>P</em> values in discovery and replication phases were calculated by <em>Z</em>- and <em>t</em>-tests, respectively. Loci that were replicated in the KORA S4 replication sample (<em>P</em> &lt; 0.05 after Bonferroni correction for 5 tests) are indicated by *. Loci are identified by most plausible causal gene or, if no plausible genes found, by nearest gene. Where multiple plausible genes could be identified at the locus (possibly for different metabolites), the gene names are separated by an underscore (“_”) in the locus name.</p></div></div></div></div><div id="S3" class="sec"><h3 id="S3title">Replication analysis</h3><p id="P10" class="p p-first-last">Replication analyses were performed in an independent sample (<em>N</em> = 1,182) from the KORA S4 cohort (hereafter KORA S4 replication sample) for the five new SNPs for serum metabolite levels that had been found in the discovery phase meta-analysis. The associations with their most strongly associated metabolite were replicated for four of these five novel SNPs; the only non-replicated association was that between SNP rs7582179 and metabolite PC ae C44:5. Although the effect sign was concordant between the discovery set and the KORA S4 replication sample (<a href="/pmc/articles/PMC4745136/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>), this association was significant in the discovery phase for the NTR and KORA cohorts only (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Fig. 4</a>).</p></div><div id="S4" class="sec"><h3 id="S4title">Integration with gene expression analysis results</h3><p id="P11" class="p p-first-last">We integrated the results of the metabolomics discovery stage GWA meta-analysis with the results of gene expression analyses in whole blood and the liver. In whole blood, both <em>cis</em> and <em>trans</em> expression quantitative trait locus (<em>cis</em>-eQTL and <em>trans</em>-eQTL, respectively) analyses were performed in two different samples originating from the United Kingdom, the Netherlands and Estonia: the Dutch NTR-NESDA sample (<em>N</em> = 5,071) and the Fehrmann-EGCUT sample comprising data from three cohorts that were meta-analysed (total <em>N</em> = 2,360; see Methods and <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Methods</a>). The results of <em>cis</em>-eQTL analysis for lead metabolomics SNPs showing overlap with <em>cis</em>-eQTL SNPs are given for the NTR-NESDA and Fehrmann-EGCUT samples in <a href="#SD4" rid="SD4" class=" supplementary-material">Supplementary Data 4</a> and <a href="#SD5" rid="SD5" class=" supplementary-material">5</a>, respectively. Significant (false discovery rate &lt; 0.05) <em>trans</em>-eQTL effects for lead metabolomic SNPs in the Fehrmann-EGCUT sample are listed in <a href="#SD6" rid="SD6" class=" supplementary-material">Supplementary Data 6</a>. We did not detect <em>trans</em>-eQTL effects for the lead metabolomic SNPs in the NTR-NESDA sample. Thirty-five lead metabolomic SNPs identified <em>cis</em>-eQTLs in at least one of the searched tissues (<em>i.e.</em>, whole blood and/or the liver) with a (<em>t</em>-,<em>Z</em>- or Kruskal-Wallis test) <em>P</em> &lt; 0.001, defining a total of 67 SNP-gene pairs and 28 different genes (see <a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Data 7</a>). The <em>cis</em>-eQTL analysis results were used to support the annotation of likely causal genes to loci that displayed significant association with variation in serum metabolite concentrations in the discovery stage meta-analysis (see <a href="#SD3" rid="SD3" class=" supplementary-material">Supplementary Data 3</a>). Of the 28 genes, 14 were predicted to be causal on the basis of our annotation and the other 14 were predicted to be non-causal.</p></div><div id="S5" class="sec sec-last"><h3 id="S5title">Variance explained</h3><p id="P12" class="p p-first-last">It has been described previously that a relatively small number of genetic variants can explain a relatively large proportion of the variance observed for serum metabolite levels<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789140">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789126">5</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789125">9</a></sup>. Therefore, we compared the variance in serum metabolite levels explained by significantly associated SNPs, with the heritability as estimated in a monozygotic twin sample from the NTR cohort (<em>N</em> = 181 pairs; see <a href="/pmc/articles/PMC4745136/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Fig. 3</span></a>)<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789098">13</a></sup> . Among all metabolites, the largest proportion of phenotypic variance was explained for C9 (13%; see <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Data 2</a>). In the current study, this metabolite associated significantly with SNPs in the <em>THEM4</em> and <em>CPS1_ACADL</em> loci. The largest proportion of heritability was explained for lysoPC a C20:4 (19%; corresponding with 10% of the phenotypic variance), which was associated with a SNP in the <em>FADS1–3</em> locus. The results of polygenic score (PGS) analyses (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Fig. 5</a> and <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Note 1</a>) suggest different genetic background of variation in serum levels for different metabolites, ranging from close-to-monogenic to highly polygenic.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4745136/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140236550176080" class="figure"><a href="/pmc/articles/PMC4745136/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4745136_emss-66678-f0003.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is emss-66678-f0003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4745136/bin/emss-66678-f0003.jpg"></a></div><div id="largeobj_idm140236550176080" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4745136/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4745136/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p4">Decomposition of variation in serum metabolite levels. This figure displays the proportions of variance in serum metabolite level explained by significantly associated SNPs; heritability not explained by significantly associated SNPs; and unexplained (environmental) variance. Seventy six metabolites are included for which both heritability estimates (monozygotic twin correlations taken from <sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789113">reference13</a></sup>; <em>N</em> = 181 pairs; Pearson correlation) were available, and that displayed genome and metabolome wide associations with SNPs in stage 1 GWA meta-analysis in the current study (<em>N</em> = [1588, 7478]). Proportion of variance explained by significantly associated SNPs was estimated as Pearson’s phi coefficient squared. Metabolites are grouped according to biochemical class.</p></div></div></div></div></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Discussion</h2><p id="P13" class="p p-first">We set out to enhance the current understanding of the genetic underpinnings of variation in circulating metabolite levels in humans. To this end, we employed a well-established targeted metabolomics platform (Biocrates) in combination with genome-wide SNP genotyping and imputation in eight independent cohorts of European descent. By meta-analysis of GWA analyses carried out for each of 129 metabolites measured in the serum samples of all individual study participants, the current study identified 123 significant SNP-metabolite associations between 59 independent SNPs and 85 different metabolites. Five of the independent SNPs were new for variation in serum metabolite levels.</p><p id="P14">Consistent with previous reports, for the majority of all 59 independent SNPs we were able to annotate a likely causal gene, which in most cases encoded a metabolite transporter protein or enzyme. The five new SNPs for serum metabolite levels are also all located nearby such genes, and their associations with metabolites tend to match the known function of these genes. SNP rs7582179 in the <em>AGPS</em> gene is associated with the choline plasmalogen PC ae C44:5. Mutations in <em>AGPS</em> (encoding the enzyme alkylglycerone phosphate synthase) are known to cause rhizomelic chondrodysplasia punctata type 3 (RCDP3; OMIM:600121 (<sup><a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789133">ref. 14</a></sup>)), a rare autosomal recessive disorder that is fatal, with death occurring often early in childhood. Clinically, RCDP3 is characterized by significantly delayed and abnormal physical and mental development, with shortness of the proximal limb bones (‘rhizomelia’) being one of the hallmarks. RCDP3 has been shown to result from reduced production of plasmalogens (a type of ether phospholipids) by alkylglycerone phosphate synthase in peroxisomes. The association in the current study of a SNP within the <em>AGPS</em> gene with the serum concentration of the choline plasmalogen PC ae C44:5 is therefore perfectly concordant with the known gene-disease link between <em>AGPS</em> and RCDP3. PC ae C44:5 also associated significantly with the new SNP rs7700133 located near the <em>ACSL1</em> gene, encoding long-chain acyl CoA synthetase 1. Previous studies have shown links between genetic and transcriptional variation of <em>ACSL1</em> and the metabolic syndrome<sup><a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789095">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789112">16</a></sup>. The new SNP rs12210538, located within the <em>SLC22A16</em> gene, associated with the two acylcarnitines C18:1 and C18:2. This gene encodes a carnitine transporter that mediates the uptake of anticancer drugs such as bleomycin and doxorubicin into tumour cells, and its activity correlates with treatment response<sup><a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789135">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789139">18</a></sup>. Significant associations were found for two SNPs (rs17657817 and rs2246012) located inside the <em>ARG1</em> gene (coding for the enzyme arginase) with serum concentrations of the amino acid ornithine that participates in the urea cycle. Importantly, the global arginine bioavailability ratio (that is, the ratio of arginine to ornithine and citrulline<sup><a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789111">19</a></sup>) is of interest as a potential biomarker for endothelial dysfunction, which is a known risk factor for the development of cardiovascular disease<sup><a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789108">20</a></sup>. The two newly identified SNPs associated with serum ornithine levels might now be used as instrumental variables in cost-effective Mendelian randomization studies in large samples of individuals, to investigate the possible causal relationship among ornithine, endothelial dysfunction and subsequent cardiovascular disease<sup><a href="#R21" rid="R21" class=" bibr popnode">21</a></sup>. Among three meta-analyses of coronary artery disease and myocardial infarction as carried out by the CARDIoGRAMplusC4D Consortium, the association with SNP rs2246012 was the strongest in the CARDIoGRAM GWA study (<em>P</em> &lt; 0.002) involving 22,233 cases and 64,762 controls<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789119">22</a></sup>. This suggests that the link between genetic variation at SNP rs2246012 and variation in serum ornithine levels as identified in the current study will indeed be useful to further establish the possible link between the <em>ARG1</em> gene and cardiovascular disease.</p><p id="P15">We compared the significant SNP-metabolite associations from the current study with those reported in 11 previous publications that employed high-resolution methods to assay the serum metabolome. For several SNPs that were associated with variation in metabolite levels in the previous studies, we identified new associations with individual metabolites. These new associations strengthen the evidence for the associations of these known SNPs with specific metabolites, demonstrating their extended effect on phenotypes that are closely related to the metabolites with which their association was discovered initially<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789127">4</a>,<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789110">23</a></sup>. Also, it has been demonstrated that GWA studies with the more refined metabolic phenotypes provided by metabolomics often yield effect sizes that are larger than those observed in GWA studies of composite measures such as high-density lipoprotein cholesterol, suggesting that these more refined metabolomic phenotypes provide better intermediate traits<sup><a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789117">8</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789148">9</a></sup>. In this context, it is interesting to note that the large proportion of explained heritability we observed for lysoPC a C20:4 (19%) was caused exclusively by the association with a single SNP in the <em>FADS1–3</em> locus, an observation that is in line with the results from previous studies<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789091">4</a>,<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789143">5</a>,<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789128">9</a></sup>.</p><p id="P16" class="p p-last">In conclusion, the results obtained in the current study contribute to the understanding of the genetic background of variation in serum metabolite levels, and are important for further progress in biomedical and pharmaceutical research.</p></div><div id="S7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S7title">Methods</h2><div id="S8" class="sec sec-first"><h3 id="S8title">Participants</h3><p id="P17" class="p p-first-last">The meta-analysis included GWA data from 7,478 participants from seven cohorts originating from five countries (the Netherlands, Germany, Australia, Estonia and the United Kingdom). The independent test sample (‘KORA S4 replication sample’) consisted of 1,182 additional KORA participants. The following local research ethics committees approved the individual studies: KORA, Ethics Committee of the Bavarian Medical Association (Bayerische Landesärztekammer); NTR, Central Ethics Committee on Research Involving Human Subjects of the VU University Medical Center, Amsterdam; EGCUT, Ethics Review Committee on Human Research of the University of Tartu; TwinsUK, St Thomas’ Hospital ethics committee; ERF, medical ethics board of the Erasmus MC Rotterdam, the Netherlands; LLS, Medical Ethical Committee of the Leiden University Medical Centre; QIMR, QIMR Human Research Ethics Committee. Informed consent was obtained from all participants. Sample characteristics for all cohorts included in this study and detailed study sample descriptions are given in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 1</a> and in the <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Methods</a>, respectively.</p></div><div id="S9" class="sec"><h3 id="S9title">Biocrates metabolite quantification</h3><p id="P18" class="p p-first-last">Targeted metabolomic measurements were performed using electrospray–flow injection analysis–tandem mass spectrometry methods and the Biocrates AbsoluteIDQ p150 kit (BIOCRATES Life Sciences AG), which enables quantification of a total of 163 metabolites (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 3</a> for an overview of all metabolites targeted by this kit)<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup>. The method of AbsoluteIDQ p150 kit has been proven to be in conformance with FDA Guideline ‘Guidance for Industry—Bioanalytical Method Validation (May 2001)’<sup><a href="#R25" rid="R25" class=" bibr popnode">25</a></sup>, which implies proof of reproducibility within a given error range. For all cohorts, metabolite measurements were carried out at the Metabolomics Platform of the Genome Analysis Center at the Helmholtz Zentrum München, Germany as per the manufacturer’s instructions<sup><a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789089">4</a>,<a href="#R24" rid="R24" class=" bibr popnode">24</a>,<a href="#R26" rid="R26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789138">26</a>,<a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789090">27</a></sup>. In brief, the used metabolomic measurement technique is based on a targeted profiling scheme that is used to quantitatively screen for known small-molecule metabolites by multiple reaction monitoring, neutral loss and precursor-ion scans. Internal standards served as reference for the calculation of all metabolite concentrations, which are reported as micromolar. Data evaluation for quantification of metabolite concentrations and quality assessment have been performed with the MetIDQ software package, which is an integral part of the AbsoluteIDQ kit. Stability of the assay was assessed using the measurement results of five aliquots of the same reference blood sample on every plate. Quality control of the Biocrates metabolite concentration measurement data was performed by each participating cohort as follows<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789149">27</a></sup>: for each cohort, metabolite profile measurements for all individuals were performed on multiple plates. For each metabolite <em>i</em> and plate <em>j</em>, the coefficient of variation (CV<em><sub>i,j</sub></em>) was calculated as: CV<em><sub>i,j</sub></em> = s.d.<em><sub>i,j</sub></em> / mean<em><sub>i,j</sub></em>, where the standard deviation (s.d.) and mean were calculated over all reference measurements per plate <em>j</em> (five per plate). Summary statistics for the metabolite concentration data for each cohort were compared with the measurement detection limit specifications as reported by the manufacturer of the AbsoluteIDQ p150 kit (BIOCRATES). A metabolite was excluded from further analyses for a particular cohort if its concentration measurement data did not meet all of the following criteria: (i) mean CV<em><sub>i</sub></em> over all plates &lt; 25%; (ii) ≤ 5% missing values; (iii) median ≥ lower limit of quantification (for metabolites reported as absolute concentrations) or ≥ limit of detection (for semiquantitatively measured metabolites). Outlying metabolite concentration values (data points) and outlying samples were also removed, and the missing data points were imputed with the ‘R’<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> package ‘mice’<sup><a href="#R27" rid="R27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789106">27</a></sup>. The resulting concentration data for each metabolite were natural log-transformed in order to attain a normal distribution. Throughout the article, names of lipids detected by the Biocrates AbsoluteIDQ p150 platform are abbreviated as follows: acylcarnitines, C<em>x</em>:<em>y</em>; hydroxylacylcarnitines, C(OH)<em>x</em>:<em>y</em>; dicarboxylacylcarnitines, C<em>x</em>:<em>y</em>-DC; sphingomyelins, SM<em>x</em>:<em>y</em>; <em>N</em>-hydroxylacyloylsphingosylphosphocholine, SM (OH)<em>x</em>:<em>y</em>; phosphatidylcholines, PC (aa = diacyl, ae = acyl-alkyl). Lipid side chain composition is abbreviated as C<em>x</em>:<em>y</em>, where <em>x</em> denotes the number of carbons in the side chain and <em>y</em> the number of double bonds.</p></div><div id="S10" class="sec"><h3 id="S10title">Association analyses and meta-analyses</h3><p id="P19" class="p p-first-last">Genome-wide SNP genotyping was performed in each cohort with standard genotyping technologies (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 2</a> and <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Methods</a>). For the samples contributing to the stage 1 (discovery) meta-analysis, imputation was conducted with reference to HapMap phase 2 build 36 release 22 or 24 CEU (Utah residents of Northern and Western European ancestry)<sup><a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789099">29</a></sup> phased genotypes. For the KORA S4 replication sample, SNP genotypes were imputed against the 1,000 g phase1 integrated haplotypes reference set. Association analysis was performed assuming a linear regression model for each SNP, adjusting for relatedness, age, sex and study specific (for example, ancestry-informative principal component scores) covariates as necessary (see <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Table 2</a>). Positions of all SNPs described in the current manuscript were mapped to those as in the HapMap 2 Build 36 release 24 reference set (hg18). Meta-analysis of GWA results obtained in the cohorts participating in stage 1 was performed as follows: the expected minor allele count (eMAC) was computed at the cohort level for each SNP as
</p><div class="disp-formula" id="FD1"><div class="f">eMAC =&nbsp;<em>N</em> <sup>∗</sup> MAF <sup>∗</sup> 2 <sup>∗</sup> <em>I</em><sub><em>A</em></sub></div><div class="l">(1)</div></div><p>
where <em>N</em> is the study sample size, MAF is the minor allele frequency and <em>I</em><em><sub>A</sub></em> is the SNP genotype imputation quality measure. SNPs for which eMAC &lt; 25 were filtered out of the GWA results for the cohort under consideration. After applying genomic control at the individual cohort level, two independent analysts carried out additive model fixed-effects meta-analysis of association data for imputed autosomal SNPs, using two different software packages (METAL<sup><a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789150">30</a></sup> and GWAMA<sup><a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789144">31</a></sup>). For a given SNP, cohort-specific effect size estimates were weighted inversely with their variance. Throughout the manuscript, we report the <em>P</em> values as resulting from <em>Z</em>-tests of association as carried out by METAL. A <em>P</em> value equal to 5.0 × 10<sup>−8</sup> was adopted as the threshold for genome-wide suggestive association between a SNP and the concentration of a metabolite, based on the approximate number of independent SNPs in samples of European ancestry<sup><a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789088">32</a></sup>. To obtain a threshold for significant association, taking account of the number of metabolites tested and their intercorrelations, the threshold value for suggestive association was divided by the number of independent tests (Meffli) in the metabolomic data as estimated using the method of Li and Ji<sup><a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789145">33</a></sup>. The value of Meffli was estimated on the basis of the metabolite profiling data in two independent cohorts (ERF and NTR). Meffli was estimated to be equal to 46 in both ERF and NTR, rendering the <em>P</em> value threshold for genome- and metabolome-wide significance in the present study to be equal to 5.0 × 10<sup>−8</sup>/46 = 1.09 × 10<sup>−9</sup>.</p></div><div id="S11" class="sec"><h3 id="S11title">Definition of loci and secondary signals analysis</h3><p id="P20" class="p p-first-last">Independent signals of association were identified in the GWA meta-analysis results for each metabolite separately, using the linkage disequilibrium-based ‘clumping’ procedure as implemented in PLINK<sup><a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789101">34</a></sup>. This procedure takes all SNPs that show a <em>P</em> value of association with a phenotype below a threshold (‘--clump-p1’), and forms clumps of these ‘index’ SNPs together with all other SNPs that are in linkage disequilibrium with (controlled by the parameter ‘--clump-r2’) and in physical proximity (controlled by parameter ‘--clump-kb’) to these index SNPs. For the current study, we used the following parameter settings: ‘--clump-p1’, 5.0 × 10<sup>−8</sup>; ‘--clump-r2’, 0.1; ‘--clump-kb’, 1,000. As input for the ‘clumping’ procedure, we used association <em>P</em> values from the discovery phase meta-analysis results and linkage disequilibrium patterns as estimated from the HapMap 2 Build 36 release 24 reference set. For each metabolite separately, secondary association signals at a locus were verified by approximate conditional analysis as implemented in GCTA<sup><a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789142">35</a></sup>. In this analysis, the association for the secondary association signal ‘top’ SNP (that is, the SNP with the lowest <em>P</em> value of association with variation in serum metabolite level at the secondary association signal) was conditioned on the top SNP for the locus and metabolite under consideration. As input for the approximate conditional analysis, we used the discovery phase GWA meta-analysis results and the imputed SNP genotype data from the NTR cohort as a reference for LD structure<sup><a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789100">36</a></sup>. We report only secondary signal top SNPs for which the <em>P</em> value remained &lt; 1.09 × 10<sup>−9</sup> in this approximate conditional analysis. In the current manuscript, the term ‘lead metabolomic SNP’ refers to a top SNP at a locus or secondary association signal for one or more metabolites. We identified genomic loci significantly associating with metabolite levels by grouping lead metabolomic SNPs located within 1 Mb from each other over all metabolites.</p></div><div id="S12" class="sec"><h3 id="S12title">Identification of new SNP–metabolite associations</h3><p id="P21" class="p p-first-last">The approach used in the current study to identify novel associations between SNPs and serum metabolite levels is described in <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Fig. 6</a> and in the <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Methods</a>. In brief, we applied two complementary methods: the first method identified novel SNPs associated with variation in serum metabolite levels, and the second method identified novel SNP-metabolite associations with respect to 11 previous GWA studies that included at least one metabolite that was also included in the current study.</p></div><div id="S13" class="sec"><h3 id="S13title">Replication analyses</h3><p id="P22" class="p p-first-last">Replication analyses were performed in the KORA S4 replication sample for the associations of the five new SNPs for serum metabolite levels (listed in <a href="/pmc/articles/PMC4745136/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>) with their lead metabolites (that is, the metabolites that were most strongly associated with these SNPs in the discovery phase meta-analysis).</p></div><div id="S14" class="sec"><h3 id="S14title">Lookup of association with cardiovascular disease</h3><p id="P23" class="p p-first-last">Data on coronary artery disease/myocardial infarction have been contributed by CARDIoGRAMplusC4D investigators and have been downloaded from <a href="http://www.CARDIOGRAMPLUSC4D.ORG" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.CARDIOGRAMPLUSC4D.ORG</a>. We performed a lookup of the associations with SNP rs2246012 in the results from all three meta-analyses as provided on this website<sup><a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789137">22</a>,<a href="#R37" rid="R37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789094">37</a>,<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789129">38</a></sup>.</p></div><div id="S15" class="sec"><h3 id="S15title">Association and Manhattan plots</h3><p id="P24" class="p p-first-last">For each lead metabolomic SNP, the LocusZoom<sup><a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789093">39</a></sup> tool was used to generate association plots in the region between 500 kb before the locus minimum position and 500 kb after the locus maximum position. Manhattan plots for each metabolite were generated based on the discovery phase meta-analysis results using in-house developed Python<sup><a href="#R40" rid="R40" class=" bibr popnode">40</a></sup> code.</p></div><div id="S16" class="sec"><h3 id="S16title">SNP annotation</h3><p id="P25" class="p p-first-last">To facilitate the manual process of selecting plausible candidate genes for each locus, we used an automated workflow developed in-house to generate reports containing the associated protein, enzyme, metabolic reaction, pathway and disease phenotypes of each gene within a ±500 kb window of each lead metabolomic SNP. SNPs within this window that were published in GWAS Catalog<sup><a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789104">41</a></sup> or in GTEx-eQTL (<a href="http://www.ncbi.nlm.nih.gov/gtex/GTEX2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gtex/GTEX2</a>) were also listed. In detail, the reports created by our workflow were based on the NCBI-Gene (<a href="http://www.ncbi.nlm.nih.gov/gene" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/gene</a>), GTEx-eQTL, GWAS Catalog, ConsensusPathDB<sup><a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789124">42</a></sup>, UniProtKB<sup><a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789105">43</a></sup>, OMIM<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup>, Gene Ontology<sup><a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789122">45</a></sup>, TCDB<sup><a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789136">46</a></sup>, ExPASy<sup><a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789097">47</a></sup> and KEGG databases<sup><a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789123">48</a></sup>. These databases had been downloaded earlier from the respective File Transfer Protocol servers and have been integrated offline in MATLAB (R2009a, The Mathworks Inc.). Overlap of lead metabolomic SNPs with <em>cis</em>-eQTL SNPs was also used as evidence to support the annotation of likely causal genes to loci. In case no biologically plausible gene could be found, the locus was given the name of the nearest gene; a similar approach was followed in the study by Shin <em>et al.</em><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789092">11</a></sup></p></div><div id="S17" class="sec"><h3 id="S17title">Variance explained</h3><p id="P26" class="p p-first-last">We estimated the proportion of phenotypic variance explained by each independent association signal (lead SNP for a locus or secondary association signal) as Pearson’s phi coefficient squared:
</p><div class="disp-formula" id="FD2"><div class="f"><em>ϕ</em><sup>2</sup> =&nbsp;<em>χ</em><sup>2</sup><sub>1</sub> ∕ <em>N</em></div><div class="l">(2)</div></div><p>
where <em>χ</em><sup>2</sup><sub>1</sub> = <em>z</em><sup>2</sup> = <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M5" overflow="scroll"><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo>∕</mo><mi>SE</mi><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M5" overflow="scroll"><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo>∕</mo><mi>SE</mi><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo stretchy="false">)</mo><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></mrow></math></script>; <em>N</em> is the sample size in the discovery phase meta-analysis for the SNP–metabolite association under consideration; <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M7" overflow="scroll"><msub><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover><mn>1</mn></msub></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M7" overflow="scroll"><msub><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover><mn>1</mn></msub></math></script> is the ordinary least-squares estimate of <em>β</em><sub>1</sub> (that is, the regression coefficient for the SNP as estimated in the discovery phase meta-analysis); and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M8" overflow="scroll"><mrow><mi>SE</mi><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo stretchy="false">)</mo></mrow></math></span><script type="math/mml"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="inline" id="M8" overflow="scroll"><mrow><mi>SE</mi><mo stretchy="false">(</mo><msub><mrow><mover accent="true"><mi>β</mi><mo>ˆ</mo></mover></mrow><mn>1</mn></msub><mo stretchy="false">)</mo></mrow></math></script> is its standard error. For each metabolite, we added up the proportions of variance in metabolite level explained by independent association signals to estimate the total proportion of phenotypic variance explained. We also estimated for each metabolite the proportion of heritability of metabolite level variability explained by approximately independent association signals. As an estimate of heritability for this analysis we used the monozygotic twin correlations for each metabolite, based on data from 181 pairs from the NTR cohort<sup><a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789114">13</a></sup>. The proportion of heritability in metabolite level variation explained by independent association signals was estimated by dividing the proportion of phenotypic variance explained by independent association signals, by the monozygotic twin correlation. The total proportion of heritability explained for a particular metabolite was estimated by adding up the proportions of variance explained by all approximately independent association signals for that metabolite.</p></div><div id="S18" class="sec"><h3 id="S18title">PGS analysis</h3><p id="P27" class="p p-first-last">We investigated evidence for the polygenic nature of variation in serum metabolite levels by building a multi-SNP predictor from the meta-analysis results for each metabolite to predict the levels of the same metabolite in an independent target cohort (KORA S4 replication sample). Such a multi-SNP predictor, or PGS, reflects the weighted sum of multiple SNPs associated with a phenotype. The discovery meta-analysis forms the basis to select SNPs based on liberal significance thresholds (for example, 0.001, 0.01 and so on). In the target sample, PGSs are calculated for each individual for each set of SNPs by multiplying the number of effect alleles per SNP (0, 1 or 2) with the beta from the meta-analysis, summed over all SNPs in the set of SNPs. We performed the PGS analysis<sup><a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789087">49</a></sup> using the regression coefficients (betas) from the discovery phase meta-analyses as weights. Analyses were performed for the 127 metabolites for which concentration data were available both in the discovery sample and in the KORA S4 target sample. For each of these metabolites, SNPs representing approximately independent association signals were selected in the discovery phase meta-analysis results using the PLINK clumping procedure. SNPs with <em>P</em> values of association with metabolite concentration levels in the discovery meta-analysis below the following thresholds were included: <em>P</em> &lt; 1.0 × 10<sup>−8</sup>; <em>P</em> &lt; 1.0 × 10<sup>−7</sup>; <em>P</em> &lt; 1.0 × 10<sup>−6</sup>; <em>P</em> &lt; 1.0 × 10<sup>−5</sup>; <em>P</em> &lt; 1.0 × 10<sup>−4</sup>; <em>P</em> &lt; 1.0 × 10<sup>−3</sup>; <em>P</em> &lt; 1.0 × 10<sup>−2</sup>; <em>P</em> &lt; 5.0 × 10<sup>−2</sup>; <em>P</em> &lt; 0.1; <em>P</em> &lt; 0.2; <em>P</em> &lt; 0.3; <em>P</em> &lt; 0.4; <em>P</em> &lt; 0.5; <em>P</em> &lt; 0.6; <em>P</em> &lt; 0.7; <em>P</em> &lt; 0.8; <em>P</em> &lt; 0.9; <em>P</em> &lt; 1.0. For each clump of SNPs, the index SNP was taken for possible inclusion in the score computation. From the resulting set of SNPs eligible for inclusion in the PGS analysis, A/T and G/C SNPs for which (0.35 &lt; MAF &lt; 0.50) were excluded because these SNPs are potentially ambiguous and therefore may lead to spurious association in the case of strand flips<sup><a href="#R50" rid="R50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789120">50</a></sup>. From the imputed SNP genotype data for the KORA S4 target sample, the SNPs corresponding with the remaining clump index SNPs were selected. A PGS was constructed for each individual in the KORA S4 replication sample using the ‘--score’ procedure as implemented in PLINK v. 1.07 (<a href="http://pngu.mgh.harvard.edu/Bpurcell/plink/profile.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://pngu.mgh.harvard.edu/Bpurcell/plink/profile.shtml</a>). The resulting PGS was included as a covariate in a multiple linear regression analysis that was similar to the regression that was carried out in the primary single SNP-based GWA analysis:
</p><div class="disp-formula" id="FD3"><div class="f"><strong>y</strong> =&nbsp;<em>α</em> +&nbsp;<em>β</em><sub>1</sub>PGS +&nbsp;<em>β</em><sub>2</sub>age +&nbsp;<em>β</em><sub>3</sub>sex +&nbsp;<em>β</em><sub>4</sub>(study&nbsp;−&nbsp;specific convariates),&nbsp;</div><div class="l">(3)</div></div><p>
where <strong>y</strong> represents log(metabolite) values, and the study-specific covariates include adjustments for, for example, population stratification (and thus <em>β</em><sub>4</sub> can be a vector). The Biocrates metabolite values were obtained and preprocessed using the same methods as described for the primary GWA in the Section ‘Biocrates metabolite quantification’. The proportion of variance explained by the PGSs was assessed by comparing the raw (that is, unadjusted) <em>R</em><sup>2</sup> values for the ‘full’ model (that is, a model including the genetic score as a covariate) with the raw <em>R</em><sup>2</sup> values when fitting a ‘reduced’ model that did not include the genetic score as a covariate<sup><a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789121">51</a></sup>. The significance of the association of the PGSs with serum metabolite levels was estimated on the basis of the <em>P</em> value of association for <em>β</em><sub>1</sub> according to the full model.</p></div><div id="S19" class="sec"><h3 id="S19title">EQTL analyses</h3><p id="P28" class="p p-first-last">Data from two independent samples originating from the United Kingdom, the Netherlands and Estonia (the Dutch NTR-NESDA sample and the Fehrmann-EGCUT sample) were used for <em>cis</em>- and <em>trans</em>-eQTL mapping in whole blood. Details of these analyses, and of the integration of the <em>cis</em>-eQTL analysis results with the results from the metabolomics genome-wide meta-analysis, are provided in the <a href="#SD8" rid="SD8" class=" supplementary-material">Supplementary Methods</a>. We also assessed the overlap of lead metabolomic SNPs with <em>cis</em>-eQTL signals in the liver as catalogued in the GTEx-eQTL database, following the method of Shin <em>et al.</em><sup><a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_511789096">11</a></sup>: for each lead metabolomic SNP, we retrieved all SNPs with <em>r</em><sup>2</sup> &gt; 0.8 in the 1000 Genomes Project pilot phase (CEU population). All <em>cis</em>-eQTLs within a 1-Mb window centred on the lead SNP were retrieved from the GTEx-eQTL database, and the best eQTL <em>P</em> value was noted. The <em>cis</em>-eQTL results for which overlap with lead metabolomic SNPs was shown and that displayed association <em>P</em> values &lt; 0.001 are given in <a href="#SD7" rid="SD7" class=" supplementary-material">Supplementary Data 7</a>.</p></div><div id="S20" class="sec sec-last"><h3 id="S20title">Meta-analysis</h3><p id="P29" class="p p-first-last">The full meta-analysis results for all metabolites are available at <a href="http://www.tweelingenregister.org/engagebiocratesgwama" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.tweelingenregister.org/engagebiocratesgwama</a></p></div></div><div id="SM" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="SMtitle">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>SupplementaryData1</h4><div class="sup-box half_rhythm" id="d37e1816"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData1.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(16K, xlsx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>SupplementaryData2</h4><div class="sup-box half_rhythm" id="d37e1820"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(25K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>SupplementaryData3</h4><div class="sup-box half_rhythm" id="d37e1824"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(13K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>SupplementaryData4</h4><div class="sup-box half_rhythm" id="d37e1828"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(9.4K, xlsx)</sup></div></div><div class="sec suppmat" id="SD5"><h4>SupplementaryData5</h4><div class="sup-box half_rhythm" id="d37e1832"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(13K, xlsx)</sup></div></div><div class="sec suppmat" id="SD6"><h4>SupplementaryData6</h4><div class="sup-box half_rhythm" id="d37e1836"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(7.2K, xlsx)</sup></div></div><div class="sec suppmat" id="SD7"><h4>SupplementaryData7</h4><div class="sup-box half_rhythm" id="d37e1840"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryData7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(12K, xlsx)</sup></div></div><div class="sec suppmat" id="SD8"><h4>SupplementaryFiguresTablesAndNotes</h4><div class="sup-box half_rhythm" id="d37e1844"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryFiguresTablesAndNotes.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(19M, pdf)</sup></div></div><div class="sec suppmat" id="SD9"><h4>SupplementaryMovie</h4><div class="sup-box half_rhythm" id="d37e1848"><a href="/pmc/articles/PMC4745136/bin/NIHMS66678-supplement-SupplementaryMovie.mp4" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(11M, mp4)</sup></div></div></div><div id="S21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S21title">Acknowledgements</h2><div class="sec"><p id="__p1">We thank all participants. This research was supported by the European Community’s Seventh Framework Programme (FP7/2007–2013), ENGAGE Consortium, grant agreement HEALTH-F4-2007-201413. The Cooperative Health Research in the Region Augsburg (KORA) study was supported by the Helmholtz Zentrum München—German Research Center for Environmental Health, which is funded by the German Federal Ministry of Education and Research (BMBF) and by the State of Bavaria. KORA research was supported within the Munich Center of Health Sciences (MC-Health), Ludwig-Maximilians-Universität, as part of LMUinnovativ. This work was financed by the German National Genome Research Network (NGFN; NGFNPlus, project number 01GS0834). K. Suhre is supported by Biomedical Research Program funds at the Weill Cornell Medical College in Qatar, funded by the Qatar Foundation. The Estonian Genome Center of University of Tartu (EGCUT) received support from the Estonian Government IUT20–60, IUT24-6, Estonian Research Roadmap through the Estonian Ministry of Education and Research (3.2.0304.11-0312), Center of Excellence in Genomics (EXCEGEN), Development Fund from the University of Tartu (SP1GVARENG), an EFSD New Horizons grant and the US National Institute of Health (R01DK075787). We acknowledge EGCUT personnel, especially Mr V. Soo. Data analyses were carried out in part in the High Performance Computing Center of the University of Tartu. TwinsUK receives support from the National Institute for Health Research (NIHR) Clinical Research Facility at Guy’s &amp; St Thomas’ NHS Foundation Trust and NIHR Biomedical Research Centre based at Guy’s and St Thomas’ NHS Foundation Trust and King’s College London. The Netherlands Twin Register (NTR) was funded by the Netherlands Organization for Scientific Research (NWO: MagW/ZonMW grants 904-61-090; 985-10-002; 904-61-193; 480-04-004; 400-05-717; Middelgroot-911-09-032; 56-464-14192); Center for Medical Systems Biology (CMSB); Netherlands Bioinformatics Centre/BioAssist/RK(2008.024); Biobanking and Biomolecular Resources Research Infrastructure (BBMRI)–NL (184.021.007) and the European Science Council (ERC-230374). H.H.M.D. is supported by an EMGO+ Fellowship. This work was carried out on the Dutch national e-infrastructure with the support from the SURF Foundation. The Erasmus Rucphen Family study (ERF) is grateful to Petra Veraart for her help in genealogy, Jeannette Vergeer for supervision of the laboratory work and Peter Snijders for help in data collection. ERF was supported by grants from NWO, Erasmus MC and CMSB. For the Queensland Institute of Medical Research (QIMR) cohort, serum metabolite level assessment was co-funded by the European Community’s Seventh Framework Programme (FP7/2007–2013), ENGAGE project, grant agreement HEALTH-F4-2007-201413 and NHMRC-European Union Collaborative Research Grant 496739.</p></div></div><div id="APP1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="APP1title">Author Contributions</h2><!--/article/back/app-group/app/--><p id="P3">Jointly supervised the research: A.I., J.T.B., M.M., B.W.P., K.W.v.D., P.A.C.H., K. Strauch, G.J.B.v.O., T.I., J.T.B., M.M., M.I.M., C.G., A.I. and D.I.B. Contributed reagents/materials/analysis tools: N.A., A.J.M.C., P.E.S., H.-J.W., L.F., J.J.H., G.W. and D.R.N. Conceived and designed the experiments: C.M.v.D., C.G. and D.I.B. Performed statistical analysis: E.M.v.L., S.B., H.H.M.D., R.P., M.K., R.J., A.-K.P. and J.B.v.K. Performing metabolomics measurements (sample preparation, mass spectrometric measurements, quality assurance): C.P. and J.A. Measurements and data quality control: W.R.-M. Wrote the paper: A.D., C.M.v.D., H.H.M.D., R.P., R.J., A.-K.P., J.B.v.K., E.J.C.G., D.R.N., K.W.v.D., P.A.C.H., K.Suhre, C.M.v.D. and D.I.B. Principal investigator: T.D.S., A.M., P.E.S., K.W.v.D., P.A.C.H., K.Strauch, N.G.M., G.J.B.v.O., T.I., C.M.v.D. and D.I.B. Analysed data: I.Y., A.A.M.V., M.B., H.H.M.D., R.P., M.K., R.J., A.-K.P., T.H., G.Z., A.D., T.E., J.B.v.K., H.D. and K.Suhre. Genotyping: T.E. Directed wet lab: G.W.M. Directed biochemistry: J.B.W. Managed samples: A.K.H.</p></div><div id="idm140236513924784" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140236513924784title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN3"></p><p id="P30" class="p p-first-last"><strong>Accession codes.</strong> The metabolomic data described in this manuscript are uploaded to MetaboLights and are available under the accession code MTBLS192.</p><p></p><p class="fn sec" id="FN4"></p><p id="P31" class="p p-first-last"><strong>Competing financial interests:</strong> The authors declare no competing financial interests.</p><p></p></div></div><div id="idm140236515320080" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140236515320080title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Suhre K, Gieger C. Genetic variation in metabolic phenotypes: study designs and applications. <span><span class="ref-journal">Nat. Rev. Genet. </span>2012;<span class="ref-vol">13</span>:759–769.</span> [<a href="/pubmed/23032255" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet&amp;title=Genetic+variation+in+metabolic+phenotypes:+study+designs+and+applications&amp;author=K+Suhre&amp;author=C+Gieger&amp;volume=13&amp;publication_year=2012&amp;pages=759-769&amp;pmid=23032255&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Suhre K, et al.  Human metabolic individuality in biomedical and pharmaceutical research. <span><span class="ref-journal">Nature. </span>2011;<span class="ref-vol">477</span>:54–60.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3832838/">PMC free article</a>]</span> [<a href="/pubmed/21886157" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Human+metabolic+individuality+in+biomedical+and+pharmaceutical+research&amp;author=K+Suhre&amp;volume=477&amp;publication_year=2011&amp;pages=54-60&amp;pmid=21886157&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Nicholson G, et al.  A genome-wide metabolic QTL analysis in Europeans implicates two loci shaped by recent positive selection. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002270.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3169529/">PMC free article</a>]</span> [<a href="/pubmed/21931564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+metabolic+QTL+analysis+in+Europeans+implicates+two+loci+shaped+by+recent+positive+selection&amp;author=G+Nicholson&amp;volume=7&amp;publication_year=2011&amp;pages=e1002270&amp;pmid=21931564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Illig T, et al.  A genome-wide perspective of genetic variation in human metabolism. <span><span class="ref-journal">Nat. Genet. </span>2010;<span class="ref-vol">42</span>:137–141.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3773904/">PMC free article</a>]</span> [<a href="/pubmed/20037589" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=A+genome-wide+perspective+of+genetic+variation+in+human+metabolism&amp;author=T+Illig&amp;volume=42&amp;publication_year=2010&amp;pages=137-141&amp;pmid=20037589&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Gieger C, et al.  Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum. <span><span class="ref-journal">PLoS Genet. </span>2008;<span class="ref-vol">4</span>:e1000282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2581785/">PMC free article</a>]</span> [<a href="/pubmed/19043545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetics+meets+metabolomics:+a+genome-wide+association+study+of+metabolite+profiles+in+human+serum&amp;author=C+Gieger&amp;volume=4&amp;publication_year=2008&amp;pages=e1000282&amp;pmid=19043545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Demirkan A, et al.  Genome-wide association study identifies novel loci associated with circulating phospho- and sphingolipid concentrations. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002490.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3280968/">PMC free article</a>]</span> [<a href="/pubmed/22359512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+identifies+novel+loci+associated+with+circulating+phospho-+and+sphingolipid+concentrations&amp;author=A+Demirkan&amp;volume=8&amp;publication_year=2012&amp;pages=e1002490&amp;pmid=22359512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Hicks AA, et al.  Genetic determinants of circulating sphingolipid concentrations in European populations. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000672.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2745562/">PMC free article</a>]</span> [<a href="/pubmed/19798445" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+determinants+of+circulating+sphingolipid+concentrations+in+European+populations&amp;author=AA+Hicks&amp;volume=5&amp;publication_year=2009&amp;pages=e1000672&amp;pmid=19798445&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Tukiainen T, et al.  Detailed metabolic and genetic characterization reveals new associations for 30 known lipid loci. <span><span class="ref-journal">Hum. Mol. Genet. </span>2012;<span class="ref-vol">21</span>:1444–1455.</span> [<a href="/pubmed/22156771" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Detailed+metabolic+and+genetic+characterization+reveals+new+associations+for+30+known+lipid+loci&amp;author=T+Tukiainen&amp;volume=21&amp;publication_year=2012&amp;pages=1444-1455&amp;pmid=22156771&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Kettunen J, et al.  Genome-wide association study identifies multiple loci influencing human serum metabolite levels. <span><span class="ref-journal">Nat. Genet. </span>2012;<span class="ref-vol">44</span>:269–276.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3605033/">PMC free article</a>]</span> [<a href="/pubmed/22286219" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+association+study+identifies+multiple+loci+influencing+human+serum+metabolite+levels&amp;author=J+Kettunen&amp;volume=44&amp;publication_year=2012&amp;pages=269-276&amp;pmid=22286219&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Rhee EP, et al.  A genome-wide association study of the human metabolome in a community-based cohort. <span><span class="ref-journal">Cell Metab. </span>2013;<span class="ref-vol">18</span>:130–143.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3973158/">PMC free article</a>]</span> [<a href="/pubmed/23823483" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metab&amp;title=A+genome-wide+association+study+of+the+human+metabolome+in+a+community-based+cohort&amp;author=EP+Rhee&amp;volume=18&amp;publication_year=2013&amp;pages=130-143&amp;pmid=23823483&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Shin SY, et al.  An atlas of genetic influences on human blood metabolites. <span><span class="ref-journal">Nat. Genet. </span>2014;<span class="ref-vol">46</span>:543–550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4064254/">PMC free article</a>]</span> [<a href="/pubmed/24816252" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=An+atlas+of+genetic+influences+on+human+blood+metabolites&amp;author=SY+Shin&amp;volume=46&amp;publication_year=2014&amp;pages=543-550&amp;pmid=24816252&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Yu B, et al.  Genetic determinants influencing human serum metabolome among African Americans. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004212.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3952826/">PMC free article</a>]</span> [<a href="/pubmed/24625756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genetic+determinants+influencing+human+serum+metabolome+among+African+Americans&amp;author=B+Yu&amp;volume=10&amp;publication_year=2014&amp;pages=e1004212&amp;pmid=24625756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Draisma HH, et al.  Familial resemblance for serum metabolite concentrations. <span><span class="ref-journal">Twin Res. Hum. Genet. </span>2013;<span class="ref-vol">16</span>:948–961.</span> [<a href="/pubmed/23985338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Twin+Res.+Hum.+Genet&amp;title=Familial+resemblance+for+serum+metabolite+concentrations&amp;author=HH+Draisma&amp;volume=16&amp;publication_year=2013&amp;pages=948-961&amp;pmid=23985338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Itzkovitz B, et al.  Functional characterization of novel mutations in GNPAT and AGPS, causing rhizomelic chondrodysplasia punctata (RCDP) types 2 and 3. <span><span class="ref-journal">Hum. Mutat. </span>2012;<span class="ref-vol">33</span>:189–197.</span> [<a href="/pubmed/21990100" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mutat&amp;title=Functional+characterization+of+novel+mutations+in+GNPAT+and+AGPS,+causing+rhizomelic+chondrodysplasia+punctata+(RCDP)+types+2+and+3&amp;author=B+Itzkovitz&amp;volume=33&amp;publication_year=2012&amp;pages=189-197&amp;pmid=21990100&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Gertow K, et al.  Fatty acid handling protein expression in adipose tissue, fatty acid composition of adipose tissue and serum, and markers of insulin resistance. <span><span class="ref-journal">Eur. J. Clin. Nutr. </span>2006;<span class="ref-vol">60</span>:1406–1413.</span> [<a href="/pubmed/16788709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Clin.+Nutr&amp;title=Fatty+acid+handling+protein+expression+in+adipose+tissue,+fatty+acid+composition+of+adipose+tissue+and+serum,+and+markers+of+insulin+resistance&amp;author=K+Gertow&amp;volume=60&amp;publication_year=2006&amp;pages=1406-1413&amp;pmid=16788709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Phillips CM, et al.  Gene-nutrient interactions with dietary fat modulate the association between genetic variation of the ACSL1 gene and metabolic syndrome. <span><span class="ref-journal">J. Lipid. Res. </span>2010;<span class="ref-vol">51</span>:1793–1800.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2882737/">PMC free article</a>]</span> [<a href="/pubmed/20176858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid.+Res&amp;title=Gene-nutrient+interactions+with+dietary+fat+modulate+the+association+between+genetic+variation+of+the+ACSL1+gene+and+metabolic+syndrome&amp;author=CM+Phillips&amp;volume=51&amp;publication_year=2010&amp;pages=1793-1800&amp;pmid=20176858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Aouida M, Poulin R, Ramotar D. The human carnitine transporter SLC22A16 mediates high affinity uptake of the anticancer polyamine analogue bleomycin-A5. <span><span class="ref-journal">J. Biol. Chem. </span>2010;<span class="ref-vol">285</span>:6275–6284.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2825423/">PMC free article</a>]</span> [<a href="/pubmed/20037140" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem&amp;title=The+human+carnitine+transporter+SLC22A16+mediates+high+affinity+uptake+of+the+anticancer+polyamine+analogue+bleomycin-A5&amp;author=M+Aouida&amp;author=R+Poulin&amp;author=D+Ramotar&amp;volume=285&amp;publication_year=2010&amp;pages=6275-6284&amp;pmid=20037140&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Bray J, et al.  Influence of pharmacogenetics on response and toxicity in breast cancer patients treated with doxorubicin and cyclophosphamide. <span><span class="ref-journal">Br. J. Cancer. </span>2010;<span class="ref-vol">102</span>:1003–1009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2844036/">PMC free article</a>]</span> [<a href="/pubmed/20179710" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=Influence+of+pharmacogenetics+on+response+and+toxicity+in+breast+cancer+patients+treated+with+doxorubicin+and+cyclophosphamide&amp;author=J+Bray&amp;volume=102&amp;publication_year=2010&amp;pages=1003-1009&amp;pmid=20179710&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Tang WH, Wang Z, Cho L, Brennan DM, Hazen SL. Diminished global arginine bioavailability and increased arginine catabolism as metabolic profile of increased cardiovascular risk. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2009;<span class="ref-vol">53</span>:2061–2067.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2755213/">PMC free article</a>]</span> [<a href="/pubmed/19477356" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol&amp;title=Diminished+global+arginine+bioavailability+and+increased+arginine+catabolism+as+metabolic+profile+of+increased+cardiovascular+risk&amp;author=WH+Tang&amp;author=Z+Wang&amp;author=L+Cho&amp;author=DM+Brennan&amp;author=SL+Hazen&amp;volume=53&amp;publication_year=2009&amp;pages=2061-2067&amp;pmid=19477356&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Tripolt NJ, et al.  Multifactorial risk factor intervention in patients with Type 2 diabetes improves arginine bioavailability ratios. <span><span class="ref-journal">Diabet. Med. </span>2012;<span class="ref-vol">29</span>:e365–e368.</span> [<a href="/pubmed/22803961" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabet.+Med&amp;title=Multifactorial+risk+factor+intervention+in+patients+with+Type+2+diabetes+improves+arginine+bioavailability+ratios&amp;author=NJ+Tripolt&amp;volume=29&amp;publication_year=2012&amp;pages=e365-e368&amp;pmid=22803961&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Brion M-JA, Benyamin B, Visscher PM, Davey Smith G. Beyond the single SNP: emerging developments in Mendelian randomization in the ‘omics’ era. <span><span class="ref-journal">Curr. Epidemiol. Rep. </span>2014;<span class="ref-vol">1</span>:228–236.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Epidemiol.+Rep&amp;title=Beyond+the+single+SNP:+emerging+developments+in+Mendelian+randomization+in+the+‘omics’+era&amp;author=M-JA+Brion&amp;author=B+Benyamin&amp;author=PM+Visscher&amp;author=G+Davey+Smith&amp;volume=1&amp;publication_year=2014&amp;pages=228-236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Schunkert H, et al.  Large-scale association analysis identifies 13 new susceptibility loci for coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2011;<span class="ref-vol">43</span>:333–338.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3119261/">PMC free article</a>]</span> [<a href="/pubmed/21378990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Large-scale+association+analysis+identifies+13+new+susceptibility+loci+for+coronary+artery+disease&amp;author=H+Schunkert&amp;volume=43&amp;publication_year=2011&amp;pages=333-338&amp;pmid=21378990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Dharuri H, et al.  Genetics of the human metabolome, what is next? Biochim. <span><span class="ref-journal">Biophys. Acta. </span>2014;<span class="ref-vol">1842</span>:1923–1931.</span> [<a href="/pubmed/24905732" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.+Acta&amp;title=Genetics+of+the+human+metabolome,+what+is+next?+Biochim&amp;author=H+Dharuri&amp;volume=1842&amp;publication_year=2014&amp;pages=1923-1931&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Römisch-Margl W, et al.  Procedure for tissue sample preparation and metabolite extraction for high-throughput targeted metabolomics. <span><span class="ref-journal">Metabolomics. </span>2012;<span class="ref-vol">8</span>:133–142.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolomics&amp;title=Procedure+for+tissue+sample+preparation+and+metabolite+extraction+for+high-throughput+targeted+metabolomics&amp;author=W+Römisch-Margl&amp;volume=8&amp;publication_year=2012&amp;pages=133-142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">US Department of Health and Human Services, F. a. D. A. Center for Drug Evaluation and Research (CDER) Center for Veterinary Medicine (CVM) Guidance for Industry  <span class="ref-journal">Bioanalytical Method Validation.</span> 2001. </span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Mittelstrass K, et al.  Discovery of sexual dimorphisms in metabolic and genetic biomarkers. <span><span class="ref-journal">PLoS Genet. </span>2011;<span class="ref-vol">7</span>:e1002215.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3154959/">PMC free article</a>]</span> [<a href="/pubmed/21852955" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Discovery+of+sexual+dimorphisms+in+metabolic+and+genetic+biomarkers&amp;author=K+Mittelstrass&amp;volume=7&amp;publication_year=2011&amp;pages=e1002215&amp;pmid=21852955&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Goek ON, et al.  Serum metabolite concentrations and decreased GFR in the general population. <span><span class="ref-journal">Am. J. Kidney Dis. </span>2012;<span class="ref-vol">60</span>:197–206.</span> [<a href="/pubmed/22464876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Kidney+Dis&amp;title=Serum+metabolite+concentrations+and+decreased+GFR+in+the+general+population&amp;author=ON+Goek&amp;volume=60&amp;publication_year=2012&amp;pages=197-206&amp;pmid=22464876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation"><span class="ref-journal">R: A language and environment for statistical computing.</span> R Foundation for Statistical Computing; 2012.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+language+and+environment+for+statistical+computing&amp;publication_year=2012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Frazer KA, et al.  A second generation human haplotype map of over 3.1 million SNPs. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">449</span>:851–861.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=KA+Frazer&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2190–2191.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2922887/">PMC free article</a>]</span> [<a href="/pubmed/20616382" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=METAL:+fast+and+efficient+meta-analysis+of+genomewide+association+scans&amp;author=CJ+Willer&amp;author=Y+Li&amp;author=GR+Abecasis&amp;volume=26&amp;publication_year=2010&amp;pages=2190-2191&amp;pmid=20616382&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Ma ¨gi R, Morris AP. GWAMA: software for genome-wide association meta-analysis. <span><span class="ref-journal">BMC Bioinformatics. </span>2010;<span class="ref-vol">11</span>:288.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2893603/">PMC free article</a>]</span> [<a href="/pubmed/20509871" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=GWAMA:+software+for+genome-wide+association+meta-analysis&amp;author=R+Ma+¨gi&amp;author=AP+Morris&amp;volume=11&amp;publication_year=2010&amp;pages=288&amp;pmid=20509871&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. <span><span class="ref-journal">Genet. Epidemiol. </span>2008;<span class="ref-vol">32</span>:381–385.</span> [<a href="/pubmed/18348202" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol&amp;title=Estimation+of+the+multiple+testing+burden+for+genomewide+association+studies+of+nearly+all+common+variants&amp;author=I+Pe’er&amp;author=R+Yelensky&amp;author=D+Altshuler&amp;author=MJ+Daly&amp;volume=32&amp;publication_year=2008&amp;pages=381-385&amp;pmid=18348202&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Li J, Ji L. Adjusting multiple testing in multilocus analyses using the eigenvalues of a correlation matrix. <span><span class="ref-journal">Heredity (Edinb) </span>2005;<span class="ref-vol">95</span>:221–227.</span> [<a href="/pubmed/16077740" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Heredity+(Edinb)&amp;title=Adjusting+multiple+testing+in+multilocus+analyses+using+the+eigenvalues+of+a+correlation+matrix&amp;author=J+Li&amp;author=L+Ji&amp;volume=95&amp;publication_year=2005&amp;pages=221-227&amp;pmid=16077740&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Yang J, Lee SH, Goddard ME, Visscher PM. GCTA: a tool for genome-wide complex trait analysis. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2011;<span class="ref-vol">88</span>:76–82.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3014363/">PMC free article</a>]</span> [<a href="/pubmed/21167468" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet&amp;title=GCTA:+a+tool+for+genome-wide+complex+trait+analysis&amp;author=J+Yang&amp;author=SH+Lee&amp;author=ME+Goddard&amp;author=PM+Visscher&amp;volume=88&amp;publication_year=2011&amp;pages=76-82&amp;pmid=21167468&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Cornelis MC, et al.  Genome-wide meta-analysis identifies six novel loci associated with habitual coffee consumption. <span><span class="ref-journal">Mol. Psychiatry. </span>2015;<span class="ref-vol">20</span>:647–656.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4388784/">PMC free article</a>]</span> [<a href="/pubmed/25288136" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Genome-wide+meta-analysis+identifies+six+novel+loci+associated+with+habitual+coffee+consumption&amp;author=MC+Cornelis&amp;volume=20&amp;publication_year=2015&amp;pages=647-656&amp;pmid=25288136&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Peden JF, Farrall M. Thirty-five common variants for coronary artery disease: the fruits of much collaborative labour. <span><span class="ref-journal">Hum. Mol. Genet. </span>2011;<span class="ref-vol">20</span>:R198–R205.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3179381/">PMC free article</a>]</span> [<a href="/pubmed/21875899" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet&amp;title=Thirty-five+common+variants+for+coronary+artery+disease:+the+fruits+of+much+collaborative+labour&amp;author=JF+Peden&amp;author=M+Farrall&amp;volume=20&amp;publication_year=2011&amp;pages=R198-R205&amp;pmid=21875899&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Deloukas P, et al.  Large-scale association analysis identifies new risk loci for coronary artery disease. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:25–33.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3679547/">PMC free article</a>]</span> [<a href="/pubmed/23202125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Large-scale+association+analysis+identifies+new+risk+loci+for+coronary+artery+disease&amp;author=P+Deloukas&amp;volume=45&amp;publication_year=2013&amp;pages=25-33&amp;pmid=23202125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Pruim RJ, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–2337.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;volume=26&amp;publication_year=2010&amp;pages=2336-2337&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Van Rossum G.  <span class="ref-journal">Python Tutorial.</span> Centrum voor Wiskunde en Informatica (CWI); 1995.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Python+Tutorial&amp;author=G+Van+Rossum&amp;publication_year=1995&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">Hindorff LA, et al.  Potential etiologic and functional implications of genomewide association loci for human diseases and traits. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2009;<span class="ref-vol">106</span>:9362–9367.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2687147/">PMC free article</a>]</span> [<a href="/pubmed/19474294" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Potential+etiologic+and+functional+implications+of+genomewide+association+loci+for+human+diseases+and+traits&amp;author=LA+Hindorff&amp;volume=106&amp;publication_year=2009&amp;pages=9362-9367&amp;pmid=19474294&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation">Kamburov A, et al.  ConsensusPathDB: toward a more complete picture of cell biology. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>:D712–D717.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013724/">PMC free article</a>]</span> [<a href="/pubmed/21071422" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ConsensusPathDB:+toward+a+more+complete+picture+of+cell+biology&amp;author=A+Kamburov&amp;volume=39&amp;publication_year=2011&amp;pages=D712-D717&amp;pmid=21071422&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Magrane M, Uniprot Consortium  UniProt Knowledgebase: a hub of integrated protein data. <span><span class="ref-journal">Database (Oxford) </span>2011;<span class="ref-vol">2011</span>:bar009.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3070428/">PMC free article</a>]</span> [<a href="/pubmed/21447597" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Database+(Oxford)&amp;title=UniProt+Knowledgebase:+a+hub+of+integrated+protein+data&amp;author=M+Magrane&amp;volume=2011&amp;publication_year=2011&amp;pages=bar009&amp;pmid=21447597&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="element-citation">McKusick V.  <span class="ref-journal">Mendelian Inheritance in Man; A Catalog of Human Genes and Genetic Disorders.</span> Johns Hopkins Univ.; 1998.  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Mendelian+Inheritance+in+Man;+A+Catalog+of+Human+Genes+and+Genetic+Disorders&amp;author=V+McKusick&amp;publication_year=1998&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="element-citation">Ashburner M, et al.  Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <span><span class="ref-journal">Nat. Genet. </span>2000;<span class="ref-vol">25</span>:25–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037419/">PMC free article</a>]</span> [<a href="/pubmed/10802651" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Gene+ontology:+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium&amp;author=M+Ashburner&amp;volume=25&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="element-citation">Saier MH, Tran CV, Barabote RD. TCDB: the Transporter Classification Database for membrane transport protein analyses and information. <span><span class="ref-journal">Nucleic Acids Res. </span>2006;<span class="ref-vol">34</span>:D181–D186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1334385/">PMC free article</a>]</span> [<a href="/pubmed/16381841" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=TCDB:+the+Transporter+Classification+Database+for+membrane+transport+protein+analyses+and+information&amp;author=MH+Saier&amp;author=CV+Tran&amp;author=RD+Barabote&amp;volume=34&amp;publication_year=2006&amp;pages=D181-D186&amp;pmid=16381841&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="element-citation">Gasteiger E, et al.  ExPASy: the proteomics server for in-depth protein knowledge and analysis. <span><span class="ref-journal">Nucleic Acids Res. </span>2003;<span class="ref-vol">31</span>:3784–3788.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168970/">PMC free article</a>]</span> [<a href="/pubmed/12824418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=ExPASy:+the+proteomics+server+for+in-depth+protein+knowledge+and+analysis&amp;author=E+Gasteiger&amp;volume=31&amp;publication_year=2003&amp;pages=3784-3788&amp;pmid=12824418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="element-citation">Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:27–30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC102409/">PMC free article</a>]</span> [<a href="/pubmed/10592173" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=KEGG:+kyoto+encyclopedia+of+genes+and+genomes&amp;author=M+Kanehisa&amp;author=S+Goto&amp;volume=28&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="element-citation">Purcell SM, et al.  Common polygenic variation contributes to risk of schizophrenia and bipolar disorder. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">460</span>:748–752.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3912837/">PMC free article</a>]</span> [<a href="/pubmed/19571811" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Common+polygenic+variation+contributes+to+risk+of+schizophrenia+and+bipolar+disorder&amp;author=SM+Purcell&amp;volume=460&amp;publication_year=2009&amp;pages=748-752&amp;pmid=19571811&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="element-citation">Demirkan A, et al.  Genetic risk profiles for depression and anxiety in adult and elderly cohorts. <span><span class="ref-journal">Mol. Psychiatry. </span>2011;<span class="ref-vol">16</span>:773–783.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3142964/">PMC free article</a>]</span> [<a href="/pubmed/20567237" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Psychiatry&amp;title=Genetic+risk+profiles+for+depression+and+anxiety+in+adult+and+elderly+cohorts&amp;author=A+Demirkan&amp;volume=16&amp;publication_year=2011&amp;pages=773-783&amp;pmid=20567237&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="element-citation">Berndt SI, et al.  Genome-wide meta-analysis identifies 11 new loci for anthropometric traits and provides insights into genetic architecture. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:501–512.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3973018/">PMC free article</a>]</span> [<a href="/pubmed/23563607" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet&amp;title=Genome-wide+meta-analysis+identifies+11+new+loci+for+anthropometric+traits+and+provides+insights+into+genetic+architecture&amp;author=SI+Berndt&amp;volume=45&amp;publication_year=2013&amp;pages=501-512&amp;pmid=23563607&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="element-citation">Shannon P, et al.  Cytoscape: a software environment for integrated models of biomolecular interaction networks. <span><span class="ref-journal">Genome Res. </span>2003;<span class="ref-vol">13</span>:2498–2504.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC403769/">PMC free article</a>]</span> [<a href="/pubmed/14597658" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Cytoscape:+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks&amp;author=P+Shannon&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4745136&amp;issue-id=264868&amp;journal-id=358&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140236519484736title">Abstract</a></li><li><a href="#S1title">Results</a></li><li><a href="#S6title">Discussion</a></li><li><a href="#S7title">Methods</a></li><li><a href="#SMtitle">Supplementary Material</a></li><li><a href="#S21title">Acknowledgements</a></li><li><a href="#APP1title">Author Contributions</a></li><li><a href="#idm140236513924784title">Footnotes</a></li><li><a href="#idm140236515320080title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4745136/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4745136/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4745136/pdf/emss-66678.pdf">PDF (1.1M)</a></li> | <li><a href="#" data-citationid="PMC4745136" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4745136%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4745136%2F&amp;text=Genome-wide%20association%20study%20identifies%20novel%20genetic%20variants%20contributing%20to%20variation%20in%20blood%20metabolite%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4745136%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26068415" data-aiid="4745136" data-aid="4745136" data-iid="264868" data-domainid="358" data-domain="wtpa" data-accid="PMC4745136" data-md5="a76b05c82e0f50feac5d6e92ffc39b52"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:46:09-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D03C1DD57CD5100000000086904CE&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4745136%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=wtpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4745136/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>