<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4326325/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326325/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations"><meta name="citation_authors" content="Laura E. Mitchell, A.J. Agopian, Angela Bhalla, Joseph T. Glessner, Cecilia E. Kim, Michael D. Swartz, Hakon Hakonarson, Elizabeth Goldmuntz"><meta name="citation_date" content="1 January 2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="265"><meta name="citation_doi" content="10.1093/hmg/ddu420"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4326325/?report=abstract"><meta name="citation_pmid" content="25138779"><meta name="DC.Title" content="Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Laura E. Mitchell"><meta name="DC.Contributor" content="A.J. Agopian"><meta name="DC.Contributor" content="Angela Bhalla"><meta name="DC.Contributor" content="Joseph T. Glessner"><meta name="DC.Contributor" content="Cecilia E. Kim"><meta name="DC.Contributor" content="Michael D. Swartz"><meta name="DC.Contributor" content="Hakon Hakonarson"><meta name="DC.Contributor" content="Elizabeth Goldmuntz"><meta name="DC.Date" content="2015 Jan 1"><meta name="DC.Identifier" content="10.1093/hmg/ddu420"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations"><meta property="og:type" content="article"><meta property="og:description" content="Congenital left-sided lesions (LSLs) are serious, heritable malformations of the heart. However, little is known about the genetic causes of LSLs. This study was undertaken to identify common variants acting through the genotype of the affected individual ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326325/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4326325/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4326325/pdf/ddu420.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE899F34DD5775F100000000015200C1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4326325%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC4326325</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2015 Jan 1; 24(1): 265–273. </span></div><div><span class="fm-vol-iss-date">Published online 2014 Aug 18. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu420" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddu420</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4326325</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25138779">25138779</a></div></div></div></div><h1 class="content-title">Genome-wide association study of maternal and inherited effects on left-sided cardiac malformations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Mitchell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441336753824" co-class="co-affbox">Laura E. Mitchell</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Agopian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441337753408" co-class="co-affbox">A.J. Agopian</a>,<sup>1</sup> <a href="/pubmed/?term=Bhalla%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441337751424" co-class="co-affbox">Angela Bhalla</a>,<sup>1</sup> <a href="/pubmed/?term=Glessner%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441337749440" co-class="co-affbox">Joseph T. Glessner</a>,<sup>2</sup> <a href="/pubmed/?term=Kim%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441368273376" co-class="co-affbox">Cecilia E. Kim</a>,<sup>2</sup> <a href="/pubmed/?term=Swartz%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441368271392" co-class="co-affbox">Michael D. Swartz</a>,<sup>4</sup> <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441368913072" co-class="co-affbox">Hakon Hakonarson</a>,<sup>2,</sup><sup>5</sup> and  <a href="/pubmed/?term=Goldmuntz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779" class="affpopup" co-rid="_co_idm140441368910352" co-class="co-affbox">Elizabeth Goldmuntz</a><sup>3,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140441336753824"><h3 class="no_margin">Laura E. Mitchell</h3><p><sup>1</sup>Division of Epidemiology, Human Genetics and Environmental Sciences, University of Texas School of Public Health, Houston, TX 77030, USA</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20LE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Laura E. Mitchell</a></div></div><div id="_co_idm140441337753408"><h3 class="no_margin">A.J. Agopian</h3><p><sup>1</sup>Division of Epidemiology, Human Genetics and Environmental Sciences, University of Texas School of Public Health, Houston, TX 77030, USA</p><div>Find articles by <a href="/pubmed/?term=Agopian%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">A.J. Agopian</a></div></div><div id="_co_idm140441337751424"><h3 class="no_margin">Angela Bhalla</h3><p><sup>1</sup>Division of Epidemiology, Human Genetics and Environmental Sciences, University of Texas School of Public Health, Houston, TX 77030, USA</p><div>Find articles by <a href="/pubmed/?term=Bhalla%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Angela Bhalla</a></div></div><div id="_co_idm140441337749440"><h3 class="no_margin">Joseph T. Glessner</h3><p><sup>2</sup>The Center for Applied Genomics,  and</p><div>Find articles by <a href="/pubmed/?term=Glessner%20JT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Joseph T. Glessner</a></div></div><div id="_co_idm140441368273376"><h3 class="no_margin">Cecilia E. Kim</h3><p><sup>2</sup>The Center for Applied Genomics,  and</p><div>Find articles by <a href="/pubmed/?term=Kim%20CE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Cecilia E. Kim</a></div></div><div id="_co_idm140441368271392"><h3 class="no_margin">Michael D. Swartz</h3><p><sup>4</sup>Division of Biostatistics, University of Texas School of Public Health, Houston, TX 77030, USA,  and</p><div>Find articles by <a href="/pubmed/?term=Swartz%20MD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Michael D. Swartz</a></div></div><div id="_co_idm140441368913072"><h3 class="no_margin">Hakon Hakonarson</h3><p><sup>2</sup>The Center for Applied Genomics,  and</p><p><sup>5</sup>Department of Pediatrics, The Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Hakonarson%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Hakon Hakonarson</a></div></div><div id="_co_idm140441368910352"><h3 class="no_margin">Elizabeth Goldmuntz</h3><p><sup>3</sup>Division of Cardiology, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</p><p><sup>5</sup>Department of Pediatrics, The Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</p><div>Find articles by <a href="/pubmed/?term=Goldmuntz%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25138779">Elizabeth Goldmuntz</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140441410585904_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140441410585904_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140441410585904_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140441410585904_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Division of Epidemiology, Human Genetics and Environmental Sciences, University of Texas School of Public Health, Houston, TX 77030, USA</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>The Center for Applied Genomics,  and</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Division of Cardiology, The Children's Hospital of Philadelphia, Philadelphia, PA 19104, USA</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Division of Biostatistics, University of Texas School of Public Health, Houston, TX 77030, USA,  and</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Department of Pediatrics, The Perelman School of Medicine, University of Pennsylvania, Philadelphia, PA 19104, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +1 7135009954; Fax: +1 7135000900; Email: <a href="mailto:dev@null" data-email="ude.cmt.htu@llehctim.e.arual" class="oemail">ude.cmt.htu@llehctim.e.arual</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140441410585904_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 May 13; Revised 2014 Aug 4; Accepted 2014 Aug 13.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140441410585904_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2014. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4326325/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4326325/bin/supp_24_1_265__index.html" data-ga-action="click_feat_suppl">supp_24_1_265__index.html</a><span style="color:gray"> (1.3K)</span></div><div class="small guid">GUID:&nbsp;E1EC1CDA-01D2-43E9-B598-672E491E15D1</div>
<div><a href="/pmc/articles/PMC4326325/bin/supp_ddu420_ddu420supp.docx" data-ga-action="click_feat_suppl">supp_ddu420_ddu420supp.docx</a><span style="color:gray"> (40K)</span></div><div class="small guid">GUID:&nbsp;5297BE9D-17AF-4DC5-BC4C-1A098F47352B</div>
<div><a href="/pmc/articles/PMC4326325/bin/supp_ddu420_ddu420supp_fig1.tif" data-ga-action="click_feat_suppl">supp_ddu420_ddu420supp_fig1.tif</a><span style="color:gray"> (4.8M)</span></div><div class="small guid">GUID:&nbsp;C4D4D830-663A-442D-91D3-CC522930D165</div>
<div><a href="/pmc/articles/PMC4326325/bin/supp_ddu420_ddu420supp_fig2.tif" data-ga-action="click_feat_suppl">supp_ddu420_ddu420supp_fig2.tif</a><span style="color:gray"> (8.7M)</span></div><div class="small guid">GUID:&nbsp;106181D2-61CE-40D7-81A0-BE1B7DED96BC</div>
<div><a href="/pmc/articles/PMC4326325/bin/supp_ddu420_ddu420supp_fig3.tif" data-ga-action="click_feat_suppl">supp_ddu420_ddu420supp_fig3.tif</a><span style="color:gray"> (38K)</span></div><div class="small guid">GUID:&nbsp;B321D195-70C8-4133-BE7D-F497D69FFA50</div>
</div></dd></dl></div><div id="idm140441332954272" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441332954272title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Congenital left-sided lesions (LSLs) are serious, heritable malformations of the heart. However, little is known about the genetic causes of LSLs. This study was undertaken to identify common variants acting through the genotype of the affected individual (i.e. case) or the mother (e.g. via an <em>in utero</em> effect) that influence the risk of LSLs. A genome-wide association study (GWAS) was performed using data from 377 LSL case-parent triads, with follow-up studies in an independent sample of 224 triads and analysis of the combined data. Associations with both the case and maternal genotypes were assessed using log-linear analyses under an additive model. An association between LSLs and the case genotype for one intergenic SNP on chromosome 16 achieved genome-wide significance in the combined data (rs8061121, combined <em>P</em> = 4.0 × 10<sup>−9</sup>; relative risk to heterozygote: 2.6, 95% CI: 1.9–3.7). In the combined data, there was also suggestive evidence of association between LSLs and the case genotype for a variant in the synaptoporin gene (rs1975649, combined <em>P</em> = 3.4 × 10<sup>−7</sup>; relative risk to heterozygote: 1.6, 95% CI: 1.4–2.0) and between LSLs and the maternal genotype for an intergenic SNP on chromosome 10 (rs11008222, combined <em>P</em> = 6.3 × 10<sup>−7</sup>; relative risk to heterozygote: 1.6, 95% CI: 1.4–2.0). This is the first GWAS of LSLs to evaluate associations with both the case and maternal genotypes. The results of this study identify three candidate LSL susceptibility loci, including one that appears to be associated with the risk of LSLs via the maternal genotype.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Congenital heart defects are the most common, serious type of birth defect (<a href="#DDU420C1" rid="DDU420C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773307">1</a>–<a href="#DDU420C3" rid="DDU420C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773311">3</a>) and are the leading cause of birth defect-related infant mortality (<a href="#DDU420C4" rid="DDU420C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773276">4</a>). In addition, as survival of affected infants has improved, congenital heart defects have increasingly become an adult healthcare issue (<a href="#DDU420C5" rid="DDU420C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773265">5</a>, <a href="#DDU420C6" rid="DDU420C6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773260">6</a>). In general, congenital heart defects are thought to be complex conditions that are influenced by the genotype of the developing embryo as well as <em>in utero</em> exposures, which may be determined or influenced by the maternal genotype [e.g. (<a href="#DDU420C7" rid="DDU420C7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773294">7</a>–<a href="#DDU420C10" rid="DDU420C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773289">10</a>)]. However, since congenital heart defects include a wide phenotypic spectrum, studies of potential risk factors are generally focused on phenotypic subsets for which there is evidence of a shared etiology (<a href="#DDU420C11" rid="DDU420C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773296">11</a>).</p><p id="__p3">Congenital left-sided lesions (LSLs), which include hypoplastic left heart syndrome, aortic valve stenosis, coarctation of the aorta, mitral valve anomalies and bicuspid aortic valve are a common and often serious subgroup of congenital heart defects. Although a growing body of evidence suggests that rare <em>de novo</em> mutations and copy number variants contribute to the burden of congenital heart defects, including LSLs [e.g. (<a href="#DDU420C12" rid="DDU420C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773268">12</a>–<a href="#DDU420C16" rid="DDU420C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773301">16</a>)], family studies demonstrate that LSLs are highly heritable (<a href="#DDU420C17" rid="DDU420C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773305">17</a>). Hence, inherited genetic variation must also contribute to the risk of LSLs, and both linkage analyses (<a href="#DDU420C18" rid="DDU420C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773280">18</a>, <a href="#DDU420C19" rid="DDU420C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773308">19</a>) and candidate-gene association studies (<a href="#DDU420C20" rid="DDU420C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773278">20</a>) have identified LSL-related genes and regions. However, little is known about the specific genetic causes of LSLs, or the extent to which the maternal genotype may influence the development of these conditions (e.g. via an effect on the <em>in utero</em> environment).</p><p id="__p4" class="p p-last">To identify novel susceptibility loci that are associated with LSLs, we conducted a case-parent triad genome-wide association study (GWAS) with follow-up of top associations in an independent sample of LSL case-parent triads and, analysis of the combined data. Because the risk of congenital malformations in offspring may be influenced by the maternal genotype [e.g. (<a href="#DDU420C10" rid="DDU420C10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773262">10</a>, <a href="#DDU420C21" rid="DDU420C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773270">21</a>–<a href="#DDU420C24" rid="DDU420C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773257">24</a>)] as well as the genotype of the child and because failure to account for maternal genetic effects can bias estimates of association with the child's genotype (<a href="#DDU420C25" rid="DDU420C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773267">25</a>), our study design and analytical strategy were selected to allow for the evaluation of associations between LSLs and both the genotypes of affected individuals (i.e. cases) and their mothers.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">RESULTS</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Stage 1 association analysis</h3><p id="__p5" class="p p-first">After making exclusions based on standard quality control metrics, data were available for 486,840 genotyped SNPs with a minor allele frequency (MAF) &gt;5% in 923 individuals, representing 377 triads (Stage 1). The demographic and clinical characteristics of the cases used in Stage 1 are summarized in Table&nbsp;<a href="/pmc/articles/PMC4326325/table/DDU420TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU420TB1" rid-ob="ob-DDU420TB1" co-legend-rid=""><span>1</span></a>. The genomic inflation factor was 1.06 for associations with the case genotype and 1.00 for associations with the maternal genotype and quartile–quartile (Q–Q) plots provided no evidence of a systematic deviation from the expected distribution for the test statistics (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material online, Figures S1A and B</a>, last accessed on 25 August 2014). Since tests involving the case genotype using triad data are not subject to bias due to population stratification (<a href="#DDU420C26" rid="DDU420C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773277">26</a>), no attempt was made to reduce the genomic inflation factor (e.g. by adjusting for principal components) for the associations with the case genotype. Log-linear analyses of these data identified five case genotypes and two maternal genotypes with association <em>P</em> &lt; 10<sup>−5</sup> (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material online, Figures S2A and B</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>, last accessed on 25 August 2014).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU420TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p6">Characteristics of cases with left-sided lesions</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441368036960" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left"></th><th rowspan="1" colspan="1" align="left">Stage 1</th><th rowspan="1" colspan="1" align="left">Stage 2</th></tr><tr><th rowspan="1" colspan="1" align="left"><em>n</em> (%)</th><th rowspan="1" colspan="1" align="left"><em>n</em> (%)</th></tr></thead><tbody><tr><td colspan="3" rowspan="1">Race/ethnicity</td></tr><tr><td rowspan="1" colspan="1"> Non-Hispanic Caucasian</td><td rowspan="1" colspan="1">284 (75.3)</td><td rowspan="1" colspan="1">216 (96.4)</td></tr><tr><td rowspan="1" colspan="1"> Other</td><td rowspan="1" colspan="1">93 (24.7)</td><td rowspan="1" colspan="1">8 (3.6)</td></tr><tr><td colspan="3" rowspan="1">Sex</td></tr><tr><td rowspan="1" colspan="1"> Male</td><td rowspan="1" colspan="1">240 (63.7)</td><td rowspan="1" colspan="1">151 (67.4)</td></tr><tr><td rowspan="1" colspan="1"> Female</td><td rowspan="1" colspan="1">137 (36.3)</td><td rowspan="1" colspan="1">72 (32.1)</td></tr><tr><td colspan="3" rowspan="1">Lesion</td></tr><tr><td rowspan="1" colspan="1"> Hypoplastic left heart syndrome</td><td rowspan="1" colspan="1">198 (52.5)</td><td rowspan="1" colspan="1">64 (28.6)</td></tr><tr><td rowspan="1" colspan="1"> Coarctation of the aorta</td><td rowspan="1" colspan="1">111 (29.4)</td><td rowspan="1" colspan="1">93 (41.5)</td></tr><tr><td rowspan="1" colspan="1"> Aortic stenosis</td><td rowspan="1" colspan="1">68 (18.0)</td><td rowspan="1" colspan="1">65 (29.0)</td></tr><tr><td rowspan="1" colspan="1"> Mitral valve anomalies</td><td rowspan="1" colspan="1">0 (0.0)</td><td rowspan="1" colspan="1">2 (0.9)</td></tr><tr><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">377 (100.0)</td><td rowspan="1" colspan="1">224 (100.0)</td></tr></tbody></table></div><div id="largeobj_idm140441368036960" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4326325/table/DDU420TB1/?report=objectonly">Open in a separate window</a></div></div><p id="__p7" class="p p-last">Using the subset of non-Hispanic Caucasian triads, data for non-genotyped autosomal variants were imputed. After making exclusions based on standard quality control metrics, data were available for 1,712,604 imputed SNPs with a MAF &gt;5% in 716 individuals, representing 284 non-Hispanic Caucasian triads. The genomic inflation factor for genotyped and imputed SNPs was 1.01 for associations with the case genotype and 1.00 for associations with the maternal genotype and Q–Q plots provided no evidence of a systematic deviation from the expected distribution for the test statistics (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material online, Figures S1C and D</a>, last accessed on 25 August 2014). Log-linear analyses of the genotyped and imputed variants in this subset of triads identified an additional 23 case and 47 maternal genotypes with association <em>P</em> &lt; 10<sup>−5</sup> (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material online, Figures S2C and D</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S1</a>, last accessed on 25 August 2014). As <em>P</em> &lt; 10<sup>−5</sup> was used to identify suggestive associations in both the analyses of genotyped SNPs only, and analyses of genotyped plus imputed SNPs, it is not surprising that more suggestive associations were identified in the latter than in the former.</p></div><div id="s2b" class="sec"><h3 id="s2btitle">Stage 2 association analysis</h3><p id="__p8" class="p p-first">We genotyped 77 SNPs with association <em>P</em>-values &lt;10<sup>−5</sup> (for associations of LSLs with either the case or maternal genotype) in an additional 224, predominantly non-Hispanic Caucasian, LSL triads (Stage 2). The demographic and clinical characteristics of the cases used in Stage 2 are summarized in Table&nbsp;<a href="/pmc/articles/PMC4326325/table/DDU420TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU420TB1" rid-ob="ob-DDU420TB1" co-legend-rid=""><span>1</span></a>. Log-linear analyses of the variants that passed the quality control metrics identified two case and one maternal genotypes that were associated with LSLs with unadjusted <em>P</em> &lt; 0.05. In the combined data, the <em>P</em>-values for the association between LSLs and these three genotypes were smaller than the corresponding <em>P</em>-values in Stage 1 and the case genotype for an intergenic variant on chromosome 16 achieved genome-wide significance (rs8061121, <em>P</em> = 4.0 × 10<sup>−9</sup>; relative risk to heterozygote: 2.7, 95% CI: 1.9–3.7) (Table&nbsp;<a href="/pmc/articles/PMC4326325/table/DDU420TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU420TB2" rid-ob="ob-DDU420TB2" co-legend-rid=""><span>2</span></a>). The estimated sibling relative risk (λ<sub>S</sub>) attributable to this variant was 1.20. There was also suggestive evidence, in the combined data, of an association between LSLs and the case genotype for a SNP in the synaptoporin (<em>SYNPR)</em> gene (rs1975649, <em>P</em> = 3.4 × 10<sup>−7</sup>; relative risk to heterozygote: 1.7, 95% CI: 1.4–2.0, λ<sub>S</sub> = 1.06) and the maternal genotype for an intergenic SNP on chromosome 10 (rs11008222, <em>P</em> = 6.3 × 10<sup>−7</sup>; relative risk to heterozygote: 1.6, 95% CI: 1.3–2.0) (Table&nbsp;<a href="/pmc/articles/PMC4326325/table/DDU420TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU420TB2" rid-ob="ob-DDU420TB2" co-legend-rid=""><span>2</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU420TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p9">Summary data for variants rs1975649, rs11008222 and rs8061121 in left-sided lesions in Stages 1 and 2</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441374681456" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">SNP (genotyped/imputed in Stage 1)</th><th rowspan="2" colspan="1" align="left">Chr<sup>a</sup></th><th rowspan="2" colspan="1" align="left">MAF<sup>b</sup></th><th rowspan="2" colspan="1" align="left">Position<sup>c</sup> (bp)</th><th rowspan="2" colspan="1" align="left">Gene<sup>d</sup></th><th rowspan="2" colspan="1" align="left">Function</th><th colspan="3" rowspan="1" align="center"><em>P</em>-values<hr></th><th colspan="3" rowspan="1" align="center">Genotype relative risk<sup>e</sup> (95% confidence interval)</th></tr><tr><th rowspan="1" colspan="1" align="left">Stage 1</th><th rowspan="1" colspan="1" align="left">Stage 2</th><th rowspan="1" colspan="1" align="left">Stage 1 + 2</th><th rowspan="1" colspan="1" align="left">Stage 1</th><th rowspan="1" colspan="1" align="left">Stage 2</th><th rowspan="1" colspan="1" align="left">Stage 1 + 2</th></tr></thead><tbody><tr><td colspan="12" rowspan="1">Inherited effects</td></tr><tr><td rowspan="1" colspan="1"> rs8061121 (imputed)<sup>f</sup></td><td rowspan="1" colspan="1">16</td><td rowspan="1" colspan="1">0.11</td><td rowspan="1" colspan="1">85750585</td><td rowspan="1" colspan="1">(<a id="__tag_426773362" class="tag_hotlink" href="/nuccore/AC136285.1" ref="/nuccore/AC136285.1"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AC136285.1","term_id":"24431621","term_text":"AC136285.1"}}</span>AC136285.1</a>)</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">2.44 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.005</td><td rowspan="1" colspan="1">4.04 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">2.94 (1.91–4.54)</td><td rowspan="1" colspan="1">2.16 (1.24–3.76)</td><td rowspan="1" colspan="1">2.65 (1.89–3.71)</td></tr><tr><td rowspan="1" colspan="1"> rs1975649 (genotyped)</td><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">0.33</td><td rowspan="1" colspan="1">63413926</td><td rowspan="1" colspan="1"><em>SYNPR</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1">1.08 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.03</td><td rowspan="1" colspan="1">3.38 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.87 (1.45–2.43)</td><td rowspan="1" colspan="1">1.38 (1.03–1.87)</td><td rowspan="1" colspan="1">1.65 (1.36–2.01)</td></tr><tr><td colspan="12" rowspan="1">Maternal effects</td></tr><tr><td rowspan="1" colspan="1"> rs11008222 (genotyped)</td><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">0.49</td><td rowspan="1" colspan="1">31082750</td><td rowspan="1" colspan="1">(<em>ZNF438</em>)</td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1">9.60 × 10<sup>−6</sup></td><td rowspan="1" colspan="1">0.02</td><td rowspan="1" colspan="1">6.27 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">1.68 (1.32–2.13)</td><td rowspan="1" colspan="1">1.55 (1.06–2.26)</td><td rowspan="1" colspan="1">1.64 (1.34–2.00)</td></tr></tbody></table></div><div id="largeobj_idm140441374681456" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4326325/table/DDU420TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><p id="__p10" class="p p-first-last"><sup>a</sup>Chromosome.</p></div><div id="TN2"><p id="__p11" class="p p-first-last"><sup>b</sup>MAF among study participant founders (i.e. mother and father).</p></div><div id="TN3"><p id="__p12" class="p p-first-last"><sup>c</sup>Hg18/NCBI build 36.</p></div><div id="TN4"><p id="__p13" class="p p-first-last"><sup>d</sup>For SNPs mapping within genes, gene names are listed, and for intergenic SNPs, the nearest gene is listed in parentheses.</p></div><div id="TN5"><p id="__p14" class="p p-first-last"><sup>e</sup>Relative risk estimate for carrying one copy of the high-risk allele compared to no copies, and corresponding 95% confidence interval.</p></div><div id="TN6"><p id="__p15" class="p p-first-last"><sup>f</sup><em>P</em>-value for this variant in Stage 1 is based on the analyses of the subgroup of non-Hispanic Caucasian triads.</p></div></div></div><p id="__p16" class="p p-last">In Stage 1, genotypes for two of the top SNPs (rs1975649 and rs1100822) were derived from the array data (see <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material online, Figures S3A and B</a> for cluster plots, last accessed on 25 August 2014) and genotypes for the third SNP (rs8061121) were based on imputation. In the samples from Stage 1 that were successfully genotyped in Stage 2, genotype concordance was ≥97% (concordance for rs8061121, which was imputed in Stage 1, was 100%). Further, regional association plots demonstrate that SNPs in linkage disequilibrium with these three variants (i.e. rs8061121, rs1975649 and rs11008222) were also nominally associated (<em>P</em> &lt; 0.05) with LSLs in Stage 1 (Fig.&nbsp;<a href="/pmc/articles/PMC4326325/figure/DDU420F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU420F1" rid-ob="ob-DDU420F1" co-legend-rid="lgnd_DDU420F1"><span>1</span></a>A–C). In particular, the maternal genotype for several SNPs in linkage disequilibrium with rs11008222 had association <em>P</em>-values between 0.001 and 0.0001. Although no other SNPs were strongly associated with LSLs in the regions surrounding rs8061121 and rs1975649 (Figs&nbsp;<a href="/pmc/articles/PMC4326325/figure/DDU420F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU420F1" rid-ob="ob-DDU420F1" co-legend-rid="lgnd_DDU420F1"><span>1</span></a>A and C), there were also no genotyped or imputed SNPs in strong linkage disequilibrium with these variants. Further, as no proxies for rs8061121 and rs1975649 were identified using SNAP (<a href="http://www.broadinstitute.org/mpg/snap/ldsearch.php">http://www.broadinstitute.org/mpg/snap/ldsearch.php</a>, default settings), imputation of a denser SNP panel would be unlikely to provide additional evidence either for or against an association in these regions.
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU420F1" co-legend-rid="lgnd_DDU420F1"><a href="/pmc/articles/PMC4326325/figure/DDU420F1/" target="figure" rid-figpopup="DDU420F1" rid-ob="ob-DDU420F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441368025696" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddu42001a.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddu42001a.jpg" src="/pmc/articles/PMC4326325/bin/ddu42001a.jpg"></div></a><div id="largeobj_idm140441368025696" class="largeobj-link align_right" style=""><a href="/pmc/articles/PMC4326325/figure/DDU420F1/" target="figure" rid-figpopup="DDU420F1" rid-ob="ob-DDU420F1"></a><a target="object" href="/pmc/articles/PMC4326325/figure/DDU420F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140441368024848" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4326325_ddu42001b.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu42001b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4326325/bin/ddu42001b.jpg"></a></div><div id="largeobj_idm140441368024848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4326325/figure/DDU420F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU420F1"><div><a class="figpopup" href="/pmc/articles/PMC4326325/figure/DDU420F1/" target="figure" rid-figpopup="DDU420F1" rid-ob="ob-DDU420F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p17">Regional association plots for three loci with genome-wide significant or suggestive associations with LSLs. Each pane: (<strong>A</strong>) rs1975649 (<strong>B</strong>) rs11008222 (<strong>C</strong>) rs8061121, shows the association statistic (−log<sub>10</sub>
<em>P</em> for the likelihood ratio test) from Stage 1 for the variant (diamond) and nearby markers (squares) on the left <em>y</em>-axis. The red shading indicates linkage disequilibrium (<em>r</em><sup>2</sup>) between the variant and nearby markers. The light blue lines indicate recombination rates across each region in 1000 Genomes Pilot 1 CEU data (right <em>y</em>-axis). Chromosome position (hg18) and nearby genes are shown on the <em>x</em>-axis.</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Pathway analysis</h3><p id="__p18" class="p p-first-last">Using data for both genotyped and imputed SNPs in the Stage 1, Caucasian subset, we assessed enrichment of Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) terms for regions containing SNPs with association <em>P</em>-values &lt;10<sup>−5</sup> using INRICH (<a href="#DDU420C27" rid="DDU420C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773303">27</a>). Analyses were performed separately for inherited and maternal genotypes and were repeated using regions containing SNPs with association <em>P</em>-values &lt;10<sup>−4</sup>. No pathways with a corrected <em>P</em>-value &lt; 0.05 were identified.</p></div><div id="s2d" class="sec sec-last"><h3 id="s2dtitle">Annotation</h3><p id="__p19" class="p p-first">The suggestive association with the maternal genotype involved a variant at 10p11.23 (rs11008222) that is located in the intergenic region between a predicted gene (<a id="__tag_426773361" class="tag_hotlink" href="/nuccore/AK302694" ref="/nuccore/AK302694"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK302694","term_id":"194380311","term_text":"AK302694"}}</span>AK302694</a>) and coding gene <em>ZNF438</em> (∼91 kb downstream), a <em>C2H2</em>-type zinc finger gene that is likely a transcriptional repressor (<a href="#DDU420C28" rid="DDU420C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773285">28</a>). The predicted gene, <a id="__tag_426773360" class="tag_hotlink" href="/nuccore/AK302694" ref="/nuccore/AK302694"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK302694","term_id":"194380311","term_text":"AK302694"}}</span>AK302694</a>, is highly similar to the F-actin-binding protein <em>Supervillin</em> and is likely to be transcribed and translated into a protein (txCdsPredict score, 1874). Upstream, the nearest known coding gene is <em>LYZL2</em> (MIM 612748) (∼124 kb), which encodes lysozyme-like 2. <em>LYZL2</em> is expressed in the placenta and may play a role in host defense against infection (<a href="#DDU420C29" rid="DDU420C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773309">29</a>). There are also clusters of transcription factor-binding sites ∼600 bp upstream (binding sites for GATA-1, GATA-2, GABP, CCNT2, RNA Pol II and TAL1) and 600 bp downstream (binding sites for EBF and EBF1) of rs11008222 and areas of open chromatin at ∼11, 30 and 64 kb downstream of this variant. The area of open chromatin at 30 kb coincides with a cluster of transcription factor binding sites.</p><p id="__p20">The GWAVA scores for rs11008222 range from 0.14 to 0.30, and the GWAVA-TSS score (0.30) falls at approximately the median value for GWAS SNPs that have been externally replicated. Among the SNPs in linkage disequilibrium (<em>r</em><sup>2</sup> &gt; 0.80) with rs11008222, the highest GWAVA scores were obtained for rs10826861 (range: 0.37–0.43), which had a <em>P</em>-value = 0.0005 in the Stage 1 analysis in the Caucasian subset. The GWAVA-TSS score (0.43) for this variant falls within the third quartile of scores for GWAS SNPs that have been externally replicated. The relatively high scores for this variant appear to be driven by evolutionary conservation at this site (GERP score: 4.09). Expression quantitative trait analyses provided no evidence that rs11008222 is associated with gene expression.</p><p id="__p21">The suggestive association with the inherited genotype involved a variant at 3p14.2 (rs1975649) that is located in intron 2 of the synaptoporin (<em>SYNPR)</em> gene. <em>SYNPR</em> is a synaptic membrane protein of synaptic vesicles and a member of the synaptophysin gene family that is involved in the regulation of neurotransmitters. In adults, <em>SYNPR</em> appears to be expressed in the brain (<a href="#DDU420C30" rid="DDU420C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773288">30</a>) and the atrium (<a href="#DDU420C31" rid="DDU420C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773312">31</a>). The GWAVA scores for this variant ranged from 0.15 to 0.18, and the GWAVA-TSS score (0.15) falls below the median value for GWAS SNPs that have been externally replicated. Expression quantitative trait analyses provided no evidence that rs1975649 is associated with gene expression.</p><p id="__p22" class="p p-last">The genome-wide significant association with the inherited genotype involved a variant at 16q24.2 (rs8061121) that is located within the predicted gene <a id="__tag_426773359" class="tag_hotlink" href="/nuccore/AK125749" ref="/nuccore/AK125749"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"AK125749","term_id":"34531953","term_text":"AK125749"}}</span>AK125749</a> (txCdsPredict score, 148). The nearest upstream genes (∼578 kb), <em>FOXC2</em> (forkhead box protein C2, MIM:602402) and <em>FOXL1</em> (forkhead box protein L1, MIM:603252), are members of the forkhead family of transcription factors that are involved in cardiac development (<a href="#DDU420C32" rid="DDU420C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773259">32</a>–<a href="#DDU420C34" rid="DDU420C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773298">34</a>). Deletion of these genes has been associated with multiple congenital malformations including hypoplastic left heart syndrome (<a href="#DDU420C35" rid="DDU420C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773299">35</a>). The closest downstream gene (∼143 kb), <em>C16orf95</em>, has unknown function, but deletion of this gene, as part of a <em>de novo</em> copy number variant, has been reported in a patient with tetralogy of Fallot (<a href="#DDU420C12" rid="DDU420C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773300">12</a>). This variant (rs8061121) is located within 50 bp of a predicted binding site for NKX2.5, a transcription factor critical to cardiac development and congenital heart disease (<a href="#DDU420C36" rid="DDU420C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773281">36</a>, <a href="#DDU420C37" rid="DDU420C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773269">37</a>). Further, a cluster of transcription factor binding sites (FOS, GATA2, JunB, JunD and NR4A1) is located ∼14 kb upstream, and a FOS binding site is located ∼15 kb downstream of rs8061121. Binding sites for the transcriptional repressor, CTCF, are also located ∼14 kb upstream and 15 kb downstream of rs8061121. There are also several regions of open chromatin between rs8061121 and <em>FOXL1</em> as well as downstream of this variant. Finally, a cluster of miRNA targets (miR-138/138ab, miR-455–5p, miR-338/338–3p, miR-9/9ab, miR-133abc, miR-26ab/1297/4465 and miR-143/1721/4770) is predicted to occur ∼18 kb upstream of rs8061121. The GWAVA scores for this variant ranged from 0.07 to 0.31, and the GWAVA-TSS score (0.08) falls below the median value for GWAS SNPs that have been externally replicated. Expression quantitative trait analyses were not performed for rs8061121 because this variant was not identified in the Geneva GenCord database.</p></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">DISCUSSION</h2><p id="__p23" class="p p-first">Congenital heart defects occur in ∼1 in 200 live births and are associated with significant infant mortality and morbidity (<a href="#DDU420C1" rid="DDU420C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773274">1</a>–<a href="#DDU420C4" rid="DDU420C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773261">4</a>). LSLs are among the most common, severe congenital heart defects, accounting for ∼13% of all affected infants. Family studies, which demonstrate that the various LSL subtypes (e.g. hypoplastic left heart, coarctation of the aorta, aortic valve stenosis) co-segregate within families, indicate that the LSL subtypes are genetically related and highly heritable (<a href="#DDU420C17" rid="DDU420C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773273">17</a>). These studies, in conjunction with emerging evidence that <em>de novo</em> mutations (<a href="#DDU420C13" rid="DDU420C13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773290">13</a>) and rare copy number variants (<a href="#DDU420C12" rid="DDU420C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773282">12</a>, <a href="#DDU420C14" rid="DDU420C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773275">14</a>–<a href="#DDU420C16" rid="DDU420C16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773279">16</a>) are associated with LSLs, suggest that both inherited variation and <em>de novo</em> events contribute to the risk of LSLs. Further, epidemiological studies have identified associations between LSLs and a range of exposures [e.g.(<a href="#DDU420C38" rid="DDU420C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_469840645">38</a>–<a href="#DDU420C41" rid="DDU420C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773297">41</a>)], many of which may be determined (e.g. maternal obesity) or modulated (e.g. exposure to medications) by the genotype of the mother, suggesting that the maternal genotype may also influence the risk of LSLs in offspring.</p><p id="__p24">To explore the potential contribution of both inherited case and maternal genotypes to LSLs, a case-parent triad GWAS was undertaken. Using log-linear models, one genome-wide significant association (rs8061121) and two suggestive associations (rs1975649 and rs11008222) were identified, one of which was associated with LSLs via the maternal genotype (rs11008222). When we began this study, there were no reported GWAS of congenital heart defects. While three such studies have subsequently been published, none of them evaluated associations between the maternal genotype and LSLs. Further, two of these studies focused on heart defects other than LSLs [i.e. tetralogy of Fallot (<a href="#DDU420C42" rid="DDU420C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773264">42</a>) and septal defects (<a href="#DDU420C43" rid="DDU420C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773256">43</a>)]. The third study included a broad range of congenital heart defects (<a href="#DDU420C44" rid="DDU420C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773284">44</a>), but conducted a subgroup analysis of LSLs (∼400 cases). However, no genome-wide significant (<em>P</em> &lt; 5 × 10<sup>−8</sup>) associations were identified in the LSL subgroup. Our data provided no evidence of association between LSLs and any of the top hits in these three studies (<em>P</em> ≥ 0.29). Further, in the GWAS of tetralogy of Fallot (the only one of the three published GWAS to provide access to summary data from all SNPs), only one of our top hits was evaluated (rs1975649), and there was no evidence for an association between this variant and TOF (<em>P</em> = 0.38). Given the etiologic complexity and heterogeneity of heart defects, it is not entirely surprising that there is no overlap between the genes/regions identified in these distinct studies.</p><p id="__p25">The results of this study should be interpreted in light of its limitations, including the relatively small sample sizes available for both Stages 1 and 2. However, as little is known about the specific genes that influence the risk of LSL, the identification of three loci with at least suggestive evidence of association with LSLs provides new insights regarding the potential genetic underpinnings of this condition. Further, our finding of a suggestive association with the maternal genotype highlights the importance of considering maternal genetic effects in studies of LSLs and other conditions that have their origin <em>in utero</em>. Nonetheless, given that genome-wide significance (i.e. <em>P</em> &lt; 10<sup>−8</sup>) was attained for only one variant and only in the combined data, these findings require follow-up in additional cohorts.</p><p id="__p26">This study also had several strengths, including well-phenotyped cases ascertained from a single centre. In addition, the family-based approach is robust to the potentially biasing influences of population stratification and provides unbiased assessment of associations with both the case and maternal genotypes.</p><p id="__p27" class="p p-last">In summary, our findings, as well as those from other recent genomic studies of LSLs, highlight the potential utility of genome-wide approaches for LSLs, despite the relatively small sample sizes that are generally available for this condition. Future studies should seek to confirm the associations reported here. Larger studies, with increased power to detect associations with LSLs, are also warranted and should include consideration of both the case and maternal genotypes.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">MATERIALS AND METHODS</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Human subjects</h3><p id="__p28" class="p p-first">Cases with an LSL and their parents were recruited from the Cardiac Center at The Children's Hospital of Philadelphia with approval from the Institutional Review Board for the Protection of Human Subjects. Written informed consent and, when appropriate, assent was obtained for each study subject.</p><p id="__p29" class="p p-last">Cases used in Stage 1 (<em>n</em> = 405 LSL case-parent triads) were recruited between 1997 and 2007, and those used in Stage 2 (<em>n</em> = 236 triads), was recruited between 1997 and 2010. Cases in both stages included males and females with an LSL including: hypoplastic left heart syndrome, coarctation of the aorta with or without bicuspid aortic valve, aortic valve stenosis and isolated mitral valve anomalies. A detailed cardiac phenotype was ascertained from all available cardiac medical records, including echocardiography, cardiac MRI, cardiac catheterization and/or operative notes. If there was any discrepancy or ambiguity in the diagnosis, then original images were inspected by one reviewer (E.G.). Individuals with variants of hypoplastic left heart syndrome, such as mal-aligned atrioventricular canal defects and double outlet right ventricle with mitral valve atresia, were excluded from the study. Medical records were also reviewed to identify other medical conditions, and cases with a recognized or suspected genetic syndrome were excluded from the study. Cases in Stage 1 included individuals of all races and ethnicities, whereas cases in Stage 2 were selected to be predominantly non-Hispanic Caucasian.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">GWAS genotyping and quality control</h3><p id="__p30" class="p p-first-last">Pre-transfusion blood samples were collected from cases at the time of admission to the hospital and blood or saliva samples were collected from available parents. DNA was extracted from the study samples using standard methods (Puregene DNA isolation kit by Gentra Systems, Inc., Minneapolis, MN, USA). For Stage 1, genotyping was performed at the Center for Applied Genomics at The Children's Hospital of Philadelphia, using the Illumina Infinium™ II HumanHap550K (V3), as previously described (<a href="#DDU420C45" rid="DDU420C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773292">45</a>). Standard quality control measures included exclusion of BeadChips with call rates &lt;97.5% and individual SNPs with &lt;95% genotyping yields. In addition, we excluded from statistical analyses data for: non-autosomal SNPs; SNPs with a MAF &lt;5%; SNPs for which the genotype distribution in parents deviated from the Hardy–Weinberg equilibrium (<em>P</em> &lt; 1 × 10<sup>−5</sup>); SNPs for which &gt;1% of triads had a Mendelian inheritance errors; triads with Mendelian inheritance errors for &gt;1% of the genotyped variants and individuals with a genotype call rate &lt;95%.</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">Imputation</h3><p id="__p31" class="p p-first-last">Imputation for non-genotyped autosomal SNPs was carried out using MACH version 1.0.16 (<a href="#DDU420C46" rid="DDU420C46" class=" bibr popnode">46</a>) (<a href="http://www.sph.umich.edu/csg/abecasis/MACH/">http://www.sph.umich.edu/csg/abecasis/MACH/</a>). Phased haplotypes for HapMap II CEU subjects (<em>n</em> = 60 founders) were used as the reference for imputing genotypes in the subset of Stage 1 triads in which all members were reported to be non-Hispanic and Caucasian (<em>n</em> = 284 triads). Imputation was based on genotyped SNPs with a MAF &gt;10% and that met all other quality control criteria. Imputed SNPs with imputation <em>r<sub>i</sub></em><sup>2</sup> (i.e. estimated squared correlation between imputed and actual genotypes) &lt;0.3 were excluded. Data for imputed SNPs that were non-autosomal or that had a MAF &lt;5% were also excluded from all statistical analyses.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle">SNP selection and genotyping</h3><p id="__p32" class="p p-first-last">SNPs (genotyped or imputed) with association <em>P</em>&lt; 1 × 10<sup>−5</sup> in Stage 1 were genotyped in Staged 2. This genotyping was performed using the Illumina Veracode GoldenGate assay (<a href="#DDU420C47" rid="DDU420C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773310">47</a>) in The Center for Applied Genomics at The Children's Hospital of Philadelphia. Samples from 24 Stage 1, case-parent triads were also genotyped using this platform, for comparisons with genotypes that were imputed in Stage 1. Quality control filters for SNPs genotyped in Stage 2 were identical to those used for genotyped SNPs in Stage 1.</p></div><div id="s4e" class="sec"><h3 id="s4etitle">Statistical analysis</h3><p id="__p33" class="p p-first">The log-linear likelihood approach (<a href="#DDU420C26" rid="DDU420C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773283">26</a>, <a href="#DDU420C48" rid="DDU420C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773302">48</a>), implemented using the MI-GWAS platform (<a href="#DDU420C49" rid="DDU420C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773291">49</a>) was used to assess the association between LSLs and both the case and maternal genotypes for each variant. This approach has been widely applied in candidate gene and genome-wide association studies of birth defects and other pediatric conditions [e.g. (<a href="#DDU420C22" rid="DDU420C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773258">22</a>, <a href="#DDU420C50" rid="DDU420C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773263">50</a>–<a href="#DDU420C56" rid="DDU420C56" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773293">56</a>)]. Briefly, each triad is characterized by the number of high-risk alleles (0, 1, 2) in the genotypes of the father, mother and child. Asymmetry in the distribution of the case genotypes, conditional on parental genotypes, permits estimation of the case genotypic effects and asymmetry in the distribution of reciprocal mating types permits estimation of maternal genotypic effects. The likelihood ratio test comparing log-linear models with and without parameters characterizing the case genotypic effects is a generalization of the transmission disequilibrium test that is not subject to population stratification bias (although the power to detect an association in a stratified sample may be reduced if the association is not present in all sub-populations) and provides results that are comparable with PLINK (<a href="http://pngu.mgh.harvard.edu/~purcell/plink/">http://pngu.mgh.harvard.edu/~purcell/plink/</a>) (<a href="#DDU420C49" rid="DDU420C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773286">49</a>). However, unlike the analyses implemented in PLINK, the log-linear approach can be used to assess the association of the maternal genotype with the offpsring phenotype. Further, use of the expectation-maximization algorithm allows for the inclusion of data from incomplete triads (e.g. triads missing data for one parent) without invalidation of the results and relatively little loss of power compared to analyses based on a similar number of complete triads (<a href="#DDU420C57" rid="DDU420C57" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773255">57</a>).</p><p id="__p34">Analyses were performed using data from Stage 1 triads and repeated using data from the subset of Stage 1 triads in which both parents were self-reported non-Hispanic Caucasian. This subset was selected for analysis, <em>a priori</em>, because the inclusion of triads with mixed parental races/ethnicities (e.g. mother non-Hispanic Caucasian&nbsp;× father Hispanic) can bias estimates of maternal genetic effects (<a href="#DDU420C58" rid="DDU420C58" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773266">58</a>). Data from this subset were also assessed for enrichment of GO and Kyoto Encyclopedia of Genes and Genomes (KEGG) terms. Enrichment analyses were performed separately for inherited and maternal genotypes using INRICH (<a href="#DDU420C27" rid="DDU420C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773306">27</a>) for regions containing SNPs with association <em>P</em>-values &lt;10<sup>−5</sup> and were repeated for regions containing SNPs with association <em>P</em>-values &lt;10<sup>−4</sup>. Pathways with <em>P</em>-values &lt;0.05, after correction for the total number of pathways evaluated, were considered to be significantly enriched.</p><p id="__p35">All log-linear analyses were performed using an additive model for the genotype being tested (e.g. case genotype) and an unrestricted model for the other (e.g. maternal) genotype. Statistical significance was assessed using a one-degree of freedom likelihood ratio test to compare models that included a parameter for the genotype being tested to models that excluded this parameter. Quartile–quartile (Q–Q) plots and estimates of lambda were generated using R version 2.15 (<a href="http://www.r-project.org/">http://www.r-project.org/</a>).</p><p id="__p36" class="p p-last">SNPs with association <em>P</em>-values &lt;10<sup>−5</sup> in Stage 1 were assessed in Stage 2 and SNPs with unadjusted <em>P</em>-values &lt;0.05 in Stage 2 were assessed in the combined data. Pooled data from the two stages were analyzed, rather than conducting a meta-analysis, because the two sets of triads were ascertained using identical protocols at a single institution during overlapping time periods. SNPs for which the <em>P</em>-value in the combined analysis was lower than that in Stage 1 were considered to be potential LSL susceptibility loci. For these loci, data from Stage 1 were used to generate regional association plots with SNAP version 2.2 (<a href="http://www.broadinstitute.org/mpg/snap/ldplot.php">http://www.broadinstitute.org/mpg/snap/ldplot.php</a>). In addition, the relative increase in risk to a sibling attributable to each potential inherited LSL susceptibility locus (λ<sub>S</sub>) was calculated (<a href="#DDU420C59" rid="DDU420C59" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773295">59</a>), using the MAF among parents to estimate the frequency of the high-risk allele and parameter estimates from the additive log-linear models to estimate genotype relative risks.</p></div><div id="s4f" class="sec sec-last"><h3 id="s4ftitle">Variant annotation</h3><p id="__p37" class="p p-first-last">For each potential susceptibility locus, the UCSC genome browser (<a href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>) was used to identify coding and predicted genes flanking variants of interest and to determine TXCdsPredict scores. Visualization of transcription factor-binding sites, regions of open chromatin and miRNA-binding sites was performed using ENCODE transcription factor binding tracks, ENCODE Open Chrom Synth tracks and the TargetScan track on the UCSC genome browser. Transcription factor binding sites were also identified using TESS (<a href="http://www.cbil.upenn.edu/tess/">http://www.cbil.upenn.edu/tess/</a>). Genome-wide annotation of variants (GWAVA) scores (<a href="http://www.sanger.ac.uk/resources/software/gwava/">http://www.sanger.ac.uk/resources/software/gwava/</a>), which range from 0 to 1, were used to assess the potential impact of non-coding variants on gene regulation (<a href="#DDU420C60" rid="DDU420C60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559057684">60</a>). In addition, the GWAVA-TSS score, which is based on comparisons with a control set of SNPs matched for distance to the nearest transcription factor start site, was compared with the distribution of GWAVA-TSS scores for GWAS SNPs that have been externally replicated (<a href="#DDU420C60" rid="DDU420C60" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_559057683">60</a>). When appropriate, GWAVA scores were also assessed for SNPs in linkage disequilibrium with our top hits. In addition, eQTL analyses were performed for each non-coding variant using Genevar (<a href="#DDU420C61" rid="DDU420C61" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_426773272">61</a>). For variants associated with the risk of LSLs via the case genotype, Genevar was run using data from three cell types (fibroblast, lymphobastoid cell lines and T-cells) derived from umbilical cords of 75 Geneva GenCord individuals. For variants associated with the risk of LSLs via the maternal genotype, Genevar was run using data from three tissue types (adipose, lymphoblastoid cell lines and skin) from twin 1 of ∼90 MuTHER healthy female twin pairs. The significance of SNP–probe associations was assessed using a non-parametric permutation <em>P</em>-value, based on 10,000 permutations, and <em>P</em> values &lt;0.05 were considered significant.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">SUPPLEMENTARY MATERIAL</h2><p id="__p38" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu420/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary material is available at <em>HMG</em> online.</a></p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">FUNDING</h2><p id="__p39" class="p p-first-last">This work was supported by the National Institutes of Health (HL74094 to E.G. and L.E.M.) and the National Center for Research Resources (RR024134), which is now the National Center for Advancing Translational Sciences (UL1TR000003), both of which supported case ascertainment; and an Institutional Development Fund to The Center for Applied Genomics from The Children's Hospital of Philadelphia, which supported the genotyping of study subjects. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140441373680640"><a href="/pmc/articles/PMC4326325/bin/supp_24_1_265__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>




</div></div><div id="idm140441374744016" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441374744016title">ACKNOWLEDGEMENTS</h2><div class="sec"><p id="__p40">Summary results are available on dbGAP (<a href="http://www.ncbi.nlm.nih.gov/gap">http://www.ncbi.nlm.nih.gov/gap</a>, accession number: phs000781.v1.p1). We thank the families for their participation, Jennifer Garbarini and Stacy Woyciechowski for case ascertainent, and Sharon Edman for data management.</p><p id="__p41"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140441368490608" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140441368490608title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDU420C1">1. <span class="element-citation">Botto L.D., Correa A., Erickson J.D. Racial and temporal variations in the prevalence of heart defects. <span><span class="ref-journal">Pediatrics. </span>2001;<span class="ref-vol">107</span>:E32.</span> [<a href="/pubmed/11230613" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatrics&amp;title=Racial+and+temporal+variations+in+the+prevalence+of+heart+defects&amp;author=L.D.+Botto&amp;author=A.+Correa&amp;author=J.D.+Erickson&amp;volume=107&amp;publication_year=2001&amp;pages=E32&amp;pmid=11230613&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C2">2. <span class="element-citation">Ferencz C., Rubin J.D., McCarter R.J., Brenner J.I., Neill C.A., Perry L.W., Hepner S.I., Downing J.W. Congenital heart disease: prevalence at livebirth. The Baltimore-Washington Infant Study. <span><span class="ref-journal">Am. J. Epidemiol. </span>1985;<span class="ref-vol">121</span>:31–36.</span> [<a href="/pubmed/3964990" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Congenital+heart+disease:+prevalence+at+livebirth.+The+Baltimore-Washington+Infant+Study&amp;author=C.+Ferencz&amp;author=J.D.+Rubin&amp;author=R.J.+McCarter&amp;author=J.I.+Brenner&amp;author=C.A.+Neill&amp;volume=121&amp;publication_year=1985&amp;pages=31-36&amp;pmid=3964990&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C3">3. <span class="element-citation">Moons P., Sluysmans T., De Wolf D., Massin M., Suys B., Benatar A., Gewillig M. Congenital heart disease in 111 225 births in Belgium: birth prevalence, treatment and survival in the 21st century. <span><span class="ref-journal">Acta Paediatr. </span>2009;<span class="ref-vol">98</span>:472–477.</span> [<a href="/pubmed/19046347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Acta+Paediatr.&amp;title=Congenital+heart+disease+in+111+225+births+in+Belgium:+birth+prevalence,+treatment+and+survival+in+the+21st+century&amp;author=P.+Moons&amp;author=T.+Sluysmans&amp;author=D.+De+Wolf&amp;author=M.+Massin&amp;author=B.+Suys&amp;volume=98&amp;publication_year=2009&amp;pages=472-477&amp;pmid=19046347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C4">4. <span class="element-citation">Yang Q., Chen H., Correa A., Devine O., Mathews T.J., Honein M.A. Racial differences in infant mortality attributable to birth defects in the United States, 1989–2002. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2006;<span class="ref-vol">76</span>:706–713.</span> [<a href="/pubmed/17022030" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=Racial+differences+in+infant+mortality+attributable+to+birth+defects+in+the+United+States,+1989–2002&amp;author=Q.+Yang&amp;author=H.+Chen&amp;author=A.+Correa&amp;author=O.+Devine&amp;author=T.J.+Mathews&amp;volume=76&amp;publication_year=2006&amp;pages=706-713&amp;pmid=17022030&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C5">5. <span class="element-citation">Opotowsky A.R., Siddiqi O.K., Webb G.D. Trends in hospitalizations for adults with congenital heart disease in the U.S. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2009;<span class="ref-vol">54</span>:460–467.</span> [<a href="/pubmed/19628123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=Trends+in+hospitalizations+for+adults+with+congenital+heart+disease+in+the+U.S&amp;author=A.R.+Opotowsky&amp;author=O.K.+Siddiqi&amp;author=G.D.+Webb&amp;volume=54&amp;publication_year=2009&amp;pages=460-467&amp;pmid=19628123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C6">6. <span class="element-citation">van der Bom T., Zomer A.C., Zwinderman A.H., Meijboom F.J., Bouma B.J., Mulder B.J. The changing epidemiology of congenital heart disease. <span><span class="ref-journal">Nat. Rev. Cardiol. </span>2011;<span class="ref-vol">8</span>:50–60.</span> [<a href="/pubmed/21045784" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cardiol.&amp;title=The+changing+epidemiology+of+congenital+heart+disease&amp;author=T.+van+der+Bom&amp;author=A.C.+Zomer&amp;author=A.H.+Zwinderman&amp;author=F.J.+Meijboom&amp;author=B.J.+Bouma&amp;volume=8&amp;publication_year=2011&amp;pages=50-60&amp;pmid=21045784&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C7">7. <span class="element-citation">van Beynum I.M., Kapusta L., den Heijer M., Vermeulen S.H., Kouwenberg M., Daniels O., Blom H.J. Maternal MTHFR 677C&gt;T is a risk factor for congenital heart defects: effect modification by periconceptional folate supplementation. <span><span class="ref-journal">Eur. Heart J. </span>2006;<span class="ref-vol">27</span>:981–987.</span> [<a href="/pubmed/16524890" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Heart+J.&amp;title=Maternal+MTHFR+677C>T+is+a+risk+factor+for+congenital+heart+defects:+effect+modification+by+periconceptional+folate+supplementation&amp;author=I.M.+van+Beynum&amp;author=L.+Kapusta&amp;author=M.+den+Heijer&amp;author=S.H.+Vermeulen&amp;author=M.+Kouwenberg&amp;volume=27&amp;publication_year=2006&amp;pages=981-987&amp;pmid=16524890&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C8">8. <span class="element-citation">Hobbs C.A., Cleves M.A., Karim M.A., Zhao W., MacLeod S.L. Maternal folate-related gene environment interactions and congenital heart defects. <span><span class="ref-journal">Obstet. Gynecol. </span>2010;<span class="ref-vol">116</span>:316–322.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3027124/">PMC free article</a>]</span> [<a href="/pubmed/20664391" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obstet.+Gynecol.&amp;title=Maternal+folate-related+gene+environment+interactions+and+congenital+heart+defects&amp;author=C.A.+Hobbs&amp;author=M.A.+Cleves&amp;author=M.A.+Karim&amp;author=W.+Zhao&amp;author=S.L.+MacLeod&amp;volume=116&amp;publication_year=2010&amp;pages=316-322&amp;pmid=20664391&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C9">9. <span class="element-citation">Hobbs C.A., James S.J., Jernigan S., Melnyk S., Lu Y., Malik S., Cleves M.A. Congenital heart defects, maternal homocysteine, smoking, and the 677 C&gt;T polymorphism in the methylenetetrahydrofolate reductase gene: evaluating gene-environment interactions. <span><span class="ref-journal">Am. J. Obstet. Gynecol. </span>2006;<span class="ref-vol">194</span>:218–224.</span> [<a href="/pubmed/16389035" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Obstet.+Gynecol.&amp;title=Congenital+heart+defects,+maternal+homocysteine,+smoking,+and+the+677+C>T+polymorphism+in+the+methylenetetrahydrofolate+reductase+gene:+evaluating+gene-environment+interactions&amp;author=C.A.+Hobbs&amp;author=S.J.+James&amp;author=S.+Jernigan&amp;author=S.+Melnyk&amp;author=Y.+Lu&amp;volume=194&amp;publication_year=2006&amp;pages=218-224&amp;pmid=16389035&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C10">10. <span class="element-citation">Lupo P.J., Canfield M.A., Chapa C., Lu W., Agopian A.J., Mitchell L.E., Shaw G.M., Waller D.K., Olshan A.F., Finnell R.H., et al.  Diabetes and obesity-related genes and the risk of neural tube defects in the national birth defects prevention study. <span><span class="ref-journal">Am. J. Epidemiol. </span>2012;<span class="ref-vol">176</span>:1101–1109.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3571234/">PMC free article</a>]</span> [<a href="/pubmed/23132673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Diabetes+and+obesity-related+genes+and+the+risk+of+neural+tube+defects+in+the+national+birth+defects+prevention+study&amp;author=P.J.+Lupo&amp;author=M.A.+Canfield&amp;author=C.+Chapa&amp;author=W.+Lu&amp;author=A.J.+Agopian&amp;volume=176&amp;publication_year=2012&amp;pages=1101-1109&amp;pmid=23132673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C11">11. <span class="element-citation">Botto L.D., Lin A.E., Riehle-Colarusso T., Malik S., Correa A. National Birth Defects Prevention Study. Seeking causes: Classifying and evaluating congenital heart defects in etiologic studies. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2007;<span class="ref-vol">79</span>:714–727.</span> [<a href="/pubmed/17729292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=Seeking+causes:+Classifying+and+evaluating+congenital+heart+defects+in+etiologic+studies&amp;author=L.D.+Botto&amp;author=A.E.+Lin&amp;author=T.+Riehle-Colarusso&amp;author=S.+Malik&amp;author=A.+Correa&amp;volume=79&amp;publication_year=2007&amp;pages=714-727&amp;pmid=17729292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C12">12. <span class="element-citation">Soemedi R., Wilson I.J., Bentham J., Darlay R., Topf A., Zelenika D., Cosgrove C., Setchfield K., Thornborough C., Granados-Riveron J., et al.  Contribution of global rare copy-number variants to the risk of sporadic congenital heart disease. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2012;<span class="ref-vol">91</span>:489–501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511986/">PMC free article</a>]</span> [<a href="/pubmed/22939634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Contribution+of+global+rare+copy-number+variants+to+the+risk+of+sporadic+congenital+heart+disease&amp;author=R.+Soemedi&amp;author=I.J.+Wilson&amp;author=J.+Bentham&amp;author=R.+Darlay&amp;author=A.+Topf&amp;volume=91&amp;publication_year=2012&amp;pages=489-501&amp;pmid=22939634&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C13">13. <span class="element-citation">Zaidi S., Choi M., Wakimoto H., Ma L., Jiang J., Overton J.D., Romano-Adesman A., Bjornson R.D., Breitbart R.E., Brown K.K., et al.  De novo mutations in histone-modifying genes in congenital heart disease. <span><span class="ref-journal">Nature. </span>2013;<span class="ref-vol">498</span>:220–223.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3706629/">PMC free article</a>]</span> [<a href="/pubmed/23665959" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=De+novo+mutations+in+histone-modifying+genes+in+congenital+heart+disease&amp;author=S.+Zaidi&amp;author=M.+Choi&amp;author=H.+Wakimoto&amp;author=L.+Ma&amp;author=J.+Jiang&amp;volume=498&amp;publication_year=2013&amp;pages=220-223&amp;pmid=23665959&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C14">14. <span class="element-citation">Hitz M.P., Lemieux-Perreault L.P., Marshall C., Feroz-Zada Y., Davies R., Yang S.W., Lionel A.C., D'Amours G., Lemyre E., Cullum R., et al.  Rare copy number variants contribute to congenital left-sided heart disease. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002903.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3435243/">PMC free article</a>]</span> [<a href="/pubmed/22969434" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Rare+copy+number+variants+contribute+to+congenital+left-sided+heart+disease&amp;author=M.P.+Hitz&amp;author=L.P.+Lemieux-Perreault&amp;author=C.+Marshall&amp;author=Y.+Feroz-Zada&amp;author=R.+Davies&amp;volume=8&amp;publication_year=2012&amp;pages=e1002903&amp;pmid=22969434&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C15">15. <span class="element-citation">Warburton D., Ronemus M., Kline J., Jobanputra V., Williams I., Anyane-Yeboa K., Chung W., Yu L., Wong N., Awad D., et al.  The contribution of de novo and rare inherited copy number changes to congenital heart disease in an unselected sample of children with conotruncal defects or hypoplastic left heart disease. <span><span class="ref-journal">Hum. Genet. </span>2014;<span class="ref-vol">133</span>:11–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3880624/">PMC free article</a>]</span> [<a href="/pubmed/23979609" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=The+contribution+of+de+novo+and+rare+inherited+copy+number+changes+to+congenital+heart+disease+in+an+unselected+sample+of+children+with+conotruncal+defects+or+hypoplastic+left+heart+disease&amp;author=D.+Warburton&amp;author=M.+Ronemus&amp;author=J.+Kline&amp;author=V.+Jobanputra&amp;author=I.+Williams&amp;volume=133&amp;publication_year=2014&amp;pages=11-27&amp;pmid=23979609&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C16">16. <span class="element-citation">Carey A.S., Liang L., Edwards J., Brandt T., Mei H., Sharp A.J., Hsu D.T., Newburger J.W., Ohye R.G., Chung W.K., et al.  Effect of copy number variants on outcomes for infants with single ventricle heart defects. <span><span class="ref-journal">Circ. Cardiovasc. Genet. </span>2013;<span class="ref-vol">6</span>:444–451.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3987966/">PMC free article</a>]</span> [<a href="/pubmed/24021551" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Cardiovasc.+Genet.&amp;title=Effect+of+copy+number+variants+on+outcomes+for+infants+with+single+ventricle+heart+defects&amp;author=A.S.+Carey&amp;author=L.+Liang&amp;author=J.+Edwards&amp;author=T.+Brandt&amp;author=H.+Mei&amp;volume=6&amp;publication_year=2013&amp;pages=444-451&amp;pmid=24021551&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C17">17. <span class="element-citation">McBride K.L., Pignatelli R., Lewin M., Ho T., Fernbach S., Menesses A., Lam W., Leal S.M., Kaplan N., Schliekelman P., et al.  Inheritance analysis of congenital left ventricular outflow tract obstruction malformations: segregation, multiplex relative risk, and heritability. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2005;<span class="ref-vol">134A</span>:180–186.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1361302/">PMC free article</a>]</span> [<a href="/pubmed/15690347" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=Inheritance+analysis+of+congenital+left+ventricular+outflow+tract+obstruction+malformations:+segregation,+multiplex+relative+risk,+and+heritability&amp;author=K.L.+McBride&amp;author=R.+Pignatelli&amp;author=M.+Lewin&amp;author=T.+Ho&amp;author=S.+Fernbach&amp;volume=134A&amp;publication_year=2005&amp;pages=180-186&amp;pmid=15690347&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C18">18. <span class="element-citation">McBride K.L., Zender G.A., Fitzgerald-Butt S.M., Koehler D., Menesses-Diaz A., Fernbach S., Lee K., Towbin J.A., Leal S., Belmont J.W. Linkage analysis of left ventricular outflow tract malformations (aortic valve stenosis, coarctation of the aorta, and hypoplastic left heart syndrome) <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2009;<span class="ref-vol">17</span>:811–819.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2916734/">PMC free article</a>]</span> [<a href="/pubmed/19142209" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Linkage+analysis+of+left+ventricular+outflow+tract+malformations+(aortic+valve+stenosis,+coarctation+of+the+aorta,+and+hypoplastic+left+heart+syndrome)&amp;author=K.L.+McBride&amp;author=G.A.+Zender&amp;author=S.M.+Fitzgerald-Butt&amp;author=D.+Koehler&amp;author=A.+Menesses-Diaz&amp;volume=17&amp;publication_year=2009&amp;pages=811-819&amp;pmid=19142209&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C19">19. <span class="element-citation">Hinton R.B., Martin L.J., Rame-Gowda S., Tabangin M.E., Cripe L.H., Benson D.W. Hypoplastic left heart syndrome links to chromosomes 10q and 6q and is genetically related to bicuspid aortic valve. <span><span class="ref-journal">J. Am. Coll. Cardiol. </span>2009;<span class="ref-vol">53</span>:1065–1071.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703749/">PMC free article</a>]</span> [<a href="/pubmed/19298921" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=Hypoplastic+left+heart+syndrome+links+to+chromosomes+10q+and+6q+and+is+genetically+related+to+bicuspid+aortic+valve&amp;author=R.B.+Hinton&amp;author=L.J.+Martin&amp;author=S.+Rame-Gowda&amp;author=M.E.+Tabangin&amp;author=L.H.+Cripe&amp;volume=53&amp;publication_year=2009&amp;pages=1065-1071&amp;pmid=19298921&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C20">20. <span class="element-citation">McBride K.L., Zender G.A., Fitzgerald-Butt S.M., Seagraves N.J., Fernbach S.D., Zapata G., Lewin M., Towbin J.A., Belmont J.W. Association of common variants in ERBB4 with congenital left ventricular outflow tract obstruction defects. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2011;<span class="ref-vol">91</span>:162–168.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3736588/">PMC free article</a>]</span> [<a href="/pubmed/21290564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=Association+of+common+variants+in+ERBB4+with+congenital+left+ventricular+outflow+tract+obstruction+defects&amp;author=K.L.+McBride&amp;author=G.A.+Zender&amp;author=S.M.+Fitzgerald-Butt&amp;author=N.J.+Seagraves&amp;author=S.D.+Fernbach&amp;volume=91&amp;publication_year=2011&amp;pages=162-168&amp;pmid=21290564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C21">21. <span class="element-citation">Boyles A.L., DeRoo L.A., Lie R.T., Taylor J.A., Jugessur A., Murray J.C., Wilcox A.J. Maternal alcohol consumption, alcohol metabolism genes, and the risk of oral clefts: a population-based case-control study in Norway, 1996–2001. <span><span class="ref-journal">Am. J. Epidemiol. </span>2010;<span class="ref-vol">172</span>:924–931.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2984244/">PMC free article</a>]</span> [<a href="/pubmed/20810466" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Maternal+alcohol+consumption,+alcohol+metabolism+genes,+and+the+risk+of+oral+clefts:+a+population-based+case-control+study+in+Norway,+1996–2001&amp;author=A.L.+Boyles&amp;author=L.A.+DeRoo&amp;author=R.T.+Lie&amp;author=J.A.+Taylor&amp;author=A.+Jugessur&amp;volume=172&amp;publication_year=2010&amp;pages=924-931&amp;pmid=20810466&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C22">22. <span class="element-citation">van der Zanden L.F., Galesloot T.E., Feitz W.F., Brouwers M.M., Shi M., Knoers N.V., Franke B., Roeleveld N., van Rooij I.A. Exploration of gene-environment interactions, maternal effects and parent of origin effects in the etiology of hypospadias. <span><span class="ref-journal">J. Urol. </span>2012;<span class="ref-vol">188</span>:2354–2360.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3602843/">PMC free article</a>]</span> [<a href="/pubmed/23088992" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Urol.&amp;title=Exploration+of+gene-environment+interactions,+maternal+effects+and+parent+of+origin+effects+in+the+etiology+of+hypospadias&amp;author=L.F.+van+der+Zanden&amp;author=T.E.+Galesloot&amp;author=W.F.+Feitz&amp;author=M.M.+Brouwers&amp;author=M.+Shi&amp;volume=188&amp;publication_year=2012&amp;pages=2354-2360&amp;pmid=23088992&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C23">23. <span class="element-citation">Luo Y.L., Cheng Y.L., Ye P., Wang W., Gao X.H., Chen Q. Association between MTHFR polymorphisms and orofacial clefts risk: a meta-analysis. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2012;<span class="ref-vol">94</span>:237–244.</span> [<a href="/pubmed/22450905" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=Association+between+MTHFR+polymorphisms+and+orofacial+clefts+risk:+a+meta-analysis&amp;author=Y.L.+Luo&amp;author=Y.L.+Cheng&amp;author=P.+Ye&amp;author=W.+Wang&amp;author=X.H.+Gao&amp;volume=94&amp;publication_year=2012&amp;pages=237-244&amp;pmid=22450905&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C24">24. <span class="element-citation">Ouyang S., Li Y., Liu Z., Chang H., Wu J. Association between MTR A2756G and MTRR A66G polymorphisms and maternal risk for neural tube defects: a meta-analysis. <span><span class="ref-journal">Gene. </span>2013;<span class="ref-vol">515</span>:308–312.</span> [<a href="/pubmed/23266814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=Association+between+MTR+A2756G+and+MTRR+A66G+polymorphisms+and+maternal+risk+for+neural+tube+defects:+a+meta-analysis&amp;author=S.+Ouyang&amp;author=Y.+Li&amp;author=Z.+Liu&amp;author=H.+Chang&amp;author=J.+Wu&amp;volume=515&amp;publication_year=2013&amp;pages=308-312&amp;pmid=23266814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C25">25. <span class="element-citation">Buyske S. Maternal genotype effects can alias case genotype effects in case-control studies. <span><span class="ref-journal">Eur. J. Hum. Genet. </span>2008;<span class="ref-vol">16</span>:783–785.</span> [<a href="/pubmed/18398431" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Maternal+genotype+effects+can+alias+case+genotype+effects+in+case-control+studies&amp;author=S.+Buyske&amp;volume=16&amp;publication_year=2008&amp;pages=783-785&amp;pmid=18398431&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C26">26. <span class="element-citation">Wilcox A.J., Weinberg C.R., Lie R.T. Distinguishing the effects of maternal and offspring genes through studies of “case-parent triads” <span><span class="ref-journal">Am. J. Epidemiol. </span>1998;<span class="ref-vol">148</span>:893–901.</span> [<a href="/pubmed/9801020" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Distinguishing+the+effects+of+maternal+and+offspring+genes+through+studies+of+“case-parent+triads”&amp;author=A.J.+Wilcox&amp;author=C.R.+Weinberg&amp;author=R.T.+Lie&amp;volume=148&amp;publication_year=1998&amp;pages=893-901&amp;pmid=9801020&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C27">27. <span class="element-citation">Lee P.H., O'Dushlaine C., Thomas B., Purcell S.M. INRICH: interval-based enrichment analysis for genome-wide association studies. <span><span class="ref-journal">Bioinformatics. </span>2012;<span class="ref-vol">28</span>:1797–1799.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3381960/">PMC free article</a>]</span> [<a href="/pubmed/22513993" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=INRICH:+interval-based+enrichment+analysis+for+genome-wide+association+studies&amp;author=P.H.+Lee&amp;author=C.+O'Dushlaine&amp;author=B.+Thomas&amp;author=S.M.+Purcell&amp;volume=28&amp;publication_year=2012&amp;pages=1797-1799&amp;pmid=22513993&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C28">28. <span class="element-citation">Zhong Z., Wan B., Qiu Y., Ni J., Tang W., Chen X., Yang Y., Shen S., Wang Y., Bai M., et al.  Identification of a novel human zinc finger gene, ZNF438, with transcription inhibition activity. <span><span class="ref-journal">J. Biochem. Mol. Biol. </span>2007;<span class="ref-vol">40</span>:517–524.</span> [<a href="/pubmed/17669267" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biochem.+Mol.+Biol.&amp;title=Identification+of+a+novel+human+zinc+finger+gene,+ZNF438,+with+transcription+inhibition+activity&amp;author=Z.+Zhong&amp;author=B.+Wan&amp;author=Y.+Qiu&amp;author=J.+Ni&amp;author=W.+Tang&amp;volume=40&amp;publication_year=2007&amp;pages=517-524&amp;pmid=17669267&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C29">29. <span class="element-citation">Zhang K., Gao R., Zhang H., Cai X., Shen C., Wu C., Zhao S., Yu L. Molecular cloning and characterization of three novel lysozyme-like genes, predominantly expressed in the male reproductive system of humans, belonging to the c-type lysozyme/alpha-lactalbumin family. <span><span class="ref-journal">Biol. Reprod. </span>2005;<span class="ref-vol">73</span>:1064–1071.</span> [<a href="/pubmed/16014814" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Reprod.&amp;title=Molecular+cloning+and+characterization+of+three+novel+lysozyme-like+genes,+predominantly+expressed+in+the+male+reproductive+system+of+humans,+belonging+to+the+c-type+lysozyme/alpha-lactalbumin+family&amp;author=K.+Zhang&amp;author=R.+Gao&amp;author=H.+Zhang&amp;author=X.+Cai&amp;author=C.+Shen&amp;volume=73&amp;publication_year=2005&amp;pages=1064-1071&amp;pmid=16014814&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C30">30. <span class="element-citation">Dai J., Ji C., Gu S., Wu Q., Wang L., Xu J., Zeng L., Ye X., Yin G., Xie Y., et al.  Cloning and sequence analysis of the human cDNA encoding the synaptoporin (delta), a highly conservative synaptic vesicle protein. <span><span class="ref-journal">Mol. Biol. Rep. </span>2003;<span class="ref-vol">30</span>:185–191.</span> [<a href="/pubmed/12974474" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Rep.&amp;title=Cloning+and+sequence+analysis+of+the+human+cDNA+encoding+the+synaptoporin+(delta),+a+highly+conservative+synaptic+vesicle+protein&amp;author=J.+Dai&amp;author=C.+Ji&amp;author=S.+Gu&amp;author=Q.+Wu&amp;author=L.+Wang&amp;volume=30&amp;publication_year=2003&amp;pages=185-191&amp;pmid=12974474&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C31">31. <span class="element-citation">Barth A.S., Merk S., Arnoldi E., Zwermann L., Kloos P., Gebauer M., Steinmeyer K., Bleich M., Kaab S., Pfeufer A., et al.  Functional profiling of human atrial and ventricular gene expression. <span><span class="ref-journal">Pflugers Arch. </span>2005;<span class="ref-vol">450</span>:201–208.</span> [<a href="/pubmed/15877233" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pflugers+Arch.&amp;title=Functional+profiling+of+human+atrial+and+ventricular+gene+expression&amp;author=A.S.+Barth&amp;author=S.+Merk&amp;author=E.+Arnoldi&amp;author=L.+Zwermann&amp;author=P.+Kloos&amp;volume=450&amp;publication_year=2005&amp;pages=201-208&amp;pmid=15877233&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C32">32. <span class="element-citation">Seo S., Kume T. Forkhead transcription factors, Foxc1 and Foxc2, are required for the morphogenesis of the cardiac outflow tract. <span><span class="ref-journal">Dev. Biol. </span>2006;<span class="ref-vol">296</span>:421–436.</span> [<a href="/pubmed/16839542" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dev.+Biol.&amp;title=Forkhead+transcription+factors,+Foxc1+and+Foxc2,+are+required+for+the+morphogenesis+of+the+cardiac+outflow+tract&amp;author=S.+Seo&amp;author=T.+Kume&amp;volume=296&amp;publication_year=2006&amp;pages=421-436&amp;pmid=16839542&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C33">33. <span class="element-citation">Shaw-Smith C. Genetic factors in esophageal atresia, tracheo-esophageal fistula and the VACTERL association: roles for FOXF1 and the 16q24.1 FOX transcription factor gene cluster, and review of the literature. <span><span class="ref-journal">Eur. J. Med. Genet. </span>2010;<span class="ref-vol">53</span>:6–13.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2809919/">PMC free article</a>]</span> [<a href="/pubmed/19822228" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Med.+Genet.&amp;title=Genetic+factors+in+esophageal+atresia,+tracheo-esophageal+fistula+and+the+VACTERL+association:+roles+for+FOXF1+and+the+16q24.1+FOX+transcription+factor+gene+cluster,+and+review+of+the+literature&amp;author=C.+Shaw-Smith&amp;volume=53&amp;publication_year=2010&amp;pages=6-13&amp;pmid=19822228&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C34">34. <span class="element-citation">Stankiewicz P., Sen P., Bhatt S.S., Storer M., Xia Z., Bejjani B.A., Ou Z., Wiszniewska J., Driscoll D.J., Maisenbacher M.K., et al.  Genomic and genic deletions of the FOX gene cluster on 16q24.1 and inactivating mutations of FOXF1 cause alveolar capillary dysplasia and other malformations. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2009;<span class="ref-vol">84</span>:780–791.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694971/">PMC free article</a>]</span> [<a href="/pubmed/19500772" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Genomic+and+genic+deletions+of+the+FOX+gene+cluster+on+16q24.1+and+inactivating+mutations+of+FOXF1+cause+alveolar+capillary+dysplasia+and+other+malformations&amp;author=P.+Stankiewicz&amp;author=P.+Sen&amp;author=S.S.+Bhatt&amp;author=M.+Storer&amp;author=Z.+Xia&amp;volume=84&amp;publication_year=2009&amp;pages=780-791&amp;pmid=19500772&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C35">35. <span class="element-citation">Iascone M., Ciccone R., Galletti L., Marchetti D., Seddio F., Lincesso A.R., Pezzoli L., Vetro A., Barachetti D., Boni L., et al.  Identification of de novo mutations and rare variants in hypoplastic left heart syndrome. <span><span class="ref-journal">Clin. Genet. </span>2012;<span class="ref-vol">81</span>:542–554.</span> [<a href="/pubmed/21457232" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Genet.&amp;title=Identification+of+de+novo+mutations+and+rare+variants+in+hypoplastic+left+heart+syndrome&amp;author=M.+Iascone&amp;author=R.+Ciccone&amp;author=L.+Galletti&amp;author=D.+Marchetti&amp;author=F.+Seddio&amp;volume=81&amp;publication_year=2012&amp;pages=542-554&amp;pmid=21457232&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C36">36. <span class="element-citation">McCulley D.J., Black B.L. Transcription factor pathways and congenital heart disease. <span><span class="ref-journal">Curr. Top. Dev. Biol. </span>2012;<span class="ref-vol">100</span>:253–277.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3684448/">PMC free article</a>]</span> [<a href="/pubmed/22449847" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Top.+Dev.+Biol.&amp;title=Transcription+factor+pathways+and+congenital+heart+disease&amp;author=D.J.+McCulley&amp;author=B.L.+Black&amp;volume=100&amp;publication_year=2012&amp;pages=253-277&amp;pmid=22449847&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C37">37. <span class="element-citation">Schott J.J., Benson D.W., Basson C.T., Pease W., Silberbach G.M., Moak J.P., Maron B.J., Seidman C.E., Seidman J.G. Congenital heart disease caused by mutations in the transcription factor NKX2–5. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">281</span>:108–111.</span> [<a href="/pubmed/9651244" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Congenital+heart+disease+caused+by+mutations+in+the+transcription+factor+NKX2–5&amp;author=J.J.+Schott&amp;author=D.W.+Benson&amp;author=C.T.+Basson&amp;author=W.+Pease&amp;author=G.M.+Silberbach&amp;volume=281&amp;publication_year=1998&amp;pages=108-111&amp;pmid=9651244&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C38">38. <span class="element-citation">Polen K.N., Rasmussen S.A., Riehle-Colarusso T., Reefhuis J. National Birth Defects Prevention Study. Association between reported venlafaxine use in early pregnancy and birth defects, national birth defects prevention study, 1997–2007. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2013;<span class="ref-vol">97</span>:28–35.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4484721/">PMC free article</a>]</span> [<a href="/pubmed/23281074" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=Association+between+reported+venlafaxine+use+in+early+pregnancy+and+birth+defects,+national+birth+defects+prevention+study,+1997–2007&amp;author=K.N.+Polen&amp;author=S.A.+Rasmussen&amp;author=T.+Riehle-Colarusso&amp;author=J.+Reefhuis&amp;volume=97&amp;publication_year=2013&amp;pages=28-35&amp;pmid=23281074&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C39">39. <span class="element-citation">Broussard C.S., Rasmussen S.A., Reefhuis J., Friedman J.M., Jann M.W., Riehle-Colarusso T., Honein M.A. National Birth Defects Prevention Study. Maternal treatment with opioid analgesics and risk for birth defects. <span><span class="ref-journal">Am. J. Obstet. Gynecol. </span>2011;<span class="ref-vol">204</span>:314.</span> e311–311. [<a href="/pubmed/21345403" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Obstet.+Gynecol.&amp;title=Maternal+treatment+with+opioid+analgesics+and+risk+for+birth+defects&amp;author=C.S.+Broussard&amp;author=S.A.+Rasmussen&amp;author=J.+Reefhuis&amp;author=J.M.+Friedman&amp;author=M.W.+Jann&amp;volume=204&amp;publication_year=2011&amp;pages=314&amp;pmid=21345403&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C40">40. <span class="element-citation">Alwan S., Reefhuis J., Botto L.D., Rasmussen S.A., Correa A., Friedman J.M. National Birth Defects Prevention Study. Maternal use of bupropion and risk for congenital heart defects. <span><span class="ref-journal">Am. J. Obstet. Gynecol. </span>2010;<span class="ref-vol">203</span>:52.</span> e51–56. [<a href="/pubmed/20417496" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Obstet.+Gynecol.&amp;title=Maternal+use+of+bupropion+and+risk+for+congenital+heart+defects&amp;author=S.+Alwan&amp;author=J.+Reefhuis&amp;author=L.D.+Botto&amp;author=S.A.+Rasmussen&amp;author=A.+Correa&amp;volume=203&amp;publication_year=2010&amp;pages=52&amp;pmid=20417496&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C41">41. <span class="element-citation">Gilboa S.M., Correa A., Botto L.D., Rasmussen S.A., Waller D.K., Hobbs C.A., Cleves M.A., Riehle-Colarusso T.J. National Birth Defects Prevention Study. Association between prepregnancy body mass index and congenital heart defects. <span><span class="ref-journal">Am. J. Obstet. Gynecol. </span>2010;<span class="ref-vol">202</span>:51.</span> e51–51 e10. [<a href="/pubmed/19796755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Obstet.+Gynecol.&amp;title=Association+between+prepregnancy+body+mass+index+and+congenital+heart+defects&amp;author=S.M.+Gilboa&amp;author=A.+Correa&amp;author=L.D.+Botto&amp;author=S.A.+Rasmussen&amp;author=D.K.+Waller&amp;volume=202&amp;publication_year=2010&amp;pages=51&amp;pmid=19796755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C42">42. <span class="element-citation">Cordell H.J., Topf A., Mamasoula C., Postma A.V., Bentham J., Zelenika D., Heath S., Blue G., Cosgrove C., Granados Riveron J., et al.  Genome-wide association study identifies loci on 12q24 and 13q32 associated with tetralogy of Fallot. <span><span class="ref-journal">Hum. Mol. Genet. </span>2013;<span class="ref-vol">22</span>:1473–1481.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3596849/">PMC free article</a>]</span> [<a href="/pubmed/23297363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Genome-wide+association+study+identifies+loci+on+12q24+and+13q32+associated+with+tetralogy+of+Fallot&amp;author=H.J.+Cordell&amp;author=A.+Topf&amp;author=C.+Mamasoula&amp;author=A.V.+Postma&amp;author=J.+Bentham&amp;volume=22&amp;publication_year=2013&amp;pages=1473-1481&amp;pmid=23297363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C43">43. <span class="element-citation">Hu Z., Shi Y., Mo X., Xu J., Zhao B., Lin Y., Yang S., Xu Z., Dai J., Pan S., et al.  A genome-wide association study identifies two risk loci for congenital heart malformations in Han Chinese populations. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:818–821.</span> [<a href="/pubmed/23708190" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+genome-wide+association+study+identifies+two+risk+loci+for+congenital+heart+malformations+in+Han+Chinese+populations&amp;author=Z.+Hu&amp;author=Y.+Shi&amp;author=X.+Mo&amp;author=J.+Xu&amp;author=B.+Zhao&amp;volume=45&amp;publication_year=2013&amp;pages=818-821&amp;pmid=23708190&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C44">44. <span class="element-citation">Cordell H.J., Bentham J., Topf A., Zelenika D., Heath S., Mamasoula C., Cosgrove C., Blue G., Granados-Riveron J., Setchfield K., et al.  Genome-wide association study of multiple congenital heart disease phenotypes identifies a susceptibility locus for atrial septal defect at chromosome 4p16. <span><span class="ref-journal">Nat. Genet. </span>2013;<span class="ref-vol">45</span>:822–824.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3793630/">PMC free article</a>]</span> [<a href="/pubmed/23708191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+of+multiple+congenital+heart+disease+phenotypes+identifies+a+susceptibility+locus+for+atrial+septal+defect+at+chromosome+4p16&amp;author=H.J.+Cordell&amp;author=J.+Bentham&amp;author=A.+Topf&amp;author=D.+Zelenika&amp;author=S.+Heath&amp;volume=45&amp;publication_year=2013&amp;pages=822-824&amp;pmid=23708191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C45">45. <span class="element-citation">Hakonarson H., Grant S.F., Bradfield J.P., Marchand L., Kim C.E., Glessner J.T., Grabs R., Casalunovo T., Taback S.P., Frackelton E.C., et al.  A genome-wide association study identifies KIAA0350 as a type 1 diabetes gene. <span><span class="ref-journal">Nature. </span>2007;<span class="ref-vol">448</span>:591–594.</span> [<a href="/pubmed/17632545" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+KIAA0350+as+a+type+1+diabetes+gene&amp;author=H.+Hakonarson&amp;author=S.F.+Grant&amp;author=J.P.+Bradfield&amp;author=L.+Marchand&amp;author=C.E.+Kim&amp;volume=448&amp;publication_year=2007&amp;pages=591-594&amp;pmid=17632545&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C46">46. <span class="element-citation">Li Y., Abecasis G. MACH 1.0: rapid haplotype reconstruction and missing genotype inference. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2006;<span class="ref-vol">S79</span>:2290.</span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=MACH+1.0:+rapid+haplotype+reconstruction+and+missing+genotype+inference&amp;author=Y.+Li&amp;author=G.+Abecasis&amp;volume=S79&amp;publication_year=2006&amp;pages=2290&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C47">47. <span class="element-citation">Zhu H., Kartiko S., Finnell R.H. Importance of gene-environment interactions in the etiology of selected birth defects. <span><span class="ref-journal">Clin. Genet. </span>2009;<span class="ref-vol">75</span>:409–423.</span> [<a href="/pubmed/19459879" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Genet.&amp;title=Importance+of+gene-environment+interactions+in+the+etiology+of+selected+birth+defects&amp;author=H.+Zhu&amp;author=S.+Kartiko&amp;author=R.H.+Finnell&amp;volume=75&amp;publication_year=2009&amp;pages=409-423&amp;pmid=19459879&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C48">48. <span class="element-citation">Weinberg C.R., Wilcox A.J., Lie R.T. A log-linear approach to case-parent-triad data: assessing effects of disease genes that act either directly or through maternal effects and that may be subject to parental imprinting. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1998;<span class="ref-vol">62</span>:969–978.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377041/">PMC free article</a>]</span> [<a href="/pubmed/9529360" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=A+log-linear+approach+to+case-parent-triad+data:+assessing+effects+of+disease+genes+that+act+either+directly+or+through+maternal+effects+and+that+may+be+subject+to+parental+imprinting&amp;author=C.R.+Weinberg&amp;author=A.J.+Wilcox&amp;author=R.T.+Lie&amp;volume=62&amp;publication_year=1998&amp;pages=969-978&amp;pmid=9529360&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C49">49. <span class="element-citation">Agopian A.J., Mitchell L.E. MI-GWAS: a SAS platform for the analysis of inherited and maternal genetic effects in genome-wide association studies using log-linear models. <span><span class="ref-journal">BMC Bioinformatics. </span>2011;<span class="ref-vol">12</span>:117.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110146/">PMC free article</a>]</span> [<a href="/pubmed/21513519" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=MI-GWAS:+a+SAS+platform+for+the+analysis+of+inherited+and+maternal+genetic+effects+in+genome-wide+association+studies+using+log-linear+models&amp;author=A.J.+Agopian&amp;author=L.E.+Mitchell&amp;volume=12&amp;publication_year=2011&amp;pages=117&amp;pmid=21513519&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C50">50. <span class="element-citation">Doolin M.T., Barbaux S., McDonnell M., Hoess K., Whitehead A.S., Mitchell L.E. Maternal genetic effects, exerted by genes involved in homocysteine remethylation, influence the risk of spina bifida. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2002;<span class="ref-vol">71</span>:1222–1226.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC385102/">PMC free article</a>]</span> [<a href="/pubmed/12375236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Maternal+genetic+effects,+exerted+by+genes+involved+in+homocysteine+remethylation,+influence+the+risk+of+spina+bifida&amp;author=M.T.+Doolin&amp;author=S.+Barbaux&amp;author=M.+McDonnell&amp;author=K.+Hoess&amp;author=A.S.+Whitehead&amp;volume=71&amp;publication_year=2002&amp;pages=1222-1226&amp;pmid=12375236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C51">51. <span class="element-citation">Jensen L.E., Wall A.M., Cook M., Hoess K., Thorn C.F., Whitehead A.S., Mitchell L.E. A common ABCC2 promoter polymorphism is not a determinant of the risk of spina bifida. <span><span class="ref-journal">Birth Defects Res. A Clin. Mol. Teratol. </span>2004;<span class="ref-vol">70</span>:396–399.</span> [<a href="/pubmed/15211708" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Birth+Defects+Res.+A+Clin.+Mol.+Teratol.&amp;title=A+common+ABCC2+promoter+polymorphism+is+not+a+determinant+of+the+risk+of+spina+bifida&amp;author=L.E.+Jensen&amp;author=A.M.+Wall&amp;author=M.+Cook&amp;author=K.+Hoess&amp;author=C.F.+Thorn&amp;volume=70&amp;publication_year=2004&amp;pages=396-399&amp;pmid=15211708&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C52">52. <span class="element-citation">Hancock D.B., Romieu I., Shi M., Sienra-Monge J.J., Wu H., Chiu G.Y., Li H., del Rio-Navarro B.E., Willis-Owen S.A., Weiss S.T., et al.  Genome-wide association study implicates chromosome 9q21.31 as a susceptibility locus for asthma in Mexican children. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000623.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722731/">PMC free article</a>]</span> [<a href="/pubmed/19714205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Genome-wide+association+study+implicates+chromosome+9q21.31+as+a+susceptibility+locus+for+asthma+in+Mexican+children&amp;author=D.B.+Hancock&amp;author=I.+Romieu&amp;author=M.+Shi&amp;author=J.J.+Sienra-Monge&amp;author=H.+Wu&amp;volume=5&amp;publication_year=2009&amp;pages=e1000623&amp;pmid=19714205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C53">53. <span class="element-citation">James S.J., Melnyk S., Jernigan S., Pavliv O., Trusty T., Lehman S., Seidel L., Gaylor D.W., Cleves M.A. A functional polymorphism in the reduced folate carrier gene and DNA hypomethylation in mothers of children with autism. <span><span class="ref-journal">Am. J. Med. Genet. B Neuropsychiatr. Genet. </span>2010;<span class="ref-vol">153B</span>:1209–1220.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2943349/">PMC free article</a>]</span> [<a href="/pubmed/20468076" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+B+Neuropsychiatr.+Genet.&amp;title=A+functional+polymorphism+in+the+reduced+folate+carrier+gene+and+DNA+hypomethylation+in+mothers+of+children+with+autism&amp;author=S.J.+James&amp;author=S.+Melnyk&amp;author=S.+Jernigan&amp;author=O.+Pavliv&amp;author=T.+Trusty&amp;volume=153B&amp;publication_year=2010&amp;pages=1209-1220&amp;pmid=20468076&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C54">54. <span class="element-citation">Martinelli M., Masiero E., Carinci F., Morselli P.G., Pezzetti F., Scapoli L. New evidence for the role of cystathionine beta-synthase in non-syndromic cleft lip with or without cleft palate. <span><span class="ref-journal">Eur. J. Oral Sci. </span>2011;<span class="ref-vol">119</span>:193–197.</span> [<a href="/pubmed/21564312" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Oral+Sci.&amp;title=New+evidence+for+the+role+of+cystathionine+beta-synthase+in+non-syndromic+cleft+lip+with+or+without+cleft+palate&amp;author=M.+Martinelli&amp;author=E.+Masiero&amp;author=F.+Carinci&amp;author=P.G.+Morselli&amp;author=F.+Pezzetti&amp;volume=119&amp;publication_year=2011&amp;pages=193-197&amp;pmid=21564312&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C55">55. <span class="element-citation">Lupo P.J., Nousome D., Okcu M.F., Chintagumpala M., Scheurer M.E. Maternal variation in EPHX1, a xenobiotic metabolism gene, is associated with childhood medulloblastoma: an exploratory case-parent triad study. <span><span class="ref-journal">Pediatr. Hematol. Oncol. </span>2012;<span class="ref-vol">29</span>:679–685.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3855527/">PMC free article</a>]</span> [<a href="/pubmed/22994552" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Hematol.+Oncol.&amp;title=Maternal+variation+in+EPHX1,+a+xenobiotic+metabolism+gene,+is+associated+with+childhood+medulloblastoma:+an+exploratory+case-parent+triad+study&amp;author=P.J.+Lupo&amp;author=D.+Nousome&amp;author=M.F.+Okcu&amp;author=M.+Chintagumpala&amp;author=M.E.+Scheurer&amp;volume=29&amp;publication_year=2012&amp;pages=679-685&amp;pmid=22994552&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C56">56. <span class="element-citation">Shi M., Murray J.C., Marazita M.L., Munger R.G., Ruczinski I., Hetmanski J.B., Wu T., Murray T., Redett R.J., Wilcox A.J., et al.  Genome wide study of maternal and parent-of-origin effects on the etiology of orofacial clefts. <span><span class="ref-journal">Am. J. Med. Genet. A. </span>2012;<span class="ref-vol">158A</span>:784–794.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3617127/">PMC free article</a>]</span> [<a href="/pubmed/22419666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.+A&amp;title=Genome+wide+study+of+maternal+and+parent-of-origin+effects+on+the+etiology+of+orofacial+clefts&amp;author=M.+Shi&amp;author=J.C.+Murray&amp;author=M.L.+Marazita&amp;author=R.G.+Munger&amp;author=I.+Ruczinski&amp;volume=158A&amp;publication_year=2012&amp;pages=784-794&amp;pmid=22419666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C57">57. <span class="element-citation">Weinberg C.R. Allowing for missing parents in genetic studies of case-parent triads. <span><span class="ref-journal">Am. J. Hum. Genet. </span>1999;<span class="ref-vol">64</span>:1186–1193.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1377843/">PMC free article</a>]</span> [<a href="/pubmed/10090904" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=Allowing+for+missing+parents+in+genetic+studies+of+case-parent+triads&amp;author=C.R.+Weinberg&amp;volume=64&amp;publication_year=1999&amp;pages=1186-1193&amp;pmid=10090904&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C58">58. <span class="element-citation">Mitchell L.E., Weinberg C.R. Evaluation of offspring and maternal genetic effects on disease risk using a family-based approach: the “pent” design. <span><span class="ref-journal">Am. J. Epidemiol. </span>2005;<span class="ref-vol">162</span>:676–685.</span> [<a href="/pubmed/16093287" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Epidemiol.&amp;title=Evaluation+of+offspring+and+maternal+genetic+effects+on+disease+risk+using+a+family-based+approach:+the+“pent”+design&amp;author=L.E.+Mitchell&amp;author=C.R.+Weinberg&amp;volume=162&amp;publication_year=2005&amp;pages=676-685&amp;pmid=16093287&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C59">59. <span class="element-citation">James J.W. Frequency in relatives for an all-or-none trait. <span><span class="ref-journal">Ann. Hum. Genet. </span>1971;<span class="ref-vol">35</span>:47–49.</span> [<a href="/pubmed/5106369" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet.&amp;title=Frequency+in+relatives+for+an+all-or-none+trait&amp;author=J.W.+James&amp;volume=35&amp;publication_year=1971&amp;pages=47-49&amp;pmid=5106369&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C60">60. <span class="element-citation">Ritchie G.R., Dunham I., Zeggini E., Flicek P. Functional annotation of noncoding sequence variants. <span><span class="ref-journal">Nat. Methods. </span>2014;<span class="ref-vol">11</span>:294–296.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5015703/">PMC free article</a>]</span> [<a href="/pubmed/24487584" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Functional+annotation+of+noncoding+sequence+variants&amp;author=G.R.+Ritchie&amp;author=I.+Dunham&amp;author=E.+Zeggini&amp;author=P.+Flicek&amp;volume=11&amp;publication_year=2014&amp;pages=294-296&amp;pmid=24487584&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU420C61">61. <span class="element-citation">Yang T.P., Beazley C., Montgomery S.B., Dimas A.S., Gutierrez-Arcelus M., Stranger B.E., Deloukas P., Dermitzakis E.T. Genevar: a database and Java application for the analysis and visualization of SNP-gene associations in eQTL studies. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2474–2476.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2944204/">PMC free article</a>]</span> [<a href="/pubmed/20702402" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Genevar:+a+database+and+Java+application+for+the+analysis+and+visualization+of+SNP-gene+associations+in+eQTL+studies&amp;author=T.P.+Yang&amp;author=C.+Beazley&amp;author=S.B.+Montgomery&amp;author=A.S.+Dimas&amp;author=M.+Gutierrez-Arcelus&amp;volume=26&amp;publication_year=2010&amp;pages=2474-2476&amp;pmid=20702402&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4326325&amp;issue-id=246385&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140441332954272title">Abstract</a></li><li><a href="#s1title">INTRODUCTION</a></li><li><a href="#s2title">RESULTS</a></li><li><a href="#s3title">DISCUSSION</a></li><li><a href="#s4title">MATERIALS AND METHODS</a></li><li><a href="#s5title">SUPPLEMENTARY MATERIAL</a></li><li><a href="#s6title">FUNDING</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140441374744016title">ACKNOWLEDGEMENTS</a></li><li><a href="#idm140441368490608title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4326325/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4326325/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4326325/pdf/ddu420.pdf">PDF (369K)</a></li> | <li><a href="#" data-citationid="PMC4326325" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4326325%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4326325%2F&amp;text=Genome-wide%20association%20study%20of%20maternal%20and%20inherited%20effects%20on%20left-sided%20cardiac%20malformations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4326325%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25138779" data-aiid="4326325" data-aid="4326325" data-iid="246385" data-domainid="740" data-domain="hmg" data-accid="PMC4326325" data-md5="2ecf062bfb5fb7c8222632b0446f7cc8"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T12:35:17-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE899F34DD5775F100000000015200C1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4326325%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4326325/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>